// @react-spectrum/s2/style/dist/main.cjs
var $53f7a8c49f8c6ec9$exports = require("./spectrum-theme.cjs");
var $cf0b57705fca33e1$exports = require("./style-macro.cjs");
var $247833060df99ecf$exports = require("./style-utils.cjs");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "space", () => $601c1b90569d4e66$export$a941ed4b947d12f8);
$parcel$export(module.exports, "fontRelative", () => $601c1b90569d4e66$export$ba024f4caf693d15);
$parcel$export(module.exports, "focusRing", () => $601c1b90569d4e66$export$8e24544ddba725ee);
$parcel$export(module.exports, "iconStyle", () => $601c1b90569d4e66$export$dcc3981cc6efe940);
$parcel$export(module.exports, "baseColor", () => $53f7a8c49f8c6ec9$exports.baseColor);
$parcel$export(module.exports, "color", () => $53f7a8c49f8c6ec9$exports.color);
$parcel$export(module.exports, "lightDark", () => $53f7a8c49f8c6ec9$exports.lightDark);
$parcel$export(module.exports, "colorMix", () => $53f7a8c49f8c6ec9$exports.colorMix);
$parcel$export(module.exports, "size", () => $53f7a8c49f8c6ec9$exports.size);
$parcel$export(module.exports, "style", () => $53f7a8c49f8c6ec9$exports.style);
$parcel$export(module.exports, "css", () => $cf0b57705fca33e1$exports.css);
$parcel$export(module.exports, "centerPadding", () => $247833060df99ecf$exports.centerPadding);
$parcel$export(module.exports, "setColorScheme", () => $247833060df99ecf$exports.setColorScheme);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 


function $601c1b90569d4e66$export$a941ed4b947d12f8(px) {
    return `[${(0, $53f7a8c49f8c6ec9$exports.space)(px)}]`;
}
function $601c1b90569d4e66$export$ba024f4caf693d15(base, baseFontSize = 14) {
    return `[${(0, $53f7a8c49f8c6ec9$exports.fontRelative)(base, baseFontSize)}]`;
}
const $601c1b90569d4e66$export$8e24544ddba725ee = ()=>({
        outlineStyle: {
            default: 'none',
            isFocusVisible: 'solid'
        },
        outlineColor: {
            default: 'focus-ring',
            forcedColors: 'Highlight'
        },
        outlineWidth: 2,
        outlineOffset: 2
    });
const $601c1b90569d4e66$var$iconSizes = {
    XS: 14,
    S: 16,
    M: 20,
    L: 22,
    XL: 26
};
function $601c1b90569d4e66$export$dcc3981cc6efe940(options) {
    let { size: size = 'M', color: color, ...styles } = options;
    if (color) styles['--iconPrimary'] = {
        type: 'fill',
        value: color
    };
    styles['size'] = $601c1b90569d4e66$var$iconSizes[size];
    // @ts-ignore
    return (0, $53f7a8c49f8c6ec9$exports.style).call(this, styles);
}
