// @react-spectrum/s2/style/dist/module.mjs
import {baseColor as $9440fa3fdd7adbfe$export$49bbad2a916ab57c, color as $9440fa3fdd7adbfe$export$35e9368ef982300f, colorMix as $9440fa3fdd7adbfe$export$9b476054b78b89cd, fontRelative as $9440fa3fdd7adbfe$export$ba024f4caf693d15, lightDark as $9440fa3fdd7adbfe$export$d19e682d444755ed, size as $9440fa3fdd7adbfe$export$346677f925de839c, space as $9440fa3fdd7adbfe$export$a941ed4b947d12f8, style as $9440fa3fdd7adbfe$export$1d567c320f4763bc} from "./spectrum-theme.mjs";
import {css as $e3c5b37c91268204$export$dbf350e5966cf602} from "./style-macro.mjs";
import {centerPadding as $fec05d0b605a7a01$export$dc7aad089b8f5c30, setColorScheme as $fec05d0b605a7a01$export$f2d00fe8ce80fd84} from "./style-utils.mjs";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 


function $4ac2adb5febfc8ac$export$a941ed4b947d12f8(px) {
    return `[${(0, $9440fa3fdd7adbfe$export$a941ed4b947d12f8)(px)}]`;
}
function $4ac2adb5febfc8ac$export$ba024f4caf693d15(base, baseFontSize = 14) {
    return `[${(0, $9440fa3fdd7adbfe$export$ba024f4caf693d15)(base, baseFontSize)}]`;
}
const $4ac2adb5febfc8ac$export$8e24544ddba725ee = ()=>({
        outlineStyle: {
            default: 'none',
            isFocusVisible: 'solid'
        },
        outlineColor: {
            default: 'focus-ring',
            forcedColors: 'Highlight'
        },
        outlineWidth: 2,
        outlineOffset: 2
    });
const $4ac2adb5febfc8ac$var$iconSizes = {
    XS: 14,
    S: 16,
    M: 20,
    L: 22,
    XL: 26
};
function $4ac2adb5febfc8ac$export$dcc3981cc6efe940(options) {
    let { size: size = 'M', color: color, ...styles } = options;
    if (color) styles['--iconPrimary'] = {
        type: 'fill',
        value: color
    };
    styles['size'] = $4ac2adb5febfc8ac$var$iconSizes[size];
    // @ts-ignore
    return (0, $9440fa3fdd7adbfe$export$1d567c320f4763bc).call(this, styles);
}


export {$4ac2adb5febfc8ac$export$a941ed4b947d12f8 as space, $4ac2adb5febfc8ac$export$ba024f4caf693d15 as fontRelative, $4ac2adb5febfc8ac$export$8e24544ddba725ee as focusRing, $4ac2adb5febfc8ac$export$dcc3981cc6efe940 as iconStyle, $9440fa3fdd7adbfe$export$49bbad2a916ab57c as baseColor, $9440fa3fdd7adbfe$export$35e9368ef982300f as color, $9440fa3fdd7adbfe$export$d19e682d444755ed as lightDark, $9440fa3fdd7adbfe$export$9b476054b78b89cd as colorMix, $9440fa3fdd7adbfe$export$346677f925de839c as size, $9440fa3fdd7adbfe$export$1d567c320f4763bc as style, $e3c5b37c91268204$export$dbf350e5966cf602 as css, $fec05d0b605a7a01$export$dc7aad089b8f5c30 as centerPadding, $fec05d0b605a7a01$export$f2d00fe8ce80fd84 as setColorScheme};
