// @react-spectrum/s2/style/dist/spectrum-theme.mjs
import {ArbitraryProperty as $e3c5b37c91268204$export$d03475ad9df06cd2, createTheme as $e3c5b37c91268204$export$25d302a5b900a763, ExpandedProperty as $e3c5b37c91268204$export$1b6028cfca42adca, MappedProperty as $e3c5b37c91268204$export$965169194d2485f7, parseArbitraryValue as $e3c5b37c91268204$export$22a8270399010c94, PercentageProperty as $e3c5b37c91268204$export$6bff0717a72a8669, SizingProperty as $e3c5b37c91268204$export$8c6b4dc3755476bb} from "./style-macro.mjs";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 
function $9440fa3fdd7adbfe$var$pxToRem(px) {
    if (typeof px === 'string') px = parseFloat(px);
    // In the docs, we need to be able to simulate font size adjustment.
    if (process.env.DOCS_ENV) return `calc(${px / 16} * var(--rem, 1rem))`;
    return px / 16 + 'rem';
}
function $9440fa3fdd7adbfe$var$hcmColor(color) {
    // In the docs, HCM colors can be simulated.
    if (process.env.DOCS_ENV) return `var(--hcm-${color.toLowerCase()}, ${color})`;
    return color;
}
const $9440fa3fdd7adbfe$var$baseColors = {
    transparent: 'transparent',
    black: 'black',
    white: 'white',
    "gray-25": {
        type: "color",
        light: "rgb(255, 255, 255)",
        dark: "rgb(17, 17, 17)"
    },
    "gray-50": {
        type: "color",
        light: "rgb(248, 248, 248)",
        dark: "rgb(27, 27, 27)"
    },
    "gray-75": {
        type: "color",
        light: "rgb(243, 243, 243)",
        dark: "rgb(34, 34, 34)"
    },
    "gray-100": {
        type: "color",
        light: "rgb(233, 233, 233)",
        dark: "rgb(44, 44, 44)"
    },
    "gray-200": {
        type: "color",
        light: "rgb(225, 225, 225)",
        dark: "rgb(50, 50, 50)"
    },
    "gray-300": {
        type: "color",
        light: "rgb(218, 218, 218)",
        dark: "rgb(57, 57, 57)"
    },
    "gray-400": {
        type: "color",
        light: "rgb(198, 198, 198)",
        dark: "rgb(68, 68, 68)"
    },
    "gray-500": {
        type: "color",
        light: "rgb(143, 143, 143)",
        dark: "rgb(109, 109, 109)"
    },
    "gray-600": {
        type: "color",
        light: "rgb(113, 113, 113)",
        dark: "rgb(138, 138, 138)"
    },
    "gray-700": {
        type: "color",
        light: "rgb(80, 80, 80)",
        dark: "rgb(175, 175, 175)"
    },
    "gray-800": {
        type: "color",
        light: "rgb(41, 41, 41)",
        dark: "rgb(219, 219, 219)"
    },
    "gray-900": {
        type: "color",
        light: "rgb(19, 19, 19)",
        dark: "rgb(242, 242, 242)"
    },
    "gray-1000": {
        type: "color",
        light: "rgb(0, 0, 0)",
        dark: "rgb(255, 255, 255)"
    },
    "blue-100": {
        type: "color",
        light: "rgb(245, 249, 255)",
        dark: "rgb(14, 23, 63)"
    },
    "blue-200": {
        type: "color",
        light: "rgb(229, 240, 254)",
        dark: "rgb(15, 28, 82)"
    },
    "blue-300": {
        type: "color",
        light: "rgb(203, 226, 254)",
        dark: "rgb(12, 33, 117)"
    },
    "blue-400": {
        type: "color",
        light: "rgb(172, 207, 253)",
        dark: "rgb(18, 45, 154)"
    },
    "blue-500": {
        type: "color",
        light: "rgb(142, 185, 252)",
        dark: "rgb(26, 58, 195)"
    },
    "blue-600": {
        type: "color",
        light: "rgb(114, 158, 253)",
        dark: "rgb(37, 73, 229)"
    },
    "blue-700": {
        type: "color",
        light: "rgb(93, 137, 255)",
        dark: "rgb(52, 91, 248)"
    },
    "blue-800": {
        type: "color",
        light: "rgb(75, 117, 255)",
        dark: "rgb(64, 105, 253)"
    },
    "blue-900": {
        type: "color",
        light: "rgb(59, 99, 251)",
        dark: "rgb(86, 129, 255)"
    },
    "blue-1000": {
        type: "color",
        light: "rgb(39, 77, 234)",
        dark: "rgb(105, 149, 254)"
    },
    "blue-1100": {
        type: "color",
        light: "rgb(29, 62, 207)",
        dark: "rgb(124, 169, 252)"
    },
    "blue-1200": {
        type: "color",
        light: "rgb(21, 50, 173)",
        dark: "rgb(152, 192, 252)"
    },
    "blue-1300": {
        type: "color",
        light: "rgb(16, 40, 140)",
        dark: "rgb(181, 213, 253)"
    },
    "blue-1400": {
        type: "color",
        light: "rgb(12, 31, 105)",
        dark: "rgb(213, 231, 254)"
    },
    "blue-1500": {
        type: "color",
        light: "rgb(14, 24, 67)",
        dark: "rgb(238, 245, 255)"
    },
    "blue-1600": {
        type: "color",
        light: "rgb(7, 11, 30)",
        dark: "rgb(255, 255, 255)"
    },
    "red-100": {
        type: "color",
        light: "rgb(255, 246, 245)",
        dark: "rgb(54, 10, 3)"
    },
    "red-200": {
        type: "color",
        light: "rgb(255, 235, 232)",
        dark: "rgb(68, 13, 5)"
    },
    "red-300": {
        type: "color",
        light: "rgb(255, 214, 209)",
        dark: "rgb(87, 17, 7)"
    },
    "red-400": {
        type: "color",
        light: "rgb(255, 188, 180)",
        dark: "rgb(115, 24, 11)"
    },
    "red-500": {
        type: "color",
        light: "rgb(255, 157, 145)",
        dark: "rgb(147, 31, 17)"
    },
    "red-600": {
        type: "color",
        light: "rgb(255, 118, 101)",
        dark: "rgb(177, 38, 23)"
    },
    "red-700": {
        type: "color",
        light: "rgb(255, 81, 61)",
        dark: "rgb(205, 46, 29)"
    },
    "red-800": {
        type: "color",
        light: "rgb(240, 56, 35)",
        dark: "rgb(223, 52, 34)"
    },
    "red-900": {
        type: "color",
        light: "rgb(215, 50, 32)",
        dark: "rgb(252, 67, 46)"
    },
    "red-1000": {
        type: "color",
        light: "rgb(183, 40, 24)",
        dark: "rgb(255, 103, 86)"
    },
    "red-1100": {
        type: "color",
        light: "rgb(156, 33, 19)",
        dark: "rgb(255, 134, 120)"
    },
    "red-1200": {
        type: "color",
        light: "rgb(129, 27, 14)",
        dark: "rgb(255, 167, 157)"
    },
    "red-1300": {
        type: "color",
        light: "rgb(104, 21, 10)",
        dark: "rgb(255, 196, 189)"
    },
    "red-1400": {
        type: "color",
        light: "rgb(80, 16, 6)",
        dark: "rgb(255, 222, 219)"
    },
    "red-1500": {
        type: "color",
        light: "rgb(59, 11, 4)",
        dark: "rgb(255, 242, 240)"
    },
    "red-1600": {
        type: "color",
        light: "rgb(29, 5, 2)",
        dark: "rgb(255, 255, 255)"
    },
    "orange-100": {
        type: "color",
        light: "rgb(255, 246, 231)",
        dark: "rgb(49, 16, 0)"
    },
    "orange-200": {
        type: "color",
        light: "rgb(255, 236, 207)",
        dark: "rgb(61, 21, 0)"
    },
    "orange-300": {
        type: "color",
        light: "rgb(255, 218, 158)",
        dark: "rgb(80, 27, 0)"
    },
    "orange-400": {
        type: "color",
        light: "rgb(255, 193, 94)",
        dark: "rgb(106, 36, 0)"
    },
    "orange-500": {
        type: "color",
        light: "rgb(255, 162, 19)",
        dark: "rgb(135, 47, 0)"
    },
    "orange-600": {
        type: "color",
        light: "rgb(252, 125, 0)",
        dark: "rgb(162, 59, 0)"
    },
    "orange-700": {
        type: "color",
        light: "rgb(232, 106, 0)",
        dark: "rgb(185, 73, 0)"
    },
    "orange-800": {
        type: "color",
        light: "rgb(212, 91, 0)",
        dark: "rgb(199, 82, 0)"
    },
    "orange-900": {
        type: "color",
        light: "rgb(194, 78, 0)",
        dark: "rgb(224, 100, 0)"
    },
    "orange-1000": {
        type: "color",
        light: "rgb(167, 62, 0)",
        dark: "rgb(243, 117, 0)"
    },
    "orange-1100": {
        type: "color",
        light: "rgb(144, 51, 0)",
        dark: "rgb(255, 137, 0)"
    },
    "orange-1200": {
        type: "color",
        light: "rgb(118, 41, 0)",
        dark: "rgb(255, 173, 45)"
    },
    "orange-1300": {
        type: "color",
        light: "rgb(95, 32, 0)",
        dark: "rgb(255, 201, 116)"
    },
    "orange-1400": {
        type: "color",
        light: "rgb(73, 24, 0)",
        dark: "rgb(255, 225, 178)"
    },
    "orange-1500": {
        type: "color",
        light: "rgb(52, 18, 0)",
        dark: "rgb(255, 243, 225)"
    },
    "orange-1600": {
        type: "color",
        light: "rgb(25, 8, 0)",
        dark: "rgb(255, 255, 255)"
    },
    "yellow-100": {
        type: "color",
        light: "rgb(255, 248, 204)",
        dark: "rgb(37, 23, 0)"
    },
    "yellow-200": {
        type: "color",
        light: "rgb(255, 241, 151)",
        dark: "rgb(47, 29, 0)"
    },
    "yellow-300": {
        type: "color",
        light: "rgb(255, 222, 44)",
        dark: "rgb(61, 39, 0)"
    },
    "yellow-400": {
        type: "color",
        light: "rgb(245, 199, 0)",
        dark: "rgb(83, 52, 0)"
    },
    "yellow-500": {
        type: "color",
        light: "rgb(230, 175, 0)",
        dark: "rgb(107, 67, 0)"
    },
    "yellow-600": {
        type: "color",
        light: "rgb(210, 149, 0)",
        dark: "rgb(130, 82, 0)"
    },
    "yellow-700": {
        type: "color",
        light: "rgb(193, 131, 0)",
        dark: "rgb(151, 97, 0)"
    },
    "yellow-800": {
        type: "color",
        light: "rgb(175, 116, 0)",
        dark: "rgb(164, 106, 0)"
    },
    "yellow-900": {
        type: "color",
        light: "rgb(158, 102, 0)",
        dark: "rgb(186, 124, 0)"
    },
    "yellow-1000": {
        type: "color",
        light: "rgb(134, 85, 0)",
        dark: "rgb(203, 141, 0)"
    },
    "yellow-1100": {
        type: "color",
        light: "rgb(114, 72, 0)",
        dark: "rgb(218, 159, 0)"
    },
    "yellow-1200": {
        type: "color",
        light: "rgb(93, 59, 0)",
        dark: "rgb(235, 183, 0)"
    },
    "yellow-1300": {
        type: "color",
        light: "rgb(75, 47, 0)",
        dark: "rgb(249, 206, 0)"
    },
    "yellow-1400": {
        type: "color",
        light: "rgb(56, 35, 0)",
        dark: "rgb(255, 230, 86)"
    },
    "yellow-1500": {
        type: "color",
        light: "rgb(40, 25, 0)",
        dark: "rgb(255, 246, 195)"
    },
    "yellow-1600": {
        type: "color",
        light: "rgb(18, 11, 0)",
        dark: "rgb(255, 255, 255)"
    },
    "chartreuse-100": {
        type: "color",
        light: "rgb(246, 251, 222)",
        dark: "rgb(23, 28, 0)"
    },
    "chartreuse-200": {
        type: "color",
        light: "rgb(234, 246, 173)",
        dark: "rgb(30, 36, 0)"
    },
    "chartreuse-300": {
        type: "color",
        light: "rgb(208, 236, 70)",
        dark: "rgb(39, 47, 0)"
    },
    "chartreuse-400": {
        type: "color",
        light: "rgb(182, 219, 0)",
        dark: "rgb(53, 63, 0)"
    },
    "chartreuse-500": {
        type: "color",
        light: "rgb(163, 196, 0)",
        dark: "rgb(68, 82, 0)"
    },
    "chartreuse-600": {
        type: "color",
        light: "rgb(143, 172, 0)",
        dark: "rgb(83, 100, 0)"
    },
    "chartreuse-700": {
        type: "color",
        light: "rgb(128, 153, 0)",
        dark: "rgb(97, 116, 0)"
    },
    "chartreuse-800": {
        type: "color",
        light: "rgb(114, 137, 0)",
        dark: "rgb(106, 127, 0)"
    },
    "chartreuse-900": {
        type: "color",
        light: "rgb(102, 122, 0)",
        dark: "rgb(122, 147, 0)"
    },
    "chartreuse-1000": {
        type: "color",
        light: "rgb(86, 103, 0)",
        dark: "rgb(136, 164, 0)"
    },
    "chartreuse-1100": {
        type: "color",
        light: "rgb(73, 87, 0)",
        dark: "rgb(151, 181, 0)"
    },
    "chartreuse-1200": {
        type: "color",
        light: "rgb(60, 71, 0)",
        dark: "rgb(169, 203, 0)"
    },
    "chartreuse-1300": {
        type: "color",
        light: "rgb(47, 57, 0)",
        dark: "rgb(187, 225, 0)"
    },
    "chartreuse-1400": {
        type: "color",
        light: "rgb(35, 43, 0)",
        dark: "rgb(219, 240, 117)"
    },
    "chartreuse-1500": {
        type: "color",
        light: "rgb(25, 30, 0)",
        dark: "rgb(242, 249, 206)"
    },
    "chartreuse-1600": {
        type: "color",
        light: "rgb(11, 14, 0)",
        dark: "rgb(255, 255, 255)"
    },
    "celery-100": {
        type: "color",
        light: "rgb(235, 255, 220)",
        dark: "rgb(11, 31, 0)"
    },
    "celery-200": {
        type: "color",
        light: "rgb(197, 255, 156)",
        dark: "rgb(15, 38, 0)"
    },
    "celery-300": {
        type: "color",
        light: "rgb(157, 247, 92)",
        dark: "rgb(21, 51, 1)"
    },
    "celery-400": {
        type: "color",
        light: "rgb(129, 228, 58)",
        dark: "rgb(31, 67, 4)"
    },
    "celery-500": {
        type: "color",
        light: "rgb(110, 206, 42)",
        dark: "rgb(41, 86, 8)"
    },
    "celery-600": {
        type: "color",
        light: "rgb(93, 180, 31)",
        dark: "rgb(50, 105, 11)"
    },
    "celery-700": {
        type: "color",
        light: "rgb(82, 161, 25)",
        dark: "rgb(60, 122, 15)"
    },
    "celery-800": {
        type: "color",
        light: "rgb(72, 144, 20)",
        dark: "rgb(66, 134, 18)"
    },
    "celery-900": {
        type: "color",
        light: "rgb(64, 129, 17)",
        dark: "rgb(78, 154, 23)"
    },
    "celery-1000": {
        type: "color",
        light: "rgb(52, 109, 12)",
        dark: "rgb(88, 172, 28)"
    },
    "celery-1100": {
        type: "color",
        light: "rgb(44, 92, 9)",
        dark: "rgb(100, 190, 35)"
    },
    "celery-1200": {
        type: "color",
        light: "rgb(35, 75, 6)",
        dark: "rgb(116, 213, 46)"
    },
    "celery-1300": {
        type: "color",
        light: "rgb(27, 60, 3)",
        dark: "rgb(136, 234, 65)"
    },
    "celery-1400": {
        type: "color",
        light: "rgb(19, 46, 0)",
        dark: "rgb(170, 251, 112)"
    },
    "celery-1500": {
        type: "color",
        light: "rgb(12, 33, 0)",
        dark: "rgb(222, 255, 198)"
    },
    "celery-1600": {
        type: "color",
        light: "rgb(4, 15, 0)",
        dark: "rgb(255, 255, 255)"
    },
    "green-100": {
        type: "color",
        light: "rgb(237, 252, 241)",
        dark: "rgb(0, 30, 23)"
    },
    "green-200": {
        type: "color",
        light: "rgb(215, 247, 225)",
        dark: "rgb(0, 38, 29)"
    },
    "green-300": {
        type: "color",
        light: "rgb(173, 238, 197)",
        dark: "rgb(0, 51, 38)"
    },
    "green-400": {
        type: "color",
        light: "rgb(107, 227, 162)",
        dark: "rgb(0, 68, 48)"
    },
    "green-500": {
        type: "color",
        light: "rgb(43, 209, 125)",
        dark: "rgb(2, 87, 58)"
    },
    "green-600": {
        type: "color",
        light: "rgb(18, 184, 103)",
        dark: "rgb(3, 106, 67)"
    },
    "green-700": {
        type: "color",
        light: "rgb(11, 164, 93)",
        dark: "rgb(4, 124, 75)"
    },
    "green-800": {
        type: "color",
        light: "rgb(7, 147, 85)",
        dark: "rgb(6, 136, 80)"
    },
    "green-900": {
        type: "color",
        light: "rgb(5, 131, 78)",
        dark: "rgb(9, 157, 89)"
    },
    "green-1000": {
        type: "color",
        light: "rgb(3, 110, 69)",
        dark: "rgb(14, 175, 98)"
    },
    "green-1100": {
        type: "color",
        light: "rgb(2, 93, 60)",
        dark: "rgb(24, 193, 110)"
    },
    "green-1200": {
        type: "color",
        light: "rgb(1, 76, 52)",
        dark: "rgb(57, 215, 134)"
    },
    "green-1300": {
        type: "color",
        light: "rgb(0, 61, 44)",
        dark: "rgb(126, 231, 172)"
    },
    "green-1400": {
        type: "color",
        light: "rgb(0, 46, 34)",
        dark: "rgb(189, 241, 208)"
    },
    "green-1500": {
        type: "color",
        light: "rgb(0, 33, 25)",
        dark: "rgb(229, 250, 236)"
    },
    "green-1600": {
        type: "color",
        light: "rgb(0, 15, 12)",
        dark: "rgb(255, 255, 255)"
    },
    "seafoam-100": {
        type: "color",
        light: "rgb(235, 251, 246)",
        dark: "rgb(0, 30, 27)"
    },
    "seafoam-200": {
        type: "color",
        light: "rgb(211, 246, 234)",
        dark: "rgb(0, 39, 35)"
    },
    "seafoam-300": {
        type: "color",
        light: "rgb(169, 237, 216)",
        dark: "rgb(0, 50, 44)"
    },
    "seafoam-400": {
        type: "color",
        light: "rgb(92, 225, 194)",
        dark: "rgb(0, 67, 59)"
    },
    "seafoam-500": {
        type: "color",
        light: "rgb(16, 207, 169)",
        dark: "rgb(2, 86, 75)"
    },
    "seafoam-600": {
        type: "color",
        light: "rgb(13, 181, 149)",
        dark: "rgb(4, 105, 89)"
    },
    "seafoam-700": {
        type: "color",
        light: "rgb(11, 162, 134)",
        dark: "rgb(6, 122, 103)"
    },
    "seafoam-800": {
        type: "color",
        light: "rgb(9, 144, 120)",
        dark: "rgb(8, 134, 112)"
    },
    "seafoam-900": {
        type: "color",
        light: "rgb(7, 129, 109)",
        dark: "rgb(10, 154, 128)"
    },
    "seafoam-1000": {
        type: "color",
        light: "rgb(5, 108, 92)",
        dark: "rgb(12, 173, 142)"
    },
    "seafoam-1100": {
        type: "color",
        light: "rgb(3, 92, 80)",
        dark: "rgb(14, 190, 156)"
    },
    "seafoam-1200": {
        type: "color",
        light: "rgb(1, 75, 67)",
        dark: "rgb(29, 214, 176)"
    },
    "seafoam-1300": {
        type: "color",
        light: "rgb(0, 60, 54)",
        dark: "rgb(122, 229, 203)"
    },
    "seafoam-1400": {
        type: "color",
        light: "rgb(0, 46, 40)",
        dark: "rgb(186, 241, 222)"
    },
    "seafoam-1500": {
        type: "color",
        light: "rgb(0, 33, 29)",
        dark: "rgb(229, 249, 243)"
    },
    "seafoam-1600": {
        type: "color",
        light: "rgb(0, 15, 14)",
        dark: "rgb(255, 255, 255)"
    },
    "cyan-100": {
        type: "color",
        light: "rgb(238, 250, 254)",
        dark: "rgb(0, 29, 39)"
    },
    "cyan-200": {
        type: "color",
        light: "rgb(217, 244, 253)",
        dark: "rgb(0, 36, 49)"
    },
    "cyan-300": {
        type: "color",
        light: "rgb(183, 231, 252)",
        dark: "rgb(0, 48, 65)"
    },
    "cyan-400": {
        type: "color",
        light: "rgb(138, 213, 255)",
        dark: "rgb(0, 64, 88)"
    },
    "cyan-500": {
        type: "color",
        light: "rgb(92, 192, 255)",
        dark: "rgb(0, 82, 113)"
    },
    "cyan-600": {
        type: "color",
        light: "rgb(48, 167, 254)",
        dark: "rgb(3, 99, 140)"
    },
    "cyan-700": {
        type: "color",
        light: "rgb(29, 149, 231)",
        dark: "rgb(8, 115, 168)"
    },
    "cyan-800": {
        type: "color",
        light: "rgb(18, 134, 205)",
        dark: "rgb(13, 125, 186)"
    },
    "cyan-900": {
        type: "color",
        light: "rgb(11, 120, 179)",
        dark: "rgb(24, 142, 220)"
    },
    "cyan-1000": {
        type: "color",
        light: "rgb(4, 102, 145)",
        dark: "rgb(38, 159, 244)"
    },
    "cyan-1100": {
        type: "color",
        light: "rgb(0, 87, 121)",
        dark: "rgb(63, 177, 255)"
    },
    "cyan-1200": {
        type: "color",
        light: "rgb(0, 71, 98)",
        dark: "rgb(107, 199, 255)"
    },
    "cyan-1300": {
        type: "color",
        light: "rgb(0, 57, 78)",
        dark: "rgb(152, 219, 255)"
    },
    "cyan-1400": {
        type: "color",
        light: "rgb(0, 43, 59)",
        dark: "rgb(195, 236, 252)"
    },
    "cyan-1500": {
        type: "color",
        light: "rgb(0, 31, 43)",
        dark: "rgb(230, 248, 253)"
    },
    "cyan-1600": {
        type: "color",
        light: "rgb(0, 14, 20)",
        dark: "rgb(255, 255, 255)"
    },
    "indigo-100": {
        type: "color",
        light: "rgb(247, 248, 255)",
        dark: "rgb(30, 0, 93)"
    },
    "indigo-200": {
        type: "color",
        light: "rgb(235, 238, 255)",
        dark: "rgb(35, 0, 110)"
    },
    "indigo-300": {
        type: "color",
        light: "rgb(216, 222, 255)",
        dark: "rgb(47, 0, 140)"
    },
    "indigo-400": {
        type: "color",
        light: "rgb(192, 201, 255)",
        dark: "rgb(62, 12, 174)"
    },
    "indigo-500": {
        type: "color",
        light: "rgb(167, 178, 255)",
        dark: "rgb(79, 30, 209)"
    },
    "indigo-600": {
        type: "color",
        light: "rgb(145, 151, 254)",
        dark: "rgb(95, 52, 235)"
    },
    "indigo-700": {
        type: "color",
        light: "rgb(132, 128, 254)",
        dark: "rgb(109, 75, 248)"
    },
    "indigo-800": {
        type: "color",
        light: "rgb(122, 106, 253)",
        dark: "rgb(116, 91, 252)"
    },
    "indigo-900": {
        type: "color",
        light: "rgb(113, 85, 250)",
        dark: "rgb(128, 119, 254)"
    },
    "indigo-1000": {
        type: "color",
        light: "rgb(99, 56, 238)",
        dark: "rgb(139, 141, 254)"
    },
    "indigo-1100": {
        type: "color",
        light: "rgb(84, 36, 219)",
        dark: "rgb(153, 161, 255)"
    },
    "indigo-1200": {
        type: "color",
        light: "rgb(69, 19, 191)",
        dark: "rgb(176, 186, 255)"
    },
    "indigo-1300": {
        type: "color",
        light: "rgb(55, 6, 160)",
        dark: "rgb(199, 208, 255)"
    },
    "indigo-1400": {
        type: "color",
        light: "rgb(42, 0, 129)",
        dark: "rgb(223, 228, 255)"
    },
    "indigo-1500": {
        type: "color",
        light: "rgb(31, 0, 98)",
        dark: "rgb(243, 244, 255)"
    },
    "indigo-1600": {
        type: "color",
        light: "rgb(17, 0, 54)",
        dark: "rgb(255, 255, 255)"
    },
    "purple-100": {
        type: "color",
        light: "rgb(251, 247, 254)",
        dark: "rgb(41, 0, 79)"
    },
    "purple-200": {
        type: "color",
        light: "rgb(244, 235, 252)",
        dark: "rgb(50, 0, 96)"
    },
    "purple-300": {
        type: "color",
        light: "rgb(235, 218, 249)",
        dark: "rgb(64, 0, 122)"
    },
    "purple-400": {
        type: "color",
        light: "rgb(221, 193, 246)",
        dark: "rgb(83, 0, 159)"
    },
    "purple-500": {
        type: "color",
        light: "rgb(208, 167, 243)",
        dark: "rgb(107, 6, 195)"
    },
    "purple-600": {
        type: "color",
        light: "rgb(191, 138, 238)",
        dark: "rgb(130, 34, 215)"
    },
    "purple-700": {
        type: "color",
        light: "rgb(178, 114, 235)",
        dark: "rgb(148, 62, 224)"
    },
    "purple-800": {
        type: "color",
        light: "rgb(166, 92, 231)",
        dark: "rgb(157, 78, 228)"
    },
    "purple-900": {
        type: "color",
        light: "rgb(154, 71, 226)",
        dark: "rgb(173, 105, 233)"
    },
    "purple-1000": {
        type: "color",
        light: "rgb(134, 40, 217)",
        dark: "rgb(186, 127, 237)"
    },
    "purple-1100": {
        type: "color",
        light: "rgb(115, 13, 204)",
        dark: "rgb(197, 149, 240)"
    },
    "purple-1200": {
        type: "color",
        light: "rgb(93, 0, 177)",
        dark: "rgb(212, 176, 244)"
    },
    "purple-1300": {
        type: "color",
        light: "rgb(75, 0, 144)",
        dark: "rgb(225, 201, 247)"
    },
    "purple-1400": {
        type: "color",
        light: "rgb(59, 0, 111)",
        dark: "rgb(238, 224, 250)"
    },
    "purple-1500": {
        type: "color",
        light: "rgb(44, 0, 84)",
        dark: "rgb(248, 243, 253)"
    },
    "purple-1600": {
        type: "color",
        light: "rgb(23, 0, 45)",
        dark: "rgb(255, 255, 255)"
    },
    "fuchsia-100": {
        type: "color",
        light: "rgb(254, 246, 255)",
        dark: "rgb(50, 0, 61)"
    },
    "fuchsia-200": {
        type: "color",
        light: "rgb(253, 233, 255)",
        dark: "rgb(61, 0, 74)"
    },
    "fuchsia-300": {
        type: "color",
        light: "rgb(250, 211, 255)",
        dark: "rgb(79, 0, 95)"
    },
    "fuchsia-400": {
        type: "color",
        light: "rgb(247, 181, 255)",
        dark: "rgb(102, 9, 120)"
    },
    "fuchsia-500": {
        type: "color",
        light: "rgb(243, 147, 255)",
        dark: "rgb(127, 23, 146)"
    },
    "fuchsia-600": {
        type: "color",
        light: "rgb(236, 105, 255)",
        dark: "rgb(151, 38, 170)"
    },
    "fuchsia-700": {
        type: "color",
        light: "rgb(223, 77, 245)",
        dark: "rgb(173, 51, 192)"
    },
    "fuchsia-800": {
        type: "color",
        light: "rgb(200, 68, 220)",
        dark: "rgb(186, 60, 206)"
    },
    "fuchsia-900": {
        type: "color",
        light: "rgb(181, 57, 200)",
        dark: "rgb(213, 73, 235)"
    },
    "fuchsia-1000": {
        type: "color",
        light: "rgb(156, 40, 175)",
        dark: "rgb(232, 91, 253)"
    },
    "fuchsia-1100": {
        type: "color",
        light: "rgb(135, 27, 154)",
        dark: "rgb(240, 122, 255)"
    },
    "fuchsia-1200": {
        type: "color",
        light: "rgb(113, 15, 131)",
        dark: "rgb(245, 159, 255)"
    },
    "fuchsia-1300": {
        type: "color",
        light: "rgb(92, 4, 109)",
        dark: "rgb(248, 191, 255)"
    },
    "fuchsia-1400": {
        type: "color",
        light: "rgb(72, 0, 88)",
        dark: "rgb(251, 219, 255)"
    },
    "fuchsia-1500": {
        type: "color",
        light: "rgb(54, 0, 66)",
        dark: "rgb(253, 241, 255)"
    },
    "fuchsia-1600": {
        type: "color",
        light: "rgb(29, 0, 35)",
        dark: "rgb(255, 255, 255)"
    },
    "magenta-100": {
        type: "color",
        light: "rgb(255, 245, 248)",
        dark: "rgb(59, 0, 22)"
    },
    "magenta-200": {
        type: "color",
        light: "rgb(255, 232, 240)",
        dark: "rgb(74, 0, 27)"
    },
    "magenta-300": {
        type: "color",
        light: "rgb(255, 213, 227)",
        dark: "rgb(93, 0, 34)"
    },
    "magenta-400": {
        type: "color",
        light: "rgb(255, 185, 208)",
        dark: "rgb(123, 0, 45)"
    },
    "magenta-500": {
        type: "color",
        light: "rgb(255, 152, 187)",
        dark: "rgb(152, 7, 60)"
    },
    "magenta-600": {
        type: "color",
        light: "rgb(255, 112, 159)",
        dark: "rgb(181, 19, 76)"
    },
    "magenta-700": {
        type: "color",
        light: "rgb(255, 72, 133)",
        dark: "rgb(207, 31, 92)"
    },
    "magenta-800": {
        type: "color",
        light: "rgb(240, 45, 110)",
        dark: "rgb(224, 38, 101)"
    },
    "magenta-900": {
        type: "color",
        light: "rgb(217, 35, 97)",
        dark: "rgb(255, 51, 119)"
    },
    "magenta-1000": {
        type: "color",
        light: "rgb(186, 22, 80)",
        dark: "rgb(255, 96, 149)"
    },
    "magenta-1100": {
        type: "color",
        light: "rgb(163, 5, 62)",
        dark: "rgb(255, 128, 171)"
    },
    "magenta-1200": {
        type: "color",
        light: "rgb(136, 0, 51)",
        dark: "rgb(255, 163, 194)"
    },
    "magenta-1300": {
        type: "color",
        light: "rgb(111, 0, 40)",
        dark: "rgb(255, 193, 214)"
    },
    "magenta-1400": {
        type: "color",
        light: "rgb(86, 0, 30)",
        dark: "rgb(255, 220, 232)"
    },
    "magenta-1500": {
        type: "color",
        light: "rgb(64, 0, 22)",
        dark: "rgb(255, 241, 246)"
    },
    "magenta-1600": {
        type: "color",
        light: "rgb(35, 0, 12)",
        dark: "rgb(255, 255, 255)"
    },
    "pink-100": {
        type: "color",
        light: "rgb(255, 246, 252)",
        dark: "rgb(58, 0, 37)"
    },
    "pink-200": {
        type: "color",
        light: "rgb(255, 232, 247)",
        dark: "rgb(71, 0, 44)"
    },
    "pink-300": {
        type: "color",
        light: "rgb(255, 211, 240)",
        dark: "rgb(90, 0, 57)"
    },
    "pink-400": {
        type: "color",
        light: "rgb(255, 181, 230)",
        dark: "rgb(115, 7, 75)"
    },
    "pink-500": {
        type: "color",
        light: "rgb(255, 148, 219)",
        dark: "rgb(143, 18, 97)"
    },
    "pink-600": {
        type: "color",
        light: "rgb(255, 103, 204)",
        dark: "rgb(171, 29, 119)"
    },
    "pink-700": {
        type: "color",
        light: "rgb(242, 76, 184)",
        dark: "rgb(196, 39, 138)"
    },
    "pink-800": {
        type: "color",
        light: "rgb(228, 52, 163)",
        dark: "rgb(213, 45, 151)"
    },
    "pink-900": {
        type: "color",
        light: "rgb(206, 42, 146)",
        dark: "rgb(236, 67, 175)"
    },
    "pink-1000": {
        type: "color",
        light: "rgb(176, 31, 123)",
        dark: "rgb(251, 90, 196)"
    },
    "pink-1100": {
        type: "color",
        light: "rgb(152, 22, 104)",
        dark: "rgb(255, 122, 210)"
    },
    "pink-1200": {
        type: "color",
        light: "rgb(128, 12, 85)",
        dark: "rgb(255, 159, 223)"
    },
    "pink-1300": {
        type: "color",
        light: "rgb(105, 3, 68)",
        dark: "rgb(255, 191, 234)"
    },
    "pink-1400": {
        type: "color",
        light: "rgb(83, 0, 53)",
        dark: "rgb(255, 219, 243)"
    },
    "pink-1500": {
        type: "color",
        light: "rgb(62, 0, 39)",
        dark: "rgb(255, 241, 250)"
    },
    "pink-1600": {
        type: "color",
        light: "rgb(33, 0, 21)",
        dark: "rgb(255, 255, 255)"
    },
    "turquoise-100": {
        type: "color",
        light: "rgb(238, 251, 251)",
        dark: "rgb(0, 30, 33)"
    },
    "turquoise-200": {
        type: "color",
        light: "rgb(209, 245, 245)",
        dark: "rgb(0, 37, 41)"
    },
    "turquoise-300": {
        type: "color",
        light: "rgb(169, 236, 237)",
        dark: "rgb(0, 49, 54)"
    },
    "turquoise-400": {
        type: "color",
        light: "rgb(111, 221, 228)",
        dark: "rgb(0, 66, 72)"
    },
    "turquoise-500": {
        type: "color",
        light: "rgb(39, 202, 216)",
        dark: "rgb(3, 84, 92)"
    },
    "turquoise-600": {
        type: "color",
        light: "rgb(15, 177, 192)",
        dark: "rgb(5, 103, 112)"
    },
    "turquoise-700": {
        type: "color",
        light: "rgb(12, 158, 171)",
        dark: "rgb(7, 120, 131)"
    },
    "turquoise-800": {
        type: "color",
        light: "rgb(10, 141, 153)",
        dark: "rgb(9, 131, 142)"
    },
    "turquoise-900": {
        type: "color",
        light: "rgb(8, 126, 137)",
        dark: "rgb(11, 151, 164)"
    },
    "turquoise-1000": {
        type: "color",
        light: "rgb(5, 107, 116)",
        dark: "rgb(13, 168, 182)"
    },
    "turquoise-1100": {
        type: "color",
        light: "rgb(3, 90, 98)",
        dark: "rgb(16, 186, 202)"
    },
    "turquoise-1200": {
        type: "color",
        light: "rgb(1, 74, 81)",
        dark: "rgb(64, 208, 220)"
    },
    "turquoise-1300": {
        type: "color",
        light: "rgb(0, 59, 65)",
        dark: "rgb(128, 225, 231)"
    },
    "turquoise-1400": {
        type: "color",
        light: "rgb(0, 44, 49)",
        dark: "rgb(183, 240, 240)"
    },
    "turquoise-1500": {
        type: "color",
        light: "rgb(0, 32, 35)",
        dark: "rgb(228, 249, 249)"
    },
    "turquoise-1600": {
        type: "color",
        light: "rgb(0, 15, 17)",
        dark: "rgb(255, 255, 255)"
    },
    "brown-100": {
        type: "color",
        light: "rgb(252, 247, 242)",
        dark: "rgb(35, 24, 8)"
    },
    "brown-200": {
        type: "color",
        light: "rgb(247, 238, 225)",
        dark: "rgb(44, 31, 11)"
    },
    "brown-300": {
        type: "color",
        light: "rgb(239, 221, 195)",
        dark: "rgb(58, 40, 14)"
    },
    "brown-400": {
        type: "color",
        light: "rgb(229, 200, 157)",
        dark: "rgb(78, 55, 19)"
    },
    "brown-500": {
        type: "color",
        light: "rgb(214, 177, 123)",
        dark: "rgb(98, 71, 30)"
    },
    "brown-600": {
        type: "color",
        light: "rgb(190, 155, 104)",
        dark: "rgb(115, 88, 47)"
    },
    "brown-700": {
        type: "color",
        light: "rgb(171, 138, 90)",
        dark: "rgb(132, 104, 61)"
    },
    "brown-800": {
        type: "color",
        light: "rgb(154, 123, 77)",
        dark: "rgb(143, 114, 69)"
    },
    "brown-900": {
        type: "color",
        light: "rgb(139, 109, 66)",
        dark: "rgb(163, 132, 84)"
    },
    "brown-1000": {
        type: "color",
        light: "rgb(119, 91, 50)",
        dark: "rgb(181, 147, 98)"
    },
    "brown-1100": {
        type: "color",
        light: "rgb(103, 76, 35)",
        dark: "rgb(199, 163, 112)"
    },
    "brown-1200": {
        type: "color",
        light: "rgb(88, 61, 21)",
        dark: "rgb(222, 185, 130)"
    },
    "brown-1300": {
        type: "color",
        light: "rgb(70, 49, 17)",
        dark: "rgb(232, 207, 169)"
    },
    "brown-1400": {
        type: "color",
        light: "rgb(52, 37, 13)",
        dark: "rgb(242, 227, 206)"
    },
    "brown-1500": {
        type: "color",
        light: "rgb(38, 26, 9)",
        dark: "rgb(250, 244, 236)"
    },
    "brown-1600": {
        type: "color",
        light: "rgb(16, 12, 4)",
        dark: "rgb(255, 255, 255)"
    },
    "silver-100": {
        type: "color",
        light: "rgb(247, 247, 247)",
        dark: "rgb(26, 26, 26)"
    },
    "silver-200": {
        type: "color",
        light: "rgb(239, 239, 239)",
        dark: "rgb(33, 33, 33)"
    },
    "silver-300": {
        type: "color",
        light: "rgb(223, 223, 223)",
        dark: "rgb(44, 44, 44)"
    },
    "silver-400": {
        type: "color",
        light: "rgb(204, 204, 204)",
        dark: "rgb(59, 59, 59)"
    },
    "silver-500": {
        type: "color",
        light: "rgb(183, 183, 183)",
        dark: "rgb(76, 76, 76)"
    },
    "silver-600": {
        type: "color",
        light: "rgb(160, 160, 160)",
        dark: "rgb(92, 92, 92)"
    },
    "silver-700": {
        type: "color",
        light: "rgb(143, 143, 143)",
        dark: "rgb(108, 108, 108)"
    },
    "silver-800": {
        type: "color",
        light: "rgb(128, 128, 128)",
        dark: "rgb(118, 118, 118)"
    },
    "silver-900": {
        type: "color",
        light: "rgb(114, 114, 114)",
        dark: "rgb(137, 137, 137)"
    },
    "silver-1000": {
        type: "color",
        light: "rgb(96, 96, 96)",
        dark: "rgb(152, 152, 152)"
    },
    "silver-1100": {
        type: "color",
        light: "rgb(81, 81, 81)",
        dark: "rgb(169, 169, 169)"
    },
    "silver-1200": {
        type: "color",
        light: "rgb(66, 66, 66)",
        dark: "rgb(190, 190, 190)"
    },
    "silver-1300": {
        type: "color",
        light: "rgb(52, 52, 52)",
        dark: "rgb(211, 211, 211)"
    },
    "silver-1400": {
        type: "color",
        light: "rgb(39, 39, 39)",
        dark: "rgb(229, 229, 229)"
    },
    "silver-1500": {
        type: "color",
        light: "rgb(28, 28, 28)",
        dark: "rgb(244, 244, 244)"
    },
    "silver-1600": {
        type: "color",
        light: "rgb(12, 12, 12)",
        dark: "rgb(255, 255, 255)"
    },
    "cinnamon-100": {
        type: "color",
        light: "rgb(253, 247, 243)",
        dark: "rgb(48, 17, 4)"
    },
    "cinnamon-200": {
        type: "color",
        light: "rgb(249, 236, 229)",
        dark: "rgb(59, 21, 5)"
    },
    "cinnamon-300": {
        type: "color",
        light: "rgb(244, 218, 203)",
        dark: "rgb(79, 28, 7)"
    },
    "cinnamon-400": {
        type: "color",
        light: "rgb(237, 196, 172)",
        dark: "rgb(100, 41, 15)"
    },
    "cinnamon-500": {
        type: "color",
        light: "rgb(229, 170, 136)",
        dark: "rgb(122, 57, 28)"
    },
    "cinnamon-600": {
        type: "color",
        light: "rgb(212, 145, 108)",
        dark: "rgb(143, 74, 40)"
    },
    "cinnamon-700": {
        type: "color",
        light: "rgb(198, 126, 88)",
        dark: "rgb(163, 88, 52)"
    },
    "cinnamon-800": {
        type: "color",
        light: "rgb(184, 109, 70)",
        dark: "rgb(176, 98, 59)"
    },
    "cinnamon-900": {
        type: "color",
        light: "rgb(170, 94, 56)",
        dark: "rgb(192, 119, 80)"
    },
    "cinnamon-1000": {
        type: "color",
        light: "rgb(147, 77, 43)",
        dark: "rgb(206, 136, 99)"
    },
    "cinnamon-1100": {
        type: "color",
        light: "rgb(128, 62, 32)",
        dark: "rgb(220, 154, 118)"
    },
    "cinnamon-1200": {
        type: "color",
        light: "rgb(110, 48, 21)",
        dark: "rgb(232, 179, 149)"
    },
    "cinnamon-1300": {
        type: "color",
        light: "rgb(92, 35, 11)",
        dark: "rgb(239, 203, 183)"
    },
    "cinnamon-1400": {
        type: "color",
        light: "rgb(72, 25, 6)",
        dark: "rgb(246, 225, 214)"
    },
    "cinnamon-1500": {
        type: "color",
        light: "rgb(52, 18, 4)",
        dark: "rgb(252, 244, 239)"
    },
    "cinnamon-1600": {
        type: "color",
        light: "rgb(24, 8, 2)",
        dark: "rgb(255, 255, 255)"
    },
    "accent-100": {
        type: "ref",
        light: "blue-100",
        dark: "blue-100"
    },
    "accent-200": {
        type: "ref",
        light: "blue-200",
        dark: "blue-200"
    },
    "accent-300": {
        type: "ref",
        light: "blue-300",
        dark: "blue-300"
    },
    "accent-400": {
        type: "ref",
        light: "blue-400",
        dark: "blue-400"
    },
    "accent-500": {
        type: "ref",
        light: "blue-500",
        dark: "blue-500"
    },
    "accent-600": {
        type: "ref",
        light: "blue-600",
        dark: "blue-600"
    },
    "accent-700": {
        type: "ref",
        light: "blue-700",
        dark: "blue-700"
    },
    "accent-800": {
        type: "ref",
        light: "blue-800",
        dark: "blue-800"
    },
    "accent-900": {
        type: "ref",
        light: "blue-900",
        dark: "blue-900"
    },
    "accent-1000": {
        type: "ref",
        light: "blue-1000",
        dark: "blue-1000"
    },
    "accent-1100": {
        type: "ref",
        light: "blue-1100",
        dark: "blue-1100"
    },
    "accent-1200": {
        type: "ref",
        light: "blue-1200",
        dark: "blue-1200"
    },
    "accent-1300": {
        type: "ref",
        light: "blue-1300",
        dark: "blue-1300"
    },
    "accent-1400": {
        type: "ref",
        light: "blue-1400",
        dark: "blue-1400"
    },
    "accent-1500": {
        type: "ref",
        light: "blue-1500",
        dark: "blue-1500"
    },
    "accent-1600": {
        type: "ref",
        light: "blue-1600",
        dark: "blue-1600"
    },
    "informative-100": {
        type: "ref",
        light: "blue-100",
        dark: "blue-100"
    },
    "informative-200": {
        type: "ref",
        light: "blue-200",
        dark: "blue-200"
    },
    "informative-300": {
        type: "ref",
        light: "blue-300",
        dark: "blue-300"
    },
    "informative-400": {
        type: "ref",
        light: "blue-400",
        dark: "blue-400"
    },
    "informative-500": {
        type: "ref",
        light: "blue-500",
        dark: "blue-500"
    },
    "informative-600": {
        type: "ref",
        light: "blue-600",
        dark: "blue-600"
    },
    "informative-700": {
        type: "ref",
        light: "blue-700",
        dark: "blue-700"
    },
    "informative-800": {
        type: "ref",
        light: "blue-800",
        dark: "blue-800"
    },
    "informative-900": {
        type: "ref",
        light: "blue-900",
        dark: "blue-900"
    },
    "informative-1000": {
        type: "ref",
        light: "blue-1000",
        dark: "blue-1000"
    },
    "informative-1100": {
        type: "ref",
        light: "blue-1100",
        dark: "blue-1100"
    },
    "informative-1200": {
        type: "ref",
        light: "blue-1200",
        dark: "blue-1200"
    },
    "informative-1300": {
        type: "ref",
        light: "blue-1300",
        dark: "blue-1300"
    },
    "informative-1400": {
        type: "ref",
        light: "blue-1400",
        dark: "blue-1400"
    },
    "informative-1500": {
        type: "ref",
        light: "blue-1500",
        dark: "blue-1500"
    },
    "informative-1600": {
        type: "ref",
        light: "blue-1600",
        dark: "blue-1600"
    },
    "negative-100": {
        type: "ref",
        light: "red-100",
        dark: "red-100"
    },
    "negative-200": {
        type: "ref",
        light: "red-200",
        dark: "red-200"
    },
    "negative-300": {
        type: "ref",
        light: "red-300",
        dark: "red-300"
    },
    "negative-400": {
        type: "ref",
        light: "red-400",
        dark: "red-400"
    },
    "negative-500": {
        type: "ref",
        light: "red-500",
        dark: "red-500"
    },
    "negative-600": {
        type: "ref",
        light: "red-600",
        dark: "red-600"
    },
    "negative-700": {
        type: "ref",
        light: "red-700",
        dark: "red-700"
    },
    "negative-800": {
        type: "ref",
        light: "red-800",
        dark: "red-800"
    },
    "negative-900": {
        type: "ref",
        light: "red-900",
        dark: "red-900"
    },
    "negative-1000": {
        type: "ref",
        light: "red-1000",
        dark: "red-1000"
    },
    "negative-1100": {
        type: "ref",
        light: "red-1100",
        dark: "red-1100"
    },
    "negative-1200": {
        type: "ref",
        light: "red-1200",
        dark: "red-1200"
    },
    "negative-1300": {
        type: "ref",
        light: "red-1300",
        dark: "red-1300"
    },
    "negative-1400": {
        type: "ref",
        light: "red-1400",
        dark: "red-1400"
    },
    "negative-1500": {
        type: "ref",
        light: "red-1500",
        dark: "red-1500"
    },
    "negative-1600": {
        type: "ref",
        light: "red-1600",
        dark: "red-1600"
    },
    "notice-100": {
        type: "ref",
        light: "orange-100",
        dark: "orange-100"
    },
    "notice-200": {
        type: "ref",
        light: "orange-200",
        dark: "orange-200"
    },
    "notice-300": {
        type: "ref",
        light: "orange-300",
        dark: "orange-300"
    },
    "notice-400": {
        type: "ref",
        light: "orange-400",
        dark: "orange-400"
    },
    "notice-500": {
        type: "ref",
        light: "orange-500",
        dark: "orange-500"
    },
    "notice-600": {
        type: "ref",
        light: "orange-600",
        dark: "orange-600"
    },
    "notice-700": {
        type: "ref",
        light: "orange-700",
        dark: "orange-700"
    },
    "notice-800": {
        type: "ref",
        light: "orange-800",
        dark: "orange-800"
    },
    "notice-900": {
        type: "ref",
        light: "orange-900",
        dark: "orange-900"
    },
    "notice-1000": {
        type: "ref",
        light: "orange-1000",
        dark: "orange-1000"
    },
    "notice-1100": {
        type: "ref",
        light: "orange-1100",
        dark: "orange-1100"
    },
    "notice-1200": {
        type: "ref",
        light: "orange-1200",
        dark: "orange-1200"
    },
    "notice-1300": {
        type: "ref",
        light: "orange-1300",
        dark: "orange-1300"
    },
    "notice-1400": {
        type: "ref",
        light: "orange-1400",
        dark: "orange-1400"
    },
    "notice-1500": {
        type: "ref",
        light: "orange-1500",
        dark: "orange-1500"
    },
    "notice-1600": {
        type: "ref",
        light: "orange-1600",
        dark: "orange-1600"
    },
    "positive-100": {
        type: "ref",
        light: "green-100",
        dark: "green-100"
    },
    "positive-200": {
        type: "ref",
        light: "green-200",
        dark: "green-200"
    },
    "positive-300": {
        type: "ref",
        light: "green-300",
        dark: "green-300"
    },
    "positive-400": {
        type: "ref",
        light: "green-400",
        dark: "green-400"
    },
    "positive-500": {
        type: "ref",
        light: "green-500",
        dark: "green-500"
    },
    "positive-600": {
        type: "ref",
        light: "green-600",
        dark: "green-600"
    },
    "positive-700": {
        type: "ref",
        light: "green-700",
        dark: "green-700"
    },
    "positive-800": {
        type: "ref",
        light: "green-800",
        dark: "green-800"
    },
    "positive-900": {
        type: "ref",
        light: "green-900",
        dark: "green-900"
    },
    "positive-1000": {
        type: "ref",
        light: "green-1000",
        dark: "green-1000"
    },
    "positive-1100": {
        type: "ref",
        light: "green-1100",
        dark: "green-1100"
    },
    "positive-1200": {
        type: "ref",
        light: "green-1200",
        dark: "green-1200"
    },
    "positive-1300": {
        type: "ref",
        light: "green-1300",
        dark: "green-1300"
    },
    "positive-1400": {
        type: "ref",
        light: "green-1400",
        dark: "green-1400"
    },
    "positive-1500": {
        type: "ref",
        light: "green-1500",
        dark: "green-1500"
    },
    "positive-1600": {
        type: "ref",
        light: "green-1600",
        dark: "green-1600"
    },
    "transparent-white-25": "rgba(255, 255, 255, 0)",
    "transparent-white-50": "rgba(255, 255, 255, 0.04)",
    "transparent-white-75": "rgba(255, 255, 255, 0.07)",
    "transparent-white-100": "rgba(255, 255, 255, 0.11)",
    "transparent-white-200": "rgba(255, 255, 255, 0.14)",
    "transparent-white-300": "rgba(255, 255, 255, 0.17)",
    "transparent-white-400": "rgba(255, 255, 255, 0.21)",
    "transparent-white-500": "rgba(255, 255, 255, 0.39)",
    "transparent-white-600": "rgba(255, 255, 255, 0.51)",
    "transparent-white-700": "rgba(255, 255, 255, 0.66)",
    "transparent-white-800": "rgba(255, 255, 255, 0.85)",
    "transparent-white-900": "rgba(255, 255, 255, 0.94)",
    "transparent-white-1000": "rgb(255, 255, 255)",
    "transparent-black-25": "rgba(0, 0, 0, 0)",
    "transparent-black-50": "rgba(0, 0, 0, 0.03)",
    "transparent-black-75": "rgba(0, 0, 0, 0.05)",
    "transparent-black-100": "rgba(0, 0, 0, 0.09)",
    "transparent-black-200": "rgba(0, 0, 0, 0.12)",
    "transparent-black-300": "rgba(0, 0, 0, 0.15)",
    "transparent-black-400": "rgba(0, 0, 0, 0.22)",
    "transparent-black-500": "rgba(0, 0, 0, 0.44)",
    "transparent-black-600": "rgba(0, 0, 0, 0.56)",
    "transparent-black-700": "rgba(0, 0, 0, 0.69)",
    "transparent-black-800": "rgba(0, 0, 0, 0.84)",
    "transparent-black-900": "rgba(0, 0, 0, 0.93)",
    "transparent-black-1000": "rgb(0, 0, 0)",
    "transparent-overlay-25": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 0)",
    "transparent-overlay-50": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 0.04)",
    "transparent-overlay-75": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 0.07)",
    "transparent-overlay-100": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 0.11)",
    "transparent-overlay-200": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 0.14)",
    "transparent-overlay-300": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 0.17)",
    "transparent-overlay-400": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 0.21)",
    "transparent-overlay-500": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 0.39)",
    "transparent-overlay-600": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 0.51)",
    "transparent-overlay-700": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 0.66)",
    "transparent-overlay-800": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 0.85)",
    "transparent-overlay-900": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 0.94)",
    "transparent-overlay-1000": "lch(from var(--s2-container-bg) calc((49.44 - l) * infinity) 0 0 / 1)",
    // High contrast mode.
    // In the docs these can be simulated via variables.
    Background: $9440fa3fdd7adbfe$var$hcmColor('Background'),
    ButtonBorder: $9440fa3fdd7adbfe$var$hcmColor('ButtonBorder'),
    ButtonFace: $9440fa3fdd7adbfe$var$hcmColor('ButtonFace'),
    ButtonText: $9440fa3fdd7adbfe$var$hcmColor('ButtonText'),
    Field: $9440fa3fdd7adbfe$var$hcmColor('Field'),
    Highlight: $9440fa3fdd7adbfe$var$hcmColor('Highlight'),
    HighlightText: $9440fa3fdd7adbfe$var$hcmColor('HighlightText'),
    GrayText: $9440fa3fdd7adbfe$var$hcmColor('GrayText'),
    Mark: $9440fa3fdd7adbfe$var$hcmColor('Mark'),
    LinkText: $9440fa3fdd7adbfe$var$hcmColor('LinkText')
};
// Resolves a color to its most basic form, following all aliases.
function $9440fa3fdd7adbfe$var$resolveColorToken(token) {
    if (typeof token === 'string') return {
        type: 'color',
        light: token,
        dark: token
    };
    if (token.type === 'color') return token;
    let lightToken = $9440fa3fdd7adbfe$var$baseColors[token.light];
    if (!lightToken) throw new Error(`${token.light} is not a valid color reference`);
    let darkToken = $9440fa3fdd7adbfe$var$baseColors[token.dark];
    if (!darkToken) throw new Error(`${token.dark} is not a valid color reference`);
    let light = $9440fa3fdd7adbfe$var$resolveColorToken(lightToken);
    let dark = $9440fa3fdd7adbfe$var$resolveColorToken(darkToken);
    return {
        type: 'color',
        light: light.light,
        dark: dark.dark
    };
}
function $9440fa3fdd7adbfe$var$colorTokenToString(token, opacity) {
    let result = token.light === token.dark ? token.light : `light-dark(${token.light}, ${token.dark})`;
    if (opacity) result = `rgb(from ${result} r g b / ${opacity}%)`;
    return result;
}
// Bumps up a color token by one stop, e.g. for hover/press states.
let $9440fa3fdd7adbfe$var$colorList = Object.keys($9440fa3fdd7adbfe$var$baseColors);
function $9440fa3fdd7adbfe$var$nextColorStop(name, token) {
    if (typeof token === 'object' && token.type === 'ref') {
        let light = $9440fa3fdd7adbfe$var$nextColorStop(token.light, $9440fa3fdd7adbfe$var$baseColors[token.light]);
        let dark = $9440fa3fdd7adbfe$var$nextColorStop(token.dark, $9440fa3fdd7adbfe$var$baseColors[token.dark]);
        return {
            type: 'color',
            light: light.light,
            dark: dark.dark,
            forcedColors: token.forcedColors
        };
    }
    let index = $9440fa3fdd7adbfe$var$colorList.indexOf(name);
    if (index === -1) throw new Error(`${name} does not support states`);
    let key = $9440fa3fdd7adbfe$var$colorList[index + 1];
    if (key.split('-')[0] !== name.split('-')[0]) throw new Error(`${name} does not support states`);
    return $9440fa3fdd7adbfe$var$resolveColorToken($9440fa3fdd7adbfe$var$baseColors[key]);
}
class $9440fa3fdd7adbfe$var$SpectrumColorProperty extends (0, $e3c5b37c91268204$export$d03475ad9df06cd2) {
    constructor(property, mapping){
        super(property);
        this.mapping = mapping;
    }
    toCSSValue(value) {
        let [colorWithOpacity, state] = value.split(':');
        let [color, opacity] = colorWithOpacity.split('/');
        let token = this.mapping[color];
        if (!token) throw new Error('Invalid color ' + value);
        if (state === 'hovered' || state === 'pressed' || state === 'focused') token = $9440fa3fdd7adbfe$var$nextColorStop(color, token);
        else token = $9440fa3fdd7adbfe$var$resolveColorToken(token);
        let result = $9440fa3fdd7adbfe$var$colorTokenToString(token, opacity);
        if (token.forcedColors) return {
            default: result,
            forcedColors: token.forcedColors
        };
        return result;
    }
}
function $9440fa3fdd7adbfe$export$49bbad2a916ab57c(base) {
    return {
        default: base,
        isHovered: `${base}:hovered`,
        isFocusVisible: `${base}:focused`,
        isPressed: `${base}:pressed`
    };
}
function $9440fa3fdd7adbfe$export$35e9368ef982300f(value) {
    let arbitrary = (0, $e3c5b37c91268204$export$22a8270399010c94)(value);
    if (arbitrary) return arbitrary;
    let [colorValue, opacity] = value.split('/');
    return $9440fa3fdd7adbfe$var$colorTokenToString($9440fa3fdd7adbfe$var$resolveColorToken($9440fa3fdd7adbfe$var$baseColors[colorValue]), opacity);
}
function $9440fa3fdd7adbfe$export$d19e682d444755ed(light, dark) {
    return `[light-dark(${$9440fa3fdd7adbfe$export$35e9368ef982300f(light)}, ${$9440fa3fdd7adbfe$export$35e9368ef982300f(dark)})]`;
}
function $9440fa3fdd7adbfe$export$9b476054b78b89cd(a, b, percent) {
    return `[color-mix(in srgb, ${$9440fa3fdd7adbfe$export$35e9368ef982300f(a)}, ${$9440fa3fdd7adbfe$export$35e9368ef982300f(b)} ${percent}%)]`;
}
function $9440fa3fdd7adbfe$export$46def8197cf4dd4c(angle, ...tokens) {
    // Generate @property rules for each gradient stop color. This allows the gradient to be animated.
    let propertyDefinitions = [];
    for(let i = 0; i < tokens.length; i++)propertyDefinitions.push(`@property --g${i} {
  syntax: '<color>';
  initial-value: #0000;
  inherits: false;
}`);
    if (this && typeof this.addAsset === 'function') this.addAsset({
        type: 'css',
        content: propertyDefinitions.join('\n\n')
    });
    return [
        {
            type: 'linear-gradient',
            angle: angle,
            stops: tokens
        }
    ];
}
// Spacing uses rems, padding does not.
function $9440fa3fdd7adbfe$var$generateSpacing(px) {
    let spacing = {};
    let padding = {};
    for (let p of px){
        spacing[p] = $9440fa3fdd7adbfe$var$pxToRem(p);
        padding[p] = p + 'px';
    }
    return {
        spacing: spacing,
        padding: padding
    };
}
const { spacing: $9440fa3fdd7adbfe$var$baseSpacing, padding: $9440fa3fdd7adbfe$var$basePadding } = $9440fa3fdd7adbfe$var$generateSpacing([
    0,
    2,
    4,
    8,
    12,
    16,
    20,
    24,
    28,
    32,
    36,
    40,
    44,
    48,
    56,
    // From here onward the values are mostly spaced by 1rem (16px)
    64,
    80,
    96 // spacing-1000
]);
// This should match the above, but negative. There's no way to negate a number
// type in typescript so this has to be done manually for now.
const { spacing: $9440fa3fdd7adbfe$var$negativeSpacing, padding: $9440fa3fdd7adbfe$var$negativePadding } = $9440fa3fdd7adbfe$var$generateSpacing([
    -2,
    -4,
    -8,
    -12,
    -16,
    -20,
    -24,
    -28,
    -32,
    -36,
    -40,
    -44,
    -48,
    -56,
    // From here onward the values are mostly spaced by 1rem (16px)
    -64,
    -80,
    -96 // spacing-1000
]);
function $9440fa3fdd7adbfe$export$ba024f4caf693d15(base, baseFontSize = 14) {
    return base / baseFontSize + 'em';
}
function $9440fa3fdd7adbfe$export$17b63977b35bb6d2(height) {
    return `calc(${$9440fa3fdd7adbfe$var$baseSpacing[height]} * 3 / 8)`;
}
function $9440fa3fdd7adbfe$export$a941ed4b947d12f8(px) {
    return $9440fa3fdd7adbfe$var$pxToRem(px);
}
const $9440fa3fdd7adbfe$var$relativeSpacing = {
    // font-size relative values
    'text-to-control': $9440fa3fdd7adbfe$export$ba024f4caf693d15(10),
    'text-to-visual': {
        default: $9440fa3fdd7adbfe$export$ba024f4caf693d15(6),
        touch: $9440fa3fdd7adbfe$export$ba024f4caf693d15(8, 17) // -> 6px, 7px, 8px, 9px, 10px, should be 7px, 7px, 8px, 9px, 11px
    },
    // height relative values
    'edge-to-text': 'calc(self(height, self(minHeight)) * 3 / 8)',
    pill: 'calc(self(height, self(minHeight)) / 2)'
};
const $9440fa3fdd7adbfe$var$spacing = {
    ...$9440fa3fdd7adbfe$var$baseSpacing,
    ...$9440fa3fdd7adbfe$var$relativeSpacing
};
const $9440fa3fdd7adbfe$var$padding = {
    ...$9440fa3fdd7adbfe$var$basePadding,
    ...$9440fa3fdd7adbfe$var$relativeSpacing
};
function $9440fa3fdd7adbfe$export$346677f925de839c(px) {
    return `calc(${$9440fa3fdd7adbfe$var$pxToRem(px)} * var(--s2-scale))`;
}
const $9440fa3fdd7adbfe$var$sizing = {
    auto: 'auto',
    full: '100%',
    min: 'min-content',
    max: 'max-content',
    fit: 'fit-content'
};
const $9440fa3fdd7adbfe$var$height = {
    ...$9440fa3fdd7adbfe$var$sizing,
    screen: '100vh'
};
const $9440fa3fdd7adbfe$var$width = {
    ...$9440fa3fdd7adbfe$var$sizing,
    screen: '100vw'
};
function $9440fa3fdd7adbfe$var$createSpectrumSizingProperty(property, values) {
    return new (0, $e3c5b37c91268204$export$8c6b4dc3755476bb)(property, values, (px)=>`calc(${$9440fa3fdd7adbfe$var$pxToRem(px)} * var(--s2-scale))`);
}
const $9440fa3fdd7adbfe$var$margin = {
    ...$9440fa3fdd7adbfe$var$spacing,
    ...$9440fa3fdd7adbfe$var$negativeSpacing,
    auto: 'auto'
};
const $9440fa3fdd7adbfe$var$inset = {
    ...$9440fa3fdd7adbfe$var$basePadding,
    ...$9440fa3fdd7adbfe$var$negativePadding,
    auto: 'auto',
    full: '100%'
};
const $9440fa3fdd7adbfe$var$translate = {
    ...$9440fa3fdd7adbfe$var$basePadding,
    ...$9440fa3fdd7adbfe$var$negativePadding,
    full: '100%'
};
const $9440fa3fdd7adbfe$var$borderWidth = {
    0: '0px',
    1: "1px",
    2: "2px",
    4: "4px"
};
const $9440fa3fdd7adbfe$var$radius = {
    none: "0px",
    sm: $9440fa3fdd7adbfe$var$pxToRem("4px"),
    default: $9440fa3fdd7adbfe$var$pxToRem("8px"),
    lg: $9440fa3fdd7adbfe$var$pxToRem("10px"),
    xl: $9440fa3fdd7adbfe$var$pxToRem("16px"),
    full: '9999px',
    pill: 'calc(self(height, self(minHeight, 9999px)) / 2)'
};
let $9440fa3fdd7adbfe$var$gridTrack = (value)=>{
    if (typeof value === 'string') return value;
    return value.map((v)=>$9440fa3fdd7adbfe$var$gridTrackSize(v)).join(' ');
};
let $9440fa3fdd7adbfe$var$gridTrackSize = (value)=>{
    return typeof value === 'number' ? $9440fa3fdd7adbfe$export$346677f925de839c(value) : value;
};
const $9440fa3fdd7adbfe$var$transitionProperty = {
    // var(--gp) is generated by the backgroundImage property when setting a gradient.
    // It includes a list of all of the custom properties used for each color stop.
    default: 'color, background-color, var(--gp, color), border-color, text-decoration-color, fill, stroke, opacity, box-shadow, transform, translate, scale, rotate, filter, backdrop-filter',
    colors: 'color, background-color, var(--gp, color), border-color, text-decoration-color, fill, stroke',
    opacity: 'opacity',
    shadow: 'box-shadow',
    transform: 'transform, translate, scale, rotate',
    all: 'all',
    none: 'none'
};
// TODO
const $9440fa3fdd7adbfe$var$timingFunction = {
    default: 'cubic-bezier(0.45, 0, 0.4, 1)',
    linear: 'linear',
    in: 'cubic-bezier(0.5, 0, 1, 1)',
    out: 'cubic-bezier(0, 0, 0.40, 1)',
    'in-out': 'cubic-bezier(0.45, 0, 0.4, 1)'
};
let $9440fa3fdd7adbfe$var$durationValue = (value)=>typeof value === 'number' ? value + 'ms' : value;
const $9440fa3fdd7adbfe$var$fontWeightBase = {
    normal: '400',
    medium: {
        default: '500'
    },
    bold: {
        default: '700',
        ':lang(ja, ko, zh)': '500' // Adobe Clean Han uses 500 as the bold weight
    },
    'extra-bold': {
        default: '800',
        ':lang(ja, ko, zh)': '700' // Adobe Clean Han uses 700 as the extra bold weight.
    },
    black: '900'
};
const $9440fa3fdd7adbfe$var$fontWeight = {
    ...$9440fa3fdd7adbfe$var$fontWeightBase,
    heading: {
        default: $9440fa3fdd7adbfe$var$fontWeightBase["extra-bold"],
        ':lang(ja, ko, zh, zh-Hant, zh-Hans)': $9440fa3fdd7adbfe$var$fontWeightBase["extra-bold"]
    },
    title: {
        default: $9440fa3fdd7adbfe$var$fontWeightBase["bold"],
        ':lang(ja, ko, zh, zh-Hant, zh-Hans)': $9440fa3fdd7adbfe$var$fontWeightBase["bold"]
    },
    detail: {
        default: $9440fa3fdd7adbfe$var$fontWeightBase["medium"],
        ':lang(ja, ko, zh, zh-Hant, zh-Hans)': $9440fa3fdd7adbfe$var$fontWeightBase["bold"]
    }
};
const $9440fa3fdd7adbfe$var$i18nFonts = {
    ':lang(ar)': 'adobe-clean-arabic, myriad-arabic, ui-sans-serif, system-ui, sans-serif',
    ':lang(he)': 'adobe-clean-hebrew, myriad-hebrew, ui-sans-serif, system-ui, sans-serif',
    ':lang(ja)': "adobe-clean-han-japanese, 'Hiragino Kaku Gothic ProN', '\u30D2\u30E9\u30AE\u30CE\u89D2\u30B4 ProN W3', Osaka, YuGothic, 'Yu Gothic', '\u30E1\u30A4\u30EA\u30AA', Meiryo, '\uFF2D\uFF33 \uFF30\u30B4\u30B7\u30C3\u30AF', 'MS PGothic', sans-serif",
    ':lang(ko)': "adobe-clean-han-korean, source-han-korean, 'Malgun Gothic', 'Apple Gothic', sans-serif",
    ':lang(zh)': "adobe-clean-han-traditional, source-han-traditional, 'MingLiu', 'Heiti TC Light', sans-serif",
    // TODO: are these fallbacks supposed to be different than above?
    ':lang(zh-hant)': "adobe-clean-han-traditional, source-han-traditional, 'MingLiu', 'Microsoft JhengHei UI', 'Microsoft JhengHei', 'Heiti TC Light', sans-serif",
    ':lang(zh-HK)': "adobe-clean-han-hong-kong, source-han-hong-kong, 'MingLiu', 'Microsoft JhengHei UI', 'Microsoft JhengHei', 'Heiti TC Light', sans-serif",
    ':lang(zh-Hans, zh-CN, zh-SG)': "adobe-clean-han-simplified-c, source-han-simplified-c, 'SimSun', 'Heiti SC Light', sans-serif"
};
const $9440fa3fdd7adbfe$var$fontSize = {
    // The default font size scale is for use within UI components.
    'ui-xs': -2,
    'ui-sm': -1,
    ui: 0,
    'ui-lg': 1,
    'ui-xl': 2,
    'ui-2xl': 3,
    'ui-3xl': 4,
    'heading-2xs': 0,
    'heading-xs': 2,
    'heading-sm': 3,
    heading: 4,
    'heading-lg': 6,
    'heading-xl': 8,
    'heading-2xl': 10,
    'heading-3xl': 12,
    'title-xs': -1,
    'title-sm': 0,
    title: 1,
    'title-lg': 2,
    'title-xl': 3,
    'title-2xl': 4,
    'title-3xl': 5,
    // Body is for large blocks of text, e.g. paragraphs, not in UI components.
    'body-2xs': -2,
    'body-xs': -1,
    'body-sm': 0,
    body: 1,
    'body-lg': 2,
    'body-xl': 3,
    'body-2xl': 4,
    'body-3xl': 5,
    'detail-sm': -1,
    detail: 0,
    'detail-lg': 1,
    'detail-xl': 2,
    'code-xs': -1,
    'code-sm': 0,
    code: 1,
    'code-lg': 2,
    'code-xl': 3
};
// Line heights linearly interpolate between 1.3 and 1.15 for font sizes between 10 and 32, rounded to the nearest 2px.
// Text above 32px always has a line height of 1.15.
const $9440fa3fdd7adbfe$var$fontSizeCalc = 'var(--s2-font-size-base, 14) * var(--fs)';
const $9440fa3fdd7adbfe$var$minFontScale = 1.15;
const $9440fa3fdd7adbfe$var$maxFontScale = 1.3;
const $9440fa3fdd7adbfe$var$minFontSize = 10;
const $9440fa3fdd7adbfe$var$maxFontSize = 32;
const $9440fa3fdd7adbfe$var$lineHeightCalc = `round(1em * (${$9440fa3fdd7adbfe$var$minFontScale} + (1 - ((min(${$9440fa3fdd7adbfe$var$maxFontSize}, ${$9440fa3fdd7adbfe$var$fontSizeCalc}) - ${$9440fa3fdd7adbfe$var$minFontSize})) / ${$9440fa3fdd7adbfe$var$maxFontSize - $9440fa3fdd7adbfe$var$minFontSize}) * ${($9440fa3fdd7adbfe$var$maxFontScale - $9440fa3fdd7adbfe$var$minFontScale).toFixed(2)}), 2px)`;
const $9440fa3fdd7adbfe$export$1d567c320f4763bc = (0, $e3c5b37c91268204$export$25d302a5b900a763)({
    properties: {
        // colors
        color: new $9440fa3fdd7adbfe$var$SpectrumColorProperty('color', {
            ...$9440fa3fdd7adbfe$var$baseColors,
            accent: {
                type: "ref",
                light: "accent-900",
                dark: "accent-900"
            },
            neutral: {
                type: "ref",
                light: "gray-800",
                dark: "gray-800"
            },
            'neutral-subdued': {
                type: "ref",
                light: "gray-700",
                dark: "gray-700"
            },
            negative: {
                type: "ref",
                light: "negative-900",
                dark: "negative-900"
            },
            disabled: {
                type: "ref",
                light: "gray-400",
                dark: "gray-400"
            },
            heading: {
                type: "ref",
                light: "gray-900",
                dark: "gray-900"
            },
            title: {
                type: "ref",
                light: "gray-900",
                dark: "gray-900"
            },
            body: {
                type: "ref",
                light: "gray-800",
                dark: "gray-800"
            },
            detail: {
                type: "ref",
                light: "gray-600",
                dark: "gray-600"
            },
            code: {
                type: "ref",
                light: "gray-800",
                dark: "gray-800"
            },
            auto: "lch(from self(backgroundColor, var(--s2-container-bg)) calc((49.44 - l) * infinity) 0 0 / 1)"
        }),
        backgroundColor: new $9440fa3fdd7adbfe$var$SpectrumColorProperty('backgroundColor', {
            ...$9440fa3fdd7adbfe$var$baseColors,
            accent: {
                type: "ref",
                light: "accent-900",
                dark: "accent-800"
            },
            'accent-subtle': {
                type: "ref",
                light: "accent-200",
                dark: "accent-300"
            },
            neutral: {
                type: "ref",
                light: "gray-800",
                dark: "gray-800"
            },
            'neutral-subdued': {
                type: "ref",
                light: "gray-700",
                dark: "gray-500"
            },
            'neutral-subtle': {
                type: "ref",
                light: "gray-100",
                dark: "gray-300"
            },
            negative: {
                type: "ref",
                light: "negative-900",
                dark: "negative-800"
            },
            'negative-subtle': {
                type: "ref",
                light: "negative-200",
                dark: "negative-300"
            },
            informative: {
                type: "ref",
                light: "informative-900",
                dark: "informative-800"
            },
            'informative-subtle': {
                type: "ref",
                light: "informative-200",
                dark: "informative-300"
            },
            positive: {
                type: "ref",
                light: "positive-900",
                dark: "positive-800"
            },
            'positive-subtle': {
                type: "ref",
                light: "positive-200",
                dark: "positive-300"
            },
            notice: {
                type: "ref",
                light: "notice-600",
                dark: "notice-900"
            },
            'notice-subtle': {
                type: "ref",
                light: "notice-200",
                dark: "notice-300"
            },
            gray: {
                type: "ref",
                light: "gray-700",
                dark: "gray-500"
            },
            'gray-subtle': {
                type: "ref",
                light: "gray-100",
                dark: "gray-300"
            },
            red: {
                type: "ref",
                light: "red-900",
                dark: "red-800"
            },
            'red-subtle': {
                type: "ref",
                light: "red-200",
                dark: "red-300"
            },
            orange: {
                type: "ref",
                light: "orange-600",
                dark: "orange-900"
            },
            'orange-subtle': {
                type: "ref",
                light: "orange-200",
                dark: "orange-300"
            },
            yellow: {
                type: "ref",
                light: "yellow-400",
                dark: "yellow-1100"
            },
            'yellow-subtle': {
                type: "ref",
                light: "yellow-200",
                dark: "yellow-300"
            },
            chartreuse: {
                type: "ref",
                light: "chartreuse-500",
                dark: "chartreuse-1000"
            },
            'chartreuse-subtle': {
                type: "ref",
                light: "chartreuse-200",
                dark: "chartreuse-300"
            },
            celery: {
                type: "ref",
                light: "celery-600",
                dark: "celery-900"
            },
            'celery-subtle': {
                type: "ref",
                light: "celery-200",
                dark: "celery-300"
            },
            green: {
                type: "ref",
                light: "green-900",
                dark: "green-800"
            },
            'green-subtle': {
                type: "ref",
                light: "green-200",
                dark: "green-300"
            },
            seafoam: {
                type: "ref",
                light: "seafoam-900",
                dark: "seafoam-800"
            },
            'seafoam-subtle': {
                type: "ref",
                light: "seafoam-200",
                dark: "seafoam-300"
            },
            cyan: {
                type: "ref",
                light: "cyan-900",
                dark: "cyan-800"
            },
            'cyan-subtle': {
                type: "ref",
                light: "cyan-200",
                dark: "cyan-300"
            },
            blue: {
                type: "ref",
                light: "blue-900",
                dark: "blue-800"
            },
            'blue-subtle': {
                type: "ref",
                light: "blue-200",
                dark: "blue-300"
            },
            indigo: {
                type: "ref",
                light: "indigo-900",
                dark: "indigo-800"
            },
            'indigo-subtle': {
                type: "ref",
                light: "indigo-200",
                dark: "indigo-300"
            },
            purple: {
                type: "ref",
                light: "purple-900",
                dark: "purple-800"
            },
            'purple-subtle': {
                type: "ref",
                light: "purple-200",
                dark: "purple-300"
            },
            fuchsia: {
                type: "ref",
                light: "fuchsia-900",
                dark: "fuchsia-800"
            },
            'fuchsia-subtle': {
                type: "ref",
                light: "fuchsia-200",
                dark: "fuchsia-300"
            },
            magenta: {
                type: "ref",
                light: "magenta-900",
                dark: "magenta-800"
            },
            'magenta-subtle': {
                type: "ref",
                light: "magenta-200",
                dark: "magenta-300"
            },
            pink: {
                type: "ref",
                light: "pink-900",
                dark: "pink-800"
            },
            'pink-subtle': {
                type: "ref",
                light: "pink-200",
                dark: "pink-300"
            },
            turquoise: {
                type: "ref",
                light: "turquoise-900",
                dark: "turquoise-800"
            },
            'turquoise-subtle': {
                type: "ref",
                light: "turquoise-200",
                dark: "turquoise-300"
            },
            cinnamon: {
                type: "ref",
                light: "cinnamon-900",
                dark: "cinnamon-800"
            },
            'cinnamon-subtle': {
                type: "ref",
                light: "cinnamon-200",
                dark: "cinnamon-300"
            },
            brown: {
                type: "ref",
                light: "brown-900",
                dark: "brown-800"
            },
            'brown-subtle': {
                type: "ref",
                light: "brown-200",
                dark: "brown-300"
            },
            silver: {
                type: "ref",
                light: "silver-900",
                dark: "silver-800"
            },
            'silver-subtle': {
                type: "ref",
                light: "silver-200",
                dark: "silver-300"
            },
            disabled: {
                type: "ref",
                light: "gray-100",
                dark: "gray-100"
            },
            base: {
                type: "ref",
                light: "gray-25",
                dark: "gray-25"
            },
            'layer-1': {
                type: "ref",
                light: "gray-50",
                dark: "gray-50"
            },
            'layer-2': {
                type: "ref",
                light: "gray-25",
                dark: "gray-75"
            },
            pasteboard: {
                type: "ref",
                light: "gray-100",
                dark: "gray-25"
            },
            elevated: {
                type: "ref",
                light: "gray-25",
                dark: "gray-75"
            }
        }),
        borderColor: new $9440fa3fdd7adbfe$var$SpectrumColorProperty('borderColor', {
            ...$9440fa3fdd7adbfe$var$baseColors,
            negative: {
                type: "ref",
                light: "negative-900",
                dark: "negative-900"
            },
            disabled: {
                type: "ref",
                light: "gray-300",
                dark: "gray-300"
            }
        }),
        outlineColor: new $9440fa3fdd7adbfe$var$SpectrumColorProperty('outlineColor', {
            ...$9440fa3fdd7adbfe$var$baseColors,
            'focus-ring': {
                type: "ref",
                light: "blue-800",
                dark: "blue-800",
                forcedColors: 'Highlight'
            }
        }),
        fill: new $9440fa3fdd7adbfe$var$SpectrumColorProperty('fill', {
            none: 'none',
            currentColor: 'currentColor',
            accent: {
                type: "ref",
                light: "accent-800",
                dark: "accent-900"
            },
            neutral: {
                type: "ref",
                light: "gray-500",
                dark: "gray-600"
            },
            negative: {
                type: "ref",
                light: "negative-800",
                dark: "negative-900"
            },
            informative: {
                type: "ref",
                light: "informative-800",
                dark: "informative-900"
            },
            positive: {
                type: "ref",
                light: "positive-800",
                dark: "positive-900"
            },
            notice: {
                type: "ref",
                light: "notice-800",
                dark: "notice-900"
            },
            gray: {
                type: "ref",
                light: "gray-500",
                dark: "gray-600"
            },
            red: {
                type: "ref",
                light: "red-800",
                dark: "red-700"
            },
            orange: {
                type: "ref",
                light: "orange-700",
                dark: "orange-900"
            },
            yellow: {
                type: "ref",
                light: "yellow-600",
                dark: "yellow-1100"
            },
            chartreuse: {
                type: "ref",
                light: "chartreuse-600",
                dark: "chartreuse-900"
            },
            celery: {
                type: "ref",
                light: "celery-700",
                dark: "celery-800"
            },
            green: {
                type: "ref",
                light: "green-700",
                dark: "green-800"
            },
            seafoam: {
                type: "ref",
                light: "seafoam-700",
                dark: "seafoam-800"
            },
            cyan: {
                type: "ref",
                light: "cyan-600",
                dark: "cyan-900"
            },
            blue: {
                type: "ref",
                light: "blue-800",
                dark: "blue-900"
            },
            indigo: {
                type: "ref",
                light: "indigo-800",
                dark: "indigo-900"
            },
            purple: {
                type: "ref",
                light: "purple-800",
                dark: "purple-900"
            },
            fuchsia: {
                type: "ref",
                light: "fuchsia-800",
                dark: "fuchsia-900"
            },
            magenta: {
                type: "ref",
                light: "magenta-800",
                dark: "magenta-900"
            },
            pink: {
                type: "ref",
                light: "pink-800",
                dark: "pink-900"
            },
            turquoise: {
                type: "ref",
                light: "turquoise-800",
                dark: "turquoise-900"
            },
            cinnamon: {
                type: "ref",
                light: "cinnamon-800",
                dark: "cinnamon-900"
            },
            brown: {
                type: "ref",
                light: "brown-800",
                dark: "brown-900"
            },
            silver: {
                type: "ref",
                light: "silver-800",
                dark: "silver-900"
            },
            ...$9440fa3fdd7adbfe$var$baseColors
        }),
        stroke: new $9440fa3fdd7adbfe$var$SpectrumColorProperty('stroke', {
            none: 'none',
            currentColor: 'currentColor',
            ...$9440fa3fdd7adbfe$var$baseColors
        }),
        // dimensions
        borderSpacing: $9440fa3fdd7adbfe$var$baseSpacing,
        flexBasis: $9440fa3fdd7adbfe$var$createSpectrumSizingProperty('flexBasis', {
            auto: 'auto',
            full: '100%'
        }),
        rowGap: $9440fa3fdd7adbfe$var$spacing,
        columnGap: $9440fa3fdd7adbfe$var$spacing,
        height: $9440fa3fdd7adbfe$var$createSpectrumSizingProperty('height', $9440fa3fdd7adbfe$var$height),
        width: $9440fa3fdd7adbfe$var$createSpectrumSizingProperty('width', $9440fa3fdd7adbfe$var$width),
        containIntrinsicWidth: $9440fa3fdd7adbfe$var$createSpectrumSizingProperty('containIntrinsicWidth', $9440fa3fdd7adbfe$var$width),
        containIntrinsicHeight: $9440fa3fdd7adbfe$var$createSpectrumSizingProperty('containIntrinsicHeight', $9440fa3fdd7adbfe$var$height),
        minHeight: $9440fa3fdd7adbfe$var$createSpectrumSizingProperty('minHeight', $9440fa3fdd7adbfe$var$height),
        maxHeight: $9440fa3fdd7adbfe$var$createSpectrumSizingProperty('maxHeight', (()=>{
            // auto is not a valid value for maxHeight
            // eslint-disable-next-line @typescript-eslint/no-unused-vars
            const { auto: auto, ...rest } = $9440fa3fdd7adbfe$var$height;
            return {
                ...rest,
                none: 'none'
            };
        })()),
        minWidth: $9440fa3fdd7adbfe$var$createSpectrumSizingProperty('minWidth', $9440fa3fdd7adbfe$var$width),
        maxWidth: $9440fa3fdd7adbfe$var$createSpectrumSizingProperty('maxWidth', (()=>{
            // auto is not a valid value for maxWidth
            // eslint-disable-next-line @typescript-eslint/no-unused-vars
            const { auto: auto, ...rest } = $9440fa3fdd7adbfe$var$width;
            return {
                ...rest,
                none: 'none'
            };
        })()),
        borderStartWidth: new (0, $e3c5b37c91268204$export$965169194d2485f7)('borderInlineStartWidth', $9440fa3fdd7adbfe$var$borderWidth),
        borderEndWidth: new (0, $e3c5b37c91268204$export$965169194d2485f7)('borderInlineEndWidth', $9440fa3fdd7adbfe$var$borderWidth),
        borderTopWidth: $9440fa3fdd7adbfe$var$borderWidth,
        borderBottomWidth: $9440fa3fdd7adbfe$var$borderWidth,
        borderStyle: [
            'solid',
            'dashed',
            'dotted',
            'double',
            'hidden',
            'none'
        ],
        strokeWidth: {
            0: '0',
            1: '1',
            2: '2'
        },
        marginStart: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('marginInlineStart', $9440fa3fdd7adbfe$var$margin),
        marginEnd: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('marginInlineEnd', $9440fa3fdd7adbfe$var$margin),
        marginTop: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('marginTop', $9440fa3fdd7adbfe$var$margin),
        marginBottom: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('marginBottom', $9440fa3fdd7adbfe$var$margin),
        paddingStart: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('paddingInlineStart', $9440fa3fdd7adbfe$var$padding),
        paddingEnd: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('paddingInlineEnd', $9440fa3fdd7adbfe$var$padding),
        paddingTop: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('paddingTop', $9440fa3fdd7adbfe$var$padding),
        paddingBottom: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('paddingBottom', $9440fa3fdd7adbfe$var$padding),
        scrollMarginStart: new (0, $e3c5b37c91268204$export$965169194d2485f7)('scrollMarginInlineStart', $9440fa3fdd7adbfe$var$baseSpacing),
        scrollMarginEnd: new (0, $e3c5b37c91268204$export$965169194d2485f7)('scrollMarginInlineEnd', $9440fa3fdd7adbfe$var$baseSpacing),
        scrollMarginTop: $9440fa3fdd7adbfe$var$baseSpacing,
        scrollMarginBottom: $9440fa3fdd7adbfe$var$baseSpacing,
        // Using rems instead of px here (unlike regular padding) because this often needs to match the height of something.
        scrollPaddingStart: new (0, $e3c5b37c91268204$export$965169194d2485f7)('scrollPaddingInlineStart', $9440fa3fdd7adbfe$var$baseSpacing),
        scrollPaddingEnd: new (0, $e3c5b37c91268204$export$965169194d2485f7)('scrollPaddingInlineEnd', $9440fa3fdd7adbfe$var$baseSpacing),
        scrollPaddingTop: $9440fa3fdd7adbfe$var$baseSpacing,
        scrollPaddingBottom: $9440fa3fdd7adbfe$var$baseSpacing,
        textIndent: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('textIndent', $9440fa3fdd7adbfe$var$baseSpacing),
        translateX: new (0, $e3c5b37c91268204$export$1b6028cfca42adca)([
            '--translateX',
            'translate'
        ], (value)=>({
                '--translateX': String(value),
                translate: 'var(--translateX, 0) var(--translateY, 0)'
            }), new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('--translateX', $9440fa3fdd7adbfe$var$translate)),
        translateY: new (0, $e3c5b37c91268204$export$1b6028cfca42adca)([
            '--translateY',
            'translate'
        ], (value)=>({
                '--translateY': String(value),
                translate: 'var(--translateX, 0) var(--translateY, 0)'
            }), new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('--translateY', $9440fa3fdd7adbfe$var$translate)),
        rotate: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('rotate', (value)=>typeof value === 'number' ? `${value}deg` : value),
        scaleX: new (0, $e3c5b37c91268204$export$1b6028cfca42adca)([
            '--scaleX',
            'scale'
        ], (value)=>({
                '--scaleX': String(value),
                scale: 'var(--scaleX, 1) var(--scaleY, 1)'
            })),
        scaleY: new (0, $e3c5b37c91268204$export$1b6028cfca42adca)([
            '--scaleY',
            'scale'
        ], (value)=>({
                '--scaleY': String(value),
                scale: 'var(--scaleX, 1) var(--scaleY, 1)'
            })),
        transform: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('transform'),
        position: [
            'absolute',
            'fixed',
            'relative',
            'sticky',
            'static'
        ],
        insetStart: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('insetInlineStart', $9440fa3fdd7adbfe$var$inset),
        insetEnd: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('insetInlineEnd', $9440fa3fdd7adbfe$var$inset),
        top: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('top', $9440fa3fdd7adbfe$var$inset),
        left: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('left', $9440fa3fdd7adbfe$var$inset),
        bottom: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('bottom', $9440fa3fdd7adbfe$var$inset),
        right: new (0, $e3c5b37c91268204$export$6bff0717a72a8669)('right', $9440fa3fdd7adbfe$var$inset),
        aspectRatio: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('aspectRatio', (value)=>{
            if (value === 'square') return '1/1';
            if (value === 'video') return '16/9';
            return value;
        }),
        // text
        fontFamily: {
            sans: {
                default: 'var(--s2-font-family-sans, adobe-clean-spectrum-vf), adobe-clean-variable, adobe-clean, ui-sans-serif, system-ui, sans-serif',
                ...$9440fa3fdd7adbfe$var$i18nFonts
            },
            serif: {
                default: 'var(--s2-font-family-serif, adobe-clean-spectrum-srf-vf), adobe-clean-serif, "Source Serif", Georgia, serif',
                ...$9440fa3fdd7adbfe$var$i18nFonts
            },
            code: 'source-code-pro, "Source Code Pro", Monaco, monospace'
        },
        fontSize: new (0, $e3c5b37c91268204$export$1b6028cfca42adca)([
            '--fs',
            'fontSize'
        ], (value)=>{
            if (typeof value === 'number') return {
                '--fs': `pow(1.125, ${value})`,
                fontSize: `round(${$9440fa3fdd7adbfe$var$fontSizeCalc} / 16 * ${process.env.DOCS_ENV ? 'var(--rem, 1rem)' : '1rem'}, 1px)`
            };
            return {
                fontSize: value
            };
        }, $9440fa3fdd7adbfe$var$fontSize),
        fontWeight: new (0, $e3c5b37c91268204$export$1b6028cfca42adca)([
            'fontWeight',
            'fontVariationSettings',
            'fontSynthesisWeight'
        ], (value)=>{
            return {
                fontWeight: value,
                fontSynthesisWeight: 'none'
            };
        }, $9440fa3fdd7adbfe$var$fontWeight),
        lineHeight: {
            // See https://spectrum.corp.adobe.com/page/typography/#Line-height
            ui: {
                // Calculate line-height based on font size.
                default: $9440fa3fdd7adbfe$var$lineHeightCalc,
                // CJK fonts use a larger line-height.
                ':lang(ja, ko, zh, zh-Hant, zh-Hans, zh-CN, zh-SG)': 1.5
            },
            heading: {
                default: $9440fa3fdd7adbfe$var$lineHeightCalc,
                ':lang(ja, ko, zh, zh-Hant, zh-Hans, zh-CN, zh-SG)': 1.5
            },
            title: {
                default: $9440fa3fdd7adbfe$var$lineHeightCalc,
                ':lang(ja, ko, zh, zh-Hant, zh-Hans, zh-CN, zh-SG)': 1.5
            },
            body: {
                // Body text uses spacious line height, 1.5 for all font sizes.
                default: 1.5,
                ':lang(ja, ko, zh, zh-Hant, zh-Hans, zh-CN, zh-SG)': 1.7
            },
            detail: {
                default: $9440fa3fdd7adbfe$var$lineHeightCalc,
                ':lang(ja, ko, zh, zh-Hant, zh-Hans, zh-CN, zh-SG)': 1.5
            },
            code: {
                default: 1.5,
                ':lang(ja, ko, zh, zh-Hant, zh-Hans, zh-CN, zh-SG)': 1.7
            }
        },
        listStyleType: [
            'none',
            'disc',
            'decimal'
        ],
        listStylePosition: [
            'inside',
            'outside'
        ],
        textTransform: [
            'uppercase',
            'lowercase',
            'capitalize',
            'none'
        ],
        textAlign: [
            'start',
            'center',
            'end',
            'justify'
        ],
        verticalAlign: [
            'baseline',
            'top',
            'middle',
            'bottom',
            'text-top',
            'text-bottom',
            'sub',
            'super'
        ],
        textDecoration: new (0, $e3c5b37c91268204$export$1b6028cfca42adca)([
            'textDecoration',
            'textUnderlineOffset'
        ], (value)=>({
                textDecoration: value === 'none' ? 'none' : `${value} ${"1px"}`,
                textUnderlineOffset: value === 'underline' ? "1px" : undefined
            })),
        textOverflow: [
            'ellipsis',
            'clip'
        ],
        lineClamp: new (0, $e3c5b37c91268204$export$1b6028cfca42adca)([
            'overflow',
            'display',
            '-webkit-box-orient',
            '-webkit-line-clamp'
        ], (value)=>({
                overflow: 'hidden',
                display: '-webkit-box',
                '-webkit-box-orient': 'vertical',
                '-webkit-line-clamp': String(value)
            })),
        hyphens: [
            'none',
            'manual',
            'auto'
        ],
        whiteSpace: [
            'normal',
            'nowrap',
            'pre',
            'pre-line',
            'pre-wrap',
            'break-spaces'
        ],
        textWrap: [
            'wrap',
            'nowrap',
            'balance',
            'pretty'
        ],
        wordBreak: [
            'normal',
            'break-all',
            'keep-all',
            'break-word'
        ],
        overflowWrap: [
            'normal',
            'anywhere',
            'break-word'
        ],
        boxDecorationBreak: [
            'slice',
            'clone'
        ],
        // effects
        boxShadow: {
            emphasized: [
                "0px 2px 8px light-dark(rgba(0, 0, 0, 0.08), rgba(0, 0, 0, 0.24))",
                "0px 1px 4px light-dark(rgba(0, 0, 0, 0.04), rgba(0, 0, 0, 0.12))",
                "0px 0px 1px light-dark(rgba(0, 0, 0, 0.08), rgba(0, 0, 0, 0.24))"
            ].join(', '),
            elevated: [
                "0px 4px 12px light-dark(rgba(0, 0, 0, 0.08), rgba(0, 0, 0, 0.24))",
                "0px 2px 6px light-dark(rgba(0, 0, 0, 0.04), rgba(0, 0, 0, 0.12))",
                "0px 0px 2px light-dark(rgba(0, 0, 0, 0.12), rgba(0, 0, 0, 0.36))"
            ].join(', '),
            dragged: [
                "0px 12px 16px light-dark(rgba(0, 0, 0, 0.08), rgba(0, 0, 0, 0.24))",
                "0px 6px 8px light-dark(rgba(0, 0, 0, 0.04), rgba(0, 0, 0, 0.12))",
                "0px 0px 6px light-dark(rgba(0, 0, 0, 0.16), rgba(0, 0, 0, 0.48))"
            ].join(', '),
            none: 'none'
        },
        filter: {
            // layer order is reversed for filter property. filters are applied in the order they are specified.
            emphasized: [
                "0px 2px 8px light-dark(rgba(0, 0, 0, 0.08), rgba(0, 0, 0, 0.24))",
                "0px 1px 4px light-dark(rgba(0, 0, 0, 0.04), rgba(0, 0, 0, 0.12))",
                "0px 0px 1px light-dark(rgba(0, 0, 0, 0.08), rgba(0, 0, 0, 0.24))"
            ].reverse().map((s)=>`drop-shadow(${s})`).join(' '),
            elevated: [
                "0px 4px 12px light-dark(rgba(0, 0, 0, 0.08), rgba(0, 0, 0, 0.24))",
                "0px 2px 6px light-dark(rgba(0, 0, 0, 0.04), rgba(0, 0, 0, 0.12))",
                "0px 0px 2px light-dark(rgba(0, 0, 0, 0.12), rgba(0, 0, 0, 0.36))"
            ].reverse().map((s)=>`drop-shadow(${s})`).join(' '),
            dragged: [
                "0px 12px 16px light-dark(rgba(0, 0, 0, 0.08), rgba(0, 0, 0, 0.24))",
                "0px 6px 8px light-dark(rgba(0, 0, 0, 0.04), rgba(0, 0, 0, 0.12))",
                "0px 0px 6px light-dark(rgba(0, 0, 0, 0.16), rgba(0, 0, 0, 0.48))"
            ].reverse().map((s)=>`drop-shadow(${s})`).join(' '),
            none: 'none'
        },
        borderTopStartRadius: new (0, $e3c5b37c91268204$export$965169194d2485f7)('borderStartStartRadius', $9440fa3fdd7adbfe$var$radius),
        borderTopEndRadius: new (0, $e3c5b37c91268204$export$965169194d2485f7)('borderStartEndRadius', $9440fa3fdd7adbfe$var$radius),
        borderBottomStartRadius: new (0, $e3c5b37c91268204$export$965169194d2485f7)('borderEndStartRadius', $9440fa3fdd7adbfe$var$radius),
        borderBottomEndRadius: new (0, $e3c5b37c91268204$export$965169194d2485f7)('borderEndEndRadius', $9440fa3fdd7adbfe$var$radius),
        forcedColorAdjust: [
            'auto',
            'none'
        ],
        colorScheme: [
            'light',
            'dark',
            'light dark'
        ],
        backgroundImage: new (0, $e3c5b37c91268204$export$1b6028cfca42adca)([
            'backgroundImage',
            '--g0',
            '--g1',
            '--g2',
            '--gp'
        ], (value)=>{
            if (typeof value === 'string') return {
                backgroundImage: value
            };
            else if (Array.isArray(value) && value[0]?.type === 'linear-gradient') {
                let values = {
                    backgroundImage: `linear-gradient(${value[0].angle}, ${value[0].stops.map(([, stop], i)=>`var(--g${i}) ${stop}%`)})`
                };
                // Create a CSS var for each color stop so the gradient can be transitioned.
                // These are registered via @property in the `linearGradient` macro.
                let properties = [];
                value[0].stops.forEach(([colorValue], i)=>{
                    properties.push(`--g${i}`);
                    values[`--g${i}`] = $9440fa3fdd7adbfe$export$35e9368ef982300f(colorValue);
                });
                // This is used by transition-property so we automatically transition all of the color stops.
                values['--gp'] = properties.join(', ');
                return values;
            } else throw new Error('Unexpected backgroundImage value: ' + JSON.stringify(value));
        }),
        // TODO: do we need separate x and y properties?
        backgroundPosition: [
            'bottom',
            'center',
            'left',
            'left bottom',
            'left top',
            'right',
            'right bottom',
            'right top',
            'top'
        ],
        backgroundSize: [
            'auto',
            'cover',
            'contain'
        ],
        backgroundAttachment: [
            'fixed',
            'local',
            'scroll'
        ],
        backgroundClip: [
            'border-box',
            'padding-box',
            'content-box',
            'text'
        ],
        backgroundRepeat: [
            'repeat',
            'no-repeat',
            'repeat-x',
            'repeat-y',
            'round',
            'space'
        ],
        backgroundOrigin: [
            'border-box',
            'padding-box',
            'content-box'
        ],
        backgroundBlendMode: [
            'normal',
            'multiply',
            'screen',
            'overlay',
            'darken',
            'lighten',
            'color-dodge',
            'color-burn',
            'hard-light',
            'soft-light',
            'difference',
            'exclusion',
            'hue',
            'saturation',
            'color',
            'luminosity'
        ],
        mixBlendMode: [
            'normal',
            'multiply',
            'screen',
            'overlay',
            'darken',
            'lighten',
            'color-dodge',
            'color-burn',
            'hard-light',
            'soft-light',
            'difference',
            'exclusion',
            'hue',
            'saturation',
            'color',
            'luminosity',
            'plus-darker',
            'plus-lighter'
        ],
        opacity: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('opacity'),
        outlineStyle: [
            'none',
            'solid',
            'dashed',
            'dotted',
            'double',
            'inset'
        ],
        outlineOffset: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('outlineOffset', (v)=>`${v}px`),
        outlineWidth: $9440fa3fdd7adbfe$var$borderWidth,
        transition: new (0, $e3c5b37c91268204$export$965169194d2485f7)('transitionProperty', $9440fa3fdd7adbfe$var$transitionProperty),
        transitionDelay: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('transitionDelay', $9440fa3fdd7adbfe$var$durationValue),
        transitionDuration: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('transitionDuration', $9440fa3fdd7adbfe$var$durationValue),
        transitionTimingFunction: $9440fa3fdd7adbfe$var$timingFunction,
        animation: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('animationName'),
        animationDuration: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('animationDuration', $9440fa3fdd7adbfe$var$durationValue),
        animationDelay: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('animationDelay', $9440fa3fdd7adbfe$var$durationValue),
        animationDirection: [
            'normal',
            'reverse',
            'alternate',
            'alternate-reverse'
        ],
        animationFillMode: [
            'none',
            'forwards',
            'backwards',
            'both'
        ],
        animationIterationCount: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('animationIterationCount'),
        animationTimingFunction: $9440fa3fdd7adbfe$var$timingFunction,
        animationPlayState: [
            'paused',
            'running'
        ],
        // layout
        display: [
            'block',
            'inline-block',
            'inline',
            'flex',
            'inline-flex',
            'grid',
            'inline-grid',
            'contents',
            'list-item',
            'none'
        ],
        alignContent: [
            'normal',
            'center',
            'start',
            'end',
            'space-between',
            'space-around',
            'space-evenly',
            'baseline',
            'stretch'
        ],
        alignItems: [
            'start',
            'end',
            'center',
            'baseline',
            'stretch'
        ],
        justifyContent: [
            'normal',
            'start',
            'end',
            'center',
            'space-between',
            'space-around',
            'space-evenly',
            'stretch'
        ],
        justifyItems: [
            'start',
            'end',
            'center',
            'stretch'
        ],
        alignSelf: [
            'auto',
            'start',
            'end',
            'center',
            'stretch',
            'baseline'
        ],
        justifySelf: [
            'auto',
            'start',
            'end',
            'center',
            'stretch'
        ],
        flexDirection: [
            'row',
            'column',
            'row-reverse',
            'column-reverse'
        ],
        flexWrap: [
            'wrap',
            'wrap-reverse',
            'nowrap'
        ],
        flexShrink: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('flexShrink'),
        flexGrow: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('flexGrow'),
        gridColumnStart: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('gridColumnStart'),
        gridColumnEnd: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('gridColumnEnd'),
        gridRowStart: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('gridRowStart'),
        gridRowEnd: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('gridRowEnd'),
        gridAutoFlow: [
            'row',
            'column',
            'dense',
            'row dense',
            'column dense'
        ],
        gridAutoRows: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('gridAutoRows', $9440fa3fdd7adbfe$var$gridTrackSize),
        gridAutoColumns: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('gridAutoColumns', $9440fa3fdd7adbfe$var$gridTrackSize),
        gridTemplateColumns: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('gridTemplateColumns', $9440fa3fdd7adbfe$var$gridTrack),
        gridTemplateRows: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('gridTemplateRows', $9440fa3fdd7adbfe$var$gridTrack),
        gridTemplateAreas: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('gridTemplateAreas', (value)=>value.map((v)=>`"${v}"`).join('')),
        float: [
            'inline-start',
            'inline-end',
            'right',
            'left',
            'none'
        ],
        clear: [
            'inline-start',
            'inline-end',
            'left',
            'right',
            'both',
            'none'
        ],
        contain: [
            'none',
            'strict',
            'content',
            'size',
            'inline-size',
            'layout',
            'style',
            'paint'
        ],
        containerType: [
            'normal',
            'size',
            'inline-size',
            'scroll-state'
        ],
        containerName: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('containerName'),
        boxSizing: [
            'border-box',
            'content-box'
        ],
        tableLayout: [
            'auto',
            'fixed'
        ],
        captionSide: [
            'top',
            'bottom'
        ],
        borderCollapse: [
            'collapse',
            'separate'
        ],
        breakBefore: [
            'auto',
            'avoid',
            'all',
            'avoid-page',
            'page',
            'left',
            'right',
            'column'
        ],
        breakInside: [
            'auto',
            'avoid',
            'avoid-page',
            'avoid-column'
        ],
        breakAfter: [
            'auto',
            'avoid',
            'all',
            'avoid-page',
            'page',
            'left',
            'right',
            'column'
        ],
        overflowX: [
            'auto',
            'hidden',
            'clip',
            'visible',
            'scroll'
        ],
        overflowY: [
            'auto',
            'hidden',
            'clip',
            'visible',
            'scroll'
        ],
        overscrollBehaviorX: [
            'auto',
            'contain',
            'none'
        ],
        overscrollBehaviorY: [
            'auto',
            'contain',
            'none'
        ],
        scrollBehavior: [
            'auto',
            'smooth'
        ],
        scrollbarWidth: [
            'none',
            'auto',
            'thin'
        ],
        order: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('order'),
        pointerEvents: [
            'none',
            'auto'
        ],
        touchAction: [
            'auto',
            'none',
            'pan-x',
            'pan-y',
            'manipulation',
            'pinch-zoom'
        ],
        userSelect: [
            'none',
            'text',
            'all',
            'auto'
        ],
        visibility: [
            'visible',
            'hidden',
            'collapse'
        ],
        isolation: [
            'isolate',
            'auto'
        ],
        transformOrigin: [
            'center',
            'top',
            'top right',
            'right',
            'bottom right',
            'bottom',
            'bottom left',
            'left'
        ],
        cursor: [
            'auto',
            'default',
            'pointer',
            'wait',
            'text',
            'move',
            'help',
            'not-allowed',
            'none',
            'context-menu',
            'progress',
            'cell',
            'crosshair',
            'vertical-text',
            'alias',
            'copy',
            'no-drop',
            'grab',
            'grabbing',
            'all-scroll',
            'col-resize',
            'row-resize',
            'n-resize',
            'e-resize',
            's-resize',
            'w-resize',
            'ne-resize',
            'nw-resize',
            'se-resize',
            'ew-resize',
            'ns-resize',
            'nesw-resize',
            'nwse-resize',
            'zoom-in',
            'zoom-out'
        ],
        resize: [
            'none',
            'vertical',
            'horizontal',
            'both'
        ],
        scrollSnapType: [
            'x',
            'y',
            'both',
            'x mandatory',
            'y mandatory',
            'both mandatory'
        ],
        scrollSnapAlign: [
            'start',
            'end',
            'center',
            'none'
        ],
        scrollSnapStop: [
            'normal',
            'always'
        ],
        appearance: [
            'none',
            'auto'
        ],
        objectFit: [
            'contain',
            'cover',
            'fill',
            'none',
            'scale-down'
        ],
        objectPosition: [
            'bottom',
            'center',
            'left',
            'left bottom',
            'left top',
            'right',
            'right bottom',
            'right top',
            'top'
        ],
        willChange: [
            'auto',
            'scroll-position',
            'contents',
            'transform'
        ],
        zIndex: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('zIndex'),
        disableTapHighlight: new (0, $e3c5b37c91268204$export$d03475ad9df06cd2)('-webkit-tap-highlight-color', // eslint-disable-next-line @typescript-eslint/no-unused-vars
        (_value)=>'rgba(0,0,0,0)'),
        unicodeBidi: [
            'normal',
            'embed',
            'bidi-override',
            'isolate',
            'isolate-override',
            'plaintext'
        ],
        caretColor: [
            'auto',
            'transparent'
        ]
    },
    shorthands: {
        padding: [
            'paddingTop',
            'paddingBottom',
            'paddingStart',
            'paddingEnd'
        ],
        paddingX: [
            'paddingStart',
            'paddingEnd'
        ],
        paddingY: [
            'paddingTop',
            'paddingBottom'
        ],
        margin: [
            'marginTop',
            'marginBottom',
            'marginStart',
            'marginEnd'
        ],
        marginX: [
            'marginStart',
            'marginEnd'
        ],
        marginY: [
            'marginTop',
            'marginBottom'
        ],
        scrollPadding: [
            'scrollPaddingTop',
            'scrollPaddingBottom',
            'scrollPaddingStart',
            'scrollPaddingEnd'
        ],
        scrollPaddingX: [
            'scrollPaddingStart',
            'scrollPaddingEnd'
        ],
        scrollPaddingY: [
            'scrollPaddingTop',
            'scrollPaddingBottom'
        ],
        scrollMargin: [
            'scrollMarginTop',
            'scrollMarginBottom',
            'scrollMarginStart',
            'scrollMarginEnd'
        ],
        scrollMarginX: [
            'scrollMarginStart',
            'scrollMarginEnd'
        ],
        scrollMarginY: [
            'scrollMarginTop',
            'scrollMarginBottom'
        ],
        borderWidth: [
            'borderTopWidth',
            'borderBottomWidth',
            'borderStartWidth',
            'borderEndWidth'
        ],
        borderXWidth: [
            'borderStartWidth',
            'borderEndWidth'
        ],
        borderYWidth: [
            'borderTopWidth',
            'borderBottomWidth'
        ],
        borderRadius: [
            'borderTopStartRadius',
            'borderTopEndRadius',
            'borderBottomStartRadius',
            'borderBottomEndRadius'
        ],
        borderTopRadius: [
            'borderTopStartRadius',
            'borderTopEndRadius'
        ],
        borderBottomRadius: [
            'borderBottomStartRadius',
            'borderBottomEndRadius'
        ],
        borderStartRadius: [
            'borderTopStartRadius',
            'borderBottomStartRadius'
        ],
        borderEndRadius: [
            'borderTopEndRadius',
            'borderBottomEndRadius'
        ],
        translate: [
            'translateX',
            'translateY'
        ],
        scale: [
            'scaleX',
            'scaleY'
        ],
        inset: [
            'top',
            'bottom',
            'insetStart',
            'insetEnd'
        ],
        insetX: [
            'insetStart',
            'insetEnd'
        ],
        insetY: [
            'top',
            'bottom'
        ],
        placeItems: [
            'alignItems',
            'justifyItems'
        ],
        placeContent: [
            'alignContent',
            'justifyContent'
        ],
        placeSelf: [
            'alignSelf',
            'justifySelf'
        ],
        gap: [
            'rowGap',
            'columnGap'
        ],
        size: [
            'width',
            'height'
        ],
        minSize: [
            'minWidth',
            'minHeight'
        ],
        maxSize: [
            'maxWidth',
            'maxHeight'
        ],
        overflow: [
            'overflowX',
            'overflowY'
        ],
        overscrollBehavior: [
            'overscrollBehaviorX',
            'overscrollBehaviorY'
        ],
        gridArea: [
            'gridColumnStart',
            'gridColumnEnd',
            'gridRowStart',
            'gridRowEnd'
        ],
        transition: (value)=>({
                transition: value,
                transitionDuration: 150,
                transitionTimingFunction: 'default'
            }),
        animation: (value)=>({
                animation: value,
                animationDuration: 150,
                animationTimingFunction: 'default'
            }),
        truncate: (value)=>({
                overflowX: value ? 'hidden' : 'visible',
                overflowY: value ? 'hidden' : 'visible',
                textOverflow: value ? 'ellipsis' : 'clip',
                whiteSpace: value ? 'nowrap' : 'normal'
            }),
        font: (value)=>{
            let type = value.split('-')[0];
            return {
                fontFamily: type === 'code' ? 'code' : 'sans',
                fontSize: value,
                fontWeight: type === 'heading' || type === 'title' || type === 'detail' ? type : 'normal',
                lineHeight: type,
                color: type === 'ui' ? 'body' : type
            };
        }
    },
    conditions: {
        // In the docs we need to be able to simulate HCM.
        forcedColors: process.env.DOCS_ENV ? [
            '@media (forced-colors: active)',
            ':is([data-hcm], [data-hcm] *)'
        ] : '@media (forced-colors: active)',
        // This detects touch primary devices as best as we can.
        // Ideally we'd use (pointer: course) but browser/device support is inconsistent.
        // Samsung Android devices claim to be mice at the hardware/OS level: (any-pointer: fine), (any-hover: hover), (hover: hover), and nothing for pointer.
        // More details: https://www.ctrl.blog/entry/css-media-hover-samsung.html
        // iPhone matches (any-hover: none), (hover: none), and nothing for any-pointer or pointer.
        // If a trackpad or Apple Pencil is connected to iPad, it matches (any-pointer: fine), (any-hover: hover), (hover: none).
        // Windows tablet matches the same as iPhone. No difference when a mouse is connected.
        // Windows touch laptop matches same as macOS: (any-pointer: fine), (pointer: fine), (any-hover: hover), (hover: hover).
        touch: '@media not ((hover: hover) and (pointer: fine))',
        xs: `@media (min-width: ${30}rem)`,
        sm: `@media (min-width: ${40}rem)`,
        md: `@media (min-width: ${48}rem)`,
        lg: `@media (min-width: ${64}rem)`,
        xl: `@media (min-width: ${80}rem)`,
        '2xl': `@media (min-width: ${96}rem)`
    }
});


export {$9440fa3fdd7adbfe$export$49bbad2a916ab57c as baseColor, $9440fa3fdd7adbfe$export$35e9368ef982300f as color, $9440fa3fdd7adbfe$export$d19e682d444755ed as lightDark, $9440fa3fdd7adbfe$export$9b476054b78b89cd as colorMix, $9440fa3fdd7adbfe$export$46def8197cf4dd4c as linearGradient, $9440fa3fdd7adbfe$export$ba024f4caf693d15 as fontRelative, $9440fa3fdd7adbfe$export$17b63977b35bb6d2 as edgeToText, $9440fa3fdd7adbfe$export$a941ed4b947d12f8 as space, $9440fa3fdd7adbfe$export$346677f925de839c as size, $9440fa3fdd7adbfe$export$1d567c320f4763bc as style};
