// @react-spectrum/s2/illustrations/linear/internal/S2_lin_paperairplane_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2vH9F$reactjsxruntime = require("react/jsx-runtime");
var $2vH9F$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $cd2462bd4d9ef980$export$2e2bcd8739ae039);




const $cd2462bd4d9ef980$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2vH9F$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2vH9F$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M147.987 38.528c-2.531-2.703-6.395-4.19-10.589-3.246l-119.72 26.96c-9.44 2.126-11.871 14.454-3.847 19.943L38 98.72v32.097c0 7.625 8.973 11.238 14.52 6.622l18.985-15.798 21.065 14.41c5.543 3.792 13.194 1.865 16.238-4.139l40.98-80.825c1.684-3.32 1.531-6.797.171-9.666-.063-.43-.198-.856-.409-1.263-.368-.708-.918-1.26-1.563-1.629m-15.768 6.12L19.436 70.046c-2.678.603-3.298 4.024-1.088 5.536L41.776 91.61zm-81.83 52.854 46.698 31.946c1.579 1.08 3.74.512 4.586-1.155l40.445-79.77-92.835 48.202zM46 104.195v26.621c0 .163.042.255.091.324.06.088.174.193.346.27.35.156.69.11.967-.121l17.02-14.163z"
        })
    });
const $cd2462bd4d9ef980$var$ForwardRef = /*#__PURE__*/ (0, $2vH9F$react.forwardRef)($cd2462bd4d9ef980$var$SvgComponent);
var $cd2462bd4d9ef980$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($cd2462bd4d9ef980$var$ForwardRef);
