// @react-spectrum/s2/illustrations/linear/internal/S2_lin_vectorDraw_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $fHI0V$jsx} from "react/jsx-runtime";
import {forwardRef as $fHI0V$forwardRef} from "react";





const $952268b2980a2914$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fHI0V$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fHI0V$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M33.723 7.34c-1.647-1.646-4.274-1.32-5.854.259l-2.273 2.273-11.263 4.808c-1.244.53-2.22 1.604-2.62 2.895l-5.2 16.8c-1.353 4.37 2.744 8.466 7.114 7.114l16.8-5.2c1.29-.4 2.364-1.377 2.895-2.62l4.808-11.264 2.273-2.273c1.579-1.579 1.905-4.207.258-5.854zM15.315 16.98l10.708-4.571 9.57 9.57-4.57 10.709c-.247.577-.755 1.034-1.336 1.214l-16.8 5.2c-.72.223-1.427.181-2.047-.05l10.05-10.01q.286.07.593.07c1.38 0 2.5-1.118 2.5-2.5 0-1.38-1.12-2.5-2.5-2.5s-2.5 1.12-2.5 2.5q.001.368.1.702L9.02 37.333c-.292-.659-.364-1.43-.12-2.219l5.2-16.8c.18-.58.636-1.089 1.214-1.335m23.32 1.386-1.56 1.56-8.998-8.998 1.56-1.56c.82-.82 1.851-.726 2.318-.259l6.938 6.938c.467.467.562 1.499-.258 2.319"
        })
    });
const $952268b2980a2914$var$ForwardRef = /*#__PURE__*/ (0, $fHI0V$forwardRef)($952268b2980a2914$var$SvgComponent);
var $952268b2980a2914$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($952268b2980a2914$var$ForwardRef);


export {$952268b2980a2914$export$2e2bcd8739ae039 as default};
