// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noFilter_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1X0Mp$jsx} from "react/jsx-runtime";
import {forwardRef as $1X0Mp$forwardRef} from "react";





const $56d1bfeb937d0b87$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1X0Mp$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1X0Mp$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M30.655 10.16c0-4.141 3.359-7.5 7.5-7.5s7.5 3.359 7.5 7.5c0 4.142-3.359 7.5-7.5 7.5s-7.5-3.358-7.5-7.5m10.859 3.359c.485-.486.485-1.268 0-1.754h.02l-1.605-1.605 1.604-1.605c.486-.485.486-1.268 0-1.753-.485-.486-1.268-.486-1.753 0l-1.605 1.605L36.57 6.8c-.486-.485-1.268-.485-1.754 0-.485.486-.485 1.269 0 1.754l1.605 1.605-1.605 1.605c-.485.486-.485 1.268 0 1.754.238.248.555.366.872.366s.634-.128.872-.366l1.605-1.605 1.605 1.605c.238.248.555.366.872.366s.634-.128.872-.366M28.456 9.92c.681-.113 1.141-.757 1.028-1.438-.109-.65-.7-1.099-1.346-1.04l-.062.002-.318.006c-.278.003-.674.006-1.167.008-.985.004-2.349.006-3.916.006-3.133 0-7.073-.005-10.411-.013-3.02-.006-4.67 3.52-2.731 5.834l10.715 12.794v13.883c0 1.941 2.293 2.972 3.744 1.683l3.012-2.674c.48-.427.756-1.04.756-1.682V26.076l5.672-6.84c.44-.532.367-1.32-.165-1.76-.531-.441-1.32-.368-1.76.164l-5.729 6.91c-.335.403-.518.911-.518 1.436v11.19l-2.512 2.23V25.988c0-.528-.186-1.04-.525-1.444L11.45 11.68c-.575-.686-.085-1.73.808-1.729 3.34.008 7.282.014 10.418.013 1.568 0 2.935-.002 3.926-.006.495-.003.898-.005 1.186-.009q.216-.002.35-.005l.122-.004h.002c.011 0 .103-.004.194-.02m-.296-2.478h.002-.014z"
        })
    });
const $56d1bfeb937d0b87$var$ForwardRef = /*#__PURE__*/ (0, $1X0Mp$forwardRef)($56d1bfeb937d0b87$var$SvgComponent);
var $56d1bfeb937d0b87$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($56d1bfeb937d0b87$var$ForwardRef);


export {$56d1bfeb937d0b87$export$2e2bcd8739ae039 as default};
