// @react-spectrum/s2/illustrations/linear/internal/S2_lin_libraries_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $lVLvi$reactjsxruntime = require("react/jsx-runtime");
var $lVLvi$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d87e56bca9fcaee1$export$2e2bcd8739ae039);




const $d87e56bca9fcaee1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lVLvi$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $lVLvi$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20.53 28.52c0-4.416 3.584-8 8-8h31.95c1.104 0 2-.896 2-2s-.896-2-2-2H28.53c-6.625 0-12 5.375-12 12v31.95c0 1.104.895 2 2 2s2-.896 2-2zm13-.99c-3.314 0-6 2.686-6 6v35.95c0 3.313 2.686 6 6 6h35.95c3.313 0 6-2.687 6-6V33.53c0-3.314-2.687-6-6-6h-.39v22.21c0 .8-.89 1.27-1.55.83l-4.87-3.25c-.34-.22-.77-.22-1.11 0l-4.87 3.25c-.66.45-1.55-.03-1.55-.83V27.53zm-10 6c0-5.523 4.476-10 10-10h35.95c5.522 0 10 4.477 10 10v35.95c0 5.522-4.478 10-10 10H33.53c-5.524 0-10-4.478-10-10z"
        })
    });
const $d87e56bca9fcaee1$var$ForwardRef = /*#__PURE__*/ (0, $lVLvi$react.forwardRef)($d87e56bca9fcaee1$var$SvgComponent);
var $d87e56bca9fcaee1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d87e56bca9fcaee1$var$ForwardRef);
