// @react-spectrum/s2/illustrations/linear/internal/S2_lin_interaction_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $akU2B$reactjsxruntime = require("react/jsx-runtime");
var $akU2B$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6e05944633a2244f$export$2e2bcd8739ae039);




const $6e05944633a2244f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $akU2B$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $akU2B$reactjsxruntime.jsx)("path", {
            fillRule: "evenodd",
            d: "M31.01 56.39c0-15.831 12.834-28.665 28.665-28.665 15.832 0 28.666 12.834 28.666 28.665 0 2.83-.41 5.556-1.168 8.128-.625 2.12.585 4.344 2.704 4.969s4.344-.586 4.969-2.705c.973-3.3 1.495-6.788 1.495-10.392 0-20.25-16.416-36.665-36.666-36.665S23.01 36.14 23.01 56.39c0 17.24 11.894 31.692 27.92 35.615 2.145.525 4.31-.788 4.836-2.934.525-2.146-.788-4.311-2.934-4.837C40.302 81.167 31.01 69.86 31.01 56.39m31.833 3.994c.673-.364 1.408-.407 2.21-.109.84.313 1.711 1.001 2.324 2.004l12.464 22.434c1.316 2.37 4.328 2.984 6.463 1.546 2.21-1.488 5.862-3.186 9.837-4.516 4.003-1.34 7.854-2.145 10.457-2.145 7.335 0 12.784 2.792 16.468 7.224 3.755 4.516 5.925 11.049 5.925 18.804 0 14.748-11.787 26.65-26.476 26.65-7.62 0-17.679-3.183-26.838-6.945-7.702-3.163-14.328-6.544-17.749-8.289-.591-.302-1.087-.555-1.476-.749-2.032-1.016-2.766-1.959-3.051-2.583-.277-.606-.4-1.501.01-2.996.483-1.753 2.39-3.139 5.266-2.321 3.393.965 7.189 2.494 11.093 4.157.895.381 1.8.771 2.706 1.161l.004.002c2.943 1.268 5.892 2.539 8.513 3.509 4.02 1.488 7.45-2.849 5.48-6.395L61.565 65.993l-.038-.068-.04-.067c-1.132-1.86-.368-4.54 1.356-5.474m11.43-2.163c-2.845-4.758-9.22-8.135-15.242-4.87-5.908 3.202-7.758 11.01-4.426 16.588L74.59 105.91q-.831-.358-1.685-.721c-3.918-1.668-8.118-3.376-12.039-4.491-6.383-1.815-13.305 1.123-15.167 7.895-.739 2.685-.862 5.622.426 8.441 1.28 2.801 3.674 4.876 6.75 6.414.353.176.816.413 1.378.7 3.38 1.725 10.34 5.279 18.385 8.583 9.257 3.802 20.57 7.545 29.878 7.545 19.167 0 34.476-15.543 34.476-34.65 0-9.172-2.564-17.654-7.773-23.92-5.28-6.35-13.028-10.108-22.62-10.108-3.859 0-8.657 1.107-12.996 2.559-2.972.994-5.95 2.226-8.501 3.552L74.335 58.33l-.03-.056z"
        })
    });
const $6e05944633a2244f$var$ForwardRef = /*#__PURE__*/ (0, $akU2B$react.forwardRef)($6e05944633a2244f$var$SvgComponent);
var $6e05944633a2244f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6e05944633a2244f$var$ForwardRef);
