// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lockClose_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $i6FhB$reactjsxruntime = require("react/jsx-runtime");
var $i6FhB$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $261e35b813e41e76$export$2e2bcd8739ae039);




const $261e35b813e41e76$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $i6FhB$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $i6FhB$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 20.5c-14.36 0-26 11.64-26 26v17h-9.636C35.129 63.5 27 70.409 27 79.688v40.624c0 9.279 8.129 16.188 17.364 16.188h71.272c9.235 0 17.364-6.909 17.364-16.188V79.688c0-9.28-8.129-16.188-17.364-16.188H106v-17c0-14.36-11.64-26-26-26m18 43v-17c0-9.941-8.059-18-18-18s-18 8.059-18 18v17zm-40 8H44.364C38.837 71.5 35 75.504 35 79.688v40.624c0 4.183 3.837 8.188 9.364 8.188h71.272c5.527 0 9.364-4.005 9.364-8.188V79.688c0-4.184-3.837-8.188-9.364-8.188H58m27.654 29.153c-.493.497-1.051.924-1.654 1.271v7.576c0 2.209-1.79 4-4 4s-4-1.791-4-4v-7.576c-.603-.347-1.16-.774-1.654-1.271-.373-.36-.707-.773-1-1.214-.28-.426-.534-.893-.733-1.373-.2-.493-.36-.986-.453-1.506-.107-.507-.16-1.04-.16-1.56s.053-1.053.16-1.56c.093-.507.253-1.014.453-1.493.2-.494.453-.947.733-1.387.293-.44.627-.854 1-1.214C75.826 87.853 77.893 87 80 87s4.173.853 5.654 2.346c.372.36.705.774 1 1.214.293.44.532.893.733 1.387.199.48.36.986.453 1.492.107.508.16 1.04.16 1.561s-.053 1.053-.16 1.56c-.094.52-.254 1.013-.453 1.506-.2.48-.44.947-.733 1.373-.295.44-.628.854-1 1.214"
        })
    });
const $261e35b813e41e76$var$ForwardRef = /*#__PURE__*/ (0, $i6FhB$react.forwardRef)($261e35b813e41e76$var$SvgComponent);
var $261e35b813e41e76$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($261e35b813e41e76$var$ForwardRef);
