// @react-spectrum/s2/illustrations/linear/internal/S2_lin_revert_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2L3Ck$reactjsxruntime = require("react/jsx-runtime");
var $2L3Ck$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $41a22d5f3c549763$export$2e2bcd8739ae039);




const $41a22d5f3c549763$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2L3Ck$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2L3Ck$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m19.943 52.95 10.935-8.574c.87-.682 2.126-.53 2.808.34.681.869.53 2.126-.34 2.808L18.777 58.948c-.869.681-2.126.53-2.808-.34L4.546 44.04c-.681-.869-.53-2.126.34-2.808s2.126-.53 2.808.34l8.309 10.596c1.858-17.47 16.643-31.08 34.606-31.08 19.22 0 34.803 15.583 34.803 34.804 0 1.104-.896 2-2 2s-2-.896-2-2c0-17.012-13.791-30.803-30.803-30.803-16.02 0-29.184 12.23-30.665 27.862m-5.86 20.947c0-1.105.895-2 2-2h69.03c1.105 0 2 .895 2 2s-.895 2-2 2h-69.03c-1.105 0-2-.896-2-2"
        })
    });
const $41a22d5f3c549763$var$ForwardRef = /*#__PURE__*/ (0, $2L3Ck$react.forwardRef)($41a22d5f3c549763$var$SvgComponent);
var $41a22d5f3c549763$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($41a22d5f3c549763$var$ForwardRef);
