// @react-spectrum/s2/illustrations/linear/internal/S2_lin_addproject_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $cfGbR$reactjsxruntime = require("react/jsx-runtime");
var $cfGbR$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $197a4eab2cef2203$export$2e2bcd8739ae039);




const $197a4eab2cef2203$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cfGbR$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cfGbR$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M26.562 27.475C27.18 25.977 28.64 25 30.26 25h32.23c1.62 0 3.08.977 3.698 2.475l2.615 6.344q-.517-.069-1.053-.069H25q-.536 0-1.053.069zm48.287 10.513q.045.11.077.221c.527 1.067.824 2.27.824 3.541v7.75c0 1.105-.895 2-2 2s-2-.895-2-2v-7.75c0-2.21-1.79-4-4-4H25c-2.21 0-4 1.79-4 4v21c0 2.21 1.79 4 4 4h29.5c1.105 0 2 .895 2 2s-.895 2-2 2H25c-4.418 0-8-3.582-8-8v-21c0-1.271.297-2.474.824-3.54q.032-.113.077-.222l4.963-12.037C24.099 22.955 27.02 21 30.26 21h32.23c3.24 0 6.16 1.955 7.396 4.95zM38.5 50.5H55c1.105 0 2-.895 2-2s-.895-2-2-2H38.5c-1.105 0-2 .895-2 2s.895 2 2 2m35.19 32.18c7.672 0 13.89-6.219 13.89-13.89S81.363 54.9 73.69 54.9c-7.67 0-13.89 6.219-13.89 13.89s6.22 13.89 13.89 13.89M75.8 62.5c0-1.105-.895-2-2-2s-2 .895-2 2v4.25h-4.359c-1.104 0-2 .895-2 2s.896 2 2 2h4.36V75c0 1.105.895 2 2 2s2-.895 2-2v-4.25h4.14c1.105 0 2-.895 2-2s-.895-2-2-2h-4.14z"
        })
    });
const $197a4eab2cef2203$var$ForwardRef = /*#__PURE__*/ (0, $cfGbR$react.forwardRef)($197a4eab2cef2203$var$SvgComponent);
var $197a4eab2cef2203$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($197a4eab2cef2203$var$ForwardRef);
