// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noElements_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2DkCt$reactjsxruntime = require("react/jsx-runtime");
var $2DkCt$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0df1df68aa8bfd36$export$2e2bcd8739ae039);




const $0df1df68aa8bfd36$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2DkCt$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2DkCt$reactjsxruntime.jsx)("path", {
                stroke: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                strokeLinecap: "round",
                strokeLinejoin: "round",
                strokeWidth: 8,
                d: "M43.904 93.334v46.91M34.523 140.244h18.765M20.451 93.334h46.91M20.451 93.334v9.382M67.361 93.334v9.382"
            }),
            /*#__PURE__*/ (0, $2DkCt$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M135.987 20.165c-1.487-2.337-4.032-3.899-6.833-4.08Zm-6.833-4.08c-1.734-.313-3.281.29-4.126.71-1.111.555-2.166 1.372-2.977 2.309l-.005.006-20.54 23.904-.006.007c-1.83 2.14-2.557 5.035-1.964 7.795.588 2.736 2.423 5.116 5.043 6.286h.002l4.249 1.904-3.764 9.281-.002.005c-1.102 2.702-.858 5.807.729 8.287 1.591 2.498 4.38 4.087 7.379 4.087 2.555 0 4.892-1.132 6.533-3.03l20.427-23.572c1.873-2.156 2.59-5.081 2.001-7.839-.59-2.756-2.444-5.144-5.095-6.302l-.007-.003-4.245-1.867 3.919-9.59.003-.007c1.096-2.698.871-5.793-.72-8.29m-7.889 4.174L107.58 48.22l-.002.003c-.197.231-.292.578-.22.913.077.359.301.58.483.661l.005.002 11.299 5.06-6.669 16.444-.004.01c-.146.356-.09.72.06.954l.004.007.005.007c.168.264.45.385.631.385.122 0 .287-.037.483-.264l20.437-23.583c.187-.216.293-.566.217-.921-.075-.353-.292-.565-.473-.643l-.006-.003-11.397-5.013 6.864-16.795.001-.003c.153-.38.093-.74-.057-.976l-.003-.005c-.175-.276-.463-.393-.631-.393h-.195q-.04.03-.084.063c-.105.08-.182.156-.226.207zM71.207 36.667c0-5.154-4.179-9.333-9.333-9.333H24.999c-5.155 0-9.334 4.179-9.334 9.333v21.25q0 .398.033.787l10.006-15.459.001-.002c3.03-4.681 9.886-4.681 12.915 0v.002l9.146 14.127 4.448-6.725.017-.026c3.045-4.529 9.72-4.543 12.77 0l6.056 8.972q.15-.817.15-1.676zM32.163 47.99 20.458 66.073c1.344.75 2.892 1.178 4.54 1.178h16.234l1.718-2.598zm33.782 18.328c-1.23.598-2.612.933-4.071.933h-11.05l7.795-11.786zm-4.071-46.984c9.573 0 17.333 7.76 17.333 17.333v21.25c0 9.573-7.76 17.334-17.333 17.334H24.999c-9.573 0-17.334-7.76-17.334-17.334v-21.25c0-9.573 7.76-17.333 17.334-17.333zm-3.9 16.285c0 2.397 1.943 4.34 4.34 4.34s4.34-1.943 4.34-4.34c0-2.398-1.943-4.34-4.34-4.34s-4.34 1.942-4.34 4.34M148.5 106.667c0-5.154-4.179-9.333-9.333-9.333h-35.834c-5.154 0-9.333 4.179-9.333 9.333v21.25c0 5.155 4.179 9.334 9.333 9.334h35.834c5.154 0 9.333-4.179 9.333-9.334zm-9.333-17.333c9.573 0 17.333 7.76 17.333 17.333v21.25c0 9.573-7.76 17.334-17.333 17.334h-35.834C93.76 145.251 86 137.49 86 127.917v-21.25c0-9.573 7.76-17.333 17.333-17.333zm-21.375 19.08 12.296 7.019c1.377.786 1.375 2.747-.004 3.53l-12.314 6.99c-1.379.783-3.1-.199-3.098-1.768l.017-14.01c.002-1.569 1.726-2.548 3.103-1.761"
            })
        ]
    });
const $0df1df68aa8bfd36$var$ForwardRef = /*#__PURE__*/ (0, $2DkCt$react.forwardRef)($0df1df68aa8bfd36$var$SvgComponent);
var $0df1df68aa8bfd36$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0df1df68aa8bfd36$var$ForwardRef);
