// @react-spectrum/s2/illustrations/linear/internal/S2_lin_replace_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3st3S$reactjsxruntime = require("react/jsx-runtime");
var $3st3S$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3fc6285ebee86302$export$2e2bcd8739ae039);




const $3fc6285ebee86302$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3st3S$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3st3S$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.229 24.25c0-4.418 3.581-8 8-8H34.85c4.419 0 8 3.582 8 8v10.622c0 4.418-3.581 8-8 8H24.228c-4.418 0-8-3.582-8-8zm8-4c-2.21 0-4 1.79-4 4v10.622c0 2.209 1.79 4 4 4H34.85c2.21 0 4-1.791 4-4V24.25c0-2.21-1.79-4-4-4zm30.646-2.889c-1.104-.037-2.03.828-2.067 1.932s.828 2.029 1.931 2.066c2.85.097 5.801.919 8.036 3.08 2.03 1.965 3.718 5.278 3.976 10.878l-4.111-4.111c-.781-.781-2.047-.781-2.828 0-.781.78-.781 2.047 0 2.828l7.144 7.145c.781.78 2.048.78 2.829 0l7.167-7.168c.781-.781.781-2.047 0-2.828-.78-.781-2.047-.781-2.828 0l-3.407 3.406c-.378-6.05-2.3-10.257-5.161-13.024-3.176-3.072-7.213-4.086-10.681-4.204M28.95 54.822c-.781-.78-2.048-.78-2.829 0l-7.167 7.168c-.781.781-.781 2.048 0 2.829.78.78 2.047.78 2.828 0l3.406-3.407c.378 6.051 2.302 10.257 5.162 13.024 3.175 3.072 7.213 4.086 10.68 4.203 1.105.038 2.03-.827 2.068-1.93.037-1.105-.828-2.03-1.932-2.067-2.849-.097-5.8-.919-8.035-3.08-2.03-1.965-3.718-5.278-3.977-10.878l4.112 4.111c.781.782 2.047.782 2.828 0 .782-.78.782-2.047 0-2.828zm32.198-1.693c-4.418 0-8 3.582-8 8V71.75c0 4.418 3.582 8 8 8H71.77c4.418 0 8-3.582 8-8V61.128c0-4.418-3.582-8-8-8zm-4 8c0-2.21 1.79-4 4-4H71.77c2.209 0 4 1.79 4 4V71.75c0 2.209-1.791 4-4 4H61.149c-2.21 0-4-1.791-4-4z"
        })
    });
const $3fc6285ebee86302$var$ForwardRef = /*#__PURE__*/ (0, $3st3S$react.forwardRef)($3fc6285ebee86302$var$SvgComponent);
var $3fc6285ebee86302$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3fc6285ebee86302$var$ForwardRef);
