// @react-spectrum/s2/illustrations/linear/internal/S2_lin_mention_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $zBJVD$jsx} from "react/jsx-runtime";
import {forwardRef as $zBJVD$forwardRef} from "react";





const $cb8de5d8e0bbbfcf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $zBJVD$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $zBJVD$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M63.1 23.29c16.545-5.425 34.284-3.612 45.82 1.593 23.659 10.674 31.47 33.612 28.012 53.264-3.429 19.484-18.327 37.023-40.869 35.464-20.758-1.436-31.9-9.307-36.412-19.61-4.341-9.914-1.93-20.8 1.58-27.28 10.008-18.484 27.797-19.679 36.932-18.844 4.498.411 7.267 4.463 6.897 8.48l-4.533 49.295c15.125-.906 25.791-13.35 28.526-28.892 2.936-16.683-3.649-35.664-23.423-44.585-9.715-4.384-25.39-6.085-40.037-1.282-14.408 4.724-27.845 15.736-33.65 37.04-4.597 16.876.774 33.035 11.113 45.01 10.4 12.046 25.48 19.448 39.591 19.041 2.208-.064 4.05 1.675 4.114 3.883.063 2.208-1.675 4.05-3.883 4.114-16.923.488-34.192-8.275-45.877-21.81-11.747-13.606-18.197-32.444-12.776-52.34 6.56-24.077 22.092-37.036 38.875-42.54m29.43 81.953 4.546-49.43c-7.944-.654-21.196.653-28.81 14.717-2.707 4.998-4.363 13.237-1.286 20.263 2.688 6.14 9.53 12.492 25.55 14.45"
        })
    });
const $cb8de5d8e0bbbfcf$var$ForwardRef = /*#__PURE__*/ (0, $zBJVD$forwardRef)($cb8de5d8e0bbbfcf$var$SvgComponent);
var $cb8de5d8e0bbbfcf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cb8de5d8e0bbbfcf$var$ForwardRef);


export {$cb8de5d8e0bbbfcf$export$2e2bcd8739ae039 as default};
