// @react-spectrum/s2/illustrations/linear/internal/S2_lin_properties_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $hnvjC$jsx} from "react/jsx-runtime";
import {forwardRef as $hnvjC$forwardRef} from "react";





const $ed5fba8ef591fd80$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hnvjC$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hnvjC$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M64.672 44.865c-5.476 0-9.916 4.44-9.916 9.916 0 5.477 4.44 9.916 9.916 9.916 5.466 0 9.9-4.423 9.916-9.885v-.061c-.016-5.463-4.45-9.886-9.916-9.886m0 27.832c8.52 0 15.65-5.947 17.468-13.916h49.61c2.209 0 4-1.79 4-4s-1.791-4-4-4H82.14c-1.817-7.969-8.948-13.916-17.468-13.916s-15.65 5.947-17.467 13.916H28.25c-2.21 0-4 1.791-4 4s1.79 4 4 4h18.955c1.817 7.97 8.948 13.916 17.467 13.916m13.188 36.522H28.25c-2.209 0-4-1.791-4-4s1.791-4 4-4h49.61c1.817-7.97 8.948-13.916 17.468-13.916s15.65 5.947 17.468 13.916h18.954c2.209 0 4 1.791 4 4s-1.791 4-4 4h-18.954c-1.818 7.969-8.948 13.915-17.468 13.915s-15.65-5.946-17.468-13.915m27.384-4.018v.039c-.012 5.466-4.447 9.894-9.916 9.894-5.476 0-9.916-4.439-9.916-9.915 0-5.477 4.44-9.916 9.916-9.916 5.471 0 9.906 4.43 9.916 9.898"
        })
    });
const $ed5fba8ef591fd80$var$ForwardRef = /*#__PURE__*/ (0, $hnvjC$forwardRef)($ed5fba8ef591fd80$var$SvgComponent);
var $ed5fba8ef591fd80$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ed5fba8ef591fd80$var$ForwardRef);


export {$ed5fba8ef591fd80$export$2e2bcd8739ae039 as default};
