// @react-spectrum/s2/illustrations/linear/internal/S2_lin_crop_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3YLPq$reactjsxruntime = require("react/jsx-runtime");
var $3YLPq$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4e8984da253526da$export$2e2bcd8739ae039);




const $4e8984da253526da$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3YLPq$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3YLPq$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M31.974 15c0-1.105-.895-2-2-2s-2 .895-2 2v12.903H15c-1.105 0-2 .896-2 2s.895 2 2 2h12.974v26.193c0 5.523 4.478 10 10 10h26.05v12.903c0 1.105.896 2 2 2s2-.895 2-2V68.096H81c1.105 0 2-.895 2-2s-.895-2-2-2H37.974c-3.313 0-6-2.686-6-6V15m5.442 12.903c-1.105 0-2 .896-2 2s.895 2 2 2h20.609c3.314 0 6 2.687 6 6v20.742c0 1.104.895 2 2 2s2-.896 2-2V37.903c0-5.523-4.477-10-10-10z"
        })
    });
const $4e8984da253526da$var$ForwardRef = /*#__PURE__*/ (0, $3YLPq$react.forwardRef)($4e8984da253526da$var$SvgComponent);
var $4e8984da253526da$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4e8984da253526da$var$ForwardRef);
