// @react-spectrum/s2/illustrations/linear/internal/S2_lin_close_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $k1XED$reactjsxruntime = require("react/jsx-runtime");
var $k1XED$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $53dcbb82cae55e04$export$2e2bcd8739ae039);




const $53dcbb82cae55e04$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $k1XED$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $k1XED$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 28c-28.719 0-52 23.281-52 52s23.281 52 52 52 52-23.281 52-52-23.281-52-52-52M20 80c0-33.137 26.863-60 60-60s60 26.863 60 60-26.863 60-60 60-60-26.863-60-60m41.172-18.828c1.562-1.562 4.094-1.562 5.656 0l13.634 13.633 13.633-13.633c1.562-1.562 4.095-1.562 5.657 0s1.562 4.094 0 5.656L86.119 80.462l13.633 13.633c1.562 1.562 1.562 4.095 0 5.657s-4.095 1.562-5.657 0L80.462 86.119 66.828 99.752c-1.562 1.562-4.094 1.562-5.656 0s-1.562-4.095 0-5.657l13.633-13.633-13.633-13.634c-1.562-1.562-1.562-4.094 0-5.656"
        })
    });
const $53dcbb82cae55e04$var$ForwardRef = /*#__PURE__*/ (0, $k1XED$react.forwardRef)($53dcbb82cae55e04$var$SvgComponent);
var $53dcbb82cae55e04$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($53dcbb82cae55e04$var$ForwardRef);
