// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cardTapPayment_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6Fusm$jsx} from "react/jsx-runtime";
import {forwardRef as $6Fusm$forwardRef} from "react";





const $7cda1f5ffe253971$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6Fusm$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6Fusm$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M69.323 15.244c.134-1.096 1.13-1.877 2.227-1.743 6.553.8 12.672 4.565 16.23 10.729 3.588 6.214 3.763 13.462 1.113 19.568-.44 1.014-1.618 1.479-2.631 1.039-1.014-.44-1.478-1.618-1.039-2.631 2.168-4.994 2.021-10.906-.906-15.977-2.904-5.029-7.893-8.103-13.252-8.757-1.096-.134-1.876-1.132-1.742-2.228m-62.68 41.14c-2.762-4.782-1.123-10.898 3.66-13.66l40.096-23.15c4.783-2.76 10.899-1.122 13.66 3.661l12.767 22.112c2.761 4.783 1.123 10.9-3.66 13.66l-40.097 23.15c-4.783 2.762-10.899 1.123-13.66-3.66zm5.66-10.195c-2.87 1.657-3.854 5.326-2.197 8.196l1.23 2.13 50.49-29.149-1.23-2.131c-1.658-2.87-5.327-3.853-8.197-2.196zm10.57 30.308L13.337 59.98l50.489-29.15 9.536 16.517c1.657 2.87.673 6.54-2.196 8.196l-40.097 23.15c-2.87 1.657-6.54.674-8.196-2.196M70.8 21.837c-1.097-.134-2.094.646-2.228 1.743s.647 2.094 1.743 2.228c2.788.34 5.38 1.938 6.888 4.551 1.521 2.635 1.599 5.707.47 8.305-.439 1.013.026 2.191 1.04 2.63 1.013.44 2.19-.024 2.63-1.037 1.61-3.711 1.506-8.119-.676-11.898-2.164-3.747-5.886-6.036-9.867-6.522M20.922 60.883c-.957.552-1.284 1.775-.732 2.732.552.956 1.776 1.284 2.732.732L42.277 53.17c.957-.553 1.285-1.776.732-2.732s-1.775-1.285-2.732-.732zm3.46 9.99c-.553-.957-.225-2.18.731-2.732l9.678-5.588c.956-.553 2.18-.225 2.732.732.552.956.224 2.18-.732 2.731l-9.678 5.589c-.956.552-2.18.224-2.732-.732M51.774 43.07c-.956.553-1.284 1.776-.732 2.732.553.957 1.776 1.285 2.733.732l9.074-5.24c.956-.552 1.284-1.775.731-2.731-.552-.957-1.775-1.285-2.732-.732z"
        })
    });
const $7cda1f5ffe253971$var$ForwardRef = /*#__PURE__*/ (0, $6Fusm$forwardRef)($7cda1f5ffe253971$var$SvgComponent);
var $7cda1f5ffe253971$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7cda1f5ffe253971$var$ForwardRef);


export {$7cda1f5ffe253971$export$2e2bcd8739ae039 as default};
