// @react-spectrum/s2/illustrations/linear/internal/S2_lin_buildTable_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $hWvag$jsx} from "react/jsx-runtime";
import {forwardRef as $hWvag$forwardRef} from "react";





const $812fcd3492cf9b6f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hWvag$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hWvag$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22 24c-3.315 0-6 2.685-6 6v4.04h64V30c0-3.315-2.685-6-6-6zm-6 42V38.04h10.02V72H22c-3.315 0-6-2.685-6-6m43.11 6H30.02V38.04H80v12.92c0 1.105.895 2 2 2s2-.895 2-2V30c0-5.525-4.475-10-10-10H22c-5.525 0-10 4.475-10 10v36c0 5.525 4.475 10 10 10h37.11c1.105 0 2-.895 2-2s-.895-2-2-2m4.5-2.61c0-7.67 6.22-13.89 13.89-13.89s13.89 6.22 13.89 13.89-6.22 13.89-13.89 13.89-13.89-6.22-13.89-13.89m15.89 1.97h4.73c1.1 0 2-.9 2-2s-.9-2-2-2H79.5v-4.73c0-1.1-.9-2-2-2s-2 .9-2 2v4.73h-4.73c-1.1 0-2 .9-2 2s.9 2 2 2h4.73v4.73c0 1.1.9 2 2 2s2-.9 2-2z"
        })
    });
const $812fcd3492cf9b6f$var$ForwardRef = /*#__PURE__*/ (0, $hWvag$forwardRef)($812fcd3492cf9b6f$var$SvgComponent);
var $812fcd3492cf9b6f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($812fcd3492cf9b6f$var$ForwardRef);


export {$812fcd3492cf9b6f$export$2e2bcd8739ae039 as default};
