// @react-spectrum/s2/illustrations/linear/internal/S2_lin_ribbon_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $kegEw$jsx} from "react/jsx-runtime";
import {forwardRef as $kegEw$forwardRef} from "react";





const $74383d9942d7b8be$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kegEw$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kegEw$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 28c-17.754 0-32.147 14.392-32.147 32.147S62.246 92.293 80 92.293s32.147-14.392 32.147-32.146S97.754 28 80 28M39.853 60.147C39.853 37.974 57.828 20 80 20s40.147 17.974 40.147 40.147c0 10.971-4.402 20.915-11.535 28.162l12.419 30.738c1.788 4.427-2.061 9.069-6.743 8.13l-9.568-1.917-5.55 8.026c-2.716 3.927-8.71 3.262-10.498-1.165l-8.671-21.463-8.662 21.438c-1.795 4.443-7.818 5.092-10.52 1.134l-5.501-8.063-9.559 1.978c-4.692.971-8.574-3.68-6.779-8.123L51.39 88.31c-7.135-7.247-11.537-17.191-11.537-28.163m18.01 33.496-10.065 24.911 7.205-1.491c2.372-.491 4.806.493 6.171 2.494l4.147 6.077 10.333-25.573c-6.527-.703-12.584-2.97-17.791-6.418m26.484 6.417c6.527-.702 12.584-2.969 17.791-6.417l10.086 24.961-7.221-1.447c-2.349-.471-4.752.5-6.114 2.47l-4.189 6.057z"
        })
    });
const $74383d9942d7b8be$var$ForwardRef = /*#__PURE__*/ (0, $kegEw$forwardRef)($74383d9942d7b8be$var$SvgComponent);
var $74383d9942d7b8be$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($74383d9942d7b8be$var$ForwardRef);


export {$74383d9942d7b8be$export$2e2bcd8739ae039 as default};
