// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pause_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $fXDuQ$reactjsxruntime = require("react/jsx-runtime");
var $fXDuQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0b06f7b56fd17f02$export$2e2bcd8739ae039);




const $0b06f7b56fd17f02$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fXDuQ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fXDuQ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M12 13c0-1.795 1.455-3.25 3.25-3.25h3.5C20.545 9.75 22 11.205 22 13v22c0 1.795-1.455 3.25-3.25 3.25h-3.5C13.455 38.25 12 36.795 12 35zm3.25-.75c-.414 0-.75.336-.75.75v22c0 .414.336.75.75.75h3.5c.414 0 .75-.336.75-.75V13c0-.414-.336-.75-.75-.75zM26 13c0-1.795 1.455-3.25 3.25-3.25h3.5C34.545 9.75 36 11.205 36 13v22c0 1.795-1.455 3.25-3.25 3.25h-3.5C27.455 38.25 26 36.795 26 35zm3.25-.75c-.414 0-.75.336-.75.75v22c0 .414.336.75.75.75h3.5c.414 0 .75-.336.75-.75V13c0-.414-.336-.75-.75-.75z"
        })
    });
const $0b06f7b56fd17f02$var$ForwardRef = /*#__PURE__*/ (0, $fXDuQ$react.forwardRef)($0b06f7b56fd17f02$var$SvgComponent);
var $0b06f7b56fd17f02$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0b06f7b56fd17f02$var$ForwardRef);
