// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cloud_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $9ZFFx$reactjsxruntime = require("react/jsx-runtime");
var $9ZFFx$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $60727f48937ec33a$export$2e2bcd8739ae039);




const $60727f48937ec33a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9ZFFx$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9ZFFx$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M56.283 50.793C61.896 37.731 74.509 28.5 89.33 28.5c19.209 0 34.684 15.462 36.079 34.765C140.627 66 152 79.795 152 96.135c0 18.261-14.217 33.365-32.117 33.365l-.083-.001-.074.001H31.881q-.247 0-.486-.029C18.254 129.016 8 117.815 8 104.385c0-10.174 5.862-19.016 14.402-22.942q-.152-1.444-.154-2.943c0-15.59 12.144-28.529 27.486-28.529 2.258 0 4.451.287 6.548.822M32.258 121.5h87.284q.102-.004.204-.004l.074.001.068.001.042.002C133.067 121.473 144 110.292 144 96.135c0-13.673-10.212-24.581-22.741-25.32-2.145-.127-3.806-1.925-3.764-4.073.005-.226.009-.329.012-.403.003-.073.005-.119.005-.229 0-16.518-12.78-29.61-28.182-29.61-12.435 0-23.13 8.505-26.818 20.503-.334 1.086-1.115 1.98-2.147 2.456s-2.217.491-3.26.041c-2.28-.983-4.765-1.529-7.371-1.529-10.6 0-19.486 9.026-19.486 20.529 0 1.586.19 3.144.53 4.675.457 2.065-.772 4.129-2.805 4.71C21.168 89.833 16 96.396 16 104.386c0 9.513 7.273 16.948 15.95 17.098q.156.002.31.017m87.7.001h-.002z"
        })
    });
const $60727f48937ec33a$var$ForwardRef = /*#__PURE__*/ (0, $9ZFFx$react.forwardRef)($60727f48937ec33a$var$SvgComponent);
var $60727f48937ec33a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($60727f48937ec33a$var$ForwardRef);
