// @react-spectrum/s2/illustrations/linear/internal/S2_lin_exposure_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $crOCw$reactjsxruntime = require("react/jsx-runtime");
var $crOCw$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $873fc8b46cbb31ce$export$2e2bcd8739ae039);




const $873fc8b46cbb31ce$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $crOCw$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $crOCw$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M130.46 79.576c0 2.751-.215 5.452-.63 8.087L85.315 55.505l24.003-17.487.059-.043c12.784 9.386 21.083 24.524 21.083 41.6m-41.332 50.558c18.14-3.655 32.834-16.829 38.646-34.087l-22.702-16.4zm-8.703.995q-.768.023-1.54.023c-19.113 0-35.798-10.397-44.708-25.842h54.401zM27.309 79.576c0 6.23 1.105 12.204 3.13 17.734h27.025L40.512 45.113C32.303 54.247 27.31 66.328 27.31 79.576m19.65-40.51C55.742 32.136 66.83 28 78.885 28c8.31 0 16.16 1.965 23.111 5.455l-45.98 33.496zm11.644 35.852q.24-.129.465-.292L78.513 60.46 98.24 74.712 91.104 97.31H65.818q-.048-.323-.152-.645zm20.281 64.234c32.903 0 59.576-26.673 59.576-59.576S111.787 20 78.884 20C45.982 20 19.31 46.673 19.31 79.576s26.673 59.576 59.575 59.576"
        })
    });
const $873fc8b46cbb31ce$var$ForwardRef = /*#__PURE__*/ (0, $crOCw$react.forwardRef)($873fc8b46cbb31ce$var$SvgComponent);
var $873fc8b46cbb31ce$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($873fc8b46cbb31ce$var$ForwardRef);
