// @react-spectrum/s2/illustrations/linear/internal/S2_lin_hand_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1GgZ0$jsx} from "react/jsx-runtime";
import {forwardRef as $1GgZ0$forwardRef} from "react";





const $cf42c714c469387f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1GgZ0$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1GgZ0$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20.963 9.526c.415-1.872 2.079-3.276 4.077-3.276 1.533 0 2.87.827 3.596 2.058.651-.422 1.427-.667 2.262-.667 2.312 0 4.177 1.88 4.177 4.189v2.286c.842-.475 1.845-.634 2.772-.452 1.925.378 3.205 2.13 2.971 4.342l-.005.046-1.268 8.864q-.016.111-.05.215c-1.036 5.59-2.769 9.286-5.233 11.575-2.533 2.352-5.647 3.044-8.981 3.044-2.528 0-4.89-1.082-6.951-2.564-2.068-1.486-3.926-3.44-5.473-5.351-1.55-1.917-2.818-3.827-3.696-5.255-.44-.715-.785-1.312-1.02-1.732-.118-.21-.208-.377-.27-.492l-.071-.133-.014-.026c-.998-1.754-.895-4.083.929-5.453 1.758-1.321 4.248-.98 5.588.763l.703.914v-9.106c0-2.308 1.864-4.188 4.176-4.188.638 0 1.241.143 1.781.399M9.993 25.022l.003.006.015.028.062.116q.083.157.25.454c.22.395.547.962.967 1.644.84 1.367 2.047 3.183 3.51 4.992 1.47 1.815 3.168 3.586 4.989 4.894 1.826 1.313 3.682 2.094 5.491 2.094 2.995 0 5.386-.617 7.28-2.376 1.928-1.79 3.53-4.937 4.517-10.422q.01-.054.024-.106l1.233-8.623c.1-1.023-.425-1.5-.968-1.606-.825-.162-1.645.335-1.738 1.282l-.002.018-.28 2.495-.279 2.496c-.074.66-.65 1.146-1.312 1.109s-1.18-.585-1.18-1.248v-10.44c0-.936-.755-1.687-1.677-1.687-.921 0-1.676.75-1.676 1.688v9.707c0 .69-.56 1.25-1.25 1.25h-.006c-.69 0-1.25-.56-1.25-1.25V10.438c0-.937-.756-1.688-1.676-1.688-.921 0-1.676.75-1.676 1.688v11.098c0 .69-.56 1.25-1.25 1.25h-.006c-.69 0-1.25-.56-1.25-1.25v-8.22c0-.938-.755-1.689-1.676-1.689s-1.676.75-1.676 1.688v12.782c0 .535-.341 1.011-.848 1.183s-1.067.003-1.393-.421L12.32 23.03c-.506-.658-1.443-.785-2.105-.288-.588.442-.79 1.293-.25 2.23z"
        })
    });
const $cf42c714c469387f$var$ForwardRef = /*#__PURE__*/ (0, $1GgZ0$forwardRef)($cf42c714c469387f$var$SvgComponent);
var $cf42c714c469387f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cf42c714c469387f$var$ForwardRef);


export {$cf42c714c469387f$export$2e2bcd8739ae039 as default};
