// @react-spectrum/s2/illustrations/linear/internal/S2_lin_emptyStateExport_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $l4MM7$reactjsxruntime = require("react/jsx-runtime");
var $l4MM7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f14f7e18c00ea9a8$export$2e2bcd8739ae039);




const $f14f7e18c00ea9a8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $l4MM7$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $l4MM7$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M108.001 28c0-2.21 1.791-4 4-4h20c2.209 0 4 1.79 4 4v44c0 2.21-1.791 4-4 4h-20c-2.209 0-4-1.79-4-4zm4-12c-6.627 0-12 5.373-12 12v44c0 6.627 5.373 12 12 12h20c6.627 0 12-5.373 12-12V28c0-6.627-5.373-12-12-12zm-10 92c2.209 0 4 1.791 4 4v20c0 2.209-1.791 4-4 4h-44c-2.21 0-4-1.791-4-4v-20c0-2.209 1.79-4 4-4zm12 4c0-6.627-5.373-12-12-12h-44c-6.627 0-12 5.373-12 12v20c0 6.627 5.373 12 12 12h44c6.627 0 12-5.373 12-12zm-82.03-36.06c-2.225 0-4.03 1.804-4.03 4.03S29.746 84 31.971 84s4.03-1.804 4.03-4.03-1.804-4.03-4.03-4.03m-12.03 4.03c0-5.253 3.367-9.72 8.06-11.36V43.33c-4.693-1.64-8.06-6.107-8.06-11.36 0-6.644 5.386-12.03 12.03-12.03 5.253 0 9.72 3.367 11.36 8.06h25.28c1.641-4.693 6.108-8.06 11.36-8.06 6.644 0 12.03 5.386 12.03 12.03 0 5.23-3.338 9.682-8 11.339V58.48c0 2.21-1.79 4-4 4s-4-1.79-4-4V43.33c-3.43-1.2-6.152-3.908-7.368-7.33H43.31c-1.21 3.402-3.907 6.1-7.309 7.309v25.323c3.422 1.216 6.131 3.938 7.33 7.368h15.31c2.209 0 4 1.79 4 4s-1.791 4-4 4H43.31c-1.657 4.662-6.108 8-11.339 8-6.644 0-12.03-5.386-12.03-12.03m8-48c0-2.226 1.805-4.03 4.03-4.03s4.03 1.804 4.03 4.03S34.197 36 31.971 36s-4.03-1.804-4.03-4.03m52.03-4.03c-2.225 0-4.03 1.804-4.03 4.03S77.746 36 79.971 36s4.03-1.804 4.03-4.03-1.804-4.03-4.03-4.03m-5.46 63.65-8.26-20.12h-.01c-1.35-3.29 1.93-6.57 5.22-5.22l20.11 8.27c3.25 1.33 3.32 5.92.11 7.35l-5.39 2.41c-.9.4-1.62 1.12-2.02 2.02l-2.41 5.4c-1.43 3.21-6.01 3.14-7.35-.11"
        })
    });
const $f14f7e18c00ea9a8$var$ForwardRef = /*#__PURE__*/ (0, $l4MM7$react.forwardRef)($f14f7e18c00ea9a8$var$SvgComponent);
var $f14f7e18c00ea9a8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f14f7e18c00ea9a8$var$ForwardRef);
