// @react-spectrum/s2/illustrations/linear/internal/S2_lin_download_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $e4iGV$jsx} from "react/jsx-runtime";
import {forwardRef as $e4iGV$forwardRef} from "react";





const $7c1521de0a273066$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e4iGV$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $e4iGV$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M17 48.846c1.105 0 2 .895 2 2v17.647c0 2.971 2.385 5.353 5.294 5.353h47.412c2.909 0 5.294-2.382 5.294-5.353V50.846c0-1.105.895-2 2-2s2 .895 2 2v17.647c0 5.15-4.146 9.353-9.294 9.353H24.294c-5.148 0-9.294-4.203-9.294-9.353V50.846c0-1.105.895-2 2-2m31-30.692c1.105 0 2 .896 2 2v35.863l10.586-10.586c.78-.78 2.047-.78 2.828 0 .781.782.781 2.048 0 2.829l-14 14c-.78.781-2.047.781-2.828 0l-14-14c-.781-.781-.781-2.047 0-2.828.78-.782 2.047-.782 2.828 0L46 56.017V20.154c0-1.104.895-2 2-2"
        })
    });
const $7c1521de0a273066$var$ForwardRef = /*#__PURE__*/ (0, $e4iGV$forwardRef)($7c1521de0a273066$var$SvgComponent);
var $7c1521de0a273066$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7c1521de0a273066$var$ForwardRef);


export {$7c1521de0a273066$export$2e2bcd8739ae039 as default};
