// @react-spectrum/s2/illustrations/linear/internal/S2_lin_replace_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $b7hAy$jsx} from "react/jsx-runtime";
import {forwardRef as $b7hAy$forwardRef} from "react";





const $5f4c475b7cbaf895$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $b7hAy$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $b7hAy$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M7.863 12.125c0-2.347 1.903-4.25 4.25-4.25h5.311c2.347 0 4.25 1.903 4.25 4.25v5.31c0 2.348-1.903 4.25-4.25 4.25h-5.31c-2.348 0-4.25-1.902-4.25-4.25zm4.25-1.75c-.966 0-1.75.784-1.75 1.75v5.31c0 .967.784 1.75 1.75 1.75h5.311c.967 0 1.75-.783 1.75-1.75v-5.31c0-.966-.783-1.75-1.75-1.75zm15.332-1.944c-.69-.023-1.268.517-1.291 1.207-.024.69.517 1.269 1.206 1.292 1.387.047 2.794.446 3.853 1.47.879.85 1.647 2.258 1.867 4.61l-1.584-1.584c-.488-.488-1.28-.488-1.768 0-.488.489-.488 1.28 0 1.768l3.573 3.573c.488.488 1.28.488 1.767 0l3.584-3.584c.488-.488.488-1.28 0-1.768s-1.28-.488-1.767 0l-1.32 1.32c-.27-2.785-1.218-4.78-2.614-6.132-1.646-1.593-3.733-2.112-5.506-2.172M14.652 27.234c-.488-.488-1.28-.488-1.768 0L9.3 30.818c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0l1.32-1.32c.268 2.785 1.217 4.78 2.613 6.132 1.647 1.593 3.734 2.111 5.506 2.172.69.023 1.268-.517 1.292-1.207.023-.69-.517-1.269-1.207-1.292-1.386-.047-2.794-.446-3.853-1.47-.878-.85-1.647-2.258-1.867-4.61l1.584 1.584c.488.488 1.28.488 1.768 0 .488-.489.488-1.28 0-1.768zm15.922-.92c-2.347 0-4.25 1.903-4.25 4.25v5.311c0 2.348 1.902 4.25 4.25 4.25h5.31c2.348 0 4.25-1.903 4.25-4.25v-5.31c0-2.348-1.902-4.25-4.25-4.25zm-1.75 4.25c0-.966.783-1.75 1.75-1.75h5.31c.967 0 1.75.784 1.75 1.75v5.311c0 .967-.783 1.75-1.75 1.75h-5.31c-.967 0-1.75-.783-1.75-1.75z"
        })
    });
const $5f4c475b7cbaf895$var$ForwardRef = /*#__PURE__*/ (0, $b7hAy$forwardRef)($5f4c475b7cbaf895$var$SvgComponent);
var $5f4c475b7cbaf895$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5f4c475b7cbaf895$var$ForwardRef);


export {$5f4c475b7cbaf895$export$2e2bcd8739ae039 as default};
