// @react-spectrum/s2/illustrations/linear/internal/S2_lin_apps_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $awy9g$reactjsxruntime = require("react/jsx-runtime");
var $awy9g$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $476e3587513b7978$export$2e2bcd8739ae039);




const $476e3587513b7978$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $awy9g$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $awy9g$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24.11 13c0-1.519 1.232-2.75 2.75-2.75h8.17c1.52 0 2.75 1.231 2.75 2.75v8.17c0 1.519-1.23 2.75-2.75 2.75h-8.17c-1.518 0-2.75-1.231-2.75-2.75zm2.75-5.25c-2.9 0-5.25 2.35-5.25 5.25v8.17c0 2.9 2.35 5.25 5.25 5.25h8.17c2.9 0 5.25-2.35 5.25-5.25V13c0-2.9-2.35-5.25-5.25-5.25zm-16.6 10.68c0-.69.56-1.25 1.25-1.25h4.23c.69 0 1.25.56 1.25 1.25v4.23c0 .69-.56 1.25-1.25 1.25h-4.23c-.69 0-1.25-.56-1.25-1.25zm1.25-3.75c-2.071 0-3.75 1.679-3.75 3.75v4.23c0 2.071 1.679 3.75 3.75 3.75h4.23c2.07 0 3.75-1.679 3.75-3.75v-4.23c0-2.071-1.68-3.75-3.75-3.75zm0 16.34c-.69 0-1.25.56-1.25 1.25v4.23c0 .69.56 1.25 1.25 1.25h4.23c.69 0 1.25-.56 1.25-1.25v-4.23c0-.69-.56-1.25-1.25-1.25zm-3.75 1.25c0-2.071 1.679-3.75 3.75-3.75h4.23c2.07 0 3.75 1.679 3.75 3.75v4.23c0 2.071-1.68 3.75-3.75 3.75h-4.23c-2.071 0-3.75-1.679-3.75-3.75zm16.35 0c0-.69.56-1.25 1.25-1.25h4.23c.69 0 1.25.56 1.25 1.25v4.23c0 .69-.56 1.25-1.25 1.25h-4.23c-.69 0-1.25-.56-1.25-1.25zm1.25-3.75c-2.07 0-3.75 1.679-3.75 3.75v4.23c0 2.071 1.68 3.75 3.75 3.75h4.23c2.071 0 3.75-1.679 3.75-3.75v-4.23c0-2.071-1.679-3.75-3.75-3.75z"
        })
    });
const $476e3587513b7978$var$ForwardRef = /*#__PURE__*/ (0, $awy9g$react.forwardRef)($476e3587513b7978$var$SvgComponent);
var $476e3587513b7978$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($476e3587513b7978$var$ForwardRef);
