// @react-spectrum/s2/illustrations/linear/internal/S2_lin_interaction_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $7NIT2$reactjsxruntime = require("react/jsx-runtime");
var $7NIT2$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6f570a7d52a41255$export$2e2bcd8739ae039);




const $6f570a7d52a41255$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7NIT2$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7NIT2$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M9.69 17.066c0-4.607 3.734-8.341 8.341-8.341s8.342 3.734 8.342 8.341c0 .996-.174 1.948-.492 2.83-.234.65.102 1.366.752 1.6s1.366-.103 1.6-.752c.414-1.15.64-2.389.64-3.678 0-5.987-4.854-10.841-10.842-10.841-5.987 0-10.842 4.854-10.842 10.841 0 5.325 3.838 9.752 8.899 10.668.679.123 1.33-.328 1.453-1.007.122-.68-.328-1.33-1.008-1.453-3.892-.705-6.844-4.113-6.844-8.208m9.307 1.24c.177-.096.37-.11.587-.028.23.085.472.275.643.555l3.603 6.485c.44.79 1.433.988 2.14.53.666-.43 1.724-.913 2.863-1.289 1.149-.379 2.24-.603 2.976-.603 2.133 0 3.711.811 4.778 2.094 1.089 1.31 1.722 3.21 1.722 5.474 0 4.29-3.428 7.75-7.699 7.75-2.221 0-5.163-.93-7.852-2.034-2.258-.928-4.2-1.919-5.205-2.431q-.262-.135-.435-.22c-.586-.294-.786-.56-.86-.723-.072-.157-.111-.4.007-.828.128-.468.64-.86 1.452-.63.948.27 2.007.692 3.104 1.156q.463.197.936.402c.772.332 1.545.665 2.241.933 1.352.52 2.49-.944 1.837-2.12l-7.183-12.93-.012-.021-.013-.02c-.31-.512-.096-1.25.37-1.503m3.385-.74c-.85-1.422-2.762-2.442-4.576-1.458-1.774.961-2.328 3.302-1.33 4.974l5.761 10.369-.44-.188c-1.107-.47-2.288-.944-3.396-1.259-1.907-.543-3.987.335-4.547 2.37-.22.8-.258 1.682.13 2.53.384.844 1.103 1.464 2.016 1.92l.404.205c.992.507 3.04 1.552 5.404 2.523 2.72 1.117 6.053 2.222 8.802 2.222 5.67 0 10.2-4.599 10.2-10.25 0-2.707-.757-5.216-2.3-7.072-1.566-1.883-3.863-2.996-6.7-2.996-1.123 0-2.507.315-3.76.729-.88.29-1.763.65-2.523 1.04L22.402 17.6l-.01-.017z"
        })
    });
const $6f570a7d52a41255$var$ForwardRef = /*#__PURE__*/ (0, $7NIT2$react.forwardRef)($6f570a7d52a41255$var$SvgComponent);
var $6f570a7d52a41255$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6f570a7d52a41255$var$ForwardRef);
