// @react-spectrum/s2/illustrations/linear/internal/S2_lin_desktop_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $buPva$reactjsxruntime = require("react/jsx-runtime");
var $buPva$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $36b56b60dd031888$export$2e2bcd8739ae039);




const $36b56b60dd031888$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $buPva$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $buPva$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20 20c-3.314 0-6 2.686-6 6v28c0 3.314 2.686 6 6 6h56c3.314 0 6-2.686 6-6V26c0-3.314-2.686-6-6-6zm-10 6c0-5.523 4.477-10 10-10h56c5.523 0 10 4.477 10 10v28c0 5.523-4.477 10-10 10H53v12h7c1.105 0 2 .895 2 2s-.895 2-2 2H36c-1.105 0-2-.895-2-2s.895-2 2-2h8V64H20c-5.523 0-10-4.477-10-10zm41 .5c0 1.38-1.12 2.5-2.5 2.5S46 27.88 46 26.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5"
        })
    });
const $36b56b60dd031888$var$ForwardRef = /*#__PURE__*/ (0, $buPva$react.forwardRef)($36b56b60dd031888$var$SvgComponent);
var $36b56b60dd031888$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($36b56b60dd031888$var$ForwardRef);
