// @react-spectrum/s2/illustrations/linear/internal/S2_lin_volumeOff_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1Upiw$reactjsxruntime = require("react/jsx-runtime");
var $1Upiw$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $717b22566910fbfb$export$2e2bcd8739ae039);




const $717b22566910fbfb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1Upiw$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1Upiw$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22.297 16.422c-1.562-1.562-4.095-1.562-5.657 0s-1.562 4.094 0 5.656l121.063 121.063c1.562 1.562 4.095 1.562 5.657 0s1.562-4.095 0-5.657zM69.38 30.15c6.312-6.258 17.04-1.787 17.04 7.102V61c0 2.21-1.79 4-4 4s-4-1.79-4-4V37.252c0-1.778-2.145-2.672-3.408-1.42l-8.922 8.845c-1.568 1.555-4.101 1.544-5.657-.024-1.555-1.569-1.544-4.102.025-5.657zM36.435 56.896c-2.21 0-4 1.79-4 4v38.076c0 2.209 1.79 4 4 4h12.802V74.249c0-2.21 1.791-4 4-4s4 1.79 4 4v32.141l17.782 17.454c1.266 1.242 3.401.346 3.401-1.427v-19.011c0-2.209 1.791-4 4-4s4 1.791 4 4v19.011c0 8.864-10.678 13.346-17.004 7.136l-18.931-18.581h-14.05c-6.628 0-12-5.373-12-12V60.896c0-6.628 5.372-12 12-12h.096c2.21 0 4 1.79 4 4s-1.79 4-4 4zm57.27.217c1.328-1.765 3.835-2.119 5.6-.79 7.09 5.335 11.687 13.833 11.687 23.404 0 2.086-.219 4.126-.636 6.096-.458 2.161-2.582 3.542-4.743 3.084-2.161-.459-3.541-2.582-3.083-4.743.302-1.428.462-2.912.462-4.437 0-6.951-3.329-13.123-8.497-17.012-1.765-1.329-2.12-3.836-.79-5.602m16.894-15.416c1.328-1.765 3.836-2.119 5.601-.79 11.752 8.843 19.364 22.922 19.364 38.777 0 9.429-2.694 18.24-7.357 25.693-1.171 1.873-3.639 2.441-5.512 1.269s-2.441-3.639-1.27-5.512c3.89-6.218 6.139-13.565 6.139-21.45 0-13.235-6.344-24.988-16.174-32.385-1.765-1.329-2.119-3.836-.791-5.602"
        })
    });
const $717b22566910fbfb$var$ForwardRef = /*#__PURE__*/ (0, $1Upiw$react.forwardRef)($717b22566910fbfb$var$SvgComponent);
var $717b22566910fbfb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($717b22566910fbfb$var$ForwardRef);
