// @react-spectrum/s2/illustrations/linear/internal/S2_lin_vectorDraw_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8srL9$jsx} from "react/jsx-runtime";
import {forwardRef as $8srL9$forwardRef} from "react";





const $fed81caf5b3a73b3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8srL9$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8srL9$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M67.092 15.034c-3.058-3.058-7.994-2.49-11 .517l-4.615 4.614-22.615 9.653c-2.355 1.006-4.205 3.04-4.96 5.48l-10.4 33.6C10.916 77.251 18.75 85.085 27.105 82.5l33.6-10.4c2.439-.756 4.473-2.605 5.478-4.96l9.654-22.615 4.614-4.614c3.007-3.007 3.575-7.943.517-11.001zm-36.66 18.463 21.728-9.274 19.62 19.619-9.275 21.727c-.55 1.288-1.68 2.306-2.984 2.71l-33.6 10.4c-1.83.566-3.622.341-5.114-.415l20.43-20.346c1.772.652 3.84.266 5.263-1.158 1.953-1.952 1.953-5.118 0-7.07-1.952-1.953-5.118-1.953-7.07 0-1.477 1.475-1.837 3.644-1.082 5.459L17.907 75.507c-.89-1.55-1.19-3.466-.584-5.427l10.4-33.6c.404-1.303 1.421-2.433 2.71-2.983m47.19 3.585-3.473 3.473-18.702-18.703 3.473-3.473c1.791-1.792 4.173-1.687 5.343-.517L78.14 31.739c1.17 1.17 1.275 3.552-.516 5.344"
        })
    });
const $fed81caf5b3a73b3$var$ForwardRef = /*#__PURE__*/ (0, $8srL9$forwardRef)($fed81caf5b3a73b3$var$SvgComponent);
var $fed81caf5b3a73b3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($fed81caf5b3a73b3$var$ForwardRef);


export {$fed81caf5b3a73b3$export$2e2bcd8739ae039 as default};
