// @react-spectrum/s2/illustrations/linear/internal/S2_lin_stepForward_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6ZA1k$jsx} from "react/jsx-runtime";
import {forwardRef as $6ZA1k$forwardRef} from "react";





const $20c5539c059cf715$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6ZA1k$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6ZA1k$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M39.145 28.478c.007-5.813 6.196-9.663 11.288-6.622L83.18 41.41c4.968 2.967 4.96 10.316-.016 13.271l-32.79 19.474c-5.1 3.028-11.28-.838-11.273-6.65zm9.238-3.188c-2.242-1.338-5.234.264-5.238 3.193L43.1 67.51c-.003 2.93 2.986 4.538 5.23 3.205l32.79-19.473c2.37-1.407 2.374-4.985.008-6.398zl1.025-1.717zM12 27c0-3.313 2.686-6 6-6h8c3.314 0 6 2.687 6 6v42c0 3.314-2.686 6-6 6h-8c-3.314 0-6-2.686-6-6zm6-2c-1.105 0-2 .896-2 2v42c0 1.105.895 2 2 2h8c1.105 0 2-.895 2-2V27c0-1.104-.895-2-2-2z"
        })
    });
const $20c5539c059cf715$var$ForwardRef = /*#__PURE__*/ (0, $6ZA1k$forwardRef)($20c5539c059cf715$var$SvgComponent);
var $20c5539c059cf715$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($20c5539c059cf715$var$ForwardRef);


export {$20c5539c059cf715$export$2e2bcd8739ae039 as default};
