// @react-spectrum/s2/illustrations/linear/internal/S2_lin_flag_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $7nsu7$reactjsxruntime = require("react/jsx-runtime");
var $7nsu7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e37e3f242d2f2d4c$export$2e2bcd8739ae039);




const $e37e3f242d2f2d4c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7nsu7$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7nsu7$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25.484 18.71c0-1.104-.895-2-2-2s-2 .896-2 2v59.563c0 1.105.896 2 2 2s2-.895 2-2v-15.78h41.172c4.418 0 8-3.582 8-8V28.535c0-4.418-3.582-8-8-8H25.484zm0 5.825v33.958h41.172c2.21 0 4-1.79 4-4V28.535c0-2.209-1.79-4-4-4z"
        })
    });
const $e37e3f242d2f2d4c$var$ForwardRef = /*#__PURE__*/ (0, $7nsu7$react.forwardRef)($e37e3f242d2f2d4c$var$SvgComponent);
var $e37e3f242d2f2d4c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e37e3f242d2f2d4c$var$ForwardRef);
