// @react-spectrum/s2/illustrations/linear/internal/S2_lin_gift_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9kv6W$jsx} from "react/jsx-runtime";
import {forwardRef as $9kv6W$forwardRef} from "react";





const $5a8df2fb4d2ecd00$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9kv6W$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9kv6W$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M29.064 25.156c0-5.782 4.687-10.468 10.468-10.468 3.481 0 6.565 1.699 8.469 4.312 1.903-2.613 4.987-4.312 8.468-4.312 5.782 0 10.468 4.686 10.468 10.468 0 2.437-.832 4.68-2.23 6.46h7.503c4.448 0 8.053 3.605 8.053 8.052v4.035c0 2.984-1.624 5.589-4.035 6.98V69.93c0 5.561-4.509 10.07-10.07 10.07H29.843c-5.561 0-10.07-4.508-10.07-10.07V50.683c-2.411-1.391-4.035-3.996-4.035-6.98v-4.035c0-4.447 3.605-8.053 8.053-8.053h7.503c-1.397-1.779-2.23-4.022-2.23-6.459M50 35.624h6.468q.22 0 .438-.009H72.21c2.238 0 4.053 1.815 4.053 4.053v4.035c0 2.238-1.815 4.052-4.053 4.052H50zm-4-4.009v-6.467c-.005-3.569-2.9-6.46-6.469-6.46-3.572 0-6.468 2.896-6.468 6.468 0 3.457 2.712 6.28 6.124 6.46zm10.812 0c3.412-.178 6.124-3.002 6.124-6.459 0-3.572-2.895-6.468-6.468-6.468-3.57 0-6.464 2.891-6.468 6.46v6.468zm-17.28 4.01q-.22 0-.438-.01H23.79c-2.238 0-4.053 1.815-4.053 4.053v4.035c0 2.238 1.815 4.052 4.053 4.052H46v-12.13zM46 51.754H23.773V69.93c0 3.352 2.718 6.07 6.07 6.07h16.158zM50 76V51.755h22.227V69.93c0 3.352-2.718 6.07-6.07 6.07z"
        })
    });
const $5a8df2fb4d2ecd00$var$ForwardRef = /*#__PURE__*/ (0, $9kv6W$forwardRef)($5a8df2fb4d2ecd00$var$SvgComponent);
var $5a8df2fb4d2ecd00$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5a8df2fb4d2ecd00$var$ForwardRef);


export {$5a8df2fb4d2ecd00$export$2e2bcd8739ae039 as default};
