// @react-spectrum/s2/illustrations/linear/internal/S2_lin_link_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $4JFYB$reactjsxruntime = require("react/jsx-runtime");
var $4JFYB$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $33a54f948b4d6bd8$export$2e2bcd8739ae039);




const $33a54f948b4d6bd8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4JFYB$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4JFYB$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M37.84 16.965c1.888-1.889 1.888-4.95 0-6.839-1.888-1.888-4.95-1.888-6.838 0l-3.5 3.5c-.488.489-1.28.489-1.768 0-.488-.488-.488-1.28 0-1.767l3.5-3.5c2.865-2.865 7.51-2.865 10.374 0 2.865 2.864 2.865 7.509 0 10.373l-8.875 8.876c-2.865 2.864-7.51 2.864-10.374 0-.488-.489-.488-1.28 0-1.768s1.28-.488 1.768 0c1.888 1.888 4.95 1.888 6.838 0zm-10.199 3.427c.489.488.489 1.28 0 1.768-.488.488-1.28.488-1.767 0-1.889-1.889-4.95-1.889-6.839 0l-8.875 8.875c-1.888 1.888-1.888 4.95 0 6.839 1.889 1.888 4.95 1.888 6.839 0l3.5-3.5c.488-.489 1.28-.489 1.767 0 .489.488.489 1.279 0 1.767l-3.5 3.5c-2.864 2.865-7.509 2.865-10.373 0-2.865-2.864-2.865-7.509 0-10.374l8.875-8.875c2.864-2.864 7.509-2.864 10.373 0"
        })
    });
const $33a54f948b4d6bd8$var$ForwardRef = /*#__PURE__*/ (0, $4JFYB$react.forwardRef)($33a54f948b4d6bd8$var$SvgComponent);
var $33a54f948b4d6bd8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($33a54f948b4d6bd8$var$ForwardRef);
