// @react-spectrum/s2/illustrations/linear/internal/S2_lin_document_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $e8zkX$reactjsxruntime = require("react/jsx-runtime");
var $e8zkX$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $cdbaa1d31bb9a4bf$export$2e2bcd8739ae039);




const $cdbaa1d31bb9a4bf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e8zkX$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $e8zkX$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M14.66 7.75c-2.592 0-4.66 2.127-4.66 4.711v23.077c0 2.585 2.068 4.712 4.66 4.712h18.18c2.592 0 4.66-2.127 4.66-4.712V21.972c0-1.243-.486-2.44-1.358-3.325l-9.367-9.51c-.872-.886-2.059-1.387-3.3-1.387zm-2.16 4.711c0-1.239.985-2.211 2.16-2.211h8.84v8.525c0 1.693 1.416 2.975 3.05 2.975h8.44q.01.11.01.222v13.566c0 1.24-.985 2.212-2.16 2.212H14.66c-1.175 0-2.16-.972-2.16-2.212zm20.727 6.789L26 11.912v6.863c0 .213.196.475.55.475z"
        })
    });
const $cdbaa1d31bb9a4bf$var$ForwardRef = /*#__PURE__*/ (0, $e8zkX$react.forwardRef)($cdbaa1d31bb9a4bf$var$SvgComponent);
var $cdbaa1d31bb9a4bf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($cdbaa1d31bb9a4bf$var$ForwardRef);
