// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cut_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2Vc2n$reactjsxruntime = require("react/jsx-runtime");
var $2Vc2n$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $bdc9977f2cedccbd$export$2e2bcd8739ae039);




const $bdc9977f2cedccbd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2Vc2n$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2Vc2n$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M99.929 23.656c2.09.716 3.204 2.99 2.489 5.08L87.936 71.038l43.076-14.67c2.092-.711 4.364.407 5.076 2.498s-.406 4.364-2.497 5.076L85.677 80.259q-.143.048-.285.085c-.249.982-.48 1.98-.714 3-.86 3.72-1.783 7.72-3.876 12.236-.41.887-.861 1.786-1.338 2.68q.333-.01.67-.01c10.58 0 19.156 8.576 19.156 19.155s-8.577 19.157-19.156 19.157c-10.58 0-19.156-8.577-19.156-19.157 0-4.155 1.322-8.001 3.57-11.139q.2-.334.472-.632.382-.49.795-.954c1.8-2.297 5.478-7.605 7.729-12.463 1.649-3.56 2.335-6.388 3.1-9.69-1.505.56-3.144 1.208-5.112 2.019-2.812 1.158-6.1 2.997-8.955 4.776-2.89 1.801-5.055 3.363-5.661 3.878q-.212.18-.439.324c-3.473 3.514-8.295 5.692-13.626 5.692-10.58 0-19.156-8.577-19.156-19.156 0-10.58 8.577-19.156 19.156-19.156 10.58 0 19.157 8.576 19.157 19.156l-.003.3c2.117-1.193 4.359-2.338 6.48-3.211 4.208-1.734 7.05-2.786 10.198-3.782l16.166-47.222c.715-2.09 2.99-3.204 5.08-2.489m-28.732 87.071c-1.394 1.862-2.22 4.174-2.22 6.678 0 6.162 4.995 11.157 11.157 11.157 6.161 0 11.156-4.995 11.156-11.157 0-6.161-4.995-11.156-11.156-11.156-3.263 0-6.199 1.401-8.239 3.634q-.389.487-.698.844M42.85 68.904c-6.16 0-11.156 4.994-11.156 11.156 0 6.161 4.995 11.156 11.156 11.156S54.008 86.22 54.008 80.06 49.013 68.904 42.85 68.904"
        })
    });
const $bdc9977f2cedccbd$var$ForwardRef = /*#__PURE__*/ (0, $2Vc2n$react.forwardRef)($bdc9977f2cedccbd$var$SvgComponent);
var $bdc9977f2cedccbd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($bdc9977f2cedccbd$var$ForwardRef);
