// @react-spectrum/s2/illustrations/linear/internal/S2_lin_paperairplane_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $huIqG$jsx} from "react/jsx-runtime";
import {forwardRef as $huIqG$forwardRef} from "react";





const $0be775f64cd932c3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $huIqG$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $huIqG$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M46.014 12.421c1.33-2.642-1.014-5.641-3.9-4.987L4.335 15.999c-2.974.674-3.723 4.566-1.21 6.296l.589-.857-.59.857 7.353 5.064v10.544c0 2.35 2.758 3.618 4.541 2.087l6.143-5.273 6.81 4.69c1.748 1.204 4.156.588 5.111-1.308zm-26.982 20.83-6.055-4.17v8.822c0 .214.25.329.413.19zm-4.714-6.282 15.071 10.38c.5.343 1.188.167 1.46-.374l12.559-24.938zM41.745 10.08 4.888 18.437c-.85.192-1.064 1.304-.346 1.799l7.438 5.122z"
        })
    });
const $0be775f64cd932c3$var$ForwardRef = /*#__PURE__*/ (0, $huIqG$forwardRef)($0be775f64cd932c3$var$SvgComponent);
var $0be775f64cd932c3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0be775f64cd932c3$var$ForwardRef);


export {$0be775f64cd932c3$export$2e2bcd8739ae039 as default};
