// @react-spectrum/s2/illustrations/linear/internal/S2_lin_upload_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8wMv8$jsx} from "react/jsx-runtime";
import {forwardRef as $8wMv8$forwardRef} from "react";





const $d129d4a0e7c7ae9b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8wMv8$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 49,
        viewBox: "0 0 48 49",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8wMv8$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8.5 24.68c.69 0 1.25.56 1.25 1.25v8.824c0 1.35 1.083 2.427 2.397 2.427h23.706c1.314 0 2.397-1.077 2.397-2.427v-8.823c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v8.823c0 2.712-2.183 4.927-4.897 4.927H12.147c-2.714 0-4.897-2.215-4.897-4.927v-8.823c0-.69.56-1.25 1.25-1.25M24 33.436c-.69 0-1.25-.56-1.25-1.25V13.604l-4.866 4.866c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.768l7-7c.488-.488 1.28-.488 1.768 0l7 7c.488.488.488 1.28 0 1.768s-1.28.488-1.768 0l-4.866-4.866v18.58c0 .69-.56 1.25-1.25 1.25"
        })
    });
const $d129d4a0e7c7ae9b$var$ForwardRef = /*#__PURE__*/ (0, $8wMv8$forwardRef)($d129d4a0e7c7ae9b$var$SvgComponent);
var $d129d4a0e7c7ae9b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d129d4a0e7c7ae9b$var$ForwardRef);


export {$d129d4a0e7c7ae9b$export$2e2bcd8739ae039 as default};
