// @react-spectrum/s2/illustrations/linear/internal/S2_lin_assets_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $iiWKg$reactjsxruntime = require("react/jsx-runtime");
var $iiWKg$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5bdfae0de7ddf77a$export$2e2bcd8739ae039);




const $5bdfae0de7ddf77a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iiWKg$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        fill: "none",
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $iiWKg$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.25 26.052c0-4.418 3.582-8 8-8H62c4.418 0 8 3.582 8 8v8.7c0 1.105-.895 2-2 2s-2-.895-2-2v-8.7c0-2.21-1.79-4-4-4H24.25c-2.21 0-4 1.79-4 4v25.656l9.885-15.322c2.013-3.12 6.592-3.07 8.535.093l.002.003 1.797 2.936c.577.942.28 2.174-.661 2.75s-2.173.28-2.75-.661l-1.796-2.934c-.403-.654-1.35-.664-1.766-.019L22.512 55.58c.526.253 1.115.396 1.738.396h12.313c1.104 0 2 .895 2 2s-.896 2-2 2H24.25c-4.418 0-8-3.582-8-8zm39.895 8.367c1.657 0 3-1.343 3-3s-1.343-3-3-3-3 1.343-3 3 1.343 3 3 3M41.552 47.75c0-4.418 3.581-8 8-8H71.75c4.418 0 8 3.582 8 8v22.198c0 4.419-3.582 8-8 8H49.552c-4.419 0-8-3.581-8-8zm8-4c-2.21 0-4 1.791-4 4v22.198c0 2.21 1.79 4 4 4H71.75c2.21 0 4-1.79 4-4V47.75c0-2.209-1.79-4-4-4zm7.265 7.53 11.214 6.432c1.256.72 1.254 2.517-.003 3.235l-11.23 6.406c-1.258.717-2.828-.183-2.826-1.621l.016-12.838c.001-1.438 1.574-2.335 2.83-1.614",
            clipRule: "evenodd"
        })
    });
const $5bdfae0de7ddf77a$var$ForwardRef = /*#__PURE__*/ (0, $iiWKg$react.forwardRef)($5bdfae0de7ddf77a$var$SvgComponent);
var $5bdfae0de7ddf77a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5bdfae0de7ddf77a$var$ForwardRef);
