// @react-spectrum/s2/illustrations/linear/internal/S2_lin_video_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9SO1y$jsx} from "react/jsx-runtime";
import {forwardRef as $9SO1y$forwardRef} from "react";





const $c2f1e4fe4c959ca0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9SO1y$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9SO1y$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M136 50c0-6.627-5.373-12-12-12H36c-6.627 0-12 5.373-12 12v60c0 6.627 5.373 12 12 12h88c6.627 0 12-5.373 12-12zm-12-20c11.046 0 20 8.954 20 20v60c0 11.046-8.954 20-20 20H36c-11.046 0-20-8.954-20-20V50c0-11.046 8.954-20 20-20zM71.811 64.479l20.933 12.119c2.344 1.357 2.34 4.743-.007 6.095L71.775 94.762c-2.348 1.352-5.278-.345-5.275-3.054l.029-24.188c.003-2.709 2.938-4.398 5.282-3.041"
        })
    });
const $c2f1e4fe4c959ca0$var$ForwardRef = /*#__PURE__*/ (0, $9SO1y$forwardRef)($c2f1e4fe4c959ca0$var$SvgComponent);
var $c2f1e4fe4c959ca0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c2f1e4fe4c959ca0$var$ForwardRef);


export {$c2f1e4fe4c959ca0$export$2e2bcd8739ae039 as default};
