// @react-spectrum/s2/illustrations/linear/internal/S2_lin_libraries_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $iP9jv$jsx} from "react/jsx-runtime";
import {forwardRef as $iP9jv$forwardRef} from "react";





const $ad09f21125526763$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iP9jv$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $iP9jv$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M10.41 15.18c0-2.62 2.13-4.75 4.75-4.75h13.67c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H15.16c-4 0-7.25 3.25-7.25 7.25v13.67c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25zm6.75-.78c-1.519 0-2.75 1.23-2.75 2.75v17.67c0 1.518 1.231 2.75 2.75 2.75h17.67c1.519 0 2.75-1.232 2.75-2.75V17.15c0-1.52-1.231-2.75-2.75-2.75h-.38v10.71c0 .4-.45.64-.78.42l-2.4-1.6c-.16-.11-.38-.11-.55 0l-2.4 1.6c-.33.22-.78-.02-.78-.42V14.4zm-5.25 2.75c0-2.9 2.35-5.25 5.25-5.25h17.67c2.9 0 5.25 2.35 5.25 5.25v17.67c0 2.9-2.35 5.25-5.25 5.25H17.16c-2.9 0-5.25-2.35-5.25-5.25z"
        })
    });
const $ad09f21125526763$var$ForwardRef = /*#__PURE__*/ (0, $iP9jv$forwardRef)($ad09f21125526763$var$SvgComponent);
var $ad09f21125526763$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ad09f21125526763$var$ForwardRef);


export {$ad09f21125526763$export$2e2bcd8739ae039 as default};
