// @react-spectrum/s2/illustrations/linear/internal/S2_lin_playTriangle_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6tmzx$jsx} from "react/jsx-runtime";
import {forwardRef as $6tmzx$forwardRef} from "react";





const $eb3c2743610aa8be$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6tmzx$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6tmzx$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22.05 26.618c.007-6.23 6.637-10.337 12.08-7.09l35.885 21.417c5.32 3.175 5.312 11.046-.017 14.208L34.063 76.481l-1.02-1.72 1.02 1.72c-5.45 3.235-12.07-.888-12.063-7.119zm10.03-3.654c-2.596-1.55-6.027.316-6.03 3.659L26 69.367c-.004 3.344 3.423 5.217 6.022 3.674l35.935-21.327c2.72-1.615 2.725-5.712.008-7.334zl.9-1.51z"
        })
    });
const $eb3c2743610aa8be$var$ForwardRef = /*#__PURE__*/ (0, $6tmzx$forwardRef)($eb3c2743610aa8be$var$SvgComponent);
var $eb3c2743610aa8be$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($eb3c2743610aa8be$var$ForwardRef);


export {$eb3c2743610aa8be$export$2e2bcd8739ae039 as default};
