// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowDown_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bp1K7$reactjsxruntime = require("react/jsx-runtime");
var $bp1K7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5884c0714277d1e1$export$2e2bcd8739ae039);




const $5884c0714277d1e1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bp1K7$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bp1K7$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M84 24c0-2.21-1.79-4-4-4s-4 1.79-4 4v104.843l-16.672-16.671c-1.562-1.563-4.094-1.563-5.656 0s-1.562 4.094 0 5.656l23.5 23.5c1.562 1.563 4.094 1.563 5.656 0l23.5-23.5c1.563-1.562 1.563-4.094 0-5.656-1.562-1.563-4.094-1.563-5.656 0L84 128.843z"
        })
    });
const $5884c0714277d1e1$var$ForwardRef = /*#__PURE__*/ (0, $bp1K7$react.forwardRef)($5884c0714277d1e1$var$SvgComponent);
var $5884c0714277d1e1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5884c0714277d1e1$var$ForwardRef);
