// @react-spectrum/s2/illustrations/linear/internal/S2_lin_conversationbubbles_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $jOJ8H$reactjsxruntime = require("react/jsx-runtime");
var $jOJ8H$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $59c6cd1931a67d9c$export$2e2bcd8739ae039);




const $59c6cd1931a67d9c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jOJ8H$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jOJ8H$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M30.56 24C22.553 24 16 30.445 16 38.468V75.62c0 8.024 6.552 14.469 14.56 14.469h5.463c-.004 4.317-.033 9.311-.063 13.363-.048 6.511 7.598 9.877 12.433 5.722l22.209-19.085h26.837c8.009 0 14.561-6.445 14.561-14.469v-37.15C112 30.444 105.448 24 97.44 24zM24 38.468C24 34.93 26.904 32 30.56 32h66.88c3.656 0 6.56 2.929 6.56 6.468V75.62c0 3.54-2.904 6.469-6.56 6.469H69.12c-.957 0-1.882.342-2.608.966l-22.545 19.374c.037-5.224.07-11.718.05-16.357-.009-2.203-1.797-3.983-4-3.983h-9.456c-3.657 0-6.561-2.93-6.561-6.469zM123 61c-2.209 0-4 1.79-4 4s1.791 4 4 4h7.64c2.995 0 5.36 2.396 5.36 5.276v32.918c0 2.88-2.365 5.276-5.36 5.276h-8.383c-2.202 0-3.99 1.781-4 3.983-.016 3.822.009 9.063.038 13.5L99.07 113.436c-.726-.623-1.65-.966-2.607-.966H71.36c-2.303 0-4.253-1.426-5.016-3.407-.794-2.062-3.108-3.09-5.17-2.296s-3.09 3.109-2.296 5.17c1.926 5 6.796 8.533 12.482 8.533h23.62l19.557 16.802c4.573 3.929 11.816.752 11.771-5.42-.026-3.448-.051-7.666-.056-11.382h4.387c7.344 0 13.36-5.91 13.36-13.276V74.276C144 66.91 137.984 61 130.64 61z"
        })
    });
const $59c6cd1931a67d9c$var$ForwardRef = /*#__PURE__*/ (0, $jOJ8H$react.forwardRef)($59c6cd1931a67d9c$var$SvgComponent);
var $59c6cd1931a67d9c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($59c6cd1931a67d9c$var$ForwardRef);
