// @react-spectrum/s2/illustrations/linear/internal/S2_lin_revert_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $nqnVu$jsx} from "react/jsx-runtime";
import {forwardRef as $nqnVu$forwardRef} from "react";





const $38d38ae7734297e5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $nqnVu$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $nqnVu$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m32.523 90.528 15.749-13.24c1.69-1.422 4.214-1.203 5.636.488 1.421 1.69 1.203 4.214-.488 5.635L30.56 102.63c-1.691 1.422-4.215 1.204-5.636-.487L5.706 79.282c-1.422-1.692-1.203-4.215.488-5.636 1.69-1.422 4.214-1.204 5.635.487l12.744 15.159c1.734-31.592 27.9-56.675 59.925-56.675 33.147 0 60.018 26.87 60.018 60.018 0 2.209-1.791 4-4 4s-4-1.791-4-4c0-28.729-23.289-52.018-52.018-52.018-28.022 0-50.87 22.159-51.975 49.91M21.54 123.383c0-2.209 1.79-4 4-4h117.885c2.209 0 4 1.791 4 4s-1.791 4-4 4H25.54c-2.21 0-4-1.791-4-4"
        })
    });
const $38d38ae7734297e5$var$ForwardRef = /*#__PURE__*/ (0, $nqnVu$forwardRef)($38d38ae7734297e5$var$SvgComponent);
var $38d38ae7734297e5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($38d38ae7734297e5$var$ForwardRef);


export {$38d38ae7734297e5$export$2e2bcd8739ae039 as default};
