// @react-spectrum/s2/illustrations/linear/internal/S2_lin_revert_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $10VV2$reactjsxruntime = require("react/jsx-runtime");
var $10VV2$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2374f687ad71be21$export$2e2bcd8739ae039);




const $2374f687ad71be21$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $10VV2$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $10VV2$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m10.857 27.221 5.516-4.376c.541-.43 1.327-.338 1.756.202.43.541.339 1.328-.202 1.756l-7.755 6.153c-.541.43-1.327.338-1.757-.202l-6.152-7.756c-.43-.54-.338-1.327.202-1.756.541-.429 1.327-.338 1.757.203l4.156 5.24c.526-9.277 8.215-16.638 17.623-16.638 9.749 0 17.651 7.903 17.651 17.651 0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25c0-8.368-6.783-15.151-15.151-15.151-8.208 0-14.892 6.527-15.144 14.674m-3.369 9.482c0-.69.56-1.25 1.25-1.25h34.516c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H8.738c-.69 0-1.25-.56-1.25-1.25"
        })
    });
const $2374f687ad71be21$var$ForwardRef = /*#__PURE__*/ (0, $10VV2$react.forwardRef)($2374f687ad71be21$var$SvgComponent);
var $2374f687ad71be21$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2374f687ad71be21$var$ForwardRef);
