// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pieChart_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bVaDP$reactjsxruntime = require("react/jsx-runtime");
var $bVaDP$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c5c86a5a573a44e8$export$2e2bcd8739ae039);




const $c5c86a5a573a44e8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bVaDP$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bVaDP$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M104.513 34.14c4.342 2.321 8.316 5.238 11.813 8.652q.596.582 1.173 1.183c2.112 2.197 4.033 4.584 5.737 7.135 4.751 7.11 7.675 15.283 8.514 23.793q.238 2.432.249 4.865L132 80l-.001.236c-.03 6.733-1.368 13.412-3.957 19.664-2.54 6.131-6.222 11.702-10.834 16.426q-.598.612-1.216 1.205-1.447 1.388-3.004 2.666c-6.61 5.425-14.456 9.136-22.843 10.804-3.318.66-6.68.992-10.038.999h-.205c-5.063-.01-10.116-.759-14.997-2.239-6.903-2.094-13.28-5.597-18.737-10.272l36.66-36.66C83.578 82.078 84 81.06 84 80V28.154c7.164.553 14.15 2.585 20.513 5.986M74.219 80.125 40.51 113.833C32.712 104.739 28 92.92 28 80c0-11.741 3.891-22.574 10.455-31.278zM76 71.043V28.152c-12.51.951-23.782 6.329-32.254 14.57zM20 80c0-33.137 26.863-60 60-60 16.176 0 30.857 6.401 41.648 16.81q.653.628 1.284 1.276c10.5 10.753 16.993 25.435 17.067 41.634v.562c-.073 16.039-6.44 30.591-16.758 41.314q-.653.678-1.327 1.336C111.138 133.454 96.416 139.953 80.176 140h-.344c-16.356-.045-31.174-6.635-41.971-17.288l-.256-.254-.031-.032-.004-.003C26.714 111.565 20 96.567 20 80"
        })
    });
const $c5c86a5a573a44e8$var$ForwardRef = /*#__PURE__*/ (0, $bVaDP$react.forwardRef)($c5c86a5a573a44e8$var$SvgComponent);
var $c5c86a5a573a44e8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c5c86a5a573a44e8$var$ForwardRef);
