// @react-spectrum/s2/illustrations/linear/internal/S2_lin_update_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $fB0Kn$jsx} from "react/jsx-runtime";
import {forwardRef as $fB0Kn$forwardRef} from "react";





const $7d037c499231f200$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fB0Kn$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fB0Kn$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M30.02 65.68c6.315-21.62 26.286-37.4 49.93-37.4 20.842 0 38.825 12.258 47.122 29.972l-13.851-6.458c-2.002-.933-4.382-.067-5.315 1.935s-.067 4.382 1.935 5.316l21.17 9.87c1.953.91 4.277.11 5.255-1.811l10.98-21.57c1.002-1.969.218-4.377-1.75-5.38-1.969-1.002-4.377-.218-5.38 1.75l-6.184 12.15c-9.732-19.99-30.24-33.775-53.981-33.775-27.295 0-50.325 18.22-57.61 43.159-.62 2.12.598 4.341 2.718 4.96 2.12.62 4.342-.597 4.961-2.717m-1.027 26.175c-1.954-.912-4.28-.111-5.257 1.81l-10.98 21.58c-1.002 1.969-.218 4.378 1.751 5.379 1.97 1.002 4.377.218 5.38-1.751l6.142-12.072c9.787 19.835 30.213 33.488 53.842 33.488 27.294 0 50.324-18.219 57.609-43.158.619-2.12-.597-4.342-2.718-4.961s-4.342.597-4.961 2.718c-6.315 21.62-26.285 37.401-49.93 37.401-20.782 0-38.721-12.188-47.05-29.82l13.959 6.515c2.001.934 4.382.069 5.316-1.933s.069-4.382-1.933-5.316z"
        })
    });
const $7d037c499231f200$var$ForwardRef = /*#__PURE__*/ (0, $fB0Kn$forwardRef)($7d037c499231f200$var$SvgComponent);
var $7d037c499231f200$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7d037c499231f200$var$ForwardRef);


export {$7d037c499231f200$export$2e2bcd8739ae039 as default};
