// @react-spectrum/s2/illustrations/linear/internal/S2_lin_threeArrowsCircle_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $exJ2K$jsx} from "react/jsx-runtime";
import {forwardRef as $exJ2K$forwardRef} from "react";





const $b47307e3a887ec59$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $exJ2K$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $exJ2K$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M72.81 23.503c.254 1.074-.412 2.152-1.487 2.406L57.56 29.162c-1.075.254-2.152-.411-2.406-1.486s.411-2.152 1.486-2.407l9.17-2.167C60.75 19.51 54.569 17.4 47.89 17.4c-9.298 0-17.638 4.09-23.323 10.578-.728.83-1.992.914-2.822.186s-.914-1.992-.186-2.823c6.412-7.316 15.833-11.941 26.331-11.941 7.384 0 14.238 2.289 19.886 6.194l-2.112-8.935c-.255-1.075.41-2.153 1.486-2.407 1.075-.254 2.152.412 2.406 1.487zm4.828 15.673c-.33-1.054.257-2.176 1.311-2.506s2.176.257 2.506 1.311c1.035 3.307 1.591 6.822 1.591 10.463 0 14.285-8.557 26.567-20.819 32.007l8.777 2.508c1.063.303 1.677 1.41 1.374 2.472s-1.41 1.678-2.472 1.374L56.307 82.92c-1.062-.303-1.677-1.41-1.373-2.472l3.885-13.598c.303-1.062 1.41-1.677 2.472-1.374s1.677 1.41 1.374 2.472l-2.594 9.078c11.151-4.697 18.975-15.728 18.975-28.582 0-3.232-.494-6.344-1.409-9.268m-64.312 4.547-6.932 5.825c-.846.71-2.107.6-2.818-.245s-.6-2.107.245-2.818l10.827-9.097c.846-.71 2.108-.601 2.818.245l9.098 10.827c.71.846.6 2.107-.245 2.818-.846.71-2.107.6-2.818-.245l-6.134-7.3c-.932 6.024-.1 12.383 2.773 18.281 4.071 8.36 11.4 14.066 19.722 16.337 1.066.29 1.694 1.39 1.403 2.456-.29 1.065-1.39 1.694-2.456 1.403-9.386-2.561-17.668-9.006-22.265-18.444-3.147-6.46-4.125-13.418-3.218-20.042"
        })
    });
const $b47307e3a887ec59$var$ForwardRef = /*#__PURE__*/ (0, $exJ2K$forwardRef)($b47307e3a887ec59$var$SvgComponent);
var $b47307e3a887ec59$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b47307e3a887ec59$var$ForwardRef);


export {$b47307e3a887ec59$export$2e2bcd8739ae039 as default};
