// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browserError_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $gb9pw$reactjsxruntime = require("react/jsx-runtime");
var $gb9pw$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f530fc27d59d1024$export$2e2bcd8739ae039);




const $f530fc27d59d1024$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gb9pw$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gb9pw$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M120 32c6.627 0 12 5.373 12 12v12H20V44c0-6.627 5.373-12 12-12zM20 64h112v25c0 2.21 1.791 4 4 4s4-1.79 4-4V44c0-11.046-8.954-20-20-20H32c-11.046 0-20 8.954-20 20v72c0 11.046 8.954 20 20 20h62c2.21 0 4-1.791 4-4s-1.79-4-4-4H32c-6.627 0-12-5.373-12-12zm21.48-19.5c0 2.486-2.013 4.5-4.5 4.5s-4.5-2.014-4.5-4.5 2.014-4.5 4.5-4.5h.012c2.475 0 4.489 2.014 4.489 4.5m62.51 79.09c0-13.53 10.97-24.5 24.5-24.5s24.5 10.97 24.5 24.5-10.97 24.5-24.5 24.5-24.5-10.97-24.5-24.5m35.42 10.93c1.56-1.57 1.56-4.1 0-5.66l-5.26-5.26 5.26-5.26c1.56-1.57 1.56-4.1 0-5.66s-4.1-1.56-5.66 0l-5.26 5.26-5.26-5.26c-1.56-1.56-4.1-1.56-5.66 0-1.56 1.57-1.56 4.1 0 5.66l5.26 5.26-5.26 5.26c-1.56 1.57-1.56 4.1 0 5.66.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17l5.26-5.26 5.26 5.26c.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17"
        })
    });
const $f530fc27d59d1024$var$ForwardRef = /*#__PURE__*/ (0, $gb9pw$react.forwardRef)($f530fc27d59d1024$var$SvgComponent);
var $f530fc27d59d1024$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f530fc27d59d1024$var$ForwardRef);
