// @react-spectrum/s2/illustrations/linear/internal/S2_lin_sparkles_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $58SIo$jsx} from "react/jsx-runtime";
import {forwardRef as $58SIo$forwardRef} from "react";





const $0a4c34d703708c51$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $58SIo$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $58SIo$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M38.402 9.842c.785-3.493-3.402-5.913-6.035-3.485l-3.9 3.594c-.278.256-.665.359-1.031.276h-.002l-5.175-1.165c-3.493-.784-5.913 3.403-3.486 6.036l3.594 3.9c.256.277.36.665.276 1.03v.003l-1.164 5.175c-.784 3.493 3.403 5.913 6.036 3.485l3.9-3.594c.277-.256.664-.359 1.03-.276h.002l5.175 1.165c3.493.784 5.913-3.403 3.486-6.036l-3.594-3.9c-.256-.277-.36-.664-.276-1.03v-.003zm-4.341-1.647c.829-.764 2.15-.004 1.902 1.1L34.8 14.466c-.265 1.17.063 2.396.876 3.277l3.593 3.9c.765.83.005 2.15-1.098 1.903l-5.173-1.164h-.001c-1.17-.265-2.396.064-3.276.876l-3.9 3.594c-.83.764-2.15.004-1.903-1.099l1.164-5.173v-.001c.264-1.17-.064-2.395-.876-3.276l-3.594-3.9c-.765-.83-.005-2.15 1.099-1.902l5.172 1.163h.001c1.17.265 2.396-.064 3.277-.876zM19.314 38.401l-3.648-.822c-.482-.108-.984.027-1.347.36l-2.748 2.535c-1.083.999-2.805.003-2.48-1.431l.821-3.648c.108-.483-.027-.984-.36-1.347L7.017 31.3c-.999-1.083-.003-2.805 1.431-2.481l3.648.822c.483.108.984-.027 1.347-.36l2.749-2.535c1.082-1 2.805-.003 2.48 1.43l-.822 3.649c-.108.483.027.984.36 1.347l2.535 2.748c1 1.083.003 2.805-1.43 2.48"
        })
    });
const $0a4c34d703708c51$var$ForwardRef = /*#__PURE__*/ (0, $58SIo$forwardRef)($0a4c34d703708c51$var$SvgComponent);
var $0a4c34d703708c51$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0a4c34d703708c51$var$ForwardRef);


export {$0a4c34d703708c51$export$2e2bcd8739ae039 as default};
