// @react-spectrum/s2/illustrations/linear/internal/S2_lin_accessibility_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $fFFyI$reactjsxruntime = require("react/jsx-runtime");
var $fFFyI$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5cf7cf5049c7c00f$export$2e2bcd8739ae039);




const $5cf7cf5049c7c00f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fFFyI$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fFFyI$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8.75 24c0-8.422 6.828-15.25 15.25-15.25S39.25 15.578 39.25 24 32.422 39.25 24 39.25 8.75 32.422 8.75 24M24 6.25C14.197 6.25 6.25 14.197 6.25 24S14.197 41.75 24 41.75 41.75 33.803 41.75 24 33.803 6.25 24 6.25m2.5 8.822c0 1.381-1.12 2.5-2.5 2.5s-2.5-1.119-2.5-2.5c0-1.38 1.12-2.5 2.5-2.5s2.5 1.12 2.5 2.5m-9.782 2.872c-.531-.152-1.085.155-1.237.686s.156 1.085.687 1.237c1.443.413 2.911.71 4.39.89.514.062.91.492.91 1.01v3.953q0 .139-.038.273l-1.258 4.444-.735 2.608-.224.8-.012.044c-.04.14-.08.287-.091.34-.108.542.243 1.07.784 1.177.542.109 1.069-.242 1.177-.784-.005.026 0 .011.024-.08l.045-.16.223-.796c.187-.668.446-1.586.734-2.606.328-1.164.694-2.459 1.036-3.664.11-.387.464-.655.867-.655s.757.267.867.655c.341 1.205.708 2.5 1.036 3.664l.735 2.606.222.796.045.16c.025.091.03.106.024.08.108.541.635.893 1.177.784.541-.108.892-.635.784-1.176-.01-.054-.052-.2-.09-.34l-.013-.045-.224-.8-.735-2.608c-.4-1.42-.857-3.034-1.258-4.444q-.038-.134-.038-.273v-3.953c0-.518.396-.948.91-1.01 1.412-.173 2.815-.451 4.196-.836l.188-.052c.532-.148.843-.7.695-1.232s-.7-.843-1.232-.695l-.187.052c-4.708 1.312-9.686 1.294-14.384-.05"
        })
    });
const $5cf7cf5049c7c00f$var$ForwardRef = /*#__PURE__*/ (0, $fFFyI$react.forwardRef)($5cf7cf5049c7c00f$var$SvgComponent);
var $5cf7cf5049c7c00f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5cf7cf5049c7c00f$var$ForwardRef);
