// @react-spectrum/s2/illustrations/linear/internal/S2_lin_shoppingBag_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1RyE2$jsx} from "react/jsx-runtime";
import {forwardRef as $1RyE2$forwardRef} from "react";





const $117cd3414ab74894$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1RyE2$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1RyE2$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M21.25 12c0-1.38 1.12-2.5 2.5-2.5 1.381 0 2.5 1.12 2.5 2.5v1.5h-5zm-2.5 4v2.75c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V16h5v2.75c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V16h4.62l1.947 19.476c.162 1.62-1.11 3.024-2.736 3.024H14.92c-1.627 0-2.899-1.405-2.737-3.024L14.132 16zm0-2.5V12c0-2.761 2.24-5 5-5 2.762 0 5 2.239 5 5v1.5h4.846c1.155 0 2.123.876 2.238 2.026l1.97 19.702c.31 3.09-2.117 5.772-5.224 5.772H14.92c-3.106 0-5.533-2.682-5.224-5.772l1.97-19.702c.116-1.15 1.083-2.026 2.24-2.026z"
        })
    });
const $117cd3414ab74894$var$ForwardRef = /*#__PURE__*/ (0, $1RyE2$forwardRef)($117cd3414ab74894$var$SvgComponent);
var $117cd3414ab74894$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($117cd3414ab74894$var$ForwardRef);


export {$117cd3414ab74894$export$2e2bcd8739ae039 as default};
