// @react-spectrum/s2/illustrations/linear/internal/S2_lin_redo_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $uN2Xh$reactjsxruntime = require("react/jsx-runtime");
var $uN2Xh$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $94968610930586de$export$2e2bcd8739ae039);




const $94968610930586de$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $uN2Xh$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $uN2Xh$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m117.577 80.078 23.5-23.5c1.563-1.562 1.563-4.094 0-5.656l-23.5-23.5c-1.562-1.562-4.094-1.562-5.656 0-1.563 1.562-1.563 4.094 0 5.656l16.629 16.63H61.978c-23.35 0-42.279 18.929-42.279 42.279s18.93 42.279 42.28 42.279h27.12c2.209 0 4-1.791 4-4s-1.791-4-4-4h-27.12c-18.932 0-34.28-15.347-34.28-34.28 0-18.931 15.348-34.278 34.28-34.278h66.656l-16.714 16.714c-1.563 1.562-1.563 4.094 0 5.656 1.562 1.563 4.094 1.563 5.656 0"
        })
    });
const $94968610930586de$var$ForwardRef = /*#__PURE__*/ (0, $uN2Xh$react.forwardRef)($94968610930586de$var$SvgComponent);
var $94968610930586de$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($94968610930586de$var$ForwardRef);
