// @react-spectrum/s2/illustrations/linear/internal/S2_lin_microphone_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7d5SR$jsx} from "react/jsx-runtime";
import {forwardRef as $7d5SR$forwardRef} from "react";





const $bc5b89171bd71e0b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7d5SR$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7d5SR$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M79 20.8c-12.812 0-23.2 10.388-23.2 23.2v28.6c0 12.814 10.388 23.2 23.2 23.2h2.6c12.814 0 23.201-10.386 23.201-23.2V44c0-12.812-10.387-23.2-23.2-23.2zM63.8 44c0-8.394 6.806-15.2 15.2-15.2h2.6c8.396 0 15.2 6.806 15.2 15.2v28.6c0 8.396-6.804 15.2-15.2 15.2H79c-8.394 0-15.2-6.804-15.2-15.2zm12 71.576c-22.462-2.256-40-21.217-40-44.275v-4.5c0-2.21 1.792-4 4-4 2.21 0 4 1.79 4 4v4.5c0 20.158 16.342 36.5 36.5 36.5s36.501-16.342 36.501-36.5v-4.5c0-2.21 1.791-4 4-4s4 1.79 4 4v4.5c0 23.399-18.06 42.579-41 44.364v12.136h20c2.209 0 4 1.791 4 4s-1.791 4-4 4h-47c-2.21 0-4-1.791-4-4s1.79-4 4-4h19z"
        })
    });
const $bc5b89171bd71e0b$var$ForwardRef = /*#__PURE__*/ (0, $7d5SR$forwardRef)($bc5b89171bd71e0b$var$SvgComponent);
var $bc5b89171bd71e0b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($bc5b89171bd71e0b$var$ForwardRef);


export {$bc5b89171bd71e0b$export$2e2bcd8739ae039 as default};
