// @react-spectrum/s2/illustrations/linear/internal/S2_lin_addproject_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $5k6KX$reactjsxruntime = require("react/jsx-runtime");
var $5k6KX$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9e94c7b9c6ece2b3$export$2e2bcd8739ae039);




const $9e94c7b9c6ece2b3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5k6KX$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5k6KX$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M44.886 46.047c.926-2.247 3.116-3.713 5.547-3.713h53.716c2.431 0 4.621 1.466 5.547 3.713l3.94 9.555q-.357-.018-.72-.018h-71.25q-.362 0-.72.018zm80.478 17.012q.078.19.135.38c.908 1.856 1.417 3.94 1.417 6.145v12.917c0 2.209-1.791 4-4 4s-4-1.791-4-4V69.584c0-3.314-2.686-6-6-6h-71.25c-3.314 0-6 2.686-6 6v35c0 3.314 2.686 6 6 6h49.167c2.209 0 4 1.791 4 4s-1.791 4-4 4H41.666c-7.732 0-14-6.268-14-14v-35c0-2.204.51-4.29 1.417-6.144q.057-.192.135-.38l8.272-20.063c2.161-5.242 7.272-8.663 12.943-8.663h53.716c5.671 0 10.782 3.42 12.943 8.663zM64.166 84.834h27.5c2.21 0 4-1.79 4-4s-1.79-4-4-4h-27.5c-2.21 0-4 1.79-4 4s1.79 4 4 4m58.652 52.966c12.785 0 23.15-10.364 23.15-23.15s-10.365-23.15-23.15-23.15-23.15 10.365-23.15 23.15c0 12.786 10.365 23.15 23.15 23.15m4.183-33.633c0-2.209-1.791-4-4-4s-4 1.791-4 4v6.417h-6.599c-2.209 0-4 1.791-4 4s1.791 4 4 4h6.599V125c0 2.21 1.791 4 4 4s4-1.79 4-4v-6.416h6.235c2.209 0 4-1.791 4-4s-1.791-4-4-4h-6.235z"
        })
    });
const $9e94c7b9c6ece2b3$var$ForwardRef = /*#__PURE__*/ (0, $5k6KX$react.forwardRef)($9e94c7b9c6ece2b3$var$SvgComponent);
var $9e94c7b9c6ece2b3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9e94c7b9c6ece2b3$var$ForwardRef);
