// @react-spectrum/s2/illustrations/linear/internal/S2_lin_clipboard_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $b01xN$reactjsxruntime = require("react/jsx-runtime");
var $b01xN$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $376ccad9ba333d61$export$2e2bcd8739ae039);




const $376ccad9ba333d61$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $b01xN$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $b01xN$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M23.513 5.95c-2.704-.003-4.733 2.182-4.761 4.795l-1.245-.002q-.069 0-.136.007h-3.648c-2.76 0-4.973 2.258-4.973 5.014v19.473c0 2.756 2.214 5.013 4.973 5.013h19.604c2.76 0 4.973-2.257 4.973-5.014V15.764c0-2.756-2.213-5.014-4.973-5.014H28.25c-.025-2.604-2.029-4.797-4.737-4.8m7.241 7.3-.004 3.252c0 .69-.561 1.249-1.251 1.248l-12-.014c-.332 0-.65-.133-.884-.367-.234-.235-.365-.553-.365-.885l.004-3.234h-2.531c-1.353 0-2.473 1.113-2.473 2.514v19.473c0 1.4 1.12 2.513 2.473 2.513h19.604c1.353 0 2.473-1.112 2.473-2.514V15.764c0-1.401-1.12-2.514-2.473-2.514zm-12 0-.002 1.987 9.5.012.002-1.993-1.257-.002c-.331 0-.65-.132-.883-.367-.235-.234-.366-.553-.365-.884l.001-1.2c.002-1.348-.991-2.351-2.24-2.353-1.255-.001-2.257 1.006-2.259 2.347l-.001 1.2c0 .69-.561 1.25-1.251 1.249h-.632v.004z"
        })
    });
const $376ccad9ba333d61$var$ForwardRef = /*#__PURE__*/ (0, $b01xN$react.forwardRef)($376ccad9ba333d61$var$SvgComponent);
var $376ccad9ba333d61$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($376ccad9ba333d61$var$ForwardRef);
