// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileVideo_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $fPFbD$reactjsxruntime = require("react/jsx-runtime");
var $fPFbD$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ba41537f636fdead$export$2e2bcd8739ae039);




const $ba41537f636fdead$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fPFbD$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fPFbD$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M10.25 12.462c0-2.631 2.176-4.712 4.796-4.712h9.167c1.262 0 2.479.49 3.38 1.37l9.742 9.51c.902.881 1.415 2.083 1.415 3.342v13.566c0 2.631-2.176 4.712-4.795 4.712h-18.91c-2.62 0-4.795-2.081-4.795-4.712zm4.796-2.212c-1.296 0-2.296 1.018-2.296 2.211v23.077c0 1.195 1 2.212 2.296 2.212h18.909c1.295 0 2.295-1.017 2.295-2.212V21.972c0-.577-.234-1.136-.662-1.553l-9.741-9.51c-.428-.419-1.016-.659-1.634-.659zm6.816 11.947 6.355 3.645c.712.408.71 1.426-.002 1.833l-6.364 3.63c-.712.406-1.602-.104-1.601-.919l.009-7.275c0-.814.892-1.323 1.603-.914"
        })
    });
const $ba41537f636fdead$var$ForwardRef = /*#__PURE__*/ (0, $fPFbD$react.forwardRef)($ba41537f636fdead$var$SvgComponent);
var $ba41537f636fdead$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ba41537f636fdead$var$ForwardRef);
