// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowRight_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1uPxt$jsx} from "react/jsx-runtime";
import {forwardRef as $1uPxt$forwardRef} from "react";





const $5fe65b5e2b7f51bf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1uPxt$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1uPxt$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M6.5 22.75c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h30.982l-4.866 4.866c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0l7-7c.488-.488.488-1.28 0-1.768l-7-7c-.488-.488-1.28-.488-1.768 0s-.488 1.28 0 1.768l4.866 4.866z"
        })
    });
const $5fe65b5e2b7f51bf$var$ForwardRef = /*#__PURE__*/ (0, $1uPxt$forwardRef)($5fe65b5e2b7f51bf$var$SvgComponent);
var $5fe65b5e2b7f51bf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5fe65b5e2b7f51bf$var$ForwardRef);


export {$5fe65b5e2b7f51bf$export$2e2bcd8739ae039 as default};
