// @react-spectrum/s2/illustrations/linear/internal/S2_lin_calendar_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $10Dhy$reactjsxruntime = require("react/jsx-runtime");
var $10Dhy$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f0f421559d861908$export$2e2bcd8739ae039);




const $f0f421559d861908$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $10Dhy$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $10Dhy$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M34 14c1.105 0 2 .895 2 2v4h24v-4c0-1.105.895-2 2-2s2 .895 2 2v4h6c5.523 0 10 4.477 10 10v40c0 5.523-4.477 10-10 10H26c-5.523 0-10-4.477-10-10V30c0-5.523 4.477-10 10-10h6v-4c0-1.105.895-2 2-2m26 10v4c0 1.105.895 2 2 2s2-.895 2-2v-4h6c3.314 0 6 2.686 6 6v7H20v-7c0-3.314 2.686-6 6-6h6v4c0 1.105.895 2 2 2s2-.895 2-2v-4zM20 41v29c0 3.314 2.686 6 6 6h44c3.314 0 6-2.686 6-6V41zm14 7c1.105 0 2 .895 2 2v4c0 1.105-.895 2-2 2h-4c-1.105 0-2-.895-2-2v-4c0-1.105.895-2 2-2zm16 0h-4c-1.105 0-2 .895-2 2v4c0 1.105.895 2 2 2h4c1.105 0 2-.895 2-2v-4c0-1.105-.895-2-2-2m10 2c0-1.105.895-2 2-2h4c1.105 0 2 .895 2 2v4c0 1.105-.895 2-2 2h-4c-1.105 0-2-.895-2-2zM30 61c-1.105 0-2 .895-2 2v4c0 1.105.895 2 2 2h4c1.105 0 2-.895 2-2v-4c0-1.105-.895-2-2-2zm16 0h4c1.105 0 2 .895 2 2v4c0 1.105-.895 2-2 2h-4c-1.105 0-2-.895-2-2v-4c0-1.105.895-2 2-2m22 2c0-1.105-.895-2-2-2h-4c-1.105 0-2 .895-2 2v4c0 1.105.895 2 2 2h4c1.105 0 2-.895 2-2z"
        })
    });
const $f0f421559d861908$var$ForwardRef = /*#__PURE__*/ (0, $10Dhy$react.forwardRef)($f0f421559d861908$var$SvgComponent);
var $f0f421559d861908$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f0f421559d861908$var$ForwardRef);
