// @react-spectrum/s2/illustrations/linear/internal/S2_lin_volume_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $4tfWt$jsx} from "react/jsx-runtime";
import {forwardRef as $4tfWt$forwardRef} from "react";





const $670abc44e9a85311$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4tfWt$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4tfWt$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M51.434 21.461c0-4.444-5.365-6.68-8.52-3.55L31.251 29.472c-.375.371-.881.58-1.408.58h-8.38c-3.313 0-6 2.686-6 6v24.022c0 3.313 2.687 6 6 6h8.391c.524 0 1.027.205 1.401.572l11.675 11.46c3.163 3.104 8.503.863 8.503-3.569zm-5.704-.71c.63-.626 1.704-.179 1.704.71v53.077c0 .887-1.068 1.335-1.7.714l-11.676-11.46-.03-.03V32.354l.04-.04zM30.027 34.05v28.027q-.085-.002-.172-.002h-8.39c-1.105 0-2-.896-2-2V36.052c0-1.104.895-2 2-2h8.378q.093 0 .184-.002m29.184.326c-.883-.664-2.137-.487-2.8.396-.665.882-.488 2.136.395 2.8 3.148 2.37 5.178 6.132 5.178 10.369 0 4.271-2.063 8.06-5.256 10.426-.887.658-1.074 1.91-.416 2.798.658.887 1.91 1.074 2.798.416 4.167-3.088 6.874-8.048 6.874-13.64 0-5.547-2.663-10.472-6.773-13.565m10.004-9.129c-.882-.664-2.136-.487-2.8.396-.665.882-.488 2.136.395 2.8 5.91 4.448 9.725 11.514 9.725 19.473 0 8.023-3.878 15.14-9.87 19.58-.888.658-1.074 1.91-.416 2.798.657.887 1.91 1.074 2.797.416 6.967-5.162 11.489-13.45 11.489-22.794 0-9.269-4.45-17.498-11.32-22.669"
        })
    });
const $670abc44e9a85311$var$ForwardRef = /*#__PURE__*/ (0, $4tfWt$forwardRef)($670abc44e9a85311$var$SvgComponent);
var $670abc44e9a85311$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($670abc44e9a85311$var$ForwardRef);


export {$670abc44e9a85311$export$2e2bcd8739ae039 as default};
