// @react-spectrum/s2/illustrations/linear/internal/S2_lin_tag_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $7tRX5$reactjsxruntime = require("react/jsx-runtime");
var $7tRX5$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $02f63499de16a120$export$2e2bcd8739ae039);




const $02f63499de16a120$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7tRX5$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7tRX5$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M9.141 5.75C7.281 5.727 5.75 7.223 5.75 9.09v10.982c0 .887.354 1.737.983 2.363L25.04 40.662c2.127 2.117 5.572 2.117 7.699 0l7.914-7.88c2.13-2.12 2.13-5.56 0-7.68L22.335 6.864c-.618-.615-1.452-.965-2.324-.976zM8.25 9.09c0-.463.381-.846.86-.84l10.87.138c.223.003.435.092.59.248L38.89 26.874c1.148 1.143 1.148 2.994 0 4.137l-7.914 7.88c-1.151 1.146-3.02 1.146-4.171 0L8.497 20.662c-.159-.157-.247-.37-.247-.591zM13 15.5c0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5-1.12-2.5-2.5-2.5-2.5 1.12-2.5 2.5"
        })
    });
const $02f63499de16a120$var$ForwardRef = /*#__PURE__*/ (0, $7tRX5$react.forwardRef)($02f63499de16a120$var$SvgComponent);
var $02f63499de16a120$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($02f63499de16a120$var$ForwardRef);
