// @react-spectrum/s2/illustrations/linear/internal/S2_lin_targeted_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $EpAn9$reactjsxruntime = require("react/jsx-runtime");
var $EpAn9$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $bed612e374f978d8$export$2e2bcd8739ae039);




const $bed612e374f978d8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $EpAn9$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $EpAn9$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M14.535 30.565c-.696.178-1.243.716-1.433 1.41-.19.692.007 1.434.515 1.942l15.729 15.729c.415.415.991.627 1.576.579l12.068-.982 22.952 22.951c1.562 1.562 4.095 1.562 5.657 0s1.562-4.094 0-5.657l-22.95-22.95.981-12.07c.048-.585-.164-1.161-.579-1.576L33.322 14.21c-.508-.508-1.25-.704-1.943-.514s-1.23.737-1.409 1.433l-3.147 12.287zm65.284-2.62c-6.31 0-12.35 1.122-17.936 3.173-2.074.762-4.372-.302-5.134-2.376s.302-4.372 2.376-5.134c6.456-2.37 13.429-3.663 20.694-3.663 33.152 0 60.028 26.875 60.028 60.027S112.971 140 79.819 140s-60.027-26.875-60.027-60.028c0-7.074 1.225-13.87 3.479-20.184.742-2.08 3.031-3.165 5.111-2.422 2.081.742 3.166 3.031 2.423 5.112-1.95 5.462-3.013 11.35-3.013 17.494 0 28.734 23.293 52.028 52.027 52.028s52.028-23.294 52.028-52.028-23.294-52.027-52.028-52.027m-.001 26.517c-3.393 0-6.62.66-9.57 1.855-2.048.83-4.38-.158-5.21-2.205s.158-4.38 2.206-5.21c3.888-1.574 8.135-2.44 12.574-2.44 18.506 0 33.508 15.003 33.508 33.509S98.324 113.48 79.818 113.48 46.309 98.477 46.309 79.97c0-4.135.751-8.105 2.129-11.774.776-2.068 3.082-3.115 5.15-2.339s3.116 3.083 2.34 5.15c-1.046 2.784-1.619 5.802-1.619 8.964 0 14.088 11.42 25.509 25.509 25.509 14.088 0 25.508-11.421 25.508-25.51 0-14.087-11.42-25.508-25.508-25.508M70.68 85.627c7.73-.72 13.881-6.873 14.6-14.602 3.09 1.861 5.158 5.25 5.158 9.12 0 5.876-4.763 10.639-10.639 10.639-3.87 0-7.257-2.067-9.119-5.157"
        })
    });
const $bed612e374f978d8$var$ForwardRef = /*#__PURE__*/ (0, $EpAn9$react.forwardRef)($bed612e374f978d8$var$SvgComponent);
var $bed612e374f978d8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($bed612e374f978d8$var$ForwardRef);
