// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowUp_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $gz6uZ$reactjsxruntime = require("react/jsx-runtime");
var $gz6uZ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $816330096bfec632$export$2e2bcd8739ae039);




const $816330096bfec632$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gz6uZ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gz6uZ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22.75 41.5c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V10.518l4.866 4.866c.488.488 1.28.488 1.768 0s.488-1.28 0-1.768l-7-7c-.488-.488-1.28-.488-1.768 0l-7 7c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0l4.866-4.866z"
        })
    });
const $816330096bfec632$var$ForwardRef = /*#__PURE__*/ (0, $gz6uZ$react.forwardRef)($816330096bfec632$var$SvgComponent);
var $816330096bfec632$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($816330096bfec632$var$ForwardRef);
