// @react-spectrum/s2/illustrations/linear/internal/S2_lin_revert_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $K4CKr$jsx} from "react/jsx-runtime";
import {forwardRef as $K4CKr$forwardRef} from "react";





const $e0f09d52901c5d81$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $K4CKr$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $K4CKr$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m10.857 27.221 5.516-4.376c.541-.43 1.327-.338 1.756.202.43.541.339 1.328-.202 1.756l-7.755 6.153c-.541.43-1.327.338-1.757-.202l-6.152-7.756c-.43-.54-.338-1.327.202-1.756.541-.429 1.327-.338 1.757.203l4.156 5.24c.526-9.277 8.215-16.638 17.623-16.638 9.749 0 17.651 7.903 17.651 17.651 0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25c0-8.368-6.783-15.151-15.151-15.151-8.208 0-14.892 6.527-15.144 14.674m-3.369 9.482c0-.69.56-1.25 1.25-1.25h34.516c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H8.738c-.69 0-1.25-.56-1.25-1.25"
        })
    });
const $e0f09d52901c5d81$var$ForwardRef = /*#__PURE__*/ (0, $K4CKr$forwardRef)($e0f09d52901c5d81$var$SvgComponent);
var $e0f09d52901c5d81$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e0f09d52901c5d81$var$ForwardRef);


export {$e0f09d52901c5d81$export$2e2bcd8739ae039 as default};
