// @react-spectrum/s2/illustrations/linear/internal/S2_lin_graphBarChart_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $767yu$jsx} from "react/jsx-runtime";
import {forwardRef as $767yu$forwardRef} from "react";





const $7495191d58c4310d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $767yu$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $767yu$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M7.96 7.19c0-.69-.559-1.25-1.25-1.25-.69 0-1.25.56-1.25 1.25v34.09c0 .69.56 1.25 1.25 1.25h34.03c.691 0 1.25-.56 1.25-1.25s-.559-1.25-1.25-1.25H7.96zm4.64 15.83c0-.415.336-.75.75-.75h1.88c.415 0 .75.335.75.75v11.56c0 .414-.335.75-.75.75h-1.88c-.414 0-.75-.336-.75-.75zm.75-3.25c-1.794 0-3.25 1.455-3.25 3.25v11.56c0 1.794 1.456 3.25 3.25 3.25h1.88c1.796 0 3.25-1.456 3.25-3.25V23.02c0-1.795-1.454-3.25-3.25-3.25zm21.16 9.83c-.414 0-.75.335-.75.75v4.23c0 .414.336.75.75.75h1.88c.415 0 .75-.336.75-.75v-4.23c0-.415-.335-.75-.75-.75zm-3.25.75c0-1.795 1.456-3.25 3.25-3.25h1.88c1.795 0 3.25 1.455 3.25 3.25v4.23c0 1.794-1.455 3.25-3.25 3.25h-1.88c-1.794 0-3.25-1.456-3.25-3.25zM23.18 13.9c0-.415.336-.75.75-.75h1.88c.415 0 .75.335.75.75v20.68c0 .414-.335.75-.75.75h-1.88c-.414 0-.75-.336-.75-.75zm.75-3.25c-1.794 0-3.25 1.455-3.25 3.25v20.68c0 1.794 1.456 3.25 3.25 3.25h1.88c1.796 0 3.25-1.456 3.25-3.25V13.9c0-1.795-1.454-3.25-3.25-3.25z"
        })
    });
const $7495191d58c4310d$var$ForwardRef = /*#__PURE__*/ (0, $767yu$forwardRef)($7495191d58c4310d$var$SvgComponent);
var $7495191d58c4310d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7495191d58c4310d$var$ForwardRef);


export {$7495191d58c4310d$export$2e2bcd8739ae039 as default};
