// @react-spectrum/s2/illustrations/linear/internal/S2_lin_translate_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $eCPcb$jsx} from "react/jsx-runtime";
import {forwardRef as $eCPcb$forwardRef} from "react";





const $a6924ba3fe69e1ee$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eCPcb$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $eCPcb$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20.04 10.643h8.1c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-2.12c-.573 4.116-2.823 7.376-5.503 9.73-.606.534-1.238 1.024-1.884 1.471 1.59 2.342 3.387 4.337 4.784 5.24.58.374.746 1.148.371 1.728-.374.58-1.148.746-1.728.372-1.84-1.189-3.893-3.557-5.563-6.036-1.727.92-3.448 1.537-4.934 1.836-.677.136-1.336-.302-1.472-.979-.137-.677.302-1.336.978-1.472 1.203-.242 2.638-.749 4.103-1.513-1.132-1.98-2.013-4.007-2.303-5.651-.12-.68.334-1.328 1.014-1.448s1.328.334 1.448 1.014c.22 1.253.949 2.977 1.978 4.785.532-.373 1.055-.781 1.558-1.224 2.27-1.995 4.077-4.626 4.623-7.853H9.152c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h8.387V9.125c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25zm5.488 28.72 2.35-5.546h7.467l2.35 5.545c.27.636 1.004.933 1.64.663.635-.269.932-1.002.662-1.638l-7.23-17.057q-.075-.186-.209-.341c-.121-.14-.276-.257-.458-.334q-.242-.101-.488-.1c-.163 0-.329.032-.488.1-.183.077-.337.193-.458.334q-.07.081-.128.177-.047.078-.082.164l-7.23 17.057c-.269.636.028 1.37.664 1.639.635.269 1.369-.028 1.638-.664m6.084-14.354-2.674 6.308h5.347z"
        })
    });
const $a6924ba3fe69e1ee$var$ForwardRef = /*#__PURE__*/ (0, $eCPcb$forwardRef)($a6924ba3fe69e1ee$var$SvgComponent);
var $a6924ba3fe69e1ee$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a6924ba3fe69e1ee$var$ForwardRef);


export {$a6924ba3fe69e1ee$export$2e2bcd8739ae039 as default};
