// @react-spectrum/s2/illustrations/linear/internal/S2_lin_minimize_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $a4XJf$jsx} from "react/jsx-runtime";
import {forwardRef as $a4XJf$forwardRef} from "react";





const $0f39421e5c1997a3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $a4XJf$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $a4XJf$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M134.764 31.828c1.562-1.562 1.562-4.094 0-5.656s-4.095-1.562-5.657 0L96.874 58.405v-19.8c0-2.209-1.79-4-4-4s-4 1.791-4 4v29.376c0 2.209 1.79 4 4 4h29.375c2.209 0 4-1.791 4-4s-1.791-4-4-4h-19.637zm-66.889 56.86c2.21 0 4 1.79 4 4v29.374c0 2.21-1.79 4-4 4s-4-1.79-4-4V102.45l-32.047 32.046c-1.562 1.562-4.094 1.562-5.656 0s-1.562-4.094 0-5.656l32.152-32.153H38.5c-2.21 0-4-1.79-4-4s1.79-4 4-4z"
        })
    });
const $0f39421e5c1997a3$var$ForwardRef = /*#__PURE__*/ (0, $a4XJf$forwardRef)($0f39421e5c1997a3$var$SvgComponent);
var $0f39421e5c1997a3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0f39421e5c1997a3$var$ForwardRef);


export {$0f39421e5c1997a3$export$2e2bcd8739ae039 as default};
