// @react-spectrum/s2/illustrations/linear/internal/S2_lin_education_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $42krD$reactjsxruntime = require("react/jsx-runtime");
var $42krD$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f09540185ccbe1c7$export$2e2bcd8739ae039);




const $f09540185ccbe1c7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $42krD$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $42krD$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.192 24.48c.49-.203 1.04-.203 1.53 0l27.58 11.413c1.648.682 1.648 3.015 0 3.696l-27.58 11.413c-.49.203-1.04.203-1.53 0l-7.428-3.073c-1.02-.423-2.19.062-2.613 1.083-.422 1.02.063 2.19 1.084 2.613l7.427 3.073c1.47.608 3.12.608 4.589 0l19.318-7.994v9.403c0 .961-.257 2.576-1.457 4.364-1.197 1.782-3.408 3.851-7.497 5.604-3.131 1.342-6.932 1.921-10.478 2.053-3.54.132-6.657-.188-8.336-.534-1.082-.223-2.14.474-2.363 1.556-.222 1.081.474 2.139 1.556 2.362 2.037.42 5.479.755 9.292.613 3.805-.141 8.149-.764 11.904-2.374 4.674-2.003 7.546-4.524 9.242-7.05 1.693-2.522 2.137-4.932 2.137-6.594V45.049l4.263-1.764c4.941-2.044 4.941-9.043 0-11.088L50.251 20.784c-1.47-.608-3.12-.608-4.589 0L18.081 32.197c-4.941 2.045-4.941 9.044 0 11.088l4.654 1.926c1.02.422 2.19-.063 2.612-1.083s-.062-2.19-1.083-2.613l-4.654-1.926c-1.647-.681-1.647-3.014 0-3.696zm1.736 13.698c.966-.537 1.313-1.754.777-2.72-.536-.965-1.754-1.313-2.72-.777l-13.594 7.553c-3.175 1.763-5.144 5.11-5.144 8.741v16.53c-1.454 2.866-2.956 6.56-3.214 9.666q-.033.286-.034.581C25 80.65 27.496 83 30.575 83s5.575-2.349 5.575-5.247v-.005q0-.357-.05-.702c-.34-3.238-2.166-7.09-3.853-9.973V50.975c0-2.179 1.181-4.186 3.086-5.245z"
        })
    });
const $f09540185ccbe1c7$var$ForwardRef = /*#__PURE__*/ (0, $42krD$react.forwardRef)($f09540185ccbe1c7$var$SvgComponent);
var $f09540185ccbe1c7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f09540185ccbe1c7$var$ForwardRef);
