// @react-spectrum/s2/illustrations/linear/internal/S2_lin_shapes_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $3J2S3$jsx} from "react/jsx-runtime";
import {forwardRef as $3J2S3$forwardRef} from "react";





const $cfddd91679e90d47$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3J2S3$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3J2S3$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M34.704 19.529c.772-1.328 2.69-1.327 3.46.002l17.501 30.203c.773 1.333-.19 3.002-1.73 3.002H18.888c-1.542 0-2.504-1.671-1.73-3.004zm6.921-2.004c-2.31-3.986-8.065-3.99-10.38-.006L13.7 47.723c-2.324 4 .562 9.013 5.188 9.013h35.047c4.623 0 7.51-5.008 5.191-9.008zM55.798 34.03q.776-.055 1.568-.055c12.087 0 21.885 9.798 21.885 21.884s-9.798 21.884-21.885 21.884c-10.233 0-18.83-7.026-21.222-16.52-.27-1.072-1.357-1.721-2.428-1.451s-1.72 1.356-1.451 2.427c2.828 11.23 12.991 19.544 25.101 19.544 14.296 0 25.885-11.589 25.885-25.884S71.66 29.975 57.366 29.975q-.932 0-1.85.065c-1.102.078-1.932 1.034-1.854 2.136s1.035 1.932 2.136 1.854"
        })
    });
const $cfddd91679e90d47$var$ForwardRef = /*#__PURE__*/ (0, $3J2S3$forwardRef)($cfddd91679e90d47$var$SvgComponent);
var $cfddd91679e90d47$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cfddd91679e90d47$var$ForwardRef);


export {$cfddd91679e90d47$export$2e2bcd8739ae039 as default};
