// @react-spectrum/s2/illustrations/linear/internal/S2_lin_gift_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $kDhPc$jsx} from "react/jsx-runtime";
import {forwardRef as $kDhPc$forwardRef} from "react";





const $befc6768d4457eea$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kDhPc$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kDhPc$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.025 41.3c0-10.21 8.278-18.487 18.488-18.487 5.87 0 11.1 2.735 14.487 7 3.387-4.265 8.618-7 14.488-7 10.21 0 18.487 8.277 18.487 18.487 0 3.894-1.204 7.507-3.26 10.487h11.703c7.927 0 14.354 6.427 14.354 14.354v6.903c0 5.2-2.764 9.753-6.903 12.272v32.597c0 9.834-7.972 17.806-17.806 17.806H48.937c-9.834 0-17.806-7.972-17.806-17.806V85.316c-4.138-2.519-6.902-7.073-6.902-12.272v-6.903c0-7.927 6.426-14.354 14.354-14.354h11.703c-2.057-2.98-3.26-6.593-3.26-10.487M84 59.788h37.418c3.509 0 6.354 2.844 6.354 6.353v6.903c0 3.51-2.845 6.355-6.354 6.355H84zm-8-18.566v10.565H65.404c-5.742-.058-10.379-4.731-10.379-10.487 0-5.792 4.696-10.487 10.488-10.487 5.766 0 10.445 4.653 10.487 10.41m18.597 10.565c5.742-.058 10.378-4.731 10.378-10.487 0-5.792-4.695-10.487-10.487-10.487-5.766 0-10.446 4.653-10.488 10.41v10.564zm-29.167 8H38.583c-3.51 0-6.355 2.845-6.355 6.354v6.903c0 3.51 2.846 6.355 6.355 6.355H76V59.788H65.43M76 87.4H39.131v30.514c0 5.416 4.39 9.806 9.806 9.806H76zm8 40.32v-40.32h36.869v30.514c0 5.416-4.39 9.806-9.806 9.806z"
        })
    });
const $befc6768d4457eea$var$ForwardRef = /*#__PURE__*/ (0, $kDhPc$forwardRef)($befc6768d4457eea$var$SvgComponent);
var $befc6768d4457eea$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($befc6768d4457eea$var$ForwardRef);


export {$befc6768d4457eea$export$2e2bcd8739ae039 as default};
