// @react-spectrum/s2/illustrations/linear/internal/S2_lin_filter_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $kKfCD$reactjsxruntime = require("react/jsx-runtime");
var $kKfCD$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $27a666dc99fc39cc$export$2e2bcd8739ae039);




const $27a666dc99fc39cc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kKfCD$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kKfCD$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M9.47 13.187C7.725 10.868 9.34 7.49 12.28 7.49H35.72c2.94 0 4.556 3.378 2.811 5.697l-.041.052-10.764 12.833V37.21c0 .685-.291 1.34-.805 1.797l-2.674 2.379c-1.562 1.39-3.971.244-3.971-1.798V26.072L9.51 13.24zm1.981-1.525 11.031 13.152c.19.225.293.51.293.803v13.731l2.45-2.18v-11.55c0-.294.104-.579.292-.804l11.031-13.152c.51-.71-.01-1.672-.83-1.672H12.282c-.818 0-1.338.962-.83 1.672m13.807 25.477Z"
        })
    });
const $27a666dc99fc39cc$var$ForwardRef = /*#__PURE__*/ (0, $kKfCD$react.forwardRef)($27a666dc99fc39cc$var$SvgComponent);
var $27a666dc99fc39cc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($27a666dc99fc39cc$var$ForwardRef);
