// @react-spectrum/s2/illustrations/linear/internal/S2_lin_exportTo_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $d22o3$reactjsxruntime = require("react/jsx-runtime");
var $d22o3$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $fe11039141328573$export$2e2bcd8739ae039);




const $fe11039141328573$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $d22o3$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $d22o3$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M134.937 20.969c2.209 0 4 1.79 4 4v25.935c0 2.21-1.791 4-4 4s-4-1.79-4-4V34.595L86.324 79.207c-1.562 1.562-4.094 1.562-5.656 0s-1.562-4.094 0-5.656l44.581-44.582h-16.248c-2.209 0-4-1.791-4-4s1.791-4 4-4zM43.75 38.453c-6.627 0-12 5.373-12 12v65.556c0 6.627 5.373 12 12 12h65.555c6.628 0 12-5.373 12-12V67.552c0-2.209 1.791-4 4-4s4 1.791 4 4v48.457c0 11.045-8.954 20-20 20H43.75c-11.046 0-20-8.955-20-20V50.453c0-11.046 8.954-20 20-20h48.456c2.21 0 4 1.791 4 4s-1.79 4-4 4z"
        })
    });
const $fe11039141328573$var$ForwardRef = /*#__PURE__*/ (0, $d22o3$react.forwardRef)($fe11039141328573$var$SvgComponent);
var $fe11039141328573$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($fe11039141328573$var$ForwardRef);
