// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lockOpen_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $lBhW3$jsx} from "react/jsx-runtime";
import {forwardRef as $lBhW3$forwardRef} from "react";





const $b83df6a72dacf9ca$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lBhW3$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $lBhW3$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 19.5c-14.36 0-26 11.64-26 26v17h-9.636C35.129 62.5 27 69.409 27 78.688v40.624c0 9.279 8.129 16.188 17.364 16.188h71.272c9.235 0 17.364-6.909 17.364-16.188V78.688c0-9.28-8.129-16.188-17.364-16.188H62v-17c0-9.941 8.059-18 18-18s18 8.059 18 18c0 2.21 1.79 4 4 4s4-1.79 4-4c0-14.36-11.64-26-26-26m-22 51H44.364C38.837 70.5 35 74.504 35 78.688v40.624c0 4.183 3.837 8.188 9.364 8.188h71.272c5.527 0 9.364-4.005 9.364-8.188V78.688c0-4.184-3.837-8.188-9.364-8.188zm26 31.424c.603-.347 1.16-.774 1.654-1.271.372-.36.705-.773 1-1.214.293-.426.532-.893.733-1.373.199-.493.36-.986.453-1.506.107-.507.16-1.04.16-1.56s-.053-1.053-.16-1.56c-.094-.507-.254-1.014-.453-1.493-.2-.494-.44-.947-.733-1.387-.295-.44-.628-.854-1-1.214C84.174 87.853 82.107 87 80 87s-4.173.853-5.654 2.346c-.373.36-.707.774-1 1.214-.28.44-.534.893-.733 1.387-.2.48-.36.986-.453 1.492-.107.508-.16 1.04-.16 1.561s.053 1.053.16 1.56c.093.52.253 1.013.453 1.506.2.48.453.947.733 1.373.293.44.627.854 1 1.214.493.497 1.051.924 1.654 1.271v7.576c0 2.209 1.79 4 4 4s4-1.791 4-4z"
        })
    });
const $b83df6a72dacf9ca$var$ForwardRef = /*#__PURE__*/ (0, $lBhW3$forwardRef)($b83df6a72dacf9ca$var$SvgComponent);
var $b83df6a72dacf9ca$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b83df6a72dacf9ca$var$ForwardRef);


export {$b83df6a72dacf9ca$export$2e2bcd8739ae039 as default};
