// @react-spectrum/s2/illustrations/linear/internal/S2_lin_folderClose_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $fQh2q$jsx} from "react/jsx-runtime";
import {forwardRef as $fQh2q$forwardRef} from "react";





const $08a57eeeb3bd5dad$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fQh2q$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fQh2q$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M7.25 11.8c0-.89.692-1.55 1.47-1.55h7.533c.432 0 .852.202 1.137.568l3.055 3.932H7.25zm-2.5 2.95V11.8c0-2.202 1.744-4.05 3.97-4.05h7.533c1.22 0 2.363.571 3.11 1.534l-.97.754.97-.754 4.249 5.466h14.234c2.956 0 5.404 2.351 5.404 5.309V34.94c0 2.958-2.448 5.309-5.404 5.309H10.154c-2.956 0-5.404-2.351-5.404-5.309V14.75m2.5 2.5v17.691c0 1.523 1.274 2.809 2.904 2.809h27.692c1.63 0 2.904-1.286 2.904-2.809V20.06c0-1.523-1.274-2.809-2.904-2.809H7.25"
        })
    });
const $08a57eeeb3bd5dad$var$ForwardRef = /*#__PURE__*/ (0, $fQh2q$forwardRef)($08a57eeeb3bd5dad$var$SvgComponent);
var $08a57eeeb3bd5dad$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($08a57eeeb3bd5dad$var$ForwardRef);


export {$08a57eeeb3bd5dad$export$2e2bcd8739ae039 as default};
