// @react-spectrum/s2/illustrations/linear/internal/S2_lin_shoppingBag_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $jyfEF$reactjsxruntime = require("react/jsx-runtime");
var $jyfEF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e448e1dfd6e36025$export$2e2bcd8739ae039);




const $e448e1dfd6e36025$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jyfEF$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jyfEF$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M71.17 36.83c0-4.877 3.954-8.83 8.83-8.83s8.83 3.953 8.83 8.83v5.406H71.17zm-8 13.406v9.686c0 2.21 1.792 4 4 4 2.21 0 4-1.79 4-4v-9.686h17.66v9.686c0 2.21 1.79 4 4 4s4-1.79 4-4v-9.686h16.328l6.433 64.335c.707 7.065-4.841 13.194-11.94 13.194H52.35c-7.1 0-12.647-6.129-11.94-13.194l6.433-64.335zm0-8V36.83C63.17 27.535 70.706 20 80 20s16.83 7.535 16.83 16.83v5.406h16.328c4.11 0 7.551 3.115 7.96 7.204l6.434 64.335c1.177 11.774-8.069 21.99-19.901 21.99H52.35c-11.833 0-21.078-10.216-19.901-21.99l6.433-64.335c.41-4.09 3.85-7.204 7.96-7.204z"
        })
    });
const $e448e1dfd6e36025$var$ForwardRef = /*#__PURE__*/ (0, $jyfEF$react.forwardRef)($e448e1dfd6e36025$var$SvgComponent);
var $e448e1dfd6e36025$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e448e1dfd6e36025$var$ForwardRef);
