// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lightbulb_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8DDJw$jsx} from "react/jsx-runtime";
import {forwardRef as $8DDJw$forwardRef} from "react";





const $003c49624a5113dd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8DDJw$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8DDJw$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M23.488 5.122c7.244-.512 13.282 5.216 13.282 12.353 0 4.167-2.06 7.85-5.209 10.093-.608.433-.977 1.166-.977 1.964v7.12c0 1.716-1.39 3.106-3.106 3.106h-6.186c-1.715 0-3.106-1.39-3.106-3.105V29.53c0-.788-.389-1.536-1.055-2.018-3.34-2.417-5.43-6.465-5.096-10.975.45-6.083 5.369-10.986 11.453-11.415M34.27 17.475c0-5.697-4.819-10.268-10.606-9.86-4.846.343-8.778 4.262-9.136 9.106-.267 3.605 1.398 6.833 4.068 8.765 1.288.932 2.09 2.42 2.09 4.044v2.898h7.398v-2.896c0-1.555.72-3.07 2.027-4 2.52-1.795 4.159-4.734 4.159-8.057m-6.186 17.453h-7.398v1.725c0 .334.271.605.606.605h6.186c.335 0 .606-.27.606-.605zm-3.71-23.522c0-.69.56-1.25 1.25-1.25 3.424 0 6.199 2.775 6.199 6.199 0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25c0-2.043-1.656-3.7-3.699-3.7-.69 0-1.25-.56-1.25-1.25"
        })
    });
const $003c49624a5113dd$var$ForwardRef = /*#__PURE__*/ (0, $8DDJw$forwardRef)($003c49624a5113dd$var$SvgComponent);
var $003c49624a5113dd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($003c49624a5113dd$var$ForwardRef);


export {$003c49624a5113dd$export$2e2bcd8739ae039 as default};
