// @react-spectrum/s2/illustrations/linear/internal/S2_lin_userGroup_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1BYDr$reactjsxruntime = require("react/jsx-runtime");
var $1BYDr$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e90a55cf1d527749$export$2e2bcd8739ae039);




const $e90a55cf1d527749$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1BYDr$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1BYDr$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M93.84 28c-10.26 0-19 8.936-19 20.5s8.74 20.5 19 20.5 19-8.936 19-20.5-8.741-20.5-19-20.5m-27 20.5c0-15.498 11.854-28.5 27-28.5s27 13.002 27 28.5-11.854 28.5-27 28.5-27-13.002-27-28.5m-3.453-20.308c2.185-.324 3.694-2.358 3.37-4.543s-2.359-3.694-4.544-3.37C48.907 22.252 39 34.306 39 48.5c0 14.243 9.977 26.333 23.352 28.242 2.187.312 4.212-1.208 4.525-3.395.312-2.187-1.208-4.213-3.395-4.525C54.368 67.522 47 59.092 47 48.5c0-10.554 7.317-18.963 16.387-20.308m-9.255 62.54c2.119-.626 3.33-2.85 2.705-4.969-.626-2.119-2.85-3.33-4.969-2.705-17.788 5.249-31.655 20.01-35.592 38.311-1.602 7.446 3.967 14.621 11.72 14.621H33.5c2.21 0 4-1.791 4-4s-1.79-4-4-4h-5.504c-2.526 0-4.457-2.345-3.9-4.939 3.316-15.409 15.03-27.892 30.036-32.32M51.79 123.05C55.955 103.693 73.346 89 93.84 89s37.885 14.693 42.049 34.051c.558 2.594-1.373 4.939-3.899 4.939h-76.3c-2.527 0-4.458-2.345-3.9-4.939M93.84 81c-24.367 0-44.935 17.427-49.87 40.369-1.603 7.446 3.966 14.621 11.72 14.621h76.3c7.753 0 13.322-7.175 11.72-14.621C138.775 98.427 118.206 81 93.84 81"
        })
    });
const $e90a55cf1d527749$var$ForwardRef = /*#__PURE__*/ (0, $1BYDr$react.forwardRef)($e90a55cf1d527749$var$SvgComponent);
var $e90a55cf1d527749$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e90a55cf1d527749$var$ForwardRef);
