// @react-spectrum/s2/illustrations/linear/internal/S2_lin_handshake_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $e8hQ7$reactjsxruntime = require("react/jsx-runtime");
var $e8hQ7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $52e3abe2cc0a1b59$export$2e2bcd8739ae039);




const $52e3abe2cc0a1b59$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e8hQ7$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $e8hQ7$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M23.685 7.517c-2.05-2.05-5.37-2.046-7.425-.003l-.003.003-10.26 10.26c-2.05 2.05-2.046 5.369-.003 7.424l.003.003 3.139 3.14q.078.086.162.17c.772.773 1.778 1.17 2.79 1.19.023 1.009.42 2.01 1.19 2.78.775.775 1.784 1.172 2.799 1.19.02 1.012.418 2.018 1.19 2.79.77.77 1.772 1.167 2.78 1.19.019 1.015.416 2.025 1.191 2.8 1.588 1.588 4.16 1.588 5.748 0l5.28-5.28.018-.018 9.721-9.722c2.05-2.05 2.046-5.369.003-7.425l-.003-.002-10.26-10.26c-2.05-2.05-5.369-2.046-7.425-.003l-.002.003-.202.2zm2.204 2.194 4.386 4.386c1.588 1.588 1.588 4.16 0 5.747-1.589 1.588-4.16 1.588-5.748 0L20.14 15.46l-9.076 9.075c-.577.577-.61 1.49-.098 2.105l.098.098q.036.036.068.073c.585.52 1.462.527 2.054.02q.045-.047.091-.093.049-.049.1-.09l2.082-2.072c.489-.488 1.28-.486 1.767.004.487.489.486 1.28-.004 1.767l-2.178 2.168v.001l-.086.083c-.524.615-.495 1.537.087 2.119.612.611 1.6.611 2.212 0q.046-.046.096-.087l2.083-2.083c.488-.489 1.28-.489 1.768 0 .488.488.488 1.28 0 1.767l-2.17 2.17c-.612.612-.612 1.6 0 2.212s1.6.612 2.212 0l2.15-2.15.007-.006.019-.018c.489-.484 1.276-.482 1.763.005.488.488.488 1.28 0 1.767l-2.18 2.18c-.612.612-.612 1.6 0 2.213.612.611 1.6.611 2.212 0l.271-.272.019-.018 14.73-14.73c1.07-1.07 1.074-2.81-.002-3.894l-.001-.002L29.978 9.515c-1.07-1.07-2.81-1.074-3.894.002l-.002.001zM8.414 24.086c.199-.48.493-.93.884-1.32l9.932-9.932.027-.027c.488-.489 1.28-.489 1.767 0l5.27 5.27c.612.611 1.6.611 2.213 0 .612-.612.612-1.6 0-2.213l-5.27-5.27q-.035-.035-.067-.071-.064-.05-.123-.109l-1.13-1.13c-1.07-1.07-2.81-1.074-3.894.002l-.001.001L7.765 19.544c-1.07 1.07-1.075 2.81.002 3.894v.001z"
        })
    });
const $52e3abe2cc0a1b59$var$ForwardRef = /*#__PURE__*/ (0, $e8hQ7$react.forwardRef)($52e3abe2cc0a1b59$var$SvgComponent);
var $52e3abe2cc0a1b59$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($52e3abe2cc0a1b59$var$ForwardRef);
