// @react-spectrum/s2/illustrations/linear/internal/S2_lin_thumbsUp_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $4jWLX$reactjsxruntime = require("react/jsx-runtime");
var $4jWLX$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d2bf0dbac331bf4d$export$2e2bcd8739ae039);




const $d2bf0dbac331bf4d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4jWLX$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4jWLX$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M19.152 38.954c1.481.552 3.05.835 4.63.835h7.381q.044 0 .087-.003h1.909c.882 0 1.852-.24 2.634-.84.823-.631 1.354-1.603 1.354-2.836 0-.329-.065-.896-.317-1.487.97-.488 1.702-1.4 1.931-2.521.194-.944.005-1.863-.45-2.613.824-.627 1.374-1.6 1.43-2.709.041-.796-.176-1.543-.577-2.161.758-.629 1.258-1.56 1.312-2.615.106-2.08-1.551-3.825-3.634-3.825h-7.591l.451-1.45c.538-1.728.883-3.51 1.03-5.313l.08-1.004c.167-2.057-1.458-3.818-3.522-3.818-1.361 0-2.602.782-3.189 2.011l-2.697 5.653q-.34.713-.781 1.369l-.472.701c-1.393 2.072-3.726 3.315-6.223 3.315h-3.283c-1.795 0-3.25 1.455-3.25 3.25v10.954c0 1.795 1.455 3.25 3.25 3.25h1.587c1.282 0 2.554.23 3.756.677zm4.63-1.665h4.135q.042-.003.086-.003h5.156c.464 0 .862-.131 1.113-.324.21-.16.375-.401.375-.852 0-.122-.034-.37-.155-.589-.1-.18-.27-.371-.655-.474q-.06-.016-.116-.037h-.052c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h1.528c.54 0 1.007-.38 1.115-.91.145-.706-.394-1.367-1.115-1.367h-.549c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h.55q.134 0 .266.01.078-.01.159-.01h.485c.606 0 1.106-.476 1.137-1.08.033-.644-.474-1.185-1.115-1.197h-.752c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h1.464c.606 0 1.106-.475 1.137-1.08.033-.651-.485-1.197-1.137-1.197h-8.015c-1.475 0-2.527-1.43-2.089-2.838l.577-1.856c.483-1.551.793-3.151.924-4.77l.081-1.004c.05-.602-.426-1.117-1.03-1.117-.398 0-.76.229-.932.588l-2.697 5.652q-.42.88-.963 1.688l-.472.701c-1.683 2.503-4.394 4.098-7.364 4.376V34.86q1.02.207 2 .572l3.164 1.18c1.201.448 2.473.677 3.756.677m-11.42-2.692V22.143h-1.717c-.415 0-.75.336-.75.75v10.954c0 .414.335.75.75.75h1.717"
        })
    });
const $d2bf0dbac331bf4d$var$ForwardRef = /*#__PURE__*/ (0, $4jWLX$react.forwardRef)($d2bf0dbac331bf4d$var$SvgComponent);
var $d2bf0dbac331bf4d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d2bf0dbac331bf4d$var$ForwardRef);
