// @react-spectrum/s2/illustrations/linear/internal/S2_lin_chartAreaStack_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1nNxh$reactjsxruntime = require("react/jsx-runtime");
var $1nNxh$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $df78cb01f816e8cd$export$2e2bcd8739ae039);




const $df78cb01f816e8cd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1nNxh$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1nNxh$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M79.813 24.483c0-4.134-5.507-5.552-7.503-1.931l-8.307 15.064-6.933-8.482c-1.718-2.1-4.983-1.919-6.456.36l-9.692 14.99-7.54-8.715c-1.49-1.721-4.113-1.855-5.77-.292l-9.48 8.943c-1.203 1.133-1.884 2.712-1.884 4.364v24.994c0 3.313 2.686 6 6 6h51.565c3.314 0 6-2.687 6-6zm-4 30.487V24.483l-8.307 15.065c-1.367 2.48-4.807 2.792-6.6.6l-6.933-8.483-9.692 14.99c-1.448 2.239-4.64 2.462-6.384.445l-7.54-8.714-9.481 8.943c-.4.378-.628.905-.628 1.455v14.529l10.335-6.932c1.447-.97 3.356-.892 4.72.193L44.2 63.66l10.764-12.693c1.52-1.793 4.252-1.895 5.901-.22l7.583 7.705zm0 4.424-5.655 2.674c-1.538.727-3.367.402-4.56-.81l-7.584-7.704-10.763 12.693c-1.398 1.648-3.852 1.888-5.542.542l-8.898-7.086L20.7 67.826q-.217.144-.452.227v5.725c0 1.104.895 2 2 2h51.565c1.105 0 2-.896 2-2z"
        })
    });
const $df78cb01f816e8cd$var$ForwardRef = /*#__PURE__*/ (0, $1nNxh$react.forwardRef)($df78cb01f816e8cd$var$SvgComponent);
var $df78cb01f816e8cd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($df78cb01f816e8cd$var$ForwardRef);
