// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noLibraries_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $cVEcX$reactjsxruntime = require("react/jsx-runtime");
var $cVEcX$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $fd44f029651dc3d5$export$2e2bcd8739ae039);




const $fd44f029651dc3d5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cVEcX$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cVEcX$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M37.214 32.867c-6.991 0-12.667 5.676-12.667 12.667v53.25c0 2.209-1.791 4-4 4s-4-1.791-4-4v-53.25c0-11.41 9.257-20.667 20.667-20.667h53.25c2.209 0 4 1.791 4 4s-1.791 4-4 4zm1 23.017c0-5.154 4.179-9.333 9.333-9.333h36.02V82.9c0 1.333 1.482 2.133 2.582 1.383l8.117-5.417c.567-.366 1.283-.366 1.85 0l8.117 5.417c1.1.733 2.583-.05 2.583-1.383V46.55h.648c5.155 0 9.333 4.179 9.333 9.333v39.117c0 2.209 1.791 4 4 4s4-1.791 4-4V55.884c0-9.573-7.76-17.333-17.333-17.333H47.547c-9.573 0-17.333 7.76-17.333 17.333v59.917c0 9.573 7.76 17.333 17.333 17.333h39.118c2.21 0 4-1.791 4-4s-1.79-4-4-4H47.547c-5.154 0-9.333-4.179-9.333-9.333zm82.119 97.282c13.577 0 24.584-11.006 24.584-24.583S133.91 104 120.333 104 95.75 115.006 95.75 128.583s11.006 24.583 24.583 24.583m4.417-35.833c0-2.209-1.791-4-4-4s-4 1.791-4 4V125h-7.667c-2.209 0-4 1.791-4 4s1.791 4 4 4h7.667v7.666c0 2.209 1.791 4 4 4s4-1.791 4-4V133h7.666c2.209 0 4-1.791 4-4s-1.791-4-4-4h-7.666z"
        })
    });
const $fd44f029651dc3d5$var$ForwardRef = /*#__PURE__*/ (0, $cVEcX$react.forwardRef)($fd44f029651dc3d5$var$SvgComponent);
var $fd44f029651dc3d5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($fd44f029651dc3d5$var$ForwardRef);
