// @react-spectrum/s2/illustrations/linear/internal/S2_lin_commentText_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $W7LQf$jsx} from "react/jsx-runtime";
import {forwardRef as $W7LQf$forwardRef} from "react";





const $63902a5e6a115b87$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $W7LQf$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $W7LQf$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M34.4 24C24.263 24 16 32.175 16 42.311V93.1c0 10.137 8.262 18.312 18.4 18.312h8.905c0 6.22-.043 13.744-.087 19.747-.056 7.596 8.874 11.591 14.555 6.696l30.693-26.443H125.6c10.139 0 18.401-8.175 18.401-18.312V42.311C144 32.175 135.738 24 125.599 24zM24 42.311C24 36.64 28.633 32 34.4 32H125.6C131.367 32 136 36.64 136 42.311V93.1c0 5.672-4.633 10.312-10.401 10.312H85.495l-1.125.969-31.819 27.414c-.166.143-.306.187-.421.201-.134.016-.296-.005-.457-.079s-.277-.179-.345-.282c-.058-.086-.112-.208-.11-.417.054-7.36.107-17.045.079-23.823l-.017-3.983H34.401c-5.768 0-10.401-4.64-10.401-10.312zM50.042 53.72c-2.209 0-4 1.79-4 4s1.791 4 4 4h59.915c2.209 0 4-1.791 4-4s-1.791-4-4-4zm0 19.971c-2.209 0-4 1.79-4 4s1.791 4 4 4h59.915c2.209 0 4-1.79 4-4s-1.791-4-4-4z"
        })
    });
const $63902a5e6a115b87$var$ForwardRef = /*#__PURE__*/ (0, $W7LQf$forwardRef)($63902a5e6a115b87$var$SvgComponent);
var $63902a5e6a115b87$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($63902a5e6a115b87$var$ForwardRef);


export {$63902a5e6a115b87$export$2e2bcd8739ae039 as default};
