// @react-spectrum/s2/illustrations/linear/internal/S2_lin_dropToUpload_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2IkPP$reactjsxruntime = require("react/jsx-runtime");
var $2IkPP$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c8b6eef40c334bb0$export$2e2bcd8739ae039);




const $c8b6eef40c334bb0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2IkPP$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2IkPP$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M33.455 15.74c-4.107 0-7.455 3.313-7.455 7.423v36.154c0 4.11 3.348 7.423 7.455 7.423h29.09c4.107 0 7.455-3.312 7.455-7.423V38.063c0-1.972-.788-3.861-2.187-5.253l-14.987-14.9c-1.399-1.39-3.294-2.17-5.267-2.17zM30 23.163c0-1.88 1.536-3.423 3.455-3.423h14.104c.92 0 1.8.364 2.447 1.007l14.986 14.9C65.64 36.29 66 37.16 66 38.063v21.254c0 1.88-1.536 3.423-3.455 3.423h-29.09c-1.92 0-3.455-1.543-3.455-3.423zm9.766 22.029c-.855-.7-2.115-.573-2.814.282-.7.856-.573 2.116.282 2.815l8.219 6.719c.366.446.92.73 1.543.732h.035c.464.007.93-.146 1.308-.46l8.438-7c.85-.706.968-1.967.262-2.817-.705-.85-1.966-.967-2.816-.262l-5.214 4.325.035-15.781c.002-1.105-.891-2.002-1.996-2.005-1.105-.002-2.002.891-2.004 1.996l-.035 15.743zM27 75.74c-1.105 0-2 .896-2 2s.895 2 2 2h7.5c1.105 0 2-.895 2-2s-.895-2-2-2zm-15-9.5c-1.105 0-2-.895-2-2v-7.5c0-1.104.895-2 2-2s2 .896 2 2v7.5c0 1.105-.895 2-2 2m70-1.5c0 1.105.895 2 2 2s2-.895 2-2v-7.5c0-1.104-.895-2-2-2s-2 .896-2 2zm-61-22c0-1.104-.895-2-2-2h-3.5c-3.038 0-5.5 2.463-5.5 5.5v3.5c0 1.105.895 2 2 2s2-.895 2-2v-3.5c0-.828.672-1.5 1.5-1.5H19c1.105 0 2-.895 2-2m56-2c-1.105 0-2 .896-2 2s.895 2 2 2h3.5c.828 0 1.5.672 1.5 1.5v3.5c0 1.105.895 2 2 2s2-.895 2-2v-3.5c0-3.037-2.462-5.5-5.5-5.5zm-65 28c-1.105 0-2 .896-2 2v3.5c0 3.038 2.462 5.5 5.5 5.5H19c1.105 0 2-.895 2-2s-.895-2-2-2h-3.5c-.828 0-1.5-.671-1.5-1.5v-3.5c0-1.104-.895-2-2-2m74 2c0-1.104-.895-2-2-2s-2 .896-2 2v3.5c0 .829-.672 1.5-1.5 1.5H77c-1.105 0-2 .896-2 2s.895 2 2 2h3.5c3.038 0 5.5-2.462 5.5-5.5zm-44 7c0-1.104.895-2 2-2h7.5c1.105 0 2 .896 2 2s-.895 2-2 2H44c-1.105 0-2-.895-2-2m19-2c-1.105 0-2 .896-2 2s.895 2 2 2h7.5c1.105 0 2-.895 2-2s-.895-2-2-2z"
        })
    });
const $c8b6eef40c334bb0$var$ForwardRef = /*#__PURE__*/ (0, $2IkPP$react.forwardRef)($c8b6eef40c334bb0$var$SvgComponent);
var $c8b6eef40c334bb0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c8b6eef40c334bb0$var$ForwardRef);
