// @react-spectrum/s2/illustrations/linear/internal/S2_lin_graphBarChart_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $ayHb7$reactjsxruntime = require("react/jsx-runtime");
var $ayHb7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b27c7324f930fcbc$export$2e2bcd8739ae039);




const $b27c7324f930fcbc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ayHb7$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $ayHb7$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24.21 24c0-2.21-1.79-4-4-4-2.208 0-4 1.79-4 4v116c0 2.209 1.792 4 4 4h115.791c2.209 0 4-1.791 4-4s-1.791-4-4-4H24.211zm15.791 55.05c0-2.21 1.79-4 4-4h4c2.21 0 4 1.79 4 4V116c0 2.209-1.79 4-4 4h-4c-2.21 0-4-1.791-4-4zm4-12c-6.627 0-12 5.373-12 12V116c0 6.627 5.373 12 12 12h4c6.627 0 12-5.373 12-12V79.05c0-6.627-5.373-12-12-12zm72 32.95c-2.209 0-4 1.791-4 4v12c0 2.209 1.791 4 4 4h4c2.209 0 4-1.791 4-4v-12c0-2.209-1.791-4-4-4zm-12 4c0-6.627 5.373-12 12-12h4c6.627 0 12 5.373 12 12v12c0 6.627-5.373 12-12 12h-4c-6.627 0-12-5.373-12-12zm-28-56c0-2.21 1.79-4 4-4h4c2.21 0 4 1.79 4 4v68c0 2.209-1.79 4-4 4h-4c-2.21 0-4-1.791-4-4zm4-12c-6.627 0-12 5.373-12 12v68c0 6.627 5.373 12 12 12h4c6.627 0 12-5.373 12-12V48c0-6.627-5.373-12-12-12z"
        })
    });
const $b27c7324f930fcbc$var$ForwardRef = /*#__PURE__*/ (0, $ayHb7$react.forwardRef)($b27c7324f930fcbc$var$SvgComponent);
var $b27c7324f930fcbc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b27c7324f930fcbc$var$ForwardRef);
