// @react-spectrum/s2/illustrations/linear/internal/S2_lin_error_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $grKcd$reactjsxruntime = require("react/jsx-runtime");
var $grKcd$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $bdadf9a98380ae0b$export$2e2bcd8739ae039);




const $bdadf9a98380ae0b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $grKcd$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $grKcd$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M24 14.99c.69 0 1.25.56 1.25 1.25v11c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-11c0-.69.56-1.25 1.25-1.25M25.75 32.74c0 .966-.784 1.75-1.75 1.75-.967 0-1.75-.784-1.75-1.75 0-.967.783-1.75 1.75-1.75.966 0 1.75.784 1.75 1.75"
            }),
            /*#__PURE__*/ (0, $grKcd$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "m28.542 9.015 13.91 24.1c2.02 3.501-.498 7.875-4.543 7.875H10.09c-4.044 0-6.562-4.373-4.542-7.875L19.38 9.147q.028-.067.066-.132c2.02-3.503 7.074-3.504 9.095 0m-6.864 1.141c1.099-1.727 3.66-1.69 4.699.108l13.91 24.1c1.06 1.84-.262 4.126-2.378 4.126H10.09c-2.115 0-3.437-2.286-2.377-4.125l13.91-24.1q.031-.054.056-.109"
            })
        ]
    });
const $bdadf9a98380ae0b$var$ForwardRef = /*#__PURE__*/ (0, $grKcd$react.forwardRef)($bdadf9a98380ae0b$var$SvgComponent);
var $bdadf9a98380ae0b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($bdadf9a98380ae0b$var$ForwardRef);
