// @react-spectrum/s2/illustrations/linear/internal/S2_lin_segmentation_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $2k7Qh$jsx} from "react/jsx-runtime";
import {forwardRef as $2k7Qh$forwardRef} from "react";





const $fb056fac4b8d0348$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2k7Qh$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2k7Qh$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M32.64 24.25c0-4.418 3.582-8 8-8h31.123c4.418 0 8 3.582 8 8v10.603c0 4.418-3.582 8-8 8H40.64c-4.419 0-8-3.582-8-8v-3.329h-8.405c-2.209 0-4 1.791-4 4v24.924c0 2.21 1.791 4 4 4h8.405v-3.3c0-4.419 3.581-8 8-8h31.122c4.418 0 8 3.581 8 8V71.75c0 4.418-3.582 8-8 8H40.64c-4.419 0-8-3.582-8-8v-3.303h-8.405c-4.418 0-8-3.582-8-8V35.524c0-4.418 3.582-8 8-8h8.405zm8-4c-2.209 0-4 1.79-4 4v10.603c0 2.21 1.791 4 4 4h31.123c2.209 0 4-1.79 4-4V24.25c0-2.21-1.791-4-4-4zm-4 40.898c0-2.21 1.791-4 4-4h31.123c2.209 0 4 1.79 4 4V71.75c0 2.209-1.791 4-4 4H40.64c-2.21 0-4-1.791-4-4z"
        })
    });
const $fb056fac4b8d0348$var$ForwardRef = /*#__PURE__*/ (0, $2k7Qh$forwardRef)($fb056fac4b8d0348$var$SvgComponent);
var $fb056fac4b8d0348$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($fb056fac4b8d0348$var$ForwardRef);


export {$fb056fac4b8d0348$export$2e2bcd8739ae039 as default};
