// @react-spectrum/s2/illustrations/linear/internal/S2_lin_education_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $jFB1P$jsx} from "react/jsx-runtime";
import {forwardRef as $jFB1P$forwardRef} from "react";





const $0fbc6852d34be7a3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jFB1P$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jFB1P$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M78.47 43.667c.979-.405 2.079-.405 3.058 0l45.898 18.992c3.294 1.363 3.294 6.03 0 7.392L81.528 89.043c-.979.406-2.08.406-3.058 0l-10.177-4.21c-2.042-.845-4.381.125-5.226 2.166s.125 4.38 2.167 5.225l10.177 4.212c2.938 1.215 6.238 1.215 9.176 0l32.886-13.609v15.786c0 1.604-.429 4.343-2.478 7.394-2.04 3.039-5.844 6.619-12.955 9.667-5.423 2.324-12.034 3.337-18.23 3.567-6.183.23-11.619-.33-14.526-.928-2.163-.446-4.279.947-4.724 3.111s.947 4.279 3.111 4.724c3.622.746 9.707 1.338 16.436 1.088 6.717-.25 14.412-1.349 21.085-4.209 8.28-3.549 13.405-8.033 16.445-12.561 3.033-4.517 3.836-8.848 3.836-11.853V79.517l5.012-2.073c9.882-4.09 9.882-18.088 0-22.177L84.587 36.275c-2.938-1.216-6.238-1.216-9.176 0L29.513 55.267c-9.882 4.09-9.882 18.087 0 22.176l5.304 2.195c2.041.845 4.38-.125 5.225-2.167.845-2.04-.125-4.38-2.166-5.225l-5.304-2.195c-3.294-1.363-3.294-6.029 0-7.392zm3.472 23.882c1.93-1.073 2.626-3.508 1.554-5.44-1.073-1.93-3.508-2.626-5.44-1.553L51.066 75.55c-3.81 2.116-6.173 6.132-6.173 10.49v33.568c-2.395 4.882-4.737 10.903-5.159 16.013q-.057.498-.058 1.011c0 5.09 4.384 9.217 9.793 9.217s9.792-4.127 9.792-9.217q0-.638-.09-1.252c-.566-5.375-3.461-11.712-6.278-16.663V86.04c0-1.452.787-2.79 2.057-3.496z"
        })
    });
const $0fbc6852d34be7a3$var$ForwardRef = /*#__PURE__*/ (0, $jFB1P$forwardRef)($0fbc6852d34be7a3$var$SvgComponent);
var $0fbc6852d34be7a3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0fbc6852d34be7a3$var$ForwardRef);


export {$0fbc6852d34be7a3$export$2e2bcd8739ae039 as default};
