// @react-spectrum/s2/illustrations/linear/internal/S2_lin_userGroup_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1zF7z$jsx} from "react/jsx-runtime";
import {forwardRef as $1zF7z$forwardRef} from "react";





const $a69ec9145c53dc41$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1zF7z$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1zF7z$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M56.31 16.83c-6.23 0-11.49 5.417-11.49 12.37s5.26 12.37 11.49 12.37c6.228 0 11.49-5.418 11.49-12.37 0-6.953-5.262-12.37-11.49-12.37M40.82 29.2c0-8.92 6.817-16.37 15.49-16.37 8.671 0 15.49 7.45 15.49 16.37S64.98 45.57 56.31 45.57c-8.673 0-15.49-7.45-15.49-16.37m-2.423-12.253c1.092-.162 1.847-1.18 1.685-2.272-.162-1.093-1.18-1.847-2.272-1.685-7.62 1.13-13.32 8.042-13.32 16.21 0 8.196 5.739 15.128 13.4 16.222 1.093.156 2.106-.604 2.262-1.698s-.604-2.106-1.698-2.262c-5.53-.79-9.964-5.892-9.964-12.262 0-6.348 4.404-11.437 9.907-12.253m-5.476 36.69c1.06-.312 1.665-1.424 1.352-2.484-.312-1.059-1.424-1.664-2.484-1.352-10.313 3.043-18.353 11.602-20.635 22.21-.894 4.158 2.215 8.157 6.536 8.157h3.228c1.105 0 2-.895 2-2s-.895-2-2-2H17.69c-1.708 0-2.998-1.584-2.626-3.315C17.037 63.69 24 56.27 32.922 53.638m-1.613 19.216c2.476-11.51 12.813-20.245 25-20.245 12.188 0 22.526 8.736 25.002 20.245l1.955-.42-1.955.42c.372 1.73-.917 3.315-2.626 3.315H33.933c-1.708 0-2.998-1.584-2.625-3.315m25-24.245c-14.123 0-26.05 10.103-28.91 23.403-.895 4.158 2.214 8.157 6.535 8.157h44.752c4.32 0 7.43-3.999 6.535-8.157-2.86-13.3-14.787-23.403-28.911-23.403"
        })
    });
const $a69ec9145c53dc41$var$ForwardRef = /*#__PURE__*/ (0, $1zF7z$forwardRef)($a69ec9145c53dc41$var$SvgComponent);
var $a69ec9145c53dc41$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a69ec9145c53dc41$var$ForwardRef);


export {$a69ec9145c53dc41$export$2e2bcd8739ae039 as default};
