// @react-spectrum/s2/illustrations/linear/internal/S2_lin_playTriangle_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $j2duC$jsx} from "react/jsx-runtime";
import {forwardRef as $j2duC$forwardRef} from "react";





const $f02843fd7cb8c371$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $j2duC$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $j2duC$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M10.775 13.31c.004-3.297 3.519-5.49 6.418-3.76l17.943 10.708c2.823 1.685 2.818 5.856-.01 7.534L17.16 38.456c-2.903 1.723-6.413-.479-6.409-3.775zm5.137-1.613c-1.12-.668-2.636.124-2.637 1.615l-.025 21.372c-.002 1.492 1.512 2.288 2.633 1.622l17.968-10.664c1.197-.71 1.2-2.523.003-3.237z"
        })
    });
const $f02843fd7cb8c371$var$ForwardRef = /*#__PURE__*/ (0, $j2duC$forwardRef)($f02843fd7cb8c371$var$SvgComponent);
var $f02843fd7cb8c371$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f02843fd7cb8c371$var$ForwardRef);


export {$f02843fd7cb8c371$export$2e2bcd8739ae039 as default};
