// @react-spectrum/s2/illustrations/linear/internal/S2_lin_remix_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3WHl7$reactjsxruntime = require("react/jsx-runtime");
var $3WHl7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7edec0b9ce330922$export$2e2bcd8739ae039);




const $7edec0b9ce330922$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3WHl7$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3WHl7$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m129.936 67.666 18.586-18.586c1.562-1.562 1.562-4.094 0-5.657l-18.586-18.585c-1.562-1.562-4.095-1.562-5.657 0s-1.562 4.094 0 5.656l11.402 11.403h-7.178c-8.729 0-16.783 2.94-23.21 7.882q-.147.113-.279.236-.064.046-.127.095c-4.735 3.718-9.15 9.122-12.607 14.075-3.471 4.976-6.188 9.807-7.457 12.609-.994 2.193-3.455 7.406-6.711 12.882-2.803 4.713-6.005 9.29-9.182 12.376q-.145.11-.282.234c-5.349 4.86-12.443 7.817-20.238 7.817-16.626 0-30.103-13.477-30.103-30.103S31.784 49.897 48.41 49.897c10.167 0 19.158 5.036 24.614 12.767l.007.01.007.01 2.082 2.911c1.285 1.797 3.783 2.213 5.58.928s2.212-3.784.927-5.58l-2.074-2.902c-6.892-9.759-18.271-16.144-31.143-16.144-21.044 0-38.103 17.059-38.103 38.103s17.06 38.103 38.103 38.103c8.72 0 16.764-2.934 23.185-7.863q.273-.153.527-.353c4.965-3.915 9.466-10.406 12.866-16.122 3.465-5.828 6.063-11.334 7.121-13.67 1.023-2.256 3.471-6.658 6.732-11.332 3.276-4.696 7.161-9.357 10.986-12.36q.13-.102.248-.211l.095-.071c5.077-3.905 11.429-6.224 18.333-6.224h7.889l-12.113 12.112c-1.562 1.563-1.562 4.095 0 5.657s4.095 1.562 5.657 0m8.019 50.437c2.21 0 4-1.791 4-4s-1.79-4-4-4h-9.452c-9.961 0-18.039-5.729-25.113-13.446l-2.717-3.283c-1.408-1.702-3.93-1.94-5.631-.531-1.703 1.408-1.94 3.93-.532 5.632l2.78 3.36.063.076.067.073c7.629 8.342 17.72 16.119 31.083 16.119z"
        })
    });
const $7edec0b9ce330922$var$ForwardRef = /*#__PURE__*/ (0, $3WHl7$react.forwardRef)($7edec0b9ce330922$var$SvgComponent);
var $7edec0b9ce330922$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7edec0b9ce330922$var$ForwardRef);
