// @react-spectrum/s2/illustrations/linear/internal/S2_lin_translate_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $c9oTA$jsx} from "react/jsx-runtime";
import {forwardRef as $c9oTA$forwardRef} from "react";





const $6983bd2e90b9323d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $c9oTA$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $c9oTA$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M39.58 21.785h16.702c1.104 0 2 .896 2 2s-.896 2-2 2h-4.679c-1.026 8.29-5.509 14.853-10.896 19.588-1.326 1.166-2.714 2.228-4.13 3.184 3.257 4.877 7.026 9.118 9.987 11.03.928.6 1.194 1.838.595 2.766-.6.928-1.837 1.194-2.765.594-3.673-2.372-7.833-7.196-11.2-12.253l-.036-.054c-3.545 1.939-7.092 3.227-10.13 3.838-1.082.218-2.136-.483-2.354-1.566-.218-1.082.483-2.137 1.566-2.355 2.579-.52 5.67-1.63 8.799-3.318-2.373-4.07-4.219-8.252-4.807-11.588-.191-1.088.535-2.125 1.623-2.317 1.087-.191 2.125.535 2.317 1.623.474 2.692 2.068 6.389 4.286 10.204 1.234-.84 2.447-1.771 3.608-2.792 4.74-4.166 8.51-9.723 9.501-16.584h-29.26c-1.105 0-2-.895-2-2s.895-2 2-2H35.58V18.25c0-1.105.896-2 2-2s2 .895 2 2zM50.598 78.53l4.83-11.396h15.267q.162 0 .318-.025l4.84 11.42c.431 1.018 1.605 1.493 2.622 1.062s1.492-1.605 1.061-2.622L65.074 42.848q-.045-.11-.103-.213-.112-.2-.261-.365c-.19-.21-.427-.382-.704-.5-.255-.108-.52-.159-.78-.159-.261 0-.526.051-.78.16-.278.117-.515.29-.704.5q-.15.163-.262.364-.057.104-.103.213l-14.462 34.12c-.43 1.018.044 2.192 1.061 2.623 1.017.43 2.191-.044 2.622-1.061m12.628-29.793-6.102 14.397h12.204z"
        })
    });
const $6983bd2e90b9323d$var$ForwardRef = /*#__PURE__*/ (0, $c9oTA$forwardRef)($6983bd2e90b9323d$var$SvgComponent);
var $6983bd2e90b9323d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6983bd2e90b9323d$var$ForwardRef);


export {$6983bd2e90b9323d$export$2e2bcd8739ae039 as default};
