// @react-spectrum/s2/illustrations/linear/internal/S2_lin_segmentation_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $k2YsX$reactjsxruntime = require("react/jsx-runtime");
var $k2YsX$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d3a7301af21e11ff$export$2e2bcd8739ae039);




const $d3a7301af21e11ff$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $k2YsX$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $k2YsX$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.071 12.125c0-2.347 1.903-4.25 4.25-4.25h15.561c2.348 0 4.25 1.903 4.25 4.25v5.302c0 2.347-1.902 4.25-4.25 4.25h-15.56c-2.348 0-4.25-1.903-4.25-4.25v-1.414h-3.953c-.966 0-1.75.783-1.75 1.75v12.462c0 .966.784 1.75 1.75 1.75h3.952v-1.4c0-2.348 1.903-4.25 4.25-4.25h15.561c2.348 0 4.25 1.902 4.25 4.25v5.3c0 2.348-1.902 4.25-4.25 4.25h-15.56c-2.348 0-4.25-1.902-4.25-4.25v-1.4h-3.953c-2.347 0-4.25-1.903-4.25-4.25V17.763c0-2.348 1.903-4.25 4.25-4.25h3.952zm4.25-1.75c-.966 0-1.75.784-1.75 1.75v5.302c0 .966.784 1.75 1.75 1.75h15.561c.967 0 1.75-.784 1.75-1.75v-5.302c0-.966-.783-1.75-1.75-1.75zm-1.75 20.2c0-.967.784-1.75 1.75-1.75h15.561c.967 0 1.75.783 1.75 1.75v5.3c0 .967-.783 1.75-1.75 1.75h-15.56c-.967 0-1.75-.783-1.75-1.75z"
        })
    });
const $d3a7301af21e11ff$var$ForwardRef = /*#__PURE__*/ (0, $k2YsX$react.forwardRef)($d3a7301af21e11ff$var$SvgComponent);
var $d3a7301af21e11ff$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d3a7301af21e11ff$var$ForwardRef);
