// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cloudStateDisconnected_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $8v72I$jsxs, jsx as $8v72I$jsx} from "react/jsx-runtime";
import {forwardRef as $8v72I$forwardRef} from "react";





const $42bd3f1809356c77$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8v72I$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8v72I$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M50.97 17.7c-8.028 0-14.861 5.031-17.856 12.148-1.17-.312-2.397-.478-3.664-.478-8.236 0-14.76 6.95-14.76 15.33q.001.869.095 1.705C10.14 48.484 6.95 53.262 6.95 58.77c0 7.284 5.62 13.34 12.767 13.46H54.6c1.105 0 2-.895 2-2s-.895-2-2-2H19.768c-4.806-.09-8.818-4.21-8.818-9.46 0-4.424 2.854-8.06 6.63-9.137 1.02-.291 1.636-1.33 1.4-2.365-.188-.832-.29-1.684-.29-2.568 0-6.34 4.897-11.33 10.76-11.33 1.449 0 2.819.3 4.075.845.522.226 1.116.22 1.633-.019.516-.238.907-.685 1.074-1.229 2.024-6.592 7.9-11.267 14.738-11.267 8.474 0 15.49 7.201 15.49 16.27v.39c0 1.059.826 1.934 1.883 1.996 3.514.207 6.67 1.851 8.954 4.414.734.825 1.999.898 2.824.163.824-.734.897-1.999.163-2.823-2.544-2.856-5.978-4.87-9.87-5.536C69.727 26.102 61.359 17.7 50.97 17.7"
            }),
            /*#__PURE__*/ (0, $8v72I$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M57.6 61.57c0-8.13 6.59-14.72 14.72-14.72s14.72 6.59 14.72 14.72-6.59 14.72-14.72 14.72S57.6 69.7 57.6 61.57m21 6.28c.78-.78.78-2.05 0-2.83h.02l-3.45-3.45 3.45-3.45c.78-.78.78-2.05 0-2.83s-2.05-.78-2.83 0l-3.45 3.45-3.45-3.45c-.78-.78-2.05-.78-2.83 0s-.78 2.05 0 2.83l3.45 3.45-3.45 3.45c-.78.78-.78 2.05 0 2.83.39.39.9.59 1.41.59s1.02-.2 1.41-.59l3.45-3.45 3.45 3.45c.39.39.9.59 1.41.59s1.02-.2 1.41-.59"
            })
        ]
    });
const $42bd3f1809356c77$var$ForwardRef = /*#__PURE__*/ (0, $8v72I$forwardRef)($42bd3f1809356c77$var$SvgComponent);
var $42bd3f1809356c77$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($42bd3f1809356c77$var$ForwardRef);


export {$42bd3f1809356c77$export$2e2bcd8739ae039 as default};
