// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cloudStateDisconnected_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $gkAc3$jsxs, jsx as $gkAc3$jsx} from "react/jsx-runtime";
import {forwardRef as $gkAc3$forwardRef} from "react";





const $e56d6087ec1e1c3e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gkAc3$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gkAc3$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M85.15 28.9c-13.458 0-24.924 8.311-30.119 20.106-1.831-.441-3.739-.676-5.701-.676-14.112 0-25.25 11.897-25.25 26.19q.001 1.239.114 2.428c-7.73 3.67-13.014 11.735-13.014 20.992 0 12.318 9.397 22.627 21.476 23.077q.253.033.514.033H87.8c2.209 0 4-1.791 4-4s-1.791-4-4-4H33.57q-.164-.016-.333-.019C25.61 112.9 19.18 106.352 19.18 97.94c0-7.061 4.571-12.849 10.561-14.565 2.027-.58 3.255-2.634 2.807-4.695-.298-1.373-.468-2.756-.468-4.16 0-10.207 7.881-18.19 17.25-18.19 2.305 0 4.496.48 6.513 1.352 1.044.451 2.231.436 3.264-.04 1.032-.477 1.813-1.371 2.147-2.458C64.542 44.47 74.083 36.9 85.15 36.9c13.71 0 25.119 11.654 25.12 26.41l-.003.06-.003.074-.001.018c-.004.097-.013.287-.013.488 0 2.117 1.649 3.868 3.763 3.993 4.227.25 8.129 1.785 11.365 4.263 1.754 1.343 4.265 1.01 5.608-.744s1.009-4.265-.745-5.608c-3.505-2.684-7.615-4.585-12.088-5.44C116.742 42.897 102.646 28.9 85.15 28.9"
            }),
            /*#__PURE__*/ (0, $gkAc3$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M95.99 102.59c0-13.53 10.97-24.5 24.5-24.5s24.5 10.97 24.5 24.5-10.97 24.5-24.5 24.5-24.5-10.97-24.5-24.5m35.42 10.93c1.56-1.57 1.56-4.1 0-5.66l-5.26-5.26 5.26-5.26c1.56-1.57 1.56-4.1 0-5.66s-4.1-1.56-5.66 0l-5.26 5.26-5.26-5.26c-1.56-1.56-4.1-1.56-5.66 0-1.56 1.57-1.56 4.1 0 5.66l5.26 5.26-5.26 5.26c-1.56 1.57-1.56 4.1 0 5.66.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17l5.26-5.26 5.26 5.26c.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17"
            })
        ]
    });
const $e56d6087ec1e1c3e$var$ForwardRef = /*#__PURE__*/ (0, $gkAc3$forwardRef)($e56d6087ec1e1c3e$var$SvgComponent);
var $e56d6087ec1e1c3e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e56d6087ec1e1c3e$var$ForwardRef);


export {$e56d6087ec1e1c3e$export$2e2bcd8739ae039 as default};
