// @react-spectrum/s2/illustrations/linear/internal/S2_lin_mailClose_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3HbFY$reactjsxruntime = require("react/jsx-runtime");
var $3HbFY$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c0c6ca96118d245f$export$2e2bcd8739ae039);




const $c0c6ca96118d245f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3HbFY$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3HbFY$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28.62 39.749C30.176 38.627 31.991 38 33.845 38h92.308c2.081 0 4.114.79 5.787 2.183l-49.563 29.06c-1.752 1.028-3.904 1.008-5.639-.053zm-4.314 6.74c-.199.898-.306 1.849-.306 2.844v61.334C24 117.32 28.782 122 33.846 122h92.308c5.064 0 9.846-4.68 9.846-11.333V49.333q-.002-1.117-.173-2.154L86.425 76.144c-4.299 2.52-9.61 2.47-13.86-.13zM33.846 30C23.616 30 16 39.05 16 49.333v61.334C16 120.95 23.616 130 33.846 130h92.308c10.23 0 17.846-9.05 17.846-19.333V49.333C144 39.05 136.384 30 126.154 30z"
        })
    });
const $c0c6ca96118d245f$var$ForwardRef = /*#__PURE__*/ (0, $3HbFY$react.forwardRef)($c0c6ca96118d245f$var$SvgComponent);
var $c0c6ca96118d245f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c0c6ca96118d245f$var$ForwardRef);
