// @react-spectrum/s2/illustrations/linear/internal/S2_lin_properties_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $c7Tr2$jsx} from "react/jsx-runtime";
import {forwardRef as $c7Tr2$forwardRef} from "react";





const $f5d0a52cdd759825$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $c7Tr2$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $c7Tr2$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M39.189 27.502c-3.314 0-6 2.686-6 6s2.686 6 6 6 6-2.686 6-6-2.687-6-6-6m-20.939 4h11.139c.926-4.564 4.962-8 9.8-8 4.837 0 8.873 3.436 9.8 8H77.75c1.105 0 2 .895 2 2s-.895 2-2 2H48.988c-.926 4.564-4.962 8-9.8 8-4.837 0-8.873-3.436-9.8-8H18.25c-1.105 0-2-.896-2-2s.895-2 2-2m28.762 28.997c.927-4.565 4.962-8 9.8-8s8.873 3.435 9.8 8h11.139c1.104 0 2 .895 2 2s-.896 2-2 2H66.612c-.927 4.565-4.962 8-9.8 8s-8.873-3.435-9.8-8H18.25c-1.104 0-2-.895-2-2s.896-2 2-2zm15.8 2c0-3.314-2.686-6-6-6s-6 2.686-6 6 2.686 6 6 6 6-2.686 6-6"
        })
    });
const $f5d0a52cdd759825$var$ForwardRef = /*#__PURE__*/ (0, $c7Tr2$forwardRef)($f5d0a52cdd759825$var$SvgComponent);
var $f5d0a52cdd759825$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f5d0a52cdd759825$var$ForwardRef);


export {$f5d0a52cdd759825$export$2e2bcd8739ae039 as default};
