// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pin_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1Xnd2$jsx} from "react/jsx-runtime";
import {forwardRef as $1Xnd2$forwardRef} from "react";





const $ed3b299800970722$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1Xnd2$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1Xnd2$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M60.579 24C52.729 24 46 30.084 46 38.031c0 6.916 4.979 13.211 12.116 14.244v17.16C51.251 70.343 46 75.738 46 82.968 46 90.916 52.73 97 60.579 97H76v39c0 2.209 1.79 4 4 4s4-1.791 4-4V97h15.421C107.27 97 114 90.915 114 82.969c0-6.916-4.979-13.211-12.116-14.244v-17.16C108.749 50.657 114 45.262 114 38.032 114 30.085 107.27 24 99.421 24zM54 38.031C54 34.898 56.742 32 60.579 32H99.42c3.836 0 6.579 2.898 6.579 6.031 0 3.138-2.358 5.67-6.19 5.67h-1.926c-2.21 0-4 1.791-4 4v24.875c0 2.21 1.79 4 4 4h1.926c3.456 0 6.19 2.88 6.19 6.393 0 3.133-2.743 6.031-6.579 6.031H60.58C56.742 89 54 86.102 54 82.969c0-3.138 2.358-5.67 6.19-5.67h1.926c2.21 0 4-1.792 4-4V48.422c0-2.209-1.79-4-4-4H60.19c-3.457 0-6.19-2.88-6.19-6.392"
        })
    });
const $ed3b299800970722$var$ForwardRef = /*#__PURE__*/ (0, $1Xnd2$forwardRef)($ed3b299800970722$var$SvgComponent);
var $ed3b299800970722$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ed3b299800970722$var$ForwardRef);


export {$ed3b299800970722$export$2e2bcd8739ae039 as default};
