// @react-spectrum/s2/illustrations/linear/internal/S2_lin_properties_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6hKRO$reactjsxruntime = require("react/jsx-runtime");
var $6hKRO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $09427c99b4458229$export$2e2bcd8739ae039);




const $09427c99b4458229$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6hKRO$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6hKRO$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M39.189 27.502c-3.314 0-6 2.686-6 6s2.686 6 6 6 6-2.686 6-6-2.687-6-6-6m-20.939 4h11.139c.926-4.564 4.962-8 9.8-8 4.837 0 8.873 3.436 9.8 8H77.75c1.105 0 2 .895 2 2s-.895 2-2 2H48.988c-.926 4.564-4.962 8-9.8 8-4.837 0-8.873-3.436-9.8-8H18.25c-1.105 0-2-.896-2-2s.895-2 2-2m28.762 28.997c.927-4.565 4.962-8 9.8-8s8.873 3.435 9.8 8h11.139c1.104 0 2 .895 2 2s-.896 2-2 2H66.612c-.927 4.565-4.962 8-9.8 8s-8.873-3.435-9.8-8H18.25c-1.104 0-2-.895-2-2s.896-2 2-2zm15.8 2c0-3.314-2.686-6-6-6s-6 2.686-6 6 2.686 6 6 6 6-2.686 6-6"
        })
    });
const $09427c99b4458229$var$ForwardRef = /*#__PURE__*/ (0, $6hKRO$react.forwardRef)($09427c99b4458229$var$SvgComponent);
var $09427c99b4458229$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($09427c99b4458229$var$ForwardRef);
