// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileZip_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $0468a$reactjsxruntime = require("react/jsx-runtime");
var $0468a$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f3c679c7780dc567$export$2e2bcd8739ae039);




const $f3c679c7780dc567$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $0468a$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $0468a$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M15.04 7.724h8.8q.055-.005.112-.005c1.188 0 2.33.458 3.19 1.282v.001l9.205 8.807c.917.878 1.432 2.096 1.432 3.36v13.463c0 2.555-2.066 4.635-4.621 4.635H15.041c-2.555 0-4.621-2.08-4.621-4.635V12.359c0-2.555 2.066-4.635 4.62-4.635m10.37 3.08c-.37-.355-.857-.562-1.37-.583l-.088.003H15.04c-1.165 0-2.121.951-2.121 2.135v22.273c0 1.183.956 2.135 2.12 2.135h18.118c1.165 0 2.12-.952 2.12-2.136V21.169c0-.587-.24-1.15-.66-1.552l-9.206-8.81zm-2.164 21.483v-2.576h1.418v2.576zm-1.114-6.732c-.797 0-1.386.652-1.386 1.386v5.627c0 1.232 1 2.219 2.219 2.219h1.98c1.232 0 2.219-1 2.219-2.219v-5.627c0-.797-.652-1.386-1.386-1.386h-.581v-3.64h1.282c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-1.282v-1.593h1.282c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-1.282v-.794c0-.69-.56-1.25-1.25-1.25h-2.228c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h.977v1.588h-.977c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h.977v1.592h-.977c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h.977v1.597z"
        })
    });
const $f3c679c7780dc567$var$ForwardRef = /*#__PURE__*/ (0, $0468a$react.forwardRef)($f3c679c7780dc567$var$SvgComponent);
var $f3c679c7780dc567$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f3c679c7780dc567$var$ForwardRef);
