// @react-spectrum/s2/illustrations/linear/internal/S2_lin_commentText_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6gC82$reactjsxruntime = require("react/jsx-runtime");
var $6gC82$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $157f011c23179c6c$export$2e2bcd8739ae039);




const $157f011c23179c6c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6gC82$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6gC82$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M9.863 7C6.8 7 4.25 9.396 4.25 12.49v14.463c0 3.094 2.551 5.49 5.613 5.49h2.793c.006 1.856.009 4.083-.004 5.858-.018 2.345 2.741 3.516 4.465 2.05l.002-.001 9.358-7.907h11.66c3.062 0 5.613-2.396 5.613-5.49V12.49c0-3.094-2.551-5.49-5.613-5.49zM6.75 12.49c0-1.637 1.355-2.99 3.113-2.99h28.274c1.758 0 3.113 1.353 3.113 2.99v14.463c0 1.638-1.355 2.99-3.113 2.99H25.562l-.35.296-9.71 8.204-.004.003c-.044.038-.08.05-.11.053-.037.004-.082-.002-.126-.022s-.072-.047-.087-.069c-.012-.017-.024-.04-.023-.089.016-2.21.008-5.11 0-7.13l-.005-1.246H9.862c-1.757 0-3.112-1.352-3.112-2.99zM15 15.492c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h18c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25zm0 6c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h18c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25z"
        })
    });
const $157f011c23179c6c$var$ForwardRef = /*#__PURE__*/ (0, $6gC82$react.forwardRef)($157f011c23179c6c$var$SvgComponent);
var $157f011c23179c6c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($157f011c23179c6c$var$ForwardRef);
