// @react-spectrum/s2/illustrations/linear/internal/S2_lin_stampClone_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $2kkH3$jsx} from "react/jsx-runtime";
import {forwardRef as $2kkH3$forwardRef} from "react";





const $118be0b484b19d31$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2kkH3$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2kkH3$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 30.117c-9.13 0-16.53 7.4-16.53 16.529 0 6.651 3.929 12.393 9.61 15.016q.269.124.51.282c1.487.835 2.326 2.57 1.95 4.325l-5.076 23.71c-.395 1.844-2.025 3.162-3.912 3.162H46.287c-6.627 0-12 5.373-12 12v4.005h91.425v-4.005c0-6.627-5.372-12-12-12H93.34c-1.914 0-3.56-1.356-3.926-3.235l-4.619-23.71c-.281-1.446.254-2.862 1.286-3.77.337-.387.754-.715 1.243-.958 5.462-2.705 9.204-8.33 9.204-14.822 0-9.129-7.4-16.529-16.529-16.529m13.169 37.225c6.824-4.35 11.359-11.99 11.359-20.696 0-13.547-10.982-24.529-24.529-24.529S55.471 33.1 55.471 46.646c0 8.817 4.652 16.541 11.623 20.862l-3.776 17.633h-17.03c-11.047 0-20 8.955-20 20v8.005c0 2.209 1.79 4 4 4h99.424c2.209 0 4-1.791 4-4v-8.005c0-11.045-8.954-20-20-20H96.636zM34.87 133.883c0-2.209 1.791-4 4-4h82.259c2.209 0 4 1.791 4 4s-1.791 4-4 4H38.871c-2.209 0-4-1.791-4-4"
        })
    });
const $118be0b484b19d31$var$ForwardRef = /*#__PURE__*/ (0, $2kkH3$forwardRef)($118be0b484b19d31$var$SvgComponent);
var $118be0b484b19d31$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($118be0b484b19d31$var$ForwardRef);


export {$118be0b484b19d31$export$2e2bcd8739ae039 as default};
