// @react-spectrum/s2/illustrations/linear/internal/S2_lin_musicNote_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $70rPg$reactjsxruntime = require("react/jsx-runtime");
var $70rPg$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6fcc0b1d1e535874$export$2e2bcd8739ae039);




const $6fcc0b1d1e535874$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $70rPg$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $70rPg$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M35.53 8.582c1.084-.216 2.093.612 2.093 1.716v1.542L20.14 15.325v-2.24c0-.834.59-1.552 1.408-1.715zm-15.39 9.292V34.88c.03 1.014-.308 2.018-.854 2.888-.818 1.301-2.164 2.423-3.813 3.132-1.649.708-3.39.912-4.896.609-1.5-.301-2.896-1.145-3.533-2.627-.637-1.483-.288-3.076.526-4.372.818-1.301 2.164-2.423 3.813-3.132 1.649-.708 3.39-.912 4.896-.61.47.095.931.243 1.361.448v-18.13c0-2.027 1.432-3.772 3.42-4.168L35.041 6.13c2.63-.524 5.081 1.487 5.081 4.168V31.29q0 .124-.023.241c-.085.813-.39 1.602-.831 2.304-.818 1.301-2.164 2.424-3.813 3.132s-3.39.912-4.896.61c-1.5-.302-2.896-1.145-3.533-2.628s-.288-3.076.526-4.372c.818-1.3 2.164-2.423 3.813-3.131 1.649-.709 3.39-.912 4.896-.61.47.095.931.243 1.361.447V14.389zM9.687 35.84c-.519.826-.566 1.542-.346 2.055.22.512.772.971 1.728 1.163.95.19 2.175.078 3.417-.456 1.243-.533 2.168-1.344 2.683-2.164.519-.826.566-1.543.346-2.055s-.772-.971-1.728-1.163c-.95-.19-2.175-.078-3.417.455-1.243.534-2.168 1.345-2.683 2.165m19.637-1.879c-.22-.512-.173-1.229.346-2.054.515-.82 1.44-1.631 2.682-2.165 1.243-.534 2.468-.646 3.418-.456.956.192 1.508.651 1.728 1.164s.173 1.228-.346 2.054c-.516.82-1.44 1.631-2.683 2.165s-2.467.646-3.417.455c-.956-.192-1.508-.65-1.728-1.163"
        })
    });
const $6fcc0b1d1e535874$var$ForwardRef = /*#__PURE__*/ (0, $70rPg$react.forwardRef)($6fcc0b1d1e535874$var$SvgComponent);
var $6fcc0b1d1e535874$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6fcc0b1d1e535874$var$ForwardRef);
