// @react-spectrum/s2/illustrations/linear/internal/S2_lin_camera_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3eoiN$reactjsxruntime = require("react/jsx-runtime");
var $3eoiN$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ac37104cc5ae4570$export$2e2bcd8739ae039);




const $ac37104cc5ae4570$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3eoiN$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3eoiN$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m39.183 21.884.001-.001C39.921 20.71 41.204 20 42.58 20h10.85c1.376 0 2.659.711 3.396 1.883v.001L61.902 30H34.109zM29.392 30l6.402-10.24.003-.004C37.26 17.428 39.817 16 42.58 16h10.85c2.763 0 5.32 1.428 6.783 3.756l.003.004L66.618 30H74c5.523 0 10 4.477 10 10v30c0 5.523-4.477 10-10 10H22c-5.523 0-10-4.477-10-10V40c0-5.523 4.477-10 10-10zm1.108 4H22c-3.314 0-6 2.686-6 6v30c0 3.314 2.686 6 6 6h52c3.314 0 6-2.686 6-6V40c0-3.314-2.686-6-6-6H30.5M48 42c-7.18 0-13 5.82-13 13s5.82 13 13 13 13-5.82 13-13-5.82-13-13-13M31 55c0-9.389 7.611-17 17-17s17 7.611 17 17-7.611 17-17 17-17-7.611-17-17m-5-14c0 1.657-1.343 3-3 3s-3-1.343-3-3 1.343-3 3-3 3 1.343 3 3"
        })
    });
const $ac37104cc5ae4570$var$ForwardRef = /*#__PURE__*/ (0, $3eoiN$react.forwardRef)($ac37104cc5ae4570$var$SvgComponent);
var $ac37104cc5ae4570$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ac37104cc5ae4570$var$ForwardRef);
