// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cloudStateDisconnected_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $lwq7B$jsxs, jsx as $lwq7B$jsx} from "react/jsx-runtime";
import {forwardRef as $lwq7B$forwardRef} from "react";





const $fb1a97d0d7e7073e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lwq7B$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lwq7B$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M25.53 8.19c-4.164 0-7.703 2.57-9.317 6.21-.563-.136-1.151-.21-1.753-.21-4.37 0-7.82 3.681-7.82 8.11q0 .375.034.743C4.28 24.183 2.65 26.675 2.65 29.54c0 3.863 2.98 7.088 6.79 7.15H27.2c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H9.471c-2.345-.043-4.32-2.057-4.32-4.65 0-2.177 1.404-3.956 3.255-4.489.634-.182 1.017-.825.875-1.47-.091-.415-.14-.845-.14-1.281 0-3.151 2.43-5.61 5.32-5.61.7 0 1.382.151 2.007.419.326.14.695.134 1.017-.014s.565-.427.67-.765c1.022-3.304 3.961-5.64 7.375-5.64 4.23 0 7.75 3.598 7.75 8.15v.2c0 .66.515 1.208 1.175 1.248 1.662.1 3.16.833 4.272 1.996.477.499 1.269.516 1.767.04.5-.478.517-1.27.04-1.768-1.273-1.331-2.931-2.266-4.79-2.622-.441-5.414-4.801-9.744-10.214-9.744"
            }),
            /*#__PURE__*/ (0, $lwq7B$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M28.94 30.98c0-4.18 3.39-7.57 7.57-7.57s7.57 3.39 7.57 7.57-3.39 7.57-7.57 7.57-7.57-3.39-7.57-7.57m10.96 3.39c.49-.49.49-1.28 0-1.77h.02l-1.62-1.62 1.62-1.62c.49-.49.49-1.28 0-1.77s-1.28-.49-1.77 0l-1.62 1.62-1.62-1.62c-.49-.49-1.28-.49-1.77 0s-.49 1.28 0 1.77l1.62 1.62-1.62 1.62c-.49.49-.49 1.28 0 1.77.24.25.56.37.88.37s.64-.13.88-.37l1.62-1.62 1.62 1.62c.24.25.56.37.88.37s.64-.13.88-.37"
            })
        ]
    });
const $fb1a97d0d7e7073e$var$ForwardRef = /*#__PURE__*/ (0, $lwq7B$forwardRef)($fb1a97d0d7e7073e$var$SvgComponent);
var $fb1a97d0d7e7073e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($fb1a97d0d7e7073e$var$ForwardRef);


export {$fb1a97d0d7e7073e$export$2e2bcd8739ae039 as default};
