// @react-spectrum/s2/illustrations/linear/internal/S2_lin_crop_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $j40oc$jsx} from "react/jsx-runtime";
import {forwardRef as $j40oc$forwardRef} from "react";





const $b0b4020c6d4d1657$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $j40oc$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $j40oc$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M53.446 24.063c0-2.21-1.791-4-4-4s-4 1.79-4 4v21.262H24.063c-2.21 0-4 1.791-4 4s1.79 4 4 4h21.383v41.35c0 11.046 8.954 20 20 20h41.108v21.263c0 2.209 1.791 4 4 4s4-1.791 4-4v-21.263h21.384c2.209 0 4-1.791 4-4s-1.791-4-4-4H65.446c-6.628 0-12-5.372-12-12V24.063m8.613 21.262c-2.209 0-4 1.791-4 4s1.791 4 4 4h32.496c6.627 0 11.999 5.373 11.999 12v32.72c0 2.209 1.791 4 4 4s4-1.791 4-4v-32.72c0-11.046-8.954-20-20-20z"
        })
    });
const $b0b4020c6d4d1657$var$ForwardRef = /*#__PURE__*/ (0, $j40oc$forwardRef)($b0b4020c6d4d1657$var$SvgComponent);
var $b0b4020c6d4d1657$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b0b4020c6d4d1657$var$ForwardRef);


export {$b0b4020c6d4d1657$export$2e2bcd8739ae039 as default};
