// @react-spectrum/s2/illustrations/linear/internal/S2_lin_apps_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $g2pl3$reactjsxruntime = require("react/jsx-runtime");
var $g2pl3$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1121238b089f7163$export$2e2bcd8739ae039);




const $1121238b089f7163$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $g2pl3$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $g2pl3$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.68 26.02c0-3.314 2.686-6 6-6h16.3c3.313 0 6 2.686 6 6v16.3c0 3.313-2.687 6-6 6h-16.3c-3.314 0-6-2.687-6-6zm6-10c-5.523 0-10 4.477-10 10v16.3c0 5.522 4.477 10 10 10h16.3c5.522 0 10-4.478 10-10v-16.3c0-5.523-4.478-10-10-10zM20.01 37.88c0-2.21 1.79-4 4-4h6.44c2.209 0 4 1.79 4 4v6.44c0 2.209-1.791 4-4 4h-6.44c-2.21 0-4-1.791-4-4zm4-8c-4.419 0-8 3.581-8 8v6.44c0 4.418 3.581 8 8 8h6.44c4.418 0 8-3.582 8-8v-6.44c0-4.419-3.582-8-8-8zm0 31.64c-2.21 0-4 1.79-4 4v6.44c0 2.209 1.79 4 4 4h6.44c2.209 0 4-1.791 4-4v-6.44c0-2.21-1.791-4-4-4zm-8 4c0-4.419 3.581-8 8-8h6.44c4.418 0 8 3.581 8 8v6.44c0 4.418-3.582 8-8 8h-6.44c-4.419 0-8-3.582-8-8zm31.67 0c0-2.21 1.79-4 4-4h6.44c2.209 0 4 1.79 4 4v6.44c0 2.209-1.791 4-4 4h-6.44c-2.21 0-4-1.791-4-4zm4-8c-4.419 0-8 3.581-8 8v6.44c0 4.418 3.581 8 8 8h6.44c4.418 0 8-3.582 8-8v-6.44c0-4.419-3.582-8-8-8z"
        })
    });
const $1121238b089f7163$var$ForwardRef = /*#__PURE__*/ (0, $g2pl3$react.forwardRef)($1121238b089f7163$var$SvgComponent);
var $1121238b089f7163$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1121238b089f7163$var$ForwardRef);
