// @react-spectrum/s2/illustrations/linear/internal/S2_lin_brush_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1zweF$jsx} from "react/jsx-runtime";
import {forwardRef as $1zweF$forwardRef} from "react";





const $f0c4efb6da3609d0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1zweF$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1zweF$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.207 46.143 72.84 20.51c2.013-2.013 5.277-2.013 7.29 0s2.013 5.277 0 7.29L53.868 54.062c-.639-1.894-1.575-3.58-2.846-5.003-1.079-1.209-2.356-2.18-3.815-2.916m-4.26-1.398 27.064-27.064c3.576-3.575 9.372-3.575 12.947 0 3.576 3.576 3.576 9.372 0 12.948L54.831 58.756q.12 1.288.12 2.647c0 5.221-1.95 9.622-5.358 12.893-3.378 3.242-8.083 5.279-13.495 6.054-7.903 1.13-18.628.05-23.458-.55-1.416-.176-2.295-1.236-2.552-2.3-.255-1.05.027-2.348 1.1-3.179 2.725-2.106 7.105-6.075 9.078-10.868 1.581-3.84 3.529-8.54 6.354-12.263 2.868-3.78 6.82-6.784 12.39-6.784q2.087 0 3.938.339m-18.982 20.23c1.616-3.923 3.383-8.127 5.841-11.367 2.415-3.183 5.312-5.202 9.205-5.202 4.328 0 7.203 1.273 9.027 3.317 1.86 2.083 2.913 5.282 2.913 9.68 0 4.15-1.518 7.503-4.127 10.007-2.64 2.534-6.504 4.294-11.293 4.98-6.332.906-14.86.274-20.18-.302 2.868-2.458 6.652-6.346 8.614-11.112"
        })
    });
const $f0c4efb6da3609d0$var$ForwardRef = /*#__PURE__*/ (0, $1zweF$forwardRef)($f0c4efb6da3609d0$var$SvgComponent);
var $f0c4efb6da3609d0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f0c4efb6da3609d0$var$ForwardRef);


export {$f0c4efb6da3609d0$export$2e2bcd8739ae039 as default};
