// @react-spectrum/s2/illustrations/linear/internal/S2_lin_desktop_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $777Fo$jsx} from "react/jsx-runtime";
import {forwardRef as $777Fo$forwardRef} from "react";





const $88ec2337e4b79844$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $777Fo$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $777Fo$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M7.25 13c0-1.519 1.231-2.75 2.75-2.75h28c1.519 0 2.75 1.231 2.75 2.75v14c0 1.519-1.231 2.75-2.75 2.75H10c-1.519 0-2.75-1.231-2.75-2.75zM10 7.75c-2.9 0-5.25 2.35-5.25 5.25v14c0 2.9 2.35 5.25 5.25 5.25h12v5.5h-4c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h12c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-4v-5.5h12c2.9 0 5.25-2.35 5.25-5.25V13c0-2.9-2.35-5.25-5.25-5.25zm15.5 5.75c0 .828-.672 1.5-1.5 1.5s-1.5-.672-1.5-1.5.672-1.5 1.5-1.5 1.5.672 1.5 1.5"
        })
    });
const $88ec2337e4b79844$var$ForwardRef = /*#__PURE__*/ (0, $777Fo$forwardRef)($88ec2337e4b79844$var$SvgComponent);
var $88ec2337e4b79844$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($88ec2337e4b79844$var$ForwardRef);


export {$88ec2337e4b79844$export$2e2bcd8739ae039 as default};
