// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileImage_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $j3n8S$reactjsxruntime = require("react/jsx-runtime");
var $j3n8S$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $08df13dc975e0ea7$export$2e2bcd8739ae039);




const $08df13dc975e0ea7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $j3n8S$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $j3n8S$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M29.008 16C24.054 16 20 19.976 20 24.929v46.193c0 4.952 4.054 8.929 9.008 8.929h37.374c4.953 0 9.007-3.977 9.007-8.93V43.967c0-2.375-.954-4.648-2.646-6.321L53.49 18.607C51.8 16.935 49.51 16 47.128 16zM24 24.929C24 22.228 26.22 20 29.008 20h18.12c1.335 0 2.611.525 3.549 1.452L69.93 40.489c.936.926 1.459 2.178 1.459 3.477v27.156c0 2.7-2.221 4.929-5.008 4.929H29.008C26.22 76.05 24 73.823 24 71.12zm32.863 24.54c-2.55 0-4.619-2.068-4.619-4.62 0-2.55 2.068-4.619 4.62-4.619 2.55 0 4.619 2.068 4.619 4.62 0 2.55-2.068 4.619-4.62 4.619M38.65 50.923c1.325-2.296 4.639-2.296 5.964 0l5.9 10.218 3.148-5.455c1.326-2.295 4.64-2.295 5.965 0l5.75 9.96c1.326 2.296-.33 5.166-2.982 5.166H33.131c-2.65 0-4.307-2.87-2.982-5.165zm9.56 14.227-6.578-11.393-7.537 13.055h13.173zm3.655 1.662h9.566l-4.786-8.29-4.395 7.612z"
        })
    });
const $08df13dc975e0ea7$var$ForwardRef = /*#__PURE__*/ (0, $j3n8S$react.forwardRef)($08df13dc975e0ea7$var$SvgComponent);
var $08df13dc975e0ea7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($08df13dc975e0ea7$var$ForwardRef);
