// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowRight_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $cKnvQ$reactjsxruntime = require("react/jsx-runtime");
var $cKnvQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $21dca3e645dbf986$export$2e2bcd8739ae039);




const $21dca3e645dbf986$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cKnvQ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cKnvQ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 76c-2.21 0-4 1.79-4 4s1.79 4 4 4h104.843l-16.671 16.672c-1.563 1.562-1.563 4.094 0 5.656 1.562 1.563 4.094 1.563 5.656 0l23.5-23.5c1.563-1.562 1.563-4.094 0-5.656l-23.5-23.5c-1.562-1.562-4.094-1.562-5.656 0-1.563 1.562-1.563 4.094 0 5.656L128.843 76z"
        })
    });
const $21dca3e645dbf986$var$ForwardRef = /*#__PURE__*/ (0, $cKnvQ$react.forwardRef)($21dca3e645dbf986$var$SvgComponent);
var $21dca3e645dbf986$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($21dca3e645dbf986$var$ForwardRef);
