// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cursor_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $5EWJy$reactjsxruntime = require("react/jsx-runtime");
var $5EWJy$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d3b9f88600fbafea$export$2e2bcd8739ae039);




const $d3b9f88600fbafea$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5EWJy$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5EWJy$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M14.927 26.56c-3.171-7.703 4.53-15.405 12.233-12.233l51.95 21.39c8.225 3.388 8.434 14.96.336 18.642l-14.732 6.696c-1.356.616-2.442 1.703-3.058 3.058l-6.697 14.733c-3.68 8.097-15.254 7.889-18.64-.336zm10.71-8.535c-4.414-1.817-8.829 2.597-7.011 7.011l-1.85.762 1.85-.762 21.391 51.95c2.053 4.987 9.07 5.113 11.3.204l6.697-14.732c1.017-2.236 2.809-4.028 5.044-5.044l14.733-6.697c4.909-2.231 4.782-9.247-.204-11.3z"
        })
    });
const $d3b9f88600fbafea$var$ForwardRef = /*#__PURE__*/ (0, $5EWJy$react.forwardRef)($d3b9f88600fbafea$var$SvgComponent);
var $d3b9f88600fbafea$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d3b9f88600fbafea$var$ForwardRef);
