// @react-spectrum/s2/illustrations/linear/internal/S2_lin_rocket_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $4C1ma$reactjsxruntime = require("react/jsx-runtime");
var $4C1ma$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4b444a621adf2934$export$2e2bcd8739ae039);




const $4b444a621adf2934$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4C1ma$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4C1ma$reactjsxruntime.jsx)("defs", {
                children: /*#__PURE__*/ (0, $4C1ma$reactjsxruntime.jsx)("clipPath", {
                    id: "e169c2__a",
                    children: /*#__PURE__*/ (0, $4C1ma$reactjsxruntime.jsx)("path", {
                        fill: "#fff",
                        d: "M0 0h48v48H0z"
                    })
                })
            }),
            /*#__PURE__*/ (0, $4C1ma$reactjsxruntime.jsx)("g", {
                clipPath: "url(#e169c2__a)",
                children: /*#__PURE__*/ (0, $4C1ma$reactjsxruntime.jsx)("path", {
                    fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                    fillRule: "evenodd",
                    d: "M37.574 11.204c-.07-.352-.334-.616-.685-.685-2.565-.507-8.06.168-13.558 5.667-4.189 4.189-5.994 6.536-6.766 7.935-.373.678-.48 1.09-.516 1.35-.028.206-.022.312-.014.48.006.11.012.244.012.449 0 .737.392 1.597 2.22 3.426 1.829 1.828 2.69 2.22 3.426 2.22.205 0 .34.007.448.012.169.008.275.014.481-.014.26-.036.672-.142 1.35-.516 1.399-.772 3.746-2.577 7.935-6.766 5.499-5.499 6.174-10.993 5.667-13.558m-.201-3.138c1.346.266 2.388 1.308 2.654 2.654.65 3.289-.23 9.114-5.413 14.825.265 1.49.05 3.236-.393 4.93-.518 1.979-1.391 4.057-2.43 5.899-1.033 1.83-2.274 3.503-3.563 4.615-1.208 1.041-2.91 1.949-4.573 1.117-1.458-.73-1.859-2.114-1.992-3.49-.067-.693-.077-1.486-.078-2.34v-.546q0-.566-.003-1.186c-1.799-.044-3.285-1.152-5.083-2.95-1.797-1.798-2.906-3.283-2.95-5.082q-.62-.004-1.187-.003h-.547c-.852-.001-1.646-.011-2.339-.078-1.376-.134-2.76-.534-3.49-1.992-.831-1.664.076-3.365 1.117-4.573 1.112-1.29 2.786-2.53 4.615-3.562 1.842-1.04 3.92-1.913 5.9-2.43 1.693-.443 3.438-.659 4.93-.394 5.71-5.184 11.536-6.063 14.825-5.414m-12.194 25.65c1.509-.831 3.65-2.48 6.98-5.695-.068.56-.187 1.17-.357 1.821-.457 1.746-1.242 3.626-2.189 5.303-.953 1.69-2.026 3.095-3.018 3.95-1.074.927-1.624.874-1.822.775-.265-.133-.512-.365-.622-1.495-.054-.56-.065-1.242-.066-2.103v-.529q.001-.7-.006-1.512c.336-.125.7-.293 1.1-.514m-5.107-17.781c-.56.068-1.171.187-1.822.357-1.746.456-3.626 1.242-5.303 2.189-1.69.953-3.095 2.026-3.95 3.018-.926 1.074-.874 1.624-.774 1.822.132.265.365.512 1.494.621.56.055 1.242.066 2.103.067h.529q.7-.002 1.512.006c.126-.337.294-.7.515-1.101.832-1.509 2.48-3.65 5.696-6.98m11.641 3.892c-.952.952-2.496.952-3.448 0s-.952-2.496 0-3.448 2.496-.952 3.448 0 .952 2.496 0 3.448M13.849 36.011c.489-.488.489-1.28 0-1.768-.488-.488-1.28-.488-1.767 0L6.155 40.17c-.488.488-.488 1.28 0 1.767.489.489 1.28.489 1.768 0zm3.017 1.25c.488.487.488 1.279 0 1.767l-3.448 3.448c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.768l3.448-3.448c.488-.488 1.28-.488 1.768 0m-6.033-4.267c.488-.488.488-1.28 0-1.768s-1.28-.488-1.768 0l-3.448 3.448c-.488.489-.488 1.28 0 1.768s1.28.488 1.768 0z"
                })
            })
        ]
    });
const $4b444a621adf2934$var$ForwardRef = /*#__PURE__*/ (0, $4C1ma$react.forwardRef)($4b444a621adf2934$var$SvgComponent);
var $4b444a621adf2934$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4b444a621adf2934$var$ForwardRef);
