// @react-spectrum/s2/illustrations/linear/internal/S2_lin_buildTable_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $7YfOQ$reactjsxruntime = require("react/jsx-runtime");
var $7YfOQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5ec991479b9bdea2$export$2e2bcd8739ae039);




const $5ec991479b9bdea2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7YfOQ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7YfOQ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22 24c-3.315 0-6 2.685-6 6v4.04h64V30c0-3.315-2.685-6-6-6zm-6 42V38.04h10.02V72H22c-3.315 0-6-2.685-6-6m43.11 6H30.02V38.04H80v12.92c0 1.105.895 2 2 2s2-.895 2-2V30c0-5.525-4.475-10-10-10H22c-5.525 0-10 4.475-10 10v36c0 5.525 4.475 10 10 10h37.11c1.105 0 2-.895 2-2s-.895-2-2-2m4.5-2.61c0-7.67 6.22-13.89 13.89-13.89s13.89 6.22 13.89 13.89-6.22 13.89-13.89 13.89-13.89-6.22-13.89-13.89m15.89 1.97h4.73c1.1 0 2-.9 2-2s-.9-2-2-2H79.5v-4.73c0-1.1-.9-2-2-2s-2 .9-2 2v4.73h-4.73c-1.1 0-2 .9-2 2s.9 2 2 2h4.73v4.73c0 1.1.9 2 2 2s2-.9 2-2z"
        })
    });
const $5ec991479b9bdea2$var$ForwardRef = /*#__PURE__*/ (0, $7YfOQ$react.forwardRef)($5ec991479b9bdea2$var$SvgComponent);
var $5ec991479b9bdea2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5ec991479b9bdea2$var$ForwardRef);
