// @react-spectrum/s2/illustrations/linear/internal/S2_lin_rotateCCW_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bTlPn$reactjsxruntime = require("react/jsx-runtime");
var $bTlPn$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b9dbcf94c3551ec3$export$2e2bcd8739ae039);




const $b9dbcf94c3551ec3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bTlPn$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bTlPn$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M39.25 24c0-8.422-6.828-15.25-15.25-15.25-6.026 0-11.238 3.495-13.714 8.573l4.671-1.472c.659-.207 1.36.159 1.568.817.207.659-.158 1.36-.817 1.568L8.266 20.58c-.658.208-1.36-.158-1.568-.816L4.355 12.32c-.207-.659.158-1.36.817-1.568s1.36.158 1.568.817l1.401 4.45c2.92-5.793 8.924-9.77 15.86-9.77 9.802 0 17.75 7.947 17.75 17.75S33.802 41.75 24 41.75c-7.668 0-14.197-4.86-16.68-11.664-.236-.649.097-1.366.746-1.603.648-.237 1.366.097 1.603.746C11.804 35.078 17.417 39.25 24 39.25c8.423 0 15.25-6.828 15.25-15.25"
        })
    });
const $b9dbcf94c3551ec3$var$ForwardRef = /*#__PURE__*/ (0, $bTlPn$react.forwardRef)($b9dbcf94c3551ec3$var$SvgComponent);
var $b9dbcf94c3551ec3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b9dbcf94c3551ec3$var$ForwardRef);
