// @react-spectrum/s2/illustrations/linear/internal/S2_lin_assets_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $di3eH$jsx} from "react/jsx-runtime";
import {forwardRef as $di3eH$forwardRef} from "react";





const $81ba89c7a10ac663$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $di3eH$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        fill: "none",
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $di3eH$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M7.875 14.027c0-2.9 2.35-5.25 5.25-5.25H30c2.9 0 5.25 2.35 5.25 5.25v2.848c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-2.848c0-1.52-1.231-2.75-2.75-2.75H13.125c-1.519 0-2.75 1.23-2.75 2.75v10.978l4.483-6.948c1.106-1.714 3.622-1.686 4.69.052l.001.002.554.904c.36.589.175 1.358-.414 1.719-.588.36-1.358.175-1.718-.414l-.553-.902v-.001c-.105-.17-.351-.172-.46-.004L11.804 27.4c.392.215.842.337 1.321.337h4.656c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-4.656c-2.9 0-5.25-2.35-5.25-5.25zm20.198 3.183c.828 0 1.5-.672 1.5-1.5s-.672-1.5-1.5-1.5c-.829 0-1.5.672-1.5 1.5s.671 1.5 1.5 1.5m-7.547 7.665c0-2.9 2.35-5.25 5.25-5.25h9.099c2.9 0 5.25 2.35 5.25 5.25v9.1c0 2.899-2.35 5.25-5.25 5.25h-9.1c-2.899 0-5.25-2.351-5.25-5.25zm5.25-2.75c-1.519 0-2.75 1.231-2.75 2.75v9.1c0 1.518 1.231 2.75 2.75 2.75h9.099c1.519 0 2.75-1.232 2.75-2.75v-9.1c0-1.519-1.231-2.75-2.75-2.75zm2.633 3.515 5.607 3.216c.628.36.627 1.259-.002 1.617l-5.615 3.203c-.629.359-1.414-.091-1.413-.81l.008-6.42c0-.718.787-1.166 1.415-.806",
            clipRule: "evenodd"
        })
    });
const $81ba89c7a10ac663$var$ForwardRef = /*#__PURE__*/ (0, $di3eH$forwardRef)($81ba89c7a10ac663$var$SvgComponent);
var $81ba89c7a10ac663$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($81ba89c7a10ac663$var$ForwardRef);


export {$81ba89c7a10ac663$export$2e2bcd8739ae039 as default};
