// @react-spectrum/s2/illustrations/linear/internal/S2_lin_transform_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $hsnyy$reactjsxruntime = require("react/jsx-runtime");
var $hsnyy$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $64384b0f7d9d0dc1$export$2e2bcd8739ae039);




const $64384b0f7d9d0dc1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hsnyy$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hsnyy$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 28c0-2.21 1.79-4 4-4h24.184c2.209 0 4 1.79 4 4s-1.791 4-4 4H37.288l42.12 42.12L121.53 32h-14.621c-2.209 0-4-1.79-4-4s1.791-4 4-4h24.184c2.209 0 4 1.79 4 4v24.184c0 2.209-1.791 4-4 4s-4-1.791-4-4V37.75L85.065 79.777l42.027 42.027v-14.896c0-2.209 1.791-4 4-4s4 1.791 4 4v24.184c0 2.209-1.791 4-4 4h-24.184c-2.209 0-4-1.791-4-4s1.791-4 4-4h14.158L79.408 85.434l-41.657 41.658h14.433c2.209 0 4 1.791 4 4s-1.791 4-4 4H28c-2.21 0-4-1.791-4-4v-24.184c0-2.209 1.79-4 4-4s4 1.791 4 4v14.621l41.751-41.752L32 38.026v14.158c0 2.209-1.79 4-4 4s-4-1.791-4-4z"
        })
    });
const $64384b0f7d9d0dc1$var$ForwardRef = /*#__PURE__*/ (0, $hsnyy$react.forwardRef)($64384b0f7d9d0dc1$var$SvgComponent);
var $64384b0f7d9d0dc1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($64384b0f7d9d0dc1$var$ForwardRef);
