// @react-spectrum/s2/illustrations/linear/internal/S2_lin_illustration_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6Glns$jsx} from "react/jsx-runtime";
import {forwardRef as $6Glns$forwardRef} from "react";





const $71941302b8a9ded3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6Glns$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6Glns$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m24.874 18.477-4.882-4.882-4.082 2.118c-.582.302-1.036.802-1.282 1.41l-3.134 7.74 3.957-3.956q.04-.039.08-.075c.02-.42.19-.834.511-1.155.684-.683 1.792-.683 2.475 0 .683.684.683 1.792 0 2.475-.355.355-.823.525-1.288.512l-.01.01-4.769 4.769 8.897-3.602c.607-.245 1.107-.7 1.409-1.282zm-5.414-7.422L21.715 8.8c1.27-1.27 3.327-1.27 4.596 0l3.352 3.351c1.27 1.27 1.27 3.327 0 4.597l-2.235 2.235-2.453 4.727c-.576 1.111-1.53 1.98-2.69 2.449l-10.742 4.348c-2.251.911-4.492-1.33-3.58-3.58l4.347-10.743c.47-1.16 1.338-2.114 2.449-2.69zm8.435 3.925-1.491 1.491-4.413-4.412 1.492-1.492c.293-.293.768-.293 1.06 0l3.352 3.352c.293.293.293.768 0 1.06m2.352 15.371 7.29-8.899c.46-.561.378-1.39-.184-1.85-.561-.46-1.39-.377-1.85.184l-7.261 8.865c.34.159.652.35.92.57.37.303.75.678 1.085 1.13m1.002 2.722 8.222-10.036c1.335-1.63 1.096-4.033-.534-5.368s-4.033-1.097-5.368.533l-8.125 9.92q-.365.015-.73.073c-1.74.28-3.5 1.326-4.542 3.476-.42.867-.731 1.545-.987 2.104-.348.76-.594 1.297-.876 1.786-.411.715-.858 1.235-1.935 2.074-.4.311-.573.834-.436 1.323.137.49.556.846 1.06.905 1.525.175 4.226.422 6.825.105 2.512-.306 5.423-1.2 6.7-3.754.563-1.124.781-2.177.726-3.141m-2.962 2.023c-.67 1.339-2.386 2.1-4.766 2.39-1.195.147-2.434.158-3.557.11.182-.247.35-.506.512-.788.33-.572.655-1.28 1.058-2.156.257-.559.546-1.186.888-1.891.658-1.36 1.69-1.937 2.69-2.098 1.077-.173 2.043.146 2.467.493.494.405.898.907 1.076 1.5.168.56.18 1.345-.368 2.44"
        })
    });
const $71941302b8a9ded3$var$ForwardRef = /*#__PURE__*/ (0, $6Glns$forwardRef)($71941302b8a9ded3$var$SvgComponent);
var $71941302b8a9ded3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($71941302b8a9ded3$var$ForwardRef);


export {$71941302b8a9ded3$export$2e2bcd8739ae039 as default};
