// @react-spectrum/s2/illustrations/linear/internal/S2_lin_wallet_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dIi3y$reactjsxruntime = require("react/jsx-runtime");
var $dIi3y$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f233cfe1605760c9$export$2e2bcd8739ae039);




const $f233cfe1605760c9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dIi3y$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dIi3y$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M23.068 31.78c-.838-3.128 1.018-6.343 4.146-7.181l30.076-8.06c3.2-.857 6.49 1.043 7.348 4.244l2.85 10.636c.286 1.067-.347 2.164-1.414 2.45-1.067.285-2.163-.348-2.45-1.415l-2.85-10.636c-.285-1.067-1.382-1.7-2.449-1.414L28.25 28.462c-.994.267-1.584 1.289-1.318 2.283l.18.674c.286 1.067-.347 2.164-1.414 2.45-1.067.285-2.163-.348-2.45-1.415zM18 36.654c-2.21 0-4 1.79-4 4v36c0 2.209 1.79 4 4 4h50c5.523 0 10-4.478 10-10v-2.5c2.21 0 4-1.791 4-4v-11c0-2.21-1.79-4-4-4v-2.5c0-5.523-4.477-10-10-10zm56 12.5v-2.5c0-3.314-2.686-6-6-6H18v36h50c3.314 0 6-2.687 6-6v-2.5H62c-2.21 0-4-1.791-4-4v-11c0-2.21 1.79-4 4-4zm-12 4h16v11H62zm5.5 7.751c1.38 0 2.5-1.119 2.5-2.5 0-1.38-1.12-2.5-2.5-2.5s-2.5 1.12-2.5 2.5c0 1.381 1.12 2.5 2.5 2.5"
        })
    });
const $f233cfe1605760c9$var$ForwardRef = /*#__PURE__*/ (0, $dIi3y$react.forwardRef)($f233cfe1605760c9$var$SvgComponent);
var $f233cfe1605760c9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f233cfe1605760c9$var$ForwardRef);
