// @react-spectrum/s2/illustrations/linear/internal/S2_lin_helpCircle_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $lhEe3$reactjsxruntime = require("react/jsx-runtime");
var $lhEe3$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $11af9817ecaef2ff$export$2e2bcd8739ae039);




const $11af9817ecaef2ff$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lhEe3$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 49,
        viewBox: "0 0 48 49",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $lhEe3$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 9.238c-8.422 0-15.25 6.828-15.25 15.25s6.828 15.25 15.25 15.25 15.25-6.827 15.25-15.25S32.422 9.238 24 9.238M6.25 24.488c0-9.803 7.947-17.75 17.75-17.75s17.75 7.947 17.75 17.75-7.947 17.75-17.75 17.75-17.75-7.947-17.75-17.75m19.25 7.586c0 .829-.672 1.5-1.5 1.5s-1.5-.671-1.5-1.5c0-.828.672-1.5 1.5-1.5s1.5.672 1.5 1.5m-2.26-14.162c.37-.157.777-.198 1.169-.118s.757.278 1.046.574.49.678.57 1.097c.082.42.04.855-.119 1.248s-.426.725-.763.956c-.576.394-1.149.932-1.588 1.566-.435.63-.79 1.43-.79 2.326v2.087c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-2.087c0-.231.096-.543.346-.903.246-.355.59-.684.944-.927.749-.512 1.328-1.238 1.669-2.08.34-.844.43-1.77.256-2.662-.173-.893-.602-1.718-1.236-2.367-.635-.65-1.447-1.097-2.336-1.278s-1.81-.088-2.646.267c-.835.354-1.544.952-2.04 1.714-.497.76-.76 1.652-.76 2.562 0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25c0-.428.125-.845.354-1.197.229-.351.552-.62.923-.778"
        })
    });
const $11af9817ecaef2ff$var$ForwardRef = /*#__PURE__*/ (0, $lhEe3$react.forwardRef)($11af9817ecaef2ff$var$SvgComponent);
var $11af9817ecaef2ff$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($11af9817ecaef2ff$var$ForwardRef);
