// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lockOpen_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6zmqs$jsx} from "react/jsx-runtime";
import {forwardRef as $6zmqs$forwardRef} from "react";





const $fa32af6a6a18abf5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6zmqs$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6zmqs$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M34 27.5C34 19.492 40.492 13 48.5 13S63 19.492 63 27.5c0 1.105-.895 2-2 2s-2-.895-2-2C59 21.701 54.299 17 48.5 17S38 21.701 38 27.5V39h30c5.523 0 10 4.477 10 10v21c0 5.523-4.477 10-10 10H28c-5.523 0-10-4.477-10-10V49c0-5.523 4.477-10 10-10h6zm12 33.084c-1.766-.772-3-2.534-3-4.584 0-2.761 2.239-5 5-5s5 2.239 5 5c0 2.05-1.234 3.812-3 4.584v4.917c0 1.105-.895 2-2 2s-2-.895-2-2zM22 49c0-3.314 2.686-6 6-6h40c3.314 0 6 2.686 6 6v21c0 3.314-2.686 6-6 6H28c-3.314 0-6-2.686-6-6z"
        })
    });
const $fa32af6a6a18abf5$var$ForwardRef = /*#__PURE__*/ (0, $6zmqs$forwardRef)($fa32af6a6a18abf5$var$SvgComponent);
var $fa32af6a6a18abf5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($fa32af6a6a18abf5$var$ForwardRef);


export {$fa32af6a6a18abf5$export$2e2bcd8739ae039 as default};
