// @react-spectrum/s2/illustrations/linear/internal/S2_lin_accessibility_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $j9IyN$reactjsxruntime = require("react/jsx-runtime");
var $j9IyN$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b6b9afa44bab8a88$export$2e2bcd8739ae039);




const $b6b9afa44bab8a88$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $j9IyN$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $j9IyN$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M17 48.492c0-17.12 13.88-31 31-31s31 13.88 31 31c0 17.121-13.88 31-31 31s-31-13.879-31-31m31-35c-19.33 0-35 15.67-35 35s15.67 35 35 35 35-15.67 35-35-15.67-35-35-35m5 17.147c0 2.761-2.238 5-5 5s-5-2.239-5-5 2.239-5 5-5 5 2.238 5 5m-19.565 5.743c-1.062-.303-2.169.311-2.473 1.373s.311 2.17 1.373 2.473c2.887.826 5.823 1.42 8.781 1.78 1.028.124 1.82.985 1.82 2.02v7.905q0 .279-.077.547c-.8 2.82-1.714 6.049-2.515 8.887-.576 2.04-1.094 3.878-1.47 5.217l-.448 1.6-.025.087c-.078.281-.16.575-.181.682-.217 1.083.486 2.136 1.569 2.353s2.137-.486 2.353-1.569c-.01.053-.002.022.048-.16l.001-.002.088-.317.446-1.592c.375-1.337.893-3.174 1.468-5.213.657-2.327 1.39-4.916 2.073-7.326.22-.776.928-1.31 1.734-1.31s1.514.534 1.734 1.31c.683 2.41 1.416 5 2.073 7.326.575 2.04 1.093 3.876 1.468 5.213l.446 1.592.036.13.052.186v.003c.05.182.06.213.049.16.216 1.083 1.27 1.786 2.353 1.569s1.786-1.27 1.57-2.353c-.022-.107-.104-.4-.182-.681l-.025-.088-.448-1.6c-.376-1.34-.894-3.178-1.47-5.217-.801-2.838-1.715-6.068-2.515-8.887q-.075-.268-.076-.547v-7.905c0-1.035.79-1.896 1.819-2.021 2.825-.344 5.63-.901 8.392-1.67l.376-.105c1.064-.297 1.686-1.4 1.39-2.463s-1.4-1.687-2.463-1.39l-.376.104c-9.415 2.623-19.372 2.588-28.768-.1"
        })
    });
const $b6b9afa44bab8a88$var$ForwardRef = /*#__PURE__*/ (0, $j9IyN$react.forwardRef)($b6b9afa44bab8a88$var$SvgComponent);
var $b6b9afa44bab8a88$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b6b9afa44bab8a88$var$ForwardRef);
