// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowDown_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1uIL8$jsx} from "react/jsx-runtime";
import {forwardRef as $1uIL8$forwardRef} from "react";





const $4e50c08b18b10b95$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1uIL8$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1uIL8$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M84 24c0-2.21-1.79-4-4-4s-4 1.79-4 4v104.843l-16.672-16.671c-1.562-1.563-4.094-1.563-5.656 0s-1.562 4.094 0 5.656l23.5 23.5c1.562 1.563 4.094 1.563 5.656 0l23.5-23.5c1.563-1.562 1.563-4.094 0-5.656-1.562-1.563-4.094-1.563-5.656 0L84 128.843z"
        })
    });
const $4e50c08b18b10b95$var$ForwardRef = /*#__PURE__*/ (0, $1uIL8$forwardRef)($4e50c08b18b10b95$var$SvgComponent);
var $4e50c08b18b10b95$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($4e50c08b18b10b95$var$ForwardRef);


export {$4e50c08b18b10b95$export$2e2bcd8739ae039 as default};
