// @react-spectrum/s2/illustrations/linear/internal/S2_lin_calendar_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $7KqtJ$reactjsxruntime = require("react/jsx-runtime");
var $7KqtJ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $efcdea8c1d566d4c$export$2e2bcd8739ae039);




const $efcdea8c1d566d4c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7KqtJ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7KqtJ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M55 20c2.21 0 4 1.79 4 4v6h42v-6c0-2.21 1.791-4 4-4s4 1.79 4 4v6h7c11.046 0 20 8.954 20 20v66c0 11.046-8.954 20-20 20H44c-11.046 0-20-8.954-20-20V50c0-11.046 8.954-20 20-20h7v-6c0-2.21 1.79-4 4-4m46 18v6c0 2.21 1.791 4 4 4s4-1.79 4-4v-6h7c6.627 0 12 5.373 12 12v10H32V50c0-6.627 5.373-12 12-12h7v6c0 2.21 1.79 4 4 4s4-1.79 4-4v-6zM32 68v48c0 6.627 5.373 12 12 12h72c6.627 0 12-5.373 12-12V68zm17 12h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4v-6c0-2.21 1.79-4 4-4m34 0h-6c-2.21 0-4 1.79-4 4v6c0 2.21 1.79 4 4 4h6c2.21 0 4-1.79 4-4v-6c0-2.21-1.79-4-4-4m22 0h6c2.209 0 4 1.79 4 4v6c0 2.21-1.791 4-4 4h-6c-2.209 0-4-1.79-4-4v-6c0-2.21 1.791-4 4-4m-50 22h-6c-2.21 0-4 1.791-4 4v6c0 2.209 1.79 4 4 4h6c2.21 0 4-1.791 4-4v-6c0-2.209-1.79-4-4-4m22 0h6c2.21 0 4 1.791 4 4v6c0 2.209-1.79 4-4 4h-6c-2.21 0-4-1.791-4-4v-6c0-2.209 1.79-4 4-4m34 0h-6c-2.209 0-4 1.791-4 4v6c0 2.209 1.791 4 4 4h6c2.209 0 4-1.791 4-4v-6c0-2.209-1.791-4-4-4"
        })
    });
const $efcdea8c1d566d4c$var$ForwardRef = /*#__PURE__*/ (0, $7KqtJ$react.forwardRef)($efcdea8c1d566d4c$var$SvgComponent);
var $efcdea8c1d566d4c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($efcdea8c1d566d4c$var$ForwardRef);
