// @react-spectrum/s2/illustrations/linear/internal/S2_lin_education_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $fejKo$reactjsxruntime = require("react/jsx-runtime");
var $fejKo$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $75ffe786800e58ca$export$2e2bcd8739ae039);




const $75ffe786800e58ca$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fejKo$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fejKo$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M23.692 12.471c.183-.076.39-.076.573 0l13.79 5.707c.618.255.618 1.13 0 1.386l-13.79 5.706c-.184.076-.39.076-.573 0l-3.714-1.537c-.638-.264-1.37.04-1.633.677-.264.638.039 1.37.677 1.633l3.714 1.537c.795.33 1.69.33 2.485 0l9.314-3.854v4.327c0 .437-.117 1.195-.687 2.043-.567.845-1.631 1.851-3.64 2.712-1.526.654-3.392.94-5.149 1.006-1.752.065-3.29-.094-4.108-.262-.676-.14-1.338.296-1.477.972s.296 1.337.973 1.477c1.04.214 2.782.383 4.705.312 1.92-.072 4.125-.386 6.042-1.208 2.373-1.017 3.85-2.306 4.73-3.615.877-1.307 1.11-2.562 1.11-3.437v-5.361l1.978-.818c2.676-1.108 2.676-4.899 0-6.006L25.22 10.16c-.796-.33-1.69-.33-2.485 0L8.945 15.868c-2.677 1.107-2.677 4.898 0 6.006l2.327.963c.638.263 1.369-.04 1.633-.678.264-.637-.04-1.369-.677-1.633L9.9 19.564c-.618-.256-.618-1.13 0-1.386zm.893 6.837c.604-.336.821-1.097.486-1.7-.335-.604-1.096-.821-1.7-.486L16.574 20.9c-1.667.925-2.7 2.682-2.7 4.589v8.772c-.642 1.346-1.244 2.953-1.358 4.328q-.015.142-.016.288c0 1.45 1.248 2.624 2.787 2.624 1.54 0 2.788-1.175 2.788-2.624q0-.18-.025-.355c-.154-1.46-.91-3.168-1.677-4.547v-8.486c0-.999.542-1.919 1.415-2.404z"
        })
    });
const $75ffe786800e58ca$var$ForwardRef = /*#__PURE__*/ (0, $fejKo$react.forwardRef)($75ffe786800e58ca$var$SvgComponent);
var $75ffe786800e58ca$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($75ffe786800e58ca$var$ForwardRef);
