// @react-spectrum/s2/illustrations/linear/internal/S2_lin_tablet_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $kgIl0$reactjsxruntime = require("react/jsx-runtime");
var $kgIl0$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $292d0e2f24a43e69$export$2e2bcd8739ae039);




const $292d0e2f24a43e69$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kgIl0$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kgIl0$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M6.25 13c0-1.519 1.231-2.75 2.75-2.75h30c1.519 0 2.75 1.231 2.75 2.75v22c0 1.519-1.231 2.75-2.75 2.75H9c-1.519 0-2.75-1.231-2.75-2.75zM9 7.75c-2.9 0-5.25 2.35-5.25 5.25v22c0 2.9 2.35 5.25 5.25 5.25h30c2.9 0 5.25-2.35 5.25-5.25V13c0-2.9-2.35-5.25-5.25-5.25zM12.5 24c0 .828-.672 1.5-1.5 1.5s-1.5-.672-1.5-1.5.672-1.5 1.5-1.5 1.5.672 1.5 1.5"
        })
    });
const $292d0e2f24a43e69$var$ForwardRef = /*#__PURE__*/ (0, $kgIl0$react.forwardRef)($292d0e2f24a43e69$var$SvgComponent);
var $292d0e2f24a43e69$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($292d0e2f24a43e69$var$ForwardRef);
