// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noReviewLinks_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1AtqL$jsx} from "react/jsx-runtime";
import {forwardRef as $1AtqL$forwardRef} from "react";





const $4f997361a833b957$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1AtqL$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1AtqL$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M14 18c0-4.41 3.571-8 7.99-8h19.45c2.047 0 4.036.783 5.526 2.218l.002.002 19.565 18.745C68.106 32.472 69 34.553 69 36.74v14.75c0 1.105-.895 2-2 2s-2-.895-2-2V36.74c0-1.092-.446-2.131-1.233-2.886l-19.57-18.75-.005-.004c-.73-.704-1.72-1.1-2.752-1.1H21.99C19.789 14 18 15.79 18 18v47.99c0 2.21 1.789 4 3.99 4H46c1.105 0 2 .895 2 2s-.895 2-2 2H21.99c-4.419 0-7.99-3.59-7.99-8zm67.49 53.74c0 8.146-6.604 14.75-14.75 14.75s-14.75-6.604-14.75-14.75 6.604-14.75 14.75-14.75 14.75 6.604 14.75 14.75m-14.502-8.75c1.105 0 2 .895 2 2v5h5c1.105 0 2 .895 2 2s-.895 2-2 2h-5v5c0 1.104-.895 2-2 2s-2-.896-2-2v-5h-5c-1.104 0-2-.896-2-2s.896-2 2-2h5v-5c0-1.105.896-2 2-2m-16.22-21.018c1.028-1.026 1.028-2.685 0-3.71-1.032-1.029-2.707-1.029-3.739 0l-2.669 2.66c-.782.78-2.048.778-2.828-.004s-.778-2.048.004-2.828l2.67-2.661c2.592-2.585 6.793-2.585 9.386 0 2.596 2.588 2.596 6.787 0 9.375l-6.768 6.748c-2.287 2.28-5.82 2.547-8.4.81-.917-.616-1.16-1.858-.544-2.775s1.859-1.16 2.775-.544c1.028.691 2.438.58 3.345-.324zM30.771 53.518c-1.028 1.026-1.028 2.685 0 3.71 1.032 1.029 2.707 1.029 3.739 0l2.669-2.66c.782-.78 2.049-.779 2.828.004.78.782.778 2.048-.004 2.828l-2.669 2.661c-2.593 2.585-6.794 2.585-9.387 0-2.596-2.588-2.596-6.787 0-9.375l6.768-6.748c2.287-2.28 5.82-2.547 8.4-.81.917.616 1.16 1.858.544 2.775s-1.859 1.16-2.775.544c-1.028-.691-2.438-.58-3.345.324z"
        })
    });
const $4f997361a833b957$var$ForwardRef = /*#__PURE__*/ (0, $1AtqL$forwardRef)($4f997361a833b957$var$SvgComponent);
var $4f997361a833b957$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($4f997361a833b957$var$ForwardRef);


export {$4f997361a833b957$export$2e2bcd8739ae039 as default};
