// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowDown_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $DS322$reactjsxruntime = require("react/jsx-runtime");
var $DS322$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c7273006c405cd8b$export$2e2bcd8739ae039);




const $c7273006c405cd8b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $DS322$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $DS322$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M50 15c0-1.105-.895-2-2-2s-2 .895-2 2v63q0 .09.008.18L35.414 67.586c-.78-.781-2.047-.781-2.828 0-.781.78-.781 2.047 0 2.828l14 14c.78.781 2.047.781 2.828 0l14-14c.781-.78.781-2.047 0-2.828-.78-.781-2.047-.781-2.828 0L49.992 78.18Q50 78.09 50 78zm-1.82 64.992Q48.09 80 48 80t-.18-.008l.18.18z"
        })
    });
const $c7273006c405cd8b$var$ForwardRef = /*#__PURE__*/ (0, $DS322$react.forwardRef)($c7273006c405cd8b$var$SvgComponent);
var $c7273006c405cd8b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c7273006c405cd8b$var$ForwardRef);
