// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cloudStateError_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $3vQgJ$jsx} from "react/jsx-runtime";
import {forwardRef as $3vQgJ$forwardRef} from "react";





const $c14e05328e4ffc5b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3vQgJ$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3vQgJ$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25.53 8.19c-4.164 0-7.703 2.57-9.317 6.209-.563-.136-1.151-.21-1.753-.21-4.37 0-7.82 3.682-7.82 8.11q0 .376.034.744C4.28 24.183 2.65 26.675 2.65 29.54c0 3.863 2.98 7.087 6.79 7.15H27.2c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H9.471c-2.345-.044-4.32-2.057-4.32-4.65 0-2.177 1.404-3.957 3.255-4.49.634-.181 1.017-.825.875-1.47-.091-.415-.14-.845-.14-1.28 0-3.152 2.43-5.61 5.32-5.61.7 0 1.382.15 2.007.418.326.14.695.135 1.017-.014.321-.148.565-.427.67-.765 1.022-3.304 3.961-5.64 7.375-5.64 4.23 0 7.75 3.599 7.75 8.15v.2c0 .662.515 1.208 1.175 1.248 1.662.1 3.16.834 4.272 1.997.477.498 1.269.516 1.767.039.5-.477.517-1.269.04-1.768-1.273-1.33-2.931-2.265-4.79-2.62-.441-5.415-4.801-9.746-10.214-9.746m3.41 22.8c0-4.18 3.39-7.57 7.57-7.57s7.57 3.39 7.57 7.57-3.39 7.57-7.57 7.57-7.57-3.39-7.57-7.57m7.57-4.84c-.69 0-1.25.56-1.25 1.25v3.75c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V27.4c0-.69-.56-1.25-1.25-1.25m1.29 8.95c.04-.09.08-.18.08-.27q.03-.135.03-.27t-.03-.27c-.01-.09-.04-.18-.08-.27-.03-.08-.07-.16-.13-.24-.05-.07-.11-.14-.17-.21-.06-.06-.14-.12-.21-.17-.08-.05-.16-.1-.24-.13-.09-.04-.18-.07-.26-.08-.18-.04-.37-.04-.55 0-.09.01-.18.04-.26.08-.09.03-.17.08-.24.13-.08.05-.15.11-.22.17-.06.07-.12.14-.17.21-.05.08-.1.16-.13.24-.04.09-.06.18-.08.27q-.03.135-.03.27t.03.27c.02.09.04.18.08.27.03.08.08.16.13.24.05.07.11.14.17.21.07.06.14.12.22.17.07.05.15.1.24.13.08.04.17.06.26.08.09.03.18.03.27.03s.19-.01.28-.03l.027-.007c.073-.018.153-.038.233-.074.08-.03.16-.08.24-.13.07-.05.15-.11.21-.17.06-.07.12-.14.17-.21.06-.08.1-.16.13-.24"
        })
    });
const $c14e05328e4ffc5b$var$ForwardRef = /*#__PURE__*/ (0, $3vQgJ$forwardRef)($c14e05328e4ffc5b$var$SvgComponent);
var $c14e05328e4ffc5b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c14e05328e4ffc5b$var$ForwardRef);


export {$c14e05328e4ffc5b$export$2e2bcd8739ae039 as default};
