// @react-spectrum/s2/illustrations/linear/internal/S2_lin_dropToUpload_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $8fa7g$reactjsxruntime = require("react/jsx-runtime");
var $8fa7g$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $693e7f9010f211e0$export$2e2bcd8739ae039);




const $693e7f9010f211e0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8fa7g$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8fa7g$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M43 36.74c0-7.18 5.82-13 13-13h23.272c3.448 0 6.754 1.37 9.192 3.808l24.728 24.728C115.63 54.714 117 58.02 117 61.468V96.74c0 7.18-5.82 13-13 13H56c-7.18 0-13-5.82-13-13zm13-5c-2.761 0-5 2.239-5 5v60c0 2.762 2.239 5 5 5h48c2.761 0 5-2.238 5-5V61.468c0-1.326-.527-2.598-1.464-3.535L82.808 33.205c-.938-.938-2.21-1.465-3.536-1.465zm5.704 41.827c1.375-1.729 3.891-2.016 5.62-.64l7.692 6.117.057-25.645c.005-2.21 1.8-3.996 4.009-3.992s3.996 1.8 3.99 4.01l-.057 25.693 7.64-6.165c1.719-1.388 4.237-1.12 5.625.6 1.387 1.719 1.118 4.237-.6 5.625L81.615 90.52c-1.457 1.176-3.536 1.184-5.002.018l-14.27-11.35c-1.729-1.376-2.016-3.892-.64-5.62M28 76.74c-2.21 0-4 1.791-4 4v3.375c0 2.21-1.79 4-4 4s-4-1.79-4-4V80.74c0-6.627 5.373-12 12-12h4.5c2.21 0 4 1.791 4 4s-1.79 4-4 4zm95.5-4c0-2.209 1.791-4 4-4h4.5c6.627 0 12 5.373 12 12v3.375c0 2.21-1.791 4-4 4s-4-1.79-4-4V80.74c0-2.209-1.791-4-4-4h-4.5c-2.209 0-4-1.79-4-4M20 93.865c2.21 0 4 1.791 4 4v8.75c0 2.209-1.79 4-4 4s-4-1.791-4-4v-8.75c0-2.209 1.79-4 4-4m120 0c2.209 0 4 1.791 4 4v8.75c0 2.209-1.791 4-4 4s-4-1.791-4-4v-8.75c0-2.209 1.791-4 4-4m-120 22.5c2.21 0 4 1.791 4 4v3.375c0 2.209 1.79 4 4 4h4.5c2.21 0 4 1.791 4 4s-1.79 4-4 4H28c-6.627 0-12-5.372-12-12v-3.375c0-2.209 1.79-4 4-4m120 0c2.209 0 4 1.791 4 4v3.375c0 6.628-5.373 12-12 12h-4.5c-2.209 0-4-1.791-4-4s1.791-4 4-4h4.5c2.209 0 4-1.791 4-4v-3.375c0-2.209 1.791-4 4-4m-37 11.375c-2.209 0-4 1.791-4 4s1.791 4 4 4h11c2.209 0 4-1.791 4-4s-1.791-4-4-4zm-32.5 4c0-2.209 1.79-4 4-4h11c2.21 0 4 1.791 4 4s-1.79 4-4 4h-11c-2.21 0-4-1.791-4-4m-24.5-4c-2.21 0-4 1.791-4 4s1.79 4 4 4h11c2.21 0 4-1.791 4-4s-1.79-4-4-4z"
        })
    });
const $693e7f9010f211e0$var$ForwardRef = /*#__PURE__*/ (0, $8fa7g$react.forwardRef)($693e7f9010f211e0$var$SvgComponent);
var $693e7f9010f211e0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($693e7f9010f211e0$var$ForwardRef);
