// @react-spectrum/s2/illustrations/linear/internal/S2_lin_mailClose_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $awinO$jsx} from "react/jsx-runtime";
import {forwardRef as $awinO$forwardRef} from "react";





const $e0ce3d84129f45a3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $awinO$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $awinO$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.82 23.204C17.842 22.436 19.054 22 20.309 22h55.384c1.394 0 2.735.538 3.819 1.47L49.629 40.992c-1.179.691-2.628.678-3.794-.036zm-2.356 3.247c-.298.826-.464 1.74-.464 2.716v36.666C14 69.981 17.005 73 20.308 73h55.384C78.995 73 82 69.981 82 65.833V29.167c0-.824-.118-1.603-.334-2.323L51.652 44.442c-2.452 1.437-5.48 1.408-7.905-.075zM20.308 18C14.434 18 10 23.19 10 29.167v36.666C10 71.811 14.434 77 20.308 77h55.384C81.566 77 86 71.81 86 65.833V29.167C86 23.189 81.566 18 75.692 18z"
        })
    });
const $e0ce3d84129f45a3$var$ForwardRef = /*#__PURE__*/ (0, $awinO$forwardRef)($e0ce3d84129f45a3$var$SvgComponent);
var $e0ce3d84129f45a3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e0ce3d84129f45a3$var$ForwardRef);


export {$e0ce3d84129f45a3$export$2e2bcd8739ae039 as default};
