// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noReviewLinks_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5w3ln$jsx} from "react/jsx-runtime";
import {forwardRef as $5w3ln$forwardRef} from "react";





const $dfc0c6b712ed2a79$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5w3ln$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5w3ln$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M6.75 9c0-2.343 1.897-4.25 4.245-4.25h9.725c1.087 0 2.144.416 2.936 1.179l.002.001 9.782 9.372c.835.8 1.31 1.906 1.31 3.068v7.375c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25V18.37c0-.478-.195-.932-.54-1.262l-9.785-9.375-.003-.003c-.317-.306-.75-.48-1.202-.48h-9.725c-.962 0-1.745.783-1.745 1.75v23.995c0 .967.783 1.75 1.745 1.75H23c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H10.995c-2.348 0-4.245-1.907-4.245-4.25zm33.994 26.87c0 4.073-3.302 7.375-7.375 7.375s-7.375-3.302-7.375-7.375 3.302-7.375 7.375-7.375 7.375 3.302 7.375 7.375m-7.25-4.625c.69 0 1.25.56 1.25 1.25v2.25h2.25c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-2.25v2.25c0 .69-.56 1.25-1.25 1.25-.691 0-1.25-.56-1.25-1.25v-2.25h-2.25c-.691 0-1.25-.56-1.25-1.25s.559-1.25 1.25-1.25h2.25v-2.25c0-.69.559-1.25 1.25-1.25m-8.287-11.937c.417.415.417 1.086 0 1.5l-3.384 3.374c-.345.345-.873.407-1.282.178-.603-.338-1.364-.122-1.701.48-.337.603-.122 1.364.48 1.701 1.358.76 3.11.567 4.268-.588l3.384-3.374c1.397-1.392 1.397-3.65 0-5.042-1.394-1.39-3.652-1.39-5.046 0l-1.192 1.189c-.49.487-.49 1.279-.003 1.768s1.279.49 1.768.002l1.192-1.188c.418-.417 1.098-.417 1.516 0m-2.866 3.92c.355-.592.165-1.36-.427-1.716-1.38-.83-3.206-.662-4.407.52l-3.426 3.374c-1.415 1.393-1.415 3.66 0 5.052 1.404 1.383 3.672 1.383 5.076 0l1.235-1.215c.492-.484.498-1.275.013-1.767-.484-.492-1.275-.499-1.767-.014l-1.235 1.215c-.43.424-1.137.424-1.568 0-.42-.414-.42-1.076 0-1.49l3.426-3.374c.365-.359.935-.417 1.364-.159.592.356 1.36.165 1.716-.426"
        })
    });
const $dfc0c6b712ed2a79$var$ForwardRef = /*#__PURE__*/ (0, $5w3ln$forwardRef)($dfc0c6b712ed2a79$var$SvgComponent);
var $dfc0c6b712ed2a79$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($dfc0c6b712ed2a79$var$ForwardRef);


export {$dfc0c6b712ed2a79$export$2e2bcd8739ae039 as default};
