// @react-spectrum/s2/illustrations/linear/internal/S2_lin_close_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $jPzaD$reactjsxruntime = require("react/jsx-runtime");
var $jPzaD$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e3d557c089ee1a2c$export$2e2bcd8739ae039);




const $e3d557c089ee1a2c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jPzaD$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jPzaD$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 17c-17.12 0-31 13.88-31 31s13.88 31 31 31 31-13.88 31-31-13.88-31-31-31M13 48c0-19.33 15.67-35 35-35s35 15.67 35 35-15.67 35-35 35-35-15.67-35-35m24.43-10.571c.78-.782 2.047-.782 2.828 0l8.11 8.11 8.11-8.11c.781-.782 2.047-.782 2.828 0 .782.78.782 2.047 0 2.828l-8.11 8.11 8.11 8.11c.782.781.782 2.047 0 2.829-.78.78-2.047.78-2.828 0l-8.11-8.11-8.11 8.11c-.781.78-2.047.78-2.829 0-.78-.782-.78-2.048 0-2.829l8.11-8.11-8.11-8.11c-.78-.781-.78-2.047 0-2.828"
        })
    });
const $e3d557c089ee1a2c$var$ForwardRef = /*#__PURE__*/ (0, $jPzaD$react.forwardRef)($e3d557c089ee1a2c$var$SvgComponent);
var $e3d557c089ee1a2c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e3d557c089ee1a2c$var$ForwardRef);
