// @react-spectrum/s2/illustrations/linear/internal/S2_lin_buildTable_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $12PXx$jsx} from "react/jsx-runtime";
import {forwardRef as $12PXx$forwardRef} from "react";





const $b853497cae00005a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $12PXx$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $12PXx$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M35.95 37.91c-6.632 0-12 5.37-12 12v4.08h112.019v-4.08c0-6.63-5.369-12-12-12zm-12 72.12V61.99H40v60.04h-4.05c-6.632 0-12-5.369-12-12m73.33 12H48V61.99h87.969v21.65c0 2.21 1.791 4 4 4s4-1.79 4-4V49.91c0-11.049-8.951-20-20-20h-88.02c-11.049 0-20 8.951-20 20v60.12c0 11.049 8.951 20 20 20h61.33c2.21 0 4-1.791 4-4s-1.79-4-4-4m10.099-4.05c0-13.55 10.98-24.53 24.53-24.53s24.53 10.98 24.53 24.53-10.98 24.53-24.53 24.53-24.53-10.98-24.53-24.53m28.54 3.96h7.88c2.21 0 4-1.79 4-4s-1.79-4-4-4h-7.88v-7.88c0-2.21-1.79-4-4-4s-4 1.79-4 4v7.88h-7.88c-2.21 0-4 1.79-4 4s1.79 4 4 4h7.88v7.88c0 2.21 1.79 4 4 4s4-1.79 4-4z"
        })
    });
const $b853497cae00005a$var$ForwardRef = /*#__PURE__*/ (0, $12PXx$forwardRef)($b853497cae00005a$var$SvgComponent);
var $b853497cae00005a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b853497cae00005a$var$ForwardRef);


export {$b853497cae00005a$export$2e2bcd8739ae039 as default};
