// @react-spectrum/s2/illustrations/linear/internal/S2_lin_rotateCW_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $9OUr8$reactjsxruntime = require("react/jsx-runtime");
var $9OUr8$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0940bbc913f47493$export$2e2bcd8739ae039);




const $0940bbc913f47493$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9OUr8$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9OUr8$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.31 48.996c0-17.12 13.88-31 31-31 12.19 0 22.739 7.035 27.803 17.271l-9.734-3.13c-1.052-.338-2.178.24-2.516 1.292-.338 1.051.24 2.178 1.292 2.516l14.056 4.52c1.052.338 2.178-.24 2.516-1.292l4.52-14.056c.338-1.052-.24-2.178-1.292-2.517-1.051-.338-2.178.24-2.516 1.292l-2.952 9.181C72.693 21.75 60.909 13.996 47.311 13.996c-19.33 0-35 15.67-35 35s15.67 35 35 35c15.117 0 27.991-9.583 32.888-23 .379-1.038-.156-2.186-1.193-2.564-1.038-.38-2.186.155-2.565 1.193-4.338 11.888-15.746 20.371-29.13 20.371-17.121 0-31-13.88-31-31"
        })
    });
const $0940bbc913f47493$var$ForwardRef = /*#__PURE__*/ (0, $9OUr8$react.forwardRef)($0940bbc913f47493$var$SvgComponent);
var $0940bbc913f47493$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0940bbc913f47493$var$ForwardRef);
