// @react-spectrum/s2/illustrations/linear/internal/S2_lin_stampClone_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $en0Ky$jsx} from "react/jsx-runtime";
import {forwardRef as $en0Ky$forwardRef} from "react";





const $27ea53c9cc8c2508$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $en0Ky$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $en0Ky$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 9.66c-2.59 0-4.69 2.1-4.69 4.69 0 1.887 1.115 3.516 2.727 4.26q.085.039.16.088c.461.262.721.802.604 1.349l-1.469 6.86c-.123.576-.633.988-1.222.988h-6.493c-1.519 0-2.75 1.231-2.75 2.75v1.695h26.267v-1.695c0-1.519-1.231-2.75-2.75-2.75H27.86c-.598 0-1.112-.424-1.227-1.011l-1.336-6.86c-.088-.45.078-.89.396-1.173q.158-.181.385-.296c1.55-.768 2.612-2.364 2.612-4.205 0-2.59-2.1-4.69-4.69-4.69m3.914 10.722c1.97-1.281 3.276-3.503 3.276-6.032 0-3.971-3.219-7.19-7.19-7.19-3.97 0-7.19 3.219-7.19 7.19 0 2.56 1.34 4.807 3.352 6.08L19.1 25.395h-5.482c-2.9 0-5.25 2.35-5.25 5.25v2.945c0 .69.56 1.25 1.25 1.25h28.767c.69 0 1.25-.56 1.25-1.25v-2.945c0-2.9-2.35-5.25-5.25-5.25H28.89zM10.85 39.59c0-.69.56-1.25 1.25-1.25h23.8c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H12.1c-.69 0-1.25-.56-1.25-1.25"
        })
    });
const $27ea53c9cc8c2508$var$ForwardRef = /*#__PURE__*/ (0, $en0Ky$forwardRef)($27ea53c9cc8c2508$var$SvgComponent);
var $27ea53c9cc8c2508$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($27ea53c9cc8c2508$var$ForwardRef);


export {$27ea53c9cc8c2508$export$2e2bcd8739ae039 as default};
