// @react-spectrum/s2/illustrations/linear/internal/S2_lin_buildings_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $gKREh$jsx} from "react/jsx-runtime";
import {forwardRef as $gKREh$forwardRef} from "react";





const $45c0c82707004262$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gKREh$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gKREh$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M21.32 19.96c-3.315 0-6 2.685-6 6V75.9h10.96v-8.45c0-4.414 3.586-8 8-8h4.17c4.415 0 8 3.586 8 8v8.45h10.96V25.96c0-3.315-2.684-6-6-6zm-10 6c0-5.524 4.476-10 10-10h30.09c5.525 0 10 4.476 10 10V77.9c0 1.105-.895 2-2 2H13.32c-1.104 0-2-.895-2-2zm53.02 4.441c0-1.105.895-2 2-2h8.34c5.524 0 10 4.475 10 10v39.5c0 1.105-.896 2-2 2H66.34c-1.105 0-2-.895-2-2s.895-2 2-2h14.34v-37.5c0-3.315-2.685-6-6-6h-8.34c-1.105 0-2-.895-2-2m-34.06 37.05c0-2.205 1.795-4 4-4h4.17c2.206 0 4 1.795 4 4v8.45H30.28zm-.09-41.01h-3.92c-1.105 0-2 .895-2 2v3.92c0 1.105.895 2 2 2h3.92c1.104 0 2-.895 2-2v-3.92c0-1.105-.896-2-2-2m12.35 0h3.92c1.104 0 2 .895 2 2v3.92c0 1.105-.896 2-2 2h-3.92c-1.105 0-2-.895-2-2v-3.92c0-1.105.895-2 2-2M30.19 42.59h-3.92c-1.105 0-2 .895-2 2v3.92c0 1.104.895 2 2 2h3.92c1.104 0 2-.896 2-2v-3.92c0-1.105-.896-2-2-2m12.35 0h3.92c1.104 0 2 .895 2 2v3.92c0 1.104-.896 2-2 2h-3.92c-1.105 0-2-.896-2-2v-3.92c0-1.105.895-2 2-2M73 42.12h-3.92c-1.105 0-2 .895-2 2v3.92c0 1.105.895 2 2 2H73c1.105 0 2-.895 2-2v-3.92c0-1.105-.895-2-2-2m-3.92 16.15H73c1.105 0 2 .895 2 2v3.92c0 1.105-.895 2-2 2h-3.92c-1.105 0-2-.895-2-2v-3.92c0-1.105.895-2 2-2"
        })
    });
const $45c0c82707004262$var$ForwardRef = /*#__PURE__*/ (0, $gKREh$forwardRef)($45c0c82707004262$var$SvgComponent);
var $45c0c82707004262$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($45c0c82707004262$var$ForwardRef);


export {$45c0c82707004262$export$2e2bcd8739ae039 as default};
