// @react-spectrum/s2/illustrations/linear/internal/S2_lin_calendar_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $gMmEe$jsx} from "react/jsx-runtime";
import {forwardRef as $gMmEe$forwardRef} from "react";





const $cfaced88bf303cb5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gMmEe$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gMmEe$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M17 6.75c.69 0 1.25.56 1.25 1.25v1.75h11.5V8c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v1.75H35c2.9 0 5.25 2.35 5.25 5.25v20c0 2.9-2.35 5.25-5.25 5.25H13c-2.9 0-5.25-2.35-5.25-5.25V15c0-2.9 2.35-5.25 5.25-5.25h2.75V8c0-.69.56-1.25 1.25-1.25m12.75 5.5V14c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-1.75H35c1.519 0 2.75 1.231 2.75 2.75v3.75h-27.5V15c0-1.519 1.231-2.75 2.75-2.75h2.75V14c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-1.75zm-19.5 9V35c0 1.519 1.231 2.75 2.75 2.75h22c1.519 0 2.75-1.231 2.75-2.75V21.25zM15 24h2c.552 0 1 .448 1 1v2c0 .552-.448 1-1 1h-2c-.552 0-1-.448-1-1v-2c0-.552.448-1 1-1m10 0h-2c-.552 0-1 .448-1 1v2c0 .552.448 1 1 1h2c.552 0 1-.448 1-1v-2c0-.552-.448-1-1-1m6 0h2c.552 0 1 .448 1 1v2c0 .552-.448 1-1 1h-2c-.552 0-1-.448-1-1v-2c0-.552.448-1 1-1m-14 7h-2c-.552 0-1 .448-1 1v2c0 .552.448 1 1 1h2c.552 0 1-.448 1-1v-2c0-.552-.448-1-1-1m6 0h2c.552 0 1 .448 1 1v2c0 .552-.448 1-1 1h-2c-.552 0-1-.448-1-1v-2c0-.552.448-1 1-1m10 0h-2c-.552 0-1 .448-1 1v2c0 .552.448 1 1 1h2c.552 0 1-.448 1-1v-2c0-.552-.448-1-1-1"
        })
    });
const $cfaced88bf303cb5$var$ForwardRef = /*#__PURE__*/ (0, $gMmEe$forwardRef)($cfaced88bf303cb5$var$SvgComponent);
var $cfaced88bf303cb5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cfaced88bf303cb5$var$ForwardRef);


export {$cfaced88bf303cb5$export$2e2bcd8739ae039 as default};
