// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cloudStateError_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5RRQ1$jsx} from "react/jsx-runtime";
import {forwardRef as $5RRQ1$forwardRef} from "react";





const $aebb551f4e8d726e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5RRQ1$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5RRQ1$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M50.97 17.7c-8.029 0-14.862 5.03-17.857 12.148-1.17-.312-2.397-.479-3.664-.479-8.236 0-14.76 6.95-14.76 15.33q.001.87.095 1.705c-4.644 2.079-7.835 6.857-7.835 12.365 0 7.284 5.62 13.34 12.767 13.46h34.883c1.105 0 2-.895 2-2s-.895-2-2-2H19.767c-4.806-.089-8.818-4.21-8.818-9.46 0-4.424 2.854-8.06 6.63-9.137 1.02-.291 1.636-1.33 1.4-2.365-.188-.832-.29-1.684-.29-2.568 0-6.339 4.897-11.33 10.76-11.33 1.449 0 2.818.3 4.075.845.522.227 1.116.22 1.633-.018.516-.239.907-.686 1.074-1.23 2.024-6.592 7.9-11.267 14.738-11.267 8.474 0 15.49 7.202 15.49 16.27v.39c0 1.06.826 1.935 1.883 1.997 3.514.206 6.67 1.85 8.954 4.413.734.825 1.999.898 2.823.164.825-.735.898-2 .164-2.824-2.544-2.856-5.978-4.87-9.87-5.536-.687-10.472-9.056-18.874-19.444-18.874m6.629 43.87c0-8.13 6.59-14.72 14.72-14.72s14.72 6.59 14.72 14.72-6.59 14.72-14.72 14.72-14.72-6.59-14.72-14.72m14.72-8.96c-1.1 0-2 .9-2 2v7.68c0 1.1.9 2 2 2s2-.9 2-2v-7.68c0-1.1-.9-2-2-2m1.98 16.74c.06-.13.1-.26.13-.4.02-.14.04-.28.04-.42s-.02-.28-.04-.42c-.03-.14-.07-.27-.13-.4-.05-.13-.12-.26-.19-.38-.08-.11-.17-.22-.27-.32q-.15-.15-.33-.27c-.11-.08-.24-.14-.37-.2-.13-.05-.26-.09-.4-.12q-.42-.075-.84 0c-.14.03-.27.07-.4.12-.13.06-.26.12-.37.2q-.18.12-.33.27c-.1.1-.19.21-.27.32-.08.12-.14.25-.2.38-.05.13-.09.26-.12.4-.04.14-.04.28-.04.42s.01.28.04.42.07.27.12.4c.06.13.12.26.2.37q.12.18.27.33t.33.27c.11.07.24.14.37.19.13.06.26.1.4.13.14.04.28.04.42.04s.28-.01.42-.04.27-.07.4-.13c.13-.05.26-.12.37-.19q.18-.12.33-.27t.27-.33c.07-.11.14-.24.19-.37"
        })
    });
const $aebb551f4e8d726e$var$ForwardRef = /*#__PURE__*/ (0, $5RRQ1$forwardRef)($aebb551f4e8d726e$var$SvgComponent);
var $aebb551f4e8d726e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($aebb551f4e8d726e$var$ForwardRef);


export {$aebb551f4e8d726e$export$2e2bcd8739ae039 as default};
