// @react-spectrum/s2/illustrations/linear/internal/S2_lin_globeGrid_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $dC5J3$jsx} from "react/jsx-runtime";
import {forwardRef as $dC5J3$forwardRef} from "react";





const $16ab3b0cd4dc117b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dC5J3$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dC5J3$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M40.119 18.025c-6.962 1.825-12.967 6.01-17.11 11.65h10.55c1.618-4.011 3.801-7.917 6.56-11.65M17.057 46.35c.238-4.555 1.461-8.85 3.459-12.675H32.14c-1.275 4.163-1.963 8.413-2.076 12.675zm17.01 0c.125-4.27.88-8.524 2.278-12.675h9.623V46.35zm15.9 0V33.676h9.686c1.398 4.151 2.153 8.404 2.278 12.675zm0-16.675h8.126c-1.93-4.341-4.593-8.537-7.999-12.487l-.09-.111-.036-.003zM65.934 46.35c-.113-4.262-.8-8.512-2.076-12.675h11.625c1.998 3.825 3.22 8.12 3.46 12.675zM62.44 29.676H72.99c-4.143-5.64-10.148-9.826-17.11-11.651 2.76 3.733 4.943 7.64 6.561 11.65m-24.535 0h8.063V17.112q-.03.04-.064.077c-3.405 3.95-6.068 8.146-7.999 12.487m8.063 20.675H34.135c.267 4.261 1.158 8.515 2.663 12.678h9.17zm0 16.678h-7.522c1.893 4.083 4.393 8.05 7.49 11.819q.015.02.032.04zm-13.4-4c-1.377-4.168-2.194-8.418-2.44-12.678H17.102c.343 4.575 1.68 8.872 3.794 12.678zm-9.024 4h10.53c1.594 3.768 3.657 7.447 6.18 10.98-6.725-1.73-12.566-5.662-16.71-10.98m26.424 0h7.585c-1.894 4.083-4.394 8.05-7.49 11.819l-.057.073-.038.002zm9.233-4h-9.233V50.35h11.896c-.267 4.261-1.158 8.515-2.663 12.678m2.724 4c-1.594 3.768-3.657 7.447-6.18 10.98 6.724-1.731 12.565-5.662 16.71-10.98zm13.177-4H63.43c1.377-4.168 2.194-8.418 2.44-12.678h13.027c-.344 4.575-1.68 8.872-3.795 12.678m-62.088-15.03c0-19.322 15.663-34.985 34.985-34.985s34.985 15.663 34.985 34.985S67.322 82.984 48 82.984 13.014 67.322 13.014 48"
        })
    });
const $16ab3b0cd4dc117b$var$ForwardRef = /*#__PURE__*/ (0, $dC5J3$forwardRef)($16ab3b0cd4dc117b$var$SvgComponent);
var $16ab3b0cd4dc117b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($16ab3b0cd4dc117b$var$ForwardRef);


export {$16ab3b0cd4dc117b$export$2e2bcd8739ae039 as default};
