// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowDown_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $b08UF$reactjsxruntime = require("react/jsx-runtime");
var $b08UF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4d0a29b0ea0ca1bb$export$2e2bcd8739ae039);




const $4d0a29b0ea0ca1bb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $b08UF$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $b08UF$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25.25 6.5c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v30.982l-4.866-4.866c-.488-.488-1.28-.488-1.768 0s-.488 1.28 0 1.768l7 7c.488.488 1.28.488 1.768 0l7-7c.488-.488.488-1.28 0-1.768s-1.28-.488-1.768 0l-4.866 4.866z"
        })
    });
const $4d0a29b0ea0ca1bb$var$ForwardRef = /*#__PURE__*/ (0, $b08UF$react.forwardRef)($4d0a29b0ea0ca1bb$var$SvgComponent);
var $4d0a29b0ea0ca1bb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4d0a29b0ea0ca1bb$var$ForwardRef);
