// @react-spectrum/s2/illustrations/linear/internal/S2_lin_microphone_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $iUUm5$reactjsxruntime = require("react/jsx-runtime");
var $iUUm5$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $bc9d549cc1109549$export$2e2bcd8739ae039);




const $bc9d549cc1109549$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iUUm5$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $iUUm5$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 5.75c-4.004 0-7.25 3.246-7.25 7.25v8c0 4.004 3.246 7.25 7.25 7.25s7.25-3.246 7.25-7.25v-8c0-4.004-3.246-7.25-7.25-7.25M19.25 13c0-2.623 2.127-4.75 4.75-4.75s4.75 2.127 4.75 4.75v8c0 2.623-2.127 4.75-4.75 4.75s-4.75-2.127-4.75-4.75zm3.5 21.192c-6.732-.63-12-6.296-12-13.192v-1.5c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25V21c0 5.937 4.813 10.75 10.75 10.75S34.75 26.937 34.75 21v-1.5c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25V21c0 6.896-5.268 12.562-12 13.192v3.558H31c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H17c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h5.75z"
        })
    });
const $bc9d549cc1109549$var$ForwardRef = /*#__PURE__*/ (0, $iUUm5$react.forwardRef)($bc9d549cc1109549$var$SvgComponent);
var $bc9d549cc1109549$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($bc9d549cc1109549$var$ForwardRef);
