// @react-spectrum/s2/illustrations/linear/internal/S2_lin_volumeOne_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6BQ7S$jsx} from "react/jsx-runtime";
import {forwardRef as $6BQ7S$forwardRef} from "react";





const $72ea22901aec7dab$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6BQ7S$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6BQ7S$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24.92 8.777c1.735-1.72 4.685-.491 4.685 1.953V37.27c0 2.437-2.936 3.67-4.676 1.962l-5.837-5.73c-.14-.137-.33-.214-.526-.214h-4.195c-1.795 0-3.25-1.455-3.25-3.25V18.026c0-1.795 1.455-3.25 3.25-3.25h4.19c.197 0 .387-.079.527-.218zm1.888 1.703c-.034.006-.077.021-.128.073l-5.778 5.727v15.496l5.778 5.671c.052.05.095.066.129.072q.062.011.143-.02c.053-.022.091-.055.115-.089.02-.027.038-.069.038-.141V10.73c0-.073-.019-.114-.038-.142-.024-.033-.063-.067-.115-.089s-.104-.026-.144-.019m-8.406 20.307V17.276h-4.03c-.415 0-.75.335-.75.75v12.01c0 .415.335.75.75.75zm13.242-13.551c.415-.552 1.199-.663 1.75-.248 2.115 1.592 3.487 4.127 3.487 6.982 0 2.88-1.394 5.432-3.539 7.021-.554.411-1.337.295-1.748-.26s-.295-1.337.26-1.748c1.535-1.138 2.527-2.96 2.527-5.013 0-2.036-.975-3.845-2.49-4.984-.551-.415-.662-1.199-.247-1.75"
        })
    });
const $72ea22901aec7dab$var$ForwardRef = /*#__PURE__*/ (0, $6BQ7S$forwardRef)($72ea22901aec7dab$var$SvgComponent);
var $72ea22901aec7dab$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($72ea22901aec7dab$var$ForwardRef);


export {$72ea22901aec7dab$export$2e2bcd8739ae039 as default};
