// @react-spectrum/s2/illustrations/linear/internal/S2_lin_bell_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $js2eZ$reactjsxruntime = require("react/jsx-runtime");
var $js2eZ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f87a350fec723e4d$export$2e2bcd8739ae039);




const $f87a350fec723e4d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $js2eZ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $js2eZ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 12.74c-12.038 0-21.81 9.722-21.81 21.731l.001.043v6.265c-.245 7.508-3.438 14.345-6.633 19.348-3.103 4.858.373 11.213 6.149 11.213H38.8v2.2c0 5.081 4.12 9.2 9.2 9.2 5.081 0 9.2-4.119 9.2-9.2v-2.2h13.093c5.775 0 9.251-6.355 6.149-11.213-3.195-5.003-6.389-11.84-6.633-19.348v-6.308c0-12.009-9.772-21.73-21.81-21.73m5.2 58.6H42.8v2.2c0 2.872 2.328 5.2 5.2 5.2s5.2-2.328 5.2-5.2zM30.19 40.812v-6.17q.003-.048.003-.098l-.001-.07-.001-.025C30.203 24.674 38.164 16.74 48 16.74c9.835 0 17.797 7.934 17.809 17.71l-.001.023-.001.071q0 .05.002.099v6.232c.27 8.53 3.868 16.092 7.262 21.406 1.402 2.194-.162 5.06-2.778 5.06H25.707c-2.616 0-4.18-2.866-2.778-5.06 3.394-5.314 6.992-12.877 7.26-21.406q.002-.032.002-.063"
        })
    });
const $f87a350fec723e4d$var$ForwardRef = /*#__PURE__*/ (0, $js2eZ$react.forwardRef)($f87a350fec723e4d$var$SvgComponent);
var $f87a350fec723e4d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f87a350fec723e4d$var$ForwardRef);
