// @react-spectrum/s2/illustrations/linear/internal/S2_lin_speedFast_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $liWPG$reactjsxruntime = require("react/jsx-runtime");
var $liWPG$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $507dd35c2b511a8e$export$2e2bcd8739ae039);




const $507dd35c2b511a8e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $liWPG$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $liWPG$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8.733 36.923c-1.464-2.528-2.318-5.454-2.362-8.572l-.002-.07v-.185c0-9.744 7.9-17.643 17.643-17.643 4.209 0 8.075 1.475 11.108 3.935.536.435.618 1.222.183 1.758s-1.222.618-1.758.183q-.396-.322-.813-.615l-1.172 2.03c-.345.597-1.11.802-1.707.457-.598-.345-.803-1.11-.458-1.708l1.182-2.047c-1.634-.787-3.43-1.29-5.327-1.443v2.386c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-2.384c-1.88.155-3.661.653-5.283 1.432l1.187 2.056c.345.598.14 1.363-.457 1.708-.598.345-1.363.14-1.708-.458l-1.179-2.041c-1.501 1.056-2.802 2.378-3.834 3.897l1.987 1.148c.598.345.803 1.11.458 1.707-.345.598-1.11.803-1.708.458l-1.966-1.136c-.744 1.619-1.211 3.39-1.34 5.255h2.218c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H8.936c.209 2.223.897 4.303 1.96 6.14.346.597.142 1.362-.455 1.708-.598.346-1.362.142-1.708-.456m31.97-14.561c-.225-.653-.936-1-1.589-.776s-1 .935-.776 1.588c.42 1.221.688 2.515.78 3.857h-2.214c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h2.185c-.21 2.223-.898 4.303-1.96 6.14-.346.597-.142 1.362.455 1.708s1.362.142 1.708-.456c1.453-2.51 2.305-5.411 2.36-8.504q.008-.067.008-.138 0-.056-.004-.11v-.075c0-2.005-.335-3.935-.954-5.734m-.95-6.198c-.585-.586-1.526-.61-2.142-.056L22.416 29.786c-1.12 1.008-1.166 2.75-.1 3.815 1.066 1.066 2.808 1.02 3.816-.1L39.81 18.306c.554-.615.53-1.557-.056-2.142"
        })
    });
const $507dd35c2b511a8e$var$ForwardRef = /*#__PURE__*/ (0, $liWPG$react.forwardRef)($507dd35c2b511a8e$var$SvgComponent);
var $507dd35c2b511a8e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($507dd35c2b511a8e$var$ForwardRef);
