// @react-spectrum/s2/illustrations/linear/internal/S2_lin_desktop_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $hC9uP$jsx} from "react/jsx-runtime";
import {forwardRef as $hC9uP$forwardRef} from "react";





const $d573973e805a3733$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hC9uP$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hC9uP$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20 20c-3.314 0-6 2.686-6 6v28c0 3.314 2.686 6 6 6h56c3.314 0 6-2.686 6-6V26c0-3.314-2.686-6-6-6zm-10 6c0-5.523 4.477-10 10-10h56c5.523 0 10 4.477 10 10v28c0 5.523-4.477 10-10 10H53v12h7c1.105 0 2 .895 2 2s-.895 2-2 2H36c-1.105 0-2-.895-2-2s.895-2 2-2h8V64H20c-5.523 0-10-4.477-10-10zm41 .5c0 1.38-1.12 2.5-2.5 2.5S46 27.88 46 26.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5"
        })
    });
const $d573973e805a3733$var$ForwardRef = /*#__PURE__*/ (0, $hC9uP$forwardRef)($d573973e805a3733$var$SvgComponent);
var $d573973e805a3733$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d573973e805a3733$var$ForwardRef);


export {$d573973e805a3733$export$2e2bcd8739ae039 as default};
