// @react-spectrum/s2/illustrations/linear/internal/S2_lin_globeGrid_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $i6Gm1$reactjsxruntime = require("react/jsx-runtime");
var $i6Gm1$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0936880dcccd78f0$export$2e2bcd8739ae039);




const $0936880dcccd78f0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $i6Gm1$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 49,
        viewBox: "0 0 48 49",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $i6Gm1$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M19.133 9.523c-3.024.983-5.646 2.857-7.556 5.311h4.688c.734-1.819 1.69-3.597 2.868-5.311m3.602-.616c-1.57 1.886-2.818 3.875-3.749 5.927h3.75zm-7.334 8.427H9.984c-.944 1.847-1.534 3.905-1.68 6.085h6.126c.071-2.045.393-4.082.971-6.085m1.531 6.085c.08-2.049.44-4.088 1.085-6.085h4.718v6.085zm-2.463 2.5H8.332c.197 2.19.843 4.25 1.845 6.088h5.438c-.629-2.004-1.012-4.044-1.146-6.088m2.065 8.588H11.87c1.903 2.302 4.448 4.052 7.361 4.978-1.078-1.612-1.979-3.277-2.697-4.978m2.736 0h3.465v5.542c-1.408-1.777-2.565-3.635-3.465-5.542m3.465-2.5h-4.49c-.698-2.002-1.123-4.043-1.27-6.088h5.76zm2.5 2.5v5.58c1.422-1.79 2.588-3.66 3.495-5.58zm4.52-2.5h-4.52V25.92h5.79c-.147 2.045-.572 4.086-1.27 6.088m1.711 2.5c-.719 1.7-1.62 3.366-2.698 4.978 2.913-.926 5.459-2.676 7.362-4.978zm6.356-2.5h-5.437c.629-2.004 1.012-4.044 1.146-6.088h6.137c-.197 2.19-.843 4.25-1.846 6.088M25.236 23.42v-6.085h4.747c.646 1.997 1.006 4.036 1.085 6.085zm8.334 0c-.07-2.045-.393-4.082-.97-6.085h5.416c.945 1.847 1.535 3.905 1.681 6.085zM28.866 9.523c1.18 1.714 2.134 3.492 2.869 5.311h4.688c-1.91-2.455-4.533-4.328-7.557-5.311m.148 5.311c-.938-2.065-2.195-4.066-3.779-5.962v5.962zM5.768 24.488C5.768 14.418 13.93 6.256 24 6.256c10.07 0 18.232 8.163 18.232 18.232 0 10.07-8.163 18.233-18.232 18.233-10.07 0-18.232-8.163-18.232-18.233"
        })
    });
const $0936880dcccd78f0$var$ForwardRef = /*#__PURE__*/ (0, $i6Gm1$react.forwardRef)($0936880dcccd78f0$var$SvgComponent);
var $0936880dcccd78f0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0936880dcccd78f0$var$ForwardRef);
