// @react-spectrum/s2/illustrations/linear/internal/S2_lin_infiniteLooping_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $hNItD$jsx} from "react/jsx-runtime";
import {forwardRef as $hNItD$forwardRef} from "react";





const $03807c024bd5d99c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hNItD$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hNItD$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M21.55 22.053c.69-.002 1.248-.563 1.247-1.253l-.018-7.506c-.001-.69-.562-1.249-1.253-1.247s-1.248.562-1.247 1.253l.01 4.456c-2.007-2.07-4.82-3.358-7.933-3.358-6.103 0-11.051 4.948-11.051 11.051S6.253 36.5 12.356 36.5c2.532 0 4.868-.852 6.732-2.286q.043-.033.083-.07l.034-.026c1.373-1.077 2.648-2.64 3.644-4.066 1-1.434 1.784-2.828 2.152-3.64.285-.629.99-2.123 1.924-3.692.803-1.352 1.717-2.656 2.618-3.532q.042-.033.083-.07c1.52-1.38 3.535-2.22 5.749-2.22 4.723 0 8.551 3.828 8.551 8.551S40.098 34 35.375 34c-2.888 0-5.442-1.43-6.992-3.627-.398-.564-1.178-.698-1.742-.3s-.699 1.178-.3 1.742c1.998 2.832 5.3 4.685 9.034 4.685 6.103 0 11.051-4.948 11.051-11.051s-4.948-11.051-11.051-11.051c-2.527 0-4.857.849-6.719 2.276q-.081.045-.158.107c-1.443 1.137-2.744 3.016-3.722 4.66-.998 1.68-1.747 3.266-2.052 3.939-.29.642-.99 1.902-1.925 3.241-.94 1.346-2.05 2.677-3.138 3.53q-.038.03-.073.063l-.024.018C16.122 33.342 14.318 34 12.356 34c-4.723 0-8.551-3.828-8.551-8.551s3.828-8.551 8.551-8.551c2.44 0 4.641 1.02 6.2 2.662l-4.518.01c-.69.002-1.249.563-1.247 1.253.001.69.562 1.249 1.253 1.247z"
        })
    });
const $03807c024bd5d99c$var$ForwardRef = /*#__PURE__*/ (0, $hNItD$forwardRef)($03807c024bd5d99c$var$SvgComponent);
var $03807c024bd5d99c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($03807c024bd5d99c$var$ForwardRef);


export {$03807c024bd5d99c$export$2e2bcd8739ae039 as default};
