// @react-spectrum/s2/illustrations/linear/internal/S2_lin_camera_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $202nH$jsx} from "react/jsx-runtime";
import {forwardRef as $202nH$forwardRef} from "react";





const $6ceb149fd95db367$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $202nH$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $202nH$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M64.864 35.757C66.32 33.422 68.886 32 71.65 32h16.7c2.758 0 5.32 1.418 6.79 3.763v.001L102.786 48h-45.57l7.647-12.24zM47.782 48l10.294-16.477.001-.002C61.002 26.837 66.136 24 71.65 24h16.7c5.521 0 10.64 2.842 13.569 7.516l.003.005L112.218 48H124c11.046 0 20 8.954 20 20v48c0 11.046-8.954 20-20 20H36c-11.046 0-20-8.954-20-20V68c0-11.046 8.954-20 20-20zM50 56H36c-6.627 0-12 5.373-12 12v48c0 6.627 5.373 12 12 12h88c6.627 0 12-5.373 12-12V68c0-6.627-5.373-12-12-12H50m30 16c-11.046 0-20 8.954-20 20s8.954 20 20 20 20-8.954 20-20-8.954-20-20-20M52 92c0-15.464 12.536-28 28-28s28 12.536 28 28-12.536 28-28 28-28-12.536-28-28m-8.5-23c0 3.314-2.686 6-6 6s-6-2.686-6-6 2.686-6 6-6 6 2.686 6 6"
        })
    });
const $6ceb149fd95db367$var$ForwardRef = /*#__PURE__*/ (0, $202nH$forwardRef)($6ceb149fd95db367$var$SvgComponent);
var $6ceb149fd95db367$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6ceb149fd95db367$var$ForwardRef);


export {$6ceb149fd95db367$export$2e2bcd8739ae039 as default};
