// @react-spectrum/s2/illustrations/linear/internal/S2_lin_update_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dCL85$reactjsxruntime = require("react/jsx-runtime");
var $dCL85$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5539f35a655797c3$export$2e2bcd8739ae039);




const $5539f35a655797c3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dCL85$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dCL85$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M9.14 19.66c1.88-6.423 7.812-11.12 14.84-11.12 6.142 0 11.443 3.573 13.937 8.766l-3.977-1.86c-.626-.292-1.37-.022-1.662.604-.292.625-.022 1.369.603 1.661l6.31 2.95c.61.286 1.337.036 1.643-.565l3.28-6.44c.314-.615.069-1.367-.546-1.68-.615-.314-1.368-.07-1.681.546l-1.8 3.532C37.159 10.116 31.05 6.039 23.982 6.039c-8.172 0-15.059 5.462-17.24 12.92-.194.662.186 1.356.849 1.55.662.194 1.357-.186 1.55-.849m-.33 7.737c-.61-.286-1.338-.036-1.644.565l-3.28 6.44c-.313.615-.068 1.368.547 1.68.615.314 1.368.07 1.681-.546l1.784-3.502c2.946 5.889 9.029 9.925 16.062 9.925 8.172 0 15.059-5.462 17.24-12.92.194-.662-.186-1.356-.849-1.55s-1.357.186-1.55.849c-1.88 6.423-7.812 11.121-14.84 11.121-6.125 0-11.415-3.555-13.917-8.726l4.017 1.878c.625.293 1.37.023 1.662-.603.292-.625.022-1.369-.603-1.661z"
        })
    });
const $5539f35a655797c3$var$ForwardRef = /*#__PURE__*/ (0, $dCL85$react.forwardRef)($5539f35a655797c3$var$SvgComponent);
var $5539f35a655797c3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5539f35a655797c3$var$ForwardRef);
