// @react-spectrum/s2/illustrations/linear/internal/S2_lin_remix_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $58lTQ$reactjsxruntime = require("react/jsx-runtime");
var $58lTQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $59e80850574faf4e$export$2e2bcd8739ae039);




const $59e80850574faf4e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $58lTQ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $58lTQ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M76.028 41.644 86.643 31.03c.781-.78.781-2.047 0-2.828L76.028 17.586c-.781-.781-2.047-.781-2.828 0-.781.78-.781 2.047 0 2.828l6.998 6.998H75.41c-4.92 0-9.46 1.657-13.083 4.443q-.076.06-.147.125-.042.03-.082.062c-2.673 2.098-5.175 5.157-7.143 7.977-1.975 2.831-3.517 5.575-4.234 7.156-.57 1.258-1.98 4.245-3.847 7.385-1.607 2.703-3.457 5.352-5.308 7.144q-.077.058-.15.124c-3.105 2.822-7.225 4.54-11.75 4.54-9.653 0-17.477-7.826-17.477-17.478s7.824-17.478 17.477-17.478c5.903 0 11.123 2.925 14.29 7.413l.008.01 2.022 2.828c.642.899 1.892 1.106 2.79.464s1.106-1.892.464-2.79l-2.02-2.824c-3.884-5.501-10.299-9.1-17.554-9.1-11.862 0-21.478 9.615-21.478 21.477s9.616 21.477 21.478 21.477c4.925 0 9.466-1.66 13.09-4.448q.143-.08.277-.184c2.791-2.202 5.34-5.87 7.28-9.13 1.97-3.316 3.45-6.45 4.052-7.779.593-1.308 2-3.838 3.87-6.518 1.878-2.69 4.115-5.38 6.333-7.12q.069-.054.132-.113l.066-.049c2.948-2.267 6.636-3.614 10.644-3.614h5.192L73.2 38.816c-.781.78-.781 2.047 0 2.828.78.781 2.047.781 2.828 0m4.781 28.723c1.105 0 2-.895 2-2s-.895-2-2-2h-5.398c-5.811 0-10.497-3.346-14.558-7.777l-2.337-2.825c-.704-.85-1.965-.97-2.816-.265-.85.704-.97 1.965-.266 2.816l2.37 2.863.031.038.033.037c4.338 4.743 10.031 9.113 17.543 9.113z"
        })
    });
const $59e80850574faf4e$var$ForwardRef = /*#__PURE__*/ (0, $58lTQ$react.forwardRef)($59e80850574faf4e$var$SvgComponent);
var $59e80850574faf4e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($59e80850574faf4e$var$ForwardRef);
