// @react-spectrum/s2/illustrations/linear/internal/S2_lin_briefcase_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $hqNSL$jsx} from "react/jsx-runtime";
import {forwardRef as $hqNSL$forwardRef} from "react";





const $d5e2b1417a1173a6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hqNSL$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hqNSL$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M19 10.25c-1.52 0-2.75 1.23-2.75 2.75v.75h15.5V13c0-1.52-1.23-2.75-2.75-2.75zM13.75 13v.75H11c-2.9 0-5.25 2.35-5.25 5.25v16c0 2.9 2.35 5.25 5.25 5.25h26c2.9 0 5.25-2.35 5.25-5.25V19c0-2.9-2.35-5.25-5.25-5.25h-2.75V13c0-2.9-2.35-5.25-5.25-5.25H19c-2.9 0-5.25 2.35-5.25 5.25M11 16.25c-1.519 0-2.75 1.231-2.75 2.75v6.75h5.5V25c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v.75h15.5V25c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v.75h5.5V19c0-1.519-1.231-2.75-2.75-2.75zm20.75 12V29c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-.75h5.5V35c0 1.519-1.231 2.75-2.75 2.75H11c-1.519 0-2.75-1.231-2.75-2.75v-6.75h5.5V29c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-.75z"
        })
    });
const $d5e2b1417a1173a6$var$ForwardRef = /*#__PURE__*/ (0, $hqNSL$forwardRef)($d5e2b1417a1173a6$var$SvgComponent);
var $d5e2b1417a1173a6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d5e2b1417a1173a6$var$ForwardRef);


export {$d5e2b1417a1173a6$export$2e2bcd8739ae039 as default};
