// @react-spectrum/s2/illustrations/linear/internal/S2_lin_brightnessContrast_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bHfBk$reactjsxruntime = require("react/jsx-runtime");
var $bHfBk$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e8338141c5018b48$export$2e2bcd8739ae039);




const $e8338141c5018b48$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bHfBk$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bHfBk$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M84 24c0-2.21-1.79-4-4-4s-4 1.79-4 4v7.938c0 2.21 1.79 4 4 4s4-1.79 4-4zM55.716 80.002c0-13.411 10.872-24.283 24.283-24.283v48.567c-13.411 0-24.283-10.872-24.283-24.284M79.999 47.72c-17.83 0-32.283 14.454-32.283 32.283s14.454 32.284 32.283 32.284 32.284-14.454 32.284-32.284S97.829 47.72 79.999 47.72M80 124.062c2.21 0 4 1.791 4 4V136c0 2.209-1.79 4-4 4s-4-1.791-4-4v-7.938c0-2.209 1.791-4 4-4M136 84c2.209 0 4-1.79 4-4s-1.791-4-4-4h-7.938c-2.209 0-4 1.79-4 4s1.791 4 4 4zM35.938 80c0 2.21-1.79 4-4 4H24c-2.21 0-4-1.79-4-4s1.79-4 4-4h7.938c2.21 0 4 1.79 4 4m86.488-36.77c1.562-1.562 1.562-4.095 0-5.657s-4.095-1.562-5.657 0l-5.613 5.613c-1.562 1.562-1.562 4.095 0 5.657s4.095 1.562 5.657 0zm-73.583 67.926c1.562 1.562 1.562 4.095 0 5.657l-5.614 5.613c-1.562 1.562-4.094 1.562-5.656 0s-1.562-4.094 0-5.657l5.613-5.613c1.562-1.562 4.095-1.562 5.657 0m-11.27-67.928c-1.563-1.562-1.563-4.095 0-5.657 1.561-1.562 4.094-1.562 5.656 0l5.613 5.613c1.562 1.562 1.562 4.095 0 5.657s-4.094 1.562-5.657 0zm73.583 67.926c-1.562 1.563-1.562 4.095 0 5.657l5.613 5.613c1.562 1.563 4.095 1.563 5.657 0s1.562-4.094 0-5.656l-5.613-5.614c-1.562-1.562-4.095-1.562-5.657 0"
        })
    });
const $e8338141c5018b48$var$ForwardRef = /*#__PURE__*/ (0, $bHfBk$react.forwardRef)($e8338141c5018b48$var$SvgComponent);
var $e8338141c5018b48$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e8338141c5018b48$var$ForwardRef);
