// @react-spectrum/s2/illustrations/linear/internal/S2_lin_location_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6cnuJ$reactjsxruntime = require("react/jsx-runtime");
var $6cnuJ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b68d6f7925c6eaec$export$2e2bcd8739ae039);




const $b68d6f7925c6eaec$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6cnuJ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6cnuJ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M76 40.794c0-16.08-13.506-29.006-29.776-27.999-13.463.835-24.542 11.39-26.045 24.816-.478 4.264.022 8.36 1.276 12.107 3.974 12.841 15.368 25.136 21.275 30.9 2.912 2.84 7.539 2.823 10.441-.01 5.539-5.406 15.815-16.473 20.378-28.34C75.12 48.764 76 44.882 76 40.794M46.471 16.788C60.417 15.924 72 26.998 72 40.794c0 3.525-.762 6.868-2.121 9.884q-.023.052-.045.106c-4.22 11.017-13.958 21.594-19.457 26.96-1.358 1.326-3.505 1.327-4.854.01C39.626 72 28.911 60.308 25.27 48.515l-.015-.046c-1.08-3.22-1.513-6.737-1.101-10.412 1.288-11.5 10.798-20.555 22.317-21.268M40 39.24c0-4.74 3.739-8.5 8.25-8.5s8.25 3.76 8.25 8.5-3.739 8.5-8.25 8.5S40 43.98 40 39.24m8.25-12.5c-6.81 0-12.25 5.643-12.25 12.5s5.44 12.5 12.25 12.5 12.25-5.642 12.25-12.5-5.44-12.5-12.25-12.5"
        })
    });
const $b68d6f7925c6eaec$var$ForwardRef = /*#__PURE__*/ (0, $6cnuJ$react.forwardRef)($b68d6f7925c6eaec$var$SvgComponent);
var $b68d6f7925c6eaec$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b68d6f7925c6eaec$var$ForwardRef);
