// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cloud_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $13I5f$reactjsxruntime = require("react/jsx-runtime");
var $13I5f$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a14fe9b43e680294$export$2e2bcd8739ae039);




const $a14fe9b43e680294$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $13I5f$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $13I5f$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M33.803 31.952c3.24-7.87 10.756-13.452 19.595-13.452 11.403 0 20.592 9.266 21.269 20.8C83.753 40.773 90.6 48.95 90.6 58.68c0 10.753-8.367 19.62-18.87 19.62H18.928q-.131 0-.259-.017C11.005 78.028 5 71.495 5 63.631c0-6.054 3.548-11.292 8.672-13.52q-.121-.983-.123-2.011c0-9.15 7.123-16.717 16.091-16.717 1.441 0 2.836.2 4.163.57M19.136 74.3h52.4q.054-.003.11-.002h.075l.03.002c8.122-.012 14.849-6.918 14.849-15.62 0-8.4-6.279-15.134-14.021-15.59-1.072-.064-1.903-.963-1.882-2.037.002-.12.005-.183.007-.231.002-.052.003-.087.003-.156 0-10.115-7.83-18.166-17.31-18.166-7.65 0-14.212 5.232-16.472 12.584-.167.543-.557.99-1.073 1.228s-1.11.246-1.631.02c-1.416-.61-2.96-.95-4.58-.95-6.597 0-12.092 5.612-12.092 12.718 0 .984.118 1.947.327 2.892.229 1.032-.386 2.064-1.402 2.355C12.214 54.565 9 58.664 9 63.63c0 5.91 4.524 10.565 9.963 10.659q.087 0 .173.01"
        })
    });
const $a14fe9b43e680294$var$ForwardRef = /*#__PURE__*/ (0, $13I5f$react.forwardRef)($a14fe9b43e680294$var$SvgComponent);
var $a14fe9b43e680294$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a14fe9b43e680294$var$ForwardRef);
