// @react-spectrum/s2/illustrations/linear/internal/S2_lin_check_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7GwAz$jsx} from "react/jsx-runtime";
import {forwardRef as $7GwAz$forwardRef} from "react";





const $1a92f311f65068af$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7GwAz$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7GwAz$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M17 48c0-17.12 13.88-31 31-31 17.121 0 31 13.88 31 31 0 17.121-13.879 31-31 31-17.12 0-31-13.879-31-31m31-35c-19.33 0-35 15.67-35 35s15.67 35 35 35 35-15.67 35-35-15.67-35-35-35m16.08 27.639c.816-.744.875-2.01.13-2.826-.743-.816-2.008-.875-2.825-.13l-9.828 8.958-6.586 6.002-1.62 1.476-8.13-7.582c-.807-.753-2.073-.709-2.826.099s-.71 2.073.098 2.827l9.479 8.84c.762.71 1.941.716 2.711.015l-1.347-1.479 1.347 1.478h.001l.001-.001.002-.002.01-.009.04-.037.16-.145.605-.552 2.163-1.971 6.586-6.003z"
        })
    });
const $1a92f311f65068af$var$ForwardRef = /*#__PURE__*/ (0, $7GwAz$forwardRef)($1a92f311f65068af$var$SvgComponent);
var $1a92f311f65068af$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1a92f311f65068af$var$ForwardRef);


export {$1a92f311f65068af$export$2e2bcd8739ae039 as default};
