// @react-spectrum/s2/illustrations/linear/internal/S2_lin_filmstrip_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $719Fr$jsx} from "react/jsx-runtime";
import {forwardRef as $719Fr$forwardRef} from "react";





const $bdf63584ecd028de$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $719Fr$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $719Fr$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M124 38c6.627 0 12 5.373 12 12v60c0 6.627-5.373 12-12 12H36c-6.627 0-12-5.373-12-12V50c0-6.627 5.373-12 12-12zm20 12c0-11.046-8.954-20-20-20H36c-11.046 0-20 8.954-20 20v60c0 11.046 8.954 20 20 20h88c11.046 0 20-8.954 20-20zm-95 3c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm-15-2c-1.105 0-2 .895-2 2v5c0 1.105.895 2 2 2h7c1.105 0 2-.895 2-2v-5c0-1.105-.895-2-2-2zm15 51c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm-15-2c-1.105 0-2 .895-2 2v5c0 1.105.895 2 2 2h7c1.105 0 2-.895 2-2v-5c0-1.105-.895-2-2-2zm32-47c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm2 47c-1.105 0-2 .895-2 2v5c0 1.105.895 2 2 2h7c1.105 0 2-.895 2-2v-5c0-1.105-.895-2-2-2zm15-47c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm2 47c-1.105 0-2 .895-2 2v5c0 1.105.895 2 2 2h7c1.105 0 2-.895 2-2v-5c0-1.105-.895-2-2-2zm15-47c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm2 47c-1.105 0-2 .895-2 2v5c0 1.105.895 2 2 2h7c1.105 0 2-.895 2-2v-5c0-1.105-.895-2-2-2zm15-47c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm2 47c-1.105 0-2 .895-2 2v5c0 1.105.895 2 2 2h7c1.105 0 2-.895 2-2v-5c0-1.105-.895-2-2-2z"
        })
    });
const $bdf63584ecd028de$var$ForwardRef = /*#__PURE__*/ (0, $719Fr$forwardRef)($bdf63584ecd028de$var$SvgComponent);
var $bdf63584ecd028de$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($bdf63584ecd028de$var$ForwardRef);


export {$bdf63584ecd028de$export$2e2bcd8739ae039 as default};
