// @react-spectrum/s2/illustrations/linear/internal/S2_lin_diamond_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1OD5M$jsx} from "react/jsx-runtime";
import {forwardRef as $1OD5M$forwardRef} from "react";





const $d22ee5e526f24116$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1OD5M$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1OD5M$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M55.5 21.842c-3.171 0-6.213 1.255-8.462 3.49l-26.136 25.99c-4.268 4.244-4.732 10.994-1.083 15.782l50.54 66.327c4.797 6.296 14.267 6.305 19.076.018l50.732-66.32c3.67-4.799 3.202-11.577-1.094-15.824l-26.289-25.996c-2.247-2.221-5.278-3.467-8.438-3.467zm-2.82 9.163c.749-.745 1.763-1.163 2.82-1.163h11.56L54.67 55.707H27.838zm10.86 24.702 12.392-25.865h8.158l12.391 25.865zm33.087 8H63.394l16.617 59.133zm-7.817 57.392 16.127-57.392h27.775zm16.542-65.392L92.961 29.842h11.385c1.053 0 2.064.415 2.813 1.156l24.988 24.709zm-78.064 8h27.797l16.239 57.792z"
        })
    });
const $d22ee5e526f24116$var$ForwardRef = /*#__PURE__*/ (0, $1OD5M$forwardRef)($d22ee5e526f24116$var$SvgComponent);
var $d22ee5e526f24116$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d22ee5e526f24116$var$ForwardRef);


export {$d22ee5e526f24116$export$2e2bcd8739ae039 as default};
