// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noReviewLinks_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $9QSE7$reactjsxruntime = require("react/jsx-runtime");
var $9QSE7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $784f8ec6f4241b5d$export$2e2bcd8739ae039);




const $784f8ec6f4241b5d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9QSE7$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9QSE7$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M36.65 16c-7.735 0-13.984 6.283-13.984 14v79.983c0 7.717 6.249 14 13.983 14h40.017c2.21 0 4-1.791 4-4s-1.79-4-4-4H36.649c-3.299 0-5.983-2.683-5.983-6V30c0-3.316 2.684-6 5.983-6h32.417c1.55 0 3.033.595 4.124 1.646l.009.009 32.616 31.25h.001c1.182 1.133 1.85 2.69 1.85 4.328v24.584c0 2.209 1.791 4 4 4s4-1.791 4-4V61.233c0-3.828-1.565-7.47-4.317-10.105L78.742 19.887l-.005-.005C76.13 17.37 72.647 16 69.066 16zm74.584 128.15c13.577 0 24.583-11.006 24.583-24.583s-11.006-24.584-24.583-24.584S86.65 105.99 86.65 119.567s11.007 24.583 24.584 24.583m4.413-35.834c0-2.209-1.791-4-4-4s-4 1.791-4 4v7.667h-7.666c-2.21 0-4 1.791-4 4s1.79 4 4 4h7.666v7.667c0 2.209 1.791 4 4 4s4-1.791 4-4v-7.667h7.667c2.209 0 4-1.791 4-4s-1.791-4-4-4h-7.667zM74.326 77.339c1.139-1.892.528-4.35-1.365-5.49-4.536-2.729-10.54-2.176-14.486 1.71L47.053 84.804c-4.65 4.578-4.65 12.027 0 16.605l2.806-2.85-2.806 2.85c4.615 4.544 12.073 4.544 16.688 0l4.142-4.078c1.574-1.55 1.594-4.082.044-5.656-1.55-1.575-4.082-1.594-5.657-.044l-4.142 4.077c-1.501 1.479-3.961 1.479-5.463 0-1.467-1.444-1.467-3.76 0-5.203l11.422-11.246c1.272-1.252 3.254-1.454 4.75-.555 1.892 1.14 4.35.528 5.489-1.365m9.817-7.858c1.452-1.449 1.452-3.791 0-5.24-1.46-1.454-3.83-1.454-5.29 0l-4.211 4.198c-1.564 1.56-4.097 1.556-5.657-.008s-1.556-4.097.009-5.657l4.21-4.198c4.582-4.568 12.005-4.568 16.587 0 4.588 4.574 4.588 11.996 0 16.57l-11.28 11.245c-3.867 3.855-9.745 4.451-14.238 1.814-1.905-1.118-2.543-3.569-1.425-5.474s3.569-2.543 5.474-1.425c1.436.843 3.316.641 4.54-.58z"
        })
    });
const $784f8ec6f4241b5d$var$ForwardRef = /*#__PURE__*/ (0, $9QSE7$react.forwardRef)($784f8ec6f4241b5d$var$SvgComponent);
var $784f8ec6f4241b5d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($784f8ec6f4241b5d$var$ForwardRef);
