// @react-spectrum/s2/illustrations/linear/internal/S2_lin_stepForward_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2PVCa$reactjsxruntime = require("react/jsx-runtime");
var $2PVCa$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ea27b97b71738876$export$2e2bcd8739ae039);




const $ea27b97b71738876$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2PVCa$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2PVCa$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M64.575 47.46c.012-10.167 10.86-16.96 19.822-11.607l54.576 32.59c8.714 5.204 8.7 18.081-.027 23.264l-54.651 32.456c-8.976 5.33-19.807-1.489-19.795-11.656zm15.72-4.739c-3.26-1.947-7.715.348-7.72 4.75l-.075 65.045c-.005 4.402 4.445 6.707 7.71 4.768l54.651-32.455c3.513-2.087 3.52-7.421.01-9.517zM20 46c0-6.628 5.373-12 12-12h11c6.627 0 12 5.372 12 12v68c0 6.627-5.373 12-12 12H32c-6.627 0-12-5.373-12-12zm12-4c-2.21 0-4 1.79-4 4v68c0 2.209 1.79 4 4 4h11c2.21 0 4-1.791 4-4V46c0-2.21-1.79-4-4-4z"
        })
    });
const $ea27b97b71738876$var$ForwardRef = /*#__PURE__*/ (0, $2PVCa$react.forwardRef)($ea27b97b71738876$var$SvgComponent);
var $ea27b97b71738876$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ea27b97b71738876$var$ForwardRef);
