// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pencil_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6IqYf$reactjsxruntime = require("react/jsx-runtime");
var $6IqYf$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $8050fdec74f24402$export$2e2bcd8739ae039);




const $8050fdec74f24402$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6IqYf$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6IqYf$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M10.716 28.847c.163-.435.417-.83.745-1.158L30.067 9.083c2.444-2.444 6.407-2.444 8.851 0s2.445 6.408 0 8.852L20.313 36.541c-.328.328-.724.582-1.159.745l-8.86 3.318c-1.808.677-3.573-1.088-2.896-2.896zm3.112.01 5.316 5.317 15.172-15.172L29 13.685zm3.068 6.605-4.356-4.357-2.608 6.965zm20.255-19.295-1.067 1.067-5.316-5.317 1.066-1.066c1.468-1.468 3.849-1.468 5.317 0s1.468 3.848 0 5.316"
        })
    });
const $8050fdec74f24402$var$ForwardRef = /*#__PURE__*/ (0, $6IqYf$react.forwardRef)($8050fdec74f24402$var$SvgComponent);
var $8050fdec74f24402$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($8050fdec74f24402$var$ForwardRef);
