// @react-spectrum/s2/illustrations/linear/internal/S2_lin_clock_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1W84e$jsx} from "react/jsx-runtime";
import {forwardRef as $1W84e$forwardRef} from "react";





const $18ea86fd25db8097$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1W84e$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1W84e$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 27.74c-28.719 0-52 23.281-52 52s23.281 52 52 52 52-23.281 52-52-23.281-52-52-52m-60 52c0-33.137 26.863-60 60-60s60 26.863 60 60-26.863 60-60 60-60-26.863-60-60m64-2.31V40.74c0-2.209-1.79-4-4-4s-4 1.791-4 4v38.924q-.004.239.02.476c.057.578.238 1.121.515 1.6s.656.908 1.128 1.247q.194.14.404.256l21.754 12.56c1.913 1.104 4.359.448 5.464-1.465 1.104-1.913.449-4.36-1.464-5.464z"
        })
    });
const $18ea86fd25db8097$var$ForwardRef = /*#__PURE__*/ (0, $1W84e$forwardRef)($18ea86fd25db8097$var$SvgComponent);
var $18ea86fd25db8097$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($18ea86fd25db8097$var$ForwardRef);


export {$18ea86fd25db8097$export$2e2bcd8739ae039 as default};
