// @react-spectrum/s2/illustrations/linear/internal/S2_lin_sparkles_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dLawb$reactjsxruntime = require("react/jsx-runtime");
var $dLawb$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $305e9f78e6fbec6c$export$2e2bcd8739ae039);




const $305e9f78e6fbec6c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dLawb$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dLawb$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M127.702 34.576c2.578-11.482-11.187-19.438-19.844-11.457l-13 11.98c-.965.89-2.31 1.249-3.585.96l-.006-.002-17.25-3.88h-.002C62.533 29.6 54.577 43.364 62.557 52.021h.001l11.98 13c.89.965 1.249 2.31.96 3.585l-.002.006-3.88 17.25v.002C69.038 97.346 82.803 105.302 91.46 97.32l13-11.98h.001c.964-.89 2.309-1.249 3.584-.96l.006.001 17.25 3.88h.002c11.482 2.58 19.438-11.186 11.457-19.843l-11.98-13h-.001c-.888-.964-1.248-2.31-.959-3.585l.002-.006 3.88-17.25zm-14.422-5.575c2.883-2.658 7.477-.015 6.617 3.821l-.001.002-3.878 17.242-.001.003c-.87 3.844.211 7.876 2.881 10.772h.001l11.979 13c2.658 2.883.015 7.477-3.821 6.617h-.002l-17.242-3.88h-.003c-3.844-.87-7.876.21-10.772 2.88v.002l-13 11.979c-2.883 2.658-7.477.015-6.617-3.821v-.002l3.88-17.242v-.004c.87-3.844-.211-7.876-2.88-10.77l-.002-.002L68.441 46.6h-.001c-2.658-2.883-.015-7.477 3.821-6.616h.002l17.242 3.878.004.001c3.844.87 7.875-.211 10.771-2.88l.001-.002zM64.24 129.81l-12.16-2.74c-1.61-.36-3.28.09-4.49 1.2l-9.16 8.45c-3.61 3.33-9.35.01-8.27-4.77l2.74-12.16c.36-1.61-.09-3.28-1.2-4.49l-8.45-9.16c-3.33-3.61-.01-9.35 4.77-8.27l12.16 2.74c1.61.36 3.28-.09 4.49-1.2l9.16-8.45c3.61-3.33 9.35-.01 8.27 4.77l-2.74 12.16c-.36 1.61.09 3.28 1.2 4.49l8.45 9.16c3.33 3.61.01 9.35-4.77 8.27"
        })
    });
const $305e9f78e6fbec6c$var$ForwardRef = /*#__PURE__*/ (0, $dLawb$react.forwardRef)($305e9f78e6fbec6c$var$SvgComponent);
var $305e9f78e6fbec6c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($305e9f78e6fbec6c$var$ForwardRef);
