// @react-spectrum/s2/illustrations/linear/internal/S2_lin_shoppingBag_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bPP5h$reactjsxruntime = require("react/jsx-runtime");
var $bPP5h$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $788d6f09da342154$export$2e2bcd8739ae039);




const $788d6f09da342154$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bPP5h$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bPP5h$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M42 22.992c0-3.037 2.463-5.5 5.5-5.5s5.5 2.463 5.5 5.5v3.5H42zm-4 7.5v6c0 1.105.896 2 2 2s2-.895 2-2v-6h11v6c0 1.105.896 2 2 2s2-.895 2-2v-6h10.19l3.941 39.403c.353 3.532-2.42 6.597-5.97 6.597h-35.32c-3.55 0-6.324-3.065-5.97-6.597l3.94-39.403zm0-4v-3.5c0-5.247 4.254-9.5 9.5-9.5 5.248 0 9.5 4.253 9.5 9.5v3.5h10.19c2.056 0 3.777 1.557 3.981 3.602l3.94 39.403c.59 5.887-4.034 10.995-9.95 10.995h-35.32c-5.916 0-10.54-5.108-9.95-10.995l3.94-39.403c.204-2.045 1.925-3.602 3.98-3.602z"
        })
    });
const $788d6f09da342154$var$ForwardRef = /*#__PURE__*/ (0, $bPP5h$react.forwardRef)($788d6f09da342154$var$SvgComponent);
var $788d6f09da342154$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($788d6f09da342154$var$ForwardRef);
