// @react-spectrum/s2/illustrations/linear/internal/S2_lin_movieCamera_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $aTGGq$jsx} from "react/jsx-runtime";
import {forwardRef as $aTGGq$forwardRef} from "react";





const $8ea24900f2086db0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aTGGq$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $aTGGq$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M11.75 34.094c0-5.523 4.477-10 10-10h31.924c5.523 0 10 4.477 10 10v3.794l11-8.163c3.96-2.939 9.576-.113 9.576 4.818v26.815c0 4.918-5.592 7.747-9.554 4.834l-11.022-8.103v3.816c0 5.523-4.477 10-10 10H21.75c-5.523 0-10-4.477-10-10zm10-6c-3.314 0-6 2.686-6 6v27.811c0 3.314 2.686 6 6 6h31.924c3.314 0 6-2.686 6-6v-7.77c0-.753.424-1.443 1.097-1.784s1.48-.273 2.088.174l14.206 10.444c1.321.971 3.185.028 3.185-1.611V34.543c0-1.644-1.872-2.586-3.192-1.606L62.866 43.469c-.607.45-1.416.52-2.091.18s-1.101-1.03-1.101-1.786v-7.77c0-3.313-2.686-6-6-6z"
        })
    });
const $8ea24900f2086db0$var$ForwardRef = /*#__PURE__*/ (0, $aTGGq$forwardRef)($8ea24900f2086db0$var$SvgComponent);
var $8ea24900f2086db0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8ea24900f2086db0$var$ForwardRef);


export {$8ea24900f2086db0$export$2e2bcd8739ae039 as default};
