// @react-spectrum/s2/illustrations/linear/internal/S2_lin_prompt_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $ateZm$reactjsxruntime = require("react/jsx-runtime");
var $ateZm$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $933308f602cd6032$export$2e2bcd8739ae039);




const $933308f602cd6032$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ateZm$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $ateZm$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16 34c0-5.523 4.477-10 10-10h44c5.523 0 10 4.477 10 10v28c0 5.523-4.477 10-10 10H26c-5.523 0-10-4.477-10-10zm10-6c-3.314 0-6 2.686-6 6v28c0 3.314 2.686 6 6 6h44c3.314 0 6-2.686 6-6V34c0-3.314-2.686-6-6-6zm16 32c0-1.105.895-2 2-2h24c1.105 0 2 .895 2 2s-.895 2-2 2H44c-1.105 0-2-.895-2-2m-15.534.848c-.692.861-.554 2.12.307 2.811.862.692 2.12.554 2.812-.307l11.4-14.204c.59-.735.587-1.783-.009-2.515l-11.4-13.996c-.697-.857-1.957-.986-2.814-.288-.856.697-.985 1.957-.287 2.814l10.378 12.743z"
        })
    });
const $933308f602cd6032$var$ForwardRef = /*#__PURE__*/ (0, $ateZm$react.forwardRef)($933308f602cd6032$var$SvgComponent);
var $933308f602cd6032$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($933308f602cd6032$var$ForwardRef);
