// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cardTapPayment_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dLZeF$reactjsxruntime = require("react/jsx-runtime");
var $dLZeF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $648d4791fe399be1$export$2e2bcd8739ae039);




const $648d4791fe399be1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dLZeF$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dLZeF$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M34.792 7.126c.084-.685.707-1.173 1.392-1.09 3.351.41 6.482 2.335 8.302 5.488 1.835 3.179 1.924 6.886.569 10.009-.275.633-1.012.924-1.645.649s-.924-1.011-.649-1.644c1.054-2.428.982-5.3-.44-7.764-1.411-2.444-3.835-3.938-6.44-4.256-.685-.083-1.173-.707-1.089-1.392M2.726 28.289c-1.45-2.511-.59-5.722 1.921-7.172L24.696 9.542c2.51-1.45 5.722-.59 7.171 1.922l6.384 11.056c1.45 2.511.589 5.722-1.922 7.172L16.281 41.267c-2.511 1.45-5.722.589-7.172-1.922zm3.171-5.007c-1.315.76-1.766 2.441-1.006 3.757l.49.848 24.811-14.324-.49-.85c-.76-1.314-2.441-1.765-3.756-1.006zm5.377 14.813-4.643-8.042 24.811-14.326 4.644 8.043c.759 1.315.308 2.997-1.007 3.756L15.031 39.102c-1.316.76-2.998.31-3.757-1.006m23.778-27.453c-.685-.083-1.309.404-1.392 1.09-.084.685.404 1.308 1.089 1.392 1.32.161 2.544.917 3.258 2.153.719 1.246.756 2.698.222 3.927-.274.634.016 1.37.65 1.645.633.275 1.369-.016 1.644-.65.835-1.924.78-4.212-.351-6.172-1.123-1.945-3.055-3.132-5.12-3.385M11.038 30.217c-.598.345-.803 1.11-.457 1.707.345.598 1.11.803 1.707.458l9.469-5.467c.597-.346.802-1.11.457-1.708s-1.11-.803-1.708-.457zm1.966 5.904c-.345-.597-.14-1.362.457-1.707l4.84-2.794c.597-.345 1.361-.14 1.707.457.345.598.14 1.363-.458 1.708l-4.839 2.794c-.598.345-1.362.14-1.707-.458M24.84 22.249c-.598.345-.803 1.11-.457 1.707.345.598 1.11.803 1.707.458l4.537-2.62c.598-.345.803-1.11.457-1.707-.345-.598-1.11-.803-1.707-.458z"
        })
    });
const $648d4791fe399be1$var$ForwardRef = /*#__PURE__*/ (0, $dLZeF$react.forwardRef)($648d4791fe399be1$var$SvgComponent);
var $648d4791fe399be1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($648d4791fe399be1$var$ForwardRef);
