// @react-spectrum/s2/illustrations/linear/internal/S2_lin_microphoneOff_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $hi4Fc$reactjsxruntime = require("react/jsx-runtime");
var $hi4Fc$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7a65a4cd764b1725$export$2e2bcd8739ae039);




const $7a65a4cd764b1725$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hi4Fc$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hi4Fc$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M29.065 23.36c-1.562-1.563-4.095-1.563-5.657 0-1.562 1.561-1.562 4.094 0 5.656l107.526 107.526c1.562 1.562 4.095 1.562 5.657 0s1.562-4.095 0-5.657zm6.31 49.913c0 23.181 17.781 42.209 40.448 44.19v12.35H56.28c-2.209 0-4 1.79-4 4s1.791 4 4 4h47.086c2.209 0 4-1.791 4-4s-1.791-4-4-4H83.823v-12.368c2.327-.21 5.19-.64 8.24-1.504 2.126-.603 3.36-2.814 2.758-4.939s-2.814-3.361-4.939-2.758c-4.3 1.218-8.211 1.389-10.147 1.389-20.08 0-36.36-16.279-36.36-36.36V68.23c0-2.21-1.79-4-4-4s-4 1.79-4 4zm21.804-34.89C59.786 31.1 66.437 22.24 79.823 22.24c9.262 0 15.47 4.223 19.233 9.451 3.623 5.036 4.946 10.927 4.946 14.755v.03c-.06 8.063-.016 21.9-.003 26.105v.006l.003 1.212c0 1.859-.283 3.66-.714 5.325-.555 2.138-2.738 3.422-4.876 2.867-2.139-.554-3.423-2.737-2.868-4.876.296-1.141.458-2.26.458-3.316L96 72.691v-.003c-.013-4.068-.057-18.103.003-26.256-.003-2.348-.91-6.55-3.44-10.068-2.397-3.33-6.278-6.124-12.74-6.124-8.9 0-13.219 5.553-15.11 10.839-.745 2.08-3.035 3.162-5.115 2.417-2.08-.744-3.162-3.034-2.418-5.114m8.579 42.874c-1.121-1.904-3.573-2.538-5.477-1.416-1.903 1.121-2.537 3.573-1.416 5.477 2.013 3.415 5.017 6.699 8.88 8.968 1.905 1.12 4.356.482 5.475-1.422 1.12-1.905.483-4.356-1.422-5.475-2.546-1.496-4.627-3.734-6.04-6.132m58.424-13.029c0-2.209-1.791-4-4-4s-4 1.791-4 4c0 12.663-1.706 18.394-4.697 23.145-1.177 1.87-.615 4.34 1.254 5.516 1.87 1.177 4.339.615 5.516-1.254 4.125-6.553 5.927-14.007 5.927-27.407"
        })
    });
const $7a65a4cd764b1725$var$ForwardRef = /*#__PURE__*/ (0, $hi4Fc$react.forwardRef)($7a65a4cd764b1725$var$SvgComponent);
var $7a65a4cd764b1725$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7a65a4cd764b1725$var$ForwardRef);
