// @react-spectrum/s2/illustrations/linear/internal/S2_lin_translate_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $02viB$jsx} from "react/jsx-runtime";
import {forwardRef as $02viB$forwardRef} from "react";





const $77917d25b4625060$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $02viB$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $02viB$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M65.853 33.829h28.57c2.209 0 4 1.79 4 4s-1.791 4-4 4H86.73c-1.913 14.38-9.737 25.764-19.091 33.986-2.193 1.927-4.482 3.693-6.818 5.294 5.591 8.294 11.975 15.42 16.958 18.639 1.856 1.199 2.389 3.675 1.19 5.53s-3.675 2.389-5.53 1.19c-6.413-4.142-13.616-12.472-19.456-21.19-6.08 3.276-12.15 5.462-17.371 6.513-2.166.436-4.275-.966-4.711-3.132s.966-4.275 3.132-4.71c4.313-.869 9.468-2.702 14.711-5.477-4.02-6.974-7.149-14.126-8.164-19.888-.384-2.176 1.069-4.25 3.245-4.634 2.175-.383 4.25 1.07 4.633 3.245.792 4.496 3.426 10.678 7.124 17.12 1.974-1.368 3.913-2.872 5.775-4.509 8.05-7.076 14.454-16.45 16.288-27.977h-50.36c-2.209 0-4-1.791-4-4s1.791-4 4-4h29.568v-5.642c0-2.209 1.79-4 4-4s4 1.791 4 4zM85 133.374l8.279-19.531h26.247q.114 0 .226-.007l8.282 19.538c.862 2.034 3.209 2.984 5.243 2.122s2.984-3.21 2.122-5.244l-25.185-59.42q-.095-.23-.218-.448-.214-.377-.495-.69c-.381-.428-.861-.78-1.423-1.018-.51-.216-1.04-.319-1.561-.318-.521 0-1.051.102-1.561.318-.563.238-1.042.59-1.423 1.018q-.281.314-.495.69-.123.218-.219.449l-25.184 59.419c-.863 2.034.087 4.382 2.121 5.244s4.382-.088 5.244-2.122m21.517-50.765-9.847 23.234h19.694z"
        })
    });
const $77917d25b4625060$var$ForwardRef = /*#__PURE__*/ (0, $02viB$forwardRef)($77917d25b4625060$var$SvgComponent);
var $77917d25b4625060$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($77917d25b4625060$var$ForwardRef);


export {$77917d25b4625060$export$2e2bcd8739ae039 as default};
