// @react-spectrum/s2/illustrations/linear/internal/S2_lin_wallet_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $4TRx7$reactjsxruntime = require("react/jsx-runtime");
var $4TRx7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $65ef580378678565$export$2e2bcd8739ae039);




const $65ef580378678565$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4TRx7$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4TRx7$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M11.293 15.708c-.455-1.698.552-3.442 2.25-3.897L28.58 7.78c1.734-.464 3.516.565 3.98 2.299l1.426 5.318c.178.667-.217 1.352-.884 1.53-.667.18-1.352-.216-1.531-.883l-1.425-5.318c-.107-.4-.519-.638-.919-.53L14.19 14.225c-.364.097-.58.471-.483.835l.09.337c.18.667-.216 1.352-.883 1.53-.667.18-1.352-.216-1.531-.883zM9 17.83c-1.243 0-2.25 1.007-2.25 2.25v18c0 1.242 1.007 2.25 2.25 2.25h25c2.9 0 5.25-2.35 5.25-5.25v-1.014c1.125-.124 2-1.078 2-2.236v-5.5c0-1.158-.875-2.112-2-2.237v-1.013c0-2.9-2.35-5.25-5.25-5.25zm27.75 6.25v-1c0-1.52-1.231-2.75-2.75-2.75H9.25v17.5H34c1.519 0 2.75-1.232 2.75-2.75v-1H31c-1.243 0-2.25-1.008-2.25-2.25v-5.5c0-1.243 1.007-2.25 2.25-2.25zm-5.5 7.5v-5h7.5v5zm2.75-.875c.828 0 1.5-.672 1.5-1.5 0-.829-.672-1.5-1.5-1.5s-1.5.671-1.5 1.5c0 .828.672 1.5 1.5 1.5"
        })
    });
const $65ef580378678565$var$ForwardRef = /*#__PURE__*/ (0, $4TRx7$react.forwardRef)($65ef580378678565$var$SvgComponent);
var $65ef580378678565$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($65ef580378678565$var$ForwardRef);
