// @react-spectrum/s2/illustrations/linear/internal/S2_lin_targeted_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $lrt9t$jsx} from "react/jsx-runtime";
import {forwardRef as $lrt9t$forwardRef} from "react";





const $bfbc31c2410be7d0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lrt9t$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $lrt9t$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M4.81 9.153c-.348.089-.621.358-.716.704-.095.347.003.717.257.971l4.6 4.6c.207.207.495.313.787.289l3.775-.307 6.374 6.373c.488.488 1.28.488 1.767 0 .488-.488.488-1.28 0-1.768l-6.373-6.373.307-3.775c.024-.293-.082-.58-.29-.788l-4.598-4.6c-.254-.253-.625-.351-.971-.256-.347.094-.616.368-.705.716l-.86 3.354zM24 8.869c-1.743 0-3.415.295-4.971.836-.652.227-1.364-.118-1.591-.77s.118-1.365.77-1.591c1.816-.632 3.766-.975 5.793-.975 9.737 0 17.631 7.894 17.631 17.631s-7.894 17.632-17.631 17.632S6.37 33.738 6.37 24c0-1.956.319-3.84.909-5.602.22-.654.927-1.007 1.582-.788.655.22 1.008.928.788 1.582-.505 1.51-.78 3.125-.78 4.808 0 8.357 6.775 15.132 15.132 15.132S39.132 32.357 39.132 24 32.358 8.87 24.001 8.87m0 7.753c-.8 0-1.57.128-2.289.362-.656.214-1.362-.145-1.576-.8-.214-.657.145-1.363.801-1.577.966-.315 1.997-.485 3.064-.485 5.456 0 9.878 4.423 9.878 9.878S29.456 33.877 24 33.877c-5.455 0-9.877-4.422-9.877-9.877 0-1.006.15-1.979.432-2.896.202-.66.9-1.031 1.56-.83.66.203 1.032.902.83 1.562-.21.683-.322 1.409-.322 2.164 0 4.074 3.303 7.378 7.377 7.378 4.075 0 7.378-3.304 7.378-7.378 0-4.075-3.303-7.378-7.378-7.378m-2.672 9.031c2.26-.21 4.059-2.01 4.27-4.269.903.544 1.507 1.535 1.507 2.666 0 1.718-1.392 3.111-3.11 3.111-1.132 0-2.122-.604-2.667-1.508"
        })
    });
const $bfbc31c2410be7d0$var$ForwardRef = /*#__PURE__*/ (0, $lrt9t$forwardRef)($bfbc31c2410be7d0$var$SvgComponent);
var $bfbc31c2410be7d0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($bfbc31c2410be7d0$var$ForwardRef);


export {$bfbc31c2410be7d0$export$2e2bcd8739ae039 as default};
