// @react-spectrum/s2/illustrations/linear/internal/S2_lin_rocket_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $6Fm9B$jsxs, jsx as $6Fm9B$jsx} from "react/jsx-runtime";
import {forwardRef as $6Fm9B$forwardRef} from "react";





const $7ab7c238e1c59c95$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6Fm9B$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6Fm9B$jsx)("defs", {
                children: /*#__PURE__*/ (0, $6Fm9B$jsx)("clipPath", {
                    id: "3249f9__a",
                    children: /*#__PURE__*/ (0, $6Fm9B$jsx)("path", {
                        fill: "#fff",
                        d: "M0 0h96v96H0z"
                    })
                })
            }),
            /*#__PURE__*/ (0, $6Fm9B$jsx)("g", {
                clipPath: "url(#3249f9__a)",
                children: /*#__PURE__*/ (0, $6Fm9B$jsx)("path", {
                    fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                    fillRule: "evenodd",
                    d: "M75.788 20.135c-.12-.454-.462-.797-.916-.916-4.934-1.3-16.541-.31-28.113 11.261-8.346 8.347-11.982 13.061-13.554 15.91-.767 1.39-1.003 2.268-1.084 2.858-.061.45-.049.712-.031 1.083.01.219.023.476.023.833 0 1.684.92 3.525 4.563 7.167s5.482 4.563 7.167 4.563c.357 0 .614.012.833.023.37.018.632.03 1.083-.031.59-.08 1.468-.317 2.857-1.084 2.85-1.572 7.564-5.208 15.91-13.554 11.572-11.572 12.56-23.18 11.262-28.113m.102-4.784c1.848.486 3.28 1.918 3.766 3.765 1.634 6.207.261 18.326-10.614 30.198.574 2.913.167 6.402-.731 9.84-1.019 3.894-2.74 7.995-4.792 11.63-2.04 3.615-4.477 6.895-6.987 9.058-2.379 2.05-5.54 3.67-8.566 2.157-2.67-1.335-3.438-3.867-3.7-6.559-.13-1.354-.15-2.911-.153-4.61v-1.086c0-.896 0-1.84-.01-2.842q-.166-.007-.26-.008c-3.464 0-6.343-2.082-9.996-5.735-3.652-3.652-5.734-6.531-5.734-9.995q-.001-.094-.008-.26c-1.003-.01-1.947-.01-2.843-.01h-.004l-1.081-.001c-1.7-.003-3.257-.023-4.611-.154-2.692-.26-5.224-1.029-6.559-3.699-1.513-3.026.106-6.187 2.156-8.566 2.164-2.51 5.444-4.946 9.06-6.987 3.634-2.051 7.734-3.773 11.63-4.791 3.437-.9 6.927-1.306 9.84-.731C57.565 15.09 69.683 13.717 75.89 15.35M50.548 65.304c3.085-1.702 7.571-5.177 14.735-12.18-.081 1.493-.364 3.186-.842 5.017-.921 3.523-2.503 7.304-4.406 10.676-1.913 3.391-4.082 6.243-6.115 7.996-2.165 1.866-3.484 1.948-4.165 1.608-.763-.381-1.284-1.07-1.507-3.367-.11-1.14-.132-2.52-.135-4.232v-1.057c0-1.027 0-2.147-.015-3.35.732-.25 1.539-.608 2.45-1.11m-8.665-35.581c-1.493.081-3.187.363-5.019.842-3.522.921-7.303 2.503-10.676 4.406-3.39 1.913-6.242 4.082-7.995 6.115-1.866 2.165-1.949 3.485-1.608 4.165.38.763 1.069 1.284 3.366 1.507 1.14.11 2.52.132 4.233.135h1.056c1.028 0 2.148 0 3.352.015.25-.732.608-1.538 1.11-2.45 1.703-3.084 5.179-7.571 12.18-14.735m21.912 8.354c-1.896 1.896-4.97 1.896-6.865 0s-1.896-4.97 0-6.865 4.97-1.896 6.865 0 1.896 4.97 0 6.865M27.88 69.956c.781-.782.781-2.048 0-2.829s-2.047-.781-2.828 0l-11.8 11.8c-.781.78-.781 2.047 0 2.828s2.047.781 2.828 0zm6.007 3.178c.78.781.78 2.047 0 2.829l-6.865 6.865c-.781.78-2.048.78-2.829 0-.78-.781-.78-2.048 0-2.829l6.865-6.865c.781-.78 2.048-.78 2.829 0m-12.015-9.186c.781-.78.781-2.047 0-2.828-.78-.781-2.047-.781-2.828 0l-6.866 6.865c-.78.781-.78 2.048 0 2.829.782.78 2.048.78 2.829 0z"
                })
            })
        ]
    });
const $7ab7c238e1c59c95$var$ForwardRef = /*#__PURE__*/ (0, $6Fm9B$forwardRef)($7ab7c238e1c59c95$var$SvgComponent);
var $7ab7c238e1c59c95$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7ab7c238e1c59c95$var$ForwardRef);


export {$7ab7c238e1c59c95$export$2e2bcd8739ae039 as default};
