// @react-spectrum/s2/illustrations/linear/internal/S2_lin_globeGrid_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $cHzDF$jsx} from "react/jsx-runtime";
import {forwardRef as $cHzDF$forwardRef} from "react";





const $d9ac223a496d7605$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cHzDF$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cHzDF$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M38.788 48.285c6.473-8.398 15.455-14.763 25.824-17.97-4.058 5.79-7.33 11.807-9.827 17.97zm-5.078 8H52c-1.957 6.682-3.04 13.486-3.268 20.313H28.11c.47-7.281 2.438-14.154 5.6-20.313m26.669 0c-2.177 6.661-3.386 13.472-3.642 20.313h19.21V56.285zM28.2 84.598h20.654c.439 6.822 1.727 13.627 3.849 20.312h-18.36c-3.35-6.127-5.503-13.003-6.143-20.312m28.67 0c.484 6.824 1.91 13.634 4.261 20.312h14.815V84.598zM39.738 112.91h15.93c2.447 5.768 5.532 11.413 9.243 16.867-9.996-3.026-18.72-8.981-25.173-16.867m24.689 0c2.987 6.322 6.832 12.478 11.521 18.363V112.91zm11.521-84.237c-5.219 6.235-9.364 12.817-12.45 19.612h12.45zm8 47.925V56.285h15.674c2.178 6.661 3.387 13.472 3.642 20.313zm19.182 8H83.947v20.312h14.921c2.352-6.678 3.777-13.488 4.261-20.312m4.168 20.312c2.121-6.685 3.409-13.49 3.848-20.312H131.8c-.641 7.31-2.793 14.185-6.144 20.312zm-11.722 8H83.947v18.495c4.737-5.925 8.618-12.126 11.627-18.495m-.485 16.867c3.712-5.454 6.796-11.099 9.244-16.867h15.929c-6.453 7.885-15.177 13.84-25.173 16.867m16.179-53.18c-.227-6.826-1.311-13.63-3.267-20.312h18.288c3.162 6.16 5.13 13.032 5.6 20.313zm-6.054-28.312h15.997c-6.472-8.398-15.455-14.763-25.824-17.97 4.058 5.79 7.329 11.807 9.827 17.97M83.947 28.547v19.738h12.557c-3.108-6.841-7.288-13.466-12.557-19.738M20 80c0-33.137 26.863-60 60-60s60 26.863 60 60-26.863 60-60 60-60-26.863-60-60"
        })
    });
const $d9ac223a496d7605$var$ForwardRef = /*#__PURE__*/ (0, $cHzDF$forwardRef)($d9ac223a496d7605$var$SvgComponent);
var $d9ac223a496d7605$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d9ac223a496d7605$var$ForwardRef);


export {$d9ac223a496d7605$export$2e2bcd8739ae039 as default};
