// @react-spectrum/s2/illustrations/linear/internal/S2_lin_ribbon_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $afQWo$jsx} from "react/jsx-runtime";
import {forwardRef as $afQWo$forwardRef} from "react";





const $b3513801a24fea4d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $afQWo$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $afQWo$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 8.75c-5.247 0-9.5 4.253-9.5 9.5s4.253 9.5 9.5 9.5 9.5-4.253 9.5-9.5-4.254-9.5-9.5-9.5m-12 9.5c0-6.627 5.372-12 12-12 6.627 0 12 5.373 12 12 0 3.261-1.301 6.218-3.412 8.381l3.407 8.435c.668 1.652-.758 3.387-2.508 3.053l-2.081-.398-1.22 1.732c-1.027 1.456-3.259 1.198-3.926-.454l-2.275-5.63-2.234 5.529c-.673 1.666-2.932 1.91-3.945.425l-1.187-1.74-2.062.428c-1.76.364-3.215-1.38-2.542-3.047l3.376-8.354c-2.1-2.16-3.392-5.11-3.392-8.36m5.412 10.032-2.904 7.186 1.768-.366c.89-.184 1.802.185 2.314.935l1.018 1.491 2.974-7.36c-1.894-.224-3.652-.887-5.17-1.886m7.977 1.888c1.895-.218 3.655-.878 5.175-1.873l2.942 7.28-1.79-.342c-.867-.166-1.751.192-2.26.914l-1.05 1.489z"
        })
    });
const $b3513801a24fea4d$var$ForwardRef = /*#__PURE__*/ (0, $afQWo$forwardRef)($b3513801a24fea4d$var$SvgComponent);
var $b3513801a24fea4d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b3513801a24fea4d$var$ForwardRef);


export {$b3513801a24fea4d$export$2e2bcd8739ae039 as default};
