// @react-spectrum/s2/illustrations/linear/internal/S2_lin_sparkles_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $36xSO$reactjsxruntime = require("react/jsx-runtime");
var $36xSO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $57490801d790e6bb$export$2e2bcd8739ae039);




const $57490801d790e6bb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $36xSO$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $36xSO$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M76.478 21.615c1.46-6.507-6.34-11.015-11.245-6.493l-7.8 7.188c-.675.623-1.617.875-2.51.672h-.004l-10.35-2.328c-6.508-1.462-11.016 6.339-6.493 11.244l7.188 7.8c.622.676.874 1.618.672 2.511l-.001.003-2.328 10.35c-1.461 6.508 6.34 11.016 11.245 6.494l7.8-7.188c.675-.623 1.617-.875 2.51-.672h.003l10.35 2.328h.001c6.507 1.462 11.015-6.339 6.493-11.244l-7.188-7.8v-.001c-.623-.675-.875-1.617-.672-2.51v-.003zm-8.534-3.552c2.018-1.86 5.233-.01 4.631 2.675l-2.327 10.347v.001c-.494 2.178.119 4.464 1.631 6.104h.001l7.188 7.8c1.86 2.019.01 5.234-2.675 4.632l-10.347-2.328h-.001c-2.178-.493-4.464.12-6.105 1.633l-7.8 7.187c-2.018 1.861-5.233.01-4.63-2.674l2.327-10.347v-.002c.493-2.178-.12-4.463-1.633-6.104l-7.187-7.8c-1.861-2.018-.01-5.233 2.674-4.63h.001l10.346 2.327h.002c2.178.493 4.463-.12 6.104-1.632zm-29.153 60.78-7.296-1.644c-.966-.216-1.968.054-2.694.72l-5.496 5.07c-2.166 1.998-5.61.006-4.962-2.862l1.644-7.296c.216-.966-.054-1.968-.72-2.694l-5.07-5.496c-1.998-2.166-.006-5.61 2.862-4.962l7.296 1.644c.966.216 1.968-.054 2.694-.72l5.496-5.07c2.166-1.998 5.61-.006 4.962 2.862l-1.644 7.296c-.216.966.054 1.968.72 2.694l5.07 5.496c1.998 2.166.006 5.61-2.862 4.962"
        })
    });
const $57490801d790e6bb$var$ForwardRef = /*#__PURE__*/ (0, $36xSO$react.forwardRef)($57490801d790e6bb$var$SvgComponent);
var $57490801d790e6bb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($57490801d790e6bb$var$ForwardRef);
