// @react-spectrum/s2/illustrations/linear/internal/S2_lin_chartAreaStack_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $cPLpc$jsx} from "react/jsx-runtime";
import {forwardRef as $cPLpc$forwardRef} from "react";





const $83d1f8e450951a98$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cPLpc$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cPLpc$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M135.852 41.038c0-8.268-11.013-11.103-15.006-3.863l-13.107 23.77-11.127-13.61c-3.434-4.202-9.965-3.838-12.911.718L67.6 72.954 55.29 58.726c-2.98-3.443-8.227-3.709-11.54-.585L26.72 74.206c-1.603 1.511-2.511 3.617-2.511 5.82v43.725c0 6.627 5.373 12 12 12h87.643c6.627 0 12-5.373 12-12zm-8 50.699V41.038l-13.108 23.77c-2.734 4.96-9.615 5.585-13.199 1.2l-11.126-13.61-16.1 24.9c-2.896 4.478-9.28 4.923-12.768.89L49.24 63.961 32.209 80.025v25.582l16.856-11.305c2.895-1.941 6.714-1.785 9.44.386l14.79 11.777L91.33 85.198c3.042-3.586 8.505-3.789 11.803-.438l12.537 12.736zm0 8.849-8.764 4.143c-3.075 1.453-6.734.803-9.12-1.621L97.431 90.372 79.397 111.64c-2.795 3.297-7.704 3.777-11.085 1.085l-14.79-11.778-20.757 13.921q-.27.181-.556.312v8.572c0 2.209 1.79 4 4 4h87.643c2.209 0 4-1.791 4-4z"
        })
    });
const $83d1f8e450951a98$var$ForwardRef = /*#__PURE__*/ (0, $cPLpc$forwardRef)($83d1f8e450951a98$var$SvgComponent);
var $83d1f8e450951a98$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($83d1f8e450951a98$var$ForwardRef);


export {$83d1f8e450951a98$export$2e2bcd8739ae039 as default};
