// @react-spectrum/s2/illustrations/linear/internal/S2_lin_check_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $fL6et$jsx} from "react/jsx-runtime";
import {forwardRef as $fL6et$forwardRef} from "react";





const $5f7caa533f6cff8e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fL6et$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fL6et$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8.25 24c0-8.698 7.052-15.75 15.75-15.75S39.75 15.302 39.75 24 32.699 39.75 24 39.75 8.25 32.699 8.25 24M24 5.75C13.92 5.75 5.75 13.92 5.75 24S13.92 42.25 24 42.25 42.25 34.08 42.25 24 34.08 5.75 24 5.75m8.336 14.68c.513-.463.555-1.253.093-1.766s-1.252-.555-1.765-.093l-9.119 8.206-3.661-3.66c-.488-.489-1.28-.489-1.768 0-.488.487-.488 1.279 0 1.767l4.366 4.366c.545.545 1.422.568 1.995.052z"
        })
    });
const $5f7caa533f6cff8e$var$ForwardRef = /*#__PURE__*/ (0, $fL6et$forwardRef)($5f7caa533f6cff8e$var$SvgComponent);
var $5f7caa533f6cff8e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5f7caa533f6cff8e$var$ForwardRef);


export {$5f7caa533f6cff8e$export$2e2bcd8739ae039 as default};
