// @react-spectrum/s2/illustrations/linear/internal/S2_lin_link_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $2lbLJ$jsx} from "react/jsx-runtime";
import {forwardRef as $2lbLJ$forwardRef} from "react";





const $96c7df4301e1b7c6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2lbLJ$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2lbLJ$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M37.84 16.965c1.888-1.889 1.888-4.95 0-6.839-1.888-1.888-4.95-1.888-6.838 0l-3.5 3.5c-.488.489-1.28.489-1.768 0-.488-.488-.488-1.28 0-1.767l3.5-3.5c2.865-2.865 7.51-2.865 10.374 0 2.865 2.864 2.865 7.509 0 10.373l-8.875 8.876c-2.865 2.864-7.51 2.864-10.374 0-.488-.489-.488-1.28 0-1.768s1.28-.488 1.768 0c1.888 1.888 4.95 1.888 6.838 0zm-10.199 3.427c.489.488.489 1.28 0 1.768-.488.488-1.28.488-1.767 0-1.889-1.889-4.95-1.889-6.839 0l-8.875 8.875c-1.888 1.888-1.888 4.95 0 6.839 1.889 1.888 4.95 1.888 6.839 0l3.5-3.5c.488-.489 1.28-.489 1.767 0 .489.488.489 1.279 0 1.767l-3.5 3.5c-2.864 2.865-7.509 2.865-10.373 0-2.865-2.864-2.865-7.509 0-10.374l8.875-8.875c2.864-2.864 7.509-2.864 10.373 0"
        })
    });
const $96c7df4301e1b7c6$var$ForwardRef = /*#__PURE__*/ (0, $2lbLJ$forwardRef)($96c7df4301e1b7c6$var$SvgComponent);
var $96c7df4301e1b7c6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($96c7df4301e1b7c6$var$ForwardRef);


export {$96c7df4301e1b7c6$export$2e2bcd8739ae039 as default};
