// @react-spectrum/s2/illustrations/linear/internal/S2_lin_heart_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $eFwuM$reactjsxruntime = require("react/jsx-runtime");
var $eFwuM$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $92fc0c79eeb5661b$export$2e2bcd8739ae039);




const $92fc0c79eeb5661b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eFwuM$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $eFwuM$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20.115 58.164C18.595 38.97 32.334 21.988 51.993 22 64.066 22.007 74.55 28.74 80 38.658 85.448 28.74 95.932 22.008 108.005 22c19.659-.012 33.399 16.967 31.878 36.16-1.318 16.653-10.843 33.862-21.387 48.024-10.612 14.254-22.698 26.002-29.946 31.807-5.012 4.013-12.114 4.012-17.125-.002-7.246-5.805-19.325-17.553-29.932-31.806-10.54-14.16-20.059-31.368-21.378-48.02M51.988 30c-14.443-.009-25.095 12.43-23.898 27.532 1.138 14.365 9.551 30.077 19.82 43.875 10.201 13.707 21.805 24.961 28.517 30.338 2.087 1.673 5.034 1.673 7.122.001 6.715-5.377 18.324-16.631 28.53-30.339 10.274-13.8 18.692-29.512 19.829-43.878 1.196-15.101-9.455-27.538-23.898-27.529-13.223.008-23.999 10.834-23.999 24.262q0 .382-.072.755l-.019.1c-.377 1.873-2.026 3.217-3.936 3.21s-3.548-1.363-3.912-3.238l-.014-.078q-.07-.371-.07-.75c0-13.427-10.778-24.253-24-24.261m24.059 24.947v.002z"
        })
    });
const $92fc0c79eeb5661b$var$ForwardRef = /*#__PURE__*/ (0, $eFwuM$react.forwardRef)($92fc0c79eeb5661b$var$SvgComponent);
var $92fc0c79eeb5661b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($92fc0c79eeb5661b$var$ForwardRef);
