// @react-spectrum/s2/illustrations/linear/internal/S2_lin_update_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $hQir0$reactjsxruntime = require("react/jsx-runtime");
var $hQir0$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $566f8cd2b0e8f243$export$2e2bcd8739ae039);




const $566f8cd2b0e8f243$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hQir0$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hQir0$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M18.28 39.32c3.749-12.85 15.623-22.24 29.69-22.24 12.725 0 23.664 7.692 28.404 18.68l-8.999-4.193c-1.001-.466-2.19-.033-2.657.968-.467 1.002-.033 2.192.968 2.658l12.45 5.8c.977.455 2.139.054 2.627-.906l6.45-12.68c.501-.985.109-2.189-.875-2.69-.985-.5-2.19-.108-2.69.876l-3.958 7.782C74.16 21.4 62.036 13.08 47.97 13.08c-15.892 0-29.297 10.61-33.53 25.12-.308 1.06.3 2.17 1.36 2.48s2.171-.3 2.48-1.36m-.414 15.798c-.976-.456-2.14-.056-2.628.905l-6.45 12.68c-.5.985-.109 2.189.876 2.69s2.189.108 2.69-.876l3.927-7.72C21.847 74.678 33.922 82.92 47.92 82.92c15.892 0 29.297-10.61 33.53-25.12.308-1.06-.3-2.17-1.36-2.48s-2.172.3-2.48 1.36c-3.749 12.85-15.623 22.24-29.69 22.24-12.696 0-23.613-7.656-28.371-18.603l9.075 4.235c1 .467 2.191.035 2.658-.966s.034-2.191-.966-2.658z"
        })
    });
const $566f8cd2b0e8f243$var$ForwardRef = /*#__PURE__*/ (0, $hQir0$react.forwardRef)($566f8cd2b0e8f243$var$SvgComponent);
var $566f8cd2b0e8f243$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($566f8cd2b0e8f243$var$ForwardRef);
