// @react-spectrum/s2/illustrations/linear/internal/S2_lin_play_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $drIC5$jsx} from "react/jsx-runtime";
import {forwardRef as $drIC5$forwardRef} from "react";





const $22953a45231357c1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $drIC5$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $drIC5$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28 79.74c0-28.719 23.281-52 52-52s52 23.281 52 52-23.281 52-52 52-52-23.281-52-52m52-60c-33.137 0-60 26.863-60 60s26.863 60 60 60 60-26.863 60-60-26.863-60-60-60m-5.189 44.94L95.744 76.8c2.344 1.357 2.34 4.743-.007 6.095L74.775 94.964c-2.348 1.351-5.278-.345-5.275-3.054l.029-24.188c.003-2.71 2.938-4.399 5.282-3.041"
        })
    });
const $22953a45231357c1$var$ForwardRef = /*#__PURE__*/ (0, $drIC5$forwardRef)($22953a45231357c1$var$SvgComponent);
var $22953a45231357c1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($22953a45231357c1$var$ForwardRef);


export {$22953a45231357c1$export$2e2bcd8739ae039 as default};
