// @react-spectrum/s2/illustrations/linear/internal/S2_lin_megaphonePromote_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $5A6l3$reactjsxruntime = require("react/jsx-runtime");
var $5A6l3$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0a8b6b96db0e62a8$export$2e2bcd8739ae039);




const $0a8b6b96db0e62a8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5A6l3$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5A6l3$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80.242 24.854c0-5.27-5.09-9.052-10.137-7.52h-.001l-25.017 7.582H27.29c-7.888 0-14.29 6.402-14.29 14.29s6.402 14.29 14.29 14.29h1.043l.702 1.605c.742 1.7 1.748 4 2.8 6.405 2.102 4.802 4.398 10.037 5.148 11.707 1.15 2.558 3.103 4.128 5.331 4.753 2.174.61 4.482.29 6.411-.647 1.8-.874 3.385-2.34 4.252-4.282.886-1.982.958-4.32-.061-6.731l-.008-.02-5.23-12.012 22.42 6.815c5.044 1.532 10.144-2.239 10.144-7.52zm-36.804 4.062v20.58H27.29c-5.679 0-10.29-4.611-10.29-10.29s4.611-10.29 10.29-10.29zm4 21.105 23.822 7.24c2.479.753 4.982-1.1 4.982-3.692V24.854c0-2.59-2.501-4.444-4.976-3.692l-23.828 7.222zM32.7 53.503c.743 1.699 1.747 3.998 2.799 6.4 2.106 4.813 4.391 10.023 5.131 11.67.674 1.498 1.7 2.243 2.765 2.542 1.12.315 2.422.17 3.582-.393 1.076-.523 1.916-1.351 2.347-2.316.412-.923.514-2.1-.09-3.533l-6.255-14.37z"
        })
    });
const $0a8b6b96db0e62a8$var$ForwardRef = /*#__PURE__*/ (0, $5A6l3$react.forwardRef)($0a8b6b96db0e62a8$var$SvgComponent);
var $0a8b6b96db0e62a8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0a8b6b96db0e62a8$var$ForwardRef);
