// @react-spectrum/s2/illustrations/linear/internal/S2_lin_layers_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $aSiLL$jsx} from "react/jsx-runtime";
import {forwardRef as $aSiLL$forwardRef} from "react";





const $598d787fd267ca3c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aSiLL$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $aSiLL$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28.622 54.894c-6.113 4.168-6.113 13.182 0 17.35l43.058 29.358c4.758 3.243 11.016 3.243 15.774 0l43.058-29.358c6.113-4.168 6.113-13.182 0-17.35L87.454 25.536c-4.758-3.243-11.016-3.243-15.773 0zm4.507 10.74c-1.456-.992-1.456-3.138 0-4.13l43.058-29.358c2.039-1.39 4.721-1.39 6.76 0l43.058 29.357c1.456.993 1.456 3.14 0 4.132L82.947 94.992c-2.039 1.39-4.721 1.39-6.76 0zm0 33.5c-1.456-.992-1.456-3.138 0-4.13l6.265-4.272c1.825-1.245 2.296-3.733 1.052-5.558-1.245-1.826-3.733-2.297-5.559-1.052l-6.265 4.272c-6.113 4.168-6.113 13.182 0 17.35l43.058 29.358c4.758 3.243 11.016 3.243 15.774 0l43.058-29.358c6.113-4.168 6.113-13.182 0-17.35l-6.265-4.272c-1.825-1.245-4.314-.774-5.558 1.052-1.245 1.825-.774 4.313 1.051 5.558l6.265 4.271c1.456.993 1.456 3.14 0 4.132l-43.058 29.357c-2.039 1.39-4.721 1.39-6.76 0z"
        })
    });
const $598d787fd267ca3c$var$ForwardRef = /*#__PURE__*/ (0, $aSiLL$forwardRef)($598d787fd267ca3c$var$SvgComponent);
var $598d787fd267ca3c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($598d787fd267ca3c$var$ForwardRef);


export {$598d787fd267ca3c$export$2e2bcd8739ae039 as default};
