// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pencil_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bvaAB$reactjsxruntime = require("react/jsx-runtime");
var $bvaAB$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ea0d6bd7664b1efd$export$2e2bcd8739ae039);




const $ea0d6bd7664b1efd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bvaAB$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bvaAB$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M21.898 57.87c.301-.803.77-1.533 1.377-2.139l37.21-37.21c4.694-4.694 12.304-4.694 16.997 0 4.694 4.692 4.694 12.302 0 16.996l-37.21 37.21c-.607.607-1.337 1.076-2.14 1.377l-17.72 6.635c-3.215 1.203-6.353-1.934-5.149-5.149zm5.049-.154 11.34 11.34 30.698-30.7-11.34-11.339zm7.747 13.404-9.812-9.812-5.873 15.685zm39.96-38.432-2.84 2.84-11.34-11.34 2.84-2.84c3.131-3.13 8.208-3.13 11.34 0 3.131 3.132 3.131 8.209 0 11.34"
        })
    });
const $ea0d6bd7664b1efd$var$ForwardRef = /*#__PURE__*/ (0, $bvaAB$react.forwardRef)($ea0d6bd7664b1efd$var$SvgComponent);
var $ea0d6bd7664b1efd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ea0d6bd7664b1efd$var$ForwardRef);
