// @react-spectrum/s2/illustrations/linear/internal/S2_lin_paperclip_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dEuip$reactjsxruntime = require("react/jsx-runtime");
var $dEuip$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9abed338efe25229$export$2e2bcd8739ae039);




const $9abed338efe25229$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dEuip$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dEuip$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M35.353 89.552c-11.138 11.083-11.137 29.061 0 40.144 11.127 11.072 29.16 11.072 40.287 0l45.068-44.847c1.566-1.558 1.573-4.09.014-5.657-1.558-1.565-4.09-1.572-5.656-.013l-45.07 44.846c-8.005 7.967-20.994 7.967-29 0-7.995-7.956-7.995-20.846 0-28.802l61.724-61.42c5.097-5.07 13.369-5.07 18.466 0 5.085 5.062 5.085 13.259 0 18.32L65.86 107.175c-2.128 2.118-5.587 2.118-7.715 0-2.117-2.107-2.117-5.514 0-7.621l38.67-38.48c1.566-1.558 1.573-4.09.014-5.657-1.558-1.566-4.09-1.572-5.656-.014l-38.67 38.48c-5.26 5.234-5.26 13.729 0 18.963 5.249 5.223 13.751 5.223 19 0l55.325-55.052c8.229-8.189 8.229-21.473 0-29.661-8.218-8.177-21.533-8.177-29.751 0z"
        })
    });
const $9abed338efe25229$var$ForwardRef = /*#__PURE__*/ (0, $dEuip$react.forwardRef)($9abed338efe25229$var$SvgComponent);
var $9abed338efe25229$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9abed338efe25229$var$ForwardRef);
