// @react-spectrum/s2/illustrations/linear/internal/S2_lin_codeBrackets_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dVUHc$reactjsxruntime = require("react/jsx-runtime");
var $dVUHc$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3a32318a8b4a6683$export$2e2bcd8739ae039);




const $3a32318a8b4a6683$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dVUHc$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dVUHc$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16 28.8c0-3.313 2.686-6 6-6h52c3.314 0 6 2.687 6 6v38.4c0 3.314-2.686 6-6 6H22c-3.314 0-6-2.686-6-6zm6-10c-5.523 0-10 4.478-10 10v38.4c0 5.524 4.477 10 10 10h52c5.523 0 10-4.476 10-10V28.8c0-5.522-4.477-10-10-10zm15.985 19.825c.78-.781.78-2.047 0-2.828-.781-.781-2.048-.781-2.829 0l-10.42 10.42c-.762.762-.783 1.99-.047 2.779l10.42 11.16c.753.807 2.019.85 2.826.097.808-.754.851-2.02.097-2.827l-9.1-9.748zm20.08-2.876c.808-.754 2.074-.71 2.828.097l10.42 11.16c.735.788.714 2.017-.048 2.779l-10.42 10.42c-.781.781-2.047.781-2.829 0-.78-.781-.78-2.047 0-2.828l9.054-9.054-9.101-9.747c-.754-.808-.71-2.073.097-2.827m-3.31-3.138c.32-1.057-.277-2.174-1.335-2.494s-2.173.277-2.494 1.334l-9.68 31.94c-.32 1.057.277 2.173 1.334 2.494 1.057.32 2.174-.277 2.494-1.334z"
        })
    });
const $3a32318a8b4a6683$var$ForwardRef = /*#__PURE__*/ (0, $dVUHc$react.forwardRef)($3a32318a8b4a6683$var$SvgComponent);
var $3a32318a8b4a6683$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3a32318a8b4a6683$var$ForwardRef);
