// @react-spectrum/s2/illustrations/linear/internal/S2_lin_rotateCW_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $hwWor$reactjsxruntime = require("react/jsx-runtime");
var $hwWor$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $785e667040a101c2$export$2e2bcd8739ae039);




const $785e667040a101c2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hwWor$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hwWor$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M26.502 80.398c0-28.718 23.281-52 52-52 19.918 0 37.227 11.2 45.962 27.655l-15.454-5.156c-2.095-.7-4.361.432-5.06 2.528s.433 4.361 2.528 5.06l24.275 8.1c2.095.699 4.361-.433 5.06-2.529l8.099-24.274c.7-2.096-.432-4.361-2.528-5.06-2.095-.7-4.361.432-5.06 2.528l-4.935 14.79c-10.118-18.831-30.003-31.642-52.887-31.642-33.137 0-60 26.863-60 60s26.863 60 60 60c25.917 0 47.986-16.429 56.38-39.428.757-2.075-.311-4.372-2.387-5.13-2.075-.757-4.371.312-5.129 2.387-7.278 19.943-26.415 34.171-48.864 34.171-28.719 0-52-23.281-52-52"
        })
    });
const $785e667040a101c2$var$ForwardRef = /*#__PURE__*/ (0, $hwWor$react.forwardRef)($785e667040a101c2$var$SvgComponent);
var $785e667040a101c2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($785e667040a101c2$var$ForwardRef);
