// @react-spectrum/s2/illustrations/linear/internal/S2_lin_channel_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $98E0g$reactjsxruntime = require("react/jsx-runtime");
var $98E0g$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f4ca5a95ead3c1b7$export$2e2bcd8739ae039);




const $f4ca5a95ead3c1b7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $98E0g$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $98E0g$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M56.912 47.773h.007l8.452 14.638c-3.96 3.345-6.758 8.021-7.704 13.338H40.783c-1.38-2.402-3.972-4.02-6.941-4.02-4.418 0-8 3.581-8 8 0 4.418 3.582 8 8 8 2.953 0 5.532-1.6 6.918-3.98h16.906c.946 5.318 3.744 9.994 7.704 13.34l-8.427 14.596h-.031c-4.419 0-8 3.581-8 8 0 4.418 3.581 8 8 8 4.418 0 8-3.582 8-8 0-1.447-.384-2.804-1.056-3.975l8.44-14.617c2.405.869 5 1.342 7.704 1.342 2.705 0 5.3-.473 7.705-1.341l8.419 14.581c-.684 1.179-1.076 2.549-1.076 4.01 0 4.418 3.582 8 8 8s8-3.582 8-8c0-4.416-3.576-7.995-7.99-8L94.63 97.088c3.96-3.344 6.759-8.021 7.705-13.339H119.2c1.386 2.38 3.965 3.98 6.918 3.98 4.418 0 8-3.582 8-8s-3.582-8-8-8c-2.97 0-5.562 1.618-6.942 4.02h-16.842c-.946-5.317-3.745-9.994-7.704-13.338l8.451-14.638c4.403-.017 7.967-3.592 7.967-8 0-4.418-3.581-8-8-8-4.418 0-8 3.582-8 8 0 1.47.396 2.846 1.088 4.03l-8.432 14.603c-2.405-.868-4.999-1.341-7.704-1.341s-5.299.473-7.704 1.341l-8.452-14.638c.68-1.175 1.068-2.54 1.068-3.995 0-4.418-3.582-8-8-8s-8 3.582-8 8 3.581 8 8 8M80 65.064c-8.11 0-14.685 6.575-14.685 14.686 0 8.11 6.575 14.685 14.685 14.685 8.111 0 14.686-6.574 14.686-14.685 0-8.11-6.575-14.686-14.686-14.686"
        })
    });
const $f4ca5a95ead3c1b7$var$ForwardRef = /*#__PURE__*/ (0, $98E0g$react.forwardRef)($f4ca5a95ead3c1b7$var$SvgComponent);
var $f4ca5a95ead3c1b7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f4ca5a95ead3c1b7$var$ForwardRef);
