// @react-spectrum/s2/illustrations/linear/internal/S2_lin_filmstrip_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $iXWxZ$jsx} from "react/jsx-runtime";
import {forwardRef as $iXWxZ$forwardRef} from "react";





const $91289bc47dea8554$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iXWxZ$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $iXWxZ$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M38 11.45c1.519 0 2.75 1.23 2.75 2.75v19.6c0 1.518-1.231 2.75-2.75 2.75H10c-1.519 0-2.75-1.232-2.75-2.75V14.2c0-1.52 1.231-2.75 2.75-2.75zm5.25 2.75c0-2.9-2.35-5.25-5.25-5.25H10c-2.9 0-5.25 2.35-5.25 5.25v19.6c0 2.899 2.35 5.25 5.25 5.25h28c2.9 0 5.25-2.351 5.25-5.25zM14.7 15.9c0-.332.268-.6.6-.6h2.1c.33 0 .6.268.6.6v1.5c0 .33-.27.6-.6.6h-2.1c-.332 0-.6-.27-.6-.6zm-4.502-.6c-.332 0-.6.268-.6.6v1.5c0 .33.268.6.6.6h2.1c.331 0 .6-.27.6-.6v-1.5c0-.332-.269-.6-.6-.6zm4.501 15.299c0-.331.269-.6.6-.6h2.1c.332 0 .6.269.6.6v1.5c0 .331-.268.6-.6.6h-2.1c-.331 0-.6-.269-.6-.6zm-4.501-.6c-.332 0-.6.269-.6.6v1.5c0 .331.268.6.6.6h2.1c.331 0 .6-.269.6-.6v-1.5c0-.331-.269-.6-.6-.6zm9.6-14.1c0-.331.27-.6.6-.6h2.1c.332 0 .6.269.6.6v1.5c0 .331-.268.6-.6.6h-2.1c-.33 0-.6-.269-.6-.6zm.6 14.1c-.33 0-.6.269-.6.6v1.5c0 .331.27.6.6.6h2.1c.332 0 .6-.269.6-.6v-1.5c0-.331-.268-.6-.6-.6zm4.5-14.1c0-.331.27-.6.6-.6h2.1c.332 0 .6.269.6.6v1.5c0 .331-.268.6-.6.6h-2.1c-.33 0-.6-.269-.6-.6zm.6 14.1c-.33 0-.6.269-.6.6v1.5c0 .331.27.6.6.6h2.1c.332 0 .6-.269.6-.6v-1.5c0-.331-.268-.6-.6-.6zM30 15.899c0-.331.269-.6.6-.6h2.1c.331 0 .6.269.6.6v1.5c0 .331-.269.6-.6.6h-2.1c-.331 0-.6-.269-.6-.6zm.6 14.1c-.331 0-.6.269-.6.6v1.5c0 .331.269.6.6.6h2.1c.331 0 .6-.269.6-.6v-1.5c0-.331-.269-.6-.6-.6zm4.498-14.1c0-.331.268-.6.6-.6h2.1c.331 0 .6.269.6.6v1.5c0 .331-.269.6-.6.6h-2.1c-.332 0-.6-.269-.6-.6zm.6 14.1c-.332 0-.6.269-.6.6v1.5c0 .331.268.6.6.6h2.1c.331 0 .6-.269.6-.6v-1.5c0-.331-.269-.6-.6-.6z"
        })
    });
const $91289bc47dea8554$var$ForwardRef = /*#__PURE__*/ (0, $iXWxZ$forwardRef)($91289bc47dea8554$var$SvgComponent);
var $91289bc47dea8554$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($91289bc47dea8554$var$ForwardRef);


export {$91289bc47dea8554$export$2e2bcd8739ae039 as default};
