// @react-spectrum/s2/illustrations/linear/internal/S2_lin_tag_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3Sw8l$reactjsxruntime = require("react/jsx-runtime");
var $3Sw8l$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $95931bf5b941578b$export$2e2bcd8739ae039);




const $95931bf5b941578b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3Sw8l$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3Sw8l$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M31.051 20C24.987 19.925 20 24.802 20 30.883V67.06c0 2.892 1.154 5.662 3.203 7.701l60.305 60.042c6.96 6.929 18.237 6.929 25.196 0l26.069-25.955c6.969-6.938 6.969-18.196 0-25.135L74.431 23.635c-2.014-2.005-4.731-3.144-7.572-3.18zM28 30.883c0-1.59 1.31-2.902 2.95-2.882l35.808.454c.765.01 1.493.317 2.028.85l60.343 60.079c3.828 3.811 3.828 9.984 0 13.796l-26.069 25.955c-3.838 3.821-10.069 3.821-13.907 0L28.847 69.092C28.303 68.55 28 67.819 28 67.06zM59 51c0 4.418-3.582 8-8 8s-8-3.582-8-8 3.582-8 8-8 8 3.582 8 8"
        })
    });
const $95931bf5b941578b$var$ForwardRef = /*#__PURE__*/ (0, $3Sw8l$react.forwardRef)($95931bf5b941578b$var$SvgComponent);
var $95931bf5b941578b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($95931bf5b941578b$var$ForwardRef);
