// @react-spectrum/s2/illustrations/linear/internal/S2_lin_imageStack_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $4v32W$reactjsxruntime = require("react/jsx-runtime");
var $4v32W$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ed0a4eae8b700138$export$2e2bcd8739ae039);




const $ed0a4eae8b700138$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4v32W$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4v32W$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M30.04 19.867c1.38 0 2.5-1.12 2.5-2.5s-1.12-2.5-2.5-2.5c-1.382 0-2.5 1.12-2.5 2.5s1.118 2.5 2.5 2.5"
            }),
            /*#__PURE__*/ (0, $4v32W$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M5.813 15.13c0-3.213 2.604-5.817 5.816-5.817h21.122c3.213 0 5.817 2.604 5.817 5.817v13.986c0 3.212-2.604 5.817-5.817 5.817H11.63c-3.212 0-5.816-2.605-5.816-5.817zm5.816-3.317c-1.831 0-3.316 1.485-3.316 3.317v13.986q0 .29.048.567l5.921-9.179c1.24-1.92 4.059-1.89 5.256.058l4.744 7.72 2.709-4.134c1.24-1.891 4.022-1.865 5.226.05l2.32 3.69 1.387 2.197c.094-.306.144-.632.144-.97V15.13c0-1.832-1.485-3.317-3.317-3.317zM34.23 32.086l-1.808-2.865-2.321-3.693c-.235-.373-.777-.378-1.02-.01l-4.53 6.915h8.2c.532 0 1.034-.125 1.48-.347m-12.668.347 1.205-1.84-5.36-8.722c-.233-.38-.782-.386-1.024-.011l-6.5 10.076c.507.315 1.105.497 1.746.497z"
            }),
            /*#__PURE__*/ (0, $4v32W$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M42.87 20.419c0-.69-.56-1.25-1.25-1.25-.691 0-1.25.56-1.25 1.25v11.56c0 2.462-1.997 4.458-4.46 4.458H17.073c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h18.839c3.843 0 6.958-3.115 6.958-6.958z"
            })
        ]
    });
const $ed0a4eae8b700138$var$ForwardRef = /*#__PURE__*/ (0, $4v32W$react.forwardRef)($ed0a4eae8b700138$var$SvgComponent);
var $ed0a4eae8b700138$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ed0a4eae8b700138$var$ForwardRef);
