// @react-spectrum/s2/illustrations/linear/internal/S2_lin_clock_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $ayiur$jsx} from "react/jsx-runtime";
import {forwardRef as $ayiur$forwardRef} from "react";





const $8a1986ae343e6319$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ayiur$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $ayiur$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 16.74c-17.12 0-31 13.88-31 31 0 17.121 13.88 31 31 31s31-13.879 31-31c0-17.12-13.88-31-31-31m-35 31c0-19.33 15.67-35 35-35s35 15.67 35 35-15.67 35-35 35-35-15.67-35-35m37-1.154V24.74c0-1.104-.895-2-2-2s-2 .896-2 2v22.962q-.002.12.01.238c.029.29.119.56.258.8.138.24.328.455.565.624q.096.07.2.127l12.826 7.405c.956.553 2.18.225 2.732-.732.552-.956.224-2.18-.732-2.732z"
        })
    });
const $8a1986ae343e6319$var$ForwardRef = /*#__PURE__*/ (0, $ayiur$forwardRef)($8a1986ae343e6319$var$SvgComponent);
var $8a1986ae343e6319$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8a1986ae343e6319$var$ForwardRef);


export {$8a1986ae343e6319$export$2e2bcd8739ae039 as default};
