// @react-spectrum/s2/illustrations/linear/internal/S2_lin_exposure_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $gUbt3$jsx} from "react/jsx-runtime";
import {forwardRef as $gUbt3$forwardRef} from "react";





const $4dcbb86f584e5d6a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gUbt3$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gUbt3$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48.755 79.829q-.397.01-.797.01c-11.989 0-22.402-6.748-27.65-16.652H54.01zM16.687 48.568c0 3.728.653 7.303 1.85 10.619h17.428l-10.503-32.34c-5.432 5.625-8.774 13.283-8.774 21.72m12.03-24.652c5.305-4.147 11.985-6.62 19.241-6.62 5.109 0 9.931 1.226 14.19 3.398L34.252 40.962zm6.805 20.957q.28-.094.533-.276l11.652-8.466 12.089 8.734-4.53 14.341q-.137-.02-.282-.02H40.171l-.004-.012zm17.556 34.548c11.735-1.932 21.268-10.398 24.74-21.537L63.212 47.332zm26.151-30.853c0-10.607-5.281-19.981-13.357-25.635l-14.76 10.724L78.815 53.67c.272-1.66.414-3.365.414-5.102m-31.27 35.27c19.479 0 35.27-15.79 35.27-35.27S67.438 13.297 47.96 13.297s-35.271 15.791-35.271 35.27 15.79 35.271 35.27 35.271"
        })
    });
const $4dcbb86f584e5d6a$var$ForwardRef = /*#__PURE__*/ (0, $gUbt3$forwardRef)($4dcbb86f584e5d6a$var$SvgComponent);
var $4dcbb86f584e5d6a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($4dcbb86f584e5d6a$var$ForwardRef);


export {$4dcbb86f584e5d6a$export$2e2bcd8739ae039 as default};
