// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noSharedFile_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1PZ9a$reactjsxruntime = require("react/jsx-runtime");
var $1PZ9a$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a6187d2a2ae66251$export$2e2bcd8739ae039);




const $a6187d2a2ae66251$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1PZ9a$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1PZ9a$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M36.65 16c-7.735 0-13.984 6.283-13.984 14v79.983c0 7.717 6.249 14 13.983 14h40.017c2.21 0 4-1.791 4-4s-1.79-4-4-4H36.649c-3.299 0-5.983-2.683-5.983-6V30c0-3.316 2.684-6 5.983-6h32.417c1.55 0 3.033.595 4.124 1.646l.009.009 32.616 31.25h.001c1.182 1.133 1.85 2.69 1.85 4.328v24.584c0 2.209 1.791 4 4 4s4-1.791 4-4V61.233c0-3.828-1.565-7.47-4.317-10.105L78.742 19.887l-.005-.005C76.13 17.37 72.647 16 69.066 16zm74.584 128.15c13.577 0 24.584-11.006 24.584-24.583s-11.007-24.583-24.584-24.583-24.583 11.006-24.583 24.583 11.006 24.583 24.583 24.583m4.414-35.834c0-2.209-1.791-4-4-4s-4 1.791-4 4v7.667h-7.667c-2.209 0-4 1.791-4 4s1.791 4 4 4h7.667v7.667c0 2.209 1.791 4 4 4s4-1.791 4-4v-7.667h7.666c2.21 0 4-1.791 4-4s-1.79-4-4-4h-7.666zm-39.415-20.72c-6.809 0-12.75 5.214-13.946 12.227-.371 2.178-2.438 3.642-4.616 3.271-2.177-.372-3.642-2.438-3.27-4.616 1.803-10.574 10.844-18.883 21.832-18.883 3.045 0 5.956.645 8.604 1.805 2.024.886 2.946 3.245 2.06 5.268-.886 2.024-3.245 2.946-5.269 2.06-1.669-.73-3.49-1.133-5.395-1.133M55.786 84.12c2.009-.92 2.891-3.294 1.971-5.303s-3.293-2.89-5.302-1.97c-6.766 3.098-11.825 9.387-13.162 16.939-.385 2.175 1.066 4.25 3.242 4.636 2.175.385 4.25-1.066 4.636-3.242.864-4.884 4.163-9.021 8.615-11.06m19.668-24.526c-2.946 0-5.333 2.388-5.333 5.334s2.387 5.333 5.333 5.333c2.945 0 5.333-2.388 5.333-5.334s-2.388-5.333-5.333-5.333M62.12 64.93c0-7.364 5.969-13.334 13.333-13.334s13.333 5.97 13.333 13.334-5.97 13.333-13.333 13.333-13.333-5.97-13.333-13.333m-7.112-3.393c0-2.676 2.272-5.052 5.334-5.052 2.209 0 4-1.79 4-4s-1.791-4-4-4c-7.248 0-13.334 5.729-13.334 13.052 0 3.722 1.595 7.064 4.104 9.42 1.61 1.512 4.142 1.432 5.654-.178s1.433-4.142-.178-5.654c-.996-.935-1.58-2.205-1.58-3.588"
        })
    });
const $a6187d2a2ae66251$var$ForwardRef = /*#__PURE__*/ (0, $1PZ9a$react.forwardRef)($a6187d2a2ae66251$var$SvgComponent);
var $a6187d2a2ae66251$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a6187d2a2ae66251$var$ForwardRef);
