// @react-spectrum/s2/illustrations/linear/internal/S2_lin_reportAbuse_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $OivPV$reactjsxruntime = require("react/jsx-runtime");
var $OivPV$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $085fedd12b3505e7$export$2e2bcd8739ae039);




const $085fedd12b3505e7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $OivPV$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $OivPV$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M26.244 16.25c-5.523 0-10 4.477-10 10v34.744c0 5.523 4.477 10 10 10h9.933l7.187 8.746c2.4 2.921 6.87 2.921 9.271 0l7.187-8.746h9.933c5.523 0 10-4.477 10-10V26.25c0-5.523-4.477-10-10-10zm-6 10c0-3.314 2.686-6 6-6h43.51c3.315 0 6 2.686 6 6v34.744c0 3.314-2.685 6-6 6H58.878c-.598 0-1.165.268-1.545.73l-7.787 9.477c-.8.973-2.29.973-3.09 0l-7.788-9.477c-.38-.462-.947-.73-1.545-.73H26.244c-3.314 0-6-2.686-6-6zM48 24.886c1.105 0 2 .895 2 2v22.21c0 1.104-.895 2-2 2s-2-.896-2-2v-22.21c0-1.105.896-2 2-2m0 37.48c1.657 0 3-1.343 3-3s-1.343-3-3-3c-1.656 0-3 1.343-3 3s1.344 3 3 3"
        })
    });
const $085fedd12b3505e7$var$ForwardRef = /*#__PURE__*/ (0, $OivPV$react.forwardRef)($085fedd12b3505e7$var$SvgComponent);
var $085fedd12b3505e7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($085fedd12b3505e7$var$ForwardRef);
