// @react-spectrum/s2/illustrations/linear/internal/S2_lin_flag_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $g4cKx$reactjsxruntime = require("react/jsx-runtime");
var $g4cKx$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $46007ca4fd1efc29$export$2e2bcd8739ae039);




const $46007ca4fd1efc29$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $g4cKx$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $g4cKx$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M12.992 9.11c0-.691-.56-1.25-1.25-1.25s-1.25.559-1.25 1.25v29.78c0 .691.56 1.25 1.25 1.25s1.25-.559 1.25-1.25v-7.64h20.336c2.347 0 4.25-1.902 4.25-4.25V14.022c0-2.348-1.903-4.25-4.25-4.25H12.992zm0 3.162V28.75h20.336c.967 0 1.75-.783 1.75-1.75V14.022c0-.967-.783-1.75-1.75-1.75z"
        })
    });
const $46007ca4fd1efc29$var$ForwardRef = /*#__PURE__*/ (0, $g4cKx$react.forwardRef)($46007ca4fd1efc29$var$SvgComponent);
var $46007ca4fd1efc29$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($46007ca4fd1efc29$var$ForwardRef);
