// @react-spectrum/s2/illustrations/linear/internal/S2_lin_filmstrip_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dtcxQ$reactjsxruntime = require("react/jsx-runtime");
var $dtcxQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1f0b71afabf0aa40$export$2e2bcd8739ae039);




const $1f0b71afabf0aa40$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dtcxQ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dtcxQ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M124 38c6.627 0 12 5.373 12 12v60c0 6.627-5.373 12-12 12H36c-6.627 0-12-5.373-12-12V50c0-6.627 5.373-12 12-12zm20 12c0-11.046-8.954-20-20-20H36c-11.046 0-20 8.954-20 20v60c0 11.046 8.954 20 20 20h88c11.046 0 20-8.954 20-20zm-95 3c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm-15-2c-1.105 0-2 .895-2 2v5c0 1.105.895 2 2 2h7c1.105 0 2-.895 2-2v-5c0-1.105-.895-2-2-2zm15 51c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm-15-2c-1.105 0-2 .895-2 2v5c0 1.105.895 2 2 2h7c1.105 0 2-.895 2-2v-5c0-1.105-.895-2-2-2zm32-47c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm2 47c-1.105 0-2 .895-2 2v5c0 1.105.895 2 2 2h7c1.105 0 2-.895 2-2v-5c0-1.105-.895-2-2-2zm15-47c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm2 47c-1.105 0-2 .895-2 2v5c0 1.105.895 2 2 2h7c1.105 0 2-.895 2-2v-5c0-1.105-.895-2-2-2zm15-47c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm2 47c-1.105 0-2 .895-2 2v5c0 1.105.895 2 2 2h7c1.105 0 2-.895 2-2v-5c0-1.105-.895-2-2-2zm15-47c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm2 47c-1.105 0-2 .895-2 2v5c0 1.105.895 2 2 2h7c1.105 0 2-.895 2-2v-5c0-1.105-.895-2-2-2z"
        })
    });
const $1f0b71afabf0aa40$var$ForwardRef = /*#__PURE__*/ (0, $dtcxQ$react.forwardRef)($1f0b71afabf0aa40$var$SvgComponent);
var $1f0b71afabf0aa40$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1f0b71afabf0aa40$var$ForwardRef);
