// @react-spectrum/s2/illustrations/linear/internal/S2_lin_trash_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $hRRdC$jsxs, jsx as $hRRdC$jsx} from "react/jsx-runtime";
import {forwardRef as $hRRdC$forwardRef} from "react";





const $0814aeec5f348f59$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hRRdC$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hRRdC$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M43 36.387c0-1.105-.895-2-2-2s-2 .895-2 2V69.29c0 1.105.895 2 2 2s2-.895 2-2zM54.549 34.387c1.104 0 2 .895 2 2V69.29c0 1.105-.896 2-2 2s-2-.895-2-2V36.387c0-1.105.895-2 2-2"
            }),
            /*#__PURE__*/ (0, $hRRdC$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M34 25v-3.47c0-3.092 2.545-5.53 5.6-5.53h16.8c3.055 0 5.6 2.438 5.6 5.53V25h16c1.105 0 2 .895 2 2s-.895 2-2 2h-2.21l-4.782 43.034C70.506 76.57 66.672 80 62.109 80H34.892c-4.562 0-8.396-3.43-8.9-7.966L21.21 29H18c-1.105 0-2-.895-2-2s.895-2 2-2zm4-3.47c0-.808.679-1.53 1.6-1.53h16.8c.921 0 1.6.722 1.6 1.53V25H38zM25.235 29h46.53l-4.732 42.593c-.279 2.509-2.4 4.407-4.924 4.407H34.89c-2.524 0-4.645-1.898-4.924-4.407z"
            })
        ]
    });
const $0814aeec5f348f59$var$ForwardRef = /*#__PURE__*/ (0, $hRRdC$forwardRef)($0814aeec5f348f59$var$SvgComponent);
var $0814aeec5f348f59$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0814aeec5f348f59$var$ForwardRef);


export {$0814aeec5f348f59$export$2e2bcd8739ae039 as default};
