// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browserNotCompatible_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bsV8A$reactjsxruntime = require("react/jsx-runtime");
var $bsV8A$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $761d32b6a953cd22$export$2e2bcd8739ae039);




const $761d32b6a953cd22$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bsV8A$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bsV8A$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M37 10.25c1.519 0 2.75 1.231 2.75 2.75v4.77H8.25V13c0-1.519 1.231-2.75 2.75-2.75zM8.25 20.27h31.5V35c0 1.519-1.231 2.75-2.75 2.75H11c-1.519 0-2.75-1.231-2.75-2.75zm34-7.27c0-2.9-2.35-5.25-5.25-5.25H11c-2.9 0-5.25 2.35-5.25 5.25v22c0 2.9 2.35 5.25 5.25 5.25h26c2.9 0 5.25-2.35 5.25-5.25zM13 15.5c-.828 0-1.5-.672-1.5-1.5s.672-1.5 1.5-1.5 1.5.672 1.5 1.5-.672 1.5-1.5 1.5m5.616 8.116c.488-.488 1.28-.488 1.768 0L24 27.232l3.616-3.616c.488-.488 1.28-.488 1.768 0s.488 1.28 0 1.768L25.768 29l3.616 3.616c.488.488.488 1.28 0 1.768s-1.28.488-1.768 0L24 30.768l-3.616 3.616c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.768L22.232 29l-3.616-3.616c-.488-.488-.488-1.28 0-1.768"
        })
    });
const $761d32b6a953cd22$var$ForwardRef = /*#__PURE__*/ (0, $bsV8A$react.forwardRef)($761d32b6a953cd22$var$SvgComponent);
var $761d32b6a953cd22$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($761d32b6a953cd22$var$ForwardRef);
