// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noComment_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $gCQek$jsx} from "react/jsx-runtime";
import {forwardRef as $gCQek$forwardRef} from "react";





const $333fd25161b8ed44$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gCQek$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gCQek$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M27.909 24c-10.14 0-18.401 8.175-18.401 18.311V93.1c0 10.137 8.262 18.312 18.4 18.312h8.904c.001 6.22-.042 13.744-.086 19.747-.056 7.596 8.874 11.591 14.555 6.696l30.693-26.443h9.659c2.209 0 4-1.791 4-4s-1.791-4-4-4h-12.63l-1.125.969-31.819 27.414c-.166.143-.306.187-.421.201-.134.016-.296-.005-.457-.079s-.277-.179-.346-.282c-.057-.086-.11-.208-.11-.417.054-7.36.108-17.045.08-23.823l-.017-3.983H27.91c-5.768 0-10.401-4.64-10.401-10.312V42.311C17.508 36.64 22.14 32 27.908 32h91.199c5.768 0 10.401 4.64 10.401 10.311v21.564c0 2.21 1.791 4 4 4s4-1.79 4-4V42.311c0-10.136-8.262-18.311-18.401-18.311zm73.584 74.635c0-13.53 10.97-24.5 24.5-24.5s24.5 10.97 24.5 24.5-10.97 24.5-24.5 24.5-24.5-10.97-24.5-24.5m35.42 10.93c1.56-1.57 1.56-4.1 0-5.66l-5.26-5.26 5.26-5.26c1.56-1.57 1.56-4.1 0-5.66s-4.1-1.56-5.66 0l-5.26 5.26-5.26-5.26c-1.56-1.56-4.1-1.56-5.66 0-1.56 1.57-1.56 4.1 0 5.66l5.26 5.26-5.26 5.26c-1.56 1.57-1.56 4.1 0 5.66.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17l5.26-5.26 5.26 5.26c.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17"
        })
    });
const $333fd25161b8ed44$var$ForwardRef = /*#__PURE__*/ (0, $gCQek$forwardRef)($333fd25161b8ed44$var$SvgComponent);
var $333fd25161b8ed44$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($333fd25161b8ed44$var$ForwardRef);


export {$333fd25161b8ed44$export$2e2bcd8739ae039 as default};
