// @react-spectrum/s2/illustrations/linear/internal/S2_lin_folderShared_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $7yiO4$jsxs, jsx as $7yiO4$jsx} from "react/jsx-runtime";
import {forwardRef as $7yiO4$forwardRef} from "react";





const $a7a15385ea512023$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7yiO4$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7yiO4$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M7.25 11.8c0-.89.692-1.55 1.47-1.55h7.533c.432 0 .852.202 1.137.568l3.055 3.932H7.25zm-2.5 2.95V11.8c0-2.202 1.744-4.05 3.97-4.05h7.533c1.22 0 2.363.571 3.11 1.534l-.97.754.97-.754 4.249 5.466h14.234c2.956 0 5.404 2.351 5.404 5.309V34.94c0 2.958-2.448 5.309-5.404 5.309H10.154c-2.956 0-5.404-2.351-5.404-5.309V14.75m2.5 2.5v17.691c0 1.523 1.274 2.809 2.904 2.809h27.692c1.63 0 2.904-1.286 2.904-2.809V20.06c0-1.523-1.274-2.809-2.904-2.809H7.25"
            }),
            /*#__PURE__*/ (0, $7yiO4$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M25.515 26.118c.812 0 1.47-.659 1.47-1.47 0-.813-.658-1.471-1.47-1.471-.813 0-1.471.658-1.471 1.47s.658 1.47 1.47 1.47m0 5.882H22.84c-.36 0-.623-.298-.548-.612.32-1.324 1.65-2.33 3.223-2.33s2.903 1.006 3.222 2.33c.076.314-.187.612-.548.612z"
            }),
            /*#__PURE__*/ (0, $7yiO4$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "m22.292 31.388 1.216.293zm6.445 0-1.215.293zm-11.22-.98c-.115.681.343 1.327 1.024 1.442.68.116 1.326-.342 1.441-1.023zm3.572-1.076c.623-.297.887-1.044.59-1.666-.298-.623-1.045-.887-1.668-.59zm.6-6.082c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25zm-2.008 2.46c.496.48 1.287.468 1.767-.028s.468-1.287-.029-1.768zm6.054-1.063c0 .122-.099.22-.22.22v2.5c1.502 0 2.72-1.217 2.72-2.72zm-.22-.22c.121 0 .22.098.22.22h2.5c0-1.502-1.218-2.72-2.72-2.72zm-.221.22c0-.122.099-.22.22-.22v-2.5c-1.502 0-2.72 1.218-2.72 2.72zm.22.22c-.121 0-.22-.098-.22-.22h-2.5c0 1.503 1.218 2.72 2.72 2.72zM22.84 33.25h2.675v-2.5H22.84zm-1.763-2.155c-.302 1.255.76 2.155 1.763 2.155v-2.5c.168 0 .355.072.496.23.145.164.238.423.172.7zm4.438-3.286c-2.075 0-3.966 1.33-4.438 3.286l2.43.586c.167-.692.936-1.372 2.008-1.372zm4.437 3.286c-.471-1.956-2.363-3.286-4.437-3.286v2.5c1.07 0 1.84.68 2.007 1.372zm-1.763 2.155c1.003 0 2.065-.9 1.763-2.155l-2.43.586c-.067-.278.026-.537.171-.7.142-.16.328-.231.496-.231zm-2.674 0h2.674v-2.5h-2.674zm-5.533-2.423c.115-.675.55-1.229 1.107-1.495l-1.078-2.256c-1.297.62-2.244 1.863-2.493 3.333zm1.32-7.19c0-.214.173-.387.386-.387v-2.5c-1.594 0-2.886 1.292-2.886 2.887zm.117.277c-.074-.072-.117-.168-.117-.277h-2.5c0 .813.338 1.55.879 2.073z"
            })
        ]
    });
const $a7a15385ea512023$var$ForwardRef = /*#__PURE__*/ (0, $7yiO4$forwardRef)($a7a15385ea512023$var$SvgComponent);
var $a7a15385ea512023$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a7a15385ea512023$var$ForwardRef);


export {$a7a15385ea512023$export$2e2bcd8739ae039 as default};
