// @react-spectrum/s2/illustrations/linear/internal/S2_lin_magicWand_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $fket7$reactjsxruntime = require("react/jsx-runtime");
var $fket7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $acc26935ea1ad44b$export$2e2bcd8739ae039);




const $acc26935ea1ad44b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fket7$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fket7$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m126.204 45.43 7.055-1.41c2.774-.553 3.177-4.359.574-5.48l-6.606-2.847c-.87-.377-1.499-1.152-1.687-2.085l-1.41-7.056c-.553-2.774-4.359-3.177-5.481-.573l-2.846 6.606c-.377.87-1.152 1.499-2.086 1.687l-7.055 1.41c-2.774.552-3.177 4.358-.574 5.48l6.606 2.846c.87.378 1.499 1.153 1.687 2.086l1.41 7.055c.553 2.774 4.359 3.178 5.481.574l2.846-6.606c.378-.87 1.152-1.499 2.086-1.687M67.91 51.228l5.828-1.165c2.29-.457 2.624-3.6.473-4.527l-5.456-2.35c-.718-.312-1.238-.952-1.393-1.723l-1.165-5.828c-.456-2.29-3.6-2.624-4.526-.474l-2.351 5.457c-.312.718-.952 1.238-1.723 1.393l-5.827 1.165c-2.292.456-2.625 3.6-.474 4.526l5.456 2.351c.718.312 1.238.952 1.393 1.723l1.165 5.827c.456 2.292 3.6 2.624 4.527.474l2.35-5.456c.312-.718.952-1.238 1.723-1.393m63.378 40.865-5.828 1.165c-.771.155-1.411.675-1.722 1.393l-2.351 5.456c-.927 2.151-4.07 1.818-4.527-.473l-1.165-5.828c-.155-.77-.675-1.41-1.393-1.722l-5.456-2.351c-2.15-.927-1.818-4.07.474-4.527l5.827-1.165c.771-.155 1.411-.675 1.723-1.393l2.351-5.456c.926-2.15 4.07-1.818 4.526.474l1.165 5.827c.155.77.675 1.411 1.393 1.723l5.457 2.35c2.15.927 1.817 4.07-.474 4.527m-44.676-37.78c5.32-5.319 13.944-5.319 19.263 0 5.319 5.32 5.319 13.944 0 19.263l-61.06 61.061c-5.32 5.319-13.944 5.319-19.263 0-5.32-5.32-5.32-13.944 0-19.263zm13.606 5.657c-2.195-2.195-5.754-2.195-7.949 0l-9.154 9.154 7.95 7.95 9.153-9.155c2.195-2.195 2.195-5.754 0-7.949m-14.81 22.76-7.95-7.949-46.25 46.25c-2.194 2.195-2.194 5.754 0 7.949 2.196 2.195 5.755 2.195 7.95 0z"
        })
    });
const $acc26935ea1ad44b$var$ForwardRef = /*#__PURE__*/ (0, $fket7$react.forwardRef)($acc26935ea1ad44b$var$SvgComponent);
var $acc26935ea1ad44b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($acc26935ea1ad44b$var$ForwardRef);
