// @react-spectrum/s2/illustrations/linear/internal/S2_lin_crop_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $88Jaa$reactjsxruntime = require("react/jsx-runtime");
var $88Jaa$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $bef9d756d4503168$export$2e2bcd8739ae039);




const $bef9d756d4503168$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $88Jaa$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $88Jaa$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.237 7.5c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v6.202H7.5c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h6.237v12.846c0 2.9 2.35 5.25 5.25 5.25h12.775V40.5c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-6.202H40.5c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H18.987c-1.519 0-2.75-1.231-2.75-2.75V7.5M19 13.702c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h10.012c1.52 0 2.75 1.23 2.75 2.75v10.063c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V18.952c0-2.9-2.35-5.25-5.25-5.25z"
        })
    });
const $bef9d756d4503168$var$ForwardRef = /*#__PURE__*/ (0, $88Jaa$react.forwardRef)($bef9d756d4503168$var$SvgComponent);
var $bef9d756d4503168$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($bef9d756d4503168$var$ForwardRef);
