// @react-spectrum/s2/illustrations/linear/internal/S2_lin_conversationbubbles_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $5Kco3$reactjsxruntime = require("react/jsx-runtime");
var $5Kco3$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $98342f62cbd0a055$export$2e2bcd8739ae039);




const $98342f62cbd0a055$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5Kco3$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5Kco3$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M9.18 7.75c-2.422 0-4.43 1.93-4.43 4.364v11.051c0 2.435 2.008 4.364 4.43 4.364h1.6c-.002 1.27-.01 2.728-.02 3.915-.014 2.016 2.351 2.995 3.814 1.753l6.677-5.668h8.069c2.422 0 4.43-1.929 4.43-4.364v-11.05c0-2.436-2.008-4.365-4.43-4.365zm-1.93 4.364c0-1.005.84-1.864 1.93-1.864h20.14c1.09 0 1.93.86 1.93 1.864v11.051c0 1.005-.84 1.864-1.93 1.864h-8.528c-.297 0-.584.105-.81.297l-6.718 5.704c.01-1.534.02-3.408.014-4.756-.003-.688-.562-1.245-1.25-1.245H9.18c-1.09 0-1.93-.859-1.93-1.864zM36.5 18.25c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h2.68c.883 0 1.57.702 1.57 1.533v9.875c0 .831-.687 1.533-1.57 1.533h-2.525c-.688 0-1.247.556-1.25 1.245-.005 1.117.002 2.64.01 3.948l-5.719-4.893c-.226-.194-.515-.3-.813-.3h-7.562c-.667 0-1.23-.404-1.46-.964-.261-.639-.991-.945-1.63-.683s-.944.992-.683 1.63c.607 1.482 2.072 2.517 3.773 2.517h7.1l5.881 5.03c1.399 1.197 3.632.239 3.618-1.666-.007-1.019-.015-2.262-.016-3.364h1.276c2.232 0 4.07-1.79 4.07-4.033v-9.875c0-2.243-1.838-4.033-4.07-4.033z"
        })
    });
const $98342f62cbd0a055$var$ForwardRef = /*#__PURE__*/ (0, $5Kco3$react.forwardRef)($98342f62cbd0a055$var$SvgComponent);
var $98342f62cbd0a055$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($98342f62cbd0a055$var$ForwardRef);
