// @react-spectrum/s2/illustrations/linear/internal/S2_lin_reportAbuse_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $43ieA$reactjsxruntime = require("react/jsx-runtime");
var $43ieA$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $319f3c13bd3137d4$export$2e2bcd8739ae039);




const $319f3c13bd3137d4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $43ieA$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $43ieA$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M13.123 7.875c-2.9 0-5.25 2.35-5.25 5.25v17.372c0 2.9 2.35 5.25 5.25 5.25h4.848l3.519 4.282c1.3 1.582 3.722 1.582 5.022 0l3.518-4.282h4.848c2.9 0 5.25-2.35 5.25-5.25V13.125c0-2.9-2.35-5.25-5.25-5.25zm-2.75 5.25c0-1.519 1.231-2.75 2.75-2.75h21.755c1.52 0 2.75 1.231 2.75 2.75v17.372c0 1.519-1.23 2.75-2.75 2.75H29.44c-.374 0-.729.167-.966.456l-3.894 4.739c-.3.365-.859.365-1.159 0l-3.893-4.739c-.238-.289-.592-.456-.966-.456h-5.439c-1.519 0-2.75-1.231-2.75-2.75zm13.628-.933c.69 0 1.25.56 1.25 1.25v11.105c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25V13.442c0-.69.56-1.25 1.25-1.25m0 19.24c.966 0 1.75-.783 1.75-1.75 0-.966-.784-1.75-1.75-1.75-.967 0-1.75.784-1.75 1.75 0 .967.783 1.75 1.75 1.75"
        })
    });
const $319f3c13bd3137d4$var$ForwardRef = /*#__PURE__*/ (0, $43ieA$react.forwardRef)($319f3c13bd3137d4$var$SvgComponent);
var $319f3c13bd3137d4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($319f3c13bd3137d4$var$ForwardRef);
