// @react-spectrum/s2/illustrations/linear/internal/S2_lin_bolt_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $5huNn$reactjsxruntime = require("react/jsx-runtime");
var $5huNn$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4a7e99759de16160$export$2e2bcd8739ae039);




const $4a7e99759de16160$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5huNn$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5huNn$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            d: "M57.81 17.62c1.29 0 2.5 1.31 1.87 2.8L52.09 38.3c-.43 1.02.04 2.19 1.06 2.62l11.72 4.96c1.27.54 1.63 2.16.71 3.18L39.78 77.7c-.43.47-.95.68-1.46.68-1.29 0-2.5-1.3-1.87-2.79l7.36-17.46c.43-1.01-.04-2.18-1.05-2.61l-11.64-5.01c-1.25-.54-1.61-2.15-.7-3.17l25.93-29.03c.43-.48.95-.68 1.46-.68m0-4.01c-1.7 0-3.28.72-4.45 2.02L27.43 44.67c-1.28 1.44-1.8 3.41-1.38 5.29s1.72 3.46 3.49 4.22l9.82 4.23-6.59 15.63c-.79 1.86-.6 3.97.5 5.62 1.11 1.68 3.05 2.72 5.06 2.72 1.69 0 3.27-.71 4.43-2l25.8-28.64c1.3-1.44 1.82-3.43 1.4-5.32s-1.73-3.47-3.52-4.22l-9.88-4.18 6.81-16.03c.79-1.87.61-3.97-.49-5.63-1.11-1.68-3.05-2.73-5.06-2.73z"
        })
    });
const $4a7e99759de16160$var$ForwardRef = /*#__PURE__*/ (0, $5huNn$react.forwardRef)($4a7e99759de16160$var$SvgComponent);
var $4a7e99759de16160$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4a7e99759de16160$var$ForwardRef);
