// @react-spectrum/s2/illustrations/linear/internal/S2_lin_image_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $fp9Fj$jsx} from "react/jsx-runtime";
import {forwardRef as $fp9Fj$forwardRef} from "react";





const $f72a6b841a21927d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fp9Fj$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fp9Fj$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M136 50c0-6.627-5.373-12-12-12H36c-6.627 0-12 5.373-12 12v60c0 .862.09 1.703.264 2.513l24.781-38.286c4.365-6.75 14.252-6.75 18.618-.002l.001.002 20.529 31.71 11.745-17.491.001-.002c4.393-6.534 14.019-6.548 18.41 0l.004.007 17.059 25.27c.382-1.172.588-2.422.588-3.721zM55.762 78.573l-26.686 41.229C31.032 121.186 33.42 122 36 122h41.407l5.934-8.837-22.393-34.59-.002-.002c-1.213-1.876-3.97-1.876-5.183 0zm74.428 41.71c-1.808 1.09-3.926 1.717-6.19 1.717H87.044l19.534-29.092.001-.002c1.232-1.83 3.914-1.817 5.127-.005l.008.013zM79.61 130H124c11.046 0 20-8.954 20-20V50c0-11.046-8.954-20-20-20H36c-11.046 0-20 8.954-20 20v60c0 11.046 8.954 20 20 20h43.609M108 58c0 4.418 3.582 8 8 8s8-3.582 8-8-3.582-8-8-8-8 3.582-8 8"
        })
    });
const $f72a6b841a21927d$var$ForwardRef = /*#__PURE__*/ (0, $fp9Fj$forwardRef)($f72a6b841a21927d$var$SvgComponent);
var $f72a6b841a21927d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f72a6b841a21927d$var$ForwardRef);


export {$f72a6b841a21927d$export$2e2bcd8739ae039 as default};
