// @react-spectrum/s2/illustrations/linear/internal/S2_lin_brightnessContrast_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $jLynE$jsx} from "react/jsx-runtime";
import {forwardRef as $jLynE$forwardRef} from "react";





const $134b08e028d4f1b6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jLynE$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jLynE$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25.25 7.48c0-.69-.56-1.25-1.25-1.25-.691 0-1.25.56-1.25 1.25v2.341c0 .69.559 1.25 1.25 1.25.69 0 1.25-.56 1.25-1.25zM16.904 24c0-3.918 3.176-7.094 7.094-7.094v14.187c-3.918 0-7.094-3.176-7.094-7.093M24 14.406c-5.299 0-9.594 4.295-9.594 9.594S18.7 33.593 24 33.593s9.593-4.295 9.593-9.593c0-5.299-4.295-9.594-9.593-9.594m0 22.522c.69 0 1.25.56 1.25 1.25v2.34c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-2.34c0-.69.56-1.25 1.25-1.25M40.52 25.25c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.34c-.691 0-1.25.56-1.25 1.25s.559 1.25 1.25 1.25zM11.07 24c0 .69-.56 1.25-1.25 1.25h-2.34c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h2.34c.69 0 1.25.56 1.25 1.25m25.493-10.799c.488-.488.488-1.28 0-1.767-.488-.489-1.28-.489-1.768 0l-1.655 1.655c-.488.488-.488 1.28 0 1.767.489.489 1.28.489 1.768 0zm-21.707 19.94c.488.488.488 1.279 0 1.767l-1.655 1.655c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.768l1.655-1.655c.489-.488 1.28-.488 1.768 0M11.434 13.2c-.489-.488-.489-1.28 0-1.767.488-.489 1.28-.489 1.767 0l1.655 1.655c.488.488.488 1.28 0 1.767-.488.489-1.28.489-1.767 0zm21.706 19.94c-.488.488-.488 1.279 0 1.767l1.655 1.655c.489.488 1.28.488 1.768 0s.488-1.28 0-1.768l-1.655-1.655c-.488-.488-1.28-.488-1.768 0"
        })
    });
const $134b08e028d4f1b6$var$ForwardRef = /*#__PURE__*/ (0, $jLynE$forwardRef)($134b08e028d4f1b6$var$SvgComponent);
var $134b08e028d4f1b6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($134b08e028d4f1b6$var$ForwardRef);


export {$134b08e028d4f1b6$export$2e2bcd8739ae039 as default};
