// @react-spectrum/s2/illustrations/linear/internal/S2_lin_volume_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $66uPy$reactjsxruntime = require("react/jsx-runtime");
var $66uPy$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b84f0665cdaa9862$export$2e2bcd8739ae039);




const $b84f0665cdaa9862$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $66uPy$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $66uPy$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M86.427 37.282c0-8.889-10.729-13.36-17.04-7.102l-18.98 18.818H36.313c-6.628 0-12 5.372-12 12v38.204c0 6.628 5.372 12 12 12h14.106l19.003 18.653c6.326 6.209 17.005 1.727 17.005-7.137zm-11.408-1.42c1.262-1.252 3.408-.358 3.408 1.42v85.436c0 1.773-2.136 2.669-3.401 1.427l-18.063-17.729V53.763zM48.963 56.998v46.204h-12.65c-2.21 0-4-1.79-4-4V60.998c0-2.21 1.79-4 4-4zm50.383-.557c-1.765-1.329-4.273-.975-5.601.79s-.974 4.273.79 5.602c5.183 3.9 8.522 10.09 8.522 17.06 0 7.028-3.394 13.262-8.648 17.155-1.775 1.316-2.148 3.821-.833 5.596s3.82 2.147 5.595.832c7.204-5.338 11.886-13.914 11.886-23.583 0-9.59-4.607-18.106-11.71-23.452m16.935-15.454c-1.765-1.328-4.273-.974-5.601.791s-.974 4.273.791 5.602c9.856 7.416 16.217 19.2 16.217 32.47 0 13.378-6.466 25.246-16.459 32.651-1.775 1.315-2.148 3.82-.833 5.595s3.821 2.148 5.596.832c11.942-8.849 19.696-23.058 19.696-39.078 0-15.89-7.629-30-19.407-38.862"
        })
    });
const $b84f0665cdaa9862$var$ForwardRef = /*#__PURE__*/ (0, $66uPy$react.forwardRef)($b84f0665cdaa9862$var$SvgComponent);
var $b84f0665cdaa9862$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b84f0665cdaa9862$var$ForwardRef);
