// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pieChart_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $aEbwC$jsx} from "react/jsx-runtime";
import {forwardRef as $aEbwC$forwardRef} from "react";





const $7b3568ec6da8f913$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aEbwC$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $aEbwC$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22.762 8.298c-3.656.284-6.961 1.816-9.49 4.171l9.49 8.7zm-11.18 6.014C9.493 16.984 8.25 20.347 8.25 24c0 3.905 1.42 7.476 3.773 10.229L22.2 24.045zm-.313 22.744q-.08-.058-.151-.129C7.803 33.624 5.75 29.051 5.75 24 5.75 13.92 13.92 5.75 24 5.75h.012c3 0 5.953.74 8.598 2.155s4.9 3.46 6.567 5.956 2.691 5.364 2.985 8.35c.294 2.987-.152 6-1.3 8.773s-2.961 5.22-5.28 7.123c-2.318 1.904-5.07 3.207-8.012 3.792-2.941.586-5.982.436-8.852-.435-2.802-.85-5.357-2.363-7.45-4.408M25.262 8.3V24c0 .331-.131.65-.366.884L13.802 35.986c1.645 1.403 3.565 2.455 5.642 3.086 2.477.752 5.1.88 7.638.375s4.913-1.629 6.914-3.272 3.565-3.754 4.556-6.147c.99-2.393 1.376-4.994 1.122-7.572s-1.139-5.053-2.577-7.207-3.384-3.919-5.666-5.14c-1.914-1.023-4.015-1.637-6.169-1.81"
        })
    });
const $7b3568ec6da8f913$var$ForwardRef = /*#__PURE__*/ (0, $aEbwC$forwardRef)($7b3568ec6da8f913$var$SvgComponent);
var $7b3568ec6da8f913$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7b3568ec6da8f913$var$ForwardRef);


export {$7b3568ec6da8f913$export$2e2bcd8739ae039 as default};
