// @react-spectrum/s2/illustrations/linear/internal/S2_lin_stepForward_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $4SppW$jsx} from "react/jsx-runtime";
import {forwardRef as $4SppW$forwardRef} from "react";





const $c3e002be668a3d96$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4SppW$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4SppW$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M5.75 13c0-1.795 1.455-3.25 3.25-3.25h3.5c1.795 0 3.25 1.455 3.25 3.25v22c0 1.795-1.455 3.25-3.25 3.25H9c-1.795 0-3.25-1.455-3.25-3.25zM9 12.25c-.414 0-.75.336-.75.75v22c0 .414.336.75.75.75h3.5c.414 0 .75-.336.75-.75V13c0-.414-.336-.75-.75-.75zm10.322 1.989c.004-3.087 3.298-5.153 6.023-3.526l16.372 9.778c2.647 1.58 2.643 5.49-.008 7.065l-16.395 9.736c-2.729 1.62-6.018-.453-6.014-3.54zm4.74-1.38c-.942-.562-2.238.098-2.24 1.383L21.8 33.755c-.002 1.285 1.293 1.948 2.237 1.388l16.396-9.737c1.021-.607 1.023-2.16.003-2.77zl.636-1.066z"
        })
    });
const $c3e002be668a3d96$var$ForwardRef = /*#__PURE__*/ (0, $4SppW$forwardRef)($c3e002be668a3d96$var$SvgComponent);
var $c3e002be668a3d96$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c3e002be668a3d96$var$ForwardRef);


export {$c3e002be668a3d96$export$2e2bcd8739ae039 as default};
