// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lightbulbRays_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $jdDne$jsx} from "react/jsx-runtime";
import {forwardRef as $jdDne$forwardRef} from "react";





const $2d32edaa22c078e3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jdDne$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jdDne$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.905 13.096c1.105 0 2 .895 2 2v3.477c0 1.105-.895 2-2 2s-2-.895-2-2v-3.477c0-1.105.896-2 2-2m1.864 19.39c-1.104 0-2 .896-2 2s.896 2 2 2c3.448 0 6.03 2.603 6.03 5.556 0 1.105.895 2 2 2s2-.895 2-2c0-5.387-4.604-9.555-10.03-9.555m-7.398 38.948v3.776c0 .171.14.31.311.31h10.397c.172 0 .311-.139.311-.31v-3.776zm11.019-4v-3.642c0-3.026 1.783-5.499 3.816-7.089 3.648-2.853 6.004-7.34 6.004-12.397 0-8.656-6.894-15.617-15.33-15.617-8.434 0-15.328 6.96-15.328 15.618 0 5.056 2.355 9.543 6.004 12.396 2.033 1.59 3.816 4.063 3.816 7.09v3.64zm4 1.943v-5.585c0-1.385.839-2.81 2.28-3.938 4.588-3.588 7.54-9.223 7.54-15.547 0-10.804-8.623-19.618-19.33-19.618-10.705 0-19.328 8.814-19.328 19.618 0 6.324 2.952 11.959 7.54 15.547 1.44 1.127 2.28 2.553 2.28 3.938V75.21c0 2.38 1.93 4.31 4.31 4.31h10.397c2.381 0 4.311-1.93 4.311-4.31v-5.833M17 44.007c0-1.105.895-2 2-2h3.472c1.104 0 2 .895 2 2s-.896 2-2 2H19c-1.105 0-2-.896-2-2m56.338-2c-1.105 0-2 .895-2 2s.895 2 2 2h3.471c1.105 0 2-.896 2-2s-.895-2-2-2zm-3.576-17.029c.78-.78.78-2.047 0-2.828-.781-.781-2.048-.781-2.829 0l-2.46 2.46c-.781.781-.781 2.048 0 2.829.78.78 2.047.78 2.828 0zM26.049 22.15c.78-.781 2.047-.781 2.828 0l2.46 2.46c.782.782.782 2.048 0 2.829-.78.78-2.047.78-2.828 0l-2.46-2.46c-.781-.782-.781-2.048 0-2.83"
        })
    });
const $2d32edaa22c078e3$var$ForwardRef = /*#__PURE__*/ (0, $jdDne$forwardRef)($2d32edaa22c078e3$var$SvgComponent);
var $2d32edaa22c078e3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2d32edaa22c078e3$var$ForwardRef);


export {$2d32edaa22c078e3$export$2e2bcd8739ae039 as default};
