// @react-spectrum/s2/illustrations/linear/internal/S2_lin_gift_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $aOqI5$reactjsxruntime = require("react/jsx-runtime");
var $aOqI5$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9f619a473b0b4c72$export$2e2bcd8739ae039);




const $9f619a473b0b4c72$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aOqI5$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $aOqI5$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M14.378 12.374c0-3.003 2.434-5.437 5.436-5.437 1.684 0 3.19.766 4.186 1.968.998-1.202 2.503-1.967 4.186-1.967 3.003 0 5.437 2.433 5.437 5.436 0 1.082-.316 2.09-.861 2.936h3.205c2.343 0 4.242 1.9 4.242 4.242v1.995c0 1.517-.797 2.848-1.994 3.598v9.366c0 2.894-2.346 5.24-5.24 5.24h-17.95c-2.894 0-5.24-2.346-5.24-5.24v-9.366c-1.197-.75-1.994-2.08-1.994-3.598v-1.995c0-2.342 1.9-4.241 4.242-4.241h3.206c-.545-.847-.86-1.855-.86-2.937m8.372-.027v2.963h-2.936c-1.621 0-2.936-1.315-2.936-2.936 0-1.622 1.315-2.937 2.936-2.937 1.613 0 2.922 1.3 2.936 2.91m2.5 2.963h2.936c1.622 0 2.937-1.315 2.937-2.936 0-1.622-1.315-2.937-2.937-2.937-1.612 0-2.921 1.3-2.936 2.91v2.963m-13.217 7.979H22.75V17.81H12.033c-.962 0-1.742.78-1.742 1.741v1.995c0 .962.78 1.742 1.742 1.742M25.25 17.81v5.478h10.717c.962 0 1.742-.78 1.742-1.742v-1.995c0-.962-.78-1.741-1.742-1.741zm-2.5 7.978H12.286v8.722c0 1.513 1.226 2.74 2.739 2.74h7.725zm2.5 11.462V25.789h10.465v8.722c0 1.513-1.227 2.74-2.74 2.74z"
        })
    });
const $9f619a473b0b4c72$var$ForwardRef = /*#__PURE__*/ (0, $aOqI5$react.forwardRef)($9f619a473b0b4c72$var$SvgComponent);
var $9f619a473b0b4c72$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9f619a473b0b4c72$var$ForwardRef);
