// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileAlert_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $dmlZT$jsx} from "react/jsx-runtime";
import {forwardRef as $dmlZT$forwardRef} from "react";





const $f9a7132594dce31d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dmlZT$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dmlZT$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28.7 36.108c0-8.903 7.396-15.999 16.13-15.999h31.33c4.107 0 8.052 1.593 11.016 4.45l32.806 31.587c3.14 3.02 4.908 7.203 4.908 11.553V85c0 2.21-1.791 4-4 4s-4-1.79-4-4V68.093q-.178.016-.36.016H85.91c-5.509 0-10.03-4.44-10.03-9.98V28.11H44.83c-4.446 0-8.13 3.644-8.13 8v.001l-.02 79.999v.001c0 4.356 3.684 7.999 8.13 7.999h32.575c2.209 0 4 1.791 4 4s-1.791 4-4 4H44.81c-8.735 0-16.13-7.096-16.13-16v-.001l.02-79.999m55.18-3.617 28.684 27.618H85.91c-1.152 0-2.03-.92-2.03-1.98zM87.5 119.5c0-13.53 10.97-24.5 24.5-24.5s24.5 10.97 24.5 24.5-10.97 24.5-24.5 24.5-24.5-10.97-24.5-24.5m24.5-15.58c-2.21 0-4 1.79-4 4v12.78c0 2.21 1.79 4 4 4s4-1.79 4-4v-12.78c0-2.21-1.79-4-4-4m3.83 28.76c.11-.25.19-.52.24-.78.05-.27.08-.54.08-.81 0-1.1-.44-2.16-1.21-2.94-.97-.96-2.4-1.4-3.75-1.13-.26.05-.53.13-.78.24-.25.1-.49.23-.71.38q-.345.225-.63.51c-.78.78-1.22 1.85-1.22 2.94 0 .27.03.54.08.81.05.26.13.53.24.78.1.25.23.49.38.71.15.23.32.44.52.63.19.2.4.37.63.52.22.15.46.28.71.38.25.11.52.19.78.24.27.05.54.08.81.08 1.09 0 2.16-.44 2.94-1.22q.285-.285.51-.63c.15-.22.28-.46.38-.71"
        })
    });
const $f9a7132594dce31d$var$ForwardRef = /*#__PURE__*/ (0, $dmlZT$forwardRef)($f9a7132594dce31d$var$SvgComponent);
var $f9a7132594dce31d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f9a7132594dce31d$var$ForwardRef);


export {$f9a7132594dce31d$export$2e2bcd8739ae039 as default};
