// @react-spectrum/s2/illustrations/linear/internal/S2_lin_helpCircle_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $f4Uwf$reactjsxruntime = require("react/jsx-runtime");
var $f4Uwf$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $091c162a8e001e42$export$2e2bcd8739ae039);




const $091c162a8e001e42$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $f4Uwf$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $f4Uwf$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28 80c0-28.719 23.281-52 52-52s52 23.281 52 52-23.281 52-52 52-52-23.281-52-52m52-60c-33.137 0-60 26.863-60 60s26.863 60 60 60 60-26.863 60-60-26.863-60-60-60m1.436 37.042c-1.378-.28-2.809-.137-4.113.416-1.306.555-2.438 1.5-3.242 2.732s-1.24 2.693-1.24 4.194c0 2.21-1.79 4-4 4-2.208 0-4-1.79-4-4 0-3.04.88-6.02 2.539-8.564s4.028-4.542 6.817-5.726 5.867-1.496 8.837-.89c2.968.605 5.681 2.095 7.801 4.267s3.552 4.926 4.132 7.911.283 6.079-.856 8.896-3.075 5.241-5.577 6.954c-1.223.838-2.414 1.974-3.266 3.207-.865 1.25-1.22 2.353-1.22 3.203v7.083c0 2.21-1.79 4-4 4s-4-1.79-4-4v-7.083c0-2.975 1.181-5.646 2.64-7.755 1.472-2.127 3.395-3.933 5.327-5.256 1.184-.811 2.122-1.974 2.679-3.352.557-1.38.705-2.902.42-4.372-.286-1.47-.988-2.807-2.004-3.848-1.014-1.04-2.294-1.735-3.674-2.017m3.064 48.11c0 2.486-2.015 4.5-4.5 4.5s-4.5-2.014-4.5-4.5c0-2.485 2.015-4.5 4.5-4.5s4.5 2.015 4.5 4.5"
        })
    });
const $091c162a8e001e42$var$ForwardRef = /*#__PURE__*/ (0, $f4Uwf$react.forwardRef)($091c162a8e001e42$var$SvgComponent);
var $091c162a8e001e42$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($091c162a8e001e42$var$ForwardRef);
