// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileZip_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $aP0fy$reactjsxruntime = require("react/jsx-runtime");
var $aP0fy$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4752c1be99ad2d77$export$2e2bcd8739ae039);




const $4752c1be99ad2d77$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aP0fy$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $aP0fy$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.56 23.74c-8.886 0-16.07 7.238-16.07 16.13v79.75c0 8.89 7.172 16.13 16.07 16.13h64.87c8.887 0 16.07-7.238 16.07-16.13V71.41c0-4.4-1.788-8.63-4.989-11.674l-.006-.006L90.56 28.214l-.002-.002c-2.877-2.76-6.666-4.33-10.623-4.445q-.23-.027-.465-.027zm0 8h31.508q.198.02.402.02c2.075 0 4.06.796 5.55 2.226l.005.005 32.96 31.53.01.01c1.595 1.514 2.505 3.641 2.505 5.88v48.209c0 4.508-3.636 8.13-8.07 8.13H47.56c-4.441 0-8.07-3.62-8.07-8.13V39.87c0-4.508 3.637-8.13 8.07-8.13m23.92 11.88c-2.209 0-4 1.791-4 4s1.791 4 4 4h3.98v6.66h-3.98c-2.209 0-4 1.791-4 4s1.791 4 4 4h3.98v6.65h-3.98c-2.209 0-4 1.791-4 4s1.791 4 4 4h3.98v6.66h-2.98c-2.209 0-4 1.791-4 4v20.09c0 4.419 3.581 8 8 8h6.01c4.42 0 8-3.581 8-8V91.59c0-2.209-1.79-4-4-4h-3.03V73.6h5.08c2.209 0 4-1.79 4-4s-1.791-4-4-4h-5.08v-6.65h5.08c2.209 0 4-1.79 4-4s-1.791-4-4-4h-5.08v-3.33c0-2.209-1.79-4-4-4zm5 60.02v8.04h6.01v-8.04z"
        })
    });
const $4752c1be99ad2d77$var$ForwardRef = /*#__PURE__*/ (0, $aP0fy$react.forwardRef)($4752c1be99ad2d77$var$SvgComponent);
var $4752c1be99ad2d77$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4752c1be99ad2d77$var$ForwardRef);
