// @react-spectrum/s2/illustrations/linear/internal/S2_lin_flag_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $kCKZq$jsx} from "react/jsx-runtime";
import {forwardRef as $kCKZq$forwardRef} from "react";





const $3c0a96293c8c17c5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kCKZq$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kCKZq$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25.484 18.71c0-1.104-.895-2-2-2s-2 .896-2 2v59.563c0 1.105.896 2 2 2s2-.895 2-2v-15.78h41.172c4.418 0 8-3.582 8-8V28.535c0-4.418-3.582-8-8-8H25.484zm0 5.825v33.958h41.172c2.21 0 4-1.79 4-4V28.535c0-2.209-1.79-4-4-4z"
        })
    });
const $3c0a96293c8c17c5$var$ForwardRef = /*#__PURE__*/ (0, $kCKZq$forwardRef)($3c0a96293c8c17c5$var$SvgComponent);
var $3c0a96293c8c17c5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($3c0a96293c8c17c5$var$ForwardRef);


export {$3c0a96293c8c17c5$export$2e2bcd8739ae039 as default};
