// @react-spectrum/s2/illustrations/linear/internal/S2_lin_location_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $h7m47$reactjsxruntime = require("react/jsx-runtime");
var $h7m47$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $30db4ee53fda1211$export$2e2bcd8739ae039);




const $30db4ee53fda1211$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $h7m47$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $h7m47$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M128 67.952c0-27.62-23.141-49.85-51.044-48.117-23.091 1.435-42.074 19.581-44.65 42.65-.818 7.323.037 14.357 2.185 20.794 6.776 21.948 26.159 42.901 36.156 52.683 5.17 5.058 13.387 5.024 18.538-.017 9.378-9.178 26.846-28.029 34.622-48.292 2.686-6.017 4.193-12.683 4.193-19.7M77.452 27.82C100.683 26.377 120 44.877 120 67.952c0 5.897-1.271 11.485-3.537 16.528q-.048.104-.088.212c-7.092 18.565-23.488 36.436-32.785 45.535-2.06 2.016-5.306 2.014-7.348.017-9.978-9.763-28.007-29.511-34.12-49.365l-.029-.094c-1.803-5.383-2.524-11.264-1.837-17.413 2.148-19.23 18.008-34.36 37.196-35.552M66 64.74c0-7.731 6.268-14 14-14s14 6.268 14 14-6.268 14-14 14-14-6.267-14-14m14-22c-12.15 0-22 9.85-22 22s9.85 22 22 22 22-9.85 22-22-9.85-22-22-22"
        })
    });
const $30db4ee53fda1211$var$ForwardRef = /*#__PURE__*/ (0, $h7m47$react.forwardRef)($30db4ee53fda1211$var$SvgComponent);
var $30db4ee53fda1211$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($30db4ee53fda1211$var$ForwardRef);
