// @react-spectrum/s2/illustrations/linear/internal/S2_lin_transform_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $8qRPD$reactjsxruntime = require("react/jsx-runtime");
var $8qRPD$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $02e3ec4afe5b8061$export$2e2bcd8739ae039);




const $02e3ec4afe5b8061$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8qRPD$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8qRPD$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8 9.25C8 8.56 8.56 8 9.25 8h6.962c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-4.264L23.77 22.32l11.82-11.82h-3.82c-.69 0-1.25-.56-1.25-1.25S31.08 8 31.77 8h6.962c.69 0 1.25.56 1.25 1.25v6.962c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-4.068L25.537 24.089l11.945 11.945v-3.909c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v6.962c0 .69-.56 1.25-1.25 1.25H31.77c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h3.98L23.77 25.857l-11.981 11.98h4.424c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H9.251c-.69 0-1.25-.56-1.25-1.25v-6.962c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v3.464L22 24.09 10.5 12.587v3.625c0 .69-.56 1.25-1.25 1.25S8 16.902 8 16.212z"
        })
    });
const $02e3ec4afe5b8061$var$ForwardRef = /*#__PURE__*/ (0, $8qRPD$react.forwardRef)($02e3ec4afe5b8061$var$SvgComponent);
var $02e3ec4afe5b8061$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($02e3ec4afe5b8061$var$ForwardRef);
