// @react-spectrum/s2/illustrations/linear/internal/S2_lin_brush_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $23g6j$jsx} from "react/jsx-runtime";
import {forwardRef as $23g6j$forwardRef} from "react";





const $9803a37cf55f312b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $23g6j$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $23g6j$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m78.656 76.415 44.678-44.678c3.358-3.358 8.803-3.358 12.161 0 3.359 3.359 3.359 8.804 0 12.162L89.787 89.607c-1.133-3.097-2.732-5.876-4.854-8.253-1.798-2.014-3.9-3.658-6.277-4.939m-8.46-2.853 47.481-47.481c6.482-6.483 16.993-6.483 23.475 0s6.483 16.993 0 23.475L91.78 98.928q.194 2.187.193 4.483c0 9.321-3.484 17.196-9.582 23.048-6.037 5.794-14.422 9.414-24.028 10.789-15.196 2.175-36.284-.273-43.48-1.231-4.708-.627-5.773-6.544-2.279-9.085 4.349-3.162 13.512-10.722 17.371-20.094 2.78-6.75 6.23-15.086 11.249-21.7C46.328 78.407 53.414 73 63.425 73c2.381 0 4.641.186 6.77.56m-32.824 36.322c2.85-6.92 5.94-14.264 10.225-19.912 4.197-5.533 9.174-8.97 15.828-8.97 7.525 0 12.441 2.21 15.54 5.68 3.17 3.55 5.008 9.052 5.008 16.729 0 7.179-2.622 12.958-7.121 17.276-4.561 4.377-11.264 7.445-19.622 8.641-10.692 1.531-24.96.557-34.324-.418 4.99-4.404 11.17-11.019 14.466-19.026"
        })
    });
const $9803a37cf55f312b$var$ForwardRef = /*#__PURE__*/ (0, $23g6j$forwardRef)($9803a37cf55f312b$var$SvgComponent);
var $9803a37cf55f312b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9803a37cf55f312b$var$ForwardRef);


export {$9803a37cf55f312b$export$2e2bcd8739ae039 as default};
