// @react-spectrum/s2/illustrations/linear/internal/S2_lin_channel_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3KtBk$reactjsxruntime = require("react/jsx-runtime");
var $3KtBk$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $670dbc8806e1cac8$export$2e2bcd8739ae039);




const $670dbc8806e1cac8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3KtBk$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3KtBk$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M14.31 11.547c0 1.38 1.119 2.5 2.5 2.5l2.626 4.548c-1.233 1.043-2.105 2.499-2.4 4.155h-5.251c-.433-.747-1.24-1.25-2.166-1.25-1.38 0-2.5 1.12-2.5 2.5s1.12 2.5 2.5 2.5c.925 0 1.733-.503 2.166-1.25h5.25c.295 1.656 1.167 3.113 2.4 4.156l-2.625 4.547c-1.381 0-2.5 1.12-2.5 2.5s1.119 2.5 2.5 2.5c1.38 0 2.5-1.12 2.5-2.5 0-.455-.122-.883-.335-1.25l2.624-4.546c.75.27 1.558.418 2.4.418.843 0 1.651-.148 2.4-.418l2.626 4.547c-.213.367-.335.794-.335 1.25 0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5-1.12-2.5-2.5-2.5l-2.626-4.549c1.233-1.042 2.104-2.499 2.4-4.155h5.25c.433.747 1.24 1.25 2.166 1.25 1.38 0 2.5-1.12 2.5-2.5s-1.12-2.5-2.5-2.5c-.925 0-1.733.503-2.166 1.25h-5.251c-.295-1.656-1.167-3.112-2.4-4.154l2.627-4.55c1.38 0 2.5-1.118 2.5-2.5 0-1.38-1.12-2.5-2.5-2.5s-2.5 1.12-2.5 2.5c0 .456.122.883.334 1.25L26.4 17.345c-.75-.27-1.557-.417-2.4-.417-.842 0-1.65.147-2.4.417l-2.624-4.547c.213-.367.335-.795.335-1.25 0-1.38-1.12-2.5-2.5-2.5-1.381 0-2.5 1.12-2.5 2.5m9.69 7.88c-2.527 0-4.575 2.048-4.575 4.574s2.048 4.574 4.574 4.574 4.574-2.048 4.574-4.574-2.048-4.574-4.574-4.574"
        })
    });
const $670dbc8806e1cac8$var$ForwardRef = /*#__PURE__*/ (0, $3KtBk$react.forwardRef)($670dbc8806e1cac8$var$SvgComponent);
var $670dbc8806e1cac8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($670dbc8806e1cac8$var$ForwardRef);
