// @react-spectrum/s2/illustrations/linear/internal/S2_lin_playTriangle_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3ZsEi$reactjsxruntime = require("react/jsx-runtime");
var $3ZsEi$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1534d2ac6b140bdc$export$2e2bcd8739ae039);




const $1534d2ac6b140bdc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3ZsEi$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3ZsEi$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22.05 26.618c.007-6.23 6.637-10.337 12.08-7.09l35.885 21.417c5.32 3.175 5.312 11.046-.017 14.208L34.063 76.481l-1.02-1.72 1.02 1.72c-5.45 3.235-12.07-.888-12.063-7.119zm10.03-3.654c-2.596-1.55-6.027.316-6.03 3.659L26 69.367c-.004 3.344 3.423 5.217 6.022 3.674l35.935-21.327c2.72-1.615 2.725-5.712.008-7.334zl.9-1.51z"
        })
    });
const $1534d2ac6b140bdc$var$ForwardRef = /*#__PURE__*/ (0, $3ZsEi$react.forwardRef)($1534d2ac6b140bdc$var$SvgComponent);
var $1534d2ac6b140bdc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1534d2ac6b140bdc$var$ForwardRef);
