// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pause_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7gX2R$jsx} from "react/jsx-runtime";
import {forwardRef as $7gX2R$forwardRef} from "react";





const $5e71098a2a3d0290$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7gX2R$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7gX2R$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M41 46c0-6.627 5.373-12 12-12h11c6.627 0 12 5.373 12 12v68c0 6.627-5.373 12-12 12H53c-6.627 0-12-5.373-12-12zm12-4c-2.21 0-4 1.79-4 4v68c0 2.209 1.79 4 4 4h11c2.21 0 4-1.791 4-4V46c0-2.21-1.79-4-4-4zm32 4c0-6.627 5.373-12 12-12h11c6.627 0 12 5.373 12 12v68c0 6.627-5.373 12-12 12H97c-6.627 0-12-5.373-12-12zm12-4c-2.21 0-4 1.79-4 4v68c0 2.209 1.79 4 4 4h11c2.209 0 4-1.791 4-4V46c0-2.21-1.791-4-4-4z"
        })
    });
const $5e71098a2a3d0290$var$ForwardRef = /*#__PURE__*/ (0, $7gX2R$forwardRef)($5e71098a2a3d0290$var$SvgComponent);
var $5e71098a2a3d0290$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5e71098a2a3d0290$var$ForwardRef);


export {$5e71098a2a3d0290$export$2e2bcd8739ae039 as default};
