// @react-spectrum/s2/illustrations/linear/internal/S2_lin_market_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $ipLyg$reactjsxruntime = require("react/jsx-runtime");
var $ipLyg$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3fab44f61b7d5c03$export$2e2bcd8739ae039);




const $3fab44f61b7d5c03$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ipLyg$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $ipLyg$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M26.198 16.684c-2.814 0-5.376 1.626-6.565 4.178l-5.09 10.943v.001C13.523 33.994 13 36.376 13 38.783c0 4.518 2.935 8.345 7 9.703v22.007c0 5.523 4.477 10 10 10h36c5.523 0 10-4.477 10-10V48.52c4.12-1.327 7.105-5.18 7.105-9.738 0-2.348-.5-4.67-1.466-6.809q-.03-.085-.069-.168l-5.09-10.943v-.001c-1.188-2.552-3.75-4.178-6.565-4.178zM72 48.98c-3.04-.246-5.707-1.813-7.421-4.127-1.87 2.525-4.877 4.161-8.263 4.161s-6.393-1.636-8.263-4.161c-1.451 1.96-3.587 3.383-6.053 3.923v15.717h30zm-34-.12c-2.643-.464-4.94-1.937-6.474-4.007-1.733 2.34-4.443 3.917-7.526 4.135v21.505c0 3.314 2.686 6 6 6h8.056Q38 76.265 38 76.02zm1.79-3.846c-3.467 0-6.264-2.797-6.264-6.23v-.407c0-1.105-.895-2-2-2s-2 .895-2 2v.406c0 3.434-2.796 6.231-6.263 6.231S17 42.217 17 38.784c0-1.828.398-3.634 1.17-5.29v-.001l5.09-10.942v-.001c.529-1.136 1.673-1.866 2.938-1.866h43.717c1.265 0 2.41.73 2.939 1.867l5.017 10.786q.028.078.064.155c.77 1.659 1.17 3.466 1.17 5.291 0 3.434-2.797 6.231-6.263 6.231s-6.263-2.797-6.263-6.23v-.407c0-1.105-.895-2-2-2s-2 .895-2 2v.406c0 3.434-2.797 6.231-6.263 6.231-3.467 0-6.263-2.797-6.263-6.23v-.407c0-1.105-.896-2-2-2s-2 .895-2 2v.406c0 3.434-2.797 6.231-6.264 6.231M42 68.494h30v1.999c0 3.314-2.686 6-6 6H41.944q.056-.228.056-.472z"
        })
    });
const $3fab44f61b7d5c03$var$ForwardRef = /*#__PURE__*/ (0, $ipLyg$react.forwardRef)($3fab44f61b7d5c03$var$SvgComponent);
var $3fab44f61b7d5c03$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3fab44f61b7d5c03$var$ForwardRef);
