// @react-spectrum/s2/illustrations/linear/internal/S2_lin_helpCircle_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $l3uqX$reactjsxruntime = require("react/jsx-runtime");
var $l3uqX$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3c9ac1ca1f4534fc$export$2e2bcd8739ae039);




const $3c9ac1ca1f4534fc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $l3uqX$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $l3uqX$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 17.488c-17.12 0-31 13.88-31 31 0 17.121 13.88 31 31 31s31-13.879 31-31c0-17.12-13.88-31-31-31m-35 31c0-19.33 15.67-35 35-35s35 15.67 35 35-15.67 35-35 35-35-15.67-35-35m37.5 14.67c0 1.38-1.12 2.5-2.5 2.5s-2.5-1.12-2.5-2.5 1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5m-4.217-28.282c.835-.355 1.751-.447 2.635-.267s1.702.626 2.35 1.29c.647.663 1.093 1.514 1.275 2.448.18.934.087 1.901-.267 2.778s-.951 1.618-1.708 2.136c-1.107.758-2.207 1.793-3.047 3.006-.833 1.205-1.492 2.709-1.492 4.367v4.174c0 1.105.895 2 2 2s2-.895 2-2v-4.174c0-.596.245-1.316.782-2.091.53-.766 1.264-1.466 2.017-1.981 1.415-.97 2.511-2.341 3.156-3.937s.814-3.35.485-5.04c-.328-1.692-1.14-3.252-2.34-4.48-1.2-1.23-2.734-2.073-4.412-2.415-1.68-.342-3.419-.166-4.997.503-1.577.67-2.918 1.8-3.857 3.241-.94 1.44-1.438 3.129-1.438 4.852 0 1.104.895 2 2 2s2-.896 2-2c0-.954.276-1.882.788-2.667s1.234-1.389 2.07-1.743"
        })
    });
const $3c9ac1ca1f4534fc$var$ForwardRef = /*#__PURE__*/ (0, $l3uqX$react.forwardRef)($3c9ac1ca1f4534fc$var$SvgComponent);
var $3c9ac1ca1f4534fc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3c9ac1ca1f4534fc$var$ForwardRef);
