// @react-spectrum/s2/illustrations/linear/internal/S2_lin_color_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $cmCv2$jsx} from "react/jsx-runtime";
import {forwardRef as $cmCv2$forwardRef} from "react";





const $96aa09e77a371f56$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cmCv2$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cmCv2$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M50.969 17C33.331 17 19 31.441 19 49.295c0 17.569 16.311 30.647 34.255 30.647 4.828 0 11.065-1.485 15.941-3.84 2.437-1.177 4.407-2.512 5.684-3.874 1.279-1.365 1.678-2.544 1.544-3.539-.22-1.63-.78-2.338-1.31-2.74-.633-.48-1.501-.741-2.758-.999q-.235-.048-.5-.099h-.001c-1.045-.203-2.405-.467-3.52-1.035-.717-.364-1.416-.884-1.973-1.647-.557-.765-.896-1.671-1.042-2.7-.39-2.753.142-4.98 1.255-6.885 1.064-1.821 2.618-3.264 4.058-4.539l.57-.503c2.774-2.442 5.284-4.651 6.238-8.608 1.218-5.051-.783-10.443-5.496-14.696C67.238 19.99 59.915 17 50.969 17M15 49.295C15 29.268 31.086 13 50.969 13c9.816 0 18.13 3.28 23.656 8.269 5.52 4.982 8.347 11.792 6.705 18.602-1.263 5.235-4.69 8.232-7.394 10.596l-.003.004-.002.001-.647.568c-1.447 1.281-2.549 2.353-3.255 3.562-.657 1.124-1.012 2.442-.748 4.306.069.489.203.752.314.904s.275.297.554.44c.574.292 1.283.433 2.356.648h.006q.303.06.648.132c1.273.26 2.95.653 4.37 1.729 1.523 1.154 2.522 2.901 2.859 5.393.355 2.633-.855 4.958-2.589 6.808-1.735 1.853-4.18 3.445-6.863 4.741-5.365 2.592-12.193 4.239-17.68 4.239C33.678 83.942 15 69.609 15 49.295M58 31c0 3.314-2.687 6-6 6-3.315 0-6-2.686-6-6s2.685-6 6-6c3.313 0 6 2.686 6 6M37 44.915c3.313 0 6-2.687 6-6s-2.687-6-6-6c-3.315 0-6 2.686-6 6s2.685 6 6 6m6 13.533c0 3.313-2.687 6-6 6-3.315 0-6-2.686-6-6s2.685-6 6-6c3.313 0 6 2.686 6 6M52 73c3.313 0 6-2.686 6-6s-2.687-6-6-6c-3.315 0-6 2.686-6 6s2.685 6 6 6"
        })
    });
const $96aa09e77a371f56$var$ForwardRef = /*#__PURE__*/ (0, $cmCv2$forwardRef)($96aa09e77a371f56$var$SvgComponent);
var $96aa09e77a371f56$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($96aa09e77a371f56$var$ForwardRef);


export {$96aa09e77a371f56$export$2e2bcd8739ae039 as default};
