// @react-spectrum/s2/illustrations/linear/internal/S2_lin_logo_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6ZIEO$reactjsxruntime = require("react/jsx-runtime");
var $6ZIEO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $141658f457efffce$export$2e2bcd8739ae039);




const $141658f457efffce$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6ZIEO$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6ZIEO$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M87.356 22.305c-4.614-2.389-10.1-2.389-14.714 0l-42.56 22.037c-5.309 2.75-8.643 8.23-8.643 14.208v41.901c0 5.979 3.334 11.459 8.644 14.208l42.56 22.037c4.613 2.389 10.1 2.389 14.713 0l42.559-22.037c5.31-2.749 8.644-8.229 8.644-14.208v-41.9c0-5.98-3.334-11.46-8.644-14.21zM76.32 29.41c2.307-1.195 5.05-1.195 7.356 0l42.56 22.036c2.655 1.375 4.322 4.115 4.322 7.104v41.901c0 2.989-1.667 5.729-4.322 7.104l-42.56 22.036c-2.306 1.195-5.05 1.195-7.356 0l-42.56-22.036c-2.655-1.375-4.322-4.115-4.322-7.104v-41.9c0-2.99 1.667-5.73 4.322-7.105zm7.787 32.403c3.095-5.25 10.687-5.25 13.782 0l13.783 23.374c3.144 5.333-.7 12.064-6.891 12.064H77.217c-6.192 0-10.036-6.73-6.892-12.064zm20.674 27.438L90.999 65.876 77.217 89.25zM68.006 64.54c1.91 1.108 2.562 3.555 1.453 5.466l-14.5 25c-1.108 1.911-3.556 2.562-5.467 1.454-1.91-1.109-2.561-3.556-1.453-5.467l14.5-25c1.108-1.911 3.556-2.562 5.467-1.454"
        })
    });
const $141658f457efffce$var$ForwardRef = /*#__PURE__*/ (0, $6ZIEO$react.forwardRef)($141658f457efffce$var$SvgComponent);
var $141658f457efffce$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($141658f457efffce$var$ForwardRef);
