// @react-spectrum/s2/illustrations/linear/internal/S2_lin_switch_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $djlbG$reactjsxruntime = require("react/jsx-runtime");
var $djlbG$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $362d229e0949c3cd$export$2e2bcd8739ae039);




const $362d229e0949c3cd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $djlbG$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $djlbG$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M19.172 51.828c-1.562-1.562-1.562-4.094 0-5.656l23-23c1.562-1.562 4.094-1.562 5.656 0 1.563 1.562 1.563 4.094 0 5.656L31.32 45.338h104.449c2.209 0 4 1.79 4 4s-1.791 4-4 4H31.995l15.833 15.834c1.563 1.562 1.563 4.094 0 5.656-1.562 1.563-4.094 1.563-5.656 0zm98.424 33.845 23 23c1.562 1.562 1.562 4.094 0 5.656l-23 23c-1.562 1.563-4.095 1.563-5.657 0s-1.562-4.094 0-5.656l15.834-15.834H24c-2.21 0-4-1.791-4-4s1.79-4 4-4h104.448l-16.509-16.51c-1.562-1.562-1.562-4.094 0-5.656s4.095-1.562 5.657 0"
        })
    });
const $362d229e0949c3cd$var$ForwardRef = /*#__PURE__*/ (0, $djlbG$react.forwardRef)($362d229e0949c3cd$var$SvgComponent);
var $362d229e0949c3cd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($362d229e0949c3cd$var$ForwardRef);
