// @react-spectrum/s2/illustrations/linear/internal/S2_lin_text_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $2cMf9$jsx} from "react/jsx-runtime";
import {forwardRef as $2cMf9$forwardRef} from "react";





const $1429af11950130ed$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2cMf9$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2cMf9$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            d: "M14.727 14.697v-1.25c-.69 0-1.25.56-1.25 1.25zm6.648 0h1.25c0-.69-.56-1.25-1.25-1.25zm0 18.546v1.25c.69 0 1.25-.56 1.25-1.25zm5.25 0h-1.25c0 .69.56 1.25 1.25 1.25zm0-18.546v-1.25c-.69 0-1.25.56-1.25 1.25zm6.647 0h1.25c0-.69-.56-1.25-1.25-1.25zm-22.545-2.625c0-.76.615-1.375 1.375-1.375v-2.5c-2.14 0-3.875 1.735-3.875 3.875zm0 4.76v-4.76h-2.5v4.76zm1.375 1.374c-.76 0-1.375-.615-1.375-1.375h-2.5c0 2.14 1.734 3.875 3.875 3.875zm1.375-1.375c0 .76-.616 1.375-1.375 1.375v2.5c2.14 0 3.875-1.735 3.875-3.875zm0-2.134v2.134h2.5v-2.134zm7.898-1.25h-6.648v2.5h6.648zm1.25 19.796V14.697h-2.5v18.546zm-3.385 1.25h2.135v-2.5H19.24zm-1.375 1.374c0-.759.615-1.374 1.375-1.374v-2.5c-2.14 0-3.875 1.734-3.875 3.875zm1.375 1.376c-.76 0-1.375-.616-1.375-1.376h-2.5c0 2.14 1.735 3.876 3.875 3.876zm9.518 0H19.24v2.5h9.518zm1.375-1.376c0 .76-.616 1.376-1.375 1.376v2.5c2.14 0 3.875-1.735 3.875-3.876zm-1.375-1.374c.76 0 1.375.615 1.375 1.374h2.5c0-2.14-1.735-3.874-3.875-3.874zm-2.133 0h2.133v-2.5h-2.133zm-1.25-19.796v18.546h2.5V14.697zm7.897-1.25h-6.647v2.5h6.647zm1.25 3.384v-2.134h-2.5v2.134zm1.375 1.375c-.76 0-1.375-.615-1.375-1.375h-2.5c0 2.14 1.735 3.875 3.875 3.875zm1.375-1.375c0 .76-.616 1.375-1.375 1.375v2.5c2.14 0 3.875-1.735 3.875-3.875zm0-4.759v4.76h2.5v-4.76zm-1.375-1.375c.76 0 1.375.616 1.375 1.375h2.5c0-2.14-1.735-3.875-3.875-3.875zm0 0v-2.5zm-11.897 0h11.897v-2.5H24zm-11.898 0H24v-2.5H12.102z"
        })
    });
const $1429af11950130ed$var$ForwardRef = /*#__PURE__*/ (0, $2cMf9$forwardRef)($1429af11950130ed$var$SvgComponent);
var $1429af11950130ed$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1429af11950130ed$var$ForwardRef);


export {$1429af11950130ed$export$2e2bcd8739ae039 as default};
