// @react-spectrum/s2/illustrations/linear/internal/S2_lin_close_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3cgXF$reactjsxruntime = require("react/jsx-runtime");
var $3cgXF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1e77cd9dadc0030b$export$2e2bcd8739ae039);




const $1e77cd9dadc0030b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3cgXF$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3cgXF$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 8.25C15.302 8.25 8.25 15.302 8.25 24S15.302 39.75 24 39.75 39.75 32.699 39.75 24 32.699 8.25 24 8.25M5.75 24C5.75 13.92 13.92 5.75 24 5.75S42.25 13.92 42.25 24 34.08 42.25 24 42.25 5.75 34.08 5.75 24m12.366-5.884c.488-.488 1.28-.488 1.768 0l4.3 4.3 4.3-4.3c.487-.488 1.279-.488 1.767 0s.488 1.28 0 1.768l-4.3 4.3 4.3 4.3c.488.487.488 1.279 0 1.767s-1.28.488-1.768 0l-4.3-4.3-4.3 4.3c-.487.488-1.279.488-1.767 0s-.488-1.28 0-1.768l4.3-4.3-4.3-4.3c-.488-.487-.488-1.279 0-1.767"
        })
    });
const $1e77cd9dadc0030b$var$ForwardRef = /*#__PURE__*/ (0, $3cgXF$react.forwardRef)($1e77cd9dadc0030b$var$SvgComponent);
var $1e77cd9dadc0030b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1e77cd9dadc0030b$var$ForwardRef);
