// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pin_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $dxu3H$jsx} from "react/jsx-runtime";
import {forwardRef as $dxu3H$forwardRef} from "react";





const $82d573fe25a571da$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dxu3H$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dxu3H$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M18.821 7.75c-2.23 0-4.071 1.785-4.071 4.028 0 1.892 1.288 3.691 3.231 4.06v4.52c-1.866.325-3.231 1.876-3.231 3.864 0 2.243 1.84 4.028 4.071 4.028h3.929V40.5c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V28.25h3.93c2.23 0 4.07-1.785 4.07-4.028 0-1.892-1.288-3.692-3.231-4.06v-4.52c1.866-.325 3.231-1.876 3.231-3.864 0-2.243-1.84-4.028-4.07-4.028zm-1.571 4.028c0-.826.685-1.528 1.571-1.528h10.358c.886 0 1.571.702 1.571 1.528s-.583 1.428-1.467 1.428h-.514c-.69 0-1.25.56-1.25 1.25v6.888c0 .69.56 1.25 1.25 1.25h.514c.77 0 1.467.682 1.467 1.628 0 .826-.685 1.528-1.57 1.528H18.82c-.886 0-1.571-.702-1.571-1.528s.583-1.428 1.467-1.428h.514c.69 0 1.25-.56 1.25-1.25v-6.888c0-.69-.56-1.25-1.25-1.25h-.514c-.77 0-1.467-.682-1.467-1.628"
        })
    });
const $82d573fe25a571da$var$ForwardRef = /*#__PURE__*/ (0, $dxu3H$forwardRef)($82d573fe25a571da$var$SvgComponent);
var $82d573fe25a571da$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($82d573fe25a571da$var$ForwardRef);


export {$82d573fe25a571da$export$2e2bcd8739ae039 as default};
