// @react-spectrum/s2/illustrations/linear/internal/S2_lin_mention_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9xr3q$jsx} from "react/jsx-runtime";
import {forwardRef as $9xr3q$forwardRef} from "react";





const $0bb945c3c9d80518$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9xr3q$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9xr3q$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M38.09 15.274c9.7-3.18 20.1-2.114 26.84.928 13.818 6.233 18.382 19.621 16.36 31.11-2.007 11.405-10.7 21.582-23.79 20.676-12.196-.844-18.633-5.454-21.228-11.381-2.51-5.733-1.118-12.05.92-15.816 5.275-9.74 14.284-11.233 19.882-11.056 3.413.109 5.498 3.2 5.222 6.203l-2.583 28.093c9.385-.333 15.974-7.962 17.637-17.413 1.761-10.004-2.19-21.413-14.065-26.77-5.83-2.63-15.197-3.642-23.948-.773-8.631 2.83-16.676 9.433-20.144 22.162-2.757 10.121.47 19.803 6.65 26.96 6.21 7.194 15.231 11.635 23.707 11.39 1.104-.031 2.025.838 2.057 1.942s-.838 2.025-1.942 2.057c-9.882.285-19.996-4.836-26.85-12.774-6.884-7.974-10.65-18.994-7.481-30.626C19.18 26.071 28.272 18.494 38.09 15.274m17.626 48.537 2.597-28.239c.103-1.117-.644-1.816-1.366-1.839-4.838-.153-11.983 1.107-16.237 8.962-1.637 3.025-2.651 8.02-.774 12.308 1.68 3.836 5.952 7.693 15.78 8.808"
        })
    });
const $0bb945c3c9d80518$var$ForwardRef = /*#__PURE__*/ (0, $9xr3q$forwardRef)($0bb945c3c9d80518$var$SvgComponent);
var $0bb945c3c9d80518$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0bb945c3c9d80518$var$ForwardRef);


export {$0bb945c3c9d80518$export$2e2bcd8739ae039 as default};
