// @react-spectrum/s2/illustrations/linear/internal/S2_lin_reportAbuse_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $br242$jsx} from "react/jsx-runtime";
import {forwardRef as $br242$forwardRef} from "react";





const $b26595de32df052d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $br242$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $br242$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M13.123 7.875c-2.9 0-5.25 2.35-5.25 5.25v17.372c0 2.9 2.35 5.25 5.25 5.25h4.848l3.519 4.282c1.3 1.582 3.722 1.582 5.022 0l3.518-4.282h4.848c2.9 0 5.25-2.35 5.25-5.25V13.125c0-2.9-2.35-5.25-5.25-5.25zm-2.75 5.25c0-1.519 1.231-2.75 2.75-2.75h21.755c1.52 0 2.75 1.231 2.75 2.75v17.372c0 1.519-1.23 2.75-2.75 2.75H29.44c-.374 0-.729.167-.966.456l-3.894 4.739c-.3.365-.859.365-1.159 0l-3.893-4.739c-.238-.289-.592-.456-.966-.456h-5.439c-1.519 0-2.75-1.231-2.75-2.75zm13.628-.933c.69 0 1.25.56 1.25 1.25v11.105c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25V13.442c0-.69.56-1.25 1.25-1.25m0 19.24c.966 0 1.75-.783 1.75-1.75 0-.966-.784-1.75-1.75-1.75-.967 0-1.75.784-1.75 1.75 0 .967.783 1.75 1.75 1.75"
        })
    });
const $b26595de32df052d$var$ForwardRef = /*#__PURE__*/ (0, $br242$forwardRef)($b26595de32df052d$var$SvgComponent);
var $b26595de32df052d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b26595de32df052d$var$ForwardRef);


export {$b26595de32df052d$export$2e2bcd8739ae039 as default};
