// @react-spectrum/s2/illustrations/linear/internal/S2_lin_shoppingCart_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $hDn2v$reactjsxruntime = require("react/jsx-runtime");
var $hDn2v$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $75f696b4b17a436a$export$2e2bcd8739ae039);




const $75f696b4b17a436a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hDn2v$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hDn2v$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22.932 24.266c-2.21 0-4 1.79-4 4s1.79 4 4 4q.406 0 1.42.007c1.523.01 4.067.028 7.619.03 3.978.004 7.36 2.867 8 6.748l9.687 58.754c1.275 7.727 7.955 13.397 15.787 13.397h54.309c2.209 0 4-1.791 4-4s-1.791-4-4-4H65.445c-3.916 0-7.256-2.835-7.893-6.699l-1.827-11.08h58.797c5.732 0 10.664-4.054 11.773-9.678l3.82-19.363c1.458-7.395-4.176-14.302-11.732-14.32-24.394-.057-59.937-.112-69.81-.01l-.71-4.302c-1.283-7.787-8.039-13.44-15.885-13.446-3.423-.003-6.335-.022-7.898-.032zM49.89 50.04l4.516 27.383h60.116c1.911 0 3.555-1.351 3.925-3.226l3.819-19.363c.488-2.476-1.395-4.766-3.901-4.772-23.658-.055-57.413-.107-68.475-.022m19.838 81.345c0 4.418-3.581 8-8 8-4.418 0-8-3.582-8-8s3.582-8 8-8 8 3.581 8 8m43.56 8c4.418 0 8-3.582 8-8s-3.582-8-8-8-8 3.581-8 8c0 4.418 3.582 8 8 8"
        })
    });
const $75f696b4b17a436a$var$ForwardRef = /*#__PURE__*/ (0, $hDn2v$react.forwardRef)($75f696b4b17a436a$var$SvgComponent);
var $75f696b4b17a436a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($75f696b4b17a436a$var$ForwardRef);
