// @react-spectrum/s2/illustrations/linear/internal/S2_lin_warning_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $kPGIZ$jsx} from "react/jsx-runtime";
import {forwardRef as $kPGIZ$forwardRef} from "react";





const $6c65c535c8f931df$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kPGIZ$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kPGIZ$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M66.144 26.743c6.15-10.673 21.552-10.667 27.71-.003l49.07 85c6.159 10.666-1.534 24-13.854 24H30.92c-12.32 0-20.013-13.334-13.854-24zm20.782 3.997c-3.082-5.336-10.78-5.33-13.85-.003l-.002.003-49.08 85c-3.08 5.335.767 12 6.926 12h98.15c6.16 0 10.007-6.665 6.926-12zM80 48.74c2.21 0 4 1.79 4 4v39c0 2.21-1.79 4-4 4s-4-1.79-4-4v-39c0-2.21 1.79-4 4-4m6 59c0 3.314-2.686 6-6 6s-6-2.686-6-6 2.686-6 6-6 6 2.686 6 6"
        })
    });
const $6c65c535c8f931df$var$ForwardRef = /*#__PURE__*/ (0, $kPGIZ$forwardRef)($6c65c535c8f931df$var$SvgComponent);
var $6c65c535c8f931df$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6c65c535c8f931df$var$ForwardRef);


export {$6c65c535c8f931df$export$2e2bcd8739ae039 as default};
