// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pause_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $jzEfM$reactjsxruntime = require("react/jsx-runtime");
var $jzEfM$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ab63fb0eccf87812$export$2e2bcd8739ae039);




const $ab63fb0eccf87812$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jzEfM$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jzEfM$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M41 46c0-6.627 5.373-12 12-12h11c6.627 0 12 5.373 12 12v68c0 6.627-5.373 12-12 12H53c-6.627 0-12-5.373-12-12zm12-4c-2.21 0-4 1.79-4 4v68c0 2.209 1.79 4 4 4h11c2.21 0 4-1.791 4-4V46c0-2.21-1.79-4-4-4zm32 4c0-6.627 5.373-12 12-12h11c6.627 0 12 5.373 12 12v68c0 6.627-5.373 12-12 12H97c-6.627 0-12-5.373-12-12zm12-4c-2.21 0-4 1.79-4 4v68c0 2.209 1.79 4 4 4h11c2.209 0 4-1.791 4-4V46c0-2.21-1.791-4-4-4z"
        })
    });
const $ab63fb0eccf87812$var$ForwardRef = /*#__PURE__*/ (0, $jzEfM$react.forwardRef)($ab63fb0eccf87812$var$SvgComponent);
var $ab63fb0eccf87812$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ab63fb0eccf87812$var$ForwardRef);
