// @react-spectrum/s2/illustrations/linear/internal/S2_lin_3Dasset_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3cvgo$reactjsxruntime = require("react/jsx-runtime");
var $3cvgo$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $fb46ec9e2fea5ef5$export$2e2bcd8739ae039);




const $fb46ec9e2fea5ef5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3cvgo$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3cvgo$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M34.6 8.293c-.982-.547-2.178-.547-3.161 0l-7.073 3.938c-.227.126-.503.126-.73 0L16.86 8.458c-1.166-.649-2.633-.583-3.738.046-1.2.683-3.1 1.731-4.913 2.722-.98.535-1.537 1.453-1.666 2.42q-.038.148-.038.307v12.248c0 1.778.9 3.435 2.392 4.404l14.41 9.35c.412.268.944.269 1.358.001l14.47-9.351c1.497-.967 2.401-2.628 2.401-4.41V13.953q0-.105-.017-.205c-.095-1-.647-1.963-1.654-2.524zm4.435 9.072-13.8 7.682V36.61l12.543-8.105c.784-.506 1.257-1.376 1.257-2.31zm-16.3 7.665L9.005 17.385v8.816c0 .931.472 1.8 1.253 2.306l12.479 8.098zm9.92-14.553c.227-.126.503-.126.73 0l5.263 2.931c.514.286.514 1.025 0 1.31l-14.647 8.155-14.602-8.13c-.516-.286-.517-1.037.008-1.324 1.812-.99 3.73-2.047 4.951-2.742.4-.228.936-.229 1.284-.035l6.778 3.773c.983.548 2.179.548 3.162 0z"
        })
    });
const $fb46ec9e2fea5ef5$var$ForwardRef = /*#__PURE__*/ (0, $3cvgo$react.forwardRef)($fb46ec9e2fea5ef5$var$SvgComponent);
var $fb46ec9e2fea5ef5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($fb46ec9e2fea5ef5$var$ForwardRef);
