// @react-spectrum/s2/illustrations/linear/internal/S2_lin_flagCross_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1TV3x$reactjsxruntime = require("react/jsx-runtime");
var $1TV3x$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f11ef6a684877f18$export$2e2bcd8739ae039);




const $f11ef6a684877f18$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1TV3x$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1TV3x$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25.414 18.71c0-1.104-.895-2-2-2s-2 .896-2 2v59.563c0 1.105.895 2 2 2s2-.895 2-2v-15.78h41.172c4.418 0 8-3.582 8-8V28.535c0-4.418-3.582-8-8-8H25.414zm0 5.825v33.958h41.172c2.21 0 4-1.79 4-4V28.535c0-2.209-1.79-4-4-4zm14.974 9.11c.78-.782 2.047-.782 2.828 0l5.042 5.041 5.041-5.042c.781-.78 2.047-.78 2.829 0 .78.781.78 2.048 0 2.829l-5.042 5.041 5.042 5.042c.78.781.78 2.047 0 2.828-.782.782-2.048.782-2.829 0l-5.041-5.041-5.042 5.041c-.781.782-2.047.782-2.828 0-.782-.78-.782-2.047 0-2.828l5.041-5.042-5.041-5.041c-.782-.781-.782-2.048 0-2.829"
        })
    });
const $f11ef6a684877f18$var$ForwardRef = /*#__PURE__*/ (0, $1TV3x$react.forwardRef)($f11ef6a684877f18$var$SvgComponent);
var $f11ef6a684877f18$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f11ef6a684877f18$var$ForwardRef);
