// @react-spectrum/s2/illustrations/linear/internal/S2_lin_replace_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $3N0s8$jsx} from "react/jsx-runtime";
import {forwardRef as $3N0s8$forwardRef} from "react";





const $4e58d8502c5dddc0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3N0s8$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3N0s8$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24.23 40.268c0-8.837 7.164-16 16-16h15.338c8.836 0 16 7.163 16 16v15.337c0 8.837-7.164 16-16 16H40.23c-8.837 0-16-7.163-16-16zm16-8c-4.418 0-8 3.581-8 8v15.337c0 4.418 3.582 8 8 8h15.338c4.418 0 8-3.582 8-8V40.268c0-4.419-3.582-8-8-8zM91.974 26.2c-2.207-.075-4.058 1.654-4.133 3.862s1.654 4.058 3.862 4.133c4.874.165 9.865 1.57 13.628 5.21 3.247 3.141 6.031 8.386 6.675 17.199l-6.177-6.178c-1.562-1.562-4.095-1.562-5.657 0s-1.562 4.095 0 5.657l12.424 12.424c1.562 1.562 4.095 1.562 5.657 0l12.464-12.464c1.562-1.562 1.562-4.095 0-5.657s-4.095-1.562-5.657 0l-5.116 5.117c-.829-10.013-4.146-17.102-9.051-21.848-5.645-5.46-12.808-7.248-18.919-7.455M47.244 91.493c-1.562-1.562-4.094-1.562-5.657 0l-12.464 12.464c-1.562 1.562-1.562 4.095 0 5.657s4.095 1.562 5.657 0l5.116-5.116c.829 10.012 4.146 17.101 9.05 21.847 5.645 5.461 12.809 7.248 18.92 7.455 2.207.075 4.058-1.654 4.133-3.862s-1.655-4.058-3.862-4.133c-4.874-.165-9.865-1.57-13.628-5.21-3.247-3.141-6.032-8.387-6.676-17.199l6.179 6.178c1.562 1.562 4.094 1.562 5.656 0s1.562-4.095 0-5.657zm57.188-3.097c-8.836 0-16 7.163-16 16v15.337c0 8.836 7.164 16 16 16h15.338c8.836 0 16-7.164 16-16v-15.337c0-8.837-7.164-16-16-16zm-8 16c0-4.419 3.582-8 8-8h15.338c4.418 0 8 3.581 8 8v15.337c0 4.418-3.582 8-8 8h-15.338c-4.418 0-8-3.582-8-8z"
        })
    });
const $4e58d8502c5dddc0$var$ForwardRef = /*#__PURE__*/ (0, $3N0s8$forwardRef)($4e58d8502c5dddc0$var$SvgComponent);
var $4e58d8502c5dddc0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($4e58d8502c5dddc0$var$ForwardRef);


export {$4e58d8502c5dddc0$export$2e2bcd8739ae039 as default};
