// @react-spectrum/s2/illustrations/linear/internal/S2_lin_exportTo_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $2UKQK$jsx} from "react/jsx-runtime";
import {forwardRef as $2UKQK$forwardRef} from "react";





const $2b6661d04fb9db49$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2UKQK$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2UKQK$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M12.499 12.75c-1.519 0-2.75 1.231-2.75 2.75v20c0 1.519 1.231 2.75 2.75 2.75h20c1.519 0 2.75-1.231 2.75-2.75V21c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v14.5c0 2.9-2.35 5.25-5.25 5.25h-20c-2.9 0-5.25-2.35-5.25-5.25v-20c0-2.9 2.35-5.25 5.25-5.25h14.5c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25zm11.116 11.667c-.488-.488-.488-1.28 0-1.768l12.637-12.636h-4.254c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h7.525c.69 0 1.25.56 1.25 1.25v7.525c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-4.762l-12.89 12.891c-.488.488-1.28.488-1.768 0"
        })
    });
const $2b6661d04fb9db49$var$ForwardRef = /*#__PURE__*/ (0, $2UKQK$forwardRef)($2b6661d04fb9db49$var$SvgComponent);
var $2b6661d04fb9db49$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2b6661d04fb9db49$var$ForwardRef);


export {$2b6661d04fb9db49$export$2e2bcd8739ae039 as default};
