// @react-spectrum/s2/illustrations/linear/internal/S2_lin_video_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $4Sn9t$reactjsxruntime = require("react/jsx-runtime");
var $4Sn9t$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3c0d0fbd405ec321$export$2e2bcd8739ae039);




const $3c0d0fbd405ec321$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4Sn9t$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4Sn9t$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M136 50c0-6.627-5.373-12-12-12H36c-6.627 0-12 5.373-12 12v60c0 6.627 5.373 12 12 12h88c6.627 0 12-5.373 12-12zm-12-20c11.046 0 20 8.954 20 20v60c0 11.046-8.954 20-20 20H36c-11.046 0-20-8.954-20-20V50c0-11.046 8.954-20 20-20zM71.811 64.479l20.933 12.119c2.344 1.357 2.34 4.743-.007 6.095L71.775 94.762c-2.348 1.352-5.278-.345-5.275-3.054l.029-24.188c.003-2.709 2.938-4.398 5.282-3.041"
        })
    });
const $3c0d0fbd405ec321$var$ForwardRef = /*#__PURE__*/ (0, $4Sn9t$react.forwardRef)($3c0d0fbd405ec321$var$SvgComponent);
var $3c0d0fbd405ec321$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3c0d0fbd405ec321$var$ForwardRef);
