// @react-spectrum/s2/illustrations/linear/internal/S2_lin_artboard_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $7aZ5Z$reactjsxruntime = require("react/jsx-runtime");
var $7aZ5Z$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7af1c3a9f2d63688$export$2e2bcd8739ae039);




const $7af1c3a9f2d63688$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7aZ5Z$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7aZ5Z$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M40 14c0-2.21-1.79-4-4-4s-4 1.79-4 4v10c0 2.21 1.79 4 4 4s4-1.79 4-4zM14 32c-2.21 0-4 1.79-4 4s1.79 4 4 4h10c2.21 0 4-1.79 4-4s-1.79-4-4-4zm34 8c-4.418 0-8 3.582-8 8v80c0 4.418 3.582 8 8 8h64c4.418 0 8-3.582 8-8V80H91c-6.627 0-12-5.373-12-12V40zm39 4.44V68c0 2.21 1.79 4 4 4h24.634zM32 48c0-8.837 7.163-16 16-16h31.163c4.138 0 8.115 1.603 11.096 4.472l32.836 31.606c3.134 3.016 4.905 7.178 4.905 11.527V128c0 8.837-7.163 16-16 16H48c-8.837 0-16-7.163-16-16z"
        })
    });
const $7af1c3a9f2d63688$var$ForwardRef = /*#__PURE__*/ (0, $7aZ5Z$react.forwardRef)($7af1c3a9f2d63688$var$SvgComponent);
var $7af1c3a9f2d63688$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7af1c3a9f2d63688$var$ForwardRef);
