// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pieChart_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $iZFcQ$reactjsxruntime = require("react/jsx-runtime");
var $iZFcQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $44aff275990f1127$export$2e2bcd8739ae039);




const $44aff275990f1127$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iZFcQ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $iZFcQ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M62.613 20.795c2.084 1.114 4.026 2.458 5.792 4.002 1.082.947 2.098 1.969 3.04 3.056q1.257 1.453 2.33 3.06c2.833 4.238 4.576 9.11 5.076 14.183q.075.768.112 1.537.036.746.036 1.502 0 .752-.035 1.497c-.172 3.552-.954 7.06-2.324 10.366-1.276 3.082-3.037 5.926-5.206 8.43-.936 1.08-1.946 2.096-3.022 3.038q-.368.321-.746.632c-3.94 3.234-8.618 5.447-13.618 6.441-5 .995-10.168.74-15.046-.74-4.24-1.285-8.146-3.462-11.462-6.375l21.874-21.875c.376-.375.586-.884.586-1.414V17.199c4.405.285 8.707 1.508 12.613 3.596M45.115 48.192 24.711 68.596C19.911 63.136 17 55.976 17 48.135c0-7.138 2.412-13.712 6.466-18.952zM46 43.646V17.198c-7.748.493-14.722 3.833-19.888 8.984zm-33 4.489c0-19.33 15.67-35 35-35h.062c8.849.015 16.927 3.315 23.082 8.744 1.143 1.008 2.22 2.09 3.223 3.24 5.055 5.785 8.236 13.25 8.598 21.447q.07 1.569 0 3.13c-.36 8.208-3.547 15.68-8.61 21.47-.996 1.139-2.066 2.215-3.206 3.22-6.17 5.445-14.274 8.749-23.15 8.749-19.33 0-34.999-15.67-34.999-35"
        })
    });
const $44aff275990f1127$var$ForwardRef = /*#__PURE__*/ (0, $iZFcQ$react.forwardRef)($44aff275990f1127$var$SvgComponent);
var $44aff275990f1127$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($44aff275990f1127$var$ForwardRef);
