// @react-spectrum/s2/illustrations/linear/internal/S2_lin_laptop_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $hE8KP$reactjsxruntime = require("react/jsx-runtime");
var $hE8KP$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f97f4b5b162884c6$export$2e2bcd8739ae039);




const $f97f4b5b162884c6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hE8KP$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hE8KP$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M15 12.25c-1.52 0-2.75 1.23-2.75 2.75v15.75h23.5V15c0-1.52-1.23-2.75-2.75-2.75zM9.75 15v15.75H8.5c-2.071 0-3.75 1.679-3.75 3.75s1.679 3.75 3.75 3.75h31c2.071 0 3.75-1.679 3.75-3.75s-1.679-3.75-3.75-3.75h-1.25V15c0-2.9-2.35-5.25-5.25-5.25H15c-2.9 0-5.25 2.35-5.25 5.25M8.5 33.25c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h31c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25zm17-17.75c0 .828-.672 1.5-1.5 1.5s-1.5-.672-1.5-1.5.672-1.5 1.5-1.5 1.5.672 1.5 1.5"
        })
    });
const $f97f4b5b162884c6$var$ForwardRef = /*#__PURE__*/ (0, $hE8KP$react.forwardRef)($f97f4b5b162884c6$var$SvgComponent);
var $f97f4b5b162884c6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f97f4b5b162884c6$var$ForwardRef);
