// @react-spectrum/s2/illustrations/linear/internal/S2_lin_musicNote_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $hrHlj$jsx} from "react/jsx-runtime";
import {forwardRef as $hrHlj$forwardRef} from "react";





const $e83ec477924c52d0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hrHlj$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hrHlj$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M120.338 27.374c3.712-.74 7.173 2.1 7.173 5.885v5.274L68.902 50.217V42.55c0-2.862 2.02-5.325 4.827-5.885zm-51.436 31v56.643q0 .09-.004.177c.102 3.345-1.008 6.661-2.819 9.542-2.706 4.305-7.164 8.025-12.633 10.374-5.47 2.35-11.236 3.023-16.222 2.022-4.966-.998-9.562-3.784-11.657-8.661-2.095-4.878-.952-10.129 1.743-14.417 2.706-4.306 7.164-8.025 12.634-10.375 5.469-2.35 11.236-3.022 16.221-2.021 1.646.33 3.25.857 4.737 1.591V42.551c0-6.677 4.715-12.425 11.263-13.73l46.609-9.292c8.661-1.727 16.737 4.898 16.737 13.73v68.951q.006.299 0 .597v.422q0 .398-.075.776c-.279 2.681-1.284 5.292-2.747 7.619-2.706 4.306-7.164 8.025-12.634 10.375-5.469 2.35-11.236 3.022-16.222 2.021-4.965-.997-9.561-3.783-11.656-8.661-2.096-4.877-.953-10.129 1.743-14.417 2.706-4.305 7.163-8.025 12.633-10.374s11.236-3.023 16.222-2.022c1.645.33 3.25.858 4.736 1.592V46.69zm-34.819 59.937c-1.749 2.783-1.927 5.23-1.166 7.002s2.66 3.328 5.882 3.975c3.203.643 7.32.263 11.49-1.528 4.168-1.791 7.279-4.516 9.017-7.281 1.75-2.783 1.927-5.231 1.166-7.003s-2.659-3.328-5.882-3.975c-3.203-.643-7.32-.262-11.489 1.529s-7.28 4.515-9.018 7.281m65.444-6.109c-.761-1.772-.583-4.22 1.166-7.003 1.738-2.765 4.849-5.49 9.018-7.28 4.169-1.792 8.286-2.172 11.489-1.53 3.223.648 5.12 2.204 5.881 3.976s.584 4.219-1.165 7.002c-1.739 2.766-4.849 5.491-9.018 7.282-4.169 1.79-8.287 2.171-11.49 1.528-3.222-.647-5.12-2.203-5.88-3.975"
        })
    });
const $e83ec477924c52d0$var$ForwardRef = /*#__PURE__*/ (0, $hrHlj$forwardRef)($e83ec477924c52d0$var$SvgComponent);
var $e83ec477924c52d0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e83ec477924c52d0$var$ForwardRef);


export {$e83ec477924c52d0$export$2e2bcd8739ae039 as default};
