// @react-spectrum/s2/illustrations/linear/internal/S2_lin_tablet_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $btPUS$reactjsxruntime = require("react/jsx-runtime");
var $btPUS$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f8b15de3cc0616cf$export$2e2bcd8739ae039);




const $f8b15de3cc0616cf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $btPUS$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $btPUS$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M13 26c0-3.314 2.686-6 6-6h58c3.314 0 6 2.686 6 6v44c0 3.314-2.686 6-6 6H19c-3.314 0-6-2.686-6-6zm6-10c-5.523 0-10 4.477-10 10v44c0 5.523 4.477 10 10 10h58c5.523 0 10-4.477 10-10V26c0-5.523-4.477-10-10-10zm4 31.5c0 1.38-1.12 2.5-2.5 2.5S18 48.88 18 47.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5"
        })
    });
const $f8b15de3cc0616cf$var$ForwardRef = /*#__PURE__*/ (0, $btPUS$react.forwardRef)($f8b15de3cc0616cf$var$SvgComponent);
var $f8b15de3cc0616cf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f8b15de3cc0616cf$var$ForwardRef);
