// @react-spectrum/s2/illustrations/linear/internal/S2_lin_layers_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bmDE9$reactjsxruntime = require("react/jsx-runtime");
var $bmDE9$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5b5ee458fe84aa55$export$2e2bcd8739ae039);




const $5b5ee458fe84aa55$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bmDE9$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bmDE9$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28.622 54.894c-6.113 4.168-6.113 13.182 0 17.35l43.058 29.358c4.758 3.243 11.016 3.243 15.774 0l43.058-29.358c6.113-4.168 6.113-13.182 0-17.35L87.454 25.536c-4.758-3.243-11.016-3.243-15.773 0zm4.507 10.74c-1.456-.992-1.456-3.138 0-4.13l43.058-29.358c2.039-1.39 4.721-1.39 6.76 0l43.058 29.357c1.456.993 1.456 3.14 0 4.132L82.947 94.992c-2.039 1.39-4.721 1.39-6.76 0zm0 33.5c-1.456-.992-1.456-3.138 0-4.13l6.265-4.272c1.825-1.245 2.296-3.733 1.052-5.558-1.245-1.826-3.733-2.297-5.559-1.052l-6.265 4.272c-6.113 4.168-6.113 13.182 0 17.35l43.058 29.358c4.758 3.243 11.016 3.243 15.774 0l43.058-29.358c6.113-4.168 6.113-13.182 0-17.35l-6.265-4.272c-1.825-1.245-4.314-.774-5.558 1.052-1.245 1.825-.774 4.313 1.051 5.558l6.265 4.271c1.456.993 1.456 3.14 0 4.132l-43.058 29.357c-2.039 1.39-4.721 1.39-6.76 0z"
        })
    });
const $5b5ee458fe84aa55$var$ForwardRef = /*#__PURE__*/ (0, $bmDE9$react.forwardRef)($5b5ee458fe84aa55$var$SvgComponent);
var $5b5ee458fe84aa55$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5b5ee458fe84aa55$var$ForwardRef);
