// @react-spectrum/s2/illustrations/linear/internal/S2_lin_interaction_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $hlqEn$reactjsxruntime = require("react/jsx-runtime");
var $hlqEn$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b07185142846619f$export$2e2bcd8739ae039);




const $b07185142846619f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hlqEn$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hlqEn$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M19 34.676c0-9.49 7.693-17.184 17.183-17.184s17.184 7.694 17.184 17.184c0 2.461-.517 4.797-1.445 6.909-.445 1.011.015 2.191 1.026 2.636 1.01.444 2.19-.015 2.636-1.026 1.147-2.61 1.783-5.493 1.783-8.52 0-11.699-9.484-21.183-21.184-21.183C24.484 13.492 15 22.976 15 34.676c0 10.853 8.161 19.798 18.682 21.037 1.097.129 2.091-.656 2.22-1.753.13-1.097-.655-2.09-1.752-2.22C25.62 50.736 19 43.477 19 34.676m18.877 2.038c.887-.48 2.274-.167 3.13 1.244l7.212 12.979c.73 1.316 2.389 1.648 3.57.883 1.379-.893 3.538-1.874 5.84-2.634 2.32-.764 4.561-1.23 6.11-1.23 4.407 0 7.707 1.681 9.94 4.368 2.27 2.728 3.56 6.65 3.56 11.266 0 8.852-7.076 16-15.897 16-4.549 0-10.51-1.894-15.894-4.105-4.538-1.864-8.444-3.856-10.451-4.88q-.52-.267-.863-.438c-1.239-.62-1.742-1.223-1.953-1.684-.207-.453-.269-1.067-.013-1.997.343-1.249 1.682-2.13 3.525-1.607 1.928.548 4.07 1.402 6.264 2.332.624.265 1.256.537 1.887.809 1.539.663 3.07 1.323 4.453 1.854 2.25.865 4.147-1.569 3.057-3.53l-14.366-25.86-.02-.034-.02-.033c-.763-1.255-.267-3.055.93-3.703m6.579-.784c-1.605-2.685-5.16-4.535-8.485-2.733-3.288 1.783-4.327 6.147-2.463 9.26l12.236 22.026-.233-.1q-.972-.421-1.993-.854c-2.21-.937-4.548-1.876-6.731-2.497-3.596-1.023-7.442.635-8.476 4.393-.42 1.525-.48 3.16.231 4.72.708 1.55 2.042 2.719 3.803 3.6q.315.157.814.413c1.986 1.014 6.06 3.093 10.769 5.027 5.432 2.231 12.021 4.405 17.414 4.405 11.06 0 19.897-8.97 19.897-20 0-5.325-1.488-10.22-4.484-13.824-3.031-3.645-7.48-5.81-13.016-5.81-2.167 0-4.876.611-7.362 1.432-1.912.63-3.822 1.427-5.401 2.275l-6.488-11.678-.016-.028z"
        })
    });
const $b07185142846619f$var$ForwardRef = /*#__PURE__*/ (0, $hlqEn$react.forwardRef)($b07185142846619f$var$SvgComponent);
var $b07185142846619f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b07185142846619f$var$ForwardRef);
