// @react-spectrum/s2/illustrations/linear/internal/S2_lin_userAvatar_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $9sETL$reactjsxruntime = require("react/jsx-runtime");
var $9sETL$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $fafba1dbd5e25163$export$2e2bcd8739ae039);




const $fafba1dbd5e25163$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9sETL$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9sETL$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 8.621c-8.422 0-15.25 6.828-15.25 15.25 0 3.877 1.446 7.415 3.829 10.106 2.364-3.837 6.609-6.421 11.422-6.421s9.056 2.586 11.42 6.42c2.383-2.69 3.829-6.228 3.829-10.105 0-8.422-6.828-15.25-15.25-15.25m0 30.5c-3.62 0-6.947-1.262-9.563-3.37 1.865-3.388 5.473-5.695 9.564-5.695 4.089 0 7.696 2.308 9.562 5.695-2.616 2.108-5.942 3.37-9.563 3.37M6.25 23.871c0-9.803 7.947-17.75 17.75-17.75s17.75 7.947 17.75 17.75-7.947 17.75-17.75 17.75-17.75-7.947-17.75-17.75m12.956-5.025c0-2.941 2.22-5.189 4.794-5.189 2.575 0 4.794 2.248 4.794 5.19 0 2.94-2.22 5.188-4.794 5.188s-4.794-2.247-4.794-5.189M24 11.157c-4.101 0-7.294 3.518-7.294 7.69 0 4.17 3.193 7.688 7.294 7.688 4.102 0 7.294-3.518 7.294-7.689 0-4.17-3.192-7.689-7.294-7.689"
        })
    });
const $fafba1dbd5e25163$var$ForwardRef = /*#__PURE__*/ (0, $9sETL$react.forwardRef)($fafba1dbd5e25163$var$SvgComponent);
var $fafba1dbd5e25163$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($fafba1dbd5e25163$var$ForwardRef);
