// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noFile_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $dUVNq$jsx} from "react/jsx-runtime";
import {forwardRef as $dUVNq$forwardRef} from "react";





const $365a382c0d13c03b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dUVNq$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dUVNq$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M14 18c0-4.41 3.571-8 7.99-8h19.45c2.047 0 4.036.783 5.526 2.218l.002.002 19.565 18.745C68.106 32.472 69 34.553 69 36.74v14.75c0 1.105-.895 2-2 2s-2-.895-2-2v-14.5H47.37c-3.333 0-6.37-2.502-6.37-5.98V14H21.99C19.789 14 18 15.79 18 18v47.99c0 2.21 1.789 4 3.99 4H46c1.105 0 2 .895 2 2s-.895 2-2 2H21.99c-4.419 0-7.99-3.59-7.99-8zm31-2.126L62.864 32.99H47.37c-1.487 0-2.37-1.058-2.37-1.98zM66.74 86.49c8.146 0 14.75-6.604 14.75-14.75s-6.604-14.75-14.75-14.75-14.75 6.604-14.75 14.75 6.604 14.75 14.75 14.75m2.248-21.5c0-1.105-.895-2-2-2s-2 .895-2 2v5h-5c-1.104 0-2 .895-2 2s.896 2 2 2h5v5c0 1.104.896 2 2 2s2-.896 2-2v-5h5c1.105 0 2-.896 2-2s-.895-2-2-2h-5z"
        })
    });
const $365a382c0d13c03b$var$ForwardRef = /*#__PURE__*/ (0, $dUVNq$forwardRef)($365a382c0d13c03b$var$SvgComponent);
var $365a382c0d13c03b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($365a382c0d13c03b$var$ForwardRef);


export {$365a382c0d13c03b$export$2e2bcd8739ae039 as default};
