// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noFilter_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $bwLQ7$jsx} from "react/jsx-runtime";
import {forwardRef as $bwLQ7$forwardRef} from "react";





const $111dd44340d901b5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bwLQ7$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bwLQ7$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M101.991 30.59c0-13.53 10.97-24.5 24.5-24.5s24.5 10.97 24.5 24.5-10.97 24.5-24.5 24.5-24.5-10.97-24.5-24.5m35.42 10.93c1.56-1.57 1.56-4.1 0-5.66l-5.26-5.26 5.26-5.26c1.56-1.57 1.56-4.1 0-5.66s-4.1-1.56-5.66 0l-5.26 5.26-5.26-5.26c-1.56-1.56-4.1-1.56-5.66 0-1.56 1.57-1.56 4.1 0 5.66l5.26 5.26-5.26 5.26c-1.56 1.57-1.56 4.1 0 5.66.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17l5.26-5.26 5.26 5.26c.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17M38.829 23.79c-9.926 0-15.56 11.364-9.552 19.264q.064.084.133.166l37.894 44.695v47.086c0 6.884 8.118 10.553 13.286 6.005l9.392-8.267c1.726-1.519 2.715-3.706 2.715-6.005v-38.82l20.012-23.603c1.429-1.685 1.221-4.21-.464-5.638-1.685-1.429-4.209-1.22-5.638.464L85.646 83.862c-.613.722-.95 1.64-.95 2.587v40.286l-9.392 8.267V86.448c0-.947-.336-1.864-.949-2.586l-38.763-45.72c-1.916-2.63-.042-6.351 3.237-6.351h52.28c2.21 0 4-1.79 4-4s-1.79-4-4-4z"
        })
    });
const $111dd44340d901b5$var$ForwardRef = /*#__PURE__*/ (0, $bwLQ7$forwardRef)($111dd44340d901b5$var$SvgComponent);
var $111dd44340d901b5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($111dd44340d901b5$var$ForwardRef);


export {$111dd44340d901b5$export$2e2bcd8739ae039 as default};
