// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowUp_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $3e7S6$jsx} from "react/jsx-runtime";
import {forwardRef as $3e7S6$forwardRef} from "react";





const $06dbc0e8984f2c42$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3e7S6$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3e7S6$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M76 136c0 2.209 1.79 4 4 4s4-1.791 4-4V31.157l16.672 16.671c1.562 1.563 4.094 1.563 5.656 0 1.563-1.562 1.563-4.094 0-5.656l-23.5-23.5c-1.562-1.562-4.094-1.562-5.656 0l-23.5 23.5c-1.562 1.562-1.562 4.094 0 5.656 1.562 1.563 4.094 1.563 5.656 0L76 31.157z"
        })
    });
const $06dbc0e8984f2c42$var$ForwardRef = /*#__PURE__*/ (0, $3e7S6$forwardRef)($06dbc0e8984f2c42$var$SvgComponent);
var $06dbc0e8984f2c42$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($06dbc0e8984f2c42$var$ForwardRef);


export {$06dbc0e8984f2c42$export$2e2bcd8739ae039 as default};
