// @react-spectrum/s2/illustrations/linear/internal/S2_lin_wallet_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $lgUWo$jsx} from "react/jsx-runtime";
import {forwardRef as $lgUWo$forwardRef} from "react";





const $260d5cc8993dca71$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lgUWo$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $lgUWo$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M36.86 51.477c-1.514-5.653 1.84-11.464 7.494-12.98l50.23-13.458c6.401-1.716 12.981 2.083 14.696 8.485l4.565 17.034c.571 2.134-.695 4.327-2.829 4.9-2.134.57-4.327-.695-4.899-2.83l-4.564-17.033c-.572-2.134-2.765-3.4-4.899-2.829l-50.229 13.46c-1.386.37-2.208 1.795-1.837 3.18l.309 1.152c.571 2.134-.695 4.327-2.829 4.9-2.134.57-4.327-.695-4.899-2.83zm-7.504 7.588c-4.419 0-8 3.582-8 8v60.307c0 4.418 3.581 8 8 8h82.459c11.046 0 20-8.955 20-20v-1.431c3.864-.566 6.831-3.894 6.831-7.915V88.411c0-4.021-2.967-7.35-6.831-7.915v-1.43c0-11.047-8.954-20-20-20zm94.459 21.346v-1.346c0-6.627-5.372-12-12-12h-82.46v60.307h82.46c6.628 0 12-5.373 12-12v-1.346h-19.323c-4.418 0-8-3.582-8-8V88.411c0-4.418 3.582-8 8-8zm-19.323 8h26.154v17.615h-26.154zm9.039 13.308c2.485 0 4.5-2.014 4.5-4.5 0-2.485-2.015-4.5-4.5-4.5s-4.5 2.015-4.5 4.5c0 2.486 2.015 4.5 4.5 4.5"
        })
    });
const $260d5cc8993dca71$var$ForwardRef = /*#__PURE__*/ (0, $lgUWo$forwardRef)($260d5cc8993dca71$var$SvgComponent);
var $260d5cc8993dca71$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($260d5cc8993dca71$var$ForwardRef);


export {$260d5cc8993dca71$export$2e2bcd8739ae039 as default};
