// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowUp_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5W2WM$jsx} from "react/jsx-runtime";
import {forwardRef as $5W2WM$forwardRef} from "react";





const $481ae9705e65ea00$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5W2WM$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5W2WM$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22.75 41.5c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V10.518l4.866 4.866c.488.488 1.28.488 1.768 0s.488-1.28 0-1.768l-7-7c-.488-.488-1.28-.488-1.768 0l-7 7c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0l4.866-4.866z"
        })
    });
const $481ae9705e65ea00$var$ForwardRef = /*#__PURE__*/ (0, $5W2WM$forwardRef)($481ae9705e65ea00$var$SvgComponent);
var $481ae9705e65ea00$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($481ae9705e65ea00$var$ForwardRef);


export {$481ae9705e65ea00$export$2e2bcd8739ae039 as default};
