// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileVideo_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $kFnN7$reactjsxruntime = require("react/jsx-runtime");
var $kFnN7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $cca165dddb7459f6$export$2e2bcd8739ae039);




const $cca165dddb7459f6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kFnN7$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kFnN7$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M30.09 16C25.117 16 21 19.95 21 24.923v46.154C21 76.05 25.116 80 30.09 80h37.82c4.974 0 9.09-3.95 9.09-8.923V43.945c0-2.383-.97-4.657-2.68-6.327L54.838 18.597C53.129 16.929 50.822 16 48.426 16zM25 24.923C25 22.25 27.234 20 30.09 20h18.336c1.366 0 2.666.53 3.617 1.459L71.526 40.48c.95.927 1.474 2.175 1.474 3.465v27.132C73 73.75 70.766 76 67.91 76H30.09C27.235 76 25 73.751 25 71.077zm19.345 20.542 11.214 6.432c1.256.72 1.254 2.517-.003 3.235l-11.23 6.406c-1.258.717-2.828-.183-2.826-1.621l.016-12.838c.001-1.438 1.573-2.335 2.83-1.614"
        })
    });
const $cca165dddb7459f6$var$ForwardRef = /*#__PURE__*/ (0, $kFnN7$react.forwardRef)($cca165dddb7459f6$var$SvgComponent);
var $cca165dddb7459f6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($cca165dddb7459f6$var$ForwardRef);
