// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cursor_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $5U5rJ$reactjsxruntime = require("react/jsx-runtime");
var $5U5rJ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $23702410112e0102$export$2e2bcd8739ae039);




const $23702410112e0102$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5U5rJ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5U5rJ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M7.231 13.375c-1.67-4.056 2.386-8.113 6.443-6.443l25.975 10.696c4.315 1.777 4.424 7.848.176 9.78l-7.366 3.347c-.623.283-1.122.783-1.405 1.405l-3.348 7.366c-1.931 4.249-8.003 4.14-9.78-.176zm5.491-4.131c-2.002-.824-4.003 1.178-3.18 3.18l-1.155.475 1.156-.476 10.696 25.975c.943 2.291 4.166 2.35 5.191.094l3.348-7.366c.534-1.173 1.474-2.113 2.646-2.646l7.367-3.349c2.255-1.025 2.197-4.248-.094-5.191z"
        })
    });
const $23702410112e0102$var$ForwardRef = /*#__PURE__*/ (0, $5U5rJ$react.forwardRef)($23702410112e0102$var$SvgComponent);
var $23702410112e0102$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($23702410112e0102$var$ForwardRef);
