// @react-spectrum/s2/illustrations/linear/internal/S2_lin_phone_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bM72c$reactjsxruntime = require("react/jsx-runtime");
var $bM72c$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f57d48f024fcbba3$export$2e2bcd8739ae039);




const $f57d48f024fcbba3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bM72c$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bM72c$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M55 40c0-4.418 3.582-8 8-8h34c4.418 0 8 3.582 8 8v80c0 4.418-3.582 8-8 8H63c-4.418 0-8-3.582-8-8zm8-16c-8.837 0-16 7.163-16 16v80c0 8.837 7.163 16 16 16h34c8.837 0 16-7.163 16-16V40c0-8.837-7.163-16-16-16zm22 92.5c0 2.485-2.015 4.5-4.5 4.5s-4.5-2.015-4.5-4.5 2.015-4.5 4.5-4.5 4.5 2.015 4.5 4.5"
        })
    });
const $f57d48f024fcbba3$var$ForwardRef = /*#__PURE__*/ (0, $bM72c$react.forwardRef)($f57d48f024fcbba3$var$SvgComponent);
var $f57d48f024fcbba3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f57d48f024fcbba3$var$ForwardRef);
