// @react-spectrum/s2/illustrations/linear/internal/S2_lin_audioWave_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $k29Ve$jsx} from "react/jsx-runtime";
import {forwardRef as $k29Ve$forwardRef} from "react";





const $f8ebd9a1f7109682$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $k29Ve$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $k29Ve$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25.25 7.492c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v33.016c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25zM9.125 20.258c.69 0 1.25.56 1.25 1.25v4.984c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-4.984c0-.69.56-1.25 1.25-1.25m29.75 0c.69 0 1.25.56 1.25 1.25v4.984c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-4.984c0-.69.56-1.25 1.25-1.25M19.042 14.25c.69 0 1.25.56 1.25 1.25v16.998c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25V15.501c0-.69.56-1.25 1.25-1.25m11.166 1.25c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v16.998c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25zM14.084 10.25c.69 0 1.25.56 1.25 1.25v25c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-25c0-.69.56-1.25 1.25-1.25m21.083 1.25c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v25c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25z"
        })
    });
const $f8ebd9a1f7109682$var$ForwardRef = /*#__PURE__*/ (0, $k29Ve$forwardRef)($f8ebd9a1f7109682$var$SvgComponent);
var $f8ebd9a1f7109682$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f8ebd9a1f7109682$var$ForwardRef);


export {$f8ebd9a1f7109682$export$2e2bcd8739ae039 as default};
