// @react-spectrum/s2/illustrations/linear/internal/S2_lin_heart_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $khAZe$jsx} from "react/jsx-runtime";
import {forwardRef as $khAZe$forwardRef} from "react";





const $a229461c3b287d25$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $khAZe$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $khAZe$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 25.513c3.294-6.587 10.308-10.992 18.287-10.471 10.742.7 18.397 10.428 17.664 20.949-.716 10.284-6.792 20.156-13.454 28.094-6.704 7.989-14.237 14.297-18.305 17.471-2.468 1.926-5.931 1.925-8.398 0-4.067-3.176-11.597-9.484-18.298-17.472-6.658-7.938-12.731-17.81-13.448-28.093-.732-10.52 6.921-20.247 17.663-20.95 7.98-.52 14.995 3.885 18.289 10.472m18.027-6.48c-8.752-.571-16.02 6.345-16.02 14.951q0 .194-.037.384l-.01.054c-.19.935-1.014 1.605-1.968 1.601-.953-.004-1.772-.68-1.955-1.616l-.008-.042q-.036-.188-.037-.381c0-8.607-7.268-15.523-16.02-14.95-8.329.544-14.526 8.161-13.933 16.68.627 9.002 6.032 18.062 12.522 25.8 6.448 7.687 13.737 13.799 17.694 16.888 1.02.797 2.455.797 3.476 0 3.959-3.089 11.251-9.2 17.702-16.888 6.493-7.738 11.9-16.798 12.528-25.801.593-8.518-5.606-16.136-13.934-16.68"
        })
    });
const $a229461c3b287d25$var$ForwardRef = /*#__PURE__*/ (0, $khAZe$forwardRef)($a229461c3b287d25$var$SvgComponent);
var $a229461c3b287d25$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a229461c3b287d25$var$ForwardRef);


export {$a229461c3b287d25$export$2e2bcd8739ae039 as default};
