// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lightbulbRays_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3YAEF$reactjsxruntime = require("react/jsx-runtime");
var $3YAEF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $801622600ea0f92e$export$2e2bcd8739ae039);




const $801622600ea0f92e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3YAEF$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3YAEF$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80.04 19.93c2.21 0 4 1.79 4 4v6.02c0 2.209-1.79 4-4 4s-4-1.791-4-4v-6.02c0-2.21 1.79-4 4-4m3.23 33.57c-2.21 0-4 1.79-4 4s1.79 4 4 4c5.702 0 9.9 4.293 9.9 9.08 0 2.209 1.79 4 4 4s4-1.791 4-4c0-9.654-8.243-17.08-17.9-17.08M63 118v-9.767c0-2.176-1.325-4.505-3.741-6.394C51.19 95.528 46 85.619 46 74.499 46 55.51 61.16 40 80 40s34 15.51 34 34.5c0 11.12-5.19 21.029-13.259 27.34-2.415 1.889-3.741 4.218-3.741 6.394V128c0 4.418-3.582 8-8 8H71c-4.418 0-8-3.582-8-8zm26-4v-5.767c0-5.459 3.213-9.88 6.812-12.696C102.002 90.697 106 83.084 106 74.5 106 59.802 94.297 48 80 48S54 59.802 54 74.5c0 8.584 3.998 16.196 10.188 21.037C67.787 98.352 71 102.774 71 108.233V114zm-18 8h18v6H71zM26 73.98c0-2.21 1.79-4 4-4h6.01c2.21 0 4 1.79 4 4s-1.79 4-4 4H30c-2.21 0-4-1.791-4-4m98.07-4c-2.209 0-4 1.79-4 4s1.791 4 4 4h6.01c2.209 0 4-1.791 4-4s-1.791-4-4-4zm-5.812-28.562c1.562-1.562 1.562-4.094 0-5.657s-4.094-1.562-5.656 0l-4.261 4.26c-1.562 1.563-1.562 4.095 0 5.657s4.095 1.562 5.657 0zm-76.436-5.657c1.562-1.562 4.095-1.562 5.657 0l4.26 4.26c1.562 1.563 1.562 4.095 0 5.657s-4.095 1.562-5.657 0l-4.26-4.26c-1.562-1.562-1.562-4.094 0-5.657"
        })
    });
const $801622600ea0f92e$var$ForwardRef = /*#__PURE__*/ (0, $3YAEF$react.forwardRef)($801622600ea0f92e$var$SvgComponent);
var $801622600ea0f92e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($801622600ea0f92e$var$ForwardRef);
