// @react-spectrum/s2/illustrations/linear/internal/S2_lin_plugin_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $gUxj3$jsx} from "react/jsx-runtime";
import {forwardRef as $gUxj3$forwardRef} from "react";





const $3f1065173becaada$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gUxj3$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gUxj3$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20.247 23.7c0-2.656 1.972-5.2 4.861-5.2h14.308c2.89 0 4.862 2.544 4.862 5.2v6h7.446v-6c0-2.656 1.972-5.2 4.861-5.2h14.308c2.89 0 4.862 2.544 4.862 5.2v6h.861c6.05 0 10.585 5.41 10.585 11.6v27.2c0 6.19-4.535 11.6-10.585 11.6h-57.23C13.335 80.1 8.8 74.69 8.8 68.5V41.3c0-6.19 4.534-11.6 10.584-11.6h.862zm4.861-1.2c-.271 0-.861.321-.861 1.2v8c0 1.105-.895 2-2 2h-2.862c-3.432 0-6.584 3.186-6.584 7.6v27.2c0 4.413 3.152 7.6 6.584 7.6h57.231c3.432 0 6.585-3.186 6.585-7.6V41.3c0-4.413-3.153-7.6-6.585-7.6h-2.861c-1.105 0-2-.895-2-2v-8c0-.879-.59-1.2-.862-1.2H56.585c-.271 0-.861.321-.861 1.2v8c0 1.105-.896 2-2 2H42.278c-1.105 0-2-.895-2-2v-8c0-.879-.59-1.2-.862-1.2z"
        })
    });
const $3f1065173becaada$var$ForwardRef = /*#__PURE__*/ (0, $gUxj3$forwardRef)($3f1065173becaada$var$SvgComponent);
var $3f1065173becaada$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($3f1065173becaada$var$ForwardRef);


export {$3f1065173becaada$export$2e2bcd8739ae039 as default};
