// @react-spectrum/s2/illustrations/linear/internal/S2_lin_playTriangle_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1iFjX$reactjsxruntime = require("react/jsx-runtime");
var $1iFjX$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2051d60e32a3a233$export$2e2bcd8739ae039);




const $2051d60e32a3a233$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1iFjX$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1iFjX$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M10.775 13.31c.004-3.297 3.519-5.49 6.418-3.76l17.943 10.708c2.823 1.685 2.818 5.856-.01 7.534L17.16 38.456c-2.903 1.723-6.413-.479-6.409-3.775zm5.137-1.613c-1.12-.668-2.636.124-2.637 1.615l-.025 21.372c-.002 1.492 1.512 2.288 2.633 1.622l17.968-10.664c1.197-.71 1.2-2.523.003-3.237z"
        })
    });
const $2051d60e32a3a233$var$ForwardRef = /*#__PURE__*/ (0, $1iFjX$react.forwardRef)($2051d60e32a3a233$var$SvgComponent);
var $2051d60e32a3a233$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2051d60e32a3a233$var$ForwardRef);
