// @react-spectrum/s2/illustrations/linear/internal/S2_lin_megaphonePromote_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $9zFZ6$reactjsxruntime = require("react/jsx-runtime");
var $9zFZ6$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4f053aac26d1d5dd$export$2e2bcd8739ae039);




const $4f053aac26d1d5dd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9zFZ6$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9zFZ6$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M40.312 12.107c0-2.89-2.79-4.964-5.558-4.123l-13.033 3.95H12.46c-4.232 0-7.667 3.434-7.667 7.666s3.435 7.667 7.667 7.667h.41l.312.715 1.462 3.344c1.097 2.507 2.297 5.24 2.689 6.114.625 1.39 1.692 2.252 2.916 2.596 1.189.334 2.445.157 3.492-.351.977-.475 1.843-1.273 2.318-2.337.487-1.09.523-2.37-.03-3.679l-.005-.012-2.553-5.863 11.28 3.429c2.765.84 5.561-1.228 5.561-4.123zm-19.627 2.326v10.334H12.46c-2.852 0-5.167-2.315-5.167-5.167s2.315-5.167 5.167-5.167zm2.5 10.661 12.292 3.737c1.162.352 2.335-.516 2.335-1.731V12.107c0-1.215-1.172-2.083-2.332-1.731l-12.295 3.726zM15.6 27.27l1.335 3.053c1.1 2.514 2.292 5.233 2.678 6.091.327.728.815 1.075 1.311 1.215.53.149 1.159.082 1.725-.193.525-.255.925-.654 1.128-1.108.19-.427.245-.982-.048-1.68l-3.212-7.378z"
        })
    });
const $4f053aac26d1d5dd$var$ForwardRef = /*#__PURE__*/ (0, $9zFZ6$react.forwardRef)($4f053aac26d1d5dd$var$SvgComponent);
var $4f053aac26d1d5dd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4f053aac26d1d5dd$var$ForwardRef);
