// @react-spectrum/s2/illustrations/linear/internal/S2_lin_explosion_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7fRPx$jsx} from "react/jsx-runtime";
import {forwardRef as $7fRPx$forwardRef} from "react";





const $301e8f0a5ec86691$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7fRPx$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7fRPx$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M32.513 26.206c-3.146-5.824 4.07-11.705 9.139-7.446l35.67 29.971 20.785-22.948c4.14-4.572 11.656-.67 10.3 5.346l-5.967 26.478h45.746c6.925 0 8.343 9.788 1.703 11.753l-39.652 11.737 12.026 14.157c3.962 4.664-.782 11.571-6.558 9.546l-18.324-6.422v39.044c0 6.661-9.234 8.361-11.606 2.136l-14.589-38.28c-9.388 5.399-22.184 12.657-30.308 17.255-5.635 3.19-11.534-3.526-7.842-8.711 4.73-6.645 11.689-16.462 16.837-23.915l-27.677-7.103c-6.065-1.557-5.99-10.197.1-11.649l28.654-6.827zm12.027 5.43 18.402 34.059-30.928 7.369 30.878 7.925-3.178 4.703c-3.758 5.562-9.749 14.087-14.965 21.453 8.814-5 19.428-11.042 26.498-15.139l4.069-2.357 14.065 36.909V87.096l22.009 7.714-15.204-17.897 38.197-11.306H92.437L98.93 36.79 78.077 59.815z"
        })
    });
const $301e8f0a5ec86691$var$ForwardRef = /*#__PURE__*/ (0, $7fRPx$forwardRef)($301e8f0a5ec86691$var$SvgComponent);
var $301e8f0a5ec86691$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($301e8f0a5ec86691$var$ForwardRef);


export {$301e8f0a5ec86691$export$2e2bcd8739ae039 as default};
