// @react-spectrum/s2/illustrations/linear/internal/S2_lin_brand_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $3Ft1C$jsxs, jsx as $3Ft1C$jsx} from "react/jsx-runtime";
import {forwardRef as $3Ft1C$forwardRef} from "react";





const $47a7dea365035fb7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3Ft1C$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3Ft1C$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M45.74 32.06c-4.41 0-8 3.59-8 8v49.63c0 23.321 18.939 42.251 42.25 42.251 23.321 0 42.25-18.939 42.25-42.25V40.06c0-4.411-3.589-8-8-8h-68.5m-16 8c0-8.829 7.171-16 16-16h68.5c8.829 0 16 7.171 16 16v49.63c0 27.729-22.51 50.251-50.25 50.251-27.728 0-50.25-22.511-50.25-50.25z"
            }),
            /*#__PURE__*/ (0, $3Ft1C$jsx)("path", {
                stroke: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                strokeWidth: 8,
                d: "M88.054 61.844c1.548-2.625 5.344-2.625 6.892 0l13.782 23.374c1.572 2.667-.35 6.032-3.445 6.032H77.718c-3.096 0-5.018-3.365-3.446-6.032z"
            }),
            /*#__PURE__*/ (0, $3Ft1C$jsx)("path", {
                stroke: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                strokeLinecap: "round",
                strokeLinejoin: "round",
                strokeWidth: 8,
                d: "m52 91 14.5-25"
            })
        ]
    });
const $47a7dea365035fb7$var$ForwardRef = /*#__PURE__*/ (0, $3Ft1C$forwardRef)($47a7dea365035fb7$var$SvgComponent);
var $47a7dea365035fb7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($47a7dea365035fb7$var$ForwardRef);


export {$47a7dea365035fb7$export$2e2bcd8739ae039 as default};
