// @react-spectrum/s2/illustrations/linear/internal/S2_lin_download_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6Qymy$reactjsxruntime = require("react/jsx-runtime");
var $6Qymy$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2b9becada138bfd3$export$2e2bcd8739ae039);




const $2b9becada138bfd3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6Qymy$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6Qymy$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8.5 24.173c.69 0 1.25.56 1.25 1.25v8.823c0 1.35 1.083 2.427 2.397 2.427h23.706c1.314 0 2.397-1.077 2.397-2.427v-8.823c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v8.823c0 2.712-2.183 4.927-4.897 4.927H12.147c-2.714 0-4.897-2.215-4.897-4.927v-8.823c0-.69.56-1.25 1.25-1.25M24 8.828c.69 0 1.25.56 1.25 1.25V28.66l4.866-4.866c.488-.488 1.28-.488 1.768 0s.488 1.28 0 1.768l-7 7c-.488.488-1.28.488-1.768 0l-7-7c-.488-.488-.488-1.28 0-1.768s1.28-.488 1.768 0l4.866 4.866v-18.58c0-.691.56-1.25 1.25-1.25"
        })
    });
const $2b9becada138bfd3$var$ForwardRef = /*#__PURE__*/ (0, $6Qymy$react.forwardRef)($2b9becada138bfd3$var$SvgComponent);
var $2b9becada138bfd3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2b9becada138bfd3$var$ForwardRef);
