// @react-spectrum/s2/illustrations/linear/internal/S2_lin_phone_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $2xPnc$jsx} from "react/jsx-runtime";
import {forwardRef as $2xPnc$forwardRef} from "react";





const $8bedad8aaa1c6cfa$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2xPnc$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2xPnc$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M55 40c0-4.418 3.582-8 8-8h34c4.418 0 8 3.582 8 8v80c0 4.418-3.582 8-8 8H63c-4.418 0-8-3.582-8-8zm8-16c-8.837 0-16 7.163-16 16v80c0 8.837 7.163 16 16 16h34c8.837 0 16-7.163 16-16V40c0-8.837-7.163-16-16-16zm22 92.5c0 2.485-2.015 4.5-4.5 4.5s-4.5-2.015-4.5-4.5 2.015-4.5 4.5-4.5 4.5 2.015 4.5 4.5"
        })
    });
const $8bedad8aaa1c6cfa$var$ForwardRef = /*#__PURE__*/ (0, $2xPnc$forwardRef)($8bedad8aaa1c6cfa$var$SvgComponent);
var $8bedad8aaa1c6cfa$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8bedad8aaa1c6cfa$var$ForwardRef);


export {$8bedad8aaa1c6cfa$export$2e2bcd8739ae039 as default};
