// @react-spectrum/s2/illustrations/linear/internal/S2_lin_filmstrip_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bF8pB$reactjsxruntime = require("react/jsx-runtime");
var $bF8pB$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ebb71e0ddd88c564$export$2e2bcd8739ae039);




const $ebb71e0ddd88c564$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bF8pB$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bF8pB$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M76 22.4c3.314 0 6 2.687 6 6v39.2c0 3.314-2.686 6-6 6H20c-3.314 0-6-2.686-6-6V28.4c0-3.313 2.686-6 6-6zm10 6c0-5.522-4.477-10-10-10H20c-5.523 0-10 4.478-10 10v39.2c0 5.523 4.477 10 10 10h56c5.523 0 10-4.477 10-10zm-56.598 3.4c0-.662.538-1.2 1.2-1.2h4.2c.663 0 1.2.538 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.662 0-1.2-.537-1.2-1.2zm-9-1.2c-.664 0-1.2.538-1.2 1.2v3c0 .663.536 1.2 1.2 1.2h4.2c.662 0 1.2-.537 1.2-1.2v-3c0-.662-.538-1.2-1.2-1.2zm9 30.6c0-.662.538-1.2 1.2-1.2h4.2c.663 0 1.2.538 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.662 0-1.2-.537-1.2-1.2zm-9-1.2c-.664 0-1.2.538-1.2 1.2v3c0 .663.536 1.2 1.2 1.2h4.2c.662 0 1.2-.537 1.2-1.2v-3c0-.662-.538-1.2-1.2-1.2zm19.2-28.2c0-.662.537-1.2 1.2-1.2h4.2c.662 0 1.2.538 1.2 1.2v3c0 .663-.538 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2zm1.2 28.2c-.663 0-1.2.538-1.2 1.2v3c0 .663.537 1.2 1.2 1.2h4.2c.662 0 1.2-.537 1.2-1.2v-3c0-.662-.538-1.2-1.2-1.2zM49.8 31.8c0-.662.537-1.2 1.2-1.2h4.2c.663 0 1.2.538 1.2 1.2v3c0 .663-.538 1.2-1.2 1.2H51c-.663 0-1.2-.537-1.2-1.2zM51 60c-.663 0-1.2.538-1.2 1.2v3c0 .663.537 1.2 1.2 1.2h4.2c.663 0 1.2-.537 1.2-1.2v-3c0-.662-.538-1.2-1.2-1.2zM60 31.8c0-.662.537-1.2 1.2-1.2h4.2c.663 0 1.2.538 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2zM61.2 60c-.663 0-1.2.538-1.2 1.2v3c0 .663.537 1.2 1.2 1.2h4.2c.663 0 1.2-.537 1.2-1.2v-3c0-.662-.537-1.2-1.2-1.2zm9.001-28.2c0-.662.537-1.2 1.2-1.2h4.2c.663 0 1.2.538 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2zm1.2 28.2c-.663 0-1.2.538-1.2 1.2v3c0 .663.537 1.2 1.2 1.2h4.2c.663 0 1.2-.537 1.2-1.2v-3c0-.662-.537-1.2-1.2-1.2z"
        })
    });
const $ebb71e0ddd88c564$var$ForwardRef = /*#__PURE__*/ (0, $bF8pB$react.forwardRef)($ebb71e0ddd88c564$var$SvgComponent);
var $ebb71e0ddd88c564$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ebb71e0ddd88c564$var$ForwardRef);
