// @react-spectrum/s2/illustrations/linear/internal/S2_lin_prompt_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $iKqdt$jsx} from "react/jsx-runtime";
import {forwardRef as $iKqdt$forwardRef} from "react";





const $002a1d2c3d9a845f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iKqdt$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $iKqdt$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M7.75 17c0-2.9 2.35-5.25 5.25-5.25h22c2.9 0 5.25 2.35 5.25 5.25v14c0 2.9-2.35 5.25-5.25 5.25H13c-2.9 0-5.25-2.35-5.25-5.25zM13 14.25c-1.519 0-2.75 1.231-2.75 2.75v14c0 1.519 1.231 2.75 2.75 2.75h22c1.519 0 2.75-1.231 2.75-2.75V17c0-1.519-1.231-2.75-2.75-2.75zM20.75 30c0-.69.56-1.25 1.25-1.25h12c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H22c-.69 0-1.25-.56-1.25-1.25m-7.712.267c-.432.539-.346 1.326.192 1.758.539.432 1.325.346 1.758-.193l5.7-7.101c.369-.46.366-1.115-.006-1.572l-5.7-6.999c-.436-.535-1.223-.615-1.759-.18-.535.437-.616 1.224-.18 1.76l5.062 6.214z"
        })
    });
const $002a1d2c3d9a845f$var$ForwardRef = /*#__PURE__*/ (0, $iKqdt$forwardRef)($002a1d2c3d9a845f$var$SvgComponent);
var $002a1d2c3d9a845f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($002a1d2c3d9a845f$var$ForwardRef);


export {$002a1d2c3d9a845f$export$2e2bcd8739ae039 as default};
