// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowLeft_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $7xpB4$reactjsxruntime = require("react/jsx-runtime");
var $7xpB4$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f2f9d3b4ce7bd973$export$2e2bcd8739ae039);




const $f2f9d3b4ce7bd973$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7xpB4$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7xpB4$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M41.5 25.25c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H10.518l4.866-4.866c.488-.488.488-1.28 0-1.768s-1.28-.488-1.768 0l-7 7c-.488.488-.488 1.28 0 1.768l7 7c.488.488 1.28.488 1.768 0s.488-1.28 0-1.768l-4.866-4.866z"
        })
    });
const $f2f9d3b4ce7bd973$var$ForwardRef = /*#__PURE__*/ (0, $7xpB4$react.forwardRef)($f2f9d3b4ce7bd973$var$SvgComponent);
var $f2f9d3b4ce7bd973$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f2f9d3b4ce7bd973$var$ForwardRef);
