// @react-spectrum/s2/illustrations/linear/internal/S2_lin_effects_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1iouu$jsx} from "react/jsx-runtime";
import {forwardRef as $1iouu$forwardRef} from "react";





const $650c27ae00117d82$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1iouu$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1iouu$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M44.126 18.956c-2.279 1.736-3.867 4.464-4.779 6.97l-3.332 10.857h17.183c.727 0 1.397.395 1.75 1.031l6.775 12.241c4.58-4.577 9.236-9.226 12.72-12.707.782-.78 2.049-.78 2.83.002.78.782.779 2.048-.003 2.829-3.691 3.686-8.694 8.683-13.531 13.517L71.27 67.3c.535.967.185 2.184-.781 2.719-.967.535-2.184.185-2.719-.781l-6.975-12.602-1.97 1.97c-3.06 3.06-5.813 5.812-7.807 7.808-.996.998-1.803 1.806-2.363 2.369l-.645.65q-.112.112-.165.167l-.02.021c-.687.751-1.845.872-2.675.249-.884-.663-1.063-1.916-.4-2.8.05-.068.097-.12.103-.127h.001l.034-.038.035-.038.023-.023.038-.04.182-.184.654-.66 2.368-2.373c1.995-1.997 4.748-4.75 7.809-7.81l2.783-2.782-6.761-12.213H34.787L22.055 82.266c-.324 1.056-1.443 1.65-2.499 1.325s-1.649-1.442-1.325-2.498l12.372-40.31H22.48c-1.104 0-2-.895-2-2s.896-2 2-2h9.35l3.708-12.079.03-.092c1.06-2.935 3.014-6.46 6.135-8.838 3.23-2.46 7.56-3.564 12.909-1.775 1.047.35 1.612 1.483 1.262 2.531s-1.483 1.613-2.531 1.262c-4.084-1.366-7.033-.498-9.217 1.164m3.687 50.678.004-.005c-.026.03-.034.036-.004.005"
        })
    });
const $650c27ae00117d82$var$ForwardRef = /*#__PURE__*/ (0, $1iouu$forwardRef)($650c27ae00117d82$var$SvgComponent);
var $650c27ae00117d82$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($650c27ae00117d82$var$ForwardRef);


export {$650c27ae00117d82$export$2e2bcd8739ae039 as default};
