// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noComments_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5C0Ml$jsx} from "react/jsx-runtime";
import {forwardRef as $5C0Ml$forwardRef} from "react";





const $14b3d36b4554751e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5C0Ml$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5C0Ml$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M45.25 25.367c0-8.146 6.603-14.75 14.75-14.75s14.75 6.604 14.75 14.75-6.603 14.75-14.75 14.75-14.75-6.603-14.75-14.75m21.043 6.293c.781-.782.781-2.054 0-2.836h.02l-3.457-3.457 3.457-3.457c.781-.781.781-2.054 0-2.836-.782-.781-2.054-.781-2.836 0l-3.457 3.457-3.457-3.457c-.782-.781-2.054-.781-2.836 0-.781.782-.781 2.055 0 2.836l3.457 3.457-3.457 3.457c-.781.782-.781 2.054 0 2.836.391.39.902.591 1.413.591s1.022-.2 1.413-.591l3.457-3.457 3.457 3.457c.39.39.902.591 1.413.591s1.022-.2 1.413-.591M10 26.61c0-4.583 3.783-8.227 8.36-8.227h22.265c1.105 0 2 .895 2 2s-.895 2-2 2H18.36c-2.448 0-4.36 1.932-4.36 4.228v22.102c0 2.296 1.912 4.228 4.36 4.228h5.696c1.101 0 1.995.89 2 1.992.013 2.951-.011 7.168-.035 10.37q0 .02.002.025c.003.004.019.022.053.037.037.017.07.02.09.017q.015-.001.026-.007.01-.002.044-.029l14.053-11.93c.361-.306.82-.475 1.294-.475H58.64c2.448 0 4.36-1.932 4.36-4.228v-4.018c0-1.104.895-2 2-2s2 .896 2 2v4.018c0 4.584-3.783 8.228-8.36 8.228H42.318L28.824 68.396c-2.616 2.22-6.83.462-6.803-3.122.02-2.53.038-5.68.038-8.333H18.36c-4.577 0-8.36-3.644-8.36-8.228zm60 14.773c0-1.105.895-2 2-2h6.36c4.194 0 7.64 3.362 7.64 7.565V66.7c0 4.204-3.446 7.566-7.64 7.566h-3.054c.002 2.324.017 5.038.034 7.232.024 3.367-3.93 5.072-6.412 2.948l-11.9-10.18H42.64c-3.645 0-6.715-2.533-7.466-5.953-.236-1.08.446-2.146 1.525-2.383s2.145.446 2.382 1.525c.35 1.59 1.797 2.81 3.56 2.81h15.125c.477 0 .938.171 1.3.481l12.272 10.498c-.02-2.824-.04-6.434-.03-8.987.005-1.1.9-1.991 2-1.991h5.052c2.035 0 3.64-1.622 3.64-3.566V46.95c0-1.945-1.605-3.566-3.64-3.566H72c-1.105 0-2-.896-2-2"
        })
    });
const $14b3d36b4554751e$var$ForwardRef = /*#__PURE__*/ (0, $5C0Ml$forwardRef)($14b3d36b4554751e$var$SvgComponent);
var $14b3d36b4554751e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($14b3d36b4554751e$var$ForwardRef);


export {$14b3d36b4554751e$export$2e2bcd8739ae039 as default};
