// @react-spectrum/s2/illustrations/linear/internal/S2_lin_filters_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8P2oq$jsx} from "react/jsx-runtime";
import {forwardRef as $8P2oq$forwardRef} from "react";





const $799a949c33f8947d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8P2oq$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8P2oq$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28.811 26.371c1.963-1.43 3.33-3.63 3.655-6.156-.757-.205-1.554-.315-2.376-.315-1.523 0-2.958.377-4.218 1.041 1.43 1.485 2.465 3.35 2.939 5.43m-2.313 1.217c-.941.332-1.953.512-3.008.512-.822 0-1.619-.11-2.376-.315.27-2.096 1.256-3.968 2.706-5.36 1.404 1.349 2.374 3.147 2.678 5.163m-4.73-6.647c-1.517 1.577-2.591 3.583-3.02 5.818-2.218-1.367-3.801-3.663-4.206-6.347.941-.331 1.953-.512 3.008-.512 1.523 0 2.958.377 4.218 1.041m2.052-1.692c1.806-1.17 3.959-1.849 6.27-1.849.802 0 1.585.082 2.34.238C31.752 13.31 28.008 10 23.49 10c-4.574 0-8.356 3.394-8.964 7.8.963-.26 1.977-.4 3.024-.4 2.311 0 4.464.68 6.27 1.849m-11.878-.398C8.398 20.823 6 24.607 6 28.951c0 6.378 5.171 11.55 11.55 11.55 2.311 0 4.464-.68 6.27-1.849 1.806 1.17 3.959 1.848 6.27 1.848 6.379 0 11.55-5.17 11.55-11.55 0-4.612-2.704-8.593-6.613-10.444C34.743 12.38 29.687 7.5 23.49 7.5c-6.312 0-11.442 5.064-11.548 11.351m.287 2.778C9.969 23.274 8.5 25.941 8.5 28.95c0 4.999 4.052 9.05 9.05 9.05 1.523 0 2.958-.376 4.218-1.04-1.882-1.955-3.081-4.571-3.215-7.466-3.158-1.495-5.53-4.38-6.324-7.865M23.82 35.477c-1.393-1.339-2.358-3.118-2.67-5.114.755.155 1.538.237 2.34.237 1.047 0 2.06-.14 3.024-.4-.284 2.062-1.264 3.902-2.694 5.277m2.052 1.482c1.956-2.032 3.175-4.779 3.226-7.81 2.933-1.632 5.08-4.504 5.735-7.908 2.584 1.593 4.307 4.45 4.307 7.71 0 4.998-4.052 9.05-9.05 9.05-1.523 0-2.958-.377-4.218-1.042"
        })
    });
const $799a949c33f8947d$var$ForwardRef = /*#__PURE__*/ (0, $8P2oq$forwardRef)($799a949c33f8947d$var$SvgComponent);
var $799a949c33f8947d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($799a949c33f8947d$var$ForwardRef);


export {$799a949c33f8947d$export$2e2bcd8739ae039 as default};
