// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cloudStateError_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1xSUP$reactjsxruntime = require("react/jsx-runtime");
var $1xSUP$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b322efe5aafb724f$export$2e2bcd8739ae039);




const $b322efe5aafb724f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1xSUP$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1xSUP$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M84.97 28.9c-13.458 0-24.924 8.311-30.118 20.107-1.832-.442-3.74-.677-5.702-.677-14.112 0-25.25 11.897-25.25 26.19q.001 1.24.115 2.429C16.285 80.619 11 88.683 11 97.94c0 12.318 9.397 22.628 21.476 23.078q.253.032.514.032h54.63c2.21 0 4-1.79 4-4s-1.79-4-4-4H33.392q-.165-.016-.334-.019C25.431 112.901 19 106.352 19 97.941c0-7.062 4.571-12.85 10.561-14.565 2.027-.58 3.256-2.635 2.808-4.695-.299-1.373-.469-2.756-.469-4.16 0-10.207 7.882-18.19 17.25-18.19 2.305 0 4.496.48 6.514 1.351 1.044.451 2.23.437 3.263-.04s1.813-1.37 2.147-2.458C64.362 44.47 73.903 36.9 84.97 36.9c13.71 0 25.12 11.655 25.12 26.41l-.002.06-.004.075-.001.017c-.004.098-.013.287-.013.488 0 2.118 1.65 3.868 3.763 3.993 4.227.251 8.129 1.786 11.365 4.263 1.754 1.343 4.265 1.01 5.608-.744s1.01-4.265-.744-5.608c-3.505-2.683-7.616-4.585-12.088-5.44C116.562 42.897 102.466 28.9 84.97 28.9m11.02 73.69c0-13.53 10.97-24.5 24.5-24.5s24.5 10.97 24.5 24.5-10.97 24.5-24.5 24.5-24.5-10.97-24.5-24.5m24.5-15.58c-2.21 0-4 1.79-4 4v12.78c0 2.21 1.79 4 4 4s4-1.79 4-4V91.01c0-2.21-1.79-4-4-4m3.83 28.76c.11-.25.19-.52.24-.78.05-.27.08-.54.08-.81 0-1.1-.44-2.16-1.21-2.94-.97-.96-2.4-1.4-3.75-1.13-.26.05-.53.13-.78.24-.25.1-.49.23-.71.38q-.345.225-.63.51c-.78.78-1.22 1.85-1.22 2.94 0 .27.03.54.08.81.05.26.13.53.24.78.1.25.23.49.38.71.15.23.32.44.52.63.19.2.4.37.63.52.22.15.46.28.71.38.25.11.52.19.78.24.27.05.54.08.81.08 1.09 0 2.16-.44 2.94-1.22q.285-.285.51-.63c.15-.22.28-.46.38-.71"
        })
    });
const $b322efe5aafb724f$var$ForwardRef = /*#__PURE__*/ (0, $1xSUP$react.forwardRef)($b322efe5aafb724f$var$SvgComponent);
var $b322efe5aafb724f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b322efe5aafb724f$var$ForwardRef);
