// @react-spectrum/s2/illustrations/linear/internal/S2_lin_clipboard_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $Uevjh$reactjsxruntime = require("react/jsx-runtime");
var $Uevjh$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b39bf0860d1a1dfe$export$2e2bcd8739ae039);




const $b39bf0860d1a1dfe$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $Uevjh$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $Uevjh$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M79.024 20c-8.837-.01-16.01 7.144-16.02 15.98l-4-.004q-.222 0-.441.024H46.409C37.306 36 30 43.449 30 52.545v66.91C30 128.551 37.306 136 46.41 136h66.181c9.104 0 16.409-7.449 16.409-16.545v-66.91C129 43.45 121.695 36 112.591 36H95.004c0-8.828-7.15-15.99-15.98-16m23.971 24-.015 12.029c-.001 1.06-.423 2.077-1.175 2.827-.751.749-1.768 1.17-2.83 1.168l-40-.048c-2.208-.003-3.997-1.796-3.995-4.005L54.996 44h-8.586C41.806 44 38 47.785 38 52.545v66.91c0 4.76 3.806 8.545 8.41 8.545h66.181c4.603 0 8.409-3.785 8.409-8.545v-66.91c0-4.76-3.806-8.545-8.409-8.545zm-40 0-.01 7.98 32 .04.01-8-4-.006c-2.21-.002-3.998-1.795-3.995-4.004l.004-4c.006-4.419-3.572-8.005-7.99-8.01s-8.004 3.572-8.01 7.99l-.004 4c-.003 2.21-1.796 3.998-4.005 3.996l-.938-.002V44z"
        })
    });
const $b39bf0860d1a1dfe$var$ForwardRef = /*#__PURE__*/ (0, $Uevjh$react.forwardRef)($b39bf0860d1a1dfe$var$SvgComponent);
var $b39bf0860d1a1dfe$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b39bf0860d1a1dfe$var$ForwardRef);
