// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noBrands_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $jQjGv$reactjsxruntime = require("react/jsx-runtime");
var $jQjGv$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4a40297ea6e0deab$export$2e2bcd8739ae039);




const $4a40297ea6e0deab$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jQjGv$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jQjGv$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M11.75 12.484c0-1.246 1.014-2.26 2.26-2.26h20.033c1.245 0 2.26 1.014 2.26 2.26V26.5c0 .69.559 1.25 1.25 1.25.69 0 1.25-.56 1.25-1.25V12.484c0-2.626-2.134-4.76-4.76-4.76H14.01c-2.627 0-4.76 2.134-4.76 4.76V27c0 8.157 6.623 14.776 14.776 14.776q1.032 0 2.03-.139c.683-.094 1.162-.725 1.067-1.409-.094-.683-.725-1.161-1.408-1.067q-.829.115-1.689.115c-6.773 0-12.276-5.5-12.276-12.276zm17.23 5.959c-.931-1.615-3.26-1.615-4.192 0l-3.673 6.367c-.93 1.612.23 3.63 2.095 3.63h7.352c1.865 0 3.025-2.018 2.095-3.63zm1.442 7.496-3.538-6.127-3.534 6.127zm-7.142.12-.07-.041-1.012-.584 1.012.584zm-2.778-7.024c.598.345.802 1.11.457 1.707l-4.085 7.075c-.346.598-1.11.802-1.708.457s-.803-1.11-.457-1.707l4.085-7.075c.345-.598 1.11-.803 1.708-.457M35.375 43.75c4.073 0 7.375-3.302 7.375-7.375S39.448 29 35.375 29 28 32.302 28 36.375s3.302 7.375 7.375 7.375M36.75 33c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v2.25H32c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h2.25V40c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-2.25H39c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.25z"
        })
    });
const $4a40297ea6e0deab$var$ForwardRef = /*#__PURE__*/ (0, $jQjGv$react.forwardRef)($4a40297ea6e0deab$var$SvgComponent);
var $4a40297ea6e0deab$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4a40297ea6e0deab$var$ForwardRef);
