// @react-spectrum/s2/illustrations/linear/internal/S2_lin_plugin_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $eFRet$jsx} from "react/jsx-runtime";
import {forwardRef as $eFRet$forwardRef} from "react";





const $95a0044c40ce8ba9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eFRet$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $eFRet$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M9.873 11.85c0-1.439 1.073-2.85 2.681-2.85h7.154c1.608 0 2.68 1.411 2.68 2.85v2.75h3.224v-2.75c0-1.439 1.072-2.85 2.68-2.85h7.154c1.609 0 2.681 1.411 2.681 2.85v2.75h.181c3.189 0 5.542 2.844 5.542 6.05v13.6c0 3.206-2.353 6.05-5.542 6.05H9.693c-3.189 0-5.543-2.844-5.543-6.05v-13.6c0-3.206 2.354-6.05 5.543-6.05h.18zm2.601-.298c-.05.056-.1.158-.1.298v4c0 .69-.56 1.25-1.25 1.25H9.692c-1.553 0-3.043 1.454-3.043 3.55v13.6c0 2.096 1.49 3.55 3.043 3.55h28.615c1.553 0 3.042-1.454 3.042-3.55v-13.6c0-2.096-1.49-3.55-3.042-3.55h-1.43c-.691 0-1.25-.56-1.25-1.25v-4c0-.14-.051-.242-.102-.298-.047-.053-.076-.052-.08-.052h-7.153c-.003 0-.033 0-.08.052-.05.056-.101.158-.101.298v4c0 .69-.56 1.25-1.25 1.25h-5.723c-.69 0-1.25-.56-1.25-1.25v-4c0-.14-.05-.242-.101-.298-.047-.053-.077-.052-.08-.052h-7.154c-.003 0-.033 0-.08.052"
        })
    });
const $95a0044c40ce8ba9$var$ForwardRef = /*#__PURE__*/ (0, $eFRet$forwardRef)($95a0044c40ce8ba9$var$SvgComponent);
var $95a0044c40ce8ba9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($95a0044c40ce8ba9$var$ForwardRef);


export {$95a0044c40ce8ba9$export$2e2bcd8739ae039 as default};
