// @react-spectrum/s2/illustrations/linear/internal/S2_lin_ruler_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $fa5DA$jsx} from "react/jsx-runtime";
import {forwardRef as $fa5DA$forwardRef} from "react";





const $7bc87af328031da6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fa5DA$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fa5DA$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28.907 120.099c-6.248-6.249-6.248-16.379 0-22.628l68.564-68.564c6.249-6.248 16.379-6.248 22.628 0l10.994 10.995c6.249 6.248 6.249 16.379 0 22.627l-68.564 68.564c-6.248 6.249-16.379 6.249-22.627 0zm5.657-16.971c-3.124 3.124-3.124 8.189 0 11.314l10.995 10.994c3.124 3.125 8.19 3.125 11.313 0l68.564-68.564c3.125-3.124 3.125-8.19 0-11.313l-10.994-10.995c-3.125-3.124-8.19-3.124-11.314 0l-5.788 5.788 7.692 7.692c1.562 1.562 1.562 4.095 0 5.657s-4.095 1.562-5.657 0l-7.692-7.692-11.454 11.454 12.344 12.344c1.562 1.562 1.562 4.095 0 5.657s-4.095 1.562-5.657 0L74.572 63.12 63.12 74.573l7.692 7.691c1.562 1.562 1.562 4.095 0 5.657s-4.095 1.562-5.657 0l-7.692-7.691L46.01 91.683l12.343 12.344c1.562 1.562 1.562 4.095 0 5.657s-4.094 1.562-5.656 0L40.352 97.34z"
        })
    });
const $7bc87af328031da6$var$ForwardRef = /*#__PURE__*/ (0, $fa5DA$forwardRef)($7bc87af328031da6$var$SvgComponent);
var $7bc87af328031da6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7bc87af328031da6$var$ForwardRef);


export {$7bc87af328031da6$export$2e2bcd8739ae039 as default};
