// @react-spectrum/s2/illustrations/linear/internal/S2_lin_briefcase_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6aM8V$reactjsxruntime = require("react/jsx-runtime");
var $6aM8V$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d83ca99db7b531c5$export$2e2bcd8739ae039);




const $d83ca99db7b531c5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6aM8V$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6aM8V$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M38 20c-3.315 0-6 2.685-6 6v2h32v-2c0-3.315-2.685-6-6-6zm-10 6v2h-6c-5.523 0-10 4.477-10 10v32c0 5.523 4.477 10 10 10h52c5.523 0 10-4.477 10-10V38c0-5.523-4.477-10-10-10h-6v-2c0-5.525-4.475-10-10-10H38c-5.525 0-10 4.475-10 10m-6 6c-3.314 0-6 2.686-6 6v14h12v-2c0-1.105.895-2 2-2s2 .895 2 2v2h32v-2c0-1.105.895-2 2-2s2 .895 2 2v2h12V38c0-3.314-2.686-6-6-6zm42 24v2c0 1.105.895 2 2 2s2-.895 2-2v-2h12v14c0 3.314-2.686 6-6 6H22c-3.314 0-6-2.686-6-6V56h12v2c0 1.105.895 2 2 2s2-.895 2-2v-2z"
        })
    });
const $d83ca99db7b531c5$var$ForwardRef = /*#__PURE__*/ (0, $6aM8V$react.forwardRef)($d83ca99db7b531c5$var$SvgComponent);
var $d83ca99db7b531c5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d83ca99db7b531c5$var$ForwardRef);
