// @react-spectrum/s2/illustrations/linear/internal/S2_lin_information_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1cYNB$jsx} from "react/jsx-runtime";
import {forwardRef as $1cYNB$forwardRef} from "react";





const $b2882be27c03166d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1cYNB$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1cYNB$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8.25 23.74c0-8.698 7.052-15.75 15.75-15.75s15.75 7.052 15.75 15.75S32.699 39.49 24 39.49 8.25 32.44 8.25 23.74M24 5.49c-10.08 0-18.25 8.171-18.25 18.25S13.92 41.99 24 41.99s18.25-8.17 18.25-18.25S34.08 5.49 24 5.49m1.25 11.25c0 .967-.784 1.75-1.75 1.75s-1.75-.783-1.75-1.75c0-.966.784-1.75 1.75-1.75s1.75.784 1.75 1.75M22 20.49c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h.75v5.5H22c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h4c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-.75v-6.75c0-.69-.56-1.25-1.25-1.25z"
        })
    });
const $b2882be27c03166d$var$ForwardRef = /*#__PURE__*/ (0, $1cYNB$forwardRef)($b2882be27c03166d$var$SvgComponent);
var $b2882be27c03166d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b2882be27c03166d$var$ForwardRef);


export {$b2882be27c03166d$export$2e2bcd8739ae039 as default};
