// @react-spectrum/s2/illustrations/linear/internal/S2_lin_location_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $aa0Kd$reactjsxruntime = require("react/jsx-runtime");
var $aa0Kd$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b67d235b008f59c4$export$2e2bcd8739ae039);




const $b67d235b008f59c4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aa0Kd$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $aa0Kd$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M39.25 20.162c0-8.464-7.41-15.165-16.214-14.643C15.744 5.95 9.674 11.43 8.848 18.49c-.262 2.238.012 4.386.7 6.35 1.09 3.372 3.189 6.633 5.37 9.393 2.194 2.775 4.526 5.11 6.143 6.62 1.63 1.521 4.2 1.511 5.824-.006 2.981-2.783 8.545-8.51 11.024-14.673.859-1.834 1.341-3.87 1.341-6.013M23.184 8.014c7.467-.442 13.566 5.222 13.566 12.148 0 1.775-.401 3.46-1.118 4.982l-.03.068c-2.251 5.623-7.462 11.044-10.423 13.809-.67.625-1.747.625-2.412.005-1.561-1.458-3.797-3.698-5.887-6.343-2.1-2.655-4-5.652-4.959-8.624l-.01-.03c-.57-1.623-.797-3.395-.58-5.247.677-5.78 5.693-10.403 11.853-10.768M20.25 18.74c0-2.07 1.679-3.75 3.75-3.75s3.75 1.68 3.75 3.75-1.679 3.75-3.75 3.75-3.75-1.679-3.75-3.75M24 12.49c-3.452 0-6.25 2.798-6.25 6.25s2.798 6.25 6.25 6.25 6.25-2.798 6.25-6.25-2.798-6.25-6.25-6.25"
        })
    });
const $b67d235b008f59c4$var$ForwardRef = /*#__PURE__*/ (0, $aa0Kd$react.forwardRef)($b67d235b008f59c4$var$SvgComponent);
var $b67d235b008f59c4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b67d235b008f59c4$var$ForwardRef);
