// @react-spectrum/s2/illustrations/linear/internal/S2_lin_filter_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5i3lE$jsx} from "react/jsx-runtime";
import {forwardRef as $5i3lE$forwardRef} from "react";





const $ad9b58e36dbf2bcf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5i3lE$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5i3lE$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M19.34 26.333c-3.248-4.316-.235-10.593 5.222-10.593h46.876c5.457 0 8.47 6.277 5.222 10.593q-.031.042-.065.083L54.95 52.222v22.455c0 1.228-.522 2.403-1.443 3.222l-5.347 4.757-1.33-1.494 1.33 1.494c-2.793 2.485-7.11.44-7.11-3.222V52.222L19.404 26.416q-.035-.04-.066-.083m3.168-2.442L44.582 50.21c.302.36.467.815.467 1.285v27.94c0 .094.025.144.045.175.026.039.069.077.126.103s.11.031.146.027c.027-.004.072-.014.136-.071l5.346-4.758 1.052 1.182-1.052-1.181c.062-.055.102-.14.102-.234V51.494c0-.47.166-.925.468-1.285l22.074-26.318c1.268-1.743.007-4.15-2.055-4.15H24.563c-2.061 0-3.322 2.407-2.054 4.15"
        })
    });
const $ad9b58e36dbf2bcf$var$ForwardRef = /*#__PURE__*/ (0, $5i3lE$forwardRef)($ad9b58e36dbf2bcf$var$SvgComponent);
var $ad9b58e36dbf2bcf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ad9b58e36dbf2bcf$var$ForwardRef);


export {$ad9b58e36dbf2bcf$export$2e2bcd8739ae039 as default};
