// @react-spectrum/s2/illustrations/linear/internal/S2_lin_bell_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8il8a$jsx} from "react/jsx-runtime";
import {forwardRef as $8il8a$forwardRef} from "react";





const $7f506d1320240553$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8il8a$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8il8a$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 20.74c-20.427 0-37.014 16.5-37.014 36.886v10.502c-.405 12.341-5.66 23.614-10.95 31.899-5.455 8.539.657 19.713 10.81 19.713H64v3c0 8.837 7.163 16 16 16s16-7.163 16-16v-3h21.155c10.152 0 16.264-11.174 10.81-19.713-5.291-8.285-10.545-19.558-10.95-31.899V57.76l-.001-.064.001-.07c0-20.386-16.587-36.886-37.015-36.886m8 99H72v3c0 4.419 3.582 8 8 8s8-3.581 8-8zM50.987 68.193V57.924l.003-.18-.002-.136-.001-.033C51.013 41.661 63.978 28.74 80 28.74c16.022 0 28.988 12.92 29.015 28.836l-.002.033-.002.135q0 .09.004.181v10.346l.002.047c.453 14.385 6.517 27.108 12.206 36.015 2.053 3.214-.235 7.407-4.068 7.407h-74.31c-3.834 0-6.121-4.193-4.069-7.407 5.69-8.907 11.754-21.63 12.207-36.015z"
        })
    });
const $7f506d1320240553$var$ForwardRef = /*#__PURE__*/ (0, $8il8a$forwardRef)($7f506d1320240553$var$SvgComponent);
var $7f506d1320240553$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7f506d1320240553$var$ForwardRef);


export {$7f506d1320240553$export$2e2bcd8739ae039 as default};
