// @react-spectrum/s2/illustrations/linear/internal/S2_lin_musicNote_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $4M7xX$reactjsxruntime = require("react/jsx-runtime");
var $4M7xX$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5ad996cc237c7b0d$export$2e2bcd8739ae039);




const $5ad996cc237c7b0d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4M7xX$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4M7xX$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M70.963 16.674c2.475-.494 4.782 1.4 4.782 3.922v3.493l-35.965 7.17v-5.087c0-1.908 1.347-3.55 3.218-3.923zM39.78 35.337v34.314q0 .054-.003.106c.062 1.917-.572 3.831-1.63 5.512-1.575 2.507-4.182 4.689-7.399 6.07s-6.594 1.771-9.496 1.189c-2.892-.581-5.515-2.191-6.705-4.963-1.191-2.77-.554-5.781 1.016-8.28 1.576-2.505 4.183-4.687 7.4-6.07 3.217-1.381 6.594-1.77 9.496-1.187 1.178.236 2.311.644 3.321 1.23V26.172c0-3.816 2.694-7.1 6.436-7.846l27.965-5.575c4.95-.987 9.564 2.8 9.564 7.845V61.99q.003.157 0 .313v.276q0 .21-.041.407c-.157 1.543-.736 3.056-1.59 4.416-1.576 2.507-4.183 4.689-7.4 6.07s-6.594 1.771-9.497 1.188c-2.892-.58-5.514-2.19-6.705-4.962s-.553-5.782 1.017-8.28c1.575-2.506 4.183-4.687 7.4-6.07 3.216-1.381 6.594-1.77 9.496-1.187 1.178.236 2.31.643 3.32 1.23V28.167zM18.95 71.415c-1.097 1.744-1.252 3.353-.728 4.572.523 1.218 1.797 2.213 3.817 2.619 2.011.404 4.564.16 7.13-.942 2.567-1.102 4.5-2.787 5.592-4.523 1.097-1.745 1.252-3.354.728-4.572-.523-1.219-1.796-2.214-3.817-2.62-2.011-.403-4.564-.16-7.13.942-2.567 1.103-4.5 2.787-5.592 4.523m39.237-3.296c-.523-1.218-.368-2.827.729-4.572 1.091-1.736 3.025-3.42 5.591-4.523s5.12-1.346 7.13-.942c2.021.406 3.294 1.401 3.818 2.62.523 1.218.368 2.827-.728 4.572-1.092 1.736-3.025 3.42-5.592 4.523s-5.12 1.346-7.13.942c-2.02-.406-3.294-1.401-3.818-2.62"
        })
    });
const $5ad996cc237c7b0d$var$ForwardRef = /*#__PURE__*/ (0, $4M7xX$react.forwardRef)($5ad996cc237c7b0d$var$SvgComponent);
var $5ad996cc237c7b0d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5ad996cc237c7b0d$var$ForwardRef);
