// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noElements_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $aE7LR$reactjsxruntime = require("react/jsx-runtime");
var $aE7LR$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a85e8c8a95474d30$export$2e2bcd8739ae039);




const $a85e8c8a95474d30$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aE7LR$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aE7LR$reactjsxruntime.jsx)("path", {
                stroke: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                strokeLinecap: "round",
                strokeLinejoin: "round",
                strokeWidth: 4,
                d: "M26.342 56v28.146M20.713 84.147H31.97M12.27 56h28.146M12.27 56v5.63M40.416 56v5.63"
            }),
            /*#__PURE__*/ (0, $aE7LR$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M43.125 22c0-3.314-2.686-6-6-6H15c-3.314 0-6 2.686-6 6v12.75c0 .535.07 1.053.201 1.546l6.558-10.132c1.66-2.567 5.418-2.567 7.078-.001l5.82 8.99 3.006-4.545.008-.013c1.67-2.482 5.328-2.49 6.998 0l.002.004 4.126 6.112c.213-.615.328-1.274.328-1.961zm-24.007 6.337-7.402 11.435c.943.618 2.072.978 3.284.978h9.955l1.294-1.956-6.77-10.457v-.001c-.028-.042-.054-.061-.076-.073q-.041-.023-.105-.024-.064.002-.104.024c-.023.012-.049.03-.076.073m21.03 11.597c-.888.519-1.92.816-3.023.816H29.75l5.243-7.926c.089-.128.273-.12.354 0l.005.006zM37.124 12c5.523 0 10 4.477 10 10v12.75c0 5.523-4.477 10-10 10H15c-5.523 0-10-4.477-10-10V22c0-5.523 4.477-10 10-10zm-1.74 8.77c0 1.44 1.166 2.605 2.605 2.605 1.438 0 2.604-1.166 2.604-2.604s-1.166-2.604-2.605-2.604c-1.438 0-2.604 1.166-2.604 2.604"
            }),
            /*#__PURE__*/ (0, $aE7LR$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "m78.052 69.259-7.377-4.212c-.827-.471-1.86.116-1.862 1.057l-.01 8.406c-.001.941 1.031 1.53 1.859 1.061l7.388-4.194c.827-.47.829-1.646.002-2.118M67.9 47.995c-1.63 0-3.2-.87-4.09-2.27-.87-1.36-1.02-3.08-.4-4.61l2.4-5.93-2.9-1.3c-1.42-.63-2.46-1.94-2.8-3.49-.33-1.53.08-3.15 1.09-4.33l12.33-14.34c.9-1.03 2.46-1.94 3.91-1.68 1.53.09 2.97.94 3.82 2.27.87 1.36 1.02 3.08.4 4.61l-2.5 6.12 2.91 1.28c1.43.62 2.49 1.93 2.82 3.5.33 1.55-.08 3.17-1.11 4.36l-12.26 14.15c-.94 1.08-2.22 1.68-3.62 1.68zm9.03-33.96c-.11.06-.26.17-.38.3l-12.32 14.33c-.2.24-.29.57-.22.89.07.31.26.57.52.68l6.43 2.88-3.86 9.51c-.13.32-.1.68.07.95.3.46.94.56 1.31.13l12.26-14.15c.2-.24.29-.58.22-.9-.07-.31-.26-.56-.52-.67l-6.48-2.85 3.97-9.72c.13-.32.11-.68-.07-.95-.16-.25-.45-.42-.72-.42h-.23z"
            }),
            /*#__PURE__*/ (0, $aE7LR$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M83.5 86.746H62c-5.51 0-10-4.49-10-10v-12.75c0-5.51 4.49-10 10-10h21.5c5.51 0 10 4.49 10 10v12.75c0 5.51-4.49 10-10 10M62 57.996c-3.31 0-6 2.69-6 6v12.75c0 3.31 2.69 6 6 6h21.5c3.31 0 6-2.69 6-6v-12.75c0-3.31-2.69-6-6-6z"
            })
        ]
    });
const $a85e8c8a95474d30$var$ForwardRef = /*#__PURE__*/ (0, $aE7LR$react.forwardRef)($a85e8c8a95474d30$var$SvgComponent);
var $a85e8c8a95474d30$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a85e8c8a95474d30$var$ForwardRef);
