// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browserNotCompatible_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5zPO9$jsx} from "react/jsx-runtime";
import {forwardRef as $5zPO9$forwardRef} from "react";





const $eebc29a0adccf87b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5zPO9$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5zPO9$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M74.5 20c3.314 0 6 2.686 6 6v6h-64v-6c0-3.314 2.686-6 6-6zm-58 16h64v34c0 3.314-2.686 6-6 6h-52c-3.314 0-6-2.686-6-6zm68-10c0-5.523-4.477-10-10-10h-52c-5.523 0-10 4.477-10 10v44c0 5.523 4.477 10 10 10h52c5.523 0 10-4.477 10-10zM24 28.5c-1.38 0-2.5-1.12-2.5-2.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5-1.12 2.5-2.5 2.5m11.005 15.167c.791-.77 2.058-.754 2.828.038l9.667 9.928 9.667-9.928c.77-.791 2.036-.808 2.828-.038s.808 2.037.038 2.828L50.29 56.5l9.742 10.005c.77.791.754 2.058-.037 2.828-.792.77-2.058.754-2.829-.038L47.5 59.367l-9.667 9.928c-.771.792-2.037.809-2.829.038-.791-.77-.808-2.037-.037-2.828L44.709 56.5l-9.742-10.005c-.77-.791-.753-2.057.038-2.828"
        })
    });
const $eebc29a0adccf87b$var$ForwardRef = /*#__PURE__*/ (0, $5zPO9$forwardRef)($eebc29a0adccf87b$var$SvgComponent);
var $eebc29a0adccf87b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($eebc29a0adccf87b$var$ForwardRef);


export {$eebc29a0adccf87b$export$2e2bcd8739ae039 as default};
