// @react-spectrum/s2/illustrations/linear/internal/S2_lin_minimize_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2rl7x$reactjsxruntime = require("react/jsx-runtime");
var $2rl7x$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d1ccbf017f828646$export$2e2bcd8739ae039);




const $d1ccbf017f828646$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2rl7x$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2rl7x$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M39.894 10.134c.488-.488.488-1.28 0-1.768s-1.28-.488-1.768 0l-8.984 8.984v-5.538c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v8.525c0 .69.56 1.25 1.25 1.25h8.525c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-5.476zM21.617 27.863c0-.69-.56-1.25-1.25-1.25h-8.525c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h5.528l-9.004 9.003c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0l8.983-8.984v5.488c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25z"
        })
    });
const $d1ccbf017f828646$var$ForwardRef = /*#__PURE__*/ (0, $2rl7x$react.forwardRef)($d1ccbf017f828646$var$SvgComponent);
var $d1ccbf017f828646$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d1ccbf017f828646$var$ForwardRef);
