// @react-spectrum/s2/illustrations/linear/internal/S2_lin_libraries_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $3CEuR$jsx} from "react/jsx-runtime";
import {forwardRef as $3CEuR$forwardRef} from "react";





const $affa7e0148f0142c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3CEuR$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3CEuR$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M31.43 47.43c0-8.841 7.159-16 16-16h53.82c2.209 0 4-1.791 4-4s-1.791-4-4-4H47.43c-13.26 0-24 10.74-24 24v53.82c0 2.209 1.79 4 4 4s4-1.791 4-4zm27-1c-6.628 0-12 5.372-12 12v58.15c0 6.627 5.372 12 12 12h58.15c6.627 0 12-5.373 12-12V58.43c0-6.092-4.54-11.124-10.42-11.897V83.96c0 1.59-1.78 2.55-3.11 1.66l-7.9-5.26c-.67-.45-1.55-.45-2.22 0l-7.9 5.26c-1.33.89-3.11-.06-3.11-1.66V46.43zm-20 12c0-11.046 8.954-20 20-20h58.15c11.045 0 20 8.954 20 20v58.15c0 11.045-8.955 20-20 20H58.43c-11.046 0-20-8.955-20-20z"
        })
    });
const $affa7e0148f0142c$var$ForwardRef = /*#__PURE__*/ (0, $3CEuR$forwardRef)($affa7e0148f0142c$var$SvgComponent);
var $affa7e0148f0142c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($affa7e0148f0142c$var$ForwardRef);


export {$affa7e0148f0142c$export$2e2bcd8739ae039 as default};
