// @react-spectrum/s2/illustrations/linear/internal/S2_lin_mailClose_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $8LXLJ$reactjsxruntime = require("react/jsx-runtime");
var $8LXLJ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b4fe76431f885bba$export$2e2bcd8739ae039);




const $b4fe76431f885bba$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8LXLJ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8LXLJ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8.878 11.845c.395-.223.832-.345 1.276-.345h27.692c.515 0 1.02.164 1.461.459l-14.618 8.57c-.51.3-1.136.294-1.64-.015zm-1.503 2.01q-.123.459-.125.978v18.334c0 1.96 1.413 3.333 2.904 3.333h27.692c1.491 0 2.904-1.374 2.904-3.333V14.833q0-.406-.078-.777l-14.719 8.63c-1.305.766-2.919.75-4.21-.04zM10.154 9C7.057 9 4.75 11.73 4.75 14.833v18.334C4.75 36.27 7.057 39 10.154 39h27.692c3.098 0 5.404-2.73 5.404-5.833V14.833C43.25 11.73 40.944 9 37.846 9z"
        })
    });
const $b4fe76431f885bba$var$ForwardRef = /*#__PURE__*/ (0, $8LXLJ$react.forwardRef)($b4fe76431f885bba$var$SvgComponent);
var $b4fe76431f885bba$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b4fe76431f885bba$var$ForwardRef);
