// @react-spectrum/s2/illustrations/linear/internal/S2_lin_bolt_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8B0Ff$jsx} from "react/jsx-runtime";
import {forwardRef as $8B0Ff$forwardRef} from "react";





const $e8efd438518e94dd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8B0Ff$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8B0Ff$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            d: "M28.7 8.38c.81 0 1.56.82 1.17 1.75l-3.6 8.49c-.27.64.03 1.37.66 1.64l5.92 2.51c.79.33 1.02 1.35.44 1.99L20.26 39.21c-.27.3-.59.42-.91.42-.81 0-1.56-.81-1.17-1.75l3.49-8.28c.27-.63-.03-1.36-.66-1.63l-5.88-2.53c-.78-.34-1.01-1.34-.44-1.98L27.78 8.81c.27-.3.59-.43.91-.43m.01-2.5c-1.07 0-2.05.45-2.78 1.26L12.83 21.79c-.8.9-1.12 2.13-.87 3.31.26 1.17 1.07 2.16 2.18 2.63l4.74 2.04-3.01 7.14c-.49 1.17-.37 2.48.31 3.51.7 1.05 1.91 1.7 3.16 1.7 1.06 0 2.04-.44 2.77-1.25l13.03-14.45c.81-.9 1.14-2.14.88-3.32s-1.08-2.17-2.2-2.64l-4.77-2.02 3.11-7.33c.49-1.17.38-2.48-.31-3.52-.7-1.05-1.91-1.71-3.16-1.71z"
        })
    });
const $e8efd438518e94dd$var$ForwardRef = /*#__PURE__*/ (0, $8B0Ff$forwardRef)($e8efd438518e94dd$var$SvgComponent);
var $e8efd438518e94dd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e8efd438518e94dd$var$ForwardRef);


export {$e8efd438518e94dd$export$2e2bcd8739ae039 as default};
