// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cardTapPayment_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $aVdZX$reactjsxruntime = require("react/jsx-runtime");
var $aVdZX$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $897764dffc558792$export$2e2bcd8739ae039);




const $897764dffc558792$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aVdZX$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $aVdZX$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M114.542 24.78c.268-2.194 2.263-3.754 4.455-3.486 11.158 1.362 21.58 7.773 27.64 18.27 6.11 10.582 6.407 22.927 1.894 33.325-.88 2.026-3.236 2.956-5.262 2.077-2.027-.88-2.956-3.236-2.077-5.262 3.547-8.173 3.306-17.845-1.483-26.14-4.75-8.229-12.911-13.258-21.681-14.33-2.193-.267-3.754-2.262-3.486-4.455M11.072 96.271c-5.523-9.566-2.246-21.798 7.32-27.32L80.94 32.838c9.566-5.522 21.797-2.245 27.32 7.32l18.742 32.462c5.522 9.566 2.245 21.797-7.321 27.32l-62.548 36.112c-9.565 5.523-21.797 2.246-27.32-7.32zm11.32-20.393c-5.74 3.314-7.706 10.653-4.392 16.393l.423.733 83.332-48.112-.423-.733c-3.314-5.74-10.653-7.706-16.392-4.392zm14.349 48.854-14.318-24.8 83.332-48.112 14.318 24.8c3.314 5.74 1.348 13.078-4.392 16.392l-62.548 36.112c-5.74 3.314-13.078 1.347-16.392-4.392m81.003-89.494c-2.193-.268-4.188 1.292-4.456 3.485-.267 2.193 1.293 4.188 3.486 4.456 4.469.545 8.619 3.106 11.036 7.292 2.437 4.221 2.562 9.141.755 13.306-.88 2.026.05 4.382 2.076 5.262s4.383-.05 5.262-2.077c2.773-6.39 2.593-13.982-1.165-20.49-3.727-6.456-10.138-10.397-16.994-11.234m-83.852 65.396c-1.913 1.105-2.568 3.551-1.464 5.464s3.551 2.569 5.465 1.464l32.38-18.697c1.913-1.105 2.568-3.551 1.464-5.464s-3.551-2.569-5.465-1.464zm5.548 17.609c-1.105-1.913-.45-4.36 1.464-5.464l16.19-9.349c1.913-1.105 4.36-.449 5.464 1.464s.45 4.359-1.464 5.464l-16.19 9.349c-1.913 1.104-4.36.449-5.464-1.464m46.068-47.408c-1.913 1.105-2.568 3.551-1.464 5.464s3.551 2.569 5.464 1.464L104.69 69c1.913-1.105 2.569-3.551 1.464-5.465s-3.551-2.568-5.464-1.463z"
        })
    });
const $897764dffc558792$var$ForwardRef = /*#__PURE__*/ (0, $aVdZX$react.forwardRef)($897764dffc558792$var$SvgComponent);
var $897764dffc558792$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($897764dffc558792$var$ForwardRef);
