// @react-spectrum/s2/illustrations/linear/internal/S2_lin_trash_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $gXcCk$reactjsxruntime = require("react/jsx-runtime");
var $gXcCk$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $cced457f9a94b528$export$2e2bcd8739ae039);




const $cced457f9a94b528$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gXcCk$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gXcCk$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M72 60c0-2.21-1.79-4-4-4s-4 1.79-4 4v56c0 2.209 1.79 4 4 4s4-1.791 4-4zM92 56c2.21 0 4 1.79 4 4v56c0 2.209-1.79 4-4 4s-4-1.791-4-4V60c0-2.21 1.79-4 4-4"
            }),
            /*#__PURE__*/ (0, $gXcCk$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M56 40v-6c0-5.523 4.477-10 10-10h28c5.523 0 10 4.477 10 10v6h31c2.209 0 4 1.79 4 4s-1.791 4-4 4h-5.438l-8.571 73.787c-.956 8.231-8.16 14.213-16.442 14.213H55.451c-8.282 0-15.486-5.982-16.442-14.213L30.438 48H25c-2.21 0-4-1.79-4-4s1.79-4 4-4zm8-6c0-1.105.895-2 2-2h28c1.105 0 2 .895 2 2v6H64zM38.492 48h83.016l-8.464 72.864c-.455 3.923-3.992 7.136-8.495 7.136H55.451c-4.503 0-8.04-3.213-8.495-7.136z"
            })
        ]
    });
const $cced457f9a94b528$var$ForwardRef = /*#__PURE__*/ (0, $gXcCk$react.forwardRef)($cced457f9a94b528$var$SvgComponent);
var $cced457f9a94b528$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($cced457f9a94b528$var$ForwardRef);
