// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noFilter_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1iwtk$reactjsxruntime = require("react/jsx-runtime");
var $1iwtk$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $fc0e81930856164b$export$2e2bcd8739ae039);




const $fc0e81930856164b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1iwtk$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1iwtk$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M61.536 20.586c0-8.147 6.603-14.75 14.75-14.75s14.75 6.603 14.75 14.75-6.603 14.75-14.75 14.75-14.75-6.603-14.75-14.75m21.043 6.293c.781-.782.781-2.054 0-2.836h.02l-3.457-3.457 3.457-3.457c.781-.782.781-2.054 0-2.836-.782-.781-2.054-.781-2.836 0l-3.457 3.457-3.457-3.457c-.782-.781-2.054-.781-2.836 0-.781.782-.781 2.054 0 2.836l3.457 3.457-3.457 3.457c-.781.782-.781 2.054 0 2.836.391.39.902.59 1.413.59s1.022-.2 1.413-.59l3.457-3.457 3.457 3.457c.39.39.902.59 1.413.59s1.022-.2 1.413-.59M24.562 15.74c-5.457 0-8.47 6.277-5.223 10.593q.032.042.066.083L41.05 52.222v27.212c0 3.663 4.318 5.707 7.111 3.222l5.347-4.757c.92-.82 1.443-1.994 1.443-3.222V52.222l6.232-7.43 5.323-6.347c.71-.846.6-2.108-.247-2.818-.846-.71-2.108-.599-2.817.248l-5.324 6.347-6.699 7.987c-.302.36-.468.815-.468 1.285v23.183c0 .094-.04.179-.102.233l-5.346 4.758c-.065.056-.109.067-.136.07-.037.005-.089 0-.146-.026s-.1-.064-.126-.103c-.02-.03-.045-.08-.045-.175v-27.94c0-.47-.165-.925-.467-1.285L22.508 23.891c-1.268-1.743-.007-4.15 2.054-4.15H56.47c1.104 0 2-.896 2-2s-.896-2-2-2z"
        })
    });
const $fc0e81930856164b$var$ForwardRef = /*#__PURE__*/ (0, $1iwtk$react.forwardRef)($fc0e81930856164b$var$SvgComponent);
var $fc0e81930856164b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($fc0e81930856164b$var$ForwardRef);
