// @react-spectrum/s2/illustrations/linear/internal/S2_lin_filter_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $cMQQ9$jsx} from "react/jsx-runtime";
import {forwardRef as $cMQQ9$forwardRef} from "react";





const $1783b5d9bc7cf245$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cMQQ9$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cMQQ9$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M29.277 43.055c-6.008-7.9-.374-19.264 9.552-19.264h82.344c9.925 0 15.56 11.364 9.551 19.264q-.064.084-.133.166L92.697 87.916v38.819c0 2.299-.989 4.486-2.715 6.005l-9.392 8.267c-5.168 4.548-13.286.879-13.286-6.005V87.916L29.41 43.22q-.069-.081-.133-.166m6.315-4.914 38.763 45.72c.613.723.95 1.64.95 2.587v48.554l9.392-8.267V86.448c0-.947.336-1.864.949-2.586l38.763-45.72c1.916-2.63.042-6.351-3.236-6.351H38.829c-3.28 0-5.153 3.72-3.237 6.35"
        })
    });
const $1783b5d9bc7cf245$var$ForwardRef = /*#__PURE__*/ (0, $cMQQ9$forwardRef)($1783b5d9bc7cf245$var$SvgComponent);
var $1783b5d9bc7cf245$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1783b5d9bc7cf245$var$ForwardRef);


export {$1783b5d9bc7cf245$export$2e2bcd8739ae039 as default};
