// @react-spectrum/s2/illustrations/linear/internal/S2_lin_explosion_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $fR5Kn$reactjsxruntime = require("react/jsx-runtime");
var $fR5Kn$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $db6089f4ce08d1a4$export$2e2bcd8739ae039);




const $db6089f4ce08d1a4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fR5Kn$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fR5Kn$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M19.243 15.136c-1.573-2.912 2.036-5.852 4.57-3.723l22.613 19L59.734 15.72c2.07-2.286 5.829-.335 5.15 2.674l-3.878 17.21H90.49c3.462 0 4.172 4.893.852 5.876l-25.846 7.65 8.14 9.584c1.982 2.332-.39 5.785-3.278 4.773l-12.296-4.31v25.21c0 3.331-4.618 4.181-5.804 1.069l-9.355-24.549c-6.002 3.455-14.548 8.3-19.48 11.09-2.835 1.604-5.751-1.784-3.916-4.359 2.932-4.115 7.688-10.817 11.016-15.647l-18.044-4.63c-3.033-.779-2.995-5.1.05-5.825l18.585-4.428zm6.014 2.715L37.111 39.79l-19.723 4.7 19.642 5.04-1.59 2.352c-2.52 3.732-6.715 9.684-10.073 14.416 5.572-3.158 12.867-7.306 17.566-10.028l2.034-1.179 9.095 23.863V53.536L68.2 58.491l-9.73-11.454 25.119-7.435H56.005l4.142-18.38-13.344 14.733z"
        })
    });
const $db6089f4ce08d1a4$var$ForwardRef = /*#__PURE__*/ (0, $fR5Kn$react.forwardRef)($db6089f4ce08d1a4$var$SvgComponent);
var $db6089f4ce08d1a4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($db6089f4ce08d1a4$var$ForwardRef);
