// @react-spectrum/s2/illustrations/linear/internal/S2_lin_laptop_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $jiJ5p$jsx} from "react/jsx-runtime";
import {forwardRef as $jiJ5p$forwardRef} from "react";





const $d07894f7d294adfa$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jiJ5p$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jiJ5p$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M23 29c0-3.315 2.685-6 6-6h38c3.315 0 6 2.685 6 6v34H23zm-4 34V29c0-5.525 4.475-10 10-10h38c5.525 0 10 4.475 10 10v34h3.63c3.866 0 7 3.134 7 7s-3.134 7-7 7H16c-3.866 0-7-3.134-7-7s3.134-7 7-7zm29.5-32c1.38 0 2.5-1.12 2.5-2.5S49.88 26 48.5 26 46 27.12 46 28.5s1.12 2.5 2.5 2.5M16 67c-1.657 0-3 1.343-3 3s1.343 3 3 3h64.63c1.657 0 3-1.343 3-3s-1.343-3-3-3z"
        })
    });
const $d07894f7d294adfa$var$ForwardRef = /*#__PURE__*/ (0, $jiJ5p$forwardRef)($d07894f7d294adfa$var$SvgComponent);
var $d07894f7d294adfa$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d07894f7d294adfa$var$ForwardRef);


export {$d07894f7d294adfa$export$2e2bcd8739ae039 as default};
