// @react-spectrum/s2/illustrations/linear/internal/S2_lin_commentText_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $Gkl3E$jsx} from "react/jsx-runtime";
import {forwardRef as $Gkl3E$forwardRef} from "react";





const $101a1e164d560972$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $Gkl3E$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $Gkl3E$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20.64 15.416c-5.838 0-10.64 4.615-10.64 10.5v28c0 5.885 4.802 10.5 10.64 10.5h5.368c.012 3.816.02 8.529-.008 12.252-.03 4.206 4.916 6.426 8.07 3.714l.002-.003L52.74 64.416h22.62c5.839 0 10.641-4.615 10.641-10.5v-28c0-5.885-4.802-10.5-10.64-10.5zM14 25.916c0-3.6 2.935-6.5 6.64-6.5h54.72c3.705 0 6.64 2.9 6.64 6.5v28c0 3.6-2.935 6.5-6.64 6.5H51.26l-.56.48-19.234 16.448-.006.005c-.592.51-1.466.065-1.461-.652.032-4.429.017-10.238 0-14.29l-.008-1.991H20.64c-3.705 0-6.64-2.9-6.64-6.5zm16 5.979c-1.105 0-2 .895-2 2s.895 2 2 2h36c1.105 0 2-.896 2-2s-.895-2-2-2zm0 12c-1.105 0-2 .895-2 2s.895 2 2 2h36c1.105 0 2-.896 2-2s-.895-2-2-2z"
        })
    });
const $101a1e164d560972$var$ForwardRef = /*#__PURE__*/ (0, $Gkl3E$forwardRef)($101a1e164d560972$var$SvgComponent);
var $101a1e164d560972$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($101a1e164d560972$var$ForwardRef);


export {$101a1e164d560972$export$2e2bcd8739ae039 as default};
