// @react-spectrum/s2/illustrations/linear/internal/S2_lin_tablet_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $eTe8G$reactjsxruntime = require("react/jsx-runtime");
var $eTe8G$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $65886b5a93fbf282$export$2e2bcd8739ae039);




const $65886b5a93fbf282$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eTe8G$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $eTe8G$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20 44c0-6.627 5.373-12 12-12h96c6.627 0 12 5.373 12 12v72c0 6.627-5.373 12-12 12H32c-6.627 0-12-5.373-12-12zm12-20c-11.046 0-20 8.954-20 20v72c0 11.046 8.954 20 20 20h96c11.046 0 20-8.954 20-20V44c0-11.046-8.954-20-20-20zm-3.5 55.5c0 2.486 2.014 4.5 4.5 4.5s4.5-2.014 4.5-4.5S35.486 75 33 75h-.011c-2.486 0-4.489 2.014-4.489 4.5"
        })
    });
const $65886b5a93fbf282$var$ForwardRef = /*#__PURE__*/ (0, $eTe8G$react.forwardRef)($65886b5a93fbf282$var$SvgComponent);
var $65886b5a93fbf282$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($65886b5a93fbf282$var$ForwardRef);
