// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileText_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $kSslR$jsxs, jsx as $kSslR$jsx} from "react/jsx-runtime";
import {forwardRef as $kSslR$forwardRef} from "react";





const $4f6982ecd944ddee$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kSslR$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kSslR$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M18.498 21.547c-.321 0-.614.121-.835.32q-.048.042-.09.09c-.202.222-.325.517-.325.84v2.4c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-1.15h3.5v9.499h-1.15c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h4.799c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-1.15v-9.499h3.5v1.15c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-2.4c0-.69-.56-1.25-1.25-1.25z"
            }),
            /*#__PURE__*/ (0, $kSslR$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M15.046 7.75c-2.62 0-4.796 2.081-4.796 4.711v23.077c0 2.631 2.176 4.712 4.796 4.712h18.909c2.62 0 4.795-2.081 4.795-4.712V21.972c0-1.26-.513-2.46-1.415-3.342l-9.742-9.51c-.901-.88-2.117-1.37-3.38-1.37zm-2.296 4.711c0-1.194 1-2.211 2.296-2.211h9.167c.618 0 1.206.24 1.634.658l9.741 9.511c.428.417.662.976.662 1.553v13.566c0 1.195-1 2.212-2.295 2.212h-18.91c-1.295 0-2.295-1.017-2.295-2.212zm0 0h-.25z"
            })
        ]
    });
const $4f6982ecd944ddee$var$ForwardRef = /*#__PURE__*/ (0, $kSslR$forwardRef)($4f6982ecd944ddee$var$SvgComponent);
var $4f6982ecd944ddee$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($4f6982ecd944ddee$var$ForwardRef);


export {$4f6982ecd944ddee$export$2e2bcd8739ae039 as default};
