// @react-spectrum/s2/illustrations/linear/internal/S2_lin_illustration_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $elYwo$jsx} from "react/jsx-runtime";
import {forwardRef as $elYwo$forwardRef} from "react";





const $64bcaee33d402e4a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $elYwo$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $elYwo$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m54.45 37.665-4.943 9.526c-1.098 2.115-2.915 3.77-5.124 4.663l-21.484 8.697c-4.093 1.657-8.167-2.418-6.51-6.51l8.696-21.485c.894-2.209 2.548-4.026 4.664-5.124l9.469-4.913 4.566-4.566c2.343-2.343 6.142-2.343 8.485 0l6.704 6.703c2.343 2.344 2.343 6.143 0 8.486zm-4.089-.805-4.405 8.488c-.658 1.27-1.748 2.262-3.074 2.799l-18.889 7.646 10.27-10.27c.863.084 1.756-.204 2.417-.865 1.172-1.171 1.172-3.07 0-4.242s-3.07-1.172-4.242 0c-.595.594-.887 1.375-.879 2.154L21.122 53.007l7.67-18.95c.537-1.325 1.53-2.416 2.799-3.074l8.488-4.405zm5.784-6.547-3.337 3.337-9.532-9.532 3.336-3.337c.782-.78 2.048-.78 2.829 0l6.704 6.704c.78.78.78 2.047 0 2.828m4.316 31.22 15-18.31c1.095-1.337.9-3.308-.437-4.403s-3.309-.9-4.404.437L55.674 57.502c.885.343 1.679.793 2.335 1.33.856.702 1.736 1.597 2.452 2.702m1.514 4.326.069.055 16.511-20.156c2.495-3.046 2.05-7.538-.997-10.033-3.045-2.495-7.537-2.048-10.032.997L51.131 56.736q-.81.02-1.623.15c-3.332.536-6.708 2.534-8.715 6.676-.823 1.7-1.441 3.048-1.954 4.166-.707 1.544-1.215 2.651-1.787 3.645-.864 1.5-1.808 2.588-3.997 4.293-.64.499-.916 1.336-.697 2.118s.89 1.354 1.697 1.447c3.04.35 8.393.838 13.531.212 4.998-.61 10.58-2.372 13.013-7.236 1.153-2.304 1.55-4.43 1.376-6.348m-4.953 4.558c-1.461 2.921-5.133 4.472-9.92 5.055-2.771.338-5.656.319-8.164.175.594-.708 1.1-1.443 1.58-2.278.65-1.127 1.283-2.506 2.08-4.237.515-1.12 1.097-2.386 1.795-3.826 1.394-2.877 3.604-4.125 5.751-4.471 2.271-.366 4.355.293 5.33 1.091 1.037.85 1.92 1.933 2.315 3.243.378 1.258.377 2.962-.767 5.248"
        })
    });
const $64bcaee33d402e4a$var$ForwardRef = /*#__PURE__*/ (0, $elYwo$forwardRef)($64bcaee33d402e4a$var$SvgComponent);
var $64bcaee33d402e4a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($64bcaee33d402e4a$var$ForwardRef);


export {$64bcaee33d402e4a$export$2e2bcd8739ae039 as default};
