// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cut_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $fDN9Y$reactjsxruntime = require("react/jsx-runtime");
var $fDN9Y$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $30622d5e64b029b3$export$2e2bcd8739ae039);




const $30622d5e64b029b3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fDN9Y$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fDN9Y$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M29.97 7.345c.654.224 1.002.934.779 1.588l-4.218 12.32 12.485-4.252c.653-.222 1.363.127 1.586.78.222.654-.127 1.364-.78 1.587L25.807 24.14q-.076.026-.154.042-.093.378-.182.77v.001c-.255 1.1-.53 2.29-1.15 3.63q-.159.344-.342.69h.081c3.162 0 5.724 2.563 5.724 5.724s-2.562 5.724-5.724 5.724c-3.16 0-5.724-2.562-5.724-5.724 0-1.24.395-2.39 1.066-3.327q.06-.1.144-.192.113-.145.235-.283c.528-.674 1.61-2.234 2.271-3.662.464-1.001.667-1.802.882-2.722-.41.155-.858.333-1.386.55-.825.34-1.792.88-2.633 1.405-.853.531-1.488.99-1.663 1.139q-.064.054-.131.097c-1.038 1.047-2.476 1.695-4.066 1.695-3.161 0-5.724-2.563-5.724-5.724s2.563-5.724 5.724-5.724c3.152 0 5.708 2.547 5.724 5.694.596-.332 1.222-.648 1.817-.893 1.265-.522 2.113-.835 3.064-1.134l4.722-13.793c.224-.653.935-1.002 1.588-.778M21.478 33.07c-.402.537-.64 1.205-.64 1.927 0 1.78 1.444 3.224 3.224 3.224s3.224-1.443 3.224-3.224c0-1.78-1.443-3.224-3.224-3.224-.942 0-1.79.404-2.379 1.048q-.113.144-.205.249m-8.42-12.321c-1.782 0-3.225 1.443-3.225 3.224 0 1.78 1.443 3.224 3.224 3.224 1.78 0 3.224-1.444 3.224-3.224s-1.443-3.224-3.224-3.224"
        })
    });
const $30622d5e64b029b3$var$ForwardRef = /*#__PURE__*/ (0, $fDN9Y$react.forwardRef)($30622d5e64b029b3$var$SvgComponent);
var $30622d5e64b029b3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($30622d5e64b029b3$var$ForwardRef);
