// @react-spectrum/s2/illustrations/linear/internal/S2_lin_desktop_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $f3dEH$reactjsxruntime = require("react/jsx-runtime");
var $f3dEH$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0fdaee257a41b43a$export$2e2bcd8739ae039);




const $0fdaee257a41b43a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $f3dEH$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $f3dEH$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M7.25 13c0-1.519 1.231-2.75 2.75-2.75h28c1.519 0 2.75 1.231 2.75 2.75v14c0 1.519-1.231 2.75-2.75 2.75H10c-1.519 0-2.75-1.231-2.75-2.75zM10 7.75c-2.9 0-5.25 2.35-5.25 5.25v14c0 2.9 2.35 5.25 5.25 5.25h12v5.5h-4c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h12c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-4v-5.5h12c2.9 0 5.25-2.35 5.25-5.25V13c0-2.9-2.35-5.25-5.25-5.25zm15.5 5.75c0 .828-.672 1.5-1.5 1.5s-1.5-.672-1.5-1.5.672-1.5 1.5-1.5 1.5.672 1.5 1.5"
        })
    });
const $0fdaee257a41b43a$var$ForwardRef = /*#__PURE__*/ (0, $f3dEH$react.forwardRef)($0fdaee257a41b43a$var$SvgComponent);
var $0fdaee257a41b43a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0fdaee257a41b43a$var$ForwardRef);
