// @react-spectrum/s2/illustrations/linear/internal/S2_lin_animation_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $gwWwt$reactjsxruntime = require("react/jsx-runtime");
var $gwWwt$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7102d6468053c63f$export$2e2bcd8739ae039);




const $7102d6468053c63f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gwWwt$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gwWwt$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M74.593 16.483c.527-.985 1.562-1.649 2.75-1.649 1.719 0 3.113 1.394 3.113 3.114 0 .64-.19 1.23-.519 1.722-.535.804-.317 1.89.488 2.426s1.89.317 2.426-.487c.698-1.05 1.105-2.31 1.105-3.661 0-3.653-2.961-6.614-6.614-6.614-2.53 0-4.724 1.42-5.835 3.5-.456.852-.135 1.912.718 2.368.852.455 1.913.134 2.368-.719m-5.71 7.902c-2.462 0-4.605 1.379-5.694 3.416-.521.974-1.733 1.342-2.707.821-.974-.52-1.342-1.732-.821-2.707 1.757-3.287 5.226-5.53 9.223-5.53 5.772 0 10.452 4.68 10.452 10.452 0 2.136-.643 4.128-1.746 5.785-.612.92-1.853 1.169-2.773.557s-1.169-1.854-.557-2.773c.68-1.021 1.076-2.246 1.076-3.569 0-3.564-2.889-6.452-6.452-6.452M48.329 40.05c1.96-2.372 4.917-3.877 8.227-3.877 5.892 0 10.668 4.776 10.668 10.668 0 4.66-2.99 8.628-7.162 10.08-1.043.362-1.595 1.502-1.232 2.545s1.503 1.595 2.546 1.232c5.73-1.993 9.848-7.442 9.848-13.857 0-8.101-6.567-14.668-14.668-14.668-4.553 0-8.623 2.077-11.31 5.328-.704.852-.584 2.112.267 2.816s2.112.584 2.816-.267M15.795 63.986c0-9.35 7.58-16.93 16.93-16.93s16.93 7.58 16.93 16.93-7.58 16.93-16.93 16.93-16.93-7.58-16.93-16.93m16.93-20.93c-11.56 0-20.93 9.371-20.93 20.93 0 11.56 9.37 20.93 20.93 20.93 11.559 0 20.93-9.37 20.93-20.93 0-11.559-9.371-20.93-20.93-20.93"
        })
    });
const $7102d6468053c63f$var$ForwardRef = /*#__PURE__*/ (0, $gwWwt$react.forwardRef)($7102d6468053c63f$var$SvgComponent);
var $7102d6468053c63f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7102d6468053c63f$var$ForwardRef);
