// @react-spectrum/s2/illustrations/linear/internal/S2_lin_playTriangle_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $cydmG$jsx} from "react/jsx-runtime";
import {forwardRef as $cydmG$forwardRef} from "react";





const $d174accb83c65fa3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cydmG$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cydmG$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M36.083 44.362c.012-10.866 11.596-18.084 21.141-12.387l59.809 35.694c9.302 5.55 9.287 19.297-.029 24.826L57.112 128.04c-9.559 5.674-21.125-1.573-21.112-12.438zm17.042-5.518c-3.851-2.298-9.036.436-9.042 5.527L44 115.612c-.006 5.091 5.174 7.838 9.03 5.55l59.891-35.547c4.099-2.433 4.107-8.633.012-11.077zl1.832-3.069z"
        })
    });
const $d174accb83c65fa3$var$ForwardRef = /*#__PURE__*/ (0, $cydmG$forwardRef)($d174accb83c65fa3$var$SvgComponent);
var $d174accb83c65fa3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d174accb83c65fa3$var$ForwardRef);


export {$d174accb83c65fa3$export$2e2bcd8739ae039 as default};
