// @react-spectrum/s2/illustrations/linear/internal/S2_lin_thumbsDown_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $8hYpP$reactjsxruntime = require("react/jsx-runtime");
var $8hYpP$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a88b9164490dba28$export$2e2bcd8739ae039);




const $a88b9164490dba28$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8hYpP$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8hYpP$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M19.152 9.237c1.481-.552 3.05-.835 4.63-.835h7.381q.044 0 .087.003h1.909c.882 0 1.852.24 2.634.84.823.632 1.354 1.604 1.354 2.836 0 .33-.065.897-.317 1.488.97.487 1.702 1.4 1.931 2.52.194.945.005 1.864-.45 2.613.824.628 1.374 1.6 1.43 2.71.041.795-.176 1.542-.577 2.16.758.63 1.258 1.56 1.312 2.616.106 2.08-1.551 3.824-3.634 3.824h-7.591l.451 1.451c.538 1.727.883 3.509 1.03 5.312l.08 1.004c.167 2.057-1.458 3.819-3.522 3.819-1.361 0-2.602-.783-3.189-2.012l-2.697-5.652q-.34-.714-.781-1.37l-.472-.7c-1.393-2.073-3.726-3.315-6.223-3.315h-3.283c-1.795 0-3.25-1.456-3.25-3.25V14.345c0-1.795 1.455-3.25 3.25-3.25h1.587c1.282 0 2.554-.23 3.756-.678zm4.63 1.665h4.135q.042.003.086.003h5.156c.464 0 .862.132 1.113.324.21.16.375.402.375.852 0 .123-.034.37-.155.589-.1.18-.27.372-.655.475q-.06.016-.116.037h-.052c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h1.528c.54 0 1.007.38 1.115.91.145.705-.394 1.366-1.115 1.366h-.549c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h.55q.134 0 .266-.01.078.011.159.011h.485c.606 0 1.106.475 1.137 1.08.033.643-.474 1.184-1.115 1.197h-.752c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h1.464c.606 0 1.106.475 1.137 1.08.033.65-.485 1.196-1.137 1.196h-8.015c-1.475 0-2.527 1.43-2.089 2.839l.577 1.855c.483 1.551.793 3.151.924 4.771l.081 1.004c.05.602-.426 1.117-1.03 1.117-.398 0-.76-.23-.932-.589l-2.697-5.652q-.42-.88-.963-1.687l-.472-.702c-1.683-2.502-4.394-4.098-7.364-4.376v-12.76q1.02-.208 2-.572l3.164-1.18c1.201-.448 2.473-.678 3.756-.678m-11.42 2.692V26.05h-1.717c-.415 0-.75-.336-.75-.75V14.345c0-.414.335-.75.75-.75h1.717"
        })
    });
const $a88b9164490dba28$var$ForwardRef = /*#__PURE__*/ (0, $8hYpP$react.forwardRef)($a88b9164490dba28$var$SvgComponent);
var $a88b9164490dba28$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a88b9164490dba28$var$ForwardRef);
