// @react-spectrum/s2/illustrations/linear/internal/S2_lin_stopwatch_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $lG5My$jsx} from "react/jsx-runtime";
import {forwardRef as $lG5My$forwardRef} from "react";





const $2dc5a6d8942429af$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lG5My$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $lG5My$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M52.043 14.117c1.105 0 2-.895 2-2s-.895-2-2-2h-9.238c-1.105 0-2 .896-2 2s.895 2 2 2h2.619v3.023C27.346 18.458 13.09 33.543 13.09 51.957c0 19.28 15.63 34.91 34.91 34.91 19.281 0 34.911-15.63 34.911-34.91 0-18.804-14.866-34.136-33.487-34.883v-2.957zM17.09 51.957c0-17.072 13.839-30.911 30.91-30.911 17.072 0 30.911 13.84 30.911 30.91 0 17.072-13.839 30.911-30.91 30.911-17.072 0-30.911-13.839-30.911-30.91m32.334-25.982c0-1.105-.896-2-2-2s-2 .895-2 2v3.464c0 1.105.895 2 2 2s2-.895 2-2zm-2 46.5c1.104 0 2 .895 2 2v3.464c0 1.105-.896 2-2 2s-2-.895-2-2v-3.464c0-1.105.895-2 2-2m26.665-18.818c1.104 0 2-.895 2-2s-.896-2-2-2h-3.591c-1.105 0-2 .896-2 2s.895 2 2 2zm-46.479-2c0 1.105-.895 2-2 2h-3.59c-1.105 0-2-.895-2-2s.895-2 2-2h3.59c1.105 0 2 .896 2 2m40.71-16.54c.78-.78.78-2.047 0-2.828-.782-.781-2.048-.781-2.829 0l-2.54 2.54c-.78.78-.78 2.047 0 2.828.782.78 2.048.78 2.83 0zm-34.28 31.45c.781.781.781 2.048 0 2.829l-2.539 2.539c-.781.78-2.047.78-2.828 0-.781-.781-.781-2.047 0-2.829l2.539-2.539c.78-.78 2.047-.78 2.828 0M28.997 36.74c-.781-.78-.781-2.047 0-2.828s2.047-.781 2.828 0l2.454 2.454c.782.781.782 2.047 0 2.829-.78.78-2.047.78-2.828 0zm33.13 30.304c-.78.78-.78 2.047 0 2.828l2.455 2.454c.781.781 2.047.781 2.828 0 .781-.78.781-2.047 0-2.828l-2.454-2.454c-.78-.781-2.047-.781-2.828 0m-13.64-16.228c-1.03-.4-2.188.111-2.588 1.14-.4 1.03.111 2.19 1.14 2.59l20.445 7.937c1.03.4 2.189-.11 2.589-1.14s-.111-2.19-1.14-2.589z"
        })
    });
const $2dc5a6d8942429af$var$ForwardRef = /*#__PURE__*/ (0, $lG5My$forwardRef)($2dc5a6d8942429af$var$SvgComponent);
var $2dc5a6d8942429af$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2dc5a6d8942429af$var$ForwardRef);


export {$2dc5a6d8942429af$export$2e2bcd8739ae039 as default};
