// @react-spectrum/s2/illustrations/linear/internal/S2_lin_ruler_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $fcHeg$reactjsxruntime = require("react/jsx-runtime");
var $fcHeg$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c79d791234b41f40$export$2e2bcd8739ae039);




const $c79d791234b41f40$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fcHeg$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fcHeg$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28.907 120.099c-6.248-6.249-6.248-16.379 0-22.628l68.564-68.564c6.249-6.248 16.379-6.248 22.628 0l10.994 10.995c6.249 6.248 6.249 16.379 0 22.627l-68.564 68.564c-6.248 6.249-16.379 6.249-22.627 0zm5.657-16.971c-3.124 3.124-3.124 8.189 0 11.314l10.995 10.994c3.124 3.125 8.19 3.125 11.313 0l68.564-68.564c3.125-3.124 3.125-8.19 0-11.313l-10.994-10.995c-3.125-3.124-8.19-3.124-11.314 0l-5.788 5.788 7.692 7.692c1.562 1.562 1.562 4.095 0 5.657s-4.095 1.562-5.657 0l-7.692-7.692-11.454 11.454 12.344 12.344c1.562 1.562 1.562 4.095 0 5.657s-4.095 1.562-5.657 0L74.572 63.12 63.12 74.573l7.692 7.691c1.562 1.562 1.562 4.095 0 5.657s-4.095 1.562-5.657 0l-7.692-7.691L46.01 91.683l12.343 12.344c1.562 1.562 1.562 4.095 0 5.657s-4.094 1.562-5.656 0L40.352 97.34z"
        })
    });
const $c79d791234b41f40$var$ForwardRef = /*#__PURE__*/ (0, $fcHeg$react.forwardRef)($c79d791234b41f40$var$SvgComponent);
var $c79d791234b41f40$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c79d791234b41f40$var$ForwardRef);
