// @react-spectrum/s2/illustrations/linear/internal/S2_lin_threeArrowsCircle_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $e7iQ5$jsx} from "react/jsx-runtime";
import {forwardRef as $e7iQ5$forwardRef} from "react";





const $e6c2acadddfc0ef6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e7iQ5$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $e7iQ5$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M135.063 40.64c.548 2.14-.743 4.319-2.883 4.867l-24.791 6.346c-2.14.547-4.319-.744-4.867-2.884s.743-4.319 2.883-4.867l15.232-3.899q-.12-.082-.238-.175c-8.853-7.027-20.047-11.222-32.23-11.222-15.564 0-29.523 6.847-39.04 17.705-1.455 1.662-3.982 1.828-5.644.372s-1.828-3.983-.372-5.644C54.084 28.72 70.204 20.806 88.168 20.806c13.866 0 26.642 4.717 36.793 12.632l-3.994-15.605c-.548-2.14.743-4.32 2.883-4.867 2.14-.548 4.319.743 4.867 2.883zm2.899 24.614c-.659-2.108.515-4.352 2.623-5.012s4.353.515 5.012 2.624c1.771 5.657 2.723 11.672 2.723 17.902 0 24.134-14.274 44.925-34.828 54.408l14.547 5.118c2.084.733 3.179 3.017 2.446 5.101-.734 2.084-3.017 3.179-5.101 2.445l-24.14-8.493c-2.084-.733-3.179-3.017-2.445-5.101l8.493-24.139c.733-2.084 3.017-3.179 5.101-2.446s3.179 3.017 2.445 5.101l-5.449 15.489c18.223-8.055 30.931-26.289 30.931-47.483 0-5.41-.827-10.62-2.358-15.514M36.94 73.845l10.065 12.338c1.397 1.711 3.917 1.967 5.628.57 1.712-1.396 1.968-3.916.571-5.627l-16.176-19.83c-1.397-1.711-3.916-1.966-5.628-.57L11.57 76.902c-1.712 1.397-1.967 3.917-.57 5.628 1.396 1.712 3.915 1.968 5.627.571L28.98 73.025c-1.473 11.234.22 23.014 5.55 33.958 7.866 16.151 22.04 27.178 38.098 31.56 2.132.581 4.33-.675 4.912-2.806s-.674-4.331-2.806-4.912c-13.93-3.801-26.197-13.353-33.011-27.345-4.656-9.559-6.109-19.838-4.782-29.635"
        })
    });
const $e6c2acadddfc0ef6$var$ForwardRef = /*#__PURE__*/ (0, $e7iQ5$forwardRef)($e6c2acadddfc0ef6$var$SvgComponent);
var $e6c2acadddfc0ef6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e6c2acadddfc0ef6$var$ForwardRef);


export {$e6c2acadddfc0ef6$export$2e2bcd8739ae039 as default};
