// @react-spectrum/s2/illustrations/linear/internal/S2_lin_AIGenerateImage_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $Jtkni$reactjsxruntime = require("react/jsx-runtime");
var $Jtkni$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $de288b31c39bc41e$export$2e2bcd8739ae039);




const $de288b31c39bc41e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $Jtkni$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $Jtkni$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M77.553 7.316c3.529-3.253 9.14-.01 8.088 4.67l-1.402 6.236v.003c-.065.283.015.583.213.797l4.33 4.7c3.253 3.529.01 9.14-4.67 8.089l-6.239-1.404c-.282-.064-.583.016-.797.214l-4.7 4.33c-3.528 3.254-9.14.01-8.089-4.67l1.404-6.238c.064-.283-.016-.583-.214-.798l-4.33-4.699c-3.253-3.529-.01-9.14 4.67-8.09l6.239 1.404c.282.064.583-.016.797-.213zm4.186 3.793c.191-.857-.833-1.444-1.475-.852l-4.7 4.33c-1.18 1.088-2.823 1.53-4.39 1.175h-.002L64.94 14.36c-.857-.192-1.445.833-.853 1.475l4.33 4.698c1.088 1.18 1.529 2.824 1.174 4.391v.002l-1.402 6.232c-.191.857.833 1.445 1.475.853l4.699-4.33c1.18-1.088 2.823-1.53 4.39-1.175l1.482-7.646v.005l-1.479 7.641 6.231 1.402c.857.192 1.445-.833.853-1.475l-4.33-4.699c-.737-.8-1.177-1.812-1.274-2.869l1.481-7.65-1.482 7.645c-.045-.502-.013-1.014.1-1.517v-.002z"
            }),
            /*#__PURE__*/ (0, $Jtkni$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "m57.767 45.296 4.395.99c1.728.39 2.928-1.684 1.724-2.989l-3.054-3.31c-.401-.438-.564-1.042-.434-1.624l.99-4.395c.391-1.728-1.684-2.928-2.988-1.724l-3.311 3.054c-.438.401-1.041.564-1.623.434l-4.396-.99c-1.727-.39-2.927 1.684-1.724 2.989l3.055 3.31c.4.438.563 1.042.433 1.624l-.99 4.395c-.39 1.728 1.684 2.928 2.99 1.724l3.31-3.054c.437-.401 1.041-.564 1.623-.434"
            }),
            /*#__PURE__*/ (0, $Jtkni$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M37.138 21.502c-11.922-3.194-24.175 3.88-27.37 15.802-2.113 7.885.266 15.918 5.573 21.378l.18-4.738c.041-1.104.97-1.965 2.073-1.923 1.104.041 1.965.97 1.924 2.074l-.361 9.547c-.04 1.077-.927 1.928-2.005 1.925l-9.896-.032c-1.105-.003-1.998-.901-1.994-2.006s.902-1.997 2.006-1.994l5.286.017c-6.309-6.435-9.152-15.944-6.65-25.283 3.767-14.056 18.214-22.397 32.27-18.63 6.098 1.634 11.124 5.282 14.548 10.035.646.896.443 2.146-.453 2.791-.896.646-2.146.443-2.792-.453-2.905-4.032-7.163-7.123-12.34-8.51"
            }),
            /*#__PURE__*/ (0, $Jtkni$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M83.64 50.263c0-3.314-2.687-6-6-6h-8.063c-1.105 0-2-.896-2-2s.895-2 2-2h8.063c5.522 0 10 4.477 10 10V79.21c0 5.523-4.478 10-10 10H34.035c-5.523 0-10-4.477-10-10V50.263c0-5.523 4.477-10 10-10h9.01c1.105 0 2 .895 2 2s-.895 2-2 2h-9.01c-3.314 0-6 2.686-6 6V79.21c0 .623.095 1.224.271 1.79l12.369-19.11c2.15-3.326 7.023-3.326 9.173 0l10.035 15.5 5.65-8.542.008-.013c2.164-3.219 6.907-3.226 9.07 0l.003.003 8.563 12.686c.298-.712.463-1.494.463-2.314zM44.033 64.064l-13.12 20.271c.909.556 1.978.876 3.122.876H54.71l2.764-4.179-10.986-16.969c-.574-.888-1.88-.888-2.456.001m36.357 20.48c-.824.427-1.76.667-2.75.667H59.507l9.357-14.148c.585-.863 1.852-.855 2.426.002l.004.006z"
            }),
            /*#__PURE__*/ (0, $Jtkni$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M67.234 54.125c0 1.657 1.344 3 3 3 1.657 0 3-1.343 3-3s-1.343-3-3-3c-1.656 0-3 1.343-3 3"
            })
        ]
    });
const $de288b31c39bc41e$var$ForwardRef = /*#__PURE__*/ (0, $Jtkni$react.forwardRef)($de288b31c39bc41e$var$SvgComponent);
var $de288b31c39bc41e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($de288b31c39bc41e$var$ForwardRef);
