// @react-spectrum/s2/illustrations/linear/internal/S2_lin_check_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $grIzQ$jsx} from "react/jsx-runtime";
import {forwardRef as $grIzQ$forwardRef} from "react";





const $49ecff2ae8abb34d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $grIzQ$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $grIzQ$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28 80c0-28.719 23.281-52 52-52s52 23.281 52 52-23.281 52-52 52-52-23.281-52-52m52-60c-33.137 0-60 26.863-60 60s26.863 60 60 60 60-26.863 60-60-26.863-60-60-60m27.694 47.956c1.633-1.488 1.75-4.018.262-5.65s-4.018-1.75-5.65-.262l-16.679 15.2L74.45 87.433l-2.334 2.127-13.388-12.484c-1.616-1.507-4.147-1.419-5.653.197-1.507 1.615-1.419 4.147.197 5.653l16.084 15c1.524 1.421 3.883 1.435 5.423.031L72.085 95l2.695 2.956v-.001l.005-.004.017-.016.07-.063.27-.247 1.027-.936 3.67-3.345 11.177-10.186z"
        })
    });
const $49ecff2ae8abb34d$var$ForwardRef = /*#__PURE__*/ (0, $grIzQ$forwardRef)($49ecff2ae8abb34d$var$SvgComponent);
var $49ecff2ae8abb34d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($49ecff2ae8abb34d$var$ForwardRef);


export {$49ecff2ae8abb34d$export$2e2bcd8739ae039 as default};
