// @react-spectrum/s2/illustrations/linear/internal/S2_lin_home_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $dVGLM$jsx} from "react/jsx-runtime";
import {forwardRef as $dVGLM$forwardRef} from "react";





const $aeb9defc5b4fbe7c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dVGLM$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dVGLM$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M92.494 20.998c-7.305-5.844-17.684-5.844-24.988 0L31.744 49.607c-4.744 3.795-7.506 9.542-7.506 15.617v50.502c0 11.045 8.955 20 20 20h71.523c11.046 0 20-8.955 20-20V65.224c0-6.075-2.762-11.822-7.506-15.617zm-19.99 6.247c4.382-3.506 10.61-3.506 14.992 0l35.761 28.609c2.847 2.277 4.504 5.725 4.504 9.37v50.502c0 6.627-5.373 12-12 12h-14.508V95.414c0-8.836-7.163-16-16-16H74.746c-8.837 0-16 7.164-16 16v32.312H44.238c-6.627 0-12-5.373-12-12V65.224c0-3.645 1.657-7.093 4.504-9.37zm-5.758 100.481h26.507V95.414c0-4.418-3.582-8-8-8H74.746c-4.419 0-8 3.582-8 8z"
        })
    });
const $aeb9defc5b4fbe7c$var$ForwardRef = /*#__PURE__*/ (0, $dVGLM$forwardRef)($aeb9defc5b4fbe7c$var$SvgComponent);
var $aeb9defc5b4fbe7c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($aeb9defc5b4fbe7c$var$ForwardRef);


export {$aeb9defc5b4fbe7c$export$2e2bcd8739ae039 as default};
