// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browser_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $56VFA$reactjsxruntime = require("react/jsx-runtime");
var $56VFA$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ba7acc1066ed85ee$export$2e2bcd8739ae039);




const $ba7acc1066ed85ee$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $56VFA$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $56VFA$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M37 10.25c1.519 0 2.75 1.231 2.75 2.75v4.77H8.25V13c0-1.519 1.231-2.75 2.75-2.75zM8.25 20.27h31.5V35c0 1.519-1.231 2.75-2.75 2.75H11c-1.519 0-2.75-1.231-2.75-2.75zm34-7.27c0-2.9-2.35-5.25-5.25-5.25H11c-2.9 0-5.25 2.35-5.25 5.25v22c0 2.9 2.35 5.25 5.25 5.25h26c2.9 0 5.25-2.35 5.25-5.25zM13 15.5c-.828 0-1.5-.672-1.5-1.5s.672-1.5 1.5-1.5 1.5.672 1.5 1.5-.672 1.5-1.5 1.5"
        })
    });
const $ba7acc1066ed85ee$var$ForwardRef = /*#__PURE__*/ (0, $56VFA$react.forwardRef)($ba7acc1066ed85ee$var$SvgComponent);
var $ba7acc1066ed85ee$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ba7acc1066ed85ee$var$ForwardRef);
