// @react-spectrum/s2/illustrations/linear/internal/S2_lin_volumeOne_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $l3JSF$jsx} from "react/jsx-runtime";
import {forwardRef as $l3JSF$forwardRef} from "react";





const $576c13eab2107645$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $l3JSF$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $l3JSF$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M81.7 30.18c6.313-6.258 17.041-1.787 17.041 7.102v85.436c0 8.864-10.679 13.346-17.005 7.137l-19.003-18.653H48.627c-6.628 0-12-5.372-12-12V60.998c0-6.628 5.372-12 12-12h14.094zm9.041 7.102c0-1.778-2.145-2.672-3.408-1.42L69.277 53.763v52.653l18.063 17.729c1.266 1.242 3.401.346 3.401-1.427zm-29.464 65.92V56.998h-12.65c-2.21 0-4 1.79-4 4v38.204c0 2.21 1.79 4 4 4zm44.782-45.97c1.329-1.766 3.837-2.12 5.602-.791 7.104 5.346 11.71 13.861 11.71 23.452 0 9.67-4.682 18.245-11.885 23.583-1.775 1.315-4.28.943-5.595-.832-1.316-1.775-.943-4.28.832-5.596 5.255-3.893 8.648-10.127 8.648-17.155 0-6.97-3.338-13.16-8.521-17.06-1.765-1.328-2.119-3.836-.791-5.601"
        })
    });
const $576c13eab2107645$var$ForwardRef = /*#__PURE__*/ (0, $l3JSF$forwardRef)($576c13eab2107645$var$SvgComponent);
var $576c13eab2107645$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($576c13eab2107645$var$ForwardRef);


export {$576c13eab2107645$export$2e2bcd8739ae039 as default};
