// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noLibraries_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5bVFF$jsx} from "react/jsx-runtime";
import {forwardRef as $5bVFF$forwardRef} from "react";





const $f4a317f1acad08e0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5bVFF$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5bVFF$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M23.53 20.52c-4.416 0-8 3.584-8 8v31.95c0 1.104-.896 2-2 2s-2-.896-2-2V28.52c0-6.625 5.375-12 12-12h31.95c1.104 0 2 .895 2 2s-.896 2-2 2zm-1 13.01c0-3.314 2.686-6 6-6h21.61v22.21c0 .8.89 1.28 1.55.83l4.87-3.25c.34-.22.77-.22 1.11 0l4.87 3.25c.66.44 1.55-.03 1.55-.83V27.53h.39c3.313 0 6 2.686 6 6V57c0 1.104.895 2 2 2s2-.896 2-2V33.53c0-5.523-4.478-10-10-10H28.53c-5.524 0-10 4.477-10 10v35.95c0 5.522 4.476 10 10 10H52c1.105 0 2-.896 2-2s-.895-2-2-2H28.529c-3.313 0-6-2.687-6-6zM72.2 91.9c8.146 0 14.75-6.604 14.75-14.75S80.346 62.4 72.2 62.4s-14.75 6.603-14.75 14.75S64.054 91.9 72.2 91.9m2.25-21.5c0-1.105-.895-2-2-2s-2 .895-2 2v5h-5c-1.104 0-2 .895-2 2s.896 2 2 2h5v5c0 1.104.896 2 2 2s2-.896 2-2v-5h5c1.105 0 2-.896 2-2s-.895-2-2-2h-5z"
        })
    });
const $f4a317f1acad08e0$var$ForwardRef = /*#__PURE__*/ (0, $5bVFF$forwardRef)($f4a317f1acad08e0$var$SvgComponent);
var $f4a317f1acad08e0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f4a317f1acad08e0$var$ForwardRef);


export {$f4a317f1acad08e0$export$2e2bcd8739ae039 as default};
