// @react-spectrum/s2/illustrations/linear/internal/S2_lin_heart_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6i6fZ$jsx} from "react/jsx-runtime";
import {forwardRef as $6i6fZ$forwardRef} from "react";





const $5736643631ea7f17$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6i6fZ$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6i6fZ$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M5.785 17.714C5.321 11.89 9.513 6.746 15.498 6.75c3.652.002 6.832 2.016 8.502 4.994 1.67-2.978 4.85-4.992 8.502-4.994 5.984-.004 10.176 5.14 9.712 10.963-.403 5.065-3.322 9.902-6.536 13.812-3.238 3.937-6.909 7.094-9.063 8.81-1.533 1.22-3.705 1.22-5.238-.001-2.153-1.716-5.822-4.872-9.058-8.81-3.213-3.91-6.13-8.746-6.534-13.81m9.711-8.464c-4.368-.003-7.58 3.736-7.219 8.265.343 4.299 2.87 8.646 5.973 12.422 3.08 3.749 6.601 6.781 8.685 8.442.622.495 1.5.495 2.123 0 2.084-1.66 5.607-4.693 8.689-8.442 3.105-3.777 5.633-8.124 5.975-12.423.361-4.53-2.85-8.267-7.219-8.264-4 .002-7.25 3.258-7.25 7.282q0 .12-.022.237l-.006.03c-.119.585-.634 1.005-1.23 1.003s-1.108-.426-1.222-1.011l-.005-.024q-.021-.116-.022-.235c0-4.024-3.25-7.28-7.25-7.282m7.269 7.497q0-.001 0 0"
        })
    });
const $5736643631ea7f17$var$ForwardRef = /*#__PURE__*/ (0, $6i6fZ$forwardRef)($5736643631ea7f17$var$SvgComponent);
var $5736643631ea7f17$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5736643631ea7f17$var$ForwardRef);


export {$5736643631ea7f17$export$2e2bcd8739ae039 as default};
