// @react-spectrum/s2/illustrations/linear/internal/S2_lin_megaphonePromote_Expressive_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $jxthb$jsx} from "react/jsx-runtime";
import {forwardRef as $jxthb$forwardRef} from "react";





const $54ad0c9df225b291$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jxthb$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jxthb$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M63.968 20.317c4.72-1.41 9.558 2.061 9.558 7.018v26.132c0 4.967-4.848 8.428-9.564 7.018L42.95 54.198l4.874 10.543.022.05c.964 2.243.898 4.432.055 6.29-.824 1.814-2.322 3.173-4.009 3.98-1.81.865-3.973 1.161-6.015.597-2.095-.58-3.94-2.036-5.028-4.417-.688-1.507-2.855-6.233-4.85-10.584l-2.657-5.793-.626-1.364h-.355C17.032 53.5 11 47.707 11 40.384 11 33.077 17.016 27.2 24.362 27.2h16.535l23.07-6.883zm5.558 7.018c0-2.197-2.18-3.854-4.413-3.186h-.001l-21.913 6.538v19.41l21.909 6.556c2.237.669 4.418-.987 4.418-3.186zM15 40.385c0-5.04 4.165-9.185 9.362-9.185h14.777v18.3H24.362C19.149 49.5 15 45.407 15 40.384M29.118 53.5l2.518 5.49c1.994 4.35 4.162 9.078 4.852 10.589.6 1.311 1.506 1.96 2.456 2.223 1.003.278 2.177.15 3.223-.35.972-.464 1.715-1.193 2.093-2.024.355-.783.45-1.789-.078-3.035L38.222 53.5zm52.411-3.605c-1.053-.334-2.177.25-2.51 1.303s.25 2.177 1.303 2.51l3.45 1.092c1.053.334 2.177-.25 2.51-1.303.334-1.053-.25-2.177-1.303-2.51zm4.754-23.363c.333 1.053-.25 2.177-1.304 2.51l-3.45 1.093c-1.053.333-2.177-.25-2.51-1.304-.333-1.053.25-2.177 1.303-2.51l3.45-1.092c1.053-.333 2.177.25 2.51 1.303m-5.357 11.486c-1.105 0-2 .895-2 2s.895 2 2 2h3.618c1.104 0 2-.896 2-2s-.896-2-2-2z"
        })
    });
const $54ad0c9df225b291$var$ForwardRef = /*#__PURE__*/ (0, $jxthb$forwardRef)($54ad0c9df225b291$var$SvgComponent);
var $54ad0c9df225b291$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($54ad0c9df225b291$var$ForwardRef);


export {$54ad0c9df225b291$export$2e2bcd8739ae039 as default};
