// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lightbulb_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $9yRl7$reactjsxruntime = require("react/jsx-runtime");
var $9yRl7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $adeb5769224ff1c6$export$2e2bcd8739ae039);




const $adeb5769224ff1c6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9yRl7$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9yRl7$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M77.028 19.846C101.052 18.13 121 37.317 121 61.124c0 13.904-6.801 26.212-17.227 33.712-2.065 1.486-3.328 4.012-3.328 6.766v23.907c0 5.611-4.513 10.231-10.167 10.231H69.722c-5.653 0-10.166-4.62-10.166-10.231v-23.913c0-2.728-1.333-5.3-3.586-6.947-11.056-8.084-17.956-21.607-16.855-36.658 1.485-20.295 17.74-36.706 37.913-38.145M113 61.124c0-19.283-16.128-34.674-35.403-33.298-16.147 1.152-29.304 14.356-30.503 30.749-.892 12.193 4.681 23.096 13.598 29.616 4.24 3.1 6.864 8.035 6.864 13.405v9.42q.218-.026.444-.026h24.445v-9.388c0-5.142 2.354-10.165 6.656-13.26C107.51 82.292 113 72.364 113 61.124M92.445 118.49H68q-.226 0-.444-.026v7.045c0 1.271 1.009 2.231 2.166 2.231h20.556c1.158 0 2.167-.96 2.167-2.231zM80.25 40.74c0-2.071 1.68-3.75 3.75-3.75 11.243 0 20.75 8.631 20.75 19.75 0 2.07-1.679 3.75-3.75 3.75s-3.75-1.68-3.75-3.75c0-6.555-5.715-12.25-13.25-12.25-2.07 0-3.75-1.68-3.75-3.75"
        })
    });
const $adeb5769224ff1c6$var$ForwardRef = /*#__PURE__*/ (0, $9yRl7$react.forwardRef)($adeb5769224ff1c6$var$SvgComponent);
var $adeb5769224ff1c6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($adeb5769224ff1c6$var$ForwardRef);
