// @react-spectrum/s2/illustrations/linear/internal/S2_lin_redo_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $hFC50$reactjsxruntime = require("react/jsx-runtime");
var $hFC50$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $48a2a9e3d7469082$export$2e2bcd8739ae039);




const $48a2a9e3d7469082$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hFC50$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hFC50$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M42.384 17.384c.488-.488.488-1.28 0-1.768l-7-7c-.488-.488-1.28-.488-1.768 0s-.488 1.28 0 1.768l4.634 4.634H18.701c-6.907 0-12.506 5.6-12.506 12.506s5.6 12.506 12.506 12.506h7.975c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H18.7c-5.526 0-10.006-4.48-10.006-10.006s4.48-10.006 10.006-10.006h20.013l-5.098 5.098c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0z"
        })
    });
const $48a2a9e3d7469082$var$ForwardRef = /*#__PURE__*/ (0, $hFC50$react.forwardRef)($48a2a9e3d7469082$var$SvgComponent);
var $48a2a9e3d7469082$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($48a2a9e3d7469082$var$ForwardRef);
