// @react-spectrum/s2/illustrations/linear/internal/S2_lin_trophy_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $26vmR$reactjsxruntime = require("react/jsx-runtime");
var $26vmR$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5203e9bef18ebfba$export$2e2bcd8739ae039);




const $5203e9bef18ebfba$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $26vmR$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $26vmR$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M56 34c0-1.1.9-2 2-2h44c1.101 0 2 .9 2 2v30c0 13.24-10.76 24-24 24S56 77.24 56 64V34m56 6v-6c0-5.52-4.481-10-10-10H58c-5.52 0-10 4.48-10 10v6H34c-5.52 0-10 4.48-10 10v2c0 15.46 12.54 28 28 28q.142 0 .282-.01C57.184 88.462 65.859 94.485 76 95.752v16.6c-8.84 1.392-16.15 7.325-19.523 15.309-1.691 3.99 1.275 8.339 5.533 8.339h35.98c4.257 0 7.223-4.347 5.535-8.337-3.374-7.985-10.684-13.919-19.525-15.311v-16.6c10.141-1.267 18.816-7.29 23.718-15.762q.14.01.282.01c15.459 0 28-12.54 28-28v-2c0-5.52-4.481-10-10-10zm0 8v16c0 2.679-.33 5.281-.952 7.769C120.644 70.299 128 62.004 128 52v-2c0-1.1-.899-2-2-2zm-80 4c0 10.005 7.356 18.3 16.952 19.769C48.33 69.28 48 66.679 48 64V48H34c-1.1 0-2 .9-2 2zm62.677 76c-3.135-4.799-8.537-7.96-14.677-7.96s-11.542 3.161-14.677 7.96zM84.61 51.17l-2.29-4.67h.01c-.89-1.81-3.46-1.81-4.35 0l-2.29 4.67c-.32.64-.93 1.09-1.64 1.19l-5.15.74c-1.99.29-2.78 2.74-1.34 4.14l3.73 3.62c.51.5.74 1.22.62 1.92l-.89 5.12c-.34 1.99 1.74 3.5 3.52 2.56l4.6-2.43c.63-.33 1.39-.33 2.02 0l4.6 2.43c1.78.94 3.86-.58 3.52-2.56l-.89-5.12c-.13-.7.11-1.42.62-1.92l3.73-3.62c1.45-1.4.65-3.85-1.34-4.14l-5.15-.74c-.71-.1-1.32-.55-1.64-1.19"
        })
    });
const $5203e9bef18ebfba$var$ForwardRef = /*#__PURE__*/ (0, $26vmR$react.forwardRef)($5203e9bef18ebfba$var$SvgComponent);
var $5203e9bef18ebfba$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5203e9bef18ebfba$var$ForwardRef);
