// @react-spectrum/s2/illustrations/linear/internal/S2_lin_play_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $gLakC$jsx} from "react/jsx-runtime";
import {forwardRef as $gLakC$forwardRef} from "react";





const $e87934b54fa23a71$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gLakC$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gLakC$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M17 47.74c0-17.12 13.88-31 31-31s31 13.88 31 31c0 17.121-13.88 31-31 31s-31-13.879-31-31m31-35c-19.33 0-35 15.67-35 35s15.67 35 35 35 35-15.67 35-35-15.67-35-35-35m-3.775 25.894 12.709 7.29c1.423.816 1.42 2.853-.005 3.666l-12.726 7.26c-1.426.813-3.205-.208-3.203-1.837l.018-14.55c.002-1.63 1.783-2.646 3.207-1.83"
        })
    });
const $e87934b54fa23a71$var$ForwardRef = /*#__PURE__*/ (0, $gLakC$forwardRef)($e87934b54fa23a71$var$SvgComponent);
var $e87934b54fa23a71$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e87934b54fa23a71$var$ForwardRef);


export {$e87934b54fa23a71$export$2e2bcd8739ae039 as default};
