// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lightbulb_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $fsp4B$jsx} from "react/jsx-runtime";
import {forwardRef as $fsp4B$forwardRef} from "react";





const $d774766ca2928412$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fsp4B$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fsp4B$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M45.285 12.889C59.731 11.312 72 22.49 72 36.549c0 7.953-3.935 14.987-9.96 19.304-1.43 1.025-2.262 2.568-2.262 4.185v13.664c0 3.351-2.736 6.038-6.074 6.038H42.296c-3.338 0-6.074-2.687-6.074-6.038h2-2V60.027c0-1.61-.828-3.147-2.254-4.168l1.164-1.626-1.164 1.626c-6.484-4.642-10.548-12.43-9.9-21.117.857-11.506 9.672-20.593 21.217-21.853M68 36.549c0-11.663-10.189-21.004-22.28-19.684-9.645 1.053-16.95 8.606-17.663 18.174-.539 7.22 2.828 13.693 8.24 17.568 2.367 1.695 3.925 4.393 3.925 7.42v6.713h15.556v-6.702c0-3.033 1.56-5.737 3.932-7.436C64.74 48.998 68 43.148 68 36.548M55.778 70.74H40.222v2.962c0 1.11.912 2.038 2.074 2.038h11.408c1.162 0 2.074-.929 2.074-2.038zM49 24.74c0-1.104.895-2 2-2 6.268 0 11 5.576 11 12 0 1.105-.895 2-2 2s-2-.895-2-2c0-4.621-3.327-8-7-8-1.105 0-2-.895-2-2"
        })
    });
const $d774766ca2928412$var$ForwardRef = /*#__PURE__*/ (0, $fsp4B$forwardRef)($d774766ca2928412$var$SvgComponent);
var $d774766ca2928412$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d774766ca2928412$var$ForwardRef);


export {$d774766ca2928412$export$2e2bcd8739ae039 as default};
