// @react-spectrum/s2/illustrations/linear/internal/S2_lin_apps_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $bmbyO$jsx} from "react/jsx-runtime";
import {forwardRef as $bmbyO$forwardRef} from "react";





const $74e1e40270e8c431$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bmbyO$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bmbyO$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.68 26.02c0-3.314 2.686-6 6-6h16.3c3.313 0 6 2.686 6 6v16.3c0 3.313-2.687 6-6 6h-16.3c-3.314 0-6-2.687-6-6zm6-10c-5.523 0-10 4.477-10 10v16.3c0 5.522 4.477 10 10 10h16.3c5.522 0 10-4.478 10-10v-16.3c0-5.523-4.478-10-10-10zM20.01 37.88c0-2.21 1.79-4 4-4h6.44c2.209 0 4 1.79 4 4v6.44c0 2.209-1.791 4-4 4h-6.44c-2.21 0-4-1.791-4-4zm4-8c-4.419 0-8 3.581-8 8v6.44c0 4.418 3.581 8 8 8h6.44c4.418 0 8-3.582 8-8v-6.44c0-4.419-3.582-8-8-8zm0 31.64c-2.21 0-4 1.79-4 4v6.44c0 2.209 1.79 4 4 4h6.44c2.209 0 4-1.791 4-4v-6.44c0-2.21-1.791-4-4-4zm-8 4c0-4.419 3.581-8 8-8h6.44c4.418 0 8 3.581 8 8v6.44c0 4.418-3.582 8-8 8h-6.44c-4.419 0-8-3.582-8-8zm31.67 0c0-2.21 1.79-4 4-4h6.44c2.209 0 4 1.79 4 4v6.44c0 2.209-1.791 4-4 4h-6.44c-2.21 0-4-1.791-4-4zm4-8c-4.419 0-8 3.581-8 8v6.44c0 4.418 3.581 8 8 8h6.44c4.418 0 8-3.582 8-8v-6.44c0-4.419-3.582-8-8-8z"
        })
    });
const $74e1e40270e8c431$var$ForwardRef = /*#__PURE__*/ (0, $bmbyO$forwardRef)($74e1e40270e8c431$var$SvgComponent);
var $74e1e40270e8c431$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($74e1e40270e8c431$var$ForwardRef);


export {$74e1e40270e8c431$export$2e2bcd8739ae039 as default};
