// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowLeft_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $8HbcF$reactjsxruntime = require("react/jsx-runtime");
var $8HbcF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $37abfcba9fa37dbc$export$2e2bcd8739ae039);




const $37abfcba9fa37dbc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8HbcF$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8HbcF$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M136 84c2.209 0 4-1.79 4-4s-1.791-4-4-4H31.157l16.671-16.672c1.563-1.562 1.563-4.094 0-5.656s-4.094-1.562-5.656 0l-23.5 23.5c-1.562 1.562-1.562 4.094 0 5.656l23.5 23.5c1.562 1.563 4.094 1.563 5.656 0 1.563-1.562 1.563-4.094 0-5.656L31.157 84z"
        })
    });
const $37abfcba9fa37dbc$var$ForwardRef = /*#__PURE__*/ (0, $8HbcF$react.forwardRef)($37abfcba9fa37dbc$var$SvgComponent);
var $37abfcba9fa37dbc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($37abfcba9fa37dbc$var$ForwardRef);
