// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noFile_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $3Pxz2$jsx} from "react/jsx-runtime";
import {forwardRef as $3Pxz2$forwardRef} from "react";





const $bffaeb396d394314$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3Pxz2$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3Pxz2$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M6.746 9c0-2.343 1.897-4.25 4.245-4.25h9.725c1.087 0 2.144.416 2.936 1.179l.002.001 9.782 9.372c.835.8 1.31 1.906 1.31 3.068v7.375c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-7h-8.565c-1.782 0-3.435-1.341-3.435-3.24V7.25h-9.255c-.962 0-1.745.783-1.745 1.75v23.995c0 .967.783 1.75 1.745 1.75h12.005c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H10.991c-2.348 0-4.245-1.907-4.245-4.25zm16-.477 8.06 7.722h-7.125c-.628 0-.935-.439-.935-.74zm10.621 34.722c4.073 0 7.375-3.302 7.375-7.375s-3.302-7.375-7.375-7.375-7.375 3.302-7.375 7.375 3.302 7.375 7.375 7.375m1.375-10.75c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v2.25h-2.25c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h2.25v2.25c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-2.25h2.25c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.25z"
        })
    });
const $bffaeb396d394314$var$ForwardRef = /*#__PURE__*/ (0, $3Pxz2$forwardRef)($bffaeb396d394314$var$SvgComponent);
var $bffaeb396d394314$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($bffaeb396d394314$var$ForwardRef);


export {$bffaeb396d394314$export$2e2bcd8739ae039 as default};
