// @react-spectrum/s2/illustrations/linear/internal/S2_lin_movieCamera_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $4gQy5$jsx} from "react/jsx-runtime";
import {forwardRef as $4gQy5$forwardRef} from "react";





const $cb0cb6cf002a0ace$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4gQy5$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4gQy5$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16 57.625c0-11.046 8.954-20 20-20h51.954c11.046 0 20 8.954 20 20V61.3l16.895-12.537c7.918-5.876 19.151-.224 19.151 9.636v43.04c0 9.836-11.183 15.495-19.108 9.669l-16.938-12.453v3.72c0 11.046-8.954 20-20 20H36c-11.046 0-20-8.954-20-20zm20-12c-6.627 0-12 5.373-12 12v44.75c0 6.627 5.373 12 12 12h51.954c6.628 0 12-5.373 12-12V90.75c0-1.508.849-2.888 2.194-3.57 1.346-.68 2.96-.546 4.176.347l23.307 17.135c2.641 1.942 6.369.056 6.369-3.223v-43.04c0-3.287-3.744-5.17-6.384-3.212l-23.278 17.275c-1.214.9-2.831 1.04-4.182.361-1.35-.68-2.202-2.062-2.202-3.573V57.625c0-6.627-5.372-12-12-12z"
        })
    });
const $cb0cb6cf002a0ace$var$ForwardRef = /*#__PURE__*/ (0, $4gQy5$forwardRef)($cb0cb6cf002a0ace$var$SvgComponent);
var $cb0cb6cf002a0ace$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cb0cb6cf002a0ace$var$ForwardRef);


export {$cb0cb6cf002a0ace$export$2e2bcd8739ae039 as default};
