// @react-spectrum/s2/illustrations/linear/internal/S2_lin_folderShared_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $a7ZcY$reactjsxruntime = require("react/jsx-runtime");
var $a7ZcY$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0ab04493f2beaf4d$export$2e2bcd8739ae039);




const $0ab04493f2beaf4d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $a7ZcY$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $a7ZcY$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M24 37.2c0-2.84 2.331-5.2 5.28-5.2h25.7c1.617 0 3.136.73 4.133 1.965L69.633 47H24zM16 47v-9.8C16 29.877 21.978 24 29.28 24h25.7c4.02 0 7.834 1.812 10.36 4.94l-3.043 2.456 3.042-2.455L79.912 47h46.242C136.031 47 144 55.063 144 64.941v53.118c0 9.877-7.969 17.941-17.846 17.941H33.846C23.97 136 16 127.936 16 118.059V47m8 8v63.059c0 5.511 4.44 9.941 9.846 9.941h92.308c5.407 0 9.846-4.43 9.846-9.941V64.941c0-5.51-4.439-9.941-9.846-9.941H24"
            }),
            /*#__PURE__*/ (0, $a7ZcY$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M85.048 87.06c2.707 0 4.902-2.195 4.902-4.903s-2.195-4.901-4.902-4.901-4.902 2.194-4.902 4.901c0 2.708 2.195 4.903 4.902 4.903m0 19.607h-8.915c-1.201 0-2.078-.993-1.825-2.04 1.063-4.413 5.498-7.764 10.74-7.764s9.678 3.351 10.741 7.764c.252 1.047-.624 2.04-1.825 2.04z"
            }),
            /*#__PURE__*/ (0, $a7ZcY$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "m74.308 104.627 3.888.937zm21.481 0-3.889.937zm-37.233-3.236c-.37 2.178 1.097 4.243 3.275 4.613s4.243-1.096 4.613-3.274zm11.669-3.767c1.993-.952 2.837-3.34 1.885-5.333s-3.34-2.838-5.334-1.886zm2.07-20.29c2.208 0 4-1.79 4-4s-1.792-4-4-4zm-6.577 8.248c1.587 1.537 4.12 1.496 5.656-.091 1.537-1.587 1.496-4.12-.091-5.656zm20.232-3.424c0 .498-.404.901-.902.901v8c4.916 0 8.902-3.985 8.902-8.901zm-.902-.903c.498 0 .902.404.902.903h8c0-4.917-3.986-8.903-8.902-8.903zm-.902.903c0-.499.404-.903.902-.903v-8c-4.916 0-8.902 3.986-8.902 8.903zm.902.901c-.498 0-.902-.403-.902-.901h-8c0 4.916 3.986 8.901 8.902 8.901zm-8.915 27.608h8.915v-8h-8.915zm-5.714-6.977c-.978 4.057 2.457 6.977 5.714 6.977v-8c.513 0 1.09.219 1.53.713.45.507.742 1.317.533 2.184zm14.63-10.827c-6.847 0-13.08 4.39-14.63 10.827l7.777 1.874c.576-2.39 3.215-4.701 6.852-4.701zm14.629 10.827c-1.551-6.436-7.783-10.827-14.63-10.827v8c3.637 0 6.277 2.311 6.852 4.701zm-5.714 6.977c3.257 0 6.692-2.92 5.714-6.977l-7.778 1.874c-.208-.867.083-1.677.534-2.184.44-.494 1.017-.713 1.53-.713zm-8.916 0h8.916v-8h-8.916zm-18.604-7.937c.39-2.302 1.874-4.195 3.78-5.106l-3.448-7.219c-4.274 2.042-7.397 6.14-8.22 10.986zm4.395-23.941c0-.804.652-1.455 1.455-1.455v-8c-5.222 0-9.455 4.233-9.455 9.455zm.444 1.046c-.279-.27-.444-.635-.444-1.046h-8c0 2.666 1.108 5.08 2.879 6.793z"
            })
        ]
    });
const $0ab04493f2beaf4d$var$ForwardRef = /*#__PURE__*/ (0, $a7ZcY$react.forwardRef)($0ab04493f2beaf4d$var$SvgComponent);
var $0ab04493f2beaf4d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0ab04493f2beaf4d$var$ForwardRef);
