// @react-spectrum/s2/illustrations/linear/internal/S2_lin_emptyStateExport_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $enmxr$reactjsxruntime = require("react/jsx-runtime");
var $enmxr$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4c6c518418cdc426$export$2e2bcd8739ae039);




const $4c6c518418cdc426$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $enmxr$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $enmxr$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M62.481 18c0-1.105.896-2 2-2h12.4c1.105 0 2 .895 2 2v26c0 1.105-.895 2-2 2h-12.4c-1.104 0-2-.895-2-2zm2-6c-3.313 0-6 2.686-6 6v26c0 3.314 2.687 6 6 6h12.4c3.314 0 6-2.686 6-6V18c0-3.314-2.686-6-6-6zm-4.6 51.6c1.104 0 2 .895 2 2V78c0 1.105-.896 2-2 2h-26c-1.105 0-2-.895-2-2V65.6c0-1.105.895-2 2-2zm6 2c0-3.314-2.686-6-6-6h-26c-3.314 0-6 2.686-6 6V78c0 3.314 2.686 6 6 6h26c3.314 0 6-2.686 6-6zM19.67 45.43c-1.408 0-2.55 1.142-2.55 2.55s1.142 2.55 2.55 2.55 2.55-1.142 2.55-2.55-1.142-2.55-2.55-2.55m-6.55 2.55c0-2.923 1.915-5.4 4.56-6.242V27.022c-2.645-.842-4.56-3.318-4.56-6.242 0-3.617 2.933-6.55 6.55-6.55 2.927 0 5.406 1.92 6.246 4.57h14.709c.84-2.65 3.318-4.57 6.245-4.57 3.618 0 6.55 2.933 6.55 6.55 0 2.916-1.906 5.388-4.54 6.236V37.02c0 1.105-.896 2-2 2s-2-.895-2-2v-9.998c-2.007-.64-3.594-2.22-4.242-4.222H25.904c-.647 1.996-2.226 3.573-4.223 4.216v14.728c2.01.648 3.597 2.24 4.235 4.256h10.035c1.105 0 2 .895 2 2s-.895 2-2 2H25.904c-.852 2.629-3.32 4.53-6.233 4.53-3.617 0-6.55-2.933-6.55-6.55m4-27.2c0-1.408 1.142-2.55 2.55-2.55s2.55 1.142 2.55 2.55-1.142 2.55-2.55 2.55-2.55-1.142-2.55-2.55m29.75-2.55c-1.408 0-2.55 1.142-2.55 2.55s1.142 2.55 2.55 2.55c1.409 0 2.55-1.142 2.55-2.55s-1.141-2.55-2.55-2.55m-2.86 36.91-5.16-12.57h.01c-.68-1.64.97-3.29 2.61-2.61l12.56 5.16c1.62.67 1.66 2.96.05 3.68l-3.72 1.66c-.45.2-.81.56-1.01 1.01l-1.66 3.72c-.72 1.61-3.01 1.58-3.68-.05"
        })
    });
const $4c6c518418cdc426$var$ForwardRef = /*#__PURE__*/ (0, $enmxr$react.forwardRef)($4c6c518418cdc426$var$SvgComponent);
var $4c6c518418cdc426$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4c6c518418cdc426$var$ForwardRef);
