// @react-spectrum/s2/illustrations/linear/internal/S2_lin_heart_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $eFR2v$reactjsxruntime = require("react/jsx-runtime");
var $eFR2v$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c240ae7f4b7f80d2$export$2e2bcd8739ae039);




const $c240ae7f4b7f80d2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eFR2v$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $eFR2v$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 25.513c3.294-6.587 10.308-10.992 18.287-10.471 10.742.7 18.397 10.428 17.664 20.949-.716 10.284-6.792 20.156-13.454 28.094-6.704 7.989-14.237 14.297-18.305 17.471-2.468 1.926-5.931 1.925-8.398 0-4.067-3.176-11.597-9.484-18.298-17.472-6.658-7.938-12.731-17.81-13.448-28.093-.732-10.52 6.921-20.247 17.663-20.95 7.98-.52 14.995 3.885 18.289 10.472m18.027-6.48c-8.752-.571-16.02 6.345-16.02 14.951q0 .194-.037.384l-.01.054c-.19.935-1.014 1.605-1.968 1.601-.953-.004-1.772-.68-1.955-1.616l-.008-.042q-.036-.188-.037-.381c0-8.607-7.268-15.523-16.02-14.95-8.329.544-14.526 8.161-13.933 16.68.627 9.002 6.032 18.062 12.522 25.8 6.448 7.687 13.737 13.799 17.694 16.888 1.02.797 2.455.797 3.476 0 3.959-3.089 11.251-9.2 17.702-16.888 6.493-7.738 11.9-16.798 12.528-25.801.593-8.518-5.606-16.136-13.934-16.68"
        })
    });
const $c240ae7f4b7f80d2$var$ForwardRef = /*#__PURE__*/ (0, $eFR2v$react.forwardRef)($c240ae7f4b7f80d2$var$SvgComponent);
var $c240ae7f4b7f80d2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c240ae7f4b7f80d2$var$ForwardRef);
