// @react-spectrum/s2/illustrations/linear/internal/S2_lin_volumeOff_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9laPS$jsx} from "react/jsx-runtime";
import {forwardRef as $9laPS$forwardRef} from "react";





const $b77c67396a6390c1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9laPS$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9laPS$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M13.57 10.742c-.78-.781-2.047-.781-2.828 0s-.781 2.047 0 2.828l71.69 71.69c.78.781 2.047.781 2.828 0s.781-2.047 0-2.829zm29.344 7.195c3.156-3.129 8.52-.893 8.52 3.551v18.2c0 1.105-.895 2-2 2s-2-.895-2-2v-18.2c0-.889-1.072-1.336-1.703-.71l-7.609 7.544c-.784.777-2.05.772-2.828-.013-.778-.784-.773-2.05.012-2.828zM21.466 34.08c-1.105 0-2 .896-2 2v24.022c0 1.105.895 2 2 2h8.39q.15 0 .297.008v-20.17c0-1.105.896-2 2-2s2 .895 2 2v21.973l11.581 11.367c.633.62 1.7.172 1.7-.714V59.251c0-1.105.896-2 2-2s2 .895 2 2v15.314c0 4.432-5.339 6.673-8.502 3.568L31.257 66.674c-.374-.367-.877-.573-1.4-.573h-8.391c-3.314 0-6-2.686-6-6V36.08c0-3.313 2.686-6 6-6h.785c1.104 0 2 .896 2 2s-.896 2-2 2zm34.945.72c.664-.883 1.918-1.06 2.8-.396 4.11 3.093 6.774 8.018 6.774 13.565 0 2.08-.375 4.077-1.063 5.924-.386 1.035-1.538 1.561-2.573 1.175s-1.561-1.537-1.175-2.572c.524-1.407.811-2.931.811-4.527 0-4.237-2.03-8-5.178-10.369-.883-.664-1.06-1.918-.396-2.8m10.005-9.13c.664-.882 1.918-1.06 2.8-.395 6.87 5.17 11.32 13.4 11.32 22.668 0 6.282-2.045 12.092-5.506 16.793-.655.89-1.907 1.08-2.796.425-.89-.655-1.08-1.907-.425-2.796 2.972-4.037 4.727-9.021 4.727-14.422 0-7.958-3.815-15.025-9.725-19.472-.883-.664-1.06-1.918-.395-2.8"
        })
    });
const $b77c67396a6390c1$var$ForwardRef = /*#__PURE__*/ (0, $9laPS$forwardRef)($b77c67396a6390c1$var$SvgComponent);
var $b77c67396a6390c1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b77c67396a6390c1$var$ForwardRef);


export {$b77c67396a6390c1$export$2e2bcd8739ae039 as default};
