// @react-spectrum/s2/illustrations/linear/internal/S2_lin_userAvatar_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $kAf0Z$jsx} from "react/jsx-runtime";
import {forwardRef as $kAf0Z$forwardRef} from "react";





const $7fea615c5a3b4546$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kAf0Z$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kAf0Z$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 28c-28.719 0-52 23.281-52 52 0 13.336 5.02 25.5 13.274 34.704 7.948-13.106 22.365-21.956 38.728-21.956 16.362 0 30.777 8.857 38.724 21.955C126.98 105.499 132 93.336 132 80c0-28.719-23.281-52-52-52m0 104c-12.42 0-23.822-4.354-32.765-11.619 6.342-11.669 18.72-19.633 32.767-19.633 14.043 0 26.419 7.968 32.763 19.633C103.823 127.646 92.42 132 80 132M20 80c0-33.137 26.863-60 60-60s60 26.863 60 60-26.863 60-60 60-60-26.863-60-60m43.489-17.164c0-10.103 7.628-17.854 16.512-17.854s16.512 7.751 16.512 17.854S88.887 80.69 80.002 80.69 63.49 72.939 63.49 62.836M80 36.982c-13.773 0-24.512 11.818-24.512 25.854S66.229 88.69 80 88.69s24.512-11.818 24.512-25.854-10.74-25.854-24.512-25.854"
        })
    });
const $7fea615c5a3b4546$var$ForwardRef = /*#__PURE__*/ (0, $kAf0Z$forwardRef)($7fea615c5a3b4546$var$SvgComponent);
var $7fea615c5a3b4546$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7fea615c5a3b4546$var$ForwardRef);


export {$7fea615c5a3b4546$export$2e2bcd8739ae039 as default};
