// @react-spectrum/s2/illustrations/linear/internal/S2_lin_animation_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $a4pCN$reactjsxruntime = require("react/jsx-runtime");
var $a4pCN$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c7fc7998e0ec6c40$export$2e2bcd8739ae039);




const $c7fc7998e0ec6c40$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $a4pCN$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $a4pCN$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M37.516 8.36c.222-.414.656-.692 1.154-.692.722 0 1.307.585 1.307 1.307 0 .269-.08.516-.217.722-.345.518-.204 1.216.313 1.56s1.215.204 1.56-.313c.375-.564.594-1.243.594-1.97 0-1.964-1.592-3.556-3.557-3.556-1.36 0-2.54.764-3.138 1.882-.293.548-.086 1.23.462 1.522.547.293 1.23.086 1.522-.462m-3.075 4.083c-1.136 0-2.124.636-2.627 1.576-.326.61-1.083.839-1.692.514-.609-.326-.838-1.083-.513-1.692.92-1.722 2.738-2.898 4.832-2.898 3.024 0 5.476 2.452 5.476 5.477 0 1.118-.337 2.162-.915 3.03-.382.575-1.158.731-1.733.349-.575-.383-.73-1.159-.348-1.734.313-.47.496-1.035.496-1.646 0-1.643-1.332-2.976-2.976-2.976m-10.085 7.743c.934-1.131 2.343-1.848 3.92-1.848 2.808 0 5.085 2.276 5.085 5.084 0 2.22-1.425 4.112-3.414 4.803-.652.227-.996.94-.77 1.592.227.652.94.996 1.592.77 2.962-1.03 5.092-3.848 5.092-7.165 0-4.189-3.396-7.584-7.584-7.584-2.355 0-4.46 1.074-5.848 2.755-.44.532-.365 1.32.167 1.76s1.32.365 1.76-.168M8.146 31.994c0-4.537 3.678-8.215 8.215-8.215s8.215 3.678 8.215 8.215-3.678 8.215-8.215 8.215-8.215-3.678-8.215-8.215m8.215-10.715c-5.917 0-10.715 4.797-10.715 10.715S10.444 42.71 16.361 42.71c5.918 0 10.715-4.797 10.715-10.715 0-5.917-4.797-10.715-10.715-10.715"
        })
    });
const $c7fc7998e0ec6c40$var$ForwardRef = /*#__PURE__*/ (0, $a4pCN$react.forwardRef)($c7fc7998e0ec6c40$var$SvgComponent);
var $c7fc7998e0ec6c40$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c7fc7998e0ec6c40$var$ForwardRef);
