// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noLibrariesBrands_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $d0NV5$jsx} from "react/jsx-runtime";
import {forwardRef as $d0NV5$forwardRef} from "react";





const $f8aea15d0b1d6d81$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $d0NV5$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $d0NV5$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M13.93 17C12.343 17 11 18.32 11 20.006v20.702C11 50.28 18.65 58 28 58c9.355 0 17-7.722 17-17.292V20.006C45 18.32 43.657 17 42.07 17zM7 20.006C7 16.169 10.077 13 13.93 13h28.14c3.853 0 6.93 3.169 6.93 7.006v20.702C49 52.428 39.621 62 28 62 16.383 62 7 52.433 7 40.708zm28.633 8.948 4.93 9.505v.001c.621 1.2.542 2.542.001 3.587C40.021 43.097 38.911 44 37.43 44h-9.86c-1.481 0-2.59-.904-3.134-1.953-.541-1.045-.62-2.387.001-3.587l4.926-9.506C30 27.727 31.201 27 32.498 27s2.499.727 3.135 1.954m-3.135 2.644L28.144 40h8.712zm-8.677.23c.457-1.006.012-2.192-.993-2.649-1.006-.457-2.192-.012-2.649.993l-5 11c-.457 1.006-.012 2.192.993 2.649 1.006.457 2.192.012 2.649-.993zM54.5 26.796c-1.105 0-2 .896-2 2s.895 2 2 2h3.919v18.95c0 .692.77 1.107 1.341.718l4.214-2.812c.294-.19.667-.19.96 0l4.215 2.812c.57.38 1.341-.026 1.341-.718v-18.95h.336c2.719 0 4.923 2.204 4.923 4.923v17.174c0 1.105.895 2 2 2s2-.895 2-2V35.719c0-4.928-3.995-8.923-8.923-8.923H54.5M34.796 66.273c0-1.105-.895-2-2-2s-2 .895-2 2v.554c0 4.928 3.995 8.922 8.922 8.922h18.04c1.105 0 2-.895 2-2s-.895-2-2-2h-18.04c-2.718 0-4.922-2.203-4.922-4.922zM77.5 87.5c8.146 0 14.75-6.604 14.75-14.75S85.646 58 77.5 58s-14.75 6.604-14.75 14.75S69.354 87.5 77.5 87.5M79.75 66c0-1.105-.895-2-2-2s-2 .895-2 2v5h-5c-1.104 0-2 .895-2 2s.896 2 2 2h5v5c0 1.105.896 2 2 2s2-.895 2-2v-5h5c1.105 0 2-.895 2-2s-.895-2-2-2h-5z"
        })
    });
const $f8aea15d0b1d6d81$var$ForwardRef = /*#__PURE__*/ (0, $d0NV5$forwardRef)($f8aea15d0b1d6d81$var$SvgComponent);
var $f8aea15d0b1d6d81$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f8aea15d0b1d6d81$var$ForwardRef);


export {$f8aea15d0b1d6d81$export$2e2bcd8739ae039 as default};
