// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cloudUpload_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $Zpm8a$jsx} from "react/jsx-runtime";
import {forwardRef as $Zpm8a$forwardRef} from "react";





const $72ada17c2f4954c4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $Zpm8a$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $Zpm8a$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M17.281 12.29c1.614-3.64 5.153-6.21 9.318-6.21 5.412 0 9.77 4.33 10.213 9.743 4.259.799 7.447 4.557 7.447 9.307 0 5.168-4.029 9.46-9.11 9.46h-.04c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h.039c3.6 0 6.61-3.068 6.61-6.96 0-3.895-2.79-6.744-6.235-6.952-.66-.04-1.174-.587-1.174-1.248v-.199c0-4.552-3.521-8.15-7.75-8.15-3.414 0-6.353 2.335-7.376 5.639-.105.338-.348.616-.67.765-.322.148-.691.154-1.017.014-.624-.268-1.307-.419-2.007-.419-2.89 0-5.32 2.459-5.32 5.61 0 .436.05.866.14 1.281.142.645-.24 1.288-.875 1.47-1.851.533-3.255 2.313-3.255 4.49 0 2.592 1.975 4.606 4.32 4.65h2.39c.69 0 1.25.559 1.25 1.25 0 .69-.56 1.25-1.25 1.25h-2.4l-.02-.001c-3.811-.062-6.79-3.287-6.79-7.15 0-2.865 1.63-5.357 4.023-6.497q-.033-.368-.033-.743c0-4.429 3.449-8.11 7.82-8.11.601 0 1.19.074 1.752.21m7.969 10.728 4.866 4.866c.488.488 1.28.488 1.768 0s.488-1.28 0-1.768l-7-7c-.488-.488-1.28-.488-1.768 0l-7 7c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0l4.866-4.866V40.76c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25z"
        })
    });
const $72ada17c2f4954c4$var$ForwardRef = /*#__PURE__*/ (0, $Zpm8a$forwardRef)($72ada17c2f4954c4$var$SvgComponent);
var $72ada17c2f4954c4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($72ada17c2f4954c4$var$ForwardRef);


export {$72ada17c2f4954c4$export$2e2bcd8739ae039 as default};
