// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cut_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5ixQF$jsx} from "react/jsx-runtime";
import {forwardRef as $5ixQF$forwardRef} from "react";





const $382641d29adfd366$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5ixQF$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5ixQF$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M59.312 14.847c1.045.357 1.602 1.495 1.245 2.54l-8.849 25.846 26.64-9.072c1.046-.356 2.182.203 2.538 1.249.356 1.045-.203 2.181-1.249 2.537l-28.776 9.8q-.017.12-.048.238c-.187.712-.353 1.43-.522 2.163-.505 2.189-1.04 4.499-2.257 7.127-.328.708-.699 1.427-1.092 2.137q.507-.047 1.025-.047c6.047 0 10.948 4.901 10.948 10.948 0 6.046-4.901 10.948-10.948 10.948-6.046 0-10.948-4.902-10.948-10.948 0-2.38.76-4.584 2.05-6.38q.105-.177.25-.335.227-.29.471-.565c1.084-1.381 3.273-4.543 4.615-7.44 1.03-2.222 1.438-3.988 1.916-6.056l.09-.385c-1.069.384-2.215.832-3.66 1.427-1.687.695-3.647 1.792-5.34 2.847-1.71 1.066-3.007 2-3.386 2.321q-.121.103-.253.184c-1.987 2.025-4.755 3.281-7.816 3.281-6.047 0-10.948-4.901-10.948-10.948 0-6.046 4.901-10.948 10.948-10.948 6.046 0 10.948 4.902 10.948 10.948q0 .413-.03.817c1.402-.813 2.927-1.613 4.353-2.2 2.361-.973 3.988-1.58 5.738-2.142l9.807-28.648c.358-1.045 1.495-1.602 2.54-1.244M42.408 66.145c-.872 1.16-1.389 2.604-1.389 4.168 0 3.837 3.11 6.948 6.948 6.948 3.837 0 6.948-3.11 6.948-6.948 0-3.837-3.11-6.948-6.948-6.948-2.035 0-3.866.875-5.137 2.27q-.237.296-.422.51M25.956 41.316c-3.837 0-6.948 3.11-6.948 6.948 0 3.837 3.11 6.948 6.948 6.948 3.837 0 6.948-3.11 6.948-6.948 0-3.837-3.11-6.948-6.948-6.948"
        })
    });
const $382641d29adfd366$var$ForwardRef = /*#__PURE__*/ (0, $5ixQF$forwardRef)($382641d29adfd366$var$SvgComponent);
var $382641d29adfd366$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($382641d29adfd366$var$ForwardRef);


export {$382641d29adfd366$export$2e2bcd8739ae039 as default};
