// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pencil_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $bc50t$jsx} from "react/jsx-runtime";
import {forwardRef as $bc50t$forwardRef} from "react";





const $8675ca876d15c0e9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bc50t$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bc50t$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M21.898 57.87c.301-.803.77-1.533 1.377-2.139l37.21-37.21c4.694-4.694 12.304-4.694 16.997 0 4.694 4.692 4.694 12.302 0 16.996l-37.21 37.21c-.607.607-1.337 1.076-2.14 1.377l-17.72 6.635c-3.215 1.203-6.353-1.934-5.149-5.149zm5.049-.154 11.34 11.34 30.698-30.7-11.34-11.339zm7.747 13.404-9.812-9.812-5.873 15.685zm39.96-38.432-2.84 2.84-11.34-11.34 2.84-2.84c3.131-3.13 8.208-3.13 11.34 0 3.131 3.132 3.131 8.209 0 11.34"
        })
    });
const $8675ca876d15c0e9$var$ForwardRef = /*#__PURE__*/ (0, $bc50t$forwardRef)($8675ca876d15c0e9$var$SvgComponent);
var $8675ca876d15c0e9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8675ca876d15c0e9$var$ForwardRef);


export {$8675ca876d15c0e9$export$2e2bcd8739ae039 as default};
