// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileZip_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $e4maQ$reactjsxruntime = require("react/jsx-runtime");
var $e4maQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c77ebacc02568dc3$export$2e2bcd8739ae039);




const $c77ebacc02568dc3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e4maQ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $e4maQ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M29.3 15.75h18.199q.1-.01.2-.01c2.305 0 4.521.888 6.185 2.487l.002.001 19.006 18.187c1.778 1.7 2.778 4.064 2.778 6.515v27.8c0 4.957-4.008 8.99-8.96 8.99H29.3c-4.952 0-8.96-4.033-8.96-8.99V24.74c0-4.957 4.008-8.99 8.96-8.99m18.4 4q.086 0 .17-.007c1.213.042 2.369.528 3.244 1.37l.003.002 19.01 18.19c.983.94 1.543 2.257 1.543 3.625v27.8c0 2.763-2.232 4.99-4.96 4.99H29.3c-2.728 0-4.96-2.227-4.96-4.99V24.74c0-2.763 2.232-4.99 4.96-4.99zm-2.04 46.72v-5.9h4.09v5.9zm-1.72-13.9c-1.31 0-2.28 1.072-2.28 2.28v11.62c0 2.221 1.802 4 4 4h4.09c2.221 0 4-1.802 4-4V54.85c0-1.31-1.072-2.28-2.28-2.28h-1.78v-8.68h3.23c1.104 0 2-.895 2-2s-.896-2-2-2h-3.23v-4.45h3.23c1.104 0 2-.895 2-2s-.896-2-2-2h-3.23v-2.22c0-1.104-.896-2-2-2h-4.6c-1.105 0-2 .896-2 2s.895 2 2 2h2.6v4.44h-2.6c-1.105 0-2 .896-2 2s.895 2 2 2h2.6v4.45h-2.6c-1.105 0-2 .896-2 2s.895 2 2 2h2.6v4.46z"
        })
    });
const $c77ebacc02568dc3$var$ForwardRef = /*#__PURE__*/ (0, $e4maQ$react.forwardRef)($c77ebacc02568dc3$var$SvgComponent);
var $c77ebacc02568dc3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c77ebacc02568dc3$var$ForwardRef);
