// @react-spectrum/s2/illustrations/linear/internal/S2_lin_video_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $2ur7e$jsx} from "react/jsx-runtime";
import {forwardRef as $2ur7e$forwardRef} from "react";





const $a262fe8d24ef0d18$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2ur7e$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2ur7e$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M40.75 14.2c0-1.52-1.231-2.75-2.75-2.75H10c-1.519 0-2.75 1.23-2.75 2.75v19.6c0 1.518 1.231 2.75 2.75 2.75h28c1.519 0 2.75-1.232 2.75-2.75zM38 8.95c2.9 0 5.25 2.35 5.25 5.25v19.6c0 2.899-2.35 5.25-5.25 5.25H10c-2.9 0-5.25-2.351-5.25-5.25V14.2c0-2.9 2.35-5.25 5.25-5.25zM21.611 19.142l6.355 3.645c.712.408.71 1.427-.002 1.833l-6.364 3.63c-.712.406-1.602-.104-1.601-.919l.009-7.274c0-.815.892-1.323 1.603-.915"
        })
    });
const $a262fe8d24ef0d18$var$ForwardRef = /*#__PURE__*/ (0, $2ur7e$forwardRef)($a262fe8d24ef0d18$var$SvgComponent);
var $a262fe8d24ef0d18$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a262fe8d24ef0d18$var$ForwardRef);


export {$a262fe8d24ef0d18$export$2e2bcd8739ae039 as default};
