// @react-spectrum/s2/illustrations/linear/internal/S2_lin_speedFast_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $bqZ4c$jsx} from "react/jsx-runtime";
import {forwardRef as $bqZ4c$forwardRef} from "react";





const $d2dfdaa664e52055$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bqZ4c$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bqZ4c$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28.047 123.972c-4.939-8.532-7.835-18.394-8.021-28.907Q20 94.841 20 94.61q0-.185.017-.367v-.26c0-33.108 26.838-59.947 59.945-59.947 13.68 0 26.301 4.588 36.391 12.306 1.754 1.342 2.089 3.853.747 5.607s-3.853 2.09-5.608.747q-.919-.703-1.869-1.365l-4.148 7.184c-1.105 1.914-3.551 2.57-5.464 1.464-1.913-1.104-2.569-3.55-1.464-5.464l4.18-7.24c-5.757-2.81-12.11-4.588-18.823-5.092v8.427c0 2.21-1.79 4-4 4s-4-1.79-4-4v-8.418c-6.651.514-12.952 2.281-18.666 5.065l4.19 7.258c1.105 1.914.45 4.36-1.463 5.464s-4.36.45-5.464-1.464l-4.163-7.21c-5.327 3.705-9.927 8.381-13.544 13.773l7.017 4.05c1.913 1.105 2.569 3.552 1.464 5.465s-3.55 2.569-5.464 1.464l-6.95-4.012c-2.66 5.701-4.314 11.966-4.737 18.565h7.841c2.21 0 4 1.791 4 4s-1.79 4-4 4H28.22c.682 7.734 3.058 14.973 6.751 21.355 1.107 1.912.454 4.359-1.458 5.466-1.912 1.106-4.359.453-5.465-1.458M137.32 76.509c-.643-2.113-2.878-3.305-4.991-2.662-2.114.643-3.306 2.878-2.663 4.99 1.144 3.759 1.873 7.7 2.133 11.772h-7.857c-2.209 0-4 1.791-4 4s1.791 4 4 4h7.762c-.682 7.734-3.057 14.973-6.751 21.355-1.107 1.912-.454 4.359 1.458 5.466 1.912 1.106 4.359.453 5.465-1.458 4.977-8.597 7.879-18.544 8.025-29.146q.006-.108.006-.217l-.002-.127.002-.5c0-6.072-.904-11.94-2.587-17.473m-3.637-23.251c-1.998-1.999-5.212-2.083-7.313-.192l-51.856 46.68c-3.823 3.442-3.979 9.385-.341 13.022s9.58 3.482 13.021-.341l46.681-51.857c1.891-2.1 1.807-5.314-.192-7.312"
        })
    });
const $d2dfdaa664e52055$var$ForwardRef = /*#__PURE__*/ (0, $bqZ4c$forwardRef)($d2dfdaa664e52055$var$SvgComponent);
var $d2dfdaa664e52055$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d2dfdaa664e52055$var$ForwardRef);


export {$d2dfdaa664e52055$export$2e2bcd8739ae039 as default};
