// @react-spectrum/s2/illustrations/linear/internal/S2_lin_userAvatar_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $kqge1$reactjsxruntime = require("react/jsx-runtime");
var $kqge1$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ca03dc11be1c4fb4$export$2e2bcd8739ae039);




const $ca03dc11be1c4fb4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kqge1$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kqge1$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 28c-28.719 0-52 23.281-52 52 0 13.336 5.02 25.5 13.274 34.704 7.948-13.106 22.365-21.956 38.728-21.956 16.362 0 30.777 8.857 38.724 21.955C126.98 105.499 132 93.336 132 80c0-28.719-23.281-52-52-52m0 104c-12.42 0-23.822-4.354-32.765-11.619 6.342-11.669 18.72-19.633 32.767-19.633 14.043 0 26.419 7.968 32.763 19.633C103.823 127.646 92.42 132 80 132M20 80c0-33.137 26.863-60 60-60s60 26.863 60 60-26.863 60-60 60-60-26.863-60-60m43.489-17.164c0-10.103 7.628-17.854 16.512-17.854s16.512 7.751 16.512 17.854S88.887 80.69 80.002 80.69 63.49 72.939 63.49 62.836M80 36.982c-13.773 0-24.512 11.818-24.512 25.854S66.229 88.69 80 88.69s24.512-11.818 24.512-25.854-10.74-25.854-24.512-25.854"
        })
    });
const $ca03dc11be1c4fb4$var$ForwardRef = /*#__PURE__*/ (0, $kqge1$react.forwardRef)($ca03dc11be1c4fb4$var$SvgComponent);
var $ca03dc11be1c4fb4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ca03dc11be1c4fb4$var$ForwardRef);
