// @react-spectrum/s2/illustrations/linear/internal/S2_lin_chatbubble_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8sqQt$jsx} from "react/jsx-runtime";
import {forwardRef as $8sqQt$forwardRef} from "react";





const $d67523c50b0524d2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8sqQt$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8sqQt$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M10 26.5C10 20.615 14.802 16 20.64 16h54.72C81.197 16 86 20.615 86 26.5v28C86 60.385 81.198 65 75.36 65H52.739L34.072 80.963l-.003.003c-3.153 2.712-8.1.493-8.069-3.715.027-3.722.02-8.435.008-12.251H20.64C14.801 65 10 60.385 10 54.5zM20.64 20C16.935 20 14 22.9 14 26.5v28c0 3.6 2.935 6.5 6.64 6.5h9.352L30 62.992c.017 4.051.032 9.86 0 14.289-.005.717.87 1.162 1.461.652l.006-.005L50.7 61.48l.561-.48H75.36c3.705 0 6.64-2.9 6.64-6.5v-28c0-3.6-2.935-6.5-6.64-6.5z"
        })
    });
const $d67523c50b0524d2$var$ForwardRef = /*#__PURE__*/ (0, $8sqQt$forwardRef)($d67523c50b0524d2$var$SvgComponent);
var $d67523c50b0524d2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d67523c50b0524d2$var$ForwardRef);


export {$d67523c50b0524d2$export$2e2bcd8739ae039 as default};
