// @react-spectrum/s2/illustrations/linear/internal/S2_lin_flagCross_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $aHTyQ$reactjsxruntime = require("react/jsx-runtime");
var $aHTyQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c705d81dab6c3de9$export$2e2bcd8739ae039);




const $c705d81dab6c3de9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aHTyQ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $aHTyQ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M12.957 9.11c0-.691-.56-1.25-1.25-1.25s-1.25.559-1.25 1.25v29.78c0 .691.56 1.25 1.25 1.25s1.25-.559 1.25-1.25v-7.64h20.336c2.347 0 4.25-1.902 4.25-4.25V14.022c0-2.348-1.903-4.25-4.25-4.25H12.957zm0 3.162V28.75h20.336c.967 0 1.75-.783 1.75-1.75V14.022c0-.967-.783-1.75-1.75-1.75zm7.06 4.127c.488-.488 1.28-.488 1.768 0l2.344 2.344 2.344-2.344c.488-.488 1.28-.488 1.768 0s.488 1.28 0 1.768l-2.344 2.344 2.344 2.344c.488.488.488 1.28 0 1.768-.489.488-1.28.488-1.768 0l-2.344-2.344-2.344 2.344c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.768l2.344-2.344-2.344-2.344c-.488-.488-.488-1.28 0-1.768"
        })
    });
const $c705d81dab6c3de9$var$ForwardRef = /*#__PURE__*/ (0, $aHTyQ$react.forwardRef)($c705d81dab6c3de9$var$SvgComponent);
var $c705d81dab6c3de9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c705d81dab6c3de9$var$ForwardRef);
