// @react-spectrum/s2/illustrations/linear/internal/S2_lin_addproject_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8e8jd$jsx} from "react/jsx-runtime";
import {forwardRef as $8e8jd$forwardRef} from "react";





const $560f3dd53ef29895$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8e8jd$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8e8jd$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M13.512 13.833c.27-.655.91-1.083 1.618-1.083h16.115c.709 0 1.348.428 1.618 1.083l1.152 2.794-.14-.002H12.5q-.07 0-.14.002zm24.144 5.065q.023.058.04.115c.275.563.429 1.194.429 1.862v3.875c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-3.875c0-.966-.783-1.75-1.75-1.75H12.5c-.966 0-1.75.784-1.75 1.75v10.5c0 .967.784 1.75 1.75 1.75h14.75c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H12.5c-2.347 0-4.25-1.903-4.25-4.25v-10.5c0-.668.154-1.3.428-1.862q.018-.058.041-.115l2.482-6.018c.656-1.591 2.207-2.63 3.929-2.63h16.115c1.722 0 3.273 1.039 3.93 2.63zM19.25 25.5h8.25c.69 0 1.25-.56 1.25-1.25S28.19 23 27.5 23h-8.25c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25m17.595 15.84c3.836 0 6.945-3.11 6.945-6.945s-3.109-6.945-6.945-6.945c-3.835 0-6.945 3.11-6.945 6.945s3.11 6.945 6.945 6.945m1.305-10.09c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v1.875h-1.93c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h1.93V37.5c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-1.875h1.82c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-1.82z"
        })
    });
const $560f3dd53ef29895$var$ForwardRef = /*#__PURE__*/ (0, $8e8jd$forwardRef)($560f3dd53ef29895$var$SvgComponent);
var $560f3dd53ef29895$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($560f3dd53ef29895$var$ForwardRef);


export {$560f3dd53ef29895$export$2e2bcd8739ae039 as default};
