// @react-spectrum/s2/illustrations/linear/internal/S2_lin_apps_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1D1bM$reactjsxruntime = require("react/jsx-runtime");
var $1D1bM$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7307fe08d1083ccc$export$2e2bcd8739ae039);




const $7307fe08d1083ccc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1D1bM$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1D1bM$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M79.98 40c0-4.418 3.582-8 8-8H120c4.419 0 8 3.582 8 8v32.02c0 4.418-3.581 8-8 8H87.98c-4.418 0-8-3.582-8-8zm8-16c-8.836 0-16 7.163-16 16v32.02c0 8.837 7.164 16 16 16H120c8.837 0 16-7.163 16-16V40c0-8.837-7.163-16-16-16zM31.99 62.03c0-3.314 2.686-6 6-6h11.99c3.314 0 6 2.686 6 6v11.99c0 3.314-2.686 6-6 6H37.99c-3.314 0-6-2.686-6-6zm6-14c-7.732 0-14 6.268-14 14v11.99c0 7.732 6.268 14 14 14h11.99c7.732 0 14-6.268 14-14V62.03c0-7.732-6.268-14-14-14zm0 55.94c-3.314 0-6 2.686-6 6v11.99c0 3.314 2.686 6 6 6h11.99c3.314 0 6-2.686 6-6v-11.99c0-3.314-2.686-6-6-6zm-14 6c0-7.732 6.268-14 14-14h11.99c7.732 0 14 6.268 14 14v11.99c0 7.732-6.268 14-14 14H37.99c-7.732 0-14-6.268-14-14zm55.99 0c0-3.314 2.687-6 6-6h11.99c3.314 0 6 2.686 6 6v11.99c0 3.314-2.686 6-6 6H85.98c-3.313 0-6-2.686-6-6zm6-14c-7.731 0-14 6.268-14 14v11.99c0 7.732 6.269 14 14 14h11.99c7.732 0 14-6.268 14-14v-11.99c0-7.732-6.268-14-14-14z"
        })
    });
const $7307fe08d1083ccc$var$ForwardRef = /*#__PURE__*/ (0, $1D1bM$react.forwardRef)($7307fe08d1083ccc$var$SvgComponent);
var $7307fe08d1083ccc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7307fe08d1083ccc$var$ForwardRef);
