// @react-spectrum/s2/illustrations/linear/internal/S2_lin_play_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $hIY0u$jsx} from "react/jsx-runtime";
import {forwardRef as $hIY0u$forwardRef} from "react";





const $3a87dc707dd2f4ee$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hIY0u$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hIY0u$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8.25 23.74c0-8.698 7.052-15.75 15.75-15.75s15.75 7.052 15.75 15.75S32.699 39.49 24 39.49 8.25 32.44 8.25 23.74M24 5.49c-10.08 0-18.25 8.171-18.25 18.25S13.92 41.99 24 41.99s18.25-8.17 18.25-18.25S34.08 5.49 24 5.49m-2.113 12.433 7.49 4.664c.832.518.83 1.811-.003 2.328l-7.5 4.644c-.834.516-1.875-.132-1.874-1.167l.01-9.307c.002-1.035 1.044-1.68 1.877-1.162"
        })
    });
const $3a87dc707dd2f4ee$var$ForwardRef = /*#__PURE__*/ (0, $hIY0u$forwardRef)($3a87dc707dd2f4ee$var$SvgComponent);
var $3a87dc707dd2f4ee$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($3a87dc707dd2f4ee$var$ForwardRef);


export {$3a87dc707dd2f4ee$export$2e2bcd8739ae039 as default};
