// @react-spectrum/s2/illustrations/linear/internal/S2_lin_ribbon_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $e3RXu$jsx} from "react/jsx-runtime";
import {forwardRef as $e3RXu$forwardRef} from "react";





const $c1038070e1cc7e9b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e3RXu$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $e3RXu$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.995 17c-10.77 0-19.5 8.73-19.5 19.5s8.73 19.5 19.5 19.5 19.5-8.73 19.5-19.5-8.73-19.5-19.5-19.5m-23.5 19.5c0-12.979 10.522-23.5 23.5-23.5 12.98 0 23.5 10.521 23.5 23.5 0 6.514-2.65 12.41-6.932 16.666l6.915 17.116c1.193 2.952-1.374 6.047-4.495 5.42l-4.462-.895-2.588 3.743c-1.81 2.62-5.806 2.176-6.999-.776L47.967 65.48l-4.9 12.129c-1.197 2.962-5.212 3.395-7.013.756l-2.562-3.753-4.45.92c-3.129.648-5.716-2.453-4.52-5.415l6.865-16.99c-4.258-4.253-6.892-10.132-6.892-16.626m10.118 19.32-6.381 15.794 4.45-.92c1.581-.328 3.204.328 4.114 1.662l2.562 3.754 6.452-15.97-.101-.25c-4.098-.395-7.888-1.843-11.096-4.07m15.42 4.093c4.173-.358 8.036-1.808 11.3-4.062l6.436 15.93-4.461-.896c-1.566-.314-3.169.334-4.077 1.648l-2.588 3.743z"
        })
    });
const $c1038070e1cc7e9b$var$ForwardRef = /*#__PURE__*/ (0, $e3RXu$forwardRef)($c1038070e1cc7e9b$var$SvgComponent);
var $c1038070e1cc7e9b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c1038070e1cc7e9b$var$ForwardRef);


export {$c1038070e1cc7e9b$export$2e2bcd8739ae039 as default};
