// @react-spectrum/s2/illustrations/linear/internal/S2_lin_dataAnalytics_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $uM9k6$reactjsxruntime = require("react/jsx-runtime");
var $uM9k6$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c695b42717e3417f$export$2e2bcd8739ae039);




const $c695b42717e3417f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $uM9k6$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $uM9k6$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M79.44 21.306c.686.866.539 2.124-.328 2.81L56.016 42.387c-.619.49-1.468.57-2.167.204l-14.718-7.69-19.708 16.716c-.843.714-2.105.61-2.82-.232s-.61-2.104.232-2.819L37.571 30.98c.62-.527 1.498-.624 2.22-.247l14.76 7.713 22.08-17.467c.866-.685 2.124-.538 2.81.328M33.76 52.35c0-3.032 2.457-5.49 5.49-5.49 3.032 0 5.49 2.458 5.49 5.49v18.597c0 3.032-2.459 5.49-5.49 5.49-3.033 0-5.49-2.458-5.49-5.49zm5.49-1.49c-.824 0-1.49.667-1.49 1.49v18.597c0 .822.666 1.49 1.49 1.49.822 0 1.49-.668 1.49-1.49V52.35c0-.823-.668-1.49-1.49-1.49m17.508-.577c-3.032 0-5.49 2.458-5.49 5.49V70.95c0 3.032 2.458 5.49 5.49 5.49s5.49-2.458 5.49-5.49V55.773c0-3.032-2.458-5.49-5.49-5.49m-1.49 5.49c0-.823.667-1.49 1.49-1.49s1.49.667 1.49 1.49V70.95c0 .823-.667 1.49-1.49 1.49s-1.49-.667-1.49-1.49zm13.508-12.045c0-3.032 2.458-5.49 5.49-5.49s5.49 2.458 5.49 5.49v27.22c0 3.032-2.458 5.49-5.49 5.49s-5.49-2.458-5.49-5.49zm5.49-1.49c-.823 0-1.49.668-1.49 1.49v27.22c0 .823.667 1.49 1.49 1.49s1.49-.667 1.49-1.49v-27.22c0-.822-.667-1.49-1.49-1.49M21.741 57.057c-3.032 0-5.49 2.458-5.49 5.49v8.4c0 3.032 2.457 5.49 5.49 5.49 3.032 0 5.49-2.458 5.49-5.49v-8.4c0-3.032-2.458-5.49-5.49-5.49m-1.49 5.49c0-.823.667-1.49 1.49-1.49s1.49.667 1.49 1.49v8.4c0 .823-.667 1.49-1.49 1.49s-1.49-.667-1.49-1.49z"
        })
    });
const $c695b42717e3417f$var$ForwardRef = /*#__PURE__*/ (0, $uM9k6$react.forwardRef)($c695b42717e3417f$var$SvgComponent);
var $c695b42717e3417f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c695b42717e3417f$var$ForwardRef);
