// @react-spectrum/s2/illustrations/linear/internal/S2_lin_chartAreaStack_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5Ygl3$jsx} from "react/jsx-runtime";
import {forwardRef as $5Ygl3$forwardRef} from "react";





const $acf4c8dbc2526bbb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5Ygl3$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5Ygl3$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M40.158 12.242c0-2.326-3.098-3.123-4.22-1.087l-3.976 7.209-3.233-3.955c-.965-1.182-2.803-1.08-3.631.202l-4.665 7.215-3.552-4.105c-.838-.968-2.314-1.043-3.245-.164l-4.741 4.471c-.651.614-1.02 1.47-1.02 2.365v12.496c0 1.795 1.455 3.25 3.25 3.25h25.783c1.794 0 3.25-1.455 3.25-3.25zm-6.185 7.653 3.685-6.682v14.114l-3.38 1.597-3.666-3.725c-.928-.943-2.465-.886-3.32.123l-5.225 6.162-4.26-3.392c-.766-.61-1.84-.654-2.654-.108l-4.778 3.204v-6.795c0-.207.085-.404.235-.546l4.551-4.292 3.6 4.16c.98 1.133 2.776 1.008 3.59-.251l4.66-7.207 3.25 3.976c1.008 1.233 2.943 1.057 3.712-.338M10.375 34.19v2.7c0 .413.336.75.75.75h25.783c.414 0 .75-.337.75-.75v-6.798l-2.471 1.168c-.865.41-1.894.226-2.565-.455l-3.6-3.657-5.205 6.137c-.786.928-2.166 1.063-3.117.305l-4.306-3.428-5.904 3.959q-.056.038-.115.069"
        })
    });
const $acf4c8dbc2526bbb$var$ForwardRef = /*#__PURE__*/ (0, $5Ygl3$forwardRef)($acf4c8dbc2526bbb$var$SvgComponent);
var $acf4c8dbc2526bbb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($acf4c8dbc2526bbb$var$ForwardRef);


export {$acf4c8dbc2526bbb$export$2e2bcd8739ae039 as default};
