// @react-spectrum/s2/illustrations/linear/internal/S2_lin_folderShared_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dJapm$reactjsxruntime = require("react/jsx-runtime");
var $dJapm$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1ed11a1287d43913$export$2e2bcd8739ae039);




const $1ed11a1287d43913$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dJapm$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dJapm$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M15 22.667C15 21.14 16.189 20 17.534 20H33.94c.747 0 1.472.349 1.961.979L42.912 30H15zM11 30v-7.333C11 19.04 13.871 16 17.534 16H33.94c2.008 0 3.888.94 5.12 2.524l-1.333 1.036 1.332-1.036L47.98 30h26.046C80.092 30 85 34.947 85 41.02v27.96C85 75.052 80.092 80 74.025 80h-52.05C15.908 80 11 75.053 11 68.98V30m4 4v34.98c0 3.883 3.136 7.02 6.975 7.02h52.05C77.865 76 81 72.863 81 68.98V41.02c0-3.883-3.136-7.02-6.975-7.02H15"
            }),
            /*#__PURE__*/ (0, $dJapm$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M51.03 52.235c1.623 0 2.94-1.316 2.94-2.94 0-1.625-1.317-2.942-2.94-2.942-1.625 0-2.942 1.317-2.942 2.941 0 1.625 1.317 2.941 2.941 2.941m0 11.765h-5.35c-.72 0-1.247-.596-1.095-1.224.638-2.648 3.3-4.658 6.444-4.658s5.807 2.01 6.445 4.658c.151.628-.375 1.224-1.095 1.224z"
            }),
            /*#__PURE__*/ (0, $dJapm$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "m44.585 62.776 1.944.468zm12.889 0-1.945.468zM35.528 60.9c-.185 1.089.548 2.122 1.637 2.307 1.09.184 2.122-.548 2.307-1.637zm6.434-2.688c.997-.476 1.42-1.67.943-2.667-.476-.996-1.67-1.418-2.667-.942zM43.376 46c1.105 0 2-.895 2-2s-.895-2-2-2zm-3.667 5.062c.793.768 2.06.747 2.828-.046.768-.794.748-2.06-.046-2.828zm12.261-1.768c0 .52-.421.941-.94.941v4c2.728 0 4.94-2.212 4.94-4.94zm-.94-.941c.519 0 .94.421.94.941h4c0-2.729-2.212-4.941-4.94-4.941zm-.942.941c0-.52.421-.941.941-.941v-4c-2.729 0-4.941 2.212-4.941 4.941zm.941.941c-.52 0-.941-.421-.941-.94h-4c0 2.728 2.212 4.94 4.941 4.94zM45.68 66h5.35v-4h-5.35zm-3.039-3.693C42.126 64.441 43.931 66 45.68 66v-4c.197 0 .433.085.619.293.192.216.322.57.23.951zm8.39-6.19c-3.948 0-7.508 2.531-8.39 6.19l3.89.937c.393-1.636 2.157-3.126 4.5-3.126zm8.388 6.19c-.882-3.659-4.441-6.19-8.389-6.19v4c2.343 0 4.106 1.491 4.5 3.127zM56.378 66c1.749 0 3.554-1.56 3.04-3.693l-3.889.937c-.092-.38.038-.735.23-.95.186-.21.423-.294.62-.294zm-5.349 0h5.35v-4h-5.35zm-11.557-4.43c.256-1.508 1.228-2.754 2.49-3.357l-1.724-3.61c-2.446 1.17-4.238 3.518-4.71 6.298zm2.631-14.297c0-.703.57-1.273 1.273-1.273v-4c-2.912 0-5.273 2.36-5.273 5.273zm.388.915c-.241-.234-.388-.556-.388-.915h-4c0 1.487.618 2.832 1.606 3.789z"
            })
        ]
    });
const $1ed11a1287d43913$var$ForwardRef = /*#__PURE__*/ (0, $dJapm$react.forwardRef)($1ed11a1287d43913$var$SvgComponent);
var $1ed11a1287d43913$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1ed11a1287d43913$var$ForwardRef);
