// @react-spectrum/s2/illustrations/linear/internal/S2_lin_movieCamera_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6TvPf$reactjsxruntime = require("react/jsx-runtime");
var $6TvPf$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $fa361e0eefbc96a0$export$2e2bcd8739ae039);




const $fa361e0eefbc96a0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6TvPf$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6TvPf$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M5.266 16.816c0-2.9 2.35-5.25 5.25-5.25H26.98c2.9 0 5.25 2.35 5.25 5.25v1.566l5.318-3.946c2.144-1.591 5.186-.06 5.186 2.61v13.855c0 2.664-3.028 4.197-5.175 2.619l-5.329-3.918v1.58c0 2.9-2.35 5.25-5.25 5.25H10.516c-2.9 0-5.25-2.35-5.25-5.25zm5.25-2.75c-1.52 0-2.75 1.232-2.75 2.75v14.366c0 1.519 1.23 2.75 2.75 2.75H26.98c1.52 0 2.75-1.231 2.75-2.75v-4.05c0-.472.265-.903.686-1.116.42-.212.925-.17 1.305.109l7.32 5.38c.494.365 1.193.011 1.193-.604V17.046c0-.616-.702-.97-1.197-.602l-7.312 5.426c-.379.282-.885.325-1.306.113s-.689-.644-.689-1.117v-4.05c0-1.518-1.23-2.75-2.75-2.75z"
        })
    });
const $fa361e0eefbc96a0$var$ForwardRef = /*#__PURE__*/ (0, $6TvPf$react.forwardRef)($fa361e0eefbc96a0$var$SvgComponent);
var $fa361e0eefbc96a0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($fa361e0eefbc96a0$var$ForwardRef);
