// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noElements_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $26kGf$jsxs, jsx as $26kGf$jsx} from "react/jsx-runtime";
import {forwardRef as $26kGf$forwardRef} from "react";





const $029a1cdffa7fef7b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $26kGf$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $26kGf$jsx)("path", {
                stroke: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                strokeLinecap: "round",
                strokeLinejoin: "round",
                strokeWidth: 2.5,
                d: "M13.172 28v14.073M10.36 42.074h5.629M6.137 28H20.21M6.137 28v2.815M20.21 28v2.815"
            }),
            /*#__PURE__*/ (0, $26kGf$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M38.615 7.285c-.01.005-.036.021-.07.062l-6.163 7.24c-.075.089-.045.225.061.273l3.452 1.564-2.016 5.04c-.02.049-.021.078-.022.09q0 .017.005.03c.008.02.034.06.089.094.055.033.102.038.124.035q.014 0 .03-.009c.01-.006.035-.02.07-.061l6.168-7.138c.077-.09.045-.229-.064-.276l-3.546-1.516 2.107-5.186c.02-.05.022-.08.022-.092q0-.016-.005-.03c-.007-.02-.033-.06-.089-.094s-.103-.038-.125-.035q-.014 0-.028.009M36.64 5.727c1.97-2.314 5.66-.075 4.516 2.74l-1.183 2.912 1.29.552c1.665.712 2.156 2.838.971 4.209l-6.168 7.137c-1.974 2.284-5.629.06-4.508-2.742l1.128-2.82-1.275-.578c-1.62-.734-2.085-2.815-.933-4.17zM21.313 11c0-1.519-1.232-2.75-2.75-2.75H7.5c-1.519 0-2.75 1.231-2.75 2.75v6.375l.001.08 2.919-4.509c.928-1.435 3.03-1.435 3.958 0l2.703 4.174 1.292-1.954.005-.008c.933-1.388 2.98-1.392 3.914 0l1.746 2.586q.024-.18.024-.369zM9.649 14.489l-3.44 5.315c.385.205.824.321 1.291.321h4.843l.483-.73zm10.06 5.386c-.348.16-.737.25-1.146.25H15.34l2.245-3.396zM18.564 5.75c2.899 0 5.25 2.35 5.25 5.25v6.375c0 2.9-2.351 5.25-5.25 5.25H7.5c-2.9 0-5.25-2.35-5.25-5.25V11c0-2.9 2.35-5.25 5.25-5.25zm-.87 4.636c0 .719.583 1.302 1.302 1.302s1.302-.583 1.302-1.302-.583-1.302-1.302-1.302-1.302.582-1.302 1.302M44.5 32c0-1.519-1.231-2.75-2.75-2.75H31c-1.519 0-2.75 1.231-2.75 2.75v6.375c0 1.519 1.231 2.75 2.75 2.75h10.75c1.519 0 2.75-1.231 2.75-2.75zm-2.75-5.25c2.9 0 5.25 2.35 5.25 5.25v6.375c0 2.9-2.35 5.25-5.25 5.25H31c-2.9 0-5.25-2.35-5.25-5.25V32c0-2.9 2.35-5.25 5.25-5.25zm-6.413 5.774 3.69 2.106c.412.236.412.824-.002 1.059l-3.694 2.097c-.414.235-.93-.06-.93-.53l.005-4.203c.001-.471.518-.765.931-.529"
            })
        ]
    });
const $029a1cdffa7fef7b$var$ForwardRef = /*#__PURE__*/ (0, $26kGf$forwardRef)($029a1cdffa7fef7b$var$SvgComponent);
var $029a1cdffa7fef7b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($029a1cdffa7fef7b$var$ForwardRef);


export {$029a1cdffa7fef7b$export$2e2bcd8739ae039 as default};
