// @react-spectrum/s2/illustrations/linear/internal/S2_lin_trophy_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $4Fai2$jsx} from "react/jsx-runtime";
import {forwardRef as $4Fai2$forwardRef} from "react";





const $5e275a05b1c211b8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4Fai2$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4Fai2$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M33.86 21c0-.555.445-1 1-1h26.3c.556 0 1 .445 1 1v6.03q-.01.098-.01.2t.01.2v11.34c0 7.815-6.343 14.15-14.15 14.15-7.814 0-14.15-6.344-14.15-14.15zm32.3 8.23v9.54c0 1.681-.228 3.309-.656 4.854C71.412 42.952 76 37.938 76 31.85v-1.62c0-.555-.444-1-1-1zm-2.332 18.444c-2.806 4.973-7.89 8.49-13.828 9.138v10.114c5.214.721 9.522 4.233 11.408 8.963l.003.008c.79 2.006-.727 4.103-2.8 4.103H37.4c-2.101 0-3.59-2.125-2.799-4.11 1.877-4.73 6.19-8.239 11.398-8.962V56.81c-5.935-.655-11.007-4.173-13.809-9.139q-.166.03-.341.029C23.095 47.7 16 40.605 16 31.85v-1.62c0-2.765 2.235-5 5-5h8.86V21c0-2.756 2.221-4.985 4.973-5H61.16c2.765 0 5 2.235 5 5v4.23H75c2.765 0 5 2.235 5 5v1.62c0 8.755-7.095 15.85-15.85 15.85q-.165 0-.322-.026M21 29.23h8.86v9.54c0 1.681.23 3.31.658 4.856C24.6 42.964 20 37.946 20 31.85v-1.62c0-.555.445-1 1-1m27 41.56c-3.852 0-7.218 2.094-9.019 5.21h18.045c-1.806-3.118-5.167-5.21-9.016-5.21zm2.69-39.43-1.51-3.08h.01c-.44-.89-1.71-.89-2.15 0l-1.51 3.08c-.18.36-.52.6-.91.66l-3.39.49c-.99.14-1.39 1.36-.67 2.05l2.46 2.39c.29.28.42.68.35 1.07l-.59 3.38c-.17.99.86 1.74 1.74 1.27l3.03-1.6c.35-.19.77-.19 1.12 0l3.03 1.6c.88.46 1.91-.29 1.74-1.27l-.59-3.38c-.06-.39.07-.79.35-1.07l2.46-2.39c.71-.7.32-1.91-.67-2.05l-3.39-.49c-.4-.06-.74-.3-.91-.66"
        })
    });
const $5e275a05b1c211b8$var$ForwardRef = /*#__PURE__*/ (0, $4Fai2$forwardRef)($5e275a05b1c211b8$var$SvgComponent);
var $5e275a05b1c211b8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5e275a05b1c211b8$var$ForwardRef);


export {$5e275a05b1c211b8$export$2e2bcd8739ae039 as default};
