// @react-spectrum/s2/illustrations/linear/internal/S2_lin_AIChat_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1w1ie$reactjsxruntime = require("react/jsx-runtime");
var $1w1ie$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7a662c176a149f0d$export$2e2bcd8739ae039);




const $7a662c176a149f0d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1w1ie$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1w1ie$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M38.615 7.111c.867.523 1.436 1.576 1.18 2.77l-.615 2.85v.002q-.006.028 0 .057.01.027.019.036l1.897 2.149c.814.922.802 2.124.323 2.99-.483.871-1.519 1.53-2.755 1.24l-2.718-.638-2.057 1.978c-.907.872-2.129.87-3.004.343-.867-.523-1.436-1.576-1.18-2.77l.615-2.85v-.002q.006-.028 0-.057-.01-.027-.019-.036l-1.897-2.149c-.814-.921-.802-2.124-.323-2.99.483-.871 1.519-1.53 2.755-1.24l2.718.638 2.057-1.978c.907-.872 2.129-.87 3.004-.343m-1.278 2.151-2.051 1.973c-.605.582-1.463.829-2.29.634h-.002l-2.716-.638-.01.014c-.015.028-.019.056-.018.073q0 .01.004.017.002.007.023.034l1.897 2.148c.55.623.764 1.474.59 2.278v.001l-.614 2.848q-.008.047-.005.057 0 .012.004.017.003.009.014.018l2.051-1.972c.605-.582 1.463-.829 2.29-.633h.002l2.716.637.01-.014c.015-.028.019-.056.018-.073q0-.01-.004-.017-.002-.007-.024-.034l-1.896-2.148c-.55-.622-.764-1.474-.59-2.278v-.002l.614-2.847q.008-.046.005-.057 0-.011-.004-.017-.003-.009-.014-.019M7.75 14.987c0-2.678 2.214-4.738 4.85-4.738h12.431c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-12.43c-1.341 0-2.351 1.024-2.351 2.238v11.487c0 1.214 1.01 2.238 2.35 2.238h4.311l.006 1.244c.006 1.624.013 3.937 0 5.739l7.944-6.69.35-.293H35.4c1.34 0 2.35-1.024 2.35-2.238v-4.6c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v4.6c0 2.678-2.214 4.738-4.85 4.738h-9.277l-7.662 6.451-.002.002c-1.553 1.315-4.058.275-4.042-1.86.01-1.388.009-3.114.004-4.593h-1.82c-2.637 0-4.851-2.06-4.851-4.738zm19.24 10.15-2.008-.453c-.265-.06-.541.015-.74.198l-1.513 1.395c-.596.55-1.544.002-1.366-.788l.453-2.007c.06-.266-.015-.542-.198-.741l-1.396-1.513c-.55-.596-.001-1.544.788-1.365l2.008.452c.265.06.541-.015.74-.198l1.513-1.395c.596-.55 1.544-.002 1.366.787l-.453 2.008c-.06.266.015.542.198.741l1.396 1.513c.55.596.001 1.543-.788 1.365"
        })
    });
const $7a662c176a149f0d$var$ForwardRef = /*#__PURE__*/ (0, $1w1ie$react.forwardRef)($7a662c176a149f0d$var$SvgComponent);
var $7a662c176a149f0d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7a662c176a149f0d$var$ForwardRef);
