// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lockClose_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $3zRlE$jsx} from "react/jsx-runtime";
import {forwardRef as $3zRlE$forwardRef} from "react";





const $6c418eb215b9e54f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3zRlE$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3zRlE$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M34.5 27.5C34.5 19.492 40.992 13 49 13s14.5 6.492 14.5 14.5V39h5c5.523 0 10 4.477 10 10v21c0 5.523-4.477 10-10 10h-40c-5.523 0-10-4.477-10-10V49c0-5.523 4.477-10 10-10h6zm4 11.5h21V27.5C59.5 21.701 54.799 17 49 17s-10.5 4.701-10.5 10.5zm23 4h-33c-3.314 0-6 2.686-6 6v21c0 3.314 2.686 6 6 6h40c3.314 0 6-2.686 6-6V49c0-3.314-2.686-6-6-6zM47 60.584c-1.766-.772-3-2.534-3-4.584 0-2.761 2.239-5 5-5s5 2.239 5 5c0 2.05-1.234 3.812-3 4.584v4.917c0 1.105-.895 2-2 2s-2-.895-2-2z"
        })
    });
const $6c418eb215b9e54f$var$ForwardRef = /*#__PURE__*/ (0, $3zRlE$forwardRef)($6c418eb215b9e54f$var$SvgComponent);
var $6c418eb215b9e54f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6c418eb215b9e54f$var$ForwardRef);


export {$6c418eb215b9e54f$export$2e2bcd8739ae039 as default};
