// @react-spectrum/s2/illustrations/linear/internal/S2_lin_brightnessContrast_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9lA1y$jsx} from "react/jsx-runtime";
import {forwardRef as $9lA1y$forwardRef} from "react";





const $c4a2e9aa6019b8af$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9lA1y$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9lA1y$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M50 15.234c0-1.104-.895-2-2-2s-2 .896-2 2v4.645c0 1.104.895 2 2 2s2-.896 2-2zM33.45 48.001c0-8.035 6.514-14.549 14.55-14.549V62.55c-8.035 0-14.55-6.514-14.55-14.55M48 29.452c-10.245 0-18.55 8.305-18.55 18.549S37.756 66.549 48 66.549c10.244 0 18.548-8.304 18.548-18.548S58.244 29.452 48 29.452m0 44.67c1.105 0 2 .895 2 2v4.644c0 1.104-.895 2-2 2s-2-.896-2-2V76.12c0-1.104.895-2 2-2M80.766 50c1.104 0 2-.895 2-2s-.896-2-2-2H76.12c-1.105 0-2 .895-2 2s.895 2 2 2zm-58.887-2c0 1.105-.896 2-2 2h-4.645c-1.104 0-2-.895-2-2s.896-2 2-2h4.645c1.104 0 2 .895 2 2m50.704-21.755c.78-.781.78-2.048 0-2.829-.781-.78-2.048-.78-2.829 0l-3.284 3.285c-.78.78-.78 2.047 0 2.828.781.781 2.047.781 2.829 0zM29.529 66.47c.781.78.781 2.047 0 2.828l-3.284 3.284c-.781.781-2.048.781-2.829 0-.78-.78-.78-2.047 0-2.828l3.285-3.284c.78-.781 2.047-.781 2.828 0m-6.113-40.225c-.78-.781-.78-2.047 0-2.829.781-.78 2.048-.78 2.829 0l3.284 3.285c.78.78.78 2.047 0 2.828-.781.781-2.047.781-2.829 0zM66.47 66.47c-.781.78-.781 2.047 0 2.828l3.284 3.284c.781.781 2.048.781 2.829 0 .78-.78.78-2.047 0-2.828l-3.285-3.284c-.78-.781-2.047-.781-2.828 0"
        })
    });
const $c4a2e9aa6019b8af$var$ForwardRef = /*#__PURE__*/ (0, $9lA1y$forwardRef)($c4a2e9aa6019b8af$var$SvgComponent);
var $c4a2e9aa6019b8af$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c4a2e9aa6019b8af$var$ForwardRef);


export {$c4a2e9aa6019b8af$export$2e2bcd8739ae039 as default};
