// @react-spectrum/s2/illustrations/linear/internal/S2_lin_3Dasset_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1MBd3$reactjsxruntime = require("react/jsx-runtime");
var $1MBd3$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6c5a3d0f212ce427$export$2e2bcd8739ae039);




const $6c5a3d0f212ce427$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1MBd3$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1MBd3$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M115.073 31.049c-3.629-2.02-8.045-2.02-11.674 0L81.947 42.992c-1.21.673-2.681.673-3.891 0l-20.27-11.284c-4.323-2.408-9.824-2.194-13.971.151-3.32 1.877-7.92 4.418-12.558 6.96-3.782 2.071-5.856 5.683-6.204 9.435q-.11.455-.112.942v36.631c0 6.775 3.43 13.09 9.113 16.777l43.725 28.375c1.323.858 3.025.859 4.349.004l43.913-28.378c5.702-3.684 9.145-10.01 9.145-16.798v-36.61q0-.172-.014-.339c-.141-3.95-2.193-7.84-6.155-10.046zM127.186 60.8l-43.23 24.067v35.409l37.743-24.39c3.421-2.21 5.487-6.006 5.487-10.079zm-51.23 24.018L32.942 60.87v24.957c0 4.065 2.058 7.854 5.468 10.066l37.548 24.366zm31.334-46.78c1.21-.673 2.682-.673 3.892 0l13.943 7.763c2.739 1.525 2.739 5.465 0 6.99L80.001 77.913 35.058 52.892c-2.75-1.531-2.75-5.527.043-7.058 4.645-2.545 9.282-5.107 12.65-7.01 1.884-1.065 4.415-1.09 6.144-.127l20.27 11.284c3.629 2.02 8.044 2.02 11.674 0z"
        })
    });
const $6c5a3d0f212ce427$var$ForwardRef = /*#__PURE__*/ (0, $1MBd3$react.forwardRef)($6c5a3d0f212ce427$var$SvgComponent);
var $6c5a3d0f212ce427$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6c5a3d0f212ce427$var$ForwardRef);
