// @react-spectrum/s2/illustrations/linear/internal/S2_lin_emptyStateExport_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6Og1A$jsx} from "react/jsx-runtime";
import {forwardRef as $6Og1A$forwardRef} from "react";





const $4a0a60fe72ece0f1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6Og1A$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6Og1A$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M31.68 8.5c0-.414.336-.75.75-.75h6.5c.414 0 .75.336.75.75V22c0 .414-.336.75-.75.75h-6.5c-.414 0-.75-.336-.75-.75zm.75-3.25c-1.795 0-3.25 1.455-3.25 3.25V22c0 1.795 1.455 3.25 3.25 3.25h6.5c1.795 0 3.25-1.455 3.25-3.25V8.5c0-1.795-1.455-3.25-3.25-3.25zm-2.25 27c.414 0 .75.336.75.75v6.5c0 .414-.336.75-.75.75h-13.5c-.415 0-.75-.336-.75-.75V33c0-.414.335-.75.75-.75zm3.25.75c0-1.795-1.455-3.25-3.25-3.25h-13.5c-1.795 0-3.25 1.455-3.25 3.25v6.5c0 1.795 1.455 3.25 3.25 3.25h13.5c1.795 0 3.25-1.455 3.25-3.25zM9.42 22.9c-.602 0-1.09.488-1.09 1.09s.488 1.09 1.09 1.09 1.09-.488 1.09-1.09-.488-1.09-1.09-1.09m-3.59 1.09c0-1.547.978-2.865 2.35-3.37v-7.26c-1.372-.505-2.35-1.823-2.35-3.37 0-1.983 1.607-3.59 3.59-3.59 1.547 0 2.865.978 3.37 2.35h7.26c.505-1.372 1.823-2.35 3.37-2.35 1.983 0 3.59 1.607 3.59 3.59 0 1.54-.969 2.853-2.33 3.363v4.677c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-4.67c-.979-.36-1.757-1.134-2.123-2.11h-7.274c-.364.97-1.134 1.74-2.103 2.103v7.274c.976.366 1.75 1.144 2.11 2.123h4.67c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-4.677c-.51 1.361-1.824 2.33-3.363 2.33-1.983 0-3.59-1.607-3.59-3.59m2.5-14c0-.602.488-1.09 1.09-1.09s1.09.488 1.09 1.09-.488 1.09-1.09 1.09-1.09-.488-1.09-1.09M23.42 8.9c-.602 0-1.09.488-1.09 1.09s.488 1.09 1.09 1.09 1.09-.488 1.09-1.09-.488-1.09-1.09-1.09m-1.45 18.84-2.71-6.6v-.01c-.33-.82.49-1.64 1.31-1.3l6.6 2.71c.82.34.83 1.48.03 1.84l-1.99.89c-.23.1-.41.28-.51.51l-.89 1.99c-.36.8-1.51.78-1.84-.03"
        })
    });
const $4a0a60fe72ece0f1$var$ForwardRef = /*#__PURE__*/ (0, $6Og1A$forwardRef)($4a0a60fe72ece0f1$var$SvgComponent);
var $4a0a60fe72ece0f1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($4a0a60fe72ece0f1$var$ForwardRef);


export {$4a0a60fe72ece0f1$export$2e2bcd8739ae039 as default};
