// @react-spectrum/s2/illustrations/linear/internal/S2_lin_user_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $4ZC0L$jsx} from "react/jsx-runtime";
import {forwardRef as $4ZC0L$forwardRef} from "react";





const $be14a8f75f9aa091$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4ZC0L$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4ZC0L$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M18.25 15.25C18.25 11.716 20.903 9 24 9s5.75 2.716 5.75 6.25S27.094 21.5 24 21.5s-5.75-2.716-5.75-6.25M24 6.5c-4.637 0-8.25 4-8.25 8.75S19.362 24 24 24c4.636 0 8.25-4 8.25-8.75S28.635 6.5 24 6.5M11.53 38.093C12.56 32.148 17.793 27.57 24 27.57c6.206 0 11.438 4.578 12.467 10.523.077.45-.277.877-.748.877H12.28c-.471 0-.825-.428-.748-.877m0 0h.001l-1.232-.213zM24 25.07c-7.454 0-13.701 5.482-14.933 12.597v.001c-.342 1.99 1.203 3.802 3.212 3.802h23.44c2.009 0 3.555-1.812 3.212-3.802v-.001C37.7 30.552 31.45 25.07 23.999 25.07"
        })
    });
const $be14a8f75f9aa091$var$ForwardRef = /*#__PURE__*/ (0, $4ZC0L$forwardRef)($be14a8f75f9aa091$var$SvgComponent);
var $be14a8f75f9aa091$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($be14a8f75f9aa091$var$ForwardRef);


export {$be14a8f75f9aa091$export$2e2bcd8739ae039 as default};
