// @react-spectrum/s2/illustrations/linear/internal/S2_lin_megaphonePromote_Expressive_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $aRGpv$reactjsxruntime = require("react/jsx-runtime");
var $aRGpv$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9328fd1738280413$export$2e2bcd8739ae039);




const $9328fd1738280413$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aRGpv$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $aRGpv$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M31.663 10.169c2.515-.752 5.1 1.098 5.1 3.748v13.067c0 2.656-2.59 4.5-5.104 3.748l-9.974-2.984 2.204 4.768.014.03c.509 1.184.478 2.352.026 3.348-.44.968-1.236 1.687-2.125 2.112-.952.455-2.096.614-3.181.314-1.12-.31-2.101-1.088-2.676-2.346-.343-.753-1.427-3.116-2.424-5.291l-1.329-2.897-.246-.536h-.017C8.134 27.25 5 24.247 5 20.442c0-3.795 3.124-6.842 6.93-6.842h8.232l11.5-3.431m2.6 3.748c0-.925-.923-1.64-1.885-1.353L21.6 15.78v9.333l10.775 3.224c.964.288 1.888-.427 1.888-1.353zM7.5 20.442c0-2.378 1.967-4.342 4.43-4.342h7.14v8.65h-7.14c-2.473 0-4.43-1.94-4.43-4.308m7.199 6.808 1.096 2.39c.997 2.175 2.082 4.54 2.427 5.296.268.588.663.863 1.067.974.436.12.961.068 1.437-.16.44-.21.765-.535.926-.89.147-.323.197-.752-.04-1.312l-2.911-6.298zm25.891-2.291c-.658-.208-1.36.156-1.569.815s.157 1.36.815 1.569l1.725.546c.658.208 1.36-.157 1.569-.815s-.157-1.36-.815-1.569zm2.54-11.518c.208.658-.157 1.36-.815 1.569l-1.725.546c-.658.208-1.36-.157-1.569-.815s.157-1.36.815-1.569l1.725-.546c.658-.208 1.36.157 1.569.815m-2.917 5.568c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h1.809c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25z"
        })
    });
const $9328fd1738280413$var$ForwardRef = /*#__PURE__*/ (0, $aRGpv$react.forwardRef)($9328fd1738280413$var$SvgComponent);
var $9328fd1738280413$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9328fd1738280413$var$ForwardRef);
