// @react-spectrum/s2/illustrations/linear/internal/S2_lin_text_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $jTROU$reactjsxruntime = require("react/jsx-runtime");
var $jTROU$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7d9185cefe6008d6$export$2e2bcd8739ae039);




const $7d9185cefe6008d6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jTROU$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jTROU$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M38.541 33.791c-2.623 0-4.75 2.127-4.75 4.75v16.583c0 2.624 2.127 4.75 4.75 4.75s4.75-2.126 4.75-4.75v-7.833c0-2.21 1.79-4 4-4h23.962c2.209 0 4 1.79 4 4v65.417c0 2.209-1.791 4-4 4h-7.836c-2.623 0-4.75 2.126-4.75 4.75s2.127 4.75 4.75 4.75h33.167c2.623 0 4.75-2.127 4.75-4.75s-2.127-4.75-4.75-4.75h-7.831c-2.21 0-4-1.791-4-4V47.291c0-2.21 1.79-4 4-4h23.955c2.209 0 4 1.79 4 4v7.833c0 2.624 2.126 4.75 4.75 4.75s4.75-2.126 4.75-4.75V38.541c0-2.623-2.126-4.75-4.75-4.75H38.541m-12.75 4.75c0-7.042 5.708-12.75 12.75-12.75h82.917c7.042 0 12.75 5.708 12.75 12.75v16.583c0 7.042-5.709 12.75-12.75 12.75s-12.75-5.708-12.75-12.75v-3.833H92.753v57.417h3.831c7.042 0 12.75 5.708 12.75 12.75s-5.708 12.75-12.75 12.75H63.417c-7.041 0-12.75-5.709-12.75-12.75s5.709-12.75 12.75-12.75h3.836V51.291H51.29v3.833c0 7.042-5.708 12.75-12.75 12.75s-12.75-5.708-12.75-12.75z"
        })
    });
const $7d9185cefe6008d6$var$ForwardRef = /*#__PURE__*/ (0, $jTROU$react.forwardRef)($7d9185cefe6008d6$var$SvgComponent);
var $7d9185cefe6008d6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7d9185cefe6008d6$var$ForwardRef);
