// @react-spectrum/s2/illustrations/linear/internal/S2_lin_segmentation_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $8Ldph$reactjsxruntime = require("react/jsx-runtime");
var $8Ldph$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7cd9d6fd521a0d76$export$2e2bcd8739ae039);




const $7cd9d6fd521a0d76$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8Ldph$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8Ldph$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M52.773 40.272c0-8.837 7.163-16 16-16h50.977c8.837 0 16 7.163 16 16v15.3c0 8.837-7.163 16-16 16H68.772c-8.836 0-16-7.163-16-16v-3.697H40.25c-4.418 0-8 3.582-8 8v40.2c0 4.418 3.582 8 8 8h12.523v-3.648c0-8.837 7.163-16 16-16h50.977c8.837 0 16 7.163 16 16v15.3c0 8.837-7.163 16-16 16H68.772c-8.836 0-16-7.163-16-16v-3.652H40.25c-8.837 0-16-7.163-16-16v-40.2c0-8.837 7.163-16 16-16h12.523zm16-8c-4.419 0-8 3.581-8 8v15.3c0 4.418 3.581 8 8 8h50.977c4.418 0 8-3.582 8-8v-15.3c0-4.419-3.582-8-8-8zm-8 72.155c0-4.419 3.581-8 8-8h50.977c4.418 0 8 3.581 8 8v15.3c0 4.418-3.582 8-8 8H68.772c-4.418 0-8-3.581-8-8z"
        })
    });
const $7cd9d6fd521a0d76$var$ForwardRef = /*#__PURE__*/ (0, $8Ldph$react.forwardRef)($7cd9d6fd521a0d76$var$SvgComponent);
var $7cd9d6fd521a0d76$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7cd9d6fd521a0d76$var$ForwardRef);
