// @react-spectrum/s2/illustrations/linear/internal/S2_lin_bell_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $aTZy3$reactjsxruntime = require("react/jsx-runtime");
var $aTZy3$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a19b6fe08f7dbac7$export$2e2bcd8739ae039);




const $a19b6fe08f7dbac7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aTZy3$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $aTZy3$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24.001 5.99c-6.156 0-11.154 4.972-11.154 11.116v3.15c-.121 3.69-1.693 7.062-3.278 9.543-1.657 2.596.2 5.991 3.286 5.991h6.296v.85c0 2.679 2.172 4.85 4.85 4.85 2.679 0 4.85-2.171 4.85-4.85v-.85h6.297c3.085 0 4.942-3.395 3.285-5.99-1.585-2.482-3.156-5.855-3.277-9.544v-3.15C35.156 10.962 30.157 5.99 24 5.99m2.35 29.8h-4.7v.85c0 1.298 1.052 2.35 2.35 2.35s2.35-1.052 2.35-2.35zM15.347 20.276V17.09c.008-4.747 3.875-8.6 8.654-8.6 4.78 0 8.647 3.853 8.655 8.6v.008l-.001.042v3.135l.001.04c.137 4.327 1.961 8.153 3.67 10.829.595.93-.068 2.145-1.178 2.145H12.855c-1.11 0-1.773-1.214-1.179-2.145 1.71-2.676 3.534-6.502 3.67-10.83z"
        })
    });
const $a19b6fe08f7dbac7$var$ForwardRef = /*#__PURE__*/ (0, $aTZy3$react.forwardRef)($a19b6fe08f7dbac7$var$SvgComponent);
var $a19b6fe08f7dbac7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a19b6fe08f7dbac7$var$ForwardRef);
