// @react-spectrum/s2/illustrations/linear/internal/S2_lin_phone_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5WyBo$jsx} from "react/jsx-runtime";
import {forwardRef as $5WyBo$forwardRef} from "react";





const $09c1bd1ff5981553$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5WyBo$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5WyBo$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.25 13c0-1.519 1.231-2.75 2.75-2.75h10c1.519 0 2.75 1.231 2.75 2.75v22c0 1.519-1.231 2.75-2.75 2.75H19c-1.519 0-2.75-1.231-2.75-2.75zM19 7.75c-2.9 0-5.25 2.35-5.25 5.25v22c0 2.9 2.35 5.25 5.25 5.25h10c2.9 0 5.25-2.35 5.25-5.25V13c0-2.9-2.35-5.25-5.25-5.25zm6.5 26.75c0 .828-.672 1.5-1.5 1.5s-1.5-.672-1.5-1.5.672-1.5 1.5-1.5 1.5.672 1.5 1.5"
        })
    });
const $09c1bd1ff5981553$var$ForwardRef = /*#__PURE__*/ (0, $5WyBo$forwardRef)($09c1bd1ff5981553$var$SvgComponent);
var $09c1bd1ff5981553$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($09c1bd1ff5981553$var$ForwardRef);


export {$09c1bd1ff5981553$export$2e2bcd8739ae039 as default};
