// @react-spectrum/s2/illustrations/linear/internal/S2_lin_trash_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $iCacr$jsxs, jsx as $iCacr$jsx} from "react/jsx-runtime";
import {forwardRef as $iCacr$forwardRef} from "react";





const $842133646bcec50b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iCacr$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iCacr$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M72 60c0-2.21-1.79-4-4-4s-4 1.79-4 4v56c0 2.209 1.79 4 4 4s4-1.791 4-4zM92 56c2.21 0 4 1.79 4 4v56c0 2.209-1.79 4-4 4s-4-1.791-4-4V60c0-2.21 1.79-4 4-4"
            }),
            /*#__PURE__*/ (0, $iCacr$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M56 40v-6c0-5.523 4.477-10 10-10h28c5.523 0 10 4.477 10 10v6h31c2.209 0 4 1.79 4 4s-1.791 4-4 4h-5.438l-8.571 73.787c-.956 8.231-8.16 14.213-16.442 14.213H55.451c-8.282 0-15.486-5.982-16.442-14.213L30.438 48H25c-2.21 0-4-1.79-4-4s1.79-4 4-4zm8-6c0-1.105.895-2 2-2h28c1.105 0 2 .895 2 2v6H64zM38.492 48h83.016l-8.464 72.864c-.455 3.923-3.992 7.136-8.495 7.136H55.451c-4.503 0-8.04-3.213-8.495-7.136z"
            })
        ]
    });
const $842133646bcec50b$var$ForwardRef = /*#__PURE__*/ (0, $iCacr$forwardRef)($842133646bcec50b$var$SvgComponent);
var $842133646bcec50b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($842133646bcec50b$var$ForwardRef);


export {$842133646bcec50b$export$2e2bcd8739ae039 as default};
