// @react-spectrum/s2/illustrations/linear/internal/S2_lin_paperclip_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8IHAW$jsx} from "react/jsx-runtime";
import {forwardRef as $8IHAW$forwardRef} from "react";





const $e1e7d582993a0fdb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8IHAW$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8IHAW$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M10.302 27.026c-3.403 3.483-3.403 9.118 0 12.6 3.42 3.499 8.975 3.499 12.394 0l13.796-14.118c.483-.494.473-1.285-.02-1.768-.494-.482-1.285-.473-1.768.02L20.907 37.88c-2.438 2.495-6.378 2.495-8.817 0-2.453-2.511-2.453-6.594 0-9.105L30.985 9.438c1.548-1.584 4.045-1.584 5.593 0 1.563 1.6 1.563 4.205 0 5.805L19.64 32.574c-.639.654-1.662.654-2.3 0-.655-.67-.655-1.767-.001-2.437l11.838-12.114c.482-.494.473-1.285-.02-1.768-.494-.482-1.286-.473-1.768.02L15.552 28.39c-1.604 1.64-1.604 4.29 0 5.931 1.62 1.658 4.258 1.658 5.877 0l16.937-17.33c2.512-2.572 2.512-6.73 0-9.3-2.529-2.588-6.64-2.588-9.169 0z"
        })
    });
const $e1e7d582993a0fdb$var$ForwardRef = /*#__PURE__*/ (0, $8IHAW$forwardRef)($e1e7d582993a0fdb$var$SvgComponent);
var $e1e7d582993a0fdb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e1e7d582993a0fdb$var$ForwardRef);


export {$e1e7d582993a0fdb$export$2e2bcd8739ae039 as default};
