// @react-spectrum/s2/illustrations/linear/internal/S2_lin_paperclip_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6c0MG$jsx} from "react/jsx-runtime";
import {forwardRef as $6c0MG$forwardRef} from "react";





const $859c7c0d374dc697$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6c0MG$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6c0MG$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M21.835 53.924c-6.447 6.423-6.446 16.84 0 23.262 6.442 6.419 16.884 6.419 23.326 0L71.835 50.61c.782-.78.784-2.046.005-2.828-.78-.783-2.046-.785-2.829-.005L42.338 74.352c-4.881 4.864-12.799 4.864-17.68 0l-1.412 1.417 1.412-1.417c-4.877-4.86-4.877-12.735 0-17.594l-1.412-1.417 1.412 1.417 36.53-36.397c3.16-3.148 8.286-3.148 11.445 0 3.156 3.144 3.156 8.238 0 11.383L39.89 64.367c-1.403 1.398-3.68 1.398-5.083 0-1.398-1.393-1.398-3.65 0-5.043l22.887-22.803c.782-.78.785-2.046.005-2.828-.78-.783-2.046-.785-2.828-.005L31.984 56.49c-2.968 2.957-2.968 7.754 0 10.71 2.964 2.954 7.765 2.954 10.729 0l32.744-32.623c4.724-4.707 4.724-12.342 0-17.05-4.721-4.703-12.371-4.703-17.092 0z"
        })
    });
const $859c7c0d374dc697$var$ForwardRef = /*#__PURE__*/ (0, $6c0MG$forwardRef)($859c7c0d374dc697$var$SvgComponent);
var $859c7c0d374dc697$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($859c7c0d374dc697$var$ForwardRef);


export {$859c7c0d374dc697$export$2e2bcd8739ae039 as default};
