// @react-spectrum/s2/illustrations/linear/internal/S2_lin_error_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $3sDLJ$jsxs, jsx as $3sDLJ$jsx} from "react/jsx-runtime";
import {forwardRef as $3sDLJ$forwardRef} from "react";





const $13a71a62fb014de9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3sDLJ$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3sDLJ$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M48 32.76c1.105 0 2 .895 2 2v22.21c0 1.105-.895 2-2 2s-2-.895-2-2V34.76c0-1.105.896-2 2-2M51 67.24c0 1.657-1.343 3-3 3s-3-1.343-3-3 1.343-3 3-3 3 1.343 3 3"
            }),
            /*#__PURE__*/ (0, $3sDLJ$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M41.236 18.603c3.172-5.085 10.675-5.003 13.706.247l29.38 50.89c3.081 5.335-.774 12-6.932 12H18.62c-6.157 0-10.013-6.665-6.932-12l29.38-50.89q.076-.131.168-.247m10.242 2.247c-1.54-2.667-5.386-2.667-6.926 0q-.066.115-.146.219l-29.254 50.67c-1.538 2.665.386 6 3.468 6h58.77c3.083 0 5.007-3.334 3.468-5.999z"
            })
        ]
    });
const $13a71a62fb014de9$var$ForwardRef = /*#__PURE__*/ (0, $3sDLJ$forwardRef)($13a71a62fb014de9$var$SvgComponent);
var $13a71a62fb014de9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($13a71a62fb014de9$var$ForwardRef);


export {$13a71a62fb014de9$export$2e2bcd8739ae039 as default};
