// @react-spectrum/s2/illustrations/linear/internal/S2_lin_switch_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9I74V$jsx} from "react/jsx-runtime";
import {forwardRef as $9I74V$forwardRef} from "react";





const $9091ec7c64e591bf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9I74V$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9I74V$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M5.366 15.134c-.488-.488-.488-1.28 0-1.768l7-7c.488-.488 1.28-.488 1.768 0s.488 1.28 0 1.768L9.268 13H40.35c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H9.268l4.866 4.866c.488.488.488 1.28 0 1.768s-1.28.488-1.768 0zm29.768 10.232 7 7c.488.488.488 1.28 0 1.768l-7 7c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.768l4.866-4.866H7.25c-.69 0-1.25-.56-1.25-1.25S6.56 32 7.25 32h30.982l-4.866-4.866c-.488-.488-.488-1.28 0-1.768s1.28-.488 1.768 0"
        })
    });
const $9091ec7c64e591bf$var$ForwardRef = /*#__PURE__*/ (0, $9I74V$forwardRef)($9091ec7c64e591bf$var$SvgComponent);
var $9091ec7c64e591bf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9091ec7c64e591bf$var$ForwardRef);


export {$9091ec7c64e591bf$export$2e2bcd8739ae039 as default};
