// @react-spectrum/s2/illustrations/linear/internal/S2_lin_volumeOff_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $d7uYe$jsx} from "react/jsx-runtime";
import {forwardRef as $d7uYe$forwardRef} from "react";





const $99ead70b99ff5c52$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $d7uYe$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $d7uYe$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M6.962 5.194c-.488-.488-1.28-.488-1.768 0s-.488 1.28 0 1.768L41.04 42.807c.488.488 1.28.488 1.768 0s.488-1.28 0-1.768zm14.32 3.597c1.735-1.72 4.685-.491 4.685 1.953v8.303c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-8.303c0-.073-.02-.114-.039-.142-.023-.033-.062-.067-.115-.089-.052-.022-.103-.026-.143-.019-.034.006-.077.021-.129.072l-3.42 3.39c-.49.487-1.28.484-1.767-.007-.486-.49-.483-1.281.008-1.767zM10.617 17.29c-.35 0-.635.284-.635.634v12.127c0 .414.336.75.75.75h4.094V21.75c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v10.101l5.715 5.61c.052.05.095.066.128.072.04.006.092.002.144-.02s.091-.055.114-.089c.02-.027.04-.069.04-.142v-6.9c0-.69.559-1.25 1.25-1.25.69 0 1.25.56 1.25 1.25v6.9c0 2.438-2.937 3.67-4.677 1.963l-5.837-5.73c-.14-.137-.33-.214-.526-.214h-4.195c-1.795 0-3.25-1.455-3.25-3.25V17.924c0-1.73 1.403-3.134 3.135-3.134h.039c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25zm17.388-.042c.415-.551 1.199-.662 1.75-.247 2.115 1.592 3.487 4.127 3.487 6.982 0 .833-.117 1.641-.336 2.407-.19.664-.882 1.048-1.546.858s-1.048-.882-.858-1.546c.156-.544.24-1.12.24-1.719 0-2.036-.976-3.845-2.49-4.984-.552-.415-.662-1.199-.247-1.75m5.002-4.564c.415-.552 1.199-.662 1.75-.247 3.496 2.63 5.76 6.818 5.76 11.534 0 2.989-.91 5.77-2.47 8.074-.386.572-1.163.721-1.735.335-.572-.387-.722-1.164-.335-1.736 1.288-1.904 2.04-4.199 2.04-6.673 0-3.898-1.868-7.358-4.763-9.537-.551-.415-.662-1.198-.247-1.75"
        })
    });
const $99ead70b99ff5c52$var$ForwardRef = /*#__PURE__*/ (0, $d7uYe$forwardRef)($99ead70b99ff5c52$var$SvgComponent);
var $99ead70b99ff5c52$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($99ead70b99ff5c52$var$ForwardRef);


export {$99ead70b99ff5c52$export$2e2bcd8739ae039 as default};
