// @react-spectrum/s2/illustrations/linear/internal/S2_lin_layers_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9Ot5z$jsx} from "react/jsx-runtime";
import {forwardRef as $9Ot5z$forwardRef} from "react";





const $2964c55e046a0f58$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9Ot5z$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9Ot5z$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M45.182 18.714c1.398-.952 3.238-.952 4.636 0l26.321 17.93c1.148.781 1.148 2.471 0 3.253l-26.321 17.93c-1.398.952-3.238.952-4.636 0l-26.321-17.93c-1.148-.782-1.148-2.472 0-3.254zm6.887-3.306c-2.756-1.877-6.382-1.877-9.138 0l-26.322 17.93c-3.479 2.37-3.479 7.496 0 9.865l26.322 17.93c2.756 1.877 6.382 1.877 9.139 0l26.32-17.93c3.48-2.37 3.48-7.496 0-9.866zM22.691 53.494c.913-.622 1.149-1.866.527-2.779s-1.866-1.149-2.779-.527l-3.83 2.609c-3.479 2.37-3.479 7.496 0 9.866l26.322 17.929c2.756 1.877 6.382 1.877 9.139 0l26.32-17.93c3.48-2.37 3.48-7.495 0-9.865l-3.829-2.609c-.913-.622-2.157-.386-2.779.527s-.386 2.157.527 2.779l3.83 2.609c1.148.782 1.148 2.472 0 3.254L49.818 77.286c-1.398.952-3.238.952-4.636 0l-26.321-17.93c-1.148-.781-1.148-2.471 0-3.253z"
        })
    });
const $2964c55e046a0f58$var$ForwardRef = /*#__PURE__*/ (0, $9Ot5z$forwardRef)($2964c55e046a0f58$var$SvgComponent);
var $2964c55e046a0f58$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2964c55e046a0f58$var$ForwardRef);


export {$2964c55e046a0f58$export$2e2bcd8739ae039 as default};
