// @react-spectrum/s2/illustrations/linear/internal/S2_lin_shapes_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9KEnM$jsx} from "react/jsx-runtime";
import {forwardRef as $9KEnM$forwardRef} from "react";





const $26644073213853aa$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9KEnM$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9KEnM$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M17.569 9.889c.29-.498 1.009-.498 1.297 0l8.75 15.102c.29.5-.07 1.126-.648 1.126H9.444c-.578 0-.939-.626-.648-1.126zm3.46-1.253c-1.251-2.16-4.368-2.16-5.622-.003L6.634 23.735c-1.259 2.166.305 4.882 2.81 4.882h17.524c2.504 0 4.067-2.713 2.812-4.88zm7.449 8.601.206-.002c5.905 0 10.692 4.787 10.692 10.692S34.589 38.62 28.684 38.62c-4.832 0-8.919-3.206-10.242-7.61-.199-.661-.896-1.036-1.557-.838-.66.2-1.036.896-.837 1.557 1.632 5.432 6.67 9.39 12.636 9.39 7.285 0 13.192-5.906 13.192-13.192 0-7.285-5.907-13.192-13.192-13.192q-.128 0-.253.003c-.69.013-1.24.583-1.227 1.273.014.69.584 1.24 1.274 1.226"
        })
    });
const $26644073213853aa$var$ForwardRef = /*#__PURE__*/ (0, $9KEnM$forwardRef)($26644073213853aa$var$SvgComponent);
var $26644073213853aa$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($26644073213853aa$var$ForwardRef);


export {$26644073213853aa$export$2e2bcd8739ae039 as default};
