// @react-spectrum/s2/illustrations/linear/internal/S2_lin_minimize_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $23gXO$reactjsxruntime = require("react/jsx-runtime");
var $23gXO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7f245b13a5d556c1$export$2e2bcd8739ae039);




const $7f245b13a5d556c1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $23gXO$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $23gXO$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M134.764 31.828c1.562-1.562 1.562-4.094 0-5.656s-4.095-1.562-5.657 0L96.874 58.405v-19.8c0-2.209-1.79-4-4-4s-4 1.791-4 4v29.376c0 2.209 1.79 4 4 4h29.375c2.209 0 4-1.791 4-4s-1.791-4-4-4h-19.637zm-66.889 56.86c2.21 0 4 1.79 4 4v29.374c0 2.21-1.79 4-4 4s-4-1.79-4-4V102.45l-32.047 32.046c-1.562 1.562-4.094 1.562-5.656 0s-1.562-4.094 0-5.656l32.152-32.153H38.5c-2.21 0-4-1.79-4-4s1.79-4 4-4z"
        })
    });
const $7f245b13a5d556c1$var$ForwardRef = /*#__PURE__*/ (0, $23gXO$react.forwardRef)($7f245b13a5d556c1$var$SvgComponent);
var $7f245b13a5d556c1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7f245b13a5d556c1$var$ForwardRef);
