// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileAlert_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7WhLh$jsx} from "react/jsx-runtime";
import {forwardRef as $7WhLh$forwardRef} from "react";





const $a38723c4d8e466a0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7WhLh$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7WhLh$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20.46 22.51c0-4.41 3.572-8 7.99-8H47.9c2.048 0 4.037.783 5.527 2.217l.002.003 19.565 18.745c1.573 1.506 2.467 3.587 2.467 5.775v11.125c0 1.104-.895 2-2 2s-2-.896-2-2V41.5H53.83c-3.333 0-6.37-2.502-6.37-5.98V18.51h-19.01c-2.201 0-3.99 1.79-3.99 4V70.5c0 2.21 1.789 4 3.99 4h21.01c1.105 0 2 .895 2 2s-.895 2-2 2H28.45c-4.419 0-7.99-3.59-7.99-8zm31-2.126L69.326 37.5H53.831c-1.487 0-2.37-1.058-2.37-1.98zm2.041 50.836c0-8.13 6.59-14.72 14.72-14.72s14.72 6.59 14.72 14.72-6.59 14.72-14.72 14.72-14.72-6.59-14.72-14.72m14.72-8.96c-1.1 0-2 .9-2 2v7.68c0 1.1.9 2 2 2s2-.9 2-2v-7.68c0-1.1-.9-2-2-2m1.98 16.73c.06-.13.1-.26.13-.4.02-.14.04-.28.04-.42s-.02-.28-.04-.42c-.03-.14-.07-.27-.13-.4-.05-.13-.12-.26-.19-.38-.08-.11-.17-.22-.27-.32q-.15-.15-.33-.27c-.11-.08-.24-.14-.37-.2-.13-.05-.26-.09-.4-.12q-.42-.075-.84 0c-.14.03-.27.07-.4.12-.13.06-.26.12-.37.2q-.18.12-.33.27c-.1.1-.19.21-.27.32-.08.12-.14.25-.2.38-.05.13-.09.26-.12.4-.04.14-.04.28-.04.42s.01.28.04.42.07.27.12.4c.06.13.12.26.2.37q.12.18.27.33t.33.27c.11.07.24.14.37.19.13.06.26.1.4.13.14.04.28.04.42.04s.28-.01.42-.04.27-.07.4-.13c.13-.05.26-.12.37-.19q.18-.12.33-.27t.27-.33c.07-.11.14-.24.19-.37"
        })
    });
const $a38723c4d8e466a0$var$ForwardRef = /*#__PURE__*/ (0, $7WhLh$forwardRef)($a38723c4d8e466a0$var$SvgComponent);
var $a38723c4d8e466a0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a38723c4d8e466a0$var$ForwardRef);


export {$a38723c4d8e466a0$export$2e2bcd8739ae039 as default};
