// @react-spectrum/s2/illustrations/linear/internal/S2_lin_ruler_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $bx1JE$jsx} from "react/jsx-runtime";
import {forwardRef as $bx1JE$forwardRef} from "react";





const $cea1d7f7652b32ce$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bx1JE$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bx1JE$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M9.487 34.99c-1.66-1.66-1.66-4.351 0-6.01L28.995 9.47c1.66-1.66 4.35-1.66 6.01 0l3.523 3.523c1.66 1.66 1.66 4.35 0 6.01L19.02 38.512c-1.66 1.66-4.35 1.66-6.01 0zm1.768-4.243c-.684.683-.684 1.791 0 2.475l3.522 3.522c.684.684 1.792.684 2.475 0L36.76 17.237c.683-.684.683-1.792 0-2.475l-3.523-3.523c-.683-.683-1.791-.683-2.474 0l-1.744 1.744 2.038 2.037c.488.488.488 1.28 0 1.768-.489.488-1.28.488-1.768 0L27.25 14.75l-2.982 2.983 3.328 3.328c.488.488.488 1.28 0 1.768s-1.28.488-1.768 0l-3.328-3.328-2.984 2.984 2.037 2.037c.488.488.488 1.28 0 1.768s-1.28.488-1.767 0l-2.038-2.037-2.983 2.982 3.329 3.33c.488.487.488 1.279 0 1.767s-1.28.488-1.768 0l-3.329-3.329z"
        })
    });
const $cea1d7f7652b32ce$var$ForwardRef = /*#__PURE__*/ (0, $bx1JE$forwardRef)($cea1d7f7652b32ce$var$SvgComponent);
var $cea1d7f7652b32ce$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cea1d7f7652b32ce$var$ForwardRef);


export {$cea1d7f7652b32ce$export$2e2bcd8739ae039 as default};
