// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cloud_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $dD39T$jsx} from "react/jsx-runtime";
import {forwardRef as $dD39T$forwardRef} from "react";





const $9d44eadae0180360$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dD39T$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dD39T$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.756 15.679C18.456 11.764 22.246 9 26.699 9c5.778 0 10.433 4.64 10.87 10.437 4.572.841 7.981 4.992 7.981 9.903 0 5.504-4.285 10.06-9.685 10.06H9.464q-.075 0-.15-.009c-3.97-.139-7.064-3.522-7.064-7.576 0-3.058 1.755-5.719 4.317-6.913q-.042-.418-.043-.852c0-4.703 3.664-8.609 8.296-8.609.667 0 1.315.083 1.936.238M35.762 36.9l.062-.001h.044l.012.001c3.91-.008 7.17-3.338 7.17-7.56 0-4.077-3.045-7.326-6.775-7.546-.67-.04-1.19-.601-1.176-1.273l.003-.121.002-.067c0-4.93-3.814-8.833-8.405-8.833-3.707 0-6.897 2.535-7.998 6.116-.104.339-.348.618-.67.767s-.693.154-1.02.013c-.677-.292-1.416-.455-2.19-.455-3.151 0-5.796 2.684-5.796 6.109 0 .472.056.935.157 1.392.143.645-.241 1.29-.876 1.472-2.02.577-3.556 2.526-3.556 4.901 0 2.83 2.162 5.035 4.736 5.08q.048 0 .094.005z"
        })
    });
const $9d44eadae0180360$var$ForwardRef = /*#__PURE__*/ (0, $dD39T$forwardRef)($9d44eadae0180360$var$SvgComponent);
var $9d44eadae0180360$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9d44eadae0180360$var$ForwardRef);


export {$9d44eadae0180360$export$2e2bcd8739ae039 as default};
