// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noLibraries_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1Joxd$jsx} from "react/jsx-runtime";
import {forwardRef as $1Joxd$forwardRef} from "react";





const $8fcb104fc7dade6d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1Joxd$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1Joxd$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M11.766 10.51c-2.07 0-3.75 1.68-3.75 3.75v15.975c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25V14.26c0-3.45 2.8-6.25 6.25-6.25H27.74c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25zm-.25 6.255c0-1.519 1.23-2.75 2.75-2.75H25.07V24.87c0 .4.445.64.775.415l2.435-1.625c.17-.11.385-.11.555 0l2.435 1.625c.33.22.775-.015.775-.415V14.015h.195c1.518 0 2.75 1.231 2.75 2.75V28.5c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V16.765c0-2.9-2.35-5.25-5.25-5.25H14.266c-2.9 0-5.25 2.35-5.25 5.25V34.74c0 2.9 2.35 5.25 5.25 5.25H26c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H14.266c-1.52 0-2.75-1.231-2.75-2.75zm24.586 29.184c4.073 0 7.375-3.302 7.375-7.375S40.175 31.2 36.102 31.2s-7.375 3.302-7.375 7.375 3.302 7.375 7.375 7.375m1.375-10.75c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v2.25h-2.25c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h2.25v2.25c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-2.25h2.25c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.25z"
        })
    });
const $8fcb104fc7dade6d$var$ForwardRef = /*#__PURE__*/ (0, $1Joxd$forwardRef)($8fcb104fc7dade6d$var$SvgComponent);
var $8fcb104fc7dade6d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8fcb104fc7dade6d$var$ForwardRef);


export {$8fcb104fc7dade6d$export$2e2bcd8739ae039 as default};
