// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browserError_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $d1SJS$jsx} from "react/jsx-runtime";
import {forwardRef as $d1SJS$forwardRef} from "react";





const $9034ae55d5e646b8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $d1SJS$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $d1SJS$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M75.5 23c0-3.314-2.686-6-6-6h-52c-3.314 0-6 2.686-6 6v6h64zm-64 44V33h64v18c0 1.105.895 2 2 2s2-.895 2-2V23c0-5.523-4.477-10-10-10h-52c-5.523 0-10 4.477-10 10v44c0 5.523 4.477 10 10 10h36c1.105 0 2-.895 2-2s-.895-2-2-2h-36c-3.314 0-6-2.686-6-6m5-44c0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5-1.12-2.5-2.5-2.5-2.5 1.12-2.5 2.5m41.75 47.5c0-8.147 6.603-14.75 14.75-14.75s14.75 6.603 14.75 14.75S81.147 85.25 73 85.25 58.25 78.647 58.25 70.5m21.043 6.293c.781-.782.781-2.054 0-2.836h.02L75.856 70.5l3.457-3.457c.781-.782.781-2.054 0-2.836-.782-.781-2.054-.781-2.836 0l-3.457 3.457-3.457-3.457c-.782-.781-2.054-.781-2.836 0-.781.782-.781 2.054 0 2.836l3.457 3.457-3.457 3.457c-.781.782-.781 2.054 0 2.836.391.39.902.591 1.413.591s1.022-.2 1.413-.591l3.457-3.457 3.457 3.457c.39.39.902.591 1.413.591s1.022-.2 1.413-.591"
        })
    });
const $9034ae55d5e646b8$var$ForwardRef = /*#__PURE__*/ (0, $d1SJS$forwardRef)($9034ae55d5e646b8$var$SvgComponent);
var $9034ae55d5e646b8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9034ae55d5e646b8$var$ForwardRef);


export {$9034ae55d5e646b8$export$2e2bcd8739ae039 as default};
