// @react-spectrum/s2/illustrations/linear/internal/S2_lin_exportTo_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $gpKr9$reactjsxruntime = require("react/jsx-runtime");
var $gpKr9$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ebbd89c35b6b48ca$export$2e2bcd8739ae039);




const $ebbd89c35b6b48ca$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gpKr9$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gpKr9$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25 25.492c-3.313 0-6 2.686-6 6v40c0 3.314 2.687 6 6 6h40c3.314 0 6-2.686 6-6v-29c0-1.104.896-2 2-2s2 .896 2 2v29c0 5.523-4.477 10-10 10H25c-5.522 0-10-4.477-10-10v-40c0-5.523 4.478-10 10-10h29c1.105 0 2 .896 2 2s-.895 2-2 2zm22.586 23.48c-.78-.78-.78-2.047 0-2.828l26.66-26.66H64c-1.105 0-2-.896-2-2s.895-2 2-2h14.967c1.104 0 2 .895 2 2V32.45c0 1.105-.896 2-2 2s-2-.895-2-2V22.42L50.415 48.972c-.781.781-2.048.781-2.829 0"
        })
    });
const $ebbd89c35b6b48ca$var$ForwardRef = /*#__PURE__*/ (0, $gpKr9$react.forwardRef)($ebbd89c35b6b48ca$var$SvgComponent);
var $ebbd89c35b6b48ca$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ebbd89c35b6b48ca$var$ForwardRef);
