// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cursor_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8a5Bn$jsx} from "react/jsx-runtime";
import {forwardRef as $8a5Bn$forwardRef} from "react";





const $07259d1f8e041d02$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8a5Bn$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8a5Bn$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24.26 44.52C18.75 31.133 32.133 17.75 45.518 23.26l86.584 35.652c14.248 5.867 14.609 25.915.582 32.292l-24.554 11.16c-2.112.961-3.805 2.654-4.766 4.766l-11.16 24.554c-6.376 14.027-26.425 13.666-32.292-.582zm18.212-13.862c-6.809-2.803-13.618 4.006-10.815 10.815l-3.698 1.523 3.698-1.523 35.653 86.584c3.2 7.771 14.133 7.968 17.61.318l11.162-24.554c1.76-3.874 4.864-6.978 8.738-8.739l24.554-11.16c7.65-3.478 7.453-14.412-.318-17.611z"
        })
    });
const $07259d1f8e041d02$var$ForwardRef = /*#__PURE__*/ (0, $8a5Bn$forwardRef)($07259d1f8e041d02$var$SvgComponent);
var $07259d1f8e041d02$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($07259d1f8e041d02$var$ForwardRef);


export {$07259d1f8e041d02$export$2e2bcd8739ae039 as default};
