// @react-spectrum/s2/illustrations/linear/internal/S2_lin_information_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8BzIJ$jsx} from "react/jsx-runtime";
import {forwardRef as $8BzIJ$forwardRef} from "react";





const $52bc408119c150ef$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8BzIJ$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8BzIJ$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28 79.74c0-28.719 23.281-52 52-52s52 23.281 52 52-23.281 52-52 52-52-23.281-52-52m52-60c-33.137 0-60 26.863-60 60s26.863 60 60 60 60-26.863 60-60-26.863-60-60-60m5.5 36c0 3.314-2.686 6-6 6s-6-2.686-6-6 2.686-6 6-6 6 2.686 6 6m-18 16c0-2.209 1.79-4 4-4h9c2.21 0 4 1.791 4 4v27h5c2.21 0 4 1.791 4 4s-1.79 4-4 4h-18c-2.21 0-4-1.791-4-4s1.79-4 4-4h5v-23h-5c-2.21 0-4-1.79-4-4"
        })
    });
const $52bc408119c150ef$var$ForwardRef = /*#__PURE__*/ (0, $8BzIJ$forwardRef)($52bc408119c150ef$var$SvgComponent);
var $52bc408119c150ef$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($52bc408119c150ef$var$ForwardRef);


export {$52bc408119c150ef$export$2e2bcd8739ae039 as default};
