// @react-spectrum/s2/illustrations/linear/internal/S2_lin_video_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $fdGH5$reactjsxruntime = require("react/jsx-runtime");
var $fdGH5$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $61eaa0e68c918e45$export$2e2bcd8739ae039);




const $61eaa0e68c918e45$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fdGH5$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fdGH5$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M82 28.4c0-3.313-2.686-6-6-6H20c-3.314 0-6 2.687-6 6v39.2c0 3.314 2.686 6 6 6h56c3.314 0 6-2.686 6-6zm-6-10c5.523 0 10 4.478 10 10v39.2c0 5.523-4.477 10-10 10H20c-5.523 0-10-4.477-10-10V28.4c0-5.522 4.477-10 10-10zM44.846 40.255l11.214 6.432c1.256.72 1.254 2.518-.004 3.235l-11.23 6.406c-1.257.717-2.827-.184-2.825-1.621l.016-12.838c.001-1.438 1.573-2.335 2.83-1.614"
        })
    });
const $61eaa0e68c918e45$var$ForwardRef = /*#__PURE__*/ (0, $fdGH5$react.forwardRef)($61eaa0e68c918e45$var$SvgComponent);
var $61eaa0e68c918e45$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($61eaa0e68c918e45$var$ForwardRef);
