// @react-spectrum/s2/illustrations/linear/internal/S2_lin_buildings_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $cGrVz$jsx} from "react/jsx-runtime";
import {forwardRef as $cGrVz$forwardRef} from "react";





const $7de23cca49855950$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cGrVz$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cGrVz$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M12.95 10.52c-2.62 0-4.75 2.13-4.75 4.75v22.22h4.849v-4.83c0-1.79 1.46-3.25 3.25-3.25h3.95c1.79 0 3.25 1.46 3.25 3.25v4.83h4.86V15.27c0-2.62-2.13-4.75-4.75-4.75zm9.457 29.47h7.202c.69 0 1.25-.56 1.25-1.25V15.27c0-4-3.25-7.25-7.25-7.25H12.95c-4 0-7.25 3.25-7.25 7.25v23.47c0 .69.56 1.25 1.25 1.25h7.192q.078.01.158.01h7.95q.08 0 .158-.01m-1.408-2.5v-4.83c0-.41-.34-.75-.75-.75h-3.95c-.41 0-.75.34-.75.75v4.83zm11.37-22.1c0-.691.56-1.25 1.25-1.25h1.43c4 0 7.25 3.249 7.25 7.25v17.35c0 .69-.56 1.25-1.25 1.25h-7.43c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h6.18v-16.1c0-2.62-2.13-4.75-4.75-4.75h-1.43c-.69 0-1.25-.56-1.25-1.25m-17.14-1.95h-1.9c-.552 0-1 .447-1 1v1.9c0 .552.448 1 1 1h1.9c.552 0 1-.448 1-1v-1.9c0-.553-.448-1-1-1m6.1 0h1.9c.552 0 1 .447 1 1v1.9c0 .552-.448 1-1 1h-1.9c-.552 0-1-.448-1-1v-1.9c0-.553.448-1 1-1m-6.1 7.94h-1.9c-.552 0-1 .447-1 1v1.9c0 .552.448 1 1 1h1.9c.552 0 1-.448 1-1v-1.9c0-.553-.448-1-1-1m6.1 0h1.9c.552 0 1 .447 1 1v1.9c0 .552-.448 1-1 1h-1.9c-.552 0-1-.448-1-1v-1.9c0-.553.448-1 1-1m14.95-.23h-1.9c-.552 0-1 .447-1 1v1.9c0 .552.448 1 1 1h1.9c.552 0 1-.448 1-1v-1.9c0-.553-.448-1-1-1m-1.9 7.94h1.9c.552 0 1 .447 1 1v1.9c0 .552-.448 1-1 1h-1.9c-.552 0-1-.448-1-1v-1.9c0-.553.448-1 1-1"
        })
    });
const $7de23cca49855950$var$ForwardRef = /*#__PURE__*/ (0, $cGrVz$forwardRef)($7de23cca49855950$var$SvgComponent);
var $7de23cca49855950$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7de23cca49855950$var$ForwardRef);


export {$7de23cca49855950$export$2e2bcd8739ae039 as default};
