// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pencil_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7S26J$jsx} from "react/jsx-runtime";
import {forwardRef as $7S26J$forwardRef} from "react";





const $78954ee8d6ae598e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7S26J$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7S26J$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M10.716 28.847c.163-.435.417-.83.745-1.158L30.067 9.083c2.444-2.444 6.407-2.444 8.851 0s2.445 6.408 0 8.852L20.313 36.541c-.328.328-.724.582-1.159.745l-8.86 3.318c-1.808.677-3.573-1.088-2.896-2.896zm3.112.01 5.316 5.317 15.172-15.172L29 13.685zm3.068 6.605-4.356-4.357-2.608 6.965zm20.255-19.295-1.067 1.067-5.316-5.317 1.066-1.066c1.468-1.468 3.849-1.468 5.317 0s1.468 3.848 0 5.316"
        })
    });
const $78954ee8d6ae598e$var$ForwardRef = /*#__PURE__*/ (0, $7S26J$forwardRef)($78954ee8d6ae598e$var$SvgComponent);
var $78954ee8d6ae598e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($78954ee8d6ae598e$var$ForwardRef);


export {$78954ee8d6ae598e$export$2e2bcd8739ae039 as default};
