// @react-spectrum/s2/illustrations/linear/internal/S2_lin_briefcase_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1DQl1$jsx} from "react/jsx-runtime";
import {forwardRef as $1DQl1$forwardRef} from "react";





const $7d883032d920ac31$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1DQl1$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1DQl1$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M59 32c-3.31 0-6 2.69-6 6v7h54v-7c0-3.31-2.689-6-6-6zm-14 6v7h-9c-11.046 0-20 8.954-20 20v51c0 11.046 8.954 20 20 20h88c11.046 0 20-8.954 20-20V65c0-11.046-8.954-20-20-20h-9v-7c0-7.73-6.271-14-14-14H59c-7.73 0-14 6.27-14 14m-9 15c-6.627 0-12 5.373-12 12v22h21v-2c0-2.21 1.79-4 4-4s4 1.79 4 4v2h54v-2c0-2.21 1.791-4 4-4s4 1.79 4 4v2h21V65c0-6.627-5.373-12-12-12zm9 42v3c0 2.209 1.79 4 4 4s4-1.791 4-4v-3h54v3c0 2.209 1.791 4 4 4s4-1.791 4-4v-3h21v21c0 6.627-5.373 12-12 12H36c-6.627 0-12-5.373-12-12V95z"
        })
    });
const $7d883032d920ac31$var$ForwardRef = /*#__PURE__*/ (0, $1DQl1$forwardRef)($7d883032d920ac31$var$SvgComponent);
var $7d883032d920ac31$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7d883032d920ac31$var$ForwardRef);


export {$7d883032d920ac31$export$2e2bcd8739ae039 as default};
