// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowUp_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8L6Wf$jsx} from "react/jsx-runtime";
import {forwardRef as $8L6Wf$forwardRef} from "react";





const $1de6accd79dbdbac$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8L6Wf$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8L6Wf$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M46 81c0 1.105.895 2 2 2s2-.895 2-2V18q0-.09-.008-.18l10.594 10.594c.78.781 2.047.781 2.828 0 .781-.78.781-2.047 0-2.828l-14-14c-.78-.781-2.047-.781-2.828 0l-14 14c-.781.78-.781 2.047 0 2.828.78.781 2.047.781 2.828 0L46.008 17.82Q46 17.91 46 18zm1.82-64.992Q47.91 16 48 16t.18.008l-.18-.18z"
        })
    });
const $1de6accd79dbdbac$var$ForwardRef = /*#__PURE__*/ (0, $8L6Wf$forwardRef)($1de6accd79dbdbac$var$SvgComponent);
var $1de6accd79dbdbac$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1de6accd79dbdbac$var$ForwardRef);


export {$1de6accd79dbdbac$export$2e2bcd8739ae039 as default};
