// @react-spectrum/s2/illustrations/linear/internal/S2_lin_ribbon_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2lQxJ$reactjsxruntime = require("react/jsx-runtime");
var $2lQxJ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1deafd3388a7628d$export$2e2bcd8739ae039);




const $1deafd3388a7628d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2lQxJ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2lQxJ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.995 17c-10.77 0-19.5 8.73-19.5 19.5s8.73 19.5 19.5 19.5 19.5-8.73 19.5-19.5-8.73-19.5-19.5-19.5m-23.5 19.5c0-12.979 10.522-23.5 23.5-23.5 12.98 0 23.5 10.521 23.5 23.5 0 6.514-2.65 12.41-6.932 16.666l6.915 17.116c1.193 2.952-1.374 6.047-4.495 5.42l-4.462-.895-2.588 3.743c-1.81 2.62-5.806 2.176-6.999-.776L47.967 65.48l-4.9 12.129c-1.197 2.962-5.212 3.395-7.013.756l-2.562-3.753-4.45.92c-3.129.648-5.716-2.453-4.52-5.415l6.865-16.99c-4.258-4.253-6.892-10.132-6.892-16.626m10.118 19.32-6.381 15.794 4.45-.92c1.581-.328 3.204.328 4.114 1.662l2.562 3.754 6.452-15.97-.101-.25c-4.098-.395-7.888-1.843-11.096-4.07m15.42 4.093c4.173-.358 8.036-1.808 11.3-4.062l6.436 15.93-4.461-.896c-1.566-.314-3.169.334-4.077 1.648l-2.588 3.743z"
        })
    });
const $1deafd3388a7628d$var$ForwardRef = /*#__PURE__*/ (0, $2lQxJ$react.forwardRef)($1deafd3388a7628d$var$SvgComponent);
var $1deafd3388a7628d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1deafd3388a7628d$var$ForwardRef);
