// @react-spectrum/s2/illustrations/linear/internal/S2_lin_speedFast_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8sVsN$jsx} from "react/jsx-runtime";
import {forwardRef as $8sVsN$forwardRef} from "react";





const $f01e8b7a062c0cf5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8sVsN$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8sVsN$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M17.908 74.088c-2.868-4.955-4.549-10.682-4.655-16.788q-.015-.12-.015-.245 0-.102.01-.2v-.17c0-19.212 15.574-34.787 34.786-34.787 8.53 0 16.346 3.073 22.397 8.169.845.711.953 1.973.241 2.818-.711.845-1.973.953-2.818.241q-1.222-1.03-2.548-1.93l-2.618 4.535c-.553.956-1.776 1.284-2.733.732-.956-.553-1.284-1.776-.732-2.732l2.635-4.563c-3.605-1.814-7.614-2.94-11.858-3.208v5.31c0 1.104-.895 2-2 2s-2-.896-2-2v-5.305c-4.209.274-8.187 1.394-11.766 3.192l2.641 4.574c.552.956.224 2.18-.732 2.732-.957.552-2.18.224-2.732-.732l-2.627-4.55c-3.389 2.297-6.296 5.251-8.537 8.68l4.43 2.558c.956.552 1.284 1.775.731 2.732-.552.956-1.775 1.284-2.732.732l-4.396-2.538c-1.72 3.574-2.772 7.532-2.99 11.71h4.96c1.105 0 2 .895 2 2s-.895 2-2 2h-4.912c.359 4.723 1.783 9.144 4.032 13.029.553.956.226 2.18-.73 2.733-.955.553-2.179.227-2.732-.73m62.837-29.265c-.377-1.039-1.524-1.575-2.562-1.199-1.039.377-1.575 1.524-1.198 2.563 1.011 2.788 1.632 5.766 1.793 8.868h-4.97c-1.104 0-2 .895-2 2s.896 2 2 2h4.923c-.36 4.723-1.784 9.144-4.032 13.029-.554.956-.227 2.18.729 2.733s2.18.227 2.733-.73c2.888-4.988 4.572-10.761 4.656-16.915q.004-.059.004-.117 0-.036-.002-.072l.001-.298c0-4.162-.731-8.158-2.075-11.862M79.517 32.82c-1.171-1.172-3.054-1.221-4.285-.113l-30.39 27.357c-2.24 2.017-2.332 5.5-.2 7.63 2.131 2.132 5.614 2.041 7.631-.2L79.63 37.107c1.108-1.231 1.058-3.114-.113-4.285"
        })
    });
const $f01e8b7a062c0cf5$var$ForwardRef = /*#__PURE__*/ (0, $8sVsN$forwardRef)($f01e8b7a062c0cf5$var$SvgComponent);
var $f01e8b7a062c0cf5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f01e8b7a062c0cf5$var$ForwardRef);


export {$f01e8b7a062c0cf5$export$2e2bcd8739ae039 as default};
