// @react-spectrum/s2/illustrations/linear/internal/S2_lin_graphBarChart_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $hTPDO$jsx} from "react/jsx-runtime";
import {forwardRef as $hTPDO$forwardRef} from "react";





const $468d16b068570448$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hTPDO$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hTPDO$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.21 15.09c0-1.105-.895-2-2-2s-2 .895-2 2V82c0 1.104.896 2 2 2H81c1.106 0 2-.896 2-2s-.894-2-2-2H16.21zm9.111 31.14c0-1.105.896-2 2-2h3.54c1.105 0 2 .895 2 2v22.54c0 1.104-.895 2-2 2h-3.54c-1.104 0-2-.896-2-2zm2-6c-3.313 0-6 2.686-6 6v22.54c0 3.314 2.687 6 6 6h3.54c3.314 0 6-2.686 6-6V46.23c0-3.314-2.686-6-6-6zm41.53 18.39c-1.104 0-2 .895-2 2v8.15c0 1.104.896 2 2 2h3.54c1.105 0 2-.896 2-2v-8.15c0-1.105-.895-2-2-2zm-6 2c0-3.314 2.687-6 6-6h3.54c3.314 0 6 2.686 6 6v8.15c0 3.314-2.686 6-6 6h-3.54c-3.313 0-6-2.686-6-6zm-16.77-32.3c0-1.105.896-2 2-2h3.54c1.105 0 2 .895 2 2v40.45c0 1.104-.895 2-2 2h-3.54c-1.104 0-2-.896-2-2zm2-6c-3.313 0-6 2.686-6 6v40.45c0 3.314 2.687 6 6 6h3.54c3.314 0 6-2.686 6-6V28.32c0-3.314-2.686-6-6-6z"
        })
    });
const $468d16b068570448$var$ForwardRef = /*#__PURE__*/ (0, $hTPDO$forwardRef)($468d16b068570448$var$SvgComponent);
var $468d16b068570448$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($468d16b068570448$var$ForwardRef);


export {$468d16b068570448$export$2e2bcd8739ae039 as default};
