// @react-spectrum/s2/illustrations/linear/internal/S2_lin_user_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6r4o1$reactjsxruntime = require("react/jsx-runtime");
var $6r4o1$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0a444b4323cb6e17$export$2e2bcd8739ae039);




const $0a444b4323cb6e17$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6r4o1$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6r4o1$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M36.39 29.5C36.39 22.475 41.706 17 48 17c6.295 0 11.61 5.475 11.61 12.5S54.295 42 48 42s-11.61-5.475-11.61-12.5M48 13c-8.738 0-15.61 7.509-15.61 16.5C32.39 38.492 39.262 46 48 46c8.739 0 15.61-7.508 15.61-16.5C63.61 20.509 56.74 13 48 13M23.01 73.598C25.107 61.72 35.585 52.59 48.002 52.59s22.894 9.14 24.99 21.007c.217 1.245-.733 2.363-1.97 2.363h-46.04c-1.244 0-2.191-1.133-1.972-2.356zM48.002 48.59c-14.402 0-26.503 10.567-28.93 24.31-.657 3.675 2.175 7.06 5.91 7.06h46.04c3.742 0 6.552-3.38 5.91-7.054v-.004C74.506 59.17 62.403 48.59 48 48.59"
        })
    });
const $0a444b4323cb6e17$var$ForwardRef = /*#__PURE__*/ (0, $6r4o1$react.forwardRef)($0a444b4323cb6e17$var$SvgComponent);
var $0a444b4323cb6e17$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0a444b4323cb6e17$var$ForwardRef);
