// @react-spectrum/s2/illustrations/linear/internal/S2_lin_3Dasset_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $jOMTD$reactjsxruntime = require("react/jsx-runtime");
var $jOMTD$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $de046d91693333e9$export$2e2bcd8739ae039);




const $de046d91693333e9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jOMTD$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jOMTD$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M68.957 17.023c-1.815-1.01-4.023-1.01-5.837 0L48.974 24.9c-.605.337-1.341.337-1.946 0l-13.555-7.546c-2.167-1.207-4.915-1.088-6.984.09-2.403 1.368-6.21 3.466-9.834 5.447-1.823.996-2.852 2.707-3.082 4.506q-.064.245-.065.51v24.495c0 3.387 1.715 6.544 4.556 8.388L46.882 79.49c.661.43 1.513.43 2.175.003l28.94-18.703c2.852-1.842 4.573-5.005 4.573-8.399V27.906q0-.182-.031-.356c-.175-1.85-1.193-3.63-3.055-4.666zm9.613 16.855L49.971 49.8v24.339L75.827 57.43c1.71-1.105 2.743-3.002 2.743-5.039zM45.971 49.767 17.508 33.921V52.4c0 2.032 1.029 3.927 2.734 5.033l25.73 16.697zm19.095-29.249c.604-.337 1.34-.337 1.945 0L77.54 26.38c1.37.763 1.37 2.733 0 3.495L48 46.319 18.553 29.924c-1.377-.766-1.374-2.762.02-3.524 3.625-1.98 7.459-4.093 9.895-5.48.942-.537 2.2-.55 3.06-.072l13.554 7.546c1.815 1.01 4.023 1.01 5.837 0z"
        })
    });
const $de046d91693333e9$var$ForwardRef = /*#__PURE__*/ (0, $jOMTD$react.forwardRef)($de046d91693333e9$var$SvgComponent);
var $de046d91693333e9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($de046d91693333e9$var$ForwardRef);
