// @react-spectrum/s2/illustrations/linear/internal/S2_lin_calendar_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1XCaY$jsx} from "react/jsx-runtime";
import {forwardRef as $1XCaY$forwardRef} from "react";





const $e84208b3ba16b508$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1XCaY$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1XCaY$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M34 14c1.105 0 2 .895 2 2v4h24v-4c0-1.105.895-2 2-2s2 .895 2 2v4h6c5.523 0 10 4.477 10 10v40c0 5.523-4.477 10-10 10H26c-5.523 0-10-4.477-10-10V30c0-5.523 4.477-10 10-10h6v-4c0-1.105.895-2 2-2m26 10v4c0 1.105.895 2 2 2s2-.895 2-2v-4h6c3.314 0 6 2.686 6 6v7H20v-7c0-3.314 2.686-6 6-6h6v4c0 1.105.895 2 2 2s2-.895 2-2v-4zM20 41v29c0 3.314 2.686 6 6 6h44c3.314 0 6-2.686 6-6V41zm14 7c1.105 0 2 .895 2 2v4c0 1.105-.895 2-2 2h-4c-1.105 0-2-.895-2-2v-4c0-1.105.895-2 2-2zm16 0h-4c-1.105 0-2 .895-2 2v4c0 1.105.895 2 2 2h4c1.105 0 2-.895 2-2v-4c0-1.105-.895-2-2-2m10 2c0-1.105.895-2 2-2h4c1.105 0 2 .895 2 2v4c0 1.105-.895 2-2 2h-4c-1.105 0-2-.895-2-2zM30 61c-1.105 0-2 .895-2 2v4c0 1.105.895 2 2 2h4c1.105 0 2-.895 2-2v-4c0-1.105-.895-2-2-2zm16 0h4c1.105 0 2 .895 2 2v4c0 1.105-.895 2-2 2h-4c-1.105 0-2-.895-2-2v-4c0-1.105.895-2 2-2m22 2c0-1.105-.895-2-2-2h-4c-1.105 0-2 .895-2 2v4c0 1.105.895 2 2 2h4c1.105 0 2-.895 2-2z"
        })
    });
const $e84208b3ba16b508$var$ForwardRef = /*#__PURE__*/ (0, $1XCaY$forwardRef)($e84208b3ba16b508$var$SvgComponent);
var $e84208b3ba16b508$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e84208b3ba16b508$var$ForwardRef);


export {$e84208b3ba16b508$export$2e2bcd8739ae039 as default};
