// @react-spectrum/s2/illustrations/linear/internal/S2_lin_3D_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $dwpdp$jsx} from "react/jsx-runtime";
import {forwardRef as $dwpdp$forwardRef} from "react";





const $bf342ae106723e6a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dwpdp$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dwpdp$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M43.246 12.868c3.141-1.843 7.035-1.834 10.167.025L77.4 27.133c3.035 1.802 4.895 5.07 4.895 8.6v26.84c0 3.624-1.96 6.964-5.122 8.73L53.187 84.706c-3.017 1.686-6.689 1.695-9.713.024l-24.31-13.428C15.979 69.541 14 66.188 14 62.548v-26.79c0-3.547 1.88-6.828 4.938-8.624zm8.125 3.465c-1.879-1.115-4.216-1.121-6.1-.015L20.963 30.584c-.474.278-.9.616-1.273 1l28.468 15.192 28.69-14.952c-.42-.488-.92-.913-1.49-1.251zM18 35.759q0-.267.023-.53l28.308 15.106V81.64q-.475-.166-.923-.412L21.098 67.8C19.188 66.744 18 64.732 18 62.548zm32.331 45.866q.465-.166.905-.412l23.986-13.401c1.897-1.06 3.073-3.064 3.073-5.238V35.732q0-.075-.002-.15L50.331 50.153z"
        })
    });
const $bf342ae106723e6a$var$ForwardRef = /*#__PURE__*/ (0, $dwpdp$forwardRef)($bf342ae106723e6a$var$SvgComponent);
var $bf342ae106723e6a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($bf342ae106723e6a$var$ForwardRef);


export {$bf342ae106723e6a$export$2e2bcd8739ae039 as default};
