// @react-spectrum/s2/illustrations/linear/internal/S2_lin_codeBrackets_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2gJyb$reactjsxruntime = require("react/jsx-runtime");
var $2gJyb$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c70e1a2ad943f8ab$export$2e2bcd8739ae039);




const $c70e1a2ad943f8ab$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2gJyb$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2gJyb$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 48c0-6.627 5.373-12 12-12h88c6.627 0 12 5.373 12 12v64c0 6.627-5.373 12-12 12H36c-6.627 0-12-5.373-12-12zm12-20c-11.046 0-20 8.954-20 20v64c0 11.046 8.954 20 20 20h88c11.046 0 20-8.954 20-20V48c0-11.046-8.954-20-20-20zm26.658 35.79c1.563-1.562 1.563-4.095.002-5.658-1.562-1.562-4.095-1.563-5.657-.001l-18.39 18.38c-1.525 1.524-1.568 3.983-.096 5.559l18.39 19.7c1.507 1.614 4.038 1.701 5.653.194s1.702-4.039.194-5.653L47.002 79.436zm34.783-5.754c1.615-1.507 4.146-1.42 5.654.194l18.39 19.7c1.471 1.576 1.429 4.035-.096 5.56l-18.39 18.379c-1.563 1.562-4.096 1.561-5.657-.001-1.562-1.563-1.561-4.096.001-5.657L113 80.564 97.247 63.69c-1.508-1.615-1.42-4.146.194-5.654m-5.073-5.056c.64-2.114-.554-4.347-2.668-4.988-2.114-.64-4.347.554-4.988 2.668l-17.08 56.36c-.64 2.114.554 4.347 2.668 4.988s4.347-.554 4.988-2.668z"
        })
    });
const $c70e1a2ad943f8ab$var$ForwardRef = /*#__PURE__*/ (0, $2gJyb$react.forwardRef)($c70e1a2ad943f8ab$var$SvgComponent);
var $c70e1a2ad943f8ab$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c70e1a2ad943f8ab$var$ForwardRef);
