// @react-spectrum/s2/illustrations/linear/internal/S2_lin_socialPost_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $iLb53$reactjsxruntime = require("react/jsx-runtime");
var $iLb53$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $910f852ccec40149$export$2e2bcd8739ae039);




const $910f852ccec40149$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iLb53$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        fill: "none",
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $iLb53$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.313 27.313c-11.046 0-20 8.954-20 20v65.375c0 11.045 8.954 20 20 20h65.374c11.046 0 20-8.955 20-20V69.093l.001-.049-.001-.048V47.313c0-11.046-8.954-20-20-20zm77.374 45.731v39.644c0 .844-.087 1.668-.253 2.463l-11.691-17.32-.005-.007c-3.996-5.96-12.755-5.945-16.752 0l-.017.025-9.52 14.393-17.253-26.624-.002-.003c-3.974-6.141-12.97-6.14-16.942.003l-16.94 26.171V73.044zm-46.469 51.644 3.416-5.165L62.48 89.965l-.002-.004c-.82-1.268-2.687-1.268-3.507 0l-.002.002-20.178 31.174c2.175 2.193 5.19 3.551 8.522 3.551zm9.592 0 14.821-22.41c.838-1.232 2.647-1.217 3.464.003l.008.013 13.593 20.135c-1.972 1.421-4.392 2.259-7.009 2.259zM35.313 65.044h89.374V47.313c0-6.628-5.372-12-12-12H47.313c-6.628 0-12 5.372-12 12zm8.062-14.866c0-2.21 1.791-4 4-4l65.375.01c2.21 0 4 1.79 4 4s-1.791 4-4.001 4l-65.375-.01c-2.209 0-4-1.791-3.999-4m65.338 32.65c0 3.313 2.686 6 6 6s6-2.687 6-6-2.686-6-6-6-6 2.686-6 6",
            clipRule: "evenodd"
        })
    });
const $910f852ccec40149$var$ForwardRef = /*#__PURE__*/ (0, $iLb53$react.forwardRef)($910f852ccec40149$var$SvgComponent);
var $910f852ccec40149$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($910f852ccec40149$var$ForwardRef);
