// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lighten_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dOion$reactjsxruntime = require("react/jsx-runtime");
var $dOion$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $41ab60ab4884f47e$export$2e2bcd8739ae039);




const $41ab60ab4884f47e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dOion$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dOion$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 20c2.21 0 4 1.79 4 4v7.938c0 2.21-1.79 4-4 4s-4-1.79-4-4V24c0-2.21 1.79-4 4-4m0 35.719c-13.412 0-24.284 10.872-24.284 24.283s10.872 24.284 24.283 24.284 24.284-10.872 24.284-24.284S93.411 55.72 79.999 55.72M47.716 80.002c0-17.83 14.454-32.283 32.283-32.283s32.284 14.454 32.284 32.283-14.454 32.284-32.284 32.284-32.283-14.454-32.283-32.284M84 128.062c0-2.209-1.79-4-4-4s-4 1.791-4 4V136c0 2.209 1.791 4 4 4s4-1.791 4-4zM140 80c0 2.21-1.791 4-4 4h-7.938c-2.209 0-4-1.79-4-4s1.791-4 4-4H136c2.209 0 4 1.79 4 4M31.938 84c2.21 0 4-1.79 4-4s-1.79-4-4-4H24c-2.21 0-4 1.79-4 4s1.79 4 4 4zm90.488-46.427c1.562 1.562 1.562 4.095 0 5.657l-5.613 5.613c-1.562 1.562-4.095 1.562-5.657 0s-1.562-4.095 0-5.657l5.613-5.613c1.562-1.562 4.095-1.562 5.657 0m-73.583 79.24c1.562-1.562 1.562-4.095 0-5.657s-4.095-1.562-5.657 0l-5.613 5.613c-1.562 1.563-1.562 4.095 0 5.657s4.094 1.562 5.657 0zm-11.27-79.242c-1.563 1.562-1.563 4.095 0 5.657l5.612 5.613c1.563 1.562 4.095 1.562 5.657 0s1.562-4.095 0-5.657l-5.613-5.613c-1.562-1.562-4.095-1.562-5.657 0m73.583 79.24c-1.562-1.562-1.562-4.094 0-5.657s4.095-1.562 5.657 0l5.613 5.614c1.562 1.562 1.562 4.094 0 5.656-1.562 1.563-4.095 1.563-5.657 0z"
        })
    });
const $41ab60ab4884f47e$var$ForwardRef = /*#__PURE__*/ (0, $dOion$react.forwardRef)($41ab60ab4884f47e$var$SvgComponent);
var $41ab60ab4884f47e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($41ab60ab4884f47e$var$ForwardRef);
