// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowUp_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2vgvf$reactjsxruntime = require("react/jsx-runtime");
var $2vgvf$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0c2ff5e0fef642dd$export$2e2bcd8739ae039);




const $0c2ff5e0fef642dd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2vgvf$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2vgvf$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M76 136c0 2.209 1.79 4 4 4s4-1.791 4-4V31.157l16.672 16.671c1.562 1.563 4.094 1.563 5.656 0 1.563-1.562 1.563-4.094 0-5.656l-23.5-23.5c-1.562-1.562-4.094-1.562-5.656 0l-23.5 23.5c-1.562 1.562-1.562 4.094 0 5.656 1.562 1.563 4.094 1.563 5.656 0L76 31.157z"
        })
    });
const $0c2ff5e0fef642dd$var$ForwardRef = /*#__PURE__*/ (0, $2vgvf$react.forwardRef)($0c2ff5e0fef642dd$var$SvgComponent);
var $0c2ff5e0fef642dd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0c2ff5e0fef642dd$var$ForwardRef);
