// @react-spectrum/s2/illustrations/linear/internal/S2_lin_trash_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $cXLh2$jsxs, jsx as $cXLh2$jsx} from "react/jsx-runtime";
import {forwardRef as $cXLh2$forwardRef} from "react";





const $e2d75e95eeef61c1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cXLh2$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cXLh2$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M22.5 18c0-.69-.56-1.25-1.25-1.25S20 17.31 20 18v16c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25zM27.25 16.75c.69 0 1.25.56 1.25 1.25v16c0 .69-.56 1.25-1.25 1.25S26 34.69 26 34V18c0-.69.56-1.25 1.25-1.25"
            }),
            /*#__PURE__*/ (0, $cXLh2$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M17 12.75v-1.985c0-1.688 1.389-3.015 3.05-3.015h8.4c1.661 0 3.05 1.327 3.05 3.015v1.985h7.75c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-.886l-2.41 20.86c-.278 2.396-2.364 4.14-4.766 4.14H17.312c-2.402 0-4.489-1.744-4.765-4.14l-2.41-20.86H9.25C8.56 15.25 8 14.69 8 14s.56-1.25 1.25-1.25zm2.5-1.985c0-.261.223-.515.55-.515h8.4c.327 0 .55.254.55.515v1.985h-9.5zm-6.847 4.485h23.194L33.47 35.824c-.122 1.058-1.072 1.926-2.282 1.926H17.312c-1.21 0-2.16-.868-2.282-1.926z"
            })
        ]
    });
const $e2d75e95eeef61c1$var$ForwardRef = /*#__PURE__*/ (0, $cXLh2$forwardRef)($e2d75e95eeef61c1$var$SvgComponent);
var $e2d75e95eeef61c1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e2d75e95eeef61c1$var$ForwardRef);


export {$e2d75e95eeef61c1$export$2e2bcd8739ae039 as default};
