// @react-spectrum/s2/illustrations/linear/internal/S2_lin_color_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $gsKpf$reactjsxruntime = require("react/jsx-runtime");
var $gsKpf$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d71adc98566caa69$export$2e2bcd8739ae039);




const $d71adc98566caa69$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gsKpf$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gsKpf$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25.485 8.5C16.665 8.5 9.5 15.72 9.5 24.648c0 8.784 8.156 15.323 17.128 15.323 2.413 0 5.532-.743 7.97-1.92 1.218-.589 2.204-1.256 2.842-1.937.64-.683.839-1.272.772-1.77-.11-.814-.39-1.169-.655-1.37-.317-.24-.75-.37-1.38-.499l-.25-.05c-.522-.1-1.202-.233-1.76-.517-.358-.182-.708-.442-.986-.824s-.448-.835-.521-1.35c-.195-1.376.07-2.49.628-3.442.532-.91 1.309-1.632 2.028-2.27q.143-.127.285-.25c1.388-1.222 2.642-2.326 3.12-4.305.609-2.526-.392-5.221-2.748-7.348C33.619 9.995 29.957 8.5 25.484 8.5M7.5 24.648C7.5 14.634 15.543 6.5 25.485 6.5c4.907 0 9.064 1.64 11.828 4.134 2.76 2.492 4.173 5.897 3.352 9.302-.631 2.617-2.345 4.115-3.697 5.298l-.002.001v.001q-.167.145-.324.284c-.723.64-1.274 1.176-1.628 1.78-.328.563-.506 1.222-.374 2.154.035.244.102.376.157.452s.138.149.277.22c.287.146.642.217 1.178.324h.003l.325.066c.636.13 1.475.327 2.185.865.76.576 1.26 1.45 1.429 2.696.178 1.316-.428 2.479-1.294 3.404-.868.926-2.09 1.722-3.432 2.37-2.683 1.296-6.097 2.12-8.84 2.12-9.789 0-19.128-7.167-19.128-17.323M29 15.5c0 1.657-1.344 3-3 3-1.657 0-3-1.343-3-3s1.343-3 3-3c1.656 0 3 1.343 3 3m-10.5 6.957c1.656 0 3-1.343 3-3s-1.344-3-3-3c-1.657 0-3 1.343-3 3s1.343 3 3 3m3 6.767c0 1.657-1.344 3-3 3-1.657 0-3-1.343-3-3s1.343-3 3-3c1.656 0 3 1.343 3 3M26 36.5c1.656 0 3-1.343 3-3s-1.344-3-3-3c-1.657 0-3 1.343-3 3s1.343 3 3 3"
        })
    });
const $d71adc98566caa69$var$ForwardRef = /*#__PURE__*/ (0, $gsKpf$react.forwardRef)($d71adc98566caa69$var$SvgComponent);
var $d71adc98566caa69$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d71adc98566caa69$var$ForwardRef);
