// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noComments_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $846YI$reactjsxruntime = require("react/jsx-runtime");
var $846YI$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0d2ab9e899b46258$export$2e2bcd8739ae039);




const $0d2ab9e899b46258$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $846YI$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $846YI$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22.531 12.602c0-4.142 3.359-7.5 7.5-7.5 4.142 0 7.5 3.358 7.5 7.5 0 4.14-3.358 7.5-7.5 7.5s-7.5-3.36-7.5-7.5M33.39 15.96c.485-.485.485-1.268 0-1.753h.02l-1.605-1.605 1.605-1.606c.485-.485.485-1.268 0-1.753-.486-.486-1.268-.486-1.754 0l-1.605 1.605-1.605-1.605c-.485-.486-1.268-.486-1.754 0-.485.485-.485 1.268 0 1.753l1.605 1.606-1.605 1.605c-.485.485-.485 1.268 0 1.753.238.248.555.367.872.367s.634-.129.872-.367l1.605-1.605 1.605 1.605c.238.248.555.367.872.367s.634-.129.872-.367M4.75 13.262c0-2.434 2.008-4.364 4.43-4.364h10.883c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H9.18c-1.09 0-1.93.86-1.93 1.864v11.052c0 1.004.84 1.864 1.93 1.864h2.848c.688 0 1.247.556 1.25 1.244.006 1.348-.004 3.222-.014 4.756l6.719-5.703c.226-.192.512-.297.809-.297h8.528c1.09 0 1.93-.86 1.93-1.864V22.52c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v1.794c0 2.434-2.008 4.364-4.43 4.364h-8.07l-6.676 5.668c-1.463 1.241-3.828.263-3.813-1.754.009-1.186.017-2.644.019-3.914h-1.6c-2.422 0-4.43-1.93-4.43-4.364zm30.5 7.386c0-.69.56-1.25 1.25-1.25h2.68c2.232 0 4.07 1.79 4.07 4.033v9.876c0 2.243-1.838 4.032-4.07 4.032h-1.276c.001 1.103.009 2.346.016 3.364.014 1.905-2.22 2.863-3.618 1.667l-5.88-5.03H21.32c-1.701 0-3.166-1.036-3.773-2.517-.261-.639.044-1.369.683-1.63.639-.262 1.369.043 1.63.682.23.56.793.964 1.46.964h7.562c.298 0 .587.107.813.3l5.72 4.893c-.01-1.308-.016-2.83-.011-3.948.003-.688.562-1.245 1.25-1.245h2.525c.883 0 1.57-.701 1.57-1.532V23.43c0-.83-.687-1.533-1.57-1.533H36.5c-.69 0-1.25-.56-1.25-1.25"
        })
    });
const $0d2ab9e899b46258$var$ForwardRef = /*#__PURE__*/ (0, $846YI$react.forwardRef)($0d2ab9e899b46258$var$SvgComponent);
var $0d2ab9e899b46258$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0d2ab9e899b46258$var$ForwardRef);
