// @react-spectrum/s2/illustrations/linear/internal/S2_lin_ruler_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2ESDk$reactjsxruntime = require("react/jsx-runtime");
var $2ESDk$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3de31ead198b105f$export$2e2bcd8739ae039);




const $3de31ead198b105f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2ESDk$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2ESDk$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M17.697 70.462c-3.124-3.124-3.124-8.19 0-11.314l41.451-41.451c3.125-3.124 8.19-3.124 11.314 0l7.842 7.841c3.124 3.124 3.124 8.19 0 11.314L36.852 78.304c-3.124 3.124-8.19 3.124-11.314 0zm2.828-8.485c-1.562 1.562-1.562 4.095 0 5.657l7.842 7.841c1.562 1.562 4.094 1.562 5.656 0l41.452-41.452c1.562-1.562 1.562-4.094 0-5.656l-7.841-7.842c-1.562-1.562-4.095-1.562-5.657 0l-4.328 4.328 4.728 4.727c.781.781.781 2.048 0 2.829-.78.78-2.047.78-2.828 0L54.82 27.68l-7.161 7.16 7.443 7.444c.781.781.781 2.048 0 2.829-.78.78-2.047.78-2.828 0l-7.443-7.444-7.162 7.162 4.728 4.727c.781.781.781 2.048 0 2.829-.781.78-2.047.78-2.829 0l-4.727-4.728-7.16 7.16 7.442 7.444c.782.781.782 2.048 0 2.829-.78.78-2.047.78-2.828 0l-7.443-7.444z"
        })
    });
const $3de31ead198b105f$var$ForwardRef = /*#__PURE__*/ (0, $2ESDk$react.forwardRef)($3de31ead198b105f$var$SvgComponent);
var $3de31ead198b105f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3de31ead198b105f$var$ForwardRef);
