// @react-spectrum/s2/illustrations/linear/internal/S2_lin_tablet_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $afr6g$jsx} from "react/jsx-runtime";
import {forwardRef as $afr6g$forwardRef} from "react";





const $2fd822ec8b0f08ee$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $afr6g$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $afr6g$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20 44c0-6.627 5.373-12 12-12h96c6.627 0 12 5.373 12 12v72c0 6.627-5.373 12-12 12H32c-6.627 0-12-5.373-12-12zm12-20c-11.046 0-20 8.954-20 20v72c0 11.046 8.954 20 20 20h96c11.046 0 20-8.954 20-20V44c0-11.046-8.954-20-20-20zm-3.5 55.5c0 2.486 2.014 4.5 4.5 4.5s4.5-2.014 4.5-4.5S35.486 75 33 75h-.011c-2.486 0-4.489 2.014-4.489 4.5"
        })
    });
const $2fd822ec8b0f08ee$var$ForwardRef = /*#__PURE__*/ (0, $afr6g$forwardRef)($2fd822ec8b0f08ee$var$SvgComponent);
var $2fd822ec8b0f08ee$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2fd822ec8b0f08ee$var$ForwardRef);


export {$2fd822ec8b0f08ee$export$2e2bcd8739ae039 as default};
