// @react-spectrum/s2/illustrations/linear/internal/S2_lin_mention_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2yhO1$reactjsxruntime = require("react/jsx-runtime");
var $2yhO1$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e50ab0fee763da52$export$2e2bcd8739ae039);




const $e50ab0fee763da52$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2yhO1$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2yhO1$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M18.967 7.154c4.91-1.61 10.173-1.073 13.6.474 7.03 3.171 9.35 9.989 8.323 15.826-1.018 5.785-5.446 11.009-12.158 10.544-6.154-.425-9.478-2.761-10.826-5.84-1.294-2.956-.575-6.197.47-8.127 2.7-4.988 7.322-5.749 10.168-5.658 1.875.06 3.001 1.754 2.852 3.374l-1.265 13.759c4.393-.304 7.497-3.936 8.297-8.485.864-4.91-1.073-10.49-6.889-13.114-2.858-1.29-7.477-1.792-11.793-.377-4.242 1.39-8.199 4.631-9.91 10.91-1.352 4.965.228 9.722 3.274 13.25 3.065 3.55 7.507 5.729 11.657 5.609.69-.02 1.265.523 1.285 1.213s-.523 1.266-1.213 1.286c-5.029.145-10.155-2.458-13.621-6.474-3.487-4.038-5.403-9.632-3.793-15.542C9.37 12.638 13.983 8.79 18.967 7.154m8.665 24.228 1.275-13.864c.018-.2-.044-.373-.134-.486q-.124-.157-.308-.16c-2.372-.076-5.828.54-7.89 4.35-.794 1.466-1.278 3.88-.379 5.934.78 1.779 2.762 3.64 7.436 4.226"
        })
    });
const $e50ab0fee763da52$var$ForwardRef = /*#__PURE__*/ (0, $2yhO1$react.forwardRef)($e50ab0fee763da52$var$SvgComponent);
var $e50ab0fee763da52$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e50ab0fee763da52$var$ForwardRef);
