// @react-spectrum/s2/illustrations/linear/internal/S2_lin_phone_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $gEn4F$reactjsxruntime = require("react/jsx-runtime");
var $gEn4F$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e1a6845ded011df0$export$2e2bcd8739ae039);




const $e1a6845ded011df0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gEn4F$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gEn4F$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M33.5 24c0-2.21 1.79-4 4-4h22c2.21 0 4 1.79 4 4v48c0 2.21-1.79 4-4 4h-22c-2.21 0-4-1.79-4-4zm4-8c-4.418 0-8 3.582-8 8v48c0 4.418 3.582 8 8 8h22c4.418 0 8-3.582 8-8V24c0-4.418-3.582-8-8-8zM51 70.5c0 1.38-1.12 2.5-2.5 2.5S46 71.88 46 70.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5"
        })
    });
const $e1a6845ded011df0$var$ForwardRef = /*#__PURE__*/ (0, $gEn4F$react.forwardRef)($e1a6845ded011df0$var$SvgComponent);
var $e1a6845ded011df0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e1a6845ded011df0$var$ForwardRef);
