// @react-spectrum/s2/illustrations/linear/internal/S2_lin_explosion_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $igvcL$reactjsxruntime = require("react/jsx-runtime");
var $igvcL$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $28462186b958700b$export$2e2bcd8739ae039);




const $28462186b958700b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $igvcL$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $igvcL$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M9.4 7.442c-.917-1.699 1.188-3.414 2.666-2.172l11.122 9.345 6.493-7.168c1.208-1.334 3.4-.196 3.004 1.56l-1.87 8.3h14.43c2.019 0 2.432 2.854.496 3.427L33.186 24.45l3.822 4.5c1.156 1.36-.228 3.374-1.913 2.784l-5.815-2.039v12.253c0 1.943-2.693 2.439-3.386.623l-4.573-12c-2.985 1.716-7.09 4.043-9.487 5.4-1.656.936-3.354-1.042-2.285-2.542 1.408-1.976 3.654-5.142 5.302-7.526l-8.675-2.226c-1.769-.454-1.747-2.974.03-3.398l8.976-2.138zm4.071 2.274 5.458 10.102-9.195 2.19 9.186 2.358-.994 1.47c-1.103 1.634-2.847 4.117-4.39 6.297 2.618-1.486 5.722-3.253 7.804-4.46l1.272-.736 4.168 10.937V26.17l6.532 2.29-4.517-5.317 11.273-3.337h-12.38l1.931-8.567-6.195 6.84z"
        })
    });
const $28462186b958700b$var$ForwardRef = /*#__PURE__*/ (0, $igvcL$react.forwardRef)($28462186b958700b$var$SvgComponent);
var $28462186b958700b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($28462186b958700b$var$ForwardRef);
