// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowRight_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $eEJ5v$reactjsxruntime = require("react/jsx-runtime");
var $eEJ5v$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7c6d9251a7ae07b3$export$2e2bcd8739ae039);




const $7c6d9251a7ae07b3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eEJ5v$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $eEJ5v$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M6.5 22.75c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h30.982l-4.866 4.866c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0l7-7c.488-.488.488-1.28 0-1.768l-7-7c-.488-.488-1.28-.488-1.768 0s-.488 1.28 0 1.768l4.866 4.866z"
        })
    });
const $7c6d9251a7ae07b3$var$ForwardRef = /*#__PURE__*/ (0, $eEJ5v$react.forwardRef)($7c6d9251a7ae07b3$var$SvgComponent);
var $7c6d9251a7ae07b3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7c6d9251a7ae07b3$var$ForwardRef);
