// @react-spectrum/s2/illustrations/linear/internal/S2_lin_tablet_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $hGWCM$jsx} from "react/jsx-runtime";
import {forwardRef as $hGWCM$forwardRef} from "react";





const $6574858b17f61f2f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hGWCM$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hGWCM$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M13 26c0-3.314 2.686-6 6-6h58c3.314 0 6 2.686 6 6v44c0 3.314-2.686 6-6 6H19c-3.314 0-6-2.686-6-6zm6-10c-5.523 0-10 4.477-10 10v44c0 5.523 4.477 10 10 10h58c5.523 0 10-4.477 10-10V26c0-5.523-4.477-10-10-10zm4 31.5c0 1.38-1.12 2.5-2.5 2.5S18 48.88 18 47.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5"
        })
    });
const $6574858b17f61f2f$var$ForwardRef = /*#__PURE__*/ (0, $hGWCM$forwardRef)($6574858b17f61f2f$var$SvgComponent);
var $6574858b17f61f2f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6574858b17f61f2f$var$ForwardRef);


export {$6574858b17f61f2f$export$2e2bcd8739ae039 as default};
