// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noBrands_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1ntmh$reactjsxruntime = require("react/jsx-runtime");
var $1ntmh$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $dc73d668a7168ff0$export$2e2bcd8739ae039);




const $dc73d668a7168ff0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1ntmh$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1ntmh$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M23 24.968c0-2.767 2.251-5.019 5.019-5.019h40.067c2.767 0 5.019 2.252 5.019 5.02v28.029c0 1.104.895 2 2 2s2-.896 2-2v-28.03c0-4.977-4.043-9.019-9.02-9.019H28.02c-4.977 0-9.019 4.042-9.019 9.02v29.029C19 70.035 32.021 83.05 48.052 83.05q2.241-.002 4.394-.331c1.092-.166 1.843-1.186 1.677-2.278s-1.186-1.843-2.278-1.677c-1.236.188-2.502.286-3.793.286C34.23 79.05 23 67.826 23 53.998zm34.528 12.167c-1.671-2.896-5.848-2.895-7.518 0l1.732 1-1.732-1-7.347 12.734c-1.668 2.892.412 6.509 3.756 6.509h14.705c3.345 0 5.424-3.617 3.756-6.509zl-1.732 1zm-4.053 1.999c.13-.227.457-.227.588 0l7.352 12.734c.134.232-.033.51-.29.51H46.418c-.258 0-.425-.279-.291-.51zm-12.723-.632c.957.553 1.285 1.776.732 2.733l-8.171 14.149c-.552.956-1.776 1.284-2.732.731-.957-.552-1.284-1.775-.732-2.732l8.171-14.149c.553-.956 1.776-1.284 2.732-.732M70.75 87.5c8.146 0 14.75-6.603 14.75-14.75S78.896 58 70.75 58 56 64.604 56 72.75 62.604 87.5 70.75 87.5M73 66c0-1.104-.895-2-2-2s-2 .896-2 2v5h-5c-1.105 0-2 .896-2 2s.895 2 2 2h5v5c0 1.105.895 2 2 2s2-.895 2-2v-5h5c1.105 0 2-.895 2-2s-.895-2-2-2h-5z"
        })
    });
const $dc73d668a7168ff0$var$ForwardRef = /*#__PURE__*/ (0, $1ntmh$react.forwardRef)($dc73d668a7168ff0$var$SvgComponent);
var $dc73d668a7168ff0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($dc73d668a7168ff0$var$ForwardRef);
