// @react-spectrum/s2/illustrations/linear/internal/S2_lin_download_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bQniA$reactjsxruntime = require("react/jsx-runtime");
var $bQniA$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $bcf654ed7b75f147$export$2e2bcd8739ae039);




const $bcf654ed7b75f147$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bQniA$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bQniA$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28.335 80.742c2.209 0 4 1.79 4 4v29.411c0 4.59 3.682 8.255 8.157 8.255h79.019c4.475 0 8.157-3.665 8.157-8.255V84.741c0-2.209 1.791-4 4-4s4 1.791 4 4v29.412c0 8.947-7.203 16.255-16.157 16.255h-79.02c-8.953 0-16.156-7.308-16.156-16.255V84.741c0-2.209 1.79-4 4-4m51.667-51.15c2.209 0 4 1.79 4 4v64.32l16.671-16.67c1.562-1.563 4.095-1.563 5.657 0 1.562 1.561 1.562 4.094 0 5.656l-23.5 23.5c-1.562 1.562-4.095 1.562-5.657 0l-23.5-23.5c-1.562-1.562-1.562-4.095 0-5.657s4.095-1.562 5.657 0l16.672 16.672V33.59c0-2.209 1.79-4 4-4"
        })
    });
const $bcf654ed7b75f147$var$ForwardRef = /*#__PURE__*/ (0, $bQniA$react.forwardRef)($bcf654ed7b75f147$var$SvgComponent);
var $bcf654ed7b75f147$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($bcf654ed7b75f147$var$ForwardRef);
