// @react-spectrum/s2/illustrations/linear/internal/S2_lin_image_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $4jLbi$reactjsxruntime = require("react/jsx-runtime");
var $4jLbi$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $eae6ae229ea6155f$export$2e2bcd8739ae039);




const $eae6ae229ea6155f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4jLbi$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4jLbi$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M82.2 28c0-3.314-2.687-6-6-6H19c-3.315 0-6 2.686-6 6v39.2c0 .678.112 1.33.32 1.939l16.042-24.785v-.001c2.462-3.806 8.038-3.806 10.5 0l12.702 19.62 7.322-11.07.009-.013c2.477-3.686 7.906-3.693 10.382 0l.002.003 11.364 16.834c.357-.768.556-1.624.556-2.527zM32.72 46.526 15.983 72.387c.886.517 1.917.813 3.017.813h27.463l3.694-5.585-13.653-21.088v-.001c-.886-1.37-2.896-1.37-3.782 0m46.051 26.096c-.78.37-1.651.578-2.572.578H51.257l11.96-18.084c.9-1.328 2.853-1.32 3.738.002l.004.007zM76.2 18c5.523 0 10 4.477 10 10v39.2c0 5.523-4.477 10-10 10H19c-5.523 0-10-4.477-10-10V28c0-5.523 4.477-10 10-10zM65.5 33c0 2.761 2.239 5 5 5s5-2.239 5-5-2.239-5-5-5-5 2.239-5 5"
        })
    });
const $eae6ae229ea6155f$var$ForwardRef = /*#__PURE__*/ (0, $4jLbi$react.forwardRef)($eae6ae229ea6155f$var$SvgComponent);
var $eae6ae229ea6155f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($eae6ae229ea6155f$var$ForwardRef);
