// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cloudUpload_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7HTKp$jsx} from "react/jsx-runtime";
import {forwardRef as $7HTKp$forwardRef} from "react";





const $b211eda84dd76f71$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7HTKp$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7HTKp$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M35.215 25.039C38.21 17.922 45.043 12.89 53.07 12.89c10.386 0 18.754 8.399 19.444 18.869 8.236 1.369 14.446 8.557 14.446 17.72 0 9.843-7.655 17.96-17.28 17.96h-.78c-1.105 0-2-.895-2-2s.895-2 2-2h.779c7.255 0 13.28-6.161 13.28-13.96 0-7.805-5.588-13.527-12.516-13.933-1.057-.062-1.883-.937-1.883-1.996v-.39c0-9.069-7.016-16.27-15.49-16.27-6.839 0-12.714 4.675-14.738 11.267-.167.544-.558.99-1.075 1.229-.517.238-1.11.245-1.632.019-1.257-.545-2.627-.845-4.075-.845-5.864 0-10.76 4.99-10.76 11.33 0 .884.101 1.735.29 2.567.235 1.036-.38 2.074-1.401 2.366-3.776 1.077-6.63 4.713-6.63 9.137 0 5.249 4.012 9.371 8.82 9.47h5.04c1.105 0 2 .895 2 2s-.895 2-2 2h-5.096c-7.144-.132-12.763-6.186-12.763-13.47 0-5.508 3.191-10.287 7.834-12.365q-.094-.837-.094-1.705c0-8.381 6.523-15.33 14.76-15.33 1.267 0 2.493.166 3.664.478M50 43.829l10.586 10.586c.78.78 2.047.78 2.828 0 .781-.781.781-2.048 0-2.829l-14-14c-.78-.78-2.047-.78-2.828 0l-14 14c-.781.781-.781 2.048 0 2.829.78.78 2.047.78 2.828 0L46 43.829v36.982c0 1.104.895 2 2 2s2-.896 2-2z"
        })
    });
const $b211eda84dd76f71$var$ForwardRef = /*#__PURE__*/ (0, $7HTKp$forwardRef)($b211eda84dd76f71$var$SvgComponent);
var $b211eda84dd76f71$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b211eda84dd76f71$var$ForwardRef);


export {$b211eda84dd76f71$export$2e2bcd8739ae039 as default};
