// @react-spectrum/s2/illustrations/linear/internal/S2_lin_effects_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6Vh0R$jsx} from "react/jsx-runtime";
import {forwardRef as $6Vh0R$forwardRef} from "react";





const $b7cf50662e96addd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6Vh0R$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6Vh0R$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22.216 9.431c-1.087.827-1.858 2.14-2.304 3.365l-1.565 5.098H26.6c.455 0 .874.247 1.094.645l3.227 5.83 6.125-6.119c.489-.488 1.28-.487 1.768.001.488.489.487 1.28-.001 1.768l-6.633 6.625 3.675 6.639c.334.604.116 1.364-.488 1.699s-1.365.115-1.7-.489l-3.326-6.01-.751.75c-1.53 1.53-2.907 2.906-3.903 3.904l-1.181 1.184-.323.325q-.056.055-.08.082l-.007.007c-.43.467-1.152.542-1.67.153-.553-.414-.665-1.197-.25-1.75.032-.042.06-.074.064-.079l.013-.014.008-.01.02-.021.033-.033.092-.093q.116-.118.328-.33l1.184-1.187c.997-1 2.374-2.376 3.905-3.906l1.258-1.258-3.218-5.813H17.58L11.268 40.96c-.203.66-.902 1.031-1.562.829-.66-.203-1.03-.902-.828-1.562l6.087-19.833h-3.723c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h4.49l1.799-5.861q.009-.03.02-.058c.538-1.494 1.538-3.305 3.15-4.533 1.68-1.28 3.932-1.847 6.685-.926.655.22 1.008.928.79 1.582-.22.655-.928 1.008-1.583.79-1.963-.657-3.351-.239-4.377.543"
        })
    });
const $b7cf50662e96addd$var$ForwardRef = /*#__PURE__*/ (0, $6Vh0R$forwardRef)($b7cf50662e96addd$var$SvgComponent);
var $b7cf50662e96addd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b7cf50662e96addd$var$ForwardRef);


export {$b7cf50662e96addd$export$2e2bcd8739ae039 as default};
