// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noBrands_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $fmL0B$reactjsxruntime = require("react/jsx-runtime");
var $fmL0B$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $346f065ea8e78c9c$export$2e2bcd8739ae039);




const $346f065ea8e78c9c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fmL0B$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fmL0B$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M39 41.614c0-4.244 3.454-7.698 7.698-7.698h66.778c4.244 0 7.698 3.454 7.698 7.698V88.33c0 2.209 1.791 4 4 4s4-1.791 4-4V41.614c0-8.662-7.035-15.698-15.698-15.698H46.698C38.036 25.916 31 32.952 31 41.614v48.383c0 27.097 22 49.087 49.087 49.087 2.555 0 5.068-.196 7.522-.575 2.184-.336 3.68-2.379 3.344-4.562s-2.379-3.681-4.562-3.344c-2.053.316-4.158.481-6.304.481-22.67 0-41.087-18.409-41.087-41.088zM96.457 61.56c-3.041-5.27-10.644-5.27-13.684 0l3.463 1.999-3.464-1.998-12.244 21.222c-3.035 5.263.749 11.848 6.838 11.848h24.508c6.089 0 9.873-6.585 6.838-11.847l-.001-.002-12.253-21.22-3.465 1.998zM77.544 86.63l12.071-20.922 12.081 20.922zm-9.29-23.036c1.913 1.105 2.568 3.551 1.463 5.464L56.1 92.64c-1.105 1.913-3.552 2.569-5.465 1.464s-2.568-3.552-1.463-5.465L62.79 65.058c1.105-1.913 3.55-2.569 5.464-1.464m49.663 82.24c13.577 0 24.584-11.007 24.584-24.584s-11.007-24.583-24.584-24.583-24.583 11.006-24.583 24.583 11.006 24.584 24.583 24.584M122.334 110c0-2.209-1.791-4-4-4s-4 1.791-4 4v7.667h-7.667c-2.209 0-4 1.791-4 4s1.791 4 4 4h7.667v7.667c0 2.209 1.791 4 4 4s4-1.791 4-4v-7.667H130c2.209 0 4-1.791 4-4s-1.791-4-4-4h-7.666z"
        })
    });
const $346f065ea8e78c9c$var$ForwardRef = /*#__PURE__*/ (0, $fmL0B$react.forwardRef)($346f065ea8e78c9c$var$SvgComponent);
var $346f065ea8e78c9c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($346f065ea8e78c9c$var$ForwardRef);
