// @react-spectrum/s2/illustrations/linear/internal/S2_lin_flag_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $3ixUn$jsx} from "react/jsx-runtime";
import {forwardRef as $3ixUn$forwardRef} from "react";





const $fb4676a90dbd3484$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3ixUn$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3ixUn$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M12.992 9.11c0-.691-.56-1.25-1.25-1.25s-1.25.559-1.25 1.25v29.78c0 .691.56 1.25 1.25 1.25s1.25-.559 1.25-1.25v-7.64h20.336c2.347 0 4.25-1.902 4.25-4.25V14.022c0-2.348-1.903-4.25-4.25-4.25H12.992zm0 3.162V28.75h20.336c.967 0 1.75-.783 1.75-1.75V14.022c0-.967-.783-1.75-1.75-1.75z"
        })
    });
const $fb4676a90dbd3484$var$ForwardRef = /*#__PURE__*/ (0, $3ixUn$forwardRef)($fb4676a90dbd3484$var$SvgComponent);
var $fb4676a90dbd3484$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($fb4676a90dbd3484$var$ForwardRef);


export {$fb4676a90dbd3484$export$2e2bcd8739ae039 as default};
