// @react-spectrum/s2/illustrations/linear/internal/S2_lin_flagCross_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $3VZEq$jsx} from "react/jsx-runtime";
import {forwardRef as $3VZEq$forwardRef} from "react";





const $26e6417be2532763$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3VZEq$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3VZEq$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M12.957 9.11c0-.691-.56-1.25-1.25-1.25s-1.25.559-1.25 1.25v29.78c0 .691.56 1.25 1.25 1.25s1.25-.559 1.25-1.25v-7.64h20.336c2.347 0 4.25-1.902 4.25-4.25V14.022c0-2.348-1.903-4.25-4.25-4.25H12.957zm0 3.162V28.75h20.336c.967 0 1.75-.783 1.75-1.75V14.022c0-.967-.783-1.75-1.75-1.75zm7.06 4.127c.488-.488 1.28-.488 1.768 0l2.344 2.344 2.344-2.344c.488-.488 1.28-.488 1.768 0s.488 1.28 0 1.768l-2.344 2.344 2.344 2.344c.488.488.488 1.28 0 1.768-.489.488-1.28.488-1.768 0l-2.344-2.344-2.344 2.344c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.768l2.344-2.344-2.344-2.344c-.488-.488-.488-1.28 0-1.768"
        })
    });
const $26e6417be2532763$var$ForwardRef = /*#__PURE__*/ (0, $3VZEq$forwardRef)($26e6417be2532763$var$SvgComponent);
var $26e6417be2532763$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($26e6417be2532763$var$ForwardRef);


export {$26e6417be2532763$export$2e2bcd8739ae039 as default};
