// @react-spectrum/s2/illustrations/linear/internal/S2_lin_crop_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6ku22$jsx} from "react/jsx-runtime";
import {forwardRef as $6ku22$forwardRef} from "react";





const $6254e954814bde00$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6ku22$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6ku22$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M31.974 15c0-1.105-.895-2-2-2s-2 .895-2 2v12.903H15c-1.105 0-2 .896-2 2s.895 2 2 2h12.974v26.193c0 5.523 4.478 10 10 10h26.05v12.903c0 1.105.896 2 2 2s2-.895 2-2V68.096H81c1.105 0 2-.895 2-2s-.895-2-2-2H37.974c-3.313 0-6-2.686-6-6V15m5.442 12.903c-1.105 0-2 .896-2 2s.895 2 2 2h20.609c3.314 0 6 2.687 6 6v20.742c0 1.104.895 2 2 2s2-.896 2-2V37.903c0-5.523-4.477-10-10-10z"
        })
    });
const $6254e954814bde00$var$ForwardRef = /*#__PURE__*/ (0, $6ku22$forwardRef)($6254e954814bde00$var$SvgComponent);
var $6254e954814bde00$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6254e954814bde00$var$ForwardRef);


export {$6254e954814bde00$export$2e2bcd8739ae039 as default};
