// @react-spectrum/s2/illustrations/linear/internal/S2_lin_warning_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $jhMY5$reactjsxruntime = require("react/jsx-runtime");
var $jhMY5$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2a7bb8c4d81f1672$export$2e2bcd8739ae039);




const $2a7bb8c4d81f1672$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jhMY5$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jhMY5$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m28.543 8.015 13.91 24.1c2.02 3.502-.499 7.875-4.543 7.875H10.09c-4.045 0-6.562-4.373-4.543-7.875L19.381 8.147q.03-.068.066-.132c2.021-3.503 7.074-3.504 9.096 0m-6.865 1.141c1.1-1.727 3.661-1.69 4.7.108l13.91 24.1c1.06 1.84-.263 4.126-2.378 4.126H10.09c-2.115 0-3.438-2.286-2.378-4.125l13.91-24.1q.032-.054.056-.109M24 13.99c.69 0 1.25.56 1.25 1.25v11c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-11c0-.69.56-1.25 1.25-1.25m1.75 17.75c0 .966-.783 1.75-1.75 1.75-.966 0-1.75-.784-1.75-1.75s.784-1.75 1.75-1.75c.967 0 1.75.784 1.75 1.75"
        })
    });
const $2a7bb8c4d81f1672$var$ForwardRef = /*#__PURE__*/ (0, $jhMY5$react.forwardRef)($2a7bb8c4d81f1672$var$SvgComponent);
var $2a7bb8c4d81f1672$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2a7bb8c4d81f1672$var$ForwardRef);
