// @react-spectrum/s2/illustrations/linear/internal/S2_lin_logo_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $hIgaM$reactjsxruntime = require("react/jsx-runtime");
var $hIgaM$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ba3322a95dc69efb$export$2e2bcd8739ae039);




const $ba3322a95dc69efb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hIgaM$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hIgaM$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M51.573 13.75c-1.96-1.015-4.292-1.015-6.253 0L17.658 28.073c-2.257 1.168-3.674 3.497-3.674 6.038V61.89c0 2.54 1.417 4.87 3.674 6.038L45.32 82.25c1.96 1.015 4.293 1.015 6.253 0l27.663-14.323c2.256-1.168 3.673-3.497 3.673-6.039V34.112c0-2.54-1.417-4.87-3.673-6.038zm-4.414 3.552c.808-.418 1.768-.418 2.575 0l27.663 14.323c.929.481 1.512 1.44 1.512 2.486V61.89c0 1.046-.583 2.005-1.512 2.486L49.734 78.698c-.807.418-1.767.418-2.575 0L19.497 64.375c-.93-.481-1.513-1.44-1.513-2.486V34.11c0-1.046.584-2.005 1.513-2.486zM51.536 38.5c1.54-2.667 5.389-2.667 6.928 0L66.258 52c1.54 2.667-.385 6-3.464 6H47.206c-3.08 0-5.004-3.333-3.464-6zm3.464 2L47.206 54h15.588zm-11.395-1.437c.957.552 1.285 1.775.732 2.732L35.621 56.89c-.553.956-1.776 1.284-2.733.731-.956-.552-1.284-1.775-.731-2.732l8.716-15.095c.553-.956 1.776-1.284 2.732-.731"
        })
    });
const $ba3322a95dc69efb$var$ForwardRef = /*#__PURE__*/ (0, $hIgaM$react.forwardRef)($ba3322a95dc69efb$var$SvgComponent);
var $ba3322a95dc69efb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ba3322a95dc69efb$var$ForwardRef);
