// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileText_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $gzE5Z$reactjsxruntime = require("react/jsx-runtime");
var $gzE5Z$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9e8a9183d619e647$export$2e2bcd8739ae039);




const $9e8a9183d619e647$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gzE5Z$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gzE5Z$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M18.498 21.547c-.321 0-.614.121-.835.32q-.048.042-.09.09c-.202.222-.325.517-.325.84v2.4c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-1.15h3.5v9.499h-1.15c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h4.799c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-1.15v-9.499h3.5v1.15c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-2.4c0-.69-.56-1.25-1.25-1.25z"
            }),
            /*#__PURE__*/ (0, $gzE5Z$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M15.046 7.75c-2.62 0-4.796 2.081-4.796 4.711v23.077c0 2.631 2.176 4.712 4.796 4.712h18.909c2.62 0 4.795-2.081 4.795-4.712V21.972c0-1.26-.513-2.46-1.415-3.342l-9.742-9.51c-.901-.88-2.117-1.37-3.38-1.37zm-2.296 4.711c0-1.194 1-2.211 2.296-2.211h9.167c.618 0 1.206.24 1.634.658l9.741 9.511c.428.417.662.976.662 1.553v13.566c0 1.195-1 2.212-2.295 2.212h-18.91c-1.295 0-2.295-1.017-2.295-2.212zm0 0h-.25z"
            })
        ]
    });
const $9e8a9183d619e647$var$ForwardRef = /*#__PURE__*/ (0, $gzE5Z$react.forwardRef)($9e8a9183d619e647$var$SvgComponent);
var $9e8a9183d619e647$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9e8a9183d619e647$var$ForwardRef);
