// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowRight_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $61lqD$jsx} from "react/jsx-runtime";
import {forwardRef as $61lqD$forwardRef} from "react";





const $1a565bfe88b13592$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $61lqD$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $61lqD$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 76c-2.21 0-4 1.79-4 4s1.79 4 4 4h104.843l-16.671 16.672c-1.563 1.562-1.563 4.094 0 5.656 1.562 1.563 4.094 1.563 5.656 0l23.5-23.5c1.563-1.562 1.563-4.094 0-5.656l-23.5-23.5c-1.562-1.562-4.094-1.562-5.656 0-1.563 1.562-1.563 4.094 0 5.656L128.843 76z"
        })
    });
const $1a565bfe88b13592$var$ForwardRef = /*#__PURE__*/ (0, $61lqD$forwardRef)($1a565bfe88b13592$var$SvgComponent);
var $1a565bfe88b13592$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1a565bfe88b13592$var$ForwardRef);


export {$1a565bfe88b13592$export$2e2bcd8739ae039 as default};
