// @react-spectrum/s2/illustrations/linear/internal/S2_lin_rotateCW_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2QA1R$reactjsxruntime = require("react/jsx-runtime");
var $2QA1R$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3b67dad3c2269345$export$2e2bcd8739ae039);




const $3b67dad3c2269345$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2QA1R$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2QA1R$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M7.754 24.299c0-8.422 6.827-15.25 15.25-15.25 6.025 0 11.238 3.495 13.714 8.572l-4.671-1.47c-.659-.208-1.36.158-1.568.816-.207.659.158 1.36.817 1.568l7.442 2.343c.658.208 1.36-.158 1.568-.817l2.343-7.442c.207-.658-.158-1.36-.817-1.567-.658-.208-1.36.158-1.568.817l-1.401 4.45c-2.92-5.794-8.924-9.77-15.86-9.77-9.802 0-17.75 7.947-17.75 17.75s7.948 17.75 17.75 17.75c7.668 0 14.197-4.86 16.68-11.664.236-.649-.098-1.367-.746-1.603-.649-.237-1.366.097-1.603.745-2.135 5.85-7.747 10.022-14.33 10.022-8.423 0-15.25-6.828-15.25-15.25"
        })
    });
const $3b67dad3c2269345$var$ForwardRef = /*#__PURE__*/ (0, $2QA1R$react.forwardRef)($3b67dad3c2269345$var$SvgComponent);
var $3b67dad3c2269345$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3b67dad3c2269345$var$ForwardRef);
