// @react-spectrum/s2/illustrations/linear/internal/S2_lin_ribbon_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6ksQP$reactjsxruntime = require("react/jsx-runtime");
var $6ksQP$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $688db776a22c2409$export$2e2bcd8739ae039);




const $688db776a22c2409$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6ksQP$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6ksQP$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 28c-17.754 0-32.147 14.392-32.147 32.147S62.246 92.293 80 92.293s32.147-14.392 32.147-32.146S97.754 28 80 28M39.853 60.147C39.853 37.974 57.828 20 80 20s40.147 17.974 40.147 40.147c0 10.971-4.402 20.915-11.535 28.162l12.419 30.738c1.788 4.427-2.061 9.069-6.743 8.13l-9.568-1.917-5.55 8.026c-2.716 3.927-8.71 3.262-10.498-1.165l-8.671-21.463-8.662 21.438c-1.795 4.443-7.818 5.092-10.52 1.134l-5.501-8.063-9.559 1.978c-4.692.971-8.574-3.68-6.779-8.123L51.39 88.31c-7.135-7.247-11.537-17.191-11.537-28.163m18.01 33.496-10.065 24.911 7.205-1.491c2.372-.491 4.806.493 6.171 2.494l4.147 6.077 10.333-25.573c-6.527-.703-12.584-2.97-17.791-6.418m26.484 6.417c6.527-.702 12.584-2.969 17.791-6.417l10.086 24.961-7.221-1.447c-2.349-.471-4.752.5-6.114 2.47l-4.189 6.057z"
        })
    });
const $688db776a22c2409$var$ForwardRef = /*#__PURE__*/ (0, $6ksQP$react.forwardRef)($688db776a22c2409$var$SvgComponent);
var $688db776a22c2409$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($688db776a22c2409$var$ForwardRef);
