// @react-spectrum/s2/illustrations/linear/internal/S2_lin_libraries_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $iUVBc$reactjsxruntime = require("react/jsx-runtime");
var $iUVBc$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $681dc36e5b5f8c2e$export$2e2bcd8739ae039);




const $681dc36e5b5f8c2e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iUVBc$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $iUVBc$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M31.43 47.43c0-8.841 7.159-16 16-16h53.82c2.209 0 4-1.791 4-4s-1.791-4-4-4H47.43c-13.26 0-24 10.74-24 24v53.82c0 2.209 1.79 4 4 4s4-1.791 4-4zm27-1c-6.628 0-12 5.372-12 12v58.15c0 6.627 5.372 12 12 12h58.15c6.627 0 12-5.373 12-12V58.43c0-6.092-4.54-11.124-10.42-11.897V83.96c0 1.59-1.78 2.55-3.11 1.66l-7.9-5.26c-.67-.45-1.55-.45-2.22 0l-7.9 5.26c-1.33.89-3.11-.06-3.11-1.66V46.43zm-20 12c0-11.046 8.954-20 20-20h58.15c11.045 0 20 8.954 20 20v58.15c0 11.045-8.955 20-20 20H58.43c-11.046 0-20-8.955-20-20z"
        })
    });
const $681dc36e5b5f8c2e$var$ForwardRef = /*#__PURE__*/ (0, $iUVBc$react.forwardRef)($681dc36e5b5f8c2e$var$SvgComponent);
var $681dc36e5b5f8c2e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($681dc36e5b5f8c2e$var$ForwardRef);
