// @react-spectrum/s2/illustrations/linear/internal/S2_lin_market_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $cfbBL$jsx} from "react/jsx-runtime";
import {forwardRef as $cfbBL$forwardRef} from "react";





const $a403085d5f7b9b22$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cfbBL$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cfbBL$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M13.1 7.846c-1.505 0-2.875.87-3.51 2.233l-2.545 5.472c-.525 1.128-.795 2.355-.795 3.595 0 2.306 1.458 4.268 3.5 5.028V35c0 2.9 2.35 5.25 5.25 5.25h18c2.9 0 5.25-2.35 5.25-5.25V24.193c2.07-.746 3.553-2.721 3.553-5.047 0-1.207-.257-2.4-.751-3.5l-.04-.095-2.545-5.471c-.636-1.365-2.006-2.234-3.51-2.234zM35.75 24.47c-1.385-.172-2.608-.869-3.46-1.886-.988 1.178-2.474 1.927-4.132 1.927s-3.144-.748-4.132-1.927c-.655.781-1.528 1.374-2.526 1.684V32h14.25zM19 24.437c-1.293-.216-2.43-.891-3.237-1.853-.863 1.03-2.107 1.732-3.513 1.892V35c0 1.519 1.231 2.75 2.75 2.75h4.022q-.021-.115-.022-.236zm-7.368-2.426c1.528 0 2.771-1.18 2.874-2.665l2.31-.874c.125.194.197.426.197.674 0 1.578 1.286 2.865 2.882 2.865s2.881-1.287 2.881-2.865v-.204c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v.204c0 1.578 1.286 2.865 2.882 2.865s2.881-1.287 2.881-2.865v-.204c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v.204c0 1.578 1.286 2.865 2.882 2.865s2.882-1.287 2.882-2.865c0-.877-.192-1.744-.562-2.54q-.02-.044-.036-.087L36.2 11.135l-19.384 7.337L36.2 11.134c-.223-.48-.707-.788-1.242-.788H13.099c-.536 0-1.02.31-1.243.789l-2.544 5.47v.002c-.371.794-.562 1.661-.562 2.539 0 1.578 1.286 2.865 2.882 2.865m5.184-3.54-2.31.875.008-.24c.021-.682.586-1.22 1.269-1.21.434.007.814.235 1.033.576M21.5 34.5h14.25v.5c0 1.519-1.231 2.75-2.75 2.75H21.478q.021-.115.022-.236z"
        })
    });
const $a403085d5f7b9b22$var$ForwardRef = /*#__PURE__*/ (0, $cfbBL$forwardRef)($a403085d5f7b9b22$var$SvgComponent);
var $a403085d5f7b9b22$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a403085d5f7b9b22$var$ForwardRef);


export {$a403085d5f7b9b22$export$2e2bcd8739ae039 as default};
