// @react-spectrum/s2/illustrations/linear/internal/S2_lin_flagCross_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $4qeNh$jsx} from "react/jsx-runtime";
import {forwardRef as $4qeNh$forwardRef} from "react";





const $8ad39ca1c4a2bdee$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4qeNh$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4qeNh$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M41.277 28.25c0-2.21-1.79-4-4-4s-4 1.79-4 4v103.5c0 2.209 1.791 4 4 4s4-1.791 4-4v-26.897h69.445c8.837 0 16-7.163 16-16V46.895c0-8.837-7.163-16-16-16H41.277zm0 10.645v57.958h69.445c4.418 0 8-3.582 8-8V46.895c0-4.419-3.582-8-8-8zM66.401 53.83c1.562-1.562 4.095-1.562 5.657 0l8.39 8.39 8.389-8.39c1.562-1.562 4.095-1.562 5.657 0s1.562 4.095 0 5.657l-8.39 8.39 8.39 8.389c1.562 1.562 1.562 4.095 0 5.657s-4.095 1.562-5.657 0l-8.39-8.39-8.39 8.39c-1.561 1.562-4.094 1.562-5.656 0s-1.562-4.095 0-5.657l8.39-8.39-8.39-8.39c-1.562-1.561-1.562-4.094 0-5.656"
        })
    });
const $8ad39ca1c4a2bdee$var$ForwardRef = /*#__PURE__*/ (0, $4qeNh$forwardRef)($8ad39ca1c4a2bdee$var$SvgComponent);
var $8ad39ca1c4a2bdee$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8ad39ca1c4a2bdee$var$ForwardRef);


export {$8ad39ca1c4a2bdee$export$2e2bcd8739ae039 as default};
