// @react-spectrum/s2/illustrations/linear/internal/S2_lin_crop_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $knu3s$jsx} from "react/jsx-runtime";
import {forwardRef as $knu3s$forwardRef} from "react";





const $07fb6aef5ee0af3a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $knu3s$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $knu3s$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.237 7.5c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v6.202H7.5c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h6.237v12.846c0 2.9 2.35 5.25 5.25 5.25h12.775V40.5c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-6.202H40.5c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H18.987c-1.519 0-2.75-1.231-2.75-2.75V7.5M19 13.702c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h10.012c1.52 0 2.75 1.23 2.75 2.75v10.063c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V18.952c0-2.9-2.35-5.25-5.25-5.25z"
        })
    });
const $07fb6aef5ee0af3a$var$ForwardRef = /*#__PURE__*/ (0, $knu3s$forwardRef)($07fb6aef5ee0af3a$var$SvgComponent);
var $07fb6aef5ee0af3a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($07fb6aef5ee0af3a$var$ForwardRef);


export {$07fb6aef5ee0af3a$export$2e2bcd8739ae039 as default};
