// @react-spectrum/s2/illustrations/linear/internal/S2_lin_market_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $h1gTX$reactjsxruntime = require("react/jsx-runtime");
var $h1gTX$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a163d08dac0a3fe4$export$2e2bcd8739ae039);




const $a163d08dac0a3fe4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $h1gTX$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $h1gTX$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M42.167 24.232c-5.08 0-9.707 2.937-11.853 7.544v.002l-8.81 18.942-.001.001v.002c-1.798 3.858-2.722 8.06-2.722 12.305 0 7.924 5.06 14.653 12.118 17.18v35.559c0 11.046 8.955 20 20 20h58.02c11.046 0 20-8.954 20-20V80.272c7.154-2.476 12.3-9.252 12.3-17.244 0-4.134-.88-8.222-2.577-11.99q-.058-.16-.13-.316L129.7 31.778v-.002c-2.146-4.607-6.773-7.544-11.853-7.544zm78.752 56.938c-4.967-.537-9.344-3.057-12.309-6.754-3.357 4.184-8.522 6.861-14.305 6.861S83.357 78.6 80 74.416c-2.371 2.956-5.645 5.16-9.402 6.199v26.919h50.321zm-58.321-.153c-4.511-.767-8.463-3.18-11.207-6.601-3.002 3.742-7.45 6.278-12.492 6.773v34.578c0 6.628 5.373 12 12 12h11.782q-.083-.395-.083-.815zm3.097-7.74c-5.705 0-10.304-4.603-10.304-10.249v-.704c0-2.21-1.791-4-4-4s-4 1.79-4 4v.704c0 5.646-4.6 10.25-10.305 10.25-5.706 0-10.305-4.604-10.305-10.25 0-3.086.672-6.134 1.974-8.93l.001-.002 8.81-18.942v-.001h.001c.828-1.776 2.619-2.92 4.6-2.92h75.68c1.982 0 3.773 1.144 4.601 2.92l8.675 18.653q.054.146.121.288c1.3 2.801 1.975 5.852 1.975 8.934 0 5.646-4.599 10.25-10.305 10.25s-10.304-4.604-10.304-10.25v-.704c0-2.21-1.791-4-4-4s-4 1.79-4 4v.704c0 5.646-4.599 10.25-10.305 10.25S84 68.673 84 63.027v-.704c0-2.21-1.79-4-4-4s-4 1.79-4 4v.704c0 5.646-4.599 10.25-10.305 10.25m55.224 42.257v.233c0 6.628-5.373 12-12 12H70.515q.082-.395.083-.815v-11.418z"
        })
    });
const $a163d08dac0a3fe4$var$ForwardRef = /*#__PURE__*/ (0, $h1gTX$react.forwardRef)($a163d08dac0a3fe4$var$SvgComponent);
var $a163d08dac0a3fe4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a163d08dac0a3fe4$var$ForwardRef);
