// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noImage_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $2kmTk$jsxs, jsx as $2kmTk$jsx} from "react/jsx-runtime";
import {forwardRef as $2kmTk$forwardRef} from "react";





const $1a1920d7957529de$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2kmTk$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2kmTk$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M76.2 18c5.522 0 10 4.477 10 10v23.125c0 1.105-.896 2-2 2s-2-.895-2-2V28c0-3.314-2.687-6-6-6H19c-3.315 0-6 2.686-6 6v39.2c0 .678.112 1.33.32 1.939l16.042-24.785v-.001c2.462-3.806 8.038-3.806 10.5 0l12.702 19.621 7.322-11.072.009-.012c2.478-3.687 7.91-3.693 10.384.003.615.918.369 2.16-.55 2.775-.917.614-2.16.368-2.774-.55-.885-1.323-2.839-1.33-3.737-.002L51.258 73.2h8.96c1.104 0 2 .895 2 2s-.896 2-2 2H18.999c-5.523 0-10-4.477-10-10V28c0-5.523 4.477-10 10-10zM32.72 46.526 15.983 72.387c.886.517 1.917.813 3.017.813h27.463l3.694-5.585-13.653-21.088v-.001c-.886-1.37-2.896-1.37-3.782 0M65.5 33c0 2.761 2.239 5 5 5s5-2.239 5-5-2.239-5-5-5-5 2.239-5 5"
            }),
            /*#__PURE__*/ (0, $2kmTk$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M79.688 55.875c-8.147 0-14.75 6.603-14.75 14.75s6.603 14.75 14.75 14.75 14.75-6.603 14.75-14.75-6.604-14.75-14.75-14.75m6.292 18.207c.782.782.782 2.054 0 2.836-.39.39-.901.591-1.413.591-.51 0-1.022-.2-1.412-.591l-3.457-3.457-3.457 3.457c-.391.39-.902.591-1.413.591s-1.023-.2-1.413-.591c-.782-.782-.782-2.054 0-2.836l3.457-3.457-3.457-3.457c-.782-.782-.782-2.054 0-2.836.781-.781 2.054-.781 2.835 0l3.457 3.457 3.458-3.457c.781-.781 2.054-.781 2.835 0 .782.782.782 2.054 0 2.836l-3.457 3.457L86 74.082z"
            })
        ]
    });
const $1a1920d7957529de$var$ForwardRef = /*#__PURE__*/ (0, $2kmTk$forwardRef)($1a1920d7957529de$var$SvgComponent);
var $1a1920d7957529de$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1a1920d7957529de$var$ForwardRef);


export {$1a1920d7957529de$export$2e2bcd8739ae039 as default};
