// @react-spectrum/s2/illustrations/linear/internal/S2_lin_stampClone_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1VFO0$jsx} from "react/jsx-runtime";
import {forwardRef as $1VFO0$forwardRef} from "react";





const $7933ea7761d9558e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1VFO0$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1VFO0$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 19.313c-5.455 0-9.878 4.422-9.878 9.879 0 3.975 2.348 7.407 5.743 8.974q.142.066.268.15c.747.417 1.168 1.285.98 2.165l-2.938 13.72c-.197.922-1.012 1.581-1.956 1.581H27.234c-3.313 0-6 2.687-6 6v3.89h53.534v-3.89c0-3.313-2.687-6-6-6H55.72c-.957 0-1.78-.678-1.963-1.617l-2.672-13.72c-.142-.73.132-1.445.658-1.898.17-.201.384-.372.636-.496 3.264-1.617 5.5-4.979 5.5-8.86 0-5.456-4.422-9.878-9.878-9.878m7.271 21.703c3.962-2.441 6.609-6.822 6.609-11.824 0-7.666-6.214-13.88-13.88-13.88-7.664 0-13.878 6.214-13.878 13.88 0 5.068 2.716 9.498 6.765 11.92l-2.285 10.67H27.234c-5.523 0-10 4.477-10 10v5.89c0 1.105.896 2 2 2h57.534c1.104 0 2-.895 2-2v-5.89c0-5.523-4.478-10-10-10h-11.4zm-33.07 38.656c0-1.105.896-2 2-2h47.6c1.105 0 2 .895 2 2s-.895 2-2 2h-47.6c-1.104 0-2-.896-2-2"
        })
    });
const $7933ea7761d9558e$var$ForwardRef = /*#__PURE__*/ (0, $1VFO0$forwardRef)($7933ea7761d9558e$var$SvgComponent);
var $7933ea7761d9558e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7933ea7761d9558e$var$ForwardRef);


export {$7933ea7761d9558e$export$2e2bcd8739ae039 as default};
