// @react-spectrum/s2/illustrations/linear/internal/S2_lin_heart_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $jYPeo$reactjsxruntime = require("react/jsx-runtime");
var $jYPeo$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f918bde16beb6c3b$export$2e2bcd8739ae039);




const $f918bde16beb6c3b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jYPeo$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jYPeo$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M5.785 17.714C5.321 11.89 9.513 6.746 15.498 6.75c3.652.002 6.832 2.016 8.502 4.994 1.67-2.978 4.85-4.992 8.502-4.994 5.984-.004 10.176 5.14 9.712 10.963-.403 5.065-3.322 9.902-6.536 13.812-3.238 3.937-6.909 7.094-9.063 8.81-1.533 1.22-3.705 1.22-5.238-.001-2.153-1.716-5.822-4.872-9.058-8.81-3.213-3.91-6.13-8.746-6.534-13.81m9.711-8.464c-4.368-.003-7.58 3.736-7.219 8.265.343 4.299 2.87 8.646 5.973 12.422 3.08 3.749 6.601 6.781 8.685 8.442.622.495 1.5.495 2.123 0 2.084-1.66 5.607-4.693 8.689-8.442 3.105-3.777 5.633-8.124 5.975-12.423.361-4.53-2.85-8.267-7.219-8.264-4 .002-7.25 3.258-7.25 7.282q0 .12-.022.237l-.006.03c-.119.585-.634 1.005-1.23 1.003s-1.108-.426-1.222-1.011l-.005-.024q-.021-.116-.022-.235c0-4.024-3.25-7.28-7.25-7.282m7.269 7.497q0-.001 0 0"
        })
    });
const $f918bde16beb6c3b$var$ForwardRef = /*#__PURE__*/ (0, $jYPeo$react.forwardRef)($f918bde16beb6c3b$var$SvgComponent);
var $f918bde16beb6c3b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f918bde16beb6c3b$var$ForwardRef);
