// @react-spectrum/s2/illustrations/linear/internal/S2_lin_exportTo_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $4CJuN$jsx} from "react/jsx-runtime";
import {forwardRef as $4CJuN$forwardRef} from "react";





const $63cce35580189705$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4CJuN$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4CJuN$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M134.937 20.969c2.209 0 4 1.79 4 4v25.935c0 2.21-1.791 4-4 4s-4-1.79-4-4V34.595L86.324 79.207c-1.562 1.562-4.094 1.562-5.656 0s-1.562-4.094 0-5.656l44.581-44.582h-16.248c-2.209 0-4-1.791-4-4s1.791-4 4-4zM43.75 38.453c-6.627 0-12 5.373-12 12v65.556c0 6.627 5.373 12 12 12h65.555c6.628 0 12-5.373 12-12V67.552c0-2.209 1.791-4 4-4s4 1.791 4 4v48.457c0 11.045-8.954 20-20 20H43.75c-11.046 0-20-8.955-20-20V50.453c0-11.046 8.954-20 20-20h48.456c2.21 0 4 1.791 4 4s-1.79 4-4 4z"
        })
    });
const $63cce35580189705$var$ForwardRef = /*#__PURE__*/ (0, $4CJuN$forwardRef)($63cce35580189705$var$SvgComponent);
var $63cce35580189705$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($63cce35580189705$var$ForwardRef);


export {$63cce35580189705$export$2e2bcd8739ae039 as default};
