// @react-spectrum/s2/illustrations/linear/internal/S2_lin_shoppingCart_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $5EYKM$reactjsxruntime = require("react/jsx-runtime");
var $5EYKM$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e6935fcac25295d7$export$2e2bcd8739ae039);




const $e6935fcac25295d7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5EYKM$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5EYKM$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M13.49 16.645c-1.104 0-2 .895-2 2s.896 2 2 2l.725.003h.001l1.237.008c1.05.006 2.456.012 4.11.01 1.988 0 3.679 1.43 3.999 3.371l5.89 35.721c.637 3.864 3.978 6.699 7.893 6.699h32.338c1.105 0 2-.895 2-2s-.895-2-2-2H37.345c-1.957 0-3.628-1.417-3.946-3.35l-1.256-7.613h35.033c2.866 0 5.332-2.027 5.887-4.839l2.47-12.522c.73-3.698-2.09-7.15-5.867-7.16-14.416-.034-36.176-.069-41.567-.004l-.59-3.582c-.642-3.893-4.021-6.721-7.947-6.72-1.645 0-3.043-.005-4.088-.01l-1.203-.008zm15.268 14.317 2.726 16.532h35.692c.956 0 1.778-.675 1.962-1.613l2.47-12.522c.244-1.237-.698-2.383-1.951-2.386-14.02-.033-34.78-.066-40.9-.011m12.25 47.531c0 2.762-2.239 5-5 5s-5-2.238-5-5 2.238-5 5-5 5 2.239 5 5m24.924 5c2.761 0 5-2.238 5-5s-2.239-5-5-5-5 2.239-5 5 2.238 5 5 5"
        })
    });
const $e6935fcac25295d7$var$ForwardRef = /*#__PURE__*/ (0, $5EYKM$react.forwardRef)($e6935fcac25295d7$var$SvgComponent);
var $e6935fcac25295d7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e6935fcac25295d7$var$ForwardRef);
