// @react-spectrum/s2/illustrations/linear/internal/S2_lin_brightnessContrast_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $5vyjS$reactjsxruntime = require("react/jsx-runtime");
var $5vyjS$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f60a780a262e6e19$export$2e2bcd8739ae039);




const $f60a780a262e6e19$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5vyjS$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5vyjS$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25.25 7.48c0-.69-.56-1.25-1.25-1.25-.691 0-1.25.56-1.25 1.25v2.341c0 .69.559 1.25 1.25 1.25.69 0 1.25-.56 1.25-1.25zM16.904 24c0-3.918 3.176-7.094 7.094-7.094v14.187c-3.918 0-7.094-3.176-7.094-7.093M24 14.406c-5.299 0-9.594 4.295-9.594 9.594S18.7 33.593 24 33.593s9.593-4.295 9.593-9.593c0-5.299-4.295-9.594-9.593-9.594m0 22.522c.69 0 1.25.56 1.25 1.25v2.34c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-2.34c0-.69.56-1.25 1.25-1.25M40.52 25.25c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.34c-.691 0-1.25.56-1.25 1.25s.559 1.25 1.25 1.25zM11.07 24c0 .69-.56 1.25-1.25 1.25h-2.34c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h2.34c.69 0 1.25.56 1.25 1.25m25.493-10.799c.488-.488.488-1.28 0-1.767-.488-.489-1.28-.489-1.768 0l-1.655 1.655c-.488.488-.488 1.28 0 1.767.489.489 1.28.489 1.768 0zm-21.707 19.94c.488.488.488 1.279 0 1.767l-1.655 1.655c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.768l1.655-1.655c.489-.488 1.28-.488 1.768 0M11.434 13.2c-.489-.488-.489-1.28 0-1.767.488-.489 1.28-.489 1.767 0l1.655 1.655c.488.488.488 1.28 0 1.767-.488.489-1.28.489-1.767 0zm21.706 19.94c-.488.488-.488 1.279 0 1.767l1.655 1.655c.489.488 1.28.488 1.768 0s.488-1.28 0-1.768l-1.655-1.655c-.488-.488-1.28-.488-1.768 0"
        })
    });
const $f60a780a262e6e19$var$ForwardRef = /*#__PURE__*/ (0, $5vyjS$react.forwardRef)($f60a780a262e6e19$var$SvgComponent);
var $f60a780a262e6e19$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f60a780a262e6e19$var$ForwardRef);
