// @react-spectrum/s2/illustrations/linear/internal/S2_lin_image_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $dD9QL$jsx} from "react/jsx-runtime";
import {forwardRef as $dD9QL$forwardRef} from "react";





const $cf716f21e24b0e26$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dD9QL$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dD9QL$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M82.2 28c0-3.314-2.687-6-6-6H19c-3.315 0-6 2.686-6 6v39.2c0 .678.112 1.33.32 1.939l16.042-24.785v-.001c2.462-3.806 8.038-3.806 10.5 0l12.702 19.62 7.322-11.07.009-.013c2.477-3.686 7.906-3.693 10.382 0l.002.003 11.364 16.834c.357-.768.556-1.624.556-2.527zM32.72 46.526 15.983 72.387c.886.517 1.917.813 3.017.813h27.463l3.694-5.585-13.653-21.088v-.001c-.886-1.37-2.896-1.37-3.782 0m46.051 26.096c-.78.37-1.651.578-2.572.578H51.257l11.96-18.084c.9-1.328 2.853-1.32 3.738.002l.004.007zM76.2 18c5.523 0 10 4.477 10 10v39.2c0 5.523-4.477 10-10 10H19c-5.523 0-10-4.477-10-10V28c0-5.523 4.477-10 10-10zM65.5 33c0 2.761 2.239 5 5 5s5-2.239 5-5-2.239-5-5-5-5 2.239-5 5"
        })
    });
const $cf716f21e24b0e26$var$ForwardRef = /*#__PURE__*/ (0, $dD9QL$forwardRef)($cf716f21e24b0e26$var$SvgComponent);
var $cf716f21e24b0e26$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cf716f21e24b0e26$var$ForwardRef);


export {$cf716f21e24b0e26$export$2e2bcd8739ae039 as default};
