// @react-spectrum/s2/illustrations/linear/internal/S2_lin_volumeOne_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $hI5nd$jsx} from "react/jsx-runtime";
import {forwardRef as $hI5nd$forwardRef} from "react";





const $e83f0f7099633e5d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hI5nd$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hI5nd$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M50.189 17.91c3.156-3.129 8.52-.893 8.52 3.551v53.077c0 4.432-5.34 6.673-8.503 3.569l-11.674-11.46c-.374-.367-.877-.573-1.401-.573h-8.39c-3.314 0-6-2.686-6-6V36.053c0-3.313 2.686-6 6-6h8.378c.527 0 1.034-.208 1.408-.58zm4.52 3.551c0-.889-1.073-1.336-1.704-.71L41.343 32.313l-.04.04v31.41l.03.03L53.01 75.252c.632.62 1.7.173 1.7-.714zM37.303 62.077V34.05l-.184.002H28.74c-1.104 0-2 .896-2 2v24.023c0 1.104.896 2 2 2h8.39q.087 0 .173.002m26.383-27.305c.664-.883 1.918-1.06 2.8-.396 4.11 3.093 6.774 8.018 6.774 13.565 0 5.592-2.708 10.552-6.875 13.64-.887.658-2.14.471-2.798-.416-.657-.888-.47-2.14.417-2.798 3.192-2.366 5.256-6.155 5.256-10.426 0-4.237-2.03-8-5.179-10.369-.883-.664-1.06-1.918-.396-2.8"
        })
    });
const $e83f0f7099633e5d$var$ForwardRef = /*#__PURE__*/ (0, $hI5nd$forwardRef)($e83f0f7099633e5d$var$SvgComponent);
var $e83f0f7099633e5d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e83f0f7099633e5d$var$ForwardRef);


export {$e83f0f7099633e5d$export$2e2bcd8739ae039 as default};
