// @react-spectrum/s2/illustrations/linear/internal/S2_lin_minimize_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $lvQJ9$jsx} from "react/jsx-runtime";
import {forwardRef as $lvQJ9$forwardRef} from "react";





const $1118fa9adfdf36d6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lvQJ9$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $lvQJ9$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M78.934 20.414c.78-.78.78-2.047 0-2.828-.781-.781-2.048-.781-2.829 0L56.922 36.769V24.258c0-1.105-.896-2-2-2s-2 .895-2 2v17.324c0 1.105.895 2 2 2h17.324c1.105 0 2-.895 2-2s-.895-2-2-2h-12.48zM16.586 77.086c-.781.78-.781 2.047 0 2.828.78.781 2.047.781 2.828 0l19.79-19.789v12.69c0 1.104.895 2 2 2s2-.896 2-2V55.49c0-1.104-.896-2-2-2H23.878c-1.105 0-2 .896-2 2s.895 2 2 2h12.302z"
        })
    });
const $1118fa9adfdf36d6$var$ForwardRef = /*#__PURE__*/ (0, $lvQJ9$forwardRef)($1118fa9adfdf36d6$var$SvgComponent);
var $1118fa9adfdf36d6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1118fa9adfdf36d6$var$ForwardRef);


export {$1118fa9adfdf36d6$export$2e2bcd8739ae039 as default};
