// @react-spectrum/s2/illustrations/linear/internal/S2_lin_buildings_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $gBz2m$reactjsxruntime = require("react/jsx-runtime");
var $gBz2m$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6d230a3d7c2f8c2f$export$2e2bcd8739ae039);




const $6d230a3d7c2f8c2f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gBz2m$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gBz2m$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M35.76 32.1c-6.621 0-12 5.379-12 12v84.13h17.98v-16.58c0-6.629 5.371-12 12-12h12.09c6.63 0 12 5.371 12 12v16.58h18V44.1c0-6.621-5.38-12-12-12zm38.09 104.13h25.98c2.209 0 4-1.791 4-4V44.1c0-11.04-8.961-20-20-20H35.76c-11.04 0-20 8.96-20 20v88.13c0 2.209 1.79 4 4 4h54.09m-4.02-8v-16.58c0-2.211-1.789-4-4-4H53.74c-2.21 0-4 1.789-4 4v16.58zm40.78-78.52c0-2.21 1.791-4 4-4h9.63c11.049 0 20 8.95 20 20v66.52c0 2.209-1.791 4-4 4h-25.63c-2.209 0-4-1.791-4-4s1.791-4 4-4h21.63V65.71c0-6.631-5.369-12-12-12h-9.63c-2.209 0-4-1.791-4-4m-62.08-6.88h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.209 1.79 4 4 4h5.76c2.209 0 4-1.791 4-4v-5.76c0-2.21-1.791-4-4-4m22.51 0h5.76c2.208 0 4 1.79 4 4v5.76c0 2.209-1.792 4-4 4h-5.76c-2.21 0-4-1.791-4-4v-5.76c0-2.21 1.79-4 4-4M48.53 70.89h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.209 1.79 4 4 4h5.76c2.209 0 4-1.791 4-4v-5.76c0-2.21-1.791-4-4-4m22.51 0h5.76c2.208 0 4 1.79 4 4v5.76c0 2.209-1.792 4-4 4h-5.76c-2.21 0-4-1.791-4-4v-5.76c0-2.21 1.79-4 4-4m51.87-.83h-5.76c-2.209 0-4 1.79-4 4v5.76c0 2.209 1.791 4 4 4h5.76c2.21 0 4-1.791 4-4v-5.76c0-2.21-1.79-4-4-4m-5.76 28.06h5.76c2.21 0 4 1.79 4 4v5.76c0 2.209-1.79 4-4 4h-5.76c-2.209 0-4-1.791-4-4v-5.76c0-2.21 1.791-4 4-4"
        })
    });
const $6d230a3d7c2f8c2f$var$ForwardRef = /*#__PURE__*/ (0, $gBz2m$react.forwardRef)($6d230a3d7c2f8c2f$var$SvgComponent);
var $6d230a3d7c2f8c2f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6d230a3d7c2f8c2f$var$ForwardRef);
