// @react-spectrum/s2/illustrations/linear/internal/S2_lin_search_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dEfQK$reactjsxruntime = require("react/jsx-runtime");
var $dEfQK$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $8fd9a27d7028012f$export$2e2bcd8739ae039);




const $8fd9a27d7028012f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dEfQK$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dEfQK$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M19.648 6.99c-7.042 0-12.75 5.708-12.75 12.75s5.708 12.75 12.75 12.75c3.07 0 5.887-1.086 8.088-2.894l.407.407-.534.534c-.489.488-.489 1.28 0 1.768l8.564 8.564c1.263 1.263 3.31 1.263 4.573 0s1.263-3.31 0-4.573l-8.564-8.564c-.488-.488-1.28-.488-1.768 0l-.504.503-.406-.406c1.808-2.201 2.894-5.018 2.894-8.089 0-7.042-5.709-12.75-12.75-12.75M9.398 19.74c0-5.66 4.589-10.25 10.25-10.25s10.25 4.59 10.25 10.25-4.59 10.25-10.25 10.25-10.25-4.59-10.25-10.25m21.9 10.644L30.26 31.42l7.681 7.68c.286.287.751.287 1.037 0 .287-.286.287-.75 0-1.037z"
        })
    });
const $8fd9a27d7028012f$var$ForwardRef = /*#__PURE__*/ (0, $dEfQK$react.forwardRef)($8fd9a27d7028012f$var$SvgComponent);
var $8fd9a27d7028012f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($8fd9a27d7028012f$var$ForwardRef);
