// @react-spectrum/s2/illustrations/linear/internal/S2_lin_rotateCCW_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7vP5n$jsx} from "react/jsx-runtime";
import {forwardRef as $7vP5n$forwardRef} from "react";





const $cd77358111f7a741$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7vP5n$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7vP5n$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M79 48.492c0-17.12-13.88-31-31-31-12.189 0-22.738 7.035-27.803 17.271l9.735-3.13c1.051-.338 2.178.24 2.516 1.292.338 1.051-.24 2.178-1.292 2.516L17.1 39.961c-1.052.338-2.179-.24-2.517-1.292l-4.52-14.056c-.338-1.052.24-2.178 1.292-2.517s2.178.24 2.516 1.292l2.953 9.181C22.618 21.247 34.4 13.492 48 13.492c19.33 0 35 15.67 35 35s-15.67 35-35 35c-15.118 0-27.992-9.583-32.888-23-.38-1.037.155-2.186 1.193-2.564 1.037-.379 2.185.155 2.564 1.193C23.21 71.01 34.616 79.492 48 79.492c17.122 0 31-13.879 31-31"
        })
    });
const $cd77358111f7a741$var$ForwardRef = /*#__PURE__*/ (0, $7vP5n$forwardRef)($cd77358111f7a741$var$SvgComponent);
var $cd77358111f7a741$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cd77358111f7a741$var$ForwardRef);


export {$cd77358111f7a741$export$2e2bcd8739ae039 as default};
