// @react-spectrum/s2/illustrations/linear/internal/S2_lin_link_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $bkDub$jsx} from "react/jsx-runtime";
import {forwardRef as $bkDub$forwardRef} from "react";





const $d60657acd4c03ad4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bkDub$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bkDub$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M123.195 57.88c5.847-5.848 5.847-15.329 0-21.176s-15.328-5.847-21.176 0L91.106 47.617c-1.562 1.562-4.094 1.562-5.656 0s-1.562-4.095 0-5.657l10.912-10.913c8.972-8.972 23.518-8.971 32.49 0 8.971 8.972 8.971 23.518 0 32.49l-27.672 27.67c-8.971 8.972-23.517 8.972-32.489 0-1.562-1.561-1.562-4.094 0-5.656s4.095-1.562 5.657 0c5.847 5.847 15.328 5.847 21.176 0zM91.54 68.564c1.562 1.562 1.562 4.094 0 5.657s-4.095 1.562-5.657 0c-5.848-5.848-15.328-5.848-21.176 0l-27.671 27.671c-5.847 5.847-5.847 15.328 0 21.175s15.328 5.848 21.176 0l10.912-10.912c1.562-1.562 4.095-1.562 5.657 0s1.562 4.094 0 5.657l-10.913 10.912c-8.971 8.972-23.517 8.972-32.489 0s-8.972-23.517 0-32.49l27.671-27.67c8.972-8.972 23.518-8.972 32.49 0"
        })
    });
const $d60657acd4c03ad4$var$ForwardRef = /*#__PURE__*/ (0, $bkDub$forwardRef)($d60657acd4c03ad4$var$SvgComponent);
var $d60657acd4c03ad4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d60657acd4c03ad4$var$ForwardRef);


export {$d60657acd4c03ad4$export$2e2bcd8739ae039 as default};
