// @react-spectrum/s2/illustrations/linear/internal/S2_lin_threeArrowsCircle_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $e0jvz$jsx} from "react/jsx-runtime";
import {forwardRef as $e0jvz$forwardRef} from "react";





const $5321ca1f306dcc3d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e0jvz$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $e0jvz$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M36.332 11.291c.157.672-.26 1.345-.932 1.502l-6.885 1.613c-.672.158-1.344-.26-1.502-.932s.26-1.344.932-1.502l4.08-.955c-2.343-1.467-5.111-2.314-8.08-2.314-4.574 0-8.676 2.012-11.473 5.204-.455.519-1.245.57-1.764.116-.52-.455-.572-1.245-.117-1.764 3.252-3.71 8.03-6.056 13.354-6.056 3.381 0 6.543.946 9.234 2.588l-.894-3.814c-.157-.672.26-1.345.932-1.502s1.345.26 1.502.932zm2.248 8.125c-.206-.659.161-1.36.82-1.566s1.36.16 1.566.82c.525 1.676.807 3.46.807 5.306 0 7.05-4.11 13.138-10.062 16.002l3.81 1.205c.657.208 1.022.91.814 1.568-.208.659-.91 1.023-1.568.815l-6.743-2.132c-.658-.208-1.023-.91-.815-1.568l2.132-6.742c.208-.659.91-1.023 1.569-.815s1.023.91.815 1.568l-1.238 3.915c5.192-2.433 8.786-7.706 8.786-13.816 0-1.59-.243-3.122-.693-4.56M6.308 22.58l-2.995 2.555c-.525.448-1.314.385-1.762-.14s-.386-1.314.14-1.762l5.379-4.59c.525-.448 1.314-.386 1.762.14l4.59 5.378c.448.526.385 1.315-.14 1.763s-1.314.385-1.762-.14l-2.706-3.171c-.243 2.675.218 5.447 1.48 8.039 2.004 4.112 5.609 6.92 9.703 8.037.666.181 1.059.869.877 1.535s-.87 1.058-1.535.877c-4.76-1.3-8.96-4.568-11.292-9.354-1.438-2.954-1.983-6.113-1.74-9.167"
        })
    });
const $5321ca1f306dcc3d$var$ForwardRef = /*#__PURE__*/ (0, $e0jvz$forwardRef)($5321ca1f306dcc3d$var$SvgComponent);
var $5321ca1f306dcc3d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5321ca1f306dcc3d$var$ForwardRef);


export {$5321ca1f306dcc3d$export$2e2bcd8739ae039 as default};
