// @react-spectrum/s2/illustrations/linear/internal/S2_lin_camera_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $eZxhV$reactjsxruntime = require("react/jsx-runtime");
var $eZxhV$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b8b82ce5553649ac$export$2e2bcd8739ae039);




const $b8b82ce5553649ac$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eZxhV$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $eZxhV$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m17.506 14.75 2.59-4.141c.147-.227.393-.359.644-.359h6.53c.251 0 .497.132.644.359l2.59 4.141zm-2.949 0 3.423-5.473.006-.01c.594-.93 1.627-1.517 2.754-1.517h6.53c1.128 0 2.16.587 2.754 1.518l.006.01 3.423 5.472H37c2.9 0 5.25 2.35 5.25 5.25v15c0 2.9-2.35 5.25-5.25 5.25H11c-2.9 0-5.25-2.35-5.25-5.25V20c0-2.9 2.35-5.25 5.25-5.25zm.693 2.5H11c-1.519 0-2.75 1.231-2.75 2.75v15c0 1.519 1.231 2.75 2.75 2.75h26c1.519 0 2.75-1.231 2.75-2.75V20c0-1.519-1.231-2.75-2.75-2.75H15.25M24 22.26c-2.894 0-5.24 2.346-5.24 5.24s2.346 5.24 5.24 5.24 5.24-2.346 5.24-5.24-2.346-5.24-5.24-5.24m-7.74 5.24c0-4.275 3.465-7.74 7.74-7.74 4.274 0 7.74 3.465 7.74 7.74s-3.466 7.74-7.74 7.74c-4.275 0-7.74-3.465-7.74-7.74m-2.01-6c0 .966-.784 1.75-1.75 1.75s-1.75-.784-1.75-1.75.784-1.75 1.75-1.75 1.75.784 1.75 1.75"
        })
    });
const $b8b82ce5553649ac$var$ForwardRef = /*#__PURE__*/ (0, $eZxhV$react.forwardRef)($b8b82ce5553649ac$var$SvgComponent);
var $b8b82ce5553649ac$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b8b82ce5553649ac$var$ForwardRef);
