// @react-spectrum/s2/illustrations/linear/internal/S2_lin_shoppingCart_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $eIe2w$jsx} from "react/jsx-runtime";
import {forwardRef as $eIe2w$forwardRef} from "react";





const $eeea38179b0a398a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eIe2w$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $eIe2w$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M13.49 16.645c-1.104 0-2 .895-2 2s.896 2 2 2l.725.003h.001l1.237.008c1.05.006 2.456.012 4.11.01 1.988 0 3.679 1.43 3.999 3.371l5.89 35.721c.637 3.864 3.978 6.699 7.893 6.699h32.338c1.105 0 2-.895 2-2s-.895-2-2-2H37.345c-1.957 0-3.628-1.417-3.946-3.35l-1.256-7.613h35.033c2.866 0 5.332-2.027 5.887-4.839l2.47-12.522c.73-3.698-2.09-7.15-5.867-7.16-14.416-.034-36.176-.069-41.567-.004l-.59-3.582c-.642-3.893-4.021-6.721-7.947-6.72-1.645 0-3.043-.005-4.088-.01l-1.203-.008zm15.268 14.317 2.726 16.532h35.692c.956 0 1.778-.675 1.962-1.613l2.47-12.522c.244-1.237-.698-2.383-1.951-2.386-14.02-.033-34.78-.066-40.9-.011m12.25 47.531c0 2.762-2.239 5-5 5s-5-2.238-5-5 2.238-5 5-5 5 2.239 5 5m24.924 5c2.761 0 5-2.238 5-5s-2.239-5-5-5-5 2.239-5 5 2.238 5 5 5"
        })
    });
const $eeea38179b0a398a$var$ForwardRef = /*#__PURE__*/ (0, $eIe2w$forwardRef)($eeea38179b0a398a$var$SvgComponent);
var $eeea38179b0a398a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($eeea38179b0a398a$var$ForwardRef);


export {$eeea38179b0a398a$export$2e2bcd8739ae039 as default};
