// @react-spectrum/s2/illustrations/linear/internal/S2_lin_logo_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $eJ4I7$jsx} from "react/jsx-runtime";
import {forwardRef as $eJ4I7$forwardRef} from "react";





const $d2f80d1efc1b3802$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eJ4I7$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $eJ4I7$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M26.16 6.409c-1.053-.545-2.305-.545-3.357 0L8.972 13.57C7.76 14.197 7 15.448 7 16.812V30.7c0 1.364.76 2.614 1.972 3.241l13.83 7.162c1.053.545 2.305.545 3.358 0l13.83-7.162c1.212-.627 1.972-1.877 1.972-3.24V16.81c0-1.363-.76-2.614-1.971-3.24zm-2.208 2.22c.332-.172.726-.172 1.058 0l13.831 7.161c.382.198.621.592.621 1.022V30.7c0 .43-.24.824-.62 1.021L25.01 38.883c-.332.172-.726.172-1.058 0l-13.83-7.162c-.382-.197-.622-.591-.622-1.02V16.81c0-.43.24-.823.621-1.02zm5.676 10.352 3.792 6.581c.941 1.635-.23 3.688-2.127 3.688H23.71c-1.898 0-3.069-2.053-2.128-3.687l3.79-6.581c.945-1.642 3.311-1.643 4.257 0M27.5 20.295l-3.718 6.455h7.437zm-6.565-.974c.598.345.803 1.11.458 1.707l-4.359 7.548c-.345.598-1.11.803-1.707.457-.598-.345-.803-1.11-.458-1.707l4.359-7.548c.345-.598 1.11-.802 1.707-.457"
        })
    });
const $d2f80d1efc1b3802$var$ForwardRef = /*#__PURE__*/ (0, $eJ4I7$forwardRef)($d2f80d1efc1b3802$var$SvgComponent);
var $d2f80d1efc1b3802$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d2f80d1efc1b3802$var$ForwardRef);


export {$d2f80d1efc1b3802$export$2e2bcd8739ae039 as default};
