// @react-spectrum/s2/illustrations/linear/internal/S2_lin_revert_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1rh9A$jsx} from "react/jsx-runtime";
import {forwardRef as $1rh9A$forwardRef} from "react";





const $12e7f7dd31898900$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1rh9A$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1rh9A$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m19.943 52.95 10.935-8.574c.87-.682 2.126-.53 2.808.34.681.869.53 2.126-.34 2.808L18.777 58.948c-.869.681-2.126.53-2.808-.34L4.546 44.04c-.681-.869-.53-2.126.34-2.808s2.126-.53 2.808.34l8.309 10.596c1.858-17.47 16.643-31.08 34.606-31.08 19.22 0 34.803 15.583 34.803 34.804 0 1.104-.896 2-2 2s-2-.896-2-2c0-17.012-13.791-30.803-30.803-30.803-16.02 0-29.184 12.23-30.665 27.862m-5.86 20.947c0-1.105.895-2 2-2h69.03c1.105 0 2 .895 2 2s-.895 2-2 2h-69.03c-1.105 0-2-.896-2-2"
        })
    });
const $12e7f7dd31898900$var$ForwardRef = /*#__PURE__*/ (0, $1rh9A$forwardRef)($12e7f7dd31898900$var$SvgComponent);
var $12e7f7dd31898900$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($12e7f7dd31898900$var$ForwardRef);


export {$12e7f7dd31898900$export$2e2bcd8739ae039 as default};
