// @react-spectrum/s2/illustrations/linear/internal/S2_lin_userAvatar_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $fOrpr$jsx} from "react/jsx-runtime";
import {forwardRef as $fOrpr$forwardRef} from "react";





const $24aca91bb28ea083$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fOrpr$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fOrpr$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 16.74c-17.12 0-31 13.88-31 31 0 8.125 3.126 15.52 8.24 21.047C29.806 60.94 38.32 55.61 48 55.61s18.193 5.335 22.76 13.178C75.874 63.26 79 55.865 79 47.74c0-17.12-13.88-31-31-31m0 62c-7.51 0-14.396-2.67-19.761-7.114C31.992 64.5 39.482 59.61 48 59.61c8.517 0 16.006 4.892 19.76 12.017C62.398 76.07 55.512 78.74 48 78.74m-35-31c0-19.33 15.67-35 35-35s35 15.67 35 35-15.67 35-35 35-35-15.67-35-35m24.913-8.987c0-6.129 4.634-10.878 10.088-10.878s10.087 4.75 10.087 10.878c0 6.13-4.633 10.878-10.087 10.878s-10.088-4.749-10.088-10.878m10.088-14.878c-7.898 0-14.088 6.782-14.088 14.878s6.19 14.878 14.088 14.878 14.087-6.782 14.087-14.878-6.19-14.878-14.087-14.878"
        })
    });
const $24aca91bb28ea083$var$ForwardRef = /*#__PURE__*/ (0, $fOrpr$forwardRef)($24aca91bb28ea083$var$SvgComponent);
var $24aca91bb28ea083$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($24aca91bb28ea083$var$ForwardRef);


export {$24aca91bb28ea083$export$2e2bcd8739ae039 as default};
