// @react-spectrum/s2/illustrations/linear/internal/S2_lin_rocket_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $hyq17$jsxs, jsx as $hyq17$jsx} from "react/jsx-runtime";
import {forwardRef as $hyq17$forwardRef} from "react";





const $38c8f7b75e12affe$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hyq17$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hyq17$jsx)("defs", {
                children: /*#__PURE__*/ (0, $hyq17$jsx)("clipPath", {
                    id: "0b4ce3__a",
                    children: /*#__PURE__*/ (0, $hyq17$jsx)("path", {
                        fill: "#fff",
                        d: "M0 0h160v160H0z"
                    })
                })
            }),
            /*#__PURE__*/ (0, $hyq17$jsx)("g", {
                clipPath: "url(#0b4ce3__a)",
                children: /*#__PURE__*/ (0, $hyq17$jsx)("path", {
                    fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                    fillRule: "evenodd",
                    d: "M127.925 34.61c-.255-1.436-1.347-2.527-2.784-2.782-8.619-1.53-26.41.909-44.194 18.693C67.2 64.268 61.268 71.98 58.728 76.583c-1.23 2.231-1.586 3.595-1.704 4.464-.094.685-.076 1.043-.048 1.604.018.357.04.797.04 1.458 0 2.463 1.314 5.313 7.314 11.313s8.85 7.314 11.314 7.314c.66 0 1.1.022 1.457.04.561.028.919.046 1.604-.048.869-.118 2.233-.473 4.464-1.704 4.604-2.54 12.315-8.472 26.062-22.219 17.784-17.784 20.224-35.575 18.694-44.194m-1.385-10.66c4.741.843 8.42 4.521 9.261 9.262 1.926 10.844-1.121 29.63-17.873 48.063.878 4.88.175 10.61-1.281 16.178-1.695 6.485-4.558 13.298-7.964 19.334-3.385 5.998-7.448 11.478-11.669 15.116-3.957 3.411-9.491 6.35-14.89 3.65-4.737-2.368-6.045-6.865-6.482-11.37-.22-2.269-.251-4.866-.257-7.665v-1.798c0-1.268 0-2.593-.01-3.986-5.901-.107-10.771-3.724-16.702-9.655s-9.548-10.801-9.654-16.702c-1.391-.01-2.714-.01-3.98-.01h-.011l-1.795-.001c-2.798-.005-5.396-.037-7.665-.257-4.505-.436-9.002-1.745-11.37-6.481-2.7-5.4.239-10.934 3.65-14.891 3.638-4.22 9.119-8.284 15.116-11.668 6.036-3.407 12.85-6.27 19.334-7.965 5.569-1.456 11.299-2.158 16.18-1.28C96.91 25.072 115.696 22.025 126.54 23.95M87.034 108.03c4.967-2.741 12.042-8.19 23.082-18.87-.22 1.92-.621 4.024-1.209 6.27-1.5 5.739-4.081 11.915-7.191 17.426-3.132 5.549-6.658 10.173-9.925 12.989-3.53 3.043-5.38 2.909-6.09 2.554-.92-.46-1.736-1.269-2.096-4.987-.18-1.841-.214-4.083-.22-6.908v-1.736c0-1.55 0-3.231-.02-5.03 1.119-.413 2.33-.97 3.669-1.708m-16.44-58.393c-1.921.219-4.025.62-6.272 1.208-5.74 1.5-11.915 4.081-17.426 7.192-5.549 3.131-10.172 6.657-12.989 9.924-3.043 3.53-2.908 5.38-2.554 6.09.46.92 1.27 1.737 4.987 2.097 1.842.178 4.084.214 6.908.22h1.736c1.55 0 3.232 0 5.031.019.414-1.118.97-2.33 1.709-3.667 2.74-4.968 8.19-12.044 18.87-23.083m37.93 12.905c-3.124 3.125-8.189 3.125-11.314 0s-3.124-8.189 0-11.313 8.19-3.124 11.314 0 3.124 8.19 0 11.313m-58.69 53.034c1.563-1.563 1.563-4.095 0-5.657s-4.094-1.562-5.656 0l-19.446 19.445c-1.562 1.562-1.562 4.095 0 5.657s4.095 1.562 5.657 0zm9.9 4.242c1.562 1.562 1.562 4.095 0 5.657L48.42 136.788c-1.562 1.562-4.095 1.562-5.657 0s-1.562-4.095 0-5.657l11.314-11.314c1.562-1.562 4.095-1.562 5.657 0m-19.8-14.142c1.562-1.562 1.562-4.095 0-5.657s-4.095-1.562-5.657 0l-11.314 11.314c-1.562 1.562-1.562 4.095 0 5.657s4.095 1.562 5.657 0z"
                })
            })
        ]
    });
const $38c8f7b75e12affe$var$ForwardRef = /*#__PURE__*/ (0, $hyq17$forwardRef)($38c8f7b75e12affe$var$SvgComponent);
var $38c8f7b75e12affe$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($38c8f7b75e12affe$var$ForwardRef);


export {$38c8f7b75e12affe$export$2e2bcd8739ae039 as default};
