// @react-spectrum/s2/illustrations/linear/internal/S2_lin_imageStack_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $lqrkp$reactjsxruntime = require("react/jsx-runtime");
var $lqrkp$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ee09f7eaa84b47a1$export$2e2bcd8739ae039);




const $ee09f7eaa84b47a1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lqrkp$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lqrkp$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M58.598 40.37c2.761 0 5-2.238 5-5 0-2.76-2.239-5-5-5s-5 2.24-5 5c0 2.762 2.238 5 5 5"
            }),
            /*#__PURE__*/ (0, $lqrkp$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M12.606 30.87c0-5.907 4.789-10.696 10.696-10.696h40.221c5.908 0 10.697 4.79 10.697 10.697v26.633c0 5.907-4.79 10.696-10.697 10.696H23.302c-5.907 0-10.696-4.789-10.696-10.696zm10.696-6.696c-3.698 0-6.696 2.998-6.696 6.697v26.633c0 .707.11 1.388.312 2.028l11.756-18.22c2.207-3.423 7.231-3.368 9.363.102l9.347 15.212 5.488-8.373c2.209-3.37 7.166-3.324 9.311.088l4.418 7.024 3.053 4.838c.364-.826.566-1.739.566-2.7V30.872c0-3.699-2.998-6.697-6.697-6.697zm43.577 39.127L58.797 50.47c-.594-.945-1.968-.958-2.58-.024L47.203 64.2h16.32c1.223 0 2.369-.327 3.355-.9m-24.459.9 2.541-3.878L34.63 43.508c-.59-.961-1.982-.976-2.594-.028L19.452 62.983c1.089.767 2.417 1.217 3.85 1.217z"
            }),
            /*#__PURE__*/ (0, $lqrkp$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M82.412 40.942c0-1.104-.896-2-2-2s-2 .896-2 2v22.013c0 4.9-3.972 8.87-8.87 8.87H33.667c-1.105 0-2 .896-2 2s.895 2 2 2h35.873c7.108 0 12.87-5.762 12.87-12.87z"
            })
        ]
    });
const $ee09f7eaa84b47a1$var$ForwardRef = /*#__PURE__*/ (0, $lqrkp$react.forwardRef)($ee09f7eaa84b47a1$var$SvgComponent);
var $ee09f7eaa84b47a1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ee09f7eaa84b47a1$var$ForwardRef);
