// @react-spectrum/s2/illustrations/linear/internal/S2_lin_shoppingBag_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2viuO$reactjsxruntime = require("react/jsx-runtime");
var $2viuO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e948f1a2aed91a00$export$2e2bcd8739ae039);




const $e948f1a2aed91a00$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2viuO$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2viuO$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M21.25 12c0-1.38 1.12-2.5 2.5-2.5 1.381 0 2.5 1.12 2.5 2.5v1.5h-5zm-2.5 4v2.75c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V16h5v2.75c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V16h4.62l1.947 19.476c.162 1.62-1.11 3.024-2.736 3.024H14.92c-1.627 0-2.899-1.405-2.737-3.024L14.132 16zm0-2.5V12c0-2.761 2.24-5 5-5 2.762 0 5 2.239 5 5v1.5h4.846c1.155 0 2.123.876 2.238 2.026l1.97 19.702c.31 3.09-2.117 5.772-5.224 5.772H14.92c-3.106 0-5.533-2.682-5.224-5.772l1.97-19.702c.116-1.15 1.083-2.026 2.24-2.026z"
        })
    });
const $e948f1a2aed91a00$var$ForwardRef = /*#__PURE__*/ (0, $2viuO$react.forwardRef)($e948f1a2aed91a00$var$SvgComponent);
var $e948f1a2aed91a00$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e948f1a2aed91a00$var$ForwardRef);
