// @react-spectrum/s2/illustrations/linear/internal/S2_lin_chatbubble_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $kK0HL$reactjsxruntime = require("react/jsx-runtime");
var $kK0HL$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a905cbdc375af210$export$2e2bcd8739ae039);




const $a905cbdc375af210$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kK0HL$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kK0HL$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16 42.311C16 32.175 24.262 24 34.4 24H125.6C135.738 24 144 32.175 144 42.311V93.1c0 10.137-8.262 18.312-18.401 18.312H88.466l-30.693 26.443c-5.681 4.895-14.61.9-14.555-6.696.044-6.003.087-13.527.087-19.747H34.4C24.26 111.411 16 103.236 16 93.099zM34.4 32C28.634 32 24 36.64 24 42.311V93.1c0 5.672 4.633 10.312 10.4 10.312h16.88l.017 3.983c.028 6.778-.025 16.463-.08 23.823 0 .209.053.331.11.417.07.103.185.208.346.282s.323.095.457.079c.115-.014.255-.058.421-.201L84.37 104.38l1.125-.969H125.6c5.768 0 10.401-4.64 10.401-10.312V42.311C136 36.64 131.367 32 125.599 32z"
        })
    });
const $a905cbdc375af210$var$ForwardRef = /*#__PURE__*/ (0, $kK0HL$react.forwardRef)($a905cbdc375af210$var$SvgComponent);
var $a905cbdc375af210$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a905cbdc375af210$var$ForwardRef);
