// @react-spectrum/s2/illustrations/linear/internal/S2_lin_artboard_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $eCPIs$jsx} from "react/jsx-runtime";
import {forwardRef as $eCPIs$forwardRef} from "react";





const $eacc2a4b700ef5ec$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eCPIs$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $eCPIs$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M40 14c0-2.21-1.79-4-4-4s-4 1.79-4 4v10c0 2.21 1.79 4 4 4s4-1.79 4-4zM14 32c-2.21 0-4 1.79-4 4s1.79 4 4 4h10c2.21 0 4-1.79 4-4s-1.79-4-4-4zm34 8c-4.418 0-8 3.582-8 8v80c0 4.418 3.582 8 8 8h64c4.418 0 8-3.582 8-8V80H91c-6.627 0-12-5.373-12-12V40zm39 4.44V68c0 2.21 1.79 4 4 4h24.634zM32 48c0-8.837 7.163-16 16-16h31.163c4.138 0 8.115 1.603 11.096 4.472l32.836 31.606c3.134 3.016 4.905 7.178 4.905 11.527V128c0 8.837-7.163 16-16 16H48c-8.837 0-16-7.163-16-16z"
        })
    });
const $eacc2a4b700ef5ec$var$ForwardRef = /*#__PURE__*/ (0, $eCPIs$forwardRef)($eacc2a4b700ef5ec$var$SvgComponent);
var $eacc2a4b700ef5ec$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($eacc2a4b700ef5ec$var$ForwardRef);


export {$eacc2a4b700ef5ec$export$2e2bcd8739ae039 as default};
