// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lighten_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bkrdd$reactjsxruntime = require("react/jsx-runtime");
var $bkrdd$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d3ea09994fed7d31$export$2e2bcd8739ae039);




const $d3ea09994fed7d31$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bkrdd$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bkrdd$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 6.23c.69 0 1.25.56 1.25 1.25v2.341c0 .69-.56 1.25-1.25 1.25-.691 0-1.25-.56-1.25-1.25v-2.34c0-.69.559-1.25 1.25-1.25m-.001 10.676c-3.918 0-7.094 3.176-7.094 7.094 0 3.917 3.176 7.093 7.094 7.093 3.917 0 7.093-3.176 7.093-7.093 0-3.918-3.176-7.094-7.093-7.094M14.405 24c0-5.299 4.295-9.594 9.594-9.594s9.593 4.295 9.593 9.594-4.295 9.593-9.593 9.593c-5.299 0-9.594-4.295-9.594-9.593M25.25 38.178c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v2.34c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25zM41.77 24c0 .69-.56 1.25-1.25 1.25h-2.34c-.691 0-1.25-.56-1.25-1.25s.559-1.25 1.25-1.25h2.34c.69 0 1.25.56 1.25 1.25M9.82 25.25c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.34c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25zm26.743-13.816c.488.488.488 1.28 0 1.767l-1.655 1.655c-.488.489-1.28.489-1.768 0-.488-.488-.488-1.28 0-1.767l1.655-1.655c.489-.489 1.28-.489 1.768 0M14.857 34.908c.488-.488.488-1.28 0-1.768s-1.28-.488-1.768 0l-1.655 1.655c-.488.489-.488 1.28 0 1.768.489.488 1.28.488 1.768 0zm-3.423-23.474c-.489.488-.489 1.28 0 1.767l1.655 1.655c.488.489 1.28.489 1.767 0 .488-.488.488-1.28 0-1.767l-1.655-1.655c-.488-.489-1.28-.489-1.767 0M33.14 34.908c-.488-.488-.488-1.28 0-1.768.489-.488 1.28-.488 1.768 0l1.655 1.655c.488.489.488 1.28 0 1.768s-1.28.488-1.768 0z"
        })
    });
const $d3ea09994fed7d31$var$ForwardRef = /*#__PURE__*/ (0, $bkrdd$react.forwardRef)($d3ea09994fed7d31$var$SvgComponent);
var $d3ea09994fed7d31$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d3ea09994fed7d31$var$ForwardRef);
