// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowLeft_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $bCjzA$jsx} from "react/jsx-runtime";
import {forwardRef as $bCjzA$forwardRef} from "react";





const $ce38cb89068a7342$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bCjzA$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bCjzA$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M136 84c2.209 0 4-1.79 4-4s-1.791-4-4-4H31.157l16.671-16.672c1.563-1.562 1.563-4.094 0-5.656s-4.094-1.562-5.656 0l-23.5 23.5c-1.562 1.562-1.562 4.094 0 5.656l23.5 23.5c1.562 1.563 4.094 1.563 5.656 0 1.563-1.562 1.563-4.094 0-5.656L31.157 84z"
        })
    });
const $ce38cb89068a7342$var$ForwardRef = /*#__PURE__*/ (0, $bCjzA$forwardRef)($ce38cb89068a7342$var$SvgComponent);
var $ce38cb89068a7342$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ce38cb89068a7342$var$ForwardRef);


export {$ce38cb89068a7342$export$2e2bcd8739ae039 as default};
