// @react-spectrum/s2/illustrations/linear/internal/S2_lin_libraries_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $b1o5C$jsx} from "react/jsx-runtime";
import {forwardRef as $b1o5C$forwardRef} from "react";





const $51a99a1996ad0f9a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $b1o5C$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $b1o5C$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20.53 28.52c0-4.416 3.584-8 8-8h31.95c1.104 0 2-.896 2-2s-.896-2-2-2H28.53c-6.625 0-12 5.375-12 12v31.95c0 1.104.895 2 2 2s2-.896 2-2zm13-.99c-3.314 0-6 2.686-6 6v35.95c0 3.313 2.686 6 6 6h35.95c3.313 0 6-2.687 6-6V33.53c0-3.314-2.687-6-6-6h-.39v22.21c0 .8-.89 1.27-1.55.83l-4.87-3.25c-.34-.22-.77-.22-1.11 0l-4.87 3.25c-.66.45-1.55-.03-1.55-.83V27.53zm-10 6c0-5.523 4.476-10 10-10h35.95c5.522 0 10 4.477 10 10v35.95c0 5.522-4.478 10-10 10H33.53c-5.524 0-10-4.478-10-10z"
        })
    });
const $51a99a1996ad0f9a$var$ForwardRef = /*#__PURE__*/ (0, $b1o5C$forwardRef)($51a99a1996ad0f9a$var$SvgComponent);
var $51a99a1996ad0f9a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($51a99a1996ad0f9a$var$ForwardRef);


export {$51a99a1996ad0f9a$export$2e2bcd8739ae039 as default};
