// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileZip_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $4dk24$jsx} from "react/jsx-runtime";
import {forwardRef as $4dk24$forwardRef} from "react";





const $ed872d435c3b3406$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4dk24$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4dk24$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M29.3 15.75h18.199q.1-.01.2-.01c2.305 0 4.521.888 6.185 2.487l.002.001 19.006 18.187c1.778 1.7 2.778 4.064 2.778 6.515v27.8c0 4.957-4.008 8.99-8.96 8.99H29.3c-4.952 0-8.96-4.033-8.96-8.99V24.74c0-4.957 4.008-8.99 8.96-8.99m18.4 4q.086 0 .17-.007c1.213.042 2.369.528 3.244 1.37l.003.002 19.01 18.19c.983.94 1.543 2.257 1.543 3.625v27.8c0 2.763-2.232 4.99-4.96 4.99H29.3c-2.728 0-4.96-2.227-4.96-4.99V24.74c0-2.763 2.232-4.99 4.96-4.99zm-2.04 46.72v-5.9h4.09v5.9zm-1.72-13.9c-1.31 0-2.28 1.072-2.28 2.28v11.62c0 2.221 1.802 4 4 4h4.09c2.221 0 4-1.802 4-4V54.85c0-1.31-1.072-2.28-2.28-2.28h-1.78v-8.68h3.23c1.104 0 2-.895 2-2s-.896-2-2-2h-3.23v-4.45h3.23c1.104 0 2-.895 2-2s-.896-2-2-2h-3.23v-2.22c0-1.104-.896-2-2-2h-4.6c-1.105 0-2 .896-2 2s.895 2 2 2h2.6v4.44h-2.6c-1.105 0-2 .896-2 2s.895 2 2 2h2.6v4.45h-2.6c-1.105 0-2 .896-2 2s.895 2 2 2h2.6v4.46z"
        })
    });
const $ed872d435c3b3406$var$ForwardRef = /*#__PURE__*/ (0, $4dk24$forwardRef)($ed872d435c3b3406$var$SvgComponent);
var $ed872d435c3b3406$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ed872d435c3b3406$var$ForwardRef);


export {$ed872d435c3b3406$export$2e2bcd8739ae039 as default};
