// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileImage_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $10pZ5$reactjsxruntime = require("react/jsx-runtime");
var $10pZ5$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9197ef47300de4d6$export$2e2bcd8739ae039);




const $9197ef47300de4d6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $10pZ5$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $10pZ5$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M15.19 7.834c-2.605 0-4.74 2.092-4.74 4.702V35.55c0 2.61 2.135 4.702 4.74 4.702h18.62c2.606 0 4.742-2.092 4.742-4.702V22.02c0-1.25-.503-2.448-1.394-3.33l-9.592-9.484c-.89-.88-2.095-1.372-3.348-1.372zm-2.24 4.702c0-1.203.99-2.202 2.24-2.202h9.028c.6 0 1.17.235 1.59.65l9.593 9.485c.418.414.651.972.651 1.552V35.55c0 1.203-.99 2.202-2.241 2.202H15.19c-1.252 0-2.242-1-2.242-2.202zm16.119 12.226c-1.271 0-2.302-1.03-2.302-2.301s1.03-2.302 2.302-2.302c1.27 0 2.301 1.03 2.301 2.302 0 1.27-1.03 2.301-2.301 2.301m-9.294.598c.758-1.313 2.653-1.313 3.41 0l2.72 4.71 1.35-2.337c.757-1.313 2.652-1.313 3.41 0l2.865 4.962c.758 1.313-.19 2.954-1.705 2.954h-14.58c-1.516 0-2.463-1.64-1.706-2.954zm4.69 7.216-2.985-5.17-3.316 5.743h5.976zm6.44.573-1.945-3.37-1.946 3.37z"
        })
    });
const $9197ef47300de4d6$var$ForwardRef = /*#__PURE__*/ (0, $10pZ5$react.forwardRef)($9197ef47300de4d6$var$SvgComponent);
var $9197ef47300de4d6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9197ef47300de4d6$var$ForwardRef);
