// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lightbulbRays_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $a9d3y$reactjsxruntime = require("react/jsx-runtime");
var $a9d3y$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0843ea9b26d9c505$export$2e2bcd8739ae039);




const $0843ea9b26d9c505$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $a9d3y$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $a9d3y$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M23.516 6c.69 0 1.25.56 1.25 1.25v1.779c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25V7.25c0-.69.56-1.25 1.25-1.25m.955 9.92c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25c1.657 0 2.864 1.244 2.864 2.615 0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25c0-2.896-2.475-5.115-5.364-5.115m-3.566 20.377v1.637h5.196v-1.637zm5.196-2.5V32.16c0-1.642.968-2.967 2.043-3.806 1.816-1.418 2.989-3.648 2.989-6.162 0-4.305-3.433-7.763-7.63-7.763-4.196 0-7.63 3.458-7.63 7.763 0 2.514 1.173 4.744 2.99 6.162 1.074.84 2.042 2.164 2.042 3.806v1.636zm2.5 1.243v-2.88c0-.613.376-1.284 1.081-1.834 2.404-1.878 3.951-4.825 3.951-8.133 0-5.65-4.518-10.263-10.13-10.263s-10.13 4.613-10.13 10.263c0 3.308 1.547 6.255 3.95 8.133.706.55 1.082 1.22 1.082 1.835V38c0 1.346 1.092 2.433 2.434 2.433h5.328c1.342 0 2.434-1.087 2.434-2.432V35.04m-21.148-13c0-.691.56-1.25 1.25-1.25h1.78c.69 0 1.25.559 1.25 1.25 0 .69-.56 1.25-1.25 1.25h-1.78c-.69 0-1.25-.56-1.25-1.25m29.095-1.25c-.69 0-1.25.559-1.25 1.25 0 .69.56 1.25 1.25 1.25h1.779c.69 0 1.25-.56 1.25-1.25 0-.691-.56-1.25-1.25-1.25zm-1.674-8.323c.488-.488.489-1.28.001-1.768-.487-.489-1.279-.49-1.767-.002l-1.261 1.26c-.49.487-.49 1.278-.002 1.767s1.28.49 1.768.001zm-22.719-1.768c.488-.489 1.28-.49 1.768-.002l1.26 1.26c.49.487.49 1.278.002 1.767-.487.488-1.279.49-1.767.001l-1.261-1.258c-.489-.488-.49-1.28-.002-1.768"
        })
    });
const $0843ea9b26d9c505$var$ForwardRef = /*#__PURE__*/ (0, $a9d3y$react.forwardRef)($0843ea9b26d9c505$var$SvgComponent);
var $0843ea9b26d9c505$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0843ea9b26d9c505$var$ForwardRef);
