// @react-spectrum/s2/illustrations/linear/internal/S2_lin_microphone_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $18m6u$reactjsxruntime = require("react/jsx-runtime");
var $18m6u$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $332d8af5a045e8a4$export$2e2bcd8739ae039);




const $332d8af5a045e8a4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $18m6u$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $18m6u$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M79 20.8c-12.812 0-23.2 10.388-23.2 23.2v28.6c0 12.814 10.388 23.2 23.2 23.2h2.6c12.814 0 23.201-10.386 23.201-23.2V44c0-12.812-10.387-23.2-23.2-23.2zM63.8 44c0-8.394 6.806-15.2 15.2-15.2h2.6c8.396 0 15.2 6.806 15.2 15.2v28.6c0 8.396-6.804 15.2-15.2 15.2H79c-8.394 0-15.2-6.804-15.2-15.2zm12 71.576c-22.462-2.256-40-21.217-40-44.275v-4.5c0-2.21 1.792-4 4-4 2.21 0 4 1.79 4 4v4.5c0 20.158 16.342 36.5 36.5 36.5s36.501-16.342 36.501-36.5v-4.5c0-2.21 1.791-4 4-4s4 1.79 4 4v4.5c0 23.399-18.06 42.579-41 44.364v12.136h20c2.209 0 4 1.791 4 4s-1.791 4-4 4h-47c-2.21 0-4-1.791-4-4s1.79-4 4-4h19z"
        })
    });
const $332d8af5a045e8a4$var$ForwardRef = /*#__PURE__*/ (0, $18m6u$react.forwardRef)($332d8af5a045e8a4$var$SvgComponent);
var $332d8af5a045e8a4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($332d8af5a045e8a4$var$ForwardRef);
