// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noImage_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2ToO5$reactjsxruntime = require("react/jsx-runtime");
var $2ToO5$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c404da5ff37fabe5$export$2e2bcd8739ae039);




const $c404da5ff37fabe5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2ToO5$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2ToO5$reactjsxruntime.jsx)("defs", {
                children: /*#__PURE__*/ (0, $2ToO5$reactjsxruntime.jsx)("clipPath", {
                    id: "c9b2ce__a",
                    children: /*#__PURE__*/ (0, $2ToO5$reactjsxruntime.jsx)("path", {
                        fill: "#fff",
                        d: "M160 0H0v160h160z"
                    })
                })
            }),
            /*#__PURE__*/ (0, $2ToO5$reactjsxruntime.jsxs)("g", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                clipPath: "url(#c9b2ce__a)",
                children: [
                    /*#__PURE__*/ (0, $2ToO5$reactjsxruntime.jsx)("path", {
                        d: "M132.49 93.09c-13.53 0-24.5 10.97-24.5 24.5s10.97 24.5 24.5 24.5 24.5-10.97 24.5-24.5-10.97-24.5-24.5-24.5m10.92 29.77c1.56 1.56 1.56 4.09 0 5.66-.78.78-1.8 1.17-2.83 1.17s-2.05-.39-2.83-1.17l-5.26-5.26-5.26 5.26c-.78.78-1.8 1.17-2.83 1.17s-2.05-.39-2.83-1.17c-1.56-1.56-1.56-4.09 0-5.66l5.26-5.26-5.26-5.26c-1.56-1.56-1.56-4.09 0-5.66 1.56-1.56 4.1-1.56 5.66 0l5.26 5.26 5.26-5.26c1.56-1.56 4.1-1.56 5.66 0s1.56 4.09 0 5.66l-5.26 5.26z"
                    }),
                    /*#__PURE__*/ (0, $2ToO5$reactjsxruntime.jsx)("path", {
                        fillRule: "evenodd",
                        d: "M124 30c11.046 0 20 8.954 20 20v32.875c0 2.21-1.791 4-4 4s-4-1.79-4-4V50c0-6.627-5.373-12-12-12H36c-6.627 0-12 5.373-12 12v60c0 .862.09 1.703.264 2.513l24.781-38.286c4.365-6.75 14.252-6.75 18.618-.002l.001.002 20.529 31.71 11.745-17.491v-.002c3.292-4.895 9.488-6.105 14.216-3.719 1.972.995 2.764 3.401 1.769 5.373s-3.401 2.764-5.373 1.77c-1.348-.681-3.067-.306-3.972 1.039L87.043 122H98c2.209 0 4 1.791 4 4s-1.791 4-4 4H36c-11.046 0-20-8.954-20-20V50c0-11.046 8.954-20 20-20zM55.762 78.573l-26.686 41.229C31.032 121.186 33.42 122 36 122h41.407l5.934-8.837-22.393-34.59-.002-.002c-1.213-1.876-3.97-1.876-5.183 0zM108 58c0 4.418 3.582 8 8 8s8-3.582 8-8-3.582-8-8-8-8 3.582-8 8"
                    })
                ]
            })
        ]
    });
const $c404da5ff37fabe5$var$ForwardRef = /*#__PURE__*/ (0, $2ToO5$react.forwardRef)($c404da5ff37fabe5$var$SvgComponent);
var $c404da5ff37fabe5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c404da5ff37fabe5$var$ForwardRef);
