// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noImage_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $9uWMR$reactjsxruntime = require("react/jsx-runtime");
var $9uWMR$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7e3a08514682c624$export$2e2bcd8739ae039);




const $7e3a08514682c624$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9uWMR$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9uWMR$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M38 8.75c2.9 0 5.25 2.35 5.25 5.25v11.563c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25V14c0-1.519-1.231-2.75-2.75-2.75H10c-1.519 0-2.75 1.231-2.75 2.75v19.97l7.722-11.93c1.33-2.055 4.34-2.055 5.669 0l6.144 9.49 3.45-5.217.005-.008c1.17-1.739 3.545-1.956 5.022-.665.52.454.572 1.244.118 1.763-.454.52-1.244.573-1.764.119-.38-.333-.997-.27-1.299.175L26.33 36.75h3.264c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H10c-2.9 0-5.25-2.35-5.25-5.25V14c0-2.9 2.35-5.25 5.25-5.25zM17.07 23.399l-8.414 13c.397.224.856.351 1.344.351h13.354q.043-.097.104-.19l1.822-2.754-6.738-10.407c-.344-.533-1.127-.533-1.471 0M32 17.5c0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5-1.12-2.5-2.5-2.5-2.5 1.12-2.5 2.5"
            }),
            /*#__PURE__*/ (0, $9uWMR$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M39.516 27.984c-4.142 0-7.5 3.359-7.5 7.5 0 4.142 3.358 7.5 7.5 7.5s7.5-3.358 7.5-7.5-3.359-7.5-7.5-7.5m3.358 9.105c.486.486.486 1.269 0 1.754-.238.238-.555.367-.872.367s-.634-.12-.871-.367l-1.605-1.605-1.606 1.605c-.237.238-.554.367-.871.367s-.634-.12-.872-.367c-.486-.485-.486-1.268 0-1.754l1.605-1.605-1.605-1.605c-.486-.485-.486-1.268 0-1.753.485-.486 1.268-.486 1.753 0l1.605 1.605 1.606-1.605c.485-.486 1.268-.486 1.753 0 .486.485.486 1.268 0 1.753l-1.605 1.605 1.605 1.605z"
            })
        ]
    });
const $7e3a08514682c624$var$ForwardRef = /*#__PURE__*/ (0, $9uWMR$react.forwardRef)($7e3a08514682c624$var$SvgComponent);
var $7e3a08514682c624$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7e3a08514682c624$var$ForwardRef);
