// @react-spectrum/s2/illustrations/linear/internal/S2_lin_warning_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $9QrFf$reactjsxruntime = require("react/jsx-runtime");
var $9QrFf$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ca38bd842fc9c673$export$2e2bcd8739ae039);




const $ca38bd842fc9c673$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9QrFf$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9QrFf$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M41.236 16.603c3.172-5.085 10.675-5.003 13.706.247l29.38 50.89c3.081 5.335-.774 12-6.932 12H18.62c-6.157 0-10.013-6.665-6.932-12l29.38-50.89q.076-.131.168-.247m10.242 2.247c-1.54-2.667-5.386-2.667-6.926 0q-.066.115-.146.219l-29.254 50.67v.001c-1.539 2.665.386 6 3.468 6h58.77c3.083 0 5.007-3.335 3.468-6zM48 30.76c1.105 0 2 .895 2 2v22.21c0 1.105-.895 2-2 2s-2-.895-2-2V32.76c0-1.104.896-2 2-2m3 34.48c0 1.657-1.343 3-3 3s-3-1.343-3-3 1.343-3 3-3 3 1.343 3 3"
        })
    });
const $ca38bd842fc9c673$var$ForwardRef = /*#__PURE__*/ (0, $9QrFf$react.forwardRef)($ca38bd842fc9c673$var$SvgComponent);
var $ca38bd842fc9c673$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ca38bd842fc9c673$var$ForwardRef);
