// @react-spectrum/s2/illustrations/linear/internal/S2_lin_transform_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $kgy9u$reactjsxruntime = require("react/jsx-runtime");
var $kgy9u$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $341f465a759f4fa0$export$2e2bcd8739ae039);




const $341f465a759f4fa0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kgy9u$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kgy9u$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M18.238 16.754c-1.104 0-2 .895-2 2V32.01c0 1.104.896 2 2 2s2-.896 2-2v-8.523l24.933 24.932L20.24 73.352v-8.375c0-1.104-.896-2-2-2s-2 .896-2 2v13.256c0 1.105.895 2 2 2h13.298c1.105 0 2-.895 2-2s-.895-2-2-2h-8.523L48 51.248l24.985 24.985h-8.523c-1.105 0-2 .895-2 2s.895 2 2 2H77.76c1.105 0 2-.895 2-2V64.977c0-1.104-.895-2-2-2s-2 .896-2 2v8.375L50.828 48.419 75.76 23.487v8.523c0 1.104.895 2 2 2s2-.896 2-2V18.754c0-1.105-.896-2-2-2H64.462c-1.104 0-2 .895-2 2s.896 2 2 2h8.374L48 45.59 23.162 20.754h8.374c1.104 0 2-.896 2-2s-.896-2-2-2z"
        })
    });
const $341f465a759f4fa0$var$ForwardRef = /*#__PURE__*/ (0, $kgy9u$react.forwardRef)($341f465a759f4fa0$var$SvgComponent);
var $341f465a759f4fa0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($341f465a759f4fa0$var$ForwardRef);
