// @react-spectrum/s2/illustrations/linear/internal/S2_lin_redo_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $kruRc$jsx} from "react/jsx-runtime";
import {forwardRef as $kruRc$forwardRef} from "react";





const $2d4fe427756c4831$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kruRc$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kruRc$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m69.414 48.414 14-14c.781-.78.781-2.047 0-2.828l-14-14c-.78-.781-2.047-.781-2.828 0-.781.78-.781 2.047 0 2.828L77.202 31.03h-39.8c-13.537 0-24.511 10.974-24.511 24.512s10.974 24.512 24.511 24.512h15.95c1.104 0 2-.896 2-2s-.896-2-2-2h-15.95c-11.328 0-20.511-9.184-20.511-20.512S26.074 35.03 37.402 35.03h39.74L66.585 45.586c-.781.78-.781 2.047 0 2.828.78.781 2.047.781 2.828 0"
        })
    });
const $2d4fe427756c4831$var$ForwardRef = /*#__PURE__*/ (0, $kruRc$forwardRef)($2d4fe427756c4831$var$SvgComponent);
var $2d4fe427756c4831$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2d4fe427756c4831$var$ForwardRef);


export {$2d4fe427756c4831$export$2e2bcd8739ae039 as default};
