// @react-spectrum/s2/illustrations/linear/internal/S2_lin_folderClose_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dkrWF$reactjsxruntime = require("react/jsx-runtime");
var $dkrWF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $856743f72953a682$export$2e2bcd8739ae039);




const $856743f72953a682$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dkrWF$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dkrWF$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 37.2c0-2.84 2.331-5.2 5.28-5.2h25.7c1.617 0 3.136.73 4.133 1.965L69.633 47H24zM16 47v-9.8C16 29.877 21.978 24 29.28 24h25.7c4.02 0 7.834 1.812 10.36 4.94l-3.043 2.456 3.042-2.455L79.912 47h46.242C136.031 47 144 55.063 144 64.941v53.118c0 9.877-7.969 17.941-17.846 17.941H33.846C23.97 136 16 127.936 16 118.059V47m8 8v63.059c0 5.511 4.44 9.941 9.846 9.941h92.308c5.407 0 9.846-4.43 9.846-9.941V64.941c0-5.51-4.439-9.941-9.846-9.941H24"
        })
    });
const $856743f72953a682$var$ForwardRef = /*#__PURE__*/ (0, $dkrWF$react.forwardRef)($856743f72953a682$var$SvgComponent);
var $856743f72953a682$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($856743f72953a682$var$ForwardRef);
