// @react-spectrum/s2/illustrations/linear/internal/S2_lin_vectorDraw_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6RNsy$reactjsxruntime = require("react/jsx-runtime");
var $6RNsy$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $00f4c8b43b37dca1$export$2e2bcd8739ae039);




const $00f4c8b43b37dca1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6RNsy$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6RNsy$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M112.433 24.117c-5.46-5.46-14.195-4.396-19.464.873l-7.71 7.71-38.096 16.262c-4.145 1.77-7.399 5.346-8.73 9.645l-17.57 56.765c-4.516 14.591 9.166 28.273 23.758 23.757l56.764-17.57c4.299-1.331 7.876-4.585 9.645-8.729l16.262-38.097 7.71-7.71c5.269-5.269 6.333-14.004.873-19.463zM50.303 56.32l36.321-15.504 32.552 32.552-15.504 36.321c-.858 2.01-2.626 3.6-4.652 4.227l-56.765 17.571c-2.61.808-5.17.609-7.384-.3l34.474-34.334c.605.147 1.237.224 1.887.224 4.419 0 8-3.581 8-8 0-4.418-3.581-8-8-8-4.418 0-8 3.582-8 8 0 .783.113 1.539.322 2.253l-34.497 34.358c-1.12-2.339-1.43-5.115-.551-7.951l17.57-56.765c.627-2.026 2.217-3.794 4.227-4.652m79.042 5.046-5.428 5.429-30.72-30.72 5.43-5.428c2.838-2.839 6.465-2.556 8.149-.873l23.442 23.442c1.684 1.684 1.966 5.312-.873 8.15"
        })
    });
const $00f4c8b43b37dca1$var$ForwardRef = /*#__PURE__*/ (0, $6RNsy$react.forwardRef)($00f4c8b43b37dca1$var$SvgComponent);
var $00f4c8b43b37dca1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($00f4c8b43b37dca1$var$ForwardRef);
