// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browser_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8lT1i$jsx} from "react/jsx-runtime";
import {forwardRef as $8lT1i$forwardRef} from "react";





const $f4a9e06ce5f374a9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8lT1i$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8lT1i$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M136 44c0-6.627-5.373-12-12-12H36c-6.627 0-12 5.373-12 12v12h112zm0 20v52c0 6.627-5.373 12-12 12H36c-6.627 0-12-5.373-12-12V64zm-12-40c11.046 0 20 8.954 20 20v72c0 11.046-8.954 20-20 20H36c-11.046 0-20-8.954-20-20V44c0-11.046 8.954-20 20-20zM40.98 49c2.487 0 4.5-2.014 4.5-4.5S43.468 40 40.993 40h-.011c-2.487 0-4.5 2.014-4.5 4.5s2.013 4.5 4.5 4.5"
        })
    });
const $f4a9e06ce5f374a9$var$ForwardRef = /*#__PURE__*/ (0, $8lT1i$forwardRef)($f4a9e06ce5f374a9$var$SvgComponent);
var $f4a9e06ce5f374a9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f4a9e06ce5f374a9$var$ForwardRef);


export {$f4a9e06ce5f374a9$export$2e2bcd8739ae039 as default};
