// @react-spectrum/s2/illustrations/linear/internal/S2_lin_brand_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $374me$jsx} from "react/jsx-runtime";
import {forwardRef as $374me$forwardRef} from "react";





const $16dbda812461eec5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $374me$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $374me$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M27.968 19.95c-2.767 0-5.019 2.25-5.019 5.018v29.03c0 13.828 11.23 25.052 25.053 25.052 13.828 0 25.052-11.23 25.052-25.052v-29.03c0-2.767-2.252-5.019-5.02-5.019zm-9.019 5.018c0-4.977 4.042-9.019 9.02-9.019h40.066c4.977 0 9.019 4.042 9.019 9.02v29.029c0 16.03-13.015 29.052-29.053 29.052-16.03 0-29.052-13.015-29.052-29.052zm31.01 12.167c1.67-2.896 5.847-2.896 7.518 0l-1.732 1 1.732-1 7.352 12.733c1.668 2.892-.41 6.51-3.756 6.51H46.37c-3.345 0-5.425-3.618-3.757-6.51zl1.732 1zm4.053 1.999c-.13-.228-.457-.228-.588 0l-7.347 12.733c-.133.232.033.51.292.51h14.704c.258 0 .425-.278.292-.51zm-12.578 2.1c.552-.956.224-2.18-.732-2.732-.957-.552-2.18-.224-2.732.732l-8.172 14.15c-.552.956-.224 2.179.732 2.731.957.553 2.18.225 2.732-.731z"
        })
    });
const $16dbda812461eec5$var$ForwardRef = /*#__PURE__*/ (0, $374me$forwardRef)($16dbda812461eec5$var$SvgComponent);
var $16dbda812461eec5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($16dbda812461eec5$var$ForwardRef);


export {$16dbda812461eec5$export$2e2bcd8739ae039 as default};
