// @react-spectrum/s2/illustrations/linear/internal/S2_lin_search_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $44EDY$reactjsxruntime = require("react/jsx-runtime");
var $44EDY$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $8ec9c5eeb6759066$export$2e2bcd8739ae039);




const $8ec9c5eeb6759066$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $44EDY$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $44EDY$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M61.501 18.077c-25.129 0-45.5 20.37-45.5 45.5s20.371 45.5 45.5 45.5c11.124 0 21.316-3.993 29.221-10.622l3.983 3.982-2.302 2.302c-1.562 1.562-1.562 4.095 0 5.657l29.507 29.507c4.372 4.372 11.46 4.372 15.832 0s4.372-11.46 0-15.832l-29.507-29.507c-1.562-1.562-4.095-1.562-5.657 0l-2.216 2.216-3.983-3.982c6.63-7.905 10.622-18.097 10.622-29.221 0-25.13-20.371-45.5-45.5-45.5m-37.5 45.5c0-20.711 16.79-37.5 37.5-37.5s37.5 16.789 37.5 37.5-16.79 37.5-37.5 37.5-37.5-16.79-37.5-37.5m81.405 39.472-4.518 4.519 26.679 26.678c1.248 1.248 3.27 1.248 4.518 0s1.248-3.27 0-4.518z"
        })
    });
const $8ec9c5eeb6759066$var$ForwardRef = /*#__PURE__*/ (0, $44EDY$react.forwardRef)($8ec9c5eeb6759066$var$SvgComponent);
var $8ec9c5eeb6759066$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($8ec9c5eeb6759066$var$ForwardRef);
