// @react-spectrum/s2/illustrations/linear/internal/S2_lin_volume_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7mBNq$jsx} from "react/jsx-runtime";
import {forwardRef as $7mBNq$forwardRef} from "react";





const $b5568cf6c3a2e71b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7mBNq$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7mBNq$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25.967 10.73c0-2.444-2.95-3.674-4.686-1.953l-5.831 5.781c-.14.14-.33.218-.528.218h-4.19c-1.795 0-3.25 1.455-3.25 3.25v12.01c0 1.796 1.455 3.25 3.25 3.25h4.196c.196 0 .385.078.525.215l5.838 5.73c1.74 1.708 4.676.475 4.676-1.962zm-2.926-.177c.052-.052.095-.067.128-.073.04-.007.091-.003.144.02.053.021.091.055.115.088.02.028.039.07.039.142V37.27c0 .072-.02.114-.039.142-.023.033-.062.066-.114.088s-.104.026-.144.02c-.033-.006-.077-.021-.128-.072l-5.778-5.671V16.28zm-8.277 6.723v13.51h-4.032c-.414 0-.75-.335-.75-.75v-12.01c0-.415.336-.75.75-.75zm14.992-.288c-.552-.415-1.336-.304-1.75.247-.416.552-.305 1.336.247 1.75 1.514 1.14 2.49 2.949 2.49 4.985 0 2.054-.992 3.875-2.528 5.013-.554.41-.67 1.194-.26 1.748.411.555 1.194.671 1.749.26 2.144-1.589 3.538-4.142 3.538-7.02 0-2.856-1.371-5.391-3.486-6.983m5.002-4.564c-.552-.415-1.335-.305-1.75.247s-.305 1.335.247 1.75c2.895 2.179 4.762 5.64 4.762 9.537 0 3.929-1.898 7.414-4.834 9.59-.554.41-.67 1.193-.26 1.748s1.194.67 1.749.26c3.544-2.626 5.846-6.844 5.846-11.598 0-4.716-2.265-8.904-5.76-11.534"
        })
    });
const $b5568cf6c3a2e71b$var$ForwardRef = /*#__PURE__*/ (0, $7mBNq$forwardRef)($b5568cf6c3a2e71b$var$SvgComponent);
var $b5568cf6c3a2e71b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b5568cf6c3a2e71b$var$ForwardRef);


export {$b5568cf6c3a2e71b$export$2e2bcd8739ae039 as default};
