// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileText_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6zftE$jsx} from "react/jsx-runtime";
import {forwardRef as $6zftE$forwardRef} from "react";





const $cbf65656593231b7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6zftE$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6zftE$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M30.09 16C25.117 16 21 19.95 21 24.923v46.154C21 76.05 25.116 80 30.09 80h37.82c4.974 0 9.09-3.95 9.09-8.923V43.945c0-2.383-.97-4.657-2.68-6.327L54.838 18.597C53.129 16.929 50.822 16 48.426 16zM25 24.923C25 22.25 27.234 20 30.09 20h18.336c1.366 0 2.666.53 3.617 1.459L71.526 40.48c.95.927 1.474 2.175 1.474 3.465v27.132C73 73.75 70.766 76 67.91 76H30.09C27.235 76 25 73.751 25 71.077zm10.879 19.013c.319-.216.703-.341 1.117-.341h23.997c1.105 0 2 .895 2 2v4.8c0 1.104-.895 2-2 2s-2-.896-2-2v-2.8h-7.998v19.997h2.8c1.104 0 2 .895 2 2s-.896 2-2 2h-9.6c-1.104 0-2-.896-2-2s.896-2 2-2h2.8V47.595h-7.999v2.8c0 1.104-.895 2-2 2s-2-.896-2-2v-4.8c0-.276.056-.54.157-.779.152-.36.406-.665.726-.88"
        })
    });
const $cbf65656593231b7$var$ForwardRef = /*#__PURE__*/ (0, $6zftE$forwardRef)($cbf65656593231b7$var$SvgComponent);
var $cbf65656593231b7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cbf65656593231b7$var$ForwardRef);


export {$cbf65656593231b7$export$2e2bcd8739ae039 as default};
