// @react-spectrum/s2/illustrations/linear/internal/S2_lin_laptop_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $avlhi$jsx} from "react/jsx-runtime";
import {forwardRef as $avlhi$forwardRef} from "react";





const $6e6494ca9d7d5e9f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $avlhi$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $avlhi$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M15 12.25c-1.52 0-2.75 1.23-2.75 2.75v15.75h23.5V15c0-1.52-1.23-2.75-2.75-2.75zM9.75 15v15.75H8.5c-2.071 0-3.75 1.679-3.75 3.75s1.679 3.75 3.75 3.75h31c2.071 0 3.75-1.679 3.75-3.75s-1.679-3.75-3.75-3.75h-1.25V15c0-2.9-2.35-5.25-5.25-5.25H15c-2.9 0-5.25 2.35-5.25 5.25M8.5 33.25c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h31c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25zm17-17.75c0 .828-.672 1.5-1.5 1.5s-1.5-.672-1.5-1.5.672-1.5 1.5-1.5 1.5.672 1.5 1.5"
        })
    });
const $6e6494ca9d7d5e9f$var$ForwardRef = /*#__PURE__*/ (0, $avlhi$forwardRef)($6e6494ca9d7d5e9f$var$SvgComponent);
var $6e6494ca9d7d5e9f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6e6494ca9d7d5e9f$var$ForwardRef);


export {$6e6494ca9d7d5e9f$export$2e2bcd8739ae039 as default};
