// @react-spectrum/s2/illustrations/linear/internal/S2_lin_trash_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $d7oQx$reactjsxruntime = require("react/jsx-runtime");
var $d7oQx$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9747368e189da2a7$export$2e2bcd8739ae039);




const $9747368e189da2a7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $d7oQx$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $d7oQx$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M22.5 18c0-.69-.56-1.25-1.25-1.25S20 17.31 20 18v16c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25zM27.25 16.75c.69 0 1.25.56 1.25 1.25v16c0 .69-.56 1.25-1.25 1.25S26 34.69 26 34V18c0-.69.56-1.25 1.25-1.25"
            }),
            /*#__PURE__*/ (0, $d7oQx$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M17 12.75v-1.985c0-1.688 1.389-3.015 3.05-3.015h8.4c1.661 0 3.05 1.327 3.05 3.015v1.985h7.75c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-.886l-2.41 20.86c-.278 2.396-2.364 4.14-4.766 4.14H17.312c-2.402 0-4.489-1.744-4.765-4.14l-2.41-20.86H9.25C8.56 15.25 8 14.69 8 14s.56-1.25 1.25-1.25zm2.5-1.985c0-.261.223-.515.55-.515h8.4c.327 0 .55.254.55.515v1.985h-9.5zm-6.847 4.485h23.194L33.47 35.824c-.122 1.058-1.072 1.926-2.282 1.926H17.312c-1.21 0-2.16-.868-2.282-1.926z"
            })
        ]
    });
const $9747368e189da2a7$var$ForwardRef = /*#__PURE__*/ (0, $d7oQx$react.forwardRef)($9747368e189da2a7$var$SvgComponent);
var $9747368e189da2a7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9747368e189da2a7$var$ForwardRef);
