// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browser_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2bIaF$reactjsxruntime = require("react/jsx-runtime");
var $2bIaF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f91eb0d2d06fb229$export$2e2bcd8739ae039);




const $f91eb0d2d06fb229$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2bIaF$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2bIaF$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M136 44c0-6.627-5.373-12-12-12H36c-6.627 0-12 5.373-12 12v12h112zm0 20v52c0 6.627-5.373 12-12 12H36c-6.627 0-12-5.373-12-12V64zm-12-40c11.046 0 20 8.954 20 20v72c0 11.046-8.954 20-20 20H36c-11.046 0-20-8.954-20-20V44c0-11.046 8.954-20 20-20zM40.98 49c2.487 0 4.5-2.014 4.5-4.5S43.468 40 40.993 40h-.011c-2.487 0-4.5 2.014-4.5 4.5s2.013 4.5 4.5 4.5"
        })
    });
const $f91eb0d2d06fb229$var$ForwardRef = /*#__PURE__*/ (0, $2bIaF$react.forwardRef)($f91eb0d2d06fb229$var$SvgComponent);
var $f91eb0d2d06fb229$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f91eb0d2d06fb229$var$ForwardRef);
