// @react-spectrum/s2/illustrations/linear/internal/S2_lin_infiniteLooping_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $kT1TJ$reactjsxruntime = require("react/jsx-runtime");
var $kT1TJ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $37bc910fb7699e65$export$2e2bcd8739ae039);




const $37bc910fb7699e65$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kT1TJ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kT1TJ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M72.328 72.502c2.21-.005 3.996-1.8 3.991-4.009l-.057-24.865c-.005-2.21-1.8-3.996-4.01-3.991s-3.995 1.8-3.99 4.009l.034 15.114c-6.64-6.98-16.023-11.337-26.426-11.337-20.141 0-36.47 16.328-36.47 36.47s16.328 36.47 36.47 36.47c8.355 0 16.065-2.814 22.216-7.545q.14-.107.267-.224l.12-.091c4.531-3.557 8.748-8.72 12.042-13.442 3.31-4.745 5.903-9.354 7.118-12.036.945-2.084 3.285-7.04 6.38-12.245 2.665-4.483 5.7-8.819 8.703-11.737q.14-.106.272-.227c5.059-4.596 11.768-7.393 19.14-7.393 15.723 0 28.47 12.747 28.47 28.47 0 15.724-12.747 28.47-28.47 28.47-9.615 0-18.118-4.762-23.28-12.075-1.273-1.805-3.769-2.235-5.574-.962-1.805 1.274-2.235 3.77-.961 5.575 6.595 9.345 17.491 15.462 29.815 15.462 20.141 0 36.47-16.328 36.47-36.47s-16.329-36.47-36.47-36.47c-8.343 0-16.038 2.806-22.182 7.52q-.267.15-.514.345c-4.756 3.75-9.057 9.957-12.295 15.403-3.305 5.556-5.781 10.806-6.79 13.032-.968 2.136-3.293 6.317-6.392 10.76-3.115 4.464-6.801 8.885-10.422 11.727q-.122.096-.235.2l-.09.067c-4.801 3.693-10.808 5.886-17.338 5.886-15.723 0-28.47-12.746-28.47-28.47s12.747-28.47 28.47-28.47c8.24 0 15.664 3.498 20.867 9.101l-15.293.036c-2.209.005-3.996 1.8-3.99 4.009.004 2.209 1.8 3.996 4.008 3.99z"
        })
    });
const $37bc910fb7699e65$var$ForwardRef = /*#__PURE__*/ (0, $kT1TJ$react.forwardRef)($37bc910fb7699e65$var$SvgComponent);
var $37bc910fb7699e65$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($37bc910fb7699e65$var$ForwardRef);
