// @react-spectrum/s2/illustrations/linear/internal/S2_lin_microphone_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $631Ik$reactjsxruntime = require("react/jsx-runtime");
var $631Ik$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3cc6017e2f8bfbd5$export$2e2bcd8739ae039);




const $3cc6017e2f8bfbd5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $631Ik$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $631Ik$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 12c-7.732 0-14 6.268-14 14v16c0 7.732 6.268 14 14 14s14-6.268 14-14V26c0-7.732-6.268-14-14-14M38 26c0-5.523 4.477-10 10-10s10 4.477 10 10v16c0 5.523-4.477 10-10 10s-10-4.477-10-10zm8 41.924C32.575 66.903 22 55.686 22 42v-3c0-1.105.895-2 2-2s2 .895 2 2v3c0 12.15 9.85 22 22 22s22-9.85 22-22v-3c0-1.105.895-2 2-2s2 .895 2 2v3c0 13.687-10.575 24.903-24 25.924V76h12c1.105 0 2 .895 2 2s-.895 2-2 2H34c-1.105 0-2-.895-2-2s.895-2 2-2h12z"
        })
    });
const $3cc6017e2f8bfbd5$var$ForwardRef = /*#__PURE__*/ (0, $631Ik$react.forwardRef)($3cc6017e2f8bfbd5$var$SvgComponent);
var $3cc6017e2f8bfbd5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3cc6017e2f8bfbd5$var$ForwardRef);
