// @react-spectrum/s2/illustrations/linear/internal/S2_lin_rotateCCW_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $hvtuF$reactjsxruntime = require("react/jsx-runtime");
var $hvtuF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b8e1b7cb64c81b15$export$2e2bcd8739ae039);




const $b8e1b7cb64c81b15$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hvtuF$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hvtuF$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M132 80c0-28.719-23.281-52-52-52-19.917 0-37.227 11.2-45.962 27.654l15.454-5.156c2.095-.699 4.361.433 5.06 2.529.7 2.095-.433 4.36-2.528 5.06l-24.275 8.1c-2.095.698-4.36-.433-5.06-2.53l-8.1-24.274c-.699-2.095.433-4.36 2.529-5.06s4.361.433 5.06 2.528l4.935 14.79C37.23 32.81 57.116 20 80 20c33.137 0 60 26.863 60 60s-26.863 60-60 60c-25.917 0-47.986-16.43-56.38-39.429-.757-2.075.311-4.371 2.387-5.129s4.371.311 5.128 2.387C38.416 117.772 57.55 132 80 132c28.719 0 52-23.281 52-52"
        })
    });
const $b8e1b7cb64c81b15$var$ForwardRef = /*#__PURE__*/ (0, $hvtuF$react.forwardRef)($b8e1b7cb64c81b15$var$SvgComponent);
var $b8e1b7cb64c81b15$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b8e1b7cb64c81b15$var$ForwardRef);
