// @react-spectrum/s2/illustrations/linear/internal/S2_lin_dataAnalytics_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dt2tH$reactjsxruntime = require("react/jsx-runtime");
var $dt2tH$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $eb9d1a9137d6d4c4$export$2e2bcd8739ae039);




const $eb9d1a9137d6d4c4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dt2tH$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dt2tH$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M39.916 10.252c.428.541.337 1.327-.205 1.756l-11.548 9.136c-.387.306-.917.356-1.354.127l-7.21-3.768-9.726 8.25c-.526.447-1.315.382-1.762-.145-.446-.526-.382-1.315.145-1.762l10.368-8.793c.388-.33.936-.39 1.387-.155l7.237 3.782 10.912-8.633c.542-.428 1.328-.337 1.756.205M16.48 26.184c0-1.795 1.455-3.25 3.25-3.25s3.25 1.455 3.25 3.25v8.788c0 1.795-1.456 3.25-3.25 3.25s-3.25-1.455-3.25-3.25zm3.25-.75c-.415 0-.75.335-.75.75v8.788c0 .414.335.75.75.75.414 0 .75-.336.75-.75v-8.788c0-.415-.336-.75-.75-.75m8.603-.79c-1.795 0-3.25 1.456-3.25 3.25v7.078c0 1.795 1.455 3.25 3.25 3.25s3.25-1.455 3.25-3.25v-7.077c0-1.795-1.455-3.25-3.25-3.25m-.75 3.25c0-.414.336-.75.75-.75s.75.336.75.75v7.078c0 .414-.336.75-.75.75s-.75-.336-.75-.75zm6.104-6.02c0-1.796 1.455-3.25 3.25-3.25s3.25 1.454 3.25 3.25v13.099c0 1.795-1.456 3.25-3.25 3.25s-3.25-1.455-3.25-3.25zm3.25-.75c-.415 0-.75.335-.75.75v13.099c0 .414.335.75.75.75.414 0 .75-.336.75-.75v-13.1c0-.414-.336-.75-.75-.75m-25.811 6.91c-1.795 0-3.25 1.454-3.25 3.25v3.69c0 1.794 1.455 3.25 3.25 3.25s3.25-1.456 3.25-3.25v-3.69c0-1.796-1.455-3.25-3.25-3.25m-.75 3.25c0-.415.336-.75.75-.75s.75.335.75.75v3.69c0 .414-.336.75-.75.75s-.75-.336-.75-.75z"
        })
    });
const $eb9d1a9137d6d4c4$var$ForwardRef = /*#__PURE__*/ (0, $dt2tH$react.forwardRef)($eb9d1a9137d6d4c4$var$SvgComponent);
var $eb9d1a9137d6d4c4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($eb9d1a9137d6d4c4$var$ForwardRef);
