// @react-spectrum/s2/illustrations/linear/internal/S2_lin_userGroup_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9UAcx$jsx} from "react/jsx-runtime";
import {forwardRef as $9UAcx$forwardRef} from "react";





const $bd1fbb23a7964d55$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9UAcx$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9UAcx$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28.235 8c-3.019 0-5.598 2.63-5.598 6.045s2.58 6.044 5.598 6.044 5.598-2.63 5.598-6.044S31.253 8 28.235 8m-8.098 6.045c0-4.644 3.552-8.545 8.098-8.545 4.545 0 8.098 3.901 8.098 8.545 0 4.643-3.553 8.544-8.098 8.544s-8.098-3.9-8.098-8.544m-.96-5.988c.683-.102 1.155-.737 1.053-1.42-.1-.683-.737-1.155-1.42-1.053-3.992.592-6.961 4.208-6.961 8.46 0 4.268 2.99 7.895 7.003 8.468.683.097 1.317-.378 1.414-1.061.098-.684-.377-1.317-1.06-1.414-2.683-.383-4.857-2.866-4.857-5.992 0-3.116 2.159-5.593 4.828-5.988M16.43 26.675c.662-.195 1.04-.89.845-1.552s-.89-1.04-1.552-.846c-5.317 1.57-9.462 5.981-10.639 11.451-.484 2.253 1.2 4.425 3.548 4.425h1.639c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H8.63c-.713 0-1.26-.663-1.102-1.399.982-4.567 4.454-8.266 8.901-9.579m-.656 9.579c1.234-5.737 6.389-10.092 12.462-10.092s11.228 4.355 12.462 10.092c.158.736-.39 1.399-1.104 1.399H16.876c-.713 0-1.26-.663-1.102-1.399m12.462-12.592c-7.284 0-13.431 5.209-14.906 12.066-.485 2.253 1.2 4.425 3.547 4.425h22.718c2.347 0 4.032-2.172 3.547-4.425-1.475-6.857-7.623-12.066-14.906-12.066"
        })
    });
const $bd1fbb23a7964d55$var$ForwardRef = /*#__PURE__*/ (0, $9UAcx$forwardRef)($bd1fbb23a7964d55$var$SvgComponent);
var $bd1fbb23a7964d55$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($bd1fbb23a7964d55$var$ForwardRef);


export {$bd1fbb23a7964d55$export$2e2bcd8739ae039 as default};
