// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lighten_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7vndO$jsx} from "react/jsx-runtime";
import {forwardRef as $7vndO$forwardRef} from "react";





const $6eb81eb46f3bf6e1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7vndO$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7vndO$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 13.234c1.105 0 2 .896 2 2v4.645c0 1.104-.895 2-2 2s-2-.896-2-2v-4.645c0-1.104.895-2 2-2m0 20.218c-8.035 0-14.55 6.514-14.55 14.549S39.965 62.55 48 62.55c8.035 0 14.548-6.514 14.548-14.55 0-8.034-6.513-14.548-14.548-14.548M29.45 48.001c0-10.244 8.305-18.549 18.55-18.549 10.244 0 18.548 8.305 18.548 18.549S58.244 66.55 48 66.55c-10.245 0-18.55-8.305-18.55-18.55M50 76.12c0-1.104-.895-2-2-2s-2 .896-2 2v4.645c0 1.104.895 2 2 2s2-.896 2-2zM82.766 48c0 1.105-.896 2-2 2H76.12c-1.105 0-2-.895-2-2s.895-2 2-2h4.645c1.104 0 2 .895 2 2m-62.887 2c1.104 0 2-.895 2-2s-.896-2-2-2h-4.645c-1.104 0-2 .895-2 2s.896 2 2 2zm52.704-26.584c.78.781.78 2.047 0 2.828l-3.284 3.285c-.782.78-2.048.78-2.829 0-.78-.781-.78-2.048 0-2.829l3.284-3.284c.781-.78 2.048-.78 2.829 0M29.529 69.298c.781-.781.781-2.048 0-2.829-.781-.78-2.047-.78-2.828 0l-3.285 3.285c-.78.78-.78 2.047 0 2.828.782.781 2.048.781 2.829 0zm-6.113-45.882c-.78.781-.78 2.047 0 2.828l3.285 3.285c.78.78 2.047.78 2.828 0 .78-.781.78-2.048 0-2.829l-3.284-3.284c-.781-.78-2.048-.78-2.829 0M66.47 69.298c-.781-.781-.781-2.048 0-2.829.781-.78 2.047-.78 2.829 0l3.284 3.285c.78.78.78 2.047 0 2.828-.781.781-2.048.781-2.829 0z"
        })
    });
const $6eb81eb46f3bf6e1$var$ForwardRef = /*#__PURE__*/ (0, $7vndO$forwardRef)($6eb81eb46f3bf6e1$var$SvgComponent);
var $6eb81eb46f3bf6e1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6eb81eb46f3bf6e1$var$ForwardRef);


export {$6eb81eb46f3bf6e1$export$2e2bcd8739ae039 as default};
