// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pin_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $eY01A$reactjsxruntime = require("react/jsx-runtime");
var $eY01A$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b3a7d8b447681374$export$2e2bcd8739ae039);




const $b3a7d8b447681374$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eY01A$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $eY01A$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M37.642 16C33.399 16 30 19.475 30 23.71c0 3.737 2.573 7.333 6.462 7.852v10.282C32.712 42.3 30 45.387 30 49.29c0 4.235 3.399 7.71 7.642 7.71H46v23c0 1.105.895 2 2 2s2-.895 2-2V57h8.358C62.601 57 66 53.525 66 49.29c0-3.737-2.573-7.333-6.462-7.852V31.156C63.288 30.7 66 27.613 66 23.71c0-4.235-3.399-7.71-7.642-7.71zM34 23.71c0-2.072 1.653-3.71 3.642-3.71h20.716C60.347 20 62 21.638 62 23.71c0 2.074-1.448 3.505-3.435 3.505h-1.027c-1.105 0-2 .895-2 2v14.16c0 1.104.895 2 2 2h1.027c1.806 0 3.435 1.648 3.435 3.915 0 2.072-1.653 3.71-3.642 3.71H37.642C35.653 53 34 51.362 34 49.29c0-2.074 1.448-3.505 3.435-3.505h1.027c1.104 0 2-.895 2-2v-14.16c0-1.104-.896-2-2-2h-1.027c-1.806 0-3.435-1.648-3.435-3.915"
        })
    });
const $b3a7d8b447681374$var$ForwardRef = /*#__PURE__*/ (0, $eY01A$react.forwardRef)($b3a7d8b447681374$var$SvgComponent);
var $b3a7d8b447681374$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b3a7d8b447681374$var$ForwardRef);
