// @react-spectrum/s2/illustrations/linear/internal/S2_lin_cloudUpload_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $5U0xf$reactjsxruntime = require("react/jsx-runtime");
var $5U0xf$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $39feb4e8728fbacb$export$2e2bcd8739ae039);




const $39feb4e8728fbacb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5U0xf$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5U0xf$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M58.355 40.065C63.548 28.3 74.992 20 88.43 20c17.478 0 31.545 13.98 32.954 31.468 13.808 2.669 24.036 15.462 24.036 30.012 0 15.805-11.646 29.025-26.808 30.404-2.2.2-4.145-1.422-4.345-3.622s1.421-4.146 3.621-4.346c10.799-.981 19.532-10.521 19.532-22.436 0-11.908-9.101-21.832-20.176-22.487-2.114-.125-3.764-1.876-3.764-3.993 0-.2.009-.39.014-.488v-.018l.006-.134C113.5 39.632 102.119 28 88.43 28c-11.046 0-20.568 7.56-23.857 18.246-.334 1.088-1.116 1.981-2.15 2.457s-2.22.49-3.265.037c-2.003-.87-4.192-1.35-6.498-1.35-9.348 0-17.21 7.963-17.21 18.15 0 1.393.17 2.776.469 4.15.448 2.062-.782 4.117-2.81 4.696-5.973 1.706-10.529 7.478-10.529 14.534 0 8.398 6.41 14.923 14.032 15.061q.166.003.33.019h4.438c2.21 0 4 1.791 4 4s-1.79 4-4 4h-4.84q-.263 0-.518-.033C23.96 111.505 14.58 101.231 14.58 88.92c0-9.242 5.263-17.299 12.984-20.963q-.113-1.186-.114-2.417c0-14.273 11.118-26.15 25.21-26.15 1.96 0 3.866.234 5.695.675M83.5 73.657l16.672 16.671c1.562 1.563 4.094 1.563 5.656 0 1.563-1.562 1.563-4.094 0-5.656l-23.5-23.5c-1.562-1.562-4.094-1.562-5.656 0l-23.5 23.5c-1.562 1.562-1.562 4.094 0 5.656 1.562 1.563 4.094 1.563 5.656 0L75.5 73.657V136c0 2.209 1.79 4 4 4s4-1.791 4-4z"
        })
    });
const $39feb4e8728fbacb$var$ForwardRef = /*#__PURE__*/ (0, $5U0xf$react.forwardRef)($39feb4e8728fbacb$var$SvgComponent);
var $39feb4e8728fbacb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($39feb4e8728fbacb$var$ForwardRef);
