// @react-spectrum/s2/illustrations/linear/internal/S2_lin_effects_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $kKrac$jsx} from "react/jsx-runtime";
import {forwardRef as $kKrac$forwardRef} from "react";





const $c7990660c0ec01d1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kKrac$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kKrac$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M73.868 30.965c-3.7 2.817-6.314 7.278-7.825 11.423l-5.35 17.434h28.054c1.454 0 2.795.79 3.5 2.063l10.991 19.857c7.514-7.507 15.095-15.08 20.809-20.786 1.563-1.56 4.096-1.559 5.657.004s1.559 4.096-.004 5.657c-6.126 6.117-14.397 14.38-22.431 22.407l12.442 22.476c1.07 1.933.37 4.367-1.563 5.437s-4.367.371-5.436-1.562l-11.33-20.468-2.719 2.718c-5.15 5.148-9.781 9.779-13.135 13.138-1.677 1.679-3.034 3.039-3.975 3.985l-1.085 1.091-.183.186-.092.093-.023.025c-1.374 1.499-3.687 1.739-5.347.494-1.767-1.325-2.125-3.833-.8-5.6.102-.136.195-.24.207-.254l.001-.001.067-.074.067-.071.107-.11.308-.313 1.103-1.11c.946-.951 2.307-2.314 3.984-3.995 3.357-3.36 7.99-7.994 13.14-13.142l4.344-4.342-10.962-19.803H58.237l-21.285 69.351c-.648 2.112-2.886 3.299-4.998 2.651s-3.298-2.886-2.65-4.998L49.87 67.822H37.06c-2.21 0-4-1.79-4-4s1.79-4 4-4h15.263l6.1-19.877q.03-.093.062-.184c1.807-5.006 5.151-11.06 10.534-15.16 5.602-4.267 13.107-6.161 22.307-3.085 2.095.701 3.226 2.968 2.525 5.063-.7 2.095-2.967 3.225-5.062 2.524-6.672-2.231-11.415-.81-14.922 1.862m6.276 85.204.012-.012c-.053.058-.07.073-.012.012"
        })
    });
const $c7990660c0ec01d1$var$ForwardRef = /*#__PURE__*/ (0, $kKrac$forwardRef)($c7990660c0ec01d1$var$SvgComponent);
var $c7990660c0ec01d1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c7990660c0ec01d1$var$ForwardRef);


export {$c7990660c0ec01d1$export$2e2bcd8739ae039 as default};
