// @react-spectrum/s2/illustrations/linear/internal/S2_lin_filters_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7oqGJ$jsx} from "react/jsx-runtime";
import {forwardRef as $7oqGJ$forwardRef} from "react";





const $ce19369b1bc6324d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7oqGJ$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7oqGJ$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M97.052 88.563c6.969-5.002 11.792-12.805 12.821-21.763-2.73-.78-5.613-1.198-8.593-1.198-5.302 0-10.296 1.322-14.668 3.656 5.15 5.216 8.85 11.87 10.44 19.305M89.625 92.5c-3.35 1.229-6.97 1.9-10.746 1.9-2.98 0-5.864-.418-8.593-1.199.868-7.556 4.435-14.29 9.714-19.216 5.114 4.772 8.622 11.243 9.625 18.515M73.388 69.258C67.935 74.779 64.11 81.91 62.69 89.876c-7.838-4.767-13.409-12.89-14.717-22.375 3.35-1.229 6.97-1.9 10.746-1.9 5.303 0 10.296 1.323 14.669 3.657M80 63.875c6.128-3.969 13.435-6.273 21.28-6.273 2.901 0 5.728.315 8.449.913C107.469 43.506 94.518 32 78.879 32c-15.842 0-28.926 11.807-30.933 27.1 3.423-.976 7.037-1.498 10.773-1.498 7.845 0 15.152 2.304 21.28 6.273m-40.315-1.35C27.657 69.219 19.52 82.06 19.52 96.8c0 21.65 17.55 39.2 39.2 39.2 7.844 0 15.151-2.304 21.28-6.273 6.128 3.969 13.435 6.273 21.28 6.273 21.649 0 39.2-17.55 39.2-39.2 0-15.654-9.177-29.166-22.444-35.448C117.072 40.561 99.909 24 78.879 24c-21.424 0-38.834 17.187-39.194 38.525m.861 8.913C32.658 77.1 27.52 86.35 27.52 96.8c0 17.232 13.968 31.2 31.2 31.2 5.302 0 10.295-1.323 14.668-3.656-6.589-6.671-10.801-15.695-11.265-25.697C51.25 93.5 43.125 83.497 40.546 71.438M80 119.618c-5.025-4.688-8.499-11.016-9.57-18.131 2.72.597 5.548.913 8.449.913 3.736 0 7.35-.523 10.773-1.499-.965 7.353-4.49 13.9-9.652 18.717m6.612 4.727c6.844-6.931 11.125-16.4 11.301-26.869 10.1-5.62 17.457-15.576 19.556-27.35 9 5.472 15.011 15.372 15.011 26.675 0 17.232-13.969 31.2-31.2 31.2-5.302 0-10.296-1.323-14.668-3.656"
        })
    });
const $ce19369b1bc6324d$var$ForwardRef = /*#__PURE__*/ (0, $7oqGJ$forwardRef)($ce19369b1bc6324d$var$SvgComponent);
var $ce19369b1bc6324d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ce19369b1bc6324d$var$ForwardRef);


export {$ce19369b1bc6324d$export$2e2bcd8739ae039 as default};
