// @react-spectrum/s2/illustrations/linear/internal/S2_lin_document_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1RiUD$reactjsxruntime = require("react/jsx-runtime");
var $1RiUD$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ba3c4207a73ae31e$export$2e2bcd8739ae039);




const $ba3c4207a73ae31e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1RiUD$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1RiUD$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M29.955 16C25.018 16 21 19.986 21 24.923v46.154C21 76.013 25.018 80 29.955 80h37.09C71.983 80 76 76.013 76 71.077V43.945c0-2.37-.946-4.64-2.626-6.313L54.266 18.61C52.586 16.938 50.31 16 47.938 16zM25 24.923C25 22.213 27.21 20 29.954 20H48v17.55c0 3.09 2.588 5.45 5.6 5.45h18.308q.091.464.092.945v27.132C72 73.787 69.79 76 67.046 76H29.954C27.21 76 25 73.787 25 71.077zM69.079 39 52 21.999V37.55c0 .721.635 1.45 1.6 1.45z"
        })
    });
const $ba3c4207a73ae31e$var$ForwardRef = /*#__PURE__*/ (0, $1RiUD$react.forwardRef)($ba3c4207a73ae31e$var$SvgComponent);
var $ba3c4207a73ae31e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ba3c4207a73ae31e$var$ForwardRef);
