// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pause_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $27Ji7$jsx} from "react/jsx-runtime";
import {forwardRef as $27Ji7$forwardRef} from "react";





const $2f09ea534e85b91c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $27Ji7$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $27Ji7$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M12 13c0-1.795 1.455-3.25 3.25-3.25h3.5C20.545 9.75 22 11.205 22 13v22c0 1.795-1.455 3.25-3.25 3.25h-3.5C13.455 38.25 12 36.795 12 35zm3.25-.75c-.414 0-.75.336-.75.75v22c0 .414.336.75.75.75h3.5c.414 0 .75-.336.75-.75V13c0-.414-.336-.75-.75-.75zM26 13c0-1.795 1.455-3.25 3.25-3.25h3.5C34.545 9.75 36 11.205 36 13v22c0 1.795-1.455 3.25-3.25 3.25h-3.5C27.455 38.25 26 36.795 26 35zm3.25-.75c-.414 0-.75.336-.75.75v22c0 .414.336.75.75.75h3.5c.414 0 .75-.336.75-.75V13c0-.414-.336-.75-.75-.75z"
        })
    });
const $2f09ea534e85b91c$var$ForwardRef = /*#__PURE__*/ (0, $27Ji7$forwardRef)($2f09ea534e85b91c$var$SvgComponent);
var $2f09ea534e85b91c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2f09ea534e85b91c$var$ForwardRef);


export {$2f09ea534e85b91c$export$2e2bcd8739ae039 as default};
