// @react-spectrum/s2/illustrations/linear/internal/S2_lin_paperairplane_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $4uSPf$reactjsxruntime = require("react/jsx-runtime");
var $4uSPf$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $21660a47b3d2faa4$export$2e2bcd8739ae039);




const $21660a47b3d2faa4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4uSPf$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4uSPf$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M91.725 24.406c2.709-5.4-2.058-11.568-7.975-10.221L8.074 31.41C1.99 32.795.473 40.771 5.597 44.314l1.137-1.645-1.137 1.645L19.5 53.93v23.43c0 4.13 4.887 5.96 7.84 3.55L41.64 69.238l13.729 9.493c3.577 2.474 8.503 1.203 10.452-2.682zM38.215 66.87 23.501 56.695V77.36c0 .218.114.43.415.562.305.133.638.099.896-.112zM25.52 53.227l32.125 22.214c1.57 1.086 3.739.534 4.601-1.186L88.15 22.613q.209-.421.29-.847zm61.023-34.984c-.567-.239-1.22-.314-1.904-.158L8.962 35.31c-2.676.61-3.365 4.141-1.09 5.714l13.893 9.608z"
        })
    });
const $21660a47b3d2faa4$var$ForwardRef = /*#__PURE__*/ (0, $4uSPf$react.forwardRef)($21660a47b3d2faa4$var$SvgComponent);
var $21660a47b3d2faa4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($21660a47b3d2faa4$var$ForwardRef);
