// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noComments_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $lxQaT$reactjsxruntime = require("react/jsx-runtime");
var $lxQaT$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $24221b858ad98a16$export$2e2bcd8739ae039);




const $24221b858ad98a16$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lxQaT$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $lxQaT$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 40.69c0-13.53 10.97-24.5 24.5-24.5S129 27.16 129 40.69s-10.97 24.5-24.5 24.5S80 54.22 80 40.69m35.42 10.93c1.56-1.57 1.56-4.1 0-5.66l-5.26-5.26 5.26-5.26c1.56-1.57 1.56-4.1 0-5.66s-4.1-1.56-5.66 0l-5.26 5.26-5.26-5.26c-1.56-1.56-4.1-1.56-5.66 0-1.56 1.57-1.56 4.1 0 5.66l5.26 5.26-5.26 5.26c-1.56 1.57-1.56 4.1 0 5.66.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17l5.26-5.26 5.26 5.26c.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17M16 43.28c0-8.025 6.552-14.47 14.56-14.47h34.815c2.21 0 4 1.791 4 4s-1.79 4-4 4H30.561c-3.657 0-6.561 2.93-6.561 6.47v37.15c0 3.54 2.904 6.468 6.56 6.468h9.458c2.202 0 3.99 1.78 4 3.983.02 4.64-.014 11.134-.05 16.358l22.544-19.375c.726-.623 1.65-.966 2.607-.966h28.32c3.657 0 6.561-2.929 6.561-6.468v-5.245c0-2.209 1.791-4 4-4s4 1.791 4 4v5.245c0 8.023-6.552 14.468-14.56 14.468H70.601l-22.21 19.086c-4.834 4.155-12.48.789-12.432-5.723.03-4.051.059-9.045.063-13.363h-5.462C22.552 94.898 16 88.453 16 80.43zm103 26.53c0-2.209 1.791-4 4-4h7.64c7.344 0 13.36 5.91 13.36 13.276v32.919c0 7.366-6.016 13.276-13.36 13.276h-4.387c.005 3.716.03 7.933.056 11.381.045 6.172-7.198 9.349-11.771 5.42L94.981 125.28H71.36c-5.686 0-10.556-3.532-12.482-8.532-.793-2.062.234-4.377 2.296-5.17s4.376.234 5.17 2.295c.763 1.981 2.713 3.407 5.016 3.407h25.102c.956 0 1.881.343 2.607.966l19.225 16.517c-.029-4.437-.054-9.677-.038-13.499.01-2.203 1.798-3.983 4-3.983h8.383c2.995 0 5.36-2.397 5.36-5.276V79.087c0-2.88-2.365-5.276-5.36-5.276H123c-2.209 0-4-1.791-4-4"
        })
    });
const $24221b858ad98a16$var$ForwardRef = /*#__PURE__*/ (0, $lxQaT$react.forwardRef)($24221b858ad98a16$var$SvgComponent);
var $24221b858ad98a16$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($24221b858ad98a16$var$ForwardRef);
