// @react-spectrum/s2/illustrations/linear/internal/S2_lin_check_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $htcs9$reactjsxruntime = require("react/jsx-runtime");
var $htcs9$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $76c7d6db66cd1da9$export$2e2bcd8739ae039);




const $76c7d6db66cd1da9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $htcs9$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $htcs9$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8.25 24c0-8.698 7.052-15.75 15.75-15.75S39.75 15.302 39.75 24 32.699 39.75 24 39.75 8.25 32.699 8.25 24M24 5.75C13.92 5.75 5.75 13.92 5.75 24S13.92 42.25 24 42.25 42.25 34.08 42.25 24 34.08 5.75 24 5.75m8.336 14.68c.513-.463.555-1.253.093-1.766s-1.252-.555-1.765-.093l-9.119 8.206-3.661-3.66c-.488-.489-1.28-.489-1.768 0-.488.487-.488 1.279 0 1.767l4.366 4.366c.545.545 1.422.568 1.995.052z"
        })
    });
const $76c7d6db66cd1da9$var$ForwardRef = /*#__PURE__*/ (0, $htcs9$react.forwardRef)($76c7d6db66cd1da9$var$SvgComponent);
var $76c7d6db66cd1da9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($76c7d6db66cd1da9$var$ForwardRef);
