// @react-spectrum/s2/illustrations/linear/internal/S2_lin_imageStack_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsxs as $6mKFA$jsxs, jsx as $6mKFA$jsx} from "react/jsx-runtime";
import {forwardRef as $6mKFA$forwardRef} from "react";





const $8ad94454391f8954$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6mKFA$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6mKFA$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M101.93 62.63c4.418 0 8-3.582 8-8s-3.582-8-8-8-8 3.582-8 8 3.581 8 8 8"
            }),
            /*#__PURE__*/ (0, $6mKFA$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M15.43 48.47c0-11.045 8.954-20 20-20h74c11.045 0 20 8.955 20 20v49c0 11.046-8.955 20-20 20h-74c-11.046 0-20-8.954-20-20zm20-12c-6.628 0-12 5.373-12 12v49c0 1.023.128 2.016.369 2.964l21.244-32.929c4.19-6.494 13.723-6.39 17.769.196l16.933 27.558 9.82-14.984c4.192-6.397 13.599-6.309 17.671.166l8.13 12.93 5.282 8.369c.505-1.327.782-2.766.782-4.27v-49c0-6.627-5.373-12-12-12zm79.722 71.551L100.463 84.7c-.97-1.542-3.209-1.563-4.207-.04l-16.26 24.81h29.434c2.072 0 4.021-.525 5.722-1.449m-44.72 1.449 4.467-6.817-18.903-30.764c-.963-1.568-3.233-1.593-4.23-.047l-22.982 35.621c1.902 1.268 4.188 2.007 6.646 2.007z"
            }),
            /*#__PURE__*/ (0, $6mKFA$jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M144.5 67c0-2.21-1.791-4-4-4s-4 1.79-4 4v40.5c0 8.837-7.163 16-16 16h-66c-2.21 0-4 1.791-4 4s1.79 4 4 4h66c13.255 0 24-10.745 24-24z"
            })
        ]
    });
const $8ad94454391f8954$var$ForwardRef = /*#__PURE__*/ (0, $6mKFA$forwardRef)($8ad94454391f8954$var$SvgComponent);
var $8ad94454391f8954$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8ad94454391f8954$var$ForwardRef);


export {$8ad94454391f8954$export$2e2bcd8739ae039 as default};
