// @react-spectrum/s2/illustrations/linear/internal/S2_lin_information_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $avq7x$reactjsxruntime = require("react/jsx-runtime");
var $avq7x$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f71e9b3b302643fa$export$2e2bcd8739ae039);




const $f71e9b3b302643fa$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $avq7x$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $avq7x$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.5 47.74c0-17.12 13.88-31 31-31s31 13.88 31 31c0 17.121-13.88 31-31 31s-31-13.879-31-31m31-35c-19.33 0-35 15.67-35 35s15.67 35 35 35 35-15.67 35-35-15.67-35-35-35m2.5 21.5c0 1.657-1.343 3-3 3s-3-1.343-3-3 1.343-3 3-3 3 1.343 3 3m-7 6c-1.105 0-2 .896-2 2s.895 2 2 2h3v14h-3c-1.105 0-2 .896-2 2s.895 2 2 2h10c1.105 0 2-.895 2-2s-.895-2-2-2h-3v-16c0-1.104-.895-2-2-2z"
        })
    });
const $f71e9b3b302643fa$var$ForwardRef = /*#__PURE__*/ (0, $avq7x$react.forwardRef)($f71e9b3b302643fa$var$SvgComponent);
var $f71e9b3b302643fa$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f71e9b3b302643fa$var$ForwardRef);
