// @react-spectrum/s2/illustrations/linear/internal/S2_lin_plugin_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $g0e92$reactjsxruntime = require("react/jsx-runtime");
var $g0e92$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5187b57d0e3026e6$export$2e2bcd8739ae039);




const $5187b57d0e3026e6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $g0e92$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $g0e92$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M33.077 41.333c0-4.722 3.517-9.333 8.77-9.333h23.845c5.252 0 8.77 4.61 8.77 9.333v9.334h11.076v-9.334c0-4.722 3.518-9.333 8.77-9.333h23.846c5.252 0 8.769 4.61 8.769 9.333v9.334h.769c10.52 0 18.308 9.386 18.308 20V116c0 10.613-7.788 20-18.308 20H32.308C21.788 136 14 126.613 14 116V70.667c0-10.614 7.788-20 18.308-20h.769zm8.379-1.105c-.196.219-.38.598-.38 1.105v17.334h-8.768c-5.284 0-10.308 4.94-10.308 12V116c0 7.06 5.024 12 10.308 12h95.384c5.284 0 10.308-4.94 10.308-12V70.667c0-7.06-5.024-12-10.308-12h-8.769V41.333c0-.507-.183-.886-.379-1.105-.188-.21-.333-.228-.39-.228H94.308c-.058 0-.202.017-.39.228-.196.219-.38.598-.38 1.105v17.334H66.463V41.333c0-.507-.184-.886-.379-1.105-.189-.21-.333-.228-.39-.228H41.845c-.057 0-.202.017-.39.228"
        })
    });
const $5187b57d0e3026e6$var$ForwardRef = /*#__PURE__*/ (0, $g0e92$react.forwardRef)($5187b57d0e3026e6$var$SvgComponent);
var $5187b57d0e3026e6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5187b57d0e3026e6$var$ForwardRef);
