// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowDown_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $4koMQ$jsx} from "react/jsx-runtime";
import {forwardRef as $4koMQ$forwardRef} from "react";





const $391acfc63860c7fd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4koMQ$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4koMQ$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25.25 6.5c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v30.982l-4.866-4.866c-.488-.488-1.28-.488-1.768 0s-.488 1.28 0 1.768l7 7c.488.488 1.28.488 1.768 0l7-7c.488-.488.488-1.28 0-1.768s-1.28-.488-1.768 0l-4.866 4.866z"
        })
    });
const $391acfc63860c7fd$var$ForwardRef = /*#__PURE__*/ (0, $4koMQ$forwardRef)($391acfc63860c7fd$var$SvgComponent);
var $391acfc63860c7fd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($391acfc63860c7fd$var$ForwardRef);


export {$391acfc63860c7fd$export$2e2bcd8739ae039 as default};
