// @react-spectrum/s2/illustrations/linear/internal/S2_lin_movieCamera_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $gTnMs$reactjsxruntime = require("react/jsx-runtime");
var $gTnMs$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $cb3cc7def1ced1cd$export$2e2bcd8739ae039);




const $cb3cc7def1ced1cd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gTnMs$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gTnMs$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M11.75 34.094c0-5.523 4.477-10 10-10h31.924c5.523 0 10 4.477 10 10v3.794l11-8.163c3.96-2.939 9.576-.113 9.576 4.818v26.815c0 4.918-5.592 7.747-9.554 4.834l-11.022-8.103v3.816c0 5.523-4.477 10-10 10H21.75c-5.523 0-10-4.477-10-10zm10-6c-3.314 0-6 2.686-6 6v27.811c0 3.314 2.686 6 6 6h31.924c3.314 0 6-2.686 6-6v-7.77c0-.753.424-1.443 1.097-1.784s1.48-.273 2.088.174l14.206 10.444c1.321.971 3.185.028 3.185-1.611V34.543c0-1.644-1.872-2.586-3.192-1.606L62.866 43.469c-.607.45-1.416.52-2.091.18s-1.101-1.03-1.101-1.786v-7.77c0-3.313-2.686-6-6-6z"
        })
    });
const $cb3cc7def1ced1cd$var$ForwardRef = /*#__PURE__*/ (0, $gTnMs$react.forwardRef)($cb3cc7def1ced1cd$var$SvgComponent);
var $cb3cc7def1ced1cd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($cb3cc7def1ced1cd$var$ForwardRef);
