// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browser_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1o6Y7$jsx} from "react/jsx-runtime";
import {forwardRef as $1o6Y7$forwardRef} from "react";





const $17cd7e24e2f42736$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1o6Y7$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1o6Y7$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M74.5 20c3.314 0 6 2.686 6 6v6h-64v-6c0-3.314 2.686-6 6-6zm-58 16h64v34c0 3.314-2.686 6-6 6h-52c-3.314 0-6-2.686-6-6zm68-10c0-5.523-4.477-10-10-10h-52c-5.523 0-10 4.477-10 10v44c0 5.523 4.477 10 10 10h52c5.523 0 10-4.477 10-10zM24 28.5c-1.38 0-2.5-1.12-2.5-2.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5-1.12 2.5-2.5 2.5"
        })
    });
const $17cd7e24e2f42736$var$ForwardRef = /*#__PURE__*/ (0, $1o6Y7$forwardRef)($17cd7e24e2f42736$var$SvgComponent);
var $17cd7e24e2f42736$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($17cd7e24e2f42736$var$ForwardRef);


export {$17cd7e24e2f42736$export$2e2bcd8739ae039 as default};
