// @react-spectrum/s2/illustrations/linear/internal/S2_lin_close_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $aDz64$jsx} from "react/jsx-runtime";
import {forwardRef as $aDz64$forwardRef} from "react";





const $c2f39f621fb76fb9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aDz64$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $aDz64$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 8.25C15.302 8.25 8.25 15.302 8.25 24S15.302 39.75 24 39.75 39.75 32.699 39.75 24 32.699 8.25 24 8.25M5.75 24C5.75 13.92 13.92 5.75 24 5.75S42.25 13.92 42.25 24 34.08 42.25 24 42.25 5.75 34.08 5.75 24m12.366-5.884c.488-.488 1.28-.488 1.768 0l4.3 4.3 4.3-4.3c.487-.488 1.279-.488 1.767 0s.488 1.28 0 1.768l-4.3 4.3 4.3 4.3c.488.487.488 1.279 0 1.767s-1.28.488-1.768 0l-4.3-4.3-4.3 4.3c-.487.488-1.279.488-1.767 0s-.488-1.28 0-1.768l4.3-4.3-4.3-4.3c-.488-.487-.488-1.279 0-1.767"
        })
    });
const $c2f39f621fb76fb9$var$ForwardRef = /*#__PURE__*/ (0, $aDz64$forwardRef)($c2f39f621fb76fb9$var$SvgComponent);
var $c2f39f621fb76fb9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c2f39f621fb76fb9$var$ForwardRef);


export {$c2f39f621fb76fb9$export$2e2bcd8739ae039 as default};
