// @react-spectrum/s2/illustrations/linear/internal/S2_lin_remix_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $iDbyQ$reactjsxruntime = require("react/jsx-runtime");
var $iDbyQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4d55f7b3b44b201b$export$2e2bcd8739ae039);




const $4d55f7b3b44b201b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iDbyQ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $iDbyQ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m38.192 20.999 5.307-5.308c.488-.488.488-1.28 0-1.767l-5.307-5.308c-.489-.488-1.28-.488-1.768 0s-.488 1.28 0 1.768l3.071 3.071h-1.789c-2.517 0-4.84.848-6.693 2.273q-.043.033-.082.069l-.035.026c-1.365 1.072-2.633 2.625-3.622 4.043-.994 1.425-1.773 2.81-2.14 3.618-.282.624-.983 2.109-1.91 3.668-.799 1.342-1.706 2.637-2.6 3.507q-.042.033-.083.07c-1.508 1.37-3.509 2.204-5.706 2.204-4.689 0-8.49-3.8-8.49-8.49 0-4.687 3.801-8.488 8.49-8.488 2.866 0 5.401 1.42 6.94 3.6l.005.007.451.63c.402.562 1.182.692 1.744.29.561-.4.691-1.182.29-1.743l-.45-.628c-1.987-2.814-5.268-4.656-8.98-4.656-6.07 0-10.99 4.92-10.99 10.989s4.92 10.989 10.99 10.989c2.512 0 4.83-.845 6.68-2.264q.082-.046.158-.106c1.434-1.131 2.728-3 3.7-4.634.992-1.668 1.736-3.245 2.039-3.913.288-.637.984-1.889 1.912-3.22.933-1.337 2.036-2.658 3.116-3.506q.038-.03.071-.06l.026-.02c1.431-1.1 3.222-1.755 5.17-1.755H39.7l-3.276 3.276c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0m2.214 14.434c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.7c-2.798 0-5.077-1.609-7.09-3.803l-.527-.638c-.44-.532-1.228-.606-1.76-.166s-.607 1.228-.166 1.76l.547.662.02.024.02.023c2.186 2.39 5.095 4.638 8.956 4.638z"
        })
    });
const $4d55f7b3b44b201b$var$ForwardRef = /*#__PURE__*/ (0, $iDbyQ$react.forwardRef)($4d55f7b3b44b201b$var$SvgComponent);
var $4d55f7b3b44b201b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4d55f7b3b44b201b$var$ForwardRef);
