// @react-spectrum/s2/illustrations/linear/internal/S2_lin_bolt_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $9Phgp$reactjsxruntime = require("react/jsx-runtime");
var $9Phgp$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $19a23a0138817549$export$2e2bcd8739ae039);




const $19a23a0138817549$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9Phgp$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9Phgp$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            d: "M95.1 29.88c2.58 0 5.01 2.61 3.74 5.59L87.27 62.73c-.86 2.03.09 4.38 2.12 5.25l19 8.05c2.53 1.07 3.25 4.32 1.41 6.36L68 128.782c-.85.95-1.9 1.35-2.92 1.35-2.58 0-5-2.6-3.74-5.58l11.21-26.6c.85-2.02-.09-4.36-2.1-5.23L51.58 84.6c-2.51-1.08-3.22-4.3-1.4-6.34l42-47.01c.85-.96 1.9-1.37 2.93-1.37m-.01-8c-3.41 0-6.57 1.43-8.89 4.04l-42 47.01c-2.56 2.87-3.6 6.83-2.77 10.59s3.44 6.91 6.97 8.43l15.23 6.55-9.67 22.94c-1.57 3.73-1.2 7.93 1 11.25 2.23 3.36 6.1 5.44 10.11 5.44 3.39 0 6.54-1.42 8.86-4l41.8-46.39c2.59-2.88 3.64-6.85 2.8-10.63s-3.47-6.94-7.04-8.45l-15.31-6.49L106.2 38.6c1.58-3.73 1.22-7.94-.98-11.26-2.23-3.37-6.11-5.46-10.12-5.46"
        })
    });
const $19a23a0138817549$var$ForwardRef = /*#__PURE__*/ (0, $9Phgp$react.forwardRef)($19a23a0138817549$var$SvgComponent);
var $19a23a0138817549$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($19a23a0138817549$var$ForwardRef);
