// @react-spectrum/s2/illustrations/linear/internal/S2_lin_conversationbubbles_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $hvPa2$jsx} from "react/jsx-runtime";
import {forwardRef as $hvPa2$forwardRef} from "react";





const $7e54904251df346c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hvPa2$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hvPa2$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M18.36 16C13.783 16 10 19.645 10 24.228V46.33c0 4.584 3.783 8.228 8.36 8.228h3.7c-.001 2.654-.02 5.803-.039 8.333-.026 3.584 4.187 5.343 6.803 3.122l13.494-11.455H58.64c4.577 0 8.36-3.644 8.36-8.228V24.228C67 19.644 63.217 16 58.64 16zM14 24.228C14 21.932 15.912 20 18.36 20h40.28c2.448 0 4.36 1.932 4.36 4.228V46.33c0 2.296-1.912 4.228-4.36 4.228H41.583c-.474 0-.933.169-1.294.476l-14.053 11.93q-.033.025-.044.029-.012.005-.025.006c-.021.003-.055 0-.091-.017-.034-.015-.05-.033-.053-.037q-.001-.004-.002-.024c.024-3.203.048-7.42.035-10.371-.005-1.101-.899-1.992-2-1.992H18.36c-2.448 0-4.36-1.932-4.36-4.228zM72 37c-1.105 0-2 .895-2 2s.895 2 2 2h6.36c2.035 0 3.64 1.621 3.64 3.566v19.75c0 1.945-1.605 3.566-3.64 3.566h-5.051c-1.102 0-1.996.89-2 1.992-.011 2.552.008 6.162.029 8.986L59.066 68.362c-.362-.31-.823-.48-1.3-.48H42.64c-1.762 0-3.21-1.221-3.559-2.81-.237-1.08-1.303-1.762-2.382-1.526-1.079.237-1.761 1.304-1.525 2.383.751 3.42 3.821 5.953 7.466 5.953h14.387l11.901 10.18c2.482 2.124 6.436.419 6.412-2.948-.017-2.194-.032-4.907-.034-7.232h3.054c4.194 0 7.64-3.362 7.64-7.566v-19.75C86 40.362 82.554 37 78.36 37z"
        })
    });
const $7e54904251df346c$var$ForwardRef = /*#__PURE__*/ (0, $hvPa2$forwardRef)($7e54904251df346c$var$SvgComponent);
var $7e54904251df346c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7e54904251df346c$var$ForwardRef);


export {$7e54904251df346c$export$2e2bcd8739ae039 as default};
