// @react-spectrum/s2/illustrations/linear/internal/S2_lin_desktop_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $i9ijI$reactjsxruntime = require("react/jsx-runtime");
var $i9ijI$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c1f36aec64edc520$export$2e2bcd8739ae039);




const $c1f36aec64edc520$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $i9ijI$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $i9ijI$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20.969 44c0-6.627 5.372-12 12-12h94.06c6.627 0 12 5.373 12 12v44c0 6.627-5.373 12-12 12h-94.06c-6.628 0-12-5.373-12-12zm12-20c-11.046 0-20 8.954-20 20v44c0 11.046 8.954 20 20 20h40.03v20h-14.01c-2.209 0-4 1.791-4 4s1.791 4 4 4h42.02c2.209 0 4-1.791 4-4s-1.791-4-4-4h-14.01v-20h40.03c11.045 0 20-8.954 20-20V44c0-11.046-8.955-20-20-20zm43.05 20.5c0 2.486 2.014 4.5 4.5 4.5s4.5-2.014 4.5-4.5-2.013-4.5-4.5-4.5h-.01c-2.476 0-4.49 2.014-4.49 4.5"
        })
    });
const $c1f36aec64edc520$var$ForwardRef = /*#__PURE__*/ (0, $i9ijI$react.forwardRef)($c1f36aec64edc520$var$SvgComponent);
var $c1f36aec64edc520$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c1f36aec64edc520$var$ForwardRef);
