// @react-spectrum/s2/illustrations/linear/internal/S2_lin_switch_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $7znRJ$reactjsxruntime = require("react/jsx-runtime");
var $7znRJ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e96461d53b8982b2$export$2e2bcd8739ae039);




const $e96461d53b8982b2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7znRJ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7znRJ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M12.586 30.414c-.781-.78-.781-2.047 0-2.828l14-14c.78-.781 2.047-.781 2.828 0 .781.78.781 2.047 0 2.828L18.828 27H81.2c1.105 0 2 .895 2 2s-.895 2-2 2H18.828l10.586 10.586c.781.78.781 2.047 0 2.828-.78.781-2.047.781-2.828 0zm70.629 35.172c.78.78.78 2.047 0 2.828l-14 14c-.781.781-2.047.781-2.829 0-.78-.78-.78-2.047 0-2.828L76.972 69H15c-1.104 0-2-.895-2-2s.896-2 2-2h61.972L66.386 54.414c-.78-.78-.78-2.047 0-2.828.782-.781 2.048-.781 2.829 0z"
        })
    });
const $e96461d53b8982b2$var$ForwardRef = /*#__PURE__*/ (0, $7znRJ$react.forwardRef)($e96461d53b8982b2$var$SvgComponent);
var $e96461d53b8982b2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e96461d53b8982b2$var$ForwardRef);
