// @react-spectrum/s2/illustrations/linear/internal/S2_lin_stopwatch_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $ieSXp$jsx} from "react/jsx-runtime";
import {forwardRef as $ieSXp$forwardRef} from "react";





const $fea9865aa259d2ba$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ieSXp$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $ieSXp$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M86.89 22.004c2.208 0 4-1.791 4-4s-1.792-4-4-4H71.143c-2.209 0-4 1.79-4 4s1.791 4 4 4h3.872v4.014C44.16 28.55 19.908 54.396 19.908 85.905c0 33.188 26.904 60.091 60.091 60.091 33.188 0 60.092-26.903 60.092-60.09 0-32.177-25.289-58.446-57.075-60.018v-3.884zM27.907 85.905c0-28.769 23.322-52.09 52.091-52.09s52.092 23.321 52.092 52.09-23.322 52.091-52.092 52.091c-28.769 0-52.09-23.322-52.09-52.09m55.108-44.283c0-2.209-1.79-4-4-4s-4 1.791-4 4v5.905c0 2.209 1.791 4 4 4s4-1.791 4-4zm-4 78.662c2.21 0 4 1.791 4 4v5.904c0 2.209-1.79 4-4 4s-4-1.791-4-4v-5.904c0-2.209 1.791-4 4-4m45.447-30.893c2.209 0 4-1.79 4-4s-1.791-4-4-4h-6.121c-2.209 0-4 1.791-4 4s1.791 4 4 4zm-78.624-4c0 2.21-1.791 4-4 4h-6.12c-2.21 0-4-1.79-4-4s1.79-4 4-4h6.12c2.209 0 4 1.791 4 4m69.209-27.768c1.562-1.562 1.562-4.095 0-5.657s-4.095-1.562-5.657 0l-4.328 4.328c-1.562 1.562-1.562 4.095 0 5.657s4.095 1.562 5.657 0zm-58.424 52.762c1.562 1.562 1.562 4.095 0 5.657l-4.328 4.327c-1.562 1.562-4.094 1.562-5.656 0s-1.562-4.094 0-5.657l4.327-4.327c1.562-1.562 4.095-1.562 5.657 0m-9.431-50c-1.563-1.563-1.563-4.095 0-5.657s4.094-1.563 5.656 0l4.183 4.182c1.562 1.562 1.562 4.095 0 5.657s-4.095 1.562-5.657 0zm56.466 50.812c-1.562 1.562-1.562 4.094 0 5.656l4.183 4.183c1.562 1.562 4.094 1.562 5.657 0s1.562-4.095 0-5.657l-4.183-4.182c-1.562-1.563-4.095-1.563-5.657 0m-22.616-27.79c-2.059-.8-4.377.222-5.176 2.281s.221 4.377 2.28 5.177l34.845 13.529c2.06.799 4.377-.222 5.177-2.281s-.222-4.377-2.281-5.177z"
        })
    });
const $fea9865aa259d2ba$var$ForwardRef = /*#__PURE__*/ (0, $ieSXp$forwardRef)($fea9865aa259d2ba$var$SvgComponent);
var $fea9865aa259d2ba$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($fea9865aa259d2ba$var$ForwardRef);


export {$fea9865aa259d2ba$export$2e2bcd8739ae039 as default};
