// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pencil_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $b15CJ$reactjsxruntime = require("react/jsx-runtime");
var $b15CJ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1991e3eb8a26eb8d$export$2e2bcd8739ae039);




const $1991e3eb8a26eb8d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $b15CJ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $b15CJ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M35.678 95.41c.4-1.071 1.026-2.043 1.835-2.852L101.7 28.37c8.265-8.264 21.665-8.264 29.93 0 8.265 8.265 8.265 21.665 0 29.93l-64.187 64.188c-.809.808-1.781 1.434-2.852 1.835l-33.874 12.683c-4.822 1.805-9.528-2.901-7.723-7.723zm71.679-61.383-4.054 4.054 18.616 18.616 4.054-4.054c5.141-5.14 5.141-13.475 0-18.616-5.14-5.14-13.475-5.14-18.616 0m-61.899 61.9 52.188-52.189 18.616 18.616-52.188 52.188zm-4.124 7.189-9.307 24.857 24.857-9.307z"
        })
    });
const $1991e3eb8a26eb8d$var$ForwardRef = /*#__PURE__*/ (0, $b15CJ$react.forwardRef)($1991e3eb8a26eb8d$var$SvgComponent);
var $1991e3eb8a26eb8d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1991e3eb8a26eb8d$var$ForwardRef);
