// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowLeft_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $cETCq$reactjsxruntime = require("react/jsx-runtime");
var $cETCq$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $969d452312829731$export$2e2bcd8739ae039);




const $969d452312829731$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cETCq$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cETCq$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M81 50c1.105 0 2-.895 2-2s-.895-2-2-2H18q-.09 0-.18.008l10.594-10.594c.781-.78.781-2.047 0-2.828-.78-.781-2.047-.781-2.828 0l-14 14c-.781.78-.781 2.047 0 2.828l14 14c.78.781 2.047.781 2.828 0 .781-.78.781-2.047 0-2.828L17.82 49.992q.09.008.18.008zm-64.992-1.82Q16 48.09 16 48t.008-.18l-.18.18z"
        })
    });
const $969d452312829731$var$ForwardRef = /*#__PURE__*/ (0, $cETCq$react.forwardRef)($969d452312829731$var$SvgComponent);
var $969d452312829731$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($969d452312829731$var$ForwardRef);
