// @react-spectrum/s2/illustrations/linear/internal/S2_lin_shoppingCart_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5ppUd$jsx} from "react/jsx-runtime";
import {forwardRef as $5ppUd$forwardRef} from "react";





const $ff10d2f048ec3d3c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5ppUd$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5ppUd$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M6.746 7.826c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25l.36.002.62.004c.525.003 1.229.006 2.057.005.873 0 1.612.628 1.752 1.476l2.945 17.86c.339 2.054 2.113 3.56 4.194 3.56h16.169c.69 0 1.25-.56 1.25-1.25 0-.691-.56-1.25-1.25-1.25h-16.17c-.856 0-1.587-.62-1.726-1.466l-.58-3.515H33.59c1.553 0 2.888-1.098 3.189-2.622l1.235-6.26c.395-2.004-1.132-3.874-3.178-3.879-7.024-.017-17.547-.034-20.572-.004l-.261-1.58c-.341-2.069-2.136-3.57-4.22-3.57-.822 0-1.52-.002-2.042-.005l-.6-.004zm7.928 7.657 1.281 7.768H33.59c.358 0 .667-.253.736-.604l1.235-6.261c.091-.465-.262-.894-.731-.895-6.808-.016-16.781-.032-20.155-.008M20.504 39c0 1.38-1.118 2.5-2.5 2.5-1.38 0-2.5-1.12-2.5-2.5s1.12-2.5 2.5-2.5c1.382 0 2.5 1.12 2.5 2.5m12.463 2.5c1.38 0 2.5-1.12 2.5-2.5s-1.12-2.5-2.5-2.5-2.5 1.12-2.5 2.5 1.12 2.5 2.5 2.5"
        })
    });
const $ff10d2f048ec3d3c$var$ForwardRef = /*#__PURE__*/ (0, $5ppUd$forwardRef)($ff10d2f048ec3d3c$var$SvgComponent);
var $ff10d2f048ec3d3c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ff10d2f048ec3d3c$var$ForwardRef);


export {$ff10d2f048ec3d3c$export$2e2bcd8739ae039 as default};
