// @react-spectrum/s2/illustrations/linear/internal/S2_lin_apps_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5ZQLV$jsx} from "react/jsx-runtime";
import {forwardRef as $5ZQLV$forwardRef} from "react";





const $8d5c954dab129624$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5ZQLV$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5ZQLV$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24.11 13c0-1.519 1.232-2.75 2.75-2.75h8.17c1.52 0 2.75 1.231 2.75 2.75v8.17c0 1.519-1.23 2.75-2.75 2.75h-8.17c-1.518 0-2.75-1.231-2.75-2.75zm2.75-5.25c-2.9 0-5.25 2.35-5.25 5.25v8.17c0 2.9 2.35 5.25 5.25 5.25h8.17c2.9 0 5.25-2.35 5.25-5.25V13c0-2.9-2.35-5.25-5.25-5.25zm-16.6 10.68c0-.69.56-1.25 1.25-1.25h4.23c.69 0 1.25.56 1.25 1.25v4.23c0 .69-.56 1.25-1.25 1.25h-4.23c-.69 0-1.25-.56-1.25-1.25zm1.25-3.75c-2.071 0-3.75 1.679-3.75 3.75v4.23c0 2.071 1.679 3.75 3.75 3.75h4.23c2.07 0 3.75-1.679 3.75-3.75v-4.23c0-2.071-1.68-3.75-3.75-3.75zm0 16.34c-.69 0-1.25.56-1.25 1.25v4.23c0 .69.56 1.25 1.25 1.25h4.23c.69 0 1.25-.56 1.25-1.25v-4.23c0-.69-.56-1.25-1.25-1.25zm-3.75 1.25c0-2.071 1.679-3.75 3.75-3.75h4.23c2.07 0 3.75 1.679 3.75 3.75v4.23c0 2.071-1.68 3.75-3.75 3.75h-4.23c-2.071 0-3.75-1.679-3.75-3.75zm16.35 0c0-.69.56-1.25 1.25-1.25h4.23c.69 0 1.25.56 1.25 1.25v4.23c0 .69-.56 1.25-1.25 1.25h-4.23c-.69 0-1.25-.56-1.25-1.25zm1.25-3.75c-2.07 0-3.75 1.679-3.75 3.75v4.23c0 2.071 1.68 3.75 3.75 3.75h4.23c2.071 0 3.75-1.679 3.75-3.75v-4.23c0-2.071-1.679-3.75-3.75-3.75z"
        })
    });
const $8d5c954dab129624$var$ForwardRef = /*#__PURE__*/ (0, $5ZQLV$forwardRef)($8d5c954dab129624$var$SvgComponent);
var $8d5c954dab129624$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8d5c954dab129624$var$ForwardRef);


export {$8d5c954dab129624$export$2e2bcd8739ae039 as default};
