// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noLibrariesBrands_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $jta2I$jsx} from "react/jsx-runtime";
import {forwardRef as $jta2I$forwardRef} from "react";





const $50cd020b1450b73f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jta2I$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jta2I$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M6.965 8.75c-.652 0-1.215.544-1.215 1.253v10.35c0 4.653 3.716 8.397 8.25 8.397 4.535 0 8.25-3.746 8.25-8.396V10.003c0-.709-.563-1.253-1.215-1.253zM3.25 10.003c0-2.053 1.647-3.753 3.715-3.753h14.07c2.068 0 3.715 1.7 3.715 3.753v10.35c0 5.996-4.798 10.897-10.75 10.897-5.95 0-10.75-4.899-10.75-10.896zm14.788 4.359 2.465 4.753c.351.677.306 1.434.001 2.024-.307.592-.939 1.111-1.789 1.111h-4.93c-.85 0-1.482-.52-1.789-1.111-.305-.59-.35-1.347 0-2.024l2.464-4.753c.36-.694 1.044-1.112 1.789-1.112s1.43.418 1.79 1.112m-1.789 1.98-1.766 3.408h3.534zm-4.111-.325c.286-.628.008-1.37-.62-1.655-.63-.286-1.37-.008-1.656.62l-2.5 5.5c-.286.63-.008 1.37.62 1.656.63.286 1.37.008 1.656-.62zm15.112-2.87c-.69 0-1.25.56-1.25 1.25 0 .691.56 1.25 1.25 1.25h1.959v9.226c0 .346.385.553.67.359l2.108-1.406c.147-.096.333-.096.48 0l2.107 1.406c.285.19.67-.013.67-.36v-9.224h.169c1.22 0 2.21.99 2.21 2.211v8.587c0 .69.56 1.25 1.25 1.25.691 0 1.25-.56 1.25-1.25V17.86c0-2.602-2.108-4.711-4.71-4.711H27.25m-9.602 19.99c0-.691-.56-1.25-1.25-1.25s-1.25.559-1.25 1.25v.276c0 2.602 2.109 4.711 4.71 4.711h9.021c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-9.02c-1.222 0-2.212-.99-2.212-2.21zM38.75 43.75c4.073 0 7.375-3.302 7.375-7.375S42.823 29 38.75 29s-7.375 3.302-7.375 7.375 3.302 7.375 7.375 7.375M40.125 33c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v2.25h-2.25c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h2.25V40c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-2.25h2.25c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.25z"
        })
    });
const $50cd020b1450b73f$var$ForwardRef = /*#__PURE__*/ (0, $jta2I$forwardRef)($50cd020b1450b73f$var$SvgComponent);
var $50cd020b1450b73f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($50cd020b1450b73f$var$ForwardRef);


export {$50cd020b1450b73f$export$2e2bcd8739ae039 as default};
