// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noComment_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3fxwH$reactjsxruntime = require("react/jsx-runtime");
var $3fxwH$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $94ac86361f2e839e$export$2e2bcd8739ae039);




const $94ac86361f2e839e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3fxwH$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3fxwH$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M7.988 7.25c-3.062 0-5.613 2.396-5.613 5.49v14.463c0 3.094 2.551 5.49 5.613 5.49h2.793c.006 1.856.009 4.083-.004 5.858-.018 2.345 2.741 3.516 4.465 2.05l.002-.001 9.358-7.907h3.585c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-4.5l-.35.296-9.71 8.204-.004.003c-.044.038-.08.05-.11.053-.037.004-.082-.002-.126-.022s-.072-.047-.087-.069c-.012-.017-.024-.04-.023-.089.016-2.21.008-5.11 0-7.13l-.005-1.246H7.986c-1.757 0-3.112-1.352-3.112-2.99V12.74c0-1.637 1.355-2.99 3.113-2.99h28.274c1.758 0 3.113 1.353 3.113 2.99v6.291c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V12.74c0-3.094-2.551-5.49-5.613-5.49zm22.637 21.686c0-4.142 3.359-7.5 7.5-7.5s7.5 3.358 7.5 7.5c0 4.14-3.359 7.5-7.5 7.5s-7.5-3.36-7.5-7.5m10.859 3.358c.485-.485.485-1.268 0-1.753h.02l-1.606-1.605 1.605-1.605c.486-.486.486-1.269 0-1.754-.485-.486-1.268-.486-1.753 0l-1.605 1.605-1.605-1.605c-.486-.486-1.268-.486-1.754 0-.485.485-.485 1.268 0 1.753l1.605 1.606-1.605 1.605c-.485.485-.485 1.268 0 1.753.238.248.555.367.872.367s.634-.129.872-.367l1.605-1.605 1.605 1.605c.238.248.555.367.872.367s.634-.129.872-.367"
        })
    });
const $94ac86361f2e839e$var$ForwardRef = /*#__PURE__*/ (0, $3fxwH$react.forwardRef)($94ac86361f2e839e$var$SvgComponent);
var $94ac86361f2e839e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($94ac86361f2e839e$var$ForwardRef);
