// @react-spectrum/s2/illustrations/linear/internal/S2_lin_megaphonePromote_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $k5cod$reactjsxruntime = require("react/jsx-runtime");
var $k5cod$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $639d3f925d96cd0b$export$2e2bcd8739ae039);




const $639d3f925d96cd0b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $k5cod$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $k5cod$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M135.96 41.009c0-9.391-9.068-16.128-18.06-13.399h-.001l-.002.001L75.276 40.53H44.98c-13.79 0-24.98 11.19-24.98 24.98 0 13.789 11.19 24.98 24.98 24.98h1.397l1.043 2.39 4.781 10.933c3.587 8.196 7.508 17.136 8.79 19.989 2.035 4.523 5.504 7.323 9.478 8.439 3.865 1.086 7.952.514 11.358-1.141 3.178-1.544 5.992-4.141 7.537-7.598 1.582-3.54 1.701-7.7-.1-11.958l-.016-.039-8.421-19.343 37.06 11.265c8.986 2.729 18.073-3.989 18.073-13.397zm-64 7.52v33.96H44.98c-9.37 0-16.98-7.61-16.98-16.98s7.61-16.98 16.98-16.98zm8 35.007 40.252 12.235h.001c3.854 1.17 7.747-1.712 7.747-5.742v-49.02c0-4.029-3.89-6.912-7.738-5.743h-.002L79.96 47.469zm-24.848 6.965 4.418 10.104c3.597 8.218 7.496 17.109 8.758 19.916 1.08 2.403 2.695 3.554 4.343 4.017 1.757.493 3.834.272 5.7-.635 1.732-.841 3.057-2.16 3.73-3.666.633-1.419.81-3.26-.157-5.56L71.379 90.501z"
        })
    });
const $639d3f925d96cd0b$var$ForwardRef = /*#__PURE__*/ (0, $k5cod$react.forwardRef)($639d3f925d96cd0b$var$SvgComponent);
var $639d3f925d96cd0b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($639d3f925d96cd0b$var$ForwardRef);
