// @react-spectrum/s2/illustrations/linear/internal/S2_lin_lockClose_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $l73q2$jsx} from "react/jsx-runtime";
import {forwardRef as $l73q2$forwardRef} from "react";





const $2b37b51a5ad8b976$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $l73q2$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $l73q2$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.75 13.75c0-4.142 3.358-7.5 7.5-7.5s7.5 3.358 7.5 7.5v5.5H34c2.9 0 5.25 2.35 5.25 5.25V35c0 2.9-2.35 5.25-5.25 5.25H14c-2.9 0-5.25-2.35-5.25-5.25V24.5c0-2.9 2.35-5.25 5.25-5.25h2.75zm2.5 5.5h10v-5.5c0-2.761-2.239-5-5-5s-5 2.239-5 5zm11.25 2.5H14c-1.519 0-2.75 1.231-2.75 2.75V35c0 1.519 1.231 2.75 2.75 2.75h20c1.519 0 2.75-1.231 2.75-2.75V24.5c0-1.519-1.231-2.75-2.75-2.75zm-7.75 7.916c-.747-.433-1.25-1.24-1.25-2.166 0-1.38 1.12-2.5 2.5-2.5s2.5 1.12 2.5 2.5c0 .925-.503 1.733-1.25 2.166V32.5c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25z"
        })
    });
const $2b37b51a5ad8b976$var$ForwardRef = /*#__PURE__*/ (0, $l73q2$forwardRef)($2b37b51a5ad8b976$var$SvgComponent);
var $2b37b51a5ad8b976$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2b37b51a5ad8b976$var$ForwardRef);


export {$2b37b51a5ad8b976$export$2e2bcd8739ae039 as default};
