// @react-spectrum/s2/illustrations/linear/internal/S2_lin_prompt_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $1CiXB$jsx} from "react/jsx-runtime";
import {forwardRef as $1CiXB$forwardRef} from "react";





const $cb7774d53cea99e2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1CiXB$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1CiXB$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24.234 58.04c0-11.047 8.955-20 20-20h71.532c11.045 0 20 8.953 20 20v43.922c0 11.046-8.955 20-20 20H44.234c-11.045 0-20-8.954-20-20zm20-12c-6.627 0-12 5.372-12 12v43.922c0 6.627 5.373 12 12 12h71.532c6.627 0 12-5.373 12-12V58.039c0-6.627-5.373-12-12-12zm24.864 54.666c0-2.21 1.79-4 4-4h41.412c2.209 0 4 1.79 4 4s-1.791 4-4 4H73.098c-2.21 0-4-1.791-4-4m-26.684 1.12c-1.383 1.723-1.107 4.241.615 5.624 1.723 1.382 4.24 1.107 5.624-.616l19.67-24.508c1.181-1.471 1.174-3.567-.017-5.03L48.635 53.144c-1.395-1.713-3.915-1.97-5.628-.575s-1.97 3.915-.575 5.627L60.06 79.841z"
        })
    });
const $cb7774d53cea99e2$var$ForwardRef = /*#__PURE__*/ (0, $1CiXB$forwardRef)($cb7774d53cea99e2$var$SvgComponent);
var $cb7774d53cea99e2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cb7774d53cea99e2$var$ForwardRef);


export {$cb7774d53cea99e2$export$2e2bcd8739ae039 as default};
