// @react-spectrum/s2/illustrations/linear/internal/S2_lin_graphBarChart_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $7XN33$reactjsxruntime = require("react/jsx-runtime");
var $7XN33$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3a8d73260d0b7eb5$export$2e2bcd8739ae039);




const $3a8d73260d0b7eb5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7XN33$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7XN33$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.21 15.09c0-1.105-.895-2-2-2s-2 .895-2 2V82c0 1.104.896 2 2 2H81c1.106 0 2-.896 2-2s-.894-2-2-2H16.21zm9.111 31.14c0-1.105.896-2 2-2h3.54c1.105 0 2 .895 2 2v22.54c0 1.104-.895 2-2 2h-3.54c-1.104 0-2-.896-2-2zm2-6c-3.313 0-6 2.686-6 6v22.54c0 3.314 2.687 6 6 6h3.54c3.314 0 6-2.686 6-6V46.23c0-3.314-2.686-6-6-6zm41.53 18.39c-1.104 0-2 .895-2 2v8.15c0 1.104.896 2 2 2h3.54c1.105 0 2-.896 2-2v-8.15c0-1.105-.895-2-2-2zm-6 2c0-3.314 2.687-6 6-6h3.54c3.314 0 6 2.686 6 6v8.15c0 3.314-2.686 6-6 6h-3.54c-3.313 0-6-2.686-6-6zm-16.77-32.3c0-1.105.896-2 2-2h3.54c1.105 0 2 .895 2 2v40.45c0 1.104-.895 2-2 2h-3.54c-1.104 0-2-.896-2-2zm2-6c-3.313 0-6 2.686-6 6v40.45c0 3.314 2.687 6 6 6h3.54c3.314 0 6-2.686 6-6V28.32c0-3.314-2.686-6-6-6z"
        })
    });
const $3a8d73260d0b7eb5$var$ForwardRef = /*#__PURE__*/ (0, $7XN33$react.forwardRef)($3a8d73260d0b7eb5$var$SvgComponent);
var $3a8d73260d0b7eb5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3a8d73260d0b7eb5$var$ForwardRef);
