// @react-spectrum/s2/illustrations/linear/internal/S2_lin_properties_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7H5o6$jsx} from "react/jsx-runtime";
import {forwardRef as $7H5o6$forwardRef} from "react";





const $214ec4685dd30ab9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7H5o6$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7H5o6$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M19.594 14c-1.518 0-2.75 1.231-2.75 2.75s1.232 2.75 2.75 2.75 2.75-1.231 2.75-2.75S21.113 14 19.594 14M9.125 15.5h5.369c.56-2.296 2.631-4 5.1-4s4.54 1.704 5.1 4h14.181c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-14.18c-.561 2.296-2.632 4-5.1 4-2.47 0-4.54-1.704-5.101-4H9.125c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25m14.18 14.499c.561-2.296 2.632-4 5.101-4s4.54 1.704 5.1 4h5.37c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-5.37c-.56 2.296-2.631 4-5.1 4s-4.54-1.704-5.1-4H9.126c-.691 0-1.25-.56-1.25-1.25s.559-1.25 1.25-1.25zm7.851 1.25c0-1.519-1.231-2.75-2.75-2.75s-2.75 1.231-2.75 2.75 1.231 2.75 2.75 2.75 2.75-1.231 2.75-2.75"
        })
    });
const $214ec4685dd30ab9$var$ForwardRef = /*#__PURE__*/ (0, $7H5o6$forwardRef)($214ec4685dd30ab9$var$SvgComponent);
var $214ec4685dd30ab9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($214ec4685dd30ab9$var$ForwardRef);


export {$214ec4685dd30ab9$export$2e2bcd8739ae039 as default};
