// @react-spectrum/s2/illustrations/linear/internal/S2_lin_search_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $f0EG3$reactjsxruntime = require("react/jsx-runtime");
var $f0EG3$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3b4eb1952a439411$export$2e2bcd8739ae039);




const $3b4eb1952a439411$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $f0EG3$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $f0EG3$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M39 14.077c-13.807 0-25 11.193-25 25s11.193 25 25 25c6.184 0 11.843-2.245 16.207-5.965l1.491 1.491-1.421 1.422c-.781.78-.781 2.047 0 2.828l17.129 17.129c2.33 2.33 6.108 2.33 8.439 0 2.33-2.33 2.33-6.109 0-8.439l-17.13-17.129c-.78-.781-2.047-.781-2.828 0l-1.36 1.36-1.491-1.49C61.755 50.92 64 45.26 64 39.077c0-13.807-11.193-25-25-25m-21 25c0-11.598 9.402-21 21-21s21 9.402 21 21-9.402 21-21 21-21-9.402-21-21m44.302 20.58-2.782 2.782 15.714 15.714c.769.769 2.014.769 2.782 0 .769-.768.769-2.013 0-2.782z"
        })
    });
const $3b4eb1952a439411$var$ForwardRef = /*#__PURE__*/ (0, $f0EG3$react.forwardRef)($3b4eb1952a439411$var$SvgComponent);
var $3b4eb1952a439411$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3b4eb1952a439411$var$ForwardRef);
