// @react-spectrum/s2/illustrations/linear/internal/S2_lin_AIGenerateImage_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $4n1bD$reactjsxruntime = require("react/jsx-runtime");
var $4n1bD$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $8634e9b86635065d$export$2e2bcd8739ae039);




const $8634e9b86635065d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4n1bD$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4n1bD$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M38.607 3.474c1.945-1.793 5.037-.005 4.457 2.574l-.701 3.12c-.014.06.003.127.046.174l2.166 2.35c1.792 1.944.005 5.036-2.574 4.457l-3.12-.702c-.06-.013-.127.004-.174.047l-2.35 2.165c-1.944 1.794-5.036.006-4.457-2.573l.701-3.12c.014-.061-.003-.127-.046-.174l-2.166-2.35c-1.793-1.945-.005-5.037 2.574-4.458l3.12.702c.06.014.127-.003.174-.047zm1.804 1.78c-.02.002-.056.008-.11.058l-2.35 2.165c-.65.6-1.556.843-2.42.648l-3.116-.701c-.072-.016-.106-.004-.124.005-.027.013-.062.04-.09.088s-.034.091-.031.121c.001.02.007.055.057.11l2.165 2.35c.6.65.843 1.555.648 2.42l-.701 3.115c-.016.073-.004.107.005.125.013.027.041.062.088.09.047.026.092.033.121.031.02-.002.056-.008.11-.058l2.35-2.165c.65-.6 1.556-.842 2.42-.647l3.116.7c.072.017.106.004.124-.004.027-.013.062-.041.09-.088.027-.048.034-.092.031-.122-.001-.02-.007-.055-.057-.11l-2.165-2.348c-.6-.65-.843-1.557-.648-2.42v-.002l.701-3.115c.016-.072.004-.106-.005-.124-.013-.027-.041-.062-.088-.09s-.092-.034-.121-.031"
            }),
            /*#__PURE__*/ (0, $4n1bD$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "m28.884 22.648 2.197.495c.864.195 1.464-.842.862-1.495l-1.527-1.655c-.2-.219-.282-.52-.217-.811l.495-2.198c.196-.864-.842-1.464-1.494-.862l-1.656 1.527c-.218.2-.52.282-.811.217l-2.198-.495c-.864-.195-1.464.842-.862 1.494l1.527 1.656c.2.218.282.52.217.811l-.495 2.198c-.195.864.842 1.464 1.494.862l1.656-1.527c.219-.2.52-.282.812-.217"
            }),
            /*#__PURE__*/ (0, $4n1bD$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M18.505 10.993c-5.828-1.562-11.818 1.896-13.38 7.724-.976 3.646.012 7.358 2.319 10.008l.067-1.763c.026-.69.606-1.227 1.296-1.201s1.228.606 1.202 1.296l-.18 4.774c-.026.673-.58 1.205-1.254 1.202l-4.948-.015c-.69-.002-1.248-.564-1.246-1.254s.564-1.248 1.254-1.246l2.063.006C2.767 27.261 1.49 22.626 2.711 18.07c1.919-7.161 9.28-11.41 16.44-9.492 3.108.832 5.669 2.692 7.413 5.113.404.56.277 1.341-.283 1.745-.56.403-1.341.276-1.745-.284-1.42-1.97-3.501-3.481-6.031-4.16"
            }),
            /*#__PURE__*/ (0, $4n1bD$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                fillRule: "evenodd",
                d: "M41.57 25.131c0-1.518-1.232-2.75-2.75-2.75h-4.032c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h4.031c2.9 0 5.25 2.35 5.25 5.25v14.474c0 2.9-2.35 5.25-5.25 5.25H17.017c-2.9 0-5.25-2.35-5.25-5.25V25.131c0-2.9 2.35-5.25 5.25-5.25h4.505c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-4.505c-1.519 0-2.75 1.232-2.75 2.75v14.474q0 .123.01.243l5.85-9.038c1.174-1.815 3.832-1.815 5.006 0l4.81 7.429 2.614-3.953.006-.007c1.18-1.757 3.769-1.761 4.95 0l3.993 5.915q.062-.285.063-.589zm-19.344 7.037-6.414 9.91c.364.178.772.277 1.205.277H27.22l1.217-1.84-5.403-8.347c-.189-.292-.62-.292-.809 0m17.596 9.999c-.311.122-.65.188-1.003.188h-8.601l4.421-6.684c.194-.284.61-.28.798 0l.002.004z"
            }),
            /*#__PURE__*/ (0, $4n1bD$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
                d: "M33.617 27.063c0 .828.671 1.5 1.5 1.5.828 0 1.5-.672 1.5-1.5 0-.829-.672-1.5-1.5-1.5-.829 0-1.5.671-1.5 1.5"
            })
        ]
    });
const $8634e9b86635065d$var$ForwardRef = /*#__PURE__*/ (0, $4n1bD$react.forwardRef)($8634e9b86635065d$var$SvgComponent);
var $8634e9b86635065d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($8634e9b86635065d$var$ForwardRef);
