// @react-spectrum/s2/illustrations/linear/internal/S2_lin_3D_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $ZCZOf$jsx} from "react/jsx-runtime";
import {forwardRef as $ZCZOf$forwardRef} from "react";





const $2a9a7548eda36bb4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ZCZOf$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $ZCZOf$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M70.118 24.051c6.281-3.686 14.07-3.667 20.333.05l34.137 20.267c6.07 3.603 9.791 10.139 9.791 17.197v38.061c0 7.246-3.919 13.925-10.245 17.46l-34.135 19.072c-6.033 3.37-13.377 3.388-19.426.047L35.951 117.08c-6.373-3.52-10.33-10.225-10.33-17.506V61.619c0-7.095 3.758-13.658 9.877-17.25zm16.25 6.93c-3.759-2.23-8.432-2.242-12.2-.03l-34.62 20.318c-.866.508-1.652 1.114-2.349 1.8l42.813 22.848 43.037-22.648c-.743-.776-1.596-1.46-2.544-2.022zM33.62 61.618q0-.681.076-1.35l42.59 22.73v47.03q-.95-.331-1.846-.826L39.82 110.078c-3.824-2.112-6.198-6.136-6.198-10.504zm50.666 68.379q.931-.333 1.81-.823l34.135-19.072c3.795-2.121 6.147-6.128 6.147-10.476v-38.06q0-.498-.041-.988l-42.05 22.129z"
        })
    });
const $2a9a7548eda36bb4$var$ForwardRef = /*#__PURE__*/ (0, $ZCZOf$forwardRef)($2a9a7548eda36bb4$var$SvgComponent);
var $2a9a7548eda36bb4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2a9a7548eda36bb4$var$ForwardRef);


export {$2a9a7548eda36bb4$export$2e2bcd8739ae039 as default};
