// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pause_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $e73s1$reactjsxruntime = require("react/jsx-runtime");
var $e73s1$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $430477958c2d4f46$export$2e2bcd8739ae039);




const $430477958c2d4f46$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e73s1$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $e73s1$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25 27c0-3.314 2.686-6 6-6h8c3.314 0 6 2.686 6 6v42c0 3.314-2.686 6-6 6h-8c-3.314 0-6-2.686-6-6zm6-2c-1.105 0-2 .895-2 2v42c0 1.105.895 2 2 2h8c1.105 0 2-.895 2-2V27c0-1.105-.895-2-2-2zm21 2c0-3.314 2.686-6 6-6h8c3.314 0 6 2.686 6 6v42c0 3.314-2.686 6-6 6h-8c-3.314 0-6-2.686-6-6zm6-2c-1.105 0-2 .895-2 2v42c0 1.105.895 2 2 2h8c1.105 0 2-.895 2-2V27c0-1.105-.895-2-2-2z"
        })
    });
const $430477958c2d4f46$var$ForwardRef = /*#__PURE__*/ (0, $e73s1$react.forwardRef)($430477958c2d4f46$var$SvgComponent);
var $430477958c2d4f46$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($430477958c2d4f46$var$ForwardRef);
