// @react-spectrum/s2/illustrations/linear/internal/S2_lin_document_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $ioPDv$jsx} from "react/jsx-runtime";
import {forwardRef as $ioPDv$forwardRef} from "react";





const $5d4af78f6847b601$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ioPDv$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $ioPDv$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48.136 24C39.267 24 32 31.12 32 40v80c0 8.879 7.267 16 16.136 16h64.728c8.869 0 16.136-7.121 16.136-16V72.97c0-4.258-1.711-8.332-4.742-11.33L90.913 28.67c-3.03-2.995-7.13-4.67-11.395-4.67zM40 40c0-4.376 3.6-8 8.136-8h31.382q.243 0 .482.014V62c0 5.615 4.638 10 10.15 10h30.79q.06.481.06.97V120c0 4.376-3.6 8-8.136 8H48.136C43.6 128 40 124.376 40 120zm75.265 24L88 37.04V62c0 1.012.869 2 2.15 2z"
        })
    });
const $5d4af78f6847b601$var$ForwardRef = /*#__PURE__*/ (0, $ioPDv$forwardRef)($5d4af78f6847b601$var$SvgComponent);
var $5d4af78f6847b601$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5d4af78f6847b601$var$ForwardRef);


export {$5d4af78f6847b601$export$2e2bcd8739ae039 as default};
