// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noComment_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5wHK5$jsx} from "react/jsx-runtime";
import {forwardRef as $5wHK5$forwardRef} from "react";





const $b2b2e2e8c27625c2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5wHK5$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5wHK5$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.516 16c-5.839 0-10.641 4.615-10.641 10.5v28c0 5.885 4.802 10.5 10.64 10.5h5.368c.012 3.816.02 8.529-.008 12.251-.03 4.207 4.916 6.427 8.07 3.715l.002-.003L48.614 65h7.292c1.105 0 2-.895 2-2s-.895-2-2-2h-8.77l-.56.48-19.234 16.448-.006.005c-.592.51-1.466.065-1.461-.652.032-4.429.017-10.238 0-14.29L25.867 61h-9.352c-3.705 0-6.64-2.9-6.64-6.5v-28c0-3.6 2.935-6.5 6.64-6.5h54.72c3.705 0 6.64 2.9 6.64 6.5v12.531c0 1.105.895 2 2 2s2-.895 2-2V26.5c0-5.885-4.802-10.5-10.64-10.5zm44.109 42.5c0-8.147 6.603-14.75 14.75-14.75s14.75 6.603 14.75 14.75-6.603 14.75-14.75 14.75-14.75-6.603-14.75-14.75m21.043 6.293c.781-.782.781-2.054 0-2.836h.02L78.23 58.5l3.457-3.457c.781-.782.781-2.054 0-2.836-.782-.781-2.054-.781-2.836 0l-3.457 3.457-3.457-3.457c-.782-.781-2.054-.781-2.836 0-.781.782-.781 2.054 0 2.836L72.56 58.5l-3.457 3.457c-.781.782-.781 2.054 0 2.836.391.39.902.591 1.413.591s1.022-.2 1.413-.591l3.457-3.457 3.457 3.457c.39.39.902.591 1.413.591s1.022-.2 1.413-.591"
        })
    });
const $b2b2e2e8c27625c2$var$ForwardRef = /*#__PURE__*/ (0, $5wHK5$forwardRef)($b2b2e2e8c27625c2$var$SvgComponent);
var $b2b2e2e8c27625c2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b2b2e2e8c27625c2$var$ForwardRef);


export {$b2b2e2e8c27625c2$export$2e2bcd8739ae039 as default};
