// @react-spectrum/s2/illustrations/linear/internal/S2_lin_revert_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $i16mA$reactjsxruntime = require("react/jsx-runtime");
var $i16mA$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1654e4ecb1a70d66$export$2e2bcd8739ae039);




const $1654e4ecb1a70d66$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $i16mA$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $i16mA$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m32.523 90.528 15.749-13.24c1.69-1.422 4.214-1.203 5.636.488 1.421 1.69 1.203 4.214-.488 5.635L30.56 102.63c-1.691 1.422-4.215 1.204-5.636-.487L5.706 79.282c-1.422-1.692-1.203-4.215.488-5.636 1.69-1.422 4.214-1.204 5.635.487l12.744 15.159c1.734-31.592 27.9-56.675 59.925-56.675 33.147 0 60.018 26.87 60.018 60.018 0 2.209-1.791 4-4 4s-4-1.791-4-4c0-28.729-23.289-52.018-52.018-52.018-28.022 0-50.87 22.159-51.975 49.91M21.54 123.383c0-2.209 1.79-4 4-4h117.885c2.209 0 4 1.791 4 4s-1.791 4-4 4H25.54c-2.21 0-4-1.791-4-4"
        })
    });
const $1654e4ecb1a70d66$var$ForwardRef = /*#__PURE__*/ (0, $i16mA$react.forwardRef)($1654e4ecb1a70d66$var$SvgComponent);
var $1654e4ecb1a70d66$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1654e4ecb1a70d66$var$ForwardRef);
