// @react-spectrum/s2/illustrations/linear/internal/S2_lin_user_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7C9DJ$jsx} from "react/jsx-runtime";
import {forwardRef as $7C9DJ$forwardRef} from "react";





const $44dc1158a7fba0a6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7C9DJ$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7C9DJ$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M61 48.5C61 36.936 69.741 28 80 28s19 8.936 19 20.5S90.26 69 80 69s-19-8.936-19-20.5M80 20c-15.146 0-27 13.002-27 28.5S64.854 77 80 77s27-13.002 27-28.5S95.146 20 80 20M37.95 123.051C42.115 103.693 59.506 89 80 89s37.885 14.693 42.049 34.051c.558 2.594-1.372 4.939-3.899 4.939h-76.3c-2.527 0-4.458-2.345-3.9-4.939M80 81c-24.367 0-44.935 17.427-49.87 40.369-1.603 7.446 3.967 14.621 11.72 14.621h76.3c7.753 0 13.322-7.175 11.72-14.621C124.935 98.427 104.366 81 80 81"
        })
    });
const $44dc1158a7fba0a6$var$ForwardRef = /*#__PURE__*/ (0, $7C9DJ$forwardRef)($44dc1158a7fba0a6$var$SvgComponent);
var $44dc1158a7fba0a6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($44dc1158a7fba0a6$var$ForwardRef);


export {$44dc1158a7fba0a6$export$2e2bcd8739ae039 as default};
