// @react-spectrum/s2/illustrations/linear/internal/S2_lin_vectorDraw_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $avRMS$reactjsxruntime = require("react/jsx-runtime");
var $avRMS$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2f60e19f1caad54e$export$2e2bcd8739ae039);




const $2f60e19f1caad54e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $avRMS$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $avRMS$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M33.723 7.34c-1.647-1.646-4.274-1.32-5.854.259l-2.273 2.273-11.263 4.808c-1.244.53-2.22 1.604-2.62 2.895l-5.2 16.8c-1.353 4.37 2.744 8.466 7.114 7.114l16.8-5.2c1.29-.4 2.364-1.377 2.895-2.62l4.808-11.264 2.273-2.273c1.579-1.579 1.905-4.207.258-5.854zM15.315 16.98l10.708-4.571 9.57 9.57-4.57 10.709c-.247.577-.755 1.034-1.336 1.214l-16.8 5.2c-.72.223-1.427.181-2.047-.05l10.05-10.01q.286.07.593.07c1.38 0 2.5-1.118 2.5-2.5 0-1.38-1.12-2.5-2.5-2.5s-2.5 1.12-2.5 2.5q.001.368.1.702L9.02 37.333c-.292-.659-.364-1.43-.12-2.219l5.2-16.8c.18-.58.636-1.089 1.214-1.335m23.32 1.386-1.56 1.56-8.998-8.998 1.56-1.56c.82-.82 1.851-.726 2.318-.259l6.938 6.938c.467.467.562 1.499-.258 2.319"
        })
    });
const $2f60e19f1caad54e$var$ForwardRef = /*#__PURE__*/ (0, $avRMS$react.forwardRef)($2f60e19f1caad54e$var$SvgComponent);
var $2f60e19f1caad54e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2f60e19f1caad54e$var$ForwardRef);
