// @react-spectrum/s2/illustrations/linear/internal/S2_lin_artboard_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9uQhd$jsx} from "react/jsx-runtime";
import {forwardRef as $9uQhd$forwardRef} from "react";





const $fa99753c29f85da6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9uQhd$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9uQhd$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M13.252 6.217c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v2.82c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25zm-7.455 4.956c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h2.82c.691 0 1.25-.56 1.25-1.25s-.559-1.25-1.25-1.25zm9.59 2.5c-1.18 0-2.135.955-2.135 2.134v22.566c0 1.18.956 2.135 2.135 2.135H33.44c1.178 0 2.134-.956 2.134-2.135V24.955h-8.058c-1.936 0-3.506-1.57-3.506-3.506v-7.777zm11.123 1.416v6.36c0 .556.45 1.006 1.006 1.006h6.647zm-15.758.718c0-2.56 2.075-4.635 4.635-4.635h8.79c1.199 0 2.35.465 3.214 1.296l9.263 8.915c.907.874 1.42 2.08 1.42 3.34v13.65c0 2.56-2.075 4.635-4.634 4.635H15.387c-2.56 0-4.635-2.075-4.635-4.635z"
        })
    });
const $fa99753c29f85da6$var$ForwardRef = /*#__PURE__*/ (0, $9uQhd$forwardRef)($fa99753c29f85da6$var$SvgComponent);
var $fa99753c29f85da6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($fa99753c29f85da6$var$ForwardRef);


export {$fa99753c29f85da6$export$2e2bcd8739ae039 as default};
