// @react-spectrum/s2/illustrations/linear/internal/S2_lin_segmentation_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $3g3Y3$jsx} from "react/jsx-runtime";
import {forwardRef as $3g3Y3$forwardRef} from "react";





const $fe5762f1abcf7943$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3g3Y3$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3g3Y3$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M52.773 40.272c0-8.837 7.163-16 16-16h50.977c8.837 0 16 7.163 16 16v15.3c0 8.837-7.163 16-16 16H68.772c-8.836 0-16-7.163-16-16v-3.697H40.25c-4.418 0-8 3.582-8 8v40.2c0 4.418 3.582 8 8 8h12.523v-3.648c0-8.837 7.163-16 16-16h50.977c8.837 0 16 7.163 16 16v15.3c0 8.837-7.163 16-16 16H68.772c-8.836 0-16-7.163-16-16v-3.652H40.25c-8.837 0-16-7.163-16-16v-40.2c0-8.837 7.163-16 16-16h12.523zm16-8c-4.419 0-8 3.581-8 8v15.3c0 4.418 3.581 8 8 8h50.977c4.418 0 8-3.582 8-8v-15.3c0-4.419-3.582-8-8-8zm-8 72.155c0-4.419 3.581-8 8-8h50.977c4.418 0 8 3.581 8 8v15.3c0 4.418-3.582 8-8 8H68.772c-4.418 0-8-3.581-8-8z"
        })
    });
const $fe5762f1abcf7943$var$ForwardRef = /*#__PURE__*/ (0, $3g3Y3$forwardRef)($fe5762f1abcf7943$var$SvgComponent);
var $fe5762f1abcf7943$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($fe5762f1abcf7943$var$ForwardRef);


export {$fe5762f1abcf7943$export$2e2bcd8739ae039 as default};
