// @react-spectrum/s2/illustrations/linear/internal/S2_lin_microphoneOff_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $73N7F$reactjsxruntime = require("react/jsx-runtime");
var $73N7F$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2d5476167a2d5a59$export$2e2bcd8739ae039);




const $2d5476167a2d5a59$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $73N7F$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $73N7F$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8.93 7.147c-.487-.488-1.279-.488-1.767 0-.488.489-.488 1.28 0 1.768l31.908 31.908c.488.489 1.28.489 1.768 0 .488-.488.488-1.28 0-1.767zm1.766 14.857c0 6.892 5.272 12.553 12.002 13.17v3.545h-5.736c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h13.973c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-5.737v-3.55c.816-.076 1.845-.245 2.922-.604.655-.218 1.01-.926.79-1.58-.217-.656-.925-1.01-1.58-.791-1.423.474-2.77.536-3.407.536-5.924 0-10.727-4.802-10.727-10.726v-1.497c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25zM19.07 8.648c1.15-1.094 2.759-1.851 4.88-1.851 2.77 0 4.631 1.264 5.758 2.83 1.084 1.507 1.48 3.268 1.48 4.416v.01c-.018 2.392-.005 6.5 0 7.746v.36c0 .788-.169 1.543-.398 2.202-.226.652-.939.997-1.59.77-.653-.226-.998-.938-.771-1.59.163-.471.26-.944.26-1.381l-.002-.328c-.004-1.205-.017-5.373.001-7.793-.001-.686-.267-1.92-1.009-2.951-.7-.974-1.834-1.791-3.73-1.791-1.471 0-2.465.505-3.156 1.162-.716.682-1.162 1.583-1.394 2.406-.187.665-.878 1.052-1.542.865-.665-.187-1.052-.878-.864-1.542.318-1.132.953-2.47 2.077-3.54m.368 14.855c-.226-.652-.938-.998-1.59-.772s-.998.938-.772 1.59c.634 1.828 2.259 3.843 4.637 4.585.659.206 1.36-.162 1.565-.821.206-.659-.162-1.36-.82-1.565-1.48-.462-2.593-1.788-3.02-3.017m17.739-2.996c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25c0 4.015-.578 5.68-1.6 7.16-.391.567-.249 1.346.32 1.738.568.392 1.346.25 1.738-.319 1.418-2.053 2.041-4.287 2.041-8.58"
        })
    });
const $2d5476167a2d5a59$var$ForwardRef = /*#__PURE__*/ (0, $73N7F$react.forwardRef)($2d5476167a2d5a59$var$SvgComponent);
var $2d5476167a2d5a59$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2d5476167a2d5a59$var$ForwardRef);
