// @react-spectrum/s2/illustrations/linear/internal/S2_lin_play_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6jbyv$reactjsxruntime = require("react/jsx-runtime");
var $6jbyv$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f319d1c9153c1fd9$export$2e2bcd8739ae039);




const $f319d1c9153c1fd9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6jbyv$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6jbyv$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28 79.74c0-28.719 23.281-52 52-52s52 23.281 52 52-23.281 52-52 52-52-23.281-52-52m52-60c-33.137 0-60 26.863-60 60s26.863 60 60 60 60-26.863 60-60-26.863-60-60-60m-5.189 44.94L95.744 76.8c2.344 1.357 2.34 4.743-.007 6.095L74.775 94.964c-2.348 1.351-5.278-.345-5.275-3.054l.029-24.188c.003-2.71 2.938-4.399 5.282-3.041"
        })
    });
const $f319d1c9153c1fd9$var$ForwardRef = /*#__PURE__*/ (0, $6jbyv$react.forwardRef)($f319d1c9153c1fd9$var$SvgComponent);
var $f319d1c9153c1fd9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f319d1c9153c1fd9$var$ForwardRef);
