// @react-spectrum/s2/illustrations/linear/internal/S2_lin_link_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $kxQ5x$jsx} from "react/jsx-runtime";
import {forwardRef as $kxQ5x$forwardRef} from "react";





const $c1569e542214e3fe$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kxQ5x$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kxQ5x$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M74.26 34.83c3.653-3.62 3.653-9.485 0-13.107-3.66-3.63-9.604-3.63-13.266 0l-6.54 6.486c-.785.778-2.051.772-2.83-.012-.777-.784-.772-2.05.013-2.828l6.54-6.486c5.222-5.177 13.68-5.177 18.9 0 5.23 5.186 5.23 13.602 0 18.788L60.491 54.117c-5.22 5.177-13.678 5.177-18.9 0-.784-.777-.789-2.044-.011-2.828.778-.785 2.044-.79 2.828-.012 3.662 3.63 9.605 3.63 13.267 0zm-19.852 7.053c.785.777.79 2.044.012 2.828-.778.785-2.044.79-2.828.012-3.662-3.63-9.605-3.63-13.267 0L21.74 61.17c-3.652 3.622-3.652 9.486 0 13.108 3.662 3.63 9.605 3.63 13.267 0l6.54-6.486c.785-.778 2.051-.772 2.83.012.777.784.772 2.05-.013 2.828l-6.54 6.486c-5.222 5.177-13.68 5.177-18.9 0l1.397-1.409-1.397 1.41c-5.23-5.187-5.23-13.603 0-18.789l16.586-16.446c5.22-5.177 13.678-5.177 18.9 0"
        })
    });
const $c1569e542214e3fe$var$ForwardRef = /*#__PURE__*/ (0, $kxQ5x$forwardRef)($c1569e542214e3fe$var$SvgComponent);
var $c1569e542214e3fe$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c1569e542214e3fe$var$ForwardRef);


export {$c1569e542214e3fe$export$2e2bcd8739ae039 as default};
