// @react-spectrum/s2/illustrations/linear/internal/S2_lin_microphoneOff_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1gzkc$reactjsxruntime = require("react/jsx-runtime");
var $1gzkc$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $42fa96cb448ffb1e$export$2e2bcd8739ae039);




const $42fa96cb448ffb1e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1gzkc$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 97,
        height: 96,
        viewBox: "0 0 97 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1gzkc$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M38.525 18.239c2.227-2.432 5.517-4.223 10.071-4.223 5.375 0 8.957 2.443 11.13 5.462 2.103 2.923 2.87 6.347 2.87 8.553v.015c-.036 4.801-.01 13.044-.002 15.535v.047l.002.669c0 2.009-.576 3.939-1.267 5.51-.445 1.01-1.625 1.469-2.636 1.024s-1.47-1.626-1.025-2.637c.552-1.253.928-2.62.928-3.897l-.002-.664c-.008-2.423-.034-10.764.002-15.609-.002-1.466-.56-4.046-2.117-6.21-1.49-2.07-3.908-3.798-7.883-3.798-3.385 0-5.619 1.284-7.12 2.924-1.553 1.697-2.392 3.86-2.72 5.589-.206 1.085-1.253 1.797-2.338 1.591s-1.798-1.253-1.592-2.338c.43-2.26 1.524-5.167 3.7-7.543m-20.388-3.638c-.781-.78-2.047-.78-2.829 0-.78.781-.78 2.048 0 2.829L79.25 81.37c.78.781 2.047.781 2.828 0s.781-2.047 0-2.828zm4.407 29.383c0 13.705 10.603 24.932 24.052 25.928v8.072h-12c-1.104 0-2 .896-2 2s.896 2 2 2h28c1.105 0 2-.895 2-2s-.895-2-2-2h-12v-8.08c1.957-.153 4.664-.57 7.426-1.676 1.026-.41 1.525-1.574 1.115-2.6-.41-1.025-1.575-1.524-2.6-1.114-3.273 1.31-6.532 1.47-7.993 1.47-12.15 0-22-9.85-22-22v-3c0-1.104-.896-2-2-2s-2 .896-2 2zm16.183 1.644c-.11-1.099-1.09-1.9-2.19-1.79s-1.9 1.09-1.79 2.19c.245 2.437 1.523 5.162 3.492 7.355 1.992 2.218 4.832 4.048 8.326 4.433 1.098.121 2.086-.67 2.207-1.769.12-1.098-.671-2.086-1.77-2.207-2.318-.255-4.306-1.48-5.787-3.13-1.503-1.674-2.342-3.629-2.488-5.082m35.87-4.644c0-1.104-.896-2-2-2s-2 .896-2 2c0 4.435-.351 7.45-1 9.769-.635 2.273-1.58 3.96-2.907 5.71-.667.88-.495 2.135.385 2.803.88.667 2.135.494 2.802-.386 1.564-2.062 2.775-4.195 3.573-7.05.786-2.812 1.146-6.233 1.146-10.846"
        })
    });
const $42fa96cb448ffb1e$var$ForwardRef = /*#__PURE__*/ (0, $1gzkc$react.forwardRef)($42fa96cb448ffb1e$var$SvgComponent);
var $42fa96cb448ffb1e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($42fa96cb448ffb1e$var$ForwardRef);
