// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileImage_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $13pK0$reactjsxruntime = require("react/jsx-runtime");
var $13pK0$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $83d375dbca7ab8a2$export$2e2bcd8739ae039);




const $83d375dbca7ab8a2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $13pK0$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $13pK0$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.136 24C38.267 24 31 31.12 31 40v80c0 8.879 7.267 16 16.136 16h64.728c8.869 0 16.136-7.121 16.136-16V72.97c0-4.258-1.711-8.332-4.742-11.33L89.913 28.67c-3.03-2.995-7.13-4.67-11.395-4.67zM39 40c0-4.376 3.6-8 8.136-8h31.382c2.173 0 4.247.854 5.77 2.36l33.345 32.97c1.521 1.504 2.367 3.534 2.367 5.64V120c0 4.376-3.6 8-8.136 8H47.136C42.6 128 39 124.376 39 120zm56.379 42.5c-4.418 0-8-3.582-8-8s3.582-8 8-8 8 3.582 8 8-3.582 8-8 8M63.37 84.75c2.502-4.333 8.757-4.333 11.259 0l9.752 16.892L89.371 93c2.501-4.333 8.756-4.333 11.258 0l9.959 17.25c2.502 4.333-.625 9.75-5.629 9.75H54.278c-5.004 0-8.132-5.417-5.63-9.75zm16.403 24.91L69 91l-12.125 21h21.572zm7.87 2.34h14.718L95 99.25l-7.147 12.377z"
        })
    });
const $83d375dbca7ab8a2$var$ForwardRef = /*#__PURE__*/ (0, $13pK0$react.forwardRef)($83d375dbca7ab8a2$var$SvgComponent);
var $83d375dbca7ab8a2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($83d375dbca7ab8a2$var$ForwardRef);
