// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noFile_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $3kBXW$jsx} from "react/jsx-runtime";
import {forwardRef as $3kBXW$forwardRef} from "react";





const $fedc109a2ab9ae21$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3kBXW$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3kBXW$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22.666 30c0-7.717 6.249-14 13.983-14h32.417c3.58 0 7.063 1.37 9.671 3.882l.005.005 32.607 31.24c2.752 2.636 4.317 6.278 4.317 10.106v24.584c0 2.209-1.791 4-4 4s-4-1.791-4-4v-23.5H78.949c-5.862 0-11.283-4.411-11.283-10.634V24H36.649c-3.299 0-5.983 2.684-5.983 6v79.983c0 3.317 2.684 6 5.983 6h40.017c2.21 0 4 1.791 4 4s-1.79 4-4 4H36.649c-7.734 0-13.983-6.283-13.983-14zm53-1.981 27.448 26.298H78.949c-2.17 0-3.283-1.523-3.283-2.634zm35.566 116.131c13.577 0 24.584-11.006 24.584-24.583s-11.007-24.584-24.584-24.584-24.583 11.007-24.583 24.584 11.006 24.583 24.583 24.583m4.416-35.833c0-2.209-1.79-4-4-4s-4 1.791-4 4v7.666h-7.666c-2.21 0-4 1.791-4 4s1.79 4 4 4h7.666v7.667c0 2.209 1.791 4 4 4s4-1.791 4-4v-7.667h7.667c2.209 0 4-1.791 4-4s-1.791-4-4-4h-7.667z"
        })
    });
const $fedc109a2ab9ae21$var$ForwardRef = /*#__PURE__*/ (0, $3kBXW$forwardRef)($fedc109a2ab9ae21$var$SvgComponent);
var $fedc109a2ab9ae21$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($fedc109a2ab9ae21$var$ForwardRef);


export {$fedc109a2ab9ae21$export$2e2bcd8739ae039 as default};
