// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browser_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $d84zF$reactjsxruntime = require("react/jsx-runtime");
var $d84zF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f4012573f2822928$export$2e2bcd8739ae039);




const $f4012573f2822928$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $d84zF$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $d84zF$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M74.5 20c3.314 0 6 2.686 6 6v6h-64v-6c0-3.314 2.686-6 6-6zm-58 16h64v34c0 3.314-2.686 6-6 6h-52c-3.314 0-6-2.686-6-6zm68-10c0-5.523-4.477-10-10-10h-52c-5.523 0-10 4.477-10 10v44c0 5.523 4.477 10 10 10h52c5.523 0 10-4.477 10-10zM24 28.5c-1.38 0-2.5-1.12-2.5-2.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5-1.12 2.5-2.5 2.5"
        })
    });
const $f4012573f2822928$var$ForwardRef = /*#__PURE__*/ (0, $d84zF$react.forwardRef)($f4012573f2822928$var$SvgComponent);
var $f4012573f2822928$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f4012573f2822928$var$ForwardRef);
