// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browserNotCompatible_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6GDu9$jsx} from "react/jsx-runtime";
import {forwardRef as $6GDu9$forwardRef} from "react";





const $12dae37f10d36235$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6GDu9$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6GDu9$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M37 10.25c1.519 0 2.75 1.231 2.75 2.75v4.77H8.25V13c0-1.519 1.231-2.75 2.75-2.75zM8.25 20.27h31.5V35c0 1.519-1.231 2.75-2.75 2.75H11c-1.519 0-2.75-1.231-2.75-2.75zm34-7.27c0-2.9-2.35-5.25-5.25-5.25H11c-2.9 0-5.25 2.35-5.25 5.25v22c0 2.9 2.35 5.25 5.25 5.25h26c2.9 0 5.25-2.35 5.25-5.25zM13 15.5c-.828 0-1.5-.672-1.5-1.5s.672-1.5 1.5-1.5 1.5.672 1.5 1.5-.672 1.5-1.5 1.5m5.616 8.116c.488-.488 1.28-.488 1.768 0L24 27.232l3.616-3.616c.488-.488 1.28-.488 1.768 0s.488 1.28 0 1.768L25.768 29l3.616 3.616c.488.488.488 1.28 0 1.768s-1.28.488-1.768 0L24 30.768l-3.616 3.616c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.768L22.232 29l-3.616-3.616c-.488-.488-.488-1.28 0-1.768"
        })
    });
const $12dae37f10d36235$var$ForwardRef = /*#__PURE__*/ (0, $6GDu9$forwardRef)($12dae37f10d36235$var$SvgComponent);
var $12dae37f10d36235$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($12dae37f10d36235$var$ForwardRef);


export {$12dae37f10d36235$export$2e2bcd8739ae039 as default};
