// @react-spectrum/s2/illustrations/linear/internal/S2_lin_tag_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9rYps$jsx} from "react/jsx-runtime";
import {forwardRef as $9rYps$forwardRef} from "react";





const $673fdd61fd7d1217$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9rYps$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9rYps$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M19.996 14c-3.566-.045-6.496 2.823-6.496 6.395V40.56c0 1.699.678 3.326 1.882 4.525l33.904 33.758c4.227 4.21 11.079 4.21 15.307 0l13.733-13.674c4.232-4.214 4.232-11.05 0-15.265L44.399 16.123c-1.184-1.179-2.781-1.848-4.451-1.87zM17.5 20.396c0-1.326 1.091-2.412 2.445-2.395l19.952.253c.632.008 1.235.262 1.68.704l33.926 33.781c2.663 2.651 2.663 6.946 0 9.596L61.77 76.008c-2.667 2.656-6.995 2.656-9.662 0L18.204 42.251c-.452-.45-.704-1.058-.704-1.69zM27 32c0 2.761 2.239 5 5 5s5-2.239 5-5-2.239-5-5-5-5 2.239-5 5"
        })
    });
const $673fdd61fd7d1217$var$ForwardRef = /*#__PURE__*/ (0, $9rYps$forwardRef)($673fdd61fd7d1217$var$SvgComponent);
var $673fdd61fd7d1217$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($673fdd61fd7d1217$var$ForwardRef);


export {$673fdd61fd7d1217$export$2e2bcd8739ae039 as default};
