// @react-spectrum/s2/illustrations/linear/internal/S2_lin_shapes_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $2HuFr$reactjsxruntime = require("react/jsx-runtime");
var $2HuFr$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7d843f4707b835bc$export$2e2bcd8739ae039);




const $7d843f4707b835bc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2HuFr$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2HuFr$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M54.689 34.641c1.543-2.656 5.38-2.654 6.92.004L87.79 79.829c1.546 2.667-.378 6.006-3.46 6.006H31.899c-3.084 0-5.008-3.343-3.459-6.01zm13.842-4.007c-4.62-7.973-16.13-7.98-20.76-.011L21.524 75.807c-4.648 8 1.124 18.028 10.376 18.028h52.43c9.247 0 15.019-10.017 10.383-18.017zm25.6 17.991c-2.208-.087-4.068 1.631-4.156 3.839-.087 2.207 1.631 4.067 3.838 4.155 18.714.743 33.66 16.152 33.66 35.05 0 19.374-15.705 35.079-35.078 35.079-15.227 0-28.198-9.705-33.046-23.279-.743-2.08-3.032-3.165-5.112-2.422s-3.165 3.032-2.422 5.113c5.949 16.657 21.866 28.588 40.58 28.588 23.791 0 43.078-19.287 43.078-43.079 0-23.21-18.354-42.131-41.342-43.044"
        })
    });
const $7d843f4707b835bc$var$ForwardRef = /*#__PURE__*/ (0, $2HuFr$react.forwardRef)($7d843f4707b835bc$var$SvgComponent);
var $7d843f4707b835bc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7d843f4707b835bc$var$ForwardRef);
