// @react-spectrum/s2/illustrations/linear/internal/S2_lin_AIGenerate_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $iqk7M$jsx} from "react/jsx-runtime";
import {forwardRef as $iqk7M$forwardRef} from "react";





const $caac5c5b6c7be7dc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iqk7M$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $iqk7M$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M38.615 7.111c.867.523 1.436 1.576 1.18 2.77l-.615 2.85v.002q-.006.028 0 .057.01.027.019.036l1.897 2.149c.814.922.802 2.124.323 2.99-.483.871-1.519 1.53-2.755 1.24l-2.718-.638-2.057 1.978c-.907.872-2.129.87-3.004.343-.867-.523-1.436-1.576-1.18-2.77l.615-2.85v-.002q.006-.028 0-.057-.01-.027-.019-.036l-1.897-2.149c-.814-.921-.802-2.124-.323-2.99.483-.871 1.519-1.53 2.755-1.24l2.718.638 2.057-1.978c.907-.872 2.129-.87 3.004-.343m-1.278 2.151-2.051 1.973c-.605.582-1.463.829-2.29.634h-.002l-2.716-.638-.01.014c-.015.028-.019.056-.018.073q0 .01.004.017.002.007.023.034l1.897 2.148c.55.623.764 1.474.59 2.278v.001l-.614 2.848q-.008.047-.005.057 0 .012.004.017.003.009.014.018l2.051-1.972c.605-.582 1.463-.829 2.29-.633h.002l2.716.637.01-.014c.015-.028.019-.056.018-.073q0-.01-.004-.017-.002-.007-.024-.034l-1.896-2.148c-.55-.622-.764-1.474-.59-2.278v-.002l.614-2.847q.008-.046.005-.057 0-.011-.004-.017-.003-.009-.014-.019M13 13.25c-1.519 0-2.75 1.232-2.75 2.75v19c0 1.52 1.231 2.75 2.75 2.75h19c1.519 0 2.75-1.23 2.75-2.75V22.25c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v12.75c0 2.9-2.35 5.25-5.25 5.25H13c-2.9 0-5.25-2.35-5.25-5.25V16c0-2.899 2.35-5.25 5.25-5.25h12c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25zm15.99 11.887-2.008-.452c-.265-.06-.541.015-.74.198l-1.513 1.395c-.596.55-1.544.002-1.366-.788l.453-2.007c.06-.266-.015-.542-.198-.741l-1.396-1.513c-.55-.596-.001-1.544.788-1.365l2.008.452c.265.06.541-.015.74-.198l1.513-1.395c.596-.55 1.544-.002 1.366.787l-.453 2.008c-.06.266.015.542.198.741l1.396 1.513c.55.596.001 1.543-.788 1.365"
        })
    });
const $caac5c5b6c7be7dc$var$ForwardRef = /*#__PURE__*/ (0, $iqk7M$forwardRef)($caac5c5b6c7be7dc$var$SvgComponent);
var $caac5c5b6c7be7dc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($caac5c5b6c7be7dc$var$ForwardRef);


export {$caac5c5b6c7be7dc$export$2e2bcd8739ae039 as default};
