// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noLibrariesBrands_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9rqgl$jsx} from "react/jsx-runtime";
import {forwardRef as $9rqgl$forwardRef} from "react";





const $d1cf01e85f3f2d39$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9rqgl$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9rqgl$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M23.216 29C20.95 29 19 30.892 19 33.343v34.503C19 83.443 31.46 96 46.667 96c15.213 0 27.666-12.563 27.666-28.154V33.343c0-2.451-1.95-4.343-4.216-4.343zM11 33.343C11 26.589 16.417 21 23.216 21h46.901c6.8 0 12.216 5.589 12.216 12.343v34.503C82.333 87.74 66.413 104 46.667 104 26.927 104 11 87.747 11 67.846zm48.979 14.606v.002l8.217 15.841v.002c1.143 2.205.997 4.67.002 6.591C67.2 72.314 65.144 74 62.383 74H45.95c-2.76 0-4.816-1.686-5.815-3.615-.994-1.92-1.14-4.386.002-6.591l8.21-15.843c1.172-2.26 3.4-3.617 5.817-3.617s4.645 1.356 5.816 3.615m-5.815 6.164L48.004 66h12.325zm-13.856-.791c.914-2.011.025-4.383-1.986-5.297s-4.383-.025-5.297 1.986l-8.333 18.334c-.915 2.01-.025 4.382 1.986 5.296 2.01.914 4.382.025 5.296-1.986zm50.526-9.328c-2.21 0-4 1.79-4 4s1.79 4 4 4h6.531V82.91c0 1.153 1.284 1.846 2.236 1.197l7.023-4.687c.491-.318 1.111-.318 1.601 0l7.024 4.687c.951.634 2.235-.044 2.235-1.197V51.994h.561c4.162 0 7.537 3.375 7.537 7.538v28.623c0 2.21 1.791 4 4 4s4-1.79 4-4V59.532c0-8.582-6.956-15.538-15.537-15.538H90.834M58.66 110.455c0-2.209-1.79-4-4-4s-4 1.791-4 4v.923c0 8.581 6.957 15.538 15.538 15.538h30.067c2.209 0 4-1.791 4-4s-1.791-4-4-4H66.198c-4.163 0-7.537-3.375-7.537-7.538zm70.507 35.379c13.577 0 24.584-11.007 24.584-24.584s-11.007-24.583-24.584-24.583-24.583 11.006-24.583 24.583 11.006 24.584 24.583 24.584M133.584 110c0-2.209-1.791-4-4-4s-4 1.791-4 4v7.667h-7.666c-2.21 0-4 1.791-4 4s1.79 4 4 4h7.666v7.667c0 2.209 1.791 4 4 4s4-1.791 4-4v-7.667h7.667c2.209 0 4-1.791 4-4s-1.791-4-4-4h-7.667z"
        })
    });
const $d1cf01e85f3f2d39$var$ForwardRef = /*#__PURE__*/ (0, $9rqgl$forwardRef)($d1cf01e85f3f2d39$var$SvgComponent);
var $d1cf01e85f3f2d39$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d1cf01e85f3f2d39$var$ForwardRef);


export {$d1cf01e85f3f2d39$export$2e2bcd8739ae039 as default};
