// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browserNotCompatible_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $joqFT$reactjsxruntime = require("react/jsx-runtime");
var $joqFT$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e19ba3669f970d9d$export$2e2bcd8739ae039);




const $e19ba3669f970d9d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $joqFT$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $joqFT$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M74.5 20c3.314 0 6 2.686 6 6v6h-64v-6c0-3.314 2.686-6 6-6zm-58 16h64v34c0 3.314-2.686 6-6 6h-52c-3.314 0-6-2.686-6-6zm68-10c0-5.523-4.477-10-10-10h-52c-5.523 0-10 4.477-10 10v44c0 5.523 4.477 10 10 10h52c5.523 0 10-4.477 10-10zM24 28.5c-1.38 0-2.5-1.12-2.5-2.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5-1.12 2.5-2.5 2.5m11.005 15.167c.791-.77 2.058-.754 2.828.038l9.667 9.928 9.667-9.928c.77-.791 2.036-.808 2.828-.038s.808 2.037.038 2.828L50.29 56.5l9.742 10.005c.77.791.754 2.058-.037 2.828-.792.77-2.058.754-2.829-.038L47.5 59.367l-9.667 9.928c-.771.792-2.037.809-2.829.038-.791-.77-.808-2.037-.037-2.828L44.709 56.5l-9.742-10.005c-.77-.791-.753-2.057.038-2.828"
        })
    });
const $e19ba3669f970d9d$var$ForwardRef = /*#__PURE__*/ (0, $joqFT$react.forwardRef)($e19ba3669f970d9d$var$SvgComponent);
var $e19ba3669f970d9d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e19ba3669f970d9d$var$ForwardRef);
