// @react-spectrum/s2/illustrations/linear/internal/S2_lin_pause_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $cRJb1$jsx} from "react/jsx-runtime";
import {forwardRef as $cRJb1$forwardRef} from "react";





const $8a25ff1c1a41ff79$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cRJb1$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cRJb1$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M25 27c0-3.314 2.686-6 6-6h8c3.314 0 6 2.686 6 6v42c0 3.314-2.686 6-6 6h-8c-3.314 0-6-2.686-6-6zm6-2c-1.105 0-2 .895-2 2v42c0 1.105.895 2 2 2h8c1.105 0 2-.895 2-2V27c0-1.105-.895-2-2-2zm21 2c0-3.314 2.686-6 6-6h8c3.314 0 6 2.686 6 6v42c0 3.314-2.686 6-6 6h-8c-3.314 0-6-2.686-6-6zm6-2c-1.105 0-2 .895-2 2v42c0 1.105.895 2 2 2h8c1.105 0 2-.895 2-2V27c0-1.105-.895-2-2-2z"
        })
    });
const $8a25ff1c1a41ff79$var$ForwardRef = /*#__PURE__*/ (0, $cRJb1$forwardRef)($8a25ff1c1a41ff79$var$SvgComponent);
var $8a25ff1c1a41ff79$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8a25ff1c1a41ff79$var$ForwardRef);


export {$8a25ff1c1a41ff79$export$2e2bcd8739ae039 as default};
