// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowRight_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $7Bq6q$reactjsxruntime = require("react/jsx-runtime");
var $7Bq6q$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9e55e25d35b3939a$export$2e2bcd8739ae039);




const $9e55e25d35b3939a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7Bq6q$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7Bq6q$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M15 46c-1.105 0-2 .895-2 2s.895 2 2 2h63q.09 0 .18-.008L67.586 60.586c-.781.78-.781 2.047 0 2.828.78.781 2.047.781 2.828 0l14-14c.781-.78.781-2.047 0-2.828l-14-14c-.78-.781-2.047-.781-2.828 0-.781.78-.781 2.047 0 2.828L78.18 46.008Q78.09 46 78 46zm64.992 1.82q.008.09.008.18t-.008.18l.18-.18z"
        })
    });
const $9e55e25d35b3939a$var$ForwardRef = /*#__PURE__*/ (0, $7Bq6q$react.forwardRef)($9e55e25d35b3939a$var$SvgComponent);
var $9e55e25d35b3939a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9e55e25d35b3939a$var$ForwardRef);
