// @react-spectrum/s2/illustrations/linear/internal/S2_lin_magicWand_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6fEw4$reactjsxruntime = require("react/jsx-runtime");
var $6fEw4$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f41fb7c32b1ea9b4$export$2e2bcd8739ae039);




const $f41fb7c32b1ea9b4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6fEw4$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6fEw4$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m75.62 27.258 4.232-.846c1.665-.332 1.906-2.616.344-3.289l-3.963-1.707c-.522-.227-.9-.692-1.012-1.252l-.846-4.233c-.332-1.664-2.615-1.906-3.289-.344L69.38 19.55c-.227.521-.692.899-1.252 1.012l-4.233.846c-1.664.331-1.906 2.615-.344 3.288l3.963 1.708c.522.226.9.691 1.013 1.251l.846 4.233c.331 1.665 2.615 1.906 3.288.345l1.708-3.964c.226-.522.691-.9 1.251-1.012m-34.977 3.478 3.496-.699c1.375-.274 1.575-2.16.285-2.716l-3.274-1.41c-.431-.188-.743-.572-.836-1.034l-.699-3.497c-.274-1.374-2.16-1.574-2.716-.284l-1.41 3.274c-.187.43-.572.743-1.034.836l-3.496.699c-1.375.274-1.575 2.16-.285 2.716l3.274 1.41c.43.187.743.571.836 1.034l.699 3.496c.274 1.375 2.16 1.575 2.716.285l1.41-3.274c.187-.431.571-.743 1.034-.836m38.026 24.52-3.496.699c-.463.093-.847.405-1.034.836l-1.41 3.273c-.556 1.29-2.442 1.09-2.716-.284l-.699-3.496c-.093-.463-.405-.847-.836-1.034l-3.274-1.41c-1.29-.557-1.09-2.443.285-2.717l3.496-.698c.463-.094.847-.405 1.034-.836l1.41-3.274c.556-1.29 2.442-1.09 2.716.284l.699 3.497c.093.462.405.846.836 1.033l3.274 1.41c1.29.557 1.09 2.443-.285 2.717M52.147 32.87c3.035-3.035 7.957-3.035 10.992 0s3.035 7.957 0 10.992L26.503 80.5c-3.035 3.035-7.957 3.035-10.992 0s-3.035-7.957 0-10.992zm8.164 2.829c-1.474-1.474-3.862-1.474-5.336 0l-6.058 6.058 5.335 5.335 6.059-6.058c1.473-1.473 1.473-3.862 0-5.335M51.424 49.92l-5.335-5.335-27.75 27.749c-1.473 1.473-1.473 3.862 0 5.335s3.862 1.473 5.335 0z"
        })
    });
const $f41fb7c32b1ea9b4$var$ForwardRef = /*#__PURE__*/ (0, $6fEw4$react.forwardRef)($f41fb7c32b1ea9b4$var$SvgComponent);
var $f41fb7c32b1ea9b4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f41fb7c32b1ea9b4$var$ForwardRef);
