// @react-spectrum/s2/illustrations/linear/internal/S2_lin_clock_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9aN25$jsx} from "react/jsx-runtime";
import {forwardRef as $9aN25$forwardRef} from "react";





const $09f1665d7517b712$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9aN25$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9aN25$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 7.99c-8.698 0-15.75 7.052-15.75 15.75S15.302 39.49 24 39.49s15.75-7.051 15.75-15.75S32.699 7.99 24 7.99M5.75 23.74C5.75 13.661 13.92 5.49 24 5.49s18.25 8.171 18.25 18.25S34.08 41.99 24 41.99 5.75 33.82 5.75 23.74m19.5-.721V12.24c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v11.476q0 .075.006.149c.018.18.075.35.161.5s.205.284.353.39q.06.044.126.08l6.408 3.7c.598.345 1.363.14 1.708-.458s.14-1.362-.458-1.707z"
        })
    });
const $09f1665d7517b712$var$ForwardRef = /*#__PURE__*/ (0, $9aN25$forwardRef)($09f1665d7517b712$var$SvgComponent);
var $09f1665d7517b712$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($09f1665d7517b712$var$ForwardRef);


export {$09f1665d7517b712$export$2e2bcd8739ae039 as default};
