// @react-spectrum/s2/illustrations/linear/internal/S2_lin_assets_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $dz6OH$reactjsxruntime = require("react/jsx-runtime");
var $dz6OH$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1026e9f2f8c77468$export$2e2bcd8739ae039);




const $1026e9f2f8c77468$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dz6OH$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        fill: "none",
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dz6OH$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24.25 41.384c0-7.732 6.268-14 14-14h66.54c7.732 0 14 6.268 14 14v12.741c0 2.21-1.791 4-4 4s-4-1.79-4-4V41.384c0-3.314-2.687-6-6-6H38.25c-3.314 0-6 2.686-6 6V84.68l16.237-25.167c3.708-5.748 12.146-5.655 15.728.173l.003.006 1.275 2.082c1.153 1.884.561 4.346-1.323 5.5s-4.347.561-5.5-1.323l-1.272-2.077v-.002c-.5-.809-1.673-.821-2.189-.022l-18.839 29.2c.591.196 1.223.301 1.88.301h19c2.21 0 4 1.791 4 4s-1.79 4-4 4h-19c-7.732 0-14-6.268-14-14zm69.918 15.773c3.314 0 6-2.686 6-6s-2.686-6-6-6-6 2.687-6 6 2.686 6 6 6M68.262 79.128c0-7.732 6.268-14 14-14h39.488c7.732 0 14 6.268 14 14v39.488c0 7.732-6.268 14-14 14H82.262c-7.732 0-14-6.268-14-14zm14-6c-3.314 0-6 2.686-6 6v39.488c0 3.314 2.686 6 6 6h39.488c3.314 0 6-2.686 6-6V79.128c0-3.314-2.686-6-6-6zm13.076 12.577 19.507 11.189c2.184 1.253 2.18 4.379-.007 5.627l-19.534 11.143c-2.188 1.247-4.919-.319-4.916-2.82l.027-22.331c.003-2.501 2.738-4.061 4.923-2.808",
            clipRule: "evenodd"
        })
    });
const $1026e9f2f8c77468$var$ForwardRef = /*#__PURE__*/ (0, $dz6OH$react.forwardRef)($1026e9f2f8c77468$var$SvgComponent);
var $1026e9f2f8c77468$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1026e9f2f8c77468$var$ForwardRef);
