// @react-spectrum/s2/illustrations/linear/internal/S2_lin_dataAnalytics_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9IU1f$jsx} from "react/jsx-runtime";
import {forwardRef as $9IU1f$forwardRef} from "react";





const $17b4cc971dcd782b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9IU1f$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9IU1f$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M135.133 32.356c1.371 1.733 1.078 4.249-.655 5.62L94.274 69.78c-1.238.98-2.936 1.139-4.334.408L64.63 56.963 30.59 85.835c-1.685 1.43-4.21 1.222-5.639-.463-1.428-1.684-1.221-4.209.464-5.638L61.509 49.12c1.241-1.053 2.997-1.249 4.44-.495l25.396 13.272 38.17-30.196c1.732-1.37 4.248-1.077 5.618.655M114.041 72.51c0-6.007 4.87-10.877 10.877-10.877s10.877 4.87 10.877 10.877v45.777c0 6.007-4.87 10.877-10.877 10.877s-10.877-4.87-10.877-10.877zm10.877-2.877c-1.589 0-2.877 1.288-2.877 2.877v45.777c0 1.589 1.288 2.877 2.877 2.877s2.877-1.288 2.877-2.877V72.51c0-1.589-1.288-2.877-2.877-2.877M54.156 87.516c0-6.007 4.87-10.877 10.877-10.877s10.877 4.87 10.877 10.877v30.771c0 6.007-4.87 10.877-10.877 10.877s-10.877-4.87-10.877-10.877zm10.877-2.877c-1.589 0-2.877 1.288-2.877 2.877v30.771c0 1.589 1.288 2.877 2.877 2.877 1.59 0 2.877-1.288 2.877-2.877V87.516c0-1.59-1.288-2.877-2.877-2.877m29.943-2.04c-6.007 0-10.877 4.87-10.877 10.878v24.81c0 6.007 4.87 10.877 10.877 10.877s10.877-4.87 10.877-10.877v-24.81c0-6.008-4.87-10.877-10.877-10.877m-2.877 10.878c0-1.59 1.288-2.877 2.877-2.877s2.877 1.288 2.877 2.877v24.81c0 1.589-1.288 2.877-2.877 2.877-1.59 0-2.877-1.288-2.877-2.877zm-57.008.914c-6.007 0-10.877 4.87-10.877 10.877v13.019c0 6.007 4.87 10.877 10.877 10.877s10.877-4.87 10.877-10.877v-13.019c0-6.007-4.87-10.877-10.877-10.877m-2.877 10.877c0-1.589 1.288-2.877 2.877-2.877s2.877 1.288 2.877 2.877v13.019c0 1.589-1.288 2.877-2.877 2.877s-2.877-1.288-2.877-2.877z"
        })
    });
const $17b4cc971dcd782b$var$ForwardRef = /*#__PURE__*/ (0, $9IU1f$forwardRef)($17b4cc971dcd782b$var$SvgComponent);
var $17b4cc971dcd782b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($17b4cc971dcd782b$var$ForwardRef);


export {$17b4cc971dcd782b$export$2e2bcd8739ae039 as default};
