// @react-spectrum/s2/illustrations/linear/internal/S2_lin_magicWand_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $hZbai$reactjsxruntime = require("react/jsx-runtime");
var $hZbai$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6bc7c36549aaf154$export$2e2bcd8739ae039);




const $6bc7c36549aaf154$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hZbai$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hZbai$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m37.81 13.629 2.116-.423c.832-.166.953-1.308.172-1.644l-1.981-.854c-.261-.113-.45-.346-.506-.626l-.423-2.116c-.166-.833-1.308-.954-1.645-.173l-.854 1.982c-.113.261-.345.45-.625.506l-2.117.423c-.832.166-.953 1.308-.172 1.644l1.982.854c.26.114.45.346.506.626l.423 2.117c.166.832 1.308.953 1.644.172l.854-1.982c.113-.26.346-.45.626-.506m-17.489 1.74 1.749-.35c.687-.136.787-1.08.142-1.357l-1.637-.706c-.215-.093-.371-.285-.418-.517l-.35-1.748c-.136-.687-1.08-.787-1.357-.142l-.706 1.637c-.093.215-.285.371-.517.418l-1.748.35c-.687.136-.787 1.08-.142 1.357l1.637.706c.215.093.371.285.418.516l.35 1.749c.136.687 1.08.787 1.357.142l.706-1.637c.093-.216.285-.372.517-.418m19.014 12.259-1.748.35c-.232.046-.424.202-.517.417l-.706 1.637c-.277.645-1.22.545-1.357-.142l-.35-1.748c-.046-.232-.202-.424-.418-.517l-1.637-.705c-.645-.278-.545-1.221.142-1.358l1.749-.35c.23-.046.423-.202.516-.418l.706-1.637c.278-.645 1.22-.545 1.358.143l.35 1.748c.046.231.202.423.417.517l1.637.705c.645.278.545 1.22-.142 1.358m-13.438-11.37c1.615-1.615 4.234-1.615 5.85 0 1.615 1.616 1.615 4.235 0 5.85L13.427 40.426c-1.615 1.615-4.234 1.615-5.85 0s-1.615-4.234 0-5.85zm4.082 1.768c-.64-.639-1.675-.639-2.314 0l-2.676 2.676 2.314 2.314 2.675-2.676c.64-.639.64-1.675 0-2.314m-4.444 6.757-2.314-2.314L9.346 36.344c-.639.64-.639 1.675 0 2.314.64.64 1.675.64 2.314 0z"
        })
    });
const $6bc7c36549aaf154$var$ForwardRef = /*#__PURE__*/ (0, $hZbai$react.forwardRef)($6bc7c36549aaf154$var$SvgComponent);
var $6bc7c36549aaf154$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6bc7c36549aaf154$var$ForwardRef);
