// @react-spectrum/s2/illustrations/linear/internal/S2_lin_buildTable_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $cyfkT$jsx} from "react/jsx-runtime";
import {forwardRef as $cyfkT$forwardRef} from "react";





const $40a892b62af8201e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cyfkT$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cyfkT$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M10.77 12.04c-1.52 0-2.75 1.23-2.75 2.75v1.89h32.05v-1.89c0-1.52-1.23-2.75-2.75-2.75zM8.02 33.21V19.18h4.62v16.78h-1.87c-1.52 0-2.75-1.231-2.75-2.75m21.45 2.75H15.14V19.18h24.93v6.16c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V14.79c0-2.901-2.35-5.25-5.25-5.25H10.77c-2.9 0-5.25 2.349-5.25 5.25v18.42c0 2.9 2.35 5.25 5.25 5.25h18.7c.69 0 1.25-.56 1.25-1.25 0-.691-.56-1.25-1.25-1.25m2.5-1.09c0-3.9 3.16-7.06 7.06-7.06s7.06 3.16 7.06 7.06-3.16 7.06-7.06 7.06-7.06-3.16-7.06-7.06m8.31 1.23h2.17c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.17v-2.17c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v2.17h-2.17c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h2.17v2.17c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25z"
        })
    });
const $40a892b62af8201e$var$ForwardRef = /*#__PURE__*/ (0, $cyfkT$forwardRef)($40a892b62af8201e$var$SvgComponent);
var $40a892b62af8201e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($40a892b62af8201e$var$ForwardRef);


export {$40a892b62af8201e$export$2e2bcd8739ae039 as default};
