// @react-spectrum/s2/illustrations/linear/internal/S2_lin_trophy_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $fhVWX$jsx} from "react/jsx-runtime";
import {forwardRef as $fhVWX$forwardRef} from "react";





const $8da6b498dc08745b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fhVWX$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $fhVWX$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M17.17 10.5c0-.14.11-.25.25-.25h13.16c.14 0 .25.11.25.25v8.89c0 3.72-2.981 6.75-6.684 6.828q-.072-.008-.146-.008-.075 0-.146.008c-3.703-.078-6.684-3.107-6.684-6.828V10.5m16.16 1.86V10.5c0-1.52-1.23-2.75-2.75-2.75H17.42c-1.52 0-2.75 1.23-2.75 2.75v1.86H10.5c-1.52 0-2.75 1.23-2.75 2.75v.81c0 4.51 3.66 8.17 8.17 8.17h.02c1.419 2.427 3.903 4.157 6.81 4.547v4.617c-2.596.438-4.727 2.217-5.68 4.59l-.003.008c-.433 1.099.337 2.408 1.623 2.408h10.6c1.267 0 2.087-1.289 1.617-2.424-.953-2.363-3.073-4.135-5.657-4.578v-4.621c2.907-.39 5.391-2.12 6.81-4.547h.02c4.51 0 8.17-3.66 8.17-8.17v-.81c0-1.52-1.23-2.75-2.75-2.75zm0 2.5v4.53c0 .726-.083 1.432-.24 2.11 2.648-.477 4.66-2.795 4.66-5.58v-.81c0-.14-.11-.25-.25-.25zm-23.08 1.06c0 2.785 2.012 5.103 4.66 5.58-.157-.678-.24-1.384-.24-2.11v-4.53H10.5c-.14 0-.25.11-.25.25zm13.74 19.73c-1.673.003-3.15.834-4.048 2.11h8.096c-.898-1.276-2.375-2.107-4.048-2.11m1.34-20.02-.64-1.31h-.01c-.26-.52-1-.52-1.26 0l-.64 1.31c-.1.2-.3.35-.53.38l-1.44.21c-.58.09-.81.79-.39 1.2l1.05 1.02c.16.16.24.39.2.62l-.25 1.44c-.1.57.51 1.01 1.02.74l1.29-.68c.2-.11.45-.11.65 0l1.29.68c.52.27 1.12-.17 1.02-.74l-.25-1.44c-.04-.23.03-.46.2-.62l1.05-1.02c.42-.41.19-1.12-.39-1.2l-1.44-.21c-.23-.03-.43-.17-.53-.38"
        })
    });
const $8da6b498dc08745b$var$ForwardRef = /*#__PURE__*/ (0, $fhVWX$forwardRef)($8da6b498dc08745b$var$SvgComponent);
var $8da6b498dc08745b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8da6b498dc08745b$var$ForwardRef);


export {$8da6b498dc08745b$export$2e2bcd8739ae039 as default};
