// @react-spectrum/s2/illustrations/linear/internal/S2_lin_text_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $8P9KK$jsx} from "react/jsx-runtime";
import {forwardRef as $8P9KK$forwardRef} from "react";





const $25717a967558eb74$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8P9KK$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $8P9KK$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M38.541 33.791c-2.623 0-4.75 2.127-4.75 4.75v16.583c0 2.624 2.127 4.75 4.75 4.75s4.75-2.126 4.75-4.75v-7.833c0-2.21 1.79-4 4-4h23.962c2.209 0 4 1.79 4 4v65.417c0 2.209-1.791 4-4 4h-7.836c-2.623 0-4.75 2.126-4.75 4.75s2.127 4.75 4.75 4.75h33.167c2.623 0 4.75-2.127 4.75-4.75s-2.127-4.75-4.75-4.75h-7.831c-2.21 0-4-1.791-4-4V47.291c0-2.21 1.79-4 4-4h23.955c2.209 0 4 1.79 4 4v7.833c0 2.624 2.126 4.75 4.75 4.75s4.75-2.126 4.75-4.75V38.541c0-2.623-2.126-4.75-4.75-4.75H38.541m-12.75 4.75c0-7.042 5.708-12.75 12.75-12.75h82.917c7.042 0 12.75 5.708 12.75 12.75v16.583c0 7.042-5.709 12.75-12.75 12.75s-12.75-5.708-12.75-12.75v-3.833H92.753v57.417h3.831c7.042 0 12.75 5.708 12.75 12.75s-5.708 12.75-12.75 12.75H63.417c-7.041 0-12.75-5.709-12.75-12.75s5.709-12.75 12.75-12.75h3.836V51.291H51.29v3.833c0 7.042-5.708 12.75-12.75 12.75s-12.75-5.708-12.75-12.75z"
        })
    });
const $25717a967558eb74$var$ForwardRef = /*#__PURE__*/ (0, $8P9KK$forwardRef)($25717a967558eb74$var$SvgComponent);
var $25717a967558eb74$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($25717a967558eb74$var$ForwardRef);


export {$25717a967558eb74$export$2e2bcd8739ae039 as default};
