// @react-spectrum/s2/illustrations/linear/internal/S2_lin_stopwatch_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $cwcz7$jsx} from "react/jsx-runtime";
import {forwardRef as $cwcz7$forwardRef} from "react";





const $dd2fd4df03d22439$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cwcz7$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cwcz7$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M26.022 7.063c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-4.62c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h1.06v1.03c-9.058.78-16.167 8.38-16.167 17.64 0 9.778 7.927 17.705 17.705 17.705 9.779 0 17.706-7.927 17.706-17.705 0-9.456-7.413-17.18-16.744-17.68v-.99zM8.795 25.733c0-8.398 6.808-15.206 15.205-15.206s15.206 6.808 15.206 15.206S32.398 40.938 24 40.938 8.795 34.13 8.795 25.733M24.962 12.74c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v1.732c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25zm-1.25 23c.69 0 1.25.56 1.25 1.25v1.732c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-1.732c0-.69.56-1.25 1.25-1.25m13.332-8.909c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H35.25c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25zm-22.989-1.25c0 .69-.56 1.25-1.25 1.25H11.01c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h1.795c.69 0 1.25.56 1.25 1.25m20.282-8.093c.488-.488.488-1.28 0-1.767-.489-.489-1.28-.489-1.768 0l-1.27 1.27c-.488.487-.488 1.279 0 1.767s1.28.488 1.768 0zM17.197 32.86c.488.488.488 1.28 0 1.768l-1.27 1.27c-.488.488-1.28.488-1.767 0-.489-.489-.489-1.28 0-1.768l1.27-1.27c.487-.488 1.279-.488 1.767 0M14.322 18.3c-.488-.488-.488-1.28 0-1.768s1.28-.488 1.767 0l1.228 1.227c.488.488.488 1.28 0 1.768-.489.488-1.28.488-1.768 0zm16.565 14.799c-.488.488-.488 1.28 0 1.767l1.227 1.227c.488.488 1.28.488 1.768 0s.488-1.28 0-1.767l-1.227-1.227c-.488-.489-1.28-.489-1.768 0m-6.553-8.17c-.643-.25-1.368.069-1.617.712-.25.644.069 1.368.712 1.618l10.223 3.969c.643.25 1.367-.07 1.617-.713s-.069-1.368-.712-1.618z"
        })
    });
const $dd2fd4df03d22439$var$ForwardRef = /*#__PURE__*/ (0, $cwcz7$forwardRef)($dd2fd4df03d22439$var$SvgComponent);
var $dd2fd4df03d22439$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($dd2fd4df03d22439$var$ForwardRef);


export {$dd2fd4df03d22439$export$2e2bcd8739ae039 as default};
