// @react-spectrum/s2/illustrations/linear/internal/S2_lin_diamond_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $kYmyv$jsx} from "react/jsx-runtime";
import {forwardRef as $kYmyv$forwardRef} from "react";





const $67edbfaef01b61f0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kYmyv$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kYmyv$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M33.682 15.193c-1.586 0-3.107.628-4.231 1.746L13.896 32.406c-2.134 2.122-2.366 5.497-.541 7.891L43.17 79.426c2.398 3.147 7.134 3.152 9.538.009l29.929-39.126c1.835-2.399 1.6-5.788-.547-7.912l-15.645-15.47c-1.123-1.11-2.64-1.734-4.219-1.734zm-1.41 4.582c.374-.372.881-.582 1.41-.582h7.357l-7.61 15.885q-.035.075-.065.15H16.731zm5.52 15.453 7.682-16.035h5.062l7.682 16.035zm19.984 4H38.234l9.77 34.774zm-5.424 34.088 9.579-34.088h16.496zm10.294-38.088q-.03-.075-.065-.15l-7.61-15.885h7.255c.526 0 1.032.208 1.406.578l15.632 15.457zm-45.077 4h16.51l9.645 34.325z"
        })
    });
const $67edbfaef01b61f0$var$ForwardRef = /*#__PURE__*/ (0, $kYmyv$forwardRef)($67edbfaef01b61f0$var$SvgComponent);
var $67edbfaef01b61f0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($67edbfaef01b61f0$var$ForwardRef);


export {$67edbfaef01b61f0$export$2e2bcd8739ae039 as default};
