// @react-spectrum/s2/illustrations/linear/internal/S2_lin_laptop_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $4jw45$reactjsxruntime = require("react/jsx-runtime");
var $4jw45$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $eba10a3920fdf7aa$export$2e2bcd8739ae039);




const $eba10a3920fdf7aa$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4jw45$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4jw45$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 36c-6.58 0-12 5.417-12 12.22V106h88V48.22c0-6.803-5.421-12.22-12-12.22zM28 48.22V106h-4c-7.18 0-13 5.82-13 13s5.82 13 13 13h112c7.18 0 13-5.82 13-13s-5.82-13-13-13h-4V48.22C132 37.103 123.101 28 112 28H48c-11.1 0-20 9.103-20 20.22M24 114c-2.761 0-5 2.239-5 5s2.239 5 5 5h112c2.761 0 5-2.239 5-5s-2.239-5-5-5zm61-68.5c0 2.485-2.015 4.5-4.5 4.5S76 47.985 76 45.5s2.015-4.5 4.5-4.5 4.5 2.015 4.5 4.5"
        })
    });
const $eba10a3920fdf7aa$var$ForwardRef = /*#__PURE__*/ (0, $4jw45$react.forwardRef)($eba10a3920fdf7aa$var$SvgComponent);
var $eba10a3920fdf7aa$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($eba10a3920fdf7aa$var$ForwardRef);
