// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browserNotCompatible_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $hvuSs$reactjsxruntime = require("react/jsx-runtime");
var $hvuSs$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $169d5662655905dd$export$2e2bcd8739ae039);




const $169d5662655905dd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hvuSs$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hvuSs$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M124 32c6.627 0 12 5.373 12 12v12H24V44c0-6.627 5.373-12 12-12zm12 84V64H24v52c0 6.627 5.373 12 12 12h88c6.627 0 12-5.373 12-12m8-72c0-11.046-8.954-20-20-20H36c-11.046 0-20 8.954-20 20v72c0 11.046 8.954 20 20 20h88c11.046 0 20-8.954 20-20zm-98.52.5c0 2.486-2.013 4.5-4.5 4.5s-4.5-2.014-4.5-4.5 2.014-4.5 4.5-4.5h.012c2.475 0 4.489 2.014 4.489 4.5m13.73 29.134c1.582-1.541 4.115-1.507 5.656.076L80.5 89.766 96.134 73.71c1.541-1.583 4.074-1.617 5.656-.076 1.583 1.541 1.617 4.074.076 5.656L86.083 95.5l15.783 16.21c1.541 1.582 1.507 4.115-.076 5.656s-4.115 1.507-5.656-.076L80.5 101.234 64.866 117.29c-1.541 1.583-4.074 1.617-5.657.076-1.582-1.541-1.616-4.074-.075-5.656L74.917 95.5 59.134 79.29c-1.541-1.582-1.507-4.115.075-5.656"
        })
    });
const $169d5662655905dd$var$ForwardRef = /*#__PURE__*/ (0, $hvuSs$react.forwardRef)($169d5662655905dd$var$SvgComponent);
var $169d5662655905dd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($169d5662655905dd$var$ForwardRef);
