// @react-spectrum/s2/illustrations/linear/internal/S2_lin_megaphonePromote_Expressive_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5tWLz$jsx} from "react/jsx-runtime";
import {forwardRef as $5tWLz$forwardRef} from "react";





const $60c0193b3cbe6eb2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5tWLz$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5tWLz$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M106.758 33.056c8.28-2.474 16.787 3.614 16.787 12.335v43.554c0 8.739-8.525 14.809-16.799 12.335L73.29 91.27l7.368 16.208.034.076c1.677 3.904 1.573 7.748.086 11.024-1.447 3.186-4.07 5.557-7 6.958-3.144 1.503-6.915 2.025-10.49 1.036-3.684-1.018-6.916-3.58-8.81-7.727-1.152-2.523-4.685-10.328-7.926-17.491l-4.314-9.541-.919-2.032h-.383C28.335 89.78 18 79.697 18 67.14 18 54.582 28.335 44.5 40.936 44.5h27.462l38.357-11.444zm8.787 12.335c0-3.2-3.189-5.66-6.498-4.67h-.002L73 51.476v31.357l36.038 10.784c3.316.991 6.507-1.468 6.507-4.67zM26 67.141C26 59.12 32.634 52.5 40.936 52.5h23.962v29.28H40.936C32.634 81.78 26 75.16 26 67.14m27.841 30.916L50.198 90h13.727l9.431 20.747c.813 1.914.649 3.397.137 4.524-.554 1.222-1.667 2.331-3.167 3.048-1.617.773-3.408.957-4.907.542-1.392-.384-2.747-1.332-3.664-3.34-1.145-2.506-4.67-10.293-7.914-17.464m82.576-15.701c-2.106-.667-4.354.5-5.021 2.607-.666 2.106.501 4.354 2.607 5.02l5.75 1.82c2.106.667 4.354-.5 5.02-2.606.667-2.106-.5-4.354-2.606-5.021zm8.356-38.503c.667 2.106-.5 4.354-2.606 5.02l-5.75 1.82c-2.106.667-4.354-.5-5.021-2.606s.501-4.354 2.607-5.02l5.75-1.82c2.106-.667 4.354.5 5.02 2.606M135.21 62.53c-2.209 0-4 1.79-4 4s1.791 4 4 4h6.03c2.209 0 4-1.791 4-4s-1.791-4-4-4z"
        })
    });
const $60c0193b3cbe6eb2$var$ForwardRef = /*#__PURE__*/ (0, $5tWLz$forwardRef)($60c0193b3cbe6eb2$var$SvgComponent);
var $60c0193b3cbe6eb2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($60c0193b3cbe6eb2$var$ForwardRef);


export {$60c0193b3cbe6eb2$export$2e2bcd8739ae039 as default};
