// @react-spectrum/s2/illustrations/linear/internal/S2_lin_socialPost_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $iJTzA$reactjsxruntime = require("react/jsx-runtime");
var $iJTzA$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b54853e7fedc33a2$export$2e2bcd8739ae039);




const $b54853e7fedc33a2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iJTzA$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        fill: "none",
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $iJTzA$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28.047 18.047c-5.523 0-10 4.477-10 10v39.906c0 5.523 4.477 10 10 10h39.906c5.523 0 10-4.477 10-10V28.047c0-5.523-4.477-10-10-10zm-6 10c0-3.314 2.686-6 6-6h39.906c3.314 0 6 2.686 6 6V39.71H22.047zm0 15.663h51.906v24.243q-.001.63-.125 1.227l-7.276-10.778-.002-.003c-2.177-3.246-6.948-3.239-9.126 0l-.008.012-5.72 8.65-10.148-15.675-1.68 1.086 1.68-1.087v.001c-2.164-3.346-7.066-3.346-9.23 0L22.047 67.255zm25.09 30.243 2.15-3.252L38.19 53.558c-.588-.909-1.925-.908-2.512.001L23.69 72.079c1.093 1.154 2.64 1.874 4.357 1.874zm4.795 0 8.815-13.327c.598-.883 1.895-.875 2.481.002l.005.006 8.227 12.188c-.986.712-2.197 1.131-3.507 1.131zM27.269 30.878c0-1.104.896-2 2-2l37.534.006c1.105 0 2 .895 2 2s-.896 2-2 2l-37.534-.005c-1.105 0-2-.896-2-2m37.66 18.746c0 1.657 1.344 3 3 3 1.657 0 3-1.343 3-3s-1.343-3-3-3c-1.656 0-3 1.343-3 3",
            clipRule: "evenodd"
        })
    });
const $b54853e7fedc33a2$var$ForwardRef = /*#__PURE__*/ (0, $iJTzA$react.forwardRef)($b54853e7fedc33a2$var$SvgComponent);
var $b54853e7fedc33a2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b54853e7fedc33a2$var$ForwardRef);
