// @react-spectrum/s2/illustrations/linear/internal/S2_lin_layers_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $77mk3$jsx} from "react/jsx-runtime";
import {forwardRef as $77mk3$forwardRef} from "react";





const $0238ee9028289dd1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $77mk3$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $77mk3$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M22.962 9.072c.63-.43 1.461-.43 2.092 0l13.376 9.112c.447.304.447.962 0 1.267l-13.376 9.112c-.631.43-1.461.43-2.092 0L9.585 19.45c-.447-.305-.447-.963 0-1.267zm3.5-2.066c-1.48-1.008-3.428-1.008-4.908 0L8.178 16.118c-1.904 1.297-1.904 4.102 0 5.4l13.376 9.11c1.48 1.009 3.427 1.009 4.908 0l13.376-9.11c1.904-1.298 1.904-4.103 0-5.4zm-14.93 19.741c.57-.389.718-1.166.33-1.737-.39-.57-1.167-.718-1.738-.33l-1.946 1.326c-1.904 1.297-1.904 4.103 0 5.4l13.376 9.112c1.48 1.008 3.427 1.008 4.908 0l13.376-9.112c1.904-1.297 1.904-4.102 0-5.4l-1.947-1.325c-.57-.389-1.348-.241-1.736.33-.39.57-.242 1.347.329 1.736l1.946 1.326c.447.304.447.962 0 1.267l-13.376 9.112c-.631.43-1.461.43-2.092 0L9.585 29.34c-.447-.305-.447-.963 0-1.267z"
        })
    });
const $0238ee9028289dd1$var$ForwardRef = /*#__PURE__*/ (0, $77mk3$forwardRef)($0238ee9028289dd1$var$SvgComponent);
var $0238ee9028289dd1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0238ee9028289dd1$var$ForwardRef);


export {$0238ee9028289dd1$export$2e2bcd8739ae039 as default};
