// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowRight_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $bAvBt$jsx} from "react/jsx-runtime";
import {forwardRef as $bAvBt$forwardRef} from "react";





const $b5cc105b159c4d29$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bAvBt$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bAvBt$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M15 46c-1.105 0-2 .895-2 2s.895 2 2 2h63q.09 0 .18-.008L67.586 60.586c-.781.78-.781 2.047 0 2.828.78.781 2.047.781 2.828 0l14-14c.781-.78.781-2.047 0-2.828l-14-14c-.78-.781-2.047-.781-2.828 0-.781.78-.781 2.047 0 2.828L78.18 46.008Q78.09 46 78 46zm64.992 1.82q.008.09.008.18t-.008.18l.18-.18z"
        })
    });
const $b5cc105b159c4d29$var$ForwardRef = /*#__PURE__*/ (0, $bAvBt$forwardRef)($b5cc105b159c4d29$var$SvgComponent);
var $b5cc105b159c4d29$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b5cc105b159c4d29$var$ForwardRef);


export {$b5cc105b159c4d29$export$2e2bcd8739ae039 as default};
