// @react-spectrum/s2/illustrations/linear/internal/S2_lin_clipboard_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $jmcah$jsx} from "react/jsx-runtime";
import {forwardRef as $jmcah$forwardRef} from "react";





const $8b5ce38de2724628$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jmcah$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jmcah$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48.025 12.4c-5.15-.006-9.016 4.178-9.022 9.19v.4l-2.99-.004q-.123 0-.242.014h-7.326C23.209 22 19 26.287 19 31.528v38.945C19 75.714 23.208 80 28.446 80h39.209c5.237 0 9.445-4.286 9.445-9.527V31.528c0-5.241-4.208-9.528-9.445-9.528H56.999l.001-.389c.006-5-3.815-9.204-8.975-9.21M62.01 26 62 33.003c-.001 1.104-.898 1.999-2.002 1.997l-24-.028c-.53-.001-1.04-.212-1.414-.588-.375-.375-.585-.884-.584-1.415L34.009 26h-5.564C25.46 26 23 28.454 23 31.528v38.945C23 73.546 25.459 76 28.445 76h39.21c2.986 0 5.445-2.454 5.445-5.527V31.528c0-3.074-2.459-5.528-5.445-5.528zm-24 0-.006 4.974 20 .024.006-4.986-3.014-.003c-.53 0-1.04-.212-1.414-.587-.374-.376-.585-.885-.584-1.415l.003-2.4c.004-2.952-2.186-5.203-4.98-5.207-2.803-.003-5.014 2.252-5.017 5.194l-.003 2.4c-.001 1.105-.898 2-2.002 1.998l-1.264-.002V26z"
        })
    });
const $8b5ce38de2724628$var$ForwardRef = /*#__PURE__*/ (0, $jmcah$forwardRef)($8b5ce38de2724628$var$SvgComponent);
var $8b5ce38de2724628$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8b5ce38de2724628$var$ForwardRef);


export {$8b5ce38de2724628$export$2e2bcd8739ae039 as default};
