// @react-spectrum/s2/illustrations/linear/internal/S2_lin_filters_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9u3PS$jsx} from "react/jsx-runtime";
import {forwardRef as $9u3PS$forwardRef} from "react";





const $6266f1c1b732a366$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9u3PS$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9u3PS$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M57.84 54.307c4.565-3.161 7.686-8.26 8.173-14.107-1.839-.59-3.799-.908-5.833-.908-3.326 0-6.454.85-9.178 2.346 3.497 3.319 5.955 7.721 6.838 12.669m-3.75 2.018c-2.198.882-4.597 1.367-7.11 1.367-2.034 0-3.995-.318-5.833-.907.424-5.098 2.852-9.628 6.493-12.8 3.531 3.076 5.921 7.43 6.45 12.34m-9.813-14.687c-3.675 3.489-6.203 8.175-6.96 13.433-5.085-2.988-8.665-8.263-9.326-14.412 2.197-.881 4.596-1.367 7.109-1.367 3.326 0 6.454.85 9.177 2.346m3.363-2.649c3.611-2.339 7.917-3.697 12.54-3.697 1.977 0 3.895.249 5.727.716-1.262-9.327-9.255-16.516-18.927-16.516-9.808 0-17.89 7.393-18.976 16.91 2.236-.72 4.62-1.11 7.096-1.11 4.623 0 8.929 1.358 12.54 3.697m-23.757-.795C16.795 42.138 12 49.705 12 58.392c0 12.758 10.342 23.1 23.1 23.1 4.623 0 8.929-1.358 12.54-3.696 3.611 2.338 7.917 3.696 12.54 3.696 12.758 0 23.1-10.342 23.1-23.1 0-9.224-5.407-17.186-13.225-20.889-.569-12.252-10.682-22.01-23.075-22.01-12.625 0-22.884 10.127-23.097 22.7m.357 4.484c-4.979 3.447-8.24 9.2-8.24 15.714 0 10.549 8.551 19.1 19.1 19.1 3.326 0 6.454-.85 9.177-2.345-4.187-3.976-6.886-9.506-7.172-15.666-6.636-3.143-11.535-9.355-12.865-16.803M47.64 72.8c-3.407-2.968-5.751-7.124-6.387-11.823 1.832.467 3.75.715 5.727.715 2.476 0 4.86-.39 7.096-1.11-.555 4.86-2.934 9.167-6.436 12.218m3.362 2.347c4.344-4.124 7.086-9.92 7.195-16.356 6.168-3.432 10.6-9.608 11.646-16.877 5.647 3.318 9.437 9.455 9.437 16.478 0 10.549-8.551 19.1-19.1 19.1-3.326 0-6.454-.85-9.178-2.345"
        })
    });
const $6266f1c1b732a366$var$ForwardRef = /*#__PURE__*/ (0, $9u3PS$forwardRef)($6266f1c1b732a366$var$SvgComponent);
var $6266f1c1b732a366$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6266f1c1b732a366$var$ForwardRef);


export {$6266f1c1b732a366$export$2e2bcd8739ae039 as default};
