// @react-spectrum/s2/illustrations/linear/internal/S2_lin_check_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $3ckGu$reactjsxruntime = require("react/jsx-runtime");
var $3ckGu$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2deca2ecd2d5bc98$export$2e2bcd8739ae039);




const $2deca2ecd2d5bc98$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3ckGu$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3ckGu$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M17 48c0-17.12 13.88-31 31-31 17.121 0 31 13.88 31 31 0 17.121-13.879 31-31 31-17.12 0-31-13.879-31-31m31-35c-19.33 0-35 15.67-35 35s15.67 35 35 35 35-15.67 35-35-15.67-35-35-35m16.08 27.639c.816-.744.875-2.01.13-2.826-.743-.816-2.008-.875-2.825-.13l-9.828 8.958-6.586 6.002-1.62 1.476-8.13-7.582c-.807-.753-2.073-.709-2.826.099s-.71 2.073.098 2.827l9.479 8.84c.762.71 1.941.716 2.711.015l-1.347-1.479 1.347 1.478h.001l.001-.001.002-.002.01-.009.04-.037.16-.145.605-.552 2.163-1.971 6.586-6.003z"
        })
    });
const $2deca2ecd2d5bc98$var$ForwardRef = /*#__PURE__*/ (0, $3ckGu$react.forwardRef)($2deca2ecd2d5bc98$var$SvgComponent);
var $2deca2ecd2d5bc98$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2deca2ecd2d5bc98$var$ForwardRef);
