// @react-spectrum/s2/illustrations/linear/internal/S2_lin_animation_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $4mVX4$jsx} from "react/jsx-runtime";
import {forwardRef as $4mVX4$forwardRef} from "react";





const $a1598252461bc6df$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4mVX4$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4mVX4$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M124.209 28.76c.736-1.376 2.178-2.3 3.831-2.3 2.396 0 4.339 1.943 4.339 4.34 0 .892-.266 1.712-.722 2.398-1.11 1.666-.658 3.917 1.009 5.026 1.667 1.11 3.917.657 5.026-1.01 1.224-1.838 1.937-4.048 1.937-6.414 0-6.4-5.189-11.59-11.589-11.59-4.433 0-8.278 2.491-10.224 6.132-.944 1.765-.278 3.962 1.488 4.906 1.765.943 3.962.277 4.905-1.488m-10.017 13.305c-3.754 0-7.023 2.1-8.685 5.21-1.042 1.948-3.465 2.683-5.413 1.641s-2.684-3.465-1.642-5.413c2.997-5.608 8.918-9.438 15.74-9.438 9.852 0 17.838 7.986 17.838 17.838 0 3.645-1.097 7.045-2.98 9.874-1.224 1.84-3.707 2.338-5.546 1.114s-2.337-3.707-1.113-5.547c1.036-1.556 1.639-3.422 1.639-5.44 0-5.435-4.404-9.84-9.838-9.84m-33.097 25.38c3.077-3.722 7.717-6.084 12.912-6.084 9.245 0 16.741 7.496 16.741 16.741 0 7.313-4.691 13.54-11.24 15.818-2.086.725-3.19 3.005-2.464 5.092s3.005 3.189 5.092 2.464c9.665-3.362 16.612-12.551 16.612-23.374 0-13.664-11.077-24.74-24.742-24.74-7.68 0-14.546 3.504-19.077 8.987-1.408 1.702-1.168 4.224.534 5.631 1.703 1.407 4.224 1.168 5.632-.535m-53.099 38.728c0-14.908 12.085-26.993 26.993-26.993s26.994 12.085 26.994 26.993-12.086 26.993-26.994 26.993-26.993-12.085-26.993-26.993M54.99 71.18c-19.326 0-34.993 15.667-34.993 34.993s15.667 34.993 34.993 34.993c19.327 0 34.994-15.667 34.994-34.993S74.315 71.18 54.989 71.18"
        })
    });
const $a1598252461bc6df$var$ForwardRef = /*#__PURE__*/ (0, $4mVX4$forwardRef)($a1598252461bc6df$var$SvgComponent);
var $a1598252461bc6df$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a1598252461bc6df$var$ForwardRef);


export {$a1598252461bc6df$export$2e2bcd8739ae039 as default};
