// @react-spectrum/s2/illustrations/linear/internal/S2_lin_ribbon_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $lXbQg$reactjsxruntime = require("react/jsx-runtime");
var $lXbQg$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $830d996f8a7f91af$export$2e2bcd8739ae039);




const $830d996f8a7f91af$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lXbQg$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $lXbQg$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24 8.75c-5.247 0-9.5 4.253-9.5 9.5s4.253 9.5 9.5 9.5 9.5-4.253 9.5-9.5-4.254-9.5-9.5-9.5m-12 9.5c0-6.627 5.372-12 12-12 6.627 0 12 5.373 12 12 0 3.261-1.301 6.218-3.412 8.381l3.407 8.435c.668 1.652-.758 3.387-2.508 3.053l-2.081-.398-1.22 1.732c-1.027 1.456-3.259 1.198-3.926-.454l-2.275-5.63-2.234 5.529c-.673 1.666-2.932 1.91-3.945.425l-1.187-1.74-2.062.428c-1.76.364-3.215-1.38-2.542-3.047l3.376-8.354c-2.1-2.16-3.392-5.11-3.392-8.36m5.412 10.032-2.904 7.186 1.768-.366c.89-.184 1.802.185 2.314.935l1.018 1.491 2.974-7.36c-1.894-.224-3.652-.887-5.17-1.886m7.977 1.888c1.895-.218 3.655-.878 5.175-1.873l2.942 7.28-1.79-.342c-.867-.166-1.751.192-2.26.914l-1.05 1.489z"
        })
    });
const $830d996f8a7f91af$var$ForwardRef = /*#__PURE__*/ (0, $lXbQg$react.forwardRef)($830d996f8a7f91af$var$SvgComponent);
var $830d996f8a7f91af$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($830d996f8a7f91af$var$ForwardRef);
