// @react-spectrum/s2/illustrations/linear/internal/S2_lin_clock_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6ZpCe$reactjsxruntime = require("react/jsx-runtime");
var $6ZpCe$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $153d632a165ad7df$export$2e2bcd8739ae039);




const $153d632a165ad7df$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6ZpCe$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6ZpCe$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M80 27.74c-28.719 0-52 23.281-52 52s23.281 52 52 52 52-23.281 52-52-23.281-52-52-52m-60 52c0-33.137 26.863-60 60-60s60 26.863 60 60-26.863 60-60 60-60-26.863-60-60m64-2.31V40.74c0-2.209-1.79-4-4-4s-4 1.791-4 4v38.924q-.004.239.02.476c.057.578.238 1.121.515 1.6s.656.908 1.128 1.247q.194.14.404.256l21.754 12.56c1.913 1.104 4.359.448 5.464-1.465 1.104-1.913.449-4.36-1.464-5.464z"
        })
    });
const $153d632a165ad7df$var$ForwardRef = /*#__PURE__*/ (0, $6ZpCe$react.forwardRef)($153d632a165ad7df$var$SvgComponent);
var $153d632a165ad7df$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($153d632a165ad7df$var$ForwardRef);
