// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noFilter_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6q0lq$jsx} from "react/jsx-runtime";
import {forwardRef as $6q0lq$forwardRef} from "react";





const $79327b7723eaef24$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6q0lq$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6q0lq$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M61.536 20.586c0-8.147 6.603-14.75 14.75-14.75s14.75 6.603 14.75 14.75-6.603 14.75-14.75 14.75-14.75-6.603-14.75-14.75m21.043 6.293c.781-.782.781-2.054 0-2.836h.02l-3.457-3.457 3.457-3.457c.781-.782.781-2.054 0-2.836-.782-.781-2.054-.781-2.836 0l-3.457 3.457-3.457-3.457c-.782-.781-2.054-.781-2.836 0-.781.782-.781 2.054 0 2.836l3.457 3.457-3.457 3.457c-.781.782-.781 2.054 0 2.836.391.39.902.59 1.413.59s1.022-.2 1.413-.59l3.457-3.457 3.457 3.457c.39.39.902.59 1.413.59s1.022-.2 1.413-.59M24.562 15.74c-5.457 0-8.47 6.277-5.223 10.593q.032.042.066.083L41.05 52.222v27.212c0 3.663 4.318 5.707 7.111 3.222l5.347-4.757c.92-.82 1.443-1.994 1.443-3.222V52.222l6.232-7.43 5.323-6.347c.71-.846.6-2.108-.247-2.818-.846-.71-2.108-.599-2.817.248l-5.324 6.347-6.699 7.987c-.302.36-.468.815-.468 1.285v23.183c0 .094-.04.179-.102.233l-5.346 4.758c-.065.056-.109.067-.136.07-.037.005-.089 0-.146-.026s-.1-.064-.126-.103c-.02-.03-.045-.08-.045-.175v-27.94c0-.47-.165-.925-.467-1.285L22.508 23.891c-1.268-1.743-.007-4.15 2.054-4.15H56.47c1.104 0 2-.896 2-2s-.896-2-2-2z"
        })
    });
const $79327b7723eaef24$var$ForwardRef = /*#__PURE__*/ (0, $6q0lq$forwardRef)($79327b7723eaef24$var$SvgComponent);
var $79327b7723eaef24$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($79327b7723eaef24$var$ForwardRef);


export {$79327b7723eaef24$export$2e2bcd8739ae039 as default};
