// @react-spectrum/s2/illustrations/linear/internal/S2_lin_commentText_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $ahs1h$reactjsxruntime = require("react/jsx-runtime");
var $ahs1h$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $72b3e8d17ec85935$export$2e2bcd8739ae039);




const $72b3e8d17ec85935$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ahs1h$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $ahs1h$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M20.64 15.416c-5.838 0-10.64 4.615-10.64 10.5v28c0 5.885 4.802 10.5 10.64 10.5h5.368c.012 3.816.02 8.529-.008 12.252-.03 4.206 4.916 6.426 8.07 3.714l.002-.003L52.74 64.416h22.62c5.839 0 10.641-4.615 10.641-10.5v-28c0-5.885-4.802-10.5-10.64-10.5zM14 25.916c0-3.6 2.935-6.5 6.64-6.5h54.72c3.705 0 6.64 2.9 6.64 6.5v28c0 3.6-2.935 6.5-6.64 6.5H51.26l-.56.48-19.234 16.448-.006.005c-.592.51-1.466.065-1.461-.652.032-4.429.017-10.238 0-14.29l-.008-1.991H20.64c-3.705 0-6.64-2.9-6.64-6.5zm16 5.979c-1.105 0-2 .895-2 2s.895 2 2 2h36c1.105 0 2-.896 2-2s-.895-2-2-2zm0 12c-1.105 0-2 .895-2 2s.895 2 2 2h36c1.105 0 2-.896 2-2s-.895-2-2-2z"
        })
    });
const $72b3e8d17ec85935$var$ForwardRef = /*#__PURE__*/ (0, $ahs1h$react.forwardRef)($72b3e8d17ec85935$var$SvgComponent);
var $72b3e8d17ec85935$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($72b3e8d17ec85935$var$ForwardRef);
