// @react-spectrum/s2/illustrations/linear/internal/S2_lin_channel_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7uNiE$jsx} from "react/jsx-runtime";
import {forwardRef as $7uNiE$forwardRef} from "react";





const $e565af79d4f5c831$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7uNiE$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7uNiE$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M33.62 28.094q.283 0 .558-.031l5.338 9.245c-2.653 2.108-4.504 5.183-5.02 8.692H23.823c-.771-1.766-2.533-3-4.584-3-2.761 0-5 2.239-5 5s2.239 5 5 5c2.05 0 3.813-1.234 4.584-3h10.674c.516 3.51 2.366 6.584 5.02 8.692l-5.338 9.246q-.276-.03-.559-.03c-2.761 0-5 2.238-5 5 0 2.76 2.239 5 5 5s5-2.24 5-5c0-1.113-.363-2.14-.977-2.97l5.336-9.243c1.554.615 3.248.953 5.021.953s3.466-.338 5.02-.953l5.337 9.243c-.614.83-.977 1.858-.977 2.97 0 2.76 2.239 5 5 5s5-2.24 5-5c0-2.762-2.238-5-5-5q-.283 0-.56.03l-5.337-9.246c2.653-2.108 4.504-5.183 5.02-8.692h10.674c.771 1.766 2.533 3 4.584 3 2.761 0 5-2.239 5-5s-2.239-5-5-5c-2.05 0-3.813 1.234-4.584 3H61.5c-.515-3.509-2.366-6.584-5.02-8.692l5.338-9.245q.276.03.56.03c2.762 0 5-2.238 5-5 0-2.76-2.238-5-5-5s-5 2.24-5 5c0 1.112.363 2.14.977 2.97l-5.337 9.242c-1.554-.614-3.247-.953-5.02-.953s-3.467.339-5.02.954l-5.337-9.243c.614-.83.977-1.857.977-2.97 0-2.76-2.238-5-5-5s-5 2.24-5 5c0 2.762 2.239 5 5 5m14.379 10.258c-5.328 0-9.648 4.32-9.648 9.648 0 5.329 4.32 9.648 9.648 9.648s9.648-4.32 9.648-9.648-4.32-9.647-9.648-9.647"
        })
    });
const $e565af79d4f5c831$var$ForwardRef = /*#__PURE__*/ (0, $7uNiE$forwardRef)($e565af79d4f5c831$var$SvgComponent);
var $e565af79d4f5c831$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e565af79d4f5c831$var$ForwardRef);


export {$e565af79d4f5c831$export$2e2bcd8739ae039 as default};
