// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileAlert_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $id4la$jsx} from "react/jsx-runtime";
import {forwardRef as $id4la$forwardRef} from "react";





const $a43fea7348cdfce4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $id4la$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $id4la$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8.43 9.96c0-2.382 2.012-4.2 4.36-4.2h9.84c1.099 0 2.172.398 2.991 1.146l.002.001 9.934 9.024.002.002c.864.78 1.37 1.883 1.37 3.057v6.045c0 .69-.559 1.25-1.25 1.25-.69 0-1.25-.56-1.25-1.25V19.26h-8.6c-1.76 0-3.38-1.324-3.38-3.19V8.26h-9.66c-1.092 0-1.86.821-1.86 1.7v24.12c0 .878.768 1.7 1.86 1.7h10.49c.691 0 1.25.56 1.25 1.25s-.559 1.25-1.25 1.25H12.79c-2.347 0-4.36-1.819-4.36-4.2zm16.52-.287 7.8 7.087h-6.92c-.6 0-.88-.416-.88-.69zM26 35.07c0-4.18 3.39-7.57 7.57-7.57s7.57 3.39 7.57 7.57-3.39 7.57-7.57 7.57S26 39.25 26 35.07m7.57-4.83c-.69 0-1.25.56-1.25 1.25v3.75c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25v-3.75c0-.69-.56-1.25-1.25-1.25m1.29 8.94c.04-.09.08-.18.08-.27q.03-.135.03-.27t-.03-.27c-.01-.09-.04-.18-.08-.27-.03-.08-.07-.16-.13-.24-.05-.07-.11-.14-.17-.21-.06-.06-.14-.12-.21-.17-.08-.05-.16-.1-.24-.13-.09-.04-.18-.07-.26-.08-.18-.04-.37-.04-.55 0-.09.01-.18.04-.26.08-.09.03-.17.08-.24.13-.08.05-.15.11-.22.17-.06.07-.12.14-.17.21-.05.08-.1.16-.13.24-.04.09-.06.18-.08.27q-.03.135-.03.27t.03.27c.02.09.04.18.08.27.03.08.08.16.13.24.05.07.11.14.17.21.07.06.14.12.22.17.07.05.15.1.24.13.08.04.17.06.26.08.09.03.18.03.27.03s.19-.01.28-.03l.027-.007c.073-.018.153-.038.233-.073.08-.03.16-.08.24-.13.07-.05.15-.11.21-.17.06-.07.12-.14.17-.21.06-.08.1-.16.13-.24"
        })
    });
const $a43fea7348cdfce4$var$ForwardRef = /*#__PURE__*/ (0, $id4la$forwardRef)($a43fea7348cdfce4$var$SvgComponent);
var $a43fea7348cdfce4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a43fea7348cdfce4$var$ForwardRef);


export {$a43fea7348cdfce4$export$2e2bcd8739ae039 as default};
