// @react-spectrum/s2/illustrations/linear/internal/S2_lin_redo_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $jQlV9$jsx} from "react/jsx-runtime";
import {forwardRef as $jQlV9$forwardRef} from "react";





const $c314fe005dd15ab7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jQlV9$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $jQlV9$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M42.384 17.384c.488-.488.488-1.28 0-1.768l-7-7c-.488-.488-1.28-.488-1.768 0s-.488 1.28 0 1.768l4.634 4.634H18.701c-6.907 0-12.506 5.6-12.506 12.506s5.6 12.506 12.506 12.506h7.975c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H18.7c-5.526 0-10.006-4.48-10.006-10.006s4.48-10.006 10.006-10.006h20.013l-5.098 5.098c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0z"
        })
    });
const $c314fe005dd15ab7$var$ForwardRef = /*#__PURE__*/ (0, $jQlV9$forwardRef)($c314fe005dd15ab7$var$SvgComponent);
var $c314fe005dd15ab7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c314fe005dd15ab7$var$ForwardRef);


export {$c314fe005dd15ab7$export$2e2bcd8739ae039 as default};
