// @react-spectrum/s2/illustrations/linear/internal/S2_lin_noSharedFile_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $dSRxo$jsx} from "react/jsx-runtime";
import {forwardRef as $dSRxo$forwardRef} from "react";





const $8c3bcca9943b7ec4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dSRxo$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dSRxo$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M21.99 10C17.571 10 14 13.59 14 18v47.99c0 4.41 3.571 8 7.99 8H46c1.105 0 2-.895 2-2s-.895-2-2-2H21.99c-2.201 0-3.99-1.79-3.99-4V18c0-2.21 1.789-4 3.99-4h19.45c1.032 0 2.022.396 2.752 1.1l.004.004 19.57 18.75C64.555 34.61 65 35.649 65 36.74v14.75c0 1.105.895 2 2 2s2-.895 2-2V36.74c0-2.188-.894-4.269-2.467-5.775L46.968 12.22l-.002-.002C45.476 10.783 43.487 10 41.44 10zm44.75 76.49c8.146 0 14.75-6.604 14.75-14.75s-6.604-14.75-14.75-14.75-14.75 6.604-14.75 14.75 6.604 14.75 14.75 14.75m2.248-21.5c0-1.105-.895-2-2-2s-2 .895-2 2v5h-5c-1.104 0-2 .895-2 2s.896 2 2 2h5v5c0 1.104.896 2 2 2s2-.896 2-2v-5h5c1.105 0 2-.896 2-2s-.895-2-2-2h-5zM45.74 52.157c-4.294 0-8.014 3.284-8.762 7.67-.186 1.088-1.219 1.82-2.308 1.635s-1.82-1.219-1.635-2.308c1.052-6.166 6.321-10.997 12.705-10.997 1.77 0 3.462.375 5.002 1.05 1.012.442 1.473 1.622 1.03 2.633-.443 1.012-1.622 1.473-2.634 1.03-1.05-.46-2.198-.713-3.398-.713m-11.834-2.048c1.004-.46 1.445-1.647.985-2.652-.46-1.004-1.647-1.445-2.651-.985-3.944 1.806-6.891 5.472-7.67 9.87-.192 1.087.533 2.125 1.62 2.318 1.088.192 2.126-.533 2.319-1.621.542-3.064 2.61-5.654 5.397-6.93m11.366-14.752c-1.988 0-3.6 1.612-3.6 3.6s1.612 3.6 3.6 3.6 3.6-1.612 3.6-3.6-1.612-3.6-3.6-3.6m-7.6 3.6c0-4.197 3.403-7.6 7.6-7.6 4.198 0 7.6 3.403 7.6 7.6s-3.402 7.6-7.6 7.6-7.6-3.403-7.6-7.6m-4.467-2.036c0-1.837 1.554-3.43 3.6-3.43 1.105 0 2-.896 2-2s-.895-2-2-2c-4.139 0-7.6 3.269-7.6 7.43 0 2.117.907 4.018 2.337 5.36.805.757 2.07.717 2.827-.088.756-.806.716-2.071-.09-2.827-.673-.633-1.074-1.498-1.074-2.445"
        })
    });
const $8c3bcca9943b7ec4$var$ForwardRef = /*#__PURE__*/ (0, $dSRxo$forwardRef)($8c3bcca9943b7ec4$var$SvgComponent);
var $8c3bcca9943b7ec4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8c3bcca9943b7ec4$var$ForwardRef);


export {$8c3bcca9943b7ec4$export$2e2bcd8739ae039 as default};
