// @react-spectrum/s2/illustrations/linear/internal/S2_lin_3D_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $ajfyp$jsx} from "react/jsx-runtime";
import {forwardRef as $ajfyp$forwardRef} from "react";





const $81bc39d434e6a83e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ajfyp$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $ajfyp$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M21.497 5.972c1.648-.968 3.693-.962 5.337.014l11.994 7.12c1.593.945 2.57 2.661 2.57 4.514v13.42c0 1.903-1.03 3.656-2.69 4.584l-11.993 6.7c-1.583.886-3.511.89-5.099.013L9.462 35.623C7.789 34.7 6.75 32.94 6.75 31.028V17.633c0-1.862.987-3.585 2.593-4.528zm4.06 2.163c-.86-.51-1.931-.514-2.795-.007l-12.154 7.134q-.176.104-.333.23l13.805 7.367 13.932-7.26q-.21-.195-.46-.344zM9.25 31.028v-13.25l13.666 7.293v15.126l-.091-.048-12.155-6.714c-.876-.484-1.42-1.406-1.42-2.407m16.166 9.158.08-.044 11.993-6.7c.87-.486 1.409-1.405 1.409-2.401V17.956l-13.483 7.026z"
        })
    });
const $81bc39d434e6a83e$var$ForwardRef = /*#__PURE__*/ (0, $ajfyp$forwardRef)($81bc39d434e6a83e$var$SvgComponent);
var $81bc39d434e6a83e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($81bc39d434e6a83e$var$ForwardRef);


export {$81bc39d434e6a83e$export$2e2bcd8739ae039 as default};
