// @react-spectrum/s2/illustrations/linear/internal/S2_lin_targeted_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bhdlY$reactjsxruntime = require("react/jsx-runtime");
var $bhdlY$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6669aab28101b0a3$export$2e2bcd8739ae039);




const $6669aab28101b0a3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bhdlY$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bhdlY$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M9.619 18.305c-.696.179-1.243.716-1.433 1.41-.19.692.007 1.434.515 1.942l9.197 9.198c.415.415.992.626 1.577.579l8.202-.668 12.447 12.447c.781.781 2.048.781 2.829 0 .78-.781.78-2.047 0-2.828L30.507 27.939l.668-8.205c.047-.585-.165-1.162-.58-1.577L21.398 8.96c-.508-.508-1.25-.705-1.943-.515s-1.23.737-1.409 1.433l-1.718 6.709zm38.38-1.066c-3.73 0-7.301.663-10.604 1.876-1.037.381-2.186-.15-2.567-1.188-.38-1.037.15-2.186 1.188-2.567C39.754 13.987 43.792 13.24 48 13.24c19.198 0 34.762 15.563 34.762 34.762S67.198 82.763 48 82.763c-19.199 0-34.762-15.564-34.762-34.762 0-4.097.71-8.033 2.014-11.689.372-1.04 1.516-1.582 2.556-1.211s1.583 1.516 1.212 2.556c-1.154 3.23-1.782 6.711-1.782 10.344 0 16.99 13.772 30.762 30.762 30.762S78.762 64.99 78.762 48.001 64.989 17.239 48 17.239m0 15.506c-2.028 0-3.959.394-5.724 1.11-1.023.414-2.19-.08-2.604-1.104-.415-1.023.079-2.19 1.103-2.604 2.233-.905 4.674-1.402 7.225-1.402 10.635 0 19.255 8.62 19.255 19.255S58.634 67.255 48 67.255 28.744 58.635 28.744 48c0-2.377.432-4.659 1.223-6.766.388-1.034 1.541-1.558 2.575-1.17s1.558 1.541 1.17 2.575c-.625 1.665-.968 3.47-.968 5.36 0 8.426 6.83 15.256 15.255 15.256S63.254 56.425 63.254 48 56.424 32.745 48 32.745m-5.344 18.562c4.52-.42 8.118-4.018 8.538-8.538 1.807 1.089 3.016 3.07 3.016 5.333 0 3.436-2.786 6.22-6.221 6.22-2.263 0-4.244-1.208-5.333-3.015"
        })
    });
const $6669aab28101b0a3$var$ForwardRef = /*#__PURE__*/ (0, $bhdlY$react.forwardRef)($6669aab28101b0a3$var$SvgComponent);
var $6669aab28101b0a3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6669aab28101b0a3$var$ForwardRef);
