// @react-spectrum/s2/illustrations/linear/internal/S2_lin_flagCheckmark_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $gqZXg$reactjsxruntime = require("react/jsx-runtime");
var $gqZXg$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2a2b9094eae3b188$export$2e2bcd8739ae039);




const $2a2b9094eae3b188$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gqZXg$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $gqZXg$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M11.742 7.86c.69 0 1.25.559 1.25 1.25v.661h20.266c2.347 0 4.25 1.903 4.25 4.25v12.98c0 2.347-1.903 4.25-4.25 4.25H12.992v7.64c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25V9.109c0-.69.56-1.25 1.25-1.25m1.25 20.89V12.273h20.266c.966 0 1.75.783 1.75 1.75V27c0 .967-.784 1.75-1.75 1.75zM30.455 18.2c.51-.466.547-1.256.082-1.767s-1.256-.546-1.766-.081L24 20.699l-.039.035-1.381 1.26-2.599-2.424c-.505-.47-1.295-.443-1.766.062s-.444 1.296.061 1.767l3.441 3.209c.477.444 1.214.448 1.695.01zm-7.884 5.494.842.923z"
        })
    });
const $2a2b9094eae3b188$var$ForwardRef = /*#__PURE__*/ (0, $gqZXg$react.forwardRef)($2a2b9094eae3b188$var$SvgComponent);
var $2a2b9094eae3b188$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2a2b9094eae3b188$var$ForwardRef);
