// @react-spectrum/s2/illustrations/linear/internal/S2_lin_arrowDown_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $7pHYN$jsx} from "react/jsx-runtime";
import {forwardRef as $7pHYN$forwardRef} from "react";





const $54be8303c48923ab$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7pHYN$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $7pHYN$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M50 15c0-1.105-.895-2-2-2s-2 .895-2 2v63q0 .09.008.18L35.414 67.586c-.78-.781-2.047-.781-2.828 0-.781.78-.781 2.047 0 2.828l14 14c.78.781 2.047.781 2.828 0l14-14c.781-.78.781-2.047 0-2.828-.78-.781-2.047-.781-2.828 0L49.992 78.18Q50 78.09 50 78zm-1.82 64.992Q48.09 80 48 80t-.18-.008l.18.18z"
        })
    });
const $54be8303c48923ab$var$ForwardRef = /*#__PURE__*/ (0, $7pHYN$forwardRef)($54be8303c48923ab$var$SvgComponent);
var $54be8303c48923ab$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($54be8303c48923ab$var$ForwardRef);


export {$54be8303c48923ab$export$2e2bcd8739ae039 as default};
