// @react-spectrum/s2/illustrations/linear/internal/S2_lin_color_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $cjbB7$reactjsxruntime = require("react/jsx-runtime");
var $cjbB7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e70b41e9ad051add$export$2e2bcd8739ae039);




const $e70b41e9ad051add$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cjbB7$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $cjbB7$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M84.287 28.219c-29.02 0-52.611 23.762-52.611 53.155 0 28.821 26.789 50.407 56.421 50.407 7.936 0 18.231-2.448 26.277-6.334 4.019-1.941 7.224-4.122 9.276-6.311 2.055-2.194 2.585-3.967 2.397-5.352-.347-2.573-1.207-3.58-1.925-4.124-.923-.7-2.235-1.115-4.327-1.544q-.374-.075-.807-.159c-1.732-.335-4.111-.795-6.064-1.789-1.265-.645-2.52-1.573-3.522-2.947-1.003-1.376-1.603-2.994-1.859-4.8-.67-4.736.245-8.599 2.177-11.904 1.833-3.136 4.498-5.603 6.895-7.726l.925-.816c4.646-4.092 8.677-7.642 10.216-14.026 1.959-8.125-1.239-16.872-8.958-23.84-7.708-6.956-19.749-11.89-34.511-11.89M23.676 81.374c0-33.739 27.1-61.155 60.611-61.155 16.503 0 30.525 5.516 39.871 13.952 9.334 8.425 14.182 20.01 11.375 31.654-2.155 8.938-8.02 14.063-12.521 17.997l-.002.001-.002.001q-.56.49-1.089.957c-2.413 2.136-4.174 3.86-5.292 5.773-1.02 1.744-1.581 3.789-1.163 6.747.103.724.292 1.055.402 1.207.112.153.296.33.688.53.868.442 1.945.658 3.734 1.016q.515.103 1.114.225c2.124.436 5.054 1.112 7.552 3.004 2.702 2.048 4.441 5.134 5.021 9.431.63 4.66-1.522 8.726-4.487 11.891-2.969 3.169-7.121 5.866-11.635 8.046-9.022 4.358-20.5 7.13-29.756 7.13-32.898 0-64.421-24.097-64.421-58.407m72.327-30.49c0 5.522-4.477 9.998-10 9.998s-9.999-4.476-9.999-9.999c0-5.522 4.477-10 10-10s9.999 4.478 9.999 10M61.005 74.072c5.523 0 10-4.477 10-10s-4.477-9.999-10-9.999-10 4.477-10 10 4.478 9.999 10 9.999m10 22.553c0 5.523-4.477 10-10 10s-10-4.477-10-10 4.478-9.999 10-9.999 10 4.477 10 10m14.999 24.253c5.522 0 9.999-4.477 9.999-10s-4.477-9.999-10-9.999-9.999 4.477-9.999 9.999 4.477 10 10 10"
        })
    });
const $e70b41e9ad051add$var$ForwardRef = /*#__PURE__*/ (0, $cjbB7$react.forwardRef)($e70b41e9ad051add$var$SvgComponent);
var $e70b41e9ad051add$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e70b41e9ad051add$var$ForwardRef);
