// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileVideo_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9PWK5$jsx} from "react/jsx-runtime";
import {forwardRef as $9PWK5$forwardRef} from "react";





const $0cddd9bebb2f3e5f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9PWK5$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9PWK5$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M10.25 12.462c0-2.631 2.176-4.712 4.796-4.712h9.167c1.262 0 2.479.49 3.38 1.37l9.742 9.51c.902.881 1.415 2.083 1.415 3.342v13.566c0 2.631-2.176 4.712-4.795 4.712h-18.91c-2.62 0-4.795-2.081-4.795-4.712zm4.796-2.212c-1.296 0-2.296 1.018-2.296 2.211v23.077c0 1.195 1 2.212 2.296 2.212h18.909c1.295 0 2.295-1.017 2.295-2.212V21.972c0-.577-.234-1.136-.662-1.553l-9.741-9.51c-.428-.419-1.016-.659-1.634-.659zm6.816 11.947 6.355 3.645c.712.408.71 1.426-.002 1.833l-6.364 3.63c-.712.406-1.602-.104-1.601-.919l.009-7.275c0-.814.892-1.323 1.603-.914"
        })
    });
const $0cddd9bebb2f3e5f$var$ForwardRef = /*#__PURE__*/ (0, $9PWK5$forwardRef)($0cddd9bebb2f3e5f$var$SvgComponent);
var $0cddd9bebb2f3e5f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0cddd9bebb2f3e5f$var$ForwardRef);


export {$0cddd9bebb2f3e5f$export$2e2bcd8739ae039 as default};
