// @react-spectrum/s2/illustrations/linear/internal/S2_lin_diamond_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $g0i2x$jsx} from "react/jsx-runtime";
import {forwardRef as $g0i2x$forwardRef} from "react";





const $5063ffd90ce7f2f6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $g0i2x$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $g0i2x$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.842 7.102c-.859 0-1.683.34-2.292.945l-7.777 7.734c-1.156 1.15-1.282 2.977-.294 4.274l14.908 19.564c1.3 1.705 3.864 1.708 5.166.005l14.965-19.563c.994-1.299.867-3.135-.296-4.285L33.399 8.04c-.608-.602-1.43-.94-2.285-.94zm-.529 2.718c.14-.14.33-.218.529-.218h3.281l-3.601 7.517h-7.55zm2.981 7.3 3.601-7.518h2.216l3.602 7.517zm9.265 2.5h-9.111l4.555 16.212zM26.77 35.225l4.386-15.607h7.553zm4.715-18.107-3.601-7.517h3.23c.197 0 .387.077.527.216l7.384 7.301zM9.29 19.62h7.56l4.418 15.719z"
        })
    });
const $5063ffd90ce7f2f6$var$ForwardRef = /*#__PURE__*/ (0, $g0i2x$forwardRef)($5063ffd90ce7f2f6$var$SvgComponent);
var $5063ffd90ce7f2f6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5063ffd90ce7f2f6$var$ForwardRef);


export {$5063ffd90ce7f2f6$export$2e2bcd8739ae039 as default};
