// @react-spectrum/s2/illustrations/linear/internal/S2_lin_clock_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $kHpmv$reactjsxruntime = require("react/jsx-runtime");
var $kHpmv$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $02003549e13ac162$export$2e2bcd8739ae039);




const $02003549e13ac162$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kHpmv$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $kHpmv$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 16.74c-17.12 0-31 13.88-31 31 0 17.121 13.88 31 31 31s31-13.879 31-31c0-17.12-13.88-31-31-31m-35 31c0-19.33 15.67-35 35-35s35 15.67 35 35-15.67 35-35 35-35-15.67-35-35m37-1.154V24.74c0-1.104-.895-2-2-2s-2 .896-2 2v22.962q-.002.12.01.238c.029.29.119.56.258.8.138.24.328.455.565.624q.096.07.2.127l12.826 7.405c.956.553 2.18.225 2.732-.732.552-.956.224-2.18-.732-2.732z"
        })
    });
const $02003549e13ac162$var$ForwardRef = /*#__PURE__*/ (0, $kHpmv$react.forwardRef)($02003549e13ac162$var$SvgComponent);
var $02003549e13ac162$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($02003549e13ac162$var$ForwardRef);
