// @react-spectrum/s2/illustrations/linear/internal/S2_lin_codeBrackets_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $hIITn$jsx} from "react/jsx-runtime";
import {forwardRef as $hIITn$forwardRef} from "react";





const $dea2a7027a4ca572$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hIITn$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $hIITn$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M8.25 14.4c0-1.518 1.231-2.75 2.75-2.75h26c1.519 0 2.75 1.232 2.75 2.75v19.2c0 1.52-1.231 2.75-2.75 2.75H11c-1.519 0-2.75-1.23-2.75-2.75zM11 9.15c-2.9 0-5.25 2.35-5.25 5.25v19.2c0 2.9 2.35 5.25 5.25 5.25h26c2.9 0 5.25-2.35 5.25-5.25V14.4c0-2.9-2.35-5.25-5.25-5.25zm8.174 10.334c.488-.488.488-1.28 0-1.768s-1.28-.488-1.768 0l-5.21 5.21c-.476.477-.49 1.244-.03 1.737l5.21 5.59c.47.505 1.261.532 1.766.062s.533-1.262.063-1.767l-4.388-4.707zm9.684-1.798c.505-.47 1.296-.443 1.767.062l5.21 5.59c.459.493.445 1.26-.031 1.736l-5.21 5.21c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.767l4.357-4.357-4.387-4.707c-.471-.505-.443-1.296.062-1.767m-1.242-1.303c.2-.66-.172-1.358-.833-1.559-.66-.2-1.359.173-1.56.834l-4.84 15.96c-.2.66.174 1.358.834 1.559.661.2 1.359-.173 1.56-.834z"
        })
    });
const $dea2a7027a4ca572$var$ForwardRef = /*#__PURE__*/ (0, $hIITn$forwardRef)($dea2a7027a4ca572$var$SvgComponent);
var $dea2a7027a4ca572$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($dea2a7027a4ca572$var$ForwardRef);


export {$dea2a7027a4ca572$export$2e2bcd8739ae039 as default};
