// @react-spectrum/s2/illustrations/linear/internal/S2_lin_image_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $2H7Hp$jsx} from "react/jsx-runtime";
import {forwardRef as $2H7Hp$forwardRef} from "react";





const $3bbaca709d56ce84$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2H7Hp$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $2H7Hp$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M40.75 14c0-1.519-1.231-2.75-2.75-2.75H10c-1.519 0-2.75 1.231-2.75 2.75v19.97l7.722-11.93c1.33-2.055 4.34-2.055 5.669 0l6.144 9.49 3.45-5.217.005-.008c1.338-1.99 4.27-1.994 5.606 0l.002.002 4.902 7.262zm-23.68 9.399-8.414 13c.397.224.856.351 1.344.351h13.354q.043-.097.104-.19l1.822-2.754-6.738-10.407c-.344-.533-1.127-.533-1.471 0m22.485 12.87c-.442.303-.978.481-1.555.481H26.33l5.987-9.053c.35-.517 1.11-.512 1.453.001l.003.004zM38 8.75c2.9 0 5.25 2.35 5.25 5.25v20c0 2.9-2.35 5.25-5.25 5.25H10c-2.9 0-5.25-2.35-5.25-5.25V14c0-2.9 2.35-5.25 5.25-5.25zm-6 8.75c0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5-1.12-2.5-2.5-2.5-2.5 1.12-2.5 2.5"
        })
    });
const $3bbaca709d56ce84$var$ForwardRef = /*#__PURE__*/ (0, $2H7Hp$forwardRef)($3bbaca709d56ce84$var$SvgComponent);
var $3bbaca709d56ce84$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($3bbaca709d56ce84$var$ForwardRef);


export {$3bbaca709d56ce84$export$2e2bcd8739ae039 as default};
