// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileText_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $arRrT$reactjsxruntime = require("react/jsx-runtime");
var $arRrT$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b9c8d7bd2aab1d3a$export$2e2bcd8739ae039);




const $b9c8d7bd2aab1d3a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $arRrT$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $arRrT$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.136 24C38.267 24 31 31.12 31 40v80c0 8.879 7.267 16 16.136 16h64.728c8.869 0 16.136-7.121 16.136-16V72.97c0-4.258-1.711-8.332-4.742-11.33L89.913 28.67c-3.03-2.995-7.13-4.67-11.395-4.67zM39 40c0-4.376 3.6-8 8.136-8h31.382c2.173 0 4.247.854 5.77 2.36l33.345 32.97c1.521 1.504 2.367 3.534 2.367 5.64V120c0 4.376-3.6 8-8.136 8H47.136C42.6 128 39 124.376 39 120zm19.066 32.498C58.639 72.18 59.298 72 60 72h40c1.942 0 3.561 1.384 3.924 3.22q.076.379.076.78v8c0 2.21-1.791 4-4 4s-4-1.79-4-4v-4H84v32h4c2.21 0 4 1.791 4 4s-1.79 4-4 4H72c-2.21 0-4-1.791-4-4s1.79-4 4-4h4V80H64v4c0 2.21-1.79 4-4 4s-4-1.79-4-4v-8c0-.552.112-1.078.314-1.557.35-.828.971-1.513 1.752-1.945"
        })
    });
const $b9c8d7bd2aab1d3a$var$ForwardRef = /*#__PURE__*/ (0, $arRrT$react.forwardRef)($b9c8d7bd2aab1d3a$var$SvgComponent);
var $b9c8d7bd2aab1d3a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b9c8d7bd2aab1d3a$var$ForwardRef);
