// @react-spectrum/s2/illustrations/linear/internal/S2_lin_rotateCCW_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $bNRT0$reactjsxruntime = require("react/jsx-runtime");
var $bNRT0$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e9cbfc4629b61465$export$2e2bcd8739ae039);




const $e9cbfc4629b61465$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bNRT0$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 97,
        viewBox: "0 0 96 97",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $bNRT0$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M79 48.492c0-17.12-13.88-31-31-31-12.189 0-22.738 7.035-27.803 17.271l9.735-3.13c1.051-.338 2.178.24 2.516 1.292.338 1.051-.24 2.178-1.292 2.516L17.1 39.961c-1.052.338-2.179-.24-2.517-1.292l-4.52-14.056c-.338-1.052.24-2.178 1.292-2.517s2.178.24 2.516 1.292l2.953 9.181C22.618 21.247 34.4 13.492 48 13.492c19.33 0 35 15.67 35 35s-15.67 35-35 35c-15.118 0-27.992-9.583-32.888-23-.38-1.037.155-2.186 1.193-2.564 1.037-.379 2.185.155 2.564 1.193C23.21 71.01 34.616 79.492 48 79.492c17.122 0 31-13.879 31-31"
        })
    });
const $e9cbfc4629b61465$var$ForwardRef = /*#__PURE__*/ (0, $bNRT0$react.forwardRef)($e9cbfc4629b61465$var$SvgComponent);
var $e9cbfc4629b61465$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e9cbfc4629b61465$var$ForwardRef);
