// @react-spectrum/s2/illustrations/linear/internal/S2_lin_folderClose_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $ljNzE$reactjsxruntime = require("react/jsx-runtime");
var $ljNzE$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a2e3fa448607a33f$export$2e2bcd8739ae039);




const $a2e3fa448607a33f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ljNzE$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $ljNzE$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M15 22.667C15 21.14 16.189 20 17.534 20H33.94c.747 0 1.472.349 1.961.979L42.912 30H15zM11 30v-7.333C11 19.04 13.871 16 17.534 16H33.94c2.008 0 3.888.94 5.12 2.524l-1.333 1.036 1.332-1.036L47.98 30h26.046C80.092 30 85 34.947 85 41.02v27.96C85 75.052 80.092 80 74.025 80h-52.05C15.908 80 11 75.053 11 68.98V30m4 4v34.98c0 3.883 3.136 7.02 6.975 7.02h52.05C77.865 76 81 72.863 81 68.98V41.02c0-3.883-3.136-7.02-6.975-7.02H15"
        })
    });
const $a2e3fa448607a33f$var$ForwardRef = /*#__PURE__*/ (0, $ljNzE$react.forwardRef)($a2e3fa448607a33f$var$SvgComponent);
var $a2e3fa448607a33f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a2e3fa448607a33f$var$ForwardRef);
