// @react-spectrum/s2/illustrations/linear/internal/S2_lin_exposure_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $eFfWs$jsx} from "react/jsx-runtime";
import {forwardRef as $eFfWs$forwardRef} from "react";





const $0846de02a8503698$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eFfWs$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $eFfWs$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M24.458 39.414q-.24.007-.479.007c-5.646 0-10.581-3.04-13.258-7.574h16.127zM8.594 24.036c0 1.866.332 3.656.941 5.311h7.922L12.431 13.87c-2.389 2.71-3.837 6.269-3.837 10.166m5.846-12.072c2.622-2.075 5.936-3.314 9.54-3.314 2.48 0 4.825.588 6.901 1.632l-13.757 9.946zm3.49 10.746q.09-.044.173-.105l5.759-4.163 5.878 4.246-2.103 6.659H20.06q-.014-.122-.054-.244zm9.253 16.377c5.311-1.125 9.61-4.99 11.35-10.047l-6.659-4.81zm5.998-27.383-7.186 5.195 13.185 9.525q.184-1.168.185-2.388c0-5.047-2.43-9.527-6.184-12.332m-9.202 30.217c9.878 0 17.886-8.007 17.886-17.885S33.857 6.15 23.979 6.15 6.094 14.158 6.094 24.036 14.1 41.92 23.979 41.92"
        })
    });
const $0846de02a8503698$var$ForwardRef = /*#__PURE__*/ (0, $eFfWs$forwardRef)($0846de02a8503698$var$SvgComponent);
var $0846de02a8503698$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0846de02a8503698$var$ForwardRef);


export {$0846de02a8503698$export$2e2bcd8739ae039 as default};
