// @react-spectrum/s2/illustrations/linear/internal/S2_lin_artboard_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $s6dMe$reactjsxruntime = require("react/jsx-runtime");
var $s6dMe$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $21fbb8611c552ceb$export$2e2bcd8739ae039);




const $21fbb8611c552ceb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $s6dMe$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $s6dMe$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M28.756 9.016c0-1.105-.895-2-2-2s-2 .895-2 2v5.79c0 1.105.895 2 2 2s2-.895 2-2zm-14.74 10.74c-1.105 0-2 .896-2 2s.895 2 2 2h5.79c1.105 0 2-.895 2-2s-.895-2-2-2zm19.69 4c-2.734 0-4.95 2.216-4.95 4.95v46.329c0 2.733 2.216 4.95 4.95 4.95h37.063c2.733 0 4.95-2.217 4.95-4.95V47.009l-.002-.088h-17.11c-3.663 0-6.632-2.97-6.632-6.633V23.76q-.11-.005-.223-.005zM55.974 25.9v14.388c0 1.454 1.179 2.633 2.633 2.633h15.05zm-31.219 2.806c0-4.943 4.007-8.95 8.95-8.95h18.046c2.315 0 4.54.897 6.207 2.502L76.975 40.56c1.753 1.687 2.743 4.015 2.743 6.448v28.026c0 4.943-4.007 8.95-8.95 8.95H33.706c-4.942 0-8.949-4.007-8.949-8.95z"
        })
    });
const $21fbb8611c552ceb$var$ForwardRef = /*#__PURE__*/ (0, $s6dMe$react.forwardRef)($21fbb8611c552ceb$var$SvgComponent);
var $21fbb8611c552ceb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($21fbb8611c552ceb$var$ForwardRef);
