// @react-spectrum/s2/illustrations/linear/internal/S2_lin_close_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $5kpDV$jsx} from "react/jsx-runtime";
import {forwardRef as $5kpDV$forwardRef} from "react";





const $721480c88d50e7b0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5kpDV$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5kpDV$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M48 17c-17.12 0-31 13.88-31 31s13.88 31 31 31 31-13.88 31-31-13.88-31-31-31M13 48c0-19.33 15.67-35 35-35s35 15.67 35 35-15.67 35-35 35-35-15.67-35-35m24.43-10.571c.78-.782 2.047-.782 2.828 0l8.11 8.11 8.11-8.11c.781-.782 2.047-.782 2.828 0 .782.78.782 2.047 0 2.828l-8.11 8.11 8.11 8.11c.782.781.782 2.047 0 2.829-.78.78-2.047.78-2.828 0l-8.11-8.11-8.11 8.11c-.781.78-2.047.78-2.829 0-.78-.782-.78-2.048 0-2.829l8.11-8.11-8.11-8.11c-.78-.781-.78-2.047 0-2.828"
        })
    });
const $721480c88d50e7b0$var$ForwardRef = /*#__PURE__*/ (0, $5kpDV$forwardRef)($721480c88d50e7b0$var$SvgComponent);
var $721480c88d50e7b0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($721480c88d50e7b0$var$ForwardRef);


export {$721480c88d50e7b0$export$2e2bcd8739ae039 as default};
