// @react-spectrum/s2/illustrations/linear/internal/S2_lin_reportAbuse_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $6SQJU$jsx} from "react/jsx-runtime";
import {forwardRef as $6SQJU$forwardRef} from "react";





const $624ca166da1672e2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6SQJU$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6SQJU$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M44.553 22.988c-11.046 0-20 8.955-20 20v55.738c0 11.046 8.954 20 20 20h14.75l11.426 13.905c4.8 5.842 13.742 5.842 18.543 0l11.427-13.905h14.749c11.046 0 20-8.954 20-20V42.988c0-11.045-8.954-20-20-20zm-12 20c0-6.627 5.372-12 12-12h70.895c6.628 0 12 5.373 12 12v55.738c0 6.627-5.372 12-12 12h-16.64c-1.197 0-2.33.536-3.09 1.46L83.09 127.552c-1.6 1.947-4.58 1.947-6.18 0l-12.628-15.366c-.76-.924-1.894-1.46-3.09-1.46h-16.64c-6.628 0-12-5.373-12-12zM80 37.92c2.209 0 4 1.79 4 4v38.402c0 2.209-1.791 4-4 4s-4-1.791-4-4V41.92c0-2.21 1.79-4 4-4m0 66.973c3.313 0 6-2.687 6-6s-2.687-6-6-6-6 2.686-6 6 2.686 6 6 6"
        })
    });
const $624ca166da1672e2$var$ForwardRef = /*#__PURE__*/ (0, $6SQJU$forwardRef)($624ca166da1672e2$var$SvgComponent);
var $624ca166da1672e2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($624ca166da1672e2$var$ForwardRef);


export {$624ca166da1672e2$export$2e2bcd8739ae039 as default};
