// @react-spectrum/s2/illustrations/linear/internal/S2_lin_fileVideo_160.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $1FkK4$reactjsxruntime = require("react/jsx-runtime");
var $1FkK4$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1a6ff28a664915a6$export$2e2bcd8739ae039);




const $1a6ff28a664915a6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1FkK4$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $1FkK4$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M47.136 24C38.267 24 31 31.12 31 40v80c0 8.879 7.267 16 16.136 16h64.728c8.869 0 16.136-7.121 16.136-16V72.97c0-4.258-1.711-8.332-4.742-11.33L89.913 28.67c-3.03-2.995-7.13-4.67-11.395-4.67zM39 40c0-4.376 3.6-8 8.136-8h31.382c2.173 0 4.247.854 5.77 2.36l33.345 32.97c1.521 1.504 2.367 3.534 2.367 5.64V120c0 4.376-3.6 8-8.136 8H47.136C42.6 128 39 124.376 39 120zm30.811 32.94 20.933 12.12c2.344 1.357 2.34 4.743-.007 6.094l-20.962 12.069c-2.348 1.352-5.278-.344-5.275-3.053l.029-24.189c.003-2.709 2.938-4.398 5.282-3.04"
        })
    });
const $1a6ff28a664915a6$var$ForwardRef = /*#__PURE__*/ (0, $1FkK4$react.forwardRef)($1a6ff28a664915a6$var$SvgComponent);
var $1a6ff28a664915a6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1a6ff28a664915a6$var$ForwardRef);
