// @react-spectrum/s2/illustrations/linear/internal/S2_lin_browserError_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $9jwSt$jsx} from "react/jsx-runtime";
import {forwardRef as $9jwSt$forwardRef} from "react";





const $323f912dd8b1c813$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9jwSt$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9jwSt$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M37.75 11.5c0-1.519-1.231-2.75-2.75-2.75H9c-1.519 0-2.75 1.231-2.75 2.75v4.77h31.5zm-31.5 22V18.77h31.5V25c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25V11.5c0-2.9-2.35-5.25-5.25-5.25H9c-2.9 0-5.25 2.35-5.25 5.25v22c0 2.9 2.35 5.25 5.25 5.25h17.5c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H9c-1.519 0-2.75-1.231-2.75-2.75m3.25-21c0 .828.672 1.5 1.5 1.5s1.5-.672 1.5-1.5S11.828 11 11 11s-1.5.672-1.5 1.5M29 35c0-4.141 3.359-7.5 7.5-7.5S44 30.859 44 35s-3.359 7.5-7.5 7.5S29 39.141 29 35m10.859 3.359c.485-.486.485-1.269 0-1.754h.02L38.273 35l1.605-1.605c.486-.486.486-1.268 0-1.754-.485-.485-1.268-.485-1.753 0l-1.605 1.605-1.605-1.605c-.486-.485-1.268-.485-1.754 0-.485.486-.485 1.269 0 1.754L34.766 35l-1.605 1.605c-.485.486-.485 1.268 0 1.754.238.247.555.366.872.366s.634-.129.872-.366l1.605-1.605 1.605 1.605c.238.247.555.366.872.366s.634-.129.872-.366"
        })
    });
const $323f912dd8b1c813$var$ForwardRef = /*#__PURE__*/ (0, $9jwSt$forwardRef)($323f912dd8b1c813$var$SvgComponent);
var $323f912dd8b1c813$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($323f912dd8b1c813$var$ForwardRef);


export {$323f912dd8b1c813$export$2e2bcd8739ae039 as default};
