// @react-spectrum/s2/illustrations/linear/internal/S2_lin_brush_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $6K7NA$reactjsxruntime = require("react/jsx-runtime");
var $6K7NA$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a913cec3c4e17d09$export$2e2bcd8739ae039);




const $a913cec3c4e17d09$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6K7NA$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $6K7NA$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "m24.018 23.01 12.578-12.578c.91-.91 2.383-.91 3.292 0 .909.909.909 2.382 0 3.291L27.035 26.576c-.322-.824-.763-1.57-1.338-2.213q-.73-.817-1.679-1.353m-2.63-.906 13.44-13.44c1.886-1.885 4.943-1.885 6.828 0s1.885 4.942 0 6.827L27.675 29.472q.05.6.05 1.23c0 2.677-1.001 4.943-2.756 6.626-1.735 1.666-4.14 2.701-6.885 3.094-3.988.571-9.381.026-11.795-.274-.828-.103-1.339-.723-1.488-1.34-.148-.606.015-1.361.64-1.845 1.354-1.047 3.5-2.998 4.46-5.332.79-1.914 1.774-4.295 3.21-6.187 1.462-1.928 3.504-3.491 6.394-3.491q.991 0 1.884.15m-9.174 10.48c.81-1.968 1.682-4.04 2.888-5.629 1.179-1.554 2.562-2.502 4.403-2.502 2.114 0 3.475.62 4.328 1.575.874.98 1.392 2.51 1.392 4.674 0 2.007-.732 3.618-1.987 4.823-1.273 1.222-3.152 2.085-5.508 2.422-2.92.419-6.798.175-9.46-.093 1.38-1.247 3.038-3.072 3.944-5.27"
        })
    });
const $a913cec3c4e17d09$var$ForwardRef = /*#__PURE__*/ (0, $6K7NA$react.forwardRef)($a913cec3c4e17d09$var$SvgComponent);
var $a913cec3c4e17d09$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a913cec3c4e17d09$var$ForwardRef);
