// @react-spectrum/s2/illustrations/linear/internal/S2_lin_chatbubble_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $9oh16$reactjsxruntime = require("react/jsx-runtime");
var $9oh16$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $40c40ca666a29178$export$2e2bcd8739ae039);




const $40c40ca666a29178$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9oh16$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $9oh16$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M4.25 12.74c0-3.094 2.551-5.49 5.613-5.49h28.274c3.062 0 5.613 2.396 5.613 5.49v14.463c0 3.094-2.551 5.49-5.613 5.49h-11.66L17.119 40.6l-.002.001c-1.724 1.466-4.483.295-4.465-2.05.013-1.775.01-4.002.004-5.858H9.863c-3.062 0-5.613-2.396-5.613-5.49zm5.613-2.99c-1.758 0-3.113 1.353-3.113 2.99v14.463c0 1.638 1.355 2.99 3.113 2.99h5.284l.005 1.245c.008 2.021.016 4.92 0 7.131 0 .048.011.072.023.09.015.021.043.048.087.068s.089.026.125.022c.03-.004.067-.015.111-.053l.004-.003 9.71-8.204.35-.296h12.575c1.758 0 3.113-1.352 3.113-2.99V12.74c0-1.637-1.355-2.99-3.113-2.99z"
        })
    });
const $40c40ca666a29178$var$ForwardRef = /*#__PURE__*/ (0, $9oh16$react.forwardRef)($40c40ca666a29178$var$SvgComponent);
var $40c40ca666a29178$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($40c40ca666a29178$var$ForwardRef);
