// @react-spectrum/s2/illustrations/linear/internal/S2_lin_bug_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $lliKQ$reactjsxruntime = require("react/jsx-runtime");
var $lliKQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6529d1118ad8d127$export$2e2bcd8739ae039);




const $6529d1118ad8d127$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lliKQ$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $lliKQ$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M35.157 15.262q.004-.084.004-.168c0-1.657-1.343-3-3-3s-3 1.343-3 3 1.343 3 3 3q.084 0 .166-.005l5.552 5.552c-4.359 2.168-8.084 5.692-10.853 10.092-2.907-1.758-5.256-3.654-7.031-5.267-.818-.742-2.083-.681-2.826.136s-.681 2.083.136 2.825c1.94 1.762 4.546 3.867 7.802 5.82-2.146 4.567-3.358 9.814-3.358 15.33q0 .497.013.991h-7.116c-1.104 0-2 .896-2 2s.896 2 2 2h7.438c.679 5.028 2.367 9.702 4.831 13.675-1.816.99-3.578 2.354-5.088 3.747-2.116 1.951-3.965 4.17-4.914 5.92-.527.972-.167 2.186.804 2.712s2.185.167 2.712-.804c.666-1.228 2.178-3.107 4.11-4.888 1.538-1.419 3.198-2.642 4.722-3.4 4.712 5.751 11.295 9.376 18.74 9.376 7.448 0 14.033-3.628 18.745-9.384 1.528.758 3.195 1.985 4.738 3.408 1.931 1.781 3.444 3.66 4.11 4.888.526.97 1.74 1.331 2.711.804s1.331-1.74.804-2.711c-.949-1.75-2.797-3.97-4.913-5.921-1.515-1.397-3.283-2.765-5.104-3.755 2.46-3.971 4.147-8.642 4.826-13.667h7.437c1.104 0 2-.895 2-2s-.896-2-2-2h-7.116q.014-.495.013-.991c0-5.515-1.21-10.761-3.356-15.329 3.258-1.822 5.967-3.878 7.907-5.926.76-.802.726-2.067-.076-2.827s-2.068-.726-2.828.076c-1.65 1.742-4.024 3.543-6.92 5.166-2.77-4.402-6.496-7.928-10.857-10.097l5.551-5.55q.082.003.166.004c1.657 0 3-1.343 3-3s-1.343-3-3-3-3 1.343-3 3q0 .084.005.168l-6.572 6.572q-.115.114-.207.243c-1.938-.542-3.967-.83-6.064-.83s-4.127.288-6.065.83q-.092-.128-.208-.244zm-6.545 23.884c-1.815 3.932-2.863 8.498-2.863 13.43 0 14.597 9.175 25.984 20.252 27.219V43.433c-6.834-.2-12.643-1.98-17.39-4.287m1.93-3.51c4.552 2.184 10.141 3.815 16.733 3.815 6.622 0 12.928-1.622 18.116-3.922C61.256 29.205 54.903 25.247 48 25.247c-6.942 0-13.327 4.003-17.46 10.389m36.798 3.403c-5.053 2.285-11.034 3.949-17.34 4.33v36.426c11.078-1.235 20.252-12.622 20.252-27.218 0-4.978-1.066-9.582-2.912-13.538"
        })
    });
const $6529d1118ad8d127$var$ForwardRef = /*#__PURE__*/ (0, $lliKQ$react.forwardRef)($6529d1118ad8d127$var$SvgComponent);
var $6529d1118ad8d127$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6529d1118ad8d127$var$ForwardRef);
