// @react-spectrum/s2/illustrations/linear/internal/S2_lin_socialPost_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $3Z8pI$jsx} from "react/jsx-runtime";
import {forwardRef as $3Z8pI$forwardRef} from "react";





const $0426469cd4ca326b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3Z8pI$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        fill: "none",
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $3Z8pI$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M14.023 8.773c-2.9 0-5.25 2.35-5.25 5.25v19.954c0 2.9 2.35 5.25 5.25 5.25h19.953c2.9 0 5.25-2.35 5.25-5.25V14.023c0-2.9-2.35-5.25-5.25-5.25zm22.703 25.093-3.244-4.806c-1.188-1.771-3.79-1.766-4.978 0l-.005.008-2.65 4.005-4.865-7.516-1.05.68 1.05-.68c-1.181-1.825-3.855-1.825-5.035 0l1.05.68-1.05-.68-4.676 7.224V22.105h25.453zm-13.293 2.86.912-1.377-5.46-8.434c-.196-.302-.641-.302-.837 0l-5.883 9.089c.49.449 1.142.723 1.858.723zm2.998 0 4.15-6.274c.2-.294.631-.29.825 0l.003.005 3.972 5.884c-.411.245-.891.386-1.405.386zM11.273 19.606h25.453v-5.582c0-1.518-1.23-2.75-2.75-2.75H14.023c-1.518 0-2.75 1.232-2.75 2.75zm2.111-4.166c0-.69.56-1.25 1.25-1.25l18.767.003c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25l-18.767-.003c-.69 0-1.25-.56-1.25-1.25m18.83 9.373c0 .966.784 1.75 1.75 1.75.967 0 1.75-.784 1.75-1.75 0-.967-.783-1.75-1.75-1.75-.966 0-1.75.783-1.75 1.75",
            clipRule: "evenodd"
        })
    });
const $0426469cd4ca326b$var$ForwardRef = /*#__PURE__*/ (0, $3Z8pI$forwardRef)($0426469cd4ca326b$var$SvgComponent);
var $0426469cd4ca326b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0426469cd4ca326b$var$ForwardRef);


export {$0426469cd4ca326b$export$2e2bcd8739ae039 as default};
