// @react-spectrum/s2/illustrations/linear/internal/S2_lin_information_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../icons/Icon.mjs";
import {jsx as $dWbIe$jsx} from "react/jsx-runtime";
import {forwardRef as $dWbIe$forwardRef} from "react";





const $50524c85d45fdb52$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dWbIe$jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $dWbIe$jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M16.5 47.74c0-17.12 13.88-31 31-31s31 13.88 31 31c0 17.121-13.88 31-31 31s-31-13.879-31-31m31-35c-19.33 0-35 15.67-35 35s15.67 35 35 35 35-15.67 35-35-15.67-35-35-35m2.5 21.5c0 1.657-1.343 3-3 3s-3-1.343-3-3 1.343-3 3-3 3 1.343 3 3m-7 6c-1.105 0-2 .896-2 2s.895 2 2 2h3v14h-3c-1.105 0-2 .896-2 2s.895 2 2 2h10c1.105 0 2-.895 2-2s-.895-2-2-2h-3v-16c0-1.104-.895-2-2-2z"
        })
    });
const $50524c85d45fdb52$var$ForwardRef = /*#__PURE__*/ (0, $dWbIe$forwardRef)($50524c85d45fdb52$var$SvgComponent);
var $50524c85d45fdb52$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($50524c85d45fdb52$var$ForwardRef);


export {$50524c85d45fdb52$export$2e2bcd8739ae039 as default};
