// @react-spectrum/s2/illustrations/linear/internal/S2_lin_infiniteLooping_96.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $4H0hP$reactjsxruntime = require("react/jsx-runtime");
var $4H0hP$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7dd12fbd249094c2$export$2e2bcd8739ae039);




const $7dd12fbd249094c2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4H0hP$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $4H0hP$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M43.097 43.606c1.105-.003 1.998-.9 1.995-2.005l-.034-15.012c-.003-1.104-.9-1.998-2.005-1.995s-1.998.9-1.995 2.004l.023 10.205c-3.959-4.595-9.825-7.508-16.37-7.508-11.932 0-21.604 9.672-21.604 21.603 0 11.93 9.672 21.602 21.603 21.602 4.95 0 9.515-1.667 13.159-4.469q.078-.06.149-.126l.082-.061c2.688-2.11 5.206-5.189 7.186-8.026 1.987-2.849 3.539-5.61 4.26-7.2.573-1.266 1.992-4.273 3.872-7.434 1.618-2.72 3.48-5.388 5.345-7.193q.078-.058.15-.125c3.128-2.842 7.277-4.571 11.835-4.571 9.722 0 17.602 7.88 17.602 17.603 0 9.721-7.88 17.602-17.602 17.602-5.945 0-11.203-2.945-14.393-7.465-.637-.903-1.885-1.118-2.787-.481-.903.637-1.118 1.885-.481 2.787 3.907 5.537 10.361 9.16 17.66 9.16 11.932 0 21.603-9.673 21.603-21.603S82.68 29.295 70.749 29.295c-4.954 0-9.522 1.67-13.167 4.475q-.144.08-.278.184c-2.807 2.214-5.372 5.905-7.323 9.185-1.984 3.337-3.472 6.49-4.078 7.828-.597 1.317-2.014 3.864-3.897 6.562-1.89 2.708-4.142 5.415-6.375 7.169q-.07.054-.133.114-.035.024-.066.049c-2.97 2.283-6.684 3.64-10.72 3.64-9.723 0-17.604-7.882-17.604-17.603s7.881-17.603 17.603-17.603c5.428 0 10.282 2.455 13.514 6.322l-10.148.023c-1.105.003-1.998.9-1.995 2.005s.9 1.998 2.004 1.995z"
        })
    });
const $7dd12fbd249094c2$var$ForwardRef = /*#__PURE__*/ (0, $4H0hP$react.forwardRef)($7dd12fbd249094c2$var$SvgComponent);
var $7dd12fbd249094c2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7dd12fbd249094c2$var$ForwardRef);
