// @react-spectrum/s2/illustrations/linear/internal/S2_lin_brand_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $5mFLt$reactjsxruntime = require("react/jsx-runtime");
var $5mFLt$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f799afe6ce3e0c47$export$2e2bcd8739ae039);




const $f799afe6ce3e0c47$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5mFLt$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $5mFLt$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M13.76 10.5c-1.246 0-2.26 1.014-2.26 2.26v14.514c0 6.776 5.503 12.276 12.276 12.276 6.776 0 12.276-5.502 12.276-12.276V12.76c0-1.245-1.014-2.259-2.26-2.259zM9 12.76C9 10.133 11.133 8 13.76 8h20.033c2.626 0 4.76 2.133 4.76 4.76v14.514c0 8.154-6.62 14.776-14.777 14.776C15.623 42.05 9 35.431 9 27.274zm19.73 5.958 3.677 6.366v.001c.93 1.612-.23 3.63-2.095 3.63H22.96c-1.866 0-3.025-2.018-2.095-3.63l3.673-6.367c.932-1.614 3.26-1.615 4.192 0m-2.096 1.37L23.1 26.214h7.072zm-3.603 6.246-.071-.04-1.01-.584 1.01.583zm-2.322-5.316c.345-.598.14-1.363-.457-1.708-.598-.345-1.363-.14-1.708.457l-4.085 7.075c-.346.598-.14 1.362.457 1.708.598.345 1.362.14 1.707-.458z"
        })
    });
const $f799afe6ce3e0c47$var$ForwardRef = /*#__PURE__*/ (0, $5mFLt$react.forwardRef)($f799afe6ce3e0c47$var$SvgComponent);
var $f799afe6ce3e0c47$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f799afe6ce3e0c47$var$ForwardRef);
