// @react-spectrum/s2/illustrations/linear/internal/S2_lin_libraries_48.cjs
var $bde97c91243ed164$exports = require("../../../icons/Icon.cjs");
var $34G6d$reactjsxruntime = require("react/jsx-runtime");
var $34G6d$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d745930d63f7ffd5$export$2e2bcd8739ae039);




const $d745930d63f7ffd5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $34G6d$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: /*#__PURE__*/ (0, $34G6d$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, light-dark(rgb(41, 41, 41), rgb(219, 219, 219)))",
            fillRule: "evenodd",
            d: "M10.41 15.18c0-2.62 2.13-4.75 4.75-4.75h13.67c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H15.16c-4 0-7.25 3.25-7.25 7.25v13.67c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25zm6.75-.78c-1.519 0-2.75 1.23-2.75 2.75v17.67c0 1.518 1.231 2.75 2.75 2.75h17.67c1.519 0 2.75-1.232 2.75-2.75V17.15c0-1.52-1.231-2.75-2.75-2.75h-.38v10.71c0 .4-.45.64-.78.42l-2.4-1.6c-.16-.11-.38-.11-.55 0l-2.4 1.6c-.33.22-.78-.02-.78-.42V14.4zm-5.25 2.75c0-2.9 2.35-5.25 5.25-5.25h17.67c2.9 0 5.25 2.35 5.25 5.25v17.67c0 2.9-2.35 5.25-5.25 5.25H17.16c-2.9 0-5.25-2.35-5.25-5.25z"
        })
    });
const $d745930d63f7ffd5$var$ForwardRef = /*#__PURE__*/ (0, $34G6d$react.forwardRef)($d745930d63f7ffd5$var$SvgComponent);
var $d745930d63f7ffd5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d745930d63f7ffd5$var$ForwardRef);
