// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_translate_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $cyM6d$jsxs, jsx as $cyM6d$jsx} from "react/jsx-runtime";
import {forwardRef as $cyM6d$forwardRef} from "react";





const $c90c92bb1bfbfc49$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cyM6d$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 97,
        height: 96,
        viewBox: "0 0 97 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cyM6d$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cyM6d$jsxs)("linearGradient", {
                        id: "fb1a60__d",
                        x1: -23.183,
                        x2: 132.895,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cyM6d$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $cyM6d$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cyM6d$jsxs)("linearGradient", {
                        id: "fb1a60__e",
                        x1: 102.07,
                        x2: 35.113,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cyM6d$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cyM6d$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cyM6d$jsxs)("linearGradient", {
                        id: "fb1a60__f",
                        x1: 20.697,
                        x2: 139.393,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cyM6d$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cyM6d$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cyM6d$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cyM6d$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cyM6d$jsxs)("radialGradient", {
                        id: "fb1a60__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.75 29.473)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cyM6d$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $cyM6d$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cyM6d$jsxs)("radialGradient", {
                        id: "fb1a60__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 37.002 54.345)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cyM6d$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cyM6d$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cyM6d$jsx)("clipPath", {
                        id: "fb1a60__b",
                        children: /*#__PURE__*/ (0, $cyM6d$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cyM6d$jsx)("clipPath", {
                id: "fb1a60__a",
                children: /*#__PURE__*/ (0, $cyM6d$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M57.225 21.786H40.524v-3.535c0-1.105-.896-2-2-2s-2 .895-2 2v3.535H19.25c-1.105 0-2 .896-2 2s.895 2 2 2h29.26c-.991 6.861-4.761 12.418-9.5 16.584-1.162 1.02-2.375 1.952-3.609 2.792-2.217-3.815-3.812-7.512-4.286-10.204-.192-1.088-1.23-1.814-2.317-1.623s-1.814 1.23-1.622 2.317c.587 3.336 2.434 7.517 4.806 11.588-3.129 1.688-6.22 2.799-8.8 3.318-1.082.218-1.783 1.273-1.565 2.355s1.272 1.784 2.355 1.566c3.037-.61 6.585-1.9 10.13-3.838l.036.054c3.366 5.057 7.527 9.881 11.2 12.254.927.599 2.165.333 2.764-.595.6-.928.333-2.166-.594-2.766-2.962-1.913-6.731-6.153-9.988-11.03 1.416-.956 2.804-2.018 4.13-3.184 5.388-4.735 9.87-11.297 10.897-19.588h4.678c1.104 0 2-.895 2-2s-.896-2-2-2m-.854 45.349-4.83 11.396c-.43 1.017-1.604 1.492-2.621 1.06-1.017-.43-1.492-1.604-1.062-2.621l14.463-34.121q.045-.11.102-.213.113-.2.262-.365c.19-.21.426-.382.703-.5.256-.108.52-.159.781-.159.26 0 .526.051.78.16.278.117.515.29.704.5q.15.163.261.364.058.104.103.213l14.462 34.12c.431 1.018-.043 2.192-1.06 2.623-1.017.43-2.191-.044-2.622-1.061L71.956 67.11q-.156.025-.317.025zm7.798-18.397-6.103 14.397h12.205z"
                })
            }),
            /*#__PURE__*/ (0, $cyM6d$jsx)("g", {
                clipPath: "url(#fb1a60__a)",
                children: /*#__PURE__*/ (0, $cyM6d$jsxs)("g", {
                    clipPath: "url(#fb1a60__b)",
                    children: [
                        /*#__PURE__*/ (0, $cyM6d$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $cyM6d$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M.941 0h96v96h-96z"
                        }),
                        /*#__PURE__*/ (0, $cyM6d$jsx)("circle", {
                            cx: 55.222,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#fb1a60__c)",
                            transform: "rotate(-.08 55.222 3.723)"
                        }),
                        /*#__PURE__*/ (0, $cyM6d$jsx)("path", {
                            fill: "url(#fb1a60__d)",
                            d: "M37.584-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.014 45.19c-7.25 4.427-7.261 14.873-.022 19.316l73.525 45.127c7.39 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.201-4.433 7.214-14.821.022-19.27z"
                        }),
                        /*#__PURE__*/ (0, $cyM6d$jsx)("path", {
                            fill: "url(#fb1a60__e)",
                            d: "M14.778 45.579c1.64-35.84 32.025-63.563 67.864-61.923s63.564 32.025 61.923 67.865-32.025 63.563-67.865 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $cyM6d$jsx)("path", {
                            fill: "url(#fb1a60__f)",
                            d: "M-37.954 69.21c14.247-6.078 29.611-5.818 42.922-.502 26.637 10.664 42.136 4.071 52.885-22.504 5.355-13.336 15.81-24.63 30.071-30.714 28.516-12.146 61.436 1.033 73.565 29.464s-1.147 61.299-29.656 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.062-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.492 12.155-61.412-1.024-73.534-29.44-12.123-28.415 1.14-61.314 29.632-73.47"
                        }),
                        /*#__PURE__*/ (0, $cyM6d$jsx)("circle", {
                            cx: 91.347,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#fb1a60__g)",
                            transform: "rotate(-.08 91.347 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $c90c92bb1bfbfc49$var$ForwardRef = /*#__PURE__*/ (0, $cyM6d$forwardRef)($c90c92bb1bfbfc49$var$SvgComponent);
var $c90c92bb1bfbfc49$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c90c92bb1bfbfc49$var$ForwardRef);


export {$c90c92bb1bfbfc49$export$2e2bcd8739ae039 as default};
