// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_prompt_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2ncJ3$jsxs, jsx as $2ncJ3$jsx} from "react/jsx-runtime";
import {forwardRef as $2ncJ3$forwardRef} from "react";





const $61ada2ed24ef0ec4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2ncJ3$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2ncJ3$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2ncJ3$jsxs)("linearGradient", {
                        id: "32aba2__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ncJ3$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2ncJ3$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ncJ3$jsxs)("linearGradient", {
                        id: "32aba2__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ncJ3$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2ncJ3$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ncJ3$jsxs)("linearGradient", {
                        id: "32aba2__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ncJ3$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2ncJ3$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2ncJ3$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2ncJ3$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ncJ3$jsxs)("radialGradient", {
                        id: "32aba2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ncJ3$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2ncJ3$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ncJ3$jsxs)("radialGradient", {
                        id: "32aba2__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ncJ3$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2ncJ3$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ncJ3$jsx)("clipPath", {
                        id: "32aba2__b",
                        children: /*#__PURE__*/ (0, $2ncJ3$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2ncJ3$jsx)("clipPath", {
                id: "32aba2__a",
                children: /*#__PURE__*/ (0, $2ncJ3$jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M44 42c-8.837 0-16 7.163-16 16v44c0 8.837 7.163 16 16 16h72c8.837 0 16-7.163 16-16V58c0-8.837-7.163-16-16-16zm-3.022 11.395c1.714-1.393 4.233-1.132 5.627.582l19.5 24c1.194 1.47 1.194 3.576 0 5.045l-19.5 24c-1.393 1.715-3.913 1.975-5.627.582-1.715-1.393-1.975-3.912-.582-5.626L57.846 80.5l-17.45-21.478c-1.393-1.714-1.133-4.233.582-5.627M66 101c0-2.21 1.791-4 4-4h41.5c2.209 0 4 1.79 4 4s-1.791 4-4 4H70c-2.209 0-4-1.791-4-4"
                })
            }),
            /*#__PURE__*/ (0, $2ncJ3$jsx)("g", {
                clipPath: "url(#32aba2__a)",
                children: /*#__PURE__*/ (0, $2ncJ3$jsxs)("g", {
                    clipPath: "url(#32aba2__b)",
                    children: [
                        /*#__PURE__*/ (0, $2ncJ3$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $2ncJ3$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $2ncJ3$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#32aba2__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $2ncJ3$jsx)("path", {
                            fill: "url(#32aba2__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $2ncJ3$jsx)("path", {
                            fill: "url(#32aba2__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $2ncJ3$jsx)("path", {
                            fill: "url(#32aba2__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $2ncJ3$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#32aba2__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $61ada2ed24ef0ec4$var$ForwardRef = /*#__PURE__*/ (0, $2ncJ3$forwardRef)($61ada2ed24ef0ec4$var$SvgComponent);
var $61ada2ed24ef0ec4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($61ada2ed24ef0ec4$var$ForwardRef);


export {$61ada2ed24ef0ec4$export$2e2bcd8739ae039 as default};
