// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_badgeVerified_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3WkMW$reactjsxruntime = require("react/jsx-runtime");
var $3WkMW$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b1def1a241190066$export$2e2bcd8739ae039);




const $b1def1a241190066$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3de3d7__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3de3d7__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3de3d7__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3de3d7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3de3d7__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("clipPath", {
                        id: "3de3d7__b",
                        children: /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("clipPath", {
                id: "3de3d7__a",
                children: /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M126.891 64.173c5.511 3.126 9.247 9.044 9.247 15.834h.015c0 6.82-3.766 12.752-9.32 15.878-2.181 1.221-3.199 3.707-2.53 6.121.742 2.632.902 5.511.305 8.491-1.41 7.11-7.095 12.81-14.205 14.22-2.995.596-5.903.436-8.549-.32-2.37-.669-4.842.335-6.078 2.472C92.621 132.321 86.746 136 80 136s-12.62-3.679-15.775-9.131c-1.236-2.137-3.693-3.141-6.078-2.472-2.646.756-5.554.916-8.55.32-7.11-1.41-12.794-7.11-14.205-14.22-.596-2.98-.436-5.859.306-8.491.668-2.414-.35-4.9-2.53-6.121-5.555-3.126-9.32-9.088-9.32-15.878S27.57 67.3 33.095 64.173c2.108-1.192 3.082-3.678 2.5-5.96-.58-2.284-.74-4.726-.377-7.256 1.148-7.866 7.502-14.191 15.368-15.325 2.574-.378 4.944-.233 7.358.422 2.413.654 4.885-.393 6.092-2.574C67.133 27.839 73.123 24 80 24s12.868 3.839 15.965 9.48c1.207 2.18 3.795 3.184 6.092 2.574s4.769-.8 7.343-.422c7.88 1.134 14.22 7.459 15.368 15.325.364 2.53.233 4.914-.378 7.255-.61 2.341.393 4.77 2.501 5.961m-19.88 3.905c1.631-1.49 1.745-4.02.255-5.651s-4.02-1.745-5.651-.255L71.712 89.497 58.349 77.071c-1.617-1.505-4.148-1.413-5.653.205s-1.412 4.149.206 5.653l16.062 14.938c1.526 1.418 3.884 1.429 5.422.023z"
                })
            }),
            /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("g", {
                clipPath: "url(#3de3d7__a)",
                children: /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#3de3d7__b)",
                    children: [
                        /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#3de3d7__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("path", {
                            fill: "url(#3de3d7__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("path", {
                            fill: "url(#3de3d7__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("path", {
                            fill: "url(#3de3d7__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $3WkMW$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#3de3d7__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $b1def1a241190066$var$ForwardRef = /*#__PURE__*/ (0, $3WkMW$react.forwardRef)($b1def1a241190066$var$SvgComponent);
var $b1def1a241190066$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b1def1a241190066$var$ForwardRef);
