// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_bell_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $lCYg9$reactjsxruntime = require("react/jsx-runtime");
var $lCYg9$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2fc945d07ec79a90$export$2e2bcd8739ae039);




const $2fc945d07ec79a90$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "822ad5__d",
                        x1: -24.021,
                        x2: 201.071,
                        y1: 134.5,
                        y2: -2.444,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "822ad5__e",
                        x1: 156.616,
                        x2: 60.052,
                        y1: 36.277,
                        y2: 140.11,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "822ad5__f",
                        x1: 39.265,
                        x2: 210.447,
                        y1: 154.441,
                        y2: 58.547,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsxs)("radialGradient", {
                        id: "822ad5__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.455 52.6)scale(60.2331)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsxs)("radialGradient", {
                        id: "822ad5__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 52.682 88.47)scale(60.2331)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("clipPath", {
                        id: "822ad5__b",
                        children: /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 10.773,
                            y: 10.775,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("clipPath", {
                id: "822ad5__a",
                children: /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M122.955 102.887c-5.292-8.32-10.748-19.933-11.153-32.867V59.93h-.004q.002-.064.004-.13C111.802 42.238 97.564 28 80 28c-17.563 0-31.8 14.238-31.8 31.801l.002.13H48.2V70.02c-.406 12.934-5.861 24.547-11.154 32.867-3.615 5.683.43 13.113 7.166 13.113h23.79v4c0 6.627 5.372 12 12 12 6.627 0 12-5.373 12-12v-4h23.789c6.735 0 10.781-7.43 7.165-13.113"
                })
            }),
            /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("g", {
                clipPath: "url(#822ad5__a)",
                children: /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#822ad5__b)",
                    children: [
                        /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 10.773,
                            y: 10.775,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M10.773 10.775h138.45v138.45H10.772z"
                        }),
                        /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("circle", {
                            cx: 89.055,
                            cy: 16.146,
                            r: 60.233,
                            fill: "url(#822ad5__c)",
                            transform: "rotate(-.08 89.055 16.146)"
                        }),
                        /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("path", {
                            fill: "url(#822ad5__d)",
                            d: "M63.616-13.68c-10.633-6.58-24.104-6.62-34.777-.105L-77.903 51.39c-10.454 6.383-10.471 21.45-.031 27.856l106.037 65.081c10.656 6.54 24.127 6.532 34.775-.023l105.818-65.142c10.385-6.393 10.404-21.376.032-27.793z"
                        }),
                        /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("path", {
                            fill: "url(#822ad5__e)",
                            d: "M30.726 76.508c2.366-51.688 46.185-91.67 97.872-89.304 51.688 2.367 91.67 46.186 89.304 97.873-2.367 51.687-46.186 91.67-97.873 89.303-51.687-2.366-91.67-46.185-89.303-97.872"
                        }),
                        /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("path", {
                            fill: "url(#822ad5__f)",
                            d: "M-45.32 110.588c20.546-8.765 42.705-8.389 61.902-.723 38.415 15.38 60.767 5.871 76.27-32.455 7.722-19.233 22.799-35.52 43.368-44.295 41.124-17.517 88.601 1.49 106.093 42.493s-1.654 88.404-42.769 105.944c-20.615 8.794-42.816 8.382-62.042.647-38.353-15.297-60.662-5.753-76.146 32.62-7.742 19.186-22.815 35.418-43.36 44.183-41.092 17.53-88.568-1.477-106.05-42.457-17.483-40.98 1.643-88.427 42.735-105.957"
                        }),
                        /*#__PURE__*/ (0, $lCYg9$reactjsxruntime.jsx)("circle", {
                            cx: 141.153,
                            cy: 35.788,
                            r: 60.233,
                            fill: "url(#822ad5__g)",
                            transform: "rotate(-.08 141.153 35.788)"
                        })
                    ]
                })
            })
        ]
    });
const $2fc945d07ec79a90$var$ForwardRef = /*#__PURE__*/ (0, $lCYg9$react.forwardRef)($2fc945d07ec79a90$var$SvgComponent);
var $2fc945d07ec79a90$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2fc945d07ec79a90$var$ForwardRef);
