// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_mailOpen_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $itBmP$reactjsxruntime = require("react/jsx-runtime");
var $itBmP$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3486558b48eca946$export$2e2bcd8739ae039);




const $3486558b48eca946$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "fbee56__d",
                        x1: -7.489,
                        x2: 60.376,
                        y1: 40.303,
                        y2: -0.986,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "fbee56__e",
                        x1: 46.97,
                        x2: 17.856,
                        y1: 10.689,
                        y2: 41.995,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "fbee56__f",
                        x1: 11.592,
                        x2: 63.203,
                        y1: 46.315,
                        y2: 17.403,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsxs)("radialGradient", {
                        id: "fbee56__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 10.992 15.611)scale(18.1603)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsxs)("radialGradient", {
                        id: "fbee56__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 15.885 26.426)scale(18.1603)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("clipPath", {
                        id: "fbee56__b",
                        children: /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3,
                            y: 3,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("clipPath", {
                id: "fbee56__a",
                children: /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M17.98 14.424c-1.987 0-3.6 1.614-3.6 3.6v1.473L9.536 16.46l13.236-8.283c.783-.49 1.773-.49 2.553.003l13.14 8.277-4.845 3.039v-1.473c0-1.986-1.614-3.6-3.6-3.6zm0 2.4c-.664 0-1.2.537-1.2 1.2V21l6.584 4.13c.387.244.888.244 1.275 0L31.224 21v-2.98c0-.662-.537-1.2-1.2-1.2H17.982zm7.934 10.34c-.582.364-1.248.547-1.914.547s-1.332-.183-1.914-.546l-13.68-8.58V35.39c0 2.652 2.148 4.803 4.8 4.803h21.588c2.652 0 4.8-2.151 4.8-4.803V18.585z"
                })
            }),
            /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("g", {
                clipPath: "url(#fbee56__a)",
                children: /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#fbee56__b)",
                    children: [
                        /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3,
                            y: 3,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3 3h41.742v41.742H3z"
                        }),
                        /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("circle", {
                            cx: 26.604,
                            cy: 4.619,
                            r: 18.16,
                            fill: "url(#fbee56__c)",
                            transform: "rotate(-.08 26.604 4.62)"
                        }),
                        /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("path", {
                            fill: "url(#fbee56__d)",
                            d: "M18.934-4.374c-3.206-1.984-7.268-1.996-10.486-.031l-32.183 19.65c-3.151 1.924-3.157 6.467-.009 8.398l31.97 19.622c3.213 1.972 7.275 1.97 10.485-.007l31.904-19.64c3.131-1.928 3.137-6.445.01-8.38z"
                        }),
                        /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("path", {
                            fill: "url(#fbee56__e)",
                            d: "M9.015 22.819C9.728 7.235 22.938-4.82 38.523-4.106S66.162 9.819 65.448 25.402c-.713 15.584-13.925 27.639-29.508 26.925C20.356 51.614 8.3 38.402 9.015 22.82"
                        }),
                        /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("path", {
                            fill: "url(#fbee56__f)",
                            d: "M-13.91 33.094c6.194-2.643 12.875-2.53 18.663-.218 11.582 4.636 18.321 1.77 22.996-9.786 2.328-5.798 6.873-10.71 13.075-13.355 12.399-5.281 26.713.45 31.987 12.812S72.312 49.2 59.916 54.489c-6.215 2.652-12.909 2.527-18.706.195-11.563-4.612-18.29-1.734-22.957 9.835-2.335 5.785-6.88 10.679-13.074 13.321-12.389 5.286-26.703-.445-31.974-12.8-5.27-12.356.496-26.661 12.885-31.947"
                        }),
                        /*#__PURE__*/ (0, $itBmP$reactjsxruntime.jsx)("circle", {
                            cx: 42.311,
                            cy: 10.541,
                            r: 18.16,
                            fill: "url(#fbee56__g)",
                            transform: "rotate(-.08 42.31 10.541)"
                        })
                    ]
                })
            })
        ]
    });
const $3486558b48eca946$var$ForwardRef = /*#__PURE__*/ (0, $itBmP$react.forwardRef)($3486558b48eca946$var$SvgComponent);
var $3486558b48eca946$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3486558b48eca946$var$ForwardRef);
