// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_lockOpen_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $b9uON$reactjsxruntime = require("react/jsx-runtime");
var $b9uON$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1fd72f627f2907a8$export$2e2bcd8739ae039);




const $1fd72f627f2907a8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3a35dd__d",
                        x1: -24.537,
                        x2: 201.681,
                        y1: 134.772,
                        y2: -2.857,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3a35dd__e",
                        x1: 157.003,
                        x2: 59.956,
                        y1: 36.058,
                        y2: 140.411,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3a35dd__f",
                        x1: 39.065,
                        x2: 211.102,
                        y1: 154.813,
                        y2: 58.44,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3a35dd__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.639 52.464)scale(60.5342)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3a35dd__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 52.948 88.515)scale(60.5342)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("clipPath", {
                        id: "3a35dd__b",
                        children: /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.429,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("clipPath", {
                id: "3a35dd__a",
                children: /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M54 45.5c0-14.36 11.64-26 26-26s26 11.64 26 26c0 2.21-1.791 4-4 4s-4-1.79-4-4c0-9.941-8.059-18-18-18s-18 8.059-18 18v21h53.636c7.381 0 13.364 5.457 13.364 12.188v40.624c0 6.731-5.983 12.188-13.364 12.188H44.364c-7.38 0-13.364-5.457-13.364-12.188V78.688C31 71.957 36.983 66.5 44.364 66.5H54zm20.346 55.153c.493.497 1.051.924 1.654 1.271v7.576c0 2.209 1.79 4 4 4s4-1.791 4-4v-7.576c.603-.347 1.16-.774 1.654-1.271.372-.36.705-.773 1-1.214.293-.426.532-.893.733-1.373.199-.493.36-.986.453-1.506.107-.507.16-1.04.16-1.56s-.053-1.053-.16-1.56c-.094-.507-.254-1.014-.453-1.493-.2-.494-.44-.947-.733-1.387-.295-.44-.628-.854-1-1.214C84.174 87.853 82.107 87 80 87s-4.173.853-5.654 2.346c-.373.36-.707.774-1 1.214-.28.44-.534.893-.733 1.387-.2.48-.36.986-.453 1.492-.107.508-.16 1.04-.16 1.561s.053 1.053.16 1.56c.093.52.253 1.013.453 1.506.2.48.453.947.733 1.373.293.44.627.854 1 1.214"
                })
            }),
            /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("g", {
                clipPath: "url(#3a35dd__a)",
                children: /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#3a35dd__b)",
                    children: [
                        /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.429,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M10.43 10.429h139.142V149.57H10.43z"
                        }),
                        /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("circle", {
                            cx: 89.103,
                            cy: 15.826,
                            r: 60.534,
                            fill: "url(#3a35dd__c)",
                            transform: "rotate(-.08 89.103 15.826)"
                        }),
                        /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("path", {
                            fill: "url(#3a35dd__d)",
                            d: "M63.539-14.15c-10.686-6.612-24.225-6.653-34.95-.104L-78.689 51.244C-89.194 57.66-89.212 72.8-78.72 79.24l106.568 65.406c10.71 6.572 24.248 6.564 34.949-.023l106.347-65.468c10.437-6.425 10.456-21.483.032-27.932z"
                        }),
                        /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("path", {
                            fill: "url(#3a35dd__e)",
                            d: "M30.483 76.49c2.378-51.946 46.417-92.128 98.362-89.75 51.946 2.378 92.128 46.417 89.75 98.362-2.378 51.946-46.416 92.128-98.362 89.75s-92.128-46.416-89.75-98.362"
                        }),
                        /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("path", {
                            fill: "url(#3a35dd__f)",
                            d: "M-45.943 110.741c20.649-8.809 42.918-8.432 62.211-.727 38.607 15.456 61.07 5.9 76.651-32.618 7.761-19.328 22.913-35.697 43.585-44.516 41.33-17.604 89.044 1.497 106.623 42.705s-1.662 88.847-42.982 106.474c-20.718 8.839-43.03 8.425-62.353.65-38.544-15.373-60.965-5.781-76.526 32.784-7.78 19.282-22.93 35.595-43.578 44.404-41.297 17.617-89.01-1.485-106.58-42.669-17.57-41.185 1.652-88.87 42.95-106.487"
                        }),
                        /*#__PURE__*/ (0, $b9uON$reactjsxruntime.jsx)("circle", {
                            cx: 141.463,
                            cy: 35.567,
                            r: 60.534,
                            fill: "url(#3a35dd__g)",
                            transform: "rotate(-.08 141.463 35.567)"
                        })
                    ]
                })
            })
        ]
    });
const $1fd72f627f2907a8$var$ForwardRef = /*#__PURE__*/ (0, $b9uON$react.forwardRef)($1fd72f627f2907a8$var$SvgComponent);
var $1fd72f627f2907a8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1fd72f627f2907a8$var$ForwardRef);
