// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_commentText_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $5zjEu$jsxs, jsx as $5zjEu$jsx} from "react/jsx-runtime";
import {forwardRef as $5zjEu$forwardRef} from "react";





const $a638d9e13c7fb81d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5zjEu$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5zjEu$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5zjEu$jsxs)("linearGradient", {
                        id: "c82d38__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.788,
                        y2: -9.168,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5zjEu$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $5zjEu$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5zjEu$jsxs)("linearGradient", {
                        id: "c82d38__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.682,
                        y2: 89.679,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5zjEu$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5zjEu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5zjEu$jsxs)("linearGradient", {
                        id: "c82d38__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.616,
                        y2: 33.124,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5zjEu$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5zjEu$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5zjEu$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5zjEu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5zjEu$jsxs)("radialGradient", {
                        id: "c82d38__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.28 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5zjEu$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $5zjEu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5zjEu$jsxs)("radialGradient", {
                        id: "c82d38__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.532 53.874)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5zjEu$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5zjEu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5zjEu$jsx)("clipPath", {
                        id: "c82d38__b",
                        children: /*#__PURE__*/ (0, $5zjEu$jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: -0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5zjEu$jsx)("clipPath", {
                id: "c82d38__a",
                children: /*#__PURE__*/ (0, $5zjEu$jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M20.64 17.178h54.72c4.772 0 8.64 3.757 8.64 8.5v28c0 4.742-3.868 8.5-8.64 8.5H52L32.767 78.626c-1.873 1.613-4.785.28-4.767-2.182.032-4.416.017-10.215 0-14.266h-7.36c-4.772 0-8.64-3.758-8.64-8.5v-28c0-4.743 3.868-8.5 8.64-8.5M30 31.656c-1.105 0-2 .896-2 2s.895 2 2 2h36c1.105 0 2-.895 2-2s-.895-2-2-2zm0 12c-1.105 0-2 .896-2 2s.895 2 2 2h36c1.105 0 2-.895 2-2s-.895-2-2-2z"
                })
            }),
            /*#__PURE__*/ (0, $5zjEu$jsx)("g", {
                clipPath: "url(#c82d38__a)",
                children: /*#__PURE__*/ (0, $5zjEu$jsxs)("g", {
                    clipPath: "url(#c82d38__b)",
                    children: [
                        /*#__PURE__*/ (0, $5zjEu$jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: -0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $5zjEu$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0-.001h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $5zjEu$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.722,
                            r: 41.765,
                            fill: "url(#c82d38__c)",
                            transform: "rotate(-.08 54.28 3.722)"
                        }),
                        /*#__PURE__*/ (0, $5zjEu$jsx)("path", {
                            fill: "url(#c82d38__d)",
                            d: "M36.643-16.96c-7.373-4.561-16.714-4.59-24.114-.071l-74.015 45.19c-7.249 4.427-7.26 14.874-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.432 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $5zjEu$jsx)("path", {
                            fill: "url(#c82d38__e)",
                            d: "M13.837 45.578C15.477 9.738 45.86-17.985 81.7-16.345s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $5zjEu$jsx)("path", {
                            fill: "url(#c82d38__f)",
                            d: "M-38.895 69.208c14.246-6.077 29.61-5.817 42.922-.5 26.636 10.663 42.135 4.07 52.885-22.505 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $5zjEu$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.343,
                            r: 41.765,
                            fill: "url(#c82d38__g)",
                            transform: "rotate(-.08 90.406 17.343)"
                        })
                    ]
                })
            })
        ]
    });
const $a638d9e13c7fb81d$var$ForwardRef = /*#__PURE__*/ (0, $5zjEu$forwardRef)($a638d9e13c7fb81d$var$SvgComponent);
var $a638d9e13c7fb81d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a638d9e13c7fb81d$var$ForwardRef);


export {$a638d9e13c7fb81d$export$2e2bcd8739ae039 as default};
