// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_folderOpen_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $chPmX$jsxs, jsx as $chPmX$jsx} from "react/jsx-runtime";
import {forwardRef as $chPmX$forwardRef} from "react";





const $64b53175ff594f67$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $chPmX$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $chPmX$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $chPmX$jsxs)("linearGradient", {
                        id: "0c99d7__d",
                        x1: -19.632,
                        x2: 125.934,
                        y1: 80.012,
                        y2: -8.549,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $chPmX$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $chPmX$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $chPmX$jsxs)("linearGradient", {
                        id: "0c99d7__e",
                        x1: 97.184,
                        x2: 34.737,
                        y1: 16.492,
                        y2: 83.641,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $chPmX$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $chPmX$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $chPmX$jsxs)("linearGradient", {
                        id: "0c99d7__f",
                        x1: 21.293,
                        x2: 131.996,
                        y1: 92.908,
                        y2: 30.894,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $chPmX$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $chPmX$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $chPmX$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $chPmX$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $chPmX$jsxs)("radialGradient", {
                        id: "0c99d7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.009 28.482)scale(38.9525)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $chPmX$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $chPmX$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $chPmX$jsxs)("radialGradient", {
                        id: "0c99d7__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 35.504 51.68)scale(38.9525)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $chPmX$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $chPmX$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $chPmX$jsx)("clipPath", {
                        id: "0c99d7__b",
                        children: /*#__PURE__*/ (0, $chPmX$jsx)("rect", {
                            width: 89.535,
                            height: 89.535,
                            x: 2.867,
                            fill: "#fff",
                            rx: 2.238
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $chPmX$jsx)("clipPath", {
                id: "0c99d7__a",
                children: /*#__PURE__*/ (0, $chPmX$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M77.702 40.8v-2.618c0-4.429-3.596-8.019-8.031-8.019H41.563l-9.1-11.357c-1.016-1.268-2.554-2.006-4.18-2.006H13.454c-2.956 0-5.353 2.393-5.353 5.345v49.037c0 .757.105 1.49.302 2.186.06-1.021.281-2.064.688-3.104l.033-.082 9.472-22.966c1.556-3.92 5.374-6.416 9.53-6.416zm.536 38.348c1.938-.268 3.603-1.54 4.314-3.36l9.487-23c1.356-3.47-1.27-7.188-5.077-7.188H28.127c-2.255 0-4.275 1.357-5.078 3.412l-9.487 23c-1.353 3.462 1.26 7.172 5.053 7.188h58.861"
                })
            }),
            /*#__PURE__*/ (0, $chPmX$jsx)("g", {
                clipPath: "url(#0c99d7__a)",
                children: /*#__PURE__*/ (0, $chPmX$jsxs)("g", {
                    clipPath: "url(#0c99d7__b)",
                    children: [
                        /*#__PURE__*/ (0, $chPmX$jsx)("rect", {
                            width: 89.535,
                            height: 89.535,
                            x: 2.867,
                            fill: "#FFECCF",
                            rx: 2.238
                        }),
                        /*#__PURE__*/ (0, $chPmX$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M2.867 0h89.535v89.535H2.867z"
                        }),
                        /*#__PURE__*/ (0, $chPmX$jsx)("circle", {
                            cx: 53.491,
                            cy: 3.473,
                            r: 38.953,
                            fill: "url(#0c99d7__c)",
                            transform: "rotate(-.08 53.491 3.473)"
                        }),
                        /*#__PURE__*/ (0, $chPmX$jsx)("path", {
                            fill: "url(#0c99d7__d)",
                            d: "M37.043-15.816c-6.877-4.255-15.589-4.28-22.49-.067l-69.03 42.147c-6.761 4.128-6.772 13.872-.02 18.015l68.573 42.088c6.891 4.229 15.603 4.224 22.489-.015l68.432-42.127c6.716-4.135 6.728-13.824.021-17.974z"
                        }),
                        /*#__PURE__*/ (0, $chPmX$jsx)("path", {
                            fill: "url(#0c99d7__e)",
                            d: "M15.772 42.51c1.53-33.427 29.868-59.283 63.294-57.753s59.282 29.868 57.752 63.294c-1.531 33.426-29.868 59.283-63.294 57.752S14.24 75.935 15.772 42.509"
                        }),
                        /*#__PURE__*/ (0, $chPmX$jsx)("path", {
                            fill: "url(#0c99d7__f)",
                            d: "M-33.407 64.548c13.286-5.668 27.616-5.425 40.031-.467 24.843 9.946 39.298 3.797 49.324-20.989 4.994-12.437 14.744-22.97 28.045-28.645 26.595-11.328 57.298.963 68.61 27.48 11.312 26.516-1.069 57.17-27.658 68.513-13.331 5.688-27.689 5.421-40.123.419-24.802-9.893-39.23-3.72-49.243 21.095-5.006 12.408-14.754 22.905-28.041 28.573-26.574 11.337-57.276-.955-68.582-27.456s1.063-57.186 27.637-68.523"
                        }),
                        /*#__PURE__*/ (0, $chPmX$jsx)("circle", {
                            cx: 87.183,
                            cy: 16.175,
                            r: 38.953,
                            fill: "url(#0c99d7__g)",
                            transform: "rotate(-.08 87.183 16.175)"
                        })
                    ]
                })
            })
        ]
    });
const $64b53175ff594f67$var$ForwardRef = /*#__PURE__*/ (0, $chPmX$forwardRef)($64b53175ff594f67$var$SvgComponent);
var $64b53175ff594f67$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($64b53175ff594f67$var$ForwardRef);


export {$64b53175ff594f67$export$2e2bcd8739ae039 as default};
