// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileImage_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $jIk8M$jsxs, jsx as $jIk8M$jsx} from "react/jsx-runtime";
import {forwardRef as $jIk8M$forwardRef} from "react";





const $67abb1740849b276$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jIk8M$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jIk8M$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jIk8M$jsxs)("linearGradient", {
                        id: "957091__d",
                        x1: -13.675,
                        x2: 121.38,
                        y1: 80.7,
                        y2: -1.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jIk8M$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $jIk8M$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jIk8M$jsxs)("linearGradient", {
                        id: "957091__e",
                        x1: 94.706,
                        x2: 36.767,
                        y1: 21.766,
                        y2: 84.066,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jIk8M$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jIk8M$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jIk8M$jsxs)("linearGradient", {
                        id: "957091__f",
                        x1: 24.296,
                        x2: 127.005,
                        y1: 92.664,
                        y2: 35.128,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jIk8M$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jIk8M$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jIk8M$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jIk8M$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jIk8M$jsxs)("radialGradient", {
                        id: "957091__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 22.242 31.929)scale(36.1398)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jIk8M$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $jIk8M$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jIk8M$jsxs)("radialGradient", {
                        id: "957091__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 31.978 53.45)scale(36.1398)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jIk8M$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jIk8M$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jIk8M$jsx)("clipPath", {
                        id: "957091__b",
                        children: /*#__PURE__*/ (0, $jIk8M$jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 7.199,
                            y: 6.465,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jIk8M$jsx)("clipPath", {
                id: "957091__a",
                children: /*#__PURE__*/ (0, $jIk8M$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M52.51 18.909 72.293 38.69c1.35 1.35 2.109 3.182 2.109 5.091V72c0 3.976-3.224 7.2-7.2 7.2h-38.4c-3.977 0-7.2-3.224-7.2-7.2V24c0-3.977 3.223-7.2 7.2-7.2h18.617c1.91 0 3.741.758 5.091 2.109m4.191 21.89c-2.485 0-4.5 2.016-4.5 4.5 0 2.486 2.015 4.5 4.5 4.5 2.486 0 4.5-2.014 4.5-4.5 0-2.484-2.014-4.5-4.5-4.5M32.404 69.6h31.194c.939 0 1.514-1.029 1.022-1.828l-6.712-10.91c-.468-.761-1.575-.761-2.044 0l-5.29 8.598-7.854-13.32c-.465-.783-1.6-.783-2.064 0l-9.284 15.648c-.474.8.102 1.813 1.032 1.813"
                })
            }),
            /*#__PURE__*/ (0, $jIk8M$jsx)("g", {
                clipPath: "url(#957091__a)",
                children: /*#__PURE__*/ (0, $jIk8M$jsxs)("g", {
                    clipPath: "url(#957091__b)",
                    children: [
                        /*#__PURE__*/ (0, $jIk8M$jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 7.199,
                            y: 6.465,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $jIk8M$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M7.2 6.465h83.07v83.07H7.2z"
                        }),
                        /*#__PURE__*/ (0, $jIk8M$jsx)("circle", {
                            cx: 54.171,
                            cy: 9.687,
                            r: 36.14,
                            fill: "url(#957091__c)",
                            transform: "rotate(-.08 54.17 9.687)"
                        }),
                        /*#__PURE__*/ (0, $jIk8M$jsx)("path", {
                            fill: "url(#957091__d)",
                            d: "M38.907-8.209c-6.38-3.947-14.462-3.972-20.866-.062l-64.045 39.104c-6.273 3.83-6.283 12.87-.02 16.714L17.6 86.595c6.394 3.924 14.477 3.92 20.865-.014l63.491-39.085c6.231-3.836 6.242-12.825.019-16.676z"
                        }),
                        /*#__PURE__*/ (0, $jIk8M$jsx)("path", {
                            fill: "url(#957091__e)",
                            d: "M19.171 45.905c1.42-31.013 27.712-55.002 58.724-53.582s55.002 27.711 53.582 58.723c-1.42 31.013-27.711 55.002-58.724 53.582-31.012-1.419-55.001-27.71-53.582-58.723"
                        }),
                        /*#__PURE__*/ (0, $jIk8M$jsx)("path", {
                            fill: "url(#957091__f)",
                            d: "M-26.455 66.352c12.328-5.259 25.623-5.033 37.141-.433 23.049 9.227 36.46 3.522 45.762-19.474 4.633-11.539 13.68-21.312 26.02-26.576 24.675-10.51 53.161.893 63.656 25.495 10.496 24.602-.992 53.043-25.661 63.567-12.369 5.276-25.69 5.029-37.225.388-23.012-9.178-36.397-3.452-45.687 19.572-4.646 11.512-13.69 21.251-26.017 26.51-24.655 10.518-53.14-.886-63.63-25.474s.986-53.057 25.641-63.575"
                        }),
                        /*#__PURE__*/ (0, $jIk8M$jsx)("circle", {
                            cx: 85.429,
                            cy: 21.472,
                            r: 36.14,
                            fill: "url(#957091__g)",
                            transform: "rotate(-.08 85.429 21.472)"
                        })
                    ]
                })
            })
        ]
    });
const $67abb1740849b276$var$ForwardRef = /*#__PURE__*/ (0, $jIk8M$forwardRef)($67abb1740849b276$var$SvgComponent);
var $67abb1740849b276$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($67abb1740849b276$var$ForwardRef);


export {$67abb1740849b276$export$2e2bcd8739ae039 as default};
