// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_volume_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gj1Kc$reactjsxruntime = require("react/jsx-runtime");
var $gj1Kc$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $dfe46ed3d941c66a$export$2e2bcd8739ae039);




const $dfe46ed3d941c66a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "302f6a__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "302f6a__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "302f6a__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsxs)("radialGradient", {
                        id: "302f6a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsxs)("radialGradient", {
                        id: "302f6a__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("clipPath", {
                        id: "302f6a__b",
                        children: /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("clipPath", {
                id: "302f6a__a",
                children: /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M46.963 52.997h-12.65c-4.419 0-8 3.582-8 8v38.205c0 4.418 3.581 8 8 8h12.65zm8-4.867v63.89L70.223 127c3.796 3.725 10.204 1.036 10.204-4.282V37.282c0-5.333-6.437-8.016-10.225-4.261zm36.782 9.102c1.328-1.766 3.836-2.12 5.601-.791 7.104 5.346 11.711 13.861 11.711 23.452 0 9.67-4.682 18.245-11.886 23.583-1.775 1.315-4.28.943-5.595-.832s-.942-4.28.833-5.596c5.254-3.894 8.648-10.127 8.648-17.155 0-6.97-3.339-13.16-8.521-17.06-1.765-1.328-2.12-3.836-.791-5.601m22.536-16.245c-1.765-1.328-4.273-.974-5.601.791s-.974 4.273.791 5.602c9.856 7.416 16.217 19.2 16.217 32.47 0 13.378-6.466 25.246-16.459 32.651-1.775 1.315-2.148 3.82-.833 5.595s3.821 2.148 5.596.832c11.942-8.849 19.696-23.058 19.696-39.078 0-15.89-7.629-30-19.407-38.862"
                })
            }),
            /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("g", {
                clipPath: "url(#302f6a__a)",
                children: /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#302f6a__b)",
                    children: [
                        /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#302f6a__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("path", {
                            fill: "url(#302f6a__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("path", {
                            fill: "url(#302f6a__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("path", {
                            fill: "url(#302f6a__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $gj1Kc$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#302f6a__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $dfe46ed3d941c66a$var$ForwardRef = /*#__PURE__*/ (0, $gj1Kc$react.forwardRef)($dfe46ed3d941c66a$var$SvgComponent);
var $dfe46ed3d941c66a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($dfe46ed3d941c66a$var$ForwardRef);
