// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_chartAreaStack_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $bA2rI$reactjsxruntime = require("react/jsx-runtime");
var $bA2rI$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $49fcdc3391013c1e$export$2e2bcd8739ae039);




const $49fcdc3391013c1e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a2451c__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a2451c__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a2451c__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a2451c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a2451c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("clipPath", {
                        id: "a2451c__b",
                        children: /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("clipPath", {
                id: "a2451c__a",
                children: /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("path", {
                    fill: "#8F8F8F",
                    fillRule: "evenodd",
                    d: "M71.905 23.235c2.151-3.895 8.094-2.38 8.094 2.083v47.806c0 3.661-2.972 6.624-6.631 6.624H22.63c-3.66 0-6.631-2.963-6.631-6.624V47.852c0-1.188.49-2.323 1.356-3.138l9.86-9.285c1.785-1.682 4.615-1.54 6.222.315l7.126 8.224c.138.159.392.14.506-.035l9.32-14.392c1.59-2.455 5.111-2.65 6.964-.388l6.44 7.867c.143.174.416.147.523-.047zm3.644 1.786c-.026.01-.08.034-.143.148L67.82 38.907l-1.751-.967 1.75.967c-1.475 2.673-5.185 3.008-7.118.647l-6.441-7.867c-.136-.166-.396-.15-.511.028l-9.32 14.392c-1.563 2.413-5.005 2.652-6.887.48l-7.126-8.223c-.117-.136-.326-.147-.457-.023l-9.86 9.285-1.336-1.419 1.336 1.42c-.063.058-.098.14-.098.225v14.527l9.075-5.837c1.77-1.14 4.064-1.047 5.737.229l7.829 5.971c.631.482 1.528.388 2.046-.214l9.924-11.525c1.896-2.203 5.265-2.33 7.322-.276l7.423 7.411 6.643-3.18v-29.64c0-.13-.035-.176-.052-.196-.03-.036-.09-.082-.184-.106-.093-.024-.17-.013-.214.005M76 59.393l-6.192 2.964c-.764.366-1.677.21-2.277-.389l-8.424-8.41c-.411-.41-1.085-.386-1.464.055l-9.924 11.525c-1.899 2.206-5.188 2.55-7.503.785l-7.829-5.972c-.334-.255-.793-.273-1.147-.045L20 67.136v5.988c0 1.446 1.175 2.624 2.631 2.624h50.737c1.456 0 2.631-1.178 2.631-2.624z"
                })
            }),
            /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("g", {
                clipPath: "url(#a2451c__a)",
                children: /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#a2451c__b)",
                    children: [
                        /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#a2451c__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("path", {
                            fill: "url(#a2451c__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("path", {
                            fill: "url(#a2451c__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("path", {
                            fill: "url(#a2451c__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $bA2rI$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#a2451c__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $49fcdc3391013c1e$var$ForwardRef = /*#__PURE__*/ (0, $bA2rI$react.forwardRef)($49fcdc3391013c1e$var$SvgComponent);
var $49fcdc3391013c1e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($49fcdc3391013c1e$var$ForwardRef);
