// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_mailClose_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $aUnJl$reactjsxruntime = require("react/jsx-runtime");
var $aUnJl$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d7fcc7991554d14c$export$2e2bcd8739ae039);




const $d7fcc7991554d14c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsxs)("linearGradient", {
                        id: "09ee95__d",
                        x1: -17.287,
                        x2: 187.234,
                        y1: 128.56,
                        y2: -3.435,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsxs)("linearGradient", {
                        id: "09ee95__e",
                        x1: 152.4,
                        x2: 67.403,
                        y1: 41.582,
                        y2: 138.536,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsxs)("linearGradient", {
                        id: "09ee95__f",
                        x1: 42.163,
                        x2: 198.293,
                        y1: 146.219,
                        y2: 53.439,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsxs)("radialGradient", {
                        id: "09ee95__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.3377 -56.5812 0 56.581 53.338)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsxs)("radialGradient", {
                        id: "09ee95__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.3377 -56.5812 0 56.581 53.338)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("clipPath", {
                        id: "09ee95__b",
                        children: /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("rect", {
                            width: 130.055,
                            height: 122.6,
                            x: 15.398,
                            y: 19,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("clipPath", {
                id: "09ee95__a",
                children: /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M33.846 34h92.308c3.691 0 7.045 1.6 9.527 4.206L82.56 69.2c-1.864 1.088-4.155 1.066-6-.056l-51.7-31.476C27.28 35.38 30.418 34 33.847 34M20.702 44.501c-.456 1.52-.702 3.144-.702 4.832v61.334C20 119.135 26.2 126 33.846 126h92.308c7.647 0 13.846-6.865 13.846-15.333V49.333c0-1.417-.174-2.79-.499-4.093l-52.91 30.87c-4.401 2.567-9.837 2.516-14.19-.134z"
                })
            }),
            /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("g", {
                clipPath: "url(#09ee95__a)",
                children: /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#09ee95__b)",
                    children: [
                        /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("rect", {
                            width: 130.055,
                            height: 122.6,
                            x: 15.398,
                            y: 19,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M15.398 19h130.055v122.6H15.398z"
                        }),
                        /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.581,
                            cy: 53.338,
                            fill: "url(#09ee95__c)",
                            rx: 56.581,
                            ry: 53.338,
                            transform: "matrix(1 -.00132 .00148 1 32.273 -29.508)"
                        }),
                        /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("path", {
                            fill: "url(#09ee95__d)",
                            d: "M65.037-2.657c-9.989-5.826-22.643-5.862-32.669-.092l-100.27 57.712c-9.82 5.653-9.837 18.994-.03 24.668l99.608 57.631c10.01 5.79 22.665 5.783 32.667-.021l99.403-57.685c9.755-5.661 9.773-18.928.029-24.611z"
                        }),
                        /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("path", {
                            fill: "url(#09ee95__e)",
                            d: "M34.142 77.207c2.223-45.77 43.386-81.175 91.939-79.08 48.554 2.096 86.112 40.899 83.889 86.669s-43.385 81.175-91.939 79.08c-48.553-2.096-86.111-40.898-83.889-86.669"
                        }),
                        /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("path", {
                            fill: "url(#09ee95__f)",
                            d: "M-37.293 107.386c19.3-7.761 40.115-7.428 58.149-.64 36.085 13.619 57.082 5.199 71.645-28.74 7.254-17.03 21.417-31.454 40.739-39.224 38.631-15.512 83.229 1.32 99.66 37.628 16.432 36.309-1.553 78.284-40.175 93.816-19.365 7.788-40.22 7.423-58.281.573-36.027-13.546-56.984-5.094-71.529 28.886-7.272 16.99-21.432 31.364-40.732 39.125-38.6 15.523-83.198-1.308-99.62-37.596-16.423-36.289 1.544-78.305 40.144-93.828"
                        }),
                        /*#__PURE__*/ (0, $aUnJl$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.581,
                            cy: 53.338,
                            fill: "url(#09ee95__g)",
                            rx: 56.581,
                            ry: 53.338,
                            transform: "matrix(1 -.00132 .00148 1 81.215 -12.113)"
                        })
                    ]
                })
            })
        ]
    });
const $d7fcc7991554d14c$var$ForwardRef = /*#__PURE__*/ (0, $aUnJl$react.forwardRef)($d7fcc7991554d14c$var$SvgComponent);
var $d7fcc7991554d14c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d7fcc7991554d14c$var$ForwardRef);
