// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_musicNote_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bGzMd$jsxs, jsx as $bGzMd$jsx} from "react/jsx-runtime";
import {forwardRef as $bGzMd$forwardRef} from "react";





const $eba3d93c23d201d6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bGzMd$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bGzMd$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bGzMd$jsxs)("linearGradient", {
                        id: "ab8bf3__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bGzMd$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bGzMd$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bGzMd$jsxs)("linearGradient", {
                        id: "ab8bf3__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bGzMd$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bGzMd$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bGzMd$jsxs)("linearGradient", {
                        id: "ab8bf3__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bGzMd$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bGzMd$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bGzMd$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bGzMd$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bGzMd$jsxs)("radialGradient", {
                        id: "ab8bf3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bGzMd$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bGzMd$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bGzMd$jsxs)("radialGradient", {
                        id: "ab8bf3__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bGzMd$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bGzMd$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bGzMd$jsx)("clipPath", {
                        id: "ab8bf3__b",
                        children: /*#__PURE__*/ (0, $bGzMd$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bGzMd$jsx)("clipPath", {
                id: "ab8bf3__a",
                children: /*#__PURE__*/ (0, $bGzMd$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M39.78 69.65V35.336l35.966-7.17v27.225c-1.01-.587-2.143-.994-3.321-1.231-2.902-.583-6.28-.194-9.496 1.188s-5.825 3.564-7.4 6.07c-1.57 2.498-2.207 5.508-1.017 8.28s3.813 4.381 6.706 4.962c2.902.583 6.28.194 9.496-1.188s5.824-3.564 7.4-6.07c.855-1.36 1.433-2.874 1.59-4.417q.041-.198.042-.407v-.276q.003-.156 0-.312V20.597c0-5.047-4.615-8.833-9.564-7.846l-27.966 5.575c-3.742.746-6.436 4.03-6.436 7.846v37.087c-1.01-.587-2.143-.994-3.32-1.231-2.903-.583-6.28-.194-9.497 1.188s-5.824 3.564-7.4 6.07c-1.57 2.498-2.207 5.508-1.017 8.28 1.191 2.771 3.814 4.381 6.706 4.962 2.902.583 6.28.194 9.496-1.188s5.824-3.564 7.4-6.07c1.056-1.681 1.69-3.595 1.63-5.51z"
                })
            }),
            /*#__PURE__*/ (0, $bGzMd$jsx)("g", {
                clipPath: "url(#ab8bf3__a)",
                children: /*#__PURE__*/ (0, $bGzMd$jsxs)("g", {
                    clipPath: "url(#ab8bf3__b)",
                    children: [
                        /*#__PURE__*/ (0, $bGzMd$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $bGzMd$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $bGzMd$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#ab8bf3__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $bGzMd$jsx)("path", {
                            fill: "url(#ab8bf3__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $bGzMd$jsx)("path", {
                            fill: "url(#ab8bf3__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $bGzMd$jsx)("path", {
                            fill: "url(#ab8bf3__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $bGzMd$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#ab8bf3__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $eba3d93c23d201d6$var$ForwardRef = /*#__PURE__*/ (0, $bGzMd$forwardRef)($eba3d93c23d201d6$var$SvgComponent);
var $eba3d93c23d201d6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($eba3d93c23d201d6$var$ForwardRef);


export {$eba3d93c23d201d6$export$2e2bcd8739ae039 as default};
