// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_data_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $67MzE$jsxs, jsx as $67MzE$jsx} from "react/jsx-runtime";
import {forwardRef as $67MzE$forwardRef} from "react";





const $e5ad017bbaac2dbb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $67MzE$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $67MzE$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $67MzE$jsxs)("linearGradient", {
                        id: "952440__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $67MzE$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $67MzE$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $67MzE$jsxs)("linearGradient", {
                        id: "952440__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $67MzE$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $67MzE$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $67MzE$jsxs)("linearGradient", {
                        id: "952440__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $67MzE$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $67MzE$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $67MzE$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $67MzE$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $67MzE$jsxs)("radialGradient", {
                        id: "952440__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $67MzE$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $67MzE$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $67MzE$jsxs)("radialGradient", {
                        id: "952440__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $67MzE$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $67MzE$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $67MzE$jsx)("clipPath", {
                        id: "952440__b",
                        children: /*#__PURE__*/ (0, $67MzE$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $67MzE$jsx)("clipPath", {
                id: "952440__a",
                children: /*#__PURE__*/ (0, $67MzE$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m127.749 44.509.001-.082c0-6.854-21.379-12.41-47.75-12.41-26.372 0-47.75 5.556-47.75 12.41l.001.082v.197c.159.516.693 1.431 2.373 2.662 2.123 1.556 5.451 3.125 9.923 4.512C53.445 54.638 65.98 56.406 80 56.406c14.02 0 26.555-1.768 35.452-4.526 4.473-1.387 7.801-2.956 9.924-4.512 1.683-1.233 2.216-2.15 2.374-2.665v-.194zM32.251 62.557v-7.204c2.78 1.615 6.164 3.001 9.927 4.168 9.856 3.056 23.227 4.885 37.822 4.885 14.594 0 27.965-1.83 37.821-4.885 3.764-1.167 7.148-2.553 9.929-4.169v7.202c-.158.515-.691 1.431-2.374 2.665-2.123 1.556-5.451 3.125-9.924 4.512-8.897 2.758-21.432 4.525-35.452 4.525s-26.556-1.767-35.452-4.525c-4.473-1.387-7.801-2.956-9.924-4.512-1.68-1.231-2.214-2.146-2.373-2.662m95.499 10.646c-2.781 1.616-6.165 3.002-9.929 4.169-9.856 3.055-23.227 4.884-37.821 4.884-14.595 0-27.966-1.829-37.822-4.884-3.763-1.167-7.147-2.553-9.927-4.168v7.16c.159.516.693 1.431 2.373 2.663 2.123 1.556 5.451 3.124 9.923 4.511C53.445 90.296 65.98 92.064 80 92.064c14.02 0 26.555-1.768 35.452-4.526 4.473-1.386 7.801-2.955 9.924-4.511 1.683-1.234 2.216-2.15 2.374-2.665zM32.25 98.26v-7.249c2.781 1.616 6.165 3.002 9.928 4.168 9.856 3.056 23.227 4.885 37.822 4.885 14.594 0 27.965-1.83 37.821-4.885 3.764-1.166 7.148-2.553 9.929-4.168v7.246c-.158.515-.691 1.431-2.374 2.665-2.123 1.556-5.451 3.125-9.924 4.512-8.897 2.758-21.432 4.525-35.452 4.525s-26.556-1.767-35.452-4.525c-4.473-1.387-7.801-2.956-9.924-4.512-1.68-1.231-2.214-2.146-2.373-2.662m0 10.647v7.616c10.25 13.938 81.25 16.563 95.5 0v-7.617c-2.781 1.616-6.165 3.002-9.929 4.169-9.856 3.055-23.227 4.884-37.821 4.884-14.595 0-27.966-1.829-37.822-4.884-3.763-1.167-7.147-2.553-9.927-4.168"
                })
            }),
            /*#__PURE__*/ (0, $67MzE$jsx)("g", {
                clipPath: "url(#952440__a)",
                children: /*#__PURE__*/ (0, $67MzE$jsxs)("g", {
                    clipPath: "url(#952440__b)",
                    children: [
                        /*#__PURE__*/ (0, $67MzE$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $67MzE$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $67MzE$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#952440__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $67MzE$jsx)("path", {
                            fill: "url(#952440__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $67MzE$jsx)("path", {
                            fill: "url(#952440__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $67MzE$jsx)("path", {
                            fill: "url(#952440__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $67MzE$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#952440__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $e5ad017bbaac2dbb$var$ForwardRef = /*#__PURE__*/ (0, $67MzE$forwardRef)($e5ad017bbaac2dbb$var$SvgComponent);
var $e5ad017bbaac2dbb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e5ad017bbaac2dbb$var$ForwardRef);


export {$e5ad017bbaac2dbb$export$2e2bcd8739ae039 as default};
