// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_bolt_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $jpaj5$jsxs, jsx as $jpaj5$jsx} from "react/jsx-runtime";
import {forwardRef as $jpaj5$forwardRef} from "react";





const $03a36bf002daf18c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jpaj5$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jpaj5$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jpaj5$jsxs)("linearGradient", {
                        id: "64e965__d",
                        x1: -8.937,
                        x2: 109.572,
                        y1: 75.484,
                        y2: -1,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jpaj5$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $jpaj5$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jpaj5$jsxs)("linearGradient", {
                        id: "64e965__e",
                        x1: 89.384,
                        x2: 40.132,
                        y1: 25.086,
                        y2: 81.265,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jpaj5$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jpaj5$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jpaj5$jsxs)("linearGradient", {
                        id: "64e965__f",
                        x1: 25.507,
                        x2: 115.976,
                        y1: 85.716,
                        y2: 31.955,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jpaj5$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jpaj5$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jpaj5$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jpaj5$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jpaj5$jsxs)("radialGradient", {
                        id: "64e965__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jpaj5$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $jpaj5$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jpaj5$jsxs)("radialGradient", {
                        id: "64e965__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jpaj5$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jpaj5$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jpaj5$jsx)("clipPath", {
                        id: "64e965__b",
                        children: /*#__PURE__*/ (0, $jpaj5$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 10,
                            y: 12,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jpaj5$jsx)("clipPath", {
                id: "64e965__a",
                children: /*#__PURE__*/ (0, $jpaj5$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m53.243 17.419-25.2 28.211c-2.044 2.291-1.25 5.915 1.577 7.13l11.321 4.873c.153.066.221.237.157.39L34.37 73.984c-1.97 4.672 4.095 8.525 7.49 4.76l25.08-27.833c2.068-2.296 1.26-5.951-1.589-7.156l-11.398-4.83h-.004c-.15-.064-.222-.236-.156-.392l6.943-16.356v-.002c1.99-4.68-4.105-8.551-7.495-4.757"
                })
            }),
            /*#__PURE__*/ (0, $jpaj5$jsx)("g", {
                clipPath: "url(#64e965__a)",
                children: /*#__PURE__*/ (0, $jpaj5$jsxs)("g", {
                    clipPath: "url(#64e965__b)",
                    children: [
                        /*#__PURE__*/ (0, $jpaj5$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 10,
                            y: 12,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $jpaj5$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M10 12h75.36v71.04H10z"
                        }),
                        /*#__PURE__*/ (0, $jpaj5$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#64e965__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 19.777 -16.107)"
                        }),
                        /*#__PURE__*/ (0, $jpaj5$jsx)("path", {
                            fill: "url(#64e965__d)",
                            d: "M38.765-.55c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.356 13.133 3.352 18.928-.012L95.962 47.09c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $jpaj5$jsx)("path", {
                            fill: "url(#64e965__e)",
                            d: "M20.86 45.728C22.148 19.207 46-1.308 74.133-.094s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822s-49.897-23.698-48.61-50.22"
                        }),
                        /*#__PURE__*/ (0, $jpaj5$jsx)("path", {
                            fill: "url(#64e965__f)",
                            d: "M-20.534 63.215c11.184-4.498 23.245-4.305 33.694-.37 20.91 7.89 33.076 3.011 41.515-16.654 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804 9.521 21.04-.901 45.361-23.28 54.361-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.448 16.738-4.213 9.845-12.418 18.173-23.601 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $jpaj5$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#64e965__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 48.137 -6.028)"
                        })
                    ]
                })
            })
        ]
    });
const $03a36bf002daf18c$var$ForwardRef = /*#__PURE__*/ (0, $jpaj5$forwardRef)($03a36bf002daf18c$var$SvgComponent);
var $03a36bf002daf18c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($03a36bf002daf18c$var$ForwardRef);


export {$03a36bf002daf18c$export$2e2bcd8739ae039 as default};
