// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_EmptyStateExport_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $8CimK$jsxs, jsx as $8CimK$jsx} from "react/jsx-runtime";
import {forwardRef as $8CimK$forwardRef} from "react";





const $abc1919d351b3809$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8CimK$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8CimK$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8CimK$jsxs)("linearGradient", {
                        id: "a53a7d__d",
                        x1: -21.932,
                        x2: 196.624,
                        y1: 133.068,
                        y2: -8.02,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8CimK$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8CimK$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8CimK$jsxs)("linearGradient", {
                        id: "a53a7d__e",
                        x1: 159.425,
                        x2: 68.606,
                        y1: 40.13,
                        y2: 143.748,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8CimK$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8CimK$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8CimK$jsxs)("linearGradient", {
                        id: "a53a7d__f",
                        x1: 41.607,
                        x2: 208.454,
                        y1: 151.936,
                        y2: 52.763,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8CimK$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8CimK$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8CimK$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8CimK$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8CimK$jsxs)("radialGradient", {
                        id: "a53a7d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 56.9922 -60.4726 0 60.473 56.992)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8CimK$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8CimK$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8CimK$jsxs)("radialGradient", {
                        id: "a53a7d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 56.9922 -60.4726 0 60.473 56.992)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8CimK$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8CimK$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8CimK$jsx)("clipPath", {
                        id: "a53a7d__b",
                        children: /*#__PURE__*/ (0, $8CimK$jsx)("rect", {
                            width: 139,
                            height: 131,
                            x: 13,
                            y: 16,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8CimK$jsx)("clipPath", {
                id: "a53a7d__a",
                children: /*#__PURE__*/ (0, $8CimK$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M111.999 16h20c6.627 0 12 5.373 12 12v44c0 6.627-5.373 12-12 12h-20c-6.628 0-12-5.373-12-12V28c0-6.627 5.372-12 12-12m-54 84h44c6.627 0 12 5.373 12 12v20c0 6.627-5.373 12-12 12h-44c-6.628 0-12-5.373-12-12v-20c0-6.627 5.372-12 12-12m8.25-28.53 8.26 20.12c1.34 3.25 5.92 3.32 7.35.11l2.41-5.4c.4-.9 1.12-1.62 2.02-2.02l5.39-2.41c3.21-1.43 3.14-6.02-.11-7.35l-20.11-8.27c-3.29-1.35-6.57 1.93-5.22 5.22zm-9.83 3.88c2.21 0 4 1.79 4 4s-1.79 4-4 4h-20.45c-1.49 3.06-4.61 5.17-8.24 5.17-5.07 0-9.17-4.11-9.17-9.17 0-3.63 2.12-6.75 5.17-8.24V35.9c-3.06-1.49-5.17-4.61-5.17-8.24 0-5.07 4.11-9.17 9.17-9.17 3.63 0 6.75 2.12 8.24 5.17h35.06c1.49-3.06 4.61-5.17 8.24-5.17 5.07 0 9.17 4.11 9.17 9.17 0 3.63-2.12 6.75-5.17 8.24v20.5c0 2.21-1.79 4-4 4s-4-1.79-4-4V35.9c-1.85-.9-3.34-2.4-4.24-4.24h-35.06c-.9 1.85-2.4 3.34-4.24 4.24v35.21c1.85.9 3.34 2.4 4.24 4.24z"
                })
            }),
            /*#__PURE__*/ (0, $8CimK$jsx)("g", {
                clipPath: "url(#a53a7d__a)",
                children: /*#__PURE__*/ (0, $8CimK$jsxs)("g", {
                    clipPath: "url(#a53a7d__b)",
                    children: [
                        /*#__PURE__*/ (0, $8CimK$jsx)("rect", {
                            width: 139,
                            height: 131,
                            x: 13,
                            y: 16,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $8CimK$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M13 16h139v131H13z"
                        }),
                        /*#__PURE__*/ (0, $8CimK$jsx)("ellipse", {
                            cx: 60.473,
                            cy: 56.992,
                            fill: "url(#a53a7d__c)",
                            rx: 60.473,
                            ry: 56.992,
                            transform: "matrix(1 -.00132 .00148 1 31.035 -35.831)"
                        }),
                        /*#__PURE__*/ (0, $8CimK$jsx)("path", {
                            fill: "url(#a53a7d__d)",
                            d: "M66.054-7.14c-10.675-6.226-24.2-6.264-34.915-.098L-76.028 54.428c-10.495 6.04-10.513 20.296-.032 26.358L30.4 142.365c10.699 6.188 24.224 6.18 34.914-.022l106.239-61.637c10.426-6.05 10.445-20.225.032-26.298z"
                        }),
                        /*#__PURE__*/ (0, $8CimK$jsx)("path", {
                            fill: "url(#a53a7d__e)",
                            d: "M33.034 78.196C35.409 29.29 79.403-8.542 131.296-6.303s92.034 43.7 89.658 92.607-46.369 86.737-98.262 84.498-92.034-43.7-89.658-92.606"
                        }),
                        /*#__PURE__*/ (0, $8CimK$jsx)("path", {
                            fill: "url(#a53a7d__f)",
                            d: "M-43.314 110.443c20.628-8.294 42.874-7.938 62.148-.684 38.567 14.552 61.008 5.555 76.573-30.71 7.753-18.197 22.89-33.608 43.54-41.911 41.288-16.574 88.953 1.41 106.515 40.207 17.562 38.796-1.66 83.647-42.938 100.243-20.697 8.321-42.987 7.932-62.29.612-38.505-14.474-60.903-5.443-76.448 30.865-7.773 18.154-22.906 33.513-43.533 41.806-41.255 16.587-88.92-1.397-106.472-40.172s1.65-83.67 42.905-100.256"
                        }),
                        /*#__PURE__*/ (0, $8CimK$jsx)("ellipse", {
                            cx: 60.473,
                            cy: 56.992,
                            fill: "url(#a53a7d__g)",
                            rx: 60.473,
                            ry: 56.992,
                            transform: "matrix(1 -.00132 .00148 1 83.34 -17.245)"
                        })
                    ]
                })
            })
        ]
    });
const $abc1919d351b3809$var$ForwardRef = /*#__PURE__*/ (0, $8CimK$forwardRef)($abc1919d351b3809$var$SvgComponent);
var $abc1919d351b3809$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($abc1919d351b3809$var$ForwardRef);


export {$abc1919d351b3809$export$2e2bcd8739ae039 as default};
