// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_globeGrid_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2oIhl$jsxs, jsx as $2oIhl$jsx} from "react/jsx-runtime";
import {forwardRef as $2oIhl$forwardRef} from "react";





const $5afb1500cab61299$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2oIhl$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2oIhl$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2oIhl$jsxs)("linearGradient", {
                        id: "a7a5c4__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2oIhl$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2oIhl$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2oIhl$jsxs)("linearGradient", {
                        id: "a7a5c4__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2oIhl$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2oIhl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2oIhl$jsxs)("linearGradient", {
                        id: "a7a5c4__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2oIhl$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2oIhl$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2oIhl$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2oIhl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2oIhl$jsxs)("radialGradient", {
                        id: "a7a5c4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2oIhl$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2oIhl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2oIhl$jsxs)("radialGradient", {
                        id: "a7a5c4__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2oIhl$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2oIhl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2oIhl$jsx)("clipPath", {
                        id: "a7a5c4__b",
                        children: /*#__PURE__*/ (0, $2oIhl$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2oIhl$jsx)("clipPath", {
                id: "a7a5c4__a",
                children: /*#__PURE__*/ (0, $2oIhl$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m50.028 15.075-.06-.004v14.605h9.21c-2.07-4.968-5.037-9.767-8.915-14.265q-.137-.159-.235-.336m13.46 14.6c-1.804-4.74-4.35-9.348-7.652-13.725 8.128 1.98 15.076 6.972 19.596 13.726zm-2.845 4H49.968v12.676h12.816c-.115-4.266-.826-8.516-2.141-12.675m6.143 12.676c-.105-4.259-.755-8.506-1.961-12.675H77.72c1.867 3.865 3.001 8.15 3.224 12.675zm-4.067 4H49.968V63.03h10.231c1.423-4.17 2.266-8.42 2.52-12.678m1.685 12.678c1.309-4.174 2.086-8.421 2.321-12.678h14.178c-.32 4.542-1.56 8.83-3.533 12.678zm-5.758 4h-8.678v13.898l.061-.004q.087-.155.203-.298c3.55-4.322 6.36-8.89 8.414-13.596M55.71 80.078c3.042-4.18 5.469-8.558 7.267-13.049h11.967c-4.548 6.429-11.34 11.158-19.234 13.049M20.568 29.676c4.505-6.733 11.424-11.714 19.518-13.707-3.295 4.371-5.836 8.974-7.637 13.707zm-2.006 4h12.55c-1.206 4.17-1.857 8.416-1.962 12.675H15.055c.223-4.532 1.36-8.824 3.232-12.694q.135.018.275.019m16.732 0c-1.316 4.16-2.026 8.41-2.142 12.675h12.816V33.676zM15.097 50.35H29.21c.236 4.257 1.012 8.505 2.32 12.678H18.63c-1.974-3.849-3.213-8.136-3.533-12.678m18.12 0c.255 4.258 1.098 8.508 2.52 12.678h10.23V50.351zM21.054 67.03h11.904c1.796 4.485 4.218 8.856 7.254 13.03-7.862-1.902-14.624-6.622-19.158-13.03m24.65 13.596q.115.141.201.294l.063.004V67.029H37.29c2.054 4.705 4.863 9.274 8.413 13.596m-.03-65.214c-3.879 4.498-6.845 9.297-8.916 14.265h9.21v-14.6l-.062.003q-.097.175-.233.332"
                })
            }),
            /*#__PURE__*/ (0, $2oIhl$jsx)("g", {
                clipPath: "url(#a7a5c4__a)",
                children: /*#__PURE__*/ (0, $2oIhl$jsxs)("g", {
                    clipPath: "url(#a7a5c4__b)",
                    children: [
                        /*#__PURE__*/ (0, $2oIhl$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $2oIhl$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $2oIhl$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#a7a5c4__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $2oIhl$jsx)("path", {
                            fill: "url(#a7a5c4__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $2oIhl$jsx)("path", {
                            fill: "url(#a7a5c4__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $2oIhl$jsx)("path", {
                            fill: "url(#a7a5c4__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $2oIhl$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#a7a5c4__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $5afb1500cab61299$var$ForwardRef = /*#__PURE__*/ (0, $2oIhl$forwardRef)($5afb1500cab61299$var$SvgComponent);
var $5afb1500cab61299$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5afb1500cab61299$var$ForwardRef);


export {$5afb1500cab61299$export$2e2bcd8739ae039 as default};
