// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_animation_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6GfAb$jsxs, jsx as $6GfAb$jsx} from "react/jsx-runtime";
import {forwardRef as $6GfAb$forwardRef} from "react";





const $83ada5141ccf0b25$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6GfAb$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6GfAb$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6GfAb$jsxs)("linearGradient", {
                        id: "085d4b__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6GfAb$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $6GfAb$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6GfAb$jsxs)("linearGradient", {
                        id: "085d4b__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6GfAb$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6GfAb$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6GfAb$jsxs)("linearGradient", {
                        id: "085d4b__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6GfAb$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6GfAb$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6GfAb$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6GfAb$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6GfAb$jsxs)("radialGradient", {
                        id: "085d4b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6GfAb$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $6GfAb$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6GfAb$jsxs)("radialGradient", {
                        id: "085d4b__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6GfAb$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6GfAb$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6GfAb$jsx)("clipPath", {
                        id: "085d4b__b",
                        children: /*#__PURE__*/ (0, $6GfAb$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6GfAb$jsx)("clipPath", {
                id: "085d4b__a",
                children: /*#__PURE__*/ (0, $6GfAb$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M41.026 9.042c0 .977-.578 1.819-1.41 2.203-.87-1.076-2.065-1.878-3.436-2.255.028-1.314 1.102-2.37 2.423-2.37 1.338 0 2.423 1.084 2.423 2.422m-2.424 6.412c0 1.735-1.05 3.225-2.551 3.867-1.178-2.234-3.288-3.9-5.813-4.48.297-2.031 2.046-3.591 4.16-3.591 2.322 0 4.204 1.882 4.204 4.204m-7.784 2.205c-.783-.348-1.65-.542-2.562-.542-2.834 0-5.23 1.868-6.027 4.44 3.015 1.695 5.219 4.663 5.882 8.175l.145.002c3.484 0 6.308-2.825 6.308-6.309 0-2.483-1.435-4.631-3.522-5.66q-.11-.055-.224-.106m-5 14.299c0 5.204-4.218 9.423-9.423 9.423-5.204 0-9.422-4.219-9.422-9.423s4.218-9.423 9.423-9.423c2.109 0 4.056.693 5.626 1.864 1.543 1.15 2.722 2.761 3.332 4.63.302.921.464 1.906.464 2.929"
                })
            }),
            /*#__PURE__*/ (0, $6GfAb$jsx)("g", {
                clipPath: "url(#085d4b__a)",
                children: /*#__PURE__*/ (0, $6GfAb$jsxs)("g", {
                    clipPath: "url(#085d4b__b)",
                    children: [
                        /*#__PURE__*/ (0, $6GfAb$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $6GfAb$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $6GfAb$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#085d4b__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $6GfAb$jsx)("path", {
                            fill: "url(#085d4b__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $6GfAb$jsx)("path", {
                            fill: "url(#085d4b__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $6GfAb$jsx)("path", {
                            fill: "url(#085d4b__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $6GfAb$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#085d4b__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $83ada5141ccf0b25$var$ForwardRef = /*#__PURE__*/ (0, $6GfAb$forwardRef)($83ada5141ccf0b25$var$SvgComponent);
var $83ada5141ccf0b25$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($83ada5141ccf0b25$var$ForwardRef);


export {$83ada5141ccf0b25$export$2e2bcd8739ae039 as default};
