// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_replace_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6FNzC$jsxs, jsx as $6FNzC$jsx} from "react/jsx-runtime";
import {forwardRef as $6FNzC$forwardRef} from "react";





const $d0f650e5bcc50f27$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6FNzC$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6FNzC$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6FNzC$jsxs)("linearGradient", {
                        id: "90bc69__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6FNzC$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $6FNzC$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6FNzC$jsxs)("linearGradient", {
                        id: "90bc69__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6FNzC$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6FNzC$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6FNzC$jsxs)("linearGradient", {
                        id: "90bc69__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6FNzC$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6FNzC$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6FNzC$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6FNzC$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6FNzC$jsxs)("radialGradient", {
                        id: "90bc69__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6FNzC$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $6FNzC$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6FNzC$jsxs)("radialGradient", {
                        id: "90bc69__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6FNzC$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6FNzC$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6FNzC$jsx)("clipPath", {
                        id: "90bc69__b",
                        children: /*#__PURE__*/ (0, $6FNzC$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6FNzC$jsx)("clipPath", {
                id: "90bc69__a",
                children: /*#__PURE__*/ (0, $6FNzC$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M40.23 24.268c-8.836 0-16 7.163-16 16v15.337c0 8.837 7.164 16 16 16h15.338c8.836 0 16-7.163 16-16V40.268c0-8.837-7.164-16-16-16zM91.974 26.2c-2.207-.075-4.058 1.654-4.133 3.862s1.654 4.058 3.862 4.133c4.874.165 9.865 1.57 13.628 5.21 3.247 3.141 6.031 8.386 6.675 17.199l-6.177-6.178c-1.562-1.562-4.095-1.562-5.657 0s-1.562 4.095 0 5.657l12.424 12.424c1.562 1.562 4.095 1.562 5.657 0l12.464-12.464c1.562-1.562 1.562-4.095 0-5.657s-4.095-1.562-5.657 0l-5.116 5.117c-.829-10.013-4.146-17.102-9.051-21.848-5.645-5.46-12.808-7.248-18.919-7.455M47.244 91.493c-1.562-1.562-4.094-1.562-5.657 0l-12.464 12.464c-1.562 1.562-1.562 4.095 0 5.657s4.095 1.562 5.657 0l5.116-5.116c.829 10.012 4.146 17.101 9.05 21.847 5.645 5.461 12.809 7.248 18.92 7.455 2.207.075 4.058-1.654 4.133-3.862s-1.655-4.058-3.862-4.133c-4.874-.165-9.865-1.57-13.628-5.21-3.247-3.141-6.032-8.387-6.676-17.199l6.179 6.178c1.562 1.562 4.094 1.562 5.656 0s1.562-4.095 0-5.657zm57.188-3.097c-8.836 0-16 7.163-16 16v15.337c0 8.836 7.164 16 16 16h15.338c8.836 0 16-7.164 16-16v-15.337c0-8.837-7.164-16-16-16z"
                })
            }),
            /*#__PURE__*/ (0, $6FNzC$jsx)("g", {
                clipPath: "url(#90bc69__a)",
                children: /*#__PURE__*/ (0, $6FNzC$jsxs)("g", {
                    clipPath: "url(#90bc69__b)",
                    children: [
                        /*#__PURE__*/ (0, $6FNzC$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $6FNzC$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $6FNzC$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#90bc69__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $6FNzC$jsx)("path", {
                            fill: "url(#90bc69__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $6FNzC$jsx)("path", {
                            fill: "url(#90bc69__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $6FNzC$jsx)("path", {
                            fill: "url(#90bc69__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $6FNzC$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#90bc69__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $d0f650e5bcc50f27$var$ForwardRef = /*#__PURE__*/ (0, $6FNzC$forwardRef)($d0f650e5bcc50f27$var$SvgComponent);
var $d0f650e5bcc50f27$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d0f650e5bcc50f27$var$ForwardRef);


export {$d0f650e5bcc50f27$export$2e2bcd8739ae039 as default};
