// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_heart_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $AAzxD$jsxs, jsx as $AAzxD$jsx} from "react/jsx-runtime";
import {forwardRef as $AAzxD$forwardRef} from "react";





const $6d9ffe691cef224b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $AAzxD$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $AAzxD$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $AAzxD$jsxs)("linearGradient", {
                        id: "f43666__d",
                        x1: -6.353,
                        x2: 59.005,
                        y1: 38.625,
                        y2: -1.138,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $AAzxD$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $AAzxD$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $AAzxD$jsxs)("linearGradient", {
                        id: "f43666__e",
                        x1: 46.098,
                        x2: 18.059,
                        y1: 10.105,
                        y2: 40.254,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $AAzxD$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $AAzxD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $AAzxD$jsxs)("linearGradient", {
                        id: "f43666__f",
                        x1: 12.025,
                        x2: 61.729,
                        y1: 44.415,
                        y2: 16.572,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $AAzxD$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $AAzxD$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $AAzxD$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $AAzxD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $AAzxD$jsxs)("radialGradient", {
                        id: "f43666__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.11 15.37)scale(17.4892)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $AAzxD$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $AAzxD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $AAzxD$jsxs)("radialGradient", {
                        id: "f43666__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 15.822 25.785)scale(17.4892)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $AAzxD$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $AAzxD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $AAzxD$jsx)("clipPath", {
                        id: "f43666__b",
                        children: /*#__PURE__*/ (0, $AAzxD$jsx)("rect", {
                            width: 40.2,
                            height: 40.2,
                            x: 3.75,
                            y: 2.7,
                            fill: "#fff",
                            rx: 1.364
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $AAzxD$jsx)("clipPath", {
                id: "f43666__a",
                children: /*#__PURE__*/ (0, $AAzxD$jsx)("path", {
                    fill: "#fff",
                    d: "M32.402 7.8c-4.64.003-8.4 3.798-8.4 8.478l-.003.02-.003-.02c0-4.68-3.76-8.475-8.4-8.478-5.115-.003-8.773 4.41-8.366 9.554.737 9.305 10.76 19.751 14.947 23.106 1.065.853 2.572.853 3.637 0 4.19-3.354 14.217-13.801 14.954-23.107.408-5.144-3.251-9.556-8.366-9.553"
                })
            }),
            /*#__PURE__*/ (0, $AAzxD$jsx)("g", {
                clipPath: "url(#f43666__a)",
                children: /*#__PURE__*/ (0, $AAzxD$jsxs)("g", {
                    clipPath: "url(#f43666__b)",
                    children: [
                        /*#__PURE__*/ (0, $AAzxD$jsx)("rect", {
                            width: 40.2,
                            height: 40.2,
                            x: 3.75,
                            y: 2.7,
                            fill: "#FFECCF",
                            rx: 1.364
                        }),
                        /*#__PURE__*/ (0, $AAzxD$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.75 2.7h40.2v40.2H3.75z"
                        }),
                        /*#__PURE__*/ (0, $AAzxD$jsx)("circle", {
                            cx: 26.479,
                            cy: 4.26,
                            r: 17.489,
                            fill: "url(#f43666__c)",
                            transform: "rotate(-.08 26.479 4.26)"
                        }),
                        /*#__PURE__*/ (0, $AAzxD$jsx)("path", {
                            fill: "url(#f43666__d)",
                            d: "M19.094-4.4c-3.088-1.911-7-1.923-10.098-.031l-30.994 18.924c-3.035 1.853-3.04 6.228-.009 8.088L8.782 41.478c3.094 1.899 7.006 1.896 10.097-.007l30.726-18.914c3.015-1.857 3.02-6.207.009-8.07z"
                        }),
                        /*#__PURE__*/ (0, $AAzxD$jsx)("path", {
                            fill: "url(#f43666__e)",
                            d: "M9.544 21.786c.687-15.008 13.41-26.617 28.418-25.93s26.618 13.41 25.93 28.418-13.41 26.617-28.418 25.93-26.617-13.41-25.93-28.418"
                        }),
                        /*#__PURE__*/ (0, $AAzxD$jsx)("path", {
                            fill: "url(#f43666__f)",
                            d: "M-12.535 31.682c5.966-2.545 12.4-2.436 17.974-.21 11.154 4.466 17.644 1.705 22.145-9.423 2.242-5.585 6.62-10.314 12.592-12.862 11.941-5.086 25.726.433 30.805 12.338 5.08 11.906-.48 25.67-12.418 30.762-5.986 2.554-12.432 2.434-18.014.188-11.136-4.442-17.614-1.67-22.11 9.472-2.248 5.57-6.624 10.284-12.59 12.829-11.931 5.09-25.717-.43-30.793-12.328s.478-25.676 12.409-30.766"
                        }),
                        /*#__PURE__*/ (0, $AAzxD$jsx)("circle", {
                            cx: 41.607,
                            cy: 9.963,
                            r: 17.489,
                            fill: "url(#f43666__g)",
                            transform: "rotate(-.08 41.607 9.963)"
                        })
                    ]
                })
            })
        ]
    });
const $6d9ffe691cef224b$var$ForwardRef = /*#__PURE__*/ (0, $AAzxD$forwardRef)($6d9ffe691cef224b$var$SvgComponent);
var $6d9ffe691cef224b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6d9ffe691cef224b$var$ForwardRef);


export {$6d9ffe691cef224b$export$2e2bcd8739ae039 as default};
