// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_update_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $kepxx$jsxs, jsx as $kepxx$jsx} from "react/jsx-runtime";
import {forwardRef as $kepxx$forwardRef} from "react";





const $2a2d7bec470074e8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kepxx$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kepxx$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kepxx$jsxs)("linearGradient", {
                        id: "1f6621__d",
                        x1: -7.585,
                        x2: 59.276,
                        y1: 38.218,
                        y2: -4.934,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kepxx$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kepxx$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kepxx$jsxs)("linearGradient", {
                        id: "1f6621__e",
                        x1: 47.89,
                        x2: 20.102,
                        y1: 9.783,
                        y2: 41.479,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kepxx$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kepxx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kepxx$jsxs)("linearGradient", {
                        id: "1f6621__f",
                        x1: 11.852,
                        x2: 62.893,
                        y1: 43.991,
                        y2: 13.659,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kepxx$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kepxx$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kepxx$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kepxx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kepxx$jsxs)("radialGradient", {
                        id: "1f6621__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.437 -18.4973 0 18.497 17.437)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kepxx$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kepxx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kepxx$jsxs)("radialGradient", {
                        id: "1f6621__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.437 -18.4973 0 18.497 17.437)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kepxx$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kepxx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kepxx$jsx)("clipPath", {
                        id: "1f6621__b",
                        children: /*#__PURE__*/ (0, $kepxx$jsx)("rect", {
                            width: 42.517,
                            height: 40.08,
                            x: 3.102,
                            y: 2.4,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kepxx$jsxs)("clipPath", {
                id: "1f6621__a",
                children: [
                    /*#__PURE__*/ (0, $kepxx$jsx)("path", {
                        fill: "#fff",
                        fillRule: "evenodd",
                        d: "M42.037 12.57c.3-.591 1.023-.825 1.614-.525h.003c.59.3.825 1.023.525 1.614l-3.294 6.471c-.213.417-.633.657-1.071.657q-.258 0-.507-.114l-6.351-2.961c-.603-.282-.864-.996-.582-1.596s.996-.861 1.596-.579l4.149 1.935c-2.547-5.415-8.097-8.988-14.133-8.988-6.891 0-13.05 4.614-14.98 11.22-.152.525-.632.864-1.151.864q-.168 0-.336-.048c-.636-.186-1.002-.852-.816-1.488 2.226-7.623 9.333-12.948 17.283-12.948 6.879 0 13.209 4.02 16.194 10.137zM38.94 28.467c.186-.636.852-1.002 1.488-.816v.003c.636.186 1.002.852.816 1.488-2.226 7.623-9.333 12.948-17.283 12.948-6.846 0-13.15-3.984-16.152-10.053l-1.845 3.627c-.213.417-.633.657-1.071.657-.183 0-.37-.042-.543-.132-.591-.3-.825-1.023-.525-1.614l3.294-6.474c.294-.576.993-.816 1.578-.543l6.35 2.961c.604.282.862.996.583 1.596-.282.6-.996.858-1.596.579L9.85 30.741c2.553 5.394 8.09 8.946 14.112 8.946 6.89 0 13.05-4.614 14.979-11.22"
                    }),
                    /*#__PURE__*/ (0, $kepxx$jsx)("path", {
                        fill: "#fff",
                        d: "m43.65 12.045-.135.268.064.032h.072zm-1.613.525.267.136zm1.617-.525.135-.268-.064-.032h-.071zm.525 1.614.267.136zm-3.294 6.471.267.137zm-1.578.543-.127.272.003.001zm-6.351-2.961-.127.272zm-.582-1.596-.272-.128zm1.596-.579-.128.271zm4.149 1.935-.127.272.75.35-.352-.75zM9.007 19.704l-.288-.084zm-1.488.816.085-.288h-.001zm-.816-1.488.288.084zm33.477-2.811-.27.132.264.54.273-.536zm.249 11.43h.3v-.225l-.216-.063zm-1.488.816-.288-.084zm1.488-.813h-.3v.225l.216.063zm.816 1.488-.288-.084zM7.81 32.037l.269-.133-.265-.536-.271.533zm-1.845 3.627.267.136zm-1.614.525.138-.267h-.002zm-.525-1.614-.267-.136zm3.294-6.474-.267-.136zm1.578-.543-.127.272zm6.35 2.961.128-.272zm.583 1.596.271.128.001-.002zm-1.596.579-.127.272zM9.85 30.741l.127-.272-.754-.352.356.752zm33.937-18.963c-.738-.375-1.642-.083-2.018.656l.535.272c.225-.443.767-.619 1.211-.393zm-.133-.033h-.003v.6h.003zm.792 2.05c.375-.738.083-1.642-.657-2.017l-.271.534c.442.225.619.767.393 1.211zm-3.294 6.471 3.294-6.47-.535-.273-3.294 6.47zm-1.338.821c.547 0 1.072-.3 1.338-.82l-.534-.273c-.16.313-.475.493-.804.493zm-.631-.14c.2.09.413.14.63.14v-.6q-.187 0-.382-.087zm-6.354-2.963 6.351 2.96.254-.543-6.351-2.96zm-.727-1.996c-.353.752-.025 1.644.727 1.996l.254-.544c-.454-.212-.648-.748-.438-1.196zm1.995-.723c-.751-.353-1.643-.025-1.995.723l.543.256c.212-.452.749-.646 1.197-.435zm4.148 1.935-4.148-1.935-.254.544 4.149 1.935zm-14.26-8.416c5.92 0 11.364 3.505 13.862 8.816l.543-.256c-2.596-5.519-8.25-9.16-14.404-9.16zM9.296 19.788c1.891-6.478 7.933-11.004 14.69-11.004v-.6c-7.023 0-13.3 4.702-15.266 11.436zm-1.44 1.08c.648 0 1.248-.423 1.44-1.08l-.576-.168c-.115.393-.474.648-.864.648zm-.422-.06q.213.06.422.06v-.6q-.129 0-.25-.036zm-1.018-1.86c-.233.795.225 1.628 1.02 1.86l.168-.576c-.478-.14-.752-.64-.612-1.116zm17.57-13.164c-8.082 0-15.307 5.413-17.57 13.164l.576.168c2.188-7.495 9.178-12.732 16.995-12.732zM40.45 16.089C37.415 9.871 30.981 5.784 23.986 5.784v.6c6.763 0 12.989 3.954 15.924 9.969zm1.32-3.655-1.857 3.651.535.272 1.857-3.651zm-1.256 14.929c-.795-.233-1.628.225-1.86 1.02l.576.168c.14-.477.64-.751 1.116-.612zm.216.291v-.003h-.6v.003zm.804 1.572c.233-.795-.225-1.628-1.02-1.86l-.168.576c.477.14.751.64.612 1.116zM23.962 42.39c8.083 0 15.308-5.413 17.57-13.164l-.575-.168c-2.189 7.495-9.178 12.732-16.995 12.732zM7.542 32.17c3.052 6.17 9.458 10.22 16.42 10.22v-.6c-6.73 0-12.93-3.918-15.883-9.886zm-1.31 3.63 1.845-3.627-.534-.272-1.845 3.627zm-1.338.821c.547 0 1.072-.3 1.338-.82l-.534-.273c-.16.313-.475.493-.804.493zm-.68-.166c.217.113.45.166.68.166v-.6c-.137 0-.275-.031-.405-.099zm-.656-2.016c-.374.738-.082 1.642.657 2.017l.272-.535c-.443-.224-.619-.766-.394-1.21zm3.295-6.474-3.294 6.474.534.272 3.294-6.474zm1.972-.679c-.732-.341-1.605-.041-1.972.679l.534.272c.22-.432.746-.612 1.184-.407zm6.35 2.961-6.35-2.96-.254.543 6.351 2.96zm.728 1.995c.349-.75.026-1.643-.727-1.995l-.254.544c.453.212.646.747.437 1.197zm-1.995.724c.75.349 1.642.026 1.994-.723l-.543-.256c-.211.45-.747.644-1.198.435zm-4.185-1.953 4.185 1.953.254-.544-4.185-1.953zm14.239 8.374c-5.904 0-11.337-3.484-13.84-8.774l-.543.256c2.602 5.498 8.245 9.118 14.383 9.118zm14.691-11.004c-1.892 6.478-7.933 11.004-14.691 11.004v.6c7.024 0 13.3-4.702 15.267-11.436z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kepxx$jsx)("g", {
                clipPath: "url(#1f6621__a)",
                children: /*#__PURE__*/ (0, $kepxx$jsxs)("g", {
                    clipPath: "url(#1f6621__b)",
                    children: [
                        /*#__PURE__*/ (0, $kepxx$jsx)("rect", {
                            width: 42.517,
                            height: 40.08,
                            x: 3.102,
                            y: 2.4,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $kepxx$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.102 2.4h42.517v40.08H3.102z"
                        }),
                        /*#__PURE__*/ (0, $kepxx$jsx)("ellipse", {
                            cx: 18.497,
                            cy: 17.437,
                            fill: "url(#1f6621__c)",
                            rx: 18.497,
                            ry: 17.437,
                            transform: "matrix(1 -.00132 .00148 1 8.617 -13.458)"
                        }),
                        /*#__PURE__*/ (0, $kepxx$jsx)("path", {
                            fill: "url(#1f6621__d)",
                            d: "M19.328-4.68c-3.265-1.904-7.402-1.916-10.68-.03l-32.78 18.868c-3.21 1.847-3.216 6.209-.01 8.064l32.564 18.84c3.272 1.893 7.41 1.891 10.68-.006l32.496-18.858c3.189-1.851 3.195-6.189.01-8.046z"
                        }),
                        /*#__PURE__*/ (0, $kepxx$jsx)("path", {
                            fill: "url(#1f6621__e)",
                            d: "M9.23 21.43C9.955 6.465 23.412-5.109 39.285-4.424S67.437 8.947 66.71 23.91c-.726 14.963-14.183 26.538-30.056 25.853S8.503 36.393 9.229 21.429"
                        }),
                        /*#__PURE__*/ (0, $kepxx$jsx)("path", {
                            fill: "url(#1f6621__f)",
                            d: "M-14.124 31.296c6.31-2.538 13.114-2.429 19.01-.21 11.797 4.453 18.661 1.7 23.422-9.395 2.372-5.568 7.002-10.283 13.318-12.823 12.63-5.071 27.21.431 32.58 12.301 5.373 11.87-.507 25.592-13.133 30.67-6.33 2.546-13.149 2.427-19.053.187-11.778-4.428-18.63-1.665-23.384 9.444-2.378 5.554-7.007 10.253-13.316 12.79-12.62 5.075-27.199-.427-32.568-12.29-5.369-11.864.505-25.6 13.124-30.674"
                        }),
                        /*#__PURE__*/ (0, $kepxx$jsx)("ellipse", {
                            cx: 18.497,
                            cy: 17.437,
                            fill: "url(#1f6621__g)",
                            rx: 18.497,
                            ry: 17.437,
                            transform: "matrix(1 -.00132 .00148 1 24.617 -7.771)"
                        })
                    ]
                })
            })
        ]
    });
const $2a2d7bec470074e8$var$ForwardRef = /*#__PURE__*/ (0, $kepxx$forwardRef)($2a2d7bec470074e8$var$SvgComponent);
var $2a2d7bec470074e8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2a2d7bec470074e8$var$ForwardRef);


export {$2a2d7bec470074e8$export$2e2bcd8739ae039 as default};
