// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_buildTable_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $gT9i9$jsxs, jsx as $gT9i9$jsx} from "react/jsx-runtime";
import {forwardRef as $gT9i9$forwardRef} from "react";





const $77ef8f4aff4390e2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gT9i9$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gT9i9$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gT9i9$jsxs)("linearGradient", {
                        id: "364c63__d",
                        x1: -4.856,
                        x2: 60.06,
                        y1: 41.735,
                        y2: -0.161,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gT9i9$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gT9i9$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gT9i9$jsxs)("linearGradient", {
                        id: "364c63__e",
                        x1: 49.005,
                        x2: 22.026,
                        y1: 14.128,
                        y2: 44.901,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gT9i9$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gT9i9$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gT9i9$jsxs)("linearGradient", {
                        id: "364c63__f",
                        x1: 14.016,
                        x2: 63.572,
                        y1: 47.34,
                        y2: 17.891,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gT9i9$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gT9i9$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gT9i9$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gT9i9$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gT9i9$jsxs)("radialGradient", {
                        id: "364c63__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.9295 -17.959 0 17.959 16.93)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gT9i9$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gT9i9$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gT9i9$jsxs)("radialGradient", {
                        id: "364c63__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.9295 -17.959 0 17.959 16.93)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gT9i9$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gT9i9$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gT9i9$jsx)("clipPath", {
                        id: "364c63__b",
                        children: /*#__PURE__*/ (0, $gT9i9$jsx)("rect", {
                            width: 41.28,
                            height: 38.913,
                            x: 5.52,
                            y: 6.96,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gT9i9$jsx)("clipPath", {
                id: "364c63__a",
                children: /*#__PURE__*/ (0, $gT9i9$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M37.19 10.17c2.652 0 4.8 2.148 4.8 4.8v1.227H5.984V14.97c0-2.652 2.148-4.8 4.8-4.8zm4.8 15.768c-.773-.198-1.583-.303-2.417-.303h-.003c-5.382 0-9.76 4.377-9.76 9.759 0 .83.106 1.64.304 2.412H14.4v-19.21h27.59zm-36.006 7.07c0 2.653 2.148 4.8 4.8 4.8H12V18.6H5.984zm33.589-4.97c.846 0 1.659.144 2.418.408h.003c2.877 1.002 4.94 3.735 4.94 6.95 0 4.066-3.293 7.36-7.358 7.36-3.22 0-5.955-2.067-6.954-4.947-.264-.756-.408-1.566-.408-2.412 0-4.065 3.294-7.36 7.359-7.36m1.203 8.544h2.364c.663 0 1.2-.537 1.2-1.2s-.537-1.2-1.2-1.2h-2.364v-2.364c0-.663-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2v2.364h-2.364c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h2.364v2.364c0 .663.537 1.2 1.2 1.2s1.2-.537 1.2-1.2z"
                })
            }),
            /*#__PURE__*/ (0, $gT9i9$jsx)("g", {
                clipPath: "url(#364c63__a)",
                children: /*#__PURE__*/ (0, $gT9i9$jsxs)("g", {
                    clipPath: "url(#364c63__b)",
                    children: [
                        /*#__PURE__*/ (0, $gT9i9$jsx)("rect", {
                            width: 41.28,
                            height: 38.913,
                            x: 5.52,
                            y: 6.96,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $gT9i9$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M5.52 6.96H46.8v38.913H5.52z"
                        }),
                        /*#__PURE__*/ (0, $gT9i9$jsx)("ellipse", {
                            cx: 17.959,
                            cy: 16.93,
                            fill: "url(#364c63__c)",
                            rx: 17.959,
                            ry: 16.93,
                            transform: "matrix(1 -.00132 .00148 1 10.875 -8.437)"
                        }),
                        /*#__PURE__*/ (0, $gT9i9$jsx)("path", {
                            fill: "url(#364c63__d)",
                            d: "M21.274.086c-3.17-1.849-7.187-1.86-10.369-.029L-20.92 18.375c-3.117 1.794-3.122 6.029-.01 7.83l31.617 18.292c3.177 1.838 7.194 1.836 10.368-.007l31.55-18.309c3.097-1.797 3.103-6.008.01-7.812z"
                        }),
                        /*#__PURE__*/ (0, $gT9i9$jsx)("path", {
                            fill: "url(#364c63__e)",
                            d: "M11.47 25.435C12.174 10.908 25.24-.33 40.65.335 56.063 1 67.984 13.316 67.279 27.844c-.706 14.527-13.771 25.765-29.182 25.1S10.764 39.963 11.469 25.435"
                        }),
                        /*#__PURE__*/ (0, $gT9i9$jsx)("path", {
                            fill: "url(#364c63__f)",
                            d: "M-11.203 35.014c6.126-2.464 12.732-2.358 18.456-.203 11.454 4.322 18.118 1.65 22.74-9.122 2.303-5.406 6.798-9.984 12.931-12.45 12.262-4.924 26.417.419 31.633 11.943 5.215 11.525-.493 24.848-12.752 29.778-6.147 2.471-12.766 2.356-18.499.181-11.435-4.299-18.087-1.616-22.703 9.169-2.308 5.392-6.803 9.955-12.929 12.418-12.251 4.927-26.407-.415-31.62-11.933-5.212-11.518.49-24.854 12.743-29.781"
                        }),
                        /*#__PURE__*/ (0, $gT9i9$jsx)("ellipse", {
                            cx: 17.959,
                            cy: 16.93,
                            fill: "url(#364c63__g)",
                            rx: 17.959,
                            ry: 16.93,
                            transform: "matrix(1 -.00132 .00148 1 26.41 -2.916)"
                        })
                    ]
                })
            })
        ]
    });
const $77ef8f4aff4390e2$var$ForwardRef = /*#__PURE__*/ (0, $gT9i9$forwardRef)($77ef8f4aff4390e2$var$SvgComponent);
var $77ef8f4aff4390e2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($77ef8f4aff4390e2$var$ForwardRef);


export {$77ef8f4aff4390e2$export$2e2bcd8739ae039 as default};
