// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_translate_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2jNdz$jsxs, jsx as $2jNdz$jsx} from "react/jsx-runtime";
import {forwardRef as $2jNdz$forwardRef} from "react";





const $9f8b9149f9815edc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2jNdz$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2jNdz$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2jNdz$jsxs)("linearGradient", {
                        id: "4cb3fd__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2jNdz$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2jNdz$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2jNdz$jsxs)("linearGradient", {
                        id: "4cb3fd__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2jNdz$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2jNdz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2jNdz$jsxs)("linearGradient", {
                        id: "4cb3fd__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2jNdz$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2jNdz$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2jNdz$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2jNdz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2jNdz$jsxs)("radialGradient", {
                        id: "4cb3fd__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2jNdz$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2jNdz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2jNdz$jsxs)("radialGradient", {
                        id: "4cb3fd__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2jNdz$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2jNdz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2jNdz$jsx)("clipPath", {
                        id: "4cb3fd__b",
                        children: /*#__PURE__*/ (0, $2jNdz$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2jNdz$jsx)("clipPath", {
                id: "4cb3fd__a",
                children: /*#__PURE__*/ (0, $2jNdz$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M28.14 10.643h-8.1V9.125c0-.69-.56-1.25-1.25-1.25-.691 0-1.25.56-1.25 1.25v1.518H9.151c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25H23.49c-.546 3.227-2.353 5.858-4.623 7.853-.503.443-1.026.85-1.558 1.224-1.03-1.808-1.757-3.532-1.978-4.785-.12-.68-.768-1.134-1.448-1.014s-1.134.768-1.014 1.448c.29 1.644 1.17 3.672 2.303 5.65-1.465.765-2.9 1.272-4.103 1.514-.676.136-1.115.795-.978 1.472.136.677.795 1.115 1.472.979 1.486-.3 3.207-.916 4.934-1.837 1.67 2.48 3.724 4.848 5.563 6.037.58.374 1.354.208 1.729-.372.374-.58.208-1.354-.372-1.728-1.397-.903-3.195-2.898-4.784-5.24.646-.447 1.278-.937 1.884-1.47 2.68-2.355 4.93-5.615 5.503-9.73h2.12c.69 0 1.25-.56 1.25-1.25 0-.691-.56-1.25-1.25-1.25m-.261 23.174-2.35 5.545c-.27.636-1.004.933-1.64.663-.635-.269-.932-1.002-.663-1.638l7.23-17.057q.034-.084.082-.164.056-.096.128-.177c.12-.14.275-.257.458-.334q.242-.101.488-.1.246-.001.488.1c.182.077.337.193.458.334q.133.155.21.341l7.23 17.057c.269.636-.028 1.37-.664 1.639s-1.37-.028-1.639-.664l-2.35-5.545zm3.732-8.808-2.674 6.308h5.348z"
                })
            }),
            /*#__PURE__*/ (0, $2jNdz$jsx)("g", {
                clipPath: "url(#4cb3fd__a)",
                children: /*#__PURE__*/ (0, $2jNdz$jsxs)("g", {
                    clipPath: "url(#4cb3fd__b)",
                    children: [
                        /*#__PURE__*/ (0, $2jNdz$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $2jNdz$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $2jNdz$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#4cb3fd__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $2jNdz$jsx)("path", {
                            fill: "url(#4cb3fd__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $2jNdz$jsx)("path", {
                            fill: "url(#4cb3fd__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $2jNdz$jsx)("path", {
                            fill: "url(#4cb3fd__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $2jNdz$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#4cb3fd__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $9f8b9149f9815edc$var$ForwardRef = /*#__PURE__*/ (0, $2jNdz$forwardRef)($9f8b9149f9815edc$var$SvgComponent);
var $9f8b9149f9815edc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9f8b9149f9815edc$var$ForwardRef);


export {$9f8b9149f9815edc$export$2e2bcd8739ae039 as default};
