// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_illustration_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $1RBtU$jsxs, jsx as $1RBtU$jsx} from "react/jsx-runtime";
import {forwardRef as $1RBtU$forwardRef} from "react";





const $eab8eccb30da5f45$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1RBtU$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 97,
        height: 96,
        viewBox: "0 0 97 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1RBtU$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1RBtU$jsxs)("linearGradient", {
                        id: "38d72b__d",
                        x1: -23.183,
                        x2: 132.895,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1RBtU$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $1RBtU$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1RBtU$jsxs)("linearGradient", {
                        id: "38d72b__e",
                        x1: 102.07,
                        x2: 35.113,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1RBtU$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1RBtU$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1RBtU$jsxs)("linearGradient", {
                        id: "38d72b__f",
                        x1: 20.697,
                        x2: 139.393,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1RBtU$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1RBtU$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1RBtU$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1RBtU$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1RBtU$jsxs)("radialGradient", {
                        id: "38d72b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.75 29.473)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1RBtU$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $1RBtU$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1RBtU$jsxs)("radialGradient", {
                        id: "38d72b__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 37.002 54.345)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1RBtU$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1RBtU$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1RBtU$jsx)("clipPath", {
                        id: "38d72b__b",
                        children: /*#__PURE__*/ (0, $1RBtU$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1RBtU$jsx)("clipPath", {
                id: "38d72b__a",
                children: /*#__PURE__*/ (0, $1RBtU$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M39.561 25.134 52.798 38.37l-4.124 7.95c-.879 1.692-2.332 3.015-4.1 3.73l-21.483 8.697c-.468.19-.935.254-1.379.217l13.425-13.392q.027-.026.051-.053c.869.091 1.77-.196 2.435-.862 1.172-1.172 1.172-3.071 0-4.243-1.171-1.171-3.071-1.171-4.243 0-.595.596-.888 1.379-.878 2.159q-.1.076-.19.167L18.958 56.063c-.01-.396.06-.809.226-1.222l8.697-21.483c.715-1.768 2.038-3.221 3.73-4.1zm6.58-5.714-3.319 3.318 12.361 12.36 3.318-3.318c1.563-1.562 1.563-4.094 0-5.657l-6.703-6.703c-1.562-1.562-4.095-1.562-5.657 0m-4.48 49.013c-2.022 4.4-2.719 5.919-6.434 8.811 1.435.165 3.358.357 5.508.448 6.869.29 16.052-.45 19.018-6.38.836-1.67 1.189-3.167 1.203-4.495.013-1.15-.227-2.173-.625-3.077-.617-1.402-1.617-2.516-2.647-3.36-.788-.646-1.91-1.157-3.2-1.43l.001.001c-3.583-.76-8.448.321-10.95 5.483-.758 1.567-1.36 2.877-1.875 3.999m18.558-11.148c-1.18-.967-2.665-1.68-4.288-2.102L70.015 37.99c1.795-2.191 5.027-2.512 7.218-.717s2.513 5.026.718 7.218L63.816 61.745c-.944-1.947-2.324-3.417-3.597-4.46"
                })
            }),
            /*#__PURE__*/ (0, $1RBtU$jsx)("g", {
                clipPath: "url(#38d72b__a)",
                children: /*#__PURE__*/ (0, $1RBtU$jsxs)("g", {
                    clipPath: "url(#38d72b__b)",
                    children: [
                        /*#__PURE__*/ (0, $1RBtU$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $1RBtU$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M.941 0h96v96h-96z"
                        }),
                        /*#__PURE__*/ (0, $1RBtU$jsx)("circle", {
                            cx: 55.222,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#38d72b__c)",
                            transform: "rotate(-.08 55.222 3.723)"
                        }),
                        /*#__PURE__*/ (0, $1RBtU$jsx)("path", {
                            fill: "url(#38d72b__d)",
                            d: "M37.584-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.014 45.19c-7.25 4.427-7.261 14.873-.022 19.316l73.525 45.127c7.39 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.201-4.433 7.214-14.821.022-19.27z"
                        }),
                        /*#__PURE__*/ (0, $1RBtU$jsx)("path", {
                            fill: "url(#38d72b__e)",
                            d: "M14.778 45.579c1.64-35.84 32.025-63.563 67.864-61.923s63.564 32.025 61.923 67.865-32.025 63.563-67.865 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $1RBtU$jsx)("path", {
                            fill: "url(#38d72b__f)",
                            d: "M-37.954 69.21c14.247-6.078 29.611-5.818 42.922-.502 26.637 10.664 42.136 4.071 52.885-22.504 5.355-13.336 15.81-24.63 30.071-30.714 28.516-12.146 61.436 1.033 73.565 29.464s-1.147 61.299-29.656 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.062-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.492 12.155-61.412-1.024-73.534-29.44-12.123-28.415 1.14-61.314 29.632-73.47"
                        }),
                        /*#__PURE__*/ (0, $1RBtU$jsx)("circle", {
                            cx: 91.347,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#38d72b__g)",
                            transform: "rotate(-.08 91.347 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $eab8eccb30da5f45$var$ForwardRef = /*#__PURE__*/ (0, $1RBtU$forwardRef)($eab8eccb30da5f45$var$SvgComponent);
var $eab8eccb30da5f45$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($eab8eccb30da5f45$var$ForwardRef);


export {$eab8eccb30da5f45$export$2e2bcd8739ae039 as default};
