// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_illustration_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $5LbQ7$jsxs, jsx as $5LbQ7$jsx} from "react/jsx-runtime";
import {forwardRef as $5LbQ7$forwardRef} from "react";





const $b2f44dce5158d0a1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5LbQ7$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5LbQ7$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5LbQ7$jsxs)("linearGradient", {
                        id: "a5beb0__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5LbQ7$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $5LbQ7$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5LbQ7$jsxs)("linearGradient", {
                        id: "a5beb0__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5LbQ7$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5LbQ7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5LbQ7$jsxs)("linearGradient", {
                        id: "a5beb0__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5LbQ7$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5LbQ7$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5LbQ7$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5LbQ7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5LbQ7$jsxs)("radialGradient", {
                        id: "a5beb0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5LbQ7$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $5LbQ7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5LbQ7$jsxs)("radialGradient", {
                        id: "a5beb0__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5LbQ7$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5LbQ7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5LbQ7$jsx)("clipPath", {
                        id: "a5beb0__b",
                        children: /*#__PURE__*/ (0, $5LbQ7$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5LbQ7$jsx)("clipPath", {
                id: "a5beb0__a",
                children: /*#__PURE__*/ (0, $5LbQ7$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m18.928 12.677 6.73 6.73-1.941 3.742c-.44.847-1.166 1.508-2.05 1.866l-10.742 4.348c-.21.085-.42.12-.622.113l6.797-6.797c.458.008.919-.162 1.268-.512.684-.684.684-1.792 0-2.475-.683-.683-1.791-.683-2.475 0-.324.324-.494.744-.51 1.17l-6.317 6.315 4.254-10.509c.358-.884 1.02-1.61 1.866-2.05zm9.703 3.202-1.482 1.482-6.18-6.18 1.481-1.482c.781-.781 2.048-.781 2.829 0l3.352 3.352c.78.781.78 2.047 0 2.828m-8.42 18.35c-1.011 2.202-1.36 2.96-3.217 4.407 3.01.347 10.315.927 12.263-2.966.78-1.559.718-2.813.288-3.788-.309-.7-.808-1.257-1.323-1.679-.394-.322-.955-.577-1.598-.714-1.792-.38-4.225.16-5.476 2.742-.38.783-.68 1.438-.938 1.999m9.595-5.96c-.596-.488-1.323-.853-2.115-1.087l6.697-8.174c.897-1.096 2.513-1.256 3.608-.359 1.096.898 1.257 2.513.36 3.609l-6.722 8.205c-.506-.954-1.198-1.677-1.828-2.193"
                })
            }),
            /*#__PURE__*/ (0, $5LbQ7$jsx)("g", {
                clipPath: "url(#a5beb0__a)",
                children: /*#__PURE__*/ (0, $5LbQ7$jsxs)("g", {
                    clipPath: "url(#a5beb0__b)",
                    children: [
                        /*#__PURE__*/ (0, $5LbQ7$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $5LbQ7$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $5LbQ7$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#a5beb0__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $5LbQ7$jsx)("path", {
                            fill: "url(#a5beb0__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $5LbQ7$jsx)("path", {
                            fill: "url(#a5beb0__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $5LbQ7$jsx)("path", {
                            fill: "url(#a5beb0__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $5LbQ7$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#a5beb0__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $b2f44dce5158d0a1$var$ForwardRef = /*#__PURE__*/ (0, $5LbQ7$forwardRef)($b2f44dce5158d0a1$var$SvgComponent);
var $b2f44dce5158d0a1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b2f44dce5158d0a1$var$ForwardRef);


export {$b2f44dce5158d0a1$export$2e2bcd8739ae039 as default};
