// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_piggyBank_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $1sNSf$jsxs, jsx as $1sNSf$jsx} from "react/jsx-runtime";
import {forwardRef as $1sNSf$forwardRef} from "react";





const $07321964dba58067$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1sNSf$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1sNSf$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1sNSf$jsxs)("linearGradient", {
                        id: "f451cc__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1sNSf$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $1sNSf$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1sNSf$jsxs)("linearGradient", {
                        id: "f451cc__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1sNSf$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1sNSf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1sNSf$jsxs)("linearGradient", {
                        id: "f451cc__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1sNSf$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1sNSf$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1sNSf$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1sNSf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1sNSf$jsxs)("radialGradient", {
                        id: "f451cc__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1sNSf$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $1sNSf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1sNSf$jsxs)("radialGradient", {
                        id: "f451cc__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1sNSf$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1sNSf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1sNSf$jsx)("clipPath", {
                        id: "f451cc__b",
                        children: /*#__PURE__*/ (0, $1sNSf$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1sNSf$jsx)("clipPath", {
                id: "f451cc__a",
                children: /*#__PURE__*/ (0, $1sNSf$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M110.974 119.029c16.846-8.923 28.046-24.74 28.046-42.766q-.001-2.502-.283-4.942c2.257-.307 4.84-1.238 7.141-3.162 3.303-2.76 5.609-7.21 5.9-13.618.1-2.207-1.608-4.077-3.815-4.177s-4.077 1.607-4.177 3.814c-.207 4.568-1.742 6.76-3.038 7.843-1.29 1.078-2.76 1.408-3.676 1.419-6.722-21.654-30.063-37.63-57.825-37.63-11.412 0-22.077 2.7-31.148 7.384-5.187-5.565-13.203-7.325-19.631-7.379-2.784-.023-4.234 2.99-2.84 5.399l7.494 12.942.057-.042c-4.113 4.195-7.438 8.96-9.782 14.135v-.03l-11.205 6.53c-2.46 1.432-3.973 4.065-3.973 6.912v9.205c0 2.848 1.514 5.482 3.976 6.914l11.202 6.52v-.023c4.654 10.272 13.17 18.932 24.05 24.713q-.044-.006-.073-.005l4.14 11.273c.868 2.362 3.116 3.931 5.633 3.931h4.919c2.514 0 4.761-1.567 5.63-3.926l1.554-4.22-.155-.053c3.3.477 6.692.725 10.152.725 3.385 0 6.704-.238 9.935-.694l-.068.022 1.554 4.22c.87 2.359 3.117 3.926 5.63 3.926h4.92c2.516 0 4.765-1.569 5.632-3.931zm-22.969-75.02c-5.42-1.208-9.917-1.259-15.653.002-2.157.474-4.29-.89-4.765-3.049-.474-2.157.89-4.29 3.048-4.765 6.885-1.513 12.562-1.456 19.11.002 2.155.48 3.514 2.618 3.034 4.774s-2.617 3.515-4.774 3.035m-38.038 13.83c0 3.315-2.686 6-6 6s-6-2.685-6-6c0-3.313 2.686-6 6-6s6 2.687 6 6"
                })
            }),
            /*#__PURE__*/ (0, $1sNSf$jsx)("g", {
                clipPath: "url(#f451cc__a)",
                children: /*#__PURE__*/ (0, $1sNSf$jsxs)("g", {
                    clipPath: "url(#f451cc__b)",
                    children: [
                        /*#__PURE__*/ (0, $1sNSf$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $1sNSf$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $1sNSf$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#f451cc__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $1sNSf$jsx)("path", {
                            fill: "url(#f451cc__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $1sNSf$jsx)("path", {
                            fill: "url(#f451cc__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $1sNSf$jsx)("path", {
                            fill: "url(#f451cc__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $1sNSf$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#f451cc__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $07321964dba58067$var$ForwardRef = /*#__PURE__*/ (0, $1sNSf$forwardRef)($07321964dba58067$var$SvgComponent);
var $07321964dba58067$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($07321964dba58067$var$ForwardRef);


export {$07321964dba58067$export$2e2bcd8739ae039 as default};
