// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_AIGenerate_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $fx3Af$jsxs, jsx as $fx3Af$jsx} from "react/jsx-runtime";
import {forwardRef as $fx3Af$forwardRef} from "react";





const $a419eacbac47fa8d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fx3Af$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fx3Af$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fx3Af$jsxs)("linearGradient", {
                        id: "21bacb__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fx3Af$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $fx3Af$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fx3Af$jsxs)("linearGradient", {
                        id: "21bacb__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fx3Af$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fx3Af$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fx3Af$jsxs)("linearGradient", {
                        id: "21bacb__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fx3Af$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fx3Af$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fx3Af$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fx3Af$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fx3Af$jsxs)("radialGradient", {
                        id: "21bacb__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fx3Af$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $fx3Af$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fx3Af$jsxs)("radialGradient", {
                        id: "21bacb__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fx3Af$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fx3Af$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fx3Af$jsx)("clipPath", {
                        id: "21bacb__b",
                        children: /*#__PURE__*/ (0, $fx3Af$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fx3Af$jsx)("clipPath", {
                id: "21bacb__a",
                children: /*#__PURE__*/ (0, $fx3Af$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m126.787 55.017 9.316 2.186c4.137.97 6.999-4.204 4.126-7.455l-6.47-7.326c-.961-1.088-1.35-2.604-1.037-4.047l2.096-9.72c.929-4.318-4.029-7.304-7.145-4.306l-7.021 6.751c-1.043 1.003-2.495 1.409-3.878 1.082l-9.316-2.187c-4.137-.969-6.999 4.204-4.126 7.456l6.47 7.326c.374.423.661.911.854 1.434q.176.478.246.985c.073.535.054 1.086-.063 1.627l-2.096 9.721c-.116.54-.14 1.059-.087 1.548q.06.551.242 1.046c.997 2.744 4.556 4.054 6.99 1.712l7.021-6.751c1.042-1.003 2.495-1.409 3.878-1.082m-32.4-18.239h-44.43c-8.836 0-16 7.164-16 16v60.087c0 8.837 7.163 16 16 16h60.087c8.837 0 16-7.163 16-16V64.183l-4.609 4.432-.002.001c-4.438 4.269-10.487 4.253-14.763 1.676-4.221-2.543-6.984-7.687-5.748-13.43l1.742-8.082-5.327-6.032-.002-.002c-1.616-1.83-2.557-3.888-2.948-5.968m2.75 49.805-6.848-1.543c-.907-.203-1.847.05-2.529.675l-5.158 4.759c-2.032 1.875-5.265.005-4.657-2.686l1.543-6.848c.203-.906-.05-1.847-.675-2.528l-4.759-5.158c-1.875-2.033-.005-5.265 2.686-4.657l6.848 1.543c.907.203 1.847-.05 2.528-.676l5.158-4.758c2.033-1.875 5.265-.006 4.657 2.686l-1.543 6.847c-.203.907.05 1.847.676 2.529l4.758 5.158c1.875 2.032.006 5.265-2.686 4.656"
                })
            }),
            /*#__PURE__*/ (0, $fx3Af$jsx)("g", {
                clipPath: "url(#21bacb__a)",
                children: /*#__PURE__*/ (0, $fx3Af$jsxs)("g", {
                    clipPath: "url(#21bacb__b)",
                    children: [
                        /*#__PURE__*/ (0, $fx3Af$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $fx3Af$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $fx3Af$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#21bacb__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $fx3Af$jsx)("path", {
                            fill: "url(#21bacb__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $fx3Af$jsx)("path", {
                            fill: "url(#21bacb__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $fx3Af$jsx)("path", {
                            fill: "url(#21bacb__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $fx3Af$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#21bacb__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $a419eacbac47fa8d$var$ForwardRef = /*#__PURE__*/ (0, $fx3Af$forwardRef)($a419eacbac47fa8d$var$SvgComponent);
var $a419eacbac47fa8d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a419eacbac47fa8d$var$ForwardRef);


export {$a419eacbac47fa8d$export$2e2bcd8739ae039 as default};
