// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_infiniteLooping_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $7SN8C$reactjsxruntime = require("react/jsx-runtime");
var $7SN8C$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $384124cbfa72b84f$export$2e2bcd8739ae039);




const $384124cbfa72b84f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d934f9__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d934f9__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d934f9__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.618,
                        y2: 33.126,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d934f9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d934f9__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("clipPath", {
                        id: "d934f9__b",
                        children: /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("clipPath", {
                id: "d934f9__a",
                children: /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M43.099 43.606c1.105-.003 1.998-.9 1.995-2.005l-.034-15.012c-.003-1.104-.9-1.998-2.005-1.995s-1.998.9-1.995 2.004l.023 10.205c-3.959-4.595-9.825-7.508-16.37-7.508-11.932 0-21.604 9.672-21.604 21.603 0 11.93 9.672 21.602 21.603 21.602 4.95 0 9.515-1.667 13.159-4.469q.078-.06.149-.126l.082-.061c2.688-2.11 5.206-5.189 7.186-8.026 1.987-2.849 3.539-5.61 4.26-7.2.573-1.266 1.992-4.273 3.872-7.434 1.618-2.72 3.48-5.388 5.345-7.193q.077-.058.15-.125c3.128-2.842 7.277-4.571 11.835-4.571 9.722 0 17.603 7.88 17.603 17.603 0 9.721-7.881 17.602-17.603 17.602-5.945 0-11.203-2.945-14.393-7.465-.637-.903-1.885-1.118-2.787-.481-.903.637-1.118 1.885-.481 2.787 3.907 5.537 10.361 9.16 17.66 9.16 11.932 0 21.604-9.673 21.604-21.603S82.68 29.295 70.75 29.295c-4.954 0-9.522 1.67-13.167 4.475q-.144.08-.278.184c-2.808 2.214-5.372 5.905-7.323 9.185-1.984 3.337-3.472 6.49-4.078 7.828-.597 1.317-2.015 3.864-3.897 6.562-1.89 2.708-4.142 5.415-6.375 7.169q-.07.054-.133.114l-.067.049c-2.968 2.283-6.683 3.64-10.72 3.64-9.722 0-17.603-7.882-17.603-17.603s7.881-17.603 17.603-17.603c5.428 0 10.282 2.455 13.514 6.322l-10.148.023c-1.105.003-1.998.9-1.995 2.005s.9 1.998 2.004 1.995z"
                })
            }),
            /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("g", {
                clipPath: "url(#d934f9__a)",
                children: /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#d934f9__b)",
                    children: [
                        /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#d934f9__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("path", {
                            fill: "url(#d934f9__d)",
                            d: "M36.643-16.957c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.426-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.432 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("path", {
                            fill: "url(#d934f9__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("path", {
                            fill: "url(#d934f9__f)",
                            d: "M-38.895 69.21c14.246-6.077 29.61-5.817 42.922-.5 26.636 10.663 42.135 4.07 52.885-22.505 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.315 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $7SN8C$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#d934f9__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $384124cbfa72b84f$var$ForwardRef = /*#__PURE__*/ (0, $7SN8C$react.forwardRef)($384124cbfa72b84f$var$SvgComponent);
var $384124cbfa72b84f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($384124cbfa72b84f$var$ForwardRef);
