// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_information_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $4uccY$jsxs, jsx as $4uccY$jsx} from "react/jsx-runtime";
import {forwardRef as $4uccY$forwardRef} from "react";





const $13173fc9f3e079c2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4uccY$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4uccY$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4uccY$jsxs)("linearGradient", {
                        id: "0657d7__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 75.604,
                        y2: -0.88,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4uccY$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4uccY$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4uccY$jsxs)("linearGradient", {
                        id: "0657d7__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 25.206,
                        y2: 81.385,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4uccY$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4uccY$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4uccY$jsxs)("linearGradient", {
                        id: "0657d7__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 85.836,
                        y2: 32.075,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4uccY$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4uccY$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4uccY$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4uccY$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4uccY$jsxs)("radialGradient", {
                        id: "0657d7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4uccY$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4uccY$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4uccY$jsxs)("radialGradient", {
                        id: "0657d7__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4uccY$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4uccY$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4uccY$jsx)("clipPath", {
                        id: "0657d7__b",
                        children: /*#__PURE__*/ (0, $4uccY$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 12.12,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4uccY$jsx)("clipPath", {
                id: "0657d7__a",
                children: /*#__PURE__*/ (0, $4uccY$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M81.2 47.6c0 18.557-15.043 33.6-33.6 33.6S14 66.157 14 47.6 29.043 14 47.6 14s33.6 15.043 33.6 33.6m-38.1-8.1c-1.325 0-2.4 1.075-2.4 2.4s1.075 2.4 2.4 2.4h3v13.8h-3c-1.325 0-2.4 1.075-2.4 2.4s1.075 2.4 2.4 2.4h10.8c1.325 0 2.4-1.075 2.4-2.4s-1.075-2.4-2.4-2.4h-3V41.9c0-1.325-1.075-2.4-2.4-2.4zm4.8-3c1.657 0 3-1.343 3-3s-1.343-3-3-3-3 1.343-3 3 1.343 3 3 3"
                })
            }),
            /*#__PURE__*/ (0, $4uccY$jsx)("g", {
                clipPath: "url(#0657d7__a)",
                children: /*#__PURE__*/ (0, $4uccY$jsxs)("g", {
                    clipPath: "url(#0657d7__b)",
                    children: [
                        /*#__PURE__*/ (0, $4uccY$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 12.12,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $4uccY$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 12.12H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $4uccY$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#0657d7__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -15.987)"
                        }),
                        /*#__PURE__*/ (0, $4uccY$jsx)("path", {
                            fill: "url(#0657d7__d)",
                            d: "M39.804-.429c-5.787-3.376-13.12-3.397-18.93-.053l-58.1 33.44c-5.69 3.276-5.7 11.007-.018 14.294l57.718 33.394c5.8 3.356 13.133 3.352 18.928-.012L97.001 47.21c5.652-3.28 5.663-10.968.017-14.26z"
                        }),
                        /*#__PURE__*/ (0, $4uccY$jsx)("path", {
                            fill: "url(#0657d7__e)",
                            d: "M21.899 45.849C23.187 19.327 47.039-1.188 75.172.026s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 72.37 21.898 45.848"
                        }),
                        /*#__PURE__*/ (0, $4uccY$jsx)("path", {
                            fill: "url(#0657d7__f)",
                            d: "M-19.495 63.335C-8.31 58.838 3.75 59.03 14.2 62.965c20.91 7.89 33.077 3.011 41.515-16.654 4.203-9.868 12.41-18.226 23.606-22.728 22.384-8.989 48.226.764 57.748 21.803s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.951-41.447 16.738-4.215 9.845-12.42 18.174-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $4uccY$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#0657d7__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -5.908)"
                        })
                    ]
                })
            })
        ]
    });
const $13173fc9f3e079c2$var$ForwardRef = /*#__PURE__*/ (0, $4uccY$forwardRef)($13173fc9f3e079c2$var$SvgComponent);
var $13173fc9f3e079c2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($13173fc9f3e079c2$var$ForwardRef);


export {$13173fc9f3e079c2$export$2e2bcd8739ae039 as default};
