// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_sparkles_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $htGsx$reactjsxruntime = require("react/jsx-runtime");
var $htGsx$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $aa55d13678439f2e$export$2e2bcd8739ae039);




const $aa55d13678439f2e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsxs)("linearGradient", {
                        id: "afd80c__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsxs)("linearGradient", {
                        id: "afd80c__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsxs)("linearGradient", {
                        id: "afd80c__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsxs)("radialGradient", {
                        id: "afd80c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsxs)("radialGradient", {
                        id: "afd80c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("clipPath", {
                        id: "afd80c__b",
                        children: /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("clipPath", {
                id: "afd80c__a",
                children: /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m75.722 50.646-10.35-2.328c-1.536-.348-3.15.084-4.308 1.152l-7.8 7.188c-3.462 3.192-8.97.012-7.938-4.584l2.328-10.35c.348-1.536-.084-3.15-1.152-4.308l-7.188-7.8c-3.192-3.462-.012-8.97 4.584-7.938l10.35 2.328c1.536.348 3.15-.084 4.308-1.152l7.8-7.188c3.462-3.192 8.97-.012 7.938 4.584L71.966 30.6c-.348 1.536.084 3.15 1.152 4.308l7.188 7.8c3.192 3.462.012 8.97-4.584 7.938m-37.163 27.27-7.296-1.644c-.966-.216-1.969.054-2.695.72l-5.495 5.07c-2.166 1.998-5.61.006-4.963-2.862l1.645-7.296c.216-.966-.054-1.968-.72-2.694l-5.07-5.496c-1.998-2.166-.006-5.61 2.861-4.962l7.296 1.644c.966.216 1.968-.054 2.695-.72l5.496-5.07c2.165-1.998 5.61-.006 4.962 2.862l-1.645 7.296c-.216.966.054 1.968.72 2.694l5.07 5.496c1.999 2.166.006 5.61-2.861 4.962"
                })
            }),
            /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("g", {
                clipPath: "url(#afd80c__a)",
                children: /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#afd80c__b)",
                    children: [
                        /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#afd80c__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("path", {
                            fill: "url(#afd80c__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("path", {
                            fill: "url(#afd80c__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("path", {
                            fill: "url(#afd80c__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $htGsx$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#afd80c__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $aa55d13678439f2e$var$ForwardRef = /*#__PURE__*/ (0, $htGsx$react.forwardRef)($aa55d13678439f2e$var$SvgComponent);
var $aa55d13678439f2e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($aa55d13678439f2e$var$ForwardRef);
