// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_lighten_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $eVjFT$reactjsxruntime = require("react/jsx-runtime");
var $eVjFT$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9b7511e8a92eb6c0$export$2e2bcd8739ae039);




const $9b7511e8a92eb6c0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d9d24e__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.894,
                        y2: -4.584,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d9d24e__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.841,
                        y2: 44.839,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d9d24e__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.808,
                        y2: 16.562,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d9d24e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d9d24e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("clipPath", {
                        id: "d9d24e__b",
                        children: /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: -0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("clipPath", {
                id: "d9d24e__a",
                children: /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M24 6.231c.69 0 1.25.56 1.25 1.25v2.341c0 .69-.56 1.25-1.25 1.25-.691 0-1.25-.56-1.25-1.25v-2.34c0-.69.559-1.25 1.25-1.25m8.342 17.77c0 4.608-3.735 8.343-8.343 8.343s-8.344-3.735-8.344-8.343 3.736-8.344 8.344-8.344 8.343 3.736 8.343 8.344M25.25 38.179c0-.69-.56-1.25-1.25-1.25s-1.25.56-1.25 1.25v2.34c0 .69.56 1.25 1.25 1.25s1.25-.56 1.25-1.25zM41.77 24c0 .69-.56 1.25-1.25 1.25h-2.34c-.691 0-1.25-.56-1.25-1.25s.559-1.25 1.25-1.25h2.34c.69 0 1.25.56 1.25 1.25M9.82 25.251c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.34c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25zm26.743-13.817c.488.489.488 1.28 0 1.768l-1.655 1.655c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.767l1.655-1.655c.489-.489 1.28-.489 1.768 0M14.857 34.91c.488-.488.488-1.28 0-1.768s-1.28-.488-1.768 0l-1.655 1.656c-.488.488-.488 1.28 0 1.767.489.488 1.28.488 1.768 0zm-3.423-23.474c-.489.488-.489 1.28 0 1.767l1.655 1.655c.488.488 1.28.488 1.767 0 .488-.488.488-1.28 0-1.767l-1.655-1.655c-.488-.489-1.28-.489-1.767 0M33.14 34.909c-.488-.488-.488-1.28 0-1.768.489-.488 1.28-.488 1.768 0l1.655 1.656c.488.488.488 1.28 0 1.767-.488.488-1.28.488-1.768 0z"
                })
            }),
            /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("g", {
                clipPath: "url(#d9d24e__a)",
                children: /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#d9d24e__b)",
                    children: [
                        /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: -0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0-.001h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.861,
                            r: 20.883,
                            fill: "url(#d9d24e__c)",
                            transform: "rotate(-.08 27.142 1.861)"
                        }),
                        /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("path", {
                            fill: "url(#d9d24e__d)",
                            d: "M18.321-8.48c-3.686-2.28-8.357-2.295-12.057-.036L-30.743 14.08c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.3c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("path", {
                            fill: "url(#d9d24e__e)",
                            d: "M6.916 22.788C7.736 4.868 22.93-8.993 40.85-8.172S72.63 7.84 71.809 25.758c-.82 17.92-16.012 31.782-33.931 30.962-17.92-.82-31.782-16.013-30.962-33.933"
                        }),
                        /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("path", {
                            fill: "url(#d9d24e__f)",
                            d: "M-19.448 34.604c7.124-3.039 14.806-2.908 21.462-.25C15.332 39.686 23.08 36.389 28.456 23.1c2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $eVjFT$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.671,
                            r: 20.883,
                            fill: "url(#d9d24e__g)",
                            transform: "rotate(-.08 45.2 8.67)"
                        })
                    ]
                })
            })
        ]
    });
const $9b7511e8a92eb6c0$var$ForwardRef = /*#__PURE__*/ (0, $eVjFT$react.forwardRef)($9b7511e8a92eb6c0$var$SvgComponent);
var $9b7511e8a92eb6c0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9b7511e8a92eb6c0$var$ForwardRef);
