// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_mention_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $h0gHm$jsxs, jsx as $h0gHm$jsx} from "react/jsx-runtime";
import {forwardRef as $h0gHm$forwardRef} from "react";





const $0e22b694626d3741$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $h0gHm$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $h0gHm$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $h0gHm$jsxs)("linearGradient", {
                        id: "0bfdc1__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h0gHm$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $h0gHm$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h0gHm$jsxs)("linearGradient", {
                        id: "0bfdc1__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h0gHm$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h0gHm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h0gHm$jsxs)("linearGradient", {
                        id: "0bfdc1__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h0gHm$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $h0gHm$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $h0gHm$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $h0gHm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h0gHm$jsxs)("radialGradient", {
                        id: "0bfdc1__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h0gHm$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $h0gHm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h0gHm$jsxs)("radialGradient", {
                        id: "0bfdc1__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h0gHm$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h0gHm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h0gHm$jsx)("clipPath", {
                        id: "0bfdc1__b",
                        children: /*#__PURE__*/ (0, $h0gHm$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $h0gHm$jsx)("clipPath", {
                id: "0bfdc1__a",
                children: /*#__PURE__*/ (0, $h0gHm$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M38.089 14.782c9.7-3.18 20.098-2.114 26.84.927 13.817 6.234 18.38 19.622 16.359 31.11-2.007 11.406-10.7 21.583-23.79 20.677-12.196-.844-18.633-5.455-21.228-11.381-2.51-5.733-1.118-12.05.921-15.817 5.274-9.74 14.283-11.233 19.88-11.055 3.415.109 5.5 3.2 5.223 6.203L59.711 63.54c9.385-.334 15.974-7.962 17.637-17.413 1.761-10.004-2.19-21.413-14.065-26.77-5.83-2.631-15.197-3.643-23.948-.773-8.631 2.83-16.676 9.433-20.144 22.162-2.757 10.12.47 19.802 6.65 26.96 6.21 7.194 15.231 11.635 23.707 11.39 1.104-.031 2.025.838 2.057 1.942s-.838 2.025-1.942 2.057c-9.881.285-19.996-4.836-26.85-12.774-6.884-7.974-10.65-18.995-7.481-30.626 3.846-14.115 12.938-21.692 22.757-24.912m17.625 48.537 2.597-28.239c.103-1.117-.644-1.816-1.366-1.839-4.838-.154-11.983 1.107-16.236 8.962-1.638 3.025-2.652 8.02-.775 12.307 1.68 3.837 5.952 7.694 15.78 8.809"
                })
            }),
            /*#__PURE__*/ (0, $h0gHm$jsx)("g", {
                clipPath: "url(#0bfdc1__a)",
                children: /*#__PURE__*/ (0, $h0gHm$jsxs)("g", {
                    clipPath: "url(#0bfdc1__b)",
                    children: [
                        /*#__PURE__*/ (0, $h0gHm$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $h0gHm$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $h0gHm$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#0bfdc1__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $h0gHm$jsx)("path", {
                            fill: "url(#0bfdc1__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $h0gHm$jsx)("path", {
                            fill: "url(#0bfdc1__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $h0gHm$jsx)("path", {
                            fill: "url(#0bfdc1__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $h0gHm$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#0bfdc1__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $0e22b694626d3741$var$ForwardRef = /*#__PURE__*/ (0, $h0gHm$forwardRef)($0e22b694626d3741$var$SvgComponent);
var $0e22b694626d3741$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0e22b694626d3741$var$ForwardRef);


export {$0e22b694626d3741$export$2e2bcd8739ae039 as default};
