// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_channel_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $c1wd8$reactjsxruntime = require("react/jsx-runtime");
var $c1wd8$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $28f3ddc1b0d879db$export$2e2bcd8739ae039);




const $28f3ddc1b0d879db$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0e6198__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0e6198__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0e6198__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsxs)("radialGradient", {
                        id: "0e6198__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsxs)("radialGradient", {
                        id: "0e6198__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("clipPath", {
                        id: "0e6198__b",
                        children: /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("clipPath", {
                id: "0e6198__a",
                children: /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M33.62 28.094q.283 0 .558-.031l5.338 9.245c-2.654 2.108-4.504 5.183-5.02 8.692H23.822c-.771-1.766-2.533-3-4.584-3-2.761 0-5 2.239-5 5s2.239 5 5 5c2.05 0 3.813-1.234 4.584-3h10.675c.515 3.509 2.366 6.584 5.02 8.691l-5.339 9.246q-.275-.03-.559-.03c-2.761 0-5 2.238-5 5 0 2.76 2.239 5 5 5s5-2.24 5-5c0-1.113-.363-2.14-.977-2.97l5.337-9.243c1.553.615 3.247.953 5.02.953s3.466-.338 5.02-.953l5.337 9.243c-.614.83-.977 1.858-.977 2.97 0 2.76 2.239 5 5 5s5-2.24 5-5c0-2.762-2.238-5-5-5q-.283 0-.56.03l-5.337-9.245c2.653-2.108 4.504-5.183 5.02-8.692h10.674c.771 1.766 2.533 3 4.584 3 2.761 0 5-2.239 5-5s-2.239-5-5-5c-2.05 0-3.813 1.234-4.584 3H61.5c-.515-3.51-2.365-6.584-5.019-8.692l5.338-9.245q.276.03.56.03c2.76 0 5-2.238 5-5 0-2.76-2.24-5-5-5-2.762 0-5 2.24-5 5 0 1.112.362 2.14.976 2.97l-5.336 9.242c-1.554-.615-3.248-.953-5.021-.953s-3.467.338-5.02.953l-5.337-9.242c.614-.83.977-1.857.977-2.97 0-2.76-2.238-5-5-5s-5 2.24-5 5c0 2.762 2.239 5 5 5"
                })
            }),
            /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("g", {
                clipPath: "url(#0e6198__a)",
                children: /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#0e6198__b)",
                    children: [
                        /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#0e6198__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("path", {
                            fill: "url(#0e6198__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("path", {
                            fill: "url(#0e6198__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("path", {
                            fill: "url(#0e6198__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $c1wd8$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#0e6198__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $28f3ddc1b0d879db$var$ForwardRef = /*#__PURE__*/ (0, $c1wd8$react.forwardRef)($28f3ddc1b0d879db$var$SvgComponent);
var $28f3ddc1b0d879db$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($28f3ddc1b0d879db$var$ForwardRef);
