// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_megaphonePromote2_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $48yk4$reactjsxruntime = require("react/jsx-runtime");
var $48yk4$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $843269319f8d3c7d$export$2e2bcd8739ae039);




const $843269319f8d3c7d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "93d563__d",
                        x1: -10.152,
                        x2: 113.452,
                        y1: 78.607,
                        y2: -1.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "93d563__e",
                        x1: 92.397,
                        x2: 41.028,
                        y1: 26.041,
                        y2: 84.636,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "93d563__f",
                        x1: 25.777,
                        x2: 120.136,
                        y1: 89.279,
                        y2: 33.206,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsxs)("radialGradient", {
                        id: "93d563__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsxs)("radialGradient", {
                        id: "93d563__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("clipPath", {
                        id: "93d563__b",
                        children: /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 9.602,
                            y: 12.393,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("clipPath", {
                id: "93d563__a",
                children: /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m41.542 28.812 23.136-7.026.006-.006c3.45-1.05 6.93 1.53 6.93 5.136v26.316c0 3.606-3.48 6.186-6.93 5.136L41.548 51.33h-.024l6.264 14.55c1.344 3.108-.084 6.714-3.186 8.058-3.108 1.344-6.714-.084-8.058-3.186l-8.442-19.416H24.91c-6.216 0-11.262-5.046-11.262-11.262s5.04-11.262 11.262-11.262zM81.85 49.416l3.45 1.092c1.26.402 1.962 1.746 1.56 3.012-.324 1.02-1.266 1.674-2.286 1.674-.24 0-.486-.036-.726-.114l-3.45-1.092c-1.26-.402-1.962-1.746-1.56-3.012.402-1.26 1.746-1.962 3.012-1.56m-.725-18.888c.24 0 .486-.036.726-.114l3.45-1.092c1.266-.402 1.962-1.752 1.56-3.012-.402-1.266-1.746-1.962-3.012-1.56l-3.45 1.092c-1.266.402-1.962 1.752-1.56 3.012.324 1.02 1.266 1.674 2.286 1.674m3.612 11.79h-3.612c-1.326 0-2.4-1.074-2.4-2.4s1.074-2.4 2.4-2.4h3.612c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4"
                })
            }),
            /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("g", {
                clipPath: "url(#93d563__a)",
                children: /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#93d563__b)",
                    children: [
                        /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 9.602,
                            y: 12.393,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M9.602 12.393h78.6v74.094h-78.6z"
                        }),
                        /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#93d563__c)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 19.8 -16.924)"
                        }),
                        /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("path", {
                            fill: "url(#93d563__d)",
                            d: "M39.6-.696c-6.035-3.52-13.684-3.542-19.742-.055l-60.6 34.879c-5.935 3.416-5.945 11.48-.018 14.908l60.2 34.83c6.049 3.5 13.697 3.495 19.742-.013l60.075-34.862c5.895-3.422 5.906-11.44.018-14.874z"
                        }),
                        /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("path", {
                            fill: "url(#93d563__e)",
                            d: "M20.927 47.571C22.27 19.91 47.147-1.488 76.491-.222c29.344 1.267 52.043 24.718 50.699 52.38-1.343 27.661-26.22 49.059-55.564 47.792S19.584 75.233 20.927 47.571"
                        }),
                        /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("path", {
                            fill: "url(#93d563__f)",
                            d: "M-22.243 65.81c11.664-4.69 24.244-4.49 35.143-.387 21.808 8.23 34.498 3.142 43.3-17.37 4.384-10.292 12.943-19.008 24.62-23.705 23.347-9.374 50.3.798 60.231 22.742 9.931 21.943-.939 47.311-24.28 56.698-11.704 4.707-24.308 4.486-35.223.346-21.773-8.186-34.439-3.078-43.23 17.458-4.394 10.268-12.952 18.955-24.616 23.645-23.328 9.382-50.281-.79-60.206-22.721s.933-47.324 24.261-56.706"
                        }),
                        /*#__PURE__*/ (0, $48yk4$reactjsxruntime.jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#93d563__g)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 49.379 -6.411)"
                        })
                    ]
                })
            })
        ]
    });
const $843269319f8d3c7d$var$ForwardRef = /*#__PURE__*/ (0, $48yk4$react.forwardRef)($843269319f8d3c7d$var$SvgComponent);
var $843269319f8d3c7d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($843269319f8d3c7d$var$ForwardRef);
