// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_cardTapPayment_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4xMcS$reactjsxruntime = require("react/jsx-runtime");
var $4xMcS$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5c8383f85212f806$export$2e2bcd8739ae039);




const $5c8383f85212f806$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "cd36c4__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "cd36c4__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "cd36c4__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsxs)("radialGradient", {
                        id: "cd36c4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsxs)("radialGradient", {
                        id: "cd36c4__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("clipPath", {
                        id: "cd36c4__b",
                        children: /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("clipPath", {
                id: "cd36c4__a",
                children: /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M34.79 7.126c.083-.685.707-1.173 1.392-1.09 3.351.41 6.481 2.335 8.301 5.488 1.836 3.179 1.925 6.886.57 10.009-.276.633-1.012.924-1.645.649s-.924-1.011-.65-1.644c1.054-2.428.983-5.3-.44-7.764-1.41-2.444-3.834-3.938-6.439-4.256-.685-.083-1.173-.707-1.09-1.392m-1.133 4.606c.084-.685.707-1.172 1.393-1.089 2.065.252 3.997 1.44 5.12 3.385 1.132 1.96 1.186 4.248.35 6.173-.274.633-1.01.923-1.644.649-.633-.275-.924-1.012-.649-1.645.534-1.23.497-2.682-.222-3.927-.714-1.236-1.939-1.992-3.258-2.153-.686-.084-1.173-.707-1.09-1.393M5.27 22.2c-1.913 1.104-2.568 3.55-1.464 5.464l.485.84 26.985-15.56-.493-.855c-1.105-1.913-3.551-2.569-5.465-1.464zm27.256-7.09L5.541 30.67l4.648 8.05c1.105 1.913 3.551 2.569 5.464 1.464l20.049-11.575c1.913-1.105 2.569-3.55 1.464-5.464zM10.58 31.923c-.346-.598-.14-1.362.457-1.707l9.468-5.467c.598-.346 1.363-.14 1.708.457.345.598.14 1.362-.458 1.708l-9.468 5.467c-.598.345-1.362.14-1.707-.458m2.88 2.49c-.598.345-.802 1.11-.457 1.708s1.11.802 1.707.457l4.84-2.794c.597-.345.802-1.11.457-1.708s-1.11-.802-1.708-.457zm10.922-10.458c-.345-.598-.14-1.363.458-1.708l4.537-2.62c.598-.345 1.362-.14 1.707.458.346.598.14 1.362-.457 1.707l-4.537 2.62c-.598.345-1.362.14-1.708-.457"
                })
            }),
            /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("g", {
                clipPath: "url(#cd36c4__a)",
                children: /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#cd36c4__b)",
                    children: [
                        /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#cd36c4__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("path", {
                            fill: "url(#cd36c4__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("path", {
                            fill: "url(#cd36c4__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("path", {
                            fill: "url(#cd36c4__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $4xMcS$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#cd36c4__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $5c8383f85212f806$var$ForwardRef = /*#__PURE__*/ (0, $4xMcS$react.forwardRef)($5c8383f85212f806$var$SvgComponent);
var $5c8383f85212f806$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5c8383f85212f806$var$ForwardRef);
