// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_userGroup_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $8WlKq$jsxs, jsx as $8WlKq$jsx} from "react/jsx-runtime";
import {forwardRef as $8WlKq$forwardRef} from "react";





const $7c2b67c1cd5c6faf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8WlKq$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8WlKq$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8WlKq$jsxs)("linearGradient", {
                        id: "6d31f8__d",
                        x1: -19.921,
                        x2: 186.087,
                        y1: 131.011,
                        y2: -1.943,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8WlKq$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8WlKq$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8WlKq$jsxs)("linearGradient", {
                        id: "6d31f8__e",
                        x1: 150.995,
                        x2: 65.38,
                        y1: 43.401,
                        y2: 141.06,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8WlKq$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8WlKq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8WlKq$jsxs)("linearGradient", {
                        id: "6d31f8__f",
                        x1: 39.959,
                        x2: 197.223,
                        y1: 148.798,
                        y2: 55.343,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8WlKq$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8WlKq$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8WlKq$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8WlKq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8WlKq$jsxs)("radialGradient", {
                        id: "6d31f8__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8WlKq$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8WlKq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8WlKq$jsxs)("radialGradient", {
                        id: "6d31f8__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8WlKq$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8WlKq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8WlKq$jsx)("clipPath", {
                        id: "6d31f8__b",
                        children: /*#__PURE__*/ (0, $8WlKq$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8WlKq$jsx)("clipPath", {
                id: "6d31f8__a",
                children: /*#__PURE__*/ (0, $8WlKq$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M116.85 50.45c0 13.531-10.298 24.5-23 24.5s-23-10.969-23-24.5c0-13.53 10.297-24.5 23-24.5s23 10.97 23 24.5M72.59 74.08c-2.04.63-4.2.98-6.44.98-12.7 0-23-10.97-23-24.5s10.3-24.5 23-24.5c2.17 0 4.28.33 6.27.93-5.9 5.92-9.58 14.16-9.58 23.46s3.75 17.7 9.75 23.63m-32.52 48.39c3.25-15.09 12.65-27.7 25.19-35.4-22.04.42-40.58 16.32-45.07 37.19-1.08 5.02 2.67 9.78 7.81 9.78h13.94c-2.04-3.46-2.74-7.56-1.87-11.57m53.77-35.52c22.43 0 41.41 16.06 45.96 37.21 1.08 5.02-2.67 9.78-7.81 9.78h-76.3c-5.14 0-8.89-4.76-7.81-9.78 4.55-21.15 23.53-37.21 45.96-37.21"
                })
            }),
            /*#__PURE__*/ (0, $8WlKq$jsx)("g", {
                clipPath: "url(#6d31f8__a)",
                children: /*#__PURE__*/ (0, $8WlKq$jsxs)("g", {
                    clipPath: "url(#6d31f8__b)",
                    children: [
                        /*#__PURE__*/ (0, $8WlKq$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $8WlKq$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M13 20.654h131v123.491H13z"
                        }),
                        /*#__PURE__*/ (0, $8WlKq$jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#6d31f8__c)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 30 -28.206)"
                        }),
                        /*#__PURE__*/ (0, $8WlKq$jsx)("path", {
                            fill: "url(#6d31f8__d)",
                            d: "M63.002-1.16C52.94-7.027 40.194-7.063 30.096-1.251l-101 58.132c-9.891 5.693-9.907 19.132-.03 24.847L29.4 139.776c10.082 5.833 22.83 5.826 32.904-.021l100.125-58.103c9.826-5.703 9.844-19.067.03-24.79z"
                        }),
                        /*#__PURE__*/ (0, $8WlKq$jsx)("path", {
                            fill: "url(#6d31f8__e)",
                            d: "M31.878 79.285C34.118 33.183 75.58-2.48 124.485-.369c48.907 2.11 86.738 41.195 84.499 87.298s-43.701 81.765-92.607 79.655c-48.906-2.111-86.738-41.196-84.499-87.299"
                        }),
                        /*#__PURE__*/ (0, $8WlKq$jsx)("path", {
                            fill: "url(#6d31f8__f)",
                            d: "M-40.075 109.683c19.44-7.818 40.406-7.483 58.57-.645 36.349 13.718 57.498 5.237 72.167-28.949 7.307-17.154 21.572-31.682 41.035-39.509 38.911-15.624 83.833 1.33 100.384 37.902s-1.565 78.853-40.467 94.497c-19.506 7.845-40.512 7.477-58.705.577-36.288-13.644-57.398-5.13-72.048 29.096-7.325 17.113-21.588 31.592-41.028 39.41-38.88 15.636-83.803-1.318-100.344-37.87-16.542-36.552 1.555-78.873 40.436-94.509"
                        }),
                        /*#__PURE__*/ (0, $8WlKq$jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#6d31f8__g)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 79.293 -10.685)"
                        })
                    ]
                })
            })
        ]
    });
const $7c2b67c1cd5c6faf$var$ForwardRef = /*#__PURE__*/ (0, $8WlKq$forwardRef)($7c2b67c1cd5c6faf$var$SvgComponent);
var $7c2b67c1cd5c6faf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7c2b67c1cd5c6faf$var$ForwardRef);


export {$7c2b67c1cd5c6faf$export$2e2bcd8739ae039 as default};
