// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_speedFast_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $1giBm$jsxs, jsx as $1giBm$jsx} from "react/jsx-runtime";
import {forwardRef as $1giBm$forwardRef} from "react";





const $1bbb4bfd7e4594e8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1giBm$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1giBm$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1giBm$jsxs)("linearGradient", {
                        id: "039432__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1giBm$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $1giBm$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1giBm$jsxs)("linearGradient", {
                        id: "039432__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1giBm$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1giBm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1giBm$jsxs)("linearGradient", {
                        id: "039432__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1giBm$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1giBm$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1giBm$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1giBm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1giBm$jsxs)("radialGradient", {
                        id: "039432__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1giBm$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $1giBm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1giBm$jsxs)("radialGradient", {
                        id: "039432__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1giBm$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1giBm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1giBm$jsx)("clipPath", {
                        id: "039432__b",
                        children: /*#__PURE__*/ (0, $1giBm$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1giBm$jsx)("clipPath", {
                id: "039432__a",
                children: /*#__PURE__*/ (0, $1giBm$jsx)("path", {
                    fill: "#D9D9D9",
                    d: "M79.95 54.607c-2.21 0-4-1.79-4-4V38.642q0-.255.03-.5c-7.404.526-14.41 2.495-20.737 5.627q.134.182.25.38l5.982 10.363c1.104 1.913.449 4.36-1.464 5.464s-4.36.45-5.465-1.464L48.565 48.15q-.1-.173-.18-.352c-5.932 4.078-11.036 9.271-15.011 15.277q.06.032.122.068l10.362 5.982c1.913 1.105 2.569 3.551 1.464 5.465s-3.55 2.568-5.464 1.464L29.495 70.07l-.048-.028c-2.974 6.298-4.806 13.242-5.228 20.564H36.01c2.209 0 4 1.79 4 4s-1.791 4-4 4H24.322c.616 7.337 2.649 14.27 5.824 20.526 1.26 2.481 3.882 3.908 6.665 3.908h86.378c2.783 0 5.405-1.427 6.664-3.908 3.176-6.256 5.209-13.189 5.824-20.526h-11.689c-2.209 0-4-1.79-4-4s1.791-4 4-4h11.793c-1.024-17.748-10.331-33.275-24.114-42.773q-.074.16-.164.316l-5.983 10.362c-1.104 1.913-3.551 2.569-5.464 1.464s-2.568-3.55-1.464-5.464l5.983-10.362q.107-.186.231-.356c-6.37-3.16-13.427-5.142-20.888-5.66q.032.25.032.508v11.965c0 2.21-1.79 4-4 4m40.542 9.21c1.599 1.6 1.666 4.17.153 5.85l-37.346 41.488c-2.753 3.058-7.508 3.183-10.418.273s-2.785-7.665.273-10.418l41.487-37.346c1.681-1.513 4.252-1.446 5.851.153"
                })
            }),
            /*#__PURE__*/ (0, $1giBm$jsx)("g", {
                clipPath: "url(#039432__a)",
                children: /*#__PURE__*/ (0, $1giBm$jsxs)("g", {
                    clipPath: "url(#039432__b)",
                    children: [
                        /*#__PURE__*/ (0, $1giBm$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $1giBm$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $1giBm$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#039432__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $1giBm$jsx)("path", {
                            fill: "url(#039432__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $1giBm$jsx)("path", {
                            fill: "url(#039432__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $1giBm$jsx)("path", {
                            fill: "url(#039432__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $1giBm$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#039432__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $1bbb4bfd7e4594e8$var$ForwardRef = /*#__PURE__*/ (0, $1giBm$forwardRef)($1bbb4bfd7e4594e8$var$SvgComponent);
var $1bbb4bfd7e4594e8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1bbb4bfd7e4594e8$var$ForwardRef);


export {$1bbb4bfd7e4594e8$export$2e2bcd8739ae039 as default};
