// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_exportTo_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $fP0cl$jsxs, jsx as $fP0cl$jsx} from "react/jsx-runtime";
import {forwardRef as $fP0cl$forwardRef} from "react";





const $7dc33c242db35dba$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fP0cl$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fP0cl$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fP0cl$jsxs)("linearGradient", {
                        id: "7d7e7c__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fP0cl$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $fP0cl$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fP0cl$jsxs)("linearGradient", {
                        id: "7d7e7c__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fP0cl$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fP0cl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fP0cl$jsxs)("linearGradient", {
                        id: "7d7e7c__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fP0cl$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fP0cl$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fP0cl$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fP0cl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fP0cl$jsxs)("radialGradient", {
                        id: "7d7e7c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fP0cl$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $fP0cl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fP0cl$jsxs)("radialGradient", {
                        id: "7d7e7c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fP0cl$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fP0cl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fP0cl$jsx)("clipPath", {
                        id: "7d7e7c__b",
                        children: /*#__PURE__*/ (0, $fP0cl$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fP0cl$jsx)("clipPath", {
                id: "7d7e7c__a",
                children: /*#__PURE__*/ (0, $fP0cl$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M16.999 31c0-4.418 3.582-8 8-8h40q.477 0 .94.055L45.463 43.53c-1.953 1.953-1.953 5.118 0 7.07 1.953 1.954 5.118 1.953 7.071 0L72.957 30.18Q73 30.584 73 31v40c0 4.418-3.582 8-8 8h-40c-4.418 0-8-3.582-8-8zm30.585 14.652 21.52-21.52 5.14-5.14H63.999c-1.104 0-2-.896-2-2s.896-2 2-2h14.967c1.104 0 2 .895 2 2v14.967c0 1.104-.896 2-2 2s-2-.896-2-2v-10.03l-5.05 5.049L50.414 48.48q-.147.146-.314.256c-.776.512-1.831.427-2.515-.256-.78-.781-.78-2.048 0-2.829"
                })
            }),
            /*#__PURE__*/ (0, $fP0cl$jsx)("g", {
                clipPath: "url(#7d7e7c__a)",
                children: /*#__PURE__*/ (0, $fP0cl$jsxs)("g", {
                    clipPath: "url(#7d7e7c__b)",
                    children: [
                        /*#__PURE__*/ (0, $fP0cl$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $fP0cl$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $fP0cl$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#7d7e7c__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $fP0cl$jsx)("path", {
                            fill: "url(#7d7e7c__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $fP0cl$jsx)("path", {
                            fill: "url(#7d7e7c__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $fP0cl$jsx)("path", {
                            fill: "url(#7d7e7c__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $fP0cl$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#7d7e7c__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $7dc33c242db35dba$var$ForwardRef = /*#__PURE__*/ (0, $fP0cl$forwardRef)($7dc33c242db35dba$var$SvgComponent);
var $7dc33c242db35dba$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7dc33c242db35dba$var$ForwardRef);


export {$7dc33c242db35dba$export$2e2bcd8739ae039 as default};
