// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_diamond_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $9u4Xj$reactjsxruntime = require("react/jsx-runtime");
var $9u4Xj$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $46295b03555cd590$export$2e2bcd8739ae039);




const $46295b03555cd590$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b4b09c__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b4b09c__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b4b09c__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b4b09c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b4b09c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("clipPath", {
                        id: "b4b09c__b",
                        children: /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("clipPath", {
                id: "b4b09c__a",
                children: /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M81.538 24.563h-3.055L64.168 55.707h31.688zm23.123 31.144L90.343 24.563h14.005c2.106 0 4.127.83 5.625 2.311l26.289 25.996c.837.827 1.455 1.797 1.851 2.837zm-7.54 8H62.903L80.01 127.25zm-8.756 63.28 17.04-63.28h31.366zM55.5 24.563H69.68L55.363 55.707H21.887c.393-1.035 1.007-2.003 1.837-2.828L49.86 26.89c1.5-1.49 3.527-2.326 5.641-2.326M23.235 63.707H54.62l17.144 63.686z"
                })
            }),
            /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("g", {
                clipPath: "url(#b4b09c__a)",
                children: /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#b4b09c__b)",
                    children: [
                        /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#b4b09c__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("path", {
                            fill: "url(#b4b09c__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("path", {
                            fill: "url(#b4b09c__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("path", {
                            fill: "url(#b4b09c__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $9u4Xj$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#b4b09c__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $46295b03555cd590$var$ForwardRef = /*#__PURE__*/ (0, $9u4Xj$react.forwardRef)($46295b03555cd590$var$SvgComponent);
var $46295b03555cd590$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($46295b03555cd590$var$ForwardRef);
