// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_magicWand_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $5NPCf$jsxs, jsx as $5NPCf$jsx} from "react/jsx-runtime";
import {forwardRef as $5NPCf$forwardRef} from "react";





const $02522126b9c9a64c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5NPCf$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5NPCf$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5NPCf$jsxs)("linearGradient", {
                        id: "ccfd51__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5NPCf$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $5NPCf$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5NPCf$jsxs)("linearGradient", {
                        id: "ccfd51__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5NPCf$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5NPCf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5NPCf$jsxs)("linearGradient", {
                        id: "ccfd51__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5NPCf$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5NPCf$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5NPCf$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5NPCf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5NPCf$jsxs)("radialGradient", {
                        id: "ccfd51__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5NPCf$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $5NPCf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5NPCf$jsxs)("radialGradient", {
                        id: "ccfd51__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5NPCf$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5NPCf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5NPCf$jsx)("clipPath", {
                        id: "ccfd51__b",
                        children: /*#__PURE__*/ (0, $5NPCf$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5NPCf$jsx)("clipPath", {
                id: "ccfd51__a",
                children: /*#__PURE__*/ (0, $5NPCf$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "m39.805 13.376-2.116.423c-.28.056-.513.245-.626.506l-.854 1.982c-.337.78-1.478.66-1.644-.172l-.423-2.117c-.057-.28-.245-.512-.506-.626l-1.982-.854c-.781-.336-.66-1.478.172-1.644l2.117-.423c.28-.056.512-.245.625-.506l.854-1.982c.337-.78 1.479-.66 1.644.172l.423 2.117c.057.28.246.512.506.626l1.982.854c.781.336.66 1.478-.172 1.644M21.95 15.19l-1.749.35c-.23.046-.423.202-.516.418l-.706 1.637c-.278.645-1.22.545-1.358-.142l-.35-1.749c-.046-.231-.202-.423-.417-.517l-1.637-.705c-.645-.278-.545-1.22.142-1.358l1.748-.35c.232-.046.424-.202.517-.417l.705-1.637c.278-.645 1.221-.545 1.358.142l.35 1.748c.046.232.202.424.418.517l1.637.705c.645.278.545 1.221-.142 1.358m15.516 12.957 1.749-.35c.687-.136.787-1.08.142-1.357l-1.637-.706c-.215-.093-.371-.285-.418-.516l-.35-1.748c-.136-.688-1.08-.788-1.357-.143l-.706 1.637c-.093.216-.285.372-.517.418l-1.748.35c-.687.137-.787 1.08-.142 1.358l1.637.705c.215.093.371.285.418.517l.35 1.748c.136.687 1.08.787 1.357.142l.706-1.637c.093-.215.285-.371.517-.418M26.66 17.313c1.127-1.127 2.954-1.127 4.081 0s1.128 2.955 0 4.082l-2.675 2.675-4.082-4.082zm-4.444 4.443 4.082 4.082-13.875 13.875c-1.127 1.127-2.954 1.127-4.082 0s-1.127-2.955 0-4.082z"
                })
            }),
            /*#__PURE__*/ (0, $5NPCf$jsx)("g", {
                clipPath: "url(#ccfd51__a)",
                children: /*#__PURE__*/ (0, $5NPCf$jsxs)("g", {
                    clipPath: "url(#ccfd51__b)",
                    children: [
                        /*#__PURE__*/ (0, $5NPCf$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $5NPCf$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $5NPCf$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#ccfd51__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $5NPCf$jsx)("path", {
                            fill: "url(#ccfd51__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $5NPCf$jsx)("path", {
                            fill: "url(#ccfd51__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $5NPCf$jsx)("path", {
                            fill: "url(#ccfd51__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $5NPCf$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#ccfd51__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $02522126b9c9a64c$var$ForwardRef = /*#__PURE__*/ (0, $5NPCf$forwardRef)($02522126b9c9a64c$var$SvgComponent);
var $02522126b9c9a64c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($02522126b9c9a64c$var$ForwardRef);


export {$02522126b9c9a64c$export$2e2bcd8739ae039 as default};
