// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_cut_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $1UYHQ$jsxs, jsx as $1UYHQ$jsx} from "react/jsx-runtime";
import {forwardRef as $1UYHQ$forwardRef} from "react";





const $c0ab80c8703da693$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1UYHQ$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1UYHQ$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1UYHQ$jsxs)("linearGradient", {
                        id: "b5dd92__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1UYHQ$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $1UYHQ$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1UYHQ$jsxs)("linearGradient", {
                        id: "b5dd92__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1UYHQ$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1UYHQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1UYHQ$jsxs)("linearGradient", {
                        id: "b5dd92__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1UYHQ$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1UYHQ$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1UYHQ$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1UYHQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1UYHQ$jsxs)("radialGradient", {
                        id: "b5dd92__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1UYHQ$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $1UYHQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1UYHQ$jsxs)("radialGradient", {
                        id: "b5dd92__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1UYHQ$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1UYHQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1UYHQ$jsx)("clipPath", {
                        id: "b5dd92__b",
                        children: /*#__PURE__*/ (0, $1UYHQ$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1UYHQ$jsx)("clipPath", {
                id: "b5dd92__a",
                children: /*#__PURE__*/ (0, $1UYHQ$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M59.312 14.848c1.045.357 1.602 1.495 1.245 2.54l-8.849 25.846 26.64-9.072c1.046-.356 2.182.203 2.538 1.248.356 1.046-.203 2.182-1.249 2.538l-28.776 9.8q-.017.12-.048.238c-.187.712-.353 1.43-.522 2.163-.505 2.188-1.04 4.499-2.257 7.127-.328.707-.699 1.427-1.092 2.137q.507-.047 1.025-.047c6.047 0 10.948 4.901 10.948 10.948 0 6.046-4.901 10.948-10.948 10.948-6.046 0-10.948-4.902-10.948-10.948 0-2.38.76-4.583 2.05-6.38q.105-.177.25-.335.227-.29.471-.565c1.084-1.381 3.273-4.543 4.615-7.44 1.03-2.222 1.438-3.988 1.916-6.056l.09-.385c-1.069.384-2.215.832-3.66 1.427-1.687.695-3.647 1.792-5.34 2.847-1.71 1.066-3.007 2-3.386 2.321q-.121.103-.253.184c-1.987 2.025-4.755 3.281-7.816 3.281-6.047 0-10.948-4.901-10.948-10.948 0-6.046 4.901-10.948 10.948-10.948 6.046 0 10.948 4.902 10.948 10.948q0 .413-.03.818c1.402-.814 2.927-1.614 4.353-2.202 2.361-.972 3.988-1.579 5.738-2.14l9.807-28.649c.358-1.045 1.495-1.602 2.54-1.244M42.408 66.146c-.872 1.16-1.389 2.604-1.389 4.168 0 3.837 3.11 6.948 6.948 6.948 3.837 0 6.948-3.11 6.948-6.948 0-3.837-3.11-6.948-6.948-6.948-2.035 0-3.866.875-5.137 2.27q-.237.296-.422.51M25.956 41.317c-3.837 0-6.948 3.11-6.948 6.948 0 3.837 3.11 6.948 6.948 6.948 3.837 0 6.948-3.11 6.948-6.948 0-3.837-3.11-6.948-6.948-6.948"
                })
            }),
            /*#__PURE__*/ (0, $1UYHQ$jsx)("g", {
                clipPath: "url(#b5dd92__a)",
                children: /*#__PURE__*/ (0, $1UYHQ$jsxs)("g", {
                    clipPath: "url(#b5dd92__b)",
                    children: [
                        /*#__PURE__*/ (0, $1UYHQ$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $1UYHQ$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $1UYHQ$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#b5dd92__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $1UYHQ$jsx)("path", {
                            fill: "url(#b5dd92__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $1UYHQ$jsx)("path", {
                            fill: "url(#b5dd92__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $1UYHQ$jsx)("path", {
                            fill: "url(#b5dd92__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $1UYHQ$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#b5dd92__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $c0ab80c8703da693$var$ForwardRef = /*#__PURE__*/ (0, $1UYHQ$forwardRef)($c0ab80c8703da693$var$SvgComponent);
var $c0ab80c8703da693$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c0ab80c8703da693$var$ForwardRef);


export {$c0ab80c8703da693$export$2e2bcd8739ae039 as default};
