// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_data_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $7wRf0$reactjsxruntime = require("react/jsx-runtime");
var $7wRf0$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0c2daa41ef8f3181$export$2e2bcd8739ae039);




const $0c2daa41ef8f3181$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsxs)("linearGradient", {
                        id: "25f53e__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsxs)("linearGradient", {
                        id: "25f53e__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsxs)("linearGradient", {
                        id: "25f53e__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsxs)("radialGradient", {
                        id: "25f53e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsxs)("radialGradient", {
                        id: "25f53e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("clipPath", {
                        id: "25f53e__b",
                        children: /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("clipPath", {
                id: "25f53e__a",
                children: /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m78.803 25.5.002-.078c0-4.513-13.792-8.172-30.805-8.172s-30.805 3.659-30.805 8.172q0 .039.002.078h-.002v.032c.049.478.415 1.25 1.755 2.233 1.411 1.034 3.58 2.049 6.442 2.936C31.09 32.467 39.083 33.59 48 33.59s16.91-1.124 22.609-2.89c2.86-.887 5.03-1.902 6.44-2.936 1.34-.982 1.706-1.753 1.756-2.232V25.5zM17.195 36.838v-5.424c1.845 1.215 4.251 2.251 7.012 3.107 6.177 1.916 14.59 3.07 23.793 3.07 9.204 0 17.616-1.154 23.793-3.07 2.761-.856 5.167-1.892 7.012-3.107v5.426c-.05.479-.416 1.25-1.755 2.231-1.41 1.034-3.58 2.05-6.441 2.936-5.698 1.767-13.692 2.89-22.609 2.89s-16.91-1.123-22.608-2.89c-2.861-.887-5.03-1.902-6.442-2.936-1.34-.982-1.706-1.754-1.755-2.233m61.61 5.883c-1.845 1.215-4.25 2.25-7.012 3.107-6.177 1.915-14.589 3.07-23.793 3.07s-17.615-1.155-23.793-3.07c-2.761-.856-5.167-1.892-7.012-3.107v5.393c.049.478.415 1.25 1.755 2.233 1.411 1.034 3.58 2.049 6.442 2.936 5.698 1.766 13.691 2.89 22.608 2.89s16.91-1.124 22.609-2.89c2.86-.887 5.03-1.902 6.44-2.937 1.34-.98 1.706-1.752 1.756-2.23zm-61.61 16.727v-5.452c1.845 1.215 4.251 2.251 7.012 3.107 6.177 1.916 14.59 3.07 23.793 3.07 9.204 0 17.616-1.154 23.793-3.07 2.761-.856 5.167-1.891 7.012-3.107v5.453c-.05.48-.416 1.25-1.755 2.232-1.41 1.034-3.58 2.049-6.441 2.936-5.698 1.766-13.692 2.89-22.609 2.89s-16.91-1.124-22.608-2.89c-2.861-.887-5.03-1.902-6.442-2.937-1.34-.982-1.706-1.754-1.755-2.233m0 5.882v5.498c1.5 4.484 15.448 7.922 30.5 7.922 11.879 0 28.375-2.375 31.11-7.922V65.33c-1.845 1.216-4.25 2.251-7.012 3.107-6.177 1.916-14.589 3.07-23.793 3.07s-17.615-1.154-23.793-3.07c-2.761-.856-5.167-1.891-7.012-3.107"
                })
            }),
            /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("g", {
                clipPath: "url(#25f53e__a)",
                children: /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#25f53e__b)",
                    children: [
                        /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#25f53e__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("path", {
                            fill: "url(#25f53e__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("path", {
                            fill: "url(#25f53e__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("path", {
                            fill: "url(#25f53e__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $7wRf0$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#25f53e__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $0c2daa41ef8f3181$var$ForwardRef = /*#__PURE__*/ (0, $7wRf0$react.forwardRef)($0c2daa41ef8f3181$var$SvgComponent);
var $0c2daa41ef8f3181$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0c2daa41ef8f3181$var$ForwardRef);
