// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_plugin_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $d0HCx$jsxs, jsx as $d0HCx$jsx} from "react/jsx-runtime";
import {forwardRef as $d0HCx$forwardRef} from "react";





const $54792e694251e064$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $d0HCx$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $d0HCx$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $d0HCx$jsxs)("linearGradient", {
                        id: "1c96ee__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d0HCx$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $d0HCx$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d0HCx$jsxs)("linearGradient", {
                        id: "1c96ee__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d0HCx$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $d0HCx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d0HCx$jsxs)("linearGradient", {
                        id: "1c96ee__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d0HCx$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $d0HCx$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $d0HCx$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $d0HCx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d0HCx$jsxs)("radialGradient", {
                        id: "1c96ee__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d0HCx$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $d0HCx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d0HCx$jsxs)("radialGradient", {
                        id: "1c96ee__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d0HCx$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $d0HCx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d0HCx$jsx)("clipPath", {
                        id: "1c96ee__b",
                        children: /*#__PURE__*/ (0, $d0HCx$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $d0HCx$jsx)("clipPath", {
                id: "1c96ee__a",
                children: /*#__PURE__*/ (0, $d0HCx$jsx)("path", {
                    fill: "#fff",
                    d: "M73.755 32.8v-8c0-1.768-1.281-3.2-2.862-3.2H56.585c-1.58 0-2.861 1.432-2.861 3.2v8H42.278v-8c0-1.768-1.281-3.2-2.862-3.2H25.11c-1.58 0-2.862 1.432-2.862 3.2v8h-2.862c-4.74 0-8.584 4.298-8.584 9.6v27.2c0 5.302 3.843 9.6 8.584 9.6h57.231c4.741 0 8.585-4.299 8.585-9.6V42.4c0-5.302-3.844-9.6-8.585-9.6z"
                })
            }),
            /*#__PURE__*/ (0, $d0HCx$jsx)("g", {
                clipPath: "url(#1c96ee__a)",
                children: /*#__PURE__*/ (0, $d0HCx$jsxs)("g", {
                    clipPath: "url(#1c96ee__b)",
                    children: [
                        /*#__PURE__*/ (0, $d0HCx$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $d0HCx$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $d0HCx$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#1c96ee__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $d0HCx$jsx)("path", {
                            fill: "url(#1c96ee__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $d0HCx$jsx)("path", {
                            fill: "url(#1c96ee__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $d0HCx$jsx)("path", {
                            fill: "url(#1c96ee__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $d0HCx$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#1c96ee__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $54792e694251e064$var$ForwardRef = /*#__PURE__*/ (0, $d0HCx$forwardRef)($54792e694251e064$var$SvgComponent);
var $54792e694251e064$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($54792e694251e064$var$ForwardRef);


export {$54792e694251e064$export$2e2bcd8739ae039 as default};
