// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_confetti_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $e0tuF$jsxs, jsx as $e0tuF$jsx} from "react/jsx-runtime";
import {forwardRef as $e0tuF$forwardRef} from "react";





const $cc1ca73b26f85c08$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e0tuF$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $e0tuF$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $e0tuF$jsxs)("linearGradient", {
                        id: "60da7a__d",
                        x1: -17.213,
                        x2: 116.852,
                        y1: 82.649,
                        y2: -3.785,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e0tuF$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $e0tuF$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e0tuF$jsxs)("linearGradient", {
                        id: "60da7a__e",
                        x1: 93.95,
                        x2: 38.201,
                        y1: 25.61,
                        y2: 89.133,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e0tuF$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $e0tuF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e0tuF$jsxs)("linearGradient", {
                        id: "60da7a__f",
                        x1: 21.733,
                        x2: 124.07,
                        y1: 94.229,
                        y2: 33.479,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e0tuF$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $e0tuF$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $e0tuF$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $e0tuF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e0tuF$jsxs)("radialGradient", {
                        id: "60da7a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.9784 -37.0667 0 37.067 34.978)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e0tuF$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $e0tuF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e0tuF$jsxs)("radialGradient", {
                        id: "60da7a__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.9784 -37.0667 0 37.067 34.978)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e0tuF$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $e0tuF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e0tuF$jsx)("clipPath", {
                        id: "60da7a__b",
                        children: /*#__PURE__*/ (0, $e0tuF$jsx)("rect", {
                            width: 85.2,
                            height: 80.4,
                            x: 4.199,
                            y: 10.8,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $e0tuF$jsx)("clipPath", {
                id: "60da7a__a",
                children: /*#__PURE__*/ (0, $e0tuF$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M65.951 19.296c1.716-1.716 1.716-4.488 0-6.204s-4.488-1.716-6.204 0-1.716 4.488 0 6.204 4.488 1.716 6.204 0M38.878 43.272c-1.038.828-1.206 2.34-.378 3.372.474.594 1.17.9 1.878.9.528 0 1.056-.174 1.494-.528 11.532-9.21 11.004-18.876 10.98-19.284-.09-1.314-1.218-2.292-2.532-2.22-1.314.078-2.322 1.212-2.256 2.526.012.312.258 7.686-9.186 15.228zM59.28 61.194c-.618-1.17-.168-2.622 1.002-3.24h.006c1.17-.618 2.622-.168 3.24 1.002 3.438 6.516 2.334 9.708.804 11.238-.498.498-1.17.948-2.07 1.212l-41.55 14.76c-.804.288-1.626.426-2.436.426-1.866 0-3.684-.738-5.064-2.118-1.98-1.98-2.628-4.854-1.692-7.5l14.592-41.118q.125-.468.33-.9c.27-.672.642-1.2 1.05-1.608 1.542-1.542 4.764-2.64 11.358.87 1.17.624 1.614 2.076.99 3.246s-2.076 1.614-3.246.99c-3.714-1.974-5.418-1.854-5.712-1.704v.012l-.108.306c-.24 1.824 2.52 8.922 11.718 18.12 6.972 6.972 12.732 10.242 15.924 11.304 1.236.252 2.112.312 2.556.252.114-.33.174-2.01-1.692-5.55m-12.786-7.536c-.618 0-1.23-.234-1.698-.702-.936-.942-.936-2.46 0-3.396l9.366-9.366c.942-.936 2.46-.936 3.396 0 .936.942.936 2.46 0 3.396l-9.366 9.366c-.468.468-1.08.702-1.698.702m4.23 2.22c9.21-11.532 18.876-11.004 19.284-10.98l-.006.006c1.326.09 2.322 1.236 2.232 2.556-.09 1.314-1.224 2.316-2.538 2.232-.354-.012-7.704-.222-15.222 9.186-.474.594-1.176.9-1.878.9-.522 0-1.05-.174-1.494-.528-1.038-.828-1.206-2.334-.378-3.372m24.3-.576c1.716-1.716 4.488-1.716 6.204 0s1.716 4.488 0 6.204-4.489 1.716-6.205 0-1.716-4.488 0-6.204M33.81 16.917l-1.153.843c-1.07.783-1.302 2.285-.519 3.354l4.688 6.406c.783 1.07 2.285 1.302 3.355.52l1.152-.844c1.07-.783 1.302-2.285.52-3.354l-4.69-6.406c-.782-1.07-2.284-1.302-3.353-.52M64.336 32.91l7.888-.894c1.317-.149 2.505.798 2.654 2.115l.161 1.42c.15 1.316-.798 2.505-2.115 2.654l-7.887.893c-1.317.15-2.506-.798-2.655-2.115l-.16-1.419c-.15-1.317.797-2.505 2.114-2.654"
                })
            }),
            /*#__PURE__*/ (0, $e0tuF$jsx)("g", {
                clipPath: "url(#60da7a__a)",
                children: /*#__PURE__*/ (0, $e0tuF$jsxs)("g", {
                    clipPath: "url(#60da7a__b)",
                    children: [
                        /*#__PURE__*/ (0, $e0tuF$jsx)("rect", {
                            width: 85.2,
                            height: 80.4,
                            x: 4.199,
                            y: 10.8,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $e0tuF$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M4.2 10.8h85.2v80.4H4.2z"
                        }),
                        /*#__PURE__*/ (0, $e0tuF$jsx)("ellipse", {
                            cx: 37.067,
                            cy: 34.978,
                            fill: "url(#60da7a__c)",
                            rx: 37.067,
                            ry: 34.978,
                            transform: "matrix(1 -.00132 .00148 1 15.254 -21.01)"
                        }),
                        /*#__PURE__*/ (0, $e0tuF$jsx)("path", {
                            fill: "url(#60da7a__d)",
                            d: "M36.717-3.402c-6.543-3.821-14.833-3.844-21.4-.06l-65.689 37.847c-6.433 3.707-6.444 12.456-.02 16.177l65.255 37.793c6.557 3.798 14.848 3.793 21.4-.013l65.119-37.83c6.391-3.712 6.402-12.413.02-16.14z"
                        }),
                        /*#__PURE__*/ (0, $e0tuF$jsx)("path", {
                            fill: "url(#60da7a__e)",
                            d: "M16.478 48.972C17.934 18.957 44.9-4.262 76.708-2.888s56.412 26.821 54.956 56.837c-1.456 30.015-28.422 53.234-60.23 51.86S15.023 78.988 16.478 48.972"
                        }),
                        /*#__PURE__*/ (0, $e0tuF$jsx)("path", {
                            fill: "url(#60da7a__f)",
                            d: "M-30.32 68.763c12.644-5.09 26.28-4.872 38.094-.42 23.64 8.931 37.395 3.41 46.936-18.847 4.752-11.169 14.03-20.627 26.688-25.723 25.307-10.172 54.523.865 65.288 24.676s-1.018 51.338-26.319 61.524c-12.686 5.107-26.349 4.868-38.18.376-23.602-8.884-37.331-3.341-46.86 18.943-4.764 11.141-14.04 20.568-26.683 25.658-25.287 10.18-54.504-.858-65.262-24.656s1.012-51.351 26.299-61.531"
                        }),
                        /*#__PURE__*/ (0, $e0tuF$jsx)("ellipse", {
                            cx: 37.067,
                            cy: 34.978,
                            fill: "url(#60da7a__g)",
                            rx: 37.067,
                            ry: 34.978,
                            transform: "matrix(1 -.00132 .00148 1 47.316 -9.604)"
                        })
                    ]
                })
            })
        ]
    });
const $cc1ca73b26f85c08$var$ForwardRef = /*#__PURE__*/ (0, $e0tuF$forwardRef)($cc1ca73b26f85c08$var$SvgComponent);
var $cc1ca73b26f85c08$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cc1ca73b26f85c08$var$ForwardRef);


export {$cc1ca73b26f85c08$export$2e2bcd8739ae039 as default};
