// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_artboard_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $caPC0$reactjsxruntime = require("react/jsx-runtime");
var $caPC0$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a3aa315ffb63e97c$export$2e2bcd8739ae039);




const $a3aa315ffb63e97c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ea8a3b__d",
                        x1: -19.253,
                        x2: 112.468,
                        y1: 76.562,
                        y2: -8.449,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ea8a3b__e",
                        x1: 90.03,
                        x2: 35.288,
                        y1: 20.545,
                        y2: 82.987,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ea8a3b__f",
                        x1: 19.033,
                        x2: 119.588,
                        y1: 87.935,
                        y2: 28.18,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ea8a3b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.3519 -36.4409 0 36.44 34.352)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ea8a3b__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.3519 -36.4409 0 36.44 34.352)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("clipPath", {
                        id: "ea8a3b__b",
                        children: /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("rect", {
                            width: 83.762,
                            height: 78.96,
                            x: 1.797,
                            y: 6,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("clipPath", {
                id: "ea8a3b__a",
                children: /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M24 14.4v-6C24 7.074 22.926 6 21.6 6s-2.4 1.074-2.4 2.4v6c0 1.326 1.074 2.4 2.4 2.4s2.4-1.074 2.4-2.4m23.4 27.612c0 3.3 2.7 5.988 6.018 5.988h20.988v28.8c0 3.978-3.204 7.2-7.158 7.2h-38.49c-3.954 0-7.158-3.222-7.158-7.2v-48c0-3.978 3.204-7.2 7.158-7.2H47.4zM72.78 43.2H53.424c-.672 0-1.218-.534-1.218-1.188V23.334l.093.08q.106.086.207.184l19.686 18.954c.21.204.402.42.588.648M8.4 19.2h6c1.326 0 2.4 1.074 2.4 2.4S15.726 24 14.4 24h-6C7.074 24 6 22.926 6 21.6s1.074-2.4 2.4-2.4"
                })
            }),
            /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ea8a3b__a)",
                children: /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ea8a3b__b)",
                    children: [
                        /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("rect", {
                            width: 83.762,
                            height: 78.96,
                            x: 1.797,
                            y: 6,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M1.797 6h83.762v78.96H1.797z"
                        }),
                        /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("ellipse", {
                            cx: 36.441,
                            cy: 34.352,
                            fill: "url(#ea8a3b__c)",
                            rx: 36.441,
                            ry: 34.352,
                            transform: "matrix(1 -.00132 .00148 1 12.668 -25.241)"
                        }),
                        /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("path", {
                            fill: "url(#ea8a3b__d)",
                            d: "M33.767-7.948c-6.433-3.752-14.583-3.775-21.04-.06l-64.579 37.17c-6.324 3.64-6.335 12.233-.019 15.887l64.152 37.117c6.447 3.73 14.598 3.725 21.04-.013L97.34 45c6.282-3.646 6.293-12.19.018-15.85z"
                        }),
                        /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("path", {
                            fill: "url(#ea8a3b__e)",
                            d: "M13.867 43.489c1.431-29.478 27.942-52.28 59.212-50.931s55.46 26.34 54.029 55.818c-1.432 29.478-27.942 52.281-59.213 50.931-31.27-1.35-55.46-26.34-54.028-55.818"
                        }),
                        /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("path", {
                            fill: "url(#ea8a3b__f)",
                            d: "M-32.14 62.925c12.43-4.999 25.836-4.785 37.45-.412 23.24 8.77 36.764 3.348 46.143-18.51C56.125 33.034 65.246 23.745 77.69 18.74c24.881-9.99 53.604.85 64.186 24.234s-1 50.418-25.874 60.422c-12.472 5.015-25.904 4.78-37.536.369-23.203-8.725-36.7-3.281-46.068 18.603-4.684 10.943-13.803 20.2-26.233 25.199-24.86 9.997-53.583-.843-64.16-24.214s.994-50.432 25.855-60.43"
                        }),
                        /*#__PURE__*/ (0, $caPC0$reactjsxruntime.jsx)("ellipse", {
                            cx: 36.441,
                            cy: 34.352,
                            fill: "url(#ea8a3b__g)",
                            rx: 36.441,
                            ry: 34.352,
                            transform: "matrix(1 -.00132 .00148 1 44.184 -14.04)"
                        })
                    ]
                })
            })
        ]
    });
const $a3aa315ffb63e97c$var$ForwardRef = /*#__PURE__*/ (0, $caPC0$react.forwardRef)($a3aa315ffb63e97c$var$SvgComponent);
var $a3aa315ffb63e97c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a3aa315ffb63e97c$var$ForwardRef);
