// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_targeted_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $6QgnL$reactjsxruntime = require("react/jsx-runtime");
var $6QgnL$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a4374e29a89bbd8b$export$2e2bcd8739ae039);




const $a4374e29a89bbd8b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsxs)("linearGradient", {
                        id: "00c6af__e",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsxs)("linearGradient", {
                        id: "00c6af__f",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsxs)("linearGradient", {
                        id: "00c6af__g",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsxs)("radialGradient", {
                        id: "00c6af__d",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsxs)("radialGradient", {
                        id: "00c6af__h",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("clipPath", {
                        id: "00c6af__a",
                        children: /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("path", {
                            fill: "#fff",
                            d: "M0 0h160v160H0z"
                        })
                    }),
                    /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("clipPath", {
                        id: "00c6af__c",
                        children: /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsxs)("g", {
                clipPath: "url(#00c6af__a)",
                children: [
                    /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("clipPath", {
                        id: "00c6af__b",
                        children: /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("path", {
                            fill: "#fff",
                            fillRule: "evenodd",
                            d: "M14.537 30.564c-.696.178-1.243.716-1.433 1.41-.19.692.007 1.434.515 1.942l15.729 15.729c.415.415.991.627 1.576.579l12.068-.982 22.952 22.951c1.562 1.562 4.094 1.562 5.657 0s1.562-4.094 0-5.657L48.65 43.586l.982-12.07c.048-.585-.164-1.161-.579-1.576l-15.73-15.73c-.507-.508-1.249-.704-1.942-.514s-1.23.736-1.409 1.433l-3.147 12.287zm65.284-2.62c-6.31 0-12.35 1.122-17.936 3.173-2.074.762-4.372-.302-5.134-2.376s.302-4.372 2.376-5.133c6.456-2.372 13.429-3.664 20.694-3.664 33.152 0 60.028 26.875 60.028 60.027S112.973 140 79.821 140s-60.027-26.875-60.027-60.028c0-7.074 1.225-13.87 3.479-20.184.742-2.08 3.03-3.165 5.111-2.422s3.166 3.031 2.423 5.112c-1.95 5.462-3.013 11.35-3.013 17.494C27.794 108.705 51.087 132 79.82 132s52.028-23.294 52.028-52.028-23.294-52.027-52.028-52.027M79.82 54.46c-3.393 0-6.62.66-9.571 1.855-2.047.83-4.38-.158-5.21-2.205-.829-2.048.159-4.38 2.207-5.21 3.888-1.574 8.135-2.44 12.574-2.44 18.506 0 33.508 15.003 33.508 33.509s-15.002 33.509-33.508 33.509c-18.507 0-33.509-15.003-33.509-33.509 0-4.136.751-8.106 2.129-11.775.776-2.068 3.082-3.115 5.15-2.339s3.115 3.083 2.34 5.15c-1.046 2.784-1.619 5.802-1.619 8.964 0 14.088 11.42 25.509 25.509 25.509 14.088 0 25.508-11.421 25.508-25.509S93.908 54.461 79.82 54.461m-9.138 31.165c7.73-.72 13.881-6.873 14.6-14.602 3.09 1.861 5.158 5.25 5.158 9.12 0 5.875-4.763 10.639-10.639 10.639-3.87 0-7.257-2.067-9.119-5.157"
                        })
                    }),
                    /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("g", {
                        clipPath: "url(#00c6af__b)",
                        children: /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsxs)("g", {
                            clipPath: "url(#00c6af__c)",
                            children: [
                                /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("rect", {
                                    width: 160,
                                    height: 160,
                                    fill: "#FFECCF",
                                    rx: 5.125
                                }),
                                /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("path", {
                                    fill: "#FFECCF",
                                    d: "M0 0h160v160H0z"
                                }),
                                /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("circle", {
                                    cx: 90.468,
                                    cy: 6.206,
                                    r: 69.609,
                                    fill: "url(#00c6af__d)",
                                    transform: "rotate(-.08 90.468 6.206)"
                                }),
                                /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("path", {
                                    fill: "url(#00c6af__e)",
                                    d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                                }),
                                /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("path", {
                                    fill: "url(#00c6af__f)",
                                    d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                                }),
                                /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("path", {
                                    fill: "url(#00c6af__g)",
                                    d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                                }),
                                /*#__PURE__*/ (0, $6QgnL$reactjsxruntime.jsx)("circle", {
                                    cx: 150.675,
                                    cy: 28.906,
                                    r: 69.609,
                                    fill: "url(#00c6af__h)",
                                    transform: "rotate(-.08 150.675 28.906)"
                                })
                            ]
                        })
                    })
                ]
            })
        ]
    });
const $a4374e29a89bbd8b$var$ForwardRef = /*#__PURE__*/ (0, $6QgnL$react.forwardRef)($a4374e29a89bbd8b$var$SvgComponent);
var $a4374e29a89bbd8b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a4374e29a89bbd8b$var$ForwardRef);
