// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_trash_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $6jmkc$reactjsxruntime = require("react/jsx-runtime");
var $6jmkc$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $fcfe2b3273fb186f$export$2e2bcd8739ae039);




const $fcfe2b3273fb186f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "36a078__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "36a078__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "36a078__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsxs)("radialGradient", {
                        id: "36a078__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsxs)("radialGradient", {
                        id: "36a078__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("clipPath", {
                        id: "36a078__b",
                        children: /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("clipPath", {
                id: "36a078__a",
                children: /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M33.602 19.6c0-3.314 2.686-6 6-6h16.8c3.314 0 6 2.686 6 6v3.6h18.6c1.325 0 2.4 1.074 2.4 2.4s-1.075 2.4-2.4 2.4h-5.7l-5.245 44.05c-.43 3.623-3.502 6.35-7.15 6.35H33.096c-3.647 0-6.718-2.727-7.15-6.35L20.703 28h-5.7c-1.326 0-2.4-1.075-2.4-2.4 0-1.326 1.074-2.4 2.4-2.4h18.6zm6-1.2c-.664 0-1.2.538-1.2 1.2v3.6h19.2v-3.6c0-.663-.537-1.2-1.2-1.2zm1.2 14.4c-1.326 0-2.4 1.074-2.4 2.4v33.6c0 1.325 1.074 2.4 2.4 2.4s2.4-1.075 2.4-2.4V35.2c0-1.326-1.075-2.4-2.4-2.4m14.4 0c-1.326 0-2.4 1.074-2.4 2.4v33.6c0 1.325 1.074 2.4 2.4 2.4s2.4-1.075 2.4-2.4V35.2c0-1.326-1.075-2.4-2.4-2.4"
                })
            }),
            /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("g", {
                clipPath: "url(#36a078__a)",
                children: /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#36a078__b)",
                    children: [
                        /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#36a078__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("path", {
                            fill: "url(#36a078__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("path", {
                            fill: "url(#36a078__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("path", {
                            fill: "url(#36a078__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $6jmkc$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#36a078__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $fcfe2b3273fb186f$var$ForwardRef = /*#__PURE__*/ (0, $6jmkc$react.forwardRef)($fcfe2b3273fb186f$var$SvgComponent);
var $fcfe2b3273fb186f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($fcfe2b3273fb186f$var$ForwardRef);
