// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_EmptyStateExport_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bklXB$jsxs, jsx as $bklXB$jsx} from "react/jsx-runtime";
import {forwardRef as $bklXB$forwardRef} from "react";





const $7cad2042809abec1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bklXB$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bklXB$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bklXB$jsxs)("linearGradient", {
                        id: "d48d83__d",
                        x1: -6.582,
                        x2: 58.985,
                        y1: 39.92,
                        y2: -2.406,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bklXB$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bklXB$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bklXB$jsxs)("linearGradient", {
                        id: "d48d83__e",
                        x1: 47.824,
                        x2: 20.578,
                        y1: 12.039,
                        y2: 43.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bklXB$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bklXB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bklXB$jsxs)("linearGradient", {
                        id: "d48d83__f",
                        x1: 12.481,
                        x2: 62.535,
                        y1: 45.58,
                        y2: 15.829,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bklXB$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bklXB$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bklXB$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bklXB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bklXB$jsxs)("radialGradient", {
                        id: "d48d83__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.0977 -18.1418 0 18.142 17.098)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bklXB$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bklXB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bklXB$jsxs)("radialGradient", {
                        id: "d48d83__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.0977 -18.1418 0 18.142 17.098)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bklXB$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bklXB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bklXB$jsx)("clipPath", {
                        id: "d48d83__b",
                        children: /*#__PURE__*/ (0, $bklXB$jsx)("rect", {
                            width: 41.7,
                            height: 39.3,
                            x: 3.898,
                            y: 4.8,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bklXB$jsx)("clipPath", {
                id: "d48d83__a",
                children: /*#__PURE__*/ (0, $bklXB$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M33.598 4.8h6c1.989 0 3.6 1.612 3.6 3.6v13.2c0 1.988-1.611 3.6-3.6 3.6h-6c-1.988 0-3.6-1.612-3.6-3.6V8.4c0-1.988 1.612-3.6 3.6-3.6M17.398 30h13.2c1.989 0 3.6 1.612 3.6 3.6v6c0 1.988-1.611 3.6-3.6 3.6h-13.2c-1.988 0-3.6-1.612-3.6-3.6v-6c0-1.988 1.612-3.6 3.6-3.6m2.476-8.56 2.478 6.037c.402.975 1.776.996 2.205.033l.723-1.62c.12-.27.336-.486.606-.606l1.617-.723c.963-.43.942-1.806-.033-2.205l-6.033-2.481c-.987-.405-1.971.579-1.566 1.566zm-2.95 1.165c.663 0 1.2.537 1.2 1.2s-.537 1.2-1.2 1.2H10.79c-.447.918-1.383 1.55-2.472 1.55-1.52 0-2.75-1.232-2.75-2.75 0-1.09.635-2.025 1.55-2.472V10.77c-.918-.447-1.55-1.383-1.55-2.472 0-1.521 1.232-2.751 2.75-2.751 1.09 0 2.025.636 2.472 1.55h10.518c.447-.917 1.383-1.55 2.472-1.55 1.521 0 2.751 1.233 2.751 2.75 0 1.09-.636 2.026-1.55 2.473v6.15c0 .663-.538 1.2-1.2 1.2-.664 0-1.2-.537-1.2-1.2v-6.15c-.556-.27-1.003-.72-1.273-1.272H10.79c-.27.555-.72 1.002-1.272 1.272v10.563c.555.27 1.002.72 1.272 1.272z"
                })
            }),
            /*#__PURE__*/ (0, $bklXB$jsx)("g", {
                clipPath: "url(#d48d83__a)",
                children: /*#__PURE__*/ (0, $bklXB$jsxs)("g", {
                    clipPath: "url(#d48d83__b)",
                    children: [
                        /*#__PURE__*/ (0, $bklXB$jsx)("rect", {
                            width: 41.7,
                            height: 39.3,
                            x: 3.898,
                            y: 4.8,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $bklXB$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.898 4.8h41.7v39.3h-41.7z"
                        }),
                        /*#__PURE__*/ (0, $bklXB$jsx)("ellipse", {
                            cx: 18.142,
                            cy: 17.098,
                            fill: "url(#d48d83__c)",
                            rx: 18.142,
                            ry: 17.098,
                            transform: "matrix(1 -.00132 .00148 1 9.309 -10.75)"
                        }),
                        /*#__PURE__*/ (0, $bklXB$jsx)("path", {
                            fill: "url(#d48d83__d)",
                            d: "M19.814-2.142c-3.203-1.868-7.26-1.88-10.475-.03l-32.15 18.5c-3.149 1.812-3.154 6.09-.01 7.908L9.117 42.71c3.21 1.856 7.267 1.854 10.474-.007l31.872-18.491c3.128-1.815 3.133-6.068.01-7.89z"
                        }),
                        /*#__PURE__*/ (0, $bklXB$jsx)("path", {
                            fill: "url(#d48d83__e)",
                            d: "M9.907 23.459c.712-14.672 13.91-26.021 29.478-25.35 15.568.672 27.61 13.11 26.898 27.782-.713 14.672-13.911 26.022-29.479 25.35S9.194 38.13 9.907 23.459"
                        }),
                        /*#__PURE__*/ (0, $bklXB$jsx)("path", {
                            fill: "url(#d48d83__f)",
                            d: "M-12.996 33.132c6.189-2.488 12.862-2.38 18.645-.205 11.57 4.366 18.302 1.667 22.972-9.213 2.325-5.459 6.866-10.082 13.062-12.573 12.386-4.972 26.685.423 31.954 12.062s-.498 25.094-12.882 30.073c-6.209 2.496-12.895 2.38-18.686.184-11.552-4.343-18.271-1.633-22.935 9.26-2.332 5.445-6.872 10.053-13.06 12.541-12.376 4.976-26.676-.42-31.941-12.052s.495-25.1 12.871-30.077"
                        }),
                        /*#__PURE__*/ (0, $bklXB$jsx)("ellipse", {
                            cx: 18.142,
                            cy: 17.098,
                            fill: "url(#d48d83__g)",
                            rx: 18.142,
                            ry: 17.098,
                            transform: "matrix(1 -.00132 .00148 1 25 -5.174)"
                        })
                    ]
                })
            })
        ]
    });
const $7cad2042809abec1$var$ForwardRef = /*#__PURE__*/ (0, $bklXB$forwardRef)($7cad2042809abec1$var$SvgComponent);
var $7cad2042809abec1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7cad2042809abec1$var$ForwardRef);


export {$7cad2042809abec1$export$2e2bcd8739ae039 as default};
