// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_libraries_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $aalkp$reactjsxruntime = require("react/jsx-runtime");
var $aalkp$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a6f578dc879ee135$export$2e2bcd8739ae039);




const $a6f578dc879ee135$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "be5f84__d",
                        x1: -3.949,
                        x2: 55.306,
                        y1: 38.702,
                        y2: 0.46,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "be5f84__e",
                        x1: 45.213,
                        x2: 20.587,
                        y1: 13.503,
                        y2: 41.593,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "be5f84__f",
                        x1: 13.273,
                        x2: 58.507,
                        y1: 43.818,
                        y2: 16.938,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsxs)("radialGradient", {
                        id: "be5f84__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsxs)("radialGradient", {
                        id: "be5f84__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("clipPath", {
                        id: "be5f84__b",
                        children: /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("clipPath", {
                id: "be5f84__a",
                children: /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M9.427 14.23c0-2.653 2.148-4.8 4.8-4.8h16.146c.663 0 1.2-.538 1.2-1.2 0-.663-.537-1.2-1.2-1.2H14.227c-3.977 0-7.2 3.222-7.2 7.2v16.145c0 .663.538 1.2 1.2 1.2.663 0 1.2-.537 1.2-1.2zm18.748 10.958V12.73H17.528c-2.651 0-4.8 2.15-4.8 4.8v17.445c0 2.651 2.149 4.8 4.8 4.8h17.445c2.65 0 4.8-2.149 4.8-4.8V17.53c0-2.49-1.898-4.538-4.326-4.777v12.436c0 .477-.534.765-.933.498l-2.37-1.578c-.201-.135-.465-.135-.666 0l-2.37 1.578c-.4.267-.933-.018-.933-.498"
                })
            }),
            /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("g", {
                clipPath: "url(#be5f84__a)",
                children: /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#be5f84__b)",
                    children: [
                        /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M5.52 6.96H43.2v35.52H5.52z"
                        }),
                        /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#be5f84__c)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 10.41 -7.094)"
                        }),
                        /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("path", {
                            fill: "url(#be5f84__d)",
                            d: "M19.902.685c-2.894-1.688-6.56-1.698-9.465-.026l-29.05 16.72c-2.845 1.638-2.85 5.503-.009 7.147l28.859 16.697c2.9 1.678 6.567 1.676 9.464-.006l28.8-16.712c2.826-1.64 2.83-5.485.008-7.131z"
                        }),
                        /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("path", {
                            fill: "url(#be5f84__e)",
                            d: "M10.951 23.824C11.595 10.564 23.521.306 37.588.913s24.949 11.849 24.305 25.11-12.57 23.518-26.637 22.91c-14.067-.606-24.949-11.848-24.305-25.109"
                        }),
                        /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("path", {
                            fill: "url(#be5f84__f)",
                            d: "M-9.747 32.567c5.591-2.248 11.622-2.152 16.847-.185 10.455 3.946 16.538 1.506 20.757-8.327 2.102-4.934 6.205-9.113 11.803-11.364 11.192-4.494 24.113.382 28.874 10.902s-.45 22.68-11.64 27.18c-5.61 2.257-11.652 2.151-16.885.167-10.438-3.925-16.51-1.476-20.724 8.368-2.107 4.923-6.21 9.087-11.8 11.336C-3.7 75.14-16.62 70.264-21.379 59.75s.447-22.686 11.63-27.184"
                        }),
                        /*#__PURE__*/ (0, $aalkp$reactjsxruntime.jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#be5f84__g)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 24.586 -2.055)"
                        })
                    ]
                })
            })
        ]
    });
const $a6f578dc879ee135$var$ForwardRef = /*#__PURE__*/ (0, $aalkp$react.forwardRef)($a6f578dc879ee135$var$SvgComponent);
var $a6f578dc879ee135$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a6f578dc879ee135$var$ForwardRef);
