// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_megaphonePromote_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $kayuV$jsxs, jsx as $kayuV$jsx} from "react/jsx-runtime";
import {forwardRef as $kayuV$forwardRef} from "react";





const $7f89270740ed1976$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kayuV$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kayuV$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kayuV$jsxs)("linearGradient", {
                        id: "30a04d__d",
                        x1: -3.949,
                        x2: 55.306,
                        y1: 38.702,
                        y2: 0.46,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kayuV$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kayuV$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kayuV$jsxs)("linearGradient", {
                        id: "30a04d__e",
                        x1: 45.213,
                        x2: 20.587,
                        y1: 13.503,
                        y2: 41.593,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kayuV$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kayuV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kayuV$jsxs)("linearGradient", {
                        id: "30a04d__f",
                        x1: 13.273,
                        x2: 58.507,
                        y1: 43.818,
                        y2: 16.938,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kayuV$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kayuV$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kayuV$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kayuV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kayuV$jsxs)("radialGradient", {
                        id: "30a04d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kayuV$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kayuV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kayuV$jsxs)("radialGradient", {
                        id: "30a04d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kayuV$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kayuV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kayuV$jsx)("clipPath", {
                        id: "30a04d__b",
                        children: /*#__PURE__*/ (0, $kayuV$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kayuV$jsx)("clipPath", {
                id: "30a04d__a",
                children: /*#__PURE__*/ (0, $kayuV$jsx)("path", {
                    fill: "#fff",
                    d: "m35.729 9.351-12.93 3.924h-9.294c-3.477 0-6.294 2.817-6.294 6.294s2.817 6.294 6.294 6.294h1.782l4.716 10.851c.75 1.734 2.766 2.532 4.5 1.782s2.532-2.766 1.782-4.5l-3.501-8.13h.012l12.927 3.933c1.926.585 3.873-.855 3.873-2.871V12.222c0-2.013-1.944-3.456-3.87-2.871z"
                })
            }),
            /*#__PURE__*/ (0, $kayuV$jsx)("g", {
                clipPath: "url(#30a04d__a)",
                children: /*#__PURE__*/ (0, $kayuV$jsxs)("g", {
                    clipPath: "url(#30a04d__b)",
                    children: [
                        /*#__PURE__*/ (0, $kayuV$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $kayuV$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M5.52 6.96H43.2v35.52H5.52z"
                        }),
                        /*#__PURE__*/ (0, $kayuV$jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#30a04d__c)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 10.41 -7.094)"
                        }),
                        /*#__PURE__*/ (0, $kayuV$jsx)("path", {
                            fill: "url(#30a04d__d)",
                            d: "M19.902.685c-2.894-1.688-6.56-1.698-9.465-.026l-29.05 16.72c-2.845 1.638-2.85 5.503-.009 7.147l28.859 16.697c2.9 1.678 6.567 1.676 9.464-.006l28.8-16.712c2.826-1.64 2.83-5.485.008-7.131z"
                        }),
                        /*#__PURE__*/ (0, $kayuV$jsx)("path", {
                            fill: "url(#30a04d__e)",
                            d: "M10.951 23.824C11.595 10.564 23.521.306 37.588.913s24.949 11.849 24.305 25.11-12.57 23.518-26.637 22.91c-14.067-.606-24.949-11.848-24.305-25.109"
                        }),
                        /*#__PURE__*/ (0, $kayuV$jsx)("path", {
                            fill: "url(#30a04d__f)",
                            d: "M-9.747 32.567c5.591-2.248 11.622-2.152 16.847-.185 10.455 3.946 16.538 1.506 20.757-8.327 2.102-4.934 6.205-9.113 11.803-11.364 11.192-4.494 24.113.382 28.874 10.902s-.45 22.68-11.64 27.18c-5.61 2.257-11.652 2.151-16.885.167-10.438-3.925-16.51-1.476-20.724 8.368-2.107 4.923-6.21 9.087-11.8 11.336C-3.7 75.14-16.62 70.264-21.379 59.75s.447-22.686 11.63-27.184"
                        }),
                        /*#__PURE__*/ (0, $kayuV$jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#30a04d__g)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 24.586 -2.055)"
                        })
                    ]
                })
            })
        ]
    });
const $7f89270740ed1976$var$ForwardRef = /*#__PURE__*/ (0, $kayuV$forwardRef)($7f89270740ed1976$var$SvgComponent);
var $7f89270740ed1976$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7f89270740ed1976$var$ForwardRef);


export {$7f89270740ed1976$export$2e2bcd8739ae039 as default};
