// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_noElements_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bEx1j$jsxs, jsx as $bEx1j$jsx} from "react/jsx-runtime";
import {forwardRef as $bEx1j$forwardRef} from "react";





const $239c129df6098817$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bEx1j$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bEx1j$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bEx1j$jsxs)("linearGradient", {
                        id: "5c4bed__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bEx1j$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bEx1j$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bEx1j$jsxs)("linearGradient", {
                        id: "5c4bed__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bEx1j$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bEx1j$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bEx1j$jsxs)("linearGradient", {
                        id: "5c4bed__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bEx1j$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bEx1j$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bEx1j$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bEx1j$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bEx1j$jsxs)("radialGradient", {
                        id: "5c4bed__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bEx1j$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bEx1j$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bEx1j$jsxs)("radialGradient", {
                        id: "5c4bed__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bEx1j$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bEx1j$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bEx1j$jsx)("clipPath", {
                        id: "5c4bed__b",
                        children: /*#__PURE__*/ (0, $bEx1j$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bEx1j$jsxs)("clipPath", {
                id: "5c4bed__a",
                children: [
                    /*#__PURE__*/ (0, $bEx1j$jsx)("path", {
                        fill: "#D9D9D9",
                        d: "M77.467 10.921c-.633-.194-1.662.38-2.254 1.064L62.076 27.273c-.648.759-.912 1.796-.699 2.786s.872 1.822 1.768 2.223l4.975 2.227-3.338 8.232c-.4.98-.304 2.09.253 2.96.562.884 1.545 1.432 2.563 1.432.857 0 1.657-.374 2.245-1.053l13.07-15.083c.659-.759.922-1.807.71-2.802s-.877-1.828-1.784-2.222l-5.005-2.202 3.45-8.442c.4-.985.309-2.09-.248-2.965-.563-.885-1.546-1.438-2.564-1.438zM10.27 56c0-1.105.896-2 2-2h28.147c1.105 0 2 .895 2 2v5.63c0 1.104-.895 2-2 2s-2-.896-2-2V58H28.343v24.147h3.63c1.104 0 2 .895 2 2s-.896 2-2 2h-11.26c-1.104 0-2-.896-2-2s.896-2 2-2h3.63V58H14.27v3.63c0 1.104-.896 2-2 2s-2-.896-2-2z"
                    }),
                    /*#__PURE__*/ (0, $bEx1j$jsx)("path", {
                        fill: "#D9D9D9",
                        fillRule: "evenodd",
                        d: "M53.924 76.688V64c0-4.418 3.582-8 8-8h21.763c4.419 0 8 3.582 8 8v12.688c0 4.418-3.581 8-8 8H61.924c-4.418 0-8-3.582-8-8m24.128-7.43-7.378-4.21c-.826-.472-1.86.115-1.861 1.056l-.01 8.406c-.001.941 1.031 1.53 1.859 1.061l7.388-4.194c.827-.47.829-1.646.002-2.118M7 34.75V22c0-4.418 3.582-8 8-8h22.125c4.418 0 8 3.582 8 8v12.75c0 1.422-.37 2.757-1.021 3.914l-4.218-6.582c-2.361-3.683-7.743-3.683-10.103 0l-1.356 2.115-3.832-6.04c-2.348-3.702-7.745-3.717-10.114-.029l-6.6 10.274C7.318 37.307 7 36.066 7 34.75M38 23.5c1.38 0 2.5-1.12 2.5-2.5s-1.12-2.5-2.5-2.5-2.5 1.12-2.5 2.5 1.12 2.5 2.5 2.5"
                    }),
                    /*#__PURE__*/ (0, $bEx1j$jsx)("path", {
                        fill: "#D9D9D9",
                        d: "M26.046 37.912 21.218 30.3c-.783-1.234-2.582-1.24-3.372-.01l-7.184 11.183c1.25.808 2.74 1.277 4.338 1.277h7.945zM27.696 42.75h9.429c1.506 0 2.914-.416 4.117-1.14l-4.724-7.37c-.787-1.227-2.58-1.227-3.368 0z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bEx1j$jsx)("g", {
                clipPath: "url(#5c4bed__a)",
                children: /*#__PURE__*/ (0, $bEx1j$jsxs)("g", {
                    clipPath: "url(#5c4bed__b)",
                    children: [
                        /*#__PURE__*/ (0, $bEx1j$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $bEx1j$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $bEx1j$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#5c4bed__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $bEx1j$jsx)("path", {
                            fill: "url(#5c4bed__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $bEx1j$jsx)("path", {
                            fill: "url(#5c4bed__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $bEx1j$jsx)("path", {
                            fill: "url(#5c4bed__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $bEx1j$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#5c4bed__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $239c129df6098817$var$ForwardRef = /*#__PURE__*/ (0, $bEx1j$forwardRef)($239c129df6098817$var$SvgComponent);
var $239c129df6098817$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($239c129df6098817$var$ForwardRef);


export {$239c129df6098817$export$2e2bcd8739ae039 as default};
