// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_paperclip_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $hW6bH$jsxs, jsx as $hW6bH$jsx} from "react/jsx-runtime";
import {forwardRef as $hW6bH$forwardRef} from "react";





const $6388778c049d581e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hW6bH$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hW6bH$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hW6bH$jsxs)("linearGradient", {
                        id: "9b19fe__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 75.904,
                        y2: -0.58,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hW6bH$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $hW6bH$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hW6bH$jsxs)("linearGradient", {
                        id: "9b19fe__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 25.506,
                        y2: 81.685,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hW6bH$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hW6bH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hW6bH$jsxs)("linearGradient", {
                        id: "9b19fe__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 86.136,
                        y2: 32.375,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hW6bH$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hW6bH$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hW6bH$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hW6bH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hW6bH$jsxs)("radialGradient", {
                        id: "9b19fe__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hW6bH$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $hW6bH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hW6bH$jsxs)("radialGradient", {
                        id: "9b19fe__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hW6bH$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hW6bH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hW6bH$jsx)("clipPath", {
                        id: "9b19fe__b",
                        children: /*#__PURE__*/ (0, $hW6bH$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 12.42,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hW6bH$jsx)("clipPath", {
                id: "9b19fe__a",
                children: /*#__PURE__*/ (0, $hW6bH$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    stroke: "#fff",
                    strokeLinecap: "round",
                    strokeLinejoin: "round",
                    strokeWidth: 1.2,
                    d: "M21.211 53.732c-6.682 6.65-6.682 17.436 0 24.086 6.677 6.643 17.496 6.643 24.172 0L72.424 50.91c.94-.935.944-2.455.009-3.394s-2.455-.944-3.394-.009L41.997 74.415c-4.803 4.78-12.596 4.78-17.4 0-4.797-4.773-4.797-12.508 0-17.28l37.034-36.852c3.058-3.044 8.021-3.044 11.08 0 3.051 3.036 3.051 7.954 0 10.99L39.515 64.306c-1.276 1.271-3.352 1.271-4.629 0-1.27-1.264-1.27-3.308 0-4.573L58.09 36.646c.94-.935.943-2.455.008-3.394s-2.455-.944-3.394-.009L31.5 56.33c-3.157 3.14-3.156 8.237 0 11.378 3.15 3.134 8.25 3.134 11.4 0l33.195-33.032c4.938-4.913 4.938-12.883 0-17.796-4.93-4.906-12.92-4.906-17.85 0z"
                })
            }),
            /*#__PURE__*/ (0, $hW6bH$jsx)("g", {
                clipPath: "url(#9b19fe__a)",
                children: /*#__PURE__*/ (0, $hW6bH$jsxs)("g", {
                    clipPath: "url(#9b19fe__b)",
                    children: [
                        /*#__PURE__*/ (0, $hW6bH$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 12.42,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $hW6bH$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 12.42H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $hW6bH$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#9b19fe__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -15.688)"
                        }),
                        /*#__PURE__*/ (0, $hW6bH$jsx)("path", {
                            fill: "url(#9b19fe__d)",
                            d: "M39.804-.13c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $hW6bH$jsx)("path", {
                            fill: "url(#9b19fe__e)",
                            d: "M21.899 46.148C23.187 19.627 47.039-.889 75.172.326s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 72.67 21.898 46.148"
                        }),
                        /*#__PURE__*/ (0, $hW6bH$jsx)("path", {
                            fill: "url(#9b19fe__f)",
                            d: "M-19.495 63.635C-8.31 59.137 3.75 59.33 14.2 63.264c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $hW6bH$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#9b19fe__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -5.608)"
                        })
                    ]
                })
            })
        ]
    });
const $6388778c049d581e$var$ForwardRef = /*#__PURE__*/ (0, $hW6bH$forwardRef)($6388778c049d581e$var$SvgComponent);
var $6388778c049d581e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6388778c049d581e$var$ForwardRef);


export {$6388778c049d581e$export$2e2bcd8739ae039 as default};
