// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_rocket_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $b58wK$reactjsxruntime = require("react/jsx-runtime");
var $b58wK$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ad8f736074564676$export$2e2bcd8739ae039);




const $ad8f736074564676$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c7b3fa__d",
                        x1: -11.953,
                        x2: 111.651,
                        y1: 80.134,
                        y2: 0.361,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c7b3fa__e",
                        x1: 90.597,
                        x2: 39.227,
                        y1: 27.568,
                        y2: 86.163,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c7b3fa__f",
                        x1: 23.977,
                        x2: 118.335,
                        y1: 90.806,
                        y2: 34.734,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsxs)("radialGradient", {
                        id: "c7b3fa__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsxs)("radialGradient", {
                        id: "c7b3fa__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("clipPath", {
                        id: "c7b3fa__b",
                        children: /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("clipPath", {
                id: "c7b3fa__a",
                children: /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M64.213 52.332c-3.696 3.528-8.274 7.188-13.932 10.914h-.006q-.71.466-1.477.798c-3.57 1.548-7.8.816-10.626-2.01L36.05 59.91l-2.124-2.124c-2.676-2.682-3.468-6.612-2.226-10.056.257-.708.593-1.398 1.02-2.046 2.694-4.092 5.358-7.614 7.956-10.65C55.777 17.4 68.857 16.26 74.827 16.938c2.207.252 3.948 1.998 4.2 4.2.63 5.58-.319 17.364-14.814 31.194m1.02-21.594c-1.987-1.98-5.203-1.98-7.183 0-1.98 1.986-1.985 5.202 0 7.182s5.203 1.986 7.183 0 1.986-5.202 0-7.182M26.466 49.164l-6.288.108v.006c-4.086.072-6.402-4.68-3.81-7.836 7.326-8.91 17.196-12.084 23.634-12.81-3.882 4.092-7.656 8.904-11.292 14.424-1.236 1.878-1.974 3.978-2.244 6.108m-9.246 21.96c-.468.468-1.086.702-1.698.702-.618 0-1.23-.234-1.698-.702-.936-.942-.936-2.46 0-3.396l6.3-6.3c.942-.936 2.46-.936 3.396 0 .936.942.936 2.46 0 3.396zm8.4-4.2c.942-.936 2.46-.936 3.396 0 .936.942.936 2.46 0 3.396l-12 12c-.468.468-1.08.702-1.698.702s-1.23-.234-1.698-.702c-.936-.942-.936-2.46 0-3.396zm5.532 5.526-6.6 6.6c-.936.936-.936 2.454 0 3.396.468.468 1.08.702 1.698.702s1.23-.234 1.698-.702l6.6-6.6c.936-.936.936-2.454 0-3.396s-2.454-.936-3.396 0m15.667-2.934-.108 6.264c-.072 4.11 4.71 6.39 7.878 3.774 8.856-7.308 12.024-17.13 12.762-23.562-3.888 3.672-8.634 7.446-14.43 11.262-1.848 1.218-3.93 1.98-6.102 2.262"
                })
            }),
            /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("g", {
                clipPath: "url(#c7b3fa__a)",
                children: /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#c7b3fa__b)",
                    children: [
                        /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M7.8 13.92h78.6v74.094H7.8z"
                        }),
                        /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#c7b3fa__c)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 18 -15.396)"
                        }),
                        /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("path", {
                            fill: "url(#c7b3fa__d)",
                            d: "M37.8.832C31.764-2.69 24.116-2.711 18.057.776l-60.6 34.88c-5.935 3.415-5.945 11.478-.018 14.907l60.2 34.83c6.05 3.5 13.697 3.495 19.742-.013l60.075-34.862c5.895-3.421 5.906-11.44.018-14.874z"
                        }),
                        /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("path", {
                            fill: "url(#c7b3fa__e)",
                            d: "M19.126 49.099C20.47 21.437 45.346.039 74.69 1.306c29.344 1.266 52.043 24.717 50.7 52.379-1.344 27.661-26.22 49.059-55.565 47.792-29.343-1.266-52.042-24.717-50.699-52.378"
                        }),
                        /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("path", {
                            fill: "url(#c7b3fa__f)",
                            d: "M-24.044 67.337c11.665-4.69 24.244-4.49 35.143-.386 21.809 8.23 34.498 3.142 43.3-17.37 4.384-10.292 12.943-19.01 24.62-23.705 23.348-9.375 50.301.797 60.231 22.74 9.931 21.945-.939 47.313-24.28 56.699-11.703 4.707-24.307 4.486-35.223.347-21.773-8.187-34.439-3.079-43.229 17.457-4.395 10.268-12.952 18.955-24.617 23.646-23.328 9.381-50.281-.791-60.206-22.722s.933-47.324 24.261-56.706"
                        }),
                        /*#__PURE__*/ (0, $b58wK$reactjsxruntime.jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#c7b3fa__g)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 47.578 -4.884)"
                        })
                    ]
                })
            })
        ]
    });
const $ad8f736074564676$var$ForwardRef = /*#__PURE__*/ (0, $b58wK$react.forwardRef)($ad8f736074564676$var$SvgComponent);
var $ad8f736074564676$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ad8f736074564676$var$ForwardRef);
