// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_gift_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8InWS$reactjsxruntime = require("react/jsx-runtime");
var $8InWS$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2b525acadde2f190$export$2e2bcd8739ae039);




const $2b525acadde2f190$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f49234__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f49234__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f49234__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f49234__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f49234__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("clipPath", {
                        id: "f49234__b",
                        children: /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("clipPath", {
                id: "f49234__a",
                children: /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M28.685 25.506c0-5.887 4.772-10.66 10.66-10.66 3.384 0 6.688 1.578 8.64 4.038 1.953-2.46 4.68-4.037 8.065-4.037 5.887 0 10.66 4.772 10.66 10.66 0 2.244-.694 4.328-1.88 6.046h-8.717c3.31-.034 5.984-2.728 5.984-6.047 0-3.34-2.707-6.047-6.047-6.047-3.324 0-5.446 2.683-5.47 6.002v6.092l-5.189-.004v-6.088c-.024-3.319-2.722-6.002-6.047-6.002-3.34 0-6.047 2.708-6.047 6.047 0 3.319 2.674 6.013 5.984 6.047h-8.717c-1.185-1.718-1.88-3.802-1.88-6.047M16 36.18c0-2.547 2.065-4.613 4.613-4.613h24.792v19.027H20.613c-2.548 0-4.613-2.065-4.613-4.612zm64 0c0-2.547-2.065-4.612-4.613-4.612H50.595v19.027h24.792c2.548 0 4.613-2.065 4.613-4.613zM50.595 55.207h25.369v14.415C75.964 75.353 71.317 80 65.586 80H50.595zm-30.559 0h25.37V80H30.413c-5.731 0-10.378-4.647-10.378-10.378z"
                })
            }),
            /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("g", {
                clipPath: "url(#f49234__a)",
                children: /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#f49234__b)",
                    children: [
                        /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#f49234__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("path", {
                            fill: "url(#f49234__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("path", {
                            fill: "url(#f49234__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("path", {
                            fill: "url(#f49234__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $8InWS$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#f49234__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $2b525acadde2f190$var$ForwardRef = /*#__PURE__*/ (0, $8InWS$react.forwardRef)($2b525acadde2f190$var$SvgComponent);
var $2b525acadde2f190$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2b525acadde2f190$var$ForwardRef);
