// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_search_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $523HE$reactjsxruntime = require("react/jsx-runtime");
var $523HE$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ef8e0f1b2519e0b0$export$2e2bcd8739ae039);




const $ef8e0f1b2519e0b0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $523HE$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "99f169__d",
                        x1: -7.103,
                        x2: 56.115,
                        y1: 38.449,
                        y2: -2.351,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "99f169__e",
                        x1: 45.348,
                        x2: 19.075,
                        y1: 11.564,
                        y2: 41.533,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "99f169__f",
                        x1: 11.275,
                        x2: 59.534,
                        y1: 43.908,
                        y2: 15.229,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsxs)("radialGradient", {
                        id: "99f169__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.4867 -17.4893 0 17.49 16.487)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsxs)("radialGradient", {
                        id: "99f169__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.4867 -17.4893 0 17.49 16.487)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("clipPath", {
                        id: "99f169__b",
                        children: /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("rect", {
                            width: 40.2,
                            height: 37.896,
                            x: 3,
                            y: 4.584,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("clipPath", {
                id: "99f169__a",
                children: /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M7.2 19.073c0-6.213 5.038-11.25 11.25-11.25 6.214 0 11.25 5.037 11.25 11.25s-5.036 11.25-11.25 11.25c-6.212 0-11.25-5.037-11.25-11.25m11.25-13.65c-7.538 0-13.65 6.111-13.65 13.65 0 7.538 6.112 13.65 13.65 13.65 3.338 0 6.396-1.198 8.767-3.187l2.043 2.043-.69.691 8.852 8.852c.843.843 2.21.843 3.052 0s.843-2.21 0-3.052l-8.852-8.852-.665.664-2.043-2.043c1.99-2.371 3.187-5.429 3.187-8.766 0-7.539-6.111-13.65-13.65-13.65"
                })
            }),
            /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("g", {
                clipPath: "url(#99f169__a)",
                children: /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#99f169__b)",
                    children: [
                        /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("rect", {
                            width: 40.2,
                            height: 37.896,
                            x: 3,
                            y: 4.584,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3 4.584h40.2V42.48H3z"
                        }),
                        /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.489,
                            cy: 16.487,
                            fill: "url(#99f169__c)",
                            rx: 17.489,
                            ry: 16.487,
                            transform: "matrix(1 -.00132 .00148 1 8.215 -10.41)"
                        }),
                        /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("path", {
                            fill: "url(#99f169__d)",
                            d: "M18.344-2.11c-3.088-1.801-7-1.812-10.098-.029l-30.994 17.84c-3.035 1.746-3.04 5.87-.009 7.624L8.032 41.139c3.094 1.79 7.006 1.787 10.097-.007l30.726-17.83c3.015-1.75 3.02-5.851.009-7.608z"
                        }),
                        /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("path", {
                            fill: "url(#99f169__e)",
                            d: "M8.794 22.576C9.481 8.43 22.204-2.515 37.212-1.868s26.618 12.642 25.93 26.79-13.41 25.091-28.418 24.443S8.107 36.724 8.794 22.576"
                        }),
                        /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("path", {
                            fill: "url(#99f169__f)",
                            d: "M-13.285 31.904c5.965-2.399 12.4-2.296 17.973-.197 11.154 4.21 17.645 1.606 22.146-8.884 2.242-5.264 6.62-9.722 12.592-12.124 11.941-4.795 25.726.408 30.805 11.63 5.08 11.224-.48 24.198-12.418 29-5.986 2.406-12.432 2.294-18.015.176-11.135-4.187-17.613-1.574-22.109 8.929-2.248 5.252-6.625 9.694-12.59 12.094-11.932 4.798-25.717-.405-30.793-11.622-5.076-11.216.477-24.203 12.409-29.002"
                        }),
                        /*#__PURE__*/ (0, $523HE$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.489,
                            cy: 16.487,
                            fill: "url(#99f169__g)",
                            rx: 17.489,
                            ry: 16.487,
                            transform: "matrix(1 -.00132 .00148 1 23.344 -5.033)"
                        })
                    ]
                })
            })
        ]
    });
const $ef8e0f1b2519e0b0$var$ForwardRef = /*#__PURE__*/ (0, $523HE$react.forwardRef)($ef8e0f1b2519e0b0$var$SvgComponent);
var $ef8e0f1b2519e0b0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ef8e0f1b2519e0b0$var$ForwardRef);
