// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_mailOpen_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $hYFZR$jsxs, jsx as $hYFZR$jsx} from "react/jsx-runtime";
import {forwardRef as $hYFZR$forwardRef} from "react";





const $4baf0b7026c07a01$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hYFZR$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hYFZR$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hYFZR$jsxs)("linearGradient", {
                        id: "051fef__d",
                        x1: -14.982,
                        x2: 120.749,
                        y1: 80.605,
                        y2: -1.972,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hYFZR$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $hYFZR$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hYFZR$jsxs)("linearGradient", {
                        id: "051fef__e",
                        x1: 93.945,
                        x2: 35.717,
                        y1: 21.377,
                        y2: 83.989,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hYFZR$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hYFZR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hYFZR$jsxs)("linearGradient", {
                        id: "051fef__f",
                        x1: 23.18,
                        x2: 126.403,
                        y1: 92.63,
                        y2: 34.806,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hYFZR$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hYFZR$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hYFZR$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hYFZR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hYFZR$jsxs)("radialGradient", {
                        id: "051fef__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 21.982 31.22)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hYFZR$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $hYFZR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hYFZR$jsxs)("radialGradient", {
                        id: "051fef__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 31.768 52.85)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hYFZR$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hYFZR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hYFZR$jsx)("clipPath", {
                        id: "051fef__b",
                        children: /*#__PURE__*/ (0, $hYFZR$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6,
                            y: 6,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hYFZR$jsx)("clipPath", {
                id: "051fef__a",
                children: /*#__PURE__*/ (0, $hYFZR$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M35.958 28.848c-3.972 0-7.2 3.228-7.2 7.2v2.946l-9.684-6.072 26.472-16.566c1.566-.978 3.546-.978 5.106.006l26.28 16.554-9.69 6.078v-2.946c0-3.972-3.228-7.2-7.2-7.2zm0 4.8c-1.325 0-2.4 1.074-2.4 2.4V42l13.17 8.262c.775.486 1.777.486 2.55 0L62.449 42v-5.958c0-1.326-1.073-2.4-2.4-2.4H35.966zm15.87 20.682c-1.164.726-2.495 1.092-3.828 1.092-1.331 0-2.663-.366-3.828-1.092l-27.36-17.16v33.612c0 5.304 4.296 9.606 9.6 9.606H69.59c5.304 0 9.6-4.302 9.6-9.606V37.17z"
                })
            }),
            /*#__PURE__*/ (0, $hYFZR$jsx)("g", {
                clipPath: "url(#051fef__a)",
                children: /*#__PURE__*/ (0, $hYFZR$jsxs)("g", {
                    clipPath: "url(#051fef__b)",
                    children: [
                        /*#__PURE__*/ (0, $hYFZR$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6,
                            y: 6,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $hYFZR$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6 6h83.485v83.485H6z"
                        }),
                        /*#__PURE__*/ (0, $hYFZR$jsx)("circle", {
                            cx: 53.203,
                            cy: 9.238,
                            r: 36.321,
                            fill: "url(#051fef__c)",
                            transform: "rotate(-.08 53.203 9.238)"
                        }),
                        /*#__PURE__*/ (0, $hYFZR$jsx)("path", {
                            fill: "url(#051fef__d)",
                            d: "M37.863-8.748c-6.411-3.967-14.535-3.991-20.97-.062l-64.366 39.3c-6.304 3.848-6.314 12.933-.019 16.797l63.94 39.244c6.426 3.943 14.55 3.938 20.97-.015l63.808-39.28c6.262-3.855 6.274-12.89.019-16.76z"
                        }),
                        /*#__PURE__*/ (0, $hYFZR$jsx)("path", {
                            fill: "url(#051fef__e)",
                            d: "M18.033 45.636C19.46 14.47 45.883-9.64 77.05-8.214s55.277 27.85 53.85 59.018-27.85 55.277-59.017 53.85-55.277-27.85-53.85-59.018"
                        }),
                        /*#__PURE__*/ (0, $hYFZR$jsx)("path", {
                            fill: "url(#051fef__f)",
                            d: "M-27.824 66.187c12.389-5.285 25.75-5.059 37.326-.436 23.164 9.274 36.643 3.54 45.991-19.57 4.657-11.598 13.748-21.42 26.15-26.71 24.799-10.563 53.427.898 63.975 25.623s-.997 53.308-25.79 63.884c-12.43 5.303-25.818 5.055-37.411.39-23.127-9.224-36.58-3.469-45.916 19.67-4.668 11.569-13.757 21.357-26.147 26.643-24.778 10.57-53.406-.891-63.948-25.602s.992-53.322 25.77-63.892"
                        }),
                        /*#__PURE__*/ (0, $hYFZR$jsx)("circle", {
                            cx: 84.617,
                            cy: 21.082,
                            r: 36.321,
                            fill: "url(#051fef__g)",
                            transform: "rotate(-.08 84.617 21.082)"
                        })
                    ]
                })
            })
        ]
    });
const $4baf0b7026c07a01$var$ForwardRef = /*#__PURE__*/ (0, $hYFZR$forwardRef)($4baf0b7026c07a01$var$SvgComponent);
var $4baf0b7026c07a01$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($4baf0b7026c07a01$var$ForwardRef);


export {$4baf0b7026c07a01$export$2e2bcd8739ae039 as default};
