// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_laptop_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $kZGkN$reactjsxruntime = require("react/jsx-runtime");
var $kZGkN$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9ea7b16c81c36399$export$2e2bcd8739ae039);




const $9ea7b16c81c36399$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4cf324__d",
                        x1: -21.18,
                        x2: 192.69,
                        y1: 132.866,
                        y2: -5.163,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4cf324__e",
                        x1: 156.261,
                        x2: 67.379,
                        y1: 41.913,
                        y2: 143.298,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4cf324__f",
                        x1: 40.989,
                        x2: 204.255,
                        y1: 151.332,
                        y2: 54.311,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4cf324__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 55.7757 -59.1674 0 59.167 55.776)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4cf324__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 55.7757 -59.1674 0 59.167 55.776)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("clipPath", {
                        id: "4cf324__b",
                        children: /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("rect", {
                            width: 136,
                            height: 128.204,
                            x: 13,
                            y: 18.298,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("clipPath", {
                id: "4cf324__a",
                children: /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M136 110h-8V48.22c0-8.96-7.16-16.22-16-16.22H48c-8.84 0-16 7.26-16 16.22V110h-8c-4.97 0-9 4.03-9 9s4.03 9 9 9h112c4.97 0 9-4.03 9-9s-4.03-9-9-9M80 49c-2.21 0-4-1.79-4-4s1.79-4 4-4 4 1.79 4 4-1.79 4-4 4"
                })
            }),
            /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("g", {
                clipPath: "url(#4cf324__a)",
                children: /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#4cf324__b)",
                    children: [
                        /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("rect", {
                            width: 136,
                            height: 128.204,
                            x: 13,
                            y: 18.298,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M13 18.298h136v128.204H13z"
                        }),
                        /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("ellipse", {
                            cx: 59.167,
                            cy: 55.776,
                            fill: "url(#4cf324__c)",
                            rx: 59.167,
                            ry: 55.776,
                            transform: "matrix(1 -.00132 .00148 1 30.648 -32.427)"
                        }),
                        /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("path", {
                            fill: "url(#4cf324__d)",
                            d: "M64.907-4.349c-10.444-6.092-23.678-6.13-34.161-.096l-104.854 60.35c-10.27 5.911-10.286 19.862-.031 25.796l104.16 60.264c10.468 6.056 23.702 6.049 34.16-.021l103.947-60.321c10.201-5.92 10.22-19.794.031-25.737z"
                        }),
                        /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("path", {
                            fill: "url(#4cf324__e)",
                            d: "M32.598 79.166C34.923 31.304 77.967-5.72 128.74-3.529s90.048 42.768 87.724 90.63c-2.325 47.863-45.369 84.886-96.141 82.695s-90.048-42.768-87.724-90.63"
                        }),
                        /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("path", {
                            fill: "url(#4cf324__f)",
                            d: "M-42.1 110.724c20.183-8.116 41.95-7.768 60.807-.669 37.735 14.241 59.692 5.436 74.92-30.054 7.586-17.809 22.396-32.892 42.601-41.017 40.397-16.22 87.033 1.38 104.216 39.348 17.182 37.969-1.625 81.863-42.012 98.104-20.25 8.144-42.058 7.763-60.945.6-37.674-14.165-59.589-5.327-74.798 30.206-7.605 17.766-22.412 32.797-42.594 40.913-40.364 16.233-87-1.368-104.174-39.315-17.173-37.947 1.615-81.883 41.98-98.116"
                        }),
                        /*#__PURE__*/ (0, $kZGkN$reactjsxruntime.jsx)("ellipse", {
                            cx: 59.167,
                            cy: 55.776,
                            fill: "url(#4cf324__g)",
                            rx: 59.167,
                            ry: 55.776,
                            transform: "matrix(1 -.00132 .00148 1 81.824 -14.238)"
                        })
                    ]
                })
            })
        ]
    });
const $9ea7b16c81c36399$var$ForwardRef = /*#__PURE__*/ (0, $kZGkN$react.forwardRef)($9ea7b16c81c36399$var$SvgComponent);
var $9ea7b16c81c36399$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9ea7b16c81c36399$var$ForwardRef);
