// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_channel_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $kJgli$reactjsxruntime = require("react/jsx-runtime");
var $kJgli$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7ab01d09a977df5a$export$2e2bcd8739ae039);




const $7ab01d09a977df5a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsxs)("linearGradient", {
                        id: "dd25c5__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsxs)("linearGradient", {
                        id: "dd25c5__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsxs)("linearGradient", {
                        id: "dd25c5__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsxs)("radialGradient", {
                        id: "dd25c5__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsxs)("radialGradient", {
                        id: "dd25c5__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("clipPath", {
                        id: "dd25c5__b",
                        children: /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("clipPath", {
                id: "dd25c5__a",
                children: /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M56.933 48.045h.008l8.451 14.638c-3.96 3.344-6.758 8.02-7.704 13.337H40.805c-1.38-2.402-3.972-4.02-6.942-4.02-4.418 0-8 3.582-8 8s3.582 8 8 8c2.953 0 5.532-1.6 6.918-3.98h16.907c.945 5.318 3.744 9.995 7.704 13.34l-8.428 14.596h-.03c-4.42 0-8 3.582-8 8s3.58 8 8 8c4.417 0 8-3.582 8-8 0-1.446-.385-2.803-1.056-3.974l8.439-14.617c2.405.868 5 1.342 7.705 1.342s5.3-.474 7.705-1.342l8.418 14.582c-.684 1.179-1.075 2.548-1.075 4.009 0 4.418 3.582 8 8 8s8-3.582 8-8c0-4.415-3.577-7.995-7.991-8L94.652 97.36c3.96-3.345 6.758-8.022 7.704-13.34h16.865c1.386 2.38 3.966 3.98 6.918 3.98 4.418 0 8-3.582 8-8s-3.582-8-8-8c-2.969 0-5.561 1.618-6.941 4.02h-16.842c-.946-5.317-3.745-9.993-7.705-13.337l8.451-14.638c4.404-.018 7.968-3.593 7.968-8 0-4.418-3.582-8-8-8s-8 3.582-8 8c0 1.47.396 2.846 1.087 4.03l-8.431 14.603c-2.405-.869-5-1.342-7.704-1.342-2.705 0-5.3.473-7.705 1.342L63.866 44.04c.679-1.176 1.067-2.54 1.067-3.995 0-4.418-3.582-8-8-8s-8 3.582-8 8 3.582 8 8 8"
                })
            }),
            /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("g", {
                clipPath: "url(#dd25c5__a)",
                children: /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#dd25c5__b)",
                    children: [
                        /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#dd25c5__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("path", {
                            fill: "url(#dd25c5__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("path", {
                            fill: "url(#dd25c5__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("path", {
                            fill: "url(#dd25c5__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $kJgli$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#dd25c5__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $7ab01d09a977df5a$var$ForwardRef = /*#__PURE__*/ (0, $kJgli$react.forwardRef)($7ab01d09a977df5a$var$SvgComponent);
var $7ab01d09a977df5a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7ab01d09a977df5a$var$ForwardRef);
