// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_dropToUpload_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $1N41r$reactjsxruntime = require("react/jsx-runtime");
var $1N41r$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $02f4216df6e07cc2$export$2e2bcd8739ae039);




const $02f4216df6e07cc2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8c4c5f__d",
                        x1: -9.883,
                        x2: 61.815,
                        y1: 40.61,
                        y2: -3.011,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8c4c5f__e",
                        x1: 47.656,
                        x2: 16.898,
                        y1: 9.323,
                        y2: 42.397,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8c4c5f__f",
                        x1: 10.275,
                        x2: 64.802,
                        y1: 46.962,
                        y2: 16.417,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8c4c5f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.612 14.523)scale(19.1859)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8c4c5f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.78 25.948)scale(19.1859)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("clipPath", {
                        id: "8c4c5f__b",
                        children: /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("rect", {
                            width: 44.1,
                            height: 44.1,
                            x: 1.199,
                            y: 1.2,
                            fill: "#fff",
                            rx: 1.331
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("clipPath", {
                id: "8c4c5f__a",
                children: /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M15.523 6.58h8.221c.843 0 1.652.335 2.248.931l8.736 8.736c.596.596.931 1.405.931 2.248v12.46c0 1.756-1.423 3.18-3.18 3.18H15.524c-1.756 0-3.18-1.424-3.18-3.18V9.76c0-1.755 1.424-3.179 3.18-3.179m9.299 9.188c.001-.586-.472-1.061-1.057-1.062-.586-.002-1.061.472-1.063 1.057l-.023 10.495-3.583-2.85c-.458-.364-1.125-.288-1.49.17-.364.458-.288 1.125.17 1.489l5.338 4.245c.389.31.94.307 1.326-.004l5.26-4.246c.455-.367.527-1.035.159-1.49s-1.035-.527-1.49-.16l-3.57 2.882zM6.72 22.658c0-.53.43-.96.96-.96h1.764c.53 0 .96-.43.96-.96s-.43-.96-.96-.96H7.681c-1.59 0-2.88 1.289-2.88 2.88v1.66c0 .53.43.96.96.96s.96-.43.96-.96zm31.836-2.88c-.53 0-.96.43-.96.96s.43.96.96.96h1.764c.53 0 .96.43.96.96v1.66c0 .53.43.96.96.96s.96-.43.96-.96v-1.66c0-1.591-1.29-2.88-2.88-2.88zm4.644 8.899c0-.53-.43-.96-.96-.96s-.96.43-.96.96v3.32c0 .531.43.96.96.96s.96-.429.96-.96zm-36.48 0c0-.53-.43-.96-.96-.96s-.96.43-.96.96v3.32c0 .531.43.96.96.96s.96-.429.96-.96zm0 7.68c0-.53-.43-.96-.96-.96s-.96.43-.96.96v1.66c0 1.59 1.29 2.88 2.88 2.88h1.764c.53 0 .96-.43.96-.96s-.43-.96-.96-.96H7.681c-.53 0-.96-.43-.96-.96zm36.48 0c0-.53-.43-.96-.96-.96s-.96.43-.96.96v1.66c0 .53-.43.96-.96.96h-1.764c-.53 0-.96.43-.96.96s.43.96.96.96h1.764c1.59 0 2.88-1.29 2.88-2.88zm-29.124 2.62c-.53 0-.96.43-.96.96s.43.96.96.96h3.528c.53 0 .96-.43.96-.96s-.43-.96-.96-.96zm8.16 0c-.53 0-.96.43-.96.96s.43.96.96.96h3.528c.53 0 .96-.43.96-.96s-.43-.96-.96-.96zm8.16 0c-.53 0-.96.43-.96.96s.43.96.96.96h3.528c.53 0 .96-.43.96-.96s-.43-.96-.96-.96z"
                })
            }),
            /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("g", {
                clipPath: "url(#8c4c5f__a)",
                children: /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#8c4c5f__b)",
                    children: [
                        /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("rect", {
                            width: 44.1,
                            height: 44.1,
                            x: 1.199,
                            y: 1.2,
                            fill: "#FFECCF",
                            rx: 1.331
                        }),
                        /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M1.2 1.2h44.1v44.1H1.2z"
                        }),
                        /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("circle", {
                            cx: 26.135,
                            cy: 2.911,
                            r: 19.186,
                            fill: "url(#8c4c5f__c)",
                            transform: "rotate(-.08 26.135 2.911)"
                        }),
                        /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("path", {
                            fill: "url(#8c4c5f__d)",
                            d: "M18.032-6.59C14.645-8.686 10.354-8.7 6.954-6.623l-34 20.76c-3.33 2.032-3.336 6.832-.01 8.872L6.72 43.74c3.394 2.084 7.685 2.081 11.076-.007l33.707-20.75c3.307-2.036 3.313-6.808.01-8.852z"
                        }),
                        /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("path", {
                            fill: "url(#8c4c5f__e)",
                            d: "M7.556 22.138C8.31 5.674 22.268-7.062 38.732-6.308S67.93 8.404 67.177 24.867s-14.711 29.2-31.175 28.446-29.2-14.711-28.446-31.175"
                        }),
                        /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("path", {
                            fill: "url(#8c4c5f__f)",
                            d: "M-16.667 32.993c6.544-2.792 13.602-2.672 19.717-.23 12.236 4.899 19.356 1.87 24.294-10.338 2.46-6.126 7.262-11.314 13.814-14.11 13.1-5.579 28.222.476 33.794 13.536s-.527 28.16-13.623 33.746c-6.567 2.801-13.638 2.67-19.763.206-12.216-4.872-19.322-1.832-24.254 10.39-2.466 6.112-7.267 11.282-13.812 14.074-13.088 5.584-28.21-.47-33.78-13.523-5.568-13.054.524-28.167 13.613-33.75"
                        }),
                        /*#__PURE__*/ (0, $1N41r$reactjsxruntime.jsx)("circle", {
                            cx: 42.728,
                            cy: 9.168,
                            r: 19.186,
                            fill: "url(#8c4c5f__g)",
                            transform: "rotate(-.08 42.728 9.168)"
                        })
                    ]
                })
            })
        ]
    });
const $02f4216df6e07cc2$var$ForwardRef = /*#__PURE__*/ (0, $1N41r$react.forwardRef)($02f4216df6e07cc2$var$SvgComponent);
var $02f4216df6e07cc2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($02f4216df6e07cc2$var$ForwardRef);
