// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_imageStack_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6IIPb$jsxs, jsx as $6IIPb$jsx} from "react/jsx-runtime";
import {forwardRef as $6IIPb$forwardRef} from "react";





const $bdbf618c3ee21275$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6IIPb$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6IIPb$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6IIPb$jsxs)("linearGradient", {
                        id: "5e3a86__d",
                        x1: -8.804,
                        x2: 115.365,
                        y1: 78.741,
                        y2: -1.397,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6IIPb$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $6IIPb$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6IIPb$jsxs)("linearGradient", {
                        id: "5e3a86__e",
                        x1: 94.214,
                        x2: 42.61,
                        y1: 25.934,
                        y2: 84.797,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6IIPb$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6IIPb$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6IIPb$jsxs)("linearGradient", {
                        id: "5e3a86__f",
                        x1: 27.289,
                        x2: 122.08,
                        y1: 89.461,
                        y2: 33.132,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6IIPb$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6IIPb$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6IIPb$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6IIPb$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6IIPb$jsxs)("radialGradient", {
                        id: "5e3a86__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.3826 -34.3518 0 34.352 32.383)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6IIPb$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $6IIPb$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6IIPb$jsxs)("radialGradient", {
                        id: "5e3a86__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.3826 -34.3518 0 34.352 32.383)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6IIPb$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6IIPb$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6IIPb$jsx)("clipPath", {
                        id: "5e3a86__b",
                        children: /*#__PURE__*/ (0, $6IIPb$jsx)("rect", {
                            width: 78.96,
                            height: 74.433,
                            x: 11.039,
                            y: 12.224,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6IIPb$jsx)("clipPath", {
                id: "5e3a86__a",
                children: /*#__PURE__*/ (0, $6IIPb$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M75.256 29.082v29.4c0 5.304-4.296 9.6-9.6 9.6h-44.4c-5.304 0-9.6-4.296-9.6-9.6v-29.4c0-5.304 4.296-9.6 9.6-9.6h44.4c5.304 0 9.6 4.296 9.6 9.6m-8.424 4.128c0-2.892-2.346-5.238-5.238-5.238s-5.238 2.346-5.238 5.238 2.346 5.238 5.238 5.238 5.238-2.346 5.238-5.238M17.386 57.732c-1.536 2.394.18 5.544 3.03 5.544v-.006h45.708c2.85 0 4.566-3.15 3.03-5.544l-7.068-11.01c-1.416-2.208-4.644-2.208-6.06 0L48.01 59.208 35.254 39.336c-1.416-2.208-4.644-2.208-6.06 0zm67.025-15.79c0-1.104-.896-2-2-2s-2 .896-2 2v22.014c0 4.899-3.972 8.87-8.87 8.87H35.666c-1.105 0-2 .896-2 2s.895 2 2 2H71.54c7.108 0 12.87-5.762 12.87-12.87z"
                })
            }),
            /*#__PURE__*/ (0, $6IIPb$jsx)("g", {
                clipPath: "url(#5e3a86__a)",
                children: /*#__PURE__*/ (0, $6IIPb$jsxs)("g", {
                    clipPath: "url(#5e3a86__b)",
                    children: [
                        /*#__PURE__*/ (0, $6IIPb$jsx)("rect", {
                            width: 78.96,
                            height: 74.433,
                            x: 11.039,
                            y: 12.224,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $6IIPb$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 12.224H90v74.433H11.04z"
                        }),
                        /*#__PURE__*/ (0, $6IIPb$jsx)("ellipse", {
                            cx: 34.352,
                            cy: 32.383,
                            fill: "url(#5e3a86__c)",
                            rx: 34.352,
                            ry: 32.383,
                            transform: "matrix(1 -.00132 .00148 1 21.285 -17.227)"
                        }),
                        /*#__PURE__*/ (0, $6IIPb$jsx)("path", {
                            fill: "url(#5e3a86__d)",
                            d: "M41.176-.925C35.112-4.462 27.43-4.484 21.343-.98l-60.877 35.04c-5.962 3.43-5.972 11.53-.018 14.975l60.474 34.99c6.078 3.515 13.76 3.51 19.833-.013l60.35-35.022c5.922-3.437 5.933-11.492.018-14.942z"
                        }),
                        /*#__PURE__*/ (0, $6IIPb$jsx)("path", {
                            fill: "url(#5e3a86__e)",
                            d: "M22.417 47.563C23.767 19.775 48.757-1.72 78.235-.448s52.281 24.83 50.931 52.618c-1.349 27.789-26.34 49.284-55.818 48.012s-52.28-24.83-50.931-52.619"
                        }),
                        /*#__PURE__*/ (0, $6IIPb$jsx)("path", {
                            fill: "url(#5e3a86__f)",
                            d: "M-20.95 65.885c11.717-4.712 24.354-4.51 35.303-.388 21.908 8.268 34.656 3.156 43.498-17.45 4.404-10.339 13.002-19.096 24.733-23.813 23.454-9.418 50.53.8 60.506 22.845s-.943 47.528-24.391 56.958c-11.757 4.728-24.419 4.506-35.384.348-21.873-8.225-34.596-3.093-43.427 17.537-4.415 10.315-13.012 19.041-24.73 23.754-23.434 9.424-50.51-.794-60.481-22.826-9.97-22.032.938-47.54 24.373-56.965"
                        }),
                        /*#__PURE__*/ (0, $6IIPb$jsx)("ellipse", {
                            cx: 34.352,
                            cy: 32.383,
                            fill: "url(#5e3a86__g)",
                            rx: 34.352,
                            ry: 32.383,
                            transform: "matrix(1 -.00132 .00148 1 50.996 -6.666)"
                        })
                    ]
                })
            })
        ]
    });
const $bdbf618c3ee21275$var$ForwardRef = /*#__PURE__*/ (0, $6IIPb$forwardRef)($bdbf618c3ee21275$var$SvgComponent);
var $bdbf618c3ee21275$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($bdbf618c3ee21275$var$ForwardRef);


export {$bdbf618c3ee21275$export$2e2bcd8739ae039 as default};
