// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_rotateCCW_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $9eOya$reactjsxruntime = require("react/jsx-runtime");
var $9eOya$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4119095d5067ce4a$export$2e2bcd8739ae039);




const $4119095d5067ce4a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8b3732__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8b3732__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8b3732__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8b3732__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8b3732__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("clipPath", {
                        id: "8b3732__b",
                        children: /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("clipPath", {
                id: "8b3732__a",
                children: /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M79.002 48c0-17.12-13.88-31-31-31-12.189 0-22.738 7.035-27.803 17.27l9.734-3.13c1.052-.337 2.178.241 2.517 1.293.338 1.051-.24 2.178-1.292 2.516L17.1 39.469c-1.051.338-2.178-.24-2.516-1.292l-4.52-14.056c-.338-1.052.24-2.179 1.292-2.517s2.178.24 2.516 1.292l2.953 9.181C22.62 20.754 34.403 13 48.002 13c19.33 0 35 15.67 35 35s-15.67 35-35 35c-15.118 0-27.992-9.583-32.888-23-.38-1.038.155-2.186 1.193-2.565 1.037-.378 2.185.156 2.564 1.194C23.21 70.517 34.618 79 48.001 79c17.122 0 31-13.88 31-31"
                })
            }),
            /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("g", {
                clipPath: "url(#8b3732__a)",
                children: /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#8b3732__b)",
                    children: [
                        /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#8b3732__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("path", {
                            fill: "url(#8b3732__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("path", {
                            fill: "url(#8b3732__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("path", {
                            fill: "url(#8b3732__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $9eOya$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#8b3732__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $4119095d5067ce4a$var$ForwardRef = /*#__PURE__*/ (0, $9eOya$react.forwardRef)($4119095d5067ce4a$var$SvgComponent);
var $4119095d5067ce4a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4119095d5067ce4a$var$ForwardRef);
