// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_lightbulbRays_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gMSU8$reactjsxruntime = require("react/jsx-runtime");
var $gMSU8$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7a8a4f3458825997$export$2e2bcd8739ae039);




const $7a8a4f3458825997$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "225c0c__d",
                        x1: -19.921,
                        x2: 186.087,
                        y1: 131.011,
                        y2: -1.943,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "225c0c__e",
                        x1: 150.995,
                        x2: 65.38,
                        y1: 43.401,
                        y2: 141.06,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "225c0c__f",
                        x1: 39.959,
                        x2: 197.223,
                        y1: 148.798,
                        y2: 55.343,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsxs)("radialGradient", {
                        id: "225c0c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsxs)("radialGradient", {
                        id: "225c0c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("clipPath", {
                        id: "225c0c__b",
                        children: /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("clipPath", {
                id: "225c0c__a",
                children: /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M84 29.95c0 2.21-1.79 4-4 4s-4-1.79-4-4v-6.02c0-2.21 1.79-4 4-4s4 1.79 4 4zm-6.2 13.73c-14.85 1.06-26.87 13.17-27.97 28.16-.81 11.14 4.28 21.12 12.45 27.09 2.65 1.94 4.27 5.01 4.27 8.32V114h26.53c.072 0 .144.01.216.021q.073.012.144.019v-6.78c0-3.23 1.48-6.31 4.08-8.19 7.7-5.54 12.72-14.63 12.72-24.91 0-17.62-14.75-31.75-32.45-30.49zm19.37 30.9c-2.21 0-4-1.79-4-4 0-5.01-4.44-9.08-9.9-9.08-2.21 0-4-1.79-4-4s1.79-4 4-4c9.87 0 17.9 7.66 17.9 17.08 0 2.21-1.79 4-4 4M66.56 122v4.8c0 2.81 2.26 5.09 5.04 5.09h16.81c2.78 0 5.04-2.28 5.04-5.09v-4.84q-.07.007-.143.018c-.072.011-.144.022-.216.022zM29.95 69.98h6.02c2.21 0 4 1.79 4 4s-1.79 4-4 4h-6.02c-2.21 0-4-1.79-4-4s1.79-4 4-4m100.09 0h-6.02c-2.21 0-4 1.79-4 4s1.79 4 4 4h6.02c2.21 0 4-1.79 4-4s-1.79-4-4-4m-17.479-34.22-4.25 4.25c-1.56 1.56-1.56 4.09 0 5.66.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17l4.25-4.25c1.56-1.56 1.56-4.09 0-5.66-1.56-1.56-4.09-1.56-5.66 0m-65.112.01 4.25 4.25c1.56 1.57 1.56 4.1 0 5.66-.78.78-1.8 1.17-2.83 1.17s-2.05-.39-2.83-1.17l-4.25-4.25c-1.56-1.57-1.56-4.1 0-5.66 1.57-1.56 4.1-1.56 5.66 0"
                })
            }),
            /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("g", {
                clipPath: "url(#225c0c__a)",
                children: /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#225c0c__b)",
                    children: [
                        /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M13 20.654h131v123.491H13z"
                        }),
                        /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#225c0c__c)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 30 -28.206)"
                        }),
                        /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("path", {
                            fill: "url(#225c0c__d)",
                            d: "M63.002-1.16C52.94-7.027 40.194-7.063 30.096-1.251l-101 58.132c-9.891 5.693-9.907 19.132-.03 24.847L29.4 139.776c10.082 5.833 22.83 5.826 32.904-.021l100.125-58.103c9.826-5.703 9.844-19.067.03-24.79z"
                        }),
                        /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("path", {
                            fill: "url(#225c0c__e)",
                            d: "M31.878 79.285C34.118 33.183 75.58-2.48 124.485-.369c48.907 2.11 86.738 41.195 84.499 87.298s-43.701 81.765-92.607 79.655c-48.906-2.111-86.738-41.196-84.499-87.299"
                        }),
                        /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("path", {
                            fill: "url(#225c0c__f)",
                            d: "M-40.075 109.683c19.44-7.818 40.406-7.483 58.57-.645 36.349 13.718 57.498 5.237 72.167-28.949 7.307-17.154 21.572-31.682 41.035-39.509 38.911-15.624 83.833 1.33 100.384 37.902s-1.565 78.853-40.467 94.497c-19.506 7.845-40.512 7.477-58.705.577-36.288-13.644-57.398-5.13-72.048 29.096-7.325 17.113-21.588 31.592-41.028 39.41-38.88 15.636-83.803-1.318-100.344-37.87-16.542-36.552 1.555-78.873 40.436-94.509"
                        }),
                        /*#__PURE__*/ (0, $gMSU8$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#225c0c__g)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 79.293 -10.685)"
                        })
                    ]
                })
            })
        ]
    });
const $7a8a4f3458825997$var$ForwardRef = /*#__PURE__*/ (0, $gMSU8$react.forwardRef)($7a8a4f3458825997$var$SvgComponent);
var $7a8a4f3458825997$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7a8a4f3458825997$var$ForwardRef);
