// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_graphBarChart_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $hJinF$reactjsxruntime = require("react/jsx-runtime");
var $hJinF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2d18060fab5b3261$export$2e2bcd8739ae039);




const $2d18060fab5b3261$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1ff87e__d",
                        x1: -9.655,
                        x2: 58.408,
                        y1: 40.96,
                        y2: -2.966,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1ff87e__e",
                        x1: 46.813,
                        x2: 18.527,
                        y1: 12.015,
                        y2: 44.28,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1ff87e__f",
                        x1: 10.129,
                        x2: 62.087,
                        y1: 46.837,
                        y2: 15.961,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsxs)("radialGradient", {
                        id: "1ff87e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.7502 -18.8296 0 18.83 17.75)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsxs)("radialGradient", {
                        id: "1ff87e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.7502 -18.8296 0 18.83 17.75)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("clipPath", {
                        id: "1ff87e__b",
                        children: /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("rect", {
                            width: 43.281,
                            height: 40.8,
                            x: 1.223,
                            y: 4.5,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("clipPath", {
                id: "1ff87e__a",
                children: /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M7.263 40.8H40.8c.663 0 1.2.537 1.2 1.2s-.537 1.2-1.2 1.2H6.063c-.663 0-1.2-.537-1.2-1.2V7.2c0-.663.537-1.2 1.2-1.2s1.2.537 1.2 1.2zM13.2 21.315h1.2c1.326 0 2.4 1.075 2.4 2.4V34.8c0 1.326-1.075 2.4-2.4 2.4h-1.2c-1.325 0-2.4-1.075-2.4-2.4V23.715c0-1.325 1.074-2.4 2.4-2.4M36 28.8h-1.2c-1.325 0-2.4 1.075-2.4 2.4v3.6c0 1.326 1.075 2.4 2.4 2.4H36c1.325 0 2.4-1.075 2.4-2.4v-3.6c0-1.325-1.075-2.4-2.4-2.4M24 12h1.2c1.326 0 2.4 1.075 2.4 2.4v20.4c0 1.326-1.075 2.4-2.4 2.4H24c-1.326 0-2.4-1.075-2.4-2.4V14.4c0-1.325 1.074-2.4 2.4-2.4"
                })
            }),
            /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("g", {
                clipPath: "url(#1ff87e__a)",
                children: /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#1ff87e__b)",
                    children: [
                        /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("rect", {
                            width: 43.281,
                            height: 40.8,
                            x: 1.223,
                            y: 4.5,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M1.223 4.5h43.28v40.8H1.224z"
                        }),
                        /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("ellipse", {
                            cx: 18.83,
                            cy: 17.75,
                            fill: "url(#1ff87e__c)",
                            rx: 18.83,
                            ry: 17.75,
                            transform: "matrix(1 -.00132 .00148 1 6.84 -11.643)"
                        }),
                        /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("path", {
                            fill: "url(#1ff87e__d)",
                            d: "M17.742-2.707c-3.324-1.94-7.536-1.951-10.872-.031l-33.369 19.206c-3.268 1.881-3.273 6.321-.01 8.21L6.64 43.855c3.331 1.927 7.543 1.925 10.871-.007l33.08-19.197c3.247-1.884 3.253-6.299.01-8.19z"
                        }),
                        /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("path", {
                            fill: "url(#1ff87e__e)",
                            d: "M7.459 23.87C8.199 8.64 21.897-3.143 38.055-2.445c16.158.697 28.657 13.61 27.917 28.842S51.534 53.41 35.376 52.713 6.719 39.103 7.459 23.87"
                        }),
                        /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("path", {
                            fill: "url(#1ff87e__f)",
                            d: "M-16.313 33.914c6.423-2.583 13.35-2.472 19.351-.213 12.009 4.532 18.996 1.73 23.843-9.564 2.414-5.668 7.127-10.468 13.557-13.053 12.856-5.162 27.698.439 33.166 12.522 5.469 12.083-.517 26.052-13.37 31.22-6.444 2.593-13.385 2.471-19.395.191-11.99-4.507-18.964-1.695-23.804 9.613-2.42 5.654-7.132 10.438-13.555 13.02-12.846 5.167-27.688-.434-33.153-12.51s.514-26.06 13.36-31.226"
                        }),
                        /*#__PURE__*/ (0, $hJinF$reactjsxruntime.jsx)("ellipse", {
                            cx: 18.83,
                            cy: 17.75,
                            fill: "url(#1ff87e__g)",
                            rx: 18.83,
                            ry: 17.75,
                            transform: "matrix(1 -.00132 .00148 1 23.125 -5.854)"
                        })
                    ]
                })
            })
        ]
    });
const $2d18060fab5b3261$var$ForwardRef = /*#__PURE__*/ (0, $hJinF$react.forwardRef)($2d18060fab5b3261$var$SvgComponent);
var $2d18060fab5b3261$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2d18060fab5b3261$var$ForwardRef);
