// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_folderShared_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $h4xU2$reactjsxruntime = require("react/jsx-runtime");
var $h4xU2$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $34a59b5ebec66a0f$export$2e2bcd8739ae039);




const $34a59b5ebec66a0f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "21adf3__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "21adf3__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "21adf3__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "21adf3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "21adf3__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("clipPath", {
                        id: "21adf3__b",
                        children: /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("clipPath", {
                id: "21adf3__a",
                children: /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M20 43h51.544l-9.318-11.547C60.466 29.27 57.8 28 54.98 28h-25.7C24.155 28 20 32.119 20 37.2zm0 8h106.154C133.796 51 140 57.247 140 64.941v53.118c0 7.694-6.204 13.941-13.846 13.941H33.846C26.204 132 20 125.753 20 118.059zm52.294 26.334c-.803 0-1.455.651-1.455 1.455 0 .41.165.776.443 1.046 1.588 1.536 1.629 4.069.092 5.656s-4.07 1.628-5.656.091c-1.77-1.714-2.879-4.127-2.879-6.793 0-5.222 4.233-9.455 9.455-9.455 2.21 0 4 1.79 4 4s-1.79 4-4 4m3.852 4.823c0-4.916 3.986-8.901 8.902-8.901s8.902 3.985 8.902 8.901-3.986 8.903-8.902 8.903-8.902-3.986-8.902-8.903M72.11 92.292c.952 1.993.108 4.381-1.886 5.333-1.906.91-3.39 2.804-3.78 5.106-.37 2.178-2.436 3.644-4.614 3.274s-3.643-2.435-3.274-4.613c.823-4.845 3.946-8.944 8.22-10.986 1.994-.952 4.382-.108 5.334 1.886m-1.691 11.399c1.55-6.436 7.782-10.827 14.63-10.827 6.846 0 13.078 4.39 14.629 10.827.977 4.057-2.457 6.977-5.714 6.977H76.133c-3.257 0-6.692-2.92-5.714-6.977"
                })
            }),
            /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("g", {
                clipPath: "url(#21adf3__a)",
                children: /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#21adf3__b)",
                    children: [
                        /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#21adf3__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("path", {
                            fill: "url(#21adf3__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("path", {
                            fill: "url(#21adf3__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("path", {
                            fill: "url(#21adf3__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $h4xU2$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#21adf3__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $34a59b5ebec66a0f$var$ForwardRef = /*#__PURE__*/ (0, $h4xU2$react.forwardRef)($34a59b5ebec66a0f$var$SvgComponent);
var $34a59b5ebec66a0f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($34a59b5ebec66a0f$var$ForwardRef);
