// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_explosion_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2A45y$jsxs, jsx as $2A45y$jsx} from "react/jsx-runtime";
import {forwardRef as $2A45y$forwardRef} from "react";





const $ad6b6da564486cea$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2A45y$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2A45y$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2A45y$jsxs)("linearGradient", {
                        id: "3d125e__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2A45y$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2A45y$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2A45y$jsxs)("linearGradient", {
                        id: "3d125e__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2A45y$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2A45y$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2A45y$jsxs)("linearGradient", {
                        id: "3d125e__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2A45y$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2A45y$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2A45y$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2A45y$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2A45y$jsxs)("radialGradient", {
                        id: "3d125e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2A45y$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2A45y$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2A45y$jsxs)("radialGradient", {
                        id: "3d125e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2A45y$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2A45y$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2A45y$jsx)("clipPath", {
                        id: "3d125e__b",
                        children: /*#__PURE__*/ (0, $2A45y$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2A45y$jsx)("clipPath", {
                id: "3d125e__a",
                children: /*#__PURE__*/ (0, $2A45y$jsx)("path", {
                    fill: "#292929",
                    d: "M19.241 14.645c-1.573-2.912 2.036-5.852 4.57-3.723l22.613 19 13.309-14.694c2.07-2.286 5.828-.335 5.15 2.673l-3.878 17.21h29.483c3.463 0 4.172 4.894.852 5.877l-25.846 7.65 8.141 9.583c1.981 2.332-.391 5.786-3.279 4.774l-12.296-4.31v25.21c0 3.331-4.617 4.181-5.804 1.069l-9.355-24.55c-6.002 3.456-14.548 8.3-19.48 11.09-2.835 1.605-5.751-1.783-3.916-4.358C22.438 63.03 27.193 56.33 30.52 51.5l-18.044-4.63c-3.033-.78-2.995-5.1.05-5.825l18.586-4.428z"
                })
            }),
            /*#__PURE__*/ (0, $2A45y$jsx)("g", {
                clipPath: "url(#3d125e__a)",
                children: /*#__PURE__*/ (0, $2A45y$jsxs)("g", {
                    clipPath: "url(#3d125e__b)",
                    children: [
                        /*#__PURE__*/ (0, $2A45y$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $2A45y$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $2A45y$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#3d125e__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $2A45y$jsx)("path", {
                            fill: "url(#3d125e__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $2A45y$jsx)("path", {
                            fill: "url(#3d125e__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $2A45y$jsx)("path", {
                            fill: "url(#3d125e__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $2A45y$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#3d125e__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $ad6b6da564486cea$var$ForwardRef = /*#__PURE__*/ (0, $2A45y$forwardRef)($ad6b6da564486cea$var$SvgComponent);
var $ad6b6da564486cea$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ad6b6da564486cea$var$ForwardRef);


export {$ad6b6da564486cea$export$2e2bcd8739ae039 as default};
