// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_accessibility_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $dA0TN$jsxs, jsx as $dA0TN$jsx} from "react/jsx-runtime";
import {forwardRef as $dA0TN$forwardRef} from "react";





const $a891eb333c8ae5a0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dA0TN$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dA0TN$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dA0TN$jsxs)("linearGradient", {
                        id: "f659fe__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dA0TN$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $dA0TN$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dA0TN$jsxs)("linearGradient", {
                        id: "f659fe__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dA0TN$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dA0TN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dA0TN$jsxs)("linearGradient", {
                        id: "f659fe__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dA0TN$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dA0TN$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dA0TN$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dA0TN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dA0TN$jsxs)("radialGradient", {
                        id: "f659fe__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dA0TN$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $dA0TN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dA0TN$jsxs)("radialGradient", {
                        id: "f659fe__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dA0TN$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dA0TN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dA0TN$jsx)("clipPath", {
                        id: "f659fe__b",
                        children: /*#__PURE__*/ (0, $dA0TN$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dA0TN$jsx)("clipPath", {
                id: "f659fe__a",
                children: /*#__PURE__*/ (0, $dA0TN$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M80 136c30.928 0 56-25.072 56-56s-25.072-56-56-56-56 25.072-56 56 25.072 56 56 56m8.484-86.296c0 4.686-3.799 8.484-8.485 8.484s-8.485-3.798-8.485-8.484c0-4.687 3.8-8.485 8.485-8.485 4.686 0 8.485 3.798 8.485 8.485M51.087 61.78c.515-1.802 2.394-2.845 4.196-2.33 15.945 4.563 32.842 4.622 48.818.172l.638-.178c1.806-.503 3.677.553 4.18 2.359.503 1.805-.553 3.677-2.359 4.18l-.637.178c-5.1 1.42-10.288 2.414-15.51 2.98-1.029.111-1.82.974-1.82 2.01v14.884q0 .279.076.546c1.371 4.829 2.945 10.39 4.322 15.269.978 3.461 1.856 6.58 2.495 8.853l.76 2.715.041.149c.133.476.272.974.309 1.156.367 1.838-.825 3.626-2.663 3.993-1.838.368-3.626-.824-3.994-2.662.019.09.004.037-.082-.272l-.15-.542-.756-2.701c-.638-2.269-1.516-5.386-2.492-8.845-1.186-4.2-2.516-8.901-3.737-13.205-.244-.86-1.03-1.454-1.923-1.454h-1.6c-.894 0-1.68.594-1.923 1.454-1.221 4.304-2.551 9.006-3.737 13.205-.976 3.459-1.854 6.576-2.492 8.845l-.756 2.701-.15.54v.002c-.086.309-.1.362-.083.272-.367 1.838-2.155 3.03-3.993 2.662s-3.03-2.155-2.662-3.993c.036-.182.175-.68.308-1.156l.041-.149.76-2.715c.639-2.273 1.518-5.392 2.495-8.853 1.377-4.88 2.95-10.44 4.322-15.269q.075-.266.076-.546V71.151c0-1.036-.791-1.898-1.82-2.01-5.448-.59-10.858-1.645-16.169-3.165-1.802-.515-2.845-2.394-2.33-4.196"
                })
            }),
            /*#__PURE__*/ (0, $dA0TN$jsx)("g", {
                clipPath: "url(#f659fe__a)",
                children: /*#__PURE__*/ (0, $dA0TN$jsxs)("g", {
                    clipPath: "url(#f659fe__b)",
                    children: [
                        /*#__PURE__*/ (0, $dA0TN$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $dA0TN$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $dA0TN$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#f659fe__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $dA0TN$jsx)("path", {
                            fill: "url(#f659fe__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $dA0TN$jsx)("path", {
                            fill: "url(#f659fe__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $dA0TN$jsx)("path", {
                            fill: "url(#f659fe__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $dA0TN$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#f659fe__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $a891eb333c8ae5a0$var$ForwardRef = /*#__PURE__*/ (0, $dA0TN$forwardRef)($a891eb333c8ae5a0$var$SvgComponent);
var $a891eb333c8ae5a0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a891eb333c8ae5a0$var$ForwardRef);


export {$a891eb333c8ae5a0$export$2e2bcd8739ae039 as default};
