// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_flagCheckmark_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $eJWHP$jsxs, jsx as $eJWHP$jsx} from "react/jsx-runtime";
import {forwardRef as $eJWHP$forwardRef} from "react";





const $923b9009c03594fa$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eJWHP$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $eJWHP$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $eJWHP$jsxs)("linearGradient", {
                        id: "80346d__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eJWHP$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $eJWHP$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eJWHP$jsxs)("linearGradient", {
                        id: "80346d__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eJWHP$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $eJWHP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eJWHP$jsxs)("linearGradient", {
                        id: "80346d__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eJWHP$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $eJWHP$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $eJWHP$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $eJWHP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eJWHP$jsxs)("radialGradient", {
                        id: "80346d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eJWHP$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $eJWHP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eJWHP$jsxs)("radialGradient", {
                        id: "80346d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eJWHP$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $eJWHP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eJWHP$jsx)("clipPath", {
                        id: "80346d__b",
                        children: /*#__PURE__*/ (0, $eJWHP$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $eJWHP$jsx)("clipPath", {
                id: "80346d__a",
                children: /*#__PURE__*/ (0, $eJWHP$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M23.484 16.219c1.105 0 2 .895 2 2v3.824h41.172c3.314 0 6 2.686 6 6v25.958c0 3.314-2.686 6-6 6H25.484v17.78c0 1.105-.895 2-2 2s-2-.895-2-2V18.22c0-1.105.896-2 2-2M60.573 36.52c.816-.744.875-2.009.13-2.825-.743-.817-2.008-.875-2.825-.131l-.062.057-.137.125-.207.188-.015.014-.015.014-.032.029-.066.06-.07.063-.073.067-.038.035-.04.036-.08.073-.085.077-.021.02-.022.02-.022.02-.023.02-.091.084-.024.021-.023.022-.024.021-.024.022-.048.044-.05.045-.1.093-.105.095-.107.097-.223.204-.058.052-.058.053-.119.108-.245.223-1.07.977-.288.262-.295.268-.3.274-.153.14-.154.14-.154.14-.04.036-.038.035-.078.071-.079.072-.078.072-.04.035-.04.036-.039.036-.04.037-.64.583-.08.074-.081.074-.04.036-.041.037-.04.037-.041.037-.163.149-.163.148-.082.075-.081.074-.041.037-.041.037-.082.075-.082.074-.04.038-.041.037-.164.149-.163.149-.041.037-.041.037-.082.075-.081.074-.082.075-.163.148-.081.074-.082.074-.162.148-.04.037-.04.037-.04.037-.041.036-.08.074-.041.037-.04.036-.08.073-.08.073-.04.037-.04.036-.04.037-.04.036-.08.072-.04.036-.039.036-.158.144-.62.566-.077.07-.038.034-.038.035-.037.034-.038.035-.038.034-.037.034-.075.068-.075.068-.037.034-.037.034-.074.067-.146.133-.072.066-.072.066-.283.258-.276.252-.068.061-.034.03-.033.031-.133.121-.033.03-.032.03-.065.06-.065.058-.031.029-.032.029-.25.227-.238.217-.029.027-.029.026-.028.026-.03.026-.112.103-.055.05-.054.05-.107.097-.026.024-.026.024-.026.023-.026.023-.016.015-5.533-5.16c-.808-.754-2.074-.71-2.827.098s-.71 2.073.099 2.827l6.882 6.418c.762.71 1.941.717 2.711.015l-1.347-1.478 1.347 1.478.002-.001v-.001l.008-.007.03-.027.116-.105.04-.037.044-.04.102-.094.028-.025.015-.013.014-.014.03-.027.016-.014.016-.014.134-.123.073-.066.077-.07.164-.15.043-.04.022-.02.022-.02.022-.02.023-.02.046-.042.094-.086.097-.089.42-.383.029-.025.028-.026.056-.051.116-.105.058-.054.06-.054.03-.027.03-.028.06-.055.062-.056.062-.056.03-.029.032-.028.032-.029.031-.029.032-.029.032-.029.064-.059.033-.03.032-.029.033-.03.033-.03.132-.12.135-.123.137-.125.139-.127.14-.128.072-.065.036-.032.035-.033.072-.066.037-.033.036-.033.146-.133.037-.033.037-.034.074-.067.037-.034.037-.034.075-.069.075-.068.076-.069.038-.034.038-.035.076-.07.62-.565.04-.036.04-.036.039-.036.04-.036.079-.072.04-.037.04-.036.32-.291.646-.59.327-.298.04-.037.041-.037.041-.037.041-.038.164-.149.081-.074.082-.075.041-.037.04-.037.083-.075.04-.037.041-.037.041-.038.041-.037.041-.037.04-.037.082-.075.041-.037.04-.037.042-.037.04-.037.041-.037.04-.037.082-.074.081-.074.04-.037.041-.037.162-.148.04-.036.04-.037.04-.037.041-.036.04-.037.04-.037.08-.073.04-.036.04-.037.08-.072.08-.073.08-.072.039-.036.04-.036.039-.036.04-.036.078-.072.078-.071.04-.036.038-.035.04-.036.038-.035.078-.071.038-.035.039-.036.154-.14.038-.034.038-.035.038-.035.038-.035.038-.034.038-.034.037-.035.038-.034.075-.068.037-.034.038-.034.294-.269.073-.066.036-.033.036-.033.036-.032.035-.033.072-.065.28-.256.069-.062.034-.032.034-.03.034-.032.034-.03.034-.031.034-.03.518-.473.914-.833.199-.181.186-.17.088-.08.084-.077.081-.074.077-.07.074-.068.035-.032.018-.016.017-.015.033-.03.017-.015.016-.015.062-.057.111-.101.095-.087z"
                })
            }),
            /*#__PURE__*/ (0, $eJWHP$jsx)("g", {
                clipPath: "url(#80346d__a)",
                children: /*#__PURE__*/ (0, $eJWHP$jsxs)("g", {
                    clipPath: "url(#80346d__b)",
                    children: [
                        /*#__PURE__*/ (0, $eJWHP$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $eJWHP$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $eJWHP$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#80346d__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $eJWHP$jsx)("path", {
                            fill: "url(#80346d__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $eJWHP$jsx)("path", {
                            fill: "url(#80346d__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $eJWHP$jsx)("path", {
                            fill: "url(#80346d__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $eJWHP$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#80346d__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $923b9009c03594fa$var$ForwardRef = /*#__PURE__*/ (0, $eJWHP$forwardRef)($923b9009c03594fa$var$SvgComponent);
var $923b9009c03594fa$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($923b9009c03594fa$var$ForwardRef);


export {$923b9009c03594fa$export$2e2bcd8739ae039 as default};
