// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_paperclip_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2GU3H$jsxs, jsx as $2GU3H$jsx} from "react/jsx-runtime";
import {forwardRef as $2GU3H$forwardRef} from "react";





const $7c00804de7c6b8d9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2GU3H$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2GU3H$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2GU3H$jsxs)("linearGradient", {
                        id: "5431f6__d",
                        x1: -13.164,
                        x2: 184.35,
                        y1: 126.507,
                        y2: -0.966,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2GU3H$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2GU3H$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2GU3H$jsxs)("linearGradient", {
                        id: "5431f6__e",
                        x1: 150.707,
                        x2: 68.621,
                        y1: 42.509,
                        y2: 136.142,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2GU3H$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2GU3H$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2GU3H$jsxs)("linearGradient", {
                        id: "5431f6__f",
                        x1: 44.245,
                        x2: 195.026,
                        y1: 143.562,
                        y2: 53.96,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2GU3H$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2GU3H$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2GU3H$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2GU3H$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2GU3H$jsxs)("radialGradient", {
                        id: "5431f6__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.5105 -54.6429 0 54.643 51.51)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2GU3H$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2GU3H$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2GU3H$jsxs)("radialGradient", {
                        id: "5431f6__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.5105 -54.6429 0 54.643 51.51)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2GU3H$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2GU3H$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2GU3H$jsx)("clipPath", {
                        id: "5431f6__b",
                        children: /*#__PURE__*/ (0, $2GU3H$jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 20.7,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2GU3H$jsx)("clipPath", {
                id: "5431f6__a",
                children: /*#__PURE__*/ (0, $2GU3H$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    stroke: "#fff",
                    strokeLinecap: "round",
                    strokeLinejoin: "round",
                    strokeWidth: 2,
                    d: "M35.353 89.552c-11.138 11.083-11.137 29.061 0 40.144 11.127 11.072 29.16 11.072 40.287 0l45.068-44.847c1.566-1.558 1.573-4.09.014-5.657-1.558-1.565-4.09-1.572-5.656-.013l-45.07 44.846c-8.005 7.967-20.994 7.967-29 0-7.995-7.956-7.995-20.846 0-28.802l61.724-61.42c5.097-5.07 13.369-5.07 18.466 0 5.085 5.062 5.085 13.259 0 18.32L65.86 107.175c-2.128 2.118-5.587 2.118-7.715 0-2.117-2.107-2.117-5.514 0-7.621l38.67-38.48c1.566-1.558 1.573-4.09.014-5.657-1.558-1.566-4.09-1.572-5.656-.014l-38.67 38.48c-5.26 5.234-5.26 13.729 0 18.963 5.249 5.223 13.751 5.223 19 0l55.325-55.052c8.229-8.189 8.229-21.473 0-29.661-8.217-8.177-21.533-8.177-29.751 0z"
                })
            }),
            /*#__PURE__*/ (0, $2GU3H$jsx)("g", {
                clipPath: "url(#5431f6__a)",
                children: /*#__PURE__*/ (0, $2GU3H$jsxs)("g", {
                    clipPath: "url(#5431f6__b)",
                    children: [
                        /*#__PURE__*/ (0, $2GU3H$jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 20.7,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $2GU3H$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M18.398 20.7h125.6v118.4h-125.6z"
                        }),
                        /*#__PURE__*/ (0, $2GU3H$jsx)("ellipse", {
                            cx: 54.643,
                            cy: 51.511,
                            fill: "url(#5431f6__c)",
                            rx: 54.643,
                            ry: 51.511,
                            transform: "matrix(1 -.00132 .00148 1 34.695 -26.145)"
                        }),
                        /*#__PURE__*/ (0, $2GU3H$jsx)("path", {
                            fill: "url(#5431f6__d)",
                            d: "M66.34-.215C56.692-5.84 44.471-5.876 34.79-.304l-96.836 55.736c-9.483 5.458-9.5 18.343-.028 23.822l96.195 55.657c9.668 5.592 21.89 5.585 31.548-.02l95.998-55.709c9.421-5.467 9.438-18.28.029-23.768z"
                        }),
                        /*#__PURE__*/ (0, $2GU3H$jsx)("path", {
                            fill: "url(#5431f6__e)",
                            d: "M36.5 76.914C38.648 32.712 78.4-1.481 125.29.543c46.89 2.023 83.162 39.497 81.015 83.7-2.147 44.201-41.899 78.394-88.789 76.37s-83.162-39.497-81.015-83.7"
                        }),
                        /*#__PURE__*/ (0, $2GU3H$jsx)("path", {
                            fill: "url(#5431f6__f)",
                            d: "M-32.49 106.059c18.64-7.495 38.741-7.174 56.157-.618 34.85 13.152 55.127 5.021 69.191-27.755 7.006-16.448 20.683-30.377 39.343-37.88 37.308-14.981 80.378 1.274 96.246 36.339 15.869 35.065-1.5 75.602-38.799 90.601-18.701 7.521-38.842 7.169-56.284.554-34.793-13.082-55.032-4.92-69.079 27.896-7.023 16.408-20.697 30.289-39.336 37.785-37.278 14.991-80.348-1.263-96.208-36.309s1.492-75.621 38.77-90.613"
                        }),
                        /*#__PURE__*/ (0, $2GU3H$jsx)("ellipse", {
                            cx: 54.643,
                            cy: 51.511,
                            fill: "url(#5431f6__g)",
                            rx: 54.643,
                            ry: 51.511,
                            transform: "matrix(1 -.00132 .00148 1 81.957 -9.348)"
                        })
                    ]
                })
            })
        ]
    });
const $7c00804de7c6b8d9$var$ForwardRef = /*#__PURE__*/ (0, $2GU3H$forwardRef)($7c00804de7c6b8d9$var$SvgComponent);
var $7c00804de7c6b8d9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7c00804de7c6b8d9$var$ForwardRef);


export {$7c00804de7c6b8d9$export$2e2bcd8739ae039 as default};
