// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_hand_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $jgltN$jsxs, jsx as $jgltN$jsx} from "react/jsx-runtime";
import {forwardRef as $jgltN$forwardRef} from "react";





const $32fb9c3459a5e7c2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jgltN$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jgltN$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jgltN$jsxs)("linearGradient", {
                        id: "80d6b5__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.982,
                        y2: -15.278,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jgltN$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $jgltN$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jgltN$jsxs)("linearGradient", {
                        id: "80d6b5__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jgltN$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jgltN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jgltN$jsxs)("linearGradient", {
                        id: "80d6b5__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jgltN$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jgltN$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jgltN$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jgltN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jgltN$jsxs)("radialGradient", {
                        id: "80d6b5__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jgltN$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $jgltN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jgltN$jsxs)("radialGradient", {
                        id: "80d6b5__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jgltN$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jgltN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jgltN$jsx)("clipPath", {
                        id: "80d6b5__b",
                        children: /*#__PURE__*/ (0, $jgltN$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jgltN$jsx)("clipPath", {
                id: "80d6b5__a",
                children: /*#__PURE__*/ (0, $jgltN$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M30.672 84.479v-.001q-.02-.04-.043-.078c-2.246-3.897-1.657-8.104 1.439-10.43 3.214-2.415 7.762-1.794 10.216 1.397l9.953 12.941c.522.68 1.418.95 2.229.675.81-.276 1.356-1.038 1.356-1.894V43.88c0-4.39 3.54-7.934 7.892-7.934 4.351 0 7.892 3.544 7.892 7.933V71.67c0 1.105.895 2 2 2h.02c1.104 0 2-.895 2-2V34.152c0-4.39 3.54-7.933 7.892-7.933 4.35 0 7.891 3.543 7.891 7.933v37.522c0 1.105.896 2 2 2h.02c1.105 0 2-.895 2-2V38.856c0-4.39 3.541-7.933 7.892-7.933 4.357 0 7.897 3.544 7.897 7.933v35.291c0 1.062.829 1.938 1.889 1.997 1.059.06 1.981-.72 2.099-1.774l.945-8.437.945-8.436.003-.028c.461-4.71 4.646-7.063 8.518-6.302 3.067.602 5.521 3.35 5.056 7.881l-4.199 29.36q-.044.133-.07.275c-3.376 18.755-8.921 29.945-15.947 36.47-6.971 6.475-15.698 8.628-26.125 8.628-6.757 0-13.481-2.907-19.866-7.496-6.373-4.581-12.26-10.732-17.294-16.953-5.027-6.212-9.158-12.435-12.034-17.11-1.437-2.335-2.558-4.279-3.317-5.636-.38-.678-.67-1.21-.863-1.57q-.145-.27-.217-.407l-.053-.101-.013-.024z"
                })
            }),
            /*#__PURE__*/ (0, $jgltN$jsx)("g", {
                clipPath: "url(#80d6b5__a)",
                children: /*#__PURE__*/ (0, $jgltN$jsxs)("g", {
                    clipPath: "url(#80d6b5__b)",
                    children: [
                        /*#__PURE__*/ (0, $jgltN$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $jgltN$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $jgltN$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#80d6b5__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $jgltN$jsx)("path", {
                            fill: "url(#80d6b5__d)",
                            d: "M61.07-28.264c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.549 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $jgltN$jsx)("path", {
                            fill: "url(#80d6b5__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $jgltN$jsx)("path", {
                            fill: "url(#80d6b5__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $jgltN$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#80d6b5__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $32fb9c3459a5e7c2$var$ForwardRef = /*#__PURE__*/ (0, $jgltN$forwardRef)($32fb9c3459a5e7c2$var$SvgComponent);
var $32fb9c3459a5e7c2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($32fb9c3459a5e7c2$var$ForwardRef);


export {$32fb9c3459a5e7c2$export$2e2bcd8739ae039 as default};
