// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_rotateCCW_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $k3Tpn$jsxs, jsx as $k3Tpn$jsx} from "react/jsx-runtime";
import {forwardRef as $k3Tpn$forwardRef} from "react";





const $b793c5c98f735f06$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $k3Tpn$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $k3Tpn$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $k3Tpn$jsxs)("linearGradient", {
                        id: "4404e0__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k3Tpn$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $k3Tpn$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k3Tpn$jsxs)("linearGradient", {
                        id: "4404e0__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k3Tpn$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $k3Tpn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k3Tpn$jsxs)("linearGradient", {
                        id: "4404e0__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k3Tpn$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $k3Tpn$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $k3Tpn$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $k3Tpn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k3Tpn$jsxs)("radialGradient", {
                        id: "4404e0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k3Tpn$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $k3Tpn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k3Tpn$jsxs)("radialGradient", {
                        id: "4404e0__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k3Tpn$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $k3Tpn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k3Tpn$jsx)("clipPath", {
                        id: "4404e0__b",
                        children: /*#__PURE__*/ (0, $k3Tpn$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $k3Tpn$jsx)("clipPath", {
                id: "4404e0__a",
                children: /*#__PURE__*/ (0, $k3Tpn$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M39.25 24c0-8.422-6.828-15.25-15.25-15.25-6.026 0-11.238 3.495-13.714 8.573l4.671-1.472c.659-.207 1.36.159 1.568.817.207.659-.158 1.36-.817 1.568L8.266 20.58c-.658.208-1.36-.158-1.568-.816L4.355 12.32c-.207-.659.158-1.36.817-1.568s1.36.158 1.568.817l1.401 4.45c2.92-5.793 8.924-9.77 15.86-9.77 9.802 0 17.75 7.947 17.75 17.75S33.802 41.75 24 41.75c-7.668 0-14.197-4.86-16.68-11.664-.236-.649.097-1.366.746-1.603.648-.237 1.366.097 1.603.746C11.804 35.078 17.417 39.25 24 39.25c8.423 0 15.25-6.828 15.25-15.25"
                })
            }),
            /*#__PURE__*/ (0, $k3Tpn$jsx)("g", {
                clipPath: "url(#4404e0__a)",
                children: /*#__PURE__*/ (0, $k3Tpn$jsxs)("g", {
                    clipPath: "url(#4404e0__b)",
                    children: [
                        /*#__PURE__*/ (0, $k3Tpn$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $k3Tpn$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $k3Tpn$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#4404e0__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $k3Tpn$jsx)("path", {
                            fill: "url(#4404e0__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $k3Tpn$jsx)("path", {
                            fill: "url(#4404e0__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $k3Tpn$jsx)("path", {
                            fill: "url(#4404e0__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $k3Tpn$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#4404e0__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $b793c5c98f735f06$var$ForwardRef = /*#__PURE__*/ (0, $k3Tpn$forwardRef)($b793c5c98f735f06$var$SvgComponent);
var $b793c5c98f735f06$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b793c5c98f735f06$var$ForwardRef);


export {$b793c5c98f735f06$export$2e2bcd8739ae039 as default};
