// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_pieChart_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $kvnWm$jsxs, jsx as $kvnWm$jsx} from "react/jsx-runtime";
import {forwardRef as $kvnWm$forwardRef} from "react";





const $d3561bdce274f6bf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kvnWm$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kvnWm$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kvnWm$jsxs)("linearGradient", {
                        id: "956ce8__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kvnWm$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kvnWm$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kvnWm$jsxs)("linearGradient", {
                        id: "956ce8__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kvnWm$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kvnWm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kvnWm$jsxs)("linearGradient", {
                        id: "956ce8__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kvnWm$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kvnWm$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kvnWm$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kvnWm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kvnWm$jsxs)("radialGradient", {
                        id: "956ce8__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kvnWm$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kvnWm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kvnWm$jsxs)("radialGradient", {
                        id: "956ce8__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kvnWm$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kvnWm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kvnWm$jsx)("clipPath", {
                        id: "956ce8__b",
                        children: /*#__PURE__*/ (0, $kvnWm$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kvnWm$jsx)("clipPath", {
                id: "956ce8__a",
                children: /*#__PURE__*/ (0, $kvnWm$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M45.602 14.496c-7.47.528-14.268 3.498-19.596 8.118L45.602 42.21zM22.616 26.004C17.504 31.896 14.402 39.582 14.402 48s3.102 16.098 8.214 21.996L44.612 48zM50.402 48V14.502c17.436 1.23 31.2 14.946 31.2 33.504s-15.042 33.6-33.6 33.6c-8.418 0-16.104-3.102-21.996-8.214L49.7 49.698q.043-.043.083-.089l-.084.09q.094-.1.175-.2.058-.074.11-.15l.001-.002.015-.02c.054-.085.096-.175.138-.265l.017-.033.013-.024q.026-.044.048-.093c.034-.082.056-.164.079-.25l.017-.062.005-.015.006-.02.014-.043.007-.02.005-.017.005-.017q.046-.234.048-.468"
                })
            }),
            /*#__PURE__*/ (0, $kvnWm$jsx)("g", {
                clipPath: "url(#956ce8__a)",
                children: /*#__PURE__*/ (0, $kvnWm$jsxs)("g", {
                    clipPath: "url(#956ce8__b)",
                    children: [
                        /*#__PURE__*/ (0, $kvnWm$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $kvnWm$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $kvnWm$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#956ce8__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $kvnWm$jsx)("path", {
                            fill: "url(#956ce8__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $kvnWm$jsx)("path", {
                            fill: "url(#956ce8__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $kvnWm$jsx)("path", {
                            fill: "url(#956ce8__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $kvnWm$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#956ce8__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $d3561bdce274f6bf$var$ForwardRef = /*#__PURE__*/ (0, $kvnWm$forwardRef)($d3561bdce274f6bf$var$SvgComponent);
var $d3561bdce274f6bf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d3561bdce274f6bf$var$ForwardRef);


export {$d3561bdce274f6bf$export$2e2bcd8739ae039 as default};
