// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_assets_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $gYgAQ$jsxs, jsx as $gYgAQ$jsx} from "react/jsx-runtime";
import {forwardRef as $gYgAQ$forwardRef} from "react";





const $6f8292e6f06faedf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gYgAQ$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gYgAQ$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gYgAQ$jsxs)("linearGradient", {
                        id: "b93fd3__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gYgAQ$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gYgAQ$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gYgAQ$jsxs)("linearGradient", {
                        id: "b93fd3__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gYgAQ$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gYgAQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gYgAQ$jsxs)("linearGradient", {
                        id: "b93fd3__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gYgAQ$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gYgAQ$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gYgAQ$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gYgAQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gYgAQ$jsxs)("radialGradient", {
                        id: "b93fd3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gYgAQ$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gYgAQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gYgAQ$jsxs)("radialGradient", {
                        id: "b93fd3__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gYgAQ$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gYgAQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gYgAQ$jsx)("clipPath", {
                        id: "b93fd3__b",
                        children: /*#__PURE__*/ (0, $gYgAQ$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gYgAQ$jsx)("clipPath", {
                id: "b93fd3__a",
                children: /*#__PURE__*/ (0, $gYgAQ$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M9.125 14.027c0-2.209 1.79-4 4-4H30c2.21 0 4 1.791 4 4v4.348h-8.224c-1.892 0-3.594.808-4.782 2.097l-1.445-2.36-.001-.002c-1.068-1.738-3.584-1.766-4.69-.052l-5.44 8.434c-.19-.464-.293-.972-.293-1.504zm10.15 10.848v4.113h-6.15c-.728 0-1.41-.194-1.998-.534l5.831-9.04c.109-.168.355-.166.46.004l2.152 3.518c-.191.612-.294 1.264-.294 1.939m8.798-7.664c.828 0 1.5-.672 1.5-1.5s-.672-1.5-1.5-1.5c-.829 0-1.5.672-1.5 1.5s.671 1.5 1.5 1.5m-6.297 7.664c0-2.21 1.79-4 4-4h9.099c2.21 0 4 1.79 4 4v9.1c0 2.208-1.79 4-4 4h-9.1c-2.208 0-4-1.792-4-4zm12.24 3.982-5.607-3.216c-.628-.36-1.414.088-1.415.807l-.008 6.419c0 .719.784 1.169 1.413.81l5.615-3.203c.629-.358.63-1.257.002-1.617"
                })
            }),
            /*#__PURE__*/ (0, $gYgAQ$jsx)("g", {
                clipPath: "url(#b93fd3__a)",
                children: /*#__PURE__*/ (0, $gYgAQ$jsxs)("g", {
                    clipPath: "url(#b93fd3__b)",
                    children: [
                        /*#__PURE__*/ (0, $gYgAQ$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $gYgAQ$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $gYgAQ$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#b93fd3__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $gYgAQ$jsx)("path", {
                            fill: "url(#b93fd3__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $gYgAQ$jsx)("path", {
                            fill: "url(#b93fd3__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $gYgAQ$jsx)("path", {
                            fill: "url(#b93fd3__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $gYgAQ$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#b93fd3__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $6f8292e6f06faedf$var$ForwardRef = /*#__PURE__*/ (0, $gYgAQ$forwardRef)($6f8292e6f06faedf$var$SvgComponent);
var $6f8292e6f06faedf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6f8292e6f06faedf$var$ForwardRef);


export {$6f8292e6f06faedf$export$2e2bcd8739ae039 as default};
