// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_calendar_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gp2XR$reactjsxruntime = require("react/jsx-runtime");
var $gp2XR$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $54ff407d22c14d9a$export$2e2bcd8739ae039);




const $54ff407d22c14d9a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ffa5ef__d",
                        x1: -14.214,
                        x2: 117.507,
                        y1: 76.562,
                        y2: -8.449,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ffa5ef__e",
                        x1: 95.069,
                        x2: 40.327,
                        y1: 20.545,
                        y2: 82.987,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ffa5ef__f",
                        x1: 24.072,
                        x2: 124.627,
                        y1: 87.935,
                        y2: 28.18,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ffa5ef__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.3519 -36.4409 0 36.44 34.352)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ffa5ef__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.3519 -36.4409 0 36.44 34.352)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("clipPath", {
                        id: "ffa5ef__b",
                        children: /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("rect", {
                            width: 83.762,
                            height: 78.96,
                            x: 6.836,
                            y: 6,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("clipPath", {
                id: "ffa5ef__a",
                children: /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M69.597 20.4h-4.2v6c0 1.326-1.074 2.4-2.4 2.4s-2.4-1.074-2.4-2.4v-6h-25.2v6c0 1.326-1.074 2.4-2.4 2.4s-2.4-1.074-2.4-2.4v-6h-4.2c-5.304 0-9.6 4.296-9.6 9.6v6h62.4v-6c0-5.304-4.296-9.6-9.6-9.6m-52.8 49.2c0 5.304 4.296 9.6 9.6 9.6h43.2c5.304 0 9.6-4.296 9.6-9.6V40.8h-62.4zm43.8-19.2c0-1.326 1.074-2.4 2.4-2.4h3.6c1.326 0 2.4 1.074 2.4 2.4V54c0 1.326-1.074 2.4-2.4 2.4h-3.6c-1.326 0-2.4-1.074-2.4-2.4zm0 13.2c0-1.326 1.074-2.4 2.4-2.4h3.6c1.326 0 2.4 1.074 2.4 2.4v3.6c0 1.326-1.074 2.4-2.4 2.4h-3.6c-1.326 0-2.4-1.074-2.4-2.4zm-16.8-13.2c0-1.326 1.074-2.4 2.4-2.4h3.6c1.326 0 2.4 1.074 2.4 2.4V54c0 1.326-1.074 2.4-2.4 2.4h-3.6c-1.326 0-2.4-1.074-2.4-2.4zm0 13.2c0-1.326 1.074-2.4 2.4-2.4h3.6c1.326 0 2.4 1.074 2.4 2.4v3.6c0 1.326-1.074 2.4-2.4 2.4h-3.6c-1.326 0-2.4-1.074-2.4-2.4zm-16.8-13.2c0-1.326 1.074-2.4 2.4-2.4h3.6c1.326 0 2.4 1.074 2.4 2.4V54c0 1.326-1.074 2.4-2.4 2.4h-3.6c-1.326 0-2.4-1.074-2.4-2.4zm0 13.2c0-1.326 1.074-2.4 2.4-2.4h3.6c1.326 0 2.4 1.074 2.4 2.4v3.6c0 1.326-1.074 2.4-2.4 2.4h-3.6c-1.326 0-2.4-1.074-2.4-2.4zm8.4-49.2c0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4v6h4.8zm30 0c0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4v6h4.8z"
                })
            }),
            /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ffa5ef__a)",
                children: /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ffa5ef__b)",
                    children: [
                        /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("rect", {
                            width: 83.762,
                            height: 78.96,
                            x: 6.836,
                            y: 6,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6.836 6h83.762v78.96H6.836z"
                        }),
                        /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("ellipse", {
                            cx: 36.441,
                            cy: 34.352,
                            fill: "url(#ffa5ef__c)",
                            rx: 36.441,
                            ry: 34.352,
                            transform: "matrix(1 -.00132 .00148 1 17.707 -25.241)"
                        }),
                        /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("path", {
                            fill: "url(#ffa5ef__d)",
                            d: "M38.806-7.948c-6.433-3.752-14.583-3.775-21.04-.06l-64.579 37.17c-6.324 3.64-6.335 12.233-.019 15.887L17.32 82.166c6.447 3.73 14.598 3.725 21.04-.013L102.378 45c6.283-3.646 6.294-12.19.019-15.85z"
                        }),
                        /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("path", {
                            fill: "url(#ffa5ef__e)",
                            d: "M18.906 43.489c1.431-29.478 27.942-52.28 59.213-50.931 31.27 1.35 55.46 26.34 54.028 55.818s-27.942 52.281-59.213 50.931c-31.27-1.35-55.46-26.34-54.028-55.818"
                        }),
                        /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("path", {
                            fill: "url(#ffa5ef__f)",
                            d: "M-27.101 62.925c12.43-4.999 25.836-4.785 37.45-.412 23.24 8.77 36.764 3.348 46.143-18.51C61.164 33.034 70.285 23.745 82.73 18.74c24.88-9.99 53.603.85 64.185 24.234s-1 50.418-25.874 60.422c-12.472 5.015-25.904 4.78-37.536.369-23.203-8.725-36.7-3.281-46.068 18.603-4.684 10.943-13.803 20.2-26.233 25.199-24.86 9.997-53.583-.843-64.16-24.214s.994-50.432 25.855-60.43"
                        }),
                        /*#__PURE__*/ (0, $gp2XR$reactjsxruntime.jsx)("ellipse", {
                            cx: 36.441,
                            cy: 34.352,
                            fill: "url(#ffa5ef__g)",
                            rx: 36.441,
                            ry: 34.352,
                            transform: "matrix(1 -.00132 .00148 1 49.223 -14.04)"
                        })
                    ]
                })
            })
        ]
    });
const $54ff407d22c14d9a$var$ForwardRef = /*#__PURE__*/ (0, $gp2XR$react.forwardRef)($54ff407d22c14d9a$var$SvgComponent);
var $54ff407d22c14d9a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($54ff407d22c14d9a$var$ForwardRef);
