// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_ribbon_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7XXTF$jsxs, jsx as $7XXTF$jsx} from "react/jsx-runtime";
import {forwardRef as $7XXTF$forwardRef} from "react";





const $df24450cf2ea55d7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7XXTF$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7XXTF$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7XXTF$jsxs)("linearGradient", {
                        id: "ed1969__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7XXTF$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $7XXTF$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7XXTF$jsxs)("linearGradient", {
                        id: "ed1969__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7XXTF$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7XXTF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7XXTF$jsxs)("linearGradient", {
                        id: "ed1969__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7XXTF$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7XXTF$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7XXTF$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7XXTF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7XXTF$jsxs)("radialGradient", {
                        id: "ed1969__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7XXTF$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $7XXTF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7XXTF$jsxs)("radialGradient", {
                        id: "ed1969__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7XXTF$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7XXTF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7XXTF$jsx)("clipPath", {
                        id: "ed1969__b",
                        children: /*#__PURE__*/ (0, $7XXTF$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7XXTF$jsx)("clipPath", {
                id: "ed1969__a",
                children: /*#__PURE__*/ (0, $7XXTF$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M34.75 18.25C34.75 24.187 29.937 29 24 29s-10.75-4.813-10.75-10.75S18.063 7.5 24 7.5s10.75 4.813 10.75 10.75m-7.325 20.118-3.158-7.87c2.847-.061 5.455-1.093 7.506-2.779l3.073 7.662c.295.734-.339 1.503-1.115 1.355l-2.245-.43c-.386-.073-.779.086-1.005.407l-1.31 1.859c-.457.648-1.45.532-1.746-.204M16.208 27.703c2.047 1.69 4.652 2.727 7.497 2.794l-3.112 7.767c-.297.742-1.303.852-1.754.191l-1.277-1.871c-.228-.334-.634-.498-1.029-.416l-2.231.462c-.78.161-1.428-.611-1.13-1.352z"
                })
            }),
            /*#__PURE__*/ (0, $7XXTF$jsx)("g", {
                clipPath: "url(#ed1969__a)",
                children: /*#__PURE__*/ (0, $7XXTF$jsxs)("g", {
                    clipPath: "url(#ed1969__b)",
                    children: [
                        /*#__PURE__*/ (0, $7XXTF$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $7XXTF$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $7XXTF$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#ed1969__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $7XXTF$jsx)("path", {
                            fill: "url(#ed1969__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $7XXTF$jsx)("path", {
                            fill: "url(#ed1969__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $7XXTF$jsx)("path", {
                            fill: "url(#ed1969__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $7XXTF$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#ed1969__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $df24450cf2ea55d7$var$ForwardRef = /*#__PURE__*/ (0, $7XXTF$forwardRef)($df24450cf2ea55d7$var$SvgComponent);
var $df24450cf2ea55d7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($df24450cf2ea55d7$var$ForwardRef);


export {$df24450cf2ea55d7$export$2e2bcd8739ae039 as default};
