// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_hand_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $zxHPv$jsxs, jsx as $zxHPv$jsx} from "react/jsx-runtime";
import {forwardRef as $zxHPv$forwardRef} from "react";





const $2c009eedc005ff8b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $zxHPv$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $zxHPv$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $zxHPv$jsxs)("linearGradient", {
                        id: "0ac701__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $zxHPv$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $zxHPv$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $zxHPv$jsxs)("linearGradient", {
                        id: "0ac701__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $zxHPv$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $zxHPv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $zxHPv$jsxs)("linearGradient", {
                        id: "0ac701__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.618,
                        y2: 33.126,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $zxHPv$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $zxHPv$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $zxHPv$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $zxHPv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $zxHPv$jsxs)("radialGradient", {
                        id: "0ac701__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $zxHPv$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $zxHPv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $zxHPv$jsxs)("radialGradient", {
                        id: "0ac701__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $zxHPv$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $zxHPv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $zxHPv$jsx)("clipPath", {
                        id: "0ac701__b",
                        children: /*#__PURE__*/ (0, $zxHPv$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $zxHPv$jsx)("clipPath", {
                id: "0ac701__a",
                children: /*#__PURE__*/ (0, $zxHPv$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m18.653 50.734-.022-.04c-1.363-2.365-1.023-4.963.9-6.408 1.982-1.489 4.786-1.106 6.299.861l5.888 7.657c.26.339.709.474 1.114.337.406-.138.678-.519.678-.947V26.63c0-2.698 2.176-4.877 4.852-4.877s4.852 2.18 4.852 4.877v16.44c0 .553.448 1 1 1h.012c.553 0 1-.447 1-1V20.878c0-2.698 2.176-4.877 4.852-4.877s4.852 2.179 4.852 4.877v22.198c0 .552.448 1 1 1h.012c.552 0 1-.448 1-1V23.66c0-2.698 2.176-4.877 4.852-4.877 2.68 0 4.855 2.18 4.855 4.877v20.878c0 .53.415.969.944.998.53.03.99-.36 1.05-.887l.559-4.991.559-4.991.001-.014c.285-2.91 2.877-4.358 5.257-3.89 1.916.376 3.424 2.095 3.138 4.864l-2.487 17.39q-.025.074-.04.152c-2 11.113-5.291 17.778-9.49 21.678C61.97 78.72 56.753 80 50.56 80c-4.05 0-8.065-1.742-11.86-4.47-3.788-2.722-7.282-6.375-10.266-10.062-2.98-3.683-5.429-7.371-7.133-10.141-.852-1.384-1.516-2.537-1.966-3.341-.226-.402-.397-.718-.512-.931l-.13-.243-.031-.06-.008-.015z"
                })
            }),
            /*#__PURE__*/ (0, $zxHPv$jsx)("g", {
                clipPath: "url(#0ac701__a)",
                children: /*#__PURE__*/ (0, $zxHPv$jsxs)("g", {
                    clipPath: "url(#0ac701__b)",
                    children: [
                        /*#__PURE__*/ (0, $zxHPv$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $zxHPv$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $zxHPv$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#0ac701__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $zxHPv$jsx)("path", {
                            fill: "url(#0ac701__d)",
                            d: "M36.643-16.957c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.426-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.432 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $zxHPv$jsx)("path", {
                            fill: "url(#0ac701__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $zxHPv$jsx)("path", {
                            fill: "url(#0ac701__f)",
                            d: "M-38.895 69.21c14.246-6.077 29.61-5.817 42.922-.5 26.636 10.663 42.135 4.07 52.885-22.505 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.315 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $zxHPv$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#0ac701__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $2c009eedc005ff8b$var$ForwardRef = /*#__PURE__*/ (0, $zxHPv$forwardRef)($2c009eedc005ff8b$var$SvgComponent);
var $2c009eedc005ff8b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2c009eedc005ff8b$var$ForwardRef);


export {$2c009eedc005ff8b$export$2e2bcd8739ae039 as default};
