// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_magicWand_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $1UM2Y$jsxs, jsx as $1UM2Y$jsx} from "react/jsx-runtime";
import {forwardRef as $1UM2Y$forwardRef} from "react";





const $d75f3e9122742920$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1UM2Y$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1UM2Y$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1UM2Y$jsxs)("linearGradient", {
                        id: "6ece83__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1UM2Y$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $1UM2Y$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1UM2Y$jsxs)("linearGradient", {
                        id: "6ece83__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1UM2Y$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1UM2Y$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1UM2Y$jsxs)("linearGradient", {
                        id: "6ece83__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1UM2Y$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1UM2Y$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1UM2Y$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1UM2Y$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1UM2Y$jsxs)("radialGradient", {
                        id: "6ece83__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1UM2Y$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $1UM2Y$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1UM2Y$jsxs)("radialGradient", {
                        id: "6ece83__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1UM2Y$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1UM2Y$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1UM2Y$jsx)("clipPath", {
                        id: "6ece83__b",
                        children: /*#__PURE__*/ (0, $1UM2Y$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1UM2Y$jsx)("clipPath", {
                id: "6ece83__a",
                children: /*#__PURE__*/ (0, $1UM2Y$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M132.689 44.59 125.634 46c-.934.189-1.709.818-2.086 1.687l-2.846 6.606c-1.122 2.604-4.928 2.2-5.481-.573l-1.41-7.055c-.188-.934-.817-1.709-1.687-2.086l-6.606-2.846c-2.603-1.122-2.2-4.928.574-5.481l7.055-1.41c.933-.188 1.708-.817 2.086-1.687l2.846-6.606c1.122-2.603 4.928-2.2 5.481.574l1.409 7.055c.189.933.818 1.708 1.687 2.086l6.606 2.846c2.604 1.122 2.201 4.928-.573 5.48m-59.522 6.043-5.827 1.165c-.771.155-1.411.675-1.723 1.393l-2.35 5.456c-.927 2.15-4.07 1.818-4.527-.473l-1.165-5.828c-.155-.77-.675-1.41-1.393-1.722l-5.457-2.351c-2.15-.927-1.817-4.07.474-4.527l5.828-1.165c.77-.155 1.41-.675 1.722-1.393l2.351-5.456c.927-2.15 4.07-1.818 4.527.474l1.165 5.827c.155.77.675 1.411 1.393 1.723l5.456 2.35c2.15.927 1.817 4.07-.474 4.527m51.723 43.195 5.827-1.164c2.292-.457 2.624-3.6.474-4.527l-5.456-2.351c-.718-.312-1.238-.952-1.393-1.723l-1.165-5.827c-.457-2.291-3.6-2.624-4.527-.474l-2.351 5.457c-.311.718-.951 1.237-1.722 1.393l-5.828 1.164c-2.291.457-2.624 3.6-.473 4.527l5.456 2.35c.718.313 1.238.953 1.393 1.724l1.165 5.827c.456 2.291 3.6 2.624 4.526.474l2.351-5.457c.312-.718.952-1.237 1.723-1.393M88.87 57.712c3.758-3.757 9.85-3.757 13.606 0s3.757 9.85 0 13.606l-9.154 9.154-13.606-13.605zM74.06 72.524 87.665 86.13l-46.25 46.25c-3.756 3.757-9.848 3.757-13.605 0s-3.757-9.849 0-13.606z"
                })
            }),
            /*#__PURE__*/ (0, $1UM2Y$jsx)("g", {
                clipPath: "url(#6ece83__a)",
                children: /*#__PURE__*/ (0, $1UM2Y$jsxs)("g", {
                    clipPath: "url(#6ece83__b)",
                    children: [
                        /*#__PURE__*/ (0, $1UM2Y$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $1UM2Y$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $1UM2Y$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#6ece83__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $1UM2Y$jsx)("path", {
                            fill: "url(#6ece83__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $1UM2Y$jsx)("path", {
                            fill: "url(#6ece83__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $1UM2Y$jsx)("path", {
                            fill: "url(#6ece83__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $1UM2Y$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#6ece83__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $d75f3e9122742920$var$ForwardRef = /*#__PURE__*/ (0, $1UM2Y$forwardRef)($d75f3e9122742920$var$SvgComponent);
var $d75f3e9122742920$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d75f3e9122742920$var$ForwardRef);


export {$d75f3e9122742920$export$2e2bcd8739ae039 as default};
