// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileVideo_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $barH4$reactjsxruntime = require("react/jsx-runtime");
var $barH4$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b224a24ce5d2a446$export$2e2bcd8739ae039);




const $b224a24ce5d2a446$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $barH4$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ac80b9__d",
                        x1: -14.724,
                        x2: 121.006,
                        y1: 80.863,
                        y2: -1.714,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ac80b9__e",
                        x1: 94.203,
                        x2: 35.974,
                        y1: 21.635,
                        y2: 84.246,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ac80b9__f",
                        x1: 23.438,
                        x2: 126.661,
                        y1: 92.888,
                        y2: 35.064,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ac80b9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 21.982 31.479)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ac80b9__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 31.768 53.108)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("clipPath", {
                        id: "ac80b9__b",
                        children: /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("clipPath", {
                id: "ac80b9__a",
                children: /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M72.41 38.644 52.527 18.763c-1.357-1.357-3.197-2.12-5.117-2.12h-18.71c-3.997 0-7.236 3.24-7.236 7.237v48.24c0 3.996 3.24 7.236 7.236 7.236h38.592c3.996 0 7.236-3.24 7.236-7.236V43.76c0-1.918-.762-3.759-2.12-5.116M57.04 55.988 42.67 64.261c-1.609.926-3.617-.237-3.615-2.094l.02-16.579c.002-1.857 2.013-3.015 3.62-2.084l14.348 8.307c1.607.93 1.604 3.25-.005 4.177"
                })
            }),
            /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ac80b9__a)",
                children: /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ac80b9__b)",
                    children: [
                        /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6.258 6.258h83.485v83.485H6.258z"
                        }),
                        /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("circle", {
                            cx: 53.461,
                            cy: 9.496,
                            r: 36.321,
                            fill: "url(#ac80b9__c)",
                            transform: "rotate(-.08 53.461 9.496)"
                        }),
                        /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("path", {
                            fill: "url(#ac80b9__d)",
                            d: "M38.121-8.49c-6.412-3.967-14.535-3.992-20.97-.062l-64.366 39.3c-6.304 3.848-6.314 12.933-.02 16.797l63.941 39.243c6.426 3.944 14.55 3.939 20.97-.014l63.808-39.28c6.262-3.855 6.274-12.89.019-16.76z"
                        }),
                        /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("path", {
                            fill: "url(#ac80b9__e)",
                            d: "M18.29 45.894c1.428-31.167 27.85-55.277 59.018-53.85s55.277 27.85 53.85 59.018-27.85 55.276-59.017 53.849c-31.168-1.426-55.277-27.85-53.85-59.017"
                        }),
                        /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("path", {
                            fill: "url(#ac80b9__f)",
                            d: "M-27.566 66.445c12.389-5.285 25.75-5.059 37.326-.436 23.164 9.274 36.642 3.54 45.99-19.57 4.657-11.598 13.749-21.42 26.152-26.71 24.798-10.563 53.426.898 63.974 25.623 10.547 24.724-.998 53.307-25.79 63.884-12.43 5.303-25.818 5.055-37.412.39-23.126-9.224-36.579-3.469-45.915 19.67C32.09 140.865 23 150.653 10.612 155.938c-24.778 10.571-53.406-.89-63.948-25.601s.991-53.322 25.77-63.892"
                        }),
                        /*#__PURE__*/ (0, $barH4$reactjsxruntime.jsx)("circle", {
                            cx: 84.875,
                            cy: 21.34,
                            r: 36.321,
                            fill: "url(#ac80b9__g)",
                            transform: "rotate(-.08 84.875 21.34)"
                        })
                    ]
                })
            })
        ]
    });
const $b224a24ce5d2a446$var$ForwardRef = /*#__PURE__*/ (0, $barH4$react.forwardRef)($b224a24ce5d2a446$var$SvgComponent);
var $b224a24ce5d2a446$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b224a24ce5d2a446$var$ForwardRef);
