// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_cut_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $e60ln$jsxs, jsx as $e60ln$jsx} from "react/jsx-runtime";
import {forwardRef as $e60ln$forwardRef} from "react";





const $118486d997689348$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e60ln$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $e60ln$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $e60ln$jsxs)("linearGradient", {
                        id: "63fb8d__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.896,
                        y2: -4.582,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e60ln$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $e60ln$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e60ln$jsxs)("linearGradient", {
                        id: "63fb8d__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.843,
                        y2: 44.841,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e60ln$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $e60ln$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e60ln$jsxs)("linearGradient", {
                        id: "63fb8d__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.81,
                        y2: 16.564,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e60ln$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $e60ln$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $e60ln$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $e60ln$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e60ln$jsxs)("radialGradient", {
                        id: "63fb8d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.503)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e60ln$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $e60ln$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e60ln$jsxs)("radialGradient", {
                        id: "63fb8d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.264 26.937)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e60ln$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $e60ln$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e60ln$jsx)("clipPath", {
                        id: "63fb8d__b",
                        children: /*#__PURE__*/ (0, $e60ln$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $e60ln$jsx)("clipPath", {
                id: "63fb8d__a",
                children: /*#__PURE__*/ (0, $e60ln$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M29.97 7.346c.654.224 1.002.934.779 1.588l-4.218 12.32 12.485-4.252c.653-.222 1.363.127 1.586.78.222.654-.127 1.364-.78 1.587L25.807 24.14q-.076.025-.154.042-.093.378-.182.77v.001c-.255 1.1-.53 2.29-1.15 3.63q-.159.344-.342.69h.081c3.162 0 5.724 2.563 5.724 5.724s-2.562 5.724-5.724 5.724c-3.16 0-5.724-2.562-5.724-5.724 0-1.24.395-2.39 1.066-3.327q.06-.1.144-.192.113-.145.235-.283c.528-.674 1.61-2.234 2.271-3.662.464-1.001.667-1.802.882-2.722-.41.155-.858.333-1.386.55-.825.34-1.792.88-2.633 1.405-.853.531-1.488.99-1.663 1.139q-.064.054-.131.097c-1.038 1.047-2.476 1.695-4.066 1.695-3.161 0-5.724-2.563-5.724-5.724s2.563-5.724 5.724-5.724c3.152 0 5.708 2.547 5.724 5.694.596-.332 1.222-.648 1.817-.893 1.265-.522 2.113-.835 3.064-1.134l4.722-13.793c.224-.653.935-1.002 1.588-.778m-8.493 25.725c-.402.537-.64 1.205-.64 1.927 0 1.78 1.444 3.224 3.224 3.224s3.224-1.443 3.224-3.224c0-1.78-1.443-3.224-3.224-3.224-.942 0-1.79.404-2.379 1.048q-.113.143-.205.249m-8.42-12.321c-1.782 0-3.225 1.443-3.225 3.224 0 1.78 1.443 3.224 3.224 3.224 1.78 0 3.224-1.444 3.224-3.224s-1.443-3.224-3.224-3.224"
                })
            }),
            /*#__PURE__*/ (0, $e60ln$jsx)("g", {
                clipPath: "url(#63fb8d__a)",
                children: /*#__PURE__*/ (0, $e60ln$jsxs)("g", {
                    clipPath: "url(#63fb8d__b)",
                    children: [
                        /*#__PURE__*/ (0, $e60ln$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $e60ln$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 .001h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $e60ln$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.863,
                            r: 20.883,
                            fill: "url(#63fb8d__c)",
                            transform: "rotate(-.08 27.142 1.863)"
                        }),
                        /*#__PURE__*/ (0, $e60ln$jsx)("path", {
                            fill: "url(#63fb8d__d)",
                            d: "M18.321-8.478c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.657L6.008 46.304c3.694 2.267 8.365 2.264 12.056-.008L54.752 23.71c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $e60ln$jsx)("path", {
                            fill: "url(#63fb8d__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.99 40.85-8.17S72.63 7.84 71.809 25.76 55.798 57.543 37.879 56.722C19.957 55.902 6.095 40.71 6.915 22.79"
                        }),
                        /*#__PURE__*/ (0, $e60ln$jsx)("path", {
                            fill: "url(#63fb8d__f)",
                            d: "M-19.448 34.606c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.331 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.649-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.279-15.033 15.318-14.246 6.077-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $e60ln$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.673,
                            r: 20.883,
                            fill: "url(#63fb8d__g)",
                            transform: "rotate(-.08 45.2 8.673)"
                        })
                    ]
                })
            })
        ]
    });
const $118486d997689348$var$ForwardRef = /*#__PURE__*/ (0, $e60ln$forwardRef)($118486d997689348$var$SvgComponent);
var $118486d997689348$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($118486d997689348$var$ForwardRef);


export {$118486d997689348$export$2e2bcd8739ae039 as default};
