// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_star_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $cWMpd$reactjsxruntime = require("react/jsx-runtime");
var $cWMpd$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d1da8531a5003e43$export$2e2bcd8739ae039);




const $d1da8531a5003e43$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "efcc3d__d",
                        x1: -14.409,
                        x2: 120.646,
                        y1: 80.7,
                        y2: -1.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "efcc3d__e",
                        x1: 93.971,
                        x2: 36.033,
                        y1: 21.766,
                        y2: 84.066,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "efcc3d__f",
                        x1: 23.561,
                        x2: 126.271,
                        y1: 92.664,
                        y2: 35.128,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsxs)("radialGradient", {
                        id: "efcc3d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 21.875 31.562)scale(36.1398)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsxs)("radialGradient", {
                        id: "efcc3d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 31.611 53.083)scale(36.1398)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("clipPath", {
                        id: "efcc3d__b",
                        children: /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("clipPath", {
                id: "efcc3d__a",
                children: /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "m52.923 19.86 5.796 11.745c.8 1.62 2.346 2.744 4.135 3.004l12.96 1.883c4.504.655 6.303 6.19 3.044 9.366L69.479 55c-1.294 1.261-1.885 3.079-1.579 4.86l2.214 12.908c.77 4.486-3.939 7.907-7.967 5.789l-11.593-6.094c-1.6-.841-3.51-.841-5.11 0l-11.592 6.094c-4.029 2.118-8.737-1.303-7.968-5.789l2.214-12.908c.306-1.781-.285-3.599-1.579-4.86l-9.378-9.142c-3.26-3.177-1.46-8.711 3.043-9.366l12.96-1.883c1.789-.26 3.335-1.383 4.135-3.004l5.796-11.744c2.014-4.081 7.834-4.081 9.848 0"
                })
            }),
            /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("g", {
                clipPath: "url(#efcc3d__a)",
                children: /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#efcc3d__b)",
                    children: [
                        /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6.465 6.465h83.07v83.07H6.465z"
                        }),
                        /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("circle", {
                            cx: 53.436,
                            cy: 9.687,
                            r: 36.14,
                            fill: "url(#efcc3d__c)",
                            transform: "rotate(-.08 53.436 9.687)"
                        }),
                        /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("path", {
                            fill: "url(#efcc3d__d)",
                            d: "M38.173-8.209c-6.38-3.947-14.463-3.972-20.866-.062L-46.74 30.833c-6.272 3.83-6.283 12.87-.019 16.714l63.623 39.048c6.393 3.924 14.477 3.92 20.865-.014l63.491-39.085c6.231-3.836 6.242-12.825.019-16.676z"
                        }),
                        /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("path", {
                            fill: "url(#efcc3d__e)",
                            d: "M18.437 45.905c1.42-31.013 27.711-55.002 58.724-53.582s55.001 27.711 53.582 58.723c-1.42 31.013-27.712 55.002-58.724 53.582-31.012-1.419-55.002-27.71-53.582-58.723"
                        }),
                        /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("path", {
                            fill: "url(#efcc3d__f)",
                            d: "M-27.19 66.352c12.328-5.259 25.623-5.033 37.142-.433C33 75.146 46.412 69.44 55.714 46.445c4.633-11.539 13.679-21.312 26.02-26.576 24.675-10.51 53.161.893 63.656 25.495s-.992 53.043-25.661 63.567c-12.369 5.276-25.69 5.029-37.226.388-23.011-9.178-36.397-3.452-45.687 19.572-4.645 11.512-13.689 21.251-26.016 26.51-24.655 10.518-53.141-.886-63.63-25.474s.986-53.057 25.64-63.575"
                        }),
                        /*#__PURE__*/ (0, $cWMpd$reactjsxruntime.jsx)("circle", {
                            cx: 84.694,
                            cy: 21.472,
                            r: 36.14,
                            fill: "url(#efcc3d__g)",
                            transform: "rotate(-.08 84.694 21.472)"
                        })
                    ]
                })
            })
        ]
    });
const $d1da8531a5003e43$var$ForwardRef = /*#__PURE__*/ (0, $cWMpd$react.forwardRef)($d1da8531a5003e43$var$SvgComponent);
var $d1da8531a5003e43$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d1da8531a5003e43$var$ForwardRef);
