// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_filters_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4Q4WC$reactjsxruntime = require("react/jsx-runtime");
var $4Q4WC$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $623137715140c516$export$2e2bcd8739ae039);




const $623137715140c516$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bf1e2f__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bf1e2f__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bf1e2f__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsxs)("radialGradient", {
                        id: "bf1e2f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsxs)("radialGradient", {
                        id: "bf1e2f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("clipPath", {
                        id: "bf1e2f__b",
                        children: /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("clipPath", {
                id: "bf1e2f__a",
                children: /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M19.52 96.801c0-11.294 4.776-21.472 12.418-28.625 1.58 15.08 10.261 28.046 22.638 35.494 1.769 12.132 8.152 22.766 17.311 30.065-4.115 1.467-8.548 2.266-13.168 2.266-21.65 0-39.2-17.55-39.2-39.2m68.592 36.934c4.116 1.467 8.548 2.266 13.168 2.266 21.649 0 39.2-17.55 39.2-39.2 0-12.289-5.655-23.257-14.505-30.444-.979 14.829-8.807 27.786-20.349 35.738-1.427 12.785-7.96 24.027-17.514 31.64m28.375-81.63C111.701 35.859 96.675 24 78.879 24c-18.134 0-33.393 12.314-37.87 29.036 5.47-2.215 11.447-3.434 17.71-3.434 7.656 0 14.886 1.822 21.28 5.058 6.395-3.236 13.625-5.058 21.281-5.058 5.32 0 10.435.88 15.207 2.503m-76.802 10.42q-.006.337-.006.675c0 12.289 5.655 23.257 14.505 30.444.902-13.665 7.621-25.742 17.703-33.776-4.115-1.467-8.548-2.266-13.168-2.266-6.907 0-13.398 1.786-19.034 4.923m78.351-1.172q.043.918.043 1.847c0 11.294-4.776 21.472-12.419 28.625-1.353-12.916-7.916-24.28-17.548-31.957 4.116-1.467 8.548-2.266 13.168-2.266 5.995 0 11.675 1.345 16.756 3.751M80 129.728c8.038-5.206 14.049-13.275 16.589-22.763-5.469 2.215-11.447 3.435-17.71 3.435-5.32 0-10.435-.881-15.207-2.504 2.676 9.085 8.554 16.797 16.328 21.832m17.92-32.927c0-13.804-7.136-25.942-17.92-32.926-10.784 6.984-17.92 19.122-17.92 32.926q0 .93.043 1.847c5.08 2.406 10.761 3.752 16.756 3.752 6.908 0 13.398-1.787 19.034-4.924q.006-.337.006-.675"
                })
            }),
            /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("g", {
                clipPath: "url(#bf1e2f__a)",
                children: /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#bf1e2f__b)",
                    children: [
                        /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#bf1e2f__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("path", {
                            fill: "url(#bf1e2f__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("path", {
                            fill: "url(#bf1e2f__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("path", {
                            fill: "url(#bf1e2f__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $4Q4WC$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#bf1e2f__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $623137715140c516$var$ForwardRef = /*#__PURE__*/ (0, $4Q4WC$react.forwardRef)($623137715140c516$var$SvgComponent);
var $623137715140c516$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($623137715140c516$var$ForwardRef);
