// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_volume_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $5k7GG$reactjsxruntime = require("react/jsx-runtime");
var $5k7GG$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $52cdbe6a6f9c351a$export$2e2bcd8739ae039);




const $52cdbe6a6f9c351a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b9415e__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b9415e__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b9415e__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b9415e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b9415e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("clipPath", {
                        id: "b9415e__b",
                        children: /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("clipPath", {
                id: "b9415e__a",
                children: /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M28.844 32.052h-8.38c-2.208 0-4 1.791-4 4v24.022c0 2.21 1.792 4 4 4h8.391q.087 0 .172.004v-32.03q-.09.004-.184.004m4.183-2.515v37.028l10.305 10.114c1.898 1.863 5.102.518 5.102-2.14V21.46c0-2.666-3.219-4.008-5.113-2.13zm22.383 5.235c.664-.883 1.918-1.06 2.8-.395 4.11 3.092 6.774 8.017 6.774 13.564 0 5.592-2.707 10.553-6.874 13.64-.888.658-2.14.472-2.798-.416s-.471-2.14.416-2.797c3.193-2.366 5.256-6.155 5.256-10.427 0-4.237-2.03-7.999-5.178-10.368-.883-.665-1.06-1.918-.396-2.801m12.805-9.524c-.882-.665-2.136-.488-2.8.395-.665.883-.488 2.137.395 2.8 5.91 4.448 9.725 11.515 9.725 19.473 0 8.023-3.878 15.14-9.87 19.58-.888.658-1.074 1.91-.416 2.798.657.888 1.91 1.074 2.797.416 6.967-5.162 11.489-13.45 11.489-22.794 0-9.268-4.45-17.498-11.32-22.668"
                })
            }),
            /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("g", {
                clipPath: "url(#b9415e__a)",
                children: /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#b9415e__b)",
                    children: [
                        /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#b9415e__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("path", {
                            fill: "url(#b9415e__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("path", {
                            fill: "url(#b9415e__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("path", {
                            fill: "url(#b9415e__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $5k7GG$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#b9415e__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $52cdbe6a6f9c351a$var$ForwardRef = /*#__PURE__*/ (0, $5k7GG$react.forwardRef)($52cdbe6a6f9c351a$var$SvgComponent);
var $52cdbe6a6f9c351a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($52cdbe6a6f9c351a$var$ForwardRef);
