// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_AIchat_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $klSRb$reactjsxruntime = require("react/jsx-runtime");
var $klSRb$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $dcf0aac86c92f860$export$2e2bcd8739ae039);




const $dcf0aac86c92f860$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1691df__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1691df__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1691df__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsxs)("radialGradient", {
                        id: "1691df__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsxs)("radialGradient", {
                        id: "1691df__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("clipPath", {
                        id: "1691df__b",
                        children: /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("clipPath", {
                id: "1691df__a",
                children: /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m30.92 18.423.008-.041.615-2.85.007-.037c.078-.412-.038-.84-.311-1.149l-1.897-2.148c-.177-.2-.28-.424-.32-.651-.155-.859.57-1.76 1.53-1.536l2.731.641c.405.096.831-.023 1.137-.317l2.059-1.98c.913-.878 2.367-.003 2.095 1.263l-.615 2.85c-.092.423.022.868.304 1.187l1.897 2.148c.832.94.025 2.43-1.162 2.196l-.028-.005-.02-.005-2.731-.64c-.406-.096-.832.023-1.137.317l-2.059 1.98c-.904.869-2.336.02-2.103-1.223m5.403 1.518 2.056.482q.316.074.619.095V26.5c0 1.942-1.585 3.482-3.54 3.482H25.39l-7.879 6.738c-.767.66-1.96.115-1.952-.894.013-1.81.006-4.185 0-5.844h-3.015c-1.955 0-3.54-1.54-3.54-3.482V15.029c0-1.943 1.585-3.482 3.54-3.482h13.964c.046.795.341 1.606.96 2.306l1.51 1.71-.494 2.294c-.377 1.753.467 3.325 1.757 4.102 1.307.788 3.159.793 4.516-.512zm-9.332 5.196-2.008-.453c-.265-.06-.541.015-.741.198l-1.512 1.396c-.596.55-1.544.001-1.366-.788l.453-2.008c.06-.265-.015-.541-.198-.74l-1.396-1.513c-.55-.596-.001-1.544.788-1.366l2.008.453c.265.06.541-.015.74-.198l1.513-1.396c.596-.55 1.544-.001 1.366.788l-.453 2.008c-.06.265.015.541.198.74l1.396 1.513c.55.596.001 1.544-.788 1.366"
                })
            }),
            /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("g", {
                clipPath: "url(#1691df__a)",
                children: /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#1691df__b)",
                    children: [
                        /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#1691df__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("path", {
                            fill: "url(#1691df__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("path", {
                            fill: "url(#1691df__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("path", {
                            fill: "url(#1691df__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $klSRb$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#1691df__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $dcf0aac86c92f860$var$ForwardRef = /*#__PURE__*/ (0, $klSRb$react.forwardRef)($dcf0aac86c92f860$var$SvgComponent);
var $dcf0aac86c92f860$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($dcf0aac86c92f860$var$ForwardRef);
