// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_rocket_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $50SSz$reactjsxruntime = require("react/jsx-runtime");
var $50SSz$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ebd9dff130560518$export$2e2bcd8739ae039);




const $ebd9dff130560518$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsxs)("linearGradient", {
                        id: "83211c__d",
                        x1: -19.921,
                        x2: 186.087,
                        y1: 133.557,
                        y2: 0.603,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsxs)("linearGradient", {
                        id: "83211c__e",
                        x1: 150.995,
                        x2: 65.38,
                        y1: 45.947,
                        y2: 143.606,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsxs)("linearGradient", {
                        id: "83211c__f",
                        x1: 39.959,
                        x2: 197.223,
                        y1: 151.344,
                        y2: 57.889,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsxs)("radialGradient", {
                        id: "83211c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsxs)("radialGradient", {
                        id: "83211c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("clipPath", {
                        id: "83211c__b",
                        children: /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 23.2,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("clipPath", {
                id: "83211c__a",
                children: /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M107.022 87.22c-6.16 5.88-13.79 11.98-23.22 18.19h-.01c-.79.52-1.61.96-2.46 1.33-5.95 2.58-13 1.36-17.71-3.35l-3.54-3.54-3.54-3.54c-4.46-4.47-5.78-11.02-3.71-16.76.43-1.18.99-2.33 1.7-3.41 4.49-6.82 8.93-12.69 13.26-17.75 25.17-29.39 46.97-31.29 56.92-30.16 3.68.42 6.58 3.33 7 7 1.05 9.3-.53 28.94-24.69 51.99m1.7-35.99c-3.31-3.3-8.67-3.3-11.97 0-3.3 3.31-3.31 8.67 0 11.97s8.67 3.31 11.97 0 3.31-8.67 0-11.97m-64.61 30.71-10.48.18v.01c-6.81.12-10.67-7.8-6.35-13.06 12.21-14.85 28.66-20.14 39.39-21.35-6.47 6.82-12.76 14.84-18.82 24.04-2.06 3.13-3.29 6.63-3.74 10.18m-15.41 36.6c-.78.78-1.81 1.17-2.83 1.17-1.03 0-2.05-.39-2.83-1.17-1.56-1.57-1.56-4.1 0-5.66l10.5-10.5c1.57-1.56 4.1-1.56 5.66 0 1.56 1.57 1.56 4.1 0 5.66zm14-7c1.57-1.56 4.1-1.56 5.66 0 1.56 1.57 1.56 4.1 0 5.66l-20 20c-.78.78-1.8 1.17-2.83 1.17s-2.05-.39-2.83-1.17c-1.56-1.57-1.56-4.1 0-5.66zm9.22 9.21-11 11c-1.56 1.56-1.56 4.09 0 5.66.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17l11-11c1.56-1.56 1.56-4.09 0-5.66s-4.09-1.56-5.66 0m26.11-4.89-.18 10.44c-.12 6.85 7.85 10.65 13.13 6.29 14.761-12.18 20.041-28.55 21.271-39.27-6.48 6.12-14.39 12.41-24.05 18.77-3.08 2.03-6.55 3.3-10.17 3.77"
                })
            }),
            /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("g", {
                clipPath: "url(#83211c__a)",
                children: /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#83211c__b)",
                    children: [
                        /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 23.2,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M13 23.2h131v123.491H13z"
                        }),
                        /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#83211c__c)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 30 -25.66)"
                        }),
                        /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("path", {
                            fill: "url(#83211c__d)",
                            d: "M63.002 1.387C52.94-4.482 40.194-4.518 30.096 1.294l-101 58.132c-9.891 5.693-9.907 19.132-.03 24.847L29.4 142.322c10.082 5.833 22.83 5.826 32.904-.021l100.125-58.104c9.826-5.702 9.844-19.066.03-24.79z"
                        }),
                        /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("path", {
                            fill: "url(#83211c__e)",
                            d: "M31.878 81.831C34.118 35.728 75.58.066 124.485 2.177c48.907 2.11 86.738 41.195 84.499 87.298s-43.701 81.765-92.607 79.654c-48.906-2.11-86.738-41.195-84.499-87.298"
                        }),
                        /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("path", {
                            fill: "url(#83211c__f)",
                            d: "M-40.075 112.229c19.44-7.818 40.406-7.483 58.57-.645 36.349 13.718 57.498 5.237 72.167-28.95 7.307-17.153 21.572-31.681 41.035-39.508 38.911-15.624 83.833 1.33 100.384 37.902s-1.565 78.852-40.467 94.497c-19.506 7.845-40.512 7.477-58.705.577-36.288-13.644-57.398-5.131-72.048 29.096-7.325 17.113-21.588 31.592-41.028 39.41-38.88 15.636-83.803-1.318-100.344-37.87-16.542-36.552 1.555-78.873 40.436-94.509"
                        }),
                        /*#__PURE__*/ (0, $50SSz$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#83211c__g)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 79.293 -8.14)"
                        })
                    ]
                })
            })
        ]
    });
const $ebd9dff130560518$var$ForwardRef = /*#__PURE__*/ (0, $50SSz$react.forwardRef)($ebd9dff130560518$var$SvgComponent);
var $ebd9dff130560518$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ebd9dff130560518$var$ForwardRef);
