// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_mention_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bLgO3$jsxs, jsx as $bLgO3$jsx} from "react/jsx-runtime";
import {forwardRef as $bLgO3$forwardRef} from "react";





const $ac1696e059a6d23f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bLgO3$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bLgO3$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bLgO3$jsxs)("linearGradient", {
                        id: "e33ed0__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bLgO3$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bLgO3$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bLgO3$jsxs)("linearGradient", {
                        id: "e33ed0__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bLgO3$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bLgO3$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bLgO3$jsxs)("linearGradient", {
                        id: "e33ed0__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bLgO3$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bLgO3$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bLgO3$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bLgO3$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bLgO3$jsxs)("radialGradient", {
                        id: "e33ed0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bLgO3$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bLgO3$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bLgO3$jsxs)("radialGradient", {
                        id: "e33ed0__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bLgO3$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bLgO3$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bLgO3$jsx)("clipPath", {
                        id: "e33ed0__b",
                        children: /*#__PURE__*/ (0, $bLgO3$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bLgO3$jsx)("clipPath", {
                id: "e33ed0__a",
                children: /*#__PURE__*/ (0, $bLgO3$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M63.1 23.29c16.545-5.425 34.284-3.612 45.82 1.593 23.659 10.674 31.47 33.612 28.012 53.264-3.429 19.484-18.327 37.023-40.869 35.464-20.758-1.436-31.9-9.307-36.412-19.61-4.341-9.914-1.93-20.8 1.58-27.28 10.008-18.484 27.797-19.679 36.932-18.844 4.498.411 7.267 4.463 6.897 8.48l-4.533 49.295c15.125-.906 25.791-13.35 28.526-28.892 2.936-16.683-3.649-35.664-23.423-44.585-9.715-4.384-25.39-6.085-40.037-1.282-14.408 4.724-27.845 15.736-33.65 37.04-4.597 16.876.774 33.035 11.113 45.01 10.4 12.046 25.48 19.448 39.591 19.041 2.208-.064 4.05 1.675 4.114 3.883.063 2.208-1.675 4.05-3.883 4.114-16.923.488-34.192-8.275-45.877-21.81-11.747-13.606-18.197-32.444-12.776-52.34 6.56-24.077 22.092-37.036 38.875-42.54m29.43 81.953 4.546-49.43c-7.944-.654-21.196.653-28.81 14.717-2.707 4.998-4.363 13.237-1.286 20.263 2.688 6.14 9.53 12.492 25.55 14.45"
                })
            }),
            /*#__PURE__*/ (0, $bLgO3$jsx)("g", {
                clipPath: "url(#e33ed0__a)",
                children: /*#__PURE__*/ (0, $bLgO3$jsxs)("g", {
                    clipPath: "url(#e33ed0__b)",
                    children: [
                        /*#__PURE__*/ (0, $bLgO3$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $bLgO3$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $bLgO3$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#e33ed0__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $bLgO3$jsx)("path", {
                            fill: "url(#e33ed0__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $bLgO3$jsx)("path", {
                            fill: "url(#e33ed0__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $bLgO3$jsx)("path", {
                            fill: "url(#e33ed0__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $bLgO3$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#e33ed0__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $ac1696e059a6d23f$var$ForwardRef = /*#__PURE__*/ (0, $bLgO3$forwardRef)($ac1696e059a6d23f$var$SvgComponent);
var $ac1696e059a6d23f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ac1696e059a6d23f$var$ForwardRef);


export {$ac1696e059a6d23f$export$2e2bcd8739ae039 as default};
