// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_desktop_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $zsj4r$reactjsxruntime = require("react/jsx-runtime");
var $zsj4r$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $05f362d4756a9adf$export$2e2bcd8739ae039);




const $05f362d4756a9adf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsxs)("linearGradient", {
                        id: "fe9f57__d",
                        x1: -17.287,
                        x2: 187.234,
                        y1: 128.56,
                        y2: -3.435,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsxs)("linearGradient", {
                        id: "fe9f57__e",
                        x1: 152.4,
                        x2: 67.403,
                        y1: 41.582,
                        y2: 138.536,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsxs)("linearGradient", {
                        id: "fe9f57__f",
                        x1: 42.163,
                        x2: 198.293,
                        y1: 146.219,
                        y2: 53.439,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsxs)("radialGradient", {
                        id: "fe9f57__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.3377 -56.5812 0 56.581 53.338)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsxs)("radialGradient", {
                        id: "fe9f57__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.3377 -56.5812 0 56.581 53.338)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("clipPath", {
                        id: "fe9f57__b",
                        children: /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("rect", {
                            width: 130.055,
                            height: 122.6,
                            x: 15.398,
                            y: 19,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("clipPath", {
                id: "fe9f57__a",
                children: /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M127.029 28h-94.06c-8.84 0-16 7.16-16 16v44c0 8.84 7.16 16 16 16h36.03v24h-10.01c-2.21 0-4 1.79-4 4s1.79 4 4 4h42.02c2.21 0 4-1.79 4-4s-1.79-4-4-4h-10.01v-24h36.03c8.84 0 16-7.16 16-16V44c0-8.84-7.16-16-16-16m-47.01 20c-2.21 0-4-1.79-4-4s1.79-4 3.99-4h.01c2.21 0 4 1.79 4 4s-1.79 4-4 4"
                })
            }),
            /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("g", {
                clipPath: "url(#fe9f57__a)",
                children: /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#fe9f57__b)",
                    children: [
                        /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("rect", {
                            width: 130.055,
                            height: 122.6,
                            x: 15.398,
                            y: 19,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M15.398 19h130.055v122.6H15.398z"
                        }),
                        /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.581,
                            cy: 53.338,
                            fill: "url(#fe9f57__c)",
                            rx: 56.581,
                            ry: 53.338,
                            transform: "matrix(1 -.00132 .00148 1 32.273 -29.508)"
                        }),
                        /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("path", {
                            fill: "url(#fe9f57__d)",
                            d: "M65.037-2.657c-9.989-5.826-22.643-5.862-32.669-.092l-100.27 57.712c-9.82 5.653-9.837 18.994-.03 24.668l99.608 57.631c10.01 5.79 22.665 5.783 32.667-.021l99.403-57.685c9.755-5.661 9.773-18.928.029-24.611z"
                        }),
                        /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("path", {
                            fill: "url(#fe9f57__e)",
                            d: "M34.142 77.207c2.223-45.77 43.386-81.175 91.939-79.08 48.554 2.096 86.112 40.899 83.889 86.669s-43.385 81.175-91.939 79.08c-48.553-2.096-86.111-40.898-83.889-86.669"
                        }),
                        /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("path", {
                            fill: "url(#fe9f57__f)",
                            d: "M-37.293 107.386c19.3-7.761 40.115-7.428 58.149-.64 36.085 13.619 57.082 5.199 71.645-28.74 7.254-17.03 21.417-31.454 40.739-39.224 38.631-15.512 83.229 1.32 99.66 37.628 16.432 36.309-1.553 78.284-40.175 93.816-19.365 7.788-40.22 7.423-58.281.573-36.027-13.546-56.984-5.094-71.529 28.886-7.272 16.99-21.432 31.364-40.732 39.125-38.6 15.523-83.198-1.308-99.62-37.596-16.423-36.289 1.544-78.305 40.144-93.828"
                        }),
                        /*#__PURE__*/ (0, $zsj4r$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.581,
                            cy: 53.338,
                            fill: "url(#fe9f57__g)",
                            rx: 56.581,
                            ry: 53.338,
                            transform: "matrix(1 -.00132 .00148 1 81.215 -12.113)"
                        })
                    ]
                })
            })
        ]
    });
const $05f362d4756a9adf$var$ForwardRef = /*#__PURE__*/ (0, $zsj4r$react.forwardRef)($05f362d4756a9adf$var$SvgComponent);
var $05f362d4756a9adf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($05f362d4756a9adf$var$ForwardRef);
