// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_shoppingBag_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gTBRd$reactjsxruntime = require("react/jsx-runtime");
var $gTBRd$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0937220ab9475ac1$export$2e2bcd8739ae039);




const $0937220ab9475ac1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4461d6__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4461d6__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4461d6__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4461d6__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4461d6__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("clipPath", {
                        id: "4461d6__b",
                        children: /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("clipPath", {
                id: "4461d6__a",
                children: /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M71.17 36.83c0-4.877 3.953-8.83 8.83-8.83 4.876 0 8.83 3.953 8.83 8.83v9.406H71.17zm-8 9.406h8v13.686c0 2.21-1.79 4-4 4s-4-1.79-4-4zm0 0H46.842c-2.055 0-3.775 1.558-3.98 3.602l-6.433 64.335c-.942 9.419 6.454 17.592 15.92 17.592h55.301c9.466 0 16.863-8.173 15.921-17.592l-6.433-64.335c-.205-2.044-1.926-3.602-3.981-3.602H96.829V36.83C96.83 27.535 89.294 20 80 20S63.17 27.535 63.17 36.83zm33.66 0v13.686c0 2.21-1.792 4-4 4-2.21 0-4-1.79-4-4V46.236z"
                })
            }),
            /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("g", {
                clipPath: "url(#4461d6__a)",
                children: /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#4461d6__b)",
                    children: [
                        /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#4461d6__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("path", {
                            fill: "url(#4461d6__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("path", {
                            fill: "url(#4461d6__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("path", {
                            fill: "url(#4461d6__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $gTBRd$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#4461d6__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $0937220ab9475ac1$var$ForwardRef = /*#__PURE__*/ (0, $gTBRd$react.forwardRef)($0937220ab9475ac1$var$SvgComponent);
var $0937220ab9475ac1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0937220ab9475ac1$var$ForwardRef);
