// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_star_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $1YWqd$reactjsxruntime = require("react/jsx-runtime");
var $1YWqd$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $33e768ec8b1faad3$export$2e2bcd8739ae039);




const $33e768ec8b1faad3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "35e4cb__d",
                        x1: -24.021,
                        x2: 201.071,
                        y1: 134.5,
                        y2: -2.444,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "35e4cb__e",
                        x1: 156.616,
                        x2: 60.052,
                        y1: 36.277,
                        y2: 140.11,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "35e4cb__f",
                        x1: 39.265,
                        x2: 210.447,
                        y1: 154.441,
                        y2: 58.547,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsxs)("radialGradient", {
                        id: "35e4cb__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.455 52.6)scale(60.2331)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsxs)("radialGradient", {
                        id: "35e4cb__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 52.682 88.47)scale(60.2331)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("clipPath", {
                        id: "35e4cb__b",
                        children: /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 10.773,
                            y: 10.775,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("clipPath", {
                id: "35e4cb__a",
                children: /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "m88.208 33.102 9.66 19.573c1.333 2.701 3.91 4.574 6.891 5.007l21.601 3.138c7.506 1.091 10.504 10.316 5.072 15.61l-15.631 15.237c-2.157 2.102-3.141 5.131-2.632 8.1l3.69 21.514c1.282 7.476-6.565 13.177-13.279 9.647l-19.32-10.157c-2.666-1.402-5.851-1.402-8.517 0l-19.32 10.157c-6.715 3.53-14.562-2.171-13.28-9.647l3.69-21.514c.51-2.969-.475-5.998-2.632-8.1L28.571 76.43c-5.432-5.295-2.435-14.52 5.072-15.61l21.6-3.14c2.981-.432 5.558-2.305 6.891-5.006l9.66-19.573c3.358-6.802 13.057-6.802 16.414 0"
                })
            }),
            /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("g", {
                clipPath: "url(#35e4cb__a)",
                children: /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#35e4cb__b)",
                    children: [
                        /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 10.773,
                            y: 10.775,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M10.773 10.775h138.45v138.45H10.772z"
                        }),
                        /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("circle", {
                            cx: 89.055,
                            cy: 16.146,
                            r: 60.233,
                            fill: "url(#35e4cb__c)",
                            transform: "rotate(-.08 89.055 16.146)"
                        }),
                        /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("path", {
                            fill: "url(#35e4cb__d)",
                            d: "M63.616-13.68c-10.633-6.58-24.104-6.62-34.777-.105L-77.903 51.39c-10.454 6.383-10.471 21.45-.031 27.856l106.037 65.081c10.656 6.54 24.127 6.532 34.775-.023l105.818-65.142c10.385-6.393 10.404-21.376.032-27.793z"
                        }),
                        /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("path", {
                            fill: "url(#35e4cb__e)",
                            d: "M30.726 76.508c2.366-51.688 46.185-91.67 97.872-89.304 51.688 2.367 91.67 46.186 89.304 97.873-2.367 51.687-46.186 91.67-97.873 89.303-51.687-2.366-91.67-46.185-89.303-97.872"
                        }),
                        /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("path", {
                            fill: "url(#35e4cb__f)",
                            d: "M-45.32 110.588c20.546-8.765 42.705-8.389 61.902-.723 38.415 15.38 60.767 5.871 76.27-32.455 7.722-19.233 22.799-35.52 43.368-44.295 41.124-17.517 88.601 1.49 106.093 42.493s-1.654 88.404-42.769 105.944c-20.615 8.794-42.816 8.382-62.042.647-38.353-15.297-60.662-5.753-76.146 32.62-7.742 19.186-22.815 35.418-43.36 44.183-41.092 17.53-88.568-1.477-106.05-42.457-17.483-40.98 1.643-88.427 42.735-105.957"
                        }),
                        /*#__PURE__*/ (0, $1YWqd$reactjsxruntime.jsx)("circle", {
                            cx: 141.153,
                            cy: 35.788,
                            r: 60.233,
                            fill: "url(#35e4cb__g)",
                            transform: "rotate(-.08 141.153 35.788)"
                        })
                    ]
                })
            })
        ]
    });
const $33e768ec8b1faad3$var$ForwardRef = /*#__PURE__*/ (0, $1YWqd$react.forwardRef)($33e768ec8b1faad3$var$SvgComponent);
var $33e768ec8b1faad3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($33e768ec8b1faad3$var$ForwardRef);
