// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_rotateCW_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $7b7vZ$reactjsxruntime = require("react/jsx-runtime");
var $7b7vZ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2ff52cbf0d94adfe$export$2e2bcd8739ae039);




const $2ff52cbf0d94adfe$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6df35f__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6df35f__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6df35f__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6df35f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6df35f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("clipPath", {
                        id: "6df35f__b",
                        children: /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("clipPath", {
                id: "6df35f__a",
                children: /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M26.5 80.398c0-28.718 23.281-52 52-52 19.918 0 37.227 11.2 45.962 27.655l-15.454-5.156c-2.095-.7-4.361.432-5.06 2.528s.433 4.361 2.528 5.06l24.275 8.1c2.095.699 4.361-.433 5.06-2.529l8.099-24.274c.7-2.096-.432-4.361-2.528-5.06-2.095-.7-4.361.432-5.06 2.528l-4.935 14.79C121.269 33.208 101.384 20.397 78.5 20.397c-33.137 0-60 26.863-60 60s26.863 60 60 60c25.917 0 47.986-16.429 56.38-39.428.757-2.075-.311-4.372-2.387-5.13-2.075-.757-4.371.312-5.129 2.387-7.278 19.943-26.415 34.171-48.864 34.171-28.719 0-52-23.281-52-52"
                })
            }),
            /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6df35f__a)",
                children: /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6df35f__b)",
                    children: [
                        /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#6df35f__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("path", {
                            fill: "url(#6df35f__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("path", {
                            fill: "url(#6df35f__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("path", {
                            fill: "url(#6df35f__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $7b7vZ$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#6df35f__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $2ff52cbf0d94adfe$var$ForwardRef = /*#__PURE__*/ (0, $7b7vZ$react.forwardRef)($2ff52cbf0d94adfe$var$SvgComponent);
var $2ff52cbf0d94adfe$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2ff52cbf0d94adfe$var$ForwardRef);
