// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_revert_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $4ybnz$jsxs, jsx as $4ybnz$jsx} from "react/jsx-runtime";
import {forwardRef as $4ybnz$forwardRef} from "react";





const $81e3500df527c319$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4ybnz$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4ybnz$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4ybnz$jsxs)("linearGradient", {
                        id: "2ed0ab__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4ybnz$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4ybnz$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4ybnz$jsxs)("linearGradient", {
                        id: "2ed0ab__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4ybnz$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4ybnz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4ybnz$jsxs)("linearGradient", {
                        id: "2ed0ab__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4ybnz$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4ybnz$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4ybnz$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4ybnz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4ybnz$jsxs)("radialGradient", {
                        id: "2ed0ab__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4ybnz$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4ybnz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4ybnz$jsxs)("radialGradient", {
                        id: "2ed0ab__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4ybnz$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4ybnz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4ybnz$jsx)("clipPath", {
                        id: "2ed0ab__b",
                        children: /*#__PURE__*/ (0, $4ybnz$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4ybnz$jsx)("clipPath", {
                id: "2ed0ab__a",
                children: /*#__PURE__*/ (0, $4ybnz$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m10.857 27.221 5.516-4.376c.541-.43 1.327-.338 1.756.202.43.541.339 1.328-.202 1.756l-7.755 6.153c-.541.43-1.327.338-1.757-.202l-6.152-7.756c-.43-.54-.338-1.327.202-1.756.541-.429 1.327-.338 1.757.203l4.156 5.24c.526-9.277 8.215-16.638 17.623-16.638 9.749 0 17.651 7.903 17.651 17.651 0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25c0-8.368-6.783-15.151-15.151-15.151-8.208 0-14.892 6.527-15.144 14.674m-3.369 9.482c0-.69.56-1.25 1.25-1.25h34.516c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25H8.738c-.69 0-1.25-.56-1.25-1.25"
                })
            }),
            /*#__PURE__*/ (0, $4ybnz$jsx)("g", {
                clipPath: "url(#2ed0ab__a)",
                children: /*#__PURE__*/ (0, $4ybnz$jsxs)("g", {
                    clipPath: "url(#2ed0ab__b)",
                    children: [
                        /*#__PURE__*/ (0, $4ybnz$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $4ybnz$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $4ybnz$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#2ed0ab__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $4ybnz$jsx)("path", {
                            fill: "url(#2ed0ab__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $4ybnz$jsx)("path", {
                            fill: "url(#2ed0ab__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $4ybnz$jsx)("path", {
                            fill: "url(#2ed0ab__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $4ybnz$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#2ed0ab__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $81e3500df527c319$var$ForwardRef = /*#__PURE__*/ (0, $4ybnz$forwardRef)($81e3500df527c319$var$SvgComponent);
var $81e3500df527c319$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($81e3500df527c319$var$ForwardRef);


export {$81e3500df527c319$export$2e2bcd8739ae039 as default};
