// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_thumbsUp_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8vVX8$reactjsxruntime = require("react/jsx-runtime");
var $8vVX8$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b07b5d086969c783$export$2e2bcd8739ae039);




const $b07b5d086969c783$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9ea0fe__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9ea0fe__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9ea0fe__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsxs)("radialGradient", {
                        id: "9ea0fe__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsxs)("radialGradient", {
                        id: "9ea0fe__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("clipPath", {
                        id: "9ea0fe__b",
                        children: /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("clipPath", {
                id: "9ea0fe__a",
                children: /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M19.274 37.614c1.57.585 3.23.886 4.907.886l-.006-.006h6.982l.043-.003h1.935c.716 0 1.458-.197 2.032-.636.581-.447.968-1.132.968-2.055 0-.298-.043-.825-.298-1.341-.12-.25-.292-.493-.527-.705 1.195-.07 2.204-.935 2.448-2.118.22-1.075-.241-2.107-1.055-2.692 1.11-.275 1.949-1.25 2.009-2.439.049-.963-.421-1.823-1.16-2.327 1.048-.315 1.828-1.261 1.888-2.408.077-1.516-1.132-2.788-2.651-2.788h-7.945c-.433 0-.742-.422-.613-.834l.573-1.84c.513-1.646.84-3.342.98-5.06l.08-.994c.12-1.481-1.052-2.754-2.542-2.754-.983 0-1.877.567-2.301 1.453l-2.669 5.59q-.375.782-.857 1.5l-.066.101c-1.51 2.248-3.876 3.742-6.506 4.167v15.828q.94.204 1.843.541zm-6.85-1.777V20.435h-.717c-1.742 0-3.152 1.41-3.152 3.153v9.09c0 1.744 1.41 3.154 3.152 3.154h.353q.181 0 .363.005"
                })
            }),
            /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("g", {
                clipPath: "url(#9ea0fe__a)",
                children: /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#9ea0fe__b)",
                    children: [
                        /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#9ea0fe__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("path", {
                            fill: "url(#9ea0fe__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("path", {
                            fill: "url(#9ea0fe__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("path", {
                            fill: "url(#9ea0fe__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $8vVX8$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#9ea0fe__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $b07b5d086969c783$var$ForwardRef = /*#__PURE__*/ (0, $8vVX8$react.forwardRef)($b07b5d086969c783$var$SvgComponent);
var $b07b5d086969c783$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b07b5d086969c783$var$ForwardRef);
