// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_badgeVerified_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $4BHDM$jsxs, jsx as $4BHDM$jsx} from "react/jsx-runtime";
import {forwardRef as $4BHDM$forwardRef} from "react";





const $877fae54b4da86ce$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4BHDM$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4BHDM$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4BHDM$jsxs)("linearGradient", {
                        id: "eec53c__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4BHDM$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4BHDM$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4BHDM$jsxs)("linearGradient", {
                        id: "eec53c__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4BHDM$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4BHDM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4BHDM$jsxs)("linearGradient", {
                        id: "eec53c__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4BHDM$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4BHDM$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4BHDM$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4BHDM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4BHDM$jsxs)("radialGradient", {
                        id: "eec53c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4BHDM$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4BHDM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4BHDM$jsxs)("radialGradient", {
                        id: "eec53c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4BHDM$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4BHDM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4BHDM$jsx)("clipPath", {
                        id: "eec53c__b",
                        children: /*#__PURE__*/ (0, $4BHDM$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4BHDM$jsx)("clipPath", {
                id: "eec53c__a",
                children: /*#__PURE__*/ (0, $4BHDM$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M37.883 19.314c1.631.925 2.738 2.678 2.738 4.688h.004c0 2.019-1.115 3.775-2.76 4.7-.645.362-.946 1.098-.748 1.813.22.78.267 1.632.09 2.514-.418 2.105-2.1 3.792-4.206 4.21-.886.177-1.747.13-2.53-.095-.703-.198-1.434.1-1.8.732-.934 1.614-2.674 2.704-4.671 2.704s-3.736-1.09-4.67-2.704c-.367-.633-1.094-.93-1.8-.732-.784.224-1.645.272-2.531.095-2.105-.417-3.789-2.105-4.206-4.21-.176-.883-.13-1.735.09-2.514.198-.715-.103-1.45-.749-1.812-1.644-.926-2.759-2.69-2.759-4.701s1.102-3.762 2.738-4.688c.624-.353.912-1.09.74-1.765-.172-.676-.22-1.399-.112-2.148.34-2.329 2.222-4.201 4.55-4.537.762-.112 1.464-.07 2.179.125.714.193 1.446-.117 1.803-.762.917-1.67 2.69-2.807 4.727-2.807 2.036 0 3.81 1.136 4.727 2.807.357.645 1.123.942 1.803.762.68-.181 1.412-.237 2.174-.125 2.333.335 4.21 2.208 4.55 4.537.108.749.07 1.455-.112 2.148-.18.693.117 1.412.74 1.765m-5.822 1.19c.51-.465.547-1.256.082-1.766s-1.256-.547-1.766-.082l-.138.126-.03.028-.018.015-.017.016-.077.07-.043.04-.022.02-.011.01-.012.01-.099.09-.493.45-.145.132-.154.14-.161.147-.169.154-.021.02-.022.02-.022.02-.022.02-.022.02-.022.02-.022.02-.022.02-.182.165-.093.085-.094.086-.024.022-.024.022-.048.044-.024.022-.025.022-.024.022-.024.022-.05.045-.024.023-.025.022-.025.023-.025.022-.024.023-.026.023-.025.023-.025.023-.025.023-.025.023-.026.023-.025.023-.051.047-.207.188-.211.193-.214.195-.027.024-.027.025-.054.05-.027.024-.028.025-.027.024-.027.025-.055.05-.054.05-.11.1-.028.025-.028.026-.027.025-.028.025-.055.05-.056.051-.055.05-.056.052-.028.025-.028.026-.056.05-.028.026-.028.026-.028.025-.028.025-.056.052-.056.05-.113.103-.028.026-.028.025-.056.052-.028.026-.029.025-.056.051-.112.103-.45.41-.224.204-.028.025-.027.026-.028.025-.028.025-.11.101-.221.201-.11.1-.027.025-.027.024-.054.05-.054.05-.027.024-.027.024-.108.098-.106.097-.026.024-.027.024-.053.048-1.532 1.397-.02.019-.021.019-.021.019-.02.018-.041.038-.04.036-.158.143-.149.136-.036.033-.035.032-.035.032-.017.016-.017.015-.029.026-3.896-3.633c-.505-.471-1.296-.444-1.767.061s-.443 1.296.062 1.767l4.739 4.42c.476.444 1.213.448 1.695.01l-.843-.925.843.924.001-.001.005-.004v-.001l.004-.004.005-.004.003-.002.003-.003.006-.006.033-.03.047-.042.128-.117.082-.075.044-.04.012-.011.012-.011.024-.022.103-.094.027-.025.014-.012.014-.013.015-.013.014-.013.03-.027.06-.055.062-.057.132-.12.017-.015.017-.016.017-.016.018-.016.017-.016.018-.016.036-.033.037-.033.018-.017.019-.017.018-.017.02-.017.018-.018.02-.017.156-.143.04-.037.02-.018.021-.019.021-.019.02-.019.021-.019.021-.019.35-.318.023-.02.022-.022.023-.02.023-.021.093-.085.047-.043.024-.021.023-.022.024-.022.024-.021.024-.022.024-.022.097-.088.024-.023.025-.022.024-.023.025-.022.05-.045.05-.046.025-.023.025-.022.05-.046.102-.093.051-.047.026-.023.026-.024.104-.094.026-.024.026-.024.026-.024.026-.024.027-.024.026-.024.053-.048.027-.024.026-.025.027-.024.026-.024.027-.025.027-.024.054-.049.108-.099.109-.099.054-.05.027-.024.028-.025.11-.1.027-.025.028-.026.055-.05.111-.101.028-.025.028-.026.027-.025.028-.026.028-.025.028-.025.028-.026.028-.025.028-.026.028-.025.056-.051.028-.026.028-.025.056-.051.113-.103.225-.205.45-.41.028-.026.028-.026.056-.05.112-.103.056-.05.056-.052.111-.101.111-.101.055-.05.028-.026.028-.025.219-.2.109-.1.054-.049.054-.049.107-.098.107-.097.106-.096.026-.024.026-.024.053-.048.207-.189.102-.092.025-.023.025-.023.025-.023.026-.023.05-.046.024-.023.025-.022.05-.045.049-.045.048-.044.025-.022.024-.023.024-.021.024-.022.024-.022.024-.022.187-.17.357-.326.022-.02.021-.02.042-.038.084-.076.02-.019.02-.018.041-.037.08-.073.078-.07.076-.07.074-.067.036-.033.035-.033.07-.063.017-.016.017-.015.017-.015.016-.016.017-.015.016-.015.017-.014.016-.015.063-.057.118-.109.015-.012.014-.013.027-.025.027-.025.013-.012.013-.012.187-.17.142-.13z"
                })
            }),
            /*#__PURE__*/ (0, $4BHDM$jsx)("g", {
                clipPath: "url(#eec53c__a)",
                children: /*#__PURE__*/ (0, $4BHDM$jsxs)("g", {
                    clipPath: "url(#eec53c__b)",
                    children: [
                        /*#__PURE__*/ (0, $4BHDM$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $4BHDM$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $4BHDM$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#eec53c__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $4BHDM$jsx)("path", {
                            fill: "url(#eec53c__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $4BHDM$jsx)("path", {
                            fill: "url(#eec53c__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $4BHDM$jsx)("path", {
                            fill: "url(#eec53c__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $4BHDM$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#eec53c__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $877fae54b4da86ce$var$ForwardRef = /*#__PURE__*/ (0, $4BHDM$forwardRef)($877fae54b4da86ce$var$SvgComponent);
var $877fae54b4da86ce$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($877fae54b4da86ce$var$ForwardRef);


export {$877fae54b4da86ce$export$2e2bcd8739ae039 as default};
