// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileZip_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $7gwdY$reactjsxruntime = require("react/jsx-runtime");
var $7gwdY$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $adc9eddc36fca2fb$export$2e2bcd8739ae039);




const $adc9eddc36fca2fb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e79f68__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e79f68__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e79f68__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsxs)("radialGradient", {
                        id: "e79f68__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsxs)("radialGradient", {
                        id: "e79f68__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("clipPath", {
                        id: "e79f68__b",
                        children: /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("clipPath", {
                id: "e79f68__a",
                children: /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m52.673 18.81 19.776 18.918v.012c1.44 1.368 2.25 3.276 2.25 5.268v28.926c0 4.02-3.246 7.278-7.242 7.278H28.535c-4.002 0-7.242-3.258-7.242-7.278v-47.85c0-4.02 3.246-7.278 7.242-7.278h19.146c1.86 0 3.648.714 4.992 2.004m2.85 23.106c0-1.326-1.074-2.4-2.4-2.4h-3.042V35.52h3.048c1.326 0 2.4-1.074 2.4-2.4s-1.074-2.4-2.4-2.4h-3.048v-1.998c0-1.326-1.074-2.4-2.4-2.4h-4.794c-1.326 0-2.4 1.074-2.4 2.4s1.074 2.4 2.4 2.4h2.394v3.996h-2.394c-1.326 0-2.4 1.074-2.4 2.4s1.074 2.4 2.4 2.4h2.394v3.996h-2.394c-1.326 0-2.4 1.074-2.4 2.4s1.074 2.4 2.4 2.4h2.394v3.996h-1.794c-1.326 0-2.4 1.074-2.4 2.4v12.054c0 2.646 2.154 4.8 4.8 4.8h3.606c2.646 0 4.8-2.154 4.8-4.8V55.11c0-1.326-1.074-2.4-2.4-2.4h-1.818v-8.394h3.048c1.326 0 2.4-1.074 2.4-2.4m-9.636 20.418H49.5v4.824h-3.612z"
                })
            }),
            /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("g", {
                clipPath: "url(#e79f68__a)",
                children: /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#e79f68__b)",
                    children: [
                        /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#e79f68__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("path", {
                            fill: "url(#e79f68__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("path", {
                            fill: "url(#e79f68__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("path", {
                            fill: "url(#e79f68__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $7gwdY$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#e79f68__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $adc9eddc36fca2fb$var$ForwardRef = /*#__PURE__*/ (0, $7gwdY$react.forwardRef)($adc9eddc36fca2fb$var$SvgComponent);
var $adc9eddc36fca2fb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($adc9eddc36fca2fb$var$ForwardRef);
