// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_artboard_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8tYPh$reactjsxruntime = require("react/jsx-runtime");
var $8tYPh$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $fc1de1a19508fa03$export$2e2bcd8739ae039);




const $fc1de1a19508fa03$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsxs)("linearGradient", {
                        id: "56d8eb__d",
                        x1: -32.085,
                        x2: 187.45,
                        y1: 127.603,
                        y2: -14.082,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsxs)("linearGradient", {
                        id: "56d8eb__e",
                        x1: 150.059,
                        x2: 58.822,
                        y1: 34.24,
                        y2: 138.311,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsxs)("linearGradient", {
                        id: "56d8eb__f",
                        x1: 31.728,
                        x2: 199.32,
                        y1: 146.558,
                        y2: 46.967,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsxs)("radialGradient", {
                        id: "56d8eb__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 57.2532 -60.7348 0 60.735 57.253)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsxs)("radialGradient", {
                        id: "56d8eb__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 57.2532 -60.7348 0 60.735 57.253)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("clipPath", {
                        id: "56d8eb__b",
                        children: /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("rect", {
                            width: 139.603,
                            height: 131.6,
                            x: 3,
                            y: 10,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("clipPath", {
                id: "56d8eb__a",
                children: /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M40 24V14c0-2.21-1.79-4-4-4s-4 1.79-4 4v10c0 2.21 1.79 4 4 4s4-1.79 4-4m39 46.02c0 5.5 4.5 9.98 10.03 9.98h34.98v48c0 6.63-5.34 12-11.93 12H47.93C41.34 140 36 134.63 36 128V48c0-6.63 5.34-12 11.93-12H79zM121.3 72H89.04c-1.12 0-2.03-.89-2.03-1.98V38.89q.075.066.155.132c.116.098.235.198.345.308l32.81 31.59c.35.34.67.7.98 1.08M14 32h10c2.21 0 4 1.79 4 4s-1.79 4-4 4H14c-2.21 0-4-1.79-4-4s1.79-4 4-4"
                })
            }),
            /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("g", {
                clipPath: "url(#56d8eb__a)",
                children: /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#56d8eb__b)",
                    children: [
                        /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("rect", {
                            width: 139.603,
                            height: 131.6,
                            x: 3,
                            y: 10,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3 10h139.603v131.6H3z"
                        }),
                        /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("ellipse", {
                            cx: 60.735,
                            cy: 57.253,
                            fill: "url(#56d8eb__c)",
                            rx: 60.735,
                            ry: 57.253,
                            transform: "matrix(1 -.00132 .00148 1 21.117 -42.068)"
                        }),
                        /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("path", {
                            fill: "url(#56d8eb__d)",
                            d: "M56.282-13.246c-10.721-6.254-24.305-6.293-35.066-.1l-107.632 61.95c-10.54 6.067-10.558 20.388-.032 26.478l106.92 61.861c10.746 6.216 24.33 6.209 35.066-.022l106.7-61.919c10.471-6.077 10.49-20.318.032-26.418z"
                        }),
                        /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("path", {
                            fill: "url(#56d8eb__e)",
                            d: "M23.12 72.48c2.386-49.13 46.57-87.134 98.688-84.885 52.118 2.25 92.434 43.9 90.048 93.031-2.387 49.13-46.571 87.135-98.688 84.885s-92.434-43.9-90.048-93.03"
                        }),
                        /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("path", {
                            fill: "url(#56d8eb__f)",
                            d: "M-53.56 104.875c20.716-8.331 43.06-7.974 62.416-.687 38.735 14.618 61.273 5.58 76.906-30.85 7.786-18.28 22.988-33.763 43.729-42.103 41.466-16.65 89.338 1.416 106.976 40.39s-1.667 84.031-43.124 100.703c-20.787 8.359-43.173 7.968-62.56.615-38.672-14.54-61.167-5.468-76.78 31.006-7.806 18.237-23.005 33.666-43.722 41.998-41.433 16.662-89.305-1.404-106.933-40.357-17.628-38.952 1.657-84.052 43.091-100.715"
                        }),
                        /*#__PURE__*/ (0, $8tYPh$reactjsxruntime.jsx)("ellipse", {
                            cx: 60.735,
                            cy: 57.253,
                            fill: "url(#56d8eb__g)",
                            rx: 60.735,
                            ry: 57.253,
                            transform: "matrix(1 -.00132 .00148 1 73.644 -23.398)"
                        })
                    ]
                })
            })
        ]
    });
const $fc1de1a19508fa03$var$ForwardRef = /*#__PURE__*/ (0, $8tYPh$react.forwardRef)($fc1de1a19508fa03$var$SvgComponent);
var $fc1de1a19508fa03$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($fc1de1a19508fa03$var$ForwardRef);
