// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_userAvatar_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $fWOFw$reactjsxruntime = require("react/jsx-runtime");
var $fWOFw$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $dea5c62dab96109e$export$2e2bcd8739ae039);




const $dea5c62dab96109e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsxs)("linearGradient", {
                        id: "cfbbd9__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsxs)("linearGradient", {
                        id: "cfbbd9__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsxs)("linearGradient", {
                        id: "cfbbd9__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsxs)("radialGradient", {
                        id: "cfbbd9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsxs)("radialGradient", {
                        id: "cfbbd9__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("clipPath", {
                        id: "cfbbd9__b",
                        children: /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("clipPath", {
                id: "cfbbd9__a",
                children: /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M34.21 36.833C31.4 39.048 27.855 40.37 24 40.37c-3.84 0-7.372-1.311-10.176-3.51q.141-.148.23-.345c1.72-3.8 5.556-6.46 9.947-6.46 4.355 0 8.163 2.618 9.903 6.367.075.162.18.3.306.41m1.849-1.7C33.88 30.67 29.284 27.554 24 27.554s-9.88 3.113-12.059 7.578C9.187 32.185 7.5 28.225 7.5 23.87c0-9.113 7.387-16.5 16.5-16.5s16.5 7.387 16.5 16.5c0 4.354-1.686 8.314-4.441 11.262M19.207 18.844c0-2.941 2.22-5.189 4.794-5.189s4.794 2.248 4.794 5.19c0 2.94-2.22 5.188-4.794 5.188s-4.794-2.247-4.794-5.189m4.794-7.689c-4.102 0-7.294 3.518-7.294 7.69 0 4.17 3.192 7.688 7.294 7.688s7.294-3.518 7.294-7.689c0-4.17-3.193-7.689-7.294-7.689"
                })
            }),
            /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("g", {
                clipPath: "url(#cfbbd9__a)",
                children: /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#cfbbd9__b)",
                    children: [
                        /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#cfbbd9__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("path", {
                            fill: "url(#cfbbd9__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("path", {
                            fill: "url(#cfbbd9__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("path", {
                            fill: "url(#cfbbd9__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $fWOFw$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#cfbbd9__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $dea5c62dab96109e$var$ForwardRef = /*#__PURE__*/ (0, $fWOFw$react.forwardRef)($dea5c62dab96109e$var$SvgComponent);
var $dea5c62dab96109e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($dea5c62dab96109e$var$ForwardRef);
