// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_CodeBrackets_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $1arRm$reactjsxruntime = require("react/jsx-runtime");
var $1arRm$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4dbfa0024ea85120$export$2e2bcd8739ae039);




const $4dbfa0024ea85120$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsxs)("linearGradient", {
                        id: "574a20__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsxs)("linearGradient", {
                        id: "574a20__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsxs)("linearGradient", {
                        id: "574a20__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsxs)("radialGradient", {
                        id: "574a20__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsxs)("radialGradient", {
                        id: "574a20__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("clipPath", {
                        id: "574a20__b",
                        children: /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("clipPath", {
                id: "574a20__a",
                children: /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M74.4 19.2H21.6c-5.304 0-9.6 4.296-9.6 9.6v38.4c0 5.304 4.296 9.6 9.6 9.6h52.8c5.304 0 9.6-4.296 9.6-9.6V28.8c0-5.304-4.296-9.6-9.6-9.6M37.65 57.786c.906.972.852 2.49-.114 3.39-.462.432-1.05.648-1.638.648-.642 0-1.284-.258-1.758-.762l-11.034-11.82c-.882-.948-.858-2.418.06-3.336L34.2 34.872c.936-.936 2.46-.936 3.396 0s.936 2.454 0 3.396l-9.39 9.39 9.45 10.122zm17.772-25.998L45.174 65.604c-.312 1.038-1.266 1.704-2.298 1.704q-.344-.001-.696-.102c-1.266-.384-1.986-1.722-1.602-2.994l10.248-33.816c.384-1.266 1.722-1.986 2.994-1.602 1.266.384 1.986 1.722 1.602 2.994m17.412 18.3L61.8 61.122c-.468.468-1.08.702-1.698.702s-1.23-.234-1.698-.702c-.936-.936-.936-2.454 0-3.396l9.39-9.39-9.45-10.122c-.906-.972-.852-2.49.114-3.39.966-.906 2.49-.852 3.39.114l11.034 11.82c.882.948.858 2.418-.06 3.336z"
                })
            }),
            /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("g", {
                clipPath: "url(#574a20__a)",
                children: /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#574a20__b)",
                    children: [
                        /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#574a20__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("path", {
                            fill: "url(#574a20__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("path", {
                            fill: "url(#574a20__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("path", {
                            fill: "url(#574a20__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $1arRm$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#574a20__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $4dbfa0024ea85120$var$ForwardRef = /*#__PURE__*/ (0, $1arRm$react.forwardRef)($4dbfa0024ea85120$var$SvgComponent);
var $4dbfa0024ea85120$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4dbfa0024ea85120$var$ForwardRef);
