// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_3D_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $b3fh2$reactjsxruntime = require("react/jsx-runtime");
var $b3fh2$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7cf8186539c98435$export$2e2bcd8739ae039);




const $7cf8186539c98435$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3964ad__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3964ad__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3964ad__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3964ad__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3964ad__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("clipPath", {
                        id: "3964ad__b",
                        children: /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("clipPath", {
                id: "3964ad__a",
                children: /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M38.19 14.18 26.196 7.06c-1.253-.743-2.81-.747-4.067-.01L9.975 14.184c-.324.19-.615.422-.868.687l14.974 7.987 15.1-7.847c-.28-.324-.613-.606-.991-.83M8 17.633q0-.254.032-.502l14.884 7.94v16.474q-.36-.116-.695-.302L10.066 34.53C8.79 33.825 8 32.484 8 31.028zm17.416 23.905q.355-.118.69-.304l11.993-6.701c1.265-.707 2.049-2.043 2.049-3.492V17.62q0-.145-.01-.289l-14.722 7.651z"
                })
            }),
            /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("g", {
                clipPath: "url(#3964ad__a)",
                children: /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#3964ad__b)",
                    children: [
                        /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#3964ad__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("path", {
                            fill: "url(#3964ad__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("path", {
                            fill: "url(#3964ad__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("path", {
                            fill: "url(#3964ad__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $b3fh2$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#3964ad__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $7cf8186539c98435$var$ForwardRef = /*#__PURE__*/ (0, $b3fh2$react.forwardRef)($7cf8186539c98435$var$SvgComponent);
var $7cf8186539c98435$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7cf8186539c98435$var$ForwardRef);
