// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_graphBarChart_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lJzvr$jsxs, jsx as $lJzvr$jsx} from "react/jsx-runtime";
import {forwardRef as $lJzvr$forwardRef} from "react";





const $36a1b4e6fd5da1f7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lJzvr$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lJzvr$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lJzvr$jsxs)("linearGradient", {
                        id: "30a420__d",
                        x1: -32.175,
                        x2: 194.7,
                        y1: 136.536,
                        y2: -9.887,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lJzvr$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lJzvr$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lJzvr$jsxs)("linearGradient", {
                        id: "30a420__e",
                        x1: 156.055,
                        x2: 61.768,
                        y1: 40.051,
                        y2: 147.602,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lJzvr$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lJzvr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lJzvr$jsxs)("linearGradient", {
                        id: "30a420__f",
                        x1: 33.771,
                        x2: 206.966,
                        y1: 156.124,
                        y2: 53.203,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lJzvr$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lJzvr$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lJzvr$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lJzvr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lJzvr$jsxs)("radialGradient", {
                        id: "30a420__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 59.1674 -62.7654 0 62.765 59.167)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lJzvr$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lJzvr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lJzvr$jsxs)("radialGradient", {
                        id: "30a420__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 59.1674 -62.7654 0 62.765 59.167)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lJzvr$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lJzvr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lJzvr$jsx)("clipPath", {
                        id: "30a420__b",
                        children: /*#__PURE__*/ (0, $lJzvr$jsx)("rect", {
                            width: 144.27,
                            height: 136,
                            x: 4.082,
                            y: 15,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lJzvr$jsx)("clipPath", {
                id: "30a420__a",
                children: /*#__PURE__*/ (0, $lJzvr$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M24.21 136h111.791c2.21 0 4 1.79 4 4s-1.79 4-4 4H20.211c-2.21 0-4-1.79-4-4V24c0-2.21 1.79-4 4-4s4 1.79 4 4zM44 71.05h4c4.418 0 8 3.582 8 8V116c0 4.418-3.582 8-8 8h-4c-4.418 0-8-3.582-8-8V79.05c0-4.418 3.582-8 8-8M120 96h-4c-4.418 0-8 3.582-8 8v12c0 4.418 3.582 8 8 8h4c4.418 0 8-3.582 8-8v-12c0-4.418-3.582-8-8-8M80 40h4c4.418 0 8 3.582 8 8v68c0 4.418-3.582 8-8 8h-4c-4.418 0-8-3.582-8-8V48c0-4.418 3.582-8 8-8"
                })
            }),
            /*#__PURE__*/ (0, $lJzvr$jsx)("g", {
                clipPath: "url(#30a420__a)",
                children: /*#__PURE__*/ (0, $lJzvr$jsxs)("g", {
                    clipPath: "url(#30a420__b)",
                    children: [
                        /*#__PURE__*/ (0, $lJzvr$jsx)("rect", {
                            width: 144.27,
                            height: 136,
                            x: 4.082,
                            y: 15,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $lJzvr$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M4.082 15h144.27v136H4.082z"
                        }),
                        /*#__PURE__*/ (0, $lJzvr$jsx)("ellipse", {
                            cx: 62.765,
                            cy: 59.167,
                            fill: "url(#30a420__c)",
                            rx: 62.765,
                            ry: 59.167,
                            transform: "matrix(1 -.00132 .00148 1 22.805 -38.81)"
                        }),
                        /*#__PURE__*/ (0, $lJzvr$jsx)("path", {
                            fill: "url(#30a420__d)",
                            d: "M59.146-9.023c-11.08-6.463-25.118-6.503-36.239-.102l-111.23 64.02c-10.893 6.27-10.912 21.07-.033 27.364L22.14 146.188c11.105 6.424 25.143 6.416 36.238-.023l110.267-63.99c10.821-6.28 10.841-20.997.033-27.3z"
                        }),
                        /*#__PURE__*/ (0, $lJzvr$jsx)("path", {
                            fill: "url(#30a420__e)",
                            d: "M24.872 79.57C27.338 28.797 73-10.478 126.86-8.154c53.86 2.325 95.524 45.369 93.058 96.142s-48.127 90.048-101.988 87.723c-53.86-2.324-95.523-45.368-93.058-96.141"
                        }),
                        /*#__PURE__*/ (0, $lJzvr$jsx)("path", {
                            fill: "url(#30a420__f)",
                            d: "M-54.37 113.047c21.41-8.61 44.5-8.24 64.504-.71 40.03 15.108 63.322 5.767 79.477-31.881 8.047-18.892 23.757-34.892 45.191-43.511 42.853-17.207 92.326 1.463 110.553 41.74s-1.723 86.841-44.566 104.071c-21.482 8.638-44.616 8.234-64.651.635-39.965-15.026-63.213-5.651-79.347 32.043-8.067 18.847-23.775 34.792-45.184 43.402-42.82 17.219-92.292-1.451-110.509-41.706s1.713-86.863 44.532-104.083"
                        }),
                        /*#__PURE__*/ (0, $lJzvr$jsx)("ellipse", {
                            cx: 62.765,
                            cy: 59.167,
                            fill: "url(#30a420__g)",
                            rx: 62.765,
                            ry: 59.167,
                            transform: "matrix(1 -.00132 .00148 1 77.09 -19.515)"
                        })
                    ]
                })
            })
        ]
    });
const $36a1b4e6fd5da1f7$var$ForwardRef = /*#__PURE__*/ (0, $lJzvr$forwardRef)($36a1b4e6fd5da1f7$var$SvgComponent);
var $36a1b4e6fd5da1f7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($36a1b4e6fd5da1f7$var$ForwardRef);


export {$36a1b4e6fd5da1f7$export$2e2bcd8739ae039 as default};
