// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_chartAreaStack_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $2IB0T$reactjsxruntime = require("react/jsx-runtime");
var $2IB0T$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $78dc405d4adc0fa1$export$2e2bcd8739ae039);




const $78dc405d4adc0fa1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9c1c14__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9c1c14__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9c1c14__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsxs)("radialGradient", {
                        id: "9c1c14__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsxs)("radialGradient", {
                        id: "9c1c14__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("clipPath", {
                        id: "9c1c14__b",
                        children: /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("clipPath", {
                id: "9c1c14__a",
                children: /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("path", {
                    fill: "#8F8F8F",
                    fillRule: "evenodd",
                    d: "M120.637 37.144c3.993-7.24 15.006-4.405 15.006 3.863v82.712c0 6.628-5.373 12-12 12H36c-6.627 0-12-5.372-12-12V79.994c0-2.203.908-4.308 2.51-5.82L43.543 58.11c3.313-3.124 8.56-2.858 11.54.585l12.31 14.228 16.1-24.9c2.946-4.558 9.477-4.921 12.912-.72l11.126 13.611zl3.503 1.932.039.021m-.039-.021-3.503-1.932Zm3.503 1.931-13.108 23.77c-2.734 4.96-9.615 5.585-13.199 1.2L90.21 52.366l-16.1 24.9c-2.896 4.478-9.28 4.924-12.768.891L49.032 63.93 32 79.994v24.14l16.119-10.386c2.48-1.598 5.696-1.469 8.04.322L72.429 106.5l19.93-23.184c2.655-3.09 7.377-3.267 10.258-.387l14.209 14.21 10.818-5.187zm0 59.816-9.914 4.753c-1.53.734-3.357.422-4.557-.778l-15.45-15.45-19.535 22.724c-2.59 3.013-7.082 3.483-10.24 1.07L51.86 100.854 32 113.651v10.068c0 2.21 1.79 4 4 4h87.643c2.209 0 4-1.79 4-4z"
                })
            }),
            /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("g", {
                clipPath: "url(#9c1c14__a)",
                children: /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#9c1c14__b)",
                    children: [
                        /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#9c1c14__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("path", {
                            fill: "url(#9c1c14__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("path", {
                            fill: "url(#9c1c14__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("path", {
                            fill: "url(#9c1c14__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $2IB0T$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#9c1c14__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $78dc405d4adc0fa1$var$ForwardRef = /*#__PURE__*/ (0, $2IB0T$react.forwardRef)($78dc405d4adc0fa1$var$SvgComponent);
var $78dc405d4adc0fa1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($78dc405d4adc0fa1$var$ForwardRef);
