// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_helpCircle_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $27frQ$reactjsxruntime = require("react/jsx-runtime");
var $27frQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3f02fb8fa22db316$export$2e2bcd8739ae039);




const $3f02fb8fa22db316$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5a5a43__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5a5a43__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5a5a43__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5a5a43__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5a5a43__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("clipPath", {
                        id: "5a5a43__b",
                        children: /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("clipPath", {
                id: "5a5a43__a",
                children: /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M80 136.016c30.928 0 56-25.072 56-56s-25.072-56-56-56-56 25.072-56 56 25.072 56 56 56m1.594-78.957c-1.378-.281-2.808-.138-4.113.416s-2.438 1.5-3.242 2.731C73.435 61.44 73 62.9 73 64.4c0 2.21-1.79 4-4 4s-4-1.79-4-4c0-3.04.88-6.02 2.538-8.564 1.66-2.544 4.028-4.542 6.818-5.726s5.867-1.495 8.836-.89c2.968.605 5.682 2.095 7.802 4.267s3.552 4.926 4.131 7.911.283 6.079-.856 8.896-3.075 5.241-5.576 6.954c-1.224.838-2.415 1.974-3.267 3.207-.865 1.25-1.22 2.353-1.22 3.203v7.083c0 2.21-1.79 4-4 4-2.208 0-4-1.79-4-4v-7.083c0-2.975 1.182-5.646 2.641-7.755 1.472-2.127 3.394-3.933 5.326-5.256 1.185-.81 2.122-1.974 2.68-3.352.557-1.379.704-2.902.419-4.372-.286-1.47-.988-2.807-2.003-3.848-1.015-1.039-2.295-1.735-3.675-2.017M85 105.016c0 2.485-2.015 4.5-4.5 4.5s-4.5-2.015-4.5-4.5c0-2.486 2.015-4.5 4.5-4.5s4.5 2.014 4.5 4.5"
                })
            }),
            /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("g", {
                clipPath: "url(#5a5a43__a)",
                children: /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#5a5a43__b)",
                    children: [
                        /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#5a5a43__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("path", {
                            fill: "url(#5a5a43__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("path", {
                            fill: "url(#5a5a43__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("path", {
                            fill: "url(#5a5a43__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $27frQ$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#5a5a43__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $3f02fb8fa22db316$var$ForwardRef = /*#__PURE__*/ (0, $27frQ$react.forwardRef)($3f02fb8fa22db316$var$SvgComponent);
var $3f02fb8fa22db316$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3f02fb8fa22db316$var$ForwardRef);
