// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_wallet_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3RY6s$reactjsxruntime = require("react/jsx-runtime");
var $3RY6s$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $934ecd9c8ef6d0f8$export$2e2bcd8739ae039);




const $934ecd9c8ef6d0f8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2c91da__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.984,
                        y2: -15.276,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2c91da__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.473,
                        y2: 149.468,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2c91da__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.03,
                        y2: 55.21,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2c91da__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.13 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2c91da__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.884 89.791)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("clipPath", {
                        id: "2c91da__b",
                        children: /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            y: 0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("clipPath", {
                id: "2c91da__a",
                children: /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m45.39 42.36 50.229-13.458c4.267-1.144 8.654 1.389 9.798 5.656l4.564 17.034H41.033l-.309-1.15c-.943-3.52 1.146-7.139 4.666-8.082m66.425 20.704h-82.46c-2.209 0-4 1.791-4 4v60.307c0 2.209 1.791 4 4 4h82.46c8.837 0 16-7.164 16-16v-1.346h-23.322c-4.419 0-8-3.582-8-8V88.41c0-4.418 3.581-8 8-8h23.322v-1.346c0-8.836-7.163-16-16-16m18.831 25.346h-26.153v17.615h26.153zm-17.115 13.309c2.485 0 4.5-2.015 4.5-4.5s-2.015-4.5-4.5-4.5c-2.486 0-4.5 2.015-4.5 4.5s2.014 4.5 4.5 4.5"
                })
            }),
            /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("g", {
                clipPath: "url(#2c91da__a)",
                children: /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#2c91da__b)",
                    children: [
                        /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            y: 0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 .001h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.207,
                            r: 69.609,
                            fill: "url(#2c91da__c)",
                            transform: "rotate(-.08 90.468 6.207)"
                        }),
                        /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("path", {
                            fill: "url(#2c91da__d)",
                            d: "M61.07-28.262c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193L20.028 154.34c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("path", {
                            fill: "url(#2c91da__e)",
                            d: "M23.058 75.966c2.735-59.733 53.375-105.94 113.108-103.204 59.732 2.734 105.938 53.374 103.204 113.107s-53.375 105.939-113.108 103.204C66.53 186.338 20.324 135.698 23.058 75.966"
                        }),
                        /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("path", {
                            fill: "url(#2c91da__f)",
                            d: "M-64.825 115.351c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.722 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $3RY6s$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.907,
                            r: 69.609,
                            fill: "url(#2c91da__g)",
                            transform: "rotate(-.08 150.675 28.907)"
                        })
                    ]
                })
            })
        ]
    });
const $934ecd9c8ef6d0f8$var$ForwardRef = /*#__PURE__*/ (0, $3RY6s$react.forwardRef)($934ecd9c8ef6d0f8$var$SvgComponent);
var $934ecd9c8ef6d0f8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($934ecd9c8ef6d0f8$var$ForwardRef);
