// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_threeArrowsCircle_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $kw3V4$reactjsxruntime = require("react/jsx-runtime");
var $kw3V4$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e64e751088b2c87e$export$2e2bcd8739ae039);




const $e64e751088b2c87e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "23ae82__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.896,
                        y2: -4.582,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "23ae82__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.843,
                        y2: 44.841,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "23ae82__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.81,
                        y2: 16.564,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsxs)("radialGradient", {
                        id: "23ae82__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.503)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsxs)("radialGradient", {
                        id: "23ae82__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.264 26.937)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("clipPath", {
                        id: "23ae82__b",
                        children: /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("clipPath", {
                id: "23ae82__a",
                children: /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M36.33 11.291c.157.672-.26 1.345-.932 1.502l-6.885 1.613c-.672.158-1.344-.26-1.502-.932s.26-1.344.932-1.502l4.08-.955c-2.343-1.467-5.111-2.314-8.08-2.314-4.574 0-8.676 2.012-11.473 5.204-.455.519-1.245.57-1.764.116-.52-.455-.572-1.245-.117-1.764 3.252-3.71 8.03-6.056 13.354-6.056 3.381 0 6.543.946 9.234 2.588l-.894-3.814c-.157-.672.26-1.345.932-1.502s1.345.26 1.502.932zm2.248 8.125c-.206-.659.161-1.36.82-1.566s1.36.16 1.566.82c.525 1.676.807 3.46.807 5.306 0 7.05-4.11 13.138-10.061 16.002l3.808 1.205c.658.208 1.023.91.815 1.568-.208.659-.91 1.023-1.568.815l-6.742-2.132c-.659-.208-1.024-.91-.816-1.568l2.132-6.742c.208-.659.91-1.023 1.569-.815s1.023.91.815 1.568l-1.238 3.915c5.193-2.433 8.786-7.706 8.786-13.816 0-1.59-.243-3.122-.693-4.56M6.306 22.58 3.31 25.135c-.525.448-1.314.385-1.762-.14s-.386-1.314.14-1.762l5.379-4.59c.525-.448 1.314-.386 1.762.14l4.59 5.378c.448.526.385 1.315-.14 1.763s-1.314.385-1.762-.14l-2.706-3.171c-.243 2.675.218 5.447 1.48 8.039 2.004 4.112 5.609 6.92 9.703 8.037.666.181 1.059.869.877 1.535s-.869 1.058-1.535.877c-4.76-1.3-8.96-4.568-11.292-9.354-1.438-2.954-1.983-6.113-1.74-9.167"
                })
            }),
            /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("g", {
                clipPath: "url(#23ae82__a)",
                children: /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#23ae82__b)",
                    children: [
                        /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 .001h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.863,
                            r: 20.883,
                            fill: "url(#23ae82__c)",
                            transform: "rotate(-.08 27.142 1.863)"
                        }),
                        /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("path", {
                            fill: "url(#23ae82__d)",
                            d: "M18.321-8.478c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.657L6.008 46.304c3.694 2.267 8.365 2.264 12.056-.008L54.752 23.71c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("path", {
                            fill: "url(#23ae82__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.99 40.85-8.17S72.63 7.84 71.809 25.76 55.798 57.543 37.879 56.722C19.957 55.902 6.095 40.71 6.915 22.79"
                        }),
                        /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("path", {
                            fill: "url(#23ae82__f)",
                            d: "M-19.448 34.606c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.331 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.649-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.279-15.033 15.318-14.246 6.077-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $kw3V4$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.673,
                            r: 20.883,
                            fill: "url(#23ae82__g)",
                            transform: "rotate(-.08 45.2 8.673)"
                        })
                    ]
                })
            })
        ]
    });
const $e64e751088b2c87e$var$ForwardRef = /*#__PURE__*/ (0, $kw3V4$react.forwardRef)($e64e751088b2c87e$var$SvgComponent);
var $e64e751088b2c87e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e64e751088b2c87e$var$ForwardRef);
