// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_speedFast_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bkHa1$jsxs, jsx as $bkHa1$jsx} from "react/jsx-runtime";
import {forwardRef as $bkHa1$forwardRef} from "react";





const $56e0193074ac0b4f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bkHa1$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bkHa1$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bkHa1$jsxs)("linearGradient", {
                        id: "cca885__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bkHa1$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bkHa1$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bkHa1$jsxs)("linearGradient", {
                        id: "cca885__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bkHa1$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bkHa1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bkHa1$jsxs)("linearGradient", {
                        id: "cca885__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bkHa1$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bkHa1$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bkHa1$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bkHa1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bkHa1$jsxs)("radialGradient", {
                        id: "cca885__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bkHa1$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bkHa1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bkHa1$jsxs)("radialGradient", {
                        id: "cca885__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bkHa1$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bkHa1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bkHa1$jsx)("clipPath", {
                        id: "cca885__b",
                        children: /*#__PURE__*/ (0, $bkHa1$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bkHa1$jsx)("clipPath", {
                id: "cca885__a",
                children: /*#__PURE__*/ (0, $bkHa1$jsx)("path", {
                    fill: "#D9D9D9",
                    d: "M47.998 32.778c-1.105 0-2-.896-2-2v-7.012q0-.17.027-.332c-4.599.273-8.942 1.494-12.838 3.471q.099.121.18.261l3.505 6.073c.553.956.225 2.18-.732 2.732-.956.552-2.18.224-2.732-.732l-3.506-6.073q-.073-.128-.127-.261c-3.704 2.483-6.875 5.702-9.301 9.448q.064.03.127.067l6.073 3.506c.956.553 1.284 1.776.732 2.732s-1.776 1.285-2.732.733L18.6 41.885l-.09-.055c-1.886 3.878-3.03 8.184-3.245 12.733h6.982c1.104 0 2 .895 2 2s-.896 2-2 2h-6.932c.333 4.57 1.604 8.88 3.621 12.74.644 1.233 1.953 1.947 3.344 1.947h51.433c1.392 0 2.7-.714 3.344-1.947 2.017-3.86 3.288-8.17 3.621-12.74h-6.873c-1.105 0-2-.896-2-2s.895-2 2-2h6.924c-.507-10.666-6.113-19.996-14.434-25.607q-.046.106-.105.21l-3.506 6.073c-.553.956-1.776 1.284-2.732.732s-1.285-1.776-.733-2.732l3.507-6.073q.069-.12.15-.225c-3.913-1.998-8.28-3.233-12.907-3.507q.027.161.028.332v7.012c0 1.104-.896 2-2 2m23.615 5.853c.933.933.972 2.433.09 3.413l-21.786 24.2c-1.606 1.785-4.38 1.858-6.077.16-1.697-1.697-1.625-4.47.16-6.077l24.2-21.785c.98-.883 2.48-.843 3.413.09"
                })
            }),
            /*#__PURE__*/ (0, $bkHa1$jsx)("g", {
                clipPath: "url(#cca885__a)",
                children: /*#__PURE__*/ (0, $bkHa1$jsxs)("g", {
                    clipPath: "url(#cca885__b)",
                    children: [
                        /*#__PURE__*/ (0, $bkHa1$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $bkHa1$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $bkHa1$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#cca885__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $bkHa1$jsx)("path", {
                            fill: "url(#cca885__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $bkHa1$jsx)("path", {
                            fill: "url(#cca885__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $bkHa1$jsx)("path", {
                            fill: "url(#cca885__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $bkHa1$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#cca885__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $56e0193074ac0b4f$var$ForwardRef = /*#__PURE__*/ (0, $bkHa1$forwardRef)($56e0193074ac0b4f$var$SvgComponent);
var $56e0193074ac0b4f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($56e0193074ac0b4f$var$ForwardRef);


export {$56e0193074ac0b4f$export$2e2bcd8739ae039 as default};
