// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_remix_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $02vDl$jsxs, jsx as $02vDl$jsx} from "react/jsx-runtime";
import {forwardRef as $02vDl$forwardRef} from "react";





const $aa6edbbd1bcd4a41$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $02vDl$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $02vDl$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $02vDl$jsxs)("linearGradient", {
                        id: "b6de79__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $02vDl$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $02vDl$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $02vDl$jsxs)("linearGradient", {
                        id: "b6de79__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $02vDl$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $02vDl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $02vDl$jsxs)("linearGradient", {
                        id: "b6de79__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.618,
                        y2: 33.126,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $02vDl$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $02vDl$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $02vDl$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $02vDl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $02vDl$jsxs)("radialGradient", {
                        id: "b6de79__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $02vDl$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $02vDl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $02vDl$jsxs)("radialGradient", {
                        id: "b6de79__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $02vDl$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $02vDl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $02vDl$jsx)("clipPath", {
                        id: "b6de79__b",
                        children: /*#__PURE__*/ (0, $02vDl$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $02vDl$jsx)("clipPath", {
                id: "b6de79__a",
                children: /*#__PURE__*/ (0, $02vDl$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M76.03 41.644 86.645 31.03c.781-.78.781-2.047 0-2.828L76.03 17.586c-.781-.781-2.047-.781-2.828 0-.781.78-.781 2.047 0 2.828l6.998 6.998h-4.787c-4.92 0-9.46 1.657-13.083 4.443q-.076.06-.148.125-.041.03-.082.062c-2.672 2.098-5.175 5.157-7.142 7.977-1.975 2.831-3.517 5.575-4.233 7.156-.57 1.258-1.98 4.245-3.848 7.385-1.607 2.703-3.457 5.352-5.308 7.144q-.077.058-.15.124c-3.105 2.822-7.225 4.54-11.75 4.54-9.653 0-17.478-7.826-17.478-17.478s7.825-17.478 17.478-17.478c5.903 0 11.123 2.925 14.29 7.413l.008.01 2.022 2.828c.642.899 1.892 1.106 2.79.464s1.106-1.892.464-2.79l-2.02-2.824c-3.884-5.501-10.299-9.1-17.554-9.1-11.862 0-21.478 9.615-21.478 21.477s9.616 21.477 21.478 21.477c4.925 0 9.466-1.66 13.09-4.448q.143-.08.277-.184c2.791-2.202 5.34-5.87 7.28-9.13 1.97-3.316 3.45-6.45 4.052-7.779.593-1.308 2-3.838 3.87-6.518 1.878-2.69 4.115-5.38 6.333-7.12q.069-.054.132-.113l.066-.049c2.948-2.267 6.636-3.614 10.644-3.614h5.192l-7.403 7.404c-.781.78-.781 2.047 0 2.828.78.781 2.047.781 2.828 0m4.781 28.723c1.105 0 2-.895 2-2s-.895-2-2-2h-5.398c-5.811 0-10.497-3.346-14.558-7.777l-2.337-2.825c-.704-.85-1.965-.97-2.816-.265-.85.704-.97 1.965-.266 2.816l2.37 2.863.031.038.033.037c4.338 4.743 10.03 9.113 17.543 9.113z"
                })
            }),
            /*#__PURE__*/ (0, $02vDl$jsx)("g", {
                clipPath: "url(#b6de79__a)",
                children: /*#__PURE__*/ (0, $02vDl$jsxs)("g", {
                    clipPath: "url(#b6de79__b)",
                    children: [
                        /*#__PURE__*/ (0, $02vDl$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $02vDl$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $02vDl$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#b6de79__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $02vDl$jsx)("path", {
                            fill: "url(#b6de79__d)",
                            d: "M36.643-16.957c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.426-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.432 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $02vDl$jsx)("path", {
                            fill: "url(#b6de79__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $02vDl$jsx)("path", {
                            fill: "url(#b6de79__f)",
                            d: "M-38.895 69.21c14.246-6.077 29.61-5.817 42.922-.5 26.636 10.663 42.135 4.07 52.885-22.505 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.315 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $02vDl$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#b6de79__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $aa6edbbd1bcd4a41$var$ForwardRef = /*#__PURE__*/ (0, $02vDl$forwardRef)($aa6edbbd1bcd4a41$var$SvgComponent);
var $aa6edbbd1bcd4a41$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($aa6edbbd1bcd4a41$var$ForwardRef);


export {$aa6edbbd1bcd4a41$export$2e2bcd8739ae039 as default};
