// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_translate_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $cj94J$jsxs, jsx as $cj94J$jsx} from "react/jsx-runtime";
import {forwardRef as $cj94J$forwardRef} from "react";





const $14af55a14f543fad$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cj94J$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cj94J$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cj94J$jsxs)("linearGradient", {
                        id: "b260a1__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cj94J$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $cj94J$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cj94J$jsxs)("linearGradient", {
                        id: "b260a1__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cj94J$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cj94J$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cj94J$jsxs)("linearGradient", {
                        id: "b260a1__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cj94J$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cj94J$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cj94J$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cj94J$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cj94J$jsxs)("radialGradient", {
                        id: "b260a1__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cj94J$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $cj94J$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cj94J$jsxs)("radialGradient", {
                        id: "b260a1__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cj94J$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cj94J$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cj94J$jsx)("clipPath", {
                        id: "b260a1__b",
                        children: /*#__PURE__*/ (0, $cj94J$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cj94J$jsxs)("clipPath", {
                id: "b260a1__a",
                children: [
                    /*#__PURE__*/ (0, $cj94J$jsx)("path", {
                        fill: "#292929",
                        d: "M94.423 33.828h-28.57v-5.642c0-2.209-1.791-4-4-4s-4 1.791-4 4v5.642H28.285c-2.209 0-4 1.79-4 4s1.791 4 4 4h50.36c-1.834 11.528-8.238 20.9-16.288 27.977-1.862 1.637-3.801 3.14-5.775 4.509-3.698-6.442-6.332-12.624-7.124-17.12-.383-2.175-2.458-3.628-4.633-3.245-2.176.384-3.629 2.458-3.245 4.634 1.015 5.762 4.144 12.914 8.164 19.888-5.243 2.775-10.398 4.608-14.711 5.476-2.166.436-3.568 2.545-3.132 4.71.436 2.167 2.545 3.569 4.71 3.133 5.222-1.051 11.292-3.237 17.372-6.513 5.84 8.718 13.043 17.048 19.456 21.19 1.855 1.199 4.331.666 5.53-1.19s.666-4.331-1.19-5.53c-4.983-3.22-11.367-10.345-16.958-18.639 2.336-1.6 4.625-3.367 6.818-5.294 9.354-8.222 17.178-19.607 19.09-33.986h7.694c2.209 0 4-1.791 4-4s-1.791-4-4-4"
                    }),
                    /*#__PURE__*/ (0, $cj94J$jsx)("path", {
                        fill: "#292929",
                        fillRule: "evenodd",
                        d: "M93.279 113.842 85 133.373c-.862 2.034-3.21 2.984-5.244 2.122s-2.984-3.21-2.121-5.244l25.184-59.42q.096-.23.219-.448.214-.376.495-.69c.381-.428.86-.78 1.423-1.018.51-.216 1.04-.319 1.561-.318.521 0 1.051.102 1.561.318.562.238 1.042.59 1.423 1.018q.28.314.495.69.123.218.218.449l25.185 59.419c.862 2.034-.088 4.382-2.122 5.244s-4.381-.088-5.243-2.122l-8.282-19.538q-.112.007-.226.007zm13.238-31.734-9.847 23.234h19.695z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cj94J$jsx)("g", {
                clipPath: "url(#b260a1__a)",
                children: /*#__PURE__*/ (0, $cj94J$jsxs)("g", {
                    clipPath: "url(#b260a1__b)",
                    children: [
                        /*#__PURE__*/ (0, $cj94J$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $cj94J$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $cj94J$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#b260a1__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $cj94J$jsx)("path", {
                            fill: "url(#b260a1__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $cj94J$jsx)("path", {
                            fill: "url(#b260a1__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $cj94J$jsx)("path", {
                            fill: "url(#b260a1__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $cj94J$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#b260a1__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $14af55a14f543fad$var$ForwardRef = /*#__PURE__*/ (0, $cj94J$forwardRef)($14af55a14f543fad$var$SvgComponent);
var $14af55a14f543fad$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($14af55a14f543fad$var$ForwardRef);


export {$14af55a14f543fad$export$2e2bcd8739ae039 as default};
