// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_buildTable_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $cGAti$jsxs, jsx as $cGAti$jsx} from "react/jsx-runtime";
import {forwardRef as $cGAti$forwardRef} from "react";





const $9429a6d022f9e576$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cGAti$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cGAti$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cGAti$jsxs)("linearGradient", {
                        id: "1feae4__d",
                        x1: -16.181,
                        x2: 200.204,
                        y1: 139.116,
                        y2: -0.536,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGAti$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $cGAti$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGAti$jsxs)("linearGradient", {
                        id: "1feae4__e",
                        x1: 163.345,
                        x2: 73.417,
                        y1: 47.092,
                        y2: 149.67,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGAti$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cGAti$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGAti$jsxs)("linearGradient", {
                        id: "1feae4__f",
                        x1: 46.716,
                        x2: 211.902,
                        y1: 157.799,
                        y2: 59.636,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGAti$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cGAti$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cGAti$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cGAti$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGAti$jsxs)("radialGradient", {
                        id: "1feae4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 56.4317 -59.8633 0 59.863 56.432)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGAti$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $cGAti$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGAti$jsxs)("radialGradient", {
                        id: "1feae4__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 56.4317 -59.8633 0 59.863 56.432)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGAti$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cGAti$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGAti$jsx)("clipPath", {
                        id: "1feae4__b",
                        children: /*#__PURE__*/ (0, $cGAti$jsx)("rect", {
                            width: 137.6,
                            height: 129.712,
                            x: 18.398,
                            y: 23.2,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cGAti$jsx)("clipPath", {
                id: "1feae4__a",
                children: /*#__PURE__*/ (0, $cGAti$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M123.969 33.9c8.84 0 16 7.16 16 16v4.09H19.949V49.9c0-8.84 7.16-16 16-16zm16.001 52.56c-2.58-.66-5.28-1.01-8.06-1.01h-.01c-17.94 0-32.53 14.59-32.53 32.53 0 2.77.35 5.47 1.01 8.04H48V61.99h91.97zM19.95 110.03c0 8.84 7.16 16 16 16H40V62H19.95zM131.91 93.46c2.82 0 5.53.48 8.06 1.36h.01c9.59 3.34 16.47 12.45 16.47 23.17 0 13.55-10.98 24.53-24.53 24.53-10.73 0-19.85-6.89-23.18-16.49-.88-2.52-1.36-5.22-1.36-8.04 0-13.55 10.98-24.53 24.53-24.53m4.01 28.48h7.88c2.21 0 4-1.79 4-4s-1.79-4-4-4h-7.88v-7.88c0-2.21-1.79-4-4-4s-4 1.79-4 4v7.88h-7.88c-2.21 0-4 1.79-4 4s1.79 4 4 4h7.88v7.88c0 2.21 1.79 4 4 4s4-1.79 4-4z"
                })
            }),
            /*#__PURE__*/ (0, $cGAti$jsx)("g", {
                clipPath: "url(#1feae4__a)",
                children: /*#__PURE__*/ (0, $cGAti$jsxs)("g", {
                    clipPath: "url(#1feae4__b)",
                    children: [
                        /*#__PURE__*/ (0, $cGAti$jsx)("rect", {
                            width: 137.6,
                            height: 129.712,
                            x: 18.398,
                            y: 23.2,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $cGAti$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M18.398 23.2h137.6v129.712h-137.6z"
                        }),
                        /*#__PURE__*/ (0, $cGAti$jsx)("ellipse", {
                            cx: 59.863,
                            cy: 56.432,
                            fill: "url(#1feae4__c)",
                            rx: 59.863,
                            ry: 56.432,
                            transform: "matrix(1 -.00132 .00148 1 36.254 -28.121)"
                        }),
                        /*#__PURE__*/ (0, $cGAti$jsx)("path", {
                            fill: "url(#1feae4__d)",
                            d: "M70.918.287C60.351-5.877 46.962-5.915 36.355.19L-69.732 61.25c-10.39 5.98-10.407 20.096-.032 26.099l105.386 60.973c10.591 6.127 23.98 6.12 34.562-.021L175.353 87.27c10.321-5.99 10.34-20.027.032-26.04z"
                        }),
                        /*#__PURE__*/ (0, $cGAti$jsx)("path", {
                            fill: "url(#1feae4__e)",
                            d: "M38.227 84.784C40.58 36.359 84.13-1.1 135.5 1.117s91.107 43.27 88.755 91.696c-2.351 48.425-45.901 85.884-97.271 83.667s-91.107-43.271-88.756-91.696"
                        }),
                        /*#__PURE__*/ (0, $cGAti$jsx)("path", {
                            fill: "url(#1feae4__f)",
                            d: "M-37.35 116.713c20.42-8.211 42.442-7.859 61.522-.676 38.179 14.408 60.394 5.5 75.802-30.408 7.674-18.019 22.659-33.278 43.101-41.5 40.872-16.41 88.057 1.397 105.442 39.812s-1.644 82.825-42.506 99.258c-20.488 8.239-42.553 7.853-61.662.606-38.117-14.332-60.29-5.39-75.678 30.561-7.694 17.976-22.675 33.183-43.095 41.395-40.84 16.424-88.024-1.384-105.399-39.777s1.634-82.847 42.473-99.271"
                        }),
                        /*#__PURE__*/ (0, $cGAti$jsx)("ellipse", {
                            cx: 59.863,
                            cy: 56.432,
                            fill: "url(#1feae4__g)",
                            rx: 59.863,
                            ry: 56.432,
                            transform: "matrix(1 -.00132 .00148 1 88.031 -9.719)"
                        })
                    ]
                })
            })
        ]
    });
const $9429a6d022f9e576$var$ForwardRef = /*#__PURE__*/ (0, $cGAti$forwardRef)($9429a6d022f9e576$var$SvgComponent);
var $9429a6d022f9e576$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9429a6d022f9e576$var$ForwardRef);


export {$9429a6d022f9e576$export$2e2bcd8739ae039 as default};
