// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_trophy_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $JEtQC$reactjsxruntime = require("react/jsx-runtime");
var $JEtQC$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $07a594069cef6fc0$export$2e2bcd8739ae039);




const $07a594069cef6fc0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b872f6__d",
                        x1: -5.676,
                        x2: 56.126,
                        y1: 38.508,
                        y2: -1.379,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b872f6__e",
                        x1: 45.597,
                        x2: 19.913,
                        y1: 12.224,
                        y2: 41.522,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b872f6__f",
                        x1: 12.287,
                        x2: 59.466,
                        y1: 43.844,
                        y2: 15.807,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b872f6__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b872f6__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("clipPath", {
                        id: "b872f6__b",
                        children: /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 5.4,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("clipPath", {
                id: "b872f6__a",
                children: /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M37.811 12H32.4v-1.8c0-.993-.807-1.8-1.8-1.8H17.4c-.993 0-1.8.807-1.8 1.8V12h-5.4c-1.653 0-3 1.347-3 3v.6c0 4.632 3.768 8.4 8.4 8.4h1.51c1.301 1.866 3.344 3.18 5.7 3.513 0 .03-.01.057-.01.087v7.326c-2.15.405-3.92 1.872-4.752 3.837-.168.396.126.834.555.834h10.794c.43 0 .723-.438.555-.834-.83-1.965-2.6-3.432-4.752-3.837V27.6c0-.03-.006-.057-.009-.087 2.358-.336 4.398-1.647 5.7-3.513h1.518c4.632 0 8.4-3.768 8.4-8.4V15c0-1.653-1.347-3-3-3zM15.6 21.6c-3.309 0-6-2.69-6-6V15c0-.33.27-.6.6-.6h5.4v4.8c0 .834.123 1.638.351 2.4zm12.237-4.449-1.119 1.086c-.153.15-.225.366-.186.576l.267 1.536c.102.594-.522 1.05-1.056.768l-1.38-.729c-.189-.099-.417-.099-.606 0l-1.38.73c-.534.281-1.158-.172-1.056-.769l.267-1.536c.036-.21-.033-.426-.186-.576l-1.119-1.086c-.432-.42-.195-1.155.402-1.242l1.545-.222c.213-.03.396-.165.492-.357l.687-1.4c.267-.544 1.038-.544 1.305 0l.687 1.4c.096.192.28.327.492.357l1.545.222c.597.087.837.822.402 1.242zm10.575-1.55c0 3.308-2.69 6-6 6h-.36c.225-.763.351-1.567.351-2.4v-4.8h5.412c.33 0 .6.27.6.6v.6z"
                })
            }),
            /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("g", {
                clipPath: "url(#b872f6__a)",
                children: /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#b872f6__b)",
                    children: [
                        /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 5.4,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M4.2 5.4h39.3v37.047H4.2z"
                        }),
                        /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#b872f6__c)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 9.3 -9.258)"
                        }),
                        /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("path", {
                            fill: "url(#b872f6__d)",
                            d: "M19.2-1.144c-3.017-1.76-6.841-1.771-9.87-.028l-30.3 17.44c-2.968 1.708-2.973 5.74-.01 7.454l30.1 17.415c3.025 1.75 6.849 1.748 9.871-.006L49.03 23.7c2.947-1.711 2.953-5.72.009-7.437z"
                        }),
                        /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("path", {
                            fill: "url(#b872f6__e)",
                            d: "M9.862 22.99C10.534 9.157 22.972-1.54 37.644-.908s26.021 12.358 25.35 26.19c-.672 13.83-13.11 24.529-27.782 23.896C20.54 48.545 9.19 36.82 9.862 22.989"
                        }),
                        /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("path", {
                            fill: "url(#b872f6__f)",
                            d: "M-11.723 32.11c5.832-2.346 12.122-2.246 17.571-.194 10.905 4.115 17.25 1.57 21.65-8.685 2.192-5.146 6.472-9.505 12.31-11.853 11.674-4.687 25.15.399 30.116 11.37 4.965 10.973-.47 23.657-12.14 28.35-5.852 2.353-12.154 2.243-17.612.173C29.286 47.178 22.953 49.732 18.558 60c-2.198 5.134-6.476 9.477-12.309 11.823-11.664 4.69-25.14-.395-30.103-11.361-4.962-10.966.467-23.662 12.131-28.353"
                        }),
                        /*#__PURE__*/ (0, $JEtQC$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#b872f6__g)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 24.086 -4.002)"
                        })
                    ]
                })
            })
        ]
    });
const $07a594069cef6fc0$var$ForwardRef = /*#__PURE__*/ (0, $JEtQC$react.forwardRef)($07a594069cef6fc0$var$SvgComponent);
var $07a594069cef6fc0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($07a594069cef6fc0$var$ForwardRef);
