// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_userGroup_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $iRUYq$reactjsxruntime = require("react/jsx-runtime");
var $iRUYq$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e3fbc4ae2c7f6c39$export$2e2bcd8739ae039);




const $e3fbc4ae2c7f6c39$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5a114e__d",
                        x1: -5.977,
                        x2: 55.826,
                        y1: 39.303,
                        y2: -0.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5a114e__e",
                        x1: 45.296,
                        x2: 19.612,
                        y1: 13.02,
                        y2: 42.317,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5a114e__f",
                        x1: 11.986,
                        x2: 59.166,
                        y1: 44.639,
                        y2: 16.603,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5a114e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5a114e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("clipPath", {
                        id: "5a114e__b",
                        children: /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.196,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("clipPath", {
                id: "5a114e__a",
                children: /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M35.054 15.135c0 4.06-3.09 7.35-6.9 7.35s-6.9-3.29-6.9-7.35 3.089-7.35 6.9-7.35c3.81 0 6.9 3.29 6.9 7.35m-13.278 7.09c-.612.188-1.26.293-1.932.293-3.81 0-6.9-3.29-6.9-7.35s3.09-7.35 6.9-7.35c.65 0 1.284.1 1.88.28-1.77 1.775-2.873 4.247-2.873 7.037s1.125 5.31 2.925 7.09M12.02 36.74c.975-4.527 3.795-8.31 7.557-10.62-6.612.126-12.174 4.896-13.521 11.157-.324 1.506.8 2.934 2.343 2.934h4.182c-.612-1.038-.822-2.268-.561-3.47m16.13-10.656c6.73 0 12.424 4.818 13.789 11.163.324 1.506-.801 2.934-2.343 2.934h-22.89c-1.542 0-2.667-1.428-2.343-2.934 1.365-6.345 7.059-11.163 13.788-11.163"
                })
            }),
            /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("g", {
                clipPath: "url(#5a114e__a)",
                children: /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#5a114e__b)",
                    children: [
                        /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.196,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.898 6.196h39.3v37.047h-39.3z"
                        }),
                        /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#5a114e__c)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 9 -8.462)"
                        }),
                        /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("path", {
                            fill: "url(#5a114e__d)",
                            d: "M18.9-.348c-3.018-1.76-6.842-1.771-9.872-.028l-30.3 17.44c-2.967 1.708-2.972 5.74-.008 7.454l30.1 17.415c3.024 1.75 6.848 1.748 9.87-.006l30.038-17.431c2.948-1.711 2.953-5.72.009-7.438z"
                        }),
                        /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("path", {
                            fill: "url(#5a114e__e)",
                            d: "M9.561 23.785C10.233 9.955 22.671-.745 37.343-.11c14.672.633 26.021 12.358 25.35 26.19-.672 13.83-13.11 24.529-27.782 23.896-14.672-.634-26.022-12.359-25.35-26.19"
                        }),
                        /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("path", {
                            fill: "url(#5a114e__f)",
                            d: "M-12.024 32.905C-6.192 30.56.098 30.66 5.548 32.712c10.904 4.115 17.249 1.57 21.65-8.685 2.192-5.146 6.471-9.505 12.31-11.853 11.673-4.687 25.15.399 30.115 11.37 4.965 10.973-.47 23.657-12.14 28.35-5.852 2.353-12.154 2.243-17.611.173-10.887-4.093-17.22-1.54-21.615 8.729-2.198 5.134-6.476 9.477-12.308 11.823-11.664 4.69-25.141-.395-30.104-11.361-4.962-10.966.467-23.662 12.131-28.353"
                        }),
                        /*#__PURE__*/ (0, $iRUYq$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#5a114e__g)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 23.785 -3.206)"
                        })
                    ]
                })
            })
        ]
    });
const $e3fbc4ae2c7f6c39$var$ForwardRef = /*#__PURE__*/ (0, $iRUYq$react.forwardRef)($e3fbc4ae2c7f6c39$var$SvgComponent);
var $e3fbc4ae2c7f6c39$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e3fbc4ae2c7f6c39$var$ForwardRef);
