// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileZip_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $hJOqE$reactjsxruntime = require("react/jsx-runtime");
var $hJOqE$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6bb621c7bec45bfd$export$2e2bcd8739ae039);




const $6bb621c7bec45bfd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f13434__d",
                        x1: -3.949,
                        x2: 55.306,
                        y1: 38.702,
                        y2: 0.46,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f13434__e",
                        x1: 45.213,
                        x2: 20.587,
                        y1: 13.503,
                        y2: 41.593,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f13434__f",
                        x1: 13.273,
                        x2: 58.507,
                        y1: 43.818,
                        y2: 16.938,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f13434__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f13434__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("clipPath", {
                        id: "f13434__b",
                        children: /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("clipPath", {
                id: "f13434__a",
                children: /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m26.338 9.405 9.888 9.46v.005c.72.684 1.125 1.638 1.125 2.634v14.463c0 2.01-1.623 3.64-3.62 3.64H14.268c-2 0-3.62-1.63-3.62-3.64V12.042c0-2.01 1.622-3.639 3.62-3.639h9.573c.93 0 1.824.357 2.496 1.002m1.425 11.553c0-.663-.537-1.2-1.2-1.2h-1.52V17.76h1.523c.663 0 1.2-.537 1.2-1.2s-.537-1.2-1.2-1.2h-1.524v-.999c0-.663-.537-1.2-1.2-1.2h-2.397c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h1.197v1.998h-1.197c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h1.197v1.998h-1.197c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h1.197v1.998h-.897c-.663 0-1.2.537-1.2 1.2v6.027c0 1.323 1.077 2.4 2.4 2.4h1.803c1.323 0 2.4-1.077 2.4-2.4v-6.027c0-.663-.537-1.2-1.2-1.2h-.909v-4.197h1.524c.663 0 1.2-.537 1.2-1.2m-4.818 10.21h1.806v2.411h-1.806z"
                })
            }),
            /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("g", {
                clipPath: "url(#f13434__a)",
                children: /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#f13434__b)",
                    children: [
                        /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M5.52 6.96H43.2v35.52H5.52z"
                        }),
                        /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#f13434__c)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 10.41 -7.094)"
                        }),
                        /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("path", {
                            fill: "url(#f13434__d)",
                            d: "M19.902.685c-2.894-1.688-6.56-1.698-9.465-.026l-29.05 16.72c-2.845 1.638-2.85 5.503-.009 7.147l28.859 16.697c2.9 1.678 6.567 1.676 9.464-.006l28.8-16.712c2.826-1.64 2.83-5.485.008-7.131z"
                        }),
                        /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("path", {
                            fill: "url(#f13434__e)",
                            d: "M10.951 23.824C11.595 10.564 23.521.306 37.588.913s24.949 11.849 24.305 25.11-12.57 23.518-26.637 22.91c-14.067-.606-24.949-11.848-24.305-25.109"
                        }),
                        /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("path", {
                            fill: "url(#f13434__f)",
                            d: "M-9.747 32.567c5.591-2.248 11.622-2.152 16.847-.185 10.455 3.946 16.538 1.506 20.757-8.327 2.102-4.934 6.205-9.113 11.803-11.364 11.192-4.494 24.113.382 28.874 10.902s-.45 22.68-11.64 27.18c-5.61 2.257-11.652 2.151-16.885.167-10.438-3.925-16.51-1.476-20.724 8.368-2.107 4.923-6.21 9.087-11.8 11.336C-3.7 75.14-16.62 70.264-21.379 59.75s.447-22.686 11.63-27.184"
                        }),
                        /*#__PURE__*/ (0, $hJOqE$reactjsxruntime.jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#f13434__g)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 24.586 -2.055)"
                        })
                    ]
                })
            })
        ]
    });
const $6bb621c7bec45bfd$var$ForwardRef = /*#__PURE__*/ (0, $hJOqE$react.forwardRef)($6bb621c7bec45bfd$var$SvgComponent);
var $6bb621c7bec45bfd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6bb621c7bec45bfd$var$ForwardRef);
