// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileVideo_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $lGqmm$reactjsxruntime = require("react/jsx-runtime");
var $lGqmm$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $917f748b396dc4e6$export$2e2bcd8739ae039);




const $917f748b396dc4e6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsxs)("linearGradient", {
                        id: "596207__d",
                        x1: -7.36,
                        x2: 60.505,
                        y1: 40.432,
                        y2: -0.857,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsxs)("linearGradient", {
                        id: "596207__e",
                        x1: 47.099,
                        x2: 17.985,
                        y1: 10.818,
                        y2: 42.124,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsxs)("linearGradient", {
                        id: "596207__f",
                        x1: 11.721,
                        x2: 63.332,
                        y1: 46.444,
                        y2: 17.532,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsxs)("radialGradient", {
                        id: "596207__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 10.992 15.74)scale(18.1603)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsxs)("radialGradient", {
                        id: "596207__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 15.885 26.555)scale(18.1603)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("clipPath", {
                        id: "596207__b",
                        children: /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("clipPath", {
                id: "596207__a",
                children: /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "m36.207 19.323-9.941-9.941c-.678-.679-1.599-1.06-2.558-1.06h-9.356c-1.998 0-3.618 1.62-3.618 3.618v24.12c0 1.998 1.62 3.618 3.618 3.618h19.296c1.999 0 3.618-1.62 3.618-3.618V21.88c0-.959-.38-1.879-1.06-2.557m-7.685 8.672-7.184 4.136c-.805.463-1.81-.118-1.808-1.047l.01-8.29c.001-.928 1.007-1.507 1.81-1.042l7.174 4.154c.804.465.802 1.625-.002 2.089"
                })
            }),
            /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("g", {
                clipPath: "url(#596207__a)",
                children: /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#596207__b)",
                    children: [
                        /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.129 3.129H44.87V44.87H3.129z"
                        }),
                        /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("circle", {
                            cx: 26.733,
                            cy: 4.748,
                            r: 18.16,
                            fill: "url(#596207__c)",
                            transform: "rotate(-.08 26.733 4.748)"
                        }),
                        /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("path", {
                            fill: "url(#596207__d)",
                            d: "M19.062-4.245c-3.205-1.984-7.267-1.996-10.485-.031l-32.183 19.65c-3.151 1.924-3.157 6.466-.01 8.398L8.356 43.394c3.213 1.972 7.275 1.97 10.485-.007l31.904-19.64c3.131-1.928 3.137-6.445.01-8.38z"
                        }),
                        /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("path", {
                            fill: "url(#596207__e)",
                            d: "M9.143 22.948C9.857 7.364 23.068-4.691 38.652-3.977S66.29 9.948 65.577 25.53c-.713 15.584-13.925 27.639-29.509 26.925S8.43 38.531 9.143 22.948"
                        }),
                        /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("path", {
                            fill: "url(#596207__f)",
                            d: "M-13.781 33.222c6.194-2.642 12.875-2.529 18.663-.218 11.582 4.637 18.321 1.77 22.995-9.785 2.329-5.798 6.874-10.71 13.076-13.355 12.399-5.281 26.713.45 31.987 12.812S72.44 49.33 60.045 54.618c-6.215 2.651-12.909 2.527-18.706.195-11.563-4.612-18.29-1.734-22.958 9.835-2.334 5.785-6.878 10.679-13.073 13.321-12.389 5.285-26.703-.445-31.974-12.8s.496-26.661 12.885-31.947"
                        }),
                        /*#__PURE__*/ (0, $lGqmm$reactjsxruntime.jsx)("circle", {
                            cx: 42.44,
                            cy: 10.67,
                            r: 18.16,
                            fill: "url(#596207__g)",
                            transform: "rotate(-.08 42.44 10.67)"
                        })
                    ]
                })
            })
        ]
    });
const $917f748b396dc4e6$var$ForwardRef = /*#__PURE__*/ (0, $lGqmm$react.forwardRef)($917f748b396dc4e6$var$SvgComponent);
var $917f748b396dc4e6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($917f748b396dc4e6$var$ForwardRef);
