// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_folderShared_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $h614n$reactjsxruntime = require("react/jsx-runtime");
var $h614n$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7386f17dbc2cc4de$export$2e2bcd8739ae039);




const $7386f17dbc2cc4de$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $h614n$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0c51ca__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.896,
                        y2: -4.582,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0c51ca__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.843,
                        y2: 44.841,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0c51ca__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.81,
                        y2: 16.564,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsxs)("radialGradient", {
                        id: "0c51ca__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.503)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsxs)("radialGradient", {
                        id: "0c51ca__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.264 26.937)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("clipPath", {
                        id: "0c51ca__b",
                        children: /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("clipPath", {
                id: "0c51ca__a",
                children: /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M6 12.9h15.463l-2.795-3.464C18.14 8.782 17.34 8.4 16.494 8.4h-7.71C7.246 8.4 6 9.636 6 11.16zm0 2.4h31.846C40.14 15.3 42 17.174 42 19.483v15.935c0 2.308-1.861 4.182-4.154 4.182H10.154C7.86 39.6 6 37.726 6 35.418zm15.688 7.95c-.213 0-.386.173-.386.387 0 .109.043.206.117.277.497.48.51 1.272.03 1.768-.481.496-1.273.509-1.769.028-.54-.523-.878-1.26-.878-2.073 0-1.595 1.292-2.887 2.886-2.887.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25m1.106 1.397c0-1.502 1.218-2.72 2.72-2.72s2.721 1.218 2.721 2.72-1.218 2.72-2.72 2.72-2.721-1.217-2.721-2.72m-1.116 3.019c.298.622.034 1.369-.59 1.666-.556.266-.991.82-1.106 1.495-.115.68-.76 1.139-1.441 1.023-.68-.115-1.139-.76-1.023-1.441.249-1.47 1.196-2.713 2.493-3.333.623-.297 1.37-.034 1.667.59m-.6 3.429c.47-1.956 2.363-3.286 4.437-3.286s3.966 1.33 4.437 3.286c.302 1.255-.76 2.155-1.763 2.155h-5.35c-1.002 0-2.064-.9-1.762-2.155"
                })
            }),
            /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("g", {
                clipPath: "url(#0c51ca__a)",
                children: /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#0c51ca__b)",
                    children: [
                        /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 .001h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.863,
                            r: 20.883,
                            fill: "url(#0c51ca__c)",
                            transform: "rotate(-.08 27.142 1.863)"
                        }),
                        /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("path", {
                            fill: "url(#0c51ca__d)",
                            d: "M18.321-8.478c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.657L6.008 46.304c3.694 2.267 8.365 2.264 12.056-.008L54.752 23.71c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("path", {
                            fill: "url(#0c51ca__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.99 40.85-8.17S72.63 7.84 71.809 25.76 55.798 57.543 37.879 56.722C19.957 55.902 6.095 40.71 6.915 22.79"
                        }),
                        /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("path", {
                            fill: "url(#0c51ca__f)",
                            d: "M-19.448 34.606c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.331 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.649-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.279-15.033 15.318-14.246 6.077-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $h614n$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.673,
                            r: 20.883,
                            fill: "url(#0c51ca__g)",
                            transform: "rotate(-.08 45.2 8.673)"
                        })
                    ]
                })
            })
        ]
    });
const $7386f17dbc2cc4de$var$ForwardRef = /*#__PURE__*/ (0, $h614n$react.forwardRef)($7386f17dbc2cc4de$var$SvgComponent);
var $7386f17dbc2cc4de$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7386f17dbc2cc4de$var$ForwardRef);
