// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_star_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $h06I2$reactjsxruntime = require("react/jsx-runtime");
var $h06I2$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $09a11459a2ca3aee$export$2e2bcd8739ae039);




const $09a11459a2ca3aee$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "86ac3c__d",
                        x1: -7.203,
                        x2: 60.325,
                        y1: 40.349,
                        y2: -0.734,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "86ac3c__e",
                        x1: 46.986,
                        x2: 18.016,
                        y1: 10.883,
                        y2: 42.033,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "86ac3c__f",
                        x1: 11.783,
                        x2: 63.137,
                        y1: 46.333,
                        y2: 17.564,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "86ac3c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 10.938 15.782)scale(18.0699)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "86ac3c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 15.806 26.543)scale(18.0699)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("clipPath", {
                        id: "86ac3c__b",
                        children: /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.234,
                            y: 3.232,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("clipPath", {
                id: "86ac3c__a",
                children: /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "m26.462 9.93 2.898 5.873c.4.81 1.173 1.372 2.067 1.502l6.48.942c2.252.327 3.151 3.094 1.522 4.683l-4.69 4.57c-.646.631-.942 1.54-.789 2.43l1.107 6.455c.385 2.242-1.97 3.953-3.984 2.894l-5.796-3.047c-.8-.42-1.755-.42-2.555 0l-5.796 3.047c-2.014 1.059-4.368-.652-3.984-2.894l1.107-6.454c.153-.891-.142-1.8-.79-2.43L8.57 22.93c-1.63-1.589-.73-4.356 1.522-4.683l6.48-.942c.895-.13 1.668-.692 2.068-1.502l2.898-5.872c1.007-2.04 3.916-2.04 4.924 0"
                })
            }),
            /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("g", {
                clipPath: "url(#86ac3c__a)",
                children: /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#86ac3c__b)",
                    children: [
                        /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.234,
                            y: 3.232,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.234 3.232H44.77v41.535H3.234z"
                        }),
                        /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("circle", {
                            cx: 26.72,
                            cy: 4.843,
                            r: 18.07,
                            fill: "url(#86ac3c__c)",
                            transform: "rotate(-.08 26.72 4.843)"
                        }),
                        /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("path", {
                            fill: "url(#86ac3c__d)",
                            d: "M19.088-4.105c-3.19-1.974-7.231-1.986-10.433-.031l-32.022 19.552c-3.137 1.915-3.142 6.435-.01 8.357L8.434 43.297c3.197 1.962 7.239 1.96 10.433-.007l31.745-19.542c3.116-1.918 3.121-6.413.01-8.338z"
                        }),
                        /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("path", {
                            fill: "url(#86ac3c__e)",
                            d: "M9.219 22.952c.71-15.506 13.855-27.5 29.361-26.79s27.501 13.855 26.791 29.361S51.516 53.024 36.01 52.314 8.509 38.46 9.219 22.952"
                        }),
                        /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("path", {
                            fill: "url(#86ac3c__f)",
                            d: "M-13.593 33.177c6.164-2.63 12.812-2.517 18.57-.217 11.525 4.614 18.23 1.761 22.882-9.737 2.316-5.77 6.84-10.656 13.01-13.288 12.337-5.255 26.58.447 31.828 12.748s-.496 26.52-12.83 31.783c-6.185 2.638-12.845 2.515-18.613.194-11.506-4.59-18.199-1.726-22.844 9.786C16.087 70.202 11.566 75.07 5.402 77.7c-12.328 5.259-26.57-.443-31.815-12.737s.493-26.528 12.82-31.787"
                        }),
                        /*#__PURE__*/ (0, $h06I2$reactjsxruntime.jsx)("circle", {
                            cx: 42.349,
                            cy: 10.736,
                            r: 18.07,
                            fill: "url(#86ac3c__g)",
                            transform: "rotate(-.08 42.35 10.736)"
                        })
                    ]
                })
            })
        ]
    });
const $09a11459a2ca3aee$var$ForwardRef = /*#__PURE__*/ (0, $h06I2$react.forwardRef)($09a11459a2ca3aee$var$SvgComponent);
var $09a11459a2ca3aee$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($09a11459a2ca3aee$var$ForwardRef);
