// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_buildTable_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $iPOYu$jsxs, jsx as $iPOYu$jsx} from "react/jsx-runtime";
import {forwardRef as $iPOYu$forwardRef} from "react";





const $1698533dc9a0c083$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iPOYu$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iPOYu$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iPOYu$jsxs)("linearGradient", {
                        id: "aa6b1d__d",
                        x1: -9.711,
                        x2: 120.12,
                        y1: 83.47,
                        y2: -0.321,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iPOYu$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $iPOYu$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iPOYu$jsxs)("linearGradient", {
                        id: "aa6b1d__e",
                        x1: 98.009,
                        x2: 44.052,
                        y1: 28.256,
                        y2: 89.803,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iPOYu$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iPOYu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iPOYu$jsxs)("linearGradient", {
                        id: "aa6b1d__f",
                        x1: 28.029,
                        x2: 127.141,
                        y1: 94.679,
                        y2: 35.782,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iPOYu$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iPOYu$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $iPOYu$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $iPOYu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iPOYu$jsxs)("radialGradient", {
                        id: "aa6b1d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 33.859 -35.918 0 35.918 33.859)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iPOYu$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $iPOYu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iPOYu$jsxs)("radialGradient", {
                        id: "aa6b1d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 33.859 -35.918 0 35.918 33.859)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iPOYu$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iPOYu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iPOYu$jsx)("clipPath", {
                        id: "aa6b1d__b",
                        children: /*#__PURE__*/ (0, $iPOYu$jsx)("rect", {
                            width: 82.56,
                            height: 77.827,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iPOYu$jsx)("clipPath", {
                id: "aa6b1d__a",
                children: /*#__PURE__*/ (0, $iPOYu$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M74.38 20.34c5.305 0 9.6 4.296 9.6 9.6v2.454H11.97V29.94c0-5.304 4.296-9.6 9.6-9.6zm9.601 31.536c-1.548-.396-3.168-.606-4.836-.606h-.006c-10.764 0-19.518 8.754-19.518 19.518 0 1.662.21 3.282.606 4.824H28.8V37.194h55.182zM11.97 66.018c0 5.304 4.296 9.6 9.6 9.6h2.43V37.2h-12.03zm67.177-9.942c1.692 0 3.318.288 4.836.816h.006c5.754 2.004 9.882 7.47 9.882 13.902 0 8.13-6.588 14.718-14.718 14.718-6.438 0-11.91-4.134-13.908-9.894-.528-1.512-.816-3.132-.816-4.824 0-8.13 6.588-14.718 14.718-14.718m2.406 17.088h4.728c1.326 0 2.4-1.074 2.4-2.4s-1.074-2.4-2.4-2.4h-4.728v-4.728c0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4v4.728h-4.728c-1.326 0-2.4 1.074-2.4 2.4s1.074 2.4 2.4 2.4h4.728v4.728c0 1.326 1.074 2.4 2.4 2.4s2.4-1.074 2.4-2.4z"
                })
            }),
            /*#__PURE__*/ (0, $iPOYu$jsx)("g", {
                clipPath: "url(#aa6b1d__a)",
                children: /*#__PURE__*/ (0, $iPOYu$jsxs)("g", {
                    clipPath: "url(#aa6b1d__b)",
                    children: [
                        /*#__PURE__*/ (0, $iPOYu$jsx)("rect", {
                            width: 82.56,
                            height: 77.827,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $iPOYu$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H93.6v77.827H11.04z"
                        }),
                        /*#__PURE__*/ (0, $iPOYu$jsx)("ellipse", {
                            cx: 35.918,
                            cy: 33.859,
                            fill: "url(#aa6b1d__c)",
                            rx: 35.918,
                            ry: 33.859,
                            transform: "matrix(1 -.00132 .00148 1 21.754 -16.873)"
                        }),
                        /*#__PURE__*/ (0, $iPOYu$jsx)("path", {
                            fill: "url(#aa6b1d__d)",
                            d: "M42.549.173C36.207-3.526 28.174-3.55 21.81.114L-41.842 36.75c-6.234 3.588-6.244 12.058-.019 15.66l63.232 36.584c6.355 3.676 14.388 3.671 20.737-.013l63.102-36.619c6.192-3.594 6.203-12.016.018-15.623z"
                        }),
                        /*#__PURE__*/ (0, $iPOYu$jsx)("path", {
                            fill: "url(#aa6b1d__e)",
                            d: "M22.939 50.87C24.349 21.817 50.479-.66 81.302.67c30.822 1.33 54.664 25.963 53.253 55.018s-27.541 51.531-58.363 50.2-54.664-25.962-53.253-55.017"
                        }),
                        /*#__PURE__*/ (0, $iPOYu$jsx)("path", {
                            fill: "url(#aa6b1d__f)",
                            d: "M-22.41 70.028c12.251-4.927 25.465-4.716 36.912-.406 22.908 8.645 36.237 3.3 45.481-18.245 4.605-10.81 13.596-19.967 25.861-24.9 24.523-9.846 52.834.838 63.265 23.888 10.431 23.048-.986 49.694-25.503 59.554-12.293 4.944-25.532 4.712-36.997.364-22.87-8.599-36.174-3.234-45.407 18.337-4.617 10.785-13.605 19.909-25.857 24.837-24.504 9.854-52.815-.831-63.24-23.867s.98-49.708 25.484-59.562"
                        }),
                        /*#__PURE__*/ (0, $iPOYu$jsx)("ellipse", {
                            cx: 35.918,
                            cy: 33.859,
                            fill: "url(#aa6b1d__g)",
                            rx: 35.918,
                            ry: 33.859,
                            transform: "matrix(1 -.00132 .00148 1 52.82 -5.831)"
                        })
                    ]
                })
            })
        ]
    });
const $1698533dc9a0c083$var$ForwardRef = /*#__PURE__*/ (0, $iPOYu$forwardRef)($1698533dc9a0c083$var$SvgComponent);
var $1698533dc9a0c083$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1698533dc9a0c083$var$ForwardRef);


export {$1698533dc9a0c083$export$2e2bcd8739ae039 as default};
