// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_userGroup_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $9qfG4$jsxs, jsx as $9qfG4$jsx} from "react/jsx-runtime";
import {forwardRef as $9qfG4$forwardRef} from "react";





const $a4f37b596d4de78a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9qfG4$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9qfG4$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9qfG4$jsxs)("linearGradient", {
                        id: "9380f7__d",
                        x1: -11.953,
                        x2: 111.651,
                        y1: 78.607,
                        y2: -1.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9qfG4$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $9qfG4$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9qfG4$jsxs)("linearGradient", {
                        id: "9380f7__e",
                        x1: 90.597,
                        x2: 39.227,
                        y1: 26.041,
                        y2: 84.636,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9qfG4$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9qfG4$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9qfG4$jsxs)("linearGradient", {
                        id: "9380f7__f",
                        x1: 23.977,
                        x2: 118.335,
                        y1: 89.279,
                        y2: 33.206,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9qfG4$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9qfG4$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9qfG4$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9qfG4$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9qfG4$jsxs)("radialGradient", {
                        id: "9380f7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9qfG4$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $9qfG4$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9qfG4$jsxs)("radialGradient", {
                        id: "9380f7__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9qfG4$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9qfG4$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9qfG4$jsx)("clipPath", {
                        id: "9380f7__b",
                        children: /*#__PURE__*/ (0, $9qfG4$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9qfG4$jsx)("clipPath", {
                id: "9380f7__a",
                children: /*#__PURE__*/ (0, $9qfG4$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M70.111 30.27c0 8.119-6.178 14.7-13.8 14.7-7.621 0-13.8-6.581-13.8-14.7s6.179-14.7 13.8-14.7 13.8 6.582 13.8 14.7M43.556 44.448c-1.224.378-2.52.588-3.864.588-7.62 0-13.8-6.582-13.8-14.7s6.18-14.7 13.8-14.7c1.302 0 2.568.198 3.762.558-3.54 3.552-5.748 8.496-5.748 14.076s2.25 10.62 5.85 14.178M24.044 73.482c1.95-9.054 7.59-16.62 15.114-21.24-13.224.252-24.348 9.792-27.042 22.314-.648 3.012 1.602 5.868 4.686 5.868h8.364c-1.224-2.076-1.644-4.536-1.122-6.942M56.305 52.17c13.458 0 24.846 9.636 27.576 22.326.648 3.012-1.602 5.868-4.686 5.868h-45.78c-3.084 0-5.334-2.856-4.686-5.868 2.73-12.69 14.118-22.326 27.576-22.326"
                })
            }),
            /*#__PURE__*/ (0, $9qfG4$jsx)("g", {
                clipPath: "url(#9380f7__a)",
                children: /*#__PURE__*/ (0, $9qfG4$jsxs)("g", {
                    clipPath: "url(#9380f7__b)",
                    children: [
                        /*#__PURE__*/ (0, $9qfG4$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $9qfG4$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M7.8 12.393h78.6v74.094H7.8z"
                        }),
                        /*#__PURE__*/ (0, $9qfG4$jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#9380f7__c)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 18 -16.924)"
                        }),
                        /*#__PURE__*/ (0, $9qfG4$jsx)("path", {
                            fill: "url(#9380f7__d)",
                            d: "M37.8-.696c-6.036-3.52-13.684-3.542-19.743-.055l-60.6 34.879c-5.935 3.416-5.945 11.48-.018 14.908l60.2 34.83c6.05 3.5 13.697 3.495 19.742-.013l60.075-34.862c5.895-3.422 5.906-11.44.018-14.874z"
                        }),
                        /*#__PURE__*/ (0, $9qfG4$jsx)("path", {
                            fill: "url(#9380f7__e)",
                            d: "M19.126 47.571C20.47 19.91 45.346-1.488 74.69-.222c29.344 1.267 52.043 24.718 50.7 52.38-1.344 27.661-26.22 49.059-55.565 47.792-29.343-1.266-52.042-24.717-50.699-52.379"
                        }),
                        /*#__PURE__*/ (0, $9qfG4$jsx)("path", {
                            fill: "url(#9380f7__f)",
                            d: "M-24.044 65.81C-12.379 61.12.2 61.32 11.1 65.423c21.809 8.23 34.498 3.142 43.3-17.37 4.384-10.292 12.943-19.008 24.62-23.705 23.348-9.374 50.301.798 60.231 22.742 9.931 21.943-.939 47.311-24.28 56.698-11.703 4.707-24.307 4.486-35.223.346-21.773-8.186-34.439-3.078-43.229 17.458-4.395 10.268-12.952 18.955-24.617 23.645-23.328 9.382-50.281-.79-60.206-22.721s.933-47.324 24.261-56.706"
                        }),
                        /*#__PURE__*/ (0, $9qfG4$jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#9380f7__g)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 47.578 -6.411)"
                        })
                    ]
                })
            })
        ]
    });
const $a4f37b596d4de78a$var$ForwardRef = /*#__PURE__*/ (0, $9qfG4$forwardRef)($a4f37b596d4de78a$var$SvgComponent);
var $a4f37b596d4de78a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a4f37b596d4de78a$var$ForwardRef);


export {$a4f37b596d4de78a$export$2e2bcd8739ae039 as default};
