// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_search_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2cDRi$jsxs, jsx as $2cDRi$jsx} from "react/jsx-runtime";
import {forwardRef as $2cDRi$forwardRef} from "react";





const $9054e404479b8145$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2cDRi$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2cDRi$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2cDRi$jsxs)("linearGradient", {
                        id: "806b6e__d",
                        x1: -23.677,
                        x2: 187.048,
                        y1: 128.165,
                        y2: -7.834,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2cDRi$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2cDRi$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2cDRi$jsxs)("linearGradient", {
                        id: "806b6e__e",
                        x1: 151.155,
                        x2: 63.579,
                        y1: 38.549,
                        y2: 138.443,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2cDRi$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2cDRi$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2cDRi$jsxs)("linearGradient", {
                        id: "806b6e__f",
                        x1: 37.577,
                        x2: 198.443,
                        y1: 146.36,
                        y2: 50.765,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2cDRi$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2cDRi$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2cDRi$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2cDRi$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2cDRi$jsxs)("radialGradient", {
                        id: "806b6e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 54.9556 -58.2975 0 58.297 54.956)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2cDRi$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2cDRi$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2cDRi$jsxs)("radialGradient", {
                        id: "806b6e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 54.9556 -58.2975 0 58.297 54.956)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2cDRi$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2cDRi$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2cDRi$jsx)("clipPath", {
                        id: "806b6e__b",
                        children: /*#__PURE__*/ (0, $2cDRi$jsx)("rect", {
                            width: 134,
                            height: 126.319,
                            x: 10,
                            y: 15.281,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2cDRi$jsx)("clipPath", {
                id: "806b6e__a",
                children: /*#__PURE__*/ (0, $2cDRi$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M24 63.576c0-20.71 16.79-37.5 37.5-37.5S99 42.866 99 63.576s-16.79 37.5-37.5 37.5S24 84.286 24 63.576m37.5-45.5c-25.129 0-45.5 20.371-45.5 45.5s20.371 45.5 45.5 45.5c11.124 0 21.316-3.992 29.221-10.622l6.81 6.811-2.302 2.302 29.508 29.507c2.809 2.81 7.365 2.81 10.175 0 2.809-2.809 2.809-7.365 0-10.175l-29.508-29.507-2.215 2.216-6.811-6.81C103.008 84.891 107 74.7 107 63.575c0-25.129-20.371-45.5-45.5-45.5"
                })
            }),
            /*#__PURE__*/ (0, $2cDRi$jsx)("g", {
                clipPath: "url(#806b6e__a)",
                children: /*#__PURE__*/ (0, $2cDRi$jsxs)("g", {
                    clipPath: "url(#806b6e__b)",
                    children: [
                        /*#__PURE__*/ (0, $2cDRi$jsx)("rect", {
                            width: 134,
                            height: 126.319,
                            x: 10,
                            y: 15.281,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $2cDRi$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M10 15.281h134v126.32H10z"
                        }),
                        /*#__PURE__*/ (0, $2cDRi$jsx)("ellipse", {
                            cx: 58.297,
                            cy: 54.956,
                            fill: "url(#806b6e__c)",
                            rx: 58.297,
                            ry: 54.956,
                            transform: "matrix(1 -.00132 .00148 1 27.387 -34.698)"
                        }),
                        /*#__PURE__*/ (0, $2cDRi$jsx)("path", {
                            fill: "url(#806b6e__d)",
                            d: "M61.144-7.032c-10.29-6.004-23.33-6.04-33.659-.095L-75.827 52.336c-10.118 5.824-10.135 19.57-.03 25.416L26.771 137.13c10.314 5.967 23.353 5.96 33.658-.021l102.418-59.434c10.051-5.834 10.069-19.503.03-25.358z"
                        }),
                        /*#__PURE__*/ (0, $2cDRi$jsx)("path", {
                            fill: "url(#806b6e__e)",
                            d: "M29.31 75.254C31.6 28.096 74.01-8.384 124.038-6.225s88.724 42.14 86.433 89.298-44.701 83.638-94.727 81.479S27.02 122.413 29.31 75.254"
                        }),
                        /*#__PURE__*/ (0, $2cDRi$jsx)("path", {
                            fill: "url(#806b6e__f)",
                            d: "M-44.29 106.349c19.886-7.997 41.332-7.654 59.913-.659 37.18 14.032 58.814 5.356 73.819-29.612 7.474-17.547 22.066-32.408 41.974-40.414 39.803-15.982 85.754 1.36 102.684 38.77s-1.601 80.658-41.394 96.661c-19.953 8.024-41.44 7.649-60.049.591-37.12-13.957-58.713-5.249-73.699 29.762-7.493 17.505-22.082 32.315-41.967 40.312-39.771 15.994-85.722-1.348-102.643-38.737-16.92-37.389 1.591-80.68 41.363-96.674"
                        }),
                        /*#__PURE__*/ (0, $2cDRi$jsx)("ellipse", {
                            cx: 58.297,
                            cy: 54.956,
                            fill: "url(#806b6e__g)",
                            rx: 58.297,
                            ry: 54.956,
                            transform: "matrix(1 -.00132 .00148 1 77.813 -16.776)"
                        })
                    ]
                })
            })
        ]
    });
const $9054e404479b8145$var$ForwardRef = /*#__PURE__*/ (0, $2cDRi$forwardRef)($9054e404479b8145$var$SvgComponent);
var $9054e404479b8145$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9054e404479b8145$var$ForwardRef);


export {$9054e404479b8145$export$2e2bcd8739ae039 as default};
