// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_megaphonePromote_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7QXfP$jsxs, jsx as $7QXfP$jsx} from "react/jsx-runtime";
import {forwardRef as $7QXfP$forwardRef} from "react";





const $6b9fbc5279156b06$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7QXfP$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7QXfP$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7QXfP$jsxs)("linearGradient", {
                        id: "d79a77__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7QXfP$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $7QXfP$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7QXfP$jsxs)("linearGradient", {
                        id: "d79a77__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7QXfP$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7QXfP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7QXfP$jsxs)("linearGradient", {
                        id: "d79a77__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7QXfP$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7QXfP$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7QXfP$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7QXfP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7QXfP$jsxs)("radialGradient", {
                        id: "d79a77__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7QXfP$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $7QXfP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7QXfP$jsxs)("radialGradient", {
                        id: "d79a77__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7QXfP$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7QXfP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7QXfP$jsx)("clipPath", {
                        id: "d79a77__b",
                        children: /*#__PURE__*/ (0, $7QXfP$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7QXfP$jsx)("clipPath", {
                id: "d79a77__a",
                children: /*#__PURE__*/ (0, $7QXfP$jsx)("path", {
                    fill: "#fff",
                    d: "m71.458 18.702-25.86 7.848H27.01c-6.954 0-12.588 5.634-12.588 12.588s5.634 12.588 12.588 12.588h3.564l9.432 21.702c1.5 3.468 5.532 5.064 9 3.564s5.064-5.532 3.564-9l-7.002-16.26h.024l25.854 7.866c3.852 1.17 7.746-1.71 7.746-5.742V24.444c0-4.026-3.888-6.912-7.74-5.742z"
                })
            }),
            /*#__PURE__*/ (0, $7QXfP$jsx)("g", {
                clipPath: "url(#d79a77__a)",
                children: /*#__PURE__*/ (0, $7QXfP$jsxs)("g", {
                    clipPath: "url(#d79a77__b)",
                    children: [
                        /*#__PURE__*/ (0, $7QXfP$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $7QXfP$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $7QXfP$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#d79a77__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $7QXfP$jsx)("path", {
                            fill: "url(#d79a77__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $7QXfP$jsx)("path", {
                            fill: "url(#d79a77__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $7QXfP$jsx)("path", {
                            fill: "url(#d79a77__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $7QXfP$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#d79a77__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $6b9fbc5279156b06$var$ForwardRef = /*#__PURE__*/ (0, $7QXfP$forwardRef)($6b9fbc5279156b06$var$SvgComponent);
var $6b9fbc5279156b06$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6b9fbc5279156b06$var$ForwardRef);


export {$6b9fbc5279156b06$export$2e2bcd8739ae039 as default};
