// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_market_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bwvFc$jsxs, jsx as $bwvFc$jsx} from "react/jsx-runtime";
import {forwardRef as $bwvFc$forwardRef} from "react";





const $7dba97bbef24b939$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bwvFc$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bwvFc$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bwvFc$jsxs)("linearGradient", {
                        id: "a3129a__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bwvFc$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bwvFc$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bwvFc$jsxs)("linearGradient", {
                        id: "a3129a__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bwvFc$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bwvFc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bwvFc$jsxs)("linearGradient", {
                        id: "a3129a__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bwvFc$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bwvFc$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bwvFc$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bwvFc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bwvFc$jsxs)("radialGradient", {
                        id: "a3129a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bwvFc$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bwvFc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bwvFc$jsxs)("radialGradient", {
                        id: "a3129a__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bwvFc$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bwvFc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bwvFc$jsx)("clipPath", {
                        id: "a3129a__b",
                        children: /*#__PURE__*/ (0, $bwvFc$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bwvFc$jsx)("clipPath", {
                id: "a3129a__a",
                children: /*#__PURE__*/ (0, $bwvFc$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M11 24.475V35c0 2.209 1.79 4 4 4h4V24.437c-1.293-.216-2.43-.891-3.237-1.853-.988 1.178-2.473 1.927-4.131 1.927q-.321 0-.632-.037m10.5-.207V32H37v-7.52q-.285.03-.579.031c-1.658 0-3.144-.748-4.132-1.927-.988 1.178-2.473 1.927-4.131 1.927s-3.144-.748-4.132-1.927c-.655.781-1.528 1.374-2.526 1.684m-9.868-2.257c1.528 0 2.771-1.18 2.874-2.665l2.31-.874c.125.194.197.426.197.674 0 1.578 1.286 2.865 2.882 2.865s2.881-1.287 2.881-2.865v-.204c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v.204c0 1.578 1.286 2.865 2.882 2.865s2.881-1.287 2.881-2.865v-.204c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v.204c0 1.578 1.286 2.865 2.882 2.865s2.882-1.287 2.882-2.865c0-.877-.192-1.744-.562-2.54q-.02-.044-.036-.087L36.2 11.135l-19.384 7.337L36.2 11.134c-.223-.48-.707-.788-1.242-.788H13.099c-.536 0-1.02.31-1.243.789l-2.544 5.47v.002c-.371.794-.562 1.661-.562 2.539 0 1.578 1.286 2.865 2.882 2.865m5.184-3.54-2.31.875.008-.24c.021-.682.586-1.22 1.269-1.21.434.007.814.235 1.033.576M21.5 34.5H37v.5c0 2.209-1.79 4-4 4H21.5z"
                })
            }),
            /*#__PURE__*/ (0, $bwvFc$jsx)("g", {
                clipPath: "url(#a3129a__a)",
                children: /*#__PURE__*/ (0, $bwvFc$jsxs)("g", {
                    clipPath: "url(#a3129a__b)",
                    children: [
                        /*#__PURE__*/ (0, $bwvFc$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $bwvFc$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $bwvFc$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#a3129a__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $bwvFc$jsx)("path", {
                            fill: "url(#a3129a__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $bwvFc$jsx)("path", {
                            fill: "url(#a3129a__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $bwvFc$jsx)("path", {
                            fill: "url(#a3129a__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $bwvFc$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#a3129a__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $7dba97bbef24b939$var$ForwardRef = /*#__PURE__*/ (0, $bwvFc$forwardRef)($7dba97bbef24b939$var$SvgComponent);
var $7dba97bbef24b939$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7dba97bbef24b939$var$ForwardRef);


export {$7dba97bbef24b939$export$2e2bcd8739ae039 as default};
