// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_stampClone_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $6S4dY$reactjsxruntime = require("react/jsx-runtime");
var $6S4dY$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9097d37243223a37$export$2e2bcd8739ae039);




const $9097d37243223a37$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3cc782__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3cc782__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3cc782__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.618,
                        y2: 33.126,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3cc782__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3cc782__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("clipPath", {
                        id: "3cc782__b",
                        children: /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("clipPath", {
                id: "3cc782__a",
                children: /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M59.882 27.683c0 4.742-2.78 8.834-6.797 10.736l2.56 13.877h13.164c4.418 0 8 3.582 8 8v3.899c0 1.104-.895 2-2 2H21.188c-1.105 0-2-.896-2-2v-3.9c0-4.417 3.581-8 8-8h13.19l2.56-13.875c-4.02-1.903-6.8-5.995-6.8-10.737 0-6.556 5.316-11.871 11.873-11.871s11.871 5.315 11.871 11.871M24.2 76.19c-1.104 0-2 .895-2 2s.896 2 2 2h47.6c1.104 0 2-.896 2-2s-.896-2-2-2z"
                })
            }),
            /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("g", {
                clipPath: "url(#3cc782__a)",
                children: /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#3cc782__b)",
                    children: [
                        /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#3cc782__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("path", {
                            fill: "url(#3cc782__d)",
                            d: "M36.643-16.957c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.426-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.432 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("path", {
                            fill: "url(#3cc782__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("path", {
                            fill: "url(#3cc782__f)",
                            d: "M-38.895 69.21c14.246-6.077 29.61-5.817 42.922-.5 26.636 10.663 42.135 4.07 52.885-22.505 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.315 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $6S4dY$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#3cc782__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $9097d37243223a37$var$ForwardRef = /*#__PURE__*/ (0, $6S4dY$react.forwardRef)($9097d37243223a37$var$SvgComponent);
var $9097d37243223a37$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9097d37243223a37$var$ForwardRef);
