// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_rocket_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $llpbM$jsxs, jsx as $llpbM$jsx} from "react/jsx-runtime";
import {forwardRef as $llpbM$forwardRef} from "react";





const $84497418bb316e8d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $llpbM$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $llpbM$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $llpbM$jsxs)("linearGradient", {
                        id: "76af74__d",
                        x1: -5.977,
                        x2: 55.826,
                        y1: 40.067,
                        y2: 0.181,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $llpbM$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $llpbM$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $llpbM$jsxs)("linearGradient", {
                        id: "76af74__e",
                        x1: 45.296,
                        x2: 19.612,
                        y1: 13.784,
                        y2: 43.081,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $llpbM$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $llpbM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $llpbM$jsxs)("linearGradient", {
                        id: "76af74__f",
                        x1: 11.986,
                        x2: 59.166,
                        y1: 45.403,
                        y2: 17.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $llpbM$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $llpbM$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $llpbM$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $llpbM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $llpbM$jsxs)("radialGradient", {
                        id: "76af74__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $llpbM$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $llpbM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $llpbM$jsxs)("radialGradient", {
                        id: "76af74__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $llpbM$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $llpbM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $llpbM$jsx)("clipPath", {
                        id: "76af74__b",
                        children: /*#__PURE__*/ (0, $llpbM$jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.96,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $llpbM$jsx)("clipPath", {
                id: "76af74__a",
                children: /*#__PURE__*/ (0, $llpbM$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M32.104 26.166c-1.848 1.764-4.137 3.594-6.966 5.457h-.003q-.355.233-.738.399c-1.785.774-3.9.408-5.313-1.005l-1.062-1.062-1.062-1.062c-1.338-1.341-1.734-3.306-1.113-5.028.13-.354.297-.7.51-1.023 1.347-2.046 2.68-3.807 3.978-5.325C27.886 8.7 34.426 8.13 37.411 8.469c1.104.126 1.974.999 2.1 2.1.315 2.79-.159 8.682-7.407 15.597m.51-10.797c-.993-.99-2.6-.99-3.59 0-.99.993-.994 2.6 0 3.59.992.99 2.6.994 3.59 0 .99-.992.993-2.6 0-3.59m-19.383 9.213-3.144.054v.003c-2.043.036-3.201-2.34-1.905-3.918 3.663-4.455 8.598-6.042 11.817-6.405-1.941 2.046-3.828 4.452-5.646 7.212-.618.939-.987 1.989-1.122 3.054m-4.623 10.98c-.234.234-.543.35-.849.35-.309 0-.615-.116-.849-.35-.468-.471-.468-1.23 0-1.698l3.15-3.15c.471-.468 1.23-.468 1.698 0 .468.47.468 1.23 0 1.698zm4.2-2.1c.471-.468 1.23-.468 1.698 0 .468.47.468 1.23 0 1.698l-6 6c-.234.234-.54.35-.849.35s-.615-.116-.849-.35c-.468-.471-.468-1.23 0-1.698zm2.766 2.763-3.3 3.3c-.468.468-.468 1.227 0 1.698.234.234.54.35.85.35.308 0 .614-.116.848-.35l3.3-3.3c.468-.468.468-1.227 0-1.698s-1.227-.468-1.698 0m7.833-1.467-.053 3.132c-.037 2.055 2.355 3.195 3.939 1.887 4.428-3.654 6.011-8.565 6.38-11.781-1.944 1.836-4.317 3.723-7.215 5.63-.923.61-1.964.99-3.05 1.132"
                })
            }),
            /*#__PURE__*/ (0, $llpbM$jsx)("g", {
                clipPath: "url(#76af74__a)",
                children: /*#__PURE__*/ (0, $llpbM$jsxs)("g", {
                    clipPath: "url(#76af74__b)",
                    children: [
                        /*#__PURE__*/ (0, $llpbM$jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.96,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $llpbM$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.898 6.96h39.3v37.047h-39.3z"
                        }),
                        /*#__PURE__*/ (0, $llpbM$jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#76af74__c)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 9 -7.698)"
                        }),
                        /*#__PURE__*/ (0, $llpbM$jsx)("path", {
                            fill: "url(#76af74__d)",
                            d: "M18.9.416c-3.018-1.76-6.842-1.771-9.872-.028l-30.3 17.44c-2.967 1.708-2.972 5.74-.008 7.454l30.1 17.415c3.024 1.75 6.848 1.747 9.87-.007l30.038-17.43c2.948-1.712 2.953-5.72.009-7.438z"
                        }),
                        /*#__PURE__*/ (0, $llpbM$jsx)("path", {
                            fill: "url(#76af74__e)",
                            d: "M9.561 24.549C10.233 10.718 22.671.019 37.343.652s26.021 12.359 25.35 26.19c-.672 13.83-13.11 24.53-27.782 23.896-14.672-.633-26.022-12.358-25.35-26.19"
                        }),
                        /*#__PURE__*/ (0, $llpbM$jsx)("path", {
                            fill: "url(#76af74__f)",
                            d: "M-12.024 33.669c5.832-2.346 12.122-2.245 17.572-.194 10.904 4.116 17.249 1.571 21.65-8.684 2.192-5.147 6.471-9.505 12.31-11.853 11.673-4.687 25.15.399 30.115 11.37s-.47 23.656-12.14 28.35c-5.852 2.353-12.154 2.243-17.611.173-10.887-4.094-17.22-1.54-21.615 8.728-2.198 5.135-6.476 9.478-12.308 11.823-11.664 4.691-25.141-.395-30.104-11.36-4.962-10.966.467-23.662 12.131-28.353"
                        }),
                        /*#__PURE__*/ (0, $llpbM$jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#76af74__g)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 23.785 -2.442)"
                        })
                    ]
                })
            })
        ]
    });
const $84497418bb316e8d$var$ForwardRef = /*#__PURE__*/ (0, $llpbM$forwardRef)($84497418bb316e8d$var$SvgComponent);
var $84497418bb316e8d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($84497418bb316e8d$var$ForwardRef);


export {$84497418bb316e8d$export$2e2bcd8739ae039 as default};
