// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_handshake_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $kDJ3l$reactjsxruntime = require("react/jsx-runtime");
var $kDJ3l$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7cf5c175fa51c34e$export$2e2bcd8739ae039);




const $7cf5c175fa51c34e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsxs)("linearGradient", {
                        id: "95da27__d",
                        x1: -11.953,
                        x2: 111.651,
                        y1: 78.607,
                        y2: -1.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsxs)("linearGradient", {
                        id: "95da27__e",
                        x1: 90.597,
                        x2: 39.227,
                        y1: 26.041,
                        y2: 84.636,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsxs)("linearGradient", {
                        id: "95da27__f",
                        x1: 23.977,
                        x2: 118.335,
                        y1: 89.279,
                        y2: 33.206,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsxs)("radialGradient", {
                        id: "95da27__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsxs)("radialGradient", {
                        id: "95da27__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("clipPath", {
                        id: "95da27__b",
                        children: /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("clipPath", {
                id: "95da27__a",
                children: /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m36.674 23.829 3.393 3.393 3.468 3.468 6.78 6.78c2.196 2.196 5.754 2.196 7.95 0s2.196-5.754 0-7.95l-6.78-6.78-3.468-3.468-1.128-1.128c-.804-.792-1.71-1.404-2.676-1.854l-.092-.045c-.107-.053-.216-.105-.328-.147-.378-.156-.768-.276-1.158-.384-3.204-.852-6.756-.03-9.27 2.478L15.107 36.45c-2.934 2.934-3.558 7.29-1.902 10.848l20.076-20.076zm25.958-5.59L80.89 36.499v.012c3.75 3.75 3.75 9.828 0 13.578L60.562 70.416l-8.568 8.568c-2.196 2.196-5.754 2.196-7.95-.006h-.006c-2.196-2.196-2.196-5.754 0-7.95-2.196 2.196-5.754 2.196-7.95-.006h-.006c-2.178-2.178-2.19-5.694-.048-7.896-2.202 2.142-5.718 2.124-7.896-.06h-.006c-2.196-2.196-2.196-5.76 0-7.956h-.006c-2.196 2.196-5.754 2.196-7.95-.006h-.006c-.174-.174-.33-.36-.474-.546-1.704-2.208-1.548-5.388.474-7.41l8.976-8.982 7.536-7.536 3.468 3.468 6.78 6.78c1.968 1.968 4.584 3.054 7.368 3.054s5.4-1.086 7.368-3.054c4.062-4.068 4.062-10.68 0-14.742l-9.822-9.822c3.54-1.626 7.866-.996 10.788 1.926"
                })
            }),
            /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("g", {
                clipPath: "url(#95da27__a)",
                children: /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#95da27__b)",
                    children: [
                        /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M7.8 12.393h78.6v74.094H7.8z"
                        }),
                        /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#95da27__c)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 18 -16.924)"
                        }),
                        /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("path", {
                            fill: "url(#95da27__d)",
                            d: "M37.8-.696c-6.036-3.52-13.684-3.542-19.743-.055l-60.6 34.879c-5.935 3.416-5.945 11.48-.018 14.908l60.2 34.83c6.05 3.5 13.697 3.495 19.742-.013l60.075-34.862c5.895-3.422 5.906-11.44.018-14.874z"
                        }),
                        /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("path", {
                            fill: "url(#95da27__e)",
                            d: "M19.126 47.571C20.47 19.91 45.346-1.488 74.69-.222c29.344 1.267 52.043 24.718 50.7 52.38-1.344 27.661-26.22 49.059-55.565 47.792-29.343-1.266-52.042-24.717-50.699-52.379"
                        }),
                        /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("path", {
                            fill: "url(#95da27__f)",
                            d: "M-24.044 65.81C-12.379 61.12.2 61.32 11.1 65.423c21.809 8.23 34.498 3.142 43.3-17.37 4.384-10.292 12.943-19.008 24.62-23.705 23.348-9.374 50.301.798 60.231 22.742 9.931 21.943-.939 47.311-24.28 56.698-11.703 4.707-24.307 4.486-35.223.346-21.773-8.186-34.439-3.078-43.229 17.458-4.395 10.268-12.952 18.955-24.617 23.645-23.328 9.382-50.281-.79-60.206-22.721s.933-47.324 24.261-56.706"
                        }),
                        /*#__PURE__*/ (0, $kDJ3l$reactjsxruntime.jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#95da27__g)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 47.578 -6.411)"
                        })
                    ]
                })
            })
        ]
    });
const $7cf5c175fa51c34e$var$ForwardRef = /*#__PURE__*/ (0, $kDJ3l$react.forwardRef)($7cf5c175fa51c34e$var$SvgComponent);
var $7cf5c175fa51c34e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7cf5c175fa51c34e$var$ForwardRef);
