// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_link_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2trGr$jsxs, jsx as $2trGr$jsx} from "react/jsx-runtime";
import {forwardRef as $2trGr$forwardRef} from "react";





const $9f6ce7a38d13f052$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2trGr$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2trGr$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2trGr$jsxs)("linearGradient", {
                        id: "5158de__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2trGr$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2trGr$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2trGr$jsxs)("linearGradient", {
                        id: "5158de__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2trGr$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2trGr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2trGr$jsxs)("linearGradient", {
                        id: "5158de__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2trGr$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2trGr$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2trGr$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2trGr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2trGr$jsxs)("radialGradient", {
                        id: "5158de__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2trGr$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2trGr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2trGr$jsxs)("radialGradient", {
                        id: "5158de__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2trGr$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2trGr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2trGr$jsx)("clipPath", {
                        id: "5158de__b",
                        children: /*#__PURE__*/ (0, $2trGr$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2trGr$jsx)("clipPath", {
                id: "5158de__a",
                children: /*#__PURE__*/ (0, $2trGr$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    stroke: "#fff",
                    strokeLinecap: "round",
                    strokeLinejoin: "round",
                    strokeWidth: 1.2,
                    d: "M73.916 34.728c3.509-3.509 3.509-9.197 0-12.706-3.508-3.508-9.197-3.508-12.705 0l-6.548 6.548c-.937.937-2.457.937-3.394 0s-.937-2.457 0-3.395l6.548-6.547c5.383-5.383 14.11-5.383 19.493 0s5.383 14.11 0 19.494L60.708 54.724c-5.383 5.383-14.11 5.383-19.494 0-.937-.937-.937-2.456 0-3.394.937-.937 2.457-.937 3.394 0 3.509 3.509 9.197 3.509 12.706 0zm-18.993 6.41c.937.937.937 2.457 0 3.394s-2.457.937-3.394 0c-3.509-3.508-9.197-3.508-12.705 0L22.22 61.135c-3.509 3.508-3.509 9.197 0 12.705s9.197 3.509 12.705 0l6.548-6.547c.937-.938 2.457-.938 3.394 0s.937 2.456 0 3.394l-6.548 6.547c-5.383 5.383-14.11 5.383-19.493 0s-5.383-14.11 0-19.493l16.602-16.603c5.383-5.383 14.11-5.383 19.494 0"
                })
            }),
            /*#__PURE__*/ (0, $2trGr$jsx)("g", {
                clipPath: "url(#5158de__a)",
                children: /*#__PURE__*/ (0, $2trGr$jsxs)("g", {
                    clipPath: "url(#5158de__b)",
                    children: [
                        /*#__PURE__*/ (0, $2trGr$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $2trGr$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $2trGr$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#5158de__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $2trGr$jsx)("path", {
                            fill: "url(#5158de__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $2trGr$jsx)("path", {
                            fill: "url(#5158de__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $2trGr$jsx)("path", {
                            fill: "url(#5158de__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $2trGr$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#5158de__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $9f6ce7a38d13f052$var$ForwardRef = /*#__PURE__*/ (0, $2trGr$forwardRef)($9f6ce7a38d13f052$var$SvgComponent);
var $9f6ce7a38d13f052$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9f6ce7a38d13f052$var$ForwardRef);


export {$9f6ce7a38d13f052$export$2e2bcd8739ae039 as default};
