// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileImage_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $e6tOe$reactjsxruntime = require("react/jsx-runtime");
var $e6tOe$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7ceb6869ec669f08$export$2e2bcd8739ae039);




const $7ceb6869ec669f08$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsxs)("linearGradient", {
                        id: "af2342__d",
                        x1: -6.836,
                        x2: 60.692,
                        y1: 40.349,
                        y2: -0.734,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsxs)("linearGradient", {
                        id: "af2342__e",
                        x1: 47.353,
                        x2: 18.384,
                        y1: 10.883,
                        y2: 42.033,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsxs)("linearGradient", {
                        id: "af2342__f",
                        x1: 12.15,
                        x2: 63.504,
                        y1: 46.333,
                        y2: 17.564,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsxs)("radialGradient", {
                        id: "af2342__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.122 15.965)scale(18.0699)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsxs)("radialGradient", {
                        id: "af2342__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 15.99 26.726)scale(18.0699)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("clipPath", {
                        id: "af2342__b",
                        children: /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.602,
                            y: 3.232,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("clipPath", {
                id: "af2342__a",
                children: /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m26.255 9.455 9.891 9.891c.675.675 1.055 1.59 1.055 2.546V36c0 1.989-1.612 3.6-3.6 3.6H14.4c-1.989 0-3.6-1.611-3.6-3.6V12c0-1.988 1.611-3.6 3.6-3.6h9.309c.954 0 1.87.38 2.545 1.055M28.351 20.4c-1.243 0-2.25 1.008-2.25 2.25 0 1.243 1.007 2.25 2.25 2.25 1.242 0 2.25-1.007 2.25-2.25 0-1.242-1.008-2.25-2.25-2.25M16.202 34.8H31.8c.47 0 .757-.514.511-.914l-3.356-5.455c-.234-.38-.788-.38-1.022 0l-2.645 4.299-3.927-6.66c-.233-.392-.8-.392-1.032 0l-4.642 7.824c-.237.4.051.906.516.906"
                })
            }),
            /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("g", {
                clipPath: "url(#af2342__a)",
                children: /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#af2342__b)",
                    children: [
                        /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.602,
                            y: 3.232,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.602 3.232h41.535v41.535H3.602z"
                        }),
                        /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("circle", {
                            cx: 27.087,
                            cy: 4.843,
                            r: 18.07,
                            fill: "url(#af2342__c)",
                            transform: "rotate(-.08 27.087 4.843)"
                        }),
                        /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("path", {
                            fill: "url(#af2342__d)",
                            d: "M19.456-4.105c-3.19-1.974-7.232-1.986-10.433-.031L-23 15.416c-3.136 1.915-3.142 6.435-.01 8.357L8.801 43.297c3.197 1.962 7.239 1.96 10.433-.007L50.98 23.748c3.115-1.918 3.12-6.413.01-8.338z"
                        }),
                        /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("path", {
                            fill: "url(#af2342__e)",
                            d: "M9.586 22.952c.71-15.506 13.855-27.5 29.362-26.79s27.5 13.855 26.79 29.361-13.855 27.501-29.361 26.791S8.876 38.46 9.586 22.952"
                        }),
                        /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("path", {
                            fill: "url(#af2342__f)",
                            d: "M-13.225 33.177c6.163-2.63 12.81-2.517 18.57-.217 11.524 4.614 18.23 1.761 22.88-9.737 2.318-5.77 6.84-10.656 13.011-13.288 12.338-5.255 26.58.447 31.828 12.748s-.496 26.52-12.83 31.783c-6.185 2.638-12.845 2.515-18.613.194-11.506-4.59-18.199-1.726-22.844 9.786C16.455 70.202 11.933 75.07 5.77 77.7c-12.328 5.259-26.57-.443-31.815-12.737s.493-26.528 12.82-31.787"
                        }),
                        /*#__PURE__*/ (0, $e6tOe$reactjsxruntime.jsx)("circle", {
                            cx: 42.716,
                            cy: 10.736,
                            r: 18.07,
                            fill: "url(#af2342__g)",
                            transform: "rotate(-.08 42.716 10.736)"
                        })
                    ]
                })
            })
        ]
    });
const $7ceb6869ec669f08$var$ForwardRef = /*#__PURE__*/ (0, $e6tOe$react.forwardRef)($7ceb6869ec669f08$var$SvgComponent);
var $7ceb6869ec669f08$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7ceb6869ec669f08$var$ForwardRef);
