// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_exposure_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4DWYn$reactjsxruntime = require("react/jsx-runtime");
var $4DWYn$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0d207aba3979e3d5$export$2e2bcd8739ae039);




const $0d207aba3979e3d5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsxs)("linearGradient", {
                        id: "24438c__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsxs)("linearGradient", {
                        id: "24438c__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsxs)("linearGradient", {
                        id: "24438c__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.618,
                        y2: 33.126,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsxs)("radialGradient", {
                        id: "24438c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsxs)("radialGradient", {
                        id: "24438c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("clipPath", {
                        id: "24438c__b",
                        children: /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("clipPath", {
                id: "24438c__a",
                children: /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M14.688 48.075c0-9.353 3.859-17.804 10.071-23.849l11.205 34.467H16.418c-1.122-3.335-1.73-6.905-1.73-10.618m19.558-7.602-6.195-19.059c5.551-4.152 12.442-6.61 19.907-6.61 5.793 0 11.24 1.48 15.984 4.084zm-16.183 22.22c5.411 11.046 16.765 18.652 29.895 18.652h.141l5.9-18.652zm34.324 18.36 10.821-34.21 16.31 11.79c-3.962 11.85-14.411 20.728-27.131 22.42m-1.279-47.892 29.504 21.326c.405-2.075.617-4.219.617-6.413 0-11.041-5.379-20.826-13.659-26.878z"
                })
            }),
            /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("g", {
                clipPath: "url(#24438c__a)",
                children: /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#24438c__b)",
                    children: [
                        /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#24438c__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("path", {
                            fill: "url(#24438c__d)",
                            d: "M36.643-16.957c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.426-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.432 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("path", {
                            fill: "url(#24438c__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("path", {
                            fill: "url(#24438c__f)",
                            d: "M-38.895 69.21c14.246-6.077 29.61-5.817 42.922-.5 26.636 10.663 42.135 4.07 52.885-22.505 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.315 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $4DWYn$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#24438c__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $0d207aba3979e3d5$var$ForwardRef = /*#__PURE__*/ (0, $4DWYn$react.forwardRef)($0d207aba3979e3d5$var$SvgComponent);
var $0d207aba3979e3d5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0d207aba3979e3d5$var$ForwardRef);
