// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_thumbsUp_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $9ch6J$reactjsxruntime = require("react/jsx-runtime");
var $9ch6J$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e77053e7325c0dc2$export$2e2bcd8739ae039);




const $e77053e7325c0dc2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsxs)("linearGradient", {
                        id: "98fcdf__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsxs)("linearGradient", {
                        id: "98fcdf__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsxs)("linearGradient", {
                        id: "98fcdf__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsxs)("radialGradient", {
                        id: "98fcdf__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsxs)("radialGradient", {
                        id: "98fcdf__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("clipPath", {
                        id: "98fcdf__b",
                        children: /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("clipPath", {
                id: "98fcdf__a",
                children: /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M63.513 129.99c5.48 2.04 11.27 3.09 17.12 3.09l-.02-.02h24.36c.05 0 .15-.01.15-.01h6.75c2.5 0 5.09-.69 7.09-2.22 2.03-1.56 3.38-3.95 3.38-7.17 0-1.04-.15-2.88-1.04-4.68-.42-.87-1.02-1.72-1.84-2.46 4.17-.24 7.69-3.26 8.54-7.39.77-3.75-.84-7.35-3.68-9.39 3.87-.96 6.8-4.36 7.01-8.51.17-3.36-1.47-6.36-4.05-8.12 3.66-1.1 6.38-4.4 6.59-8.4.27-5.29-3.95-9.73-9.25-9.73h-27.72c-1.51 0-2.59-1.47-2.14-2.91l2-6.42c1.79-5.74 2.93-11.66 3.42-17.65l.28-3.47c.42-5.17-3.67-9.61-8.87-9.61-3.43 0-6.55 1.98-8.03 5.07l-9.31 19.5c-.87 1.82-1.87 3.57-2.99 5.24l-.23.35c-5.28 7.856-13.553 13.07-22.745 14.543v55.209c2.197.475 4.362 1.109 6.475 1.898zm-23.225-6.18V70.05h-3.175c-6.08 0-11 4.92-11 11v31.72c0 6.08 4.92 11 11 11h1.23q.975 0 1.945.04"
                })
            }),
            /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("g", {
                clipPath: "url(#98fcdf__a)",
                children: /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#98fcdf__b)",
                    children: [
                        /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#98fcdf__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("path", {
                            fill: "url(#98fcdf__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("path", {
                            fill: "url(#98fcdf__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("path", {
                            fill: "url(#98fcdf__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $9ch6J$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#98fcdf__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $e77053e7325c0dc2$var$ForwardRef = /*#__PURE__*/ (0, $9ch6J$react.forwardRef)($e77053e7325c0dc2$var$SvgComponent);
var $e77053e7325c0dc2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e77053e7325c0dc2$var$ForwardRef);
