// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_shapes_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $71KK6$jsxs, jsx as $71KK6$jsx} from "react/jsx-runtime";
import {forwardRef as $71KK6$forwardRef} from "react";





const $ebd417cb3a1c0e2a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $71KK6$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 97,
        height: 96,
        viewBox: "0 0 97 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $71KK6$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $71KK6$jsxs)("linearGradient", {
                        id: "29f49a__d",
                        x1: -23.183,
                        x2: 132.895,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $71KK6$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $71KK6$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $71KK6$jsxs)("linearGradient", {
                        id: "29f49a__e",
                        x1: 102.07,
                        x2: 35.113,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $71KK6$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $71KK6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $71KK6$jsxs)("linearGradient", {
                        id: "29f49a__f",
                        x1: 20.697,
                        x2: 139.393,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $71KK6$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $71KK6$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $71KK6$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $71KK6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $71KK6$jsxs)("radialGradient", {
                        id: "29f49a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.75 29.473)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $71KK6$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $71KK6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $71KK6$jsxs)("radialGradient", {
                        id: "29f49a__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 37.002 54.345)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $71KK6$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $71KK6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $71KK6$jsx)("clipPath", {
                        id: "29f49a__b",
                        children: /*#__PURE__*/ (0, $71KK6$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $71KK6$jsx)("clipPath", {
                id: "29f49a__a",
                children: /*#__PURE__*/ (0, $71KK6$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M33.851 18.386 16.305 48.59c-1.55 2.667.375 6.01 3.459 6.01H54.81c3.082 0 5.006-3.34 3.46-6.006l-8.756-15.111-8.744-15.092c-1.54-2.658-5.377-2.66-6.92-.004m24.392 61.219c-12.217 0-22.291-9.173-23.712-21.006h20.28c6.164 0 10.012-6.678 6.922-12.011l-8.27-14.273c1.544-.313 3.143-.478 4.78-.478 13.19 0 23.883 10.693 23.883 23.884 0 13.19-10.693 23.884-23.883 23.884"
                })
            }),
            /*#__PURE__*/ (0, $71KK6$jsx)("g", {
                clipPath: "url(#29f49a__a)",
                children: /*#__PURE__*/ (0, $71KK6$jsxs)("g", {
                    clipPath: "url(#29f49a__b)",
                    children: [
                        /*#__PURE__*/ (0, $71KK6$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $71KK6$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M.941 0h96v96h-96z"
                        }),
                        /*#__PURE__*/ (0, $71KK6$jsx)("circle", {
                            cx: 55.222,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#29f49a__c)",
                            transform: "rotate(-.08 55.222 3.723)"
                        }),
                        /*#__PURE__*/ (0, $71KK6$jsx)("path", {
                            fill: "url(#29f49a__d)",
                            d: "M37.584-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.014 45.19c-7.25 4.427-7.261 14.873-.022 19.316l73.525 45.127c7.39 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.201-4.433 7.214-14.821.022-19.27z"
                        }),
                        /*#__PURE__*/ (0, $71KK6$jsx)("path", {
                            fill: "url(#29f49a__e)",
                            d: "M14.778 45.579c1.64-35.84 32.025-63.563 67.864-61.923s63.564 32.025 61.923 67.865-32.025 63.563-67.865 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $71KK6$jsx)("path", {
                            fill: "url(#29f49a__f)",
                            d: "M-37.954 69.21c14.247-6.078 29.611-5.818 42.922-.502 26.637 10.664 42.136 4.071 52.885-22.504 5.355-13.336 15.81-24.63 30.071-30.714 28.516-12.146 61.436 1.033 73.565 29.464s-1.147 61.299-29.656 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.062-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.492 12.155-61.412-1.024-73.534-29.44-12.123-28.415 1.14-61.314 29.632-73.47"
                        }),
                        /*#__PURE__*/ (0, $71KK6$jsx)("circle", {
                            cx: 91.347,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#29f49a__g)",
                            transform: "rotate(-.08 91.347 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $ebd417cb3a1c0e2a$var$ForwardRef = /*#__PURE__*/ (0, $71KK6$forwardRef)($ebd417cb3a1c0e2a$var$SvgComponent);
var $ebd417cb3a1c0e2a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ebd417cb3a1c0e2a$var$ForwardRef);


export {$ebd417cb3a1c0e2a$export$2e2bcd8739ae039 as default};
