// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_trophy_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lSffN$jsxs, jsx as $lSffN$jsx} from "react/jsx-runtime";
import {forwardRef as $lSffN$forwardRef} from "react";





const $480fa60a2c704260$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lSffN$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lSffN$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lSffN$jsxs)("linearGradient", {
                        id: "462ab7__d",
                        x1: -11.355,
                        x2: 112.249,
                        y1: 77.014,
                        y2: -2.759,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lSffN$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lSffN$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lSffN$jsxs)("linearGradient", {
                        id: "462ab7__e",
                        x1: 91.194,
                        x2: 39.825,
                        y1: 24.448,
                        y2: 83.043,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lSffN$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lSffN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lSffN$jsxs)("linearGradient", {
                        id: "462ab7__f",
                        x1: 24.574,
                        x2: 118.933,
                        y1: 87.686,
                        y2: 31.613,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lSffN$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lSffN$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lSffN$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lSffN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lSffN$jsxs)("radialGradient", {
                        id: "462ab7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lSffN$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lSffN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lSffN$jsxs)("radialGradient", {
                        id: "462ab7__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lSffN$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lSffN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lSffN$jsx)("clipPath", {
                        id: "462ab7__b",
                        children: /*#__PURE__*/ (0, $lSffN$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 10.8,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lSffN$jsx)("clipPath", {
                id: "462ab7__a",
                children: /*#__PURE__*/ (0, $lSffN$jsx)("path", {
                    fill: "#fff",
                    d: "M75.622 24H64.798v-3.6c0-1.986-1.614-3.6-3.6-3.6h-26.4c-1.986 0-3.6 1.614-3.6 3.6V24h-10.8c-3.306 0-6 2.694-6 6v1.2c0 9.264 7.536 16.8 16.8 16.8h3.018c2.604 3.732 6.69 6.36 11.4 7.026 0 .06-.018.114-.018.174v14.652c-4.302.81-7.842 3.744-9.504 7.674-.336.792.252 1.668 1.11 1.668h21.588c.858 0 1.446-.876 1.11-1.668-1.662-3.93-5.202-6.864-9.504-7.674V55.2c0-.06-.012-.114-.018-.174 4.716-.672 8.796-3.294 11.4-7.026h3.036c9.264 0 16.8-7.536 16.8-16.8V30c0-3.306-2.694-6-6-6zM31.198 43.2c-6.618 0-12-5.382-12-12V30c0-.66.54-1.2 1.2-1.2h10.8v9.6c0 1.668.246 3.276.702 4.8zm24.474-8.898-2.238 2.172c-.306.3-.45.732-.372 1.152l.534 3.072c.204 1.188-1.044 2.1-2.112 1.536l-2.76-1.458c-.378-.198-.834-.198-1.212 0l-2.76 1.458c-1.068.564-2.316-.342-2.112-1.536l.534-3.072c.072-.42-.066-.852-.372-1.152l-2.238-2.172c-.864-.84-.39-2.31.804-2.484l3.09-.444c.426-.06.792-.33.984-.714l1.374-2.802c.534-1.086 2.076-1.086 2.61 0L50.8 30.66c.192.384.558.654.984.714l3.09.444c1.194.174 1.674 1.644.804 2.484zm21.15-3.102c0 6.618-5.382 12-12 12h-.72c.45-1.524.702-3.132.702-4.8v-9.6h10.824c.66 0 1.2.54 1.2 1.2v1.2z"
                })
            }),
            /*#__PURE__*/ (0, $lSffN$jsx)("g", {
                clipPath: "url(#462ab7__a)",
                children: /*#__PURE__*/ (0, $lSffN$jsxs)("g", {
                    clipPath: "url(#462ab7__b)",
                    children: [
                        /*#__PURE__*/ (0, $lSffN$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 10.8,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $lSffN$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M8.398 10.8h78.6v74.094h-78.6z"
                        }),
                        /*#__PURE__*/ (0, $lSffN$jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#462ab7__c)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 18.598 -18.517)"
                        }),
                        /*#__PURE__*/ (0, $lSffN$jsx)("path", {
                            fill: "url(#462ab7__d)",
                            d: "M38.398-2.288C32.36-5.81 24.713-5.831 18.654-2.344l-60.6 34.88c-5.934 3.415-5.944 11.478-.017 14.907l60.199 34.83c6.05 3.5 13.698 3.495 19.742-.013l60.076-34.862c5.895-3.422 5.906-11.44.018-14.874z"
                        }),
                        /*#__PURE__*/ (0, $lSffN$jsx)("path", {
                            fill: "url(#462ab7__e)",
                            d: "M19.724 45.978C21.067 18.317 45.944-3.08 75.288-1.814s52.043 24.717 50.699 52.379c-1.343 27.661-26.22 49.059-55.564 47.792S18.38 73.64 19.723 45.978"
                        }),
                        /*#__PURE__*/ (0, $lSffN$jsx)("path", {
                            fill: "url(#462ab7__f)",
                            d: "M-23.446 64.217c11.664-4.69 24.244-4.49 35.143-.386 21.808 8.23 34.498 3.141 43.3-17.37 4.383-10.292 12.943-19.01 24.62-23.705 23.347-9.375 50.3.797 60.231 22.74 9.931 21.944-.939 47.312-24.28 56.699-11.704 4.707-24.308 4.486-35.223.347-21.773-8.187-34.439-3.079-43.23 17.457-4.394 10.268-12.952 18.955-24.616 23.646-23.328 9.381-50.282-.791-60.207-22.722s.934-47.324 24.262-56.706"
                        }),
                        /*#__PURE__*/ (0, $lSffN$jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#462ab7__g)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 48.176 -8.004)"
                        })
                    ]
                })
            })
        ]
    });
const $480fa60a2c704260$var$ForwardRef = /*#__PURE__*/ (0, $lSffN$forwardRef)($480fa60a2c704260$var$SvgComponent);
var $480fa60a2c704260$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($480fa60a2c704260$var$ForwardRef);


export {$480fa60a2c704260$export$2e2bcd8739ae039 as default};
