// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_ribbon_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gbGVF$reactjsxruntime = require("react/jsx-runtime");
var $gbGVF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $539ea49ef298d345$export$2e2bcd8739ae039);




const $539ea49ef298d345$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "40ed61__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "40ed61__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "40ed61__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsxs)("radialGradient", {
                        id: "40ed61__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsxs)("radialGradient", {
                        id: "40ed61__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("clipPath", {
                        id: "40ed61__b",
                        children: /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("clipPath", {
                id: "40ed61__a",
                children: /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M116.147 60.147c0 19.963-16.184 36.146-36.147 36.146S43.854 80.11 43.854 60.147 60.037 24 80 24s36.147 16.183 36.147 36.147m-23.764 70.468-10.641-26.432c9.508-.385 18.24-3.782 25.268-9.264l10.318 25.63c.594 1.475-.689 3.02-2.248 2.708l-10.864-2.178c-.782-.157-1.583.167-2.038.824l-6.295 9.103c-.906 1.31-2.905 1.087-3.5-.391M53.028 95.062c7.056 5.444 15.808 8.797 25.328 9.131l-10.723 26.413c-.6 1.479-2.606 1.693-3.505.375l-6.258-9.17c-.455-.667-1.266-.995-2.057-.831l-10.857 2.246c-1.566.324-2.86-1.229-2.259-2.71z"
                })
            }),
            /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("g", {
                clipPath: "url(#40ed61__a)",
                children: /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#40ed61__b)",
                    children: [
                        /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#40ed61__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("path", {
                            fill: "url(#40ed61__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("path", {
                            fill: "url(#40ed61__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("path", {
                            fill: "url(#40ed61__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $gbGVF$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#40ed61__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $539ea49ef298d345$var$ForwardRef = /*#__PURE__*/ (0, $gbGVF$react.forwardRef)($539ea49ef298d345$var$SvgComponent);
var $539ea49ef298d345$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($539ea49ef298d345$var$ForwardRef);
