// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_cut_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $8TIKT$jsxs, jsx as $8TIKT$jsx} from "react/jsx-runtime";
import {forwardRef as $8TIKT$forwardRef} from "react";





const $8b090bb585b10510$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8TIKT$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8TIKT$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8TIKT$jsxs)("linearGradient", {
                        id: "b40601__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8TIKT$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8TIKT$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8TIKT$jsxs)("linearGradient", {
                        id: "b40601__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8TIKT$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8TIKT$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8TIKT$jsxs)("linearGradient", {
                        id: "b40601__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8TIKT$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8TIKT$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8TIKT$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8TIKT$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8TIKT$jsxs)("radialGradient", {
                        id: "b40601__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8TIKT$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8TIKT$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8TIKT$jsxs)("radialGradient", {
                        id: "b40601__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8TIKT$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8TIKT$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8TIKT$jsx)("clipPath", {
                        id: "b40601__b",
                        children: /*#__PURE__*/ (0, $8TIKT$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8TIKT$jsx)("clipPath", {
                id: "b40601__a",
                children: /*#__PURE__*/ (0, $8TIKT$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M99.929 23.655c2.09.716 3.204 2.99 2.489 5.08L87.936 71.037l43.076-14.67c2.092-.711 4.364.407 5.076 2.498s-.406 4.364-2.497 5.076L85.677 80.258q-.143.048-.285.085c-.249.982-.48 1.98-.714 3-.86 3.72-1.783 7.72-3.876 12.236-.41.887-.861 1.786-1.338 2.68q.333-.01.67-.01c10.58 0 19.156 8.576 19.156 19.155s-8.577 19.157-19.156 19.157c-10.58 0-19.156-8.577-19.156-19.157 0-4.155 1.322-8.001 3.57-11.139q.2-.334.472-.632.382-.49.795-.954c1.8-2.297 5.478-7.605 7.729-12.463 1.649-3.56 2.335-6.388 3.1-9.69-1.505.56-3.144 1.208-5.112 2.019-2.812 1.158-6.1 2.997-8.955 4.776-2.89 1.801-5.055 3.363-5.661 3.878q-.212.18-.439.324c-3.473 3.514-8.295 5.692-13.626 5.692-10.58 0-19.156-8.577-19.156-19.156 0-10.58 8.577-19.156 19.156-19.156 10.58 0 19.157 8.576 19.157 19.156l-.003.3c2.117-1.193 4.359-2.338 6.48-3.211 4.208-1.734 7.05-2.786 10.198-3.782l16.166-47.222c.715-2.09 2.99-3.204 5.08-2.489m-28.732 87.071c-1.394 1.862-2.22 4.174-2.22 6.678 0 6.162 4.995 11.157 11.157 11.157 6.161 0 11.156-4.995 11.156-11.157 0-6.161-4.995-11.156-11.156-11.156-3.263 0-6.199 1.401-8.239 3.634q-.389.487-.698.844M42.85 68.903c-6.16 0-11.156 4.994-11.156 11.156 0 6.161 4.995 11.156 11.156 11.156s11.157-4.995 11.157-11.156-4.995-11.156-11.157-11.156"
                })
            }),
            /*#__PURE__*/ (0, $8TIKT$jsx)("g", {
                clipPath: "url(#b40601__a)",
                children: /*#__PURE__*/ (0, $8TIKT$jsxs)("g", {
                    clipPath: "url(#b40601__b)",
                    children: [
                        /*#__PURE__*/ (0, $8TIKT$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $8TIKT$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $8TIKT$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#b40601__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $8TIKT$jsx)("path", {
                            fill: "url(#b40601__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $8TIKT$jsx)("path", {
                            fill: "url(#b40601__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $8TIKT$jsx)("path", {
                            fill: "url(#b40601__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $8TIKT$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#b40601__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $8b090bb585b10510$var$ForwardRef = /*#__PURE__*/ (0, $8TIKT$forwardRef)($8b090bb585b10510$var$SvgComponent);
var $8b090bb585b10510$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8b090bb585b10510$var$ForwardRef);


export {$8b090bb585b10510$export$2e2bcd8739ae039 as default};
