// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_speedFast_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $ll3TZ$jsxs, jsx as $ll3TZ$jsx} from "react/jsx-runtime";
import {forwardRef as $ll3TZ$forwardRef} from "react";





const $5135efc63b5638b3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ll3TZ$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $ll3TZ$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $ll3TZ$jsxs)("linearGradient", {
                        id: "057a6c__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.896,
                        y2: -4.582,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ll3TZ$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $ll3TZ$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ll3TZ$jsxs)("linearGradient", {
                        id: "057a6c__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.843,
                        y2: 44.841,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ll3TZ$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ll3TZ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ll3TZ$jsxs)("linearGradient", {
                        id: "057a6c__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.81,
                        y2: 16.564,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ll3TZ$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $ll3TZ$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $ll3TZ$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $ll3TZ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ll3TZ$jsxs)("radialGradient", {
                        id: "057a6c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.503)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ll3TZ$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $ll3TZ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ll3TZ$jsxs)("radialGradient", {
                        id: "057a6c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.264 26.937)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ll3TZ$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ll3TZ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ll3TZ$jsx)("clipPath", {
                        id: "057a6c__b",
                        children: /*#__PURE__*/ (0, $ll3TZ$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ll3TZ$jsx)("clipPath", {
                id: "057a6c__a",
                children: /*#__PURE__*/ (0, $ll3TZ$jsx)("path", {
                    fill: "#D9D9D9",
                    d: "M22.75 11.882v3.506c0 .69.559 1.25 1.25 1.25.69 0 1.25-.56 1.25-1.25v-3.506q0-.132-.027-.255c2.16.157 4.205.733 6.053 1.646q-.07.084-.129.183l-1.753 3.036c-.345.598-.14 1.363.458 1.708s1.362.14 1.707-.458l1.753-3.036q.05-.087.085-.178c4.004 2.793 6.704 7.326 7.005 12.502h-3.499c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h3.457c-.2 2.172-.825 4.222-1.789 6.066-.322.616-.976.973-1.671.973H11.128c-.696 0-1.35-.357-1.672-.973-.964-1.844-1.588-3.893-1.789-6.066h3.457c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H7.625c.122-2.102.64-4.099 1.481-5.916l.07.044 3.036 1.753c.598.345 1.363.14 1.708-.458s.14-1.362-.458-1.707l-3.036-1.753q-.051-.03-.104-.054c1.149-1.733 2.618-3.235 4.324-4.422q.035.096.089.19l1.753 3.035c.345.598 1.11.803 1.707.458.598-.345.803-1.11.458-1.708L16.9 13.456q-.06-.103-.135-.19c1.836-.905 3.866-1.477 6.01-1.637q-.026.123-.026.253m.027 19.029c-.902.811-.939 2.214-.08 3.072.858.858 2.26.821 3.072-.08l11.014-12.236c.447-.496.427-1.254-.045-1.726-.471-.471-1.23-.491-1.725-.045z"
                })
            }),
            /*#__PURE__*/ (0, $ll3TZ$jsx)("g", {
                clipPath: "url(#057a6c__a)",
                children: /*#__PURE__*/ (0, $ll3TZ$jsxs)("g", {
                    clipPath: "url(#057a6c__b)",
                    children: [
                        /*#__PURE__*/ (0, $ll3TZ$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $ll3TZ$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 .001h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $ll3TZ$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.863,
                            r: 20.883,
                            fill: "url(#057a6c__c)",
                            transform: "rotate(-.08 27.142 1.863)"
                        }),
                        /*#__PURE__*/ (0, $ll3TZ$jsx)("path", {
                            fill: "url(#057a6c__d)",
                            d: "M18.321-8.478c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.657L6.008 46.304c3.694 2.267 8.365 2.264 12.056-.008L54.752 23.71c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $ll3TZ$jsx)("path", {
                            fill: "url(#057a6c__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.99 40.85-8.17S72.63 7.84 71.809 25.76 55.798 57.543 37.879 56.722C19.957 55.902 6.095 40.71 6.915 22.79"
                        }),
                        /*#__PURE__*/ (0, $ll3TZ$jsx)("path", {
                            fill: "url(#057a6c__f)",
                            d: "M-19.448 34.606c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.331 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.649-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.279-15.033 15.318-14.246 6.077-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $ll3TZ$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.673,
                            r: 20.883,
                            fill: "url(#057a6c__g)",
                            transform: "rotate(-.08 45.2 8.673)"
                        })
                    ]
                })
            })
        ]
    });
const $5135efc63b5638b3$var$ForwardRef = /*#__PURE__*/ (0, $ll3TZ$forwardRef)($5135efc63b5638b3$var$SvgComponent);
var $5135efc63b5638b3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5135efc63b5638b3$var$ForwardRef);


export {$5135efc63b5638b3$export$2e2bcd8739ae039 as default};
