// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_crop_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $511nf$jsxs, jsx as $511nf$jsx} from "react/jsx-runtime";
import {forwardRef as $511nf$forwardRef} from "react";





const $23a34370329c1122$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $511nf$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 97,
        height: 96,
        viewBox: "0 0 97 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $511nf$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $511nf$jsxs)("linearGradient", {
                        id: "8d7b6e__d",
                        x1: -23.183,
                        x2: 132.895,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $511nf$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $511nf$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $511nf$jsxs)("linearGradient", {
                        id: "8d7b6e__e",
                        x1: 102.07,
                        x2: 35.113,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $511nf$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $511nf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $511nf$jsxs)("linearGradient", {
                        id: "8d7b6e__f",
                        x1: 20.697,
                        x2: 139.393,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $511nf$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $511nf$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $511nf$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $511nf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $511nf$jsxs)("radialGradient", {
                        id: "8d7b6e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.75 29.473)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $511nf$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $511nf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $511nf$jsxs)("radialGradient", {
                        id: "8d7b6e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 37.002 54.345)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $511nf$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $511nf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $511nf$jsx)("clipPath", {
                        id: "8d7b6e__b",
                        children: /*#__PURE__*/ (0, $511nf$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $511nf$jsx)("clipPath", {
                id: "8d7b6e__a",
                children: /*#__PURE__*/ (0, $511nf$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M32.916 15c0-1.105-.896-2-2-2s-2 .895-2 2v12.903H15.94c-1.104 0-2 .896-2 2s.896 2 2 2h12.975v26.193c0 5.523 4.477 10 10 10h26.05v12.903c0 1.105.896 2 2 2s2-.895 2-2V68.096h12.975c1.104 0 2-.895 2-2s-.896-2-2-2H38.916c-3.314 0-6-2.686-6-6V15m5.441 12.903c-1.104 0-2 .896-2 2s.896 2 2 2h20.61c3.313 0 6 2.687 6 6v20.742c0 1.104.895 2 2 2s2-.896 2-2V37.903c0-5.523-4.478-10-10-10z"
                })
            }),
            /*#__PURE__*/ (0, $511nf$jsx)("g", {
                clipPath: "url(#8d7b6e__a)",
                children: /*#__PURE__*/ (0, $511nf$jsxs)("g", {
                    clipPath: "url(#8d7b6e__b)",
                    children: [
                        /*#__PURE__*/ (0, $511nf$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $511nf$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M.941 0h96v96h-96z"
                        }),
                        /*#__PURE__*/ (0, $511nf$jsx)("circle", {
                            cx: 55.222,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#8d7b6e__c)",
                            transform: "rotate(-.08 55.222 3.723)"
                        }),
                        /*#__PURE__*/ (0, $511nf$jsx)("path", {
                            fill: "url(#8d7b6e__d)",
                            d: "M37.584-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.014 45.19c-7.25 4.427-7.261 14.873-.022 19.316l73.525 45.127c7.39 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.201-4.433 7.214-14.821.022-19.27z"
                        }),
                        /*#__PURE__*/ (0, $511nf$jsx)("path", {
                            fill: "url(#8d7b6e__e)",
                            d: "M14.778 45.579c1.64-35.84 32.025-63.563 67.864-61.923s63.564 32.025 61.923 67.865-32.025 63.563-67.865 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $511nf$jsx)("path", {
                            fill: "url(#8d7b6e__f)",
                            d: "M-37.954 69.21c14.247-6.078 29.611-5.818 42.922-.502 26.637 10.664 42.136 4.071 52.885-22.504 5.355-13.336 15.81-24.63 30.071-30.714 28.516-12.146 61.436 1.033 73.565 29.464s-1.147 61.299-29.656 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.062-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.492 12.155-61.412-1.024-73.534-29.44-12.123-28.415 1.14-61.314 29.632-73.47"
                        }),
                        /*#__PURE__*/ (0, $511nf$jsx)("circle", {
                            cx: 91.347,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#8d7b6e__g)",
                            transform: "rotate(-.08 91.347 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $23a34370329c1122$var$ForwardRef = /*#__PURE__*/ (0, $511nf$forwardRef)($23a34370329c1122$var$SvgComponent);
var $23a34370329c1122$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($23a34370329c1122$var$ForwardRef);


export {$23a34370329c1122$export$2e2bcd8739ae039 as default};
