// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileZip_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $9Ixv0$reactjsxruntime = require("react/jsx-runtime");
var $9Ixv0$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e4346ecd234f8ebf$export$2e2bcd8739ae039);




const $e4346ecd234f8ebf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6ec67d__d",
                        x1: -13.164,
                        x2: 184.35,
                        y1: 129.007,
                        y2: 1.534,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6ec67d__e",
                        x1: 150.707,
                        x2: 68.621,
                        y1: 45.009,
                        y2: 138.642,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6ec67d__f",
                        x1: 44.245,
                        x2: 195.026,
                        y1: 146.062,
                        y2: 56.46,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6ec67d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.5105 -54.6429 0 54.643 51.51)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6ec67d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.5105 -54.6429 0 54.643 51.51)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("clipPath", {
                        id: "6ec67d__b",
                        children: /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.2,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("clipPath", {
                id: "6ec67d__a",
                children: /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m87.792 31.35 32.96 31.53v.02c2.4 2.28 3.75 5.46 3.75 8.78v48.21c0 6.7-5.41 12.13-12.07 12.13h-64.87c-6.67 0-12.07-5.43-12.07-12.13V40.14c0-6.7 5.41-12.13 12.07-12.13h31.91c3.1 0 6.08 1.19 8.32 3.34m4.75 38.51c0-2.21-1.79-4-4-4h-5.07V59.2h5.08c2.21 0 4-1.79 4-4s-1.79-4-4-4h-5.08v-3.33c0-2.21-1.79-4-4-4h-7.99c-2.21 0-4 1.79-4 4s1.79 4 4 4h3.99v6.66h-3.99c-2.21 0-4 1.79-4 4s1.79 4 4 4h3.99v6.66h-3.99c-2.21 0-4 1.79-4 4s1.79 4 4 4h3.99v6.66h-2.99c-2.21 0-4 1.79-4 4v20.09c0 4.41 3.59 8 8 8h6.01c4.41 0 8-3.59 8-8V91.85c0-2.21-1.79-4-4-4h-3.03V73.86h5.08c2.21 0 4-1.79 4-4m-16.06 34.03h6.02v8.04h-6.02z"
                })
            }),
            /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6ec67d__a)",
                children: /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6ec67d__b)",
                    children: [
                        /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.2,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M18.398 23.2h125.6v118.4h-125.6z"
                        }),
                        /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("ellipse", {
                            cx: 54.643,
                            cy: 51.511,
                            fill: "url(#6ec67d__c)",
                            rx: 54.643,
                            ry: 51.511,
                            transform: "matrix(1 -.00132 .00148 1 34.695 -23.645)"
                        }),
                        /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("path", {
                            fill: "url(#6ec67d__d)",
                            d: "M66.34 2.285c-9.647-5.626-21.868-5.661-31.55-.089l-96.836 55.736c-9.483 5.458-9.5 18.343-.028 23.822l96.195 55.657c9.668 5.592 21.89 5.585 31.548-.02l95.998-55.709c9.421-5.467 9.438-18.28.029-23.768z"
                        }),
                        /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("path", {
                            fill: "url(#6ec67d__e)",
                            d: "M36.5 79.414c2.147-44.202 41.9-78.395 88.79-76.371 46.89 2.023 83.162 39.497 81.015 83.7-2.147 44.201-41.899 78.394-88.789 76.37s-83.162-39.497-81.015-83.7"
                        }),
                        /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("path", {
                            fill: "url(#6ec67d__f)",
                            d: "M-32.49 108.559c18.64-7.495 38.741-7.174 56.157-.618 34.85 13.152 55.127 5.021 69.191-27.755 7.006-16.448 20.683-30.377 39.343-37.88 37.308-14.981 80.378 1.274 96.246 36.339 15.869 35.065-1.5 75.602-38.799 90.601-18.701 7.521-38.842 7.169-56.284.554-34.793-13.082-55.032-4.92-69.079 27.896-7.023 16.408-20.697 30.289-39.336 37.785-37.278 14.991-80.348-1.263-96.208-36.309s1.492-75.621 38.77-90.613"
                        }),
                        /*#__PURE__*/ (0, $9Ixv0$reactjsxruntime.jsx)("ellipse", {
                            cx: 54.643,
                            cy: 51.511,
                            fill: "url(#6ec67d__g)",
                            rx: 54.643,
                            ry: 51.511,
                            transform: "matrix(1 -.00132 .00148 1 81.957 -6.848)"
                        })
                    ]
                })
            })
        ]
    });
const $e4346ecd234f8ebf$var$ForwardRef = /*#__PURE__*/ (0, $9Ixv0$react.forwardRef)($e4346ecd234f8ebf$var$SvgComponent);
var $e4346ecd234f8ebf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e4346ecd234f8ebf$var$ForwardRef);
