// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileText_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gFZfF$reactjsxruntime = require("react/jsx-runtime");
var $gFZfF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2e97e4fa49d683e5$export$2e2bcd8739ae039);




const $2e97e4fa49d683e5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "99fb20__d",
                        x1: -7.203,
                        x2: 60.325,
                        y1: 40.349,
                        y2: -0.734,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "99fb20__e",
                        x1: 46.986,
                        x2: 18.016,
                        y1: 10.883,
                        y2: 42.033,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "99fb20__f",
                        x1: 11.783,
                        x2: 63.137,
                        y1: 46.333,
                        y2: 17.564,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsxs)("radialGradient", {
                        id: "99fb20__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 10.938 15.782)scale(18.0699)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsxs)("radialGradient", {
                        id: "99fb20__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 15.806 26.543)scale(18.0699)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("clipPath", {
                        id: "99fb20__b",
                        children: /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.234,
                            y: 3.232,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("clipPath", {
                id: "99fb20__a",
                children: /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "m36.146 19.346-9.89-9.891C25.58 8.78 24.663 8.4 23.71 8.4H14.4c-1.988 0-3.6 1.612-3.6 3.6v24c0 1.989 1.612 3.6 3.6 3.6h19.2c1.989 0 3.6-1.611 3.6-3.6V21.892c0-.955-.379-1.87-1.054-2.546M31.201 25.2c0 .663-.538 1.2-1.2 1.2-.663 0-1.2-.537-1.2-1.2V24h-3.6v9.6h1.2c.663 0 1.2.538 1.2 1.2 0 .663-.537 1.2-1.2 1.2h-4.8c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.2-1.2h1.2V24h-3.6v1.2c0 .663-.538 1.2-1.2 1.2-.663 0-1.2-.537-1.2-1.2v-2.4c0-.662.537-1.2 1.2-1.2h12c.662 0 1.2.538 1.2 1.2z"
                })
            }),
            /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("g", {
                clipPath: "url(#99fb20__a)",
                children: /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#99fb20__b)",
                    children: [
                        /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.234,
                            y: 3.232,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.234 3.232H44.77v41.535H3.234z"
                        }),
                        /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("circle", {
                            cx: 26.72,
                            cy: 4.843,
                            r: 18.07,
                            fill: "url(#99fb20__c)",
                            transform: "rotate(-.08 26.72 4.843)"
                        }),
                        /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("path", {
                            fill: "url(#99fb20__d)",
                            d: "M19.088-4.105c-3.19-1.974-7.231-1.986-10.433-.031l-32.022 19.552c-3.137 1.915-3.142 6.435-.01 8.357L8.434 43.297c3.197 1.962 7.239 1.96 10.433-.007l31.745-19.542c3.116-1.918 3.121-6.413.01-8.338z"
                        }),
                        /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("path", {
                            fill: "url(#99fb20__e)",
                            d: "M9.219 22.952c.71-15.506 13.855-27.5 29.361-26.79s27.501 13.855 26.791 29.361S51.516 53.024 36.01 52.314 8.509 38.46 9.219 22.952"
                        }),
                        /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("path", {
                            fill: "url(#99fb20__f)",
                            d: "M-13.593 33.177c6.164-2.63 12.812-2.517 18.57-.217 11.525 4.614 18.23 1.761 22.882-9.737 2.316-5.77 6.84-10.656 13.01-13.288 12.337-5.255 26.58.447 31.828 12.748s-.496 26.52-12.83 31.783c-6.185 2.638-12.845 2.515-18.613.194-11.506-4.59-18.199-1.726-22.844 9.786C16.087 70.202 11.566 75.07 5.402 77.7c-12.328 5.259-26.57-.443-31.815-12.737s.493-26.528 12.82-31.787"
                        }),
                        /*#__PURE__*/ (0, $gFZfF$reactjsxruntime.jsx)("circle", {
                            cx: 42.349,
                            cy: 10.736,
                            r: 18.07,
                            fill: "url(#99fb20__g)",
                            transform: "rotate(-.08 42.35 10.736)"
                        })
                    ]
                })
            })
        ]
    });
const $2e97e4fa49d683e5$var$ForwardRef = /*#__PURE__*/ (0, $gFZfF$react.forwardRef)($2e97e4fa49d683e5$var$SvgComponent);
var $2e97e4fa49d683e5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2e97e4fa49d683e5$var$ForwardRef);
