// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_gift_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lR9WB$jsxs, jsx as $lR9WB$jsx} from "react/jsx-runtime";
import {forwardRef as $lR9WB$forwardRef} from "react";





const $822c9273e463fb59$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lR9WB$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lR9WB$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lR9WB$jsxs)("linearGradient", {
                        id: "5eadaf__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lR9WB$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lR9WB$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lR9WB$jsxs)("linearGradient", {
                        id: "5eadaf__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lR9WB$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lR9WB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lR9WB$jsxs)("linearGradient", {
                        id: "5eadaf__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lR9WB$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lR9WB$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lR9WB$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lR9WB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lR9WB$jsxs)("radialGradient", {
                        id: "5eadaf__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lR9WB$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lR9WB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lR9WB$jsxs)("radialGradient", {
                        id: "5eadaf__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lR9WB$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lR9WB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lR9WB$jsx)("clipPath", {
                        id: "5eadaf__b",
                        children: /*#__PURE__*/ (0, $lR9WB$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lR9WB$jsx)("clipPath", {
                id: "5eadaf__a",
                children: /*#__PURE__*/ (0, $lR9WB$jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M14.357 12.342c0-2.95 2.392-5.342 5.342-5.342 1.696 0 3.352.79 4.33 2.023C25.01 7.79 26.376 7 28.073 7c2.95 0 5.342 2.392 5.342 5.342 0 1.126-.348 2.17-.942 3.03h-4.369c1.66-.016 3-1.367 3-3.03 0-1.673-1.358-3.03-3.031-3.03-1.666 0-2.73 1.344-2.742 3.008v3.053l-2.6-.002V12.32c-.013-1.664-1.365-3.008-3.031-3.008-1.674 0-3.03 1.357-3.03 3.03 0 1.663 1.34 3.014 2.999 3.03h-4.37c-.593-.86-.941-1.904-.941-3.03M8 17.692c0-1.277 1.035-2.312 2.312-2.312h12.425v9.536H10.312C9.035 24.916 8 23.881 8 22.604zm32.075 0c0-1.277-1.035-2.312-2.312-2.312H25.339v9.536h12.426c1.276 0 2.311-1.035 2.311-2.312zm-14.738 9.536h12.715v7.224c0 2.873-2.329 5.201-5.201 5.201h-7.514zm-15.314 0h12.715v12.425h-7.513c-2.873 0-5.202-2.328-5.202-5.201z"
                })
            }),
            /*#__PURE__*/ (0, $lR9WB$jsx)("g", {
                clipPath: "url(#5eadaf__a)",
                children: /*#__PURE__*/ (0, $lR9WB$jsxs)("g", {
                    clipPath: "url(#5eadaf__b)",
                    children: [
                        /*#__PURE__*/ (0, $lR9WB$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $lR9WB$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $lR9WB$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#5eadaf__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $lR9WB$jsx)("path", {
                            fill: "url(#5eadaf__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $lR9WB$jsx)("path", {
                            fill: "url(#5eadaf__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $lR9WB$jsx)("path", {
                            fill: "url(#5eadaf__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $lR9WB$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#5eadaf__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $822c9273e463fb59$var$ForwardRef = /*#__PURE__*/ (0, $lR9WB$forwardRef)($822c9273e463fb59$var$SvgComponent);
var $822c9273e463fb59$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($822c9273e463fb59$var$ForwardRef);


export {$822c9273e463fb59$export$2e2bcd8739ae039 as default};
