// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fireworks_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $1dtfx$reactjsxruntime = require("react/jsx-runtime");
var $1dtfx$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $14c8378f774599cc$export$2e2bcd8739ae039);




const $14c8378f774599cc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ee1f53__d",
                        x1: -19.921,
                        x2: 186.087,
                        y1: 131.011,
                        y2: -1.943,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ee1f53__e",
                        x1: 150.995,
                        x2: 65.38,
                        y1: 43.401,
                        y2: 141.06,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ee1f53__f",
                        x1: 39.959,
                        x2: 197.223,
                        y1: 148.798,
                        y2: 55.343,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ee1f53__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ee1f53__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("clipPath", {
                        id: "ee1f53__b",
                        children: /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("clipPath", {
                id: "ee1f53__a",
                children: /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M128.11 28.244c.035-2.21-1.727-4.028-3.936-4.064-2.209-.035-4.028 1.727-4.063 3.936l-.056 3.498-3.503-.054c-2.209-.034-4.027 1.73-4.061 3.938-.034 2.21 1.729 4.027 3.938 4.061l3.498.054-.056 3.503c-.036 2.209 1.726 4.028 3.935 4.063s4.028-1.727 4.064-3.935l.056-3.508 3.503.053c2.209.034 4.027-1.73 4.061-3.938s-1.73-4.027-3.938-4.06l-3.498-.055zM66.99 30.79c0-2.21-1.79-4-4-4s-4 1.79-4 4v10.14c0 2.209 1.79 4 4 4s4-1.791 4-4zm-25.03 2.823c1.788-1.298 4.29-.9 5.587.887l5.96 8.21c1.298 1.788.9 4.289-.887 5.587-1.788 1.297-4.29.9-5.587-.887l-5.96-8.21c-1.298-1.788-.9-4.29.887-5.587m-7.965 15.332c-2.1-.682-4.357.468-5.04 2.57-.682 2.1.469 4.357 2.57 5.04l9.64 3.13c2.1.681 4.357-.469 5.04-2.57s-.469-4.358-2.57-5.04zm12.21 19.08c.681 2.102-.47 4.358-2.57 5.04l-9.65 3.13c-2.102.681-4.358-.47-5.04-2.571-.681-2.102.47-4.358 2.57-5.04l9.65-3.13c2.102-.68 4.358.47 5.04 2.572m7.29 14.416c1.3-1.787.904-4.288-.883-5.587s-4.289-.903-5.588.884l-5.96 8.2c-1.298 1.787-.903 4.288.884 5.587 1.787 1.3 4.289.903 5.588-.884zM66.98 84.22c0-2.21-1.79-4-4-4s-4 1.79-4 4v10.14c0 2.209 1.79 4 4 4s4-1.791 4-4zm17.052-50.596c1.787 1.299 2.183 3.8.884 5.587l-5.96 8.2c-1.3 1.787-3.8 2.183-5.588.884-1.787-1.299-2.182-3.8-.884-5.587l5.96-8.2c1.3-1.787 3.8-2.183 5.588-.884m10.423 22.94c2.101-.682 3.252-2.938 2.57-5.04-.683-2.1-2.94-3.251-5.04-2.569l-9.64 3.13c-2.101.682-3.252 2.939-2.57 5.04.683 2.101 2.94 3.251 5.04 2.57zm-14.68 11.47c.683-2.1 2.94-3.251 5.04-2.569l9.64 3.13c2.101.682 3.252 2.939 2.57 5.04-.683 2.101-2.94 3.251-5.04 2.57l-9.64-3.13c-2.101-.683-3.252-2.94-2.57-5.04m-.828 9.706c-1.298-1.788-3.8-2.185-5.587-.887-1.788 1.297-2.185 3.799-.887 5.587l5.96 8.21c1.298 1.787 3.8 2.185 5.587.887 1.788-1.298 2.185-3.8.887-5.587zm30.09 37.22c2.209.009 3.992 1.807 3.984 4.016l-.02 4.95c-.009 2.209-1.807 3.993-4.017 3.984-2.209-.009-3.992-1.807-3.983-4.016l.02-4.95c.009-2.21 1.807-3.993 4.016-3.984m-8.94 1.182c1.917-1.097 2.583-3.541 1.486-5.459s-3.541-2.582-5.459-1.485l-4.3 2.46c-1.917 1.097-2.582 3.54-1.485 5.458s3.54 2.583 5.458 1.486zm-9.734-20.574c1.111-1.91 3.56-2.557 5.469-1.446l4.28 2.49c1.91 1.111 2.557 3.56 1.446 5.469s-3.56 2.557-5.469 1.446l-4.28-2.49c-1.91-1.11-2.557-3.56-1.446-5.469m22.658-6.742c.008-2.21-1.775-4.007-3.984-4.016s-4.007 1.774-4.016 3.983l-.02 4.95c-.009 2.21 1.774 4.008 3.983 4.017s4.008-1.775 4.017-3.984zm14.672 6.777c1.097 1.918.431 4.362-1.486 5.459l-4.3 2.46c-1.918 1.097-4.361.432-5.458-1.486s-.432-4.361 1.485-5.458l4.3-2.46c1.918-1.097 4.362-.432 5.459 1.485m-5.751 13.589c-1.909-1.111-4.358-.463-5.469 1.446s-.463 4.358 1.446 5.469l4.28 2.49c1.91 1.111 4.358.464 5.469-1.446s.464-4.358-1.446-5.469zm-98.461 15.296c.034-2.209 1.852-3.972 4.06-3.938l3.496.054.054-3.506c.034-2.209 1.852-3.972 4.06-3.938 2.21.034 3.973 1.852 3.939 4.061l-.054 3.505 3.506.054c2.209.034 3.972 1.852 3.938 4.061s-1.852 3.972-4.06 3.938l-3.506-.054-.054 3.496c-.034 2.209-1.852 3.972-4.06 3.938-2.21-.034-3.973-1.852-3.94-4.061l.055-3.495-3.496-.054c-2.209-.034-3.972-1.852-3.938-4.061"
                })
            }),
            /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ee1f53__a)",
                children: /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ee1f53__b)",
                    children: [
                        /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M13 20.654h131v123.491H13z"
                        }),
                        /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#ee1f53__c)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 30 -28.206)"
                        }),
                        /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("path", {
                            fill: "url(#ee1f53__d)",
                            d: "M63.002-1.16C52.94-7.027 40.194-7.063 30.096-1.251l-101 58.132c-9.891 5.693-9.907 19.132-.03 24.847L29.4 139.776c10.082 5.833 22.83 5.826 32.904-.021l100.125-58.103c9.826-5.703 9.844-19.067.03-24.79z"
                        }),
                        /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("path", {
                            fill: "url(#ee1f53__e)",
                            d: "M31.878 79.285C34.118 33.183 75.58-2.48 124.485-.369c48.907 2.11 86.738 41.195 84.499 87.298s-43.701 81.765-92.607 79.655c-48.906-2.111-86.738-41.196-84.499-87.299"
                        }),
                        /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("path", {
                            fill: "url(#ee1f53__f)",
                            d: "M-40.075 109.683c19.44-7.818 40.406-7.483 58.57-.645 36.349 13.718 57.498 5.237 72.167-28.949 7.307-17.154 21.572-31.682 41.035-39.509 38.911-15.624 83.833 1.33 100.384 37.902s-1.565 78.853-40.467 94.497c-19.506 7.845-40.512 7.477-58.705.577-36.288-13.644-57.398-5.13-72.048 29.096-7.325 17.113-21.588 31.592-41.028 39.41-38.88 15.636-83.803-1.318-100.344-37.87-16.542-36.552 1.555-78.873 40.436-94.509"
                        }),
                        /*#__PURE__*/ (0, $1dtfx$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#ee1f53__g)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 79.293 -10.685)"
                        })
                    ]
                })
            })
        ]
    });
const $14c8378f774599cc$var$ForwardRef = /*#__PURE__*/ (0, $1dtfx$react.forwardRef)($14c8378f774599cc$var$SvgComponent);
var $14c8378f774599cc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($14c8378f774599cc$var$ForwardRef);
