// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_bell_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $3YDW5$jsxs, jsx as $3YDW5$jsx} from "react/jsx-runtime";
import {forwardRef as $3YDW5$forwardRef} from "react";





const $574af2a85ded0d2e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3YDW5$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3YDW5$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3YDW5$jsxs)("linearGradient", {
                        id: "7e4da7__d",
                        x1: -14.409,
                        x2: 120.646,
                        y1: 80.7,
                        y2: -1.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3YDW5$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $3YDW5$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3YDW5$jsxs)("linearGradient", {
                        id: "7e4da7__e",
                        x1: 93.971,
                        x2: 36.033,
                        y1: 21.766,
                        y2: 84.066,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3YDW5$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3YDW5$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3YDW5$jsxs)("linearGradient", {
                        id: "7e4da7__f",
                        x1: 23.561,
                        x2: 126.271,
                        y1: 92.664,
                        y2: 35.128,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3YDW5$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3YDW5$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3YDW5$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3YDW5$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3YDW5$jsxs)("radialGradient", {
                        id: "7e4da7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 21.875 31.562)scale(36.1398)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3YDW5$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $3YDW5$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3YDW5$jsxs)("radialGradient", {
                        id: "7e4da7__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 31.611 53.083)scale(36.1398)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3YDW5$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3YDW5$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3YDW5$jsx)("clipPath", {
                        id: "7e4da7__b",
                        children: /*#__PURE__*/ (0, $3YDW5$jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3YDW5$jsx)("clipPath", {
                id: "7e4da7__a",
                children: /*#__PURE__*/ (0, $3YDW5$jsx)("path", {
                    fill: "#fff",
                    d: "M73.772 61.732c-3.175-4.992-6.448-11.96-6.692-19.72v-6.053h-.002l.002-.079c0-10.537-8.543-19.08-19.08-19.08-10.539 0-19.081 8.543-19.081 19.08l.002.078h-.002v6.054c-.244 7.76-3.517 14.728-6.693 19.72-2.169 3.41.258 7.868 4.3 7.868h14.273V72c0 3.976 3.224 7.2 7.2 7.2s7.2-3.224 7.2-7.2v-2.4h14.274c4.041 0 6.469-4.458 4.3-7.868"
                })
            }),
            /*#__PURE__*/ (0, $3YDW5$jsx)("g", {
                clipPath: "url(#7e4da7__a)",
                children: /*#__PURE__*/ (0, $3YDW5$jsxs)("g", {
                    clipPath: "url(#7e4da7__b)",
                    children: [
                        /*#__PURE__*/ (0, $3YDW5$jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $3YDW5$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6.465 6.465h83.07v83.07H6.465z"
                        }),
                        /*#__PURE__*/ (0, $3YDW5$jsx)("circle", {
                            cx: 53.436,
                            cy: 9.687,
                            r: 36.14,
                            fill: "url(#7e4da7__c)",
                            transform: "rotate(-.08 53.436 9.687)"
                        }),
                        /*#__PURE__*/ (0, $3YDW5$jsx)("path", {
                            fill: "url(#7e4da7__d)",
                            d: "M38.173-8.209c-6.38-3.947-14.463-3.972-20.866-.062L-46.74 30.833c-6.272 3.83-6.283 12.87-.019 16.714l63.623 39.048c6.393 3.924 14.477 3.92 20.865-.014l63.491-39.085c6.231-3.836 6.242-12.825.019-16.676z"
                        }),
                        /*#__PURE__*/ (0, $3YDW5$jsx)("path", {
                            fill: "url(#7e4da7__e)",
                            d: "M18.437 45.905c1.42-31.013 27.711-55.002 58.724-53.582s55.001 27.711 53.582 58.723c-1.42 31.013-27.712 55.002-58.724 53.582-31.012-1.419-55.002-27.71-53.582-58.723"
                        }),
                        /*#__PURE__*/ (0, $3YDW5$jsx)("path", {
                            fill: "url(#7e4da7__f)",
                            d: "M-27.19 66.352c12.328-5.259 25.623-5.033 37.142-.433C33 75.146 46.412 69.44 55.714 46.445c4.633-11.539 13.679-21.312 26.02-26.576 24.675-10.51 53.161.893 63.656 25.495s-.992 53.043-25.661 63.567c-12.369 5.276-25.69 5.029-37.226.388-23.011-9.178-36.397-3.452-45.687 19.572-4.645 11.512-13.689 21.251-26.016 26.51-24.655 10.518-53.141-.886-63.63-25.474s.986-53.057 25.64-63.575"
                        }),
                        /*#__PURE__*/ (0, $3YDW5$jsx)("circle", {
                            cx: 84.694,
                            cy: 21.472,
                            r: 36.14,
                            fill: "url(#7e4da7__g)",
                            transform: "rotate(-.08 84.694 21.472)"
                        })
                    ]
                })
            })
        ]
    });
const $574af2a85ded0d2e$var$ForwardRef = /*#__PURE__*/ (0, $3YDW5$forwardRef)($574af2a85ded0d2e$var$SvgComponent);
var $574af2a85ded0d2e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($574af2a85ded0d2e$var$ForwardRef);


export {$574af2a85ded0d2e$export$2e2bcd8739ae039 as default};
