// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_clipboard_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $2MlYe$reactjsxruntime = require("react/jsx-runtime");
var $2MlYe$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2e858587ec46f18d$export$2e2bcd8739ae039);




const $2e858587ec46f18d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8f7e16__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8f7e16__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8f7e16__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8f7e16__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8f7e16__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("clipPath", {
                        id: "8f7e16__b",
                        children: /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("clipPath", {
                id: "8f7e16__a",
                children: /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M32.999 24h-5.155c-4.112 0-7.446 3.37-7.446 7.527v40.146c0 4.157 3.334 7.527 7.446 7.527h39.709c4.112 0 7.445-3.37 7.445-7.528V31.527c0-4.157-3.333-7.527-7.445-7.527h-5.754v.017l-.012 9.6c0 .637-.254 1.247-.705 1.696-.45.45-1.061.702-1.698.701l-24-.029c-1.325-.001-2.398-1.077-2.397-2.402zm4.791 7.188.006-4.8 2.4.003c1.325.002 2.401-1.071 2.403-2.397l.002-2.4c.004-2.65 2.155-4.797 4.806-4.794s4.798 2.155 4.794 4.805l-.002 2.4c-.002 1.326 1.071 2.402 2.397 2.403l2.4.003-.006 4.8z"
                })
            }),
            /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("g", {
                clipPath: "url(#8f7e16__a)",
                children: /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#8f7e16__b)",
                    children: [
                        /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#8f7e16__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("path", {
                            fill: "url(#8f7e16__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("path", {
                            fill: "url(#8f7e16__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("path", {
                            fill: "url(#8f7e16__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $2MlYe$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#8f7e16__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $2e858587ec46f18d$var$ForwardRef = /*#__PURE__*/ (0, $2MlYe$react.forwardRef)($2e858587ec46f18d$var$SvgComponent);
var $2e858587ec46f18d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2e858587ec46f18d$var$ForwardRef);
