// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_handshake_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7Dn9i$jsxs, jsx as $7Dn9i$jsx} from "react/jsx-runtime";
import {forwardRef as $7Dn9i$forwardRef} from "react";





const $1379b8340e50b909$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7Dn9i$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7Dn9i$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7Dn9i$jsxs)("linearGradient", {
                        id: "23af9f__d",
                        x1: -19.921,
                        x2: 186.087,
                        y1: 131.011,
                        y2: -1.943,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Dn9i$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $7Dn9i$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Dn9i$jsxs)("linearGradient", {
                        id: "23af9f__e",
                        x1: 150.995,
                        x2: 65.38,
                        y1: 43.401,
                        y2: 141.06,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Dn9i$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7Dn9i$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Dn9i$jsxs)("linearGradient", {
                        id: "23af9f__f",
                        x1: 39.959,
                        x2: 197.223,
                        y1: 148.798,
                        y2: 55.343,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Dn9i$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7Dn9i$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7Dn9i$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7Dn9i$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Dn9i$jsxs)("radialGradient", {
                        id: "23af9f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Dn9i$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $7Dn9i$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Dn9i$jsxs)("radialGradient", {
                        id: "23af9f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Dn9i$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7Dn9i$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Dn9i$jsx)("clipPath", {
                        id: "23af9f__b",
                        children: /*#__PURE__*/ (0, $7Dn9i$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7Dn9i$jsx)("clipPath", {
                id: "23af9f__a",
                children: /*#__PURE__*/ (0, $7Dn9i$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m61.124 39.715 5.655 5.655 5.78 5.78 11.3 11.3c3.66 3.66 9.59 3.66 13.25 0s3.66-9.59 0-13.25l-11.3-11.3-5.78-5.78-1.88-1.88c-1.34-1.32-2.85-2.34-4.46-3.09l-.154-.074c-.18-.088-.36-.176-.546-.246-.63-.26-1.28-.46-1.93-.64-5.34-1.42-11.26-.05-15.45 4.13l-30.43 30.43c-4.89 4.89-5.93 12.15-3.17 18.08l33.46-33.46zm43.263-9.315 30.43 30.43v.02c6.25 6.25 6.25 16.38 0 22.63l-33.88 33.88-14.28 14.28c-3.66 3.66-9.59 3.66-13.25-.01h-.01c-3.66-3.66-3.66-9.59 0-13.25-3.66 3.66-9.59 3.66-13.25-.01h-.01c-3.63-3.63-3.65-9.49-.08-13.16-3.67 3.57-9.53 3.54-13.16-.1h-.01c-3.66-3.66-3.66-9.6 0-13.26h-.01c-3.66 3.66-9.59 3.66-13.25-.01h-.01c-.29-.29-.55-.6-.79-.91-2.84-3.68-2.58-8.98.79-12.35l14.96-14.97 12.56-12.56 5.78 5.78 11.3 11.3c3.28 3.28 7.64 5.09 12.28 5.09s9-1.81 12.28-5.09c6.77-6.78 6.77-17.8 0-24.57l-16.37-16.37c5.9-2.71 13.11-1.66 17.98 3.21"
                })
            }),
            /*#__PURE__*/ (0, $7Dn9i$jsx)("g", {
                clipPath: "url(#23af9f__a)",
                children: /*#__PURE__*/ (0, $7Dn9i$jsxs)("g", {
                    clipPath: "url(#23af9f__b)",
                    children: [
                        /*#__PURE__*/ (0, $7Dn9i$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $7Dn9i$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M13 20.654h131v123.491H13z"
                        }),
                        /*#__PURE__*/ (0, $7Dn9i$jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#23af9f__c)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 30 -28.206)"
                        }),
                        /*#__PURE__*/ (0, $7Dn9i$jsx)("path", {
                            fill: "url(#23af9f__d)",
                            d: "M63.002-1.16C52.94-7.027 40.194-7.063 30.096-1.251l-101 58.132c-9.891 5.693-9.907 19.132-.03 24.847L29.4 139.776c10.082 5.833 22.83 5.826 32.904-.021l100.125-58.103c9.826-5.703 9.844-19.067.03-24.79z"
                        }),
                        /*#__PURE__*/ (0, $7Dn9i$jsx)("path", {
                            fill: "url(#23af9f__e)",
                            d: "M31.878 79.285C34.118 33.183 75.58-2.48 124.485-.369c48.907 2.11 86.738 41.195 84.499 87.298s-43.701 81.765-92.607 79.655c-48.906-2.111-86.738-41.196-84.499-87.299"
                        }),
                        /*#__PURE__*/ (0, $7Dn9i$jsx)("path", {
                            fill: "url(#23af9f__f)",
                            d: "M-40.075 109.683c19.44-7.818 40.406-7.483 58.57-.645 36.349 13.718 57.498 5.237 72.167-28.949 7.307-17.154 21.572-31.682 41.035-39.509 38.911-15.624 83.833 1.33 100.384 37.902s-1.565 78.853-40.467 94.497c-19.506 7.845-40.512 7.477-58.705.577-36.288-13.644-57.398-5.13-72.048 29.096-7.325 17.113-21.588 31.592-41.028 39.41-38.88 15.636-83.803-1.318-100.344-37.87-16.542-36.552 1.555-78.873 40.436-94.509"
                        }),
                        /*#__PURE__*/ (0, $7Dn9i$jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#23af9f__g)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 79.293 -10.685)"
                        })
                    ]
                })
            })
        ]
    });
const $1379b8340e50b909$var$ForwardRef = /*#__PURE__*/ (0, $7Dn9i$forwardRef)($1379b8340e50b909$var$SvgComponent);
var $1379b8340e50b909$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1379b8340e50b909$var$ForwardRef);


export {$1379b8340e50b909$export$2e2bcd8739ae039 as default};
