// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_information_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $aacYo$reactjsxruntime = require("react/jsx-runtime");
var $aacYo$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $025e4ece6f8504ed$export$2e2bcd8739ae039);




const $025e4ece6f8504ed$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsxs)("linearGradient", {
                        id: "dcbd9a__d",
                        x1: -13.164,
                        x2: 184.35,
                        y1: 126.007,
                        y2: -1.466,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsxs)("linearGradient", {
                        id: "dcbd9a__e",
                        x1: 150.707,
                        x2: 68.621,
                        y1: 42.009,
                        y2: 135.642,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsxs)("linearGradient", {
                        id: "dcbd9a__f",
                        x1: 44.245,
                        x2: 195.026,
                        y1: 143.062,
                        y2: 53.46,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsxs)("radialGradient", {
                        id: "dcbd9a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.5105 -54.6429 0 54.643 51.51)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsxs)("radialGradient", {
                        id: "dcbd9a__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.5105 -54.6429 0 54.643 51.51)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("clipPath", {
                        id: "dcbd9a__b",
                        children: /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 20.2,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("clipPath", {
                id: "dcbd9a__a",
                children: /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M136 80c0 30.928-25.072 56-56 56s-56-25.072-56-56 25.072-56 56-56 56 25.072 56 56M72.5 66.5c-2.209 0-4 1.79-4 4s1.791 4 4 4h5v23h-5c-2.209 0-4 1.79-4 4s1.791 4 4 4h18c2.209 0 4-1.791 4-4s-1.791-4-4-4h-5v-27c0-2.21-1.791-4-4-4zm8-5c2.761 0 5-2.239 5-5s-2.239-5-5-5-5 2.239-5 5 2.239 5 5 5"
                })
            }),
            /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("g", {
                clipPath: "url(#dcbd9a__a)",
                children: /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#dcbd9a__b)",
                    children: [
                        /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 20.2,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M18.398 20.2h125.6v118.4h-125.6z"
                        }),
                        /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("ellipse", {
                            cx: 54.643,
                            cy: 51.511,
                            fill: "url(#dcbd9a__c)",
                            rx: 54.643,
                            ry: 51.511,
                            transform: "matrix(1 -.00132 .00148 1 34.695 -26.645)"
                        }),
                        /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("path", {
                            fill: "url(#dcbd9a__d)",
                            d: "M66.34-.715C56.692-6.34 44.471-6.376 34.79-.804l-96.836 55.736c-9.483 5.458-9.5 18.343-.028 23.822l96.195 55.657c9.668 5.592 21.89 5.585 31.548-.02l95.998-55.709c9.421-5.467 9.438-18.28.029-23.768z"
                        }),
                        /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("path", {
                            fill: "url(#dcbd9a__e)",
                            d: "M36.5 76.414C38.648 32.212 78.4-1.981 125.29.043c46.89 2.023 83.162 39.497 81.015 83.7-2.147 44.201-41.899 78.394-88.789 76.37s-83.162-39.497-81.015-83.7"
                        }),
                        /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("path", {
                            fill: "url(#dcbd9a__f)",
                            d: "M-32.49 105.559c18.64-7.495 38.741-7.174 56.157-.618 34.85 13.152 55.127 5.021 69.191-27.755 7.006-16.448 20.683-30.377 39.343-37.88 37.308-14.981 80.378 1.274 96.246 36.339 15.869 35.065-1.5 75.602-38.799 90.601-18.701 7.521-38.842 7.169-56.284.554-34.793-13.082-55.032-4.92-69.079 27.896-7.023 16.408-20.697 30.289-39.336 37.785-37.278 14.991-80.348-1.263-96.208-36.309s1.492-75.621 38.77-90.613"
                        }),
                        /*#__PURE__*/ (0, $aacYo$reactjsxruntime.jsx)("ellipse", {
                            cx: 54.643,
                            cy: 51.511,
                            fill: "url(#dcbd9a__g)",
                            rx: 54.643,
                            ry: 51.511,
                            transform: "matrix(1 -.00132 .00148 1 81.957 -9.848)"
                        })
                    ]
                })
            })
        ]
    });
const $025e4ece6f8504ed$var$ForwardRef = /*#__PURE__*/ (0, $aacYo$react.forwardRef)($025e4ece6f8504ed$var$SvgComponent);
var $025e4ece6f8504ed$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($025e4ece6f8504ed$var$ForwardRef);
