// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_tablet_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4wujl$reactjsxruntime = require("react/jsx-runtime");
var $4wujl$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e08754e553f2cc0f$export$2e2bcd8739ae039);




const $e08754e553f2cc0f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f74860__d",
                        x1: -15.112,
                        x2: 116.984,
                        y1: 80.61,
                        y2: -4.643,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f74860__e",
                        x1: 94.487,
                        x2: 39.588,
                        y1: 24.434,
                        y2: 87.054,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f74860__f",
                        x1: 23.289,
                        x2: 124.13,
                        y1: 92.016,
                        y2: 32.091,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f74860__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.4497 -36.5446 0 36.545 34.45)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f74860__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.4497 -36.5446 0 36.545 34.45)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("clipPath", {
                        id: "f74860__b",
                        children: /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("rect", {
                            width: 84,
                            height: 79.185,
                            x: 6,
                            y: 9.848,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("clipPath", {
                id: "f74860__a",
                children: /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M76.802 16.8h-57.6c-5.304 0-9.6 4.296-9.6 9.6v43.2c0 5.304 4.296 9.6 9.6 9.6h57.6c5.304 0 9.6-4.296 9.6-9.6V26.4c0-5.304-4.296-9.6-9.6-9.6m-56.4 33.6c-1.326 0-2.4-1.074-2.4-2.4s1.068-2.4 2.394-2.4h.006c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4"
                })
            }),
            /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("g", {
                clipPath: "url(#f74860__a)",
                children: /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#f74860__b)",
                    children: [
                        /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("rect", {
                            width: 84,
                            height: 79.185,
                            x: 6,
                            y: 9.848,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6 9.848h84v79.185H6z"
                        }),
                        /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("ellipse", {
                            cx: 36.545,
                            cy: 34.45,
                            fill: "url(#f74860__c)",
                            rx: 36.545,
                            ry: 34.45,
                            transform: "matrix(1 -.00132 .00148 1 16.898 -21.482)"
                        }),
                        /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("path", {
                            fill: "url(#f74860__d)",
                            d: "M38.06-4.14c-6.452-3.763-14.625-3.786-21.1-.06l-64.763 37.276c-6.343 3.65-6.353 12.267-.02 15.932L16.513 86.23c6.466 3.74 14.64 3.736 21.1-.013l64.202-37.257c6.3-3.657 6.312-12.226.019-15.896z"
                        }),
                        /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("path", {
                            fill: "url(#f74860__e)",
                            d: "M18.107 47.443c1.435-29.562 28.021-52.43 59.38-51.076s55.619 26.415 54.183 55.977-28.022 52.43-59.381 51.077S16.67 77.005 18.106 47.443"
                        }),
                        /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("path", {
                            fill: "url(#f74860__f)",
                            d: "M-28.03 66.935c12.465-5.013 25.91-4.798 37.557-.413 23.306 8.796 36.868 3.357 46.274-18.563 4.685-11 13.833-20.315 26.312-25.334 24.951-10.018 53.756.852 64.369 24.303 10.613 23.452-1.004 50.562-25.949 60.594-12.507 5.03-25.977 4.794-37.642.37-23.27-8.749-36.805-3.29-46.2 18.657-4.696 10.973-13.842 20.257-26.307 25.27-24.931 10.026-53.736-.845-64.343-24.283s.998-50.575 25.929-60.6"
                        }),
                        /*#__PURE__*/ (0, $4wujl$reactjsxruntime.jsx)("ellipse", {
                            cx: 36.545,
                            cy: 34.45,
                            fill: "url(#f74860__g)",
                            rx: 36.545,
                            ry: 34.45,
                            transform: "matrix(1 -.00132 .00148 1 48.508 -10.248)"
                        })
                    ]
                })
            })
        ]
    });
const $e08754e553f2cc0f$var$ForwardRef = /*#__PURE__*/ (0, $4wujl$react.forwardRef)($e08754e553f2cc0f$var$SvgComponent);
var $e08754e553f2cc0f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e08754e553f2cc0f$var$ForwardRef);
