// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_search_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6P1E0$jsxs, jsx as $6P1E0$jsx} from "react/jsx-runtime";
import {forwardRef as $6P1E0$forwardRef} from "react";





const $fbaeba4942c00fa6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6P1E0$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6P1E0$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6P1E0$jsxs)("linearGradient", {
                        id: "d098c2__d",
                        x1: -14.205,
                        x2: 112.23,
                        y1: 76.9,
                        y2: -4.7,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6P1E0$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $6P1E0$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6P1E0$jsxs)("linearGradient", {
                        id: "d098c2__e",
                        x1: 90.695,
                        x2: 38.15,
                        y1: 23.13,
                        y2: 83.067,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6P1E0$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6P1E0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6P1E0$jsxs)("linearGradient", {
                        id: "d098c2__f",
                        x1: 22.545,
                        x2: 119.065,
                        y1: 87.816,
                        y2: 30.459,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6P1E0$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6P1E0$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6P1E0$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6P1E0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6P1E0$jsxs)("radialGradient", {
                        id: "d098c2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.9734 -34.9785 0 34.978 32.973)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6P1E0$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $6P1E0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6P1E0$jsxs)("radialGradient", {
                        id: "d098c2__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.9734 -34.9785 0 34.978 32.973)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6P1E0$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6P1E0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6P1E0$jsx)("clipPath", {
                        id: "d098c2__b",
                        children: /*#__PURE__*/ (0, $6P1E0$jsx)("rect", {
                            width: 80.4,
                            height: 75.791,
                            x: 6,
                            y: 9.169,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6P1E0$jsx)("clipPath", {
                id: "d098c2__a",
                children: /*#__PURE__*/ (0, $6P1E0$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M14.402 38.146c0-12.427 10.073-22.5 22.5-22.5s22.5 10.073 22.5 22.5-10.074 22.5-22.5 22.5-22.5-10.074-22.5-22.5m22.5-27.3c-15.078 0-27.3 12.222-27.3 27.3 0 15.077 12.222 27.3 27.3 27.3 6.674 0 12.79-2.396 17.532-6.373l4.087 4.086-1.382 1.381 17.705 17.705c1.685 1.685 4.419 1.685 6.105 0 1.685-1.686 1.685-4.42 0-6.105L63.244 58.435l-1.33 1.33-4.086-4.087c3.978-4.743 6.374-10.858 6.374-17.532 0-15.078-12.223-27.3-27.3-27.3"
                })
            }),
            /*#__PURE__*/ (0, $6P1E0$jsx)("g", {
                clipPath: "url(#d098c2__a)",
                children: /*#__PURE__*/ (0, $6P1E0$jsxs)("g", {
                    clipPath: "url(#d098c2__b)",
                    children: [
                        /*#__PURE__*/ (0, $6P1E0$jsx)("rect", {
                            width: 80.4,
                            height: 75.791,
                            x: 6,
                            y: 9.169,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $6P1E0$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6 9.169h80.4v75.79H6z"
                        }),
                        /*#__PURE__*/ (0, $6P1E0$jsx)("ellipse", {
                            cx: 34.978,
                            cy: 32.973,
                            fill: "url(#d098c2__c)",
                            rx: 34.978,
                            ry: 32.973,
                            transform: "matrix(1 -.00132 .00148 1 16.434 -20.818)"
                        }),
                        /*#__PURE__*/ (0, $6P1E0$jsx)("path", {
                            fill: "url(#d098c2__d)",
                            d: "M36.687-4.219C30.513-7.82 22.69-7.843 16.492-4.276l-61.987 35.678c-6.071 3.495-6.081 11.742-.019 15.25l61.578 35.627c6.188 3.58 14.012 3.576 20.194-.013l61.451-35.66c6.031-3.5 6.042-11.702.019-15.215z"
                        }),
                        /*#__PURE__*/ (0, $6P1E0$jsx)("path", {
                            fill: "url(#d098c2__e)",
                            d: "M17.588 45.153C18.962 16.858 44.41-5.03 74.425-3.734s53.234 25.283 51.86 53.578-26.82 50.183-56.836 48.888-53.235-25.284-51.86-53.579"
                        }),
                        /*#__PURE__*/ (0, $6P1E0$jsx)("path", {
                            fill: "url(#d098c2__f)",
                            d: "M-26.574 63.81c11.93-4.798 24.799-4.593 35.947-.396 22.308 8.42 35.288 3.214 44.291-17.767 4.485-10.528 13.24-19.445 25.185-24.248 23.882-9.59 51.452.815 61.61 23.261 10.158 22.447-.96 48.396-24.836 57.997-11.972 4.815-24.864 4.589-36.03.355-22.272-8.374-35.227-3.15-44.219 17.857-4.496 10.503-13.25 19.389-25.18 24.187-23.863 9.596-51.433-.808-61.586-23.242s.955-48.408 24.818-58.004"
                        }),
                        /*#__PURE__*/ (0, $6P1E0$jsx)("ellipse", {
                            cx: 34.978,
                            cy: 32.973,
                            fill: "url(#d098c2__g)",
                            rx: 34.978,
                            ry: 32.973,
                            transform: "matrix(1 -.00132 .00148 1 46.688 -10.065)"
                        })
                    ]
                })
            })
        ]
    });
const $fbaeba4942c00fa6$var$ForwardRef = /*#__PURE__*/ (0, $6P1E0$forwardRef)($fbaeba4942c00fa6$var$SvgComponent);
var $fbaeba4942c00fa6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($fbaeba4942c00fa6$var$ForwardRef);


export {$fbaeba4942c00fa6$export$2e2bcd8739ae039 as default};
