// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_threeArrowsCircle_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $giIR5$reactjsxruntime = require("react/jsx-runtime");
var $giIR5$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6235b589d7d7c3f8$export$2e2bcd8739ae039);




const $6235b589d7d7c3f8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2d8640__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.788,
                        y2: -9.168,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2d8640__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.682,
                        y2: 89.679,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2d8640__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.616,
                        y2: 33.124,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2d8640__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.28 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2d8640__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.532 53.874)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("clipPath", {
                        id: "2d8640__b",
                        children: /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: -0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("clipPath", {
                id: "2d8640__a",
                children: /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M72.808 23.01c.254 1.075-.412 2.153-1.487 2.407L57.558 28.67c-1.075.254-2.152-.411-2.406-1.486s.411-2.153 1.486-2.407l9.17-2.167c-5.06-3.591-11.241-5.701-17.92-5.701-9.298 0-17.638 4.09-23.323 10.578-.728.83-1.992.914-2.822.186s-.914-1.992-.186-2.823C27.969 17.533 37.39 12.91 47.888 12.91c7.384 0 14.238 2.288 19.886 6.194l-2.112-8.935c-.255-1.075.41-2.153 1.486-2.407 1.075-.254 2.152.411 2.406 1.486zm4.827 15.674c-.33-1.055.258-2.177 1.312-2.506 1.054-.33 2.176.257 2.506 1.311 1.034 3.307 1.591 6.822 1.591 10.463 0 14.285-8.556 26.567-20.818 32.007l8.776 2.508c1.063.303 1.678 1.41 1.374 2.472s-1.41 1.677-2.472 1.374l-13.598-3.885c-1.062-.303-1.677-1.41-1.374-2.472l3.885-13.598c.303-1.062 1.41-1.678 2.472-1.374s1.677 1.41 1.374 2.472l-2.593 9.078c11.15-4.697 18.974-15.728 18.974-28.582 0-3.232-.494-6.344-1.409-9.268M13.324 43.23l-6.932 5.825c-.846.71-2.107.6-2.818-.245-.71-.846-.6-2.107.245-2.818l10.827-9.097c.846-.71 2.108-.601 2.818.244l9.098 10.828c.71.845.6 2.107-.245 2.818-.846.71-2.107.6-2.818-.245l-6.134-7.301c-.932 6.025-.1 12.383 2.773 18.282 4.071 8.36 11.4 14.066 19.722 16.337 1.066.29 1.694 1.39 1.403 2.456-.29 1.065-1.39 1.693-2.456 1.403-9.386-2.562-17.668-9.006-22.265-18.445-3.147-6.46-4.125-13.418-3.218-20.042"
                })
            }),
            /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("g", {
                clipPath: "url(#2d8640__a)",
                children: /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#2d8640__b)",
                    children: [
                        /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: -0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0-.001h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.722,
                            r: 41.765,
                            fill: "url(#2d8640__c)",
                            transform: "rotate(-.08 54.28 3.722)"
                        }),
                        /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("path", {
                            fill: "url(#2d8640__d)",
                            d: "M36.643-16.96c-7.373-4.561-16.714-4.59-24.114-.071l-74.015 45.19c-7.249 4.427-7.26 14.874-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.432 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("path", {
                            fill: "url(#2d8640__e)",
                            d: "M13.837 45.578C15.477 9.738 45.86-17.985 81.7-16.345s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("path", {
                            fill: "url(#2d8640__f)",
                            d: "M-38.895 69.208c14.246-6.077 29.61-5.817 42.922-.5 26.636 10.663 42.135 4.07 52.885-22.505 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $giIR5$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.343,
                            r: 41.765,
                            fill: "url(#2d8640__g)",
                            transform: "rotate(-.08 90.406 17.343)"
                        })
                    ]
                })
            })
        ]
    });
const $6235b589d7d7c3f8$var$ForwardRef = /*#__PURE__*/ (0, $giIR5$react.forwardRef)($6235b589d7d7c3f8$var$SvgComponent);
var $6235b589d7d7c3f8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6235b589d7d7c3f8$var$ForwardRef);
