// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_AIchat_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $g5DQc$jsxs, jsx as $g5DQc$jsx} from "react/jsx-runtime";
import {forwardRef as $g5DQc$forwardRef} from "react";





const $7bc49841ac291555$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $g5DQc$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $g5DQc$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $g5DQc$jsxs)("linearGradient", {
                        id: "7650e2__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g5DQc$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $g5DQc$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g5DQc$jsxs)("linearGradient", {
                        id: "7650e2__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.684,
                        y2: 89.681,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g5DQc$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $g5DQc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g5DQc$jsxs)("linearGradient", {
                        id: "7650e2__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.618,
                        y2: 33.126,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g5DQc$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $g5DQc$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $g5DQc$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $g5DQc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g5DQc$jsxs)("radialGradient", {
                        id: "7650e2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.278 29.003)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g5DQc$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $g5DQc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g5DQc$jsxs)("radialGradient", {
                        id: "7650e2__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.53 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g5DQc$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $g5DQc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g5DQc$jsx)("clipPath", {
                        id: "7650e2__b",
                        children: /*#__PURE__*/ (0, $g5DQc$jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: 0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $g5DQc$jsx)("clipPath", {
                id: "7650e2__a",
                children: /*#__PURE__*/ (0, $g5DQc$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M61.817 36.974q.015-.105.038-.21l1.229-5.7q.045-.213.058-.427c.04-.71-.197-1.415-.666-1.946l-3.794-4.296c-.36-.407-.566-.865-.645-1.329-.29-1.709 1.156-3.49 3.064-3.043l5.463 1.282c.811.191 1.663-.046 2.274-.635l4.117-3.959c1.828-1.758 4.735-.006 4.19 2.525l-1.229 5.7-.014.072q-.008.043-.015.088-.023.142-.03.284c-.035.704.202 1.402.667 1.93l3.794 4.295c1.647 1.864.081 4.804-2.255 4.406q-.083-.014-.164-.033l-5.463-1.282c-.811-.192-1.663.046-2.274.634l-4.117 3.959c-1.777 1.71-4.574.101-4.228-2.315m10.526 1.808 4.643 1.09q.55.128 1.076.164v12.936c0 3.885-3.169 6.964-7.079 6.964H50.847L35.09 73.412c-1.534 1.322-3.92.23-3.906-1.788.027-3.618.014-8.369 0-11.688h-6.03c-3.91 0-7.079-3.079-7.079-6.964V30.03c0-3.886 3.17-6.965 7.08-6.965h28.86c.088 1.372.602 2.768 1.67 3.977l3.355 3.8-1.095 5.08c-.658 3.058.81 5.788 3.052 7.138 2.269 1.367 5.468 1.375 7.821-.889h.001zM53.98 50.273l-4.015-.904c-.532-.12-1.083.03-1.483.396l-3.025 2.79c-1.192 1.1-3.087.004-2.73-1.575l.904-4.015c.12-.532-.03-1.083-.396-1.483l-2.79-3.025c-1.1-1.192-.004-3.087 1.575-2.73l4.015.904c.532.12 1.083-.03 1.483-.396l3.025-2.79c1.192-1.1 3.087-.004 2.73 1.575l-.904 4.015c-.12.532.03 1.083.396 1.483l2.79 3.025c1.1 1.192.004 3.087-1.575 2.73"
                })
            }),
            /*#__PURE__*/ (0, $g5DQc$jsx)("g", {
                clipPath: "url(#7650e2__a)",
                children: /*#__PURE__*/ (0, $g5DQc$jsxs)("g", {
                    clipPath: "url(#7650e2__b)",
                    children: [
                        /*#__PURE__*/ (0, $g5DQc$jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: 0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $g5DQc$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 .001h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $g5DQc$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.724,
                            r: 41.765,
                            fill: "url(#7650e2__c)",
                            transform: "rotate(-.08 54.28 3.724)"
                        }),
                        /*#__PURE__*/ (0, $g5DQc$jsx)("path", {
                            fill: "url(#7650e2__d)",
                            d: "M36.643-16.957c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.426-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.432 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $g5DQc$jsx)("path", {
                            fill: "url(#7650e2__e)",
                            d: "M13.837 45.58C15.477 9.74 45.86-17.983 81.7-16.343s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922S12.196 81.419 13.837 45.58"
                        }),
                        /*#__PURE__*/ (0, $g5DQc$jsx)("path", {
                            fill: "url(#7650e2__f)",
                            d: "M-38.895 69.21c14.246-6.077 29.61-5.817 42.922-.5 26.636 10.663 42.135 4.07 52.885-22.505 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.315 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $g5DQc$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.345,
                            r: 41.765,
                            fill: "url(#7650e2__g)",
                            transform: "rotate(-.08 90.406 17.345)"
                        })
                    ]
                })
            })
        ]
    });
const $7bc49841ac291555$var$ForwardRef = /*#__PURE__*/ (0, $g5DQc$forwardRef)($7bc49841ac291555$var$SvgComponent);
var $7bc49841ac291555$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7bc49841ac291555$var$ForwardRef);


export {$7bc49841ac291555$export$2e2bcd8739ae039 as default};
