// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_transform_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6ydh3$jsxs, jsx as $6ydh3$jsx} from "react/jsx-runtime";
import {forwardRef as $6ydh3$forwardRef} from "react";





const $514b4d718176a86e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6ydh3$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6ydh3$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6ydh3$jsxs)("linearGradient", {
                        id: "6ba99f__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6ydh3$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $6ydh3$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6ydh3$jsxs)("linearGradient", {
                        id: "6ba99f__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.684,
                        y2: 89.681,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6ydh3$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6ydh3$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6ydh3$jsxs)("linearGradient", {
                        id: "6ba99f__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.618,
                        y2: 33.126,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6ydh3$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6ydh3$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6ydh3$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $6ydh3$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6ydh3$jsxs)("radialGradient", {
                        id: "6ba99f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.278 29.003)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6ydh3$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $6ydh3$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6ydh3$jsxs)("radialGradient", {
                        id: "6ba99f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.53 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6ydh3$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6ydh3$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6ydh3$jsx)("clipPath", {
                        id: "6ba99f__b",
                        children: /*#__PURE__*/ (0, $6ydh3$jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: 0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6ydh3$jsx)("clipPath", {
                id: "6ba99f__a",
                children: /*#__PURE__*/ (0, $6ydh3$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M18.238 16.262c-1.104 0-2 .895-2 2v13.255c0 1.105.896 2 2 2s2-.895 2-2v-8.523l24.933 24.933L20.24 72.86v-8.375c0-1.104-.896-2-2-2s-2 .896-2 2v13.256c0 1.104.895 2 2 2h13.298c1.105 0 2-.896 2-2s-.895-2-2-2h-8.523L48 50.756 72.985 75.74h-8.523c-1.105 0-2 .895-2 2s.895 2 2 2H77.76c1.105 0 2-.896 2-2V64.485c0-1.104-.895-2-2-2s-2 .896-2 2v8.375L50.828 47.927 75.76 22.994v8.523c0 1.105.895 2 2 2s2-.895 2-2V18.262c0-1.105-.896-2-2-2H64.462c-1.104 0-2 .895-2 2s.896 2 2 2h8.374L48 45.099 23.162 20.262h8.374c1.104 0 2-.896 2-2s-.896-2-2-2z"
                })
            }),
            /*#__PURE__*/ (0, $6ydh3$jsx)("g", {
                clipPath: "url(#6ba99f__a)",
                children: /*#__PURE__*/ (0, $6ydh3$jsxs)("g", {
                    clipPath: "url(#6ba99f__b)",
                    children: [
                        /*#__PURE__*/ (0, $6ydh3$jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: 0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $6ydh3$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 .001h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $6ydh3$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.724,
                            r: 41.765,
                            fill: "url(#6ba99f__c)",
                            transform: "rotate(-.08 54.28 3.724)"
                        }),
                        /*#__PURE__*/ (0, $6ydh3$jsx)("path", {
                            fill: "url(#6ba99f__d)",
                            d: "M36.643-16.957c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.426-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.432 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $6ydh3$jsx)("path", {
                            fill: "url(#6ba99f__e)",
                            d: "M13.837 45.58C15.477 9.74 45.86-17.983 81.7-16.343s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922S12.196 81.419 13.837 45.58"
                        }),
                        /*#__PURE__*/ (0, $6ydh3$jsx)("path", {
                            fill: "url(#6ba99f__f)",
                            d: "M-38.895 69.21c14.246-6.077 29.61-5.817 42.922-.5 26.636 10.663 42.135 4.07 52.885-22.505 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.315 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $6ydh3$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.345,
                            r: 41.765,
                            fill: "url(#6ba99f__g)",
                            transform: "rotate(-.08 90.406 17.345)"
                        })
                    ]
                })
            })
        ]
    });
const $514b4d718176a86e$var$ForwardRef = /*#__PURE__*/ (0, $6ydh3$forwardRef)($514b4d718176a86e$var$SvgComponent);
var $514b4d718176a86e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($514b4d718176a86e$var$ForwardRef);


export {$514b4d718176a86e$export$2e2bcd8739ae039 as default};
