// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_userAvatar_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8dKsY$reactjsxruntime = require("react/jsx-runtime");
var $8dKsY$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a5035e6c49f05da2$export$2e2bcd8739ae039);




const $a5035e6c49f05da2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b88f7b__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b88f7b__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b88f7b__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b88f7b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b88f7b__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("clipPath", {
                        id: "b88f7b__b",
                        children: /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("clipPath", {
                id: "b88f7b__a",
                children: /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M68.816 73.348C63.138 77.97 55.892 80.74 48 80.74c-7.86 0-15.078-2.748-20.746-7.335.162-.164.298-.359.398-.58C31.17 65.051 39.012 59.608 48 59.608c8.914 0 16.701 5.358 20.26 13.025.133.287.326.528.556.714m2.932-2.694C67.505 61.803 58.431 55.609 48 55.609c-10.43 0-19.505 6.188-23.749 15.044C18.523 64.717 15 56.64 15 47.74c0-18.225 14.775-33 33-33s33 14.775 33 33c0 8.9-3.523 16.978-9.252 22.914m-33.834-31.9c0-6.13 4.634-10.879 10.088-10.879s10.087 4.75 10.087 10.878c0 6.13-4.633 10.878-10.087 10.878s-10.088-4.749-10.088-10.878m10.088-14.879c-7.898 0-14.088 6.782-14.088 14.878s6.19 14.878 14.088 14.878 14.087-6.782 14.087-14.878-6.19-14.878-14.087-14.878"
                })
            }),
            /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("g", {
                clipPath: "url(#b88f7b__a)",
                children: /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#b88f7b__b)",
                    children: [
                        /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#b88f7b__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("path", {
                            fill: "url(#b88f7b__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("path", {
                            fill: "url(#b88f7b__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("path", {
                            fill: "url(#b88f7b__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $8dKsY$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#b88f7b__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $a5035e6c49f05da2$var$ForwardRef = /*#__PURE__*/ (0, $8dKsY$react.forwardRef)($a5035e6c49f05da2$var$SvgComponent);
var $a5035e6c49f05da2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a5035e6c49f05da2$var$ForwardRef);
