// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_update_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $jK7e1$jsxs, jsx as $jK7e1$jsx} from "react/jsx-runtime";
import {forwardRef as $jK7e1$forwardRef} from "react";





const $eaba833c137e6329$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jK7e1$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jK7e1$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jK7e1$jsxs)("linearGradient", {
                        id: "b4606e__d",
                        x1: -25.276,
                        x2: 197.596,
                        y1: 127.391,
                        y2: -16.448,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jK7e1$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $jK7e1$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jK7e1$jsxs)("linearGradient", {
                        id: "b4606e__e",
                        x1: 159.631,
                        x2: 67.007,
                        y1: 32.609,
                        y2: 138.262,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jK7e1$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jK7e1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jK7e1$jsxs)("linearGradient", {
                        id: "b4606e__f",
                        x1: 39.505,
                        x2: 209.643,
                        y1: 146.634,
                        y2: 45.529,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jK7e1$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jK7e1$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jK7e1$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jK7e1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jK7e1$jsxs)("radialGradient", {
                        id: "b4606e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 58.1233 -61.6578 0 61.658 58.123)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jK7e1$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $jK7e1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jK7e1$jsxs)("radialGradient", {
                        id: "b4606e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 58.1233 -61.6578 0 61.658 58.123)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jK7e1$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jK7e1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jK7e1$jsx)("clipPath", {
                        id: "b4606e__b",
                        children: /*#__PURE__*/ (0, $jK7e1$jsx)("rect", {
                            width: 141.724,
                            height: 133.6,
                            x: 10.34,
                            y: 8,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jK7e1$jsxs)("clipPath", {
                id: "b4606e__a",
                children: [
                    /*#__PURE__*/ (0, $jK7e1$jsx)("path", {
                        fill: "#fff",
                        fillRule: "evenodd",
                        d: "M140.117 41.9c1-1.97 3.41-2.75 5.38-1.75h.01c1.97 1 2.75 3.41 1.75 5.38l-10.98 21.57c-.71 1.39-2.11 2.19-3.57 2.19-.57 0-1.14-.13-1.69-.38l-21.17-9.87c-2.01-.94-2.88-3.32-1.94-5.32s3.32-2.87 5.32-1.93l13.83 6.45c-8.49-18.05-26.99-29.96-47.11-29.96-22.97 0-43.5 15.38-49.93 37.4-.51 1.75-2.11 2.88-3.84 2.88-.37 0-.75-.05-1.12-.16-2.12-.62-3.34-2.84-2.72-4.96 7.42-25.41 31.11-43.16 57.61-43.16 22.93 0 44.03 13.4 53.98 33.79zm-10.319 52.99c.62-2.12 2.84-3.34 4.96-2.72v.01c2.12.62 3.34 2.84 2.72 4.96-7.42 25.41-31.11 43.16-57.61 43.16-22.82 0-43.83-13.28-53.84-33.51l-6.15 12.09c-.71 1.39-2.11 2.19-3.57 2.19-.61 0-1.23-.14-1.81-.44-1.97-1-2.75-3.41-1.75-5.38l10.98-21.58c.98-1.92 3.31-2.72 5.26-1.81l21.17 9.87c2.01.94 2.87 3.32 1.94 5.32-.94 2-3.32 2.86-5.32 1.93l-13.95-6.51c8.51 17.98 26.97 29.82 47.04 29.82 22.97 0 43.5-15.38 49.93-37.4"
                    }),
                    /*#__PURE__*/ (0, $jK7e1$jsx)("path", {
                        fill: "#fff",
                        d: "m145.497 40.15-.452.892.213.108h.239zm-5.38 1.75.892.454v-.001zm5.39-1.75.453-.891-.213-.109h-.24zm1.75 5.38.891.454.001-.001zm-10.98 21.57.891.455v-.001zm-5.26 1.81-.422.907.008.004zm-21.17-9.87-.423.906h.001zm3.38-7.25-.425.905.003.002zm13.83 6.45-.422.907 2.503 1.167-1.176-2.5zm-97.04 7.44-.96-.28zm-4.96 2.72.285-.958-.004-.001zm-2.72-4.96.96.281zm111.59-9.37-.898.439.88 1.803.91-1.788zm.831 38.1h1v-.75l-.719-.21zm-4.96 2.72-.96-.28zm4.96-2.71h-1v.75l.719.21zm2.72 4.96-.96-.28zm-111.45 9.65.896-.443-.884-1.787-.903 1.777zm-6.15 12.09.89.455.001-.001zm-5.38 1.75.46-.888-.007-.003zm-1.75-5.38-.891-.453v.001zm10.98-21.58-.89-.454h-.001zm5.26-1.81-.423.906zm21.17 9.87.424-.906h-.002zm1.94 5.32.905.426.002-.004zm-5.32 1.93-.423.906.001.001zm-13.95-6.51.423-.906-2.513-1.173 1.186 2.507zM145.95 39.259c-2.46-1.249-5.473-.277-6.724 2.189l1.783.905c.749-1.475 2.555-2.062 4.036-1.311zm-.443-.109h-.01v2h.01zm2.642 6.833c1.249-2.46.276-5.473-2.189-6.724l-.905 1.783c1.474.749 2.062 2.555 1.311 4.036zm-10.981 21.57 10.98-21.57-1.782-.906-10.98 21.57zm-4.461 2.737c1.824 0 3.575-1 4.461-2.735l-1.781-.91c-.534 1.045-1.583 1.645-2.68 1.645zm-2.104-.47c.67.305 1.38.47 2.104.47v-2c-.415 0-.845-.094-1.276-.29zm-21.178-9.873 21.17 9.87.845-1.813-21.17-9.87zm-2.423-6.652c-1.177 2.505-.084 5.48 2.422 6.651l.847-1.812c-1.514-.708-2.161-2.494-1.459-3.988zm6.651-2.41c-2.505-1.177-5.478-.085-6.651 2.41l1.81.85c.708-1.504 2.494-2.153 3.99-1.45zm13.827 6.449-13.83-6.45-.845 1.813 13.83 6.45zM79.947 29.28c19.729 0 37.879 11.683 46.205 29.386l1.81-.851c-8.653-18.397-27.503-30.535-48.015-30.535zm-48.97 36.68c6.305-21.592 26.444-36.68 48.97-36.68v-2c-23.413 0-44.335 15.673-50.89 38.12zm-4.8 3.6c2.16 0 4.162-1.41 4.8-3.6l-1.92-.56c-.381 1.31-1.58 2.16-2.88 2.16zm-1.405-.201c.47.14.946.201 1.405.201v-2c-.28 0-.564-.038-.835-.118zm-3.394-6.2c-.776 2.652.75 5.427 3.399 6.201l.561-1.92c-1.592-.465-2.505-2.13-2.04-3.719zm58.57-43.879c-26.944 0-51.026 18.043-58.57 43.88l1.92.56c7.295-24.982 30.593-42.44 56.65-42.44zm54.878 34.352C124.71 32.902 103.264 19.28 79.947 19.28v2c22.544 0 43.297 13.179 53.082 33.229zm4.4-12.185-6.19 12.17 1.783.907 6.19-12.17zm-4.187 49.763c-2.652-.775-5.426.751-6.201 3.4l1.92.561c.465-1.592 2.131-2.505 3.719-2.04zm.719.97v-.01h-2v.01zm2.68 5.241c.775-2.652-.751-5.426-3.399-6.2l-.562 1.92c1.592.465 2.506 2.13 2.041 3.719zM79.868 141.3c26.944 0 51.025-18.042 58.57-43.88l-1.92-.56c-7.295 24.983-30.594 42.44-56.65 42.44zm-54.736-34.066C35.308 127.8 56.662 141.3 79.868 141.3v-2c-22.435 0-43.1-13.06-52.944-32.953zm-4.363 12.1 6.15-12.09-1.782-.907-6.15 12.09zm-4.461 2.736c1.823 0 3.574-1 4.46-2.735l-1.78-.91c-.534 1.045-1.584 1.645-2.68 1.645zm-2.27-.552c.728.377 1.506.552 2.27.552v-2c-.456 0-.918-.104-1.35-.328zm-2.182-6.72c-1.248 2.459-.276 5.473 2.19 6.724l.905-1.783c-1.475-.749-2.063-2.556-1.311-4.036zm10.98-21.581-10.98 21.58 1.783.907 10.98-21.58zm6.575-2.263c-2.438-1.138-5.349-.138-6.574 2.262l1.782.909c.735-1.44 2.484-2.04 3.946-1.359zm21.17 9.87-21.17-9.87-.846 1.813 21.17 9.87zm2.424 6.648c1.162-2.5.087-5.474-2.423-6.648l-.848 1.812c1.51.706 2.155 2.492 1.457 3.993zm-6.649 2.415c2.499 1.162 5.472.088 6.647-2.411l-1.81-.851c-.706 1.501-2.492 2.147-3.993 1.449zm-13.95-6.511 13.949 6.51.846-1.812-13.95-6.51zm47.462 27.914c-19.68 0-37.79-11.614-46.136-29.248l-1.808.856c8.674 18.326 27.484 30.392 47.944 30.392zm48.97-36.68c-6.305 21.593-26.444 36.68-48.97 36.68v2c23.414 0 44.335-15.672 50.89-38.12z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jK7e1$jsx)("g", {
                clipPath: "url(#b4606e__a)",
                children: /*#__PURE__*/ (0, $jK7e1$jsxs)("g", {
                    clipPath: "url(#b4606e__b)",
                    children: [
                        /*#__PURE__*/ (0, $jK7e1$jsx)("rect", {
                            width: 141.724,
                            height: 133.6,
                            x: 10.34,
                            y: 8,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $jK7e1$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M10.34 8h141.724v133.6H10.34z"
                        }),
                        /*#__PURE__*/ (0, $jK7e1$jsx)("ellipse", {
                            cx: 61.658,
                            cy: 58.123,
                            fill: "url(#b4606e__c)",
                            rx: 61.658,
                            ry: 58.123,
                            transform: "matrix(1 -.00132 .00148 1 28.73 -44.86)"
                        }),
                        /*#__PURE__*/ (0, $jK7e1$jsx)("path", {
                            fill: "url(#b4606e__d)",
                            d: "M64.435-15.6c-10.885-6.349-24.675-6.388-35.6-.1L-80.432 47.19c-10.701 6.16-10.72 20.699-.032 26.882L28.08 136.873c10.908 6.31 24.699 6.303 35.598-.023L172 73.99c10.63-6.17 10.65-20.627.033-26.82z"
                        }),
                        /*#__PURE__*/ (0, $jK7e1$jsx)("path", {
                            fill: "url(#b4606e__e)",
                            d: "M30.763 71.43c2.422-49.876 47.278-88.458 100.188-86.175s93.838 44.568 91.415 94.445-47.277 88.459-100.187 86.175c-52.91-2.283-93.838-44.568-91.416-94.444"
                        }),
                        /*#__PURE__*/ (0, $jK7e1$jsx)("path", {
                            fill: "url(#b4606e__f)",
                            d: "M-47.08 104.317c21.03-8.458 43.713-8.095 63.365-.697 39.323 14.84 62.204 5.665 78.074-31.32 7.905-18.558 23.338-34.275 44.393-42.743 42.098-16.903 90.697 1.438 108.603 41.005s-1.693 85.308-43.78 102.233c-21.103 8.487-43.829 8.089-63.51.625-39.26-14.762-62.097-5.551-77.947 31.477-7.925 18.514-23.355 34.178-44.387 42.636-42.063 16.916-90.662-1.426-108.558-40.97-17.896-39.545 1.683-85.33 43.746-102.246"
                        }),
                        /*#__PURE__*/ (0, $jK7e1$jsx)("ellipse", {
                            cx: 61.658,
                            cy: 58.123,
                            fill: "url(#b4606e__g)",
                            rx: 61.658,
                            ry: 58.123,
                            transform: "matrix(1 -.00132 .00148 1 82.059 -25.905)"
                        })
                    ]
                })
            })
        ]
    });
const $eaba833c137e6329$var$ForwardRef = /*#__PURE__*/ (0, $jK7e1$forwardRef)($eaba833c137e6329$var$SvgComponent);
var $eaba833c137e6329$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($eaba833c137e6329$var$ForwardRef);


export {$eaba833c137e6329$export$2e2bcd8739ae039 as default};
