// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_desktop_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $1yYpH$jsxs, jsx as $1yYpH$jsx} from "react/jsx-runtime";
import {forwardRef as $1yYpH$forwardRef} from "react";





const $9b6b05811ed7bc5d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1yYpH$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1yYpH$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1yYpH$jsxs)("linearGradient", {
                        id: "d5bd05__d",
                        x1: -5.189,
                        x2: 56.167,
                        y1: 38.568,
                        y2: -1.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1yYpH$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $1yYpH$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1yYpH$jsxs)("linearGradient", {
                        id: "d5bd05__e",
                        x1: 45.718,
                        x2: 20.218,
                        y1: 12.476,
                        y2: 41.562,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1yYpH$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1yYpH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1yYpH$jsxs)("linearGradient", {
                        id: "d5bd05__f",
                        x1: 12.646,
                        x2: 59.485,
                        y1: 43.865,
                        y2: 16.031,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1yYpH$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1yYpH$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1yYpH$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1yYpH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1yYpH$jsxs)("radialGradient", {
                        id: "d5bd05__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.0013 -16.9744 0 16.974 16.001)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1yYpH$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $1yYpH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1yYpH$jsxs)("radialGradient", {
                        id: "d5bd05__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.0013 -16.9744 0 16.974 16.001)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1yYpH$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1yYpH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1yYpH$jsx)("clipPath", {
                        id: "d5bd05__b",
                        children: /*#__PURE__*/ (0, $1yYpH$jsx)("rect", {
                            width: 39.017,
                            height: 36.78,
                            x: 4.617,
                            y: 5.7,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1yYpH$jsx)("clipPath", {
                id: "d5bd05__a",
                children: /*#__PURE__*/ (0, $1yYpH$jsx)("path", {
                    fill: "#fff",
                    d: "M38.108 8.4H9.89c-2.652 0-4.8 2.148-4.8 4.8v13.2c0 2.652 2.148 4.8 4.8 4.8h10.809v7.2h-3.003c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h12.606c.663 0 1.2-.537 1.2-1.2s-.537-1.2-1.2-1.2h-3.003v-7.2h10.809c2.652 0 4.8-2.148 4.8-4.8V13.2c0-2.652-2.148-4.8-4.8-4.8m-14.103 6c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.197-1.2h.003c.663 0 1.2.537 1.2 1.2s-.537 1.2-1.2 1.2"
                })
            }),
            /*#__PURE__*/ (0, $1yYpH$jsx)("g", {
                clipPath: "url(#d5bd05__a)",
                children: /*#__PURE__*/ (0, $1yYpH$jsxs)("g", {
                    clipPath: "url(#d5bd05__b)",
                    children: [
                        /*#__PURE__*/ (0, $1yYpH$jsx)("rect", {
                            width: 39.017,
                            height: 36.78,
                            x: 4.617,
                            y: 5.7,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $1yYpH$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M4.617 5.7h39.017v36.78H4.617z"
                        }),
                        /*#__PURE__*/ (0, $1yYpH$jsx)("ellipse", {
                            cx: 16.974,
                            cy: 16.001,
                            fill: "url(#d5bd05__c)",
                            rx: 16.974,
                            ry: 16.001,
                            transform: "matrix(1 -.00132 .00148 1 9.68 -8.853)"
                        }),
                        /*#__PURE__*/ (0, $1yYpH$jsx)("path", {
                            fill: "url(#d5bd05__d)",
                            d: "M19.508-.797c-2.996-1.748-6.793-1.758-9.8-.027l-30.081 17.313c-2.947 1.696-2.951 5.698-.01 7.4L9.5 41.18c3.003 1.737 6.8 1.735 9.8-.006l29.821-17.306c2.926-1.698 2.932-5.678.009-7.383z"
                        }),
                        /*#__PURE__*/ (0, $1yYpH$jsx)("path", {
                            fill: "url(#d5bd05__e)",
                            d: "M10.24 23.163C10.907 9.432 23.256-1.19 37.822-.561s25.834 12.27 25.167 26-13.016 24.353-27.582 23.725c-14.566-.63-25.833-12.27-25.167-26.001"
                        }),
                        /*#__PURE__*/ (0, $1yYpH$jsx)("path", {
                            fill: "url(#d5bd05__f)",
                            d: "M-11.191 32.216c5.79-2.329 12.035-2.229 17.445-.192 10.825 4.085 17.124 1.56 21.493-8.622 2.177-5.11 6.425-9.436 12.222-11.768 11.59-4.653 24.969.396 29.898 11.289 4.93 10.893-.466 23.485-12.053 28.145-5.81 2.336-12.066 2.227-17.484.172-10.808-4.064-17.095-1.529-21.459 8.665-2.181 5.097-6.43 9.41-12.22 11.738-11.58 4.657-24.959-.392-29.885-11.28-4.927-10.886.463-23.49 12.043-28.147"
                        }),
                        /*#__PURE__*/ (0, $1yYpH$jsx)("ellipse", {
                            cx: 16.974,
                            cy: 16.001,
                            fill: "url(#d5bd05__g)",
                            rx: 16.974,
                            ry: 16.001,
                            transform: "matrix(1 -.00132 .00148 1 24.363 -3.634)"
                        })
                    ]
                })
            })
        ]
    });
const $9b6b05811ed7bc5d$var$ForwardRef = /*#__PURE__*/ (0, $1yYpH$forwardRef)($9b6b05811ed7bc5d$var$SvgComponent);
var $9b6b05811ed7bc5d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9b6b05811ed7bc5d$var$ForwardRef);


export {$9b6b05811ed7bc5d$export$2e2bcd8739ae039 as default};
