// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_volume_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $gN0C8$jsxs, jsx as $gN0C8$jsx} from "react/jsx-runtime";
import {forwardRef as $gN0C8$forwardRef} from "react";





const $973197010429fa31$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gN0C8$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gN0C8$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gN0C8$jsxs)("linearGradient", {
                        id: "be5d26__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gN0C8$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gN0C8$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gN0C8$jsxs)("linearGradient", {
                        id: "be5d26__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gN0C8$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gN0C8$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gN0C8$jsxs)("linearGradient", {
                        id: "be5d26__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gN0C8$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gN0C8$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gN0C8$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gN0C8$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gN0C8$jsxs)("radialGradient", {
                        id: "be5d26__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gN0C8$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gN0C8$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gN0C8$jsxs)("radialGradient", {
                        id: "be5d26__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gN0C8$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gN0C8$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gN0C8$jsx)("clipPath", {
                        id: "be5d26__b",
                        children: /*#__PURE__*/ (0, $gN0C8$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gN0C8$jsx)("clipPath", {
                id: "be5d26__a",
                children: /*#__PURE__*/ (0, $gN0C8$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M14.14 16.026h-4.03c-1.105 0-2 .896-2 2v12.011c0 1.105.895 2 2 2h4.03zm2.5-1.505v19.007l4.903 4.812c.949.93 2.55.259 2.55-1.07V10.73c0-1.333-1.609-2.003-2.555-1.065zm10.742 2.715c.415-.552 1.199-.663 1.75-.248 2.116 1.592 3.487 4.127 3.487 6.983 0 2.878-1.394 5.432-3.538 7.02-.555.412-1.338.295-1.749-.26-.41-.554-.294-1.337.26-1.748 1.536-1.138 2.527-2.96 2.527-5.012 0-2.037-.975-3.845-2.49-4.985-.551-.415-.662-1.199-.247-1.75m6.753-4.812c-.552-.415-1.335-.304-1.75.247-.416.552-.305 1.336.247 1.75 2.894 2.179 4.762 5.64 4.762 9.537 0 3.93-1.898 7.415-4.834 9.59-.554.41-.67 1.193-.26 1.748s1.194.671 1.749.26c3.544-2.626 5.845-6.843 5.845-11.598 0-4.716-2.264-8.904-5.76-11.534"
                })
            }),
            /*#__PURE__*/ (0, $gN0C8$jsx)("g", {
                clipPath: "url(#be5d26__a)",
                children: /*#__PURE__*/ (0, $gN0C8$jsxs)("g", {
                    clipPath: "url(#be5d26__b)",
                    children: [
                        /*#__PURE__*/ (0, $gN0C8$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $gN0C8$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $gN0C8$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#be5d26__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $gN0C8$jsx)("path", {
                            fill: "url(#be5d26__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $gN0C8$jsx)("path", {
                            fill: "url(#be5d26__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $gN0C8$jsx)("path", {
                            fill: "url(#be5d26__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $gN0C8$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#be5d26__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $973197010429fa31$var$ForwardRef = /*#__PURE__*/ (0, $gN0C8$forwardRef)($973197010429fa31$var$SvgComponent);
var $973197010429fa31$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($973197010429fa31$var$ForwardRef);


export {$973197010429fa31$export$2e2bcd8739ae039 as default};
