// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_effects_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8XSho$reactjsxruntime = require("react/jsx-runtime");
var $8XSho$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $067afce6177c9ab9$export$2e2bcd8739ae039);




const $067afce6177c9ab9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ac1546__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ac1546__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ac1546__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ac1546__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ac1546__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("clipPath", {
                        id: "ac1546__b",
                        children: /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("clipPath", {
                id: "ac1546__a",
                children: /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M44.124 18.464c-2.279 1.736-3.867 4.464-4.779 6.97L36.013 36.29h17.183c.727 0 1.397.395 1.75 1.031l6.775 12.24 12.721-12.706c.782-.78 2.048-.78 2.828.002s.78 2.048-.001 2.829c-3.692 3.686-8.695 8.683-13.532 13.517l7.531 13.605c.535.967.185 2.184-.781 2.719s-2.183.185-2.718-.782l-6.976-12.6-1.97 1.969c-3.06 3.06-5.813 5.812-7.807 7.808l-2.363 2.368-.645.65-.165.168-.02.02c-.687.752-1.845.873-2.675.25-.884-.663-1.063-1.916-.4-2.8.05-.068.097-.12.103-.127l.001-.001.034-.037.035-.038.023-.024.038-.04.182-.183.654-.66 2.368-2.373c1.995-1.997 4.748-4.75 7.809-7.81l2.783-2.782-6.761-12.213H34.785L22.053 81.774c-.324 1.056-1.443 1.65-2.499 1.325s-1.649-1.443-1.325-2.499l12.372-40.31h-8.122c-1.105 0-2-.895-2-2s.895-2 2-2h9.35l3.707-12.078.03-.092c1.06-2.936 3.014-6.46 6.135-8.838 3.23-2.46 7.56-3.564 12.909-1.775 1.047.35 1.612 1.483 1.262 2.53-.35 1.048-1.483 1.614-2.531 1.263-4.084-1.366-7.033-.499-9.217 1.164m3.687 50.677.004-.004c-.026.03-.034.036-.004.004"
                })
            }),
            /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ac1546__a)",
                children: /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ac1546__b)",
                    children: [
                        /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#ac1546__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("path", {
                            fill: "url(#ac1546__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("path", {
                            fill: "url(#ac1546__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("path", {
                            fill: "url(#ac1546__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $8XSho$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#ac1546__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $067afce6177c9ab9$var$ForwardRef = /*#__PURE__*/ (0, $8XSho$react.forwardRef)($067afce6177c9ab9$var$SvgComponent);
var $067afce6177c9ab9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($067afce6177c9ab9$var$ForwardRef);
