// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_video_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $aJhkG$jsxs, jsx as $aJhkG$jsx} from "react/jsx-runtime";
import {forwardRef as $aJhkG$forwardRef} from "react";





const $d57a7e79dbe553d0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aJhkG$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aJhkG$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aJhkG$jsxs)("linearGradient", {
                        id: "029657__d",
                        x1: -7.489,
                        x2: 60.376,
                        y1: 40.303,
                        y2: -0.986,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aJhkG$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $aJhkG$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aJhkG$jsxs)("linearGradient", {
                        id: "029657__e",
                        x1: 46.97,
                        x2: 17.856,
                        y1: 10.689,
                        y2: 41.995,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aJhkG$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aJhkG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aJhkG$jsxs)("linearGradient", {
                        id: "029657__f",
                        x1: 11.592,
                        x2: 63.203,
                        y1: 46.315,
                        y2: 17.403,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aJhkG$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aJhkG$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aJhkG$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aJhkG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aJhkG$jsxs)("radialGradient", {
                        id: "029657__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 10.992 15.611)scale(18.1603)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aJhkG$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $aJhkG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aJhkG$jsxs)("radialGradient", {
                        id: "029657__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 15.885 26.426)scale(18.1603)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aJhkG$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aJhkG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aJhkG$jsx)("clipPath", {
                        id: "029657__b",
                        children: /*#__PURE__*/ (0, $aJhkG$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3,
                            y: 3,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aJhkG$jsx)("clipPath", {
                id: "029657__a",
                children: /*#__PURE__*/ (0, $aJhkG$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M6 15v18c0 2.651 2.149 4.8 4.8 4.8h26.4c2.651 0 4.8-2.149 4.8-4.8V15c0-2.65-2.149-4.8-4.8-4.8H10.8C8.149 10.2 6 12.35 6 15m15.543 4.344 6.28 3.636c.704.407.702 1.423-.002 1.828l-6.289 3.62c-.704.406-1.583-.103-1.582-.915l.009-7.257c0-.812.881-1.32 1.584-.912"
                })
            }),
            /*#__PURE__*/ (0, $aJhkG$jsx)("g", {
                clipPath: "url(#029657__a)",
                children: /*#__PURE__*/ (0, $aJhkG$jsxs)("g", {
                    clipPath: "url(#029657__b)",
                    children: [
                        /*#__PURE__*/ (0, $aJhkG$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3,
                            y: 3,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $aJhkG$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3 3h41.742v41.742H3z"
                        }),
                        /*#__PURE__*/ (0, $aJhkG$jsx)("circle", {
                            cx: 26.604,
                            cy: 4.619,
                            r: 18.16,
                            fill: "url(#029657__c)",
                            transform: "rotate(-.08 26.604 4.62)"
                        }),
                        /*#__PURE__*/ (0, $aJhkG$jsx)("path", {
                            fill: "url(#029657__d)",
                            d: "M18.934-4.374c-3.206-1.984-7.268-1.996-10.486-.031l-32.183 19.65c-3.151 1.924-3.157 6.467-.009 8.398l31.97 19.622c3.213 1.972 7.275 1.97 10.485-.007l31.904-19.64c3.131-1.928 3.137-6.445.01-8.38z"
                        }),
                        /*#__PURE__*/ (0, $aJhkG$jsx)("path", {
                            fill: "url(#029657__e)",
                            d: "M9.015 22.819C9.728 7.235 22.938-4.82 38.523-4.106S66.162 9.819 65.448 25.402c-.713 15.584-13.925 27.639-29.508 26.925C20.356 51.614 8.3 38.402 9.015 22.82"
                        }),
                        /*#__PURE__*/ (0, $aJhkG$jsx)("path", {
                            fill: "url(#029657__f)",
                            d: "M-13.91 33.094c6.194-2.643 12.875-2.53 18.663-.218 11.582 4.636 18.321 1.77 22.996-9.786 2.328-5.798 6.873-10.71 13.075-13.355 12.399-5.281 26.713.45 31.987 12.812S72.312 49.2 59.916 54.489c-6.215 2.652-12.909 2.527-18.706.195-11.563-4.612-18.29-1.734-22.957 9.835-2.335 5.785-6.88 10.679-13.074 13.321-12.389 5.286-26.703-.445-31.974-12.8-5.27-12.356.496-26.661 12.885-31.947"
                        }),
                        /*#__PURE__*/ (0, $aJhkG$jsx)("circle", {
                            cx: 42.311,
                            cy: 10.541,
                            r: 18.16,
                            fill: "url(#029657__g)",
                            transform: "rotate(-.08 42.31 10.541)"
                        })
                    ]
                })
            })
        ]
    });
const $d57a7e79dbe553d0$var$ForwardRef = /*#__PURE__*/ (0, $aJhkG$forwardRef)($d57a7e79dbe553d0$var$SvgComponent);
var $d57a7e79dbe553d0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d57a7e79dbe553d0$var$ForwardRef);


export {$d57a7e79dbe553d0$export$2e2bcd8739ae039 as default};
