// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_AIGenerateImage_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $53IEy$jsxs, jsx as $53IEy$jsx} from "react/jsx-runtime";
import {forwardRef as $53IEy$forwardRef} from "react";





const $7949dfdae3dde924$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $53IEy$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $53IEy$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $53IEy$jsxs)("linearGradient", {
                        id: "f99fb7__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $53IEy$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $53IEy$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $53IEy$jsxs)("linearGradient", {
                        id: "f99fb7__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $53IEy$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $53IEy$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $53IEy$jsxs)("linearGradient", {
                        id: "f99fb7__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $53IEy$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $53IEy$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $53IEy$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $53IEy$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $53IEy$jsxs)("radialGradient", {
                        id: "f99fb7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $53IEy$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $53IEy$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $53IEy$jsxs)("radialGradient", {
                        id: "f99fb7__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $53IEy$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $53IEy$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $53IEy$jsx)("clipPath", {
                        id: "f99fb7__b",
                        children: /*#__PURE__*/ (0, $53IEy$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $53IEy$jsx)("clipPath", {
                id: "f99fb7__a",
                children: /*#__PURE__*/ (0, $53IEy$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m142.816 50.766-10.391-2.338c-1.543-.35-3.163.085-4.326 1.157l-7.831 7.217c-3.476 3.205-9.007.012-7.97-4.603l2.337-10.391c.349-1.543-.084-3.163-1.157-4.326l-7.217-7.831c-3.205-3.476-.012-9.007 4.603-7.97l10.392 2.337c1.542.35 3.162-.084 4.325-1.157l7.832-7.217c3.476-3.205 9.006-.012 7.97 4.603l-2.338 10.392c-.349 1.542.085 3.162 1.157 4.325l7.217 7.831c3.205 3.477.012 9.007-4.603 7.97M18.824 63.347c5.228-19.513 25.286-31.094 44.8-25.865 8.472 2.27 15.441 7.329 20.197 13.929 1.291 1.792 3.791 2.198 5.583.907 1.793-1.292 2.199-3.792.907-5.584-5.794-8.041-14.3-14.215-24.617-16.98-23.781-6.372-48.225 7.741-54.598 31.522-4.089 15.261.258 30.788 10.173 41.64l-7.253-.023c-2.21-.007-4.006 1.778-4.013 3.987s1.778 4.006 3.987 4.013l16.494.053c2.155.006 3.928-1.696 4.01-3.849l.601-15.913c.084-2.207-1.638-4.065-3.846-4.148-2.207-.084-4.065 1.638-4.148 3.846l-.238 6.293c-7.922-8.893-11.35-21.473-8.04-33.828m87.964 8.09.183.199.003.003 1.404 1.522c2.006 2.175.006 5.633-2.873 4.982l-2.031-.457-1.869-.421-.411-.093-3.015-.68c-.97-.216-1.976.055-2.705.724l-2.272 2.096-.31.285-1.41 1.302-.004.003-1.522 1.404c-2.175 2.006-5.633.006-4.982-2.873l.458-2.031.42-1.87.093-.41.68-3.015q.075-.334.073-.67c-.002-.747-.28-1.473-.796-2.035l-5.09-5.518c-2.007-2.175-.007-5.633 2.873-4.982l7.325 1.65c.97.218 1.976-.054 2.705-.722l5.519-5.09c2.174-2.007 5.632-.007 4.982 2.873l-1.651 7.325c-.217.97.054 1.976.723 2.705zm9.772 0h15.073c7.179 0 13 5.82 13 13v48.915c0 2.214-.554 4.299-1.531 6.124l-16.296-24.123c-3.871-5.77-12.353-5.755-16.223.001l-.018.028-8.848 13.394-16.175-24.985v-.002h-.001c-3.847-5.947-12.557-5.948-16.405-.003l-.003.005-23.842 36.672V84.438c0-7.18 5.82-13 13-13h18.604l1.35 1.463-1.074 4.769v.002l-.001.002C74.79 88.22 87.444 95.508 95.38 88.186l3.593-3.314 4.769 1.075h.004c9.013 2.034 15.646-6.912 12.814-14.51m-65.555 74.915h39.102l6.797-10.29-18.079-27.926-.001-.002c-.695-1.075-2.278-1.075-2.972 0l-.006.009zm69.168-26.527 17.064 25.26c-1.697.812-3.598 1.267-5.604 1.267H99.695l17.534-26.544c.711-1.044 2.243-1.029 2.934.004zM120.666 98c-3.314 0-6-2.686-6-6s2.686-6 6-6 6 2.686 6 6-2.686 6-6 6"
                })
            }),
            /*#__PURE__*/ (0, $53IEy$jsx)("g", {
                clipPath: "url(#f99fb7__a)",
                children: /*#__PURE__*/ (0, $53IEy$jsxs)("g", {
                    clipPath: "url(#f99fb7__b)",
                    children: [
                        /*#__PURE__*/ (0, $53IEy$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $53IEy$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $53IEy$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#f99fb7__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $53IEy$jsx)("path", {
                            fill: "url(#f99fb7__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $53IEy$jsx)("path", {
                            fill: "url(#f99fb7__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $53IEy$jsx)("path", {
                            fill: "url(#f99fb7__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $53IEy$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#f99fb7__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $7949dfdae3dde924$var$ForwardRef = /*#__PURE__*/ (0, $53IEy$forwardRef)($7949dfdae3dde924$var$SvgComponent);
var $7949dfdae3dde924$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7949dfdae3dde924$var$ForwardRef);


export {$7949dfdae3dde924$export$2e2bcd8739ae039 as default};
