// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_libraries_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gVD5F$reactjsxruntime = require("react/jsx-runtime");
var $gVD5F$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2a3ab50e61a16d24$export$2e2bcd8739ae039);




const $2a3ab50e61a16d24$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3d804e__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3d804e__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3d804e__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3d804e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3d804e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("clipPath", {
                        id: "3d804e__b",
                        children: /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("clipPath", {
                id: "3d804e__a",
                children: /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M19.531 27.52c0-4.416 3.585-8 8-8h31.95c1.105 0 2-.896 2-2s-.895-2-2-2h-31.95c-6.624 0-12 5.375-12 12v31.95c0 1.104.896 2 2 2s2-.896 2-2zm36.823 22.856V25.458H35.058c-5.302 0-9.6 4.298-9.6 9.6v34.89c0 5.302 4.298 9.6 9.6 9.6h34.89c5.302 0 9.6-4.299 9.6-9.6v-34.89c0-4.982-3.795-9.078-8.652-9.554v24.872c0 .954-1.068 1.53-1.866.996l-4.74-3.156c-.401-.27-.93-.27-1.331 0l-4.74 3.156c-.798.534-1.867-.036-1.867-.996"
                })
            }),
            /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("g", {
                clipPath: "url(#3d804e__a)",
                children: /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#3d804e__b)",
                    children: [
                        /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#3d804e__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("path", {
                            fill: "url(#3d804e__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("path", {
                            fill: "url(#3d804e__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("path", {
                            fill: "url(#3d804e__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $gVD5F$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#3d804e__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $2a3ab50e61a16d24$var$ForwardRef = /*#__PURE__*/ (0, $gVD5F$react.forwardRef)($2a3ab50e61a16d24$var$SvgComponent);
var $2a3ab50e61a16d24$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2a3ab50e61a16d24$var$ForwardRef);
