// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_calendar_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $lS5Fp$reactjsxruntime = require("react/jsx-runtime");
var $lS5Fp$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5538c8e28618ac16$export$2e2bcd8739ae039);




const $5538c8e28618ac16$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6653d8__d",
                        x1: -23.686,
                        x2: 195.849,
                        y1: 127.603,
                        y2: -14.082,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6653d8__e",
                        x1: 158.458,
                        x2: 67.22,
                        y1: 34.24,
                        y2: 138.311,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6653d8__f",
                        x1: 40.127,
                        x2: 207.718,
                        y1: 146.558,
                        y2: 46.967,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6653d8__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 57.2532 -60.7348 0 60.735 57.253)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6653d8__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 57.2532 -60.7348 0 60.735 57.253)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("clipPath", {
                        id: "6653d8__b",
                        children: /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("rect", {
                            width: 139.603,
                            height: 131.6,
                            x: 11.398,
                            y: 10,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("clipPath", {
                id: "6653d8__a",
                children: /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M116 34h-7v10c0 2.21-1.79 4-4 4s-4-1.79-4-4V34H59v10c0 2.21-1.79 4-4 4s-4-1.79-4-4V34h-7c-8.84 0-16 7.16-16 16v10h104V50c0-8.84-7.16-16-16-16m-88 82c0 8.84 7.16 16 16 16h72c8.84 0 16-7.16 16-16V68H28zm73-32c0-2.21 1.79-4 4-4h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4zm0 22c0-2.21 1.79-4 4-4h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4zM73 84c0-2.21 1.79-4 4-4h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4zm0 22c0-2.21 1.79-4 4-4h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4zM45 84c0-2.21 1.79-4 4-4h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4zm0 22c0-2.21 1.79-4 4-4h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4zm14-82c0-2.21-1.79-4-4-4s-4 1.79-4 4v10h8zm50 0c0-2.21-1.79-4-4-4s-4 1.79-4 4v10h8z"
                })
            }),
            /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6653d8__a)",
                children: /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6653d8__b)",
                    children: [
                        /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("rect", {
                            width: 139.603,
                            height: 131.6,
                            x: 11.398,
                            y: 10,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.398 10h139.603v131.6H11.398z"
                        }),
                        /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("ellipse", {
                            cx: 60.735,
                            cy: 57.253,
                            fill: "url(#6653d8__c)",
                            rx: 60.735,
                            ry: 57.253,
                            transform: "matrix(1 -.00132 .00148 1 29.516 -42.068)"
                        }),
                        /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("path", {
                            fill: "url(#6653d8__d)",
                            d: "M64.68-13.246c-10.72-6.254-24.305-6.293-35.066-.1l-107.631 61.95c-10.541 6.067-10.559 20.388-.032 26.478l106.92 61.861c10.745 6.216 24.33 6.209 35.065-.022l106.7-61.919c10.471-6.077 10.49-20.318.032-26.418z"
                        }),
                        /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("path", {
                            fill: "url(#6653d8__e)",
                            d: "M31.519 72.48c2.386-49.13 46.57-87.134 98.688-84.885 52.118 2.25 92.433 43.9 90.047 93.031-2.386 49.13-46.57 87.135-98.688 84.885s-92.433-43.9-90.047-93.03"
                        }),
                        /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("path", {
                            fill: "url(#6653d8__f)",
                            d: "M-45.162 104.875c20.717-8.331 43.06-7.974 62.417-.687 38.734 14.618 61.273 5.58 76.905-30.85 7.787-18.28 22.989-33.763 43.729-42.103 41.467-16.65 89.339 1.416 106.977 40.39 17.637 38.974-1.668 84.031-43.125 100.703-20.787 8.359-43.173 7.968-62.56.615-38.671-14.54-61.167-5.468-76.78 31.006-7.806 18.237-23.004 33.666-43.721 41.998-41.434 16.662-89.306-1.404-106.934-40.357-17.628-38.952 1.658-84.052 43.092-100.715"
                        }),
                        /*#__PURE__*/ (0, $lS5Fp$reactjsxruntime.jsx)("ellipse", {
                            cx: 60.735,
                            cy: 57.253,
                            fill: "url(#6653d8__g)",
                            rx: 60.735,
                            ry: 57.253,
                            transform: "matrix(1 -.00132 .00148 1 82.043 -23.398)"
                        })
                    ]
                })
            })
        ]
    });
const $5538c8e28618ac16$var$ForwardRef = /*#__PURE__*/ (0, $lS5Fp$react.forwardRef)($5538c8e28618ac16$var$SvgComponent);
var $5538c8e28618ac16$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5538c8e28618ac16$var$ForwardRef);
