// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_paperairplane_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $hEWdC$reactjsxruntime = require("react/jsx-runtime");
var $hEWdC$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $efe5f034e27554dc$export$2e2bcd8739ae039);




const $efe5f034e27554dc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b451e2__d",
                        x1: -22.923,
                        x2: 194.398,
                        y1: 132.145,
                        y2: -8.112,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b451e2__e",
                        x1: 157.381,
                        x2: 67.064,
                        y1: 39.723,
                        y2: 142.745,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b451e2__f",
                        x1: 40.244,
                        x2: 206.146,
                        y1: 150.908,
                        y2: 52.321,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b451e2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 56.676 -60.1225 0 60.123 56.676)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b451e2__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 56.676 -60.1225 0 60.123 56.676)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("clipPath", {
                        id: "b451e2__b",
                        children: /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("rect", {
                            width: 138.195,
                            height: 130.273,
                            x: 11.805,
                            y: 15.727,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("clipPath", {
                id: "b451e2__a",
                children: /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M138.276 39.184c1.644-.37 3.202-.162 4.534.446L39.157 93.45q-.49.257-.876.615L16.09 78.884c-5.117-3.5-3.592-11.376 2.467-12.74zM94.829 132.75l-49.19-33.65L146.843 46.55q.075-.038.146-.08c-.053.93-.298 1.879-.768 2.807l-40.98 80.825c-1.945 3.835-6.852 5.083-10.412 2.647M41 131.316v-25.382l23.492 16.006-15.53 12.924c-3.127 2.602-7.962.447-7.962-3.548"
                })
            }),
            /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("g", {
                clipPath: "url(#b451e2__a)",
                children: /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#b451e2__b)",
                    children: [
                        /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("rect", {
                            width: 138.195,
                            height: 130.273,
                            x: 11.805,
                            y: 15.727,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.805 15.727H150V146H11.805z"
                        }),
                        /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("ellipse", {
                            cx: 60.123,
                            cy: 56.676,
                            fill: "url(#b451e2__c)",
                            rx: 60.123,
                            ry: 56.676,
                            transform: "matrix(1 -.00132 .00148 1 29.738 -35.817)"
                        }),
                        /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("path", {
                            fill: "url(#b451e2__d)",
                            d: "M64.553-7.285c-10.613-6.191-24.06-6.23-34.713-.098L-76.706 53.94c-10.435 6.006-10.453 20.183-.032 26.212L29.104 141.39c10.637 6.153 24.084 6.145 34.712-.022L169.44 80.074c10.365-6.016 10.384-20.114.032-26.152z"
                        }),
                        /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("path", {
                            fill: "url(#b451e2__e)",
                            d: "M31.722 77.578c2.362-48.635 46.1-86.256 97.693-84.03 51.592 2.227 91.501 43.458 89.139 92.093s-46.101 86.256-97.693 84.03c-51.592-2.227-91.501-43.458-89.14-92.093"
                        }),
                        /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("path", {
                            fill: "url(#b451e2__f)",
                            d: "M-44.185 109.645c20.508-8.248 42.625-7.894 61.788-.68 38.344 14.471 60.655 5.524 76.13-30.54 7.707-18.096 22.757-33.422 43.288-41.678 41.048-16.483 88.438 1.402 105.898 39.983 17.46 38.582-1.651 83.184-42.69 99.688-20.577 8.275-42.738 7.887-61.929.608-38.282-14.393-60.55-5.412-76.006 30.694-7.727 18.053-22.773 33.327-43.28 41.574-41.017 16.495-88.406-1.39-105.856-39.949-17.451-38.56 1.64-83.206 42.657-99.7"
                        }),
                        /*#__PURE__*/ (0, $hEWdC$reactjsxruntime.jsx)("ellipse", {
                            cx: 60.123,
                            cy: 56.676,
                            fill: "url(#b451e2__g)",
                            rx: 60.123,
                            ry: 56.676,
                            transform: "matrix(1 -.00132 .00148 1 81.738 -17.335)"
                        })
                    ]
                })
            })
        ]
    });
const $efe5f034e27554dc$var$ForwardRef = /*#__PURE__*/ (0, $hEWdC$react.forwardRef)($efe5f034e27554dc$var$SvgComponent);
var $efe5f034e27554dc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($efe5f034e27554dc$var$ForwardRef);
