// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileImage_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4D8Gj$reactjsxruntime = require("react/jsx-runtime");
var $4D8Gj$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1a412de727c682f8$export$2e2bcd8739ae039);




const $1a412de727c682f8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "fcf3be__d",
                        x1: -22.794,
                        x2: 202.298,
                        y1: 134.5,
                        y2: -2.444,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "fcf3be__e",
                        x1: 157.843,
                        x2: 61.279,
                        y1: 36.277,
                        y2: 140.11,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "fcf3be__f",
                        x1: 40.492,
                        x2: 211.674,
                        y1: 154.441,
                        y2: 58.547,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsxs)("radialGradient", {
                        id: "fcf3be__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 37.068 53.214)scale(60.2331)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsxs)("radialGradient", {
                        id: "fcf3be__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 53.296 89.084)scale(60.2331)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("clipPath", {
                        id: "fcf3be__b",
                        children: /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 12,
                            y: 10.775,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("clipPath", {
                id: "fcf3be__a",
                children: /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m87.515 31.515 32.97 32.97c2.251 2.25 3.515 5.303 3.515 8.486V120c0 6.627-5.373 12-12 12H48c-6.627 0-12-5.373-12-12V40c0-6.627 5.373-12 12-12h31.03c3.182 0 6.234 1.264 8.485 3.515M94.499 68c-4.142 0-7.5 3.358-7.5 7.5s3.358 7.5 7.5 7.5c4.143 0 7.501-3.358 7.501-7.5S98.642 68 94.5 68m-40.494 48h51.989c1.565 0 2.524-1.715 1.704-3.048L96.511 94.769c-.78-1.27-2.626-1.27-3.407 0l-8.816 14.329-13.091-22.199c-.775-1.306-2.666-1.306-3.44 0L52.284 112.98c-.79 1.333.17 3.02 1.72 3.02"
                })
            }),
            /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("g", {
                clipPath: "url(#fcf3be__a)",
                children: /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#fcf3be__b)",
                    children: [
                        /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 12,
                            y: 10.775,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M12 10.775h138.449v138.45H12z"
                        }),
                        /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("circle", {
                            cx: 90.282,
                            cy: 16.146,
                            r: 60.233,
                            fill: "url(#fcf3be__c)",
                            transform: "rotate(-.08 90.282 16.146)"
                        }),
                        /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("path", {
                            fill: "url(#fcf3be__d)",
                            d: "M64.843-13.68c-10.633-6.58-24.105-6.62-34.777-.105L-76.676 51.39c-10.454 6.383-10.472 21.45-.032 27.856l106.037 65.081c10.656 6.54 24.128 6.532 34.775-.023l105.819-65.142c10.384-6.393 10.403-21.376.031-27.793z"
                        }),
                        /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("path", {
                            fill: "url(#fcf3be__e)",
                            d: "M31.952 76.508c2.367-51.688 46.186-91.67 97.873-89.304 51.687 2.367 91.67 46.186 89.303 97.873-2.366 51.687-46.185 91.67-97.872 89.303-51.687-2.366-91.67-46.185-89.304-97.872"
                        }),
                        /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("path", {
                            fill: "url(#fcf3be__f)",
                            d: "M-44.093 110.588c20.546-8.765 42.704-8.389 61.902-.723 38.414 15.38 60.766 5.871 76.27-32.455 7.722-19.233 22.799-35.52 43.367-44.295 41.125-17.517 88.601 1.49 106.093 42.493s-1.654 88.404-42.768 105.944c-20.615 8.794-42.816 8.382-62.043.647-38.352-15.297-60.662-5.753-76.145 32.62-7.742 19.186-22.815 35.418-43.361 44.183-41.092 17.53-88.568-1.477-106.05-42.457-17.483-40.98 1.644-88.427 42.735-105.957"
                        }),
                        /*#__PURE__*/ (0, $4D8Gj$reactjsxruntime.jsx)("circle", {
                            cx: 142.38,
                            cy: 35.788,
                            r: 60.233,
                            fill: "url(#fcf3be__g)",
                            transform: "rotate(-.08 142.38 35.788)"
                        })
                    ]
                })
            })
        ]
    });
const $1a412de727c682f8$var$ForwardRef = /*#__PURE__*/ (0, $4D8Gj$react.forwardRef)($1a412de727c682f8$var$SvgComponent);
var $1a412de727c682f8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1a412de727c682f8$var$ForwardRef);
