// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_infiniteLooping_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $2ziSO$reactjsxruntime = require("react/jsx-runtime");
var $2ziSO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $767a63a4eccff0fb$export$2e2bcd8739ae039);




const $767a63a4eccff0fb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a13a81__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.982,
                        y2: -15.278,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a13a81__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a13a81__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a13a81__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a13a81__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("clipPath", {
                        id: "a13a81__b",
                        children: /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("clipPath", {
                id: "a13a81__a",
                children: /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M72.33 72.502c2.21-.005 3.996-1.8 3.99-4.009l-.056-24.865c-.005-2.21-1.8-3.996-4.01-3.991s-3.996 1.8-3.99 4.009l.035 15.114c-6.64-6.98-16.024-11.337-26.427-11.337-20.141 0-36.47 16.328-36.47 36.47s16.328 36.47 36.47 36.47c8.355 0 16.065-2.814 22.215-7.545q.14-.107.268-.224l.12-.091c4.531-3.557 8.748-8.72 12.042-13.442 3.31-4.745 5.903-9.354 7.118-12.036.945-2.084 3.285-7.04 6.38-12.245 2.665-4.483 5.7-8.819 8.703-11.737q.14-.106.272-.227c5.059-4.596 11.768-7.393 19.14-7.393 15.723 0 28.47 12.747 28.47 28.47 0 15.724-12.747 28.47-28.47 28.47-9.615 0-18.118-4.762-23.28-12.075-1.273-1.805-3.769-2.235-5.574-.962-1.804 1.274-2.235 3.77-.961 5.575 6.595 9.345 17.49 15.462 29.815 15.462 20.141 0 36.47-16.328 36.47-36.47s-16.329-36.47-36.47-36.47c-8.343 0-16.038 2.806-22.182 7.52q-.267.15-.514.345c-4.756 3.75-9.057 9.957-12.295 15.403-3.305 5.556-5.781 10.806-6.79 13.032-.968 2.136-3.293 6.317-6.392 10.76-3.115 4.464-6.801 8.885-10.422 11.727q-.122.096-.235.2l-.09.067c-4.801 3.693-10.808 5.886-17.338 5.886-15.723 0-28.47-12.746-28.47-28.47s12.747-28.47 28.47-28.47c8.24 0 15.664 3.498 20.867 9.101l-15.293.036c-2.21.005-3.996 1.8-3.99 4.009.004 2.209 1.8 3.996 4.008 3.99z"
                })
            }),
            /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("g", {
                clipPath: "url(#a13a81__a)",
                children: /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#a13a81__b)",
                    children: [
                        /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#a13a81__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("path", {
                            fill: "url(#a13a81__d)",
                            d: "M61.07-28.264c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.549 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("path", {
                            fill: "url(#a13a81__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("path", {
                            fill: "url(#a13a81__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $2ziSO$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#a13a81__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $767a63a4eccff0fb$var$ForwardRef = /*#__PURE__*/ (0, $2ziSO$react.forwardRef)($767a63a4eccff0fb$var$SvgComponent);
var $767a63a4eccff0fb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($767a63a4eccff0fb$var$ForwardRef);
