// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_targeted_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $l6piw$reactjsxruntime = require("react/jsx-runtime");
var $l6piw$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f91a2d8ce00f1f60$export$2e2bcd8739ae039);




const $f91a2d8ce00f1f60$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4e1436__e",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4e1436__f",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4e1436__g",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4e1436__d",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4e1436__h",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("clipPath", {
                        id: "4e1436__a",
                        children: /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("path", {
                            fill: "#fff",
                            d: "M0 0h96v96H0z"
                        })
                    }),
                    /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("clipPath", {
                        id: "4e1436__c",
                        children: /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsxs)("g", {
                clipPath: "url(#4e1436__a)",
                children: [
                    /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("clipPath", {
                        id: "4e1436__b",
                        children: /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("path", {
                            fill: "#fff",
                            fillRule: "evenodd",
                            d: "M9.619 18.305c-.696.179-1.243.716-1.433 1.41-.19.692.007 1.434.515 1.942l9.197 9.198c.415.415.992.626 1.577.579l8.202-.668 12.447 12.447c.781.781 2.048.781 2.829 0 .78-.781.78-2.047 0-2.828L30.507 27.939l.668-8.205c.047-.585-.165-1.162-.58-1.577L21.398 8.96c-.508-.508-1.25-.705-1.943-.515s-1.23.737-1.409 1.433l-1.718 6.709zm38.38-1.066c-3.73 0-7.301.663-10.604 1.876-1.037.381-2.186-.15-2.567-1.188-.38-1.037.15-2.186 1.188-2.567C39.754 13.987 43.792 13.24 48 13.24c19.198 0 34.762 15.563 34.762 34.762S67.198 82.763 48 82.763c-19.199 0-34.762-15.564-34.762-34.762 0-4.097.71-8.033 2.014-11.689.372-1.04 1.516-1.582 2.556-1.211s1.583 1.516 1.212 2.556c-1.154 3.23-1.782 6.711-1.782 10.344 0 16.99 13.772 30.762 30.762 30.762S78.762 64.99 78.762 48.001 64.989 17.239 48 17.239m0 15.506c-2.028 0-3.959.394-5.724 1.11-1.023.414-2.19-.08-2.604-1.104-.415-1.023.079-2.19 1.103-2.604 2.233-.905 4.674-1.402 7.225-1.402 10.635 0 19.255 8.62 19.255 19.255S58.634 67.255 48 67.255 28.744 58.635 28.744 48c0-2.377.432-4.659 1.223-6.766.388-1.034 1.541-1.558 2.575-1.17s1.558 1.541 1.17 2.575c-.625 1.665-.968 3.47-.968 5.36 0 8.426 6.83 15.256 15.255 15.256S63.254 56.425 63.254 48 56.424 32.745 48 32.745m-5.344 18.562c4.52-.42 8.118-4.018 8.538-8.538 1.807 1.089 3.016 3.07 3.016 5.333 0 3.436-2.786 6.22-6.221 6.22-2.263 0-4.244-1.208-5.333-3.015"
                        })
                    }),
                    /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("g", {
                        clipPath: "url(#4e1436__b)",
                        children: /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsxs)("g", {
                            clipPath: "url(#4e1436__c)",
                            children: [
                                /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("rect", {
                                    width: 96,
                                    height: 96,
                                    fill: "#FFECCF",
                                    rx: 5.125
                                }),
                                /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("path", {
                                    fill: "#FFECCF",
                                    d: "M0 0h96v96H0z"
                                }),
                                /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("circle", {
                                    cx: 54.281,
                                    cy: 3.723,
                                    r: 41.765,
                                    fill: "url(#4e1436__d)",
                                    transform: "rotate(-.08 54.28 3.723)"
                                }),
                                /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("path", {
                                    fill: "url(#4e1436__e)",
                                    d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                                }),
                                /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("path", {
                                    fill: "url(#4e1436__f)",
                                    d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                                }),
                                /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("path", {
                                    fill: "url(#4e1436__g)",
                                    d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                                }),
                                /*#__PURE__*/ (0, $l6piw$reactjsxruntime.jsx)("circle", {
                                    cx: 90.406,
                                    cy: 17.344,
                                    r: 41.765,
                                    fill: "url(#4e1436__h)",
                                    transform: "rotate(-.08 90.406 17.344)"
                                })
                            ]
                        })
                    })
                ]
            })
        ]
    });
const $f91a2d8ce00f1f60$var$ForwardRef = /*#__PURE__*/ (0, $l6piw$react.forwardRef)($f91a2d8ce00f1f60$var$SvgComponent);
var $f91a2d8ce00f1f60$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f91a2d8ce00f1f60$var$ForwardRef);
