// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_filters_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $ktNaH$jsxs, jsx as $ktNaH$jsx} from "react/jsx-runtime";
import {forwardRef as $ktNaH$forwardRef} from "react";





const $77aaed70c75f195d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ktNaH$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $ktNaH$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $ktNaH$jsxs)("linearGradient", {
                        id: "33e29d__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ktNaH$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $ktNaH$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ktNaH$jsxs)("linearGradient", {
                        id: "33e29d__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ktNaH$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ktNaH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ktNaH$jsxs)("linearGradient", {
                        id: "33e29d__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ktNaH$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $ktNaH$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $ktNaH$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $ktNaH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ktNaH$jsxs)("radialGradient", {
                        id: "33e29d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ktNaH$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $ktNaH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ktNaH$jsxs)("radialGradient", {
                        id: "33e29d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ktNaH$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ktNaH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ktNaH$jsx)("clipPath", {
                        id: "33e29d__b",
                        children: /*#__PURE__*/ (0, $ktNaH$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ktNaH$jsx)("clipPath", {
                id: "33e29d__a",
                children: /*#__PURE__*/ (0, $ktNaH$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M6 28.95c0-3.263 1.353-6.21 3.529-8.31.498 4.419 3.045 8.217 6.669 10.422.535 3.552 2.4 6.669 5.072 8.826-1.167.397-2.418.612-3.72.612C11.171 40.5 6 35.33 6 28.95m20.37 10.938c1.167.397 2.418.612 3.72.612 6.379 0 11.55-5.17 11.55-11.55 0-3.557-1.608-6.74-4.138-8.858-.319 4.348-2.617 8.147-5.997 10.499-.436 3.747-2.346 7.045-5.135 9.297m8.15-24.275c-1.463-4.7-5.848-8.113-11.03-8.113-5.28 0-9.733 3.544-11.11 8.382 1.6-.633 3.344-.982 5.17-.982 2.253 0 4.383.53 6.27 1.474 1.887-.943 4.017-1.474 6.27-1.474 1.548 0 3.037.25 4.43.713m-22.578 3.238-.002.199c0 3.558 1.609 6.74 4.138 8.859.293-3.995 2.255-7.525 5.192-9.896-1.167-.397-2.418-.613-3.72-.613-2.035 0-3.948.527-5.608 1.451M23.82 38.652c2.32-1.502 4.066-3.813 4.84-6.534-1.6.634-3.344.982-5.17.982-1.548 0-3.037-.25-4.43-.713.81 2.604 2.518 4.813 4.76 6.265M35.04 19.05c0 3.263-1.353 6.21-3.529 8.31-.424-3.768-2.34-7.085-5.142-9.347 1.168-.397 2.42-.613 3.721-.613 1.767 0 3.44.397 4.937 1.106q.013.27.013.544m-5.94 9.9c0-4.067-2.103-7.643-5.28-9.701-3.178 2.058-5.28 5.634-5.28 9.701q0 .274.013.544c1.497.71 3.17 1.106 4.937 1.106 2.035 0 3.948-.526 5.608-1.45z"
                })
            }),
            /*#__PURE__*/ (0, $ktNaH$jsx)("g", {
                clipPath: "url(#33e29d__a)",
                children: /*#__PURE__*/ (0, $ktNaH$jsxs)("g", {
                    clipPath: "url(#33e29d__b)",
                    children: [
                        /*#__PURE__*/ (0, $ktNaH$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $ktNaH$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $ktNaH$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#33e29d__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $ktNaH$jsx)("path", {
                            fill: "url(#33e29d__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $ktNaH$jsx)("path", {
                            fill: "url(#33e29d__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $ktNaH$jsx)("path", {
                            fill: "url(#33e29d__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $ktNaH$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#33e29d__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $77aaed70c75f195d$var$ForwardRef = /*#__PURE__*/ (0, $ktNaH$forwardRef)($77aaed70c75f195d$var$SvgComponent);
var $77aaed70c75f195d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($77aaed70c75f195d$var$ForwardRef);


export {$77aaed70c75f195d$export$2e2bcd8739ae039 as default};
