// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_assets_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $k1Emf$reactjsxruntime = require("react/jsx-runtime");
var $k1Emf$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f70da74a98d4bfb0$export$2e2bcd8739ae039);




const $f70da74a98d4bfb0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5d12de__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5d12de__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5d12de__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5d12de__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5d12de__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("clipPath", {
                        id: "5d12de__b",
                        children: /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("clipPath", {
                id: "5d12de__a",
                children: /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M28.25 41.384c0-5.523 4.477-10 10-10h66.54c5.523 0 10 4.477 10 10v19.744H82.262c-5.367 0-10.186 2.35-13.484 6.076l-4.553-7.499-.011-.018c-3.581-5.828-12.02-5.921-15.728-.173l-.008.013-19.754 30.876c-.308-.962-.474-1.987-.474-3.051zm36.012 37.744v18.224H38.25c-1.404 0-2.741-.29-3.954-.813l20.913-32.688.002-.004c.517-.794 1.683-.782 2.183.022l.004.006 7.188 11.838c-.213 1.106-.324 2.247-.324 3.415m29.906-21.97c3.314 0 6-2.687 6-6 0-3.315-2.686-6-6-6s-6 2.685-6 6c0 3.313 2.686 6 6 6m-21.906 21.97c0-5.523 4.477-10 10-10h39.488c5.523 0 10 4.477 10 10v39.488c0 5.523-4.477 10-10 10H82.262c-5.523 0-10-4.477-10-10zm42.583 17.766-19.507-11.19c-2.185-1.252-4.92.308-4.923 2.809l-.027 22.331c-.003 2.501 2.728 4.067 4.916 2.819l19.534-11.142c2.187-1.248 2.191-4.374.007-5.627"
                })
            }),
            /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("g", {
                clipPath: "url(#5d12de__a)",
                children: /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#5d12de__b)",
                    children: [
                        /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#5d12de__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("path", {
                            fill: "url(#5d12de__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("path", {
                            fill: "url(#5d12de__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("path", {
                            fill: "url(#5d12de__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $k1Emf$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#5d12de__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $f70da74a98d4bfb0$var$ForwardRef = /*#__PURE__*/ (0, $k1Emf$react.forwardRef)($f70da74a98d4bfb0$var$SvgComponent);
var $f70da74a98d4bfb0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f70da74a98d4bfb0$var$ForwardRef);
