// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_artboard_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $i8ylg$jsxs, jsx as $i8ylg$jsx} from "react/jsx-runtime";
import {forwardRef as $i8ylg$forwardRef} from "react";





const $2750d7ea242a4139$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $i8ylg$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $i8ylg$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $i8ylg$jsxs)("linearGradient", {
                        id: "ea280d__d",
                        x1: -9.627,
                        x2: 56.234,
                        y1: 38.282,
                        y2: -4.224,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i8ylg$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $i8ylg$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i8ylg$jsxs)("linearGradient", {
                        id: "ea280d__e",
                        x1: 45.015,
                        x2: 17.644,
                        y1: 10.272,
                        y2: 41.493,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i8ylg$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $i8ylg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i8ylg$jsxs)("linearGradient", {
                        id: "ea280d__f",
                        x1: 9.519,
                        x2: 59.796,
                        y1: 43.968,
                        y2: 14.091,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i8ylg$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $i8ylg$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $i8ylg$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $i8ylg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i8ylg$jsxs)("radialGradient", {
                        id: "ea280d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.176 -18.2204 0 18.22 17.176)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i8ylg$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $i8ylg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i8ylg$jsxs)("radialGradient", {
                        id: "ea280d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.176 -18.2204 0 18.22 17.176)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i8ylg$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $i8ylg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i8ylg$jsx)("clipPath", {
                        id: "ea280d__b",
                        children: /*#__PURE__*/ (0, $i8ylg$jsx)("rect", {
                            width: 41.881,
                            height: 39.48,
                            x: 0.898,
                            y: 3,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $i8ylg$jsx)("clipPath", {
                id: "ea280d__a",
                children: /*#__PURE__*/ (0, $i8ylg$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M12 7.2v-3c0-.663-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2v3c0 .663.537 1.2 1.2 1.2S12 7.863 12 7.2m11.7 13.806c0 1.65 1.35 2.994 3.009 2.994h10.494v14.4c0 1.989-1.602 3.6-3.579 3.6H14.379c-1.977 0-3.579-1.611-3.579-3.6v-24c0-1.989 1.602-3.6 3.579-3.6H23.7zm12.69.594h-9.678c-.336 0-.61-.267-.61-.594v-9.339l.047.04q.053.043.104.092l9.843 9.477c.105.102.2.21.294.324M4.2 9.6h3c.663 0 1.2.537 1.2 1.2S7.863 12 7.2 12h-3c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.2-1.2"
                })
            }),
            /*#__PURE__*/ (0, $i8ylg$jsx)("g", {
                clipPath: "url(#ea280d__a)",
                children: /*#__PURE__*/ (0, $i8ylg$jsxs)("g", {
                    clipPath: "url(#ea280d__b)",
                    children: [
                        /*#__PURE__*/ (0, $i8ylg$jsx)("rect", {
                            width: 41.881,
                            height: 39.48,
                            x: 0.898,
                            y: 3,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $i8ylg$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M.898 3H42.78v39.48H.9z"
                        }),
                        /*#__PURE__*/ (0, $i8ylg$jsx)("ellipse", {
                            cx: 18.22,
                            cy: 17.176,
                            fill: "url(#ea280d__c)",
                            rx: 18.22,
                            ry: 17.176,
                            transform: "matrix(1 -.00132 .00148 1 6.332 -12.621)"
                        }),
                        /*#__PURE__*/ (0, $i8ylg$jsx)("path", {
                            fill: "url(#ea280d__d)",
                            d: "M16.884-3.973c-3.217-1.877-7.292-1.888-10.52-.03l-32.29 18.585c-3.162 1.82-3.168 6.116-.01 7.943L6.142 41.084c3.223 1.864 7.298 1.862 10.52-.007L48.67 22.5c3.14-1.823 3.146-6.095.009-7.925z"
                        }),
                        /*#__PURE__*/ (0, $i8ylg$jsx)("path", {
                            fill: "url(#ea280d__e)",
                            d: "M6.933 21.744C7.65 7.004 20.904-4.396 36.54-3.722c15.635.675 27.73 13.17 27.014 27.91s-13.971 26.14-29.607 25.465-27.73-13.17-27.014-27.909"
                        }),
                        /*#__PURE__*/ (0, $i8ylg$jsx)("path", {
                            fill: "url(#ea280d__f)",
                            d: "M-16.068 31.463c6.215-2.5 12.918-2.392 18.725-.206 11.62 4.385 18.382 1.674 23.072-9.255 2.335-5.484 6.896-10.129 13.118-12.631 12.44-4.995 26.802.425 32.093 12.117s-.5 25.21-12.937 30.21c-6.236 2.509-12.952 2.391-18.768.185-11.602-4.362-18.35-1.64-23.034 9.302-2.342 5.471-6.902 10.1-13.117 12.6-12.43 4.998-26.791-.422-32.08-12.107-5.288-11.686.498-25.216 12.928-30.215"
                        }),
                        /*#__PURE__*/ (0, $i8ylg$jsx)("ellipse", {
                            cx: 18.22,
                            cy: 17.176,
                            fill: "url(#ea280d__g)",
                            rx: 18.22,
                            ry: 17.176,
                            transform: "matrix(1 -.00132 .00148 1 22.094 -7.02)"
                        })
                    ]
                })
            })
        ]
    });
const $2750d7ea242a4139$var$ForwardRef = /*#__PURE__*/ (0, $i8ylg$forwardRef)($2750d7ea242a4139$var$SvgComponent);
var $2750d7ea242a4139$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2750d7ea242a4139$var$ForwardRef);


export {$2750d7ea242a4139$export$2e2bcd8739ae039 as default};
