// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_logo_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $1kdjU$reactjsxruntime = require("react/jsx-runtime");
var $1kdjU$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d9b6a8f08253e2d5$export$2e2bcd8739ae039);




const $d9b6a8f08253e2d5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0a13d7__d",
                        x1: -18.921,
                        x2: 187.087,
                        y1: 128.357,
                        y2: -4.598,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0a13d7__e",
                        x1: 151.995,
                        x2: 66.38,
                        y1: 40.747,
                        y2: 138.405,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0a13d7__f",
                        x1: 40.959,
                        x2: 198.223,
                        y1: 146.143,
                        y2: 52.689,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsxs)("radialGradient", {
                        id: "0a13d7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsxs)("radialGradient", {
                        id: "0a13d7__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("clipPath", {
                        id: "0a13d7__b",
                        children: /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 18,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("clipPath", {
                id: "0a13d7__a",
                children: /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m85.358 26.74 42.569 21.24c4.07 2.03 6.641 6.19 6.641 10.74v42.46c0 4.54-2.561 8.7-6.621 10.73l-42.57 21.33c-3.38 1.7-7.37 1.7-10.75 0l-42.57-21.33c-4.06-2.04-6.62-6.19-6.62-10.73V58.72c0-4.55 2.57-8.71 6.64-10.74l42.57-21.24c3.37-1.68 7.34-1.68 10.71 0M51.318 97.6c1.39 0 2.73-.72 3.47-2h-.01l14.65-25.37c1.11-1.91.45-4.36-1.46-5.46-1.91-1.11-4.36-.45-5.46 1.46L47.858 91.6c-1.11 1.91-.45 4.36 1.46 5.46.63.37 1.32.54 2 .54m53.34 0c2.89 0 5.479-1.5 6.929-4 1.45-2.51 1.45-5.5 0-8l-13.53-23.43c-1.45-2.5-4.04-4-6.93-4s-5.48 1.5-6.93 4L70.668 85.6c-1.45 2.5-1.45 5.5 0 8s4.04 4 6.93 4zm-13.52-31.43 13.529 23.43h-27.06z"
                })
            }),
            /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("g", {
                clipPath: "url(#0a13d7__a)",
                children: /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#0a13d7__b)",
                    children: [
                        /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 18,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M14 18h131v123.491H14z"
                        }),
                        /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#0a13d7__c)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 31 -30.86)"
                        }),
                        /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("path", {
                            fill: "url(#0a13d7__d)",
                            d: "M64.001-3.814c-10.06-5.868-22.807-5.904-32.905-.092l-101 58.131c-9.891 5.694-9.907 19.133-.03 24.847L30.4 137.122c10.082 5.833 22.83 5.826 32.904-.021l100.125-58.104c9.826-5.703 9.844-19.066.03-24.79z"
                        }),
                        /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("path", {
                            fill: "url(#0a13d7__e)",
                            d: "M32.878 76.631c2.24-46.103 43.701-81.765 92.607-79.655 48.907 2.111 86.738 41.196 84.499 87.299-2.239 46.102-43.701 81.765-92.607 79.654-48.906-2.11-86.738-41.195-84.499-87.298"
                        }),
                        /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("path", {
                            fill: "url(#0a13d7__f)",
                            d: "M-39.075 107.029c19.44-7.818 40.406-7.483 58.57-.645 36.349 13.717 57.498 5.236 72.167-28.95 7.307-17.153 21.572-31.681 41.035-39.508 38.911-15.625 83.833 1.329 100.384 37.902s-1.565 78.852-40.467 94.497c-19.506 7.844-40.512 7.477-58.705.577-36.288-13.644-57.398-5.131-72.048 29.096-7.325 17.113-21.588 31.591-41.028 39.409-38.88 15.636-83.803-1.317-100.344-37.869-16.542-36.553 1.555-78.874 40.436-94.509"
                        }),
                        /*#__PURE__*/ (0, $1kdjU$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#0a13d7__g)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 80.293 -13.34)"
                        })
                    ]
                })
            })
        ]
    });
const $d9b6a8f08253e2d5$var$ForwardRef = /*#__PURE__*/ (0, $1kdjU$react.forwardRef)($d9b6a8f08253e2d5$var$SvgComponent);
var $d9b6a8f08253e2d5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d9b6a8f08253e2d5$var$ForwardRef);
