// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_piggyBank_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $hVBtE$jsxs, jsx as $hVBtE$jsx} from "react/jsx-runtime";
import {forwardRef as $hVBtE$forwardRef} from "react";





const $6bfe4464b873b97d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hVBtE$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hVBtE$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hVBtE$jsxs)("linearGradient", {
                        id: "acf84a__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hVBtE$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $hVBtE$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hVBtE$jsxs)("linearGradient", {
                        id: "acf84a__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hVBtE$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hVBtE$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hVBtE$jsxs)("linearGradient", {
                        id: "acf84a__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hVBtE$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hVBtE$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hVBtE$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hVBtE$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hVBtE$jsxs)("radialGradient", {
                        id: "acf84a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hVBtE$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $hVBtE$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hVBtE$jsxs)("radialGradient", {
                        id: "acf84a__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hVBtE$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hVBtE$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hVBtE$jsx)("clipPath", {
                        id: "acf84a__b",
                        children: /*#__PURE__*/ (0, $hVBtE$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hVBtE$jsx)("clipPath", {
                id: "acf84a__a",
                children: /*#__PURE__*/ (0, $hVBtE$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M65.762 70.376c9.658-5.116 16.08-14.184 16.08-24.519q0-1.518-.183-2.997c1.303-.13 2.844-.64 4.207-1.78 1.823-1.523 3.119-3.995 3.283-7.608.05-1.103-.804-2.038-1.907-2.088-1.104-.05-2.039.804-2.089 1.907-.122 2.693-1.033 4.036-1.852 4.72-.876.733-1.888.914-2.46.881-3.693-12.61-17.187-21.96-33.268-21.96-6.543 0-12.657 1.548-17.857 4.233-3.089-3.312-7.926-4.273-11.676-4.23-1.392.017-2.115 1.527-1.418 2.731l4.507 7.784.032-.024c-2.358 2.405-4.264 5.137-5.607 8.104v-.017l-6.716 3.913c-1.23.716-1.986 2.032-1.986 3.456v5.952c0 1.424.757 2.74 1.987 3.457l6.715 3.908v-.014c2.667 5.89 7.55 10.854 13.788 14.168l-.042-.003 2.48 6.752c.433 1.18 1.557 1.965 2.815 1.965h3.434c1.257 0 2.381-.783 2.815-1.963l.997-2.707-.089-.03c1.892.273 3.837.415 5.82.415 1.941 0 3.844-.136 5.697-.398l-.04.013.998 2.707c.434 1.18 1.558 1.963 2.815 1.963h3.434c1.258 0 2.382-.784 2.816-1.965zm-13.231-43.23c-3.156-.702-5.783-.732-9.12.002-1.08.237-2.146-.446-2.383-1.524-.237-1.08.445-2.146 1.524-2.383 3.912-.86 7.129-.827 10.848.001 1.078.24 1.758 1.309 1.518 2.387s-1.31 1.758-2.387 1.517m-22.81 8.236c0 1.657-1.343 3-3 3s-3-1.343-3-3 1.343-3 3-3 3 1.343 3 3"
                })
            }),
            /*#__PURE__*/ (0, $hVBtE$jsx)("g", {
                clipPath: "url(#acf84a__a)",
                children: /*#__PURE__*/ (0, $hVBtE$jsxs)("g", {
                    clipPath: "url(#acf84a__b)",
                    children: [
                        /*#__PURE__*/ (0, $hVBtE$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $hVBtE$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $hVBtE$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#acf84a__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $hVBtE$jsx)("path", {
                            fill: "url(#acf84a__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $hVBtE$jsx)("path", {
                            fill: "url(#acf84a__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $hVBtE$jsx)("path", {
                            fill: "url(#acf84a__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $hVBtE$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#acf84a__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $6bfe4464b873b97d$var$ForwardRef = /*#__PURE__*/ (0, $hVBtE$forwardRef)($6bfe4464b873b97d$var$SvgComponent);
var $6bfe4464b873b97d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6bfe4464b873b97d$var$ForwardRef);


export {$6bfe4464b873b97d$export$2e2bcd8739ae039 as default};
