// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_volumeOne_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $b5DCP$jsxs, jsx as $b5DCP$jsx} from "react/jsx-runtime";
import {forwardRef as $b5DCP$forwardRef} from "react";





const $40d93b9e18d00ed4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $b5DCP$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $b5DCP$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $b5DCP$jsxs)("linearGradient", {
                        id: "c432dc__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b5DCP$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $b5DCP$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b5DCP$jsxs)("linearGradient", {
                        id: "c432dc__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b5DCP$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $b5DCP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b5DCP$jsxs)("linearGradient", {
                        id: "c432dc__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b5DCP$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $b5DCP$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $b5DCP$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $b5DCP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b5DCP$jsxs)("radialGradient", {
                        id: "c432dc__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b5DCP$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $b5DCP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b5DCP$jsxs)("radialGradient", {
                        id: "c432dc__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b5DCP$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $b5DCP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b5DCP$jsx)("clipPath", {
                        id: "c432dc__b",
                        children: /*#__PURE__*/ (0, $b5DCP$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $b5DCP$jsx)("clipPath", {
                id: "c432dc__a",
                children: /*#__PURE__*/ (0, $b5DCP$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M46.629 52.997h12.65v54.205h-12.65c-4.418 0-8-3.582-8-8V60.997c0-4.418 3.582-8 8-8m20.65 59.023V48.13l15.24-15.11c3.787-3.754 10.224-1.072 10.224 4.261v85.437c0 5.318-6.407 8.007-10.203 4.282zm42.384-55.58c-1.766-1.328-4.273-.974-5.602.792-1.328 1.765-.974 4.273.791 5.6 5.183 3.9 8.521 10.09 8.521 17.061 0 7.028-3.393 13.261-8.648 17.155-1.775 1.316-2.148 3.821-.832 5.596 1.315 1.775 3.82 2.147 5.595.832 7.203-5.338 11.885-13.914 11.885-23.583 0-9.59-4.606-18.106-11.71-23.452"
                })
            }),
            /*#__PURE__*/ (0, $b5DCP$jsx)("g", {
                clipPath: "url(#c432dc__a)",
                children: /*#__PURE__*/ (0, $b5DCP$jsxs)("g", {
                    clipPath: "url(#c432dc__b)",
                    children: [
                        /*#__PURE__*/ (0, $b5DCP$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $b5DCP$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $b5DCP$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#c432dc__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $b5DCP$jsx)("path", {
                            fill: "url(#c432dc__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $b5DCP$jsx)("path", {
                            fill: "url(#c432dc__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $b5DCP$jsx)("path", {
                            fill: "url(#c432dc__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $b5DCP$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#c432dc__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $40d93b9e18d00ed4$var$ForwardRef = /*#__PURE__*/ (0, $b5DCP$forwardRef)($40d93b9e18d00ed4$var$SvgComponent);
var $40d93b9e18d00ed4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($40d93b9e18d00ed4$var$ForwardRef);


export {$40d93b9e18d00ed4$export$2e2bcd8739ae039 as default};
