// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_data_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $h9VQW$jsxs, jsx as $h9VQW$jsx} from "react/jsx-runtime";
import {forwardRef as $h9VQW$forwardRef} from "react";





const $bc787fd591e54415$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $h9VQW$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $h9VQW$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $h9VQW$jsxs)("linearGradient", {
                        id: "8a2177__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h9VQW$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $h9VQW$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h9VQW$jsxs)("linearGradient", {
                        id: "8a2177__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h9VQW$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h9VQW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h9VQW$jsxs)("linearGradient", {
                        id: "8a2177__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h9VQW$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $h9VQW$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $h9VQW$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $h9VQW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h9VQW$jsxs)("radialGradient", {
                        id: "8a2177__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h9VQW$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $h9VQW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h9VQW$jsxs)("radialGradient", {
                        id: "8a2177__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h9VQW$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h9VQW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h9VQW$jsx)("clipPath", {
                        id: "8a2177__b",
                        children: /*#__PURE__*/ (0, $h9VQW$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $h9VQW$jsx)("clipPath", {
                id: "8a2177__a",
                children: /*#__PURE__*/ (0, $h9VQW$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M35.97 17.516c1.21-.375 2.295-.821 3.182-1.342v2.232c-.02.146-.15.469-.776.928-.674.494-1.73.992-3.146 1.43C32.41 21.64 28.44 22.2 24 22.2s-8.412-.56-11.23-1.434c-1.418-.44-2.473-.937-3.148-1.431-.623-.457-.753-.779-.774-.925v-2.235c.886.52 1.971.967 3.18 1.342 3.12.967 7.352 1.546 11.971 1.546 4.62 0 8.852-.58 11.97-1.546m3.181-4.743c-.048-2.156-6.813-3.899-15.152-3.899S8.896 10.617 8.848 12.773c.021.146.152.468.774.924.675.494 1.73.992 3.147 1.431 2.82.874 6.79 1.434 11.23 1.434s8.412-.56 11.23-1.434c1.418-.44 2.473-.937 3.148-1.43.621-.457.752-.778.774-.925m0 9.037c-.886.521-1.971.968-3.181 1.343-3.119.967-7.35 1.546-11.97 1.546s-8.852-.58-11.971-1.546c-1.21-.375-2.295-.822-3.181-1.343v2.235c.02.147.151.47.774.926.675.494 1.73.991 3.147 1.43 2.82.875 6.79 1.434 11.23 1.434s8.412-.56 11.23-1.433c1.418-.44 2.473-.937 3.148-1.431.626-.46.754-.782.774-.928zM8.848 29.713v-2.266c.886.521 1.971.968 3.18 1.343 3.12.966 7.352 1.545 11.971 1.545 4.62 0 8.852-.579 11.97-1.546 1.21-.375 2.296-.821 3.182-1.342v2.264c-.02.146-.148.469-.775.928-.674.494-1.73.991-3.146 1.43-2.82.874-6.79 1.434-11.23 1.434s-8.412-.56-11.23-1.433c-1.418-.44-2.473-.937-3.148-1.431-.623-.457-.753-.78-.774-.926m0 3.402v2.212c.402 2 7.598 3.798 15.002 3.798 5.842 0 13.916-1.094 15.301-3.735v-2.275c-.886.521-1.971.967-3.181 1.342-3.119.967-7.35 1.546-11.97 1.546s-8.852-.579-11.971-1.546c-1.21-.375-2.295-.82-3.181-1.342"
                })
            }),
            /*#__PURE__*/ (0, $h9VQW$jsx)("g", {
                clipPath: "url(#8a2177__a)",
                children: /*#__PURE__*/ (0, $h9VQW$jsxs)("g", {
                    clipPath: "url(#8a2177__b)",
                    children: [
                        /*#__PURE__*/ (0, $h9VQW$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $h9VQW$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $h9VQW$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#8a2177__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $h9VQW$jsx)("path", {
                            fill: "url(#8a2177__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $h9VQW$jsx)("path", {
                            fill: "url(#8a2177__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $h9VQW$jsx)("path", {
                            fill: "url(#8a2177__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $h9VQW$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#8a2177__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $bc787fd591e54415$var$ForwardRef = /*#__PURE__*/ (0, $h9VQW$forwardRef)($bc787fd591e54415$var$SvgComponent);
var $bc787fd591e54415$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($bc787fd591e54415$var$ForwardRef);


export {$bc787fd591e54415$export$2e2bcd8739ae039 as default};
