// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_folderOpen_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $dJBpe$jsxs, jsx as $dJBpe$jsx} from "react/jsx-runtime";
import {forwardRef as $dJBpe$forwardRef} from "react";





const $23862386e96bbe63$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dJBpe$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dJBpe$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dJBpe$jsxs)("linearGradient", {
                        id: "92ffa5__d",
                        x1: -32.728,
                        x2: 209.883,
                        y1: 133.353,
                        y2: -14.249,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dJBpe$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $dJBpe$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dJBpe$jsxs)("linearGradient", {
                        id: "92ffa5__e",
                        x1: 161.965,
                        x2: 57.886,
                        y1: 27.487,
                        y2: 139.401,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dJBpe$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dJBpe$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dJBpe$jsxs)("linearGradient", {
                        id: "92ffa5__f",
                        x1: 35.482,
                        x2: 219.987,
                        y1: 154.848,
                        y2: 51.491,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dJBpe$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dJBpe$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dJBpe$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dJBpe$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dJBpe$jsxs)("radialGradient", {
                        id: "92ffa5__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 41.68 47.468)scale(64.9209)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dJBpe$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $dJBpe$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dJBpe$jsxs)("radialGradient", {
                        id: "92ffa5__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 59.17 86.13)scale(64.9209)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dJBpe$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dJBpe$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dJBpe$jsx)("clipPath", {
                        id: "92ffa5__b",
                        children: /*#__PURE__*/ (0, $dJBpe$jsx)("rect", {
                            width: 149.225,
                            height: 149.225,
                            x: 4.773,
                            fill: "#fff",
                            rx: 3.731
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dJBpe$jsx)("clipPath", {
                id: "92ffa5__a",
                children: /*#__PURE__*/ (0, $dJBpe$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M129.5 68v-4.364c0-7.38-5.992-13.363-13.385-13.363H69.269l-15.167-18.93C52.41 29.23 49.845 28 47.135 28H22.423c-4.928 0-8.923 3.989-8.923 8.91v81.726c0 1.264.176 2.487.504 3.646.1-1.703.47-3.442 1.147-5.174l.055-.138 15.788-38.276C33.586 72.16 39.949 68 46.876 68zm-98.435 64h98.059q.357 0 .71-.027c3.466-.265 6.498-2.453 7.752-5.66l15.811-38.333c2.119-5.42-1.597-11.207-7.297-11.909q-.57-.07-1.165-.071H46.876c-3.758 0-7.125 2.263-8.463 5.687L22.602 120.02c-2.26 5.782 2.118 11.98 8.462 11.98"
                })
            }),
            /*#__PURE__*/ (0, $dJBpe$jsx)("g", {
                clipPath: "url(#92ffa5__a)",
                children: /*#__PURE__*/ (0, $dJBpe$jsxs)("g", {
                    clipPath: "url(#92ffa5__b)",
                    children: [
                        /*#__PURE__*/ (0, $dJBpe$jsx)("rect", {
                            width: 149.225,
                            height: 149.225,
                            x: 4.773,
                            fill: "#FFECCF",
                            rx: 3.731
                        }),
                        /*#__PURE__*/ (0, $dJBpe$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M4.773 0h149.225v149.225H4.773z"
                        }),
                        /*#__PURE__*/ (0, $dJBpe$jsx)("circle", {
                            cx: 89.148,
                            cy: 5.788,
                            r: 64.921,
                            fill: "url(#92ffa5__c)",
                            transform: "rotate(-.08 89.148 5.788)"
                        }),
                        /*#__PURE__*/ (0, $dJBpe$jsx)("path", {
                            fill: "url(#92ffa5__d)",
                            d: "M61.73-26.36c-11.46-7.091-25.98-7.135-37.484-.112l-115.05 70.246c-11.267 6.88-11.286 23.118-.034 30.024l114.29 70.146c11.486 7.049 26.006 7.04 37.482-.025l114.054-70.211c11.193-6.891 11.213-23.04.034-29.957z"
                        }),
                        /*#__PURE__*/ (0, $dJBpe$jsx)("path", {
                            fill: "url(#92ffa5__e)",
                            d: "M26.277 70.849c2.55-55.71 49.78-98.804 105.49-96.254s98.804 49.78 96.253 105.49c-2.55 55.71-49.779 98.804-105.489 96.254-55.71-2.551-98.805-49.78-96.254-105.49"
                        }),
                        /*#__PURE__*/ (0, $dJBpe$jsx)("path", {
                            fill: "url(#92ffa5__f)",
                            d: "M-55.685 107.582c22.144-9.447 46.027-9.042 66.719-.779 41.404 16.576 65.496 6.327 82.206-34.982 8.323-20.729 24.573-38.284 46.743-47.742 44.324-18.88 95.496 1.606 114.349 45.8s-1.782 95.284-46.097 114.189c-22.219 9.479-46.148 9.035-66.871.698-41.337-16.488-65.383-6.2-82.072 35.158-8.344 20.68-24.59 38.175-46.735 47.622-44.29 18.895-95.461-1.592-114.304-45.761s1.772-95.309 46.062-114.203"
                        }),
                        /*#__PURE__*/ (0, $dJBpe$jsx)("circle", {
                            cx: 145.301,
                            cy: 26.96,
                            r: 64.921,
                            fill: "url(#92ffa5__g)",
                            transform: "rotate(-.08 145.301 26.96)"
                        })
                    ]
                })
            })
        ]
    });
const $23862386e96bbe63$var$ForwardRef = /*#__PURE__*/ (0, $dJBpe$forwardRef)($23862386e96bbe63$var$SvgComponent);
var $23862386e96bbe63$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($23862386e96bbe63$var$ForwardRef);


export {$23862386e96bbe63$export$2e2bcd8739ae039 as default};
