// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_remix_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $fRQbH$jsxs, jsx as $fRQbH$jsx} from "react/jsx-runtime";
import {forwardRef as $fRQbH$forwardRef} from "react";





const $be3f79ee28b4a064$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fRQbH$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fRQbH$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fRQbH$jsxs)("linearGradient", {
                        id: "13f63f__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.982,
                        y2: -15.278,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fRQbH$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $fRQbH$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fRQbH$jsxs)("linearGradient", {
                        id: "13f63f__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fRQbH$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fRQbH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fRQbH$jsxs)("linearGradient", {
                        id: "13f63f__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fRQbH$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fRQbH$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fRQbH$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fRQbH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fRQbH$jsxs)("radialGradient", {
                        id: "13f63f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fRQbH$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $fRQbH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fRQbH$jsxs)("radialGradient", {
                        id: "13f63f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fRQbH$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fRQbH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fRQbH$jsx)("clipPath", {
                        id: "13f63f__b",
                        children: /*#__PURE__*/ (0, $fRQbH$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fRQbH$jsx)("clipPath", {
                id: "13f63f__a",
                children: /*#__PURE__*/ (0, $fRQbH$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M129.934 67.666 148.52 49.08c1.562-1.562 1.562-4.094 0-5.657l-18.586-18.585c-1.562-1.562-4.095-1.562-5.657 0s-1.562 4.094 0 5.656l11.402 11.403h-7.178c-8.729 0-16.783 2.94-23.21 7.882q-.147.113-.279.236-.064.046-.127.095c-4.735 3.718-9.15 9.122-12.607 14.075-3.471 4.976-6.188 9.807-7.458 12.609-.993 2.193-3.454 7.406-6.71 12.882-2.803 4.713-6.005 9.29-9.182 12.376q-.145.11-.282.234c-5.349 4.86-12.443 7.817-20.238 7.817-16.626 0-30.103-13.477-30.103-30.103s13.477-30.103 30.103-30.103c10.167 0 19.158 5.036 24.614 12.767l.007.01.008.01 2.081 2.911c1.285 1.797 3.783 2.213 5.58.928s2.213-3.784.928-5.58L79.55 58.04C72.66 48.282 61.28 41.897 48.408 41.897c-21.044 0-38.103 17.059-38.103 38.103s17.06 38.103 38.103 38.103c8.72 0 16.764-2.934 23.185-7.863q.273-.153.527-.353c4.965-3.915 9.466-10.406 12.866-16.122 3.465-5.828 6.063-11.334 7.121-13.67 1.023-2.256 3.471-6.658 6.732-11.332 3.276-4.696 7.161-9.357 10.986-12.36q.13-.102.248-.211l.095-.071c5.077-3.905 11.429-6.224 18.333-6.224h7.889l-12.113 12.112c-1.562 1.563-1.562 4.095 0 5.657s4.095 1.562 5.657 0m8.019 50.437c2.21 0 4-1.791 4-4s-1.79-4-4-4h-9.452c-9.961 0-18.039-5.729-25.113-13.446l-2.717-3.283c-1.408-1.702-3.93-1.94-5.631-.531s-1.94 3.93-.532 5.632l2.78 3.36.063.076.067.073c7.629 8.342 17.72 16.119 31.083 16.119z"
                })
            }),
            /*#__PURE__*/ (0, $fRQbH$jsx)("g", {
                clipPath: "url(#13f63f__a)",
                children: /*#__PURE__*/ (0, $fRQbH$jsxs)("g", {
                    clipPath: "url(#13f63f__b)",
                    children: [
                        /*#__PURE__*/ (0, $fRQbH$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $fRQbH$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $fRQbH$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#13f63f__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $fRQbH$jsx)("path", {
                            fill: "url(#13f63f__d)",
                            d: "M61.07-28.264c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.549 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $fRQbH$jsx)("path", {
                            fill: "url(#13f63f__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $fRQbH$jsx)("path", {
                            fill: "url(#13f63f__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $fRQbH$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#13f63f__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $be3f79ee28b4a064$var$ForwardRef = /*#__PURE__*/ (0, $fRQbH$forwardRef)($be3f79ee28b4a064$var$SvgComponent);
var $be3f79ee28b4a064$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($be3f79ee28b4a064$var$ForwardRef);


export {$be3f79ee28b4a064$export$2e2bcd8739ae039 as default};
