// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_megaphonePromote_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $9K2i0$jsxs, jsx as $9K2i0$jsx} from "react/jsx-runtime";
import {forwardRef as $9K2i0$forwardRef} from "react";





const $eb8473e6ed3d1c0b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9K2i0$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9K2i0$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9K2i0$jsxs)("linearGradient", {
                        id: "24a4e3__d",
                        x1: -13.164,
                        x2: 184.35,
                        y1: 129.007,
                        y2: 1.534,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9K2i0$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $9K2i0$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9K2i0$jsxs)("linearGradient", {
                        id: "24a4e3__e",
                        x1: 150.707,
                        x2: 68.621,
                        y1: 45.009,
                        y2: 138.642,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9K2i0$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9K2i0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9K2i0$jsxs)("linearGradient", {
                        id: "24a4e3__f",
                        x1: 44.245,
                        x2: 195.026,
                        y1: 146.062,
                        y2: 56.46,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9K2i0$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9K2i0$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9K2i0$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9K2i0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9K2i0$jsxs)("radialGradient", {
                        id: "24a4e3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.5105 -54.6429 0 54.643 51.51)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9K2i0$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $9K2i0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9K2i0$jsxs)("radialGradient", {
                        id: "24a4e3__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.5105 -54.6429 0 54.643 51.51)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9K2i0$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9K2i0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9K2i0$jsx)("clipPath", {
                        id: "24a4e3__b",
                        children: /*#__PURE__*/ (0, $9K2i0$jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.2,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9K2i0$jsx)("clipPath", {
                id: "24a4e3__a",
                children: /*#__PURE__*/ (0, $9K2i0$jsx)("path", {
                    fill: "#fff",
                    d: "m119.099 31.17-43.1 13.08H45.02c-11.59 0-20.98 9.39-20.98 20.98s9.39 20.98 20.98 20.98h5.94l15.72 36.17c2.5 5.78 9.22 8.44 15 5.94s8.44-9.22 5.94-15l-11.67-27.1h.04l43.09 13.11c6.42 1.95 12.91-2.85 12.91-9.57V40.74c0-6.71-6.48-11.52-12.9-9.57z"
                })
            }),
            /*#__PURE__*/ (0, $9K2i0$jsx)("g", {
                clipPath: "url(#24a4e3__a)",
                children: /*#__PURE__*/ (0, $9K2i0$jsxs)("g", {
                    clipPath: "url(#24a4e3__b)",
                    children: [
                        /*#__PURE__*/ (0, $9K2i0$jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.2,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $9K2i0$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M18.398 23.2h125.6v118.4h-125.6z"
                        }),
                        /*#__PURE__*/ (0, $9K2i0$jsx)("ellipse", {
                            cx: 54.643,
                            cy: 51.511,
                            fill: "url(#24a4e3__c)",
                            rx: 54.643,
                            ry: 51.511,
                            transform: "matrix(1 -.00132 .00148 1 34.695 -23.645)"
                        }),
                        /*#__PURE__*/ (0, $9K2i0$jsx)("path", {
                            fill: "url(#24a4e3__d)",
                            d: "M66.34 2.285c-9.647-5.626-21.868-5.661-31.55-.089l-96.836 55.736c-9.483 5.458-9.5 18.343-.028 23.822l96.195 55.657c9.668 5.592 21.89 5.585 31.548-.02l95.998-55.709c9.421-5.467 9.438-18.28.029-23.768z"
                        }),
                        /*#__PURE__*/ (0, $9K2i0$jsx)("path", {
                            fill: "url(#24a4e3__e)",
                            d: "M36.5 79.414c2.147-44.202 41.9-78.395 88.79-76.371 46.89 2.023 83.162 39.497 81.015 83.7-2.147 44.201-41.899 78.394-88.789 76.37s-83.162-39.497-81.015-83.7"
                        }),
                        /*#__PURE__*/ (0, $9K2i0$jsx)("path", {
                            fill: "url(#24a4e3__f)",
                            d: "M-32.49 108.559c18.64-7.495 38.741-7.174 56.157-.618 34.85 13.152 55.127 5.021 69.191-27.755 7.006-16.448 20.683-30.377 39.343-37.88 37.308-14.981 80.378 1.274 96.246 36.339 15.869 35.065-1.5 75.602-38.799 90.601-18.701 7.521-38.842 7.169-56.284.554-34.793-13.082-55.032-4.92-69.079 27.896-7.023 16.408-20.697 30.289-39.336 37.785-37.278 14.991-80.348-1.263-96.208-36.309s1.492-75.621 38.77-90.613"
                        }),
                        /*#__PURE__*/ (0, $9K2i0$jsx)("ellipse", {
                            cx: 54.643,
                            cy: 51.511,
                            fill: "url(#24a4e3__g)",
                            rx: 54.643,
                            ry: 51.511,
                            transform: "matrix(1 -.00132 .00148 1 81.957 -6.848)"
                        })
                    ]
                })
            })
        ]
    });
const $eb8473e6ed3d1c0b$var$ForwardRef = /*#__PURE__*/ (0, $9K2i0$forwardRef)($eb8473e6ed3d1c0b$var$SvgComponent);
var $eb8473e6ed3d1c0b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($eb8473e6ed3d1c0b$var$ForwardRef);


export {$eb8473e6ed3d1c0b$export$2e2bcd8739ae039 as default};
