// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_mention_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $98CT8$jsxs, jsx as $98CT8$jsx} from "react/jsx-runtime";
import {forwardRef as $98CT8$forwardRef} from "react";





const $83de6c33f251ef57$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $98CT8$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $98CT8$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $98CT8$jsxs)("linearGradient", {
                        id: "c86ca2__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98CT8$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $98CT8$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98CT8$jsxs)("linearGradient", {
                        id: "c86ca2__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98CT8$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $98CT8$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98CT8$jsxs)("linearGradient", {
                        id: "c86ca2__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98CT8$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $98CT8$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $98CT8$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $98CT8$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98CT8$jsxs)("radialGradient", {
                        id: "c86ca2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98CT8$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $98CT8$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98CT8$jsxs)("radialGradient", {
                        id: "c86ca2__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98CT8$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $98CT8$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98CT8$jsx)("clipPath", {
                        id: "c86ca2__b",
                        children: /*#__PURE__*/ (0, $98CT8$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $98CT8$jsx)("clipPath", {
                id: "c86ca2__a",
                children: /*#__PURE__*/ (0, $98CT8$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M18.967 7.153c4.91-1.61 10.173-1.073 13.6.474 7.03 3.171 9.35 9.989 8.323 15.826-1.018 5.785-5.446 11.009-12.158 10.544-6.154-.425-9.478-2.762-10.826-5.84-1.294-2.956-.575-6.197.47-8.127 2.7-4.988 7.322-5.749 10.168-5.658 1.875.06 3.001 1.754 2.852 3.374l-1.265 13.759c4.393-.304 7.497-3.936 8.297-8.485.864-4.91-1.073-10.49-6.889-13.114-2.858-1.29-7.477-1.792-11.793-.377-4.242 1.39-8.199 4.632-9.91 10.91-1.352 4.965.228 9.722 3.274 13.25 3.065 3.55 7.507 5.728 11.657 5.609.69-.02 1.265.523 1.285 1.213s-.523 1.266-1.213 1.286c-5.029.145-10.155-2.458-13.621-6.474-3.487-4.038-5.403-9.632-3.793-15.542C9.37 12.637 13.983 8.788 18.967 7.153m8.665 24.228 1.275-13.864c.018-.2-.044-.373-.134-.486q-.124-.157-.308-.16c-2.372-.076-5.828.54-7.89 4.35-.794 1.466-1.278 3.88-.379 5.934.78 1.779 2.762 3.64 7.436 4.226"
                })
            }),
            /*#__PURE__*/ (0, $98CT8$jsx)("g", {
                clipPath: "url(#c86ca2__a)",
                children: /*#__PURE__*/ (0, $98CT8$jsxs)("g", {
                    clipPath: "url(#c86ca2__b)",
                    children: [
                        /*#__PURE__*/ (0, $98CT8$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $98CT8$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $98CT8$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#c86ca2__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $98CT8$jsx)("path", {
                            fill: "url(#c86ca2__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $98CT8$jsx)("path", {
                            fill: "url(#c86ca2__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $98CT8$jsx)("path", {
                            fill: "url(#c86ca2__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $98CT8$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#c86ca2__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $83de6c33f251ef57$var$ForwardRef = /*#__PURE__*/ (0, $98CT8$forwardRef)($83de6c33f251ef57$var$SvgComponent);
var $83de6c33f251ef57$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($83de6c33f251ef57$var$ForwardRef);


export {$83de6c33f251ef57$export$2e2bcd8739ae039 as default};
