// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_segmentation_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $hNKF6$jsxs, jsx as $hNKF6$jsx} from "react/jsx-runtime";
import {forwardRef as $hNKF6$forwardRef} from "react";





const $fcb50903d25b6916$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hNKF6$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hNKF6$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hNKF6$jsxs)("linearGradient", {
                        id: "d2a8b7__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.896,
                        y2: -4.582,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hNKF6$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $hNKF6$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hNKF6$jsxs)("linearGradient", {
                        id: "d2a8b7__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.843,
                        y2: 44.841,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hNKF6$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hNKF6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hNKF6$jsxs)("linearGradient", {
                        id: "d2a8b7__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.81,
                        y2: 16.564,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hNKF6$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hNKF6$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hNKF6$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hNKF6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hNKF6$jsxs)("radialGradient", {
                        id: "d2a8b7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.503)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hNKF6$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $hNKF6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hNKF6$jsxs)("radialGradient", {
                        id: "d2a8b7__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.264 26.937)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hNKF6$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hNKF6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hNKF6$jsx)("clipPath", {
                        id: "d2a8b7__b",
                        children: /*#__PURE__*/ (0, $hNKF6$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hNKF6$jsx)("clipPath", {
                id: "d2a8b7__a",
                children: /*#__PURE__*/ (0, $hNKF6$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M17.944 12.125c0-1.657 1.344-3 3-3h15.561c1.657 0 3 1.343 3 3v5.302c0 1.656-1.343 3-3 3h-15.56c-1.657 0-3-1.344-3-3v-1.535c-.163.077-.344.12-.535.12h-4.668c-.966 0-1.75.783-1.75 1.75v12.462c0 .966.784 1.75 1.75 1.75h4.668c.191 0 .372.043.534.12v-1.52c0-1.657 1.344-3 3-3h15.561c1.657 0 3 1.343 3 3v5.302c0 1.657-1.343 3-3 3h-15.56c-1.657 0-3-1.343-3-3v-1.522c-.163.077-.344.12-.535.12h-4.668c-2.347 0-4.25-1.903-4.25-4.25V17.762c0-2.347 1.903-4.25 4.25-4.25h4.668c.191 0 .372.043.534.12z"
                })
            }),
            /*#__PURE__*/ (0, $hNKF6$jsx)("g", {
                clipPath: "url(#d2a8b7__a)",
                children: /*#__PURE__*/ (0, $hNKF6$jsxs)("g", {
                    clipPath: "url(#d2a8b7__b)",
                    children: [
                        /*#__PURE__*/ (0, $hNKF6$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $hNKF6$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 .001h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $hNKF6$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.863,
                            r: 20.883,
                            fill: "url(#d2a8b7__c)",
                            transform: "rotate(-.08 27.142 1.863)"
                        }),
                        /*#__PURE__*/ (0, $hNKF6$jsx)("path", {
                            fill: "url(#d2a8b7__d)",
                            d: "M18.321-8.478c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.657L6.008 46.304c3.694 2.267 8.365 2.264 12.056-.008L54.752 23.71c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $hNKF6$jsx)("path", {
                            fill: "url(#d2a8b7__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.99 40.85-8.17S72.63 7.84 71.809 25.76 55.798 57.543 37.879 56.722C19.957 55.902 6.095 40.71 6.915 22.79"
                        }),
                        /*#__PURE__*/ (0, $hNKF6$jsx)("path", {
                            fill: "url(#d2a8b7__f)",
                            d: "M-19.448 34.606c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.331 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.649-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.279-15.033 15.318-14.246 6.077-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $hNKF6$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.673,
                            r: 20.883,
                            fill: "url(#d2a8b7__g)",
                            transform: "rotate(-.08 45.2 8.673)"
                        })
                    ]
                })
            })
        ]
    });
const $fcb50903d25b6916$var$ForwardRef = /*#__PURE__*/ (0, $hNKF6$forwardRef)($fcb50903d25b6916$var$SvgComponent);
var $fcb50903d25b6916$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($fcb50903d25b6916$var$ForwardRef);


export {$fcb50903d25b6916$export$2e2bcd8739ae039 as default};
