// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_apps_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $9p9c6$reactjsxruntime = require("react/jsx-runtime");
var $9p9c6$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $680e8998839ba685$export$2e2bcd8739ae039);




const $680e8998839ba685$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5a1465__d",
                        x1: -5.676,
                        x2: 56.126,
                        y1: 39.707,
                        y2: -0.18,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5a1465__e",
                        x1: 45.597,
                        x2: 19.913,
                        y1: 13.423,
                        y2: 42.721,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5a1465__f",
                        x1: 12.287,
                        x2: 59.466,
                        y1: 45.043,
                        y2: 17.006,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5a1465__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5a1465__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("clipPath", {
                        id: "5a1465__b",
                        children: /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 6.6,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("clipPath", {
                id: "5a1465__a",
                children: /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M27.093 6.504h9.59c2.651 0 4.8 2.149 4.8 4.8v9.59c0 2.652-2.148 4.8-4.8 4.8h-9.59c-2.651 0-4.8-2.148-4.8-4.8v-9.59c0-2.651 2.149-4.8 4.8-4.8m-16.377 5.838h4.956c2.32 0 4.2 1.88 4.2 4.2v4.956c0 2.32-1.88 4.2-4.2 4.2h-4.956c-2.32 0-4.2-1.88-4.2-4.2v-4.956c0-2.32 1.88-4.2 4.2-4.2m4.956 15.798h-4.956c-2.32 0-4.2 1.88-4.2 4.2v4.956c0 2.32 1.88 4.2 4.2 4.2h4.956c2.32 0 4.2-1.88 4.2-4.2V32.34c0-2.32-1.88-4.2-4.2-4.2m10.82 0h4.957c2.32 0 4.2 1.88 4.2 4.2v4.956c0 2.32-1.88 4.2-4.2 4.2h-4.956c-2.32 0-4.2-1.88-4.2-4.2V32.34c0-2.32 1.88-4.2 4.2-4.2"
                })
            }),
            /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("g", {
                clipPath: "url(#5a1465__a)",
                children: /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#5a1465__b)",
                    children: [
                        /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 6.6,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M4.2 6.6h39.3v37.047H4.2z"
                        }),
                        /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#5a1465__c)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 9.3 -8.059)"
                        }),
                        /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("path", {
                            fill: "url(#5a1465__d)",
                            d: "M19.2.056C16.184-1.705 12.36-1.716 9.33.028l-30.3 17.44c-2.968 1.707-2.973 5.739-.01 7.453l30.1 17.415c3.025 1.75 6.849 1.748 9.871-.006L49.03 24.899c2.947-1.711 2.953-5.72.009-7.437z"
                        }),
                        /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("path", {
                            fill: "url(#5a1465__e)",
                            d: "M9.862 24.188c.672-13.83 13.11-24.53 27.782-23.896 14.672.633 26.021 12.359 25.35 26.19-.672 13.83-13.11 24.53-27.782 23.896-14.672-.633-26.022-12.359-25.35-26.19"
                        }),
                        /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("path", {
                            fill: "url(#5a1465__f)",
                            d: "M-11.723 33.308c5.832-2.345 12.122-2.244 17.571-.193 10.905 4.115 17.25 1.57 21.65-8.685 2.192-5.146 6.472-9.504 12.31-11.852 11.674-4.688 25.15.398 30.116 11.37 4.965 10.972-.47 23.656-12.14 28.35-5.852 2.353-12.154 2.242-17.612.172-10.886-4.093-17.219-1.539-21.614 8.73-2.198 5.133-6.476 9.477-12.309 11.822-11.664 4.69-25.14-.395-30.103-11.36-4.962-10.966.467-23.663 12.131-28.354"
                        }),
                        /*#__PURE__*/ (0, $9p9c6$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#5a1465__g)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 24.086 -2.803)"
                        })
                    ]
                })
            })
        ]
    });
const $680e8998839ba685$var$ForwardRef = /*#__PURE__*/ (0, $9p9c6$react.forwardRef)($680e8998839ba685$var$SvgComponent);
var $680e8998839ba685$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($680e8998839ba685$var$ForwardRef);
