// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_lockOpen_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $3cu9D$jsxs, jsx as $3cu9D$jsx} from "react/jsx-runtime";
import {forwardRef as $3cu9D$forwardRef} from "react";





const $861cec183ba16df9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3cu9D$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3cu9D$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3cu9D$jsxs)("linearGradient", {
                        id: "7a9831__d",
                        x1: -7.36,
                        x2: 60.505,
                        y1: 40.432,
                        y2: -0.857,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3cu9D$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $3cu9D$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3cu9D$jsxs)("linearGradient", {
                        id: "7a9831__e",
                        x1: 47.099,
                        x2: 17.985,
                        y1: 10.818,
                        y2: 42.124,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3cu9D$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3cu9D$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3cu9D$jsxs)("linearGradient", {
                        id: "7a9831__f",
                        x1: 11.721,
                        x2: 63.332,
                        y1: 46.444,
                        y2: 17.532,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3cu9D$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3cu9D$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3cu9D$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3cu9D$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3cu9D$jsxs)("radialGradient", {
                        id: "7a9831__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 10.992 15.74)scale(18.1603)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3cu9D$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $3cu9D$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3cu9D$jsxs)("radialGradient", {
                        id: "7a9831__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 15.885 26.555)scale(18.1603)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3cu9D$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3cu9D$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3cu9D$jsx)("clipPath", {
                        id: "7a9831__b",
                        children: /*#__PURE__*/ (0, $3cu9D$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3cu9D$jsx)("clipPath", {
                id: "7a9831__a",
                children: /*#__PURE__*/ (0, $3cu9D$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M16.197 13.65c0-4.308 3.492-7.8 7.8-7.8s7.8 3.492 7.8 7.8c0 .662-.537 1.2-1.2 1.2s-1.2-.538-1.2-1.2c0-2.983-2.418-5.4-5.4-5.4s-5.4 2.417-5.4 5.4v6.3h16.09c2.215 0 4.01 1.637 4.01 3.656v12.187c0 2.02-1.795 3.657-4.01 3.657H13.307c-2.214 0-4.01-1.637-4.01-3.657V23.606c0-2.02 1.796-3.656 4.01-3.656h2.89zm6.105 16.545q.223.225.496.382v2.273c0 .662.537 1.2 1.2 1.2s1.2-.538 1.2-1.2v-2.273c.18-.104.348-.232.496-.382q.168-.164.3-.364.131-.193.22-.412c.06-.148.108-.295.136-.451.032-.152.048-.312.048-.468s-.016-.316-.048-.469c-.028-.151-.076-.303-.136-.447-.06-.148-.132-.284-.22-.416q-.132-.2-.3-.364c-.444-.448-1.064-.704-1.696-.704s-1.252.255-1.696.703q-.168.164-.3.365c-.084.132-.16.268-.22.416-.06.143-.108.296-.136.447-.032.153-.048.313-.048.469s.016.316.048.468c.028.156.076.303.136.451q.092.219.22.412.132.2.3.364"
                })
            }),
            /*#__PURE__*/ (0, $3cu9D$jsx)("g", {
                clipPath: "url(#7a9831__a)",
                children: /*#__PURE__*/ (0, $3cu9D$jsxs)("g", {
                    clipPath: "url(#7a9831__b)",
                    children: [
                        /*#__PURE__*/ (0, $3cu9D$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $3cu9D$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.129 3.129H44.87V44.87H3.129z"
                        }),
                        /*#__PURE__*/ (0, $3cu9D$jsx)("circle", {
                            cx: 26.733,
                            cy: 4.748,
                            r: 18.16,
                            fill: "url(#7a9831__c)",
                            transform: "rotate(-.08 26.733 4.748)"
                        }),
                        /*#__PURE__*/ (0, $3cu9D$jsx)("path", {
                            fill: "url(#7a9831__d)",
                            d: "M19.062-4.245c-3.205-1.984-7.267-1.996-10.485-.031l-32.183 19.65c-3.151 1.924-3.157 6.466-.01 8.398L8.356 43.394c3.213 1.972 7.275 1.97 10.485-.007l31.904-19.64c3.131-1.928 3.137-6.445.01-8.38z"
                        }),
                        /*#__PURE__*/ (0, $3cu9D$jsx)("path", {
                            fill: "url(#7a9831__e)",
                            d: "M9.143 22.948C9.857 7.364 23.068-4.691 38.652-3.977S66.29 9.948 65.577 25.53c-.713 15.584-13.925 27.639-29.509 26.925S8.43 38.531 9.143 22.948"
                        }),
                        /*#__PURE__*/ (0, $3cu9D$jsx)("path", {
                            fill: "url(#7a9831__f)",
                            d: "M-13.781 33.222c6.194-2.642 12.875-2.529 18.663-.218 11.582 4.637 18.321 1.77 22.995-9.785 2.329-5.798 6.874-10.71 13.076-13.355 12.399-5.281 26.713.45 31.987 12.812S72.44 49.33 60.045 54.618c-6.215 2.651-12.909 2.527-18.706.195-11.563-4.612-18.29-1.734-22.958 9.835-2.334 5.785-6.878 10.679-13.073 13.321-12.389 5.285-26.703-.445-31.974-12.8s.496-26.661 12.885-31.947"
                        }),
                        /*#__PURE__*/ (0, $3cu9D$jsx)("circle", {
                            cx: 42.44,
                            cy: 10.67,
                            r: 18.16,
                            fill: "url(#7a9831__g)",
                            transform: "rotate(-.08 42.44 10.67)"
                        })
                    ]
                })
            })
        ]
    });
const $861cec183ba16df9$var$ForwardRef = /*#__PURE__*/ (0, $3cu9D$forwardRef)($861cec183ba16df9$var$SvgComponent);
var $861cec183ba16df9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($861cec183ba16df9$var$ForwardRef);


export {$861cec183ba16df9$export$2e2bcd8739ae039 as default};
