// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_update_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $3V7ny$jsxs, jsx as $3V7ny$jsx} from "react/jsx-runtime";
import {forwardRef as $3V7ny$forwardRef} from "react";





const $6ebce5f5c5a716ef$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3V7ny$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3V7ny$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3V7ny$jsxs)("linearGradient", {
                        id: "adffa5__d",
                        x1: -15.166,
                        x2: 118.557,
                        y1: 76.434,
                        y2: -9.869,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3V7ny$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $3V7ny$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3V7ny$jsxs)("linearGradient", {
                        id: "adffa5__e",
                        x1: 95.779,
                        x2: 40.205,
                        y1: 19.566,
                        y2: 82.957,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3V7ny$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3V7ny$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3V7ny$jsxs)("linearGradient", {
                        id: "adffa5__f",
                        x1: 23.703,
                        x2: 125.786,
                        y1: 87.98,
                        y2: 27.317,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3V7ny$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3V7ny$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3V7ny$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3V7ny$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3V7ny$jsxs)("radialGradient", {
                        id: "adffa5__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.874 -36.9947 0 36.995 34.874)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3V7ny$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $3V7ny$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3V7ny$jsxs)("radialGradient", {
                        id: "adffa5__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.874 -36.9947 0 36.995 34.874)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3V7ny$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3V7ny$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3V7ny$jsx)("clipPath", {
                        id: "adffa5__b",
                        children: /*#__PURE__*/ (0, $3V7ny$jsx)("rect", {
                            width: 85.035,
                            height: 80.16,
                            x: 6.203,
                            y: 4.8,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3V7ny$jsxs)("clipPath", {
                id: "adffa5__a",
                children: [
                    /*#__PURE__*/ (0, $3V7ny$jsx)("path", {
                        fill: "#fff",
                        fillRule: "evenodd",
                        d: "M84.074 25.14c.6-1.182 2.046-1.65 3.228-1.05h.005c1.183.6 1.65 2.046 1.05 3.228L81.77 40.26c-.425.834-1.266 1.314-2.142 1.314-.341 0-.683-.078-1.013-.228l-12.702-5.922c-1.207-.564-1.729-1.992-1.165-3.192s1.993-1.722 3.192-1.158l8.299 3.87c-5.094-10.83-16.194-17.976-28.266-17.976-13.782 0-26.1 9.228-29.958 22.44-.306 1.05-1.267 1.728-2.304 1.728q-.336.002-.673-.096c-1.272-.372-2.004-1.704-1.632-2.976 4.453-15.246 18.667-25.896 34.567-25.896 13.758 0 26.417 8.04 32.387 20.274zm-6.192 31.794c.372-1.272 1.704-2.004 2.976-1.632v.006c1.272.372 2.004 1.704 1.632 2.976C78.038 73.53 63.824 84.18 47.924 84.18c-13.692 0-26.298-7.968-32.304-20.106l-3.69 7.254c-.426.834-1.266 1.314-2.142 1.314-.366 0-.738-.084-1.086-.264-1.182-.6-1.65-2.046-1.05-3.228l6.588-12.948c.588-1.152 1.986-1.632 3.156-1.086l12.702 5.922c1.206.564 1.722 1.992 1.164 3.192-.564 1.2-1.992 1.716-3.192 1.158l-8.37-3.906c5.106 10.788 16.182 17.892 28.224 17.892 13.782 0 26.1-9.228 29.958-22.44"
                    }),
                    /*#__PURE__*/ (0, $3V7ny$jsx)("path", {
                        fill: "#fff",
                        d: "m87.302 24.09-.272.535.128.065h.144zm-3.229 1.05.535.272zm3.235-1.05.271-.535-.128-.065h-.143zm1.05 3.228.534.272zM81.769 40.26l.535.273zm-3.156 1.086-.253.544.005.002zm-12.701-5.922-.255.544h.001zm-1.165-3.192-.543-.255zm3.192-1.158-.255.543h.002zm8.299 3.87-.254.544 1.502.7-.706-1.5zm-58.224 4.464-.576-.168zm-2.976 1.632.17-.575h-.002zm-1.633-2.976.576.168zm66.955-5.622-.54.263.528 1.082.546-1.073zm.498 22.86h.6v-.45l-.432-.126zm-2.976 1.632-.576-.168zm2.976-1.626h-.6v.45l.431.126zm1.632 2.976-.576-.168zm-66.87 5.79.538-.266-.53-1.072-.543 1.066zm-3.69 7.254.534.273V71.6zm-3.228 1.05.276-.533-.005-.002zm-1.05-3.228-.535-.272zm6.588-12.948-.534-.273zm3.156-1.086-.254.544zm12.702 5.922.254-.543zm1.164 3.192.543.255v-.002zm-3.192 1.158-.254.544zm-8.37-3.906.254-.544-1.508-.704.712 1.505zm67.873-37.927c-1.476-.75-3.284-.166-4.034 1.313l1.07.544c.449-.885 1.533-1.238 2.42-.787zm-.265-.065H87.3v1.2h.007zm1.584 4.1c.75-1.476.166-3.284-1.313-4.035l-.543 1.07c.885.45 1.237 1.533.787 2.421zm-6.588 12.942 6.588-12.942-1.07-.544-6.587 12.942zm-2.677 1.642c1.095 0 2.145-.6 2.677-1.641l-1.069-.546c-.32.627-.95.987-1.608.987zm-1.262-.282c.402.183.828.282 1.263.282v-1.2q-.377 0-.766-.174zm-12.707-5.924L78.36 41.89l.507-1.088-12.702-5.922zm-1.454-3.991c-.706 1.503-.05 3.287 1.453 3.99l.509-1.087c-.909-.424-1.297-1.496-.876-2.393zm3.99-1.446c-1.502-.706-3.286-.051-3.99 1.446l1.087.51c.424-.903 1.496-1.292 2.393-.87zm8.297 3.87-8.298-3.87-.507 1.087 8.298 3.87zm-28.52-16.833c11.838 0 22.728 7.01 27.724 17.631l1.085-.51C71.588 23.65 60.278 16.368 47.971 16.368zM18.59 39.576c3.784-12.955 15.867-22.008 29.383-22.008v-1.2c-14.049 0-26.602 9.404-30.534 22.872zm-2.88 2.16c1.297 0 2.498-.846 2.88-2.16l-1.151-.336c-.23.786-.949 1.296-1.728 1.296zm-.842-.12c.281.083.567.12.842.12v-1.2c-.168 0-.338-.023-.5-.071zm-2.037-3.72c-.466 1.59.45 3.255 2.04 3.72l.336-1.152c-.955-.28-1.503-1.279-1.225-2.232zm35.142-26.328c-16.167 0-30.616 10.826-35.142 26.328l1.151.336c4.378-14.99 18.357-25.464 33.99-25.464zm32.927 20.61c-6.07-12.437-18.938-20.61-32.927-20.61v1.2c13.526 0 25.978 7.907 31.848 19.937zm2.64-7.31-3.714 7.302 1.07.544 3.713-7.302zm-2.513 29.858c-1.59-.465-3.255.45-3.72 2.04l1.152.336c.28-.955 1.279-1.503 2.231-1.224zm.432.582v-.006h-1.2v.006zm1.608 3.144c.465-1.59-.45-3.255-2.04-3.72l-.337 1.152c.956.28 1.504 1.279 1.225 2.232zM47.924 84.78c16.166 0 30.615-10.826 35.142-26.328l-1.152-.336c-4.377 14.99-18.356 25.464-33.99 25.464zM15.082 64.34c6.106 12.34 18.919 20.44 32.842 20.44v-1.2c-13.461 0-25.86-7.836-31.766-19.772zm-2.617 7.26 3.69-7.254-1.07-.544-3.69 7.254zm-2.677 1.642c1.094 0 2.144-.6 2.676-1.641l-1.068-.546c-.32.627-.95.987-1.608.987zm-1.362-.331c.437.226.904.331 1.362.331v-1.2c-.274 0-.55-.063-.81-.197zm-1.31-4.033c-.748 1.476-.165 3.284 1.314 4.035l.543-1.07c-.884-.45-1.237-1.533-.786-2.422zm6.59-12.948L7.116 68.878l1.07.544 6.588-12.948zm3.944-1.358c-1.463-.682-3.21-.083-3.944 1.357l1.068.546c.441-.865 1.491-1.225 2.368-.815zm12.701 5.922L17.65 54.572l-.507 1.088 12.702 5.922zm1.455 3.989c.697-1.5.052-3.284-1.454-3.989l-.508 1.087c.906.424 1.293 1.496.874 2.396zm-3.99 1.449c1.5.697 3.284.053 3.989-1.447l-1.086-.51c-.423.9-1.495 1.288-2.396.869zm-8.37-3.906 8.37 3.906.508-1.088-8.37-3.906zm28.478 16.748c-11.808 0-22.674-6.969-27.682-17.549l-1.084.514c5.204 10.996 16.49 18.235 28.766 18.235zm29.382-22.008C73.523 69.722 61.44 78.774 47.924 78.774v1.2c14.048 0 26.601-9.404 30.534-22.872z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3V7ny$jsx)("g", {
                clipPath: "url(#adffa5__a)",
                children: /*#__PURE__*/ (0, $3V7ny$jsxs)("g", {
                    clipPath: "url(#adffa5__b)",
                    children: [
                        /*#__PURE__*/ (0, $3V7ny$jsx)("rect", {
                            width: 85.035,
                            height: 80.16,
                            x: 6.203,
                            y: 4.8,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $3V7ny$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6.203 4.8h85.035v80.16H6.203z"
                        }),
                        /*#__PURE__*/ (0, $3V7ny$jsx)("ellipse", {
                            cx: 36.995,
                            cy: 34.874,
                            fill: "url(#adffa5__c)",
                            rx: 36.995,
                            ry: 34.874,
                            transform: "matrix(1 -.00132 .00148 1 17.238 -26.916)"
                        }),
                        /*#__PURE__*/ (0, $3V7ny$jsx)("path", {
                            fill: "url(#adffa5__d)",
                            d: "M38.66-9.36c-6.53-3.81-14.805-3.833-21.36-.06l-65.56 37.734c-6.42 3.696-6.431 12.419-.02 16.129l65.128 37.68c6.545 3.787 14.819 3.782 21.358-.013L103.2 44.394c6.378-3.702 6.39-12.376.02-16.092z"
                        }),
                        /*#__PURE__*/ (0, $3V7ny$jsx)("path", {
                            fill: "url(#adffa5__e)",
                            d: "M18.459 42.858C19.912 12.932 46.825-10.217 78.57-8.847s56.303 26.74 54.85 56.667c-1.454 29.926-28.367 53.075-60.113 51.705s-56.303-26.74-54.85-56.667"
                        }),
                        /*#__PURE__*/ (0, $3V7ny$jsx)("path", {
                            fill: "url(#adffa5__f)",
                            d: "M-28.248 62.59c12.62-5.075 26.229-4.857 38.02-.418 23.594 8.904 37.322 3.399 46.844-18.792C61.36 32.245 70.62 22.815 83.252 17.734c25.259-10.142 54.418.863 65.162 24.603 10.743 23.74-1.016 51.185-26.268 61.34-12.662 5.092-26.297 4.853-38.106.375-23.556-8.857-37.259-3.331-46.768 18.886-4.755 11.109-14.013 20.507-26.632 25.582-25.238 10.149-54.398-.856-65.136-24.582-10.737-23.727 1.01-51.198 26.248-61.348"
                        }),
                        /*#__PURE__*/ (0, $3V7ny$jsx)("ellipse", {
                            cx: 36.995,
                            cy: 34.874,
                            fill: "url(#adffa5__g)",
                            rx: 36.995,
                            ry: 34.874,
                            transform: "matrix(1 -.00132 .00148 1 49.234 -15.543)"
                        })
                    ]
                })
            })
        ]
    });
const $6ebce5f5c5a716ef$var$ForwardRef = /*#__PURE__*/ (0, $3V7ny$forwardRef)($6ebce5f5c5a716ef$var$SvgComponent);
var $6ebce5f5c5a716ef$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6ebce5f5c5a716ef$var$ForwardRef);


export {$6ebce5f5c5a716ef$export$2e2bcd8739ae039 as default};
