// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_dropToUpload_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $fZsNu$reactjsxruntime = require("react/jsx-runtime");
var $fZsNu$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $09dbc1bede6c6f52$export$2e2bcd8739ae039);




const $09dbc1bede6c6f52$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "851f4c__d",
                        x1: -19.766,
                        x2: 123.63,
                        y1: 81.22,
                        y2: -6.021,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "851f4c__e",
                        x1: 95.308,
                        x2: 33.791,
                        y1: 18.646,
                        y2: 84.794,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "851f4c__f",
                        x1: 20.551,
                        x2: 129.603,
                        y1: 93.923,
                        y2: 32.834,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsxs)("radialGradient", {
                        id: "851f4c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 23.224 29.045)scale(38.3718)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsxs)("radialGradient", {
                        id: "851f4c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 33.56 51.896)scale(38.3718)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("clipPath", {
                        id: "851f4c__b",
                        children: /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("rect", {
                            width: 88.2,
                            height: 88.2,
                            x: 2.398,
                            y: 2.4,
                            fill: "#fff",
                            rx: 2.663
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("clipPath", {
                id: "851f4c__a",
                children: /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M31.046 13.16h16.442c1.687 0 3.304.67 4.496 1.862l17.472 17.47c1.192 1.193 1.862 2.81 1.862 4.497V61.91c0 3.512-2.847 6.359-6.359 6.359H31.046c-3.512 0-6.36-2.847-6.36-6.359V19.518c0-3.512 2.848-6.359 6.36-6.359m18.6 18.374c.002-1.17-.945-2.121-2.115-2.124s-2.122.944-2.125 2.115l-.047 20.99-7.165-5.7c-.916-.729-2.25-.577-2.979.34-.728.916-.576 2.25.34 2.978l10.675 8.49c.777.619 1.879.615 2.651-.009l10.52-8.49c.911-.736 1.054-2.07.319-2.981s-2.07-1.054-2.981-.319l-7.14 5.764zm-36.204 13.78c0-1.06.86-1.92 1.92-1.92h3.528c1.06 0 1.92-.86 1.92-1.92s-.86-1.92-1.92-1.92h-3.528c-3.182 0-5.76 2.578-5.76 5.76v3.32c0 1.061.86 1.92 1.92 1.92s1.92-.859 1.92-1.92zm63.67-5.76c-1.06 0-1.92.86-1.92 1.92s.86 1.92 1.92 1.92h3.53c1.06 0 1.92.86 1.92 1.92v3.32c0 1.061.86 1.92 1.92 1.92s1.92-.859 1.92-1.92v-3.32c0-3.182-2.58-5.76-5.76-5.76zm9.29 17.798c0-1.06-.86-1.92-1.92-1.92s-1.92.86-1.92 1.92v6.643c0 1.06.86 1.92 1.92 1.92s1.92-.86 1.92-1.92zm-72.96 0c0-1.06-.86-1.92-1.92-1.92s-1.92.86-1.92 1.92v6.643c0 1.06.86 1.92 1.92 1.92s1.92-.86 1.92-1.92zm0 15.36c0-1.06-.86-1.92-1.92-1.92s-1.92.86-1.92 1.92v3.322c0 3.18 2.578 5.76 5.76 5.76h3.528c1.06 0 1.92-.86 1.92-1.92s-.86-1.92-1.92-1.92h-3.528c-1.06 0-1.92-.86-1.92-1.92zm72.96 0c0-1.06-.86-1.92-1.92-1.92s-1.92.86-1.92 1.92v3.322c0 1.06-.86 1.92-1.92 1.92h-3.53c-1.06 0-1.92.86-1.92 1.92s.86 1.92 1.92 1.92h3.53c3.18 0 5.76-2.58 5.76-5.76zm-58.25 5.242c-1.06 0-1.92.86-1.92 1.92s.86 1.92 1.92 1.92h7.058c1.06 0 1.92-.86 1.92-1.92s-.86-1.92-1.92-1.92zm16.32 0c-1.06 0-1.92.86-1.92 1.92s.86 1.92 1.92 1.92h7.058c1.06 0 1.92-.86 1.92-1.92s-.86-1.92-1.92-1.92zm16.32 0c-1.06 0-1.92.86-1.92 1.92s.86 1.92 1.92 1.92h7.058c1.06 0 1.92-.86 1.92-1.92s-.86-1.92-1.92-1.92z"
                })
            }),
            /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("g", {
                clipPath: "url(#851f4c__a)",
                children: /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#851f4c__b)",
                    children: [
                        /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("rect", {
                            width: 88.2,
                            height: 88.2,
                            x: 2.398,
                            y: 2.4,
                            fill: "#FFECCF",
                            rx: 2.663
                        }),
                        /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M2.398 2.4h88.2v88.2h-88.2z"
                        }),
                        /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("circle", {
                            cx: 52.269,
                            cy: 5.821,
                            r: 38.372,
                            fill: "url(#851f4c__c)",
                            transform: "rotate(-.08 52.27 5.821)"
                        }),
                        /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("path", {
                            fill: "url(#851f4c__d)",
                            d: "M36.063-13.18c-6.773-4.19-15.356-4.216-22.154-.065l-68.001 41.519c-6.66 4.066-6.671 13.664-.02 17.746l67.551 41.46c6.789 4.166 15.371 4.161 22.154-.015l67.412-41.499c6.616-4.073 6.628-13.617.02-17.706z"
                        }),
                        /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("path", {
                            fill: "url(#851f4c__e)",
                            d: "M15.11 44.276c1.507-32.928 29.422-58.4 62.35-56.892s58.398 29.423 56.891 62.35-29.423 58.399-62.35 56.892c-32.928-1.508-58.4-29.423-56.892-62.35"
                        }),
                        /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("path", {
                            fill: "url(#851f4c__f)",
                            d: "M-33.334 65.987c13.089-5.584 27.205-5.345 39.435-.46 24.472 9.797 38.711 3.74 48.588-20.677 4.92-12.252 14.524-22.628 27.627-28.218 26.199-11.16 56.444.95 67.587 27.07s-1.053 56.319-27.245 67.493c-13.133 5.602-27.276 5.34-39.525.412-24.433-9.745-38.645-3.665-48.509 20.78-4.932 12.223-14.534 22.564-27.623 28.148-26.178 11.167-56.423-.941-67.56-27.048s1.047-56.333 27.225-67.5"
                        }),
                        /*#__PURE__*/ (0, $fZsNu$reactjsxruntime.jsx)("circle", {
                            cx: 85.457,
                            cy: 18.335,
                            r: 38.372,
                            fill: "url(#851f4c__g)",
                            transform: "rotate(-.08 85.457 18.335)"
                        })
                    ]
                })
            })
        ]
    });
const $09dbc1bede6c6f52$var$ForwardRef = /*#__PURE__*/ (0, $fZsNu$react.forwardRef)($09dbc1bede6c6f52$var$SvgComponent);
var $09dbc1bede6c6f52$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($09dbc1bede6c6f52$var$ForwardRef);
