// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileShared_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $aewCZ$jsxs, jsx as $aewCZ$jsx} from "react/jsx-runtime";
import {forwardRef as $aewCZ$forwardRef} from "react";





const $a03fa7a1824b9898$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aewCZ$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aewCZ$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aewCZ$jsxs)("linearGradient", {
                        id: "807425__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.896,
                        y2: -4.582,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aewCZ$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $aewCZ$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aewCZ$jsxs)("linearGradient", {
                        id: "807425__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.843,
                        y2: 44.841,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aewCZ$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aewCZ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aewCZ$jsxs)("linearGradient", {
                        id: "807425__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.81,
                        y2: 16.564,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aewCZ$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aewCZ$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aewCZ$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aewCZ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aewCZ$jsxs)("radialGradient", {
                        id: "807425__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.503)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aewCZ$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $aewCZ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aewCZ$jsxs)("radialGradient", {
                        id: "807425__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.264 26.937)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aewCZ$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aewCZ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aewCZ$jsx)("clipPath", {
                        id: "807425__b",
                        children: /*#__PURE__*/ (0, $aewCZ$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aewCZ$jsx)("clipPath", {
                id: "807425__a",
                children: /*#__PURE__*/ (0, $aewCZ$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "m26.454 8.577 9.892 9.891c.675.675 1.054 1.59 1.054 2.546v14.109c0 1.988-1.612 3.6-3.6 3.6H14.6c-1.988 0-3.6-1.612-3.6-3.6v-24c0-1.989 1.612-3.6 3.6-3.6h9.309c.955 0 1.87.379 2.545 1.054m-5.152 14.582c0-.213.173-.386.387-.386.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25c-1.595 0-2.887 1.292-2.887 2.886 0 .814.338 1.55.879 2.074.496.48 1.287.467 1.767-.029s.468-1.287-.028-1.767c-.074-.072-.118-.169-.118-.278m4.213-1.71c-1.503 0-2.72 1.218-2.72 2.72s1.217 2.721 2.72 2.721c1.502 0 2.72-1.218 2.72-2.72s-1.218-2.721-2.72-2.721m-4.426 7.406c.623-.298.887-1.044.59-1.667s-1.044-.887-1.667-.59c-1.298.62-2.245 1.864-2.494 3.333-.116.68.342 1.326 1.023 1.442s1.326-.343 1.442-1.023c.114-.675.549-1.23 1.106-1.495m4.426-1.524c-2.074 0-3.966 1.33-4.438 3.287-.302 1.254.76 2.155 1.763 2.155h5.35c1.002 0 2.065-.9 1.762-2.156-.471-1.956-2.363-3.286-4.437-3.286"
                })
            }),
            /*#__PURE__*/ (0, $aewCZ$jsx)("g", {
                clipPath: "url(#807425__a)",
                children: /*#__PURE__*/ (0, $aewCZ$jsxs)("g", {
                    clipPath: "url(#807425__b)",
                    children: [
                        /*#__PURE__*/ (0, $aewCZ$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $aewCZ$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 .001h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $aewCZ$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.863,
                            r: 20.883,
                            fill: "url(#807425__c)",
                            transform: "rotate(-.08 27.142 1.863)"
                        }),
                        /*#__PURE__*/ (0, $aewCZ$jsx)("path", {
                            fill: "url(#807425__d)",
                            d: "M18.321-8.478c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.657L6.008 46.304c3.694 2.267 8.365 2.264 12.056-.008L54.752 23.71c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $aewCZ$jsx)("path", {
                            fill: "url(#807425__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.99 40.85-8.17S72.63 7.84 71.809 25.76 55.798 57.543 37.879 56.722C19.957 55.902 6.095 40.71 6.915 22.79"
                        }),
                        /*#__PURE__*/ (0, $aewCZ$jsx)("path", {
                            fill: "url(#807425__f)",
                            d: "M-19.448 34.606c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.331 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.649-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.279-15.033 15.318-14.246 6.077-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $aewCZ$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.673,
                            r: 20.883,
                            fill: "url(#807425__g)",
                            transform: "rotate(-.08 45.2 8.673)"
                        })
                    ]
                })
            })
        ]
    });
const $a03fa7a1824b9898$var$ForwardRef = /*#__PURE__*/ (0, $aewCZ$forwardRef)($a03fa7a1824b9898$var$SvgComponent);
var $a03fa7a1824b9898$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a03fa7a1824b9898$var$ForwardRef);


export {$a03fa7a1824b9898$export$2e2bcd8739ae039 as default};
