// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_education_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $cT1u1$jsxs, jsx as $cT1u1$jsx} from "react/jsx-runtime";
import {forwardRef as $cT1u1$forwardRef} from "react";





const $8fba0b4db4b4d0d5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cT1u1$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cT1u1$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cT1u1$jsxs)("linearGradient", {
                        id: "a15cfe__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cT1u1$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $cT1u1$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cT1u1$jsxs)("linearGradient", {
                        id: "a15cfe__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cT1u1$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cT1u1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cT1u1$jsxs)("linearGradient", {
                        id: "a15cfe__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cT1u1$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cT1u1$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cT1u1$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cT1u1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cT1u1$jsxs)("radialGradient", {
                        id: "a15cfe__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cT1u1$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $cT1u1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cT1u1$jsxs)("radialGradient", {
                        id: "a15cfe__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cT1u1$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cT1u1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cT1u1$jsx)("clipPath", {
                        id: "a15cfe__b",
                        children: /*#__PURE__*/ (0, $cT1u1$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cT1u1$jsx)("clipPath", {
                id: "a15cfe__a",
                children: /*#__PURE__*/ (0, $cT1u1$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m18.526 30.379 27.96-11.422c.97-.396 2.056-.396 3.025 0l27.96 11.421c3.316 1.355 3.316 6.052 0 7.406l-3.903 1.595v13.068c0 1.662-.444 4.073-2.136 6.594-1.697 2.526-4.569 5.048-9.242 7.051-3.756 1.61-8.1 2.233-11.905 2.374-3.813.142-7.255-.194-9.292-.614-1.082-.222-1.778-1.28-1.555-2.362.222-1.082 1.28-1.778 2.362-1.555 1.68.345 4.797.665 8.336.534 3.546-.132 7.347-.712 10.478-2.054 4.089-1.752 6.3-3.822 7.497-5.604 1.2-1.788 1.457-3.402 1.457-4.364V41.013l-20.057 8.193c-.97.396-2.055.396-3.025 0L33.354 43.84c.51-.718 1.18-1.328 1.978-1.771l13.595-7.553c.966-.536 1.314-1.754.777-2.72-.536-.965-1.754-1.313-2.72-.776L33.39 38.573c-1.609.894-2.908 2.194-3.798 3.732l-11.066-4.52c-3.316-1.355-3.316-6.052 0-7.406m9.72 33.466v-16.53c0-1.791.48-3.514 1.346-5.01l3.762 1.536c-.709.998-1.108 2.207-1.108 3.474v16.098c1.688 2.882 3.513 6.735 3.853 9.973q.05.345.05.702v.005c0 2.898-2.496 5.247-5.575 5.247s-5.576-2.35-5.576-5.247q0-.295.034-.581c.258-3.106 1.76-6.8 3.214-9.667"
                })
            }),
            /*#__PURE__*/ (0, $cT1u1$jsx)("g", {
                clipPath: "url(#a15cfe__a)",
                children: /*#__PURE__*/ (0, $cT1u1$jsxs)("g", {
                    clipPath: "url(#a15cfe__b)",
                    children: [
                        /*#__PURE__*/ (0, $cT1u1$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $cT1u1$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $cT1u1$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#a15cfe__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $cT1u1$jsx)("path", {
                            fill: "url(#a15cfe__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $cT1u1$jsx)("path", {
                            fill: "url(#a15cfe__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $cT1u1$jsx)("path", {
                            fill: "url(#a15cfe__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $cT1u1$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#a15cfe__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $8fba0b4db4b4d0d5$var$ForwardRef = /*#__PURE__*/ (0, $cT1u1$forwardRef)($8fba0b4db4b4d0d5$var$SvgComponent);
var $8fba0b4db4b4d0d5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8fba0b4db4b4d0d5$var$ForwardRef);


export {$8fba0b4db4b4d0d5$export$2e2bcd8739ae039 as default};
