// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_dataAnalytics_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $i8u4f$reactjsxruntime = require("react/jsx-runtime");
var $i8u4f$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $da79c3caad3bb2b7$export$2e2bcd8739ae039);




const $da79c3caad3bb2b7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsxs)("linearGradient", {
                        id: "514d2c__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsxs)("linearGradient", {
                        id: "514d2c__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsxs)("linearGradient", {
                        id: "514d2c__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsxs)("radialGradient", {
                        id: "514d2c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsxs)("radialGradient", {
                        id: "514d2c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("clipPath", {
                        id: "514d2c__b",
                        children: /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("clipPath", {
                id: "514d2c__a",
                children: /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M79.44 20.813c.686.866.539 2.124-.328 2.81L56.016 41.893c-.619.49-1.468.57-2.167.204L39.13 34.407 19.423 51.124c-.843.714-2.105.61-2.82-.232s-.61-2.104.232-2.819l20.736-17.587c.62-.527 1.498-.625 2.22-.248l14.76 7.714 22.08-17.467c.866-.685 2.124-.538 2.81.328M35.76 51.856c0-1.927 1.562-3.49 3.49-3.49 1.927 0 3.49 1.563 3.49 3.49v18.597c0 1.928-1.563 3.49-3.49 3.49s-3.49-1.562-3.49-3.49zm20.998-.066c-1.928 0-3.49 1.563-3.49 3.49v15.175c0 1.928 1.562 3.49 3.49 3.49 1.927 0 3.49-1.562 3.49-3.49V55.28c0-1.927-1.563-3.49-3.49-3.49m14.018-8.555c0-1.927 1.563-3.49 3.49-3.49s3.49 1.563 3.49 3.49v27.22c0 1.927-1.562 3.49-3.49 3.49s-3.49-1.563-3.49-3.49zM21.74 58.564c-1.928 0-3.49 1.562-3.49 3.49v8.4c0 1.928 1.562 3.49 3.49 3.49 1.927 0 3.49-1.562 3.49-3.49v-8.4c0-1.928-1.563-3.49-3.49-3.49"
                })
            }),
            /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("g", {
                clipPath: "url(#514d2c__a)",
                children: /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#514d2c__b)",
                    children: [
                        /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#514d2c__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("path", {
                            fill: "url(#514d2c__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("path", {
                            fill: "url(#514d2c__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("path", {
                            fill: "url(#514d2c__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $i8u4f$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#514d2c__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $da79c3caad3bb2b7$var$ForwardRef = /*#__PURE__*/ (0, $i8u4f$react.forwardRef)($da79c3caad3bb2b7$var$SvgComponent);
var $da79c3caad3bb2b7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($da79c3caad3bb2b7$var$ForwardRef);
