// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_text_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bOqWo$jsxs, jsx as $bOqWo$jsx} from "react/jsx-runtime";
import {forwardRef as $bOqWo$forwardRef} from "react";





const $092e8f97f30bc640$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bOqWo$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bOqWo$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bOqWo$jsxs)("linearGradient", {
                        id: "8086e6__e",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bOqWo$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bOqWo$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bOqWo$jsxs)("linearGradient", {
                        id: "8086e6__f",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bOqWo$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bOqWo$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bOqWo$jsxs)("linearGradient", {
                        id: "8086e6__g",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bOqWo$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bOqWo$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bOqWo$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bOqWo$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bOqWo$jsxs)("radialGradient", {
                        id: "8086e6__d",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bOqWo$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bOqWo$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bOqWo$jsxs)("radialGradient", {
                        id: "8086e6__h",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bOqWo$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bOqWo$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bOqWo$jsx)("clipPath", {
                        id: "8086e6__c",
                        children: /*#__PURE__*/ (0, $bOqWo$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bOqWo$jsxs)("clipPath", {
                id: "8086e6__b",
                children: [
                    /*#__PURE__*/ (0, $bOqWo$jsx)("path", {
                        fill: "#292929",
                        fillRule: "evenodd",
                        d: "M37.32 32.07c-2.9 0-5.25 2.35-5.25 5.25v17.072c0 2.9 2.35 5.25 5.25 5.25s5.25-2.35 5.25-5.25V42.569h32.183v74.861H62.928c-2.9 0-5.25 2.35-5.25 5.25s2.35 5.25 5.25 5.25h16.964l.111.001.112-.001h16.958c2.9 0 5.25-2.351 5.25-5.25s-2.35-5.25-5.25-5.25h-11.82V42.57h32.178v11.822c0 2.9 2.351 5.25 5.25 5.25s5.25-2.35 5.25-5.25V37.407l.001-.088c0-2.9-2.351-5.25-5.25-5.25H37.32"
                    }),
                    /*#__PURE__*/ (0, $bOqWo$jsx)("path", {
                        fill: "#292929",
                        d: "M42.57 42.57v-2c-1.104 0-2 .895-2 2zm32.183 0h2c0-1.105-.895-2-2-2zm0 74.86v2c1.105 0 2-.896 2-2zm5.139 10.5.041-2h-.041zm.223 0v-2h-.042zm5.138-10.5h-2c0 1.104.896 2 2 2zm0-74.86v-2c-1.104 0-2 .895-2 2zm32.178 0h2c0-1.105-.895-2-2-2zm10.5-5.163-1.999-.033-.001.032zm-93.86-.088c0-1.795 1.454-3.25 3.25-3.25v-4c-4.005 0-7.25 3.246-7.25 7.25zm0 17.073V37.319h-4v17.073zm3.25 3.25c-1.796 0-3.25-1.455-3.25-3.25h-4c0 4.004 3.245 7.25 7.25 7.25zm3.25-3.25c0 1.795-1.456 3.25-3.25 3.25v4c4.003 0 7.25-3.246 7.25-7.25zm0-11.823v11.823h4V42.569zm34.182-2H42.57v4h32.183zm2 76.861V42.57h-4v74.86zm-13.825 2h11.825v-4H62.928zm-3.25 3.25c0-1.795 1.455-3.25 3.25-3.25v-4c-4.004 0-7.25 3.246-7.25 7.25zm3.25 3.25c-1.795 0-3.25-1.455-3.25-3.25h-4c0 4.004 3.246 7.25 7.25 7.25zm16.964 0H62.928v4h16.964zm.111.001-.07-.001-.083 3.999.153.002zm.07-.001-.07.001v4l.153-.002zm17 0H80.115v4h16.958zm3.25-3.25c0 1.795-1.455 3.25-3.25 3.25v4c4.004 0 7.25-3.246 7.25-7.25zm-3.25-3.25c1.795 0 3.25 1.455 3.25 3.25h4c0-4.004-3.246-7.25-7.25-7.25zm-11.82 0h11.82v-4h-11.82zm-2-76.86v74.86h4V42.57zm34.178-2H85.253v4h32.178zm2 13.822V42.569h-4v11.823zm3.25 3.25c-1.795 0-3.25-1.455-3.25-3.25h-4c0 4.004 3.246 7.25 7.25 7.25zm3.25-3.25c0 1.795-1.455 3.25-3.25 3.25v4c4.004 0 7.25-3.246 7.25-7.25zm0-16.985v16.985h4V37.407zm.001-.088v.055l3.999.065.001-.12zm-3.25-3.25c1.795 0 3.25 1.455 3.25 3.25h4c0-4.004-3.246-7.25-7.25-7.25zm-.001 0h.001v-4h-.001zm-42.678 0h42.678v-4H80.003zm-42.683 0h42.683v-4H37.32z",
                        clipPath: "url(#8086e6__a)"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bOqWo$jsx)("g", {
                clipPath: "url(#8086e6__b)",
                children: /*#__PURE__*/ (0, $bOqWo$jsxs)("g", {
                    clipPath: "url(#8086e6__c)",
                    children: [
                        /*#__PURE__*/ (0, $bOqWo$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $bOqWo$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $bOqWo$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#8086e6__d)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $bOqWo$jsx)("path", {
                            fill: "url(#8086e6__e)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $bOqWo$jsx)("path", {
                            fill: "url(#8086e6__f)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $bOqWo$jsx)("path", {
                            fill: "url(#8086e6__g)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $bOqWo$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#8086e6__h)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $092e8f97f30bc640$var$ForwardRef = /*#__PURE__*/ (0, $bOqWo$forwardRef)($092e8f97f30bc640$var$SvgComponent);
var $092e8f97f30bc640$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($092e8f97f30bc640$var$ForwardRef);


export {$092e8f97f30bc640$export$2e2bcd8739ae039 as default};
