// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_AIchat_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $dd3ri$reactjsxruntime = require("react/jsx-runtime");
var $dd3ri$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3bdeaa05621eefc5$export$2e2bcd8739ae039);




const $3bdeaa05621eefc5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f9adf7__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f9adf7__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f9adf7__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f9adf7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f9adf7__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("clipPath", {
                        id: "f9adf7__b",
                        children: /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("clipPath", {
                id: "f9adf7__a",
                children: /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m106.073 50.902-2.121 9.84c-.94 4.37 4.078 7.393 7.233 4.358l7.107-6.833c1.055-1.016 2.525-1.427 3.925-1.096l9.43 2.214q.156.036.31.061c4.018.66 6.703-4.4 3.867-7.609l-6.55-7.416c-.84-.95-1.248-2.224-1.14-3.494q.015-.163.04-.327.021-.138.051-.274l2.121-9.84c.94-4.37-4.078-7.394-7.233-4.359l-7.107 6.834c-1.055 1.016-2.525 1.426-3.925 1.095l-9.43-2.213c-3.251-.762-5.725 2.225-5.307 5.14.12.839.48 1.672 1.13 2.407l6.55 7.416c.832.942 1.24 2.201 1.143 3.46q-.025.319-.094.636m-16.757-13.92H40.489c-6.76 0-12.239 5.323-12.239 12.04v39.66c0 6.717 5.48 12.04 12.239 12.04h10.424c.024 5.739.046 13.952 0 20.207-.026 3.488 4.1 5.375 6.752 3.09l27.242-23.297h34.811c6.759 0 12.239-5.323 12.239-12.04V67.491q-1.044-.063-2.135-.317l-.003-.001-7.433-1.745-5.655 5.438-.001.001c-4.458 4.287-10.531 4.271-14.826 1.683-4.24-2.554-7.014-7.72-5.773-13.49l1.774-8.23-5.426-6.142-.001-.002c-2.066-2.339-3.034-5.051-3.162-7.704m1.042 47.08-6.93-1.562c-.919-.205-1.87.051-2.56.684l-5.221 4.817c-2.058 1.898-5.33.005-4.714-2.72l1.561-6.93c.206-.918-.05-1.87-.684-2.56l-4.816-5.221c-1.898-2.058-.006-5.33 2.719-4.714l6.931 1.562c.918.205 1.87-.052 2.56-.684l5.22-4.817c2.058-1.898 5.33-.005 4.715 2.72l-1.562 6.93c-.206.918.051 1.87.684 2.56l4.816 5.221c1.898 2.058.006 5.33-2.719 4.714"
                })
            }),
            /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("g", {
                clipPath: "url(#f9adf7__a)",
                children: /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#f9adf7__b)",
                    children: [
                        /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#f9adf7__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("path", {
                            fill: "url(#f9adf7__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("path", {
                            fill: "url(#f9adf7__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("path", {
                            fill: "url(#f9adf7__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $dd3ri$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#f9adf7__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $3bdeaa05621eefc5$var$ForwardRef = /*#__PURE__*/ (0, $dd3ri$react.forwardRef)($3bdeaa05621eefc5$var$SvgComponent);
var $3bdeaa05621eefc5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3bdeaa05621eefc5$var$ForwardRef);
