// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_lockOpen_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $80TjC$jsxs, jsx as $80TjC$jsx} from "react/jsx-runtime";
import {forwardRef as $80TjC$forwardRef} from "react";





const $b79173967cd67039$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $80TjC$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $80TjC$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $80TjC$jsxs)("linearGradient", {
                        id: "32cc98__d",
                        x1: -14.724,
                        x2: 121.006,
                        y1: 80.863,
                        y2: -1.714,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $80TjC$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $80TjC$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $80TjC$jsxs)("linearGradient", {
                        id: "32cc98__e",
                        x1: 94.203,
                        x2: 35.974,
                        y1: 21.635,
                        y2: 84.246,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $80TjC$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $80TjC$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $80TjC$jsxs)("linearGradient", {
                        id: "32cc98__f",
                        x1: 23.438,
                        x2: 126.661,
                        y1: 92.888,
                        y2: 35.064,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $80TjC$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $80TjC$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $80TjC$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $80TjC$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $80TjC$jsxs)("radialGradient", {
                        id: "32cc98__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 21.982 31.479)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $80TjC$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $80TjC$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $80TjC$jsxs)("radialGradient", {
                        id: "32cc98__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 31.768 53.108)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $80TjC$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $80TjC$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $80TjC$jsx)("clipPath", {
                        id: "32cc98__b",
                        children: /*#__PURE__*/ (0, $80TjC$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $80TjC$jsx)("clipPath", {
                id: "32cc98__a",
                children: /*#__PURE__*/ (0, $80TjC$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M32.402 27.3c0-8.615 6.984-15.6 15.6-15.6s15.6 6.985 15.6 15.6c0 1.326-1.075 2.4-2.4 2.4-1.326 0-2.4-1.074-2.4-2.4 0-5.965-4.836-10.8-10.8-10.8s-10.8 4.836-10.8 10.8v12.6h32.181c4.429 0 8.019 3.274 8.019 7.313v24.375c0 4.038-3.59 7.312-8.019 7.312H26.62c-4.429 0-8.018-3.274-8.018-7.312V47.213c0-4.039 3.59-7.313 8.018-7.313h5.782zM44.61 60.392c.296.298.63.554.992.763V65.7c0 1.326 1.075 2.4 2.4 2.4s2.4-1.074 2.4-2.4v-4.545c.362-.209.696-.465.992-.763.224-.216.424-.464.6-.728.176-.256.32-.536.44-.824.12-.296.216-.592.272-.904.064-.304.096-.624.096-.936s-.032-.632-.096-.936c-.056-.304-.153-.608-.272-.896-.12-.296-.264-.568-.44-.832s-.376-.512-.6-.728c-.888-.896-2.128-1.408-3.392-1.408s-2.504.512-3.392 1.408c-.224.216-.424.464-.6.728-.168.264-.32.536-.44.832-.12.288-.217.592-.272.896-.064.304-.096.624-.096.936s.032.632.096.936c.055.312.152.608.272.904.12.288.272.568.44.824.176.264.376.512.6.728"
                })
            }),
            /*#__PURE__*/ (0, $80TjC$jsx)("g", {
                clipPath: "url(#32cc98__a)",
                children: /*#__PURE__*/ (0, $80TjC$jsxs)("g", {
                    clipPath: "url(#32cc98__b)",
                    children: [
                        /*#__PURE__*/ (0, $80TjC$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $80TjC$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6.258 6.258h83.485v83.485H6.258z"
                        }),
                        /*#__PURE__*/ (0, $80TjC$jsx)("circle", {
                            cx: 53.461,
                            cy: 9.496,
                            r: 36.321,
                            fill: "url(#32cc98__c)",
                            transform: "rotate(-.08 53.461 9.496)"
                        }),
                        /*#__PURE__*/ (0, $80TjC$jsx)("path", {
                            fill: "url(#32cc98__d)",
                            d: "M38.121-8.49c-6.412-3.967-14.535-3.992-20.97-.062l-64.366 39.3c-6.304 3.848-6.314 12.933-.02 16.797l63.941 39.243c6.426 3.944 14.55 3.939 20.97-.014l63.808-39.28c6.262-3.855 6.274-12.89.019-16.76z"
                        }),
                        /*#__PURE__*/ (0, $80TjC$jsx)("path", {
                            fill: "url(#32cc98__e)",
                            d: "M18.29 45.894c1.428-31.167 27.85-55.277 59.018-53.85s55.277 27.85 53.85 59.018-27.85 55.276-59.017 53.849c-31.168-1.426-55.277-27.85-53.85-59.017"
                        }),
                        /*#__PURE__*/ (0, $80TjC$jsx)("path", {
                            fill: "url(#32cc98__f)",
                            d: "M-27.566 66.445c12.389-5.285 25.75-5.059 37.326-.436 23.164 9.274 36.642 3.54 45.99-19.57 4.657-11.598 13.749-21.42 26.152-26.71 24.798-10.563 53.426.898 63.974 25.623 10.547 24.724-.998 53.307-25.79 63.884-12.43 5.303-25.818 5.055-37.412.39-23.126-9.224-36.579-3.469-45.915 19.67C32.09 140.865 23 150.653 10.612 155.938c-24.778 10.571-53.406-.89-63.948-25.601s.991-53.322 25.77-63.892"
                        }),
                        /*#__PURE__*/ (0, $80TjC$jsx)("circle", {
                            cx: 84.875,
                            cy: 21.34,
                            r: 36.321,
                            fill: "url(#32cc98__g)",
                            transform: "rotate(-.08 84.875 21.34)"
                        })
                    ]
                })
            })
        ]
    });
const $b79173967cd67039$var$ForwardRef = /*#__PURE__*/ (0, $80TjC$forwardRef)($b79173967cd67039$var$SvgComponent);
var $b79173967cd67039$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b79173967cd67039$var$ForwardRef);


export {$b79173967cd67039$export$2e2bcd8739ae039 as default};
