// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_lightbulbRays_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $28dID$reactjsxruntime = require("react/jsx-runtime");
var $28dID$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $93af24be18fc060d$export$2e2bcd8739ae039);




const $93af24be18fc060d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $28dID$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0c0b09__d",
                        x1: -5.977,
                        x2: 55.826,
                        y1: 39.303,
                        y2: -0.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0c0b09__e",
                        x1: 45.296,
                        x2: 19.612,
                        y1: 13.02,
                        y2: 42.317,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0c0b09__f",
                        x1: 11.986,
                        x2: 59.166,
                        y1: 44.639,
                        y2: 16.603,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsxs)("radialGradient", {
                        id: "0c0b09__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsxs)("radialGradient", {
                        id: "0c0b09__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("clipPath", {
                        id: "0c0b09__b",
                        children: /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.196,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("clipPath", {
                id: "0c0b09__a",
                children: /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M25.2 8.985c0 .663-.537 1.2-1.2 1.2s-1.2-.537-1.2-1.2V7.179c0-.663.537-1.2 1.2-1.2s1.2.537 1.2 1.2zm-1.86 4.119c-4.454.318-8.06 3.95-8.39 8.448-.243 3.342 1.284 6.336 3.735 8.127.795.582 1.28 1.503 1.28 2.496V34.2h7.96q.032 0 .064.006l.043.005v-2.033c0-.97.445-1.893 1.224-2.457 2.31-1.663 3.816-4.39 3.816-7.474 0-5.285-4.425-9.524-9.735-9.146zm5.811 9.27c-.663 0-1.2-.537-1.2-1.2 0-1.503-1.332-2.724-2.97-2.724-.663 0-1.2-.537-1.2-1.2s.538-1.2 1.2-1.2c2.961 0 5.37 2.297 5.37 5.124 0 .663-.537 1.2-1.2 1.2M19.97 36.6v1.44c0 .843.678 1.526 1.512 1.526h5.043c.834 0 1.512-.684 1.512-1.527v-1.452l-.044.006q-.032.005-.064.006zM8.985 20.993h1.806c.663 0 1.2.537 1.2 1.2s-.537 1.2-1.2 1.2H8.985c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.2-1.2m30.027 0h-1.806c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h1.806c.663 0 1.2-.537 1.2-1.2s-.537-1.2-1.2-1.2M33.77 10.728l-1.275 1.274c-.468.468-.468 1.227 0 1.698.234.235.54.352.849.352s.615-.117.849-.351l1.275-1.276c.468-.468.468-1.227 0-1.698-.468-.468-1.227-.468-1.698 0m-19.534.002 1.275 1.275c.468.471.468 1.23 0 1.698-.234.235-.54.352-.849.352s-.615-.117-.849-.351l-1.275-1.276c-.468-.47-.468-1.23 0-1.698.471-.468 1.23-.468 1.698 0"
                })
            }),
            /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("g", {
                clipPath: "url(#0c0b09__a)",
                children: /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#0c0b09__b)",
                    children: [
                        /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.196,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.898 6.196h39.3v37.047h-39.3z"
                        }),
                        /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#0c0b09__c)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 9 -8.462)"
                        }),
                        /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("path", {
                            fill: "url(#0c0b09__d)",
                            d: "M18.9-.348c-3.018-1.76-6.842-1.771-9.872-.028l-30.3 17.44c-2.967 1.708-2.972 5.74-.008 7.454l30.1 17.415c3.024 1.75 6.848 1.748 9.87-.006l30.038-17.431c2.948-1.711 2.953-5.72.009-7.438z"
                        }),
                        /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("path", {
                            fill: "url(#0c0b09__e)",
                            d: "M9.561 23.785C10.233 9.955 22.671-.745 37.343-.11c14.672.633 26.021 12.358 25.35 26.19-.672 13.83-13.11 24.529-27.782 23.896-14.672-.634-26.022-12.359-25.35-26.19"
                        }),
                        /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("path", {
                            fill: "url(#0c0b09__f)",
                            d: "M-12.024 32.905C-6.192 30.56.098 30.66 5.548 32.712c10.904 4.115 17.249 1.57 21.65-8.685 2.192-5.146 6.471-9.505 12.31-11.853 11.673-4.687 25.15.399 30.115 11.37 4.965 10.973-.47 23.657-12.14 28.35-5.852 2.353-12.154 2.243-17.611.173-10.887-4.093-17.22-1.54-21.615 8.729-2.198 5.134-6.476 9.477-12.308 11.823-11.664 4.69-25.141-.395-30.104-11.361-4.962-10.966.467-23.662 12.131-28.353"
                        }),
                        /*#__PURE__*/ (0, $28dID$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#0c0b09__g)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 23.785 -3.206)"
                        })
                    ]
                })
            })
        ]
    });
const $93af24be18fc060d$var$ForwardRef = /*#__PURE__*/ (0, $28dID$react.forwardRef)($93af24be18fc060d$var$SvgComponent);
var $93af24be18fc060d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($93af24be18fc060d$var$ForwardRef);
