// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_market_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $1ORL6$reactjsxruntime = require("react/jsx-runtime");
var $1ORL6$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $bc74f6669671e484$export$2e2bcd8739ae039);




const $bc74f6669671e484$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4a7d68__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4a7d68__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4a7d68__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4a7d68__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4a7d68__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("clipPath", {
                        id: "4a7d68__b",
                        children: /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("clipPath", {
                id: "4a7d68__a",
                children: /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M22 48.446V70c0 4.418 3.582 8 8 8h8V48.367c-2.643-.463-4.94-1.936-6.474-4.006-1.87 2.525-4.877 4.161-8.263 4.161q-.641 0-1.263-.076m20-.162V64h32V48.458q-.57.064-1.158.064c-3.386 0-6.393-1.636-8.263-4.161-1.87 2.525-4.877 4.161-8.263 4.161s-6.393-1.636-8.263-4.161c-1.451 1.96-3.587 3.383-6.053 3.923m-2.21-3.762c-3.467 0-6.264-2.797-6.264-6.23v-.407c0-1.105-.895-2-2-2s-2 .895-2 2v.406c0 3.434-2.796 6.231-6.263 6.231S17 41.725 17 38.292c0-1.828.398-3.634 1.17-5.29V33l5.09-10.942v-.001c.529-1.136 1.673-1.867 2.938-1.867h43.717c1.265 0 2.41.731 2.939 1.867l5.017 10.787q.028.078.064.155c.77 1.659 1.17 3.465 1.17 5.291 0 3.434-2.797 6.231-6.263 6.231s-6.263-2.797-6.263-6.23v-.407c0-1.105-.895-2-2-2s-2 .895-2 2v.406c0 3.434-2.797 6.231-6.263 6.231-3.467 0-6.263-2.797-6.263-6.23v-.407c0-1.105-.896-2-2-2s-2 .895-2 2v.406c0 3.434-2.797 6.231-6.264 6.231M42 68h32v2c0 4.418-3.582 8-8 8H42z"
                })
            }),
            /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("g", {
                clipPath: "url(#4a7d68__a)",
                children: /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#4a7d68__b)",
                    children: [
                        /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#4a7d68__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("path", {
                            fill: "url(#4a7d68__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("path", {
                            fill: "url(#4a7d68__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("path", {
                            fill: "url(#4a7d68__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $1ORL6$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#4a7d68__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $bc74f6669671e484$var$ForwardRef = /*#__PURE__*/ (0, $1ORL6$react.forwardRef)($bc74f6669671e484$var$SvgComponent);
var $bc74f6669671e484$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($bc74f6669671e484$var$ForwardRef);
