// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_link_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4DHv2$reactjsxruntime = require("react/jsx-runtime");
var $4DHv2$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0e91baee79e331fb$export$2e2bcd8739ae039);




const $0e91baee79e331fb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3eddb0__d",
                        x1: -3.949,
                        x2: 55.306,
                        y1: 38.702,
                        y2: 0.46,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3eddb0__e",
                        x1: 45.213,
                        x2: 20.587,
                        y1: 13.503,
                        y2: 41.593,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3eddb0__f",
                        x1: 13.273,
                        x2: 58.507,
                        y1: 43.818,
                        y2: 16.938,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3eddb0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3eddb0__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("clipPath", {
                        id: "3eddb0__b",
                        children: /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("clipPath", {
                id: "3eddb0__a",
                children: /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    stroke: "#fff",
                    strokeLinecap: "round",
                    strokeLinejoin: "round",
                    strokeWidth: 0.6,
                    d: "M36.96 17.364c1.754-1.755 1.754-4.599 0-6.353s-4.598-1.754-6.353 0l-3.273 3.274c-.469.469-1.229.469-1.697 0-.47-.469-.47-1.229 0-1.697l3.273-3.274c2.692-2.692 7.056-2.692 9.747 0 2.692 2.691 2.692 7.055 0 9.747l-8.301 8.301c-2.692 2.692-7.056 2.692-9.747 0-.469-.468-.469-1.228 0-1.697.469-.468 1.228-.468 1.697 0 1.754 1.754 4.598 1.754 6.353 0zm-9.497 3.205c.47.469.47 1.228 0 1.697-.468.469-1.228.469-1.697 0-1.754-1.754-4.598-1.754-6.352 0l-8.302 8.301c-1.754 1.755-1.754 4.599 0 6.353s4.599 1.754 6.353 0l3.274-3.274c.468-.468 1.228-.468 1.697 0 .468.469.468 1.229 0 1.697l-3.274 3.274c-2.691 2.692-7.055 2.692-9.747 0-2.691-2.691-2.691-7.055 0-9.747l8.302-8.301c2.691-2.692 7.055-2.692 9.746 0"
                })
            }),
            /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("g", {
                clipPath: "url(#3eddb0__a)",
                children: /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#3eddb0__b)",
                    children: [
                        /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M5.52 6.96H43.2v35.52H5.52z"
                        }),
                        /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#3eddb0__c)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 10.41 -7.094)"
                        }),
                        /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("path", {
                            fill: "url(#3eddb0__d)",
                            d: "M19.902.685c-2.894-1.688-6.56-1.698-9.465-.026l-29.05 16.72c-2.845 1.638-2.85 5.503-.009 7.147l28.859 16.697c2.9 1.678 6.567 1.676 9.464-.006l28.8-16.712c2.826-1.64 2.83-5.485.008-7.131z"
                        }),
                        /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("path", {
                            fill: "url(#3eddb0__e)",
                            d: "M10.951 23.824C11.595 10.564 23.521.306 37.588.913s24.949 11.849 24.305 25.11-12.57 23.518-26.637 22.91c-14.067-.606-24.949-11.848-24.305-25.109"
                        }),
                        /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("path", {
                            fill: "url(#3eddb0__f)",
                            d: "M-9.747 32.567c5.591-2.248 11.622-2.152 16.847-.185 10.455 3.946 16.538 1.506 20.757-8.327 2.102-4.934 6.205-9.113 11.803-11.364 11.192-4.494 24.113.382 28.874 10.902s-.45 22.68-11.64 27.18c-5.61 2.257-11.652 2.151-16.885.167-10.438-3.925-16.51-1.476-20.724 8.368-2.107 4.923-6.21 9.087-11.8 11.336C-3.7 75.14-16.62 70.264-21.379 59.75s.447-22.686 11.63-27.184"
                        }),
                        /*#__PURE__*/ (0, $4DHv2$reactjsxruntime.jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#3eddb0__g)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 24.586 -2.055)"
                        })
                    ]
                })
            })
        ]
    });
const $0e91baee79e331fb$var$ForwardRef = /*#__PURE__*/ (0, $4DHv2$react.forwardRef)($0e91baee79e331fb$var$SvgComponent);
var $0e91baee79e331fb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0e91baee79e331fb$var$ForwardRef);
