// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_clipboard_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $3Y7g1$jsxs, jsx as $3Y7g1$jsx} from "react/jsx-runtime";
import {forwardRef as $3Y7g1$forwardRef} from "react";





const $79eb0c6d58a6bc97$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3Y7g1$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3Y7g1$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3Y7g1$jsxs)("linearGradient", {
                        id: "08eb16__d",
                        x1: -3.949,
                        x2: 55.306,
                        y1: 38.702,
                        y2: 0.46,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3Y7g1$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $3Y7g1$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3Y7g1$jsxs)("linearGradient", {
                        id: "08eb16__e",
                        x1: 45.213,
                        x2: 20.587,
                        y1: 13.503,
                        y2: 41.593,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3Y7g1$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3Y7g1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3Y7g1$jsxs)("linearGradient", {
                        id: "08eb16__f",
                        x1: 13.273,
                        x2: 58.507,
                        y1: 43.818,
                        y2: 16.938,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3Y7g1$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3Y7g1$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3Y7g1$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3Y7g1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3Y7g1$jsxs)("radialGradient", {
                        id: "08eb16__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3Y7g1$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $3Y7g1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3Y7g1$jsxs)("radialGradient", {
                        id: "08eb16__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3Y7g1$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3Y7g1$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3Y7g1$jsx)("clipPath", {
                        id: "08eb16__b",
                        children: /*#__PURE__*/ (0, $3Y7g1$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3Y7g1$jsx)("clipPath", {
                id: "08eb16__a",
                children: /*#__PURE__*/ (0, $3Y7g1$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M16.502 12h-2.576c-2.056 0-3.723 1.685-3.723 3.764v20.073c0 2.078 1.667 3.763 3.723 3.763H33.78c2.056 0 3.723-1.685 3.723-3.763V15.764c0-2.078-1.667-3.764-3.723-3.764h-2.878v.009l-.006 4.8c0 .318-.127.623-.352.848s-.53.35-.849.35l-12-.014c-.663 0-1.2-.538-1.198-1.201zm2.396 3.595.003-2.4 1.2.001c.663 0 1.2-.536 1.201-1.198l.002-1.2c.001-1.326 1.077-2.4 2.403-2.398 1.325.002 2.398 1.078 2.397 2.403l-.002 1.2c0 .663.536 1.2 1.199 1.202l1.2.001-.003 2.4z"
                })
            }),
            /*#__PURE__*/ (0, $3Y7g1$jsx)("g", {
                clipPath: "url(#08eb16__a)",
                children: /*#__PURE__*/ (0, $3Y7g1$jsxs)("g", {
                    clipPath: "url(#08eb16__b)",
                    children: [
                        /*#__PURE__*/ (0, $3Y7g1$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $3Y7g1$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M5.52 6.96H43.2v35.52H5.52z"
                        }),
                        /*#__PURE__*/ (0, $3Y7g1$jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#08eb16__c)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 10.41 -7.094)"
                        }),
                        /*#__PURE__*/ (0, $3Y7g1$jsx)("path", {
                            fill: "url(#08eb16__d)",
                            d: "M19.902.685c-2.894-1.688-6.56-1.698-9.465-.026l-29.05 16.72c-2.845 1.638-2.85 5.503-.009 7.147l28.859 16.697c2.9 1.678 6.567 1.676 9.464-.006l28.8-16.712c2.826-1.64 2.83-5.485.008-7.131z"
                        }),
                        /*#__PURE__*/ (0, $3Y7g1$jsx)("path", {
                            fill: "url(#08eb16__e)",
                            d: "M10.951 23.824C11.595 10.564 23.521.306 37.588.913s24.949 11.849 24.305 25.11-12.57 23.518-26.637 22.91c-14.067-.606-24.949-11.848-24.305-25.109"
                        }),
                        /*#__PURE__*/ (0, $3Y7g1$jsx)("path", {
                            fill: "url(#08eb16__f)",
                            d: "M-9.747 32.567c5.591-2.248 11.622-2.152 16.847-.185 10.455 3.946 16.538 1.506 20.757-8.327 2.102-4.934 6.205-9.113 11.803-11.364 11.192-4.494 24.113.382 28.874 10.902s-.45 22.68-11.64 27.18c-5.61 2.257-11.652 2.151-16.885.167-10.438-3.925-16.51-1.476-20.724 8.368-2.107 4.923-6.21 9.087-11.8 11.336C-3.7 75.14-16.62 70.264-21.379 59.75s.447-22.686 11.63-27.184"
                        }),
                        /*#__PURE__*/ (0, $3Y7g1$jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#08eb16__g)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 24.586 -2.055)"
                        })
                    ]
                })
            })
        ]
    });
const $79eb0c6d58a6bc97$var$ForwardRef = /*#__PURE__*/ (0, $3Y7g1$forwardRef)($79eb0c6d58a6bc97$var$SvgComponent);
var $79eb0c6d58a6bc97$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($79eb0c6d58a6bc97$var$ForwardRef);


export {$79eb0c6d58a6bc97$export$2e2bcd8739ae039 as default};
