// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_confetti_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $kQWsv$jsxs, jsx as $kQWsv$jsx} from "react/jsx-runtime";
import {forwardRef as $kQWsv$forwardRef} from "react";





const $526d81c50c77492d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kQWsv$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kQWsv$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kQWsv$jsxs)("linearGradient", {
                        id: "6a042f__d",
                        x1: -28.688,
                        x2: 194.754,
                        y1: 137.748,
                        y2: -6.308,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kQWsv$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kQWsv$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kQWsv$jsxs)("linearGradient", {
                        id: "6a042f__e",
                        x1: 156.584,
                        x2: 63.67,
                        y1: 42.682,
                        y2: 148.555,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kQWsv$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kQWsv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kQWsv$jsxs)("linearGradient", {
                        id: "6a042f__f",
                        x1: 36.222,
                        x2: 206.784,
                        y1: 157.049,
                        y2: 55.798,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kQWsv$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kQWsv$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kQWsv$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kQWsv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kQWsv$jsxs)("radialGradient", {
                        id: "6a042f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 58.2973 -61.7778 0 61.778 58.297)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kQWsv$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kQWsv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kQWsv$jsxs)("radialGradient", {
                        id: "6a042f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 58.2973 -61.7778 0 61.778 58.297)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kQWsv$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kQWsv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kQWsv$jsx)("clipPath", {
                        id: "6a042f__b",
                        children: /*#__PURE__*/ (0, $kQWsv$jsx)("rect", {
                            width: 142,
                            height: 134,
                            x: 7,
                            y: 18,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kQWsv$jsx)("clipPath", {
                id: "6a042f__a",
                children: /*#__PURE__*/ (0, $kQWsv$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M109.922 32.16c2.86-2.86 2.86-7.48 0-10.34s-7.48-2.86-10.34 0-2.86 7.48 0 10.34 7.48 2.86 10.34 0m-45.12 39.96c-1.73 1.38-2.01 3.9-.63 5.62.79.99 1.95 1.5 3.13 1.5.88 0 1.76-.29 2.49-.88 19.22-15.35 18.34-31.46 18.299-32.14-.15-2.19-2.03-3.82-4.22-3.7-2.19.13-3.87 2.02-3.76 4.21.02.52.43 12.81-15.31 25.38zm34 29.87c-1.03-1.95-.28-4.37 1.671-5.4h.01c1.95-1.03 4.37-.28 5.4 1.67 5.73 10.86 3.89 16.18 1.34 18.73-.83.83-1.95 1.58-3.45 2.02l-69.25 24.6c-1.34.48-2.71.71-4.06.71-3.11 0-6.14-1.23-8.44-3.53-3.3-3.3-4.38-8.09-2.82-12.5l24.32-68.53c.14-.52.32-1.02.55-1.5.45-1.12 1.07-2 1.75-2.68 2.57-2.57 7.94-4.4 18.93 1.45 1.95 1.04 2.69 3.46 1.65 5.41s-3.46 2.69-5.41 1.65c-6.19-3.29-9.03-3.09-9.52-2.84v.02l-.18.51c-.4 3.04 4.2 14.87 19.53 30.2 11.62 11.62 21.22 17.07 26.54 18.84 2.06.42 3.52.52 4.26.42.19-.55.29-3.35-2.82-9.25m-21.31-12.56c-1.03 0-2.05-.39-2.83-1.17-1.56-1.57-1.56-4.1 0-5.66l15.61-15.61c1.57-1.56 4.1-1.56 5.66 0 1.56 1.57 1.56 4.1 0 5.66l-15.61 15.61c-.78.78-1.8 1.17-2.83 1.17m7.05 3.7c15.35-19.22 31.46-18.34 32.14-18.3l-.01.01c2.21.15 3.87 2.06 3.72 4.26-.15 2.19-2.04 3.86-4.23 3.72-.59-.02-12.84-.37-25.37 15.31-.79.99-1.96 1.5-3.13 1.5-.87 0-1.75-.29-2.49-.88-1.73-1.38-2.01-3.89-.63-5.62m40.499-.96c2.86-2.86 7.48-2.86 10.34 0s2.86 7.48 0 10.34-7.48 2.86-10.34 0-2.86-7.48 0-10.34M56.354 28.195l-1.921 1.406c-1.783 1.305-2.17 3.807-.865 5.59l7.813 10.676c1.305 1.783 3.808 2.17 5.59.866l1.921-1.406c1.783-1.305 2.17-3.808.865-5.59L61.944 29.06c-1.305-1.783-3.808-2.17-5.59-.866m50.876 26.656 13.146-1.488c2.196-.249 4.177 1.33 4.425 3.524l.268 2.365c.248 2.195-1.33 4.176-3.525 4.425l-13.146 1.488c-2.195.249-4.176-1.329-4.424-3.524l-.268-2.365c-.249-2.195 1.329-4.176 3.524-4.425"
                })
            }),
            /*#__PURE__*/ (0, $kQWsv$jsx)("g", {
                clipPath: "url(#6a042f__a)",
                children: /*#__PURE__*/ (0, $kQWsv$jsxs)("g", {
                    clipPath: "url(#6a042f__b)",
                    children: [
                        /*#__PURE__*/ (0, $kQWsv$jsx)("rect", {
                            width: 142,
                            height: 134,
                            x: 7,
                            y: 18,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $kQWsv$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M7 18h142v134H7z"
                        }),
                        /*#__PURE__*/ (0, $kQWsv$jsx)("ellipse", {
                            cx: 61.778,
                            cy: 58.297,
                            fill: "url(#6a042f__c)",
                            rx: 61.778,
                            ry: 58.297,
                            transform: "matrix(1 -.00132 .00148 1 25.426 -35.019)"
                        }),
                        /*#__PURE__*/ (0, $kQWsv$jsx)("path", {
                            fill: "url(#6a042f__d)",
                            d: "M61.197-5.67c-10.906-6.368-24.723-6.407-35.669-.1l-109.48 63.078c-10.722 6.178-10.74 20.76-.032 26.961l108.756 62.99c10.93 6.329 24.747 6.321 35.667-.023l108.533-63.048c10.651-6.188 10.67-20.689.032-26.9z"
                        }),
                        /*#__PURE__*/ (0, $kQWsv$jsx)("path", {
                            fill: "url(#6a042f__e)",
                            d: "M27.465 81.62c2.427-50.026 47.37-88.724 100.383-86.434s94.02 44.702 91.593 94.728-47.369 88.723-100.382 86.433-94.02-44.701-91.594-94.727"
                        }),
                        /*#__PURE__*/ (0, $kQWsv$jsx)("path", {
                            fill: "url(#6a042f__f)",
                            d: "M-50.532 114.605c21.073-8.483 43.8-8.119 63.49-.699 39.399 14.885 62.324 5.682 78.225-31.413 7.92-18.614 23.384-34.378 44.48-42.871 42.179-16.954 90.873 1.442 108.813 41.127 17.941 39.685-1.696 85.563-43.865 102.539-21.143 8.512-43.914 8.114-63.634.627-39.335-14.806-62.217-5.568-78.098 31.571-7.94 18.57-23.4 34.28-44.472 42.764-42.146 16.966-90.84-1.43-108.77-41.093-17.931-39.663 1.686-85.585 43.831-102.552"
                        }),
                        /*#__PURE__*/ (0, $kQWsv$jsx)("ellipse", {
                            cx: 61.778,
                            cy: 58.297,
                            fill: "url(#6a042f__g)",
                            rx: 61.778,
                            ry: 58.297,
                            transform: "matrix(1 -.00132 .00148 1 78.86 -16.007)"
                        })
                    ]
                })
            })
        ]
    });
const $526d81c50c77492d$var$ForwardRef = /*#__PURE__*/ (0, $kQWsv$forwardRef)($526d81c50c77492d$var$SvgComponent);
var $526d81c50c77492d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($526d81c50c77492d$var$ForwardRef);


export {$526d81c50c77492d$export$2e2bcd8739ae039 as default};
