// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_folderShared_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $5JO1S$reactjsxruntime = require("react/jsx-runtime");
var $5JO1S$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1f3098001db7afb6$export$2e2bcd8739ae039);




const $1f3098001db7afb6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8754f3__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8754f3__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8754f3__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8754f3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8754f3__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("clipPath", {
                        id: "8754f3__b",
                        children: /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("clipPath", {
                id: "8754f3__a",
                children: /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M12 25.8h30.927l-5.591-6.928c-1.057-1.31-2.657-2.072-4.348-2.072h-15.42C14.493 16.8 12 19.27 12 22.32zm0 4.8h63.692c4.586 0 8.308 3.748 8.308 8.365v31.87c0 4.617-3.722 8.365-8.308 8.365H20.308C15.723 79.2 12 75.452 12 70.835zm31.377 15.399c-.704 0-1.274.57-1.274 1.273 0 .36.147.681.388.915.794.768.814 2.034.046 2.828s-2.035.814-2.828.046c-.988-.957-1.606-2.302-1.606-3.789 0-2.912 2.361-5.273 5.273-5.273 1.105 0 2 .895 2 2s-.895 2-2 2m2.711 3.294c0-2.729 2.212-4.94 4.941-4.94 2.73 0 4.941 2.211 4.941 4.94s-2.212 4.941-4.94 4.941c-2.73 0-4.942-2.212-4.942-4.94m-3.183 6.252c.476.997.054 2.191-.943 2.667-1.262.603-2.234 1.85-2.49 3.357-.185 1.09-1.218 1.822-2.307 1.638s-1.822-1.218-1.637-2.307c.472-2.78 2.264-5.129 4.71-6.297.997-.476 2.191-.054 2.667.942m-.265 6.761c.882-3.659 4.442-6.19 8.39-6.19 3.947 0 7.506 2.531 8.388 6.19.514 2.134-1.29 3.693-3.04 3.693H45.68c-1.749 0-3.554-1.56-3.04-3.693"
                })
            }),
            /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("g", {
                clipPath: "url(#8754f3__a)",
                children: /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#8754f3__b)",
                    children: [
                        /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#8754f3__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("path", {
                            fill: "url(#8754f3__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("path", {
                            fill: "url(#8754f3__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("path", {
                            fill: "url(#8754f3__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $5JO1S$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#8754f3__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $1f3098001db7afb6$var$ForwardRef = /*#__PURE__*/ (0, $5JO1S$react.forwardRef)($1f3098001db7afb6$var$SvgComponent);
var $1f3098001db7afb6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1f3098001db7afb6$var$ForwardRef);
