// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_remix_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $kDNWt$jsxs, jsx as $kDNWt$jsx} from "react/jsx-runtime";
import {forwardRef as $kDNWt$forwardRef} from "react";





const $600200914f8e387b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kDNWt$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kDNWt$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kDNWt$jsxs)("linearGradient", {
                        id: "b18be3__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kDNWt$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kDNWt$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kDNWt$jsxs)("linearGradient", {
                        id: "b18be3__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kDNWt$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kDNWt$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kDNWt$jsxs)("linearGradient", {
                        id: "b18be3__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kDNWt$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kDNWt$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kDNWt$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kDNWt$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kDNWt$jsxs)("radialGradient", {
                        id: "b18be3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kDNWt$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kDNWt$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kDNWt$jsxs)("radialGradient", {
                        id: "b18be3__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kDNWt$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kDNWt$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kDNWt$jsx)("clipPath", {
                        id: "b18be3__b",
                        children: /*#__PURE__*/ (0, $kDNWt$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kDNWt$jsx)("clipPath", {
                id: "b18be3__a",
                children: /*#__PURE__*/ (0, $kDNWt$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m38.19 20.999 5.307-5.308c.488-.488.488-1.28 0-1.767L38.19 8.616c-.488-.488-1.28-.488-1.768 0s-.488 1.28 0 1.768l3.071 3.071h-1.788c-2.518 0-4.84.848-6.694 2.273q-.043.033-.082.069l-.035.026c-1.364 1.072-2.632 2.625-3.622 4.043-.994 1.425-1.773 2.81-2.14 3.618-.282.624-.983 2.109-1.91 3.668-.799 1.342-1.706 2.637-2.6 3.507q-.042.033-.083.07c-1.508 1.37-3.508 2.204-5.707 2.204-4.688 0-8.488-3.8-8.488-8.49 0-4.687 3.8-8.488 8.488-8.488 2.867 0 5.403 1.42 6.941 3.6l.005.007.451.63c.402.562 1.182.692 1.744.29.562-.4.691-1.182.29-1.743l-.45-.628c-1.987-2.814-5.268-4.656-8.98-4.656-6.07 0-10.99 4.92-10.99 10.989s4.92 10.989 10.99 10.989c2.512 0 4.83-.845 6.68-2.264q.082-.046.158-.106c1.434-1.131 2.728-3 3.7-4.634.992-1.668 1.736-3.245 2.039-3.913.288-.637.984-1.889 1.912-3.22.934-1.337 2.036-2.658 3.116-3.506l.072-.06.025-.02c1.432-1.1 3.222-1.755 5.17-1.755h1.993l-3.276 3.276c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0m2.214 14.434c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.7c-2.798 0-5.077-1.609-7.09-3.803l-.527-.638c-.44-.532-1.228-.606-1.76-.166s-.606 1.228-.166 1.76l.547.662.02.024.02.023c2.187 2.39 5.095 4.638 8.957 4.638z"
                })
            }),
            /*#__PURE__*/ (0, $kDNWt$jsx)("g", {
                clipPath: "url(#b18be3__a)",
                children: /*#__PURE__*/ (0, $kDNWt$jsxs)("g", {
                    clipPath: "url(#b18be3__b)",
                    children: [
                        /*#__PURE__*/ (0, $kDNWt$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $kDNWt$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $kDNWt$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#b18be3__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $kDNWt$jsx)("path", {
                            fill: "url(#b18be3__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $kDNWt$jsx)("path", {
                            fill: "url(#b18be3__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $kDNWt$jsx)("path", {
                            fill: "url(#b18be3__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $kDNWt$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#b18be3__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $600200914f8e387b$var$ForwardRef = /*#__PURE__*/ (0, $kDNWt$forwardRef)($600200914f8e387b$var$SvgComponent);
var $600200914f8e387b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($600200914f8e387b$var$ForwardRef);


export {$600200914f8e387b$export$2e2bcd8739ae039 as default};
