// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_replace_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $9VbKF$jsxs, jsx as $9VbKF$jsx} from "react/jsx-runtime";
import {forwardRef as $9VbKF$forwardRef} from "react";





const $eb26f6566fd87275$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9VbKF$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9VbKF$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9VbKF$jsxs)("linearGradient", {
                        id: "b7390d__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9VbKF$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $9VbKF$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9VbKF$jsxs)("linearGradient", {
                        id: "b7390d__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9VbKF$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9VbKF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9VbKF$jsxs)("linearGradient", {
                        id: "b7390d__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9VbKF$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9VbKF$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9VbKF$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9VbKF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9VbKF$jsxs)("radialGradient", {
                        id: "b7390d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9VbKF$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $9VbKF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9VbKF$jsxs)("radialGradient", {
                        id: "b7390d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9VbKF$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9VbKF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9VbKF$jsx)("clipPath", {
                        id: "b7390d__b",
                        children: /*#__PURE__*/ (0, $9VbKF$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9VbKF$jsx)("clipPath", {
                id: "b7390d__a",
                children: /*#__PURE__*/ (0, $9VbKF$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M12.113 7.875c-2.347 0-4.25 1.903-4.25 4.25v5.31c0 2.348 1.903 4.25 4.25 4.25h5.311c2.347 0 4.25-1.902 4.25-4.25v-5.31c0-2.347-1.903-4.25-4.25-4.25zm15.332.555c-.69-.023-1.268.517-1.291 1.207-.024.69.517 1.269 1.206 1.292 1.387.047 2.794.446 3.853 1.47.879.85 1.647 2.258 1.867 4.61l-1.584-1.583c-.488-.489-1.28-.489-1.768 0-.488.488-.488 1.28 0 1.767l3.573 3.573c.488.488 1.28.488 1.767 0l3.584-3.584c.488-.488.488-1.28 0-1.768s-1.28-.488-1.767 0l-1.32 1.32c-.27-2.785-1.218-4.78-2.614-6.132-1.646-1.593-3.733-2.112-5.506-2.172M14.652 27.234c-.488-.488-1.28-.488-1.768 0L9.3 30.818c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0l1.32-1.32c.268 2.785 1.217 4.78 2.613 6.132 1.647 1.593 3.734 2.111 5.506 2.172.69.023 1.268-.517 1.292-1.207.023-.69-.517-1.269-1.207-1.292-1.386-.047-2.794-.446-3.853-1.47-.878-.85-1.647-2.258-1.867-4.61l1.584 1.584c.488.488 1.28.488 1.768 0 .488-.489.488-1.28 0-1.768zm15.922-.92c-2.347 0-4.25 1.903-4.25 4.25v5.311c0 2.348 1.902 4.25 4.25 4.25h5.31c2.348 0 4.25-1.903 4.25-4.25v-5.31c0-2.348-1.902-4.25-4.25-4.25z"
                })
            }),
            /*#__PURE__*/ (0, $9VbKF$jsx)("g", {
                clipPath: "url(#b7390d__a)",
                children: /*#__PURE__*/ (0, $9VbKF$jsxs)("g", {
                    clipPath: "url(#b7390d__b)",
                    children: [
                        /*#__PURE__*/ (0, $9VbKF$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $9VbKF$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $9VbKF$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#b7390d__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $9VbKF$jsx)("path", {
                            fill: "url(#b7390d__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $9VbKF$jsx)("path", {
                            fill: "url(#b7390d__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $9VbKF$jsx)("path", {
                            fill: "url(#b7390d__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $9VbKF$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#b7390d__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $eb26f6566fd87275$var$ForwardRef = /*#__PURE__*/ (0, $9VbKF$forwardRef)($eb26f6566fd87275$var$SvgComponent);
var $eb26f6566fd87275$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($eb26f6566fd87275$var$ForwardRef);


export {$eb26f6566fd87275$export$2e2bcd8739ae039 as default};
