// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_EmptyStateExport_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $FAkPb$reactjsxruntime = require("react/jsx-runtime");
var $FAkPb$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4b53156365fc9597$export$2e2bcd8739ae039);




const $4b53156365fc9597$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "28bfb9__d",
                        x1: -13.157,
                        x2: 117.977,
                        y1: 79.84,
                        y2: -4.813,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "28bfb9__e",
                        x1: 95.656,
                        x2: 41.164,
                        y1: 24.077,
                        y2: 86.248,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "28bfb9__f",
                        x1: 24.965,
                        x2: 125.073,
                        y1: 91.161,
                        y2: 31.657,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsxs)("radialGradient", {
                        id: "28bfb9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.1953 -36.2836 0 36.284 34.195)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsxs)("radialGradient", {
                        id: "28bfb9__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.1953 -36.2836 0 36.284 34.195)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("clipPath", {
                        id: "28bfb9__b",
                        children: /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("rect", {
                            width: 83.4,
                            height: 78.6,
                            x: 7.801,
                            y: 9.6,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("clipPath", {
                id: "28bfb9__a",
                children: /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M67.2 9.6h12c3.977 0 7.2 3.223 7.2 7.2v26.4c0 3.976-3.223 7.2-7.2 7.2h-12c-3.976 0-7.2-3.224-7.2-7.2V16.8c0-3.977 3.224-7.2 7.2-7.2M34.8 60h26.4c3.977 0 7.2 3.223 7.2 7.2v12c0 3.976-3.223 7.2-7.2 7.2H34.8c-3.976 0-7.2-3.224-7.2-7.2v-12c0-3.977 3.224-7.2 7.2-7.2m4.951-17.118 4.956 12.072c.804 1.95 3.552 1.992 4.41.066l1.446-3.24c.24-.54.672-.973 1.212-1.212l3.234-1.447c1.926-.858 1.884-3.611-.066-4.41L42.877 39.75c-1.974-.81-3.942 1.158-3.132 3.133zm-5.898 2.328c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4h-12.27c-.894 1.836-2.766 3.102-4.944 3.102-3.042 0-5.502-2.466-5.502-5.502 0-2.178 1.272-4.05 3.102-4.944V21.54c-1.836-.894-3.102-2.766-3.102-4.944 0-3.042 2.466-5.502 5.502-5.502 2.178 0 4.05 1.272 4.944 3.102h21.036c.894-1.836 2.766-3.102 4.944-3.102 3.042 0 5.502 2.466 5.502 5.502 0 2.178-1.272 4.05-3.102 4.944v12.3c0 1.326-1.074 2.4-2.4 2.4s-2.4-1.074-2.4-2.4v-12.3c-1.11-.54-2.004-1.44-2.544-2.544H21.583c-.54 1.11-1.44 2.004-2.544 2.544v21.126c1.11.54 2.004 1.44 2.544 2.544z"
                })
            }),
            /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("g", {
                clipPath: "url(#28bfb9__a)",
                children: /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#28bfb9__b)",
                    children: [
                        /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("rect", {
                            width: 83.4,
                            height: 78.6,
                            x: 7.801,
                            y: 9.6,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M7.8 9.6h83.4v78.6H7.8z"
                        }),
                        /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("ellipse", {
                            cx: 36.284,
                            cy: 34.195,
                            fill: "url(#28bfb9__c)",
                            rx: 36.284,
                            ry: 34.195,
                            transform: "matrix(1 -.00132 .00148 1 18.621 -21.499)"
                        }),
                        /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("path", {
                            fill: "url(#28bfb9__d)",
                            d: "M39.635-4.285c-6.405-3.735-14.52-3.758-20.95-.06l-64.3 37.001c-6.297 3.624-6.307 12.177-.018 15.815l63.875 36.947c6.419 3.713 14.534 3.708 20.948-.013l63.743-36.982c6.256-3.63 6.267-12.135.02-15.779z"
                        }),
                        /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("path", {
                            fill: "url(#28bfb9__e)",
                            d: "M19.82 46.917c1.426-29.344 27.822-52.043 58.958-50.7 31.136 1.344 55.221 26.22 53.795 55.565-1.425 29.343-27.821 52.042-58.957 50.699s-55.22-26.22-53.795-55.564"
                        }),
                        /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("path", {
                            fill: "url(#28bfb9__f)",
                            d: "M-25.987 66.265c12.376-4.976 25.724-4.763 37.288-.41 23.14 8.73 36.605 3.332 45.944-18.426 4.652-10.919 13.734-20.165 26.124-25.147 24.773-9.945 53.372.846 63.909 24.124s-.996 50.189-25.763 60.146c-12.418 4.993-25.792 4.759-37.374.367-23.103-8.684-36.542-3.265-45.869 18.519-4.663 10.893-13.743 20.108-26.12 25.084-24.753 9.952-53.352-.839-63.883-24.104-10.53-23.264.99-50.201 25.744-60.153"
                        }),
                        /*#__PURE__*/ (0, $FAkPb$reactjsxruntime.jsx)("ellipse", {
                            cx: 36.284,
                            cy: 34.195,
                            fill: "url(#28bfb9__g)",
                            rx: 36.284,
                            ry: 34.195,
                            transform: "matrix(1 -.00132 .00148 1 50.004 -10.348)"
                        })
                    ]
                })
            })
        ]
    });
const $4b53156365fc9597$var$ForwardRef = /*#__PURE__*/ (0, $FAkPb$react.forwardRef)($4b53156365fc9597$var$SvgComponent);
var $4b53156365fc9597$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4b53156365fc9597$var$ForwardRef);
