// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_pin_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $W4BVZ$reactjsxruntime = require("react/jsx-runtime");
var $W4BVZ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a2b6e8e18b23edd3$export$2e2bcd8739ae039);




const $a2b6e8e18b23edd3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "542bd3__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "542bd3__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "542bd3__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "542bd3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "542bd3__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("clipPath", {
                        id: "542bd3__b",
                        children: /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("clipPath", {
                id: "542bd3__a",
                children: /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M58.73 43.546h1.156c3.505 0 6.114 2.911 6.114 6.235S63.158 55.8 59.653 55.8H50.4v25.8c0 1.325-1.075 2.4-2.4 2.4s-2.4-1.075-2.4-2.4V55.8h-9.253C32.842 55.8 30 53.105 30 49.78s2.608-5.802 6.114-5.802h1.156V29.054h-1.156c-3.505 0-6.114-2.911-6.114-6.235s2.842-6.02 6.347-6.02h23.306c3.505 0 6.347 2.695 6.347 6.02s-2.608 5.802-6.114 5.802H58.73z"
                })
            }),
            /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("g", {
                clipPath: "url(#542bd3__a)",
                children: /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#542bd3__b)",
                    children: [
                        /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#542bd3__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("path", {
                            fill: "url(#542bd3__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("path", {
                            fill: "url(#542bd3__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("path", {
                            fill: "url(#542bd3__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $W4BVZ$reactjsxruntime.jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#542bd3__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $a2b6e8e18b23edd3$var$ForwardRef = /*#__PURE__*/ (0, $W4BVZ$react.forwardRef)($a2b6e8e18b23edd3$var$SvgComponent);
var $a2b6e8e18b23edd3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a2b6e8e18b23edd3$var$ForwardRef);
