// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_filmstrip_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bLoul$jsxs, jsx as $bLoul$jsx} from "react/jsx-runtime";
import {forwardRef as $bLoul$forwardRef} from "react";





const $c3bef3dac87f14fe$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bLoul$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bLoul$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bLoul$jsxs)("linearGradient", {
                        id: "4b5cda__d",
                        x1: -14.982,
                        x2: 120.749,
                        y1: 80.605,
                        y2: -1.972,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bLoul$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bLoul$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bLoul$jsxs)("linearGradient", {
                        id: "4b5cda__e",
                        x1: 93.945,
                        x2: 35.717,
                        y1: 21.377,
                        y2: 83.989,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bLoul$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bLoul$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bLoul$jsxs)("linearGradient", {
                        id: "4b5cda__f",
                        x1: 23.18,
                        x2: 126.403,
                        y1: 92.63,
                        y2: 34.806,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bLoul$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bLoul$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bLoul$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bLoul$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bLoul$jsxs)("radialGradient", {
                        id: "4b5cda__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 21.982 31.22)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bLoul$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bLoul$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bLoul$jsxs)("radialGradient", {
                        id: "4b5cda__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 31.768 52.85)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bLoul$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bLoul$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bLoul$jsx)("clipPath", {
                        id: "4b5cda__b",
                        children: /*#__PURE__*/ (0, $bLoul$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6,
                            y: 6,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bLoul$jsx)("clipPath", {
                id: "4b5cda__a",
                children: /*#__PURE__*/ (0, $bLoul$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M21.6 20.5c-5.302 0-9.6 4.298-9.6 9.6v36c0 5.302 4.298 9.6 9.6 9.6h52.8c5.302 0 9.6-4.298 9.6-9.6v-36c0-5.302-4.298-9.6-9.6-9.6zm-2.4 11.4c0-.663.537-1.2 1.2-1.2h4.2c.663 0 1.2.537 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2zm0 29.4c0-.663.537-1.2 1.2-1.2h4.2c.663 0 1.2.537 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2zm10.2-29.4c0-.663.537-1.2 1.2-1.2h4.2c.663 0 1.2.537 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2zm0 29.4c0-.663.537-1.2 1.2-1.2h4.2c.663 0 1.2.537 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2zm10.2-29.4c0-.663.537-1.2 1.2-1.2H45c.663 0 1.2.537 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2zm0 29.4c0-.663.537-1.2 1.2-1.2H45c.663 0 1.2.537 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2zm10.2-29.4c0-.663.537-1.2 1.2-1.2h4.2c.663 0 1.2.537 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2H51c-.663 0-1.2-.537-1.2-1.2zm0 29.4c0-.663.537-1.2 1.2-1.2h4.2c.663 0 1.2.537 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2H51c-.663 0-1.2-.537-1.2-1.2zM60 31.9c0-.663.537-1.2 1.2-1.2h4.2c.663 0 1.2.537 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2zm0 29.4c0-.663.537-1.2 1.2-1.2h4.2c.663 0 1.2.537 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2zm10.2-29.4c0-.663.537-1.2 1.2-1.2h4.2c.663 0 1.2.537 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2zm0 29.4c0-.663.537-1.2 1.2-1.2h4.2c.663 0 1.2.537 1.2 1.2v3c0 .663-.537 1.2-1.2 1.2h-4.2c-.663 0-1.2-.537-1.2-1.2z"
                })
            }),
            /*#__PURE__*/ (0, $bLoul$jsx)("g", {
                clipPath: "url(#4b5cda__a)",
                children: /*#__PURE__*/ (0, $bLoul$jsxs)("g", {
                    clipPath: "url(#4b5cda__b)",
                    children: [
                        /*#__PURE__*/ (0, $bLoul$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6,
                            y: 6,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $bLoul$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6 6h83.485v83.485H6z"
                        }),
                        /*#__PURE__*/ (0, $bLoul$jsx)("circle", {
                            cx: 53.203,
                            cy: 9.238,
                            r: 36.321,
                            fill: "url(#4b5cda__c)",
                            transform: "rotate(-.08 53.203 9.238)"
                        }),
                        /*#__PURE__*/ (0, $bLoul$jsx)("path", {
                            fill: "url(#4b5cda__d)",
                            d: "M37.863-8.748c-6.411-3.967-14.535-3.991-20.97-.062l-64.366 39.3c-6.304 3.848-6.314 12.933-.019 16.797l63.94 39.244c6.426 3.943 14.55 3.938 20.97-.015l63.808-39.28c6.262-3.855 6.274-12.89.019-16.76z"
                        }),
                        /*#__PURE__*/ (0, $bLoul$jsx)("path", {
                            fill: "url(#4b5cda__e)",
                            d: "M18.033 45.636C19.46 14.47 45.883-9.64 77.05-8.214s55.277 27.85 53.85 59.018-27.85 55.277-59.017 53.85-55.277-27.85-53.85-59.018"
                        }),
                        /*#__PURE__*/ (0, $bLoul$jsx)("path", {
                            fill: "url(#4b5cda__f)",
                            d: "M-27.824 66.187c12.389-5.285 25.75-5.059 37.326-.436 23.164 9.274 36.643 3.54 45.991-19.57 4.657-11.598 13.748-21.42 26.15-26.71 24.799-10.563 53.427.898 63.975 25.623s-.997 53.308-25.79 63.884c-12.43 5.303-25.818 5.055-37.411.39-23.127-9.224-36.58-3.469-45.916 19.67-4.668 11.569-13.757 21.357-26.147 26.643-24.778 10.57-53.406-.891-63.948-25.602s.992-53.322 25.77-63.892"
                        }),
                        /*#__PURE__*/ (0, $bLoul$jsx)("circle", {
                            cx: 84.617,
                            cy: 21.082,
                            r: 36.321,
                            fill: "url(#4b5cda__g)",
                            transform: "rotate(-.08 84.617 21.082)"
                        })
                    ]
                })
            })
        ]
    });
const $c3bef3dac87f14fe$var$ForwardRef = /*#__PURE__*/ (0, $bLoul$forwardRef)($c3bef3dac87f14fe$var$SvgComponent);
var $c3bef3dac87f14fe$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c3bef3dac87f14fe$var$ForwardRef);


export {$c3bef3dac87f14fe$export$2e2bcd8739ae039 as default};
