// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_movieCamera_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $5ddyM$reactjsxruntime = require("react/jsx-runtime");
var $5ddyM$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $14f6268d62cde97c$export$2e2bcd8739ae039);




const $14f6268d62cde97c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9556c4__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 83.29,
                        y2: -11.667,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9556c4__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 15.183,
                        y2: 87.18,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9556c4__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 97.117,
                        y2: 30.625,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsxs)("radialGradient", {
                        id: "9556c4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 26.529 27.752)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsxs)("radialGradient", {
                        id: "9556c4__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 37.781 52.625)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("clipPath", {
                        id: "9556c4__b",
                        children: /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: -2.5,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("clipPath", {
                id: "9556c4__a",
                children: /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    d: "M61.674 34.095v7.769l14.192-10.532c2.64-1.96 6.384-.075 6.384 3.212v26.815c0 3.278-3.728 5.164-6.37 3.222L61.675 54.137v7.77c0 4.418-3.582 8-8 8H21.75c-4.418 0-8-3.582-8-8V34.094c0-4.419 3.582-8 8-8h31.924c4.418 0 8 3.582 8 8"
                })
            }),
            /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("g", {
                clipPath: "url(#9556c4__a)",
                children: /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#9556c4__b)",
                    children: [
                        /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: -2.5,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0-2.5h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 1.223,
                            r: 41.765,
                            fill: "url(#9556c4__c)",
                            transform: "rotate(-.08 54.28 1.223)"
                        }),
                        /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("path", {
                            fill: "url(#9556c4__d)",
                            d: "M36.643-19.458c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("path", {
                            fill: "url(#9556c4__e)",
                            d: "M13.837 43.079C15.477 7.239 45.86-20.484 81.7-18.844s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("path", {
                            fill: "url(#9556c4__f)",
                            d: "M-38.895 66.71c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $5ddyM$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 14.844,
                            r: 41.765,
                            fill: "url(#9556c4__g)",
                            transform: "rotate(-.08 90.406 14.844)"
                        })
                    ]
                })
            })
        ]
    });
const $14f6268d62cde97c$var$ForwardRef = /*#__PURE__*/ (0, $5ddyM$react.forwardRef)($14f6268d62cde97c$var$SvgComponent);
var $14f6268d62cde97c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($14f6268d62cde97c$var$ForwardRef);
