// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_pieChart_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $ktYZr$reactjsxruntime = require("react/jsx-runtime");
var $ktYZr$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3c344f3ee7ca57f8$export$2e2bcd8739ae039);




const $3c344f3ee7ca57f8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "809480__d",
                        x1: -3.945,
                        x2: 55.309,
                        y1: 38.702,
                        y2: 0.46,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "809480__e",
                        x1: 45.217,
                        x2: 20.591,
                        y1: 13.503,
                        y2: 41.593,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "809480__f",
                        x1: 13.277,
                        x2: 58.511,
                        y1: 43.818,
                        y2: 16.938,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsxs)("radialGradient", {
                        id: "809480__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsxs)("radialGradient", {
                        id: "809480__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("clipPath", {
                        id: "809480__b",
                        children: /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.523,
                            y: 6.96,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("clipPath", {
                id: "809480__a",
                children: /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M22.8 7.248c-3.735.264-7.134 1.749-9.798 4.059l9.798 9.798zm-11.494 5.754C8.75 15.948 7.2 19.792 7.2 24c0 4.21 1.551 8.05 4.107 10.998L22.304 24zM25.2 24V7.251c8.718.615 15.6 7.473 15.6 16.752s-7.521 16.8-16.8 16.8c-4.209 0-8.052-1.551-10.998-4.107l11.847-11.847.01-.01.048-.053q.049-.06.092-.123.039-.064.069-.132l.006-.012.01-.016q.012-.022.023-.047.024-.061.04-.125l.008-.031.003-.01.006-.017.012-.039q.023-.117.024-.234m-.036.273.012-.039-.01.027q-.006.019-.011.04.003-.015.009-.028"
                })
            }),
            /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("g", {
                clipPath: "url(#809480__a)",
                children: /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#809480__b)",
                    children: [
                        /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.523,
                            y: 6.96,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M5.523 6.96h37.68v35.52H5.523z"
                        }),
                        /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#809480__c)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 10.414 -7.094)"
                        }),
                        /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("path", {
                            fill: "url(#809480__d)",
                            d: "M19.906.685c-2.894-1.688-6.56-1.698-9.465-.026l-29.05 16.72c-2.845 1.638-2.85 5.503-.009 7.147l28.859 16.697c2.9 1.678 6.566 1.676 9.464-.006l28.8-16.712c2.826-1.64 2.83-5.485.008-7.131z"
                        }),
                        /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("path", {
                            fill: "url(#809480__e)",
                            d: "M10.955 23.824C11.6 10.564 23.525.306 37.592.913s24.949 11.849 24.305 25.11-12.57 23.518-26.637 22.91c-14.067-.606-24.949-11.848-24.305-25.109"
                        }),
                        /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("path", {
                            fill: "url(#809480__f)",
                            d: "M-9.743 32.567c5.591-2.248 11.622-2.152 16.847-.185 10.454 3.946 16.538 1.506 20.757-8.327 2.102-4.934 6.205-9.113 11.803-11.364 11.192-4.494 24.113.382 28.874 10.902s-.45 22.68-11.64 27.18c-5.61 2.257-11.653 2.151-16.885.166-10.438-3.924-16.51-1.475-20.724 8.37-2.107 4.922-6.21 9.086-11.8 11.335-11.184 4.497-24.105-.38-28.863-10.893s.447-22.686 11.63-27.184"
                        }),
                        /*#__PURE__*/ (0, $ktYZr$reactjsxruntime.jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#809480__g)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 24.59 -2.055)"
                        })
                    ]
                })
            })
        ]
    });
const $3c344f3ee7ca57f8$var$ForwardRef = /*#__PURE__*/ (0, $ktYZr$react.forwardRef)($3c344f3ee7ca57f8$var$SvgComponent);
var $3c344f3ee7ca57f8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3c344f3ee7ca57f8$var$ForwardRef);
