// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_chartAreaStack_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $dwZmO$reactjsxruntime = require("react/jsx-runtime");
var $dwZmO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $696993301adcbbce$export$2e2bcd8739ae039);




const $696993301adcbbce$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5b8d90__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.894,
                        y2: -4.584,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5b8d90__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.841,
                        y2: 44.839,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5b8d90__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.808,
                        y2: 16.562,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5b8d90__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5b8d90__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("clipPath", {
                        id: "5b8d90__b",
                        children: /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: -0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("clipPath", {
                id: "5b8d90__a",
                children: /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("path", {
                    fill: "#8F8F8F",
                    fillRule: "evenodd",
                    d: "M35.743 11.242c1.195-2.164 4.497-1.323 4.497 1.157v23.697c0 1.958-1.59 3.542-3.545 3.542h-25.15C9.59 39.638 8 38.054 8 36.096V23.569c0-.66.273-1.291.753-1.744l4.887-4.602c.993-.935 2.565-.855 3.458.175l3.444 3.974 4.546-7.02c.883-1.364 2.84-1.473 3.869-.216l3.098 3.784zM20.468 21.486Zm17.272-8.69-3.57 6.464-1.094-.604 1.094.604c-.82 1.485-2.88 1.671-3.955.36l-3.104-3.792-4.545 7.017c-.868 1.34-2.78 1.473-3.825.266l-3.463-3.996-4.778 4.5v6.68l4.1-2.637c.969-.623 2.224-.573 3.14.126l3.88 2.96c.205.156.495.125.662-.07l4.92-5.712c1.037-1.206 2.881-1.275 4.007-.151l3.55 3.544 2.981-1.427zm0 16.904-2.699 1.292c-.478.229-1.048.131-1.423-.243l-4.175-4.169c-.098-.097-.258-.091-.347.013l-4.92 5.713c-1.03 1.197-2.815 1.384-4.071.426l-3.88-2.96c-.08-.06-.189-.065-.273-.01L10.5 33.267v2.828c0 .573.466 1.042 1.046 1.042h25.149c.579 0 1.045-.469 1.045-1.042zm-27.273-6.055-.072-.076zm16.72-7.934-.086-.055z"
                })
            }),
            /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("g", {
                clipPath: "url(#5b8d90__a)",
                children: /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#5b8d90__b)",
                    children: [
                        /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: -0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0-.001h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.861,
                            r: 20.883,
                            fill: "url(#5b8d90__c)",
                            transform: "rotate(-.08 27.142 1.861)"
                        }),
                        /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("path", {
                            fill: "url(#5b8d90__d)",
                            d: "M18.321-8.48c-3.686-2.28-8.357-2.295-12.057-.036L-30.743 14.08c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.3c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("path", {
                            fill: "url(#5b8d90__e)",
                            d: "M6.916 22.788C7.736 4.868 22.93-8.993 40.85-8.172S72.63 7.84 71.809 25.758c-.82 17.92-16.012 31.782-33.931 30.962-17.92-.82-31.782-16.013-30.962-33.933"
                        }),
                        /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("path", {
                            fill: "url(#5b8d90__f)",
                            d: "M-19.448 34.604c7.124-3.039 14.806-2.908 21.462-.25C15.332 39.686 23.08 36.389 28.456 23.1c2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $dwZmO$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.671,
                            r: 20.883,
                            fill: "url(#5b8d90__g)",
                            transform: "rotate(-.08 45.2 8.67)"
                        })
                    ]
                })
            })
        ]
    });
const $696993301adcbbce$var$ForwardRef = /*#__PURE__*/ (0, $dwZmO$react.forwardRef)($696993301adcbbce$var$SvgComponent);
var $696993301adcbbce$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($696993301adcbbce$var$ForwardRef);
