// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_replace_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gfj8N$reactjsxruntime = require("react/jsx-runtime");
var $gfj8N$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2443512063ae9773$export$2e2bcd8739ae039);




const $2443512063ae9773$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ee8876__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ee8876__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ee8876__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ee8876__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ee8876__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("clipPath", {
                        id: "ee8876__b",
                        children: /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("clipPath", {
                id: "ee8876__a",
                children: /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M24.23 16.25c-4.418 0-8 3.582-8 8v10.622c0 4.418 3.582 8 8 8h10.622c4.419 0 8-3.582 8-8V24.25c0-4.418-3.581-8-8-8zm30.647 1.11c-1.104-.037-2.03.828-2.067 1.932s.827 2.029 1.931 2.066c2.85.097 5.801.919 8.036 3.08 2.03 1.965 3.718 5.278 3.976 10.878l-4.111-4.111c-.781-.781-2.048-.781-2.829 0-.78.78-.78 2.047 0 2.828l7.145 7.145c.781.78 2.047.78 2.829 0l7.167-7.168c.781-.781.781-2.047 0-2.828-.78-.781-2.047-.781-2.828 0l-3.407 3.406c-.378-6.05-2.3-10.257-5.161-13.024-3.176-3.072-7.214-4.086-10.681-4.204M28.953 54.82c-.781-.78-2.048-.78-2.829 0l-7.167 7.168c-.782.781-.782 2.048 0 2.829.78.78 2.047.78 2.828 0l3.406-3.407c.379 6.051 2.302 10.257 5.162 13.024 3.175 3.072 7.213 4.086 10.68 4.204 1.105.037 2.03-.828 2.067-1.932.038-1.103-.827-2.029-1.93-2.066-2.85-.097-5.802-.919-8.036-3.08-2.03-1.965-3.718-5.278-3.977-10.878l4.112 4.112c.781.78 2.047.78 2.828 0 .781-.781.781-2.048 0-2.829zM61.15 53.13c-4.418 0-8 3.582-8 8V71.75c0 4.418 3.582 8 8 8h10.622c4.418 0 8-3.582 8-8V61.129c0-4.418-3.582-8-8-8z"
                })
            }),
            /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ee8876__a)",
                children: /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ee8876__b)",
                    children: [
                        /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#ee8876__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("path", {
                            fill: "url(#ee8876__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("path", {
                            fill: "url(#ee8876__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("path", {
                            fill: "url(#ee8876__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $gfj8N$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#ee8876__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $2443512063ae9773$var$ForwardRef = /*#__PURE__*/ (0, $gfj8N$react.forwardRef)($2443512063ae9773$var$SvgComponent);
var $2443512063ae9773$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2443512063ae9773$var$ForwardRef);
