// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_sparkles_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $aPotf$jsxs, jsx as $aPotf$jsx} from "react/jsx-runtime";
import {forwardRef as $aPotf$forwardRef} from "react";





const $83de9b6bc3c4dcca$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aPotf$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aPotf$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aPotf$jsxs)("linearGradient", {
                        id: "9668db__d",
                        x1: -3.949,
                        x2: 55.306,
                        y1: 38.702,
                        y2: 0.46,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aPotf$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $aPotf$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aPotf$jsxs)("linearGradient", {
                        id: "9668db__e",
                        x1: 45.213,
                        x2: 20.587,
                        y1: 13.503,
                        y2: 41.593,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aPotf$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aPotf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aPotf$jsxs)("linearGradient", {
                        id: "9668db__f",
                        x1: 13.273,
                        x2: 58.507,
                        y1: 43.818,
                        y2: 16.938,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aPotf$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aPotf$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aPotf$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aPotf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aPotf$jsxs)("radialGradient", {
                        id: "9668db__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aPotf$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $aPotf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aPotf$jsxs)("radialGradient", {
                        id: "9668db__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aPotf$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aPotf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aPotf$jsx)("clipPath", {
                        id: "9668db__b",
                        children: /*#__PURE__*/ (0, $aPotf$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aPotf$jsx)("clipPath", {
                id: "9668db__a",
                children: /*#__PURE__*/ (0, $aPotf$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m37.86 25.323-5.174-1.164c-.768-.174-1.575.042-2.154.576l-3.9 3.594c-1.731 1.596-4.485.006-3.97-2.292l1.165-5.175c.174-.768-.042-1.575-.576-2.154l-3.594-3.9c-1.596-1.73-.006-4.485 2.292-3.969l5.175 1.164c.768.174 1.575-.042 2.154-.576l3.9-3.594c1.73-1.596 4.485-.006 3.969 2.292L35.983 15.3c-.174.768.042 1.575.576 2.154l3.594 3.9c1.596 1.731.006 4.485-2.292 3.97M19.28 38.958l-3.649-.822c-.483-.108-.984.027-1.347.36l-2.748 2.535c-1.083 1-2.805.003-2.48-1.43l.821-3.649c.108-.483-.027-.984-.36-1.347l-2.535-2.748c-.999-1.083-.003-2.805 1.431-2.48l3.648.821c.483.108.984-.027 1.347-.36l2.748-2.535c1.083-.999 2.805-.003 2.481 1.431l-.822 3.648c-.108.483.027.984.36 1.347l2.535 2.748c1 1.083.003 2.805-1.43 2.481"
                })
            }),
            /*#__PURE__*/ (0, $aPotf$jsx)("g", {
                clipPath: "url(#9668db__a)",
                children: /*#__PURE__*/ (0, $aPotf$jsxs)("g", {
                    clipPath: "url(#9668db__b)",
                    children: [
                        /*#__PURE__*/ (0, $aPotf$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $aPotf$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M5.52 6.96H43.2v35.52H5.52z"
                        }),
                        /*#__PURE__*/ (0, $aPotf$jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#9668db__c)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 10.41 -7.094)"
                        }),
                        /*#__PURE__*/ (0, $aPotf$jsx)("path", {
                            fill: "url(#9668db__d)",
                            d: "M19.902.685c-2.894-1.688-6.56-1.698-9.465-.026l-29.05 16.72c-2.845 1.638-2.85 5.503-.009 7.147l28.859 16.697c2.9 1.678 6.567 1.676 9.464-.006l28.8-16.712c2.826-1.64 2.83-5.485.008-7.131z"
                        }),
                        /*#__PURE__*/ (0, $aPotf$jsx)("path", {
                            fill: "url(#9668db__e)",
                            d: "M10.951 23.824C11.595 10.564 23.521.306 37.588.913s24.949 11.849 24.305 25.11-12.57 23.518-26.637 22.91c-14.067-.606-24.949-11.848-24.305-25.109"
                        }),
                        /*#__PURE__*/ (0, $aPotf$jsx)("path", {
                            fill: "url(#9668db__f)",
                            d: "M-9.747 32.567c5.591-2.248 11.622-2.152 16.847-.185 10.455 3.946 16.538 1.506 20.757-8.327 2.102-4.934 6.205-9.113 11.803-11.364 11.192-4.494 24.113.382 28.874 10.902s-.45 22.68-11.64 27.18c-5.61 2.257-11.652 2.151-16.885.167-10.438-3.925-16.51-1.476-20.724 8.368-2.107 4.923-6.21 9.087-11.8 11.336C-3.7 75.14-16.62 70.264-21.379 59.75s.447-22.686 11.63-27.184"
                        }),
                        /*#__PURE__*/ (0, $aPotf$jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#9668db__g)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 24.586 -2.055)"
                        })
                    ]
                })
            })
        ]
    });
const $83de9b6bc3c4dcca$var$ForwardRef = /*#__PURE__*/ (0, $aPotf$forwardRef)($83de9b6bc3c4dcca$var$SvgComponent);
var $83de9b6bc3c4dcca$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($83de9b6bc3c4dcca$var$ForwardRef);


export {$83de9b6bc3c4dcca$export$2e2bcd8739ae039 as default};
