// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_folderOpen_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $14EEG$jsxs, jsx as $14EEG$jsx} from "react/jsx-runtime";
import {forwardRef as $14EEG$forwardRef} from "react";





const $b13faf629a84668a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $14EEG$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $14EEG$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $14EEG$jsxs)("linearGradient", {
                        id: "95b76e__d",
                        x1: -9.82,
                        x2: 62.963,
                        y1: 40.006,
                        y2: -4.275,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $14EEG$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $14EEG$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $14EEG$jsxs)("linearGradient", {
                        id: "95b76e__e",
                        x1: 48.588,
                        x2: 17.364,
                        y1: 8.246,
                        y2: 41.82,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $14EEG$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $14EEG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $14EEG$jsxs)("linearGradient", {
                        id: "95b76e__f",
                        x1: 10.645,
                        x2: 65.996,
                        y1: 46.455,
                        y2: 15.447,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $14EEG$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $14EEG$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $14EEG$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $14EEG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $14EEG$jsxs)("radialGradient", {
                        id: "95b76e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.503 14.239)scale(19.4763)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $14EEG$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $14EEG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $14EEG$jsxs)("radialGradient", {
                        id: "95b76e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 17.75 25.839)scale(19.4763)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $14EEG$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $14EEG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $14EEG$jsx)("clipPath", {
                        id: "95b76e__b",
                        children: /*#__PURE__*/ (0, $14EEG$jsx)("rect", {
                            width: 44.767,
                            height: 44.767,
                            x: 1.43,
                            fill: "#fff",
                            rx: 1.119
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $14EEG$jsx)("clipPath", {
                id: "95b76e__a",
                children: /*#__PURE__*/ (0, $14EEG$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M38.847 20.4v-1.309c0-2.214-1.798-4.009-4.016-4.009H20.778l-4.55-5.679C15.718 8.77 14.95 8.4 14.138 8.4H6.723c-1.479 0-2.677 1.197-2.677 2.673v24.518q.002.572.152 1.096c.03-.511.14-1.034.344-1.554q.007-.021.017-.042L9.296 23.61c.778-1.96 2.686-3.209 4.764-3.209zm4.63 2.4H14.06c-1.127 0-2.137.68-2.538 1.706l-4.744 11.5c-.676 1.731.63 3.586 2.527 3.594h29.428c1.128 0 2.138-.678 2.54-1.706l4.743-11.5c.652-1.666-.536-3.448-2.318-3.585q-.11-.009-.22-.009"
                })
            }),
            /*#__PURE__*/ (0, $14EEG$jsx)("g", {
                clipPath: "url(#95b76e__a)",
                children: /*#__PURE__*/ (0, $14EEG$jsxs)("g", {
                    clipPath: "url(#95b76e__b)",
                    children: [
                        /*#__PURE__*/ (0, $14EEG$jsx)("rect", {
                            width: 44.767,
                            height: 44.767,
                            x: 1.43,
                            fill: "#FFECCF",
                            rx: 1.119
                        }),
                        /*#__PURE__*/ (0, $14EEG$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M1.43 0h44.767v44.767H1.43z"
                        }),
                        /*#__PURE__*/ (0, $14EEG$jsx)("circle", {
                            cx: 26.742,
                            cy: 1.736,
                            r: 19.476,
                            fill: "url(#95b76e__c)",
                            transform: "rotate(-.08 26.742 1.736)"
                        }),
                        /*#__PURE__*/ (0, $14EEG$jsx)("path", {
                            fill: "url(#95b76e__d)",
                            d: "M18.517-7.908c-3.438-2.127-7.794-2.14-11.245-.034l-34.515 21.074c-3.38 2.064-3.386 6.936-.01 9.008L7.034 43.183c3.446 2.115 7.802 2.112 11.245-.007l34.216-21.064c3.358-2.067 3.364-6.911.01-8.987z"
                        }),
                        /*#__PURE__*/ (0, $14EEG$jsx)("path", {
                            fill: "url(#95b76e__e)",
                            d: "M7.882 21.255C8.647 4.542 22.816-8.387 39.529-7.621S69.17 7.313 68.405 24.026s-14.934 29.64-31.647 28.876C20.045 52.136 7.117 37.968 7.882 21.255"
                        }),
                        /*#__PURE__*/ (0, $14EEG$jsx)("path", {
                            fill: "url(#95b76e__f)",
                            d: "M-16.706 32.275c6.644-2.834 13.809-2.713 20.016-.234 12.421 4.973 19.649 1.898 24.662-10.494 2.497-6.22 7.372-11.486 14.023-14.323 13.297-5.664 28.649.482 34.305 13.74s-.535 28.585-13.83 34.257c-6.665 2.843-13.844 2.71-20.06.209-12.402-4.946-19.616-1.86-24.622 10.548-2.504 6.203-7.377 11.452-14.02 14.286-13.288 5.668-28.64-.477-34.292-13.728s.532-28.593 13.818-34.261"
                        }),
                        /*#__PURE__*/ (0, $14EEG$jsx)("circle", {
                            cx: 43.589,
                            cy: 8.088,
                            r: 19.476,
                            fill: "url(#95b76e__g)",
                            transform: "rotate(-.08 43.59 8.088)"
                        })
                    ]
                })
            })
        ]
    });
const $b13faf629a84668a$var$ForwardRef = /*#__PURE__*/ (0, $14EEG$forwardRef)($b13faf629a84668a$var$SvgComponent);
var $b13faf629a84668a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b13faf629a84668a$var$ForwardRef);


export {$b13faf629a84668a$export$2e2bcd8739ae039 as default};
