// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_calendar_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lNGeV$jsxs, jsx as $lNGeV$jsx} from "react/jsx-runtime";
import {forwardRef as $lNGeV$forwardRef} from "react";





const $35b9d39a08606d15$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lNGeV$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lNGeV$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lNGeV$jsxs)("linearGradient", {
                        id: "4a63bb__d",
                        x1: -7.107,
                        x2: 58.753,
                        y1: 38.282,
                        y2: -4.224,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lNGeV$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lNGeV$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lNGeV$jsxs)("linearGradient", {
                        id: "4a63bb__e",
                        x1: 47.535,
                        x2: 20.163,
                        y1: 10.272,
                        y2: 41.493,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lNGeV$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lNGeV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lNGeV$jsxs)("linearGradient", {
                        id: "4a63bb__f",
                        x1: 12.038,
                        x2: 62.315,
                        y1: 43.968,
                        y2: 14.091,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lNGeV$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lNGeV$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lNGeV$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lNGeV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lNGeV$jsxs)("radialGradient", {
                        id: "4a63bb__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.176 -18.2204 0 18.22 17.176)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lNGeV$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lNGeV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lNGeV$jsxs)("radialGradient", {
                        id: "4a63bb__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.176 -18.2204 0 18.22 17.176)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lNGeV$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lNGeV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lNGeV$jsx)("clipPath", {
                        id: "4a63bb__b",
                        children: /*#__PURE__*/ (0, $lNGeV$jsx)("rect", {
                            width: 41.881,
                            height: 39.48,
                            x: 3.418,
                            y: 3,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lNGeV$jsx)("clipPath", {
                id: "4a63bb__a",
                children: /*#__PURE__*/ (0, $lNGeV$jsx)("path", {
                    fill: "#fff",
                    d: "M34.798 10.2h-2.1v3c0 .663-.537 1.2-1.2 1.2s-1.2-.537-1.2-1.2v-3h-12.6v3c0 .663-.537 1.2-1.2 1.2s-1.2-.537-1.2-1.2v-3h-2.1c-2.652 0-4.8 2.148-4.8 4.8v3h31.2v-3c0-2.652-2.148-4.8-4.8-4.8m-26.4 24.6c0 2.652 2.148 4.8 4.8 4.8h21.6c2.652 0 4.8-2.148 4.8-4.8V20.4h-31.2zm21.9-9.6c0-.663.537-1.2 1.2-1.2h1.8c.663 0 1.2.537 1.2 1.2V27c0 .663-.537 1.2-1.2 1.2h-1.8c-.663 0-1.2-.537-1.2-1.2zm0 6.6c0-.663.537-1.2 1.2-1.2h1.8c.663 0 1.2.537 1.2 1.2v1.8c0 .663-.537 1.2-1.2 1.2h-1.8c-.663 0-1.2-.537-1.2-1.2zm-8.4-6.6c0-.663.537-1.2 1.2-1.2h1.8c.663 0 1.2.537 1.2 1.2V27c0 .663-.537 1.2-1.2 1.2h-1.8c-.663 0-1.2-.537-1.2-1.2zm0 6.6c0-.663.537-1.2 1.2-1.2h1.8c.663 0 1.2.537 1.2 1.2v1.8c0 .663-.537 1.2-1.2 1.2h-1.8c-.663 0-1.2-.537-1.2-1.2zm-8.4-6.6c0-.663.537-1.2 1.2-1.2h1.8c.663 0 1.2.537 1.2 1.2V27c0 .663-.537 1.2-1.2 1.2h-1.8c-.663 0-1.2-.537-1.2-1.2zm0 6.6c0-.663.537-1.2 1.2-1.2h1.8c.663 0 1.2.537 1.2 1.2v1.8c0 .663-.537 1.2-1.2 1.2h-1.8c-.663 0-1.2-.537-1.2-1.2zm4.2-24.6c0-.663-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2v3h2.4zm15 0c0-.663-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2v3h2.4z"
                })
            }),
            /*#__PURE__*/ (0, $lNGeV$jsx)("g", {
                clipPath: "url(#4a63bb__a)",
                children: /*#__PURE__*/ (0, $lNGeV$jsxs)("g", {
                    clipPath: "url(#4a63bb__b)",
                    children: [
                        /*#__PURE__*/ (0, $lNGeV$jsx)("rect", {
                            width: 41.881,
                            height: 39.48,
                            x: 3.418,
                            y: 3,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $lNGeV$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.418 3h41.881v39.48H3.418z"
                        }),
                        /*#__PURE__*/ (0, $lNGeV$jsx)("ellipse", {
                            cx: 18.22,
                            cy: 17.176,
                            fill: "url(#4a63bb__c)",
                            rx: 18.22,
                            ry: 17.176,
                            transform: "matrix(1 -.00132 .00148 1 8.852 -12.621)"
                        }),
                        /*#__PURE__*/ (0, $lNGeV$jsx)("path", {
                            fill: "url(#4a63bb__d)",
                            d: "M19.403-3.973c-3.216-1.877-7.291-1.888-10.52-.03l-32.29 18.585c-3.162 1.82-3.167 6.116-.009 7.943L8.66 41.084c3.224 1.864 7.299 1.862 10.52-.007L51.19 22.5c3.141-1.823 3.147-6.095.01-7.925z"
                        }),
                        /*#__PURE__*/ (0, $lNGeV$jsx)("path", {
                            fill: "url(#4a63bb__e)",
                            d: "M9.453 21.744c.716-14.74 13.97-26.14 29.606-25.466 15.636.675 27.73 13.17 27.015 27.91-.716 14.739-13.972 26.14-29.607 25.465S8.737 36.483 9.453 21.744"
                        }),
                        /*#__PURE__*/ (0, $lNGeV$jsx)("path", {
                            fill: "url(#4a63bb__f)",
                            d: "M-13.549 31.463c6.215-2.5 12.918-2.392 18.725-.206 11.62 4.385 18.382 1.674 23.072-9.255 2.336-5.484 6.897-10.129 13.119-12.631 12.44-4.995 26.801.425 32.093 12.117 5.291 11.692-.5 25.21-12.938 30.21-6.236 2.509-12.951 2.391-18.768.185-11.601-4.362-18.35-1.64-23.033 9.302-2.342 5.471-6.902 10.1-13.117 12.6-12.43 4.998-26.792-.422-32.08-12.107-5.289-11.686.497-25.216 12.927-30.215"
                        }),
                        /*#__PURE__*/ (0, $lNGeV$jsx)("ellipse", {
                            cx: 18.22,
                            cy: 17.176,
                            fill: "url(#4a63bb__g)",
                            rx: 18.22,
                            ry: 17.176,
                            transform: "matrix(1 -.00132 .00148 1 24.613 -7.02)"
                        })
                    ]
                })
            })
        ]
    });
const $35b9d39a08606d15$var$ForwardRef = /*#__PURE__*/ (0, $lNGeV$forwardRef)($35b9d39a08606d15$var$SvgComponent);
var $35b9d39a08606d15$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($35b9d39a08606d15$var$ForwardRef);


export {$35b9d39a08606d15$export$2e2bcd8739ae039 as default};
