// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_cloudUpload_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $cGa7x$jsxs, jsx as $cGa7x$jsx} from "react/jsx-runtime";
import {forwardRef as $cGa7x$forwardRef} from "react";





const $8389d8a100638f59$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cGa7x$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cGa7x$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cGa7x$jsxs)("linearGradient", {
                        id: "937f15__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGa7x$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $cGa7x$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGa7x$jsxs)("linearGradient", {
                        id: "937f15__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGa7x$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cGa7x$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGa7x$jsxs)("linearGradient", {
                        id: "937f15__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGa7x$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cGa7x$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cGa7x$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cGa7x$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGa7x$jsxs)("radialGradient", {
                        id: "937f15__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGa7x$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $cGa7x$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGa7x$jsxs)("radialGradient", {
                        id: "937f15__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGa7x$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cGa7x$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGa7x$jsx)("clipPath", {
                        id: "937f15__b",
                        children: /*#__PURE__*/ (0, $cGa7x$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cGa7x$jsx)("clipPath", {
                id: "937f15__a",
                children: /*#__PURE__*/ (0, $cGa7x$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M70.524 32.79q-.005.098-.006.192c8.022.474 14.388 7.11 14.388 15.912S78.078 64.83 69.648 64.83H50.106V44.442l10.5 8.424c.444.354.972.528 1.5.528.702 0 1.398-.312 1.872-.9.828-1.032.66-2.544-.372-3.372l-14.202-11.4c-.882-.708-2.142-.702-3.018.012l-13.998 11.4c-1.032.84-1.188 2.352-.348 3.378.84 1.032 2.352 1.188 3.378.348l9.882-8.052V64.83h4.8V81.6c0 1.326-1.074 2.4-2.4 2.4s-2.4-1.074-2.4-2.4V64.83H21.882v-.012c-5.97-.102-10.788-5.184-10.788-11.448 0-5.274 3.42-9.708 8.07-11.04-.21-.966-.336-1.968-.336-3.006 0-7.356 5.706-13.314 12.744-13.314 1.722 0 3.36.36 4.86 1.008 2.262-7.344 8.85-12.66 16.632-12.66 9.648 0 17.466 8.166 17.466 18.24q0 .095-.006.192"
                })
            }),
            /*#__PURE__*/ (0, $cGa7x$jsx)("g", {
                clipPath: "url(#937f15__a)",
                children: /*#__PURE__*/ (0, $cGa7x$jsxs)("g", {
                    clipPath: "url(#937f15__b)",
                    children: [
                        /*#__PURE__*/ (0, $cGa7x$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $cGa7x$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $cGa7x$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#937f15__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $cGa7x$jsx)("path", {
                            fill: "url(#937f15__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $cGa7x$jsx)("path", {
                            fill: "url(#937f15__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $cGa7x$jsx)("path", {
                            fill: "url(#937f15__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $cGa7x$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#937f15__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $8389d8a100638f59$var$ForwardRef = /*#__PURE__*/ (0, $cGa7x$forwardRef)($8389d8a100638f59$var$SvgComponent);
var $8389d8a100638f59$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8389d8a100638f59$var$ForwardRef);


export {$8389d8a100638f59$export$2e2bcd8739ae039 as default};
