// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_microphone_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $4e7Ei$jsxs, jsx as $4e7Ei$jsx} from "react/jsx-runtime";
import {forwardRef as $4e7Ei$forwardRef} from "react";





const $a5e57ff4d5ed008e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4e7Ei$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4e7Ei$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4e7Ei$jsxs)("linearGradient", {
                        id: "1872ce__d",
                        x1: -24.966,
                        x2: 201.252,
                        y1: 134.343,
                        y2: -3.286,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4e7Ei$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4e7Ei$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4e7Ei$jsxs)("linearGradient", {
                        id: "1872ce__e",
                        x1: 156.573,
                        x2: 59.526,
                        y1: 35.63,
                        y2: 139.982,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4e7Ei$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4e7Ei$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4e7Ei$jsxs)("linearGradient", {
                        id: "1872ce__f",
                        x1: 38.635,
                        x2: 210.673,
                        y1: 154.384,
                        y2: 58.011,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4e7Ei$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4e7Ei$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4e7Ei$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4e7Ei$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4e7Ei$jsxs)("radialGradient", {
                        id: "1872ce__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.638 52.035)scale(60.5342)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4e7Ei$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4e7Ei$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4e7Ei$jsxs)("radialGradient", {
                        id: "1872ce__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 52.947 88.086)scale(60.5342)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4e7Ei$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4e7Ei$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4e7Ei$jsx)("clipPath", {
                        id: "1872ce__b",
                        children: /*#__PURE__*/ (0, $4e7Ei$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10,
                            y: 10,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4e7Ei$jsx)("clipPath", {
                id: "1872ce__a",
                children: /*#__PURE__*/ (0, $4e7Ei$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M79 24.8c-10.603 0-19.2 8.596-19.2 19.2v28.6c0 10.604 8.597 19.2 19.2 19.2h2.6c10.605 0 19.201-8.596 19.201-19.2V44c0-10.604-8.596-19.2-19.2-19.2zm-3.2 90.775c-22.462-2.256-40-21.217-40-44.275v-4.5c0-2.21 1.792-4 4-4 2.21 0 4 1.79 4 4v4.5c0 20.158 16.342 36.5 36.5 36.5s36.501-16.342 36.501-36.5v-4.5c0-2.21 1.791-4 4-4s4 1.79 4 4v4.5c0 23.399-18.06 42.579-41 44.364V127.8h20c2.209 0 4 1.791 4 4s-1.791 4-4 4h-47c-2.21 0-4-1.791-4-4s1.79-4 4-4h19z"
                })
            }),
            /*#__PURE__*/ (0, $4e7Ei$jsx)("g", {
                clipPath: "url(#1872ce__a)",
                children: /*#__PURE__*/ (0, $4e7Ei$jsxs)("g", {
                    clipPath: "url(#1872ce__b)",
                    children: [
                        /*#__PURE__*/ (0, $4e7Ei$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10,
                            y: 10,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $4e7Ei$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M10 10h139.142v139.142H10z"
                        }),
                        /*#__PURE__*/ (0, $4e7Ei$jsx)("circle", {
                            cx: 88.673,
                            cy: 15.397,
                            r: 60.534,
                            fill: "url(#1872ce__c)",
                            transform: "rotate(-.08 88.673 15.397)"
                        }),
                        /*#__PURE__*/ (0, $4e7Ei$jsx)("path", {
                            fill: "url(#1872ce__d)",
                            d: "M63.11-14.579c-10.687-6.612-24.226-6.653-34.951-.104l-107.276 65.5c-10.507 6.414-10.524 21.556-.032 27.995l106.567 65.406c10.71 6.572 24.249 6.565 34.949-.023l106.348-65.468c10.436-6.425 10.455-21.482.031-27.932z"
                        }),
                        /*#__PURE__*/ (0, $4e7Ei$jsx)("path", {
                            fill: "url(#1872ce__e)",
                            d: "M30.054 76.061c2.378-51.945 46.416-92.128 98.362-89.75s92.128 46.417 89.749 98.363c-2.378 51.945-46.416 92.128-98.362 89.749-51.945-2.378-92.128-46.416-89.75-98.362"
                        }),
                        /*#__PURE__*/ (0, $4e7Ei$jsx)("path", {
                            fill: "url(#1872ce__f)",
                            d: "M-46.373 110.312c20.649-8.809 42.918-8.431 62.212-.726 38.606 15.456 61.07 5.9 76.65-32.618 7.762-19.329 22.914-35.698 43.585-44.517 41.33-17.604 89.044 1.498 106.624 42.706 17.579 41.207-1.662 88.846-42.983 106.473-20.718 8.839-43.03 8.425-62.353.651-38.543-15.374-60.965-5.782-76.526 32.783-7.78 19.282-22.929 35.595-43.577 44.404-41.297 17.618-89.011-1.484-106.58-42.669-17.571-41.185 1.651-88.87 42.948-106.487"
                        }),
                        /*#__PURE__*/ (0, $4e7Ei$jsx)("circle", {
                            cx: 141.033,
                            cy: 35.138,
                            r: 60.534,
                            fill: "url(#1872ce__g)",
                            transform: "rotate(-.08 141.033 35.138)"
                        })
                    ]
                })
            })
        ]
    });
const $a5e57ff4d5ed008e$var$ForwardRef = /*#__PURE__*/ (0, $4e7Ei$forwardRef)($a5e57ff4d5ed008e$var$SvgComponent);
var $a5e57ff4d5ed008e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a5e57ff4d5ed008e$var$ForwardRef);


export {$a5e57ff4d5ed008e$export$2e2bcd8739ae039 as default};
