// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_shoppingCart_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $5LaGP$reactjsxruntime = require("react/jsx-runtime");
var $5LaGP$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $151acd79a3e2e665$export$2e2bcd8739ae039);




const $151acd79a3e2e665$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6688a0__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6688a0__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6688a0__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6688a0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6688a0__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("clipPath", {
                        id: "6688a0__b",
                        children: /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("clipPath", {
                id: "6688a0__a",
                children: /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M6.746 7.826c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25l.36.002.62.004c.525.003 1.229.006 2.057.005.873 0 1.612.628 1.752 1.476l2.945 17.86c.339 2.054 2.113 3.56 4.194 3.56h16.169c.69 0 1.25-.56 1.25-1.25 0-.691-.56-1.25-1.25-1.25h-16.17c-.856 0-1.587-.62-1.726-1.466l-.58-3.516H33.59c1.553 0 2.888-1.098 3.189-2.62l1.235-6.262c.395-2.002-1.132-3.873-3.178-3.878-7.024-.017-17.547-.034-20.572-.005l-.261-1.58c-.341-2.068-2.136-3.57-4.22-3.569-.822 0-1.52-.002-2.042-.005l-.6-.004zM20.505 39c0 1.38-1.12 2.5-2.5 2.5s-2.5-1.12-2.5-2.5 1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5m12.462 2.5c1.38 0 2.5-1.12 2.5-2.5s-1.12-2.5-2.5-2.5-2.5 1.12-2.5 2.5 1.12 2.5 2.5 2.5"
                })
            }),
            /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6688a0__a)",
                children: /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6688a0__b)",
                    children: [
                        /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#6688a0__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("path", {
                            fill: "url(#6688a0__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("path", {
                            fill: "url(#6688a0__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("path", {
                            fill: "url(#6688a0__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $5LaGP$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#6688a0__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $151acd79a3e2e665$var$ForwardRef = /*#__PURE__*/ (0, $5LaGP$react.forwardRef)($151acd79a3e2e665$var$SvgComponent);
var $151acd79a3e2e665$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($151acd79a3e2e665$var$ForwardRef);
