// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_vectorDraw_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $20wqg$jsxs, jsx as $20wqg$jsx} from "react/jsx-runtime";
import {forwardRef as $20wqg$forwardRef} from "react";





const $ef3602e8e9ebfe54$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $20wqg$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $20wqg$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $20wqg$jsxs)("linearGradient", {
                        id: "c6545f__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $20wqg$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $20wqg$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $20wqg$jsxs)("linearGradient", {
                        id: "c6545f__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $20wqg$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $20wqg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $20wqg$jsxs)("linearGradient", {
                        id: "c6545f__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $20wqg$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $20wqg$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $20wqg$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $20wqg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $20wqg$jsxs)("radialGradient", {
                        id: "c6545f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $20wqg$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $20wqg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $20wqg$jsxs)("radialGradient", {
                        id: "c6545f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $20wqg$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $20wqg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $20wqg$jsx)("clipPath", {
                        id: "c6545f__b",
                        children: /*#__PURE__*/ (0, $20wqg$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $20wqg$jsx)("clipPath", {
                id: "c6545f__a",
                children: /*#__PURE__*/ (0, $20wqg$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M54.073 20.4 75.604 41.93l3.434-3.433c2.4-2.4 2.63-6.058.516-8.172L65.68 16.45c-2.114-2.114-5.773-1.883-8.173.516zm19.255 24.912L50.693 22.676 29.648 31.66c-1.821.778-3.255 2.36-3.834 4.231l-10.4 33.6c-.845 2.729-.327 5.39 1.035 7.465l21.873-21.873c-.719-1.8-.35-3.934 1.108-5.392 1.953-1.953 5.119-1.953 7.071 0 1.953 1.953 1.953 5.118 0 7.07-1.442 1.443-3.545 1.82-5.333 1.132l-21.84 21.84c2.032 1.229 4.578 1.666 7.186.858l33.6-10.4c1.872-.579 3.453-2.013 4.231-3.834z"
                })
            }),
            /*#__PURE__*/ (0, $20wqg$jsx)("g", {
                clipPath: "url(#c6545f__a)",
                children: /*#__PURE__*/ (0, $20wqg$jsxs)("g", {
                    clipPath: "url(#c6545f__b)",
                    children: [
                        /*#__PURE__*/ (0, $20wqg$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $20wqg$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $20wqg$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#c6545f__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $20wqg$jsx)("path", {
                            fill: "url(#c6545f__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $20wqg$jsx)("path", {
                            fill: "url(#c6545f__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $20wqg$jsx)("path", {
                            fill: "url(#c6545f__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $20wqg$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#c6545f__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $ef3602e8e9ebfe54$var$ForwardRef = /*#__PURE__*/ (0, $20wqg$forwardRef)($ef3602e8e9ebfe54$var$SvgComponent);
var $ef3602e8e9ebfe54$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ef3602e8e9ebfe54$var$ForwardRef);


export {$ef3602e8e9ebfe54$export$2e2bcd8739ae039 as default};
