// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_targeted_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $jAHSs$jsxs, jsx as $jAHSs$jsx} from "react/jsx-runtime";
import {forwardRef as $jAHSs$forwardRef} from "react";





const $0c828ac65116aad2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jAHSs$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jAHSs$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jAHSs$jsxs)("linearGradient", {
                        id: "018305__e",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jAHSs$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $jAHSs$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jAHSs$jsxs)("linearGradient", {
                        id: "018305__f",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jAHSs$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jAHSs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jAHSs$jsxs)("linearGradient", {
                        id: "018305__g",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jAHSs$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jAHSs$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jAHSs$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jAHSs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jAHSs$jsxs)("radialGradient", {
                        id: "018305__d",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jAHSs$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $jAHSs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jAHSs$jsxs)("radialGradient", {
                        id: "018305__h",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jAHSs$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jAHSs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jAHSs$jsx)("clipPath", {
                        id: "018305__a",
                        children: /*#__PURE__*/ (0, $jAHSs$jsx)("path", {
                            fill: "#fff",
                            d: "M0 0h48v48H0z"
                        })
                    }),
                    /*#__PURE__*/ (0, $jAHSs$jsx)("clipPath", {
                        id: "018305__c",
                        children: /*#__PURE__*/ (0, $jAHSs$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jAHSs$jsxs)("g", {
                clipPath: "url(#018305__a)",
                children: [
                    /*#__PURE__*/ (0, $jAHSs$jsx)("clipPath", {
                        id: "018305__b",
                        children: /*#__PURE__*/ (0, $jAHSs$jsx)("path", {
                            fill: "#fff",
                            fillRule: "evenodd",
                            d: "M4.808 9.153c-.348.089-.621.358-.716.704-.095.347.003.717.257.971l4.6 4.6c.207.207.495.313.787.289l3.775-.307 6.374 6.373c.488.488 1.28.488 1.767 0 .489-.488.489-1.28 0-1.768l-6.373-6.373.307-3.775c.024-.293-.082-.58-.29-.788l-4.598-4.6c-.254-.253-.625-.351-.971-.256-.347.094-.616.368-.705.716l-.86 3.354zM24 8.869c-1.744 0-3.416.295-4.972.836-.652.227-1.364-.118-1.591-.77s.118-1.365.77-1.591c1.816-.632 3.766-.975 5.793-.975C33.736 6.37 41.63 14.263 41.63 24s-7.894 17.632-17.631 17.632S6.368 33.738 6.368 24c0-1.956.319-3.84.909-5.602.22-.654.928-1.007 1.582-.788.655.22 1.008.928.789 1.582-.506 1.51-.78 3.125-.78 4.808 0 8.357 6.774 15.132 15.131 15.132S39.13 32.357 39.13 24 32.356 8.87 23.999 8.87m0 7.753c-.802 0-1.57.128-2.29.362-.656.214-1.362-.145-1.576-.8-.214-.657.145-1.363.801-1.577.966-.315 1.997-.485 3.064-.485 5.456 0 9.878 4.423 9.878 9.878s-4.422 9.877-9.878 9.877c-5.455 0-9.877-4.422-9.877-9.877 0-1.006.15-1.979.431-2.896.203-.66.902-1.031 1.562-.83.66.203 1.031.902.829 1.562-.21.683-.322 1.409-.322 2.164 0 4.074 3.303 7.378 7.377 7.378 4.075 0 7.378-3.304 7.378-7.378 0-4.075-3.303-7.378-7.378-7.378m-2.673 9.031c2.26-.21 4.059-2.01 4.27-4.269.903.544 1.507 1.535 1.507 2.666 0 1.718-1.392 3.111-3.11 3.111-1.132 0-2.122-.604-2.667-1.508"
                        })
                    }),
                    /*#__PURE__*/ (0, $jAHSs$jsx)("g", {
                        clipPath: "url(#018305__b)",
                        children: /*#__PURE__*/ (0, $jAHSs$jsxs)("g", {
                            clipPath: "url(#018305__c)",
                            children: [
                                /*#__PURE__*/ (0, $jAHSs$jsx)("rect", {
                                    width: 48,
                                    height: 48,
                                    fill: "#FFECCF",
                                    rx: 5.125
                                }),
                                /*#__PURE__*/ (0, $jAHSs$jsx)("path", {
                                    fill: "#FFECCF",
                                    d: "M0 0h48v48H0z"
                                }),
                                /*#__PURE__*/ (0, $jAHSs$jsx)("circle", {
                                    cx: 27.142,
                                    cy: 1.862,
                                    r: 20.883,
                                    fill: "url(#018305__d)",
                                    transform: "rotate(-.08 27.142 1.862)"
                                }),
                                /*#__PURE__*/ (0, $jAHSs$jsx)("path", {
                                    fill: "url(#018305__e)",
                                    d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                                }),
                                /*#__PURE__*/ (0, $jAHSs$jsx)("path", {
                                    fill: "url(#018305__f)",
                                    d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                                }),
                                /*#__PURE__*/ (0, $jAHSs$jsx)("path", {
                                    fill: "url(#018305__g)",
                                    d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                                }),
                                /*#__PURE__*/ (0, $jAHSs$jsx)("circle", {
                                    cx: 45.201,
                                    cy: 8.672,
                                    r: 20.883,
                                    fill: "url(#018305__h)",
                                    transform: "rotate(-.08 45.2 8.672)"
                                })
                            ]
                        })
                    })
                ]
            })
        ]
    });
const $0c828ac65116aad2$var$ForwardRef = /*#__PURE__*/ (0, $jAHSs$forwardRef)($0c828ac65116aad2$var$SvgComponent);
var $0c828ac65116aad2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0c828ac65116aad2$var$ForwardRef);


export {$0c828ac65116aad2$export$2e2bcd8739ae039 as default};
