// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_conversationbubbles_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $9qwy4$reactjsxruntime = require("react/jsx-runtime");
var $9qwy4$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0b75f62138993cea$export$2e2bcd8739ae039);




const $0b75f62138993cea$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2c41be__d",
                        x1: -7.36,
                        x2: 60.505,
                        y1: 40.432,
                        y2: -0.857,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2c41be__e",
                        x1: 47.099,
                        x2: 17.985,
                        y1: 10.818,
                        y2: 42.124,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2c41be__f",
                        x1: 11.721,
                        x2: 63.332,
                        y1: 46.444,
                        y2: 17.532,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2c41be__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 10.992 15.74)scale(18.1603)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2c41be__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 15.885 26.555)scale(18.1603)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("clipPath", {
                        id: "2c41be__b",
                        children: /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsxs)("clipPath", {
                id: "2c41be__a",
                children: [
                    /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("path", {
                        fill: "#fff",
                        d: "M39.484 20.808h-4.631v3.436c0 2.968-2.51 5.383-5.596 5.383h-7.562v2.235c0 1.342 1.167 2.43 2.605 2.43h5.672l5.756 4.655c.565.456 1.443.08 1.437-.618-.01-1.25-.02-2.89-.014-4.037h2.333c1.439 0 2.605-1.088 2.605-2.43v-8.624c0-1.342-1.166-2.43-2.605-2.43"
                    }),
                    /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("path", {
                        fill: "#fff",
                        d: "M29.258 27.214c1.759 0 3.184-1.33 3.184-2.97v-10.54c0-1.64-1.425-2.97-3.184-2.97H9.094c-1.758 0-3.184 1.33-3.184 2.97v10.54c0 1.64 1.426 2.97 3.184 2.97h2.851c.007 1.402-.005 3.408-.017 4.935-.007.852 1.067 1.313 1.757.755l7.035-5.69z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("g", {
                clipPath: "url(#2c41be__a)",
                children: /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#2c41be__b)",
                    children: [
                        /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.129 3.129H44.87V44.87H3.129z"
                        }),
                        /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("circle", {
                            cx: 26.733,
                            cy: 4.748,
                            r: 18.16,
                            fill: "url(#2c41be__c)",
                            transform: "rotate(-.08 26.733 4.748)"
                        }),
                        /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("path", {
                            fill: "url(#2c41be__d)",
                            d: "M19.062-4.245c-3.205-1.984-7.267-1.996-10.485-.031l-32.183 19.65c-3.151 1.924-3.157 6.466-.01 8.398L8.356 43.394c3.213 1.972 7.275 1.97 10.485-.007l31.904-19.64c3.131-1.928 3.137-6.445.01-8.38z"
                        }),
                        /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("path", {
                            fill: "url(#2c41be__e)",
                            d: "M9.143 22.948C9.857 7.364 23.068-4.691 38.652-3.977S66.29 9.948 65.577 25.53c-.713 15.584-13.925 27.639-29.509 26.925S8.43 38.531 9.143 22.948"
                        }),
                        /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("path", {
                            fill: "url(#2c41be__f)",
                            d: "M-13.781 33.222c6.194-2.642 12.875-2.529 18.663-.218 11.582 4.637 18.321 1.77 22.995-9.785 2.329-5.798 6.874-10.71 13.076-13.355 12.399-5.281 26.713.45 31.987 12.812S72.44 49.33 60.045 54.618c-6.215 2.651-12.909 2.527-18.706.195-11.563-4.612-18.29-1.734-22.958 9.835-2.334 5.785-6.878 10.679-13.073 13.321-12.389 5.285-26.703-.445-31.974-12.8s.496-26.661 12.885-31.947"
                        }),
                        /*#__PURE__*/ (0, $9qwy4$reactjsxruntime.jsx)("circle", {
                            cx: 42.44,
                            cy: 10.67,
                            r: 18.16,
                            fill: "url(#2c41be__g)",
                            transform: "rotate(-.08 42.44 10.67)"
                        })
                    ]
                })
            })
        ]
    });
const $0b75f62138993cea$var$ForwardRef = /*#__PURE__*/ (0, $9qwy4$react.forwardRef)($0b75f62138993cea$var$SvgComponent);
var $0b75f62138993cea$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0b75f62138993cea$var$ForwardRef);
