// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_laptop_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $ckAXv$reactjsxruntime = require("react/jsx-runtime");
var $ckAXv$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $62ef9a995e683297$export$2e2bcd8739ae039);




const $62ef9a995e683297$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsxs)("linearGradient", {
                        id: "071fd8__d",
                        x1: -6.355,
                        x2: 57.806,
                        y1: 39.86,
                        y2: -1.549,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsxs)("linearGradient", {
                        id: "071fd8__e",
                        x1: 46.878,
                        x2: 20.213,
                        y1: 12.574,
                        y2: 42.989,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsxs)("linearGradient", {
                        id: "071fd8__f",
                        x1: 12.294,
                        x2: 61.274,
                        y1: 45.4,
                        y2: 16.294,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsxs)("radialGradient", {
                        id: "071fd8__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.7327 -17.7502 0 17.75 16.733)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsxs)("radialGradient", {
                        id: "071fd8__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.7327 -17.7502 0 17.75 16.733)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("clipPath", {
                        id: "071fd8__b",
                        children: /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("rect", {
                            width: 40.8,
                            height: 38.461,
                            x: 3.898,
                            y: 5.489,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("clipPath", {
                id: "071fd8__a",
                children: /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M40.8 33h-2.4V14.466c0-2.688-2.148-4.866-4.8-4.866H14.4c-2.652 0-4.8 2.178-4.8 4.866V33H7.2c-1.491 0-2.7 1.209-2.7 2.7 0 1.49 1.209 2.7 2.7 2.7h33.6c1.491 0 2.7-1.21 2.7-2.7S42.291 33 40.8 33M24 14.7c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.2-1.2 1.2.537 1.2 1.2-.537 1.2-1.2 1.2"
                })
            }),
            /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("g", {
                clipPath: "url(#071fd8__a)",
                children: /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#071fd8__b)",
                    children: [
                        /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("rect", {
                            width: 40.8,
                            height: 38.461,
                            x: 3.898,
                            y: 5.489,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.898 5.49h40.8v38.46h-40.8z"
                        }),
                        /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.75,
                            cy: 16.733,
                            fill: "url(#071fd8__c)",
                            rx: 17.75,
                            ry: 16.733,
                            transform: "matrix(1 -.00132 .00148 1 9.191 -9.729)"
                        }),
                        /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("path", {
                            fill: "url(#071fd8__d)",
                            d: "M19.471-1.305c-3.133-1.828-7.103-1.839-10.248-.029l-31.456 18.105c-3.081 1.773-3.086 5.959-.01 7.739L9.006 42.59c3.14 1.816 7.11 1.814 10.248-.007l31.184-18.096c3.06-1.777 3.066-5.939.009-7.721z"
                        }),
                        /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("path", {
                            fill: "url(#071fd8__e)",
                            d: "M9.78 23.75C10.475 9.39 23.39-1.716 38.62-1.06c15.232.658 27.015 12.83 26.317 27.19-.697 14.358-13.61 25.465-28.842 24.808-15.232-.658-27.014-12.83-26.317-27.19"
                        }),
                        /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("path", {
                            fill: "url(#071fd8__f)",
                            d: "M-12.633 33.218c6.055-2.435 12.585-2.33 18.242-.201 11.32 4.272 17.908 1.63 22.477-9.016 2.275-5.343 6.718-9.868 12.78-12.305C52.985 6.83 66.976 12.11 72.13 23.5c5.155 11.39-.487 24.559-12.603 29.431-6.075 2.444-12.618 2.33-18.284.18-11.302-4.25-17.876-1.598-22.44 9.062-2.28 5.33-6.723 9.84-12.777 12.274-12.11 4.87-26.1-.41-31.252-11.794s.484-24.565 12.593-29.435"
                        }),
                        /*#__PURE__*/ (0, $ckAXv$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.75,
                            cy: 16.733,
                            fill: "url(#071fd8__g)",
                            rx: 17.75,
                            ry: 16.733,
                            transform: "matrix(1 -.00132 .00148 1 24.547 -4.271)"
                        })
                    ]
                })
            })
        ]
    });
const $62ef9a995e683297$var$ForwardRef = /*#__PURE__*/ (0, $ckAXv$react.forwardRef)($62ef9a995e683297$var$SvgComponent);
var $62ef9a995e683297$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($62ef9a995e683297$var$ForwardRef);
