// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_globeGrid_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $fynGA$jsxs, jsx as $fynGA$jsx} from "react/jsx-runtime";
import {forwardRef as $fynGA$forwardRef} from "react";





const $1d09e5f7fe75a8f9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fynGA$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fynGA$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fynGA$jsxs)("linearGradient", {
                        id: "f5dd6c__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fynGA$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $fynGA$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fynGA$jsxs)("linearGradient", {
                        id: "f5dd6c__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fynGA$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fynGA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fynGA$jsxs)("linearGradient", {
                        id: "f5dd6c__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fynGA$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fynGA$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fynGA$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fynGA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fynGA$jsxs)("radialGradient", {
                        id: "f5dd6c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fynGA$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $fynGA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fynGA$jsxs)("radialGradient", {
                        id: "f5dd6c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fynGA$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fynGA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fynGA$jsx)("clipPath", {
                        id: "f5dd6c__b",
                        children: /*#__PURE__*/ (0, $fynGA$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fynGA$jsx)("clipPath", {
                id: "f5dd6c__a",
                children: /*#__PURE__*/ (0, $fynGA$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M83.947 25.206v23.08h14.48c-3.451-8.022-8.27-15.778-14.48-23.08m23.127 23.08c-2.976-7.693-7.08-15.182-12.331-22.325 12.936 3.52 24.009 11.563 31.417 22.324zm-5.681 8H83.947v20.312h20.999c-.288-6.837-1.468-13.642-3.553-20.313m11.56 20.312c-.258-6.824-1.323-13.624-3.208-20.313h20.13q.427-.001.83-.086c2.94 6.254 4.758 13.14 5.193 20.399zm-8.077 8H83.947v20.312h17.089c2.151-6.693 3.435-13.498 3.84-20.312m4.52 20.312c1.953-6.699 3.121-13.5 3.491-20.312h22.927c-.59 7.252-2.561 14.115-5.651 20.322q-.142-.01-.288-.01zm-11.368 8h-14.08v22.953l.082-.006c5.9-7.309 10.573-15.018 13.998-22.947m-2.506 20.911c4.624-6.732 8.346-13.74 11.148-20.911h18.644c-7.238 9.948-17.671 17.421-29.792 20.911M33.84 48.285c7.382-10.724 18.405-18.748 31.285-22.288-5.24 7.132-9.334 14.61-12.305 22.288zm-3.215 8H50.15c-1.886 6.689-2.95 13.49-3.209 20.313h-22.84c.438-7.306 2.277-14.234 5.25-20.52.4.134.829.207 1.274.207m27.876 0c-2.085 6.671-3.265 13.477-3.552 20.313h20.998V56.284zM24.186 84.598h22.821c.37 6.813 1.538 13.613 3.491 20.312H30.625q-.39 0-.758.072c-3.107-6.224-5.09-13.108-5.681-20.385m30.833 0c.404 6.814 1.688 13.619 3.84 20.312h17.088V84.598zM34.686 112.91h18.538c2.797 7.157 6.51 14.153 11.122 20.873-12.065-3.506-22.449-10.961-29.66-20.873m27.18 0c3.424 7.927 8.095 15.632 13.991 22.939l.09.006V112.91zm14.081-87.704c-6.21 7.302-11.029 15.058-14.48 23.079h14.48z"
                })
            }),
            /*#__PURE__*/ (0, $fynGA$jsx)("g", {
                clipPath: "url(#f5dd6c__a)",
                children: /*#__PURE__*/ (0, $fynGA$jsxs)("g", {
                    clipPath: "url(#f5dd6c__b)",
                    children: [
                        /*#__PURE__*/ (0, $fynGA$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $fynGA$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $fynGA$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#f5dd6c__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $fynGA$jsx)("path", {
                            fill: "url(#f5dd6c__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $fynGA$jsx)("path", {
                            fill: "url(#f5dd6c__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $fynGA$jsx)("path", {
                            fill: "url(#f5dd6c__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $fynGA$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#f5dd6c__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $1d09e5f7fe75a8f9$var$ForwardRef = /*#__PURE__*/ (0, $fynGA$forwardRef)($1d09e5f7fe75a8f9$var$SvgComponent);
var $1d09e5f7fe75a8f9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1d09e5f7fe75a8f9$var$ForwardRef);


export {$1d09e5f7fe75a8f9$export$2e2bcd8739ae039 as default};
