// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_thumbsUp_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7vElD$jsxs, jsx as $7vElD$jsx} from "react/jsx-runtime";
import {forwardRef as $7vElD$forwardRef} from "react";





const $82cd178b4e833f93$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7vElD$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 100,
        height: 96,
        viewBox: "0 0 100 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7vElD$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7vElD$jsxs)("linearGradient", {
                        id: "9b1a52__d",
                        x1: -22.152,
                        x2: 133.926,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7vElD$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $7vElD$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7vElD$jsxs)("linearGradient", {
                        id: "9b1a52__e",
                        x1: 103.101,
                        x2: 36.144,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7vElD$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7vElD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7vElD$jsxs)("linearGradient", {
                        id: "9b1a52__f",
                        x1: 21.728,
                        x2: 140.424,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7vElD$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7vElD$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7vElD$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7vElD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7vElD$jsxs)("radialGradient", {
                        id: "9b1a52__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 26.265 29.988)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7vElD$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $7vElD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7vElD$jsxs)("radialGradient", {
                        id: "9b1a52__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 37.517 54.861)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7vElD$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7vElD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7vElD$jsx)("clipPath", {
                        id: "9b1a52__b",
                        children: /*#__PURE__*/ (0, $7vElD$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 1.973,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7vElD$jsx)("clipPath", {
                id: "9b1a52__a",
                children: /*#__PURE__*/ (0, $7vElD$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M40.284 76.965c3.22 1.2 6.623 1.816 10.06 1.816l-.011-.012h14.315c.03 0 .089-.005.089-.005h3.966c1.47 0 2.992-.406 4.167-1.305 1.193-.917 1.986-2.321 1.986-4.214 0-.61-.088-1.692-.61-2.75-.248-.511-.6-1.01-1.082-1.446 2.45-.14 4.519-1.915 5.019-4.343.452-2.203-.494-4.319-2.163-5.518 2.274-.564 3.996-2.562 4.12-5 .1-1.975-.864-3.738-2.38-4.773 2.15-.646 3.749-2.585 3.872-4.936.159-3.109-2.321-5.718-5.436-5.718h-16.29c-.887 0-1.522-.864-1.258-1.71l1.176-3.773c1.052-3.373 1.721-6.852 2.01-10.372l.164-2.04c.247-3.038-2.157-5.647-5.213-5.647-2.015 0-3.849 1.163-4.719 2.98l-5.47 11.459q-.77 1.606-1.758 3.08l-.135.205c-3.166 4.71-8.163 7.805-13.696 8.597v32.325c1.404.286 2.787.681 4.134 1.185zm-13.277-3.614v-31.61h-2.238c-3.573 0-6.464 2.89-6.464 6.464v18.64c0 3.574 2.891 6.465 6.464 6.465h.723q.758 0 1.515.041"
                })
            }),
            /*#__PURE__*/ (0, $7vElD$jsx)("g", {
                clipPath: "url(#9b1a52__a)",
                children: /*#__PURE__*/ (0, $7vElD$jsxs)("g", {
                    clipPath: "url(#9b1a52__b)",
                    children: [
                        /*#__PURE__*/ (0, $7vElD$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 1.973,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $7vElD$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M1.973 0h96v96h-96z"
                        }),
                        /*#__PURE__*/ (0, $7vElD$jsx)("circle", {
                            cx: 56.253,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#9b1a52__c)",
                            transform: "rotate(-.08 56.253 3.723)"
                        }),
                        /*#__PURE__*/ (0, $7vElD$jsx)("path", {
                            fill: "url(#9b1a52__d)",
                            d: "M38.615-16.958c-7.372-4.562-16.714-4.59-24.114-.072l-74.014 45.19c-7.249 4.427-7.261 14.873-.022 19.316L13.99 92.603c7.39 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.201-4.433 7.214-14.821.022-19.27z"
                        }),
                        /*#__PURE__*/ (0, $7vElD$jsx)("path", {
                            fill: "url(#9b1a52__e)",
                            d: "M15.81 45.579c1.64-35.84 32.024-63.563 67.864-61.923 35.839 1.641 63.563 32.025 61.922 67.865s-32.025 63.563-67.864 61.922-63.564-32.025-61.923-67.864"
                        }),
                        /*#__PURE__*/ (0, $7vElD$jsx)("path", {
                            fill: "url(#9b1a52__f)",
                            d: "M-36.922 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.354-13.336 15.808-24.63 30.07-30.714 28.516-12.146 61.436 1.033 73.565 29.464s-1.147 61.299-29.656 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.062-3.989-52.798 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $7vElD$jsx)("circle", {
                            cx: 92.378,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#9b1a52__g)",
                            transform: "rotate(-.08 92.378 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $82cd178b4e833f93$var$ForwardRef = /*#__PURE__*/ (0, $7vElD$forwardRef)($82cd178b4e833f93$var$SvgComponent);
var $82cd178b4e833f93$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($82cd178b4e833f93$var$ForwardRef);


export {$82cd178b4e833f93$export$2e2bcd8739ae039 as default};
