// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_logo_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $ib1ED$reactjsxruntime = require("react/jsx-runtime");
var $ib1ED$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e160c8b0e262c4f0$export$2e2bcd8739ae039);




const $e160c8b0e262c4f0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a7c756__d",
                        x1: -5.676,
                        x2: 56.126,
                        y1: 38.508,
                        y2: -1.379,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a7c756__e",
                        x1: 45.597,
                        x2: 19.913,
                        y1: 12.224,
                        y2: 41.522,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a7c756__f",
                        x1: 12.287,
                        x2: 59.466,
                        y1: 43.844,
                        y2: 15.807,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a7c756__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a7c756__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("clipPath", {
                        id: "a7c756__b",
                        children: /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 5.4,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("clipPath", {
                id: "a7c756__a",
                children: /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m25.609 8.022 12.77 6.372c1.222.609 1.993 1.857 1.993 3.222v12.738c0 1.362-.768 2.61-1.986 3.219l-12.771 6.399c-1.014.51-2.211.51-3.225 0l-12.771-6.4c-1.218-.611-1.986-1.856-1.986-3.218V17.616c0-1.365.77-2.613 1.992-3.222l12.77-6.372c1.012-.504 2.203-.504 3.214 0M15.397 29.28c.417 0 .819-.216 1.04-.6h-.002l4.395-7.611c.333-.573.135-1.308-.438-1.638-.573-.333-1.308-.135-1.638.438l-4.395 7.61c-.333.574-.135 1.309.438 1.639.189.11.396.162.6.162m16.002 0c.867 0 1.644-.45 2.079-1.2.435-.753.435-1.65 0-2.4l-4.06-7.03c-.434-.75-1.211-1.2-2.078-1.2s-1.644.45-2.08 1.2l-4.058 7.03c-.435.75-.435 1.65 0 2.4s1.212 1.2 2.079 1.2zm-4.056-9.43 4.059 7.03h-8.118z"
                })
            }),
            /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("g", {
                clipPath: "url(#a7c756__a)",
                children: /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#a7c756__b)",
                    children: [
                        /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 5.4,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M4.2 5.4h39.3v37.047H4.2z"
                        }),
                        /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#a7c756__c)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 9.3 -9.258)"
                        }),
                        /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("path", {
                            fill: "url(#a7c756__d)",
                            d: "M19.2-1.144c-3.017-1.76-6.841-1.771-9.87-.028l-30.3 17.44c-2.968 1.708-2.973 5.74-.01 7.454l30.1 17.415c3.025 1.75 6.849 1.748 9.871-.006L49.03 23.7c2.947-1.711 2.953-5.72.009-7.437z"
                        }),
                        /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("path", {
                            fill: "url(#a7c756__e)",
                            d: "M9.862 22.99C10.534 9.157 22.972-1.54 37.644-.908s26.021 12.358 25.35 26.19c-.672 13.83-13.11 24.529-27.782 23.896C20.54 48.545 9.19 36.82 9.862 22.989"
                        }),
                        /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("path", {
                            fill: "url(#a7c756__f)",
                            d: "M-11.723 32.11c5.832-2.346 12.122-2.246 17.571-.194 10.905 4.115 17.25 1.57 21.65-8.685 2.192-5.146 6.472-9.505 12.31-11.853 11.674-4.687 25.15.399 30.116 11.37 4.965 10.973-.47 23.657-12.14 28.35-5.852 2.353-12.154 2.243-17.612.173C29.286 47.178 22.953 49.732 18.558 60c-2.198 5.134-6.476 9.477-12.309 11.823-11.664 4.69-25.14-.395-30.103-11.361-4.962-10.966.467-23.662 12.131-28.353"
                        }),
                        /*#__PURE__*/ (0, $ib1ED$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#a7c756__g)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 24.086 -4.002)"
                        })
                    ]
                })
            })
        ]
    });
const $e160c8b0e262c4f0$var$ForwardRef = /*#__PURE__*/ (0, $ib1ED$react.forwardRef)($e160c8b0e262c4f0$var$SvgComponent);
var $e160c8b0e262c4f0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e160c8b0e262c4f0$var$ForwardRef);
