// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_AIGenerate_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4heh2$reactjsxruntime = require("react/jsx-runtime");
var $4heh2$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c720b031bbb608e3$export$2e2bcd8739ae039);




const $c720b031bbb608e3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3ccbc6__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3ccbc6__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3ccbc6__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3ccbc6__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3ccbc6__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("clipPath", {
                        id: "3ccbc6__b",
                        children: /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("clipPath", {
                id: "3ccbc6__a",
                children: /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M38.075 7.775c.916-.741 2.268.119 2.004 1.343l-.614 2.85c-.092.423.022.868.304 1.187l1.897 2.148c.556.63.379 1.505-.155 1.945-.274.221-.639.329-1.046.234l-2.732-.642c-.405-.095-.831.024-1.137.318l-2.058 1.98c-.914.878-2.367.002-2.095-1.263l.614-2.85c.092-.423-.022-.868-.304-1.187l-1.897-2.148c-.556-.63-.379-1.505.155-1.945.274-.221.639-.33 1.046-.234l2.732.641c.373.089.765-.005 1.063-.251q.042-.034.082-.073l2.059-1.98q.04-.038.082-.073M28.078 11.5H14.516c-2.21 0-4 1.79-4 4v19c0 2.21 1.79 4 4 4h19c2.209 0 4-1.79 4-4V19.742l-1.245 1.197c-1.357 1.305-3.209 1.3-4.516.513-1.29-.778-2.133-2.35-1.756-4.103l.494-2.293-1.51-1.71c-.5-.566-.788-1.203-.905-1.846m-1.59 14.684 2.008.453c.79.178 1.338-.77.788-1.366l-1.395-1.512c-.184-.2-.258-.476-.198-.741l.452-2.008c.178-.79-.77-1.338-1.366-.788l-1.512 1.395c-.2.184-.475.258-.741.199l-2.008-.453c-.789-.178-1.337.77-.787 1.366l1.395 1.512c.183.2.257.476.198.741l-.453 2.008c-.178.79.77 1.337 1.366.788l1.512-1.396c.2-.183.476-.257.742-.198"
                })
            }),
            /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("g", {
                clipPath: "url(#3ccbc6__a)",
                children: /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#3ccbc6__b)",
                    children: [
                        /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#3ccbc6__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("path", {
                            fill: "url(#3ccbc6__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("path", {
                            fill: "url(#3ccbc6__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("path", {
                            fill: "url(#3ccbc6__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $4heh2$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#3ccbc6__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $c720b031bbb608e3$var$ForwardRef = /*#__PURE__*/ (0, $4heh2$react.forwardRef)($c720b031bbb608e3$var$SvgComponent);
var $c720b031bbb608e3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c720b031bbb608e3$var$ForwardRef);
