// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileText_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $bQVHB$reactjsxruntime = require("react/jsx-runtime");
var $bQVHB$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $55eab672a1ba96bd$export$2e2bcd8739ae039);




const $55eab672a1ba96bd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsxs)("linearGradient", {
                        id: "899127__d",
                        x1: -24.021,
                        x2: 201.071,
                        y1: 134.5,
                        y2: -2.444,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsxs)("linearGradient", {
                        id: "899127__e",
                        x1: 156.616,
                        x2: 60.052,
                        y1: 36.277,
                        y2: 140.11,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsxs)("linearGradient", {
                        id: "899127__f",
                        x1: 39.265,
                        x2: 210.447,
                        y1: 154.441,
                        y2: 58.547,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsxs)("radialGradient", {
                        id: "899127__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.455 52.6)scale(60.2331)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsxs)("radialGradient", {
                        id: "899127__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 52.682 88.47)scale(60.2331)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("clipPath", {
                        id: "899127__b",
                        children: /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 10.773,
                            y: 10.775,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("clipPath", {
                id: "899127__a",
                children: /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "m120.485 64.485-32.97-32.97C85.265 29.265 82.212 28 79.029 28H48c-6.627 0-12 5.373-12 12v80c0 6.627 5.373 12 12 12h64c6.627 0 12-5.373 12-12V72.97c0-3.182-1.264-6.234-3.515-8.485M104 84c0 2.209-1.791 4-4 4s-4-1.791-4-4v-4H84v32h4c2.209 0 4 1.791 4 4s-1.791 4-4 4H72c-2.209 0-4-1.791-4-4s1.791-4 4-4h4V80H64v4c0 2.209-1.791 4-4 4s-4-1.791-4-4v-8c0-2.209 1.791-4 4-4h40c2.209 0 4 1.791 4 4z"
                })
            }),
            /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("g", {
                clipPath: "url(#899127__a)",
                children: /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#899127__b)",
                    children: [
                        /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 10.773,
                            y: 10.775,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M10.773 10.775h138.45v138.45H10.772z"
                        }),
                        /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("circle", {
                            cx: 89.055,
                            cy: 16.146,
                            r: 60.233,
                            fill: "url(#899127__c)",
                            transform: "rotate(-.08 89.055 16.146)"
                        }),
                        /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("path", {
                            fill: "url(#899127__d)",
                            d: "M63.616-13.68c-10.633-6.58-24.104-6.62-34.777-.105L-77.903 51.39c-10.454 6.383-10.471 21.45-.031 27.856l106.037 65.081c10.656 6.54 24.127 6.532 34.775-.023l105.818-65.142c10.385-6.393 10.404-21.376.032-27.793z"
                        }),
                        /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("path", {
                            fill: "url(#899127__e)",
                            d: "M30.726 76.508c2.366-51.688 46.185-91.67 97.872-89.304 51.688 2.367 91.67 46.186 89.304 97.873-2.367 51.687-46.186 91.67-97.873 89.303-51.687-2.366-91.67-46.185-89.303-97.872"
                        }),
                        /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("path", {
                            fill: "url(#899127__f)",
                            d: "M-45.32 110.588c20.546-8.765 42.705-8.389 61.902-.723 38.415 15.38 60.767 5.871 76.27-32.455 7.722-19.233 22.799-35.52 43.368-44.295 41.124-17.517 88.601 1.49 106.093 42.493s-1.654 88.404-42.769 105.944c-20.615 8.794-42.816 8.382-62.042.647-38.353-15.297-60.662-5.753-76.146 32.62-7.742 19.186-22.815 35.418-43.36 44.183-41.092 17.53-88.568-1.477-106.05-42.457-17.483-40.98 1.643-88.427 42.735-105.957"
                        }),
                        /*#__PURE__*/ (0, $bQVHB$reactjsxruntime.jsx)("circle", {
                            cx: 141.153,
                            cy: 35.788,
                            r: 60.233,
                            fill: "url(#899127__g)",
                            transform: "rotate(-.08 141.153 35.788)"
                        })
                    ]
                })
            })
        ]
    });
const $55eab672a1ba96bd$var$ForwardRef = /*#__PURE__*/ (0, $bQVHB$react.forwardRef)($55eab672a1ba96bd$var$SvgComponent);
var $55eab672a1ba96bd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($55eab672a1ba96bd$var$ForwardRef);
