// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_cardTapPayment_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $hd55o$reactjsxruntime = require("react/jsx-runtime");
var $hd55o$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e23d3dbf2f2f1ebc$export$2e2bcd8739ae039);




const $e23d3dbf2f2f1ebc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsxs)("linearGradient", {
                        id: "112f9f__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsxs)("linearGradient", {
                        id: "112f9f__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsxs)("linearGradient", {
                        id: "112f9f__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsxs)("radialGradient", {
                        id: "112f9f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsxs)("radialGradient", {
                        id: "112f9f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("clipPath", {
                        id: "112f9f__b",
                        children: /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("clipPath", {
                id: "112f9f__a",
                children: /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M68.322 15.752c.134-1.097 1.132-1.877 2.228-1.743 6.553.8 12.672 4.565 16.23 10.728 3.588 6.214 3.763 13.463 1.112 19.57-.44 1.012-1.617 1.477-2.63 1.038-1.014-.44-1.479-1.618-1.039-2.631 2.168-4.994 2.02-10.906-.906-15.977-2.904-5.029-7.893-8.103-13.252-8.757-1.096-.134-1.876-1.132-1.743-2.228m-.75 8.334c.135-1.096 1.132-1.877 2.228-1.743 3.982.486 7.704 2.775 9.868 6.523 2.181 3.778 2.286 8.186.676 11.897-.44 1.013-1.618 1.478-2.631 1.038s-1.478-1.618-1.038-2.63c1.127-2.6 1.05-5.67-.471-8.305-1.51-2.614-4.1-4.212-6.888-4.552-1.097-.134-1.877-1.132-1.743-2.228m-57.27 20.878c-3.826 2.209-5.137 7.102-2.928 10.928l1.211 2.098 53.978-31.106-1.236-2.142c-2.21-3.826-7.102-5.137-10.928-2.928zm54.261-14.616L10.585 61.454l9.555 16.55c2.21 3.827 7.102 5.138 10.929 2.928l40.096-23.15c3.827-2.208 5.138-7.101 2.928-10.928zM19.19 64.122c-.553-.956-.225-2.18.732-2.732l19.355-11.176c.956-.553 2.18-.225 2.732.732.552.956.225 2.18-.732 2.732L21.922 64.854c-.957.552-2.18.225-2.732-.732m4.923 4.526c-.956.552-1.284 1.775-.732 2.732.552.956 1.776 1.284 2.732.732l9.678-5.588c.956-.553 1.284-1.776.732-2.732-.553-.957-1.776-1.285-2.732-.732zm25.93-22.338c-.552-.957-.224-2.18.732-2.733l9.074-5.24c.957-.552 2.18-.224 2.732.733.553.956.225 2.18-.732 2.732l-9.074 5.24c-.956.552-2.18.224-2.732-.733"
                })
            }),
            /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("g", {
                clipPath: "url(#112f9f__a)",
                children: /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#112f9f__b)",
                    children: [
                        /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#112f9f__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("path", {
                            fill: "url(#112f9f__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("path", {
                            fill: "url(#112f9f__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("path", {
                            fill: "url(#112f9f__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $hd55o$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#112f9f__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $e23d3dbf2f2f1ebc$var$ForwardRef = /*#__PURE__*/ (0, $hd55o$react.forwardRef)($e23d3dbf2f2f1ebc$var$SvgComponent);
var $e23d3dbf2f2f1ebc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e23d3dbf2f2f1ebc$var$ForwardRef);
