// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_explosion_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7SYAK$jsxs, jsx as $7SYAK$jsx} from "react/jsx-runtime";
import {forwardRef as $7SYAK$forwardRef} from "react";





const $337e065a06fc9b8f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7SYAK$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7SYAK$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7SYAK$jsxs)("linearGradient", {
                        id: "ed988a__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.896,
                        y2: -4.582,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SYAK$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $7SYAK$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SYAK$jsxs)("linearGradient", {
                        id: "ed988a__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.843,
                        y2: 44.841,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SYAK$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7SYAK$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SYAK$jsxs)("linearGradient", {
                        id: "ed988a__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.81,
                        y2: 16.564,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SYAK$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7SYAK$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7SYAK$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7SYAK$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SYAK$jsxs)("radialGradient", {
                        id: "ed988a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.503)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SYAK$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $7SYAK$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SYAK$jsxs)("radialGradient", {
                        id: "ed988a__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.264 26.937)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SYAK$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7SYAK$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SYAK$jsx)("clipPath", {
                        id: "ed988a__b",
                        children: /*#__PURE__*/ (0, $7SYAK$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7SYAK$jsx)("clipPath", {
                id: "ed988a__a",
                children: /*#__PURE__*/ (0, $7SYAK$jsx)("path", {
                    fill: "#292929",
                    d: "M9.403 7.442c-.918-1.699 1.187-3.414 2.665-2.172l11.122 9.345 6.493-7.168c1.208-1.334 3.4-.196 3.004 1.56l-1.87 8.3h14.43c2.019 0 2.432 2.854.496 3.427L33.188 24.45l3.822 4.5c1.156 1.36-.228 3.374-1.913 2.784l-5.815-2.039v12.253c0 1.943-2.694 2.439-3.386.623l-4.573-12c-2.985 1.716-7.09 4.043-9.487 5.4-1.656.936-3.354-1.042-2.285-2.542 1.408-1.976 3.654-5.142 5.302-7.526l-8.675-2.226c-1.769-.454-1.747-2.974.03-3.398l8.975-2.138z"
                })
            }),
            /*#__PURE__*/ (0, $7SYAK$jsx)("g", {
                clipPath: "url(#ed988a__a)",
                children: /*#__PURE__*/ (0, $7SYAK$jsxs)("g", {
                    clipPath: "url(#ed988a__b)",
                    children: [
                        /*#__PURE__*/ (0, $7SYAK$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: 0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $7SYAK$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 .001h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $7SYAK$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.863,
                            r: 20.883,
                            fill: "url(#ed988a__c)",
                            transform: "rotate(-.08 27.142 1.863)"
                        }),
                        /*#__PURE__*/ (0, $7SYAK$jsx)("path", {
                            fill: "url(#ed988a__d)",
                            d: "M18.321-8.478c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.657L6.008 46.304c3.694 2.267 8.365 2.264 12.056-.008L54.752 23.71c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $7SYAK$jsx)("path", {
                            fill: "url(#ed988a__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.99 40.85-8.17S72.63 7.84 71.809 25.76 55.798 57.543 37.879 56.722C19.957 55.902 6.095 40.71 6.915 22.79"
                        }),
                        /*#__PURE__*/ (0, $7SYAK$jsx)("path", {
                            fill: "url(#ed988a__f)",
                            d: "M-19.448 34.606c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.331 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.649-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.279-15.033 15.318-14.246 6.077-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $7SYAK$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.673,
                            r: 20.883,
                            fill: "url(#ed988a__g)",
                            transform: "rotate(-.08 45.2 8.673)"
                        })
                    ]
                })
            })
        ]
    });
const $337e065a06fc9b8f$var$ForwardRef = /*#__PURE__*/ (0, $7SYAK$forwardRef)($337e065a06fc9b8f$var$SvgComponent);
var $337e065a06fc9b8f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($337e065a06fc9b8f$var$ForwardRef);


export {$337e065a06fc9b8f$export$2e2bcd8739ae039 as default};
