// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_threeArrowsCircle_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $kyEvW$jsxs, jsx as $kyEvW$jsx} from "react/jsx-runtime";
import {forwardRef as $kyEvW$forwardRef} from "react";





const $e3ed4e5ec83ec0db$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kyEvW$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kyEvW$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kyEvW$jsxs)("linearGradient", {
                        id: "413b28__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kyEvW$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kyEvW$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kyEvW$jsxs)("linearGradient", {
                        id: "413b28__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kyEvW$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kyEvW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kyEvW$jsxs)("linearGradient", {
                        id: "413b28__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kyEvW$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kyEvW$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kyEvW$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kyEvW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kyEvW$jsxs)("radialGradient", {
                        id: "413b28__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kyEvW$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kyEvW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kyEvW$jsxs)("radialGradient", {
                        id: "413b28__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kyEvW$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kyEvW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kyEvW$jsx)("clipPath", {
                        id: "413b28__b",
                        children: /*#__PURE__*/ (0, $kyEvW$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kyEvW$jsx)("clipPath", {
                id: "413b28__a",
                children: /*#__PURE__*/ (0, $kyEvW$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M126.811 40.186c.548 2.14-.743 4.319-2.883 4.867l-24.791 6.345c-2.14.548-4.32-.743-4.867-2.883-.548-2.14.743-4.319 2.883-4.867l15.233-3.899q-.122-.082-.239-.175c-8.853-7.027-20.047-11.222-32.23-11.222-15.564 0-29.523 6.847-39.04 17.705-1.455 1.662-3.982 1.828-5.644.372s-1.828-3.983-.372-5.644c10.971-12.519 27.091-20.433 45.055-20.433 13.867 0 26.642 4.717 36.794 12.632l-3.995-15.605c-.548-2.14.743-4.32 2.883-4.867 2.14-.548 4.319.743 4.867 2.883zM129.71 64.8c-.659-2.108.515-4.352 2.623-5.012s4.353.515 5.012 2.623c1.771 5.658 2.723 11.673 2.723 17.903 0 24.134-14.273 44.925-34.828 54.407l14.547 5.119c2.084.733 3.179 3.017 2.446 5.101s-3.017 3.179-5.101 2.445l-24.14-8.493c-2.084-.733-3.179-3.017-2.445-5.101l8.493-24.139c.733-2.084 3.017-3.179 5.101-2.446s3.179 3.017 2.445 5.101l-5.449 15.489c18.223-8.055 30.931-26.289 30.931-47.483 0-5.41-.826-10.62-2.358-15.514M28.689 73.391l10.065 12.337c1.396 1.712 3.916 1.968 5.627.571 1.712-1.396 1.968-3.916.571-5.628L28.776 60.843c-1.397-1.712-3.916-1.968-5.628-.571L3.319 76.448c-1.712 1.397-1.967 3.917-.57 5.628 1.396 1.712 3.916 1.968 5.627.571l12.351-10.076c-1.473 11.234.22 23.014 5.55 33.958 7.866 16.15 22.04 27.178 38.098 31.56 2.132.581 4.33-.675 4.912-2.806s-.674-4.331-2.806-4.912c-13.93-3.801-26.197-13.353-33.01-27.345-4.656-9.559-6.11-19.838-4.782-29.635"
                })
            }),
            /*#__PURE__*/ (0, $kyEvW$jsx)("g", {
                clipPath: "url(#413b28__a)",
                children: /*#__PURE__*/ (0, $kyEvW$jsxs)("g", {
                    clipPath: "url(#413b28__b)",
                    children: [
                        /*#__PURE__*/ (0, $kyEvW$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $kyEvW$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $kyEvW$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#413b28__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $kyEvW$jsx)("path", {
                            fill: "url(#413b28__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $kyEvW$jsx)("path", {
                            fill: "url(#413b28__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $kyEvW$jsx)("path", {
                            fill: "url(#413b28__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $kyEvW$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#413b28__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $e3ed4e5ec83ec0db$var$ForwardRef = /*#__PURE__*/ (0, $kyEvW$forwardRef)($e3ed4e5ec83ec0db$var$SvgComponent);
var $e3ed4e5ec83ec0db$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e3ed4e5ec83ec0db$var$ForwardRef);


export {$e3ed4e5ec83ec0db$export$2e2bcd8739ae039 as default};
