// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_stepForward_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $kWFIc$reactjsxruntime = require("react/jsx-runtime");
var $kWFIc$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $404e2285dbb15ff6$export$2e2bcd8739ae039);




const $404e2285dbb15ff6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8cbb14__d",
                        x1: -14.062,
                        x2: 63.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8cbb14__e",
                        x1: 48.562,
                        x2: 15.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8cbb14__f",
                        x1: 7.878,
                        x2: 67.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8cbb14__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.64 13.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8cbb14__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 17.265 25.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("clipPath", {
                        id: "8cbb14__b",
                        children: /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            x: -2,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("clipPath", {
                id: "8cbb14__a",
                children: /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "m39.577 21.564-16.373-9.777c-1.834-1.095-4.13.268-4.132 2.453l-.022 19.514c-.003 2.185 2.29 3.554 4.125 2.464l16.396-9.737c1.836-1.09 1.84-3.822.006-4.917M7 13c0-1.105.895-2 2-2h3.5c1.105 0 2 .895 2 2v22c0 1.105-.895 2-2 2H9c-1.105 0-2-.895-2-2z"
                })
            }),
            /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("g", {
                clipPath: "url(#8cbb14__a)",
                children: /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#8cbb14__b)",
                    children: [
                        /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            x: -2,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M-2 0h48v48H-2z"
                        }),
                        /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("circle", {
                            cx: 25.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#8cbb14__c)",
                            transform: "rotate(-.08 25.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("path", {
                            fill: "url(#8cbb14__d)",
                            d: "M16.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L4.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("path", {
                            fill: "url(#8cbb14__e)",
                            d: "M4.916 22.79C5.736 4.87 20.93-8.993 38.85-8.173S70.63 7.841 69.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("path", {
                            fill: "url(#8cbb14__f)",
                            d: "M-21.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $kWFIc$reactjsxruntime.jsx)("circle", {
                            cx: 43.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#8cbb14__g)",
                            transform: "rotate(-.08 43.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $404e2285dbb15ff6$var$ForwardRef = /*#__PURE__*/ (0, $kWFIc$react.forwardRef)($404e2285dbb15ff6$var$SvgComponent);
var $404e2285dbb15ff6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($404e2285dbb15ff6$var$ForwardRef);
