// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_globe_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7Q9xA$jsxs, jsx as $7Q9xA$jsx} from "react/jsx-runtime";
import {forwardRef as $7Q9xA$forwardRef} from "react";





const $b00efae3e3e42c23$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7Q9xA$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7Q9xA$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7Q9xA$jsxs)("linearGradient", {
                        id: "24dc54__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Q9xA$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $7Q9xA$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Q9xA$jsxs)("linearGradient", {
                        id: "24dc54__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Q9xA$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7Q9xA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Q9xA$jsxs)("linearGradient", {
                        id: "24dc54__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Q9xA$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7Q9xA$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7Q9xA$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7Q9xA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Q9xA$jsxs)("radialGradient", {
                        id: "24dc54__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Q9xA$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $7Q9xA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Q9xA$jsxs)("radialGradient", {
                        id: "24dc54__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Q9xA$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7Q9xA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Q9xA$jsx)("clipPath", {
                        id: "24dc54__b",
                        children: /*#__PURE__*/ (0, $7Q9xA$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7Q9xA$jsx)("clipPath", {
                id: "24dc54__a",
                children: /*#__PURE__*/ (0, $7Q9xA$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M48 14.74c1.787 0 3.541.142 5.252.416-.539.73-1.25 1.565-2.082 2.276-1.154.985-2.349 1.568-3.517 1.568-1.23 0-2.546-.54-3.716-1.414-.997-.745-1.719-1.608-2.095-2.272 1.995-.377 4.053-.574 6.158-.574m9.122 1.854q.157-.244.238-.508.724.214 1.434.46-.038.129-.058.265c-.394 2.607-.878 6.313-.463 9.631.411 3.293 1.829 6.914 5.851 8.1 3.074.905 5.989.302 8.271-.828 1.56-.772 2.917-1.836 3.901-2.964 1.485 2.468 2.656 5.146 3.458 7.979q-.466-.101-1.018-.211c-1.724-.344-4.044-.733-6.478-.895-2.41-.16-5.063-.105-7.398.519-2.34.625-4.636 1.906-5.809 4.395-1.725 3.662-1.522 6.785-.697 9.117.804 2.27 2.15 3.67 2.548 4.068 1.456 1.457 2.943 2.134 4.077 2.652q.208.094.398.183c1.122.524 1.969 1.003 2.817 2.228.327.473.62 1.225.833 2.277.209 1.03.316 2.217.36 3.46.058 1.661.005 3.163-.041 4.49-.024.695-.047 1.342-.05 1.939-5.748 4.86-13.18 7.79-21.296 7.79q-2.038-.002-4.012-.242c.27-.604.567-1.246.882-1.927l.463-1.003c.674-1.466 1.39-3.062 2.015-4.673 1.205-3.11 2.253-6.712 1.644-9.76-.38-1.896-1.298-3.562-3.041-4.793-1.66-1.171-3.899-1.828-6.708-2.14-4.388-.488-6.51-.946-8.434-2.486-1.775-1.42-2.255-2.75-2.334-3.746-.085-1.076.28-2.053.637-2.678.389-.68 1.252-1.165 2.89-1.336 1.423-.15 2.96-.016 4.368.106l.546.047c1.651.138 3.348-.068 4.53-1.315 1.115-1.177 1.276-2.791 1.276-4.075 0-.768.215-1.306.58-1.775.4-.516 1.048-1.027 2.03-1.572 1.583-.88 2.71-2.408 3.002-4.11.307-1.792-.345-3.69-2.135-4.884-.94-.626-2.03-.857-2.98-.955-.964-.098-2.004-.077-2.955-.047l-.41.014c-1.873.06-3.415.112-4.65-.241-.724-.207-1.876-1.09-2.993-2.567-1.086-1.434-1.8-2.982-1.959-4.011q-.046-.294-.167-.55c2.41-1.56 5.035-2.815 7.822-3.71q.03.1.071.197c.63 1.499 1.958 3.08 3.564 4.28 1.62 1.212 3.76 2.211 6.111 2.211 2.505 0 4.596-1.23 6.114-2.526 1.534-1.309 2.684-2.854 3.355-3.88m16.282 52.21C78.148 63.09 81 55.749 81 47.74c0-1.598-.114-3.17-.333-4.708l-.096-.021-.423-.096-.198-.045c-.5-.114-1.182-.267-1.995-.43-1.632-.325-3.766-.68-5.962-.826-2.22-.147-4.374-.07-6.1.392-1.72.46-2.743 1.217-3.223 2.236-1.253 2.658-1.037 4.686-.545 6.078.515 1.454 1.385 2.353 1.606 2.574.923.924 1.778 1.318 2.912 1.84l.427.199c1.403.655 2.983 1.51 4.413 3.575.764 1.105 1.2 2.453 1.464 3.76.27 1.326.39 2.754.437 4.115.028.793.03 1.62.02 2.422m-32.145 8.048c-.47 1.014-.926 2.002-1.312 2.899C25.614 76.156 15 63.188 15 47.74c0-10.08 4.519-19.104 11.64-25.157.497 1.524 1.382 3.093 2.382 4.413 1.28 1.69 3.092 3.43 5.084 3.999 1.878.536 4.063.458 5.768.397l.517-.018c.96-.03 1.755-.04 2.42.029.681.07 1.017.202 1.169.303.236.158.331.308.375.414.047.115.07.268.037.466-.072.421-.4.955-1.002 1.289-1.202.668-2.374 1.492-3.246 2.613-.91 1.17-1.422 2.572-1.422 4.23 0 .524-.041.872-.099 1.1-.032.128-.063.192-.078.217-.076.041-.395.163-1.297.088l-.509-.044c-1.37-.121-3.317-.294-5.153-.102-2.048.215-4.583.944-5.947 3.33-.613 1.074-1.319 2.856-1.151 4.979.174 2.204 1.272 4.513 3.822 6.553 2.927 2.342 6.146 2.855 10.491 3.338 2.502.278 3.973.819 4.843 1.433.787.556 1.216 1.264 1.425 2.31.361 1.805-.25 4.431-1.45 7.53-.579 1.491-1.253 2.995-1.92 4.448zm32.627-49.58C70.858 23.446 67 20.308 62.585 18.13c-.354 2.473-.65 5.35-.342 7.816.342 2.735 1.307 4.256 3.012 4.758 1.899.56 3.77.216 5.366-.575 1.64-.812 2.791-1.998 3.232-2.798zM38.537 42.05q0 0 .002-.003zm.02-.02.009-.007z"
                })
            }),
            /*#__PURE__*/ (0, $7Q9xA$jsx)("g", {
                clipPath: "url(#24dc54__a)",
                children: /*#__PURE__*/ (0, $7Q9xA$jsxs)("g", {
                    clipPath: "url(#24dc54__b)",
                    children: [
                        /*#__PURE__*/ (0, $7Q9xA$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $7Q9xA$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $7Q9xA$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#24dc54__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $7Q9xA$jsx)("path", {
                            fill: "url(#24dc54__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $7Q9xA$jsx)("path", {
                            fill: "url(#24dc54__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $7Q9xA$jsx)("path", {
                            fill: "url(#24dc54__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $7Q9xA$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#24dc54__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $b00efae3e3e42c23$var$ForwardRef = /*#__PURE__*/ (0, $7Q9xA$forwardRef)($b00efae3e3e42c23$var$SvgComponent);
var $b00efae3e3e42c23$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b00efae3e3e42c23$var$ForwardRef);


export {$b00efae3e3e42c23$export$2e2bcd8739ae039 as default};
