// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_AIGenerate_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $fkX8P$reactjsxruntime = require("react/jsx-runtime");
var $fkX8P$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5d3f1a518ea9c900$export$2e2bcd8739ae039);




const $5d3f1a518ea9c900$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsxs)("linearGradient", {
                        id: "038618__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsxs)("linearGradient", {
                        id: "038618__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsxs)("linearGradient", {
                        id: "038618__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsxs)("radialGradient", {
                        id: "038618__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsxs)("radialGradient", {
                        id: "038618__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("clipPath", {
                        id: "038618__b",
                        children: /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("clipPath", {
                id: "038618__a",
                children: /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M80.934 33.977c-.293 1.304-1.518 2.355-3.034 2l-5.463-1.282q-.208-.05-.418-.061c-.677-.039-1.349.208-1.856.696l-4.117 3.959c-1.827 1.758-4.734.006-4.19-2.525l1.23-5.7c.183-.847-.045-1.736-.609-2.373l-3.794-4.296q-.168-.192-.293-.395c-.344-.562-.446-1.185-.36-1.77.205-1.397 1.477-2.581 3.073-2.207l5.463 1.282c.81.191 1.662-.046 2.273-.635l4.118-3.959c1.827-1.758 4.734-.006 4.19 2.525l-1.23 5.7q-.034.16-.049.32c-.072.745.165 1.495.658 2.053l3.794 4.296c.51.577.711 1.257.678 1.91q-.012.233-.064.462M54.143 24c.222 1.06.716 2.108 1.542 3.042v.001l3.356 3.8-1.095 5.08c-.658 3.058.81 5.788 3.051 7.138 2.27 1.367 5.468 1.375 7.822-.889l3.2-3.078V70c0 4.418-3.581 8-8 8h-38c-4.418 0-8-3.582-8-8V32c0-4.419 3.582-8 8-8zm-2.177 27.368 4.016.905c1.578.357 2.674-1.539 1.575-2.73l-2.79-3.025c-.367-.4-.516-.951-.397-1.483l.905-4.015c.357-1.579-1.539-2.675-2.73-1.576l-3.026 2.79c-.4.367-.95.516-1.482.397l-4.016-.905c-1.578-.356-2.674 1.539-1.575 2.731l2.79 3.025c.367.4.516.95.397 1.483l-.905 4.015c-.357 1.578 1.539 2.675 2.73 1.575l3.026-2.79c.4-.367.95-.516 1.482-.397"
                })
            }),
            /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("g", {
                clipPath: "url(#038618__a)",
                children: /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#038618__b)",
                    children: [
                        /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#038618__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("path", {
                            fill: "url(#038618__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("path", {
                            fill: "url(#038618__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("path", {
                            fill: "url(#038618__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $fkX8P$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#038618__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $5d3f1a518ea9c900$var$ForwardRef = /*#__PURE__*/ (0, $fkX8P$react.forwardRef)($5d3f1a518ea9c900$var$SvgComponent);
var $5d3f1a518ea9c900$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5d3f1a518ea9c900$var$ForwardRef);
