// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_lighten_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $lOoWu$reactjsxruntime = require("react/jsx-runtime");
var $lOoWu$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a1c973f1320f85bc$export$2e2bcd8739ae039);




const $a1c973f1320f85bc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 97,
        height: 96,
        viewBox: "0 0 97 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c32554__d",
                        x1: -23.183,
                        x2: 132.895,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c32554__e",
                        x1: 102.07,
                        x2: 35.113,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c32554__f",
                        x1: 20.697,
                        x2: 139.393,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsxs)("radialGradient", {
                        id: "c32554__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.75 29.473)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsxs)("radialGradient", {
                        id: "c32554__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 37.002 54.345)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("clipPath", {
                        id: "c32554__b",
                        children: /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("clipPath", {
                id: "c32554__a",
                children: /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M48.941 13.234c1.105 0 2 .896 2 2v4.645c0 1.104-.895 2-2 2s-2-.896-2-2v-4.645c0-1.104.896-2 2-2M65.49 48.001c0 9.14-7.41 16.549-16.549 16.549-9.14 0-16.549-7.41-16.549-16.55 0-9.139 7.41-16.548 16.549-16.548 9.14 0 16.549 7.41 16.549 16.549M50.94 76.12c0-1.104-.895-2-2-2s-2 .896-2 2v4.645c0 1.104.896 2 2 2s2-.896 2-2zM83.707 48c0 1.105-.895 2-2 2h-4.644c-1.105 0-2-.895-2-2s.895-2 2-2h4.644c1.105 0 2 .895 2 2M20.82 50c1.105 0 2-.895 2-2s-.895-2-2-2h-4.644c-1.105 0-2 .895-2 2s.895 2 2 2zm52.704-26.584c.781.781.781 2.047 0 2.828L70.24 29.53c-.781.78-2.047.78-2.828 0-.782-.781-.782-2.048 0-2.829l3.284-3.284c.78-.78 2.047-.78 2.828 0M30.471 69.298c.78-.781.78-2.048 0-2.829-.782-.78-2.048-.78-2.829 0l-3.284 3.285c-.781.78-.781 2.047 0 2.828.78.781 2.047.781 2.828 0zm-6.113-45.882c-.781.781-.781 2.047 0 2.828l3.284 3.285c.78.78 2.047.78 2.828 0 .781-.781.781-2.048 0-2.829l-3.284-3.284c-.781-.78-2.047-.78-2.828 0M67.41 69.298c-.78-.781-.78-2.048 0-2.829.781-.78 2.048-.78 2.829 0l3.284 3.285c.781.78.781 2.047 0 2.828s-2.047.781-2.828 0z"
                })
            }),
            /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("g", {
                clipPath: "url(#c32554__a)",
                children: /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#c32554__b)",
                    children: [
                        /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M.941 0h96v96h-96z"
                        }),
                        /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("circle", {
                            cx: 55.222,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#c32554__c)",
                            transform: "rotate(-.08 55.222 3.723)"
                        }),
                        /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("path", {
                            fill: "url(#c32554__d)",
                            d: "M37.584-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.014 45.19c-7.25 4.427-7.261 14.873-.022 19.316l73.525 45.127c7.39 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.201-4.433 7.214-14.821.022-19.27z"
                        }),
                        /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("path", {
                            fill: "url(#c32554__e)",
                            d: "M14.778 45.579c1.64-35.84 32.025-63.563 67.864-61.923s63.564 32.025 61.923 67.865-32.025 63.563-67.865 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("path", {
                            fill: "url(#c32554__f)",
                            d: "M-37.954 69.21c14.247-6.078 29.611-5.818 42.922-.502 26.637 10.664 42.136 4.071 52.885-22.504 5.355-13.336 15.81-24.63 30.071-30.714 28.516-12.146 61.436 1.033 73.565 29.464s-1.147 61.299-29.656 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.062-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.492 12.155-61.412-1.024-73.534-29.44-12.123-28.415 1.14-61.314 29.632-73.47"
                        }),
                        /*#__PURE__*/ (0, $lOoWu$reactjsxruntime.jsx)("circle", {
                            cx: 91.347,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#c32554__g)",
                            transform: "rotate(-.08 91.347 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $a1c973f1320f85bc$var$ForwardRef = /*#__PURE__*/ (0, $lOoWu$react.forwardRef)($a1c973f1320f85bc$var$SvgComponent);
var $a1c973f1320f85bc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a1c973f1320f85bc$var$ForwardRef);
