// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_3Dasset_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $kVleY$jsxs, jsx as $kVleY$jsx} from "react/jsx-runtime";
import {forwardRef as $kVleY$forwardRef} from "react";





const $ecde968127664d0b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kVleY$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kVleY$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kVleY$jsxs)("linearGradient", {
                        id: "ec27ce__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.894,
                        y2: -4.584,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kVleY$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kVleY$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kVleY$jsxs)("linearGradient", {
                        id: "ec27ce__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.841,
                        y2: 44.839,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kVleY$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kVleY$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kVleY$jsxs)("linearGradient", {
                        id: "ec27ce__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.808,
                        y2: 16.562,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kVleY$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kVleY$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kVleY$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kVleY$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kVleY$jsxs)("radialGradient", {
                        id: "ec27ce__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kVleY$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kVleY$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kVleY$jsxs)("radialGradient", {
                        id: "ec27ce__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kVleY$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kVleY$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kVleY$jsx)("clipPath", {
                        id: "ec27ce__b",
                        children: /*#__PURE__*/ (0, $kVleY$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: -0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kVleY$jsx)("clipPath", {
                id: "ec27ce__a",
                children: /*#__PURE__*/ (0, $kVleY$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M8.808 12.323c1.812-.99 3.722-2.043 4.932-2.732.752-.429 1.754-.462 2.51-.04l6.778 3.772c.605.337 1.34.337 1.946 0l7.073-3.938c.605-.336 1.34-.336 1.946 0l5.263 2.93c.594.331.93.888 1.01 1.476L24.02 22.86 7.783 13.795c.084-.589.426-1.146 1.025-1.472m-1.054 4.32V26.2c0 1.354.686 2.617 1.822 3.355l12.237 7.94c.302.196.624.348.957.454V25.025zM25.27 37.93q.466-.157.891-.43l12.295-7.946c1.14-.737 1.83-2.002 1.83-3.36v-9.553L25.27 25.025z"
                })
            }),
            /*#__PURE__*/ (0, $kVleY$jsx)("g", {
                clipPath: "url(#ec27ce__a)",
                children: /*#__PURE__*/ (0, $kVleY$jsxs)("g", {
                    clipPath: "url(#ec27ce__b)",
                    children: [
                        /*#__PURE__*/ (0, $kVleY$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: -0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $kVleY$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0-.001h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $kVleY$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.861,
                            r: 20.883,
                            fill: "url(#ec27ce__c)",
                            transform: "rotate(-.08 27.142 1.861)"
                        }),
                        /*#__PURE__*/ (0, $kVleY$jsx)("path", {
                            fill: "url(#ec27ce__d)",
                            d: "M18.321-8.48c-3.686-2.28-8.357-2.295-12.057-.036L-30.743 14.08c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.3c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $kVleY$jsx)("path", {
                            fill: "url(#ec27ce__e)",
                            d: "M6.916 22.788C7.736 4.868 22.93-8.993 40.85-8.172S72.63 7.84 71.809 25.758c-.82 17.92-16.012 31.782-33.931 30.962-17.92-.82-31.782-16.013-30.962-33.933"
                        }),
                        /*#__PURE__*/ (0, $kVleY$jsx)("path", {
                            fill: "url(#ec27ce__f)",
                            d: "M-19.448 34.604c7.124-3.039 14.806-2.908 21.462-.25C15.332 39.686 23.08 36.389 28.456 23.1c2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $kVleY$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.671,
                            r: 20.883,
                            fill: "url(#ec27ce__g)",
                            transform: "rotate(-.08 45.2 8.67)"
                        })
                    ]
                })
            })
        ]
    });
const $ecde968127664d0b$var$ForwardRef = /*#__PURE__*/ (0, $kVleY$forwardRef)($ecde968127664d0b$var$SvgComponent);
var $ecde968127664d0b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ecde968127664d0b$var$ForwardRef);


export {$ecde968127664d0b$export$2e2bcd8739ae039 as default};
