// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_noElements_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $ilsmd$jsxs, jsx as $ilsmd$jsx} from "react/jsx-runtime";
import {forwardRef as $ilsmd$forwardRef} from "react";





const $2efe4515335d399e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ilsmd$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $ilsmd$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $ilsmd$jsxs)("linearGradient", {
                        id: "e400c7__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.894,
                        y2: -4.584,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ilsmd$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $ilsmd$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ilsmd$jsxs)("linearGradient", {
                        id: "e400c7__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.841,
                        y2: 44.839,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ilsmd$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ilsmd$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ilsmd$jsxs)("linearGradient", {
                        id: "e400c7__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.808,
                        y2: 16.562,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ilsmd$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $ilsmd$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $ilsmd$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $ilsmd$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ilsmd$jsxs)("radialGradient", {
                        id: "e400c7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ilsmd$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $ilsmd$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ilsmd$jsxs)("radialGradient", {
                        id: "e400c7__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ilsmd$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ilsmd$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ilsmd$jsx)("clipPath", {
                        id: "e400c7__b",
                        children: /*#__PURE__*/ (0, $ilsmd$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: -0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ilsmd$jsxs)("clipPath", {
                id: "e400c7__a",
                children: [
                    /*#__PURE__*/ (0, $ilsmd$jsx)("path", {
                        d: "M37.522 6.517c.278-.321.76-.59 1.058-.499l.002.002c.477 0 .938.26 1.202.675.261.41.304.928.117 1.39l-1.619 3.96 2.348 1.031c.425.186.736.576.836 1.043s-.024.958-.333 1.314l-6.13 7.073c-.275.319-.65.494-1.052.494-.477 0-.938-.257-1.202-.672-.261-.407-.306-.928-.119-1.388l1.566-3.86-2.333-1.045c-.42-.187-.73-.578-.83-1.042s.025-.95.329-1.306z"
                    }),
                    /*#__PURE__*/ (0, $ilsmd$jsx)("path", {
                        fillRule: "evenodd",
                        d: "M27 38.375V32c0-2.21 1.79-4 4-4h10.75c2.21 0 4 1.79 4 4v6.375c0 2.21-1.79 4-4 4H31c-2.21 0-4-1.79-4-4m12.026-3.746-3.689-2.105c-.413-.236-.93.057-.93.528l-.006 4.203c0 .47.516.765.93.53l3.694-2.096c.414-.235.414-.824.001-1.06M3.5 17.375V11c0-2.21 1.79-4 4-4h11.063c2.209 0 4 1.79 4 4v6.375c0 .593-.13 1.157-.361 1.663l-2.141-3.34c-1.28-1.995-4.194-1.995-5.473 0l-.39.61-1.603-2.527c-1.272-2.005-4.196-2.013-5.48-.015l-3.307 5.15c-.199-.474-.308-.995-.308-1.541M19.25 11.5c.69 0 1.25-.56 1.25-1.25S19.94 9 19.25 9 18 9.56 18 10.25s.56 1.25 1.25 1.25"
                    }),
                    /*#__PURE__*/ (0, $ilsmd$jsx)("path", {
                        d: "m10.484 15.12 2.225 3.51-1.457 2.272c-.095.148-.154.31-.181.473H7.5c-.716 0-1.387-.188-1.969-.517l3.688-5.74c.297-.462.971-.46 1.265.003M13.919 21.375h4.643c.676 0 1.313-.168 1.87-.463l-2.476-3.865c-.295-.46-.968-.46-1.263 0zM4.887 28c0-.69.56-1.25 1.25-1.25H20.21c.69 0 1.25.56 1.25 1.25v2.815c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25V29.25h-4.537v11.573h1.565c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-5.63c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h1.565V29.25H7.387v1.565c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ilsmd$jsx)("g", {
                clipPath: "url(#e400c7__a)",
                children: /*#__PURE__*/ (0, $ilsmd$jsxs)("g", {
                    clipPath: "url(#e400c7__b)",
                    children: [
                        /*#__PURE__*/ (0, $ilsmd$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: -0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $ilsmd$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0-.001h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $ilsmd$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.861,
                            r: 20.883,
                            fill: "url(#e400c7__c)",
                            transform: "rotate(-.08 27.142 1.861)"
                        }),
                        /*#__PURE__*/ (0, $ilsmd$jsx)("path", {
                            fill: "url(#e400c7__d)",
                            d: "M18.321-8.48c-3.686-2.28-8.357-2.295-12.057-.036L-30.743 14.08c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.3c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $ilsmd$jsx)("path", {
                            fill: "url(#e400c7__e)",
                            d: "M6.916 22.788C7.736 4.868 22.93-8.993 40.85-8.172S72.63 7.84 71.809 25.758c-.82 17.92-16.012 31.782-33.931 30.962-17.92-.82-31.782-16.013-30.962-33.933"
                        }),
                        /*#__PURE__*/ (0, $ilsmd$jsx)("path", {
                            fill: "url(#e400c7__f)",
                            d: "M-19.448 34.604c7.124-3.039 14.806-2.908 21.462-.25C15.332 39.686 23.08 36.389 28.456 23.1c2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $ilsmd$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.671,
                            r: 20.883,
                            fill: "url(#e400c7__g)",
                            transform: "rotate(-.08 45.2 8.67)"
                        })
                    ]
                })
            })
        ]
    });
const $2efe4515335d399e$var$ForwardRef = /*#__PURE__*/ (0, $ilsmd$forwardRef)($2efe4515335d399e$var$SvgComponent);
var $2efe4515335d399e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2efe4515335d399e$var$ForwardRef);


export {$2efe4515335d399e$export$2e2bcd8739ae039 as default};
