// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_microphone_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8Cu8K$reactjsxruntime = require("react/jsx-runtime");
var $8Cu8K$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $98e83ed7e2af8558$export$2e2bcd8739ae039);




const $98e83ed7e2af8558$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e64a03__d",
                        x1: -14.982,
                        x2: 120.749,
                        y1: 80.605,
                        y2: -1.972,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e64a03__e",
                        x1: 93.945,
                        x2: 35.717,
                        y1: 21.377,
                        y2: 83.989,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e64a03__f",
                        x1: 23.18,
                        x2: 126.403,
                        y1: 92.63,
                        y2: 34.806,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsxs)("radialGradient", {
                        id: "e64a03__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 21.982 31.22)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsxs)("radialGradient", {
                        id: "e64a03__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 31.768 52.85)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("clipPath", {
                        id: "e64a03__b",
                        children: /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6,
                            y: 6,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("clipPath", {
                id: "e64a03__a",
                children: /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M47.4 14.88c-6.362 0-11.52 5.158-11.52 11.52v17.16c0 6.362 5.158 11.52 11.52 11.52h1.56c6.363 0 11.52-5.158 11.52-11.52V26.4c0-6.362-5.157-11.52-11.52-11.52zm-1.92 54.465c-13.477-1.353-24-12.73-24-26.565v-2.7c0-1.326 1.075-2.4 2.4-2.4 1.326 0 2.4 1.074 2.4 2.4v2.7c0 12.095 9.805 21.9 21.9 21.9 12.096 0 21.9-9.805 21.9-21.9v-2.7c0-1.326 1.075-2.4 2.4-2.4 1.326 0 2.4 1.074 2.4 2.4v2.7c0 14.04-10.835 25.548-24.6 26.618v7.282h12c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4h-28.2c-1.325 0-2.4-1.075-2.4-2.4 0-1.326 1.075-2.4 2.4-2.4h11.4z"
                })
            }),
            /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("g", {
                clipPath: "url(#e64a03__a)",
                children: /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#e64a03__b)",
                    children: [
                        /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6,
                            y: 6,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6 6h83.485v83.485H6z"
                        }),
                        /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("circle", {
                            cx: 53.203,
                            cy: 9.238,
                            r: 36.321,
                            fill: "url(#e64a03__c)",
                            transform: "rotate(-.08 53.203 9.238)"
                        }),
                        /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("path", {
                            fill: "url(#e64a03__d)",
                            d: "M37.863-8.748c-6.411-3.967-14.535-3.991-20.97-.062l-64.366 39.3c-6.304 3.848-6.314 12.933-.019 16.797l63.94 39.244c6.426 3.943 14.55 3.938 20.97-.015l63.808-39.28c6.262-3.855 6.274-12.89.019-16.76z"
                        }),
                        /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("path", {
                            fill: "url(#e64a03__e)",
                            d: "M18.033 45.636C19.46 14.47 45.883-9.64 77.05-8.214s55.277 27.85 53.85 59.018-27.85 55.277-59.017 53.85-55.277-27.85-53.85-59.018"
                        }),
                        /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("path", {
                            fill: "url(#e64a03__f)",
                            d: "M-27.824 66.187c12.389-5.285 25.75-5.059 37.326-.436 23.164 9.274 36.643 3.54 45.991-19.57 4.657-11.598 13.748-21.42 26.15-26.71 24.799-10.563 53.427.898 63.975 25.623s-.997 53.308-25.79 63.884c-12.43 5.303-25.818 5.055-37.411.39-23.127-9.224-36.58-3.469-45.916 19.67-4.668 11.569-13.757 21.357-26.147 26.643-24.778 10.57-53.406-.891-63.948-25.602s.992-53.322 25.77-63.892"
                        }),
                        /*#__PURE__*/ (0, $8Cu8K$reactjsxruntime.jsx)("circle", {
                            cx: 84.617,
                            cy: 21.082,
                            r: 36.321,
                            fill: "url(#e64a03__g)",
                            transform: "rotate(-.08 84.617 21.082)"
                        })
                    ]
                })
            })
        ]
    });
const $98e83ed7e2af8558$var$ForwardRef = /*#__PURE__*/ (0, $8Cu8K$react.forwardRef)($98e83ed7e2af8558$var$SvgComponent);
var $98e83ed7e2af8558$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($98e83ed7e2af8558$var$ForwardRef);
