// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_information_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7Chyi$jsxs, jsx as $7Chyi$jsx} from "react/jsx-runtime";
import {forwardRef as $7Chyi$forwardRef} from "react";





const $934b9d6052411f2a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7Chyi$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7Chyi$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7Chyi$jsxs)("linearGradient", {
                        id: "0fe3d6__d",
                        x1: -3.949,
                        x2: 55.306,
                        y1: 37.802,
                        y2: -0.441,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Chyi$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $7Chyi$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Chyi$jsxs)("linearGradient", {
                        id: "0fe3d6__e",
                        x1: 45.213,
                        x2: 20.587,
                        y1: 12.602,
                        y2: 40.692,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Chyi$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7Chyi$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Chyi$jsxs)("linearGradient", {
                        id: "0fe3d6__f",
                        x1: 13.273,
                        x2: 58.507,
                        y1: 42.918,
                        y2: 16.037,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Chyi$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7Chyi$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7Chyi$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7Chyi$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Chyi$jsxs)("radialGradient", {
                        id: "0fe3d6__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Chyi$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $7Chyi$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Chyi$jsxs)("radialGradient", {
                        id: "0fe3d6__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Chyi$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7Chyi$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Chyi$jsx)("clipPath", {
                        id: "0fe3d6__b",
                        children: /*#__PURE__*/ (0, $7Chyi$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.06,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7Chyi$jsx)("clipPath", {
                id: "0fe3d6__a",
                children: /*#__PURE__*/ (0, $7Chyi$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M40.8 24c0 9.279-7.522 16.8-16.8 16.8-9.28 0-16.8-7.521-16.8-16.8S14.72 7.2 24 7.2c9.278 0 16.8 7.522 16.8 16.8m-19.05-4.05c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h1.5v6.9h-1.5c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h5.4c.662 0 1.2-.537 1.2-1.2s-.538-1.2-1.2-1.2h-1.5v-8.1c0-.663-.538-1.2-1.2-1.2zm2.4-1.5c.828 0 1.5-.671 1.5-1.5 0-.828-.672-1.5-1.5-1.5-.83 0-1.5.672-1.5 1.5 0 .829.67 1.5 1.5 1.5"
                })
            }),
            /*#__PURE__*/ (0, $7Chyi$jsx)("g", {
                clipPath: "url(#0fe3d6__a)",
                children: /*#__PURE__*/ (0, $7Chyi$jsxs)("g", {
                    clipPath: "url(#0fe3d6__b)",
                    children: [
                        /*#__PURE__*/ (0, $7Chyi$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.06,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $7Chyi$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M5.52 6.06H43.2v35.52H5.52z"
                        }),
                        /*#__PURE__*/ (0, $7Chyi$jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#0fe3d6__c)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 10.41 -7.994)"
                        }),
                        /*#__PURE__*/ (0, $7Chyi$jsx)("path", {
                            fill: "url(#0fe3d6__d)",
                            d: "M19.902-.215c-2.894-1.688-6.56-1.698-9.465-.027l-29.05 16.72c-2.845 1.638-2.85 5.504-.009 7.148l28.859 16.697c2.9 1.677 6.567 1.675 9.464-.006l28.8-16.713c2.826-1.64 2.83-5.484.008-7.13z"
                        }),
                        /*#__PURE__*/ (0, $7Chyi$jsx)("path", {
                            fill: "url(#0fe3d6__e)",
                            d: "M10.951 22.924C11.595 9.664 23.521-.595 37.588.012c14.067.608 24.949 11.85 24.305 25.11s-12.57 23.519-26.637 22.912c-14.067-.608-24.949-11.85-24.305-25.11"
                        }),
                        /*#__PURE__*/ (0, $7Chyi$jsx)("path", {
                            fill: "url(#0fe3d6__f)",
                            d: "M-9.747 31.667c5.591-2.249 11.622-2.152 16.847-.185 10.455 3.945 16.538 1.506 20.757-8.327 2.102-4.934 6.205-9.113 11.803-11.364 11.192-4.494 24.113.382 28.874 10.902s-.45 22.68-11.64 27.18c-5.61 2.256-11.652 2.15-16.885.166-10.438-3.924-16.51-1.476-20.724 8.369-2.107 4.922-6.21 9.087-11.8 11.335-11.184 4.498-24.105-.378-28.863-10.892s.447-22.687 11.63-27.184"
                        }),
                        /*#__PURE__*/ (0, $7Chyi$jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#0fe3d6__g)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 24.586 -2.955)"
                        })
                    ]
                })
            })
        ]
    });
const $934b9d6052411f2a$var$ForwardRef = /*#__PURE__*/ (0, $7Chyi$forwardRef)($934b9d6052411f2a$var$SvgComponent);
var $934b9d6052411f2a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($934b9d6052411f2a$var$ForwardRef);


export {$934b9d6052411f2a$export$2e2bcd8739ae039 as default};
