// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_handshake_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $h1rDf$reactjsxruntime = require("react/jsx-runtime");
var $h1rDf$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0df0d2812dd99289$export$2e2bcd8739ae039);




const $0df0d2812dd99289$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsxs)("linearGradient", {
                        id: "624371__d",
                        x1: -5.977,
                        x2: 55.826,
                        y1: 39.303,
                        y2: -0.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsxs)("linearGradient", {
                        id: "624371__e",
                        x1: 45.296,
                        x2: 19.612,
                        y1: 13.02,
                        y2: 42.317,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsxs)("linearGradient", {
                        id: "624371__f",
                        x1: 11.986,
                        x2: 59.166,
                        y1: 44.639,
                        y2: 16.603,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsxs)("radialGradient", {
                        id: "624371__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsxs)("radialGradient", {
                        id: "624371__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("clipPath", {
                        id: "624371__b",
                        children: /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.196,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("clipPath", {
                id: "624371__a",
                children: /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m18.337 11.914 1.697 1.697 1.734 1.734 3.39 3.39c1.098 1.098 2.877 1.098 3.975 0s1.098-2.877 0-3.975l-3.39-3.39-1.734-1.734-.564-.564c-.402-.396-.855-.702-1.338-.927l-.046-.023q-.08-.04-.164-.073c-.19-.078-.384-.138-.58-.192-1.601-.426-3.377-.015-4.634 1.239l-9.13 9.129c-1.466 1.467-1.778 3.645-.95 5.424L16.64 13.61zm12.98-2.794 9.128 9.129v.006c1.875 1.875 1.875 4.914 0 6.789L30.281 35.208l-4.284 4.284c-1.098 1.098-2.877 1.098-3.975-.003h-.003c-1.098-1.098-1.098-2.877 0-3.975-1.098 1.098-2.877 1.098-3.975-.003h-.003c-1.089-1.09-1.095-2.847-.024-3.948-1.1 1.07-2.859 1.062-3.948-.03h-.003c-1.098-1.098-1.098-2.88 0-3.978h-.003c-1.098 1.098-2.877 1.098-3.975-.003h-.003q-.13-.132-.237-.273c-.852-1.104-.774-2.694.237-3.705l4.488-4.491 3.768-3.768 1.734 1.734 3.39 3.39c.984.984 2.292 1.527 3.684 1.527s2.7-.543 3.684-1.527c2.031-2.034 2.031-5.34 0-7.371l-4.91-4.911c1.77-.813 3.932-.498 5.393.963"
                })
            }),
            /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("g", {
                clipPath: "url(#624371__a)",
                children: /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#624371__b)",
                    children: [
                        /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.196,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.898 6.196h39.3v37.047h-39.3z"
                        }),
                        /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#624371__c)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 9 -8.462)"
                        }),
                        /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("path", {
                            fill: "url(#624371__d)",
                            d: "M18.9-.348c-3.018-1.76-6.842-1.771-9.872-.028l-30.3 17.44c-2.967 1.708-2.972 5.74-.008 7.454l30.1 17.415c3.024 1.75 6.848 1.748 9.87-.006l30.038-17.431c2.948-1.711 2.953-5.72.009-7.438z"
                        }),
                        /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("path", {
                            fill: "url(#624371__e)",
                            d: "M9.561 23.785C10.233 9.955 22.671-.745 37.343-.11c14.672.633 26.021 12.358 25.35 26.19-.672 13.83-13.11 24.529-27.782 23.896-14.672-.634-26.022-12.359-25.35-26.19"
                        }),
                        /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("path", {
                            fill: "url(#624371__f)",
                            d: "M-12.024 32.905C-6.192 30.56.098 30.66 5.548 32.712c10.904 4.115 17.249 1.57 21.65-8.685 2.192-5.146 6.471-9.505 12.31-11.853 11.673-4.687 25.15.399 30.115 11.37 4.965 10.973-.47 23.657-12.14 28.35-5.852 2.353-12.154 2.243-17.611.173-10.887-4.093-17.22-1.54-21.615 8.729-2.198 5.134-6.476 9.477-12.308 11.823-11.664 4.69-25.141-.395-30.104-11.361-4.962-10.966.467-23.662 12.131-28.353"
                        }),
                        /*#__PURE__*/ (0, $h1rDf$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#624371__g)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 23.785 -3.206)"
                        })
                    ]
                })
            })
        ]
    });
const $0df0d2812dd99289$var$ForwardRef = /*#__PURE__*/ (0, $h1rDf$react.forwardRef)($0df0d2812dd99289$var$SvgComponent);
var $0df0d2812dd99289$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0df0d2812dd99289$var$ForwardRef);
