// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_cursor_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7wycW$jsxs, jsx as $7wycW$jsx} from "react/jsx-runtime";
import {forwardRef as $7wycW$forwardRef} from "react";





const $36d0a2352bb7e4d3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7wycW$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7wycW$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7wycW$jsxs)("linearGradient", {
                        id: "bc84c4__d",
                        x1: -7.898,
                        x2: 110.611,
                        y1: 77.404,
                        y2: 0.92,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7wycW$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $7wycW$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7wycW$jsxs)("linearGradient", {
                        id: "bc84c4__e",
                        x1: 90.423,
                        x2: 41.171,
                        y1: 27.006,
                        y2: 83.185,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7wycW$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7wycW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7wycW$jsxs)("linearGradient", {
                        id: "bc84c4__f",
                        x1: 26.546,
                        x2: 117.015,
                        y1: 87.636,
                        y2: 33.875,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7wycW$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7wycW$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7wycW$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $7wycW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7wycW$jsxs)("radialGradient", {
                        id: "bc84c4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7wycW$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $7wycW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7wycW$jsxs)("radialGradient", {
                        id: "bc84c4__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 30.9063 -32.7857 0 32.786 30.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7wycW$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7wycW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7wycW$jsx)("clipPath", {
                        id: "bc84c4__b",
                        children: /*#__PURE__*/ (0, $7wycW$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7wycW$jsx)("clipPath", {
                id: "bc84c4__a",
                children: /*#__PURE__*/ (0, $7wycW$jsx)("path", {
                    fill: "#fff",
                    d: "m16.775 25.797 21.391 51.95c2.72 6.606 12.015 6.774 14.97.27l6.697-14.732c.817-1.796 2.256-3.235 4.051-4.051l14.733-6.697c6.503-2.956 6.336-12.25-.27-14.97l-51.95-21.392c-6.058-2.494-12.117 3.564-9.622 9.622"
                })
            }),
            /*#__PURE__*/ (0, $7wycW$jsx)("g", {
                clipPath: "url(#bc84c4__a)",
                children: /*#__PURE__*/ (0, $7wycW$jsxs)("g", {
                    clipPath: "url(#bc84c4__b)",
                    children: [
                        /*#__PURE__*/ (0, $7wycW$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $7wycW$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M11.04 13.92H86.4v71.04H11.04z"
                        }),
                        /*#__PURE__*/ (0, $7wycW$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#bc84c4__c)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 20.816 -14.188)"
                        }),
                        /*#__PURE__*/ (0, $7wycW$jsx)("path", {
                            fill: "url(#bc84c4__d)",
                            d: "M39.804 1.37c-5.787-3.375-13.12-3.396-18.93-.053l-58.1 33.442c-5.69 3.275-5.7 11.006-.018 14.293l57.718 33.394c5.8 3.355 13.133 3.351 18.928-.012l57.599-33.425c5.652-3.28 5.663-10.968.017-14.261z"
                        }),
                        /*#__PURE__*/ (0, $7wycW$jsx)("path", {
                            fill: "url(#bc84c4__e)",
                            d: "M21.899 47.648C23.187 21.127 47.039.611 75.172 1.826s49.897 23.698 48.609 50.22c-1.288 26.52-25.139 47.036-53.273 45.822S20.61 74.17 21.898 47.648"
                        }),
                        /*#__PURE__*/ (0, $7wycW$jsx)("path", {
                            fill: "url(#bc84c4__f)",
                            d: "M-19.495 65.135C-8.31 60.637 3.75 60.83 14.2 64.764c20.91 7.891 33.077 3.012 41.515-16.653 4.203-9.869 12.41-18.226 23.606-22.729 22.384-8.988 48.226.765 57.748 21.804s-.901 45.361-23.28 54.361c-11.221 4.513-23.305 4.301-33.77.332-20.876-7.849-33.02-2.952-41.447 16.738-4.215 9.844-12.42 18.173-23.602 22.671-22.367 8.995-48.21-.758-57.725-21.785s.895-45.373 23.261-54.368"
                        }),
                        /*#__PURE__*/ (0, $7wycW$jsx)("ellipse", {
                            cx: 32.786,
                            cy: 30.906,
                            fill: "url(#bc84c4__g)",
                            rx: 32.786,
                            ry: 30.906,
                            transform: "matrix(1 -.00132 .00148 1 49.176 -4.108)"
                        })
                    ]
                })
            })
        ]
    });
const $36d0a2352bb7e4d3$var$ForwardRef = /*#__PURE__*/ (0, $7wycW$forwardRef)($36d0a2352bb7e4d3$var$SvgComponent);
var $36d0a2352bb7e4d3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($36d0a2352bb7e4d3$var$ForwardRef);


export {$36d0a2352bb7e4d3$export$2e2bcd8739ae039 as default};
