// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_badgeVerified_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $9BLaE$reactjsxruntime = require("react/jsx-runtime");
var $9BLaE$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ad093adbda6a1331$export$2e2bcd8739ae039);




const $ad093adbda6a1331$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "03bd4f__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "03bd4f__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.684,
                        y2: 89.681,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "03bd4f__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.618,
                        y2: 33.126,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsxs)("radialGradient", {
                        id: "03bd4f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.278 29.003)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsxs)("radialGradient", {
                        id: "03bd4f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.53 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("clipPath", {
                        id: "03bd4f__b",
                        children: /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: 0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("clipPath", {
                id: "03bd4f__a",
                children: /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M75.766 38.628c3.263 1.851 5.475 5.355 5.475 9.376h.009c0 4.038-2.23 7.55-5.519 9.401-1.291.724-1.894 2.196-1.498 3.625.44 1.558.534 3.263.181 5.028-.835 4.21-4.201 7.585-8.412 8.42-1.773.353-3.495.258-5.062-.19-1.403-.395-2.867.199-3.599 1.464-1.868 3.229-5.346 5.407-9.341 5.407s-7.473-2.178-9.341-5.407c-.732-1.265-2.187-1.86-3.6-1.463-1.566.447-3.288.542-5.061.189-4.21-.835-7.577-4.21-8.412-8.42-.353-1.765-.258-3.47.18-5.028.397-1.43-.206-2.901-1.497-3.625-3.29-1.85-5.519-5.38-5.519-9.401 0-4.02 2.204-7.525 5.476-9.376 1.248-.706 1.825-2.178 1.48-3.53-.344-1.352-.439-2.798-.223-4.296.68-4.658 4.442-8.403 9.1-9.074 1.524-.224 2.927-.138 4.356.25 1.43.387 2.893-.233 3.608-1.525 1.834-3.34 5.38-5.613 9.453-5.613s7.62 2.273 9.453 5.613c.715 1.292 2.247 1.886 3.608 1.524 1.36-.361 2.824-.473 4.348-.25 4.666.672 8.42 4.417 9.1 9.075.215 1.498.138 2.91-.224 4.296s.232 2.824 1.48 3.53m-11.981 2.01c.816-.744.875-2.009.13-2.825-.743-.816-2.008-.875-2.825-.131l-3.177 2.896-3.112 2.836-.053.049-.054.048-.053.05-.054.048-.214.196-.434.395-.11.1-.11.1-.109.1-.055.05-.055.05-.055.05-.056.05-.055.051-.055.05-.222.203-.223.203-.111.102-.112.101-.112.102-.056.051-.056.052-.112.102-.113.102-.45.41-.056.052-.056.05-.056.052-.057.051-.225.206-1.79 1.632-.878.8-.054.048-.054.05-.108.098-.107.098-.107.098-.107.097-.053.048-.053.049-.053.048-.053.048-.053.048-.053.048-.052.048-.052.048-.053.048-.052.047-.104.095-.104.094-.051.048-.052.047-.102.093-.102.093-.102.092-.201.184-.1.09-.099.091-.05.045-.049.045-.049.045-.049.044-.048.045-.049.044-.048.044-.048.044-.38.346-.369.336-.045.041-.045.041-.045.04-.044.042-.177.16-.087.08-.043.039-.043.04-.17.154-1.618 1.475-8.13-7.581c-.808-.754-2.074-.71-2.827.098s-.71 2.074.099 2.827l9.478 8.84c.762.71 1.941.717 2.711.015l-1.347-1.478 1.347 1.478h.001l.003-.003.002-.002.002-.002.006-.006.017-.015.005-.005.006-.005.013-.012m19.342-17.629-3.177 2.896Zm-3.177 2.896-.043.04Zm-.043.04-.044.039Zm-.044.039-.043.04Zm-.043.04-.044.04Zm-.044.04-.044.04Zm-.044.04-.044.04Zm-.044.04-.089.08Zm-.089.08-.363.332Zm-.363.332-.047.042Zm-.047.042-.046.042Zm-.046.042-.093.085Zm-.093.085-.095.086Zm-.095.086-.047.043Zm-.047.043-.047.044Zm-.047.044-.048.043Zm-.048.043-.048.044Zm-.048.044-.048.044Zm-.048.044-.048.044Zm-.048.044-.048.044Zm-.048.044-.05.044Zm-.05.044-.097.089Zm-.097.089-.396.36Zm-.396.36-.05.047Zm-.05.047-.05.045Zm-.05.045-.05.046Zm-.05.046-.051.047Zm-.051.047-.102.092Zm-.102.092-.051.047Zm-.051.047-.051.046Zm-.051.046-.206.188Zm-.206.188-.208.19Zm-.208.19-.105.095Zm-.105.095-.053.048Zm-.053.048-.052.048Zm-.052.048-.212.193Zm-.212.193-.862.785Zm-.862.785-.439.4Zm-.439.4-.055.05Zm-.055.05-.055.051Zm-.055.051-.055.05Zm-.055.05-.056.05Zm-.056.05-.11.102Zm-.11.102-.112.101Zm-.112.101-.222.203Zm-.222.203-.224.204Zm-.224.204-.224.204Zm-.224.204-.112.102Zm-.112.102-.056.051Zm-.056.051-.056.052Zm-.056.052-.056.05Zm-.056.05-.056.052Zm-.056.052-.057.051Zm-.057.051-.056.051Zm-.056.051-.056.052Zm-.056.052-.056.05Zm-.056.05-.113.103Zm-.113.103-.056.052Zm-.056.052-.057.051Zm-.057.051-.112.103Zm-.112.103-.226.205Zm-.226.205-1.79 1.633Zm-1.79 1.633-.056.05Zm-.056.05-.055.05Zm-.055.05-.055.05Zm-.055.05-.055.05Zm-.055.05-.055.05Zm-.055.05-.055.051Zm-.055.051-.11.1Zm-.11.1-.219.2Zm-.219.2-.054.049Zm-.054.049-.055.05Zm-.055.05-.108.099Zm-.108.099-.054.049Zm-.054.049-.054.05Zm-.054.05-.054.048Zm-.054.048-.054.05Zm-.054.05-.054.048Zm-.054.048-.054.05Zm-.054.05-.053.048Zm-.053.048-.054.049Zm-.054.049-.106.097Zm-.106.097-.107.097Zm-.107.097-.211.193Zm-.211.193-.417.38Zm-.417.38-.052.047Zm-.052.047-.051.047Zm-.051.047-.103.093Zm-.103.093-.05.047Zm-.05.047-.052.046Zm-.052.046-.05.047Zm-.05.047-.051.046Zm-.051.046-.201.183Zm-.201.183-.1.091Zm-.1.091-.05.045Zm-.05.045-.05.046Zm-.05.046-.196.179Zm-.196.179-.194.176Zm-.194.176-.38.346Zm-.38.346-.368.336Zm-.368.336-.18.164Zm-.18.164-.088.08Zm-.088.08-.044.04Zm-.044.04-.044.04Zm-.044.04-.087.08Zm-.087.08-.043.04Zm-.043.04-.043.038Zm-.043.038-.043.04Zm-.043.04-.042.038Zm-.042.038-.085.077Zm-.085.077-.083.076Zm-.083.076-.083.076Zm-.083.076-.041.037Zm-.041.037-.04.037Zm-.04.037-.041.037Zm-.041.037-.04.037Zm-.04.037-.04.036Zm-.04.036-.04.036Zm-.04.036-.04.037Zm-.04.037-.04.035Zm-.04.035-.154.142Zm-.154.142-.298.271Zm-.298.271-.143.13Zm-.143.13-.07.064Zm-.07.064-.034.031Zm-.034.031-.035.032Zm-.035.032-.067.061Zm-.067.061-.067.06Zm-.067.06-.032.03Zm-.032.03-.033.03Zm-.033.03-.064.059Zm-.064.059-.032.029Zm-.032.029-.031.028Zm-.031.028-.062.057Zm-.062.057-.12.11Zm-.12.11-.433.393Zm-.433.393-.348.318Zm-.348.318-.257.234Zm-.257.234-.05.045Zm-.05.045-.01.01Zm-.01.01-.011.01Zm-.011.01-.021.019Zm-.021.019-.037.033Zm-.037.033-.002.002Z"
                })
            }),
            /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("g", {
                clipPath: "url(#03bd4f__a)",
                children: /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#03bd4f__b)",
                    children: [
                        /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: 0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 .001h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.724,
                            r: 41.765,
                            fill: "url(#03bd4f__c)",
                            transform: "rotate(-.08 54.28 3.724)"
                        }),
                        /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("path", {
                            fill: "url(#03bd4f__d)",
                            d: "M36.643-16.957c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.426-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.432 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("path", {
                            fill: "url(#03bd4f__e)",
                            d: "M13.837 45.58C15.477 9.74 45.86-17.983 81.7-16.343s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922S12.196 81.419 13.837 45.58"
                        }),
                        /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("path", {
                            fill: "url(#03bd4f__f)",
                            d: "M-38.895 69.21c14.246-6.077 29.61-5.817 42.922-.5 26.636 10.663 42.135 4.07 52.885-22.505 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.315 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $9BLaE$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.345,
                            r: 41.765,
                            fill: "url(#03bd4f__g)",
                            transform: "rotate(-.08 90.406 17.345)"
                        })
                    ]
                })
            })
        ]
    });
const $ad093adbda6a1331$var$ForwardRef = /*#__PURE__*/ (0, $9BLaE$react.forwardRef)($ad093adbda6a1331$var$SvgComponent);
var $ad093adbda6a1331$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ad093adbda6a1331$var$ForwardRef);
