// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_confetti_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4KNHr$reactjsxruntime = require("react/jsx-runtime");
var $4KNHr$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $47137d1ab6548785$export$2e2bcd8739ae039);




const $47137d1ab6548785$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0639ef__d",
                        x1: -8.603,
                        x2: 58.43,
                        y1: 41.325,
                        y2: -1.892,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0639ef__e",
                        x1: 46.977,
                        x2: 19.103,
                        y1: 12.805,
                        y2: 44.567,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0639ef__f",
                        x1: 10.87,
                        x2: 62.039,
                        y1: 47.115,
                        y2: 16.74,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsxs)("radialGradient", {
                        id: "0639ef__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.4892 -18.5333 0 18.533 17.49)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsxs)("radialGradient", {
                        id: "0639ef__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.4892 -18.5333 0 18.533 17.49)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("clipPath", {
                        id: "0639ef__b",
                        children: /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("rect", {
                            width: 42.6,
                            height: 40.2,
                            x: 2.102,
                            y: 5.4,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("clipPath", {
                id: "0639ef__a",
                children: /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M32.978 9.648c.858-.858.858-2.244 0-3.102s-2.245-.858-3.103 0-.858 2.244 0 3.102 2.245.858 3.102 0M19.44 21.636c-.519.414-.603 1.17-.189 1.686.237.297.585.45.94.45.263 0 .527-.087.746-.264 5.766-4.605 5.502-9.438 5.49-9.642-.045-.657-.609-1.146-1.266-1.11-.657.039-1.16.606-1.128 1.263.006.156.13 3.843-4.593 7.614zm10.2 8.96c-.308-.584-.083-1.31.502-1.62h.003c.585-.308 1.31-.083 1.62.502 1.719 3.258 1.167 4.854.402 5.619-.25.249-.585.474-1.035.606l-20.775 7.38c-.402.144-.813.213-1.218.213-.933 0-1.842-.37-2.532-1.06-.99-.99-1.314-2.426-.846-3.75l7.296-20.558q.062-.234.165-.45c.135-.336.32-.6.525-.804.77-.771 2.382-1.32 5.679.435.585.312.807 1.038.495 1.623s-1.038.807-1.623.495c-1.857-.987-2.71-.927-2.856-.852v.006l-.054.153c-.12.912 1.26 4.46 5.859 9.06 3.486 3.486 6.366 5.12 7.962 5.652.618.126 1.056.156 1.278.126.057-.165.087-1.005-.846-2.775m-6.393-3.767c-.309 0-.615-.117-.849-.351-.468-.471-.468-1.23 0-1.698l4.683-4.683c.471-.468 1.23-.468 1.698 0 .468.47.468 1.23 0 1.698l-4.683 4.683c-.234.234-.54.35-.849.35m2.115 1.11c4.605-5.766 9.438-5.502 9.642-5.49l-.003.003c.663.045 1.161.618 1.116 1.278-.045.657-.612 1.158-1.269 1.116-.177-.006-3.852-.111-7.61 4.593-.238.297-.589.45-.94.45-.26 0-.525-.087-.747-.264-.519-.414-.603-1.167-.189-1.686m12.15-.288c.858-.858 2.244-.858 3.102 0s.858 2.244 0 3.102-2.244.858-3.102 0-.858-2.244 0-3.102M16.907 8.458l-.576.422c-.535.392-.651 1.142-.26 1.677l2.344 3.203c.392.535 1.142.651 1.677.26l.576-.422c.535-.391.651-1.142.26-1.677l-2.344-3.203c-.392-.535-1.142-.651-1.677-.26m15.263 7.997 3.944-.446c.658-.075 1.253.399 1.327 1.057l.08.71c.075.658-.398 1.252-1.057 1.327l-3.944.446c-.658.075-1.253-.398-1.327-1.057l-.08-.71c-.075-.658.398-1.252 1.057-1.327"
                })
            }),
            /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("g", {
                clipPath: "url(#0639ef__a)",
                children: /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#0639ef__b)",
                    children: [
                        /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("rect", {
                            width: 42.6,
                            height: 40.2,
                            x: 2.102,
                            y: 5.4,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M2.102 5.4h42.6v40.2h-42.6z"
                        }),
                        /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("ellipse", {
                            cx: 18.533,
                            cy: 17.489,
                            fill: "url(#0639ef__c)",
                            rx: 18.533,
                            ry: 17.489,
                            transform: "matrix(1 -.00132 .00148 1 7.629 -10.505)"
                        }),
                        /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("path", {
                            fill: "url(#0639ef__d)",
                            d: "M18.363-1.7c-3.272-1.911-7.417-1.923-10.701-.03l-32.844 18.923c-3.217 1.853-3.222 6.228-.01 8.088L7.435 44.178c3.28 1.899 7.424 1.897 10.7-.007l32.56-18.914c3.195-1.857 3.201-6.207.01-8.07z"
                        }),
                        /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("path", {
                            fill: "url(#0639ef__e)",
                            d: "M8.241 24.486C8.97 9.478 22.452-2.13 38.356-1.444s28.206 13.41 27.478 28.418-14.211 26.617-30.115 25.93S7.513 39.494 8.241 24.486"
                        }),
                        /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("path", {
                            fill: "url(#0639ef__f)",
                            d: "M-15.156 34.383c6.322-2.546 13.14-2.436 19.047-.21 11.82 4.465 18.697 1.704 23.468-9.424 2.376-5.584 7.015-10.314 13.344-12.862 12.653-5.086 27.262.433 32.644 12.339 5.382 11.905-.509 25.669-13.16 30.761-6.343 2.554-13.174 2.434-19.09.188-11.8-4.441-18.665-1.67-23.43 9.472-2.381 5.57-7.02 10.284-13.341 12.829-12.644 5.09-27.252-.43-32.631-12.328s.506-25.676 13.15-30.765"
                        }),
                        /*#__PURE__*/ (0, $4KNHr$reactjsxruntime.jsx)("ellipse", {
                            cx: 18.533,
                            cy: 17.489,
                            fill: "url(#0639ef__g)",
                            rx: 18.533,
                            ry: 17.489,
                            transform: "matrix(1 -.00132 .00148 1 23.66 -4.802)"
                        })
                    ]
                })
            })
        ]
    });
const $47137d1ab6548785$var$ForwardRef = /*#__PURE__*/ (0, $4KNHr$react.forwardRef)($47137d1ab6548785$var$SvgComponent);
var $47137d1ab6548785$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($47137d1ab6548785$var$ForwardRef);
