// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_tablet_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $8yqKP$jsxs, jsx as $8yqKP$jsx} from "react/jsx-runtime";
import {forwardRef as $8yqKP$forwardRef} from "react";





const $9efff894808945de$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8yqKP$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8yqKP$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8yqKP$jsxs)("linearGradient", {
                        id: "77745d__d",
                        x1: -7.554,
                        x2: 58.494,
                        y1: 40.305,
                        y2: -2.321,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8yqKP$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8yqKP$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8yqKP$jsxs)("linearGradient", {
                        id: "77745d__e",
                        x1: 47.243,
                        x2: 19.794,
                        y1: 12.217,
                        y2: 43.527,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8yqKP$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8yqKP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8yqKP$jsxs)("linearGradient", {
                        id: "77745d__f",
                        x1: 11.644,
                        x2: 62.065,
                        y1: 46.008,
                        y2: 16.046,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8yqKP$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8yqKP$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8yqKP$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8yqKP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8yqKP$jsxs)("radialGradient", {
                        id: "77745d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.2248 -18.2723 0 18.272 17.225)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8yqKP$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8yqKP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8yqKP$jsxs)("radialGradient", {
                        id: "77745d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.2248 -18.2723 0 18.272 17.225)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8yqKP$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8yqKP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8yqKP$jsx)("clipPath", {
                        id: "77745d__b",
                        children: /*#__PURE__*/ (0, $8yqKP$jsx)("rect", {
                            width: 42,
                            height: 39.592,
                            x: 3,
                            y: 4.924,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8yqKP$jsx)("clipPath", {
                id: "77745d__a",
                children: /*#__PURE__*/ (0, $8yqKP$jsx)("path", {
                    fill: "#fff",
                    d: "M38.4 8.4H9.6c-2.651 0-4.8 2.148-4.8 4.8v21.6c0 2.652 2.149 4.8 4.8 4.8h28.8c2.653 0 4.8-2.148 4.8-4.8V13.2c0-2.652-2.147-4.8-4.8-4.8M10.2 25.2c-.662 0-1.2-.537-1.2-1.2s.535-1.2 1.198-1.2h.003c.663 0 1.2.537 1.2 1.2s-.537 1.2-1.2 1.2"
                })
            }),
            /*#__PURE__*/ (0, $8yqKP$jsx)("g", {
                clipPath: "url(#77745d__a)",
                children: /*#__PURE__*/ (0, $8yqKP$jsxs)("g", {
                    clipPath: "url(#77745d__b)",
                    children: [
                        /*#__PURE__*/ (0, $8yqKP$jsx)("rect", {
                            width: 42,
                            height: 39.592,
                            x: 3,
                            y: 4.924,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $8yqKP$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3 4.924h42v39.592H3z"
                        }),
                        /*#__PURE__*/ (0, $8yqKP$jsx)("ellipse", {
                            cx: 18.272,
                            cy: 17.225,
                            fill: "url(#77745d__c)",
                            rx: 18.272,
                            ry: 17.225,
                            transform: "matrix(1 -.00132 .00148 1 8.45 -10.741)"
                        }),
                        /*#__PURE__*/ (0, $8yqKP$jsx)("path", {
                            fill: "url(#77745d__d)",
                            d: "M19.032-2.07c-3.226-1.882-7.313-1.893-10.55-.03L-23.9 16.538c-3.17 1.825-3.176 6.134-.01 7.966L8.259 43.115c3.233 1.87 7.32 1.868 10.55-.007l32.1-18.628c3.151-1.828 3.157-6.113.01-7.948z"
                        }),
                        /*#__PURE__*/ (0, $8yqKP$jsx)("path", {
                            fill: "url(#77745d__e)",
                            d: "M9.053 23.722C9.771 8.94 23.064-2.493 38.743-1.816s27.81 13.207 27.092 27.988-14.01 26.215-29.69 25.538S8.334 38.503 9.052 23.722"
                        }),
                        /*#__PURE__*/ (0, $8yqKP$jsx)("path", {
                            fill: "url(#77745d__f)",
                            d: "M-14.015 33.468c6.233-2.507 12.955-2.4 18.778-.207 11.654 4.398 18.434 1.679 23.138-9.282 2.342-5.5 6.916-10.157 13.156-12.666 12.475-5.01 26.877.426 32.184 12.151 5.306 11.726-.502 25.281-12.974 30.297-6.254 2.515-12.989 2.397-18.822.185-11.634-4.374-18.402-1.645-23.1 9.328-2.348 5.487-6.92 10.129-13.153 12.635-12.466 5.013-26.868-.422-32.171-12.14-5.304-11.72.498-25.288 12.964-30.301"
                        }),
                        /*#__PURE__*/ (0, $8yqKP$jsx)("ellipse", {
                            cx: 18.272,
                            cy: 17.225,
                            fill: "url(#77745d__g)",
                            rx: 18.272,
                            ry: 17.225,
                            transform: "matrix(1 -.00132 .00148 1 24.254 -5.124)"
                        })
                    ]
                })
            })
        ]
    });
const $9efff894808945de$var$ForwardRef = /*#__PURE__*/ (0, $8yqKP$forwardRef)($9efff894808945de$var$SvgComponent);
var $9efff894808945de$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9efff894808945de$var$ForwardRef);


export {$9efff894808945de$export$2e2bcd8739ae039 as default};
