// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_mailClose_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $lRc16$reactjsxruntime = require("react/jsx-runtime");
var $lRc16$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $dec09854cae2b7af$export$2e2bcd8739ae039);




const $dec09854cae2b7af$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsxs)("linearGradient", {
                        id: "86b8d2__d",
                        x1: -10.37,
                        x2: 112.343,
                        y1: 77.137,
                        y2: -2.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsxs)("linearGradient", {
                        id: "86b8d2__e",
                        x1: 91.443,
                        x2: 40.445,
                        y1: 24.95,
                        y2: 83.122,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsxs)("linearGradient", {
                        id: "86b8d2__f",
                        x1: 25.3,
                        x2: 118.977,
                        y1: 87.732,
                        y2: 32.064,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsxs)("radialGradient", {
                        id: "86b8d2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.0026 -33.9487 0 33.949 32.003)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsxs)("radialGradient", {
                        id: "86b8d2__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.0026 -33.9487 0 33.949 32.003)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("clipPath", {
                        id: "86b8d2__b",
                        children: /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("rect", {
                            width: 78.033,
                            height: 73.56,
                            x: 9.242,
                            y: 11.4,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("clipPath", {
                id: "86b8d2__a",
                children: /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M20.308 20.4h55.384c2.215 0 4.227.96 5.717 2.524L49.536 41.52c-1.118.652-2.493.64-3.6-.034l-31.02-18.885c1.451-1.372 3.334-2.2 5.392-2.2m-7.887 6.301c-.273.912-.421 1.886-.421 2.9v36.8c0 5.08 3.72 9.2 8.308 9.2h55.384c4.589 0 8.308-4.12 8.308-9.2V29.6c0-.851-.104-1.675-.3-2.457L51.956 45.666c-2.641 1.54-5.903 1.51-8.515-.08z"
                })
            }),
            /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("g", {
                clipPath: "url(#86b8d2__a)",
                children: /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#86b8d2__b)",
                    children: [
                        /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("rect", {
                            width: 78.033,
                            height: 73.56,
                            x: 9.242,
                            y: 11.4,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M9.242 11.4h78.033v73.56H9.242z"
                        }),
                        /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("ellipse", {
                            cx: 33.949,
                            cy: 32.003,
                            fill: "url(#86b8d2__c)",
                            rx: 33.949,
                            ry: 32.003,
                            transform: "matrix(1 -.00132 .00148 1 19.367 -17.704)"
                        }),
                        /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("path", {
                            fill: "url(#86b8d2__d)",
                            d: "M39.024-1.594c-5.993-3.495-13.585-3.517-19.6-.055L-40.74 32.98c-5.892 3.391-5.902 11.396-.018 14.8l59.765 34.579c6.006 3.474 13.6 3.47 19.6-.013l59.642-34.61c5.853-3.398 5.863-11.358.018-14.767z"
                        }),
                        /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("path", {
                            fill: "url(#86b8d2__e)",
                            d: "M20.489 46.325C21.822 18.863 46.519-2.38 75.652-1.123s51.667 24.539 50.333 52.001c-1.333 27.462-26.03 48.705-55.163 47.448S19.155 73.787 20.489 46.325"
                        }),
                        /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("path", {
                            fill: "url(#86b8d2__f)",
                            d: "M-22.374 64.432c11.58-4.657 24.069-4.457 34.889-.384 21.651 8.171 34.25 3.12 42.987-17.244 4.353-10.218 12.85-18.872 24.443-23.534 23.179-9.307 49.938.791 59.797 22.577 9.859 21.785-.932 46.97-24.106 56.289-11.619 4.673-24.132 4.454-34.968.344-21.616-8.127-34.19-3.056-42.917 17.332-4.364 10.193-12.86 18.818-24.44 23.475-23.16 9.314-49.918-.785-59.772-22.558s.927-46.983 24.087-56.297"
                        }),
                        /*#__PURE__*/ (0, $lRc16$reactjsxruntime.jsx)("ellipse", {
                            cx: 33.949,
                            cy: 32.003,
                            fill: "url(#86b8d2__g)",
                            rx: 33.949,
                            ry: 32.003,
                            transform: "matrix(1 -.00132 .00148 1 48.73 -7.268)"
                        })
                    ]
                })
            })
        ]
    });
const $dec09854cae2b7af$var$ForwardRef = /*#__PURE__*/ (0, $lRc16$react.forwardRef)($dec09854cae2b7af$var$SvgComponent);
var $dec09854cae2b7af$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($dec09854cae2b7af$var$ForwardRef);
