// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_megaphonePromote2_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gcbLH$reactjsxruntime = require("react/jsx-runtime");
var $gcbLH$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a661435c437ad6d9$export$2e2bcd8739ae039);




const $a661435c437ad6d9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsxs)("linearGradient", {
                        id: "27b1b9__d",
                        x1: -5.074,
                        x2: 56.728,
                        y1: 39.303,
                        y2: -0.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsxs)("linearGradient", {
                        id: "27b1b9__e",
                        x1: 46.199,
                        x2: 20.514,
                        y1: 13.02,
                        y2: 42.317,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsxs)("linearGradient", {
                        id: "27b1b9__f",
                        x1: 12.889,
                        x2: 60.068,
                        y1: 44.639,
                        y2: 16.603,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsxs)("radialGradient", {
                        id: "27b1b9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsxs)("radialGradient", {
                        id: "27b1b9__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("clipPath", {
                        id: "27b1b9__b",
                        children: /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.801,
                            y: 6.196,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("clipPath", {
                id: "27b1b9__a",
                children: /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m20.771 14.406 11.568-3.513.003-.003c1.725-.525 3.465.765 3.465 2.568v13.158c0 1.803-1.74 3.093-3.465 2.568l-11.568-3.52h-.012l3.132 7.276c.672 1.554-.042 3.357-1.593 4.029-1.554.672-3.357-.042-4.029-1.593l-4.22-9.708h-1.597c-3.108 0-5.63-2.523-5.63-5.631s2.52-5.631 5.63-5.631zm20.154 10.302 1.725.546c.63.2.981.873.78 1.506-.162.51-.633.837-1.143.837q-.181 0-.363-.057l-1.725-.546c-.63-.201-.98-.873-.78-1.506.201-.63.873-.981 1.506-.78m-.363-9.444q.181 0 .363-.057l1.725-.546c.633-.201.981-.876.78-1.506-.2-.633-.873-.981-1.506-.78l-1.725.546c-.633.2-.98.876-.78 1.506.162.51.633.837 1.143.837m1.806 5.895h-1.806c-.662 0-1.2-.537-1.2-1.2s.538-1.2 1.2-1.2h1.806c.663 0 1.2.537 1.2 1.2s-.536 1.2-1.2 1.2"
                })
            }),
            /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("g", {
                clipPath: "url(#27b1b9__a)",
                children: /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#27b1b9__b)",
                    children: [
                        /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.801,
                            y: 6.196,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M4.8 6.196h39.3v37.047H4.8z"
                        }),
                        /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#27b1b9__c)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 9.902 -8.462)"
                        }),
                        /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("path", {
                            fill: "url(#27b1b9__d)",
                            d: "M19.802-.348c-3.018-1.76-6.842-1.771-9.871-.028l-30.3 17.44c-2.968 1.708-2.972 5.74-.01 7.454l30.1 17.415c3.025 1.75 6.85 1.748 9.872-.006L49.63 24.496c2.948-1.711 2.953-5.72.01-7.438z"
                        }),
                        /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("path", {
                            fill: "url(#27b1b9__e)",
                            d: "M10.464 23.785C11.135 9.955 23.574-.745 38.246-.11c14.671.633 26.02 12.358 25.35 26.19-.673 13.83-13.111 24.529-27.783 23.896-14.672-.634-26.021-12.359-25.35-26.19"
                        }),
                        /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("path", {
                            fill: "url(#27b1b9__f)",
                            d: "M-11.121 32.905C-5.29 30.56 1 30.66 6.449 32.712c10.905 4.115 17.25 1.57 21.65-8.685 2.193-5.146 6.473-9.505 12.311-11.853 11.674-4.687 25.15.399 30.116 11.37 4.965 10.973-.47 23.657-12.14 28.35-5.852 2.353-12.154 2.243-17.612.173-10.887-4.093-17.22-1.54-21.615 8.729-2.197 5.134-6.476 9.477-12.308 11.823-11.664 4.69-25.14-.395-30.103-11.361s.466-23.662 12.13-28.353"
                        }),
                        /*#__PURE__*/ (0, $gcbLH$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#27b1b9__g)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 24.688 -3.206)"
                        })
                    ]
                })
            })
        ]
    });
const $a661435c437ad6d9$var$ForwardRef = /*#__PURE__*/ (0, $gcbLH$react.forwardRef)($a661435c437ad6d9$var$SvgComponent);
var $a661435c437ad6d9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a661435c437ad6d9$var$ForwardRef);
