// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileShared_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $5xBON$jsxs, jsx as $5xBON$jsx} from "react/jsx-runtime";
import {forwardRef as $5xBON$forwardRef} from "react";





const $965d2b59a413c8f4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5xBON$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5xBON$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5xBON$jsxs)("linearGradient", {
                        id: "0b906f__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5xBON$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $5xBON$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5xBON$jsxs)("linearGradient", {
                        id: "0b906f__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5xBON$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5xBON$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5xBON$jsxs)("linearGradient", {
                        id: "0b906f__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5xBON$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5xBON$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5xBON$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5xBON$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5xBON$jsxs)("radialGradient", {
                        id: "0b906f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5xBON$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $5xBON$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5xBON$jsxs)("radialGradient", {
                        id: "0b906f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5xBON$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5xBON$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5xBON$jsx)("clipPath", {
                        id: "0b906f__b",
                        children: /*#__PURE__*/ (0, $5xBON$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5xBON$jsx)("clipPath", {
                id: "0b906f__a",
                children: /*#__PURE__*/ (0, $5xBON$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "m87.515 31.515 32.97 32.97c2.251 2.25 3.515 5.303 3.515 8.486V120c0 6.627-5.373 12-12 12H48c-6.627 0-12-5.373-12-12V40c0-6.627 5.373-12 12-12h31.03c3.182 0 6.234 1.264 8.485 3.515M69.84 78.789c0-.804.651-1.455 1.455-1.455 2.209 0 4-1.79 4-4s-1.791-4-4-4c-5.222 0-9.455 4.233-9.455 9.455 0 2.666 1.108 5.08 2.879 6.793 1.587 1.537 4.119 1.496 5.656-.091s1.495-4.12-.092-5.656c-.278-.27-.443-.635-.443-1.046m14.209-5.534c-4.917 0-8.902 3.986-8.902 8.903s3.985 8.901 8.902 8.901 8.902-3.985 8.902-8.901-3.986-8.903-8.902-8.903m-14.824 24.37c1.994-.953 2.838-3.34 1.886-5.334s-3.34-2.838-5.334-1.886c-4.275 2.042-7.397 6.14-8.22 10.986-.37 2.178 1.096 4.243 3.274 4.613s4.243-1.096 4.613-3.274c.391-2.302 1.874-4.195 3.781-5.106m14.824-4.762c-6.847 0-13.079 4.39-14.63 10.827-.977 4.057 2.458 6.977 5.714 6.977h17.832c3.256 0 6.691-2.92 5.713-6.977-1.55-6.436-7.782-10.827-14.63-10.827"
                })
            }),
            /*#__PURE__*/ (0, $5xBON$jsx)("g", {
                clipPath: "url(#0b906f__a)",
                children: /*#__PURE__*/ (0, $5xBON$jsxs)("g", {
                    clipPath: "url(#0b906f__b)",
                    children: [
                        /*#__PURE__*/ (0, $5xBON$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $5xBON$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $5xBON$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#0b906f__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $5xBON$jsx)("path", {
                            fill: "url(#0b906f__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $5xBON$jsx)("path", {
                            fill: "url(#0b906f__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $5xBON$jsx)("path", {
                            fill: "url(#0b906f__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $5xBON$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#0b906f__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $965d2b59a413c8f4$var$ForwardRef = /*#__PURE__*/ (0, $5xBON$forwardRef)($965d2b59a413c8f4$var$SvgComponent);
var $965d2b59a413c8f4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($965d2b59a413c8f4$var$ForwardRef);


export {$965d2b59a413c8f4$export$2e2bcd8739ae039 as default};
