// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_flagCheckmark_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3UoPt$reactjsxruntime = require("react/jsx-runtime");
var $3UoPt$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2ce5eaa496badfee$export$2e2bcd8739ae039);




const $2ce5eaa496badfee$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsxs)("linearGradient", {
                        id: "deebe8__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsxs)("linearGradient", {
                        id: "deebe8__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsxs)("linearGradient", {
                        id: "deebe8__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsxs)("radialGradient", {
                        id: "deebe8__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsxs)("radialGradient", {
                        id: "deebe8__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("clipPath", {
                        id: "deebe8__b",
                        children: /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("clipPath", {
                id: "deebe8__a",
                children: /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M37.277 24.25c2.21 0 4 1.79 4 4v6.645h69.445c6.628 0 12 5.372 12 12v41.958c0 6.628-5.372 12-12 12H41.277v30.897c0 2.209-1.79 4-4 4s-4-1.791-4-4V28.25c0-2.21 1.791-4 4-4m64.802 35.827c1.632-1.488 1.75-4.018.261-5.65s-4.018-1.75-5.65-.262l-1.178 1.073-.066.06-.068.062-.14.128-.147.134-.153.14-.323.294-.346.315-.368.335-.047.043-.048.044-.048.044-.048.044-.197.179-.832.758-1.861 1.697-2.066 1.883-.54.492-.273.248-.069.063-.068.063-.069.062-.069.063-.138.125-.069.063-.069.064-.069.063-.07.063-.138.126-.56.51-.28.256-.07.064-.07.064-.071.064-.07.065-.567.515-.07.065-.072.065-.07.064-.071.065-.284.259-.071.064-.072.065-.142.13-.284.259-.568.518-.567.516-.282.258-.07.064-.071.064-.07.064-.07.064-.282.256-.28.255-.555.507-.07.063-.069.063-.137.125-.274.25-.273.249-.136.123-.067.062-.068.062-.134.122-.135.123-.066.06-.067.061-.067.061-.066.06-1.043.95-.127.117-.063.057-.064.058-.25.229-.249.226-.122.112-.061.055-.061.056-.06.055-.061.055-.12.11-.06.054-.06.055-.118.108-.235.213-.058.053-.057.053-.058.052-.057.052-.114.104-.056.052-.057.05-.222.204-.219.199-.214.195-.81.739-.227.206-9.261-8.638c-1.616-1.506-4.147-1.418-5.654.198s-1.418 4.146.197 5.653l11.96 11.153c1.523 1.42 3.882 1.434 5.422.03l-2.695-2.956 2.695 2.956.004-.004.013-.011.052-.047.2-.184.07-.063.039-.035.02-.018.02-.018.176-.161.206-.188.233-.213.26-.236.285-.26.643-.586L80 80.2l.045-.04.045-.041.045-.042.046-.041.185-.17.095-.085.096-.088.395-.36.206-.187.051-.048.053-.047.105-.097.214-.195.109-.099.11-.1.449-.41.23-.21.234-.213.06-.054.059-.054.12-.108.12-.11.12-.11.062-.056.06-.056.062-.055.062-.056.247-.226.251-.23.127-.115.127-.116.257-.234.26-.236.065-.06.065-.06.131-.119.066-.06.066-.06.133-.12 1.078-.984.55-.501.556-.507.07-.064.07-.063.14-.128.281-.256.14-.128.071-.064.07-.065.071-.064.071-.064.141-.13.567-.516.071-.064.07-.065.143-.13.284-.259.284-.258.071-.065.072-.065.141-.13.285-.258.141-.13.071-.064.071-.065.283-.258.071-.064.07-.065.142-.129.07-.064.071-.064.07-.064.07-.064.141-.129.07-.064.07-.064.141-.127.14-.128.279-.254.139-.127.069-.063.07-.063.068-.063.07-.063.137-.126.138-.125.137-.125.068-.062.069-.062.136-.125.068-.062.068-.061.067-.062.068-.062.135-.123.067-.06.067-.062 1.054-.96.129-.118.064-.059.064-.058.064-.058.064-.058.063-.058.064-.058.063-.057.063-.058.063-.057.062-.057.125-.114.062-.056.062-.057.061-.056.062-.056.06-.055.062-.056.242-.22.06-.055.06-.055.118-.108.06-.054.058-.053.117-.107.058-.053.058-.053.058-.052.057-.053.114-.104.057-.051.057-.052.056-.051.056-.051.056-.05.055-.051.055-.05.055-.05.055-.05.054-.05.215-.196.21-.192.206-.187.051-.047.05-.046.05-.045.05-.046.756-.688.345-.315.324-.295.077-.07.076-.07.146-.133.141-.128.068-.062.033-.03.033-.03.472-.43.359-.327z"
                })
            }),
            /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("g", {
                clipPath: "url(#deebe8__a)",
                children: /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#deebe8__b)",
                    children: [
                        /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#deebe8__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("path", {
                            fill: "url(#deebe8__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("path", {
                            fill: "url(#deebe8__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("path", {
                            fill: "url(#deebe8__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $3UoPt$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#deebe8__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $2ce5eaa496badfee$var$ForwardRef = /*#__PURE__*/ (0, $3UoPt$react.forwardRef)($2ce5eaa496badfee$var$SvgComponent);
var $2ce5eaa496badfee$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2ce5eaa496badfee$var$ForwardRef);
