// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_audioWave_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $kCRv6$jsxs, jsx as $kCRv6$jsx} from "react/jsx-runtime";
import {forwardRef as $kCRv6$forwardRef} from "react";





const $181829d2e1da7e11$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kCRv6$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kCRv6$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kCRv6$jsxs)("linearGradient", {
                        id: "b2b0f9__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kCRv6$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kCRv6$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kCRv6$jsxs)("linearGradient", {
                        id: "b2b0f9__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kCRv6$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kCRv6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kCRv6$jsxs)("linearGradient", {
                        id: "b2b0f9__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kCRv6$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kCRv6$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kCRv6$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kCRv6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kCRv6$jsxs)("radialGradient", {
                        id: "b2b0f9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kCRv6$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kCRv6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kCRv6$jsxs)("radialGradient", {
                        id: "b2b0f9__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kCRv6$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kCRv6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kCRv6$jsx)("clipPath", {
                        id: "b2b0f9__b",
                        children: /*#__PURE__*/ (0, $kCRv6$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kCRv6$jsx)("clipPath", {
                id: "b2b0f9__a",
                children: /*#__PURE__*/ (0, $kCRv6$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M50 14.984c0-1.104-.896-2-2-2s-2 .896-2 2v66.032c0 1.104.895 2 2 2s2-.896 2-2zM18.25 41.016c1.105 0 2 .895 2 2v9.968c0 1.105-.895 2-2 2s-2-.895-2-2v-9.968c0-1.105.895-2 2-2m59.5 0c1.104 0 2 .895 2 2v9.968c0 1.105-.896 2-2 2s-2-.895-2-2v-9.968c0-1.105.895-2 2-2M38.082 29.002c1.105 0 2 .895 2 2v33.996c0 1.105-.895 2-2 2s-2-.895-2-2V31.002c0-1.105.896-2 2-2m21.833 2c0-1.105-.895-2-2-2s-2 .895-2 2v33.996c0 1.105.896 2 2 2s2-.895 2-2zM28.166 21c1.105 0 2 .895 2 2v50c0 1.105-.895 2-2 2s-2-.895-2-2V23c0-1.105.896-2 2-2m41.667 2c0-1.105-.896-2-2-2s-2 .895-2 2v50c0 1.105.895 2 2 2s2-.895 2-2z"
                })
            }),
            /*#__PURE__*/ (0, $kCRv6$jsx)("g", {
                clipPath: "url(#b2b0f9__a)",
                children: /*#__PURE__*/ (0, $kCRv6$jsxs)("g", {
                    clipPath: "url(#b2b0f9__b)",
                    children: [
                        /*#__PURE__*/ (0, $kCRv6$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $kCRv6$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $kCRv6$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#b2b0f9__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $kCRv6$jsx)("path", {
                            fill: "url(#b2b0f9__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $kCRv6$jsx)("path", {
                            fill: "url(#b2b0f9__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $kCRv6$jsx)("path", {
                            fill: "url(#b2b0f9__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $kCRv6$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#b2b0f9__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $181829d2e1da7e11$var$ForwardRef = /*#__PURE__*/ (0, $kCRv6$forwardRef)($181829d2e1da7e11$var$SvgComponent);
var $181829d2e1da7e11$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($181829d2e1da7e11$var$ForwardRef);


export {$181829d2e1da7e11$export$2e2bcd8739ae039 as default};
