// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_3D_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lHUtW$jsxs, jsx as $lHUtW$jsx} from "react/jsx-runtime";
import {forwardRef as $lHUtW$forwardRef} from "react";





const $bfa181027a1fe357$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lHUtW$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lHUtW$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lHUtW$jsxs)("linearGradient", {
                        id: "444a3b__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lHUtW$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lHUtW$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lHUtW$jsxs)("linearGradient", {
                        id: "444a3b__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lHUtW$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lHUtW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lHUtW$jsxs)("linearGradient", {
                        id: "444a3b__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lHUtW$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lHUtW$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lHUtW$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lHUtW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lHUtW$jsxs)("radialGradient", {
                        id: "444a3b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lHUtW$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lHUtW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lHUtW$jsxs)("radialGradient", {
                        id: "444a3b__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lHUtW$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lHUtW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lHUtW$jsx)("clipPath", {
                        id: "444a3b__b",
                        children: /*#__PURE__*/ (0, $lHUtW$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lHUtW$jsx)("clipPath", {
                id: "444a3b__a",
                children: /*#__PURE__*/ (0, $lHUtW$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M122.546 47.807 88.409 27.541c-5.01-2.974-11.241-2.99-16.266-.04L37.523 47.82c-1.522.892-2.861 2.015-3.984 3.309q.135.06.269.13L80.02 75.923l46.815-24.398c-1.179-1.463-2.624-2.728-4.29-3.717M29.621 61.618c0-1.134.12-2.252.352-3.337l.068.037 46.246 24.68v51.204c-1.303-.333-2.574-.832-3.78-1.498l-34.622-19.125c-5.099-2.816-8.264-8.181-8.264-14.005zm54.666 72.561c1.297-.338 2.561-.843 3.76-1.513l34.136-19.072c5.06-2.828 8.196-8.171 8.196-13.968v-38.06q-.001-1.395-.237-2.743L84.287 82.719z"
                })
            }),
            /*#__PURE__*/ (0, $lHUtW$jsx)("g", {
                clipPath: "url(#444a3b__a)",
                children: /*#__PURE__*/ (0, $lHUtW$jsxs)("g", {
                    clipPath: "url(#444a3b__b)",
                    children: [
                        /*#__PURE__*/ (0, $lHUtW$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $lHUtW$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $lHUtW$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#444a3b__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $lHUtW$jsx)("path", {
                            fill: "url(#444a3b__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $lHUtW$jsx)("path", {
                            fill: "url(#444a3b__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $lHUtW$jsx)("path", {
                            fill: "url(#444a3b__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $lHUtW$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#444a3b__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $bfa181027a1fe357$var$ForwardRef = /*#__PURE__*/ (0, $lHUtW$forwardRef)($bfa181027a1fe357$var$SvgComponent);
var $bfa181027a1fe357$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($bfa181027a1fe357$var$ForwardRef);


export {$bfa181027a1fe357$export$2e2bcd8739ae039 as default};
