// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_lightbulbRays_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $5vhIs$jsxs, jsx as $5vhIs$jsx} from "react/jsx-runtime";
import {forwardRef as $5vhIs$forwardRef} from "react";





const $a940b7ad063a2d1d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5vhIs$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5vhIs$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5vhIs$jsxs)("linearGradient", {
                        id: "b3942d__d",
                        x1: -11.953,
                        x2: 111.651,
                        y1: 78.607,
                        y2: -1.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5vhIs$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $5vhIs$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5vhIs$jsxs)("linearGradient", {
                        id: "b3942d__e",
                        x1: 90.597,
                        x2: 39.227,
                        y1: 26.041,
                        y2: 84.636,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5vhIs$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5vhIs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5vhIs$jsxs)("linearGradient", {
                        id: "b3942d__f",
                        x1: 23.977,
                        x2: 118.335,
                        y1: 89.279,
                        y2: 33.206,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5vhIs$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5vhIs$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5vhIs$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5vhIs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5vhIs$jsxs)("radialGradient", {
                        id: "b3942d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5vhIs$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $5vhIs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5vhIs$jsxs)("radialGradient", {
                        id: "b3942d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5vhIs$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5vhIs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5vhIs$jsx)("clipPath", {
                        id: "b3942d__b",
                        children: /*#__PURE__*/ (0, $5vhIs$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5vhIs$jsx)("clipPath", {
                id: "b3942d__a",
                children: /*#__PURE__*/ (0, $5vhIs$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M50.4 17.97c0 1.326-1.073 2.4-2.4 2.4-1.325 0-2.4-1.074-2.4-2.4v-3.612c0-1.326 1.075-2.4 2.4-2.4 1.327 0 2.4 1.074 2.4 2.4zm-3.719 8.238c-8.91.636-16.122 7.902-16.782 16.896-.486 6.684 2.568 12.672 7.47 16.254 1.59 1.164 2.562 3.006 2.562 4.992v4.05h15.918q.064.001.13.013.043.007.086.011v-4.068c0-1.938.888-3.786 2.448-4.914 4.62-3.324 7.632-8.778 7.632-14.946 0-10.572-8.85-19.05-19.47-18.294zm11.622 18.54c-1.326 0-2.4-1.074-2.4-2.4 0-3.006-2.664-5.448-5.94-5.448-1.326 0-2.4-1.074-2.4-2.4s1.074-2.4 2.4-2.4c5.922 0 10.74 4.596 10.74 10.248 0 1.326-1.074 2.4-2.4 2.4M39.937 73.2v2.88c0 1.686 1.356 3.054 3.024 3.054h10.086c1.668 0 3.024-1.368 3.024-3.054v-2.904q-.043.004-.086.011-.066.012-.13.013zM17.97 41.988h3.612c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4H17.97c-1.326 0-2.4-1.074-2.4-2.4s1.074-2.4 2.4-2.4m60.055 0h-3.613c-1.326 0-2.4 1.074-2.4 2.4s1.075 2.4 2.4 2.4h3.612c1.326 0 2.4-1.074 2.4-2.4s-1.074-2.4-2.4-2.4M67.537 21.456l-2.55 2.55c-.936.936-.936 2.454 0 3.396.468.468 1.08.702 1.698.702s1.23-.234 1.698-.702l2.55-2.55c.936-.936.936-2.454 0-3.396-.936-.936-2.454-.936-3.396 0m-39.067.006 2.55 2.55c.936.942.936 2.46 0 3.396-.468.468-1.08.702-1.698.702s-1.23-.234-1.698-.702l-2.55-2.55c-.936-.942-.936-2.46 0-3.396.942-.936 2.46-.936 3.396 0"
                })
            }),
            /*#__PURE__*/ (0, $5vhIs$jsx)("g", {
                clipPath: "url(#b3942d__a)",
                children: /*#__PURE__*/ (0, $5vhIs$jsxs)("g", {
                    clipPath: "url(#b3942d__b)",
                    children: [
                        /*#__PURE__*/ (0, $5vhIs$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $5vhIs$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M7.8 12.393h78.6v74.094H7.8z"
                        }),
                        /*#__PURE__*/ (0, $5vhIs$jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#b3942d__c)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 18 -16.924)"
                        }),
                        /*#__PURE__*/ (0, $5vhIs$jsx)("path", {
                            fill: "url(#b3942d__d)",
                            d: "M37.8-.696c-6.036-3.52-13.684-3.542-19.743-.055l-60.6 34.879c-5.935 3.416-5.945 11.48-.018 14.908l60.2 34.83c6.05 3.5 13.697 3.495 19.742-.013l60.075-34.862c5.895-3.422 5.906-11.44.018-14.874z"
                        }),
                        /*#__PURE__*/ (0, $5vhIs$jsx)("path", {
                            fill: "url(#b3942d__e)",
                            d: "M19.126 47.571C20.47 19.91 45.346-1.488 74.69-.222c29.344 1.267 52.043 24.718 50.7 52.38-1.344 27.661-26.22 49.059-55.565 47.792-29.343-1.266-52.042-24.717-50.699-52.379"
                        }),
                        /*#__PURE__*/ (0, $5vhIs$jsx)("path", {
                            fill: "url(#b3942d__f)",
                            d: "M-24.044 65.81C-12.379 61.12.2 61.32 11.1 65.423c21.809 8.23 34.498 3.142 43.3-17.37 4.384-10.292 12.943-19.008 24.62-23.705 23.348-9.374 50.301.798 60.231 22.742 9.931 21.943-.939 47.311-24.28 56.698-11.703 4.707-24.307 4.486-35.223.346-21.773-8.186-34.439-3.078-43.229 17.458-4.395 10.268-12.952 18.955-24.617 23.645-23.328 9.382-50.281-.79-60.206-22.721s.933-47.324 24.261-56.706"
                        }),
                        /*#__PURE__*/ (0, $5vhIs$jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#b3942d__g)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 47.578 -6.411)"
                        })
                    ]
                })
            })
        ]
    });
const $a940b7ad063a2d1d$var$ForwardRef = /*#__PURE__*/ (0, $5vhIs$forwardRef)($a940b7ad063a2d1d$var$SvgComponent);
var $a940b7ad063a2d1d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a940b7ad063a2d1d$var$ForwardRef);


export {$a940b7ad063a2d1d$export$2e2bcd8739ae039 as default};
