// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_effects_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $jhSOJ$reactjsxruntime = require("react/jsx-runtime");
var $jhSOJ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0a644b688cc1d2dd$export$2e2bcd8739ae039);




const $0a644b688cc1d2dd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "db9671__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "db9671__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "db9671__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "db9671__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "db9671__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("clipPath", {
                        id: "db9671__b",
                        children: /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("clipPath", {
                id: "db9671__a",
                children: /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M22.214 9.431c-1.086.827-1.858 2.14-2.304 3.365l-1.565 5.098h8.253c.455 0 .874.247 1.094.645l3.227 5.83 6.125-6.119c.489-.488 1.28-.487 1.768.001.488.489.487 1.28-.001 1.768-1.812 1.81-4.256 4.25-6.632 6.625l3.674 6.639c.334.604.116 1.364-.488 1.699s-1.365.115-1.7-.489l-3.326-6.01-.751.75c-1.53 1.53-2.907 2.906-3.903 3.904l-1.181 1.184-.322.325-.082.082-.006.007c-.43.467-1.152.542-1.67.153-.553-.414-.665-1.197-.25-1.75.032-.042.06-.074.064-.079l.013-.014.008-.01.021-.021.032-.033.092-.093q.116-.118.328-.33l1.184-1.187c.997-1 2.374-2.376 3.905-3.906l1.258-1.258-3.218-5.813h-8.283L11.266 40.96c-.203.66-.902 1.031-1.562.829-.66-.203-1.03-.902-.828-1.562l6.087-19.833H11.24c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h4.49l1.799-5.861q.009-.03.02-.058c.538-1.494 1.538-3.305 3.15-4.533 1.68-1.28 3.932-1.847 6.685-.926.655.22 1.008.928.79 1.582-.22.655-.928 1.008-1.583.79-1.963-.657-3.351-.239-4.377.543"
                })
            }),
            /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("g", {
                clipPath: "url(#db9671__a)",
                children: /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#db9671__b)",
                    children: [
                        /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#db9671__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("path", {
                            fill: "url(#db9671__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("path", {
                            fill: "url(#db9671__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("path", {
                            fill: "url(#db9671__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $jhSOJ$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#db9671__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $0a644b688cc1d2dd$var$ForwardRef = /*#__PURE__*/ (0, $jhSOJ$react.forwardRef)($0a644b688cc1d2dd$var$SvgComponent);
var $0a644b688cc1d2dd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0a644b688cc1d2dd$var$ForwardRef);
