// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_lightbulb_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2tXKq$jsxs, jsx as $2tXKq$jsx} from "react/jsx-runtime";
import {forwardRef as $2tXKq$forwardRef} from "react";





const $f25be4e91e2eb8b2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2tXKq$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2tXKq$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2tXKq$jsxs)("linearGradient", {
                        id: "d9cd18__d",
                        x1: -7.36,
                        x2: 60.505,
                        y1: 40.432,
                        y2: -0.857,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2tXKq$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2tXKq$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2tXKq$jsxs)("linearGradient", {
                        id: "d9cd18__e",
                        x1: 47.099,
                        x2: 17.985,
                        y1: 10.818,
                        y2: 42.124,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2tXKq$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2tXKq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2tXKq$jsxs)("linearGradient", {
                        id: "d9cd18__f",
                        x1: 11.721,
                        x2: 63.332,
                        y1: 46.444,
                        y2: 17.532,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2tXKq$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2tXKq$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2tXKq$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2tXKq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2tXKq$jsxs)("radialGradient", {
                        id: "d9cd18__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 10.992 15.74)scale(18.1603)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2tXKq$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2tXKq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2tXKq$jsxs)("radialGradient", {
                        id: "d9cd18__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 15.885 26.555)scale(18.1603)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2tXKq$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2tXKq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2tXKq$jsx)("clipPath", {
                        id: "d9cd18__b",
                        children: /*#__PURE__*/ (0, $2tXKq$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2tXKq$jsx)("clipPath", {
                id: "d9cd18__a",
                children: /*#__PURE__*/ (0, $2tXKq$jsx)("path", {
                    fill: "#fff",
                    d: "M23.21 8.155c-5.327.376-9.641 4.675-10.035 10.001-.292 3.955 1.537 7.5 4.467 9.62.952.69 1.532 1.78 1.532 2.955v2.797h9.649v-2.795c0-1.146.53-2.242 1.464-2.907 2.763-1.967 4.566-5.195 4.566-8.845 0-6.255-5.292-11.275-11.643-10.826m6.819 10.75c-.625 0-1.131-.506-1.131-1.13 0-2.037-1.657-3.693-3.693-3.693-.625 0-1.131-.507-1.131-1.131 0-.625.506-1.13 1.13-1.13 3.284 0 5.955 2.67 5.955 5.954 0 .624-.506 1.13-1.13 1.13M19.176 37.674c0 1 .81 1.81 1.809 1.81h6.03c.999 0 1.809-.81 1.809-1.81V35.79h-9.648z"
                })
            }),
            /*#__PURE__*/ (0, $2tXKq$jsx)("g", {
                clipPath: "url(#d9cd18__a)",
                children: /*#__PURE__*/ (0, $2tXKq$jsxs)("g", {
                    clipPath: "url(#d9cd18__b)",
                    children: [
                        /*#__PURE__*/ (0, $2tXKq$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $2tXKq$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.129 3.129H44.87V44.87H3.129z"
                        }),
                        /*#__PURE__*/ (0, $2tXKq$jsx)("circle", {
                            cx: 26.733,
                            cy: 4.748,
                            r: 18.16,
                            fill: "url(#d9cd18__c)",
                            transform: "rotate(-.08 26.733 4.748)"
                        }),
                        /*#__PURE__*/ (0, $2tXKq$jsx)("path", {
                            fill: "url(#d9cd18__d)",
                            d: "M19.062-4.245c-3.205-1.984-7.267-1.996-10.485-.031l-32.183 19.65c-3.151 1.924-3.157 6.466-.01 8.398L8.356 43.394c3.213 1.972 7.275 1.97 10.485-.007l31.904-19.64c3.131-1.928 3.137-6.445.01-8.38z"
                        }),
                        /*#__PURE__*/ (0, $2tXKq$jsx)("path", {
                            fill: "url(#d9cd18__e)",
                            d: "M9.143 22.948C9.857 7.364 23.068-4.691 38.652-3.977S66.29 9.948 65.577 25.53c-.713 15.584-13.925 27.639-29.509 26.925S8.43 38.531 9.143 22.948"
                        }),
                        /*#__PURE__*/ (0, $2tXKq$jsx)("path", {
                            fill: "url(#d9cd18__f)",
                            d: "M-13.781 33.222c6.194-2.642 12.875-2.529 18.663-.218 11.582 4.637 18.321 1.77 22.995-9.785 2.329-5.798 6.874-10.71 13.076-13.355 12.399-5.281 26.713.45 31.987 12.812S72.44 49.33 60.045 54.618c-6.215 2.651-12.909 2.527-18.706.195-11.563-4.612-18.29-1.734-22.958 9.835-2.334 5.785-6.878 10.679-13.073 13.321-12.389 5.285-26.703-.445-31.974-12.8s.496-26.661 12.885-31.947"
                        }),
                        /*#__PURE__*/ (0, $2tXKq$jsx)("circle", {
                            cx: 42.44,
                            cy: 10.67,
                            r: 18.16,
                            fill: "url(#d9cd18__g)",
                            transform: "rotate(-.08 42.44 10.67)"
                        })
                    ]
                })
            })
        ]
    });
const $f25be4e91e2eb8b2$var$ForwardRef = /*#__PURE__*/ (0, $2tXKq$forwardRef)($f25be4e91e2eb8b2$var$SvgComponent);
var $f25be4e91e2eb8b2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f25be4e91e2eb8b2$var$ForwardRef);


export {$f25be4e91e2eb8b2$export$2e2bcd8739ae039 as default};
