// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_noElements_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $dVz4T$jsxs, jsx as $dVz4T$jsx} from "react/jsx-runtime";
import {forwardRef as $dVz4T$forwardRef} from "react";





const $c320d4ab1547f71d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dVz4T$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dVz4T$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dVz4T$jsxs)("linearGradient", {
                        id: "32aa7d__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dVz4T$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $dVz4T$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dVz4T$jsxs)("linearGradient", {
                        id: "32aa7d__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dVz4T$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dVz4T$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dVz4T$jsxs)("linearGradient", {
                        id: "32aa7d__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dVz4T$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dVz4T$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dVz4T$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dVz4T$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dVz4T$jsxs)("radialGradient", {
                        id: "32aa7d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dVz4T$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $dVz4T$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dVz4T$jsxs)("radialGradient", {
                        id: "32aa7d__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dVz4T$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dVz4T$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dVz4T$jsx)("clipPath", {
                        id: "32aa7d__b",
                        children: /*#__PURE__*/ (0, $dVz4T$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dVz4T$jsxs)("clipPath", {
                id: "32aa7d__a",
                children: [
                    /*#__PURE__*/ (0, $dVz4T$jsx)("path", {
                        fill: "#D9D9D9",
                        d: "M125.071 21.723c.926-1.07 2.534-1.967 3.524-1.663l.008.008c1.592 0 3.128.864 4.007 2.248.871 1.366 1.014 3.095.388 4.635l-5.393 13.196 7.824 3.441c1.418.618 2.455 1.919 2.788 3.474s-.079 3.195-1.109 4.38L116.676 75.02c-.918 1.062-2.17 1.647-3.508 1.647-1.592 0-3.128-.856-4.007-2.24-.871-1.358-1.022-3.095-.396-4.626l5.219-12.868-7.777-3.482c-1.402-.626-2.431-1.926-2.764-3.474s.079-3.17 1.093-4.355zM16.448 93.333c0-2.21 1.79-4 4-4h46.91c2.21 0 4 1.79 4 4v9.382c0 2.209-1.79 4-4 4s-4-1.791-4-4v-5.382H47.9v38.911h5.383c2.21 0 4 1.791 4 4s-1.79 4-4 4H34.52c-2.21 0-4-1.791-4-4s1.79-4 4-4h5.38V97.333H24.449v5.382c0 2.209-1.791 4-4 4s-4-1.791-4-4z"
                    }),
                    /*#__PURE__*/ (0, $dVz4T$jsx)("path", {
                        fill: "#D9D9D9",
                        fillRule: "evenodd",
                        d: "M89.998 106.666v21.25c0 7.364 5.97 13.334 13.333 13.334h35.833c7.364 0 13.334-5.97 13.334-13.334v-21.25c0-7.363-5.97-13.333-13.334-13.333h-35.833c-7.364 0-13.333 5.97-13.333 13.333m27.791 1.747 12.296 7.019c1.377.787 1.375 2.748-.004 3.531l-12.313 6.99c-1.379.783-3.101-.2-3.099-1.769l.017-14.009c.002-1.569 1.726-2.548 3.103-1.762M11.664 57.916v-21.25c0-7.363 5.97-13.333 13.333-13.333h36.875c7.364 0 13.334 5.97 13.334 13.333v21.25c0 2.039-.458 3.97-1.276 5.699l-6.794-10.601c-3.935-6.139-12.904-6.139-16.839 0l-2.689 4.196-7.097-11.096c-3.931-6.146-12.907-6.15-16.844-.008L12.735 63.162c-.69-1.61-1.07-3.383-1.07-5.246M62.496 40c2.485 0 4.5-2.015 4.5-4.5s-2.015-4.5-4.5-4.5-4.5 2.015-4.5 4.5 2.015 4.5 4.5 4.5"
                    }),
                    /*#__PURE__*/ (0, $dVz4T$jsx)("path", {
                        fill: "#D9D9D9",
                        d: "M42.855 64.626 33.77 50.424c-.786-1.229-2.581-1.23-3.368-.001l-12.17 18.986c1.983 1.17 4.296 1.84 6.764 1.84H38.61zM48.112 71.25h13.76c2.32 0 4.502-.593 6.402-1.635L60.4 57.331c-.786-1.228-2.58-1.228-3.367 0z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dVz4T$jsx)("g", {
                clipPath: "url(#32aa7d__a)",
                children: /*#__PURE__*/ (0, $dVz4T$jsxs)("g", {
                    clipPath: "url(#32aa7d__b)",
                    children: [
                        /*#__PURE__*/ (0, $dVz4T$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $dVz4T$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $dVz4T$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#32aa7d__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $dVz4T$jsx)("path", {
                            fill: "url(#32aa7d__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $dVz4T$jsx)("path", {
                            fill: "url(#32aa7d__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $dVz4T$jsx)("path", {
                            fill: "url(#32aa7d__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $dVz4T$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#32aa7d__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $c320d4ab1547f71d$var$ForwardRef = /*#__PURE__*/ (0, $dVz4T$forwardRef)($c320d4ab1547f71d$var$SvgComponent);
var $c320d4ab1547f71d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c320d4ab1547f71d$var$ForwardRef);


export {$c320d4ab1547f71d$export$2e2bcd8739ae039 as default};
