// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_filters_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3AeJI$reactjsxruntime = require("react/jsx-runtime");
var $3AeJI$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9ba0a7ddd26a6458$export$2e2bcd8739ae039);




const $9ba0a7ddd26a6458$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "420398__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "420398__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "420398__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsxs)("radialGradient", {
                        id: "420398__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsxs)("radialGradient", {
                        id: "420398__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("clipPath", {
                        id: "420398__b",
                        children: /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("clipPath", {
                id: "420398__a",
                children: /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M12 57.9c0-6.975 3.092-13.228 7.98-17.464.768 8.999 5.933 16.743 13.338 21.074.965 7.252 4.799 13.598 10.323 17.86C40.999 80.422 38.117 81 35.1 81 22.342 81 12 70.658 12 57.9m39.639 21.47c2.642 1.052 5.524 1.63 8.54 1.63 12.759 0 23.1-10.342 23.1-23.1 0-7.554-3.625-14.26-9.23-18.475-.426 8.833-5.08 16.557-11.983 21.191-.76 7.624-4.677 14.318-10.427 18.754M69.37 32.398C66.832 22.399 57.771 15 46.98 15c-10.996 0-20.197 7.682-22.529 17.972C27.72 31.574 31.32 30.8 35.1 30.8c4.525 0 8.79 1.109 12.54 3.07 3.75-1.961 8.015-3.07 12.54-3.07 3.226 0 6.321.564 9.191 1.598m-45.488 5.303-.003.399c0 7.554 3.626 14.26 9.232 18.475.394-8.198 4.432-15.44 10.529-20.144-2.642-1.052-5.524-1.63-8.541-1.63-4.07 0-7.895 1.052-11.217 2.9M47.64 77.303c4.978-3.224 8.637-8.311 9.989-14.275C54.36 64.426 50.76 65.2 46.98 65.2c-3.226 0-6.321-.564-9.191-1.598 1.45 5.716 5.033 10.581 9.851 13.701M70.08 38.1c0 6.975-3.092 13.228-7.98 17.464-.664-7.781-4.616-14.624-10.461-19.133 2.642-1.052 5.524-1.63 8.54-1.63 3.534 0 6.882.792 9.876 2.21q.025.541.025 1.089M58.2 57.9c0-8.135-4.205-15.288-10.56-19.403-6.355 4.115-10.56 11.268-10.56 19.403q0 .548.025 1.089C40.1 60.407 43.447 61.2 46.98 61.2c4.07 0 7.895-1.053 11.217-2.901z"
                })
            }),
            /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("g", {
                clipPath: "url(#420398__a)",
                children: /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#420398__b)",
                    children: [
                        /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#420398__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("path", {
                            fill: "url(#420398__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("path", {
                            fill: "url(#420398__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("path", {
                            fill: "url(#420398__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $3AeJI$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#420398__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $9ba0a7ddd26a6458$var$ForwardRef = /*#__PURE__*/ (0, $3AeJI$react.forwardRef)($9ba0a7ddd26a6458$var$SvgComponent);
var $9ba0a7ddd26a6458$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9ba0a7ddd26a6458$var$ForwardRef);
