// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_hand_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2lu6o$jsxs, jsx as $2lu6o$jsx} from "react/jsx-runtime";
import {forwardRef as $2lu6o$forwardRef} from "react";





const $d419ad27d714053b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2lu6o$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2lu6o$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2lu6o$jsxs)("linearGradient", {
                        id: "5942a1__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2lu6o$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2lu6o$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2lu6o$jsxs)("linearGradient", {
                        id: "5942a1__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2lu6o$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2lu6o$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2lu6o$jsxs)("linearGradient", {
                        id: "5942a1__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2lu6o$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2lu6o$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2lu6o$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2lu6o$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2lu6o$jsxs)("radialGradient", {
                        id: "5942a1__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2lu6o$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2lu6o$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2lu6o$jsxs)("radialGradient", {
                        id: "5942a1__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2lu6o$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2lu6o$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2lu6o$jsx)("clipPath", {
                        id: "5942a1__b",
                        children: /*#__PURE__*/ (0, $2lu6o$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2lu6o$jsx)("clipPath", {
                id: "5942a1__a",
                children: /*#__PURE__*/ (0, $2lu6o$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m9.326 25.367-.01-.02c-.682-1.183-.512-2.482.45-3.204.99-.744 2.393-.553 3.149.43l2.944 3.829c.13.17.354.237.557.168s.34-.259.34-.473V13.315c0-1.348 1.087-2.438 2.425-2.438s2.426 1.09 2.426 2.438v8.22c0 .277.224.5.5.5h.006c.276 0 .5-.223.5-.5V10.439C22.613 9.09 23.701 8 25.04 8s2.426 1.09 2.426 2.438v11.1c0 .275.224.5.5.5h.006c.276 0 .5-.225.5-.5V11.83c0-1.349 1.088-2.439 2.426-2.439 1.34 0 2.428 1.09 2.428 2.439v10.439c0 .265.207.484.472.5.265.014.495-.18.524-.444l.28-2.496.28-2.496v-.007c.143-1.455 1.439-2.178 2.629-1.945.958.188 1.711 1.048 1.568 2.433l-1.243 8.695q-.012.037-.02.076c-1 5.556-2.646 8.889-4.745 10.838C30.985 39.36 28.377 40 25.28 40c-2.025 0-4.033-.871-5.93-2.235-1.894-1.361-3.64-3.187-5.133-5.031-1.49-1.842-2.714-3.686-3.567-5.07-.425-.693-.757-1.269-.983-1.671-.112-.201-.198-.359-.256-.466l-.064-.12-.016-.031-.004-.007z"
                })
            }),
            /*#__PURE__*/ (0, $2lu6o$jsx)("g", {
                clipPath: "url(#5942a1__a)",
                children: /*#__PURE__*/ (0, $2lu6o$jsxs)("g", {
                    clipPath: "url(#5942a1__b)",
                    children: [
                        /*#__PURE__*/ (0, $2lu6o$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $2lu6o$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $2lu6o$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#5942a1__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $2lu6o$jsx)("path", {
                            fill: "url(#5942a1__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $2lu6o$jsx)("path", {
                            fill: "url(#5942a1__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $2lu6o$jsx)("path", {
                            fill: "url(#5942a1__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $2lu6o$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#5942a1__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $d419ad27d714053b$var$ForwardRef = /*#__PURE__*/ (0, $2lu6o$forwardRef)($d419ad27d714053b$var$SvgComponent);
var $d419ad27d714053b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d419ad27d714053b$var$ForwardRef);


export {$d419ad27d714053b$export$2e2bcd8739ae039 as default};
