// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_piggyBank_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8ZZeI$reactjsxruntime = require("react/jsx-runtime");
var $8ZZeI$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $22aeb89de557e071$export$2e2bcd8739ae039);




const $22aeb89de557e071$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "12de2a__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "12de2a__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "12de2a__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsxs)("radialGradient", {
                        id: "12de2a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsxs)("radialGradient", {
                        id: "12de2a__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("clipPath", {
                        id: "12de2a__b",
                        children: /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("clipPath", {
                id: "12de2a__a",
                children: /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M32.845 35.216c4.84-2.564 8.06-7.11 8.06-12.29q0-.638-.065-1.261c.668-.09 1.432-.364 2.113-.932.974-.814 1.646-2.121 1.73-3.985.032-.69-.502-1.274-1.192-1.305-.69-.032-1.274.502-1.305 1.192-.059 1.288-.49 1.89-.837 2.18-.363.303-.78.387-1.018.383-1.949-6.2-8.644-10.77-16.603-10.77-3.28 0-6.344.775-8.951 2.121-1.55-1.661-3.976-2.142-5.856-2.12-.696.009-1.058.764-.709 1.366l2.26 3.905.017-.012c-1.182 1.205-2.137 2.574-2.81 4.061v-.008l-3.37 1.963c-.614.358-.993 1.016-.993 1.728v2.989c0 .712.379 1.37.994 1.728l3.368 1.96v-.006c1.337 2.952 3.785 5.44 6.911 7.102q-.012-.003-.02-.002l1.243 3.386c.217.59.78.983 1.408.983h1.727c.628 0 1.19-.392 1.407-.981l.5-1.36-.044-.015q1.424.207 2.918.208 1.46-.001 2.855-.199l-.02.006.5 1.36c.218.59.78.981 1.408.981h1.727c.629 0 1.191-.392 1.408-.983zM26.21 13.818c-1.543-.344-2.82-.358-4.452 0-.675.149-1.341-.278-1.49-.952-.148-.674.279-1.341.953-1.49 1.992-.437 3.637-.42 5.532.002.674.15 1.099.817.948 1.491-.15.674-.817 1.099-1.491.949M15.36 17.69c0 .967-.783 1.75-1.75 1.75-.966 0-1.75-.783-1.75-1.75 0-.966.784-1.75 1.75-1.75.967 0 1.75.784 1.75 1.75"
                })
            }),
            /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("g", {
                clipPath: "url(#12de2a__a)",
                children: /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#12de2a__b)",
                    children: [
                        /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#12de2a__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("path", {
                            fill: "url(#12de2a__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("path", {
                            fill: "url(#12de2a__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("path", {
                            fill: "url(#12de2a__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $8ZZeI$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#12de2a__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $22aeb89de557e071$var$ForwardRef = /*#__PURE__*/ (0, $8ZZeI$react.forwardRef)($22aeb89de557e071$var$SvgComponent);
var $22aeb89de557e071$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($22aeb89de557e071$var$ForwardRef);
