// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_assets_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $fzF93$reactjsxruntime = require("react/jsx-runtime");
var $fzF93$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $dcea516be1ca0d52$export$2e2bcd8739ae039);




const $dcea516be1ca0d52$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsxs)("linearGradient", {
                        id: "383fc0__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.788,
                        y2: -9.168,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsxs)("linearGradient", {
                        id: "383fc0__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.682,
                        y2: 89.679,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsxs)("linearGradient", {
                        id: "383fc0__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.616,
                        y2: 33.124,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsxs)("radialGradient", {
                        id: "383fc0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.28 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsxs)("radialGradient", {
                        id: "383fc0__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.532 53.874)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("clipPath", {
                        id: "383fc0__b",
                        children: /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: -0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("clipPath", {
                id: "383fc0__a",
                children: /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M18.25 26.05c0-3.313 2.686-6 6-6H62c3.314 0 6 2.687 6 6v11.7H49.552c-3.144 0-5.948 1.45-7.782 3.718l-3.1-4.99-.003-.006c-1.945-3.158-6.52-3.206-8.532-.089l-.001.003-11.459 17.811c-.274-.687-.425-1.437-.425-2.222zm17.017 12.53 4.477 7.207q-.19.954-.192 1.963v10.225H24.25c-1.012 0-1.966-.251-2.803-.694l12.05-18.73c.417-.644 1.363-.634 1.765.02zm23.878-7.162c0 1.657-1.343 3-3 3s-3-1.343-3-3 1.343-3 3-3 3 1.343 3 3M43.552 47.75c0-3.314 2.686-6 6-6H71.75c3.314 0 6 2.686 6 6v22.198c0 3.314-2.686 6-6 6H49.552c-3.314 0-6-2.686-6-6zm24.48 9.96-11.215-6.432c-1.256-.72-2.828.176-2.83 1.614l-.015 12.838c-.002 1.437 1.568 2.338 2.826 1.62l11.23-6.405c1.257-.718 1.26-2.515.003-3.235"
                })
            }),
            /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("g", {
                clipPath: "url(#383fc0__a)",
                children: /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#383fc0__b)",
                    children: [
                        /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: -0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0-.001h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.722,
                            r: 41.765,
                            fill: "url(#383fc0__c)",
                            transform: "rotate(-.08 54.28 3.722)"
                        }),
                        /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("path", {
                            fill: "url(#383fc0__d)",
                            d: "M36.643-16.96c-7.373-4.561-16.714-4.59-24.114-.071l-74.015 45.19c-7.249 4.427-7.26 14.874-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.432 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("path", {
                            fill: "url(#383fc0__e)",
                            d: "M13.837 45.578C15.477 9.738 45.86-17.985 81.7-16.345s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("path", {
                            fill: "url(#383fc0__f)",
                            d: "M-38.895 69.208c14.246-6.077 29.61-5.817 42.922-.5 26.636 10.663 42.135 4.07 52.885-22.505 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $fzF93$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.343,
                            r: 41.765,
                            fill: "url(#383fc0__g)",
                            transform: "rotate(-.08 90.406 17.343)"
                        })
                    ]
                })
            })
        ]
    });
const $dcea516be1ca0d52$var$ForwardRef = /*#__PURE__*/ (0, $fzF93$react.forwardRef)($dcea516be1ca0d52$var$SvgComponent);
var $dcea516be1ca0d52$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($dcea516be1ca0d52$var$ForwardRef);
