// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_globe_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $3uxKc$jsxs, jsx as $3uxKc$jsx} from "react/jsx-runtime";
import {forwardRef as $3uxKc$forwardRef} from "react";





const $7bda43a27b06dceb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3uxKc$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3uxKc$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3uxKc$jsxs)("linearGradient", {
                        id: "2264ae__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3uxKc$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $3uxKc$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3uxKc$jsxs)("linearGradient", {
                        id: "2264ae__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3uxKc$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3uxKc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3uxKc$jsxs)("linearGradient", {
                        id: "2264ae__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3uxKc$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3uxKc$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3uxKc$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3uxKc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3uxKc$jsxs)("radialGradient", {
                        id: "2264ae__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3uxKc$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $3uxKc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3uxKc$jsxs)("radialGradient", {
                        id: "2264ae__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3uxKc$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3uxKc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3uxKc$jsx)("clipPath", {
                        id: "2264ae__b",
                        children: /*#__PURE__*/ (0, $3uxKc$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3uxKc$jsx)("clipPath", {
                id: "2264ae__a",
                children: /*#__PURE__*/ (0, $3uxKc$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M23.977 79.987c0-17.009 7.579-32.247 19.544-42.521.923 2.022 2.32 4.67 4.003 7.132 1.05 1.537 2.278 3.099 3.645 4.41 1.329 1.273 3.038 2.551 5.077 3.134 3.287.939 7.097.801 9.977.697q.456-.017.878-.03c1.63-.052 2.943-.066 4.03.045 1.114.114 1.57.325 1.709.417.307.205.386.367.412.43.033.08.067.223.026.458-.089.522-.525 1.277-1.398 1.762-2.073 1.152-4.143 2.599-5.695 4.594-1.626 2.09-2.542 4.602-2.542 7.554 0 .862-.068 1.398-.148 1.719l-.009.032c-.273.055-.798.104-1.69.03l-.857-.075c-2.318-.206-5.688-.504-8.868-.171-3.54.37-8.095 1.644-10.558 5.955-1.08 1.89-2.33 5.037-2.032 8.799.31 3.924 2.267 7.993 6.713 11.55 5.12 4.097 10.752 4.977 18.123 5.796 4.2.467 6.581 1.367 7.939 2.325 1.19.841 1.846 1.898 2.175 3.546.575 2.877-.395 7.187-2.433 12.445-.975 2.516-2.113 5.054-3.246 7.518l-.743 1.612c-.812 1.755-1.607 3.473-2.273 5.028-24.032-6.309-41.76-28.18-41.76-54.191m26.362-46.902q-.104-.255-.237-.486c3.834-2.424 7.982-4.395 12.37-5.84q.075.452.258.89c2.056 4.898 8.302 10.941 16.68 10.941 4.327 0 7.798-1.855 10.35-3.972 2.527-2.096 4.343-4.6 5.473-6.33.413-.632.624-1.338.649-2.042q1.035.305 2.051.649-.085.276-.129.57c-.673 4.46-1.437 10.7-.74 16.343.686 5.544 3.007 11.908 9.759 14.252 5.356 1.86 10.656.945 14.796-.9 2.737-1.22 5.23-2.963 7.071-4.904 2.109 3.695 3.81 7.653 5.04 11.814-.535-.113-1.121-.233-1.75-.357-3.013-.59-7.064-1.262-11.28-1.542-4.159-.276-8.77-.193-12.777.906-4.021 1.102-8.103 3.444-9.84 8.134-4.859 13.115 1.7 21.061 3.393 22.755 2.554 2.553 5.174 3.747 7.1 4.624q.351.159.67.307c1.861.87 3.188 1.627 4.54 3.58.467.675.945 1.887 1.3 3.74.343 1.796.517 3.894.583 6.106.065 2.199.021 4.415-.043 6.435-.017.551-.036 1.1-.055 1.634v.007c-.038 1.074-.074 2.09-.087 2.944-9.662 7.916-22.018 12.667-35.484 12.667q-3.135-.001-6.181-.337c.435-.967.91-1.993 1.413-3.082v-.002q.385-.83.79-1.709c1.145-2.492 2.367-5.214 3.435-7.968 2.047-5.281 3.89-11.544 2.82-16.905-.67-3.35-2.304-6.323-5.407-8.513-2.936-2.073-6.855-3.206-11.67-3.741-7.456-.83-10.891-1.598-14.008-4.091-2.896-2.318-3.617-4.428-3.736-5.935-.132-1.67.435-3.206 1.003-4.2.513-.897 1.725-1.682 4.444-1.967 2.354-.246 4.903-.025 7.3.182h.001l.928.08c2.863.238 5.98-.099 8.18-2.42 2.064-2.18 2.331-5.133 2.331-7.334 0-1.168.32-1.952.857-2.641.61-.786 1.63-1.605 3.265-2.513 2.835-1.575 4.87-4.32 5.398-7.403.56-3.262-.636-6.73-3.886-8.896-1.715-1.144-3.68-1.55-5.332-1.72-1.682-.172-3.485-.134-5.098-.083l-.653.022c-3.224.105-5.75.187-7.75-.385-.268-.076-.862-.377-1.739-1.217-.837-.802-1.715-1.888-2.577-3.149-1.728-2.528-3.124-5.366-3.789-6.998m73.361 81.962c7.71-9.598 12.324-21.79 12.324-35.06 0-2.487-.162-4.937-.476-7.338l-.125-.022c-.135-.026-.285-.057-.423-.087-.199-.044-.454-.1-.739-.165l-.348-.078c-.873-.195-2.06-.456-3.471-.733-2.838-.556-6.524-1.162-10.272-1.41-3.805-.254-7.384-.116-10.131.638-2.733.749-3.97 1.891-4.454 3.198-3.383 9.13 1.142 13.912 1.549 14.32 1.487 1.487 2.844 2.113 4.772 3.003l.726.336c2.423 1.132 5.216 2.641 7.731 6.274 1.387 2.003 2.136 4.472 2.58 6.79.455 2.376.65 4.941.722 7.373.03 1 .039 1.994.035 2.961m.277-69.772c-4.974-6.294-11.271-11.495-18.471-15.183-.572 4.097-.995 8.747-.502 12.734.564 4.557 2.101 6.864 4.442 7.676 2.957 1.027 6.112.6 8.917-.65 2.913-1.297 4.847-3.21 5.456-4.315q.075-.136.158-.262M80 23.963c2.755 0 5.463.199 8.111.583-.9 1.306-2.058 2.753-3.458 3.915-1.606 1.332-3.332 2.129-5.242 2.129-4.395 0-7.991-3.273-9.188-5.777 3.175-.559 6.442-.85 9.777-.85"
                })
            }),
            /*#__PURE__*/ (0, $3uxKc$jsx)("g", {
                clipPath: "url(#2264ae__a)",
                children: /*#__PURE__*/ (0, $3uxKc$jsxs)("g", {
                    clipPath: "url(#2264ae__b)",
                    children: [
                        /*#__PURE__*/ (0, $3uxKc$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $3uxKc$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $3uxKc$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#2264ae__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $3uxKc$jsx)("path", {
                            fill: "url(#2264ae__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $3uxKc$jsx)("path", {
                            fill: "url(#2264ae__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $3uxKc$jsx)("path", {
                            fill: "url(#2264ae__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $3uxKc$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#2264ae__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $7bda43a27b06dceb$var$ForwardRef = /*#__PURE__*/ (0, $3uxKc$forwardRef)($7bda43a27b06dceb$var$SvgComponent);
var $7bda43a27b06dceb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7bda43a27b06dceb$var$ForwardRef);


export {$7bda43a27b06dceb$export$2e2bcd8739ae039 as default};
