// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_interaction_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $jWHEU$jsxs, jsx as $jWHEU$jsx} from "react/jsx-runtime";
import {forwardRef as $jWHEU$forwardRef} from "react";





const $ac7b668699132334$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jWHEU$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jWHEU$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jWHEU$jsxs)("linearGradient", {
                        id: "e98d8b__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jWHEU$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $jWHEU$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jWHEU$jsxs)("linearGradient", {
                        id: "e98d8b__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jWHEU$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jWHEU$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jWHEU$jsxs)("linearGradient", {
                        id: "e98d8b__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jWHEU$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jWHEU$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jWHEU$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jWHEU$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jWHEU$jsxs)("radialGradient", {
                        id: "e98d8b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jWHEU$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $jWHEU$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jWHEU$jsxs)("radialGradient", {
                        id: "e98d8b__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jWHEU$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jWHEU$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jWHEU$jsx)("clipPath", {
                        id: "e98d8b__b",
                        children: /*#__PURE__*/ (0, $jWHEU$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jWHEU$jsx)("clipPath", {
                id: "e98d8b__a",
                children: /*#__PURE__*/ (0, $jWHEU$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M27.625 17.066c0 1.551-.368 3.017-1.022 4.313q-.275.108-.546.223l-2.58-4.643-.02-.034c-1.084-1.813-3.67-3.311-6.245-1.916-2.422 1.313-3.148 4.444-1.817 6.697l2.75 4.952h-.112c-5.297 0-9.592-4.294-9.592-9.592 0-5.297 4.295-9.591 9.592-9.591s9.592 4.294 9.592 9.591m-6.84 9.192 3.96 7.128c.11.2-.082.428-.296.346-.672-.258-1.408-.575-2.169-.903-1.4-.603-2.887-1.244-4.219-1.623-1.36-.386-2.655.249-3 1.5-.337 1.228 0 2.25 1.5 3l.42.213c2.166 1.106 9.021 4.605 13.631 4.605 4.97 0 8.95-4.03 8.95-9 0-4.971-2.78-8.818-7.75-8.818-1.86 0-4.952 1.078-6.52 2.093-.125.082-.295.043-.367-.088l-3.614-6.504c-.615-1.03-1.854-1.572-2.907-1-.65.351-1.051 1.016-1.147 1.725-.07.52.023 1.063.304 1.525l3.223 5.801"
                })
            }),
            /*#__PURE__*/ (0, $jWHEU$jsx)("g", {
                clipPath: "url(#e98d8b__a)",
                children: /*#__PURE__*/ (0, $jWHEU$jsxs)("g", {
                    clipPath: "url(#e98d8b__b)",
                    children: [
                        /*#__PURE__*/ (0, $jWHEU$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $jWHEU$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $jWHEU$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#e98d8b__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $jWHEU$jsx)("path", {
                            fill: "url(#e98d8b__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $jWHEU$jsx)("path", {
                            fill: "url(#e98d8b__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $jWHEU$jsx)("path", {
                            fill: "url(#e98d8b__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $jWHEU$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#e98d8b__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $ac7b668699132334$var$ForwardRef = /*#__PURE__*/ (0, $jWHEU$forwardRef)($ac7b668699132334$var$SvgComponent);
var $ac7b668699132334$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ac7b668699132334$var$ForwardRef);


export {$ac7b668699132334$export$2e2bcd8739ae039 as default};
