// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileText_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $c4MBb$reactjsxruntime = require("react/jsx-runtime");
var $c4MBb$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c9c538b9d9682143$export$2e2bcd8739ae039);




const $c9c538b9d9682143$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "aa0643__d",
                        x1: -14.409,
                        x2: 120.646,
                        y1: 80.7,
                        y2: -1.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "aa0643__e",
                        x1: 93.971,
                        x2: 36.033,
                        y1: 21.766,
                        y2: 84.066,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "aa0643__f",
                        x1: 23.561,
                        x2: 126.271,
                        y1: 92.664,
                        y2: 35.128,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsxs)("radialGradient", {
                        id: "aa0643__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 21.875 31.562)scale(36.1398)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsxs)("radialGradient", {
                        id: "aa0643__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 31.611 53.083)scale(36.1398)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("clipPath", {
                        id: "aa0643__b",
                        children: /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("clipPath", {
                id: "aa0643__a",
                children: /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M72.293 38.691 52.51 18.909c-1.35-1.35-3.181-2.11-5.09-2.11H28.801c-3.977 0-7.2 3.224-7.2 7.2v48c0 3.977 3.223 7.2 7.2 7.2h38.4c3.976 0 7.2-3.223 7.2-7.2V43.783c0-1.91-.759-3.74-2.11-5.091M62.402 50.4c0 1.325-1.075 2.4-2.4 2.4-1.326 0-2.4-1.075-2.4-2.4V48h-7.2v19.2h2.4c1.325 0 2.4 1.074 2.4 2.4s-1.075 2.4-2.4 2.4h-9.6c-1.326 0-2.4-1.075-2.4-2.4 0-1.326 1.074-2.4 2.4-2.4h2.4V48h-7.2v2.4c0 1.325-1.075 2.4-2.4 2.4-1.326 0-2.4-1.075-2.4-2.4v-4.8c0-1.326 1.074-2.4 2.4-2.4h24c1.325 0 2.4 1.074 2.4 2.4z"
                })
            }),
            /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("g", {
                clipPath: "url(#aa0643__a)",
                children: /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#aa0643__b)",
                    children: [
                        /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6.465 6.465h83.07v83.07H6.465z"
                        }),
                        /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("circle", {
                            cx: 53.436,
                            cy: 9.687,
                            r: 36.14,
                            fill: "url(#aa0643__c)",
                            transform: "rotate(-.08 53.436 9.687)"
                        }),
                        /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("path", {
                            fill: "url(#aa0643__d)",
                            d: "M38.173-8.209c-6.38-3.947-14.463-3.972-20.866-.062L-46.74 30.833c-6.272 3.83-6.283 12.87-.019 16.714l63.623 39.048c6.393 3.924 14.477 3.92 20.865-.014l63.491-39.085c6.231-3.836 6.242-12.825.019-16.676z"
                        }),
                        /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("path", {
                            fill: "url(#aa0643__e)",
                            d: "M18.437 45.905c1.42-31.013 27.711-55.002 58.724-53.582s55.001 27.711 53.582 58.723c-1.42 31.013-27.712 55.002-58.724 53.582-31.012-1.419-55.002-27.71-53.582-58.723"
                        }),
                        /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("path", {
                            fill: "url(#aa0643__f)",
                            d: "M-27.19 66.352c12.328-5.259 25.623-5.033 37.142-.433C33 75.146 46.412 69.44 55.714 46.445c4.633-11.539 13.679-21.312 26.02-26.576 24.675-10.51 53.161.893 63.656 25.495s-.992 53.043-25.661 63.567c-12.369 5.276-25.69 5.029-37.226.388-23.011-9.178-36.397-3.452-45.687 19.572-4.645 11.512-13.689 21.251-26.016 26.51-24.655 10.518-53.141-.886-63.63-25.474s.986-53.057 25.64-63.575"
                        }),
                        /*#__PURE__*/ (0, $c4MBb$reactjsxruntime.jsx)("circle", {
                            cx: 84.694,
                            cy: 21.472,
                            r: 36.14,
                            fill: "url(#aa0643__g)",
                            transform: "rotate(-.08 84.694 21.472)"
                        })
                    ]
                })
            })
        ]
    });
const $c9c538b9d9682143$var$ForwardRef = /*#__PURE__*/ (0, $c4MBb$react.forwardRef)($c9c538b9d9682143$var$SvgComponent);
var $c9c538b9d9682143$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c9c538b9d9682143$var$ForwardRef);
