// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_buildings_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $hxLF9$reactjsxruntime = require("react/jsx-runtime");
var $hxLF9$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $840214519690970e$export$2e2bcd8739ae039);




const $840214519690970e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "275b76__d",
                        x1: -8.603,
                        x2: 58.43,
                        y1: 41.325,
                        y2: -1.892,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "275b76__e",
                        x1: 46.977,
                        x2: 19.103,
                        y1: 12.805,
                        y2: 44.567,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "275b76__f",
                        x1: 10.87,
                        x2: 62.039,
                        y1: 47.115,
                        y2: 16.74,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsxs)("radialGradient", {
                        id: "275b76__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.4892 -18.5333 0 18.533 17.49)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsxs)("radialGradient", {
                        id: "275b76__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.4892 -18.5333 0 18.533 17.49)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("clipPath", {
                        id: "275b76__b",
                        children: /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("rect", {
                            width: 42.6,
                            height: 40.2,
                            x: 2.102,
                            y: 5.4,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("clipPath", {
                id: "275b76__a",
                children: /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M10.726 8.379h14.42c2.653 0 4.8 2.148 4.8 4.8v26.439H22.15v-6.174c0-1.326-1.074-2.4-2.4-2.4h-3.627c-1.326 0-2.4 1.074-2.4 2.4v6.174H5.926v-26.44c0-2.651 2.148-4.8 4.8-4.8m3.837 16.968c.663 0 1.2-.537 1.2-1.2v-1.728c0-.663-.537-1.2-1.2-1.2h-1.728c-.663 0-1.2.537-1.2 1.2v1.728c0 .663.537 1.2 1.2 1.2zm0-8.418c.663 0 1.2-.537 1.2-1.2V14c0-.663-.537-1.2-1.2-1.2h-1.728c-.663 0-1.2.537-1.2 1.2v1.728c0 .663.537 1.2 1.2 1.2zm8.478 8.418c.663 0 1.2-.537 1.2-1.2v-1.728c0-.663-.537-1.2-1.2-1.2h-1.728c-.663 0-1.2.537-1.2 1.2v1.728c0 .663.537 1.2 1.2 1.2zm0-8.418c.663 0 1.2-.537 1.2-1.2V14c0-.663-.537-1.2-1.2-1.2h-1.728c-.663 0-1.2.537-1.2 1.2v1.728c0 .663.537 1.2 1.2 1.2zm14.232-2.061c2.652 0 4.8 2.148 4.8 4.8v19.95H32.35v-24.75zm-.4 18.648c.664 0 1.2-.537 1.2-1.2v-1.728c0-.663-.536-1.2-1.2-1.2h-1.727c-.663 0-1.2.537-1.2 1.2v1.728c0 .663.537 1.2 1.2 1.2zm0-8.418c.664 0 1.2-.537 1.2-1.2V22.17c0-.663-.536-1.2-1.2-1.2h-1.727c-.663 0-1.2.537-1.2 1.2v1.728c0 .663.537 1.2 1.2 1.2z"
                })
            }),
            /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("g", {
                clipPath: "url(#275b76__a)",
                children: /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#275b76__b)",
                    children: [
                        /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("rect", {
                            width: 42.6,
                            height: 40.2,
                            x: 2.102,
                            y: 5.4,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M2.102 5.4h42.6v40.2h-42.6z"
                        }),
                        /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("ellipse", {
                            cx: 18.533,
                            cy: 17.489,
                            fill: "url(#275b76__c)",
                            rx: 18.533,
                            ry: 17.489,
                            transform: "matrix(1 -.00132 .00148 1 7.629 -10.505)"
                        }),
                        /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("path", {
                            fill: "url(#275b76__d)",
                            d: "M18.363-1.7c-3.272-1.911-7.417-1.923-10.701-.03l-32.844 18.923c-3.217 1.853-3.222 6.228-.01 8.088L7.435 44.178c3.28 1.899 7.424 1.897 10.7-.007l32.56-18.914c3.195-1.857 3.201-6.207.01-8.07z"
                        }),
                        /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("path", {
                            fill: "url(#275b76__e)",
                            d: "M8.241 24.486C8.97 9.478 22.452-2.13 38.356-1.444s28.206 13.41 27.478 28.418-14.211 26.617-30.115 25.93S7.513 39.494 8.241 24.486"
                        }),
                        /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("path", {
                            fill: "url(#275b76__f)",
                            d: "M-15.156 34.383c6.322-2.546 13.14-2.436 19.047-.21 11.82 4.465 18.697 1.704 23.468-9.424 2.376-5.584 7.015-10.314 13.344-12.862 12.653-5.086 27.262.433 32.644 12.339 5.382 11.905-.509 25.669-13.16 30.761-6.343 2.554-13.174 2.434-19.09.188-11.8-4.441-18.665-1.67-23.43 9.472-2.381 5.57-7.02 10.284-13.341 12.829-12.644 5.09-27.252-.43-32.631-12.328s.506-25.676 13.15-30.765"
                        }),
                        /*#__PURE__*/ (0, $hxLF9$reactjsxruntime.jsx)("ellipse", {
                            cx: 18.533,
                            cy: 17.489,
                            fill: "url(#275b76__g)",
                            rx: 18.533,
                            ry: 17.489,
                            transform: "matrix(1 -.00132 .00148 1 23.66 -4.802)"
                        })
                    ]
                })
            })
        ]
    });
const $840214519690970e$var$ForwardRef = /*#__PURE__*/ (0, $hxLF9$react.forwardRef)($840214519690970e$var$SvgComponent);
var $840214519690970e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($840214519690970e$var$ForwardRef);
