// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_globe_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $lF6sX$reactjsxruntime = require("react/jsx-runtime");
var $lF6sX$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3d9f93ad20ef11f1$export$2e2bcd8739ae039);




const $3d9f93ad20ef11f1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsxs)("linearGradient", {
                        id: "06859c__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsxs)("linearGradient", {
                        id: "06859c__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsxs)("linearGradient", {
                        id: "06859c__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsxs)("radialGradient", {
                        id: "06859c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsxs)("radialGradient", {
                        id: "06859c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("clipPath", {
                        id: "06859c__b",
                        children: /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("clipPath", {
                id: "06859c__a",
                children: /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m29.252 8.222-.347-.112q-.049.136-.13.261c-.337.516-.925 1.324-1.715 2.015-.777.68-1.884 1.364-3.233 1.364-2.736 0-4.442-2.084-5.106-3.323q-.045-.083-.076-.17c-1.19.408-2.317.948-3.367 1.603q.057.106.093.226c.146.487.529 1.293 1.037 2.018.25.357.508.662.752.886.255.233.43.32.511.343.578.165 1.31.14 2.267.11l.186-.006c.475-.015 1.01-.027 1.511.025.492.05 1.084.171 1.603.518.982.655 1.345 1.705 1.176 2.692-.16.93-.774 1.757-1.626 2.23-.478.266-.769.502-.94.722-.148.19-.237.406-.237.734 0 .65-.078 1.546-.707 2.21-.67.707-1.617.805-2.467.734l-.274-.023c-.707-.061-1.45-.126-2.136-.054-.794.083-1.12.31-1.254.543-.162.285-.323.723-.286 1.196.033.422.234 1.026 1.073 1.697.9.72 1.892.946 4.09 1.19 1.423.158 2.59.494 3.47 1.114.93.658 1.42 1.55 1.621 2.552.32 1.602-.231 3.464-.834 5.02-.315.813-.676 1.616-1.013 2.35l-.233.505q-.218.47-.407.886.861.09 1.746.091c3.896 0 7.477-1.35 10.3-3.61l-.001-.064c0-.325.02-.748.04-1.189l.016-.347c.026-.585.048-1.22.039-1.847-.01-.632-.053-1.226-.146-1.733-.095-.523-.229-.858-.357-1.043-.388-.56-.765-.777-1.308-1.03q-.093-.045-.197-.09c-.567-.259-1.35-.615-2.112-1.377-.51-.51-2.46-2.88-1.016-6.778.524-1.415 1.76-2.113 2.96-2.438 1.197-.324 2.571-.346 3.807-.262 1.253.085 2.457.286 3.351.462l.427.087c-.37-1.232-.88-2.402-1.512-3.494-.522.532-1.206 1.061-2.006 1.458-1.175.582-2.677.913-4.294.436-2.183-.643-2.87-2.645-3.045-4.332-.177-1.712.102-3.618.297-4.904zm11.119 13.57-.127-.026-.22-.05-.102-.023c-.259-.057-.61-.135-1.028-.217-.84-.165-1.93-.345-3.039-.42-1.125-.077-2.179-.038-2.983.18-.801.217-1.14.543-1.27.894-.983 2.653.332 4.033.44 4.141.429.429.817.608 1.385.87l.214.1c.719.335 1.554.787 2.306 1.872.417.603.636 1.335.76 2.015.128.696.175 1.442.186 2.144.005.304.003.606-.003.898 2.26-2.823 3.61-6.404 3.61-10.3q0-1.057-.13-2.078M7.5 23.869c0-4.978 2.204-9.44 5.69-12.466.274.676.688 1.447 1.171 2.136.32.458.695.913 1.11 1.293.404.37.915.733 1.513.904.98.28 2.114.24 2.96.208l.26-.009c.48-.015.862-.019 1.177.014.323.033.444.093.47.11.078.053.093.089.095.095.004.01.013.038.004.095-.023.13-.137.335-.376.469-.615.34-1.234.773-1.7 1.371-.488.629-.763 1.384-.763 2.269 0 .22-.015.363-.034.453-.082.01-.219.018-.432 0l-.252-.022c-.682-.06-1.682-.15-2.626-.05-1.05.11-2.422.49-3.165 1.79-.322.564-.696 1.506-.607 2.632.093 1.178.68 2.394 2.004 3.452 1.526 1.222 3.204 1.482 5.374 1.723 1.232.137 1.92.4 2.305.672.333.235.518.53.611 1 .165.825-.113 2.076-.713 3.625-.286.738-.62 1.484-.954 2.21l-.218.474c-.241.52-.478 1.032-.676 1.494C12.686 37.929 7.5 31.505 7.5 23.869m29.33-10.376c-1.418-1.751-3.188-3.207-5.202-4.258-.17 1.195-.314 2.532-.195 3.687.142 1.374.584 1.99 1.265 2.19.87.256 1.715.1 2.476-.278.75-.372 1.342-.924 1.656-1.34M24 7.37q1.177.001 2.31.16c-.246.323-.55.673-.897.976-.561.491-1.1.745-1.587.745-1.186 0-2.129-.82-2.679-1.635Q22.54 7.37 24 7.369"
                })
            }),
            /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("g", {
                clipPath: "url(#06859c__a)",
                children: /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#06859c__b)",
                    children: [
                        /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#06859c__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("path", {
                            fill: "url(#06859c__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("path", {
                            fill: "url(#06859c__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("path", {
                            fill: "url(#06859c__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $lF6sX$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#06859c__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $3d9f93ad20ef11f1$var$ForwardRef = /*#__PURE__*/ (0, $lF6sX$react.forwardRef)($3d9f93ad20ef11f1$var$SvgComponent);
var $3d9f93ad20ef11f1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3d9f93ad20ef11f1$var$ForwardRef);
