// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_AIGenerateImage_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $hAEwo$jsxs, jsx as $hAEwo$jsx} from "react/jsx-runtime";
import {forwardRef as $hAEwo$forwardRef} from "react";





const $59ba4b20d790f8d2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hAEwo$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hAEwo$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hAEwo$jsxs)("linearGradient", {
                        id: "8aa66b__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hAEwo$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $hAEwo$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hAEwo$jsxs)("linearGradient", {
                        id: "8aa66b__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hAEwo$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hAEwo$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hAEwo$jsxs)("linearGradient", {
                        id: "8aa66b__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hAEwo$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hAEwo$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hAEwo$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hAEwo$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hAEwo$jsxs)("radialGradient", {
                        id: "8aa66b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hAEwo$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $hAEwo$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hAEwo$jsxs)("radialGradient", {
                        id: "8aa66b__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hAEwo$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hAEwo$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hAEwo$jsx)("clipPath", {
                        id: "8aa66b__b",
                        children: /*#__PURE__*/ (0, $hAEwo$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hAEwo$jsx)("clipPath", {
                id: "8aa66b__a",
                children: /*#__PURE__*/ (0, $hAEwo$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m39.729 13.728 3.117.701c1.385.311 2.343-1.348 1.381-2.39l-2.165-2.35c-.322-.349-.452-.835-.347-1.298l.701-3.117c.311-1.385-1.348-2.343-2.39-1.381l-2.35 2.165c-.35.322-.835.452-1.298.347l-3.117-.701c-1.385-.311-2.343 1.348-1.381 2.39l2.165 2.35c.322.35.452.835.347 1.298l-.701 3.117c-.311 1.385 1.348 2.343 2.39 1.381l2.35-2.165c.349-.322.835-.452 1.298-.347M5.696 18.217c1.561-5.827 7.551-9.286 13.379-7.724 2.53.678 4.61 2.188 6.031 4.16.404.56 1.185.686 1.745.283s.687-1.185.283-1.745c-1.744-2.421-4.306-4.28-7.412-5.113C12.56 6.159 5.2 10.408 3.282 17.57c-1.222 4.556.054 9.19 2.986 12.454l-2.063-.007c-.69-.002-1.252.556-1.254 1.247-.002.69.555 1.251 1.246 1.253l4.948.016c.673.002 1.228-.53 1.253-1.203l.18-4.773c.026-.69-.512-1.27-1.202-1.297s-1.27.512-1.296 1.202l-.067 1.763c-2.306-2.65-3.294-6.362-2.317-10.008m26.34 2.414.056.06v.001l.422.457q.112.124.171.26c.277.633-.277 1.405-1.033 1.235l-.61-.138-.681-.153-.174-.04-.733-.165c-.29-.065-.592.017-.811.217l-.552.51-.131.12-.515.475-.001.001-.457.421c-.652.602-1.69.002-1.494-.862l.137-.609.154-.682.039-.174.165-.732q.022-.101.022-.202c0-.224-.085-.442-.239-.61l-1.527-1.655c-.602-.653-.002-1.69.862-1.495l2.198.495c.29.065.593-.016.811-.216l1.656-1.528c.652-.601 1.69-.001 1.494.862l-.495 2.198c-.065.291.016.593.217.812zm3.04 0h4.313c2.21 0 4 1.79 4 4v14.474c0 .675-.167 1.311-.462 1.869l-4.843-7.194-.001-.001c-1.181-1.762-3.77-1.758-4.95-.001l-.006.008-2.614 3.955-4.81-7.43c-1.174-1.815-3.831-1.815-5.005-.002l-.001.002-7.11 10.931V24.631c0-2.21 1.79-4 4-4h5.345l.433.47-.31 1.378v.001h-.001c-.735 3.26 3.175 5.513 5.629 3.25l1.039-.959 1.38.311c2.773.626 4.817-2.11 3.973-4.451m-19.72 22.475h11.612l2.04-3.089-5.404-8.348c-.189-.293-.62-.293-.808 0l-.002.002zm20.652-7.932 5.085 7.552c-.517.243-1.095.38-1.704.38h-9.425l5.245-7.936c.193-.284.61-.28.797 0zm-.007-6.424c-.967 0-1.75-.784-1.75-1.75s.783-1.75 1.75-1.75c.966 0 1.75.784 1.75 1.75s-.784 1.75-1.75 1.75"
                })
            }),
            /*#__PURE__*/ (0, $hAEwo$jsx)("g", {
                clipPath: "url(#8aa66b__a)",
                children: /*#__PURE__*/ (0, $hAEwo$jsxs)("g", {
                    clipPath: "url(#8aa66b__b)",
                    children: [
                        /*#__PURE__*/ (0, $hAEwo$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $hAEwo$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $hAEwo$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#8aa66b__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $hAEwo$jsx)("path", {
                            fill: "url(#8aa66b__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $hAEwo$jsx)("path", {
                            fill: "url(#8aa66b__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $hAEwo$jsx)("path", {
                            fill: "url(#8aa66b__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $hAEwo$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#8aa66b__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $59ba4b20d790f8d2$var$ForwardRef = /*#__PURE__*/ (0, $hAEwo$forwardRef)($59ba4b20d790f8d2$var$SvgComponent);
var $59ba4b20d790f8d2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($59ba4b20d790f8d2$var$ForwardRef);


export {$59ba4b20d790f8d2$export$2e2bcd8739ae039 as default};
