// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_brand_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $92AQ3$reactjsxruntime = require("react/jsx-runtime");
var $92AQ3$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c2d094db041b4277$export$2e2bcd8739ae039);




const $c2d094db041b4277$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsxs)("linearGradient", {
                        id: "de1123__d",
                        x1: -5.676,
                        x2: 56.126,
                        y1: 38.508,
                        y2: -1.379,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsxs)("linearGradient", {
                        id: "de1123__e",
                        x1: 45.597,
                        x2: 19.913,
                        y1: 12.224,
                        y2: 41.522,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsxs)("linearGradient", {
                        id: "de1123__f",
                        x1: 12.287,
                        x2: 59.466,
                        y1: 43.844,
                        y2: 15.807,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsxs)("radialGradient", {
                        id: "de1123__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsxs)("radialGradient", {
                        id: "de1123__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("clipPath", {
                        id: "de1123__b",
                        children: /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 5.4,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("clipPath", {
                id: "de1123__a",
                children: /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M13.725 7.818h20.55c1.989 0 3.6 1.611 3.6 3.6v14.89c0 7.664-6.21 13.874-13.875 13.874S10.125 33.97 10.125 26.307V11.418c0-1.989 1.611-3.6 3.6-3.6m1.224 19.725c.188.111.395.162.598.162.417 0 .818-.216 1.04-.6l4.188-7.254c.333-.573.135-1.308-.438-1.638-.573-.333-1.308-.135-1.638.438l-4.188 7.254c-.333.573-.135 1.308.438 1.638m15.756.162c.867 0 1.644-.45 2.079-1.2.435-.753.435-1.65 0-2.4l-3.771-6.53c-.435-.75-1.212-1.2-2.079-1.2s-1.644.45-2.079 1.2l-3.771 6.53c-.435.75-.435 1.65 0 2.4s1.212 1.2 2.079 1.2zm-3.77-8.934 3.77 6.534h-7.542z"
                })
            }),
            /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("g", {
                clipPath: "url(#de1123__a)",
                children: /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#de1123__b)",
                    children: [
                        /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 5.4,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M4.2 5.4h39.3v37.047H4.2z"
                        }),
                        /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#de1123__c)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 9.3 -9.258)"
                        }),
                        /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("path", {
                            fill: "url(#de1123__d)",
                            d: "M19.2-1.144c-3.017-1.76-6.841-1.771-9.87-.028l-30.3 17.44c-2.968 1.708-2.973 5.74-.01 7.454l30.1 17.415c3.025 1.75 6.849 1.748 9.871-.006L49.03 23.7c2.947-1.711 2.953-5.72.009-7.437z"
                        }),
                        /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("path", {
                            fill: "url(#de1123__e)",
                            d: "M9.862 22.99C10.534 9.157 22.972-1.54 37.644-.908s26.021 12.358 25.35 26.19c-.672 13.83-13.11 24.529-27.782 23.896C20.54 48.545 9.19 36.82 9.862 22.989"
                        }),
                        /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("path", {
                            fill: "url(#de1123__f)",
                            d: "M-11.723 32.11c5.832-2.346 12.122-2.246 17.571-.194 10.905 4.115 17.25 1.57 21.65-8.685 2.192-5.146 6.472-9.505 12.31-11.853 11.674-4.687 25.15.399 30.116 11.37 4.965 10.973-.47 23.657-12.14 28.35-5.852 2.353-12.154 2.243-17.612.173C29.286 47.178 22.953 49.732 18.558 60c-2.198 5.134-6.476 9.477-12.309 11.823-11.664 4.69-25.14-.395-30.103-11.361-4.962-10.966.467-23.662 12.131-28.353"
                        }),
                        /*#__PURE__*/ (0, $92AQ3$reactjsxruntime.jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#de1123__g)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 24.086 -4.002)"
                        })
                    ]
                })
            })
        ]
    });
const $c2d094db041b4277$var$ForwardRef = /*#__PURE__*/ (0, $92AQ3$react.forwardRef)($c2d094db041b4277$var$SvgComponent);
var $c2d094db041b4277$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c2d094db041b4277$var$ForwardRef);
