// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_imageStack_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $AWctw$jsxs, jsx as $AWctw$jsx} from "react/jsx-runtime";
import {forwardRef as $AWctw$forwardRef} from "react";





const $d471f76329b700eb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $AWctw$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $AWctw$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $AWctw$jsxs)("linearGradient", {
                        id: "f10493__d",
                        x1: -4.4,
                        x2: 57.685,
                        y1: 39.37,
                        y2: -0.699,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $AWctw$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $AWctw$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $AWctw$jsxs)("linearGradient", {
                        id: "f10493__e",
                        x1: 47.109,
                        x2: 21.307,
                        y1: 12.967,
                        y2: 42.398,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $AWctw$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $AWctw$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $AWctw$jsxs)("linearGradient", {
                        id: "f10493__f",
                        x1: 13.643,
                        x2: 61.038,
                        y1: 44.73,
                        y2: 16.566,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $AWctw$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $AWctw$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $AWctw$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $AWctw$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $AWctw$jsxs)("radialGradient", {
                        id: "f10493__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1913 -17.1759 0 17.176 16.191)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $AWctw$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $AWctw$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $AWctw$jsxs)("radialGradient", {
                        id: "f10493__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1913 -17.1759 0 17.176 16.191)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $AWctw$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $AWctw$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $AWctw$jsx)("clipPath", {
                        id: "f10493__b",
                        children: /*#__PURE__*/ (0, $AWctw$jsx)("rect", {
                            width: 39.48,
                            height: 37.217,
                            x: 5.52,
                            y: 6.111,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $AWctw$jsx)("clipPath", {
                id: "f10493__a",
                children: /*#__PURE__*/ (0, $AWctw$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M37.628 14.541v14.7c0 2.652-2.148 4.8-4.8 4.8h-22.2c-2.652 0-4.8-2.148-4.8-4.8v-14.7c0-2.652 2.148-4.8 4.8-4.8h22.2c2.652 0 4.8 2.148 4.8 4.8m-4.212 2.064c0-1.446-1.173-2.619-2.619-2.619s-2.619 1.173-2.619 2.62c0 1.445 1.173 2.618 2.62 2.618 1.445 0 2.618-1.173 2.618-2.619M8.693 28.866c-.768 1.197.09 2.772 1.515 2.772v-.003h22.854c1.425 0 2.283-1.575 1.515-2.772l-3.534-5.505c-.708-1.104-2.322-1.104-3.03 0l-4.008 6.246-6.378-9.936c-.708-1.104-2.322-1.104-3.03 0zm32.277-8.919c0-.663.537-1.2 1.2-1.2h.003c.663 0 1.2.537 1.2 1.2v12.315c0 3.97-3.23 7.2-7.2 7.2l-19.782-.015c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.2-1.2l19.78.015c2.645 0 4.8-2.154 4.8-4.8z"
                })
            }),
            /*#__PURE__*/ (0, $AWctw$jsx)("g", {
                clipPath: "url(#f10493__a)",
                children: /*#__PURE__*/ (0, $AWctw$jsxs)("g", {
                    clipPath: "url(#f10493__b)",
                    children: [
                        /*#__PURE__*/ (0, $AWctw$jsx)("rect", {
                            width: 39.48,
                            height: 37.217,
                            x: 5.52,
                            y: 6.111,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $AWctw$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M5.52 6.111H45v37.217H5.52z"
                        }),
                        /*#__PURE__*/ (0, $AWctw$jsx)("ellipse", {
                            cx: 17.176,
                            cy: 16.191,
                            fill: "url(#f10493__c)",
                            rx: 17.176,
                            ry: 16.191,
                            transform: "matrix(1 -.00132 .00148 1 10.645 -8.614)"
                        }),
                        /*#__PURE__*/ (0, $AWctw$jsx)("path", {
                            fill: "url(#f10493__d)",
                            d: "M20.59-.463c-3.032-1.768-6.873-1.78-9.917-.028l-30.438 17.52c-2.981 1.715-2.986 5.765-.01 7.488L10.464 42.01c3.039 1.758 6.88 1.756 9.917-.006l30.174-17.51c2.962-1.72 2.967-5.747.01-7.472z"
                        }),
                        /*#__PURE__*/ (0, $AWctw$jsx)("path", {
                            fill: "url(#f10493__e)",
                            d: "M11.21 23.781C11.885 9.887 24.38-.86 39.12-.225c14.739.637 26.14 12.416 25.465 26.31S51.415 50.727 36.676 50.09c-14.739-.636-26.14-12.415-25.466-26.309"
                        }),
                        /*#__PURE__*/ (0, $AWctw$jsx)("path", {
                            fill: "url(#f10493__f)",
                            d: "M-10.477 32.942c5.859-2.356 12.177-2.255 17.652-.194 10.954 4.134 17.328 1.578 21.748-8.725 2.203-5.17 6.502-9.548 12.367-11.907 11.727-4.708 25.265.4 30.253 11.423s-.471 23.764-12.196 28.479c-5.878 2.364-12.209 2.253-17.691.174-10.937-4.112-17.299-1.547-21.714 8.768-2.207 5.158-6.506 9.521-12.365 11.877-11.717 4.713-25.255-.397-30.24-11.413-4.986-11.015.468-23.77 12.186-28.482"
                        }),
                        /*#__PURE__*/ (0, $AWctw$jsx)("ellipse", {
                            cx: 17.176,
                            cy: 16.191,
                            fill: "url(#f10493__g)",
                            rx: 17.176,
                            ry: 16.191,
                            transform: "matrix(1 -.00132 .00148 1 25.5 -3.334)"
                        })
                    ]
                })
            })
        ]
    });
const $d471f76329b700eb$var$ForwardRef = /*#__PURE__*/ (0, $AWctw$forwardRef)($d471f76329b700eb$var$SvgComponent);
var $d471f76329b700eb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d471f76329b700eb$var$ForwardRef);


export {$d471f76329b700eb$export$2e2bcd8739ae039 as default};
