// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_dataAnalytics_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $8l7jn$jsxs, jsx as $8l7jn$jsx} from "react/jsx-runtime";
import {forwardRef as $8l7jn$forwardRef} from "react";





const $21e07b05fabb6e7f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8l7jn$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8l7jn$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8l7jn$jsxs)("linearGradient", {
                        id: "ec1bf2__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8l7jn$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8l7jn$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8l7jn$jsxs)("linearGradient", {
                        id: "ec1bf2__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8l7jn$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8l7jn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8l7jn$jsxs)("linearGradient", {
                        id: "ec1bf2__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8l7jn$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8l7jn$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8l7jn$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8l7jn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8l7jn$jsxs)("radialGradient", {
                        id: "ec1bf2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8l7jn$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8l7jn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8l7jn$jsxs)("radialGradient", {
                        id: "ec1bf2__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8l7jn$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8l7jn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8l7jn$jsx)("clipPath", {
                        id: "ec1bf2__b",
                        children: /*#__PURE__*/ (0, $8l7jn$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8l7jn$jsx)("clipPath", {
                id: "ec1bf2__a",
                children: /*#__PURE__*/ (0, $8l7jn$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M135.135 32.355c1.371 1.733 1.078 4.249-.655 5.62L94.276 69.78c-1.238.98-2.936 1.139-4.334.408l-25.31-13.226-34.04 28.873c-1.685 1.428-4.21 1.221-5.639-.464-1.428-1.684-1.221-4.209.464-5.637L61.51 49.119c1.241-1.053 2.997-1.249 4.44-.495l25.396 13.272L129.516 31.7c1.733-1.37 4.249-1.077 5.619.655M118.043 72.51c0-3.798 3.079-6.877 6.877-6.877s6.877 3.079 6.877 6.877v45.777c0 3.798-3.079 6.877-6.877 6.877s-6.877-3.079-6.877-6.877zM58.158 87.515c0-3.798 3.08-6.877 6.877-6.877 3.798 0 6.877 3.079 6.877 6.877v30.771c0 3.798-3.079 6.877-6.877 6.877s-6.877-3.079-6.877-6.877zm36.82-.916c-3.798 0-6.877 3.079-6.877 6.877v24.81c0 3.798 3.079 6.877 6.877 6.877s6.877-3.079 6.877-6.877v-24.81c0-3.798-3.08-6.877-6.877-6.877m-59.885 11.79c-3.798 0-6.877 3.08-6.877 6.878v13.019c0 3.798 3.079 6.877 6.877 6.877s6.877-3.079 6.877-6.877v-13.019c0-3.798-3.08-6.877-6.877-6.877"
                })
            }),
            /*#__PURE__*/ (0, $8l7jn$jsx)("g", {
                clipPath: "url(#ec1bf2__a)",
                children: /*#__PURE__*/ (0, $8l7jn$jsxs)("g", {
                    clipPath: "url(#ec1bf2__b)",
                    children: [
                        /*#__PURE__*/ (0, $8l7jn$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $8l7jn$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $8l7jn$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#ec1bf2__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $8l7jn$jsx)("path", {
                            fill: "url(#ec1bf2__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $8l7jn$jsx)("path", {
                            fill: "url(#ec1bf2__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $8l7jn$jsx)("path", {
                            fill: "url(#ec1bf2__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $8l7jn$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#ec1bf2__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $21e07b05fabb6e7f$var$ForwardRef = /*#__PURE__*/ (0, $8l7jn$forwardRef)($21e07b05fabb6e7f$var$SvgComponent);
var $21e07b05fabb6e7f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($21e07b05fabb6e7f$var$ForwardRef);


export {$21e07b05fabb6e7f$export$2e2bcd8739ae039 as default};
