// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_paperairplane_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $gnDai$jsxs, jsx as $gnDai$jsx} from "react/jsx-runtime";
import {forwardRef as $gnDai$forwardRef} from "react";





const $776b6db67b6f8efd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gnDai$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gnDai$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gnDai$jsxs)("linearGradient", {
                        id: "dff60e__d",
                        x1: -6.877,
                        x2: 58.319,
                        y1: 39.643,
                        y2: -2.434,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gnDai$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gnDai$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gnDai$jsxs)("linearGradient", {
                        id: "dff60e__e",
                        x1: 47.216,
                        x2: 20.121,
                        y1: 11.917,
                        y2: 42.823,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gnDai$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gnDai$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gnDai$jsxs)("linearGradient", {
                        id: "dff60e__f",
                        x1: 12.076,
                        x2: 61.847,
                        y1: 45.272,
                        y2: 15.696,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gnDai$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gnDai$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gnDai$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gnDai$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gnDai$jsxs)("radialGradient", {
                        id: "dff60e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.0028 -18.0367 0 18.037 17.003)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gnDai$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gnDai$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gnDai$jsxs)("radialGradient", {
                        id: "dff60e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 17.0028 -18.0367 0 18.037 17.003)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gnDai$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gnDai$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gnDai$jsx)("clipPath", {
                        id: "dff60e__b",
                        children: /*#__PURE__*/ (0, $gnDai$jsx)("rect", {
                            width: 41.459,
                            height: 39.082,
                            x: 3.543,
                            y: 4.718,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gnDai$jsx)("clipPath", {
                id: "dff60e__a",
                children: /*#__PURE__*/ (0, $gnDai$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M41.481 11.755c.493-.11.96-.048 1.36.134L11.746 28.035q-.147.077-.263.185l-6.658-4.555c-1.535-1.05-1.077-3.412.74-3.822zm-13.034 28.07L13.69 29.73l30.361-15.765q.023-.011.044-.024c-.016.279-.09.564-.23.842L31.57 39.031c-.584 1.15-2.056 1.525-3.124.794m-16.149-.43V31.78l7.048 4.802-4.66 3.877c-.938.78-2.388.134-2.388-1.064"
                })
            }),
            /*#__PURE__*/ (0, $gnDai$jsx)("g", {
                clipPath: "url(#dff60e__a)",
                children: /*#__PURE__*/ (0, $gnDai$jsxs)("g", {
                    clipPath: "url(#dff60e__b)",
                    children: [
                        /*#__PURE__*/ (0, $gnDai$jsx)("rect", {
                            width: 41.459,
                            height: 39.082,
                            x: 3.543,
                            y: 4.718,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $gnDai$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.543 4.718h41.459V43.8H3.543z"
                        }),
                        /*#__PURE__*/ (0, $gnDai$jsx)("ellipse", {
                            cx: 18.037,
                            cy: 17.003,
                            fill: "url(#dff60e__c)",
                            rx: 18.037,
                            ry: 17.003,
                            transform: "matrix(1 -.00132 .00148 1 8.922 -10.745)"
                        }),
                        /*#__PURE__*/ (0, $gnDai$jsx)("path", {
                            fill: "url(#dff60e__d)",
                            d: "M19.366-2.186c-3.184-1.857-7.219-1.869-10.414-.03l-31.964 18.398c-3.13 1.802-3.136 6.055-.01 7.864l31.753 18.37c3.19 1.847 7.225 1.845 10.413-.006l31.688-18.388c3.11-1.805 3.115-6.034.01-7.846z"
                        }),
                        /*#__PURE__*/ (0, $gnDai$jsx)("path", {
                            fill: "url(#dff60e__e)",
                            d: "M9.518 23.273c.71-14.59 13.83-25.877 29.308-25.209s27.45 13.038 26.742 27.628-13.83 25.877-29.308 25.209S8.81 37.863 9.518 23.273"
                        }),
                        /*#__PURE__*/ (0, $gnDai$jsx)("path", {
                            fill: "url(#dff60e__f)",
                            d: "M-13.252 32.893c6.152-2.474 12.787-2.368 18.536-.204 11.503 4.341 18.196 1.657 22.839-9.162 2.312-5.429 6.827-10.026 12.986-12.503 12.315-4.945 26.532.42 31.77 11.995s-.496 24.955-12.807 29.906c-6.173 2.482-12.821 2.366-18.579.183-11.484-4.318-18.165-1.624-22.802 9.208-2.318 5.416-6.831 9.998-12.984 12.472-12.305 4.948-26.521-.417-31.757-11.985-5.235-11.568.493-24.962 12.798-29.91"
                        }),
                        /*#__PURE__*/ (0, $gnDai$jsx)("ellipse", {
                            cx: 18.037,
                            cy: 17.003,
                            fill: "url(#dff60e__g)",
                            rx: 18.037,
                            ry: 17.003,
                            transform: "matrix(1 -.00132 .00148 1 24.523 -5.2)"
                        })
                    ]
                })
            })
        ]
    });
const $776b6db67b6f8efd$var$ForwardRef = /*#__PURE__*/ (0, $gnDai$forwardRef)($776b6db67b6f8efd$var$SvgComponent);
var $776b6db67b6f8efd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($776b6db67b6f8efd$var$ForwardRef);


export {$776b6db67b6f8efd$export$2e2bcd8739ae039 as default};
