// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_market_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $aYR8N$jsxs, jsx as $aYR8N$jsx} from "react/jsx-runtime";
import {forwardRef as $aYR8N$forwardRef} from "react";





const $d47c98b04fb03dce$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aYR8N$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aYR8N$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aYR8N$jsxs)("linearGradient", {
                        id: "b20d79__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYR8N$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $aYR8N$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aYR8N$jsxs)("linearGradient", {
                        id: "b20d79__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYR8N$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aYR8N$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aYR8N$jsxs)("linearGradient", {
                        id: "b20d79__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYR8N$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aYR8N$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aYR8N$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $aYR8N$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aYR8N$jsxs)("radialGradient", {
                        id: "b20d79__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYR8N$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $aYR8N$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aYR8N$jsxs)("radialGradient", {
                        id: "b20d79__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYR8N$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aYR8N$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aYR8N$jsx)("clipPath", {
                        id: "b20d79__b",
                        children: /*#__PURE__*/ (0, $aYR8N$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aYR8N$jsx)("clipPath", {
                id: "b20d79__a",
                children: /*#__PURE__*/ (0, $aYR8N$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M34.9 81.148v34.62c0 8.836 7.163 16 16 16h11.703l-.005-50.751c-4.51-.767-8.463-3.18-11.207-6.601-3.357 4.184-8.522 6.861-14.305 6.861q-1.11 0-2.187-.129m35.698-.533.003 26.918h54.318V81.169q-.987.108-2.005.108c-5.783 0-10.948-2.677-14.304-6.861-3.357 4.184-8.522 6.861-14.305 6.861S83.357 78.6 80 74.416c-2.371 2.956-5.645 5.16-9.402 6.199m-4.903-7.338c-5.705 0-10.304-4.603-10.304-10.249v-.704c0-2.21-1.791-4-4-4s-4 1.79-4 4v.704c0 5.646-4.6 10.25-10.305 10.25-5.706 0-10.305-4.604-10.305-10.25 0-3.086.672-6.134 1.974-8.93l.001-.002 8.81-18.942v-.001h.001c.828-1.776 2.619-2.92 4.6-2.92h75.68c1.982 0 3.773 1.144 4.601 2.92l8.675 18.653q.054.146.121.288c1.3 2.801 1.975 5.852 1.975 8.934 0 5.646-4.599 10.25-10.305 10.25s-10.304-4.604-10.304-10.25v-.704c0-2.21-1.791-4-4-4s-4 1.79-4 4v.704c0 5.646-4.599 10.25-10.305 10.25S84 68.673 84 63.027v-.704c0-2.21-1.79-4-4-4s-4 1.79-4 4v.704c0 5.646-4.599 10.25-10.305 10.25m4.907 42.256h54.317v.235c0 8.836-7.163 16-16 16H70.603z"
                })
            }),
            /*#__PURE__*/ (0, $aYR8N$jsx)("g", {
                clipPath: "url(#b20d79__a)",
                children: /*#__PURE__*/ (0, $aYR8N$jsxs)("g", {
                    clipPath: "url(#b20d79__b)",
                    children: [
                        /*#__PURE__*/ (0, $aYR8N$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $aYR8N$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $aYR8N$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#b20d79__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $aYR8N$jsx)("path", {
                            fill: "url(#b20d79__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $aYR8N$jsx)("path", {
                            fill: "url(#b20d79__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $aYR8N$jsx)("path", {
                            fill: "url(#b20d79__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $aYR8N$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#b20d79__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $d47c98b04fb03dce$var$ForwardRef = /*#__PURE__*/ (0, $aYR8N$forwardRef)($d47c98b04fb03dce$var$SvgComponent);
var $d47c98b04fb03dce$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d47c98b04fb03dce$var$ForwardRef);


export {$d47c98b04fb03dce$export$2e2bcd8739ae039 as default};
