// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_heart_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lcNps$jsxs, jsx as $lcNps$jsx} from "react/jsx-runtime";
import {forwardRef as $lcNps$forwardRef} from "react";





const $6e41b260f59fa199$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lcNps$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lcNps$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lcNps$jsxs)("linearGradient", {
                        id: "4ac1b9__d",
                        x1: -21.177,
                        x2: 196.682,
                        y1: 128.748,
                        y2: -3.795,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lcNps$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lcNps$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lcNps$jsxs)("linearGradient", {
                        id: "4ac1b9__e",
                        x1: 153.655,
                        x2: 60.194,
                        y1: 33.682,
                        y2: 134.178,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lcNps$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lcNps$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lcNps$jsxs)("linearGradient", {
                        id: "4ac1b9__f",
                        x1: 40.078,
                        x2: 205.758,
                        y1: 148.049,
                        y2: 55.237,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lcNps$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lcNps$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lcNps$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lcNps$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lcNps$jsxs)("radialGradient", {
                        id: "4ac1b9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 37.034 51.231)scale(58.2973)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lcNps$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lcNps$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lcNps$jsxs)("radialGradient", {
                        id: "4ac1b9__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 52.741 85.95)scale(58.2973)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lcNps$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lcNps$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lcNps$jsx)("clipPath", {
                        id: "4ac1b9__b",
                        children: /*#__PURE__*/ (0, $lcNps$jsx)("rect", {
                            width: 134,
                            height: 134,
                            x: 12.5,
                            y: 9,
                            fill: "#fff",
                            rx: 4.547
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lcNps$jsx)("clipPath", {
                id: "4ac1b9__a",
                children: /*#__PURE__*/ (0, $lcNps$jsx)("path", {
                    fill: "#fff",
                    d: "M108.008 26c-15.463.01-27.996 12.66-27.996 28.262-.004.014-.008.045-.012.065-.004-.02-.008-.05-.012-.065 0-15.603-12.533-28.252-27.996-28.262-17.052-.01-29.247 14.7-27.889 31.848 2.458 31.016 35.866 65.837 49.824 77.019 3.549 2.843 8.573 2.844 12.123.001 13.964-11.181 47.39-46.005 49.847-77.023C137.255 40.698 125.059 25.99 108.008 26"
                })
            }),
            /*#__PURE__*/ (0, $lcNps$jsx)("g", {
                clipPath: "url(#4ac1b9__a)",
                children: /*#__PURE__*/ (0, $lcNps$jsxs)("g", {
                    clipPath: "url(#4ac1b9__b)",
                    children: [
                        /*#__PURE__*/ (0, $lcNps$jsx)("rect", {
                            width: 134,
                            height: 134,
                            x: 12.5,
                            y: 9,
                            fill: "#FFECCF",
                            rx: 4.547
                        }),
                        /*#__PURE__*/ (0, $lcNps$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M12.5 9h134v134h-134z"
                        }),
                        /*#__PURE__*/ (0, $lcNps$jsx)("circle", {
                            cx: 88.266,
                            cy: 14.197,
                            r: 58.297,
                            fill: "url(#4ac1b9__c)",
                            transform: "rotate(-.08 88.266 14.197)"
                        }),
                        /*#__PURE__*/ (0, $lcNps$jsx)("path", {
                            fill: "url(#4ac1b9__d)",
                            d: "M63.644-14.67c-10.291-6.368-23.33-6.407-33.66-.1L-73.326 48.307c-10.118 6.178-10.135 20.76-.03 26.961l102.629 62.99c10.313 6.329 23.352 6.322 33.657-.023l102.418-63.048c10.051-6.188 10.069-20.689.031-26.9z"
                        }),
                        /*#__PURE__*/ (0, $lcNps$jsx)("path", {
                            fill: "url(#4ac1b9__e)",
                            d: "M31.81 72.62c2.29-50.026 44.702-88.724 94.728-86.434s88.723 44.702 86.433 94.728-44.701 88.723-94.727 86.433S29.52 122.646 31.811 72.62"
                        }),
                        /*#__PURE__*/ (0, $lcNps$jsx)("path", {
                            fill: "url(#4ac1b9__f)",
                            d: "M-41.789 105.605c19.886-8.483 41.332-8.12 59.913-.699 37.18 14.885 58.813 5.682 73.818-31.413 7.474-18.614 22.066-34.378 41.974-42.871 39.803-16.954 85.754 1.442 102.684 41.127s-1.601 85.563-41.394 102.539c-19.953 8.512-41.44 8.114-60.049.627-37.12-14.806-58.713-5.568-73.699 31.571-7.492 18.57-22.081 34.28-41.967 42.764-39.77 16.966-85.721-1.43-102.642-41.093s1.591-85.585 41.362-102.552"
                        }),
                        /*#__PURE__*/ (0, $lcNps$jsx)("circle", {
                            cx: 138.691,
                            cy: 33.209,
                            r: 58.297,
                            fill: "url(#4ac1b9__g)",
                            transform: "rotate(-.08 138.691 33.209)"
                        })
                    ]
                })
            })
        ]
    });
const $6e41b260f59fa199$var$ForwardRef = /*#__PURE__*/ (0, $lcNps$forwardRef)($6e41b260f59fa199$var$SvgComponent);
var $6e41b260f59fa199$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6e41b260f59fa199$var$ForwardRef);


export {$6e41b260f59fa199$export$2e2bcd8739ae039 as default};
