// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_tablet_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $lg0ig$reactjsxruntime = require("react/jsx-runtime");
var $lg0ig$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $33861e6bf7fec9fd$export$2e2bcd8739ae039);




const $33861e6bf7fec9fd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4d7d2f__d",
                        x1: -25.185,
                        x2: 194.975,
                        y1: 134.352,
                        y2: -7.737,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4d7d2f__e",
                        x1: 157.478,
                        x2: 65.981,
                        y1: 40.722,
                        y2: 145.09,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4d7d2f__f",
                        x1: 38.811,
                        x2: 206.879,
                        y1: 153.36,
                        y2: 53.485,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4d7d2f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 57.4161 -60.9076 0 60.908 57.416)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4d7d2f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 57.4161 -60.9076 0 60.908 57.416)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("clipPath", {
                        id: "4d7d2f__b",
                        children: /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("rect", {
                            width: 140,
                            height: 131.975,
                            x: 10,
                            y: 16.413,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("clipPath", {
                id: "4d7d2f__a",
                children: /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M128 28H32c-8.84 0-16 7.16-16 16v72c0 8.84 7.16 16 16 16h96c8.84 0 16-7.16 16-16V44c0-8.84-7.16-16-16-16M34 84c-2.21 0-4-1.79-4-4s1.78-4 3.99-4H34c2.21 0 4 1.79 4 4s-1.79 4-4 4"
                })
            }),
            /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("g", {
                clipPath: "url(#4d7d2f__a)",
                children: /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#4d7d2f__b)",
                    children: [
                        /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("rect", {
                            width: 140,
                            height: 131.975,
                            x: 10,
                            y: 16.413,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M10 16.413h140v131.975H10z"
                        }),
                        /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("ellipse", {
                            cx: 60.908,
                            cy: 57.416,
                            fill: "url(#4d7d2f__c)",
                            rx: 60.908,
                            ry: 57.416,
                            transform: "matrix(1 -.00132 .00148 1 28.168 -35.804)"
                        }),
                        /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("path", {
                            fill: "url(#4d7d2f__d)",
                            d: "M63.434-6.899c-10.752-6.272-24.375-6.31-35.167-.1L-79.67 55.128c-10.571 6.085-10.59 20.446-.032 26.554l107.224 62.037c10.776 6.234 24.399 6.226 35.165-.022L169.69 81.601c10.501-6.095 10.521-20.376.033-26.494z"
                        }),
                        /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("path", {
                            fill: "url(#4d7d2f__e)",
                            d: "M30.178 79.071C32.57 29.801 76.88-8.31 129.146-6.055 181.412-3.8 221.843 37.97 219.45 87.24s-46.703 87.383-98.969 85.127-92.696-44.026-90.303-93.296"
                        }),
                        /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("path", {
                            fill: "url(#4d7d2f__f)",
                            d: "M-46.721 111.558c20.776-8.355 43.182-7.997 62.595-.689 38.844 14.66 61.447 5.596 77.124-30.938 7.808-18.332 23.054-33.858 43.853-42.223 41.585-16.697 89.593 1.42 107.281 40.506s-1.672 84.27-43.247 100.989c-20.846 8.383-43.296 7.991-62.738.617-38.781-14.582-61.34-5.484-76.998 31.094-7.828 18.289-23.07 33.762-43.846 42.117-41.552 16.71-89.56-1.408-107.238-40.471s1.662-84.292 43.214-101.002"
                        }),
                        /*#__PURE__*/ (0, $lg0ig$reactjsxruntime.jsx)("ellipse", {
                            cx: 60.908,
                            cy: 57.416,
                            fill: "url(#4d7d2f__g)",
                            rx: 60.908,
                            ry: 57.416,
                            transform: "matrix(1 -.00132 .00148 1 80.848 -17.08)"
                        })
                    ]
                })
            })
        ]
    });
const $33861e6bf7fec9fd$var$ForwardRef = /*#__PURE__*/ (0, $lg0ig$react.forwardRef)($33861e6bf7fec9fd$var$SvgComponent);
var $33861e6bf7fec9fd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($33861e6bf7fec9fd$var$ForwardRef);
