// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_graphBarChart_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $5ukbI$reactjsxruntime = require("react/jsx-runtime");
var $5ukbI$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5238c0d9a22d5d8a$export$2e2bcd8739ae039);




const $5238c0d9a22d5d8a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "afe6b0__d",
                        x1: -19.305,
                        x2: 116.82,
                        y1: 81.922,
                        y2: -5.932,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "afe6b0__e",
                        x1: 93.635,
                        x2: 37.062,
                        y1: 24.031,
                        y2: 88.561,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "afe6b0__f",
                        x1: 20.262,
                        x2: 124.179,
                        y1: 93.675,
                        y2: 31.922,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsxs)("radialGradient", {
                        id: "afe6b0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 35.5005 -37.6593 0 37.66 35.5)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsxs)("radialGradient", {
                        id: "afe6b0__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 35.5005 -37.6593 0 37.66 35.5)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("clipPath", {
                        id: "afe6b0__b",
                        children: /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("rect", {
                            width: 86.562,
                            height: 81.6,
                            x: 2.449,
                            y: 9,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("clipPath", {
                id: "afe6b0__a",
                children: /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M14.527 81.6H81.6c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4H12.127c-1.326 0-2.4-1.074-2.4-2.4V14.4c0-1.326 1.074-2.4 2.4-2.4s2.4 1.074 2.4 2.4zM26.4 42.63h2.4c2.651 0 4.8 2.15 4.8 4.8V69.6c0 2.651-2.149 4.8-4.8 4.8h-2.4c-2.651 0-4.8-2.149-4.8-4.8V47.43c0-2.65 2.149-4.8 4.8-4.8M72 57.6h-2.4c-2.651 0-4.8 2.149-4.8 4.8v7.2c0 2.651 2.149 4.8 4.8 4.8H72c2.651 0 4.8-2.149 4.8-4.8v-7.2c0-2.651-2.149-4.8-4.8-4.8M48 24h2.4c2.651 0 4.8 2.149 4.8 4.8v40.8c0 2.651-2.149 4.8-4.8 4.8H48c-2.651 0-4.8-2.149-4.8-4.8V28.8c0-2.651 2.149-4.8 4.8-4.8"
                })
            }),
            /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("g", {
                clipPath: "url(#afe6b0__a)",
                children: /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#afe6b0__b)",
                    children: [
                        /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("rect", {
                            width: 86.562,
                            height: 81.6,
                            x: 2.449,
                            y: 9,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M2.45 9H89.01v81.6H2.45z"
                        }),
                        /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("ellipse", {
                            cx: 37.659,
                            cy: 35.501,
                            fill: "url(#afe6b0__c)",
                            rx: 37.659,
                            ry: 35.501,
                            transform: "matrix(1 -.00132 .00148 1 13.684 -23.286)"
                        }),
                        /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("path", {
                            fill: "url(#afe6b0__d)",
                            d: "M35.488-5.414c-6.648-3.878-15.071-3.901-21.744-.061l-66.738 38.412c-6.536 3.762-6.547 12.642-.02 16.418l66.298 38.358c6.662 3.854 15.085 3.85 21.742-.014l66.16-38.393c6.493-3.768 6.505-12.599.02-16.381z"
                        }),
                        /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("path", {
                            fill: "url(#afe6b0__e)",
                            d: "M14.925 47.742c1.48-30.464 28.876-54.029 61.193-52.634s57.314 27.221 55.834 57.685-28.876 54.029-61.192 52.634-57.315-27.221-55.835-57.685"
                        }),
                        /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("path", {
                            fill: "url(#afe6b0__f)",
                            d: "M-32.623 67.829c12.846-5.166 26.7-4.945 38.703-.426 24.018 9.064 37.993 3.46 47.686-19.13C58.594 36.94 68.02 27.34 80.88 22.168c25.712-10.324 55.396.878 66.332 25.045 10.937 24.166-1.034 52.104-26.74 62.442-12.888 5.183-26.77 4.94-38.79.381-23.98-9.016-37.928-3.39-47.608 19.226-4.84 11.308-14.265 20.875-27.11 26.041-25.692 10.332-55.376-.871-66.306-25.024s1.028-52.117 26.72-62.45"
                        }),
                        /*#__PURE__*/ (0, $5ukbI$reactjsxruntime.jsx)("ellipse", {
                            cx: 37.659,
                            cy: 35.501,
                            fill: "url(#afe6b0__g)",
                            rx: 37.659,
                            ry: 35.501,
                            transform: "matrix(1 -.00132 .00148 1 46.254 -11.709)"
                        })
                    ]
                })
            })
        ]
    });
const $5238c0d9a22d5d8a$var$ForwardRef = /*#__PURE__*/ (0, $5ukbI$react.forwardRef)($5238c0d9a22d5d8a$var$SvgComponent);
var $5238c0d9a22d5d8a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5238c0d9a22d5d8a$var$ForwardRef);
