// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_AIGenerateImage_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $hvn4O$reactjsxruntime = require("react/jsx-runtime");
var $hvn4O$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $453c8e497aaca26a$export$2e2bcd8739ae039);




const $453c8e497aaca26a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsxs)("linearGradient", {
                        id: "90836c__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsxs)("linearGradient", {
                        id: "90836c__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsxs)("linearGradient", {
                        id: "90836c__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsxs)("radialGradient", {
                        id: "90836c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsxs)("radialGradient", {
                        id: "90836c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("clipPath", {
                        id: "90836c__b",
                        children: /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("clipPath", {
                id: "90836c__a",
                children: /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m85.692 28.859-6.235-1.403c-.925-.21-1.897.05-2.595.694l-4.699 4.33c-2.085 1.923-5.404.008-4.782-2.761l1.403-6.235c.21-.926-.051-1.898-.695-2.596l-4.33-4.698c-1.923-2.086-.007-5.404 2.762-4.782l6.235 1.402c.925.21 1.897-.05 2.595-.694l4.699-4.33c2.086-1.923 5.404-.007 4.782 2.761l-1.402 6.235c-.21.926.05 1.898.694 2.596l4.33 4.698c1.923 2.086.007 5.404-2.762 4.783M10.91 36.305c3.194-11.921 15.448-18.996 27.37-15.802 5.175 1.387 9.433 4.478 12.339 8.51.645.896 1.895 1.1 2.791.453.897-.645 1.1-1.895.454-2.791-3.424-4.753-8.451-8.402-14.55-10.035-14.055-3.767-28.502 4.574-32.268 18.63-2.503 9.34.34 18.848 6.65 25.283l-5.286-.017c-1.104-.003-2.002.89-2.006 1.994-.003 1.104.89 2.003 1.994 2.006l9.896.032c1.078.003 1.964-.848 2.005-1.925l.36-9.547c.042-1.104-.819-2.033-1.922-2.074-1.104-.042-2.033.819-2.075 1.923l-.179 4.739c-5.307-5.46-7.686-13.494-5.573-21.379m40.076 9.756c-.39 1.727 1.685 2.928 2.99 1.724l3.31-3.054c.438-.402 1.042-.564 1.624-.434l4.395.99c1.727.39 2.927-1.684 1.724-2.989l-.956-1.036-2.098-2.275c-.402-.437-.564-1.04-.434-1.623l.99-4.395c.39-1.728-1.684-2.928-2.989-1.724l-3.31 3.054c-.438.401-1.042.564-1.624.434l-4.395-.99c-1.728-.391-2.928 1.684-1.724 2.989l3.054 3.31c.309.337.476.772.478 1.22q0 .202-.044.404zm-3.76-4.8h-12.05c-4.419 0-8 3.582-8 8v34.214l14.631-23 1.688 1.072-1.687-1.073-.001.001c2.148-3.383 7.042-3.383 9.19 0l-1.687 1.072 1.688-1.072 10.075 15.848 5.6-8.475.009-.013c2.164-3.22 6.908-3.226 9.07 0l.002.003 9.824 14.592c.763-1.225 1.203-2.672 1.203-4.221V49.262c0-4.419-3.582-8-8-8h-9.71c1.766 4.062-1.784 9.024-6.645 7.927h-.002l-3.272-.737-2.464 2.273c-4.186 3.861-10.858.017-9.604-5.544v-.002l.738-3.271zM30.175 86.21H54.54l4.117-6.23L47.62 62.62c-.578-.91-1.861-.91-2.438 0h-.001zm42.257-16.141L82.634 85.22c-1.143.63-2.457.988-3.854.988H59.334l10.67-16.147c.585-.863 1.852-.855 2.425.002zM72.001 57c-1.657 0-3-1.343-3-3s1.343-3 3-3 3 1.343 3 3-1.343 3-3 3"
                })
            }),
            /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("g", {
                clipPath: "url(#90836c__a)",
                children: /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#90836c__b)",
                    children: [
                        /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#90836c__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("path", {
                            fill: "url(#90836c__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("path", {
                            fill: "url(#90836c__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("path", {
                            fill: "url(#90836c__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $hvn4O$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#90836c__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $453c8e497aaca26a$var$ForwardRef = /*#__PURE__*/ (0, $hvn4O$react.forwardRef)($453c8e497aaca26a$var$SvgComponent);
var $453c8e497aaca26a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($453c8e497aaca26a$var$ForwardRef);
