// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_explosion_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bzzZ5$jsxs, jsx as $bzzZ5$jsx} from "react/jsx-runtime";
import {forwardRef as $bzzZ5$forwardRef} from "react";





const $9b55b64239f696cc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bzzZ5$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bzzZ5$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bzzZ5$jsxs)("linearGradient", {
                        id: "fbfb67__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bzzZ5$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bzzZ5$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bzzZ5$jsxs)("linearGradient", {
                        id: "fbfb67__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bzzZ5$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bzzZ5$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bzzZ5$jsxs)("linearGradient", {
                        id: "fbfb67__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bzzZ5$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bzzZ5$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bzzZ5$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bzzZ5$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bzzZ5$jsxs)("radialGradient", {
                        id: "fbfb67__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bzzZ5$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bzzZ5$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bzzZ5$jsxs)("radialGradient", {
                        id: "fbfb67__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bzzZ5$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bzzZ5$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bzzZ5$jsx)("clipPath", {
                        id: "fbfb67__b",
                        children: /*#__PURE__*/ (0, $bzzZ5$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bzzZ5$jsx)("clipPath", {
                id: "fbfb67__a",
                children: /*#__PURE__*/ (0, $bzzZ5$jsx)("path", {
                    fill: "#fff",
                    d: "M32.513 26.206c-3.146-5.824 4.07-11.705 9.139-7.446l35.67 29.971 20.785-22.948c4.14-4.572 11.656-.67 10.3 5.346l-5.967 26.478h45.746c6.925 0 8.343 9.788 1.703 11.753l-39.652 11.737 12.026 14.157c3.962 4.664-.782 11.571-6.558 9.546l-18.324-6.422v39.044c0 6.661-9.234 8.361-11.606 2.136l-14.589-38.28c-9.388 5.399-22.184 12.657-30.308 17.255-5.635 3.19-11.534-3.526-7.842-8.711 4.73-6.645 11.689-16.462 16.837-23.915l-27.677-7.103c-6.065-1.557-5.99-10.197.1-11.649l28.654-6.827z"
                })
            }),
            /*#__PURE__*/ (0, $bzzZ5$jsx)("g", {
                clipPath: "url(#fbfb67__a)",
                children: /*#__PURE__*/ (0, $bzzZ5$jsxs)("g", {
                    clipPath: "url(#fbfb67__b)",
                    children: [
                        /*#__PURE__*/ (0, $bzzZ5$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $bzzZ5$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $bzzZ5$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#fbfb67__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $bzzZ5$jsx)("path", {
                            fill: "url(#fbfb67__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $bzzZ5$jsx)("path", {
                            fill: "url(#fbfb67__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $bzzZ5$jsx)("path", {
                            fill: "url(#fbfb67__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $bzzZ5$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#fbfb67__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $9b55b64239f696cc$var$ForwardRef = /*#__PURE__*/ (0, $bzzZ5$forwardRef)($9b55b64239f696cc$var$SvgComponent);
var $9b55b64239f696cc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9b55b64239f696cc$var$ForwardRef);


export {$9b55b64239f696cc$export$2e2bcd8739ae039 as default};
