// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_brand_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $kRJ0R$reactjsxruntime = require("react/jsx-runtime");
var $kRJ0R$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $eb544d839b0ac1a3$export$2e2bcd8739ae039);




const $eb544d839b0ac1a3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsxs)("linearGradient", {
                        id: "eed3bc__d",
                        x1: -18.921,
                        x2: 187.087,
                        y1: 128.357,
                        y2: -4.598,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsxs)("linearGradient", {
                        id: "eed3bc__e",
                        x1: 151.995,
                        x2: 66.38,
                        y1: 40.747,
                        y2: 138.405,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsxs)("linearGradient", {
                        id: "eed3bc__f",
                        x1: 40.959,
                        x2: 198.223,
                        y1: 146.143,
                        y2: 52.689,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsxs)("radialGradient", {
                        id: "eed3bc__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsxs)("radialGradient", {
                        id: "eed3bc__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("clipPath", {
                        id: "eed3bc__b",
                        children: /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 18,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("clipPath", {
                id: "eed3bc__a",
                children: /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M45.75 26.06h68.5c6.63 0 12 5.37 12 12v49.63c0 25.55-20.7 46.25-46.25 46.25s-46.25-20.71-46.25-46.25V38.06c0-6.63 5.37-12 12-12m4.08 65.75c.629.369 1.317.539 1.995.54 1.388-.002 2.726-.722 3.465-2l13.96-24.18c1.11-1.91.45-4.36-1.46-5.46-1.91-1.11-4.36-.45-5.46 1.46L48.37 86.35c-1.11 1.91-.45 4.36 1.46 5.46m52.52.54c2.89 0 5.48-1.5 6.93-4 1.45-2.51 1.45-5.5 0-8L96.71 58.58c-1.45-2.5-4.04-4-6.93-4s-5.48 1.5-6.93 4L70.28 80.35c-1.45 2.5-1.45 5.5 0 8s4.04 4 6.93 4zM89.78 62.57l12.571 21.78h-25.14z"
                })
            }),
            /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("g", {
                clipPath: "url(#eed3bc__a)",
                children: /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#eed3bc__b)",
                    children: [
                        /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 18,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M14 18h131v123.491H14z"
                        }),
                        /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#eed3bc__c)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 31 -30.86)"
                        }),
                        /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("path", {
                            fill: "url(#eed3bc__d)",
                            d: "M64.001-3.814c-10.06-5.868-22.807-5.904-32.905-.092l-101 58.131c-9.891 5.694-9.907 19.133-.03 24.847L30.4 137.122c10.082 5.833 22.83 5.826 32.904-.021l100.125-58.104c9.826-5.703 9.844-19.066.03-24.79z"
                        }),
                        /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("path", {
                            fill: "url(#eed3bc__e)",
                            d: "M32.878 76.631c2.24-46.103 43.701-81.765 92.607-79.655 48.907 2.111 86.738 41.196 84.499 87.299-2.239 46.102-43.701 81.765-92.607 79.654-48.906-2.11-86.738-41.195-84.499-87.298"
                        }),
                        /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("path", {
                            fill: "url(#eed3bc__f)",
                            d: "M-39.075 107.029c19.44-7.818 40.406-7.483 58.57-.645 36.349 13.717 57.498 5.236 72.167-28.95 7.307-17.153 21.572-31.681 41.035-39.508 38.911-15.625 83.833 1.329 100.384 37.902s-1.565 78.852-40.467 94.497c-19.506 7.844-40.512 7.477-58.705.577-36.288-13.644-57.398-5.131-72.048 29.096-7.325 17.113-21.588 31.591-41.028 39.409-38.88 15.636-83.803-1.317-100.344-37.869-16.542-36.553 1.555-78.874 40.436-94.509"
                        }),
                        /*#__PURE__*/ (0, $kRJ0R$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#eed3bc__g)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 80.293 -13.34)"
                        })
                    ]
                })
            })
        ]
    });
const $eb544d839b0ac1a3$var$ForwardRef = /*#__PURE__*/ (0, $kRJ0R$react.forwardRef)($eb544d839b0ac1a3$var$SvgComponent);
var $eb544d839b0ac1a3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($eb544d839b0ac1a3$var$ForwardRef);
