// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_buildings_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $8TIrV$jsxs, jsx as $8TIrV$jsx} from "react/jsx-runtime";
import {forwardRef as $8TIrV$forwardRef} from "react";





const $eb23e8b8f40b3783$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8TIrV$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8TIrV$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8TIrV$jsxs)("linearGradient", {
                        id: "3be0ba__d",
                        x1: -17.213,
                        x2: 116.852,
                        y1: 82.649,
                        y2: -3.785,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8TIrV$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8TIrV$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8TIrV$jsxs)("linearGradient", {
                        id: "3be0ba__e",
                        x1: 93.95,
                        x2: 38.201,
                        y1: 25.61,
                        y2: 89.133,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8TIrV$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8TIrV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8TIrV$jsxs)("linearGradient", {
                        id: "3be0ba__f",
                        x1: 21.733,
                        x2: 124.07,
                        y1: 94.229,
                        y2: 33.479,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8TIrV$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8TIrV$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8TIrV$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8TIrV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8TIrV$jsxs)("radialGradient", {
                        id: "3be0ba__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.9784 -37.0667 0 37.067 34.978)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8TIrV$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8TIrV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8TIrV$jsxs)("radialGradient", {
                        id: "3be0ba__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.9784 -37.0667 0 37.067 34.978)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8TIrV$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8TIrV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8TIrV$jsx)("clipPath", {
                        id: "3be0ba__b",
                        children: /*#__PURE__*/ (0, $8TIrV$jsx)("rect", {
                            width: 85.2,
                            height: 80.4,
                            x: 4.199,
                            y: 10.8,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8TIrV$jsx)("clipPath", {
                id: "3be0ba__a",
                children: /*#__PURE__*/ (0, $8TIrV$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M21.452 16.758h28.842c5.304 0 9.6 4.296 9.6 9.6v52.878H44.3V66.888c0-2.652-2.148-4.8-4.8-4.8h-7.254c-2.652 0-4.8 2.148-4.8 4.8v12.348H11.852V26.358c0-5.304 4.296-9.6 9.6-9.6m7.674 33.936c1.326 0 2.4-1.074 2.4-2.4v-3.456c0-1.326-1.074-2.4-2.4-2.4H25.67c-1.326 0-2.4 1.074-2.4 2.4v3.456c0 1.326 1.074 2.4 2.4 2.4zm0-16.836c1.326 0 2.4-1.074 2.4-2.4v-3.456c0-1.326-1.074-2.4-2.4-2.4H25.67c-1.326 0-2.4 1.074-2.4 2.4v3.456c0 1.326 1.074 2.4 2.4 2.4zm16.956 16.836c1.326 0 2.4-1.074 2.4-2.4v-3.456c0-1.326-1.074-2.4-2.4-2.4h-3.456c-1.326 0-2.4 1.074-2.4 2.4v3.456c0 1.326 1.074 2.4 2.4 2.4zm0-16.836c1.326 0 2.4-1.074 2.4-2.4v-3.456c0-1.326-1.074-2.4-2.4-2.4h-3.456c-1.326 0-2.4 1.074-2.4 2.4v3.456c0 1.326 1.074 2.4 2.4 2.4zm28.464-4.122c5.304 0 9.6 4.296 9.6 9.6v39.9H64.7v-49.5zm-.798 37.296c1.326 0 2.4-1.074 2.4-2.4v-3.456c0-1.326-1.074-2.4-2.4-2.4h-3.456c-1.326 0-2.4 1.074-2.4 2.4v3.456c0 1.326 1.074 2.4 2.4 2.4zm0-16.836c1.326 0 2.4-1.074 2.4-2.4V44.34c0-1.326-1.074-2.4-2.4-2.4h-3.456c-1.326 0-2.4 1.074-2.4 2.4v3.456c0 1.326 1.074 2.4 2.4 2.4z"
                })
            }),
            /*#__PURE__*/ (0, $8TIrV$jsx)("g", {
                clipPath: "url(#3be0ba__a)",
                children: /*#__PURE__*/ (0, $8TIrV$jsxs)("g", {
                    clipPath: "url(#3be0ba__b)",
                    children: [
                        /*#__PURE__*/ (0, $8TIrV$jsx)("rect", {
                            width: 85.2,
                            height: 80.4,
                            x: 4.199,
                            y: 10.8,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $8TIrV$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M4.2 10.8h85.2v80.4H4.2z"
                        }),
                        /*#__PURE__*/ (0, $8TIrV$jsx)("ellipse", {
                            cx: 37.067,
                            cy: 34.978,
                            fill: "url(#3be0ba__c)",
                            rx: 37.067,
                            ry: 34.978,
                            transform: "matrix(1 -.00132 .00148 1 15.254 -21.01)"
                        }),
                        /*#__PURE__*/ (0, $8TIrV$jsx)("path", {
                            fill: "url(#3be0ba__d)",
                            d: "M36.717-3.402c-6.543-3.821-14.833-3.844-21.4-.06l-65.689 37.847c-6.433 3.707-6.444 12.456-.02 16.177l65.255 37.793c6.557 3.798 14.848 3.793 21.4-.013l65.119-37.83c6.391-3.712 6.402-12.413.02-16.14z"
                        }),
                        /*#__PURE__*/ (0, $8TIrV$jsx)("path", {
                            fill: "url(#3be0ba__e)",
                            d: "M16.478 48.972C17.934 18.957 44.9-4.262 76.708-2.888s56.412 26.821 54.956 56.837c-1.456 30.015-28.422 53.234-60.23 51.86S15.023 78.988 16.478 48.972"
                        }),
                        /*#__PURE__*/ (0, $8TIrV$jsx)("path", {
                            fill: "url(#3be0ba__f)",
                            d: "M-30.32 68.763c12.644-5.09 26.28-4.872 38.094-.42 23.64 8.931 37.395 3.41 46.936-18.847 4.752-11.169 14.03-20.627 26.688-25.723 25.307-10.172 54.523.865 65.288 24.676s-1.018 51.338-26.319 61.524c-12.686 5.107-26.349 4.868-38.18.376-23.602-8.884-37.331-3.341-46.86 18.943-4.764 11.141-14.04 20.568-26.683 25.658-25.287 10.18-54.504-.858-65.262-24.656s1.012-51.351 26.299-61.531"
                        }),
                        /*#__PURE__*/ (0, $8TIrV$jsx)("ellipse", {
                            cx: 37.067,
                            cy: 34.978,
                            fill: "url(#3be0ba__g)",
                            rx: 37.067,
                            ry: 34.978,
                            transform: "matrix(1 -.00132 .00148 1 47.316 -9.604)"
                        })
                    ]
                })
            })
        ]
    });
const $eb23e8b8f40b3783$var$ForwardRef = /*#__PURE__*/ (0, $8TIrV$forwardRef)($eb23e8b8f40b3783$var$SvgComponent);
var $eb23e8b8f40b3783$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($eb23e8b8f40b3783$var$ForwardRef);


export {$eb23e8b8f40b3783$export$2e2bcd8739ae039 as default};
