// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_apps_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $dA9GI$jsxs, jsx as $dA9GI$jsx} from "react/jsx-runtime";
import {forwardRef as $dA9GI$forwardRef} from "react";





const $e463a51e72170bbd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dA9GI$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dA9GI$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dA9GI$jsxs)("linearGradient", {
                        id: "09f9ef__d",
                        x1: -18.921,
                        x2: 187.087,
                        y1: 132.357,
                        y2: -0.598,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dA9GI$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $dA9GI$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dA9GI$jsxs)("linearGradient", {
                        id: "09f9ef__e",
                        x1: 151.995,
                        x2: 66.38,
                        y1: 44.747,
                        y2: 142.405,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dA9GI$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dA9GI$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dA9GI$jsxs)("linearGradient", {
                        id: "09f9ef__f",
                        x1: 40.959,
                        x2: 198.223,
                        y1: 150.143,
                        y2: 56.689,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dA9GI$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dA9GI$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dA9GI$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dA9GI$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dA9GI$jsxs)("radialGradient", {
                        id: "09f9ef__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dA9GI$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $dA9GI$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dA9GI$jsxs)("radialGradient", {
                        id: "09f9ef__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dA9GI$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dA9GI$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dA9GI$jsx)("clipPath", {
                        id: "09f9ef__b",
                        children: /*#__PURE__*/ (0, $dA9GI$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 22,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dA9GI$jsx)("clipPath", {
                id: "09f9ef__a",
                children: /*#__PURE__*/ (0, $dA9GI$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M90.309 21.68h31.97c8.836 0 16 7.163 16 16v31.97c0 8.836-7.164 16-16 16h-31.97c-8.837 0-16-7.164-16-16V37.68c0-8.837 7.163-16 16-16m-54.59 19.46h16.52c7.732 0 14 6.268 14 14v16.52c0 7.732-6.268 14-14 14h-16.52c-7.732 0-14-6.268-14-14V55.14c0-7.732 6.268-14 14-14m16.52 52.66h-16.52c-7.732 0-14 6.268-14 14v16.52c0 7.732 6.268 14 14 14h16.52c7.732 0 14-6.268 14-14V107.8c0-7.732-6.268-14-14-14m36.07 0h16.52c7.732 0 14 6.268 14 14v16.52c0 7.732-6.268 14-14 14h-16.52c-7.732 0-14-6.268-14-14V107.8c0-7.732 6.268-14 14-14"
                })
            }),
            /*#__PURE__*/ (0, $dA9GI$jsx)("g", {
                clipPath: "url(#09f9ef__a)",
                children: /*#__PURE__*/ (0, $dA9GI$jsxs)("g", {
                    clipPath: "url(#09f9ef__b)",
                    children: [
                        /*#__PURE__*/ (0, $dA9GI$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 22,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $dA9GI$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M14 22h131v123.491H14z"
                        }),
                        /*#__PURE__*/ (0, $dA9GI$jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#09f9ef__c)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 31 -26.86)"
                        }),
                        /*#__PURE__*/ (0, $dA9GI$jsx)("path", {
                            fill: "url(#09f9ef__d)",
                            d: "M64.001.186C53.941-5.682 41.194-5.718 31.096.094l-101 58.131c-9.891 5.694-9.907 19.133-.03 24.847L30.4 141.122c10.082 5.833 22.83 5.826 32.904-.021l100.125-58.104c9.826-5.703 9.844-19.066.03-24.79z"
                        }),
                        /*#__PURE__*/ (0, $dA9GI$jsx)("path", {
                            fill: "url(#09f9ef__e)",
                            d: "M32.878 80.631C35.118 34.528 76.58-1.134 125.485.976c48.907 2.111 86.738 41.196 84.499 87.299-2.239 46.102-43.701 81.765-92.607 79.654-48.906-2.11-86.738-41.195-84.499-87.298"
                        }),
                        /*#__PURE__*/ (0, $dA9GI$jsx)("path", {
                            fill: "url(#09f9ef__f)",
                            d: "M-39.075 111.029c19.44-7.818 40.406-7.483 58.57-.645 36.349 13.717 57.498 5.236 72.167-28.95 7.307-17.153 21.572-31.681 41.035-39.508 38.911-15.625 83.833 1.329 100.384 37.902s-1.565 78.852-40.467 94.497c-19.506 7.844-40.512 7.477-58.705.577-36.288-13.644-57.398-5.131-72.048 29.096-7.325 17.113-21.588 31.591-41.028 39.409-38.88 15.636-83.803-1.317-100.344-37.869-16.542-36.553 1.555-78.874 40.436-94.509"
                        }),
                        /*#__PURE__*/ (0, $dA9GI$jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#09f9ef__g)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 80.293 -9.34)"
                        })
                    ]
                })
            })
        ]
    });
const $e463a51e72170bbd$var$ForwardRef = /*#__PURE__*/ (0, $dA9GI$forwardRef)($e463a51e72170bbd$var$SvgComponent);
var $e463a51e72170bbd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e463a51e72170bbd$var$ForwardRef);


export {$e463a51e72170bbd$export$2e2bcd8739ae039 as default};
