// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_logo_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $bZhYT$reactjsxruntime = require("react/jsx-runtime");
var $bZhYT$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $71d181f7b20cc92d$export$2e2bcd8739ae039);




const $71d181f7b20cc92d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5e84c9__d",
                        x1: -11.355,
                        x2: 112.249,
                        y1: 77.014,
                        y2: -2.759,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5e84c9__e",
                        x1: 91.194,
                        x2: 39.825,
                        y1: 24.448,
                        y2: 83.043,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5e84c9__f",
                        x1: 24.574,
                        x2: 118.933,
                        y1: 87.686,
                        y2: 31.613,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5e84c9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5e84c9__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("clipPath", {
                        id: "5e84c9__b",
                        children: /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 10.8,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("clipPath", {
                id: "5e84c9__a",
                children: /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M51.218 16.044 76.76 28.788c2.442 1.218 3.984 3.714 3.984 6.444v25.476c0 2.724-1.536 5.22-3.972 6.438L51.23 79.944c-2.028 1.02-4.422 1.02-6.45 0L19.238 67.146c-2.436-1.224-3.972-3.714-3.972-6.438V35.232c0-2.73 1.542-5.226 3.984-6.444l25.542-12.744c2.022-1.008 4.404-1.008 6.426 0M30.794 58.56c.834 0 1.638-.432 2.082-1.2h-.006l8.79-15.222c.666-1.146.27-2.616-.876-3.276-1.146-.666-2.616-.27-3.276.876l-8.79 15.222c-.666 1.146-.27 2.616.876 3.276.378.222.792.324 1.2.324m32.004 0c1.734 0 3.288-.9 4.158-2.4.87-1.506.87-3.3 0-4.8l-8.118-14.058c-.87-1.5-2.424-2.4-4.158-2.4s-3.288.9-4.158 2.4L42.404 51.36c-.87 1.5-.87 3.3 0 4.8s2.424 2.4 4.158 2.4zm-8.112-18.858 8.118 14.058H46.568z"
                })
            }),
            /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("g", {
                clipPath: "url(#5e84c9__a)",
                children: /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#5e84c9__b)",
                    children: [
                        /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 10.8,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M8.398 10.8h78.6v74.094h-78.6z"
                        }),
                        /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#5e84c9__c)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 18.598 -18.517)"
                        }),
                        /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("path", {
                            fill: "url(#5e84c9__d)",
                            d: "M38.398-2.288C32.36-5.81 24.713-5.831 18.654-2.344l-60.6 34.88c-5.934 3.415-5.944 11.478-.017 14.907l60.199 34.83c6.05 3.5 13.698 3.495 19.742-.013l60.076-34.862c5.895-3.422 5.906-11.44.018-14.874z"
                        }),
                        /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("path", {
                            fill: "url(#5e84c9__e)",
                            d: "M19.724 45.978C21.067 18.317 45.944-3.08 75.288-1.814s52.043 24.717 50.699 52.379c-1.343 27.661-26.22 49.059-55.564 47.792S18.38 73.64 19.723 45.978"
                        }),
                        /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("path", {
                            fill: "url(#5e84c9__f)",
                            d: "M-23.446 64.217c11.664-4.69 24.244-4.49 35.143-.386 21.808 8.23 34.498 3.141 43.3-17.37 4.383-10.292 12.943-19.01 24.62-23.705 23.347-9.375 50.3.797 60.231 22.74 9.931 21.944-.939 47.312-24.28 56.699-11.704 4.707-24.308 4.486-35.223.347-21.773-8.187-34.439-3.079-43.23 17.457-4.394 10.268-12.952 18.955-24.616 23.646-23.328 9.381-50.282-.791-60.207-22.722s.934-47.324 24.262-56.706"
                        }),
                        /*#__PURE__*/ (0, $bZhYT$reactjsxruntime.jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#5e84c9__g)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 48.176 -8.004)"
                        })
                    ]
                })
            })
        ]
    });
const $71d181f7b20cc92d$var$ForwardRef = /*#__PURE__*/ (0, $bZhYT$react.forwardRef)($71d181f7b20cc92d$var$SvgComponent);
var $71d181f7b20cc92d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($71d181f7b20cc92d$var$ForwardRef);
