// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_dropToUpload_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $9UZYb$jsxs, jsx as $9UZYb$jsx} from "react/jsx-runtime";
import {forwardRef as $9UZYb$forwardRef} from "react";





const $7aed909c2e1fd136$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9UZYb$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9UZYb$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9UZYb$jsxs)("linearGradient", {
                        id: "784b02__d",
                        x1: -32.943,
                        x2: 206.051,
                        y1: 135.365,
                        y2: -10.037,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9UZYb$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $9UZYb$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9UZYb$jsxs)("linearGradient", {
                        id: "784b02__e",
                        x1: 158.85,
                        x2: 56.322,
                        y1: 31.077,
                        y2: 141.323,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9UZYb$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9UZYb$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9UZYb$jsxs)("linearGradient", {
                        id: "784b02__f",
                        x1: 34.252,
                        x2: 216.005,
                        y1: 156.538,
                        y2: 54.722,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9UZYb$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9UZYb$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9UZYb$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $9UZYb$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9UZYb$jsxs)("radialGradient", {
                        id: "784b02__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 38.708 48.409)scale(63.953)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9UZYb$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $9UZYb$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9UZYb$jsxs)("radialGradient", {
                        id: "784b02__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 55.938 86.495)scale(63.953)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9UZYb$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9UZYb$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9UZYb$jsx)("clipPath", {
                        id: "784b02__b",
                        children: /*#__PURE__*/ (0, $9UZYb$jsx)("rect", {
                            width: 147,
                            height: 147,
                            x: 4,
                            y: 4,
                            fill: "#fff",
                            rx: 4.438
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9UZYb$jsx)("clipPath", {
                id: "784b02__a",
                children: /*#__PURE__*/ (0, $9UZYb$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M51.739 21.933h27.404c2.81 0 5.506 1.116 7.494 3.104l29.118 29.118c1.988 1.988 3.104 4.683 3.104 7.494v41.535c0 5.853-4.745 10.598-10.598 10.598H51.739c-5.853 0-10.598-4.745-10.598-10.598V32.531c0-5.853 4.745-10.598 10.598-10.598m31 30.625c.004-1.951-1.574-3.536-3.525-3.54s-3.536 1.573-3.54 3.524l-.079 34.983-11.943-9.5c-1.527-1.214-3.75-.96-4.964.566-1.214 1.528-.96 3.75.566 4.964l17.793 14.152c1.294 1.03 3.13 1.023 4.418-.016L98.999 83.54c1.518-1.226 1.755-3.45.53-4.968s-3.45-1.756-4.968-.53l-11.9 9.605zM22.4 75.523c0-1.767 1.433-3.2 3.2-3.2h5.881c1.767 0 3.2-1.432 3.2-3.2 0-1.767-1.433-3.2-3.2-3.2H25.6c-5.302 0-9.6 4.298-9.6 9.6v5.535c0 1.768 1.433 3.2 3.2 3.2s3.2-1.432 3.2-3.2zm106.119-9.6c-1.767 0-3.2 1.433-3.2 3.2s1.433 3.2 3.2 3.2h5.881c1.767 0 3.2 1.433 3.2 3.2v5.535c0 1.768 1.433 3.2 3.2 3.2s3.2-1.432 3.2-3.2v-5.535c0-5.302-4.298-9.6-9.6-9.6zM144 95.588c0-1.767-1.433-3.2-3.2-3.2s-3.2 1.433-3.2 3.2v11.07c0 1.768 1.433 3.2 3.2 3.2s3.2-1.432 3.2-3.2zm-121.6 0c0-1.767-1.433-3.2-3.2-3.2s-3.2 1.433-3.2 3.2v11.07c0 1.768 1.433 3.2 3.2 3.2s3.2-1.432 3.2-3.2zm0 25.6c0-1.767-1.433-3.2-3.2-3.2s-3.2 1.433-3.2 3.2v5.535c0 5.302 4.298 9.6 9.6 9.6h5.881c1.767 0 3.2-1.432 3.2-3.2 0-1.767-1.433-3.2-3.2-3.2H25.6c-1.767 0-3.2-1.432-3.2-3.2zm121.6 0c0-1.767-1.433-3.2-3.2-3.2s-3.2 1.433-3.2 3.2v5.535c0 1.768-1.433 3.2-3.2 3.2h-5.881c-1.767 0-3.2 1.433-3.2 3.2s1.433 3.2 3.2 3.2h5.881c5.302 0 9.6-4.298 9.6-9.6zm-97.081 8.735c-1.767 0-3.2 1.433-3.2 3.2s1.433 3.2 3.2 3.2H58.68c1.767 0 3.2-1.432 3.2-3.2 0-1.767-1.433-3.2-3.2-3.2zm27.2 0c-1.767 0-3.2 1.433-3.2 3.2s1.433 3.2 3.2 3.2H85.88c1.767 0 3.2-1.432 3.2-3.2 0-1.767-1.433-3.2-3.2-3.2zm27.2 0c-1.767 0-3.2 1.433-3.2 3.2s1.433 3.2 3.2 3.2h11.762c1.767 0 3.2-1.432 3.2-3.2 0-1.767-1.433-3.2-3.2-3.2z"
                })
            }),
            /*#__PURE__*/ (0, $9UZYb$jsx)("g", {
                clipPath: "url(#784b02__a)",
                children: /*#__PURE__*/ (0, $9UZYb$jsxs)("g", {
                    clipPath: "url(#784b02__b)",
                    children: [
                        /*#__PURE__*/ (0, $9UZYb$jsx)("rect", {
                            width: 147,
                            height: 147,
                            x: 4,
                            y: 4,
                            fill: "#FFECCF",
                            rx: 4.438
                        }),
                        /*#__PURE__*/ (0, $9UZYb$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M4 4h147v147H4z"
                        }),
                        /*#__PURE__*/ (0, $9UZYb$jsx)("circle", {
                            cx: 87.117,
                            cy: 9.701,
                            r: 63.953,
                            fill: "url(#784b02__c)",
                            transform: "rotate(-.08 87.117 9.701)"
                        }),
                        /*#__PURE__*/ (0, $9UZYb$jsx)("path", {
                            fill: "url(#784b02__d)",
                            d: "M60.107-21.967c-11.29-6.986-25.593-7.029-36.925-.11L-90.152 47.12c-11.1 6.777-11.119 22.774-.034 29.577L22.4 145.8c11.314 6.943 25.618 6.935 36.923-.025l112.354-69.165c11.025-6.789 11.046-22.696.033-29.51z"
                        }),
                        /*#__PURE__*/ (0, $9UZYb$jsx)("path", {
                            fill: "url(#784b02__e)",
                            d: "M25.184 73.792c2.513-54.88 49.038-97.331 103.918-94.818 54.879 2.512 97.331 49.037 94.818 103.917-2.512 54.879-49.037 97.331-103.917 94.818-54.88-2.512-97.331-49.037-94.819-103.917"
                        }),
                        /*#__PURE__*/ (0, $9UZYb$jsx)("path", {
                            fill: "url(#784b02__f)",
                            d: "M-55.557 109.977c21.815-9.306 45.342-8.907 65.725-.767 40.787 16.329 64.52 6.233 80.98-34.46 8.199-20.42 24.207-37.714 46.046-47.03 43.664-18.6 94.073 1.581 112.645 45.116 18.572 43.536-1.756 93.864-45.41 112.488-21.888 9.337-45.46 8.9-65.874.687-40.721-16.242-64.408-6.108-80.848 34.634-8.22 20.371-24.224 37.606-46.039 46.912-43.63 18.613-94.038-1.568-112.6-45.079s1.746-93.888 45.375-112.501"
                        }),
                        /*#__PURE__*/ (0, $9UZYb$jsx)("circle", {
                            cx: 142.433,
                            cy: 30.558,
                            r: 63.953,
                            fill: "url(#784b02__g)",
                            transform: "rotate(-.08 142.433 30.558)"
                        })
                    ]
                })
            })
        ]
    });
const $7aed909c2e1fd136$var$ForwardRef = /*#__PURE__*/ (0, $9UZYb$forwardRef)($7aed909c2e1fd136$var$SvgComponent);
var $7aed909c2e1fd136$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7aed909c2e1fd136$var$ForwardRef);


export {$7aed909c2e1fd136$export$2e2bcd8739ae039 as default};
