// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_heart_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $iGYEi$reactjsxruntime = require("react/jsx-runtime");
var $iGYEi$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $dfbecb2fdb2b11cd$export$2e2bcd8739ae039);




const $dfbecb2fdb2b11cd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsxs)("linearGradient", {
                        id: "baa08c__d",
                        x1: -12.705,
                        x2: 118.01,
                        y1: 77.25,
                        y2: -2.276,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsxs)("linearGradient", {
                        id: "baa08c__e",
                        x1: 92.195,
                        x2: 36.119,
                        y1: 20.21,
                        y2: 80.508,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsxs)("linearGradient", {
                        id: "baa08c__f",
                        x1: 24.046,
                        x2: 123.454,
                        y1: 88.83,
                        y2: 33.143,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsxs)("radialGradient", {
                        id: "baa08c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 22.22 30.74)scale(34.9784)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsxs)("radialGradient", {
                        id: "baa08c__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 31.645 51.57)scale(34.9784)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("clipPath", {
                        id: "baa08c__b",
                        children: /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("rect", {
                            width: 80.4,
                            height: 80.4,
                            x: 7.5,
                            y: 5.4,
                            fill: "#fff",
                            rx: 2.728
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("clipPath", {
                id: "baa08c__a",
                children: /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M64.807 15.6c-9.278.005-16.797 7.595-16.797 16.957-.003.008-.005.027-.008.039-.002-.012-.004-.03-.007-.04 0-9.36-7.52-16.95-16.798-16.956-10.23-.007-17.548 8.82-16.733 19.108 1.475 18.61 21.52 39.503 29.894 46.212 2.13 1.706 5.145 1.706 7.274 0 8.379-6.708 28.434-27.602 29.908-46.213.815-10.289-6.502-19.114-16.733-19.107"
                })
            }),
            /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("g", {
                clipPath: "url(#baa08c__a)",
                children: /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#baa08c__b)",
                    children: [
                        /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("rect", {
                            width: 80.4,
                            height: 80.4,
                            x: 7.5,
                            y: 5.4,
                            fill: "#FFECCF",
                            rx: 2.728
                        }),
                        /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M7.5 5.4h80.4v80.4H7.5z"
                        }),
                        /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("circle", {
                            cx: 52.961,
                            cy: 8.519,
                            r: 34.978,
                            fill: "url(#baa08c__c)",
                            transform: "rotate(-.08 52.96 8.52)"
                        }),
                        /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("path", {
                            fill: "url(#baa08c__d)",
                            d: "M38.187-8.802c-6.174-3.82-13.998-3.844-20.195-.06l-61.987 37.847c-6.071 3.707-6.081 12.456-.019 16.177l61.578 37.794c6.188 3.797 14.011 3.793 20.194-.014L99.21 45.113c6.03-3.712 6.041-12.413.018-16.14z"
                        }),
                        /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("path", {
                            fill: "url(#baa08c__e)",
                            d: "M19.089 43.573c1.374-30.016 26.82-53.234 56.836-51.86s53.234 26.82 51.86 56.836-26.821 53.234-56.836 51.86c-30.016-1.374-53.235-26.82-51.86-56.836"
                        }),
                        /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("path", {
                            fill: "url(#baa08c__f)",
                            d: "M-25.074 63.364c11.931-5.09 24.799-4.872 35.947-.42 22.308 8.93 35.289 3.41 44.292-18.848 4.484-11.168 13.24-20.627 25.184-25.722 23.882-10.173 51.452.865 61.61 24.676 10.158 23.81-.96 51.338-24.836 61.523-11.972 5.108-24.864 4.868-36.03.376-22.272-8.883-35.227-3.34-44.219 18.943-4.495 11.142-13.249 20.568-25.18 25.658-23.863 10.18-51.433-.858-61.585-24.655s.954-51.351 24.817-61.531"
                        }),
                        /*#__PURE__*/ (0, $iGYEi$reactjsxruntime.jsx)("circle", {
                            cx: 83.215,
                            cy: 19.926,
                            r: 34.978,
                            fill: "url(#baa08c__g)",
                            transform: "rotate(-.08 83.215 19.926)"
                        })
                    ]
                })
            })
        ]
    });
const $dfbecb2fdb2b11cd$var$ForwardRef = /*#__PURE__*/ (0, $iGYEi$react.forwardRef)($dfbecb2fdb2b11cd$var$SvgComponent);
var $dfbecb2fdb2b11cd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($dfbecb2fdb2b11cd$var$ForwardRef);
