// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fireworks_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $dgK2z$jsxs, jsx as $dgK2z$jsx} from "react/jsx-runtime";
import {forwardRef as $dgK2z$forwardRef} from "react";





const $6669275b8c61d396$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dgK2z$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dgK2z$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dgK2z$jsxs)("linearGradient", {
                        id: "3a33da__d",
                        x1: -5.977,
                        x2: 55.826,
                        y1: 39.303,
                        y2: -0.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dgK2z$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $dgK2z$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dgK2z$jsxs)("linearGradient", {
                        id: "3a33da__e",
                        x1: 45.296,
                        x2: 19.612,
                        y1: 13.02,
                        y2: 42.317,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dgK2z$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dgK2z$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dgK2z$jsxs)("linearGradient", {
                        id: "3a33da__f",
                        x1: 11.986,
                        x2: 59.166,
                        y1: 44.639,
                        y2: 16.603,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dgK2z$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dgK2z$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dgK2z$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $dgK2z$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dgK2z$jsxs)("radialGradient", {
                        id: "3a33da__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dgK2z$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $dgK2z$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dgK2z$jsxs)("radialGradient", {
                        id: "3a33da__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 16.1176 -17.0977 0 17.098 16.118)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dgK2z$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dgK2z$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dgK2z$jsx)("clipPath", {
                        id: "3a33da__b",
                        children: /*#__PURE__*/ (0, $dgK2z$jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.196,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dgK2z$jsx)("clipPath", {
                id: "3a33da__a",
                children: /*#__PURE__*/ (0, $dgK2z$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M38.432 8.473c.01-.663-.518-1.208-1.18-1.219-.663-.01-1.21.518-1.22 1.18l-.016 1.05-1.052-.016c-.662-.01-1.208.519-1.218 1.181-.01.663.519 1.209 1.181 1.219l1.05.016-.017 1.05c-.01.663.518 1.21 1.181 1.22s1.208-.518 1.219-1.18l.017-1.053 1.05.016c.663.01 1.209-.519 1.219-1.182s-.519-1.208-1.182-1.218l-1.049-.016zm-18.335.764c0-.663-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2v3.042c0 .662.537 1.2 1.2 1.2s1.2-.537 1.2-1.2zm-7.51.847c.537-.39 1.287-.27 1.677.266l1.788 2.463c.39.536.27 1.287-.266 1.676-.537.39-1.287.27-1.676-.266l-1.788-2.463c-.39-.537-.27-1.287.266-1.676m-2.389 4.6c-.63-.205-1.307.14-1.512.77-.204.63.14 1.308.771 1.512l2.892.94c.63.204 1.307-.141 1.512-.772.205-.63-.14-1.307-.77-1.511zm3.664 5.724c.204.63-.141 1.307-.771 1.511l-2.895.94c-.63.204-1.308-.141-1.512-.772-.205-.63.14-1.307.771-1.511l2.895-.94c.63-.204 1.307.141 1.512.772m2.186 4.324c.39-.536.27-1.286-.265-1.676-.537-.39-1.287-.27-1.677.265l-1.788 2.46c-.39.537-.27 1.287.266 1.677s1.286.27 1.676-.266zm4.045.534c0-.663-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2v3.042c0 .663.538 1.2 1.2 1.2.663 0 1.2-.537 1.2-1.2zm5.116-15.179c.536.39.655 1.14.265 1.676l-1.788 2.46c-.39.536-1.14.655-1.676.265s-.655-1.14-.266-1.676l1.788-2.46c.39-.536 1.14-.654 1.677-.265m3.127 6.882c.63-.204.976-.881.771-1.512-.205-.63-.882-.975-1.512-.77l-2.892.939c-.63.204-.975.881-.77 1.511s.88.976 1.511.771zm-4.404 3.441c.205-.63.882-.975 1.512-.77l2.892.938c.63.205.976.882.771 1.512s-.882.976-1.512.771l-2.892-.939c-.63-.204-.975-.881-.77-1.512m-.248 2.912c-.389-.536-1.14-.655-1.676-.266-.536.39-.655 1.14-.266 1.676l1.788 2.463c.39.536 1.14.655 1.676.266.537-.39.656-1.14.266-1.676zm9.027 11.166c.663.003 1.198.542 1.195 1.205l-.006 1.485c-.002.663-.542 1.198-1.205 1.195-.662-.003-1.198-.542-1.195-1.205l.006-1.485c.003-.663.542-1.198 1.205-1.195m-2.682.354c.575-.329.775-1.062.446-1.637-.33-.575-1.062-.775-1.638-.446l-1.29.738c-.575.33-.774 1.063-.445 1.638s1.062.775 1.637.445zm-2.921-6.172c.333-.573 1.068-.767 1.64-.433l1.285.747c.572.333.767 1.067.433 1.64-.333.573-1.067.767-1.64.434l-1.284-.747c-.573-.333-.767-1.068-.434-1.64m6.798-2.022c.003-.663-.532-1.203-1.195-1.205s-1.202.532-1.205 1.195l-.006 1.485c-.003.663.533 1.202 1.195 1.205.663.003 1.203-.533 1.205-1.195zm4.4 2.033c.33.575.13 1.308-.445 1.637l-1.29.738c-.576.33-1.309.13-1.638-.445s-.13-1.309.446-1.638l1.29-.738c.575-.329 1.308-.13 1.637.446m-1.724 4.077c-.573-.334-1.307-.14-1.64.433-.334.573-.14 1.308.433 1.641l1.284.747c.573.333 1.308.14 1.641-.434.333-.573.139-1.307-.434-1.64zM7.043 37.346c.01-.662.556-1.191 1.218-1.181l1.05.016.015-1.051c.01-.663.556-1.192 1.219-1.182.662.01 1.191.556 1.181 1.218l-.016 1.052 1.051.016c.663.01 1.192.556 1.182 1.218-.01.663-.556 1.192-1.218 1.182l-1.052-.016-.016 1.048c-.01.663-.556 1.192-1.218 1.182-.663-.01-1.192-.556-1.182-1.218l.016-1.05-1.048-.015c-.663-.01-1.192-.556-1.182-1.218"
                })
            }),
            /*#__PURE__*/ (0, $dgK2z$jsx)("g", {
                clipPath: "url(#3a33da__a)",
                children: /*#__PURE__*/ (0, $dgK2z$jsxs)("g", {
                    clipPath: "url(#3a33da__b)",
                    children: [
                        /*#__PURE__*/ (0, $dgK2z$jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.196,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $dgK2z$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M3.898 6.196h39.3v37.047h-39.3z"
                        }),
                        /*#__PURE__*/ (0, $dgK2z$jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#3a33da__c)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 9 -8.462)"
                        }),
                        /*#__PURE__*/ (0, $dgK2z$jsx)("path", {
                            fill: "url(#3a33da__d)",
                            d: "M18.9-.348c-3.018-1.76-6.842-1.771-9.872-.028l-30.3 17.44c-2.967 1.708-2.972 5.74-.008 7.454l30.1 17.415c3.024 1.75 6.848 1.748 9.87-.006l30.038-17.431c2.948-1.711 2.953-5.72.009-7.438z"
                        }),
                        /*#__PURE__*/ (0, $dgK2z$jsx)("path", {
                            fill: "url(#3a33da__e)",
                            d: "M9.561 23.785C10.233 9.955 22.671-.745 37.343-.11c14.672.633 26.021 12.358 25.35 26.19-.672 13.83-13.11 24.529-27.782 23.896-14.672-.634-26.022-12.359-25.35-26.19"
                        }),
                        /*#__PURE__*/ (0, $dgK2z$jsx)("path", {
                            fill: "url(#3a33da__f)",
                            d: "M-12.024 32.905C-6.192 30.56.098 30.66 5.548 32.712c10.904 4.115 17.249 1.57 21.65-8.685 2.192-5.146 6.471-9.505 12.31-11.853 11.673-4.687 25.15.399 30.115 11.37 4.965 10.973-.47 23.657-12.14 28.35-5.852 2.353-12.154 2.243-17.611.173-10.887-4.093-17.22-1.54-21.615 8.729-2.198 5.134-6.476 9.477-12.308 11.823-11.664 4.69-25.141-.395-30.104-11.361-4.962-10.966.467-23.662 12.131-28.353"
                        }),
                        /*#__PURE__*/ (0, $dgK2z$jsx)("ellipse", {
                            cx: 17.098,
                            cy: 16.118,
                            fill: "url(#3a33da__g)",
                            rx: 17.098,
                            ry: 16.118,
                            transform: "matrix(1 -.00132 .00148 1 23.785 -3.206)"
                        })
                    ]
                })
            })
        ]
    });
const $6669275b8c61d396$var$ForwardRef = /*#__PURE__*/ (0, $dgK2z$forwardRef)($6669275b8c61d396$var$SvgComponent);
var $6669275b8c61d396$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6669275b8c61d396$var$ForwardRef);


export {$6669275b8c61d396$export$2e2bcd8739ae039 as default};
