// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_laptop_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $42bwu$jsxs, jsx as $42bwu$jsx} from "react/jsx-runtime";
import {forwardRef as $42bwu$forwardRef} from "react";





const $1da4cfac16bbcf46$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $42bwu$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $42bwu$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $42bwu$jsxs)("linearGradient", {
                        id: "626acc__d",
                        x1: -12.705,
                        x2: 115.616,
                        y1: 79.719,
                        y2: -3.098,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $42bwu$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $42bwu$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $42bwu$jsxs)("linearGradient", {
                        id: "626acc__e",
                        x1: 93.76,
                        x2: 40.43,
                        y1: 25.147,
                        y2: 85.978,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $42bwu$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $42bwu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $42bwu$jsxs)("linearGradient", {
                        id: "626acc__f",
                        x1: 24.596,
                        x2: 122.555,
                        y1: 90.799,
                        y2: 32.586,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $42bwu$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $42bwu$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $42bwu$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $42bwu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $42bwu$jsxs)("radialGradient", {
                        id: "626acc__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 33.4654 -35.5005 0 35.5 33.465)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $42bwu$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $42bwu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $42bwu$jsxs)("radialGradient", {
                        id: "626acc__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 33.4654 -35.5005 0 35.5 33.465)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $42bwu$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $42bwu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $42bwu$jsx)("clipPath", {
                        id: "626acc__b",
                        children: /*#__PURE__*/ (0, $42bwu$jsx)("rect", {
                            width: 81.6,
                            height: 76.922,
                            x: 7.801,
                            y: 10.979,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $42bwu$jsx)("clipPath", {
                id: "626acc__a",
                children: /*#__PURE__*/ (0, $42bwu$jsx)("path", {
                    fill: "#fff",
                    d: "M81.6 66h-4.8V28.932c0-5.376-4.296-9.732-9.6-9.732H28.8c-5.304 0-9.6 4.356-9.6 9.732V66h-4.8C11.418 66 9 68.418 9 71.4s2.418 5.4 5.4 5.4h67.2c2.982 0 5.4-2.418 5.4-5.4S84.582 66 81.6 66M48 29.4c-1.326 0-2.4-1.074-2.4-2.4s1.074-2.4 2.4-2.4 2.4 1.074 2.4 2.4-1.074 2.4-2.4 2.4"
                })
            }),
            /*#__PURE__*/ (0, $42bwu$jsx)("g", {
                clipPath: "url(#626acc__a)",
                children: /*#__PURE__*/ (0, $42bwu$jsxs)("g", {
                    clipPath: "url(#626acc__b)",
                    children: [
                        /*#__PURE__*/ (0, $42bwu$jsx)("rect", {
                            width: 81.6,
                            height: 76.922,
                            x: 7.801,
                            y: 10.979,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $42bwu$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M7.8 10.979h81.6V87.9H7.8z"
                        }),
                        /*#__PURE__*/ (0, $42bwu$jsx)("ellipse", {
                            cx: 35.501,
                            cy: 33.465,
                            fill: "url(#626acc__c)",
                            rx: 35.501,
                            ry: 33.465,
                            transform: "matrix(1 -.00132 .00148 1 18.39 -19.456)"
                        }),
                        /*#__PURE__*/ (0, $42bwu$jsx)("path", {
                            fill: "url(#626acc__d)",
                            d: "M38.947-2.61C32.68-6.265 24.74-6.288 18.45-2.668l-62.912 36.21c-6.162 3.547-6.172 11.918-.02 15.478l62.497 36.159c6.281 3.633 14.221 3.629 20.496-.013l62.368-36.193c6.121-3.552 6.132-11.876.019-15.442z"
                        }),
                        /*#__PURE__*/ (0, $42bwu$jsx)("path", {
                            fill: "url(#626acc__e)",
                            d: "M19.562 47.5C20.957 18.781 46.783-3.433 77.247-2.119 107.71-.803 131.275 23.543 129.88 52.26c-1.395 28.718-27.221 50.932-57.685 49.617-30.464-1.314-54.029-25.66-52.634-54.378"
                        }),
                        /*#__PURE__*/ (0, $42bwu$jsx)("path", {
                            fill: "url(#626acc__f)",
                            d: "M-25.257 66.435c12.109-4.87 25.169-4.662 36.484-.402 22.64 8.545 35.814 3.262 44.952-18.032 4.551-10.686 13.437-19.735 25.56-24.61 24.238-9.733 52.22.827 62.53 23.608 10.309 22.782-.975 49.118-25.207 58.863-12.151 4.886-25.235 4.657-36.567.359-22.605-8.499-35.754-3.196-44.88 18.124-4.562 10.66-13.446 19.678-25.556 24.548-24.218 9.74-52.2-.821-62.504-23.589s.969-49.13 25.188-58.87"
                        }),
                        /*#__PURE__*/ (0, $42bwu$jsx)("ellipse", {
                            cx: 35.501,
                            cy: 33.465,
                            fill: "url(#626acc__g)",
                            rx: 35.501,
                            ry: 33.465,
                            transform: "matrix(1 -.00132 .00148 1 49.094 -8.543)"
                        })
                    ]
                })
            })
        ]
    });
const $1da4cfac16bbcf46$var$ForwardRef = /*#__PURE__*/ (0, $42bwu$forwardRef)($1da4cfac16bbcf46$var$SvgComponent);
var $1da4cfac16bbcf46$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1da4cfac16bbcf46$var$ForwardRef);


export {$1da4cfac16bbcf46$export$2e2bcd8739ae039 as default};
