// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_chatbubble_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4iQFX$reactjsxruntime = require("react/jsx-runtime");
var $4iQFX$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $51f801fb4eeabe9d$export$2e2bcd8739ae039);




const $51f801fb4eeabe9d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2ef790__d",
                        x1: -24.537,
                        x2: 201.681,
                        y1: 134.772,
                        y2: -2.857,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2ef790__e",
                        x1: 157.003,
                        x2: 59.956,
                        y1: 36.058,
                        y2: 140.411,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2ef790__f",
                        x1: 39.065,
                        x2: 211.102,
                        y1: 154.813,
                        y2: 58.44,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2ef790__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.639 52.464)scale(60.5342)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2ef790__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 52.948 88.515)scale(60.5342)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("clipPath", {
                        id: "2ef790__b",
                        children: /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.429,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("clipPath", {
                id: "2ef790__a",
                children: /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M119.641 35.937H40.362c-6.913 0-12.518 5.228-12.518 11.677v41.442c0 6.45 5.605 11.678 12.518 11.678h11.21c.025 5.51-.021 13.396-.068 19.402-.026 3.349 4.194 5.161 6.906 2.967l27.66-22.369h33.571c6.914 0 12.518-5.228 12.518-11.678V47.614c0-6.45-5.604-11.677-12.518-11.677"
                })
            }),
            /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("g", {
                clipPath: "url(#2ef790__a)",
                children: /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#2ef790__b)",
                    children: [
                        /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.429,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M10.43 10.429h139.142V149.57H10.43z"
                        }),
                        /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("circle", {
                            cx: 89.103,
                            cy: 15.826,
                            r: 60.534,
                            fill: "url(#2ef790__c)",
                            transform: "rotate(-.08 89.103 15.826)"
                        }),
                        /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("path", {
                            fill: "url(#2ef790__d)",
                            d: "M63.539-14.15c-10.686-6.612-24.225-6.653-34.95-.104L-78.689 51.244C-89.194 57.66-89.212 72.8-78.72 79.24l106.568 65.406c10.71 6.572 24.248 6.564 34.949-.023l106.347-65.468c10.437-6.425 10.456-21.483.032-27.932z"
                        }),
                        /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("path", {
                            fill: "url(#2ef790__e)",
                            d: "M30.483 76.49c2.378-51.946 46.417-92.128 98.362-89.75 51.946 2.378 92.128 46.417 89.75 98.362-2.378 51.946-46.416 92.128-98.362 89.75s-92.128-46.416-89.75-98.362"
                        }),
                        /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("path", {
                            fill: "url(#2ef790__f)",
                            d: "M-45.943 110.741c20.649-8.809 42.918-8.432 62.211-.727 38.607 15.456 61.07 5.9 76.651-32.618 7.761-19.328 22.913-35.697 43.585-44.516 41.33-17.604 89.044 1.497 106.623 42.705s-1.662 88.847-42.982 106.474c-20.718 8.839-43.03 8.425-62.353.65-38.544-15.373-60.965-5.781-76.526 32.784-7.78 19.282-22.93 35.595-43.578 44.404-41.297 17.617-89.01-1.485-106.58-42.669-17.57-41.185 1.652-88.87 42.95-106.487"
                        }),
                        /*#__PURE__*/ (0, $4iQFX$reactjsxruntime.jsx)("circle", {
                            cx: 141.463,
                            cy: 35.567,
                            r: 60.534,
                            fill: "url(#2ef790__g)",
                            transform: "rotate(-.08 141.463 35.567)"
                        })
                    ]
                })
            })
        ]
    });
const $51f801fb4eeabe9d$var$ForwardRef = /*#__PURE__*/ (0, $4iQFX$react.forwardRef)($51f801fb4eeabe9d$var$SvgComponent);
var $51f801fb4eeabe9d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($51f801fb4eeabe9d$var$ForwardRef);
