// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_userAvatar_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gLCJk$reactjsxruntime = require("react/jsx-runtime");
var $gLCJk$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0774e78ddfadff03$export$2e2bcd8739ae039);




const $0774e78ddfadff03$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsxs)("linearGradient", {
                        id: "41e2d5__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsxs)("linearGradient", {
                        id: "41e2d5__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsxs)("linearGradient", {
                        id: "41e2d5__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsxs)("radialGradient", {
                        id: "41e2d5__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsxs)("radialGradient", {
                        id: "41e2d5__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("clipPath", {
                        id: "41e2d5__b",
                        children: /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("clipPath", {
                id: "41e2d5__a",
                children: /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M114.843 123.843C105.28 131.453 93.171 136 80 136c-13.12 0-25.184-4.511-34.73-12.067.304-.318.56-.692.752-1.116 5.874-12.982 18.974-22.069 33.978-22.069 14.878 0 27.885 8.944 33.831 21.752.246.532.595.984 1.012 1.343m5.904-5.428C113.408 103.306 97.864 92.748 80 92.748c-17.863 0-33.409 10.548-40.748 25.666C29.796 108.387 24 94.87 24 80c0-30.928 25.072-56 56-56s56 25.072 56 56c0 14.871-5.797 28.388-15.253 38.415m-57.26-55.579c0-10.103 7.628-17.854 16.513-17.854 8.884 0 16.512 7.751 16.512 17.854S88.884 80.69 80 80.69s-16.513-7.751-16.513-17.854M80 36.982c-13.773 0-24.513 11.818-24.513 25.854S66.227 88.69 80 88.69c13.772 0 24.512-11.818 24.512-25.854S93.772 36.982 80 36.982"
                })
            }),
            /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("g", {
                clipPath: "url(#41e2d5__a)",
                children: /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#41e2d5__b)",
                    children: [
                        /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#41e2d5__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("path", {
                            fill: "url(#41e2d5__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("path", {
                            fill: "url(#41e2d5__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("path", {
                            fill: "url(#41e2d5__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $gLCJk$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#41e2d5__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $0774e78ddfadff03$var$ForwardRef = /*#__PURE__*/ (0, $gLCJk$react.forwardRef)($0774e78ddfadff03$var$SvgComponent);
var $0774e78ddfadff03$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0774e78ddfadff03$var$ForwardRef);
