// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_imageStack_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $jVPuM$reactjsxruntime = require("react/jsx-runtime");
var $jVPuM$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $299474de19a76f44$export$2e2bcd8739ae039);




const $299474de19a76f44$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d8faef__d",
                        x1: -14.673,
                        x2: 192.277,
                        y1: 131.234,
                        y2: -2.329,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d8faef__e",
                        x1: 157.026,
                        x2: 71.019,
                        y1: 43.223,
                        y2: 141.328,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d8faef__f",
                        x1: 45.482,
                        x2: 203.466,
                        y1: 149.102,
                        y2: 55.22,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d8faef__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.971 -57.253 0 57.253 53.971)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d8faef__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.971 -57.253 0 57.253 53.971)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("clipPath", {
                        id: "d8faef__b",
                        children: /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("rect", {
                            width: 131.6,
                            height: 124.056,
                            x: 18.398,
                            y: 20.372,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("clipPath", {
                id: "d8faef__a",
                children: /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M125.43 48.47v49c0 8.84-7.16 16-16 16h-74c-8.84 0-16-7.16-16-16v-49c0-8.84 7.16-16 16-16h74c8.84 0 16 7.16 16 16m-14.04 6.88c0-4.82-3.91-8.73-8.73-8.73s-8.73 3.91-8.73 8.73 3.91 8.73 8.73 8.73 8.73-3.91 8.73-8.73M28.98 96.22c-2.56 3.99.3 9.24 5.05 9.24v-.01h76.18c4.75 0 7.61-5.25 5.05-9.24l-11.78-18.35c-2.36-3.68-7.74-3.68-10.1 0L80.02 98.68 58.76 65.56c-2.36-3.68-7.74-3.68-10.1 0zM136.57 66.49c0-2.21 1.79-4 4-4h.01c2.21 0 4 1.79 4 4v41.05c0 13.23-10.77 24-24 24l-65.94-.05c-2.21 0-4-1.79-4-4s1.79-4 4-4l65.93.05c8.82 0 16-7.18 16-16z"
                })
            }),
            /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("g", {
                clipPath: "url(#d8faef__a)",
                children: /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#d8faef__b)",
                    children: [
                        /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("rect", {
                            width: 131.6,
                            height: 124.056,
                            x: 18.398,
                            y: 20.372,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M18.398 20.372h131.6v124.056h-131.6z"
                        }),
                        /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("ellipse", {
                            cx: 57.253,
                            cy: 53.971,
                            fill: "url(#d8faef__c)",
                            rx: 57.253,
                            ry: 53.971,
                            transform: "matrix(1 -.00132 .00148 1 35.477 -28.712)"
                        }),
                        /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("path", {
                            fill: "url(#d8faef__d)",
                            d: "M68.628-1.541c-10.106-5.896-22.911-5.932-33.056-.094l-101.46 58.398c-9.938 5.72-9.954 19.22-.031 24.96l100.79 58.316c10.13 5.859 22.935 5.852 33.055-.021l100.584-58.37c9.87-5.729 9.888-19.153.03-24.903z"
                        }),
                        /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("path", {
                            fill: "url(#d8faef__e)",
                            d: "M37.364 79.27c2.25-46.313 43.9-82.138 93.031-80.018 49.13 2.12 87.134 41.384 84.885 87.697-2.25 46.314-43.901 82.14-93.031 80.019s-87.134-41.383-84.885-87.697"
                        }),
                        /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("path", {
                            fill: "url(#d8faef__f)",
                            d: "M-34.917 109.808c19.529-7.854 40.591-7.517 58.839-.647 36.513 13.78 57.76 5.26 72.496-29.082 7.34-17.233 21.671-31.827 41.222-39.69 39.09-15.696 84.217 1.335 100.844 38.076 16.627 36.739-1.572 79.213-40.652 94.929-19.595 7.88-40.698 7.511-58.974.58-36.454-13.707-57.66-5.154-72.378 29.229-7.358 17.191-21.686 31.736-41.215 39.589-39.059 15.708-84.186-1.323-100.804-38.042s1.563-79.234 40.622-94.942"
                        }),
                        /*#__PURE__*/ (0, $jVPuM$reactjsxruntime.jsx)("ellipse", {
                            cx: 57.253,
                            cy: 53.971,
                            fill: "url(#d8faef__g)",
                            rx: 57.253,
                            ry: 53.971,
                            transform: "matrix(1 -.00132 .00148 1 84.996 -11.111)"
                        })
                    ]
                })
            })
        ]
    });
const $299474de19a76f44$var$ForwardRef = /*#__PURE__*/ (0, $jVPuM$react.forwardRef)($299474de19a76f44$var$SvgComponent);
var $299474de19a76f44$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($299474de19a76f44$var$ForwardRef);
