// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_infiniteLooping_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $96wS6$jsxs, jsx as $96wS6$jsx} from "react/jsx-runtime";
import {forwardRef as $96wS6$forwardRef} from "react";





const $564caba43b0d6982$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $96wS6$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $96wS6$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $96wS6$jsxs)("linearGradient", {
                        id: "f673fa__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.894,
                        y2: -4.584,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $96wS6$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $96wS6$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $96wS6$jsxs)("linearGradient", {
                        id: "f673fa__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.841,
                        y2: 44.839,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $96wS6$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $96wS6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $96wS6$jsxs)("linearGradient", {
                        id: "f673fa__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.808,
                        y2: 16.562,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $96wS6$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $96wS6$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $96wS6$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $96wS6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $96wS6$jsxs)("radialGradient", {
                        id: "f673fa__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $96wS6$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $96wS6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $96wS6$jsxs)("radialGradient", {
                        id: "f673fa__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $96wS6$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $96wS6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $96wS6$jsx)("clipPath", {
                        id: "f673fa__b",
                        children: /*#__PURE__*/ (0, $96wS6$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: -0.001,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $96wS6$jsx)("clipPath", {
                id: "f673fa__a",
                children: /*#__PURE__*/ (0, $96wS6$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M21.55 22.054c.69-.002 1.248-.563 1.247-1.253l-.018-7.506c-.001-.69-.562-1.249-1.253-1.247-.69.001-1.248.562-1.247 1.253l.01 4.455c-2.007-2.07-4.82-3.358-7.933-3.358-6.103 0-11.051 4.948-11.051 11.051S6.253 36.5 12.356 36.5c2.532 0 4.868-.852 6.732-2.286q.043-.033.083-.07l.034-.026c1.373-1.077 2.648-2.64 3.644-4.066 1-1.434 1.784-2.828 2.152-3.64.285-.629.99-2.123 1.924-3.692.803-1.352 1.717-2.656 2.618-3.532q.042-.033.083-.07c1.52-1.38 3.535-2.22 5.749-2.22 4.723 0 8.551 3.828 8.551 8.551S40.098 34 35.375 34c-2.888 0-5.442-1.43-6.992-3.627-.398-.564-1.178-.698-1.742-.3s-.699 1.178-.3 1.742c1.998 2.832 5.3 4.685 9.034 4.685 6.103 0 11.051-4.948 11.051-11.051s-4.948-11.051-11.051-11.051c-2.527 0-4.857.849-6.719 2.276q-.081.045-.158.107c-1.443 1.137-2.744 3.016-3.722 4.66-.998 1.68-1.747 3.266-2.052 3.939-.29.642-.99 1.902-1.925 3.241-.94 1.346-2.05 2.677-3.138 3.53q-.038.03-.073.063l-.024.018C16.122 33.342 14.318 34 12.356 34c-4.723 0-8.551-3.828-8.551-8.551s3.828-8.551 8.551-8.551c2.44 0 4.642 1.02 6.201 2.663l-4.52.01c-.69.002-1.248.563-1.246 1.253.001.69.562 1.249 1.253 1.247z"
                })
            }),
            /*#__PURE__*/ (0, $96wS6$jsx)("g", {
                clipPath: "url(#f673fa__a)",
                children: /*#__PURE__*/ (0, $96wS6$jsxs)("g", {
                    clipPath: "url(#f673fa__b)",
                    children: [
                        /*#__PURE__*/ (0, $96wS6$jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: -0.001,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $96wS6$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0-.001h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $96wS6$jsx)("circle", {
                            cx: 27.142,
                            cy: 1.861,
                            r: 20.883,
                            fill: "url(#f673fa__c)",
                            transform: "rotate(-.08 27.142 1.861)"
                        }),
                        /*#__PURE__*/ (0, $96wS6$jsx)("path", {
                            fill: "url(#f673fa__d)",
                            d: "M18.321-8.48c-3.686-2.28-8.357-2.295-12.057-.036L-30.743 14.08c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.3c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $96wS6$jsx)("path", {
                            fill: "url(#f673fa__e)",
                            d: "M6.916 22.788C7.736 4.868 22.93-8.993 40.85-8.172S72.63 7.84 71.809 25.758c-.82 17.92-16.012 31.782-33.931 30.962-17.92-.82-31.782-16.013-30.962-33.933"
                        }),
                        /*#__PURE__*/ (0, $96wS6$jsx)("path", {
                            fill: "url(#f673fa__f)",
                            d: "M-19.448 34.604c7.124-3.039 14.806-2.908 21.462-.25C15.332 39.686 23.08 36.389 28.456 23.1c2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $96wS6$jsx)("circle", {
                            cx: 45.201,
                            cy: 8.671,
                            r: 20.883,
                            fill: "url(#f673fa__g)",
                            transform: "rotate(-.08 45.2 8.67)"
                        })
                    ]
                })
            })
        ]
    });
const $564caba43b0d6982$var$ForwardRef = /*#__PURE__*/ (0, $96wS6$forwardRef)($564caba43b0d6982$var$SvgComponent);
var $564caba43b0d6982$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($564caba43b0d6982$var$ForwardRef);


export {$564caba43b0d6982$export$2e2bcd8739ae039 as default};
