// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_conversationbubbles_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $jIQch$jsxs, jsx as $jIQch$jsx} from "react/jsx-runtime";
import {forwardRef as $jIQch$forwardRef} from "react";





const $29d24e34c83405fc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jIQch$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jIQch$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jIQch$jsxs)("linearGradient", {
                        id: "797dbd__d",
                        x1: -24.537,
                        x2: 201.681,
                        y1: 134.772,
                        y2: -2.857,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jIQch$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $jIQch$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jIQch$jsxs)("linearGradient", {
                        id: "797dbd__e",
                        x1: 157.003,
                        x2: 59.956,
                        y1: 36.058,
                        y2: 140.411,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jIQch$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jIQch$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jIQch$jsxs)("linearGradient", {
                        id: "797dbd__f",
                        x1: 39.065,
                        x2: 211.102,
                        y1: 154.813,
                        y2: 58.44,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jIQch$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jIQch$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jIQch$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $jIQch$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jIQch$jsxs)("radialGradient", {
                        id: "797dbd__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.639 52.464)scale(60.5342)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jIQch$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $jIQch$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jIQch$jsxs)("radialGradient", {
                        id: "797dbd__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 52.948 88.515)scale(60.5342)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jIQch$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jIQch$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jIQch$jsx)("clipPath", {
                        id: "797dbd__b",
                        children: /*#__PURE__*/ (0, $jIQch$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.429,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jIQch$jsxs)("clipPath", {
                id: "797dbd__a",
                children: [
                    /*#__PURE__*/ (0, $jIQch$jsx)("path", {
                        fill: "#fff",
                        d: "M131.619 69.36h-15.436v11.455c0 9.893-8.368 17.94-18.654 17.94H72.325v7.452c0 4.473 3.888 8.1 8.684 8.1h18.906l19.186 15.517c1.882 1.521 4.809.265 4.79-2.059-.032-4.165-.064-9.636-.047-13.458h7.776c4.796 0 8.684-3.626 8.684-8.1V77.46c0-4.474-3.888-8.1-8.684-8.1"
                    }),
                    /*#__PURE__*/ (0, $jIQch$jsx)("path", {
                        fill: "#fff",
                        d: "M97.53 90.716c5.861 0 10.613-4.433 10.613-9.901V45.68c0-5.468-4.752-9.9-10.613-9.9H30.317c-5.862 0-10.614 4.432-10.614 9.9v35.135c0 5.468 4.752 9.9 10.614 9.9h9.504c.02 4.672-.019 11.358-.058 16.449-.022 2.84 3.555 4.376 5.855 2.516l23.45-18.964z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jIQch$jsx)("g", {
                clipPath: "url(#797dbd__a)",
                children: /*#__PURE__*/ (0, $jIQch$jsxs)("g", {
                    clipPath: "url(#797dbd__b)",
                    children: [
                        /*#__PURE__*/ (0, $jIQch$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.429,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $jIQch$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M10.43 10.429h139.142V149.57H10.43z"
                        }),
                        /*#__PURE__*/ (0, $jIQch$jsx)("circle", {
                            cx: 89.103,
                            cy: 15.826,
                            r: 60.534,
                            fill: "url(#797dbd__c)",
                            transform: "rotate(-.08 89.103 15.826)"
                        }),
                        /*#__PURE__*/ (0, $jIQch$jsx)("path", {
                            fill: "url(#797dbd__d)",
                            d: "M63.539-14.15c-10.686-6.612-24.225-6.653-34.95-.104L-78.689 51.244C-89.194 57.66-89.212 72.8-78.72 79.24l106.568 65.406c10.71 6.572 24.248 6.564 34.949-.023l106.347-65.468c10.437-6.425 10.456-21.483.032-27.932z"
                        }),
                        /*#__PURE__*/ (0, $jIQch$jsx)("path", {
                            fill: "url(#797dbd__e)",
                            d: "M30.483 76.49c2.378-51.946 46.417-92.128 98.362-89.75 51.946 2.378 92.128 46.417 89.75 98.362-2.378 51.946-46.416 92.128-98.362 89.75s-92.128-46.416-89.75-98.362"
                        }),
                        /*#__PURE__*/ (0, $jIQch$jsx)("path", {
                            fill: "url(#797dbd__f)",
                            d: "M-45.943 110.741c20.649-8.809 42.918-8.432 62.211-.727 38.607 15.456 61.07 5.9 76.651-32.618 7.761-19.328 22.913-35.697 43.585-44.516 41.33-17.604 89.044 1.497 106.623 42.705s-1.662 88.847-42.982 106.474c-20.718 8.839-43.03 8.425-62.353.65-38.544-15.373-60.965-5.781-76.526 32.784-7.78 19.282-22.93 35.595-43.578 44.404-41.297 17.617-89.01-1.485-106.58-42.669-17.57-41.185 1.652-88.87 42.95-106.487"
                        }),
                        /*#__PURE__*/ (0, $jIQch$jsx)("circle", {
                            cx: 141.463,
                            cy: 35.567,
                            r: 60.534,
                            fill: "url(#797dbd__g)",
                            transform: "rotate(-.08 141.463 35.567)"
                        })
                    ]
                })
            })
        ]
    });
const $29d24e34c83405fc$var$ForwardRef = /*#__PURE__*/ (0, $jIQch$forwardRef)($29d24e34c83405fc$var$SvgComponent);
var $29d24e34c83405fc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($29d24e34c83405fc$var$ForwardRef);


export {$29d24e34c83405fc$export$2e2bcd8739ae039 as default};
