// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_magicWand_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $kSHqN$jsxs, jsx as $kSHqN$jsx} from "react/jsx-runtime";
import {forwardRef as $kSHqN$forwardRef} from "react";





const $2e2952cf8a17c9cf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kSHqN$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kSHqN$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kSHqN$jsxs)("linearGradient", {
                        id: "bee420__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.167,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kSHqN$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kSHqN$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kSHqN$jsxs)("linearGradient", {
                        id: "bee420__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kSHqN$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kSHqN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kSHqN$jsxs)("linearGradient", {
                        id: "bee420__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.617,
                        y2: 33.125,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kSHqN$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kSHqN$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kSHqN$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kSHqN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kSHqN$jsxs)("radialGradient", {
                        id: "bee420__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kSHqN$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kSHqN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kSHqN$jsxs)("radialGradient", {
                        id: "bee420__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kSHqN$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kSHqN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kSHqN$jsx)("clipPath", {
                        id: "bee420__b",
                        children: /*#__PURE__*/ (0, $kSHqN$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kSHqN$jsx)("clipPath", {
                id: "bee420__a",
                children: /*#__PURE__*/ (0, $kSHqN$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "m79.614 26.756-4.233.845c-.56.113-1.025.49-1.252 1.013l-1.707 3.963c-.673 1.562-2.957 1.32-3.289-.344L68.287 28c-.113-.56-.49-1.025-1.012-1.252l-3.963-1.707c-1.562-.673-1.32-2.957.344-3.289l4.233-.846c.56-.112 1.025-.49 1.251-1.012l1.708-3.963c.673-1.562 2.957-1.32 3.288.344l.846 4.233c.113.56.49 1.025 1.013 1.251l3.963 1.708c1.562.673 1.32 2.957-.344 3.289M43.901 30.38l-3.496.698c-.463.094-.847.406-1.034.837l-1.41 3.273c-.556 1.29-2.443 1.09-2.716-.284l-.7-3.496c-.093-.463-.404-.847-.835-1.034l-3.274-1.41c-1.29-.556-1.09-2.443.284-2.717l3.497-.698c.462-.094.846-.405 1.033-.836l1.41-3.274c.557-1.29 2.443-1.09 2.717.284l.699 3.497c.093.462.405.846.835 1.033l3.274 1.41c1.29.557 1.09 2.443-.284 2.717m31.034 25.917 3.496-.699c1.375-.273 1.575-2.16.284-2.715l-3.273-1.411c-.431-.187-.743-.571-.836-1.034l-.7-3.496c-.273-1.375-2.16-1.575-2.715-.284l-1.41 3.273c-.188.431-.572.743-1.034.836l-3.497.7c-1.374.273-1.574 2.16-.284 2.715l3.274 1.41c.43.188.743.572.836 1.034l.699 3.497c.273 1.374 2.16 1.574 2.715.284l1.411-3.274c.187-.43.571-.742 1.034-.836m-21.612-21.67c2.254-2.254 5.91-2.254 8.164 0s2.254 5.91 0 8.163l-6.058 6.058-8.164-8.163zm-8.886 8.886 8.163 8.164-27.75 27.75c-2.254 2.254-5.909 2.254-8.163 0s-2.255-5.91 0-8.164z"
                })
            }),
            /*#__PURE__*/ (0, $kSHqN$jsx)("g", {
                clipPath: "url(#bee420__a)",
                children: /*#__PURE__*/ (0, $kSHqN$jsxs)("g", {
                    clipPath: "url(#bee420__b)",
                    children: [
                        /*#__PURE__*/ (0, $kSHqN$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $kSHqN$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $kSHqN$jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#bee420__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $kSHqN$jsx)("path", {
                            fill: "url(#bee420__d)",
                            d: "M36.643-16.958c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.427-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.433 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $kSHqN$jsx)("path", {
                            fill: "url(#bee420__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $kSHqN$jsx)("path", {
                            fill: "url(#bee420__f)",
                            d: "M-38.895 69.21c14.246-6.078 29.61-5.818 42.922-.502 26.636 10.664 42.135 4.071 52.885-22.504 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.314 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $kSHqN$jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#bee420__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $2e2952cf8a17c9cf$var$ForwardRef = /*#__PURE__*/ (0, $kSHqN$forwardRef)($2e2952cf8a17c9cf$var$SvgComponent);
var $2e2952cf8a17c9cf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2e2952cf8a17c9cf$var$ForwardRef);


export {$2e2952cf8a17c9cf$export$2e2bcd8739ae039 as default};
