// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_lightbulb_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $d9HCc$reactjsxruntime = require("react/jsx-runtime");
var $d9HCc$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1b80fa1290ba20a2$export$2e2bcd8739ae039);




const $1b80fa1290ba20a2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ff94ee__d",
                        x1: -14.724,
                        x2: 121.006,
                        y1: 80.862,
                        y2: -1.715,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ff94ee__e",
                        x1: 94.203,
                        x2: 35.974,
                        y1: 21.634,
                        y2: 84.246,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ff94ee__f",
                        x1: 23.438,
                        x2: 126.661,
                        y1: 92.887,
                        y2: 35.063,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ff94ee__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 21.983 31.478)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ff94ee__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 31.768 53.107)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("clipPath", {
                        id: "ff94ee__b",
                        children: /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.257,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("clipPath", {
                id: "ff94ee__a",
                children: /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M46.42 16.31c-10.654.752-19.283 9.35-20.07 20.002-.585 7.91 3.074 15.002 8.933 19.242 1.905 1.378 3.066 3.557 3.066 5.909v5.594h19.296v-5.59c0-2.294 1.061-4.485 2.929-5.815 5.526-3.934 9.131-10.39 9.131-17.69 0-12.51-10.583-22.55-23.285-21.652M60.057 37.81c-1.249 0-2.261-1.012-2.261-2.261 0-4.073-3.314-7.387-7.387-7.387-1.249 0-2.261-1.012-2.261-2.261s1.012-2.261 2.261-2.261c6.567 0 11.91 5.342 11.91 11.909 0 1.249-1.013 2.261-2.262 2.261M38.352 75.348c0 1.998 1.62 3.618 3.618 3.618h12.06c1.998 0 3.618-1.62 3.618-3.618v-3.769H38.352z"
                })
            }),
            /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ff94ee__a)",
                children: /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ff94ee__b)",
                    children: [
                        /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.257,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6.258 6.257h83.485v83.485H6.258z"
                        }),
                        /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("circle", {
                            cx: 53.461,
                            cy: 9.495,
                            r: 36.321,
                            fill: "url(#ff94ee__c)",
                            transform: "rotate(-.08 53.461 9.495)"
                        }),
                        /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("path", {
                            fill: "url(#ff94ee__d)",
                            d: "M38.121-8.49c-6.412-3.968-14.535-3.992-20.97-.063l-64.366 39.3c-6.304 3.848-6.314 12.933-.02 16.797l63.941 39.244c6.426 3.943 14.55 3.938 20.97-.015l63.808-39.28c6.262-3.855 6.274-12.89.019-16.76z"
                        }),
                        /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("path", {
                            fill: "url(#ff94ee__e)",
                            d: "M18.29 45.893c1.428-31.167 27.85-55.277 59.018-53.85s55.277 27.85 53.85 59.018-27.85 55.276-59.017 53.849c-31.168-1.426-55.277-27.85-53.85-59.017"
                        }),
                        /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("path", {
                            fill: "url(#ff94ee__f)",
                            d: "M-27.566 66.444c12.389-5.286 25.75-5.059 37.326-.436 23.164 9.274 36.642 3.54 45.99-19.57 4.657-11.598 13.749-21.42 26.152-26.71 24.798-10.563 53.426.898 63.974 25.623 10.547 24.724-.998 53.307-25.79 63.884-12.43 5.303-25.818 5.055-37.412.39-23.126-9.224-36.579-3.469-45.915 19.67C32.09 140.864 23 150.652 10.612 155.937c-24.778 10.571-53.406-.89-63.948-25.601s.991-53.322 25.77-63.892"
                        }),
                        /*#__PURE__*/ (0, $d9HCc$reactjsxruntime.jsx)("circle", {
                            cx: 84.875,
                            cy: 21.339,
                            r: 36.321,
                            fill: "url(#ff94ee__g)",
                            transform: "rotate(-.08 84.875 21.34)"
                        })
                    ]
                })
            })
        ]
    });
const $1b80fa1290ba20a2$var$ForwardRef = /*#__PURE__*/ (0, $d9HCc$react.forwardRef)($1b80fa1290ba20a2$var$SvgComponent);
var $1b80fa1290ba20a2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1b80fa1290ba20a2$var$ForwardRef);
