// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_paperclip_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bbgZ6$jsxs, jsx as $bbgZ6$jsx} from "react/jsx-runtime";
import {forwardRef as $bbgZ6$forwardRef} from "react";





const $cd82304e618bcb82$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bbgZ6$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bbgZ6$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bbgZ6$jsxs)("linearGradient", {
                        id: "cd14ba__d",
                        x1: -3.949,
                        x2: 55.306,
                        y1: 37.952,
                        y2: -0.29,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bbgZ6$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bbgZ6$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bbgZ6$jsxs)("linearGradient", {
                        id: "cd14ba__e",
                        x1: 45.213,
                        x2: 20.587,
                        y1: 12.753,
                        y2: 40.843,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bbgZ6$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bbgZ6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bbgZ6$jsxs)("linearGradient", {
                        id: "cd14ba__f",
                        x1: 13.273,
                        x2: 58.507,
                        y1: 43.068,
                        y2: 16.188,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bbgZ6$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bbgZ6$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bbgZ6$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bbgZ6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bbgZ6$jsxs)("radialGradient", {
                        id: "cd14ba__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bbgZ6$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bbgZ6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bbgZ6$jsxs)("radialGradient", {
                        id: "cd14ba__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bbgZ6$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bbgZ6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bbgZ6$jsx)("clipPath", {
                        id: "cd14ba__b",
                        children: /*#__PURE__*/ (0, $bbgZ6$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.21,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bbgZ6$jsx)("clipPath", {
                id: "cd14ba__a",
                children: /*#__PURE__*/ (0, $bbgZ6$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    stroke: "#fff",
                    strokeLinecap: "round",
                    strokeLinejoin: "round",
                    strokeWidth: 0.6,
                    d: "M10.608 26.865c-3.342 3.325-3.342 8.719 0 12.043 3.338 3.322 8.747 3.322 12.086 0l13.52-13.454c.47-.467.472-1.227.004-1.697s-1.227-.471-1.697-.004l-13.52 13.454c-2.402 2.39-6.299 2.39-8.7 0-2.4-2.387-2.4-6.254 0-8.64L30.817 10.14c1.529-1.522 4.01-1.522 5.54 0 1.525 1.518 1.525 3.977 0 5.495L19.76 32.152c-.639.636-1.676.636-2.315 0-.635-.632-.635-1.654 0-2.286l11.601-11.544c.47-.468.472-1.227.004-1.697s-1.227-.472-1.697-.004l-11.6 11.544c-1.579 1.57-1.579 4.118 0 5.688 1.574 1.567 4.125 1.567 5.7 0L38.05 17.338c2.469-2.457 2.469-6.442 0-8.898-2.465-2.454-6.46-2.454-8.925 0z"
                })
            }),
            /*#__PURE__*/ (0, $bbgZ6$jsx)("g", {
                clipPath: "url(#cd14ba__a)",
                children: /*#__PURE__*/ (0, $bbgZ6$jsxs)("g", {
                    clipPath: "url(#cd14ba__b)",
                    children: [
                        /*#__PURE__*/ (0, $bbgZ6$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.21,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $bbgZ6$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M5.52 6.21H43.2v35.52H5.52z"
                        }),
                        /*#__PURE__*/ (0, $bbgZ6$jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#cd14ba__c)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 10.41 -7.844)"
                        }),
                        /*#__PURE__*/ (0, $bbgZ6$jsx)("path", {
                            fill: "url(#cd14ba__d)",
                            d: "M19.902-.065c-2.894-1.688-6.56-1.698-9.465-.026l-29.05 16.72c-2.845 1.638-2.85 5.503-.009 7.147l28.859 16.697c2.9 1.678 6.567 1.676 9.464-.006l28.8-16.712c2.826-1.64 2.83-5.485.008-7.131z"
                        }),
                        /*#__PURE__*/ (0, $bbgZ6$jsx)("path", {
                            fill: "url(#cd14ba__e)",
                            d: "M10.951 23.074C11.595 9.814 23.521-.444 37.588.163s24.949 11.849 24.305 25.11-12.57 23.518-26.637 22.91c-14.067-.606-24.949-11.848-24.305-25.109"
                        }),
                        /*#__PURE__*/ (0, $bbgZ6$jsx)("path", {
                            fill: "url(#cd14ba__f)",
                            d: "M-9.747 31.817c5.591-2.248 11.622-2.152 16.847-.185 10.455 3.946 16.538 1.506 20.757-8.327 2.102-4.934 6.205-9.113 11.803-11.364 11.192-4.494 24.113.382 28.874 10.902s-.45 22.68-11.64 27.18c-5.61 2.257-11.652 2.151-16.885.167-10.438-3.925-16.51-1.476-20.724 8.368-2.107 4.923-6.21 9.087-11.8 11.336C-3.7 74.39-16.62 69.514-21.379 59s.447-22.686 11.63-27.184"
                        }),
                        /*#__PURE__*/ (0, $bbgZ6$jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#cd14ba__g)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 24.586 -2.805)"
                        })
                    ]
                })
            })
        ]
    });
const $cd82304e618bcb82$var$ForwardRef = /*#__PURE__*/ (0, $bbgZ6$forwardRef)($cd82304e618bcb82$var$SvgComponent);
var $cd82304e618bcb82$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cd82304e618bcb82$var$ForwardRef);


export {$cd82304e618bcb82$export$2e2bcd8739ae039 as default};
