// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_megaphonePromote2_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $i5QHx$reactjsxruntime = require("react/jsx-runtime");
var $i5QHx$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ac1681db40d1da7f$export$2e2bcd8739ae039);




const $ac1681db40d1da7f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ecad3a__d",
                        x1: -16.921,
                        x2: 189.087,
                        y1: 131.011,
                        y2: -1.943,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ecad3a__e",
                        x1: 153.995,
                        x2: 68.38,
                        y1: 43.401,
                        y2: 141.06,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ecad3a__f",
                        x1: 42.959,
                        x2: 200.223,
                        y1: 148.798,
                        y2: 55.343,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ecad3a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ecad3a__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("clipPath", {
                        id: "ecad3a__b",
                        children: /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 16,
                            y: 20.654,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("clipPath", {
                id: "ecad3a__a",
                children: /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m69.24 48.02 38.56-11.71.01-.01c5.75-1.75 11.55 2.55 11.55 8.56v43.86c0 6.01-5.8 10.31-11.55 8.56L69.25 85.55h-.04l10.44 24.25c2.24 5.18-.14 11.19-5.31 13.43-5.18 2.24-11.19-.14-13.43-5.31L46.84 85.56h-5.32c-10.36 0-18.77-8.41-18.77-18.77s8.4-18.77 18.77-18.77zm67.18 34.34 5.75 1.82c2.1.67 3.27 2.91 2.6 5.02-.54 1.7-2.11 2.79-3.81 2.79-.4 0-.81-.06-1.21-.19L134 89.98c-2.1-.67-3.27-2.91-2.6-5.02.67-2.1 2.91-3.27 5.02-2.6m-1.209-31.48c.4 0 .81-.06 1.21-.19l5.75-1.82c2.11-.67 3.27-2.92 2.6-5.02-.67-2.11-2.91-3.27-5.02-2.6l-5.75 1.82c-2.11.67-3.27 2.92-2.6 5.02.54 1.7 2.11 2.79 3.81 2.79m6.02 19.65h-6.02c-2.21 0-4-1.79-4-4s1.79-4 4-4h6.02c2.21 0 4 1.79 4 4s-1.79 4-4 4"
                })
            }),
            /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ecad3a__a)",
                children: /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ecad3a__b)",
                    children: [
                        /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 16,
                            y: 20.654,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M16 20.654h131v123.491H16z"
                        }),
                        /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#ecad3a__c)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 33 -28.206)"
                        }),
                        /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("path", {
                            fill: "url(#ecad3a__d)",
                            d: "M66.002-1.16C55.94-7.027 43.193-7.063 33.095-1.251l-101 58.132c-9.891 5.693-9.907 19.132-.03 24.847L32.4 139.776c10.082 5.833 22.83 5.826 32.904-.021l100.125-58.103c9.826-5.703 9.844-19.067.03-24.79z"
                        }),
                        /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("path", {
                            fill: "url(#ecad3a__e)",
                            d: "M34.878 79.285C37.118 33.183 78.58-2.48 127.485-.369c48.907 2.11 86.738 41.195 84.499 87.298s-43.701 81.765-92.607 79.655c-48.906-2.111-86.738-41.196-84.499-87.299"
                        }),
                        /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("path", {
                            fill: "url(#ecad3a__f)",
                            d: "M-37.075 109.683c19.44-7.818 40.406-7.483 58.57-.645 36.349 13.718 57.498 5.237 72.167-28.949 7.307-17.154 21.572-31.682 41.035-39.509 38.911-15.624 83.833 1.33 100.384 37.902s-1.565 78.853-40.467 94.497c-19.506 7.845-40.512 7.477-58.705.577-36.288-13.644-57.398-5.13-72.048 29.096-7.325 17.113-21.588 31.592-41.028 39.41-38.88 15.636-83.803-1.318-100.344-37.87-16.542-36.552 1.555-78.873 40.436-94.509"
                        }),
                        /*#__PURE__*/ (0, $i5QHx$reactjsxruntime.jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#ecad3a__g)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 82.293 -10.685)"
                        })
                    ]
                })
            })
        ]
    });
const $ac1681db40d1da7f$var$ForwardRef = /*#__PURE__*/ (0, $i5QHx$react.forwardRef)($ac1681db40d1da7f$var$SvgComponent);
var $ac1681db40d1da7f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ac1681db40d1da7f$var$ForwardRef);
