// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_cursor_generic1_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $cFxCn$jsxs, jsx as $cFxCn$jsx} from "react/jsx-runtime";
import {forwardRef as $cFxCn$forwardRef} from "react";





const $1037d4403ade5bc8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cFxCn$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cFxCn$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cFxCn$jsxs)("linearGradient", {
                        id: "c78e8e__d",
                        x1: -3.949,
                        x2: 55.306,
                        y1: 38.702,
                        y2: 0.46,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cFxCn$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $cFxCn$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cFxCn$jsxs)("linearGradient", {
                        id: "c78e8e__e",
                        x1: 45.213,
                        x2: 20.587,
                        y1: 13.503,
                        y2: 41.593,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cFxCn$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cFxCn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cFxCn$jsxs)("linearGradient", {
                        id: "c78e8e__f",
                        x1: 13.273,
                        x2: 58.507,
                        y1: 43.818,
                        y2: 16.938,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cFxCn$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cFxCn$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cFxCn$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $cFxCn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cFxCn$jsxs)("radialGradient", {
                        id: "c78e8e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cFxCn$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $cFxCn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cFxCn$jsxs)("radialGradient", {
                        id: "c78e8e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 15.4531 -16.3929 0 16.393 15.453)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cFxCn$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cFxCn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cFxCn$jsx)("clipPath", {
                        id: "c78e8e__b",
                        children: /*#__PURE__*/ (0, $cFxCn$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#fff",
                            rx: 1.038
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cFxCn$jsx)("clipPath", {
                id: "c78e8e__a",
                children: /*#__PURE__*/ (0, $cFxCn$jsx)("path", {
                    fill: "#fff",
                    d: "m8.39 12.899 10.695 25.975c1.36 3.303 6.007 3.386 7.485.135l3.349-7.367c.408-.897 1.127-1.617 2.025-2.025l7.366-3.348c3.252-1.478 3.168-6.126-.134-7.486L13.2 8.088c-3.029-1.248-6.058 1.781-4.81 4.81"
                })
            }),
            /*#__PURE__*/ (0, $cFxCn$jsx)("g", {
                clipPath: "url(#c78e8e__a)",
                children: /*#__PURE__*/ (0, $cFxCn$jsxs)("g", {
                    clipPath: "url(#c78e8e__b)",
                    children: [
                        /*#__PURE__*/ (0, $cFxCn$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#FFECCF",
                            rx: 1.038
                        }),
                        /*#__PURE__*/ (0, $cFxCn$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M5.52 6.96H43.2v35.52H5.52z"
                        }),
                        /*#__PURE__*/ (0, $cFxCn$jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#c78e8e__c)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 10.41 -7.094)"
                        }),
                        /*#__PURE__*/ (0, $cFxCn$jsx)("path", {
                            fill: "url(#c78e8e__d)",
                            d: "M19.902.685c-2.894-1.688-6.56-1.698-9.465-.026l-29.05 16.72c-2.845 1.638-2.85 5.503-.009 7.147l28.859 16.697c2.9 1.678 6.567 1.676 9.464-.006l28.8-16.712c2.826-1.64 2.83-5.485.008-7.131z"
                        }),
                        /*#__PURE__*/ (0, $cFxCn$jsx)("path", {
                            fill: "url(#c78e8e__e)",
                            d: "M10.951 23.824C11.595 10.564 23.521.306 37.588.913s24.949 11.849 24.305 25.11-12.57 23.518-26.637 22.91c-14.067-.606-24.949-11.848-24.305-25.109"
                        }),
                        /*#__PURE__*/ (0, $cFxCn$jsx)("path", {
                            fill: "url(#c78e8e__f)",
                            d: "M-9.747 32.567c5.591-2.248 11.622-2.152 16.847-.185 10.455 3.946 16.538 1.506 20.757-8.327 2.102-4.934 6.205-9.113 11.803-11.364 11.192-4.494 24.113.382 28.874 10.902s-.45 22.68-11.64 27.18c-5.61 2.257-11.652 2.151-16.885.167-10.438-3.925-16.51-1.476-20.724 8.368-2.107 4.923-6.21 9.087-11.8 11.336C-3.7 75.14-16.62 70.264-21.379 59.75s.447-22.686 11.63-27.184"
                        }),
                        /*#__PURE__*/ (0, $cFxCn$jsx)("ellipse", {
                            cx: 16.393,
                            cy: 15.453,
                            fill: "url(#c78e8e__g)",
                            rx: 16.393,
                            ry: 15.453,
                            transform: "matrix(1 -.00132 .00148 1 24.586 -2.055)"
                        })
                    ]
                })
            })
        ]
    });
const $1037d4403ade5bc8$var$ForwardRef = /*#__PURE__*/ (0, $cFxCn$forwardRef)($1037d4403ade5bc8$var$SvgComponent);
var $1037d4403ade5bc8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1037d4403ade5bc8$var$ForwardRef);


export {$1037d4403ade5bc8$export$2e2bcd8739ae039 as default};
