// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_education_generic1_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $2gBTv$reactjsxruntime = require("react/jsx-runtime");
var $2gBTv$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1293cf3d58d532c1$export$2e2bcd8739ae039);




const $1293cf3d58d532c1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsxs)("linearGradient", {
                        id: "92faca__d",
                        x1: -12.062,
                        x2: 65.977,
                        y1: 42.895,
                        y2: -4.583,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsxs)("linearGradient", {
                        id: "92faca__e",
                        x1: 50.562,
                        x2: 17.084,
                        y1: 8.842,
                        y2: 44.84,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsxs)("linearGradient", {
                        id: "92faca__f",
                        x1: 9.878,
                        x2: 69.226,
                        y1: 49.809,
                        y2: 16.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsxs)("radialGradient", {
                        id: "92faca__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.64 14.502)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsxs)("radialGradient", {
                        id: "92faca__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.265 26.936)scale(20.8826)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("clipPath", {
                        id: "92faca__b",
                        children: /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("clipPath", {
                id: "92faca__a",
                children: /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m9.31 15.316 13.913-5.683c.485-.198 1.028-.198 1.513 0l13.912 5.683c1.659.678 1.659 3.026 0 3.703l-1.612.659v6.669c0 .874-.234 2.13-1.11 3.436-.88 1.31-2.357 2.599-4.73 3.616-1.918.821-4.123 1.136-6.042 1.207-1.924.071-3.665-.098-4.706-.312-.676-.14-1.111-.8-.972-1.476.14-.676.8-1.112 1.476-.972.818.168 2.356.327 4.109.262 1.756-.066 3.623-.353 5.15-1.007 2.007-.86 3.072-1.867 3.639-2.712.57-.848.686-1.605.686-2.042v-5.648l-9.8 4.003c-.485.198-1.028.198-1.513 0l-6.298-2.572c.228-.304.52-.562.864-.753l6.798-3.777c.603-.335.82-1.096.485-1.7s-1.096-.82-1.7-.485l-6.797 3.776c-.85.473-1.536 1.162-2.004 1.977L9.31 19.02c-1.66-.677-1.66-3.025 0-3.703m4.565 17.237V23.78c0-.934.248-1.831.696-2.613l2.354.962c-.351.468-.55 1.045-.55 1.65v8.487c.765 1.378 1.522 3.087 1.676 4.546q.025.174.025.355c0 1.45-1.248 2.624-2.787 2.624-1.54 0-2.788-1.174-2.788-2.623q0-.147.017-.288c.113-1.376.715-2.983 1.357-4.328"
                })
            }),
            /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("g", {
                clipPath: "url(#92faca__a)",
                children: /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#92faca__b)",
                    children: [
                        /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h48v48H0z"
                        }),
                        /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("circle", {
                            cx: 27.142,
                            cy: 1.862,
                            r: 20.883,
                            fill: "url(#92faca__c)",
                            transform: "rotate(-.08 27.142 1.862)"
                        }),
                        /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("path", {
                            fill: "url(#92faca__d)",
                            d: "M18.321-8.479c-3.686-2.28-8.357-2.295-12.057-.036l-37.007 22.596c-3.624 2.213-3.63 7.436-.01 9.658L6.008 46.302c3.694 2.267 8.365 2.264 12.056-.008l36.687-22.585c3.6-2.216 3.607-7.41.011-9.635z"
                        }),
                        /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("path", {
                            fill: "url(#92faca__e)",
                            d: "M6.916 22.79C7.736 4.87 22.93-8.993 40.85-8.173S72.63 7.841 71.809 25.76c-.82 17.92-16.012 31.782-33.931 30.961-17.92-.82-31.782-16.012-30.962-33.932"
                        }),
                        /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("path", {
                            fill: "url(#92faca__f)",
                            d: "M-19.448 34.605c7.124-3.039 14.806-2.908 21.462-.25 13.318 5.332 21.067 2.035 26.442-11.253 2.677-6.667 7.904-12.314 15.036-15.357 14.257-6.073 30.717.517 36.782 14.733 6.064 14.215-.574 30.65-14.828 36.73-7.147 3.05-14.844 2.906-21.51.224-13.297-5.303-21.031-1.994-26.4 11.31-2.684 6.651-7.91 12.28-15.033 15.318-14.246 6.078-30.706-.512-36.767-14.72-6.06-14.207.57-30.657 14.816-36.735"
                        }),
                        /*#__PURE__*/ (0, $2gBTv$reactjsxruntime.jsx)("circle", {
                            cx: 45.201,
                            cy: 8.672,
                            r: 20.883,
                            fill: "url(#92faca__g)",
                            transform: "rotate(-.08 45.2 8.672)"
                        })
                    ]
                })
            })
        ]
    });
const $1293cf3d58d532c1$var$ForwardRef = /*#__PURE__*/ (0, $2gBTv$react.forwardRef)($1293cf3d58d532c1$var$SvgComponent);
var $1293cf3d58d532c1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1293cf3d58d532c1$var$ForwardRef);
