// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_briefcase_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8gHVK$reactjsxruntime = require("react/jsx-runtime");
var $8gHVK$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5470cb1a707e234e$export$2e2bcd8739ae039);




const $5470cb1a707e234e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1910bd__d",
                        x1: -13.164,
                        x2: 184.35,
                        y1: 129.007,
                        y2: 1.534,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1910bd__e",
                        x1: 150.707,
                        x2: 68.621,
                        y1: 45.009,
                        y2: 138.642,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1910bd__f",
                        x1: 44.245,
                        x2: 195.026,
                        y1: 146.062,
                        y2: 56.46,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsxs)("radialGradient", {
                        id: "1910bd__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.5105 -54.6429 0 54.643 51.51)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsxs)("radialGradient", {
                        id: "1910bd__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.5105 -54.6429 0 54.643 51.51)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("clipPath", {
                        id: "1910bd__b",
                        children: /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.2,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("clipPath", {
                id: "1910bd__a",
                children: /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M115 49h9c8.84 0 16 7.16 16 16v22h-25v-2c0-2.21-1.79-4-4-4s-4 1.79-4 4v2H53v-2c0-2.21-1.79-4-4-4s-4 1.79-4 4v2H20V65c0-8.84 7.16-16 16-16h9V38c0-7.72 6.28-14 14-14h42c7.72 0 14 6.28 14 14zm-62 0h54V38c0-3.31-2.69-6-6-6H59c-3.31 0-6 2.69-6 6zm58 53c2.21 0 4-1.79 4-4v-3h25v21c0 8.84-7.16 16-16 16H36c-8.84 0-16-7.16-16-16V95h25v3c0 2.21 1.79 4 4 4s4-1.79 4-4v-3h54v3c0 2.21 1.79 4 4 4"
                })
            }),
            /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("g", {
                clipPath: "url(#1910bd__a)",
                children: /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#1910bd__b)",
                    children: [
                        /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.2,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M18.398 23.2h125.6v118.4h-125.6z"
                        }),
                        /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("ellipse", {
                            cx: 54.643,
                            cy: 51.511,
                            fill: "url(#1910bd__c)",
                            rx: 54.643,
                            ry: 51.511,
                            transform: "matrix(1 -.00132 .00148 1 34.695 -23.645)"
                        }),
                        /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("path", {
                            fill: "url(#1910bd__d)",
                            d: "M66.34 2.285c-9.647-5.626-21.868-5.661-31.55-.089l-96.836 55.736c-9.483 5.458-9.5 18.343-.028 23.822l96.195 55.657c9.668 5.592 21.89 5.585 31.548-.02l95.998-55.709c9.421-5.467 9.438-18.28.029-23.768z"
                        }),
                        /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("path", {
                            fill: "url(#1910bd__e)",
                            d: "M36.5 79.414c2.147-44.202 41.9-78.395 88.79-76.371 46.89 2.023 83.162 39.497 81.015 83.7-2.147 44.201-41.899 78.394-88.789 76.37s-83.162-39.497-81.015-83.7"
                        }),
                        /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("path", {
                            fill: "url(#1910bd__f)",
                            d: "M-32.49 108.559c18.64-7.495 38.741-7.174 56.157-.618 34.85 13.152 55.127 5.021 69.191-27.755 7.006-16.448 20.683-30.377 39.343-37.88 37.308-14.981 80.378 1.274 96.246 36.339 15.869 35.065-1.5 75.602-38.799 90.601-18.701 7.521-38.842 7.169-56.284.554-34.793-13.082-55.032-4.92-69.079 27.896-7.023 16.408-20.697 30.289-39.336 37.785-37.278 14.991-80.348-1.263-96.208-36.309s1.492-75.621 38.77-90.613"
                        }),
                        /*#__PURE__*/ (0, $8gHVK$reactjsxruntime.jsx)("ellipse", {
                            cx: 54.643,
                            cy: 51.511,
                            fill: "url(#1910bd__g)",
                            rx: 54.643,
                            ry: 51.511,
                            transform: "matrix(1 -.00132 .00148 1 81.957 -6.848)"
                        })
                    ]
                })
            })
        ]
    });
const $5470cb1a707e234e$var$ForwardRef = /*#__PURE__*/ (0, $8gHVK$react.forwardRef)($5470cb1a707e234e$var$SvgComponent);
var $5470cb1a707e234e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5470cb1a707e234e$var$ForwardRef);
