// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_brightnessContrast_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $e21CG$jsxs, jsx as $e21CG$jsx} from "react/jsx-runtime";
import {forwardRef as $e21CG$forwardRef} from "react";





const $888d57c16750b60b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e21CG$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $e21CG$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $e21CG$jsxs)("linearGradient", {
                        id: "0f9d21__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e21CG$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $e21CG$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e21CG$jsxs)("linearGradient", {
                        id: "0f9d21__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e21CG$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $e21CG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e21CG$jsxs)("linearGradient", {
                        id: "0f9d21__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e21CG$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $e21CG$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $e21CG$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $e21CG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e21CG$jsxs)("radialGradient", {
                        id: "0f9d21__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e21CG$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $e21CG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e21CG$jsxs)("radialGradient", {
                        id: "0f9d21__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e21CG$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $e21CG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e21CG$jsx)("clipPath", {
                        id: "0f9d21__b",
                        children: /*#__PURE__*/ (0, $e21CG$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $e21CG$jsx)("clipPath", {
                id: "0f9d21__a",
                children: /*#__PURE__*/ (0, $e21CG$jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M80 20c2.21 0 4 1.79 4 4v7.938c0 2.21-1.79 4-4 4s-4-1.79-4-4V24c0-2.21 1.791-4 4-4m-.228 35.72c-13.306.123-24.055 10.947-24.055 24.282s10.749 24.16 24.055 24.283zm.23-8.001c17.829 0 32.283 14.454 32.283 32.283s-14.454 32.284-32.284 32.284-32.284-14.454-32.284-32.284S62.171 47.72 80.001 47.72M84 128.062c0-2.21-1.791-4-4-4s-4 1.79-4 4V136c0 2.209 1.79 4 4 4s4-1.791 4-4zM140 80c0 2.21-1.791 4-4 4h-7.939c-2.209 0-4-1.79-4-4s1.791-4 4-4h7.939c2.209 0 4 1.79 4 4M31.938 84c2.21 0 4-1.79 4-4s-1.79-4-4-4H24c-2.21 0-4 1.79-4 4s1.79 4 4 4zm90.488-46.427c1.562 1.562 1.562 4.095 0 5.657l-5.613 5.613c-1.562 1.562-4.095 1.562-5.657 0s-1.562-4.095 0-5.657l5.613-5.613c1.562-1.562 4.095-1.562 5.657 0m-73.582 79.24c1.562-1.562 1.562-4.094 0-5.657s-4.095-1.562-5.657 0l-5.613 5.614c-1.562 1.562-1.562 4.094 0 5.656s4.095 1.562 5.657 0zm-11.27-79.242c-1.563 1.562-1.563 4.095 0 5.657l5.612 5.613c1.562 1.562 4.095 1.562 5.657 0s1.562-4.095 0-5.657l-5.613-5.613c-1.562-1.562-4.095-1.562-5.657 0m73.583 79.24c-1.562-1.562-1.562-4.094 0-5.657s4.095-1.562 5.657 0l5.613 5.614c1.562 1.562 1.562 4.094 0 5.656-1.562 1.563-4.095 1.563-5.657 0z"
                })
            }),
            /*#__PURE__*/ (0, $e21CG$jsx)("g", {
                clipPath: "url(#0f9d21__a)",
                children: /*#__PURE__*/ (0, $e21CG$jsxs)("g", {
                    clipPath: "url(#0f9d21__b)",
                    children: [
                        /*#__PURE__*/ (0, $e21CG$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $e21CG$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $e21CG$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#0f9d21__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $e21CG$jsx)("path", {
                            fill: "url(#0f9d21__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $e21CG$jsx)("path", {
                            fill: "url(#0f9d21__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $e21CG$jsx)("path", {
                            fill: "url(#0f9d21__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $e21CG$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#0f9d21__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $888d57c16750b60b$var$ForwardRef = /*#__PURE__*/ (0, $e21CG$forwardRef)($888d57c16750b60b$var$SvgComponent);
var $888d57c16750b60b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($888d57c16750b60b$var$ForwardRef);


export {$888d57c16750b60b$export$2e2bcd8739ae039 as default};
