// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_3Dasset_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $fGeIl$jsxs, jsx as $fGeIl$jsx} from "react/jsx-runtime";
import {forwardRef as $fGeIl$forwardRef} from "react";





const $0e9940588064d2e1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fGeIl$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fGeIl$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fGeIl$jsxs)("linearGradient", {
                        id: "14b0f8__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fGeIl$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $fGeIl$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fGeIl$jsxs)("linearGradient", {
                        id: "14b0f8__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fGeIl$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fGeIl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fGeIl$jsxs)("linearGradient", {
                        id: "14b0f8__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fGeIl$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fGeIl$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fGeIl$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fGeIl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fGeIl$jsxs)("radialGradient", {
                        id: "14b0f8__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fGeIl$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $fGeIl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fGeIl$jsxs)("radialGradient", {
                        id: "14b0f8__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fGeIl$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fGeIl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fGeIl$jsx)("clipPath", {
                        id: "14b0f8__b",
                        children: /*#__PURE__*/ (0, $fGeIl$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fGeIl$jsx)("clipPath", {
                id: "14b0f8__a",
                children: /*#__PURE__*/ (0, $fGeIl$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M33.116 43.031c4.642-2.543 9.26-5.094 12.605-6.985 3.015-1.704 7.03-1.824 10.058-.139l20.269 11.285c2.42 1.346 5.363 1.346 7.782 0l21.452-11.943c2.419-1.348 5.363-1.348 7.783 0l13.944 7.762c2.753 1.533 4.122 4.286 4.108 7.032L79.939 78.615l-51-28.472c-.019-2.784 1.377-5.578 4.177-7.112m-4.237 16.24v27.261c0 5.42 2.744 10.472 7.29 13.422l39.77 25.807V85.544zm55.06 66.444 39.869-25.764c4.561-2.948 7.315-8.008 7.315-13.438V59.202L83.939 85.544z"
                })
            }),
            /*#__PURE__*/ (0, $fGeIl$jsx)("g", {
                clipPath: "url(#14b0f8__a)",
                children: /*#__PURE__*/ (0, $fGeIl$jsxs)("g", {
                    clipPath: "url(#14b0f8__b)",
                    children: [
                        /*#__PURE__*/ (0, $fGeIl$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $fGeIl$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $fGeIl$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#14b0f8__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $fGeIl$jsx)("path", {
                            fill: "url(#14b0f8__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $fGeIl$jsx)("path", {
                            fill: "url(#14b0f8__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $fGeIl$jsx)("path", {
                            fill: "url(#14b0f8__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $fGeIl$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#14b0f8__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $0e9940588064d2e1$var$ForwardRef = /*#__PURE__*/ (0, $fGeIl$forwardRef)($0e9940588064d2e1$var$SvgComponent);
var $0e9940588064d2e1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0e9940588064d2e1$var$ForwardRef);


export {$0e9940588064d2e1$export$2e2bcd8739ae039 as default};
