// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_trophy_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $do3ec$jsxs, jsx as $do3ec$jsx} from "react/jsx-runtime";
import {forwardRef as $do3ec$forwardRef} from "react";





const $00ef68fa1deffef3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $do3ec$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $do3ec$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $do3ec$jsxs)("linearGradient", {
                        id: "aa9809__d",
                        x1: -18.921,
                        x2: 187.087,
                        y1: 128.357,
                        y2: -4.598,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $do3ec$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $do3ec$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $do3ec$jsxs)("linearGradient", {
                        id: "aa9809__e",
                        x1: 151.995,
                        x2: 66.38,
                        y1: 40.747,
                        y2: 138.405,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $do3ec$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $do3ec$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $do3ec$jsxs)("linearGradient", {
                        id: "aa9809__f",
                        x1: 40.959,
                        x2: 198.223,
                        y1: 146.143,
                        y2: 52.689,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $do3ec$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $do3ec$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $do3ec$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $do3ec$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $do3ec$jsxs)("radialGradient", {
                        id: "aa9809__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $do3ec$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $do3ec$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $do3ec$jsxs)("radialGradient", {
                        id: "aa9809__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7253 -56.9923 0 56.992 53.725)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $do3ec$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $do3ec$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $do3ec$jsx)("clipPath", {
                        id: "aa9809__b",
                        children: /*#__PURE__*/ (0, $do3ec$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 18,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $do3ec$jsx)("clipPath", {
                id: "aa9809__a",
                children: /*#__PURE__*/ (0, $do3ec$jsx)("path", {
                    fill: "#fff",
                    d: "M126.04 40H108v-6c0-3.31-2.69-6-6-6H58c-3.31 0-6 2.69-6 6v6H34c-5.51 0-10 4.49-10 10v2c0 15.44 12.56 28 28 28h5.03c4.34 6.22 11.15 10.6 19 11.71 0 .1-.03.19-.03.29v24.42c-7.17 1.35-13.07 6.24-15.84 12.79-.56 1.32.42 2.78 1.85 2.78h35.98c1.43 0 2.41-1.46 1.85-2.78-2.77-6.55-8.67-11.44-15.84-12.79V92c0-.1-.02-.19-.03-.29 7.86-1.12 14.66-5.49 19-11.71h5.06c15.44 0 28-12.56 28-28v-2c0-5.51-4.49-10-10-10zM52 72c-11.03 0-20-8.97-20-20v-2c0-1.1.9-2 2-2h18v16c0 2.78.41 5.46 1.17 8zm40.79-14.83-3.73 3.62c-.51.5-.75 1.22-.62 1.92l.89 5.12c.34 1.98-1.74 3.5-3.52 2.56l-4.6-2.43c-.63-.33-1.39-.33-2.02 0l-4.6 2.43c-1.78.94-3.86-.57-3.52-2.56l.89-5.12c.12-.7-.11-1.42-.62-1.92l-3.73-3.62c-1.44-1.4-.65-3.85 1.34-4.14l5.15-.74c.71-.1 1.32-.55 1.64-1.19l2.29-4.67c.89-1.81 3.46-1.81 4.35 0l2.29 4.67c.32.64.93 1.09 1.64 1.19l5.15.74c1.99.29 2.79 2.74 1.34 4.14zM128.04 52c0 11.03-8.97 20-20 20h-1.2c.75-2.54 1.17-5.22 1.17-8V48h18.04c1.1 0 2 .9 2 2v2z"
                })
            }),
            /*#__PURE__*/ (0, $do3ec$jsx)("g", {
                clipPath: "url(#aa9809__a)",
                children: /*#__PURE__*/ (0, $do3ec$jsxs)("g", {
                    clipPath: "url(#aa9809__b)",
                    children: [
                        /*#__PURE__*/ (0, $do3ec$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 18,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $do3ec$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M14 18h131v123.491H14z"
                        }),
                        /*#__PURE__*/ (0, $do3ec$jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#aa9809__c)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 31 -30.86)"
                        }),
                        /*#__PURE__*/ (0, $do3ec$jsx)("path", {
                            fill: "url(#aa9809__d)",
                            d: "M64.001-3.814c-10.06-5.868-22.807-5.904-32.905-.092l-101 58.131c-9.891 5.694-9.907 19.133-.03 24.847L30.4 137.122c10.082 5.833 22.83 5.826 32.904-.021l100.125-58.104c9.826-5.703 9.844-19.066.03-24.79z"
                        }),
                        /*#__PURE__*/ (0, $do3ec$jsx)("path", {
                            fill: "url(#aa9809__e)",
                            d: "M32.878 76.631c2.24-46.103 43.701-81.765 92.607-79.655 48.907 2.111 86.738 41.196 84.499 87.299-2.239 46.102-43.701 81.765-92.607 79.654-48.906-2.11-86.738-41.195-84.499-87.298"
                        }),
                        /*#__PURE__*/ (0, $do3ec$jsx)("path", {
                            fill: "url(#aa9809__f)",
                            d: "M-39.075 107.029c19.44-7.818 40.406-7.483 58.57-.645 36.349 13.717 57.498 5.236 72.167-28.95 7.307-17.153 21.572-31.681 41.035-39.508 38.911-15.625 83.833 1.329 100.384 37.902s-1.565 78.852-40.467 94.497c-19.506 7.844-40.512 7.477-58.705.577-36.288-13.644-57.398-5.131-72.048 29.096-7.325 17.113-21.588 31.591-41.028 39.409-38.88 15.636-83.803-1.317-100.344-37.869-16.542-36.553 1.555-78.874 40.436-94.509"
                        }),
                        /*#__PURE__*/ (0, $do3ec$jsx)("ellipse", {
                            cx: 56.992,
                            cy: 53.725,
                            fill: "url(#aa9809__g)",
                            rx: 56.992,
                            ry: 53.725,
                            transform: "matrix(1 -.00132 .00148 1 80.293 -13.34)"
                        })
                    ]
                })
            })
        ]
    });
const $00ef68fa1deffef3$var$ForwardRef = /*#__PURE__*/ (0, $do3ec$forwardRef)($00ef68fa1deffef3$var$SvgComponent);
var $00ef68fa1deffef3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($00ef68fa1deffef3$var$ForwardRef);


export {$00ef68fa1deffef3$export$2e2bcd8739ae039 as default};
