// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_buildings_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bYBrO$jsxs, jsx as $bYBrO$jsx} from "react/jsx-runtime";
import {forwardRef as $bYBrO$forwardRef} from "react";





const $a1ef02eaf635d4f0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bYBrO$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bYBrO$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bYBrO$jsxs)("linearGradient", {
                        id: "6fc07f__d",
                        x1: -28.688,
                        x2: 194.754,
                        y1: 137.748,
                        y2: -6.308,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bYBrO$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $bYBrO$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bYBrO$jsxs)("linearGradient", {
                        id: "6fc07f__e",
                        x1: 156.584,
                        x2: 63.67,
                        y1: 42.682,
                        y2: 148.555,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bYBrO$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bYBrO$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bYBrO$jsxs)("linearGradient", {
                        id: "6fc07f__f",
                        x1: 36.222,
                        x2: 206.784,
                        y1: 157.049,
                        y2: 55.798,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bYBrO$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bYBrO$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bYBrO$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $bYBrO$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bYBrO$jsxs)("radialGradient", {
                        id: "6fc07f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 58.2973 -61.7778 0 61.778 58.297)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bYBrO$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $bYBrO$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bYBrO$jsxs)("radialGradient", {
                        id: "6fc07f__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 58.2973 -61.7778 0 61.778 58.297)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bYBrO$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bYBrO$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bYBrO$jsx)("clipPath", {
                        id: "6fc07f__b",
                        children: /*#__PURE__*/ (0, $bYBrO$jsx)("rect", {
                            width: 142,
                            height: 134,
                            x: 7,
                            y: 18,
                            fill: "#fff",
                            rx: 3.461
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bYBrO$jsx)("clipPath", {
                id: "6fc07f__a",
                children: /*#__PURE__*/ (0, $bYBrO$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M35.75 27.93h48.07c8.84 0 16 7.16 16 16v88.13H73.83v-20.58c0-4.42-3.58-8-8-8H53.74c-4.42 0-8 3.58-8 8v20.58H19.75V43.93c0-8.84 7.16-16 16-16m12.79 56.56c2.21 0 4-1.79 4-4v-5.76c0-2.21-1.79-4-4-4h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.21 1.79 4 4 4zm0-28.06c2.21 0 4-1.79 4-4v-5.76c0-2.21-1.79-4-4-4h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.21 1.79 4 4 4zM76.8 84.49c2.21 0 4-1.79 4-4v-5.76c0-2.21-1.79-4-4-4h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.21 1.79 4 4 4zm0-28.06c2.21 0 4-1.79 4-4v-5.76c0-2.21-1.79-4-4-4h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.21 1.79 4 4 4zm47.44-6.87c8.84 0 16 7.16 16 16v66.5h-32.41v-82.5zm-1.33 62.16c2.21 0 4-1.79 4-4v-5.76c0-2.21-1.79-4-4-4h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.21 1.79 4 4 4zm0-28.06c2.21 0 4-1.79 4-4V73.9c0-2.21-1.79-4-4-4h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.21 1.79 4 4 4z"
                })
            }),
            /*#__PURE__*/ (0, $bYBrO$jsx)("g", {
                clipPath: "url(#6fc07f__a)",
                children: /*#__PURE__*/ (0, $bYBrO$jsxs)("g", {
                    clipPath: "url(#6fc07f__b)",
                    children: [
                        /*#__PURE__*/ (0, $bYBrO$jsx)("rect", {
                            width: 142,
                            height: 134,
                            x: 7,
                            y: 18,
                            fill: "#FFECCF",
                            rx: 3.461
                        }),
                        /*#__PURE__*/ (0, $bYBrO$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M7 18h142v134H7z"
                        }),
                        /*#__PURE__*/ (0, $bYBrO$jsx)("ellipse", {
                            cx: 61.778,
                            cy: 58.297,
                            fill: "url(#6fc07f__c)",
                            rx: 61.778,
                            ry: 58.297,
                            transform: "matrix(1 -.00132 .00148 1 25.426 -35.019)"
                        }),
                        /*#__PURE__*/ (0, $bYBrO$jsx)("path", {
                            fill: "url(#6fc07f__d)",
                            d: "M61.197-5.67c-10.906-6.368-24.723-6.407-35.669-.1l-109.48 63.078c-10.722 6.178-10.74 20.76-.032 26.961l108.756 62.99c10.93 6.329 24.747 6.321 35.667-.023l108.533-63.048c10.651-6.188 10.67-20.689.032-26.9z"
                        }),
                        /*#__PURE__*/ (0, $bYBrO$jsx)("path", {
                            fill: "url(#6fc07f__e)",
                            d: "M27.465 81.62c2.427-50.026 47.37-88.724 100.383-86.434s94.02 44.702 91.593 94.728-47.369 88.723-100.382 86.433-94.02-44.701-91.594-94.727"
                        }),
                        /*#__PURE__*/ (0, $bYBrO$jsx)("path", {
                            fill: "url(#6fc07f__f)",
                            d: "M-50.532 114.605c21.073-8.483 43.8-8.119 63.49-.699 39.399 14.885 62.324 5.682 78.225-31.413 7.92-18.614 23.384-34.378 44.48-42.871 42.179-16.954 90.873 1.442 108.813 41.127 17.941 39.685-1.696 85.563-43.865 102.539-21.143 8.512-43.914 8.114-63.634.627-39.335-14.806-62.217-5.568-78.098 31.571-7.94 18.57-23.4 34.28-44.472 42.764-42.146 16.966-90.84-1.43-108.77-41.093-17.931-39.663 1.686-85.585 43.831-102.552"
                        }),
                        /*#__PURE__*/ (0, $bYBrO$jsx)("ellipse", {
                            cx: 61.778,
                            cy: 58.297,
                            fill: "url(#6fc07f__g)",
                            rx: 61.778,
                            ry: 58.297,
                            transform: "matrix(1 -.00132 .00148 1 78.86 -16.007)"
                        })
                    ]
                })
            })
        ]
    });
const $a1ef02eaf635d4f0$var$ForwardRef = /*#__PURE__*/ (0, $bYBrO$forwardRef)($a1ef02eaf635d4f0$var$SvgComponent);
var $a1ef02eaf635d4f0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a1ef02eaf635d4f0$var$ForwardRef);


export {$a1ef02eaf635d4f0$export$2e2bcd8739ae039 as default};
