// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fireworks_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3UJnA$reactjsxruntime = require("react/jsx-runtime");
var $3UJnA$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f644594ea9d6a50b$export$2e2bcd8739ae039);




const $f644594ea9d6a50b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b9507e__d",
                        x1: -11.953,
                        x2: 111.651,
                        y1: 78.607,
                        y2: -1.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b9507e__e",
                        x1: 90.597,
                        x2: 39.227,
                        y1: 26.041,
                        y2: 84.636,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b9507e__f",
                        x1: 23.977,
                        x2: 118.335,
                        y1: 89.279,
                        y2: 33.206,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b9507e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b9507e__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("clipPath", {
                        id: "b9507e__b",
                        children: /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("clipPath", {
                id: "b9507e__a",
                children: /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M76.868 16.946c.02-1.325-1.036-2.417-2.362-2.438-1.325-.021-2.417 1.036-2.438 2.361l-.033 2.1-2.103-.033c-1.325-.02-2.416 1.038-2.436 2.363s1.037 2.416 2.363 2.436l2.099.033-.034 2.101c-.021 1.326 1.036 2.417 2.361 2.439 1.326.02 2.417-1.037 2.439-2.362l.033-2.105 2.102.032c1.325.02 2.416-1.037 2.436-2.362s-1.038-2.417-2.363-2.437l-2.098-.032zm-36.674 1.528c0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4v6.084c0 1.325 1.075 2.4 2.4 2.4 1.326 0 2.4-1.075 2.4-2.4zm-15.016 1.694c1.073-.779 2.573-.54 3.352.532l3.576 4.926c.779 1.073.54 2.573-.532 3.352s-2.574.54-3.352-.532l-3.576-4.926c-.779-1.073-.54-2.574.532-3.352m-4.78 9.2c-1.26-.41-2.614.28-3.024 1.54s.281 2.615 1.542 3.025L24.7 35.81c1.26.409 2.614-.281 3.024-1.542.409-1.26-.281-2.614-1.542-3.024zm7.326 11.447c.409 1.261-.282 2.615-1.543 3.024l-5.79 1.878c-1.26.409-2.614-.282-3.023-1.543s.282-2.614 1.542-3.023l5.79-1.878c1.261-.409 2.615.282 3.024 1.542m4.375 8.65c.78-1.072.542-2.573-.53-3.353-1.072-.779-2.573-.541-3.352.53l-3.576 4.92c-.78 1.073-.542 2.574.53 3.353s2.573.542 3.352-.53zm8.09 1.067c0-1.326-1.075-2.4-2.4-2.4-1.326 0-2.4 1.074-2.4 2.4v6.084c0 1.325 1.074 2.4 2.4 2.4s2.4-1.075 2.4-2.4zM50.42 20.175c1.073.779 1.31 2.28.53 3.352l-3.575 4.92c-.78 1.072-2.28 1.31-3.353.53-1.072-.779-1.31-2.28-.53-3.352l3.576-4.92c.78-1.072 2.28-1.31 3.352-.53m6.253 13.764c1.26-.41 1.95-1.764 1.541-3.024s-1.763-1.951-3.023-1.542l-5.784 1.878c-1.261.41-1.951 1.763-1.542 3.024.41 1.26 1.763 1.95 3.024 1.542zm-8.808 6.882c.41-1.261 1.763-1.951 3.024-1.542l5.784 1.878c1.26.41 1.95 1.763 1.541 3.024-.409 1.26-1.763 1.95-3.023 1.542l-5.784-1.878c-1.261-.41-1.951-1.764-1.542-3.024m-.495 5.823c-.779-1.073-2.28-1.311-3.352-.532s-1.311 2.28-.533 3.352l3.576 4.926c.779 1.072 2.28 1.31 3.352.532 1.073-.779 1.311-2.28.533-3.352zm18.051 22.332c1.326.005 2.396 1.084 2.39 2.41l-.012 2.97c-.005 1.325-1.084 2.395-2.41 2.39-1.325-.005-2.395-1.084-2.39-2.41l.012-2.97c.006-1.325 1.085-2.395 2.41-2.39m-5.364.71c1.15-.66 1.55-2.125.892-3.276-.659-1.15-2.125-1.55-3.275-.891l-2.58 1.476c-1.151.658-1.55 2.124-.892 3.275s2.125 1.55 3.275.891zm-5.84-12.345c.666-1.146 2.136-1.534 3.281-.868l2.568 1.494c1.146.667 1.535 2.136.868 3.282s-2.136 1.534-3.281.867l-2.568-1.494c-1.146-.666-1.535-2.135-.868-3.281m13.594-4.045c.006-1.326-1.064-2.405-2.39-2.41-1.325-.005-2.404 1.065-2.41 2.39L63 56.246c-.005 1.326 1.065 2.404 2.39 2.41 1.326.005 2.405-1.065 2.41-2.39zm8.805 4.066c.658 1.15.259 2.617-.892 3.275l-2.58 1.476c-1.15.658-2.617.26-3.275-.891-.658-1.15-.259-2.617.892-3.275l2.58-1.476c1.15-.658 2.617-.26 3.275.891m-3.45 8.153c-1.146-.666-2.616-.278-3.282.868-.667 1.146-.278 2.615.867 3.281l2.569 1.494c1.145.667 2.614.278 3.28-.867.667-1.146.279-2.615-.867-3.282zM14.09 74.693c.02-1.325 1.111-2.383 2.437-2.363l2.097.032.033-2.103c.02-1.325 1.11-2.383 2.436-2.363s2.383 1.111 2.363 2.437l-.032 2.103 2.103.032c1.325.02 2.383 1.111 2.363 2.437s-1.112 2.383-2.437 2.363l-2.103-.033-.032 2.098c-.02 1.325-1.111 2.383-2.436 2.363-1.326-.02-2.384-1.112-2.363-2.437l.032-2.097-2.098-.032c-1.325-.02-2.383-1.112-2.363-2.437"
                })
            }),
            /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("g", {
                clipPath: "url(#b9507e__a)",
                children: /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#b9507e__b)",
                    children: [
                        /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M7.8 12.393h78.6v74.094H7.8z"
                        }),
                        /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#b9507e__c)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 18 -16.924)"
                        }),
                        /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("path", {
                            fill: "url(#b9507e__d)",
                            d: "M37.8-.696c-6.036-3.52-13.684-3.542-19.743-.055l-60.6 34.879c-5.935 3.416-5.945 11.48-.018 14.908l60.2 34.83c6.05 3.5 13.697 3.495 19.742-.013l60.075-34.862c5.895-3.422 5.906-11.44.018-14.874z"
                        }),
                        /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("path", {
                            fill: "url(#b9507e__e)",
                            d: "M19.126 47.571C20.47 19.91 45.346-1.488 74.69-.222c29.344 1.267 52.043 24.718 50.7 52.38-1.344 27.661-26.22 49.059-55.565 47.792-29.343-1.266-52.042-24.717-50.699-52.379"
                        }),
                        /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("path", {
                            fill: "url(#b9507e__f)",
                            d: "M-24.044 65.81C-12.379 61.12.2 61.32 11.1 65.423c21.809 8.23 34.498 3.142 43.3-17.37 4.384-10.292 12.943-19.008 24.62-23.705 23.348-9.374 50.301.798 60.231 22.742 9.931 21.943-.939 47.311-24.28 56.698-11.703 4.707-24.307 4.486-35.223.346-21.773-8.186-34.439-3.078-43.229 17.458-4.395 10.268-12.952 18.955-24.617 23.645-23.328 9.382-50.281-.79-60.206-22.721s.933-47.324 24.261-56.706"
                        }),
                        /*#__PURE__*/ (0, $3UJnA$reactjsxruntime.jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#b9507e__g)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 47.578 -6.411)"
                        })
                    ]
                })
            })
        ]
    });
const $f644594ea9d6a50b$var$ForwardRef = /*#__PURE__*/ (0, $3UJnA$react.forwardRef)($f644594ea9d6a50b$var$SvgComponent);
var $f644594ea9d6a50b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f644594ea9d6a50b$var$ForwardRef);
