// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_apps_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $8OjNs$jsxs, jsx as $8OjNs$jsx} from "react/jsx-runtime";
import {forwardRef as $8OjNs$forwardRef} from "react";





const $28daa713e58e9a25$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8OjNs$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8OjNs$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8OjNs$jsxs)("linearGradient", {
                        id: "a9fb23__d",
                        x1: -11.355,
                        x2: 112.249,
                        y1: 79.415,
                        y2: -0.358,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8OjNs$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $8OjNs$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8OjNs$jsxs)("linearGradient", {
                        id: "a9fb23__e",
                        x1: 91.194,
                        x2: 39.825,
                        y1: 26.849,
                        y2: 85.444,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8OjNs$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8OjNs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8OjNs$jsxs)("linearGradient", {
                        id: "a9fb23__f",
                        x1: 24.574,
                        x2: 118.933,
                        y1: 90.087,
                        y2: 34.014,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8OjNs$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8OjNs$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8OjNs$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $8OjNs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8OjNs$jsxs)("radialGradient", {
                        id: "a9fb23__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8OjNs$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $8OjNs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8OjNs$jsxs)("radialGradient", {
                        id: "a9fb23__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 32.2352 -34.1954 0 34.195 32.235)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8OjNs$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8OjNs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8OjNs$jsx)("clipPath", {
                        id: "a9fb23__b",
                        children: /*#__PURE__*/ (0, $8OjNs$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 13.2,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8OjNs$jsx)("clipPath", {
                id: "a9fb23__a",
                children: /*#__PURE__*/ (0, $8OjNs$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M54.185 13.008h19.182c5.302 0 9.6 4.298 9.6 9.6V41.79c0 5.302-4.298 9.6-9.6 9.6H54.185c-5.302 0-9.6-4.298-9.6-9.6V22.608c0-5.302 4.298-9.6 9.6-9.6M21.431 24.684h9.912c4.64 0 8.4 3.76 8.4 8.4v9.912c0 4.639-3.76 8.4-8.4 8.4h-9.912c-4.639 0-8.4-3.761-8.4-8.4v-9.912c0-4.64 3.761-8.4 8.4-8.4m9.912 31.596h-9.912c-4.639 0-8.4 3.76-8.4 8.4v9.912c0 4.639 3.761 8.4 8.4 8.4h9.912c4.64 0 8.4-3.761 8.4-8.4V64.68c0-4.64-3.76-8.4-8.4-8.4m21.642 0h9.912c4.64 0 8.4 3.76 8.4 8.4v9.912c0 4.639-3.76 8.4-8.4 8.4h-9.912c-4.639 0-8.4-3.761-8.4-8.4V64.68c0-4.64 3.761-8.4 8.4-8.4"
                })
            }),
            /*#__PURE__*/ (0, $8OjNs$jsx)("g", {
                clipPath: "url(#a9fb23__a)",
                children: /*#__PURE__*/ (0, $8OjNs$jsxs)("g", {
                    clipPath: "url(#a9fb23__b)",
                    children: [
                        /*#__PURE__*/ (0, $8OjNs$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 13.2,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $8OjNs$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M8.398 13.2h78.6v74.094h-78.6z"
                        }),
                        /*#__PURE__*/ (0, $8OjNs$jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#a9fb23__c)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 18.598 -16.116)"
                        }),
                        /*#__PURE__*/ (0, $8OjNs$jsx)("path", {
                            fill: "url(#a9fb23__d)",
                            d: "M38.398.112C32.36-3.409 24.713-3.43 18.654.056l-60.6 34.88c-5.934 3.416-5.944 11.479-.017 14.908l60.199 34.83c6.05 3.499 13.698 3.495 19.742-.013l60.076-34.863c5.895-3.421 5.906-11.44.018-14.874z"
                        }),
                        /*#__PURE__*/ (0, $8OjNs$jsx)("path", {
                            fill: "url(#a9fb23__e)",
                            d: "M19.724 48.379C21.067 20.717 45.944-.681 75.288.586c29.344 1.266 52.043 24.717 50.699 52.379-1.343 27.662-26.22 49.059-55.564 47.793-29.344-1.267-52.043-24.717-50.7-52.38"
                        }),
                        /*#__PURE__*/ (0, $8OjNs$jsx)("path", {
                            fill: "url(#a9fb23__f)",
                            d: "M-23.446 66.618c11.664-4.691 24.244-4.49 35.143-.387 21.808 8.23 34.498 3.142 43.3-17.37C59.38 38.57 67.94 29.852 79.617 25.156c23.347-9.374 50.3.797 60.231 22.741s-.939 47.312-24.28 56.699c-11.704 4.706-24.308 4.486-35.223.346-21.773-8.187-34.439-3.079-43.23 17.457-4.394 10.268-12.952 18.955-24.616 23.646-23.328 9.382-50.282-.79-60.207-22.722-9.925-21.931.934-47.324 24.262-56.705"
                        }),
                        /*#__PURE__*/ (0, $8OjNs$jsx)("ellipse", {
                            cx: 34.195,
                            cy: 32.235,
                            fill: "url(#a9fb23__g)",
                            rx: 34.195,
                            ry: 32.235,
                            transform: "matrix(1 -.00132 .00148 1 48.176 -5.604)"
                        })
                    ]
                })
            })
        ]
    });
const $28daa713e58e9a25$var$ForwardRef = /*#__PURE__*/ (0, $8OjNs$forwardRef)($28daa713e58e9a25$var$SvgComponent);
var $28daa713e58e9a25$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($28daa713e58e9a25$var$ForwardRef);


export {$28daa713e58e9a25$export$2e2bcd8739ae039 as default};
