// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_illustration_generic1_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $kARon$jsxs, jsx as $kARon$jsx} from "react/jsx-runtime";
import {forwardRef as $kARon$forwardRef} from "react";





const $72203d28f680a3b2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kARon$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kARon$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kARon$jsxs)("linearGradient", {
                        id: "180dd4__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kARon$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $kARon$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kARon$jsxs)("linearGradient", {
                        id: "180dd4__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kARon$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kARon$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kARon$jsxs)("linearGradient", {
                        id: "180dd4__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kARon$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kARon$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kARon$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $kARon$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kARon$jsxs)("radialGradient", {
                        id: "180dd4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kARon$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $kARon$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kARon$jsxs)("radialGradient", {
                        id: "180dd4__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kARon$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kARon$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kARon$jsx)("clipPath", {
                        id: "180dd4__b",
                        children: /*#__PURE__*/ (0, $kARon$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kARon$jsx)("clipPath", {
                id: "180dd4__a",
                children: /*#__PURE__*/ (0, $kARon$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M62.722 40.774 85.93 63.983l-6.52 12.566c-1.757 3.384-4.664 6.03-8.199 7.461L36.426 98.091c-1.087.44-2.174.543-3.185.384l22.76-22.834c1.603.054 3.223-.53 4.447-1.754 2.343-2.343 2.343-6.142 0-8.485s-6.142-2.343-8.486 0c-1.289 1.29-1.869 3.019-1.74 4.704L28.169 92.233c.036-.64.179-1.297.445-1.954l14.081-34.785c1.43-3.535 4.077-6.442 7.462-8.198zm34.032 10.59-6.065 6.064-21.436-21.436 6.065-6.065c2.343-2.343 6.142-2.343 8.485 0l12.95 12.951c2.344 2.343 2.344 6.142 0 8.485m-4.381 40.438 23.686-28.915c3.113-3.8 8.717-4.357 12.517-1.244s4.357 8.717 1.244 12.517l-23.774 29.022c-1.711-3.335-4.109-5.858-6.303-7.655-2.06-1.687-4.6-2.944-7.37-3.725m2.3 9.913c1.788 1.464 3.52 3.396 4.592 5.829 1.396 3.169 1.668 7.189-.53 12.13q-.22.494-.473 1.001c-4.643 9.282-18.071 11.232-29.403 11.154q-1.106-.009-2.181-.04c-4.29-.125-8.155-.505-10.95-.827l.5-.393c2.922-2.318 4.716-4.146 6.18-6.267q.24-.348.47-.708c1.327-2.077 2.436-4.493 4.007-7.914.894-1.946 1.936-4.216 3.253-6.934q.407-.838.86-1.596c4.522-7.577 12.301-9.147 18.127-7.913 2.235.473 4.183 1.36 5.549 2.478"
                })
            }),
            /*#__PURE__*/ (0, $kARon$jsx)("g", {
                clipPath: "url(#180dd4__a)",
                children: /*#__PURE__*/ (0, $kARon$jsxs)("g", {
                    clipPath: "url(#180dd4__b)",
                    children: [
                        /*#__PURE__*/ (0, $kARon$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $kARon$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $kARon$jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#180dd4__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $kARon$jsx)("path", {
                            fill: "url(#180dd4__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $kARon$jsx)("path", {
                            fill: "url(#180dd4__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $kARon$jsx)("path", {
                            fill: "url(#180dd4__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $kARon$jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#180dd4__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $72203d28f680a3b2$var$ForwardRef = /*#__PURE__*/ (0, $kARon$forwardRef)($72203d28f680a3b2$var$SvgComponent);
var $72203d28f680a3b2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($72203d28f680a3b2$var$ForwardRef);


export {$72203d28f680a3b2$export$2e2bcd8739ae039 as default};
