// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_fileShared_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $lvdRF$reactjsxruntime = require("react/jsx-runtime");
var $lvdRF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9b4f7e8dd47bce34$export$2e2bcd8739ae039);




const $9b4f7e8dd47bce34$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ce80f8__d",
                        x1: -24.124,
                        x2: 131.953,
                        y1: 85.79,
                        y2: -9.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ce80f8__e",
                        x1: 101.128,
                        x2: 34.171,
                        y1: 17.683,
                        y2: 89.68,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ce80f8__f",
                        x1: 19.755,
                        x2: 138.452,
                        y1: 99.618,
                        y2: 33.126,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ce80f8__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 25.279 29.002)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ce80f8__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 36.531 53.875)scale(41.7653)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("clipPath", {
                        id: "ce80f8__b",
                        children: /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("clipPath", {
                id: "ce80f8__a",
                children: /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M52.909 18.631 72.69 38.414c1.35 1.35 2.109 3.181 2.109 5.09v28.219c0 3.976-3.224 7.2-7.2 7.2H29.2c-3.976 0-7.2-3.224-7.2-7.2v-48c0-3.977 3.224-7.2 7.2-7.2h18.618c1.91 0 3.74.758 5.09 2.108M41.104 46.795c0-.704.57-1.273 1.273-1.273 1.104 0 2-.896 2-2s-.896-2-2-2c-2.913 0-5.273 2.36-5.273 5.273 0 1.486.617 2.832 1.605 3.788.794.768 2.06.748 2.828-.046.769-.793.748-2.06-.045-2.828-.242-.234-.388-.555-.388-.915m8.925-2.92c-2.729 0-4.94 2.212-4.94 4.94 0 2.73 2.211 4.942 4.94 4.942s4.942-2.212 4.942-4.941c0-2.73-2.213-4.942-4.942-4.942m-9.066 13.86c.996-.477 1.418-1.67.942-2.667s-1.67-1.419-2.666-.943c-2.447 1.169-4.239 3.518-4.71 6.297-.185 1.09.548 2.122 1.637 2.307s2.121-.548 2.306-1.637c.256-1.508 1.228-2.754 2.49-3.357m9.066-2.096c-3.947 0-7.507 2.53-8.388 6.19-.514 2.133 1.29 3.693 3.04 3.693h10.698c1.748 0 3.553-1.56 3.04-3.693-.882-3.66-4.442-6.19-8.39-6.19"
                })
            }),
            /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ce80f8__a)",
                children: /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ce80f8__b)",
                    children: [
                        /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h96v96H0z"
                        }),
                        /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("circle", {
                            cx: 54.281,
                            cy: 3.723,
                            r: 41.765,
                            fill: "url(#ce80f8__c)",
                            transform: "rotate(-.08 54.28 3.723)"
                        }),
                        /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("path", {
                            fill: "url(#ce80f8__d)",
                            d: "M36.643-16.957c-7.373-4.562-16.714-4.59-24.114-.072l-74.015 45.19c-7.249 4.426-7.26 14.873-.022 19.316l73.526 45.127c7.389 4.534 16.73 4.529 24.113-.016l73.374-45.17c7.2-4.432 7.213-14.821.021-19.27z"
                        }),
                        /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("path", {
                            fill: "url(#ce80f8__e)",
                            d: "M13.837 45.579C15.477 9.739 45.86-17.984 81.7-16.344s63.563 32.025 61.922 67.865-32.024 63.563-67.864 61.922-63.563-32.025-61.922-67.864"
                        }),
                        /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("path", {
                            fill: "url(#ce80f8__f)",
                            d: "M-38.895 69.21c14.246-6.077 29.61-5.817 42.922-.5 26.636 10.663 42.135 4.07 52.885-22.505 5.355-13.336 15.809-24.63 30.071-30.714 28.515-12.146 61.435 1.033 73.564 29.464s-1.147 61.299-29.655 73.461c-14.294 6.098-29.688 5.813-43.02.449-26.593-10.607-42.063-3.989-52.799 22.618-5.368 13.304-15.82 24.559-30.066 30.637-28.493 12.155-61.413-1.024-73.535-29.44-12.122-28.415 1.14-61.315 29.633-73.47"
                        }),
                        /*#__PURE__*/ (0, $lvdRF$reactjsxruntime.jsx)("circle", {
                            cx: 90.406,
                            cy: 17.344,
                            r: 41.765,
                            fill: "url(#ce80f8__g)",
                            transform: "rotate(-.08 90.406 17.344)"
                        })
                    ]
                })
            })
        ]
    });
const $9b4f7e8dd47bce34$var$ForwardRef = /*#__PURE__*/ (0, $lvdRF$react.forwardRef)($9b4f7e8dd47bce34$var$SvgComponent);
var $9b4f7e8dd47bce34$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9b4f7e8dd47bce34$var$ForwardRef);
