// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_document_generic1_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $gqYoA$jsxs, jsx as $gqYoA$jsx} from "react/jsx-runtime";
import {forwardRef as $gqYoA$forwardRef} from "react";





const $c90f02ade5184c44$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gqYoA$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gqYoA$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gqYoA$jsxs)("linearGradient", {
                        id: "868fc6__d",
                        x1: -14.724,
                        x2: 121.006,
                        y1: 80.863,
                        y2: -1.714,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gqYoA$jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $gqYoA$jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gqYoA$jsxs)("linearGradient", {
                        id: "868fc6__e",
                        x1: 94.203,
                        x2: 35.974,
                        y1: 21.635,
                        y2: 84.246,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gqYoA$jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gqYoA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gqYoA$jsxs)("linearGradient", {
                        id: "868fc6__f",
                        x1: 23.438,
                        x2: 126.661,
                        y1: 92.888,
                        y2: 35.064,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gqYoA$jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gqYoA$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gqYoA$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $gqYoA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gqYoA$jsxs)("radialGradient", {
                        id: "868fc6__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 21.982 31.479)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gqYoA$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $gqYoA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gqYoA$jsxs)("radialGradient", {
                        id: "868fc6__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 31.768 53.108)scale(36.3205)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gqYoA$jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gqYoA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gqYoA$jsx)("clipPath", {
                        id: "868fc6__b",
                        children: /*#__PURE__*/ (0, $gqYoA$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gqYoA$jsxs)("clipPath", {
                id: "868fc6__a",
                children: [
                    /*#__PURE__*/ (0, $gqYoA$jsx)("path", {
                        fill: "#fff",
                        d: "M47.997 37.146V16.673c-.194-.016-.39-.03-.585-.03H28.7c-3.997 0-7.236 3.24-7.236 7.237v48.24c0 3.996 3.24 7.236 7.236 7.236h38.592c3.996 0 7.236-3.24 7.236-7.236V43.76c0-.195-.013-.39-.03-.584H54.028c-3.325 0-6.03-2.705-6.03-6.03"
                    }),
                    /*#__PURE__*/ (0, $gqYoA$jsx)("path", {
                        fill: "#fff",
                        d: "M54.026 38.352h18.09L52.82 19.056v18.09c0 .665.541 1.206 1.206 1.206"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gqYoA$jsx)("g", {
                clipPath: "url(#868fc6__a)",
                children: /*#__PURE__*/ (0, $gqYoA$jsxs)("g", {
                    clipPath: "url(#868fc6__b)",
                    children: [
                        /*#__PURE__*/ (0, $gqYoA$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $gqYoA$jsx)("path", {
                            fill: "#FFECCF",
                            d: "M6.258 6.258h83.485v83.485H6.258z"
                        }),
                        /*#__PURE__*/ (0, $gqYoA$jsx)("circle", {
                            cx: 53.461,
                            cy: 9.496,
                            r: 36.321,
                            fill: "url(#868fc6__c)",
                            transform: "rotate(-.08 53.461 9.496)"
                        }),
                        /*#__PURE__*/ (0, $gqYoA$jsx)("path", {
                            fill: "url(#868fc6__d)",
                            d: "M38.121-8.49c-6.412-3.967-14.535-3.992-20.97-.062l-64.366 39.3c-6.304 3.848-6.314 12.933-.02 16.797l63.941 39.243c6.426 3.944 14.55 3.939 20.97-.014l63.808-39.28c6.262-3.855 6.274-12.89.019-16.76z"
                        }),
                        /*#__PURE__*/ (0, $gqYoA$jsx)("path", {
                            fill: "url(#868fc6__e)",
                            d: "M18.29 45.894c1.428-31.167 27.85-55.277 59.018-53.85s55.277 27.85 53.85 59.018-27.85 55.276-59.017 53.849c-31.168-1.426-55.277-27.85-53.85-59.017"
                        }),
                        /*#__PURE__*/ (0, $gqYoA$jsx)("path", {
                            fill: "url(#868fc6__f)",
                            d: "M-27.566 66.445c12.389-5.285 25.75-5.059 37.326-.436 23.164 9.274 36.642 3.54 45.99-19.57 4.657-11.598 13.749-21.42 26.152-26.71 24.798-10.563 53.426.898 63.974 25.623 10.547 24.724-.998 53.307-25.79 63.884-12.43 5.303-25.818 5.055-37.412.39-23.126-9.224-36.579-3.469-45.915 19.67C32.09 140.865 23 150.653 10.612 155.938c-24.778 10.571-53.406-.89-63.948-25.601s.991-53.322 25.77-63.892"
                        }),
                        /*#__PURE__*/ (0, $gqYoA$jsx)("circle", {
                            cx: 84.875,
                            cy: 21.34,
                            r: 36.321,
                            fill: "url(#868fc6__g)",
                            transform: "rotate(-.08 84.875 21.34)"
                        })
                    ]
                })
            })
        ]
    });
const $c90f02ade5184c44$var$ForwardRef = /*#__PURE__*/ (0, $gqYoA$forwardRef)($c90f02ade5184c44$var$SvgComponent);
var $c90f02ade5184c44$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c90f02ade5184c44$var$ForwardRef);


export {$c90f02ade5184c44$export$2e2bcd8739ae039 as default};
