// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_paperairplane_generic1_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $fYxNA$reactjsxruntime = require("react/jsx-runtime");
var $fYxNA$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $13577c0741e5fd8c$export$2e2bcd8739ae039);




const $13577c0741e5fd8c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsxs)("linearGradient", {
                        id: "56e5ea__d",
                        x1: -13.755,
                        x2: 116.638,
                        y1: 79.287,
                        y2: -4.867,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsxs)("linearGradient", {
                        id: "56e5ea__e",
                        x1: 94.428,
                        x2: 40.238,
                        y1: 23.834,
                        y2: 85.647,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsxs)("linearGradient", {
                        id: "56e5ea__f",
                        x1: 24.145,
                        x2: 123.686,
                        y1: 90.546,
                        y2: 31.393,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsxs)("radialGradient", {
                        id: "56e5ea__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.0056 -36.0735 0 36.074 34.006)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsxs)("radialGradient", {
                        id: "56e5ea__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 34.0056 -36.0735 0 36.074 34.006)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("clipPath", {
                        id: "56e5ea__b",
                        children: /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("rect", {
                            width: 82.917,
                            height: 78.164,
                            x: 7.082,
                            y: 9.437,
                            fill: "#fff",
                            rx: 2.077
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("clipPath", {
                id: "56e5ea__a",
                children: /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M82.963 23.51c.986-.221 1.92-.097 2.72.269L23.49 56.07q-.294.154-.526.37L9.651 47.33c-3.07-2.1-2.155-6.824 1.48-7.643zM56.894 79.65 27.38 59.46l60.723-31.53q.045-.022.087-.047c-.031.557-.178 1.127-.46 1.683L63.14 78.061c-1.167 2.302-4.11 3.05-6.247 1.589m-32.296-.86V63.56l14.095 9.604-9.318 7.754c-1.876 1.562-4.777.269-4.777-2.128"
                })
            }),
            /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("g", {
                clipPath: "url(#56e5ea__a)",
                children: /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#56e5ea__b)",
                    children: [
                        /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("rect", {
                            width: 82.917,
                            height: 78.164,
                            x: 7.082,
                            y: 9.437,
                            fill: "#FFECCF",
                            rx: 2.077
                        }),
                        /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M7.082 9.437h82.917V87.6H7.082z"
                        }),
                        /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("ellipse", {
                            cx: 36.074,
                            cy: 34.006,
                            fill: "url(#56e5ea__c)",
                            rx: 36.074,
                            ry: 34.006,
                            transform: "matrix(1 -.00132 .00148 1 17.84 -21.49)"
                        }),
                        /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("path", {
                            fill: "url(#56e5ea__d)",
                            d: "M38.73-4.37c-6.367-3.716-14.435-3.738-20.827-.06l-63.928 36.795c-6.26 3.604-6.27 12.11-.019 15.727l63.506 36.743c6.382 3.691 14.45 3.687 20.827-.014l63.374-36.777c6.219-3.61 6.231-12.068.019-15.69z"
                        }),
                        /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("path", {
                            fill: "url(#56e5ea__e)",
                            d: "M19.033 46.547C20.45 17.367 46.693-5.207 77.65-3.871c30.955 1.336 54.901 26.075 53.483 55.256-1.417 29.18-27.66 51.754-58.615 50.418S17.616 75.728 19.033 46.547"
                        }),
                        /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("path", {
                            fill: "url(#56e5ea__f)",
                            d: "M-26.513 65.788c12.305-4.948 25.576-4.736 37.073-.408 23.006 8.683 36.393 3.314 45.678-18.323 4.625-10.858 13.654-20.054 25.973-25.008 24.629-9.89 53.063.841 63.539 23.99 10.476 23.15-.991 49.91-25.614 59.813-12.347 4.965-25.643 4.732-37.158.365-22.969-8.636-36.33-3.248-45.603 18.416-4.636 10.832-13.664 19.996-25.969 24.945-24.61 9.896-53.043-.834-63.513-23.97s.985-49.923 25.594-59.82"
                        }),
                        /*#__PURE__*/ (0, $fYxNA$reactjsxruntime.jsx)("ellipse", {
                            cx: 36.074,
                            cy: 34.006,
                            fill: "url(#56e5ea__g)",
                            rx: 36.074,
                            ry: 34.006,
                            transform: "matrix(1 -.00132 .00148 1 49.043 -10.4)"
                        })
                    ]
                })
            })
        ]
    });
const $13577c0741e5fd8c$var$ForwardRef = /*#__PURE__*/ (0, $fYxNA$react.forwardRef)($13577c0741e5fd8c$var$SvgComponent);
var $13577c0741e5fd8c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($13577c0741e5fd8c$var$ForwardRef);
