// @react-spectrum/s2/illustrations/gradient/generic1/internal/S2_fill_education_generic1_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4atVd$reactjsxruntime = require("react/jsx-runtime");
var $4atVd$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $074035d53bf14c41$export$2e2bcd8739ae039);




const $074035d53bf14c41$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f3d3ff__d",
                        x1: -40.209,
                        x2: 219.921,
                        y1: 142.983,
                        y2: -15.277,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("stop", {
                                offset: 0.06,
                                stopColor: "#8480FE"
                            }),
                            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("stop", {
                                offset: 0.6,
                                stopColor: "#8480FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f3d3ff__e",
                        x1: 168.544,
                        x2: 56.949,
                        y1: 29.472,
                        y2: 149.467,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("stop", {
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f3d3ff__f",
                        x1: 32.925,
                        x2: 230.753,
                        y1: 166.029,
                        y2: 55.209,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#FC7D00"
                            }),
                            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FC7D00",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f3d3ff__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 42.131 48.337)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#FF709F"
                            }),
                            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF709F",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f3d3ff__g",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 60.885 89.79)scale(69.6088)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("stop", {
                                offset: 0.167,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("clipPath", {
                        id: "f3d3ff__b",
                        children: /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 5.125
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("clipPath", {
                id: "f3d3ff__a",
                children: /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m30.444 50.95 46.532-19.008c1.939-.792 4.111-.792 6.05 0L129.56 50.95c6.633 2.71 6.633 12.102 0 14.812l-4.084 1.668v23.183c0 3.005-.803 7.337-3.836 11.853-3.04 4.528-8.165 9.013-16.445 12.562-6.673 2.859-14.368 3.959-21.085 4.208-6.73.251-12.815-.341-16.436-1.087-2.164-.446-3.557-2.561-3.111-4.724s2.56-3.557 4.724-3.112c2.907.599 8.343 1.159 14.526.929 6.196-.23 12.806-1.243 18.23-3.568 7.111-3.047 10.914-6.627 12.955-9.667 2.048-3.05 2.478-5.79 2.478-7.394V70.699L83.026 84.77c-1.939.792-4.111.792-6.05 0L53.93 75.356c.29-.32.633-.596 1.022-.812L81.943 59.55c1.931-1.073 2.627-3.508 1.554-5.439s-3.508-2.627-5.439-1.554L51.067 67.551c-2.014 1.119-3.623 2.768-4.693 4.718l-15.93-6.507c-6.633-2.71-6.633-12.103 0-14.812m14.45 60.659V78.041c0-2.054.526-4.033 1.48-5.772l7.556 3.087c-.656.724-1.035 1.677-1.035 2.685v32.678c2.817 4.951 5.712 11.287 6.278 16.662q.09.615.09 1.253c0 5.09-4.384 9.216-9.793 9.216s-9.792-4.126-9.792-9.216q0-.513.058-1.011c.422-5.11 2.764-11.132 5.159-16.014"
                })
            }),
            /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("g", {
                clipPath: "url(#f3d3ff__a)",
                children: /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#f3d3ff__b)",
                    children: [
                        /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#FFECCF",
                            rx: 5.125
                        }),
                        /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("path", {
                            fill: "#FFECCF",
                            d: "M0 0h160v160H0z"
                        }),
                        /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("circle", {
                            cx: 90.468,
                            cy: 6.206,
                            r: 69.609,
                            fill: "url(#f3d3ff__c)",
                            transform: "rotate(-.08 90.468 6.206)"
                        }),
                        /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("path", {
                            fill: "url(#f3d3ff__d)",
                            d: "M61.07-28.263c-12.288-7.603-27.857-7.65-40.19-.12l-123.358 75.318c-12.081 7.377-12.101 24.788-.036 32.193l122.542 75.211c12.315 7.557 27.884 7.548 40.188-.027l122.29-75.281c12.001-7.39 12.023-24.703.037-32.12z"
                        }),
                        /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("path", {
                            fill: "url(#f3d3ff__e)",
                            d: "M23.058 75.965C25.793 16.232 76.433-29.975 136.166-27.24c59.732 2.735 105.938 53.375 103.204 113.108s-53.375 105.939-113.108 103.204C66.53 186.337 20.324 135.697 23.058 75.965"
                        }),
                        /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("path", {
                            fill: "url(#f3d3ff__f)",
                            d: "M-64.825 115.35c23.744-10.129 49.351-9.695 71.537-.835 44.394 17.773 70.225 6.784 88.141-37.508 8.925-22.226 26.348-41.049 50.119-51.19 47.525-20.243 102.392 1.723 122.607 49.108 20.214 47.385-1.912 102.165-49.426 122.435-23.824 10.163-49.48 9.687-71.7.747-44.322-17.678-70.104-6.648-87.998 37.698-8.947 22.173-26.366 40.931-50.11 51.061-47.488 20.258-102.354-1.707-122.558-49.066-20.203-47.359 1.9-102.191 49.388-122.45"
                        }),
                        /*#__PURE__*/ (0, $4atVd$reactjsxruntime.jsx)("circle", {
                            cx: 150.675,
                            cy: 28.906,
                            r: 69.609,
                            fill: "url(#f3d3ff__g)",
                            transform: "rotate(-.08 150.675 28.906)"
                        })
                    ]
                })
            })
        ]
    });
const $074035d53bf14c41$var$ForwardRef = /*#__PURE__*/ (0, $4atVd$react.forwardRef)($074035d53bf14c41$var$SvgComponent);
var $074035d53bf14c41$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($074035d53bf14c41$var$ForwardRef);
