// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_dataAnalytics_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gh0WL$reactjsxruntime = require("react/jsx-runtime");
var $gh0WL$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d544414c847fd12a$export$2e2bcd8739ae039);




const $d544414c847fd12a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsxs)("linearGradient", {
                        id: "64b345__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsxs)("linearGradient", {
                        id: "64b345__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsxs)("linearGradient", {
                        id: "64b345__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsxs)("radialGradient", {
                        id: "64b345__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("clipPath", {
                        id: "64b345__b",
                        children: /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("clipPath", {
                id: "64b345__a",
                children: /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M39.914 10.252c.428.541.337 1.327-.205 1.756l-11.548 9.136c-.387.306-.917.356-1.354.127l-7.21-3.768-9.726 8.25c-.526.447-1.315.382-1.762-.145-.446-.526-.382-1.315.145-1.762l10.368-8.793c.388-.33.936-.39 1.387-.155l7.237 3.782 10.912-8.633c.542-.428 1.328-.337 1.756.205M17.727 26.184c0-1.105.895-2 2-2s2 .895 2 2v8.788c0 1.105-.896 2-2 2s-2-.895-2-2zm10.603-.29c-1.104 0-2 .896-2 2v7.078c0 1.105.896 2 2 2s2-.895 2-2v-7.077c0-1.105-.895-2-2-2m6.604-4.02c0-1.105.895-2 2-2s2 .895 2 2v13.099c0 1.105-.896 2-2 2s-2-.895-2-2zm-23.81 7.41c-1.106 0-2 .895-2 2v3.69c0 1.104.894 2 2 2 1.104 0 2-.896 2-2v-3.69c0-1.105-.896-2-2-2"
                })
            }),
            /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("g", {
                clipPath: "url(#64b345__a)",
                children: /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#64b345__b)",
                    children: [
                        /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#64b345__c)"
                        }),
                        /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("path", {
                            fill: "url(#64b345__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("path", {
                            fill: "url(#64b345__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $gh0WL$reactjsxruntime.jsx)("path", {
                            fill: "url(#64b345__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $d544414c847fd12a$var$ForwardRef = /*#__PURE__*/ (0, $gh0WL$react.forwardRef)($d544414c847fd12a$var$SvgComponent);
var $d544414c847fd12a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d544414c847fd12a$var$ForwardRef);
