// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lockOpen_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $axZuq$jsxs, jsx as $axZuq$jsx} from "react/jsx-runtime";
import {forwardRef as $axZuq$forwardRef} from "react";





const $b3b7f1ad44d0b92c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $axZuq$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $axZuq$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $axZuq$jsxs)("linearGradient", {
                        id: "d718df__d",
                        x1: 66.664,
                        x2: 18.316,
                        y1: 3.573,
                        y2: 66.258,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $axZuq$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $axZuq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $axZuq$jsxs)("linearGradient", {
                        id: "d718df__e",
                        x1: 37.208,
                        x2: 46.535,
                        y1: -25.285,
                        y2: 34.406,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $axZuq$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $axZuq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $axZuq$jsxs)("linearGradient", {
                        id: "d718df__f",
                        x1: -70.407,
                        x2: 86.52,
                        y1: 94.721,
                        y2: 63.707,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $axZuq$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $axZuq$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $axZuq$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $axZuq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $axZuq$jsxs)("radialGradient", {
                        id: "d718df__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 9.766 70.908)scale(120.537)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $axZuq$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $axZuq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $axZuq$jsx)("clipPath", {
                        id: "d718df__b",
                        children: /*#__PURE__*/ (0, $axZuq$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $axZuq$jsx)("clipPath", {
                id: "d718df__a",
                children: /*#__PURE__*/ (0, $axZuq$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M32.402 27.3c0-8.616 6.984-15.6 15.6-15.6s15.6 6.984 15.6 15.6c0 1.325-1.075 2.4-2.4 2.4-1.326 0-2.4-1.075-2.4-2.4 0-5.966-4.836-10.8-10.8-10.8s-10.8 4.834-10.8 10.8v12.6h32.181c4.429 0 8.019 3.273 8.019 7.312v24.375c0 4.038-3.59 7.312-8.019 7.312H26.62c-4.429 0-8.018-3.274-8.018-7.312V47.212c0-4.039 3.59-7.313 8.018-7.313h5.782zM44.61 60.39c.296.298.63.554.992.763v4.545c0 1.326 1.075 2.4 2.4 2.4s2.4-1.074 2.4-2.4v-4.545c.362-.209.696-.465.992-.763.224-.216.424-.464.6-.728.176-.256.32-.536.44-.824.12-.296.216-.592.272-.904.064-.304.096-.624.096-.936s-.032-.632-.096-.936c-.056-.304-.153-.608-.272-.896-.12-.296-.264-.568-.44-.832s-.376-.512-.6-.728c-.888-.896-2.128-1.408-3.392-1.408s-2.504.512-3.392 1.408c-.224.216-.424.464-.6.728-.168.264-.32.536-.44.832-.12.288-.217.592-.272.896-.064.304-.096.624-.096.936s.032.632.096.936c.055.312.152.608.272.904.12.288.272.568.44.824.176.264.376.512.6.728"
                })
            }),
            /*#__PURE__*/ (0, $axZuq$jsx)("g", {
                clipPath: "url(#d718df__a)",
                children: /*#__PURE__*/ (0, $axZuq$jsxs)("g", {
                    clipPath: "url(#d718df__b)",
                    children: [
                        /*#__PURE__*/ (0, $axZuq$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $axZuq$jsx)("circle", {
                            cx: 80.674,
                            cy: 61.143,
                            r: 120.537,
                            fill: "url(#d718df__c)"
                        }),
                        /*#__PURE__*/ (0, $axZuq$jsx)("path", {
                            fill: "url(#d718df__d)",
                            d: "M-1.255 32.406C-2.595 3.55 19.71-20.928 48.567-22.268s53.335 20.966 54.675 49.822S82.275 80.89 53.419 82.23.085 61.263-1.255 32.406"
                        }),
                        /*#__PURE__*/ (0, $axZuq$jsx)("path", {
                            fill: "url(#d718df__e)",
                            d: "M27.349-51.511c-6.51-4.028-14.757-4.053-21.291-.064l-65.35 39.9c-6.4 3.908-6.41 13.132-.018 17.054L5.607 45.223c6.524 4.003 14.771 3.998 21.29-.015L91.68 5.328c6.358-3.915 6.37-13.087.02-17.016z"
                        }),
                        /*#__PURE__*/ (0, $axZuq$jsx)("path", {
                            fill: "url(#d718df__f)",
                            d: "M-35.36 46.189c13.47 0 25.671 5.451 34.505 14.246 17.667 17.619 32.314 17.634 49.981.045 8.834-8.84 21.05-14.306 34.535-14.306 26.953.015 48.788 21.79 48.788 48.671s-21.835 48.641-48.789 48.641c-13.514 0-25.73-5.482-34.564-14.321-17.652-17.559-32.284-17.544-49.951.075-8.834 8.81-21.035 14.246-34.504 14.246-26.939 0-48.774-21.775-48.774-48.641 0-26.865 21.835-48.656 48.774-48.656"
                        })
                    ]
                })
            })
        ]
    });
const $b3b7f1ad44d0b92c$var$ForwardRef = /*#__PURE__*/ (0, $axZuq$forwardRef)($b3b7f1ad44d0b92c$var$SvgComponent);
var $b3b7f1ad44d0b92c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b3b7f1ad44d0b92c$var$ForwardRef);


export {$b3b7f1ad44d0b92c$export$2e2bcd8739ae039 as default};
