// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_stepForward_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $6C8aT$reactjsxruntime = require("react/jsx-runtime");
var $6C8aT$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5032f7c653d82bd2$export$2e2bcd8739ae039);




const $5032f7c653d82bd2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4004e7__d",
                        x1: 32.731,
                        x2: 4.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4004e7__e",
                        x1: 15.793,
                        x2: 21.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4004e7__f",
                        x1: -46.078,
                        x2: 44.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4004e7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 4.616 36.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("clipPath", {
                        id: "4004e7__b",
                        children: /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            x: -2,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("clipPath", {
                id: "4004e7__a",
                children: /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "m39.577 21.564-16.373-9.777c-1.834-1.095-4.13.268-4.132 2.453l-.022 19.514c-.003 2.185 2.29 3.554 4.125 2.464l16.396-9.737c1.836-1.09 1.84-3.822.006-4.917M7 13c0-1.105.895-2 2-2h3.5c1.105 0 2 .895 2 2v22c0 1.105-.895 2-2 2H9c-1.105 0-2-.895-2-2z"
                })
            }),
            /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("g", {
                clipPath: "url(#4004e7__a)",
                children: /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#4004e7__b)",
                    children: [
                        /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            x: -2,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("circle", {
                            cx: 40.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#4004e7__c)"
                        }),
                        /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("path", {
                            fill: "url(#4004e7__d)",
                            d: "M-6.32 15.035C-7.09-1.557 5.735-15.63 22.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-5.55 31.625-6.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("path", {
                            fill: "url(#4004e7__e)",
                            d: "M10.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-39.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805l37.324 22.909c3.75 2.301 8.493 2.299 12.24-.009L47.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $6C8aT$reactjsxruntime.jsx)("path", {
                            fill: "url(#4004e7__f)",
                            d: "M-25.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S58 78.899 42.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $5032f7c653d82bd2$var$ForwardRef = /*#__PURE__*/ (0, $6C8aT$react.forwardRef)($5032f7c653d82bd2$var$SvgComponent);
var $5032f7c653d82bd2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5032f7c653d82bd2$var$ForwardRef);
