// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lightbulb_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $eRJrc$jsxs, jsx as $eRJrc$jsx} from "react/jsx-runtime";
import {forwardRef as $eRJrc$forwardRef} from "react";





const $ded27dcd9a9fd319$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eRJrc$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $eRJrc$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $eRJrc$jsxs)("linearGradient", {
                        id: "7379dd__d",
                        x1: 33.33,
                        x2: 9.156,
                        y1: 1.786,
                        y2: 33.128,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eRJrc$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $eRJrc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eRJrc$jsxs)("linearGradient", {
                        id: "7379dd__e",
                        x1: 18.602,
                        x2: 23.265,
                        y1: -12.643,
                        y2: 17.203,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eRJrc$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $eRJrc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eRJrc$jsxs)("linearGradient", {
                        id: "7379dd__f",
                        x1: -35.206,
                        x2: 43.258,
                        y1: 47.361,
                        y2: 31.854,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eRJrc$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $eRJrc$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $eRJrc$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $eRJrc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eRJrc$jsxs)("radialGradient", {
                        id: "7379dd__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 4.884 35.455)scale(60.2686)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eRJrc$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $eRJrc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eRJrc$jsx)("clipPath", {
                        id: "7379dd__b",
                        children: /*#__PURE__*/ (0, $eRJrc$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $eRJrc$jsx)("clipPath", {
                id: "7379dd__a",
                children: /*#__PURE__*/ (0, $eRJrc$jsx)("path", {
                    fill: "#fff",
                    d: "M23.21 8.157c-5.327.376-9.641 4.675-10.035 10-.292 3.956 1.537 7.502 4.467 9.622.952.69 1.532 1.779 1.532 2.954v2.797h9.649v-2.795c0-1.146.53-2.242 1.464-2.907 2.763-1.967 4.566-5.195 4.566-8.845 0-6.255-5.292-11.275-11.643-10.826m6.819 10.75c-.625 0-1.131-.506-1.131-1.13 0-2.037-1.657-3.693-3.693-3.693-.625 0-1.131-.507-1.131-1.131 0-.625.506-1.13 1.13-1.13 3.284 0 5.955 2.67 5.955 5.954 0 .624-.506 1.13-1.13 1.13M19.176 37.675c0 1 .81 1.81 1.809 1.81h6.03c.999 0 1.809-.81 1.809-1.81v-1.884h-9.648z"
                })
            }),
            /*#__PURE__*/ (0, $eRJrc$jsx)("g", {
                clipPath: "url(#7379dd__a)",
                children: /*#__PURE__*/ (0, $eRJrc$jsxs)("g", {
                    clipPath: "url(#7379dd__b)",
                    children: [
                        /*#__PURE__*/ (0, $eRJrc$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $eRJrc$jsx)("circle", {
                            cx: 40.339,
                            cy: 30.571,
                            r: 60.269,
                            fill: "url(#7379dd__c)"
                        }),
                        /*#__PURE__*/ (0, $eRJrc$jsx)("path", {
                            fill: "url(#7379dd__d)",
                            d: "M-.63 16.202C-1.3 1.774 9.854-10.465 24.282-11.135S50.949-.652 51.619 13.776 41.136 40.444 26.708 41.113.04 30.63-.63 16.203"
                        }),
                        /*#__PURE__*/ (0, $eRJrc$jsx)("path", {
                            fill: "url(#7379dd__e)",
                            d: "M13.672-25.756c-3.254-2.014-7.378-2.027-10.645-.032l-32.675 19.95c-3.2 1.954-3.205 6.566-.01 8.527l32.46 19.922c3.261 2.002 7.385 2 10.644-.007l32.392-19.94c3.179-1.958 3.185-6.544.01-8.508z"
                        }),
                        /*#__PURE__*/ (0, $eRJrc$jsx)("path", {
                            fill: "url(#7379dd__f)",
                            d: "M-17.682 23.095c6.735 0 12.836 2.726 17.252 7.123 8.834 8.81 16.158 8.817 24.991.023 4.417-4.42 10.525-7.153 17.267-7.153 13.477.007 24.395 10.895 24.395 24.335S55.305 71.745 41.828 71.745c-6.757 0-12.865-2.74-17.282-7.16-8.826-8.78-16.142-8.773-24.976.037-4.416 4.405-10.517 7.123-17.252 7.123-13.469 0-24.386-10.888-24.386-24.32s10.917-24.329 24.386-24.329"
                        })
                    ]
                })
            })
        ]
    });
const $ded27dcd9a9fd319$var$ForwardRef = /*#__PURE__*/ (0, $eRJrc$forwardRef)($ded27dcd9a9fd319$var$SvgComponent);
var $ded27dcd9a9fd319$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ded27dcd9a9fd319$var$ForwardRef);


export {$ded27dcd9a9fd319$export$2e2bcd8739ae039 as default};
