// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_hand_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $bkqMi$reactjsxruntime = require("react/jsx-runtime");
var $bkqMi$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f51c1ea7104ca3e9$export$2e2bcd8739ae039);




const $f51c1ea7104ca3e9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d66b74__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d66b74__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d66b74__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d66b74__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("clipPath", {
                        id: "d66b74__b",
                        children: /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("clipPath", {
                id: "d66b74__a",
                children: /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m18.653 50.734-.022-.04c-1.363-2.365-1.023-4.963.9-6.408 1.982-1.489 4.786-1.106 6.299.861l5.888 7.657c.26.339.709.474 1.114.337.406-.138.678-.519.678-.947V26.63c0-2.698 2.176-4.877 4.852-4.877s4.852 2.18 4.852 4.877v16.44c0 .553.448 1 1 1h.012c.553 0 1-.447 1-1V20.878c0-2.698 2.176-4.877 4.852-4.877s4.852 2.179 4.852 4.877v22.198c0 .552.448 1 1 1h.012c.552 0 1-.448 1-1V23.66c0-2.698 2.176-4.877 4.852-4.877 2.68 0 4.855 2.18 4.855 4.877v20.878c0 .53.415.969.944.998.53.03.99-.36 1.05-.887l.559-4.991.559-4.991.001-.014c.285-2.91 2.877-4.358 5.257-3.89 1.916.376 3.424 2.095 3.138 4.864l-2.487 17.39q-.025.074-.04.152c-2 11.113-5.291 17.778-9.49 21.678C61.97 78.72 56.753 80 50.56 80c-4.05 0-8.065-1.742-11.86-4.47-3.788-2.722-7.282-6.375-10.266-10.062-2.98-3.683-5.429-7.371-7.133-10.141-.852-1.384-1.516-2.537-1.966-3.341-.226-.402-.397-.718-.512-.931l-.13-.243-.031-.06-.008-.015z"
                })
            }),
            /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("g", {
                clipPath: "url(#d66b74__a)",
                children: /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#d66b74__b)",
                    children: [
                        /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#d66b74__c)"
                        }),
                        /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("path", {
                            fill: "url(#d66b74__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("path", {
                            fill: "url(#d66b74__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $bkqMi$reactjsxruntime.jsx)("path", {
                            fill: "url(#d66b74__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $f51c1ea7104ca3e9$var$ForwardRef = /*#__PURE__*/ (0, $bkqMi$react.forwardRef)($f51c1ea7104ca3e9$var$SvgComponent);
var $f51c1ea7104ca3e9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f51c1ea7104ca3e9$var$ForwardRef);
