// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_threeArrowsCircle_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6cbSi$jsxs, jsx as $6cbSi$jsx} from "react/jsx-runtime";
import {forwardRef as $6cbSi$forwardRef} from "react";





const $d8d4db8a3a942e8b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6cbSi$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6cbSi$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6cbSi$jsxs)("linearGradient", {
                        id: "a7b623__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6cbSi$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6cbSi$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6cbSi$jsxs)("linearGradient", {
                        id: "a7b623__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6cbSi$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6cbSi$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6cbSi$jsxs)("linearGradient", {
                        id: "a7b623__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6cbSi$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6cbSi$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6cbSi$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6cbSi$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6cbSi$jsxs)("radialGradient", {
                        id: "a7b623__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6cbSi$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6cbSi$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6cbSi$jsx)("clipPath", {
                        id: "a7b623__b",
                        children: /*#__PURE__*/ (0, $6cbSi$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6cbSi$jsx)("clipPath", {
                id: "a7b623__a",
                children: /*#__PURE__*/ (0, $6cbSi$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M126.811 40.187c.548 2.14-.743 4.319-2.883 4.867l-24.791 6.345c-2.14.548-4.32-.743-4.867-2.883-.548-2.14.743-4.319 2.883-4.867l15.233-3.899q-.122-.082-.239-.175c-8.853-7.027-20.047-11.222-32.23-11.222-15.564 0-29.523 6.846-39.04 17.705-1.455 1.662-3.982 1.828-5.644.372s-1.828-3.983-.372-5.644c10.971-12.519 27.091-20.433 45.055-20.433 13.867 0 26.642 4.717 36.794 12.632l-3.995-15.605c-.548-2.14.743-4.32 2.883-4.867 2.14-.548 4.319.743 4.867 2.883zM129.71 64.8c-.659-2.108.515-4.352 2.623-5.012s4.353.515 5.012 2.623c1.771 5.658 2.723 11.673 2.723 17.903 0 24.134-14.273 44.925-34.828 54.407l14.547 5.119c2.084.733 3.179 3.017 2.446 5.101s-3.017 3.179-5.101 2.445l-24.14-8.493c-2.084-.733-3.179-3.017-2.445-5.101l8.493-24.139c.733-2.084 3.017-3.179 5.101-2.446s3.179 3.017 2.445 5.101l-5.449 15.489c18.223-8.055 30.931-26.289 30.931-47.483 0-5.41-.826-10.62-2.358-15.514M28.689 73.392 38.754 85.73c1.396 1.712 3.916 1.968 5.627.571 1.712-1.396 1.968-3.916.571-5.628L28.776 60.844c-1.397-1.712-3.916-1.968-5.628-.571L3.319 76.449c-1.712 1.397-1.967 3.916-.57 5.628 1.396 1.712 3.916 1.968 5.627.571l12.351-10.076c-1.473 11.234.22 23.014 5.55 33.958 7.866 16.15 22.04 27.178 38.098 31.56 2.132.581 4.33-.675 4.912-2.806s-.674-4.331-2.806-4.912c-13.93-3.801-26.197-13.353-33.01-27.345-4.656-9.559-6.11-19.838-4.782-29.635"
                })
            }),
            /*#__PURE__*/ (0, $6cbSi$jsx)("g", {
                clipPath: "url(#a7b623__a)",
                children: /*#__PURE__*/ (0, $6cbSi$jsxs)("g", {
                    clipPath: "url(#a7b623__b)",
                    children: [
                        /*#__PURE__*/ (0, $6cbSi$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6cbSi$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#a7b623__c)"
                        }),
                        /*#__PURE__*/ (0, $6cbSi$jsx)("path", {
                            fill: "url(#a7b623__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $6cbSi$jsx)("path", {
                            fill: "url(#a7b623__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $6cbSi$jsx)("path", {
                            fill: "url(#a7b623__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $d8d4db8a3a942e8b$var$ForwardRef = /*#__PURE__*/ (0, $6cbSi$forwardRef)($d8d4db8a3a942e8b$var$SvgComponent);
var $d8d4db8a3a942e8b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d8d4db8a3a942e8b$var$ForwardRef);


export {$d8d4db8a3a942e8b$export$2e2bcd8739ae039 as default};
