// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_laptop_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4A8ZG$reactjsxruntime = require("react/jsx-runtime");
var $4A8ZG$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0f9025c7f4e040cf$export$2e2bcd8739ae039);




const $0f9025c7f4e040cf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsxs)("linearGradient", {
                        id: "588828__d",
                        x1: 111.403,
                        x2: 38.379,
                        y1: 14.175,
                        y2: 114.609,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsxs)("linearGradient", {
                        id: "588828__e",
                        x1: 63.417,
                        x2: 76.955,
                        y1: -30.14,
                        y2: 61.768,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsxs)("linearGradient", {
                        id: "588828__f",
                        x1: -111.894,
                        x2: 142.547,
                        y1: 154.147,
                        y2: 100.803,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsxs)("radialGradient", {
                        id: "588828__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 185.103 -196.359 0 134.227 102.582)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("clipPath", {
                        id: "588828__b",
                        children: /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("rect", {
                            width: 136,
                            height: 128.204,
                            x: 13,
                            y: 18.298,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("clipPath", {
                id: "588828__a",
                children: /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M136 110h-8V48.22c0-8.96-7.16-16.22-16-16.22H48c-8.84 0-16 7.26-16 16.22V110h-8c-4.97 0-9 4.03-9 9s4.03 9 9 9h112c4.97 0 9-4.03 9-9s-4.03-9-9-9M80 49c-2.21 0-4-1.79-4-4s1.79-4 4-4 4 1.79 4 4-1.79 4-4 4"
                })
            }),
            /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("g", {
                clipPath: "url(#588828__a)",
                children: /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#588828__b)",
                    children: [
                        /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("rect", {
                            width: 136,
                            height: 128.204,
                            x: 13,
                            y: 18.298,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("ellipse", {
                            cx: 134.227,
                            cy: 102.582,
                            fill: "url(#588828__c)",
                            rx: 196.359,
                            ry: 185.103
                        }),
                        /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("path", {
                            fill: "url(#588828__d)",
                            d: "M.76 58.452C-1.423 14.14 34.915-23.45 81.922-25.509s86.884 32.197 89.067 76.51-34.155 81.903-81.162 83.961S2.943 102.765.76 58.452"
                        }),
                        /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("path", {
                            fill: "url(#588828__e)",
                            d: "M47.356-70.415c-10.604-6.186-24.04-6.224-34.683-.098L-93.783-9.24c-10.426 6-10.444 20.165-.032 26.189L11.938 78.135c10.628 6.148 24.063 6.14 34.682-.022L152.154 16.87c10.357-6.011 10.376-20.097.032-26.13z"
                        }),
                        /*#__PURE__*/ (0, $4A8ZG$reactjsxruntime.jsx)("path", {
                            fill: "url(#588828__f)",
                            d: "M-54.8 79.619c21.943 0 41.818 8.371 56.209 21.877 28.78 27.057 52.641 27.08 81.422.069 14.39-13.575 34.29-21.97 56.257-21.97 43.909.024 79.478 33.463 79.478 74.743s-35.569 74.695-79.478 74.695c-22.016 0-41.916-8.417-56.307-21.992-28.756-26.965-52.592-26.941-81.372.115-14.39 13.529-34.266 21.877-56.208 21.877-43.885 0-79.454-33.439-79.454-74.695s35.57-74.72 79.454-74.72"
                        })
                    ]
                })
            })
        ]
    });
const $0f9025c7f4e040cf$var$ForwardRef = /*#__PURE__*/ (0, $4A8ZG$react.forwardRef)($0f9025c7f4e040cf$var$SvgComponent);
var $0f9025c7f4e040cf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0f9025c7f4e040cf$var$ForwardRef);
