// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_mailOpen_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $kTmaQ$reactjsxruntime = require("react/jsx-runtime");
var $kTmaQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $8020d73b6cd8e8f6$export$2e2bcd8739ae039);




const $8020d73b6cd8e8f6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "15b3a1__d",
                        x1: 66.406,
                        x2: 18.058,
                        y1: 3.316,
                        y2: 66.001,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "15b3a1__e",
                        x1: 36.95,
                        x2: 46.277,
                        y1: -25.543,
                        y2: 34.148,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "15b3a1__f",
                        x1: -70.665,
                        x2: 86.262,
                        y1: 94.463,
                        y2: 63.449,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "15b3a1__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 9.766 70.65)scale(120.537)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("clipPath", {
                        id: "15b3a1__b",
                        children: /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6,
                            y: 6,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("clipPath", {
                id: "15b3a1__a",
                children: /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M35.958 28.848c-3.972 0-7.2 3.228-7.2 7.2v2.946l-9.684-6.072 26.472-16.566c1.566-.978 3.546-.978 5.106.006l26.28 16.554-9.69 6.078v-2.946c0-3.972-3.228-7.2-7.2-7.2zm0 4.8c-1.325 0-2.4 1.074-2.4 2.4V42l13.17 8.262c.775.486 1.777.486 2.55 0L62.449 42v-5.958c0-1.326-1.073-2.4-2.4-2.4H35.966zm15.87 20.682c-1.164.726-2.495 1.092-3.828 1.092-1.331 0-2.663-.366-3.828-1.092l-27.36-17.16v33.612c0 5.304 4.296 9.606 9.6 9.606H69.59c5.304 0 9.6-4.302 9.6-9.606V37.17z"
                })
            }),
            /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("g", {
                clipPath: "url(#15b3a1__a)",
                children: /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#15b3a1__b)",
                    children: [
                        /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6,
                            y: 6,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("circle", {
                            cx: 80.416,
                            cy: 60.885,
                            r: 120.537,
                            fill: "url(#15b3a1__c)"
                        }),
                        /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("path", {
                            fill: "url(#15b3a1__d)",
                            d: "M-1.513 32.149c-1.34-28.856 20.966-53.335 49.822-54.675s53.335 20.966 54.675 49.822-20.966 53.335-49.822 54.675S-.173 61.005-1.513 32.149"
                        }),
                        /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("path", {
                            fill: "url(#15b3a1__e)",
                            d: "M27.091-51.77c-6.51-4.027-14.757-4.052-21.29-.063l-65.35 39.9c-6.4 3.908-6.41 13.132-.02 17.054L5.35 44.965c6.524 4.003 14.772 3.999 21.29-.015L91.422 5.07c6.358-3.914 6.37-13.087.02-17.016z"
                        }),
                        /*#__PURE__*/ (0, $kTmaQ$reactjsxruntime.jsx)("path", {
                            fill: "url(#15b3a1__f)",
                            d: "M-35.617 45.931c13.47 0 25.67 5.452 34.504 14.246 17.667 17.62 32.314 17.634 49.982.045 8.833-8.84 21.05-14.306 34.534-14.306 26.954.015 48.788 21.79 48.788 48.671s-21.834 48.641-48.788 48.641c-13.515 0-25.731-5.481-34.565-14.321-17.652-17.559-32.284-17.544-49.951.075-8.834 8.81-21.035 14.246-34.504 14.246-26.939 0-48.774-21.775-48.774-48.64S-62.556 45.93-35.617 45.93"
                        })
                    ]
                })
            })
        ]
    });
const $8020d73b6cd8e8f6$var$ForwardRef = /*#__PURE__*/ (0, $kTmaQ$react.forwardRef)($8020d73b6cd8e8f6$var$SvgComponent);
var $8020d73b6cd8e8f6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($8020d73b6cd8e8f6$var$ForwardRef);
