// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_buildings_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8bZ6W$reactjsxruntime = require("react/jsx-runtime");
var $8bZ6W$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $dac70890289eeec4$export$2e2bcd8739ae039);




const $dac70890289eeec4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8d53ee__d",
                        x1: 65.847,
                        x2: 20.038,
                        y1: 8.215,
                        y2: 71.154,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8d53ee__e",
                        x1: 35.785,
                        x2: 44.283,
                        y1: -19.576,
                        y2: 38.059,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8d53ee__f",
                        x1: -74.043,
                        x2: 85.371,
                        y1: 95.995,
                        y2: 62.609,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8d53ee__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 116.083 -123.013 0 80.146 63.657)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("clipPath", {
                        id: "8d53ee__b",
                        children: /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("rect", {
                            width: 85.2,
                            height: 80.4,
                            x: 4.199,
                            y: 10.801,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("clipPath", {
                id: "8d53ee__a",
                children: /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M21.452 16.758h28.842c5.304 0 9.6 4.296 9.6 9.6v52.878H44.3V66.888c0-2.652-2.148-4.8-4.8-4.8h-7.254c-2.652 0-4.8 2.148-4.8 4.8v12.348H11.852V26.358c0-5.304 4.296-9.6 9.6-9.6m7.674 33.936c1.326 0 2.4-1.074 2.4-2.4v-3.456c0-1.326-1.074-2.4-2.4-2.4H25.67c-1.326 0-2.4 1.074-2.4 2.4v3.456c0 1.326 1.074 2.4 2.4 2.4zm0-16.836c1.326 0 2.4-1.074 2.4-2.4v-3.456c0-1.326-1.074-2.4-2.4-2.4H25.67c-1.326 0-2.4 1.074-2.4 2.4v3.456c0 1.326 1.074 2.4 2.4 2.4zm16.956 16.836c1.326 0 2.4-1.074 2.4-2.4v-3.456c0-1.326-1.074-2.4-2.4-2.4h-3.456c-1.326 0-2.4 1.074-2.4 2.4v3.456c0 1.326 1.074 2.4 2.4 2.4zm0-16.836c1.326 0 2.4-1.074 2.4-2.4v-3.456c0-1.326-1.074-2.4-2.4-2.4h-3.456c-1.326 0-2.4 1.074-2.4 2.4v3.456c0 1.326 1.074 2.4 2.4 2.4zm28.464-4.122c5.304 0 9.6 4.296 9.6 9.6v39.9H64.7v-49.5zm-.798 37.296c1.326 0 2.4-1.074 2.4-2.4v-3.456c0-1.326-1.074-2.4-2.4-2.4h-3.456c-1.326 0-2.4 1.074-2.4 2.4v3.456c0 1.326 1.074 2.4 2.4 2.4zm0-16.836c1.326 0 2.4-1.074 2.4-2.4V44.34c0-1.326-1.074-2.4-2.4-2.4h-3.456c-1.326 0-2.4 1.074-2.4 2.4v3.456c0 1.326 1.074 2.4 2.4 2.4z"
                })
            }),
            /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("g", {
                clipPath: "url(#8d53ee__a)",
                children: /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#8d53ee__b)",
                    children: [
                        /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("rect", {
                            width: 85.2,
                            height: 80.4,
                            x: 4.199,
                            y: 10.801,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("ellipse", {
                            cx: 80.146,
                            cy: 63.657,
                            fill: "url(#8d53ee__c)",
                            rx: 123.013,
                            ry: 116.083
                        }),
                        /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("path", {
                            fill: "url(#8d53ee__d)",
                            d: "M-3.467 35.982C-4.835 8.192 17.93-15.382 47.379-16.672s54.43 20.191 55.797 47.981c1.368 27.79-21.397 51.364-50.845 52.654-29.45 1.29-54.43-20.191-55.798-47.98"
                        }),
                        /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("path", {
                            fill: "url(#8d53ee__e)",
                            d: "M25.724-44.834c-6.644-3.879-15.06-3.903-21.729-.061L-62.696-6.469c-6.532 3.763-6.543 12.646-.02 16.424l66.25 38.37c6.659 3.856 15.076 3.852 21.728-.013L91.376 9.905c6.489-3.77 6.5-12.603.02-16.387z"
                        }),
                        /*#__PURE__*/ (0, $8bZ6W$reactjsxruntime.jsx)("path", {
                            fill: "url(#8d53ee__f)",
                            d: "M-38.275 49.257c13.746 0 26.197 5.25 35.213 13.72 18.03 16.967 32.978 16.982 51.008.043 9.015-8.513 21.482-13.778 35.244-13.778 27.507.015 49.79 20.986 49.79 46.873s-22.283 46.843-49.79 46.843c-13.793 0-26.26-5.279-35.275-13.792-18.015-16.91-32.947-16.895-50.977.073-9.016 8.484-21.467 13.719-35.213 13.719-27.492 0-49.776-20.97-49.776-46.843s22.284-46.858 49.776-46.858"
                        })
                    ]
                })
            })
        ]
    });
const $dac70890289eeec4$var$ForwardRef = /*#__PURE__*/ (0, $8bZ6W$react.forwardRef)($dac70890289eeec4$var$SvgComponent);
var $dac70890289eeec4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($dac70890289eeec4$var$ForwardRef);
