// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_checkmark_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $jNyEu$jsxs, jsx as $jNyEu$jsx} from "react/jsx-runtime";
import {forwardRef as $jNyEu$forwardRef} from "react";





const $6860d7daeee01f03$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jNyEu$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jNyEu$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jNyEu$jsxs)("linearGradient", {
                        id: "0d1e0f__d",
                        x1: 33.286,
                        x2: 9.233,
                        y1: 1.897,
                        y2: 33.084,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jNyEu$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $jNyEu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jNyEu$jsxs)("linearGradient", {
                        id: "0d1e0f__e",
                        x1: 18.631,
                        x2: 23.271,
                        y1: -12.46,
                        y2: 17.237,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jNyEu$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $jNyEu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jNyEu$jsxs)("linearGradient", {
                        id: "0d1e0f__f",
                        x1: -34.908,
                        x2: 43.165,
                        y1: 47.245,
                        y2: 31.815,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jNyEu$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jNyEu$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $jNyEu$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $jNyEu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jNyEu$jsxs)("radialGradient", {
                        id: "0d1e0f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 59.9687 -59.9688 0 40.258 30.54)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jNyEu$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jNyEu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jNyEu$jsx)("clipPath", {
                        id: "0d1e0f__b",
                        children: /*#__PURE__*/ (0, $jNyEu$jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.234,
                            y: 3.232,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jNyEu$jsx)("clipPath", {
                id: "0d1e0f__a",
                children: /*#__PURE__*/ (0, $jNyEu$jsx)("path", {
                    fill: "#fff",
                    d: "M23.998 8.4c-8.615 0-15.6 6.985-15.6 15.6 0 8.616 6.985 15.6 15.6 15.6 8.616 0 15.6-6.984 15.6-15.6s-6.984-15.6-15.6-15.6m6.208 12.85-7.2 7.199c-.225.225-.53.351-.848.351h-.007c-.32-.002-.627-.132-.851-.36l-3.519-3.6c-.463-.475-.455-1.235.02-1.698.473-.463 1.233-.454 1.696.02l2.67 2.732 6.343-6.343c.469-.468 1.229-.467 1.697.001.468.47.468 1.23-.001 1.697"
                })
            }),
            /*#__PURE__*/ (0, $jNyEu$jsx)("g", {
                clipPath: "url(#0d1e0f__a)",
                children: /*#__PURE__*/ (0, $jNyEu$jsxs)("g", {
                    clipPath: "url(#0d1e0f__b)",
                    children: [
                        /*#__PURE__*/ (0, $jNyEu$jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.234,
                            y: 3.232,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $jNyEu$jsx)("circle", {
                            cx: 40.258,
                            cy: 30.539,
                            r: 59.969,
                            fill: "url(#0d1e0f__c)"
                        }),
                        /*#__PURE__*/ (0, $jNyEu$jsx)("path", {
                            fill: "url(#0d1e0f__d)",
                            d: "M-.504 16.242c-.667-14.356 10.43-26.535 24.787-27.201S50.818-.53 51.484 13.828s-10.43 26.535-24.787 27.201S.162 30.6-.504 16.242"
                        }),
                        /*#__PURE__*/ (0, $jNyEu$jsx)("path", {
                            fill: "url(#0d1e0f__e)",
                            d: "M13.726-25.508c-3.238-2.004-7.342-2.016-10.592-.032L-29.378-5.689c-3.184 1.944-3.19 6.533-.01 8.485L2.91 22.618c3.246 1.992 7.35 1.99 10.592-.007l32.23-19.84c3.164-1.948 3.17-6.511.01-8.466z"
                        }),
                        /*#__PURE__*/ (0, $jNyEu$jsx)("path", {
                            fill: "url(#0d1e0f__f)",
                            d: "M-17.471 23.1c6.701 0 12.771 2.711 17.166 7.087 8.79 8.766 16.077 8.773 24.867.022 4.394-4.398 10.472-7.117 17.18-7.117 13.41.007 24.274 10.84 24.274 24.214s-10.864 24.2-24.273 24.2c-6.724 0-12.802-2.727-17.197-7.125-8.782-8.736-16.061-8.728-24.851.037-4.395 4.383-10.465 7.088-17.166 7.088-13.402 0-24.266-10.834-24.266-24.2S-30.873 23.1-17.47 23.1"
                        })
                    ]
                })
            })
        ]
    });
const $6860d7daeee01f03$var$ForwardRef = /*#__PURE__*/ (0, $jNyEu$forwardRef)($6860d7daeee01f03$var$SvgComponent);
var $6860d7daeee01f03$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6860d7daeee01f03$var$ForwardRef);


export {$6860d7daeee01f03$export$2e2bcd8739ae039 as default};
