// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_chatbubble_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $iUfdQ$jsxs, jsx as $iUfdQ$jsx} from "react/jsx-runtime";
import {forwardRef as $iUfdQ$forwardRef} from "react";





const $cff38676cb9e0d6c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iUfdQ$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iUfdQ$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iUfdQ$jsxs)("linearGradient", {
                        id: "b55145__d",
                        x1: 66.664,
                        x2: 18.316,
                        y1: 3.573,
                        y2: 66.258,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iUfdQ$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $iUfdQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iUfdQ$jsxs)("linearGradient", {
                        id: "b55145__e",
                        x1: 37.208,
                        x2: 46.535,
                        y1: -25.284,
                        y2: 34.407,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iUfdQ$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $iUfdQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iUfdQ$jsxs)("linearGradient", {
                        id: "b55145__f",
                        x1: -70.407,
                        x2: 86.52,
                        y1: 94.721,
                        y2: 63.707,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iUfdQ$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iUfdQ$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $iUfdQ$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $iUfdQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iUfdQ$jsxs)("radialGradient", {
                        id: "b55145__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 9.766 70.908)scale(120.537)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iUfdQ$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iUfdQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iUfdQ$jsx)("clipPath", {
                        id: "b55145__b",
                        children: /*#__PURE__*/ (0, $iUfdQ$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iUfdQ$jsx)("clipPath", {
                id: "b55145__a",
                children: /*#__PURE__*/ (0, $iUfdQ$jsx)("path", {
                    fill: "#fff",
                    d: "M71.781 21.563H24.214c-4.148 0-7.51 3.137-7.51 7.006v24.865c0 3.87 3.362 7.007 7.51 7.007h6.727c.014 3.306-.014 8.038-.042 11.641-.015 2.01 2.516 3.097 4.144 1.78l16.596-13.42H71.78c4.149 0 7.511-3.138 7.511-7.008V28.57c0-3.87-3.362-7.006-7.51-7.006"
                })
            }),
            /*#__PURE__*/ (0, $iUfdQ$jsx)("g", {
                clipPath: "url(#b55145__a)",
                children: /*#__PURE__*/ (0, $iUfdQ$jsxs)("g", {
                    clipPath: "url(#b55145__b)",
                    children: [
                        /*#__PURE__*/ (0, $iUfdQ$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $iUfdQ$jsx)("circle", {
                            cx: 80.674,
                            cy: 61.143,
                            r: 120.537,
                            fill: "url(#b55145__c)"
                        }),
                        /*#__PURE__*/ (0, $iUfdQ$jsx)("path", {
                            fill: "url(#b55145__d)",
                            d: "M-1.255 32.406C-2.595 3.55 19.71-20.928 48.567-22.268s53.335 20.966 54.675 49.822S82.275 80.89 53.419 82.23.085 61.263-1.255 32.406"
                        }),
                        /*#__PURE__*/ (0, $iUfdQ$jsx)("path", {
                            fill: "url(#b55145__e)",
                            d: "M27.349-51.51c-6.51-4.029-14.757-4.053-21.291-.064l-65.35 39.9c-6.4 3.908-6.41 13.132-.018 17.054L5.607 45.224c6.524 4.003 14.771 3.998 21.29-.015L91.68 5.33c6.358-3.915 6.37-13.087.02-17.016z"
                        }),
                        /*#__PURE__*/ (0, $iUfdQ$jsx)("path", {
                            fill: "url(#b55145__f)",
                            d: "M-35.36 46.189c13.47 0 25.671 5.451 34.505 14.246 17.667 17.619 32.314 17.634 49.981.045 8.834-8.84 21.05-14.306 34.535-14.306 26.953.015 48.788 21.79 48.788 48.671s-21.835 48.641-48.789 48.641c-13.514 0-25.73-5.482-34.564-14.321-17.652-17.559-32.284-17.544-49.951.075-8.834 8.81-21.035 14.246-34.504 14.246-26.939 0-48.774-21.775-48.774-48.641 0-26.865 21.835-48.656 48.774-48.656"
                        })
                    ]
                })
            })
        ]
    });
const $cff38676cb9e0d6c$var$ForwardRef = /*#__PURE__*/ (0, $iUfdQ$forwardRef)($cff38676cb9e0d6c$var$SvgComponent);
var $cff38676cb9e0d6c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cff38676cb9e0d6c$var$ForwardRef);


export {$cff38676cb9e0d6c$export$2e2bcd8739ae039 as default};
