// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_filters_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $cfmW7$reactjsxruntime = require("react/jsx-runtime");
var $cfmW7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6eaad014217e2609$export$2e2bcd8739ae039);




const $6eaad014217e2609$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8b3455__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8b3455__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8b3455__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8b3455__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("clipPath", {
                        id: "8b3455__b",
                        children: /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("clipPath", {
                id: "8b3455__a",
                children: /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M6 28.95c0-3.263 1.353-6.21 3.529-8.31.498 4.419 3.045 8.217 6.669 10.422.535 3.552 2.4 6.669 5.072 8.826-1.167.397-2.418.612-3.72.612C11.171 40.5 6 35.33 6 28.95m20.37 10.938c1.167.397 2.418.612 3.72.612 6.379 0 11.55-5.17 11.55-11.55 0-3.557-1.608-6.74-4.138-8.858-.319 4.348-2.617 8.147-5.997 10.499-.436 3.747-2.346 7.045-5.135 9.297m8.15-24.275c-1.463-4.7-5.848-8.113-11.03-8.113-5.28 0-9.733 3.544-11.11 8.382 1.6-.633 3.344-.982 5.17-.982 2.253 0 4.383.53 6.27 1.474 1.887-.943 4.017-1.474 6.27-1.474 1.548 0 3.037.25 4.43.713m-22.578 3.238-.002.199c0 3.558 1.609 6.74 4.138 8.859.293-3.995 2.255-7.525 5.192-9.896-1.167-.397-2.418-.613-3.72-.613-2.035 0-3.948.527-5.608 1.451M23.82 38.652c2.32-1.502 4.066-3.813 4.84-6.534-1.6.634-3.344.982-5.17.982-1.548 0-3.037-.25-4.43-.713.81 2.604 2.518 4.813 4.76 6.265M35.04 19.05c0 3.263-1.353 6.21-3.529 8.31-.424-3.768-2.34-7.085-5.142-9.347 1.168-.397 2.42-.613 3.721-.613 1.767 0 3.44.397 4.937 1.106q.013.27.013.544m-5.94 9.9c0-4.067-2.103-7.643-5.28-9.701-3.178 2.058-5.28 5.634-5.28 9.701q0 .274.013.544c1.497.71 3.17 1.106 4.937 1.106 2.035 0 3.948-.526 5.608-1.45z"
                })
            }),
            /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("g", {
                clipPath: "url(#8b3455__a)",
                children: /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#8b3455__b)",
                    children: [
                        /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#8b3455__c)"
                        }),
                        /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("path", {
                            fill: "url(#8b3455__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("path", {
                            fill: "url(#8b3455__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $cfmW7$reactjsxruntime.jsx)("path", {
                            fill: "url(#8b3455__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $6eaad014217e2609$var$ForwardRef = /*#__PURE__*/ (0, $cfmW7$react.forwardRef)($6eaad014217e2609$var$SvgComponent);
var $6eaad014217e2609$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6eaad014217e2609$var$ForwardRef);
