// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_thumbsUp_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $gDWlX$jsxs, jsx as $gDWlX$jsx} from "react/jsx-runtime";
import {forwardRef as $gDWlX$forwardRef} from "react";





const $e7b0e8e074a3ac17$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gDWlX$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gDWlX$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gDWlX$jsxs)("linearGradient", {
                        id: "8d7fac__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gDWlX$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $gDWlX$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gDWlX$jsxs)("linearGradient", {
                        id: "8d7fac__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gDWlX$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $gDWlX$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gDWlX$jsxs)("linearGradient", {
                        id: "8d7fac__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gDWlX$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gDWlX$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $gDWlX$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $gDWlX$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gDWlX$jsxs)("radialGradient", {
                        id: "8d7fac__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gDWlX$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gDWlX$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gDWlX$jsx)("clipPath", {
                        id: "8d7fac__b",
                        children: /*#__PURE__*/ (0, $gDWlX$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gDWlX$jsx)("clipPath", {
                id: "8d7fac__a",
                children: /*#__PURE__*/ (0, $gDWlX$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M63.513 129.99c5.48 2.04 11.27 3.09 17.12 3.09l-.02-.02h24.36c.05 0 .15-.01.15-.01h6.75c2.5 0 5.09-.69 7.09-2.22 2.03-1.56 3.38-3.95 3.38-7.17 0-1.04-.15-2.88-1.04-4.68-.42-.87-1.02-1.72-1.84-2.46 4.17-.24 7.69-3.26 8.54-7.39.77-3.75-.84-7.35-3.68-9.39 3.87-.96 6.8-4.36 7.01-8.51.17-3.36-1.47-6.36-4.05-8.12 3.66-1.1 6.38-4.4 6.59-8.4.27-5.29-3.95-9.73-9.25-9.73h-27.72c-1.51 0-2.59-1.47-2.14-2.91l2-6.42c1.79-5.74 2.93-11.66 3.42-17.65l.28-3.47c.42-5.17-3.67-9.61-8.87-9.61-3.43 0-6.55 1.98-8.03 5.07l-9.31 19.5c-.87 1.82-1.87 3.57-2.99 5.24l-.23.35c-5.28 7.856-13.553 13.07-22.745 14.543v55.209c2.197.475 4.362 1.109 6.475 1.898zm-23.225-6.18V70.05h-3.175c-6.08 0-11 4.92-11 11v31.72c0 6.08 4.92 11 11 11h1.23q.975 0 1.945.04"
                })
            }),
            /*#__PURE__*/ (0, $gDWlX$jsx)("g", {
                clipPath: "url(#8d7fac__a)",
                children: /*#__PURE__*/ (0, $gDWlX$jsxs)("g", {
                    clipPath: "url(#8d7fac__b)",
                    children: [
                        /*#__PURE__*/ (0, $gDWlX$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $gDWlX$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#8d7fac__c)"
                        }),
                        /*#__PURE__*/ (0, $gDWlX$jsx)("path", {
                            fill: "url(#8d7fac__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $gDWlX$jsx)("path", {
                            fill: "url(#8d7fac__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $gDWlX$jsx)("path", {
                            fill: "url(#8d7fac__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $e7b0e8e074a3ac17$var$ForwardRef = /*#__PURE__*/ (0, $gDWlX$forwardRef)($e7b0e8e074a3ac17$var$SvgComponent);
var $e7b0e8e074a3ac17$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e7b0e8e074a3ac17$var$ForwardRef);


export {$e7b0e8e074a3ac17$export$2e2bcd8739ae039 as default};
