// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fileZip_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $02E67$reactjsxruntime = require("react/jsx-runtime");
var $02E67$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7bb49cad6d31f7e1$export$2e2bcd8739ae039);




const $7bb49cad6d31f7e1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $02E67$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6de6e4__d",
                        x1: 65.566,
                        x2: 25.102,
                        y1: 11.636,
                        y2: 67.288,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6de6e4__e",
                        x1: 38.978,
                        x2: 46.479,
                        y1: -12.92,
                        y2: 38.008,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6de6e4__f",
                        x1: -58.165,
                        x2: 82.825,
                        y1: 89.197,
                        y2: 59.638,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6de6e4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 78.212 60.624)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("clipPath", {
                        id: "6de6e4__b",
                        children: /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("clipPath", {
                id: "6de6e4__a",
                children: /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M52.673 18.81 72.449 37.73v.012c1.44 1.368 2.25 3.276 2.25 5.268v28.926c0 4.02-3.246 7.278-7.242 7.278H28.535c-4.002 0-7.242-3.258-7.242-7.278v-47.85c0-4.02 3.246-7.278 7.242-7.278h19.146c1.86 0 3.648.714 4.992 2.004m2.85 23.107c0-1.326-1.074-2.4-2.4-2.4h-3.042V35.52h3.048c1.326 0 2.4-1.074 2.4-2.4s-1.074-2.4-2.4-2.4h-3.048v-1.998c0-1.326-1.074-2.4-2.4-2.4h-4.794c-1.326 0-2.4 1.074-2.4 2.4s1.074 2.4 2.4 2.4h2.394v3.996h-2.394c-1.326 0-2.4 1.074-2.4 2.4s1.074 2.4 2.4 2.4h2.394v3.996h-2.394c-1.326 0-2.4 1.074-2.4 2.4s1.074 2.4 2.4 2.4h2.394v3.996h-1.794c-1.326 0-2.4 1.074-2.4 2.4v12.054c0 2.646 2.154 4.8 4.8 4.8h3.606c2.646 0 4.8-2.154 4.8-4.8V55.11c0-1.326-1.074-2.4-2.4-2.4h-1.818v-8.394h3.048c1.326 0 2.4-1.074 2.4-2.4m-9.636 20.418H49.5v4.824h-3.612z"
                })
            }),
            /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6de6e4__a)",
                children: /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6de6e4__b)",
                    children: [
                        /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("ellipse", {
                            cx: 78.212,
                            cy: 60.624,
                            fill: "url(#6de6e4__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("path", {
                            fill: "url(#6de6e4__d)",
                            d: "M4.256 36.171C3.046 11.616 23.182-9.213 49.23-10.353s48.144 17.84 49.353 42.395S79.658 77.426 53.61 78.566 5.466 60.726 4.256 36.171"
                        }),
                        /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("path", {
                            fill: "url(#6de6e4__e)",
                            d: "M30.078-35.237c-5.876-3.427-13.321-3.448-19.219-.054L-48.13-1.339c-5.777 3.325-5.787 11.174-.017 14.512l58.599 33.904c5.889 3.407 13.334 3.403 19.218-.012L88.148 13.13c5.74-3.33 5.75-11.135.018-14.479z"
                        }),
                        /*#__PURE__*/ (0, $02E67$reactjsxruntime.jsx)("path", {
                            fill: "url(#6de6e4__f)",
                            d: "M-26.528 47.9c12.159 0 23.172 4.638 31.146 12.122 15.948 14.993 29.17 15.005 45.118.038 7.974-7.522 19-12.173 31.173-12.173 24.33.013 44.04 18.542 44.04 41.415 0 22.874-19.71 41.391-44.04 41.391-12.2 0-23.227-4.665-31.2-12.187-15.935-14.941-29.143-14.928-45.09.064-7.975 7.497-18.988 12.123-31.147 12.123-24.317 0-44.026-18.53-44.026-41.39 0-22.861 19.71-41.403 44.026-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $7bb49cad6d31f7e1$var$ForwardRef = /*#__PURE__*/ (0, $02E67$react.forwardRef)($7bb49cad6d31f7e1$var$SvgComponent);
var $7bb49cad6d31f7e1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7bb49cad6d31f7e1$var$ForwardRef);
