// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_gift_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $i7Pvk$reactjsxruntime = require("react/jsx-runtime");
var $i7Pvk$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $197b3ac9df5bf20f$export$2e2bcd8739ae039);




const $197b3ac9df5bf20f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1cb4ae__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1cb4ae__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1cb4ae__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsxs)("radialGradient", {
                        id: "1cb4ae__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("clipPath", {
                        id: "1cb4ae__b",
                        children: /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("clipPath", {
                id: "1cb4ae__a",
                children: /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M46 41.488C46 31.278 54.277 23 64.488 23c5.87 0 11.6 2.736 14.987 7.001C82.862 25.736 87.593 23 93.462 23c10.211 0 18.488 8.277 18.488 18.488 0 3.894-1.204 7.507-3.26 10.487H93.571c5.742-.059 10.379-4.732 10.379-10.487C103.95 35.696 99.254 31 93.462 31c-5.766 0-9.445 4.654-9.487 10.41v10.565l-9-.006v-10.56C74.933 35.655 70.254 31 64.488 31 58.695 31 54 35.696 54 41.488c0 5.755 4.637 10.428 10.379 10.487H49.26C47.204 48.995 46 45.382 46 41.488M24 60c0-4.418 3.582-8 8-8h43v33H32c-4.418 0-8-3.582-8-8zm111 0c0-4.418-3.582-8-8-8H84v33h43c4.418 0 8-3.582 8-8zM84 93h44v25c0 9.941-8.059 18-18 18H84zm-53 0h44v43H49c-9.941 0-18-8.059-18-18z"
                })
            }),
            /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("g", {
                clipPath: "url(#1cb4ae__a)",
                children: /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#1cb4ae__b)",
                    children: [
                        /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#1cb4ae__c)"
                        }),
                        /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("path", {
                            fill: "url(#1cb4ae__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("path", {
                            fill: "url(#1cb4ae__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $i7Pvk$reactjsxruntime.jsx)("path", {
                            fill: "url(#1cb4ae__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $197b3ac9df5bf20f$var$ForwardRef = /*#__PURE__*/ (0, $i7Pvk$react.forwardRef)($197b3ac9df5bf20f$var$SvgComponent);
var $197b3ac9df5bf20f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($197b3ac9df5bf20f$var$ForwardRef);
