// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fileZip_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $eWeW8$reactjsxruntime = require("react/jsx-runtime");
var $eWeW8$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $203b89bcc602abce$export$2e2bcd8739ae039);




const $203b89bcc602abce$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6edf05__d",
                        x1: 109.277,
                        x2: 41.838,
                        y1: 19.392,
                        y2: 112.146,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6edf05__e",
                        x1: 64.961,
                        x2: 77.463,
                        y1: -21.535,
                        y2: 63.345,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6edf05__f",
                        x1: -96.944,
                        x2: 138.04,
                        y1: 148.659,
                        y2: 99.394,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6edf05__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 170.948 -181.344 0 130.355 101.038)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("clipPath", {
                        id: "6edf05__b",
                        children: /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.199,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("clipPath", {
                id: "6edf05__a",
                children: /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m87.792 31.35 32.96 31.53v.02c2.4 2.28 3.75 5.46 3.75 8.78v48.21c0 6.7-5.41 12.13-12.07 12.13h-64.87c-6.67 0-12.07-5.43-12.07-12.13V40.14c0-6.7 5.41-12.13 12.07-12.13h31.91c3.1 0 6.08 1.19 8.32 3.34m4.75 38.51c0-2.21-1.79-4-4-4h-5.07V59.2h5.08c2.21 0 4-1.79 4-4s-1.79-4-4-4h-5.08v-3.33c0-2.21-1.79-4-4-4h-7.99c-2.21 0-4 1.79-4 4s1.79 4 4 4h3.99v6.66h-3.99c-2.21 0-4 1.79-4 4s1.79 4 4 4h3.99v6.66h-3.99c-2.21 0-4 1.79-4 4s1.79 4 4 4h3.99v6.66h-2.99c-2.21 0-4 1.79-4 4v20.09c0 4.41 3.59 8 8 8h6.01c4.41 0 8-3.59 8-8V91.85c0-2.21-1.79-4-4-4h-3.03V73.86h5.08c2.21 0 4-1.79 4-4m-16.06 34.03h6.02v8.04h-6.02z"
                })
            }),
            /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6edf05__a)",
                children: /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6edf05__b)",
                    children: [
                        /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.199,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("ellipse", {
                            cx: 130.355,
                            cy: 101.038,
                            fill: "url(#6edf05__c)",
                            rx: 181.344,
                            ry: 170.948
                        }),
                        /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("path", {
                            fill: "url(#6edf05__d)",
                            d: "M7.095 60.284C5.08 19.359 38.638-15.356 82.051-17.257s80.24 29.735 82.256 70.66c2.016 40.923-31.543 75.639-74.956 77.54-43.413 1.9-80.24-29.735-82.256-70.66"
                        }),
                        /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("path", {
                            fill: "url(#6edf05__e)",
                            d: "M50.129-58.73c-9.794-5.713-22.202-5.748-32.032-.09L-80.218-2.233c-9.629 5.542-9.645 18.623-.03 24.186L17.42 78.46c9.814 5.678 22.223 5.671 32.03-.02l97.464-56.56c9.564-5.551 9.582-18.56.029-24.131z"
                        }),
                        /*#__PURE__*/ (0, $eWeW8$reactjsxruntime.jsx)("path", {
                            fill: "url(#6edf05__f)",
                            d: "M-44.215 79.83c20.264 0 38.62 7.731 51.91 20.204 26.58 24.988 48.615 25.009 75.195.064 13.29-12.537 31.668-20.29 51.955-20.29 40.551.022 73.401 30.904 73.401 69.027s-32.85 68.984-73.401 68.984c-20.332 0-38.71-7.774-52-20.311-26.557-24.903-48.57-24.881-75.15.107-13.29 12.494-31.646 20.204-51.91 20.204-40.529 0-73.378-30.883-73.378-68.984s32.85-69.005 73.378-69.005"
                        })
                    ]
                })
            })
        ]
    });
const $203b89bcc602abce$var$ForwardRef = /*#__PURE__*/ (0, $eWeW8$react.forwardRef)($203b89bcc602abce$var$SvgComponent);
var $203b89bcc602abce$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($203b89bcc602abce$var$ForwardRef);
