// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_upload_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2ix91$jsxs, jsx as $2ix91$jsx} from "react/jsx-runtime";
import {forwardRef as $2ix91$forwardRef} from "react";





const $5f3cfa81272cdebe$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2ix91$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2ix91$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2ix91$jsxs)("linearGradient", {
                        id: "f2f1c8__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ix91$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2ix91$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ix91$jsxs)("linearGradient", {
                        id: "f2f1c8__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ix91$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2ix91$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ix91$jsxs)("linearGradient", {
                        id: "f2f1c8__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ix91$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2ix91$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2ix91$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2ix91$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ix91$jsxs)("radialGradient", {
                        id: "f2f1c8__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ix91$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2ix91$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ix91$jsx)("clipPath", {
                        id: "f2f1c8__b",
                        children: /*#__PURE__*/ (0, $2ix91$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2ix91$jsx)("clipPath", {
                id: "f2f1c8__a",
                children: /*#__PURE__*/ (0, $2ix91$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M17 48.6c1.105 0 2 .895 2 2v17.647c0 2.971 2.385 5.353 5.294 5.353h47.412c2.909 0 5.294-2.382 5.294-5.353V50.6c0-1.105.895-2 2-2s2 .895 2 2v17.647c0 5.15-4.146 9.353-9.294 9.353H24.294C19.146 77.6 15 73.397 15 68.247V50.6c0-1.105.895-2 2-2m31 14c-1.105 0-2-.896-2-2V24.582L35.414 35.168c-.78.781-2.047.781-2.828 0-.781-.78-.781-2.047 0-2.828l14-14c.78-.781 2.047-.781 2.828 0l14 14c.781.78.781 2.047 0 2.828-.78.781-2.047.781-2.828 0L50 24.582V60.6c0 1.104-.895 2-2 2"
                })
            }),
            /*#__PURE__*/ (0, $2ix91$jsx)("g", {
                clipPath: "url(#f2f1c8__a)",
                children: /*#__PURE__*/ (0, $2ix91$jsxs)("g", {
                    clipPath: "url(#f2f1c8__b)",
                    children: [
                        /*#__PURE__*/ (0, $2ix91$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2ix91$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#f2f1c8__c)"
                        }),
                        /*#__PURE__*/ (0, $2ix91$jsx)("path", {
                            fill: "url(#f2f1c8__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $2ix91$jsx)("path", {
                            fill: "url(#f2f1c8__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $2ix91$jsx)("path", {
                            fill: "url(#f2f1c8__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $5f3cfa81272cdebe$var$ForwardRef = /*#__PURE__*/ (0, $2ix91$forwardRef)($5f3cfa81272cdebe$var$SvgComponent);
var $5f3cfa81272cdebe$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5f3cfa81272cdebe$var$ForwardRef);


export {$5f3cfa81272cdebe$export$2e2bcd8739ae039 as default};
