// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_noElements_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4Y9Us$reactjsxruntime = require("react/jsx-runtime");
var $4Y9Us$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $cfb7ecd6ab970409$export$2e2bcd8739ae039);




const $cfb7ecd6ab970409$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ea619e__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ea619e__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ea619e__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ea619e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("clipPath", {
                        id: "ea619e__b",
                        children: /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsxs)("clipPath", {
                id: "ea619e__a",
                children: [
                    /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("path", {
                        fill: "#D9D9D9",
                        d: "M77.467 10.921c-.633-.194-1.662.38-2.254 1.064L62.076 27.273c-.648.759-.912 1.796-.699 2.786s.872 1.822 1.768 2.223l4.975 2.227-3.338 8.232c-.4.98-.304 2.09.253 2.96.562.884 1.545 1.432 2.563 1.432.857 0 1.657-.374 2.245-1.053l13.07-15.083c.659-.759.922-1.807.71-2.802s-.877-1.828-1.784-2.222l-5.005-2.202 3.45-8.442c.4-.985.309-2.09-.248-2.965-.563-.885-1.546-1.438-2.564-1.438zM10.27 56c0-1.105.896-2 2-2h28.147c1.105 0 2 .895 2 2v5.63c0 1.104-.895 2-2 2s-2-.896-2-2V58H28.343v24.147h3.63c1.104 0 2 .895 2 2s-.896 2-2 2h-11.26c-1.104 0-2-.896-2-2s.896-2 2-2h3.63V58H14.27v3.63c0 1.104-.896 2-2 2s-2-.896-2-2z"
                    }),
                    /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("path", {
                        fill: "#D9D9D9",
                        fillRule: "evenodd",
                        d: "M53.924 76.688V64c0-4.418 3.582-8 8-8h21.763c4.419 0 8 3.582 8 8v12.688c0 4.418-3.581 8-8 8H61.924c-4.418 0-8-3.582-8-8m24.128-7.43-7.378-4.21c-.826-.472-1.86.115-1.861 1.056l-.01 8.406c-.001.941 1.031 1.53 1.859 1.061l7.388-4.194c.827-.47.829-1.646.002-2.118M7 34.75V22c0-4.418 3.582-8 8-8h22.125c4.418 0 8 3.582 8 8v12.75c0 1.422-.37 2.757-1.021 3.914l-4.218-6.582c-2.361-3.683-7.743-3.683-10.103 0l-1.356 2.115-3.832-6.04c-2.348-3.702-7.745-3.717-10.114-.029l-6.6 10.274C7.318 37.307 7 36.066 7 34.75M38 23.5c1.38 0 2.5-1.12 2.5-2.5s-1.12-2.5-2.5-2.5-2.5 1.12-2.5 2.5 1.12 2.5 2.5 2.5"
                    }),
                    /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("path", {
                        fill: "#D9D9D9",
                        d: "M26.046 37.912 21.218 30.3c-.783-1.234-2.582-1.24-3.372-.01l-7.184 11.183c1.25.808 2.74 1.277 4.338 1.277h7.945zM27.696 42.75h9.429c1.506 0 2.914-.416 4.117-1.14l-4.724-7.37c-.787-1.227-2.58-1.227-3.368 0z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ea619e__a)",
                children: /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ea619e__b)",
                    children: [
                        /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#ea619e__c)"
                        }),
                        /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("path", {
                            fill: "url(#ea619e__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("path", {
                            fill: "url(#ea619e__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $4Y9Us$reactjsxruntime.jsx)("path", {
                            fill: "url(#ea619e__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $cfb7ecd6ab970409$var$ForwardRef = /*#__PURE__*/ (0, $4Y9Us$react.forwardRef)($cfb7ecd6ab970409$var$SvgComponent);
var $cfb7ecd6ab970409$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($cfb7ecd6ab970409$var$ForwardRef);
