// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_confetti_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3OpSE$reactjsxruntime = require("react/jsx-runtime");
var $3OpSE$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f912d325b73ab7ec$export$2e2bcd8739ae039);




const $f912d325b73ab7ec$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f51283__d",
                        x1: 32.926,
                        x2: 10.021,
                        y1: 4.108,
                        y2: 35.578,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f51283__e",
                        x1: 17.893,
                        x2: 22.142,
                        y1: -9.787,
                        y2: 19.031,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f51283__f",
                        x1: -37.02,
                        x2: 42.687,
                        y1: 47.998,
                        y2: 31.304,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f51283__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 58.0415 -61.5067 0 40.073 31.829)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("clipPath", {
                        id: "f51283__b",
                        children: /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("rect", {
                            width: 42.6,
                            height: 40.2,
                            x: 2.102,
                            y: 5.4,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("clipPath", {
                id: "f51283__a",
                children: /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M32.978 9.648c.858-.858.858-2.244 0-3.102s-2.245-.858-3.103 0-.858 2.244 0 3.102 2.245.858 3.102 0M19.44 21.636c-.519.414-.603 1.17-.189 1.686.237.297.585.45.94.45.263 0 .527-.087.746-.264 5.766-4.605 5.502-9.438 5.49-9.642-.045-.657-.609-1.146-1.266-1.11-.657.039-1.16.606-1.128 1.263.006.156.13 3.843-4.593 7.614zm10.2 8.96c-.308-.584-.083-1.31.502-1.62h.003c.585-.308 1.31-.083 1.62.502 1.719 3.258 1.167 4.854.402 5.619-.25.249-.585.474-1.035.606l-20.775 7.38c-.402.144-.813.213-1.218.213-.933 0-1.842-.37-2.532-1.06-.99-.99-1.314-2.426-.846-3.75l7.296-20.558q.062-.234.165-.45c.135-.336.32-.6.525-.804.77-.771 2.382-1.32 5.679.435.585.312.807 1.038.495 1.623s-1.038.807-1.623.495c-1.857-.987-2.71-.927-2.856-.852v.006l-.054.153c-.12.912 1.26 4.46 5.859 9.06 3.486 3.486 6.366 5.12 7.962 5.652.618.126 1.056.156 1.278.126.057-.165.087-1.005-.846-2.775m-6.393-3.767c-.309 0-.615-.117-.849-.351-.468-.471-.468-1.23 0-1.698l4.683-4.683c.471-.468 1.23-.468 1.698 0 .468.47.468 1.23 0 1.698l-4.683 4.683c-.234.234-.54.35-.849.35m2.115 1.11c4.605-5.766 9.438-5.502 9.642-5.49l-.003.003c.663.045 1.161.618 1.116 1.278-.045.657-.612 1.158-1.269 1.116-.177-.006-3.852-.111-7.61 4.593-.238.297-.589.45-.94.45-.26 0-.525-.087-.747-.264-.519-.414-.603-1.167-.189-1.686m12.15-.288c.858-.858 2.244-.858 3.102 0s.858 2.244 0 3.102-2.244.858-3.102 0-.858-2.244 0-3.102M16.907 8.458l-.576.422c-.535.392-.651 1.142-.26 1.677l2.344 3.203c.392.535 1.142.651 1.677.26l.576-.422c.535-.391.651-1.142.26-1.677l-2.344-3.203c-.392-.535-1.142-.651-1.677-.26m15.263 7.997 3.944-.446c.658-.075 1.253.399 1.327 1.057l.08.71c.075.658-.398 1.252-1.057 1.327l-3.944.446c-.658.075-1.253-.398-1.327-1.057l-.08-.71c-.075-.658.398-1.252 1.057-1.327"
                })
            }),
            /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("g", {
                clipPath: "url(#f51283__a)",
                children: /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#f51283__b)",
                    children: [
                        /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("rect", {
                            width: 42.6,
                            height: 40.2,
                            x: 2.102,
                            y: 5.4,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("ellipse", {
                            cx: 40.073,
                            cy: 31.829,
                            fill: "url(#f51283__c)",
                            rx: 61.507,
                            ry: 58.041
                        }),
                        /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("path", {
                            fill: "url(#f51283__d)",
                            d: "M-1.732 17.992C-2.415 4.097 8.967-7.69 23.691-8.335c14.725-.645 27.215 10.096 27.9 23.99.683 13.895-10.7 25.682-25.424 26.328-14.724.645-27.215-10.096-27.899-23.991"
                        }),
                        /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("path", {
                            fill: "url(#f51283__e)",
                            d: "M12.862-22.416c-3.322-1.94-7.53-1.951-10.864-.03L-31.348-3.235c-3.266 1.882-3.271 6.323-.01 8.212L1.767 24.164c3.33 1.928 7.538 1.925 10.864-.007L45.688 4.953c3.244-1.884 3.25-6.301.01-8.193z"
                        }),
                        /*#__PURE__*/ (0, $3OpSE$reactjsxruntime.jsx)("path", {
                            fill: "url(#f51283__f)",
                            d: "M-19.136 24.628c6.873 0 13.1 2.625 17.607 6.86 9.015 8.484 16.489 8.491 25.504.022 4.508-4.257 10.741-6.889 17.622-6.889 13.753.007 24.895 10.493 24.895 23.436 0 12.944-11.142 23.422-24.895 23.422-6.896 0-13.13-2.64-17.638-6.896-9.007-8.455-16.473-8.448-25.488.037-4.508 4.242-10.734 6.86-17.607 6.86-13.746 0-24.887-10.486-24.887-23.423s11.141-23.429 24.887-23.429"
                        })
                    ]
                })
            })
        ]
    });
const $f912d325b73ab7ec$var$ForwardRef = /*#__PURE__*/ (0, $3OpSE$react.forwardRef)($f912d325b73ab7ec$var$SvgComponent);
var $f912d325b73ab7ec$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f912d325b73ab7ec$var$ForwardRef);
