// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_paperclip_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $eWGS6$jsxs, jsx as $eWGS6$jsx} from "react/jsx-runtime";
import {forwardRef as $eWGS6$forwardRef} from "react";





const $a796f13f80a4696c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eWGS6$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $eWGS6$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $eWGS6$jsxs)("linearGradient", {
                        id: "0375f7__d",
                        x1: 32.783,
                        x2: 12.551,
                        y1: 5.068,
                        y2: 32.894,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eWGS6$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $eWGS6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eWGS6$jsxs)("linearGradient", {
                        id: "0375f7__e",
                        x1: 19.489,
                        x2: 23.239,
                        y1: -7.21,
                        y2: 18.254,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eWGS6$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $eWGS6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eWGS6$jsxs)("linearGradient", {
                        id: "0375f7__f",
                        x1: -29.084,
                        x2: 41.411,
                        y1: 43.849,
                        y2: 29.069,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eWGS6$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $eWGS6$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $eWGS6$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $eWGS6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eWGS6$jsxs)("radialGradient", {
                        id: "0375f7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.2844 -54.4031 0 39.106 29.562)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eWGS6$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $eWGS6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eWGS6$jsx)("clipPath", {
                        id: "0375f7__b",
                        children: /*#__PURE__*/ (0, $eWGS6$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.21,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $eWGS6$jsx)("clipPath", {
                id: "0375f7__a",
                children: /*#__PURE__*/ (0, $eWGS6$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    stroke: "#fff",
                    strokeLinecap: "round",
                    strokeLinejoin: "round",
                    strokeWidth: 0.6,
                    d: "M10.608 26.865c-3.342 3.325-3.342 8.719 0 12.043 3.338 3.322 8.747 3.322 12.086 0l13.52-13.454c.47-.467.472-1.227.004-1.697s-1.227-.471-1.697-.004l-13.52 13.454c-2.402 2.39-6.299 2.39-8.7 0-2.4-2.387-2.4-6.254 0-8.64L30.817 10.14c1.529-1.522 4.01-1.522 5.54 0 1.525 1.518 1.525 3.977 0 5.495L19.76 32.152c-.639.636-1.676.636-2.315 0-.635-.632-.635-1.654 0-2.286l11.601-11.544c.47-.468.472-1.227.004-1.697s-1.227-.472-1.697-.004l-11.6 11.544c-1.579 1.57-1.579 4.118 0 5.688 1.574 1.567 4.125 1.567 5.7 0L38.05 17.338c2.469-2.457 2.469-6.442 0-8.898-2.465-2.454-6.46-2.454-8.925 0z"
                })
            }),
            /*#__PURE__*/ (0, $eWGS6$jsx)("g", {
                clipPath: "url(#0375f7__a)",
                children: /*#__PURE__*/ (0, $eWGS6$jsxs)("g", {
                    clipPath: "url(#0375f7__b)",
                    children: [
                        /*#__PURE__*/ (0, $eWGS6$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.21,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $eWGS6$jsx)("ellipse", {
                            cx: 39.106,
                            cy: 29.562,
                            fill: "url(#0375f7__c)",
                            rx: 54.403,
                            ry: 51.284
                        }),
                        /*#__PURE__*/ (0, $eWGS6$jsx)("path", {
                            fill: "url(#0375f7__d)",
                            d: "M2.128 17.336C1.523 5.057 11.591-5.358 24.615-5.928s24.072 8.92 24.677 21.198c.605 12.277-9.463 22.692-22.487 23.262S2.733 29.613 2.128 17.336"
                        }),
                        /*#__PURE__*/ (0, $eWGS6$jsx)("path", {
                            fill: "url(#0375f7__e)",
                            d: "M15.039-18.368c-2.938-1.714-6.66-1.725-9.61-.027L-24.064-1.42c-2.889 1.662-2.893 5.587-.009 7.256l29.3 16.952c2.944 1.703 6.667 1.7 9.609-.007l29.24-16.967c2.869-1.666 2.874-5.568.008-7.24z"
                        }),
                        /*#__PURE__*/ (0, $eWGS6$jsx)("path", {
                            fill: "url(#0375f7__f)",
                            d: "M-13.266 23.2c6.08 0 11.586 2.32 15.573 6.06 7.974 7.497 14.585 7.504 22.559.02 3.987-3.76 9.5-6.087 15.586-6.087 12.166.007 22.02 9.272 22.02 20.708 0 11.437-9.854 20.695-22.02 20.695-6.1 0-11.613-2.332-15.6-6.093-7.967-7.47-14.57-7.464-22.545.032-3.987 3.748-9.494 6.061-15.573 6.061-12.158 0-22.013-9.264-22.013-20.695 0-11.43 9.855-20.701 22.013-20.701"
                        })
                    ]
                })
            })
        ]
    });
const $a796f13f80a4696c$var$ForwardRef = /*#__PURE__*/ (0, $eWGS6$forwardRef)($a796f13f80a4696c$var$SvgComponent);
var $a796f13f80a4696c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a796f13f80a4696c$var$ForwardRef);


export {$a796f13f80a4696c$export$2e2bcd8739ae039 as default};
