// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_trophy_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $5beOJ$jsxs, jsx as $5beOJ$jsx} from "react/jsx-runtime";
import {forwardRef as $5beOJ$forwardRef} from "react";





const $de5f83b17fe9a8bb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5beOJ$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5beOJ$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5beOJ$jsxs)("linearGradient", {
                        id: "79a529__d",
                        x1: 108.785,
                        x2: 38.447,
                        y1: 14.029,
                        y2: 110.771,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5beOJ$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $5beOJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5beOJ$jsxs)("linearGradient", {
                        id: "79a529__e",
                        x1: 62.565,
                        x2: 75.605,
                        y1: -28.658,
                        y2: 59.871,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5beOJ$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $5beOJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5beOJ$jsxs)("linearGradient", {
                        id: "79a529__f",
                        x1: -106.301,
                        x2: 138.787,
                        y1: 148.856,
                        y2: 97.472,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5beOJ$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5beOJ$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $5beOJ$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $5beOJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5beOJ$jsxs)("radialGradient", {
                        id: "79a529__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 178.298 -189.141 0 130.774 99.185)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5beOJ$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5beOJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5beOJ$jsx)("clipPath", {
                        id: "79a529__b",
                        children: /*#__PURE__*/ (0, $5beOJ$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 18,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5beOJ$jsx)("clipPath", {
                id: "79a529__a",
                children: /*#__PURE__*/ (0, $5beOJ$jsx)("path", {
                    fill: "#fff",
                    d: "M126.04 40H108v-6c0-3.31-2.69-6-6-6H58c-3.31 0-6 2.69-6 6v6H34c-5.51 0-10 4.49-10 10v2c0 15.44 12.56 28 28 28h5.03c4.34 6.22 11.15 10.6 19 11.71 0 .1-.03.19-.03.29v24.42c-7.17 1.35-13.07 6.24-15.84 12.79-.56 1.32.42 2.78 1.85 2.78h35.98c1.43 0 2.41-1.46 1.85-2.78-2.77-6.55-8.67-11.44-15.84-12.79V92c0-.1-.02-.19-.03-.29 7.86-1.12 14.66-5.49 19-11.71h5.06c15.44 0 28-12.56 28-28v-2c0-5.51-4.49-10-10-10zM52 72c-11.03 0-20-8.97-20-20v-2c0-1.1.9-2 2-2h18v16c0 2.78.41 5.46 1.17 8zm40.79-14.83-3.73 3.62c-.51.5-.75 1.22-.62 1.92l.89 5.12c.34 1.98-1.74 3.5-3.52 2.56l-4.6-2.43c-.63-.33-1.39-.33-2.02 0l-4.6 2.43c-1.78.94-3.86-.57-3.52-2.56l.89-5.12c.12-.7-.11-1.42-.62-1.92l-3.73-3.62c-1.44-1.4-.65-3.85 1.34-4.14l5.15-.74c.71-.1 1.32-.55 1.64-1.19l2.29-4.67c.89-1.81 3.46-1.81 4.35 0l2.29 4.67c.32.64.93 1.09 1.64 1.19l5.15.74c1.99.29 2.79 2.74 1.34 4.14zM128.04 52c0 11.03-8.97 20-20 20h-1.2c.75-2.54 1.17-5.22 1.17-8V48h18.04c1.1 0 2 .9 2 2v2z"
                })
            }),
            /*#__PURE__*/ (0, $5beOJ$jsx)("g", {
                clipPath: "url(#79a529__a)",
                children: /*#__PURE__*/ (0, $5beOJ$jsxs)("g", {
                    clipPath: "url(#79a529__b)",
                    children: [
                        /*#__PURE__*/ (0, $5beOJ$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 18,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $5beOJ$jsx)("ellipse", {
                            cx: 130.774,
                            cy: 99.185,
                            fill: "url(#79a529__c)",
                            rx: 189.141,
                            ry: 178.298
                        }),
                        /*#__PURE__*/ (0, $5beOJ$jsx)("path", {
                            fill: "url(#79a529__d)",
                            d: "M2.21 56.679C.107 13.995 35.11-22.214 80.389-24.196s83.69 31.013 85.792 73.697-32.899 78.893-78.178 80.875c-45.28 1.982-83.69-31.014-85.793-73.697"
                        }),
                        /*#__PURE__*/ (0, $5beOJ$jsx)("path", {
                            fill: "url(#79a529__e)",
                            d: "M47.095-67.453c-10.215-5.958-23.156-5.995-33.409-.094L-88.856-8.527c-10.043 5.78-10.06 19.424-.03 25.227L12.977 75.636c10.237 5.922 23.179 5.915 33.407-.021L148.04 16.623c9.976-5.79 9.994-19.357.031-25.169z"
                        }),
                        /*#__PURE__*/ (0, $5beOJ$jsx)("path", {
                            fill: "url(#79a529__f)",
                            d: "M-51.305 77.067c21.136 0 40.28 8.064 54.142 21.073 27.723 26.062 50.706 26.084 78.429.067 13.861-13.076 33.03-21.162 54.189-21.162 42.295.022 76.557 32.233 76.557 71.994s-34.262 71.95-76.557 71.95c-21.207 0-40.375-8.108-54.237-21.184-27.699-25.973-50.658-25.951-78.38.111-13.862 13.031-33.007 21.073-54.143 21.073-42.27 0-76.533-32.21-76.533-71.95 0-39.739 34.262-71.972 76.533-71.972"
                        })
                    ]
                })
            })
        ]
    });
const $de5f83b17fe9a8bb$var$ForwardRef = /*#__PURE__*/ (0, $5beOJ$forwardRef)($de5f83b17fe9a8bb$var$SvgComponent);
var $de5f83b17fe9a8bb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($de5f83b17fe9a8bb$var$ForwardRef);


export {$de5f83b17fe9a8bb$export$2e2bcd8739ae039 as default};
