// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_magicWand_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $aQ37k$reactjsxruntime = require("react/jsx-runtime");
var $aQ37k$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ac53e11234fb53a3$export$2e2bcd8739ae039);




const $ac53e11234fb53a3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsxs)("linearGradient", {
                        id: "fd120b__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsxs)("linearGradient", {
                        id: "fd120b__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsxs)("linearGradient", {
                        id: "fd120b__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsxs)("radialGradient", {
                        id: "fd120b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("clipPath", {
                        id: "fd120b__b",
                        children: /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("clipPath", {
                id: "fd120b__a",
                children: /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "m79.614 26.756-4.233.845c-.56.113-1.025.49-1.252 1.013l-1.707 3.963c-.673 1.562-2.957 1.32-3.289-.344L68.287 28c-.113-.56-.49-1.025-1.012-1.252l-3.963-1.707c-1.562-.673-1.32-2.957.344-3.289l4.233-.846c.56-.112 1.025-.49 1.251-1.012l1.708-3.963c.673-1.562 2.957-1.32 3.288.344l.846 4.233c.113.56.49 1.025 1.013 1.251l3.963 1.708c1.562.673 1.32 2.957-.344 3.289M43.901 30.38l-3.496.698c-.463.094-.847.406-1.034.837l-1.41 3.273c-.556 1.29-2.443 1.09-2.716-.284l-.7-3.496c-.093-.463-.404-.847-.835-1.034l-3.274-1.41c-1.29-.556-1.09-2.443.284-2.717l3.497-.698c.462-.094.846-.405 1.033-.836l1.41-3.274c.557-1.29 2.443-1.09 2.717.284l.699 3.497c.093.462.405.846.835 1.033l3.274 1.41c1.29.557 1.09 2.443-.284 2.717m31.034 25.917 3.496-.699c1.375-.273 1.575-2.16.284-2.715l-3.273-1.411c-.431-.187-.743-.571-.836-1.034l-.7-3.496c-.273-1.375-2.16-1.575-2.715-.284l-1.41 3.273c-.188.431-.572.743-1.034.836l-3.497.7c-1.374.273-1.574 2.16-.284 2.715l3.274 1.41c.43.188.743.572.836 1.034l.699 3.497c.273 1.374 2.16 1.574 2.715.284l1.411-3.274c.187-.43.571-.742 1.034-.836m-21.612-21.67c2.254-2.254 5.91-2.254 8.164 0s2.254 5.91 0 8.163l-6.058 6.058-8.164-8.163zm-8.886 8.886 8.163 8.164-27.75 27.75c-2.254 2.254-5.909 2.254-8.163 0s-2.255-5.91 0-8.164z"
                })
            }),
            /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("g", {
                clipPath: "url(#fd120b__a)",
                children: /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#fd120b__b)",
                    children: [
                        /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#fd120b__c)"
                        }),
                        /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("path", {
                            fill: "url(#fd120b__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("path", {
                            fill: "url(#fd120b__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $aQ37k$reactjsxruntime.jsx)("path", {
                            fill: "url(#fd120b__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $ac53e11234fb53a3$var$ForwardRef = /*#__PURE__*/ (0, $aQ37k$react.forwardRef)($ac53e11234fb53a3$var$SvgComponent);
var $ac53e11234fb53a3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ac53e11234fb53a3$var$ForwardRef);
