// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_document_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8ZI9O$reactjsxruntime = require("react/jsx-runtime");
var $8ZI9O$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2a9e71f3a84d8b25$export$2e2bcd8739ae039);




const $2a9e71f3a84d8b25$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsxs)("linearGradient", {
                        id: "09b037__d",
                        x1: 33.33,
                        x2: 9.156,
                        y1: 1.787,
                        y2: 33.129,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsxs)("linearGradient", {
                        id: "09b037__e",
                        x1: 18.602,
                        x2: 23.265,
                        y1: -12.642,
                        y2: 17.203,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsxs)("linearGradient", {
                        id: "09b037__f",
                        x1: -35.206,
                        x2: 43.258,
                        y1: 47.361,
                        y2: 31.853,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsxs)("radialGradient", {
                        id: "09b037__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 4.884 35.455)scale(60.2686)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("clipPath", {
                        id: "09b037__b",
                        children: /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsxs)("clipPath", {
                id: "09b037__a",
                children: [
                    /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("path", {
                        fill: "#fff",
                        d: "M24 18.573V8.337q-.145-.014-.292-.015h-9.356c-1.998 0-3.618 1.62-3.618 3.618v24.12c0 1.998 1.62 3.618 3.618 3.618h19.296c1.999 0 3.618-1.62 3.618-3.618V21.88q-.001-.146-.014-.292H27.015c-1.662 0-3.015-1.352-3.015-3.015"
                    }),
                    /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("path", {
                        fill: "#fff",
                        d: "M27.013 19.176h9.045L26.41 9.528v9.045c0 .333.27.603.603.603"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("g", {
                clipPath: "url(#09b037__a)",
                children: /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#09b037__b)",
                    children: [
                        /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("circle", {
                            cx: 40.339,
                            cy: 30.571,
                            r: 60.269,
                            fill: "url(#09b037__c)"
                        }),
                        /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("path", {
                            fill: "url(#09b037__d)",
                            d: "M-.63 16.203C-1.3 1.775 9.854-10.464 24.282-11.134S50.949-.651 51.619 13.777 41.136 40.444 26.708 41.114.04 30.631-.63 16.204"
                        }),
                        /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("path", {
                            fill: "url(#09b037__e)",
                            d: "M13.672-25.755c-3.254-2.014-7.378-2.027-10.645-.032l-32.675 19.95c-3.2 1.954-3.205 6.566-.01 8.527l32.46 19.922c3.261 2.002 7.385 2 10.644-.007l32.392-19.94c3.179-1.958 3.185-6.544.01-8.508z"
                        }),
                        /*#__PURE__*/ (0, $8ZI9O$reactjsxruntime.jsx)("path", {
                            fill: "url(#09b037__f)",
                            d: "M-17.682 23.094c6.735 0 12.836 2.726 17.252 7.123 8.834 8.81 16.158 8.818 24.991.023 4.417-4.42 10.525-7.153 17.267-7.153 13.477.007 24.395 10.895 24.395 24.336s-10.918 24.32-24.395 24.32c-6.757 0-12.865-2.74-17.282-7.16-8.826-8.78-16.142-8.773-24.976.037-4.416 4.405-10.517 7.123-17.252 7.123-13.469 0-24.386-10.888-24.386-24.32s10.917-24.329 24.386-24.329"
                        })
                    ]
                })
            })
        ]
    });
const $2a9e71f3a84d8b25$var$ForwardRef = /*#__PURE__*/ (0, $8ZI9O$react.forwardRef)($2a9e71f3a84d8b25$var$SvgComponent);
var $2a9e71f3a84d8b25$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2a9e71f3a84d8b25$var$ForwardRef);
