// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_AIchat_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $98tJH$reactjsxruntime = require("react/jsx-runtime");
var $98tJH$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5b0dd2a1d7ceb89c$export$2e2bcd8739ae039);




const $5b0dd2a1d7ceb89c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a89f50__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a89f50__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a89f50__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a89f50__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("clipPath", {
                        id: "a89f50__b",
                        children: /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("clipPath", {
                id: "a89f50__a",
                children: /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m106.073 50.901-2.121 9.84c-.94 4.37 4.078 7.393 7.233 4.359l7.107-6.834c1.055-1.016 2.525-1.426 3.925-1.096l9.43 2.214q.156.036.31.061c4.018.66 6.703-4.399 3.867-7.609l-6.55-7.416c-.84-.95-1.248-2.224-1.14-3.494q.015-.163.04-.327.021-.137.051-.274l2.121-9.84c.94-4.37-4.078-7.393-7.233-4.359l-7.107 6.834c-1.055 1.016-2.525 1.426-3.925 1.096l-9.43-2.214c-3.251-.762-5.725 2.225-5.307 5.14.12.839.48 1.672 1.13 2.407l6.55 7.416c.832.942 1.24 2.201 1.143 3.46q-.025.319-.094.636m-16.757-13.92H40.489c-6.76 0-12.239 5.323-12.239 12.04v39.66c0 6.717 5.48 12.04 12.239 12.04h10.424c.024 5.739.046 13.952 0 20.207-.026 3.488 4.1 5.375 6.752 3.09l27.242-23.297h34.811c6.759 0 12.239-5.323 12.239-12.04V67.49q-1.044-.063-2.135-.317l-.003-.001-7.433-1.745-5.655 5.438-.001.001c-4.458 4.287-10.531 4.271-14.826 1.683-4.24-2.554-7.014-7.72-5.773-13.49l1.774-8.229-5.426-6.143-.001-.002c-2.066-2.339-3.034-5.051-3.162-7.704m1.042 47.08-6.93-1.562c-.919-.205-1.87.051-2.56.684L75.647 88c-2.058 1.898-5.33.005-4.714-2.72l1.561-6.93c.206-.918-.05-1.87-.684-2.56l-4.816-5.221c-1.898-2.058-.006-5.33 2.719-4.714l6.931 1.562c.918.205 1.87-.052 2.56-.684l5.22-4.817c2.058-1.898 5.33-.005 4.715 2.72l-1.562 6.93c-.206.918.051 1.87.684 2.56l4.816 5.221c1.898 2.058.006 5.33-2.719 4.714"
                })
            }),
            /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("g", {
                clipPath: "url(#a89f50__a)",
                children: /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#a89f50__b)",
                    children: [
                        /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#a89f50__c)"
                        }),
                        /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("path", {
                            fill: "url(#a89f50__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("path", {
                            fill: "url(#a89f50__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $98tJH$reactjsxruntime.jsx)("path", {
                            fill: "url(#a89f50__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $5b0dd2a1d7ceb89c$var$ForwardRef = /*#__PURE__*/ (0, $98tJH$react.forwardRef)($5b0dd2a1d7ceb89c$var$SvgComponent);
var $5b0dd2a1d7ceb89c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5b0dd2a1d7ceb89c$var$ForwardRef);
