// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_badgeVerified_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4G2Em$reactjsxruntime = require("react/jsx-runtime");
var $4G2Em$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0143f4b5862cc0c1$export$2e2bcd8739ae039);




const $0143f4b5862cc0c1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsxs)("linearGradient", {
                        id: "555e6d__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsxs)("linearGradient", {
                        id: "555e6d__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsxs)("linearGradient", {
                        id: "555e6d__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsxs)("radialGradient", {
                        id: "555e6d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("clipPath", {
                        id: "555e6d__b",
                        children: /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("clipPath", {
                id: "555e6d__a",
                children: /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M75.766 38.628c3.263 1.851 5.475 5.355 5.475 9.376h.009c0 4.038-2.23 7.55-5.519 9.401-1.291.724-1.894 2.196-1.498 3.625.44 1.558.534 3.263.181 5.028-.835 4.21-4.201 7.585-8.412 8.42-1.773.353-3.495.258-5.062-.19-1.403-.395-2.867.199-3.599 1.464-1.868 3.229-5.346 5.407-9.341 5.407s-7.473-2.178-9.341-5.407c-.732-1.265-2.187-1.86-3.6-1.463-1.566.447-3.288.542-5.061.189-4.21-.835-7.577-4.21-8.412-8.42-.353-1.765-.258-3.47.18-5.028.397-1.43-.206-2.901-1.497-3.625-3.29-1.85-5.519-5.38-5.519-9.401 0-4.02 2.204-7.525 5.476-9.376 1.248-.706 1.825-2.178 1.48-3.53-.344-1.352-.439-2.798-.223-4.296.68-4.658 4.442-8.403 9.1-9.074 1.524-.224 2.927-.138 4.356.25 1.43.387 2.893-.233 3.608-1.525 1.834-3.34 5.38-5.613 9.453-5.613s7.62 2.273 9.453 5.613c.715 1.292 2.247 1.886 3.608 1.524 1.36-.361 2.824-.473 4.348-.25 4.666.672 8.42 4.417 9.1 9.075.215 1.498.138 2.91-.224 4.296s.232 2.824 1.48 3.53m-11.981 2.01c.816-.744.875-2.009.13-2.825-.743-.816-2.008-.875-2.825-.131l-3.177 2.896-3.112 2.836-.053.049-.054.048-.053.05-.054.048-.214.196-.434.395-.11.1-.11.1-.109.1-.055.05-.055.05-.055.05-.056.05-.055.051-.055.05-.222.203-.223.203-.111.102-.112.101-.112.102-.056.051-.056.052-.112.102-.113.102-.45.41-.056.052-.056.05-.056.052-.057.051-.225.206-1.79 1.632-.878.8-.054.048-.054.05-.108.098-.107.098-.107.098-.107.097-.053.048-.053.049-.053.048-.053.048-.053.048-.053.048-.052.048-.052.048-.053.048-.052.047-.104.095-.104.094-.051.048-.052.047-.102.093-.102.093-.102.092-.201.184-.1.09-.099.091-.05.045-.049.045-.049.045-.049.044-.048.045-.049.044-.048.044-.048.044-.38.346-.369.336-.045.041-.045.041-.045.04-.044.042-.177.16-.087.08-.043.039-.043.04-.17.154-1.618 1.475-8.13-7.581c-.808-.754-2.074-.71-2.827.098s-.71 2.074.099 2.827l9.478 8.84c.762.71 1.941.717 2.711.015l-1.347-1.478 1.347 1.478h.001l.003-.003.002-.002.002-.002.006-.006.017-.015.005-.005.006-.005.013-.012m19.342-17.629-3.177 2.896Zm-3.177 2.896-.043.04Zm-.043.04-.044.039Zm-.044.039-.043.04Zm-.043.04-.044.04Zm-.044.04-.044.04Zm-.044.04-.044.04Zm-.044.04-.089.08Zm-.089.08-.363.332Zm-.363.332-.047.042Zm-.047.042-.046.042Zm-.046.042-.093.085Zm-.093.085-.095.086Zm-.095.086-.047.043Zm-.047.043-.047.044Zm-.047.044-.048.043Zm-.048.043-.048.044Zm-.048.044-.048.044Zm-.048.044-.048.044Zm-.048.044-.048.044Zm-.048.044-.05.044Zm-.05.044-.097.089Zm-.097.089-.396.36Zm-.396.36-.05.047Zm-.05.047-.05.045Zm-.05.045-.05.046Zm-.05.046-.051.047Zm-.051.047-.102.092Zm-.102.092-.051.047Zm-.051.047-.051.046Zm-.051.046-.206.188Zm-.206.188-.208.19Zm-.208.19-.105.095Zm-.105.095-.053.048Zm-.053.048-.052.048Zm-.052.048-.212.193Zm-.212.193-.862.785Zm-.862.785-.439.4Zm-.439.4-.055.05Zm-.055.05-.055.051Zm-.055.051-.055.05Zm-.055.05-.056.05Zm-.056.05-.11.102Zm-.11.102-.112.101Zm-.112.101-.222.203Zm-.222.203-.224.204Zm-.224.204-.224.204Zm-.224.204-.112.102Zm-.112.102-.056.051Zm-.056.051-.056.052Zm-.056.052-.056.05Zm-.056.05-.056.052Zm-.056.052-.057.051Zm-.057.051-.056.051Zm-.056.051-.056.052Zm-.056.052-.056.05Zm-.056.05-.113.103Zm-.113.103-.056.052Zm-.056.052-.057.051Zm-.057.051-.112.103Zm-.112.103-.226.205Zm-.226.205-1.79 1.633Zm-1.79 1.633-.056.05Zm-.056.05-.055.05Zm-.055.05-.055.05Zm-.055.05-.055.05Zm-.055.05-.055.05Zm-.055.05-.055.051Zm-.055.051-.11.1Zm-.11.1-.219.2Zm-.219.2-.054.049Zm-.054.049-.055.05Zm-.055.05-.108.099Zm-.108.099-.054.049Zm-.054.049-.054.05Zm-.054.05-.054.048Zm-.054.048-.054.05Zm-.054.05-.054.048Zm-.054.048-.054.05Zm-.054.05-.053.048Zm-.053.048-.054.049Zm-.054.049-.106.097Zm-.106.097-.107.097Zm-.107.097-.211.193Zm-.211.193-.417.38Zm-.417.38-.052.047Zm-.052.047-.051.047Zm-.051.047-.103.093Zm-.103.093-.05.047Zm-.05.047-.052.046Zm-.052.046-.05.047Zm-.05.047-.051.046Zm-.051.046-.201.183Zm-.201.183-.1.091Zm-.1.091-.05.045Zm-.05.045-.05.046Zm-.05.046-.196.179Zm-.196.179-.194.176Zm-.194.176-.38.346Zm-.38.346-.368.336Zm-.368.336-.18.164Zm-.18.164-.088.08Zm-.088.08-.044.04Zm-.044.04-.044.04Zm-.044.04-.087.08Zm-.087.08-.043.04Zm-.043.04-.043.038Zm-.043.038-.043.04Zm-.043.04-.042.038Zm-.042.038-.085.077Zm-.085.077-.083.076Zm-.083.076-.083.076Zm-.083.076-.041.037Zm-.041.037-.04.037Zm-.04.037-.041.037Zm-.041.037-.04.037Zm-.04.037-.04.036Zm-.04.036-.04.036Zm-.04.036-.04.037Zm-.04.037-.04.035Zm-.04.035-.154.142Zm-.154.142-.298.271Zm-.298.271-.143.13Zm-.143.13-.07.064Zm-.07.064-.034.031Zm-.034.031-.035.032Zm-.035.032-.067.061Zm-.067.061-.067.06Zm-.067.06-.032.03Zm-.032.03-.033.03Zm-.033.03-.064.059Zm-.064.059-.032.029Zm-.032.029-.031.028Zm-.031.028-.062.057Zm-.062.057-.12.11Zm-.12.11-.433.393Zm-.433.393-.348.318Zm-.348.318-.257.234Zm-.257.234-.05.045Zm-.05.045-.01.01Zm-.01.01-.011.01Zm-.011.01-.021.019Zm-.021.019-.037.033Zm-.037.033-.002.002Z"
                })
            }),
            /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("g", {
                clipPath: "url(#555e6d__a)",
                children: /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#555e6d__b)",
                    children: [
                        /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#555e6d__c)"
                        }),
                        /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("path", {
                            fill: "url(#555e6d__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("path", {
                            fill: "url(#555e6d__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $4G2Em$reactjsxruntime.jsx)("path", {
                            fill: "url(#555e6d__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $0143f4b5862cc0c1$var$ForwardRef = /*#__PURE__*/ (0, $4G2Em$react.forwardRef)($0143f4b5862cc0c1$var$SvgComponent);
var $0143f4b5862cc0c1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0143f4b5862cc0c1$var$ForwardRef);
