// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_gift_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $1GOOH$reactjsxruntime = require("react/jsx-runtime");
var $1GOOH$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $57324e92ffd6d3d2$export$2e2bcd8739ae039);




const $57324e92ffd6d3d2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsxs)("linearGradient", {
                        id: "15700f__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsxs)("linearGradient", {
                        id: "15700f__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsxs)("linearGradient", {
                        id: "15700f__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsxs)("radialGradient", {
                        id: "15700f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("clipPath", {
                        id: "15700f__b",
                        children: /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("clipPath", {
                id: "15700f__a",
                children: /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M28.685 25.507c0-5.887 4.772-10.66 10.66-10.66 3.384 0 6.688 1.578 8.64 4.037 1.953-2.46 4.68-4.036 8.065-4.036 5.887 0 10.66 4.772 10.66 10.66 0 2.245-.694 4.328-1.88 6.046h-8.717c3.31-.034 5.984-2.728 5.984-6.047 0-3.34-2.707-6.047-6.047-6.047-3.324 0-5.446 2.683-5.47 6.002v6.092l-5.189-.004v-6.088c-.024-3.319-2.722-6.002-6.047-6.002-3.34 0-6.047 2.708-6.047 6.047 0 3.319 2.674 6.013 5.984 6.047h-8.717c-1.185-1.718-1.88-3.801-1.88-6.047M16 36.181c0-2.547 2.065-4.613 4.613-4.613h24.792v19.027H20.613c-2.548 0-4.613-2.065-4.613-4.612zm64 0c0-2.547-2.065-4.612-4.613-4.612H50.595v19.027h24.792c2.548 0 4.613-2.066 4.613-4.613zM50.595 55.208h25.369v14.415C75.964 75.354 71.317 80 65.586 80H50.595zm-30.559 0h25.37v24.793H30.413c-5.731 0-10.378-4.647-10.378-10.378z"
                })
            }),
            /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("g", {
                clipPath: "url(#15700f__a)",
                children: /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#15700f__b)",
                    children: [
                        /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#15700f__c)"
                        }),
                        /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("path", {
                            fill: "url(#15700f__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("path", {
                            fill: "url(#15700f__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $1GOOH$reactjsxruntime.jsx)("path", {
                            fill: "url(#15700f__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $57324e92ffd6d3d2$var$ForwardRef = /*#__PURE__*/ (0, $1GOOH$react.forwardRef)($57324e92ffd6d3d2$var$SvgComponent);
var $57324e92ffd6d3d2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($57324e92ffd6d3d2$var$ForwardRef);
