// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_calendar_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bpCLM$jsxs, jsx as $bpCLM$jsx} from "react/jsx-runtime";
import {forwardRef as $bpCLM$forwardRef} from "react";





const $c9109896d83f2fa6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bpCLM$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bpCLM$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bpCLM$jsxs)("linearGradient", {
                        id: "fb97b9__d",
                        x1: 67.441,
                        x2: 22.466,
                        y1: 3.461,
                        y2: 65.318,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bpCLM$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $bpCLM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bpCLM$jsxs)("linearGradient", {
                        id: "fb97b9__e",
                        x1: 37.887,
                        x2: 46.225,
                        y1: -23.833,
                        y2: 32.773,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bpCLM$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $bpCLM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bpCLM$jsxs)("linearGradient", {
                        id: "fb97b9__f",
                        x1: -70.084,
                        x2: 86.625,
                        y1: 89.668,
                        y2: 56.813,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bpCLM$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bpCLM$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $bpCLM$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $bpCLM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bpCLM$jsxs)("radialGradient", {
                        id: "fb97b9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 114.004 -120.937 0 81.499 57.91)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bpCLM$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bpCLM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bpCLM$jsx)("clipPath", {
                        id: "fb97b9__b",
                        children: /*#__PURE__*/ (0, $bpCLM$jsx)("rect", {
                            width: 83.762,
                            height: 78.96,
                            x: 6.836,
                            y: 6,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bpCLM$jsx)("clipPath", {
                id: "fb97b9__a",
                children: /*#__PURE__*/ (0, $bpCLM$jsx)("path", {
                    fill: "#fff",
                    d: "M69.597 20.4h-4.2v6c0 1.326-1.074 2.4-2.4 2.4s-2.4-1.074-2.4-2.4v-6h-25.2v6c0 1.326-1.074 2.4-2.4 2.4s-2.4-1.074-2.4-2.4v-6h-4.2c-5.304 0-9.6 4.296-9.6 9.6v6h62.4v-6c0-5.304-4.296-9.6-9.6-9.6m-52.8 49.2c0 5.304 4.296 9.6 9.6 9.6h43.2c5.304 0 9.6-4.296 9.6-9.6V40.8h-62.4zm43.8-19.2c0-1.326 1.074-2.4 2.4-2.4h3.6c1.326 0 2.4 1.074 2.4 2.4V54c0 1.326-1.074 2.4-2.4 2.4h-3.6c-1.326 0-2.4-1.074-2.4-2.4zm0 13.2c0-1.326 1.074-2.4 2.4-2.4h3.6c1.326 0 2.4 1.074 2.4 2.4v3.6c0 1.326-1.074 2.4-2.4 2.4h-3.6c-1.326 0-2.4-1.074-2.4-2.4zm-16.8-13.2c0-1.326 1.074-2.4 2.4-2.4h3.6c1.326 0 2.4 1.074 2.4 2.4V54c0 1.326-1.074 2.4-2.4 2.4h-3.6c-1.326 0-2.4-1.074-2.4-2.4zm0 13.2c0-1.326 1.074-2.4 2.4-2.4h3.6c1.326 0 2.4 1.074 2.4 2.4v3.6c0 1.326-1.074 2.4-2.4 2.4h-3.6c-1.326 0-2.4-1.074-2.4-2.4zm-16.8-13.2c0-1.326 1.074-2.4 2.4-2.4h3.6c1.326 0 2.4 1.074 2.4 2.4V54c0 1.326-1.074 2.4-2.4 2.4h-3.6c-1.326 0-2.4-1.074-2.4-2.4zm0 13.2c0-1.326 1.074-2.4 2.4-2.4h3.6c1.326 0 2.4 1.074 2.4 2.4v3.6c0 1.326-1.074 2.4-2.4 2.4h-3.6c-1.326 0-2.4-1.074-2.4-2.4zm8.4-49.2c0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4v6h4.8zm30 0c0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4v6h4.8z"
                })
            }),
            /*#__PURE__*/ (0, $bpCLM$jsx)("g", {
                clipPath: "url(#fb97b9__a)",
                children: /*#__PURE__*/ (0, $bpCLM$jsxs)("g", {
                    clipPath: "url(#fb97b9__b)",
                    children: [
                        /*#__PURE__*/ (0, $bpCLM$jsx)("rect", {
                            width: 83.762,
                            height: 78.96,
                            x: 6.836,
                            y: 6,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $bpCLM$jsx)("ellipse", {
                            cx: 81.499,
                            cy: 57.91,
                            fill: "url(#fb97b9__c)",
                            rx: 120.937,
                            ry: 114.004
                        }),
                        /*#__PURE__*/ (0, $bpCLM$jsx)("path", {
                            fill: "url(#fb97b9__d)",
                            d: "M-.703 30.731C-2.048 3.44 20.333-19.713 49.284-20.979c28.952-1.268 53.512 19.829 54.856 47.121S83.104 76.586 54.153 77.853C25.2 79.121.64 58.023-.703 30.731"
                        }),
                        /*#__PURE__*/ (0, $bpCLM$jsx)("path", {
                            fill: "url(#fb97b9__e)",
                            d: "M27.996-48.638c-6.532-3.81-14.806-3.833-21.362-.06l-65.566 37.737c-6.42 3.696-6.431 12.42-.019 16.13L6.181 42.853c6.546 3.786 14.821 3.782 21.36-.014l65-37.719c6.378-3.702 6.39-12.377.019-16.093z"
                        }),
                        /*#__PURE__*/ (0, $bpCLM$jsx)("path", {
                            fill: "url(#fb97b9__f)",
                            d: "M-34.92 43.766c13.515 0 25.756 5.156 34.62 13.474 17.725 16.664 32.42 16.678 50.146.043 8.863-8.36 21.12-13.531 34.649-13.531 27.043.014 48.95 20.61 48.95 46.033s-21.907 46.005-48.95 46.005c-13.56 0-25.816-5.185-34.679-13.545-17.71-16.608-32.391-16.593-50.117.071-8.863 8.332-21.104 13.474-34.618 13.474-27.028 0-48.935-20.596-48.935-46.005s21.907-46.019 48.935-46.019"
                        })
                    ]
                })
            })
        ]
    });
const $c9109896d83f2fa6$var$ForwardRef = /*#__PURE__*/ (0, $bpCLM$forwardRef)($c9109896d83f2fa6$var$SvgComponent);
var $c9109896d83f2fa6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c9109896d83f2fa6$var$ForwardRef);


export {$c9109896d83f2fa6$export$2e2bcd8739ae039 as default};
