// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_handshake_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $blfBs$reactjsxruntime = require("react/jsx-runtime");
var $blfBs$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $73a789772c0ccd62$export$2e2bcd8739ae039);




const $73a789772c0ccd62$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ef2e90__d",
                        x1: 32.334,
                        x2: 11.233,
                        y1: 5.007,
                        y2: 34.029,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ef2e90__e",
                        x1: 18.466,
                        x2: 22.378,
                        y1: -7.8,
                        y2: 18.758,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ef2e90__f",
                        x1: -32.193,
                        x2: 41.333,
                        y1: 45.453,
                        y2: 30.038,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ef2e90__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.4895 -56.7422 0 38.93 30.552)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("clipPath", {
                        id: "ef2e90__b",
                        children: /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.197,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("clipPath", {
                id: "ef2e90__a",
                children: /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m18.337 11.915 1.697 1.697 1.734 1.734 3.39 3.39c1.098 1.098 2.877 1.098 3.975 0s1.098-2.877 0-3.975l-3.39-3.39-1.734-1.734-.564-.564c-.402-.396-.855-.702-1.338-.927l-.046-.023q-.08-.04-.164-.073c-.19-.078-.384-.138-.58-.192-1.601-.426-3.377-.015-4.634 1.239l-9.13 9.129c-1.466 1.467-1.778 3.645-.95 5.424L16.64 13.612zm12.98-2.794 9.128 9.129v.006c1.875 1.875 1.875 4.914 0 6.789L30.281 35.209l-4.284 4.284c-1.098 1.098-2.877 1.098-3.975-.003h-.003c-1.098-1.098-1.098-2.877 0-3.975-1.098 1.098-2.877 1.098-3.975-.003h-.003c-1.089-1.09-1.095-2.847-.024-3.948-1.1 1.07-2.859 1.062-3.948-.03h-.003c-1.098-1.098-1.098-2.88 0-3.978h-.003c-1.098 1.098-2.877 1.098-3.975-.003h-.003q-.13-.132-.237-.273c-.852-1.104-.774-2.694.237-3.705l4.488-4.491 3.768-3.768 1.734 1.734 3.39 3.39c.984.984 2.292 1.527 3.684 1.527s2.7-.543 3.684-1.527c2.031-2.034 2.031-5.34 0-7.371l-4.91-4.911c1.77-.813 3.932-.498 5.393.963"
                })
            }),
            /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ef2e90__a)",
                children: /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ef2e90__b)",
                    children: [
                        /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.197,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("ellipse", {
                            cx: 38.93,
                            cy: 30.552,
                            fill: "url(#ef2e90__c)",
                            rx: 56.742,
                            ry: 53.489
                        }),
                        /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("path", {
                            fill: "url(#ef2e90__d)",
                            d: "M.362 17.801C-.268 4.996 10.232-5.866 23.815-6.46s25.107 9.304 25.738 22.11c.631 12.804-9.87 23.667-23.453 24.262S.993 30.607.362 17.8"
                        }),
                        /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("path", {
                            fill: "url(#ef2e90__e)",
                            d: "M13.825-19.439c-3.064-1.787-6.947-1.798-10.022-.028L-26.96-1.761c-3.013 1.734-3.018 5.827-.01 7.568l30.56 17.68c3.071 1.777 6.954 1.775 10.022-.005L44.11 5.784c2.993-1.737 2.998-5.807.01-7.55z"
                        }),
                        /*#__PURE__*/ (0, $blfBs$reactjsxruntime.jsx)("path", {
                            fill: "url(#ef2e90__f)",
                            d: "M-15.694 23.917c6.34 0 12.084 2.419 16.242 6.322 8.317 7.818 15.212 7.825 23.529.02 4.158-3.923 9.909-6.349 16.257-6.349 12.688.007 22.967 9.67 22.967 21.598 0 11.929-10.279 21.585-22.967 21.585-6.362 0-12.113-2.432-16.271-6.355-8.31-7.792-15.198-7.785-23.515.034-4.158 3.909-9.902 6.321-16.242 6.321-12.681 0-22.96-9.663-22.96-21.585 0-11.921 10.279-21.591 22.96-21.591"
                        })
                    ]
                })
            })
        ]
    });
const $73a789772c0ccd62$var$ForwardRef = /*#__PURE__*/ (0, $blfBs$react.forwardRef)($73a789772c0ccd62$var$SvgComponent);
var $73a789772c0ccd62$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($73a789772c0ccd62$var$ForwardRef);
