// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_chartAreaStack_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $dHIGT$jsxs, jsx as $dHIGT$jsx} from "react/jsx-runtime";
import {forwardRef as $dHIGT$forwardRef} from "react";





const $a7bec63edf0c8b26$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dHIGT$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dHIGT$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dHIGT$jsxs)("linearGradient", {
                        id: "22adc0__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dHIGT$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dHIGT$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dHIGT$jsxs)("linearGradient", {
                        id: "22adc0__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dHIGT$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dHIGT$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dHIGT$jsxs)("linearGradient", {
                        id: "22adc0__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dHIGT$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dHIGT$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dHIGT$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dHIGT$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dHIGT$jsxs)("radialGradient", {
                        id: "22adc0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dHIGT$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dHIGT$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dHIGT$jsx)("clipPath", {
                        id: "22adc0__b",
                        children: /*#__PURE__*/ (0, $dHIGT$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dHIGT$jsx)("clipPath", {
                id: "22adc0__a",
                children: /*#__PURE__*/ (0, $dHIGT$jsx)("path", {
                    fill: "#8F8F8F",
                    fillRule: "evenodd",
                    d: "M120.637 37.144c3.993-7.24 15.006-4.405 15.006 3.863v82.712c0 6.628-5.373 12-12 12H36c-6.627 0-12-5.372-12-12V79.994c0-2.203.908-4.308 2.51-5.82L43.543 58.11c3.313-3.124 8.56-2.858 11.54.585l12.31 14.228 16.1-24.9c2.946-4.558 9.477-4.921 12.912-.72l11.126 13.611zl3.503 1.932.039.021m-.039-.021-3.503-1.932Zm3.503 1.931-13.108 23.77c-2.734 4.96-9.615 5.585-13.199 1.2L90.21 52.366l-16.1 24.9c-2.896 4.478-9.28 4.924-12.768.891L49.032 63.93 32 79.994v24.14l16.119-10.386c2.48-1.598 5.696-1.469 8.04.322L72.429 106.5l19.93-23.184c2.655-3.09 7.377-3.267 10.258-.387l14.209 14.21 10.818-5.187zm0 59.816-9.914 4.753c-1.53.734-3.357.422-4.557-.778l-15.45-15.45-19.535 22.724c-2.59 3.013-7.082 3.483-10.24 1.07L51.86 100.854 32 113.651v10.068c0 2.21 1.79 4 4 4h87.643c2.209 0 4-1.79 4-4z"
                })
            }),
            /*#__PURE__*/ (0, $dHIGT$jsx)("g", {
                clipPath: "url(#22adc0__a)",
                children: /*#__PURE__*/ (0, $dHIGT$jsxs)("g", {
                    clipPath: "url(#22adc0__b)",
                    children: [
                        /*#__PURE__*/ (0, $dHIGT$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dHIGT$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#22adc0__c)"
                        }),
                        /*#__PURE__*/ (0, $dHIGT$jsx)("path", {
                            fill: "url(#22adc0__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $dHIGT$jsx)("path", {
                            fill: "url(#22adc0__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $dHIGT$jsx)("path", {
                            fill: "url(#22adc0__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $a7bec63edf0c8b26$var$ForwardRef = /*#__PURE__*/ (0, $dHIGT$forwardRef)($a7bec63edf0c8b26$var$SvgComponent);
var $a7bec63edf0c8b26$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a7bec63edf0c8b26$var$ForwardRef);


export {$a7bec63edf0c8b26$export$2e2bcd8739ae039 as default};
