// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_paperclip_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6yVEe$jsxs, jsx as $6yVEe$jsx} from "react/jsx-runtime";
import {forwardRef as $6yVEe$forwardRef} from "react";





const $c8f9bc4959cd1094$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6yVEe$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6yVEe$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6yVEe$jsxs)("linearGradient", {
                        id: "9f8d31__d",
                        x1: 65.566,
                        x2: 25.102,
                        y1: 10.135,
                        y2: 65.787,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6yVEe$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6yVEe$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6yVEe$jsxs)("linearGradient", {
                        id: "9f8d31__e",
                        x1: 38.978,
                        x2: 46.479,
                        y1: -14.421,
                        y2: 36.507,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6yVEe$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6yVEe$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6yVEe$jsxs)("linearGradient", {
                        id: "9f8d31__f",
                        x1: -58.165,
                        x2: 82.825,
                        y1: 87.696,
                        y2: 58.137,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6yVEe$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6yVEe$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6yVEe$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6yVEe$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6yVEe$jsxs)("radialGradient", {
                        id: "9f8d31__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 78.212 59.123)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6yVEe$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6yVEe$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6yVEe$jsx)("clipPath", {
                        id: "9f8d31__b",
                        children: /*#__PURE__*/ (0, $6yVEe$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 12.42,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6yVEe$jsx)("clipPath", {
                id: "9f8d31__a",
                children: /*#__PURE__*/ (0, $6yVEe$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    stroke: "#fff",
                    strokeLinecap: "round",
                    strokeLinejoin: "round",
                    strokeWidth: 1.2,
                    d: "M21.211 53.732c-6.682 6.65-6.682 17.436 0 24.086 6.677 6.643 17.496 6.643 24.172 0L72.424 50.91c.94-.935.944-2.455.009-3.394s-2.455-.944-3.394-.009L41.997 74.415c-4.803 4.78-12.596 4.78-17.4 0-4.797-4.773-4.797-12.508 0-17.28l37.034-36.852c3.058-3.044 8.021-3.044 11.08 0 3.051 3.036 3.051 7.954 0 10.99L39.515 64.306c-1.276 1.271-3.352 1.271-4.629 0-1.27-1.264-1.27-3.308 0-4.573L58.09 36.646c.94-.935.943-2.455.008-3.394s-2.455-.944-3.394-.009L31.5 56.33c-3.157 3.14-3.156 8.237 0 11.378 3.15 3.134 8.25 3.134 11.4 0l33.195-33.032c4.938-4.913 4.938-12.883 0-17.796-4.93-4.906-12.92-4.906-17.85 0z"
                })
            }),
            /*#__PURE__*/ (0, $6yVEe$jsx)("g", {
                clipPath: "url(#9f8d31__a)",
                children: /*#__PURE__*/ (0, $6yVEe$jsxs)("g", {
                    clipPath: "url(#9f8d31__b)",
                    children: [
                        /*#__PURE__*/ (0, $6yVEe$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 12.42,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6yVEe$jsx)("ellipse", {
                            cx: 78.212,
                            cy: 59.123,
                            fill: "url(#9f8d31__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $6yVEe$jsx)("path", {
                            fill: "url(#9f8d31__d)",
                            d: "M4.256 34.67C3.046 10.115 23.182-10.714 49.23-11.854S97.374 5.986 98.583 30.54 79.658 75.925 53.61 77.065 5.466 59.225 4.256 34.67"
                        }),
                        /*#__PURE__*/ (0, $6yVEe$jsx)("path", {
                            fill: "url(#9f8d31__e)",
                            d: "M30.078-36.738c-5.876-3.427-13.321-3.448-19.219-.054L-48.13-2.84c-5.777 3.326-5.787 11.174-.017 14.512l58.599 33.904c5.889 3.407 13.334 3.403 19.218-.012l58.478-33.936c5.74-3.33 5.75-11.135.018-14.479z"
                        }),
                        /*#__PURE__*/ (0, $6yVEe$jsx)("path", {
                            fill: "url(#9f8d31__f)",
                            d: "M-26.528 46.399c12.159 0 23.172 4.638 31.146 12.122 15.948 14.993 29.17 15.005 45.118.038 7.974-7.522 19-12.173 31.173-12.173 24.33.013 44.04 18.542 44.04 41.416s-19.71 41.39-44.04 41.39c-12.2 0-23.227-4.665-31.2-12.187-15.935-14.941-29.143-14.928-45.09.064-7.975 7.497-18.988 12.123-31.147 12.123-24.317 0-44.026-18.53-44.026-41.39 0-22.861 19.71-41.403 44.026-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $c8f9bc4959cd1094$var$ForwardRef = /*#__PURE__*/ (0, $6yVEe$forwardRef)($c8f9bc4959cd1094$var$SvgComponent);
var $c8f9bc4959cd1094$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c8f9bc4959cd1094$var$ForwardRef);


export {$c8f9bc4959cd1094$export$2e2bcd8739ae039 as default};
