// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_interaction_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $af1Ks$reactjsxruntime = require("react/jsx-runtime");
var $af1Ks$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f84fd2ed95780682$export$2e2bcd8739ae039);




const $f84fd2ed95780682$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsxs)("linearGradient", {
                        id: "230d95__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsxs)("linearGradient", {
                        id: "230d95__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsxs)("linearGradient", {
                        id: "230d95__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsxs)("radialGradient", {
                        id: "230d95__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("clipPath", {
                        id: "230d95__b",
                        children: /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("clipPath", {
                id: "230d95__a",
                children: /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M55.367 34.184c0 3.673-1.032 7.105-2.823 10.022q-.377.161-.745.33l-5.594-10.07-.032-.055c-1.98-3.311-6.615-5.926-11.155-3.464-4.326 2.345-5.64 7.975-3.244 12.016L37.53 53.32q-.667.045-1.345.046C25.589 53.367 17 44.778 17 34.184 17 23.589 25.589 15 36.184 15c10.594 0 19.183 8.589 19.183 19.184m-12.628 2.28 6.528 11.75.7 1.26c.145.26.484.338.735.175 3.136-2.031 9.32-4.185 13.037-4.185 9.941 0 15.5 7.693 15.5 17.634s-7.956 18-17.897 18c-9.22 0-22.93-6.997-27.264-9.21q-.51-.26-.838-.425c-3-1.5-3.676-3.545-3-6 .688-2.503 3.28-3.773 6-3 2.664.758 5.636 2.039 8.437 3.246 1.523.656 2.994 1.29 4.337 1.806.428.165.814-.292.591-.692l-7.92-14.256-6.446-11.603c-1.328-2.186-.563-5.282 1.685-6.5 2.016-1.093 4.368-.15 5.646 1.734q.088.13.17.266"
                })
            }),
            /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("g", {
                clipPath: "url(#230d95__a)",
                children: /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#230d95__b)",
                    children: [
                        /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#230d95__c)"
                        }),
                        /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("path", {
                            fill: "url(#230d95__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("path", {
                            fill: "url(#230d95__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $af1Ks$reactjsxruntime.jsx)("path", {
                            fill: "url(#230d95__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $f84fd2ed95780682$var$ForwardRef = /*#__PURE__*/ (0, $af1Ks$react.forwardRef)($f84fd2ed95780682$var$SvgComponent);
var $f84fd2ed95780682$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f84fd2ed95780682$var$ForwardRef);
