// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_filmstrip_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6NGQh$jsxs, jsx as $6NGQh$jsx} from "react/jsx-runtime";
import {forwardRef as $6NGQh$forwardRef} from "react";





const $c573ea22c8285f0b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6NGQh$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6NGQh$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6NGQh$jsxs)("linearGradient", {
                        id: "566679__d",
                        x1: 33.201,
                        x2: 9.027,
                        y1: 1.657,
                        y2: 32.999,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6NGQh$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6NGQh$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6NGQh$jsxs)("linearGradient", {
                        id: "566679__e",
                        x1: 18.473,
                        x2: 23.137,
                        y1: -12.772,
                        y2: 17.074,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6NGQh$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6NGQh$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6NGQh$jsxs)("linearGradient", {
                        id: "566679__f",
                        x1: -35.335,
                        x2: 43.129,
                        y1: 47.233,
                        y2: 31.725,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6NGQh$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6NGQh$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6NGQh$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6NGQh$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6NGQh$jsxs)("radialGradient", {
                        id: "566679__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 4.884 35.326)scale(60.2686)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6NGQh$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6NGQh$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6NGQh$jsx)("clipPath", {
                        id: "566679__b",
                        children: /*#__PURE__*/ (0, $6NGQh$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3,
                            y: 3,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6NGQh$jsx)("clipPath", {
                id: "566679__a",
                children: /*#__PURE__*/ (0, $6NGQh$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M10.8 10.3c-2.651 0-4.8 2.15-4.8 4.8v18c0 2.652 2.149 4.8 4.8 4.8h26.4c2.651 0 4.8-2.148 4.8-4.8v-18c0-2.65-2.149-4.8-4.8-4.8zM9.6 16c0-.33.269-.6.6-.6h2.1c.331 0 .6.27.6.6v1.5c0 .332-.269.6-.6.6h-2.1c-.331 0-.6-.268-.6-.6zm0 14.7c0-.33.269-.6.6-.6h2.1c.331 0 .6.27.6.6v1.5c0 .332-.269.6-.6.6h-2.1c-.331 0-.6-.268-.6-.6zM14.7 16c0-.33.269-.6.6-.6h2.1c.331 0 .6.27.6.6v1.5c0 .332-.269.6-.6.6h-2.1c-.331 0-.6-.268-.6-.6zm0 14.7c0-.33.269-.6.6-.6h2.1c.331 0 .6.27.6.6v1.5c0 .332-.269.6-.6.6h-2.1c-.331 0-.6-.268-.6-.6zM19.8 16c0-.33.269-.6.6-.6h2.1c.331 0 .6.27.6.6v1.5c0 .332-.269.6-.6.6h-2.1c-.331 0-.6-.268-.6-.6zm0 14.7c0-.33.269-.6.6-.6h2.1c.331 0 .6.27.6.6v1.5c0 .332-.269.6-.6.6h-2.1c-.331 0-.6-.268-.6-.6zM24.9 16c0-.33.269-.6.6-.6h2.1c.331 0 .6.27.6.6v1.5c0 .332-.269.6-.6.6h-2.1c-.331 0-.6-.268-.6-.6zm0 14.7c0-.33.269-.6.6-.6h2.1c.331 0 .6.27.6.6v1.5c0 .332-.269.6-.6.6h-2.1c-.331 0-.6-.268-.6-.6zM30 16c0-.33.269-.6.6-.6h2.1c.331 0 .6.27.6.6v1.5c0 .332-.269.6-.6.6h-2.1c-.331 0-.6-.268-.6-.6zm0 14.7c0-.33.269-.6.6-.6h2.1c.331 0 .6.27.6.6v1.5c0 .332-.269.6-.6.6h-2.1c-.331 0-.6-.268-.6-.6zM35.1 16c0-.33.269-.6.6-.6h2.1c.331 0 .6.27.6.6v1.5c0 .332-.269.6-.6.6h-2.1c-.331 0-.6-.268-.6-.6zm0 14.7c0-.33.269-.6.6-.6h2.1c.331 0 .6.27.6.6v1.5c0 .332-.269.6-.6.6h-2.1c-.331 0-.6-.268-.6-.6z"
                })
            }),
            /*#__PURE__*/ (0, $6NGQh$jsx)("g", {
                clipPath: "url(#566679__a)",
                children: /*#__PURE__*/ (0, $6NGQh$jsxs)("g", {
                    clipPath: "url(#566679__b)",
                    children: [
                        /*#__PURE__*/ (0, $6NGQh$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3,
                            y: 3,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6NGQh$jsx)("circle", {
                            cx: 40.21,
                            cy: 30.442,
                            r: 60.269,
                            fill: "url(#566679__c)"
                        }),
                        /*#__PURE__*/ (0, $6NGQh$jsx)("path", {
                            fill: "url(#566679__d)",
                            d: "M-.758 16.073c-.67-14.428 10.483-26.667 24.91-27.337C38.582-11.934 50.82-.781 51.49 13.647S41.007 40.315 26.58 40.985-.088 30.5-.758 16.072"
                        }),
                        /*#__PURE__*/ (0, $6NGQh$jsx)("path", {
                            fill: "url(#566679__e)",
                            d: "M13.544-25.885c-3.255-2.014-7.379-2.026-10.646-.032l-32.674 19.95c-3.2 1.954-3.206 6.566-.01 8.527L2.673 22.482c3.262 2.002 7.385 2 10.645-.007l32.391-19.94c3.179-1.958 3.185-6.544.01-8.508z"
                        }),
                        /*#__PURE__*/ (0, $6NGQh$jsx)("path", {
                            fill: "url(#566679__f)",
                            d: "M-17.81 22.967c6.734 0 12.835 2.725 17.252 7.123 8.833 8.81 16.157 8.817 24.99.022 4.417-4.42 10.525-7.153 17.267-7.153 13.477.008 24.395 10.895 24.395 24.336s-10.918 24.32-24.395 24.32c-6.757 0-12.865-2.74-17.282-7.16-8.826-8.78-16.142-8.772-24.975.037-4.417 4.405-10.518 7.123-17.252 7.123-13.47 0-24.387-10.888-24.387-24.32S-31.28 22.967-17.81 22.967"
                        })
                    ]
                })
            })
        ]
    });
const $c573ea22c8285f0b$var$ForwardRef = /*#__PURE__*/ (0, $6NGQh$forwardRef)($c573ea22c8285f0b$var$SvgComponent);
var $c573ea22c8285f0b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c573ea22c8285f0b$var$ForwardRef);


export {$c573ea22c8285f0b$export$2e2bcd8739ae039 as default};
