// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_dataAnalytics_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $IgYrU$reactjsxruntime = require("react/jsx-runtime");
var $IgYrU$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $86fb6b93ff075edc$export$2e2bcd8739ae039);




const $86fb6b93ff075edc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8080ea__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8080ea__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8080ea__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8080ea__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("clipPath", {
                        id: "8080ea__b",
                        children: /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("clipPath", {
                id: "8080ea__a",
                children: /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M135.135 32.356c1.371 1.733 1.078 4.249-.655 5.62L94.276 69.78c-1.238.98-2.936 1.139-4.334.408l-25.31-13.226-34.04 28.872c-1.685 1.43-4.21 1.222-5.639-.463-1.428-1.684-1.221-4.209.464-5.638L61.51 49.12c1.241-1.053 2.997-1.249 4.44-.495l25.396 13.272 38.169-30.196c1.733-1.37 4.249-1.077 5.619.655M118.043 72.51c0-3.798 3.079-6.877 6.877-6.877s6.877 3.079 6.877 6.877v45.777c0 3.798-3.079 6.877-6.877 6.877s-6.877-3.079-6.877-6.877zM58.158 87.516c0-3.798 3.08-6.877 6.877-6.877 3.798 0 6.877 3.079 6.877 6.877v30.771c0 3.798-3.079 6.877-6.877 6.877s-6.877-3.079-6.877-6.877zm36.82-.916c-3.798 0-6.877 3.079-6.877 6.877v24.81c0 3.798 3.079 6.877 6.877 6.877s6.877-3.079 6.877-6.877v-24.81c0-3.798-3.08-6.877-6.877-6.877M35.093 98.39c-3.798 0-6.877 3.08-6.877 6.878v13.019c0 3.798 3.079 6.877 6.877 6.877s6.877-3.079 6.877-6.877v-13.019c0-3.798-3.08-6.877-6.877-6.877"
                })
            }),
            /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("g", {
                clipPath: "url(#8080ea__a)",
                children: /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#8080ea__b)",
                    children: [
                        /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#8080ea__c)"
                        }),
                        /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("path", {
                            fill: "url(#8080ea__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("path", {
                            fill: "url(#8080ea__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $IgYrU$reactjsxruntime.jsx)("path", {
                            fill: "url(#8080ea__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $86fb6b93ff075edc$var$ForwardRef = /*#__PURE__*/ (0, $IgYrU$react.forwardRef)($86fb6b93ff075edc$var$SvgComponent);
var $86fb6b93ff075edc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($86fb6b93ff075edc$var$ForwardRef);
