// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_cut_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $iZGt6$reactjsxruntime = require("react/jsx-runtime");
var $iZGt6$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5df10e5870e7e40c$export$2e2bcd8739ae039);




const $5df10e5870e7e40c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "10904d__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "10904d__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "10904d__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsxs)("radialGradient", {
                        id: "10904d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("clipPath", {
                        id: "10904d__b",
                        children: /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("clipPath", {
                id: "10904d__a",
                children: /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M29.97 7.345c.654.224 1.002.934.779 1.588l-4.218 12.32 12.485-4.252c.653-.222 1.363.127 1.586.78.222.654-.127 1.364-.78 1.587L25.807 24.14q-.076.026-.154.042-.093.378-.182.77v.001c-.255 1.1-.53 2.29-1.15 3.63q-.159.344-.342.69h.081c3.162 0 5.724 2.563 5.724 5.724s-2.562 5.724-5.724 5.724c-3.16 0-5.724-2.562-5.724-5.724 0-1.24.395-2.39 1.066-3.327q.06-.1.144-.192.113-.145.235-.283c.528-.674 1.61-2.234 2.271-3.662.464-1.001.667-1.802.882-2.722-.41.155-.858.333-1.386.55-.825.34-1.792.88-2.633 1.405-.853.531-1.488.99-1.663 1.139q-.064.054-.131.097c-1.038 1.047-2.476 1.695-4.066 1.695-3.161 0-5.724-2.563-5.724-5.724s2.563-5.724 5.724-5.724c3.152 0 5.708 2.547 5.724 5.694.596-.332 1.222-.648 1.817-.893 1.265-.522 2.113-.835 3.064-1.134l4.722-13.793c.224-.653.935-1.002 1.588-.778M21.478 33.07c-.402.537-.64 1.205-.64 1.927 0 1.78 1.444 3.224 3.224 3.224s3.224-1.443 3.224-3.224c0-1.78-1.443-3.224-3.224-3.224-.942 0-1.79.404-2.379 1.048q-.113.144-.205.249m-8.42-12.321c-1.782 0-3.225 1.443-3.225 3.224 0 1.78 1.443 3.224 3.224 3.224 1.78 0 3.224-1.444 3.224-3.224s-1.443-3.224-3.224-3.224"
                })
            }),
            /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("g", {
                clipPath: "url(#10904d__a)",
                children: /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#10904d__b)",
                    children: [
                        /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#10904d__c)"
                        }),
                        /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("path", {
                            fill: "url(#10904d__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("path", {
                            fill: "url(#10904d__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $iZGt6$reactjsxruntime.jsx)("path", {
                            fill: "url(#10904d__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $5df10e5870e7e40c$var$ForwardRef = /*#__PURE__*/ (0, $iZGt6$react.forwardRef)($5df10e5870e7e40c$var$SvgComponent);
var $5df10e5870e7e40c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5df10e5870e7e40c$var$ForwardRef);
