// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_vectorDraw_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gjJEu$reactjsxruntime = require("react/jsx-runtime");
var $gjJEu$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d9c79520082ea8f7$export$2e2bcd8739ae039);




const $d9c79520082ea8f7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "10ec28__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "10ec28__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "10ec28__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsxs)("radialGradient", {
                        id: "10ec28__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("clipPath", {
                        id: "10ec28__b",
                        children: /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("clipPath", {
                id: "10ec28__a",
                children: /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M54.073 20.4 75.604 41.93l3.434-3.433c2.4-2.4 2.63-6.058.516-8.172L65.68 16.45c-2.114-2.114-5.773-1.883-8.173.516zm19.255 24.912L50.693 22.676 29.648 31.66c-1.821.778-3.255 2.36-3.834 4.231l-10.4 33.6c-.845 2.729-.327 5.39 1.035 7.465l21.873-21.873c-.719-1.8-.35-3.934 1.108-5.392 1.953-1.953 5.119-1.953 7.071 0 1.953 1.953 1.953 5.118 0 7.07-1.442 1.443-3.545 1.82-5.333 1.132l-21.84 21.84c2.032 1.229 4.578 1.666 7.186.858l33.6-10.4c1.872-.579 3.453-2.013 4.231-3.834z"
                })
            }),
            /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("g", {
                clipPath: "url(#10ec28__a)",
                children: /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#10ec28__b)",
                    children: [
                        /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#10ec28__c)"
                        }),
                        /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("path", {
                            fill: "url(#10ec28__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("path", {
                            fill: "url(#10ec28__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $gjJEu$reactjsxruntime.jsx)("path", {
                            fill: "url(#10ec28__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $d9c79520082ea8f7$var$ForwardRef = /*#__PURE__*/ (0, $gjJEu$react.forwardRef)($d9c79520082ea8f7$var$SvgComponent);
var $d9c79520082ea8f7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d9c79520082ea8f7$var$ForwardRef);
