// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_update_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $39YMu$reactjsxruntime = require("react/jsx-runtime");
var $39YMu$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1715336fd0541f2d$export$2e2bcd8739ae039);




const $1715336fd0541f2d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5ccad6__d",
                        x1: 33.867,
                        x2: 11.037,
                        y1: 1.112,
                        y2: 32.511,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5ccad6__e",
                        x1: 18.863,
                        x2: 23.095,
                        y1: -12.743,
                        y2: 15.99,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5ccad6__f",
                        x1: -35.944,
                        x2: 43.601,
                        y1: 44.871,
                        y2: 28.194,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5ccad6__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 57.8682 -61.3872 0 41 28.75)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("clipPath", {
                        id: "5ccad6__b",
                        children: /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("rect", {
                            width: 42.517,
                            height: 40.08,
                            x: 3.102,
                            y: 2.4,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsxs)("clipPath", {
                id: "5ccad6__a",
                children: [
                    /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("path", {
                        fill: "#fff",
                        fillRule: "evenodd",
                        d: "M42.037 12.57c.3-.591 1.023-.825 1.614-.525h.003c.59.3.825 1.023.525 1.614l-3.294 6.471c-.213.417-.633.657-1.071.657q-.258 0-.507-.114l-6.351-2.961c-.603-.282-.864-.996-.582-1.596s.996-.861 1.596-.579l4.149 1.935c-2.547-5.415-8.097-8.988-14.133-8.988-6.891 0-13.05 4.614-14.98 11.22-.152.525-.632.864-1.151.864q-.168 0-.336-.048c-.636-.186-1.002-.852-.816-1.488 2.226-7.623 9.333-12.948 17.283-12.948 6.879 0 13.209 4.02 16.194 10.137zM38.94 28.467c.186-.636.852-1.002 1.488-.816v.003c.636.186 1.002.852.816 1.488-2.226 7.623-9.333 12.948-17.283 12.948-6.846 0-13.15-3.984-16.152-10.053l-1.845 3.627c-.213.417-.633.657-1.071.657-.183 0-.37-.042-.543-.132-.591-.3-.825-1.023-.525-1.614l3.294-6.474c.294-.576.993-.816 1.578-.543l6.35 2.961c.604.282.862.996.583 1.596-.282.6-.996.858-1.596.579L9.85 30.741c2.553 5.394 8.09 8.946 14.112 8.946 6.89 0 13.05-4.614 14.979-11.22"
                    }),
                    /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("path", {
                        fill: "#fff",
                        d: "m43.65 12.045-.135.268.064.032h.072zm-1.613.525.267.136zm1.617-.525.135-.268-.064-.032h-.071zm.525 1.614.267.136zm-3.294 6.471.267.137zm-1.578.543-.127.272.003.001zm-6.351-2.961-.127.272zm-.582-1.596-.272-.128zm1.596-.579-.128.271zm4.149 1.935-.127.272.75.35-.352-.75zM9.007 19.704l-.288-.084zm-1.488.816.085-.288h-.001zm-.816-1.488.288.084zm33.477-2.811-.27.132.264.54.273-.536zm.249 11.43h.3v-.225l-.216-.063zm-1.488.816-.288-.084zm1.488-.813h-.3v.225l.216.063zm.816 1.488-.288-.084zM7.81 32.037l.269-.133-.265-.536-.271.533zm-1.845 3.627.267.136zm-1.614.525.138-.267h-.002zm-.525-1.614-.267-.136zm3.294-6.474-.267-.136zm1.578-.543-.127.272zm6.35 2.961.128-.272zm.583 1.596.271.128.001-.002zm-1.596.579-.127.272zM9.85 30.741l.127-.272-.754-.352.356.752zm33.937-18.963c-.738-.375-1.642-.083-2.018.656l.535.272c.225-.443.767-.619 1.211-.393zm-.133-.033h-.003v.6h.003zm.792 2.05c.375-.738.083-1.642-.657-2.017l-.271.534c.442.225.619.767.393 1.211zm-3.294 6.471 3.294-6.47-.535-.273-3.294 6.47zm-1.338.821c.547 0 1.072-.3 1.338-.82l-.534-.273c-.16.313-.475.493-.804.493zm-.631-.14c.2.09.413.14.63.14v-.6q-.187 0-.382-.087zm-6.354-2.963 6.351 2.96.254-.543-6.351-2.96zm-.727-1.996c-.353.752-.025 1.644.727 1.996l.254-.544c-.454-.212-.648-.748-.438-1.196zm1.995-.723c-.751-.353-1.643-.025-1.995.723l.543.256c.212-.452.749-.646 1.197-.435zm4.148 1.935-4.148-1.935-.254.544 4.149 1.935zm-14.26-8.416c5.92 0 11.364 3.505 13.862 8.816l.543-.256c-2.596-5.519-8.25-9.16-14.404-9.16zM9.296 19.788c1.891-6.478 7.933-11.004 14.69-11.004v-.6c-7.023 0-13.3 4.702-15.266 11.436zm-1.44 1.08c.648 0 1.248-.423 1.44-1.08l-.576-.168c-.115.393-.474.648-.864.648zm-.422-.06q.213.06.422.06v-.6q-.129 0-.25-.036zm-1.018-1.86c-.233.795.225 1.628 1.02 1.86l.168-.576c-.478-.14-.752-.64-.612-1.116zm17.57-13.164c-8.082 0-15.307 5.413-17.57 13.164l.576.168c2.188-7.495 9.178-12.732 16.995-12.732zM40.45 16.089C37.415 9.871 30.981 5.784 23.986 5.784v.6c6.763 0 12.989 3.954 15.924 9.969zm1.32-3.655-1.857 3.651.535.272 1.857-3.651zm-1.256 14.929c-.795-.233-1.628.225-1.86 1.02l.576.168c.14-.477.64-.751 1.116-.612zm.216.291v-.003h-.6v.003zm.804 1.572c.233-.795-.225-1.628-1.02-1.86l-.168.576c.477.14.751.64.612 1.116zM23.962 42.39c8.083 0 15.308-5.413 17.57-13.164l-.575-.168c-2.189 7.495-9.178 12.732-16.995 12.732zM7.542 32.17c3.052 6.17 9.458 10.22 16.42 10.22v-.6c-6.73 0-12.93-3.918-15.883-9.886zm-1.31 3.63 1.845-3.627-.534-.272-1.845 3.627zm-1.338.821c.547 0 1.072-.3 1.338-.82l-.534-.273c-.16.313-.475.493-.804.493zm-.68-.166c.217.113.45.166.68.166v-.6c-.137 0-.275-.031-.405-.099zm-.656-2.016c-.374.738-.082 1.642.657 2.017l.272-.535c-.443-.224-.619-.766-.394-1.21zm3.295-6.474-3.294 6.474.534.272 3.294-6.474zm1.972-.679c-.732-.341-1.605-.041-1.972.679l.534.272c.22-.432.746-.612 1.184-.407zm6.35 2.961-6.35-2.96-.254.543 6.351 2.96zm.728 1.995c.349-.75.026-1.643-.727-1.995l-.254.544c.453.212.646.747.437 1.197zm-1.995.724c.75.349 1.642.026 1.994-.723l-.543-.256c-.211.45-.747.644-1.198.435zm-4.185-1.953 4.185 1.953.254-.544-4.185-1.953zm14.239 8.374c-5.904 0-11.337-3.484-13.84-8.774l-.543.256c2.602 5.498 8.245 9.118 14.383 9.118zm14.691-11.004c-1.892 6.478-7.933 11.004-14.691 11.004v.6c7.024 0 13.3-4.702 15.267-11.436z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("g", {
                clipPath: "url(#5ccad6__a)",
                children: /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#5ccad6__b)",
                    children: [
                        /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("rect", {
                            width: 42.517,
                            height: 40.08,
                            x: 3.102,
                            y: 2.4,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("ellipse", {
                            cx: 41.001,
                            cy: 28.749,
                            fill: "url(#5ccad6__c)",
                            rx: 61.387,
                            ry: 57.868
                        }),
                        /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("path", {
                            fill: "url(#5ccad6__d)",
                            d: "M-.723 14.955C-1.406 1.1 9.954-10.651 24.65-11.294s27.163 10.066 27.845 23.92c.682 13.853-10.678 25.604-25.374 26.248C12.426 39.517-.04 28.808-.723 14.954"
                        }),
                        /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("path", {
                            fill: "url(#5ccad6__e)",
                            d: "M13.841-25.334c-3.315-1.934-7.515-1.946-10.843-.03L-30.282-6.21c-3.26 1.876-3.265 6.304-.01 8.187l33.06 19.128c3.323 1.922 7.523 1.92 10.843-.007L46.604 1.953c3.238-1.879 3.244-6.282.01-8.169z"
                        }),
                        /*#__PURE__*/ (0, $39YMu$reactjsxruntime.jsx)("path", {
                            fill: "url(#5ccad6__f)",
                            d: "M-18.094 21.572c6.86 0 13.073 2.617 17.572 6.839 8.997 8.459 16.457 8.466 25.455.022 4.498-4.244 10.72-6.869 17.587-6.869 13.727.008 24.847 10.462 24.847 23.367S56.247 68.283 42.52 68.283c-6.883 0-13.104-2.632-17.603-6.876-8.99-8.43-16.442-8.422-25.44.036-4.498 4.23-10.712 6.84-17.571 6.84-13.72 0-24.84-10.454-24.84-23.352s11.12-23.36 24.84-23.36"
                        })
                    ]
                })
            })
        ]
    });
const $1715336fd0541f2d$var$ForwardRef = /*#__PURE__*/ (0, $39YMu$react.forwardRef)($1715336fd0541f2d$var$SvgComponent);
var $1715336fd0541f2d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1715336fd0541f2d$var$ForwardRef);
