// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_camera_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lbs0S$jsxs, jsx as $lbs0S$jsx} from "react/jsx-runtime";
import {forwardRef as $lbs0S$forwardRef} from "react";





const $92a4f2a38fdc66d4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lbs0S$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lbs0S$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lbs0S$jsxs)("linearGradient", {
                        id: "22b059__d",
                        x1: 65.566,
                        x2: 25.102,
                        y1: 11.636,
                        y2: 67.288,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lbs0S$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $lbs0S$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lbs0S$jsxs)("linearGradient", {
                        id: "22b059__e",
                        x1: 38.978,
                        x2: 46.479,
                        y1: -12.921,
                        y2: 38.007,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lbs0S$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $lbs0S$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lbs0S$jsxs)("linearGradient", {
                        id: "22b059__f",
                        x1: -58.165,
                        x2: 82.825,
                        y1: 89.197,
                        y2: 59.638,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lbs0S$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lbs0S$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $lbs0S$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $lbs0S$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lbs0S$jsxs)("radialGradient", {
                        id: "22b059__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 78.212 60.623)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lbs0S$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lbs0S$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lbs0S$jsx)("clipPath", {
                        id: "22b059__b",
                        children: /*#__PURE__*/ (0, $lbs0S$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lbs0S$jsx)("clipPath", {
                id: "22b059__a",
                children: /*#__PURE__*/ (0, $lbs0S$jsx)("path", {
                    fill: "#D9D9D9",
                    d: "M63.001 26.4H33l3.883-6.215c1.314-2.106 3.624-3.384 6.108-3.384h10.02c2.484 0 4.788 1.278 6.108 3.384zM21.6 31.2h52.8c5.302 0 9.6 4.299 9.6 9.6v28.8c0 5.303-4.298 9.6-9.6 9.6H21.6c-5.302 0-9.6-4.297-9.6-9.6V40.8c0-5.301 4.298-9.6 9.6-9.6m.9 13.8c1.988 0 3.6-1.611 3.6-3.6 0-1.987-1.612-3.6-3.6-3.6s-3.6 1.613-3.6 3.6c0 1.989 1.612 3.6 3.6 3.6M48 69.6c7.953 0 14.4-6.446 14.4-14.4 0-7.952-6.447-14.4-14.4-14.4s-14.4 6.448-14.4 14.4c0 7.954 6.447 14.4 14.4 14.4"
                })
            }),
            /*#__PURE__*/ (0, $lbs0S$jsx)("g", {
                clipPath: "url(#22b059__a)",
                children: /*#__PURE__*/ (0, $lbs0S$jsxs)("g", {
                    clipPath: "url(#22b059__b)",
                    children: [
                        /*#__PURE__*/ (0, $lbs0S$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $lbs0S$jsx)("ellipse", {
                            cx: 78.212,
                            cy: 60.623,
                            fill: "url(#22b059__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $lbs0S$jsx)("path", {
                            fill: "url(#22b059__d)",
                            d: "M4.256 36.171C3.046 11.616 23.182-9.213 49.23-10.353s48.144 17.84 49.353 42.395S79.658 77.426 53.61 78.566 5.466 60.726 4.256 36.171"
                        }),
                        /*#__PURE__*/ (0, $lbs0S$jsx)("path", {
                            fill: "url(#22b059__e)",
                            d: "M30.078-35.238c-5.876-3.427-13.321-3.448-19.219-.054L-48.13-1.34c-5.777 3.326-5.787 11.174-.017 14.512l58.599 33.904c5.889 3.407 13.334 3.403 19.218-.012l58.478-33.936c5.74-3.33 5.75-11.135.018-14.479z"
                        }),
                        /*#__PURE__*/ (0, $lbs0S$jsx)("path", {
                            fill: "url(#22b059__f)",
                            d: "M-26.528 47.9c12.159 0 23.172 4.638 31.146 12.122 15.948 14.993 29.17 15.005 45.118.038 7.974-7.522 19-12.173 31.173-12.173 24.33.013 44.04 18.542 44.04 41.415 0 22.874-19.71 41.391-44.04 41.391-12.2 0-23.227-4.665-31.2-12.187-15.935-14.941-29.143-14.928-45.09.064-7.975 7.497-18.988 12.123-31.147 12.123-24.317 0-44.026-18.53-44.026-41.39 0-22.861 19.71-41.403 44.026-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $92a4f2a38fdc66d4$var$ForwardRef = /*#__PURE__*/ (0, $lbs0S$forwardRef)($92a4f2a38fdc66d4$var$SvgComponent);
var $92a4f2a38fdc66d4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($92a4f2a38fdc66d4$var$ForwardRef);


export {$92a4f2a38fdc66d4$export$2e2bcd8739ae039 as default};
