// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_userGroup_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $ek7uG$jsxs, jsx as $ek7uG$jsx} from "react/jsx-runtime";
import {forwardRef as $ek7uG$forwardRef} from "react";





const $46698db7ddcb3e17$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ek7uG$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $ek7uG$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $ek7uG$jsxs)("linearGradient", {
                        id: "3ba791__d",
                        x1: 32.334,
                        x2: 11.233,
                        y1: 5.007,
                        y2: 34.029,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ek7uG$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $ek7uG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ek7uG$jsxs)("linearGradient", {
                        id: "3ba791__e",
                        x1: 18.466,
                        x2: 22.378,
                        y1: -7.8,
                        y2: 18.758,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ek7uG$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $ek7uG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ek7uG$jsxs)("linearGradient", {
                        id: "3ba791__f",
                        x1: -32.193,
                        x2: 41.333,
                        y1: 45.453,
                        y2: 30.038,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ek7uG$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $ek7uG$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $ek7uG$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $ek7uG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ek7uG$jsxs)("radialGradient", {
                        id: "3ba791__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.4895 -56.7422 0 38.93 30.552)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ek7uG$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ek7uG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ek7uG$jsx)("clipPath", {
                        id: "3ba791__b",
                        children: /*#__PURE__*/ (0, $ek7uG$jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.197,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ek7uG$jsx)("clipPath", {
                id: "3ba791__a",
                children: /*#__PURE__*/ (0, $ek7uG$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M35.054 15.135c0 4.06-3.09 7.35-6.9 7.35s-6.9-3.29-6.9-7.35 3.089-7.35 6.9-7.35c3.81 0 6.9 3.29 6.9 7.35m-13.278 7.09c-.612.188-1.26.293-1.932.293-3.81 0-6.9-3.29-6.9-7.35s3.09-7.35 6.9-7.35c.65 0 1.284.1 1.88.28-1.77 1.775-2.873 4.247-2.873 7.037s1.125 5.31 2.925 7.09M12.02 36.74c.975-4.527 3.795-8.31 7.557-10.62-6.612.126-12.174 4.896-13.521 11.157-.324 1.506.8 2.934 2.343 2.934h4.182c-.612-1.038-.822-2.268-.561-3.47m16.13-10.656c6.73 0 12.424 4.818 13.789 11.163.324 1.506-.801 2.934-2.343 2.934h-22.89c-1.542 0-2.667-1.428-2.343-2.934 1.365-6.345 7.059-11.163 13.788-11.163"
                })
            }),
            /*#__PURE__*/ (0, $ek7uG$jsx)("g", {
                clipPath: "url(#3ba791__a)",
                children: /*#__PURE__*/ (0, $ek7uG$jsxs)("g", {
                    clipPath: "url(#3ba791__b)",
                    children: [
                        /*#__PURE__*/ (0, $ek7uG$jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.197,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $ek7uG$jsx)("ellipse", {
                            cx: 38.93,
                            cy: 30.552,
                            fill: "url(#3ba791__c)",
                            rx: 56.742,
                            ry: 53.489
                        }),
                        /*#__PURE__*/ (0, $ek7uG$jsx)("path", {
                            fill: "url(#3ba791__d)",
                            d: "M.362 17.801C-.268 4.996 10.232-5.866 23.815-6.46s25.107 9.304 25.738 22.11c.631 12.804-9.87 23.667-23.453 24.262S.993 30.607.362 17.8"
                        }),
                        /*#__PURE__*/ (0, $ek7uG$jsx)("path", {
                            fill: "url(#3ba791__e)",
                            d: "M13.825-19.439c-3.064-1.787-6.947-1.798-10.022-.028L-26.96-1.761c-3.013 1.734-3.018 5.827-.01 7.568l30.56 17.68c3.071 1.777 6.954 1.775 10.022-.005L44.11 5.784c2.993-1.737 2.998-5.807.01-7.55z"
                        }),
                        /*#__PURE__*/ (0, $ek7uG$jsx)("path", {
                            fill: "url(#3ba791__f)",
                            d: "M-15.694 23.917c6.34 0 12.084 2.419 16.242 6.322 8.317 7.818 15.212 7.825 23.529.02 4.158-3.923 9.909-6.349 16.257-6.349 12.688.007 22.967 9.67 22.967 21.598 0 11.929-10.279 21.585-22.967 21.585-6.362 0-12.113-2.432-16.271-6.355-8.31-7.792-15.198-7.785-23.515.034-4.158 3.909-9.902 6.321-16.242 6.321-12.681 0-22.96-9.663-22.96-21.585 0-11.921 10.279-21.591 22.96-21.591"
                        })
                    ]
                })
            })
        ]
    });
const $46698db7ddcb3e17$var$ForwardRef = /*#__PURE__*/ (0, $ek7uG$forwardRef)($46698db7ddcb3e17$var$SvgComponent);
var $46698db7ddcb3e17$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($46698db7ddcb3e17$var$ForwardRef);


export {$46698db7ddcb3e17$export$2e2bcd8739ae039 as default};
