// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_trash_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $fEvFw$jsxs, jsx as $fEvFw$jsx} from "react/jsx-runtime";
import {forwardRef as $fEvFw$forwardRef} from "react";





const $1f4b177844c5b786$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fEvFw$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fEvFw$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fEvFw$jsxs)("linearGradient", {
                        id: "92c420__d",
                        x1: 65.566,
                        x2: 25.102,
                        y1: 11.636,
                        y2: 67.288,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fEvFw$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $fEvFw$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fEvFw$jsxs)("linearGradient", {
                        id: "92c420__e",
                        x1: 38.978,
                        x2: 46.479,
                        y1: -12.921,
                        y2: 38.007,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fEvFw$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $fEvFw$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fEvFw$jsxs)("linearGradient", {
                        id: "92c420__f",
                        x1: -58.165,
                        x2: 82.825,
                        y1: 89.197,
                        y2: 59.638,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fEvFw$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fEvFw$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $fEvFw$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $fEvFw$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fEvFw$jsxs)("radialGradient", {
                        id: "92c420__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 78.212 60.623)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fEvFw$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fEvFw$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fEvFw$jsx)("clipPath", {
                        id: "92c420__b",
                        children: /*#__PURE__*/ (0, $fEvFw$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fEvFw$jsx)("clipPath", {
                id: "92c420__a",
                children: /*#__PURE__*/ (0, $fEvFw$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M33.602 19.6c0-3.314 2.686-6 6-6h16.8c3.314 0 6 2.686 6 6v3.6h18.6c1.325 0 2.4 1.074 2.4 2.4s-1.075 2.4-2.4 2.4h-5.7l-5.245 44.05c-.43 3.623-3.502 6.35-7.15 6.35H33.096c-3.647 0-6.718-2.727-7.15-6.35L20.703 28h-5.7c-1.326 0-2.4-1.075-2.4-2.4 0-1.326 1.074-2.4 2.4-2.4h18.6zm6-1.2c-.664 0-1.2.538-1.2 1.2v3.6h19.2v-3.6c0-.663-.537-1.2-1.2-1.2zm1.2 14.4c-1.326 0-2.4 1.074-2.4 2.4v33.6c0 1.325 1.074 2.4 2.4 2.4s2.4-1.075 2.4-2.4V35.2c0-1.326-1.075-2.4-2.4-2.4m14.4 0c-1.326 0-2.4 1.074-2.4 2.4v33.6c0 1.325 1.074 2.4 2.4 2.4s2.4-1.075 2.4-2.4V35.2c0-1.326-1.075-2.4-2.4-2.4"
                })
            }),
            /*#__PURE__*/ (0, $fEvFw$jsx)("g", {
                clipPath: "url(#92c420__a)",
                children: /*#__PURE__*/ (0, $fEvFw$jsxs)("g", {
                    clipPath: "url(#92c420__b)",
                    children: [
                        /*#__PURE__*/ (0, $fEvFw$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $fEvFw$jsx)("ellipse", {
                            cx: 78.212,
                            cy: 60.623,
                            fill: "url(#92c420__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $fEvFw$jsx)("path", {
                            fill: "url(#92c420__d)",
                            d: "M4.256 36.171C3.046 11.616 23.182-9.213 49.23-10.353s48.144 17.84 49.353 42.395S79.658 77.426 53.61 78.566 5.466 60.726 4.256 36.171"
                        }),
                        /*#__PURE__*/ (0, $fEvFw$jsx)("path", {
                            fill: "url(#92c420__e)",
                            d: "M30.078-35.238c-5.876-3.427-13.321-3.448-19.219-.054L-48.13-1.34c-5.777 3.326-5.787 11.174-.017 14.512l58.599 33.904c5.889 3.407 13.334 3.403 19.218-.012l58.478-33.936c5.74-3.33 5.75-11.135.018-14.479z"
                        }),
                        /*#__PURE__*/ (0, $fEvFw$jsx)("path", {
                            fill: "url(#92c420__f)",
                            d: "M-26.528 47.9c12.159 0 23.172 4.638 31.146 12.122 15.948 14.993 29.17 15.005 45.118.038 7.974-7.522 19-12.173 31.173-12.173 24.33.013 44.04 18.542 44.04 41.415 0 22.874-19.71 41.391-44.04 41.391-12.2 0-23.227-4.665-31.2-12.187-15.935-14.941-29.143-14.928-45.09.064-7.975 7.497-18.988 12.123-31.147 12.123-24.317 0-44.026-18.53-44.026-41.39 0-22.861 19.71-41.403 44.026-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $1f4b177844c5b786$var$ForwardRef = /*#__PURE__*/ (0, $fEvFw$forwardRef)($1f4b177844c5b786$var$SvgComponent);
var $1f4b177844c5b786$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1f4b177844c5b786$var$ForwardRef);


export {$1f4b177844c5b786$export$2e2bcd8739ae039 as default};
