// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_threeArrowsCircle_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $9RQDT$jsxs, jsx as $9RQDT$jsx} from "react/jsx-runtime";
import {forwardRef as $9RQDT$forwardRef} from "react";





const $066a6e84a70bdfe4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9RQDT$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9RQDT$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9RQDT$jsxs)("linearGradient", {
                        id: "df23e2__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9RQDT$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $9RQDT$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9RQDT$jsxs)("linearGradient", {
                        id: "df23e2__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9RQDT$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $9RQDT$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9RQDT$jsxs)("linearGradient", {
                        id: "df23e2__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9RQDT$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9RQDT$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $9RQDT$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $9RQDT$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9RQDT$jsxs)("radialGradient", {
                        id: "df23e2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9RQDT$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9RQDT$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9RQDT$jsx)("clipPath", {
                        id: "df23e2__b",
                        children: /*#__PURE__*/ (0, $9RQDT$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9RQDT$jsx)("clipPath", {
                id: "df23e2__a",
                children: /*#__PURE__*/ (0, $9RQDT$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M72.808 23.01c.254 1.075-.412 2.153-1.487 2.407L57.558 28.67c-1.075.254-2.152-.411-2.406-1.486s.411-2.153 1.486-2.407l9.17-2.167c-5.06-3.591-11.241-5.701-17.92-5.701-9.298 0-17.638 4.09-23.323 10.578-.728.83-1.992.914-2.822.186s-.914-1.992-.186-2.823C27.969 17.533 37.39 12.91 47.888 12.91c7.384 0 14.238 2.288 19.886 6.194l-2.112-8.935c-.255-1.075.41-2.153 1.486-2.407 1.075-.254 2.152.411 2.406 1.486zm4.827 15.674c-.33-1.055.258-2.177 1.312-2.506 1.054-.33 2.176.257 2.506 1.311 1.034 3.307 1.591 6.822 1.591 10.463 0 14.285-8.556 26.567-20.818 32.007l8.776 2.508c1.063.303 1.678 1.41 1.374 2.472s-1.41 1.677-2.472 1.374l-13.598-3.885c-1.062-.303-1.677-1.41-1.374-2.472l3.885-13.598c.303-1.062 1.41-1.678 2.472-1.374s1.677 1.41 1.374 2.472l-2.593 9.078c11.15-4.697 18.974-15.728 18.974-28.582 0-3.232-.494-6.344-1.409-9.268M13.324 43.23l-6.932 5.825c-.846.71-2.107.6-2.818-.245-.71-.846-.6-2.107.245-2.818l10.827-9.097c.846-.71 2.108-.601 2.818.244l9.098 10.828c.71.845.6 2.107-.245 2.818-.846.71-2.107.6-2.818-.245l-6.134-7.301c-.932 6.025-.1 12.383 2.773 18.282 4.071 8.36 11.4 14.066 19.722 16.337 1.066.29 1.694 1.39 1.403 2.456-.29 1.065-1.39 1.693-2.456 1.403-9.386-2.562-17.668-9.006-22.265-18.445-3.147-6.46-4.125-13.418-3.218-20.042"
                })
            }),
            /*#__PURE__*/ (0, $9RQDT$jsx)("g", {
                clipPath: "url(#df23e2__a)",
                children: /*#__PURE__*/ (0, $9RQDT$jsxs)("g", {
                    clipPath: "url(#df23e2__b)",
                    children: [
                        /*#__PURE__*/ (0, $9RQDT$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $9RQDT$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#df23e2__c)"
                        }),
                        /*#__PURE__*/ (0, $9RQDT$jsx)("path", {
                            fill: "url(#df23e2__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $9RQDT$jsx)("path", {
                            fill: "url(#df23e2__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $9RQDT$jsx)("path", {
                            fill: "url(#df23e2__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $066a6e84a70bdfe4$var$ForwardRef = /*#__PURE__*/ (0, $9RQDT$forwardRef)($066a6e84a70bdfe4$var$SvgComponent);
var $066a6e84a70bdfe4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($066a6e84a70bdfe4$var$ForwardRef);


export {$066a6e84a70bdfe4$export$2e2bcd8739ae039 as default};
