// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_effects_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $1eAR8$reactjsxruntime = require("react/jsx-runtime");
var $1eAR8$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $618caa4647225270$export$2e2bcd8739ae039);




const $618caa4647225270$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f48bd4__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f48bd4__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f48bd4__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f48bd4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("clipPath", {
                        id: "f48bd4__b",
                        children: /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("clipPath", {
                id: "f48bd4__a",
                children: /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M44.124 18.464c-2.279 1.736-3.867 4.464-4.779 6.97L36.013 36.29h17.183c.727 0 1.397.395 1.75 1.031l6.775 12.24 12.721-12.706c.782-.78 2.048-.78 2.828.002s.78 2.048-.001 2.829c-3.692 3.686-8.695 8.683-13.532 13.517l7.531 13.605c.535.967.185 2.184-.781 2.719s-2.183.185-2.718-.782l-6.976-12.6-1.97 1.969c-3.06 3.06-5.813 5.812-7.807 7.808l-2.363 2.368-.645.65-.165.168-.02.02c-.687.752-1.845.873-2.675.25-.884-.663-1.063-1.916-.4-2.8.05-.068.097-.12.103-.127l.001-.001.034-.037.035-.038.023-.024.038-.04.182-.183.654-.66 2.368-2.373c1.995-1.997 4.748-4.75 7.809-7.81l2.783-2.782-6.761-12.213H34.785L22.053 81.774c-.324 1.056-1.443 1.65-2.499 1.325s-1.649-1.443-1.325-2.499l12.372-40.31h-8.122c-1.105 0-2-.895-2-2s.895-2 2-2h9.35l3.707-12.078.03-.092c1.06-2.936 3.014-6.46 6.135-8.838 3.23-2.46 7.56-3.564 12.909-1.775 1.047.35 1.612 1.483 1.262 2.53-.35 1.048-1.483 1.614-2.531 1.263-4.084-1.366-7.033-.499-9.217 1.164m3.687 50.677.004-.004c-.026.03-.034.036-.004.004"
                })
            }),
            /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("g", {
                clipPath: "url(#f48bd4__a)",
                children: /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#f48bd4__b)",
                    children: [
                        /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#f48bd4__c)"
                        }),
                        /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("path", {
                            fill: "url(#f48bd4__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("path", {
                            fill: "url(#f48bd4__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $1eAR8$reactjsxruntime.jsx)("path", {
                            fill: "url(#f48bd4__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $618caa4647225270$var$ForwardRef = /*#__PURE__*/ (0, $1eAR8$react.forwardRef)($618caa4647225270$var$SvgComponent);
var $618caa4647225270$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($618caa4647225270$var$ForwardRef);
