// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_CodeBrackets_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $b7siJ$jsxs, jsx as $b7siJ$jsx} from "react/jsx-runtime";
import {forwardRef as $b7siJ$forwardRef} from "react";





const $045787fe26b83c0a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $b7siJ$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $b7siJ$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $b7siJ$jsxs)("linearGradient", {
                        id: "f1e65d__d",
                        x1: 109.277,
                        x2: 41.838,
                        y1: 19.392,
                        y2: 112.146,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b7siJ$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $b7siJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b7siJ$jsxs)("linearGradient", {
                        id: "f1e65d__e",
                        x1: 64.961,
                        x2: 77.463,
                        y1: -21.535,
                        y2: 63.345,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b7siJ$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $b7siJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b7siJ$jsxs)("linearGradient", {
                        id: "f1e65d__f",
                        x1: -96.944,
                        x2: 138.04,
                        y1: 148.659,
                        y2: 99.394,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b7siJ$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $b7siJ$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $b7siJ$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $b7siJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b7siJ$jsxs)("radialGradient", {
                        id: "f1e65d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 170.948 -181.344 0 130.355 101.038)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $b7siJ$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $b7siJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $b7siJ$jsx)("clipPath", {
                        id: "f1e65d__b",
                        children: /*#__PURE__*/ (0, $b7siJ$jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.199,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $b7siJ$jsx)("clipPath", {
                id: "f1e65d__a",
                children: /*#__PURE__*/ (0, $b7siJ$jsx)("path", {
                    fill: "#fff",
                    d: "M124 32H36c-8.84 0-16 7.16-16 16v64c0 8.84 7.16 16 16 16h88c8.84 0 16-7.16 16-16V48c0-8.84-7.16-16-16-16M62.75 96.31c1.51 1.62 1.42 4.15-.19 5.65-.77.72-1.75 1.08-2.73 1.08-1.07 0-2.14-.43-2.93-1.27l-18.39-19.7c-1.47-1.58-1.43-4.03.1-5.56L57 58.12c1.56-1.56 4.1-1.56 5.66 0s1.56 4.09 0 5.66L47.01 79.43 62.76 96.3zm29.62-43.33-17.08 56.36c-.52 1.73-2.11 2.84-3.83 2.84-.38 0-.77-.06-1.16-.17-2.11-.64-3.31-2.87-2.67-4.99l17.08-56.36c.64-2.11 2.87-3.31 4.99-2.67 2.11.64 3.31 2.87 2.67 4.99m29.02 30.5L103 101.87c-.78.78-1.8 1.17-2.83 1.17s-2.05-.39-2.83-1.17c-1.56-1.56-1.56-4.09 0-5.66l15.65-15.65-15.75-16.87c-1.51-1.62-1.42-4.15.19-5.65 1.61-1.51 4.15-1.42 5.65.19l18.39 19.7c1.47 1.58 1.43 4.03-.1 5.56z"
                })
            }),
            /*#__PURE__*/ (0, $b7siJ$jsx)("g", {
                clipPath: "url(#f1e65d__a)",
                children: /*#__PURE__*/ (0, $b7siJ$jsxs)("g", {
                    clipPath: "url(#f1e65d__b)",
                    children: [
                        /*#__PURE__*/ (0, $b7siJ$jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.199,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $b7siJ$jsx)("ellipse", {
                            cx: 130.355,
                            cy: 101.038,
                            fill: "url(#f1e65d__c)",
                            rx: 181.344,
                            ry: 170.948
                        }),
                        /*#__PURE__*/ (0, $b7siJ$jsx)("path", {
                            fill: "url(#f1e65d__d)",
                            d: "M7.095 60.284C5.08 19.359 38.638-15.356 82.051-17.257s80.24 29.735 82.256 70.66c2.016 40.923-31.543 75.639-74.956 77.54-43.413 1.9-80.24-29.735-82.256-70.66"
                        }),
                        /*#__PURE__*/ (0, $b7siJ$jsx)("path", {
                            fill: "url(#f1e65d__e)",
                            d: "M50.129-58.73c-9.794-5.713-22.202-5.748-32.032-.09L-80.218-2.233c-9.629 5.542-9.645 18.623-.03 24.186L17.42 78.46c9.814 5.678 22.223 5.671 32.03-.02l97.464-56.56c9.564-5.551 9.582-18.56.029-24.131z"
                        }),
                        /*#__PURE__*/ (0, $b7siJ$jsx)("path", {
                            fill: "url(#f1e65d__f)",
                            d: "M-44.215 79.83c20.264 0 38.62 7.731 51.91 20.204 26.58 24.988 48.615 25.009 75.195.064 13.29-12.537 31.668-20.29 51.955-20.29 40.551.022 73.401 30.904 73.401 69.027s-32.85 68.984-73.401 68.984c-20.332 0-38.71-7.774-52-20.311-26.557-24.903-48.57-24.881-75.15.107-13.29 12.494-31.646 20.204-51.91 20.204-40.529 0-73.378-30.883-73.378-68.984s32.85-69.005 73.378-69.005"
                        })
                    ]
                })
            })
        ]
    });
const $045787fe26b83c0a$var$ForwardRef = /*#__PURE__*/ (0, $b7siJ$forwardRef)($045787fe26b83c0a$var$SvgComponent);
var $045787fe26b83c0a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($045787fe26b83c0a$var$ForwardRef);


export {$045787fe26b83c0a$export$2e2bcd8739ae039 as default};
