// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_rotateCW_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $cSvMD$jsxs, jsx as $cSvMD$jsx} from "react/jsx-runtime";
import {forwardRef as $cSvMD$forwardRef} from "react";





const $6b53068258b71260$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cSvMD$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cSvMD$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cSvMD$jsxs)("linearGradient", {
                        id: "53bdda__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cSvMD$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $cSvMD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cSvMD$jsxs)("linearGradient", {
                        id: "53bdda__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cSvMD$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $cSvMD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cSvMD$jsxs)("linearGradient", {
                        id: "53bdda__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cSvMD$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cSvMD$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $cSvMD$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $cSvMD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cSvMD$jsxs)("radialGradient", {
                        id: "53bdda__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cSvMD$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cSvMD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cSvMD$jsx)("clipPath", {
                        id: "53bdda__b",
                        children: /*#__PURE__*/ (0, $cSvMD$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cSvMD$jsx)("clipPath", {
                id: "53bdda__a",
                children: /*#__PURE__*/ (0, $cSvMD$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M16.31 48.996c0-17.12 13.88-31 31-31 12.19 0 22.739 7.035 27.803 17.271l-9.734-3.13c-1.052-.338-2.178.24-2.516 1.292-.338 1.051.24 2.178 1.292 2.516l14.056 4.52c1.052.338 2.178-.24 2.516-1.292l4.52-14.056c.338-1.052-.24-2.178-1.292-2.517-1.051-.338-2.178.24-2.516 1.292l-2.952 9.181C72.693 21.75 60.909 13.996 47.311 13.996c-19.33 0-35 15.67-35 35s15.67 35 35 35c15.117 0 27.991-9.583 32.888-23 .379-1.038-.156-2.186-1.193-2.564-1.038-.38-2.186.155-2.565 1.193-4.338 11.888-15.746 20.371-29.13 20.371-17.121 0-31-13.88-31-31"
                })
            }),
            /*#__PURE__*/ (0, $cSvMD$jsx)("g", {
                clipPath: "url(#53bdda__a)",
                children: /*#__PURE__*/ (0, $cSvMD$jsxs)("g", {
                    clipPath: "url(#53bdda__b)",
                    children: [
                        /*#__PURE__*/ (0, $cSvMD$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $cSvMD$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#53bdda__c)"
                        }),
                        /*#__PURE__*/ (0, $cSvMD$jsx)("path", {
                            fill: "url(#53bdda__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $cSvMD$jsx)("path", {
                            fill: "url(#53bdda__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $cSvMD$jsx)("path", {
                            fill: "url(#53bdda__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $6b53068258b71260$var$ForwardRef = /*#__PURE__*/ (0, $cSvMD$forwardRef)($6b53068258b71260$var$SvgComponent);
var $6b53068258b71260$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6b53068258b71260$var$ForwardRef);


export {$6b53068258b71260$export$2e2bcd8739ae039 as default};
