// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_bell_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7NRkl$jsxs, jsx as $7NRkl$jsx} from "react/jsx-runtime";
import {forwardRef as $7NRkl$forwardRef} from "react";





const $414d8387c0b89aa9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7NRkl$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7NRkl$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7NRkl$jsxs)("linearGradient", {
                        id: "1ef97d__d",
                        x1: 33.286,
                        x2: 9.233,
                        y1: 1.897,
                        y2: 33.084,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7NRkl$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7NRkl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7NRkl$jsxs)("linearGradient", {
                        id: "1ef97d__e",
                        x1: 18.631,
                        x2: 23.271,
                        y1: -12.46,
                        y2: 17.237,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7NRkl$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7NRkl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7NRkl$jsxs)("linearGradient", {
                        id: "1ef97d__f",
                        x1: -34.908,
                        x2: 43.165,
                        y1: 47.245,
                        y2: 31.815,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7NRkl$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7NRkl$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7NRkl$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7NRkl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7NRkl$jsxs)("radialGradient", {
                        id: "1ef97d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 59.9687 -59.9688 0 40.258 30.54)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7NRkl$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7NRkl$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7NRkl$jsx)("clipPath", {
                        id: "1ef97d__b",
                        children: /*#__PURE__*/ (0, $7NRkl$jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.234,
                            y: 3.232,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7NRkl$jsx)("clipPath", {
                id: "1ef97d__a",
                children: /*#__PURE__*/ (0, $7NRkl$jsx)("path", {
                    fill: "#fff",
                    d: "M36.886 30.866c-1.588-2.495-3.224-5.98-3.346-9.86V17.94c0-5.268-4.271-9.54-9.54-9.54-5.27 0-9.54 4.272-9.54 9.54v3.066c-.122 3.88-1.759 7.365-3.347 9.86-1.084 1.705.13 3.934 2.15 3.934H20.4V36c0 1.989 1.612 3.6 3.6 3.6s3.6-1.611 3.6-3.6v-1.2h7.136c2.021 0 3.235-2.229 2.15-3.934"
                })
            }),
            /*#__PURE__*/ (0, $7NRkl$jsx)("g", {
                clipPath: "url(#1ef97d__a)",
                children: /*#__PURE__*/ (0, $7NRkl$jsxs)("g", {
                    clipPath: "url(#1ef97d__b)",
                    children: [
                        /*#__PURE__*/ (0, $7NRkl$jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.234,
                            y: 3.232,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7NRkl$jsx)("circle", {
                            cx: 40.258,
                            cy: 30.539,
                            r: 59.969,
                            fill: "url(#1ef97d__c)"
                        }),
                        /*#__PURE__*/ (0, $7NRkl$jsx)("path", {
                            fill: "url(#1ef97d__d)",
                            d: "M-.504 16.242c-.667-14.356 10.43-26.535 24.787-27.201S50.818-.53 51.484 13.828s-10.43 26.535-24.787 27.201S.162 30.6-.504 16.242"
                        }),
                        /*#__PURE__*/ (0, $7NRkl$jsx)("path", {
                            fill: "url(#1ef97d__e)",
                            d: "M13.726-25.508c-3.238-2.004-7.342-2.016-10.592-.032L-29.378-5.689c-3.184 1.944-3.19 6.533-.01 8.485L2.91 22.618c3.246 1.992 7.35 1.99 10.592-.007l32.23-19.84c3.164-1.948 3.17-6.511.01-8.466z"
                        }),
                        /*#__PURE__*/ (0, $7NRkl$jsx)("path", {
                            fill: "url(#1ef97d__f)",
                            d: "M-17.471 23.1c6.701 0 12.771 2.711 17.166 7.087 8.79 8.766 16.077 8.773 24.867.022 4.394-4.398 10.472-7.117 17.18-7.117 13.41.007 24.274 10.84 24.274 24.214s-10.864 24.2-24.273 24.2c-6.724 0-12.802-2.727-17.197-7.125-8.782-8.736-16.061-8.728-24.851.037-4.395 4.383-10.465 7.088-17.166 7.088-13.402 0-24.266-10.834-24.266-24.2S-30.873 23.1-17.47 23.1"
                        })
                    ]
                })
            })
        ]
    });
const $414d8387c0b89aa9$var$ForwardRef = /*#__PURE__*/ (0, $7NRkl$forwardRef)($414d8387c0b89aa9$var$SvgComponent);
var $414d8387c0b89aa9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($414d8387c0b89aa9$var$ForwardRef);


export {$414d8387c0b89aa9$export$2e2bcd8739ae039 as default};
