// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_effects_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4fv6A$reactjsxruntime = require("react/jsx-runtime");
var $4fv6A$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5fef272950f5f11e$export$2e2bcd8739ae039);




const $5fef272950f5f11e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f400a1__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f400a1__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f400a1__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f400a1__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("clipPath", {
                        id: "f400a1__b",
                        children: /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("clipPath", {
                id: "f400a1__a",
                children: /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M22.214 9.431c-1.086.827-1.858 2.14-2.304 3.365l-1.565 5.098h8.253c.455 0 .874.247 1.094.645l3.227 5.83 6.125-6.119c.489-.488 1.28-.487 1.768.001.488.489.487 1.28-.001 1.768-1.812 1.81-4.256 4.25-6.632 6.625l3.674 6.639c.334.604.116 1.364-.488 1.699s-1.365.115-1.7-.489l-3.326-6.01-.751.75c-1.53 1.53-2.907 2.906-3.903 3.904l-1.181 1.184-.322.325-.082.082-.006.007c-.43.467-1.152.542-1.67.153-.553-.414-.665-1.197-.25-1.75.032-.042.06-.074.064-.079l.013-.014.008-.01.021-.021.032-.033.092-.093q.116-.118.328-.33l1.184-1.187c.997-1 2.374-2.376 3.905-3.906l1.258-1.258-3.218-5.813h-8.283L11.266 40.96c-.203.66-.902 1.031-1.562.829-.66-.203-1.03-.902-.828-1.562l6.087-19.833H11.24c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h4.49l1.799-5.861q.009-.03.02-.058c.538-1.494 1.538-3.305 3.15-4.533 1.68-1.28 3.932-1.847 6.685-.926.655.22 1.008.928.79 1.582-.22.655-.928 1.008-1.583.79-1.963-.657-3.351-.239-4.377.543"
                })
            }),
            /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("g", {
                clipPath: "url(#f400a1__a)",
                children: /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#f400a1__b)",
                    children: [
                        /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#f400a1__c)"
                        }),
                        /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("path", {
                            fill: "url(#f400a1__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("path", {
                            fill: "url(#f400a1__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $4fv6A$reactjsxruntime.jsx)("path", {
                            fill: "url(#f400a1__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $5fef272950f5f11e$var$ForwardRef = /*#__PURE__*/ (0, $4fv6A$react.forwardRef)($5fef272950f5f11e$var$SvgComponent);
var $5fef272950f5f11e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5fef272950f5f11e$var$ForwardRef);
