// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_ribbon_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $3E9Mz$jsxs, jsx as $3E9Mz$jsx} from "react/jsx-runtime";
import {forwardRef as $3E9Mz$forwardRef} from "react";





const $d73b9b93c1b0d2ec$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3E9Mz$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3E9Mz$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3E9Mz$jsxs)("linearGradient", {
                        id: "79d72d__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3E9Mz$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3E9Mz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3E9Mz$jsxs)("linearGradient", {
                        id: "79d72d__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3E9Mz$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3E9Mz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3E9Mz$jsxs)("linearGradient", {
                        id: "79d72d__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3E9Mz$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3E9Mz$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3E9Mz$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3E9Mz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3E9Mz$jsxs)("radialGradient", {
                        id: "79d72d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3E9Mz$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3E9Mz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3E9Mz$jsx)("clipPath", {
                        id: "79d72d__b",
                        children: /*#__PURE__*/ (0, $3E9Mz$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3E9Mz$jsx)("clipPath", {
                id: "79d72d__a",
                children: /*#__PURE__*/ (0, $3E9Mz$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M34.75 18.25C34.75 24.187 29.937 29 24 29s-10.75-4.813-10.75-10.75S18.063 7.5 24 7.5s10.75 4.813 10.75 10.75m-7.325 20.118-3.158-7.87c2.847-.061 5.455-1.093 7.506-2.779l3.073 7.662c.295.734-.339 1.503-1.115 1.355l-2.245-.43c-.386-.073-.779.086-1.005.407l-1.31 1.859c-.457.648-1.45.532-1.746-.204M16.208 27.703c2.047 1.69 4.652 2.727 7.497 2.794l-3.112 7.767c-.297.742-1.303.852-1.754.191l-1.277-1.871c-.228-.334-.634-.498-1.029-.416l-2.231.462c-.78.161-1.428-.611-1.13-1.352z"
                })
            }),
            /*#__PURE__*/ (0, $3E9Mz$jsx)("g", {
                clipPath: "url(#79d72d__a)",
                children: /*#__PURE__*/ (0, $3E9Mz$jsxs)("g", {
                    clipPath: "url(#79d72d__b)",
                    children: [
                        /*#__PURE__*/ (0, $3E9Mz$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3E9Mz$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#79d72d__c)"
                        }),
                        /*#__PURE__*/ (0, $3E9Mz$jsx)("path", {
                            fill: "url(#79d72d__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $3E9Mz$jsx)("path", {
                            fill: "url(#79d72d__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $3E9Mz$jsx)("path", {
                            fill: "url(#79d72d__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $d73b9b93c1b0d2ec$var$ForwardRef = /*#__PURE__*/ (0, $3E9Mz$forwardRef)($d73b9b93c1b0d2ec$var$SvgComponent);
var $d73b9b93c1b0d2ec$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d73b9b93c1b0d2ec$var$ForwardRef);


export {$d73b9b93c1b0d2ec$export$2e2bcd8739ae039 as default};
