// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_noElements_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $ax5Nl$reactjsxruntime = require("react/jsx-runtime");
var $ax5Nl$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2492de924697a999$export$2e2bcd8739ae039);




const $2492de924697a999$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsxs)("linearGradient", {
                        id: "40fece__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsxs)("linearGradient", {
                        id: "40fece__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsxs)("linearGradient", {
                        id: "40fece__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsxs)("radialGradient", {
                        id: "40fece__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("clipPath", {
                        id: "40fece__b",
                        children: /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsxs)("clipPath", {
                id: "40fece__a",
                children: [
                    /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("path", {
                        fill: "#D9D9D9",
                        d: "M125.071 21.723c.926-1.07 2.534-1.967 3.524-1.663l.008.008c1.592 0 3.128.864 4.007 2.248.871 1.366 1.014 3.095.388 4.635l-5.393 13.196 7.824 3.441c1.418.618 2.455 1.919 2.788 3.474s-.079 3.195-1.109 4.38L116.676 75.02c-.918 1.062-2.17 1.647-3.508 1.647-1.592 0-3.128-.856-4.007-2.24-.871-1.358-1.022-3.095-.396-4.626l5.219-12.868-7.777-3.482c-1.402-.626-2.431-1.926-2.764-3.474s.079-3.17 1.093-4.355zM16.448 93.333c0-2.21 1.79-4 4-4h46.91c2.21 0 4 1.79 4 4v9.382c0 2.209-1.79 4-4 4s-4-1.791-4-4v-5.382H47.9v38.911h5.383c2.21 0 4 1.791 4 4s-1.79 4-4 4H34.52c-2.21 0-4-1.791-4-4s1.79-4 4-4h5.38V97.333H24.449v5.382c0 2.209-1.791 4-4 4s-4-1.791-4-4z"
                    }),
                    /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("path", {
                        fill: "#D9D9D9",
                        fillRule: "evenodd",
                        d: "M89.998 106.666v21.25c0 7.364 5.97 13.334 13.333 13.334h35.833c7.364 0 13.334-5.97 13.334-13.334v-21.25c0-7.363-5.97-13.333-13.334-13.333h-35.833c-7.364 0-13.333 5.97-13.333 13.333m27.791 1.747 12.296 7.019c1.377.787 1.375 2.748-.004 3.531l-12.313 6.99c-1.379.783-3.101-.2-3.099-1.769l.017-14.009c.002-1.569 1.726-2.548 3.103-1.762M11.664 57.916v-21.25c0-7.363 5.97-13.333 13.333-13.333h36.875c7.364 0 13.334 5.97 13.334 13.333v21.25c0 2.039-.458 3.97-1.276 5.699l-6.794-10.601c-3.935-6.139-12.904-6.139-16.839 0l-2.689 4.196-7.097-11.096c-3.931-6.146-12.907-6.15-16.844-.008L12.735 63.162c-.69-1.61-1.07-3.383-1.07-5.246M62.496 40c2.485 0 4.5-2.015 4.5-4.5s-2.015-4.5-4.5-4.5-4.5 2.015-4.5 4.5 2.015 4.5 4.5 4.5"
                    }),
                    /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("path", {
                        fill: "#D9D9D9",
                        d: "M42.855 64.626 33.77 50.424c-.786-1.229-2.581-1.23-3.368-.001l-12.17 18.986c1.983 1.17 4.296 1.84 6.764 1.84H38.61zM48.112 71.25h13.76c2.32 0 4.502-.593 6.402-1.635L60.4 57.331c-.786-1.228-2.58-1.228-3.367 0z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("g", {
                clipPath: "url(#40fece__a)",
                children: /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#40fece__b)",
                    children: [
                        /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#40fece__c)"
                        }),
                        /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("path", {
                            fill: "url(#40fece__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("path", {
                            fill: "url(#40fece__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $ax5Nl$reactjsxruntime.jsx)("path", {
                            fill: "url(#40fece__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $2492de924697a999$var$ForwardRef = /*#__PURE__*/ (0, $ax5Nl$react.forwardRef)($2492de924697a999$var$SvgComponent);
var $2492de924697a999$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2492de924697a999$var$ForwardRef);
