// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_data_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $a0mtC$jsxs, jsx as $a0mtC$jsx} from "react/jsx-runtime";
import {forwardRef as $a0mtC$forwardRef} from "react";





const $9688bc507674c163$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $a0mtC$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $a0mtC$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $a0mtC$jsxs)("linearGradient", {
                        id: "c713cd__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a0mtC$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $a0mtC$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a0mtC$jsxs)("linearGradient", {
                        id: "c713cd__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a0mtC$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $a0mtC$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a0mtC$jsxs)("linearGradient", {
                        id: "c713cd__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a0mtC$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $a0mtC$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $a0mtC$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $a0mtC$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a0mtC$jsxs)("radialGradient", {
                        id: "c713cd__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a0mtC$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $a0mtC$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a0mtC$jsx)("clipPath", {
                        id: "c713cd__b",
                        children: /*#__PURE__*/ (0, $a0mtC$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $a0mtC$jsx)("clipPath", {
                id: "c713cd__a",
                children: /*#__PURE__*/ (0, $a0mtC$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M35.97 17.517c1.21-.375 2.295-.822 3.182-1.343v2.233c-.02.146-.15.469-.776.928-.674.494-1.73.991-3.146 1.43C32.41 21.64 28.44 22.2 24 22.2s-8.412-.56-11.23-1.434c-1.418-.44-2.473-.937-3.148-1.431-.623-.457-.753-.779-.774-.925v-2.235c.886.52 1.971.967 3.18 1.342 3.12.967 7.352 1.546 11.971 1.546 4.62 0 8.852-.58 11.97-1.546m3.181-4.743c-.048-2.156-6.813-3.899-15.152-3.899S8.896 10.618 8.848 12.774c.021.146.152.468.774.924.675.494 1.73.992 3.147 1.431 2.82.874 6.79 1.434 11.23 1.434s8.412-.56 11.23-1.434c1.418-.44 2.473-.937 3.148-1.43.621-.457.752-.778.774-.925m0 9.037c-.886.521-1.971.968-3.181 1.343C32.85 24.12 28.62 24.7 24 24.7s-8.852-.58-11.971-1.546c-1.21-.375-2.295-.822-3.181-1.343v2.235c.02.147.151.47.774.926.675.494 1.73.991 3.147 1.43 2.82.875 6.79 1.434 11.23 1.434s8.412-.56 11.23-1.433c1.418-.44 2.473-.937 3.148-1.431.626-.46.754-.782.774-.928zM8.848 29.714v-2.266c.886.521 1.971.967 3.18 1.342 3.12.967 7.352 1.546 11.971 1.546 4.62 0 8.852-.579 11.97-1.546 1.21-.375 2.296-.821 3.182-1.342v2.264c-.02.146-.148.469-.775.928-.674.494-1.73.991-3.146 1.43-2.82.874-6.79 1.434-11.23 1.434s-8.412-.56-11.23-1.433c-1.418-.44-2.473-.937-3.148-1.431-.623-.457-.753-.78-.774-.926m0 3.402v2.212c.402 2 7.598 3.798 15.002 3.798 5.842 0 13.916-1.094 15.301-3.735v-2.275c-.886.52-1.971.967-3.181 1.342-3.119.967-7.35 1.546-11.97 1.546s-8.852-.579-11.971-1.546c-1.21-.375-2.295-.821-3.181-1.342"
                })
            }),
            /*#__PURE__*/ (0, $a0mtC$jsx)("g", {
                clipPath: "url(#c713cd__a)",
                children: /*#__PURE__*/ (0, $a0mtC$jsxs)("g", {
                    clipPath: "url(#c713cd__b)",
                    children: [
                        /*#__PURE__*/ (0, $a0mtC$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $a0mtC$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#c713cd__c)"
                        }),
                        /*#__PURE__*/ (0, $a0mtC$jsx)("path", {
                            fill: "url(#c713cd__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $a0mtC$jsx)("path", {
                            fill: "url(#c713cd__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $a0mtC$jsx)("path", {
                            fill: "url(#c713cd__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $9688bc507674c163$var$ForwardRef = /*#__PURE__*/ (0, $a0mtC$forwardRef)($9688bc507674c163$var$SvgComponent);
var $9688bc507674c163$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9688bc507674c163$var$ForwardRef);


export {$9688bc507674c163$export$2e2bcd8739ae039 as default};
