// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_pieChart_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $hsdEO$reactjsxruntime = require("react/jsx-runtime");
var $hsdEO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c8046f6753ca90f9$export$2e2bcd8739ae039);




const $c8046f6753ca90f9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0e26b3__d",
                        x1: 32.787,
                        x2: 12.555,
                        y1: 5.819,
                        y2: 33.645,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0e26b3__e",
                        x1: 19.493,
                        x2: 23.243,
                        y1: -6.459,
                        y2: 19.005,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0e26b3__f",
                        x1: -29.081,
                        x2: 41.415,
                        y1: 44.599,
                        y2: 29.819,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsxs)("radialGradient", {
                        id: "0e26b3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.2844 -54.4031 0 39.11 30.312)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("clipPath", {
                        id: "0e26b3__b",
                        children: /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.523,
                            y: 6.961,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("clipPath", {
                id: "0e26b3__a",
                children: /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M22.8 7.248c-3.735.264-7.134 1.749-9.798 4.059l9.798 9.798zm-11.494 5.754C8.75 15.948 7.2 19.792 7.2 24c0 4.21 1.551 8.05 4.107 10.998L22.304 24zM25.2 24V7.251c8.718.615 15.6 7.473 15.6 16.752s-7.521 16.8-16.8 16.8c-4.209 0-8.052-1.551-10.998-4.107l11.847-11.847.01-.01.048-.053q.049-.06.092-.123.039-.064.069-.132l.006-.012.01-.016q.012-.022.023-.047.024-.061.04-.125l.008-.031.003-.01.006-.017.012-.039q.023-.117.024-.234m-.036.273.012-.039-.01.027q-.006.019-.011.04.003-.015.009-.028"
                })
            }),
            /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("g", {
                clipPath: "url(#0e26b3__a)",
                children: /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#0e26b3__b)",
                    children: [
                        /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.523,
                            y: 6.961,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("ellipse", {
                            cx: 39.11,
                            cy: 30.312,
                            fill: "url(#0e26b3__c)",
                            rx: 54.403,
                            ry: 51.284
                        }),
                        /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("path", {
                            fill: "url(#0e26b3__d)",
                            d: "M2.132 18.087C1.527 5.809 11.595-4.607 24.62-5.177s24.072 8.92 24.677 21.198c.604 12.277-9.463 22.692-22.487 23.262S2.737 30.364 2.132 18.087"
                        }),
                        /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("path", {
                            fill: "url(#0e26b3__e)",
                            d: "M15.043-17.617c-2.938-1.714-6.66-1.725-9.61-.027L-24.06-.669c-2.889 1.662-2.893 5.587-.009 7.256L5.23 23.54c2.944 1.703 6.667 1.7 9.609-.006l29.24-16.968c2.868-1.666 2.874-5.568.008-7.24z"
                        }),
                        /*#__PURE__*/ (0, $hsdEO$reactjsxruntime.jsx)("path", {
                            fill: "url(#0e26b3__f)",
                            d: "M-13.262 23.95c6.08 0 11.586 2.32 15.573 6.06 7.974 7.497 14.585 7.504 22.559.02 3.987-3.76 9.5-6.087 15.586-6.087 12.166.007 22.02 9.272 22.02 20.708 0 11.437-9.854 20.695-22.02 20.695-6.1 0-11.613-2.332-15.6-6.093-7.967-7.47-14.571-7.464-22.545.032-3.987 3.748-9.494 6.061-15.573 6.061-12.158 0-22.013-9.264-22.013-20.695 0-11.43 9.855-20.701 22.013-20.701"
                        })
                    ]
                })
            })
        ]
    });
const $c8046f6753ca90f9$var$ForwardRef = /*#__PURE__*/ (0, $hsdEO$react.forwardRef)($c8046f6753ca90f9$var$SvgComponent);
var $c8046f6753ca90f9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c8046f6753ca90f9$var$ForwardRef);
