// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_apps_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2hR29$jsxs, jsx as $2hR29$jsx} from "react/jsx-runtime";
import {forwardRef as $2hR29$forwardRef} from "react";





const $f5805c1918ce8693$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2hR29$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2hR29$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2hR29$jsxs)("linearGradient", {
                        id: "2b3566__d",
                        x1: 108.785,
                        x2: 38.447,
                        y1: 18.029,
                        y2: 114.771,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2hR29$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2hR29$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2hR29$jsxs)("linearGradient", {
                        id: "2b3566__e",
                        x1: 62.565,
                        x2: 75.605,
                        y1: -24.658,
                        y2: 63.871,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2hR29$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2hR29$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2hR29$jsxs)("linearGradient", {
                        id: "2b3566__f",
                        x1: -106.301,
                        x2: 138.787,
                        y1: 152.856,
                        y2: 101.472,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2hR29$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2hR29$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2hR29$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2hR29$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2hR29$jsxs)("radialGradient", {
                        id: "2b3566__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 178.298 -189.141 0 130.774 103.185)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2hR29$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2hR29$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2hR29$jsx)("clipPath", {
                        id: "2b3566__b",
                        children: /*#__PURE__*/ (0, $2hR29$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 22,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2hR29$jsx)("clipPath", {
                id: "2b3566__a",
                children: /*#__PURE__*/ (0, $2hR29$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M90.309 21.68h31.97c8.836 0 16 7.163 16 16v31.97c0 8.836-7.164 16-16 16h-31.97c-8.837 0-16-7.164-16-16V37.68c0-8.837 7.163-16 16-16m-54.59 19.46h16.52c7.732 0 14 6.268 14 14v16.52c0 7.732-6.268 14-14 14h-16.52c-7.732 0-14-6.268-14-14V55.14c0-7.732 6.268-14 14-14m16.52 52.66h-16.52c-7.732 0-14 6.268-14 14v16.52c0 7.732 6.268 14 14 14h16.52c7.732 0 14-6.268 14-14V107.8c0-7.732-6.268-14-14-14m36.07 0h16.52c7.732 0 14 6.268 14 14v16.52c0 7.732-6.268 14-14 14h-16.52c-7.732 0-14-6.268-14-14V107.8c0-7.732 6.268-14 14-14"
                })
            }),
            /*#__PURE__*/ (0, $2hR29$jsx)("g", {
                clipPath: "url(#2b3566__a)",
                children: /*#__PURE__*/ (0, $2hR29$jsxs)("g", {
                    clipPath: "url(#2b3566__b)",
                    children: [
                        /*#__PURE__*/ (0, $2hR29$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 22,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2hR29$jsx)("ellipse", {
                            cx: 130.774,
                            cy: 103.185,
                            fill: "url(#2b3566__c)",
                            rx: 189.141,
                            ry: 178.298
                        }),
                        /*#__PURE__*/ (0, $2hR29$jsx)("path", {
                            fill: "url(#2b3566__d)",
                            d: "M2.21 60.679C.107 17.995 35.11-18.214 80.389-20.196s83.69 31.013 85.792 73.697-32.899 78.893-78.178 80.875c-45.28 1.982-83.69-31.014-85.793-73.697"
                        }),
                        /*#__PURE__*/ (0, $2hR29$jsx)("path", {
                            fill: "url(#2b3566__e)",
                            d: "M47.095-63.453c-10.215-5.958-23.156-5.995-33.409-.094L-88.856-4.527c-10.043 5.78-10.06 19.424-.03 25.227L12.977 79.636c10.237 5.922 23.179 5.915 33.407-.021L148.04 20.623c9.976-5.79 9.994-19.357.031-25.169z"
                        }),
                        /*#__PURE__*/ (0, $2hR29$jsx)("path", {
                            fill: "url(#2b3566__f)",
                            d: "M-51.305 81.067c21.136 0 40.28 8.064 54.142 21.073 27.723 26.062 50.706 26.084 78.429.067 13.861-13.076 33.03-21.162 54.189-21.162 42.295.022 76.557 32.233 76.557 71.994s-34.262 71.95-76.557 71.95c-21.207 0-40.375-8.108-54.237-21.184-27.699-25.973-50.658-25.951-78.38.111-13.862 13.031-33.007 21.073-54.143 21.073-42.27 0-76.533-32.21-76.533-71.95 0-39.739 34.262-71.972 76.533-71.972"
                        })
                    ]
                })
            })
        ]
    });
const $f5805c1918ce8693$var$ForwardRef = /*#__PURE__*/ (0, $2hR29$forwardRef)($f5805c1918ce8693$var$SvgComponent);
var $f5805c1918ce8693$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f5805c1918ce8693$var$ForwardRef);


export {$f5805c1918ce8693$export$2e2bcd8739ae039 as default};
