// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_remix_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $1dmnk$reactjsxruntime = require("react/jsx-runtime");
var $1dmnk$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $18c32becf1f24a4d$export$2e2bcd8739ae039);




const $18c32becf1f24a4d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c88459__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c88459__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c88459__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsxs)("radialGradient", {
                        id: "c88459__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("clipPath", {
                        id: "c88459__b",
                        children: /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("clipPath", {
                id: "c88459__a",
                children: /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m38.19 20.999 5.307-5.308c.488-.488.488-1.28 0-1.767L38.19 8.616c-.488-.488-1.28-.488-1.768 0s-.488 1.28 0 1.768l3.071 3.071h-1.788c-2.518 0-4.84.848-6.694 2.273q-.043.033-.082.069l-.035.026c-1.364 1.072-2.632 2.625-3.622 4.043-.994 1.425-1.773 2.81-2.14 3.618-.282.624-.983 2.109-1.91 3.668-.799 1.342-1.706 2.637-2.6 3.507q-.042.033-.083.07c-1.508 1.37-3.508 2.204-5.707 2.204-4.688 0-8.488-3.8-8.488-8.49 0-4.687 3.8-8.488 8.488-8.488 2.867 0 5.403 1.42 6.941 3.6l.005.007.451.63c.402.562 1.182.692 1.744.29.562-.4.691-1.182.29-1.743l-.45-.628c-1.987-2.814-5.268-4.656-8.98-4.656-6.07 0-10.99 4.92-10.99 10.989s4.92 10.989 10.99 10.989c2.512 0 4.83-.845 6.68-2.264q.082-.046.158-.106c1.434-1.131 2.728-3 3.7-4.634.992-1.668 1.736-3.245 2.039-3.913.288-.637.984-1.889 1.912-3.22.934-1.337 2.036-2.658 3.116-3.506l.072-.06.025-.02c1.432-1.1 3.222-1.755 5.17-1.755h1.993l-3.276 3.276c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0m2.214 14.434c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.7c-2.798 0-5.077-1.609-7.09-3.803l-.527-.638c-.44-.532-1.228-.606-1.76-.166s-.606 1.228-.166 1.76l.547.662.02.024.02.023c2.187 2.39 5.095 4.638 8.957 4.638z"
                })
            }),
            /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("g", {
                clipPath: "url(#c88459__a)",
                children: /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#c88459__b)",
                    children: [
                        /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#c88459__c)"
                        }),
                        /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("path", {
                            fill: "url(#c88459__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("path", {
                            fill: "url(#c88459__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $1dmnk$reactjsxruntime.jsx)("path", {
                            fill: "url(#c88459__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $18c32becf1f24a4d$var$ForwardRef = /*#__PURE__*/ (0, $1dmnk$react.forwardRef)($18c32becf1f24a4d$var$SvgComponent);
var $18c32becf1f24a4d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($18c32becf1f24a4d$var$ForwardRef);
