// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_filmstrip_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $55X5P$jsxs, jsx as $55X5P$jsx} from "react/jsx-runtime";
import {forwardRef as $55X5P$forwardRef} from "react";





const $15a298dc83baf558$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $55X5P$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $55X5P$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $55X5P$jsxs)("linearGradient", {
                        id: "b45755__d",
                        x1: 110.677,
                        x2: 30.097,
                        y1: 5.525,
                        y2: 110,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $55X5P$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $55X5P$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $55X5P$jsxs)("linearGradient", {
                        id: "b45755__e",
                        x1: 61.581,
                        x2: 77.126,
                        y1: -42.571,
                        y2: 56.914,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $55X5P$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $55X5P$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $55X5P$jsxs)("linearGradient", {
                        id: "b45755__f",
                        x1: -117.778,
                        x2: 143.767,
                        y1: 157.439,
                        y2: 105.749,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $55X5P$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $55X5P$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $55X5P$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $55X5P$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $55X5P$jsxs)("radialGradient", {
                        id: "b45755__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.277 117.75)scale(200.895)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $55X5P$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $55X5P$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $55X5P$jsx)("clipPath", {
                        id: "b45755__b",
                        children: /*#__PURE__*/ (0, $55X5P$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10,
                            y: 10,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $55X5P$jsx)("clipPath", {
                id: "b45755__a",
                children: /*#__PURE__*/ (0, $55X5P$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M36 34c-8.837 0-16 7.163-16 16v60c0 8.837 7.163 16 16 16h88c8.837 0 16-7.163 16-16V50c0-8.837-7.163-16-16-16zm-4 19c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm0 49c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm17-49c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm0 49c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm17-49c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm0 49c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm17-49c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm0 49c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm17-49c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm0 49c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm17-49c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2zm0 49c0-1.105.895-2 2-2h7c1.105 0 2 .895 2 2v5c0 1.105-.895 2-2 2h-7c-1.105 0-2-.895-2-2z"
                })
            }),
            /*#__PURE__*/ (0, $55X5P$jsx)("g", {
                clipPath: "url(#b45755__a)",
                children: /*#__PURE__*/ (0, $55X5P$jsxs)("g", {
                    clipPath: "url(#b45755__b)",
                    children: [
                        /*#__PURE__*/ (0, $55X5P$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10,
                            y: 10,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $55X5P$jsx)("circle", {
                            cx: 134.028,
                            cy: 101.473,
                            r: 200.895,
                            fill: "url(#b45755__c)"
                        }),
                        /*#__PURE__*/ (0, $55X5P$jsx)("path", {
                            fill: "url(#b45755__d)",
                            d: "M-2.522 53.58C-4.755 5.487 32.422-35.31 80.516-37.544S169.407-2.6 171.64 45.493s-34.944 88.892-83.037 91.125S-.288 101.674-2.522 53.58"
                        }),
                        /*#__PURE__*/ (0, $55X5P$jsx)("path", {
                            fill: "url(#b45755__e)",
                            d: "M45.149-86.282c-10.85-6.713-24.595-6.754-35.485-.106l-108.915 66.5c-10.667 6.513-10.685 21.886-.032 28.424L8.912 74.942c10.874 6.672 24.62 6.664 35.484-.024L152.368 8.45c10.596-6.524 10.615-21.81.032-28.36z"
                        }),
                        /*#__PURE__*/ (0, $55X5P$jsx)("path", {
                            fill: "url(#b45755__f)",
                            d: "M-59.364 76.552c22.449 0 42.784 9.086 57.506 23.744 29.446 29.365 53.858 29.39 83.303.075 14.723-14.733 35.083-23.844 57.557-23.844 44.923.025 81.314 36.318 81.314 81.119s-36.391 81.068-81.314 81.068c-22.525 0-42.885-9.136-57.607-23.868-29.42-29.265-53.807-29.24-83.253.125-14.722 14.683-35.057 23.743-57.506 23.743-44.898 0-81.29-36.292-81.29-81.068s36.392-81.094 81.29-81.094"
                        })
                    ]
                })
            })
        ]
    });
const $15a298dc83baf558$var$ForwardRef = /*#__PURE__*/ (0, $55X5P$forwardRef)($15a298dc83baf558$var$SvgComponent);
var $15a298dc83baf558$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($15a298dc83baf558$var$ForwardRef);


export {$15a298dc83baf558$export$2e2bcd8739ae039 as default};
