// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_data_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $g4O6t$jsxs, jsx as $g4O6t$jsx} from "react/jsx-runtime";
import {forwardRef as $g4O6t$forwardRef} from "react";





const $42b4a54b12f9ffce$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $g4O6t$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $g4O6t$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $g4O6t$jsxs)("linearGradient", {
                        id: "3dd061__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g4O6t$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $g4O6t$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g4O6t$jsxs)("linearGradient", {
                        id: "3dd061__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g4O6t$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $g4O6t$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g4O6t$jsxs)("linearGradient", {
                        id: "3dd061__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g4O6t$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $g4O6t$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $g4O6t$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $g4O6t$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g4O6t$jsxs)("radialGradient", {
                        id: "3dd061__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g4O6t$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $g4O6t$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g4O6t$jsx)("clipPath", {
                        id: "3dd061__b",
                        children: /*#__PURE__*/ (0, $g4O6t$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $g4O6t$jsx)("clipPath", {
                id: "3dd061__a",
                children: /*#__PURE__*/ (0, $g4O6t$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m78.803 25.5.002-.078c0-4.513-13.792-8.172-30.805-8.172s-30.805 3.659-30.805 8.172q0 .039.002.078h-.002v.032c.049.478.415 1.25 1.755 2.233 1.411 1.034 3.58 2.049 6.442 2.936C31.09 32.467 39.083 33.59 48 33.59s16.91-1.124 22.609-2.89c2.86-.887 5.03-1.902 6.44-2.936 1.34-.982 1.706-1.753 1.756-2.232V25.5zM17.195 36.838v-5.424c1.845 1.215 4.251 2.251 7.012 3.107 6.177 1.916 14.59 3.07 23.793 3.07 9.204 0 17.616-1.154 23.793-3.07 2.761-.856 5.167-1.892 7.012-3.107v5.426c-.05.479-.416 1.25-1.755 2.231-1.41 1.034-3.58 2.05-6.441 2.936-5.698 1.767-13.692 2.89-22.609 2.89s-16.91-1.123-22.608-2.89c-2.861-.887-5.03-1.902-6.442-2.936-1.34-.982-1.706-1.754-1.755-2.233m61.61 5.883c-1.845 1.215-4.25 2.25-7.012 3.107-6.177 1.915-14.589 3.07-23.793 3.07s-17.615-1.155-23.793-3.07c-2.761-.856-5.167-1.892-7.012-3.107v5.393c.049.478.415 1.25 1.755 2.233 1.411 1.034 3.58 2.049 6.442 2.936 5.698 1.766 13.691 2.89 22.608 2.89s16.91-1.124 22.609-2.89c2.86-.887 5.03-1.902 6.44-2.937 1.34-.98 1.706-1.752 1.756-2.23zm-61.61 16.727v-5.452c1.845 1.215 4.251 2.251 7.012 3.107 6.177 1.916 14.59 3.07 23.793 3.07 9.204 0 17.616-1.154 23.793-3.07 2.761-.856 5.167-1.891 7.012-3.107v5.453c-.05.48-.416 1.25-1.755 2.232-1.41 1.034-3.58 2.049-6.441 2.936-5.698 1.766-13.692 2.89-22.609 2.89s-16.91-1.124-22.608-2.89c-2.861-.887-5.03-1.902-6.442-2.937-1.34-.982-1.706-1.754-1.755-2.233m0 5.882v5.498c1.5 4.484 15.448 7.922 30.5 7.922 11.879 0 28.375-2.375 31.11-7.922V65.33c-1.845 1.216-4.25 2.251-7.012 3.107-6.177 1.916-14.589 3.07-23.793 3.07s-17.615-1.154-23.793-3.07c-2.761-.856-5.167-1.891-7.012-3.107"
                })
            }),
            /*#__PURE__*/ (0, $g4O6t$jsx)("g", {
                clipPath: "url(#3dd061__a)",
                children: /*#__PURE__*/ (0, $g4O6t$jsxs)("g", {
                    clipPath: "url(#3dd061__b)",
                    children: [
                        /*#__PURE__*/ (0, $g4O6t$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $g4O6t$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#3dd061__c)"
                        }),
                        /*#__PURE__*/ (0, $g4O6t$jsx)("path", {
                            fill: "url(#3dd061__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $g4O6t$jsx)("path", {
                            fill: "url(#3dd061__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $g4O6t$jsx)("path", {
                            fill: "url(#3dd061__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $42b4a54b12f9ffce$var$ForwardRef = /*#__PURE__*/ (0, $g4O6t$forwardRef)($42b4a54b12f9ffce$var$SvgComponent);
var $42b4a54b12f9ffce$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($42b4a54b12f9ffce$var$ForwardRef);


export {$42b4a54b12f9ffce$export$2e2bcd8739ae039 as default};
