// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_cardTapPayment_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $9o4L2$reactjsxruntime = require("react/jsx-runtime");
var $9o4L2$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3d9373e69d93e887$export$2e2bcd8739ae039);




const $3d9373e69d93e887$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6f1835__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6f1835__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6f1835__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6f1835__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("clipPath", {
                        id: "6f1835__b",
                        children: /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("clipPath", {
                id: "6f1835__a",
                children: /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M68.322 15.752c.134-1.097 1.132-1.877 2.228-1.743 6.553.8 12.672 4.565 16.23 10.728 3.588 6.214 3.763 13.463 1.112 19.57-.44 1.012-1.617 1.477-2.63 1.038-1.014-.44-1.479-1.618-1.039-2.631 2.168-4.994 2.02-10.906-.906-15.977-2.904-5.029-7.893-8.103-13.252-8.757-1.096-.134-1.876-1.132-1.743-2.228m-.75 8.334c.135-1.096 1.132-1.877 2.228-1.743 3.982.486 7.704 2.775 9.868 6.523 2.181 3.778 2.286 8.186.676 11.897-.44 1.013-1.618 1.478-2.631 1.038s-1.478-1.618-1.038-2.63c1.127-2.6 1.05-5.67-.471-8.305-1.51-2.614-4.1-4.212-6.888-4.552-1.097-.134-1.877-1.132-1.743-2.228m-57.27 20.878c-3.826 2.209-5.137 7.102-2.928 10.928l1.211 2.098 53.978-31.106-1.236-2.142c-2.21-3.826-7.102-5.137-10.928-2.928zm54.261-14.616L10.585 61.454l9.555 16.55c2.21 3.827 7.102 5.138 10.929 2.928l40.096-23.15c3.827-2.208 5.138-7.101 2.928-10.928zM19.19 64.122c-.553-.956-.225-2.18.732-2.732l19.355-11.176c.956-.553 2.18-.225 2.732.732.552.956.225 2.18-.732 2.732L21.922 64.854c-.957.552-2.18.225-2.732-.732m4.923 4.526c-.956.552-1.284 1.775-.732 2.732.552.956 1.776 1.284 2.732.732l9.678-5.588c.956-.553 1.284-1.776.732-2.732-.553-.957-1.776-1.285-2.732-.732zm25.93-22.338c-.552-.957-.224-2.18.732-2.733l9.074-5.24c.957-.552 2.18-.224 2.732.733.553.956.225 2.18-.732 2.732l-9.074 5.24c-.956.552-2.18.224-2.732-.733"
                })
            }),
            /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6f1835__a)",
                children: /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6f1835__b)",
                    children: [
                        /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#6f1835__c)"
                        }),
                        /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("path", {
                            fill: "url(#6f1835__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("path", {
                            fill: "url(#6f1835__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $9o4L2$reactjsxruntime.jsx)("path", {
                            fill: "url(#6f1835__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $3d9373e69d93e887$var$ForwardRef = /*#__PURE__*/ (0, $9o4L2$react.forwardRef)($3d9373e69d93e887$var$SvgComponent);
var $3d9373e69d93e887$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3d9373e69d93e887$var$ForwardRef);
