// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_volume_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4qUv8$reactjsxruntime = require("react/jsx-runtime");
var $4qUv8$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1c451bd8ef45e3b0$export$2e2bcd8739ae039);




const $1c451bd8ef45e3b0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bde72c__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bde72c__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bde72c__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsxs)("radialGradient", {
                        id: "bde72c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("clipPath", {
                        id: "bde72c__b",
                        children: /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("clipPath", {
                id: "bde72c__a",
                children: /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M28.844 32.052h-8.38c-2.208 0-4 1.791-4 4v24.022c0 2.21 1.792 4 4 4h8.391q.087 0 .172.004v-32.03q-.09.004-.184.004m4.183-2.515v37.028l10.305 10.114c1.898 1.863 5.102.518 5.102-2.14V21.46c0-2.666-3.219-4.008-5.113-2.13zm22.383 5.235c.664-.883 1.918-1.06 2.8-.395 4.11 3.092 6.774 8.017 6.774 13.564 0 5.592-2.707 10.553-6.874 13.64-.888.658-2.14.472-2.798-.416s-.471-2.14.416-2.797c3.193-2.366 5.256-6.155 5.256-10.427 0-4.237-2.03-7.999-5.178-10.368-.883-.665-1.06-1.918-.396-2.801m12.805-9.524c-.882-.665-2.136-.488-2.8.395-.665.883-.488 2.137.395 2.8 5.91 4.448 9.725 11.515 9.725 19.473 0 8.023-3.878 15.14-9.87 19.58-.888.658-1.074 1.91-.416 2.798.657.888 1.91 1.074 2.797.416 6.967-5.162 11.489-13.45 11.489-22.794 0-9.268-4.45-17.498-11.32-22.668"
                })
            }),
            /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("g", {
                clipPath: "url(#bde72c__a)",
                children: /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#bde72c__b)",
                    children: [
                        /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#bde72c__c)"
                        }),
                        /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("path", {
                            fill: "url(#bde72c__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("path", {
                            fill: "url(#bde72c__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $4qUv8$reactjsxruntime.jsx)("path", {
                            fill: "url(#bde72c__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $1c451bd8ef45e3b0$var$ForwardRef = /*#__PURE__*/ (0, $4qUv8$react.forwardRef)($1c451bd8ef45e3b0$var$SvgComponent);
var $1c451bd8ef45e3b0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1c451bd8ef45e3b0$var$ForwardRef);
