// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_shapes_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $hza99$jsxs, jsx as $hza99$jsx} from "react/jsx-runtime";
import {forwardRef as $hza99$forwardRef} from "react";





const $bc6370db9cfd4bc8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hza99$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hza99$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hza99$jsxs)("linearGradient", {
                        id: "aa2afb__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hza99$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $hza99$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hza99$jsxs)("linearGradient", {
                        id: "aa2afb__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hza99$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $hza99$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hza99$jsxs)("linearGradient", {
                        id: "aa2afb__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hza99$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hza99$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $hza99$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $hza99$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hza99$jsxs)("radialGradient", {
                        id: "aa2afb__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hza99$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hza99$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hza99$jsx)("clipPath", {
                        id: "aa2afb__b",
                        children: /*#__PURE__*/ (0, $hza99$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hza99$jsx)("clipPath", {
                id: "aa2afb__a",
                children: /*#__PURE__*/ (0, $hza99$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M16.453 9.195 7.68 24.296c-.775 1.334.187 3.005 1.73 3.005h17.523q.144 0 .282-.02c1.372-.188 2.172-1.733 1.448-2.983l-4.378-7.555-4.372-7.546c-.77-1.33-2.689-1.33-3.46-.002m12.2 30.608c-5.935 0-10.858-4.33-11.785-10.002h10.065c3.467 0 5.632-3.756 3.893-6.756l-4.044-6.98q.916-.145 1.871-.146c6.596 0 11.942 5.346 11.942 11.942 0 6.595-5.346 11.942-11.942 11.942"
                })
            }),
            /*#__PURE__*/ (0, $hza99$jsx)("g", {
                clipPath: "url(#aa2afb__a)",
                children: /*#__PURE__*/ (0, $hza99$jsxs)("g", {
                    clipPath: "url(#aa2afb__b)",
                    children: [
                        /*#__PURE__*/ (0, $hza99$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $hza99$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#aa2afb__c)"
                        }),
                        /*#__PURE__*/ (0, $hza99$jsx)("path", {
                            fill: "url(#aa2afb__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $hza99$jsx)("path", {
                            fill: "url(#aa2afb__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $hza99$jsx)("path", {
                            fill: "url(#aa2afb__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $bc6370db9cfd4bc8$var$ForwardRef = /*#__PURE__*/ (0, $hza99$forwardRef)($bc6370db9cfd4bc8$var$SvgComponent);
var $bc6370db9cfd4bc8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($bc6370db9cfd4bc8$var$ForwardRef);


export {$bc6370db9cfd4bc8$export$2e2bcd8739ae039 as default};
