// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lockOpen_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $cNszj$jsxs, jsx as $cNszj$jsx} from "react/jsx-runtime";
import {forwardRef as $cNszj$forwardRef} from "react";





const $dda6bb3c94099e7e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cNszj$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cNszj$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cNszj$jsxs)("linearGradient", {
                        id: "caac42__d",
                        x1: 111.107,
                        x2: 30.526,
                        y1: 5.955,
                        y2: 110.43,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cNszj$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $cNszj$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cNszj$jsxs)("linearGradient", {
                        id: "caac42__e",
                        x1: 62.011,
                        x2: 77.555,
                        y1: -42.141,
                        y2: 57.344,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cNszj$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $cNszj$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cNszj$jsxs)("linearGradient", {
                        id: "caac42__f",
                        x1: -117.348,
                        x2: 144.197,
                        y1: 157.869,
                        y2: 106.178,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cNszj$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cNszj$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $cNszj$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $cNszj$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cNszj$jsxs)("radialGradient", {
                        id: "caac42__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.277 118.181)scale(200.895)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cNszj$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cNszj$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cNszj$jsx)("clipPath", {
                        id: "caac42__b",
                        children: /*#__PURE__*/ (0, $cNszj$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.43,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cNszj$jsx)("clipPath", {
                id: "caac42__a",
                children: /*#__PURE__*/ (0, $cNszj$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M54 45.5c0-14.36 11.64-26 26-26s26 11.64 26 26c0 2.21-1.791 4-4 4s-4-1.79-4-4c0-9.941-8.059-18-18-18s-18 8.059-18 18v21h53.636c7.381 0 13.364 5.457 13.364 12.188v40.624c0 6.731-5.983 12.188-13.364 12.188H44.364c-7.38 0-13.364-5.457-13.364-12.188V78.688C31 71.957 36.983 66.5 44.364 66.5H54zm20.346 55.153c.493.497 1.051.924 1.654 1.271v7.576c0 2.209 1.79 4 4 4s4-1.791 4-4v-7.576c.603-.347 1.16-.774 1.654-1.271.372-.36.705-.773 1-1.214.293-.426.532-.893.733-1.373.199-.493.36-.986.453-1.506.107-.507.16-1.04.16-1.56s-.053-1.053-.16-1.56c-.094-.507-.254-1.014-.453-1.493-.2-.494-.44-.947-.733-1.387-.295-.44-.628-.854-1-1.214C84.174 87.853 82.107 87 80 87s-4.173.853-5.654 2.346c-.373.36-.707.774-1 1.214-.28.44-.534.893-.733 1.387-.2.48-.36.986-.453 1.492-.107.508-.16 1.04-.16 1.561s.053 1.053.16 1.56c.093.52.253 1.013.453 1.506.2.48.453.947.733 1.373.293.44.627.854 1 1.214"
                })
            }),
            /*#__PURE__*/ (0, $cNszj$jsx)("g", {
                clipPath: "url(#caac42__a)",
                children: /*#__PURE__*/ (0, $cNszj$jsxs)("g", {
                    clipPath: "url(#caac42__b)",
                    children: [
                        /*#__PURE__*/ (0, $cNszj$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.43,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $cNszj$jsx)("circle", {
                            cx: 134.458,
                            cy: 101.904,
                            r: 200.895,
                            fill: "url(#caac42__c)"
                        }),
                        /*#__PURE__*/ (0, $cNszj$jsx)("path", {
                            fill: "url(#caac42__d)",
                            d: "M-2.092 54.01C-4.325 5.917 32.852-34.88 80.945-37.114S169.837-2.17 172.07 45.923s-34.944 88.891-83.038 91.124C40.94 139.281.142 102.104-2.092 54.01"
                        }),
                        /*#__PURE__*/ (0, $cNszj$jsx)("path", {
                            fill: "url(#caac42__e)",
                            d: "M45.579-85.852c-10.85-6.714-24.595-6.755-35.485-.106l-108.915 66.5c-10.667 6.513-10.685 21.886-.032 28.424L9.341 75.37c10.873 6.673 24.62 6.665 35.483-.023L152.798 8.88c10.596-6.524 10.615-21.811.032-28.36z"
                        }),
                        /*#__PURE__*/ (0, $cNszj$jsx)("path", {
                            fill: "url(#caac42__f)",
                            d: "M-58.935 76.982c22.45 0 42.784 9.086 57.507 23.743 29.445 29.366 53.857 29.391 83.303.076 14.722-14.733 35.082-23.844 57.557-23.844 44.923.025 81.314 36.318 81.314 81.119s-36.391 81.068-81.314 81.068c-22.525 0-42.885-9.136-57.608-23.869-29.42-29.265-53.807-29.24-83.252.126-14.723 14.682-35.058 23.743-57.507 23.743-44.897 0-81.289-36.293-81.289-81.068 0-44.776 36.392-81.094 81.29-81.094"
                        })
                    ]
                })
            })
        ]
    });
const $dda6bb3c94099e7e$var$ForwardRef = /*#__PURE__*/ (0, $cNszj$forwardRef)($dda6bb3c94099e7e$var$SvgComponent);
var $dda6bb3c94099e7e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($dda6bb3c94099e7e$var$ForwardRef);


export {$dda6bb3c94099e7e$export$2e2bcd8739ae039 as default};
