// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_paperairplane_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $dIQrA$jsxs, jsx as $dIQrA$jsx} from "react/jsx-runtime";
import {forwardRef as $dIQrA$forwardRef} from "react";





const $32f25613dbe4482c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dIQrA$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dIQrA$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dIQrA$jsxs)("linearGradient", {
                        id: "d49c59__d",
                        x1: 33.541,
                        x2: 11.28,
                        y1: 3.461,
                        y2: 34.078,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dIQrA$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dIQrA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dIQrA$jsxs)("linearGradient", {
                        id: "d49c59__e",
                        x1: 18.914,
                        x2: 23.041,
                        y1: -10.048,
                        y2: 17.969,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dIQrA$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dIQrA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dIQrA$jsxs)("linearGradient", {
                        id: "d49c59__f",
                        x1: -34.531,
                        x2: 43.033,
                        y1: 46.131,
                        y2: 29.869,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dIQrA$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dIQrA$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dIQrA$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dIQrA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dIQrA$jsxs)("radialGradient", {
                        id: "d49c59__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 56.4273 -59.8586 0 40.5 30.41)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dIQrA$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dIQrA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dIQrA$jsx)("clipPath", {
                        id: "d49c59__b",
                        children: /*#__PURE__*/ (0, $dIQrA$jsx)("rect", {
                            width: 41.459,
                            height: 39.082,
                            x: 3.543,
                            y: 4.718,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dIQrA$jsx)("clipPath", {
                id: "d49c59__a",
                children: /*#__PURE__*/ (0, $dIQrA$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M41.481 11.755c.493-.11.96-.048 1.36.134L11.746 28.035q-.147.077-.263.185l-6.658-4.555c-1.535-1.05-1.077-3.412.74-3.822zm-13.034 28.07L13.69 29.73l30.361-15.765q.023-.011.044-.024c-.016.279-.09.564-.23.842L31.57 39.031c-.584 1.15-2.056 1.525-3.124.794m-16.149-.43V31.78l7.048 4.802-4.66 3.877c-.938.78-2.388.134-2.388-1.064"
                })
            }),
            /*#__PURE__*/ (0, $dIQrA$jsx)("g", {
                clipPath: "url(#d49c59__a)",
                children: /*#__PURE__*/ (0, $dIQrA$jsxs)("g", {
                    clipPath: "url(#d49c59__b)",
                    children: [
                        /*#__PURE__*/ (0, $dIQrA$jsx)("rect", {
                            width: 41.459,
                            height: 39.082,
                            x: 3.543,
                            y: 4.718,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dIQrA$jsx)("ellipse", {
                            cx: 40.499,
                            cy: 30.411,
                            fill: "url(#d49c59__c)",
                            rx: 59.859,
                            ry: 56.427
                        }),
                        /*#__PURE__*/ (0, $dIQrA$jsx)("path", {
                            fill: "url(#d49c59__d)",
                            d: "M-.188 16.959C-.853 3.45 10.224-8.01 24.554-8.636S51.04 1.179 51.705 14.687c.666 13.509-10.411 24.968-24.741 25.595C12.634 40.91.478 30.467-.188 16.96"
                        }),
                        /*#__PURE__*/ (0, $dIQrA$jsx)("path", {
                            fill: "url(#d49c59__e)",
                            d: "M14.018-22.326c-3.233-1.885-7.329-1.897-10.573-.03L-29.008-3.677c-3.178 1.83-3.183 6.147-.01 7.984L3.222 22.959c3.24 1.874 7.335 1.872 10.572-.007l32.172-18.67c3.157-1.832 3.162-6.126.01-7.965z"
                        }),
                        /*#__PURE__*/ (0, $dIQrA$jsx)("path", {
                            fill: "url(#d49c59__f)",
                            d: "M-17.126 23.411c6.688 0 12.747 2.552 17.134 6.67 8.774 8.247 16.048 8.255 24.821.02 4.387-4.138 10.453-6.697 17.15-6.697 13.385.007 24.228 10.201 24.228 22.785s-10.843 22.77-24.228 22.77c-6.712 0-12.778-2.566-17.165-6.704-8.766-8.22-16.032-8.213-24.806.035-4.387 4.124-10.446 6.67-17.134 6.67-13.378 0-24.221-10.195-24.221-22.771s10.843-22.778 24.22-22.778"
                        })
                    ]
                })
            })
        ]
    });
const $32f25613dbe4482c$var$ForwardRef = /*#__PURE__*/ (0, $dIQrA$forwardRef)($32f25613dbe4482c$var$SvgComponent);
var $32f25613dbe4482c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($32f25613dbe4482c$var$ForwardRef);


export {$32f25613dbe4482c$export$2e2bcd8739ae039 as default};
