// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_speedFast_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2Or0E$jsxs, jsx as $2Or0E$jsx} from "react/jsx-runtime";
import {forwardRef as $2Or0E$forwardRef} from "react";





const $aa4b4cef897d5703$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2Or0E$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2Or0E$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2Or0E$jsxs)("linearGradient", {
                        id: "a6fb6d__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2Or0E$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2Or0E$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2Or0E$jsxs)("linearGradient", {
                        id: "a6fb6d__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2Or0E$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2Or0E$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2Or0E$jsxs)("linearGradient", {
                        id: "a6fb6d__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2Or0E$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2Or0E$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2Or0E$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2Or0E$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2Or0E$jsxs)("radialGradient", {
                        id: "a6fb6d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2Or0E$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2Or0E$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2Or0E$jsx)("clipPath", {
                        id: "a6fb6d__b",
                        children: /*#__PURE__*/ (0, $2Or0E$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2Or0E$jsx)("clipPath", {
                id: "a6fb6d__a",
                children: /*#__PURE__*/ (0, $2Or0E$jsx)("path", {
                    fill: "#D9D9D9",
                    d: "M47.998 32.778c-1.105 0-2-.896-2-2v-7.012q0-.17.027-.332c-4.599.273-8.942 1.494-12.838 3.471q.099.121.18.261l3.505 6.073c.553.956.225 2.18-.732 2.732-.956.552-2.18.224-2.732-.732l-3.506-6.073q-.073-.128-.127-.261c-3.704 2.483-6.875 5.702-9.301 9.448q.064.03.127.067l6.073 3.506c.956.553 1.284 1.776.732 2.732s-1.776 1.285-2.732.733L18.6 41.885l-.09-.055c-1.886 3.878-3.03 8.184-3.245 12.733h6.982c1.104 0 2 .895 2 2s-.896 2-2 2h-6.932c.333 4.57 1.604 8.88 3.621 12.74.644 1.233 1.953 1.947 3.344 1.947h51.433c1.392 0 2.7-.714 3.344-1.947 2.017-3.86 3.288-8.17 3.621-12.74h-6.873c-1.105 0-2-.896-2-2s.895-2 2-2h6.924c-.507-10.666-6.113-19.996-14.434-25.607q-.046.106-.105.21l-3.506 6.073c-.553.956-1.776 1.284-2.732.732s-1.285-1.776-.733-2.732l3.507-6.073q.069-.12.15-.225c-3.913-1.998-8.28-3.233-12.907-3.507q.027.161.028.332v7.012c0 1.104-.896 2-2 2m23.615 5.853c.933.933.972 2.433.09 3.413l-21.786 24.2c-1.606 1.785-4.38 1.858-6.077.16-1.697-1.697-1.625-4.47.16-6.077l24.2-21.785c.98-.883 2.48-.843 3.413.09"
                })
            }),
            /*#__PURE__*/ (0, $2Or0E$jsx)("g", {
                clipPath: "url(#a6fb6d__a)",
                children: /*#__PURE__*/ (0, $2Or0E$jsxs)("g", {
                    clipPath: "url(#a6fb6d__b)",
                    children: [
                        /*#__PURE__*/ (0, $2Or0E$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2Or0E$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#a6fb6d__c)"
                        }),
                        /*#__PURE__*/ (0, $2Or0E$jsx)("path", {
                            fill: "url(#a6fb6d__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $2Or0E$jsx)("path", {
                            fill: "url(#a6fb6d__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $2Or0E$jsx)("path", {
                            fill: "url(#a6fb6d__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $aa4b4cef897d5703$var$ForwardRef = /*#__PURE__*/ (0, $2Or0E$forwardRef)($aa4b4cef897d5703$var$SvgComponent);
var $aa4b4cef897d5703$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($aa4b4cef897d5703$var$ForwardRef);


export {$aa4b4cef897d5703$export$2e2bcd8739ae039 as default};
