// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_rotateCCW_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $d0Gcq$reactjsxruntime = require("react/jsx-runtime");
var $d0Gcq$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c9706e945ee6198c$export$2e2bcd8739ae039);




const $c9706e945ee6198c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e531f3__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e531f3__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e531f3__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsxs)("radialGradient", {
                        id: "e531f3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("clipPath", {
                        id: "e531f3__b",
                        children: /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("clipPath", {
                id: "e531f3__a",
                children: /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M79.002 48c0-17.12-13.88-31-31-31-12.189 0-22.738 7.035-27.803 17.27l9.734-3.13c1.052-.337 2.178.241 2.517 1.293.338 1.051-.24 2.178-1.292 2.516L17.1 39.469c-1.051.338-2.178-.24-2.516-1.292l-4.52-14.056c-.338-1.052.24-2.179 1.292-2.517s2.178.24 2.516 1.292l2.953 9.181C22.62 20.754 34.403 13 48.002 13c19.33 0 35 15.67 35 35s-15.67 35-35 35c-15.118 0-27.992-9.583-32.888-23-.38-1.038.155-2.186 1.193-2.565 1.037-.378 2.185.156 2.564 1.194C23.21 70.517 34.618 79 48.001 79c17.122 0 31-13.88 31-31"
                })
            }),
            /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("g", {
                clipPath: "url(#e531f3__a)",
                children: /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#e531f3__b)",
                    children: [
                        /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#e531f3__c)"
                        }),
                        /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("path", {
                            fill: "url(#e531f3__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("path", {
                            fill: "url(#e531f3__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $d0Gcq$reactjsxruntime.jsx)("path", {
                            fill: "url(#e531f3__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $c9706e945ee6198c$var$ForwardRef = /*#__PURE__*/ (0, $d0Gcq$react.forwardRef)($c9706e945ee6198c$var$SvgComponent);
var $c9706e945ee6198c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c9706e945ee6198c$var$ForwardRef);
