// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_translate_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $cd9gg$reactjsxruntime = require("react/jsx-runtime");
var $cd9gg$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $289131aed370dc7b$export$2e2bcd8739ae039);




const $289131aed370dc7b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 97,
        height: 96,
        viewBox: "0 0 97 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a8931f__d",
                        x1: 70.403,
                        x2: 14.807,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a8931f__e",
                        x1: 36.531,
                        x2: 47.256,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a8931f__f",
                        x1: -87.219,
                        x2: 93.232,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a8931f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.7 74.813)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("clipPath", {
                        id: "a8931f__b",
                        children: /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("clipPath", {
                id: "a8931f__a",
                children: /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M57.225 21.785H40.524V18.25c0-1.105-.896-2-2-2s-2 .895-2 2v3.535H19.25c-1.105 0-2 .896-2 2s.895 2 2 2h29.26c-.991 6.861-4.761 12.418-9.5 16.584-1.162 1.02-2.375 1.952-3.609 2.792-2.217-3.815-3.812-7.512-4.286-10.204-.192-1.088-1.23-1.814-2.317-1.623s-1.814 1.23-1.622 2.317c.587 3.336 2.434 7.517 4.806 11.588-3.129 1.688-6.22 2.799-8.8 3.318-1.082.218-1.783 1.273-1.565 2.355s1.272 1.784 2.355 1.566c3.037-.61 6.585-1.9 10.13-3.838l.036.054c3.366 5.057 7.527 9.881 11.2 12.253.927.6 2.165.334 2.764-.594.6-.928.333-2.166-.594-2.766-2.962-1.913-6.731-6.153-9.988-11.03 1.416-.956 2.804-2.018 4.13-3.184 5.388-4.735 9.87-11.297 10.897-19.588h4.678c1.104 0 2-.895 2-2s-.896-2-2-2m-.854 45.349-4.83 11.396c-.43 1.017-1.604 1.492-2.621 1.06-1.017-.43-1.492-1.604-1.062-2.621l14.463-34.121q.045-.11.102-.213.113-.2.262-.365c.19-.21.426-.382.703-.5.256-.108.52-.159.781-.159.26 0 .526.051.78.16.278.117.515.29.704.5q.15.163.261.364.058.104.103.213l14.462 34.12c.431 1.018-.043 2.192-1.06 2.623-1.017.43-2.191-.044-2.622-1.061l-4.841-11.421q-.156.025-.317.025zm7.798-18.397-6.103 14.397h12.205z"
                })
            }),
            /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("g", {
                clipPath: "url(#a8931f__a)",
                children: /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#a8931f__b)",
                    children: [
                        /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("circle", {
                            cx: 86.513,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#a8931f__c)"
                        }),
                        /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("path", {
                            fill: "url(#a8931f__d)",
                            d: "M-7.698 30.067c-1.54-33.182 24.11-61.33 57.291-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.541 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.87-57.291"
                        }),
                        /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("path", {
                            fill: "url(#a8931f__e)",
                            d: "M25.194-66.43c-7.485-4.632-16.97-4.66-24.482-.073l-75.146 45.881c-7.36 4.494-7.372 15.1-.022 19.61L.193 44.806c7.502 4.604 16.986 4.598 24.481-.016L99.17-1.071c7.311-4.5 7.324-15.048.023-19.566z"
                        }),
                        /*#__PURE__*/ (0, $cd9gg$reactjsxruntime.jsx)("path", {
                            fill: "url(#a8931f__f)",
                            d: "M-46.917 45.918c15.488 0 29.518 6.268 39.676 16.381 20.316 20.26 37.159 20.278 57.474.052 10.158-10.165 24.206-16.45 39.711-16.45 30.995.017 56.103 25.056 56.103 55.967S120.939 157.8 89.944 157.8c-15.54 0-29.588-6.303-39.745-16.468-20.299-20.191-37.124-20.174-57.44.087-10.158 10.13-24.188 16.381-39.676 16.381-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $289131aed370dc7b$var$ForwardRef = /*#__PURE__*/ (0, $cd9gg$react.forwardRef)($289131aed370dc7b$var$SvgComponent);
var $289131aed370dc7b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($289131aed370dc7b$var$ForwardRef);
