// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_paperairplane_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2MuNS$jsxs, jsx as $2MuNS$jsx} from "react/jsx-runtime";
import {forwardRef as $2MuNS$forwardRef} from "react";





const $134945f1c3f6aee9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2MuNS$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2MuNS$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2MuNS$jsxs)("linearGradient", {
                        id: "91c490__d",
                        x1: 111.795,
                        x2: 37.593,
                        y1: 11.537,
                        y2: 113.592,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2MuNS$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2MuNS$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2MuNS$jsxs)("linearGradient", {
                        id: "91c490__e",
                        x1: 63.038,
                        x2: 76.794,
                        y1: -33.494,
                        y2: 59.898,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2MuNS$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2MuNS$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2MuNS$jsxs)("linearGradient", {
                        id: "91c490__f",
                        x1: -115.103,
                        x2: 143.445,
                        y1: 153.768,
                        y2: 99.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2MuNS$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2MuNS$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2MuNS$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2MuNS$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2MuNS$jsxs)("radialGradient", {
                        id: "91c490__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 188.091 -199.529 0 134.99 101.37)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2MuNS$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2MuNS$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2MuNS$jsx)("clipPath", {
                        id: "91c490__b",
                        children: /*#__PURE__*/ (0, $2MuNS$jsx)("rect", {
                            width: 138.195,
                            height: 130.273,
                            x: 11.805,
                            y: 15.727,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2MuNS$jsx)("clipPath", {
                id: "91c490__a",
                children: /*#__PURE__*/ (0, $2MuNS$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M138.276 39.184c1.644-.37 3.202-.162 4.534.446L39.157 93.45q-.49.257-.876.615L16.09 78.884c-5.117-3.5-3.592-11.376 2.467-12.74zM94.829 132.75l-49.19-33.65L146.843 46.55q.075-.038.146-.08c-.053.93-.298 1.879-.768 2.807l-40.98 80.825c-1.945 3.835-6.852 5.083-10.412 2.647M41 131.316v-25.382l23.492 16.006-15.53 12.924c-3.127 2.602-7.962.447-7.962-3.548"
                })
            }),
            /*#__PURE__*/ (0, $2MuNS$jsx)("g", {
                clipPath: "url(#91c490__a)",
                children: /*#__PURE__*/ (0, $2MuNS$jsxs)("g", {
                    clipPath: "url(#91c490__b)",
                    children: [
                        /*#__PURE__*/ (0, $2MuNS$jsx)("rect", {
                            width: 138.195,
                            height: 130.273,
                            x: 11.805,
                            y: 15.727,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2MuNS$jsx)("ellipse", {
                            cx: 134.99,
                            cy: 101.37,
                            fill: "url(#91c490__c)",
                            rx: 199.529,
                            ry: 188.091
                        }),
                        /*#__PURE__*/ (0, $2MuNS$jsx)("path", {
                            fill: "url(#91c490__d)",
                            d: "M-.634 56.53C-2.852 11.5 34.073-26.697 81.84-28.788s88.286 32.717 90.504 77.745-34.706 83.225-82.472 85.316S1.584 101.557-.634 56.529"
                        }),
                        /*#__PURE__*/ (0, $2MuNS$jsx)("path", {
                            fill: "url(#91c490__e)",
                            d: "M46.718-74.419c-10.776-6.286-24.428-6.324-35.244-.1L-96.7-12.256c-10.594 6.098-10.612 20.491-.032 26.612L10.728 76.53c10.799 6.247 24.451 6.24 35.241-.023l107.238-62.231c10.524-6.108 10.544-20.42.033-26.552z"
                        }),
                        /*#__PURE__*/ (0, $2MuNS$jsx)("path", {
                            fill: "url(#91c490__f)",
                            d: "M-57.087 78.037c22.296 0 42.493 8.507 57.116 22.23 29.245 27.494 53.49 27.517 82.736.071 14.622-13.794 34.844-22.324 57.165-22.324 44.617.023 80.761 34.003 80.761 75.948s-36.144 75.901-80.761 75.901c-22.371 0-42.593-8.553-57.215-22.347-29.22-27.4-53.441-27.376-82.686.117-14.623 13.747-34.82 22.23-57.116 22.23-44.592 0-80.736-33.979-80.736-75.901s36.144-75.925 80.736-75.925"
                        })
                    ]
                })
            })
        ]
    });
const $134945f1c3f6aee9$var$ForwardRef = /*#__PURE__*/ (0, $2MuNS$forwardRef)($134945f1c3f6aee9$var$SvgComponent);
var $134945f1c3f6aee9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($134945f1c3f6aee9$var$ForwardRef);


export {$134945f1c3f6aee9$export$2e2bcd8739ae039 as default};
