// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_cardTapPayment_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $eHQyi$reactjsxruntime = require("react/jsx-runtime");
var $eHQyi$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $22755c071e1b3f33$export$2e2bcd8739ae039);




const $22755c071e1b3f33$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a5d179__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a5d179__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a5d179__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a5d179__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("clipPath", {
                        id: "a5d179__b",
                        children: /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("clipPath", {
                id: "a5d179__a",
                children: /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M112.544 26.78c.267-2.194 2.262-3.754 4.455-3.486 11.158 1.362 21.579 7.773 27.64 18.27 6.11 10.582 6.406 22.927 1.893 33.325-.879 2.026-3.235 2.956-5.262 2.077-2.026-.88-2.956-3.236-2.076-5.262 3.547-8.173 3.306-17.845-1.483-26.14-4.751-8.229-12.912-13.258-21.682-14.33-2.193-.267-3.753-2.262-3.485-4.455m-1.254 13.944c.267-2.193 2.262-3.753 4.455-3.485 6.857.837 13.268 4.778 16.995 11.234 3.757 6.508 3.938 14.1 1.164 20.49-.879 2.027-3.235 2.956-5.262 2.077-2.026-.88-2.956-3.236-2.076-5.262 1.807-4.165 1.682-9.085-.754-13.306-2.418-4.186-6.568-6.747-11.037-7.293-2.193-.267-3.753-2.262-3.485-4.455M18.393 74.416C10.741 78.834 8.12 88.62 12.537 96.272l.42.729 90.245-52.14-.405-.7c-4.418-7.653-14.203-10.275-21.856-5.857zm88.809-22.627-90.244 52.14 14.32 24.804c4.419 7.653 14.204 10.275 21.857 5.857l62.547-36.112c7.653-4.419 10.275-14.204 5.857-21.857zM30.429 108.1c-1.104-1.913-.449-4.36 1.464-5.465l32.38-18.697c1.913-1.104 4.36-.449 5.465 1.464s.449 4.36-1.464 5.464l-32.38 18.697c-1.914 1.105-4.36.45-5.465-1.463m8.476 6.68c-1.913 1.105-2.569 3.551-1.464 5.464s3.551 2.569 5.464 1.464l16.19-9.349c1.913-1.104 2.569-3.551 1.464-5.464s-3.55-2.568-5.464-1.464zm43.14-36.48c-1.104-1.912-.449-4.359 1.464-5.463l15.18-8.766c1.914-1.105 4.36-.45 5.465 1.464s.449 4.36-1.464 5.464l-15.18 8.766c-1.913 1.104-4.36.449-5.464-1.464"
                })
            }),
            /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("g", {
                clipPath: "url(#a5d179__a)",
                children: /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#a5d179__b)",
                    children: [
                        /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#a5d179__c)"
                        }),
                        /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("path", {
                            fill: "url(#a5d179__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("path", {
                            fill: "url(#a5d179__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $eHQyi$reactjsxruntime.jsx)("path", {
                            fill: "url(#a5d179__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $22755c071e1b3f33$var$ForwardRef = /*#__PURE__*/ (0, $eHQyi$react.forwardRef)($22755c071e1b3f33$var$SvgComponent);
var $22755c071e1b3f33$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($22755c071e1b3f33$var$ForwardRef);
