// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_badgeVerified_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $n3Oz3$reactjsxruntime = require("react/jsx-runtime");
var $n3Oz3$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $13ace852a6cd83e1$export$2e2bcd8739ae039);




const $13ace852a6cd83e1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsxs)("linearGradient", {
                        id: "21c79c__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsxs)("linearGradient", {
                        id: "21c79c__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsxs)("linearGradient", {
                        id: "21c79c__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsxs)("radialGradient", {
                        id: "21c79c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("clipPath", {
                        id: "21c79c__b",
                        children: /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("clipPath", {
                id: "21c79c__a",
                children: /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M37.883 19.314c1.631.925 2.738 2.678 2.738 4.688h.004c0 2.019-1.115 3.775-2.76 4.7-.645.362-.946 1.098-.748 1.813.22.78.267 1.632.09 2.514-.418 2.105-2.1 3.792-4.206 4.21-.886.177-1.747.13-2.53-.095-.703-.198-1.434.1-1.8.732-.934 1.614-2.674 2.704-4.671 2.704s-3.736-1.09-4.67-2.704c-.367-.633-1.094-.93-1.8-.732-.784.224-1.645.272-2.531.095-2.105-.417-3.789-2.105-4.206-4.21-.176-.883-.13-1.735.09-2.514.198-.715-.103-1.45-.749-1.812-1.644-.926-2.759-2.69-2.759-4.701s1.102-3.762 2.738-4.688c.624-.353.912-1.09.74-1.765-.172-.676-.22-1.399-.112-2.148.34-2.329 2.222-4.201 4.55-4.537.762-.112 1.464-.07 2.179.125.714.193 1.446-.117 1.803-.762.917-1.67 2.69-2.807 4.727-2.807 2.036 0 3.81 1.136 4.727 2.807.357.645 1.123.942 1.803.762.68-.181 1.412-.237 2.174-.125 2.333.335 4.21 2.208 4.55 4.537.108.749.07 1.455-.112 2.148-.18.693.117 1.412.74 1.765m-5.822 1.19c.51-.465.547-1.256.082-1.766s-1.256-.547-1.766-.082l-.138.126-.03.028-.018.015-.017.016-.077.07-.043.04-.022.02-.011.01-.012.01-.099.09-.493.45-.145.132-.154.14-.161.147-.169.154-.021.02-.022.02-.022.02-.022.02-.022.02-.022.02-.022.02-.022.02-.182.165-.093.085-.094.086-.024.022-.024.022-.048.044-.024.022-.025.022-.024.022-.024.022-.05.045-.024.023-.025.022-.025.023-.025.022-.024.023-.026.023-.025.023-.025.023-.025.023-.025.023-.026.023-.025.023-.051.047-.207.188-.211.193-.214.195-.027.024-.027.025-.054.05-.027.024-.028.025-.027.024-.027.025-.055.05-.054.05-.11.1-.028.025-.028.026-.027.025-.028.025-.055.05-.056.051-.055.05-.056.052-.028.025-.028.026-.056.05-.028.026-.028.026-.028.025-.028.025-.056.052-.056.05-.113.103-.028.026-.028.025-.056.052-.028.026-.029.025-.056.051-.112.103-.45.41-.224.204-.028.025-.027.026-.028.025-.028.025-.11.101-.221.201-.11.1-.027.025-.027.024-.054.05-.054.05-.027.024-.027.024-.108.098-.106.097-.026.024-.027.024-.053.048-1.532 1.397-.02.019-.021.019-.021.019-.02.018-.041.038-.04.036-.158.143-.149.136-.036.033-.035.032-.035.032-.017.016-.017.015-.029.026-3.896-3.633c-.505-.471-1.296-.444-1.767.061s-.443 1.296.062 1.767l4.739 4.42c.476.444 1.213.448 1.695.01l-.843-.925.843.924.001-.001.005-.004v-.001l.004-.004.005-.004.003-.002.003-.003.006-.006.033-.03.047-.042.128-.117.082-.075.044-.04.012-.011.012-.011.024-.022.103-.094.027-.025.014-.012.014-.013.015-.013.014-.013.03-.027.06-.055.062-.057.132-.12.017-.015.017-.016.017-.016.018-.016.017-.016.018-.016.036-.033.037-.033.018-.017.019-.017.018-.017.02-.017.018-.018.02-.017.156-.143.04-.037.02-.018.021-.019.021-.019.02-.019.021-.019.021-.019.35-.318.023-.02.022-.022.023-.02.023-.021.093-.085.047-.043.024-.021.023-.022.024-.022.024-.021.024-.022.024-.022.097-.088.024-.023.025-.022.024-.023.025-.022.05-.045.05-.046.025-.023.025-.022.05-.046.102-.093.051-.047.026-.023.026-.024.104-.094.026-.024.026-.024.026-.024.026-.024.027-.024.026-.024.053-.048.027-.024.026-.025.027-.024.026-.024.027-.025.027-.024.054-.049.108-.099.109-.099.054-.05.027-.024.028-.025.11-.1.027-.025.028-.026.055-.05.111-.101.028-.025.028-.026.027-.025.028-.026.028-.025.028-.025.028-.026.028-.025.028-.026.028-.025.056-.051.028-.026.028-.025.056-.051.113-.103.225-.205.45-.41.028-.026.028-.026.056-.05.112-.103.056-.05.056-.052.111-.101.111-.101.055-.05.028-.026.028-.025.219-.2.109-.1.054-.049.054-.049.107-.098.107-.097.106-.096.026-.024.026-.024.053-.048.207-.189.102-.092.025-.023.025-.023.025-.023.026-.023.05-.046.024-.023.025-.022.05-.045.049-.045.048-.044.025-.022.024-.023.024-.021.024-.022.024-.022.024-.022.187-.17.357-.326.022-.02.021-.02.042-.038.084-.076.02-.019.02-.018.041-.037.08-.073.078-.07.076-.07.074-.067.036-.033.035-.033.07-.063.017-.016.017-.015.017-.015.016-.016.017-.015.016-.015.017-.014.016-.015.063-.057.118-.109.015-.012.014-.013.027-.025.027-.025.013-.012.013-.012.187-.17.142-.13z"
                })
            }),
            /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("g", {
                clipPath: "url(#21c79c__a)",
                children: /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#21c79c__b)",
                    children: [
                        /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#21c79c__c)"
                        }),
                        /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("path", {
                            fill: "url(#21c79c__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("path", {
                            fill: "url(#21c79c__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $n3Oz3$reactjsxruntime.jsx)("path", {
                            fill: "url(#21c79c__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $13ace852a6cd83e1$var$ForwardRef = /*#__PURE__*/ (0, $n3Oz3$react.forwardRef)($13ace852a6cd83e1$var$SvgComponent);
var $13ace852a6cd83e1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($13ace852a6cd83e1$var$ForwardRef);
