// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_trophy_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $h448a$jsxs, jsx as $h448a$jsx} from "react/jsx-runtime";
import {forwardRef as $h448a$forwardRef} from "react";





const $405c6e1843950634$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $h448a$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $h448a$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $h448a$jsxs)("linearGradient", {
                        id: "d40309__d",
                        x1: 32.635,
                        x2: 11.534,
                        y1: 4.21,
                        y2: 33.232,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h448a$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $h448a$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h448a$jsxs)("linearGradient", {
                        id: "d40309__e",
                        x1: 18.767,
                        x2: 22.679,
                        y1: -8.597,
                        y2: 17.962,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h448a$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $h448a$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h448a$jsxs)("linearGradient", {
                        id: "d40309__f",
                        x1: -31.892,
                        x2: 41.634,
                        y1: 44.657,
                        y2: 29.241,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h448a$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $h448a$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $h448a$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $h448a$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h448a$jsxs)("radialGradient", {
                        id: "d40309__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.4895 -56.7422 0 39.23 29.755)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h448a$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h448a$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h448a$jsx)("clipPath", {
                        id: "d40309__b",
                        children: /*#__PURE__*/ (0, $h448a$jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 5.4,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $h448a$jsx)("clipPath", {
                id: "d40309__a",
                children: /*#__PURE__*/ (0, $h448a$jsx)("path", {
                    fill: "#fff",
                    d: "M37.811 12H32.4v-1.8c0-.993-.807-1.8-1.8-1.8H17.4c-.993 0-1.8.807-1.8 1.8V12h-5.4c-1.653 0-3 1.347-3 3v.6c0 4.632 3.768 8.4 8.4 8.4h1.51c1.301 1.866 3.344 3.18 5.7 3.513 0 .03-.01.057-.01.087v7.326c-2.15.405-3.92 1.872-4.752 3.837-.168.396.126.834.555.834h10.794c.43 0 .723-.438.555-.834-.83-1.965-2.6-3.432-4.752-3.837V27.6c0-.03-.006-.057-.009-.087 2.358-.336 4.398-1.647 5.7-3.513h1.518c4.632 0 8.4-3.768 8.4-8.4V15c0-1.653-1.347-3-3-3zM15.6 21.6c-3.309 0-6-2.69-6-6V15c0-.33.27-.6.6-.6h5.4v4.8c0 .834.123 1.638.351 2.4zm12.237-4.449-1.119 1.086c-.153.15-.225.366-.186.576l.267 1.536c.102.594-.522 1.05-1.056.768l-1.38-.729c-.189-.099-.417-.099-.606 0l-1.38.73c-.534.281-1.158-.172-1.056-.769l.267-1.536c.036-.21-.033-.426-.186-.576l-1.119-1.086c-.432-.42-.195-1.155.402-1.242l1.545-.222c.213-.03.396-.165.492-.357l.687-1.4c.267-.544 1.038-.544 1.305 0l.687 1.4c.096.192.28.327.492.357l1.545.222c.597.087.837.822.402 1.242zm10.575-1.55c0 3.308-2.69 6-6 6h-.36c.225-.763.351-1.567.351-2.4v-4.8h5.412c.33 0 .6.27.6.6v.6z"
                })
            }),
            /*#__PURE__*/ (0, $h448a$jsx)("g", {
                clipPath: "url(#d40309__a)",
                children: /*#__PURE__*/ (0, $h448a$jsxs)("g", {
                    clipPath: "url(#d40309__b)",
                    children: [
                        /*#__PURE__*/ (0, $h448a$jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 5.4,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $h448a$jsx)("ellipse", {
                            cx: 39.23,
                            cy: 29.755,
                            fill: "url(#d40309__c)",
                            rx: 56.742,
                            ry: 53.489
                        }),
                        /*#__PURE__*/ (0, $h448a$jsx)("path", {
                            fill: "url(#d40309__d)",
                            d: "M.663 17.005C.032 4.199 10.533-6.663 24.116-7.258c13.584-.594 25.107 9.304 25.738 22.11.63 12.805-9.87 23.667-23.454 24.262-13.583.594-25.107-9.304-25.737-22.11"
                        }),
                        /*#__PURE__*/ (0, $h448a$jsx)("path", {
                            fill: "url(#d40309__e)",
                            d: "M14.126-20.236c-3.064-1.787-6.947-1.798-10.022-.028L-26.66-2.558c-3.013 1.734-3.018 5.827-.01 7.568l30.56 17.681c3.071 1.777 6.954 1.774 10.022-.006L44.41 4.987c2.993-1.737 2.998-5.807.009-7.55z"
                        }),
                        /*#__PURE__*/ (0, $h448a$jsx)("path", {
                            fill: "url(#d40309__f)",
                            d: "M-15.393 23.12c6.34 0 12.084 2.42 16.242 6.322 8.317 7.818 15.212 7.825 23.529.02 4.158-3.923 9.909-6.349 16.257-6.349 12.688.007 22.967 9.67 22.967 21.599s-10.28 21.584-22.967 21.584c-6.362 0-12.113-2.432-16.271-6.355-8.31-7.792-15.198-7.785-23.515.034-4.158 3.909-9.902 6.321-16.242 6.321-12.682 0-22.96-9.663-22.96-21.584 0-11.922 10.278-21.592 22.96-21.592"
                        })
                    ]
                })
            })
        ]
    });
const $405c6e1843950634$var$ForwardRef = /*#__PURE__*/ (0, $h448a$forwardRef)($405c6e1843950634$var$SvgComponent);
var $405c6e1843950634$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($405c6e1843950634$var$ForwardRef);


export {$405c6e1843950634$export$2e2bcd8739ae039 as default};
