// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_ribbon_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $7iRdC$reactjsxruntime = require("react/jsx-runtime");
var $7iRdC$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $cacf6bf28e0237e1$export$2e2bcd8739ae039);




const $cacf6bf28e0237e1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "90fc4e__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "90fc4e__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "90fc4e__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsxs)("radialGradient", {
                        id: "90fc4e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("clipPath", {
                        id: "90fc4e__b",
                        children: /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("clipPath", {
                id: "90fc4e__a",
                children: /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M69.497 36.5c0 11.874-9.626 21.5-21.5 21.5s-21.5-9.626-21.5-21.5S36.123 15 47.997 15s21.5 9.626 21.5 21.5M54.79 77.018l-6.461-16.02c5.702-.075 10.934-2.098 15.06-5.434l6.238 15.465c.595 1.476-.688 3.023-2.249 2.71l-4.464-.896c-.783-.157-1.584.167-2.038.823l-2.587 3.741c-.905 1.31-2.904 1.088-3.5-.389m-22.199-21.47c4.125 3.341 9.356 5.37 15.058 5.45l-6.435 15.867c-.6 1.48-2.605 1.694-3.505.376l-2.565-3.758c-.455-.667-1.266-.995-2.057-.832l-4.445.92c-1.565.324-2.86-1.229-2.259-2.71z"
                })
            }),
            /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("g", {
                clipPath: "url(#90fc4e__a)",
                children: /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#90fc4e__b)",
                    children: [
                        /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#90fc4e__c)"
                        }),
                        /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("path", {
                            fill: "url(#90fc4e__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("path", {
                            fill: "url(#90fc4e__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $7iRdC$reactjsxruntime.jsx)("path", {
                            fill: "url(#90fc4e__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $cacf6bf28e0237e1$var$ForwardRef = /*#__PURE__*/ (0, $7iRdC$react.forwardRef)($cacf6bf28e0237e1$var$SvgComponent);
var $cacf6bf28e0237e1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($cacf6bf28e0237e1$var$ForwardRef);
