// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_volumeOne_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $jJXLP$jsxs, jsx as $jJXLP$jsx} from "react/jsx-runtime";
import {forwardRef as $jJXLP$forwardRef} from "react";





const $aff3c06c119a9174$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jJXLP$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jJXLP$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jJXLP$jsxs)("linearGradient", {
                        id: "92330b__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jJXLP$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $jJXLP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jJXLP$jsxs)("linearGradient", {
                        id: "92330b__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jJXLP$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $jJXLP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jJXLP$jsxs)("linearGradient", {
                        id: "92330b__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jJXLP$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jJXLP$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $jJXLP$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $jJXLP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jJXLP$jsxs)("radialGradient", {
                        id: "92330b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jJXLP$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jJXLP$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jJXLP$jsx)("clipPath", {
                        id: "92330b__b",
                        children: /*#__PURE__*/ (0, $jJXLP$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jJXLP$jsx)("clipPath", {
                id: "92330b__a",
                children: /*#__PURE__*/ (0, $jJXLP$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M27.742 32.052h8.379q.092 0 .184-.004v32.03q-.086-.003-.172-.004h-8.39c-2.21 0-4-1.79-4-4V36.052c0-2.209 1.79-4 4-4m12.563 34.513V29.537L50.599 19.33c1.893-1.878 5.112-.536 5.112 2.13v53.077c0 2.66-3.204 4.004-5.102 2.141zm25.183-32.188c-.882-.665-2.136-.488-2.8.395-.665.883-.488 2.136.395 2.8 3.149 2.37 5.179 6.132 5.179 10.37 0 4.27-2.064 8.06-5.256 10.426-.888.657-1.074 1.91-.417 2.797.658.888 1.91 1.074 2.798.416 4.167-3.087 6.875-8.048 6.875-13.64 0-5.547-2.664-10.472-6.774-13.565"
                })
            }),
            /*#__PURE__*/ (0, $jJXLP$jsx)("g", {
                clipPath: "url(#92330b__a)",
                children: /*#__PURE__*/ (0, $jJXLP$jsxs)("g", {
                    clipPath: "url(#92330b__b)",
                    children: [
                        /*#__PURE__*/ (0, $jJXLP$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $jJXLP$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#92330b__c)"
                        }),
                        /*#__PURE__*/ (0, $jJXLP$jsx)("path", {
                            fill: "url(#92330b__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $jJXLP$jsx)("path", {
                            fill: "url(#92330b__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $jJXLP$jsx)("path", {
                            fill: "url(#92330b__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $aff3c06c119a9174$var$ForwardRef = /*#__PURE__*/ (0, $jJXLP$forwardRef)($aff3c06c119a9174$var$SvgComponent);
var $aff3c06c119a9174$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($aff3c06c119a9174$var$ForwardRef);


export {$aff3c06c119a9174$export$2e2bcd8739ae039 as default};
