// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_remix_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $2o3DI$reactjsxruntime = require("react/jsx-runtime");
var $2o3DI$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $bec66b467ac46338$export$2e2bcd8739ae039);




const $bec66b467ac46338$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3a2764__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3a2764__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3a2764__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3a2764__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("clipPath", {
                        id: "3a2764__b",
                        children: /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("clipPath", {
                id: "3a2764__a",
                children: /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M76.03 41.644 86.645 31.03c.781-.78.781-2.047 0-2.828L76.03 17.586c-.781-.781-2.047-.781-2.828 0-.781.78-.781 2.047 0 2.828l6.998 6.998h-4.787c-4.92 0-9.46 1.657-13.083 4.443q-.076.06-.148.125-.041.03-.082.062c-2.672 2.098-5.175 5.157-7.142 7.977-1.975 2.831-3.517 5.575-4.233 7.156-.57 1.258-1.98 4.245-3.848 7.385-1.607 2.703-3.457 5.352-5.308 7.144q-.077.058-.15.124c-3.105 2.822-7.225 4.54-11.75 4.54-9.653 0-17.478-7.826-17.478-17.478s7.825-17.478 17.478-17.478c5.903 0 11.123 2.925 14.29 7.413l.008.01 2.022 2.828c.642.899 1.892 1.106 2.79.464s1.106-1.892.464-2.79l-2.02-2.824c-3.884-5.501-10.299-9.1-17.554-9.1-11.862 0-21.478 9.615-21.478 21.477s9.616 21.477 21.478 21.477c4.925 0 9.466-1.66 13.09-4.448q.143-.08.277-.184c2.791-2.202 5.34-5.87 7.28-9.13 1.97-3.316 3.45-6.45 4.052-7.779.593-1.308 2-3.838 3.87-6.518 1.878-2.69 4.115-5.38 6.333-7.12q.069-.054.132-.113l.066-.049c2.948-2.267 6.636-3.614 10.644-3.614h5.192l-7.403 7.404c-.781.78-.781 2.047 0 2.828.78.781 2.047.781 2.828 0m4.781 28.723c1.105 0 2-.895 2-2s-.895-2-2-2h-5.398c-5.811 0-10.497-3.346-14.558-7.777l-2.337-2.825c-.704-.85-1.965-.97-2.816-.265-.85.704-.97 1.965-.266 2.816l2.37 2.863.031.038.033.037c4.338 4.743 10.03 9.113 17.543 9.113z"
                })
            }),
            /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("g", {
                clipPath: "url(#3a2764__a)",
                children: /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#3a2764__b)",
                    children: [
                        /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#3a2764__c)"
                        }),
                        /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("path", {
                            fill: "url(#3a2764__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("path", {
                            fill: "url(#3a2764__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $2o3DI$reactjsxruntime.jsx)("path", {
                            fill: "url(#3a2764__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $bec66b467ac46338$var$ForwardRef = /*#__PURE__*/ (0, $2o3DI$react.forwardRef)($bec66b467ac46338$var$SvgComponent);
var $bec66b467ac46338$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($bec66b467ac46338$var$ForwardRef);
