// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_information_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $k46eu$reactjsxruntime = require("react/jsx-runtime");
var $k46eu$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6f0812c68050458f$export$2e2bcd8739ae039);




const $6f0812c68050458f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "120c25__d",
                        x1: 65.566,
                        x2: 25.102,
                        y1: 9.835,
                        y2: 65.487,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "120c25__e",
                        x1: 38.978,
                        x2: 46.479,
                        y1: -14.72,
                        y2: 36.207,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "120c25__f",
                        x1: -58.165,
                        x2: 82.825,
                        y1: 87.396,
                        y2: 57.837,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsxs)("radialGradient", {
                        id: "120c25__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 78.212 58.823)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("clipPath", {
                        id: "120c25__b",
                        children: /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 12.119,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("clipPath", {
                id: "120c25__a",
                children: /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M81.2 47.6c0 18.557-15.043 33.6-33.6 33.6S14 66.157 14 47.6 29.043 14 47.6 14s33.6 15.043 33.6 33.6m-38.1-8.1c-1.325 0-2.4 1.075-2.4 2.4s1.075 2.4 2.4 2.4h3v13.8h-3c-1.325 0-2.4 1.075-2.4 2.4s1.075 2.4 2.4 2.4h10.8c1.325 0 2.4-1.075 2.4-2.4s-1.075-2.4-2.4-2.4h-3V41.9c0-1.325-1.075-2.4-2.4-2.4zm4.8-3c1.657 0 3-1.343 3-3s-1.343-3-3-3-3 1.343-3 3 1.343 3 3 3"
                })
            }),
            /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("g", {
                clipPath: "url(#120c25__a)",
                children: /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#120c25__b)",
                    children: [
                        /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 12.119,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("ellipse", {
                            cx: 78.212,
                            cy: 58.823,
                            fill: "url(#120c25__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("path", {
                            fill: "url(#120c25__d)",
                            d: "M4.256 34.37C3.046 9.816 23.182-11.014 49.23-12.154S97.374 5.686 98.583 30.24 79.658 75.626 53.61 76.766 5.466 58.925 4.256 34.37"
                        }),
                        /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("path", {
                            fill: "url(#120c25__e)",
                            d: "M30.078-37.038c-5.876-3.427-13.321-3.448-19.219-.054L-48.13-3.14c-5.777 3.326-5.787 11.175-.017 14.512l58.599 33.905c5.889 3.406 13.334 3.402 19.218-.013l58.478-33.936c5.74-3.33 5.75-11.135.018-14.478z"
                        }),
                        /*#__PURE__*/ (0, $k46eu$reactjsxruntime.jsx)("path", {
                            fill: "url(#120c25__f)",
                            d: "M-26.528 46.099c12.159 0 23.172 4.639 31.146 12.122 15.948 14.993 29.17 15.006 45.118.039 7.974-7.523 19-12.174 31.173-12.174 24.33.013 44.04 18.542 44.04 41.416s-19.71 41.39-44.04 41.39c-12.2 0-23.227-4.664-31.2-12.186-15.935-14.942-29.143-14.929-45.09.064-7.975 7.496-18.988 12.122-31.147 12.122-24.317 0-44.026-18.53-44.026-41.39 0-22.861 19.71-41.403 44.026-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $6f0812c68050458f$var$ForwardRef = /*#__PURE__*/ (0, $k46eu$react.forwardRef)($6f0812c68050458f$var$SvgComponent);
var $6f0812c68050458f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6f0812c68050458f$var$ForwardRef);
