// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_heart_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $1OoGS$reactjsxruntime = require("react/jsx-runtime");
var $1OoGS$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f45eddeb794088f5$export$2e2bcd8739ae039);




const $f45eddeb794088f5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ef1dcd__d",
                        x1: 32.836,
                        x2: 9.555,
                        y1: 1.407,
                        y2: 31.591,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ef1dcd__e",
                        x1: 18.652,
                        x2: 23.143,
                        y1: -12.488,
                        y2: 16.254,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ef1dcd__f",
                        x1: -33.169,
                        x2: 42.395,
                        y1: 45.297,
                        y2: 30.363,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ef1dcd__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.228 34.356)scale(58.0415)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("clipPath", {
                        id: "ef1dcd__b",
                        children: /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("rect", {
                            width: 40.2,
                            height: 40.2,
                            x: 3.75,
                            y: 2.7,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("clipPath", {
                id: "ef1dcd__a",
                children: /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M32.402 7.8c-4.64.003-8.4 3.798-8.4 8.478l-.003.02-.003-.02c0-4.68-3.76-8.475-8.4-8.478-5.115-.003-8.773 4.41-8.366 9.554.737 9.305 10.76 19.751 14.947 23.106 1.065.853 2.572.853 3.637 0 4.19-3.354 14.217-13.801 14.954-23.107.408-5.144-3.251-9.556-8.366-9.553"
                })
            }),
            /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ef1dcd__a)",
                children: /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ef1dcd__b)",
                    children: [
                        /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("rect", {
                            width: 40.2,
                            height: 40.2,
                            x: 3.75,
                            y: 2.7,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("circle", {
                            cx: 39.584,
                            cy: 29.128,
                            r: 58.041,
                            fill: "url(#ef1dcd__c)"
                        }),
                        /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("path", {
                            fill: "url(#ef1dcd__d)",
                            d: "M.131 15.29C-.514 1.397 10.227-10.39 24.121-11.035 38.018-11.682 49.805-.94 50.45 12.954c.645 13.895-10.096 25.682-23.99 26.328C12.562 39.927.775 29.186.13 15.29"
                        }),
                        /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("path", {
                            fill: "url(#ef1dcd__e)",
                            d: "M13.905-25.117c-3.135-1.94-7.106-1.951-10.252-.03L-27.814-5.936c-3.082 1.882-3.087 6.323-.01 8.212l31.26 19.186c3.14 1.927 7.112 1.925 10.251-.007L44.882 2.252c3.061-1.884 3.067-6.301.01-8.193z"
                        }),
                        /*#__PURE__*/ (0, $1OoGS$reactjsxruntime.jsx)("path", {
                            fill: "url(#ef1dcd__f)",
                            d: "M-16.292 21.928c6.486 0 12.36 2.625 16.614 6.86 8.508 8.484 15.56 8.491 24.068.022 4.253-4.257 10.136-6.89 16.629-6.89 12.979.008 23.493 10.494 23.493 23.437S53.998 68.78 41.019 68.78c-6.508 0-12.39-2.64-16.644-6.896-8.5-8.455-15.545-8.448-24.053.036-4.253 4.242-10.128 6.86-16.614 6.86-12.972 0-23.486-10.485-23.486-23.422 0-12.936 10.514-23.429 23.486-23.429"
                        })
                    ]
                })
            })
        ]
    });
const $f45eddeb794088f5$var$ForwardRef = /*#__PURE__*/ (0, $1OoGS$react.forwardRef)($f45eddeb794088f5$var$SvgComponent);
var $f45eddeb794088f5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f45eddeb794088f5$var$ForwardRef);
