// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_shapes_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $9Jg12$jsxs, jsx as $9Jg12$jsx} from "react/jsx-runtime";
import {forwardRef as $9Jg12$forwardRef} from "react";





const $5fffa4b3551a8d16$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9Jg12$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 97,
        height: 96,
        viewBox: "0 0 97 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9Jg12$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9Jg12$jsxs)("linearGradient", {
                        id: "bdd008__d",
                        x1: 70.403,
                        x2: 14.807,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Jg12$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $9Jg12$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Jg12$jsxs)("linearGradient", {
                        id: "bdd008__e",
                        x1: 36.531,
                        x2: 47.256,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Jg12$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $9Jg12$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Jg12$jsxs)("linearGradient", {
                        id: "bdd008__f",
                        x1: -87.219,
                        x2: 93.232,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Jg12$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9Jg12$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $9Jg12$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $9Jg12$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Jg12$jsxs)("radialGradient", {
                        id: "bdd008__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.7 74.813)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Jg12$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9Jg12$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Jg12$jsx)("clipPath", {
                        id: "bdd008__b",
                        children: /*#__PURE__*/ (0, $9Jg12$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9Jg12$jsx)("clipPath", {
                id: "bdd008__a",
                children: /*#__PURE__*/ (0, $9Jg12$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M33.851 18.385 16.305 48.588c-1.55 2.667.375 6.01 3.459 6.01H54.81c3.082 0 5.006-3.34 3.46-6.006l-8.756-15.111-8.744-15.092c-1.54-2.658-5.377-2.66-6.92-.004m24.392 61.218c-12.217 0-22.291-9.172-23.712-21.005h20.28c6.164 0 10.012-6.678 6.922-12.011l-8.27-14.273c1.544-.313 3.143-.478 4.78-.478 13.19 0 23.883 10.693 23.883 23.884 0 13.19-10.693 23.883-23.883 23.883"
                })
            }),
            /*#__PURE__*/ (0, $9Jg12$jsx)("g", {
                clipPath: "url(#bdd008__a)",
                children: /*#__PURE__*/ (0, $9Jg12$jsxs)("g", {
                    clipPath: "url(#bdd008__b)",
                    children: [
                        /*#__PURE__*/ (0, $9Jg12$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $9Jg12$jsx)("circle", {
                            cx: 86.513,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#bdd008__c)"
                        }),
                        /*#__PURE__*/ (0, $9Jg12$jsx)("path", {
                            fill: "url(#bdd008__d)",
                            d: "M-7.698 30.067c-1.54-33.182 24.11-61.33 57.291-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.541 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.87-57.291"
                        }),
                        /*#__PURE__*/ (0, $9Jg12$jsx)("path", {
                            fill: "url(#bdd008__e)",
                            d: "M25.194-66.43c-7.485-4.632-16.97-4.66-24.482-.073l-75.146 45.881c-7.36 4.494-7.372 15.1-.022 19.61L.193 44.806c7.502 4.604 16.986 4.598 24.481-.016L99.17-1.071c7.311-4.5 7.324-15.048.023-19.566z"
                        }),
                        /*#__PURE__*/ (0, $9Jg12$jsx)("path", {
                            fill: "url(#bdd008__f)",
                            d: "M-46.917 45.918c15.488 0 29.518 6.268 39.676 16.381 20.316 20.26 37.159 20.278 57.474.052 10.158-10.165 24.206-16.45 39.711-16.45 30.995.017 56.103 25.056 56.103 55.967S120.939 157.8 89.944 157.8c-15.54 0-29.588-6.303-39.745-16.468-20.299-20.191-37.124-20.174-57.44.087-10.158 10.13-24.188 16.381-39.676 16.381-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $5fffa4b3551a8d16$var$ForwardRef = /*#__PURE__*/ (0, $9Jg12$forwardRef)($5fffa4b3551a8d16$var$SvgComponent);
var $5fffa4b3551a8d16$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5fffa4b3551a8d16$var$ForwardRef);


export {$5fffa4b3551a8d16$export$2e2bcd8739ae039 as default};
