// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_cloudUpload_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $4juzv$jsxs, jsx as $4juzv$jsx} from "react/jsx-runtime";
import {forwardRef as $4juzv$forwardRef} from "react";





const $3467e722ad5dc363$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4juzv$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4juzv$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4juzv$jsxs)("linearGradient", {
                        id: "8d0f31__d",
                        x1: 32.783,
                        x2: 12.551,
                        y1: 5.819,
                        y2: 33.645,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4juzv$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $4juzv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4juzv$jsxs)("linearGradient", {
                        id: "8d0f31__e",
                        x1: 19.489,
                        x2: 23.239,
                        y1: -6.459,
                        y2: 19.005,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4juzv$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $4juzv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4juzv$jsxs)("linearGradient", {
                        id: "8d0f31__f",
                        x1: -29.084,
                        x2: 41.411,
                        y1: 44.599,
                        y2: 29.819,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4juzv$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4juzv$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $4juzv$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $4juzv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4juzv$jsxs)("radialGradient", {
                        id: "8d0f31__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.2844 -54.4031 0 39.106 30.312)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4juzv$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4juzv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4juzv$jsx)("clipPath", {
                        id: "8d0f31__b",
                        children: /*#__PURE__*/ (0, $4juzv$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.961,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4juzv$jsx)("clipPath", {
                id: "8d0f31__a",
                children: /*#__PURE__*/ (0, $4juzv$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m35.262 16.396-.003.096c4.01.237 7.194 3.555 7.194 7.956s-3.414 7.968-7.63 7.968h-9.771V22.222l5.25 4.212c.223.177.487.264.75.264.352 0 .7-.156.937-.45.414-.516.33-1.272-.186-1.686l-7.101-5.7c-.441-.354-1.071-.351-1.51.006l-6.998 5.7c-.516.42-.594 1.176-.174 1.689.42.516 1.176.594 1.689.174l4.94-4.026v10.01h2.4v8.386c0 .663-.536 1.2-1.2 1.2-.662 0-1.2-.537-1.2-1.2v-8.385H10.942v-.006c-2.985-.051-5.394-2.592-5.394-5.724 0-2.637 1.71-4.854 4.035-5.52-.105-.483-.168-.984-.168-1.503 0-3.678 2.853-6.657 6.372-6.657.86 0 1.68.18 2.43.504 1.13-3.672 4.425-6.33 8.316-6.33 4.824 0 8.733 4.083 8.733 9.12z"
                })
            }),
            /*#__PURE__*/ (0, $4juzv$jsx)("g", {
                clipPath: "url(#8d0f31__a)",
                children: /*#__PURE__*/ (0, $4juzv$jsxs)("g", {
                    clipPath: "url(#8d0f31__b)",
                    children: [
                        /*#__PURE__*/ (0, $4juzv$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.961,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $4juzv$jsx)("ellipse", {
                            cx: 39.106,
                            cy: 30.312,
                            fill: "url(#8d0f31__c)",
                            rx: 54.403,
                            ry: 51.284
                        }),
                        /*#__PURE__*/ (0, $4juzv$jsx)("path", {
                            fill: "url(#8d0f31__d)",
                            d: "M2.128 18.087C1.523 5.809 11.591-4.607 24.615-5.177s24.072 8.92 24.677 21.198c.605 12.277-9.463 22.692-22.487 23.262S2.733 30.364 2.128 18.087"
                        }),
                        /*#__PURE__*/ (0, $4juzv$jsx)("path", {
                            fill: "url(#8d0f31__e)",
                            d: "M15.039-17.617c-2.938-1.714-6.66-1.725-9.61-.027L-24.064-.669c-2.889 1.662-2.893 5.587-.009 7.256l29.3 16.952c2.944 1.703 6.667 1.7 9.609-.006l29.24-16.968c2.869-1.666 2.874-5.568.008-7.24z"
                        }),
                        /*#__PURE__*/ (0, $4juzv$jsx)("path", {
                            fill: "url(#8d0f31__f)",
                            d: "M-13.266 23.95c6.08 0 11.586 2.32 15.573 6.06 7.974 7.497 14.585 7.504 22.559.02 3.987-3.76 9.5-6.087 15.586-6.087 12.166.007 22.02 9.272 22.02 20.708 0 11.437-9.854 20.695-22.02 20.695-6.1 0-11.613-2.332-15.6-6.093-7.967-7.47-14.57-7.464-22.545.032-3.987 3.748-9.494 6.061-15.573 6.061-12.158 0-22.013-9.264-22.013-20.695 0-11.43 9.855-20.701 22.013-20.701"
                        })
                    ]
                })
            })
        ]
    });
const $3467e722ad5dc363$var$ForwardRef = /*#__PURE__*/ (0, $4juzv$forwardRef)($3467e722ad5dc363$var$SvgComponent);
var $3467e722ad5dc363$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($3467e722ad5dc363$var$ForwardRef);


export {$3467e722ad5dc363$export$2e2bcd8739ae039 as default};
