// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_laptop_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $aW0Nv$jsxs, jsx as $aW0Nv$jsx} from "react/jsx-runtime";
import {forwardRef as $aW0Nv$forwardRef} from "react";





const $7be2246609b7a188$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aW0Nv$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aW0Nv$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aW0Nv$jsxs)("linearGradient", {
                        id: "8fe01b__d",
                        x1: 66.842,
                        x2: 23.028,
                        y1: 8.504,
                        y2: 68.765,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aW0Nv$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $aW0Nv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aW0Nv$jsxs)("linearGradient", {
                        id: "8fe01b__e",
                        x1: 38.053,
                        x2: 46.176,
                        y1: -18.084,
                        y2: 37.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aW0Nv$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $aW0Nv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aW0Nv$jsxs)("linearGradient", {
                        id: "8fe01b__f",
                        x1: -67.136,
                        x2: 85.529,
                        y1: 92.488,
                        y2: 60.481,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aW0Nv$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aW0Nv$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $aW0Nv$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $aW0Nv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aW0Nv$jsxs)("radialGradient", {
                        id: "8fe01b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 111.062 -117.816 0 80.538 61.548)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aW0Nv$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aW0Nv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aW0Nv$jsx)("clipPath", {
                        id: "8fe01b__b",
                        children: /*#__PURE__*/ (0, $aW0Nv$jsx)("rect", {
                            width: 81.6,
                            height: 76.922,
                            x: 7.801,
                            y: 10.979,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aW0Nv$jsx)("clipPath", {
                id: "8fe01b__a",
                children: /*#__PURE__*/ (0, $aW0Nv$jsx)("path", {
                    fill: "#fff",
                    d: "M81.6 66h-4.8V28.932c0-5.376-4.296-9.732-9.6-9.732H28.8c-5.304 0-9.6 4.356-9.6 9.732V66h-4.8C11.418 66 9 68.418 9 71.4s2.418 5.4 5.4 5.4h67.2c2.982 0 5.4-2.418 5.4-5.4S84.582 66 81.6 66M48 29.4c-1.326 0-2.4-1.074-2.4-2.4s1.074-2.4 2.4-2.4 2.4 1.074 2.4 2.4-1.074 2.4-2.4 2.4"
                })
            }),
            /*#__PURE__*/ (0, $aW0Nv$jsx)("g", {
                clipPath: "url(#8fe01b__a)",
                children: /*#__PURE__*/ (0, $aW0Nv$jsxs)("g", {
                    clipPath: "url(#8fe01b__b)",
                    children: [
                        /*#__PURE__*/ (0, $aW0Nv$jsx)("rect", {
                            width: 81.6,
                            height: 76.922,
                            x: 7.801,
                            y: 10.979,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $aW0Nv$jsx)("ellipse", {
                            cx: 80.538,
                            cy: 61.548,
                            fill: "url(#8fe01b__c)",
                            rx: 117.816,
                            ry: 111.062
                        }),
                        /*#__PURE__*/ (0, $aW0Nv$jsx)("path", {
                            fill: "url(#8fe01b__d)",
                            d: "M.456 35.07C-.854 8.484 20.949-14.07 49.153-15.305S101.284 4.012 102.594 30.6c1.309 26.588-20.493 49.142-48.698 50.377C25.692 82.21 1.766 61.658.456 35.07"
                        }),
                        /*#__PURE__*/ (0, $aW0Nv$jsx)("path", {
                            fill: "url(#8fe01b__e)",
                            d: "M28.417-42.25c-6.363-3.71-14.424-3.733-20.81-.058L-56.267-5.544c-6.255 3.6-6.266 12.099-.019 15.713L7.166 46.881c6.376 3.688 14.438 3.684 20.809-.014l63.32-36.745c6.215-3.607 6.226-12.058.02-15.678z"
                        }),
                        /*#__PURE__*/ (0, $aW0Nv$jsx)("path", {
                            fill: "url(#8fe01b__f)",
                            d: "M-32.879 47.77c13.165 0 25.09 5.024 33.725 13.127 17.268 16.234 31.585 16.248 48.853.041 8.634-8.144 20.575-13.181 33.755-13.181 26.345.014 47.687 20.078 47.687 44.845 0 24.768-21.342 44.817-47.687 44.817-13.21 0-25.15-5.05-33.784-13.195-17.254-16.179-31.556-16.165-48.824.069-8.634 8.117-20.56 13.126-33.725 13.126-26.33 0-47.672-20.063-47.672-44.817S-59.21 47.771-32.88 47.771"
                        })
                    ]
                })
            })
        ]
    });
const $7be2246609b7a188$var$ForwardRef = /*#__PURE__*/ (0, $aW0Nv$forwardRef)($7be2246609b7a188$var$SvgComponent);
var $7be2246609b7a188$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7be2246609b7a188$var$ForwardRef);


export {$7be2246609b7a188$export$2e2bcd8739ae039 as default};
