// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_magicWand_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2QalS$jsxs, jsx as $2QalS$jsx} from "react/jsx-runtime";
import {forwardRef as $2QalS$forwardRef} from "react";





const $7be499652da58a68$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2QalS$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2QalS$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2QalS$jsxs)("linearGradient", {
                        id: "16495d__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2QalS$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2QalS$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2QalS$jsxs)("linearGradient", {
                        id: "16495d__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2QalS$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2QalS$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2QalS$jsxs)("linearGradient", {
                        id: "16495d__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2QalS$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2QalS$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2QalS$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2QalS$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2QalS$jsxs)("radialGradient", {
                        id: "16495d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2QalS$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2QalS$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2QalS$jsx)("clipPath", {
                        id: "16495d__b",
                        children: /*#__PURE__*/ (0, $2QalS$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2QalS$jsx)("clipPath", {
                id: "16495d__a",
                children: /*#__PURE__*/ (0, $2QalS$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "m39.805 13.376-2.116.423c-.28.056-.513.245-.626.506l-.854 1.982c-.337.78-1.478.66-1.644-.172l-.423-2.117c-.057-.28-.245-.512-.506-.626l-1.982-.854c-.781-.336-.66-1.478.172-1.644l2.117-.423c.28-.056.512-.245.625-.506l.854-1.982c.337-.78 1.479-.66 1.644.172l.423 2.117c.057.28.246.512.506.626l1.982.854c.781.336.66 1.478-.172 1.644M21.95 15.19l-1.749.35c-.23.046-.423.202-.516.418l-.706 1.637c-.278.645-1.22.545-1.358-.142l-.35-1.749c-.046-.231-.202-.423-.417-.517l-1.637-.705c-.645-.278-.545-1.22.142-1.358l1.748-.35c.232-.046.424-.202.517-.417l.705-1.637c.278-.645 1.221-.545 1.358.142l.35 1.748c.046.232.202.424.418.517l1.637.705c.645.278.545 1.221-.142 1.358m15.516 12.957 1.749-.35c.687-.136.787-1.08.142-1.357l-1.637-.706c-.215-.093-.371-.285-.418-.516l-.35-1.748c-.136-.688-1.08-.788-1.357-.143l-.706 1.637c-.093.216-.285.372-.517.418l-1.748.35c-.687.137-.787 1.08-.142 1.358l1.637.705c.215.093.371.285.418.517l.35 1.748c.136.687 1.08.787 1.357.142l.706-1.637c.093-.215.285-.371.517-.418M26.66 17.313c1.127-1.127 2.954-1.127 4.081 0s1.128 2.955 0 4.082l-2.675 2.675-4.082-4.082zm-4.444 4.443 4.082 4.082-13.875 13.875c-1.127 1.127-2.954 1.127-4.082 0s-1.127-2.955 0-4.082z"
                })
            }),
            /*#__PURE__*/ (0, $2QalS$jsx)("g", {
                clipPath: "url(#16495d__a)",
                children: /*#__PURE__*/ (0, $2QalS$jsxs)("g", {
                    clipPath: "url(#16495d__b)",
                    children: [
                        /*#__PURE__*/ (0, $2QalS$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2QalS$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#16495d__c)"
                        }),
                        /*#__PURE__*/ (0, $2QalS$jsx)("path", {
                            fill: "url(#16495d__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $2QalS$jsx)("path", {
                            fill: "url(#16495d__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $2QalS$jsx)("path", {
                            fill: "url(#16495d__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $7be499652da58a68$var$ForwardRef = /*#__PURE__*/ (0, $2QalS$forwardRef)($7be499652da58a68$var$SvgComponent);
var $7be499652da58a68$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7be499652da58a68$var$ForwardRef);


export {$7be499652da58a68$export$2e2bcd8739ae039 as default};
