// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_buildings_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8Vc42$reactjsxruntime = require("react/jsx-runtime");
var $8Vc42$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2243cd4f69b180e1$export$2e2bcd8739ae039);




const $2243cd4f69b180e1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1b4bc3__d",
                        x1: 32.926,
                        x2: 10.021,
                        y1: 4.108,
                        y2: 35.578,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1b4bc3__e",
                        x1: 17.893,
                        x2: 22.142,
                        y1: -9.787,
                        y2: 19.031,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1b4bc3__f",
                        x1: -37.02,
                        x2: 42.687,
                        y1: 47.998,
                        y2: 31.304,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsxs)("radialGradient", {
                        id: "1b4bc3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 58.0415 -61.5067 0 40.073 31.829)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("clipPath", {
                        id: "1b4bc3__b",
                        children: /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("rect", {
                            width: 42.6,
                            height: 40.2,
                            x: 2.102,
                            y: 5.4,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("clipPath", {
                id: "1b4bc3__a",
                children: /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M10.726 8.379h14.42c2.653 0 4.8 2.148 4.8 4.8v26.439H22.15v-6.174c0-1.326-1.074-2.4-2.4-2.4h-3.627c-1.326 0-2.4 1.074-2.4 2.4v6.174H5.926v-26.44c0-2.651 2.148-4.8 4.8-4.8m3.837 16.968c.663 0 1.2-.537 1.2-1.2v-1.728c0-.663-.537-1.2-1.2-1.2h-1.728c-.663 0-1.2.537-1.2 1.2v1.728c0 .663.537 1.2 1.2 1.2zm0-8.418c.663 0 1.2-.537 1.2-1.2V14c0-.663-.537-1.2-1.2-1.2h-1.728c-.663 0-1.2.537-1.2 1.2v1.728c0 .663.537 1.2 1.2 1.2zm8.478 8.418c.663 0 1.2-.537 1.2-1.2v-1.728c0-.663-.537-1.2-1.2-1.2h-1.728c-.663 0-1.2.537-1.2 1.2v1.728c0 .663.537 1.2 1.2 1.2zm0-8.418c.663 0 1.2-.537 1.2-1.2V14c0-.663-.537-1.2-1.2-1.2h-1.728c-.663 0-1.2.537-1.2 1.2v1.728c0 .663.537 1.2 1.2 1.2zm14.232-2.061c2.652 0 4.8 2.148 4.8 4.8v19.95H32.35v-24.75zm-.4 18.648c.664 0 1.2-.537 1.2-1.2v-1.728c0-.663-.536-1.2-1.2-1.2h-1.727c-.663 0-1.2.537-1.2 1.2v1.728c0 .663.537 1.2 1.2 1.2zm0-8.418c.664 0 1.2-.537 1.2-1.2V22.17c0-.663-.536-1.2-1.2-1.2h-1.727c-.663 0-1.2.537-1.2 1.2v1.728c0 .663.537 1.2 1.2 1.2z"
                })
            }),
            /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("g", {
                clipPath: "url(#1b4bc3__a)",
                children: /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#1b4bc3__b)",
                    children: [
                        /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("rect", {
                            width: 42.6,
                            height: 40.2,
                            x: 2.102,
                            y: 5.4,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("ellipse", {
                            cx: 40.073,
                            cy: 31.829,
                            fill: "url(#1b4bc3__c)",
                            rx: 61.507,
                            ry: 58.041
                        }),
                        /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("path", {
                            fill: "url(#1b4bc3__d)",
                            d: "M-1.732 17.992C-2.415 4.097 8.967-7.69 23.691-8.335c14.725-.645 27.215 10.096 27.9 23.99.683 13.895-10.7 25.682-25.424 26.328-14.724.645-27.215-10.096-27.899-23.991"
                        }),
                        /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("path", {
                            fill: "url(#1b4bc3__e)",
                            d: "M12.862-22.416c-3.322-1.94-7.53-1.951-10.864-.03L-31.348-3.235c-3.266 1.882-3.271 6.323-.01 8.212L1.767 24.164c3.33 1.928 7.538 1.925 10.864-.007L45.688 4.953c3.244-1.884 3.25-6.301.01-8.193z"
                        }),
                        /*#__PURE__*/ (0, $8Vc42$reactjsxruntime.jsx)("path", {
                            fill: "url(#1b4bc3__f)",
                            d: "M-19.136 24.628c6.873 0 13.1 2.625 17.607 6.86 9.015 8.484 16.489 8.491 25.504.022 4.508-4.257 10.741-6.889 17.622-6.889 13.753.007 24.895 10.493 24.895 23.436 0 12.944-11.142 23.422-24.895 23.422-6.896 0-13.13-2.64-17.638-6.896-9.007-8.455-16.473-8.448-25.488.037-4.508 4.242-10.734 6.86-17.607 6.86-13.746 0-24.887-10.486-24.887-23.423s11.141-23.429 24.887-23.429"
                        })
                    ]
                })
            })
        ]
    });
const $2243cd4f69b180e1$var$ForwardRef = /*#__PURE__*/ (0, $8Vc42$react.forwardRef)($2243cd4f69b180e1$var$SvgComponent);
var $2243cd4f69b180e1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2243cd4f69b180e1$var$ForwardRef);
