// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_sparkles_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $fUPM7$jsxs, jsx as $fUPM7$jsx} from "react/jsx-runtime";
import {forwardRef as $fUPM7$forwardRef} from "react";





const $85098dac1489026d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fUPM7$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fUPM7$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fUPM7$jsxs)("linearGradient", {
                        id: "d30bc1__d",
                        x1: 65.566,
                        x2: 25.102,
                        y1: 11.636,
                        y2: 67.288,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fUPM7$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $fUPM7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fUPM7$jsxs)("linearGradient", {
                        id: "d30bc1__e",
                        x1: 38.978,
                        x2: 46.479,
                        y1: -12.92,
                        y2: 38.008,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fUPM7$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $fUPM7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fUPM7$jsxs)("linearGradient", {
                        id: "d30bc1__f",
                        x1: -58.165,
                        x2: 82.825,
                        y1: 89.197,
                        y2: 59.638,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fUPM7$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fUPM7$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $fUPM7$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $fUPM7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fUPM7$jsxs)("radialGradient", {
                        id: "d30bc1__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 78.212 60.624)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fUPM7$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fUPM7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fUPM7$jsx)("clipPath", {
                        id: "d30bc1__b",
                        children: /*#__PURE__*/ (0, $fUPM7$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fUPM7$jsx)("clipPath", {
                id: "d30bc1__a",
                children: /*#__PURE__*/ (0, $fUPM7$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m75.722 50.646-10.35-2.328c-1.536-.348-3.15.084-4.308 1.152l-7.8 7.188c-3.462 3.192-8.97.012-7.938-4.584l2.328-10.35c.348-1.536-.084-3.15-1.152-4.308l-7.188-7.8c-3.192-3.462-.012-8.97 4.584-7.938l10.35 2.328c1.536.348 3.15-.084 4.308-1.152l7.8-7.188c3.462-3.192 8.97-.012 7.938 4.584L71.966 30.6c-.348 1.536.084 3.15 1.152 4.308l7.188 7.8c3.192 3.462.012 8.97-4.584 7.938m-37.163 27.27-7.296-1.644c-.966-.216-1.969.054-2.695.72l-5.495 5.07c-2.166 1.998-5.61.006-4.963-2.862l1.645-7.296c.216-.966-.054-1.968-.72-2.694l-5.07-5.496c-1.998-2.166-.006-5.61 2.861-4.962l7.296 1.644c.966.216 1.968-.054 2.695-.72l5.496-5.07c2.165-1.998 5.61-.006 4.962 2.862l-1.645 7.296c-.216.966.054 1.968.72 2.694l5.07 5.496c1.999 2.166.006 5.61-2.861 4.962"
                })
            }),
            /*#__PURE__*/ (0, $fUPM7$jsx)("g", {
                clipPath: "url(#d30bc1__a)",
                children: /*#__PURE__*/ (0, $fUPM7$jsxs)("g", {
                    clipPath: "url(#d30bc1__b)",
                    children: [
                        /*#__PURE__*/ (0, $fUPM7$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $fUPM7$jsx)("ellipse", {
                            cx: 78.212,
                            cy: 60.624,
                            fill: "url(#d30bc1__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $fUPM7$jsx)("path", {
                            fill: "url(#d30bc1__d)",
                            d: "M4.256 36.171C3.046 11.616 23.182-9.213 49.23-10.353s48.144 17.84 49.353 42.395S79.658 77.426 53.61 78.566 5.466 60.726 4.256 36.171"
                        }),
                        /*#__PURE__*/ (0, $fUPM7$jsx)("path", {
                            fill: "url(#d30bc1__e)",
                            d: "M30.078-35.237c-5.876-3.427-13.321-3.448-19.219-.054L-48.13-1.339c-5.777 3.325-5.787 11.174-.017 14.512l58.599 33.904c5.889 3.407 13.334 3.403 19.218-.012L88.148 13.13c5.74-3.33 5.75-11.135.018-14.479z"
                        }),
                        /*#__PURE__*/ (0, $fUPM7$jsx)("path", {
                            fill: "url(#d30bc1__f)",
                            d: "M-26.528 47.9c12.159 0 23.172 4.638 31.146 12.122 15.948 14.993 29.17 15.005 45.118.038 7.974-7.522 19-12.173 31.173-12.173 24.33.013 44.04 18.542 44.04 41.415 0 22.874-19.71 41.391-44.04 41.391-12.2 0-23.227-4.665-31.2-12.187-15.935-14.941-29.143-14.928-45.09.064-7.975 7.497-18.988 12.123-31.147 12.123-24.317 0-44.026-18.53-44.026-41.39 0-22.861 19.71-41.403 44.026-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $85098dac1489026d$var$ForwardRef = /*#__PURE__*/ (0, $fUPM7$forwardRef)($85098dac1489026d$var$SvgComponent);
var $85098dac1489026d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($85098dac1489026d$var$ForwardRef);


export {$85098dac1489026d$export$2e2bcd8739ae039 as default};
