// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_cloud_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $bvBQM$reactjsxruntime = require("react/jsx-runtime");
var $bvBQM$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f8f0f83706244f81$export$2e2bcd8739ae039);




const $f8f0f83706244f81$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6397b3__d",
                        x1: 66.664,
                        x2: 18.316,
                        y1: 3.572,
                        y2: 66.257,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6397b3__e",
                        x1: 37.208,
                        x2: 46.535,
                        y1: -25.285,
                        y2: 34.406,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6397b3__f",
                        x1: -70.407,
                        x2: 86.52,
                        y1: 94.721,
                        y2: 63.707,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6397b3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 9.766 70.908)scale(120.537)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("clipPath", {
                        id: "6397b3__b",
                        children: /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("clipPath", {
                id: "6397b3__a",
                children: /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M69.914 41.524c.002-.126.009-.25.009-.378 0-9.87-7.666-17.87-17.123-17.87-7.629 0-14.09 5.208-16.303 12.403-1.469-.633-3.076-.987-4.764-.987-6.901 0-12.496 5.839-12.496 13.042 0 1.015.122 1.998.332 2.946-4.558 1.303-7.913 5.646-7.913 10.817 0 6.138 4.72 11.117 10.578 11.218v.01h46.739v-.003q.042.002.083.003c8.263 0 14.96-6.99 14.96-15.614 0-8.323-6.24-15.123-14.102-15.587"
                })
            }),
            /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6397b3__a)",
                children: /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6397b3__b)",
                    children: [
                        /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("circle", {
                            cx: 80.674,
                            cy: 61.143,
                            r: 120.537,
                            fill: "url(#6397b3__c)"
                        }),
                        /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("path", {
                            fill: "url(#6397b3__d)",
                            d: "M-1.255 32.406C-2.595 3.549 19.71-20.93 48.567-22.27s53.335 20.966 54.675 49.822-20.967 53.335-49.823 54.675S.085 61.262-1.255 32.406"
                        }),
                        /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("path", {
                            fill: "url(#6397b3__e)",
                            d: "M27.349-51.511c-6.51-4.028-14.757-4.053-21.291-.064l-65.35 39.9c-6.4 3.908-6.41 13.132-.018 17.054L5.607 45.223c6.524 4.003 14.771 3.998 21.29-.015L91.68 5.328c6.358-3.915 6.37-13.087.02-17.016z"
                        }),
                        /*#__PURE__*/ (0, $bvBQM$reactjsxruntime.jsx)("path", {
                            fill: "url(#6397b3__f)",
                            d: "M-35.36 46.189c13.47 0 25.671 5.451 34.505 14.246 17.667 17.619 32.314 17.634 49.981.045 8.834-8.84 21.05-14.306 34.535-14.306 26.953.015 48.788 21.79 48.788 48.671s-21.835 48.641-48.789 48.641c-13.514 0-25.73-5.482-34.564-14.321-17.652-17.559-32.284-17.544-49.951.075-8.834 8.81-21.035 14.246-34.504 14.246-26.939 0-48.774-21.775-48.774-48.641 0-26.865 21.835-48.656 48.774-48.656"
                        })
                    ]
                })
            })
        ]
    });
const $f8f0f83706244f81$var$ForwardRef = /*#__PURE__*/ (0, $bvBQM$react.forwardRef)($f8f0f83706244f81$var$SvgComponent);
var $f8f0f83706244f81$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f8f0f83706244f81$var$ForwardRef);
