// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_star_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7SGNg$jsxs, jsx as $7SGNg$jsx} from "react/jsx-runtime";
import {forwardRef as $7SGNg$forwardRef} from "react";





const $2f4c9455983f2a06$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7SGNg$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7SGNg$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7SGNg$jsxs)("linearGradient", {
                        id: "f2571a__d",
                        x1: 66.569,
                        x2: 18.461,
                        y1: 3.793,
                        y2: 66.165,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SGNg$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7SGNg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SGNg$jsxs)("linearGradient", {
                        id: "f2571a__e",
                        x1: 37.259,
                        x2: 46.539,
                        y1: -24.922,
                        y2: 34.472,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SGNg$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7SGNg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SGNg$jsxs)("linearGradient", {
                        id: "f2571a__f",
                        x1: -69.82,
                        x2: 86.326,
                        y1: 94.487,
                        y2: 63.627,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SGNg$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7SGNg$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7SGNg$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7SGNg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SGNg$jsxs)("radialGradient", {
                        id: "f2571a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 119.937 -119.938 0 80.512 61.076)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7SGNg$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7SGNg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7SGNg$jsx)("clipPath", {
                        id: "f2571a__b",
                        children: /*#__PURE__*/ (0, $7SGNg$jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7SGNg$jsx)("clipPath", {
                id: "f2571a__a",
                children: /*#__PURE__*/ (0, $7SGNg$jsx)("path", {
                    fill: "#fff",
                    d: "m52.923 19.862 5.796 11.744c.8 1.62 2.346 2.744 4.135 3.004l12.96 1.883c4.504.654 6.303 6.19 3.044 9.366l-9.379 9.142c-1.294 1.261-1.885 3.079-1.579 4.86l2.214 12.908c.77 4.486-3.939 7.907-7.967 5.789l-11.593-6.095c-1.6-.84-3.51-.84-5.11 0l-11.592 6.095c-4.029 2.118-8.737-1.303-7.968-5.789l2.214-12.908c.306-1.781-.285-3.599-1.579-4.86l-9.378-9.142c-3.26-3.177-1.46-8.711 3.043-9.366l12.96-1.883c1.789-.26 3.335-1.383 4.135-3.004l5.796-11.744c2.014-4.081 7.834-4.081 9.848 0"
                })
            }),
            /*#__PURE__*/ (0, $7SGNg$jsx)("g", {
                clipPath: "url(#f2571a__a)",
                children: /*#__PURE__*/ (0, $7SGNg$jsxs)("g", {
                    clipPath: "url(#f2571a__b)",
                    children: [
                        /*#__PURE__*/ (0, $7SGNg$jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7SGNg$jsx)("ellipse", {
                            cx: 80.512,
                            cy: 61.076,
                            fill: "url(#f2571a__c)",
                            rx: 119.938,
                            ry: 119.937
                        }),
                        /*#__PURE__*/ (0, $7SGNg$jsx)("path", {
                            fill: "url(#f2571a__d)",
                            d: "M-1.012 32.482C-2.346 3.77 19.85-20.588 48.562-21.92c28.713-1.334 53.07 20.862 54.403 49.574 1.333 28.713-20.862 53.07-49.575 54.403C24.678 83.39.321 61.195-1.012 32.482"
                        }),
                        /*#__PURE__*/ (0, $7SGNg$jsx)("path", {
                            fill: "url(#f2571a__e)",
                            d: "M27.449-51.018c-6.478-4.008-14.684-4.032-21.185-.063L-58.76-11.38c-6.369 3.889-6.38 13.067-.02 16.97L5.815 45.235c6.491 3.984 14.698 3.979 21.184-.014l64.46-39.683c6.327-3.894 6.338-13.02.02-16.93z"
                        }),
                        /*#__PURE__*/ (0, $7SGNg$jsx)("path", {
                            fill: "url(#f2571a__f)",
                            d: "M-34.946 46.197c13.402 0 25.542 5.424 34.332 14.175 17.58 17.531 32.154 17.546 49.733.045 8.79-8.796 20.945-14.235 34.362-14.235 26.82.015 48.546 21.682 48.546 48.429 0 26.746-21.726 48.399-48.546 48.399-13.447 0-25.602-5.455-34.392-14.25-17.564-17.472-32.124-17.457-49.703.075-8.79 8.765-20.93 14.175-34.332 14.175-26.805 0-48.531-21.667-48.531-48.4 0-26.731 21.726-48.413 48.53-48.413"
                        })
                    ]
                })
            })
        ]
    });
const $2f4c9455983f2a06$var$ForwardRef = /*#__PURE__*/ (0, $7SGNg$forwardRef)($2f4c9455983f2a06$var$SvgComponent);
var $2f4c9455983f2a06$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2f4c9455983f2a06$var$ForwardRef);


export {$2f4c9455983f2a06$export$2e2bcd8739ae039 as default};
