// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fireworks_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $1M228$jsxs, jsx as $1M228$jsx} from "react/jsx-runtime";
import {forwardRef as $1M228$forwardRef} from "react";





const $7808f3d2bedfe657$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1M228$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1M228$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1M228$jsxs)("linearGradient", {
                        id: "4deb84__d",
                        x1: 107.785,
                        x2: 37.447,
                        y1: 16.683,
                        y2: 113.425,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1M228$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $1M228$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1M228$jsxs)("linearGradient", {
                        id: "4deb84__e",
                        x1: 61.565,
                        x2: 74.605,
                        y1: -26.004,
                        y2: 62.525,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1M228$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $1M228$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1M228$jsxs)("linearGradient", {
                        id: "4deb84__f",
                        x1: -107.301,
                        x2: 137.787,
                        y1: 151.51,
                        y2: 100.127,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1M228$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1M228$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $1M228$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $1M228$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1M228$jsxs)("radialGradient", {
                        id: "4deb84__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 178.298 -189.141 0 129.774 101.839)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1M228$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1M228$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1M228$jsx)("clipPath", {
                        id: "4deb84__b",
                        children: /*#__PURE__*/ (0, $1M228$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1M228$jsx)("clipPath", {
                id: "4deb84__a",
                children: /*#__PURE__*/ (0, $1M228$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M128.11 28.244c.035-2.21-1.727-4.028-3.936-4.064-2.209-.035-4.028 1.727-4.063 3.936l-.056 3.498-3.503-.054c-2.209-.034-4.027 1.73-4.061 3.938-.034 2.21 1.729 4.027 3.938 4.061l3.498.054-.056 3.503c-.036 2.209 1.726 4.028 3.935 4.063s4.028-1.727 4.064-3.935l.056-3.508 3.503.053c2.209.034 4.027-1.73 4.061-3.938s-1.73-4.027-3.938-4.06l-3.498-.055zM66.99 30.79c0-2.21-1.79-4-4-4s-4 1.79-4 4v10.14c0 2.209 1.79 4 4 4s4-1.791 4-4zm-25.03 2.823c1.788-1.298 4.29-.9 5.587.887l5.96 8.21c1.298 1.788.9 4.289-.887 5.587-1.788 1.297-4.29.9-5.587-.887l-5.96-8.21c-1.298-1.788-.9-4.29.887-5.587m-7.965 15.332c-2.1-.682-4.357.468-5.04 2.57-.682 2.1.469 4.357 2.57 5.04l9.64 3.13c2.1.681 4.357-.469 5.04-2.57s-.469-4.358-2.57-5.04zm12.21 19.08c.681 2.102-.47 4.358-2.57 5.04l-9.65 3.13c-2.102.681-4.358-.47-5.04-2.571-.681-2.102.47-4.358 2.57-5.04l9.65-3.13c2.102-.68 4.358.47 5.04 2.572m7.29 14.416c1.3-1.787.904-4.288-.883-5.587s-4.289-.903-5.588.884l-5.96 8.2c-1.298 1.787-.903 4.288.884 5.587 1.787 1.3 4.289.903 5.588-.884zM66.98 84.22c0-2.21-1.79-4-4-4s-4 1.79-4 4v10.14c0 2.209 1.79 4 4 4s4-1.791 4-4zm17.052-50.596c1.787 1.299 2.183 3.8.884 5.587l-5.96 8.2c-1.3 1.787-3.8 2.183-5.588.884-1.787-1.299-2.182-3.8-.884-5.587l5.96-8.2c1.3-1.787 3.8-2.183 5.588-.884m10.423 22.94c2.101-.682 3.252-2.938 2.57-5.04-.683-2.1-2.94-3.251-5.04-2.569l-9.64 3.13c-2.101.682-3.252 2.939-2.57 5.04.683 2.101 2.94 3.251 5.04 2.57zm-14.68 11.47c.683-2.1 2.94-3.251 5.04-2.569l9.64 3.13c2.101.682 3.252 2.939 2.57 5.04-.683 2.101-2.94 3.251-5.04 2.57l-9.64-3.13c-2.101-.683-3.252-2.94-2.57-5.04m-.828 9.706c-1.298-1.788-3.8-2.185-5.587-.887-1.788 1.297-2.185 3.799-.887 5.587l5.96 8.21c1.298 1.787 3.8 2.185 5.587.887 1.788-1.298 2.185-3.8.887-5.587zm30.09 37.22c2.209.009 3.992 1.807 3.984 4.016l-.02 4.95c-.009 2.209-1.807 3.993-4.017 3.984-2.209-.009-3.992-1.807-3.983-4.016l.02-4.95c.009-2.21 1.807-3.993 4.016-3.984m-8.94 1.182c1.917-1.097 2.583-3.541 1.486-5.459s-3.541-2.582-5.459-1.485l-4.3 2.46c-1.917 1.097-2.582 3.54-1.485 5.458s3.54 2.583 5.458 1.486zm-9.734-20.574c1.111-1.91 3.56-2.557 5.469-1.446l4.28 2.49c1.91 1.111 2.557 3.56 1.446 5.469s-3.56 2.557-5.469 1.446l-4.28-2.49c-1.91-1.11-2.557-3.56-1.446-5.469m22.658-6.742c.008-2.21-1.775-4.007-3.984-4.016s-4.007 1.774-4.016 3.983l-.02 4.95c-.009 2.21 1.774 4.008 3.983 4.017s4.008-1.775 4.017-3.984zm14.672 6.777c1.097 1.918.431 4.362-1.486 5.459l-4.3 2.46c-1.918 1.097-4.361.432-5.458-1.486s-.432-4.361 1.485-5.458l4.3-2.46c1.918-1.097 4.362-.432 5.459 1.485m-5.751 13.589c-1.909-1.111-4.358-.463-5.469 1.446s-.463 4.358 1.446 5.469l4.28 2.49c1.91 1.111 4.358.464 5.469-1.446s.464-4.358-1.446-5.469zm-98.461 15.296c.034-2.209 1.852-3.972 4.06-3.938l3.496.054.054-3.506c.034-2.209 1.852-3.972 4.06-3.938 2.21.034 3.973 1.852 3.939 4.061l-.054 3.505 3.506.054c2.209.034 3.972 1.852 3.938 4.061s-1.852 3.972-4.06 3.938l-3.506-.054-.054 3.496c-.034 2.209-1.852 3.972-4.06 3.938-2.21-.034-3.973-1.852-3.94-4.061l.055-3.495-3.496-.054c-2.209-.034-3.972-1.852-3.938-4.061"
                })
            }),
            /*#__PURE__*/ (0, $1M228$jsx)("g", {
                clipPath: "url(#4deb84__a)",
                children: /*#__PURE__*/ (0, $1M228$jsxs)("g", {
                    clipPath: "url(#4deb84__b)",
                    children: [
                        /*#__PURE__*/ (0, $1M228$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $1M228$jsx)("ellipse", {
                            cx: 129.774,
                            cy: 101.839,
                            fill: "url(#4deb84__c)",
                            rx: 189.141,
                            ry: 178.298
                        }),
                        /*#__PURE__*/ (0, $1M228$jsx)("path", {
                            fill: "url(#4deb84__d)",
                            d: "M1.21 59.333C-.893 16.649 34.11-19.56 79.389-21.542s83.69 31.014 85.792 73.698-32.899 78.892-78.178 80.874c-45.28 1.982-83.69-31.013-85.793-73.697"
                        }),
                        /*#__PURE__*/ (0, $1M228$jsx)("path", {
                            fill: "url(#4deb84__e)",
                            d: "M46.095-64.799c-10.215-5.958-23.156-5.994-33.409-.094L-89.856-5.873c-10.043 5.78-10.06 19.425-.03 25.227L11.977 78.29c10.237 5.922 23.179 5.915 33.407-.02L147.04 19.277c9.976-5.79 9.994-19.358.031-25.17z"
                        }),
                        /*#__PURE__*/ (0, $1M228$jsx)("path", {
                            fill: "url(#4deb84__f)",
                            d: "M-52.305 79.722c21.136 0 40.28 8.063 54.142 21.072 27.723 26.062 50.706 26.085 78.429.067 13.861-13.076 33.03-21.162 54.189-21.162 42.295.023 76.557 32.233 76.557 71.994s-34.262 71.95-76.557 71.95c-21.207 0-40.375-8.108-54.237-21.184-27.699-25.973-50.658-25.951-78.38.112-13.862 13.031-33.007 21.072-54.143 21.072-42.27 0-76.533-32.21-76.533-71.95 0-39.739 34.262-71.971 76.533-71.971"
                        })
                    ]
                })
            })
        ]
    });
const $7808f3d2bedfe657$var$ForwardRef = /*#__PURE__*/ (0, $1M228$forwardRef)($7808f3d2bedfe657$var$SvgComponent);
var $7808f3d2bedfe657$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7808f3d2bedfe657$var$ForwardRef);


export {$7808f3d2bedfe657$export$2e2bcd8739ae039 as default};
