// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_artboard_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6rLj0$jsxs, jsx as $6rLj0$jsx} from "react/jsx-runtime";
import {forwardRef as $6rLj0$forwardRef} from "react";





const $ced0dd0ae04e6883$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6rLj0$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6rLj0$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6rLj0$jsxs)("linearGradient", {
                        id: "2046d8__d",
                        x1: 31.203,
                        x2: 8.716,
                        y1: 1.731,
                        y2: 32.659,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6rLj0$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6rLj0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6rLj0$jsxs)("linearGradient", {
                        id: "2046d8__e",
                        x1: 16.426,
                        x2: 20.595,
                        y1: -11.916,
                        y2: 16.386,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6rLj0$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6rLj0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6rLj0$jsxs)("linearGradient", {
                        id: "2046d8__f",
                        x1: -37.561,
                        x2: 40.793,
                        y1: 44.835,
                        y2: 28.408,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6rLj0$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6rLj0$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6rLj0$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6rLj0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6rLj0$jsxs)("radialGradient", {
                        id: "2046d8__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 57.0019 -60.4683 0 38.23 28.955)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6rLj0$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6rLj0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6rLj0$jsx)("clipPath", {
                        id: "2046d8__b",
                        children: /*#__PURE__*/ (0, $6rLj0$jsx)("rect", {
                            width: 41.881,
                            height: 39.48,
                            x: 0.898,
                            y: 3,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6rLj0$jsx)("clipPath", {
                id: "2046d8__a",
                children: /*#__PURE__*/ (0, $6rLj0$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M12 7.2v-3c0-.663-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2v3c0 .663.537 1.2 1.2 1.2S12 7.863 12 7.2m11.7 13.806c0 1.65 1.35 2.994 3.009 2.994h10.494v14.4c0 1.989-1.602 3.6-3.579 3.6H14.379c-1.977 0-3.579-1.611-3.579-3.6v-24c0-1.989 1.602-3.6 3.579-3.6H23.7zm12.69.594h-9.678c-.336 0-.61-.267-.61-.594v-9.339l.047.04q.053.043.104.092l9.843 9.477c.105.102.2.21.294.324M4.2 9.6h3c.663 0 1.2.537 1.2 1.2S7.863 12 7.2 12h-3c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.2-1.2"
                })
            }),
            /*#__PURE__*/ (0, $6rLj0$jsx)("g", {
                clipPath: "url(#2046d8__a)",
                children: /*#__PURE__*/ (0, $6rLj0$jsxs)("g", {
                    clipPath: "url(#2046d8__b)",
                    children: [
                        /*#__PURE__*/ (0, $6rLj0$jsx)("rect", {
                            width: 41.881,
                            height: 39.48,
                            x: 0.898,
                            y: 3,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6rLj0$jsx)("ellipse", {
                            cx: 38.23,
                            cy: 28.955,
                            fill: "url(#2046d8__c)",
                            rx: 60.468,
                            ry: 57.002
                        }),
                        /*#__PURE__*/ (0, $6rLj0$jsx)("path", {
                            fill: "url(#2046d8__d)",
                            d: "M-2.87 15.366C-3.54 1.72 7.65-9.856 22.126-10.49 36.6-11.124 48.88-.575 49.553 13.071c.672 13.646-10.518 25.222-24.994 25.856-14.476.633-26.756-9.915-27.428-23.561"
                        }),
                        /*#__PURE__*/ (0, $6rLj0$jsx)("path", {
                            fill: "url(#2046d8__e)",
                            d: "M11.48-24.319c-3.265-1.905-7.403-1.917-10.68-.03L-31.983-5.48c-3.211 1.848-3.216 6.21-.01 8.064L.573 21.426c3.273 1.894 7.41 1.891 10.68-.006l32.5-18.86c3.189-1.85 3.195-6.188.01-8.046z"
                        }),
                        /*#__PURE__*/ (0, $6rLj0$jsx)("path", {
                            fill: "url(#2046d8__f)",
                            d: "M-19.98 21.884c6.758 0 12.879 2.578 17.31 6.737 8.863 8.332 16.21 8.34 25.074.021 4.431-4.18 10.56-6.765 17.324-6.765 13.521.007 24.475 10.305 24.475 23.017S53.25 67.896 39.728 67.896c-6.78 0-12.908-2.592-17.34-6.773-8.855-8.303-16.195-8.296-25.058.036-4.431 4.166-10.552 6.737-17.31 6.737-13.513 0-24.467-10.298-24.467-23.002 0-12.705 10.954-23.01 24.468-23.01"
                        })
                    ]
                })
            })
        ]
    });
const $ced0dd0ae04e6883$var$ForwardRef = /*#__PURE__*/ (0, $6rLj0$forwardRef)($ced0dd0ae04e6883$var$SvgComponent);
var $ced0dd0ae04e6883$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ced0dd0ae04e6883$var$ForwardRef);


export {$ced0dd0ae04e6883$export$2e2bcd8739ae039 as default};
