// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_tablet_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $dwp9j$reactjsxruntime = require("react/jsx-runtime");
var $dwp9j$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d1c5e241043a6053$export$2e2bcd8739ae039);




const $d1c5e241043a6053$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4261e5__d",
                        x1: 66.779,
                        x2: 21.676,
                        y1: 7.301,
                        y2: 69.334,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4261e5__e",
                        x1: 37.142,
                        x2: 45.503,
                        y1: -20.07,
                        y2: 36.696,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4261e5__f",
                        x1: -71.139,
                        x2: 86.016,
                        y1: 93.754,
                        y2: 60.806,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4261e5__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 114.328 -121.281 0 80.874 61.906)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("clipPath", {
                        id: "4261e5__b",
                        children: /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("rect", {
                            width: 84,
                            height: 79.185,
                            x: 6,
                            y: 9.848,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("clipPath", {
                id: "4261e5__a",
                children: /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M76.802 16.8h-57.6c-5.304 0-9.6 4.296-9.6 9.6v43.2c0 5.304 4.296 9.6 9.6 9.6h57.6c5.304 0 9.6-4.296 9.6-9.6V26.4c0-5.304-4.296-9.6-9.6-9.6m-56.4 33.6c-1.326 0-2.4-1.074-2.4-2.4s1.068-2.4 2.394-2.4h.006c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4"
                })
            }),
            /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("g", {
                clipPath: "url(#4261e5__a)",
                children: /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#4261e5__b)",
                    children: [
                        /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("rect", {
                            width: 84,
                            height: 79.185,
                            x: 6,
                            y: 9.848,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("ellipse", {
                            cx: 80.874,
                            cy: 61.906,
                            fill: "url(#4261e5__c)",
                            rx: 121.281,
                            ry: 114.328
                        }),
                        /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("path", {
                            fill: "url(#4261e5__d)",
                            d: "M-1.56 34.649C-2.906 7.279 19.537-15.94 48.57-17.21c29.035-1.27 53.664 19.887 55.012 47.256 1.349 27.37-21.095 50.588-50.13 51.859C24.42 83.175-.21 62.018-1.558 34.649"
                        }),
                        /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("path", {
                            fill: "url(#4261e5__e)",
                            d: "M27.222-44.946c-6.55-3.82-14.849-3.844-21.423-.06L-59.953-7.163c-6.44 3.707-6.45 12.455-.02 16.176L5.346 46.805c6.565 3.797 14.863 3.793 21.422-.014L91.95 8.965c6.397-3.713 6.408-12.412.02-16.139z"
                        }),
                        /*#__PURE__*/ (0, $dwp9j$reactjsxruntime.jsx)("path", {
                            fill: "url(#4261e5__f)",
                            d: "M-35.875 47.722c13.553 0 25.829 5.17 34.717 13.512 17.776 16.712 32.514 16.726 50.29.043 8.888-8.384 21.18-13.569 34.747-13.569 27.12.014 49.09 20.668 49.09 46.164s-21.97 46.136-49.09 46.136c-13.598 0-25.89-5.199-34.777-13.584-17.761-16.654-32.484-16.64-50.26.072-8.888 8.355-21.164 13.512-34.717 13.512-27.105 0-49.074-20.654-49.074-46.136 0-25.481 21.97-46.15 49.074-46.15"
                        })
                    ]
                })
            })
        ]
    });
const $d1c5e241043a6053$var$ForwardRef = /*#__PURE__*/ (0, $dwp9j$react.forwardRef)($d1c5e241043a6053$var$SvgComponent);
var $d1c5e241043a6053$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d1c5e241043a6053$var$ForwardRef);
