// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_market_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $10UbJ$reactjsxruntime = require("react/jsx-runtime");
var $10UbJ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f36fe3b47f2d944e$export$2e2bcd8739ae039);




const $f36fe3b47f2d944e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7ef25c__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7ef25c__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7ef25c__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "7ef25c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("clipPath", {
                        id: "7ef25c__b",
                        children: /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("clipPath", {
                id: "7ef25c__a",
                children: /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M22 48.446V70c0 4.418 3.582 8 8 8h8V48.367c-2.643-.463-4.94-1.936-6.474-4.006-1.87 2.525-4.877 4.161-8.263 4.161q-.641 0-1.263-.076m20-.162V64h32V48.458q-.57.064-1.158.064c-3.386 0-6.393-1.636-8.263-4.161-1.87 2.525-4.877 4.161-8.263 4.161s-6.393-1.636-8.263-4.161c-1.451 1.96-3.587 3.383-6.053 3.923m-2.21-3.762c-3.467 0-6.264-2.797-6.264-6.23v-.407c0-1.105-.895-2-2-2s-2 .895-2 2v.406c0 3.434-2.796 6.231-6.263 6.231S17 41.725 17 38.292c0-1.828.398-3.634 1.17-5.29V33l5.09-10.942v-.001c.529-1.136 1.673-1.867 2.938-1.867h43.717c1.265 0 2.41.731 2.939 1.867l5.017 10.787q.028.078.064.155c.77 1.659 1.17 3.465 1.17 5.291 0 3.434-2.797 6.231-6.263 6.231s-6.263-2.797-6.263-6.23v-.407c0-1.105-.895-2-2-2s-2 .895-2 2v.406c0 3.434-2.797 6.231-6.263 6.231-3.467 0-6.263-2.797-6.263-6.23v-.407c0-1.105-.896-2-2-2s-2 .895-2 2v.406c0 3.434-2.797 6.231-6.264 6.231M42 68h32v2c0 4.418-3.582 8-8 8H42z"
                })
            }),
            /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("g", {
                clipPath: "url(#7ef25c__a)",
                children: /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#7ef25c__b)",
                    children: [
                        /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#7ef25c__c)"
                        }),
                        /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("path", {
                            fill: "url(#7ef25c__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("path", {
                            fill: "url(#7ef25c__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $10UbJ$reactjsxruntime.jsx)("path", {
                            fill: "url(#7ef25c__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $f36fe3b47f2d944e$var$ForwardRef = /*#__PURE__*/ (0, $10UbJ$react.forwardRef)($f36fe3b47f2d944e$var$SvgComponent);
var $f36fe3b47f2d944e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f36fe3b47f2d944e$var$ForwardRef);
