// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lockClose_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $dxcyO$reactjsxruntime = require("react/jsx-runtime");
var $dxcyO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $815a9c282ebb2846$export$2e2bcd8739ae039);




const $815a9c282ebb2846$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b66b12__d",
                        x1: 111.107,
                        x2: 30.526,
                        y1: 5.955,
                        y2: 110.43,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b66b12__e",
                        x1: 62.011,
                        x2: 77.555,
                        y1: -42.141,
                        y2: 57.344,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b66b12__f",
                        x1: -117.348,
                        x2: 144.197,
                        y1: 157.869,
                        y2: 106.178,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b66b12__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.277 118.181)scale(200.895)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("clipPath", {
                        id: "b66b12__b",
                        children: /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.43,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("clipPath", {
                id: "b66b12__a",
                children: /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M54 47c0-14.36 11.64-26 26-26s26 11.64 26 26v20.5h10c7.18 0 13 5.82 13 13v39c0 7.18-5.82 13-13 13H44c-7.18 0-13-5.82-13-13v-39c0-7.18 5.82-13 13-13h10zm8 20.5h36V47c0-9.941-8.059-18-18-18s-18 8.059-18 18zm22 34.43c2.391-1.383 4-3.969 4-6.93 0-4.418-3.582-8-8-8s-8 3.582-8 8c0 2.961 1.609 5.547 4 6.93V110c0 2.209 1.79 4 4 4s4-1.791 4-4z",
                    opacity: 0.9
                })
            }),
            /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("g", {
                clipPath: "url(#b66b12__a)",
                children: /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#b66b12__b)",
                    children: [
                        /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.43,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("circle", {
                            cx: 134.458,
                            cy: 101.904,
                            r: 200.895,
                            fill: "url(#b66b12__c)"
                        }),
                        /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("path", {
                            fill: "url(#b66b12__d)",
                            d: "M-2.092 54.01C-4.325 5.917 32.852-34.88 80.945-37.114S169.837-2.17 172.07 45.923s-34.944 88.891-83.038 91.124C40.94 139.281.142 102.104-2.092 54.01"
                        }),
                        /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("path", {
                            fill: "url(#b66b12__e)",
                            d: "M45.579-85.852c-10.85-6.714-24.595-6.755-35.485-.106l-108.915 66.5c-10.667 6.513-10.685 21.886-.032 28.424L9.341 75.37c10.873 6.673 24.62 6.665 35.483-.023L152.798 8.88c10.596-6.524 10.615-21.811.032-28.36z"
                        }),
                        /*#__PURE__*/ (0, $dxcyO$reactjsxruntime.jsx)("path", {
                            fill: "url(#b66b12__f)",
                            d: "M-58.935 76.982c22.45 0 42.784 9.086 57.507 23.743 29.445 29.366 53.857 29.391 83.303.076 14.722-14.733 35.082-23.844 57.557-23.844 44.923.025 81.314 36.318 81.314 81.119s-36.391 81.068-81.314 81.068c-22.525 0-42.885-9.136-57.608-23.869-29.42-29.265-53.807-29.24-83.252.126-14.723 14.682-35.058 23.743-57.507 23.743-44.897 0-81.289-36.293-81.289-81.068 0-44.776 36.392-81.094 81.29-81.094"
                        })
                    ]
                })
            })
        ]
    });
const $815a9c282ebb2846$var$ForwardRef = /*#__PURE__*/ (0, $dxcyO$react.forwardRef)($815a9c282ebb2846$var$SvgComponent);
var $815a9c282ebb2846$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($815a9c282ebb2846$var$ForwardRef);
