// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_helpCircle_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $aL8dF$reactjsxruntime = require("react/jsx-runtime");
var $aL8dF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $aafafb5dd7f61d58$export$2e2bcd8739ae039);




const $aafafb5dd7f61d58$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "24f328__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "24f328__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "24f328__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsxs)("radialGradient", {
                        id: "24f328__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("clipPath", {
                        id: "24f328__b",
                        children: /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("clipPath", {
                id: "24f328__a",
                children: /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M48 81.016c18.225 0 33-14.775 33-33s-14.775-33-33-33-33 14.774-33 33 14.775 33 33 33m.743-46.88c-.883-.18-1.8-.088-2.635.267-.836.355-1.558.959-2.07 1.744s-.788 1.713-.788 2.666c0 1.105-.895 2-2 2s-2-.895-2-2c0-1.723.498-3.411 1.438-4.852.94-1.44 2.28-2.57 3.857-3.24 1.579-.67 3.318-.846 4.997-.504s3.213 1.185 4.413 2.415 2.011 2.789 2.34 4.48c.328 1.69.16 3.444-.486 5.04-.645 1.596-1.74 2.968-3.156 3.937-.753.516-1.487 1.215-2.017 1.981-.536.775-.782 1.495-.782 2.091v4.174c0 1.105-.895 2-2 2s-2-.895-2-2v-4.174c0-1.658.659-3.162 1.492-4.367.84-1.213 1.94-2.247 3.047-3.005.757-.518 1.354-1.26 1.708-2.136.355-.877.448-1.845.267-2.779s-.628-1.785-1.276-2.448c-.647-.663-1.465-1.11-2.349-1.29m1.757 28.38c0 1.38-1.12 2.5-2.5 2.5s-2.5-1.12-2.5-2.5c0-1.381 1.12-2.5 2.5-2.5s2.5 1.119 2.5 2.5"
                })
            }),
            /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("g", {
                clipPath: "url(#24f328__a)",
                children: /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#24f328__b)",
                    children: [
                        /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#24f328__c)"
                        }),
                        /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("path", {
                            fill: "url(#24f328__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("path", {
                            fill: "url(#24f328__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $aL8dF$reactjsxruntime.jsx)("path", {
                            fill: "url(#24f328__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $aafafb5dd7f61d58$var$ForwardRef = /*#__PURE__*/ (0, $aL8dF$react.forwardRef)($aafafb5dd7f61d58$var$SvgComponent);
var $aafafb5dd7f61d58$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($aafafb5dd7f61d58$var$ForwardRef);
