// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_folderShared_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $itDk6$reactjsxruntime = require("react/jsx-runtime");
var $itDk6$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d22d9a949fc26b75$export$2e2bcd8739ae039);




const $d22d9a949fc26b75$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "44ffff__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "44ffff__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "44ffff__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsxs)("radialGradient", {
                        id: "44ffff__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("clipPath", {
                        id: "44ffff__b",
                        children: /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("clipPath", {
                id: "44ffff__a",
                children: /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M20 43h51.544l-9.318-11.547C60.466 29.27 57.8 28 54.98 28h-25.7C24.155 28 20 32.119 20 37.2zm0 8h106.154C133.796 51 140 57.247 140 64.941v53.118c0 7.694-6.204 13.941-13.846 13.941H33.846C26.204 132 20 125.753 20 118.059zm52.294 26.334c-.803 0-1.455.651-1.455 1.455 0 .41.165.776.443 1.046 1.588 1.536 1.629 4.069.092 5.656s-4.07 1.628-5.656.091c-1.77-1.714-2.879-4.127-2.879-6.793 0-5.222 4.233-9.455 9.455-9.455 2.21 0 4 1.79 4 4s-1.79 4-4 4m3.852 4.823c0-4.916 3.986-8.901 8.902-8.901s8.902 3.985 8.902 8.901-3.986 8.903-8.902 8.903-8.902-3.986-8.902-8.903M72.11 92.292c.952 1.993.108 4.381-1.886 5.333-1.906.91-3.39 2.804-3.78 5.106-.37 2.178-2.436 3.644-4.614 3.274s-3.643-2.435-3.274-4.613c.823-4.845 3.946-8.944 8.22-10.986 1.994-.952 4.382-.108 5.334 1.886m-1.691 11.399c1.55-6.436 7.782-10.827 14.63-10.827 6.846 0 13.078 4.39 14.629 10.827.977 4.057-2.457 6.977-5.714 6.977H76.133c-3.257 0-6.692-2.92-5.714-6.977"
                })
            }),
            /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("g", {
                clipPath: "url(#44ffff__a)",
                children: /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#44ffff__b)",
                    children: [
                        /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#44ffff__c)"
                        }),
                        /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("path", {
                            fill: "url(#44ffff__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("path", {
                            fill: "url(#44ffff__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $itDk6$reactjsxruntime.jsx)("path", {
                            fill: "url(#44ffff__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $d22d9a949fc26b75$var$ForwardRef = /*#__PURE__*/ (0, $itDk6$react.forwardRef)($d22d9a949fc26b75$var$SvgComponent);
var $d22d9a949fc26b75$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d22d9a949fc26b75$var$ForwardRef);
