// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_buildings_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8qJfg$reactjsxruntime = require("react/jsx-runtime");
var $8qJfg$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $58325690b231931d$export$2e2bcd8739ae039);




const $58325690b231931d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsxs)("linearGradient", {
                        id: "640a9f__d",
                        x1: 109.744,
                        x2: 33.395,
                        y1: 13.691,
                        y2: 118.59,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsxs)("linearGradient", {
                        id: "640a9f__e",
                        x1: 59.643,
                        x2: 73.807,
                        y1: -32.629,
                        y2: 63.43,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsxs)("linearGradient", {
                        id: "640a9f__f",
                        x1: -123.403,
                        x2: 142.287,
                        y1: 159.99,
                        y2: 104.346,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsxs)("radialGradient", {
                        id: "640a9f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 193.472 -205.022 0 133.577 106.095)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("clipPath", {
                        id: "640a9f__b",
                        children: /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("rect", {
                            width: 142,
                            height: 134,
                            x: 7,
                            y: 18,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("clipPath", {
                id: "640a9f__a",
                children: /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M35.75 27.93h48.07c8.84 0 16 7.16 16 16v88.13H73.83v-20.58c0-4.42-3.58-8-8-8H53.74c-4.42 0-8 3.58-8 8v20.58H19.75V43.93c0-8.84 7.16-16 16-16m12.79 56.56c2.21 0 4-1.79 4-4v-5.76c0-2.21-1.79-4-4-4h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.21 1.79 4 4 4zm0-28.06c2.21 0 4-1.79 4-4v-5.76c0-2.21-1.79-4-4-4h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.21 1.79 4 4 4zM76.8 84.49c2.21 0 4-1.79 4-4v-5.76c0-2.21-1.79-4-4-4h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.21 1.79 4 4 4zm0-28.06c2.21 0 4-1.79 4-4v-5.76c0-2.21-1.79-4-4-4h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.21 1.79 4 4 4zm47.44-6.87c8.84 0 16 7.16 16 16v66.5h-32.41v-82.5zm-1.33 62.16c2.21 0 4-1.79 4-4v-5.76c0-2.21-1.79-4-4-4h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.21 1.79 4 4 4zm0-28.06c2.21 0 4-1.79 4-4V73.9c0-2.21-1.79-4-4-4h-5.76c-2.21 0-4 1.79-4 4v5.76c0 2.21 1.79 4 4 4z"
                })
            }),
            /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("g", {
                clipPath: "url(#640a9f__a)",
                children: /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#640a9f__b)",
                    children: [
                        /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("rect", {
                            width: 142,
                            height: 134,
                            x: 7,
                            y: 18,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("ellipse", {
                            cx: 133.577,
                            cy: 106.095,
                            fill: "url(#640a9f__c)",
                            rx: 205.022,
                            ry: 193.472
                        }),
                        /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("path", {
                            fill: "url(#640a9f__d)",
                            d: "M-5.78 59.97c-2.28-46.316 35.662-85.606 84.743-87.757 49.081-2.15 90.717 33.653 92.996 79.969s-35.661 85.606-84.743 87.757S-3.5 106.286-5.78 59.97"
                        }),
                        /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("path", {
                            fill: "url(#640a9f__e)",
                            d: "M42.874-74.725c-11.072-6.465-25.1-6.505-36.214-.102l-111.153 64.043c-10.886 6.272-10.904 21.077-.033 27.373L5.893 80.541c11.096 6.426 25.125 6.418 36.212-.023l110.19-64.012c10.814-6.282 10.834-21.004.033-27.31z"
                        }),
                        /*#__PURE__*/ (0, $8qJfg$reactjsxruntime.jsx)("path", {
                            fill: "url(#640a9f__f)",
                            d: "M-63.79 82.093c22.91 0 43.663 8.75 58.689 22.865 30.05 28.281 54.963 28.305 85.014.073 15.025-14.189 35.803-22.963 58.739-22.963 45.846.025 82.985 34.976 82.985 78.121 0 43.146-37.139 78.073-82.985 78.073-22.987 0-43.766-8.798-58.79-22.987-30.025-28.183-54.913-28.159-84.963.121-15.026 14.14-35.778 22.866-58.688 22.866-45.821 0-82.96-34.951-82.96-78.073s37.139-78.096 82.96-78.096"
                        })
                    ]
                })
            })
        ]
    });
const $58325690b231931d$var$ForwardRef = /*#__PURE__*/ (0, $8qJfg$react.forwardRef)($58325690b231931d$var$SvgComponent);
var $58325690b231931d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($58325690b231931d$var$ForwardRef);
