// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_piggyBank_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $k6tOg$reactjsxruntime = require("react/jsx-runtime");
var $k6tOg$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $484828917e3f8119$export$2e2bcd8739ae039);




const $484828917e3f8119$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsxs)("linearGradient", {
                        id: "68f5d3__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsxs)("linearGradient", {
                        id: "68f5d3__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsxs)("linearGradient", {
                        id: "68f5d3__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsxs)("radialGradient", {
                        id: "68f5d3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("clipPath", {
                        id: "68f5d3__b",
                        children: /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("clipPath", {
                id: "68f5d3__a",
                children: /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M65.762 70.377c9.658-5.116 16.08-14.184 16.08-24.519q0-1.518-.183-2.997c1.303-.13 2.844-.64 4.207-1.78 1.823-1.523 3.119-3.995 3.283-7.608.05-1.103-.804-2.038-1.907-2.088-1.104-.05-2.039.804-2.089 1.907-.122 2.693-1.033 4.036-1.852 4.72-.876.733-1.888.914-2.46.881-3.693-12.61-17.187-21.96-33.268-21.96-6.543 0-12.657 1.548-17.857 4.233-3.089-3.313-7.926-4.273-11.676-4.23-1.392.017-2.115 1.527-1.418 2.731l4.507 7.784.032-.024c-2.358 2.405-4.264 5.137-5.607 8.104v-.017l-6.716 3.913c-1.23.716-1.986 2.032-1.986 3.456v5.952c0 1.424.757 2.74 1.987 3.457l6.715 3.908v-.014c2.667 5.89 7.55 10.854 13.788 14.169q-.025-.004-.042-.004l2.48 6.752c.433 1.18 1.557 1.965 2.815 1.965h3.434c1.257 0 2.381-.783 2.815-1.963l.997-2.707-.089-.03c1.892.273 3.837.415 5.82.415 1.941 0 3.844-.136 5.697-.398l-.04.013.998 2.707c.434 1.18 1.558 1.963 2.815 1.963h3.434c1.258 0 2.382-.784 2.816-1.965zm-13.231-43.23c-3.156-.702-5.783-.732-9.12.002-1.08.237-2.146-.446-2.383-1.524-.237-1.08.445-2.146 1.524-2.383 3.912-.86 7.129-.827 10.848.001 1.078.24 1.758 1.309 1.518 2.387s-1.31 1.758-2.387 1.517m-22.81 8.236c0 1.657-1.343 3-3 3s-3-1.343-3-3 1.343-3 3-3 3 1.343 3 3"
                })
            }),
            /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("g", {
                clipPath: "url(#68f5d3__a)",
                children: /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#68f5d3__b)",
                    children: [
                        /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#68f5d3__c)"
                        }),
                        /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("path", {
                            fill: "url(#68f5d3__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("path", {
                            fill: "url(#68f5d3__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $k6tOg$reactjsxruntime.jsx)("path", {
                            fill: "url(#68f5d3__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $484828917e3f8119$var$ForwardRef = /*#__PURE__*/ (0, $k6tOg$react.forwardRef)($484828917e3f8119$var$SvgComponent);
var $484828917e3f8119$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($484828917e3f8119$var$ForwardRef);
