// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_cut_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $j0DgR$jsxs, jsx as $j0DgR$jsx} from "react/jsx-runtime";
import {forwardRef as $j0DgR$forwardRef} from "react";





const $27e83759ae389909$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $j0DgR$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $j0DgR$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $j0DgR$jsxs)("linearGradient", {
                        id: "43eb96__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $j0DgR$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $j0DgR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $j0DgR$jsxs)("linearGradient", {
                        id: "43eb96__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $j0DgR$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $j0DgR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $j0DgR$jsxs)("linearGradient", {
                        id: "43eb96__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $j0DgR$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $j0DgR$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $j0DgR$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $j0DgR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $j0DgR$jsxs)("radialGradient", {
                        id: "43eb96__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $j0DgR$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $j0DgR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $j0DgR$jsx)("clipPath", {
                        id: "43eb96__b",
                        children: /*#__PURE__*/ (0, $j0DgR$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $j0DgR$jsx)("clipPath", {
                id: "43eb96__a",
                children: /*#__PURE__*/ (0, $j0DgR$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M59.312 14.847c1.045.357 1.602 1.495 1.245 2.54l-8.849 25.846 26.64-9.072c1.046-.356 2.182.203 2.538 1.249.356 1.045-.203 2.181-1.249 2.537l-28.776 9.8q-.017.12-.048.238c-.187.712-.353 1.43-.522 2.163-.505 2.189-1.04 4.499-2.257 7.127-.328.708-.699 1.427-1.092 2.137q.507-.047 1.025-.047c6.047 0 10.948 4.901 10.948 10.948 0 6.046-4.901 10.948-10.948 10.948-6.046 0-10.948-4.902-10.948-10.948 0-2.38.76-4.584 2.05-6.38q.105-.177.25-.335.227-.29.471-.565c1.084-1.381 3.273-4.543 4.615-7.44 1.03-2.222 1.438-3.988 1.916-6.056l.09-.385c-1.069.384-2.215.832-3.66 1.427-1.687.695-3.647 1.792-5.34 2.847-1.71 1.066-3.007 2-3.386 2.321q-.121.103-.253.184c-1.987 2.025-4.755 3.281-7.816 3.281-6.047 0-10.948-4.901-10.948-10.948 0-6.046 4.901-10.948 10.948-10.948 6.046 0 10.948 4.902 10.948 10.948q0 .413-.03.817c1.402-.813 2.927-1.613 4.353-2.2 2.361-.973 3.988-1.58 5.738-2.142l9.807-28.648c.358-1.045 1.495-1.602 2.54-1.244M42.408 66.145c-.872 1.16-1.389 2.604-1.389 4.168 0 3.837 3.11 6.948 6.948 6.948 3.837 0 6.948-3.11 6.948-6.948 0-3.837-3.11-6.948-6.948-6.948-2.035 0-3.866.875-5.137 2.27q-.237.296-.422.51M25.956 41.316c-3.837 0-6.948 3.11-6.948 6.948 0 3.837 3.11 6.948 6.948 6.948 3.837 0 6.948-3.11 6.948-6.948 0-3.837-3.11-6.948-6.948-6.948"
                })
            }),
            /*#__PURE__*/ (0, $j0DgR$jsx)("g", {
                clipPath: "url(#43eb96__a)",
                children: /*#__PURE__*/ (0, $j0DgR$jsxs)("g", {
                    clipPath: "url(#43eb96__b)",
                    children: [
                        /*#__PURE__*/ (0, $j0DgR$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $j0DgR$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#43eb96__c)"
                        }),
                        /*#__PURE__*/ (0, $j0DgR$jsx)("path", {
                            fill: "url(#43eb96__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $j0DgR$jsx)("path", {
                            fill: "url(#43eb96__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $j0DgR$jsx)("path", {
                            fill: "url(#43eb96__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $27e83759ae389909$var$ForwardRef = /*#__PURE__*/ (0, $j0DgR$forwardRef)($27e83759ae389909$var$SvgComponent);
var $27e83759ae389909$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($27e83759ae389909$var$ForwardRef);


export {$27e83759ae389909$export$2e2bcd8739ae039 as default};
