// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_globeGrid_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lvphz$jsxs, jsx as $lvphz$jsx} from "react/jsx-runtime";
import {forwardRef as $lvphz$forwardRef} from "react";





const $936814a283a522d7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lvphz$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lvphz$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lvphz$jsxs)("linearGradient", {
                        id: "4ccecb__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvphz$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $lvphz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvphz$jsxs)("linearGradient", {
                        id: "4ccecb__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvphz$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $lvphz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvphz$jsxs)("linearGradient", {
                        id: "4ccecb__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvphz$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lvphz$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $lvphz$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $lvphz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvphz$jsxs)("radialGradient", {
                        id: "4ccecb__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvphz$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lvphz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvphz$jsx)("clipPath", {
                        id: "4ccecb__b",
                        children: /*#__PURE__*/ (0, $lvphz$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lvphz$jsx)("clipPath", {
                id: "4ccecb__a",
                children: /*#__PURE__*/ (0, $lvphz$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m25.235 7.063.053.003c1.941 2.303 3.435 4.75 4.488 7.28h-4.54zm3.74.695c1.482 2.116 2.644 4.325 3.49 6.588h5.508c-2.15-3.104-5.307-5.458-8.999-6.588m-3.74 9.088h5.432c.595 2 .927 4.038 1 6.084h-6.432zm8.032 0c.537 2.006.837 4.042.902 6.084h6.78c-.134-2.162-.674-4.215-1.543-6.084zm-8.032 8.584h6.392c-.139 2.044-.534 4.083-1.182 6.09h-5.21zm-2.5 6.09v-6.09h-6.39c.138 2.044.533 4.083 1.181 6.09zm-4.261 2.5h4.261v6.762q0 .079.01.155l-.052-.004c-1.77-2.204-3.18-4.526-4.22-6.913m-3.561-2.5c-.588-2.008-.947-4.047-1.073-6.09H7.077c.181 2.172.77 4.228 1.692 6.09zm-4.626 2.5h5.483c.841 2.139 1.949 4.23 3.317 6.241-3.572-1.077-6.648-3.3-8.8-6.24m14.948 0h4.262c-1.04 2.388-2.45 4.71-4.22 6.916l-.051.003q.01-.077.01-.157zm6.966 0c-.843 2.142-1.953 4.237-3.324 6.252 3.587-1.073 6.676-3.301 8.836-6.252zm7.03-2.5c.921-1.862 1.51-3.918 1.692-6.09H34.13c-.126 2.043-.485 4.082-1.073 6.09zM18.989 7.77c-3.676 1.133-6.82 3.482-8.962 6.576h5.479c.845-2.26 2.004-4.464 3.483-6.576m-4.284 9.076H8.75q-.077 0-.152-.009c-.872 1.871-1.413 3.927-1.547 6.093h6.752c.064-2.042.364-4.078.902-6.084m1.6 6.084c.072-2.046.404-4.083 1-6.084h5.43v6.084zm1.89-8.584c1.053-2.53 2.546-4.976 4.486-7.277l.054-.004v7.281z"
                })
            }),
            /*#__PURE__*/ (0, $lvphz$jsx)("g", {
                clipPath: "url(#4ccecb__a)",
                children: /*#__PURE__*/ (0, $lvphz$jsxs)("g", {
                    clipPath: "url(#4ccecb__b)",
                    children: [
                        /*#__PURE__*/ (0, $lvphz$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $lvphz$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#4ccecb__c)"
                        }),
                        /*#__PURE__*/ (0, $lvphz$jsx)("path", {
                            fill: "url(#4ccecb__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $lvphz$jsx)("path", {
                            fill: "url(#4ccecb__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $lvphz$jsx)("path", {
                            fill: "url(#4ccecb__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $936814a283a522d7$var$ForwardRef = /*#__PURE__*/ (0, $lvphz$forwardRef)($936814a283a522d7$var$SvgComponent);
var $936814a283a522d7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($936814a283a522d7$var$ForwardRef);


export {$936814a283a522d7$export$2e2bcd8739ae039 as default};
