// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lightbulbRays_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $9PUXC$reactjsxruntime = require("react/jsx-runtime");
var $9PUXC$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $97c093271528eaee$export$2e2bcd8739ae039);




const $97c093271528eaee$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "728493__d",
                        x1: 107.785,
                        x2: 37.447,
                        y1: 16.683,
                        y2: 113.425,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "728493__e",
                        x1: 61.565,
                        x2: 74.605,
                        y1: -26.004,
                        y2: 62.525,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "728493__f",
                        x1: -107.301,
                        x2: 137.787,
                        y1: 151.51,
                        y2: 100.127,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsxs)("radialGradient", {
                        id: "728493__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 178.298 -189.141 0 129.774 101.839)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("clipPath", {
                        id: "728493__b",
                        children: /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("clipPath", {
                id: "728493__a",
                children: /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M84 29.95c0 2.21-1.79 4-4 4s-4-1.79-4-4v-6.02c0-2.21 1.79-4 4-4s4 1.79 4 4zm-6.2 13.73c-14.85 1.06-26.87 13.17-27.97 28.16-.81 11.14 4.28 21.12 12.45 27.09 2.65 1.94 4.27 5.01 4.27 8.32V114h26.53c.072 0 .144.01.216.021q.073.012.144.019v-6.78c0-3.23 1.48-6.31 4.08-8.19 7.7-5.54 12.72-14.63 12.72-24.91 0-17.62-14.75-31.75-32.45-30.49zm19.37 30.9c-2.21 0-4-1.79-4-4 0-5.01-4.44-9.08-9.9-9.08-2.21 0-4-1.79-4-4s1.79-4 4-4c9.87 0 17.9 7.66 17.9 17.08 0 2.21-1.79 4-4 4M66.56 122v4.8c0 2.81 2.26 5.09 5.04 5.09h16.81c2.78 0 5.04-2.28 5.04-5.09v-4.84q-.07.007-.143.018c-.072.011-.144.022-.216.022zM29.95 69.98h6.02c2.21 0 4 1.79 4 4s-1.79 4-4 4h-6.02c-2.21 0-4-1.79-4-4s1.79-4 4-4m100.09 0h-6.02c-2.21 0-4 1.79-4 4s1.79 4 4 4h6.02c2.21 0 4-1.79 4-4s-1.79-4-4-4m-17.479-34.22-4.25 4.25c-1.56 1.56-1.56 4.09 0 5.66.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17l4.25-4.25c1.56-1.56 1.56-4.09 0-5.66-1.56-1.56-4.09-1.56-5.66 0m-65.112.01 4.25 4.25c1.56 1.57 1.56 4.1 0 5.66-.78.78-1.8 1.17-2.83 1.17s-2.05-.39-2.83-1.17l-4.25-4.25c-1.56-1.57-1.56-4.1 0-5.66 1.57-1.56 4.1-1.56 5.66 0"
                })
            }),
            /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("g", {
                clipPath: "url(#728493__a)",
                children: /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#728493__b)",
                    children: [
                        /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("ellipse", {
                            cx: 129.774,
                            cy: 101.839,
                            fill: "url(#728493__c)",
                            rx: 189.141,
                            ry: 178.298
                        }),
                        /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("path", {
                            fill: "url(#728493__d)",
                            d: "M1.21 59.333C-.893 16.649 34.11-19.56 79.389-21.542s83.69 31.014 85.792 73.698-32.899 78.892-78.178 80.874c-45.28 1.982-83.69-31.013-85.793-73.697"
                        }),
                        /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("path", {
                            fill: "url(#728493__e)",
                            d: "M46.095-64.799c-10.215-5.958-23.156-5.994-33.409-.094L-89.856-5.873c-10.043 5.78-10.06 19.425-.03 25.227L11.977 78.29c10.237 5.922 23.179 5.915 33.407-.02L147.04 19.277c9.976-5.79 9.994-19.358.031-25.17z"
                        }),
                        /*#__PURE__*/ (0, $9PUXC$reactjsxruntime.jsx)("path", {
                            fill: "url(#728493__f)",
                            d: "M-52.305 79.722c21.136 0 40.28 8.063 54.142 21.072 27.723 26.062 50.706 26.085 78.429.067 13.861-13.076 33.03-21.162 54.189-21.162 42.295.023 76.557 32.233 76.557 71.994s-34.262 71.95-76.557 71.95c-21.207 0-40.375-8.108-54.237-21.184-27.699-25.973-50.658-25.951-78.38.112-13.862 13.031-33.007 21.072-54.143 21.072-42.27 0-76.533-32.21-76.533-71.95 0-39.739 34.262-71.971 76.533-71.971"
                        })
                    ]
                })
            })
        ]
    });
const $97c093271528eaee$var$ForwardRef = /*#__PURE__*/ (0, $9PUXC$react.forwardRef)($97c093271528eaee$var$SvgComponent);
var $97c093271528eaee$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($97c093271528eaee$var$ForwardRef);
