// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fileImage_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $d4MFr$jsxs, jsx as $d4MFr$jsx} from "react/jsx-runtime";
import {forwardRef as $d4MFr$forwardRef} from "react";





const $7969f4c7ed93c1aa$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $d4MFr$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $d4MFr$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $d4MFr$jsxs)("linearGradient", {
                        id: "b71164__d",
                        x1: 112.176,
                        x2: 31.997,
                        y1: 6.324,
                        y2: 110.278,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d4MFr$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $d4MFr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d4MFr$jsxs)("linearGradient", {
                        id: "b71164__e",
                        x1: 63.327,
                        x2: 78.794,
                        y1: -41.534,
                        y2: 57.456,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d4MFr$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $d4MFr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d4MFr$jsxs)("linearGradient", {
                        id: "b71164__f",
                        x1: -115.143,
                        x2: 145.101,
                        y1: 157.48,
                        y2: 106.047,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d4MFr$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $d4MFr$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $d4MFr$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $d4MFr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d4MFr$jsxs)("radialGradient", {
                        id: "b71164__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.808 118.603)scale(199.896)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d4MFr$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $d4MFr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d4MFr$jsx)("clipPath", {
                        id: "b71164__b",
                        children: /*#__PURE__*/ (0, $d4MFr$jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 12,
                            y: 10.775,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $d4MFr$jsx)("clipPath", {
                id: "b71164__a",
                children: /*#__PURE__*/ (0, $d4MFr$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m87.515 31.515 32.97 32.97c2.251 2.25 3.515 5.303 3.515 8.486V120c0 6.627-5.373 12-12 12H48c-6.627 0-12-5.373-12-12V40c0-6.627 5.373-12 12-12h31.03c3.182 0 6.234 1.264 8.485 3.515M94.499 68c-4.142 0-7.5 3.358-7.5 7.5s3.358 7.5 7.5 7.5c4.143 0 7.501-3.358 7.501-7.5S98.642 68 94.5 68m-40.494 48h51.989c1.565 0 2.524-1.715 1.704-3.048L96.511 94.769c-.78-1.27-2.626-1.27-3.407 0l-8.816 14.329-13.091-22.199c-.775-1.306-2.666-1.306-3.44 0L52.284 112.98c-.79 1.333.17 3.02 1.72 3.02"
                })
            }),
            /*#__PURE__*/ (0, $d4MFr$jsx)("g", {
                clipPath: "url(#b71164__a)",
                children: /*#__PURE__*/ (0, $d4MFr$jsxs)("g", {
                    clipPath: "url(#b71164__b)",
                    children: [
                        /*#__PURE__*/ (0, $d4MFr$jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 12,
                            y: 10.775,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $d4MFr$jsx)("circle", {
                            cx: 135.411,
                            cy: 101.794,
                            r: 199.896,
                            fill: "url(#b71164__c)"
                        }),
                        /*#__PURE__*/ (0, $d4MFr$jsx)("path", {
                            fill: "url(#b71164__d)",
                            d: "M-.46 54.14C-2.68 6.284 34.31-34.31 82.166-36.533s88.449 34.77 90.671 82.625-34.77 88.449-82.624 90.671S1.762 101.994-.46 54.139"
                        }),
                        /*#__PURE__*/ (0, $d4MFr$jsx)("path", {
                            fill: "url(#b71164__e)",
                            d: "M46.977-85.028c-10.795-6.68-24.473-6.72-35.308-.105L-96.705-18.964c-10.614 6.48-10.631 21.777-.032 28.282L10.92 75.394c10.82 6.639 24.497 6.631 35.307-.024L153.662 9.233c10.544-6.491 10.563-21.702.032-28.218z"
                        }),
                        /*#__PURE__*/ (0, $d4MFr$jsx)("path", {
                            fill: "url(#b71164__f)",
                            d: "M-57.02 76.996c22.338 0 42.571 9.04 57.22 23.625 29.3 29.219 53.59 29.244 82.889.075 14.65-14.66 34.908-23.725 57.271-23.725 44.699.025 80.91 36.137 80.91 80.715s-36.211 80.665-80.91 80.665c-22.413 0-42.672-9.091-57.321-23.75-29.274-29.12-53.54-29.095-82.838.125-14.65 14.609-34.883 23.625-57.22 23.625-44.675 0-80.886-36.112-80.886-80.665s36.211-80.69 80.885-80.69"
                        })
                    ]
                })
            })
        ]
    });
const $7969f4c7ed93c1aa$var$ForwardRef = /*#__PURE__*/ (0, $d4MFr$forwardRef)($7969f4c7ed93c1aa$var$SvgComponent);
var $7969f4c7ed93c1aa$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7969f4c7ed93c1aa$var$ForwardRef);


export {$7969f4c7ed93c1aa$export$2e2bcd8739ae039 as default};
