// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_flagCheckmark_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bOH0T$jsxs, jsx as $bOH0T$jsx} from "react/jsx-runtime";
import {forwardRef as $bOH0T$forwardRef} from "react";





const $8d34ea5e8528db47$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bOH0T$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bOH0T$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bOH0T$jsxs)("linearGradient", {
                        id: "111e67__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bOH0T$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $bOH0T$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bOH0T$jsxs)("linearGradient", {
                        id: "111e67__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bOH0T$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $bOH0T$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bOH0T$jsxs)("linearGradient", {
                        id: "111e67__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bOH0T$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bOH0T$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $bOH0T$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $bOH0T$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bOH0T$jsxs)("radialGradient", {
                        id: "111e67__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bOH0T$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bOH0T$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bOH0T$jsx)("clipPath", {
                        id: "111e67__b",
                        children: /*#__PURE__*/ (0, $bOH0T$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bOH0T$jsx)("clipPath", {
                id: "111e67__a",
                children: /*#__PURE__*/ (0, $bOH0T$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M23.484 16.219c1.105 0 2 .895 2 2v3.824h41.172c3.314 0 6 2.686 6 6v25.958c0 3.314-2.686 6-6 6H25.484v17.78c0 1.105-.895 2-2 2s-2-.895-2-2V18.22c0-1.105.896-2 2-2M60.573 36.52c.816-.744.875-2.009.13-2.825-.743-.817-2.008-.875-2.825-.131l-.062.057-.137.125-.207.188-.015.014-.015.014-.032.029-.066.06-.07.063-.073.067-.038.035-.04.036-.08.073-.085.077-.021.02-.022.02-.022.02-.023.02-.091.084-.024.021-.023.022-.024.021-.024.022-.048.044-.05.045-.1.093-.105.095-.107.097-.223.204-.058.052-.058.053-.119.108-.245.223-1.07.977-.288.262-.295.268-.3.274-.153.14-.154.14-.154.14-.04.036-.038.035-.078.071-.079.072-.078.072-.04.035-.04.036-.039.036-.04.037-.64.583-.08.074-.081.074-.04.036-.041.037-.04.037-.041.037-.163.149-.163.148-.082.075-.081.074-.041.037-.041.037-.082.075-.082.074-.04.038-.041.037-.164.149-.163.149-.041.037-.041.037-.082.075-.081.074-.082.075-.163.148-.081.074-.082.074-.162.148-.04.037-.04.037-.04.037-.041.036-.08.074-.041.037-.04.036-.08.073-.08.073-.04.037-.04.036-.04.037-.04.036-.08.072-.04.036-.039.036-.158.144-.62.566-.077.07-.038.034-.038.035-.037.034-.038.035-.038.034-.037.034-.075.068-.075.068-.037.034-.037.034-.074.067-.146.133-.072.066-.072.066-.283.258-.276.252-.068.061-.034.03-.033.031-.133.121-.033.03-.032.03-.065.06-.065.058-.031.029-.032.029-.25.227-.238.217-.029.027-.029.026-.028.026-.03.026-.112.103-.055.05-.054.05-.107.097-.026.024-.026.024-.026.023-.026.023-.016.015-5.533-5.16c-.808-.754-2.074-.71-2.827.098s-.71 2.073.099 2.827l6.882 6.418c.762.71 1.941.717 2.711.015l-1.347-1.478 1.347 1.478.002-.001v-.001l.008-.007.03-.027.116-.105.04-.037.044-.04.102-.094.028-.025.015-.013.014-.014.03-.027.016-.014.016-.014.134-.123.073-.066.077-.07.164-.15.043-.04.022-.02.022-.02.022-.02.023-.02.046-.042.094-.086.097-.089.42-.383.029-.025.028-.026.056-.051.116-.105.058-.054.06-.054.03-.027.03-.028.06-.055.062-.056.062-.056.03-.029.032-.028.032-.029.031-.029.032-.029.032-.029.064-.059.033-.03.032-.029.033-.03.033-.03.132-.12.135-.123.137-.125.139-.127.14-.128.072-.065.036-.032.035-.033.072-.066.037-.033.036-.033.146-.133.037-.033.037-.034.074-.067.037-.034.037-.034.075-.069.075-.068.076-.069.038-.034.038-.035.076-.07.62-.565.04-.036.04-.036.039-.036.04-.036.079-.072.04-.037.04-.036.32-.291.646-.59.327-.298.04-.037.041-.037.041-.037.041-.038.164-.149.081-.074.082-.075.041-.037.04-.037.083-.075.04-.037.041-.037.041-.038.041-.037.041-.037.04-.037.082-.075.041-.037.04-.037.042-.037.04-.037.041-.037.04-.037.082-.074.081-.074.04-.037.041-.037.162-.148.04-.036.04-.037.04-.037.041-.036.04-.037.04-.037.08-.073.04-.036.04-.037.08-.072.08-.073.08-.072.039-.036.04-.036.039-.036.04-.036.078-.072.078-.071.04-.036.038-.035.04-.036.038-.035.078-.071.038-.035.039-.036.154-.14.038-.034.038-.035.038-.035.038-.035.038-.034.038-.034.037-.035.038-.034.075-.068.037-.034.038-.034.294-.269.073-.066.036-.033.036-.033.036-.032.035-.033.072-.065.28-.256.069-.062.034-.032.034-.03.034-.032.034-.03.034-.031.034-.03.518-.473.914-.833.199-.181.186-.17.088-.08.084-.077.081-.074.077-.07.074-.068.035-.032.018-.016.017-.015.033-.03.017-.015.016-.015.062-.057.111-.101.095-.087z"
                })
            }),
            /*#__PURE__*/ (0, $bOH0T$jsx)("g", {
                clipPath: "url(#111e67__a)",
                children: /*#__PURE__*/ (0, $bOH0T$jsxs)("g", {
                    clipPath: "url(#111e67__b)",
                    children: [
                        /*#__PURE__*/ (0, $bOH0T$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $bOH0T$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#111e67__c)"
                        }),
                        /*#__PURE__*/ (0, $bOH0T$jsx)("path", {
                            fill: "url(#111e67__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $bOH0T$jsx)("path", {
                            fill: "url(#111e67__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $bOH0T$jsx)("path", {
                            fill: "url(#111e67__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $8d34ea5e8528db47$var$ForwardRef = /*#__PURE__*/ (0, $bOH0T$forwardRef)($8d34ea5e8528db47$var$SvgComponent);
var $8d34ea5e8528db47$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8d34ea5e8528db47$var$ForwardRef);


export {$8d34ea5e8528db47$export$2e2bcd8739ae039 as default};
