// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_imageStack_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $1zr5m$reactjsxruntime = require("react/jsx-runtime");
var $1zr5m$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b84d531d87acab37$export$2e2bcd8739ae039);




const $b84d531d87acab37$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsxs)("linearGradient", {
                        id: "895fba__d",
                        x1: 34.086,
                        x2: 12.888,
                        y1: 4.914,
                        y2: 34.07,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsxs)("linearGradient", {
                        id: "895fba__e",
                        x1: 20.157,
                        x2: 24.087,
                        y1: -7.95,
                        y2: 18.73,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsxs)("linearGradient", {
                        id: "895fba__f",
                        x1: -30.735,
                        x2: 43.127,
                        y1: 45.548,
                        y2: 30.063,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsxs)("radialGradient", {
                        id: "895fba__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.7342 -57.0018 0 40.713 30.578)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("clipPath", {
                        id: "895fba__b",
                        children: /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("rect", {
                            width: 39.48,
                            height: 37.217,
                            x: 5.52,
                            y: 6.111,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("clipPath", {
                id: "895fba__a",
                children: /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M37.628 14.54v14.7c0 2.652-2.148 4.8-4.8 4.8h-22.2c-2.652 0-4.8-2.148-4.8-4.8v-14.7c0-2.652 2.148-4.8 4.8-4.8h22.2c2.652 0 4.8 2.148 4.8 4.8m-4.212 2.064c0-1.446-1.173-2.619-2.619-2.619s-2.619 1.173-2.619 2.62c0 1.445 1.173 2.618 2.62 2.618 1.445 0 2.618-1.173 2.618-2.619M8.693 28.865c-.768 1.197.09 2.772 1.515 2.772v-.003h22.854c1.425 0 2.283-1.575 1.515-2.772l-3.534-5.505c-.708-1.104-2.322-1.104-3.03 0l-4.008 6.246-6.378-9.936c-.708-1.104-2.322-1.104-3.03 0zm32.277-8.919c0-.663.537-1.2 1.2-1.2h.003c.663 0 1.2.537 1.2 1.2v12.315c0 3.97-3.23 7.2-7.2 7.2l-19.782-.015c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.2-1.2l19.78.015c2.645 0 4.8-2.154 4.8-4.8z"
                })
            }),
            /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("g", {
                clipPath: "url(#895fba__a)",
                children: /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#895fba__b)",
                    children: [
                        /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("rect", {
                            width: 39.48,
                            height: 37.217,
                            x: 5.52,
                            y: 6.111,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("ellipse", {
                            cx: 40.713,
                            cy: 30.578,
                            fill: "url(#895fba__c)",
                            rx: 57.002,
                            ry: 53.734
                        }),
                        /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("path", {
                            fill: "url(#895fba__d)",
                            d: "M1.968 17.768C1.334 4.904 11.883-6.008 25.528-6.606c13.646-.597 25.222 9.347 25.856 22.21s-9.915 23.777-23.561 24.374-25.222-9.347-25.855-22.21"
                        }),
                        /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("path", {
                            fill: "url(#895fba__e)",
                            d: "M15.495-19.642c-3.078-1.795-6.979-1.806-10.069-.028L-25.477-1.883c-3.027 1.742-3.032 5.854-.01 7.603l30.7 17.761c3.085 1.785 6.985 1.783 10.068-.006L45.917 5.697c3.006-1.745 3.012-5.834.01-7.586z"
                        }),
                        /*#__PURE__*/ (0, $1zr5m$reactjsxruntime.jsx)("path", {
                            fill: "url(#895fba__f)",
                            d: "M-14.16 23.913c6.369 0 12.138 2.43 16.316 6.35 8.355 7.855 15.281 7.862 23.636.02 4.177-3.94 9.954-6.377 16.331-6.377 12.747.007 23.072 9.714 23.072 21.697S54.87 67.287 42.123 67.287c-6.39 0-12.168-2.444-16.345-6.384-8.348-7.828-15.267-7.821-23.622.033-4.178 3.928-9.947 6.351-16.317 6.351-12.74 0-23.065-9.707-23.065-21.684s10.326-21.69 23.065-21.69"
                        })
                    ]
                })
            })
        ]
    });
const $b84d531d87acab37$var$ForwardRef = /*#__PURE__*/ (0, $1zr5m$react.forwardRef)($b84d531d87acab37$var$SvgComponent);
var $b84d531d87acab37$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b84d531d87acab37$var$ForwardRef);
