// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_graphBarChart_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6tawx$jsxs, jsx as $6tawx$jsx} from "react/jsx-runtime";
import {forwardRef as $6tawx$forwardRef} from "react";





const $6027da17b155fcaf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6tawx$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6tawx$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6tawx$jsxs)("linearGradient", {
                        id: "e0eedd__d",
                        x1: 32.537,
                        x2: 9.298,
                        y1: 3.187,
                        y2: 35.15,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6tawx$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6tawx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6tawx$jsxs)("linearGradient", {
                        id: "e0eedd__e",
                        x1: 17.266,
                        x2: 21.574,
                        y1: -10.915,
                        y2: 18.334,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6tawx$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6tawx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6tawx$jsxs)("linearGradient", {
                        id: "e0eedd__f",
                        x1: -38.522,
                        x2: 42.452,
                        y1: 47.733,
                        y2: 30.757,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6tawx$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6tawx$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6tawx$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6tawx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6tawx$jsxs)("radialGradient", {
                        id: "e0eedd__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 58.9078 -62.49 0 39.803 31.322)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6tawx$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6tawx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6tawx$jsx)("clipPath", {
                        id: "e0eedd__b",
                        children: /*#__PURE__*/ (0, $6tawx$jsx)("rect", {
                            width: 43.281,
                            height: 40.8,
                            x: 1.223,
                            y: 4.5,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6tawx$jsx)("clipPath", {
                id: "e0eedd__a",
                children: /*#__PURE__*/ (0, $6tawx$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M7.263 40.8H40.8c.663 0 1.2.537 1.2 1.2s-.537 1.2-1.2 1.2H6.063c-.663 0-1.2-.537-1.2-1.2V7.2c0-.663.537-1.2 1.2-1.2s1.2.537 1.2 1.2zM13.2 21.315h1.2c1.326 0 2.4 1.075 2.4 2.4V34.8c0 1.326-1.075 2.4-2.4 2.4h-1.2c-1.325 0-2.4-1.075-2.4-2.4V23.715c0-1.325 1.074-2.4 2.4-2.4M36 28.8h-1.2c-1.325 0-2.4 1.075-2.4 2.4v3.6c0 1.326 1.075 2.4 2.4 2.4H36c1.325 0 2.4-1.075 2.4-2.4v-3.6c0-1.325-1.075-2.4-2.4-2.4M24 12h1.2c1.326 0 2.4 1.075 2.4 2.4v20.4c0 1.326-1.075 2.4-2.4 2.4H24c-1.326 0-2.4-1.075-2.4-2.4V14.4c0-1.325 1.074-2.4 2.4-2.4"
                })
            }),
            /*#__PURE__*/ (0, $6tawx$jsx)("g", {
                clipPath: "url(#e0eedd__a)",
                children: /*#__PURE__*/ (0, $6tawx$jsxs)("g", {
                    clipPath: "url(#e0eedd__b)",
                    children: [
                        /*#__PURE__*/ (0, $6tawx$jsx)("rect", {
                            width: 43.281,
                            height: 40.8,
                            x: 1.223,
                            y: 4.5,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6tawx$jsx)("ellipse", {
                            cx: 39.803,
                            cy: 31.322,
                            fill: "url(#e0eedd__c)",
                            rx: 62.49,
                            ry: 58.908
                        }),
                        /*#__PURE__*/ (0, $6tawx$jsx)("path", {
                            fill: "url(#e0eedd__d)",
                            d: "M-2.674 17.278C-3.37 3.176 8.196-8.787 23.156-9.442S50.805.805 51.5 14.907s-10.87 26.065-25.83 26.72S-1.98 31.381-2.673 17.278"
                        }),
                        /*#__PURE__*/ (0, $6tawx$jsx)("path", {
                            fill: "url(#e0eedd__e)",
                            d: "M12.155-23.732c-3.375-1.969-7.65-1.98-11.038-.031l-33.88 19.5c-3.317 1.91-3.323 6.417-.01 8.334L.884 23.543c3.382 1.957 7.658 1.954 11.037-.007l33.586-19.49c3.296-1.913 3.302-6.396.01-8.316z"
                        }),
                        /*#__PURE__*/ (0, $6tawx$jsx)("path", {
                            fill: "url(#e0eedd__f)",
                            d: "M-20.352 24.015c6.983 0 13.308 2.664 17.888 6.962 9.159 8.611 16.752 8.618 25.912.022 4.58-4.32 10.912-6.991 17.903-6.991 13.974.007 25.293 10.65 25.293 23.786s-11.32 23.771-25.293 23.771c-7.006 0-13.34-2.679-17.92-6.999-9.15-8.58-16.736-8.574-25.895.037-4.58 4.306-10.905 6.962-17.888 6.962-13.966 0-25.286-10.642-25.286-23.771s11.32-23.779 25.286-23.779"
                        })
                    ]
                })
            })
        ]
    });
const $6027da17b155fcaf$var$ForwardRef = /*#__PURE__*/ (0, $6tawx$forwardRef)($6027da17b155fcaf$var$SvgComponent);
var $6027da17b155fcaf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6027da17b155fcaf$var$ForwardRef);


export {$6027da17b155fcaf$export$2e2bcd8739ae039 as default};
