// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_stampClone_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $lWLoE$reactjsxruntime = require("react/jsx-runtime");
var $lWLoE$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6ab8b43ab74d63ad$export$2e2bcd8739ae039);




const $6ab8b43ab74d63ad$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e006f2__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e006f2__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e006f2__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsxs)("radialGradient", {
                        id: "e006f2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("clipPath", {
                        id: "e006f2__b",
                        children: /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("clipPath", {
                id: "e006f2__a",
                children: /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M26.544 19.074c2.009-.95 3.399-2.997 3.399-5.367 0-3.278-2.658-5.936-5.936-5.936s-5.935 2.658-5.935 5.936c0 2.37 1.39 4.415 3.398 5.367l-1.28 6.936h-6.592c-2.21 0-4 1.79-4 4v1.948c0 .553.447 1 1 1h26.807c.552 0 1-.447 1-1V30.01c0-2.21-1.791-4-4-4h-6.582zM12.102 37.728c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h23.8c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25z"
                })
            }),
            /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("g", {
                clipPath: "url(#e006f2__a)",
                children: /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#e006f2__b)",
                    children: [
                        /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#e006f2__c)"
                        }),
                        /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("path", {
                            fill: "url(#e006f2__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("path", {
                            fill: "url(#e006f2__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $lWLoE$reactjsxruntime.jsx)("path", {
                            fill: "url(#e006f2__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $6ab8b43ab74d63ad$var$ForwardRef = /*#__PURE__*/ (0, $lWLoE$react.forwardRef)($6ab8b43ab74d63ad$var$SvgComponent);
var $6ab8b43ab74d63ad$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6ab8b43ab74d63ad$var$ForwardRef);
