// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_hand_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $iYlUQ$jsxs, jsx as $iYlUQ$jsx} from "react/jsx-runtime";
import {forwardRef as $iYlUQ$forwardRef} from "react";





const $047708f6c33296f4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iYlUQ$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iYlUQ$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iYlUQ$jsxs)("linearGradient", {
                        id: "8d8cd1__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iYlUQ$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $iYlUQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iYlUQ$jsxs)("linearGradient", {
                        id: "8d8cd1__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iYlUQ$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $iYlUQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iYlUQ$jsxs)("linearGradient", {
                        id: "8d8cd1__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iYlUQ$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iYlUQ$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $iYlUQ$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $iYlUQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iYlUQ$jsxs)("radialGradient", {
                        id: "8d8cd1__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iYlUQ$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iYlUQ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iYlUQ$jsx)("clipPath", {
                        id: "8d8cd1__b",
                        children: /*#__PURE__*/ (0, $iYlUQ$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iYlUQ$jsx)("clipPath", {
                id: "8d8cd1__a",
                children: /*#__PURE__*/ (0, $iYlUQ$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M30.672 84.479v-.001q-.02-.04-.043-.078c-2.246-3.897-1.657-8.104 1.439-10.43 3.214-2.415 7.762-1.794 10.216 1.397l9.953 12.941c.522.68 1.418.95 2.229.675.81-.276 1.356-1.038 1.356-1.894V43.88c0-4.39 3.54-7.934 7.892-7.934 4.351 0 7.892 3.544 7.892 7.933V71.67c0 1.105.895 2 2 2h.02c1.104 0 2-.895 2-2V34.152c0-4.39 3.54-7.933 7.892-7.933 4.35 0 7.891 3.543 7.891 7.933v37.522c0 1.105.896 2 2 2h.02c1.105 0 2-.895 2-2V38.856c0-4.39 3.541-7.933 7.892-7.933 4.357 0 7.897 3.544 7.897 7.933v35.291c0 1.062.829 1.938 1.889 1.997 1.059.06 1.981-.72 2.099-1.774l.945-8.437.945-8.436.003-.028c.461-4.71 4.646-7.063 8.518-6.302 3.067.602 5.521 3.35 5.056 7.881l-4.199 29.36q-.044.133-.07.275c-3.376 18.755-8.921 29.945-15.947 36.47-6.971 6.475-15.698 8.628-26.125 8.628-6.757 0-13.481-2.907-19.866-7.496-6.373-4.581-12.26-10.732-17.294-16.953-5.027-6.212-9.158-12.435-12.034-17.11-1.437-2.335-2.558-4.279-3.317-5.636-.38-.678-.67-1.21-.863-1.57q-.145-.27-.217-.407l-.053-.101-.013-.024z"
                })
            }),
            /*#__PURE__*/ (0, $iYlUQ$jsx)("g", {
                clipPath: "url(#8d8cd1__a)",
                children: /*#__PURE__*/ (0, $iYlUQ$jsxs)("g", {
                    clipPath: "url(#8d8cd1__b)",
                    children: [
                        /*#__PURE__*/ (0, $iYlUQ$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $iYlUQ$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#8d8cd1__c)"
                        }),
                        /*#__PURE__*/ (0, $iYlUQ$jsx)("path", {
                            fill: "url(#8d8cd1__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $iYlUQ$jsx)("path", {
                            fill: "url(#8d8cd1__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $iYlUQ$jsx)("path", {
                            fill: "url(#8d8cd1__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $047708f6c33296f4$var$ForwardRef = /*#__PURE__*/ (0, $iYlUQ$forwardRef)($047708f6c33296f4$var$SvgComponent);
var $047708f6c33296f4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($047708f6c33296f4$var$ForwardRef);


export {$047708f6c33296f4$export$2e2bcd8739ae039 as default};
