// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_update_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2m7eD$jsxs, jsx as $2m7eD$jsx} from "react/jsx-runtime";
import {forwardRef as $2m7eD$forwardRef} from "react";





const $169659c71a2b4781$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2m7eD$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2m7eD$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2m7eD$jsxs)("linearGradient", {
                        id: "1a58eb__d",
                        x1: 67.729,
                        x2: 22.071,
                        y1: 2.223,
                        y2: 65.019,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2m7eD$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2m7eD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2m7eD$jsxs)("linearGradient", {
                        id: "1a58eb__e",
                        x1: 37.729,
                        x2: 46.193,
                        y1: -25.484,
                        y2: 31.981,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2m7eD$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2m7eD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2m7eD$jsxs)("linearGradient", {
                        id: "1a58eb__f",
                        x1: -71.888,
                        x2: 87.203,
                        y1: 89.74,
                        y2: 56.387,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2m7eD$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2m7eD$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2m7eD$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2m7eD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2m7eD$jsxs)("radialGradient", {
                        id: "1a58eb__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 115.736 -122.774 0 82.001 57.5)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2m7eD$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2m7eD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2m7eD$jsx)("clipPath", {
                        id: "1a58eb__b",
                        children: /*#__PURE__*/ (0, $2m7eD$jsx)("rect", {
                            width: 85.035,
                            height: 80.16,
                            x: 6.203,
                            y: 4.801,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2m7eD$jsxs)("clipPath", {
                id: "1a58eb__a",
                children: [
                    /*#__PURE__*/ (0, $2m7eD$jsx)("path", {
                        fill: "#fff",
                        fillRule: "evenodd",
                        d: "M84.074 25.14c.6-1.182 2.046-1.65 3.228-1.05h.005c1.183.6 1.65 2.046 1.05 3.228L81.77 40.26c-.425.834-1.266 1.314-2.142 1.314-.341 0-.683-.078-1.013-.228l-12.702-5.922c-1.207-.564-1.729-1.992-1.165-3.192s1.993-1.722 3.192-1.158l8.299 3.87c-5.094-10.83-16.194-17.976-28.266-17.976-13.782 0-26.1 9.228-29.958 22.44-.306 1.05-1.267 1.728-2.304 1.728q-.336.002-.673-.096c-1.272-.372-2.004-1.704-1.632-2.976 4.453-15.246 18.667-25.896 34.567-25.896 13.758 0 26.417 8.04 32.387 20.274zm-6.192 31.794c.372-1.272 1.704-2.004 2.976-1.632v.006c1.272.372 2.004 1.704 1.632 2.976C78.038 73.53 63.824 84.18 47.924 84.18c-13.692 0-26.298-7.968-32.304-20.106l-3.69 7.254c-.426.834-1.266 1.314-2.142 1.314-.366 0-.738-.084-1.086-.264-1.182-.6-1.65-2.046-1.05-3.228l6.588-12.948c.588-1.152 1.986-1.632 3.156-1.086l12.702 5.922c1.206.564 1.722 1.992 1.164 3.192-.564 1.2-1.992 1.716-3.192 1.158l-8.37-3.906c5.106 10.788 16.182 17.892 28.224 17.892 13.782 0 26.1-9.228 29.958-22.44"
                    }),
                    /*#__PURE__*/ (0, $2m7eD$jsx)("path", {
                        fill: "#fff",
                        d: "m87.302 24.09-.272.535.128.065h.144zm-3.229 1.05.535.272zm3.235-1.05.271-.535-.128-.065h-.143zm1.05 3.228.534.272zM81.769 40.26l.535.273zm-3.156 1.086-.253.544.005.002zm-12.701-5.922-.255.544h.001zm-1.165-3.192-.543-.255zm3.192-1.158-.255.543h.002zm8.299 3.87-.254.544 1.502.7-.706-1.5zm-58.224 4.464-.576-.168zm-2.976 1.632.17-.575h-.002zm-1.633-2.976.576.168zm66.955-5.622-.54.263.528 1.082.546-1.073zm.498 22.86h.6v-.45l-.432-.126zm-2.976 1.632-.576-.168zm2.976-1.626h-.6v.45l.431.126zm1.632 2.976-.576-.168zm-66.87 5.79.538-.266-.53-1.072-.543 1.066zm-3.69 7.254.534.273V71.6zm-3.228 1.05.276-.533-.005-.002zm-1.05-3.228-.535-.272zm6.588-12.948-.534-.273zm3.156-1.086-.254.544zm12.702 5.922.254-.543zm1.164 3.192.543.255v-.002zm-3.192 1.158-.254.544zm-8.37-3.906.254-.544-1.508-.704.712 1.505zm67.873-37.927c-1.476-.75-3.284-.166-4.034 1.313l1.07.544c.449-.885 1.533-1.238 2.42-.787zm-.265-.065H87.3v1.2h.007zm1.584 4.1c.75-1.476.166-3.284-1.313-4.035l-.543 1.07c.885.45 1.237 1.533.787 2.421zm-6.588 12.942 6.588-12.942-1.07-.544-6.587 12.942zm-2.677 1.642c1.095 0 2.145-.6 2.677-1.641l-1.069-.546c-.32.627-.95.987-1.608.987zm-1.262-.282c.402.183.828.282 1.263.282v-1.2q-.377 0-.766-.174zm-12.707-5.924L78.36 41.89l.507-1.088-12.702-5.922zm-1.454-3.991c-.706 1.503-.05 3.287 1.453 3.99l.509-1.087c-.909-.424-1.297-1.496-.876-2.393zm3.99-1.446c-1.502-.706-3.286-.051-3.99 1.446l1.087.51c.424-.903 1.496-1.292 2.393-.87zm8.297 3.87-8.298-3.87-.507 1.087 8.298 3.87zm-28.52-16.833c11.838 0 22.728 7.01 27.724 17.631l1.085-.51C71.588 23.65 60.278 16.368 47.971 16.368zM18.59 39.576c3.784-12.955 15.867-22.008 29.383-22.008v-1.2c-14.049 0-26.602 9.404-30.534 22.872zm-2.88 2.16c1.297 0 2.498-.846 2.88-2.16l-1.151-.336c-.23.786-.949 1.296-1.728 1.296zm-.842-.12c.281.083.567.12.842.12v-1.2c-.168 0-.338-.023-.5-.071zm-2.037-3.72c-.466 1.59.45 3.255 2.04 3.72l.336-1.152c-.955-.28-1.503-1.279-1.225-2.232zm35.142-26.328c-16.167 0-30.616 10.826-35.142 26.328l1.151.336c4.378-14.99 18.357-25.464 33.99-25.464zm32.927 20.61c-6.07-12.437-18.938-20.61-32.927-20.61v1.2c13.526 0 25.978 7.907 31.848 19.937zm2.64-7.31-3.714 7.302 1.07.544 3.713-7.302zm-2.513 29.858c-1.59-.465-3.255.45-3.72 2.04l1.152.336c.28-.955 1.279-1.503 2.231-1.224zm.432.582v-.006h-1.2v.006zm1.608 3.144c.465-1.59-.45-3.255-2.04-3.72l-.337 1.152c.956.28 1.504 1.279 1.225 2.232zM47.924 84.78c16.166 0 30.615-10.826 35.142-26.328l-1.152-.336c-4.377 14.99-18.356 25.464-33.99 25.464zM15.082 64.34c6.106 12.34 18.919 20.44 32.842 20.44v-1.2c-13.461 0-25.86-7.836-31.766-19.772zm-2.617 7.26 3.69-7.254-1.07-.544-3.69 7.254zm-2.677 1.642c1.094 0 2.144-.6 2.676-1.641l-1.068-.546c-.32.627-.95.987-1.608.987zm-1.362-.331c.437.226.904.331 1.362.331v-1.2c-.274 0-.55-.063-.81-.197zm-1.31-4.033c-.748 1.476-.165 3.284 1.314 4.035l.543-1.07c-.884-.45-1.237-1.533-.786-2.422zm6.59-12.948L7.116 68.878l1.07.544 6.588-12.948zm3.944-1.358c-1.463-.682-3.21-.083-3.944 1.357l1.068.546c.441-.865 1.491-1.225 2.368-.815zm12.701 5.922L17.65 54.572l-.507 1.088 12.702 5.922zm1.455 3.989c.697-1.5.052-3.284-1.454-3.989l-.508 1.087c.906.424 1.293 1.496.874 2.396zm-3.99 1.449c1.5.697 3.284.053 3.989-1.447l-1.086-.51c-.423.9-1.495 1.288-2.396.869zm-8.37-3.906 8.37 3.906.508-1.088-8.37-3.906zm28.478 16.748c-11.808 0-22.674-6.969-27.682-17.549l-1.084.514c5.204 10.996 16.49 18.235 28.766 18.235zm29.382-22.008C73.523 69.722 61.44 78.774 47.924 78.774v1.2c14.048 0 26.601-9.404 30.534-22.872z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2m7eD$jsx)("g", {
                clipPath: "url(#1a58eb__a)",
                children: /*#__PURE__*/ (0, $2m7eD$jsxs)("g", {
                    clipPath: "url(#1a58eb__b)",
                    children: [
                        /*#__PURE__*/ (0, $2m7eD$jsx)("rect", {
                            width: 85.035,
                            height: 80.16,
                            x: 6.203,
                            y: 4.801,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2m7eD$jsx)("ellipse", {
                            cx: 82.001,
                            cy: 57.5,
                            fill: "url(#1a58eb__c)",
                            rx: 122.774,
                            ry: 115.736
                        }),
                        /*#__PURE__*/ (0, $2m7eD$jsx)("path", {
                            fill: "url(#1a58eb__d)",
                            d: "M-1.45 29.907C-2.816 2.201 19.904-21.303 49.296-22.59c29.391-1.286 54.324 20.132 55.689 47.838 1.365 27.707-21.356 51.211-50.747 52.498C24.847 79.032-.086 57.614-1.451 29.907"
                        }),
                        /*#__PURE__*/ (0, $2m7eD$jsx)("path", {
                            fill: "url(#1a58eb__e)",
                            d: "M27.687-50.667c-6.63-3.867-15.031-3.891-21.686-.06l-66.562 38.31c-6.52 3.752-6.53 12.609-.02 16.375L5.541 42.215c6.645 3.844 15.046 3.84 21.685-.014L93.212 3.909C99.688.15 99.7-8.657 93.232-12.43z"
                        }),
                        /*#__PURE__*/ (0, $2m7eD$jsx)("path", {
                            fill: "url(#1a58eb__f)",
                            d: "M-36.189 43.141c13.72 0 26.147 5.235 35.145 13.679 17.995 16.917 32.914 16.932 50.909.043 8.997-8.487 21.44-13.736 35.175-13.736 27.454.014 49.694 20.923 49.694 46.733s-22.24 46.703-49.694 46.703c-13.765 0-26.208-5.263-35.206-13.75-17.98-16.86-32.883-16.846-50.878.072-8.998 8.459-21.425 13.678-35.145 13.678-27.438 0-49.679-20.908-49.679-46.703s22.24-46.719 49.68-46.719"
                        })
                    ]
                })
            })
        ]
    });
const $169659c71a2b4781$var$ForwardRef = /*#__PURE__*/ (0, $2m7eD$forwardRef)($169659c71a2b4781$var$SvgComponent);
var $169659c71a2b4781$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($169659c71a2b4781$var$ForwardRef);


export {$169659c71a2b4781$export$2e2bcd8739ae039 as default};
