// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_market_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2eGiY$jsxs, jsx as $2eGiY$jsx} from "react/jsx-runtime";
import {forwardRef as $2eGiY$forwardRef} from "react";





const $3ddc8e07d1a2105d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2eGiY$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2eGiY$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2eGiY$jsxs)("linearGradient", {
                        id: "50ac4b__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2eGiY$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2eGiY$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2eGiY$jsxs)("linearGradient", {
                        id: "50ac4b__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2eGiY$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2eGiY$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2eGiY$jsxs)("linearGradient", {
                        id: "50ac4b__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2eGiY$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2eGiY$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2eGiY$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2eGiY$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2eGiY$jsxs)("radialGradient", {
                        id: "50ac4b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2eGiY$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2eGiY$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2eGiY$jsx)("clipPath", {
                        id: "50ac4b__b",
                        children: /*#__PURE__*/ (0, $2eGiY$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2eGiY$jsx)("clipPath", {
                id: "50ac4b__a",
                children: /*#__PURE__*/ (0, $2eGiY$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M34.9 81.148v34.62c0 8.836 7.163 16 16 16h11.703l-.005-50.751c-4.51-.767-8.463-3.18-11.207-6.601-3.357 4.184-8.522 6.861-14.305 6.861q-1.11 0-2.187-.129m35.698-.533.003 26.918h54.318V81.169q-.987.108-2.005.108c-5.783 0-10.948-2.677-14.304-6.861-3.357 4.184-8.522 6.861-14.305 6.861S83.357 78.6 80 74.416c-2.371 2.956-5.645 5.16-9.402 6.199m-4.903-7.338c-5.705 0-10.304-4.603-10.304-10.249v-.704c0-2.21-1.791-4-4-4s-4 1.79-4 4v.704c0 5.646-4.6 10.25-10.305 10.25-5.706 0-10.305-4.604-10.305-10.25 0-3.086.672-6.134 1.974-8.93l.001-.002 8.81-18.942v-.001h.001c.828-1.776 2.619-2.92 4.6-2.92h75.68c1.982 0 3.773 1.144 4.601 2.92l8.675 18.653q.054.146.121.288c1.3 2.801 1.975 5.852 1.975 8.934 0 5.646-4.599 10.25-10.305 10.25s-10.304-4.604-10.304-10.25v-.704c0-2.21-1.791-4-4-4s-4 1.79-4 4v.704c0 5.646-4.599 10.25-10.305 10.25S84 68.673 84 63.027v-.704c0-2.21-1.79-4-4-4s-4 1.79-4 4v.704c0 5.646-4.599 10.25-10.305 10.25m4.907 42.256h54.317v.235c0 8.836-7.163 16-16 16H70.603z"
                })
            }),
            /*#__PURE__*/ (0, $2eGiY$jsx)("g", {
                clipPath: "url(#50ac4b__a)",
                children: /*#__PURE__*/ (0, $2eGiY$jsxs)("g", {
                    clipPath: "url(#50ac4b__b)",
                    children: [
                        /*#__PURE__*/ (0, $2eGiY$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2eGiY$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#50ac4b__c)"
                        }),
                        /*#__PURE__*/ (0, $2eGiY$jsx)("path", {
                            fill: "url(#50ac4b__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $2eGiY$jsx)("path", {
                            fill: "url(#50ac4b__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $2eGiY$jsx)("path", {
                            fill: "url(#50ac4b__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $3ddc8e07d1a2105d$var$ForwardRef = /*#__PURE__*/ (0, $2eGiY$forwardRef)($3ddc8e07d1a2105d$var$SvgComponent);
var $3ddc8e07d1a2105d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($3ddc8e07d1a2105d$var$ForwardRef);


export {$3ddc8e07d1a2105d$export$2e2bcd8739ae039 as default};
