// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_megaphonePromote2_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $20Fu1$reactjsxruntime = require("react/jsx-runtime");
var $20Fu1$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $65432ebf5d11d2cf$export$2e2bcd8739ae039);




const $65432ebf5d11d2cf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsxs)("linearGradient", {
                        id: "58889f__d",
                        x1: 33.237,
                        x2: 12.135,
                        y1: 5.007,
                        y2: 34.029,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsxs)("linearGradient", {
                        id: "58889f__e",
                        x1: 19.369,
                        x2: 23.281,
                        y1: -7.8,
                        y2: 18.758,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsxs)("linearGradient", {
                        id: "58889f__f",
                        x1: -31.291,
                        x2: 42.236,
                        y1: 45.453,
                        y2: 30.038,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsxs)("radialGradient", {
                        id: "58889f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.4895 -56.7422 0 39.832 30.552)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("clipPath", {
                        id: "58889f__b",
                        children: /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.801,
                            y: 6.197,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("clipPath", {
                id: "58889f__a",
                children: /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m20.771 14.406 11.568-3.513.003-.003c1.725-.525 3.465.765 3.465 2.568v13.158c0 1.803-1.74 3.093-3.465 2.568l-11.568-3.52h-.012l3.132 7.276c.672 1.554-.042 3.357-1.593 4.029-1.554.672-3.357-.042-4.029-1.593l-4.22-9.708h-1.597c-3.108 0-5.63-2.523-5.63-5.631s2.52-5.631 5.63-5.631zm20.154 10.302 1.725.546c.63.2.981.873.78 1.506-.162.51-.633.837-1.143.837q-.181 0-.363-.057l-1.725-.546c-.63-.201-.98-.873-.78-1.506.201-.63.873-.981 1.506-.78m-.363-9.444q.181 0 .363-.057l1.725-.546c.633-.201.981-.876.78-1.506-.2-.633-.873-.981-1.506-.78l-1.725.546c-.633.2-.98.876-.78 1.506.162.51.633.837 1.143.837m1.806 5.895h-1.806c-.662 0-1.2-.537-1.2-1.2s.538-1.2 1.2-1.2h1.806c.663 0 1.2.537 1.2 1.2s-.536 1.2-1.2 1.2"
                })
            }),
            /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("g", {
                clipPath: "url(#58889f__a)",
                children: /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#58889f__b)",
                    children: [
                        /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.801,
                            y: 6.197,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("ellipse", {
                            cx: 39.832,
                            cy: 30.552,
                            fill: "url(#58889f__c)",
                            rx: 56.742,
                            ry: 53.489
                        }),
                        /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("path", {
                            fill: "url(#58889f__d)",
                            d: "M1.264 17.801C.634 4.996 11.134-5.866 24.718-6.46s25.107 9.304 25.738 22.11c.63 12.804-9.87 23.667-23.454 24.262S1.895 30.607 1.264 17.8"
                        }),
                        /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("path", {
                            fill: "url(#58889f__e)",
                            d: "M14.728-19.439c-3.065-1.787-6.947-1.798-10.023-.028L-26.058-1.761c-3.012 1.734-3.018 5.827-.009 7.568l30.56 17.68c3.07 1.777 6.953 1.775 10.022-.005L45.01 5.784c2.993-1.737 2.999-5.807.01-7.55z"
                        }),
                        /*#__PURE__*/ (0, $20Fu1$reactjsxruntime.jsx)("path", {
                            fill: "url(#58889f__f)",
                            d: "M-14.792 23.917c6.34 0 12.084 2.419 16.243 6.322 8.316 7.818 15.212 7.825 23.528.02 4.159-3.923 9.91-6.349 16.257-6.349 12.689.007 22.967 9.67 22.967 21.598 0 11.929-10.279 21.585-22.967 21.585-6.362 0-12.112-2.432-16.27-6.355-8.31-7.792-15.199-7.785-23.515.034-4.159 3.909-9.902 6.321-16.243 6.321-12.681 0-22.96-9.663-22.96-21.585 0-11.921 10.279-21.591 22.96-21.591"
                        })
                    ]
                })
            })
        ]
    });
const $65432ebf5d11d2cf$var$ForwardRef = /*#__PURE__*/ (0, $20Fu1$react.forwardRef)($65432ebf5d11d2cf$var$SvgComponent);
var $65432ebf5d11d2cf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($65432ebf5d11d2cf$var$ForwardRef);
