// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_dropToUpload_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $geMuF$jsxs, jsx as $geMuF$jsx} from "react/jsx-runtime";
import {forwardRef as $geMuF$forwardRef} from "react";





const $fa2aa1a4b80eda9a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $geMuF$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $geMuF$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $geMuF$jsxs)("linearGradient", {
                        id: "9326a3__d",
                        x1: 66.215,
                        x2: 15.136,
                        y1: -0.437,
                        y2: 65.788,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $geMuF$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $geMuF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $geMuF$jsxs)("linearGradient", {
                        id: "9326a3__e",
                        x1: 35.097,
                        x2: 44.951,
                        y1: -30.924,
                        y2: 32.138,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $geMuF$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $geMuF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $geMuF$jsxs)("linearGradient", {
                        id: "9326a3__f",
                        x1: -78.599,
                        x2: 87.191,
                        y1: 95.859,
                        y2: 63.093,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $geMuF$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $geMuF$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $geMuF$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $geMuF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $geMuF$jsxs)("radialGradient", {
                        id: "9326a3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 10.316 70.7)scale(127.345)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $geMuF$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $geMuF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $geMuF$jsx)("clipPath", {
                        id: "9326a3__b",
                        children: /*#__PURE__*/ (0, $geMuF$jsx)("rect", {
                            width: 88.2,
                            height: 88.2,
                            x: 2.398,
                            y: 2.4,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $geMuF$jsx)("clipPath", {
                id: "9326a3__a",
                children: /*#__PURE__*/ (0, $geMuF$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M31.046 13.16h16.442c1.687 0 3.304.67 4.496 1.863l17.472 17.47c1.192 1.193 1.862 2.81 1.862 4.497v24.921c0 3.512-2.847 6.359-6.359 6.359H31.046c-3.512 0-6.36-2.847-6.36-6.359V19.519c0-3.512 2.848-6.359 6.36-6.359m18.6 18.375c.002-1.17-.945-2.121-2.115-2.124s-2.122.944-2.125 2.115l-.047 20.99-7.165-5.7c-.916-.729-2.25-.577-2.979.34-.728.916-.576 2.25.34 2.978l10.675 8.49c.777.618 1.879.615 2.651-.009l10.52-8.49c.911-.736 1.054-2.07.319-2.981s-2.07-1.054-2.981-.319l-7.14 5.764zm-36.204 13.78c0-1.06.86-1.92 1.92-1.92h3.528c1.06 0 1.92-.86 1.92-1.92s-.86-1.92-1.92-1.92h-3.528c-3.182 0-5.76 2.578-5.76 5.76v3.32c0 1.061.86 1.92 1.92 1.92s1.92-.859 1.92-1.92zm63.67-5.76c-1.06 0-1.92.86-1.92 1.92s.86 1.92 1.92 1.92h3.53c1.06 0 1.92.86 1.92 1.92v3.32c0 1.061.86 1.92 1.92 1.92s1.92-.859 1.92-1.92v-3.32c0-3.182-2.58-5.76-5.76-5.76zm9.29 17.798c0-1.06-.86-1.92-1.92-1.92s-1.92.86-1.92 1.92v6.643c0 1.06.86 1.92 1.92 1.92s1.92-.86 1.92-1.92zm-72.96 0c0-1.06-.86-1.92-1.92-1.92s-1.92.86-1.92 1.92v6.643c0 1.06.86 1.92 1.92 1.92s1.92-.86 1.92-1.92zm0 15.36c0-1.06-.86-1.92-1.92-1.92s-1.92.86-1.92 1.92v3.322c0 3.18 2.578 5.76 5.76 5.76h3.528c1.06 0 1.92-.86 1.92-1.92s-.86-1.92-1.92-1.92h-3.528c-1.06 0-1.92-.86-1.92-1.92zm72.96 0c0-1.06-.86-1.92-1.92-1.92s-1.92.86-1.92 1.92v3.322c0 1.06-.86 1.92-1.92 1.92h-3.53c-1.06 0-1.92.86-1.92 1.92s.86 1.92 1.92 1.92h3.53c3.18 0 5.76-2.58 5.76-5.76zm-58.25 5.242c-1.06 0-1.92.86-1.92 1.92s.86 1.92 1.92 1.92h7.058c1.06 0 1.92-.86 1.92-1.92s-.86-1.92-1.92-1.92zm16.32 0c-1.06 0-1.92.86-1.92 1.92s.86 1.92 1.92 1.92h7.058c1.06 0 1.92-.86 1.92-1.92s-.86-1.92-1.92-1.92zm16.32 0c-1.06 0-1.92.86-1.92 1.92s.86 1.92 1.92 1.92h7.058c1.06 0 1.92-.86 1.92-1.92s-.86-1.92-1.92-1.92z"
                })
            }),
            /*#__PURE__*/ (0, $geMuF$jsx)("g", {
                clipPath: "url(#9326a3__a)",
                children: /*#__PURE__*/ (0, $geMuF$jsxs)("g", {
                    clipPath: "url(#9326a3__b)",
                    children: [
                        /*#__PURE__*/ (0, $geMuF$jsx)("rect", {
                            width: 88.2,
                            height: 88.2,
                            x: 2.398,
                            y: 2.4,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $geMuF$jsx)("circle", {
                            cx: 81.017,
                            cy: 60.384,
                            r: 127.345,
                            fill: "url(#9326a3__c)"
                        }),
                        /*#__PURE__*/ (0, $geMuF$jsx)("path", {
                            fill: "url(#9326a3__d)",
                            d: "M-5.54 30.025C-6.956-.461 16.61-26.322 47.096-27.738s56.347 22.15 57.763 52.636-22.15 56.347-52.636 57.763S-4.125 60.51-5.54 30.025"
                        }),
                        /*#__PURE__*/ (0, $geMuF$jsx)("path", {
                            fill: "url(#9326a3__e)",
                            d: "M24.681-58.632c-6.877-4.255-15.59-4.282-22.493-.067l-69.04 42.153c-6.761 4.129-6.773 13.874-.02 18.018L1.71 43.566c6.893 4.23 15.606 4.224 22.493-.016L92.646 1.417c6.717-4.135 6.729-13.825.02-17.976z"
                        }),
                        /*#__PURE__*/ (0, $geMuF$jsx)("path", {
                            fill: "url(#9326a3__f)",
                            d: "M-41.57 44.586c14.23 0 27.12 5.76 36.452 15.05 18.665 18.615 34.14 18.63 52.804.048C57.02 50.347 69.925 44.57 84.171 44.57c28.476.016 51.544 23.022 51.544 51.42s-23.068 51.388-51.544 51.388c-14.278 0-27.184-5.791-36.517-15.13-18.649-18.55-34.107-18.534-52.772.08-9.333 9.307-22.223 15.05-36.453 15.05-28.46 0-51.528-23.005-51.528-51.388s23.068-51.404 51.528-51.404"
                        })
                    ]
                })
            })
        ]
    });
const $fa2aa1a4b80eda9a$var$ForwardRef = /*#__PURE__*/ (0, $geMuF$forwardRef)($fa2aa1a4b80eda9a$var$SvgComponent);
var $fa2aa1a4b80eda9a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($fa2aa1a4b80eda9a$var$ForwardRef);


export {$fa2aa1a4b80eda9a$export$2e2bcd8739ae039 as default};
