// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_infiniteLooping_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $sbqgg$reactjsxruntime = require("react/jsx-runtime");
var $sbqgg$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $756854c9df2baa84$export$2e2bcd8739ae039);




const $756854c9df2baa84$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsxs)("linearGradient", {
                        id: "714e9e__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsxs)("linearGradient", {
                        id: "714e9e__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsxs)("linearGradient", {
                        id: "714e9e__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsxs)("radialGradient", {
                        id: "714e9e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("clipPath", {
                        id: "714e9e__b",
                        children: /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("clipPath", {
                id: "714e9e__a",
                children: /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M21.55 22.053c.69-.002 1.248-.563 1.247-1.253l-.018-7.506c-.001-.69-.562-1.249-1.253-1.247s-1.248.562-1.247 1.253l.01 4.455c-2.007-2.07-4.82-3.358-7.933-3.358-6.103 0-11.051 4.948-11.051 11.051 0 6.104 4.948 11.051 11.051 11.051 2.532 0 4.868-.852 6.732-2.286q.043-.033.083-.07l.034-.026c1.373-1.077 2.648-2.64 3.644-4.066 1-1.434 1.784-2.828 2.152-3.64.285-.629.99-2.123 1.924-3.692.803-1.352 1.717-2.656 2.618-3.532q.042-.033.083-.07c1.52-1.38 3.535-2.22 5.749-2.22 4.723 0 8.551 3.828 8.551 8.551s-3.828 8.551-8.551 8.551c-2.888 0-5.442-1.43-6.992-3.627-.398-.564-1.178-.698-1.742-.3s-.699 1.178-.3 1.742c1.998 2.832 5.3 4.685 9.034 4.685 6.103 0 11.051-4.948 11.051-11.051s-4.948-11.051-11.051-11.051c-2.527 0-4.857.849-6.719 2.276q-.081.046-.158.107c-1.443 1.137-2.744 3.016-3.722 4.66-.998 1.68-1.747 3.266-2.052 3.939-.29.642-.99 1.902-1.925 3.241-.94 1.346-2.05 2.677-3.138 3.531q-.038.03-.073.062l-.024.018C16.122 33.341 14.318 34 12.356 34c-4.723 0-8.551-3.828-8.551-8.551s3.828-8.551 8.551-8.551c2.44 0 4.642 1.02 6.201 2.663l-4.52.01c-.69.002-1.248.563-1.246 1.253.001.69.562 1.249 1.253 1.247z"
                })
            }),
            /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("g", {
                clipPath: "url(#714e9e__a)",
                children: /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#714e9e__b)",
                    children: [
                        /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#714e9e__c)"
                        }),
                        /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("path", {
                            fill: "url(#714e9e__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("path", {
                            fill: "url(#714e9e__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $sbqgg$reactjsxruntime.jsx)("path", {
                            fill: "url(#714e9e__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $756854c9df2baa84$var$ForwardRef = /*#__PURE__*/ (0, $sbqgg$react.forwardRef)($756854c9df2baa84$var$SvgComponent);
var $756854c9df2baa84$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($756854c9df2baa84$var$ForwardRef);
