// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_education_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $dPboR$reactjsxruntime = require("react/jsx-runtime");
var $dPboR$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $763c8ec8d29edaa0$export$2e2bcd8739ae039);




const $763c8ec8d29edaa0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsxs)("linearGradient", {
                        id: "59619b__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsxs)("linearGradient", {
                        id: "59619b__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsxs)("linearGradient", {
                        id: "59619b__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsxs)("radialGradient", {
                        id: "59619b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("clipPath", {
                        id: "59619b__b",
                        children: /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("clipPath", {
                id: "59619b__a",
                children: /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m9.31 15.316 13.913-5.683c.485-.198 1.028-.198 1.513 0l13.912 5.683c1.659.678 1.659 3.026 0 3.703l-1.612.659v6.669c0 .874-.234 2.13-1.11 3.436-.88 1.31-2.357 2.599-4.73 3.616-1.918.821-4.123 1.136-6.042 1.207-1.924.071-3.665-.098-4.706-.312-.676-.14-1.111-.8-.972-1.476.14-.676.8-1.112 1.476-.972.818.168 2.356.327 4.109.262 1.756-.066 3.623-.353 5.15-1.007 2.007-.86 3.072-1.867 3.639-2.712.57-.848.686-1.605.686-2.042v-5.648l-9.8 4.003c-.485.198-1.028.198-1.513 0l-6.298-2.572c.228-.304.52-.562.864-.753l6.798-3.777c.603-.335.82-1.096.485-1.7s-1.096-.82-1.7-.485l-6.797 3.776c-.85.473-1.536 1.162-2.004 1.977L9.31 19.02c-1.66-.677-1.66-3.025 0-3.703m4.565 17.237V23.78c0-.934.248-1.831.696-2.613l2.354.962c-.351.468-.55 1.045-.55 1.65v8.487c.765 1.378 1.522 3.087 1.676 4.546q.025.174.025.355c0 1.45-1.248 2.624-2.787 2.624-1.54 0-2.788-1.174-2.788-2.623q0-.147.017-.288c.113-1.376.715-2.983 1.357-4.328"
                })
            }),
            /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("g", {
                clipPath: "url(#59619b__a)",
                children: /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#59619b__b)",
                    children: [
                        /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#59619b__c)"
                        }),
                        /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("path", {
                            fill: "url(#59619b__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("path", {
                            fill: "url(#59619b__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $dPboR$reactjsxruntime.jsx)("path", {
                            fill: "url(#59619b__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $763c8ec8d29edaa0$var$ForwardRef = /*#__PURE__*/ (0, $dPboR$react.forwardRef)($763c8ec8d29edaa0$var$SvgComponent);
var $763c8ec8d29edaa0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($763c8ec8d29edaa0$var$ForwardRef);
