// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_remix_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $iXbGh$reactjsxruntime = require("react/jsx-runtime");
var $iXbGh$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0df96fc8f253a50d$export$2e2bcd8739ae039);




const $0df96fc8f253a50d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2669b1__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2669b1__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2669b1__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2669b1__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("clipPath", {
                        id: "2669b1__b",
                        children: /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("clipPath", {
                id: "2669b1__a",
                children: /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M129.934 67.666 148.52 49.08c1.562-1.562 1.562-4.094 0-5.657l-18.586-18.585c-1.562-1.562-4.095-1.562-5.657 0s-1.562 4.094 0 5.656l11.402 11.403h-7.178c-8.729 0-16.783 2.94-23.21 7.882q-.147.113-.279.236-.064.046-.127.095c-4.735 3.718-9.15 9.122-12.607 14.075-3.471 4.976-6.188 9.807-7.458 12.609-.993 2.193-3.454 7.406-6.71 12.882-2.803 4.713-6.005 9.29-9.182 12.376q-.145.11-.282.234c-5.349 4.86-12.443 7.817-20.238 7.817-16.626 0-30.103-13.477-30.103-30.103s13.477-30.103 30.103-30.103c10.167 0 19.158 5.036 24.614 12.767l.007.01.008.01 2.081 2.911c1.285 1.797 3.783 2.213 5.58.928s2.213-3.784.928-5.58L79.55 58.04C72.66 48.282 61.28 41.897 48.408 41.897c-21.044 0-38.103 17.059-38.103 38.103s17.06 38.103 38.103 38.103c8.72 0 16.764-2.934 23.185-7.863q.273-.153.527-.353c4.965-3.915 9.466-10.406 12.866-16.122 3.465-5.828 6.063-11.334 7.121-13.67 1.023-2.256 3.471-6.658 6.732-11.332 3.276-4.696 7.161-9.357 10.986-12.36q.13-.102.248-.211l.095-.071c5.077-3.905 11.429-6.224 18.333-6.224h7.889l-12.113 12.112c-1.562 1.563-1.562 4.095 0 5.657s4.095 1.562 5.657 0m8.019 50.437c2.21 0 4-1.791 4-4s-1.79-4-4-4h-9.452c-9.961 0-18.039-5.729-25.113-13.446l-2.717-3.283c-1.408-1.702-3.93-1.94-5.631-.531s-1.94 3.93-.532 5.632l2.78 3.36.063.076.067.073c7.629 8.342 17.72 16.119 31.083 16.119z"
                })
            }),
            /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("g", {
                clipPath: "url(#2669b1__a)",
                children: /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#2669b1__b)",
                    children: [
                        /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#2669b1__c)"
                        }),
                        /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("path", {
                            fill: "url(#2669b1__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("path", {
                            fill: "url(#2669b1__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $iXbGh$reactjsxruntime.jsx)("path", {
                            fill: "url(#2669b1__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $0df96fc8f253a50d$var$ForwardRef = /*#__PURE__*/ (0, $iXbGh$react.forwardRef)($0df96fc8f253a50d$var$SvgComponent);
var $0df96fc8f253a50d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0df96fc8f253a50d$var$ForwardRef);
