// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_calendar_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $Q2aXk$jsxs, jsx as $Q2aXk$jsx} from "react/jsx-runtime";
import {forwardRef as $Q2aXk$forwardRef} from "react";





const $f0a973200e0e2f51$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $Q2aXk$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $Q2aXk$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $Q2aXk$jsxs)("linearGradient", {
                        id: "c18bd7__d",
                        x1: 33.722,
                        x2: 11.235,
                        y1: 1.731,
                        y2: 32.659,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $Q2aXk$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $Q2aXk$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $Q2aXk$jsxs)("linearGradient", {
                        id: "c18bd7__e",
                        x1: 18.946,
                        x2: 23.114,
                        y1: -11.916,
                        y2: 16.386,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $Q2aXk$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $Q2aXk$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $Q2aXk$jsxs)("linearGradient", {
                        id: "c18bd7__f",
                        x1: -35.042,
                        x2: 43.313,
                        y1: 44.835,
                        y2: 28.408,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $Q2aXk$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $Q2aXk$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $Q2aXk$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $Q2aXk$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $Q2aXk$jsxs)("radialGradient", {
                        id: "c18bd7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 57.0019 -60.4683 0 40.75 28.955)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $Q2aXk$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $Q2aXk$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $Q2aXk$jsx)("clipPath", {
                        id: "c18bd7__b",
                        children: /*#__PURE__*/ (0, $Q2aXk$jsx)("rect", {
                            width: 41.881,
                            height: 39.48,
                            x: 3.418,
                            y: 3,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $Q2aXk$jsx)("clipPath", {
                id: "c18bd7__a",
                children: /*#__PURE__*/ (0, $Q2aXk$jsx)("path", {
                    fill: "#fff",
                    d: "M34.798 10.2h-2.1v3c0 .663-.537 1.2-1.2 1.2s-1.2-.537-1.2-1.2v-3h-12.6v3c0 .663-.537 1.2-1.2 1.2s-1.2-.537-1.2-1.2v-3h-2.1c-2.652 0-4.8 2.148-4.8 4.8v3h31.2v-3c0-2.652-2.148-4.8-4.8-4.8m-26.4 24.6c0 2.652 2.148 4.8 4.8 4.8h21.6c2.652 0 4.8-2.148 4.8-4.8V20.4h-31.2zm21.9-9.6c0-.663.537-1.2 1.2-1.2h1.8c.663 0 1.2.537 1.2 1.2V27c0 .663-.537 1.2-1.2 1.2h-1.8c-.663 0-1.2-.537-1.2-1.2zm0 6.6c0-.663.537-1.2 1.2-1.2h1.8c.663 0 1.2.537 1.2 1.2v1.8c0 .663-.537 1.2-1.2 1.2h-1.8c-.663 0-1.2-.537-1.2-1.2zm-8.4-6.6c0-.663.537-1.2 1.2-1.2h1.8c.663 0 1.2.537 1.2 1.2V27c0 .663-.537 1.2-1.2 1.2h-1.8c-.663 0-1.2-.537-1.2-1.2zm0 6.6c0-.663.537-1.2 1.2-1.2h1.8c.663 0 1.2.537 1.2 1.2v1.8c0 .663-.537 1.2-1.2 1.2h-1.8c-.663 0-1.2-.537-1.2-1.2zm-8.4-6.6c0-.663.537-1.2 1.2-1.2h1.8c.663 0 1.2.537 1.2 1.2V27c0 .663-.537 1.2-1.2 1.2h-1.8c-.663 0-1.2-.537-1.2-1.2zm0 6.6c0-.663.537-1.2 1.2-1.2h1.8c.663 0 1.2.537 1.2 1.2v1.8c0 .663-.537 1.2-1.2 1.2h-1.8c-.663 0-1.2-.537-1.2-1.2zm4.2-24.6c0-.663-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2v3h2.4zm15 0c0-.663-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2v3h2.4z"
                })
            }),
            /*#__PURE__*/ (0, $Q2aXk$jsx)("g", {
                clipPath: "url(#c18bd7__a)",
                children: /*#__PURE__*/ (0, $Q2aXk$jsxs)("g", {
                    clipPath: "url(#c18bd7__b)",
                    children: [
                        /*#__PURE__*/ (0, $Q2aXk$jsx)("rect", {
                            width: 41.881,
                            height: 39.48,
                            x: 3.418,
                            y: 3,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $Q2aXk$jsx)("ellipse", {
                            cx: 40.749,
                            cy: 28.955,
                            fill: "url(#c18bd7__c)",
                            rx: 60.468,
                            ry: 57.002
                        }),
                        /*#__PURE__*/ (0, $Q2aXk$jsx)("path", {
                            fill: "url(#c18bd7__d)",
                            d: "M-.35 15.366C-1.022 1.72 10.168-9.856 24.644-10.49S51.4-.575 52.072 13.071 41.554 38.293 27.078 38.927C12.602 39.56.323 29.012-.35 15.366"
                        }),
                        /*#__PURE__*/ (0, $Q2aXk$jsx)("path", {
                            fill: "url(#c18bd7__e)",
                            d: "M14-24.319c-3.266-1.905-7.403-1.917-10.681-.03L-29.464-5.48c-3.21 1.848-3.216 6.21-.01 8.064L3.093 21.426c3.272 1.894 7.41 1.891 10.68-.006L46.272 2.56c3.19-1.85 3.195-6.188.01-8.046z"
                        }),
                        /*#__PURE__*/ (0, $Q2aXk$jsx)("path", {
                            fill: "url(#c18bd7__f)",
                            d: "M-17.46 21.884c6.758 0 12.878 2.578 17.31 6.737 8.862 8.332 16.21 8.34 25.073.021 4.432-4.18 10.56-6.765 17.325-6.765 13.521.007 24.475 10.305 24.475 23.017S55.769 67.896 42.248 67.896c-6.78 0-12.908-2.592-17.34-6.773-8.855-8.303-16.196-8.296-25.058.036-4.432 4.166-10.552 6.737-17.31 6.737-13.513 0-24.467-10.298-24.467-23.002 0-12.705 10.953-23.01 24.468-23.01"
                        })
                    ]
                })
            })
        ]
    });
const $f0a973200e0e2f51$var$ForwardRef = /*#__PURE__*/ (0, $Q2aXk$forwardRef)($f0a973200e0e2f51$var$SvgComponent);
var $f0a973200e0e2f51$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f0a973200e0e2f51$var$ForwardRef);


export {$f0a973200e0e2f51$export$2e2bcd8739ae039 as default};
