// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_trash_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $kgNUH$reactjsxruntime = require("react/jsx-runtime");
var $kgNUH$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0b28360132b82be0$export$2e2bcd8739ae039);




const $0b28360132b82be0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsxs)("linearGradient", {
                        id: "513f63__d",
                        x1: 32.783,
                        x2: 12.551,
                        y1: 5.818,
                        y2: 33.644,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsxs)("linearGradient", {
                        id: "513f63__e",
                        x1: 19.489,
                        x2: 23.239,
                        y1: -6.46,
                        y2: 19.004,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsxs)("linearGradient", {
                        id: "513f63__f",
                        x1: -29.084,
                        x2: 41.411,
                        y1: 44.599,
                        y2: 29.819,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsxs)("radialGradient", {
                        id: "513f63__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.2844 -54.4031 0 39.106 30.312)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("clipPath", {
                        id: "513f63__b",
                        children: /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("clipPath", {
                id: "513f63__a",
                children: /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M16.8 9.8c0-1.657 1.344-3 3-3h8.4c1.658 0 3 1.343 3 3v1.8h9.3c.663 0 1.2.537 1.2 1.2s-.537 1.2-1.2 1.2h-2.85L35.03 36.025c-.216 1.811-1.752 3.175-3.575 3.175H16.548c-1.824 0-3.36-1.364-3.575-3.175L10.35 14H7.5c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.2-1.2h9.3zm3-.6c-.331 0-.6.269-.6.6v1.8h9.6V9.8c0-.332-.268-.6-.6-.6zm.6 7.2c-.662 0-1.2.537-1.2 1.2v16.8c0 .663.538 1.2 1.2 1.2.664 0 1.2-.538 1.2-1.2V17.6c0-.663-.537-1.2-1.2-1.2m7.2 0c-.662 0-1.2.537-1.2 1.2v16.8c0 .663.538 1.2 1.2 1.2.663 0 1.2-.538 1.2-1.2V17.6c0-.663-.537-1.2-1.2-1.2"
                })
            }),
            /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("g", {
                clipPath: "url(#513f63__a)",
                children: /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#513f63__b)",
                    children: [
                        /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("ellipse", {
                            cx: 39.106,
                            cy: 30.312,
                            fill: "url(#513f63__c)",
                            rx: 54.403,
                            ry: 51.284
                        }),
                        /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("path", {
                            fill: "url(#513f63__d)",
                            d: "M2.128 18.086C1.523 5.807 11.591-4.608 24.615-5.178s24.072 8.92 24.677 21.198c.605 12.277-9.463 22.692-22.487 23.262S2.733 30.363 2.128 18.086"
                        }),
                        /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("path", {
                            fill: "url(#513f63__e)",
                            d: "M15.039-17.618c-2.938-1.714-6.66-1.725-9.61-.027L-24.064-.67c-2.889 1.662-2.893 5.587-.009 7.256l29.3 16.952c2.944 1.703 6.667 1.7 9.609-.007l29.24-16.967c2.869-1.666 2.874-5.568.008-7.24z"
                        }),
                        /*#__PURE__*/ (0, $kgNUH$reactjsxruntime.jsx)("path", {
                            fill: "url(#513f63__f)",
                            d: "M-13.266 23.95c6.08 0 11.586 2.32 15.573 6.06 7.974 7.497 14.585 7.504 22.559.02 3.987-3.76 9.5-6.087 15.586-6.087 12.166.007 22.02 9.272 22.02 20.708 0 11.437-9.854 20.695-22.02 20.695-6.1 0-11.613-2.332-15.6-6.093-7.967-7.47-14.57-7.464-22.545.032-3.987 3.748-9.494 6.061-15.573 6.061-12.158 0-22.013-9.264-22.013-20.695 0-11.43 9.855-20.701 22.013-20.701"
                        })
                    ]
                })
            })
        ]
    });
const $0b28360132b82be0$var$ForwardRef = /*#__PURE__*/ (0, $kgNUH$react.forwardRef)($0b28360132b82be0$var$SvgComponent);
var $0b28360132b82be0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0b28360132b82be0$var$ForwardRef);
