// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_upload_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $bciud$jsxs, jsx as $bciud$jsx} from "react/jsx-runtime";
import {forwardRef as $bciud$forwardRef} from "react";





const $e6a9b775136f3b54$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bciud$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bciud$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bciud$jsxs)("linearGradient", {
                        id: "cbb2b3__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.544,
                        y2: 34.497,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bciud$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $bciud$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bciud$jsxs)("linearGradient", {
                        id: "cbb2b3__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.136,
                        y2: 16.183,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bciud$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $bciud$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bciud$jsxs)("linearGradient", {
                        id: "cbb2b3__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.031,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bciud$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bciud$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $bciud$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $bciud$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bciud$jsxs)("radialGradient", {
                        id: "cbb2b3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.172)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bciud$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bciud$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bciud$jsx)("clipPath", {
                        id: "cbb2b3__b",
                        children: /*#__PURE__*/ (0, $bciud$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bciud$jsx)("clipPath", {
                id: "cbb2b3__a",
                children: /*#__PURE__*/ (0, $bciud$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M8.5 24.173c.69 0 1.25.56 1.25 1.25v8.823c0 1.35 1.083 2.427 2.397 2.427h23.706c1.314 0 2.397-1.077 2.397-2.427v-8.823c0-.69.56-1.25 1.25-1.25s1.25.56 1.25 1.25v8.823c0 2.712-2.183 4.927-4.897 4.927H12.147c-2.714 0-4.897-2.215-4.897-4.927v-8.823c0-.69.56-1.25 1.25-1.25M24 32.927c-.69 0-1.25-.56-1.25-1.25V13.096l-4.866 4.866c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.768l7-7c.488-.488 1.28-.488 1.768 0l7 7c.488.488.488 1.28 0 1.768s-1.28.488-1.768 0l-4.866-4.866v18.58c0 .691-.56 1.25-1.25 1.25"
                })
            }),
            /*#__PURE__*/ (0, $bciud$jsx)("g", {
                clipPath: "url(#cbb2b3__a)",
                children: /*#__PURE__*/ (0, $bciud$jsxs)("g", {
                    clipPath: "url(#cbb2b3__b)",
                    children: [
                        /*#__PURE__*/ (0, $bciud$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $bciud$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.556,
                            r: 69.303,
                            fill: "url(#cbb2b3__c)"
                        }),
                        /*#__PURE__*/ (0, $bciud$jsx)("path", {
                            fill: "url(#cbb2b3__d)",
                            d: "M-4.32 15.034C-5.09-1.558 7.735-15.631 24.326-16.402c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.624-4.32 15.033"
                        }),
                        /*#__PURE__*/ (0, $bciud$jsx)("path", {
                            fill: "url(#cbb2b3__e)",
                            d: "M12.124-33.215c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.403c3.75 2.301 8.493 2.299 12.24-.009L49.112-.536c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $bciud$jsx)("path", {
                            fill: "url(#cbb2b3__f)",
                            d: "M-23.927 22.959c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.9 44.503 78.9c-7.77 0-14.794-3.151-19.872-8.234C14.48 60.571 6.069 60.58-4.09 70.71-9.17 75.775-16.183 78.9-23.927 78.9c-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $e6a9b775136f3b54$var$ForwardRef = /*#__PURE__*/ (0, $bciud$forwardRef)($e6a9b775136f3b54$var$SvgComponent);
var $e6a9b775136f3b54$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e6a9b775136f3b54$var$ForwardRef);


export {$e6a9b775136f3b54$export$2e2bcd8739ae039 as default};
