// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_globeGrid_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $gydh2$jsxs, jsx as $gydh2$jsx} from "react/jsx-runtime";
import {forwardRef as $gydh2$forwardRef} from "react";





const $d5948431898416d5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gydh2$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gydh2$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gydh2$jsxs)("linearGradient", {
                        id: "d7fb2e__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gydh2$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $gydh2$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gydh2$jsxs)("linearGradient", {
                        id: "d7fb2e__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gydh2$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $gydh2$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gydh2$jsxs)("linearGradient", {
                        id: "d7fb2e__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gydh2$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gydh2$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $gydh2$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $gydh2$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gydh2$jsxs)("radialGradient", {
                        id: "d7fb2e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gydh2$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gydh2$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gydh2$jsx)("clipPath", {
                        id: "d7fb2e__b",
                        children: /*#__PURE__*/ (0, $gydh2$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gydh2$jsx)("clipPath", {
                id: "d7fb2e__a",
                children: /*#__PURE__*/ (0, $gydh2$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m50.028 15.076-.06-.004v14.605h9.21c-2.07-4.968-5.037-9.767-8.915-14.265q-.137-.16-.235-.336m13.46 14.6c-1.804-4.74-4.35-9.348-7.652-13.725 8.128 1.98 15.076 6.972 19.596 13.726zm-2.845 4H49.968v12.676h12.816c-.115-4.266-.826-8.516-2.141-12.675m6.143 12.676c-.105-4.259-.755-8.506-1.961-12.675H77.72c1.867 3.865 3.001 8.15 3.224 12.675zm-4.067 4H49.968V63.03h10.231c1.423-4.17 2.266-8.42 2.52-12.678m1.685 12.678c1.309-4.174 2.086-8.421 2.321-12.678h14.178c-.32 4.542-1.56 8.83-3.533 12.678zm-5.758 4h-8.678v13.898l.061-.004q.087-.156.203-.298c3.55-4.322 6.36-8.89 8.414-13.596M55.71 80.08c3.042-4.18 5.469-8.558 7.267-13.049h11.967c-4.548 6.429-11.34 11.158-19.234 13.049M20.568 29.677c4.505-6.733 11.424-11.714 19.518-13.707-3.295 4.371-5.836 8.974-7.637 13.707zm-2.006 4h12.55c-1.206 4.17-1.857 8.416-1.962 12.675H15.055c.223-4.532 1.36-8.824 3.232-12.694q.135.019.275.019m16.732 0c-1.316 4.16-2.026 8.409-2.142 12.675h12.816V33.677zM15.097 50.352H29.21c.236 4.257 1.012 8.505 2.32 12.678H18.63c-1.974-3.849-3.213-8.136-3.533-12.678m18.12 0c.255 4.258 1.098 8.508 2.52 12.678h10.23V50.352zM21.054 67.03h11.904c1.796 4.485 4.218 8.856 7.254 13.03-7.862-1.902-14.624-6.622-19.158-13.03m24.65 13.596q.115.14.201.294l.063.004V67.03H37.29c2.054 4.705 4.863 9.274 8.413 13.596m-.03-65.214c-3.879 4.498-6.845 9.297-8.916 14.265h9.21v-14.6l-.062.003q-.097.174-.233.332"
                })
            }),
            /*#__PURE__*/ (0, $gydh2$jsx)("g", {
                clipPath: "url(#d7fb2e__a)",
                children: /*#__PURE__*/ (0, $gydh2$jsxs)("g", {
                    clipPath: "url(#d7fb2e__b)",
                    children: [
                        /*#__PURE__*/ (0, $gydh2$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $gydh2$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#d7fb2e__c)"
                        }),
                        /*#__PURE__*/ (0, $gydh2$jsx)("path", {
                            fill: "url(#d7fb2e__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $gydh2$jsx)("path", {
                            fill: "url(#d7fb2e__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $gydh2$jsx)("path", {
                            fill: "url(#d7fb2e__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $d5948431898416d5$var$ForwardRef = /*#__PURE__*/ (0, $gydh2$forwardRef)($d5948431898416d5$var$SvgComponent);
var $d5948431898416d5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($d5948431898416d5$var$ForwardRef);


export {$d5948431898416d5$export$2e2bcd8739ae039 as default};
