// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lightbulbRays_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $45uhR$reactjsxruntime = require("react/jsx-runtime");
var $45uhR$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $394215a7886439e4$export$2e2bcd8739ae039);




const $394215a7886439e4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsxs)("linearGradient", {
                        id: "490f1f__d",
                        x1: 32.334,
                        x2: 11.233,
                        y1: 5.007,
                        y2: 34.029,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsxs)("linearGradient", {
                        id: "490f1f__e",
                        x1: 18.466,
                        x2: 22.378,
                        y1: -7.8,
                        y2: 18.758,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsxs)("linearGradient", {
                        id: "490f1f__f",
                        x1: -32.193,
                        x2: 41.333,
                        y1: 45.453,
                        y2: 30.038,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsxs)("radialGradient", {
                        id: "490f1f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.4895 -56.7422 0 38.93 30.552)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("clipPath", {
                        id: "490f1f__b",
                        children: /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.197,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("clipPath", {
                id: "490f1f__a",
                children: /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M25.2 8.985c0 .663-.537 1.2-1.2 1.2s-1.2-.537-1.2-1.2V7.179c0-.663.537-1.2 1.2-1.2s1.2.537 1.2 1.2zm-1.86 4.119c-4.454.318-8.06 3.95-8.39 8.448-.243 3.342 1.284 6.336 3.735 8.127.795.582 1.28 1.503 1.28 2.496V34.2h7.96q.032 0 .064.006l.043.005v-2.033c0-.97.445-1.893 1.224-2.457 2.31-1.663 3.816-4.39 3.816-7.474 0-5.285-4.425-9.524-9.735-9.146zm5.811 9.27c-.663 0-1.2-.537-1.2-1.2 0-1.503-1.332-2.724-2.97-2.724-.663 0-1.2-.537-1.2-1.2s.538-1.2 1.2-1.2c2.961 0 5.37 2.297 5.37 5.124 0 .663-.537 1.2-1.2 1.2M19.97 36.6v1.44c0 .843.678 1.526 1.512 1.526h5.043c.834 0 1.512-.684 1.512-1.527v-1.452l-.044.006q-.032.005-.064.006zM8.985 20.993h1.806c.663 0 1.2.537 1.2 1.2s-.537 1.2-1.2 1.2H8.985c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.2-1.2m30.027 0h-1.806c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h1.806c.663 0 1.2-.537 1.2-1.2s-.537-1.2-1.2-1.2M33.77 10.728l-1.275 1.274c-.468.468-.468 1.227 0 1.698.234.235.54.352.849.352s.615-.117.849-.351l1.275-1.276c.468-.468.468-1.227 0-1.698-.468-.468-1.227-.468-1.698 0m-19.534.002 1.275 1.275c.468.471.468 1.23 0 1.698-.234.235-.54.352-.849.352s-.615-.117-.849-.351l-1.275-1.276c-.468-.47-.468-1.23 0-1.698.471-.468 1.23-.468 1.698 0"
                })
            }),
            /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("g", {
                clipPath: "url(#490f1f__a)",
                children: /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#490f1f__b)",
                    children: [
                        /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.197,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("ellipse", {
                            cx: 38.93,
                            cy: 30.552,
                            fill: "url(#490f1f__c)",
                            rx: 56.742,
                            ry: 53.489
                        }),
                        /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("path", {
                            fill: "url(#490f1f__d)",
                            d: "M.362 17.801C-.268 4.996 10.232-5.866 23.815-6.46s25.107 9.304 25.738 22.11c.631 12.804-9.87 23.667-23.453 24.262S.993 30.607.362 17.8"
                        }),
                        /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("path", {
                            fill: "url(#490f1f__e)",
                            d: "M13.825-19.439c-3.064-1.787-6.947-1.798-10.022-.028L-26.96-1.761c-3.013 1.734-3.018 5.827-.01 7.568l30.56 17.68c3.071 1.777 6.954 1.775 10.022-.005L44.11 5.784c2.993-1.737 2.998-5.807.01-7.55z"
                        }),
                        /*#__PURE__*/ (0, $45uhR$reactjsxruntime.jsx)("path", {
                            fill: "url(#490f1f__f)",
                            d: "M-15.694 23.917c6.34 0 12.084 2.419 16.242 6.322 8.317 7.818 15.212 7.825 23.529.02 4.158-3.923 9.909-6.349 16.257-6.349 12.688.007 22.967 9.67 22.967 21.598 0 11.929-10.279 21.585-22.967 21.585-6.362 0-12.113-2.432-16.271-6.355-8.31-7.792-15.198-7.785-23.515.034-4.158 3.909-9.902 6.321-16.242 6.321-12.681 0-22.96-9.663-22.96-21.585 0-11.921 10.279-21.591 22.96-21.591"
                        })
                    ]
                })
            })
        ]
    });
const $394215a7886439e4$var$ForwardRef = /*#__PURE__*/ (0, $45uhR$react.forwardRef)($394215a7886439e4$var$SvgComponent);
var $394215a7886439e4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($394215a7886439e4$var$ForwardRef);
