// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_userGroup_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $7Bb9u$reactjsxruntime = require("react/jsx-runtime");
var $7Bb9u$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $8320e3ea90d690c6$export$2e2bcd8739ae039);




const $8320e3ea90d690c6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1bf28f__d",
                        x1: 64.673,
                        x2: 22.47,
                        y1: 10.009,
                        y2: 68.054,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1bf28f__e",
                        x1: 36.941,
                        x2: 44.765,
                        y1: -15.603,
                        y2: 37.515,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1bf28f__f",
                        x1: -64.382,
                        x2: 82.671,
                        y1: 90.905,
                        y2: 60.075,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsxs)("radialGradient", {
                        id: "1bf28f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 106.979 -113.484 0 77.863 61.104)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("clipPath", {
                        id: "1bf28f__b",
                        children: /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("clipPath", {
                id: "1bf28f__a",
                children: /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M70.111 30.27c0 8.119-6.178 14.7-13.8 14.7-7.621 0-13.8-6.581-13.8-14.7s6.179-14.7 13.8-14.7 13.8 6.582 13.8 14.7M43.556 44.448c-1.224.378-2.52.588-3.864.588-7.62 0-13.8-6.582-13.8-14.7s6.18-14.7 13.8-14.7c1.302 0 2.568.198 3.762.558-3.54 3.552-5.748 8.496-5.748 14.076s2.25 10.62 5.85 14.178M24.044 73.482c1.95-9.054 7.59-16.62 15.114-21.24-13.224.252-24.348 9.792-27.042 22.314-.648 3.012 1.602 5.868 4.686 5.868h8.364c-1.224-2.076-1.644-4.536-1.122-6.942M56.305 52.17c13.458 0 24.846 9.636 27.576 22.326.648 3.012-1.602 5.868-4.686 5.868h-45.78c-3.084 0-5.334-2.856-4.686-5.868 2.73-12.69 14.118-22.326 27.576-22.326"
                })
            }),
            /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("g", {
                clipPath: "url(#1bf28f__a)",
                children: /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#1bf28f__b)",
                    children: [
                        /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("ellipse", {
                            cx: 77.863,
                            cy: 61.104,
                            fill: "url(#1bf28f__c)",
                            rx: 113.484,
                            ry: 106.979
                        }),
                        /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("path", {
                            fill: "url(#1bf28f__d)",
                            d: "M.728 35.599c-1.262-25.61 19.74-47.336 46.907-48.525S97.849 5.682 99.11 31.293c1.262 25.61-19.74 47.335-46.907 48.524C25.035 81.007 1.99 61.21.728 35.6"
                        }),
                        /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("path", {
                            fill: "url(#1bf28f__e)",
                            d: "M27.658-38.88c-6.128-3.575-13.893-3.596-20.045-.056L-53.912-3.524c-6.026 3.468-6.036 11.655-.019 15.136L7.19 46.974c6.142 3.553 13.907 3.549 20.044-.013l60.993-35.395c5.985-3.474 5.996-11.614.018-15.101z"
                        }),
                        /*#__PURE__*/ (0, $7Bb9u$reactjsxruntime.jsx)("path", {
                            fill: "url(#1bf28f__f)",
                            d: "M-31.384 47.832c12.68 0 24.168 4.838 32.485 12.643 16.633 15.638 30.423 15.651 47.057.04 8.317-7.845 19.818-12.697 32.513-12.697 25.377.014 45.934 19.34 45.934 43.197s-20.557 43.17-45.934 43.17c-12.724 0-24.225-4.865-32.542-12.711-16.619-15.584-30.395-15.57-47.028.067-8.317 7.819-19.804 12.644-32.485 12.644-25.363 0-45.92-19.326-45.92-43.17s20.557-43.183 45.92-43.183"
                        })
                    ]
                })
            })
        ]
    });
const $8320e3ea90d690c6$var$ForwardRef = /*#__PURE__*/ (0, $7Bb9u$react.forwardRef)($8320e3ea90d690c6$var$SvgComponent);
var $8320e3ea90d690c6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($8320e3ea90d690c6$var$ForwardRef);
