// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_interaction_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $4uzBf$jsxs, jsx as $4uzBf$jsx} from "react/jsx-runtime";
import {forwardRef as $4uzBf$forwardRef} from "react";





const $37cdda5efc2429fd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4uzBf$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4uzBf$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4uzBf$jsxs)("linearGradient", {
                        id: "91a060__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4uzBf$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $4uzBf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4uzBf$jsxs)("linearGradient", {
                        id: "91a060__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4uzBf$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $4uzBf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4uzBf$jsxs)("linearGradient", {
                        id: "91a060__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4uzBf$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4uzBf$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $4uzBf$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $4uzBf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4uzBf$jsxs)("radialGradient", {
                        id: "91a060__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4uzBf$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4uzBf$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4uzBf$jsx)("clipPath", {
                        id: "91a060__b",
                        children: /*#__PURE__*/ (0, $4uzBf$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4uzBf$jsx)("clipPath", {
                id: "91a060__a",
                children: /*#__PURE__*/ (0, $4uzBf$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M92.339 56.39c0 5.56-1.39 10.796-3.84 15.38q-.87.348-1.724.719l-9.007-16.212-.065-.109c-3.593-6.011-12.126-10.917-20.581-6.334-7.981 4.327-10.383 14.665-5.988 22.098l9.506 17.11q-.481.014-.967.014c-18.04 0-32.665-14.625-32.665-32.666s14.625-32.665 32.665-32.665c18.041 0 32.666 14.624 32.666 32.665M69.042 87.692l13.932 25.078c.223.401-.164.86-.594.701-2.513-.93-5.323-2.141-8.243-3.399-4.77-2.055-9.832-4.237-14.368-5.527-4.63-1.316-9.044.846-10.216 5.109-1.15 4.18 0 7.662 5.108 10.216.37.186.85.43 1.427.725 7.38 3.766 30.725 15.681 46.425 15.681 16.928 0 30.476-13.722 30.476-30.65s-9.466-30.028-26.393-30.028c-6.461 0-17.3 3.823-22.528 7.343-.248.167-.587.09-.733-.17L70.837 60.272q-.11-.184-.23-.364-.09-.138-.187-.273c-2.203-3.084-6.12-4.593-9.485-2.769-3.829 2.076-5.131 7.346-2.87 11.068z"
                })
            }),
            /*#__PURE__*/ (0, $4uzBf$jsx)("g", {
                clipPath: "url(#91a060__a)",
                children: /*#__PURE__*/ (0, $4uzBf$jsxs)("g", {
                    clipPath: "url(#91a060__b)",
                    children: [
                        /*#__PURE__*/ (0, $4uzBf$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $4uzBf$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#91a060__c)"
                        }),
                        /*#__PURE__*/ (0, $4uzBf$jsx)("path", {
                            fill: "url(#91a060__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $4uzBf$jsx)("path", {
                            fill: "url(#91a060__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $4uzBf$jsx)("path", {
                            fill: "url(#91a060__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $37cdda5efc2429fd$var$ForwardRef = /*#__PURE__*/ (0, $4uzBf$forwardRef)($37cdda5efc2429fd$var$SvgComponent);
var $37cdda5efc2429fd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($37cdda5efc2429fd$var$ForwardRef);


export {$37cdda5efc2429fd$export$2e2bcd8739ae039 as default};
