// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_shoppingCart_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $55N9h$jsxs, jsx as $55N9h$jsx} from "react/jsx-runtime";
import {forwardRef as $55N9h$forwardRef} from "react";





const $b460da4ed553067e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $55N9h$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $55N9h$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $55N9h$jsxs)("linearGradient", {
                        id: "2e00d3__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $55N9h$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $55N9h$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $55N9h$jsxs)("linearGradient", {
                        id: "2e00d3__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $55N9h$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $55N9h$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $55N9h$jsxs)("linearGradient", {
                        id: "2e00d3__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $55N9h$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $55N9h$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $55N9h$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $55N9h$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $55N9h$jsxs)("radialGradient", {
                        id: "2e00d3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $55N9h$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $55N9h$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $55N9h$jsx)("clipPath", {
                        id: "2e00d3__b",
                        children: /*#__PURE__*/ (0, $55N9h$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $55N9h$jsx)("clipPath", {
                id: "2e00d3__a",
                children: /*#__PURE__*/ (0, $55N9h$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M6.746 7.826c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25l.36.002.62.004c.525.003 1.229.006 2.057.005.873 0 1.612.628 1.752 1.476l2.945 17.86c.339 2.054 2.113 3.56 4.194 3.56h16.169c.69 0 1.25-.56 1.25-1.25 0-.691-.56-1.25-1.25-1.25h-16.17c-.856 0-1.587-.62-1.726-1.466l-.58-3.516H33.59c1.553 0 2.888-1.098 3.189-2.62l1.235-6.262c.395-2.002-1.132-3.873-3.178-3.878-7.024-.017-17.547-.034-20.572-.005l-.261-1.58c-.341-2.068-2.136-3.57-4.22-3.569-.822 0-1.52-.002-2.042-.005l-.6-.004zM20.505 39c0 1.38-1.12 2.5-2.5 2.5s-2.5-1.12-2.5-2.5 1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5m12.462 2.5c1.38 0 2.5-1.12 2.5-2.5s-1.12-2.5-2.5-2.5-2.5 1.12-2.5 2.5 1.12 2.5 2.5 2.5"
                })
            }),
            /*#__PURE__*/ (0, $55N9h$jsx)("g", {
                clipPath: "url(#2e00d3__a)",
                children: /*#__PURE__*/ (0, $55N9h$jsxs)("g", {
                    clipPath: "url(#2e00d3__b)",
                    children: [
                        /*#__PURE__*/ (0, $55N9h$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $55N9h$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#2e00d3__c)"
                        }),
                        /*#__PURE__*/ (0, $55N9h$jsx)("path", {
                            fill: "url(#2e00d3__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $55N9h$jsx)("path", {
                            fill: "url(#2e00d3__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $55N9h$jsx)("path", {
                            fill: "url(#2e00d3__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $b460da4ed553067e$var$ForwardRef = /*#__PURE__*/ (0, $55N9h$forwardRef)($b460da4ed553067e$var$SvgComponent);
var $b460da4ed553067e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b460da4ed553067e$var$ForwardRef);


export {$b460da4ed553067e$export$2e2bcd8739ae039 as default};
