// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fireworks_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $euYe7$jsxs, jsx as $euYe7$jsx} from "react/jsx-runtime";
import {forwardRef as $euYe7$forwardRef} from "react";





const $5dea715cf5f6ddc0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $euYe7$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $euYe7$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $euYe7$jsxs)("linearGradient", {
                        id: "5b7c5b__d",
                        x1: 64.673,
                        x2: 22.47,
                        y1: 10.009,
                        y2: 68.054,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $euYe7$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $euYe7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $euYe7$jsxs)("linearGradient", {
                        id: "5b7c5b__e",
                        x1: 36.941,
                        x2: 44.765,
                        y1: -15.603,
                        y2: 37.515,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $euYe7$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $euYe7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $euYe7$jsxs)("linearGradient", {
                        id: "5b7c5b__f",
                        x1: -64.382,
                        x2: 82.671,
                        y1: 90.905,
                        y2: 60.075,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $euYe7$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $euYe7$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $euYe7$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $euYe7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $euYe7$jsxs)("radialGradient", {
                        id: "5b7c5b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 106.979 -113.484 0 77.863 61.104)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $euYe7$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $euYe7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $euYe7$jsx)("clipPath", {
                        id: "5b7c5b__b",
                        children: /*#__PURE__*/ (0, $euYe7$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $euYe7$jsx)("clipPath", {
                id: "5b7c5b__a",
                children: /*#__PURE__*/ (0, $euYe7$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M76.868 16.946c.02-1.325-1.036-2.417-2.362-2.438-1.325-.021-2.417 1.036-2.438 2.361l-.033 2.1-2.103-.033c-1.325-.02-2.416 1.038-2.436 2.363s1.037 2.416 2.363 2.436l2.099.033-.034 2.101c-.021 1.326 1.036 2.417 2.361 2.439 1.326.02 2.417-1.037 2.439-2.362l.033-2.105 2.102.032c1.325.02 2.416-1.037 2.436-2.362s-1.038-2.417-2.363-2.437l-2.098-.032zm-36.674 1.528c0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4v6.084c0 1.325 1.075 2.4 2.4 2.4 1.326 0 2.4-1.075 2.4-2.4zm-15.016 1.694c1.073-.779 2.573-.54 3.352.532l3.576 4.926c.779 1.073.54 2.573-.532 3.352s-2.574.54-3.352-.532l-3.576-4.926c-.779-1.073-.54-2.574.532-3.352m-4.78 9.2c-1.26-.41-2.614.28-3.024 1.54s.281 2.615 1.542 3.025L24.7 35.81c1.26.409 2.614-.281 3.024-1.542.409-1.26-.281-2.614-1.542-3.024zm7.326 11.447c.409 1.261-.282 2.615-1.543 3.024l-5.79 1.878c-1.26.409-2.614-.282-3.023-1.543s.282-2.614 1.542-3.023l5.79-1.878c1.261-.409 2.615.282 3.024 1.542m4.375 8.65c.78-1.072.542-2.573-.53-3.353-1.072-.779-2.573-.541-3.352.53l-3.576 4.92c-.78 1.073-.542 2.574.53 3.353s2.573.542 3.352-.53zm8.09 1.067c0-1.326-1.075-2.4-2.4-2.4-1.326 0-2.4 1.074-2.4 2.4v6.084c0 1.325 1.074 2.4 2.4 2.4s2.4-1.075 2.4-2.4zM50.42 20.175c1.073.779 1.31 2.28.53 3.352l-3.575 4.92c-.78 1.072-2.28 1.31-3.353.53-1.072-.779-1.31-2.28-.53-3.352l3.576-4.92c.78-1.072 2.28-1.31 3.352-.53m6.253 13.764c1.26-.41 1.95-1.764 1.541-3.024s-1.763-1.951-3.023-1.542l-5.784 1.878c-1.261.41-1.951 1.763-1.542 3.024.41 1.26 1.763 1.95 3.024 1.542zm-8.808 6.882c.41-1.261 1.763-1.951 3.024-1.542l5.784 1.878c1.26.41 1.95 1.763 1.541 3.024-.409 1.26-1.763 1.95-3.023 1.542l-5.784-1.878c-1.261-.41-1.951-1.764-1.542-3.024m-.495 5.823c-.779-1.073-2.28-1.311-3.352-.532s-1.311 2.28-.533 3.352l3.576 4.926c.779 1.072 2.28 1.31 3.352.532 1.073-.779 1.311-2.28.533-3.352zm18.051 22.332c1.326.005 2.396 1.084 2.39 2.41l-.012 2.97c-.005 1.325-1.084 2.395-2.41 2.39-1.325-.005-2.395-1.084-2.39-2.41l.012-2.97c.006-1.325 1.085-2.395 2.41-2.39m-5.364.71c1.15-.66 1.55-2.125.892-3.276-.659-1.15-2.125-1.55-3.275-.891l-2.58 1.476c-1.151.658-1.55 2.124-.892 3.275s2.125 1.55 3.275.891zm-5.84-12.345c.666-1.146 2.136-1.534 3.281-.868l2.568 1.494c1.146.667 1.535 2.136.868 3.282s-2.136 1.534-3.281.867l-2.568-1.494c-1.146-.666-1.535-2.135-.868-3.281m13.594-4.045c.006-1.326-1.064-2.405-2.39-2.41-1.325-.005-2.404 1.065-2.41 2.39L63 56.246c-.005 1.326 1.065 2.404 2.39 2.41 1.326.005 2.405-1.065 2.41-2.39zm8.805 4.066c.658 1.15.259 2.617-.892 3.275l-2.58 1.476c-1.15.658-2.617.26-3.275-.891-.658-1.15-.259-2.617.892-3.275l2.58-1.476c1.15-.658 2.617-.26 3.275.891m-3.45 8.153c-1.146-.666-2.616-.278-3.282.868-.667 1.146-.278 2.615.867 3.281l2.569 1.494c1.145.667 2.614.278 3.28-.867.667-1.146.279-2.615-.867-3.282zM14.09 74.693c.02-1.325 1.111-2.383 2.437-2.363l2.097.032.033-2.103c.02-1.325 1.11-2.383 2.436-2.363s2.383 1.111 2.363 2.437l-.032 2.103 2.103.032c1.325.02 2.383 1.111 2.363 2.437s-1.112 2.383-2.437 2.363l-2.103-.033-.032 2.098c-.02 1.325-1.111 2.383-2.436 2.363-1.326-.02-2.384-1.112-2.363-2.437l.032-2.097-2.098-.032c-1.325-.02-2.383-1.112-2.363-2.437"
                })
            }),
            /*#__PURE__*/ (0, $euYe7$jsx)("g", {
                clipPath: "url(#5b7c5b__a)",
                children: /*#__PURE__*/ (0, $euYe7$jsxs)("g", {
                    clipPath: "url(#5b7c5b__b)",
                    children: [
                        /*#__PURE__*/ (0, $euYe7$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $euYe7$jsx)("ellipse", {
                            cx: 77.863,
                            cy: 61.104,
                            fill: "url(#5b7c5b__c)",
                            rx: 113.484,
                            ry: 106.979
                        }),
                        /*#__PURE__*/ (0, $euYe7$jsx)("path", {
                            fill: "url(#5b7c5b__d)",
                            d: "M.728 35.599c-1.262-25.61 19.74-47.336 46.907-48.525S97.849 5.682 99.11 31.293c1.262 25.61-19.74 47.335-46.907 48.524C25.035 81.007 1.99 61.21.728 35.6"
                        }),
                        /*#__PURE__*/ (0, $euYe7$jsx)("path", {
                            fill: "url(#5b7c5b__e)",
                            d: "M27.658-38.88c-6.128-3.575-13.893-3.596-20.045-.056L-53.912-3.524c-6.026 3.468-6.036 11.655-.019 15.136L7.19 46.974c6.142 3.553 13.907 3.549 20.044-.013l60.993-35.395c5.985-3.474 5.996-11.614.018-15.101z"
                        }),
                        /*#__PURE__*/ (0, $euYe7$jsx)("path", {
                            fill: "url(#5b7c5b__f)",
                            d: "M-31.384 47.832c12.68 0 24.168 4.838 32.485 12.643 16.633 15.638 30.423 15.651 47.057.04 8.317-7.845 19.818-12.697 32.513-12.697 25.377.014 45.934 19.34 45.934 43.197s-20.557 43.17-45.934 43.17c-12.724 0-24.225-4.865-32.542-12.711-16.619-15.584-30.395-15.57-47.028.067-8.317 7.819-19.804 12.644-32.485 12.644-25.363 0-45.92-19.326-45.92-43.17s20.557-43.183 45.92-43.183"
                        })
                    ]
                })
            })
        ]
    });
const $5dea715cf5f6ddc0$var$ForwardRef = /*#__PURE__*/ (0, $euYe7$forwardRef)($5dea715cf5f6ddc0$var$SvgComponent);
var $5dea715cf5f6ddc0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5dea715cf5f6ddc0$var$ForwardRef);


export {$5dea715cf5f6ddc0$export$2e2bcd8739ae039 as default};
