// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_volumeOne_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $9n94R$jsxs, jsx as $9n94R$jsx} from "react/jsx-runtime";
import {forwardRef as $9n94R$forwardRef} from "react";





const $f50349b9a293197e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9n94R$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9n94R$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9n94R$jsxs)("linearGradient", {
                        id: "21b252__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9n94R$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $9n94R$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9n94R$jsxs)("linearGradient", {
                        id: "21b252__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9n94R$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $9n94R$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9n94R$jsxs)("linearGradient", {
                        id: "21b252__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9n94R$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9n94R$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $9n94R$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $9n94R$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9n94R$jsxs)("radialGradient", {
                        id: "21b252__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9n94R$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9n94R$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9n94R$jsx)("clipPath", {
                        id: "21b252__b",
                        children: /*#__PURE__*/ (0, $9n94R$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9n94R$jsx)("clipPath", {
                id: "21b252__a",
                children: /*#__PURE__*/ (0, $9n94R$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M46.629 52.997h12.65v54.205h-12.65c-4.418 0-8-3.582-8-8V60.997c0-4.418 3.582-8 8-8m20.65 59.023V48.13l15.24-15.11c3.787-3.754 10.224-1.072 10.224 4.261v85.437c0 5.318-6.407 8.007-10.203 4.282zm42.384-55.58c-1.766-1.328-4.273-.974-5.602.792-1.328 1.765-.974 4.273.791 5.6 5.183 3.9 8.521 10.09 8.521 17.061 0 7.028-3.393 13.261-8.648 17.155-1.775 1.316-2.148 3.821-.832 5.596 1.315 1.775 3.82 2.147 5.595.832 7.203-5.338 11.885-13.914 11.885-23.583 0-9.59-4.606-18.106-11.71-23.452"
                })
            }),
            /*#__PURE__*/ (0, $9n94R$jsx)("g", {
                clipPath: "url(#21b252__a)",
                children: /*#__PURE__*/ (0, $9n94R$jsxs)("g", {
                    clipPath: "url(#21b252__b)",
                    children: [
                        /*#__PURE__*/ (0, $9n94R$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $9n94R$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#21b252__c)"
                        }),
                        /*#__PURE__*/ (0, $9n94R$jsx)("path", {
                            fill: "url(#21b252__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $9n94R$jsx)("path", {
                            fill: "url(#21b252__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $9n94R$jsx)("path", {
                            fill: "url(#21b252__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $f50349b9a293197e$var$ForwardRef = /*#__PURE__*/ (0, $9n94R$forwardRef)($f50349b9a293197e$var$SvgComponent);
var $f50349b9a293197e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f50349b9a293197e$var$ForwardRef);


export {$f50349b9a293197e$export$2e2bcd8739ae039 as default};
