// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_rotateCCW_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $itL3k$jsxs, jsx as $itL3k$jsx} from "react/jsx-runtime";
import {forwardRef as $itL3k$forwardRef} from "react";





const $668c76556c9271e9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $itL3k$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $itL3k$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $itL3k$jsxs)("linearGradient", {
                        id: "518361__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itL3k$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $itL3k$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itL3k$jsxs)("linearGradient", {
                        id: "518361__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itL3k$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $itL3k$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itL3k$jsxs)("linearGradient", {
                        id: "518361__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itL3k$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $itL3k$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $itL3k$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $itL3k$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itL3k$jsxs)("radialGradient", {
                        id: "518361__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $itL3k$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $itL3k$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $itL3k$jsx)("clipPath", {
                        id: "518361__b",
                        children: /*#__PURE__*/ (0, $itL3k$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $itL3k$jsx)("clipPath", {
                id: "518361__a",
                children: /*#__PURE__*/ (0, $itL3k$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M39.25 24c0-8.422-6.828-15.25-15.25-15.25-6.026 0-11.238 3.495-13.714 8.573l4.671-1.472c.659-.207 1.36.159 1.568.817.207.659-.158 1.36-.817 1.568L8.266 20.58c-.658.208-1.36-.158-1.568-.816L4.355 12.32c-.207-.659.158-1.36.817-1.568s1.36.158 1.568.817l1.401 4.45c2.92-5.793 8.924-9.77 15.86-9.77 9.802 0 17.75 7.947 17.75 17.75S33.802 41.75 24 41.75c-7.668 0-14.197-4.86-16.68-11.664-.236-.649.097-1.366.746-1.603.648-.237 1.366.097 1.603.746C11.804 35.078 17.417 39.25 24 39.25c8.423 0 15.25-6.828 15.25-15.25"
                })
            }),
            /*#__PURE__*/ (0, $itL3k$jsx)("g", {
                clipPath: "url(#518361__a)",
                children: /*#__PURE__*/ (0, $itL3k$jsxs)("g", {
                    clipPath: "url(#518361__b)",
                    children: [
                        /*#__PURE__*/ (0, $itL3k$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $itL3k$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#518361__c)"
                        }),
                        /*#__PURE__*/ (0, $itL3k$jsx)("path", {
                            fill: "url(#518361__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $itL3k$jsx)("path", {
                            fill: "url(#518361__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $itL3k$jsx)("path", {
                            fill: "url(#518361__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $668c76556c9271e9$var$ForwardRef = /*#__PURE__*/ (0, $itL3k$forwardRef)($668c76556c9271e9$var$SvgComponent);
var $668c76556c9271e9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($668c76556c9271e9$var$ForwardRef);


export {$668c76556c9271e9$export$2e2bcd8739ae039 as default};
