// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_browser_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $eNklW$jsxs, jsx as $eNklW$jsx} from "react/jsx-runtime";
import {forwardRef as $eNklW$forwardRef} from "react";





const $dba940f7f8689010$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eNklW$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $eNklW$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $eNklW$jsxs)("linearGradient", {
                        id: "e07e3d__d",
                        x1: 32.783,
                        x2: 12.551,
                        y1: 5.818,
                        y2: 33.644,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eNklW$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $eNklW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eNklW$jsxs)("linearGradient", {
                        id: "e07e3d__e",
                        x1: 19.489,
                        x2: 23.239,
                        y1: -6.46,
                        y2: 19.004,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eNklW$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $eNklW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eNklW$jsxs)("linearGradient", {
                        id: "e07e3d__f",
                        x1: -29.084,
                        x2: 41.411,
                        y1: 44.599,
                        y2: 29.819,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eNklW$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $eNklW$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $eNklW$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $eNklW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eNklW$jsxs)("radialGradient", {
                        id: "e07e3d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.2844 -54.4031 0 39.106 30.312)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eNklW$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $eNklW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eNklW$jsx)("clipPath", {
                        id: "e07e3d__b",
                        children: /*#__PURE__*/ (0, $eNklW$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $eNklW$jsx)("clipPath", {
                id: "e07e3d__a",
                children: /*#__PURE__*/ (0, $eNklW$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M10.8 8.4h26.4c2.652 0 4.8 2.148 4.8 4.8v3.6H6v-3.6c0-2.652 2.148-4.8 4.8-4.8m-1.194 4.8c0 .663.537 1.2 1.2 1.2s1.2-.537 1.2-1.2-.537-1.2-1.2-1.2h-.003c-.66 0-1.197.537-1.197 1.2M10.8 39.6c-2.652 0-4.8-2.148-4.8-4.8V19.2h36v15.6c0 2.652-2.148 4.8-4.8 4.8z"
                })
            }),
            /*#__PURE__*/ (0, $eNklW$jsx)("g", {
                clipPath: "url(#e07e3d__a)",
                children: /*#__PURE__*/ (0, $eNklW$jsxs)("g", {
                    clipPath: "url(#e07e3d__b)",
                    children: [
                        /*#__PURE__*/ (0, $eNklW$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $eNklW$jsx)("ellipse", {
                            cx: 39.106,
                            cy: 30.312,
                            fill: "url(#e07e3d__c)",
                            rx: 54.403,
                            ry: 51.284
                        }),
                        /*#__PURE__*/ (0, $eNklW$jsx)("path", {
                            fill: "url(#e07e3d__d)",
                            d: "M2.128 18.086C1.523 5.807 11.591-4.608 24.615-5.178s24.072 8.92 24.677 21.198c.605 12.277-9.463 22.692-22.487 23.262S2.733 30.363 2.128 18.086"
                        }),
                        /*#__PURE__*/ (0, $eNklW$jsx)("path", {
                            fill: "url(#e07e3d__e)",
                            d: "M15.039-17.618c-2.938-1.714-6.66-1.725-9.61-.027L-24.064-.67c-2.889 1.662-2.893 5.587-.009 7.256l29.3 16.952c2.944 1.703 6.667 1.7 9.609-.007l29.24-16.967c2.869-1.666 2.874-5.568.008-7.24z"
                        }),
                        /*#__PURE__*/ (0, $eNklW$jsx)("path", {
                            fill: "url(#e07e3d__f)",
                            d: "M-13.266 23.95c6.08 0 11.586 2.32 15.573 6.06 7.974 7.497 14.585 7.504 22.559.02 3.987-3.76 9.5-6.087 15.586-6.087 12.166.007 22.02 9.272 22.02 20.708 0 11.437-9.854 20.695-22.02 20.695-6.1 0-11.613-2.332-15.6-6.093-7.967-7.47-14.57-7.464-22.545.032-3.987 3.748-9.494 6.061-15.573 6.061-12.158 0-22.013-9.264-22.013-20.695 0-11.43 9.855-20.701 22.013-20.701"
                        })
                    ]
                })
            })
        ]
    });
const $dba940f7f8689010$var$ForwardRef = /*#__PURE__*/ (0, $eNklW$forwardRef)($dba940f7f8689010$var$SvgComponent);
var $dba940f7f8689010$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($dba940f7f8689010$var$ForwardRef);


export {$dba940f7f8689010$export$2e2bcd8739ae039 as default};
