// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lightbulbRays_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3oBWG$reactjsxruntime = require("react/jsx-runtime");
var $3oBWG$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e3c84856ef476b8b$export$2e2bcd8739ae039);




const $e3c84856ef476b8b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ca6a9d__d",
                        x1: 64.673,
                        x2: 22.47,
                        y1: 10.009,
                        y2: 68.054,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ca6a9d__e",
                        x1: 36.941,
                        x2: 44.765,
                        y1: -15.603,
                        y2: 37.515,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ca6a9d__f",
                        x1: -64.382,
                        x2: 82.671,
                        y1: 90.905,
                        y2: 60.075,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ca6a9d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 106.979 -113.484 0 77.863 61.104)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("clipPath", {
                        id: "ca6a9d__b",
                        children: /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("clipPath", {
                id: "ca6a9d__a",
                children: /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M50.4 17.969c0 1.326-1.073 2.4-2.4 2.4-1.325 0-2.4-1.074-2.4-2.4v-3.612c0-1.326 1.075-2.4 2.4-2.4 1.327 0 2.4 1.074 2.4 2.4zm-3.719 8.238c-8.91.636-16.122 7.902-16.782 16.896-.486 6.684 2.568 12.672 7.47 16.254 1.59 1.164 2.562 3.006 2.562 4.992v4.05h15.918c.043 0 .086.007.13.013q.043.007.086.011v-4.068c0-1.938.888-3.786 2.448-4.914 4.62-3.324 7.632-8.778 7.632-14.946 0-10.572-8.85-19.05-19.47-18.294zm11.622 18.54c-1.326 0-2.4-1.074-2.4-2.4 0-3.006-2.664-5.448-5.94-5.448-1.326 0-2.4-1.074-2.4-2.4s1.074-2.4 2.4-2.4c5.922 0 10.74 4.596 10.74 10.248 0 1.326-1.074 2.4-2.4 2.4M39.937 73.199v2.88c0 1.686 1.356 3.054 3.024 3.054h10.086c1.668 0 3.024-1.368 3.024-3.054v-2.904q-.043.004-.086.011-.066.012-.13.013zM17.97 41.987h3.612c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4H17.97c-1.326 0-2.4-1.074-2.4-2.4s1.074-2.4 2.4-2.4m60.055 0h-3.613c-1.326 0-2.4 1.074-2.4 2.4s1.075 2.4 2.4 2.4h3.612c1.326 0 2.4-1.074 2.4-2.4s-1.074-2.4-2.4-2.4M67.537 21.455l-2.55 2.55c-.936.936-.936 2.454 0 3.396.468.468 1.08.702 1.698.702s1.23-.234 1.698-.702l2.55-2.55c.936-.936.936-2.454 0-3.396-.936-.936-2.454-.936-3.396 0m-39.067.006 2.55 2.55c.936.942.936 2.46 0 3.396-.468.468-1.08.702-1.698.702s-1.23-.234-1.698-.702l-2.55-2.55c-.936-.942-.936-2.46 0-3.396.942-.936 2.46-.936 3.396 0"
                })
            }),
            /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ca6a9d__a)",
                children: /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ca6a9d__b)",
                    children: [
                        /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("ellipse", {
                            cx: 77.863,
                            cy: 61.104,
                            fill: "url(#ca6a9d__c)",
                            rx: 113.484,
                            ry: 106.979
                        }),
                        /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("path", {
                            fill: "url(#ca6a9d__d)",
                            d: "M.728 35.599c-1.262-25.61 19.74-47.336 46.907-48.525S97.849 5.682 99.11 31.293c1.262 25.61-19.74 47.335-46.907 48.524C25.035 81.007 1.99 61.21.728 35.6"
                        }),
                        /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("path", {
                            fill: "url(#ca6a9d__e)",
                            d: "M27.658-38.88c-6.128-3.575-13.893-3.596-20.045-.056L-53.912-3.524c-6.026 3.468-6.036 11.655-.019 15.136L7.19 46.974c6.142 3.553 13.907 3.549 20.044-.013l60.993-35.395c5.985-3.474 5.996-11.614.018-15.101z"
                        }),
                        /*#__PURE__*/ (0, $3oBWG$reactjsxruntime.jsx)("path", {
                            fill: "url(#ca6a9d__f)",
                            d: "M-31.384 47.832c12.68 0 24.168 4.838 32.485 12.643 16.633 15.638 30.423 15.651 47.057.04 8.317-7.845 19.818-12.697 32.513-12.697 25.377.014 45.934 19.34 45.934 43.197s-20.557 43.17-45.934 43.17c-12.724 0-24.225-4.865-32.542-12.711-16.619-15.584-30.395-15.57-47.028.067-8.317 7.819-19.804 12.644-32.485 12.644-25.363 0-45.92-19.326-45.92-43.17s20.557-43.183 45.92-43.183"
                        })
                    ]
                })
            })
        ]
    });
const $e3c84856ef476b8b$var$ForwardRef = /*#__PURE__*/ (0, $3oBWG$react.forwardRef)($e3c84856ef476b8b$var$SvgComponent);
var $e3c84856ef476b8b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e3c84856ef476b8b$var$ForwardRef);
