// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_desktop_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3rWGt$reactjsxruntime = require("react/jsx-runtime");
var $3rWGt$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3df8a29136fd909c$export$2e2bcd8739ae039);




const $3df8a29136fd909c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4c7481__d",
                        x1: 65.703,
                        x2: 23.805,
                        y1: 9.034,
                        y2: 66.661,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4c7481__e",
                        x1: 38.171,
                        x2: 45.938,
                        y1: -16.392,
                        y2: 36.343,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4c7481__f",
                        x1: -62.418,
                        x2: 83.573,
                        y1: 89.347,
                        y2: 58.739,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4c7481__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 106.207 -112.666 0 78.799 59.76)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("clipPath", {
                        id: "4c7481__b",
                        children: /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("rect", {
                            width: 78.033,
                            height: 73.56,
                            x: 9.242,
                            y: 11.4,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("clipPath", {
                id: "4c7481__a",
                children: /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M76.22 16.8H19.784c-5.304 0-9.6 4.296-9.6 9.6v26.4c0 5.304 4.296 9.6 9.6 9.6h21.618v14.4h-6.006c-1.326 0-2.4 1.074-2.4 2.4s1.074 2.4 2.4 2.4h25.212c1.326 0 2.4-1.074 2.4-2.4s-1.074-2.4-2.4-2.4h-6.006V62.4H76.22c5.304 0 9.6-4.296 9.6-9.6V26.4c0-5.304-4.296-9.6-9.6-9.6m-28.206 12c-1.326 0-2.4-1.074-2.4-2.4s1.074-2.4 2.394-2.4h.006c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4"
                })
            }),
            /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("g", {
                clipPath: "url(#4c7481__a)",
                children: /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#4c7481__b)",
                    children: [
                        /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("rect", {
                            width: 78.033,
                            height: 73.56,
                            x: 9.242,
                            y: 11.4,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("ellipse", {
                            cx: 78.799,
                            cy: 59.76,
                            fill: "url(#4c7481__c)",
                            rx: 112.666,
                            ry: 106.207
                        }),
                        /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("path", {
                            fill: "url(#4c7481__d)",
                            d: "M2.22 34.44C.966 9.014 21.816-12.555 48.787-13.735c26.972-1.18 49.852 18.474 51.105 43.9 1.252 25.425-19.598 46.993-46.57 48.174-26.971 1.18-49.851-18.474-51.104-43.9"
                        }),
                        /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("path", {
                            fill: "url(#4c7481__e)",
                            d: "M28.955-39.5c-6.084-3.55-13.793-3.572-19.9-.057L-52.027-4.4c-5.982 3.443-5.992 11.57-.018 15.027L8.633 45.733c6.098 3.528 13.807 3.524 19.9-.012l60.553-35.14c5.942-3.449 5.953-11.53.018-14.992z"
                        }),
                        /*#__PURE__*/ (0, $3rWGt$reactjsxruntime.jsx)("path", {
                            fill: "url(#4c7481__f)",
                            d: "M-29.659 46.585c12.59 0 23.994 4.803 32.251 12.552 16.514 15.525 30.204 15.538 46.718.04 8.256-7.79 19.675-12.606 32.279-12.606 25.193.014 45.602 19.2 45.602 42.885s-20.409 42.859-45.602 42.859c-12.632 0-24.05-4.83-32.307-12.619-16.5-15.472-30.176-15.458-46.69.066-8.257 7.763-19.66 12.553-32.25 12.553-25.18 0-45.59-19.187-45.59-42.859 0-23.671 20.41-42.871 45.59-42.871"
                        })
                    ]
                })
            })
        ]
    });
const $3df8a29136fd909c$var$ForwardRef = /*#__PURE__*/ (0, $3rWGt$react.forwardRef)($3df8a29136fd909c$var$SvgComponent);
var $3df8a29136fd909c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3df8a29136fd909c$var$ForwardRef);
