// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_AIGenerate_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8MwHb$reactjsxruntime = require("react/jsx-runtime");
var $8MwHb$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $2f5c5d2ede9da698$export$2e2bcd8739ae039);




const $2f5c5d2ede9da698$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d4bd07__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d4bd07__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d4bd07__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d4bd07__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("clipPath", {
                        id: "d4bd07__b",
                        children: /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("clipPath", {
                id: "d4bd07__a",
                children: /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m126.787 55.018 9.316 2.186c4.137.97 6.999-4.204 4.126-7.455l-6.47-7.326c-.961-1.088-1.35-2.604-1.037-4.047l2.096-9.72c.929-4.318-4.029-7.304-7.145-4.306l-7.021 6.75c-1.043 1.004-2.495 1.41-3.878 1.083l-9.316-2.187c-4.137-.969-6.999 4.204-4.126 7.456l6.47 7.326c.374.423.661.911.854 1.434q.176.478.246.985c.073.535.054 1.086-.063 1.627l-2.096 9.721c-.116.54-.14 1.059-.087 1.548q.06.551.242 1.046c.997 2.744 4.556 4.054 6.99 1.712l7.021-6.751c1.042-1.003 2.495-1.409 3.878-1.082m-32.4-18.239h-44.43c-8.836 0-16 7.164-16 16v60.087c0 8.837 7.163 16 16 16h60.087c8.837 0 16-7.163 16-16V64.184l-4.609 4.432-.002.001c-4.438 4.269-10.487 4.253-14.763 1.676-4.221-2.543-6.984-7.687-5.748-13.43l1.742-8.082-5.327-6.032-.002-.002c-1.616-1.83-2.557-3.888-2.948-5.968m2.75 49.805-6.848-1.543c-.907-.203-1.847.05-2.529.675l-5.158 4.759c-2.032 1.875-5.265.005-4.657-2.686l1.543-6.848c.203-.906-.05-1.847-.675-2.528l-4.759-5.158c-1.875-2.033-.005-5.265 2.686-4.657l6.848 1.543c.907.203 1.847-.05 2.528-.676l5.158-4.758c2.033-1.875 5.265-.006 4.657 2.686l-1.543 6.847c-.203.907.05 1.847.676 2.529l4.758 5.158c1.875 2.032.006 5.265-2.686 4.656"
                })
            }),
            /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("g", {
                clipPath: "url(#d4bd07__a)",
                children: /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#d4bd07__b)",
                    children: [
                        /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#d4bd07__c)"
                        }),
                        /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("path", {
                            fill: "url(#d4bd07__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("path", {
                            fill: "url(#d4bd07__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $8MwHb$reactjsxruntime.jsx)("path", {
                            fill: "url(#d4bd07__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $2f5c5d2ede9da698$var$ForwardRef = /*#__PURE__*/ (0, $8MwHb$react.forwardRef)($2f5c5d2ede9da698$var$SvgComponent);
var $2f5c5d2ede9da698$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($2f5c5d2ede9da698$var$ForwardRef);
