// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_piggyBank_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $gyl6d$jsxs, jsx as $gyl6d$jsx} from "react/jsx-runtime";
import {forwardRef as $gyl6d$forwardRef} from "react";





const $5f927347cd9fe340$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gyl6d$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gyl6d$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gyl6d$jsxs)("linearGradient", {
                        id: "034067__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gyl6d$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $gyl6d$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gyl6d$jsxs)("linearGradient", {
                        id: "034067__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gyl6d$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $gyl6d$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gyl6d$jsxs)("linearGradient", {
                        id: "034067__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gyl6d$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gyl6d$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $gyl6d$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $gyl6d$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gyl6d$jsxs)("radialGradient", {
                        id: "034067__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gyl6d$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gyl6d$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gyl6d$jsx)("clipPath", {
                        id: "034067__b",
                        children: /*#__PURE__*/ (0, $gyl6d$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gyl6d$jsx)("clipPath", {
                id: "034067__a",
                children: /*#__PURE__*/ (0, $gyl6d$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M110.974 119.029c16.846-8.923 28.046-24.74 28.046-42.766q-.001-2.502-.283-4.942c2.257-.307 4.84-1.238 7.141-3.162 3.303-2.76 5.609-7.21 5.9-13.618.1-2.207-1.608-4.077-3.815-4.177s-4.077 1.607-4.177 3.814c-.207 4.568-1.742 6.76-3.038 7.843-1.29 1.078-2.76 1.408-3.676 1.419-6.722-21.654-30.063-37.63-57.825-37.63-11.412 0-22.077 2.7-31.148 7.384-5.187-5.565-13.203-7.325-19.631-7.379-2.784-.023-4.234 2.99-2.84 5.399l7.494 12.942.057-.042c-4.113 4.195-7.438 8.96-9.782 14.135v-.03l-11.205 6.53c-2.46 1.432-3.973 4.065-3.973 6.912v9.205c0 2.848 1.514 5.482 3.976 6.914l11.202 6.52v-.023c4.654 10.272 13.17 18.932 24.05 24.713q-.044-.006-.073-.005l4.14 11.273c.868 2.362 3.116 3.931 5.633 3.931h4.919c2.514 0 4.761-1.567 5.63-3.926l1.554-4.22-.155-.053c3.3.477 6.692.725 10.152.725 3.385 0 6.704-.238 9.935-.694l-.068.022 1.554 4.22c.87 2.359 3.117 3.926 5.63 3.926h4.92c2.516 0 4.765-1.569 5.632-3.931zm-22.969-75.02c-5.42-1.208-9.917-1.259-15.653.002-2.157.474-4.29-.89-4.765-3.049-.474-2.157.89-4.29 3.048-4.765 6.885-1.513 12.562-1.456 19.11.002 2.155.48 3.514 2.618 3.034 4.774s-2.617 3.515-4.774 3.035m-38.038 13.83c0 3.315-2.686 6-6 6s-6-2.685-6-6c0-3.313 2.686-6 6-6s6 2.687 6 6"
                })
            }),
            /*#__PURE__*/ (0, $gyl6d$jsx)("g", {
                clipPath: "url(#034067__a)",
                children: /*#__PURE__*/ (0, $gyl6d$jsxs)("g", {
                    clipPath: "url(#034067__b)",
                    children: [
                        /*#__PURE__*/ (0, $gyl6d$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $gyl6d$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#034067__c)"
                        }),
                        /*#__PURE__*/ (0, $gyl6d$jsx)("path", {
                            fill: "url(#034067__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $gyl6d$jsx)("path", {
                            fill: "url(#034067__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $gyl6d$jsx)("path", {
                            fill: "url(#034067__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $5f927347cd9fe340$var$ForwardRef = /*#__PURE__*/ (0, $gyl6d$forwardRef)($5f927347cd9fe340$var$SvgComponent);
var $5f927347cd9fe340$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5f927347cd9fe340$var$ForwardRef);


export {$5f927347cd9fe340$export$2e2bcd8739ae039 as default};
