// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_globe_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $hBNJQ$reactjsxruntime = require("react/jsx-runtime");
var $hBNJQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4cfe68a38654df38$export$2e2bcd8739ae039);




const $4cfe68a38654df38$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b840b7__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b840b7__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b840b7__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b840b7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("clipPath", {
                        id: "b840b7__b",
                        children: /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("clipPath", {
                id: "b840b7__a",
                children: /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m29.252 8.222-.347-.112q-.049.136-.13.261c-.337.516-.925 1.324-1.715 2.015-.777.68-1.884 1.364-3.233 1.364-2.736 0-4.442-2.084-5.106-3.323q-.045-.083-.076-.17c-1.19.408-2.317.948-3.367 1.603q.057.106.093.226c.146.487.529 1.293 1.037 2.018.25.357.508.662.752.886.255.233.43.32.511.343.578.165 1.31.14 2.267.11l.186-.006c.475-.015 1.01-.027 1.511.025.492.05 1.084.171 1.603.518.982.655 1.345 1.705 1.176 2.692-.16.93-.774 1.757-1.626 2.23-.478.266-.769.502-.94.722-.148.19-.237.406-.237.734 0 .65-.078 1.546-.707 2.21-.67.707-1.617.805-2.467.734l-.274-.023c-.707-.061-1.45-.126-2.136-.054-.794.083-1.12.31-1.254.543-.162.285-.323.723-.286 1.196.033.422.234 1.026 1.073 1.697.9.72 1.892.946 4.09 1.19 1.423.158 2.59.494 3.47 1.114.93.658 1.42 1.55 1.621 2.552.32 1.602-.231 3.464-.834 5.02-.315.813-.676 1.616-1.013 2.35l-.233.505q-.218.47-.407.886.861.09 1.746.091c3.896 0 7.477-1.35 10.3-3.61l-.001-.064c0-.325.02-.748.04-1.189l.016-.347c.026-.585.048-1.22.039-1.847-.01-.632-.053-1.226-.146-1.733-.095-.523-.229-.858-.357-1.043-.388-.56-.765-.777-1.308-1.03q-.093-.045-.197-.09c-.567-.259-1.35-.615-2.112-1.377-.51-.51-2.46-2.88-1.016-6.778.524-1.415 1.76-2.113 2.96-2.438 1.197-.324 2.571-.346 3.807-.262 1.253.085 2.457.286 3.351.462l.427.087c-.37-1.232-.88-2.402-1.512-3.494-.522.532-1.206 1.061-2.006 1.458-1.175.582-2.677.913-4.294.436-2.183-.643-2.87-2.645-3.045-4.332-.177-1.712.102-3.618.297-4.904zm11.119 13.57-.127-.026-.22-.05-.102-.023c-.259-.057-.61-.135-1.028-.217-.84-.165-1.93-.345-3.039-.42-1.125-.077-2.179-.038-2.983.18-.801.217-1.14.543-1.27.894-.983 2.653.332 4.033.44 4.141.429.429.817.608 1.385.87l.214.1c.719.335 1.554.787 2.306 1.872.417.603.636 1.335.76 2.015.128.696.175 1.442.186 2.144.005.304.003.606-.003.898 2.26-2.823 3.61-6.404 3.61-10.3q0-1.057-.13-2.078M7.5 23.869c0-4.978 2.204-9.44 5.69-12.466.274.676.688 1.447 1.171 2.136.32.458.695.913 1.11 1.293.404.37.915.733 1.513.904.98.28 2.114.24 2.96.208l.26-.009c.48-.015.862-.019 1.177.014.323.033.444.093.47.11.078.053.093.089.095.095.004.01.013.038.004.095-.023.13-.137.335-.376.469-.615.34-1.234.773-1.7 1.371-.488.629-.763 1.384-.763 2.269 0 .22-.015.363-.034.453-.082.01-.219.018-.432 0l-.252-.022c-.682-.06-1.682-.15-2.626-.05-1.05.11-2.422.49-3.165 1.79-.322.564-.696 1.506-.607 2.632.093 1.178.68 2.394 2.004 3.452 1.526 1.222 3.204 1.482 5.374 1.723 1.232.137 1.92.4 2.305.672.333.235.518.53.611 1 .165.825-.113 2.076-.713 3.625-.286.738-.62 1.484-.954 2.21l-.218.474c-.241.52-.478 1.032-.676 1.494C12.686 37.929 7.5 31.505 7.5 23.869m29.33-10.376c-1.418-1.751-3.188-3.207-5.202-4.258-.17 1.195-.314 2.532-.195 3.687.142 1.374.584 1.99 1.265 2.19.87.256 1.715.1 2.476-.278.75-.372 1.342-.924 1.656-1.34M24 7.37q1.177.001 2.31.16c-.246.323-.55.673-.897.976-.561.491-1.1.745-1.587.745-1.186 0-2.129-.82-2.679-1.635Q22.54 7.37 24 7.369"
                })
            }),
            /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("g", {
                clipPath: "url(#b840b7__a)",
                children: /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#b840b7__b)",
                    children: [
                        /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#b840b7__c)"
                        }),
                        /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("path", {
                            fill: "url(#b840b7__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("path", {
                            fill: "url(#b840b7__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $hBNJQ$reactjsxruntime.jsx)("path", {
                            fill: "url(#b840b7__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $4cfe68a38654df38$var$ForwardRef = /*#__PURE__*/ (0, $hBNJQ$react.forwardRef)($4cfe68a38654df38$var$SvgComponent);
var $4cfe68a38654df38$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4cfe68a38654df38$var$ForwardRef);
