// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_logo_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2IiPs$jsxs, jsx as $2IiPs$jsx} from "react/jsx-runtime";
import {forwardRef as $2IiPs$forwardRef} from "react";





const $5365213d91469648$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2IiPs$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2IiPs$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2IiPs$jsxs)("linearGradient", {
                        id: "3925e3__d",
                        x1: 108.785,
                        x2: 38.447,
                        y1: 14.029,
                        y2: 110.771,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2IiPs$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2IiPs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2IiPs$jsxs)("linearGradient", {
                        id: "3925e3__e",
                        x1: 62.565,
                        x2: 75.605,
                        y1: -28.658,
                        y2: 59.871,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2IiPs$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2IiPs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2IiPs$jsxs)("linearGradient", {
                        id: "3925e3__f",
                        x1: -106.301,
                        x2: 138.787,
                        y1: 148.856,
                        y2: 97.472,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2IiPs$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2IiPs$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2IiPs$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2IiPs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2IiPs$jsxs)("radialGradient", {
                        id: "3925e3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 178.298 -189.141 0 130.774 99.185)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2IiPs$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2IiPs$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2IiPs$jsx)("clipPath", {
                        id: "3925e3__b",
                        children: /*#__PURE__*/ (0, $2IiPs$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 18,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2IiPs$jsx)("clipPath", {
                id: "3925e3__a",
                children: /*#__PURE__*/ (0, $2IiPs$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m85.358 26.74 42.569 21.24c4.07 2.03 6.641 6.19 6.641 10.74v42.46c0 4.54-2.561 8.7-6.621 10.73l-42.57 21.33c-3.38 1.7-7.37 1.7-10.75 0l-42.57-21.33c-4.06-2.04-6.62-6.19-6.62-10.73V58.72c0-4.55 2.57-8.71 6.64-10.74l42.57-21.24c3.37-1.68 7.34-1.68 10.71 0M51.318 97.6c1.39 0 2.73-.72 3.47-2h-.01l14.65-25.37c1.11-1.91.45-4.36-1.46-5.46-1.91-1.11-4.36-.45-5.46 1.46L47.858 91.6c-1.11 1.91-.45 4.36 1.46 5.46.63.37 1.32.54 2 .54m53.34 0c2.89 0 5.479-1.5 6.929-4 1.45-2.51 1.45-5.5 0-8l-13.53-23.43c-1.45-2.5-4.04-4-6.93-4s-5.48 1.5-6.93 4L70.668 85.6c-1.45 2.5-1.45 5.5 0 8s4.04 4 6.93 4zm-13.52-31.43 13.529 23.43h-27.06z"
                })
            }),
            /*#__PURE__*/ (0, $2IiPs$jsx)("g", {
                clipPath: "url(#3925e3__a)",
                children: /*#__PURE__*/ (0, $2IiPs$jsxs)("g", {
                    clipPath: "url(#3925e3__b)",
                    children: [
                        /*#__PURE__*/ (0, $2IiPs$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 18,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2IiPs$jsx)("ellipse", {
                            cx: 130.774,
                            cy: 99.185,
                            fill: "url(#3925e3__c)",
                            rx: 189.141,
                            ry: 178.298
                        }),
                        /*#__PURE__*/ (0, $2IiPs$jsx)("path", {
                            fill: "url(#3925e3__d)",
                            d: "M2.21 56.679C.107 13.995 35.11-22.214 80.389-24.196s83.69 31.013 85.792 73.697-32.899 78.893-78.178 80.875c-45.28 1.982-83.69-31.014-85.793-73.697"
                        }),
                        /*#__PURE__*/ (0, $2IiPs$jsx)("path", {
                            fill: "url(#3925e3__e)",
                            d: "M47.095-67.453c-10.215-5.958-23.156-5.995-33.409-.094L-88.856-8.527c-10.043 5.78-10.06 19.424-.03 25.227L12.977 75.636c10.237 5.922 23.179 5.915 33.407-.021L148.04 16.623c9.976-5.79 9.994-19.357.031-25.169z"
                        }),
                        /*#__PURE__*/ (0, $2IiPs$jsx)("path", {
                            fill: "url(#3925e3__f)",
                            d: "M-51.305 77.067c21.136 0 40.28 8.064 54.142 21.073 27.723 26.062 50.706 26.084 78.429.067 13.861-13.076 33.03-21.162 54.189-21.162 42.295.022 76.557 32.233 76.557 71.994s-34.262 71.95-76.557 71.95c-21.207 0-40.375-8.108-54.237-21.184-27.699-25.973-50.658-25.951-78.38.111-13.862 13.031-33.007 21.073-54.143 21.073-42.27 0-76.533-32.21-76.533-71.95 0-39.739 34.262-71.972 76.533-71.972"
                        })
                    ]
                })
            })
        ]
    });
const $5365213d91469648$var$ForwardRef = /*#__PURE__*/ (0, $2IiPs$forwardRef)($5365213d91469648$var$SvgComponent);
var $5365213d91469648$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5365213d91469648$var$ForwardRef);


export {$5365213d91469648$export$2e2bcd8739ae039 as default};
