// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_infiniteLooping_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $cJp6w$reactjsxruntime = require("react/jsx-runtime");
var $cJp6w$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5a92aaf9bc0c7f89$export$2e2bcd8739ae039);




const $5a92aaf9bc0c7f89$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9a9ba7__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9a9ba7__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9a9ba7__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsxs)("radialGradient", {
                        id: "9a9ba7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("clipPath", {
                        id: "9a9ba7__b",
                        children: /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("clipPath", {
                id: "9a9ba7__a",
                children: /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M43.099 43.606c1.105-.003 1.998-.9 1.995-2.005l-.034-15.012c-.003-1.104-.9-1.998-2.005-1.995s-1.998.9-1.995 2.004l.023 10.205c-3.959-4.595-9.825-7.508-16.37-7.508-11.932 0-21.604 9.672-21.604 21.603 0 11.93 9.672 21.602 21.603 21.602 4.95 0 9.515-1.667 13.159-4.469q.078-.06.149-.126l.082-.061c2.688-2.11 5.206-5.189 7.186-8.026 1.987-2.849 3.539-5.61 4.26-7.2.573-1.266 1.992-4.273 3.872-7.434 1.618-2.72 3.48-5.388 5.345-7.193q.077-.058.15-.125c3.128-2.842 7.277-4.571 11.835-4.571 9.722 0 17.603 7.88 17.603 17.603 0 9.721-7.881 17.602-17.603 17.602-5.945 0-11.203-2.945-14.393-7.465-.637-.903-1.885-1.118-2.787-.481-.903.637-1.118 1.885-.481 2.787 3.907 5.537 10.361 9.16 17.66 9.16 11.932 0 21.604-9.673 21.604-21.603S82.68 29.295 70.75 29.295c-4.954 0-9.522 1.67-13.167 4.475q-.144.08-.278.184c-2.808 2.214-5.372 5.905-7.323 9.185-1.984 3.337-3.472 6.49-4.078 7.828-.597 1.317-2.015 3.864-3.897 6.562-1.89 2.708-4.142 5.415-6.375 7.169q-.07.054-.133.114l-.067.049c-2.968 2.283-6.683 3.64-10.72 3.64-9.722 0-17.603-7.882-17.603-17.603s7.881-17.603 17.603-17.603c5.428 0 10.282 2.455 13.514 6.322l-10.148.023c-1.105.003-1.998.9-1.995 2.005s.9 1.998 2.004 1.995z"
                })
            }),
            /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("g", {
                clipPath: "url(#9a9ba7__a)",
                children: /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#9a9ba7__b)",
                    children: [
                        /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#9a9ba7__c)"
                        }),
                        /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("path", {
                            fill: "url(#9a9ba7__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("path", {
                            fill: "url(#9a9ba7__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $cJp6w$reactjsxruntime.jsx)("path", {
                            fill: "url(#9a9ba7__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $5a92aaf9bc0c7f89$var$ForwardRef = /*#__PURE__*/ (0, $cJp6w$react.forwardRef)($5a92aaf9bc0c7f89$var$SvgComponent);
var $5a92aaf9bc0c7f89$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5a92aaf9bc0c7f89$var$ForwardRef);
