// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_handshake_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7epnX$jsxs, jsx as $7epnX$jsx} from "react/jsx-runtime";
import {forwardRef as $7epnX$forwardRef} from "react";





const $a3ab1e4804a756c1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7epnX$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7epnX$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7epnX$jsxs)("linearGradient", {
                        id: "a4859c__d",
                        x1: 64.673,
                        x2: 22.47,
                        y1: 10.009,
                        y2: 68.054,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7epnX$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7epnX$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7epnX$jsxs)("linearGradient", {
                        id: "a4859c__e",
                        x1: 36.941,
                        x2: 44.765,
                        y1: -15.603,
                        y2: 37.515,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7epnX$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7epnX$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7epnX$jsxs)("linearGradient", {
                        id: "a4859c__f",
                        x1: -64.382,
                        x2: 82.671,
                        y1: 90.905,
                        y2: 60.075,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7epnX$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7epnX$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7epnX$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7epnX$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7epnX$jsxs)("radialGradient", {
                        id: "a4859c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 106.979 -113.484 0 77.863 61.104)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7epnX$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7epnX$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7epnX$jsx)("clipPath", {
                        id: "a4859c__b",
                        children: /*#__PURE__*/ (0, $7epnX$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7epnX$jsx)("clipPath", {
                id: "a4859c__a",
                children: /*#__PURE__*/ (0, $7epnX$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m36.674 23.829 3.393 3.393 3.468 3.468 6.78 6.78c2.196 2.196 5.754 2.196 7.95 0s2.196-5.754 0-7.95l-6.78-6.78-3.468-3.468-1.128-1.128c-.804-.792-1.71-1.404-2.676-1.854l-.092-.045c-.107-.053-.216-.105-.328-.147-.378-.156-.768-.276-1.158-.384-3.204-.852-6.756-.03-9.27 2.478L15.107 36.45c-2.934 2.934-3.558 7.29-1.902 10.848l20.076-20.076zm25.958-5.59L80.89 36.499v.012c3.75 3.75 3.75 9.828 0 13.578L60.562 70.416l-8.568 8.568c-2.196 2.196-5.754 2.196-7.95-.006h-.006c-2.196-2.196-2.196-5.754 0-7.95-2.196 2.196-5.754 2.196-7.95-.006h-.006c-2.178-2.178-2.19-5.694-.048-7.896-2.202 2.142-5.718 2.124-7.896-.06h-.006c-2.196-2.196-2.196-5.76 0-7.956h-.006c-2.196 2.196-5.754 2.196-7.95-.006h-.006c-.174-.174-.33-.36-.474-.546-1.704-2.208-1.548-5.388.474-7.41l8.976-8.982 7.536-7.536 3.468 3.468 6.78 6.78c1.968 1.968 4.584 3.054 7.368 3.054s5.4-1.086 7.368-3.054c4.062-4.068 4.062-10.68 0-14.742l-9.822-9.822c3.54-1.626 7.866-.996 10.788 1.926"
                })
            }),
            /*#__PURE__*/ (0, $7epnX$jsx)("g", {
                clipPath: "url(#a4859c__a)",
                children: /*#__PURE__*/ (0, $7epnX$jsxs)("g", {
                    clipPath: "url(#a4859c__b)",
                    children: [
                        /*#__PURE__*/ (0, $7epnX$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 12.393,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7epnX$jsx)("ellipse", {
                            cx: 77.863,
                            cy: 61.104,
                            fill: "url(#a4859c__c)",
                            rx: 113.484,
                            ry: 106.979
                        }),
                        /*#__PURE__*/ (0, $7epnX$jsx)("path", {
                            fill: "url(#a4859c__d)",
                            d: "M.728 35.599c-1.262-25.61 19.74-47.336 46.907-48.525S97.849 5.682 99.11 31.293c1.262 25.61-19.74 47.335-46.907 48.524C25.035 81.007 1.99 61.21.728 35.6"
                        }),
                        /*#__PURE__*/ (0, $7epnX$jsx)("path", {
                            fill: "url(#a4859c__e)",
                            d: "M27.658-38.88c-6.128-3.575-13.893-3.596-20.045-.056L-53.912-3.524c-6.026 3.468-6.036 11.655-.019 15.136L7.19 46.974c6.142 3.553 13.907 3.549 20.044-.013l60.993-35.395c5.985-3.474 5.996-11.614.018-15.101z"
                        }),
                        /*#__PURE__*/ (0, $7epnX$jsx)("path", {
                            fill: "url(#a4859c__f)",
                            d: "M-31.384 47.832c12.68 0 24.168 4.838 32.485 12.643 16.633 15.638 30.423 15.651 47.057.04 8.317-7.845 19.818-12.697 32.513-12.697 25.377.014 45.934 19.34 45.934 43.197s-20.557 43.17-45.934 43.17c-12.724 0-24.225-4.865-32.542-12.711-16.619-15.584-30.395-15.57-47.028.067-8.317 7.819-19.804 12.644-32.485 12.644-25.363 0-45.92-19.326-45.92-43.17s20.557-43.183 45.92-43.183"
                        })
                    ]
                })
            })
        ]
    });
const $a3ab1e4804a756c1$var$ForwardRef = /*#__PURE__*/ (0, $7epnX$forwardRef)($a3ab1e4804a756c1$var$SvgComponent);
var $a3ab1e4804a756c1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a3ab1e4804a756c1$var$ForwardRef);


export {$a3ab1e4804a756c1$export$2e2bcd8739ae039 as default};
