// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_cloud_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $5Ri2U$jsxs, jsx as $5Ri2U$jsx} from "react/jsx-runtime";
import {forwardRef as $5Ri2U$forwardRef} from "react";





const $4cc2f23496eabc62$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5Ri2U$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5Ri2U$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5Ri2U$jsxs)("linearGradient", {
                        id: "db6940__d",
                        x1: 111.107,
                        x2: 30.526,
                        y1: 5.954,
                        y2: 110.429,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5Ri2U$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $5Ri2U$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5Ri2U$jsxs)("linearGradient", {
                        id: "db6940__e",
                        x1: 62.011,
                        x2: 77.555,
                        y1: -42.142,
                        y2: 57.343,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5Ri2U$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $5Ri2U$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5Ri2U$jsxs)("linearGradient", {
                        id: "db6940__f",
                        x1: -117.348,
                        x2: 144.197,
                        y1: 157.868,
                        y2: 106.177,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5Ri2U$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5Ri2U$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $5Ri2U$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $5Ri2U$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5Ri2U$jsxs)("radialGradient", {
                        id: "db6940__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.277 118.18)scale(200.895)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5Ri2U$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5Ri2U$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5Ri2U$jsx)("clipPath", {
                        id: "db6940__b",
                        children: /*#__PURE__*/ (0, $5Ri2U$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.429,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5Ri2U$jsx)("clipPath", {
                id: "db6940__a",
                children: /*#__PURE__*/ (0, $5Ri2U$jsx)("path", {
                    fill: "#fff",
                    d: "M116.525 69.207c.005-.21.016-.419.016-.63 0-16.45-12.777-29.785-28.538-29.785-12.715 0-23.483 8.681-27.172 20.672-2.448-1.055-5.127-1.644-7.94-1.644-11.502 0-20.826 9.731-20.826 21.736 0 1.691.203 3.33.553 4.91-7.598 2.172-13.188 9.41-13.188 18.029 0 10.229 7.867 18.528 17.63 18.697v.015h77.897v-.003q.07.001.14.003c13.77 0 24.933-11.651 24.933-26.023 0-13.87-10.4-25.204-23.505-25.977"
                })
            }),
            /*#__PURE__*/ (0, $5Ri2U$jsx)("g", {
                clipPath: "url(#db6940__a)",
                children: /*#__PURE__*/ (0, $5Ri2U$jsxs)("g", {
                    clipPath: "url(#db6940__b)",
                    children: [
                        /*#__PURE__*/ (0, $5Ri2U$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.429,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $5Ri2U$jsx)("circle", {
                            cx: 134.458,
                            cy: 101.903,
                            r: 200.895,
                            fill: "url(#db6940__c)"
                        }),
                        /*#__PURE__*/ (0, $5Ri2U$jsx)("path", {
                            fill: "url(#db6940__d)",
                            d: "M-2.092 54.01C-4.325 5.915 32.852-34.883 80.945-37.116s88.892 34.944 91.125 83.037-34.944 88.891-83.038 91.124C40.94 139.28.142 102.103-2.092 54.009"
                        }),
                        /*#__PURE__*/ (0, $5Ri2U$jsx)("path", {
                            fill: "url(#db6940__e)",
                            d: "M45.579-85.853c-10.85-6.714-24.595-6.755-35.485-.106l-108.915 66.5c-10.667 6.513-10.685 21.886-.032 28.424L9.341 75.37c10.873 6.673 24.62 6.665 35.483-.023L152.798 8.879c10.596-6.524 10.615-21.811.032-28.36z"
                        }),
                        /*#__PURE__*/ (0, $5Ri2U$jsx)("path", {
                            fill: "url(#db6940__f)",
                            d: "M-58.935 76.981c22.45 0 42.784 9.086 57.507 23.743 29.445 29.366 53.857 29.391 83.303.076 14.722-14.733 35.082-23.844 57.557-23.844 44.923.025 81.314 36.318 81.314 81.119s-36.391 81.068-81.314 81.068c-22.525 0-42.885-9.136-57.608-23.869-29.42-29.265-53.807-29.24-83.252.126-14.723 14.682-35.058 23.743-57.507 23.743-44.897 0-81.289-36.292-81.289-81.068s36.392-81.094 81.29-81.094"
                        })
                    ]
                })
            })
        ]
    });
const $4cc2f23496eabc62$var$ForwardRef = /*#__PURE__*/ (0, $5Ri2U$forwardRef)($4cc2f23496eabc62$var$SvgComponent);
var $4cc2f23496eabc62$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($4cc2f23496eabc62$var$ForwardRef);


export {$4cc2f23496eabc62$export$2e2bcd8739ae039 as default};
