// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_pieChart_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $j9dug$jsxs, jsx as $j9dug$jsx} from "react/jsx-runtime";
import {forwardRef as $j9dug$forwardRef} from "react";





const $f172e480556e58c9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $j9dug$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $j9dug$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $j9dug$jsxs)("linearGradient", {
                        id: "2586d8__d",
                        x1: 65.566,
                        x2: 25.102,
                        y1: 11.636,
                        y2: 67.288,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $j9dug$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $j9dug$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $j9dug$jsxs)("linearGradient", {
                        id: "2586d8__e",
                        x1: 38.978,
                        x2: 46.479,
                        y1: -12.92,
                        y2: 38.008,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $j9dug$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $j9dug$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $j9dug$jsxs)("linearGradient", {
                        id: "2586d8__f",
                        x1: -58.165,
                        x2: 82.825,
                        y1: 89.197,
                        y2: 59.638,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $j9dug$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $j9dug$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $j9dug$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $j9dug$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $j9dug$jsxs)("radialGradient", {
                        id: "2586d8__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 78.212 60.624)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $j9dug$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $j9dug$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $j9dug$jsx)("clipPath", {
                        id: "2586d8__b",
                        children: /*#__PURE__*/ (0, $j9dug$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $j9dug$jsx)("clipPath", {
                id: "2586d8__a",
                children: /*#__PURE__*/ (0, $j9dug$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M45.602 14.496c-7.47.528-14.268 3.498-19.596 8.118L45.602 42.21zM22.616 26.004C17.504 31.896 14.402 39.582 14.402 48s3.102 16.098 8.214 21.996L44.612 48zM50.402 48V14.502c17.436 1.23 31.2 14.946 31.2 33.504s-15.042 33.6-33.6 33.6c-8.418 0-16.104-3.102-21.996-8.214L49.7 49.698q.043-.043.083-.089l-.084.09q.094-.1.175-.2.058-.074.11-.15l.001-.002.015-.02c.054-.085.096-.175.138-.265l.017-.033.013-.024q.026-.044.048-.093c.034-.082.056-.164.079-.25l.017-.062.005-.015.006-.02.014-.043.007-.02.005-.017.005-.017q.046-.234.048-.468"
                })
            }),
            /*#__PURE__*/ (0, $j9dug$jsx)("g", {
                clipPath: "url(#2586d8__a)",
                children: /*#__PURE__*/ (0, $j9dug$jsxs)("g", {
                    clipPath: "url(#2586d8__b)",
                    children: [
                        /*#__PURE__*/ (0, $j9dug$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $j9dug$jsx)("ellipse", {
                            cx: 78.212,
                            cy: 60.624,
                            fill: "url(#2586d8__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $j9dug$jsx)("path", {
                            fill: "url(#2586d8__d)",
                            d: "M4.256 36.171C3.046 11.616 23.182-9.213 49.23-10.353s48.144 17.84 49.353 42.395S79.658 77.426 53.61 78.566 5.466 60.726 4.256 36.171"
                        }),
                        /*#__PURE__*/ (0, $j9dug$jsx)("path", {
                            fill: "url(#2586d8__e)",
                            d: "M30.078-35.237c-5.876-3.427-13.321-3.448-19.219-.054L-48.13-1.339c-5.777 3.325-5.787 11.174-.017 14.512l58.599 33.904c5.889 3.407 13.334 3.403 19.218-.012L88.148 13.13c5.74-3.33 5.75-11.135.018-14.479z"
                        }),
                        /*#__PURE__*/ (0, $j9dug$jsx)("path", {
                            fill: "url(#2586d8__f)",
                            d: "M-26.528 47.9c12.159 0 23.172 4.638 31.146 12.122 15.948 14.993 29.17 15.005 45.118.038 7.974-7.522 19-12.173 31.173-12.173 24.33.013 44.04 18.542 44.04 41.415 0 22.874-19.71 41.391-44.04 41.391-12.2 0-23.227-4.665-31.2-12.187-15.935-14.941-29.143-14.928-45.09.064-7.975 7.497-18.988 12.123-31.147 12.123-24.317 0-44.026-18.53-44.026-41.39 0-22.861 19.71-41.403 44.026-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $f172e480556e58c9$var$ForwardRef = /*#__PURE__*/ (0, $j9dug$forwardRef)($f172e480556e58c9$var$SvgComponent);
var $f172e480556e58c9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f172e480556e58c9$var$ForwardRef);


export {$f172e480556e58c9$export$2e2bcd8739ae039 as default};
