// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_cut_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $dsC4T$reactjsxruntime = require("react/jsx-runtime");
var $dsC4T$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $443daa725d4f2faa$export$2e2bcd8739ae039);




const $443daa725d4f2faa$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsxs)("linearGradient", {
                        id: "441b5f__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsxs)("linearGradient", {
                        id: "441b5f__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsxs)("linearGradient", {
                        id: "441b5f__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsxs)("radialGradient", {
                        id: "441b5f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("clipPath", {
                        id: "441b5f__b",
                        children: /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("clipPath", {
                id: "441b5f__a",
                children: /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M99.929 23.656c2.09.716 3.204 2.99 2.489 5.08L87.936 71.038l43.076-14.67c2.092-.711 4.364.407 5.076 2.498s-.406 4.364-2.497 5.076L85.677 80.259q-.143.048-.285.085c-.249.982-.48 1.98-.714 3-.86 3.72-1.783 7.72-3.876 12.236-.41.887-.861 1.786-1.338 2.68q.333-.01.67-.01c10.58 0 19.156 8.576 19.156 19.155s-8.577 19.157-19.156 19.157c-10.58 0-19.156-8.577-19.156-19.157 0-4.155 1.322-8.001 3.57-11.139q.2-.334.472-.632.382-.49.795-.954c1.8-2.297 5.478-7.605 7.729-12.463 1.649-3.56 2.335-6.388 3.1-9.69-1.505.56-3.144 1.208-5.112 2.019-2.812 1.158-6.1 2.997-8.955 4.776-2.89 1.801-5.055 3.363-5.661 3.878q-.212.18-.439.324c-3.473 3.514-8.295 5.692-13.626 5.692-10.58 0-19.156-8.577-19.156-19.156 0-10.58 8.577-19.156 19.156-19.156 10.58 0 19.157 8.576 19.157 19.156l-.003.3c2.117-1.193 4.359-2.338 6.48-3.211 4.208-1.734 7.05-2.786 10.198-3.782l16.166-47.222c.715-2.09 2.99-3.204 5.08-2.489m-28.732 87.071c-1.394 1.862-2.22 4.174-2.22 6.678 0 6.162 4.995 11.157 11.157 11.157 6.161 0 11.156-4.995 11.156-11.157 0-6.161-4.995-11.156-11.156-11.156-3.263 0-6.199 1.401-8.239 3.634q-.389.487-.698.844M42.85 68.904c-6.16 0-11.156 4.994-11.156 11.156 0 6.161 4.995 11.156 11.156 11.156S54.008 86.22 54.008 80.06 49.013 68.904 42.85 68.904"
                })
            }),
            /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("g", {
                clipPath: "url(#441b5f__a)",
                children: /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#441b5f__b)",
                    children: [
                        /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#441b5f__c)"
                        }),
                        /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("path", {
                            fill: "url(#441b5f__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("path", {
                            fill: "url(#441b5f__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $dsC4T$reactjsxruntime.jsx)("path", {
                            fill: "url(#441b5f__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $443daa725d4f2faa$var$ForwardRef = /*#__PURE__*/ (0, $dsC4T$react.forwardRef)($443daa725d4f2faa$var$SvgComponent);
var $443daa725d4f2faa$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($443daa725d4f2faa$var$ForwardRef);
