// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_paperairplane_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $hluaE$reactjsxruntime = require("react/jsx-runtime");
var $hluaE$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3b5606e76cd5fdfc$export$2e2bcd8739ae039);




const $3b5606e76cd5fdfc$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f4bcf0__d",
                        x1: 67.078,
                        x2: 22.557,
                        y1: 6.922,
                        y2: 68.155,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f4bcf0__e",
                        x1: 37.82,
                        x2: 46.073,
                        y1: -20.095,
                        y2: 35.94,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f4bcf0__f",
                        x1: -69.063,
                        x2: 86.066,
                        y1: 92.261,
                        y2: 59.738,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f4bcf0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 112.855 -119.717 0 80.995 60.823)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("clipPath", {
                        id: "f4bcf0__b",
                        children: /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("rect", {
                            width: 82.917,
                            height: 78.164,
                            x: 7.082,
                            y: 9.437,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("clipPath", {
                id: "f4bcf0__a",
                children: /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M82.963 23.51c.986-.221 1.92-.097 2.72.269L23.49 56.07q-.294.154-.526.37L9.651 47.33c-3.07-2.1-2.155-6.824 1.48-7.643zM56.894 79.65 27.38 59.46l60.723-31.53q.045-.022.087-.047c-.031.557-.178 1.127-.46 1.683L63.14 78.061c-1.167 2.302-4.11 3.05-6.247 1.589m-32.296-.86V63.56l14.095 9.604-9.318 7.754c-1.876 1.562-4.777.269-4.777-2.128"
                })
            }),
            /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("g", {
                clipPath: "url(#f4bcf0__a)",
                children: /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#f4bcf0__b)",
                    children: [
                        /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("rect", {
                            width: 82.917,
                            height: 78.164,
                            x: 7.082,
                            y: 9.437,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("ellipse", {
                            cx: 80.995,
                            cy: 60.823,
                            fill: "url(#f4bcf0__c)",
                            rx: 119.717,
                            ry: 112.855
                        }),
                        /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("path", {
                            fill: "url(#f4bcf0__d)",
                            d: "M-.38 33.918C-1.71 6.9 20.445-16.018 49.105-17.272c28.66-1.255 52.972 19.63 54.303 46.647S82.583 79.31 53.923 80.565.951 60.934-.379 33.917"
                        }),
                        /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("path", {
                            fill: "url(#f4bcf0__e)",
                            d: "M28.028-44.65c-6.466-3.772-14.657-3.795-21.146-.06L-58.023-7.353c-6.357 3.659-6.367 12.295-.02 15.967L6.434 45.918c6.48 3.749 14.672 3.744 21.146-.013L91.92 8.565c6.315-3.664 6.326-12.252.02-15.93z"
                        }),
                        /*#__PURE__*/ (0, $hluaE$reactjsxruntime.jsx)("path", {
                            fill: "url(#f4bcf0__f)",
                            d: "M-34.253 46.823c13.378 0 25.496 5.104 34.27 13.338 17.546 16.496 32.094 16.51 49.641.042 8.773-8.276 20.906-13.394 34.3-13.394 26.77.014 48.456 20.401 48.456 45.569 0 25.167-21.686 45.54-48.457 45.54-13.422 0-25.555-5.132-34.329-13.408-17.532-16.44-32.065-16.426-49.612.07-8.773 8.249-20.891 13.338-34.269 13.338-26.755 0-48.442-20.387-48.442-45.54 0-25.154 21.687-45.555 48.442-45.555"
                        })
                    ]
                })
            })
        ]
    });
const $3b5606e76cd5fdfc$var$ForwardRef = /*#__PURE__*/ (0, $hluaE$react.forwardRef)($3b5606e76cd5fdfc$var$SvgComponent);
var $3b5606e76cd5fdfc$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3b5606e76cd5fdfc$var$ForwardRef);
