// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_microphone_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $dlW8E$reactjsxruntime = require("react/jsx-runtime");
var $dlW8E$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $915e12781c4c61c4$export$2e2bcd8739ae039);




const $915e12781c4c61c4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7889ba__d",
                        x1: 66.406,
                        x2: 18.058,
                        y1: 3.316,
                        y2: 66.001,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7889ba__e",
                        x1: 36.95,
                        x2: 46.277,
                        y1: -25.541,
                        y2: 34.149,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7889ba__f",
                        x1: -70.665,
                        x2: 86.262,
                        y1: 94.463,
                        y2: 63.449,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsxs)("radialGradient", {
                        id: "7889ba__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 9.766 70.65)scale(120.537)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("clipPath", {
                        id: "7889ba__b",
                        children: /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6,
                            y: 6,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("clipPath", {
                id: "7889ba__a",
                children: /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M47.4 14.879c-6.362 0-11.52 5.158-11.52 11.52v17.16c0 6.362 5.158 11.52 11.52 11.52h1.56c6.363 0 11.52-5.158 11.52-11.52v-17.16c0-6.362-5.157-11.52-11.52-11.52zm-1.92 54.465c-13.477-1.353-24-12.73-24-26.565v-2.7c0-1.326 1.075-2.4 2.4-2.4 1.326 0 2.4 1.074 2.4 2.4v2.7c0 12.095 9.805 21.9 21.9 21.9 12.096 0 21.9-9.805 21.9-21.9v-2.7c0-1.326 1.075-2.4 2.4-2.4 1.326 0 2.4 1.074 2.4 2.4v2.7c0 14.04-10.835 25.548-24.6 26.618v7.282h12c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4h-28.2c-1.325 0-2.4-1.075-2.4-2.4 0-1.326 1.075-2.4 2.4-2.4h11.4z"
                })
            }),
            /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("g", {
                clipPath: "url(#7889ba__a)",
                children: /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#7889ba__b)",
                    children: [
                        /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6,
                            y: 6,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("circle", {
                            cx: 80.416,
                            cy: 60.885,
                            r: 120.537,
                            fill: "url(#7889ba__c)"
                        }),
                        /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("path", {
                            fill: "url(#7889ba__d)",
                            d: "M-1.513 32.149c-1.34-28.856 20.966-53.335 49.822-54.675s53.335 20.966 54.675 49.822-20.966 53.335-49.822 54.675S-.173 61.005-1.513 32.149"
                        }),
                        /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("path", {
                            fill: "url(#7889ba__e)",
                            d: "M27.091-51.768c-6.51-4.028-14.757-4.053-21.29-.064l-65.35 39.9c-6.4 3.908-6.41 13.132-.02 17.054L5.35 44.966c6.524 4.003 14.772 3.999 21.29-.014L91.422 5.07c6.358-3.914 6.37-13.087.02-17.016z"
                        }),
                        /*#__PURE__*/ (0, $dlW8E$reactjsxruntime.jsx)("path", {
                            fill: "url(#7889ba__f)",
                            d: "M-35.617 45.931c13.47 0 25.67 5.452 34.504 14.246 17.667 17.62 32.314 17.634 49.982.045 8.833-8.84 21.05-14.306 34.534-14.306 26.954.015 48.788 21.79 48.788 48.671s-21.834 48.641-48.788 48.641c-13.515 0-25.731-5.481-34.565-14.321-17.652-17.559-32.284-17.544-49.951.075-8.834 8.81-21.035 14.246-34.504 14.246-26.939 0-48.774-21.775-48.774-48.64S-62.556 45.93-35.617 45.93"
                        })
                    ]
                })
            })
        ]
    });
const $915e12781c4c61c4$var$ForwardRef = /*#__PURE__*/ (0, $dlW8E$react.forwardRef)($915e12781c4c61c4$var$SvgComponent);
var $915e12781c4c61c4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($915e12781c4c61c4$var$ForwardRef);
