// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_noElements_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $jhy9k$reactjsxruntime = require("react/jsx-runtime");
var $jhy9k$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d01ee3cb93227922$export$2e2bcd8739ae039);




const $d01ee3cb93227922$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d96c2b__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d96c2b__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d96c2b__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d96c2b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("clipPath", {
                        id: "d96c2b__b",
                        children: /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsxs)("clipPath", {
                id: "d96c2b__a",
                children: [
                    /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("path", {
                        d: "M37.522 6.517c.278-.321.76-.59 1.058-.499l.002.002c.477 0 .938.26 1.202.675.261.41.304.928.117 1.39l-1.619 3.96 2.348 1.031c.425.186.736.576.836 1.043s-.024.958-.333 1.314l-6.13 7.073c-.275.319-.65.494-1.052.494-.477 0-.938-.257-1.202-.672-.261-.407-.306-.928-.119-1.388l1.566-3.86-2.333-1.045c-.42-.187-.73-.578-.83-1.042s.025-.95.329-1.306z"
                    }),
                    /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("path", {
                        fillRule: "evenodd",
                        d: "M27 38.375V32c0-2.21 1.79-4 4-4h10.75c2.21 0 4 1.79 4 4v6.375c0 2.21-1.79 4-4 4H31c-2.21 0-4-1.79-4-4m12.026-3.746-3.689-2.105c-.413-.236-.93.057-.93.528l-.006 4.203c0 .47.516.765.93.53l3.694-2.096c.414-.235.414-.824.001-1.06M3.5 17.375V11c0-2.21 1.79-4 4-4h11.063c2.209 0 4 1.79 4 4v6.375c0 .593-.13 1.157-.361 1.663l-2.141-3.34c-1.28-1.995-4.194-1.995-5.473 0l-.39.61-1.603-2.527c-1.272-2.005-4.196-2.013-5.48-.015l-3.307 5.15c-.199-.474-.308-.995-.308-1.541M19.25 11.5c.69 0 1.25-.56 1.25-1.25S19.94 9 19.25 9 18 9.56 18 10.25s.56 1.25 1.25 1.25"
                    }),
                    /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("path", {
                        d: "m10.484 15.12 2.225 3.51-1.457 2.272c-.095.148-.154.31-.181.473H7.5c-.716 0-1.387-.188-1.969-.517l3.688-5.74c.297-.462.971-.46 1.265.003M13.919 21.375h4.643c.676 0 1.313-.168 1.87-.463l-2.476-3.865c-.295-.46-.968-.46-1.263 0zM4.887 28c0-.69.56-1.25 1.25-1.25H20.21c.69 0 1.25.56 1.25 1.25v2.815c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25V29.25h-4.537v11.573h1.565c.69 0 1.25.56 1.25 1.25s-.56 1.25-1.25 1.25h-5.63c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h1.565V29.25H7.387v1.565c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("g", {
                clipPath: "url(#d96c2b__a)",
                children: /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#d96c2b__b)",
                    children: [
                        /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#d96c2b__c)"
                        }),
                        /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("path", {
                            fill: "url(#d96c2b__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("path", {
                            fill: "url(#d96c2b__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $jhy9k$reactjsxruntime.jsx)("path", {
                            fill: "url(#d96c2b__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $d01ee3cb93227922$var$ForwardRef = /*#__PURE__*/ (0, $jhy9k$react.forwardRef)($d01ee3cb93227922$var$SvgComponent);
var $d01ee3cb93227922$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d01ee3cb93227922$var$ForwardRef);
