// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_rocket_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $hRE9x$jsxs, jsx as $hRE9x$jsx} from "react/jsx-runtime";
import {forwardRef as $hRE9x$forwardRef} from "react";





const $20256b446a243adf$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hRE9x$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hRE9x$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hRE9x$jsxs)("linearGradient", {
                        id: "aae9a3__d",
                        x1: 64.673,
                        x2: 22.47,
                        y1: 11.537,
                        y2: 69.582,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hRE9x$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $hRE9x$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hRE9x$jsxs)("linearGradient", {
                        id: "aae9a3__e",
                        x1: 36.941,
                        x2: 44.765,
                        y1: -14.075,
                        y2: 39.042,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hRE9x$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $hRE9x$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hRE9x$jsxs)("linearGradient", {
                        id: "aae9a3__f",
                        x1: -64.382,
                        x2: 82.671,
                        y1: 92.432,
                        y2: 61.602,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hRE9x$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hRE9x$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $hRE9x$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $hRE9x$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hRE9x$jsxs)("radialGradient", {
                        id: "aae9a3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 106.979 -113.484 0 77.863 62.631)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hRE9x$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hRE9x$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hRE9x$jsx)("clipPath", {
                        id: "aae9a3__b",
                        children: /*#__PURE__*/ (0, $hRE9x$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 13.92,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hRE9x$jsx)("clipPath", {
                id: "aae9a3__a",
                children: /*#__PURE__*/ (0, $hRE9x$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M64.213 52.331c-3.696 3.528-8.274 7.188-13.932 10.914h-.006q-.71.466-1.477.798c-3.57 1.548-7.8.816-10.626-2.01L36.05 59.91l-2.124-2.124c-2.676-2.682-3.468-6.612-2.226-10.056.257-.708.593-1.398 1.02-2.046 2.694-4.092 5.358-7.614 7.956-10.65C55.777 17.4 68.857 16.26 74.827 16.937c2.207.252 3.948 1.998 4.2 4.2.63 5.58-.319 17.364-14.814 31.194m1.02-21.594c-1.987-1.98-5.203-1.98-7.183 0-1.98 1.986-1.985 5.202 0 7.182s5.203 1.986 7.183 0 1.986-5.202 0-7.182M26.466 49.163l-6.288.108v.006c-4.086.072-6.402-4.68-3.81-7.836 7.326-8.91 17.196-12.084 23.634-12.81-3.882 4.092-7.656 8.904-11.292 14.424-1.236 1.878-1.974 3.978-2.244 6.108m-9.246 21.96c-.468.468-1.086.702-1.698.702-.618 0-1.23-.234-1.698-.702-.936-.942-.936-2.46 0-3.396l6.3-6.3c.942-.936 2.46-.936 3.396 0 .936.942.936 2.46 0 3.396zm8.4-4.2c.942-.936 2.46-.936 3.396 0 .936.942.936 2.46 0 3.396l-12 12c-.468.468-1.08.702-1.698.702s-1.23-.234-1.698-.702c-.936-.942-.936-2.46 0-3.396zm5.532 5.526-6.6 6.6c-.936.936-.936 2.454 0 3.396.468.468 1.08.702 1.698.702s1.23-.234 1.698-.702l6.6-6.6c.936-.936.936-2.454 0-3.396s-2.454-.936-3.396 0m15.667-2.934-.108 6.264c-.072 4.11 4.71 6.39 7.878 3.774 8.856-7.308 12.024-17.13 12.762-23.562-3.888 3.672-8.634 7.446-14.43 11.262-1.848 1.218-3.93 1.98-6.102 2.262"
                })
            }),
            /*#__PURE__*/ (0, $hRE9x$jsx)("g", {
                clipPath: "url(#aae9a3__a)",
                children: /*#__PURE__*/ (0, $hRE9x$jsxs)("g", {
                    clipPath: "url(#aae9a3__b)",
                    children: [
                        /*#__PURE__*/ (0, $hRE9x$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 7.801,
                            y: 13.92,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $hRE9x$jsx)("ellipse", {
                            cx: 77.863,
                            cy: 62.631,
                            fill: "url(#aae9a3__c)",
                            rx: 113.484,
                            ry: 106.979
                        }),
                        /*#__PURE__*/ (0, $hRE9x$jsx)("path", {
                            fill: "url(#aae9a3__d)",
                            d: "M.728 37.126c-1.262-25.61 19.74-47.335 46.907-48.524S97.849 7.21 99.11 32.82 79.37 80.155 52.203 81.345C25.035 82.534 1.99 62.737.728 37.126"
                        }),
                        /*#__PURE__*/ (0, $hRE9x$jsx)("path", {
                            fill: "url(#aae9a3__e)",
                            d: "M27.658-37.352c-6.128-3.575-13.893-3.597-20.045-.057L-53.912-1.997c-6.026 3.469-6.036 11.655-.019 15.136L7.19 48.501c6.142 3.553 13.907 3.549 20.044-.013l60.993-35.394c5.985-3.474 5.996-11.615.018-15.102z"
                        }),
                        /*#__PURE__*/ (0, $hRE9x$jsx)("path", {
                            fill: "url(#aae9a3__f)",
                            d: "M-31.384 49.36c12.68 0 24.168 4.837 32.485 12.643 16.633 15.637 30.423 15.65 47.057.04 8.317-7.846 19.818-12.697 32.513-12.697 25.377.013 45.934 19.34 45.934 43.196 0 23.857-20.557 43.17-45.934 43.17-12.724 0-24.225-4.865-32.542-12.71-16.619-15.584-30.395-15.571-47.028.067-8.317 7.818-19.804 12.643-32.485 12.643-25.363 0-45.92-19.326-45.92-43.17 0-23.843 20.557-43.183 45.92-43.183"
                        })
                    ]
                })
            })
        ]
    });
const $20256b446a243adf$var$ForwardRef = /*#__PURE__*/ (0, $hRE9x$forwardRef)($20256b446a243adf$var$SvgComponent);
var $20256b446a243adf$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($20256b446a243adf$var$ForwardRef);


export {$20256b446a243adf$export$2e2bcd8739ae039 as default};
