// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_tablet_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $7sf07$reactjsxruntime = require("react/jsx-runtime");
var $7sf07$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $98e55e0d89433d2c$export$2e2bcd8739ae039);




const $98e55e0d89433d2c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsxs)("linearGradient", {
                        id: "721d7a__d",
                        x1: 111.296,
                        x2: 36.125,
                        y1: 12.168,
                        y2: 115.556,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsxs)("linearGradient", {
                        id: "721d7a__e",
                        x1: 61.9,
                        x2: 75.835,
                        y1: -33.45,
                        y2: 61.161,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsxs)("linearGradient", {
                        id: "721d7a__f",
                        x1: -118.567,
                        x2: 143.358,
                        y1: 156.258,
                        y2: 101.344,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsxs)("radialGradient", {
                        id: "721d7a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 190.547 -202.135 0 134.795 103.175)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("clipPath", {
                        id: "721d7a__b",
                        children: /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("rect", {
                            width: 140,
                            height: 131.975,
                            x: 10,
                            y: 16.413,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("clipPath", {
                id: "721d7a__a",
                children: /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M128 28H32c-8.84 0-16 7.16-16 16v72c0 8.84 7.16 16 16 16h96c8.84 0 16-7.16 16-16V44c0-8.84-7.16-16-16-16M34 84c-2.21 0-4-1.79-4-4s1.78-4 3.99-4H34c2.21 0 4 1.79 4 4s-1.79 4-4 4"
                })
            }),
            /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("g", {
                clipPath: "url(#721d7a__a)",
                children: /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#721d7a__b)",
                    children: [
                        /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("rect", {
                            width: 140,
                            height: 131.975,
                            x: 10,
                            y: 16.413,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("ellipse", {
                            cx: 134.795,
                            cy: 103.175,
                            fill: "url(#721d7a__c)",
                            rx: 202.135,
                            ry: 190.547
                        }),
                        /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("path", {
                            fill: "url(#721d7a__d)",
                            d: "M-2.601 57.748c-2.247-45.616 35.16-84.312 83.55-86.43s89.439 33.143 91.686 78.76c2.247 45.616-35.16 84.312-83.55 86.43S-.355 103.364-2.6 57.748"
                        }),
                        /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("path", {
                            fill: "url(#721d7a__e)",
                            d: "M45.367-74.91c-10.917-6.368-24.747-6.407-35.704-.1L-99.924-11.936c-10.733 6.178-10.751 20.759-.033 26.96L8.906 78.009c10.94 6.329 24.772 6.321 35.702-.022l108.639-63.045c10.661-6.187 10.681-20.687.033-26.898z"
                        }),
                        /*#__PURE__*/ (0, $7sf07$reactjsxruntime.jsx)("path", {
                            fill: "url(#721d7a__f)",
                            d: "M-59.793 79.538c22.588 0 43.048 8.617 57.862 22.52 29.627 27.852 54.19 27.876 83.816.071 14.814-13.974 35.299-22.615 57.912-22.615 45.2.023 81.816 34.447 81.816 76.94s-36.616 76.892-81.816 76.892c-22.663 0-43.149-8.665-57.962-22.639-29.602-27.757-54.14-27.734-83.766.119-14.814 13.927-35.274 22.52-57.862 22.52-45.175 0-81.791-34.423-81.791-76.892 0-42.47 36.616-76.917 81.791-76.917"
                        })
                    ]
                })
            })
        ]
    });
const $98e55e0d89433d2c$var$ForwardRef = /*#__PURE__*/ (0, $7sf07$react.forwardRef)($98e55e0d89433d2c$var$SvgComponent);
var $98e55e0d89433d2c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($98e55e0d89433d2c$var$ForwardRef);
