// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_play_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $aYuRK$reactjsxruntime = require("react/jsx-runtime");
var $aYuRK$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $165d1b47127c5989$export$2e2bcd8739ae039);




const $165d1b47127c5989$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ed36b9__d",
                        x1: 33.286,
                        x2: 9.233,
                        y1: 1.897,
                        y2: 33.084,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ed36b9__e",
                        x1: 18.631,
                        x2: 23.271,
                        y1: -12.46,
                        y2: 17.237,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ed36b9__f",
                        x1: -34.908,
                        x2: 43.165,
                        y1: 47.245,
                        y2: 31.815,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ed36b9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 59.9687 -59.9688 0 40.258 30.54)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("clipPath", {
                        id: "ed36b9__b",
                        children: /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.234,
                            y: 3.232,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("clipPath", {
                id: "ed36b9__a",
                children: /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M23.998 8.4c-8.615 0-15.6 6.985-15.6 15.6 0 8.616 6.985 15.6 15.6 15.6 8.616 0 15.6-6.984 15.6-15.6s-6.984-15.6-15.6-15.6m5.1 16.649-7.2 4.145c-.801.461-1.8-.117-1.8-1.041l.01-8.308c.001-.924 1.002-1.5 1.802-1.037l7.19 4.163c.8.462.798 1.617-.003 2.078"
                })
            }),
            /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ed36b9__a)",
                children: /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ed36b9__b)",
                    children: [
                        /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.234,
                            y: 3.232,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("circle", {
                            cx: 40.258,
                            cy: 30.539,
                            r: 59.969,
                            fill: "url(#ed36b9__c)"
                        }),
                        /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("path", {
                            fill: "url(#ed36b9__d)",
                            d: "M-.504 16.242c-.667-14.356 10.43-26.535 24.787-27.201S50.818-.53 51.484 13.828s-10.43 26.535-24.787 27.201S.162 30.6-.504 16.242"
                        }),
                        /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("path", {
                            fill: "url(#ed36b9__e)",
                            d: "M13.726-25.508c-3.238-2.004-7.342-2.016-10.592-.032L-29.378-5.689c-3.184 1.944-3.19 6.533-.01 8.485L2.91 22.618c3.246 1.992 7.35 1.99 10.592-.007l32.23-19.84c3.164-1.948 3.17-6.511.01-8.466z"
                        }),
                        /*#__PURE__*/ (0, $aYuRK$reactjsxruntime.jsx)("path", {
                            fill: "url(#ed36b9__f)",
                            d: "M-17.471 23.1c6.701 0 12.771 2.711 17.166 7.087 8.79 8.766 16.077 8.773 24.867.022 4.394-4.398 10.472-7.117 17.18-7.117 13.41.007 24.274 10.84 24.274 24.214s-10.864 24.2-24.273 24.2c-6.724 0-12.802-2.727-17.197-7.125-8.782-8.736-16.061-8.728-24.851.037-4.395 4.383-10.465 7.088-17.166 7.088-13.402 0-24.266-10.834-24.266-24.2S-30.873 23.1-17.47 23.1"
                        })
                    ]
                })
            })
        ]
    });
const $165d1b47127c5989$var$ForwardRef = /*#__PURE__*/ (0, $aYuRK$react.forwardRef)($165d1b47127c5989$var$SvgComponent);
var $165d1b47127c5989$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($165d1b47127c5989$var$ForwardRef);
