// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fileVideo_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $87W12$reactjsxruntime = require("react/jsx-runtime");
var $87W12$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $137be996d9742a86$export$2e2bcd8739ae039);




const $137be996d9742a86$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $87W12$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c2ca91__d",
                        x1: 33.33,
                        x2: 9.156,
                        y1: 1.787,
                        y2: 33.129,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c2ca91__e",
                        x1: 18.602,
                        x2: 23.265,
                        y1: -12.642,
                        y2: 17.203,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c2ca91__f",
                        x1: -35.206,
                        x2: 43.258,
                        y1: 47.361,
                        y2: 31.853,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsxs)("radialGradient", {
                        id: "c2ca91__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 4.884 35.455)scale(60.2686)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("clipPath", {
                        id: "c2ca91__b",
                        children: /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("clipPath", {
                id: "c2ca91__a",
                children: /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "m36.207 19.323-9.941-9.941c-.678-.679-1.599-1.06-2.558-1.06h-9.356c-1.998 0-3.618 1.62-3.618 3.618v24.12c0 1.998 1.62 3.618 3.618 3.618h19.296c1.999 0 3.618-1.62 3.618-3.618V21.88c0-.959-.38-1.879-1.06-2.557m-7.685 8.672-7.184 4.136c-.805.463-1.81-.118-1.808-1.047l.01-8.29c.001-.928 1.007-1.507 1.81-1.042l7.174 4.154c.804.465.802 1.625-.002 2.089"
                })
            }),
            /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("g", {
                clipPath: "url(#c2ca91__a)",
                children: /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#c2ca91__b)",
                    children: [
                        /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("circle", {
                            cx: 40.339,
                            cy: 30.571,
                            r: 60.269,
                            fill: "url(#c2ca91__c)"
                        }),
                        /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("path", {
                            fill: "url(#c2ca91__d)",
                            d: "M-.63 16.203C-1.3 1.775 9.854-10.464 24.282-11.134S50.949-.651 51.619 13.777 41.136 40.444 26.708 41.114.04 30.631-.63 16.204"
                        }),
                        /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("path", {
                            fill: "url(#c2ca91__e)",
                            d: "M13.672-25.755c-3.254-2.014-7.378-2.027-10.645-.032l-32.675 19.95c-3.2 1.954-3.205 6.566-.01 8.527l32.46 19.922c3.261 2.002 7.385 2 10.644-.007l32.392-19.94c3.179-1.958 3.185-6.544.01-8.508z"
                        }),
                        /*#__PURE__*/ (0, $87W12$reactjsxruntime.jsx)("path", {
                            fill: "url(#c2ca91__f)",
                            d: "M-17.682 23.094c6.735 0 12.836 2.726 17.252 7.123 8.834 8.81 16.158 8.818 24.991.023 4.417-4.42 10.525-7.153 17.267-7.153 13.477.007 24.395 10.895 24.395 24.336s-10.918 24.32-24.395 24.32c-6.757 0-12.865-2.74-17.282-7.16-8.826-8.78-16.142-8.773-24.976.037-4.416 4.405-10.517 7.123-17.252 7.123-13.469 0-24.386-10.888-24.386-24.32s10.917-24.329 24.386-24.329"
                        })
                    ]
                })
            })
        ]
    });
const $137be996d9742a86$var$ForwardRef = /*#__PURE__*/ (0, $87W12$react.forwardRef)($137be996d9742a86$var$SvgComponent);
var $137be996d9742a86$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($137be996d9742a86$var$ForwardRef);
