// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_userGroup_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $csRxY$reactjsxruntime = require("react/jsx-runtime");
var $csRxY$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3ffec038811e59ad$export$2e2bcd8739ae039);




const $3ffec038811e59ad$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7541ca__d",
                        x1: 107.785,
                        x2: 37.447,
                        y1: 16.683,
                        y2: 113.425,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7541ca__e",
                        x1: 61.565,
                        x2: 74.605,
                        y1: -26.004,
                        y2: 62.525,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7541ca__f",
                        x1: -107.301,
                        x2: 137.787,
                        y1: 151.51,
                        y2: 100.127,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsxs)("radialGradient", {
                        id: "7541ca__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 178.298 -189.141 0 129.774 101.839)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("clipPath", {
                        id: "7541ca__b",
                        children: /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("clipPath", {
                id: "7541ca__a",
                children: /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M116.85 50.45c0 13.53-10.298 24.5-23 24.5s-23-10.97-23-24.5c0-13.532 10.297-24.5 23-24.5s23 10.968 23 24.5M72.59 74.08c-2.04.63-4.2.98-6.44.98-12.7 0-23-10.97-23-24.5s10.3-24.5 23-24.5c2.17 0 4.28.33 6.27.93-5.9 5.92-9.58 14.16-9.58 23.46s3.75 17.7 9.75 23.63m-32.52 48.389c3.25-15.09 12.65-27.7 25.19-35.4-22.04.42-40.58 16.32-45.07 37.19-1.08 5.02 2.67 9.78 7.81 9.78h13.94c-2.04-3.46-2.74-7.56-1.87-11.57m53.77-35.52c22.43 0 41.41 16.06 45.96 37.21 1.08 5.02-2.67 9.78-7.81 9.78h-76.3c-5.14 0-8.89-4.76-7.81-9.78 4.55-21.15 23.53-37.21 45.96-37.21"
                })
            }),
            /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("g", {
                clipPath: "url(#7541ca__a)",
                children: /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#7541ca__b)",
                    children: [
                        /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("ellipse", {
                            cx: 129.774,
                            cy: 101.839,
                            fill: "url(#7541ca__c)",
                            rx: 189.141,
                            ry: 178.298
                        }),
                        /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("path", {
                            fill: "url(#7541ca__d)",
                            d: "M1.21 59.333C-.893 16.649 34.11-19.56 79.389-21.542s83.69 31.014 85.792 73.698-32.899 78.892-78.178 80.874c-45.28 1.982-83.69-31.013-85.793-73.697"
                        }),
                        /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("path", {
                            fill: "url(#7541ca__e)",
                            d: "M46.095-64.799c-10.215-5.958-23.156-5.994-33.409-.094L-89.856-5.873c-10.043 5.78-10.06 19.425-.03 25.227L11.977 78.29c10.237 5.922 23.179 5.915 33.407-.02L147.04 19.277c9.976-5.79 9.994-19.358.031-25.17z"
                        }),
                        /*#__PURE__*/ (0, $csRxY$reactjsxruntime.jsx)("path", {
                            fill: "url(#7541ca__f)",
                            d: "M-52.305 79.722c21.136 0 40.28 8.063 54.142 21.072 27.723 26.062 50.706 26.085 78.429.067 13.861-13.076 33.03-21.162 54.189-21.162 42.295.023 76.557 32.233 76.557 71.994s-34.262 71.95-76.557 71.95c-21.207 0-40.375-8.108-54.237-21.184-27.699-25.973-50.658-25.951-78.38.112-13.862 13.031-33.007 21.072-54.143 21.072-42.27 0-76.533-32.21-76.533-71.95 0-39.739 34.262-71.971 76.533-71.971"
                        })
                    ]
                })
            })
        ]
    });
const $3ffec038811e59ad$var$ForwardRef = /*#__PURE__*/ (0, $csRxY$react.forwardRef)($3ffec038811e59ad$var$SvgComponent);
var $3ffec038811e59ad$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3ffec038811e59ad$var$ForwardRef);
