// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_thumbsUp_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $enEbA$jsxs, jsx as $enEbA$jsx} from "react/jsx-runtime";
import {forwardRef as $enEbA$forwardRef} from "react";





const $3b778573adf81725$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $enEbA$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $enEbA$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $enEbA$jsxs)("linearGradient", {
                        id: "d930ce__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $enEbA$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $enEbA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $enEbA$jsxs)("linearGradient", {
                        id: "d930ce__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $enEbA$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $enEbA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $enEbA$jsxs)("linearGradient", {
                        id: "d930ce__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $enEbA$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $enEbA$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $enEbA$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $enEbA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $enEbA$jsxs)("radialGradient", {
                        id: "d930ce__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $enEbA$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $enEbA$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $enEbA$jsx)("clipPath", {
                        id: "d930ce__b",
                        children: /*#__PURE__*/ (0, $enEbA$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $enEbA$jsx)("clipPath", {
                id: "d930ce__a",
                children: /*#__PURE__*/ (0, $enEbA$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M19.274 37.614c1.57.585 3.23.886 4.907.886l-.006-.006h6.982l.043-.003h1.935c.716 0 1.458-.197 2.032-.636.581-.447.968-1.132.968-2.055 0-.298-.043-.825-.298-1.341-.12-.25-.292-.493-.527-.705 1.195-.07 2.204-.935 2.448-2.118.22-1.075-.241-2.107-1.055-2.692 1.11-.275 1.949-1.25 2.009-2.439.049-.963-.421-1.823-1.16-2.327 1.048-.315 1.828-1.261 1.888-2.408.077-1.516-1.132-2.788-2.651-2.788h-7.945c-.433 0-.742-.422-.613-.834l.573-1.84c.513-1.646.84-3.342.98-5.06l.08-.994c.12-1.481-1.052-2.754-2.542-2.754-.983 0-1.877.567-2.301 1.453l-2.669 5.59q-.375.782-.857 1.5l-.066.101c-1.51 2.248-3.876 3.742-6.506 4.167v15.828q.94.204 1.843.541zm-6.85-1.777V20.435h-.717c-1.742 0-3.152 1.41-3.152 3.153v9.09c0 1.744 1.41 3.154 3.152 3.154h.353q.181 0 .363.005"
                })
            }),
            /*#__PURE__*/ (0, $enEbA$jsx)("g", {
                clipPath: "url(#d930ce__a)",
                children: /*#__PURE__*/ (0, $enEbA$jsxs)("g", {
                    clipPath: "url(#d930ce__b)",
                    children: [
                        /*#__PURE__*/ (0, $enEbA$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $enEbA$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#d930ce__c)"
                        }),
                        /*#__PURE__*/ (0, $enEbA$jsx)("path", {
                            fill: "url(#d930ce__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $enEbA$jsx)("path", {
                            fill: "url(#d930ce__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $enEbA$jsx)("path", {
                            fill: "url(#d930ce__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $3b778573adf81725$var$ForwardRef = /*#__PURE__*/ (0, $enEbA$forwardRef)($3b778573adf81725$var$SvgComponent);
var $3b778573adf81725$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($3b778573adf81725$var$ForwardRef);


export {$3b778573adf81725$export$2e2bcd8739ae039 as default};
