// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_threeArrowsCircle_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $joyrj$jsxs, jsx as $joyrj$jsx} from "react/jsx-runtime";
import {forwardRef as $joyrj$forwardRef} from "react";





const $ec97aa368266b458$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $joyrj$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $joyrj$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $joyrj$jsxs)("linearGradient", {
                        id: "121b20__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $joyrj$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $joyrj$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $joyrj$jsxs)("linearGradient", {
                        id: "121b20__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $joyrj$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $joyrj$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $joyrj$jsxs)("linearGradient", {
                        id: "121b20__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $joyrj$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $joyrj$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $joyrj$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $joyrj$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $joyrj$jsxs)("radialGradient", {
                        id: "121b20__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $joyrj$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $joyrj$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $joyrj$jsx)("clipPath", {
                        id: "121b20__b",
                        children: /*#__PURE__*/ (0, $joyrj$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $joyrj$jsx)("clipPath", {
                id: "121b20__a",
                children: /*#__PURE__*/ (0, $joyrj$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M36.33 11.291c.157.672-.26 1.345-.932 1.502l-6.885 1.613c-.672.158-1.344-.26-1.502-.932s.26-1.344.932-1.502l4.08-.955c-2.343-1.467-5.111-2.314-8.08-2.314-4.574 0-8.676 2.012-11.473 5.204-.455.519-1.245.57-1.764.116-.52-.455-.572-1.245-.117-1.764 3.252-3.71 8.03-6.056 13.354-6.056 3.381 0 6.543.946 9.234 2.588l-.894-3.814c-.157-.672.26-1.345.932-1.502s1.345.26 1.502.932zm2.248 8.125c-.206-.659.161-1.36.82-1.566s1.36.16 1.566.82c.525 1.676.807 3.46.807 5.306 0 7.05-4.11 13.138-10.061 16.002l3.808 1.205c.658.208 1.023.91.815 1.568-.208.659-.91 1.023-1.568.815l-6.742-2.132c-.659-.208-1.024-.91-.816-1.568l2.132-6.742c.208-.659.91-1.023 1.569-.815s1.023.91.815 1.568l-1.238 3.915c5.193-2.433 8.786-7.706 8.786-13.816 0-1.59-.243-3.122-.693-4.56M6.306 22.58 3.31 25.135c-.525.448-1.314.385-1.762-.14s-.386-1.314.14-1.762l5.379-4.59c.525-.448 1.314-.386 1.762.14l4.59 5.378c.448.526.385 1.315-.14 1.763s-1.314.385-1.762-.14l-2.706-3.171c-.243 2.675.218 5.447 1.48 8.039 2.004 4.112 5.609 6.92 9.703 8.037.666.181 1.059.869.877 1.535s-.869 1.058-1.535.877c-4.76-1.3-8.96-4.568-11.292-9.354-1.438-2.954-1.983-6.113-1.74-9.167"
                })
            }),
            /*#__PURE__*/ (0, $joyrj$jsx)("g", {
                clipPath: "url(#121b20__a)",
                children: /*#__PURE__*/ (0, $joyrj$jsxs)("g", {
                    clipPath: "url(#121b20__b)",
                    children: [
                        /*#__PURE__*/ (0, $joyrj$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $joyrj$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#121b20__c)"
                        }),
                        /*#__PURE__*/ (0, $joyrj$jsx)("path", {
                            fill: "url(#121b20__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $joyrj$jsx)("path", {
                            fill: "url(#121b20__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $joyrj$jsx)("path", {
                            fill: "url(#121b20__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $ec97aa368266b458$var$ForwardRef = /*#__PURE__*/ (0, $joyrj$forwardRef)($ec97aa368266b458$var$SvgComponent);
var $ec97aa368266b458$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ec97aa368266b458$var$ForwardRef);


export {$ec97aa368266b458$export$2e2bcd8739ae039 as default};
