// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_stampClone_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3loWF$reactjsxruntime = require("react/jsx-runtime");
var $3loWF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $be960c83df0fd183$export$2e2bcd8739ae039);




const $be960c83df0fd183$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2a2756__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2a2756__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2a2756__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2a2756__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("clipPath", {
                        id: "2a2756__b",
                        children: /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("clipPath", {
                id: "2a2756__a",
                children: /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M59.882 27.684c0 4.741-2.78 8.834-6.797 10.736l2.56 13.877h13.164c4.418 0 8 3.582 8 8v3.898c0 1.105-.895 2-2 2H21.188c-1.105 0-2-.895-2-2v-3.898c0-4.418 3.581-8 8-8h13.19l2.56-13.876c-4.02-1.903-6.8-5.995-6.8-10.737 0-6.556 5.316-11.872 11.873-11.872s11.871 5.316 11.871 11.872M24.2 76.19c-1.104 0-2 .895-2 2s.896 2 2 2h47.6c1.104 0 2-.896 2-2s-.896-2-2-2z"
                })
            }),
            /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("g", {
                clipPath: "url(#2a2756__a)",
                children: /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#2a2756__b)",
                    children: [
                        /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#2a2756__c)"
                        }),
                        /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("path", {
                            fill: "url(#2a2756__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("path", {
                            fill: "url(#2a2756__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $3loWF$reactjsxruntime.jsx)("path", {
                            fill: "url(#2a2756__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $be960c83df0fd183$var$ForwardRef = /*#__PURE__*/ (0, $3loWF$react.forwardRef)($be960c83df0fd183$var$SvgComponent);
var $be960c83df0fd183$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($be960c83df0fd183$var$ForwardRef);
