// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_bell_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $2AZDQ$reactjsxruntime = require("react/jsx-runtime");
var $2AZDQ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $642d53c72521f0d5$export$2e2bcd8739ae039);




const $642d53c72521f0d5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b66e63__d",
                        x1: 66.569,
                        x2: 18.461,
                        y1: 3.793,
                        y2: 66.165,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b66e63__e",
                        x1: 37.259,
                        x2: 46.539,
                        y1: -24.922,
                        y2: 34.472,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b66e63__f",
                        x1: -69.82,
                        x2: 86.326,
                        y1: 94.487,
                        y2: 63.627,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b66e63__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 119.937 -119.938 0 80.512 61.076)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("clipPath", {
                        id: "b66e63__b",
                        children: /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("clipPath", {
                id: "b66e63__a",
                children: /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M73.772 61.733c-3.175-4.992-6.448-11.96-6.692-19.72v-6.054h-.002l.002-.078c0-10.538-8.543-19.08-19.08-19.08-10.539 0-19.081 8.543-19.081 19.08l.002.078h-.002v6.054c-.244 7.76-3.517 14.728-6.693 19.72-2.169 3.41.258 7.868 4.3 7.868h14.273V72c0 3.976 3.224 7.2 7.2 7.2s7.2-3.224 7.2-7.2v-2.4h14.274c4.041 0 6.469-4.458 4.3-7.868"
                })
            }),
            /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("g", {
                clipPath: "url(#b66e63__a)",
                children: /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#b66e63__b)",
                    children: [
                        /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("ellipse", {
                            cx: 80.512,
                            cy: 61.076,
                            fill: "url(#b66e63__c)",
                            rx: 119.938,
                            ry: 119.937
                        }),
                        /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("path", {
                            fill: "url(#b66e63__d)",
                            d: "M-1.012 32.482C-2.346 3.77 19.85-20.588 48.562-21.92c28.713-1.334 53.07 20.862 54.403 49.574 1.333 28.713-20.862 53.07-49.575 54.403C24.678 83.39.321 61.195-1.012 32.482"
                        }),
                        /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("path", {
                            fill: "url(#b66e63__e)",
                            d: "M27.449-51.018c-6.478-4.008-14.684-4.032-21.185-.063L-58.76-11.38c-6.369 3.889-6.38 13.067-.02 16.97L5.815 45.235c6.491 3.984 14.698 3.979 21.184-.014l64.46-39.683c6.327-3.894 6.338-13.02.02-16.93z"
                        }),
                        /*#__PURE__*/ (0, $2AZDQ$reactjsxruntime.jsx)("path", {
                            fill: "url(#b66e63__f)",
                            d: "M-34.946 46.197c13.402 0 25.542 5.424 34.332 14.175 17.58 17.531 32.154 17.546 49.733.045 8.79-8.796 20.945-14.235 34.362-14.235 26.82.015 48.546 21.682 48.546 48.429 0 26.746-21.726 48.399-48.546 48.399-13.447 0-25.602-5.455-34.392-14.25-17.564-17.472-32.124-17.457-49.703.075-8.79 8.765-20.93 14.175-34.332 14.175-26.805 0-48.531-21.667-48.531-48.4 0-26.731 21.726-48.413 48.53-48.413"
                        })
                    ]
                })
            })
        ]
    });
const $642d53c72521f0d5$var$ForwardRef = /*#__PURE__*/ (0, $2AZDQ$react.forwardRef)($642d53c72521f0d5$var$SvgComponent);
var $642d53c72521f0d5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($642d53c72521f0d5$var$ForwardRef);
