// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_interaction_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lk841$jsxs, jsx as $lk841$jsx} from "react/jsx-runtime";
import {forwardRef as $lk841$forwardRef} from "react";





const $8496935dc16ea86e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lk841$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lk841$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lk841$jsxs)("linearGradient", {
                        id: "1d463a__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lk841$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $lk841$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lk841$jsxs)("linearGradient", {
                        id: "1d463a__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lk841$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $lk841$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lk841$jsxs)("linearGradient", {
                        id: "1d463a__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lk841$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lk841$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $lk841$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $lk841$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lk841$jsxs)("radialGradient", {
                        id: "1d463a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lk841$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lk841$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lk841$jsx)("clipPath", {
                        id: "1d463a__b",
                        children: /*#__PURE__*/ (0, $lk841$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lk841$jsx)("clipPath", {
                id: "1d463a__a",
                children: /*#__PURE__*/ (0, $lk841$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M27.625 17.066c0 1.551-.368 3.017-1.022 4.313q-.275.108-.546.223l-2.58-4.643-.02-.034c-1.084-1.813-3.67-3.311-6.245-1.916-2.422 1.313-3.148 4.444-1.817 6.697l2.75 4.952h-.112c-5.297 0-9.592-4.294-9.592-9.592 0-5.297 4.295-9.591 9.592-9.591s9.592 4.294 9.592 9.591m-6.84 9.192 3.96 7.128c.11.2-.082.428-.296.346-.672-.258-1.408-.575-2.169-.903-1.4-.603-2.887-1.244-4.219-1.623-1.36-.386-2.655.249-3 1.5-.337 1.228 0 2.25 1.5 3l.42.213c2.166 1.106 9.021 4.605 13.631 4.605 4.97 0 8.95-4.03 8.95-9 0-4.971-2.78-8.818-7.75-8.818-1.86 0-4.952 1.078-6.52 2.093-.125.082-.295.043-.367-.088l-3.614-6.504c-.615-1.03-1.854-1.572-2.907-1-.65.351-1.051 1.016-1.147 1.725-.07.52.023 1.063.304 1.525l3.223 5.801"
                })
            }),
            /*#__PURE__*/ (0, $lk841$jsx)("g", {
                clipPath: "url(#1d463a__a)",
                children: /*#__PURE__*/ (0, $lk841$jsxs)("g", {
                    clipPath: "url(#1d463a__b)",
                    children: [
                        /*#__PURE__*/ (0, $lk841$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $lk841$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#1d463a__c)"
                        }),
                        /*#__PURE__*/ (0, $lk841$jsx)("path", {
                            fill: "url(#1d463a__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $lk841$jsx)("path", {
                            fill: "url(#1d463a__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $lk841$jsx)("path", {
                            fill: "url(#1d463a__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $8496935dc16ea86e$var$ForwardRef = /*#__PURE__*/ (0, $lk841$forwardRef)($8496935dc16ea86e$var$SvgComponent);
var $8496935dc16ea86e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8496935dc16ea86e$var$ForwardRef);


export {$8496935dc16ea86e$export$2e2bcd8739ae039 as default};
