// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_helpCircle_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $2Gfwc$reactjsxruntime = require("react/jsx-runtime");
var $2Gfwc$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9ec6bb7ccab938e8$export$2e2bcd8739ae039);




const $9ec6bb7ccab938e8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "df0257__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "df0257__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "df0257__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsxs)("radialGradient", {
                        id: "df0257__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("clipPath", {
                        id: "df0257__b",
                        children: /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("clipPath", {
                id: "df0257__a",
                children: /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M80 136.016c30.928 0 56-25.072 56-56s-25.072-56-56-56-56 25.072-56 56 25.072 56 56 56m1.594-78.957c-1.378-.281-2.808-.138-4.113.416s-2.438 1.5-3.242 2.731C73.435 61.44 73 62.9 73 64.4c0 2.21-1.79 4-4 4s-4-1.79-4-4c0-3.04.88-6.02 2.538-8.564 1.66-2.544 4.028-4.542 6.818-5.726s5.867-1.495 8.836-.89c2.968.605 5.682 2.095 7.802 4.267s3.552 4.926 4.131 7.911.283 6.079-.856 8.896-3.075 5.241-5.576 6.954c-1.224.838-2.415 1.974-3.267 3.207-.865 1.25-1.22 2.353-1.22 3.203v7.083c0 2.21-1.79 4-4 4-2.208 0-4-1.79-4-4v-7.083c0-2.975 1.182-5.646 2.641-7.755 1.472-2.127 3.394-3.933 5.326-5.256 1.185-.81 2.122-1.974 2.68-3.352.557-1.379.704-2.902.419-4.372-.286-1.47-.988-2.807-2.003-3.848-1.015-1.039-2.295-1.735-3.675-2.017M85 105.016c0 2.485-2.015 4.5-4.5 4.5s-4.5-2.015-4.5-4.5c0-2.486 2.015-4.5 4.5-4.5s4.5 2.014 4.5 4.5"
                })
            }),
            /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("g", {
                clipPath: "url(#df0257__a)",
                children: /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#df0257__b)",
                    children: [
                        /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#df0257__c)"
                        }),
                        /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("path", {
                            fill: "url(#df0257__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("path", {
                            fill: "url(#df0257__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $2Gfwc$reactjsxruntime.jsx)("path", {
                            fill: "url(#df0257__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $9ec6bb7ccab938e8$var$ForwardRef = /*#__PURE__*/ (0, $2Gfwc$react.forwardRef)($9ec6bb7ccab938e8$var$SvgComponent);
var $9ec6bb7ccab938e8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9ec6bb7ccab938e8$var$ForwardRef);
