// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_play_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $dR6TJ$jsxs, jsx as $dR6TJ$jsx} from "react/jsx-runtime";
import {forwardRef as $dR6TJ$forwardRef} from "react";





const $0b7de2ec2be9402e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dR6TJ$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dR6TJ$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dR6TJ$jsxs)("linearGradient", {
                        id: "aad4f6__d",
                        x1: 66.569,
                        x2: 18.461,
                        y1: 3.793,
                        y2: 66.165,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dR6TJ$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dR6TJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dR6TJ$jsxs)("linearGradient", {
                        id: "aad4f6__e",
                        x1: 37.259,
                        x2: 46.539,
                        y1: -24.922,
                        y2: 34.472,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dR6TJ$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dR6TJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dR6TJ$jsxs)("linearGradient", {
                        id: "aad4f6__f",
                        x1: -69.82,
                        x2: 86.326,
                        y1: 94.487,
                        y2: 63.627,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dR6TJ$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dR6TJ$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dR6TJ$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dR6TJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dR6TJ$jsxs)("radialGradient", {
                        id: "aad4f6__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 119.937 -119.938 0 80.512 61.076)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dR6TJ$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dR6TJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dR6TJ$jsx)("clipPath", {
                        id: "aad4f6__b",
                        children: /*#__PURE__*/ (0, $dR6TJ$jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dR6TJ$jsx)("clipPath", {
                id: "aad4f6__a",
                children: /*#__PURE__*/ (0, $dR6TJ$jsx)("path", {
                    fill: "#fff",
                    d: "M48 16.8c-17.23 0-31.2 13.97-31.2 31.2 0 17.232 13.97 31.2 31.2 31.2 17.232 0 31.2-13.968 31.2-31.2 0-17.23-13.968-31.2-31.2-31.2m10.198 33.298-14.4 8.29c-1.6.923-3.6-.234-3.597-2.082l.02-16.616c.002-1.848 2.003-3 3.602-2.074l14.38 8.325c1.6.926 1.596 3.235-.005 4.157"
                })
            }),
            /*#__PURE__*/ (0, $dR6TJ$jsx)("g", {
                clipPath: "url(#aad4f6__a)",
                children: /*#__PURE__*/ (0, $dR6TJ$jsxs)("g", {
                    clipPath: "url(#aad4f6__b)",
                    children: [
                        /*#__PURE__*/ (0, $dR6TJ$jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dR6TJ$jsx)("ellipse", {
                            cx: 80.512,
                            cy: 61.076,
                            fill: "url(#aad4f6__c)",
                            rx: 119.938,
                            ry: 119.937
                        }),
                        /*#__PURE__*/ (0, $dR6TJ$jsx)("path", {
                            fill: "url(#aad4f6__d)",
                            d: "M-1.012 32.482C-2.346 3.77 19.85-20.588 48.562-21.92c28.713-1.334 53.07 20.862 54.403 49.574 1.333 28.713-20.862 53.07-49.575 54.403C24.678 83.39.321 61.195-1.012 32.482"
                        }),
                        /*#__PURE__*/ (0, $dR6TJ$jsx)("path", {
                            fill: "url(#aad4f6__e)",
                            d: "M27.449-51.018c-6.478-4.008-14.684-4.032-21.185-.063L-58.76-11.38c-6.369 3.889-6.38 13.067-.02 16.97L5.815 45.235c6.491 3.984 14.698 3.979 21.184-.014l64.46-39.683c6.327-3.894 6.338-13.02.02-16.93z"
                        }),
                        /*#__PURE__*/ (0, $dR6TJ$jsx)("path", {
                            fill: "url(#aad4f6__f)",
                            d: "M-34.946 46.197c13.402 0 25.542 5.424 34.332 14.175 17.58 17.531 32.154 17.546 49.733.045 8.79-8.796 20.945-14.235 34.362-14.235 26.82.015 48.546 21.682 48.546 48.429 0 26.746-21.726 48.399-48.546 48.399-13.447 0-25.602-5.455-34.392-14.25-17.564-17.472-32.124-17.457-49.703.075-8.79 8.765-20.93 14.175-34.332 14.175-26.805 0-48.531-21.667-48.531-48.4 0-26.731 21.726-48.413 48.53-48.413"
                        })
                    ]
                })
            })
        ]
    });
const $0b7de2ec2be9402e$var$ForwardRef = /*#__PURE__*/ (0, $dR6TJ$forwardRef)($0b7de2ec2be9402e$var$SvgComponent);
var $0b7de2ec2be9402e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($0b7de2ec2be9402e$var$ForwardRef);


export {$0b7de2ec2be9402e$export$2e2bcd8739ae039 as default};
