// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_exportTo_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $bTJEA$reactjsxruntime = require("react/jsx-runtime");
var $bTJEA$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $51c2a7f0912df1f3$export$2e2bcd8739ae039);




const $51c2a7f0912df1f3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7e3628__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7e3628__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7e3628__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsxs)("radialGradient", {
                        id: "7e3628__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("clipPath", {
                        id: "7e3628__b",
                        children: /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("clipPath", {
                id: "7e3628__a",
                children: /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M8.498 15.5c0-2.21 1.79-4 4-4h19.234q.132.013.266.013h.631L22.553 21.589c-1.074 1.074-1.074 2.815 0 3.889s2.815 1.074 3.89 0l10.054-10.055.001.077v20c0 2.21-1.79 4-4 4h-20c-2.21 0-4-1.79-4-4zm15.116 7.15 10.71-10.71 1.927-1.927h-4.253c-.69 0-1.25-.56-1.25-1.25s.56-1.25 1.25-1.25h7.525c.69 0 1.25.56 1.25 1.25v7.525c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-4.763l-2.193 2.193-10.698 10.7q-.092.09-.196.16c-.486.32-1.145.266-1.572-.16-.488-.49-.488-1.28 0-1.769"
                })
            }),
            /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("g", {
                clipPath: "url(#7e3628__a)",
                children: /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#7e3628__b)",
                    children: [
                        /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#7e3628__c)"
                        }),
                        /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("path", {
                            fill: "url(#7e3628__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("path", {
                            fill: "url(#7e3628__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $bTJEA$reactjsxruntime.jsx)("path", {
                            fill: "url(#7e3628__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $51c2a7f0912df1f3$var$ForwardRef = /*#__PURE__*/ (0, $bTJEA$react.forwardRef)($51c2a7f0912df1f3$var$SvgComponent);
var $51c2a7f0912df1f3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($51c2a7f0912df1f3$var$ForwardRef);
