// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_infiniteLooping_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $98irm$jsxs, jsx as $98irm$jsx} from "react/jsx-runtime";
import {forwardRef as $98irm$forwardRef} from "react";





const $5d491a66086ac44a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $98irm$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $98irm$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $98irm$jsxs)("linearGradient", {
                        id: "1848bb__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98irm$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $98irm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98irm$jsxs)("linearGradient", {
                        id: "1848bb__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98irm$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $98irm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98irm$jsxs)("linearGradient", {
                        id: "1848bb__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98irm$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $98irm$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $98irm$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $98irm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98irm$jsxs)("radialGradient", {
                        id: "1848bb__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $98irm$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $98irm$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $98irm$jsx)("clipPath", {
                        id: "1848bb__b",
                        children: /*#__PURE__*/ (0, $98irm$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $98irm$jsx)("clipPath", {
                id: "1848bb__a",
                children: /*#__PURE__*/ (0, $98irm$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M72.33 72.502c2.21-.005 3.996-1.8 3.99-4.009l-.056-24.865c-.005-2.21-1.8-3.996-4.01-3.991s-3.996 1.8-3.99 4.009l.035 15.114c-6.64-6.98-16.024-11.337-26.427-11.337-20.141 0-36.47 16.328-36.47 36.47s16.328 36.47 36.47 36.47c8.355 0 16.065-2.814 22.215-7.545q.14-.107.268-.224l.12-.091c4.531-3.557 8.748-8.72 12.042-13.442 3.31-4.745 5.903-9.354 7.118-12.036.945-2.084 3.285-7.04 6.38-12.245 2.665-4.483 5.7-8.819 8.703-11.737q.14-.106.272-.227c5.059-4.596 11.768-7.393 19.14-7.393 15.723 0 28.47 12.747 28.47 28.47 0 15.724-12.747 28.47-28.47 28.47-9.615 0-18.118-4.762-23.28-12.075-1.273-1.805-3.769-2.235-5.574-.962-1.804 1.274-2.235 3.77-.961 5.575 6.595 9.345 17.49 15.462 29.815 15.462 20.141 0 36.47-16.328 36.47-36.47s-16.329-36.47-36.47-36.47c-8.343 0-16.038 2.806-22.182 7.52q-.267.15-.514.345c-4.756 3.75-9.057 9.957-12.295 15.403-3.305 5.556-5.781 10.806-6.79 13.032-.968 2.136-3.293 6.317-6.392 10.76-3.115 4.464-6.801 8.885-10.422 11.727q-.122.096-.235.2l-.09.067c-4.801 3.693-10.808 5.886-17.338 5.886-15.723 0-28.47-12.746-28.47-28.47s12.747-28.47 28.47-28.47c8.24 0 15.664 3.498 20.867 9.101l-15.293.036c-2.21.005-3.996 1.8-3.99 4.009.004 2.209 1.8 3.996 4.008 3.99z"
                })
            }),
            /*#__PURE__*/ (0, $98irm$jsx)("g", {
                clipPath: "url(#1848bb__a)",
                children: /*#__PURE__*/ (0, $98irm$jsxs)("g", {
                    clipPath: "url(#1848bb__b)",
                    children: [
                        /*#__PURE__*/ (0, $98irm$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $98irm$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#1848bb__c)"
                        }),
                        /*#__PURE__*/ (0, $98irm$jsx)("path", {
                            fill: "url(#1848bb__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $98irm$jsx)("path", {
                            fill: "url(#1848bb__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $98irm$jsx)("path", {
                            fill: "url(#1848bb__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $5d491a66086ac44a$var$ForwardRef = /*#__PURE__*/ (0, $98irm$forwardRef)($5d491a66086ac44a$var$SvgComponent);
var $5d491a66086ac44a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5d491a66086ac44a$var$ForwardRef);


export {$5d491a66086ac44a$export$2e2bcd8739ae039 as default};
