// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_translate_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $hpJSU$reactjsxruntime = require("react/jsx-runtime");
var $hpJSU$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c81b442373b54a0c$export$2e2bcd8739ae039);




const $c81b442373b54a0c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bafc0c__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bafc0c__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bafc0c__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsxs)("radialGradient", {
                        id: "bafc0c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("clipPath", {
                        id: "bafc0c__b",
                        children: /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("clipPath", {
                id: "bafc0c__a",
                children: /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M28.14 10.643h-8.1V9.125c0-.69-.56-1.25-1.25-1.25-.691 0-1.25.56-1.25 1.25v1.518H9.151c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25H23.49c-.546 3.227-2.353 5.858-4.623 7.853-.503.443-1.026.85-1.558 1.224-1.03-1.808-1.757-3.532-1.978-4.785-.12-.68-.768-1.134-1.448-1.014s-1.134.768-1.014 1.448c.29 1.644 1.17 3.672 2.303 5.65-1.465.765-2.9 1.272-4.103 1.514-.676.136-1.115.795-.978 1.472.136.677.795 1.115 1.472.979 1.486-.3 3.207-.916 4.934-1.837 1.67 2.48 3.724 4.848 5.563 6.037.58.374 1.354.208 1.729-.372.374-.58.208-1.354-.372-1.728-1.397-.903-3.195-2.898-4.784-5.24.646-.447 1.278-.937 1.884-1.47 2.68-2.355 4.93-5.615 5.503-9.73h2.12c.69 0 1.25-.56 1.25-1.25 0-.691-.56-1.25-1.25-1.25m-.261 23.174-2.35 5.545c-.27.636-1.004.933-1.64.663-.635-.269-.932-1.002-.663-1.638l7.23-17.057q.034-.084.082-.164.056-.096.128-.177c.12-.14.275-.257.458-.334q.242-.101.488-.1.246-.001.488.1c.182.077.337.193.458.334q.133.155.21.341l7.23 17.057c.269.636-.028 1.37-.664 1.639s-1.37-.028-1.639-.664l-2.35-5.545zm3.732-8.808-2.674 6.308h5.348z"
                })
            }),
            /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("g", {
                clipPath: "url(#bafc0c__a)",
                children: /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#bafc0c__b)",
                    children: [
                        /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#bafc0c__c)"
                        }),
                        /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("path", {
                            fill: "url(#bafc0c__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("path", {
                            fill: "url(#bafc0c__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $hpJSU$reactjsxruntime.jsx)("path", {
                            fill: "url(#bafc0c__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $c81b442373b54a0c$var$ForwardRef = /*#__PURE__*/ (0, $hpJSU$react.forwardRef)($c81b442373b54a0c$var$SvgComponent);
var $c81b442373b54a0c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c81b442373b54a0c$var$ForwardRef);
