// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_explosion_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $2swtq$reactjsxruntime = require("react/jsx-runtime");
var $2swtq$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $999cdc5a2b9e1a71$export$2e2bcd8739ae039);




const $999cdc5a2b9e1a71$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsxs)("linearGradient", {
                        id: "90ff61__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsxs)("linearGradient", {
                        id: "90ff61__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsxs)("linearGradient", {
                        id: "90ff61__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsxs)("radialGradient", {
                        id: "90ff61__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("clipPath", {
                        id: "90ff61__b",
                        children: /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("clipPath", {
                id: "90ff61__a",
                children: /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M32.513 26.206c-3.146-5.824 4.07-11.705 9.139-7.446l35.67 29.971 20.785-22.948c4.14-4.572 11.656-.67 10.3 5.346l-5.967 26.478h45.746c6.925 0 8.343 9.788 1.703 11.753l-39.652 11.737 12.026 14.157c3.962 4.664-.782 11.571-6.558 9.546l-18.324-6.422v39.044c0 6.661-9.234 8.361-11.606 2.136l-14.589-38.28c-9.388 5.399-22.184 12.657-30.308 17.255-5.635 3.19-11.534-3.526-7.842-8.711 4.73-6.645 11.689-16.462 16.837-23.915l-27.677-7.103c-6.065-1.557-5.99-10.197.1-11.649l28.654-6.827z"
                })
            }),
            /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("g", {
                clipPath: "url(#90ff61__a)",
                children: /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#90ff61__b)",
                    children: [
                        /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#90ff61__c)"
                        }),
                        /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("path", {
                            fill: "url(#90ff61__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("path", {
                            fill: "url(#90ff61__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $2swtq$reactjsxruntime.jsx)("path", {
                            fill: "url(#90ff61__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $999cdc5a2b9e1a71$var$ForwardRef = /*#__PURE__*/ (0, $2swtq$react.forwardRef)($999cdc5a2b9e1a71$var$SvgComponent);
var $999cdc5a2b9e1a71$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($999cdc5a2b9e1a71$var$ForwardRef);
