// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_dropToUpload_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $3JYI6$jsxs, jsx as $3JYI6$jsx} from "react/jsx-runtime";
import {forwardRef as $3JYI6$forwardRef} from "react";





const $b843ba8f6900ba82$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3JYI6$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3JYI6$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3JYI6$jsxs)("linearGradient", {
                        id: "a700d8__d",
                        x1: 33.108,
                        x2: 7.568,
                        y1: -0.218,
                        y2: 32.894,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3JYI6$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3JYI6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3JYI6$jsxs)("linearGradient", {
                        id: "a700d8__e",
                        x1: 17.549,
                        x2: 22.475,
                        y1: -15.463,
                        y2: 16.068,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3JYI6$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3JYI6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3JYI6$jsxs)("linearGradient", {
                        id: "a700d8__f",
                        x1: -39.297,
                        x2: 43.598,
                        y1: 47.93,
                        y2: 31.547,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3JYI6$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3JYI6$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3JYI6$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3JYI6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3JYI6$jsxs)("radialGradient", {
                        id: "a700d8__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.158 35.35)scale(63.6724)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3JYI6$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3JYI6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3JYI6$jsx)("clipPath", {
                        id: "a700d8__b",
                        children: /*#__PURE__*/ (0, $3JYI6$jsx)("rect", {
                            width: 44.1,
                            height: 44.1,
                            x: 1.199,
                            y: 1.199,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3JYI6$jsx)("clipPath", {
                id: "a700d8__a",
                children: /*#__PURE__*/ (0, $3JYI6$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M15.523 6.58h8.221c.843 0 1.652.335 2.248.931l8.736 8.736c.596.596.931 1.405.931 2.248v12.46c0 1.756-1.423 3.18-3.18 3.18H15.524c-1.756 0-3.18-1.424-3.18-3.18V9.76c0-1.755 1.424-3.179 3.18-3.179m9.299 9.188c.001-.586-.472-1.061-1.057-1.062-.586-.002-1.061.472-1.063 1.057l-.023 10.495-3.583-2.85c-.458-.364-1.125-.288-1.49.17-.364.458-.288 1.125.17 1.489l5.338 4.245c.389.31.94.307 1.326-.004l5.26-4.246c.455-.367.527-1.035.159-1.49s-1.035-.527-1.49-.16l-3.57 2.882zM6.72 22.658c0-.53.43-.96.96-.96h1.764c.53 0 .96-.43.96-.96s-.43-.96-.96-.96H7.681c-1.59 0-2.88 1.289-2.88 2.88v1.66c0 .53.43.96.96.96s.96-.43.96-.96zm31.836-2.88c-.53 0-.96.43-.96.96s.43.96.96.96h1.764c.53 0 .96.43.96.96v1.66c0 .53.43.96.96.96s.96-.43.96-.96v-1.66c0-1.591-1.29-2.88-2.88-2.88zm4.644 8.899c0-.53-.43-.96-.96-.96s-.96.43-.96.96v3.32c0 .531.43.96.96.96s.96-.429.96-.96zm-36.48 0c0-.53-.43-.96-.96-.96s-.96.43-.96.96v3.32c0 .531.43.96.96.96s.96-.429.96-.96zm0 7.68c0-.53-.43-.96-.96-.96s-.96.43-.96.96v1.66c0 1.59 1.29 2.88 2.88 2.88h1.764c.53 0 .96-.43.96-.96s-.43-.96-.96-.96H7.681c-.53 0-.96-.43-.96-.96zm36.48 0c0-.53-.43-.96-.96-.96s-.96.43-.96.96v1.66c0 .53-.43.96-.96.96h-1.764c-.53 0-.96.43-.96.96s.43.96.96.96h1.764c1.59 0 2.88-1.29 2.88-2.88zm-29.124 2.62c-.53 0-.96.43-.96.96s.43.96.96.96h3.528c.53 0 .96-.43.96-.96s-.43-.96-.96-.96zm8.16 0c-.53 0-.96.43-.96.96s.43.96.96.96h3.528c.53 0 .96-.43.96-.96s-.43-.96-.96-.96zm8.16 0c-.53 0-.96.43-.96.96s.43.96.96.96h3.528c.53 0 .96-.43.96-.96s-.43-.96-.96-.96z"
                })
            }),
            /*#__PURE__*/ (0, $3JYI6$jsx)("g", {
                clipPath: "url(#a700d8__a)",
                children: /*#__PURE__*/ (0, $3JYI6$jsxs)("g", {
                    clipPath: "url(#a700d8__b)",
                    children: [
                        /*#__PURE__*/ (0, $3JYI6$jsx)("rect", {
                            width: 44.1,
                            height: 44.1,
                            x: 1.199,
                            y: 1.199,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3JYI6$jsx)("circle", {
                            cx: 40.508,
                            cy: 30.192,
                            r: 63.672,
                            fill: "url(#a700d8__c)"
                        }),
                        /*#__PURE__*/ (0, $3JYI6$jsx)("path", {
                            fill: "url(#a700d8__d)",
                            d: "M-2.77 15.012C-3.478-.23 8.305-13.16 23.548-13.869S51.722-2.794 52.43 12.449 41.354 40.623 26.111 41.33-2.062 30.255-2.77 15.012"
                        }),
                        /*#__PURE__*/ (0, $3JYI6$jsx)("path", {
                            fill: "url(#a700d8__e)",
                            d: "M12.34-29.317c-3.438-2.128-7.795-2.14-11.246-.034l-34.52 21.077c-3.38 2.064-3.386 6.937-.01 9.009L.856 21.782c3.446 2.115 7.803 2.112 11.246-.008L46.323.708c3.358-2.068 3.364-6.913.01-8.989z"
                        }),
                        /*#__PURE__*/ (0, $3JYI6$jsx)("path", {
                            fill: "url(#a700d8__f)",
                            d: "M-20.783 22.293c7.115 0 13.56 2.88 18.226 7.525 9.332 9.308 17.07 9.315 26.402.024 4.666-4.67 11.12-7.557 18.242-7.557 14.238.008 25.772 11.51 25.772 25.71S56.325 73.69 42.087 73.69c-7.139 0-13.592-2.895-18.258-7.565-9.324-9.275-17.054-9.267-26.386.04-4.666 4.653-11.111 7.525-18.226 7.525-14.23 0-25.765-11.502-25.765-25.694s11.534-25.702 25.765-25.702"
                        })
                    ]
                })
            })
        ]
    });
const $b843ba8f6900ba82$var$ForwardRef = /*#__PURE__*/ (0, $3JYI6$forwardRef)($b843ba8f6900ba82$var$SvgComponent);
var $b843ba8f6900ba82$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b843ba8f6900ba82$var$ForwardRef);


export {$b843ba8f6900ba82$export$2e2bcd8739ae039 as default};
