// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_apps_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lqp5K$jsxs, jsx as $lqp5K$jsx} from "react/jsx-runtime";
import {forwardRef as $lqp5K$forwardRef} from "react";





const $7dd0e0afd65997e2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lqp5K$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lqp5K$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lqp5K$jsxs)("linearGradient", {
                        id: "0d95ef__d",
                        x1: 65.27,
                        x2: 23.067,
                        y1: 10.816,
                        y2: 68.861,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lqp5K$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $lqp5K$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lqp5K$jsxs)("linearGradient", {
                        id: "0d95ef__e",
                        x1: 37.538,
                        x2: 45.362,
                        y1: -14.796,
                        y2: 38.321,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lqp5K$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $lqp5K$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lqp5K$jsxs)("linearGradient", {
                        id: "0d95ef__f",
                        x1: -63.784,
                        x2: 83.268,
                        y1: 91.712,
                        y2: 60.882,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lqp5K$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lqp5K$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $lqp5K$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $lqp5K$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lqp5K$jsxs)("radialGradient", {
                        id: "0d95ef__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 106.979 -113.484 0 78.461 61.91)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lqp5K$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lqp5K$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lqp5K$jsx)("clipPath", {
                        id: "0d95ef__b",
                        children: /*#__PURE__*/ (0, $lqp5K$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 13.199,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lqp5K$jsx)("clipPath", {
                id: "0d95ef__a",
                children: /*#__PURE__*/ (0, $lqp5K$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M54.185 13.008h19.182c5.302 0 9.6 4.298 9.6 9.6V41.79c0 5.302-4.298 9.6-9.6 9.6H54.185c-5.302 0-9.6-4.298-9.6-9.6V22.608c0-5.302 4.298-9.6 9.6-9.6M21.431 24.684h9.912c4.64 0 8.4 3.76 8.4 8.4v9.912c0 4.639-3.76 8.4-8.4 8.4h-9.912c-4.639 0-8.4-3.761-8.4-8.4v-9.912c0-4.64 3.761-8.4 8.4-8.4m9.912 31.596h-9.912c-4.639 0-8.4 3.76-8.4 8.4v9.912c0 4.639 3.761 8.4 8.4 8.4h9.912c4.64 0 8.4-3.761 8.4-8.4V64.68c0-4.64-3.76-8.4-8.4-8.4m21.642 0h9.912c4.64 0 8.4 3.76 8.4 8.4v9.912c0 4.639-3.76 8.4-8.4 8.4h-9.912c-4.639 0-8.4-3.761-8.4-8.4V64.68c0-4.64 3.761-8.4 8.4-8.4"
                })
            }),
            /*#__PURE__*/ (0, $lqp5K$jsx)("g", {
                clipPath: "url(#0d95ef__a)",
                children: /*#__PURE__*/ (0, $lqp5K$jsxs)("g", {
                    clipPath: "url(#0d95ef__b)",
                    children: [
                        /*#__PURE__*/ (0, $lqp5K$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 13.199,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $lqp5K$jsx)("ellipse", {
                            cx: 78.461,
                            cy: 61.911,
                            fill: "url(#0d95ef__c)",
                            rx: 113.484,
                            ry: 106.979
                        }),
                        /*#__PURE__*/ (0, $lqp5K$jsx)("path", {
                            fill: "url(#0d95ef__d)",
                            d: "M1.325 36.406C.064 10.796 21.065-10.93 48.233-12.12 75.4-13.31 98.447 6.489 99.708 32.099s-19.74 47.336-46.907 48.525C25.633 81.814 2.587 62.016 1.325 36.406"
                        }),
                        /*#__PURE__*/ (0, $lqp5K$jsx)("path", {
                            fill: "url(#0d95ef__e)",
                            d: "M28.256-38.073c-6.129-3.575-13.894-3.597-20.045-.056L-53.314-2.717C-59.34.75-59.35 8.937-53.333 12.419L7.787 47.78c6.141 3.553 13.906 3.55 20.043-.012l60.993-35.395C94.81 8.899 94.82.758 88.842-2.73z"
                        }),
                        /*#__PURE__*/ (0, $lqp5K$jsx)("path", {
                            fill: "url(#0d95ef__f)",
                            d: "M-30.787 48.638c12.682 0 24.169 4.839 32.485 12.644 16.634 15.637 30.424 15.65 47.058.04 8.316-7.845 19.818-12.697 32.513-12.697 25.377.013 45.934 19.34 45.934 43.197s-20.557 43.169-45.934 43.169c-12.724 0-24.225-4.865-32.542-12.71-16.62-15.584-30.395-15.571-47.029.067-8.316 7.819-19.803 12.643-32.485 12.643-25.362 0-45.92-19.326-45.92-43.17 0-23.843 20.558-43.183 45.92-43.183"
                        })
                    ]
                })
            })
        ]
    });
const $7dd0e0afd65997e2$var$ForwardRef = /*#__PURE__*/ (0, $lqp5K$forwardRef)($7dd0e0afd65997e2$var$SvgComponent);
var $7dd0e0afd65997e2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7dd0e0afd65997e2$var$ForwardRef);


export {$7dd0e0afd65997e2$export$2e2bcd8739ae039 as default};
