// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_bell_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $5solc$reactjsxruntime = require("react/jsx-runtime");
var $5solc$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $50bbc5666fb43d86$export$2e2bcd8739ae039);




const $50bbc5666fb43d86$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5solc$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4c5be4__d",
                        x1: 110.95,
                        x2: 30.77,
                        y1: 6.324,
                        y2: 110.278,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4c5be4__e",
                        x1: 62.1,
                        x2: 77.567,
                        y1: -41.534,
                        y2: 57.456,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4c5be4__f",
                        x1: -116.369,
                        x2: 143.875,
                        y1: 157.48,
                        y2: 106.047,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4c5be4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.195 117.99)scale(199.896)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("clipPath", {
                        id: "4c5be4__b",
                        children: /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 10.773,
                            y: 10.775,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("clipPath", {
                id: "4c5be4__a",
                children: /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M122.955 102.887c-5.292-8.32-10.748-19.933-11.153-32.867V59.93h-.004q.002-.064.004-.13C111.802 42.238 97.564 28 80 28c-17.563 0-31.8 14.238-31.8 31.801l.002.13H48.2V70.02c-.406 12.934-5.861 24.547-11.154 32.867-3.615 5.683.43 13.113 7.166 13.113h23.79v4c0 6.627 5.372 12 12 12 6.627 0 12-5.373 12-12v-4h23.789c6.735 0 10.781-7.43 7.165-13.113"
                })
            }),
            /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("g", {
                clipPath: "url(#4c5be4__a)",
                children: /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#4c5be4__b)",
                    children: [
                        /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 10.773,
                            y: 10.775,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("circle", {
                            cx: 134.185,
                            cy: 101.794,
                            r: 199.896,
                            fill: "url(#4c5be4__c)"
                        }),
                        /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("path", {
                            fill: "url(#4c5be4__d)",
                            d: "M-1.686 54.14c-2.222-47.855 34.77-88.45 82.624-90.672 47.855-2.222 88.449 34.77 90.671 82.625s-34.77 88.449-82.624 90.671-88.449-34.77-90.67-82.625"
                        }),
                        /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("path", {
                            fill: "url(#4c5be4__e)",
                            d: "M45.75-85.028c-10.795-6.68-24.473-6.72-35.308-.105L-97.931-18.964c-10.614 6.48-10.632 21.777-.032 28.282L9.694 75.394C20.513 82.033 34.19 82.025 45 75.37L152.436 9.233c10.543-6.491 10.562-21.702.032-28.218z"
                        }),
                        /*#__PURE__*/ (0, $5solc$reactjsxruntime.jsx)("path", {
                            fill: "url(#4c5be4__f)",
                            d: "M-58.246 76.996c22.337 0 42.57 9.04 57.22 23.625 29.3 29.219 53.59 29.244 82.889.075 14.649-14.66 34.908-23.725 57.27-23.725 44.7.025 80.91 36.137 80.91 80.715s-36.21 80.665-80.91 80.665c-22.412 0-42.671-9.091-57.32-23.75-29.275-29.12-53.54-29.095-82.839.125-14.65 14.609-34.883 23.625-57.22 23.625-44.675 0-80.885-36.112-80.885-80.665s36.21-80.69 80.885-80.69"
                        })
                    ]
                })
            })
        ]
    });
const $50bbc5666fb43d86$var$ForwardRef = /*#__PURE__*/ (0, $5solc$react.forwardRef)($50bbc5666fb43d86$var$SvgComponent);
var $50bbc5666fb43d86$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($50bbc5666fb43d86$var$ForwardRef);
