// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_layers_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $7MMdU$reactjsxruntime = require("react/jsx-runtime");
var $7MMdU$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $bf555e934c550385$export$2e2bcd8739ae039);




const $bf555e934c550385$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "11984b__d",
                        x1: 109.277,
                        x2: 41.838,
                        y1: 19.392,
                        y2: 112.146,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "11984b__e",
                        x1: 64.961,
                        x2: 77.463,
                        y1: -21.535,
                        y2: 63.345,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "11984b__f",
                        x1: -96.944,
                        x2: 138.04,
                        y1: 148.659,
                        y2: 99.394,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsxs)("radialGradient", {
                        id: "11984b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 170.948 -181.344 0 130.355 101.038)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("clipPath", {
                        id: "11984b__b",
                        children: /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.199,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("clipPath", {
                id: "11984b__a",
                children: /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m110.262 79.956.007.005 17.42-11.88c4.66-3.18 4.66-10.04 0-13.22l-43.18-29.45c-2.72-1.85-6.3-1.85-9.01 0L32.32 54.86c-4.66 3.18-4.66 10.04 0 13.22l17.42 11.88.008-.005L75.5 97.52c2.72 1.85 6.3 1.85 9.01 0zm-30.263 26.965c-3.23 0-6.35-.96-9.01-2.78l-28.35-19.34-10.32 7.04c-4.66 3.18-4.66 10.04 0 13.22l43.18 29.45c2.72 1.85 6.3 1.85 9.01 0l43.18-29.45c4.66-3.18 4.66-10.04 0-13.22l-10.32-7.04-28.35 19.34c-2.67 1.82-5.78 2.78-9.01 2.78z"
                })
            }),
            /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("g", {
                clipPath: "url(#11984b__a)",
                children: /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#11984b__b)",
                    children: [
                        /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.199,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("ellipse", {
                            cx: 130.355,
                            cy: 101.038,
                            fill: "url(#11984b__c)",
                            rx: 181.344,
                            ry: 170.948
                        }),
                        /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("path", {
                            fill: "url(#11984b__d)",
                            d: "M7.095 60.284C5.08 19.359 38.638-15.356 82.051-17.257s80.24 29.735 82.256 70.66c2.016 40.923-31.543 75.639-74.956 77.54-43.413 1.9-80.24-29.735-82.256-70.66"
                        }),
                        /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("path", {
                            fill: "url(#11984b__e)",
                            d: "M50.129-58.73c-9.794-5.713-22.202-5.748-32.032-.09L-80.218-2.233c-9.629 5.542-9.645 18.623-.03 24.186L17.42 78.46c9.814 5.678 22.223 5.671 32.03-.02l97.464-56.56c9.564-5.551 9.582-18.56.029-24.131z"
                        }),
                        /*#__PURE__*/ (0, $7MMdU$reactjsxruntime.jsx)("path", {
                            fill: "url(#11984b__f)",
                            d: "M-44.215 79.83c20.264 0 38.62 7.731 51.91 20.204 26.58 24.988 48.615 25.009 75.195.064 13.29-12.537 31.668-20.29 51.955-20.29 40.551.022 73.401 30.904 73.401 69.027s-32.85 68.984-73.401 68.984c-20.332 0-38.71-7.774-52-20.311-26.557-24.903-48.57-24.881-75.15.107-13.29 12.494-31.646 20.204-51.91 20.204-40.529 0-73.378-30.883-73.378-68.984s32.85-69.005 73.378-69.005"
                        })
                    ]
                })
            })
        ]
    });
const $bf555e934c550385$var$ForwardRef = /*#__PURE__*/ (0, $7MMdU$react.forwardRef)($bf555e934c550385$var$SvgComponent);
var $bf555e934c550385$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($bf555e934c550385$var$ForwardRef);
