// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_cloudUpload_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $alC74$reactjsxruntime = require("react/jsx-runtime");
var $alC74$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e988af5318c7f8e1$export$2e2bcd8739ae039);




const $e988af5318c7f8e1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $alC74$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsxs)("linearGradient", {
                        id: "381d10__d",
                        x1: 109.277,
                        x2: 41.838,
                        y1: 19.392,
                        y2: 112.146,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsxs)("linearGradient", {
                        id: "381d10__e",
                        x1: 64.961,
                        x2: 77.463,
                        y1: -21.535,
                        y2: 63.345,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsxs)("linearGradient", {
                        id: "381d10__f",
                        x1: -96.944,
                        x2: 138.04,
                        y1: 148.659,
                        y2: 99.394,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsxs)("radialGradient", {
                        id: "381d10__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 170.948 -181.344 0 130.355 101.038)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("clipPath", {
                        id: "381d10__b",
                        children: /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.199,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("clipPath", {
                id: "381d10__a",
                children: /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M117.542 54.65q-.009.162-.01.32c13.37.79 23.98 11.85 23.98 26.52s-11.38 26.56-25.43 26.56h-32.57V74.07l17.5 14.04c.74.59 1.62.88 2.5.88 1.17 0 2.33-.52 3.12-1.5 1.38-1.72 1.1-4.24-.62-5.62l-23.67-19c-1.47-1.18-3.57-1.17-5.03.02l-23.33 19c-1.72 1.4-1.98 3.92-.58 5.63 1.4 1.72 3.92 1.98 5.63.58l16.47-13.42v33.37h8V136c0 2.21-1.79 4-4 4s-4-1.79-4-4v-27.95h-39.03v-.02c-9.95-.17-17.98-8.64-17.98-19.08 0-8.79 5.7-16.18 13.45-18.4-.35-1.61-.56-3.28-.56-5.01 0-12.26 9.51-22.19 21.24-22.19 2.87 0 5.6.6 8.1 1.68 3.77-12.24 14.75-21.1 27.72-21.1 16.08 0 29.11 13.61 29.11 30.4q-.001.158-.01.32"
                })
            }),
            /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("g", {
                clipPath: "url(#381d10__a)",
                children: /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#381d10__b)",
                    children: [
                        /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.199,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("ellipse", {
                            cx: 130.355,
                            cy: 101.038,
                            fill: "url(#381d10__c)",
                            rx: 181.344,
                            ry: 170.948
                        }),
                        /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("path", {
                            fill: "url(#381d10__d)",
                            d: "M7.095 60.284C5.08 19.359 38.638-15.356 82.051-17.257s80.24 29.735 82.256 70.66c2.016 40.923-31.543 75.639-74.956 77.54-43.413 1.9-80.24-29.735-82.256-70.66"
                        }),
                        /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("path", {
                            fill: "url(#381d10__e)",
                            d: "M50.129-58.73c-9.794-5.713-22.202-5.748-32.032-.09L-80.218-2.233c-9.629 5.542-9.645 18.623-.03 24.186L17.42 78.46c9.814 5.678 22.223 5.671 32.03-.02l97.464-56.56c9.564-5.551 9.582-18.56.029-24.131z"
                        }),
                        /*#__PURE__*/ (0, $alC74$reactjsxruntime.jsx)("path", {
                            fill: "url(#381d10__f)",
                            d: "M-44.215 79.83c20.264 0 38.62 7.731 51.91 20.204 26.58 24.988 48.615 25.009 75.195.064 13.29-12.537 31.668-20.29 51.955-20.29 40.551.022 73.401 30.904 73.401 69.027s-32.85 68.984-73.401 68.984c-20.332 0-38.71-7.774-52-20.311-26.557-24.903-48.57-24.881-75.15.107-13.29 12.494-31.646 20.204-51.91 20.204-40.529 0-73.378-30.883-73.378-68.984s32.85-69.005 73.378-69.005"
                        })
                    ]
                })
            })
        ]
    });
const $e988af5318c7f8e1$var$ForwardRef = /*#__PURE__*/ (0, $alC74$react.forwardRef)($e988af5318c7f8e1$var$SvgComponent);
var $e988af5318c7f8e1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e988af5318c7f8e1$var$ForwardRef);
