// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_channel_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $htIQ6$reactjsxruntime = require("react/jsx-runtime");
var $htIQ6$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $68bf88e5730b8be7$export$2e2bcd8739ae039);




const $68bf88e5730b8be7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "cd0ab2__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "cd0ab2__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsxs)("linearGradient", {
                        id: "cd0ab2__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsxs)("radialGradient", {
                        id: "cd0ab2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("clipPath", {
                        id: "cd0ab2__b",
                        children: /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("clipPath", {
                id: "cd0ab2__a",
                children: /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M14.312 11.547c0 1.38 1.119 2.5 2.5 2.5l2.625 4.548c-1.233 1.042-2.105 2.499-2.4 4.155h-5.25c-.433-.747-1.24-1.25-2.166-1.25-1.38 0-2.5 1.12-2.5 2.5s1.12 2.5 2.5 2.5c.925 0 1.733-.503 2.166-1.25h5.25c.296 1.656 1.167 3.113 2.4 4.155l-2.626 4.548c-1.38 0-2.5 1.12-2.5 2.5s1.12 2.5 2.5 2.5c1.381 0 2.5-1.12 2.5-2.5 0-.455-.121-.883-.334-1.25l2.625-4.547c.749.27 1.557.418 2.4.418.842 0 1.65-.148 2.4-.418l2.624 4.547c-.212.368-.334.795-.334 1.25 0 1.38 1.12 2.5 2.5 2.5s2.5-1.12 2.5-2.5-1.12-2.5-2.5-2.5l-2.627-4.549c1.233-1.042 2.105-2.498 2.4-4.154h5.251c.433.747 1.24 1.25 2.166 1.25 1.38 0 2.5-1.12 2.5-2.5s-1.12-2.5-2.5-2.5c-.925 0-1.733.503-2.166 1.25h-5.25c-.296-1.656-1.168-3.113-2.4-4.155l2.625-4.548c1.382 0 2.5-1.12 2.5-2.5s-1.118-2.5-2.5-2.5c-1.38 0-2.5 1.12-2.5 2.5 0 .455.123.882.335 1.25l-2.625 4.546c-.749-.27-1.557-.417-2.4-.417-.842 0-1.65.147-2.4.417l-2.624-4.546c.213-.367.334-.795.334-1.25 0-1.38-1.119-2.5-2.5-2.5-1.38 0-2.5 1.12-2.5 2.5"
                })
            }),
            /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("g", {
                clipPath: "url(#cd0ab2__a)",
                children: /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#cd0ab2__b)",
                    children: [
                        /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#cd0ab2__c)"
                        }),
                        /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("path", {
                            fill: "url(#cd0ab2__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("path", {
                            fill: "url(#cd0ab2__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $htIQ6$reactjsxruntime.jsx)("path", {
                            fill: "url(#cd0ab2__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $68bf88e5730b8be7$var$ForwardRef = /*#__PURE__*/ (0, $htIQ6$react.forwardRef)($68bf88e5730b8be7$var$SvgComponent);
var $68bf88e5730b8be7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($68bf88e5730b8be7$var$ForwardRef);
