// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_apps_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gyo8E$reactjsxruntime = require("react/jsx-runtime");
var $gyo8E$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6de75ab6beff77f7$export$2e2bcd8739ae039);




const $6de75ab6beff77f7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsxs)("linearGradient", {
                        id: "697d2b__d",
                        x1: 32.635,
                        x2: 11.534,
                        y1: 5.409,
                        y2: 34.431,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsxs)("linearGradient", {
                        id: "697d2b__e",
                        x1: 18.767,
                        x2: 22.679,
                        y1: -7.398,
                        y2: 19.161,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsxs)("linearGradient", {
                        id: "697d2b__f",
                        x1: -31.892,
                        x2: 41.634,
                        y1: 45.856,
                        y2: 30.441,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsxs)("radialGradient", {
                        id: "697d2b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.4895 -56.7422 0 39.23 30.954)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("clipPath", {
                        id: "697d2b__b",
                        children: /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 6.6,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("clipPath", {
                id: "697d2b__a",
                children: /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M27.093 6.504h9.59c2.651 0 4.8 2.149 4.8 4.8v9.59c0 2.652-2.148 4.8-4.8 4.8h-9.59c-2.651 0-4.8-2.148-4.8-4.8v-9.59c0-2.651 2.149-4.8 4.8-4.8m-16.377 5.838h4.956c2.32 0 4.2 1.88 4.2 4.2v4.956c0 2.32-1.88 4.2-4.2 4.2h-4.956c-2.32 0-4.2-1.88-4.2-4.2v-4.956c0-2.32 1.88-4.2 4.2-4.2m4.956 15.798h-4.956c-2.32 0-4.2 1.88-4.2 4.2v4.956c0 2.32 1.88 4.2 4.2 4.2h4.956c2.32 0 4.2-1.88 4.2-4.2V32.34c0-2.32-1.88-4.2-4.2-4.2m10.82 0h4.957c2.32 0 4.2 1.88 4.2 4.2v4.956c0 2.32-1.88 4.2-4.2 4.2h-4.956c-2.32 0-4.2-1.88-4.2-4.2V32.34c0-2.32 1.88-4.2 4.2-4.2"
                })
            }),
            /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("g", {
                clipPath: "url(#697d2b__a)",
                children: /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#697d2b__b)",
                    children: [
                        /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 6.6,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("ellipse", {
                            cx: 39.23,
                            cy: 30.954,
                            fill: "url(#697d2b__c)",
                            rx: 56.742,
                            ry: 53.489
                        }),
                        /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("path", {
                            fill: "url(#697d2b__d)",
                            d: "M.663 18.204C.032 5.399 10.533-5.464 24.116-6.06c13.584-.594 25.107 9.304 25.738 22.11.63 12.805-9.87 23.667-23.454 24.262-13.583.594-25.107-9.304-25.737-22.11"
                        }),
                        /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("path", {
                            fill: "url(#697d2b__e)",
                            d: "M14.126-19.036c-3.064-1.788-6.947-1.799-10.022-.029L-26.66-1.359c-3.013 1.734-3.018 5.828-.01 7.568L3.892 23.89c3.071 1.777 6.954 1.775 10.022-.006L44.41 6.186c2.993-1.737 2.998-5.807.009-7.55z"
                        }),
                        /*#__PURE__*/ (0, $gyo8E$reactjsxruntime.jsx)("path", {
                            fill: "url(#697d2b__f)",
                            d: "M-15.393 24.32c6.34 0 12.084 2.418 16.242 6.321 8.317 7.819 15.212 7.825 23.529.02 4.158-3.923 9.909-6.349 16.257-6.349 12.688.007 22.967 9.67 22.967 21.599s-10.28 21.585-22.967 21.585c-6.362 0-12.113-2.433-16.271-6.355-8.31-7.792-15.198-7.786-23.515.033-4.158 3.91-9.902 6.322-16.242 6.322-12.682 0-22.96-9.663-22.96-21.585s10.278-21.592 22.96-21.592"
                        })
                    ]
                })
            })
        ]
    });
const $6de75ab6beff77f7$var$ForwardRef = /*#__PURE__*/ (0, $gyo8E$react.forwardRef)($6de75ab6beff77f7$var$SvgComponent);
var $6de75ab6beff77f7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6de75ab6beff77f7$var$ForwardRef);
