// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_imageStack_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2wmDF$jsxs, jsx as $2wmDF$jsx} from "react/jsx-runtime";
import {forwardRef as $2wmDF$forwardRef} from "react";





const $f36f33d6a3061ec2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2wmDF$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2wmDF$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2wmDF$jsxs)("linearGradient", {
                        id: "ca8798__d",
                        x1: 113.619,
                        x2: 42.958,
                        y1: 16.384,
                        y2: 113.569,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2wmDF$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2wmDF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2wmDF$jsxs)("linearGradient", {
                        id: "ca8798__e",
                        x1: 67.187,
                        x2: 80.286,
                        y1: -26.497,
                        y2: 62.437,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2wmDF$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2wmDF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2wmDF$jsxs)("linearGradient", {
                        id: "ca8798__f",
                        x1: -102.452,
                        x2: 143.757,
                        y1: 151.827,
                        y2: 100.208,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2wmDF$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2wmDF$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2wmDF$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2wmDF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2wmDF$jsxs)("radialGradient", {
                        id: "ca8798__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 179.114 -190.006 0 135.705 101.93)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2wmDF$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2wmDF$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2wmDF$jsx)("clipPath", {
                        id: "ca8798__b",
                        children: /*#__PURE__*/ (0, $2wmDF$jsx)("rect", {
                            width: 131.6,
                            height: 124.056,
                            x: 18.398,
                            y: 20.373,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2wmDF$jsx)("clipPath", {
                id: "ca8798__a",
                children: /*#__PURE__*/ (0, $2wmDF$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M125.43 48.47v49c0 8.841-7.16 16.001-16 16.001h-74c-8.84 0-16-7.16-16-16v-49c0-8.84 7.16-16 16-16h74c8.84 0 16 7.16 16 16m-14.04 6.88c0-4.82-3.91-8.73-8.73-8.73s-8.73 3.91-8.73 8.73 3.91 8.73 8.73 8.73 8.73-3.91 8.73-8.73M28.98 96.22c-2.56 3.991.3 9.241 5.05 9.241v-.01h76.18c4.75 0 7.61-5.25 5.05-9.24l-11.78-18.35c-2.36-3.68-7.74-3.68-10.1 0L80.02 98.68 58.76 65.56c-2.36-3.68-7.74-3.68-10.1 0zM136.57 66.49c0-2.21 1.79-4 4-4h.01c2.21 0 4 1.79 4 4v41.051c0 13.23-10.77 24-24 24l-65.94-.05c-2.21 0-4-1.79-4-4s1.79-4 4-4l65.93.05c8.82 0 16-7.18 16-16z"
                })
            }),
            /*#__PURE__*/ (0, $2wmDF$jsx)("g", {
                clipPath: "url(#ca8798__a)",
                children: /*#__PURE__*/ (0, $2wmDF$jsxs)("g", {
                    clipPath: "url(#ca8798__b)",
                    children: [
                        /*#__PURE__*/ (0, $2wmDF$jsx)("rect", {
                            width: 131.6,
                            height: 124.056,
                            x: 18.398,
                            y: 20.373,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2wmDF$jsx)("ellipse", {
                            cx: 135.705,
                            cy: 101.93,
                            fill: "url(#ca8798__c)",
                            rx: 190.006,
                            ry: 179.114
                        }),
                        /*#__PURE__*/ (0, $2wmDF$jsx)("path", {
                            fill: "url(#ca8798__d)",
                            d: "M6.556 59.229C4.444 16.35 39.606-20.024 85.092-22.015s84.073 31.155 86.185 74.034-33.05 79.253-78.536 81.244S8.668 102.108 6.556 59.229"
                        }),
                        /*#__PURE__*/ (0, $2wmDF$jsx)("path", {
                            fill: "url(#ca8798__e)",
                            d: "M51.646-65.47c-10.262-5.985-23.262-6.022-33.562-.094L-84.927-6.274c-10.089 5.807-10.106 19.513-.03 25.342l102.33 59.206c10.284 5.949 23.285 5.942 33.56-.021l102.12-59.262c10.022-5.816 10.04-19.446.03-25.284z"
                        }),
                        /*#__PURE__*/ (0, $2wmDF$jsx)("path", {
                            fill: "url(#ca8798__f)",
                            d: "M-47.204 79.71c21.232 0 40.464 8.1 54.389 21.169 27.85 26.181 50.938 26.204 78.787.067 13.925-13.135 33.182-21.258 54.437-21.258 42.489.022 76.907 32.38 76.907 72.323 0 39.944-34.418 72.279-76.907 72.279-21.303 0-40.56-8.145-54.484-21.281-27.826-26.092-50.89-26.07-78.74.112-13.925 13.091-33.157 21.169-54.39 21.169-42.464 0-76.883-32.358-76.883-72.279s34.42-72.301 76.884-72.301"
                        })
                    ]
                })
            })
        ]
    });
const $f36f33d6a3061ec2$var$ForwardRef = /*#__PURE__*/ (0, $2wmDF$forwardRef)($f36f33d6a3061ec2$var$SvgComponent);
var $f36f33d6a3061ec2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f36f33d6a3061ec2$var$ForwardRef);


export {$f36f33d6a3061ec2$export$2e2bcd8739ae039 as default};
