// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_EmptyStateExport_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $hhQ6Y$reactjsxruntime = require("react/jsx-runtime");
var $hhQ6Y$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7f082910a821318d$export$2e2bcd8739ae039);




const $7f082910a821318d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d98257__d",
                        x1: 113.572,
                        x2: 38.961,
                        y1: 11.788,
                        y2: 114.429,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d98257__e",
                        x1: 64.53,
                        x2: 78.36,
                        y1: -33.495,
                        y2: 60.419,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d98257__f",
                        x1: -114.649,
                        x2: 145.4,
                        y1: 154.812,
                        y2: 100.279,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d98257__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 189.14 -200.691 0 136.902 102.123)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("clipPath", {
                        id: "d98257__b",
                        children: /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("rect", {
                            width: 139,
                            height: 131,
                            x: 13,
                            y: 16,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("clipPath", {
                id: "d98257__a",
                children: /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M111.999 16h20c6.627 0 12 5.373 12 12v44c0 6.627-5.373 12-12 12h-20c-6.628 0-12-5.373-12-12V28c0-6.627 5.372-12 12-12m-54 84h44c6.627 0 12 5.373 12 12v20c0 6.627-5.373 12-12 12h-44c-6.628 0-12-5.373-12-12v-20c0-6.627 5.372-12 12-12m8.25-28.53 8.26 20.12c1.34 3.25 5.92 3.32 7.35.11l2.41-5.4c.4-.9 1.12-1.62 2.02-2.02l5.39-2.41c3.21-1.43 3.14-6.02-.11-7.35l-20.11-8.27c-3.29-1.35-6.57 1.93-5.22 5.22zm-9.83 3.88c2.21 0 4 1.79 4 4s-1.79 4-4 4h-20.45c-1.49 3.06-4.61 5.17-8.24 5.17-5.07 0-9.17-4.11-9.17-9.17 0-3.63 2.12-6.75 5.17-8.24V35.9c-3.06-1.49-5.17-4.61-5.17-8.24 0-5.07 4.11-9.17 9.17-9.17 3.63 0 6.75 2.12 8.24 5.17h35.06c1.49-3.06 4.61-5.17 8.24-5.17 5.07 0 9.17 4.11 9.17 9.17 0 3.63-2.12 6.75-5.17 8.24v20.5c0 2.21-1.79 4-4 4s-4-1.79-4-4V35.9c-1.85-.9-3.34-2.4-4.24-4.24h-35.06c-.9 1.85-2.4 3.34-4.24 4.24v35.21c1.85.9 3.34 2.4 4.24 4.24z"
                })
            }),
            /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("g", {
                clipPath: "url(#d98257__a)",
                children: /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#d98257__b)",
                    children: [
                        /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("rect", {
                            width: 139,
                            height: 131,
                            x: 13,
                            y: 16,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("ellipse", {
                            cx: 136.902,
                            cy: 102.123,
                            fill: "url(#d98257__c)",
                            rx: 200.691,
                            ry: 189.14
                        }),
                        /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("path", {
                            fill: "url(#d98257__d)",
                            d: "M.488 57.03C-1.743 11.752 35.396-26.66 83.44-28.761c48.044-2.102 88.8 32.9 91.031 78.179s-34.908 83.69-82.953 85.792C43.476 137.312 2.72 102.31.488 57.031"
                        }),
                        /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("path", {
                            fill: "url(#d98257__e)",
                            d: "M48.115-74.648c-10.838-6.321-24.57-6.36-35.449-.1L-96.138-12.14c-10.656 6.132-10.674 20.605-.032 26.76l108.085 62.52c10.862 6.282 24.594 6.275 35.447-.022L155.225 14.54c10.585-6.142 10.604-20.534.032-26.7z"
                        }),
                        /*#__PURE__*/ (0, $hhQ6Y$reactjsxruntime.jsx)("path", {
                            fill: "url(#d98257__f)",
                            d: "M-56.294 78.658c22.426 0 42.74 8.554 57.448 22.354 29.415 27.647 53.802 27.671 83.218.071 14.707-13.87 35.047-22.448 57.498-22.448 44.877.023 81.232 34.192 81.232 76.372s-36.355 76.325-81.232 76.325c-22.502 0-42.841-8.602-57.549-22.472-29.39-27.553-53.752-27.529-83.167.118-14.708 13.823-35.022 22.354-57.448 22.354-44.853 0-81.207-34.169-81.207-76.325s36.354-76.349 81.207-76.349"
                        })
                    ]
                })
            })
        ]
    });
const $7f082910a821318d$var$ForwardRef = /*#__PURE__*/ (0, $hhQ6Y$react.forwardRef)($7f082910a821318d$var$SvgComponent);
var $7f082910a821318d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7f082910a821318d$var$ForwardRef);
