// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_EmptyStateExport_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $g2olN$jsxs, jsx as $g2olN$jsx} from "react/jsx-runtime";
import {forwardRef as $g2olN$forwardRef} from "react";





const $3d37444acebdd226$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $g2olN$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $g2olN$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $g2olN$jsxs)("linearGradient", {
                        id: "de4de9__d",
                        x1: 68.145,
                        x2: 23.379,
                        y1: 7.071,
                        y2: 68.656,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g2olN$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $g2olN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g2olN$jsxs)("linearGradient", {
                        id: "de4de9__e",
                        x1: 38.718,
                        x2: 47.016,
                        y1: -20.098,
                        y2: 36.25,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g2olN$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $g2olN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g2olN$jsxs)("linearGradient", {
                        id: "de4de9__f",
                        x1: -68.787,
                        x2: 87.242,
                        y1: 92.886,
                        y2: 60.166,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g2olN$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $g2olN$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $g2olN$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $g2olN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g2olN$jsxs)("radialGradient", {
                        id: "de4de9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 113.484 -120.414 0 82.141 61.273)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $g2olN$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $g2olN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $g2olN$jsx)("clipPath", {
                        id: "de4de9__b",
                        children: /*#__PURE__*/ (0, $g2olN$jsx)("rect", {
                            width: 83.4,
                            height: 78.6,
                            x: 7.801,
                            y: 9.6,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $g2olN$jsx)("clipPath", {
                id: "de4de9__a",
                children: /*#__PURE__*/ (0, $g2olN$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M67.2 9.6h12c3.977 0 7.2 3.223 7.2 7.2v26.4c0 3.976-3.223 7.2-7.2 7.2h-12c-3.976 0-7.2-3.224-7.2-7.2V16.8c0-3.977 3.224-7.2 7.2-7.2M34.8 60h26.4c3.977 0 7.2 3.223 7.2 7.2v12c0 3.976-3.223 7.2-7.2 7.2H34.8c-3.976 0-7.2-3.224-7.2-7.2v-12c0-3.977 3.224-7.2 7.2-7.2m4.951-17.118 4.956 12.072c.804 1.95 3.552 1.992 4.41.066l1.446-3.24c.24-.54.672-.973 1.212-1.212l3.234-1.447c1.926-.858 1.884-3.611-.066-4.41L42.877 39.75c-1.974-.81-3.942 1.158-3.132 3.133zm-5.898 2.328c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4h-12.27c-.894 1.836-2.766 3.102-4.944 3.102-3.042 0-5.502-2.466-5.502-5.502 0-2.178 1.272-4.05 3.102-4.944V21.54c-1.836-.894-3.102-2.766-3.102-4.944 0-3.042 2.466-5.502 5.502-5.502 2.178 0 4.05 1.272 4.944 3.102h21.036c.894-1.836 2.766-3.102 4.944-3.102 3.042 0 5.502 2.466 5.502 5.502 0 2.178-1.272 4.05-3.102 4.944v12.3c0 1.326-1.074 2.4-2.4 2.4s-2.4-1.074-2.4-2.4v-12.3c-1.11-.54-2.004-1.44-2.544-2.544H21.583c-.54 1.11-1.44 2.004-2.544 2.544v21.126c1.11.54 2.004 1.44 2.544 2.544z"
                })
            }),
            /*#__PURE__*/ (0, $g2olN$jsx)("g", {
                clipPath: "url(#de4de9__a)",
                children: /*#__PURE__*/ (0, $g2olN$jsxs)("g", {
                    clipPath: "url(#de4de9__b)",
                    children: [
                        /*#__PURE__*/ (0, $g2olN$jsx)("rect", {
                            width: 83.4,
                            height: 78.6,
                            x: 7.801,
                            y: 9.6,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $g2olN$jsx)("ellipse", {
                            cx: 82.141,
                            cy: 61.273,
                            fill: "url(#de4de9__c)",
                            rx: 120.414,
                            ry: 113.484
                        }),
                        /*#__PURE__*/ (0, $g2olN$jsx)("path", {
                            fill: "url(#de4de9__d)",
                            d: "M.295 34.217C-1.043 7.049 21.24-15.997 50.067-17.26c28.827-1.261 53.28 19.74 54.619 46.908 1.338 27.167-20.945 50.213-49.772 51.475C26.087 82.385 1.634 61.384.295 34.217"
                        }),
                        /*#__PURE__*/ (0, $g2olN$jsx)("path", {
                            fill: "url(#de4de9__e)",
                            d: "M28.869-44.79c-6.503-3.792-14.742-3.815-21.27-.06L-57.682-7.284c-6.393 3.679-6.404 12.363-.02 16.056L7.15 46.284c6.517 3.77 14.757 3.765 21.268-.013L93.135 8.723c6.351-3.685 6.363-12.32.02-16.02z"
                        }),
                        /*#__PURE__*/ (0, $g2olN$jsx)("path", {
                            fill: "url(#de4de9__f)",
                            d: "M-33.774 47.194c13.455 0 25.644 5.132 34.469 13.412 17.649 16.589 32.281 16.603 49.93.043 8.825-8.323 21.028-13.47 34.5-13.47 26.926.015 48.738 20.516 48.738 45.824s-21.812 45.795-48.739 45.795c-13.5 0-25.704-5.161-34.529-13.483-17.634-16.532-32.251-16.518-49.9.07-8.825 8.295-21.014 13.413-34.47 13.413-26.91 0-48.723-20.502-48.723-45.795s21.812-45.81 48.724-45.81"
                        })
                    ]
                })
            })
        ]
    });
const $3d37444acebdd226$var$ForwardRef = /*#__PURE__*/ (0, $g2olN$forwardRef)($3d37444acebdd226$var$SvgComponent);
var $3d37444acebdd226$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($3d37444acebdd226$var$ForwardRef);


export {$3d37444acebdd226$export$2e2bcd8739ae039 as default};
