// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_education_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $btgP4$reactjsxruntime = require("react/jsx-runtime");
var $btgP4$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f8220d0c991aa3e5$export$2e2bcd8739ae039);




const $f8220d0c991aa3e5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6a6f9c__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6a6f9c__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6a6f9c__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6a6f9c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("clipPath", {
                        id: "6a6f9c__b",
                        children: /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("clipPath", {
                id: "6a6f9c__a",
                children: /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m18.526 30.379 27.96-11.422c.97-.396 2.056-.396 3.025 0l27.96 11.421c3.316 1.355 3.316 6.052 0 7.406l-3.903 1.595v13.068c0 1.662-.444 4.073-2.136 6.594-1.697 2.526-4.569 5.048-9.242 7.051-3.756 1.61-8.1 2.233-11.905 2.374-3.813.142-7.255-.194-9.292-.614-1.082-.222-1.778-1.28-1.555-2.362.222-1.082 1.28-1.778 2.362-1.555 1.68.345 4.797.665 8.336.534 3.546-.132 7.347-.712 10.478-2.054 4.089-1.752 6.3-3.822 7.497-5.604 1.2-1.788 1.457-3.402 1.457-4.364V41.013l-20.057 8.193c-.97.396-2.055.396-3.025 0L33.354 43.84c.51-.718 1.18-1.328 1.978-1.771l13.595-7.553c.966-.536 1.314-1.754.777-2.72-.536-.965-1.754-1.313-2.72-.776L33.39 38.573c-1.609.894-2.908 2.194-3.798 3.732l-11.066-4.52c-3.316-1.355-3.316-6.052 0-7.406m9.72 33.466v-16.53c0-1.791.48-3.514 1.346-5.01l3.762 1.536c-.709.998-1.108 2.207-1.108 3.474v16.098c1.688 2.882 3.513 6.735 3.853 9.973q.05.345.05.702v.005c0 2.898-2.496 5.247-5.575 5.247s-5.576-2.35-5.576-5.247q0-.295.034-.581c.258-3.106 1.76-6.8 3.214-9.667"
                })
            }),
            /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6a6f9c__a)",
                children: /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6a6f9c__b)",
                    children: [
                        /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#6a6f9c__c)"
                        }),
                        /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("path", {
                            fill: "url(#6a6f9c__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("path", {
                            fill: "url(#6a6f9c__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $btgP4$reactjsxruntime.jsx)("path", {
                            fill: "url(#6a6f9c__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $f8220d0c991aa3e5$var$ForwardRef = /*#__PURE__*/ (0, $btgP4$react.forwardRef)($f8220d0c991aa3e5$var$SvgComponent);
var $f8220d0c991aa3e5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f8220d0c991aa3e5$var$ForwardRef);
