// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_gift_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $jkCK9$reactjsxruntime = require("react/jsx-runtime");
var $jkCK9$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a6fb9594917b6173$export$2e2bcd8739ae039);




const $a6fb9594917b6173$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "44ae19__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "44ae19__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "44ae19__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsxs)("radialGradient", {
                        id: "44ae19__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("clipPath", {
                        id: "44ae19__b",
                        children: /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("clipPath", {
                id: "44ae19__a",
                children: /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M14.357 12.342c0-2.95 2.392-5.342 5.342-5.342 1.696 0 3.352.79 4.33 2.023C25.01 7.79 26.376 7 28.073 7c2.95 0 5.342 2.392 5.342 5.342 0 1.126-.348 2.17-.942 3.03h-4.369c1.66-.016 3-1.367 3-3.03 0-1.673-1.358-3.03-3.031-3.03-1.666 0-2.73 1.344-2.742 3.008v3.053l-2.6-.002V12.32c-.013-1.664-1.365-3.008-3.031-3.008-1.674 0-3.03 1.357-3.03 3.03 0 1.663 1.34 3.014 2.999 3.03h-4.37c-.593-.86-.941-1.904-.941-3.03M8 17.692c0-1.277 1.035-2.312 2.312-2.312h12.425v9.536H10.312C9.035 24.916 8 23.881 8 22.604zm32.075 0c0-1.277-1.035-2.312-2.312-2.312H25.339v9.536h12.426c1.276 0 2.311-1.035 2.311-2.312zm-14.738 9.536h12.715v7.224c0 2.873-2.329 5.201-5.201 5.201h-7.514zm-15.314 0h12.715v12.425h-7.513c-2.873 0-5.202-2.328-5.202-5.201z"
                })
            }),
            /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("g", {
                clipPath: "url(#44ae19__a)",
                children: /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#44ae19__b)",
                    children: [
                        /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#44ae19__c)"
                        }),
                        /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("path", {
                            fill: "url(#44ae19__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("path", {
                            fill: "url(#44ae19__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $jkCK9$reactjsxruntime.jsx)("path", {
                            fill: "url(#44ae19__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $a6fb9594917b6173$var$ForwardRef = /*#__PURE__*/ (0, $jkCK9$react.forwardRef)($a6fb9594917b6173$var$SvgComponent);
var $a6fb9594917b6173$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a6fb9594917b6173$var$ForwardRef);
