// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fileText_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $iWAds$jsxs, jsx as $iWAds$jsx} from "react/jsx-runtime";
import {forwardRef as $iWAds$forwardRef} from "react";





const $e4dc23b3bdb52396$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iWAds$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iWAds$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iWAds$jsxs)("linearGradient", {
                        id: "1f6183__d",
                        x1: 66.569,
                        x2: 18.461,
                        y1: 3.793,
                        y2: 66.165,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iWAds$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $iWAds$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iWAds$jsxs)("linearGradient", {
                        id: "1f6183__e",
                        x1: 37.259,
                        x2: 46.539,
                        y1: -24.921,
                        y2: 34.473,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iWAds$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $iWAds$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iWAds$jsxs)("linearGradient", {
                        id: "1f6183__f",
                        x1: -69.82,
                        x2: 86.326,
                        y1: 94.487,
                        y2: 63.627,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iWAds$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iWAds$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $iWAds$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $iWAds$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iWAds$jsxs)("radialGradient", {
                        id: "1f6183__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 119.937 -119.938 0 80.512 61.076)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iWAds$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iWAds$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iWAds$jsx)("clipPath", {
                        id: "1f6183__b",
                        children: /*#__PURE__*/ (0, $iWAds$jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iWAds$jsx)("clipPath", {
                id: "1f6183__a",
                children: /*#__PURE__*/ (0, $iWAds$jsx)("path", {
                    fill: "#fff",
                    d: "M72.293 38.692 52.51 18.91c-1.35-1.35-3.181-2.11-5.09-2.11H28.801c-3.977 0-7.2 3.224-7.2 7.2v48c0 3.977 3.223 7.2 7.2 7.2h38.4c3.976 0 7.2-3.223 7.2-7.2V43.784c0-1.91-.759-3.74-2.11-5.091m-9.891 11.709c0 1.325-1.075 2.4-2.4 2.4-1.326 0-2.4-1.075-2.4-2.4V48h-7.2v19.2h2.4c1.325 0 2.4 1.074 2.4 2.4s-1.075 2.4-2.4 2.4h-9.6c-1.326 0-2.4-1.075-2.4-2.4 0-1.326 1.074-2.4 2.4-2.4h2.4V48h-7.2v2.4c0 1.325-1.075 2.4-2.4 2.4-1.326 0-2.4-1.075-2.4-2.4v-4.8c0-1.326 1.074-2.4 2.4-2.4h24c1.325 0 2.4 1.074 2.4 2.4z"
                })
            }),
            /*#__PURE__*/ (0, $iWAds$jsx)("g", {
                clipPath: "url(#1f6183__a)",
                children: /*#__PURE__*/ (0, $iWAds$jsxs)("g", {
                    clipPath: "url(#1f6183__b)",
                    children: [
                        /*#__PURE__*/ (0, $iWAds$jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 6.465,
                            y: 6.465,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $iWAds$jsx)("ellipse", {
                            cx: 80.512,
                            cy: 61.076,
                            fill: "url(#1f6183__c)",
                            rx: 119.938,
                            ry: 119.937
                        }),
                        /*#__PURE__*/ (0, $iWAds$jsx)("path", {
                            fill: "url(#1f6183__d)",
                            d: "M-1.012 32.482C-2.346 3.77 19.85-20.588 48.562-21.92c28.713-1.334 53.07 20.862 54.403 49.574 1.333 28.713-20.862 53.07-49.575 54.403C24.678 83.39.321 61.195-1.012 32.482"
                        }),
                        /*#__PURE__*/ (0, $iWAds$jsx)("path", {
                            fill: "url(#1f6183__e)",
                            d: "M27.449-51.017c-6.478-4.008-14.684-4.032-21.185-.063L-58.76-11.38c-6.369 3.889-6.38 13.067-.02 16.97L5.815 45.236c6.491 3.984 14.698 3.979 21.184-.014l64.46-39.683c6.327-3.894 6.338-13.02.02-16.93z"
                        }),
                        /*#__PURE__*/ (0, $iWAds$jsx)("path", {
                            fill: "url(#1f6183__f)",
                            d: "M-34.946 46.197c13.402 0 25.542 5.424 34.332 14.175 17.58 17.531 32.154 17.546 49.733.045 8.79-8.796 20.945-14.235 34.362-14.235 26.82.015 48.546 21.682 48.546 48.429 0 26.746-21.726 48.399-48.546 48.399-13.447 0-25.602-5.455-34.392-14.25-17.564-17.472-32.124-17.457-49.703.075-8.79 8.765-20.93 14.175-34.332 14.175-26.805 0-48.531-21.667-48.531-48.4 0-26.731 21.726-48.413 48.53-48.413"
                        })
                    ]
                })
            })
        ]
    });
const $e4dc23b3bdb52396$var$ForwardRef = /*#__PURE__*/ (0, $iWAds$forwardRef)($e4dc23b3bdb52396$var$SvgComponent);
var $e4dc23b3bdb52396$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e4dc23b3bdb52396$var$ForwardRef);


export {$e4dc23b3bdb52396$export$2e2bcd8739ae039 as default};
