// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lighten_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $2ozJP$reactjsxruntime = require("react/jsx-runtime");
var $2ozJP$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $fa97b35f9bece410$export$2e2bcd8739ae039);




const $fa97b35f9bece410$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 97,
        height: 96,
        viewBox: "0 0 97 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d7588b__d",
                        x1: 70.403,
                        x2: 14.807,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d7588b__e",
                        x1: 36.531,
                        x2: 47.256,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d7588b__f",
                        x1: -87.219,
                        x2: 93.232,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d7588b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.7 74.813)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("clipPath", {
                        id: "d7588b__b",
                        children: /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("clipPath", {
                id: "d7588b__a",
                children: /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M48.941 13.234c1.105 0 2 .896 2 2v4.645c0 1.104-.895 2-2 2s-2-.896-2-2v-4.645c0-1.104.896-2 2-2M65.49 48.001c0 9.14-7.41 16.549-16.549 16.549-9.14 0-16.549-7.41-16.549-16.55 0-9.139 7.41-16.548 16.549-16.548 9.14 0 16.549 7.41 16.549 16.549M50.94 76.12c0-1.104-.895-2-2-2s-2 .896-2 2v4.645c0 1.104.896 2 2 2s2-.896 2-2zM83.707 48c0 1.105-.895 2-2 2h-4.644c-1.105 0-2-.895-2-2s.895-2 2-2h4.644c1.105 0 2 .895 2 2M20.82 50c1.105 0 2-.895 2-2s-.895-2-2-2h-4.644c-1.105 0-2 .895-2 2s.895 2 2 2zm52.704-26.584c.781.781.781 2.047 0 2.828L70.24 29.53c-.781.78-2.047.78-2.828 0-.782-.781-.782-2.048 0-2.829l3.284-3.284c.78-.78 2.047-.78 2.828 0M30.471 69.298c.78-.781.78-2.048 0-2.829-.782-.78-2.048-.78-2.829 0l-3.284 3.285c-.781.78-.781 2.047 0 2.828.78.781 2.047.781 2.828 0zm-6.113-45.882c-.781.781-.781 2.047 0 2.828l3.284 3.285c.78.78 2.047.78 2.828 0 .781-.781.781-2.048 0-2.829l-3.284-3.284c-.781-.78-2.047-.78-2.828 0M67.41 69.298c-.78-.781-.78-2.048 0-2.829.781-.78 2.048-.78 2.829 0l3.284 3.285c.781.78.781 2.047 0 2.828s-2.047.781-2.828 0z"
                })
            }),
            /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("g", {
                clipPath: "url(#d7588b__a)",
                children: /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#d7588b__b)",
                    children: [
                        /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("circle", {
                            cx: 86.513,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#d7588b__c)"
                        }),
                        /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("path", {
                            fill: "url(#d7588b__d)",
                            d: "M-7.698 30.067c-1.54-33.182 24.11-61.33 57.291-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.541 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.87-57.291"
                        }),
                        /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("path", {
                            fill: "url(#d7588b__e)",
                            d: "M25.194-66.43c-7.485-4.632-16.97-4.66-24.482-.073l-75.146 45.881c-7.36 4.494-7.372 15.1-.022 19.61L.193 44.806c7.502 4.604 16.986 4.598 24.481-.016L99.17-1.071c7.311-4.5 7.324-15.048.023-19.566z"
                        }),
                        /*#__PURE__*/ (0, $2ozJP$reactjsxruntime.jsx)("path", {
                            fill: "url(#d7588b__f)",
                            d: "M-46.917 45.918c15.488 0 29.518 6.268 39.676 16.381 20.316 20.26 37.159 20.278 57.474.052 10.158-10.165 24.206-16.45 39.711-16.45 30.995.017 56.103 25.056 56.103 55.967S120.939 157.8 89.944 157.8c-15.54 0-29.588-6.303-39.745-16.468-20.299-20.191-37.124-20.174-57.44.087-10.158 10.13-24.188 16.381-39.676 16.381-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $fa97b35f9bece410$var$ForwardRef = /*#__PURE__*/ (0, $2ozJP$react.forwardRef)($fa97b35f9bece410$var$SvgComponent);
var $fa97b35f9bece410$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($fa97b35f9bece410$var$ForwardRef);
