// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_flagCheckmark_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $dVGli$jsxs, jsx as $dVGli$jsx} from "react/jsx-runtime";
import {forwardRef as $dVGli$forwardRef} from "react";





const $896d5c26126567dd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dVGli$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dVGli$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dVGli$jsxs)("linearGradient", {
                        id: "8de5e6__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dVGli$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dVGli$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dVGli$jsxs)("linearGradient", {
                        id: "8de5e6__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dVGli$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dVGli$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dVGli$jsxs)("linearGradient", {
                        id: "8de5e6__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dVGli$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dVGli$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dVGli$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dVGli$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dVGli$jsxs)("radialGradient", {
                        id: "8de5e6__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dVGli$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dVGli$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dVGli$jsx)("clipPath", {
                        id: "8de5e6__b",
                        children: /*#__PURE__*/ (0, $dVGli$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dVGli$jsx)("clipPath", {
                id: "8de5e6__a",
                children: /*#__PURE__*/ (0, $dVGli$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M37.277 24.25c2.21 0 4 1.79 4 4v6.645h69.445c6.628 0 12 5.372 12 12v41.958c0 6.628-5.372 12-12 12H41.277v30.897c0 2.209-1.79 4-4 4s-4-1.791-4-4V28.25c0-2.21 1.791-4 4-4m64.802 35.827c1.632-1.488 1.75-4.018.261-5.65s-4.018-1.75-5.65-.262l-1.178 1.073-.066.06-.068.062-.14.128-.147.134-.153.14-.323.294-.346.315-.368.335-.047.043-.048.044-.048.044-.048.044-.197.179-.832.758-1.861 1.697-2.066 1.883-.54.492-.273.248-.069.063-.068.063-.069.062-.069.063-.138.125-.069.063-.069.064-.069.063-.07.063-.138.126-.56.51-.28.256-.07.064-.07.064-.071.064-.07.065-.567.515-.07.065-.072.065-.07.064-.071.065-.284.259-.071.064-.072.065-.142.13-.284.259-.568.518-.567.516-.282.258-.07.064-.071.064-.07.064-.07.064-.282.256-.28.255-.555.507-.07.063-.069.063-.137.125-.274.25-.273.249-.136.123-.067.062-.068.062-.134.122-.135.123-.066.06-.067.061-.067.061-.066.06-1.043.95-.127.117-.063.057-.064.058-.25.229-.249.226-.122.112-.061.055-.061.056-.06.055-.061.055-.12.11-.06.054-.06.055-.118.108-.235.213-.058.053-.057.053-.058.052-.057.052-.114.104-.056.052-.057.05-.222.204-.219.199-.214.195-.81.739-.227.206-9.261-8.638c-1.616-1.506-4.147-1.418-5.654.198s-1.418 4.146.197 5.653l11.96 11.153c1.523 1.42 3.882 1.434 5.422.03l-2.695-2.956 2.695 2.956.004-.004.013-.011.052-.047.2-.184.07-.063.039-.035.02-.018.02-.018.176-.161.206-.188.233-.213.26-.236.285-.26.643-.586L80 80.2l.045-.04.045-.041.045-.042.046-.041.185-.17.095-.085.096-.088.395-.36.206-.187.051-.048.053-.047.105-.097.214-.195.109-.099.11-.1.449-.41.23-.21.234-.213.06-.054.059-.054.12-.108.12-.11.12-.11.062-.056.06-.056.062-.055.062-.056.247-.226.251-.23.127-.115.127-.116.257-.234.26-.236.065-.06.065-.06.131-.119.066-.06.066-.06.133-.12 1.078-.984.55-.501.556-.507.07-.064.07-.063.14-.128.281-.256.14-.128.071-.064.07-.065.071-.064.071-.064.141-.13.567-.516.071-.064.07-.065.143-.13.284-.259.284-.258.071-.065.072-.065.141-.13.285-.258.141-.13.071-.064.071-.065.283-.258.071-.064.07-.065.142-.129.07-.064.071-.064.07-.064.07-.064.141-.129.07-.064.07-.064.141-.127.14-.128.279-.254.139-.127.069-.063.07-.063.068-.063.07-.063.137-.126.138-.125.137-.125.068-.062.069-.062.136-.125.068-.062.068-.061.067-.062.068-.062.135-.123.067-.06.067-.062 1.054-.96.129-.118.064-.059.064-.058.064-.058.064-.058.063-.058.064-.058.063-.057.063-.058.063-.057.062-.057.125-.114.062-.056.062-.057.061-.056.062-.056.06-.055.062-.056.242-.22.06-.055.06-.055.118-.108.06-.054.058-.053.117-.107.058-.053.058-.053.058-.052.057-.053.114-.104.057-.051.057-.052.056-.051.056-.051.056-.05.055-.051.055-.05.055-.05.055-.05.054-.05.215-.196.21-.192.206-.187.051-.047.05-.046.05-.045.05-.046.756-.688.345-.315.324-.295.077-.07.076-.07.146-.133.141-.128.068-.062.033-.03.033-.03.472-.43.359-.327z"
                })
            }),
            /*#__PURE__*/ (0, $dVGli$jsx)("g", {
                clipPath: "url(#8de5e6__a)",
                children: /*#__PURE__*/ (0, $dVGli$jsxs)("g", {
                    clipPath: "url(#8de5e6__b)",
                    children: [
                        /*#__PURE__*/ (0, $dVGli$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dVGli$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#8de5e6__c)"
                        }),
                        /*#__PURE__*/ (0, $dVGli$jsx)("path", {
                            fill: "url(#8de5e6__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $dVGli$jsx)("path", {
                            fill: "url(#8de5e6__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $dVGli$jsx)("path", {
                            fill: "url(#8de5e6__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $896d5c26126567dd$var$ForwardRef = /*#__PURE__*/ (0, $dVGli$forwardRef)($896d5c26126567dd$var$SvgComponent);
var $896d5c26126567dd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($896d5c26126567dd$var$ForwardRef);


export {$896d5c26126567dd$export$2e2bcd8739ae039 as default};
