// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lightbulb_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $hwEa9$reactjsxruntime = require("react/jsx-runtime");
var $hwEa9$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9d3e9953224a8eca$export$2e2bcd8739ae039);




const $9d3e9953224a8eca$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8515c7__d",
                        x1: 111.107,
                        x2: 30.526,
                        y1: 5.955,
                        y2: 110.43,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8515c7__e",
                        x1: 62.011,
                        x2: 77.555,
                        y1: -42.141,
                        y2: 57.344,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8515c7__f",
                        x1: -117.348,
                        x2: 144.197,
                        y1: 157.869,
                        y2: 106.178,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8515c7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.277 118.18)scale(200.895)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("clipPath", {
                        id: "8515c7__b",
                        children: /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.43,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("clipPath", {
                id: "8515c7__a",
                children: /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M77.372 27.183c-17.757 1.254-32.138 15.584-33.45 33.337-.974 13.183 5.124 25.003 14.889 32.07 3.175 2.297 5.11 5.929 5.11 9.848v9.323h32.16v-9.317c0-3.822 1.768-7.475 4.881-9.69 9.209-6.558 15.218-17.317 15.218-29.484 0-20.851-17.638-37.582-38.808-36.087M100.1 63.019c-2.081 0-3.768-1.687-3.768-3.77 0-6.788-5.523-12.31-12.312-12.31-2.081 0-3.768-1.687-3.768-3.77 0-2.08 1.687-3.768 3.768-3.768 10.945 0 19.849 8.904 19.849 19.849 0 2.081-1.687 3.769-3.769 3.769M63.922 125.58c0 3.33 2.7 6.03 6.03 6.03h20.1c3.33 0 6.03-2.7 6.03-6.03v-6.281h-32.16z"
                })
            }),
            /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("g", {
                clipPath: "url(#8515c7__a)",
                children: /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#8515c7__b)",
                    children: [
                        /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.43,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("circle", {
                            cx: 134.458,
                            cy: 101.903,
                            r: 200.895,
                            fill: "url(#8515c7__c)"
                        }),
                        /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("path", {
                            fill: "url(#8515c7__d)",
                            d: "M-2.092 54.01C-4.325 5.917 32.852-34.88 80.945-37.114S169.837-2.17 172.07 45.923s-34.944 88.891-83.038 91.124C40.94 139.281.142 102.104-2.092 54.01"
                        }),
                        /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("path", {
                            fill: "url(#8515c7__e)",
                            d: "M45.579-85.852c-10.85-6.714-24.595-6.755-35.485-.106l-108.915 66.5c-10.667 6.513-10.685 21.886-.032 28.424L9.341 75.37c10.873 6.673 24.62 6.665 35.483-.023L152.798 8.88c10.596-6.524 10.615-21.811.032-28.36z"
                        }),
                        /*#__PURE__*/ (0, $hwEa9$reactjsxruntime.jsx)("path", {
                            fill: "url(#8515c7__f)",
                            d: "M-58.935 76.982c22.45 0 42.784 9.086 57.507 23.743 29.445 29.366 53.857 29.391 83.303.076 14.722-14.733 35.082-23.844 57.557-23.844 44.923.025 81.314 36.318 81.314 81.119s-36.391 81.068-81.314 81.068c-22.525 0-42.885-9.136-57.608-23.869-29.42-29.265-53.807-29.24-83.252.126-14.723 14.682-35.058 23.743-57.507 23.743-44.897 0-81.289-36.293-81.289-81.068 0-44.776 36.392-81.094 81.29-81.094"
                        })
                    ]
                })
            })
        ]
    });
const $9d3e9953224a8eca$var$ForwardRef = /*#__PURE__*/ (0, $hwEa9$react.forwardRef)($9d3e9953224a8eca$var$SvgComponent);
var $9d3e9953224a8eca$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9d3e9953224a8eca$var$ForwardRef);
