// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_helpCircle_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $50cXp$reactjsxruntime = require("react/jsx-runtime");
var $50cXp$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $8f2bcdf94806109f$export$2e2bcd8739ae039);




const $8f2bcdf94806109f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3c0caa__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3c0caa__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3c0caa__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3c0caa__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("clipPath", {
                        id: "3c0caa__b",
                        children: /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("clipPath", {
                id: "3c0caa__a",
                children: /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M24 40.023c9.113 0 16.5-7.387 16.5-16.5s-7.387-16.5-16.5-16.5-16.5 7.388-16.5 16.5 7.387 16.5 16.5 16.5m.447-23.194c-.392-.08-.8-.04-1.17.118-.372.158-.694.427-.924.778-.229.352-.353.769-.353 1.197 0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25c0-.91.263-1.801.76-2.562.496-.762 1.205-1.36 2.04-1.714.836-.355 1.757-.448 2.646-.267s1.7.628 2.335 1.278 1.063 1.474 1.237 2.367c.173.893.084 1.819-.256 2.661-.341.843-.92 1.569-1.67 2.082-.354.242-.698.57-.944.926-.249.36-.346.672-.346.903v2.087c0 .69-.56 1.25-1.25 1.25s-1.25-.56-1.25-1.25v-2.087c0-.895.355-1.696.79-2.325.44-.635 1.013-1.173 1.588-1.567.337-.231.605-.562.764-.956s.201-.828.12-1.248-.282-.8-.572-1.097c-.289-.296-.653-.494-1.045-.574M25.5 31.023c0 .829-.672 1.5-1.5 1.5s-1.5-.671-1.5-1.5c0-.828.672-1.5 1.5-1.5s1.5.672 1.5 1.5"
                })
            }),
            /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("g", {
                clipPath: "url(#3c0caa__a)",
                children: /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#3c0caa__b)",
                    children: [
                        /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#3c0caa__c)"
                        }),
                        /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("path", {
                            fill: "url(#3c0caa__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("path", {
                            fill: "url(#3c0caa__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $50cXp$reactjsxruntime.jsx)("path", {
                            fill: "url(#3c0caa__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $8f2bcdf94806109f$var$ForwardRef = /*#__PURE__*/ (0, $50cXp$react.forwardRef)($8f2bcdf94806109f$var$SvgComponent);
var $8f2bcdf94806109f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($8f2bcdf94806109f$var$ForwardRef);
