// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_rocket_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lCs70$jsxs, jsx as $lCs70$jsx} from "react/jsx-runtime";
import {forwardRef as $lCs70$forwardRef} from "react";





const $c483420601f911ec$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lCs70$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lCs70$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lCs70$jsxs)("linearGradient", {
                        id: "c637f2__d",
                        x1: 32.334,
                        x2: 11.233,
                        y1: 5.77,
                        y2: 34.793,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lCs70$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $lCs70$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lCs70$jsxs)("linearGradient", {
                        id: "c637f2__e",
                        x1: 18.466,
                        x2: 22.378,
                        y1: -7.037,
                        y2: 19.522,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lCs70$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $lCs70$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lCs70$jsxs)("linearGradient", {
                        id: "c637f2__f",
                        x1: -32.193,
                        x2: 41.333,
                        y1: 46.217,
                        y2: 30.802,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lCs70$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lCs70$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $lCs70$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $lCs70$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lCs70$jsxs)("radialGradient", {
                        id: "c637f2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.4895 -56.7422 0 38.93 31.316)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lCs70$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lCs70$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lCs70$jsx)("clipPath", {
                        id: "c637f2__b",
                        children: /*#__PURE__*/ (0, $lCs70$jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.961,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lCs70$jsx)("clipPath", {
                id: "c637f2__a",
                children: /*#__PURE__*/ (0, $lCs70$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M32.104 26.166c-1.848 1.764-4.137 3.594-6.966 5.457h-.003q-.355.233-.738.399c-1.785.774-3.9.408-5.313-1.005l-1.062-1.062-1.062-1.062c-1.338-1.341-1.734-3.306-1.113-5.028.13-.354.297-.7.51-1.023 1.347-2.046 2.68-3.807 3.978-5.325C27.886 8.7 34.426 8.13 37.411 8.469c1.104.126 1.974.999 2.1 2.1.315 2.79-.159 8.682-7.407 15.597m.51-10.797c-.993-.99-2.6-.99-3.59 0-.99.993-.994 2.6 0 3.59.992.99 2.6.994 3.59 0 .99-.992.993-2.6 0-3.59m-19.383 9.213-3.144.054v.003c-2.043.036-3.201-2.34-1.905-3.918 3.663-4.455 8.598-6.042 11.817-6.405-1.941 2.046-3.828 4.452-5.646 7.212-.618.939-.987 1.989-1.122 3.054m-4.623 10.98c-.234.234-.543.35-.849.35-.309 0-.615-.116-.849-.35-.468-.471-.468-1.23 0-1.698l3.15-3.15c.471-.468 1.23-.468 1.698 0 .468.47.468 1.23 0 1.698zm4.2-2.1c.471-.468 1.23-.468 1.698 0 .468.47.468 1.23 0 1.698l-6 6c-.234.234-.54.35-.849.35s-.615-.116-.849-.35c-.468-.471-.468-1.23 0-1.698zm2.766 2.763-3.3 3.3c-.468.468-.468 1.227 0 1.698.234.234.54.35.85.35.308 0 .614-.116.848-.35l3.3-3.3c.468-.468.468-1.227 0-1.698s-1.227-.468-1.698 0m7.833-1.467-.053 3.132c-.037 2.055 2.355 3.195 3.939 1.887 4.428-3.654 6.011-8.565 6.38-11.781-1.944 1.836-4.317 3.723-7.215 5.63-.923.61-1.964.99-3.05 1.132"
                })
            }),
            /*#__PURE__*/ (0, $lCs70$jsx)("g", {
                clipPath: "url(#c637f2__a)",
                children: /*#__PURE__*/ (0, $lCs70$jsxs)("g", {
                    clipPath: "url(#c637f2__b)",
                    children: [
                        /*#__PURE__*/ (0, $lCs70$jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.961,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $lCs70$jsx)("ellipse", {
                            cx: 38.93,
                            cy: 31.316,
                            fill: "url(#c637f2__c)",
                            rx: 56.742,
                            ry: 53.489
                        }),
                        /*#__PURE__*/ (0, $lCs70$jsx)("path", {
                            fill: "url(#c637f2__d)",
                            d: "M.362 18.565C-.268 5.76 10.232-5.103 23.815-5.697s25.107 9.304 25.738 22.109S39.683 40.08 26.1 40.674.993 31.37.362 18.565"
                        }),
                        /*#__PURE__*/ (0, $lCs70$jsx)("path", {
                            fill: "url(#c637f2__e)",
                            d: "M13.825-18.675c-3.064-1.788-6.947-1.799-10.022-.028L-26.96-.997c-3.013 1.734-3.018 5.827-.01 7.568L3.59 24.25c3.071 1.777 6.954 1.775 10.022-.006L44.11 6.548c2.993-1.737 2.998-5.808.01-7.551z"
                        }),
                        /*#__PURE__*/ (0, $lCs70$jsx)("path", {
                            fill: "url(#c637f2__f)",
                            d: "M-15.694 24.68c6.34 0 12.084 2.42 16.242 6.322 8.317 7.819 15.212 7.826 23.529.02 4.158-3.922 9.909-6.348 16.257-6.348 12.688.006 22.967 9.67 22.967 21.598 0 11.929-10.279 21.585-22.967 21.585-6.362 0-12.113-2.433-16.271-6.355-8.31-7.792-15.198-7.785-23.515.033-4.158 3.91-9.902 6.322-16.242 6.322-12.681 0-22.96-9.663-22.96-21.585s10.279-21.591 22.96-21.591"
                        })
                    ]
                })
            })
        ]
    });
const $c483420601f911ec$var$ForwardRef = /*#__PURE__*/ (0, $lCs70$forwardRef)($c483420601f911ec$var$SvgComponent);
var $c483420601f911ec$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c483420601f911ec$var$ForwardRef);


export {$c483420601f911ec$export$2e2bcd8739ae039 as default};
