// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_clock_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gwED2$reactjsxruntime = require("react/jsx-runtime");
var $gwED2$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $906f010768c20364$export$2e2bcd8739ae039);




const $906f010768c20364$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "25c351__d",
                        x1: 32.783,
                        x2: 12.551,
                        y1: 5.819,
                        y2: 33.645,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "25c351__e",
                        x1: 19.489,
                        x2: 23.239,
                        y1: -6.459,
                        y2: 19.005,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "25c351__f",
                        x1: -29.084,
                        x2: 41.411,
                        y1: 44.599,
                        y2: 29.819,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "25c351__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.2844 -54.4031 0 39.106 30.312)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("clipPath", {
                        id: "25c351__b",
                        children: /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.961,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("clipPath", {
                id: "25c351__a",
                children: /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M40.8 24c0 9.278-7.522 16.8-16.8 16.8-9.28 0-16.8-7.522-16.8-16.8 0-9.28 7.52-16.8 16.8-16.8 9.278 0 16.8 7.52 16.8 16.8m-18-11.7v11.666q-.003.137.023.27.028.136.084.26c.106.233.285.426.507.55l6.532 3.772c.574.331 1.308.135 1.64-.44.33-.573.134-1.307-.44-1.639L25.2 23.306V12.3c0-.662-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2"
                })
            }),
            /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("g", {
                clipPath: "url(#25c351__a)",
                children: /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#25c351__b)",
                    children: [
                        /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.961,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("ellipse", {
                            cx: 39.106,
                            cy: 30.312,
                            fill: "url(#25c351__c)",
                            rx: 54.403,
                            ry: 51.284
                        }),
                        /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("path", {
                            fill: "url(#25c351__d)",
                            d: "M2.128 18.087C1.523 5.809 11.591-4.607 24.615-5.177s24.072 8.92 24.677 21.198c.605 12.277-9.463 22.692-22.487 23.262S2.733 30.364 2.128 18.087"
                        }),
                        /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("path", {
                            fill: "url(#25c351__e)",
                            d: "M15.039-17.617c-2.938-1.714-6.66-1.725-9.61-.027L-24.064-.669c-2.889 1.662-2.893 5.587-.009 7.256l29.3 16.952c2.944 1.703 6.667 1.7 9.609-.006l29.24-16.968c2.869-1.666 2.874-5.568.008-7.24z"
                        }),
                        /*#__PURE__*/ (0, $gwED2$reactjsxruntime.jsx)("path", {
                            fill: "url(#25c351__f)",
                            d: "M-13.266 23.95c6.08 0 11.586 2.32 15.573 6.06 7.974 7.497 14.585 7.504 22.559.02 3.987-3.76 9.5-6.087 15.586-6.087 12.166.007 22.02 9.272 22.02 20.708 0 11.437-9.854 20.695-22.02 20.695-6.1 0-11.613-2.332-15.6-6.093-7.967-7.47-14.57-7.464-22.545.032-3.987 3.748-9.494 6.061-15.573 6.061-12.158 0-22.013-9.264-22.013-20.695 0-11.43 9.855-20.701 22.013-20.701"
                        })
                    ]
                })
            })
        ]
    });
const $906f010768c20364$var$ForwardRef = /*#__PURE__*/ (0, $gwED2$react.forwardRef)($906f010768c20364$var$SvgComponent);
var $906f010768c20364$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($906f010768c20364$var$ForwardRef);
