// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_accessibility_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7QUJu$jsxs, jsx as $7QUJu$jsx} from "react/jsx-runtime";
import {forwardRef as $7QUJu$forwardRef} from "react";





const $a1c11c0c863db271$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7QUJu$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7QUJu$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7QUJu$jsxs)("linearGradient", {
                        id: "69ad45__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7QUJu$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7QUJu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7QUJu$jsxs)("linearGradient", {
                        id: "69ad45__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7QUJu$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7QUJu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7QUJu$jsxs)("linearGradient", {
                        id: "69ad45__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7QUJu$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7QUJu$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7QUJu$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7QUJu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7QUJu$jsxs)("radialGradient", {
                        id: "69ad45__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7QUJu$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7QUJu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7QUJu$jsx)("clipPath", {
                        id: "69ad45__b",
                        children: /*#__PURE__*/ (0, $7QUJu$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7QUJu$jsx)("clipPath", {
                id: "69ad45__a",
                children: /*#__PURE__*/ (0, $7QUJu$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M24 40.5c9.113 0 16.5-7.387 16.5-16.5S33.113 7.5 24 7.5 7.5 14.887 7.5 24 14.887 40.5 24 40.5m2.5-25.428c0 1.381-1.12 2.5-2.5 2.5s-2.5-1.119-2.5-2.5c0-1.38 1.12-2.5 2.5-2.5s2.5 1.12 2.5 2.5m-11.019 3.56c.152-.532.706-.839 1.237-.687 4.698 1.344 9.676 1.362 14.384.05l.187-.052c.532-.148 1.084.163 1.232.695s-.163 1.084-.695 1.232l-.188.052c-1.381.385-2.784.663-4.196.835-.514.063-.91.493-.91 1.011v3.953q0 .139.038.273l1.258 4.444.735 2.608.224.8.012.044c.04.14.08.287.091.34.108.542-.243 1.07-.784 1.177-.542.109-1.069-.242-1.177-.784.005.026 0 .011-.024-.08l-.045-.16-.223-.796c-.187-.668-.446-1.587-.734-2.606-.328-1.164-.694-2.459-1.036-3.664-.11-.387-.464-.655-.867-.655s-.757.268-.867.655c-.341 1.205-.708 2.5-1.036 3.664l-.735 2.606-.222.796-.045.16c-.025.091-.03.106-.024.08-.108.541-.635.893-1.177.784-.541-.108-.892-.635-.784-1.176.01-.054.052-.2.09-.34l.013-.045.224-.8.735-2.608c.4-1.42.857-3.034 1.258-4.444q.038-.133.038-.273v-3.953c0-.518-.396-.948-.91-1.01-1.479-.18-2.947-.477-4.39-.89-.531-.152-.839-.706-.687-1.237"
                })
            }),
            /*#__PURE__*/ (0, $7QUJu$jsx)("g", {
                clipPath: "url(#69ad45__a)",
                children: /*#__PURE__*/ (0, $7QUJu$jsxs)("g", {
                    clipPath: "url(#69ad45__b)",
                    children: [
                        /*#__PURE__*/ (0, $7QUJu$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7QUJu$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#69ad45__c)"
                        }),
                        /*#__PURE__*/ (0, $7QUJu$jsx)("path", {
                            fill: "url(#69ad45__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $7QUJu$jsx)("path", {
                            fill: "url(#69ad45__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $7QUJu$jsx)("path", {
                            fill: "url(#69ad45__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $a1c11c0c863db271$var$ForwardRef = /*#__PURE__*/ (0, $7QUJu$forwardRef)($a1c11c0c863db271$var$SvgComponent);
var $a1c11c0c863db271$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a1c11c0c863db271$var$ForwardRef);


export {$a1c11c0c863db271$export$2e2bcd8739ae039 as default};
