// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_badgeVerified_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $9Fe0Z$jsxs, jsx as $9Fe0Z$jsx} from "react/jsx-runtime";
import {forwardRef as $9Fe0Z$forwardRef} from "react";





const $685fff3392588d42$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9Fe0Z$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9Fe0Z$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9Fe0Z$jsxs)("linearGradient", {
                        id: "8f2c98__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Fe0Z$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $9Fe0Z$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Fe0Z$jsxs)("linearGradient", {
                        id: "8f2c98__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Fe0Z$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $9Fe0Z$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Fe0Z$jsxs)("linearGradient", {
                        id: "8f2c98__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Fe0Z$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9Fe0Z$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $9Fe0Z$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $9Fe0Z$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Fe0Z$jsxs)("radialGradient", {
                        id: "8f2c98__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9Fe0Z$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9Fe0Z$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9Fe0Z$jsx)("clipPath", {
                        id: "8f2c98__b",
                        children: /*#__PURE__*/ (0, $9Fe0Z$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9Fe0Z$jsx)("clipPath", {
                id: "8f2c98__a",
                children: /*#__PURE__*/ (0, $9Fe0Z$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M126.891 64.173c5.511 3.126 9.247 9.044 9.247 15.834h.015c0 6.82-3.766 12.752-9.32 15.878-2.181 1.221-3.199 3.707-2.53 6.121.742 2.632.902 5.511.305 8.491-1.41 7.11-7.095 12.81-14.205 14.22-2.995.596-5.903.436-8.549-.32-2.37-.669-4.842.335-6.078 2.472C92.621 132.321 86.746 136 80 136s-12.62-3.679-15.775-9.131c-1.236-2.137-3.693-3.141-6.078-2.472-2.646.756-5.554.916-8.55.32-7.11-1.41-12.794-7.11-14.205-14.22-.596-2.98-.436-5.859.306-8.491.668-2.414-.35-4.9-2.53-6.121-5.555-3.126-9.32-9.088-9.32-15.878S27.57 67.3 33.095 64.173c2.108-1.192 3.082-3.678 2.5-5.96-.58-2.284-.74-4.726-.377-7.256 1.148-7.866 7.502-14.191 15.368-15.325 2.574-.378 4.944-.233 7.358.422 2.413.654 4.885-.393 6.092-2.574C67.133 27.839 73.123 24 80 24s12.868 3.839 15.965 9.48c1.207 2.18 3.795 3.184 6.092 2.574s4.769-.8 7.343-.422c7.88 1.134 14.22 7.459 15.368 15.325.364 2.53.233 4.914-.378 7.255-.61 2.341.393 4.77 2.501 5.961m-19.88 3.905c1.631-1.49 1.745-4.02.255-5.651s-4.02-1.745-5.651-.255L71.712 89.497 58.349 77.071c-1.617-1.505-4.148-1.413-5.653.205s-1.412 4.149.206 5.653l16.062 14.938c1.526 1.418 3.884 1.429 5.422.023z"
                })
            }),
            /*#__PURE__*/ (0, $9Fe0Z$jsx)("g", {
                clipPath: "url(#8f2c98__a)",
                children: /*#__PURE__*/ (0, $9Fe0Z$jsxs)("g", {
                    clipPath: "url(#8f2c98__b)",
                    children: [
                        /*#__PURE__*/ (0, $9Fe0Z$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $9Fe0Z$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#8f2c98__c)"
                        }),
                        /*#__PURE__*/ (0, $9Fe0Z$jsx)("path", {
                            fill: "url(#8f2c98__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $9Fe0Z$jsx)("path", {
                            fill: "url(#8f2c98__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $9Fe0Z$jsx)("path", {
                            fill: "url(#8f2c98__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $685fff3392588d42$var$ForwardRef = /*#__PURE__*/ (0, $9Fe0Z$forwardRef)($685fff3392588d42$var$SvgComponent);
var $685fff3392588d42$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($685fff3392588d42$var$ForwardRef);


export {$685fff3392588d42$export$2e2bcd8739ae039 as default};
