// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_buildTable_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $aEjHr$reactjsxruntime = require("react/jsx-runtime");
var $aEjHr$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $ae32589f443c1d04$export$2e2bcd8739ae039);




const $ae32589f443c1d04$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "21bbb2__d",
                        x1: 117.96,
                        x2: 44.078,
                        y1: 19.028,
                        y2: 120.644,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "21bbb2__e",
                        x1: 69.409,
                        x2: 83.106,
                        y1: -25.808,
                        y2: 67.181,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "21bbb2__f",
                        x1: -107.965,
                        x2: 149.469,
                        y1: 160.645,
                        y2: 106.673,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsxs)("radialGradient", {
                        id: "21bbb2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 187.28 -198.669 0 141.052 108.474)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("clipPath", {
                        id: "21bbb2__b",
                        children: /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("rect", {
                            width: 137.6,
                            height: 129.712,
                            x: 18.398,
                            y: 23.199,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("clipPath", {
                id: "21bbb2__a",
                children: /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M123.969 33.9c8.84 0 16 7.16 16 16v4.09H19.949V49.9c0-8.84 7.16-16 16-16zm16.001 52.56c-2.58-.66-5.28-1.01-8.06-1.01h-.01c-17.94 0-32.53 14.59-32.53 32.53 0 2.77.35 5.47 1.01 8.04H48V61.99h91.97zM19.95 110.03c0 8.84 7.16 16 16 16H40V62H19.95zM131.91 93.46c2.82 0 5.53.48 8.06 1.36h.01c9.59 3.34 16.47 12.45 16.47 23.17 0 13.55-10.98 24.53-24.53 24.53-10.73 0-19.85-6.89-23.18-16.49-.88-2.52-1.36-5.22-1.36-8.04 0-13.55 10.98-24.53 24.53-24.53m4.01 28.48h7.88c2.21 0 4-1.79 4-4s-1.79-4-4-4h-7.88v-7.88c0-2.21-1.79-4-4-4s-4 1.79-4 4v7.88h-7.88c-2.21 0-4 1.79-4 4s1.79 4 4 4h7.88v7.88c0 2.21 1.79 4 4 4s4-1.79 4-4z"
                })
            }),
            /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("g", {
                clipPath: "url(#21bbb2__a)",
                children: /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#21bbb2__b)",
                    children: [
                        /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("rect", {
                            width: 137.6,
                            height: 129.712,
                            x: 18.398,
                            y: 23.199,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("ellipse", {
                            cx: 141.052,
                            cy: 108.474,
                            fill: "url(#21bbb2__c)",
                            rx: 198.669,
                            ry: 187.28
                        }),
                        /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("path", {
                            fill: "url(#21bbb2__d)",
                            d: "M6.016 63.827C3.807 18.993 40.573-19.04 88.133-21.123c47.56-2.08 87.906 32.577 90.114 77.41 2.209 44.835-34.556 82.867-82.117 84.949-47.56 2.082-87.906-32.575-90.114-77.41"
                        }),
                        /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("path", {
                            fill: "url(#21bbb2__e)",
                            d: "M53.16-66.557c-10.73-6.258-24.323-6.296-35.092-.099L-89.64-4.662c-10.549 6.071-10.567 20.403-.032 26.497L17.324 83.74c10.753 6.22 24.346 6.213 35.09-.022L159.19 21.755c10.478-6.082 10.498-20.333.032-26.437z"
                        }),
                        /*#__PURE__*/ (0, $aEjHr$reactjsxruntime.jsx)("path", {
                            fill: "url(#21bbb2__f)",
                            d: "M-50.198 85.24c22.2 0 42.31 8.47 56.869 22.134 29.119 27.376 53.26 27.399 82.38.071 14.559-13.735 34.693-22.228 56.918-22.228 44.426.023 80.414 33.856 80.414 75.621s-35.988 75.574-80.414 75.574c-22.274 0-42.409-8.517-56.968-22.251-29.095-27.282-53.211-27.258-82.33.117-14.56 13.688-34.67 22.134-56.87 22.134-44.4 0-80.388-33.833-80.388-75.574S-94.599 85.24-50.198 85.24"
                        })
                    ]
                })
            })
        ]
    });
const $ae32589f443c1d04$var$ForwardRef = /*#__PURE__*/ (0, $aEjHr$react.forwardRef)($ae32589f443c1d04$var$SvgComponent);
var $ae32589f443c1d04$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($ae32589f443c1d04$var$ForwardRef);
