// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_speedFast_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $jHMvC$reactjsxruntime = require("react/jsx-runtime");
var $jHMvC$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $098e36acc40c417d$export$2e2bcd8739ae039);




const $098e36acc40c417d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "374ab1__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "374ab1__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "374ab1__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsxs)("radialGradient", {
                        id: "374ab1__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("clipPath", {
                        id: "374ab1__b",
                        children: /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("clipPath", {
                id: "374ab1__a",
                children: /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("path", {
                    fill: "#D9D9D9",
                    d: "M79.95 54.607c-2.21 0-4-1.79-4-4V38.642q0-.255.03-.5c-7.404.526-14.41 2.495-20.737 5.627q.134.182.25.38l5.982 10.363c1.104 1.913.449 4.36-1.464 5.464s-4.36.45-5.465-1.464L48.565 48.15q-.1-.173-.18-.352c-5.932 4.078-11.036 9.271-15.011 15.277q.06.032.122.068l10.362 5.982c1.913 1.105 2.569 3.551 1.464 5.465s-3.55 2.568-5.464 1.464L29.495 70.07l-.048-.028c-2.974 6.298-4.806 13.242-5.228 20.564H36.01c2.209 0 4 1.79 4 4s-1.791 4-4 4H24.322c.616 7.337 2.649 14.27 5.824 20.526 1.26 2.481 3.882 3.908 6.665 3.908h86.378c2.783 0 5.405-1.427 6.664-3.908 3.176-6.256 5.209-13.189 5.824-20.526h-11.689c-2.209 0-4-1.79-4-4s1.791-4 4-4h11.793c-1.024-17.748-10.331-33.275-24.114-42.773q-.074.16-.164.316l-5.983 10.362c-1.104 1.913-3.551 2.569-5.464 1.464s-2.568-3.55-1.464-5.464l5.983-10.362q.107-.186.231-.356c-6.37-3.16-13.427-5.142-20.888-5.66q.032.25.032.508v11.965c0 2.21-1.79 4-4 4m40.542 9.21c1.599 1.6 1.666 4.17.153 5.85l-37.346 41.488c-2.753 3.058-7.508 3.183-10.418.273s-2.785-7.665.273-10.418l41.487-37.346c1.681-1.513 4.252-1.446 5.851.153"
                })
            }),
            /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("g", {
                clipPath: "url(#374ab1__a)",
                children: /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#374ab1__b)",
                    children: [
                        /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#374ab1__c)"
                        }),
                        /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("path", {
                            fill: "url(#374ab1__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("path", {
                            fill: "url(#374ab1__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $jHMvC$reactjsxruntime.jsx)("path", {
                            fill: "url(#374ab1__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $098e36acc40c417d$var$ForwardRef = /*#__PURE__*/ (0, $jHMvC$react.forwardRef)($098e36acc40c417d$var$SvgComponent);
var $098e36acc40c417d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($098e36acc40c417d$var$ForwardRef);
