// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_conversationbubbles_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $gF1qB$jsxs, jsx as $gF1qB$jsx} from "react/jsx-runtime";
import {forwardRef as $gF1qB$forwardRef} from "react";





const $cfac90a4ba149f5e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gF1qB$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gF1qB$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gF1qB$jsxs)("linearGradient", {
                        id: "3d0e86__d",
                        x1: 33.33,
                        x2: 9.156,
                        y1: 1.786,
                        y2: 33.128,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gF1qB$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $gF1qB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gF1qB$jsxs)("linearGradient", {
                        id: "3d0e86__e",
                        x1: 18.602,
                        x2: 23.265,
                        y1: -12.643,
                        y2: 17.203,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gF1qB$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $gF1qB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gF1qB$jsxs)("linearGradient", {
                        id: "3d0e86__f",
                        x1: -35.206,
                        x2: 43.258,
                        y1: 47.361,
                        y2: 31.854,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gF1qB$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gF1qB$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $gF1qB$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $gF1qB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gF1qB$jsxs)("radialGradient", {
                        id: "3d0e86__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 4.884 35.455)scale(60.2686)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gF1qB$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gF1qB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gF1qB$jsx)("clipPath", {
                        id: "3d0e86__b",
                        children: /*#__PURE__*/ (0, $gF1qB$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gF1qB$jsxs)("clipPath", {
                id: "3d0e86__a",
                children: [
                    /*#__PURE__*/ (0, $gF1qB$jsx)("path", {
                        fill: "#fff",
                        d: "M39.484 20.809h-4.631v3.436c0 2.968-2.51 5.383-5.596 5.383h-7.562v2.235c0 1.342 1.167 2.43 2.605 2.43h5.672l5.756 4.655c.565.456 1.443.08 1.437-.618-.01-1.25-.02-2.89-.014-4.037h2.333c1.439 0 2.605-1.088 2.605-2.43v-8.624c0-1.342-1.166-2.43-2.605-2.43"
                    }),
                    /*#__PURE__*/ (0, $gF1qB$jsx)("path", {
                        fill: "#fff",
                        d: "M29.258 27.215c1.759 0 3.184-1.33 3.184-2.97v-10.54c0-1.64-1.425-2.97-3.184-2.97H9.094c-1.758 0-3.184 1.33-3.184 2.97v10.54c0 1.64 1.426 2.97 3.184 2.97h2.851c.007 1.402-.005 3.407-.017 4.935-.007.852 1.067 1.313 1.757.755l7.035-5.69z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gF1qB$jsx)("g", {
                clipPath: "url(#3d0e86__a)",
                children: /*#__PURE__*/ (0, $gF1qB$jsxs)("g", {
                    clipPath: "url(#3d0e86__b)",
                    children: [
                        /*#__PURE__*/ (0, $gF1qB$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $gF1qB$jsx)("circle", {
                            cx: 40.339,
                            cy: 30.571,
                            r: 60.269,
                            fill: "url(#3d0e86__c)"
                        }),
                        /*#__PURE__*/ (0, $gF1qB$jsx)("path", {
                            fill: "url(#3d0e86__d)",
                            d: "M-.63 16.202C-1.3 1.774 9.854-10.465 24.282-11.135S50.949-.652 51.619 13.776 41.136 40.444 26.708 41.113.04 30.63-.63 16.203"
                        }),
                        /*#__PURE__*/ (0, $gF1qB$jsx)("path", {
                            fill: "url(#3d0e86__e)",
                            d: "M13.672-25.756c-3.254-2.014-7.378-2.027-10.645-.032l-32.675 19.95c-3.2 1.954-3.205 6.566-.01 8.527l32.46 19.922c3.261 2.002 7.385 2 10.644-.007l32.392-19.94c3.179-1.958 3.185-6.544.01-8.508z"
                        }),
                        /*#__PURE__*/ (0, $gF1qB$jsx)("path", {
                            fill: "url(#3d0e86__f)",
                            d: "M-17.682 23.095c6.735 0 12.836 2.726 17.252 7.123 8.834 8.81 16.158 8.817 24.991.023 4.417-4.42 10.525-7.153 17.267-7.153 13.477.007 24.395 10.895 24.395 24.335S55.305 71.745 41.828 71.745c-6.757 0-12.865-2.74-17.282-7.16-8.826-8.78-16.142-8.773-24.976.037-4.416 4.405-10.517 7.123-17.252 7.123-13.469 0-24.386-10.888-24.386-24.32s10.917-24.329 24.386-24.329"
                        })
                    ]
                })
            })
        ]
    });
const $cfac90a4ba149f5e$var$ForwardRef = /*#__PURE__*/ (0, $gF1qB$forwardRef)($cfac90a4ba149f5e$var$SvgComponent);
var $cfac90a4ba149f5e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cfac90a4ba149f5e$var$ForwardRef);


export {$cfac90a4ba149f5e$export$2e2bcd8739ae039 as default};
