// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lightbulb_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6ePTx$jsxs, jsx as $6ePTx$jsx} from "react/jsx-runtime";
import {forwardRef as $6ePTx$forwardRef} from "react";





const $55337cd8a83ca0d6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6ePTx$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6ePTx$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6ePTx$jsxs)("linearGradient", {
                        id: "b27d4d__d",
                        x1: 66.664,
                        x2: 18.316,
                        y1: 3.573,
                        y2: 66.258,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6ePTx$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6ePTx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6ePTx$jsxs)("linearGradient", {
                        id: "b27d4d__e",
                        x1: 37.208,
                        x2: 46.535,
                        y1: -25.284,
                        y2: 34.407,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6ePTx$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6ePTx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6ePTx$jsxs)("linearGradient", {
                        id: "b27d4d__f",
                        x1: -70.407,
                        x2: 86.52,
                        y1: 94.721,
                        y2: 63.707,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6ePTx$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6ePTx$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6ePTx$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6ePTx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6ePTx$jsxs)("radialGradient", {
                        id: "b27d4d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 9.766 70.908)scale(120.537)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6ePTx$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6ePTx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6ePTx$jsx)("clipPath", {
                        id: "b27d4d__b",
                        children: /*#__PURE__*/ (0, $6ePTx$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6ePTx$jsx)("clipPath", {
                id: "b27d4d__a",
                children: /*#__PURE__*/ (0, $6ePTx$jsx)("path", {
                    fill: "#fff",
                    d: "M46.42 16.31c-10.654.752-19.283 9.35-20.07 20.002-.585 7.91 3.074 15.002 8.933 19.242 1.905 1.378 3.066 3.557 3.066 5.909v5.594h19.296v-5.59c0-2.294 1.061-4.485 2.929-5.815 5.526-3.934 9.131-10.39 9.131-17.69 0-12.51-10.583-22.55-23.285-21.652M60.057 37.81c-1.249 0-2.261-1.012-2.261-2.261 0-4.073-3.314-7.387-7.387-7.387-1.249 0-2.261-1.012-2.261-2.261s1.012-2.261 2.261-2.261c6.567 0 11.91 5.342 11.91 11.909 0 1.249-1.013 2.261-2.262 2.261M38.352 75.349c0 1.998 1.62 3.618 3.618 3.618h12.06c1.998 0 3.618-1.62 3.618-3.618V71.58H38.352z"
                })
            }),
            /*#__PURE__*/ (0, $6ePTx$jsx)("g", {
                clipPath: "url(#b27d4d__a)",
                children: /*#__PURE__*/ (0, $6ePTx$jsxs)("g", {
                    clipPath: "url(#b27d4d__b)",
                    children: [
                        /*#__PURE__*/ (0, $6ePTx$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6ePTx$jsx)("circle", {
                            cx: 80.674,
                            cy: 61.143,
                            r: 120.537,
                            fill: "url(#b27d4d__c)"
                        }),
                        /*#__PURE__*/ (0, $6ePTx$jsx)("path", {
                            fill: "url(#b27d4d__d)",
                            d: "M-1.255 32.406C-2.595 3.55 19.71-20.928 48.567-22.268s53.335 20.966 54.675 49.822S82.275 80.89 53.419 82.23.085 61.263-1.255 32.406"
                        }),
                        /*#__PURE__*/ (0, $6ePTx$jsx)("path", {
                            fill: "url(#b27d4d__e)",
                            d: "M27.349-51.51c-6.51-4.029-14.757-4.053-21.291-.064l-65.35 39.9c-6.4 3.908-6.41 13.132-.018 17.054L5.607 45.224c6.524 4.003 14.771 3.998 21.29-.015L91.68 5.33c6.358-3.915 6.37-13.087.02-17.016z"
                        }),
                        /*#__PURE__*/ (0, $6ePTx$jsx)("path", {
                            fill: "url(#b27d4d__f)",
                            d: "M-35.36 46.189c13.47 0 25.671 5.451 34.505 14.246 17.667 17.619 32.314 17.634 49.981.045 8.834-8.84 21.05-14.306 34.535-14.306 26.953.015 48.788 21.79 48.788 48.671s-21.835 48.641-48.789 48.641c-13.514 0-25.73-5.482-34.564-14.321-17.652-17.559-32.284-17.544-49.951.075-8.834 8.81-21.035 14.246-34.504 14.246-26.939 0-48.774-21.775-48.774-48.641 0-26.865 21.835-48.656 48.774-48.656"
                        })
                    ]
                })
            })
        ]
    });
const $55337cd8a83ca0d6$var$ForwardRef = /*#__PURE__*/ (0, $6ePTx$forwardRef)($55337cd8a83ca0d6$var$SvgComponent);
var $55337cd8a83ca0d6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($55337cd8a83ca0d6$var$ForwardRef);


export {$55337cd8a83ca0d6$export$2e2bcd8739ae039 as default};
