// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_confetti_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $f8mOr$jsxs, jsx as $f8mOr$jsx} from "react/jsx-runtime";
import {forwardRef as $f8mOr$forwardRef} from "react";





const $29983e86a191c2a1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $f8mOr$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $f8mOr$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $f8mOr$jsxs)("linearGradient", {
                        id: "5b3327__d",
                        x1: 109.744,
                        x2: 33.395,
                        y1: 13.691,
                        y2: 118.59,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $f8mOr$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $f8mOr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $f8mOr$jsxs)("linearGradient", {
                        id: "5b3327__e",
                        x1: 59.643,
                        x2: 73.807,
                        y1: -32.629,
                        y2: 63.43,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $f8mOr$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $f8mOr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $f8mOr$jsxs)("linearGradient", {
                        id: "5b3327__f",
                        x1: -123.403,
                        x2: 142.287,
                        y1: 159.99,
                        y2: 104.346,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $f8mOr$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $f8mOr$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $f8mOr$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $f8mOr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $f8mOr$jsxs)("radialGradient", {
                        id: "5b3327__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 193.472 -205.022 0 133.577 106.095)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $f8mOr$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $f8mOr$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $f8mOr$jsx)("clipPath", {
                        id: "5b3327__b",
                        children: /*#__PURE__*/ (0, $f8mOr$jsx)("rect", {
                            width: 142,
                            height: 134,
                            x: 7,
                            y: 18,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $f8mOr$jsx)("clipPath", {
                id: "5b3327__a",
                children: /*#__PURE__*/ (0, $f8mOr$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M109.922 32.16c2.86-2.86 2.86-7.48 0-10.34s-7.48-2.86-10.34 0-2.86 7.48 0 10.34 7.48 2.86 10.34 0m-45.12 39.96c-1.73 1.38-2.01 3.9-.63 5.62.79.99 1.95 1.5 3.13 1.5.88 0 1.76-.29 2.49-.88 19.22-15.35 18.34-31.46 18.299-32.14-.15-2.19-2.03-3.82-4.22-3.7-2.19.13-3.87 2.02-3.76 4.21.02.52.43 12.81-15.31 25.38zm34 29.871c-1.03-1.95-.28-4.37 1.671-5.4h.01c1.95-1.03 4.37-.28 5.4 1.67 5.73 10.86 3.89 16.18 1.34 18.73-.83.83-1.95 1.58-3.45 2.02l-69.25 24.6c-1.34.48-2.71.71-4.06.71-3.11 0-6.14-1.23-8.44-3.53-3.3-3.3-4.38-8.09-2.82-12.5l24.32-68.53c.14-.52.32-1.02.55-1.5.45-1.12 1.07-2 1.75-2.68 2.57-2.57 7.94-4.4 18.93 1.45 1.95 1.04 2.69 3.46 1.65 5.41s-3.46 2.69-5.41 1.65c-6.19-3.29-9.03-3.09-9.52-2.84v.02l-.18.51c-.4 3.04 4.2 14.87 19.53 30.2 11.62 11.62 21.22 17.07 26.54 18.84 2.06.42 3.52.52 4.26.42.19-.55.29-3.35-2.82-9.25m-21.31-12.56c-1.03 0-2.05-.39-2.83-1.17-1.56-1.57-1.56-4.1 0-5.66l15.61-15.61c1.57-1.56 4.1-1.56 5.66 0 1.56 1.57 1.56 4.1 0 5.66l-15.61 15.61c-.78.78-1.8 1.17-2.83 1.17m7.05 3.7c15.35-19.22 31.46-18.34 32.14-18.3l-.01.01c2.21.15 3.87 2.06 3.72 4.26-.15 2.19-2.04 3.86-4.23 3.72-.59-.02-12.84-.37-25.37 15.31-.79.99-1.96 1.5-3.13 1.5-.87 0-1.75-.29-2.49-.88-1.73-1.38-2.01-3.89-.63-5.62m40.499-.96c2.86-2.86 7.48-2.86 10.34 0s2.86 7.48 0 10.34-7.48 2.86-10.34 0-2.86-7.48 0-10.34M56.354 28.196l-1.921 1.406c-1.783 1.305-2.17 3.807-.865 5.59l7.813 10.676c1.305 1.783 3.808 2.17 5.59.866l1.921-1.406c1.783-1.305 2.17-3.808.865-5.59l-7.813-10.676c-1.305-1.783-3.808-2.17-5.59-.866m50.876 26.656 13.146-1.488c2.196-.249 4.177 1.33 4.425 3.524l.268 2.365c.248 2.195-1.33 4.176-3.525 4.425l-13.146 1.488c-2.195.249-4.176-1.329-4.424-3.524l-.268-2.365c-.249-2.195 1.329-4.176 3.524-4.425"
                })
            }),
            /*#__PURE__*/ (0, $f8mOr$jsx)("g", {
                clipPath: "url(#5b3327__a)",
                children: /*#__PURE__*/ (0, $f8mOr$jsxs)("g", {
                    clipPath: "url(#5b3327__b)",
                    children: [
                        /*#__PURE__*/ (0, $f8mOr$jsx)("rect", {
                            width: 142,
                            height: 134,
                            x: 7,
                            y: 18,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $f8mOr$jsx)("ellipse", {
                            cx: 133.577,
                            cy: 106.095,
                            fill: "url(#5b3327__c)",
                            rx: 205.022,
                            ry: 193.472
                        }),
                        /*#__PURE__*/ (0, $f8mOr$jsx)("path", {
                            fill: "url(#5b3327__d)",
                            d: "M-5.78 59.97c-2.28-46.316 35.662-85.606 84.743-87.757 49.081-2.15 90.717 33.653 92.996 79.969s-35.661 85.606-84.743 87.757S-3.5 106.286-5.78 59.97"
                        }),
                        /*#__PURE__*/ (0, $f8mOr$jsx)("path", {
                            fill: "url(#5b3327__e)",
                            d: "M42.874-74.725c-11.072-6.465-25.1-6.505-36.214-.102l-111.153 64.043c-10.886 6.272-10.904 21.077-.033 27.373L5.893 80.541c11.096 6.426 25.125 6.418 36.212-.023l110.19-64.012c10.814-6.282 10.834-21.004.033-27.31z"
                        }),
                        /*#__PURE__*/ (0, $f8mOr$jsx)("path", {
                            fill: "url(#5b3327__f)",
                            d: "M-63.79 82.093c22.91 0 43.663 8.75 58.689 22.865 30.05 28.281 54.963 28.305 85.014.073 15.025-14.189 35.803-22.963 58.739-22.963 45.846.025 82.985 34.976 82.985 78.121 0 43.146-37.139 78.073-82.985 78.073-22.987 0-43.766-8.798-58.79-22.987-30.025-28.183-54.913-28.159-84.963.121-15.026 14.14-35.778 22.866-58.688 22.866-45.821 0-82.96-34.951-82.96-78.073s37.139-78.096 82.96-78.096"
                        })
                    ]
                })
            })
        ]
    });
const $29983e86a191c2a1$var$ForwardRef = /*#__PURE__*/ (0, $f8mOr$forwardRef)($29983e86a191c2a1$var$SvgComponent);
var $29983e86a191c2a1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($29983e86a191c2a1$var$ForwardRef);


export {$29983e86a191c2a1$export$2e2bcd8739ae039 as default};
