// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_paperclip_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $2rtXy$reactjsxruntime = require("react/jsx-runtime");
var $2rtXy$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $218260507ddecd49$export$2e2bcd8739ae039);




const $218260507ddecd49$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0977db__d",
                        x1: 109.277,
                        x2: 41.838,
                        y1: 16.892,
                        y2: 109.646,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0977db__e",
                        x1: 64.961,
                        x2: 77.463,
                        y1: -24.034,
                        y2: 60.846,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsxs)("linearGradient", {
                        id: "0977db__f",
                        x1: -96.944,
                        x2: 138.04,
                        y1: 146.161,
                        y2: 96.896,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsxs)("radialGradient", {
                        id: "0977db__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 170.948 -181.344 0 130.355 98.538)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("clipPath", {
                        id: "0977db__b",
                        children: /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 20.7,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("clipPath", {
                id: "0977db__a",
                children: /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    stroke: "#fff",
                    strokeLinecap: "round",
                    strokeLinejoin: "round",
                    strokeWidth: 2,
                    d: "M35.353 89.552c-11.138 11.083-11.137 29.061 0 40.144 11.127 11.072 29.16 11.072 40.287 0l45.068-44.847c1.566-1.558 1.573-4.09.014-5.657-1.558-1.565-4.09-1.572-5.656-.013l-45.07 44.846c-8.005 7.967-20.994 7.967-29 0-7.995-7.956-7.995-20.846 0-28.802l61.724-61.42c5.097-5.07 13.369-5.07 18.466 0 5.085 5.062 5.085 13.259 0 18.32L65.86 107.175c-2.128 2.118-5.587 2.118-7.715 0-2.117-2.107-2.117-5.514 0-7.621l38.67-38.48c1.566-1.558 1.573-4.09.014-5.657-1.558-1.566-4.09-1.572-5.656-.014l-38.67 38.48c-5.26 5.234-5.26 13.729 0 18.963 5.249 5.223 13.751 5.223 19 0l55.325-55.052c8.229-8.189 8.229-21.473 0-29.661-8.217-8.177-21.533-8.177-29.751 0z"
                })
            }),
            /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("g", {
                clipPath: "url(#0977db__a)",
                children: /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#0977db__b)",
                    children: [
                        /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 20.7,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("ellipse", {
                            cx: 130.355,
                            cy: 98.538,
                            fill: "url(#0977db__c)",
                            rx: 181.344,
                            ry: 170.948
                        }),
                        /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("path", {
                            fill: "url(#0977db__d)",
                            d: "M7.095 57.784C5.08 16.859 38.638-17.856 82.051-19.757s80.24 29.735 82.256 70.66c2.016 40.923-31.543 75.639-74.956 77.54-43.413 1.9-80.24-29.735-82.256-70.66"
                        }),
                        /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("path", {
                            fill: "url(#0977db__e)",
                            d: "M50.129-61.229c-9.794-5.713-22.202-5.748-32.032-.09L-80.218-4.732c-9.629 5.542-9.645 18.623-.03 24.186L17.42 75.961c9.814 5.678 22.223 5.671 32.03-.02l97.464-56.56c9.564-5.551 9.582-18.56.029-24.131z"
                        }),
                        /*#__PURE__*/ (0, $2rtXy$reactjsxruntime.jsx)("path", {
                            fill: "url(#0977db__f)",
                            d: "M-44.215 77.332c20.264 0 38.62 7.731 51.91 20.204 26.58 24.988 48.615 25.009 75.195.064 13.29-12.537 31.668-20.29 51.955-20.29 40.551.022 73.401 30.904 73.401 69.027 0 38.122-32.85 68.984-73.401 68.984-20.332 0-38.71-7.774-52-20.311-26.557-24.903-48.57-24.881-75.15.107-13.29 12.494-31.646 20.204-51.91 20.204-40.529 0-73.378-30.883-73.378-68.984s32.85-69.005 73.378-69.005"
                        })
                    ]
                })
            })
        ]
    });
const $218260507ddecd49$var$ForwardRef = /*#__PURE__*/ (0, $2rtXy$react.forwardRef)($218260507ddecd49$var$SvgComponent);
var $218260507ddecd49$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($218260507ddecd49$var$ForwardRef);
