// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_filters_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $etLOc$jsxs, jsx as $etLOc$jsx} from "react/jsx-runtime";
import {forwardRef as $etLOc$forwardRef} from "react";





const $1cb7c47f3fb3ff81$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $etLOc$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $etLOc$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $etLOc$jsxs)("linearGradient", {
                        id: "a01c0c__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $etLOc$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $etLOc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $etLOc$jsxs)("linearGradient", {
                        id: "a01c0c__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $etLOc$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $etLOc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $etLOc$jsxs)("linearGradient", {
                        id: "a01c0c__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $etLOc$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $etLOc$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $etLOc$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $etLOc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $etLOc$jsxs)("radialGradient", {
                        id: "a01c0c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $etLOc$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $etLOc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $etLOc$jsx)("clipPath", {
                        id: "a01c0c__b",
                        children: /*#__PURE__*/ (0, $etLOc$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $etLOc$jsx)("clipPath", {
                id: "a01c0c__a",
                children: /*#__PURE__*/ (0, $etLOc$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M19.52 96.801c0-11.294 4.776-21.472 12.418-28.625 1.58 15.08 10.261 28.046 22.638 35.494 1.769 12.132 8.152 22.766 17.311 30.065-4.115 1.467-8.548 2.266-13.168 2.266-21.65 0-39.2-17.55-39.2-39.2m68.592 36.934c4.116 1.467 8.548 2.266 13.168 2.266 21.649 0 39.2-17.55 39.2-39.2 0-12.289-5.655-23.257-14.505-30.444-.979 14.829-8.807 27.786-20.349 35.738-1.427 12.785-7.96 24.027-17.514 31.64m28.375-81.63C111.701 35.859 96.675 24 78.879 24c-18.134 0-33.393 12.314-37.87 29.036 5.47-2.215 11.447-3.434 17.71-3.434 7.656 0 14.886 1.822 21.28 5.058 6.395-3.236 13.625-5.058 21.281-5.058 5.32 0 10.435.88 15.207 2.503m-76.802 10.42q-.006.337-.006.675c0 12.289 5.655 23.257 14.505 30.444.902-13.665 7.621-25.742 17.703-33.776-4.115-1.467-8.548-2.266-13.168-2.266-6.907 0-13.398 1.786-19.034 4.923m78.351-1.172q.043.918.043 1.847c0 11.294-4.776 21.472-12.419 28.625-1.353-12.916-7.916-24.28-17.548-31.957 4.116-1.467 8.548-2.266 13.168-2.266 5.995 0 11.675 1.345 16.756 3.751M80 129.728c8.038-5.206 14.049-13.275 16.589-22.763-5.469 2.215-11.447 3.435-17.71 3.435-5.32 0-10.435-.881-15.207-2.504 2.676 9.085 8.554 16.797 16.328 21.832m17.92-32.927c0-13.804-7.136-25.942-17.92-32.926-10.784 6.984-17.92 19.122-17.92 32.926q0 .93.043 1.847c5.08 2.406 10.761 3.752 16.756 3.752 6.908 0 13.398-1.787 19.034-4.924q.006-.337.006-.675"
                })
            }),
            /*#__PURE__*/ (0, $etLOc$jsx)("g", {
                clipPath: "url(#a01c0c__a)",
                children: /*#__PURE__*/ (0, $etLOc$jsxs)("g", {
                    clipPath: "url(#a01c0c__b)",
                    children: [
                        /*#__PURE__*/ (0, $etLOc$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $etLOc$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#a01c0c__c)"
                        }),
                        /*#__PURE__*/ (0, $etLOc$jsx)("path", {
                            fill: "url(#a01c0c__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $etLOc$jsx)("path", {
                            fill: "url(#a01c0c__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $etLOc$jsx)("path", {
                            fill: "url(#a01c0c__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $1cb7c47f3fb3ff81$var$ForwardRef = /*#__PURE__*/ (0, $etLOc$forwardRef)($1cb7c47f3fb3ff81$var$SvgComponent);
var $1cb7c47f3fb3ff81$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1cb7c47f3fb3ff81$var$ForwardRef);


export {$1cb7c47f3fb3ff81$export$2e2bcd8739ae039 as default};
