// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_AIchat_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lwl56$jsxs, jsx as $lwl56$jsx} from "react/jsx-runtime";
import {forwardRef as $lwl56$forwardRef} from "react";





const $803c7e8c7ec3321b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lwl56$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lwl56$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lwl56$jsxs)("linearGradient", {
                        id: "4f798c__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lwl56$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $lwl56$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lwl56$jsxs)("linearGradient", {
                        id: "4f798c__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lwl56$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $lwl56$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lwl56$jsxs)("linearGradient", {
                        id: "4f798c__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lwl56$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lwl56$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $lwl56$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $lwl56$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lwl56$jsxs)("radialGradient", {
                        id: "4f798c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lwl56$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lwl56$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lwl56$jsx)("clipPath", {
                        id: "4f798c__b",
                        children: /*#__PURE__*/ (0, $lwl56$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lwl56$jsx)("clipPath", {
                id: "4f798c__a",
                children: /*#__PURE__*/ (0, $lwl56$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m30.92 18.423.008-.041.615-2.85.007-.037c.078-.412-.038-.84-.311-1.149l-1.897-2.148c-.177-.2-.28-.424-.32-.651-.155-.859.57-1.76 1.53-1.536l2.731.641c.405.096.831-.023 1.137-.317l2.059-1.98c.913-.878 2.367-.003 2.095 1.263l-.615 2.85c-.092.423.022.868.304 1.187l1.897 2.148c.832.94.025 2.43-1.162 2.196l-.028-.005-.02-.005-2.731-.64c-.406-.096-.832.023-1.137.317l-2.059 1.98c-.904.869-2.336.02-2.103-1.223m5.403 1.518 2.056.482q.316.074.619.095V26.5c0 1.942-1.585 3.482-3.54 3.482H25.39l-7.879 6.738c-.767.66-1.96.115-1.952-.894.013-1.81.006-4.185 0-5.844h-3.015c-1.955 0-3.54-1.54-3.54-3.482V15.029c0-1.943 1.585-3.482 3.54-3.482h13.964c.046.795.341 1.606.96 2.306l1.51 1.71-.494 2.294c-.377 1.753.467 3.325 1.757 4.102 1.307.788 3.159.793 4.516-.512zm-9.332 5.196-2.008-.453c-.265-.06-.541.015-.741.198l-1.512 1.396c-.596.55-1.544.001-1.366-.788l.453-2.008c.06-.265-.015-.541-.198-.74l-1.396-1.513c-.55-.596-.001-1.544.788-1.366l2.008.453c.265.06.541-.015.74-.198l1.513-1.396c.596-.55 1.544-.001 1.366.788l-.453 2.008c-.06.265.015.541.198.74l1.396 1.513c.55.596.001 1.544-.788 1.366"
                })
            }),
            /*#__PURE__*/ (0, $lwl56$jsx)("g", {
                clipPath: "url(#4f798c__a)",
                children: /*#__PURE__*/ (0, $lwl56$jsxs)("g", {
                    clipPath: "url(#4f798c__b)",
                    children: [
                        /*#__PURE__*/ (0, $lwl56$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $lwl56$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#4f798c__c)"
                        }),
                        /*#__PURE__*/ (0, $lwl56$jsx)("path", {
                            fill: "url(#4f798c__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $lwl56$jsx)("path", {
                            fill: "url(#4f798c__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $lwl56$jsx)("path", {
                            fill: "url(#4f798c__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $803c7e8c7ec3321b$var$ForwardRef = /*#__PURE__*/ (0, $lwl56$forwardRef)($803c7e8c7ec3321b$var$SvgComponent);
var $803c7e8c7ec3321b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($803c7e8c7ec3321b$var$ForwardRef);


export {$803c7e8c7ec3321b$export$2e2bcd8739ae039 as default};
