// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_megaphonePromote2_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3JYuB$reactjsxruntime = require("react/jsx-runtime");
var $3JYuB$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9638b910190f53ad$export$2e2bcd8739ae039);




const $9638b910190f53ad$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6cedc7__d",
                        x1: 110.785,
                        x2: 40.447,
                        y1: 16.683,
                        y2: 113.425,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6cedc7__e",
                        x1: 64.565,
                        x2: 77.605,
                        y1: -26.004,
                        y2: 62.525,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6cedc7__f",
                        x1: -104.301,
                        x2: 140.787,
                        y1: 151.51,
                        y2: 100.127,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6cedc7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 178.298 -189.141 0 132.774 101.839)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("clipPath", {
                        id: "6cedc7__b",
                        children: /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 16,
                            y: 20.654,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("clipPath", {
                id: "6cedc7__a",
                children: /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m69.24 48.02 38.56-11.71.01-.01c5.75-1.75 11.55 2.55 11.55 8.56v43.86c0 6.01-5.8 10.31-11.55 8.56L69.25 85.55h-.04l10.44 24.25c2.24 5.18-.14 11.19-5.31 13.43-5.18 2.24-11.19-.14-13.43-5.31L46.84 85.56h-5.32c-10.36 0-18.77-8.41-18.77-18.77s8.4-18.77 18.77-18.77zm67.18 34.34 5.75 1.82c2.1.67 3.27 2.91 2.6 5.02-.54 1.7-2.11 2.79-3.81 2.79-.4 0-.81-.06-1.21-.19L134 89.98c-2.1-.67-3.27-2.91-2.6-5.02.67-2.1 2.91-3.27 5.02-2.6m-1.209-31.48c.4 0 .81-.06 1.21-.19l5.75-1.82c2.11-.67 3.27-2.92 2.6-5.02-.67-2.11-2.91-3.27-5.02-2.6l-5.75 1.82c-2.11.67-3.27 2.92-2.6 5.02.54 1.7 2.11 2.79 3.81 2.79m6.02 19.65h-6.02c-2.21 0-4-1.79-4-4s1.79-4 4-4h6.02c2.21 0 4 1.79 4 4s-1.79 4-4 4"
                })
            }),
            /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6cedc7__a)",
                children: /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6cedc7__b)",
                    children: [
                        /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 16,
                            y: 20.654,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("ellipse", {
                            cx: 132.774,
                            cy: 101.839,
                            fill: "url(#6cedc7__c)",
                            rx: 189.141,
                            ry: 178.298
                        }),
                        /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("path", {
                            fill: "url(#6cedc7__d)",
                            d: "M4.21 59.333C2.107 16.649 37.11-19.56 82.389-21.542s83.69 31.014 85.792 73.698-32.899 78.892-78.178 80.874c-45.28 1.982-83.69-31.013-85.793-73.697"
                        }),
                        /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("path", {
                            fill: "url(#6cedc7__e)",
                            d: "M49.095-64.799c-10.215-5.958-23.156-5.994-33.409-.094L-86.856-5.873c-10.043 5.78-10.06 19.425-.03 25.227L14.977 78.29c10.237 5.922 23.179 5.915 33.407-.02L150.04 19.277c9.976-5.79 9.994-19.358.031-25.17z"
                        }),
                        /*#__PURE__*/ (0, $3JYuB$reactjsxruntime.jsx)("path", {
                            fill: "url(#6cedc7__f)",
                            d: "M-49.305 79.722c21.136 0 40.28 8.063 54.142 21.072 27.723 26.062 50.706 26.085 78.429.067 13.861-13.076 33.03-21.162 54.189-21.162 42.295.023 76.557 32.233 76.557 71.994s-34.262 71.95-76.557 71.95c-21.207 0-40.375-8.108-54.237-21.184-27.699-25.973-50.658-25.951-78.38.112-13.862 13.031-33.007 21.072-54.143 21.072-42.27 0-76.533-32.21-76.533-71.95 0-39.739 34.262-71.971 76.533-71.971"
                        })
                    ]
                })
            })
        ]
    });
const $9638b910190f53ad$var$ForwardRef = /*#__PURE__*/ (0, $3JYuB$react.forwardRef)($9638b910190f53ad$var$SvgComponent);
var $9638b910190f53ad$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9638b910190f53ad$var$ForwardRef);
