// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_AIGenerate_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $45TbG$jsxs, jsx as $45TbG$jsx} from "react/jsx-runtime";
import {forwardRef as $45TbG$forwardRef} from "react";





const $ab483b9fed39b291$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $45TbG$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $45TbG$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $45TbG$jsxs)("linearGradient", {
                        id: "62cb81__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $45TbG$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $45TbG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $45TbG$jsxs)("linearGradient", {
                        id: "62cb81__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $45TbG$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $45TbG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $45TbG$jsxs)("linearGradient", {
                        id: "62cb81__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $45TbG$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $45TbG$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $45TbG$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $45TbG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $45TbG$jsxs)("radialGradient", {
                        id: "62cb81__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $45TbG$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $45TbG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $45TbG$jsx)("clipPath", {
                        id: "62cb81__b",
                        children: /*#__PURE__*/ (0, $45TbG$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $45TbG$jsx)("clipPath", {
                id: "62cb81__a",
                children: /*#__PURE__*/ (0, $45TbG$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M38.075 7.775c.916-.741 2.268.119 2.004 1.343l-.614 2.85c-.092.423.022.868.304 1.187l1.897 2.148c.556.63.379 1.505-.155 1.945-.274.221-.639.329-1.046.234l-2.732-.642c-.405-.095-.831.024-1.137.318l-2.058 1.98c-.914.878-2.367.002-2.095-1.263l.614-2.85c.092-.423-.022-.868-.304-1.187l-1.897-2.148c-.556-.63-.379-1.505.155-1.945.274-.221.639-.33 1.046-.234l2.732.641c.373.089.765-.005 1.063-.251q.042-.034.082-.073l2.059-1.98q.04-.038.082-.073M28.078 11.5H14.516c-2.21 0-4 1.79-4 4v19c0 2.21 1.79 4 4 4h19c2.209 0 4-1.79 4-4V19.742l-1.245 1.197c-1.357 1.305-3.209 1.3-4.516.513-1.29-.778-2.133-2.35-1.756-4.103l.494-2.293-1.51-1.71c-.5-.566-.788-1.203-.905-1.846m-1.59 14.684 2.008.453c.79.178 1.338-.77.788-1.366l-1.395-1.512c-.184-.2-.258-.476-.198-.741l.452-2.008c.178-.79-.77-1.338-1.366-.788l-1.512 1.395c-.2.184-.475.258-.741.199l-2.008-.453c-.789-.178-1.337.77-.787 1.366l1.395 1.512c.183.2.257.476.198.741l-.453 2.008c-.178.79.77 1.337 1.366.788l1.512-1.396c.2-.183.476-.257.742-.198"
                })
            }),
            /*#__PURE__*/ (0, $45TbG$jsx)("g", {
                clipPath: "url(#62cb81__a)",
                children: /*#__PURE__*/ (0, $45TbG$jsxs)("g", {
                    clipPath: "url(#62cb81__b)",
                    children: [
                        /*#__PURE__*/ (0, $45TbG$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $45TbG$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#62cb81__c)"
                        }),
                        /*#__PURE__*/ (0, $45TbG$jsx)("path", {
                            fill: "url(#62cb81__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $45TbG$jsx)("path", {
                            fill: "url(#62cb81__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $45TbG$jsx)("path", {
                            fill: "url(#62cb81__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $ab483b9fed39b291$var$ForwardRef = /*#__PURE__*/ (0, $45TbG$forwardRef)($ab483b9fed39b291$var$SvgComponent);
var $ab483b9fed39b291$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ab483b9fed39b291$var$ForwardRef);


export {$ab483b9fed39b291$export$2e2bcd8739ae039 as default};
