// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_plugin_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $1Vq4X$jsxs, jsx as $1Vq4X$jsx} from "react/jsx-runtime";
import {forwardRef as $1Vq4X$forwardRef} from "react";





const $9f47b19733905c46$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1Vq4X$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1Vq4X$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1Vq4X$jsxs)("linearGradient", {
                        id: "4c64ee__d",
                        x1: 65.566,
                        x2: 25.102,
                        y1: 11.636,
                        y2: 67.288,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1Vq4X$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $1Vq4X$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1Vq4X$jsxs)("linearGradient", {
                        id: "4c64ee__e",
                        x1: 38.978,
                        x2: 46.479,
                        y1: -12.921,
                        y2: 38.007,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1Vq4X$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $1Vq4X$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1Vq4X$jsxs)("linearGradient", {
                        id: "4c64ee__f",
                        x1: -58.165,
                        x2: 82.825,
                        y1: 89.197,
                        y2: 59.638,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1Vq4X$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1Vq4X$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $1Vq4X$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $1Vq4X$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1Vq4X$jsxs)("radialGradient", {
                        id: "4c64ee__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 78.212 60.623)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1Vq4X$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1Vq4X$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1Vq4X$jsx)("clipPath", {
                        id: "4c64ee__b",
                        children: /*#__PURE__*/ (0, $1Vq4X$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1Vq4X$jsx)("clipPath", {
                id: "4c64ee__a",
                children: /*#__PURE__*/ (0, $1Vq4X$jsx)("path", {
                    fill: "#fff",
                    d: "M73.755 32.8v-8c0-1.768-1.281-3.2-2.862-3.2H56.585c-1.58 0-2.861 1.432-2.861 3.2v8H42.278v-8c0-1.768-1.281-3.2-2.862-3.2H25.11c-1.58 0-2.862 1.432-2.862 3.2v8h-2.862c-4.74 0-8.584 4.298-8.584 9.6v27.2c0 5.302 3.843 9.6 8.584 9.6h57.231c4.741 0 8.585-4.299 8.585-9.6V42.4c0-5.302-3.844-9.6-8.585-9.6z"
                })
            }),
            /*#__PURE__*/ (0, $1Vq4X$jsx)("g", {
                clipPath: "url(#4c64ee__a)",
                children: /*#__PURE__*/ (0, $1Vq4X$jsxs)("g", {
                    clipPath: "url(#4c64ee__b)",
                    children: [
                        /*#__PURE__*/ (0, $1Vq4X$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $1Vq4X$jsx)("ellipse", {
                            cx: 78.212,
                            cy: 60.623,
                            fill: "url(#4c64ee__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $1Vq4X$jsx)("path", {
                            fill: "url(#4c64ee__d)",
                            d: "M4.256 36.171C3.046 11.616 23.182-9.213 49.23-10.353s48.144 17.84 49.353 42.395S79.658 77.426 53.61 78.566 5.466 60.726 4.256 36.171"
                        }),
                        /*#__PURE__*/ (0, $1Vq4X$jsx)("path", {
                            fill: "url(#4c64ee__e)",
                            d: "M30.078-35.238c-5.876-3.427-13.321-3.448-19.219-.054L-48.13-1.34c-5.777 3.326-5.787 11.174-.017 14.512l58.599 33.904c5.889 3.407 13.334 3.403 19.218-.012l58.478-33.936c5.74-3.33 5.75-11.135.018-14.479z"
                        }),
                        /*#__PURE__*/ (0, $1Vq4X$jsx)("path", {
                            fill: "url(#4c64ee__f)",
                            d: "M-26.528 47.9c12.159 0 23.172 4.638 31.146 12.122 15.948 14.993 29.17 15.005 45.118.038 7.974-7.522 19-12.173 31.173-12.173 24.33.013 44.04 18.542 44.04 41.415 0 22.874-19.71 41.391-44.04 41.391-12.2 0-23.227-4.665-31.2-12.187-15.935-14.941-29.143-14.928-45.09.064-7.975 7.497-18.988 12.123-31.147 12.123-24.317 0-44.026-18.53-44.026-41.39 0-22.861 19.71-41.403 44.026-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $9f47b19733905c46$var$ForwardRef = /*#__PURE__*/ (0, $1Vq4X$forwardRef)($9f47b19733905c46$var$SvgComponent);
var $9f47b19733905c46$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9f47b19733905c46$var$ForwardRef);


export {$9f47b19733905c46$export$2e2bcd8739ae039 as default};
