// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_shoppingCart_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $68UEd$jsxs, jsx as $68UEd$jsx} from "react/jsx-runtime";
import {forwardRef as $68UEd$forwardRef} from "react";





const $83d67d3c68619f56$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $68UEd$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $68UEd$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $68UEd$jsxs)("linearGradient", {
                        id: "20c82e__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $68UEd$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $68UEd$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $68UEd$jsxs)("linearGradient", {
                        id: "20c82e__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $68UEd$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $68UEd$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $68UEd$jsxs)("linearGradient", {
                        id: "20c82e__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $68UEd$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $68UEd$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $68UEd$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $68UEd$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $68UEd$jsxs)("radialGradient", {
                        id: "20c82e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $68UEd$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $68UEd$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $68UEd$jsx)("clipPath", {
                        id: "20c82e__b",
                        children: /*#__PURE__*/ (0, $68UEd$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $68UEd$jsx)("clipPath", {
                id: "20c82e__a",
                children: /*#__PURE__*/ (0, $68UEd$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M13.492 16.152c-1.104 0-2 .896-2 2s.896 2 2 2l.725.004h.001l1.236.008c1.05.006 2.457.011 4.112.01 1.987 0 3.678 1.43 3.998 3.371l5.89 35.721c.637 3.864 3.977 6.699 7.893 6.699h32.339c1.104 0 2-.896 2-2s-.896-2-2-2H37.347c-1.958 0-3.628-1.418-3.946-3.35l-1.256-7.612h35.033c2.866 0 5.332-2.027 5.886-4.839l2.47-12.522c.73-3.698-2.089-7.152-5.868-7.157-12.455-.018-30.5-.002-38.842.006h-.002l-2.718.002-.593-3.599c-.642-3.893-4.021-6.72-7.947-6.72-1.645.001-3.043-.004-4.088-.01l-1.203-.007zM41.01 78c0 2.761-2.239 5-5 5s-5-2.239-5-5 2.238-5 5-5 5 2.239 5 5m24.924 5c2.761 0 5-2.239 5-5s-2.239-5-5-5-5 2.239-5 5 2.238 5 5 5"
                })
            }),
            /*#__PURE__*/ (0, $68UEd$jsx)("g", {
                clipPath: "url(#20c82e__a)",
                children: /*#__PURE__*/ (0, $68UEd$jsxs)("g", {
                    clipPath: "url(#20c82e__b)",
                    children: [
                        /*#__PURE__*/ (0, $68UEd$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $68UEd$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#20c82e__c)"
                        }),
                        /*#__PURE__*/ (0, $68UEd$jsx)("path", {
                            fill: "url(#20c82e__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $68UEd$jsx)("path", {
                            fill: "url(#20c82e__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $68UEd$jsx)("path", {
                            fill: "url(#20c82e__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $83d67d3c68619f56$var$ForwardRef = /*#__PURE__*/ (0, $68UEd$forwardRef)($83d67d3c68619f56$var$SvgComponent);
var $83d67d3c68619f56$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($83d67d3c68619f56$var$ForwardRef);


export {$83d67d3c68619f56$export$2e2bcd8739ae039 as default};
