// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fileText_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $chRZo$jsxs, jsx as $chRZo$jsx} from "react/jsx-runtime";
import {forwardRef as $chRZo$forwardRef} from "react";





const $5a06725625eaabed$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $chRZo$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $chRZo$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $chRZo$jsxs)("linearGradient", {
                        id: "94a763__d",
                        x1: 33.286,
                        x2: 9.233,
                        y1: 1.896,
                        y2: 33.083,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $chRZo$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $chRZo$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $chRZo$jsxs)("linearGradient", {
                        id: "94a763__e",
                        x1: 18.631,
                        x2: 23.271,
                        y1: -12.461,
                        y2: 17.236,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $chRZo$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $chRZo$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $chRZo$jsxs)("linearGradient", {
                        id: "94a763__f",
                        x1: -34.908,
                        x2: 43.165,
                        y1: 47.244,
                        y2: 31.814,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $chRZo$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $chRZo$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $chRZo$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $chRZo$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $chRZo$jsxs)("radialGradient", {
                        id: "94a763__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 59.9687 -59.9688 0 40.258 30.538)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $chRZo$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $chRZo$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $chRZo$jsx)("clipPath", {
                        id: "94a763__b",
                        children: /*#__PURE__*/ (0, $chRZo$jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.234,
                            y: 3.232,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $chRZo$jsx)("clipPath", {
                id: "94a763__a",
                children: /*#__PURE__*/ (0, $chRZo$jsx)("path", {
                    fill: "#fff",
                    d: "m36.146 19.346-9.89-9.891C25.58 8.78 24.663 8.4 23.71 8.4H14.4c-1.988 0-3.6 1.612-3.6 3.6v24c0 1.989 1.612 3.6 3.6 3.6h19.2c1.989 0 3.6-1.611 3.6-3.6V21.892c0-.955-.379-1.87-1.054-2.546M31.201 25.2c0 .663-.538 1.2-1.2 1.2-.663 0-1.2-.537-1.2-1.2V24h-3.6v9.6h1.2c.663 0 1.2.538 1.2 1.2 0 .663-.537 1.2-1.2 1.2h-4.8c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.2-1.2h1.2V24h-3.6v1.2c0 .663-.538 1.2-1.2 1.2-.663 0-1.2-.537-1.2-1.2v-2.4c0-.662.537-1.2 1.2-1.2h12c.662 0 1.2.538 1.2 1.2z"
                })
            }),
            /*#__PURE__*/ (0, $chRZo$jsx)("g", {
                clipPath: "url(#94a763__a)",
                children: /*#__PURE__*/ (0, $chRZo$jsxs)("g", {
                    clipPath: "url(#94a763__b)",
                    children: [
                        /*#__PURE__*/ (0, $chRZo$jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.234,
                            y: 3.232,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $chRZo$jsx)("circle", {
                            cx: 40.258,
                            cy: 30.538,
                            r: 59.969,
                            fill: "url(#94a763__c)"
                        }),
                        /*#__PURE__*/ (0, $chRZo$jsx)("path", {
                            fill: "url(#94a763__d)",
                            d: "M-.504 16.241C-1.171 1.885 9.926-10.294 24.283-10.96S50.818-.53 51.484 13.827s-10.43 26.535-24.787 27.201S.162 30.598-.504 16.241"
                        }),
                        /*#__PURE__*/ (0, $chRZo$jsx)("path", {
                            fill: "url(#94a763__e)",
                            d: "M13.726-25.509c-3.238-2.004-7.342-2.016-10.592-.032L-29.378-5.69c-3.184 1.944-3.19 6.533-.01 8.485L2.91 22.617c3.246 1.992 7.35 1.99 10.592-.007l32.23-19.84c3.164-1.948 3.17-6.511.01-8.466z"
                        }),
                        /*#__PURE__*/ (0, $chRZo$jsx)("path", {
                            fill: "url(#94a763__f)",
                            d: "M-17.471 23.098c6.701 0 12.771 2.713 17.166 7.088 8.79 8.766 16.077 8.773 24.867.022 4.394-4.397 10.472-7.117 17.18-7.117 13.41.007 24.274 10.84 24.274 24.214s-10.864 24.2-24.273 24.2c-6.724 0-12.802-2.727-17.197-7.125-8.782-8.736-16.061-8.728-24.851.037C-4.7 68.8-10.77 71.505-17.471 71.505c-13.402 0-24.266-10.834-24.266-24.2s10.864-24.207 24.266-24.207"
                        })
                    ]
                })
            })
        ]
    });
const $5a06725625eaabed$var$ForwardRef = /*#__PURE__*/ (0, $chRZo$forwardRef)($5a06725625eaabed$var$SvgComponent);
var $5a06725625eaabed$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5a06725625eaabed$var$ForwardRef);


export {$5a06725625eaabed$export$2e2bcd8739ae039 as default};
