// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_diamond_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $djjnM$jsxs, jsx as $djjnM$jsx} from "react/jsx-runtime";
import {forwardRef as $djjnM$forwardRef} from "react";





const $dd2f99d1b14c030e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $djjnM$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $djjnM$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $djjnM$jsxs)("linearGradient", {
                        id: "a5057f__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $djjnM$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $djjnM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $djjnM$jsxs)("linearGradient", {
                        id: "a5057f__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $djjnM$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $djjnM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $djjnM$jsxs)("linearGradient", {
                        id: "a5057f__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $djjnM$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $djjnM$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $djjnM$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $djjnM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $djjnM$jsxs)("radialGradient", {
                        id: "a5057f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $djjnM$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $djjnM$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $djjnM$jsx)("clipPath", {
                        id: "a5057f__b",
                        children: /*#__PURE__*/ (0, $djjnM$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $djjnM$jsx)("clipPath", {
                id: "a5057f__a",
                children: /*#__PURE__*/ (0, $djjnM$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M24.515 8.352h-1.023l-3.943 8.767h8.908l-3.915-8.704q-.015-.03-.027-.063m6.684 8.767-3.944-8.767h3.859c.526 0 1.031.207 1.406.577l7.822 7.736q.21.207.345.454zm-2.434 2.5h-9.524l4.762 18.032zm-2.123 17.832 4.709-17.832h8.931zm-9.8-29.1h3.91l-3.945 8.768H7.313q.135-.245.34-.452l7.778-7.734c.375-.372.882-.581 1.41-.581M7.717 19.62h8.937l4.74 17.95z"
                })
            }),
            /*#__PURE__*/ (0, $djjnM$jsx)("g", {
                clipPath: "url(#a5057f__a)",
                children: /*#__PURE__*/ (0, $djjnM$jsxs)("g", {
                    clipPath: "url(#a5057f__b)",
                    children: [
                        /*#__PURE__*/ (0, $djjnM$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $djjnM$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#a5057f__c)"
                        }),
                        /*#__PURE__*/ (0, $djjnM$jsx)("path", {
                            fill: "url(#a5057f__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $djjnM$jsx)("path", {
                            fill: "url(#a5057f__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $djjnM$jsx)("path", {
                            fill: "url(#a5057f__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $dd2f99d1b14c030e$var$ForwardRef = /*#__PURE__*/ (0, $djjnM$forwardRef)($dd2f99d1b14c030e$var$SvgComponent);
var $dd2f99d1b14c030e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($dd2f99d1b14c030e$var$ForwardRef);


export {$dd2f99d1b14c030e$export$2e2bcd8739ae039 as default};
