// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_mention_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $iRXtj$jsxs, jsx as $iRXtj$jsx} from "react/jsx-runtime";
import {forwardRef as $iRXtj$forwardRef} from "react";





const $9644cff9fff771bb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iRXtj$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iRXtj$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iRXtj$jsxs)("linearGradient", {
                        id: "d77ecf__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iRXtj$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $iRXtj$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iRXtj$jsxs)("linearGradient", {
                        id: "d77ecf__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iRXtj$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $iRXtj$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iRXtj$jsxs)("linearGradient", {
                        id: "d77ecf__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iRXtj$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iRXtj$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $iRXtj$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $iRXtj$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iRXtj$jsxs)("radialGradient", {
                        id: "d77ecf__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iRXtj$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iRXtj$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iRXtj$jsx)("clipPath", {
                        id: "d77ecf__b",
                        children: /*#__PURE__*/ (0, $iRXtj$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iRXtj$jsx)("clipPath", {
                id: "d77ecf__a",
                children: /*#__PURE__*/ (0, $iRXtj$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M18.967 7.154c4.91-1.61 10.173-1.073 13.6.474 7.03 3.171 9.35 9.989 8.323 15.826-1.018 5.785-5.446 11.009-12.158 10.544-6.154-.425-9.478-2.761-10.826-5.84-1.294-2.956-.575-6.197.47-8.127 2.7-4.988 7.322-5.749 10.168-5.658 1.875.06 3.001 1.754 2.852 3.374l-1.265 13.759c4.393-.304 7.497-3.936 8.297-8.485.864-4.91-1.073-10.49-6.889-13.114-2.858-1.29-7.477-1.792-11.793-.377-4.242 1.39-8.199 4.631-9.91 10.91-1.352 4.965.228 9.722 3.274 13.25 3.065 3.55 7.507 5.729 11.657 5.609.69-.02 1.265.523 1.285 1.213s-.523 1.266-1.213 1.286c-5.029.145-10.155-2.458-13.621-6.474-3.487-4.038-5.403-9.632-3.793-15.542C9.37 12.638 13.983 8.79 18.967 7.154m8.665 24.228 1.275-13.864c.018-.2-.044-.373-.134-.486q-.124-.157-.308-.16c-2.372-.076-5.828.54-7.89 4.35-.794 1.466-1.278 3.88-.379 5.934.78 1.779 2.762 3.64 7.436 4.226"
                })
            }),
            /*#__PURE__*/ (0, $iRXtj$jsx)("g", {
                clipPath: "url(#d77ecf__a)",
                children: /*#__PURE__*/ (0, $iRXtj$jsxs)("g", {
                    clipPath: "url(#d77ecf__b)",
                    children: [
                        /*#__PURE__*/ (0, $iRXtj$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $iRXtj$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#d77ecf__c)"
                        }),
                        /*#__PURE__*/ (0, $iRXtj$jsx)("path", {
                            fill: "url(#d77ecf__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $iRXtj$jsx)("path", {
                            fill: "url(#d77ecf__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $iRXtj$jsx)("path", {
                            fill: "url(#d77ecf__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $9644cff9fff771bb$var$ForwardRef = /*#__PURE__*/ (0, $iRXtj$forwardRef)($9644cff9fff771bb$var$SvgComponent);
var $9644cff9fff771bb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9644cff9fff771bb$var$ForwardRef);


export {$9644cff9fff771bb$export$2e2bcd8739ae039 as default};
