// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lockClose_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $75WR6$jsxs, jsx as $75WR6$jsx} from "react/jsx-runtime";
import {forwardRef as $75WR6$forwardRef} from "react";





const $854faadb7be633ab$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $75WR6$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $75WR6$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $75WR6$jsxs)("linearGradient", {
                        id: "a638a9__d",
                        x1: 66.664,
                        x2: 18.316,
                        y1: 3.573,
                        y2: 66.258,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $75WR6$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $75WR6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $75WR6$jsxs)("linearGradient", {
                        id: "a638a9__e",
                        x1: 37.208,
                        x2: 46.535,
                        y1: -25.285,
                        y2: 34.406,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $75WR6$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $75WR6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $75WR6$jsxs)("linearGradient", {
                        id: "a638a9__f",
                        x1: -70.407,
                        x2: 86.52,
                        y1: 94.721,
                        y2: 63.707,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $75WR6$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $75WR6$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $75WR6$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $75WR6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $75WR6$jsxs)("radialGradient", {
                        id: "a638a9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 9.766 70.908)scale(120.537)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $75WR6$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $75WR6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $75WR6$jsx)("clipPath", {
                        id: "a638a9__b",
                        children: /*#__PURE__*/ (0, $75WR6$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $75WR6$jsx)("clipPath", {
                id: "a638a9__a",
                children: /*#__PURE__*/ (0, $75WR6$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M32.331 27.82c0-8.654 7.016-15.67 15.67-15.67s15.67 7.016 15.67 15.67v12.356h6.027c4.327 0 7.834 3.507 7.834 7.834v23.506c0 4.327-3.507 7.834-7.834 7.834H26.304c-4.327 0-7.835-3.507-7.835-7.834V48.01c0-4.327 3.508-7.834 7.835-7.834h6.027zm4.822 12.356H58.85V27.82c0-5.991-4.857-10.848-10.849-10.848-5.991 0-10.848 4.857-10.848 10.848zm13.26 20.75c1.44-.834 2.41-2.392 2.41-4.176 0-2.663-2.159-4.822-4.822-4.822-2.662 0-4.821 2.159-4.821 4.822 0 1.784.97 3.343 2.41 4.176v4.864c0 1.331 1.08 2.41 2.411 2.41 1.332 0 2.411-1.079 2.411-2.41z",
                    opacity: 0.9
                })
            }),
            /*#__PURE__*/ (0, $75WR6$jsx)("g", {
                clipPath: "url(#a638a9__a)",
                children: /*#__PURE__*/ (0, $75WR6$jsxs)("g", {
                    clipPath: "url(#a638a9__b)",
                    children: [
                        /*#__PURE__*/ (0, $75WR6$jsx)("rect", {
                            width: 83.485,
                            height: 83.485,
                            x: 6.258,
                            y: 6.258,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $75WR6$jsx)("circle", {
                            cx: 80.674,
                            cy: 61.143,
                            r: 120.537,
                            fill: "url(#a638a9__c)"
                        }),
                        /*#__PURE__*/ (0, $75WR6$jsx)("path", {
                            fill: "url(#a638a9__d)",
                            d: "M-1.255 32.406C-2.595 3.55 19.71-20.928 48.567-22.268s53.335 20.966 54.675 49.822S82.275 80.89 53.419 82.23.085 61.263-1.255 32.406"
                        }),
                        /*#__PURE__*/ (0, $75WR6$jsx)("path", {
                            fill: "url(#a638a9__e)",
                            d: "M27.349-51.511c-6.51-4.028-14.757-4.053-21.291-.064l-65.35 39.9c-6.4 3.908-6.41 13.132-.018 17.054L5.607 45.223c6.524 4.003 14.771 3.998 21.29-.015L91.68 5.328c6.358-3.915 6.37-13.087.02-17.016z"
                        }),
                        /*#__PURE__*/ (0, $75WR6$jsx)("path", {
                            fill: "url(#a638a9__f)",
                            d: "M-35.36 46.189c13.47 0 25.671 5.451 34.505 14.246 17.667 17.619 32.314 17.634 49.981.045 8.834-8.84 21.05-14.306 34.535-14.306 26.953.015 48.788 21.79 48.788 48.671s-21.835 48.641-48.789 48.641c-13.514 0-25.73-5.482-34.564-14.321-17.652-17.559-32.284-17.544-49.951.075-8.834 8.81-21.035 14.246-34.504 14.246-26.939 0-48.774-21.775-48.774-48.641 0-26.865 21.835-48.656 48.774-48.656"
                        })
                    ]
                })
            })
        ]
    });
const $854faadb7be633ab$var$ForwardRef = /*#__PURE__*/ (0, $75WR6$forwardRef)($854faadb7be633ab$var$SvgComponent);
var $854faadb7be633ab$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($854faadb7be633ab$var$ForwardRef);


export {$854faadb7be633ab$export$2e2bcd8739ae039 as default};
