// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_calendar_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $evHNP$reactjsxruntime = require("react/jsx-runtime");
var $evHNP$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $efad6e282c0367fe$export$2e2bcd8739ae039);




const $efad6e282c0367fe$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1ac8e0__d",
                        x1: 112.407,
                        x2: 37.449,
                        y1: 5.767,
                        y2: 108.862,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1ac8e0__e",
                        x1: 63.152,
                        x2: 77.048,
                        y1: -39.722,
                        y2: 54.621,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1ac8e0__f",
                        x1: -116.805,
                        x2: 144.377,
                        y1: 149.448,
                        y2: 94.691,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsxs)("radialGradient", {
                        id: "1ac8e0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 190.006 -201.561 0 135.838 96.516)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("clipPath", {
                        id: "1ac8e0__b",
                        children: /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("rect", {
                            width: 139.603,
                            height: 131.6,
                            x: 11.398,
                            y: 10,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("clipPath", {
                id: "1ac8e0__a",
                children: /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M116 34h-7v10c0 2.21-1.79 4-4 4s-4-1.79-4-4V34H59v10c0 2.21-1.79 4-4 4s-4-1.79-4-4V34h-7c-8.84 0-16 7.16-16 16v10h104V50c0-8.84-7.16-16-16-16m-88 82c0 8.84 7.16 16 16 16h72c8.84 0 16-7.16 16-16V68H28zm73-32c0-2.21 1.79-4 4-4h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4zm0 22c0-2.21 1.79-4 4-4h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4zM73 84c0-2.21 1.79-4 4-4h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4zm0 22c0-2.21 1.79-4 4-4h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4zM45 84c0-2.21 1.79-4 4-4h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4zm0 22c0-2.21 1.79-4 4-4h6c2.21 0 4 1.79 4 4v6c0 2.21-1.79 4-4 4h-6c-2.21 0-4-1.79-4-4zm14-82c0-2.21-1.79-4-4-4s-4 1.79-4 4v10h8zm50 0c0-2.21-1.79-4-4-4s-4 1.79-4 4v10h8z"
                })
            }),
            /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("g", {
                clipPath: "url(#1ac8e0__a)",
                children: /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#1ac8e0__b)",
                    children: [
                        /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("rect", {
                            width: 139.603,
                            height: 131.6,
                            x: 11.398,
                            y: 10,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("ellipse", {
                            cx: 135.838,
                            cy: 96.516,
                            fill: "url(#1ac8e0__c)",
                            rx: 201.561,
                            ry: 190.006
                        }),
                        /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("path", {
                            fill: "url(#1ac8e0__d)",
                            d: "M-1.167 51.218c-2.24-45.487 35.06-84.074 83.313-86.186s89.185 33.05 91.426 78.537c2.241 45.486-35.06 84.073-83.312 86.185-48.253 2.112-89.186-33.05-91.427-78.536"
                        }),
                        /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("path", {
                            fill: "url(#1ac8e0__e)",
                            d: "M46.666-81.064c-10.885-6.35-24.677-6.389-35.603-.1L-98.213-18.27c-10.702 6.16-10.72 20.7-.032 26.883L10.309 71.421c10.91 6.31 24.7 6.303 35.6-.023L154.24 8.533c10.631-6.17 10.65-20.629.032-26.822z"
                        }),
                        /*#__PURE__*/ (0, $evHNP$reactjsxruntime.jsx)("path", {
                            fill: "url(#1ac8e0__f)",
                            d: "M-58.197 72.946c22.523 0 42.925 8.593 57.697 22.456 29.543 27.774 54.036 27.797 83.579.071 14.771-13.934 35.199-22.551 57.747-22.551 45.072.024 81.584 34.349 81.584 76.722s-36.512 76.674-81.584 76.674c-22.599 0-43.026-8.641-57.798-22.575-29.518-27.679-53.985-27.655-83.528.119-14.772 13.887-35.174 22.456-57.697 22.456-45.047 0-81.559-34.325-81.559-76.674s36.512-76.698 81.559-76.698"
                        })
                    ]
                })
            })
        ]
    });
const $efad6e282c0367fe$var$ForwardRef = /*#__PURE__*/ (0, $evHNP$react.forwardRef)($efad6e282c0367fe$var$SvgComponent);
var $efad6e282c0367fe$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($efad6e282c0367fe$var$ForwardRef);
