// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_heart_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $9VsDH$jsxs, jsx as $9VsDH$jsx} from "react/jsx-runtime";
import {forwardRef as $9VsDH$forwardRef} from "react";





const $317ff72c698c8338$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9VsDH$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9VsDH$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9VsDH$jsxs)("linearGradient", {
                        id: "b568de__d",
                        x1: 109.454,
                        x2: 31.851,
                        y1: 4.691,
                        y2: 105.305,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9VsDH$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $9VsDH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9VsDH$jsxs)("linearGradient", {
                        id: "b568de__e",
                        x1: 62.178,
                        x2: 77.148,
                        y1: -41.628,
                        y2: 54.181,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9VsDH$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $9VsDH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9VsDH$jsxs)("linearGradient", {
                        id: "b568de__f",
                        x1: -110.558,
                        x2: 141.322,
                        y1: 150.99,
                        y2: 101.21,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9VsDH$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9VsDH$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $9VsDH$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $9VsDH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9VsDH$jsxs)("radialGradient", {
                        id: "b568de__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 17.425 114.519)scale(193.472)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9VsDH$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9VsDH$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9VsDH$jsx)("clipPath", {
                        id: "b568de__b",
                        children: /*#__PURE__*/ (0, $9VsDH$jsx)("rect", {
                            width: 134,
                            height: 134,
                            x: 12.5,
                            y: 9,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9VsDH$jsx)("clipPath", {
                id: "b568de__a",
                children: /*#__PURE__*/ (0, $9VsDH$jsx)("path", {
                    fill: "#fff",
                    d: "M108.008 26c-15.463.01-27.996 12.66-27.996 28.262-.004.014-.008.045-.012.065-.004-.02-.008-.05-.012-.065 0-15.603-12.533-28.252-27.996-28.262-17.052-.01-29.247 14.7-27.889 31.848 2.458 31.016 35.866 65.837 49.824 77.019 3.549 2.843 8.573 2.844 12.123.001 13.964-11.181 47.39-46.005 49.847-77.023C137.255 40.698 125.059 25.99 108.008 26"
                })
            }),
            /*#__PURE__*/ (0, $9VsDH$jsx)("g", {
                clipPath: "url(#b568de__a)",
                children: /*#__PURE__*/ (0, $9VsDH$jsxs)("g", {
                    clipPath: "url(#b568de__b)",
                    children: [
                        /*#__PURE__*/ (0, $9VsDH$jsx)("rect", {
                            width: 134,
                            height: 134,
                            x: 12.5,
                            y: 9,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $9VsDH$jsx)("circle", {
                            cx: 131.944,
                            cy: 97.094,
                            r: 193.472,
                            fill: "url(#b568de__c)"
                        }),
                        /*#__PURE__*/ (0, $9VsDH$jsx)("path", {
                            fill: "url(#b568de__d)",
                            d: "M.439 50.97C-1.712 4.654 34.09-34.636 80.407-36.787c46.317-2.15 85.607 33.653 87.757 79.969s-33.652 85.606-79.968 87.757S2.589 97.286.439 50.97"
                        }),
                        /*#__PURE__*/ (0, $9VsDH$jsx)("path", {
                            fill: "url(#b568de__e)",
                            d: "M46.353-83.724c-10.448-6.465-23.686-6.505-34.174-.102L-92.71-19.783C-102.984-13.511-103 1.294-92.742 7.59L11.455 71.542c10.472 6.426 23.71 6.418 34.172-.023L149.61 7.507c10.204-6.282 10.223-21.004.031-27.31z"
                        }),
                        /*#__PURE__*/ (0, $9VsDH$jsx)("path", {
                            fill: "url(#b568de__f)",
                            d: "M-54.303 73.093c21.62 0 41.203 8.75 55.382 22.865 28.357 28.281 51.867 28.305 80.224.073 14.179-14.189 33.787-22.963 55.43-22.963 43.263.025 78.31 34.976 78.31 78.121 0 43.146-35.047 78.073-78.31 78.073-21.692 0-41.3-8.798-55.478-22.987-28.333-28.183-51.819-28.159-80.176.121-14.179 14.14-33.762 22.866-55.382 22.866-43.239 0-78.285-34.951-78.285-78.073s35.046-78.096 78.285-78.096"
                        })
                    ]
                })
            })
        ]
    });
const $317ff72c698c8338$var$ForwardRef = /*#__PURE__*/ (0, $9VsDH$forwardRef)($317ff72c698c8338$var$SvgComponent);
var $317ff72c698c8338$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($317ff72c698c8338$var$ForwardRef);


export {$317ff72c698c8338$export$2e2bcd8739ae039 as default};
