// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_sparkles_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $57cnW$jsxs, jsx as $57cnW$jsx} from "react/jsx-runtime";
import {forwardRef as $57cnW$forwardRef} from "react";





const $b31633ceae6a5845$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $57cnW$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $57cnW$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $57cnW$jsxs)("linearGradient", {
                        id: "a93643__d",
                        x1: 32.783,
                        x2: 12.551,
                        y1: 5.819,
                        y2: 33.645,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $57cnW$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $57cnW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $57cnW$jsxs)("linearGradient", {
                        id: "a93643__e",
                        x1: 19.489,
                        x2: 23.239,
                        y1: -6.459,
                        y2: 19.005,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $57cnW$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $57cnW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $57cnW$jsxs)("linearGradient", {
                        id: "a93643__f",
                        x1: -29.084,
                        x2: 41.411,
                        y1: 44.599,
                        y2: 29.819,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $57cnW$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $57cnW$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $57cnW$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $57cnW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $57cnW$jsxs)("radialGradient", {
                        id: "a93643__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.2844 -54.4031 0 39.106 30.312)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $57cnW$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $57cnW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $57cnW$jsx)("clipPath", {
                        id: "a93643__b",
                        children: /*#__PURE__*/ (0, $57cnW$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.961,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $57cnW$jsx)("clipPath", {
                id: "a93643__a",
                children: /*#__PURE__*/ (0, $57cnW$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m37.86 25.323-5.174-1.164c-.768-.174-1.575.042-2.154.576l-3.9 3.594c-1.731 1.596-4.485.006-3.97-2.292l1.165-5.175c.174-.768-.042-1.575-.576-2.154l-3.594-3.9c-1.596-1.73-.006-4.485 2.292-3.969l5.175 1.164c.768.174 1.575-.042 2.154-.576l3.9-3.594c1.73-1.596 4.485-.006 3.969 2.292L35.983 15.3c-.174.768.042 1.575.576 2.154l3.594 3.9c1.596 1.731.006 4.485-2.292 3.97M19.28 38.958l-3.649-.822c-.483-.108-.984.027-1.347.36l-2.748 2.535c-1.083 1-2.805.003-2.48-1.43l.821-3.649c.108-.483-.027-.984-.36-1.347l-2.535-2.748c-.999-1.083-.003-2.805 1.431-2.48l3.648.821c.483.108.984-.027 1.347-.36l2.748-2.535c1.083-.999 2.805-.003 2.481 1.431l-.822 3.648c-.108.483.027.984.36 1.347l2.535 2.748c1 1.083.003 2.805-1.43 2.481"
                })
            }),
            /*#__PURE__*/ (0, $57cnW$jsx)("g", {
                clipPath: "url(#a93643__a)",
                children: /*#__PURE__*/ (0, $57cnW$jsxs)("g", {
                    clipPath: "url(#a93643__b)",
                    children: [
                        /*#__PURE__*/ (0, $57cnW$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.961,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $57cnW$jsx)("ellipse", {
                            cx: 39.106,
                            cy: 30.312,
                            fill: "url(#a93643__c)",
                            rx: 54.403,
                            ry: 51.284
                        }),
                        /*#__PURE__*/ (0, $57cnW$jsx)("path", {
                            fill: "url(#a93643__d)",
                            d: "M2.128 18.087C1.523 5.809 11.591-4.607 24.615-5.177s24.072 8.92 24.677 21.198c.605 12.277-9.463 22.692-22.487 23.262S2.733 30.364 2.128 18.087"
                        }),
                        /*#__PURE__*/ (0, $57cnW$jsx)("path", {
                            fill: "url(#a93643__e)",
                            d: "M15.039-17.617c-2.938-1.714-6.66-1.725-9.61-.027L-24.064-.669c-2.889 1.662-2.893 5.587-.009 7.256l29.3 16.952c2.944 1.703 6.667 1.7 9.609-.006l29.24-16.968c2.869-1.666 2.874-5.568.008-7.24z"
                        }),
                        /*#__PURE__*/ (0, $57cnW$jsx)("path", {
                            fill: "url(#a93643__f)",
                            d: "M-13.266 23.95c6.08 0 11.586 2.32 15.573 6.06 7.974 7.497 14.585 7.504 22.559.02 3.987-3.76 9.5-6.087 15.586-6.087 12.166.007 22.02 9.272 22.02 20.708 0 11.437-9.854 20.695-22.02 20.695-6.1 0-11.613-2.332-15.6-6.093-7.967-7.47-14.57-7.464-22.545.032-3.987 3.748-9.494 6.061-15.573 6.061-12.158 0-22.013-9.264-22.013-20.695 0-11.43 9.855-20.701 22.013-20.701"
                        })
                    ]
                })
            })
        ]
    });
const $b31633ceae6a5845$var$ForwardRef = /*#__PURE__*/ (0, $57cnW$forwardRef)($b31633ceae6a5845$var$SvgComponent);
var $b31633ceae6a5845$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b31633ceae6a5845$var$ForwardRef);


export {$b31633ceae6a5845$export$2e2bcd8739ae039 as default};
