// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_rocket_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $dty0i$jsxs, jsx as $dty0i$jsx} from "react/jsx-runtime";
import {forwardRef as $dty0i$forwardRef} from "react";





const $73de90c82d6561f0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dty0i$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dty0i$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dty0i$jsxs)("linearGradient", {
                        id: "378ae4__d",
                        x1: 107.785,
                        x2: 37.447,
                        y1: 19.228,
                        y2: 115.97,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dty0i$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dty0i$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dty0i$jsxs)("linearGradient", {
                        id: "378ae4__e",
                        x1: 61.565,
                        x2: 74.605,
                        y1: -23.459,
                        y2: 65.07,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dty0i$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dty0i$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dty0i$jsxs)("linearGradient", {
                        id: "378ae4__f",
                        x1: -107.301,
                        x2: 137.787,
                        y1: 154.055,
                        y2: 102.672,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dty0i$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dty0i$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dty0i$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dty0i$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dty0i$jsxs)("radialGradient", {
                        id: "378ae4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 178.298 -189.141 0 129.774 104.384)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dty0i$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dty0i$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dty0i$jsx)("clipPath", {
                        id: "378ae4__b",
                        children: /*#__PURE__*/ (0, $dty0i$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 23.199,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dty0i$jsx)("clipPath", {
                id: "378ae4__a",
                children: /*#__PURE__*/ (0, $dty0i$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M107.022 87.22c-6.16 5.88-13.79 11.98-23.22 18.19h-.01c-.79.52-1.61.96-2.46 1.33-5.95 2.58-13 1.36-17.71-3.35l-3.54-3.54-3.54-3.54c-4.46-4.47-5.78-11.02-3.71-16.76.43-1.18.99-2.33 1.7-3.41 4.49-6.82 8.93-12.69 13.26-17.75 25.17-29.39 46.97-31.29 56.92-30.16 3.68.42 6.58 3.33 7 7 1.05 9.3-.53 28.94-24.69 51.99m1.7-35.99c-3.31-3.3-8.67-3.3-11.97 0-3.3 3.31-3.31 8.67 0 11.97s8.67 3.31 11.97 0 3.31-8.67 0-11.97m-64.61 30.71-10.48.18v.01c-6.81.12-10.67-7.8-6.35-13.06 12.21-14.85 28.66-20.14 39.39-21.35-6.47 6.82-12.76 14.84-18.82 24.04-2.06 3.13-3.29 6.63-3.74 10.18m-15.41 36.6c-.78.78-1.81 1.17-2.83 1.17-1.03 0-2.05-.39-2.83-1.17-1.56-1.57-1.56-4.1 0-5.66l10.5-10.5c1.57-1.56 4.1-1.56 5.66 0 1.56 1.57 1.56 4.1 0 5.66zm14-7c1.57-1.56 4.1-1.56 5.66 0 1.56 1.57 1.56 4.1 0 5.66l-20 20c-.78.78-1.8 1.17-2.83 1.17s-2.05-.39-2.83-1.17c-1.56-1.57-1.56-4.1 0-5.66zm9.22 9.21-11 11c-1.56 1.56-1.56 4.09 0 5.66.78.78 1.8 1.17 2.83 1.17s2.05-.39 2.83-1.17l11-11c1.56-1.56 1.56-4.09 0-5.66s-4.09-1.56-5.66 0m26.11-4.89-.18 10.44c-.12 6.85 7.85 10.65 13.13 6.29 14.761-12.18 20.041-28.55 21.271-39.27-6.48 6.12-14.39 12.41-24.05 18.77-3.08 2.03-6.55 3.3-10.17 3.77"
                })
            }),
            /*#__PURE__*/ (0, $dty0i$jsx)("g", {
                clipPath: "url(#378ae4__a)",
                children: /*#__PURE__*/ (0, $dty0i$jsxs)("g", {
                    clipPath: "url(#378ae4__b)",
                    children: [
                        /*#__PURE__*/ (0, $dty0i$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 23.199,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dty0i$jsx)("ellipse", {
                            cx: 129.774,
                            cy: 104.384,
                            fill: "url(#378ae4__c)",
                            rx: 189.141,
                            ry: 178.298
                        }),
                        /*#__PURE__*/ (0, $dty0i$jsx)("path", {
                            fill: "url(#378ae4__d)",
                            d: "M1.21 61.878c-2.103-42.684 32.9-78.893 78.179-80.875s83.69 31.014 85.792 73.697-32.899 78.893-78.178 80.875c-45.28 1.982-83.69-31.013-85.793-73.697"
                        }),
                        /*#__PURE__*/ (0, $dty0i$jsx)("path", {
                            fill: "url(#378ae4__e)",
                            d: "M46.095-62.254c-10.215-5.958-23.156-5.995-33.409-.094L-89.856-3.328c-10.043 5.78-10.06 19.425-.03 25.227L11.977 80.835c10.237 5.922 23.179 5.915 33.407-.021L147.04 21.822c9.976-5.79 9.994-19.357.031-25.169z"
                        }),
                        /*#__PURE__*/ (0, $dty0i$jsx)("path", {
                            fill: "url(#378ae4__f)",
                            d: "M-52.305 82.266c21.136 0 40.28 8.064 54.142 21.073 27.723 26.062 50.706 26.085 78.429.067 13.861-13.076 33.03-21.162 54.189-21.162 42.295.022 76.557 32.233 76.557 71.994s-34.262 71.95-76.557 71.95c-21.207 0-40.375-8.108-54.237-21.184-27.699-25.973-50.658-25.951-78.38.112-13.862 13.031-33.007 21.072-54.143 21.072-42.27 0-76.533-32.21-76.533-71.95 0-39.739 34.262-71.972 76.533-71.972"
                        })
                    ]
                })
            })
        ]
    });
const $73de90c82d6561f0$var$ForwardRef = /*#__PURE__*/ (0, $dty0i$forwardRef)($73de90c82d6561f0$var$SvgComponent);
var $73de90c82d6561f0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($73de90c82d6561f0$var$ForwardRef);


export {$73de90c82d6561f0$export$2e2bcd8739ae039 as default};
