// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_mention_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $7fMSe$reactjsxruntime = require("react/jsx-runtime");
var $7fMSe$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $fd42787555ce0435$export$2e2bcd8739ae039);




const $fd42787555ce0435$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bbd80a__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bbd80a__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bbd80a__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsxs)("radialGradient", {
                        id: "bbd80a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("clipPath", {
                        id: "bbd80a__b",
                        children: /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("clipPath", {
                id: "bbd80a__a",
                children: /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M63.1 23.29c16.545-5.425 34.284-3.612 45.82 1.593 23.659 10.674 31.47 33.612 28.012 53.264-3.429 19.484-18.327 37.023-40.869 35.464-20.758-1.436-31.9-9.307-36.412-19.61-4.341-9.914-1.93-20.8 1.58-27.28 10.008-18.484 27.797-19.679 36.932-18.844 4.498.411 7.267 4.463 6.897 8.48l-4.533 49.295c15.125-.906 25.791-13.35 28.526-28.892 2.936-16.683-3.649-35.664-23.423-44.585-9.715-4.384-25.39-6.085-40.037-1.282-14.408 4.724-27.845 15.736-33.65 37.04-4.597 16.876.774 33.035 11.113 45.01 10.4 12.046 25.48 19.448 39.591 19.041 2.208-.064 4.05 1.675 4.114 3.883.063 2.208-1.675 4.05-3.883 4.114-16.923.488-34.192-8.275-45.877-21.81-11.747-13.606-18.197-32.444-12.776-52.34 6.56-24.077 22.092-37.036 38.875-42.54m29.43 81.953 4.546-49.43c-7.944-.654-21.196.653-28.81 14.717-2.707 4.998-4.363 13.237-1.286 20.263 2.688 6.14 9.53 12.492 25.55 14.45"
                })
            }),
            /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("g", {
                clipPath: "url(#bbd80a__a)",
                children: /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#bbd80a__b)",
                    children: [
                        /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#bbd80a__c)"
                        }),
                        /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("path", {
                            fill: "url(#bbd80a__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("path", {
                            fill: "url(#bbd80a__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $7fMSe$reactjsxruntime.jsx)("path", {
                            fill: "url(#bbd80a__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $fd42787555ce0435$var$ForwardRef = /*#__PURE__*/ (0, $7fMSe$react.forwardRef)($fd42787555ce0435$var$SvgComponent);
var $fd42787555ce0435$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($fd42787555ce0435$var$ForwardRef);
