// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_replace_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $iK7Ki$jsxs, jsx as $iK7Ki$jsx} from "react/jsx-runtime";
import {forwardRef as $iK7Ki$forwardRef} from "react";





const $83bd93d92dae85d1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iK7Ki$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iK7Ki$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iK7Ki$jsxs)("linearGradient", {
                        id: "8d6839__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iK7Ki$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $iK7Ki$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iK7Ki$jsxs)("linearGradient", {
                        id: "8d6839__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iK7Ki$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $iK7Ki$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iK7Ki$jsxs)("linearGradient", {
                        id: "8d6839__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iK7Ki$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iK7Ki$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $iK7Ki$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $iK7Ki$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iK7Ki$jsxs)("radialGradient", {
                        id: "8d6839__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iK7Ki$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iK7Ki$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iK7Ki$jsx)("clipPath", {
                        id: "8d6839__b",
                        children: /*#__PURE__*/ (0, $iK7Ki$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iK7Ki$jsx)("clipPath", {
                id: "8d6839__a",
                children: /*#__PURE__*/ (0, $iK7Ki$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M24.23 16.25c-4.418 0-8 3.582-8 8v10.622c0 4.418 3.582 8 8 8h10.622c4.419 0 8-3.582 8-8V24.25c0-4.418-3.581-8-8-8zm30.647 1.11c-1.104-.037-2.03.828-2.067 1.932s.827 2.029 1.931 2.066c2.85.097 5.801.919 8.036 3.08 2.03 1.965 3.718 5.278 3.976 10.878l-4.111-4.111c-.781-.781-2.048-.781-2.829 0-.78.78-.78 2.047 0 2.828l7.145 7.145c.781.78 2.047.78 2.829 0l7.167-7.168c.781-.781.781-2.047 0-2.828-.78-.781-2.047-.781-2.828 0l-3.407 3.406c-.378-6.05-2.3-10.257-5.161-13.024-3.176-3.072-7.214-4.086-10.681-4.204M28.953 54.82c-.781-.78-2.048-.78-2.829 0l-7.167 7.168c-.782.781-.782 2.048 0 2.829.78.78 2.047.78 2.828 0l3.406-3.407c.379 6.051 2.302 10.257 5.162 13.024 3.175 3.072 7.213 4.086 10.68 4.204 1.105.037 2.03-.828 2.067-1.932.038-1.103-.827-2.029-1.93-2.066-2.85-.097-5.802-.919-8.036-3.08-2.03-1.965-3.718-5.278-3.977-10.878l4.112 4.112c.781.78 2.047.78 2.828 0 .781-.781.781-2.048 0-2.829zM61.15 53.13c-4.418 0-8 3.582-8 8V71.75c0 4.418 3.582 8 8 8h10.622c4.418 0 8-3.582 8-8V61.129c0-4.418-3.582-8-8-8z"
                })
            }),
            /*#__PURE__*/ (0, $iK7Ki$jsx)("g", {
                clipPath: "url(#8d6839__a)",
                children: /*#__PURE__*/ (0, $iK7Ki$jsxs)("g", {
                    clipPath: "url(#8d6839__b)",
                    children: [
                        /*#__PURE__*/ (0, $iK7Ki$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $iK7Ki$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#8d6839__c)"
                        }),
                        /*#__PURE__*/ (0, $iK7Ki$jsx)("path", {
                            fill: "url(#8d6839__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $iK7Ki$jsx)("path", {
                            fill: "url(#8d6839__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $iK7Ki$jsx)("path", {
                            fill: "url(#8d6839__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $83bd93d92dae85d1$var$ForwardRef = /*#__PURE__*/ (0, $iK7Ki$forwardRef)($83bd93d92dae85d1$var$SvgComponent);
var $83bd93d92dae85d1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($83bd93d92dae85d1$var$ForwardRef);


export {$83bd93d92dae85d1$export$2e2bcd8739ae039 as default};
