// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fileImage_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $bW3qM$reactjsxruntime = require("react/jsx-runtime");
var $bW3qM$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $fecea141be81428c$export$2e2bcd8739ae039);




const $fecea141be81428c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a7b887__d",
                        x1: 67.303,
                        x2: 19.196,
                        y1: 3.793,
                        y2: 66.165,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a7b887__e",
                        x1: 37.993,
                        x2: 47.273,
                        y1: -24.921,
                        y2: 34.473,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a7b887__f",
                        x1: -69.086,
                        x2: 87.061,
                        y1: 94.487,
                        y2: 63.627,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a7b887__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 119.937 -119.938 0 81.246 61.076)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("clipPath", {
                        id: "a7b887__b",
                        children: /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 7.199,
                            y: 6.465,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("clipPath", {
                id: "a7b887__a",
                children: /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m52.51 18.91 19.783 19.782c1.35 1.35 2.109 3.182 2.109 5.091v28.218c0 3.976-3.224 7.2-7.2 7.2h-38.4c-3.977 0-7.2-3.224-7.2-7.2V24c0-3.977 3.223-7.2 7.2-7.2h18.617c1.91 0 3.741.758 5.091 2.109m4.191 21.89c-2.485 0-4.5 2.016-4.5 4.5 0 2.486 2.015 4.5 4.5 4.5 2.486 0 4.5-2.014 4.5-4.5 0-2.484-2.014-4.5-4.5-4.5M32.404 69.6h31.194c.939 0 1.514-1.028 1.022-1.828l-6.712-10.91c-.468-.762-1.575-.762-2.044 0l-5.29 8.598-7.854-13.32c-.465-.783-1.6-.783-2.064 0l-9.284 15.648c-.474.8.102 1.813 1.032 1.813"
                })
            }),
            /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("g", {
                clipPath: "url(#a7b887__a)",
                children: /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#a7b887__b)",
                    children: [
                        /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("rect", {
                            width: 83.07,
                            height: 83.07,
                            x: 7.199,
                            y: 6.465,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("ellipse", {
                            cx: 81.246,
                            cy: 61.076,
                            fill: "url(#a7b887__c)",
                            rx: 119.938,
                            ry: 119.937
                        }),
                        /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("path", {
                            fill: "url(#a7b887__d)",
                            d: "M-.278 32.482C-1.61 3.77 20.584-20.588 49.297-21.92c28.712-1.334 53.069 20.862 54.402 49.574 1.333 28.713-20.862 53.07-49.574 54.403C25.412 83.39 1.055 61.195-.278 32.482"
                        }),
                        /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("path", {
                            fill: "url(#a7b887__e)",
                            d: "M28.183-51.017c-6.477-4.008-14.684-4.032-21.185-.063L-58.026-11.38c-6.368 3.889-6.379 13.067-.02 16.97L6.55 45.236c6.492 3.984 14.698 3.979 21.184-.014L92.194 5.539c6.326-3.894 6.338-13.02.02-16.93z"
                        }),
                        /*#__PURE__*/ (0, $bW3qM$reactjsxruntime.jsx)("path", {
                            fill: "url(#a7b887__f)",
                            d: "M-34.212 46.197c13.403 0 25.543 5.424 34.332 14.175 17.58 17.531 32.154 17.546 49.733.045 8.79-8.796 20.945-14.235 34.363-14.235 26.819.015 48.546 21.682 48.546 48.429 0 26.746-21.727 48.399-48.546 48.399-13.448 0-25.603-5.455-34.393-14.25C32.26 111.288 17.7 111.303.12 128.835c-8.79 8.765-20.93 14.175-34.332 14.175-26.805 0-48.53-21.667-48.53-48.4 0-26.731 21.725-48.413 48.53-48.413"
                        })
                    ]
                })
            })
        ]
    });
const $fecea141be81428c$var$ForwardRef = /*#__PURE__*/ (0, $bW3qM$react.forwardRef)($fecea141be81428c$var$SvgComponent);
var $fecea141be81428c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($fecea141be81428c$var$ForwardRef);
