// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_dropToUpload_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $h5hQC$reactjsxruntime = require("react/jsx-runtime");
var $h5hQC$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $336a592a9ed1b0c6$export$2e2bcd8739ae039);




const $336a592a9ed1b0c6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b0a054__d",
                        x1: 110.361,
                        x2: 25.23,
                        y1: -0.728,
                        y2: 109.647,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b0a054__e",
                        x1: 58.496,
                        x2: 74.918,
                        y1: -51.54,
                        y2: 53.563,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsxs)("linearGradient", {
                        id: "b0a054__f",
                        x1: -130.994,
                        x2: 145.323,
                        y1: 159.767,
                        y2: 105.157,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsxs)("radialGradient", {
                        id: "b0a054__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 17.197 117.837)scale(212.241)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("clipPath", {
                        id: "b0a054__b",
                        children: /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("rect", {
                            width: 147,
                            height: 147,
                            x: 4,
                            y: 4,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("clipPath", {
                id: "b0a054__a",
                children: /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M51.739 21.932h27.404c2.81 0 5.506 1.116 7.494 3.104l29.118 29.118c1.988 1.988 3.104 4.683 3.104 7.494v41.535c0 5.853-4.745 10.598-10.598 10.598H51.739c-5.853 0-10.598-4.745-10.598-10.598V32.53c0-5.853 4.745-10.598 10.598-10.598m31 30.625c.004-1.951-1.574-3.536-3.525-3.54s-3.536 1.573-3.54 3.524l-.079 34.983-11.943-9.5c-1.527-1.214-3.75-.96-4.964.567s-.96 3.749.566 4.963l17.793 14.152c1.294 1.03 3.13 1.023 4.418-.016L98.999 83.54c1.518-1.226 1.755-3.45.53-4.968s-3.45-1.756-4.968-.53l-11.9 9.605zM22.4 75.522c0-1.767 1.433-3.2 3.2-3.2h5.881c1.767 0 3.2-1.432 3.2-3.2 0-1.767-1.433-3.2-3.2-3.2H25.6c-5.302 0-9.6 4.298-9.6 9.6v5.535c0 1.768 1.433 3.2 3.2 3.2s3.2-1.432 3.2-3.2zm106.119-9.6c-1.767 0-3.2 1.433-3.2 3.2s1.433 3.2 3.2 3.2h5.881c1.767 0 3.2 1.433 3.2 3.2v5.535c0 1.768 1.433 3.2 3.2 3.2s3.2-1.432 3.2-3.2v-5.535c0-5.302-4.298-9.6-9.6-9.6zM144 95.587c0-1.767-1.433-3.2-3.2-3.2s-3.2 1.433-3.2 3.2v11.07c0 1.768 1.433 3.2 3.2 3.2s3.2-1.432 3.2-3.2zm-121.6 0c0-1.767-1.433-3.2-3.2-3.2s-3.2 1.433-3.2 3.2v11.07c0 1.768 1.433 3.2 3.2 3.2s3.2-1.432 3.2-3.2zm0 25.6c0-1.767-1.433-3.2-3.2-3.2s-3.2 1.433-3.2 3.2v5.535c0 5.302 4.298 9.6 9.6 9.6h5.881c1.767 0 3.2-1.432 3.2-3.2 0-1.767-1.433-3.2-3.2-3.2H25.6c-1.767 0-3.2-1.432-3.2-3.2zm121.6 0c0-1.767-1.433-3.2-3.2-3.2s-3.2 1.433-3.2 3.2v5.535c0 1.768-1.433 3.2-3.2 3.2h-5.881c-1.767 0-3.2 1.433-3.2 3.2s1.433 3.2 3.2 3.2h5.881c5.302 0 9.6-4.298 9.6-9.6zm-97.081 8.735c-1.767 0-3.2 1.433-3.2 3.2s1.433 3.2 3.2 3.2H58.68c1.767 0 3.2-1.432 3.2-3.2 0-1.767-1.433-3.2-3.2-3.2zm27.2 0c-1.767 0-3.2 1.433-3.2 3.2s1.433 3.2 3.2 3.2H85.88c1.767 0 3.2-1.432 3.2-3.2 0-1.767-1.433-3.2-3.2-3.2zm27.2 0c-1.767 0-3.2 1.433-3.2 3.2s1.433 3.2 3.2 3.2h11.762c1.767 0 3.2-1.432 3.2-3.2 0-1.767-1.433-3.2-3.2-3.2z"
                })
            }),
            /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("g", {
                clipPath: "url(#b0a054__a)",
                children: /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#b0a054__b)",
                    children: [
                        /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("rect", {
                            width: 147,
                            height: 147,
                            x: 4,
                            y: 4,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("circle", {
                            cx: 135.034,
                            cy: 100.64,
                            r: 212.241,
                            fill: "url(#b0a054__c)"
                        }),
                        /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("path", {
                            fill: "url(#b0a054__d)",
                            d: "M-9.23 50.04C-11.59-.768 27.686-43.87 78.495-46.23c50.81-2.36 93.912 36.918 96.271 87.727S137.85 135.41 87.04 137.768-6.87 100.851-9.23 50.041"
                        }),
                        /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("path", {
                            fill: "url(#b0a054__e)",
                            d: "M41.136-97.72c-11.462-7.093-25.985-7.136-37.49-.112l-115.065 70.255c-11.27 6.881-11.289 23.122-.035 30.03L2.852 72.608c11.488 7.049 26.01 7.04 37.488-.026L154.41 2.362c11.195-6.893 11.215-23.043.034-29.961z"
                        }),
                        /*#__PURE__*/ (0, $h5hQC$reactjsxruntime.jsx)("path", {
                            fill: "url(#b0a054__f)",
                            d: "M-69.281 74.312c23.717 0 45.2 9.599 60.754 25.084 31.109 31.024 56.9 31.05 88.008.08 15.554-15.565 37.064-25.19 60.807-25.19 47.461.026 85.907 38.368 85.907 85.699s-38.446 85.647-85.907 85.647c-23.796 0-45.306-9.652-60.86-25.217-31.082-30.917-56.846-30.891-87.955.133-15.554 15.512-37.037 25.084-60.754 25.084-47.434 0-85.88-38.342-85.88-85.647s38.446-85.673 85.88-85.673"
                        })
                    ]
                })
            })
        ]
    });
const $336a592a9ed1b0c6$var$ForwardRef = /*#__PURE__*/ (0, $h5hQC$react.forwardRef)($336a592a9ed1b0c6$var$SvgComponent);
var $336a592a9ed1b0c6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($336a592a9ed1b0c6$var$ForwardRef);
