// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_buildTable_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gGpGZ$reactjsxruntime = require("react/jsx-runtime");
var $gGpGZ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $64823e4d0952ba4e$export$2e2bcd8739ae039);




const $64823e4d0952ba4e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1bdb67__d",
                        x1: 35.386,
                        x2: 13.221,
                        y1: 5.709,
                        y2: 36.193,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1bdb67__e",
                        x1: 20.822,
                        x2: 24.931,
                        y1: -7.742,
                        y2: 20.155,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1bdb67__f",
                        x1: -32.39,
                        x2: 44.84,
                        y1: 48.195,
                        y2: 32.003,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "1bdb67__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 56.1841 -59.6007 0 42.316 32.543)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("clipPath", {
                        id: "1bdb67__b",
                        children: /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("rect", {
                            width: 41.28,
                            height: 38.913,
                            x: 5.52,
                            y: 6.961,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("clipPath", {
                id: "1bdb67__a",
                children: /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M37.19 10.17c2.652 0 4.8 2.148 4.8 4.8v1.227H5.984V14.97c0-2.652 2.148-4.8 4.8-4.8zm4.8 15.768c-.773-.198-1.583-.303-2.417-.303h-.003c-5.382 0-9.76 4.377-9.76 9.759 0 .83.106 1.64.304 2.412H14.4v-19.21h27.59zm-36.006 7.07c0 2.653 2.148 4.8 4.8 4.8H12V18.6H5.984zm33.589-4.97c.846 0 1.659.144 2.418.408h.003c2.877 1.002 4.94 3.735 4.94 6.95 0 4.066-3.293 7.36-7.358 7.36-3.22 0-5.955-2.067-6.954-4.947-.264-.756-.408-1.566-.408-2.412 0-4.065 3.294-7.36 7.359-7.36m1.203 8.544h2.364c.663 0 1.2-.537 1.2-1.2s-.537-1.2-1.2-1.2h-2.364v-2.364c0-.663-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2v2.364h-2.364c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h2.364v2.364c0 .663.537 1.2 1.2 1.2s1.2-.537 1.2-1.2z"
                })
            }),
            /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("g", {
                clipPath: "url(#1bdb67__a)",
                children: /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#1bdb67__b)",
                    children: [
                        /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("rect", {
                            width: 41.28,
                            height: 38.913,
                            x: 5.52,
                            y: 6.961,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("ellipse", {
                            cx: 42.316,
                            cy: 32.543,
                            fill: "url(#1bdb67__c)",
                            rx: 59.601,
                            ry: 56.184
                        }),
                        /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("path", {
                            fill: "url(#1bdb67__d)",
                            d: "M1.803 19.148C1.14 5.698 12.17-5.712 26.438-6.336S52.81 3.436 53.472 16.887c.663 13.45-10.367 24.86-24.635 25.484S2.465 32.598 1.803 19.148"
                        }),
                        /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("path", {
                            fill: "url(#1bdb67__e)",
                            d: "M15.947-19.967c-3.219-1.877-7.297-1.889-10.527-.03L-26.893-1.397c-3.165 1.821-3.17 6.12-.01 7.949l32.1 18.571c3.225 1.867 7.303 1.864 10.526-.006l32.033-18.59c3.144-1.824 3.15-6.099.01-7.93z"
                        }),
                        /*#__PURE__*/ (0, $gGpGZ$reactjsxruntime.jsx)("path", {
                            fill: "url(#1bdb67__f)",
                            d: "M-15.06 25.573c6.66 0 12.693 2.541 17.06 6.64 8.736 8.213 15.979 8.22 24.714.022 4.368-4.12 10.409-6.669 17.076-6.669 13.328.007 24.124 10.157 24.124 22.687S57.118 70.925 43.79 70.925c-6.682 0-12.723-2.555-17.09-6.675-8.729-8.185-15.964-8.178-24.7.035-4.367 4.106-10.4 6.64-17.06 6.64-13.32 0-24.117-10.15-24.117-22.672s10.797-22.68 24.117-22.68"
                        })
                    ]
                })
            })
        ]
    });
const $64823e4d0952ba4e$var$ForwardRef = /*#__PURE__*/ (0, $gGpGZ$react.forwardRef)($64823e4d0952ba4e$var$SvgComponent);
var $64823e4d0952ba4e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($64823e4d0952ba4e$var$ForwardRef);
