// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_3Dasset_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $e221O$reactjsxruntime = require("react/jsx-runtime");
var $e221O$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $71713a5a05c8fe41$export$2e2bcd8739ae039);




const $71713a5a05c8fe41$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e221O$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6ed3df__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6ed3df__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6ed3df__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6ed3df__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("clipPath", {
                        id: "6ed3df__b",
                        children: /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("clipPath", {
                id: "6ed3df__a",
                children: /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M33.116 43.031c4.642-2.543 9.26-5.094 12.605-6.985 3.015-1.704 7.03-1.824 10.058-.139l20.269 11.285c2.42 1.346 5.363 1.346 7.782 0l21.452-11.943c2.419-1.348 5.363-1.348 7.783 0l13.944 7.762c2.753 1.533 4.122 4.286 4.108 7.032L79.939 78.615l-51-28.472c-.019-2.784 1.377-5.578 4.177-7.112m-4.237 16.24v27.261c0 5.42 2.744 10.472 7.29 13.422l39.77 25.807V85.544zm55.06 66.444 39.869-25.764c4.561-2.948 7.315-8.008 7.315-13.438V59.202L83.939 85.544z"
                })
            }),
            /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6ed3df__a)",
                children: /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6ed3df__b)",
                    children: [
                        /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#6ed3df__c)"
                        }),
                        /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("path", {
                            fill: "url(#6ed3df__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("path", {
                            fill: "url(#6ed3df__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $e221O$reactjsxruntime.jsx)("path", {
                            fill: "url(#6ed3df__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $71713a5a05c8fe41$var$ForwardRef = /*#__PURE__*/ (0, $e221O$react.forwardRef)($71713a5a05c8fe41$var$SvgComponent);
var $71713a5a05c8fe41$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($71713a5a05c8fe41$var$ForwardRef);
