// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_filters_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $ifh4u$jsxs, jsx as $ifh4u$jsx} from "react/jsx-runtime";
import {forwardRef as $ifh4u$forwardRef} from "react";





const $f22fbf251ce027f3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ifh4u$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $ifh4u$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $ifh4u$jsxs)("linearGradient", {
                        id: "5302ff__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ifh4u$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $ifh4u$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ifh4u$jsxs)("linearGradient", {
                        id: "5302ff__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ifh4u$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $ifh4u$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ifh4u$jsxs)("linearGradient", {
                        id: "5302ff__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ifh4u$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $ifh4u$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $ifh4u$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $ifh4u$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ifh4u$jsxs)("radialGradient", {
                        id: "5302ff__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ifh4u$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ifh4u$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ifh4u$jsx)("clipPath", {
                        id: "5302ff__b",
                        children: /*#__PURE__*/ (0, $ifh4u$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ifh4u$jsx)("clipPath", {
                id: "5302ff__a",
                children: /*#__PURE__*/ (0, $ifh4u$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M12 57.9c0-6.975 3.092-13.228 7.98-17.464.768 8.999 5.933 16.743 13.338 21.074.965 7.252 4.799 13.598 10.323 17.86C40.999 80.422 38.117 81 35.1 81 22.342 81 12 70.658 12 57.9m39.639 21.47c2.642 1.052 5.524 1.63 8.54 1.63 12.759 0 23.1-10.342 23.1-23.1 0-7.554-3.625-14.26-9.23-18.475-.426 8.833-5.08 16.557-11.983 21.191-.76 7.624-4.677 14.318-10.427 18.754M69.37 32.398C66.832 22.399 57.771 15 46.98 15c-10.996 0-20.197 7.682-22.529 17.972C27.72 31.574 31.32 30.8 35.1 30.8c4.525 0 8.79 1.109 12.54 3.07 3.75-1.961 8.015-3.07 12.54-3.07 3.226 0 6.321.564 9.191 1.598m-45.488 5.303-.003.399c0 7.554 3.626 14.26 9.232 18.475.394-8.198 4.432-15.44 10.529-20.144-2.642-1.052-5.524-1.63-8.541-1.63-4.07 0-7.895 1.052-11.217 2.9M47.64 77.303c4.978-3.224 8.637-8.311 9.989-14.275C54.36 64.426 50.76 65.2 46.98 65.2c-3.226 0-6.321-.564-9.191-1.598 1.45 5.716 5.033 10.581 9.851 13.701M70.08 38.1c0 6.975-3.092 13.228-7.98 17.464-.664-7.781-4.616-14.624-10.461-19.133 2.642-1.052 5.524-1.63 8.54-1.63 3.534 0 6.882.792 9.876 2.21q.025.541.025 1.089M58.2 57.9c0-8.135-4.205-15.288-10.56-19.403-6.355 4.115-10.56 11.268-10.56 19.403q0 .548.025 1.089C40.1 60.407 43.447 61.2 46.98 61.2c4.07 0 7.895-1.053 11.217-2.901z"
                })
            }),
            /*#__PURE__*/ (0, $ifh4u$jsx)("g", {
                clipPath: "url(#5302ff__a)",
                children: /*#__PURE__*/ (0, $ifh4u$jsxs)("g", {
                    clipPath: "url(#5302ff__b)",
                    children: [
                        /*#__PURE__*/ (0, $ifh4u$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $ifh4u$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#5302ff__c)"
                        }),
                        /*#__PURE__*/ (0, $ifh4u$jsx)("path", {
                            fill: "url(#5302ff__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $ifh4u$jsx)("path", {
                            fill: "url(#5302ff__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $ifh4u$jsx)("path", {
                            fill: "url(#5302ff__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $f22fbf251ce027f3$var$ForwardRef = /*#__PURE__*/ (0, $ifh4u$forwardRef)($f22fbf251ce027f3$var$SvgComponent);
var $f22fbf251ce027f3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f22fbf251ce027f3$var$ForwardRef);


export {$f22fbf251ce027f3$export$2e2bcd8739ae039 as default};
