// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fireworks_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3fszR$reactjsxruntime = require("react/jsx-runtime");
var $3fszR$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $eaa0bbf6ef17de51$export$2e2bcd8739ae039);




const $eaa0bbf6ef17de51$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ac07df__d",
                        x1: 32.334,
                        x2: 11.233,
                        y1: 5.007,
                        y2: 34.029,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ac07df__e",
                        x1: 18.466,
                        x2: 22.378,
                        y1: -7.8,
                        y2: 18.758,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ac07df__f",
                        x1: -32.193,
                        x2: 41.333,
                        y1: 45.453,
                        y2: 30.038,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ac07df__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.4895 -56.7422 0 38.93 30.552)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("clipPath", {
                        id: "ac07df__b",
                        children: /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.197,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("clipPath", {
                id: "ac07df__a",
                children: /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M38.432 8.473c.01-.663-.518-1.208-1.18-1.219-.663-.01-1.21.518-1.22 1.18l-.016 1.05-1.052-.016c-.662-.01-1.208.519-1.218 1.181-.01.663.519 1.209 1.181 1.219l1.05.016-.017 1.05c-.01.663.518 1.21 1.181 1.22s1.208-.518 1.219-1.18l.017-1.053 1.05.016c.663.01 1.209-.519 1.219-1.182s-.519-1.208-1.182-1.218l-1.049-.016zm-18.335.764c0-.663-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2v3.042c0 .662.537 1.2 1.2 1.2s1.2-.537 1.2-1.2zm-7.51.847c.537-.39 1.287-.27 1.677.266l1.788 2.463c.39.536.27 1.287-.266 1.676-.537.39-1.287.27-1.676-.266l-1.788-2.463c-.39-.537-.27-1.287.266-1.676m-2.389 4.6c-.63-.205-1.307.14-1.512.77-.204.63.14 1.308.771 1.512l2.892.94c.63.204 1.307-.141 1.512-.772.205-.63-.14-1.307-.77-1.511zm3.664 5.724c.204.63-.141 1.307-.771 1.511l-2.895.94c-.63.204-1.308-.141-1.512-.772-.205-.63.14-1.307.771-1.511l2.895-.94c.63-.204 1.307.141 1.512.772m2.186 4.324c.39-.536.27-1.286-.265-1.676-.537-.39-1.287-.27-1.677.265l-1.788 2.46c-.39.537-.27 1.287.266 1.677s1.286.27 1.676-.266zm4.045.534c0-.663-.537-1.2-1.2-1.2s-1.2.537-1.2 1.2v3.042c0 .663.538 1.2 1.2 1.2.663 0 1.2-.537 1.2-1.2zm5.116-15.179c.536.39.655 1.14.265 1.676l-1.788 2.46c-.39.536-1.14.655-1.676.265s-.655-1.14-.266-1.676l1.788-2.46c.39-.536 1.14-.654 1.677-.265m3.127 6.882c.63-.204.976-.881.771-1.512-.205-.63-.882-.975-1.512-.77l-2.892.939c-.63.204-.975.881-.77 1.511s.88.976 1.511.771zm-4.404 3.441c.205-.63.882-.975 1.512-.77l2.892.938c.63.205.976.882.771 1.512s-.882.976-1.512.771l-2.892-.939c-.63-.204-.975-.881-.77-1.512m-.248 2.912c-.389-.536-1.14-.655-1.676-.266-.536.39-.655 1.14-.266 1.676l1.788 2.463c.39.536 1.14.655 1.676.266.537-.39.656-1.14.266-1.676zm9.027 11.166c.663.003 1.198.542 1.195 1.205l-.006 1.485c-.002.663-.542 1.198-1.205 1.195-.662-.003-1.198-.542-1.195-1.205l.006-1.485c.003-.663.542-1.198 1.205-1.195m-2.682.354c.575-.329.775-1.062.446-1.637-.33-.575-1.062-.775-1.638-.446l-1.29.738c-.575.33-.774 1.063-.445 1.638s1.062.775 1.637.445zm-2.921-6.172c.333-.573 1.068-.767 1.64-.433l1.285.747c.572.333.767 1.067.433 1.64-.333.573-1.067.767-1.64.434l-1.284-.747c-.573-.333-.767-1.068-.434-1.64m6.798-2.022c.003-.663-.532-1.203-1.195-1.205s-1.202.532-1.205 1.195l-.006 1.485c-.003.663.533 1.202 1.195 1.205.663.003 1.203-.533 1.205-1.195zm4.4 2.033c.33.575.13 1.308-.445 1.637l-1.29.738c-.576.33-1.309.13-1.638-.445s-.13-1.309.446-1.638l1.29-.738c.575-.329 1.308-.13 1.637.446m-1.724 4.077c-.573-.334-1.307-.14-1.64.433-.334.573-.14 1.308.433 1.641l1.284.747c.573.333 1.308.14 1.641-.434.333-.573.139-1.307-.434-1.64zM7.043 37.346c.01-.662.556-1.191 1.218-1.181l1.05.016.015-1.051c.01-.663.556-1.192 1.219-1.182.662.01 1.191.556 1.181 1.218l-.016 1.052 1.051.016c.663.01 1.192.556 1.182 1.218-.01.663-.556 1.192-1.218 1.182l-1.052-.016-.016 1.048c-.01.663-.556 1.192-1.218 1.182-.663-.01-1.192-.556-1.182-1.218l.016-1.05-1.048-.015c-.663-.01-1.192-.556-1.182-1.218"
                })
            }),
            /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ac07df__a)",
                children: /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ac07df__b)",
                    children: [
                        /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 3.898,
                            y: 6.197,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("ellipse", {
                            cx: 38.93,
                            cy: 30.552,
                            fill: "url(#ac07df__c)",
                            rx: 56.742,
                            ry: 53.489
                        }),
                        /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("path", {
                            fill: "url(#ac07df__d)",
                            d: "M.362 17.801C-.268 4.996 10.232-5.866 23.815-6.46s25.107 9.304 25.738 22.11c.631 12.804-9.87 23.667-23.453 24.262S.993 30.607.362 17.8"
                        }),
                        /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("path", {
                            fill: "url(#ac07df__e)",
                            d: "M13.825-19.439c-3.064-1.787-6.947-1.798-10.022-.028L-26.96-1.761c-3.013 1.734-3.018 5.827-.01 7.568l30.56 17.68c3.071 1.777 6.954 1.775 10.022-.005L44.11 5.784c2.993-1.737 2.998-5.807.01-7.55z"
                        }),
                        /*#__PURE__*/ (0, $3fszR$reactjsxruntime.jsx)("path", {
                            fill: "url(#ac07df__f)",
                            d: "M-15.694 23.917c6.34 0 12.084 2.419 16.242 6.322 8.317 7.818 15.212 7.825 23.529.02 4.158-3.923 9.909-6.349 16.257-6.349 12.688.007 22.967 9.67 22.967 21.598 0 11.929-10.279 21.585-22.967 21.585-6.362 0-12.113-2.432-16.271-6.355-8.31-7.792-15.198-7.785-23.515.034-4.158 3.909-9.902 6.321-16.242 6.321-12.681 0-22.96-9.663-22.96-21.585 0-11.921 10.279-21.591 22.96-21.591"
                        })
                    ]
                })
            })
        ]
    });
const $eaa0bbf6ef17de51$var$ForwardRef = /*#__PURE__*/ (0, $3fszR$react.forwardRef)($eaa0bbf6ef17de51$var$SvgComponent);
var $eaa0bbf6ef17de51$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($eaa0bbf6ef17de51$var$ForwardRef);
