// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_artboard_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $1oDDV$jsxs, jsx as $1oDDV$jsx} from "react/jsx-runtime";
import {forwardRef as $1oDDV$forwardRef} from "react";





const $f984f09c186acd0a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1oDDV$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1oDDV$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1oDDV$jsxs)("linearGradient", {
                        id: "5b8d16__d",
                        x1: 104.009,
                        x2: 29.051,
                        y1: 5.767,
                        y2: 108.862,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1oDDV$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $1oDDV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1oDDV$jsxs)("linearGradient", {
                        id: "5b8d16__e",
                        x1: 54.754,
                        x2: 68.65,
                        y1: -39.722,
                        y2: 54.621,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1oDDV$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $1oDDV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1oDDV$jsxs)("linearGradient", {
                        id: "5b8d16__f",
                        x1: -125.203,
                        x2: 135.978,
                        y1: 149.448,
                        y2: 94.691,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1oDDV$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1oDDV$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $1oDDV$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $1oDDV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1oDDV$jsxs)("radialGradient", {
                        id: "5b8d16__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 190.006 -201.561 0 127.44 96.516)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1oDDV$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1oDDV$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1oDDV$jsx)("clipPath", {
                        id: "5b8d16__b",
                        children: /*#__PURE__*/ (0, $1oDDV$jsx)("rect", {
                            width: 139.603,
                            height: 131.6,
                            x: 3,
                            y: 10,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1oDDV$jsx)("clipPath", {
                id: "5b8d16__a",
                children: /*#__PURE__*/ (0, $1oDDV$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M40 24V14c0-2.21-1.79-4-4-4s-4 1.79-4 4v10c0 2.21 1.79 4 4 4s4-1.79 4-4m39 46.02c0 5.5 4.5 9.98 10.03 9.98h34.98v48c0 6.63-5.34 12-11.93 12H47.93C41.34 140 36 134.63 36 128V48c0-6.63 5.34-12 11.93-12H79zM121.3 72H89.04c-1.12 0-2.03-.89-2.03-1.98V38.89q.075.066.155.132c.116.098.235.198.345.308l32.81 31.59c.35.34.67.7.98 1.08M14 32h10c2.21 0 4 1.79 4 4s-1.79 4-4 4H14c-2.21 0-4-1.79-4-4s1.79-4 4-4"
                })
            }),
            /*#__PURE__*/ (0, $1oDDV$jsx)("g", {
                clipPath: "url(#5b8d16__a)",
                children: /*#__PURE__*/ (0, $1oDDV$jsxs)("g", {
                    clipPath: "url(#5b8d16__b)",
                    children: [
                        /*#__PURE__*/ (0, $1oDDV$jsx)("rect", {
                            width: 139.603,
                            height: 131.6,
                            x: 3,
                            y: 10,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $1oDDV$jsx)("ellipse", {
                            cx: 127.44,
                            cy: 96.516,
                            fill: "url(#5b8d16__c)",
                            rx: 201.561,
                            ry: 190.006
                        }),
                        /*#__PURE__*/ (0, $1oDDV$jsx)("path", {
                            fill: "url(#5b8d16__d)",
                            d: "M-9.565 51.218c-2.24-45.487 35.06-84.074 83.312-86.186C122-37.08 162.933-1.918 165.174 43.57c2.24 45.486-35.06 84.073-83.313 86.185S-7.325 96.704-9.565 51.218"
                        }),
                        /*#__PURE__*/ (0, $1oDDV$jsx)("path", {
                            fill: "url(#5b8d16__e)",
                            d: "M38.267-81.064c-10.885-6.35-24.676-6.389-35.602-.1L-106.611-18.27c-10.702 6.16-10.72 20.7-.032 26.883L1.911 71.421c10.909 6.31 24.7 6.303 35.6-.023L145.842 8.533c10.631-6.17 10.65-20.629.032-26.822z"
                        }),
                        /*#__PURE__*/ (0, $1oDDV$jsx)("path", {
                            fill: "url(#5b8d16__f)",
                            d: "M-66.596 72.946c22.524 0 42.926 8.593 57.697 22.456 29.543 27.774 54.036 27.797 83.58.071 14.77-13.934 35.198-22.551 57.747-22.551 45.072.024 81.584 34.349 81.584 76.722s-36.512 76.674-81.584 76.674c-22.599 0-43.027-8.641-57.798-22.575-29.518-27.679-53.986-27.655-83.529.119-14.771 13.887-35.173 22.456-57.697 22.456-45.046 0-81.558-34.325-81.558-76.674s36.512-76.698 81.558-76.698"
                        })
                    ]
                })
            })
        ]
    });
const $f984f09c186acd0a$var$ForwardRef = /*#__PURE__*/ (0, $1oDDV$forwardRef)($f984f09c186acd0a$var$SvgComponent);
var $f984f09c186acd0a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f984f09c186acd0a$var$ForwardRef);


export {$f984f09c186acd0a$export$2e2bcd8739ae039 as default};
