// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_tablet_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $832jD$reactjsxruntime = require("react/jsx-runtime");
var $832jD$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $6d6f2e1e2f625302$export$2e2bcd8739ae039);




const $6d6f2e1e2f625302$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $832jD$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ea35b5__d",
                        x1: 33.389,
                        x2: 10.838,
                        y1: 3.65,
                        y2: 34.667,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ea35b5__e",
                        x1: 18.571,
                        x2: 22.751,
                        y1: -10.035,
                        y2: 18.348,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ea35b5__f",
                        x1: -35.57,
                        x2: 43.008,
                        y1: 46.877,
                        y2: 30.403,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ea35b5__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 57.1642 -60.6404 0 40.437 30.952)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("clipPath", {
                        id: "ea35b5__b",
                        children: /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("rect", {
                            width: 42,
                            height: 39.592,
                            x: 3,
                            y: 4.924,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("clipPath", {
                id: "ea35b5__a",
                children: /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M38.4 8.4H9.6c-2.651 0-4.8 2.148-4.8 4.8v21.6c0 2.652 2.149 4.8 4.8 4.8h28.8c2.653 0 4.8-2.148 4.8-4.8V13.2c0-2.652-2.147-4.8-4.8-4.8M10.2 25.2c-.662 0-1.2-.537-1.2-1.2s.535-1.2 1.198-1.2h.003c.663 0 1.2.537 1.2 1.2s-.537 1.2-1.2 1.2"
                })
            }),
            /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ea35b5__a)",
                children: /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ea35b5__b)",
                    children: [
                        /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("rect", {
                            width: 42,
                            height: 39.592,
                            x: 3,
                            y: 4.924,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("ellipse", {
                            cx: 40.437,
                            cy: 30.952,
                            fill: "url(#ea35b5__c)",
                            rx: 60.64,
                            ry: 57.164
                        }),
                        /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("path", {
                            fill: "url(#ea35b5__d)",
                            d: "M-.78 17.324C-1.454 3.64 9.768-7.969 24.285-8.605s26.832 9.943 27.506 23.628-10.548 25.294-25.065 25.93S-.106 31.008-.78 17.323"
                        }),
                        /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("path", {
                            fill: "url(#ea35b5__e)",
                            d: "M13.61-22.473c-3.274-1.91-7.423-1.922-10.71-.03L-29.976-3.581c-3.22 1.853-3.226 6.228-.01 8.088L2.673 23.403c3.282 1.898 7.431 1.896 10.71-.007L45.975 4.483c3.198-1.857 3.204-6.207.01-8.07z"
                        }),
                        /*#__PURE__*/ (0, $832jD$reactjsxruntime.jsx)("path", {
                            fill: "url(#ea35b5__f)",
                            d: "M-17.937 23.86c6.776 0 12.914 2.586 17.358 6.757 8.888 8.356 16.257 8.363 25.145.021 4.444-4.192 10.59-6.785 17.374-6.785 13.56.008 24.544 10.335 24.544 23.083S55.5 70.003 41.94 70.003c-6.8 0-12.945-2.6-17.39-6.791-8.88-8.328-16.24-8.32-25.129.035-4.444 4.178-10.582 6.756-17.358 6.756-13.553 0-24.538-10.327-24.538-23.067S-31.49 23.86-17.937 23.86"
                        })
                    ]
                })
            })
        ]
    });
const $6d6f2e1e2f625302$var$ForwardRef = /*#__PURE__*/ (0, $832jD$react.forwardRef)($6d6f2e1e2f625302$var$SvgComponent);
var $6d6f2e1e2f625302$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($6d6f2e1e2f625302$var$ForwardRef);
