// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_location_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $a52Ic$reactjsxruntime = require("react/jsx-runtime");
var $a52Ic$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $e667a7a2a8e474fb$export$2e2bcd8739ae039);




const $e667a7a2a8e474fb$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2325ac__d",
                        x1: 32.783,
                        x2: 12.551,
                        y1: 5.819,
                        y2: 33.645,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2325ac__e",
                        x1: 19.489,
                        x2: 23.239,
                        y1: -6.459,
                        y2: 19.005,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2325ac__f",
                        x1: -29.084,
                        x2: 41.411,
                        y1: 44.599,
                        y2: 29.819,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2325ac__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.2844 -54.4031 0 39.106 30.312)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("clipPath", {
                        id: "2325ac__b",
                        children: /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.961,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("clipPath", {
                id: "2325ac__a",
                children: /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M10.885 18.956c.709-6.345 5.935-11.337 12.277-11.73 7.67-.477 14.039 5.632 14.039 13.237 0 1.943-.42 3.787-1.167 5.45-2.227 5.83-7.313 11.346-10.117 14.09-1.082 1.058-2.801 1.063-3.883.005-2.997-2.933-8.612-9.041-10.543-15.314-.595-1.775-.832-3.715-.606-5.738m13.116 5.943c2.982 0 5.4-2.417 5.4-5.4 0-2.982-2.418-5.4-5.4-5.4s-5.4 2.418-5.4 5.4 2.417 5.4 5.4 5.4"
                })
            }),
            /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("g", {
                clipPath: "url(#2325ac__a)",
                children: /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#2325ac__b)",
                    children: [
                        /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.961,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("ellipse", {
                            cx: 39.106,
                            cy: 30.312,
                            fill: "url(#2325ac__c)",
                            rx: 54.403,
                            ry: 51.284
                        }),
                        /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("path", {
                            fill: "url(#2325ac__d)",
                            d: "M2.128 18.087C1.523 5.809 11.591-4.607 24.615-5.177s24.072 8.92 24.677 21.198c.605 12.277-9.463 22.692-22.487 23.262S2.733 30.364 2.128 18.087"
                        }),
                        /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("path", {
                            fill: "url(#2325ac__e)",
                            d: "M15.039-17.617c-2.938-1.714-6.66-1.725-9.61-.027L-24.064-.669c-2.889 1.662-2.893 5.587-.009 7.256l29.3 16.952c2.944 1.703 6.667 1.7 9.609-.006l29.24-16.968c2.869-1.666 2.874-5.568.008-7.24z"
                        }),
                        /*#__PURE__*/ (0, $a52Ic$reactjsxruntime.jsx)("path", {
                            fill: "url(#2325ac__f)",
                            d: "M-13.266 23.95c6.08 0 11.586 2.32 15.573 6.06 7.974 7.497 14.585 7.504 22.559.02 3.987-3.76 9.5-6.087 15.586-6.087 12.166.007 22.02 9.272 22.02 20.708 0 11.437-9.854 20.695-22.02 20.695-6.1 0-11.613-2.332-15.6-6.093-7.967-7.47-14.57-7.464-22.545.032-3.987 3.748-9.494 6.061-15.573 6.061-12.158 0-22.013-9.264-22.013-20.695 0-11.43 9.855-20.701 22.013-20.701"
                        })
                    ]
                })
            })
        ]
    });
const $e667a7a2a8e474fb$var$ForwardRef = /*#__PURE__*/ (0, $a52Ic$react.forwardRef)($e667a7a2a8e474fb$var$SvgComponent);
var $e667a7a2a8e474fb$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($e667a7a2a8e474fb$var$ForwardRef);
