// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_illustration_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8qIAO$reactjsxruntime = require("react/jsx-runtime");
var $8qIAO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $69dbc03f3ea24da2$export$2e2bcd8739ae039);




const $69dbc03f3ea24da2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 97,
        height: 96,
        viewBox: "0 0 97 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f090a6__d",
                        x1: 70.403,
                        x2: 14.807,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f090a6__e",
                        x1: 36.531,
                        x2: 47.256,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f090a6__f",
                        x1: -87.219,
                        x2: 93.232,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f090a6__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.7 74.813)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("clipPath", {
                        id: "f090a6__b",
                        children: /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("clipPath", {
                id: "f090a6__a",
                children: /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M39.561 25.134 52.798 38.37l-4.124 7.95c-.879 1.692-2.332 3.015-4.1 3.73l-21.483 8.697c-.468.19-.935.254-1.379.217l13.425-13.392q.027-.026.051-.053c.869.091 1.77-.196 2.435-.862 1.172-1.172 1.172-3.071 0-4.243-1.171-1.171-3.071-1.171-4.243 0-.595.596-.888 1.379-.878 2.159q-.1.076-.19.167L18.958 56.063c-.01-.396.06-.809.226-1.222l8.697-21.483c.715-1.768 2.038-3.221 3.73-4.1zm6.58-5.714-3.319 3.318 12.361 12.36 3.318-3.318c1.563-1.562 1.563-4.094 0-5.657l-6.703-6.703c-1.562-1.562-4.095-1.562-5.657 0m-4.48 49.013c-2.022 4.4-2.719 5.919-6.434 8.811 1.435.165 3.358.357 5.508.448 6.869.29 16.052-.45 19.018-6.38.836-1.67 1.189-3.167 1.203-4.495.013-1.15-.227-2.173-.625-3.077-.617-1.402-1.617-2.516-2.647-3.36-.788-.646-1.91-1.157-3.2-1.43l.001.001c-3.583-.76-8.448.321-10.95 5.483-.758 1.567-1.36 2.877-1.875 3.999m18.558-11.148c-1.18-.967-2.665-1.68-4.288-2.102L70.015 37.99c1.795-2.191 5.027-2.512 7.218-.717s2.513 5.026.718 7.218L63.816 61.745c-.944-1.947-2.324-3.417-3.597-4.46"
                })
            }),
            /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("g", {
                clipPath: "url(#f090a6__a)",
                children: /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#f090a6__b)",
                    children: [
                        /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("circle", {
                            cx: 86.513,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#f090a6__c)"
                        }),
                        /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("path", {
                            fill: "url(#f090a6__d)",
                            d: "M-7.698 30.067c-1.54-33.182 24.11-61.33 57.291-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.541 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.87-57.291"
                        }),
                        /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("path", {
                            fill: "url(#f090a6__e)",
                            d: "M25.194-66.43c-7.485-4.632-16.97-4.66-24.482-.073l-75.146 45.881c-7.36 4.494-7.372 15.1-.022 19.61L.193 44.806c7.502 4.604 16.986 4.598 24.481-.016L99.17-1.071c7.311-4.5 7.324-15.048.023-19.566z"
                        }),
                        /*#__PURE__*/ (0, $8qIAO$reactjsxruntime.jsx)("path", {
                            fill: "url(#f090a6__f)",
                            d: "M-46.917 45.918c15.488 0 29.518 6.268 39.676 16.381 20.316 20.26 37.159 20.278 57.474.052 10.158-10.165 24.206-16.45 39.711-16.45 30.995.017 56.103 25.056 56.103 55.967S120.939 157.8 89.944 157.8c-15.54 0-29.588-6.303-39.745-16.468-20.299-20.191-37.124-20.174-57.44.087-10.158 10.13-24.188 16.381-39.676 16.381-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $69dbc03f3ea24da2$var$ForwardRef = /*#__PURE__*/ (0, $8qIAO$react.forwardRef)($69dbc03f3ea24da2$var$SvgComponent);
var $69dbc03f3ea24da2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($69dbc03f3ea24da2$var$ForwardRef);
