// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_volume_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $rC62T$jsxs, jsx as $rC62T$jsx} from "react/jsx-runtime";
import {forwardRef as $rC62T$forwardRef} from "react";





const $ef7a894ad7c6a397$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $rC62T$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $rC62T$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $rC62T$jsxs)("linearGradient", {
                        id: "7dbaf0__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $rC62T$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $rC62T$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $rC62T$jsxs)("linearGradient", {
                        id: "7dbaf0__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $rC62T$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $rC62T$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $rC62T$jsxs)("linearGradient", {
                        id: "7dbaf0__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $rC62T$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $rC62T$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $rC62T$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $rC62T$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $rC62T$jsxs)("radialGradient", {
                        id: "7dbaf0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $rC62T$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $rC62T$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $rC62T$jsx)("clipPath", {
                        id: "7dbaf0__b",
                        children: /*#__PURE__*/ (0, $rC62T$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $rC62T$jsx)("clipPath", {
                id: "7dbaf0__a",
                children: /*#__PURE__*/ (0, $rC62T$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M46.963 52.997h-12.65c-4.419 0-8 3.582-8 8v38.205c0 4.418 3.581 8 8 8h12.65zm8-4.867v63.89L70.223 127c3.796 3.725 10.204 1.036 10.204-4.282V37.282c0-5.333-6.437-8.016-10.225-4.261zm36.782 9.102c1.328-1.766 3.836-2.12 5.601-.791 7.104 5.346 11.711 13.861 11.711 23.452 0 9.67-4.682 18.245-11.886 23.583-1.775 1.315-4.28.943-5.595-.832s-.942-4.28.833-5.596c5.254-3.894 8.648-10.127 8.648-17.155 0-6.97-3.339-13.16-8.521-17.06-1.765-1.328-2.12-3.836-.791-5.601m22.536-16.245c-1.765-1.328-4.273-.974-5.601.791s-.974 4.273.791 5.602c9.856 7.416 16.217 19.2 16.217 32.47 0 13.378-6.466 25.246-16.459 32.651-1.775 1.315-2.148 3.82-.833 5.595s3.821 2.148 5.596.832c11.942-8.849 19.696-23.058 19.696-39.078 0-15.89-7.629-30-19.407-38.862"
                })
            }),
            /*#__PURE__*/ (0, $rC62T$jsx)("g", {
                clipPath: "url(#7dbaf0__a)",
                children: /*#__PURE__*/ (0, $rC62T$jsxs)("g", {
                    clipPath: "url(#7dbaf0__b)",
                    children: [
                        /*#__PURE__*/ (0, $rC62T$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $rC62T$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#7dbaf0__c)"
                        }),
                        /*#__PURE__*/ (0, $rC62T$jsx)("path", {
                            fill: "url(#7dbaf0__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $rC62T$jsx)("path", {
                            fill: "url(#7dbaf0__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $rC62T$jsx)("path", {
                            fill: "url(#7dbaf0__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $ef7a894ad7c6a397$var$ForwardRef = /*#__PURE__*/ (0, $rC62T$forwardRef)($ef7a894ad7c6a397$var$SvgComponent);
var $ef7a894ad7c6a397$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($ef7a894ad7c6a397$var$ForwardRef);


export {$ef7a894ad7c6a397$export$2e2bcd8739ae039 as default};
