// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_folderShared_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $lY0pz$reactjsxruntime = require("react/jsx-runtime");
var $lY0pz$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $439b8f926ca1b653$export$2e2bcd8739ae039);




const $439b8f926ca1b653$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1f063c__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1f063c__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsxs)("linearGradient", {
                        id: "1f063c__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsxs)("radialGradient", {
                        id: "1f063c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("clipPath", {
                        id: "1f063c__b",
                        children: /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("clipPath", {
                id: "1f063c__a",
                children: /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M12 25.8h30.927l-5.591-6.928c-1.057-1.309-2.657-2.071-4.348-2.071h-15.42C14.493 16.8 12 19.272 12 22.32zm0 4.8h63.692C80.278 30.6 84 34.35 84 38.966v31.871c0 4.617-3.722 8.365-8.308 8.365H20.308C15.723 79.2 12 75.453 12 70.836zM43.377 46c-.704 0-1.274.57-1.274 1.273 0 .36.147.681.388.915.794.768.814 2.034.046 2.828-.768.793-2.035.814-2.828.046-.988-.957-1.606-2.302-1.606-3.789 0-2.912 2.361-5.273 5.273-5.273 1.105 0 2 .895 2 2s-.895 2-2 2m2.711 3.294c0-2.729 2.212-4.941 4.941-4.941 2.73 0 4.941 2.212 4.941 4.941 0 2.73-2.212 4.941-4.94 4.941-2.73 0-4.942-2.212-4.942-4.94m-3.183 6.252c.476.997.054 2.191-.943 2.667-1.262.603-2.234 1.85-2.49 3.358-.185 1.089-1.218 1.821-2.307 1.637-1.089-.185-1.822-1.218-1.637-2.307.472-2.78 2.264-5.129 4.71-6.297.997-.476 2.191-.054 2.667.942m-.265 6.761c.882-3.659 4.442-6.19 8.39-6.19 3.947 0 7.506 2.531 8.388 6.19.514 2.134-1.29 3.693-3.04 3.693H45.68c-1.749 0-3.554-1.56-3.04-3.693"
                })
            }),
            /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("g", {
                clipPath: "url(#1f063c__a)",
                children: /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#1f063c__b)",
                    children: [
                        /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#1f063c__c)"
                        }),
                        /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("path", {
                            fill: "url(#1f063c__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("path", {
                            fill: "url(#1f063c__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $lY0pz$reactjsxruntime.jsx)("path", {
                            fill: "url(#1f063c__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $439b8f926ca1b653$var$ForwardRef = /*#__PURE__*/ (0, $lY0pz$react.forwardRef)($439b8f926ca1b653$var$SvgComponent);
var $439b8f926ca1b653$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($439b8f926ca1b653$var$ForwardRef);
