// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_chatbubble_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3c5KM$reactjsxruntime = require("react/jsx-runtime");
var $3c5KM$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $207a28571f488c0e$export$2e2bcd8739ae039);




const $207a28571f488c0e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8bb3ff__d",
                        x1: 33.33,
                        x2: 9.156,
                        y1: 1.786,
                        y2: 33.128,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8bb3ff__e",
                        x1: 18.602,
                        x2: 23.265,
                        y1: -12.643,
                        y2: 17.203,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8bb3ff__f",
                        x1: -35.206,
                        x2: 43.258,
                        y1: 47.361,
                        y2: 31.854,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8bb3ff__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 4.884 35.455)scale(60.2686)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("clipPath", {
                        id: "8bb3ff__b",
                        children: /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("clipPath", {
                id: "8bb3ff__a",
                children: /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M35.89 10.781H12.108c-2.074 0-3.755 1.569-3.755 3.504v12.432c0 1.935 1.681 3.504 3.755 3.504h3.363c.008 1.653-.006 4.018-.02 5.82-.008 1.005 1.258 1.548 2.072.89l8.297-6.71h10.072c2.074 0 3.755-1.569 3.755-3.504V14.285c0-1.935-1.681-3.504-3.755-3.504"
                })
            }),
            /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("g", {
                clipPath: "url(#8bb3ff__a)",
                children: /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#8bb3ff__b)",
                    children: [
                        /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("circle", {
                            cx: 40.339,
                            cy: 30.571,
                            r: 60.269,
                            fill: "url(#8bb3ff__c)"
                        }),
                        /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("path", {
                            fill: "url(#8bb3ff__d)",
                            d: "M-.63 16.202C-1.3 1.774 9.854-10.465 24.282-11.135S50.949-.652 51.619 13.776 41.136 40.444 26.708 41.113.04 30.63-.63 16.203"
                        }),
                        /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("path", {
                            fill: "url(#8bb3ff__e)",
                            d: "M13.672-25.756c-3.254-2.014-7.378-2.027-10.645-.032l-32.675 19.95c-3.2 1.954-3.205 6.566-.01 8.527l32.46 19.922c3.261 2.002 7.385 2 10.644-.007l32.392-19.94c3.179-1.958 3.185-6.544.01-8.508z"
                        }),
                        /*#__PURE__*/ (0, $3c5KM$reactjsxruntime.jsx)("path", {
                            fill: "url(#8bb3ff__f)",
                            d: "M-17.682 23.095c6.735 0 12.836 2.726 17.252 7.123 8.834 8.81 16.158 8.817 24.991.023 4.417-4.42 10.525-7.153 17.267-7.153 13.477.007 24.395 10.895 24.395 24.335S55.305 71.745 41.828 71.745c-6.757 0-12.865-2.74-17.282-7.16-8.826-8.78-16.142-8.773-24.976.037-4.416 4.405-10.517 7.123-17.252 7.123-13.469 0-24.386-10.888-24.386-24.32s10.917-24.329 24.386-24.329"
                        })
                    ]
                })
            })
        ]
    });
const $207a28571f488c0e$var$ForwardRef = /*#__PURE__*/ (0, $3c5KM$react.forwardRef)($207a28571f488c0e$var$SvgComponent);
var $207a28571f488c0e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($207a28571f488c0e$var$ForwardRef);
