// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_audioWave_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $88dQr$reactjsxruntime = require("react/jsx-runtime");
var $88dQr$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d965aeb47db39976$export$2e2bcd8739ae039);




const $d965aeb47db39976$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a7270b__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a7270b__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a7270b__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a7270b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("clipPath", {
                        id: "a7270b__b",
                        children: /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("clipPath", {
                id: "a7270b__a",
                children: /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M84 22.568c0-2.209-1.79-4-4-4s-4 1.791-4 4v114.863c0 2.209 1.791 4 4 4s4-1.791 4-4zM28.25 67.33c2.21 0 4 1.79 4 4v17.341c0 2.21-1.79 4-4 4s-4-1.79-4-4v-17.34c0-2.21 1.79-4 4-4m103.5 0c2.209 0 4 1.79 4 4v17.341c0 2.21-1.791 4-4 4s-4-1.79-4-4v-17.34c0-2.21 1.791-4 4-4m-69-20.898c2.21 0 4 1.79 4 4v59.137c0 2.209-1.79 4-4 4s-4-1.791-4-4V50.432c0-2.21 1.79-4 4-4m38.5 4c0-2.21-1.79-4-4-4s-4 1.79-4 4v59.137c0 2.209 1.79 4 4 4s4-1.791 4-4zM45.5 32.512c2.209 0 4 1.79 4 4v86.976c0 2.209-1.791 4-4 4s-4-1.791-4-4V36.512c0-2.21 1.79-4 4-4m73 4c0-2.21-1.791-4-4-4s-4 1.79-4 4v86.976c0 2.209 1.79 4 4 4s4-1.791 4-4z"
                })
            }),
            /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("g", {
                clipPath: "url(#a7270b__a)",
                children: /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#a7270b__b)",
                    children: [
                        /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#a7270b__c)"
                        }),
                        /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("path", {
                            fill: "url(#a7270b__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("path", {
                            fill: "url(#a7270b__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $88dQr$reactjsxruntime.jsx)("path", {
                            fill: "url(#a7270b__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $d965aeb47db39976$var$ForwardRef = /*#__PURE__*/ (0, $88dQr$react.forwardRef)($d965aeb47db39976$var$SvgComponent);
var $d965aeb47db39976$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d965aeb47db39976$var$ForwardRef);
