// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_userAvatar_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $aTvJS$reactjsxruntime = require("react/jsx-runtime");
var $aTvJS$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $38fd447c37fbfcff$export$2e2bcd8739ae039);




const $38fd447c37fbfcff$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "be9e22__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "be9e22__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "be9e22__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsxs)("radialGradient", {
                        id: "be9e22__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("clipPath", {
                        id: "be9e22__b",
                        children: /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("clipPath", {
                id: "be9e22__a",
                children: /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M68.816 73.348C63.138 77.97 55.892 80.74 48 80.74c-7.86 0-15.078-2.748-20.746-7.335.162-.164.298-.359.398-.58C31.17 65.051 39.012 59.608 48 59.608c8.914 0 16.701 5.358 20.26 13.025.133.287.326.528.556.714m2.932-2.694C67.505 61.803 58.431 55.609 48 55.609c-10.43 0-19.505 6.188-23.749 15.044C18.523 64.717 15 56.64 15 47.74c0-18.225 14.775-33 33-33s33 14.775 33 33c0 8.9-3.523 16.978-9.252 22.914m-33.834-31.9c0-6.13 4.634-10.879 10.088-10.879s10.087 4.75 10.087 10.878c0 6.13-4.633 10.878-10.087 10.878s-10.088-4.749-10.088-10.878m10.088-14.879c-7.898 0-14.088 6.782-14.088 14.878s6.19 14.878 14.088 14.878 14.087-6.782 14.087-14.878-6.19-14.878-14.087-14.878"
                })
            }),
            /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("g", {
                clipPath: "url(#be9e22__a)",
                children: /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#be9e22__b)",
                    children: [
                        /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#be9e22__c)"
                        }),
                        /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("path", {
                            fill: "url(#be9e22__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("path", {
                            fill: "url(#be9e22__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $aTvJS$reactjsxruntime.jsx)("path", {
                            fill: "url(#be9e22__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $38fd447c37fbfcff$var$ForwardRef = /*#__PURE__*/ (0, $aTvJS$react.forwardRef)($38fd447c37fbfcff$var$SvgComponent);
var $38fd447c37fbfcff$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($38fd447c37fbfcff$var$ForwardRef);
