// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_accessibility_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $hXIeI$jsxs, jsx as $hXIeI$jsx} from "react/jsx-runtime";
import {forwardRef as $hXIeI$forwardRef} from "react";





const $a88c0ef3cb256ab4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $hXIeI$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $hXIeI$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $hXIeI$jsxs)("linearGradient", {
                        id: "a54e75__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hXIeI$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $hXIeI$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hXIeI$jsxs)("linearGradient", {
                        id: "a54e75__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hXIeI$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $hXIeI$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hXIeI$jsxs)("linearGradient", {
                        id: "a54e75__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hXIeI$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $hXIeI$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $hXIeI$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $hXIeI$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hXIeI$jsxs)("radialGradient", {
                        id: "a54e75__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $hXIeI$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $hXIeI$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $hXIeI$jsx)("clipPath", {
                        id: "a54e75__b",
                        children: /*#__PURE__*/ (0, $hXIeI$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $hXIeI$jsx)("clipPath", {
                id: "a54e75__a",
                children: /*#__PURE__*/ (0, $hXIeI$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M48 81c18.225 0 33-14.775 33-33S66.225 15 48 15 15 29.775 15 48s14.775 33 33 33m5-50.853c0 2.76-2.238 5-5 5s-5-2.24-5-5c0-2.762 2.239-5 5-5s5 2.238 5 5m-22.038 7.116c.304-1.062 1.412-1.677 2.473-1.373 9.396 2.689 19.353 2.724 28.768.101l.376-.105c1.064-.296 2.167.326 2.463 1.39s-.326 2.167-1.39 2.464l-.376.104c-2.762.77-5.568 1.326-8.392 1.67-1.028.126-1.82.986-1.82 2.022v7.905q0 .279.077.546c.8 2.82 1.714 6.05 2.515 8.887.576 2.04 1.094 3.878 1.47 5.218l.448 1.6.025.087c.078.281.16.575.181.681.217 1.084-.486 2.137-1.569 2.354s-2.137-.486-2.353-1.57c.01.054.002.023-.048-.16l-.001-.002-.088-.317-.446-1.592c-.375-1.337-.893-3.173-1.468-5.212-.657-2.327-1.39-4.917-2.073-7.326-.22-.776-.928-1.31-1.734-1.31s-1.514.534-1.734 1.31c-.683 2.41-1.416 4.999-2.073 7.326-.575 2.039-1.093 3.875-1.468 5.212l-.446 1.592-.06.218-.028.102c-.05.182-.06.213-.049.16-.216 1.083-1.27 1.785-2.353 1.569-1.083-.217-1.786-1.27-1.57-2.353.022-.108.104-.401.182-.682l.025-.087.448-1.6c.376-1.34.894-3.178 1.47-5.218.801-2.838 1.715-6.067 2.515-8.887q.075-.267.076-.546v-7.905c0-1.035-.79-1.896-1.819-2.02-2.958-.361-5.894-.954-8.78-1.78-1.063-.304-1.677-1.411-1.374-2.473"
                })
            }),
            /*#__PURE__*/ (0, $hXIeI$jsx)("g", {
                clipPath: "url(#a54e75__a)",
                children: /*#__PURE__*/ (0, $hXIeI$jsxs)("g", {
                    clipPath: "url(#a54e75__b)",
                    children: [
                        /*#__PURE__*/ (0, $hXIeI$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $hXIeI$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#a54e75__c)"
                        }),
                        /*#__PURE__*/ (0, $hXIeI$jsx)("path", {
                            fill: "url(#a54e75__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $hXIeI$jsx)("path", {
                            fill: "url(#a54e75__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $hXIeI$jsx)("path", {
                            fill: "url(#a54e75__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $a88c0ef3cb256ab4$var$ForwardRef = /*#__PURE__*/ (0, $hXIeI$forwardRef)($a88c0ef3cb256ab4$var$SvgComponent);
var $a88c0ef3cb256ab4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a88c0ef3cb256ab4$var$ForwardRef);


export {$a88c0ef3cb256ab4$export$2e2bcd8739ae039 as default};
