// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_link_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $054zD$jsxs, jsx as $054zD$jsx} from "react/jsx-runtime";
import {forwardRef as $054zD$forwardRef} from "react";





const $5325bd54cdb4479d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $054zD$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $054zD$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $054zD$jsxs)("linearGradient", {
                        id: "ca8b0a__d",
                        x1: 65.566,
                        x2: 25.102,
                        y1: 11.635,
                        y2: 67.287,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $054zD$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $054zD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $054zD$jsxs)("linearGradient", {
                        id: "ca8b0a__e",
                        x1: 38.978,
                        x2: 46.479,
                        y1: -12.921,
                        y2: 38.007,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $054zD$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $054zD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $054zD$jsxs)("linearGradient", {
                        id: "ca8b0a__f",
                        x1: -58.165,
                        x2: 82.825,
                        y1: 89.196,
                        y2: 59.637,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $054zD$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $054zD$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $054zD$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $054zD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $054zD$jsxs)("radialGradient", {
                        id: "ca8b0a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 78.212 60.623)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $054zD$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $054zD$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $054zD$jsx)("clipPath", {
                        id: "ca8b0a__b",
                        children: /*#__PURE__*/ (0, $054zD$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $054zD$jsx)("clipPath", {
                id: "ca8b0a__a",
                children: /*#__PURE__*/ (0, $054zD$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    stroke: "#fff",
                    strokeLinecap: "round",
                    strokeLinejoin: "round",
                    strokeWidth: 1.2,
                    d: "M73.916 34.728c3.509-3.509 3.509-9.197 0-12.706-3.508-3.508-9.197-3.508-12.705 0l-6.548 6.548c-.937.937-2.457.937-3.394 0s-.937-2.457 0-3.395l6.548-6.547c5.383-5.383 14.11-5.383 19.493 0s5.383 14.11 0 19.494L60.708 54.724c-5.383 5.383-14.11 5.383-19.494 0-.937-.937-.937-2.456 0-3.394.937-.937 2.457-.937 3.394 0 3.509 3.509 9.197 3.509 12.706 0zm-18.993 6.41c.937.937.937 2.457 0 3.394s-2.457.937-3.394 0c-3.509-3.508-9.197-3.508-12.705 0L22.22 61.135c-3.509 3.508-3.509 9.197 0 12.705s9.197 3.509 12.705 0l6.548-6.547c.937-.938 2.457-.938 3.394 0s.937 2.456 0 3.394l-6.548 6.547c-5.383 5.383-14.11 5.383-19.493 0s-5.383-14.11 0-19.493l16.602-16.603c5.383-5.383 14.11-5.383 19.494 0"
                })
            }),
            /*#__PURE__*/ (0, $054zD$jsx)("g", {
                clipPath: "url(#ca8b0a__a)",
                children: /*#__PURE__*/ (0, $054zD$jsxs)("g", {
                    clipPath: "url(#ca8b0a__b)",
                    children: [
                        /*#__PURE__*/ (0, $054zD$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $054zD$jsx)("ellipse", {
                            cx: 78.212,
                            cy: 60.623,
                            fill: "url(#ca8b0a__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $054zD$jsx)("path", {
                            fill: "url(#ca8b0a__d)",
                            d: "M4.256 36.17C3.046 11.615 23.182-9.214 49.23-10.354S97.374 7.486 98.583 32.04 79.658 77.425 53.61 78.565 5.466 60.725 4.256 36.17"
                        }),
                        /*#__PURE__*/ (0, $054zD$jsx)("path", {
                            fill: "url(#ca8b0a__e)",
                            d: "M30.078-35.238c-5.876-3.427-13.321-3.448-19.219-.054L-48.13-1.34c-5.777 3.326-5.787 11.174-.017 14.512l58.599 33.904c5.889 3.407 13.334 3.403 19.218-.012l58.478-33.936c5.74-3.33 5.75-11.135.018-14.479z"
                        }),
                        /*#__PURE__*/ (0, $054zD$jsx)("path", {
                            fill: "url(#ca8b0a__f)",
                            d: "M-26.528 47.899c12.159 0 23.172 4.638 31.146 12.122 15.948 14.993 29.17 15.005 45.118.038 7.974-7.522 19-12.173 31.173-12.173 24.33.013 44.04 18.542 44.04 41.416s-19.71 41.39-44.04 41.39c-12.2 0-23.227-4.665-31.2-12.187-15.935-14.941-29.143-14.928-45.09.064-7.975 7.497-18.988 12.123-31.147 12.123-24.317 0-44.026-18.53-44.026-41.39 0-22.861 19.71-41.403 44.026-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $5325bd54cdb4479d$var$ForwardRef = /*#__PURE__*/ (0, $054zD$forwardRef)($5325bd54cdb4479d$var$SvgComponent);
var $5325bd54cdb4479d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($5325bd54cdb4479d$var$ForwardRef);


export {$5325bd54cdb4479d$export$2e2bcd8739ae039 as default};
