// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_clipboard_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $h9nlt$jsxs, jsx as $h9nlt$jsx} from "react/jsx-runtime";
import {forwardRef as $h9nlt$forwardRef} from "react";





const $7631a89e1029654f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $h9nlt$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $h9nlt$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $h9nlt$jsxs)("linearGradient", {
                        id: "a80734__d",
                        x1: 65.566,
                        x2: 25.102,
                        y1: 11.636,
                        y2: 67.288,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h9nlt$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $h9nlt$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h9nlt$jsxs)("linearGradient", {
                        id: "a80734__e",
                        x1: 38.978,
                        x2: 46.479,
                        y1: -12.921,
                        y2: 38.007,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h9nlt$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $h9nlt$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h9nlt$jsxs)("linearGradient", {
                        id: "a80734__f",
                        x1: -58.165,
                        x2: 82.825,
                        y1: 89.197,
                        y2: 59.638,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h9nlt$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $h9nlt$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $h9nlt$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $h9nlt$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h9nlt$jsxs)("radialGradient", {
                        id: "a80734__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 78.212 60.623)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $h9nlt$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $h9nlt$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $h9nlt$jsx)("clipPath", {
                        id: "a80734__b",
                        children: /*#__PURE__*/ (0, $h9nlt$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $h9nlt$jsx)("clipPath", {
                id: "a80734__a",
                children: /*#__PURE__*/ (0, $h9nlt$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M32.999 24h-5.155c-4.112 0-7.446 3.37-7.446 7.528v40.146c0 4.157 3.334 7.527 7.446 7.527h39.709c4.112 0 7.445-3.37 7.445-7.527V31.528c0-4.157-3.333-7.527-7.445-7.527h-5.754v.017l-.012 9.6c0 .636-.254 1.247-.705 1.696-.45.45-1.061.702-1.698.701l-24-.029c-1.325-.001-2.398-1.077-2.397-2.402zm4.791 7.19.006-4.8 2.4.002c1.325.002 2.401-1.072 2.403-2.397l.002-2.4c.004-2.65 2.155-4.797 4.806-4.794s4.798 2.155 4.794 4.806l-.002 2.4c-.002 1.325 1.071 2.4 2.397 2.402l2.4.003-.006 4.8z"
                })
            }),
            /*#__PURE__*/ (0, $h9nlt$jsx)("g", {
                clipPath: "url(#a80734__a)",
                children: /*#__PURE__*/ (0, $h9nlt$jsxs)("g", {
                    clipPath: "url(#a80734__b)",
                    children: [
                        /*#__PURE__*/ (0, $h9nlt$jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $h9nlt$jsx)("ellipse", {
                            cx: 78.212,
                            cy: 60.623,
                            fill: "url(#a80734__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $h9nlt$jsx)("path", {
                            fill: "url(#a80734__d)",
                            d: "M4.256 36.171C3.046 11.616 23.182-9.213 49.23-10.353s48.144 17.84 49.353 42.395S79.658 77.426 53.61 78.566 5.466 60.726 4.256 36.171"
                        }),
                        /*#__PURE__*/ (0, $h9nlt$jsx)("path", {
                            fill: "url(#a80734__e)",
                            d: "M30.078-35.238c-5.876-3.427-13.321-3.448-19.219-.054L-48.13-1.34c-5.777 3.326-5.787 11.174-.017 14.512l58.599 33.904c5.889 3.407 13.334 3.403 19.218-.012l58.478-33.936c5.74-3.33 5.75-11.135.018-14.479z"
                        }),
                        /*#__PURE__*/ (0, $h9nlt$jsx)("path", {
                            fill: "url(#a80734__f)",
                            d: "M-26.528 47.9c12.159 0 23.172 4.638 31.146 12.122 15.948 14.993 29.17 15.005 45.118.038 7.974-7.522 19-12.173 31.173-12.173 24.33.013 44.04 18.542 44.04 41.415 0 22.874-19.71 41.391-44.04 41.391-12.2 0-23.227-4.665-31.2-12.187-15.935-14.941-29.143-14.928-45.09.064-7.975 7.497-18.988 12.123-31.147 12.123-24.317 0-44.026-18.53-44.026-41.39 0-22.861 19.71-41.403 44.026-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $7631a89e1029654f$var$ForwardRef = /*#__PURE__*/ (0, $h9nlt$forwardRef)($7631a89e1029654f$var$SvgComponent);
var $7631a89e1029654f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7631a89e1029654f$var$ForwardRef);


export {$7631a89e1029654f$export$2e2bcd8739ae039 as default};
