// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_data_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $a8He0$jsxs, jsx as $a8He0$jsx} from "react/jsx-runtime";
import {forwardRef as $a8He0$forwardRef} from "react";





const $76900bfca826e04c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $a8He0$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $a8He0$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $a8He0$jsxs)("linearGradient", {
                        id: "08bfd7__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a8He0$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $a8He0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a8He0$jsxs)("linearGradient", {
                        id: "08bfd7__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a8He0$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $a8He0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a8He0$jsxs)("linearGradient", {
                        id: "08bfd7__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a8He0$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $a8He0$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $a8He0$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $a8He0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a8He0$jsxs)("radialGradient", {
                        id: "08bfd7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a8He0$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $a8He0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a8He0$jsx)("clipPath", {
                        id: "08bfd7__b",
                        children: /*#__PURE__*/ (0, $a8He0$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $a8He0$jsx)("clipPath", {
                id: "08bfd7__a",
                children: /*#__PURE__*/ (0, $a8He0$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m127.749 44.509.001-.082c0-6.854-21.379-12.41-47.75-12.41-26.372 0-47.75 5.556-47.75 12.41l.001.082v.197c.159.516.693 1.431 2.373 2.662 2.123 1.556 5.451 3.125 9.923 4.512C53.445 54.638 65.98 56.406 80 56.406c14.02 0 26.555-1.768 35.452-4.526 4.473-1.387 7.801-2.956 9.924-4.512 1.683-1.233 2.216-2.15 2.374-2.665v-.194zM32.251 62.557v-7.204c2.78 1.615 6.164 3.001 9.927 4.168 9.856 3.056 23.227 4.885 37.822 4.885 14.594 0 27.965-1.83 37.821-4.885 3.764-1.167 7.148-2.553 9.929-4.169v7.202c-.158.515-.691 1.431-2.374 2.665-2.123 1.556-5.451 3.125-9.924 4.512-8.897 2.758-21.432 4.525-35.452 4.525s-26.556-1.767-35.452-4.525c-4.473-1.387-7.801-2.956-9.924-4.512-1.68-1.231-2.214-2.146-2.373-2.662m95.499 10.646c-2.781 1.616-6.165 3.002-9.929 4.169-9.856 3.055-23.227 4.884-37.821 4.884-14.595 0-27.966-1.829-37.822-4.884-3.763-1.167-7.147-2.553-9.927-4.168v7.16c.159.516.693 1.431 2.373 2.663 2.123 1.556 5.451 3.124 9.923 4.511C53.445 90.296 65.98 92.064 80 92.064c14.02 0 26.555-1.768 35.452-4.526 4.473-1.386 7.801-2.955 9.924-4.511 1.683-1.234 2.216-2.15 2.374-2.665zM32.25 98.26v-7.249c2.781 1.616 6.165 3.002 9.928 4.168 9.856 3.056 23.227 4.885 37.822 4.885 14.594 0 27.965-1.83 37.821-4.885 3.764-1.166 7.148-2.553 9.929-4.168v7.246c-.158.515-.691 1.431-2.374 2.665-2.123 1.556-5.451 3.125-9.924 4.512-8.897 2.758-21.432 4.525-35.452 4.525s-26.556-1.767-35.452-4.525c-4.473-1.387-7.801-2.956-9.924-4.512-1.68-1.231-2.214-2.146-2.373-2.662m0 10.647v7.616c10.25 13.938 81.25 16.563 95.5 0v-7.617c-2.781 1.616-6.165 3.002-9.929 4.169-9.856 3.055-23.227 4.884-37.821 4.884-14.595 0-27.966-1.829-37.822-4.884-3.763-1.167-7.147-2.553-9.927-4.168"
                })
            }),
            /*#__PURE__*/ (0, $a8He0$jsx)("g", {
                clipPath: "url(#08bfd7__a)",
                children: /*#__PURE__*/ (0, $a8He0$jsxs)("g", {
                    clipPath: "url(#08bfd7__b)",
                    children: [
                        /*#__PURE__*/ (0, $a8He0$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $a8He0$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#08bfd7__c)"
                        }),
                        /*#__PURE__*/ (0, $a8He0$jsx)("path", {
                            fill: "url(#08bfd7__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $a8He0$jsx)("path", {
                            fill: "url(#08bfd7__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $a8He0$jsx)("path", {
                            fill: "url(#08bfd7__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $76900bfca826e04c$var$ForwardRef = /*#__PURE__*/ (0, $a8He0$forwardRef)($76900bfca826e04c$var$SvgComponent);
var $76900bfca826e04c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($76900bfca826e04c$var$ForwardRef);


export {$76900bfca826e04c$export$2e2bcd8739ae039 as default};
