// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_brand_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $Fh0Jx$jsxs, jsx as $Fh0Jx$jsx} from "react/jsx-runtime";
import {forwardRef as $Fh0Jx$forwardRef} from "react";





const $e98377240c9ae52e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $Fh0Jx$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $Fh0Jx$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $Fh0Jx$jsxs)("linearGradient", {
                        id: "51323d__d",
                        x1: 32.635,
                        x2: 11.534,
                        y1: 4.21,
                        y2: 33.232,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $Fh0Jx$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $Fh0Jx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $Fh0Jx$jsxs)("linearGradient", {
                        id: "51323d__e",
                        x1: 18.767,
                        x2: 22.679,
                        y1: -8.597,
                        y2: 17.962,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $Fh0Jx$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $Fh0Jx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $Fh0Jx$jsxs)("linearGradient", {
                        id: "51323d__f",
                        x1: -31.892,
                        x2: 41.634,
                        y1: 44.657,
                        y2: 29.241,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $Fh0Jx$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $Fh0Jx$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $Fh0Jx$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $Fh0Jx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $Fh0Jx$jsxs)("radialGradient", {
                        id: "51323d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.4895 -56.7422 0 39.23 29.755)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $Fh0Jx$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $Fh0Jx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $Fh0Jx$jsx)("clipPath", {
                        id: "51323d__b",
                        children: /*#__PURE__*/ (0, $Fh0Jx$jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 5.4,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $Fh0Jx$jsx)("clipPath", {
                id: "51323d__a",
                children: /*#__PURE__*/ (0, $Fh0Jx$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M13.725 7.818h20.55c1.989 0 3.6 1.611 3.6 3.6v14.89c0 7.664-6.21 13.874-13.875 13.874S10.125 33.97 10.125 26.307V11.418c0-1.989 1.611-3.6 3.6-3.6m1.224 19.725c.188.111.395.162.598.162.417 0 .818-.216 1.04-.6l4.188-7.254c.333-.573.135-1.308-.438-1.638-.573-.333-1.308-.135-1.638.438l-4.188 7.254c-.333.573-.135 1.308.438 1.638m15.756.162c.867 0 1.644-.45 2.079-1.2.435-.753.435-1.65 0-2.4l-3.771-6.53c-.435-.75-1.212-1.2-2.079-1.2s-1.644.45-2.079 1.2l-3.771 6.53c-.435.75-.435 1.65 0 2.4s1.212 1.2 2.079 1.2zm-3.77-8.934 3.77 6.534h-7.542z"
                })
            }),
            /*#__PURE__*/ (0, $Fh0Jx$jsx)("g", {
                clipPath: "url(#51323d__a)",
                children: /*#__PURE__*/ (0, $Fh0Jx$jsxs)("g", {
                    clipPath: "url(#51323d__b)",
                    children: [
                        /*#__PURE__*/ (0, $Fh0Jx$jsx)("rect", {
                            width: 39.3,
                            height: 37.047,
                            x: 4.199,
                            y: 5.4,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $Fh0Jx$jsx)("ellipse", {
                            cx: 39.23,
                            cy: 29.755,
                            fill: "url(#51323d__c)",
                            rx: 56.742,
                            ry: 53.489
                        }),
                        /*#__PURE__*/ (0, $Fh0Jx$jsx)("path", {
                            fill: "url(#51323d__d)",
                            d: "M.663 17.005C.032 4.199 10.533-6.663 24.116-7.258c13.584-.594 25.107 9.304 25.738 22.11.63 12.805-9.87 23.667-23.454 24.262-13.583.594-25.107-9.304-25.737-22.11"
                        }),
                        /*#__PURE__*/ (0, $Fh0Jx$jsx)("path", {
                            fill: "url(#51323d__e)",
                            d: "M14.126-20.236c-3.064-1.787-6.947-1.798-10.022-.028L-26.66-2.558c-3.013 1.734-3.018 5.827-.01 7.568l30.56 17.681c3.071 1.777 6.954 1.774 10.022-.006L44.41 4.987c2.993-1.737 2.998-5.807.009-7.55z"
                        }),
                        /*#__PURE__*/ (0, $Fh0Jx$jsx)("path", {
                            fill: "url(#51323d__f)",
                            d: "M-15.393 23.12c6.34 0 12.084 2.42 16.242 6.322 8.317 7.818 15.212 7.825 23.529.02 4.158-3.923 9.909-6.349 16.257-6.349 12.688.007 22.967 9.67 22.967 21.599s-10.28 21.584-22.967 21.584c-6.362 0-12.113-2.432-16.271-6.355-8.31-7.792-15.198-7.785-23.515.034-4.158 3.909-9.902 6.321-16.242 6.321-12.682 0-22.96-9.663-22.96-21.584 0-11.922 10.278-21.592 22.96-21.592"
                        })
                    ]
                })
            })
        ]
    });
const $e98377240c9ae52e$var$ForwardRef = /*#__PURE__*/ (0, $Fh0Jx$forwardRef)($e98377240c9ae52e$var$SvgComponent);
var $e98377240c9ae52e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e98377240c9ae52e$var$ForwardRef);


export {$e98377240c9ae52e$export$2e2bcd8739ae039 as default};
