// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_logo_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $kxtpN$jsxs, jsx as $kxtpN$jsx} from "react/jsx-runtime";
import {forwardRef as $kxtpN$forwardRef} from "react";





const $7fe56a28ec3f586e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $kxtpN$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $kxtpN$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $kxtpN$jsxs)("linearGradient", {
                        id: "e6c863__d",
                        x1: 65.27,
                        x2: 23.067,
                        y1: 8.417,
                        y2: 66.462,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kxtpN$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $kxtpN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kxtpN$jsxs)("linearGradient", {
                        id: "e6c863__e",
                        x1: 37.538,
                        x2: 45.362,
                        y1: -17.195,
                        y2: 35.923,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kxtpN$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $kxtpN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kxtpN$jsxs)("linearGradient", {
                        id: "e6c863__f",
                        x1: -63.784,
                        x2: 83.268,
                        y1: 89.313,
                        y2: 58.483,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kxtpN$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $kxtpN$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $kxtpN$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $kxtpN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kxtpN$jsxs)("radialGradient", {
                        id: "e6c863__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 106.979 -113.484 0 78.461 59.512)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $kxtpN$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $kxtpN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $kxtpN$jsx)("clipPath", {
                        id: "e6c863__b",
                        children: /*#__PURE__*/ (0, $kxtpN$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 10.801,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $kxtpN$jsx)("clipPath", {
                id: "e6c863__a",
                children: /*#__PURE__*/ (0, $kxtpN$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M51.218 16.043 76.76 28.787c2.442 1.218 3.984 3.714 3.984 6.444v25.476c0 2.724-1.536 5.22-3.972 6.438L51.23 79.943c-2.028 1.02-4.422 1.02-6.45 0L19.238 67.145c-2.436-1.224-3.972-3.714-3.972-6.438V35.231c0-2.73 1.542-5.226 3.984-6.444l25.542-12.744c2.022-1.008 4.404-1.008 6.426 0M30.794 58.56c.834 0 1.638-.432 2.082-1.2h-.006l8.79-15.222c.666-1.146.27-2.616-.876-3.276-1.146-.666-2.616-.27-3.276.876l-8.79 15.222c-.666 1.146-.27 2.616.876 3.276.378.222.792.324 1.2.324m32.004 0c1.734 0 3.288-.9 4.158-2.4.87-1.506.87-3.3 0-4.8l-8.118-14.058c-.87-1.5-2.424-2.4-4.158-2.4s-3.288.9-4.158 2.4L42.404 51.36c-.87 1.5-.87 3.3 0 4.8s2.424 2.4 4.158 2.4zm-8.112-18.858 8.118 14.058H46.568z"
                })
            }),
            /*#__PURE__*/ (0, $kxtpN$jsx)("g", {
                clipPath: "url(#e6c863__a)",
                children: /*#__PURE__*/ (0, $kxtpN$jsxs)("g", {
                    clipPath: "url(#e6c863__b)",
                    children: [
                        /*#__PURE__*/ (0, $kxtpN$jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 10.801,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $kxtpN$jsx)("ellipse", {
                            cx: 78.461,
                            cy: 59.512,
                            fill: "url(#e6c863__c)",
                            rx: 113.484,
                            ry: 106.979
                        }),
                        /*#__PURE__*/ (0, $kxtpN$jsx)("path", {
                            fill: "url(#e6c863__d)",
                            d: "M1.325 34.007C.064 8.397 21.065-13.328 48.233-14.518 75.4-15.707 98.447 4.09 99.708 29.701c1.262 25.61-19.74 47.335-46.907 48.524-27.168 1.19-50.214-18.607-51.476-44.218"
                        }),
                        /*#__PURE__*/ (0, $kxtpN$jsx)("path", {
                            fill: "url(#e6c863__e)",
                            d: "M28.256-40.471c-6.129-3.575-13.894-3.597-20.045-.057L-53.314-5.116C-59.34-1.648-59.35 6.54-53.333 10.02l61.12 35.362c6.141 3.553 13.906 3.549 20.043-.013L88.823 9.974C94.81 6.5 94.82-1.64 88.842-5.127z"
                        }),
                        /*#__PURE__*/ (0, $kxtpN$jsx)("path", {
                            fill: "url(#e6c863__f)",
                            d: "M-30.787 46.24c12.682 0 24.169 4.838 32.485 12.643 16.634 15.638 30.424 15.651 47.058.04 8.316-7.845 19.818-12.696 32.513-12.696 25.377.013 45.934 19.34 45.934 43.196 0 23.857-20.557 43.17-45.934 43.17-12.724 0-24.225-4.865-32.542-12.71-16.62-15.584-30.395-15.571-47.029.066-8.316 7.819-19.803 12.644-32.485 12.644-25.362 0-45.92-19.326-45.92-43.17s20.558-43.183 45.92-43.183"
                        })
                    ]
                })
            })
        ]
    });
const $7fe56a28ec3f586e$var$ForwardRef = /*#__PURE__*/ (0, $kxtpN$forwardRef)($7fe56a28ec3f586e$var$SvgComponent);
var $7fe56a28ec3f586e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7fe56a28ec3f586e$var$ForwardRef);


export {$7fe56a28ec3f586e$export$2e2bcd8739ae039 as default};
