// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_tag_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $8Hxo7$reactjsxruntime = require("react/jsx-runtime");
var $8Hxo7$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $8c931a1a434c0101$export$2e2bcd8739ae039);




const $8c931a1a434c0101$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f6ceea__d",
                        x1: 109.277,
                        x2: 41.838,
                        y1: 19.392,
                        y2: 112.146,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f6ceea__e",
                        x1: 64.961,
                        x2: 77.463,
                        y1: -21.534,
                        y2: 63.346,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsxs)("linearGradient", {
                        id: "f6ceea__f",
                        x1: -96.944,
                        x2: 138.04,
                        y1: 148.661,
                        y2: 99.396,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsxs)("radialGradient", {
                        id: "f6ceea__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 170.948 -181.344 0 130.355 101.038)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("clipPath", {
                        id: "f6ceea__b",
                        children: /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.2,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("clipPath", {
                id: "f6ceea__a",
                children: /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M68.79 29.3c-.54-.53-1.26-.84-2.03-.85L30.95 28c-1.64-.02-2.95 1.29-2.95 2.88v36.18c0 .76.3 1.49.85 2.03l60.31 60.04c3.84 3.82 10.07 3.82 13.91 0l26.07-25.95c3.83-3.81 3.83-9.98 0-13.8L68.8 29.3zM51 59c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8"
                })
            }),
            /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("g", {
                clipPath: "url(#f6ceea__a)",
                children: /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#f6ceea__b)",
                    children: [
                        /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.2,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("ellipse", {
                            cx: 130.355,
                            cy: 101.038,
                            fill: "url(#f6ceea__c)",
                            rx: 181.344,
                            ry: 170.948
                        }),
                        /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("path", {
                            fill: "url(#f6ceea__d)",
                            d: "M7.095 60.284C5.08 19.359 38.638-15.356 82.051-17.257s80.24 29.735 82.256 70.66c2.016 40.923-31.543 75.639-74.956 77.54-43.413 1.9-80.24-29.735-82.256-70.66"
                        }),
                        /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("path", {
                            fill: "url(#f6ceea__e)",
                            d: "M50.129-58.729c-9.794-5.713-22.202-5.748-32.032-.09L-80.218-2.232c-9.629 5.542-9.645 18.623-.03 24.186L17.42 78.461c9.814 5.678 22.223 5.671 32.03-.02l97.464-56.56c9.564-5.551 9.582-18.56.029-24.131z"
                        }),
                        /*#__PURE__*/ (0, $8Hxo7$reactjsxruntime.jsx)("path", {
                            fill: "url(#f6ceea__f)",
                            d: "M-44.215 79.832c20.264 0 38.62 7.731 51.91 20.204 26.58 24.988 48.615 25.009 75.195.064 13.29-12.537 31.668-20.29 51.955-20.29 40.551.022 73.401 30.904 73.401 69.027 0 38.122-32.85 68.984-73.401 68.984-20.332 0-38.71-7.774-52-20.311-26.557-24.903-48.57-24.881-75.15.107-13.29 12.494-31.646 20.204-51.91 20.204-40.529 0-73.378-30.883-73.378-68.984s32.85-69.005 73.378-69.005"
                        })
                    ]
                })
            })
        ]
    });
const $8c931a1a434c0101$var$ForwardRef = /*#__PURE__*/ (0, $8Hxo7$react.forwardRef)($8c931a1a434c0101$var$SvgComponent);
var $8c931a1a434c0101$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($8c931a1a434c0101$var$ForwardRef);
