// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_illustration_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $j6zjy$jsxs, jsx as $j6zjy$jsx} from "react/jsx-runtime";
import {forwardRef as $j6zjy$forwardRef} from "react";





const $1e4ba3c63722a5e7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $j6zjy$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $j6zjy$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $j6zjy$jsxs)("linearGradient", {
                        id: "ed584a__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $j6zjy$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $j6zjy$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $j6zjy$jsxs)("linearGradient", {
                        id: "ed584a__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $j6zjy$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $j6zjy$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $j6zjy$jsxs)("linearGradient", {
                        id: "ed584a__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $j6zjy$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $j6zjy$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $j6zjy$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $j6zjy$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $j6zjy$jsxs)("radialGradient", {
                        id: "ed584a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $j6zjy$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $j6zjy$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $j6zjy$jsx)("clipPath", {
                        id: "ed584a__b",
                        children: /*#__PURE__*/ (0, $j6zjy$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $j6zjy$jsx)("clipPath", {
                id: "ed584a__a",
                children: /*#__PURE__*/ (0, $j6zjy$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m18.928 12.677 6.73 6.73-1.941 3.742c-.44.847-1.166 1.508-2.05 1.866l-10.742 4.348c-.21.085-.42.12-.622.113l6.797-6.797c.458.008.919-.162 1.268-.512.684-.684.684-1.792 0-2.475-.683-.683-1.791-.683-2.475 0-.324.324-.494.744-.51 1.17l-6.317 6.315 4.254-10.509c.358-.884 1.02-1.61 1.866-2.05zm9.703 3.202-1.482 1.482-6.18-6.18 1.481-1.482c.781-.781 2.048-.781 2.829 0l3.352 3.352c.78.781.78 2.047 0 2.828m-8.42 18.35c-1.011 2.202-1.36 2.96-3.217 4.407 3.01.347 10.315.927 12.263-2.966.78-1.559.718-2.813.288-3.788-.309-.7-.808-1.257-1.323-1.679-.394-.322-.955-.577-1.598-.714-1.792-.38-4.225.16-5.476 2.742-.38.783-.68 1.438-.938 1.999m9.595-5.96c-.596-.488-1.323-.853-2.115-1.087l6.697-8.174c.897-1.096 2.513-1.256 3.608-.359 1.096.898 1.257 2.513.36 3.609l-6.722 8.205c-.506-.954-1.198-1.677-1.828-2.193"
                })
            }),
            /*#__PURE__*/ (0, $j6zjy$jsx)("g", {
                clipPath: "url(#ed584a__a)",
                children: /*#__PURE__*/ (0, $j6zjy$jsxs)("g", {
                    clipPath: "url(#ed584a__b)",
                    children: [
                        /*#__PURE__*/ (0, $j6zjy$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $j6zjy$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#ed584a__c)"
                        }),
                        /*#__PURE__*/ (0, $j6zjy$jsx)("path", {
                            fill: "url(#ed584a__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $j6zjy$jsx)("path", {
                            fill: "url(#ed584a__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $j6zjy$jsx)("path", {
                            fill: "url(#ed584a__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $1e4ba3c63722a5e7$var$ForwardRef = /*#__PURE__*/ (0, $j6zjy$forwardRef)($1e4ba3c63722a5e7$var$SvgComponent);
var $1e4ba3c63722a5e7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1e4ba3c63722a5e7$var$ForwardRef);


export {$1e4ba3c63722a5e7$export$2e2bcd8739ae039 as default};
