// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_exposure_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $7oU0u$reactjsxruntime = require("react/jsx-runtime");
var $7oU0u$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $85a476ce2ce9f646$export$2e2bcd8739ae039);




const $85a476ce2ce9f646$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsxs)("linearGradient", {
                        id: "95df62__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsxs)("linearGradient", {
                        id: "95df62__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsxs)("linearGradient", {
                        id: "95df62__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsxs)("radialGradient", {
                        id: "95df62__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("clipPath", {
                        id: "95df62__b",
                        children: /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("clipPath", {
                id: "95df62__a",
                children: /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M14.688 48.076c0-9.353 3.859-17.804 10.071-23.849l11.205 34.467H16.418c-1.122-3.335-1.73-6.906-1.73-10.619m19.558-7.602-6.195-19.059c5.551-4.152 12.442-6.61 19.907-6.61 5.793 0 11.24 1.48 15.984 4.084zm-16.183 22.22c5.411 11.046 16.765 18.652 29.895 18.652h.141l5.9-18.652zm34.324 18.36 10.821-34.21 16.31 11.79c-3.962 11.85-14.411 20.728-27.131 22.42m-1.279-47.892 29.504 21.326c.405-2.075.617-4.219.617-6.413 0-11.041-5.379-20.827-13.659-26.878z"
                })
            }),
            /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("g", {
                clipPath: "url(#95df62__a)",
                children: /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#95df62__b)",
                    children: [
                        /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#95df62__c)"
                        }),
                        /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("path", {
                            fill: "url(#95df62__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("path", {
                            fill: "url(#95df62__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $7oU0u$reactjsxruntime.jsx)("path", {
                            fill: "url(#95df62__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $85a476ce2ce9f646$var$ForwardRef = /*#__PURE__*/ (0, $7oU0u$react.forwardRef)($85a476ce2ce9f646$var$SvgComponent);
var $85a476ce2ce9f646$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($85a476ce2ce9f646$var$ForwardRef);
