// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_channel_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $dg0BA$reactjsxruntime = require("react/jsx-runtime");
var $dg0BA$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $467d61e510280ae8$export$2e2bcd8739ae039);




const $467d61e510280ae8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsxs)("linearGradient", {
                        id: "46a504__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsxs)("linearGradient", {
                        id: "46a504__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsxs)("linearGradient", {
                        id: "46a504__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsxs)("radialGradient", {
                        id: "46a504__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("clipPath", {
                        id: "46a504__b",
                        children: /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("clipPath", {
                id: "46a504__a",
                children: /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M33.62 28.094q.283 0 .558-.031l5.338 9.245c-2.654 2.108-4.504 5.183-5.02 8.692H23.822c-.771-1.766-2.533-3-4.584-3-2.761 0-5 2.239-5 5s2.239 5 5 5c2.05 0 3.813-1.234 4.584-3h10.675c.515 3.509 2.366 6.584 5.02 8.691l-5.339 9.246q-.275-.03-.559-.03c-2.761 0-5 2.238-5 5 0 2.76 2.239 5 5 5s5-2.24 5-5c0-1.113-.363-2.14-.977-2.97l5.337-9.243c1.553.615 3.247.953 5.02.953s3.466-.338 5.02-.953l5.337 9.243c-.614.83-.977 1.858-.977 2.97 0 2.76 2.239 5 5 5s5-2.24 5-5c0-2.762-2.238-5-5-5q-.283 0-.56.03l-5.337-9.245c2.653-2.108 4.504-5.183 5.02-8.692h10.674c.771 1.766 2.533 3 4.584 3 2.761 0 5-2.239 5-5s-2.239-5-5-5c-2.05 0-3.813 1.234-4.584 3H61.5c-.515-3.51-2.365-6.584-5.019-8.692l5.338-9.245q.276.03.56.03c2.76 0 5-2.238 5-5 0-2.76-2.24-5-5-5-2.762 0-5 2.24-5 5 0 1.112.362 2.14.976 2.97l-5.336 9.242c-1.554-.615-3.248-.953-5.021-.953s-3.467.338-5.02.953l-5.337-9.242c.614-.83.977-1.857.977-2.97 0-2.76-2.238-5-5-5s-5 2.24-5 5c0 2.762 2.239 5 5 5"
                })
            }),
            /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("g", {
                clipPath: "url(#46a504__a)",
                children: /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#46a504__b)",
                    children: [
                        /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#46a504__c)"
                        }),
                        /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("path", {
                            fill: "url(#46a504__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("path", {
                            fill: "url(#46a504__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $dg0BA$reactjsxruntime.jsx)("path", {
                            fill: "url(#46a504__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $467d61e510280ae8$var$ForwardRef = /*#__PURE__*/ (0, $dg0BA$react.forwardRef)($467d61e510280ae8$var$SvgComponent);
var $467d61e510280ae8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($467d61e510280ae8$var$ForwardRef);
