// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_megaphonePromote2_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $fnviM$reactjsxruntime = require("react/jsx-runtime");
var $fnviM$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $46a93a1ee10783da$export$2e2bcd8739ae039);




const $46a93a1ee10783da$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "110ab6__d",
                        x1: 66.474,
                        x2: 24.27,
                        y1: 10.009,
                        y2: 68.054,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "110ab6__e",
                        x1: 38.742,
                        x2: 46.565,
                        y1: -15.603,
                        y2: 37.515,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsxs)("linearGradient", {
                        id: "110ab6__f",
                        x1: -62.581,
                        x2: 84.471,
                        y1: 90.905,
                        y2: 60.075,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsxs)("radialGradient", {
                        id: "110ab6__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 106.979 -113.484 0 79.664 61.104)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("clipPath", {
                        id: "110ab6__b",
                        children: /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 9.602,
                            y: 12.393,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("clipPath", {
                id: "110ab6__a",
                children: /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m41.542 28.811 23.136-7.026.006-.006c3.45-1.05 6.93 1.53 6.93 5.136v26.316c0 3.606-3.48 6.186-6.93 5.136L41.548 51.33h-.024l6.264 14.55c1.344 3.108-.084 6.714-3.186 8.058-3.108 1.344-6.714-.084-8.058-3.186l-8.442-19.416H24.91c-6.216 0-11.262-5.046-11.262-11.262s5.04-11.262 11.262-11.262zM81.85 49.415l3.45 1.092c1.26.402 1.962 1.746 1.56 3.012-.324 1.02-1.266 1.674-2.286 1.674-.24 0-.486-.036-.726-.114l-3.45-1.092c-1.26-.402-1.962-1.746-1.56-3.012.402-1.26 1.746-1.962 3.012-1.56m-.725-18.888c.24 0 .486-.036.726-.114l3.45-1.092c1.266-.402 1.962-1.752 1.56-3.012-.402-1.266-1.746-1.962-3.012-1.56l-3.45 1.092c-1.266.402-1.962 1.752-1.56 3.012.324 1.02 1.266 1.674 2.286 1.674m3.612 11.79h-3.612c-1.326 0-2.4-1.074-2.4-2.4s1.074-2.4 2.4-2.4h3.612c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4"
                })
            }),
            /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("g", {
                clipPath: "url(#110ab6__a)",
                children: /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#110ab6__b)",
                    children: [
                        /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 9.602,
                            y: 12.393,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("ellipse", {
                            cx: 79.664,
                            cy: 61.104,
                            fill: "url(#110ab6__c)",
                            rx: 113.484,
                            ry: 106.979
                        }),
                        /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("path", {
                            fill: "url(#110ab6__d)",
                            d: "M2.528 35.599c-1.261-25.61 19.74-47.336 46.908-48.525C76.603-14.115 99.65 5.682 100.91 31.293c1.262 25.61-19.74 47.335-46.907 48.524C26.836 81.007 3.79 61.21 2.528 35.6"
                        }),
                        /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("path", {
                            fill: "url(#110ab6__e)",
                            d: "M29.46-38.88c-6.13-3.575-13.894-3.596-20.046-.056L-52.11-3.524c-6.026 3.468-6.036 11.655-.019 15.136L8.99 46.974c6.142 3.553 13.907 3.549 20.044-.013l60.992-35.395c5.986-3.474 5.997-11.614.019-15.101z"
                        }),
                        /*#__PURE__*/ (0, $fnviM$reactjsxruntime.jsx)("path", {
                            fill: "url(#110ab6__f)",
                            d: "M-29.584 47.832c12.682 0 24.169 4.838 32.485 12.643 16.634 15.638 30.424 15.651 47.058.04 8.316-7.845 19.818-12.697 32.513-12.697 25.377.014 45.934 19.34 45.934 43.197s-20.557 43.17-45.934 43.17c-12.724 0-24.225-4.865-32.542-12.711-16.62-15.584-30.395-15.57-47.029.067-8.316 7.819-19.803 12.644-32.485 12.644-25.362 0-45.92-19.326-45.92-43.17s20.558-43.183 45.92-43.183"
                        })
                    ]
                })
            })
        ]
    });
const $46a93a1ee10783da$var$ForwardRef = /*#__PURE__*/ (0, $fnviM$react.forwardRef)($46a93a1ee10783da$var$SvgComponent);
var $46a93a1ee10783da$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($46a93a1ee10783da$var$ForwardRef);
