// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_download_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lxEWK$jsxs, jsx as $lxEWK$jsx} from "react/jsx-runtime";
import {forwardRef as $lxEWK$forwardRef} from "react";





const $505922aa714c4995$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lxEWK$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lxEWK$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lxEWK$jsxs)("linearGradient", {
                        id: "8775a0__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.146,
                        y2: 114.99,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lxEWK$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $lxEWK$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lxEWK$jsxs)("linearGradient", {
                        id: "8775a0__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.451,
                        y2: 53.947,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lxEWK$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $lxEWK$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lxEWK$jsxs)("linearGradient", {
                        id: "8775a0__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lxEWK$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lxEWK$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $lxEWK$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $lxEWK$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lxEWK$jsxs)("radialGradient", {
                        id: "8775a0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lxEWK$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lxEWK$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lxEWK$jsx)("clipPath", {
                        id: "8775a0__b",
                        children: /*#__PURE__*/ (0, $lxEWK$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lxEWK$jsx)("clipPath", {
                id: "8775a0__a",
                children: /*#__PURE__*/ (0, $lxEWK$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M28.335 80.742c2.209 0 4 1.79 4 4v29.411c0 4.59 3.682 8.255 8.157 8.255h79.019c4.475 0 8.157-3.665 8.157-8.255V84.741c0-2.209 1.791-4 4-4s4 1.791 4 4v29.412c0 8.947-7.203 16.255-16.157 16.255h-79.02c-8.953 0-16.156-7.308-16.156-16.255V84.741c0-2.209 1.79-4 4-4m51.667-51.15c2.209 0 4 1.79 4 4v64.32l16.671-16.67c1.562-1.563 4.095-1.563 5.657 0 1.562 1.561 1.562 4.094 0 5.656l-23.5 23.5c-1.562 1.562-4.095 1.562-5.657 0l-23.5-23.5c-1.562-1.562-1.562-4.095 0-5.657s4.095-1.562 5.657 0l16.672 16.672V33.59c0-2.209 1.79-4 4-4"
                })
            }),
            /*#__PURE__*/ (0, $lxEWK$jsx)("g", {
                clipPath: "url(#8775a0__a)",
                children: /*#__PURE__*/ (0, $lxEWK$jsxs)("g", {
                    clipPath: "url(#8775a0__b)",
                    children: [
                        /*#__PURE__*/ (0, $lxEWK$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $lxEWK$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#8775a0__c)"
                        }),
                        /*#__PURE__*/ (0, $lxEWK$jsx)("path", {
                            fill: "url(#8775a0__d)",
                            d: "M-14.4 50.113C-16.968-5.19 25.782-52.104 81.085-54.672S183.302-14.49 185.87 40.813 145.688 143.03 90.385 145.598-11.832 105.416-14.4 50.113"
                        }),
                        /*#__PURE__*/ (0, $lxEWK$jsx)("path", {
                            fill: "url(#8775a0__e)",
                            d: "M40.418-110.715c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.677c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.782c12.185-7.502 12.207-25.08.037-32.61z"
                        }),
                        /*#__PURE__*/ (0, $lxEWK$jsx)("path", {
                            fill: "url(#8775a0__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $505922aa714c4995$var$ForwardRef = /*#__PURE__*/ (0, $lxEWK$forwardRef)($505922aa714c4995$var$SvgComponent);
var $505922aa714c4995$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($505922aa714c4995$var$ForwardRef);


export {$505922aa714c4995$export$2e2bcd8739ae039 as default};
