// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_document_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $cfJnW$jsxs, jsx as $cfJnW$jsx} from "react/jsx-runtime";
import {forwardRef as $cfJnW$forwardRef} from "react";





const $82a15d4e06960336$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cfJnW$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cfJnW$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cfJnW$jsxs)("linearGradient", {
                        id: "c8b9fd__d",
                        x1: 111.107,
                        x2: 30.526,
                        y1: 5.955,
                        y2: 110.43,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cfJnW$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $cfJnW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cfJnW$jsxs)("linearGradient", {
                        id: "c8b9fd__e",
                        x1: 62.011,
                        x2: 77.555,
                        y1: -42.141,
                        y2: 57.344,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cfJnW$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $cfJnW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cfJnW$jsxs)("linearGradient", {
                        id: "c8b9fd__f",
                        x1: -117.348,
                        x2: 144.197,
                        y1: 157.869,
                        y2: 106.178,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cfJnW$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cfJnW$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $cfJnW$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $cfJnW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cfJnW$jsxs)("radialGradient", {
                        id: "c8b9fd__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.277 118.181)scale(200.895)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cfJnW$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cfJnW$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cfJnW$jsx)("clipPath", {
                        id: "c8b9fd__b",
                        children: /*#__PURE__*/ (0, $cfJnW$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.43,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cfJnW$jsxs)("clipPath", {
                id: "c8b9fd__a",
                children: [
                    /*#__PURE__*/ (0, $cfJnW$jsx)("path", {
                        fill: "#fff",
                        d: "M80.001 61.91V27.79c-.323-.027-.648-.049-.975-.049H47.84c-6.66 0-12.06 5.4-12.06 12.06v80.4c0 6.661 5.4 12.06 12.06 12.06h64.32c6.661 0 12.06-5.399 12.06-12.06V72.936c0-.328-.022-.652-.048-.976H90.051c-5.541 0-10.05-4.508-10.05-10.05"
                    }),
                    /*#__PURE__*/ (0, $cfJnW$jsx)("path", {
                        fill: "#fff",
                        d: "M90.053 63.92h30.15l-32.16-32.16v30.15c0 1.108.902 2.01 2.01 2.01"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cfJnW$jsx)("g", {
                clipPath: "url(#c8b9fd__a)",
                children: /*#__PURE__*/ (0, $cfJnW$jsxs)("g", {
                    clipPath: "url(#c8b9fd__b)",
                    children: [
                        /*#__PURE__*/ (0, $cfJnW$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.43,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $cfJnW$jsx)("circle", {
                            cx: 134.458,
                            cy: 101.904,
                            r: 200.895,
                            fill: "url(#c8b9fd__c)"
                        }),
                        /*#__PURE__*/ (0, $cfJnW$jsx)("path", {
                            fill: "url(#c8b9fd__d)",
                            d: "M-2.092 54.01C-4.325 5.917 32.852-34.88 80.945-37.114S169.837-2.17 172.07 45.923s-34.944 88.891-83.038 91.124C40.94 139.281.142 102.104-2.092 54.01"
                        }),
                        /*#__PURE__*/ (0, $cfJnW$jsx)("path", {
                            fill: "url(#c8b9fd__e)",
                            d: "M45.579-85.852c-10.85-6.714-24.595-6.755-35.485-.106l-108.915 66.5c-10.667 6.513-10.685 21.886-.032 28.424L9.341 75.37c10.873 6.673 24.62 6.665 35.483-.023L152.798 8.88c10.596-6.524 10.615-21.811.032-28.36z"
                        }),
                        /*#__PURE__*/ (0, $cfJnW$jsx)("path", {
                            fill: "url(#c8b9fd__f)",
                            d: "M-58.935 76.982c22.45 0 42.784 9.086 57.507 23.743 29.445 29.366 53.857 29.391 83.303.076 14.722-14.733 35.082-23.844 57.557-23.844 44.923.025 81.314 36.318 81.314 81.119s-36.391 81.068-81.314 81.068c-22.525 0-42.885-9.136-57.608-23.869-29.42-29.265-53.807-29.24-83.252.126-14.723 14.682-35.058 23.743-57.507 23.743-44.897 0-81.289-36.293-81.289-81.068 0-44.776 36.392-81.094 81.29-81.094"
                        })
                    ]
                })
            })
        ]
    });
const $82a15d4e06960336$var$ForwardRef = /*#__PURE__*/ (0, $cfJnW$forwardRef)($82a15d4e06960336$var$SvgComponent);
var $82a15d4e06960336$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($82a15d4e06960336$var$ForwardRef);


export {$82a15d4e06960336$export$2e2bcd8739ae039 as default};
