// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_AIchat_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $3ztt0$jsxs, jsx as $3ztt0$jsx} from "react/jsx-runtime";
import {forwardRef as $3ztt0$forwardRef} from "react";





const $92754e927c130bfa$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3ztt0$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3ztt0$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3ztt0$jsxs)("linearGradient", {
                        id: "dd83b4__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3ztt0$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3ztt0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3ztt0$jsxs)("linearGradient", {
                        id: "dd83b4__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3ztt0$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3ztt0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3ztt0$jsxs)("linearGradient", {
                        id: "dd83b4__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3ztt0$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3ztt0$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3ztt0$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3ztt0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3ztt0$jsxs)("radialGradient", {
                        id: "dd83b4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3ztt0$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3ztt0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3ztt0$jsx)("clipPath", {
                        id: "dd83b4__b",
                        children: /*#__PURE__*/ (0, $3ztt0$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3ztt0$jsx)("clipPath", {
                id: "dd83b4__a",
                children: /*#__PURE__*/ (0, $3ztt0$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M61.817 36.974q.015-.105.038-.21l1.229-5.7q.045-.213.058-.427c.04-.71-.197-1.415-.666-1.946l-3.794-4.296c-.36-.407-.566-.865-.645-1.329-.29-1.709 1.156-3.49 3.064-3.043l5.463 1.282c.811.191 1.663-.046 2.274-.635l4.117-3.959c1.828-1.758 4.735-.006 4.19 2.525l-1.229 5.7-.014.072q-.008.043-.015.088-.023.142-.03.284c-.035.704.202 1.402.667 1.93l3.794 4.295c1.647 1.864.081 4.804-2.255 4.406q-.083-.014-.164-.033l-5.463-1.282c-.811-.192-1.663.046-2.274.634l-4.117 3.959c-1.777 1.71-4.574.101-4.228-2.315m10.526 1.808 4.643 1.09q.55.128 1.076.164v12.936c0 3.885-3.169 6.964-7.079 6.964H50.847L35.09 73.412c-1.534 1.322-3.92.23-3.906-1.788.027-3.618.014-8.369 0-11.688h-6.03c-3.91 0-7.079-3.079-7.079-6.964V30.03c0-3.886 3.17-6.965 7.08-6.965h28.86c.088 1.372.602 2.768 1.67 3.977l3.355 3.8-1.095 5.08c-.658 3.058.81 5.788 3.052 7.138 2.269 1.367 5.468 1.375 7.821-.889h.001zM53.98 50.273l-4.015-.904c-.532-.12-1.083.03-1.483.396l-3.025 2.79c-1.192 1.1-3.087.004-2.73-1.575l.904-4.015c.12-.532-.03-1.083-.396-1.483l-2.79-3.025c-1.1-1.192-.004-3.087 1.575-2.73l4.015.904c.532.12 1.083-.03 1.483-.396l3.025-2.79c1.192-1.1 3.087-.004 2.73 1.575l-.904 4.015c-.12.532.03 1.083.396 1.483l2.79 3.025c1.1 1.192.004 3.087-1.575 2.73"
                })
            }),
            /*#__PURE__*/ (0, $3ztt0$jsx)("g", {
                clipPath: "url(#dd83b4__a)",
                children: /*#__PURE__*/ (0, $3ztt0$jsxs)("g", {
                    clipPath: "url(#dd83b4__b)",
                    children: [
                        /*#__PURE__*/ (0, $3ztt0$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3ztt0$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#dd83b4__c)"
                        }),
                        /*#__PURE__*/ (0, $3ztt0$jsx)("path", {
                            fill: "url(#dd83b4__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $3ztt0$jsx)("path", {
                            fill: "url(#dd83b4__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $3ztt0$jsx)("path", {
                            fill: "url(#dd83b4__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $92754e927c130bfa$var$ForwardRef = /*#__PURE__*/ (0, $3ztt0$forwardRef)($92754e927c130bfa$var$SvgComponent);
var $92754e927c130bfa$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($92754e927c130bfa$var$ForwardRef);


export {$92754e927c130bfa$export$2e2bcd8739ae039 as default};
