// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_chartAreaStack_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $ipHu2$reactjsxruntime = require("react/jsx-runtime");
var $ipHu2$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1bf2cd035cd19eb1$export$2e2bcd8739ae039);




const $1bf2cd035cd19eb1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a071be__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a071be__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a071be__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a071be__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("clipPath", {
                        id: "a071be__b",
                        children: /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("clipPath", {
                id: "a071be__a",
                children: /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("path", {
                    fill: "#8F8F8F",
                    fillRule: "evenodd",
                    d: "M71.905 23.235c2.151-3.895 8.094-2.38 8.094 2.083v47.806c0 3.661-2.972 6.624-6.631 6.624H22.63c-3.66 0-6.631-2.963-6.631-6.624V47.852c0-1.188.49-2.323 1.356-3.138l9.86-9.285c1.785-1.682 4.615-1.54 6.222.315l7.126 8.224c.138.159.392.14.506-.035l9.32-14.392c1.59-2.455 5.111-2.65 6.964-.388l6.44 7.867c.143.174.416.147.523-.047zm3.644 1.786c-.026.01-.08.034-.143.148L67.82 38.907l-1.751-.967 1.75.967c-1.475 2.673-5.185 3.008-7.118.647l-6.441-7.867c-.136-.166-.396-.15-.511.028l-9.32 14.392c-1.563 2.413-5.005 2.652-6.887.48l-7.126-8.223c-.117-.136-.326-.147-.457-.023l-9.86 9.285-1.336-1.419 1.336 1.42c-.063.058-.098.14-.098.225v14.527l9.075-5.837c1.77-1.14 4.064-1.047 5.737.229l7.829 5.971c.631.482 1.528.388 2.046-.214l9.924-11.525c1.896-2.203 5.265-2.33 7.322-.276l7.423 7.411 6.643-3.18v-29.64c0-.13-.035-.176-.052-.196-.03-.036-.09-.082-.184-.106-.093-.024-.17-.013-.214.005M76 59.393l-6.192 2.964c-.764.366-1.677.21-2.277-.389l-8.424-8.41c-.411-.41-1.085-.386-1.464.055l-9.924 11.525c-1.899 2.206-5.188 2.55-7.503.785l-7.829-5.972c-.334-.255-.793-.273-1.147-.045L20 67.136v5.988c0 1.446 1.175 2.624 2.631 2.624h50.737c1.456 0 2.631-1.178 2.631-2.624z"
                })
            }),
            /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("g", {
                clipPath: "url(#a071be__a)",
                children: /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#a071be__b)",
                    children: [
                        /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#a071be__c)"
                        }),
                        /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("path", {
                            fill: "url(#a071be__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("path", {
                            fill: "url(#a071be__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $ipHu2$reactjsxruntime.jsx)("path", {
                            fill: "url(#a071be__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $1bf2cd035cd19eb1$var$ForwardRef = /*#__PURE__*/ (0, $ipHu2$react.forwardRef)($1bf2cd035cd19eb1$var$SvgComponent);
var $1bf2cd035cd19eb1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1bf2cd035cd19eb1$var$ForwardRef);
