// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_volume_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $i1obI$reactjsxruntime = require("react/jsx-runtime");
var $i1obI$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $f0d051f0195c1869$export$2e2bcd8739ae039);




const $f0d051f0195c1869$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7719da__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7719da__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7719da__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsxs)("radialGradient", {
                        id: "7719da__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("clipPath", {
                        id: "7719da__b",
                        children: /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("clipPath", {
                id: "7719da__a",
                children: /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M14.14 16.027h-4.03c-1.105 0-2 .895-2 2v12.011c0 1.105.895 2 2 2h4.03zm2.5-1.505v19.007l4.903 4.811c.949.932 2.55.26 2.55-1.07V10.731c0-1.333-1.609-2.003-2.555-1.065zm10.742 2.715c.415-.552 1.199-.663 1.75-.247 2.116 1.591 3.487 4.126 3.487 6.982 0 2.878-1.394 5.432-3.538 7.02-.555.412-1.338.295-1.749-.26-.41-.554-.294-1.337.26-1.748 1.536-1.138 2.527-2.96 2.527-5.012 0-2.037-.975-3.845-2.49-4.985-.551-.415-.662-1.199-.247-1.75m6.753-4.812c-.552-.415-1.335-.304-1.75.247-.416.552-.305 1.335.247 1.75 2.894 2.179 4.762 5.64 4.762 9.537 0 3.93-1.898 7.414-4.834 9.59-.554.41-.67 1.193-.26 1.748s1.194.671 1.749.26c3.544-2.626 5.845-6.844 5.845-11.598 0-4.716-2.264-8.904-5.76-11.534"
                })
            }),
            /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("g", {
                clipPath: "url(#7719da__a)",
                children: /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#7719da__b)",
                    children: [
                        /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#7719da__c)"
                        }),
                        /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("path", {
                            fill: "url(#7719da__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("path", {
                            fill: "url(#7719da__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $i1obI$reactjsxruntime.jsx)("path", {
                            fill: "url(#7719da__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $f0d051f0195c1869$var$ForwardRef = /*#__PURE__*/ (0, $i1obI$react.forwardRef)($f0d051f0195c1869$var$SvgComponent);
var $f0d051f0195c1869$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($f0d051f0195c1869$var$ForwardRef);
