// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_brand_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $fALcr$reactjsxruntime = require("react/jsx-runtime");
var $fALcr$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $29cc6e6806a33d2c$export$2e2bcd8739ae039);




const $29cc6e6806a33d2c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ee893b__d",
                        x1: 108.785,
                        x2: 38.447,
                        y1: 14.029,
                        y2: 110.771,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ee893b__e",
                        x1: 62.565,
                        x2: 75.605,
                        y1: -28.658,
                        y2: 59.871,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ee893b__f",
                        x1: -106.301,
                        x2: 138.787,
                        y1: 148.856,
                        y2: 97.472,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ee893b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 178.298 -189.141 0 130.774 99.185)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("clipPath", {
                        id: "ee893b__b",
                        children: /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 18,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("clipPath", {
                id: "ee893b__a",
                children: /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M45.75 26.06h68.5c6.63 0 12 5.37 12 12v49.63c0 25.551-20.7 46.251-46.25 46.251s-46.25-20.71-46.25-46.25V38.06c0-6.63 5.37-12 12-12m4.08 65.75c.629.37 1.317.54 1.995.54 1.388-.001 2.726-.721 3.465-2l13.96-24.18c1.11-1.91.45-4.36-1.46-5.46-1.91-1.11-4.36-.45-5.46 1.46L48.37 86.35c-1.11 1.91-.45 4.36 1.46 5.46m52.52.54c2.89 0 5.48-1.5 6.93-4 1.45-2.51 1.45-5.5 0-8L96.71 58.58c-1.45-2.5-4.04-4-6.93-4s-5.48 1.5-6.93 4L70.28 80.35c-1.45 2.5-1.45 5.5 0 8s4.04 4 6.93 4zM89.78 62.57l12.571 21.78h-25.14z"
                })
            }),
            /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ee893b__a)",
                children: /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ee893b__b)",
                    children: [
                        /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 14,
                            y: 18,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("ellipse", {
                            cx: 130.774,
                            cy: 99.185,
                            fill: "url(#ee893b__c)",
                            rx: 189.141,
                            ry: 178.298
                        }),
                        /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("path", {
                            fill: "url(#ee893b__d)",
                            d: "M2.21 56.679C.107 13.995 35.11-22.214 80.389-24.196s83.69 31.013 85.792 73.697-32.899 78.893-78.178 80.875c-45.28 1.982-83.69-31.014-85.793-73.697"
                        }),
                        /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("path", {
                            fill: "url(#ee893b__e)",
                            d: "M47.095-67.453c-10.215-5.958-23.156-5.995-33.409-.094L-88.856-8.527c-10.043 5.78-10.06 19.424-.03 25.227L12.977 75.636c10.237 5.922 23.179 5.915 33.407-.021L148.04 16.623c9.976-5.79 9.994-19.357.031-25.169z"
                        }),
                        /*#__PURE__*/ (0, $fALcr$reactjsxruntime.jsx)("path", {
                            fill: "url(#ee893b__f)",
                            d: "M-51.305 77.067c21.136 0 40.28 8.064 54.142 21.073 27.723 26.062 50.706 26.084 78.429.067 13.861-13.076 33.03-21.162 54.189-21.162 42.295.022 76.557 32.233 76.557 71.994s-34.262 71.95-76.557 71.95c-21.207 0-40.375-8.108-54.237-21.184-27.699-25.973-50.658-25.951-78.38.111-13.862 13.031-33.007 21.073-54.143 21.073-42.27 0-76.533-32.21-76.533-71.95 0-39.739 34.262-71.972 76.533-71.972"
                        })
                    ]
                })
            })
        ]
    });
const $29cc6e6806a33d2c$var$ForwardRef = /*#__PURE__*/ (0, $fALcr$react.forwardRef)($29cc6e6806a33d2c$var$SvgComponent);
var $29cc6e6806a33d2c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($29cc6e6806a33d2c$var$ForwardRef);
