// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_buildTable_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $bqEvT$reactjsxruntime = require("react/jsx-runtime");
var $bqEvT$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $927b6a08cc10ebdd$export$2e2bcd8739ae039);




const $927b6a08cc10ebdd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsxs)("linearGradient", {
                        id: "387f1f__d",
                        x1: 70.776,
                        x2: 26.447,
                        y1: 11.417,
                        y2: 72.386,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsxs)("linearGradient", {
                        id: "387f1f__e",
                        x1: 41.644,
                        x2: 49.862,
                        y1: -15.484,
                        y2: 40.309,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsxs)("linearGradient", {
                        id: "387f1f__f",
                        x1: -64.776,
                        x2: 89.684,
                        y1: 96.388,
                        y2: 64.005,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsxs)("radialGradient", {
                        id: "387f1f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 112.368 -119.201 0 84.631 65.085)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("clipPath", {
                        id: "387f1f__b",
                        children: /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("rect", {
                            width: 82.56,
                            height: 77.827,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("clipPath", {
                id: "387f1f__a",
                children: /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M74.38 20.34c5.305 0 9.6 4.296 9.6 9.6v2.454H11.97V29.94c0-5.304 4.296-9.6 9.6-9.6zm9.601 31.536c-1.548-.396-3.168-.606-4.836-.606h-.006c-10.764 0-19.518 8.754-19.518 19.518 0 1.662.21 3.282.606 4.824H28.8V37.194h55.182zM11.97 66.018c0 5.304 4.296 9.6 9.6 9.6h2.43V37.2h-12.03zm67.177-9.942c1.692 0 3.318.288 4.836.816h.006c5.754 2.004 9.882 7.47 9.882 13.902 0 8.13-6.588 14.718-14.718 14.718-6.438 0-11.91-4.134-13.908-9.894-.528-1.512-.816-3.132-.816-4.824 0-8.13 6.588-14.718 14.718-14.718m2.406 17.088h4.728c1.326 0 2.4-1.074 2.4-2.4s-1.074-2.4-2.4-2.4h-4.728v-4.728c0-1.326-1.074-2.4-2.4-2.4s-2.4 1.074-2.4 2.4v4.728h-4.728c-1.326 0-2.4 1.074-2.4 2.4s1.074 2.4 2.4 2.4h4.728v4.728c0 1.326 1.074 2.4 2.4 2.4s2.4-1.074 2.4-2.4z"
                })
            }),
            /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("g", {
                clipPath: "url(#387f1f__a)",
                children: /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#387f1f__b)",
                    children: [
                        /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("rect", {
                            width: 82.56,
                            height: 77.827,
                            x: 11.039,
                            y: 13.92,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("ellipse", {
                            cx: 84.631,
                            cy: 65.085,
                            fill: "url(#387f1f__c)",
                            rx: 119.201,
                            ry: 112.368
                        }),
                        /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("path", {
                            fill: "url(#387f1f__d)",
                            d: "M3.61 38.296c-1.326-26.9 20.734-49.72 49.27-50.969s52.743 19.546 54.068 46.446-20.734 49.72-49.27 50.97C29.142 85.99 4.935 65.196 3.61 38.295"
                        }),
                        /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("path", {
                            fill: "url(#387f1f__e)",
                            d: "M31.894-39.933c-6.437-3.755-14.594-3.778-21.055-.06L-53.786-2.797c-6.329 3.643-6.34 12.242-.019 15.899l64.198 37.143c6.451 3.732 14.608 3.728 21.054-.013l64.065-37.178c6.288-3.65 6.299-12.2.02-15.863z"
                        }),
                        /*#__PURE__*/ (0, $bqEvT$reactjsxruntime.jsx)("path", {
                            fill: "url(#387f1f__f)",
                            d: "M-30.117 51.145c13.32 0 25.386 5.082 34.122 13.28 17.471 16.426 31.956 16.44 49.428.043 8.735-8.241 20.816-13.337 34.151-13.337 26.655.014 48.248 20.314 48.248 45.373s-21.593 45.344-48.248 45.344c-13.365 0-25.446-5.11-34.181-13.351-17.457-16.369-31.927-16.355-49.398.071-8.736 8.212-20.801 13.28-34.122 13.28-26.64 0-48.233-20.3-48.233-45.345s21.593-45.358 48.233-45.358"
                        })
                    ]
                })
            })
        ]
    });
const $927b6a08cc10ebdd$var$ForwardRef = /*#__PURE__*/ (0, $bqEvT$react.forwardRef)($927b6a08cc10ebdd$var$SvgComponent);
var $927b6a08cc10ebdd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($927b6a08cc10ebdd$var$ForwardRef);
