// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_prompt_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2idYL$jsxs, jsx as $2idYL$jsx} from "react/jsx-runtime";
import {forwardRef as $2idYL$forwardRef} from "react";





const $6150c9c9bc5bd333$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2idYL$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2idYL$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2idYL$jsxs)("linearGradient", {
                        id: "ca9a0e__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2idYL$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2idYL$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2idYL$jsxs)("linearGradient", {
                        id: "ca9a0e__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2idYL$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2idYL$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2idYL$jsxs)("linearGradient", {
                        id: "ca9a0e__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2idYL$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2idYL$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2idYL$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2idYL$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2idYL$jsxs)("radialGradient", {
                        id: "ca9a0e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2idYL$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2idYL$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2idYL$jsx)("clipPath", {
                        id: "ca9a0e__b",
                        children: /*#__PURE__*/ (0, $2idYL$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2idYL$jsx)("clipPath", {
                id: "ca9a0e__a",
                children: /*#__PURE__*/ (0, $2idYL$jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M44 42c-8.837 0-16 7.163-16 16v44c0 8.837 7.163 16 16 16h72c8.837 0 16-7.163 16-16V58c0-8.837-7.163-16-16-16zm-3.022 11.395c1.714-1.393 4.233-1.132 5.627.582l19.5 24c1.194 1.47 1.194 3.576 0 5.045l-19.5 24c-1.393 1.715-3.913 1.975-5.627.582-1.715-1.393-1.975-3.912-.582-5.626L57.846 80.5l-17.45-21.478c-1.393-1.714-1.133-4.233.582-5.627M66 101c0-2.21 1.791-4 4-4h41.5c2.209 0 4 1.79 4 4s-1.791 4-4 4H70c-2.209 0-4-1.791-4-4"
                })
            }),
            /*#__PURE__*/ (0, $2idYL$jsx)("g", {
                clipPath: "url(#ca9a0e__a)",
                children: /*#__PURE__*/ (0, $2idYL$jsxs)("g", {
                    clipPath: "url(#ca9a0e__b)",
                    children: [
                        /*#__PURE__*/ (0, $2idYL$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2idYL$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#ca9a0e__c)"
                        }),
                        /*#__PURE__*/ (0, $2idYL$jsx)("path", {
                            fill: "url(#ca9a0e__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $2idYL$jsx)("path", {
                            fill: "url(#ca9a0e__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $2idYL$jsx)("path", {
                            fill: "url(#ca9a0e__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $6150c9c9bc5bd333$var$ForwardRef = /*#__PURE__*/ (0, $2idYL$forwardRef)($6150c9c9bc5bd333$var$SvgComponent);
var $6150c9c9bc5bd333$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($6150c9c9bc5bd333$var$ForwardRef);


export {$6150c9c9bc5bd333$export$2e2bcd8739ae039 as default};
