// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_mention_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $i7Mj7$jsxs, jsx as $i7Mj7$jsx} from "react/jsx-runtime";
import {forwardRef as $i7Mj7$forwardRef} from "react";





const $2966a455393e017d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $i7Mj7$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $i7Mj7$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $i7Mj7$jsxs)("linearGradient", {
                        id: "174b5b__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i7Mj7$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $i7Mj7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i7Mj7$jsxs)("linearGradient", {
                        id: "174b5b__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i7Mj7$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $i7Mj7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i7Mj7$jsxs)("linearGradient", {
                        id: "174b5b__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i7Mj7$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $i7Mj7$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $i7Mj7$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $i7Mj7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i7Mj7$jsxs)("radialGradient", {
                        id: "174b5b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i7Mj7$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $i7Mj7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i7Mj7$jsx)("clipPath", {
                        id: "174b5b__b",
                        children: /*#__PURE__*/ (0, $i7Mj7$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $i7Mj7$jsx)("clipPath", {
                id: "174b5b__a",
                children: /*#__PURE__*/ (0, $i7Mj7$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M38.089 14.782c9.7-3.18 20.098-2.114 26.84.927 13.817 6.234 18.38 19.622 16.359 31.11-2.007 11.406-10.7 21.583-23.79 20.677-12.196-.844-18.633-5.455-21.228-11.381-2.51-5.733-1.118-12.05.921-15.817 5.274-9.74 14.283-11.233 19.88-11.055 3.415.109 5.5 3.2 5.223 6.203L59.711 63.54c9.385-.334 15.974-7.962 17.637-17.413 1.761-10.004-2.19-21.413-14.065-26.77-5.83-2.631-15.197-3.643-23.948-.773-8.631 2.83-16.676 9.433-20.144 22.162-2.757 10.12.47 19.802 6.65 26.96 6.21 7.194 15.231 11.635 23.707 11.39 1.104-.031 2.025.838 2.057 1.942s-.838 2.025-1.942 2.057c-9.881.285-19.996-4.836-26.85-12.774-6.884-7.974-10.65-18.995-7.481-30.626 3.846-14.115 12.938-21.692 22.757-24.912m17.625 48.537 2.597-28.239c.103-1.117-.644-1.816-1.366-1.839-4.838-.154-11.983 1.107-16.236 8.962-1.638 3.025-2.652 8.02-.775 12.307 1.68 3.837 5.952 7.694 15.78 8.809"
                })
            }),
            /*#__PURE__*/ (0, $i7Mj7$jsx)("g", {
                clipPath: "url(#174b5b__a)",
                children: /*#__PURE__*/ (0, $i7Mj7$jsxs)("g", {
                    clipPath: "url(#174b5b__b)",
                    children: [
                        /*#__PURE__*/ (0, $i7Mj7$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $i7Mj7$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#174b5b__c)"
                        }),
                        /*#__PURE__*/ (0, $i7Mj7$jsx)("path", {
                            fill: "url(#174b5b__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $i7Mj7$jsx)("path", {
                            fill: "url(#174b5b__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $i7Mj7$jsx)("path", {
                            fill: "url(#174b5b__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $2966a455393e017d$var$ForwardRef = /*#__PURE__*/ (0, $i7Mj7$forwardRef)($2966a455393e017d$var$SvgComponent);
var $2966a455393e017d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2966a455393e017d$var$ForwardRef);


export {$2966a455393e017d$export$2e2bcd8739ae039 as default};
