// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_piggyBank_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $cX0eq$jsxs, jsx as $cX0eq$jsx} from "react/jsx-runtime";
import {forwardRef as $cX0eq$forwardRef} from "react";





const $a99831aa7858da66$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cX0eq$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cX0eq$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cX0eq$jsxs)("linearGradient", {
                        id: "26d391__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cX0eq$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $cX0eq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cX0eq$jsxs)("linearGradient", {
                        id: "26d391__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cX0eq$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $cX0eq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cX0eq$jsxs)("linearGradient", {
                        id: "26d391__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cX0eq$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cX0eq$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $cX0eq$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $cX0eq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cX0eq$jsxs)("radialGradient", {
                        id: "26d391__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cX0eq$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cX0eq$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cX0eq$jsx)("clipPath", {
                        id: "26d391__b",
                        children: /*#__PURE__*/ (0, $cX0eq$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cX0eq$jsx)("clipPath", {
                id: "26d391__a",
                children: /*#__PURE__*/ (0, $cX0eq$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M32.845 35.216c4.84-2.564 8.06-7.11 8.06-12.29q0-.638-.065-1.261c.668-.09 1.432-.364 2.113-.932.974-.814 1.646-2.121 1.73-3.985.032-.69-.502-1.274-1.192-1.305-.69-.032-1.274.502-1.305 1.192-.059 1.288-.49 1.89-.837 2.18-.363.303-.78.387-1.018.383-1.949-6.2-8.644-10.77-16.603-10.77-3.28 0-6.344.775-8.951 2.121-1.55-1.661-3.976-2.142-5.856-2.12-.696.009-1.058.764-.709 1.366l2.26 3.905.017-.012c-1.182 1.205-2.137 2.574-2.81 4.061v-.008l-3.37 1.963c-.614.358-.993 1.016-.993 1.728v2.989c0 .712.379 1.37.994 1.728l3.368 1.96v-.006c1.337 2.952 3.785 5.44 6.911 7.102q-.012-.003-.02-.002l1.243 3.386c.217.59.78.983 1.408.983h1.727c.628 0 1.19-.392 1.407-.981l.5-1.36-.044-.015q1.424.207 2.918.208 1.46-.001 2.855-.199l-.02.006.5 1.36c.218.59.78.981 1.408.981h1.727c.629 0 1.191-.392 1.408-.983zM26.21 13.818c-1.543-.344-2.82-.358-4.452 0-.675.149-1.341-.278-1.49-.952-.148-.674.279-1.341.953-1.49 1.992-.437 3.637-.42 5.532.002.674.15 1.099.817.948 1.491-.15.674-.817 1.099-1.491.949M15.36 17.69c0 .967-.783 1.75-1.75 1.75-.966 0-1.75-.783-1.75-1.75 0-.966.784-1.75 1.75-1.75.967 0 1.75.784 1.75 1.75"
                })
            }),
            /*#__PURE__*/ (0, $cX0eq$jsx)("g", {
                clipPath: "url(#26d391__a)",
                children: /*#__PURE__*/ (0, $cX0eq$jsxs)("g", {
                    clipPath: "url(#26d391__b)",
                    children: [
                        /*#__PURE__*/ (0, $cX0eq$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $cX0eq$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#26d391__c)"
                        }),
                        /*#__PURE__*/ (0, $cX0eq$jsx)("path", {
                            fill: "url(#26d391__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $cX0eq$jsx)("path", {
                            fill: "url(#26d391__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $cX0eq$jsx)("path", {
                            fill: "url(#26d391__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $a99831aa7858da66$var$ForwardRef = /*#__PURE__*/ (0, $cX0eq$forwardRef)($a99831aa7858da66$var$SvgComponent);
var $a99831aa7858da66$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a99831aa7858da66$var$ForwardRef);


export {$a99831aa7858da66$export$2e2bcd8739ae039 as default};
