// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_shoppingBag_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $d81AU$reactjsxruntime = require("react/jsx-runtime");
var $d81AU$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9415a6263dafcc19$export$2e2bcd8739ae039);




const $9415a6263dafcc19$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4b6211__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4b6211__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4b6211__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4b6211__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("clipPath", {
                        id: "4b6211__b",
                        children: /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("clipPath", {
                id: "4b6211__a",
                children: /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M42 22.5c0-3.038 2.463-5.5 5.5-5.5s5.5 2.462 5.5 5.5V28H42zM38 28h4v8c0 1.105-.895 2-2 2s-2-.895-2-2zm0 0H27.81c-1.027 0-1.887.779-1.99 1.801l-3.94 39.403C21.41 73.914 25.108 78 29.84 78h35.32c4.734 0 8.432-4.087 7.961-8.796l-3.94-39.403C69.079 28.779 68.218 28 67.19 28H57v-5.5c0-5.247-4.254-9.5-9.5-9.5S38 17.253 38 22.5zm19 0v8c0 1.105-.895 2-2 2s-2-.895-2-2v-8z"
                })
            }),
            /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("g", {
                clipPath: "url(#4b6211__a)",
                children: /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#4b6211__b)",
                    children: [
                        /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#4b6211__c)"
                        }),
                        /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("path", {
                            fill: "url(#4b6211__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("path", {
                            fill: "url(#4b6211__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $d81AU$reactjsxruntime.jsx)("path", {
                            fill: "url(#4b6211__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $9415a6263dafcc19$var$ForwardRef = /*#__PURE__*/ (0, $d81AU$react.forwardRef)($9415a6263dafcc19$var$SvgComponent);
var $9415a6263dafcc19$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9415a6263dafcc19$var$ForwardRef);
