// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fileShared_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $eIS25$jsxs, jsx as $eIS25$jsx} from "react/jsx-runtime";
import {forwardRef as $eIS25$forwardRef} from "react";





const $05530c6509dba824$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eIS25$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $eIS25$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $eIS25$jsxs)("linearGradient", {
                        id: "954896__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eIS25$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $eIS25$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eIS25$jsxs)("linearGradient", {
                        id: "954896__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eIS25$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $eIS25$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eIS25$jsxs)("linearGradient", {
                        id: "954896__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eIS25$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $eIS25$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $eIS25$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $eIS25$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eIS25$jsxs)("radialGradient", {
                        id: "954896__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eIS25$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $eIS25$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eIS25$jsx)("clipPath", {
                        id: "954896__b",
                        children: /*#__PURE__*/ (0, $eIS25$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $eIS25$jsx)("clipPath", {
                id: "954896__a",
                children: /*#__PURE__*/ (0, $eIS25$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "m26.454 8.578 9.892 9.891c.675.675 1.054 1.59 1.054 2.546v14.108c0 1.989-1.612 3.6-3.6 3.6H14.6c-1.988 0-3.6-1.611-3.6-3.6v-24c0-1.988 1.612-3.6 3.6-3.6h9.309c.955 0 1.87.38 2.545 1.055M21.302 23.16c0-.213.173-.387.387-.387.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25c-1.595 0-2.887 1.293-2.887 2.887 0 .814.338 1.55.879 2.074.496.48 1.287.467 1.767-.029s.468-1.287-.028-1.767c-.074-.072-.118-.169-.118-.278m4.213-1.71c-1.503 0-2.72 1.218-2.72 2.72s1.217 2.721 2.72 2.721c1.502 0 2.72-1.218 2.72-2.72s-1.218-2.721-2.72-2.721m-4.426 7.406c.623-.298.887-1.044.59-1.667s-1.044-.887-1.667-.59c-1.298.62-2.245 1.864-2.494 3.333-.116.68.342 1.326 1.023 1.442s1.326-.343 1.442-1.023c.114-.675.549-1.23 1.106-1.495m4.426-1.524c-2.074 0-3.966 1.33-4.438 3.287-.302 1.254.76 2.154 1.763 2.154h5.35c1.002 0 2.065-.9 1.762-2.155-.471-1.955-2.363-3.286-4.437-3.286"
                })
            }),
            /*#__PURE__*/ (0, $eIS25$jsx)("g", {
                clipPath: "url(#954896__a)",
                children: /*#__PURE__*/ (0, $eIS25$jsxs)("g", {
                    clipPath: "url(#954896__b)",
                    children: [
                        /*#__PURE__*/ (0, $eIS25$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $eIS25$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#954896__c)"
                        }),
                        /*#__PURE__*/ (0, $eIS25$jsx)("path", {
                            fill: "url(#954896__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $eIS25$jsx)("path", {
                            fill: "url(#954896__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $eIS25$jsx)("path", {
                            fill: "url(#954896__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $05530c6509dba824$var$ForwardRef = /*#__PURE__*/ (0, $eIS25$forwardRef)($05530c6509dba824$var$SvgComponent);
var $05530c6509dba824$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($05530c6509dba824$var$ForwardRef);


export {$05530c6509dba824$export$2e2bcd8739ae039 as default};
