// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_confetti_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $egTD9$reactjsxruntime = require("react/jsx-runtime");
var $egTD9$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $c397f8fe0005fe41$export$2e2bcd8739ae039);




const $c397f8fe0005fe41$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2930ac__d",
                        x1: 65.847,
                        x2: 20.038,
                        y1: 8.215,
                        y2: 71.154,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2930ac__e",
                        x1: 35.785,
                        x2: 44.283,
                        y1: -19.576,
                        y2: 38.059,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2930ac__f",
                        x1: -74.043,
                        x2: 85.371,
                        y1: 95.995,
                        y2: 62.609,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2930ac__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 116.083 -123.013 0 80.146 63.657)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("clipPath", {
                        id: "2930ac__b",
                        children: /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("rect", {
                            width: 85.2,
                            height: 80.4,
                            x: 4.199,
                            y: 10.801,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("clipPath", {
                id: "2930ac__a",
                children: /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M65.951 19.296c1.716-1.716 1.716-4.488 0-6.204s-4.488-1.716-6.204 0-1.716 4.488 0 6.204 4.488 1.716 6.204 0M38.878 43.272c-1.038.828-1.206 2.34-.378 3.372.474.594 1.17.9 1.878.9.528 0 1.056-.174 1.494-.528 11.532-9.21 11.004-18.876 10.98-19.284-.09-1.314-1.218-2.292-2.532-2.22-1.314.078-2.322 1.212-2.256 2.526.012.312.258 7.686-9.186 15.228zM59.28 61.194c-.618-1.17-.168-2.622 1.002-3.24h.006c1.17-.618 2.622-.168 3.24 1.002 3.438 6.516 2.334 9.708.804 11.238-.498.498-1.17.948-2.07 1.212l-41.55 14.76c-.804.288-1.626.426-2.436.426-1.866 0-3.684-.738-5.064-2.118-1.98-1.98-2.628-4.854-1.692-7.5l14.592-41.118q.125-.468.33-.9c.27-.672.642-1.2 1.05-1.608 1.542-1.542 4.764-2.64 11.358.87 1.17.624 1.614 2.076.99 3.246s-2.076 1.614-3.246.99c-3.714-1.974-5.418-1.854-5.712-1.704v.012l-.108.306c-.24 1.824 2.52 8.922 11.718 18.12 6.972 6.972 12.732 10.242 15.924 11.304 1.236.252 2.112.312 2.556.252.114-.33.174-2.01-1.692-5.55m-12.786-7.536c-.618 0-1.23-.234-1.698-.702-.936-.942-.936-2.46 0-3.396l9.366-9.366c.942-.936 2.46-.936 3.396 0 .936.942.936 2.46 0 3.396l-9.366 9.366c-.468.468-1.08.702-1.698.702m4.23 2.22c9.21-11.532 18.876-11.004 19.284-10.98l-.006.006c1.326.09 2.322 1.236 2.232 2.556-.09 1.314-1.224 2.316-2.538 2.232-.354-.012-7.704-.222-15.222 9.186-.474.594-1.176.9-1.878.9-.522 0-1.05-.174-1.494-.528-1.038-.828-1.206-2.334-.378-3.372m24.3-.576c1.716-1.716 4.488-1.716 6.204 0s1.716 4.488 0 6.204-4.489 1.716-6.205 0-1.716-4.488 0-6.204M33.81 16.917l-1.153.843c-1.07.783-1.302 2.285-.519 3.354l4.688 6.406c.783 1.07 2.285 1.302 3.355.52l1.152-.844c1.07-.783 1.302-2.285.52-3.354l-4.69-6.406c-.782-1.07-2.284-1.302-3.353-.52M64.336 32.91l7.888-.894c1.317-.149 2.505.798 2.654 2.115l.161 1.42c.15 1.316-.798 2.505-2.115 2.654l-7.887.893c-1.317.15-2.506-.798-2.655-2.115l-.16-1.419c-.15-1.317.797-2.505 2.114-2.654"
                })
            }),
            /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("g", {
                clipPath: "url(#2930ac__a)",
                children: /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#2930ac__b)",
                    children: [
                        /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("rect", {
                            width: 85.2,
                            height: 80.4,
                            x: 4.199,
                            y: 10.801,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("ellipse", {
                            cx: 80.146,
                            cy: 63.657,
                            fill: "url(#2930ac__c)",
                            rx: 123.013,
                            ry: 116.083
                        }),
                        /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("path", {
                            fill: "url(#2930ac__d)",
                            d: "M-3.467 35.982C-4.835 8.192 17.93-15.382 47.379-16.672s54.43 20.191 55.797 47.981c1.368 27.79-21.397 51.364-50.845 52.654-29.45 1.29-54.43-20.191-55.798-47.98"
                        }),
                        /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("path", {
                            fill: "url(#2930ac__e)",
                            d: "M25.724-44.834c-6.644-3.879-15.06-3.903-21.729-.061L-62.696-6.469c-6.532 3.763-6.543 12.646-.02 16.424l66.25 38.37c6.659 3.856 15.076 3.852 21.728-.013L91.376 9.905c6.489-3.77 6.5-12.603.02-16.387z"
                        }),
                        /*#__PURE__*/ (0, $egTD9$reactjsxruntime.jsx)("path", {
                            fill: "url(#2930ac__f)",
                            d: "M-38.275 49.257c13.746 0 26.197 5.25 35.213 13.72 18.03 16.967 32.978 16.982 51.008.043 9.015-8.513 21.482-13.778 35.244-13.778 27.507.015 49.79 20.986 49.79 46.873s-22.283 46.843-49.79 46.843c-13.793 0-26.26-5.279-35.275-13.792-18.015-16.91-32.947-16.895-50.977.073-9.016 8.484-21.467 13.719-35.213 13.719-27.492 0-49.776-20.97-49.776-46.843s22.284-46.858 49.776-46.858"
                        })
                    ]
                })
            })
        ]
    });
const $c397f8fe0005fe41$var$ForwardRef = /*#__PURE__*/ (0, $egTD9$react.forwardRef)($c397f8fe0005fe41$var$SvgComponent);
var $c397f8fe0005fe41$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($c397f8fe0005fe41$var$ForwardRef);
