// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_graphBarChart_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $d9J3D$jsxs, jsx as $d9J3D$jsx} from "react/jsx-runtime";
import {forwardRef as $d9J3D$forwardRef} from "react";





const $944bd3eb049079dd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $d9J3D$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $d9J3D$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $d9J3D$jsxs)("linearGradient", {
                        id: "e549cc__d",
                        x1: 108.469,
                        x2: 31.005,
                        y1: 10.626,
                        y2: 117.168,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d9J3D$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $d9J3D$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d9J3D$jsxs)("linearGradient", {
                        id: "e549cc__e",
                        x1: 57.566,
                        x2: 71.927,
                        y1: -36.383,
                        y2: 61.114,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d9J3D$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $d9J3D$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d9J3D$jsxs)("linearGradient", {
                        id: "e549cc__f",
                        x1: -128.408,
                        x2: 141.506,
                        y1: 159.11,
                        y2: 102.522,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d9J3D$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $d9J3D$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $d9J3D$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $d9J3D$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d9J3D$jsxs)("radialGradient", {
                        id: "e549cc__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 196.359 -208.3 0 132.683 104.408)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $d9J3D$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $d9J3D$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $d9J3D$jsx)("clipPath", {
                        id: "e549cc__b",
                        children: /*#__PURE__*/ (0, $d9J3D$jsx)("rect", {
                            width: 144.27,
                            height: 136,
                            x: 4.082,
                            y: 15,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $d9J3D$jsx)("clipPath", {
                id: "e549cc__a",
                children: /*#__PURE__*/ (0, $d9J3D$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M24.21 136h111.791c2.21 0 4 1.79 4 4s-1.79 4-4 4H20.211c-2.21 0-4-1.79-4-4V24c0-2.21 1.79-4 4-4s4 1.79 4 4zM44 71.05h4c4.418 0 8 3.582 8 8V116c0 4.418-3.582 8-8 8h-4c-4.418 0-8-3.582-8-8V79.05c0-4.418 3.582-8 8-8M120 96h-4c-4.418 0-8 3.582-8 8v12c0 4.418 3.582 8 8 8h4c4.418 0 8-3.582 8-8v-12c0-4.418-3.582-8-8-8M80 40h4c4.418 0 8 3.582 8 8v68c0 4.418-3.582 8-8 8h-4c-4.418 0-8-3.582-8-8V48c0-4.418 3.582-8 8-8"
                })
            }),
            /*#__PURE__*/ (0, $d9J3D$jsx)("g", {
                clipPath: "url(#e549cc__a)",
                children: /*#__PURE__*/ (0, $d9J3D$jsxs)("g", {
                    clipPath: "url(#e549cc__b)",
                    children: [
                        /*#__PURE__*/ (0, $d9J3D$jsx)("rect", {
                            width: 144.27,
                            height: 136,
                            x: 4.082,
                            y: 15,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $d9J3D$jsx)("ellipse", {
                            cx: 132.683,
                            cy: 104.408,
                            fill: "url(#e549cc__c)",
                            rx: 208.3,
                            ry: 196.359
                        }),
                        /*#__PURE__*/ (0, $d9J3D$jsx)("path", {
                            fill: "url(#e549cc__d)",
                            d: "M-8.903 57.596C-11.218 10.59 27.33-29.288 77.195-31.47s92.168 34.154 94.484 81.162-36.232 86.884-86.098 89.067-92.168-34.155-94.484-81.163"
                        }),
                        /*#__PURE__*/ (0, $d9J3D$jsx)("path", {
                            fill: "url(#e549cc__e)",
                            d: "M40.528-79.107c-11.249-6.562-25.501-6.602-36.792-.104l-112.93 64.999c-11.06 6.366-11.078 21.392-.033 27.782L2.956 78.476c11.274 6.522 25.527 6.514 36.791-.023L151.7 13.486c10.987-6.377 11.007-21.319.034-27.719z"
                        }),
                        /*#__PURE__*/ (0, $d9J3D$jsx)("path", {
                            fill: "url(#e549cc__f)",
                            d: "M-67.841 80.05c23.276 0 44.36 8.88 59.626 23.207 30.53 28.702 55.843 28.727 86.373.074 15.266-14.4 36.376-23.306 59.679-23.306 46.579.025 84.311 35.498 84.311 79.287s-37.732 79.238-84.311 79.238c-23.355 0-44.465-8.929-59.73-23.329-30.505-28.605-55.791-28.58-86.322.122-15.265 14.351-36.35 23.207-59.626 23.207-46.553 0-84.286-35.473-84.286-79.238s37.733-79.262 84.286-79.262"
                        })
                    ]
                })
            })
        ]
    });
const $944bd3eb049079dd$var$ForwardRef = /*#__PURE__*/ (0, $d9J3D$forwardRef)($944bd3eb049079dd$var$SvgComponent);
var $944bd3eb049079dd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($944bd3eb049079dd$var$ForwardRef);


export {$944bd3eb049079dd$export$2e2bcd8739ae039 as default};
