// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_brightnessContrast_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $9iQqp$reactjsxruntime = require("react/jsx-runtime");
var $9iQqp$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $9da23652d4aea71c$export$2e2bcd8739ae039);




const $9da23652d4aea71c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "175020__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.087,
                        y2: 68.995,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "175020__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.271,
                        y2: 32.368,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "175020__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsxs)("radialGradient", {
                        id: "175020__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("clipPath", {
                        id: "175020__b",
                        children: /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("clipPath", {
                id: "175020__a",
                children: /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M48 13.234c1.105 0 2 .896 2 2v4.645c0 1.104-.895 2-2 2s-2-.896-2-2v-4.645c0-1.104.895-2 2-2M48 29.452c10.244 0 18.548 8.305 18.548 18.549S58.244 66.55 48 66.55c-10.245 0-18.55-8.305-18.55-18.55 0-10.243 8.305-18.548 18.55-18.548m-.228 4.002C39.842 33.576 33.45 40.042 33.45 48s6.39 14.425 14.32 14.547zM50 76.121c0-1.104-.895-2-2-2s-2 .896-2 2v4.645c0 1.104.895 2 2 2s2-.896 2-2zM82.766 48c0 1.105-.896 2-2 2H76.12c-1.105 0-2-.895-2-2s.895-2 2-2h4.645c1.104 0 2 .895 2 2M19.879 50c1.104 0 2-.895 2-2s-.896-2-2-2h-4.645c-1.104 0-2 .895-2 2s.896 2 2 2zM72.583 23.416c.78.781.78 2.047 0 2.828l-3.284 3.285c-.782.78-2.048.78-2.829 0-.78-.781-.78-2.048 0-2.829l3.284-3.284c.781-.78 2.048-.78 2.829 0M29.53 69.298c.78-.781.78-2.048 0-2.829-.782-.78-2.048-.78-2.83 0l-3.284 3.285c-.78.78-.78 2.047 0 2.828.782.781 2.048.781 2.829 0zM23.416 23.416c-.78.781-.78 2.047 0 2.828l3.285 3.285c.78.78 2.047.78 2.828 0 .78-.781.78-2.048 0-2.829l-3.284-3.284c-.781-.78-2.048-.78-2.829 0M66.47 69.298c-.781-.781-.781-2.048 0-2.829.781-.78 2.047-.78 2.829 0l3.284 3.285c.78.78.78 2.047 0 2.828-.781.781-2.048.781-2.829 0z"
                })
            }),
            /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("g", {
                clipPath: "url(#175020__a)",
                children: /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#175020__b)",
                    children: [
                        /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#175020__c)"
                        }),
                        /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("path", {
                            fill: "url(#175020__d)",
                            d: "M-8.64 30.068c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871C21.05 88.9-7.098 63.25-8.64 30.068"
                        }),
                        /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("path", {
                            fill: "url(#175020__e)",
                            d: "M24.253-66.429c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016L98.229-1.07c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $9iQqp$reactjsxruntime.jsx)("path", {
                            fill: "url(#175020__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $9da23652d4aea71c$var$ForwardRef = /*#__PURE__*/ (0, $9iQqp$react.forwardRef)($9da23652d4aea71c$var$SvgComponent);
var $9da23652d4aea71c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($9da23652d4aea71c$var$ForwardRef);
