// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_camera_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $lvqh6$jsxs, jsx as $lvqh6$jsx} from "react/jsx-runtime";
import {forwardRef as $lvqh6$forwardRef} from "react";





const $f7eb1de1012e7e2c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lvqh6$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lvqh6$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lvqh6$jsxs)("linearGradient", {
                        id: "3ef66f__d",
                        x1: 32.787,
                        x2: 12.555,
                        y1: 5.818,
                        y2: 33.644,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvqh6$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $lvqh6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvqh6$jsxs)("linearGradient", {
                        id: "3ef66f__e",
                        x1: 19.493,
                        x2: 23.243,
                        y1: -6.46,
                        y2: 19.004,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvqh6$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $lvqh6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvqh6$jsxs)("linearGradient", {
                        id: "3ef66f__f",
                        x1: -29.081,
                        x2: 41.415,
                        y1: 44.599,
                        y2: 29.819,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvqh6$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lvqh6$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $lvqh6$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $lvqh6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvqh6$jsxs)("radialGradient", {
                        id: "3ef66f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.2844 -54.4031 0 39.11 30.312)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lvqh6$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lvqh6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lvqh6$jsx)("clipPath", {
                        id: "3ef66f__b",
                        children: /*#__PURE__*/ (0, $lvqh6$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.523,
                            y: 6.96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lvqh6$jsx)("clipPath", {
                id: "3ef66f__a",
                children: /*#__PURE__*/ (0, $lvqh6$jsx)("path", {
                    fill: "#D9D9D9",
                    d: "M31.5 13.2h-15l1.941-3.108C19.098 9.04 20.253 8.4 21.495 8.4h5.01c1.242 0 2.394.64 3.054 1.692zm-20.7 2.4h26.4c2.651 0 4.8 2.15 4.8 4.8v14.4c0 2.652-2.149 4.8-4.8 4.8H10.8c-2.651 0-4.8-2.148-4.8-4.8V20.4c0-2.65 2.149-4.8 4.8-4.8m.45 6.9c.994 0 1.8-.805 1.8-1.8 0-.994-.806-1.8-1.8-1.8s-1.8.806-1.8 1.8c0 .995.806 1.8 1.8 1.8M24 34.8c3.977 0 7.2-3.223 7.2-7.2 0-3.976-3.223-7.2-7.2-7.2s-7.2 3.224-7.2 7.2 3.223 7.2 7.2 7.2"
                })
            }),
            /*#__PURE__*/ (0, $lvqh6$jsx)("g", {
                clipPath: "url(#3ef66f__a)",
                children: /*#__PURE__*/ (0, $lvqh6$jsxs)("g", {
                    clipPath: "url(#3ef66f__b)",
                    children: [
                        /*#__PURE__*/ (0, $lvqh6$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.523,
                            y: 6.96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $lvqh6$jsx)("ellipse", {
                            cx: 39.11,
                            cy: 30.312,
                            fill: "url(#3ef66f__c)",
                            rx: 54.403,
                            ry: 51.284
                        }),
                        /*#__PURE__*/ (0, $lvqh6$jsx)("path", {
                            fill: "url(#3ef66f__d)",
                            d: "M2.132 18.086C1.527 5.807 11.595-4.608 24.62-5.178s24.072 8.92 24.677 21.198c.604 12.277-9.463 22.692-22.487 23.262S2.737 30.363 2.132 18.086"
                        }),
                        /*#__PURE__*/ (0, $lvqh6$jsx)("path", {
                            fill: "url(#3ef66f__e)",
                            d: "M15.043-17.618c-2.938-1.714-6.66-1.725-9.61-.027L-24.06-.67c-2.889 1.662-2.893 5.587-.009 7.256l29.3 16.952c2.944 1.703 6.667 1.7 9.609-.007l29.24-16.967c2.868-1.666 2.874-5.568.008-7.24z"
                        }),
                        /*#__PURE__*/ (0, $lvqh6$jsx)("path", {
                            fill: "url(#3ef66f__f)",
                            d: "M-13.262 23.95c6.08 0 11.586 2.32 15.573 6.06 7.974 7.497 14.585 7.504 22.559.02 3.987-3.76 9.5-6.087 15.586-6.087 12.166.007 22.02 9.272 22.02 20.708 0 11.437-9.854 20.695-22.02 20.695-6.1 0-11.613-2.332-15.6-6.093-7.967-7.47-14.571-7.464-22.545.032-3.987 3.748-9.494 6.061-15.573 6.061-12.158 0-22.013-9.264-22.013-20.695 0-11.43 9.855-20.701 22.013-20.701"
                        })
                    ]
                })
            })
        ]
    });
const $f7eb1de1012e7e2c$var$ForwardRef = /*#__PURE__*/ (0, $lvqh6$forwardRef)($f7eb1de1012e7e2c$var$SvgComponent);
var $f7eb1de1012e7e2c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f7eb1de1012e7e2c$var$ForwardRef);


export {$f7eb1de1012e7e2c$export$2e2bcd8739ae039 as default};
