// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_education_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $83zna$reactjsxruntime = require("react/jsx-runtime");
var $83zna$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5e7c507c7d0e8e9a$export$2e2bcd8739ae039);




const $5e7c507c7d0e8e9a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $83zna$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ac4031__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ac4031__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsxs)("linearGradient", {
                        id: "ac4031__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsxs)("radialGradient", {
                        id: "ac4031__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("clipPath", {
                        id: "ac4031__b",
                        children: /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("clipPath", {
                id: "ac4031__a",
                children: /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m30.444 50.95 46.532-19.008c1.939-.792 4.111-.792 6.05 0L129.56 50.95c6.633 2.71 6.633 12.102 0 14.812l-4.084 1.668v23.183c0 3.005-.803 7.337-3.836 11.853-3.04 4.528-8.165 9.013-16.445 12.562-6.673 2.859-14.368 3.959-21.085 4.208-6.73.251-12.815-.341-16.436-1.087-2.164-.446-3.557-2.561-3.111-4.724s2.56-3.557 4.724-3.112c2.907.599 8.343 1.159 14.526.929 6.196-.23 12.806-1.243 18.23-3.568 7.111-3.047 10.914-6.627 12.955-9.667 2.048-3.05 2.478-5.79 2.478-7.394V70.699L83.026 84.77c-1.939.792-4.111.792-6.05 0L53.93 75.356c.29-.32.633-.596 1.022-.812L81.943 59.55c1.931-1.073 2.627-3.508 1.554-5.439s-3.508-2.627-5.439-1.554L51.067 67.551c-2.014 1.119-3.623 2.768-4.693 4.718l-15.93-6.507c-6.633-2.71-6.633-12.103 0-14.812m14.45 60.659V78.041c0-2.054.526-4.033 1.48-5.772l7.556 3.087c-.656.724-1.035 1.677-1.035 2.685v32.678c2.817 4.951 5.712 11.287 6.278 16.662q.09.615.09 1.253c0 5.09-4.384 9.216-9.793 9.216s-9.792-4.126-9.792-9.216q0-.513.058-1.011c.422-5.11 2.764-11.132 5.159-16.014"
                })
            }),
            /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("g", {
                clipPath: "url(#ac4031__a)",
                children: /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#ac4031__b)",
                    children: [
                        /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#ac4031__c)"
                        }),
                        /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("path", {
                            fill: "url(#ac4031__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("path", {
                            fill: "url(#ac4031__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $83zna$reactjsxruntime.jsx)("path", {
                            fill: "url(#ac4031__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $5e7c507c7d0e8e9a$var$ForwardRef = /*#__PURE__*/ (0, $83zna$react.forwardRef)($5e7c507c7d0e8e9a$var$SvgComponent);
var $5e7c507c7d0e8e9a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5e7c507c7d0e8e9a$var$ForwardRef);
