// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lockOpen_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $3MK15$jsxs, jsx as $3MK15$jsx} from "react/jsx-runtime";
import {forwardRef as $3MK15$forwardRef} from "react";





const $53d703ec6b3283c3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3MK15$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3MK15$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3MK15$jsxs)("linearGradient", {
                        id: "b9143f__d",
                        x1: 33.33,
                        x2: 9.156,
                        y1: 1.787,
                        y2: 33.129,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3MK15$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3MK15$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3MK15$jsxs)("linearGradient", {
                        id: "b9143f__e",
                        x1: 18.602,
                        x2: 23.265,
                        y1: -12.642,
                        y2: 17.203,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3MK15$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3MK15$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3MK15$jsxs)("linearGradient", {
                        id: "b9143f__f",
                        x1: -35.206,
                        x2: 43.258,
                        y1: 47.361,
                        y2: 31.853,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3MK15$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3MK15$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3MK15$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3MK15$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3MK15$jsxs)("radialGradient", {
                        id: "b9143f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 4.884 35.455)scale(60.2686)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3MK15$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3MK15$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3MK15$jsx)("clipPath", {
                        id: "b9143f__b",
                        children: /*#__PURE__*/ (0, $3MK15$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3MK15$jsx)("clipPath", {
                id: "b9143f__a",
                children: /*#__PURE__*/ (0, $3MK15$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M16.197 13.65c0-4.308 3.492-7.8 7.8-7.8s7.8 3.492 7.8 7.8c0 .662-.537 1.2-1.2 1.2s-1.2-.538-1.2-1.2c0-2.983-2.418-5.4-5.4-5.4s-5.4 2.417-5.4 5.4v6.3h16.09c2.215 0 4.01 1.637 4.01 3.656v12.187c0 2.02-1.795 3.657-4.01 3.657H13.307c-2.214 0-4.01-1.637-4.01-3.657V23.606c0-2.02 1.796-3.656 4.01-3.656h2.89zm6.105 16.545q.223.225.496.382v2.273c0 .662.537 1.2 1.2 1.2s1.2-.538 1.2-1.2v-2.273c.18-.104.348-.232.496-.382q.168-.164.3-.364.131-.193.22-.412c.06-.148.108-.295.136-.451.032-.152.048-.312.048-.468s-.016-.316-.048-.469c-.028-.151-.076-.303-.136-.447-.06-.148-.132-.284-.22-.416q-.132-.2-.3-.364c-.444-.448-1.064-.704-1.696-.704s-1.252.255-1.696.703q-.168.164-.3.365c-.084.132-.16.268-.22.416-.06.143-.108.296-.136.447-.032.153-.048.313-.048.469s.016.316.048.468c.028.156.076.303.136.451q.092.219.22.412.132.2.3.364"
                })
            }),
            /*#__PURE__*/ (0, $3MK15$jsx)("g", {
                clipPath: "url(#b9143f__a)",
                children: /*#__PURE__*/ (0, $3MK15$jsxs)("g", {
                    clipPath: "url(#b9143f__b)",
                    children: [
                        /*#__PURE__*/ (0, $3MK15$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3.129,
                            y: 3.129,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3MK15$jsx)("circle", {
                            cx: 40.339,
                            cy: 30.571,
                            r: 60.269,
                            fill: "url(#b9143f__c)"
                        }),
                        /*#__PURE__*/ (0, $3MK15$jsx)("path", {
                            fill: "url(#b9143f__d)",
                            d: "M-.63 16.203C-1.3 1.775 9.854-10.464 24.282-11.134S50.949-.651 51.619 13.777 41.136 40.444 26.708 41.114.04 30.631-.63 16.204"
                        }),
                        /*#__PURE__*/ (0, $3MK15$jsx)("path", {
                            fill: "url(#b9143f__e)",
                            d: "M13.672-25.755c-3.254-2.014-7.378-2.027-10.645-.032l-32.675 19.95c-3.2 1.954-3.205 6.566-.01 8.527l32.46 19.922c3.261 2.002 7.385 2 10.644-.007l32.392-19.94c3.179-1.958 3.185-6.544.01-8.508z"
                        }),
                        /*#__PURE__*/ (0, $3MK15$jsx)("path", {
                            fill: "url(#b9143f__f)",
                            d: "M-17.682 23.094c6.735 0 12.836 2.726 17.252 7.123 8.834 8.81 16.158 8.818 24.991.023 4.417-4.42 10.525-7.153 17.267-7.153 13.477.007 24.395 10.895 24.395 24.336s-10.918 24.32-24.395 24.32c-6.757 0-12.865-2.74-17.282-7.16-8.826-8.78-16.142-8.773-24.976.037-4.416 4.405-10.517 7.123-17.252 7.123-13.469 0-24.386-10.888-24.386-24.32s10.917-24.329 24.386-24.329"
                        })
                    ]
                })
            })
        ]
    });
const $53d703ec6b3283c3$var$ForwardRef = /*#__PURE__*/ (0, $3MK15$forwardRef)($53d703ec6b3283c3$var$SvgComponent);
var $53d703ec6b3283c3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($53d703ec6b3283c3$var$ForwardRef);


export {$53d703ec6b3283c3$export$2e2bcd8739ae039 as default};
