// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_heart_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $32RgJ$jsxs, jsx as $32RgJ$jsx} from "react/jsx-runtime";
import {forwardRef as $32RgJ$forwardRef} from "react";





const $cecbd351e93c157a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $32RgJ$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $32RgJ$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $32RgJ$jsxs)("linearGradient", {
                        id: "c54e03__d",
                        x1: 65.672,
                        x2: 19.11,
                        y1: 2.815,
                        y2: 63.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $32RgJ$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $32RgJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $32RgJ$jsxs)("linearGradient", {
                        id: "c54e03__e",
                        x1: 37.304,
                        x2: 46.286,
                        y1: -24.976,
                        y2: 32.508,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $32RgJ$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $32RgJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $32RgJ$jsxs)("linearGradient", {
                        id: "c54e03__f",
                        x1: -66.333,
                        x2: 84.795,
                        y1: 90.595,
                        y2: 60.727,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $32RgJ$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $32RgJ$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $32RgJ$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $32RgJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $32RgJ$jsxs)("radialGradient", {
                        id: "c54e03__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 10.456 68.713)scale(116.083)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $32RgJ$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $32RgJ$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $32RgJ$jsx)("clipPath", {
                        id: "c54e03__b",
                        children: /*#__PURE__*/ (0, $32RgJ$jsx)("rect", {
                            width: 80.4,
                            height: 80.4,
                            x: 7.5,
                            y: 5.4,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $32RgJ$jsx)("clipPath", {
                id: "c54e03__a",
                children: /*#__PURE__*/ (0, $32RgJ$jsx)("path", {
                    fill: "#fff",
                    d: "M64.807 15.6c-9.278.005-16.797 7.595-16.797 16.957-.003.008-.005.027-.008.039-.002-.012-.004-.03-.007-.04 0-9.36-7.52-16.95-16.798-16.956-10.23-.007-17.548 8.82-16.733 19.108 1.475 18.61 21.52 39.503 29.894 46.212 2.13 1.706 5.145 1.706 7.274 0 8.379-6.708 28.434-27.602 29.908-46.213.815-10.289-6.502-19.114-16.733-19.107"
                })
            }),
            /*#__PURE__*/ (0, $32RgJ$jsx)("g", {
                clipPath: "url(#c54e03__a)",
                children: /*#__PURE__*/ (0, $32RgJ$jsxs)("g", {
                    clipPath: "url(#c54e03__b)",
                    children: [
                        /*#__PURE__*/ (0, $32RgJ$jsx)("rect", {
                            width: 80.4,
                            height: 80.4,
                            x: 7.5,
                            y: 5.4,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $32RgJ$jsx)("circle", {
                            cx: 79.169,
                            cy: 58.257,
                            r: 116.083,
                            fill: "url(#c54e03__c)"
                        }),
                        /*#__PURE__*/ (0, $32RgJ$jsx)("path", {
                            fill: "url(#c54e03__d)",
                            d: "M.262 30.583c-1.29-27.79 20.192-51.364 47.982-52.655 27.79-1.29 51.364 20.192 52.654 47.982S80.706 77.274 52.917 78.564 1.553 58.372.262 30.583"
                        }),
                        /*#__PURE__*/ (0, $32RgJ$jsx)("path", {
                            fill: "url(#c54e03__e)",
                            d: "M27.81-50.234c-6.27-3.88-14.212-3.903-20.505-.061L-55.629-11.87c-6.163 3.764-6.174 12.647-.019 16.424L6.871 42.925c6.283 3.856 14.226 3.851 20.503-.013l62.39-38.407c6.122-3.77 6.133-12.603.018-16.387z"
                        }),
                        /*#__PURE__*/ (0, $32RgJ$jsx)("path", {
                            fill: "url(#c54e03__f)",
                            d: "M-32.58 43.856c12.971 0 24.722 5.25 33.229 13.72 17.014 16.968 31.12 16.982 48.134.043 8.508-8.513 20.272-13.777 33.259-13.777 25.957.014 46.985 20.985 46.985 46.872 0 25.888-21.028 46.844-46.985 46.844-13.016 0-24.78-5.279-33.288-13.792-17-16.91-31.09-16.896-48.105.073-8.507 8.484-20.257 13.719-33.23 13.719-25.942 0-46.97-20.971-46.97-46.844 0-25.872 21.028-46.858 46.97-46.858"
                        })
                    ]
                })
            })
        ]
    });
const $cecbd351e93c157a$var$ForwardRef = /*#__PURE__*/ (0, $32RgJ$forwardRef)($cecbd351e93c157a$var$SvgComponent);
var $cecbd351e93c157a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($cecbd351e93c157a$var$ForwardRef);


export {$cecbd351e93c157a$export$2e2bcd8739ae039 as default};
