// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_mailOpen_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7Tav7$jsxs, jsx as $7Tav7$jsx} from "react/jsx-runtime";
import {forwardRef as $7Tav7$forwardRef} from "react";





const $1eed032a259ada2b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7Tav7$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7Tav7$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7Tav7$jsxs)("linearGradient", {
                        id: "fe752b__d",
                        x1: 33.201,
                        x2: 9.027,
                        y1: 1.658,
                        y2: 33,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Tav7$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7Tav7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Tav7$jsxs)("linearGradient", {
                        id: "fe752b__e",
                        x1: 18.473,
                        x2: 23.137,
                        y1: -12.771,
                        y2: 17.075,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Tav7$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7Tav7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Tav7$jsxs)("linearGradient", {
                        id: "fe752b__f",
                        x1: -35.335,
                        x2: 43.129,
                        y1: 47.232,
                        y2: 31.724,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Tav7$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7Tav7$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7Tav7$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7Tav7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Tav7$jsxs)("radialGradient", {
                        id: "fe752b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 4.884 35.326)scale(60.2686)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7Tav7$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7Tav7$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7Tav7$jsx)("clipPath", {
                        id: "fe752b__b",
                        children: /*#__PURE__*/ (0, $7Tav7$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3,
                            y: 3,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7Tav7$jsx)("clipPath", {
                id: "fe752b__a",
                children: /*#__PURE__*/ (0, $7Tav7$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M17.98 14.424c-1.987 0-3.6 1.614-3.6 3.6v1.473L9.536 16.46l13.236-8.283c.783-.49 1.773-.49 2.553.003l13.14 8.277-4.845 3.039v-1.473c0-1.986-1.614-3.6-3.6-3.6zm0 2.4c-.664 0-1.2.537-1.2 1.2V21l6.584 4.13c.387.244.888.244 1.275 0L31.224 21v-2.98c0-.662-.537-1.2-1.2-1.2H17.982zm7.934 10.34c-.582.364-1.248.547-1.914.547s-1.332-.183-1.914-.546l-13.68-8.58V35.39c0 2.652 2.148 4.803 4.8 4.803h21.588c2.652 0 4.8-2.151 4.8-4.803V18.585z"
                })
            }),
            /*#__PURE__*/ (0, $7Tav7$jsx)("g", {
                clipPath: "url(#fe752b__a)",
                children: /*#__PURE__*/ (0, $7Tav7$jsxs)("g", {
                    clipPath: "url(#fe752b__b)",
                    children: [
                        /*#__PURE__*/ (0, $7Tav7$jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3,
                            y: 3,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7Tav7$jsx)("circle", {
                            cx: 40.21,
                            cy: 30.442,
                            r: 60.269,
                            fill: "url(#fe752b__c)"
                        }),
                        /*#__PURE__*/ (0, $7Tav7$jsx)("path", {
                            fill: "url(#fe752b__d)",
                            d: "M-.758 16.074c-.67-14.428 10.483-26.667 24.91-27.337C38.582-11.933 50.82-.78 51.49 13.648S41.007 40.316 26.58 40.986-.088 30.502-.758 16.074"
                        }),
                        /*#__PURE__*/ (0, $7Tav7$jsx)("path", {
                            fill: "url(#fe752b__e)",
                            d: "M13.544-25.884c-3.255-2.014-7.379-2.026-10.646-.032l-32.674 19.95c-3.2 1.954-3.206 6.566-.01 8.527L2.673 22.483c3.262 2.002 7.385 2 10.645-.007l32.391-19.94c3.179-1.958 3.185-6.544.01-8.508z"
                        }),
                        /*#__PURE__*/ (0, $7Tav7$jsx)("path", {
                            fill: "url(#fe752b__f)",
                            d: "M-17.81 22.966c6.734 0 12.835 2.725 17.252 7.122 8.833 8.81 16.157 8.818 24.99.023 4.417-4.42 10.525-7.153 17.267-7.153 13.477.008 24.395 10.895 24.395 24.336s-10.918 24.32-24.395 24.32c-6.757 0-12.865-2.74-17.282-7.16-8.826-8.78-16.142-8.772-24.975.037-4.417 4.405-10.518 7.123-17.252 7.123-13.47 0-24.387-10.888-24.387-24.32S-31.28 22.966-17.81 22.966"
                        })
                    ]
                })
            })
        ]
    });
const $1eed032a259ada2b$var$ForwardRef = /*#__PURE__*/ (0, $7Tav7$forwardRef)($1eed032a259ada2b$var$SvgComponent);
var $1eed032a259ada2b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1eed032a259ada2b$var$ForwardRef);


export {$1eed032a259ada2b$export$2e2bcd8739ae039 as default};
