// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_microphone_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $czy1n$jsxs, jsx as $czy1n$jsx} from "react/jsx-runtime";
import {forwardRef as $czy1n$forwardRef} from "react";





const $18828935cba43aa6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $czy1n$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $czy1n$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $czy1n$jsxs)("linearGradient", {
                        id: "78ee73__d",
                        x1: 110.677,
                        x2: 30.097,
                        y1: 5.525,
                        y2: 110,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $czy1n$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $czy1n$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $czy1n$jsxs)("linearGradient", {
                        id: "78ee73__e",
                        x1: 61.581,
                        x2: 77.126,
                        y1: -42.571,
                        y2: 56.914,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $czy1n$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $czy1n$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $czy1n$jsxs)("linearGradient", {
                        id: "78ee73__f",
                        x1: -117.778,
                        x2: 143.767,
                        y1: 157.439,
                        y2: 105.749,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $czy1n$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $czy1n$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $czy1n$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $czy1n$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $czy1n$jsxs)("radialGradient", {
                        id: "78ee73__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.277 117.75)scale(200.895)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $czy1n$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $czy1n$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $czy1n$jsx)("clipPath", {
                        id: "78ee73__b",
                        children: /*#__PURE__*/ (0, $czy1n$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10,
                            y: 10,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $czy1n$jsx)("clipPath", {
                id: "78ee73__a",
                children: /*#__PURE__*/ (0, $czy1n$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M79 24.8c-10.603 0-19.2 8.597-19.2 19.2v28.6c0 10.605 8.597 19.2 19.2 19.2h2.6c10.605 0 19.201-8.595 19.201-19.2V44c0-10.603-8.596-19.2-19.2-19.2zm-3.2 90.776c-22.462-2.256-40-21.217-40-44.275v-4.5c0-2.21 1.792-4 4-4 2.21 0 4 1.79 4 4v4.5c0 20.158 16.342 36.5 36.5 36.5s36.501-16.342 36.501-36.5v-4.5c0-2.21 1.791-4 4-4s4 1.79 4 4v4.5c0 23.399-18.06 42.579-41 44.364v12.136h20c2.209 0 4 1.791 4 4s-1.791 4-4 4h-47c-2.21 0-4-1.791-4-4s1.79-4 4-4h19z"
                })
            }),
            /*#__PURE__*/ (0, $czy1n$jsx)("g", {
                clipPath: "url(#78ee73__a)",
                children: /*#__PURE__*/ (0, $czy1n$jsxs)("g", {
                    clipPath: "url(#78ee73__b)",
                    children: [
                        /*#__PURE__*/ (0, $czy1n$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10,
                            y: 10,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $czy1n$jsx)("circle", {
                            cx: 134.028,
                            cy: 101.473,
                            r: 200.895,
                            fill: "url(#78ee73__c)"
                        }),
                        /*#__PURE__*/ (0, $czy1n$jsx)("path", {
                            fill: "url(#78ee73__d)",
                            d: "M-2.522 53.58C-4.755 5.487 32.422-35.31 80.516-37.544S169.407-2.6 171.64 45.493s-34.944 88.892-83.037 91.125S-.288 101.674-2.522 53.58"
                        }),
                        /*#__PURE__*/ (0, $czy1n$jsx)("path", {
                            fill: "url(#78ee73__e)",
                            d: "M45.149-86.282c-10.85-6.713-24.595-6.754-35.485-.106l-108.915 66.5c-10.667 6.513-10.685 21.886-.032 28.424L8.912 74.942c10.874 6.672 24.62 6.664 35.484-.024L152.368 8.45c10.596-6.524 10.615-21.81.032-28.36z"
                        }),
                        /*#__PURE__*/ (0, $czy1n$jsx)("path", {
                            fill: "url(#78ee73__f)",
                            d: "M-59.364 76.552c22.449 0 42.784 9.086 57.506 23.744 29.446 29.365 53.858 29.39 83.303.075 14.723-14.733 35.083-23.844 57.557-23.844 44.923.025 81.314 36.318 81.314 81.119s-36.391 81.068-81.314 81.068c-22.525 0-42.885-9.136-57.607-23.868-29.42-29.265-53.807-29.24-83.253.125-14.722 14.683-35.057 23.743-57.506 23.743-44.898 0-81.29-36.292-81.29-81.068s36.392-81.094 81.29-81.094"
                        })
                    ]
                })
            })
        ]
    });
const $18828935cba43aa6$var$ForwardRef = /*#__PURE__*/ (0, $czy1n$forwardRef)($18828935cba43aa6$var$SvgComponent);
var $18828935cba43aa6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($18828935cba43aa6$var$ForwardRef);


export {$18828935cba43aa6$export$2e2bcd8739ae039 as default};
