// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_flag_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $3ugPg$jsxs, jsx as $3ugPg$jsx} from "react/jsx-runtime";
import {forwardRef as $3ugPg$forwardRef} from "react";





const $e449ae416a1d3fc8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3ugPg$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3ugPg$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3ugPg$jsxs)("linearGradient", {
                        id: "534fe0__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3ugPg$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3ugPg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3ugPg$jsxs)("linearGradient", {
                        id: "534fe0__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3ugPg$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3ugPg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3ugPg$jsxs)("linearGradient", {
                        id: "534fe0__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3ugPg$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3ugPg$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3ugPg$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3ugPg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3ugPg$jsxs)("radialGradient", {
                        id: "534fe0__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3ugPg$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3ugPg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3ugPg$jsx)("clipPath", {
                        id: "534fe0__b",
                        children: /*#__PURE__*/ (0, $3ugPg$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3ugPg$jsx)("clipPath", {
                id: "534fe0__a",
                children: /*#__PURE__*/ (0, $3ugPg$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M25.484 18.219c0-1.105-.895-2-2-2s-2 .895-2 2V77.78c0 1.105.896 2 2 2s2-.895 2-2v-17.78h41.172c3.314 0 6-2.686 6-6V28.043c0-3.314-2.686-6-6-6H25.484z"
                })
            }),
            /*#__PURE__*/ (0, $3ugPg$jsx)("g", {
                clipPath: "url(#534fe0__a)",
                children: /*#__PURE__*/ (0, $3ugPg$jsxs)("g", {
                    clipPath: "url(#534fe0__b)",
                    children: [
                        /*#__PURE__*/ (0, $3ugPg$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3ugPg$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#534fe0__c)"
                        }),
                        /*#__PURE__*/ (0, $3ugPg$jsx)("path", {
                            fill: "url(#534fe0__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $3ugPg$jsx)("path", {
                            fill: "url(#534fe0__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $3ugPg$jsx)("path", {
                            fill: "url(#534fe0__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $e449ae416a1d3fc8$var$ForwardRef = /*#__PURE__*/ (0, $3ugPg$forwardRef)($e449ae416a1d3fc8$var$SvgComponent);
var $e449ae416a1d3fc8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e449ae416a1d3fc8$var$ForwardRef);


export {$e449ae416a1d3fc8$export$2e2bcd8739ae039 as default};
