// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_flagCheckmark_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $81NOz$jsxs, jsx as $81NOz$jsx} from "react/jsx-runtime";
import {forwardRef as $81NOz$forwardRef} from "react";





const $2461ddb526b9c3cd$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $81NOz$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $81NOz$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $81NOz$jsxs)("linearGradient", {
                        id: "7a499a__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $81NOz$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $81NOz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $81NOz$jsxs)("linearGradient", {
                        id: "7a499a__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $81NOz$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $81NOz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $81NOz$jsxs)("linearGradient", {
                        id: "7a499a__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $81NOz$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $81NOz$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $81NOz$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $81NOz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $81NOz$jsxs)("radialGradient", {
                        id: "7a499a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $81NOz$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $81NOz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $81NOz$jsx)("clipPath", {
                        id: "7a499a__b",
                        children: /*#__PURE__*/ (0, $81NOz$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $81NOz$jsx)("clipPath", {
                id: "7a499a__a",
                children: /*#__PURE__*/ (0, $81NOz$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M11.742 7.86c.69 0 1.25.559 1.25 1.25v1.912h20.336c1.657 0 3 1.343 3 3V27c0 1.657-1.343 3-3 3H12.992v8.89c0 .691-.56 1.25-1.25 1.25s-1.25-.559-1.25-1.25V9.11c0-.691.56-1.25 1.25-1.25M30.455 18.2c.51-.466.547-1.256.082-1.767s-1.256-.546-1.766-.081L24 20.699l-.039.035-1.381 1.26-2.599-2.424c-.505-.47-1.295-.443-1.766.062s-.444 1.296.061 1.767l3.441 3.209c.477.444 1.214.448 1.695.01zm-7.884 5.494.842.923z"
                })
            }),
            /*#__PURE__*/ (0, $81NOz$jsx)("g", {
                clipPath: "url(#7a499a__a)",
                children: /*#__PURE__*/ (0, $81NOz$jsxs)("g", {
                    clipPath: "url(#7a499a__b)",
                    children: [
                        /*#__PURE__*/ (0, $81NOz$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $81NOz$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#7a499a__c)"
                        }),
                        /*#__PURE__*/ (0, $81NOz$jsx)("path", {
                            fill: "url(#7a499a__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $81NOz$jsx)("path", {
                            fill: "url(#7a499a__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $81NOz$jsx)("path", {
                            fill: "url(#7a499a__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $2461ddb526b9c3cd$var$ForwardRef = /*#__PURE__*/ (0, $81NOz$forwardRef)($2461ddb526b9c3cd$var$SvgComponent);
var $2461ddb526b9c3cd$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2461ddb526b9c3cd$var$ForwardRef);


export {$2461ddb526b9c3cd$export$2e2bcd8739ae039 as default};
