// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_desktop_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $iUWSs$reactjsxruntime = require("react/jsx-runtime");
var $iUWSs$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a246d75d54bdccb1$export$2e2bcd8739ae039);




const $a246d75d54bdccb1$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6a4477__d",
                        x1: 32.846,
                        x2: 11.896,
                        y1: 4.517,
                        y2: 33.33,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6a4477__e",
                        x1: 19.083,
                        x2: 22.967,
                        y1: -8.196,
                        y2: 18.171,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6a4477__f",
                        x1: -31.211,
                        x2: 41.785,
                        y1: 44.673,
                        y2: 29.369,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6a4477__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.1036 -56.3329 0 39.395 29.88)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("clipPath", {
                        id: "6a4477__b",
                        children: /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("rect", {
                            width: 39.017,
                            height: 36.78,
                            x: 4.617,
                            y: 5.7,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("clipPath", {
                id: "6a4477__a",
                children: /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M38.108 8.4H9.89c-2.652 0-4.8 2.148-4.8 4.8v13.2c0 2.652 2.148 4.8 4.8 4.8h10.809v7.2h-3.003c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h12.606c.663 0 1.2-.537 1.2-1.2s-.537-1.2-1.2-1.2h-3.003v-7.2h10.809c2.652 0 4.8-2.148 4.8-4.8V13.2c0-2.652-2.148-4.8-4.8-4.8m-14.103 6c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.197-1.2h.003c.663 0 1.2.537 1.2 1.2s-.537 1.2-1.2 1.2"
                })
            }),
            /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6a4477__a)",
                children: /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6a4477__b)",
                    children: [
                        /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("rect", {
                            width: 39.017,
                            height: 36.78,
                            x: 4.617,
                            y: 5.7,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("ellipse", {
                            cx: 39.395,
                            cy: 29.88,
                            fill: "url(#6a4477__c)",
                            rx: 56.333,
                            ry: 53.104
                        }),
                        /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("path", {
                            fill: "url(#6a4477__d)",
                            d: "M1.104 17.22C.478 4.507 10.902-6.277 24.388-6.868c13.486-.59 24.926 9.237 25.552 21.95S40.142 38.58 26.656 39.17 1.73 29.933 1.104 17.22"
                        }),
                        /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("path", {
                            fill: "url(#6a4477__e)",
                            d: "M14.476-19.75c-3.043-1.775-6.897-1.786-9.95-.028L-26.016-2.2c-2.991 1.722-2.997 5.785-.01 7.513l30.34 17.554c3.049 1.764 6.903 1.761 9.95-.007L44.54 5.29c2.971-1.724 2.977-5.765.009-7.496z"
                        }),
                        /*#__PURE__*/ (0, $iUWSs$reactjsxruntime.jsx)("path", {
                            fill: "url(#6a4477__f)",
                            d: "M-14.831 23.292c6.295 0 11.997 2.401 16.125 6.276 8.257 7.762 15.102 7.769 23.359.02 4.128-3.895 9.837-6.303 16.14-6.303 12.596.007 22.8 9.6 22.8 21.443 0 11.842-10.204 21.429-22.8 21.429-6.317 0-12.026-2.415-16.154-6.31-8.25-7.735-15.088-7.729-23.345.034-4.128 3.88-9.83 6.276-16.125 6.276-12.59 0-22.795-9.593-22.795-21.43 0-11.835 10.205-21.435 22.795-21.435"
                        })
                    ]
                })
            })
        ]
    });
const $a246d75d54bdccb1$var$ForwardRef = /*#__PURE__*/ (0, $iUWSs$react.forwardRef)($a246d75d54bdccb1$var$SvgComponent);
var $a246d75d54bdccb1$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a246d75d54bdccb1$var$ForwardRef);
