// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_handshake_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $epfR0$jsxs, jsx as $epfR0$jsx} from "react/jsx-runtime";
import {forwardRef as $epfR0$forwardRef} from "react";





const $1b3c1b3c2f5bb88d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $epfR0$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $epfR0$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $epfR0$jsxs)("linearGradient", {
                        id: "f77013__d",
                        x1: 107.785,
                        x2: 37.447,
                        y1: 16.683,
                        y2: 113.425,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $epfR0$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $epfR0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $epfR0$jsxs)("linearGradient", {
                        id: "f77013__e",
                        x1: 61.565,
                        x2: 74.605,
                        y1: -26.004,
                        y2: 62.525,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $epfR0$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $epfR0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $epfR0$jsxs)("linearGradient", {
                        id: "f77013__f",
                        x1: -107.301,
                        x2: 137.787,
                        y1: 151.51,
                        y2: 100.127,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $epfR0$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $epfR0$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $epfR0$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $epfR0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $epfR0$jsxs)("radialGradient", {
                        id: "f77013__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 178.298 -189.141 0 129.774 101.839)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $epfR0$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $epfR0$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $epfR0$jsx)("clipPath", {
                        id: "f77013__b",
                        children: /*#__PURE__*/ (0, $epfR0$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $epfR0$jsx)("clipPath", {
                id: "f77013__a",
                children: /*#__PURE__*/ (0, $epfR0$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m61.124 39.715 5.655 5.655 5.78 5.78 11.3 11.3c3.66 3.66 9.59 3.66 13.25 0s3.66-9.59 0-13.25l-11.3-11.3-5.78-5.78-1.88-1.88c-1.34-1.32-2.85-2.34-4.46-3.09l-.154-.074c-.18-.088-.36-.176-.546-.246-.63-.26-1.28-.46-1.93-.64-5.34-1.42-11.26-.05-15.45 4.13l-30.43 30.43c-4.89 4.89-5.93 12.15-3.17 18.08l33.46-33.46zm43.263-9.315 30.43 30.43v.02c6.25 6.25 6.25 16.38 0 22.63l-33.88 33.88-14.28 14.28c-3.66 3.66-9.59 3.66-13.25-.01h-.01c-3.66-3.66-3.66-9.59 0-13.25-3.66 3.66-9.59 3.66-13.25-.01h-.01c-3.63-3.63-3.65-9.49-.08-13.16-3.67 3.57-9.53 3.54-13.16-.1h-.01c-3.66-3.66-3.66-9.6 0-13.26h-.01c-3.66 3.66-9.59 3.66-13.25-.01h-.01c-.29-.29-.55-.6-.79-.91-2.84-3.68-2.58-8.98.79-12.35l14.96-14.97 12.56-12.56 5.78 5.78 11.3 11.3c3.28 3.28 7.64 5.09 12.28 5.09s9-1.81 12.28-5.09c6.77-6.78 6.77-17.8 0-24.57l-16.37-16.37c5.9-2.71 13.11-1.66 17.98 3.21"
                })
            }),
            /*#__PURE__*/ (0, $epfR0$jsx)("g", {
                clipPath: "url(#f77013__a)",
                children: /*#__PURE__*/ (0, $epfR0$jsxs)("g", {
                    clipPath: "url(#f77013__b)",
                    children: [
                        /*#__PURE__*/ (0, $epfR0$jsx)("rect", {
                            width: 131,
                            height: 123.491,
                            x: 13,
                            y: 20.654,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $epfR0$jsx)("ellipse", {
                            cx: 129.774,
                            cy: 101.839,
                            fill: "url(#f77013__c)",
                            rx: 189.141,
                            ry: 178.298
                        }),
                        /*#__PURE__*/ (0, $epfR0$jsx)("path", {
                            fill: "url(#f77013__d)",
                            d: "M1.21 59.333C-.893 16.649 34.11-19.56 79.389-21.542s83.69 31.014 85.792 73.698-32.899 78.892-78.178 80.874c-45.28 1.982-83.69-31.013-85.793-73.697"
                        }),
                        /*#__PURE__*/ (0, $epfR0$jsx)("path", {
                            fill: "url(#f77013__e)",
                            d: "M46.095-64.799c-10.215-5.958-23.156-5.994-33.409-.094L-89.856-5.873c-10.043 5.78-10.06 19.425-.03 25.227L11.977 78.29c10.237 5.922 23.179 5.915 33.407-.02L147.04 19.277c9.976-5.79 9.994-19.358.031-25.17z"
                        }),
                        /*#__PURE__*/ (0, $epfR0$jsx)("path", {
                            fill: "url(#f77013__f)",
                            d: "M-52.305 79.722c21.136 0 40.28 8.063 54.142 21.072 27.723 26.062 50.706 26.085 78.429.067 13.861-13.076 33.03-21.162 54.189-21.162 42.295.023 76.557 32.233 76.557 71.994s-34.262 71.95-76.557 71.95c-21.207 0-40.375-8.108-54.237-21.184-27.699-25.973-50.658-25.951-78.38.112-13.862 13.031-33.007 21.072-54.143 21.072-42.27 0-76.533-32.21-76.533-71.95 0-39.739 34.262-71.971 76.533-71.971"
                        })
                    ]
                })
            })
        ]
    });
const $1b3c1b3c2f5bb88d$var$ForwardRef = /*#__PURE__*/ (0, $epfR0$forwardRef)($1b3c1b3c2f5bb88d$var$SvgComponent);
var $1b3c1b3c2f5bb88d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1b3c1b3c2f5bb88d$var$ForwardRef);


export {$1b3c1b3c2f5bb88d$export$2e2bcd8739ae039 as default};
