// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_effects_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $gTEQz$jsxs, jsx as $gTEQz$jsx} from "react/jsx-runtime";
import {forwardRef as $gTEQz$forwardRef} from "react";





const $aa8dda8d17fd15b2$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gTEQz$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gTEQz$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gTEQz$jsxs)("linearGradient", {
                        id: "3414c4__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gTEQz$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $gTEQz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gTEQz$jsxs)("linearGradient", {
                        id: "3414c4__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gTEQz$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $gTEQz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gTEQz$jsxs)("linearGradient", {
                        id: "3414c4__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gTEQz$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gTEQz$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $gTEQz$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $gTEQz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gTEQz$jsxs)("radialGradient", {
                        id: "3414c4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gTEQz$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gTEQz$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gTEQz$jsx)("clipPath", {
                        id: "3414c4__b",
                        children: /*#__PURE__*/ (0, $gTEQz$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gTEQz$jsx)("clipPath", {
                id: "3414c4__a",
                children: /*#__PURE__*/ (0, $gTEQz$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M73.866 30.965c-3.7 2.817-6.314 7.278-7.825 11.423l-5.35 17.434h28.054c1.454 0 2.795.79 3.5 2.063l10.991 19.857c7.514-7.507 15.095-15.08 20.809-20.786 1.563-1.56 4.096-1.559 5.657.004s1.559 4.096-.004 5.657c-6.126 6.117-14.397 14.38-22.431 22.407l12.442 22.476c1.07 1.933.37 4.367-1.563 5.437s-4.367.371-5.436-1.562l-11.33-20.468-2.719 2.718c-5.15 5.148-9.781 9.779-13.135 13.138-1.677 1.679-3.033 3.039-3.975 3.985-.472.473-.836.84-1.085 1.091l-.182.186-.093.093-.023.025c-1.374 1.499-3.687 1.739-5.347.494-1.767-1.325-2.125-3.833-.8-5.6.102-.136.195-.24.207-.254l.001-.001.067-.074.067-.071.107-.11.308-.313 1.103-1.11c.946-.951 2.307-2.314 3.984-3.995 3.357-3.36 7.99-7.994 13.14-13.142l4.344-4.342-10.962-19.803H58.235L36.95 137.173c-.648 2.112-2.886 3.299-4.998 2.651s-3.298-2.886-2.65-4.998l20.565-67.004H37.059c-2.21 0-4-1.79-4-4s1.79-4 4-4h15.263l6.1-19.877q.03-.093.063-.184c1.806-5.006 5.15-11.06 10.533-15.16 5.602-4.267 13.107-6.161 22.307-3.085 2.095.701 3.226 2.968 2.525 5.063-.7 2.095-2.967 3.225-5.062 2.524-6.672-2.231-11.415-.81-14.922 1.862m6.276 85.204.012-.012c-.053.058-.07.073-.012.012"
                })
            }),
            /*#__PURE__*/ (0, $gTEQz$jsx)("g", {
                clipPath: "url(#3414c4__a)",
                children: /*#__PURE__*/ (0, $gTEQz$jsxs)("g", {
                    clipPath: "url(#3414c4__b)",
                    children: [
                        /*#__PURE__*/ (0, $gTEQz$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $gTEQz$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#3414c4__c)"
                        }),
                        /*#__PURE__*/ (0, $gTEQz$jsx)("path", {
                            fill: "url(#3414c4__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $gTEQz$jsx)("path", {
                            fill: "url(#3414c4__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $gTEQz$jsx)("path", {
                            fill: "url(#3414c4__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $aa8dda8d17fd15b2$var$ForwardRef = /*#__PURE__*/ (0, $gTEQz$forwardRef)($aa8dda8d17fd15b2$var$SvgComponent);
var $aa8dda8d17fd15b2$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($aa8dda8d17fd15b2$var$ForwardRef);


export {$aa8dda8d17fd15b2$export$2e2bcd8739ae039 as default};
