// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_mailClose_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $jPQpP$reactjsxruntime = require("react/jsx-runtime");
var $jPQpP$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $d82a8bb2d9c5d590$export$2e2bcd8739ae039);




const $d82a8bb2d9c5d590$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6d7c86__d",
                        x1: 32.846,
                        x2: 11.896,
                        y1: 4.517,
                        y2: 33.33,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6d7c86__e",
                        x1: 19.083,
                        x2: 22.967,
                        y1: -8.196,
                        y2: 18.171,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsxs)("linearGradient", {
                        id: "6d7c86__f",
                        x1: -31.211,
                        x2: 41.785,
                        y1: 44.673,
                        y2: 29.369,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsxs)("radialGradient", {
                        id: "6d7c86__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 53.1036 -56.3329 0 39.395 29.88)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("clipPath", {
                        id: "6d7c86__b",
                        children: /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("rect", {
                            width: 39.017,
                            height: 36.78,
                            x: 4.617,
                            y: 5.7,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("clipPath", {
                id: "6d7c86__a",
                children: /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M10.154 10.2h27.692c1.108 0 2.114.48 2.858 1.262L24.768 20.76c-.56.326-1.247.32-1.8-.017L7.458 11.3c.726-.686 1.667-1.1 2.696-1.1M6.21 13.35c-.136.456-.21.944-.21 1.45v18.4c0 2.54 1.86 4.6 4.154 4.6h27.692C40.14 37.8 42 35.74 42 33.2V14.8q-.002-.64-.15-1.228l-15.873 9.261c-1.32.77-2.951.755-4.257-.04z"
                })
            }),
            /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("g", {
                clipPath: "url(#6d7c86__a)",
                children: /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#6d7c86__b)",
                    children: [
                        /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("rect", {
                            width: 39.017,
                            height: 36.78,
                            x: 4.617,
                            y: 5.7,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("ellipse", {
                            cx: 39.395,
                            cy: 29.88,
                            fill: "url(#6d7c86__c)",
                            rx: 56.333,
                            ry: 53.104
                        }),
                        /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("path", {
                            fill: "url(#6d7c86__d)",
                            d: "M1.104 17.22C.478 4.507 10.902-6.277 24.388-6.868c13.486-.59 24.926 9.237 25.552 21.95S40.142 38.58 26.656 39.17 1.73 29.933 1.104 17.22"
                        }),
                        /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("path", {
                            fill: "url(#6d7c86__e)",
                            d: "M14.476-19.75c-3.043-1.775-6.897-1.786-9.95-.028L-26.016-2.2c-2.991 1.722-2.997 5.785-.01 7.513l30.34 17.554c3.049 1.764 6.903 1.761 9.95-.007L44.54 5.29c2.971-1.724 2.977-5.765.009-7.496z"
                        }),
                        /*#__PURE__*/ (0, $jPQpP$reactjsxruntime.jsx)("path", {
                            fill: "url(#6d7c86__f)",
                            d: "M-14.831 23.292c6.295 0 11.997 2.401 16.125 6.276 8.257 7.762 15.102 7.769 23.359.02 4.128-3.895 9.837-6.303 16.14-6.303 12.596.007 22.8 9.6 22.8 21.443 0 11.842-10.204 21.429-22.8 21.429-6.317 0-12.026-2.415-16.154-6.31-8.25-7.735-15.088-7.729-23.345.034-4.128 3.88-9.83 6.276-16.125 6.276-12.59 0-22.795-9.593-22.795-21.43 0-11.835 10.205-21.435 22.795-21.435"
                        })
                    ]
                })
            })
        ]
    });
const $d82a8bb2d9c5d590$var$ForwardRef = /*#__PURE__*/ (0, $jPQpP$react.forwardRef)($d82a8bb2d9c5d590$var$SvgComponent);
var $d82a8bb2d9c5d590$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($d82a8bb2d9c5d590$var$ForwardRef);
