// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_translate_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $5QDUB$jsxs, jsx as $5QDUB$jsx} from "react/jsx-runtime";
import {forwardRef as $5QDUB$forwardRef} from "react";





const $885023002aed4f3d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5QDUB$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5QDUB$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5QDUB$jsxs)("linearGradient", {
                        id: "026e8d__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5QDUB$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $5QDUB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5QDUB$jsxs)("linearGradient", {
                        id: "026e8d__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5QDUB$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $5QDUB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5QDUB$jsxs)("linearGradient", {
                        id: "026e8d__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5QDUB$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5QDUB$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $5QDUB$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $5QDUB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5QDUB$jsxs)("radialGradient", {
                        id: "026e8d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5QDUB$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5QDUB$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5QDUB$jsx)("clipPath", {
                        id: "026e8d__b",
                        children: /*#__PURE__*/ (0, $5QDUB$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5QDUB$jsxs)("clipPath", {
                id: "026e8d__a",
                children: [
                    /*#__PURE__*/ (0, $5QDUB$jsx)("path", {
                        fill: "#292929",
                        d: "M94.423 33.829h-28.57v-5.642c0-2.209-1.791-4-4-4s-4 1.791-4 4v5.642H28.285c-2.209 0-4 1.79-4 4s1.791 4 4 4h50.36c-1.834 11.528-8.238 20.9-16.288 27.977-1.862 1.637-3.801 3.14-5.775 4.509-3.698-6.442-6.332-12.624-7.124-17.12-.383-2.175-2.458-3.628-4.633-3.245-2.176.383-3.629 2.458-3.245 4.634 1.015 5.762 4.144 12.914 8.164 19.888-5.243 2.775-10.398 4.608-14.711 5.476-2.166.436-3.568 2.545-3.132 4.71.436 2.167 2.545 3.569 4.71 3.133 5.222-1.051 11.292-3.237 17.372-6.513 5.84 8.718 13.043 17.048 19.456 21.19 1.855 1.199 4.331.666 5.53-1.19s.666-4.331-1.19-5.53c-4.983-3.22-11.367-10.345-16.958-18.639 2.336-1.6 4.625-3.367 6.818-5.294 9.354-8.222 17.178-19.607 19.09-33.986h7.694c2.209 0 4-1.791 4-4s-1.791-4-4-4"
                    }),
                    /*#__PURE__*/ (0, $5QDUB$jsx)("path", {
                        fill: "#292929",
                        fillRule: "evenodd",
                        d: "M93.279 113.843 85 133.374c-.862 2.034-3.21 2.984-5.244 2.122s-2.984-3.21-2.121-5.244l25.184-59.42q.096-.23.219-.448.214-.377.495-.69c.381-.428.86-.78 1.423-1.018.51-.216 1.04-.319 1.561-.318.521 0 1.051.102 1.561.318.562.238 1.042.59 1.423 1.018q.28.314.495.69.123.218.218.449l25.185 59.419c.862 2.034-.088 4.382-2.122 5.244s-4.381-.088-5.243-2.122l-8.282-19.538q-.112.006-.226.007zm13.238-31.734-9.847 23.234h19.695z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5QDUB$jsx)("g", {
                clipPath: "url(#026e8d__a)",
                children: /*#__PURE__*/ (0, $5QDUB$jsxs)("g", {
                    clipPath: "url(#026e8d__b)",
                    children: [
                        /*#__PURE__*/ (0, $5QDUB$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $5QDUB$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#026e8d__c)"
                        }),
                        /*#__PURE__*/ (0, $5QDUB$jsx)("path", {
                            fill: "url(#026e8d__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $5QDUB$jsx)("path", {
                            fill: "url(#026e8d__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $5QDUB$jsx)("path", {
                            fill: "url(#026e8d__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $885023002aed4f3d$var$ForwardRef = /*#__PURE__*/ (0, $5QDUB$forwardRef)($885023002aed4f3d$var$SvgComponent);
var $885023002aed4f3d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($885023002aed4f3d$var$ForwardRef);


export {$885023002aed4f3d$export$2e2bcd8739ae039 as default};
