// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_replace_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $2nzAz$reactjsxruntime = require("react/jsx-runtime");
var $2nzAz$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $124eada49c1f5f4f$export$2e2bcd8739ae039);




const $124eada49c1f5f4f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsxs)("linearGradient", {
                        id: "be0b7e__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsxs)("linearGradient", {
                        id: "be0b7e__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsxs)("linearGradient", {
                        id: "be0b7e__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsxs)("radialGradient", {
                        id: "be0b7e__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("clipPath", {
                        id: "be0b7e__b",
                        children: /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("clipPath", {
                id: "be0b7e__a",
                children: /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M12.113 7.875c-2.347 0-4.25 1.903-4.25 4.25v5.31c0 2.348 1.903 4.25 4.25 4.25h5.311c2.347 0 4.25-1.902 4.25-4.25v-5.31c0-2.347-1.903-4.25-4.25-4.25zm15.332.555c-.69-.023-1.268.517-1.291 1.207-.024.69.517 1.269 1.206 1.292 1.387.047 2.794.446 3.853 1.47.879.85 1.647 2.258 1.867 4.61l-1.584-1.583c-.488-.489-1.28-.489-1.768 0-.488.488-.488 1.28 0 1.767l3.573 3.573c.488.488 1.28.488 1.767 0l3.584-3.584c.488-.488.488-1.28 0-1.768s-1.28-.488-1.767 0l-1.32 1.32c-.27-2.785-1.218-4.78-2.614-6.132-1.646-1.593-3.733-2.112-5.506-2.172M14.652 27.234c-.488-.488-1.28-.488-1.768 0L9.3 30.818c-.488.488-.488 1.28 0 1.768s1.28.488 1.768 0l1.32-1.32c.268 2.785 1.217 4.78 2.613 6.132 1.647 1.593 3.734 2.111 5.506 2.172.69.023 1.268-.517 1.292-1.207.023-.69-.517-1.269-1.207-1.292-1.386-.047-2.794-.446-3.853-1.47-.878-.85-1.647-2.258-1.867-4.61l1.584 1.584c.488.488 1.28.488 1.768 0 .488-.489.488-1.28 0-1.768zm15.922-.92c-2.347 0-4.25 1.903-4.25 4.25v5.311c0 2.348 1.902 4.25 4.25 4.25h5.31c2.348 0 4.25-1.903 4.25-4.25v-5.31c0-2.348-1.902-4.25-4.25-4.25z"
                })
            }),
            /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("g", {
                clipPath: "url(#be0b7e__a)",
                children: /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#be0b7e__b)",
                    children: [
                        /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#be0b7e__c)"
                        }),
                        /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("path", {
                            fill: "url(#be0b7e__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("path", {
                            fill: "url(#be0b7e__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $2nzAz$reactjsxruntime.jsx)("path", {
                            fill: "url(#be0b7e__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $124eada49c1f5f4f$var$ForwardRef = /*#__PURE__*/ (0, $2nzAz$react.forwardRef)($124eada49c1f5f4f$var$SvgComponent);
var $124eada49c1f5f4f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($124eada49c1f5f4f$var$ForwardRef);
