// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_globeGrid_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6cqtx$jsxs, jsx as $6cqtx$jsx} from "react/jsx-runtime";
import {forwardRef as $6cqtx$forwardRef} from "react";





const $265b89930520f9ea$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6cqtx$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6cqtx$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6cqtx$jsxs)("linearGradient", {
                        id: "d9dee2__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6cqtx$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6cqtx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6cqtx$jsxs)("linearGradient", {
                        id: "d9dee2__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6cqtx$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6cqtx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6cqtx$jsxs)("linearGradient", {
                        id: "d9dee2__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6cqtx$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6cqtx$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6cqtx$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6cqtx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6cqtx$jsxs)("radialGradient", {
                        id: "d9dee2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6cqtx$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6cqtx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6cqtx$jsx)("clipPath", {
                        id: "d9dee2__b",
                        children: /*#__PURE__*/ (0, $6cqtx$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6cqtx$jsx)("clipPath", {
                id: "d9dee2__a",
                children: /*#__PURE__*/ (0, $6cqtx$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M83.947 25.207v23.079h14.48c-3.451-8.02-8.27-15.777-14.48-23.079m23.127 23.079c-2.976-7.692-7.08-15.18-12.331-22.324 12.936 3.52 24.009 11.563 31.417 22.324zm-5.681 8H83.947v20.313h20.999c-.288-6.837-1.468-13.642-3.553-20.313m11.56 20.313c-.258-6.824-1.323-13.625-3.208-20.313h20.13q.427-.001.83-.086c2.94 6.253 4.758 13.14 5.193 20.399zm-8.077 8H83.947v20.312h17.089c2.151-6.693 3.435-13.498 3.84-20.312m4.52 20.312c1.953-6.699 3.121-13.5 3.491-20.312h22.927c-.59 7.252-2.561 14.115-5.651 20.322q-.142-.01-.288-.01zm-11.368 8h-14.08v22.953l.082-.006c5.9-7.309 10.573-15.018 13.998-22.947m-2.506 20.911c4.624-6.732 8.346-13.74 11.148-20.911h18.644c-7.238 9.948-17.671 17.421-29.792 20.911M33.84 48.286c7.382-10.724 18.405-18.748 31.285-22.288-5.24 7.132-9.334 14.61-12.305 22.288zm-3.215 8H50.15c-1.886 6.688-2.95 13.49-3.209 20.313h-22.84c.438-7.306 2.277-14.234 5.25-20.52.4.134.829.207 1.274.207m27.876 0c-2.085 6.671-3.265 13.476-3.552 20.313h20.998V56.286zM24.186 84.599h22.821c.37 6.813 1.538 13.613 3.491 20.312H30.625q-.39 0-.758.072c-3.107-6.224-5.09-13.108-5.681-20.385m30.833 0c.404 6.814 1.688 13.619 3.84 20.312h17.088V84.599zM34.686 112.91h18.538c2.797 7.157 6.51 14.153 11.122 20.873-12.065-3.506-22.449-10.961-29.66-20.873m27.18 0c3.424 7.927 8.095 15.632 13.991 22.939l.09.006v-22.945zm14.081-87.704c-6.21 7.302-11.029 15.058-14.48 23.079h14.48z"
                })
            }),
            /*#__PURE__*/ (0, $6cqtx$jsx)("g", {
                clipPath: "url(#d9dee2__a)",
                children: /*#__PURE__*/ (0, $6cqtx$jsxs)("g", {
                    clipPath: "url(#d9dee2__b)",
                    children: [
                        /*#__PURE__*/ (0, $6cqtx$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6cqtx$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#d9dee2__c)"
                        }),
                        /*#__PURE__*/ (0, $6cqtx$jsx)("path", {
                            fill: "url(#d9dee2__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $6cqtx$jsx)("path", {
                            fill: "url(#d9dee2__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $6cqtx$jsx)("path", {
                            fill: "url(#d9dee2__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $265b89930520f9ea$var$ForwardRef = /*#__PURE__*/ (0, $6cqtx$forwardRef)($265b89930520f9ea$var$SvgComponent);
var $265b89930520f9ea$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($265b89930520f9ea$var$ForwardRef);


export {$265b89930520f9ea$export$2e2bcd8739ae039 as default};
