// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_information_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $ghlUI$jsxs, jsx as $ghlUI$jsx} from "react/jsx-runtime";
import {forwardRef as $ghlUI$forwardRef} from "react";





const $9e3f1fedd4494517$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ghlUI$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $ghlUI$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $ghlUI$jsxs)("linearGradient", {
                        id: "5a1822__d",
                        x1: 109.277,
                        x2: 41.838,
                        y1: 16.392,
                        y2: 109.146,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ghlUI$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $ghlUI$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ghlUI$jsxs)("linearGradient", {
                        id: "5a1822__e",
                        x1: 64.961,
                        x2: 77.463,
                        y1: -24.535,
                        y2: 60.345,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ghlUI$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $ghlUI$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ghlUI$jsxs)("linearGradient", {
                        id: "5a1822__f",
                        x1: -96.944,
                        x2: 138.04,
                        y1: 145.659,
                        y2: 96.394,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ghlUI$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $ghlUI$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $ghlUI$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $ghlUI$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ghlUI$jsxs)("radialGradient", {
                        id: "5a1822__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 170.948 -181.344 0 130.355 98.038)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ghlUI$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ghlUI$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ghlUI$jsx)("clipPath", {
                        id: "5a1822__b",
                        children: /*#__PURE__*/ (0, $ghlUI$jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 20.199,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ghlUI$jsx)("clipPath", {
                id: "5a1822__a",
                children: /*#__PURE__*/ (0, $ghlUI$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M136 80c0 30.928-25.072 56-56 56s-56-25.072-56-56 25.072-56 56-56 56 25.072 56 56M72.5 66.5c-2.209 0-4 1.79-4 4s1.791 4 4 4h5v23h-5c-2.209 0-4 1.79-4 4s1.791 4 4 4h18c2.209 0 4-1.791 4-4s-1.791-4-4-4h-5v-27c0-2.21-1.791-4-4-4zm8-5c2.761 0 5-2.239 5-5s-2.239-5-5-5-5 2.239-5 5 2.239 5 5 5"
                })
            }),
            /*#__PURE__*/ (0, $ghlUI$jsx)("g", {
                clipPath: "url(#5a1822__a)",
                children: /*#__PURE__*/ (0, $ghlUI$jsxs)("g", {
                    clipPath: "url(#5a1822__b)",
                    children: [
                        /*#__PURE__*/ (0, $ghlUI$jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 20.199,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $ghlUI$jsx)("ellipse", {
                            cx: 130.355,
                            cy: 98.038,
                            fill: "url(#5a1822__c)",
                            rx: 181.344,
                            ry: 170.948
                        }),
                        /*#__PURE__*/ (0, $ghlUI$jsx)("path", {
                            fill: "url(#5a1822__d)",
                            d: "M7.095 57.284C5.08 16.359 38.638-18.356 82.051-20.257s80.24 29.735 82.256 70.66c2.016 40.923-31.543 75.639-74.956 77.54-43.413 1.9-80.24-29.735-82.256-70.66"
                        }),
                        /*#__PURE__*/ (0, $ghlUI$jsx)("path", {
                            fill: "url(#5a1822__e)",
                            d: "M50.129-61.73c-9.794-5.713-22.202-5.748-32.032-.09L-80.218-5.233c-9.629 5.542-9.645 18.623-.03 24.186L17.42 75.46c9.814 5.678 22.223 5.671 32.03-.02l97.464-56.56c9.564-5.551 9.582-18.56.029-24.131z"
                        }),
                        /*#__PURE__*/ (0, $ghlUI$jsx)("path", {
                            fill: "url(#5a1822__f)",
                            d: "M-44.215 76.83c20.264 0 38.62 7.731 51.91 20.204 26.58 24.988 48.615 25.009 75.195.064 13.29-12.537 31.668-20.29 51.955-20.29 40.551.022 73.401 30.904 73.401 69.027s-32.85 68.984-73.401 68.984c-20.332 0-38.71-7.774-52-20.311-26.557-24.903-48.57-24.881-75.15.107-13.29 12.494-31.646 20.204-51.91 20.204-40.529 0-73.378-30.883-73.378-68.984s32.85-69.005 73.378-69.005"
                        })
                    ]
                })
            })
        ]
    });
const $9e3f1fedd4494517$var$ForwardRef = /*#__PURE__*/ (0, $ghlUI$forwardRef)($9e3f1fedd4494517$var$SvgComponent);
var $9e3f1fedd4494517$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($9e3f1fedd4494517$var$ForwardRef);


export {$9e3f1fedd4494517$export$2e2bcd8739ae039 as default};
