// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_revert_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $5oGqd$reactjsxruntime = require("react/jsx-runtime");
var $5oGqd$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $5916dbe1704c32b4$export$2e2bcd8739ae039);




const $5916dbe1704c32b4$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bb9868__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bb9868__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsxs)("linearGradient", {
                        id: "bb9868__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsxs)("radialGradient", {
                        id: "bb9868__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("clipPath", {
                        id: "bb9868__b",
                        children: /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("clipPath", {
                id: "bb9868__a",
                children: /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m19.941 52.458 10.935-8.574c.87-.682 2.126-.53 2.808.34.682.869.53 2.126-.34 2.808L18.776 58.456c-.87.681-2.127.53-2.809-.34L4.544 43.547c-.681-.87-.529-2.126.34-2.808.87-.681 2.127-.53 2.808.34L16 51.675c1.858-17.47 16.643-31.08 34.606-31.08 19.22 0 34.803 15.583 34.803 34.804 0 1.104-.896 2-2 2s-2-.896-2-2c0-17.012-13.791-30.803-30.803-30.803-16.02 0-29.184 12.23-30.665 27.862m-5.86 20.946c0-1.104.895-2 2-2h69.03c1.105 0 2 .896 2 2s-.895 2-2 2h-69.03c-1.105 0-2-.895-2-2"
                })
            }),
            /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("g", {
                clipPath: "url(#bb9868__a)",
                children: /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#bb9868__b)",
                    children: [
                        /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#bb9868__c)"
                        }),
                        /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("path", {
                            fill: "url(#bb9868__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("path", {
                            fill: "url(#bb9868__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $5oGqd$reactjsxruntime.jsx)("path", {
                            fill: "url(#bb9868__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $5916dbe1704c32b4$var$ForwardRef = /*#__PURE__*/ (0, $5oGqd$react.forwardRef)($5916dbe1704c32b4$var$SvgComponent);
var $5916dbe1704c32b4$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($5916dbe1704c32b4$var$ForwardRef);
