// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_text_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $aqhh4$jsxs, jsx as $aqhh4$jsx} from "react/jsx-runtime";
import {forwardRef as $aqhh4$forwardRef} from "react";





const $c1c7125dca9fcf7c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aqhh4$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aqhh4$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aqhh4$jsxs)("linearGradient", {
                        id: "5a3d02__e",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aqhh4$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $aqhh4$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aqhh4$jsxs)("linearGradient", {
                        id: "5a3d02__f",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aqhh4$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $aqhh4$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aqhh4$jsxs)("linearGradient", {
                        id: "5a3d02__g",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aqhh4$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aqhh4$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $aqhh4$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $aqhh4$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aqhh4$jsxs)("radialGradient", {
                        id: "5a3d02__d",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aqhh4$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aqhh4$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aqhh4$jsx)("clipPath", {
                        id: "5a3d02__c",
                        children: /*#__PURE__*/ (0, $aqhh4$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aqhh4$jsxs)("clipPath", {
                id: "5a3d02__b",
                children: [
                    /*#__PURE__*/ (0, $aqhh4$jsx)("path", {
                        fill: "#292929",
                        fillRule: "evenodd",
                        d: "M37.32 32.07c-2.9 0-5.25 2.35-5.25 5.25v17.073c0 2.9 2.35 5.25 5.25 5.25s5.25-2.35 5.25-5.25V42.57h32.183v74.861H62.928c-2.9 0-5.25 2.35-5.25 5.25s2.35 5.25 5.25 5.25h16.964l.111.001.112-.001h16.958c2.9 0 5.25-2.351 5.25-5.25s-2.35-5.25-5.25-5.25h-11.82v-74.86h32.178v11.822c0 2.9 2.351 5.25 5.25 5.25s5.25-2.35 5.25-5.25V37.408l.001-.088c0-2.9-2.351-5.25-5.25-5.25H37.32"
                    }),
                    /*#__PURE__*/ (0, $aqhh4$jsx)("path", {
                        fill: "#292929",
                        d: "M42.57 42.57v-2c-1.104 0-2 .896-2 2zm32.183 0h2c0-1.104-.895-2-2-2zm0 74.861v2c1.105 0 2-.896 2-2zm5.139 10.5.041-2h-.041zm.223 0v-2h-.042zm5.138-10.5h-2c0 1.104.896 2 2 2zm0-74.86v-2c-1.104 0-2 .895-2 2zm32.178 0h2c0-1.105-.895-2-2-2zm10.5-5.163-1.999-.033-.001.032zm-93.86-.088c0-1.795 1.454-3.25 3.25-3.25v-4c-4.005 0-7.25 3.246-7.25 7.25zm0 17.073V37.32h-4v17.073zm3.25 3.25c-1.796 0-3.25-1.455-3.25-3.25h-4c0 4.004 3.245 7.25 7.25 7.25zm3.25-3.25c0 1.795-1.456 3.25-3.25 3.25v4c4.003 0 7.25-3.246 7.25-7.25zm0-11.823v11.823h4V42.57zm34.182-2H42.57v4h32.183zm2 76.861v-74.86h-4v74.86zm-13.825 2h11.825v-4H62.928zm-3.25 3.25c0-1.795 1.455-3.25 3.25-3.25v-4c-4.004 0-7.25 3.246-7.25 7.25zm3.25 3.25c-1.795 0-3.25-1.455-3.25-3.25h-4c0 4.004 3.246 7.25 7.25 7.25zm16.964 0H62.928v4h16.964zm.111.001-.07-.001-.083 3.999.153.002zm.07-.001-.07.001v4l.153-.002zm17 0H80.115v4h16.958zm3.25-3.25c0 1.795-1.455 3.25-3.25 3.25v4c4.004 0 7.25-3.246 7.25-7.25zm-3.25-3.25c1.795 0 3.25 1.455 3.25 3.25h4c0-4.004-3.246-7.25-7.25-7.25zm-11.82 0h11.82v-4h-11.82zm-2-76.86v74.86h4v-74.86zm34.178-2H85.253v4h32.178zm2 13.822V42.57h-4v11.823zm3.25 3.25c-1.795 0-3.25-1.455-3.25-3.25h-4c0 4.004 3.246 7.25 7.25 7.25zm3.25-3.25c0 1.795-1.455 3.25-3.25 3.25v4c4.004 0 7.25-3.246 7.25-7.25zm0-16.985v16.985h4V37.408zm.001-.088v.055l3.999.065.001-.12zm-3.25-3.25c1.795 0 3.25 1.455 3.25 3.25h4c0-4.004-3.246-7.25-7.25-7.25zm-.001 0h.001v-4h-.001zm-42.678 0h42.678v-4H80.003zm-42.683 0h42.683v-4H37.32z",
                        clipPath: "url(#5a3d02__a)"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aqhh4$jsx)("g", {
                clipPath: "url(#5a3d02__b)",
                children: /*#__PURE__*/ (0, $aqhh4$jsxs)("g", {
                    clipPath: "url(#5a3d02__c)",
                    children: [
                        /*#__PURE__*/ (0, $aqhh4$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $aqhh4$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#5a3d02__d)"
                        }),
                        /*#__PURE__*/ (0, $aqhh4$jsx)("path", {
                            fill: "url(#5a3d02__e)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $aqhh4$jsx)("path", {
                            fill: "url(#5a3d02__f)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $aqhh4$jsx)("path", {
                            fill: "url(#5a3d02__g)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $c1c7125dca9fcf7c$var$ForwardRef = /*#__PURE__*/ (0, $aqhh4$forwardRef)($c1c7125dca9fcf7c$var$SvgComponent);
var $c1c7125dca9fcf7c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c1c7125dca9fcf7c$var$ForwardRef);


export {$c1c7125dca9fcf7c$export$2e2bcd8739ae039 as default};
