// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_information_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3FsdO$reactjsxruntime = require("react/jsx-runtime");
var $3FsdO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $00e5b4b3e17931f5$export$2e2bcd8739ae039);




const $00e5b4b3e17931f5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5858b4__d",
                        x1: 32.783,
                        x2: 12.551,
                        y1: 4.919,
                        y2: 32.745,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5858b4__e",
                        x1: 19.489,
                        x2: 23.239,
                        y1: -7.359,
                        y2: 18.105,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "5858b4__f",
                        x1: -29.084,
                        x2: 41.411,
                        y1: 43.698,
                        y2: 28.919,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsxs)("radialGradient", {
                        id: "5858b4__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.2844 -54.4031 0 39.106 29.411)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("clipPath", {
                        id: "5858b4__b",
                        children: /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.061,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("clipPath", {
                id: "5858b4__a",
                children: /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M40.8 24c0 9.278-7.522 16.8-16.8 16.8-9.28 0-16.8-7.522-16.8-16.8 0-9.28 7.52-16.8 16.8-16.8 9.278 0 16.8 7.52 16.8 16.8m-19.05-4.05c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h1.5v6.9h-1.5c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h5.4c.662 0 1.2-.538 1.2-1.2 0-.663-.538-1.2-1.2-1.2h-1.5v-8.1c0-.663-.538-1.2-1.2-1.2zm2.4-1.5c.828 0 1.5-.672 1.5-1.5 0-.83-.672-1.5-1.5-1.5-.83 0-1.5.67-1.5 1.5 0 .828.67 1.5 1.5 1.5"
                })
            }),
            /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("g", {
                clipPath: "url(#5858b4__a)",
                children: /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#5858b4__b)",
                    children: [
                        /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.061,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("ellipse", {
                            cx: 39.106,
                            cy: 29.411,
                            fill: "url(#5858b4__c)",
                            rx: 54.403,
                            ry: 51.284
                        }),
                        /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("path", {
                            fill: "url(#5858b4__d)",
                            d: "M2.128 17.186C1.523 4.91 11.591-5.506 24.615-6.076s24.072 8.92 24.677 21.198c.605 12.277-9.463 22.692-22.487 23.262S2.733 29.464 2.128 17.186"
                        }),
                        /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("path", {
                            fill: "url(#5858b4__e)",
                            d: "M15.039-18.518c-2.938-1.714-6.66-1.724-9.61-.027L-24.064-1.569c-2.889 1.663-2.893 5.587-.009 7.256l29.3 16.952c2.944 1.704 6.667 1.701 9.609-.006l29.24-16.968c2.869-1.665 2.874-5.568.008-7.24z"
                        }),
                        /*#__PURE__*/ (0, $3FsdO$reactjsxruntime.jsx)("path", {
                            fill: "url(#5858b4__f)",
                            d: "M-13.266 23.05c6.08 0 11.586 2.319 15.573 6.06 7.974 7.497 14.585 7.503 22.559.02 3.987-3.761 9.5-6.087 15.586-6.087 12.166.006 22.02 9.271 22.02 20.708s-9.854 20.695-22.02 20.695c-6.1 0-11.613-2.332-15.6-6.093-7.967-7.471-14.57-7.465-22.545.032-3.987 3.748-9.494 6.061-15.573 6.061-12.158 0-22.013-9.265-22.013-20.695s9.855-20.702 22.013-20.702"
                        })
                    ]
                })
            })
        ]
    });
const $00e5b4b3e17931f5$var$ForwardRef = /*#__PURE__*/ (0, $3FsdO$react.forwardRef)($00e5b4b3e17931f5$var$SvgComponent);
var $00e5b4b3e17931f5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($00e5b4b3e17931f5$var$ForwardRef);
