// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_mailClose_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $1Ww8w$jsxs, jsx as $1Ww8w$jsx} from "react/jsx-runtime";
import {forwardRef as $1Ww8w$forwardRef} from "react";





const $3108f7f3de51c155$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1Ww8w$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1Ww8w$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1Ww8w$jsxs)("linearGradient", {
                        id: "e95a39__d",
                        x1: 65.703,
                        x2: 23.805,
                        y1: 9.034,
                        y2: 66.661,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1Ww8w$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $1Ww8w$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1Ww8w$jsxs)("linearGradient", {
                        id: "e95a39__e",
                        x1: 38.171,
                        x2: 45.938,
                        y1: -16.392,
                        y2: 36.343,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1Ww8w$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $1Ww8w$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1Ww8w$jsxs)("linearGradient", {
                        id: "e95a39__f",
                        x1: -62.418,
                        x2: 83.573,
                        y1: 89.346,
                        y2: 58.738,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1Ww8w$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1Ww8w$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $1Ww8w$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $1Ww8w$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1Ww8w$jsxs)("radialGradient", {
                        id: "e95a39__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 106.207 -112.666 0 78.799 59.76)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1Ww8w$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1Ww8w$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1Ww8w$jsx)("clipPath", {
                        id: "e95a39__b",
                        children: /*#__PURE__*/ (0, $1Ww8w$jsx)("rect", {
                            width: 78.033,
                            height: 73.56,
                            x: 9.242,
                            y: 11.4,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1Ww8w$jsx)("clipPath", {
                id: "e95a39__a",
                children: /*#__PURE__*/ (0, $1Ww8w$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M20.308 20.4h55.384c2.215 0 4.227.96 5.717 2.524L49.536 41.52c-1.118.652-2.493.64-3.6-.034l-31.02-18.885c1.451-1.372 3.334-2.2 5.392-2.2m-7.887 6.301c-.273.912-.421 1.886-.421 2.9v36.8c0 5.08 3.72 9.2 8.308 9.2h55.384c4.589 0 8.308-4.12 8.308-9.2V29.6c0-.851-.104-1.675-.3-2.457L51.956 45.666c-2.641 1.54-5.903 1.51-8.515-.08z"
                })
            }),
            /*#__PURE__*/ (0, $1Ww8w$jsx)("g", {
                clipPath: "url(#e95a39__a)",
                children: /*#__PURE__*/ (0, $1Ww8w$jsxs)("g", {
                    clipPath: "url(#e95a39__b)",
                    children: [
                        /*#__PURE__*/ (0, $1Ww8w$jsx)("rect", {
                            width: 78.033,
                            height: 73.56,
                            x: 9.242,
                            y: 11.4,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $1Ww8w$jsx)("ellipse", {
                            cx: 78.799,
                            cy: 59.76,
                            fill: "url(#e95a39__c)",
                            rx: 112.666,
                            ry: 106.207
                        }),
                        /*#__PURE__*/ (0, $1Ww8w$jsx)("path", {
                            fill: "url(#e95a39__d)",
                            d: "M2.22 34.44C.966 9.014 21.816-12.555 48.787-13.735c26.972-1.18 49.852 18.474 51.105 43.9 1.252 25.425-19.598 46.993-46.57 48.174-26.971 1.18-49.851-18.474-51.104-43.9"
                        }),
                        /*#__PURE__*/ (0, $1Ww8w$jsx)("path", {
                            fill: "url(#e95a39__e)",
                            d: "M28.955-39.5c-6.084-3.55-13.793-3.572-19.9-.057L-52.027-4.4c-5.982 3.443-5.992 11.57-.018 15.027L8.633 45.733c6.098 3.528 13.807 3.524 19.9-.012l60.553-35.14c5.942-3.449 5.953-11.53.018-14.992z"
                        }),
                        /*#__PURE__*/ (0, $1Ww8w$jsx)("path", {
                            fill: "url(#e95a39__f)",
                            d: "M-29.659 46.584c12.59 0 23.994 4.803 32.251 12.552 16.514 15.524 30.204 15.538 46.718.04 8.256-7.79 19.675-12.606 32.279-12.606 25.193.014 45.602 19.2 45.602 42.885s-20.409 42.859-45.602 42.859c-12.632 0-24.05-4.83-32.307-12.619-16.5-15.471-30.176-15.458-46.69.066-8.257 7.763-19.66 12.553-32.25 12.553-25.18 0-45.59-19.187-45.59-42.859 0-23.671 20.41-42.871 45.59-42.871"
                        })
                    ]
                })
            })
        ]
    });
const $3108f7f3de51c155$var$ForwardRef = /*#__PURE__*/ (0, $1Ww8w$forwardRef)($3108f7f3de51c155$var$SvgComponent);
var $3108f7f3de51c155$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($3108f7f3de51c155$var$ForwardRef);


export {$3108f7f3de51c155$export$2e2bcd8739ae039 as default};
