// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_brand_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $cr5V1$reactjsxruntime = require("react/jsx-runtime");
var $cr5V1$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a46f92506653527e$export$2e2bcd8739ae039);




const $a46f92506653527e$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7c864f__d",
                        x1: 65.27,
                        x2: 23.067,
                        y1: 8.417,
                        y2: 66.462,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7c864f__e",
                        x1: 37.538,
                        x2: 45.362,
                        y1: -17.195,
                        y2: 35.923,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7c864f__f",
                        x1: -63.784,
                        x2: 83.268,
                        y1: 89.313,
                        y2: 58.483,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsxs)("radialGradient", {
                        id: "7c864f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 106.979 -113.484 0 78.461 59.512)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("clipPath", {
                        id: "7c864f__b",
                        children: /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 10.801,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("clipPath", {
                id: "7c864f__a",
                children: /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M27.45 15.637h41.1c3.978 0 7.2 3.222 7.2 7.2v29.778c0 15.33-12.42 27.75-27.75 27.75s-27.75-12.426-27.75-27.75V22.837c0-3.978 3.222-7.2 7.2-7.2m2.448 39.45c.377.221.79.323 1.197.324.833-.001 1.636-.433 2.079-1.2l8.376-14.508c.666-1.146.27-2.616-.876-3.276-1.146-.666-2.616-.27-3.276.876L29.022 51.81c-.666 1.146-.27 2.616.876 3.276m31.512.324c1.734 0 3.288-.9 4.158-2.4.87-1.506.87-3.3 0-4.8l-7.542-13.062c-.87-1.5-2.424-2.4-4.158-2.4s-3.288.9-4.158 2.4L42.168 48.21c-.87 1.5-.87 3.3 0 4.8s2.424 2.4 4.158 2.4zm-7.541-17.868L61.41 50.61H46.327z"
                })
            }),
            /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("g", {
                clipPath: "url(#7c864f__a)",
                children: /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#7c864f__b)",
                    children: [
                        /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 10.801,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("ellipse", {
                            cx: 78.461,
                            cy: 59.512,
                            fill: "url(#7c864f__c)",
                            rx: 113.484,
                            ry: 106.979
                        }),
                        /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("path", {
                            fill: "url(#7c864f__d)",
                            d: "M1.325 34.007C.064 8.397 21.065-13.328 48.233-14.518 75.4-15.707 98.447 4.09 99.708 29.701c1.262 25.61-19.74 47.335-46.907 48.524-27.168 1.19-50.214-18.607-51.476-44.218"
                        }),
                        /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("path", {
                            fill: "url(#7c864f__e)",
                            d: "M28.256-40.471c-6.129-3.575-13.894-3.597-20.045-.057L-53.314-5.116C-59.34-1.648-59.35 6.54-53.333 10.02l61.12 35.362c6.141 3.553 13.906 3.549 20.043-.013L88.823 9.974C94.81 6.5 94.82-1.64 88.842-5.127z"
                        }),
                        /*#__PURE__*/ (0, $cr5V1$reactjsxruntime.jsx)("path", {
                            fill: "url(#7c864f__f)",
                            d: "M-30.787 46.24c12.682 0 24.169 4.838 32.485 12.643 16.634 15.638 30.424 15.651 47.058.04 8.316-7.845 19.818-12.696 32.513-12.696 25.377.013 45.934 19.34 45.934 43.196 0 23.857-20.557 43.17-45.934 43.17-12.724 0-24.225-4.865-32.542-12.71-16.62-15.584-30.395-15.571-47.029.066-8.316 7.819-19.803 12.644-32.485 12.644-25.362 0-45.92-19.326-45.92-43.17s20.558-43.183 45.92-43.183"
                        })
                    ]
                })
            })
        ]
    });
const $a46f92506653527e$var$ForwardRef = /*#__PURE__*/ (0, $cr5V1$react.forwardRef)($a46f92506653527e$var$SvgComponent);
var $a46f92506653527e$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a46f92506653527e$var$ForwardRef);
