// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fileImage_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $i7Oqx$jsxs, jsx as $i7Oqx$jsx} from "react/jsx-runtime";
import {forwardRef as $i7Oqx$forwardRef} from "react";





const $4cc8ee625d936b83$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $i7Oqx$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $i7Oqx$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $i7Oqx$jsxs)("linearGradient", {
                        id: "893378__d",
                        x1: 33.654,
                        x2: 9.6,
                        y1: 1.896,
                        y2: 33.083,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i7Oqx$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $i7Oqx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i7Oqx$jsxs)("linearGradient", {
                        id: "893378__e",
                        x1: 18.998,
                        x2: 23.639,
                        y1: -12.461,
                        y2: 17.236,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i7Oqx$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $i7Oqx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i7Oqx$jsxs)("linearGradient", {
                        id: "893378__f",
                        x1: -34.541,
                        x2: 43.532,
                        y1: 47.244,
                        y2: 31.814,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i7Oqx$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $i7Oqx$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $i7Oqx$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $i7Oqx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i7Oqx$jsxs)("radialGradient", {
                        id: "893378__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 59.9687 -59.9688 0 40.625 30.538)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $i7Oqx$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $i7Oqx$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $i7Oqx$jsx)("clipPath", {
                        id: "893378__b",
                        children: /*#__PURE__*/ (0, $i7Oqx$jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.602,
                            y: 3.232,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $i7Oqx$jsx)("clipPath", {
                id: "893378__a",
                children: /*#__PURE__*/ (0, $i7Oqx$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m26.255 9.455 9.891 9.891c.675.675 1.055 1.59 1.055 2.546V36c0 1.989-1.612 3.6-3.6 3.6H14.4c-1.989 0-3.6-1.611-3.6-3.6V12c0-1.988 1.611-3.6 3.6-3.6h9.309c.954 0 1.87.38 2.545 1.055M28.351 20.4c-1.243 0-2.25 1.008-2.25 2.25 0 1.243 1.007 2.25 2.25 2.25 1.242 0 2.25-1.007 2.25-2.25 0-1.242-1.008-2.25-2.25-2.25M16.202 34.8H31.8c.47 0 .757-.514.511-.914l-3.356-5.455c-.234-.38-.788-.38-1.022 0l-2.645 4.299-3.927-6.66c-.233-.392-.8-.392-1.032 0l-4.642 7.824c-.237.4.051.906.516.906"
                })
            }),
            /*#__PURE__*/ (0, $i7Oqx$jsx)("g", {
                clipPath: "url(#893378__a)",
                children: /*#__PURE__*/ (0, $i7Oqx$jsxs)("g", {
                    clipPath: "url(#893378__b)",
                    children: [
                        /*#__PURE__*/ (0, $i7Oqx$jsx)("rect", {
                            width: 41.535,
                            height: 41.535,
                            x: 3.602,
                            y: 3.232,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $i7Oqx$jsx)("circle", {
                            cx: 40.625,
                            cy: 30.538,
                            r: 59.969,
                            fill: "url(#893378__c)"
                        }),
                        /*#__PURE__*/ (0, $i7Oqx$jsx)("path", {
                            fill: "url(#893378__d)",
                            d: "M-.137 16.241C-.804 1.885 10.294-10.294 24.65-10.96S51.185-.53 51.852 13.827c.666 14.356-10.431 26.535-24.788 27.201C12.708 41.695.53 30.598-.137 16.241"
                        }),
                        /*#__PURE__*/ (0, $i7Oqx$jsx)("path", {
                            fill: "url(#893378__e)",
                            d: "M14.093-25.509c-3.238-2.004-7.341-2.016-10.592-.032L-29.011-5.69c-3.184 1.944-3.19 6.533-.01 8.485L3.276 22.617c3.246 1.992 7.35 1.99 10.593-.007l32.23-19.84c3.163-1.948 3.169-6.511.01-8.466z"
                        }),
                        /*#__PURE__*/ (0, $i7Oqx$jsx)("path", {
                            fill: "url(#893378__f)",
                            d: "M-17.104 23.098c6.701 0 12.771 2.713 17.166 7.088 8.79 8.766 16.077 8.773 24.867.022 4.395-4.397 10.472-7.117 17.18-7.117 13.41.007 24.274 10.84 24.274 24.214s-10.863 24.2-24.273 24.2c-6.724 0-12.801-2.727-17.196-7.125-8.783-8.736-16.062-8.728-24.852.037-4.395 4.383-10.465 7.088-17.166 7.088-13.402 0-24.265-10.834-24.265-24.2s10.863-24.207 24.265-24.207"
                        })
                    ]
                })
            })
        ]
    });
const $4cc8ee625d936b83$var$ForwardRef = /*#__PURE__*/ (0, $i7Oqx$forwardRef)($4cc8ee625d936b83$var$SvgComponent);
var $4cc8ee625d936b83$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($4cc8ee625d936b83$var$ForwardRef);


export {$4cc8ee625d936b83$export$2e2bcd8739ae039 as default};
