// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_imageStack_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $9esAn$jsxs, jsx as $9esAn$jsx} from "react/jsx-runtime";
import {forwardRef as $9esAn$forwardRef} from "react";





const $1832face75489d38$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9esAn$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9esAn$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9esAn$jsxs)("linearGradient", {
                        id: "db5bf7__d",
                        x1: 68.169,
                        x2: 25.773,
                        y1: 9.829,
                        y2: 68.139,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9esAn$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $9esAn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9esAn$jsxs)("linearGradient", {
                        id: "db5bf7__e",
                        x1: 40.311,
                        x2: 48.17,
                        y1: -15.9,
                        y2: 37.461,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9esAn$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $9esAn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9esAn$jsxs)("linearGradient", {
                        id: "db5bf7__f",
                        x1: -61.47,
                        x2: 86.255,
                        y1: 91.094,
                        y2: 60.123,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9esAn$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9esAn$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $9esAn$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $9esAn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9esAn$jsxs)("radialGradient", {
                        id: "db5bf7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 107.468 -114.004 0 81.421 61.156)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9esAn$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9esAn$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9esAn$jsx)("clipPath", {
                        id: "db5bf7__b",
                        children: /*#__PURE__*/ (0, $9esAn$jsx)("rect", {
                            width: 78.96,
                            height: 74.433,
                            x: 11.039,
                            y: 12.223,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9esAn$jsx)("clipPath", {
                id: "db5bf7__a",
                children: /*#__PURE__*/ (0, $9esAn$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M75.256 29.082v29.4c0 5.304-4.296 9.6-9.6 9.6h-44.4c-5.304 0-9.6-4.296-9.6-9.6v-29.4c0-5.304 4.296-9.6 9.6-9.6h44.4c5.304 0 9.6 4.296 9.6 9.6m-8.424 4.128c0-2.892-2.346-5.238-5.238-5.238s-5.238 2.346-5.238 5.238 2.346 5.238 5.238 5.238 5.238-2.346 5.238-5.238M17.386 57.732c-1.536 2.394.18 5.544 3.03 5.544v-.006h45.708c2.85 0 4.566-3.15 3.03-5.544l-7.068-11.01c-1.416-2.208-4.644-2.208-6.06 0L48.01 59.208 35.254 39.336c-1.416-2.208-4.644-2.208-6.06 0zm67.025-15.79c0-1.104-.896-2-2-2s-2 .896-2 2v22.014c0 4.899-3.972 8.87-8.87 8.87H35.666c-1.105 0-2 .896-2 2s.895 2 2 2H71.54c7.108 0 12.87-5.762 12.87-12.87z"
                })
            }),
            /*#__PURE__*/ (0, $9esAn$jsx)("g", {
                clipPath: "url(#db5bf7__a)",
                children: /*#__PURE__*/ (0, $9esAn$jsxs)("g", {
                    clipPath: "url(#db5bf7__b)",
                    children: [
                        /*#__PURE__*/ (0, $9esAn$jsx)("rect", {
                            width: 78.96,
                            height: 74.433,
                            x: 11.039,
                            y: 12.223,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $9esAn$jsx)("ellipse", {
                            cx: 81.421,
                            cy: 61.156,
                            fill: "url(#db5bf7__c)",
                            rx: 114.004,
                            ry: 107.468
                        }),
                        /*#__PURE__*/ (0, $9esAn$jsx)("path", {
                            fill: "url(#db5bf7__d)",
                            d: "M3.931 35.536C2.664 9.807 23.761-12.017 51.053-13.211s50.444 18.693 51.711 44.42c1.267 25.728-19.83 47.552-47.122 48.747S5.198 61.263 3.932 35.536"
                        }),
                        /*#__PURE__*/ (0, $9esAn$jsx)("path", {
                            fill: "url(#db5bf7__e)",
                            d: "M30.986-39.283c-6.157-3.592-13.957-3.614-20.137-.057L-50.958-3.766c-6.053 3.484-6.063 11.708-.018 15.205l61.398 35.524c6.17 3.57 13.971 3.565 20.136-.013L91.83 11.393c6.013-3.49 6.024-11.667.019-15.17z"
                        }),
                        /*#__PURE__*/ (0, $9esAn$jsx)("path", {
                            fill: "url(#db5bf7__f)",
                            d: "M-28.322 47.824c12.74 0 24.279 4.86 32.634 12.701 16.71 15.71 30.563 15.723 47.272.04 8.355-7.88 19.909-12.754 32.662-12.754 25.493.013 46.145 19.428 46.145 43.394s-20.652 43.367-46.144 43.367c-12.783 0-24.337-4.887-32.691-12.769-16.696-15.655-30.535-15.641-47.244.068-8.355 7.854-19.895 12.701-32.634 12.701-25.478 0-46.13-19.415-46.13-43.367 0-23.953 20.652-43.381 46.13-43.381"
                        })
                    ]
                })
            })
        ]
    });
const $1832face75489d38$var$ForwardRef = /*#__PURE__*/ (0, $9esAn$forwardRef)($1832face75489d38$var$SvgComponent);
var $1832face75489d38$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($1832face75489d38$var$ForwardRef);


export {$1832face75489d38$export$2e2bcd8739ae039 as default};
