// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_chartAreaStack_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $cJAgw$reactjsxruntime = require("react/jsx-runtime");
var $cJAgw$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7810e02fa9ffbc37$export$2e2bcd8739ae039);




const $7810e02fa9ffbc37$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a480ad__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a480ad__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a480ad__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a480ad__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("clipPath", {
                        id: "a480ad__b",
                        children: /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("clipPath", {
                id: "a480ad__a",
                children: /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("path", {
                    fill: "#8F8F8F",
                    fillRule: "evenodd",
                    d: "M35.743 11.242c1.195-2.164 4.497-1.323 4.497 1.157v23.697c0 1.958-1.59 3.542-3.545 3.542h-25.15C9.59 39.638 8 38.054 8 36.096V23.569c0-.66.273-1.291.753-1.744l4.887-4.602c.993-.935 2.565-.855 3.458.175l3.444 3.974 4.546-7.02c.883-1.364 2.84-1.473 3.869-.216l3.098 3.784zM20.468 21.486Zm17.272-8.69-3.57 6.464-1.094-.604 1.094.604c-.82 1.485-2.88 1.671-3.955.36l-3.104-3.792-4.545 7.017c-.868 1.34-2.78 1.473-3.825.266l-3.463-3.996-4.778 4.5v6.68l4.1-2.637c.969-.623 2.224-.573 3.14.126l3.88 2.96c.205.156.495.125.662-.07l4.92-5.712c1.037-1.206 2.881-1.275 4.007-.151l3.55 3.544 2.981-1.427zm0 16.904-2.699 1.292c-.478.229-1.048.131-1.423-.243l-4.175-4.169c-.098-.097-.258-.091-.347.013l-4.92 5.713c-1.03 1.197-2.815 1.384-4.071.426l-3.88-2.96c-.08-.06-.189-.065-.273-.01L10.5 33.267v2.828c0 .573.466 1.042 1.046 1.042h25.149c.579 0 1.045-.469 1.045-1.042zm-27.273-6.055-.072-.076zm16.72-7.934-.086-.055z"
                })
            }),
            /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("g", {
                clipPath: "url(#a480ad__a)",
                children: /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#a480ad__b)",
                    children: [
                        /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#a480ad__c)"
                        }),
                        /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("path", {
                            fill: "url(#a480ad__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("path", {
                            fill: "url(#a480ad__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $cJAgw$reactjsxruntime.jsx)("path", {
                            fill: "url(#a480ad__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $7810e02fa9ffbc37$var$ForwardRef = /*#__PURE__*/ (0, $cJAgw$react.forwardRef)($7810e02fa9ffbc37$var$SvgComponent);
var $7810e02fa9ffbc37$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7810e02fa9ffbc37$var$ForwardRef);
