// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_artboard_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $iQneE$jsxs, jsx as $iQneE$jsx} from "react/jsx-runtime";
import {forwardRef as $iQneE$forwardRef} from "react";





const $f87806a3423ee80b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iQneE$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iQneE$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iQneE$jsxs)("linearGradient", {
                        id: "71d067__d",
                        x1: 62.402,
                        x2: 17.427,
                        y1: 3.461,
                        y2: 65.318,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iQneE$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $iQneE$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iQneE$jsxs)("linearGradient", {
                        id: "71d067__e",
                        x1: 32.848,
                        x2: 41.186,
                        y1: -23.833,
                        y2: 32.773,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iQneE$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $iQneE$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iQneE$jsxs)("linearGradient", {
                        id: "71d067__f",
                        x1: -75.123,
                        x2: 81.586,
                        y1: 89.668,
                        y2: 56.813,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iQneE$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iQneE$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $iQneE$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $iQneE$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iQneE$jsxs)("radialGradient", {
                        id: "71d067__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 114.004 -120.937 0 76.46 57.91)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iQneE$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iQneE$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iQneE$jsx)("clipPath", {
                        id: "71d067__b",
                        children: /*#__PURE__*/ (0, $iQneE$jsx)("rect", {
                            width: 83.762,
                            height: 78.96,
                            x: 1.797,
                            y: 6,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iQneE$jsx)("clipPath", {
                id: "71d067__a",
                children: /*#__PURE__*/ (0, $iQneE$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M24 14.4v-6C24 7.074 22.926 6 21.6 6s-2.4 1.074-2.4 2.4v6c0 1.326 1.074 2.4 2.4 2.4s2.4-1.074 2.4-2.4m23.4 27.612c0 3.3 2.7 5.988 6.018 5.988h20.988v28.8c0 3.978-3.204 7.2-7.158 7.2h-38.49c-3.954 0-7.158-3.222-7.158-7.2v-48c0-3.978 3.204-7.2 7.158-7.2H47.4zM72.78 43.2H53.424c-.672 0-1.218-.534-1.218-1.188V23.334l.093.08q.106.086.207.184l19.686 18.954c.21.204.402.42.588.648M8.4 19.2h6c1.326 0 2.4 1.074 2.4 2.4S15.726 24 14.4 24h-6C7.074 24 6 22.926 6 21.6s1.074-2.4 2.4-2.4"
                })
            }),
            /*#__PURE__*/ (0, $iQneE$jsx)("g", {
                clipPath: "url(#71d067__a)",
                children: /*#__PURE__*/ (0, $iQneE$jsxs)("g", {
                    clipPath: "url(#71d067__b)",
                    children: [
                        /*#__PURE__*/ (0, $iQneE$jsx)("rect", {
                            width: 83.762,
                            height: 78.96,
                            x: 1.797,
                            y: 6,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $iQneE$jsx)("ellipse", {
                            cx: 76.46,
                            cy: 57.91,
                            fill: "url(#71d067__c)",
                            rx: 120.937,
                            ry: 114.004
                        }),
                        /*#__PURE__*/ (0, $iQneE$jsx)("path", {
                            fill: "url(#71d067__d)",
                            d: "M-5.742 30.731C-7.087 3.44 15.294-19.713 44.245-20.979 73.197-22.248 97.757-1.15 99.101 26.141S78.065 76.586 49.114 77.853C20.162 79.121-4.398 58.023-5.742 30.731"
                        }),
                        /*#__PURE__*/ (0, $iQneE$jsx)("path", {
                            fill: "url(#71d067__e)",
                            d: "M22.957-48.638c-6.532-3.81-14.807-3.833-21.362-.06L-63.97-10.961c-6.422 3.696-6.433 12.42-.02 16.13L1.142 42.853c6.546 3.786 14.82 3.782 21.36-.014L87.502 5.12c6.379-3.702 6.39-12.377.02-16.093z"
                        }),
                        /*#__PURE__*/ (0, $iQneE$jsx)("path", {
                            fill: "url(#71d067__f)",
                            d: "M-39.958 43.766c13.514 0 25.755 5.156 34.618 13.474 17.726 16.664 32.422 16.678 50.147.043 8.863-8.36 21.12-13.531 34.649-13.531 27.043.014 48.95 20.61 48.95 46.033s-21.907 46.005-48.95 46.005c-13.56 0-25.816-5.185-34.679-13.545-17.71-16.608-32.391-16.593-50.117.071-8.863 8.332-21.104 13.474-34.618 13.474-27.028 0-48.935-20.596-48.935-46.005s21.907-46.019 48.935-46.019"
                        })
                    ]
                })
            })
        ]
    });
const $f87806a3423ee80b$var$ForwardRef = /*#__PURE__*/ (0, $iQneE$forwardRef)($f87806a3423ee80b$var$SvgComponent);
var $f87806a3423ee80b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($f87806a3423ee80b$var$ForwardRef);


export {$f87806a3423ee80b$export$2e2bcd8739ae039 as default};
