// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_crop_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $7dkuj$reactjsxruntime = require("react/jsx-runtime");
var $7dkuj$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7f4210d2f3826c71$export$2e2bcd8739ae039);




const $7f4210d2f3826c71$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 97,
        height: 96,
        viewBox: "0 0 97 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "eb6bda__d",
                        x1: 70.403,
                        x2: 14.807,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "eb6bda__e",
                        x1: 36.531,
                        x2: 47.256,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "eb6bda__f",
                        x1: -87.219,
                        x2: 93.232,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsxs)("radialGradient", {
                        id: "eb6bda__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.7 74.813)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("clipPath", {
                        id: "eb6bda__b",
                        children: /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("clipPath", {
                id: "eb6bda__a",
                children: /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M32.916 15c0-1.105-.896-2-2-2s-2 .895-2 2v12.903H15.94c-1.104 0-2 .896-2 2s.896 2 2 2h12.975v26.193c0 5.523 4.477 10 10 10h26.05v12.903c0 1.105.896 2 2 2s2-.895 2-2V68.096h12.975c1.104 0 2-.895 2-2s-.896-2-2-2H38.916c-3.314 0-6-2.686-6-6V15m5.441 12.903c-1.104 0-2 .896-2 2s.896 2 2 2h20.61c3.313 0 6 2.687 6 6v20.742c0 1.104.895 2 2 2s2-.896 2-2V37.903c0-5.523-4.478-10-10-10z"
                })
            }),
            /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("g", {
                clipPath: "url(#eb6bda__a)",
                children: /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#eb6bda__b)",
                    children: [
                        /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 0.941,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("circle", {
                            cx: 86.513,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#eb6bda__c)"
                        }),
                        /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("path", {
                            fill: "url(#eb6bda__d)",
                            d: "M-7.698 30.067c-1.54-33.182 24.11-61.33 57.291-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.541 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.87-57.291"
                        }),
                        /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("path", {
                            fill: "url(#eb6bda__e)",
                            d: "M25.194-66.43c-7.485-4.632-16.97-4.66-24.482-.073l-75.146 45.881c-7.36 4.494-7.372 15.1-.022 19.61L.193 44.806c7.502 4.604 16.986 4.598 24.481-.016L99.17-1.071c7.311-4.5 7.324-15.048.023-19.566z"
                        }),
                        /*#__PURE__*/ (0, $7dkuj$reactjsxruntime.jsx)("path", {
                            fill: "url(#eb6bda__f)",
                            d: "M-46.917 45.918c15.488 0 29.518 6.268 39.676 16.381 20.316 20.26 37.159 20.278 57.474.052 10.158-10.165 24.206-16.45 39.711-16.45 30.995.017 56.103 25.056 56.103 55.967S120.939 157.8 89.944 157.8c-15.54 0-29.588-6.303-39.745-16.468-20.299-20.191-37.124-20.174-57.44.087-10.158 10.13-24.188 16.381-39.676 16.381-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $7f4210d2f3826c71$var$ForwardRef = /*#__PURE__*/ (0, $7dkuj$react.forwardRef)($7f4210d2f3826c71$var$SvgComponent);
var $7f4210d2f3826c71$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7f4210d2f3826c71$var$ForwardRef);
