// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_laptop_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $2OLth$jsxs, jsx as $2OLth$jsx} from "react/jsx-runtime";
import {forwardRef as $2OLth$forwardRef} from "react";





const $a913778df922eb62$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $2OLth$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $2OLth$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $2OLth$jsxs)("linearGradient", {
                        id: "a70e0d__d",
                        x1: 33.419,
                        x2: 11.512,
                        y1: 4.253,
                        y2: 34.383,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2OLth$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $2OLth$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2OLth$jsxs)("linearGradient", {
                        id: "a70e0d__e",
                        x1: 19.025,
                        x2: 23.086,
                        y1: -9.042,
                        y2: 18.53,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2OLth$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $2OLth$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2OLth$jsxs)("linearGradient", {
                        id: "a70e0d__f",
                        x1: -33.568,
                        x2: 42.764,
                        y1: 46.244,
                        y2: 30.241,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2OLth$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $2OLth$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $2OLth$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $2OLth$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2OLth$jsxs)("radialGradient", {
                        id: "a70e0d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 55.5309 -58.9078 0 40.267 30.774)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $2OLth$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $2OLth$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2OLth$jsx)("clipPath", {
                        id: "a70e0d__b",
                        children: /*#__PURE__*/ (0, $2OLth$jsx)("rect", {
                            width: 40.8,
                            height: 38.461,
                            x: 3.898,
                            y: 5.489,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $2OLth$jsx)("clipPath", {
                id: "a70e0d__a",
                children: /*#__PURE__*/ (0, $2OLth$jsx)("path", {
                    fill: "#fff",
                    d: "M40.8 33h-2.4V14.466c0-2.688-2.148-4.866-4.8-4.866H14.4c-2.652 0-4.8 2.178-4.8 4.866V33H7.2c-1.491 0-2.7 1.209-2.7 2.7 0 1.49 1.209 2.7 2.7 2.7h33.6c1.491 0 2.7-1.21 2.7-2.7S42.291 33 40.8 33M24 14.7c-.663 0-1.2-.537-1.2-1.2s.537-1.2 1.2-1.2 1.2.537 1.2 1.2-.537 1.2-1.2 1.2"
                })
            }),
            /*#__PURE__*/ (0, $2OLth$jsx)("g", {
                clipPath: "url(#a70e0d__a)",
                children: /*#__PURE__*/ (0, $2OLth$jsxs)("g", {
                    clipPath: "url(#a70e0d__b)",
                    children: [
                        /*#__PURE__*/ (0, $2OLth$jsx)("rect", {
                            width: 40.8,
                            height: 38.461,
                            x: 3.898,
                            y: 5.489,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $2OLth$jsx)("ellipse", {
                            cx: 40.267,
                            cy: 30.774,
                            fill: "url(#a70e0d__c)",
                            rx: 58.908,
                            ry: 55.531
                        }),
                        /*#__PURE__*/ (0, $2OLth$jsx)("path", {
                            fill: "url(#a70e0d__d)",
                            d: "M.226 17.536C-.429 4.242 10.473-7.035 24.575-7.652 38.677-8.27 50.64 2.007 51.295 15.3S41.048 39.871 26.946 40.49.881 30.829.226 17.536"
                        }),
                        /*#__PURE__*/ (0, $2OLth$jsx)("path", {
                            fill: "url(#a70e0d__e)",
                            d: "M14.207-21.125c-3.182-1.855-7.212-1.867-10.406-.029L-28.135-2.772c-3.128 1.8-3.133 6.05-.01 7.857L3.581 23.44c3.188 1.845 7.219 1.842 10.405-.006L45.646 5.06c3.107-1.803 3.113-6.03.01-7.84z"
                        }),
                        /*#__PURE__*/ (0, $2OLth$jsx)("path", {
                            fill: "url(#a70e0d__f)",
                            d: "M-16.44 23.886c6.583 0 12.546 2.511 16.863 6.563 8.634 8.117 15.792 8.124 24.427.02 4.317-4.072 10.287-6.59 16.877-6.59C54.899 23.886 65.57 33.918 65.57 46.3s-10.67 22.41-23.843 22.41c-6.605 0-12.575-2.526-16.892-6.599-8.627-8.089-15.778-8.082-24.412.035-4.317 4.059-10.28 6.563-16.862 6.563-13.166 0-23.837-10.032-23.837-22.408s10.671-22.416 23.837-22.416"
                        })
                    ]
                })
            })
        ]
    });
const $a913778df922eb62$var$ForwardRef = /*#__PURE__*/ (0, $2OLth$forwardRef)($a913778df922eb62$var$SvgComponent);
var $a913778df922eb62$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($a913778df922eb62$var$ForwardRef);


export {$a913778df922eb62$export$2e2bcd8739ae039 as default};
