// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_trophy_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $9sKcu$reactjsxruntime = require("react/jsx-runtime");
var $9sKcu$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $0a823444820d8aec$export$2e2bcd8739ae039);




const $0a823444820d8aec$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9b2c92__d",
                        x1: 65.27,
                        x2: 23.067,
                        y1: 8.417,
                        y2: 66.462,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9b2c92__e",
                        x1: 37.538,
                        x2: 45.362,
                        y1: -17.195,
                        y2: 35.923,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsxs)("linearGradient", {
                        id: "9b2c92__f",
                        x1: -63.784,
                        x2: 83.268,
                        y1: 89.313,
                        y2: 58.483,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsxs)("radialGradient", {
                        id: "9b2c92__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 106.979 -113.484 0 78.461 59.512)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("clipPath", {
                        id: "9b2c92__b",
                        children: /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 10.801,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("clipPath", {
                id: "9b2c92__a",
                children: /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M75.622 24H64.798v-3.6c0-1.985-1.614-3.6-3.6-3.6h-26.4c-1.986 0-3.6 1.615-3.6 3.6V24h-10.8c-3.306 0-6 2.695-6 6v1.2c0 9.265 7.536 16.8 16.8 16.8h3.018c2.604 3.733 6.69 6.36 11.4 7.027 0 .06-.018.114-.018.174v14.652c-4.302.81-7.842 3.744-9.504 7.674-.336.792.252 1.668 1.11 1.668h21.588c.858 0 1.446-.876 1.11-1.668-1.662-3.93-5.202-6.864-9.504-7.674V55.2c0-.06-.012-.114-.018-.174 4.716-.672 8.796-3.294 11.4-7.026h3.036c9.264 0 16.8-7.536 16.8-16.8V30c0-3.306-2.694-6-6-6zM31.198 43.2c-6.618 0-12-5.381-12-12V30c0-.66.54-1.2 1.2-1.2h10.8v9.6c0 1.669.246 3.277.702 4.8zm24.474-8.897-2.238 2.172c-.306.3-.45.732-.372 1.152l.534 3.072c.204 1.188-1.044 2.1-2.112 1.536l-2.76-1.458c-.378-.198-.834-.198-1.212 0l-2.76 1.458c-1.068.564-2.316-.342-2.112-1.536l.534-3.072c.072-.42-.066-.852-.372-1.152l-2.238-2.172c-.864-.84-.39-2.31.804-2.484l3.09-.444c.426-.06.792-.33.984-.714l1.374-2.802c.534-1.086 2.076-1.086 2.61 0L50.8 30.66c.192.384.558.654.984.714l3.09.444c1.194.174 1.674 1.644.804 2.484zm21.15-3.102c0 6.618-5.382 12-12 12h-.72c.45-1.524.702-3.132.702-4.8v-9.6h10.824c.66 0 1.2.54 1.2 1.2v1.2z"
                })
            }),
            /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("g", {
                clipPath: "url(#9b2c92__a)",
                children: /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#9b2c92__b)",
                    children: [
                        /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("rect", {
                            width: 78.6,
                            height: 74.094,
                            x: 8.398,
                            y: 10.801,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("ellipse", {
                            cx: 78.461,
                            cy: 59.512,
                            fill: "url(#9b2c92__c)",
                            rx: 113.484,
                            ry: 106.979
                        }),
                        /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("path", {
                            fill: "url(#9b2c92__d)",
                            d: "M1.325 34.007C.064 8.397 21.065-13.328 48.233-14.518 75.4-15.707 98.447 4.09 99.708 29.701c1.262 25.61-19.74 47.335-46.907 48.524-27.168 1.19-50.214-18.607-51.476-44.218"
                        }),
                        /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("path", {
                            fill: "url(#9b2c92__e)",
                            d: "M28.256-40.471c-6.129-3.575-13.894-3.597-20.045-.057L-53.314-5.116C-59.34-1.648-59.35 6.54-53.333 10.02l61.12 35.362c6.141 3.553 13.906 3.549 20.043-.013L88.823 9.974C94.81 6.5 94.82-1.64 88.842-5.127z"
                        }),
                        /*#__PURE__*/ (0, $9sKcu$reactjsxruntime.jsx)("path", {
                            fill: "url(#9b2c92__f)",
                            d: "M-30.787 46.24c12.682 0 24.169 4.838 32.485 12.643 16.634 15.638 30.424 15.651 47.058.04 8.316-7.845 19.818-12.696 32.513-12.696 25.377.013 45.934 19.34 45.934 43.196 0 23.857-20.557 43.17-45.934 43.17-12.724 0-24.225-4.865-32.542-12.71-16.62-15.584-30.395-15.571-47.029.066-8.316 7.819-19.803 12.644-32.485 12.644-25.362 0-45.92-19.326-45.92-43.17s20.558-43.183 45.92-43.183"
                        })
                    ]
                })
            })
        ]
    });
const $0a823444820d8aec$var$ForwardRef = /*#__PURE__*/ (0, $9sKcu$react.forwardRef)($0a823444820d8aec$var$SvgComponent);
var $0a823444820d8aec$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($0a823444820d8aec$var$ForwardRef);
