// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fileVideo_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $fs8Z2$jsxs, jsx as $fs8Z2$jsx} from "react/jsx-runtime";
import {forwardRef as $fs8Z2$forwardRef} from "react";





const $eee0bc866d4c57c7$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $fs8Z2$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $fs8Z2$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $fs8Z2$jsxs)("linearGradient", {
                        id: "9ccfeb__d",
                        x1: 111.107,
                        x2: 30.526,
                        y1: 5.955,
                        y2: 110.43,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fs8Z2$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $fs8Z2$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fs8Z2$jsxs)("linearGradient", {
                        id: "9ccfeb__e",
                        x1: 62.011,
                        x2: 77.555,
                        y1: -42.141,
                        y2: 57.344,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fs8Z2$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $fs8Z2$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fs8Z2$jsxs)("linearGradient", {
                        id: "9ccfeb__f",
                        x1: -117.348,
                        x2: 144.197,
                        y1: 157.869,
                        y2: 106.178,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fs8Z2$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $fs8Z2$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $fs8Z2$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $fs8Z2$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fs8Z2$jsxs)("radialGradient", {
                        id: "9ccfeb__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.277 118.181)scale(200.895)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $fs8Z2$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $fs8Z2$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fs8Z2$jsx)("clipPath", {
                        id: "9ccfeb__b",
                        children: /*#__PURE__*/ (0, $fs8Z2$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.43,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $fs8Z2$jsx)("clipPath", {
                id: "9ccfeb__a",
                children: /*#__PURE__*/ (0, $fs8Z2$jsx)("path", {
                    fill: "#fff",
                    d: "M120.689 64.408 87.554 31.273c-2.262-2.262-5.33-3.533-8.528-3.533H47.84c-6.66 0-12.06 5.4-12.06 12.06v80.4c0 6.661 5.4 12.06 12.06 12.06h64.32c6.661 0 12.06-5.399 12.06-12.06V72.936c0-3.199-1.27-6.266-3.532-8.528M95.072 93.315l-23.946 13.787c-2.682 1.544-6.03-.394-6.026-3.489l.033-27.632c.004-3.094 3.356-5.024 6.034-3.474L95.08 86.352c2.678 1.55 2.674 5.418-.008 6.963"
                })
            }),
            /*#__PURE__*/ (0, $fs8Z2$jsx)("g", {
                clipPath: "url(#9ccfeb__a)",
                children: /*#__PURE__*/ (0, $fs8Z2$jsxs)("g", {
                    clipPath: "url(#9ccfeb__b)",
                    children: [
                        /*#__PURE__*/ (0, $fs8Z2$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.43,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $fs8Z2$jsx)("circle", {
                            cx: 134.458,
                            cy: 101.904,
                            r: 200.895,
                            fill: "url(#9ccfeb__c)"
                        }),
                        /*#__PURE__*/ (0, $fs8Z2$jsx)("path", {
                            fill: "url(#9ccfeb__d)",
                            d: "M-2.092 54.01C-4.325 5.917 32.852-34.88 80.945-37.114S169.837-2.17 172.07 45.923s-34.944 88.891-83.038 91.124C40.94 139.281.142 102.104-2.092 54.01"
                        }),
                        /*#__PURE__*/ (0, $fs8Z2$jsx)("path", {
                            fill: "url(#9ccfeb__e)",
                            d: "M45.579-85.852c-10.85-6.714-24.595-6.755-35.485-.106l-108.915 66.5c-10.667 6.513-10.685 21.886-.032 28.424L9.341 75.37c10.873 6.673 24.62 6.665 35.483-.023L152.798 8.88c10.596-6.524 10.615-21.811.032-28.36z"
                        }),
                        /*#__PURE__*/ (0, $fs8Z2$jsx)("path", {
                            fill: "url(#9ccfeb__f)",
                            d: "M-58.935 76.982c22.45 0 42.784 9.086 57.507 23.743 29.445 29.366 53.857 29.391 83.303.076 14.722-14.733 35.082-23.844 57.557-23.844 44.923.025 81.314 36.318 81.314 81.119s-36.391 81.068-81.314 81.068c-22.525 0-42.885-9.136-57.608-23.869-29.42-29.265-53.807-29.24-83.252.126-14.723 14.682-35.058 23.743-57.507 23.743-44.897 0-81.289-36.293-81.289-81.068 0-44.776 36.392-81.094 81.29-81.094"
                        })
                    ]
                })
            })
        ]
    });
const $eee0bc866d4c57c7$var$ForwardRef = /*#__PURE__*/ (0, $fs8Z2$forwardRef)($eee0bc866d4c57c7$var$SvgComponent);
var $eee0bc866d4c57c7$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($eee0bc866d4c57c7$var$ForwardRef);


export {$eee0bc866d4c57c7$export$2e2bcd8739ae039 as default};
