// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_revert_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6dOKN$jsxs, jsx as $6dOKN$jsx} from "react/jsx-runtime";
import {forwardRef as $6dOKN$forwardRef} from "react";





const $4fb7f3d16105e1a5$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6dOKN$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6dOKN$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6dOKN$jsxs)("linearGradient", {
                        id: "345c3c__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6dOKN$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6dOKN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6dOKN$jsxs)("linearGradient", {
                        id: "345c3c__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6dOKN$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6dOKN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6dOKN$jsxs)("linearGradient", {
                        id: "345c3c__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6dOKN$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6dOKN$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6dOKN$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6dOKN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6dOKN$jsxs)("radialGradient", {
                        id: "345c3c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6dOKN$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6dOKN$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6dOKN$jsx)("clipPath", {
                        id: "345c3c__b",
                        children: /*#__PURE__*/ (0, $6dOKN$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6dOKN$jsx)("clipPath", {
                id: "345c3c__a",
                children: /*#__PURE__*/ (0, $6dOKN$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m32.525 90.528 15.749-13.24c1.69-1.422 4.214-1.203 5.636.488 1.421 1.69 1.203 4.214-.488 5.635l-22.86 19.218c-1.692 1.422-4.215 1.204-5.636-.487L5.708 79.282c-1.422-1.692-1.203-4.215.488-5.636 1.69-1.422 4.214-1.204 5.635.487l12.744 15.159C26.309 57.7 52.475 32.617 84.5 32.617c33.147 0 60.018 26.87 60.018 60.018 0 2.209-1.791 4-4 4s-4-1.791-4-4c0-28.729-23.289-52.018-52.018-52.018-28.022 0-50.87 22.159-51.975 49.91m-10.983 32.855c0-2.209 1.79-4 4-4h117.884c2.21 0 4 1.791 4 4s-1.79 4-4 4H25.542c-2.21 0-4-1.791-4-4"
                })
            }),
            /*#__PURE__*/ (0, $6dOKN$jsx)("g", {
                clipPath: "url(#345c3c__a)",
                children: /*#__PURE__*/ (0, $6dOKN$jsxs)("g", {
                    clipPath: "url(#345c3c__b)",
                    children: [
                        /*#__PURE__*/ (0, $6dOKN$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6dOKN$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#345c3c__c)"
                        }),
                        /*#__PURE__*/ (0, $6dOKN$jsx)("path", {
                            fill: "url(#345c3c__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $6dOKN$jsx)("path", {
                            fill: "url(#345c3c__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $6dOKN$jsx)("path", {
                            fill: "url(#345c3c__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $4fb7f3d16105e1a5$var$ForwardRef = /*#__PURE__*/ (0, $6dOKN$forwardRef)($4fb7f3d16105e1a5$var$SvgComponent);
var $4fb7f3d16105e1a5$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($4fb7f3d16105e1a5$var$ForwardRef);


export {$4fb7f3d16105e1a5$export$2e2bcd8739ae039 as default};
