// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_cardTapPayment_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $5QV7f$jsxs, jsx as $5QV7f$jsx} from "react/jsx-runtime";
import {forwardRef as $5QV7f$forwardRef} from "react";





const $83620ec60270ade3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $5QV7f$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $5QV7f$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $5QV7f$jsxs)("linearGradient", {
                        id: "3cb847__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5QV7f$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $5QV7f$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5QV7f$jsxs)("linearGradient", {
                        id: "3cb847__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5QV7f$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $5QV7f$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5QV7f$jsxs)("linearGradient", {
                        id: "3cb847__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5QV7f$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $5QV7f$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $5QV7f$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $5QV7f$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5QV7f$jsxs)("radialGradient", {
                        id: "3cb847__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $5QV7f$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $5QV7f$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $5QV7f$jsx)("clipPath", {
                        id: "3cb847__b",
                        children: /*#__PURE__*/ (0, $5QV7f$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $5QV7f$jsx)("clipPath", {
                id: "3cb847__a",
                children: /*#__PURE__*/ (0, $5QV7f$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M34.79 7.126c.083-.685.707-1.173 1.392-1.09 3.351.41 6.481 2.335 8.301 5.488 1.836 3.179 1.925 6.886.57 10.009-.276.633-1.012.924-1.645.649s-.924-1.011-.65-1.644c1.054-2.428.983-5.3-.44-7.764-1.41-2.444-3.834-3.938-6.439-4.256-.685-.083-1.173-.707-1.09-1.392m-1.133 4.606c.084-.685.707-1.172 1.393-1.089 2.065.252 3.997 1.44 5.12 3.385 1.132 1.96 1.186 4.248.35 6.173-.274.633-1.01.923-1.644.649-.633-.275-.924-1.012-.649-1.645.534-1.23.497-2.682-.222-3.927-.714-1.236-1.939-1.992-3.258-2.153-.686-.084-1.173-.707-1.09-1.393M5.27 22.2c-1.913 1.104-2.568 3.55-1.464 5.464l.485.84 26.985-15.56-.493-.855c-1.105-1.913-3.551-2.569-5.465-1.464zm27.256-7.09L5.541 30.67l4.648 8.05c1.105 1.913 3.551 2.569 5.464 1.464l20.049-11.575c1.913-1.105 2.569-3.55 1.464-5.464zM10.58 31.923c-.346-.598-.14-1.362.457-1.707l9.468-5.467c.598-.346 1.363-.14 1.708.457.345.598.14 1.362-.458 1.708l-9.468 5.467c-.598.345-1.362.14-1.707-.458m2.88 2.49c-.598.345-.802 1.11-.457 1.708s1.11.802 1.707.457l4.84-2.794c.597-.345.802-1.11.457-1.708s-1.11-.802-1.708-.457zm10.922-10.458c-.345-.598-.14-1.363.458-1.708l4.537-2.62c.598-.345 1.362-.14 1.707.458.346.598.14 1.362-.457 1.707l-4.537 2.62c-.598.345-1.362.14-1.708-.457"
                })
            }),
            /*#__PURE__*/ (0, $5QV7f$jsx)("g", {
                clipPath: "url(#3cb847__a)",
                children: /*#__PURE__*/ (0, $5QV7f$jsxs)("g", {
                    clipPath: "url(#3cb847__b)",
                    children: [
                        /*#__PURE__*/ (0, $5QV7f$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $5QV7f$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#3cb847__c)"
                        }),
                        /*#__PURE__*/ (0, $5QV7f$jsx)("path", {
                            fill: "url(#3cb847__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $5QV7f$jsx)("path", {
                            fill: "url(#3cb847__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $5QV7f$jsx)("path", {
                            fill: "url(#3cb847__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $83620ec60270ade3$var$ForwardRef = /*#__PURE__*/ (0, $5QV7f$forwardRef)($83620ec60270ade3$var$SvgComponent);
var $83620ec60270ade3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($83620ec60270ade3$var$ForwardRef);


export {$83620ec60270ade3$export$2e2bcd8739ae039 as default};
