// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_EmptyStateExport_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $cWcLt$jsxs, jsx as $cWcLt$jsx} from "react/jsx-runtime";
import {forwardRef as $cWcLt$forwardRef} from "react";





const $8aca56409e9a50e0$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cWcLt$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cWcLt$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cWcLt$jsxs)("linearGradient", {
                        id: "f8e9d6__d",
                        x1: 34.069,
                        x2: 11.686,
                        y1: 3.537,
                        y2: 34.33,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cWcLt$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $cWcLt$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cWcLt$jsxs)("linearGradient", {
                        id: "f8e9d6__e",
                        x1: 19.359,
                        x2: 23.508,
                        y1: -10.048,
                        y2: 18.126,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cWcLt$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $cWcLt$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cWcLt$jsxs)("linearGradient", {
                        id: "f8e9d6__f",
                        x1: -34.395,
                        x2: 43.619,
                        y1: 46.445,
                        y2: 30.085,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cWcLt$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cWcLt$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $cWcLt$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $cWcLt$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cWcLt$jsxs)("radialGradient", {
                        id: "f8e9d6__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 56.7421 -60.2072 0 41.07 30.637)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cWcLt$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cWcLt$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cWcLt$jsx)("clipPath", {
                        id: "f8e9d6__b",
                        children: /*#__PURE__*/ (0, $cWcLt$jsx)("rect", {
                            width: 41.7,
                            height: 39.3,
                            x: 3.898,
                            y: 4.801,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cWcLt$jsx)("clipPath", {
                id: "f8e9d6__a",
                children: /*#__PURE__*/ (0, $cWcLt$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M33.598 4.8h6c1.989 0 3.6 1.613 3.6 3.6v13.2c0 1.989-1.611 3.6-3.6 3.6h-6c-1.988 0-3.6-1.611-3.6-3.6V8.4c0-1.987 1.612-3.6 3.6-3.6M17.398 30h13.2c1.989 0 3.6 1.613 3.6 3.6v6c0 1.989-1.611 3.6-3.6 3.6h-13.2c-1.988 0-3.6-1.611-3.6-3.6v-6c0-1.987 1.612-3.6 3.6-3.6m2.476-8.558 2.478 6.036c.402.975 1.776.996 2.205.033l.723-1.62c.12-.27.336-.486.606-.606l1.617-.723c.963-.43.942-1.806-.033-2.205l-6.033-2.481c-.987-.405-1.971.579-1.566 1.566zm-2.95 1.164c.663 0 1.2.537 1.2 1.2s-.537 1.2-1.2 1.2H10.79c-.447.918-1.383 1.55-2.472 1.55-1.52 0-2.75-1.232-2.75-2.75 0-1.09.635-2.025 1.55-2.472V10.77c-.918-.447-1.55-1.383-1.55-2.472 0-1.521 1.232-2.751 2.75-2.751 1.09 0 2.025.636 2.472 1.55h10.518c.447-.917 1.383-1.55 2.472-1.55 1.521 0 2.751 1.233 2.751 2.75 0 1.09-.636 2.026-1.55 2.473v6.15c0 .663-.538 1.2-1.2 1.2-.664 0-1.2-.537-1.2-1.2v-6.15c-.556-.27-1.003-.72-1.273-1.272H10.79c-.27.555-.72 1.002-1.272 1.272v10.563c.555.27 1.002.72 1.272 1.272z"
                })
            }),
            /*#__PURE__*/ (0, $cWcLt$jsx)("g", {
                clipPath: "url(#f8e9d6__a)",
                children: /*#__PURE__*/ (0, $cWcLt$jsxs)("g", {
                    clipPath: "url(#f8e9d6__b)",
                    children: [
                        /*#__PURE__*/ (0, $cWcLt$jsx)("rect", {
                            width: 41.7,
                            height: 39.3,
                            x: 3.898,
                            y: 4.801,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $cWcLt$jsx)("ellipse", {
                            cx: 41.071,
                            cy: 30.637,
                            fill: "url(#f8e9d6__c)",
                            rx: 60.207,
                            ry: 56.742
                        }),
                        /*#__PURE__*/ (0, $cWcLt$jsx)("path", {
                            fill: "url(#f8e9d6__d)",
                            d: "M.144 17.11C-.526 3.527 10.616-7.997 25.029-8.627c14.414-.631 26.64 9.87 27.31 23.453S41.867 39.933 27.453 40.564C13.04 41.194.813 30.694.143 17.11"
                        }),
                        /*#__PURE__*/ (0, $cWcLt$jsx)("path", {
                            fill: "url(#f8e9d6__e)",
                            d: "M14.435-22.394c-3.252-1.896-7.372-1.908-10.635-.03L-28.841-3.64c-3.197 1.84-3.203 6.181-.01 8.028L3.575 23.143c3.258 1.885 7.378 1.882 10.634-.007L46.568 4.363c3.175-1.843 3.18-6.16.01-8.01z"
                        }),
                        /*#__PURE__*/ (0, $cWcLt$jsx)("path", {
                            fill: "url(#f8e9d6__f)",
                            d: "M-16.89 23.599c6.729 0 12.823 2.566 17.235 6.706 8.825 8.294 16.141 8.301 24.966.021 4.412-4.16 10.514-6.734 17.25-6.734 13.462.007 24.369 10.258 24.369 22.911S56.023 69.401 42.56 69.401c-6.75 0-12.852-2.58-17.264-6.742-8.817-8.265-16.126-8.258-24.95.036C-4.068 66.842-10.162 69.4-16.89 69.4c-13.456 0-24.362-10.25-24.362-22.898 0-12.646 10.906-22.904 24.362-22.904"
                        })
                    ]
                })
            })
        ]
    });
const $8aca56409e9a50e0$var$ForwardRef = /*#__PURE__*/ (0, $cWcLt$forwardRef)($8aca56409e9a50e0$var$SvgComponent);
var $8aca56409e9a50e0$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($8aca56409e9a50e0$var$ForwardRef);


export {$8aca56409e9a50e0$export$2e2bcd8739ae039 as default};
