// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fileZip_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $c7pVG$jsxs, jsx as $c7pVG$jsx} from "react/jsx-runtime";
import {forwardRef as $c7pVG$forwardRef} from "react";





const $7773040f63f94807$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $c7pVG$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $c7pVG$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $c7pVG$jsxs)("linearGradient", {
                        id: "a5d1a3__d",
                        x1: 32.783,
                        x2: 12.551,
                        y1: 5.819,
                        y2: 33.645,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c7pVG$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $c7pVG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c7pVG$jsxs)("linearGradient", {
                        id: "a5d1a3__e",
                        x1: 19.489,
                        x2: 23.239,
                        y1: -6.459,
                        y2: 19.005,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c7pVG$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $c7pVG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c7pVG$jsxs)("linearGradient", {
                        id: "a5d1a3__f",
                        x1: -29.084,
                        x2: 41.411,
                        y1: 44.599,
                        y2: 29.819,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c7pVG$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $c7pVG$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $c7pVG$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $c7pVG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c7pVG$jsxs)("radialGradient", {
                        id: "a5d1a3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 51.2844 -54.4031 0 39.106 30.312)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $c7pVG$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $c7pVG$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $c7pVG$jsx)("clipPath", {
                        id: "a5d1a3__b",
                        children: /*#__PURE__*/ (0, $c7pVG$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.961,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $c7pVG$jsx)("clipPath", {
                id: "a5d1a3__a",
                children: /*#__PURE__*/ (0, $c7pVG$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m26.338 9.404 9.888 9.46v.005c.72.684 1.125 1.638 1.125 2.634v14.463c0 2.01-1.623 3.64-3.62 3.64H14.268c-2 0-3.62-1.63-3.62-3.64V12.041c0-2.01 1.622-3.639 3.62-3.639h9.573c.93 0 1.824.357 2.496 1.002m1.425 11.553c0-.663-.537-1.2-1.2-1.2h-1.52V17.76h1.523c.663 0 1.2-.537 1.2-1.2s-.537-1.2-1.2-1.2h-1.524v-.999c0-.663-.537-1.2-1.2-1.2h-2.397c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h1.197v1.998h-1.197c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h1.197v1.998h-1.197c-.663 0-1.2.537-1.2 1.2s.537 1.2 1.2 1.2h1.197v1.998h-.897c-.663 0-1.2.537-1.2 1.2v6.027c0 1.323 1.077 2.4 2.4 2.4h1.803c1.323 0 2.4-1.077 2.4-2.4v-6.027c0-.663-.537-1.2-1.2-1.2h-.909v-4.197h1.524c.663 0 1.2-.537 1.2-1.2m-4.818 10.21h1.806v2.411h-1.806z"
                })
            }),
            /*#__PURE__*/ (0, $c7pVG$jsx)("g", {
                clipPath: "url(#a5d1a3__a)",
                children: /*#__PURE__*/ (0, $c7pVG$jsxs)("g", {
                    clipPath: "url(#a5d1a3__b)",
                    children: [
                        /*#__PURE__*/ (0, $c7pVG$jsx)("rect", {
                            width: 37.68,
                            height: 35.52,
                            x: 5.52,
                            y: 6.961,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $c7pVG$jsx)("ellipse", {
                            cx: 39.106,
                            cy: 30.312,
                            fill: "url(#a5d1a3__c)",
                            rx: 54.403,
                            ry: 51.284
                        }),
                        /*#__PURE__*/ (0, $c7pVG$jsx)("path", {
                            fill: "url(#a5d1a3__d)",
                            d: "M2.128 18.087C1.523 5.809 11.591-4.607 24.615-5.177s24.072 8.92 24.677 21.198c.605 12.277-9.463 22.692-22.487 23.262S2.733 30.364 2.128 18.087"
                        }),
                        /*#__PURE__*/ (0, $c7pVG$jsx)("path", {
                            fill: "url(#a5d1a3__e)",
                            d: "M15.039-17.617c-2.938-1.714-6.66-1.725-9.61-.027L-24.064-.669c-2.889 1.662-2.893 5.587-.009 7.256l29.3 16.952c2.944 1.703 6.667 1.7 9.609-.006l29.24-16.968c2.869-1.666 2.874-5.568.008-7.24z"
                        }),
                        /*#__PURE__*/ (0, $c7pVG$jsx)("path", {
                            fill: "url(#a5d1a3__f)",
                            d: "M-13.266 23.95c6.08 0 11.586 2.32 15.573 6.06 7.974 7.497 14.585 7.504 22.559.02 3.987-3.76 9.5-6.087 15.586-6.087 12.166.007 22.02 9.272 22.02 20.708 0 11.437-9.854 20.695-22.02 20.695-6.1 0-11.613-2.332-15.6-6.093-7.967-7.47-14.57-7.464-22.545.032-3.987 3.748-9.494 6.061-15.573 6.061-12.158 0-22.013-9.264-22.013-20.695 0-11.43 9.855-20.701 22.013-20.701"
                        })
                    ]
                })
            })
        ]
    });
const $7773040f63f94807$var$ForwardRef = /*#__PURE__*/ (0, $c7pVG$forwardRef)($7773040f63f94807$var$SvgComponent);
var $7773040f63f94807$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($7773040f63f94807$var$ForwardRef);


export {$7773040f63f94807$export$2e2bcd8739ae039 as default};
