// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_AIGenerate_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $ht68n$reactjsxruntime = require("react/jsx-runtime");
var $ht68n$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $cbbd7f9e7304c90c$export$2e2bcd8739ae039);




const $cbbd7f9e7304c90c$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8bb15d__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8bb15d__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsxs)("linearGradient", {
                        id: "8bb15d__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsxs)("radialGradient", {
                        id: "8bb15d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("clipPath", {
                        id: "8bb15d__b",
                        children: /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("clipPath", {
                id: "8bb15d__a",
                children: /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M80.934 33.977c-.293 1.304-1.518 2.355-3.034 2l-5.463-1.282q-.208-.05-.418-.061c-.677-.039-1.349.208-1.856.696l-4.117 3.959c-1.827 1.758-4.734.006-4.19-2.525l1.23-5.7c.183-.847-.045-1.736-.609-2.373l-3.794-4.296q-.168-.192-.293-.395c-.344-.562-.446-1.185-.36-1.77.205-1.397 1.477-2.581 3.073-2.207l5.463 1.282c.81.191 1.662-.046 2.273-.635l4.118-3.959c1.827-1.758 4.734-.006 4.19 2.525l-1.23 5.7q-.034.16-.049.32c-.072.745.165 1.495.658 2.053l3.794 4.296c.51.577.711 1.257.678 1.91q-.012.233-.064.462M54.143 24c.222 1.06.716 2.108 1.542 3.042v.001l3.356 3.8-1.095 5.08c-.658 3.058.81 5.788 3.051 7.138 2.27 1.367 5.468 1.375 7.822-.889l3.2-3.078V70c0 4.418-3.581 8-8 8h-38c-4.418 0-8-3.582-8-8V32c0-4.419 3.582-8 8-8zm-2.177 27.368 4.016.905c1.578.357 2.674-1.539 1.575-2.73l-2.79-3.025c-.367-.4-.516-.951-.397-1.483l.905-4.015c.357-1.579-1.539-2.675-2.73-1.576l-3.026 2.79c-.4.367-.95.516-1.482.397l-4.016-.905c-1.578-.356-2.674 1.539-1.575 2.731l2.79 3.025c.367.4.516.95.397 1.483l-.905 4.015c-.357 1.578 1.539 2.675 2.73 1.575l3.026-2.79c.4-.367.95-.516 1.482-.397"
                })
            }),
            /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("g", {
                clipPath: "url(#8bb15d__a)",
                children: /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#8bb15d__b)",
                    children: [
                        /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#8bb15d__c)"
                        }),
                        /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("path", {
                            fill: "url(#8bb15d__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("path", {
                            fill: "url(#8bb15d__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $ht68n$reactjsxruntime.jsx)("path", {
                            fill: "url(#8bb15d__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $cbbd7f9e7304c90c$var$ForwardRef = /*#__PURE__*/ (0, $ht68n$react.forwardRef)($cbbd7f9e7304c90c$var$SvgComponent);
var $cbbd7f9e7304c90c$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($cbbd7f9e7304c90c$var$ForwardRef);
