// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_video_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $jXfIj$reactjsxruntime = require("react/jsx-runtime");
var $jXfIj$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $17e13b3be8033707$export$2e2bcd8739ae039);




const $17e13b3be8033707$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "78f171__d",
                        x1: 33.201,
                        x2: 9.027,
                        y1: 1.657,
                        y2: 32.999,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "78f171__e",
                        x1: 18.473,
                        x2: 23.137,
                        y1: -12.772,
                        y2: 17.074,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "78f171__f",
                        x1: -35.335,
                        x2: 43.129,
                        y1: 47.233,
                        y2: 31.725,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsxs)("radialGradient", {
                        id: "78f171__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 4.884 35.326)scale(60.2686)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("clipPath", {
                        id: "78f171__b",
                        children: /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3,
                            y: 3,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("clipPath", {
                id: "78f171__a",
                children: /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M6 15v18c0 2.65 2.149 4.8 4.8 4.8h26.4c2.651 0 4.8-2.15 4.8-4.8V15c0-2.652-2.149-4.8-4.8-4.8H10.8C8.149 10.2 6 12.347 6 15m15.543 4.343 6.28 3.636c.704.407.702 1.423-.002 1.828l-6.289 3.62c-.704.406-1.583-.103-1.582-.915l.009-7.257c0-.812.881-1.32 1.584-.912"
                })
            }),
            /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("g", {
                clipPath: "url(#78f171__a)",
                children: /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#78f171__b)",
                    children: [
                        /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("rect", {
                            width: 41.742,
                            height: 41.742,
                            x: 3,
                            y: 3,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("circle", {
                            cx: 40.21,
                            cy: 30.442,
                            r: 60.269,
                            fill: "url(#78f171__c)"
                        }),
                        /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("path", {
                            fill: "url(#78f171__d)",
                            d: "M-.758 16.073c-.67-14.428 10.483-26.667 24.91-27.337C38.582-11.934 50.82-.781 51.49 13.647S41.007 40.315 26.58 40.985-.088 30.5-.758 16.072"
                        }),
                        /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("path", {
                            fill: "url(#78f171__e)",
                            d: "M13.544-25.885c-3.255-2.014-7.379-2.026-10.646-.032l-32.674 19.95c-3.2 1.954-3.206 6.566-.01 8.527L2.673 22.482c3.262 2.002 7.385 2 10.645-.007l32.391-19.94c3.179-1.958 3.185-6.544.01-8.508z"
                        }),
                        /*#__PURE__*/ (0, $jXfIj$reactjsxruntime.jsx)("path", {
                            fill: "url(#78f171__f)",
                            d: "M-17.81 22.967c6.734 0 12.835 2.725 17.252 7.123 8.833 8.81 16.157 8.817 24.99.022 4.417-4.42 10.525-7.153 17.267-7.153 13.477.008 24.395 10.895 24.395 24.336s-10.918 24.32-24.395 24.32c-6.757 0-12.865-2.74-17.282-7.16-8.826-8.78-16.142-8.772-24.975.037-4.417 4.405-10.518 7.123-17.252 7.123-13.47 0-24.387-10.888-24.387-24.32S-31.28 22.967-17.81 22.967"
                        })
                    ]
                })
            })
        ]
    });
const $17e13b3be8033707$var$ForwardRef = /*#__PURE__*/ (0, $jXfIj$react.forwardRef)($17e13b3be8033707$var$SvgComponent);
var $17e13b3be8033707$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($17e13b3be8033707$var$ForwardRef);
