// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_brightnessContrast_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $lpIxb$reactjsxruntime = require("react/jsx-runtime");
var $lpIxb$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7ead961f45ac8db3$export$2e2bcd8739ae039);




const $7ead961f45ac8db3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "cb9bbc__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.545,
                        y2: 34.496,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "cb9bbc__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.137,
                        y2: 16.183,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsxs)("linearGradient", {
                        id: "cb9bbc__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsxs)("radialGradient", {
                        id: "cb9bbc__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("clipPath", {
                        id: "cb9bbc__b",
                        children: /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: -0.001,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("clipPath", {
                id: "cb9bbc__a",
                children: /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("path", {
                    fill: "#D9D9D9",
                    fillRule: "evenodd",
                    d: "M24 6.231c.69 0 1.25.56 1.25 1.25v2.341c0 .69-.56 1.25-1.25 1.25-.691 0-1.25-.56-1.25-1.25v-2.34c0-.69.559-1.25 1.25-1.25M23.999 14.407c5.298 0 9.593 4.295 9.593 9.594S29.297 33.594 24 33.594c-5.299 0-9.594-4.295-9.594-9.593 0-5.299 4.295-9.594 9.594-9.594m-.228 2.504c-3.812.12-6.866 3.248-6.866 7.09s3.054 6.97 6.866 7.09zM25.25 38.179c0-.69-.56-1.25-1.25-1.25-.691 0-1.25.56-1.25 1.25v2.34c0 .69.559 1.25 1.25 1.25.69 0 1.25-.56 1.25-1.25zM41.769 24.001c0 .69-.56 1.25-1.25 1.25h-2.34c-.691 0-1.25-.56-1.25-1.25s.559-1.25 1.25-1.25h2.34c.69 0 1.25.56 1.25 1.25M9.821 25.251c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25h-2.34c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25zM36.564 11.435c.488.488.488 1.28 0 1.767l-1.655 1.655c-.488.488-1.28.488-1.768 0s-.488-1.28 0-1.767l1.655-1.655c.489-.489 1.28-.489 1.768 0M14.857 34.91c.488-.489.488-1.28 0-1.769s-1.28-.488-1.768 0l-1.655 1.656c-.488.488-.488 1.28 0 1.767.489.488 1.28.488 1.768 0zM11.434 11.435c-.489.488-.489 1.28 0 1.767l1.655 1.655c.488.488 1.28.488 1.767 0 .488-.488.488-1.28 0-1.767l-1.655-1.655c-.488-.489-1.28-.489-1.767 0M33.14 34.91c-.488-.489-.488-1.28 0-1.769.489-.488 1.28-.488 1.768 0l1.655 1.656c.488.488.488 1.28 0 1.767-.488.488-1.28.488-1.768 0z"
                })
            }),
            /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("g", {
                clipPath: "url(#cb9bbc__a)",
                children: /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#cb9bbc__b)",
                    children: [
                        /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            y: -0.001,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#cb9bbc__c)"
                        }),
                        /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("path", {
                            fill: "url(#cb9bbc__d)",
                            d: "M-4.32 15.033C-5.09-1.559 7.735-15.632 24.326-16.403c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.623-4.32 15.033"
                        }),
                        /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("path", {
                            fill: "url(#cb9bbc__e)",
                            d: "M12.124-33.216c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.312c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.402c3.75 2.301 8.493 2.299 12.24-.009L49.112-.537c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $lpIxb$reactjsxruntime.jsx)("path", {
                            fill: "url(#cb9bbc__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $7ead961f45ac8db3$var$ForwardRef = /*#__PURE__*/ (0, $lpIxb$react.forwardRef)($7ead961f45ac8db3$var$SvgComponent);
var $7ead961f45ac8db3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7ead961f45ac8db3$var$ForwardRef);
