// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_vectorDraw_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $eRDMN$reactjsxruntime = require("react/jsx-runtime");
var $eRDMN$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7fb6c10475b4781d$export$2e2bcd8739ae039);




const $7fb6c10475b4781d$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4bd648__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4bd648__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4bd648__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4bd648__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("clipPath", {
                        id: "4bd648__b",
                        children: /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("clipPath", {
                id: "4bd648__a",
                children: /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "m90.334 33.289 36.376 36.376 5.722-5.722c4.053-4.053 4.444-10.234.872-13.806l-23.442-23.442c-3.571-3.572-9.753-3.181-13.806.872zm31.973 43.286L83.424 37.692 48.99 52.39c-3.078 1.314-5.5 3.986-6.478 7.149l-17.57 56.764c-1.364 4.404-.627 8.702 1.447 12.133l37.15-37.15c-.201-.702-.309-1.443-.309-2.209 0-4.418 3.582-8 8-8s8 3.582 8 8-3.582 8-8 8c-.667 0-1.314-.081-1.933-.235l-37.12 37.12c3.322 1.801 7.374 2.377 11.518 1.095l56.764-17.571c3.163-.978 5.835-3.4 7.149-6.478z"
                })
            }),
            /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("g", {
                clipPath: "url(#4bd648__a)",
                children: /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#4bd648__b)",
                    children: [
                        /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#4bd648__c)"
                        }),
                        /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("path", {
                            fill: "url(#4bd648__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("path", {
                            fill: "url(#4bd648__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $eRDMN$reactjsxruntime.jsx)("path", {
                            fill: "url(#4bd648__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $7fb6c10475b4781d$var$ForwardRef = /*#__PURE__*/ (0, $eRDMN$react.forwardRef)($7fb6c10475b4781d$var$SvgComponent);
var $7fb6c10475b4781d$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7fb6c10475b4781d$var$ForwardRef);
