// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_globe_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $3lDAS$reactjsxruntime = require("react/jsx-runtime");
var $3lDAS$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $dc860085f3082184$export$2e2bcd8739ae039);




const $dc860085f3082184$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a4557d__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a4557d__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsxs)("linearGradient", {
                        id: "a4557d__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsxs)("radialGradient", {
                        id: "a4557d__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("clipPath", {
                        id: "a4557d__b",
                        children: /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("clipPath", {
                id: "a4557d__a",
                children: /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M23.977 79.987c0-17.009 7.579-32.247 19.544-42.521.923 2.022 2.32 4.67 4.003 7.132 1.05 1.537 2.278 3.099 3.645 4.41 1.329 1.273 3.038 2.551 5.077 3.134 3.287.939 7.097.801 9.977.697q.456-.017.878-.03c1.63-.052 2.943-.066 4.03.045 1.114.114 1.57.325 1.709.417.307.205.386.367.412.43.033.08.067.223.026.458-.089.522-.525 1.277-1.398 1.762-2.073 1.152-4.143 2.599-5.695 4.594-1.626 2.09-2.542 4.602-2.542 7.554 0 .862-.068 1.398-.148 1.719l-.009.032c-.273.055-.798.104-1.69.03l-.857-.075c-2.318-.206-5.688-.504-8.868-.171-3.54.37-8.095 1.644-10.558 5.955-1.08 1.89-2.33 5.037-2.032 8.799.31 3.924 2.267 7.993 6.713 11.55 5.12 4.097 10.752 4.977 18.123 5.796 4.2.467 6.581 1.367 7.939 2.325 1.19.841 1.846 1.898 2.175 3.546.575 2.877-.395 7.187-2.433 12.445-.975 2.516-2.113 5.054-3.246 7.518l-.743 1.612c-.812 1.755-1.607 3.473-2.273 5.028-24.032-6.309-41.76-28.18-41.76-54.191m26.362-46.902q-.104-.255-.237-.486c3.834-2.424 7.982-4.395 12.37-5.84q.075.452.258.89c2.056 4.898 8.302 10.941 16.68 10.941 4.327 0 7.798-1.855 10.35-3.972 2.527-2.096 4.343-4.6 5.473-6.33.413-.632.624-1.338.649-2.042q1.035.305 2.051.649-.085.276-.129.57c-.673 4.46-1.437 10.7-.74 16.343.686 5.544 3.007 11.908 9.759 14.252 5.356 1.86 10.656.945 14.796-.9 2.737-1.22 5.23-2.963 7.071-4.904 2.109 3.695 3.81 7.653 5.04 11.814-.535-.113-1.121-.233-1.75-.357-3.013-.59-7.064-1.262-11.28-1.542-4.159-.276-8.77-.193-12.777.906-4.021 1.102-8.103 3.444-9.84 8.134-4.859 13.115 1.7 21.061 3.393 22.755 2.554 2.553 5.174 3.747 7.1 4.624q.351.159.67.307c1.861.87 3.188 1.627 4.54 3.58.467.675.945 1.887 1.3 3.74.343 1.796.517 3.894.583 6.106.065 2.199.021 4.415-.043 6.435-.017.551-.036 1.1-.055 1.634v.007c-.038 1.074-.074 2.09-.087 2.944-9.662 7.916-22.018 12.667-35.484 12.667q-3.135-.001-6.181-.337c.435-.967.91-1.993 1.413-3.082v-.002q.385-.83.79-1.709c1.145-2.492 2.367-5.214 3.435-7.968 2.047-5.281 3.89-11.544 2.82-16.905-.67-3.35-2.304-6.323-5.407-8.513-2.936-2.073-6.855-3.206-11.67-3.741-7.456-.83-10.891-1.598-14.008-4.091-2.896-2.318-3.617-4.428-3.736-5.935-.132-1.67.435-3.206 1.003-4.2.513-.897 1.725-1.682 4.444-1.967 2.354-.246 4.903-.025 7.3.182h.001l.928.08c2.863.238 5.98-.099 8.18-2.42 2.064-2.18 2.331-5.133 2.331-7.334 0-1.168.32-1.952.857-2.641.61-.786 1.63-1.605 3.265-2.513 2.835-1.575 4.87-4.32 5.398-7.403.56-3.262-.636-6.73-3.886-8.896-1.715-1.144-3.68-1.55-5.332-1.72-1.682-.172-3.485-.134-5.098-.083l-.653.022c-3.224.105-5.75.187-7.75-.385-.268-.076-.862-.377-1.739-1.217-.837-.802-1.715-1.888-2.577-3.149-1.728-2.528-3.124-5.366-3.789-6.998m73.361 81.962c7.71-9.598 12.324-21.79 12.324-35.06 0-2.487-.162-4.937-.476-7.338l-.125-.022c-.135-.026-.285-.057-.423-.087-.199-.044-.454-.1-.739-.165l-.348-.078c-.873-.195-2.06-.456-3.471-.733-2.838-.556-6.524-1.162-10.272-1.41-3.805-.254-7.384-.116-10.131.638-2.733.749-3.97 1.891-4.454 3.198-3.383 9.13 1.142 13.912 1.549 14.32 1.487 1.487 2.844 2.113 4.772 3.003l.726.336c2.423 1.132 5.216 2.641 7.731 6.274 1.387 2.003 2.136 4.472 2.58 6.79.455 2.376.65 4.941.722 7.373.03 1 .039 1.994.035 2.961m.277-69.772c-4.974-6.294-11.271-11.495-18.471-15.183-.572 4.097-.995 8.747-.502 12.734.564 4.557 2.101 6.864 4.442 7.676 2.957 1.027 6.112.6 8.917-.65 2.913-1.297 4.847-3.21 5.456-4.315q.075-.136.158-.262M80 23.963c2.755 0 5.463.199 8.111.583-.9 1.306-2.058 2.753-3.458 3.915-1.606 1.332-3.332 2.129-5.242 2.129-4.395 0-7.991-3.273-9.188-5.777 3.175-.559 6.442-.85 9.777-.85"
                })
            }),
            /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("g", {
                clipPath: "url(#a4557d__a)",
                children: /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#a4557d__b)",
                    children: [
                        /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#a4557d__c)"
                        }),
                        /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("path", {
                            fill: "url(#a4557d__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("path", {
                            fill: "url(#a4557d__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $3lDAS$reactjsxruntime.jsx)("path", {
                            fill: "url(#a4557d__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $dc860085f3082184$var$ForwardRef = /*#__PURE__*/ (0, $3lDAS$react.forwardRef)($dc860085f3082184$var$SvgComponent);
var $dc860085f3082184$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($dc860085f3082184$var$ForwardRef);
