// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_targeted_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $9CUsR$jsxs, jsx as $9CUsR$jsx} from "react/jsx-runtime";
import {forwardRef as $9CUsR$forwardRef} from "react";





const $3f212341c8c5e9b3$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9CUsR$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9CUsR$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9CUsR$jsxs)("linearGradient", {
                        id: "1d58a0__e",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9CUsR$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $9CUsR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9CUsR$jsxs)("linearGradient", {
                        id: "1d58a0__f",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9CUsR$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $9CUsR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9CUsR$jsxs)("linearGradient", {
                        id: "1d58a0__g",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9CUsR$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9CUsR$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $9CUsR$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $9CUsR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9CUsR$jsx)("clipPath", {
                        id: "1d58a0__a",
                        children: /*#__PURE__*/ (0, $9CUsR$jsx)("path", {
                            fill: "#fff",
                            d: "M0 0h160v160H0z"
                        })
                    }),
                    /*#__PURE__*/ (0, $9CUsR$jsx)("clipPath", {
                        id: "1d58a0__c",
                        children: /*#__PURE__*/ (0, $9CUsR$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    }),
                    /*#__PURE__*/ (0, $9CUsR$jsxs)("radialGradient", {
                        id: "1d58a0__d",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9CUsR$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9CUsR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9CUsR$jsxs)("g", {
                clipPath: "url(#1d58a0__a)",
                children: [
                    /*#__PURE__*/ (0, $9CUsR$jsx)("clipPath", {
                        id: "1d58a0__b",
                        children: /*#__PURE__*/ (0, $9CUsR$jsx)("path", {
                            fill: "#fff",
                            fillRule: "evenodd",
                            d: "M14.537 30.565c-.696.178-1.243.716-1.433 1.41-.19.692.007 1.434.515 1.942l15.729 15.729c.415.415.991.627 1.576.579l12.068-.982 22.952 22.951c1.562 1.562 4.094 1.562 5.657 0s1.562-4.094 0-5.657L48.65 43.587l.982-12.07c.048-.585-.164-1.161-.579-1.576l-15.73-15.73c-.507-.508-1.249-.704-1.942-.514s-1.23.737-1.409 1.433l-3.147 12.287zm65.284-2.62c-6.31 0-12.35 1.122-17.936 3.173-2.074.762-4.372-.302-5.134-2.376s.302-4.372 2.376-5.134c6.456-2.37 13.429-3.663 20.694-3.663 33.152 0 60.028 26.875 60.028 60.027S112.973 140 79.821 140s-60.027-26.875-60.027-60.028c0-7.074 1.225-13.87 3.479-20.184.742-2.08 3.03-3.165 5.111-2.422s3.166 3.031 2.423 5.112c-1.95 5.462-3.013 11.35-3.013 17.494C27.794 108.706 51.087 132 79.82 132s52.028-23.294 52.028-52.028-23.294-52.027-52.028-52.027m-.001 26.517c-3.393 0-6.62.66-9.571 1.855-2.047.83-4.38-.158-5.21-2.205-.829-2.048.159-4.38 2.207-5.21 3.888-1.574 8.135-2.44 12.574-2.44 18.506 0 33.508 15.003 33.508 33.509S98.326 113.48 79.82 113.48c-18.507 0-33.509-15.003-33.509-33.51 0-4.135.751-8.105 2.129-11.774.776-2.068 3.082-3.115 5.15-2.339s3.115 3.083 2.34 5.15c-1.046 2.784-1.619 5.802-1.619 8.964 0 14.088 11.42 25.509 25.509 25.509 14.088 0 25.508-11.421 25.508-25.51 0-14.087-11.42-25.508-25.508-25.508m-9.138 31.165c7.73-.72 13.881-6.873 14.6-14.602 3.09 1.861 5.158 5.25 5.158 9.12 0 5.876-4.763 10.639-10.639 10.639-3.87 0-7.257-2.067-9.119-5.157"
                        })
                    }),
                    /*#__PURE__*/ (0, $9CUsR$jsx)("g", {
                        clipPath: "url(#1d58a0__b)",
                        children: /*#__PURE__*/ (0, $9CUsR$jsxs)("g", {
                            clipPath: "url(#1d58a0__c)",
                            children: [
                                /*#__PURE__*/ (0, $9CUsR$jsx)("rect", {
                                    width: 160,
                                    height: 160,
                                    fill: "#D9F4FD",
                                    rx: 10
                                }),
                                /*#__PURE__*/ (0, $9CUsR$jsx)("circle", {
                                    cx: 142.62,
                                    cy: 105.187,
                                    r: 231.011,
                                    fill: "url(#1d58a0__d)"
                                }),
                                /*#__PURE__*/ (0, $9CUsR$jsx)("path", {
                                    fill: "url(#1d58a0__e)",
                                    d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                                }),
                                /*#__PURE__*/ (0, $9CUsR$jsx)("path", {
                                    fill: "url(#1d58a0__f)",
                                    d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                                }),
                                /*#__PURE__*/ (0, $9CUsR$jsx)("path", {
                                    fill: "url(#1d58a0__g)",
                                    d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                                })
                            ]
                        })
                    })
                ]
            })
        ]
    });
const $3f212341c8c5e9b3$var$ForwardRef = /*#__PURE__*/ (0, $9CUsR$forwardRef)($3f212341c8c5e9b3$var$SvgComponent);
var $3f212341c8c5e9b3$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($3f212341c8c5e9b3$var$ForwardRef);


export {$3f212341c8c5e9b3$export$2e2bcd8739ae039 as default};
