// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_bolt_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $iKgtO$reactjsxruntime = require("react/jsx-runtime");
var $iKgtO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $69f74d9819b590b8$export$2e2bcd8739ae039);




const $69f74d9819b590b8$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e209da__d",
                        x1: 64.526,
                        x2: 24.063,
                        y1: 9.716,
                        y2: 65.368,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e209da__e",
                        x1: 37.938,
                        x2: 45.44,
                        y1: -14.84,
                        y2: 36.088,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e209da__f",
                        x1: -59.204,
                        x2: 81.786,
                        y1: 87.277,
                        y2: 57.718,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsxs)("radialGradient", {
                        id: "e209da__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 77.173 58.704)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("clipPath", {
                        id: "e209da__b",
                        children: /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 10,
                            y: 12,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("clipPath", {
                id: "e209da__a",
                children: /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "m53.243 17.418-25.2 28.211c-2.044 2.291-1.25 5.915 1.577 7.13l11.321 4.873c.153.066.221.237.157.39L34.37 73.983c-1.97 4.672 4.095 8.526 7.49 4.76l25.08-27.833c2.068-2.296 1.26-5.951-1.589-7.156l-11.398-4.829-.004-.002c-.15-.063-.222-.235-.156-.39l6.943-16.357v-.002c1.99-4.68-4.105-8.551-7.495-4.757"
                })
            }),
            /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("g", {
                clipPath: "url(#e209da__a)",
                children: /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#e209da__b)",
                    children: [
                        /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 10,
                            y: 12,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("ellipse", {
                            cx: 77.173,
                            cy: 58.704,
                            fill: "url(#e209da__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("path", {
                            fill: "url(#e209da__d)",
                            d: "M3.217 34.251C2.007 9.697 22.143-11.133 48.191-12.273s48.144 17.84 49.353 42.395S78.62 75.506 52.571 76.647 4.427 58.806 3.217 34.25"
                        }),
                        /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("path", {
                            fill: "url(#e209da__e)",
                            d: "M29.039-37.157c-5.876-3.427-13.321-3.448-19.219-.054L-49.17-3.259c-5.776 3.326-5.786 11.174-.017 14.512l58.6 33.904c5.889 3.407 13.334 3.403 19.218-.012L87.109 11.21c5.74-3.33 5.75-11.135.018-14.479z"
                        }),
                        /*#__PURE__*/ (0, $iKgtO$reactjsxruntime.jsx)("path", {
                            fill: "url(#e209da__f)",
                            d: "M-27.567 45.98c12.159 0 23.172 4.638 31.146 12.122 15.948 14.993 29.17 15.006 45.118.038 7.973-7.522 19-12.173 31.173-12.173 24.33.013 44.04 18.542 44.04 41.416s-19.71 41.39-44.04 41.39c-12.2 0-23.227-4.665-31.2-12.187-15.935-14.941-29.143-14.928-45.09.064-7.975 7.497-18.988 12.123-31.147 12.123-24.317 0-44.027-18.53-44.027-41.39 0-22.861 19.71-41.403 44.027-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $69f74d9819b590b8$var$ForwardRef = /*#__PURE__*/ (0, $iKgtO$react.forwardRef)($69f74d9819b590b8$var$SvgComponent);
var $69f74d9819b590b8$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($69f74d9819b590b8$var$ForwardRef);
