// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_channel_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $f6JIv$jsxs, jsx as $f6JIv$jsx} from "react/jsx-runtime";
import {forwardRef as $f6JIv$forwardRef} from "react";





const $e308489103bc1625$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $f6JIv$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $f6JIv$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $f6JIv$jsxs)("linearGradient", {
                        id: "ad16d7__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $f6JIv$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $f6JIv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $f6JIv$jsxs)("linearGradient", {
                        id: "ad16d7__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $f6JIv$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $f6JIv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $f6JIv$jsxs)("linearGradient", {
                        id: "ad16d7__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $f6JIv$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $f6JIv$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $f6JIv$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $f6JIv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $f6JIv$jsxs)("radialGradient", {
                        id: "ad16d7__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $f6JIv$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $f6JIv$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $f6JIv$jsx)("clipPath", {
                        id: "ad16d7__b",
                        children: /*#__PURE__*/ (0, $f6JIv$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $f6JIv$jsx)("clipPath", {
                id: "ad16d7__a",
                children: /*#__PURE__*/ (0, $f6JIv$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M56.933 48.045h.008l8.451 14.638c-3.96 3.344-6.758 8.02-7.704 13.337H40.805c-1.38-2.402-3.972-4.02-6.942-4.02-4.418 0-8 3.582-8 8s3.582 8 8 8c2.953 0 5.532-1.6 6.918-3.98h16.907c.945 5.318 3.744 9.995 7.704 13.34l-8.428 14.596h-.03c-4.42 0-8 3.582-8 8s3.58 8 8 8c4.417 0 8-3.582 8-8 0-1.446-.385-2.803-1.056-3.974l8.439-14.617c2.405.868 5 1.342 7.705 1.342s5.3-.474 7.705-1.342l8.418 14.582c-.684 1.179-1.075 2.548-1.075 4.009 0 4.418 3.582 8 8 8s8-3.582 8-8c0-4.415-3.577-7.995-7.991-8L94.652 97.36c3.96-3.345 6.758-8.022 7.704-13.34h16.865c1.386 2.38 3.966 3.98 6.918 3.98 4.418 0 8-3.582 8-8s-3.582-8-8-8c-2.969 0-5.561 1.618-6.941 4.02h-16.842c-.946-5.317-3.745-9.993-7.705-13.337l8.451-14.638c4.404-.018 7.968-3.593 7.968-8 0-4.418-3.582-8-8-8s-8 3.582-8 8c0 1.47.396 2.846 1.087 4.03l-8.431 14.603c-2.405-.869-5-1.342-7.704-1.342-2.705 0-5.3.473-7.705 1.342L63.866 44.04c.679-1.176 1.067-2.54 1.067-3.995 0-4.418-3.582-8-8-8s-8 3.582-8 8 3.582 8 8 8"
                })
            }),
            /*#__PURE__*/ (0, $f6JIv$jsx)("g", {
                clipPath: "url(#ad16d7__a)",
                children: /*#__PURE__*/ (0, $f6JIv$jsxs)("g", {
                    clipPath: "url(#ad16d7__b)",
                    children: [
                        /*#__PURE__*/ (0, $f6JIv$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $f6JIv$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#ad16d7__c)"
                        }),
                        /*#__PURE__*/ (0, $f6JIv$jsx)("path", {
                            fill: "url(#ad16d7__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $f6JIv$jsx)("path", {
                            fill: "url(#ad16d7__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $f6JIv$jsx)("path", {
                            fill: "url(#ad16d7__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $e308489103bc1625$var$ForwardRef = /*#__PURE__*/ (0, $f6JIv$forwardRef)($e308489103bc1625$var$SvgComponent);
var $e308489103bc1625$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e308489103bc1625$var$ForwardRef);


export {$e308489103bc1625$export$2e2bcd8739ae039 as default};
