// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_targeted_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $aYCzW$reactjsxruntime = require("react/jsx-runtime");
var $aYCzW$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $fcb3957471fb8aed$export$2e2bcd8739ae039);




const $fcb3957471fb8aed$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d318f8__e",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d318f8__f",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d318f8__g",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("clipPath", {
                        id: "d318f8__a",
                        children: /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("path", {
                            fill: "#fff",
                            d: "M0 0h96v96H0z"
                        })
                    }),
                    /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("clipPath", {
                        id: "d318f8__c",
                        children: /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    }),
                    /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d318f8__d",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    })
                ]
            }),
            /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsxs)("g", {
                clipPath: "url(#d318f8__a)",
                children: [
                    /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("clipPath", {
                        id: "d318f8__b",
                        children: /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("path", {
                            fill: "#fff",
                            fillRule: "evenodd",
                            d: "M9.619 18.305c-.696.179-1.243.716-1.433 1.41-.19.692.007 1.434.515 1.942l9.197 9.198c.415.415.992.626 1.577.579l8.202-.668 12.447 12.447c.781.781 2.048.781 2.829 0 .78-.781.78-2.047 0-2.828L30.507 27.939l.668-8.205c.047-.585-.165-1.162-.58-1.577L21.398 8.96c-.508-.508-1.25-.705-1.943-.515s-1.23.737-1.409 1.433l-1.718 6.709zm38.38-1.066c-3.73 0-7.301.663-10.604 1.876-1.037.381-2.186-.15-2.567-1.188-.38-1.037.15-2.186 1.188-2.567C39.754 13.987 43.792 13.24 48 13.24c19.198 0 34.762 15.563 34.762 34.762S67.198 82.763 48 82.763c-19.199 0-34.762-15.564-34.762-34.762 0-4.097.71-8.033 2.014-11.689.372-1.04 1.516-1.582 2.556-1.211s1.583 1.516 1.212 2.556c-1.154 3.23-1.782 6.711-1.782 10.344 0 16.99 13.772 30.762 30.762 30.762S78.762 64.99 78.762 48.001 64.989 17.239 48 17.239m0 15.506c-2.028 0-3.959.394-5.724 1.11-1.023.414-2.19-.08-2.604-1.104-.415-1.023.079-2.19 1.103-2.604 2.233-.905 4.674-1.402 7.225-1.402 10.635 0 19.255 8.62 19.255 19.255S58.634 67.255 48 67.255 28.744 58.635 28.744 48c0-2.377.432-4.659 1.223-6.766.388-1.034 1.541-1.558 2.575-1.17s1.558 1.541 1.17 2.575c-.625 1.665-.968 3.47-.968 5.36 0 8.426 6.83 15.256 15.255 15.256S63.254 56.425 63.254 48 56.424 32.745 48 32.745m-5.344 18.562c4.52-.42 8.118-4.018 8.538-8.538 1.807 1.089 3.016 3.07 3.016 5.333 0 3.436-2.786 6.22-6.221 6.22-2.263 0-4.244-1.208-5.333-3.015"
                        })
                    }),
                    /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("g", {
                        clipPath: "url(#d318f8__b)",
                        children: /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsxs)("g", {
                            clipPath: "url(#d318f8__c)",
                            children: [
                                /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("rect", {
                                    width: 96,
                                    height: 96,
                                    fill: "#D9F4FD",
                                    rx: 10
                                }),
                                /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("circle", {
                                    cx: 85.571,
                                    cy: 63.112,
                                    r: 138.607,
                                    fill: "url(#d318f8__d)"
                                }),
                                /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("path", {
                                    fill: "url(#d318f8__e)",
                                    d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                                }),
                                /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("path", {
                                    fill: "url(#d318f8__f)",
                                    d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                                }),
                                /*#__PURE__*/ (0, $aYCzW$reactjsxruntime.jsx)("path", {
                                    fill: "url(#d318f8__g)",
                                    d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                                })
                            ]
                        })
                    })
                ]
            })
        ]
    });
const $fcb3957471fb8aed$var$ForwardRef = /*#__PURE__*/ (0, $aYCzW$react.forwardRef)($fcb3957471fb8aed$var$SvgComponent);
var $fcb3957471fb8aed$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($fcb3957471fb8aed$var$ForwardRef);
