// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_dataAnalytics_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $4lYns$reactjsxruntime = require("react/jsx-runtime");
var $4lYns$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $bbd8e0deb8a20b49$export$2e2bcd8739ae039);




const $bbd8e0deb8a20b49$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e3ee1a__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e3ee1a__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsxs)("linearGradient", {
                        id: "e3ee1a__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsxs)("radialGradient", {
                        id: "e3ee1a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("clipPath", {
                        id: "e3ee1a__b",
                        children: /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("clipPath", {
                id: "e3ee1a__a",
                children: /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M79.44 20.814c.686.866.539 2.124-.328 2.81L56.016 41.894c-.619.49-1.468.57-2.167.204L39.13 34.408 19.423 51.125c-.843.714-2.105.61-2.82-.232s-.61-2.104.232-2.819l20.736-17.587c.62-.527 1.498-.625 2.22-.248l14.76 7.714 22.08-17.467c.866-.685 2.124-.538 2.81.328M35.76 51.857c0-1.927 1.562-3.49 3.49-3.49 1.927 0 3.49 1.563 3.49 3.49v18.597c0 1.928-1.563 3.49-3.49 3.49s-3.49-1.562-3.49-3.49zm20.998-.066c-1.928 0-3.49 1.563-3.49 3.49v15.175c0 1.928 1.562 3.49 3.49 3.49 1.927 0 3.49-1.562 3.49-3.49V55.281c0-1.927-1.563-3.49-3.49-3.49m14.018-8.555c0-1.927 1.563-3.49 3.49-3.49s3.49 1.563 3.49 3.49v27.22c0 1.927-1.562 3.49-3.49 3.49s-3.49-1.563-3.49-3.49zM21.74 58.565c-1.928 0-3.49 1.562-3.49 3.49v8.4c0 1.927 1.562 3.49 3.49 3.49 1.927 0 3.49-1.563 3.49-3.49v-8.4c0-1.928-1.563-3.49-3.49-3.49"
                })
            }),
            /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("g", {
                clipPath: "url(#e3ee1a__a)",
                children: /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#e3ee1a__b)",
                    children: [
                        /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#e3ee1a__c)"
                        }),
                        /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("path", {
                            fill: "url(#e3ee1a__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("path", {
                            fill: "url(#e3ee1a__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $4lYns$reactjsxruntime.jsx)("path", {
                            fill: "url(#e3ee1a__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $bbd8e0deb8a20b49$var$ForwardRef = /*#__PURE__*/ (0, $4lYns$react.forwardRef)($bbd8e0deb8a20b49$var$SvgComponent);
var $bbd8e0deb8a20b49$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($bbd8e0deb8a20b49$var$ForwardRef);
