// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_movieCamera_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $7kNwa$jsxs, jsx as $7kNwa$jsx} from "react/jsx-runtime";
import {forwardRef as $7kNwa$forwardRef} from "react";





const $4d682407a4e444a9$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $7kNwa$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $7kNwa$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $7kNwa$jsxs)("linearGradient", {
                        id: "a2f5b9__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -5.588,
                        y2: 66.493,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7kNwa$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $7kNwa$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7kNwa$jsxs)("linearGradient", {
                        id: "a2f5b9__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -38.772,
                        y2: 29.867,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7kNwa$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $7kNwa$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7kNwa$jsxs)("linearGradient", {
                        id: "a2f5b9__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 99.225,
                        y2: 63.562,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7kNwa$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $7kNwa$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $7kNwa$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $7kNwa$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7kNwa$jsxs)("radialGradient", {
                        id: "a2f5b9__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.48 73.092)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $7kNwa$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $7kNwa$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7kNwa$jsx)("clipPath", {
                        id: "a2f5b9__b",
                        children: /*#__PURE__*/ (0, $7kNwa$jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: -2.5,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $7kNwa$jsx)("clipPath", {
                id: "a2f5b9__a",
                children: /*#__PURE__*/ (0, $7kNwa$jsx)("path", {
                    fill: "#292929",
                    d: "M61.674 34.094v7.769l14.192-10.532c2.64-1.96 6.384-.075 6.384 3.212v26.815c0 3.278-3.728 5.164-6.37 3.222L61.675 54.136v7.77c0 4.417-3.582 8-8 8H21.75c-4.418 0-8-3.583-8-8V34.093c0-4.419 3.582-8 8-8h31.924c4.418 0 8 3.581 8 8"
                })
            }),
            /*#__PURE__*/ (0, $7kNwa$jsx)("g", {
                clipPath: "url(#a2f5b9__a)",
                children: /*#__PURE__*/ (0, $7kNwa$jsxs)("g", {
                    clipPath: "url(#a2f5b9__b)",
                    children: [
                        /*#__PURE__*/ (0, $7kNwa$jsx)("rect", {
                            width: 96,
                            height: 96,
                            y: -2.5,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $7kNwa$jsx)("circle", {
                            cx: 85.571,
                            cy: 60.612,
                            r: 138.607,
                            fill: "url(#a2f5b9__c)"
                        }),
                        /*#__PURE__*/ (0, $7kNwa$jsx)("path", {
                            fill: "url(#a2f5b9__d)",
                            d: "M-8.64 27.567c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $7kNwa$jsx)("path", {
                            fill: "url(#a2f5b9__e)",
                            d: "M24.253-68.93c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $7kNwa$jsx)("path", {
                            fill: "url(#a2f5b9__f)",
                            d: "M-47.859 43.418c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 155.3 89.003 155.3c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 149.049-32.37 155.3-47.858 155.3c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $4d682407a4e444a9$var$ForwardRef = /*#__PURE__*/ (0, $7kNwa$forwardRef)($4d682407a4e444a9$var$SvgComponent);
var $4d682407a4e444a9$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($4d682407a4e444a9$var$ForwardRef);


export {$4d682407a4e444a9$export$2e2bcd8739ae039 as default};
