// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_fileShared_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6HowO$jsxs, jsx as $6HowO$jsx} from "react/jsx-runtime";
import {forwardRef as $6HowO$forwardRef} from "react";





const $c9292eacfbde1b33$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6HowO$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6HowO$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6HowO$jsxs)("linearGradient", {
                        id: "5a4bb3__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6HowO$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6HowO$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6HowO$jsxs)("linearGradient", {
                        id: "5a4bb3__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6HowO$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6HowO$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6HowO$jsxs)("linearGradient", {
                        id: "5a4bb3__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6HowO$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6HowO$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6HowO$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6HowO$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6HowO$jsxs)("radialGradient", {
                        id: "5a4bb3__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6HowO$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6HowO$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6HowO$jsx)("clipPath", {
                        id: "5a4bb3__b",
                        children: /*#__PURE__*/ (0, $6HowO$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6HowO$jsx)("clipPath", {
                id: "5a4bb3__a",
                children: /*#__PURE__*/ (0, $6HowO$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M52.909 18.632 72.69 38.415c1.35 1.35 2.109 3.181 2.109 5.09v28.218c0 3.977-3.224 7.2-7.2 7.2H29.2c-3.976 0-7.2-3.223-7.2-7.2v-48c0-3.976 3.224-7.2 7.2-7.2h18.618c1.91 0 3.74.759 5.09 2.11M41.104 46.795c0-.703.57-1.273 1.273-1.273 1.104 0 2-.895 2-2s-.896-2-2-2c-2.913 0-5.273 2.361-5.273 5.273 0 1.487.617 2.833 1.605 3.79.794.767 2.06.747 2.828-.047.769-.793.748-2.06-.045-2.828-.242-.234-.388-.555-.388-.915m8.925-2.92c-2.729 0-4.94 2.213-4.94 4.942s2.211 4.94 4.94 4.94 4.942-2.211 4.942-4.94-2.213-4.941-4.942-4.941m-9.066 13.86c.996-.476 1.418-1.67.942-2.666-.476-.997-1.67-1.419-2.666-.943-2.447 1.169-4.239 3.518-4.71 6.297-.185 1.09.548 2.122 1.637 2.307s2.121-.548 2.306-1.637c.256-1.508 1.228-2.754 2.49-3.358m9.066-2.095c-3.947 0-7.507 2.53-8.388 6.19-.514 2.133 1.29 3.692 3.04 3.692h10.698c1.748 0 3.553-1.559 3.04-3.692-.882-3.66-4.442-6.19-8.39-6.19"
                })
            }),
            /*#__PURE__*/ (0, $6HowO$jsx)("g", {
                clipPath: "url(#5a4bb3__a)",
                children: /*#__PURE__*/ (0, $6HowO$jsxs)("g", {
                    clipPath: "url(#5a4bb3__b)",
                    children: [
                        /*#__PURE__*/ (0, $6HowO$jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6HowO$jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#5a4bb3__c)"
                        }),
                        /*#__PURE__*/ (0, $6HowO$jsx)("path", {
                            fill: "url(#5a4bb3__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $6HowO$jsx)("path", {
                            fill: "url(#5a4bb3__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $6HowO$jsx)("path", {
                            fill: "url(#5a4bb3__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $c9292eacfbde1b33$var$ForwardRef = /*#__PURE__*/ (0, $6HowO$forwardRef)($c9292eacfbde1b33$var$SvgComponent);
var $c9292eacfbde1b33$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($c9292eacfbde1b33$var$ForwardRef);


export {$c9292eacfbde1b33$export$2e2bcd8739ae039 as default};
