// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_star_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $65Qbg$jsxs, jsx as $65Qbg$jsx} from "react/jsx-runtime";
import {forwardRef as $65Qbg$forwardRef} from "react";





const $33bee991bd143042$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $65Qbg$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $65Qbg$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $65Qbg$jsxs)("linearGradient", {
                        id: "df6ab1__d",
                        x1: 110.95,
                        x2: 30.77,
                        y1: 6.324,
                        y2: 110.278,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $65Qbg$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $65Qbg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $65Qbg$jsxs)("linearGradient", {
                        id: "df6ab1__e",
                        x1: 62.1,
                        x2: 77.567,
                        y1: -41.534,
                        y2: 57.456,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $65Qbg$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $65Qbg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $65Qbg$jsxs)("linearGradient", {
                        id: "df6ab1__f",
                        x1: -116.369,
                        x2: 143.875,
                        y1: 157.48,
                        y2: 106.047,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $65Qbg$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $65Qbg$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $65Qbg$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $65Qbg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $65Qbg$jsxs)("radialGradient", {
                        id: "df6ab1__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.195 117.99)scale(199.896)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $65Qbg$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $65Qbg$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $65Qbg$jsx)("clipPath", {
                        id: "df6ab1__b",
                        children: /*#__PURE__*/ (0, $65Qbg$jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 10.773,
                            y: 10.775,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $65Qbg$jsx)("clipPath", {
                id: "df6ab1__a",
                children: /*#__PURE__*/ (0, $65Qbg$jsx)("path", {
                    fill: "#fff",
                    d: "m88.208 33.102 9.66 19.573c1.333 2.701 3.91 4.574 6.891 5.007l21.601 3.138c7.506 1.091 10.504 10.316 5.072 15.61l-15.631 15.237c-2.157 2.102-3.141 5.131-2.632 8.1l3.69 21.514c1.282 7.476-6.565 13.177-13.279 9.647l-19.32-10.157c-2.666-1.402-5.851-1.402-8.517 0l-19.32 10.157c-6.715 3.53-14.562-2.171-13.28-9.647l3.69-21.514c.51-2.969-.475-5.998-2.632-8.1L28.571 76.43c-5.432-5.295-2.435-14.52 5.072-15.61l21.6-3.14c2.981-.432 5.558-2.305 6.891-5.006l9.66-19.573c3.358-6.802 13.057-6.802 16.414 0"
                })
            }),
            /*#__PURE__*/ (0, $65Qbg$jsx)("g", {
                clipPath: "url(#df6ab1__a)",
                children: /*#__PURE__*/ (0, $65Qbg$jsxs)("g", {
                    clipPath: "url(#df6ab1__b)",
                    children: [
                        /*#__PURE__*/ (0, $65Qbg$jsx)("rect", {
                            width: 138.449,
                            height: 138.449,
                            x: 10.773,
                            y: 10.775,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $65Qbg$jsx)("circle", {
                            cx: 134.185,
                            cy: 101.794,
                            r: 199.896,
                            fill: "url(#df6ab1__c)"
                        }),
                        /*#__PURE__*/ (0, $65Qbg$jsx)("path", {
                            fill: "url(#df6ab1__d)",
                            d: "M-1.686 54.14c-2.222-47.855 34.77-88.45 82.624-90.672 47.855-2.222 88.449 34.77 90.671 82.625s-34.77 88.449-82.624 90.671-88.449-34.77-90.67-82.625"
                        }),
                        /*#__PURE__*/ (0, $65Qbg$jsx)("path", {
                            fill: "url(#df6ab1__e)",
                            d: "M45.75-85.028c-10.795-6.68-24.473-6.72-35.308-.105L-97.931-18.964c-10.614 6.48-10.632 21.777-.032 28.282L9.694 75.394C20.513 82.033 34.19 82.025 45 75.37L152.436 9.233c10.543-6.491 10.562-21.702.032-28.218z"
                        }),
                        /*#__PURE__*/ (0, $65Qbg$jsx)("path", {
                            fill: "url(#df6ab1__f)",
                            d: "M-58.246 76.996c22.337 0 42.57 9.04 57.22 23.625 29.3 29.219 53.59 29.244 82.889.075 14.649-14.66 34.908-23.725 57.27-23.725 44.7.025 80.91 36.137 80.91 80.715s-36.21 80.665-80.91 80.665c-22.412 0-42.671-9.091-57.32-23.75-29.275-29.12-53.54-29.095-82.839.125-14.65 14.609-34.883 23.625-57.22 23.625-44.675 0-80.885-36.112-80.885-80.665s36.21-80.69 80.885-80.69"
                        })
                    ]
                })
            })
        ]
    });
const $33bee991bd143042$var$ForwardRef = /*#__PURE__*/ (0, $65Qbg$forwardRef)($33bee991bd143042$var$SvgComponent);
var $33bee991bd143042$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($33bee991bd143042$var$ForwardRef);


export {$33bee991bd143042$export$2e2bcd8739ae039 as default};
