// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_speedFast_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $6gmYI$reactjsxruntime = require("react/jsx-runtime");
var $6gmYI$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $4fb21081ba4e1271$export$2e2bcd8739ae039);




const $4fb21081ba4e1271$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7afe9f__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7afe9f__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7afe9f__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsxs)("radialGradient", {
                        id: "7afe9f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("clipPath", {
                        id: "7afe9f__b",
                        children: /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("clipPath", {
                id: "7afe9f__a",
                children: /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("path", {
                    fill: "#D9D9D9",
                    d: "M22.75 11.882v3.506c0 .69.559 1.25 1.25 1.25.69 0 1.25-.56 1.25-1.25v-3.506q0-.132-.027-.255c2.16.157 4.205.733 6.053 1.646q-.07.084-.129.183l-1.753 3.036c-.345.598-.14 1.363.458 1.708s1.362.14 1.707-.458l1.753-3.036q.05-.087.085-.178c4.004 2.793 6.704 7.326 7.005 12.502h-3.499c-.69 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25h3.457c-.2 2.172-.825 4.222-1.789 6.066-.322.616-.976.973-1.671.973H11.128c-.696 0-1.35-.357-1.672-.973-.964-1.844-1.588-3.893-1.789-6.066h3.457c.69 0 1.25-.56 1.25-1.25s-.56-1.25-1.25-1.25H7.625c.122-2.102.64-4.099 1.481-5.916l.07.044 3.036 1.753c.598.345 1.363.14 1.708-.458s.14-1.362-.458-1.707l-3.036-1.753q-.051-.03-.104-.054c1.149-1.733 2.618-3.235 4.324-4.422q.035.096.089.19l1.753 3.035c.345.598 1.11.803 1.707.458.598-.345.803-1.11.458-1.708L16.9 13.456q-.06-.103-.135-.19c1.836-.905 3.866-1.477 6.01-1.637q-.026.123-.026.253m.027 19.029c-.902.811-.939 2.214-.08 3.072.858.858 2.26.821 3.072-.08l11.014-12.236c.447-.496.427-1.254-.045-1.726-.471-.471-1.23-.491-1.725-.045z"
                })
            }),
            /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("g", {
                clipPath: "url(#7afe9f__a)",
                children: /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#7afe9f__b)",
                    children: [
                        /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#7afe9f__c)"
                        }),
                        /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("path", {
                            fill: "url(#7afe9f__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("path", {
                            fill: "url(#7afe9f__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $6gmYI$reactjsxruntime.jsx)("path", {
                            fill: "url(#7afe9f__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $4fb21081ba4e1271$var$ForwardRef = /*#__PURE__*/ (0, $6gmYI$react.forwardRef)($4fb21081ba4e1271$var$SvgComponent);
var $4fb21081ba4e1271$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($4fb21081ba4e1271$var$ForwardRef);
