// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_graphBarChart_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $iffrk$reactjsxruntime = require("react/jsx-runtime");
var $iffrk$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $89dafdea5df227b6$export$2e2bcd8739ae039);




const $89dafdea5df227b6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3ba1fd__d",
                        x1: 65.082,
                        x2: 18.604,
                        y1: 6.377,
                        y2: 70.302,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3ba1fd__e",
                        x1: 34.54,
                        x2: 43.156,
                        y1: -21.829,
                        y2: 36.669,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsxs)("linearGradient", {
                        id: "3ba1fd__f",
                        x1: -77.045,
                        x2: 84.904,
                        y1: 95.467,
                        y2: 61.514,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsxs)("radialGradient", {
                        id: "3ba1fd__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 117.816 -124.98 0 79.609 62.646)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("clipPath", {
                        id: "3ba1fd__b",
                        children: /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("rect", {
                            width: 86.562,
                            height: 81.6,
                            x: 2.449,
                            y: 9,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("clipPath", {
                id: "3ba1fd__a",
                children: /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M14.527 81.6H81.6c1.326 0 2.4 1.074 2.4 2.4s-1.074 2.4-2.4 2.4H12.127c-1.326 0-2.4-1.074-2.4-2.4V14.4c0-1.326 1.074-2.4 2.4-2.4s2.4 1.074 2.4 2.4zM26.4 42.63h2.4c2.651 0 4.8 2.15 4.8 4.8V69.6c0 2.651-2.149 4.8-4.8 4.8h-2.4c-2.651 0-4.8-2.149-4.8-4.8V47.43c0-2.65 2.149-4.8 4.8-4.8M72 57.6h-2.4c-2.651 0-4.8 2.149-4.8 4.8v7.2c0 2.651 2.149 4.8 4.8 4.8H72c2.651 0 4.8-2.149 4.8-4.8v-7.2c0-2.651-2.149-4.8-4.8-4.8M48 24h2.4c2.651 0 4.8 2.149 4.8 4.8v40.8c0 2.651-2.149 4.8-4.8 4.8H48c-2.651 0-4.8-2.149-4.8-4.8V28.8c0-2.651 2.149-4.8 4.8-4.8"
                })
            }),
            /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("g", {
                clipPath: "url(#3ba1fd__a)",
                children: /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#3ba1fd__b)",
                    children: [
                        /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("rect", {
                            width: 86.562,
                            height: 81.6,
                            x: 2.449,
                            y: 9,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("ellipse", {
                            cx: 79.609,
                            cy: 62.646,
                            fill: "url(#3ba1fd__c)",
                            rx: 124.98,
                            ry: 117.816
                        }),
                        /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("path", {
                            fill: "url(#3ba1fd__d)",
                            d: "M-5.34 34.559c-1.39-28.205 21.738-52.13 51.658-53.44s55.301 20.492 56.69 48.697-21.74 52.13-51.659 53.44C21.43 84.566-3.95 62.763-5.34 34.56"
                        }),
                        /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("path", {
                            fill: "url(#3ba1fd__e)",
                            d: "M24.317-47.464c-6.75-3.937-15.3-3.961-22.075-.062l-67.758 39c-6.636 3.819-6.647 12.834-.02 16.668l67.31 38.944C8.538 51 17.09 50.995 23.848 47.072L91.02 8.092c6.592-3.826 6.604-12.791.02-16.631z"
                        }),
                        /*#__PURE__*/ (0, $iffrk$reactjsxruntime.jsx)("path", {
                            fill: "url(#3ba1fd__f)",
                            d: "M-40.705 48.03c13.966 0 26.617 5.329 35.776 13.925 18.319 17.221 33.506 17.236 51.824.044 9.16-8.64 21.826-13.983 35.807-13.983 27.947.014 50.587 21.298 50.587 47.572 0 26.273-22.64 47.543-50.587 47.543-14.013 0-26.68-5.358-35.838-13.998-18.303-17.163-33.474-17.148-51.793.073-9.16 8.611-21.81 13.925-35.776 13.925-27.931 0-50.57-21.284-50.57-47.543 0-26.26 22.639-47.558 50.57-47.558"
                        })
                    ]
                })
            })
        ]
    });
const $89dafdea5df227b6$var$ForwardRef = /*#__PURE__*/ (0, $iffrk$react.forwardRef)($89dafdea5df227b6$var$SvgComponent);
var $89dafdea5df227b6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($89dafdea5df227b6$var$ForwardRef);
