// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_location_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $jD6st$jsxs, jsx as $jD6st$jsx} from "react/jsx-runtime";
import {forwardRef as $jD6st$forwardRef} from "react";





const $b68d60985b2d447f$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $jD6st$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $jD6st$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $jD6st$jsxs)("linearGradient", {
                        id: "c7a621__d",
                        x1: 109.277,
                        x2: 41.838,
                        y1: 19.392,
                        y2: 112.146,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jD6st$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $jD6st$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jD6st$jsxs)("linearGradient", {
                        id: "c7a621__e",
                        x1: 64.961,
                        x2: 77.463,
                        y1: -21.535,
                        y2: 63.345,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jD6st$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $jD6st$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jD6st$jsxs)("linearGradient", {
                        id: "c7a621__f",
                        x1: -96.944,
                        x2: 138.04,
                        y1: 148.659,
                        y2: 99.394,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jD6st$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $jD6st$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $jD6st$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $jD6st$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jD6st$jsxs)("radialGradient", {
                        id: "c7a621__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 170.948 -181.344 0 130.355 101.038)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $jD6st$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $jD6st$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $jD6st$jsx)("clipPath", {
                        id: "c7a621__b",
                        children: /*#__PURE__*/ (0, $jD6st$jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.199,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $jD6st$jsx)("clipPath", {
                id: "c7a621__a",
                children: /*#__PURE__*/ (0, $jD6st$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M36.281 63.188c2.362-21.15 19.783-37.788 40.923-39.101C102.77 22.499 124 42.864 124 68.212c0 6.477-1.398 12.623-3.889 18.167-7.423 19.434-24.378 37.82-33.724 46.967-3.605 3.528-9.337 3.544-12.942.016-9.991-9.776-28.708-30.137-35.145-51.047-1.981-5.916-2.772-12.383-2.019-19.127M80 83c9.941 0 18-8.059 18-18s-8.059-18-18-18-18 8.059-18 18 8.059 18 18 18"
                })
            }),
            /*#__PURE__*/ (0, $jD6st$jsx)("g", {
                clipPath: "url(#c7a621__a)",
                children: /*#__PURE__*/ (0, $jD6st$jsxs)("g", {
                    clipPath: "url(#c7a621__b)",
                    children: [
                        /*#__PURE__*/ (0, $jD6st$jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.199,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $jD6st$jsx)("ellipse", {
                            cx: 130.355,
                            cy: 101.038,
                            fill: "url(#c7a621__c)",
                            rx: 181.344,
                            ry: 170.948
                        }),
                        /*#__PURE__*/ (0, $jD6st$jsx)("path", {
                            fill: "url(#c7a621__d)",
                            d: "M7.095 60.284C5.08 19.359 38.638-15.356 82.051-17.257s80.24 29.735 82.256 70.66c2.016 40.923-31.543 75.639-74.956 77.54-43.413 1.9-80.24-29.735-82.256-70.66"
                        }),
                        /*#__PURE__*/ (0, $jD6st$jsx)("path", {
                            fill: "url(#c7a621__e)",
                            d: "M50.129-58.73c-9.794-5.713-22.202-5.748-32.032-.09L-80.218-2.233c-9.629 5.542-9.645 18.623-.03 24.186L17.42 78.46c9.814 5.678 22.223 5.671 32.03-.02l97.464-56.56c9.564-5.551 9.582-18.56.029-24.131z"
                        }),
                        /*#__PURE__*/ (0, $jD6st$jsx)("path", {
                            fill: "url(#c7a621__f)",
                            d: "M-44.215 79.83c20.264 0 38.62 7.731 51.91 20.204 26.58 24.988 48.615 25.009 75.195.064 13.29-12.537 31.668-20.29 51.955-20.29 40.551.022 73.401 30.904 73.401 69.027s-32.85 68.984-73.401 68.984c-20.332 0-38.71-7.774-52-20.311-26.557-24.903-48.57-24.881-75.15.107-13.29 12.494-31.646 20.204-51.91 20.204-40.529 0-73.378-30.883-73.378-68.984s32.85-69.005 73.378-69.005"
                        })
                    ]
                })
            })
        ]
    });
const $b68d60985b2d447f$var$ForwardRef = /*#__PURE__*/ (0, $jD6st$forwardRef)($b68d60985b2d447f$var$SvgComponent);
var $b68d60985b2d447f$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($b68d60985b2d447f$var$ForwardRef);


export {$b68d60985b2d447f$export$2e2bcd8739ae039 as default};
