// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_animation_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $6z7hc$jsxs, jsx as $6z7hc$jsx} from "react/jsx-runtime";
import {forwardRef as $6z7hc$forwardRef} from "react";





const $321f829af758868a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6z7hc$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6z7hc$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6z7hc$jsxs)("linearGradient", {
                        id: "d08fdd__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6z7hc$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6z7hc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6z7hc$jsxs)("linearGradient", {
                        id: "d08fdd__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6z7hc$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6z7hc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6z7hc$jsxs)("linearGradient", {
                        id: "d08fdd__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6z7hc$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6z7hc$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6z7hc$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6z7hc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6z7hc$jsxs)("radialGradient", {
                        id: "d08fdd__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6z7hc$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6z7hc$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6z7hc$jsx)("clipPath", {
                        id: "d08fdd__b",
                        children: /*#__PURE__*/ (0, $6z7hc$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6z7hc$jsx)("clipPath", {
                id: "d08fdd__a",
                children: /*#__PURE__*/ (0, $6z7hc$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M136.004 30.807c0 3.293-1.998 6.12-4.848 7.335-2.815-3.466-6.671-6.053-11.093-7.282l-.001-.053c0-4.402 3.569-7.971 7.971-7.971s7.971 3.569 7.971 7.97m-7.973 21.085c0 5.786-3.554 10.742-8.598 12.805-3.844-7.275-10.719-12.702-18.943-14.6.88-6.789 6.684-12.032 13.713-12.032 7.637 0 13.828 6.19 13.828 13.827m-25.604 7.253c3.042 1.354 5.699 3.418 7.758 5.982 2.855 3.555 4.563 8.07 4.563 12.983 0 11.46-9.29 20.749-20.75 20.749q-.352 0-.703-.012c-2.176-11.45-9.36-21.127-19.186-26.666 2.552-8.57 10.49-14.82 19.89-14.82 3 0 5.853.637 8.428 1.784m-16.445 47.028c0 17.117-13.876 30.993-30.993 30.993s-30.993-13.876-30.993-30.993S37.872 75.18 54.99 75.18c6.937 0 13.342 2.279 18.506 6.129 5.075 3.783 8.953 9.084 10.961 15.23.99 3.032 1.526 6.271 1.526 9.634"
                })
            }),
            /*#__PURE__*/ (0, $6z7hc$jsx)("g", {
                clipPath: "url(#d08fdd__a)",
                children: /*#__PURE__*/ (0, $6z7hc$jsxs)("g", {
                    clipPath: "url(#d08fdd__b)",
                    children: [
                        /*#__PURE__*/ (0, $6z7hc$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6z7hc$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#d08fdd__c)"
                        }),
                        /*#__PURE__*/ (0, $6z7hc$jsx)("path", {
                            fill: "url(#d08fdd__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $6z7hc$jsx)("path", {
                            fill: "url(#d08fdd__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $6z7hc$jsx)("path", {
                            fill: "url(#d08fdd__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $321f829af758868a$var$ForwardRef = /*#__PURE__*/ (0, $6z7hc$forwardRef)($321f829af758868a$var$SvgComponent);
var $321f829af758868a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($321f829af758868a$var$ForwardRef);


export {$321f829af758868a$export$2e2bcd8739ae039 as default};
