// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_volumeOne_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $gBsND$reactjsxruntime = require("react/jsx-runtime");
var $gBsND$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1db67a9ea031d64b$export$2e2bcd8739ae039);




const $1db67a9ea031d64b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2fef3c__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2fef3c__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsxs)("linearGradient", {
                        id: "2fef3c__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsxs)("radialGradient", {
                        id: "2fef3c__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("clipPath", {
                        id: "2fef3c__b",
                        children: /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("clipPath", {
                id: "2fef3c__a",
                children: /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M13.746 16.027h4.031v16.011h-4.03c-1.105 0-2-.895-2-2v-12.01c0-1.105.895-2 2-2m6.531 17.502V14.522l4.897-4.856c.947-.938 2.557-.268 2.557 1.066V37.27c0 1.33-1.602 2.002-2.551 1.07zm12.492-16.54c-.551-.415-1.335-.304-1.75.248-.415.551-.304 1.335.247 1.75 1.515 1.14 2.49 2.948 2.49 4.985 0 2.053-.992 3.874-2.527 5.012-.555.411-.671 1.194-.26 1.749.41.554 1.194.67 1.748.26 2.145-1.59 3.539-4.143 3.539-7.021 0-2.856-1.372-5.391-3.487-6.983"
                })
            }),
            /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("g", {
                clipPath: "url(#2fef3c__a)",
                children: /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#2fef3c__b)",
                    children: [
                        /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#2fef3c__c)"
                        }),
                        /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("path", {
                            fill: "url(#2fef3c__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("path", {
                            fill: "url(#2fef3c__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $gBsND$reactjsxruntime.jsx)("path", {
                            fill: "url(#2fef3c__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $1db67a9ea031d64b$var$ForwardRef = /*#__PURE__*/ (0, $gBsND$react.forwardRef)($1db67a9ea031d64b$var$SvgComponent);
var $1db67a9ea031d64b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($1db67a9ea031d64b$var$ForwardRef);
