// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_pieChart_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $a9JKa$reactjsxruntime = require("react/jsx-runtime");
var $a9JKa$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $7606a52c1c8864a6$export$2e2bcd8739ae039);




const $7606a52c1c8864a6$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsxs)("linearGradient", {
                        id: "976932__d",
                        x1: 109.277,
                        x2: 41.838,
                        y1: 19.392,
                        y2: 112.146,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsxs)("linearGradient", {
                        id: "976932__e",
                        x1: 64.961,
                        x2: 77.463,
                        y1: -21.535,
                        y2: 63.345,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsxs)("linearGradient", {
                        id: "976932__f",
                        x1: -96.944,
                        x2: 138.04,
                        y1: 148.659,
                        y2: 99.394,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsxs)("radialGradient", {
                        id: "976932__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 170.948 -181.344 0 130.355 101.038)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("clipPath", {
                        id: "976932__b",
                        children: /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.199,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("clipPath", {
                id: "976932__a",
                children: /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M76 24.16c-12.45.88-23.78 5.83-32.66 13.53L76 70.35zM37.69 43.34C29.17 53.16 24 65.97 24 80s5.17 26.83 13.69 36.66L74.35 80zM84 80V24.17c29.06 2.05 52 24.91 52 55.84s-25.07 56-56 56c-14.03 0-26.84-5.17-36.66-13.69l39.49-39.49.006-.006q.12-.125.227-.253.14-.175.267-.36c.09-.14.16-.29.23-.44l.01-.02.012-.023.028-.053q.042-.074.08-.155c.056-.136.093-.273.131-.415v-.001L83.85 81l.013-.042.016-.048z"
                })
            }),
            /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("g", {
                clipPath: "url(#976932__a)",
                children: /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#976932__b)",
                    children: [
                        /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("rect", {
                            width: 125.6,
                            height: 118.4,
                            x: 18.398,
                            y: 23.199,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("ellipse", {
                            cx: 130.355,
                            cy: 101.038,
                            fill: "url(#976932__c)",
                            rx: 181.344,
                            ry: 170.948
                        }),
                        /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("path", {
                            fill: "url(#976932__d)",
                            d: "M7.095 60.284C5.08 19.359 38.638-15.356 82.051-17.257s80.24 29.735 82.256 70.66c2.016 40.923-31.543 75.639-74.956 77.54-43.413 1.9-80.24-29.735-82.256-70.66"
                        }),
                        /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("path", {
                            fill: "url(#976932__e)",
                            d: "M50.129-58.73c-9.794-5.713-22.202-5.748-32.032-.09L-80.218-2.233c-9.629 5.542-9.645 18.623-.03 24.186L17.42 78.46c9.814 5.678 22.223 5.671 32.03-.02l97.464-56.56c9.564-5.551 9.582-18.56.029-24.131z"
                        }),
                        /*#__PURE__*/ (0, $a9JKa$reactjsxruntime.jsx)("path", {
                            fill: "url(#976932__f)",
                            d: "M-44.215 79.83c20.264 0 38.62 7.731 51.91 20.204 26.58 24.988 48.615 25.009 75.195.064 13.29-12.537 31.668-20.29 51.955-20.29 40.551.022 73.401 30.904 73.401 69.027s-32.85 68.984-73.401 68.984c-20.332 0-38.71-7.774-52-20.311-26.557-24.903-48.57-24.881-75.15.107-13.29 12.494-31.646 20.204-51.91 20.204-40.529 0-73.378-30.883-73.378-68.984s32.85-69.005 73.378-69.005"
                        })
                    ]
                })
            })
        ]
    });
const $7606a52c1c8864a6$var$ForwardRef = /*#__PURE__*/ (0, $a9JKa$react.forwardRef)($7606a52c1c8864a6$var$SvgComponent);
var $7606a52c1c8864a6$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($7606a52c1c8864a6$var$ForwardRef);
