// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_update_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $6lQwp$reactjsxruntime = require("react/jsx-runtime");
var $6lQwp$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $03b7d923d4d1b289$export$2e2bcd8739ae039);




const $03b7d923d4d1b289$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7a232f__d",
                        x1: 112.885,
                        x2: 36.788,
                        y1: 3.703,
                        y2: 108.364,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7a232f__e",
                        x1: 62.879,
                        x2: 76.986,
                        y1: -42.476,
                        y2: 53.3,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7a232f__f",
                        x1: -119.811,
                        x2: 145.339,
                        y1: 149.566,
                        y2: 93.977,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsxs)("radialGradient", {
                        id: "7a232f__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 192.894 -204.624 0 136.671 95.832)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("clipPath", {
                        id: "7a232f__b",
                        children: /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("rect", {
                            width: 141.724,
                            height: 133.6,
                            x: 10.34,
                            y: 8,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsxs)("clipPath", {
                id: "7a232f__a",
                children: [
                    /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("path", {
                        fill: "#fff",
                        fillRule: "evenodd",
                        d: "M140.117 41.9c1-1.97 3.41-2.75 5.38-1.75h.01c1.97 1 2.75 3.41 1.75 5.38l-10.98 21.57c-.71 1.39-2.11 2.19-3.57 2.19-.57 0-1.14-.13-1.69-.38l-21.17-9.87c-2.01-.94-2.88-3.32-1.94-5.32s3.32-2.87 5.32-1.93l13.83 6.45c-8.49-18.05-26.99-29.96-47.11-29.96-22.97 0-43.5 15.38-49.93 37.4-.51 1.75-2.11 2.88-3.84 2.88-.37 0-.75-.05-1.12-.16-2.12-.62-3.34-2.84-2.72-4.96 7.42-25.41 31.11-43.16 57.61-43.16 22.93 0 44.03 13.4 53.98 33.79zm-10.319 52.99c.62-2.12 2.84-3.34 4.96-2.72v.01c2.12.62 3.34 2.84 2.72 4.96-7.42 25.409-31.11 43.159-57.61 43.159-22.82 0-43.83-13.28-53.84-33.51l-6.15 12.09c-.71 1.39-2.11 2.19-3.57 2.19-.61 0-1.23-.14-1.81-.44-1.97-1-2.75-3.41-1.75-5.38l10.98-21.58c.98-1.92 3.31-2.72 5.26-1.81l21.17 9.87c2.01.94 2.87 3.32 1.94 5.32-.94 2-3.32 2.86-5.32 1.93l-13.95-6.51c8.51 17.98 26.97 29.82 47.04 29.82 22.97 0 43.5-15.38 49.93-37.4"
                    }),
                    /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("path", {
                        fill: "#fff",
                        d: "m145.497 40.15-.452.891.213.108h.239zm-5.38 1.75.892.453v-.001zm5.39-1.75.453-.892-.213-.109h-.24zm1.75 5.38.891.453.001-.001zm-10.98 21.57.891.454v-.001zm-5.26 1.81-.422.906.008.004zm-21.17-9.87-.423.905h.001zm3.38-7.25-.425.904.003.002zm13.83 6.45-.422.906 2.503 1.167-1.176-2.5zm-97.04 7.44-.96-.281zm-4.96 2.72.285-.96h-.004zm-2.72-4.96.96.28zm111.59-9.37-.898.438.88 1.803.91-1.788zm.831 38.1h1v-.75l-.719-.21zm-4.96 2.72-.96-.281zm4.96-2.71h-1v.749l.719.21zm2.72 4.96-.96-.281zm-111.45 9.649.896-.443-.884-1.787-.903 1.777zm-6.15 12.09.89.455.001-.001zm-5.38 1.75.46-.888-.007-.003zm-1.75-5.38-.891-.453v.001zm10.98-21.58-.89-.454h-.001zm5.26-1.81-.423.907zm21.17 9.87.424-.906h-.002zm1.94 5.32.905.426.002-.004zm-5.32 1.93-.423.906.001.001zm-13.95-6.51.423-.906-2.513-1.173 1.186 2.507zM145.95 39.258c-2.46-1.249-5.473-.277-6.724 2.189l1.783.905c.749-1.475 2.555-2.062 4.036-1.311zm-.443-.109h-.01v2h.01zm2.642 6.833c1.249-2.46.276-5.473-2.189-6.724l-.905 1.783c1.474.749 2.062 2.555 1.311 4.036zm-10.981 21.57 10.98-21.57-1.782-.906-10.98 21.57zm-4.461 2.737c1.824 0 3.575-1 4.461-2.735l-1.781-.91c-.534 1.045-1.583 1.645-2.68 1.645zm-2.104-.47c.67.305 1.38.47 2.104.47v-2c-.415 0-.845-.094-1.276-.29zm-21.178-9.873 21.17 9.87.845-1.813-21.17-9.87zm-2.423-6.652c-1.177 2.505-.084 5.48 2.422 6.651l.847-1.812c-1.514-.708-2.161-2.494-1.459-3.988zm6.651-2.41c-2.505-1.177-5.478-.085-6.651 2.41l1.81.85c.708-1.504 2.494-2.153 3.99-1.45zm13.827 6.449-13.83-6.45-.845 1.813 13.83 6.45zM79.947 29.279c19.729 0 37.879 11.683 46.205 29.386l1.81-.851c-8.653-18.397-27.503-30.535-48.015-30.535zm-48.97 36.68c6.305-21.592 26.444-36.68 48.97-36.68v-2c-23.413 0-44.335 15.673-50.89 38.12zm-4.8 3.6c2.16 0 4.162-1.41 4.8-3.6l-1.92-.56c-.381 1.31-1.58 2.16-2.88 2.16zm-1.405-.201c.47.14.946.201 1.405.201v-2c-.28 0-.564-.038-.835-.118zm-3.394-6.2c-.776 2.653.75 5.427 3.399 6.201l.561-1.92c-1.592-.465-2.505-2.13-2.04-3.719zm58.57-43.879c-26.944 0-51.026 18.043-58.57 43.88l1.92.56c7.295-24.982 30.593-42.44 56.65-42.44zm54.878 34.352C124.71 32.9 103.264 19.279 79.947 19.279v2c22.544 0 43.297 13.179 53.082 33.229zm4.4-12.185-6.19 12.17 1.783.907 6.19-12.17zm-4.187 49.764c-2.652-.776-5.426.75-6.201 3.399l1.92.561c.465-1.592 2.131-2.505 3.719-2.04zm.719.97v-.01h-2v.01zm2.68 5.24c.775-2.652-.751-5.426-3.399-6.2l-.562 1.92c1.592.465 2.506 2.13 2.041 3.719zm-58.57 43.879c26.944 0 51.025-18.042 58.57-43.88l-1.92-.56c-7.295 24.983-30.594 42.44-56.65 42.44zm-54.736-34.066c10.176 20.566 31.53 34.066 54.736 34.066v-2c-22.435 0-43.1-13.06-52.944-32.953zm-4.363 12.1 6.15-12.09-1.782-.907-6.15 12.09zm-4.461 2.736c1.823 0 3.574-1 4.46-2.735l-1.78-.91c-.534 1.045-1.584 1.645-2.68 1.645zm-2.27-.551c.728.376 1.506.551 2.27.551v-2c-.456 0-.918-.104-1.35-.328zm-2.182-6.721c-1.248 2.459-.276 5.473 2.19 6.724l.905-1.783c-1.475-.749-2.063-2.556-1.311-4.036zm10.98-21.581-10.98 21.58 1.783.907 10.98-21.58zm6.575-2.263c-2.438-1.138-5.349-.138-6.574 2.262l1.782.909c.735-1.44 2.484-2.04 3.946-1.358zm21.17 9.87-21.17-9.87-.846 1.813 21.17 9.87zm2.424 6.648c1.162-2.5.087-5.474-2.423-6.648l-.848 1.812c1.51.706 2.155 2.492 1.457 3.993zm-6.649 2.415c2.499 1.162 5.472.088 6.647-2.411l-1.81-.851c-.706 1.501-2.492 2.147-3.993 1.449zm-13.95-6.511 13.949 6.51.846-1.812-13.95-6.51zm47.462 27.914c-19.68 0-37.79-11.614-46.136-29.248l-1.808.856c8.674 18.326 27.484 30.392 47.944 30.392zm48.97-36.68c-6.305 21.593-26.444 36.68-48.97 36.68v2c23.414 0 44.335-15.672 50.89-38.12z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("g", {
                clipPath: "url(#7a232f__a)",
                children: /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#7a232f__b)",
                    children: [
                        /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("rect", {
                            width: 141.724,
                            height: 133.6,
                            x: 10.34,
                            y: 8,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("ellipse", {
                            cx: 136.671,
                            cy: 95.832,
                            fill: "url(#7a232f__c)",
                            rx: 204.624,
                            ry: 192.894
                        }),
                        /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("path", {
                            fill: "url(#7a232f__d)",
                            d: "M-2.415 49.844C-4.69 3.666 33.177-35.507 82.163-37.651S172.704-4.1 174.98 42.079 139.387 127.43 90.4 129.574-.14 96.022-2.415 49.844"
                        }),
                        /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("path", {
                            fill: "url(#7a232f__e)",
                            d: "M46.142-84.446c-11.05-6.446-25.052-6.486-36.143-.102l-110.937 63.851c-10.865 6.254-10.883 21.015-.033 27.292L9.233 70.355c11.075 6.408 25.076 6.4 36.141-.022l109.977-63.82c10.793-6.264 10.813-20.943.033-27.23z"
                        }),
                        /*#__PURE__*/ (0, $6lQwp$reactjsxruntime.jsx)("path", {
                            fill: "url(#7a232f__f)",
                            d: "M-60.313 71.901c22.865 0 43.578 8.724 58.574 22.798 29.992 28.195 54.857 28.22 84.849.072 14.996-14.146 35.734-22.894 58.625-22.894 45.757.024 82.824 34.871 82.824 77.888 0 43.016-37.067 77.839-82.824 77.839-22.943 0-43.68-8.772-58.677-22.918-29.966-28.099-54.805-28.075-84.797.121-14.996 14.098-35.709 22.797-58.574 22.797-45.732 0-82.798-34.847-82.798-77.839 0-42.993 37.066-77.864 82.798-77.864"
                        })
                    ]
                })
            })
        ]
    });
const $03b7d923d4d1b289$var$ForwardRef = /*#__PURE__*/ (0, $6lQwp$react.forwardRef)($03b7d923d4d1b289$var$SvgComponent);
var $03b7d923d4d1b289$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($03b7d923d4d1b289$var$ForwardRef);
