// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_targeted_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $dwu2M$reactjsxruntime = require("react/jsx-runtime");
var $dwu2M$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $3fc7ad5d43305d6a$export$2e2bcd8739ae039);




const $3fc7ad5d43305d6a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7d556d__e",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7d556d__f",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsxs)("linearGradient", {
                        id: "7d556d__g",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("clipPath", {
                        id: "7d556d__a",
                        children: /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("path", {
                            fill: "#fff",
                            d: "M0 0h48v48H0z"
                        })
                    }),
                    /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("clipPath", {
                        id: "7d556d__c",
                        children: /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    }),
                    /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsxs)("radialGradient", {
                        id: "7d556d__d",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsxs)("g", {
                clipPath: "url(#7d556d__a)",
                children: [
                    /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("clipPath", {
                        id: "7d556d__b",
                        children: /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("path", {
                            fill: "#fff",
                            fillRule: "evenodd",
                            d: "M4.808 9.153c-.348.089-.621.358-.716.704-.095.347.003.717.257.971l4.6 4.6c.207.207.495.313.787.289l3.775-.307 6.374 6.373c.488.488 1.28.488 1.767 0 .489-.488.489-1.28 0-1.768l-6.373-6.373.307-3.775c.024-.293-.082-.58-.29-.788l-4.598-4.6c-.254-.253-.625-.351-.971-.256-.347.094-.616.368-.705.716l-.86 3.354zM24 8.869c-1.744 0-3.416.295-4.972.836-.652.227-1.364-.118-1.591-.77s.118-1.365.77-1.591c1.816-.632 3.766-.975 5.793-.975C33.736 6.37 41.63 14.263 41.63 24s-7.894 17.632-17.631 17.632S6.368 33.738 6.368 24c0-1.956.319-3.84.909-5.602.22-.654.928-1.007 1.582-.788.655.22 1.008.928.789 1.582-.506 1.51-.78 3.125-.78 4.808 0 8.357 6.774 15.132 15.131 15.132S39.13 32.357 39.13 24 32.356 8.87 23.999 8.87m0 7.753c-.802 0-1.57.128-2.29.362-.656.214-1.362-.145-1.576-.8-.214-.657.145-1.363.801-1.577.966-.315 1.997-.485 3.064-.485 5.456 0 9.878 4.423 9.878 9.878s-4.422 9.877-9.878 9.877c-5.455 0-9.877-4.422-9.877-9.877 0-1.006.15-1.979.431-2.896.203-.66.902-1.031 1.562-.83.66.203 1.031.902.829 1.562-.21.683-.322 1.409-.322 2.164 0 4.074 3.303 7.378 7.377 7.378 4.075 0 7.378-3.304 7.378-7.378 0-4.075-3.303-7.378-7.378-7.378m-2.673 9.031c2.26-.21 4.059-2.01 4.27-4.269.903.544 1.507 1.535 1.507 2.666 0 1.718-1.392 3.111-3.11 3.111-1.132 0-2.122-.604-2.667-1.508"
                        })
                    }),
                    /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("g", {
                        clipPath: "url(#7d556d__b)",
                        children: /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsxs)("g", {
                            clipPath: "url(#7d556d__c)",
                            children: [
                                /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("rect", {
                                    width: 48,
                                    height: 48,
                                    fill: "#D9F4FD",
                                    rx: 10
                                }),
                                /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("circle", {
                                    cx: 42.788,
                                    cy: 31.555,
                                    r: 69.303,
                                    fill: "url(#7d556d__d)"
                                }),
                                /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("path", {
                                    fill: "url(#7d556d__e)",
                                    d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                                }),
                                /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("path", {
                                    fill: "url(#7d556d__f)",
                                    d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                                }),
                                /*#__PURE__*/ (0, $dwu2M$reactjsxruntime.jsx)("path", {
                                    fill: "url(#7d556d__g)",
                                    d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                                })
                            ]
                        })
                    })
                ]
            })
        ]
    });
const $3fc7ad5d43305d6a$var$ForwardRef = /*#__PURE__*/ (0, $dwu2M$react.forwardRef)($3fc7ad5d43305d6a$var$SvgComponent);
var $3fc7ad5d43305d6a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($3fc7ad5d43305d6a$var$ForwardRef);
