// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_rotateCW_generic2_48.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $U32HU$reactjsxruntime = require("react/jsx-runtime");
var $U32HU$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $8c4a80062171da68$export$2e2bcd8739ae039);




const $8c4a80062171da68$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "342de2__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "342de2__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsxs)("linearGradient", {
                        id: "342de2__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsxs)("radialGradient", {
                        id: "342de2__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("clipPath", {
                        id: "342de2__b",
                        children: /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("clipPath", {
                id: "342de2__a",
                children: /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M7.754 24.299c0-8.422 6.827-15.25 15.25-15.25 6.025 0 11.238 3.495 13.714 8.572l-4.671-1.47c-.659-.208-1.36.158-1.568.816-.207.659.158 1.36.817 1.568l7.442 2.343c.658.208 1.36-.158 1.568-.817l2.343-7.442c.207-.658-.158-1.36-.817-1.567-.658-.208-1.36.158-1.568.817l-1.401 4.45c-2.92-5.794-8.924-9.77-15.86-9.77-9.802 0-17.75 7.947-17.75 17.75s7.948 17.75 17.75 17.75c7.668 0 14.197-4.86 16.68-11.664.236-.649-.098-1.367-.746-1.603-.649-.237-1.366.097-1.603.745-2.135 5.85-7.747 10.022-14.33 10.022-8.423 0-15.25-6.828-15.25-15.25"
                })
            }),
            /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("g", {
                clipPath: "url(#342de2__a)",
                children: /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#342de2__b)",
                    children: [
                        /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#342de2__c)"
                        }),
                        /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("path", {
                            fill: "url(#342de2__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("path", {
                            fill: "url(#342de2__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $U32HU$reactjsxruntime.jsx)("path", {
                            fill: "url(#342de2__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $8c4a80062171da68$var$ForwardRef = /*#__PURE__*/ (0, $U32HU$react.forwardRef)($8c4a80062171da68$var$SvgComponent);
var $8c4a80062171da68$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($8c4a80062171da68$var$ForwardRef);
