// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_rotateCW_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $cGjQu$jsxs, jsx as $cGjQu$jsx} from "react/jsx-runtime";
import {forwardRef as $cGjQu$forwardRef} from "react";





const $992ee835805ed1ca$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $cGjQu$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $cGjQu$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $cGjQu$jsxs)("linearGradient", {
                        id: "65f308__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGjQu$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $cGjQu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGjQu$jsxs)("linearGradient", {
                        id: "65f308__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGjQu$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $cGjQu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGjQu$jsxs)("linearGradient", {
                        id: "65f308__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGjQu$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $cGjQu$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $cGjQu$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $cGjQu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGjQu$jsxs)("radialGradient", {
                        id: "65f308__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $cGjQu$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $cGjQu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $cGjQu$jsx)("clipPath", {
                        id: "65f308__b",
                        children: /*#__PURE__*/ (0, $cGjQu$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $cGjQu$jsx)("clipPath", {
                id: "65f308__a",
                children: /*#__PURE__*/ (0, $cGjQu$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M26.5 80.398c0-28.718 23.281-52 52-52 19.918 0 37.227 11.2 45.962 27.655l-15.454-5.156c-2.095-.7-4.361.432-5.06 2.528s.433 4.361 2.528 5.06l24.275 8.1c2.095.699 4.361-.433 5.06-2.529l8.099-24.274c.7-2.096-.432-4.361-2.528-5.06-2.095-.7-4.361.432-5.06 2.528l-4.935 14.79C121.269 33.208 101.384 20.397 78.5 20.397c-33.137 0-60 26.863-60 60s26.863 60 60 60c25.917 0 47.986-16.429 56.38-39.428.757-2.075-.311-4.372-2.387-5.13-2.075-.757-4.371.312-5.129 2.387-7.278 19.943-26.415 34.171-48.864 34.171-28.719 0-52-23.281-52-52"
                })
            }),
            /*#__PURE__*/ (0, $cGjQu$jsx)("g", {
                clipPath: "url(#65f308__a)",
                children: /*#__PURE__*/ (0, $cGjQu$jsxs)("g", {
                    clipPath: "url(#65f308__b)",
                    children: [
                        /*#__PURE__*/ (0, $cGjQu$jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $cGjQu$jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#65f308__c)"
                        }),
                        /*#__PURE__*/ (0, $cGjQu$jsx)("path", {
                            fill: "url(#65f308__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $cGjQu$jsx)("path", {
                            fill: "url(#65f308__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $cGjQu$jsx)("path", {
                            fill: "url(#65f308__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $992ee835805ed1ca$var$ForwardRef = /*#__PURE__*/ (0, $cGjQu$forwardRef)($992ee835805ed1ca$var$SvgComponent);
var $992ee835805ed1ca$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($992ee835805ed1ca$var$ForwardRef);


export {$992ee835805ed1ca$export$2e2bcd8739ae039 as default};
