// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_mailClose_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $9IJ5v$reactjsxruntime = require("react/jsx-runtime");
var $9IJ5v$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b2a167ba15387418$export$2e2bcd8739ae039);




const $b2a167ba15387418$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsxs)("linearGradient", {
                        id: "809965__d",
                        x1: 109.5,
                        x2: 39.669,
                        y1: 15.057,
                        y2: 111.101,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsxs)("linearGradient", {
                        id: "809965__e",
                        x1: 63.614,
                        x2: 76.559,
                        y1: -27.321,
                        y2: 60.569,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsxs)("linearGradient", {
                        id: "809965__f",
                        x1: -104.036,
                        x2: 139.284,
                        y1: 148.91,
                        y2: 97.897,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsxs)("radialGradient", {
                        id: "809965__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 177.012 -187.776 0 131.327 99.599)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("clipPath", {
                        id: "809965__b",
                        children: /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("rect", {
                            width: 130.055,
                            height: 122.6,
                            x: 15.398,
                            y: 19,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("clipPath", {
                id: "809965__a",
                children: /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M33.846 34h92.308c3.691 0 7.045 1.6 9.527 4.206L82.56 69.2c-1.864 1.088-4.155 1.066-6-.056l-51.7-31.476C27.28 35.38 30.418 34 33.847 34M20.702 44.501c-.456 1.52-.702 3.144-.702 4.832v61.334C20 119.135 26.2 126 33.846 126h92.308c7.647 0 13.846-6.865 13.846-15.333V49.333c0-1.417-.174-2.79-.499-4.093l-52.91 30.87c-4.401 2.567-9.837 2.516-14.19-.134z"
                })
            }),
            /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("g", {
                clipPath: "url(#809965__a)",
                children: /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#809965__b)",
                    children: [
                        /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("rect", {
                            width: 130.055,
                            height: 122.6,
                            x: 15.398,
                            y: 19,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("ellipse", {
                            cx: 131.327,
                            cy: 99.599,
                            fill: "url(#809965__c)",
                            rx: 187.776,
                            ry: 177.012
                        }),
                        /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("path", {
                            fill: "url(#809965__d)",
                            d: "M3.694 57.4C1.606 15.022 36.356-20.925 81.308-22.893s83.087 30.79 85.174 73.166-32.662 78.323-77.614 80.291C43.915 132.532 5.78 99.775 3.694 57.399"
                        }),
                        /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("path", {
                            fill: "url(#809965__e)",
                            d: "M48.255-65.836c-10.14-5.915-22.99-5.951-33.168-.093L-86.715-7.335c-9.97 5.739-9.987 19.284-.03 25.044l101.13 58.512C24.548 82.1 37.396 82.093 47.55 76.2l100.921-58.566c9.904-5.748 9.922-19.218.03-24.988z"
                        }),
                        /*#__PURE__*/ (0, $9IJ5v$reactjsxruntime.jsx)("path", {
                            fill: "url(#809965__f)",
                            d: "M-49.436 77.64c20.983 0 39.99 8.005 53.75 20.92 27.523 25.874 50.341 25.896 77.864.066 13.761-12.981 32.791-21.009 53.798-21.009 41.989.022 76.004 32 76.004 71.475s-34.015 71.431-76.004 71.431c-21.054 0-40.084-8.05-53.845-21.031-27.5-25.786-50.294-25.764-77.816.11-13.761 12.937-32.768 20.921-53.751 20.921-41.966 0-75.981-31.978-75.981-71.431s34.015-71.453 75.98-71.453"
                        })
                    ]
                })
            })
        ]
    });
const $b2a167ba15387418$var$ForwardRef = /*#__PURE__*/ (0, $9IJ5v$react.forwardRef)($b2a167ba15387418$var$SvgComponent);
var $b2a167ba15387418$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($b2a167ba15387418$var$ForwardRef);
