// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_browser_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $f2dIF$reactjsxruntime = require("react/jsx-runtime");
var $f2dIF$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $a22292c5526fbc46$export$2e2bcd8739ae039);




const $a22292c5526fbc46$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "36c8fe__d",
                        x1: 65.566,
                        x2: 25.102,
                        y1: 11.635,
                        y2: 67.287,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "36c8fe__e",
                        x1: 38.978,
                        x2: 46.479,
                        y1: -12.921,
                        y2: 38.007,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsxs)("linearGradient", {
                        id: "36c8fe__f",
                        x1: -58.165,
                        x2: 82.825,
                        y1: 89.196,
                        y2: 59.637,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsxs)("radialGradient", {
                        id: "36c8fe__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 78.212 60.623)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("clipPath", {
                        id: "36c8fe__b",
                        children: /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("clipPath", {
                id: "36c8fe__a",
                children: /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M21.6 16.8h52.8c5.304 0 9.6 4.296 9.6 9.6v7.2H12v-7.2c0-5.304 4.296-9.6 9.6-9.6m-2.388 9.6c0 1.326 1.074 2.4 2.4 2.4s2.4-1.074 2.4-2.4-1.074-2.4-2.4-2.4h-.006c-1.32 0-2.394 1.074-2.394 2.4M21.6 79.2c-5.304 0-9.6-4.296-9.6-9.6V38.4h72v31.2c0 5.304-4.296 9.6-9.6 9.6z"
                })
            }),
            /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("g", {
                clipPath: "url(#36c8fe__a)",
                children: /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#36c8fe__b)",
                    children: [
                        /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("ellipse", {
                            cx: 78.212,
                            cy: 60.623,
                            fill: "url(#36c8fe__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("path", {
                            fill: "url(#36c8fe__d)",
                            d: "M4.256 36.17C3.046 11.615 23.182-9.214 49.23-10.354S97.374 7.486 98.583 32.04 79.658 77.425 53.61 78.565 5.466 60.725 4.256 36.17"
                        }),
                        /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("path", {
                            fill: "url(#36c8fe__e)",
                            d: "M30.078-35.238c-5.876-3.427-13.321-3.448-19.219-.054L-48.13-1.34c-5.777 3.326-5.787 11.174-.017 14.512l58.599 33.904c5.889 3.407 13.334 3.403 19.218-.012l58.478-33.936c5.74-3.33 5.75-11.135.018-14.479z"
                        }),
                        /*#__PURE__*/ (0, $f2dIF$reactjsxruntime.jsx)("path", {
                            fill: "url(#36c8fe__f)",
                            d: "M-26.528 47.899c12.159 0 23.172 4.638 31.146 12.122 15.948 14.993 29.17 15.005 45.118.038 7.974-7.522 19-12.173 31.173-12.173 24.33.013 44.04 18.542 44.04 41.416s-19.71 41.39-44.04 41.39c-12.2 0-23.227-4.665-31.2-12.187-15.935-14.941-29.143-14.928-45.09.064-7.975 7.497-18.988 12.123-31.147 12.123-24.317 0-44.026-18.53-44.026-41.39 0-22.861 19.71-41.403 44.026-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $a22292c5526fbc46$var$ForwardRef = /*#__PURE__*/ (0, $f2dIF$react.forwardRef)($a22292c5526fbc46$var$SvgComponent);
var $a22292c5526fbc46$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($a22292c5526fbc46$var$ForwardRef);
