// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_animation_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $6MJNO$reactjsxruntime = require("react/jsx-runtime");
var $6MJNO$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $cd61630fe7601677$export$2e2bcd8739ae039);




const $cd61630fe7601677$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4f8d2a__d",
                        x1: 69.462,
                        x2: 13.866,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4f8d2a__e",
                        x1: 35.59,
                        x2: 46.315,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsxs)("linearGradient", {
                        id: "4f8d2a__f",
                        x1: -88.161,
                        x2: 92.291,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsxs)("radialGradient", {
                        id: "4f8d2a__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 11.23 74.342)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("clipPath", {
                        id: "4f8d2a__b",
                        children: /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("clipPath", {
                id: "4f8d2a__a",
                children: /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M82.053 18.087c0 2.331-1.646 4.278-3.84 4.742-1.505-1.755-3.494-3.083-5.758-3.78q-.095-.468-.096-.962c0-2.677 2.17-4.847 4.847-4.847s4.847 2.17 4.847 4.847m-4.848 12.821c0 3.827-2.556 7.056-6.054 8.074-2.213-4.11-6.106-7.18-10.76-8.293.117-4.542 3.835-8.189 8.405-8.189 4.644 0 8.409 3.765 8.409 8.408M51.637 63.915c0 10.408-8.438 18.846-18.846 18.846s-18.846-8.438-18.846-18.846S22.383 45.07 32.791 45.07c4.218 0 8.113 1.386 11.253 3.727 3.086 2.3 5.443 5.523 6.664 9.26.603 1.844.929 3.813.929 5.86m17.49-17.065c0 6.968-5.648 12.617-12.616 12.617q-.669 0-1.32-.069c-1.31-6.533-5.408-12.059-10.999-15.285 1.25-5.652 6.291-9.88 12.32-9.88 6.967 0 12.616 5.649 12.616 12.617"
                })
            }),
            /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("g", {
                clipPath: "url(#4f8d2a__a)",
                children: /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#4f8d2a__b)",
                    children: [
                        /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("rect", {
                            width: 96,
                            height: 96,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("circle", {
                            cx: 85.571,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#4f8d2a__c)"
                        }),
                        /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("path", {
                            fill: "url(#4f8d2a__d)",
                            d: "M-8.64 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.182-1.541 61.33 24.109 62.871 57.29 1.54 33.182-24.11 61.33-57.291 62.871-33.182 1.54-61.33-24.11-62.871-57.291"
                        }),
                        /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("path", {
                            fill: "url(#4f8d2a__e)",
                            d: "M24.253-66.43c-7.486-4.632-16.97-4.66-24.483-.073l-75.145 45.881c-7.36 4.494-7.372 15.1-.023 19.61l74.65 45.817c7.501 4.604 16.985 4.598 24.48-.016l74.496-45.86c7.31-4.5 7.324-15.048.022-19.566z"
                        }),
                        /*#__PURE__*/ (0, $6MJNO$reactjsxruntime.jsx)("path", {
                            fill: "url(#4f8d2a__f)",
                            d: "M-47.859 45.918c15.489 0 29.519 6.268 39.677 16.381 20.315 20.26 37.158 20.278 57.474.052 10.158-10.165 24.205-16.45 39.711-16.45 30.994.017 56.102 25.056 56.102 55.967S119.997 157.8 89.003 157.8c-15.54 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087C-18.34 151.549-32.37 157.8-47.858 157.8c-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $cd61630fe7601677$var$ForwardRef = /*#__PURE__*/ (0, $6MJNO$react.forwardRef)($cd61630fe7601677$var$SvgComponent);
var $cd61630fe7601677$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($cd61630fe7601677$var$ForwardRef);
