// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_thumbsUp_generic2_96.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $gFRI6$jsxs, jsx as $gFRI6$jsx} from "react/jsx-runtime";
import {forwardRef as $gFRI6$forwardRef} from "react";





const $2fccb1c1010e8818$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $gFRI6$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 100,
        height: 96,
        viewBox: "0 0 100 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $gFRI6$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $gFRI6$jsxs)("linearGradient", {
                        id: "26a870__d",
                        x1: 71.434,
                        x2: 15.838,
                        y1: -3.088,
                        y2: 68.993,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gFRI6$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $gFRI6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gFRI6$jsxs)("linearGradient", {
                        id: "26a870__e",
                        x1: 37.562,
                        x2: 48.287,
                        y1: -36.272,
                        y2: 32.367,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gFRI6$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $gFRI6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gFRI6$jsxs)("linearGradient", {
                        id: "26a870__f",
                        x1: -86.188,
                        x2: 94.263,
                        y1: 101.725,
                        y2: 66.061,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gFRI6$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $gFRI6$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $gFRI6$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $gFRI6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gFRI6$jsxs)("radialGradient", {
                        id: "26a870__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 12.216 75.328)scale(138.607)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $gFRI6$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $gFRI6$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $gFRI6$jsx)("clipPath", {
                        id: "26a870__b",
                        children: /*#__PURE__*/ (0, $gFRI6$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 1.973,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $gFRI6$jsx)("clipPath", {
                id: "26a870__a",
                children: /*#__PURE__*/ (0, $gFRI6$jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M40.284 76.965c3.22 1.2 6.623 1.816 10.06 1.816l-.011-.012h14.315c.03 0 .089-.005.089-.005h3.966c1.47 0 2.992-.406 4.167-1.305 1.193-.917 1.986-2.321 1.986-4.214 0-.61-.088-1.692-.61-2.75-.248-.511-.6-1.01-1.082-1.446 2.45-.14 4.519-1.915 5.019-4.343.452-2.203-.494-4.319-2.163-5.518 2.274-.564 3.996-2.562 4.12-5 .1-1.975-.864-3.738-2.38-4.773 2.15-.646 3.749-2.585 3.872-4.936.159-3.109-2.321-5.718-5.436-5.718h-16.29c-.887 0-1.522-.864-1.258-1.71l1.176-3.773c1.052-3.373 1.721-6.852 2.01-10.372l.164-2.04c.247-3.038-2.157-5.647-5.213-5.647-2.015 0-3.849 1.163-4.719 2.98l-5.47 11.459q-.77 1.606-1.758 3.08l-.135.205c-3.166 4.71-8.163 7.805-13.696 8.597v32.325c1.404.286 2.787.681 4.134 1.185zm-13.277-3.614v-31.61h-2.238c-3.573 0-6.464 2.89-6.464 6.464v18.64c0 3.574 2.891 6.465 6.464 6.465h.723q.758 0 1.515.041"
                })
            }),
            /*#__PURE__*/ (0, $gFRI6$jsx)("g", {
                clipPath: "url(#26a870__a)",
                children: /*#__PURE__*/ (0, $gFRI6$jsxs)("g", {
                    clipPath: "url(#26a870__b)",
                    children: [
                        /*#__PURE__*/ (0, $gFRI6$jsx)("rect", {
                            width: 96,
                            height: 96,
                            x: 1.973,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $gFRI6$jsx)("circle", {
                            cx: 87.544,
                            cy: 63.112,
                            r: 138.607,
                            fill: "url(#26a870__c)"
                        }),
                        /*#__PURE__*/ (0, $gFRI6$jsx)("path", {
                            fill: "url(#26a870__d)",
                            d: "M-6.667 30.067c-1.54-33.182 24.11-61.33 57.292-62.87 33.181-1.541 61.33 24.109 62.87 57.29 1.541 33.182-24.109 61.33-57.29 62.871-33.183 1.54-61.33-24.11-62.872-57.291"
                        }),
                        /*#__PURE__*/ (0, $gFRI6$jsx)("path", {
                            fill: "url(#26a870__e)",
                            d: "M26.225-66.43c-7.485-4.632-16.969-4.66-24.482-.073l-75.146 45.881c-7.36 4.494-7.371 15.1-.022 19.61L1.224 44.806c7.502 4.604 16.986 4.598 24.481-.016L100.2-1.071c7.311-4.5 7.324-15.048.023-19.566z"
                        }),
                        /*#__PURE__*/ (0, $gFRI6$jsx)("path", {
                            fill: "url(#26a870__f)",
                            d: "M-45.886 45.918c15.489 0 29.519 6.268 39.676 16.381 20.316 20.26 37.159 20.278 57.475.052 10.157-10.165 24.205-16.45 39.71-16.45 30.995.017 56.103 25.056 56.103 55.967S121.97 157.8 90.976 157.8c-15.541 0-29.588-6.303-39.746-16.468-20.298-20.191-37.124-20.174-57.44.087-10.157 10.13-24.187 16.381-39.676 16.381-30.977 0-56.085-25.039-56.085-55.932s25.108-55.95 56.085-55.95"
                        })
                    ]
                })
            })
        ]
    });
const $2fccb1c1010e8818$var$ForwardRef = /*#__PURE__*/ (0, $gFRI6$forwardRef)($2fccb1c1010e8818$var$SvgComponent);
var $2fccb1c1010e8818$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($2fccb1c1010e8818$var$ForwardRef);


export {$2fccb1c1010e8818$export$2e2bcd8739ae039 as default};
