// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_lighten_generic2_160.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $e2CTj$reactjsxruntime = require("react/jsx-runtime");
var $e2CTj$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $21bacf084e198765$export$2e2bcd8739ae039);




const $21bacf084e198765$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c435ac__d",
                        x1: 115.768,
                        x2: 23.108,
                        y1: -5.145,
                        y2: 114.991,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c435ac__e",
                        x1: 59.314,
                        x2: 77.188,
                        y1: -60.452,
                        y2: 53.946,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsxs)("linearGradient", {
                        id: "c435ac__f",
                        x1: -146.934,
                        x2: 153.819,
                        y1: 169.541,
                        y2: 110.102,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsxs)("radialGradient", {
                        id: "c435ac__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 18.716 123.904)scale(231.011)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("clipPath", {
                        id: "c435ac__b",
                        children: /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("clipPath", {
                id: "c435ac__a",
                children: /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    fillRule: "evenodd",
                    d: "M80 20c2.21 0 4 1.79 4 4v7.938c0 2.21-1.79 4-4 4s-4-1.79-4-4V24c0-2.21 1.79-4 4-4m28.283 60.002c0 15.621-12.663 28.284-28.284 28.284S51.717 95.623 51.717 80.002 64.379 51.72 79.999 51.72s28.284 12.663 28.284 28.283M84 128.062c0-2.209-1.79-4-4-4s-4 1.791-4 4V136c0 2.209 1.791 4 4 4s4-1.791 4-4zM140 80c0 2.21-1.791 4-4 4h-7.938c-2.209 0-4-1.79-4-4s1.791-4 4-4H136c2.209 0 4 1.79 4 4M31.938 84c2.21 0 4-1.79 4-4s-1.79-4-4-4H24c-2.21 0-4 1.79-4 4s1.79 4 4 4zm90.488-46.427c1.562 1.562 1.562 4.095 0 5.657l-5.613 5.613c-1.562 1.562-4.095 1.562-5.657 0s-1.562-4.095 0-5.657l5.613-5.613c1.562-1.562 4.095-1.562 5.657 0m-73.583 79.24c1.562-1.562 1.562-4.095 0-5.657s-4.095-1.562-5.657 0l-5.613 5.613c-1.562 1.563-1.562 4.095 0 5.657s4.094 1.562 5.657 0zm-11.27-79.242c-1.563 1.562-1.563 4.095 0 5.657l5.612 5.613c1.563 1.562 4.095 1.562 5.657 0s1.562-4.095 0-5.657l-5.613-5.613c-1.562-1.562-4.095-1.562-5.657 0m73.583 79.24c-1.562-1.562-1.562-4.094 0-5.657s4.095-1.562 5.657 0l5.613 5.614c1.562 1.562 1.562 4.094 0 5.656-1.562 1.563-4.095 1.563-5.657 0z"
                })
            }),
            /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("g", {
                clipPath: "url(#c435ac__a)",
                children: /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#c435ac__b)",
                    children: [
                        /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("rect", {
                            width: 160,
                            height: 160,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("circle", {
                            cx: 142.62,
                            cy: 105.187,
                            r: 231.011,
                            fill: "url(#c435ac__c)"
                        }),
                        /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("path", {
                            fill: "url(#c435ac__d)",
                            d: "M-14.4 50.114C-16.968-5.19 25.782-52.103 81.085-54.671S183.302-14.489 185.87 40.814 145.688 143.031 90.385 145.6-11.832 105.417-14.4 50.114"
                        }),
                        /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("path", {
                            fill: "url(#c435ac__e)",
                            d: "M40.418-110.716c-12.475-7.72-28.282-7.767-40.804-.122l-125.242 76.469c-12.266 7.49-12.286 25.167-.037 32.684L-1.25 74.676c12.503 7.673 28.31 7.664 40.802-.027L163.71-1.783c12.185-7.502 12.207-25.08.038-32.61z"
                        }),
                        /*#__PURE__*/ (0, $e2CTj$reactjsxruntime.jsx)("path", {
                            fill: "url(#c435ac__f)",
                            d: "M-79.763 76.529c25.814 0 49.197 10.448 66.127 27.302 33.86 33.768 61.931 33.797 95.79.087C99.084 86.977 122.497 76.5 148.34 76.5c51.657.029 93.504 41.762 93.504 93.279S199.997 263 148.34 263c-25.901 0-49.314-10.505-66.243-27.447-33.83-33.652-61.873-33.623-95.733.144C-30.566 252.581-53.949 263-79.763 263c-51.629 0-93.475-41.733-93.475-93.221s41.846-93.25 93.475-93.25"
                        })
                    ]
                })
            })
        ]
    });
const $21bacf084e198765$var$ForwardRef = /*#__PURE__*/ (0, $e2CTj$react.forwardRef)($21bacf084e198765$var$SvgComponent);
var $21bacf084e198765$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($21bacf084e198765$var$ForwardRef);
