// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_vectorDraw_generic2_48.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $dbJtR$jsxs, jsx as $dbJtR$jsx} from "react/jsx-runtime";
import {forwardRef as $dbJtR$forwardRef} from "react";





const $e78af3fae4788729$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dbJtR$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 48,
        height: 48,
        viewBox: "0 0 48 48",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dbJtR$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dbJtR$jsxs)("linearGradient", {
                        id: "dea082__d",
                        x1: 34.731,
                        x2: 6.933,
                        y1: -1.543,
                        y2: 34.498,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dbJtR$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dbJtR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dbJtR$jsxs)("linearGradient", {
                        id: "dea082__e",
                        x1: 17.793,
                        x2: 23.155,
                        y1: -18.135,
                        y2: 16.184,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dbJtR$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dbJtR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dbJtR$jsxs)("linearGradient", {
                        id: "dea082__f",
                        x1: -44.078,
                        x2: 46.147,
                        y1: 50.862,
                        y2: 33.03,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dbJtR$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dbJtR$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dbJtR$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dbJtR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dbJtR$jsxs)("radialGradient", {
                        id: "dea082__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 5.616 37.171)scale(69.3033)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dbJtR$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dbJtR$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dbJtR$jsx)("clipPath", {
                        id: "dea082__b",
                        children: /*#__PURE__*/ (0, $dbJtR$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dbJtR$jsx)("clipPath", {
                id: "dea082__a",
                children: /*#__PURE__*/ (0, $dbJtR$jsx)("path", {
                    fill: "#292929",
                    fillRule: "evenodd",
                    d: "M27.195 10.04 37.96 20.806l1.557-1.557c1.2-1.2 1.315-3.03.258-4.086l-6.938-6.938c-1.057-1.057-2.886-.942-4.086.258zm9.351 12.887L25.073 11.454l-10.25 4.376c-.912.388-1.628 1.18-1.918 2.115l-5.2 16.8c-.393 1.27-.196 2.51.381 3.511L19.07 27.273q-.087-.317-.088-.662c0-1.38 1.12-2.5 2.5-2.5s2.5 1.12 2.5 2.5c0 1.381-1.12 2.5-2.5 2.5q-.329-.001-.634-.08L9.887 39.991c.976.517 2.159.678 3.37.303l16.799-5.2c.936-.29 1.727-1.006 2.115-1.917z"
                })
            }),
            /*#__PURE__*/ (0, $dbJtR$jsx)("g", {
                clipPath: "url(#dea082__a)",
                children: /*#__PURE__*/ (0, $dbJtR$jsxs)("g", {
                    clipPath: "url(#dea082__b)",
                    children: [
                        /*#__PURE__*/ (0, $dbJtR$jsx)("rect", {
                            width: 48,
                            height: 48,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dbJtR$jsx)("circle", {
                            cx: 42.788,
                            cy: 31.555,
                            r: 69.303,
                            fill: "url(#dea082__c)"
                        }),
                        /*#__PURE__*/ (0, $dbJtR$jsx)("path", {
                            fill: "url(#dea082__d)",
                            d: "M-4.32 15.035C-5.09-1.557 7.735-15.63 24.326-16.401c16.59-.77 30.665 12.055 31.435 28.646.77 16.59-12.054 30.665-28.645 31.435S-3.55 31.625-4.32 15.035"
                        }),
                        /*#__PURE__*/ (0, $dbJtR$jsx)("path", {
                            fill: "url(#dea082__e)",
                            d: "M12.124-33.214c-3.742-2.316-8.484-2.33-12.24-.037L-37.69-10.31c-3.68 2.247-3.686 7.55-.01 9.805L-.377 22.404c3.75 2.301 8.493 2.299 12.24-.009L49.112-.535c3.655-2.25 3.662-7.523.011-9.782z"
                        }),
                        /*#__PURE__*/ (0, $dbJtR$jsx)("path", {
                            fill: "url(#dea082__f)",
                            d: "M-23.927 22.958c7.744 0 14.759 3.134 19.838 8.19 10.158 10.13 18.58 10.14 28.737.027 5.079-5.083 12.103-8.226 19.855-8.226 15.498.009 28.052 12.529 28.052 27.984S60 78.899 44.503 78.899c-7.77 0-14.794-3.151-19.872-8.234-10.15-10.095-18.562-10.087-28.72.043-5.08 5.066-12.094 8.191-19.838 8.191-15.489 0-28.043-12.52-28.043-27.966s12.554-27.975 28.043-27.975"
                        })
                    ]
                })
            })
        ]
    });
const $e78af3fae4788729$var$ForwardRef = /*#__PURE__*/ (0, $dbJtR$forwardRef)($e78af3fae4788729$var$SvgComponent);
var $e78af3fae4788729$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($e78af3fae4788729$var$ForwardRef);


export {$e78af3fae4788729$export$2e2bcd8739ae039 as default};
