// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_server_generic2_96.cjs
var $bde97c91243ed164$exports = require("../../../../icons/Icon.cjs");
var $1aaYJ$reactjsxruntime = require("react/jsx-runtime");
var $1aaYJ$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $563042bfd0604e6b$export$2e2bcd8739ae039);




const $563042bfd0604e6b$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 96,
        height: 96,
        viewBox: "0 0 96 96",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d1cc3b__d",
                        x1: 65.566,
                        x2: 25.102,
                        y1: 11.636,
                        y2: 67.288,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d1cc3b__e",
                        x1: 38.978,
                        x2: 46.479,
                        y1: -12.92,
                        y2: 38.008,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsxs)("linearGradient", {
                        id: "d1cc3b__f",
                        x1: -58.165,
                        x2: 82.825,
                        y1: 89.197,
                        y2: 59.638,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsxs)("radialGradient", {
                        id: "d1cc3b__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "matrix(0 102.569 -108.806 0 78.212 60.624)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("clipPath", {
                        id: "d1cc3b__b",
                        children: /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("clipPath", {
                id: "d1cc3b__a",
                children: /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("path", {
                    fill: "#fff",
                    d: "M69.898 40.8c6.462 0 11.7-5.237 11.7-11.7v-.6c0-6.461-5.238-11.7-11.7-11.7h-43.8c-6.462 0-11.7 5.239-11.7 11.7v.6c0 6.463 5.238 11.7 11.7 11.7h19.5v14.4h-19.5c-6.462 0-11.7 5.239-11.7 11.7v.6c0 6.463 5.238 11.7 11.7 11.7h43.8c6.462 0 11.7-5.237 11.7-11.7v-.6c0-6.461-5.238-11.7-11.7-11.7h-19.5V40.8zm-43.5-9.6c-1.326 0-2.4-1.073-2.4-2.4 0-1.325 1.074-2.4 2.4-2.4s2.4 1.075 2.4 2.4c0 1.327-1.074 2.4-2.4 2.4m0 38.4c-1.326 0-2.4-1.073-2.4-2.4 0-1.325 1.074-2.4 2.4-2.4s2.4 1.075 2.4 2.4c0 1.327-1.074 2.4-2.4 2.4m18 0h-10.8c-1.326 0-2.4-1.073-2.4-2.4 0-1.325 1.074-2.4 2.4-2.4h10.8c1.326 0 2.4 1.075 2.4 2.4 0 1.327-1.074 2.4-2.4 2.4m0-38.4h-10.8c-1.326 0-2.4-1.073-2.4-2.4 0-1.325 1.074-2.4 2.4-2.4h10.8c1.326 0 2.4 1.075 2.4 2.4 0 1.327-1.074 2.4-2.4 2.4"
                })
            }),
            /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("g", {
                clipPath: "url(#d1cc3b__a)",
                children: /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsxs)("g", {
                    clipPath: "url(#d1cc3b__b)",
                    children: [
                        /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("rect", {
                            width: 75.36,
                            height: 71.04,
                            x: 11.039,
                            y: 13.92,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("ellipse", {
                            cx: 78.212,
                            cy: 60.624,
                            fill: "url(#d1cc3b__c)",
                            rx: 108.806,
                            ry: 102.569
                        }),
                        /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("path", {
                            fill: "url(#d1cc3b__d)",
                            d: "M4.256 36.171C3.046 11.616 23.182-9.213 49.23-10.353s48.144 17.84 49.353 42.395S79.658 77.426 53.61 78.566 5.466 60.726 4.256 36.171"
                        }),
                        /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("path", {
                            fill: "url(#d1cc3b__e)",
                            d: "M30.078-35.237c-5.876-3.427-13.321-3.448-19.219-.054L-48.13-1.339c-5.777 3.325-5.787 11.174-.017 14.512l58.599 33.904c5.889 3.407 13.334 3.403 19.218-.012L88.148 13.13c5.74-3.33 5.75-11.135.018-14.479z"
                        }),
                        /*#__PURE__*/ (0, $1aaYJ$reactjsxruntime.jsx)("path", {
                            fill: "url(#d1cc3b__f)",
                            d: "M-26.528 47.9c12.159 0 23.172 4.638 31.146 12.122 15.948 14.993 29.17 15.005 45.118.038 7.974-7.522 19-12.173 31.173-12.173 24.33.013 44.04 18.542 44.04 41.415 0 22.874-19.71 41.391-44.04 41.391-12.2 0-23.227-4.665-31.2-12.187-15.935-14.941-29.143-14.928-45.09.064-7.975 7.497-18.988 12.123-31.147 12.123-24.317 0-44.026-18.53-44.026-41.39 0-22.861 19.71-41.403 44.026-41.403"
                        })
                    ]
                })
            })
        ]
    });
const $563042bfd0604e6b$var$ForwardRef = /*#__PURE__*/ (0, $1aaYJ$react.forwardRef)($563042bfd0604e6b$var$SvgComponent);
var $563042bfd0604e6b$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $bde97c91243ed164$exports.createIllustration)($563042bfd0604e6b$var$ForwardRef);
