// @react-spectrum/s2/illustrations/gradient/generic2/internal/S2_fill_conversationbubbles_generic2_160.mjs
import {createIllustration as $ac8c32e6775ddd1f$export$4d00e7be716a0093} from "../../../../icons/Icon.mjs";
import {jsxs as $dOmlu$jsxs, jsx as $dOmlu$jsx} from "react/jsx-runtime";
import {forwardRef as $dOmlu$forwardRef} from "react";





const $481c9508aba16f1a$var$SvgComponent = (props, ref)=>/*#__PURE__*/ (0, $dOmlu$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 160,
        height: 160,
        viewBox: "0 0 160 160",
        ref: ref,
        ...props,
        children: [
            /*#__PURE__*/ (0, $dOmlu$jsxs)("defs", {
                children: [
                    /*#__PURE__*/ (0, $dOmlu$jsxs)("linearGradient", {
                        id: "d65a19__d",
                        x1: 111.107,
                        x2: 30.526,
                        y1: 5.955,
                        y2: 110.43,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dOmlu$jsx)("stop", {
                                stopColor: "#FF4885"
                            }),
                            /*#__PURE__*/ (0, $dOmlu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#FF4885",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dOmlu$jsxs)("linearGradient", {
                        id: "d65a19__e",
                        x1: 62.011,
                        x2: 77.555,
                        y1: -42.141,
                        y2: 57.344,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dOmlu$jsx)("stop", {
                                offset: 0.156,
                                stopColor: "#7A6AFD"
                            }),
                            /*#__PURE__*/ (0, $dOmlu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#7A6AFD",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dOmlu$jsxs)("linearGradient", {
                        id: "d65a19__f",
                        x1: -117.348,
                        x2: 144.197,
                        y1: 157.869,
                        y2: 106.178,
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dOmlu$jsx)("stop", {
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            }),
                            /*#__PURE__*/ (0, $dOmlu$jsx)("stop", {
                                offset: 0.432,
                                stopColor: "#30A7FE",
                                stopOpacity: 0.995
                            }),
                            /*#__PURE__*/ (0, $dOmlu$jsx)("stop", {
                                offset: 0.609,
                                stopColor: "#30A7FE"
                            }),
                            /*#__PURE__*/ (0, $dOmlu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#30A7FE",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dOmlu$jsxs)("radialGradient", {
                        id: "d65a19__c",
                        cx: 0,
                        cy: 0,
                        r: 1,
                        gradientTransform: "rotate(90 16.277 118.18)scale(200.895)",
                        gradientUnits: "userSpaceOnUse",
                        children: [
                            /*#__PURE__*/ (0, $dOmlu$jsx)("stop", {
                                offset: 0.089,
                                stopColor: "#EB1000"
                            }),
                            /*#__PURE__*/ (0, $dOmlu$jsx)("stop", {
                                offset: 1,
                                stopColor: "#EB1000",
                                stopOpacity: 0
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $dOmlu$jsx)("clipPath", {
                        id: "d65a19__b",
                        children: /*#__PURE__*/ (0, $dOmlu$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.43,
                            fill: "#fff",
                            rx: 10
                        })
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dOmlu$jsxs)("clipPath", {
                id: "d65a19__a",
                children: [
                    /*#__PURE__*/ (0, $dOmlu$jsx)("path", {
                        fill: "#fff",
                        d: "M131.619 69.36h-15.436v11.455c0 9.893-8.368 17.94-18.654 17.94H72.325v7.452c0 4.473 3.888 8.1 8.684 8.1h18.906l19.186 15.517c1.882 1.521 4.809.265 4.79-2.059-.032-4.165-.064-9.636-.047-13.458h7.776c4.796 0 8.684-3.626 8.684-8.1V77.46c0-4.474-3.888-8.1-8.684-8.1"
                    }),
                    /*#__PURE__*/ (0, $dOmlu$jsx)("path", {
                        fill: "#fff",
                        d: "M97.53 90.716c5.861 0 10.613-4.433 10.613-9.901V45.68c0-5.468-4.752-9.9-10.613-9.9H30.317c-5.862 0-10.614 4.432-10.614 9.9v35.135c0 5.468 4.752 9.9 10.614 9.9h9.504c.02 4.672-.019 11.358-.058 16.449-.022 2.84 3.555 4.376 5.855 2.516l23.45-18.964z"
                    })
                ]
            }),
            /*#__PURE__*/ (0, $dOmlu$jsx)("g", {
                clipPath: "url(#d65a19__a)",
                children: /*#__PURE__*/ (0, $dOmlu$jsxs)("g", {
                    clipPath: "url(#d65a19__b)",
                    children: [
                        /*#__PURE__*/ (0, $dOmlu$jsx)("rect", {
                            width: 139.142,
                            height: 139.142,
                            x: 10.43,
                            y: 10.43,
                            fill: "#D9F4FD",
                            rx: 10
                        }),
                        /*#__PURE__*/ (0, $dOmlu$jsx)("circle", {
                            cx: 134.458,
                            cy: 101.903,
                            r: 200.895,
                            fill: "url(#d65a19__c)"
                        }),
                        /*#__PURE__*/ (0, $dOmlu$jsx)("path", {
                            fill: "url(#d65a19__d)",
                            d: "M-2.092 54.01C-4.325 5.917 32.852-34.88 80.945-37.114S169.837-2.17 172.07 45.923s-34.944 88.891-83.038 91.124C40.94 139.281.142 102.104-2.092 54.01"
                        }),
                        /*#__PURE__*/ (0, $dOmlu$jsx)("path", {
                            fill: "url(#d65a19__e)",
                            d: "M45.579-85.852c-10.85-6.714-24.595-6.755-35.485-.106l-108.915 66.5c-10.667 6.513-10.685 21.886-.032 28.424L9.341 75.37c10.873 6.673 24.62 6.665 35.483-.023L152.798 8.88c10.596-6.524 10.615-21.811.032-28.36z"
                        }),
                        /*#__PURE__*/ (0, $dOmlu$jsx)("path", {
                            fill: "url(#d65a19__f)",
                            d: "M-58.935 76.982c22.45 0 42.784 9.086 57.507 23.743 29.445 29.366 53.857 29.391 83.303.076 14.722-14.733 35.082-23.844 57.557-23.844 44.923.025 81.314 36.318 81.314 81.119s-36.391 81.068-81.314 81.068c-22.525 0-42.885-9.136-57.608-23.869-29.42-29.265-53.807-29.24-83.252.126-14.723 14.682-35.058 23.743-57.507 23.743-44.897 0-81.289-36.293-81.289-81.068 0-44.776 36.392-81.094 81.29-81.094"
                        })
                    ]
                })
            })
        ]
    });
const $481c9508aba16f1a$var$ForwardRef = /*#__PURE__*/ (0, $dOmlu$forwardRef)($481c9508aba16f1a$var$SvgComponent);
var $481c9508aba16f1a$export$2e2bcd8739ae039 = /*#__PURE__*/ (0, $ac8c32e6775ddd1f$export$4d00e7be716a0093)($481c9508aba16f1a$var$ForwardRef);


export {$481c9508aba16f1a$export$2e2bcd8739ae039 as default};
