// @react-spectrum/s2/dist/private/Accordion.mjs
import "./Accordion.css";
import {Disclosure as $cf1d656285911959$export$74a362b31437ec83, DisclosureContext as $cf1d656285911959$export$d665dd135a51b28a, DisclosureHeader as $cf1d656285911959$export$1e037ddd32f5b8ac, DisclosurePanel as $cf1d656285911959$export$feabaa331e1d464c, DisclosureTitle as $cf1d656285911959$export$7843c6a5b3e340a2} from "./Disclosure.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $cFlas$jsx} from "react/jsx-runtime";
import {DisclosureGroup as $cFlas$DisclosureGroup} from "react-aria-components/DisclosureGroup";
import {createContext as $cFlas$createContext, forwardRef as $cFlas$forwardRef} from "react";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





const $e0a5d45f488fa6ba$var$accordion = function anonymous(props, overrides) {
    let rules = " ";
    rules += (String(overrides || '').match(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L|F|M|K)[^\s]+/g) || []).join('');
    rules += ' sd131';
    rules += ' _ta131';
    return rules;
};
const $e0a5d45f488fa6ba$export$6bd5bb82c987ab74 = /*#__PURE__*/ (0, $cFlas$createContext)(null);
const $e0a5d45f488fa6ba$export$a766cd26d0d69044 = /*#__PURE__*/ (0, $cFlas$forwardRef)(function Accordion(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $e0a5d45f488fa6ba$export$6bd5bb82c987ab74);
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let { UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '', size: size = 'M', density: density = 'regular', isQuiet: isQuiet } = props;
    return /*#__PURE__*/ (0, $cFlas$jsx)((0, $cf1d656285911959$export$d665dd135a51b28a).Provider, {
        value: {
            size: size,
            isQuiet: isQuiet,
            density: density
        },
        children: /*#__PURE__*/ (0, $cFlas$jsx)((0, $cFlas$DisclosureGroup), {
            ...props,
            ref: domRef,
            style: UNSAFE_style,
            className: (UNSAFE_className ?? '') + $e0a5d45f488fa6ba$var$accordion(null, props.styles),
            children: props.children
        })
    });
});
const $e0a5d45f488fa6ba$export$d99097c13d4dac9f = /*#__PURE__*/ (0, $cFlas$forwardRef)(function AccordionItem(props, ref) {
    return /*#__PURE__*/ (0, $cFlas$jsx)((0, $cf1d656285911959$export$74a362b31437ec83), {
        ...props,
        ref: ref
    });
});
const $e0a5d45f488fa6ba$export$c30e8bcae12973af = /*#__PURE__*/ (0, $cFlas$forwardRef)(function AccordionItemTitle(props, ref) {
    return /*#__PURE__*/ (0, $cFlas$jsx)((0, $cf1d656285911959$export$7843c6a5b3e340a2), {
        ...props,
        ref: ref
    });
});
const $e0a5d45f488fa6ba$export$96fae341779b6372 = /*#__PURE__*/ (0, $cFlas$forwardRef)(function AccordionItemHeader(props, ref) {
    return /*#__PURE__*/ (0, $cFlas$jsx)((0, $cf1d656285911959$export$1e037ddd32f5b8ac), {
        ...props,
        ref: ref
    });
});
const $e0a5d45f488fa6ba$export$d60a3c6bf8395a37 = /*#__PURE__*/ (0, $cFlas$forwardRef)(function AccordionItemPanel(props, ref) {
    return /*#__PURE__*/ (0, $cFlas$jsx)((0, $cf1d656285911959$export$feabaa331e1d464c), {
        ...props,
        ref: ref
    });
});


export {$e0a5d45f488fa6ba$export$6bd5bb82c987ab74 as AccordionContext, $e0a5d45f488fa6ba$export$a766cd26d0d69044 as Accordion, $e0a5d45f488fa6ba$export$d99097c13d4dac9f as AccordionItem, $e0a5d45f488fa6ba$export$c30e8bcae12973af as AccordionItemTitle, $e0a5d45f488fa6ba$export$96fae341779b6372 as AccordionItemHeader, $e0a5d45f488fa6ba$export$d60a3c6bf8395a37 as AccordionItemPanel};
