// @react-spectrum/s2/dist/private/Accordion.cjs
require("./Accordion.css");
var $4e301a98f0cc4e4f$exports = require("./Disclosure.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $etdk1$reactjsxruntime = require("react/jsx-runtime");
var $etdk1$reactariacomponentsDisclosureGroup = require("react-aria-components/DisclosureGroup");
var $etdk1$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "AccordionContext", () => $152a25dfd79f123a$export$6bd5bb82c987ab74);
$parcel$export(module.exports, "Accordion", () => $152a25dfd79f123a$export$a766cd26d0d69044);
$parcel$export(module.exports, "AccordionItem", () => $152a25dfd79f123a$export$d99097c13d4dac9f);
$parcel$export(module.exports, "AccordionItemTitle", () => $152a25dfd79f123a$export$c30e8bcae12973af);
$parcel$export(module.exports, "AccordionItemHeader", () => $152a25dfd79f123a$export$96fae341779b6372);
$parcel$export(module.exports, "AccordionItemPanel", () => $152a25dfd79f123a$export$d60a3c6bf8395a37);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





const $152a25dfd79f123a$var$accordion = function anonymous(props, overrides) {
    let rules = " ";
    rules += (String(overrides || '').match(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L|F|M|K)[^\s]+/g) || []).join('');
    rules += ' sd131';
    rules += ' _ta131';
    return rules;
};
const $152a25dfd79f123a$export$6bd5bb82c987ab74 = /*#__PURE__*/ (0, $etdk1$react.createContext)(null);
const $152a25dfd79f123a$export$a766cd26d0d69044 = /*#__PURE__*/ (0, $etdk1$react.forwardRef)(function Accordion(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $152a25dfd79f123a$export$6bd5bb82c987ab74);
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let { UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '', size: size = 'M', density: density = 'regular', isQuiet: isQuiet } = props;
    return /*#__PURE__*/ (0, $etdk1$reactjsxruntime.jsx)((0, $4e301a98f0cc4e4f$exports.DisclosureContext).Provider, {
        value: {
            size: size,
            isQuiet: isQuiet,
            density: density
        },
        children: /*#__PURE__*/ (0, $etdk1$reactjsxruntime.jsx)((0, $etdk1$reactariacomponentsDisclosureGroup.DisclosureGroup), {
            ...props,
            ref: domRef,
            style: UNSAFE_style,
            className: (UNSAFE_className ?? '') + $152a25dfd79f123a$var$accordion(null, props.styles),
            children: props.children
        })
    });
});
const $152a25dfd79f123a$export$d99097c13d4dac9f = /*#__PURE__*/ (0, $etdk1$react.forwardRef)(function AccordionItem(props, ref) {
    return /*#__PURE__*/ (0, $etdk1$reactjsxruntime.jsx)((0, $4e301a98f0cc4e4f$exports.Disclosure), {
        ...props,
        ref: ref
    });
});
const $152a25dfd79f123a$export$c30e8bcae12973af = /*#__PURE__*/ (0, $etdk1$react.forwardRef)(function AccordionItemTitle(props, ref) {
    return /*#__PURE__*/ (0, $etdk1$reactjsxruntime.jsx)((0, $4e301a98f0cc4e4f$exports.DisclosureTitle), {
        ...props,
        ref: ref
    });
});
const $152a25dfd79f123a$export$96fae341779b6372 = /*#__PURE__*/ (0, $etdk1$react.forwardRef)(function AccordionItemHeader(props, ref) {
    return /*#__PURE__*/ (0, $etdk1$reactjsxruntime.jsx)((0, $4e301a98f0cc4e4f$exports.DisclosureHeader), {
        ...props,
        ref: ref
    });
});
const $152a25dfd79f123a$export$d60a3c6bf8395a37 = /*#__PURE__*/ (0, $etdk1$react.forwardRef)(function AccordionItemPanel(props, ref) {
    return /*#__PURE__*/ (0, $etdk1$reactjsxruntime.jsx)((0, $4e301a98f0cc4e4f$exports.DisclosurePanel), {
        ...props,
        ref: ref
    });
});
