// @react-spectrum/s2/dist/private/ActionBar.cjs
require("./ActionBar.css");
var $7a26131f6144af2b$exports = require("./ActionButtonGroup.cjs");
var $2f907cb84c6e9e75$exports = require("./CloseButton.cjs");
var $4526404114e78c80$exports = require("./intlStrings.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $cRsNx$reactjsxruntime = require("react/jsx-runtime");
var $cRsNx$reactariaprivateliveannouncerLiveAnnouncer = require("react-aria/private/live-announcer/LiveAnnouncer");
var $cRsNx$react = require("react");
var $cRsNx$reactariaFocusScope = require("react-aria/FocusScope");
var $cRsNx$reactstatelyuseControlledState = require("react-stately/useControlledState");
var $cRsNx$reactariaprivateutilsanimation = require("react-aria/private/utils/animation");
var $cRsNx$reactariauseKeyboard = require("react-aria/useKeyboard");
var $cRsNx$reactariauseLocalizedStringFormatter = require("react-aria/useLocalizedStringFormatter");
var $cRsNx$reactariauseObjectRef = require("react-aria/useObjectRef");
var $cRsNx$reactariaprivateutilsuseResizeObserver = require("react-aria/private/utils/useResizeObserver");


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "ActionBarContext", () => $85bf454149fe9ddc$export$9f306827c48feef4);
$parcel$export(module.exports, "ActionBar", () => $85bf454149fe9ddc$export$e213cebad6250b4a);
$parcel$export(module.exports, "useActionBarContainer", () => $85bf454149fe9ddc$export$13f32e21845e01d0);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 














const $85bf454149fe9ddc$var$actionBarStyles = function anonymous(props) {
    let rules = " ";
    rules += ' oc131';
    rules += ' nc131';
    rules += ' kc131';
    rules += ' jc131';
    if (props.isEmphasized) rules += ' -Ovbhqd-g0131';
    else rules += ' -Ovbhqd-g_g131';
    rules += ' gEzkoQe131';
    rules += ' _nLeasBb131';
    rules += ' _oa131';
    rules += ' _Lf131';
    rules += ' _Mb131';
    if (props.isEmphasized) rules += ' Oj131';
    else rules += ' Oye39g131';
    rules += ' Ola131';
    rules += ' St131';
    rules += ' Rt131';
    rules += ' Te131';
    rules += ' Qe131';
    rules += ' sd131';
    rules += ' Ul131';
    rules += ' ql131';
    rules += ' eb131';
    if (props.isInContainer) rules += ' _Pa131';
    rules += ' _lr131';
    rules += ' _AI131';
    rules += ' -F3Iohc-_zI131';
    if (props.isInContainer) rules += ' Zd131';
    else rules += ' Za131';
    rules += ' IM131';
    rules += ' HM131';
    rules += ' Ll131';
    rules += ' Yg131';
    rules += ' Xc131';
    rules += ' _2b131';
    if (props.isExiting) {
        rules += ' -_8PloMd-t131';
        rules += ' __Ya131';
    } else if (props.isEntering) {
        rules += ' -_8PloMd-t131';
        rules += ' __Ya131';
    } else {
        rules += ' -_8PloMd-q131';
        rules += ' __Ya131';
    }
    return rules;
};
const $85bf454149fe9ddc$export$9f306827c48feef4 = /*#__PURE__*/ (0, $cRsNx$react.createContext)(null);
const $85bf454149fe9ddc$export$e213cebad6250b4a = /*#__PURE__*/ (0, $cRsNx$react.forwardRef)(function ActionBar(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $85bf454149fe9ddc$export$9f306827c48feef4);
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let isOpen = props.selectedItemCount !== 0;
    let isExiting = (0, $cRsNx$reactariaprivateutilsanimation.useExitAnimation)(domRef, isOpen && props.scrollRef != null);
    if (!isOpen && !isExiting) return null;
    return /*#__PURE__*/ (0, $cRsNx$reactjsxruntime.jsx)($85bf454149fe9ddc$var$ActionBarInner, {
        ...props,
        ref: domRef,
        isExiting: isExiting
    });
});
const $85bf454149fe9ddc$var$ActionBarInner = /*#__PURE__*/ (0, $cRsNx$react.forwardRef)(function ActionBarInner(props1, ref) {
    let { isEmphasized: isEmphasized, selectedItemCount: selectedItemCount = 0, children: children, onClearSelection: onClearSelection, isExiting: isExiting, slot: // eslint-disable-next-line @typescript-eslint/no-unused-vars
    slot, scrollRef: scrollRef, ...otherProps } = props1;
    let stringFormatter = (0, $cRsNx$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    // Store the last count greater than zero so that we can retain it while rendering the fade-out animation.
    let [lastCount, setLastCount] = (0, $cRsNx$react.useState)(selectedItemCount);
    if ((selectedItemCount === 'all' || selectedItemCount > 0) && selectedItemCount !== lastCount) setLastCount(selectedItemCount);
    // Measure the width of the collection's scrollbar and offset the action bar by that amount.
    let [scrollbarWidth, setScrollbarWidth] = (0, $cRsNx$react.useState)(0);
    let updateScrollbarWidth = (0, $cRsNx$react.useCallback)(()=>{
        let el = scrollRef?.current;
        if (el) {
            let w = el.offsetWidth - el.clientWidth;
            setScrollbarWidth(w);
        }
    }, [
        scrollRef
    ]);
    (0, $cRsNx$reactariaprivateutilsuseResizeObserver.useResizeObserver)({
        ref: scrollRef,
        onResize: updateScrollbarWidth
    });
    let { keyboardProps: keyboardProps } = (0, $cRsNx$reactariauseKeyboard.useKeyboard)({
        onKeyDown (e) {
            if (e.key === 'Escape') {
                e.preventDefault();
                onClearSelection?.();
            } else e.continuePropagation();
        }
    });
    // Announce "actions available" on mount.
    let isInitial = (0, $cRsNx$react.useRef)(true);
    (0, $cRsNx$react.useEffect)(()=>{
        if (isInitial.current && scrollRef) {
            isInitial.current = false;
            (0, $cRsNx$reactariaprivateliveannouncerLiveAnnouncer.announce)(stringFormatter.format('actionbar.actionsAvailable'));
        }
    }, [
        stringFormatter,
        scrollRef
    ]);
    let objectRef = (0, $cRsNx$reactariauseObjectRef.useObjectRef)(ref);
    let isEntering = (0, $cRsNx$reactariaprivateutilsanimation.useEnterAnimation)(objectRef, !!scrollRef);
    return /*#__PURE__*/ (0, $cRsNx$reactjsxruntime.jsx)((0, $cRsNx$reactariaFocusScope.FocusScope), {
        restoreFocus: true,
        children: /*#__PURE__*/ (0, $cRsNx$reactjsxruntime.jsxs)("div", {
            ref: objectRef,
            ...otherProps,
            ...keyboardProps,
            className: (props1.UNSAFE_className || '') + $85bf454149fe9ddc$var$actionBarStyles({
                isEmphasized: isEmphasized,
                isInContainer: !!scrollRef,
                isEntering: isEntering,
                isExiting: isExiting
            }),
            style: {
                insetInlineEnd: `calc(var(--insetEnd) + ${scrollbarWidth}px)`,
                ...props1.UNSAFE_style
            },
            children: [
                /*#__PURE__*/ (0, $cRsNx$reactjsxruntime.jsx)("div", {
                    className: " _Jb131 IM131",
                    children: /*#__PURE__*/ (0, $cRsNx$reactjsxruntime.jsx)((0, $7a26131f6144af2b$exports.ActionButtonGroup), {
                        staticColor: isEmphasized ? 'auto' : undefined,
                        isQuiet: true,
                        "aria-label": stringFormatter.format('actionbar.actions'),
                        children: children
                    })
                }),
                /*#__PURE__*/ (0, $cRsNx$reactjsxruntime.jsxs)("div", {
                    className: " _Ja131 sd131 eb131 Ub131 qb131",
                    children: [
                        /*#__PURE__*/ (0, $cRsNx$reactjsxruntime.jsx)((0, $2f907cb84c6e9e75$exports.CloseButton), {
                            staticColor: isEmphasized ? 'auto' : undefined,
                            "aria-label": stringFormatter.format('actionbar.clearSelection'),
                            onPress: ()=>onClearSelection?.()
                        }),
                        /*#__PURE__*/ (0, $cRsNx$reactjsxruntime.jsx)("span", {
                            className: function anonymous(props) {
                                let rules = " ";
                                rules += ' uk131';
                                rules += ' ucJ9TBTb131';
                                rules += ' ud3Euai131';
                                rules += ' uea131';
                                rules += ' ugb131';
                                rules += ' uhd131';
                                rules += ' uje131';
                                rules += ' u2NhKxcl131';
                                rules += ' uic131';
                                rules += ' -_6BNtrc-c131';
                                rules += ' vx131';
                                rules += ' xb131';
                                rules += ' _xa131';
                                rules += ' _Fd131';
                                rules += ' _FnuYUweb131';
                                if (props.isEmphasized) rules += ' pr131';
                                else rules += ' pt131';
                                return rules;
                            }({
                                isEmphasized: isEmphasized
                            }),
                            children: lastCount === 'all' ? stringFormatter.format('actionbar.selectedAll') : stringFormatter.format('actionbar.selected', {
                                count: lastCount
                            })
                        })
                    ]
                })
            ]
        })
    });
});
function $85bf454149fe9ddc$export$13f32e21845e01d0(props) {
    let { renderActionBar: renderActionBar, scrollRef: scrollRef } = props;
    let [selectedKeys, setSelectedKeys] = (0, $cRsNx$reactstatelyuseControlledState.useControlledState)(props.selectedKeys, props.defaultSelectedKeys || new Set(), props.onSelectionChange);
    let selectedKeysSet = (0, $cRsNx$react.useMemo)(()=>selectedKeys === 'all' ? selectedKeys : new Set(selectedKeys), [
        selectedKeys
    ]);
    let actionBar = (0, $cRsNx$react.useMemo)(()=>renderActionBar?.(selectedKeysSet), [
        renderActionBar,
        selectedKeysSet
    ]);
    let selectedItemCount = selectedKeysSet === 'all' ? 'all' : selectedKeysSet.size;
    let [actionBarHeight, setActionBarHeight] = (0, $cRsNx$react.useState)(0);
    let actionBarRef = (0, $cRsNx$react.useCallback)((ref)=>{
        let actionBar = ref?.UNSAFE_getDOMNode();
        if (actionBar) setActionBarHeight(actionBar.offsetHeight + 8);
        else setActionBarHeight(0);
    }, []);
    let actionBarContext = (0, $cRsNx$react.useMemo)(()=>({
            ref: actionBarRef,
            scrollRef: scrollRef,
            selectedItemCount: selectedItemCount,
            onClearSelection: ()=>setSelectedKeys(new Set())
        }), [
        scrollRef,
        actionBarRef,
        selectedItemCount,
        setSelectedKeys
    ]);
    let wrappedActionBar = (0, $cRsNx$react.useMemo)(()=>/*#__PURE__*/ (0, $cRsNx$reactjsxruntime.jsx)($85bf454149fe9ddc$export$9f306827c48feef4.Provider, {
            value: actionBarContext,
            children: actionBar
        }), [
        actionBarContext,
        actionBar
    ]);
    return {
        selectedKeys: selectedKeys,
        onSelectionChange: setSelectedKeys,
        actionBar: wrappedActionBar,
        actionBarHeight: actionBarHeight
    };
}
