// @react-spectrum/s2/dist/private/ActionBar.mjs
import "./ActionBar.css";
import {ActionButtonGroup as $10401bdc118cbc90$export$73d695ae27330056} from "./ActionButtonGroup.mjs";
import {CloseButton as $a9cda54c4f47ce52$export$de65de8213222d10} from "./CloseButton.mjs";
import $kxwJw$intlStringsmjs from "./intlStrings.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $kxwJw$jsx, jsxs as $kxwJw$jsxs} from "react/jsx-runtime";
import {announce as $kxwJw$announce} from "react-aria/private/live-announcer/LiveAnnouncer";
import {createContext as $kxwJw$createContext, forwardRef as $kxwJw$forwardRef, useState as $kxwJw$useState, useCallback as $kxwJw$useCallback, useRef as $kxwJw$useRef, useEffect as $kxwJw$useEffect, useMemo as $kxwJw$useMemo} from "react";
import {FocusScope as $kxwJw$FocusScope} from "react-aria/FocusScope";
import {useControlledState as $kxwJw$useControlledState} from "react-stately/useControlledState";
import {useExitAnimation as $kxwJw$useExitAnimation, useEnterAnimation as $kxwJw$useEnterAnimation} from "react-aria/private/utils/animation";
import {useKeyboard as $kxwJw$useKeyboard} from "react-aria/useKeyboard";
import {useLocalizedStringFormatter as $kxwJw$useLocalizedStringFormatter} from "react-aria/useLocalizedStringFormatter";
import {useObjectRef as $kxwJw$useObjectRef} from "react-aria/useObjectRef";
import {useResizeObserver as $kxwJw$useResizeObserver} from "react-aria/private/utils/useResizeObserver";


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 














const $f21f2186348fbc5b$var$actionBarStyles = function anonymous(props) {
    let rules = " ";
    rules += ' oc131';
    rules += ' nc131';
    rules += ' kc131';
    rules += ' jc131';
    if (props.isEmphasized) rules += ' -Ovbhqd-g0131';
    else rules += ' -Ovbhqd-g_g131';
    rules += ' gEzkoQe131';
    rules += ' _nLeasBb131';
    rules += ' _oa131';
    rules += ' _Lf131';
    rules += ' _Mb131';
    if (props.isEmphasized) rules += ' Oj131';
    else rules += ' Oye39g131';
    rules += ' Ola131';
    rules += ' St131';
    rules += ' Rt131';
    rules += ' Te131';
    rules += ' Qe131';
    rules += ' sd131';
    rules += ' Ul131';
    rules += ' ql131';
    rules += ' eb131';
    if (props.isInContainer) rules += ' _Pa131';
    rules += ' _lr131';
    rules += ' _AI131';
    rules += ' -F3Iohc-_zI131';
    if (props.isInContainer) rules += ' Zd131';
    else rules += ' Za131';
    rules += ' IM131';
    rules += ' HM131';
    rules += ' Ll131';
    rules += ' Yg131';
    rules += ' Xc131';
    rules += ' _2b131';
    if (props.isExiting) {
        rules += ' -_8PloMd-t131';
        rules += ' __Ya131';
    } else if (props.isEntering) {
        rules += ' -_8PloMd-t131';
        rules += ' __Ya131';
    } else {
        rules += ' -_8PloMd-q131';
        rules += ' __Ya131';
    }
    return rules;
};
const $f21f2186348fbc5b$export$9f306827c48feef4 = /*#__PURE__*/ (0, $kxwJw$createContext)(null);
const $f21f2186348fbc5b$export$e213cebad6250b4a = /*#__PURE__*/ (0, $kxwJw$forwardRef)(function ActionBar(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $f21f2186348fbc5b$export$9f306827c48feef4);
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let isOpen = props.selectedItemCount !== 0;
    let isExiting = (0, $kxwJw$useExitAnimation)(domRef, isOpen && props.scrollRef != null);
    if (!isOpen && !isExiting) return null;
    return /*#__PURE__*/ (0, $kxwJw$jsx)($f21f2186348fbc5b$var$ActionBarInner, {
        ...props,
        ref: domRef,
        isExiting: isExiting
    });
});
const $f21f2186348fbc5b$var$ActionBarInner = /*#__PURE__*/ (0, $kxwJw$forwardRef)(function ActionBarInner(props1, ref) {
    let { isEmphasized: isEmphasized, selectedItemCount: selectedItemCount = 0, children: children, onClearSelection: onClearSelection, isExiting: isExiting, slot: // eslint-disable-next-line @typescript-eslint/no-unused-vars
    slot, scrollRef: scrollRef, ...otherProps } = props1;
    let stringFormatter = (0, $kxwJw$useLocalizedStringFormatter)((0, ($parcel$interopDefault($kxwJw$intlStringsmjs))), '@react-spectrum/s2');
    // Store the last count greater than zero so that we can retain it while rendering the fade-out animation.
    let [lastCount, setLastCount] = (0, $kxwJw$useState)(selectedItemCount);
    if ((selectedItemCount === 'all' || selectedItemCount > 0) && selectedItemCount !== lastCount) setLastCount(selectedItemCount);
    // Measure the width of the collection's scrollbar and offset the action bar by that amount.
    let [scrollbarWidth, setScrollbarWidth] = (0, $kxwJw$useState)(0);
    let updateScrollbarWidth = (0, $kxwJw$useCallback)(()=>{
        let el = scrollRef?.current;
        if (el) {
            let w = el.offsetWidth - el.clientWidth;
            setScrollbarWidth(w);
        }
    }, [
        scrollRef
    ]);
    (0, $kxwJw$useResizeObserver)({
        ref: scrollRef,
        onResize: updateScrollbarWidth
    });
    let { keyboardProps: keyboardProps } = (0, $kxwJw$useKeyboard)({
        onKeyDown (e) {
            if (e.key === 'Escape') {
                e.preventDefault();
                onClearSelection?.();
            } else e.continuePropagation();
        }
    });
    // Announce "actions available" on mount.
    let isInitial = (0, $kxwJw$useRef)(true);
    (0, $kxwJw$useEffect)(()=>{
        if (isInitial.current && scrollRef) {
            isInitial.current = false;
            (0, $kxwJw$announce)(stringFormatter.format('actionbar.actionsAvailable'));
        }
    }, [
        stringFormatter,
        scrollRef
    ]);
    let objectRef = (0, $kxwJw$useObjectRef)(ref);
    let isEntering = (0, $kxwJw$useEnterAnimation)(objectRef, !!scrollRef);
    return /*#__PURE__*/ (0, $kxwJw$jsx)((0, $kxwJw$FocusScope), {
        restoreFocus: true,
        children: /*#__PURE__*/ (0, $kxwJw$jsxs)("div", {
            ref: objectRef,
            ...otherProps,
            ...keyboardProps,
            className: (props1.UNSAFE_className || '') + $f21f2186348fbc5b$var$actionBarStyles({
                isEmphasized: isEmphasized,
                isInContainer: !!scrollRef,
                isEntering: isEntering,
                isExiting: isExiting
            }),
            style: {
                insetInlineEnd: `calc(var(--insetEnd) + ${scrollbarWidth}px)`,
                ...props1.UNSAFE_style
            },
            children: [
                /*#__PURE__*/ (0, $kxwJw$jsx)("div", {
                    className: " _Jb131 IM131",
                    children: /*#__PURE__*/ (0, $kxwJw$jsx)((0, $10401bdc118cbc90$export$73d695ae27330056), {
                        staticColor: isEmphasized ? 'auto' : undefined,
                        isQuiet: true,
                        "aria-label": stringFormatter.format('actionbar.actions'),
                        children: children
                    })
                }),
                /*#__PURE__*/ (0, $kxwJw$jsxs)("div", {
                    className: " _Ja131 sd131 eb131 Ub131 qb131",
                    children: [
                        /*#__PURE__*/ (0, $kxwJw$jsx)((0, $a9cda54c4f47ce52$export$de65de8213222d10), {
                            staticColor: isEmphasized ? 'auto' : undefined,
                            "aria-label": stringFormatter.format('actionbar.clearSelection'),
                            onPress: ()=>onClearSelection?.()
                        }),
                        /*#__PURE__*/ (0, $kxwJw$jsx)("span", {
                            className: function anonymous(props) {
                                let rules = " ";
                                rules += ' uk131';
                                rules += ' ucJ9TBTb131';
                                rules += ' ud3Euai131';
                                rules += ' uea131';
                                rules += ' ugb131';
                                rules += ' uhd131';
                                rules += ' uje131';
                                rules += ' u2NhKxcl131';
                                rules += ' uic131';
                                rules += ' -_6BNtrc-c131';
                                rules += ' vx131';
                                rules += ' xb131';
                                rules += ' _xa131';
                                rules += ' _Fd131';
                                rules += ' _FnuYUweb131';
                                if (props.isEmphasized) rules += ' pr131';
                                else rules += ' pt131';
                                return rules;
                            }({
                                isEmphasized: isEmphasized
                            }),
                            children: lastCount === 'all' ? stringFormatter.format('actionbar.selectedAll') : stringFormatter.format('actionbar.selected', {
                                count: lastCount
                            })
                        })
                    ]
                })
            ]
        })
    });
});
function $f21f2186348fbc5b$export$13f32e21845e01d0(props) {
    let { renderActionBar: renderActionBar, scrollRef: scrollRef } = props;
    let [selectedKeys, setSelectedKeys] = (0, $kxwJw$useControlledState)(props.selectedKeys, props.defaultSelectedKeys || new Set(), props.onSelectionChange);
    let selectedKeysSet = (0, $kxwJw$useMemo)(()=>selectedKeys === 'all' ? selectedKeys : new Set(selectedKeys), [
        selectedKeys
    ]);
    let actionBar = (0, $kxwJw$useMemo)(()=>renderActionBar?.(selectedKeysSet), [
        renderActionBar,
        selectedKeysSet
    ]);
    let selectedItemCount = selectedKeysSet === 'all' ? 'all' : selectedKeysSet.size;
    let [actionBarHeight, setActionBarHeight] = (0, $kxwJw$useState)(0);
    let actionBarRef = (0, $kxwJw$useCallback)((ref)=>{
        let actionBar = ref?.UNSAFE_getDOMNode();
        if (actionBar) setActionBarHeight(actionBar.offsetHeight + 8);
        else setActionBarHeight(0);
    }, []);
    let actionBarContext = (0, $kxwJw$useMemo)(()=>({
            ref: actionBarRef,
            scrollRef: scrollRef,
            selectedItemCount: selectedItemCount,
            onClearSelection: ()=>setSelectedKeys(new Set())
        }), [
        scrollRef,
        actionBarRef,
        selectedItemCount,
        setSelectedKeys
    ]);
    let wrappedActionBar = (0, $kxwJw$useMemo)(()=>/*#__PURE__*/ (0, $kxwJw$jsx)($f21f2186348fbc5b$export$9f306827c48feef4.Provider, {
            value: actionBarContext,
            children: actionBar
        }), [
        actionBarContext,
        actionBar
    ]);
    return {
        selectedKeys: selectedKeys,
        onSelectionChange: setSelectedKeys,
        actionBar: wrappedActionBar,
        actionBarHeight: actionBarHeight
    };
}


export {$f21f2186348fbc5b$export$9f306827c48feef4 as ActionBarContext, $f21f2186348fbc5b$export$e213cebad6250b4a as ActionBar, $f21f2186348fbc5b$export$13f32e21845e01d0 as useActionBarContainer};
