// @react-spectrum/s2/dist/private/ActionButton.cjs
require("./ActionButton.css");
var $7a26131f6144af2b$exports = require("./ActionButtonGroup.cjs");
var $38071d9ce246d4cf$exports = require("./Avatar.cjs");
var $e991cbcdf82ced71$exports = require("./CenterBaseline.cjs");
var $411768293e842abb$exports = require("./CornerTriangle.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $053b76ed3d29e13b$exports = require("./Image.cjs");
var $4526404114e78c80$exports = require("./intlStrings.cjs");
var $cd7301fb33ba8690$exports = require("./NotificationBadge.cjs");
var $2061c83559b50a66$exports = require("./pressScale.cjs");
var $1c116ea4b0d2d023$exports = require("./ProgressCircle.cjs");
var $5eb75e0c130e0669$exports = require("../../icons/Skeleton.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $bfa4962d90c8af48$exports = require("./Form.cjs");
var $25d06cf8d4e72761$exports = require("./Button.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $9ZEgK$reactjsxruntime = require("react/jsx-runtime");
var $9ZEgK$reactariacomponentsButton = require("react-aria-components/Button");
var $9ZEgK$reactariacomponentsslots = require("react-aria-components/slots");
var $9ZEgK$react = require("react");
var $9ZEgK$reactariacomponentsDialog = require("react-aria-components/Dialog");
var $9ZEgK$reactariaI18nProvider = require("react-aria/I18nProvider");
var $9ZEgK$reactariauseLocalizedStringFormatter = require("react-aria/useLocalizedStringFormatter");


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "btnStyles", () => $6e265ff388155b91$export$446543cff2a0f496);
$parcel$export(module.exports, "ActionButtonContext", () => $6e265ff388155b91$export$2ff682723a46d413);
$parcel$export(module.exports, "ActionButton", () => $6e265ff388155b91$export$cfc7921d29ef7b80);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 






















// These styles handle both ActionButton and ToggleButton
const $6e265ff388155b91$var$iconOnly = ':has([slot=icon], [slot=avatar]):not(:has([data-rsp-slot=text]))';
const $6e265ff388155b91$var$avatarOnly = ':has([slot=avatar]):not(:has([slot=icon], [data-rsp-slot=text]))';
const $6e265ff388155b91$var$textOnly = ':has([data-rsp-slot=text]):not(:has([slot=icon], [slot=avatar]))';
const $6e265ff388155b91$var$controlStyle = {
    font: {
        "default": "ui",
        size: {
            XS: "ui-xs",
            S: "ui-sm",
            L: "ui-lg",
            XL: "ui-xl"
        }
    },
    display: "flex",
    alignItems: "center",
    boxSizing: "border-box",
    paddingX: {
        "default": "edge-to-text",
        ":has([slot=icon]):not(:has([data-rsp-slot=text]))": 0
    },
    minWidth: {
        "default": 32,
        size: {
            XS: 20,
            S: 24,
            L: 40,
            XL: 48
        }
    },
    "--size": {
        type: "order",
        value: {
            "default": 1,
            size: {
                XS: 0.7901234567901234,
                S: 0.8888888888888888,
                L: 1.125,
                XL: 1.265625
            }
        }
    },
    "--radius": {
        type: "borderTopStartRadius",
        value: "default"
    },
    borderRadius: "round(var(--radius) * var(--size), 1px)",
    columnGap: "text-to-visual",
    "--iconMargin": {
        type: "marginStart",
        value: {
            "default": "[-0.14285714285714285em]",
            ":has([slot=icon]):not(:has([data-rsp-slot=text]))": 0
        }
    },
    height: {
        "default": 32,
        size: {
            XS: 20,
            S: 24,
            L: 40,
            XL: 48
        }
    }
};
const $6e265ff388155b91$export$446543cff2a0f496 = function anonymous(props, overrides) {
    let rules = " ";
    let minWidth = false;
    let flexShrink = false;
    let flexGrow = false;
    let flexBasis = false;
    let width = false;
    let position = false;
    let zIndex = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "N") minWidth = true;
        if (p[1] === "_v") flexShrink = true;
        if (p[1] === "_u") flexGrow = true;
        if (p[1] === "_s") flexBasis = true;
        if (p[1] === "Z") width = true;
        if (p[1] === "_P") position = true;
        if (p[1] === "_9") zIndex = true;
        rules += p[0];
    }
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    if (props.isStaticColor) rules += ' Oe131';
    else rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (props.staticColor === "white") rules += ' -Ovbhqd-gcwrnr131';
    else if (props.staticColor === "black") rules += ' -Ovbhqd-g4hn3s131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    rules += ' xc131';
    rules += ' _xa131';
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isStaticColor) {
        if (props.isDisabled) rules += ' pi131';
        else if (props.isSelected) rules += ' ph131';
        else rules += ' pl131';
    } else if (props.isDisabled) rules += ' pp131';
    else if (props.isSelected) {
        if (props.isEmphasized) rules += ' px131';
        else rules += ' pr131';
    } else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    }
    if (props.isDisabled) rules += ' plc131';
    else if (props.isSelected) rules += ' ple131';
    else rules += ' plb131';
    rules += ' se131';
    rules += ' eb131';
    rules += ' _oa131';
    rules += ' Sv131';
    rules += ' SgLWx3bd131';
    rules += ' SsAUsHd131';
    rules += ' Rv131';
    rules += ' RgLWx3bd131';
    rules += ' RsAUsHd131';
    if (props.size === "XL") {
        if (!minWidth) rules += ' Nl131';
    } else if (props.size === "L") {
        if (!minWidth) rules += ' Nj131';
    } else if (props.size === "S") {
        if (!minWidth) rules += ' Nf131';
    } else if (props.size === "XS") {
        if (!minWidth) rules += ' Ne131';
    } else if (!minWidth) rules += ' Nk131';
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-ob131';
    if (props.density === "compact") {
        rules += ' od131';
        rules += ' obj131';
    } else rules += ' oj131';
    if (props.density === "compact") {
        if (props.orientation === "vertical") {
            rules += ' nd131';
            rules += ' nbj131';
        } else if (props.orientation === "horizontal") {
            rules += ' nd131';
            rules += ' nkj131';
        } else rules += ' nd131';
    } else rules += ' nj131';
    if (props.density === "compact") {
        if (props.orientation === "vertical") {
            rules += ' kd131';
            rules += ' kkj131';
        } else if (props.orientation === "horizontal") {
            rules += ' kd131';
            rules += ' kbj131';
        } else rules += ' kd131';
    } else rules += ' kj131';
    if (props.density === "compact") {
        rules += ' jd131';
        rules += ' jkj131';
    } else rules += ' jj131';
    rules += ' qc131';
    rules += ' qsd131';
    rules += ' -Qg41Lb-IAxCW0c131';
    rules += ' -Qg41Lb-y5UFLdIy131';
    rules += ' -Qg41Lb-sAUsHIy131';
    rules += ' FRuFGYc131';
    rules += ' _Ca131';
    if (props.isInGroup) {
        if (!flexShrink) rules += ' _va131';
    } else if (!flexShrink) rules += ' _vb131';
    if (props.isJustified) {
        if (!flexGrow) rules += ' _ub131';
    }
    if (props.isJustified) {
        if (!flexBasis) rules += ' _sb131';
    }
    if (!width) rules += ' ZJ131';
    rules += ' _4c131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' _yb131';
    if (!position) rules += ' _Pc131';
    rules += ' CjpxCZ131';
    rules += ' Cy5UFLd2sgPTb131';
    rules += ' CwAP22cUolIUb131';
    rules += ' DZQmDFe131';
    rules += ' Dy5UFLdQgYBrc131';
    rules += ' DwAP22cQgYBrc131';
    rules += ' gqMFGYc131';
    rules += ' -_8sjo0b-t5ZbAob131';
    rules += ' _je131';
    if (props.isFocusVisible) {
        if (!zIndex) rules += ' _9d131';
    }
    rules += ' __ca131';
    rules += ' -B5H9Ae-WXN7eQb131';
    rules += ' -B5H9Ae-wAP22cWr131';
    rules += ' -SqFa4c-ZuUmpre131';
    rules += ' -dcAs1c-ZaP94G131';
    rules += ' -dcAs1c-wAP22cZa131';
    if (props.size === "XL") rules += ' -F_-Fz131';
    else if (props.size === "L") rules += ' -F_-Fv131';
    else if (props.size === "S") rules += ' -F_-Fp131';
    else if (props.size === "XS") rules += ' -F_-Fn131';
    else rules += ' -F_-Fx131';
    if (props.isStaticColor) {
        if (props.isSelected) {
            if (props.isDisabled) {
                if (props.isQuiet) rules += ' -g_-g9131';
                else rules += ' -g_-gf131';
            } else {
                if (props.isPressed) rules += ' -g_-gj131';
                else if (props.isFocusVisible) rules += ' -g_-gj131';
                else if (props.isHovered) rules += ' -g_-gj131';
                else rules += ' -g_-gi131';
            }
        } else if (props.isPressed) rules += ' -g_-gg131';
        else if (props.isFocusVisible) rules += ' -g_-gg131';
        else if (props.isHovered) rules += ' -g_-gg131';
        else if (props.isQuiet) rules += ' -g_-g9131';
        else rules += ' -g_-gf131';
    } else if (props.isSelected) {
        if (props.isDisabled) {
            if (props.isQuiet) rules += ' -g_-g9131';
            else rules += ' -g_-gH131';
        } else if (props.isEmphasized) {
            if (props.isFocusVisible) rules += ' -g_-gUgARdd131';
            else if (props.isPressed) rules += ' -g_-gUgARdd131';
            else if (props.isHovered) rules += ' -g_-gUgARdd131';
            else rules += ' -g_-g5qAiPc131';
        } else {
            if (props.isPressed) rules += ' -g_-gu131';
            else if (props.isFocusVisible) rules += ' -g_-gu131';
            else if (props.isHovered) rules += ' -g_-gu131';
            else rules += ' -g_-g0131';
        }
    } else {
        if (props.isPressed) rules += ' -g_-gF131';
        else if (props.isFocusVisible) rules += ' -g_-gF131';
        else if (props.isHovered) rules += ' -g_-gF131';
        else if (props.isQuiet) rules += ' -g_-g9131';
        else rules += ' -g_-gH131';
    }
    if (props.isSelected) {
        if (props.isDisabled) rules += ' -g_-lgb131';
        else rules += ' -g_-lge131';
    } else rules += ' -g_-lgb131';
    return rules;
};
// Matching icon sizes. TBD.
const $6e265ff388155b91$var$avatarSize = {
    XS: 14,
    S: 16,
    M: 20,
    L: 22,
    XL: 26
};
const $6e265ff388155b91$export$2ff682723a46d413 = /*#__PURE__*/ (0, $9ZEgK$react.createContext)(null);
const $6e265ff388155b91$export$cfc7921d29ef7b80 = /*#__PURE__*/ (0, $9ZEgK$react.forwardRef)(function ActionButton(props1, ref) {
    [props1, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props1, ref, $6e265ff388155b91$export$2ff682723a46d413);
    props1 = (0, $bfa4962d90c8af48$exports.useFormProps)(props1);
    let stringFormatter = (0, $9ZEgK$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    let { isPending: isPending = false, holdAffordance: holdAffordance } = props1;
    let domRef = (0, $150382364eecc4bc$exports.useFocusableRef)(ref);
    let overlayTriggerState = (0, $9ZEgK$react.useContext)((0, $9ZEgK$reactariacomponentsDialog.OverlayTriggerStateContext));
    let ctx = (0, $9ZEgK$reactariacomponentsslots.useSlottedContext)((0, $7a26131f6144af2b$exports.ActionButtonGroupContext));
    let isInGroup = !!ctx;
    let { density: density = 'regular', isJustified: isJustified, orientation: orientation = 'horizontal', staticColor: staticColor = props1.staticColor, isQuiet: isQuiet = props1.isQuiet, size: size = props1.size || 'M', isDisabled: isDisabled } = ctx || {};
    let { isProgressVisible: isProgressVisible } = (0, $25d06cf8d4e72761$exports.usePendingState)(isPending);
    let { direction: direction } = (0, $9ZEgK$reactariaI18nProvider.useLocale)();
    return /*#__PURE__*/ (0, $9ZEgK$reactjsxruntime.jsx)((0, $9ZEgK$reactariacomponentsButton.Button), {
        ...props1,
        isDisabled: props1.isDisabled ?? isDisabled,
        ref: domRef,
        style: (0, $2061c83559b50a66$exports.pressScale)(domRef, props1.UNSAFE_style),
        className: (renderProps)=>(props1.UNSAFE_className || '') + $6e265ff388155b91$export$446543cff2a0f496({
                ...renderProps,
                // Retain hover styles when an overlay is open.
                isHovered: renderProps.isHovered || overlayTriggerState?.isOpen || false,
                isDisabled: renderProps.isDisabled || isProgressVisible,
                staticColor: staticColor,
                isStaticColor: !!staticColor,
                size: size,
                isQuiet: isQuiet,
                density: density,
                isJustified: isJustified,
                orientation: orientation,
                isInGroup: isInGroup
            }, props1.styles),
        children: ({ isDisabled: isDisabled })=>/*#__PURE__*/ (0, $9ZEgK$reactjsxruntime.jsxs)((0, $9ZEgK$reactjsxruntime.Fragment), {
                children: [
                    /*#__PURE__*/ (0, $9ZEgK$reactjsxruntime.jsxs)((0, $9ZEgK$reactariacomponentsslots.Provider), {
                        values: [
                            [
                                (0, $5eb75e0c130e0669$exports.SkeletonContext),
                                null
                            ],
                            [
                                (0, $6367bc87eb7d24ad$exports.TextContext),
                                {
                                    styles: function anonymous(props) {
                                        let rules = " ";
                                        rules += ' zYoLErc131';
                                        rules += ' yYoLErc131';
                                        rules += ' BYoLErc131';
                                        rules += ' AYoLErc131';
                                        rules += ' _Nc131';
                                        rules += ' Pc131';
                                        rules += ' _Yb131';
                                        rules += ' _7c131';
                                        if (props.isProgressVisible) rules += ' _6b131';
                                        return rules;
                                    }({
                                        isProgressVisible: isProgressVisible
                                    })
                                }
                            ],
                            [
                                (0, $bde97c91243ed164$exports.IconContext),
                                {
                                    render: (0, $e991cbcdf82ced71$exports.centerBaseline)({
                                        slot: 'icon',
                                        styles: " z2U5Crc131 y2U5Crc131 B2U5Crc131 A2U5Crc131"
                                    }),
                                    styles: function anonymous(props) {
                                        let rules = " ";
                                        rules += ' ZuUmpre131';
                                        rules += ' FuUmpre131';
                                        rules += ' I67logd131';
                                        if (props.isProgressVisible) rules += ' _6b131';
                                        return rules;
                                    }({
                                        isProgressVisible: isProgressVisible
                                    })
                                }
                            ],
                            [
                                (0, $38071d9ce246d4cf$exports.AvatarContext),
                                {
                                    size: $6e265ff388155b91$var$avatarSize[size],
                                    styles: " I67logd131 z2U5Crc131 y2U5Crc131 B2U5Crc131 A2U5Crc131"
                                }
                            ],
                            [
                                (0, $053b76ed3d29e13b$exports.ImageContext),
                                {
                                    styles: function anonymous(props) {
                                        let rules = " ";
                                        if (props.isProgressVisible) rules += ' _6b131';
                                        return rules;
                                    }({
                                        isProgressVisible: isProgressVisible
                                    })
                                }
                            ],
                            [
                                (0, $cd7301fb33ba8690$exports.NotificationBadgeContext),
                                {
                                    staticColor: staticColor,
                                    size: props1.size === 'XS' ? undefined : props1.size,
                                    isDisabled: isDisabled,
                                    styles: function anonymous(props) {
                                        let rules = " ";
                                        rules += ' _Pa131';
                                        rules += ' WtedXZc131';
                                        rules += ' JlZStSb131';
                                        rules += ' Ijgro3c131';
                                        rules += ' z2mU131';
                                        rules += ' _A1IiWH131';
                                        if (props.isProgressVisible) rules += ' _6b131';
                                        return rules;
                                    }({
                                        isProgressVisible: isProgressVisible
                                    })
                                }
                            ]
                        ],
                        children: [
                            typeof props1.children === 'string' ? /*#__PURE__*/ (0, $9ZEgK$reactjsxruntime.jsx)((0, $6367bc87eb7d24ad$exports.Text), {
                                children: props1.children
                            }) : props1.children,
                            isPending && /*#__PURE__*/ (0, $9ZEgK$reactjsxruntime.jsx)("div", {
                                className: function anonymous(props) {
                                    let rules = " ";
                                    rules += ' _Pa131';
                                    rules += ' WF131';
                                    rules += ' _EF131';
                                    rules += ' _Zc131';
                                    if (props.isProgressVisible) rules += ' _6c131';
                                    else rules += ' _6b131';
                                    return rules;
                                }({
                                    isProgressVisible: isProgressVisible,
                                    isPending: isPending
                                }),
                                children: /*#__PURE__*/ (0, $9ZEgK$reactjsxruntime.jsx)((0, $1c116ea4b0d2d023$exports.ProgressCircle), {
                                    isIndeterminate: true,
                                    "aria-label": stringFormatter.format('button.pending'),
                                    size: "S",
                                    staticColor: staticColor,
                                    styles: function anonymous(props) {
                                        let rules = " ";
                                        if (props.size === "XL") rules += ' Zo131';
                                        else if (props.size === "L") rules += ' Zm131';
                                        else if (props.size === "M") rules += ' Zl131';
                                        else if (props.size === "S") rules += ' Zk131';
                                        else if (props.size === "XS") rules += ' Zj131';
                                        if (props.size === "XL") rules += ' Fp131';
                                        else if (props.size === "L") rules += ' Fn131';
                                        else if (props.size === "M") rules += ' Fm131';
                                        else if (props.size === "S") rules += ' Fl131';
                                        else if (props.size === "XS") rules += ' Fk131';
                                        return rules;
                                    }({
                                        size: size
                                    })
                                })
                            })
                        ]
                    }),
                    holdAffordance && /*#__PURE__*/ (0, $9ZEgK$reactjsxruntime.jsx)((0, $411768293e842abb$exports.default), {
                        size: size === 'XS' ? 'S' : size,
                        className: function anonymous(props) {
                            let rules = " ";
                            rules += ' _Pa131';
                            if (props.size === "XL") rules += ' _zWJpoB131';
                            else if (props.size === "L") rules += ' _zC1otk131';
                            else if (props.size === "M") rules += ' _zE131';
                            else if (props.size === "S") rules += ' _zwtfhvc131';
                            else if (props.size === "XS") rules += ' _zwtfhvc131';
                            if (props.size === "XL") rules += ' _lWJpoB131';
                            else if (props.size === "L") rules += ' _lC1otk131';
                            else if (props.size === "M") rules += ' _lE131';
                            else if (props.size === "S") rules += ' _lwtfhvc131';
                            else if (props.size === "XS") rules += ' _lwtfhvc131';
                            if (props.direction === "rtl") {
                                rules += ' -Z4Pn4c-a131';
                                rules += ' __Ha131';
                            }
                            return rules;
                        }({
                            direction: direction,
                            size: size
                        })
                    })
                ]
            })
    });
});
