// @react-spectrum/s2/dist/private/ActionButtonGroup.cjs
require("./ActionButtonGroup.css");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $jGK4W$reactjsxruntime = require("react/jsx-runtime");
var $jGK4W$react = require("react");
var $jGK4W$reactariacomponentsToolbar = require("react-aria-components/Toolbar");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "actionGroupStyle", () => $7a26131f6144af2b$export$641cbcd1f6f6802f);
$parcel$export(module.exports, "ActionButtonGroupContext", () => $7a26131f6144af2b$export$33f5f2f2cb85d743);
$parcel$export(module.exports, "ActionButtonGroup", () => $7a26131f6144af2b$export$73d695ae27330056);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 



const $7a26131f6144af2b$export$641cbcd1f6f6802f = function anonymous(props, overrides) {
    let rules = " ";
    rules += (String(overrides || '').match(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L|F|M|K)[^\s]+/g) || []).join('');
    rules += ' sd131';
    if (props.orientation === "vertical") rules += ' _ta131';
    else if (props.orientation === "horizontal") rules += ' _tc131';
    if (props.density === "regular") {
        if (props.size === "XL") rules += ' Ue131';
        else if (props.size === "L") rules += ' Ue131';
        else if (props.size === "M") rules += ' Ue131';
        else if (props.size === "S") rules += ' Ub131';
        else if (props.size === "XS") rules += ' Ub131';
    } else if (props.density === "compact") rules += ' Ua131';
    if (props.density === "regular") {
        if (props.size === "XL") rules += ' qe131';
        else if (props.size === "L") rules += ' qe131';
        else if (props.size === "M") rules += ' qe131';
        else if (props.size === "S") rules += ' qb131';
        else if (props.size === "XS") rules += ' qb131';
    } else if (props.density === "compact") rules += ' qa131';
    return rules;
};
const $7a26131f6144af2b$export$33f5f2f2cb85d743 = /*#__PURE__*/ (0, $jGK4W$react.createContext)(null);
const $7a26131f6144af2b$export$73d695ae27330056 = /*#__PURE__*/ (0, $jGK4W$react.forwardRef)(function ActionButtonGroup(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $7a26131f6144af2b$export$33f5f2f2cb85d743);
    let { density: density = 'regular', size: size = 'M', orientation: orientation = 'horizontal', isJustified: isJustified, children: children, UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, styles: styles } = props;
    return /*#__PURE__*/ (0, $jGK4W$reactjsxruntime.jsx)((0, $jGK4W$reactariacomponentsToolbar.Toolbar), {
        ...props,
        ref: ref,
        style: UNSAFE_style,
        className: UNSAFE_className + $7a26131f6144af2b$export$641cbcd1f6f6802f({
            size: size,
            density: density,
            orientation: orientation,
            isJustified: isJustified
        }, styles),
        children: /*#__PURE__*/ (0, $jGK4W$reactjsxruntime.jsx)($7a26131f6144af2b$export$33f5f2f2cb85d743.Provider, {
            value: props,
            children: children
        })
    });
});
