// @react-spectrum/s2/dist/private/AvatarGroup.cjs
require("./AvatarGroup.css");
var $38071d9ce246d4cf$exports = require("./Avatar.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $6mB16$reactjsxruntime = require("react/jsx-runtime");
var $6mB16$react = require("react");
var $6mB16$reactariafilterDOMProps = require("react-aria/filterDOMProps");
var $6mB16$reactariauseLabel = require("react-aria/useLabel");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "AvatarGroupContext", () => $5b7eb34c05dfeb90$export$4f18251748129091);
$parcel$export(module.exports, "AvatarGroup", () => $5b7eb34c05dfeb90$export$5b26ebaf2b105908);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 






const $5b7eb34c05dfeb90$export$4f18251748129091 = /*#__PURE__*/ (0, $6mB16$react.createContext)(null);
const $5b7eb34c05dfeb90$var$avatar = " I4gnV8d131 Iby131";
const $5b7eb34c05dfeb90$var$text = function anonymous(props) {
    let rules = " ";
    rules += ' Iv131';
    rules += ' _Nc131';
    rules += ' Pc131';
    rules += ' _Yb131';
    rules += ' _7c131';
    if (props.size === "40") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "36") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "32") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "28") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "24") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "20") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "16") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "40") {
        rules += ' -_6BNtrc-g131';
        rules += ' vx131';
    } else if (props.size === "36") {
        rules += ' -_6BNtrc-f131';
        rules += ' vx131';
    } else if (props.size === "32") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "28") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "24") {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    } else if (props.size === "20") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "16") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    }
    if (props.size === "40") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "36") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "32") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "28") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "24") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "20") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "16") {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "40") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "36") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "32") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "28") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "24") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "20") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "16") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.size === "40") rules += ' pt131';
    else if (props.size === "36") rules += ' pt131';
    else if (props.size === "32") rules += ' pt131';
    else if (props.size === "28") rules += ' pt131';
    else if (props.size === "24") rules += ' pt131';
    else if (props.size === "20") rules += ' pt131';
    else if (props.size === "16") rules += ' pt131';
    return rules;
};
const $5b7eb34c05dfeb90$var$container = function anonymous(props, overrides) {
    let rules = " ";
    rules += (String(overrides || '').match(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6)[^\s]+/g) || []).join('');
    rules += ' sd131';
    rules += ' eb131';
    return rules;
};
const $5b7eb34c05dfeb90$export$5b26ebaf2b105908 = /*#__PURE__*/ (0, $6mB16$react.forwardRef)(function AvatarGroup(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $5b7eb34c05dfeb90$export$4f18251748129091);
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let { children: children, label: label, size: size = 24, styles: styles, UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className, ...otherProps } = props;
    let { labelProps: labelProps, fieldProps: fieldProps } = (0, $6mB16$reactariauseLabel.useLabel)({
        ...props,
        labelElementType: 'span'
    });
    return /*#__PURE__*/ (0, $6mB16$reactjsxruntime.jsx)((0, $38071d9ce246d4cf$exports.AvatarContext).Provider, {
        value: {
            styles: $5b7eb34c05dfeb90$var$avatar,
            size: size,
            isOverBackground: true
        },
        children: /*#__PURE__*/ (0, $6mB16$reactjsxruntime.jsxs)("div", {
            ref: domRef,
            ...(0, $6mB16$reactariafilterDOMProps.filterDOMProps)(otherProps),
            ...fieldProps,
            role: "group",
            className: (UNSAFE_className ?? '') + $5b7eb34c05dfeb90$var$container(null, styles),
            style: {
                ...UNSAFE_style,
                '--size': size / 16 + 'rem'
            },
            children: [
                children,
                label && /*#__PURE__*/ (0, $6mB16$reactjsxruntime.jsx)("span", {
                    ...labelProps,
                    className: $5b7eb34c05dfeb90$var$text({
                        size: String(size)
                    }),
                    children: label
                })
            ]
        })
    });
});
