// @react-spectrum/s2/dist/private/Badge.cjs
require("./Badge.css");
var $e991cbcdf82ced71$exports = require("./CenterBaseline.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $5eb75e0c130e0669$exports = require("../../icons/Skeleton.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $26Nq1$reactjsxruntime = require("react/jsx-runtime");
var $26Nq1$reactariacomponentsslots = require("react-aria-components/slots");
var $26Nq1$reactariafilterDOMProps = require("react-aria/filterDOMProps");
var $26Nq1$reactariacomponentsText = require("react-aria-components/Text");
var $26Nq1$react = require("react");


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "BadgeContext", () => $660a621e3a266afc$export$1d4627584e49159a);
$parcel$export(module.exports, "Badge", () => $660a621e3a266afc$export$37acb3580601e69a);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 










const $660a621e3a266afc$export$1d4627584e49159a = /*#__PURE__*/ (0, $26Nq1$react.createContext)(null);
const $660a621e3a266afc$var$badge = function anonymous(props, overrides) {
    let rules = " ";
    let minWidth = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "N") minWidth = true;
        rules += p[0];
    }
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.fillStyle === "outline") rules += ' pm131';
    else if (props.fillStyle === "subtle") rules += ' pm131';
    else if (props.fillStyle === "bold") {
        if (props.variant === "celery") rules += ' pg131';
        else if (props.variant === "chartreuse") rules += ' pg131';
        else if (props.variant === "yellow") rules += ' pg131';
        else if (props.variant === "orange") rules += ' pg131';
        else if (props.variant === "notice") rules += ' pg131';
        else rules += ' px131';
    }
    rules += ' sd131';
    rules += ' ea131';
    rules += ' egLWx3bb131';
    rules += ' _oa131';
    rules += ' Sv131';
    rules += ' SgLWx3bd131';
    rules += ' Rv131';
    rules += ' RgLWx3bd131';
    if (props.size === "XL") {
        if (!minWidth) rules += ' Nl131';
    } else if (props.size === "L") {
        if (!minWidth) rules += ' Nj131';
    } else if (props.size === "S") {
        if (!minWidth) rules += ' Nf131';
    } else if (props.size === "XS") {
        if (!minWidth) rules += ' Ne131';
    } else if (!minWidth) rules += ' Nk131';
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-ob131';
    rules += ' oj131';
    rules += ' nj131';
    rules += ' kj131';
    rules += ' jj131';
    rules += ' qc131';
    rules += ' qsd131';
    rules += ' -Qg41Lb-IAxCW0c131';
    rules += ' -Qg41Lb-gLWx3bIy131';
    rules += ' MAyFGYc131';
    rules += ' Td131';
    rules += ' Qd131';
    rules += ' -aRvJDd-Tu7Fubd131';
    rules += ' _Ca131';
    if (props.fillStyle === "outline") rules += ' g_g131';
    else if (props.fillStyle === "subtle") {
        if (props.variant === "silver") rules += ' g_f131';
        else if (props.variant === "cinnamon") rules += ' gP131';
        else if (props.variant === "brown") rules += ' gO131';
        else if (props.variant === "turquoise") rules += ' gy131';
        else if (props.variant === "pink") rules += ' gT131';
        else if (props.variant === "magenta") rules += ' gS131';
        else if (props.variant === "fuchsia") rules += ' gR131';
        else if (props.variant === "purple") rules += ' gM131';
        else if (props.variant === "indigo") rules += ' gK131';
        else if (props.variant === "blue") rules += ' gG131';
        else if (props.variant === "cyan") rules += ' gC131';
        else if (props.variant === "seafoam") rules += ' gz131';
        else if (props.variant === "green") rules += ' gA131';
        else if (props.variant === "celery") rules += ' gv131';
        else if (props.variant === "chartreuse") rules += ' gJ131';
        else if (props.variant === "yellow") rules += ' gW131';
        else if (props.variant === "orange") rules += ' gV131';
        else if (props.variant === "red") rules += ' gU131';
        else if (props.variant === "gray") rules += ' gI131';
        else if (props.variant === "negative") rules += ' gU131';
        else if (props.variant === "notice") rules += ' gV131';
        else if (props.variant === "positive") rules += ' gA131';
        else if (props.variant === "neutral") rules += ' gI131';
        else if (props.variant === "informative") rules += ' gG131';
        else if (props.variant === "accent") rules += ' gG131';
    } else if (props.fillStyle === "bold") {
        if (props.variant === "silver") rules += ' gm131';
        else if (props.variant === "cinnamon") rules += ' gq131';
        else if (props.variant === "brown") rules += ' gn131';
        else if (props.variant === "turquoise") rules += ' g5131';
        else if (props.variant === "pink") rules += ' gx131';
        else if (props.variant === "magenta") rules += ' gD131';
        else if (props.variant === "fuchsia") rules += ' gr131';
        else if (props.variant === "purple") rules += ' go131';
        else if (props.variant === "indigo") rules += ' gl131';
        else if (props.variant === "blue") rules += ' g2131';
        else if (props.variant === "cyan") rules += ' gk131';
        else if (props.variant === "seafoam") rules += ' g4131';
        else if (props.variant === "green") rules += ' g1131';
        else if (props.variant === "celery") rules += ' g7131';
        else if (props.variant === "chartreuse") rules += ' gp131';
        else if (props.variant === "yellow") rules += ' gN131';
        else if (props.variant === "orange") rules += ' gQ131';
        else if (props.variant === "red") rules += ' gB131';
        else if (props.variant === "gray") rules += ' g6131';
        else if (props.variant === "negative") rules += ' gB131';
        else if (props.variant === "notice") rules += ' gQ131';
        else if (props.variant === "positive") rules += ' g1131';
        else if (props.variant === "neutral") rules += ' g6131';
        else if (props.variant === "informative") rules += ' g2131';
        else if (props.variant === "accent") rules += ' g2131';
    }
    rules += ' _jf131';
    rules += ' _kJ8iPYd131';
    rules += ' hXMFGYc131';
    rules += ' mc131';
    rules += ' lc131';
    if (props.fillStyle === "outline") {
        if (props.variant === "negative") rules += ' iFLVMS131';
        else if (props.variant === "notice") rules += ' i9VrKkd131';
        else if (props.variant === "positive") rules += ' ip17Jm131';
        else if (props.variant === "neutral") rules += ' i0ruhAb131';
        else if (props.variant === "informative") rules += ' iG1ze4d131';
        else if (props.variant === "accent") rules += ' iG1ze4d131';
        else rules += ' iA131';
    } else rules += ' iA131';
    rules += ' -_8sjo0b-t5ZbAob131';
    if (props.size === "XL") rules += ' -M_-Mk131';
    else if (props.size === "L") rules += ' -M_-Mh131';
    else if (props.size === "S") rules += ' -M_-Mg131';
    else if (props.size === "XS") rules += ' -M_-Mf131';
    else rules += ' -M_-Mi131';
    rules += ' -_k_-_kc131';
    rules += ' -h_-hc131';
    return rules;
};
const $660a621e3a266afc$export$37acb3580601e69a = /*#__PURE__*/ (0, $26Nq1$react.forwardRef)(function Badge(props1, ref) {
    [props1, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props1, ref, $660a621e3a266afc$export$1d4627584e49159a);
    let { children: children, variant: variant = 'neutral', size: size = 'S', fillStyle: fillStyle = 'bold', overflowMode: overflowMode = 'wrap', ...otherProps } = props1; // useProviderProps(props) in v3
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let isTextOnly = (0, ($parcel$interopDefault($26Nq1$react))).Children.toArray(props1.children).every((c)=>!/*#__PURE__*/ (0, ($parcel$interopDefault($26Nq1$react))).isValidElement(c));
    return /*#__PURE__*/ (0, $26Nq1$reactjsxruntime.jsx)((0, $26Nq1$reactariacomponentsslots.Provider), {
        values: [
            [
                (0, $6367bc87eb7d24ad$exports.TextContext),
                {
                    styles: function anonymous(props) {
                        let rules = " ";
                        rules += ' TAzEcG131';
                        rules += ' QAzEcG131';
                        rules += ' _Jb131';
                        rules += ' _Nc131';
                        rules += ' Pc131';
                        rules += ' _Yb131';
                        if (props.overflowMode === "wrap") rules += ' _7b131';
                        else if (props.overflowMode === "truncate") rules += ' _7c131';
                        return rules;
                    }({
                        overflowMode: overflowMode
                    })
                }
            ],
            [
                (0, $26Nq1$reactariacomponentsText.TextContext),
                {}
            ],
            [
                (0, $bde97c91243ed164$exports.IconContext),
                {
                    render: (0, $e991cbcdf82ced71$exports.centerBaseline)({
                        slot: 'icon',
                        styles: " _Ja131"
                    }),
                    styles: " ZuUmpre131 FuUmpre131 I67logd131 _va131"
                }
            ]
        ],
        children: /*#__PURE__*/ (0, $26Nq1$reactjsxruntime.jsx)((0, $5eb75e0c130e0669$exports.SkeletonWrapper), {
            children: /*#__PURE__*/ (0, $26Nq1$reactjsxruntime.jsx)("span", {
                ...(0, $26Nq1$reactariafilterDOMProps.filterDOMProps)(otherProps),
                role: "presentation",
                className: (props1.UNSAFE_className || '') + $660a621e3a266afc$var$badge({
                    variant: variant,
                    size: size,
                    fillStyle: fillStyle
                }, props1.styles),
                style: props1.UNSAFE_style,
                ref: domRef,
                children: typeof children === 'string' || isTextOnly ? /*#__PURE__*/ (0, $26Nq1$reactjsxruntime.jsx)((0, $6367bc87eb7d24ad$exports.Text), {
                    children: children
                }) : children
            })
        })
    });
});
