// @react-spectrum/s2/dist/private/Breadcrumbs.cjs
require("./Breadcrumbs.css");
var $6e265ff388155b91$exports = require("./ActionButton.cjs");
var $1df9f1c9262ce5df$exports = require("./Chevron.cjs");
var $280c0303c82c816f$exports = require("../../icons/FolderBreadcrumb.cjs");
var $4526404114e78c80$exports = require("./intlStrings.cjs");
var $e741ea6b88ce4866$exports = require("./Menu.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $PCemd$reactjsxruntime = require("react/jsx-runtime");
var $PCemd$react = require("react");
var $PCemd$reactariacomponentsBreadcrumbs = require("react-aria-components/Breadcrumbs");
var $PCemd$reactariacomponentsCollectionBuilder = require("react-aria-components/CollectionBuilder");
var $PCemd$reactariacomponentsslots = require("react-aria-components/slots");
var $PCemd$reactariacomponentsHeading = require("react-aria-components/Heading");
var $PCemd$reactariaprivateutilsinertValue = require("react-aria/private/utils/inertValue");
var $PCemd$reactariacomponentsLink = require("react-aria-components/Link");
var $PCemd$reactariaprivateutilsuseLayoutEffect = require("react-aria/private/utils/useLayoutEffect");
var $PCemd$reactariaI18nProvider = require("react-aria/I18nProvider");
var $PCemd$reactariauseLocalizedStringFormatter = require("react-aria/useLocalizedStringFormatter");
var $PCemd$reactariaprivateutilsuseResizeObserver = require("react-aria/private/utils/useResizeObserver");


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "BreadcrumbsContext", () => $4130f4e17f7081aa$export$65596d3621b0a4a0);
$parcel$export(module.exports, "Breadcrumbs", () => $4130f4e17f7081aa$export$2dc68d50d56fbbd);
$parcel$export(module.exports, "Breadcrumb", () => $4130f4e17f7081aa$export$dabcc1ec9dd9d1cc);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 



















const $4130f4e17f7081aa$var$MIN_VISIBLE_ITEMS = 1;
const $4130f4e17f7081aa$var$MAX_VISIBLE_ITEMS = 4;
const $4130f4e17f7081aa$export$65596d3621b0a4a0 = /*#__PURE__*/ (0, $PCemd$react.createContext)(null);
const $4130f4e17f7081aa$var$wrapper = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let flexGrow = false;
    let flexShrink = false;
    let flexBasis = false;
    let marginTop = false;
    let marginBottom = false;
    let marginInlineStart = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "_u") flexGrow = true;
        if (p[1] === "_v") flexShrink = true;
        if (p[1] === "_s") flexBasis = true;
        if (p[1] === "J") marginTop = true;
        if (p[1] === "G") marginBottom = true;
        if (p[1] === "I") marginInlineStart = true;
        rules += p[0];
    }
    if (!position) rules += ' _Pc131';
    rules += ' sd131';
    rules += ' _Cg131';
    rules += ' _Gc131';
    rules += ' _wa131';
    if (!flexGrow) rules += ' _ub131';
    if (!flexShrink) rules += ' _va131';
    if (!flexBasis) rules += ' _sb131';
    if (props.size === "L") rules += ' Urun3rd131';
    else if (props.size === "M") rules += ' UssxDec131';
    if (props.size === "L") rules += ' qrun3rd131';
    else if (props.size === "M") rules += ' qssxDec131';
    rules += ' Td131';
    rules += ' Qd131';
    rules += ' Sd131';
    rules += ' Rd131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (!marginTop) rules += ' Jy131';
    if (!marginBottom) rules += ' Gy131';
    if (props.size === "L") {
        if (!marginInlineStart) rules += ' Irun3rd131';
    } else if (props.size === "M") {
        if (!marginInlineStart) rules += ' IssxDec131';
    }
    return rules;
};
const $4130f4e17f7081aa$var$InternalBreadcrumbsContext = /*#__PURE__*/ (0, $PCemd$react.createContext)({});
const $4130f4e17f7081aa$export$2dc68d50d56fbbd = /*#__PURE__*/ (0, $PCemd$react.forwardRef)(function Breadcrumbs(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $4130f4e17f7081aa$export$65596d3621b0a4a0);
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let { UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, styles: styles, size: size = 'M', children: children, isDisabled: isDisabled, ...otherProps } = props;
    return /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $PCemd$reactariacomponentsslots.Provider), {
        values: [
            [
                $4130f4e17f7081aa$var$InternalBreadcrumbsContext,
                {
                    size: size,
                    isDisabled: isDisabled
                }
            ]
        ],
        children: /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)($4130f4e17f7081aa$var$CollapsingCollection, {
            containerRef: domRef,
            onAction: props.onAction,
            children: /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $PCemd$reactariacomponentsBreadcrumbs.Breadcrumbs), {
                ...otherProps,
                isDisabled: isDisabled,
                ref: domRef,
                style: UNSAFE_style,
                className: UNSAFE_className + $4130f4e17f7081aa$var$wrapper({
                    size: size
                }, styles),
                children: children
            })
        })
    });
});
let $4130f4e17f7081aa$var$BreadcrumbMenu = (props)=>{
    let stringFormatter = (0, $PCemd$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    let { items: items, onAction: onAction } = props;
    let { direction: direction } = (0, $PCemd$reactariaI18nProvider.useLocale)();
    let { size: size, isDisabled: isDisabled } = (0, $PCemd$react.useContext)($4130f4e17f7081aa$var$InternalBreadcrumbsContext);
    let label = stringFormatter.format('breadcrumbs.more');
    return /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $PCemd$reactariacomponentsCollectionBuilder.CollectionRendererContext).Provider, {
        value: (0, $PCemd$reactariacomponentsCollectionBuilder.DefaultCollectionRenderer),
        children: /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsxs)("li", {
            className: $4130f4e17f7081aa$var$breadcrumbStyles({
                size: size,
                isDisabled: isDisabled,
                isMenu: true
            }),
            children: [
                /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsxs)((0, $e741ea6b88ce4866$exports.MenuTrigger), {
                    children: [
                        /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $6e265ff388155b91$exports.ActionButton), {
                            isDisabled: isDisabled,
                            isQuiet: true,
                            "aria-label": label,
                            children: /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $280c0303c82c816f$exports.default), {})
                        }),
                        /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $e741ea6b88ce4866$exports.Menu), {
                            items: items,
                            onAction: onAction,
                            children: (item)=>/*#__PURE__*/ (0, $PCemd$react.createElement)((0, $e741ea6b88ce4866$exports.MenuItem), {
                                    ...item.props.originalProps,
                                    key: item.key
                                }, /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $6367bc87eb7d24ad$exports.Text), {
                                    slot: "label",
                                    children: item.props.children({
                                        size: size,
                                        isCurrent: false,
                                        isMenu: true
                                    })
                                }))
                        })
                    ]
                }),
                /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $1df9f1c9262ce5df$exports.default), {
                    size: size,
                    className: $4130f4e17f7081aa$var$chevronStyles({
                        direction: direction,
                        isMenu: true
                    })
                })
            ]
        })
    });
};
let $4130f4e17f7081aa$var$HiddenBreadcrumbs = function(props) {
    let { listRef: listRef, items: items, size: size } = props;
    return /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsxs)("div", {
        // @ts-ignore
        inert: (0, $PCemd$reactariaprivateutilsinertValue.inertValue)(true),
        ref: listRef,
        className: " s40ub4c131 U40ub4c131 q40ub4c131 _w40ub4c131 _Pa131 Wr131 _lr131 _6b131 _Nc131 Pc131 _Ia131",
        children: [
            items.map((item, idx)=>{
                // pull off individual props as an allow list, don't want refs or other props getting through
                return /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)("div", {
                    "data-hidden-breadcrumb": true,
                    style: item.props.UNSAFE_style,
                    className: item.props.className({
                        size: size,
                        isCurrent: idx === items.length - 1
                    }),
                    children: item.props.children({
                        size: size,
                        isCurrent: idx === items.length - 1
                    })
                }, item.key);
            }),
            /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $6e265ff388155b91$exports.ActionButton), {
                "data-hidden-button": true,
                isQuiet: true,
                children: /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $280c0303c82c816f$exports.default), {})
            })
        ]
    });
};
const $4130f4e17f7081aa$var$breadcrumbStyles = function anonymous(props) {
    let rules = " ";
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _Cg131';
    if (props.size === "XL") rules += ' Fz131';
    else if (props.size === "L") rules += ' Fv131';
    else if (props.size === "S") rules += ' Fp131';
    else if (props.size === "XS") rules += ' Fn131';
    else rules += ' Fx131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' _Pc131';
    rules += ' _va131';
    if (props.isDisabled) rules += ' pp131';
    else rules += ' pt131';
    if (props.isDisabled) rules += ' plc131';
    else rules += ' plb131';
    rules += ' _je131';
    if (props.isMenu) rules += ' I34mdr131';
    return rules;
};
const $4130f4e17f7081aa$var$chevronStyles = function anonymous(props) {
    let rules = " ";
    if (props.direction === "rtl") {
        rules += ' -Z4Pn4c-a131';
        rules += ' __Ha131';
    }
    if (props.direction === "rtl") {
        rules += ' -_04Pn4c-a131';
        rules += ' __Ha131';
    }
    if (props.isMenu) rules += ' Iy131';
    else {
        rules += ' It131';
        rules += ' Isu131';
    }
    rules += ' pt131';
    rules += ' plf131';
    rules += ' -_8sjo0b-t5ZbAob131';
    return rules;
};
const $4130f4e17f7081aa$var$linkStyles = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    rules += ' oa131';
    rules += ' na131';
    rules += ' ka131';
    rules += ' ja131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isCurrent) {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    } else if (props.isDisabled) rules += ' pp131';
    else {
        if (props.isPressed) rules += ' pt131';
        else if (props.isFocusVisible) rules += ' pt131';
        else if (props.isHovered) rules += ' pt131';
        else rules += ' pv131';
    }
    if (props.isCurrent) rules += ' plc131';
    else if (props.isDisabled) rules += ' plc131';
    else rules += ' plf131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.isDisabled) rules += ' _Xa131';
    else if (props.isFocusVisible) {
        rules += ' _Xb131';
        rules += ' __Va131';
    } else if (props.isHovered) {
        rules += ' _Xb131';
        rules += ' __Va131';
    } else rules += ' _Xa131';
    if (props.isDisabled) rules += ' ri131';
    else rules += ' ry131';
    rules += ' __ca131';
    return rules;
};
const $4130f4e17f7081aa$var$currentStyles = function anonymous(props) {
    let rules = " ";
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    rules += ' xd131';
    rules += ' _xa131';
    rules += ' xX0cczbc131';
    rules += ' _xX0cczba131';
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    rules += ' pt131';
    rules += ' plb131';
    return rules;
};
// TODO: support user heading size customization, for now just set it to large
const $4130f4e17f7081aa$var$heading = " Jy131 Gy131 Iy131 Hy131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-h131 vx131 xe131 _xa131 xX0cczbd131 _xX0cczba131 _Fd131 _FnuYUweb131 po131";
const $4130f4e17f7081aa$export$dabcc1ec9dd9d1cc = /*#__PURE__*/ (0, $PCemd$react.forwardRef)(function Breadcrumb({ children: children, ...props }, ref) {
    let { href: href, target: target, rel: rel, download: download, ping: ping, referrerPolicy: referrerPolicy } = props;
    let { size: size = 'M' } = (0, $PCemd$react.useContext)($4130f4e17f7081aa$var$InternalBreadcrumbsContext) ?? {};
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let { direction: direction } = (0, $PCemd$reactariaI18nProvider.useLocale)();
    return /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $PCemd$reactariacomponentsBreadcrumbs.Breadcrumb), {
        ...props,
        ref: domRef,
        // @ts-ignore
        originalProps: props,
        className: ({ isCurrent: isCurrent, isDisabled: isDisabled })=>$4130f4e17f7081aa$var$breadcrumbStyles({
                size: size,
                isCurrent: isCurrent,
                isDisabled: isDisabled
            }),
        children: ({ isCurrent: isCurrent, isDisabled: isDisabled, isMenu: // @ts-ignore
        isMenu })=>{
            if (isMenu) return children;
            return isCurrent ? /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)("div", {
                className: $4130f4e17f7081aa$var$currentStyles({
                    size: size
                }),
                children: /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $PCemd$reactariacomponentsslots.Provider), {
                    values: [
                        [
                            (0, $PCemd$reactariacomponentsHeading.HeadingContext),
                            {
                                className: $4130f4e17f7081aa$var$heading
                            }
                        ]
                    ],
                    children: children
                })
            }) : /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsxs)((0, $PCemd$reactjsxruntime.Fragment), {
                children: [
                    /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $PCemd$reactariacomponentsLink.Link), {
                        style: ({ isFocusVisible: isFocusVisible })=>({
                                clipPath: isFocusVisible ? 'none' : 'margin-box'
                            }),
                        href: href,
                        target: target,
                        rel: rel,
                        download: download,
                        ping: ping,
                        referrerPolicy: referrerPolicy,
                        isDisabled: isDisabled || isCurrent,
                        className: ({ isFocused: isFocused, isFocusVisible: isFocusVisible, isHovered: isHovered, isDisabled: isDisabled, isPressed: isPressed })=>$4130f4e17f7081aa$var$linkStyles({
                                isFocused: isFocused,
                                isFocusVisible: isFocusVisible,
                                isHovered: isHovered,
                                isDisabled: isDisabled,
                                size: size,
                                isPressed: isPressed,
                                isCurrent: isCurrent
                            }),
                        children: children
                    }),
                    /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $1df9f1c9262ce5df$exports.default), {
                        size: "M",
                        className: $4130f4e17f7081aa$var$chevronStyles({
                            direction: direction
                        })
                    })
                ]
            });
        }
    });
});
// Context for passing the count for the custom renderer
let $4130f4e17f7081aa$var$CollapseContext = /*#__PURE__*/ (0, $PCemd$react.createContext)(null);
function $4130f4e17f7081aa$var$CollapsingCollection({ children: children, containerRef: containerRef, onAction: onAction }) {
    return /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)($4130f4e17f7081aa$var$CollapseContext.Provider, {
        value: {
            containerRef: containerRef,
            onAction: onAction
        },
        children: /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $PCemd$reactariacomponentsCollectionBuilder.CollectionRendererContext).Provider, {
            value: $4130f4e17f7081aa$var$CollapsingCollectionRenderer,
            children: children
        })
    });
}
let $4130f4e17f7081aa$var$CollapsingCollectionRenderer = {
    CollectionRoot ({ collection: collection }) {
        return $4130f4e17f7081aa$var$useCollectionRender(collection);
    },
    CollectionBranch ({ collection: collection }) {
        return $4130f4e17f7081aa$var$useCollectionRender(collection);
    }
};
let $4130f4e17f7081aa$var$useCollectionRender = (collection)=>{
    let { containerRef: containerRef, onAction: onAction } = (0, $PCemd$react.useContext)($4130f4e17f7081aa$var$CollapseContext) ?? {};
    let [visibleItems, setVisibleItems] = (0, $PCemd$react.useState)(collection.size);
    let { size: size = 'M' } = (0, $PCemd$react.useContext)($4130f4e17f7081aa$var$InternalBreadcrumbsContext);
    let children = (0, $PCemd$react.useMemo)(()=>{
        let result = [];
        for (let key of collection.getKeys())result.push(collection.getItem(key));
        return result;
    }, [
        collection
    ]);
    let listRef = (0, $PCemd$react.useRef)(null);
    let updateOverflow = (0, $PCemd$react.useCallback)(()=>{
        let currListRef = listRef.current;
        if (!currListRef) {
            setVisibleItems(collection.size);
            return;
        }
        let container = currListRef.parentElement;
        if (!container) {
            setVisibleItems(collection.size);
            return;
        }
        let listItems = Array.from(currListRef.querySelectorAll('[data-hidden-breadcrumb]'));
        let folder = currListRef.querySelector('button');
        if (listItems.length <= 0) {
            setVisibleItems(collection.size);
            return;
        }
        let containerWidth = container.offsetWidth;
        let containerGap = parseInt(getComputedStyle(container).gap, 10);
        let folderGap = parseInt(getComputedStyle(folder).marginInlineStart, 10);
        let newVisibleItems = 0;
        let maxVisibleItems = $4130f4e17f7081aa$var$MAX_VISIBLE_ITEMS;
        let widths = [];
        let totalWidth = 0;
        for (let breadcrumb of listItems){
            let width = breadcrumb.offsetWidth + 1; // offsetWidth is rounded down
            widths.push(width);
            totalWidth += width;
        }
        // can we fit all the items without collapsing
        if (totalWidth <= containerWidth - collection.size * containerGap && collection.size <= $4130f4e17f7081aa$var$MAX_VISIBLE_ITEMS) {
            setVisibleItems(collection.size);
            return;
        }
        // we know there is always at least one item because of the listItems.length check up above
        let widthOfFirst = widths.shift();
        let availableWidth = containerWidth - widthOfFirst - folderGap - folder.offsetWidth - containerGap;
        maxVisibleItems -= 2; // account for the first item and folder
        for (let width of widths.reverse()){
            availableWidth -= width;
            if (availableWidth <= 0) break;
            availableWidth -= containerGap;
            newVisibleItems++;
        }
        setVisibleItems(Math.max($4130f4e17f7081aa$var$MIN_VISIBLE_ITEMS, Math.min(maxVisibleItems, newVisibleItems)));
    }, [
        collection.size,
        setVisibleItems
    ]);
    // making bad assumption that i can listen to containerRef when it's declared in the parent?
    (0, $PCemd$reactariaprivateutilsuseResizeObserver.useResizeObserver)({
        ref: containerRef,
        onResize: updateOverflow
    });
    (0, $PCemd$reactariaprivateutilsuseLayoutEffect.useLayoutEffect)(()=>{
        if (collection.size > 0) queueMicrotask(updateOverflow);
    }, [
        collection.size,
        updateOverflow
    ]);
    (0, $PCemd$react.useEffect)(()=>{
        // Recalculate visible tags when fonts are loaded.
        document.fonts?.ready.then(()=>updateOverflow());
    // eslint-disable-next-line react-hooks/exhaustive-deps
    }, []);
    let sliceIndex = collection.size - visibleItems;
    return /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsxs)((0, $PCemd$reactjsxruntime.Fragment), {
        children: [
            /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)($4130f4e17f7081aa$var$HiddenBreadcrumbs, {
                items: children,
                size: size,
                listRef: listRef
            }),
            visibleItems < collection.size && collection.size > 2 ? /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsxs)((0, $PCemd$reactjsxruntime.Fragment), {
                children: [
                    children[0].render?.(children[0]),
                    /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)($4130f4e17f7081aa$var$BreadcrumbMenu, {
                        items: children.slice(1, sliceIndex),
                        onAction: onAction
                    }),
                    children.slice(sliceIndex).map((node)=>/*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $PCemd$react.Fragment), {
                            children: node.render?.(node)
                        }, node.key))
                ]
            }) : /*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $PCemd$reactjsxruntime.Fragment), {
                children: children.map((node)=>/*#__PURE__*/ (0, $PCemd$reactjsxruntime.jsx)((0, $PCemd$react.Fragment), {
                        children: node.render?.(node)
                    }, node.key))
            })
        ]
    });
};
