// @react-spectrum/s2/dist/private/Button.mjs
import "./Button.css";
import {centerBaseline as $1f4b04be3f24aae3$export$9d7e2342a7e53afa} from "./CenterBaseline.mjs";
import {IconContext as $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed} from "../../icons/Icon.mjs";
import $7BBHC$intlStringsmjs from "./intlStrings.mjs";
import {pressScale as $10ea7662e51a285b$export$56e8cba416805d8d} from "./pressScale.mjs";
import {ProgressCircle as $41ddd91dbbf0c389$export$c79b9d6b4cc92af7} from "./ProgressCircle.mjs";
import {SkeletonContext as $5ad421ec19460c48$export$74e166679b1f49ee} from "../../icons/Skeleton.mjs";
import {Text as $8e847109a6ab556d$export$5f1af8db9871e1d6, TextContext as $8e847109a6ab556d$export$9afb8bc826b033ea} from "./Content.mjs";
import {useFocusableRef as $80ecb974df713a30$export$96a734597687c040} from "./useDOMRef.mjs";
import {useFormProps as $9b916426527cebe7$export$a6b5be5c6b451665} from "./Form.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $7BBHC$jsx, jsxs as $7BBHC$jsxs, Fragment as $7BBHC$Fragment} from "react/jsx-runtime";
import {Button as $7BBHC$Button} from "react-aria-components/Button";
import {Provider as $7BBHC$Provider} from "react-aria-components/slots";
import {createContext as $7BBHC$createContext, useState as $7BBHC$useState, useEffect as $7BBHC$useEffect, forwardRef as $7BBHC$forwardRef, useContext as $7BBHC$useContext} from "react";
import {Link as $7BBHC$Link} from "react-aria-components/Link";
import {OverlayTriggerStateContext as $7BBHC$OverlayTriggerStateContext} from "react-aria-components/Dialog";
import {useLocalizedStringFormatter as $7BBHC$useLocalizedStringFormatter} from "react-aria/useLocalizedStringFormatter";


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 
















const $067ea9f64ccd4e8e$export$24d547caef80ccd1 = /*#__PURE__*/ (0, $7BBHC$createContext)(null);
const $067ea9f64ccd4e8e$export$1c1a176f3b9e48d3 = /*#__PURE__*/ (0, $7BBHC$createContext)(null);
const $067ea9f64ccd4e8e$var$button = function anonymous(props, overrides) {
    let rules = " ";
    let minWidth = false;
    let position = false;
    let width = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "N") minWidth = true;
        if (p[1] === "_P") position = true;
        if (p[1] === "Z") width = true;
        rules += p[0];
    }
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    if (props.isStaticColor) rules += ' Oe131';
    else rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (props.staticColor === "white") rules += ' -Ovbhqd-gcwrnr131';
    else if (props.staticColor === "black") rules += ' -Ovbhqd-g4hn3s131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    rules += ' xd131';
    rules += ' _xa131';
    rules += ' xX0cczbc131';
    rules += ' _xX0cczba131';
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isStaticColor) {
        if (props.isDisabled) rules += ' pi131';
        else if (props.fillStyle === "outline") {
            if (props.variant === "genai") rules += ' px131';
            else if (props.variant === "premium") rules += ' px131';
            else rules += ' px131';
        } else if (props.fillStyle === "fill") {
            if (props.variant === "genai") rules += ' px131';
            else if (props.variant === "premium") rules += ' px131';
            else if (props.variant === "secondary") rules += ' px131';
            else if (props.variant === "primary") rules += ' ph131';
        }
    } else if (props.fillStyle === "outline") {
        if (props.isDisabled) rules += ' pp131';
        else if (props.variant === "genai") rules += ' px131';
        else if (props.variant === "premium") rules += ' px131';
        else {
            if (props.isPressed) rules += ' po131';
            else if (props.isFocusVisible) rules += ' po131';
            else if (props.isHovered) rules += ' po131';
            else rules += ' pt131';
        }
    } else if (props.fillStyle === "fill") {
        if (props.isDisabled) rules += ' pp131';
        else if (props.variant === "genai") rules += ' px131';
        else if (props.variant === "premium") rules += ' px131';
        else if (props.variant === "negative") rules += ' px131';
        else if (props.variant === "accent") rules += ' px131';
        else if (props.variant === "secondary") {
            if (props.isPressed) rules += ' po131';
            else if (props.isFocusVisible) rules += ' po131';
            else if (props.isHovered) rules += ' po131';
            else rules += ' pt131';
        } else if (props.variant === "primary") rules += ' pr131';
    }
    if (props.fillStyle === "outline") {
        if (props.isDisabled) rules += ' plc131';
        else rules += ' plb131';
    } else if (props.fillStyle === "fill") {
        if (props.isDisabled) rules += ' ple131';
        else rules += ' pla131';
    }
    rules += ' sd131';
    rules += ' ea131';
    rules += ' egLWx3bb131';
    rules += ' _oa131';
    rules += ' Sw131';
    rules += ' SgLWx3bd131';
    rules += ' Rw131';
    rules += ' RgLWx3bd131';
    if (props.size === "XL") {
        if (!minWidth) rules += ' Nl131';
    } else if (props.size === "L") {
        if (!minWidth) rules += ' Nj131';
    } else if (props.size === "S") {
        if (!minWidth) rules += ' Nf131';
    } else if (props.size === "XS") {
        if (!minWidth) rules += ' Ne131';
    } else if (!minWidth) rules += ' Nk131';
    rules += ' oh131';
    rules += ' nh131';
    rules += ' kh131';
    rules += ' jh131';
    rules += ' qc131';
    rules += ' qsd131';
    rules += ' -Qg41Lb-IAxCW0c131';
    rules += ' -Qg41Lb-gLWx3bIy131';
    rules += ' MAyFGYc131';
    rules += ' Td131';
    rules += ' Qd131';
    rules += ' -aRvJDd-Tu7Fubd131';
    if (!position) rules += ' _Pc131';
    rules += ' _Ca131';
    rules += ' _Wd131';
    rules += ' _4c131';
    if (!width) rules += ' ZJ131';
    rules += ' _Xa131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' _jf131';
    rules += ' _kJ8iPYd131';
    rules += ' hXMFGYc131';
    if (props.variant === "genai") rules += ' ma131';
    else if (props.variant === "premium") rules += ' ma131';
    else if (props.fillStyle === "outline") rules += ' mc131';
    else if (props.fillStyle === "fill") rules += ' ma131';
    if (props.variant === "genai") rules += ' la131';
    else if (props.variant === "premium") rules += ' la131';
    else if (props.fillStyle === "outline") rules += ' lc131';
    else if (props.fillStyle === "fill") rules += ' la131';
    if (props.isStaticColor) {
        if (props.isDisabled) rules += ' ig131';
        else if (props.variant === "secondary") {
            if (props.isPressed) rules += ' ih131';
            else if (props.isFocusVisible) rules += ' ih131';
            else if (props.isHovered) rules += ' ih131';
            else rules += ' ig131';
        } else if (props.variant === "primary") {
            if (props.isPressed) rules += ' ij131';
            else if (props.isFocusVisible) rules += ' ij131';
            else if (props.isHovered) rules += ' ij131';
            else rules += ' ii131';
        }
    } else if (props.isDisabled) rules += ' iq131';
    else if (props.variant === "secondary") {
        if (props.isPressed) rules += ' io131';
        else if (props.isFocusVisible) rules += ' io131';
        else if (props.isHovered) rules += ' io131';
        else rules += ' iq131';
    } else if (props.variant === "primary") {
        if (props.isPressed) rules += ' in131';
        else if (props.isFocusVisible) rules += ' in131';
        else if (props.isHovered) rules += ' in131';
        else rules += ' iv131';
    }
    if (props.isDisabled) rules += ' ilc131';
    else if (props.isHovered) rules += ' ild131';
    else rules += ' ila131';
    rules += ' gqMFGYc131';
    rules += ' -_8sjo0b-t5ZbAob131';
    rules += ' _yb131';
    rules += ' __ca131';
    if (props.size === "XL") rules += ' -M_-Mk131';
    else if (props.size === "L") rules += ' -M_-Mh131';
    else if (props.size === "S") rules += ' -M_-Mg131';
    else if (props.size === "XS") rules += ' -M_-Mf131';
    else rules += ' -M_-Mi131';
    if (props.variant === "genai") rules += ' -_k_-_ka131';
    else if (props.variant === "premium") rules += ' -_k_-_ka131';
    else if (props.fillStyle === "outline") rules += ' -_k_-_kc131';
    else if (props.fillStyle === "fill") rules += ' -_k_-_ka131';
    if (props.variant === "genai") rules += ' -h_-ha131';
    else if (props.variant === "premium") rules += ' -h_-ha131';
    else if (props.fillStyle === "outline") rules += ' -h_-hc131';
    else if (props.fillStyle === "fill") rules += ' -h_-ha131';
    if (props.isStaticColor) {
        if (props.fillStyle === "outline") {
            if (props.isDisabled) {
                if (props.variant === "genai") rules += ' -g_-gf131';
                else if (props.variant === "premium") rules += ' -g_-gf131';
                else rules += ' -g_-g9131';
            } else if (props.isFocusVisible) rules += ' -g_-gf131';
            else if (props.isPressed) rules += ' -g_-gf131';
            else if (props.isHovered) rules += ' -g_-gf131';
            else if (props.variant === "genai") rules += ' -g_-gf131';
            else if (props.variant === "premium") rules += ' -g_-gf131';
            else rules += ' -g_-g9131';
        } else if (props.fillStyle === "fill") {
            if (props.isDisabled) rules += ' -g_-gf131';
            else if (props.variant === "genai") rules += ' -g_-gf131';
            else if (props.variant === "premium") rules += ' -g_-gf131';
            else if (props.variant === "secondary") {
                if (props.isPressed) rules += ' -g_-gg131';
                else if (props.isFocusVisible) rules += ' -g_-gg131';
                else if (props.isHovered) rules += ' -g_-gg131';
                else rules += ' -g_-gf131';
            } else if (props.variant === "primary") {
                if (props.isPressed) rules += ' -g_-gj131';
                else if (props.isFocusVisible) rules += ' -g_-gj131';
                else if (props.isHovered) rules += ' -g_-gj131';
                else rules += ' -g_-gi131';
            }
        }
    } else if (props.fillStyle === "outline") {
        if (props.isDisabled) {
            if (props.variant === "genai") rules += ' -g_-gH131';
            else if (props.variant === "premium") rules += ' -g_-gH131';
            else rules += ' -g_-g9131';
        } else if (props.isFocusVisible) rules += ' -g_-gH131';
        else if (props.isPressed) rules += ' -g_-gH131';
        else if (props.isHovered) rules += ' -g_-gH131';
        else if (props.variant === "genai") rules += ' -g_-gH131';
        else if (props.variant === "premium") rules += ' -g_-gH131';
        else rules += ' -g_-g9131';
    } else if (props.fillStyle === "fill") {
        if (props.isDisabled) rules += ' -g_-gH131';
        else if (props.variant === "genai") rules += ' -g_-gH131';
        else if (props.variant === "premium") rules += ' -g_-gH131';
        else if (props.variant === "negative") {
            if (props.isFocusVisible) rules += ' -g_-gYvV9s131';
            else if (props.isPressed) rules += ' -g_-gYvV9s131';
            else if (props.isHovered) rules += ' -g_-gYvV9s131';
            else rules += ' -g_-g7aei8131';
        } else if (props.variant === "accent") {
            if (props.isFocusVisible) rules += ' -g_-gUgARdd131';
            else if (props.isPressed) rules += ' -g_-gUgARdd131';
            else if (props.isHovered) rules += ' -g_-gUgARdd131';
            else rules += ' -g_-g5qAiPc131';
        } else if (props.variant === "secondary") {
            if (props.isPressed) rules += ' -g_-gF131';
            else if (props.isFocusVisible) rules += ' -g_-gF131';
            else if (props.isHovered) rules += ' -g_-gF131';
            else rules += ' -g_-gH131';
        } else if (props.variant === "primary") {
            if (props.isPressed) rules += ' -g_-gu131';
            else if (props.isFocusVisible) rules += ' -g_-gu131';
            else if (props.isHovered) rules += ' -g_-gu131';
            else rules += ' -g_-g0131';
        }
    }
    if (props.fillStyle === "outline") rules += ' -g_-lgb131';
    else if (props.fillStyle === "fill") {
        if (props.isDisabled) rules += ' -g_-lgd131';
        else if (props.isHovered) rules += ' -g_-lge131';
        else rules += ' -g_-lgc131';
    }
    return rules;
};
// Put the gradient background on a separate element from the button to work around a Safari
// bug where transitions of custom properties cause layout flickering if any properties use rems. 🤣
// https://bugs.webkit.org/show_bug.cgi?id=285622
const $067ea9f64ccd4e8e$var$gradient = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' Wr131';
    rules += ' _lr131';
    rules += ' _Ar131';
    rules += ' _zr131';
    rules += ' _9a131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' oi131';
    rules += ' ni131';
    rules += ' ki131';
    rules += ' ji131';
    if (props.isDisabled) rules += ' _ic131';
    else if (props.variant === "genai") {
        if (props.isFocusVisible) {
            rules += ' _ia131';
            rules += ' -yBNtrc-37ZLpd131';
            rules += ' -zBNtrc-plLfSc131';
            rules += ' -ABNtrc-z2bZib131';
            rules += ' -ACNtrc-tQ56Od131';
        } else if (props.isPressed) {
            rules += ' _ia131';
            rules += ' -yBNtrc-37ZLpd131';
            rules += ' -zBNtrc-plLfSc131';
            rules += ' -ABNtrc-z2bZib131';
            rules += ' -ACNtrc-tQ56Od131';
        } else if (props.isHovered) {
            rules += ' _ia131';
            rules += ' -yBNtrc-37ZLpd131';
            rules += ' -zBNtrc-plLfSc131';
            rules += ' -ABNtrc-z2bZib131';
            rules += ' -ACNtrc-tQ56Od131';
        } else {
            rules += ' _ia131';
            rules += ' -yBNtrc-Nitqwc131';
            rules += ' -zBNtrc-ugAmz131';
            rules += ' -ABNtrc-j7ulPc131';
            rules += ' -ACNtrc-tQ56Od131';
        }
    } else if (props.variant === "premium") {
        if (props.isFocusVisible) {
            rules += ' _ib131';
            rules += ' -yBNtrc-1gYRFc131';
            rules += ' -zBNtrc-z2bZib131';
            rules += ' -ABNtrc-BorWLb131';
            rules += ' -ACNtrc-tQ56Od131';
        } else if (props.isPressed) {
            rules += ' _ib131';
            rules += ' -yBNtrc-1gYRFc131';
            rules += ' -zBNtrc-z2bZib131';
            rules += ' -ABNtrc-BorWLb131';
            rules += ' -ACNtrc-tQ56Od131';
        } else if (props.isHovered) {
            rules += ' _ib131';
            rules += ' -yBNtrc-1gYRFc131';
            rules += ' -zBNtrc-z2bZib131';
            rules += ' -ABNtrc-BorWLb131';
            rules += ' -ACNtrc-tQ56Od131';
        } else {
            rules += ' _ib131';
            rules += ' -yBNtrc-X7W3Sb131';
            rules += ' -zBNtrc-j7ulPc131';
            rules += ' -ABNtrc-fHZGqb131';
            rules += ' -ACNtrc-tQ56Od131';
        }
    }
    rules += ' _ilc131';
    if (props.variant === "genai") rules += ' _pb131';
    else if (props.variant === "premium") rules += ' _pb131';
    return rules;
};
function $067ea9f64ccd4e8e$export$edc9ced5c3e5ff00(isPending) {
    let [isProgressVisible, setIsProgressVisible] = (0, $7BBHC$useState)(false);
    (0, $7BBHC$useEffect)(()=>{
        let timeout;
        if (isPending) timeout = setTimeout(()=>{
            setIsProgressVisible(true);
        }, 1000);
        else setIsProgressVisible(false);
        return ()=>{
            clearTimeout(timeout);
        };
    }, [
        isPending
    ]);
    return {
        isProgressVisible: isProgressVisible
    };
}
const $067ea9f64ccd4e8e$export$353f5b6fc5456de1 = /*#__PURE__*/ (0, $7BBHC$forwardRef)(function Button(props1, ref) {
    [props1, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props1, ref, $067ea9f64ccd4e8e$export$24d547caef80ccd1);
    props1 = (0, $9b916426527cebe7$export$a6b5be5c6b451665)(props1);
    let stringFormatter = (0, $7BBHC$useLocalizedStringFormatter)((0, ($parcel$interopDefault($7BBHC$intlStringsmjs))), '@react-spectrum/s2');
    let { isPending: isPending = false, variant: variant = 'primary', fillStyle: fillStyle = 'fill', size: size = 'M', staticColor: staticColor } = props1;
    let domRef = (0, $80ecb974df713a30$export$96a734597687c040)(ref);
    let overlayTriggerState = (0, $7BBHC$useContext)((0, $7BBHC$OverlayTriggerStateContext));
    let { isProgressVisible: isProgressVisible } = $067ea9f64ccd4e8e$export$edc9ced5c3e5ff00(isPending);
    return /*#__PURE__*/ (0, $7BBHC$jsx)((0, $7BBHC$Button), {
        ...props1,
        ref: domRef,
        style: (0, $10ea7662e51a285b$export$56e8cba416805d8d)(domRef, props1.UNSAFE_style),
        className: (renderProps)=>(props1.UNSAFE_className || '') + $067ea9f64ccd4e8e$var$button({
                ...renderProps,
                // Retain hover styles when an overlay is open.
                isHovered: renderProps.isHovered || overlayTriggerState?.isOpen || false,
                isDisabled: renderProps.isDisabled || isProgressVisible,
                variant: variant,
                fillStyle: fillStyle,
                size: size,
                staticColor: staticColor,
                isStaticColor: !!staticColor
            }, props1.styles),
        children: (renderProps)=>/*#__PURE__*/ (0, $7BBHC$jsxs)((0, $7BBHC$Fragment), {
                children: [
                    variant === 'genai' || variant === 'premium' ? /*#__PURE__*/ (0, $7BBHC$jsx)("span", {
                        className: $067ea9f64ccd4e8e$var$gradient({
                            ...renderProps,
                            // Retain hover styles when an overlay is open.
                            isHovered: renderProps.isHovered || overlayTriggerState?.isOpen || false,
                            isDisabled: renderProps.isDisabled || isProgressVisible,
                            variant: variant
                        })
                    }) : null,
                    /*#__PURE__*/ (0, $7BBHC$jsxs)((0, $7BBHC$Provider), {
                        values: [
                            [
                                (0, $5ad421ec19460c48$export$74e166679b1f49ee),
                                null
                            ],
                            [
                                (0, $8e847109a6ab556d$export$9afb8bc826b033ea),
                                {
                                    styles: function anonymous(props) {
                                        let rules = " ";
                                        rules += ' TAzEcG131';
                                        rules += ' QAzEcG131';
                                        rules += ' _Jb131';
                                        if (props.isProgressVisible) rules += ' _6b131';
                                        return rules;
                                    }({
                                        isProgressVisible: isProgressVisible
                                    }),
                                    // @ts-ignore data-attributes allowed on all JSX elements, but adding to DOMProps has been problematic in the past
                                    'data-rsp-slot': 'text'
                                }
                            ],
                            [
                                (0, $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed),
                                {
                                    render: (0, $1f4b04be3f24aae3$export$9d7e2342a7e53afa)({
                                        slot: 'icon',
                                        styles: " _Ja131"
                                    }),
                                    styles: function anonymous(props) {
                                        let rules = " ";
                                        rules += ' ZuUmpre131';
                                        rules += ' FuUmpre131';
                                        rules += ' I67logd131';
                                        rules += ' _va131';
                                        if (props.isProgressVisible) rules += ' _6b131';
                                        return rules;
                                    }({
                                        isProgressVisible: isProgressVisible
                                    })
                                }
                            ]
                        ],
                        children: [
                            typeof props1.children === 'string' ? /*#__PURE__*/ (0, $7BBHC$jsx)((0, $8e847109a6ab556d$export$5f1af8db9871e1d6), {
                                children: props1.children
                            }) : props1.children,
                            isPending && /*#__PURE__*/ (0, $7BBHC$jsx)("div", {
                                className: function anonymous(props) {
                                    let rules = " ";
                                    rules += ' _Pa131';
                                    rules += ' WF131';
                                    rules += ' _EF131';
                                    rules += ' _Zc131';
                                    if (props.isProgressVisible) rules += ' _6c131';
                                    else rules += ' _6b131';
                                    return rules;
                                }({
                                    isProgressVisible: isProgressVisible,
                                    isPending: isPending
                                }),
                                children: /*#__PURE__*/ (0, $7BBHC$jsx)((0, $41ddd91dbbf0c389$export$c79b9d6b4cc92af7), {
                                    isIndeterminate: true,
                                    "aria-label": stringFormatter.format('button.pending'),
                                    size: "S",
                                    staticColor: staticColor,
                                    styles: function anonymous(props) {
                                        let rules = " ";
                                        if (props.size === "XL") rules += ' Zo131';
                                        else if (props.size === "L") rules += ' Zm131';
                                        else if (props.size === "M") rules += ' Zl131';
                                        else if (props.size === "S") rules += ' Zk131';
                                        if (props.size === "XL") rules += ' Fp131';
                                        else if (props.size === "L") rules += ' Fn131';
                                        else if (props.size === "M") rules += ' Fm131';
                                        else if (props.size === "S") rules += ' Fl131';
                                        return rules;
                                    }({
                                        size: size
                                    })
                                })
                            })
                        ]
                    })
                ]
            })
    });
});
const $067ea9f64ccd4e8e$export$29d11c0fe2fc51d8 = /*#__PURE__*/ (0, $7BBHC$forwardRef)(function LinkButton(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $067ea9f64ccd4e8e$export$1c1a176f3b9e48d3);
    props = (0, $9b916426527cebe7$export$a6b5be5c6b451665)(props);
    let domRef = (0, $80ecb974df713a30$export$96a734597687c040)(ref);
    let overlayTriggerState = (0, $7BBHC$useContext)((0, $7BBHC$OverlayTriggerStateContext));
    let { fillStyle: fillStyle = 'fill', size: size = 'M', variant: variant = 'primary', staticColor: staticColor, styles: styles, children: children } = props;
    return /*#__PURE__*/ (0, $7BBHC$jsx)((0, $7BBHC$Link), {
        ...props,
        ref: domRef,
        style: (0, $10ea7662e51a285b$export$56e8cba416805d8d)(domRef, props.UNSAFE_style),
        className: (renderProps)=>(props.UNSAFE_className || '') + $067ea9f64ccd4e8e$var$button({
                ...renderProps,
                // Retain hover styles when an overlay is open.
                isHovered: renderProps.isHovered || overlayTriggerState?.isOpen || false,
                variant: variant,
                fillStyle: fillStyle,
                size: size,
                staticColor: staticColor,
                isStaticColor: !!staticColor,
                isPending: false
            }, styles),
        children: (renderProps)=>/*#__PURE__*/ (0, $7BBHC$jsxs)((0, $7BBHC$Fragment), {
                children: [
                    variant === 'genai' || variant === 'premium' ? /*#__PURE__*/ (0, $7BBHC$jsx)("span", {
                        className: $067ea9f64ccd4e8e$var$gradient({
                            ...renderProps,
                            // Retain hover styles when an overlay is open.
                            isHovered: renderProps.isHovered || overlayTriggerState?.isOpen || false,
                            variant: variant
                        })
                    }) : null,
                    /*#__PURE__*/ (0, $7BBHC$jsx)((0, $7BBHC$Provider), {
                        values: [
                            [
                                (0, $5ad421ec19460c48$export$74e166679b1f49ee),
                                null
                            ],
                            [
                                (0, $8e847109a6ab556d$export$9afb8bc826b033ea),
                                {
                                    styles: " TAzEcG131 QAzEcG131 _Jb131",
                                    // @ts-ignore data-attributes allowed on all JSX elements, but adding to DOMProps has been problematic in the past
                                    'data-rsp-slot': 'text'
                                }
                            ],
                            [
                                (0, $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed),
                                {
                                    render: (0, $1f4b04be3f24aae3$export$9d7e2342a7e53afa)({
                                        slot: 'icon',
                                        styles: " _Ja131"
                                    }),
                                    styles: " ZuUmpre131 FuUmpre131 I67logd131 _va131"
                                }
                            ]
                        ],
                        children: typeof children === 'string' ? /*#__PURE__*/ (0, $7BBHC$jsx)((0, $8e847109a6ab556d$export$5f1af8db9871e1d6), {
                            children: children
                        }) : children
                    })
                ]
            })
    });
});


export {$067ea9f64ccd4e8e$export$24d547caef80ccd1 as ButtonContext, $067ea9f64ccd4e8e$export$1c1a176f3b9e48d3 as LinkButtonContext, $067ea9f64ccd4e8e$export$edc9ced5c3e5ff00 as usePendingState, $067ea9f64ccd4e8e$export$353f5b6fc5456de1 as Button, $067ea9f64ccd4e8e$export$29d11c0fe2fc51d8 as LinkButton};
