// @react-spectrum/s2/dist/private/Calendar.mjs
import "./Calendar.css";
import {ActionButton as $da878a05ab4a403e$export$cfc7921d29ef7b80} from "./ActionButton.mjs";
import $5c3fed25abf344cc$export$2e2bcd8739ae039 from "../../icons/ChevronLeft.mjs";
import $04f4b15c8c66bb2a$export$2e2bcd8739ae039 from "../../icons/ChevronRight.mjs";
import {Header as $8e847109a6ab556d$export$8b251419efc915eb, HeaderContext as $8e847109a6ab556d$export$e0e4026c12a8bdbb, HeadingContext as $8e847109a6ab556d$export$d688439359537581} from "./Content.mjs";
import {helpTextStyles as $9afd80978c252040$export$13b1fd5431595bf6} from "./Field.mjs";
import $gwkOZ$intlStringsmjs from "./intlStrings.mjs";
import {pressScale as $10ea7662e51a285b$export$56e8cba416805d8d} from "./pressScale.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $gwkOZ$jsx, jsxs as $gwkOZ$jsxs, Fragment as $gwkOZ$Fragment} from "react/jsx-runtime";
import {Calendar as $gwkOZ$Calendar, CalendarHeading as $gwkOZ$CalendarHeading, CalendarContext as $gwkOZ$CalendarContext, CalendarGrid as $gwkOZ$CalendarGrid, CalendarGridHeader as $gwkOZ$CalendarGridHeader, CalendarGridBody as $gwkOZ$CalendarGridBody, CalendarHeaderCell as $gwkOZ$CalendarHeaderCell, CalendarStateContext as $gwkOZ$CalendarStateContext, CalendarCell as $gwkOZ$CalendarCell} from "react-aria-components/Calendar";
import {getDayOfWeek as $gwkOZ$getDayOfWeek, startOfMonth as $gwkOZ$startOfMonth} from "@internationalized/date";
import {Provider as $gwkOZ$Provider, useSlottedContext as $gwkOZ$useSlottedContext} from "react-aria-components/slots";
import {RangeCalendarContext as $gwkOZ$RangeCalendarContext, RangeCalendarStateContext as $gwkOZ$RangeCalendarStateContext} from "react-aria-components/RangeCalendar";
import {createContext as $gwkOZ$createContext, forwardRef as $gwkOZ$forwardRef, useContext as $gwkOZ$useContext, useRef as $gwkOZ$useRef, useMemo as $gwkOZ$useMemo} from "react";
import {Text as $gwkOZ$Text} from "react-aria-components/Text";
import {useLocale as $gwkOZ$useLocale} from "react-aria/I18nProvider";
import {useLocalizedStringFormatter as $gwkOZ$useLocalizedStringFormatter} from "react-aria/useLocalizedStringFormatter";


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 
















const $f06f4d5597ad704c$export$3b805cea1f178355 = /*#__PURE__*/ (0, $gwkOZ$createContext)(null);
const $f06f4d5597ad704c$var$calendarStyles = function anonymous(props, overrides) {
    let rules = " ";
    let width = false;
    let maxWidth = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "Z") width = true;
        if (p[1] === "L") maxWidth = true;
        rules += p[0];
    }
    rules += ' sd131';
    if (props.isMultiMonth) rules += ' -_4DHzJc-e4mUs131';
    else rules += ' -_4DHzJc-0u6p5b131';
    rules += ' _ta131';
    rules += ' Uj131';
    rules += ' qj131';
    rules += ' __ca131';
    rules += ' -uq4sQ-Sp131';
    rules += ' -LnVzy-ZC131';
    if (props.isMultiMonth) rules += ' -cYDWLb-ZFFQGNb131';
    else rules += ' -cYDWLb-ZnqRBTd131';
    if (props.isMultiMonth) {
        if (!width) rules += ' ZJ131';
    } else if (!width) rules += ' ZRqDA2c131';
    if (props.isMultiMonth) {
        if (!maxWidth) rules += ' Le4mUs131';
    } else if (!maxWidth) rules += ' La131';
    return rules;
};
const $f06f4d5597ad704c$var$headerStyles = " sd131 eb131 _Ce131 qj131";
const $f06f4d5597ad704c$var$headingStyles = " sd131 eb131 _Ce131 Jy131 Gy131 Iy131 Hy131 _ub131 Za131";
const $f06f4d5597ad704c$var$titleStyles = " uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-e131 vx131 xd131 _xa131 xX0cczbc131 _xX0cczba131 xfd131 _xfa131 xfX0cczbc131 _xfX0cczba131 _Fd131 _FnuYUweb131 po131 _Wa131 _ub131 _va131 Jy131 Gy131";
const $f06f4d5597ad704c$var$headerCellStyles = " uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-c131 vx131 xd131 _xa131 xX0cczbc131 _xX0cczba131 xfd131 _xfa131 xfX0cczbc131 _xfX0cczba131 _Fd131 _FnuYUweb131 po131 ri131 _Wa131 Sp131 Sbd131 Rp131 Rkd131 Qe131";
const $f06f4d5597ad704c$var$cellStyles = function anonymous(props) {
    let rules = " ";
    rules += ' _Le131';
    rules += ' _ob131';
    if (props.isFirstChild) rules += ' Sd131';
    else rules += ' Sp131';
    if (props.isLastChild) rules += ' Rd131';
    else rules += ' Rp131';
    if (props.isFirstWeek) rules += ' Td131';
    else rules += ' Tj131';
    if (props.isLastWeek) rules += ' Qd131';
    else rules += ' Qj131';
    rules += ' _Pc131';
    if (props.isOutsideMonth) rules += ' sk131';
    else rules += ' sd131';
    rules += ' eb131';
    rules += ' _Ca131';
    rules += ' __ca131';
    rules += ' Z0q26ub131';
    rules += ' F0q26ub131';
    return rules;
};
const $f06f4d5597ad704c$var$cellInnerStyles = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    if (props.isSelectionEnd) rules += ' _Kd131';
    else if (props.isSelectionStart) rules += ' _Kd131';
    else if (props.isSelected) {
        if (props.selectionMode === "range") rules += ' _Kb131';
        else if (props.selectionMode === "single") rules += ' _Kd131';
    } else rules += ' _Kb131';
    rules += ' Yd131';
    rules += ' Yle131';
    rules += ' Xb131';
    rules += ' Xlb131';
    rules += ' _2b131';
    rules += ' _2lb131';
    rules += ' _Pc131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-c131';
    rules += ' vx131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fb131';
    rules += ' _FnuYUwec131';
    if (props.isDisabled) rules += ' pp131';
    else if (props.isSelectionEnd) rules += ' px131';
    else if (props.isSelectionStart) rules += ' px131';
    else if (props.isSelected) {
        if (props.selectionMode === "range") rules += ' pt131';
        else rules += ' px131';
    } else rules += ' pt131';
    if (props.isDisabled) rules += ' plc131';
    else if (props.isSelectionEnd) rules += ' ple131';
    else if (props.isSelectionStart) rules += ' ple131';
    else if (props.isSelected) rules += ' ple131';
    else rules += ' plb131';
    rules += ' ri131';
    rules += ' Za131';
    rules += ' Fb131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _Ca131';
    rules += ' _yb131';
    if (props.isSelectionEnd) {
        if (props.isInvalid) {
            if (props.isFocusVisible) {
                if (props.isUnavailable) rules += ' g7aei8131';
                else rules += ' gYvV9s131';
            } else if (props.isPressed) rules += ' gYvV9s131';
            else if (props.isHovered) {
                if (props.isUnavailable) rules += ' g7aei8131';
                else rules += ' gYvV9s131';
            } else rules += ' g7aei8131';
        } else if (props.isDisabled) rules += ' g9131';
        else if (props.isFocusVisible) rules += ' gUgARdd131';
        else if (props.isPressed) rules += ' gUgARdd131';
        else if (props.isHovered) rules += ' gUgARdd131';
        else rules += ' g5qAiPc131';
    } else if (props.isSelectionStart) {
        if (props.isInvalid) {
            if (props.isFocusVisible) {
                if (props.isUnavailable) rules += ' g7aei8131';
                else rules += ' gYvV9s131';
            } else if (props.isPressed) rules += ' gYvV9s131';
            else if (props.isHovered) {
                if (props.isUnavailable) rules += ' g7aei8131';
                else rules += ' gYvV9s131';
            } else rules += ' g7aei8131';
        } else if (props.isDisabled) rules += ' g9131';
        else if (props.isFocusVisible) rules += ' gUgARdd131';
        else if (props.isPressed) rules += ' gUgARdd131';
        else if (props.isHovered) rules += ' gUgARdd131';
        else rules += ' g5qAiPc131';
    } else if (props.isSelected) {
        if (props.selectionMode === "range") {
            if (props.isHovered) {
                if (props.isInvalid) {
                    if (props.isUnavailable) rules += ' g9131';
                    else rules += ' gJj9Tpd131';
                } else rules += ' gNAacne131';
            }
        } else if (props.selectionMode === "single") {
            if (props.isInvalid) {
                if (props.isFocusVisible) rules += ' gYvV9s131';
                else if (props.isPressed) rules += ' gYvV9s131';
                else if (props.isHovered) rules += ' gYvV9s131';
                else rules += ' g7aei8131';
            } else if (props.isDisabled) rules += ' g9131';
            else if (props.isFocusVisible) rules += ' gUgARdd131';
            else if (props.isPressed) rules += ' gUgARdd131';
            else if (props.isHovered) rules += ' gUgARdd131';
            else rules += ' g5qAiPc131';
        }
    } else if (props.isDisabled) rules += ' g9131';
    else if (props.isPressed) rules += ' gH131';
    else if (props.isHovered) {
        if (props.isUnavailable) rules += ' g9131';
        else rules += ' gH131';
    } else rules += ' g9131';
    if (props.isUnavailable) rules += ' gl9131';
    else if (props.isSelectionEnd) rules += ' gle131';
    else if (props.isSelectionStart) rules += ' gle131';
    else if (props.isSelected) {
        if (props.selectionMode === "range") {
            if (props.isHovered) rules += ' gle131';
        } else if (props.selectionMode === "single") rules += ' gle131';
    } else if (props.isHovered) rules += ' gle131';
    else rules += ' gl9131';
    return rules;
};
const $f06f4d5597ad704c$var$todayStyles = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' _lakDin131';
    rules += ' _EF131';
    rules += ' _ZRWeLS131';
    rules += ' ZS16uu131';
    rules += ' Fe131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    rules += ' g5ZbAob131';
    if (props.isToday) rules += ' sb131';
    else rules += ' sk131';
    return rules;
};
const $f06f4d5597ad704c$var$unavailableStyles = " _Pa131 WzErnCb131 _EjhYzGc131 _RjhYzGc131 FlAZvq131 _ZN7ydnb131 og131 ng131 kg131 jg131 g5ZbAob131";
const $f06f4d5597ad704c$var$selectionBackgroundStyles = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' _9a131';
    rules += ' Wr131';
    if (props.isPreviousDayNotSelected) rules += ' _Ar131';
    else if (props.isSelectionStart) rules += ' _Ar131';
    else if (props.isFirstDayInWeek) rules += ' _Ar131';
    else rules += ' _Al131';
    if (props.isNextDayNotSelected) rules += ' _zr131';
    else if (props.isSelectionEnd) rules += ' _zr131';
    else if (props.isLastDayInWeek) rules += ' _zr131';
    else rules += ' _zl131';
    rules += ' _lr131';
    if (props.isPreviousDayNotSelected) rules += ' og131';
    else if (props.isSelectionStart) rules += ' og131';
    else if (props.isFirstDayInWeek) rules += ' og131';
    else rules += ' od131';
    if (props.isPreviousDayNotSelected) rules += ' kg131';
    else if (props.isSelectionStart) rules += ' kg131';
    else if (props.isFirstDayInWeek) rules += ' kg131';
    else rules += ' kd131';
    if (props.isNextDayNotSelected) rules += ' ng131';
    else if (props.isSelectionEnd) rules += ' ng131';
    else if (props.isLastDayInWeek) rules += ' ng131';
    else rules += ' nd131';
    if (props.isNextDayNotSelected) rules += ' jg131';
    else if (props.isSelectionEnd) rules += ' jg131';
    else if (props.isLastDayInWeek) rules += ' jg131';
    else rules += ' jd131';
    if (props.isInvalid) rules += ' gZbQnx131';
    else rules += ' gG131';
    rules += ' gle131';
    rules += ' _yb131';
    return rules;
};
const $f06f4d5597ad704c$var$selectionBorderStyles = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' _9c131';
    rules += ' Wr131';
    if (props.isPreviousDayNotSelected) rules += ' _Ar131';
    else if (props.isSelectionStart) rules += ' _Ar131';
    else if (props.isFirstDayInWeek) rules += ' _Ar131';
    else rules += ' _Al131';
    if (props.isNextDayNotSelected) rules += ' _zr131';
    else if (props.isSelectionEnd) rules += ' _zr131';
    else if (props.isLastDayInWeek) rules += ' _zr131';
    else rules += ' _zl131';
    rules += ' _lr131';
    if (props.isPreviousDayNotSelected) rules += ' mb131';
    else if (props.isSelectionStart) rules += ' mb131';
    else if (props.isFirstDayInWeek) rules += ' mb131';
    else rules += ' ma131';
    rules += ' _kb131';
    if (props.isNextDayNotSelected) rules += ' lb131';
    else if (props.isSelectionEnd) rules += ' lb131';
    else if (props.isLastDayInWeek) rules += ' lb131';
    else rules += ' la131';
    rules += ' hb131';
    rules += ' _jf131';
    if (props.isInvalid) rules += ' ip131';
    else rules += ' ix131';
    rules += ' ilb131';
    if (props.isPreviousDayNotSelected) rules += ' og131';
    else if (props.isSelectionStart) rules += ' og131';
    else if (props.isFirstDayInWeek) rules += ' og131';
    else rules += ' od131';
    if (props.isPreviousDayNotSelected) rules += ' kg131';
    else if (props.isSelectionStart) rules += ' kg131';
    else if (props.isFirstDayInWeek) rules += ' kg131';
    else rules += ' kd131';
    if (props.isNextDayNotSelected) rules += ' ng131';
    else if (props.isSelectionEnd) rules += ' ng131';
    else if (props.isLastDayInWeek) rules += ' ng131';
    else rules += ' nd131';
    if (props.isNextDayNotSelected) rules += ' jg131';
    else if (props.isSelectionEnd) rules += ' jg131';
    else if (props.isLastDayInWeek) rules += ' jg131';
    else rules += ' jd131';
    return rules;
};
const $f06f4d5597ad704c$export$e1aef45b828286de = /*#__PURE__*/ (0, $gwkOZ$forwardRef)(function Calendar(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $f06f4d5597ad704c$export$3b805cea1f178355);
    let { visibleMonths: visibleMonths = 1, errorMessage: errorMessage, UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className, styles: styles, ...otherProps } = props;
    let stringFormatter = (0, $gwkOZ$useLocalizedStringFormatter)((0, ($parcel$interopDefault($gwkOZ$intlStringsmjs))), '@react-spectrum/s2');
    let isMultiMonth = visibleMonths > 1;
    return /*#__PURE__*/ (0, $gwkOZ$jsx)((0, $gwkOZ$Calendar), {
        ...otherProps,
        ref: ref,
        visibleDuration: {
            months: visibleMonths
        },
        style: UNSAFE_style,
        className: (UNSAFE_className || '') + $f06f4d5597ad704c$var$calendarStyles({
            isMultiMonth: isMultiMonth
        }, styles),
        children: ({ isInvalid: isInvalid, isDisabled: isDisabled })=>{
            return /*#__PURE__*/ (0, $gwkOZ$jsxs)((0, $gwkOZ$Fragment), {
                children: [
                    /*#__PURE__*/ (0, $gwkOZ$jsx)((0, $gwkOZ$Provider), {
                        values: [
                            [
                                (0, $8e847109a6ab556d$export$e0e4026c12a8bdbb),
                                null
                            ],
                            [
                                (0, $8e847109a6ab556d$export$d688439359537581),
                                null
                            ]
                        ],
                        children: /*#__PURE__*/ (0, $gwkOZ$jsx)($f06f4d5597ad704c$export$9c75275274db0b56, {
                            visibleMonths: visibleMonths
                        })
                    }),
                    /*#__PURE__*/ (0, $gwkOZ$jsx)("div", {
                        className: " sd131 _tc131 Uj131 qj131 Za131 ed131",
                        children: Array.from({
                            length: visibleMonths
                        }).map((_, i)=>/*#__PURE__*/ (0, $gwkOZ$jsx)($f06f4d5597ad704c$export$5bd780d491cfc46c, {
                                months: i
                            }, i))
                    }),
                    isInvalid && /*#__PURE__*/ (0, $gwkOZ$jsx)((0, $gwkOZ$Text), {
                        slot: "errorMessage",
                        className: (0, $9afd80978c252040$export$13b1fd5431595bf6)({
                            isInvalid: isInvalid,
                            isDisabled: isDisabled,
                            size: 'M'
                        }),
                        children: errorMessage || stringFormatter.format('calendar.invalidSelection', {
                            selectedCount: 1
                        })
                    })
                ]
            });
        }
    });
});
const $f06f4d5597ad704c$export$9c75275274db0b56 = ({ visibleMonths: visibleMonths })=>{
    return /*#__PURE__*/ (0, $gwkOZ$jsx)((0, $8e847109a6ab556d$export$8b251419efc915eb), {
        styles: $f06f4d5597ad704c$var$headerStyles,
        children: Array.from({
            length: visibleMonths
        }).map((_, i)=>/*#__PURE__*/ (0, $gwkOZ$jsxs)("div", {
                className: $f06f4d5597ad704c$var$headingStyles,
                children: [
                    i === 0 && /*#__PURE__*/ (0, $gwkOZ$jsx)($f06f4d5597ad704c$export$adb29da0ac001538, {
                        slot: "previous",
                        children: /*#__PURE__*/ (0, $gwkOZ$jsx)((0, $5c3fed25abf344cc$export$2e2bcd8739ae039), {})
                    }),
                    /*#__PURE__*/ (0, $gwkOZ$jsx)((0, $gwkOZ$CalendarHeading), {
                        offset: {
                            months: i
                        },
                        className: $f06f4d5597ad704c$var$titleStyles
                    }),
                    i === visibleMonths - 1 && /*#__PURE__*/ (0, $gwkOZ$jsx)($f06f4d5597ad704c$export$adb29da0ac001538, {
                        slot: "next",
                        children: /*#__PURE__*/ (0, $gwkOZ$jsx)((0, $04f4b15c8c66bb2a$export$2e2bcd8739ae039), {})
                    })
                ]
            }, i))
    });
};
const $f06f4d5597ad704c$export$5bd780d491cfc46c = (props)=>{
    let rangeCalendarProps = (0, $gwkOZ$useSlottedContext)((0, $gwkOZ$RangeCalendarContext));
    let calendarProps = (0, $gwkOZ$useSlottedContext)((0, $gwkOZ$CalendarContext));
    let firstDayOfWeek = rangeCalendarProps?.firstDayOfWeek ?? calendarProps?.firstDayOfWeek;
    // use isolation to start a new stacking context so that we can use zIndex -1 for the selection span.
    return /*#__PURE__*/ (0, $gwkOZ$jsxs)((0, $gwkOZ$CalendarGrid), {
        className: " __na131 __oe131 _Bb131",
        offset: {
            months: props.months
        },
        children: [
            /*#__PURE__*/ (0, $gwkOZ$jsx)((0, $gwkOZ$CalendarGridHeader), {
                className: "",
                children: (day)=>/*#__PURE__*/ (0, $gwkOZ$jsx)($f06f4d5597ad704c$var$CalendarHeaderCell, {
                        children: day
                    })
            }),
            /*#__PURE__*/ (0, $gwkOZ$jsx)((0, $gwkOZ$CalendarGridBody), {
                className: "",
                children: (date)=>/*#__PURE__*/ (0, $gwkOZ$jsx)($f06f4d5597ad704c$var$CalendarCell, {
                        date: date,
                        firstDayOfWeek: firstDayOfWeek
                    })
            })
        ]
    });
};
const $f06f4d5597ad704c$export$adb29da0ac001538 = (props1)=>{
    let { direction: direction } = (0, $gwkOZ$useLocale)();
    return /*#__PURE__*/ (0, $gwkOZ$jsx)("div", {
        className: function anonymous(props) {
            let rules = " ";
            if (props.direction === "rtl") {
                rules += ' -Z4Pn4c-a131';
                rules += ' __Ha131';
            }
            if (props.direction === "rtl") {
                rules += ' -_04Pn4c-a131';
                rules += ' __Ha131';
            }
            return rules;
        }({
            direction: direction
        }),
        children: /*#__PURE__*/ (0, $gwkOZ$jsx)((0, $da878a05ab4a403e$export$cfc7921d29ef7b80), {
            ...props1,
            isQuiet: true,
            children: props1.children
        })
    });
};
const $f06f4d5597ad704c$var$CalendarHeaderCell = (props)=>{
    return /*#__PURE__*/ (0, $gwkOZ$jsx)((0, $gwkOZ$CalendarHeaderCell), {
        className: $f06f4d5597ad704c$var$headerCellStyles,
        children: props.children
    });
};
const $f06f4d5597ad704c$var$CalendarCell = (props)=>{
    let { locale: locale } = (0, $gwkOZ$useLocale)();
    let firstDayOfWeek = props.firstDayOfWeek;
    // Calculate the day and week index based on the date.
    let { dayIndex: dayIndex, weekIndex: weekIndex, lastWeekIndex: lastWeekIndex } = $f06f4d5597ad704c$var$useWeekAndDayIndices(props.date, locale, firstDayOfWeek);
    let calendarStateContext = (0, $gwkOZ$useContext)((0, $gwkOZ$CalendarStateContext));
    let rangeCalendarStateContext = (0, $gwkOZ$useContext)((0, $gwkOZ$RangeCalendarStateContext));
    let state = calendarStateContext ?? rangeCalendarStateContext;
    let isFirstWeek = weekIndex === 0;
    let isLastWeek = weekIndex === lastWeekIndex;
    let isFirstChild = dayIndex === 0;
    let isLastChild = dayIndex === 6;
    return /*#__PURE__*/ (0, $gwkOZ$jsx)((0, $gwkOZ$CalendarCell), {
        date: props.date,
        className: (renderProps)=>$f06f4d5597ad704c$var$cellStyles({
                ...renderProps,
                isFirstChild: isFirstChild,
                isLastChild: isLastChild,
                isFirstWeek: isFirstWeek,
                isLastWeek: isLastWeek
            }),
        children: (renderProps)=>/*#__PURE__*/ (0, $gwkOZ$jsx)($f06f4d5597ad704c$var$CalendarCellInner, {
                ...props,
                weekIndex: weekIndex,
                dayIndex: dayIndex,
                state: state,
                isRangeSelection: !!rangeCalendarStateContext,
                renderProps: renderProps
            })
    });
};
const $f06f4d5597ad704c$var$CalendarCellInner = (props)=>{
    let { dayIndex: dayIndex, date: date, renderProps: renderProps, state: state, isRangeSelection: isRangeSelection } = props;
    let ref = (0, $gwkOZ$useRef)(null);
    let { isUnavailable: isUnavailable, formattedDate: formattedDate, isSelected: isSelected, isSelectionStart: isSelectionStart, isSelectionEnd: isSelectionEnd, isInvalid: isInvalid } = renderProps;
    // only apply the selection start/end styles if the start/end date is actually selectable (aka not unavailable)
    // or if the range is invalid and thus we still want to show the styles even if the start/end date is an unavailable one
    isSelectionStart = isSelectionStart && (!isUnavailable || isInvalid);
    isSelectionEnd = isSelectionEnd && (!isUnavailable || isInvalid);
    let isDateInRange = (checkDate)=>{
        if (!('highlightedRange' in state) || !state.highlightedRange) return state.isSelected(checkDate);
        // if invalid, check if date is within the full range boundaries
        if (isInvalid) return checkDate.compare(state.highlightedRange.start) >= 0 && checkDate.compare(state.highlightedRange.end) <= 0;
        return state.isSelected(checkDate);
    };
    let prevDay = date.subtract({
        days: 1
    });
    let nextDay = date.add({
        days: 1
    });
    let isFirstDayInWeek = dayIndex === 0;
    let isLastDayInWeek = dayIndex === 6;
    let isPreviousDayNotSelected = !prevDay || !isDateInRange(prevDay) || prevDay.month !== props.date.month;
    let isNextDayNotSelected = !nextDay || !isDateInRange(nextDay) || nextDay.month !== props.date.month;
    // when invalid, show background for all selected dates (including unavailable) to make continuous range appearance
    // when valid, only show background for available selected dates
    let isBackgroundStyleApplied = isSelected && isRangeSelection && (isInvalid || !isUnavailable) && (isDateInRange(prevDay) || nextDay.month === date.month && isDateInRange(nextDay));
    return /*#__PURE__*/ (0, $gwkOZ$jsxs)("div", {
        className: " _Pc131 Za131 Fb131",
        children: [
            /*#__PURE__*/ (0, $gwkOZ$jsxs)("div", {
                ref: ref,
                style: (0, $10ea7662e51a285b$export$56e8cba416805d8d)(ref, {})(renderProps),
                className: $f06f4d5597ad704c$var$cellInnerStyles({
                    ...renderProps,
                    isSelectionStart: isSelectionStart,
                    isSelectionEnd: isSelectionEnd,
                    selectionMode: isRangeSelection ? 'range' : 'single'
                }),
                children: [
                    /*#__PURE__*/ (0, $gwkOZ$jsx)("div", {
                        className: $f06f4d5597ad704c$var$todayStyles(renderProps),
                        role: "presentation"
                    }),
                    /*#__PURE__*/ (0, $gwkOZ$jsx)("div", {
                        children: formattedDate
                    }),
                    isUnavailable && /*#__PURE__*/ (0, $gwkOZ$jsx)("div", {
                        className: $f06f4d5597ad704c$var$unavailableStyles,
                        role: "presentation"
                    })
                ]
            }),
            isBackgroundStyleApplied && /*#__PURE__*/ (0, $gwkOZ$jsx)("div", {
                className: $f06f4d5597ad704c$var$selectionBackgroundStyles({
                    isInvalid: isInvalid,
                    isFirstDayInWeek: isFirstDayInWeek,
                    isLastDayInWeek: isLastDayInWeek,
                    isSelectionStart: isSelectionStart,
                    isSelectionEnd: isSelectionEnd,
                    isPreviousDayNotSelected: isPreviousDayNotSelected,
                    isNextDayNotSelected: isNextDayNotSelected
                }),
                role: "presentation"
            }),
            isBackgroundStyleApplied && /*#__PURE__*/ (0, $gwkOZ$jsx)("div", {
                className: $f06f4d5597ad704c$var$selectionBorderStyles({
                    isInvalid: isInvalid,
                    isFirstDayInWeek: isFirstDayInWeek,
                    isLastDayInWeek: isLastDayInWeek,
                    isSelectionStart: isSelectionStart,
                    isSelectionEnd: isSelectionEnd,
                    isPreviousDayNotSelected: isPreviousDayNotSelected,
                    isNextDayNotSelected: isNextDayNotSelected
                }),
                role: "presentation"
            })
        ]
    });
};
/**
 * Calculate the week index (0-based) and day index (0-based) for a given date within a month in a
 * calendar.
 *
 * @param date - The date to calculate indices for.
 * @param locale - The locale string (e.g., 'en-US', 'fr-FR', 'hi-IN-u-ca-indian').
 * @param firstDayOfWeek - Optional override for the first day of the week ('sun', 'mon', 'tue',
 *   etc.).
 * @returns Object with weekIndex and dayIndex.
 */ function $f06f4d5597ad704c$var$useWeekAndDayIndices(date, locale, firstDayOfWeek) {
    let result = (0, $gwkOZ$useMemo)(()=>{
        // Get the day index within the week (0-6)
        const dayIndex = (0, $gwkOZ$getDayOfWeek)(date, locale, firstDayOfWeek);
        const monthStart = (0, $gwkOZ$startOfMonth)(date);
        // Calculate the week index by finding which week this date falls into
        // within the month's calendar grid
        const monthStartDayOfWeek = (0, $gwkOZ$getDayOfWeek)(monthStart, locale, firstDayOfWeek);
        const dayOfMonth = date.day;
        const weekIndex = Math.floor((dayOfMonth + monthStartDayOfWeek - 1) / 7);
        const lastDayOfMonth = (0, $gwkOZ$startOfMonth)(date).add({
            months: 1
        }).subtract({
            days: 1
        });
        const lastWeekIndex = Math.floor((lastDayOfMonth.day + monthStartDayOfWeek - 1) / 7);
        return {
            weekIndex: weekIndex,
            lastWeekIndex: lastWeekIndex,
            dayIndex: dayIndex
        };
    }, [
        date,
        locale,
        firstDayOfWeek
    ]);
    return result;
}


export {$f06f4d5597ad704c$export$3b805cea1f178355 as CalendarContext, $f06f4d5597ad704c$export$e1aef45b828286de as Calendar, $f06f4d5597ad704c$export$9c75275274db0b56 as CalendarHeader, $f06f4d5597ad704c$export$5bd780d491cfc46c as CalendarGrid, $f06f4d5597ad704c$export$adb29da0ac001538 as CalendarButton};
