// @react-spectrum/s2/dist/private/Calendar.cjs
require("./Calendar.css");
var $6e265ff388155b91$exports = require("./ActionButton.cjs");
var $4f683da24e35b31a$exports = require("../../icons/ChevronLeft.cjs");
var $f76eebe9b8ebc714$exports = require("../../icons/ChevronRight.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $4526404114e78c80$exports = require("./intlStrings.cjs");
var $2061c83559b50a66$exports = require("./pressScale.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $425Kz$reactjsxruntime = require("react/jsx-runtime");
var $425Kz$reactariacomponentsCalendar = require("react-aria-components/Calendar");
var $425Kz$internationalizeddate = require("@internationalized/date");
var $425Kz$reactariacomponentsslots = require("react-aria-components/slots");
var $425Kz$reactariacomponentsRangeCalendar = require("react-aria-components/RangeCalendar");
var $425Kz$react = require("react");
var $425Kz$reactariacomponentsText = require("react-aria-components/Text");
var $425Kz$reactariaI18nProvider = require("react-aria/I18nProvider");
var $425Kz$reactariauseLocalizedStringFormatter = require("react-aria/useLocalizedStringFormatter");


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "CalendarContext", () => $e9e9a0d15c3364d7$export$3b805cea1f178355);
$parcel$export(module.exports, "Calendar", () => $e9e9a0d15c3364d7$export$e1aef45b828286de);
$parcel$export(module.exports, "CalendarHeader", () => $e9e9a0d15c3364d7$export$9c75275274db0b56);
$parcel$export(module.exports, "CalendarGrid", () => $e9e9a0d15c3364d7$export$5bd780d491cfc46c);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 
















const $e9e9a0d15c3364d7$export$3b805cea1f178355 = /*#__PURE__*/ (0, $425Kz$react.createContext)(null);
const $e9e9a0d15c3364d7$var$calendarStyles = function anonymous(props, overrides) {
    let rules = " ";
    let width = false;
    let maxWidth = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "Z") width = true;
        if (p[1] === "L") maxWidth = true;
        rules += p[0];
    }
    rules += ' sd131';
    if (props.isMultiMonth) rules += ' -_4DHzJc-e4mUs131';
    else rules += ' -_4DHzJc-0u6p5b131';
    rules += ' _ta131';
    rules += ' Uj131';
    rules += ' qj131';
    rules += ' __ca131';
    rules += ' -uq4sQ-Sp131';
    rules += ' -LnVzy-ZC131';
    if (props.isMultiMonth) rules += ' -cYDWLb-ZFFQGNb131';
    else rules += ' -cYDWLb-ZnqRBTd131';
    if (props.isMultiMonth) {
        if (!width) rules += ' ZJ131';
    } else if (!width) rules += ' ZRqDA2c131';
    if (props.isMultiMonth) {
        if (!maxWidth) rules += ' Le4mUs131';
    } else if (!maxWidth) rules += ' La131';
    return rules;
};
const $e9e9a0d15c3364d7$var$headerStyles = " sd131 eb131 _Ce131 qj131";
const $e9e9a0d15c3364d7$var$headingStyles = " sd131 eb131 _Ce131 Jy131 Gy131 Iy131 Hy131 _ub131 Za131";
const $e9e9a0d15c3364d7$var$titleStyles = " uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-e131 vx131 xd131 _xa131 xX0cczbc131 _xX0cczba131 xfd131 _xfa131 xfX0cczbc131 _xfX0cczba131 _Fd131 _FnuYUweb131 po131 _Wa131 _ub131 _va131 Jy131 Gy131";
const $e9e9a0d15c3364d7$var$headerCellStyles = " uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-c131 vx131 xd131 _xa131 xX0cczbc131 _xX0cczba131 xfd131 _xfa131 xfX0cczbc131 _xfX0cczba131 _Fd131 _FnuYUweb131 po131 ri131 _Wa131 Sp131 Sbd131 Rp131 Rkd131 Qe131";
const $e9e9a0d15c3364d7$var$cellStyles = function anonymous(props) {
    let rules = " ";
    rules += ' _Le131';
    rules += ' _ob131';
    if (props.isFirstChild) rules += ' Sd131';
    else rules += ' Sp131';
    if (props.isLastChild) rules += ' Rd131';
    else rules += ' Rp131';
    if (props.isFirstWeek) rules += ' Td131';
    else rules += ' Tj131';
    if (props.isLastWeek) rules += ' Qd131';
    else rules += ' Qj131';
    rules += ' _Pc131';
    if (props.isOutsideMonth) rules += ' sk131';
    else rules += ' sd131';
    rules += ' eb131';
    rules += ' _Ca131';
    rules += ' __ca131';
    rules += ' Z0q26ub131';
    rules += ' F0q26ub131';
    return rules;
};
const $e9e9a0d15c3364d7$var$cellInnerStyles = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    if (props.isSelectionEnd) rules += ' _Kd131';
    else if (props.isSelectionStart) rules += ' _Kd131';
    else if (props.isSelected) {
        if (props.selectionMode === "range") rules += ' _Kb131';
        else if (props.selectionMode === "single") rules += ' _Kd131';
    } else rules += ' _Kb131';
    rules += ' Yd131';
    rules += ' Yle131';
    rules += ' Xb131';
    rules += ' Xlb131';
    rules += ' _2b131';
    rules += ' _2lb131';
    rules += ' _Pc131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-c131';
    rules += ' vx131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fb131';
    rules += ' _FnuYUwec131';
    if (props.isDisabled) rules += ' pp131';
    else if (props.isSelectionEnd) rules += ' px131';
    else if (props.isSelectionStart) rules += ' px131';
    else if (props.isSelected) {
        if (props.selectionMode === "range") rules += ' pt131';
        else rules += ' px131';
    } else rules += ' pt131';
    if (props.isDisabled) rules += ' plc131';
    else if (props.isSelectionEnd) rules += ' ple131';
    else if (props.isSelectionStart) rules += ' ple131';
    else if (props.isSelected) rules += ' ple131';
    else rules += ' plb131';
    rules += ' ri131';
    rules += ' Za131';
    rules += ' Fb131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _Ca131';
    rules += ' _yb131';
    if (props.isSelectionEnd) {
        if (props.isInvalid) {
            if (props.isFocusVisible) {
                if (props.isUnavailable) rules += ' g7aei8131';
                else rules += ' gYvV9s131';
            } else if (props.isPressed) rules += ' gYvV9s131';
            else if (props.isHovered) {
                if (props.isUnavailable) rules += ' g7aei8131';
                else rules += ' gYvV9s131';
            } else rules += ' g7aei8131';
        } else if (props.isDisabled) rules += ' g9131';
        else if (props.isFocusVisible) rules += ' gUgARdd131';
        else if (props.isPressed) rules += ' gUgARdd131';
        else if (props.isHovered) rules += ' gUgARdd131';
        else rules += ' g5qAiPc131';
    } else if (props.isSelectionStart) {
        if (props.isInvalid) {
            if (props.isFocusVisible) {
                if (props.isUnavailable) rules += ' g7aei8131';
                else rules += ' gYvV9s131';
            } else if (props.isPressed) rules += ' gYvV9s131';
            else if (props.isHovered) {
                if (props.isUnavailable) rules += ' g7aei8131';
                else rules += ' gYvV9s131';
            } else rules += ' g7aei8131';
        } else if (props.isDisabled) rules += ' g9131';
        else if (props.isFocusVisible) rules += ' gUgARdd131';
        else if (props.isPressed) rules += ' gUgARdd131';
        else if (props.isHovered) rules += ' gUgARdd131';
        else rules += ' g5qAiPc131';
    } else if (props.isSelected) {
        if (props.selectionMode === "range") {
            if (props.isHovered) {
                if (props.isInvalid) {
                    if (props.isUnavailable) rules += ' g9131';
                    else rules += ' gJj9Tpd131';
                } else rules += ' gNAacne131';
            }
        } else if (props.selectionMode === "single") {
            if (props.isInvalid) {
                if (props.isFocusVisible) rules += ' gYvV9s131';
                else if (props.isPressed) rules += ' gYvV9s131';
                else if (props.isHovered) rules += ' gYvV9s131';
                else rules += ' g7aei8131';
            } else if (props.isDisabled) rules += ' g9131';
            else if (props.isFocusVisible) rules += ' gUgARdd131';
            else if (props.isPressed) rules += ' gUgARdd131';
            else if (props.isHovered) rules += ' gUgARdd131';
            else rules += ' g5qAiPc131';
        }
    } else if (props.isDisabled) rules += ' g9131';
    else if (props.isPressed) rules += ' gH131';
    else if (props.isHovered) {
        if (props.isUnavailable) rules += ' g9131';
        else rules += ' gH131';
    } else rules += ' g9131';
    if (props.isUnavailable) rules += ' gl9131';
    else if (props.isSelectionEnd) rules += ' gle131';
    else if (props.isSelectionStart) rules += ' gle131';
    else if (props.isSelected) {
        if (props.selectionMode === "range") {
            if (props.isHovered) rules += ' gle131';
        } else if (props.selectionMode === "single") rules += ' gle131';
    } else if (props.isHovered) rules += ' gle131';
    else rules += ' gl9131';
    return rules;
};
const $e9e9a0d15c3364d7$var$todayStyles = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' _lakDin131';
    rules += ' _EF131';
    rules += ' _ZRWeLS131';
    rules += ' ZS16uu131';
    rules += ' Fe131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    rules += ' g5ZbAob131';
    if (props.isToday) rules += ' sb131';
    else rules += ' sk131';
    return rules;
};
const $e9e9a0d15c3364d7$var$unavailableStyles = " _Pa131 WzErnCb131 _EjhYzGc131 _RjhYzGc131 FlAZvq131 _ZN7ydnb131 og131 ng131 kg131 jg131 g5ZbAob131";
const $e9e9a0d15c3364d7$var$selectionBackgroundStyles = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' _9a131';
    rules += ' Wr131';
    if (props.isPreviousDayNotSelected) rules += ' _Ar131';
    else if (props.isSelectionStart) rules += ' _Ar131';
    else if (props.isFirstDayInWeek) rules += ' _Ar131';
    else rules += ' _Al131';
    if (props.isNextDayNotSelected) rules += ' _zr131';
    else if (props.isSelectionEnd) rules += ' _zr131';
    else if (props.isLastDayInWeek) rules += ' _zr131';
    else rules += ' _zl131';
    rules += ' _lr131';
    if (props.isPreviousDayNotSelected) rules += ' og131';
    else if (props.isSelectionStart) rules += ' og131';
    else if (props.isFirstDayInWeek) rules += ' og131';
    else rules += ' od131';
    if (props.isPreviousDayNotSelected) rules += ' kg131';
    else if (props.isSelectionStart) rules += ' kg131';
    else if (props.isFirstDayInWeek) rules += ' kg131';
    else rules += ' kd131';
    if (props.isNextDayNotSelected) rules += ' ng131';
    else if (props.isSelectionEnd) rules += ' ng131';
    else if (props.isLastDayInWeek) rules += ' ng131';
    else rules += ' nd131';
    if (props.isNextDayNotSelected) rules += ' jg131';
    else if (props.isSelectionEnd) rules += ' jg131';
    else if (props.isLastDayInWeek) rules += ' jg131';
    else rules += ' jd131';
    if (props.isInvalid) rules += ' gZbQnx131';
    else rules += ' gG131';
    rules += ' gle131';
    rules += ' _yb131';
    return rules;
};
const $e9e9a0d15c3364d7$var$selectionBorderStyles = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' _9c131';
    rules += ' Wr131';
    if (props.isPreviousDayNotSelected) rules += ' _Ar131';
    else if (props.isSelectionStart) rules += ' _Ar131';
    else if (props.isFirstDayInWeek) rules += ' _Ar131';
    else rules += ' _Al131';
    if (props.isNextDayNotSelected) rules += ' _zr131';
    else if (props.isSelectionEnd) rules += ' _zr131';
    else if (props.isLastDayInWeek) rules += ' _zr131';
    else rules += ' _zl131';
    rules += ' _lr131';
    if (props.isPreviousDayNotSelected) rules += ' mb131';
    else if (props.isSelectionStart) rules += ' mb131';
    else if (props.isFirstDayInWeek) rules += ' mb131';
    else rules += ' ma131';
    rules += ' _kb131';
    if (props.isNextDayNotSelected) rules += ' lb131';
    else if (props.isSelectionEnd) rules += ' lb131';
    else if (props.isLastDayInWeek) rules += ' lb131';
    else rules += ' la131';
    rules += ' hb131';
    rules += ' _jf131';
    if (props.isInvalid) rules += ' ip131';
    else rules += ' ix131';
    rules += ' ilb131';
    if (props.isPreviousDayNotSelected) rules += ' og131';
    else if (props.isSelectionStart) rules += ' og131';
    else if (props.isFirstDayInWeek) rules += ' og131';
    else rules += ' od131';
    if (props.isPreviousDayNotSelected) rules += ' kg131';
    else if (props.isSelectionStart) rules += ' kg131';
    else if (props.isFirstDayInWeek) rules += ' kg131';
    else rules += ' kd131';
    if (props.isNextDayNotSelected) rules += ' ng131';
    else if (props.isSelectionEnd) rules += ' ng131';
    else if (props.isLastDayInWeek) rules += ' ng131';
    else rules += ' nd131';
    if (props.isNextDayNotSelected) rules += ' jg131';
    else if (props.isSelectionEnd) rules += ' jg131';
    else if (props.isLastDayInWeek) rules += ' jg131';
    else rules += ' jd131';
    return rules;
};
const $e9e9a0d15c3364d7$export$e1aef45b828286de = /*#__PURE__*/ (0, $425Kz$react.forwardRef)(function Calendar(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $e9e9a0d15c3364d7$export$3b805cea1f178355);
    let { visibleMonths: visibleMonths = 1, errorMessage: errorMessage, UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className, styles: styles, ...otherProps } = props;
    let stringFormatter = (0, $425Kz$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    let isMultiMonth = visibleMonths > 1;
    return /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)((0, $425Kz$reactariacomponentsCalendar.Calendar), {
        ...otherProps,
        ref: ref,
        visibleDuration: {
            months: visibleMonths
        },
        style: UNSAFE_style,
        className: (UNSAFE_className || '') + $e9e9a0d15c3364d7$var$calendarStyles({
            isMultiMonth: isMultiMonth
        }, styles),
        children: ({ isInvalid: isInvalid, isDisabled: isDisabled })=>{
            return /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsxs)((0, $425Kz$reactjsxruntime.Fragment), {
                children: [
                    /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)((0, $425Kz$reactariacomponentsslots.Provider), {
                        values: [
                            [
                                (0, $6367bc87eb7d24ad$exports.HeaderContext),
                                null
                            ],
                            [
                                (0, $6367bc87eb7d24ad$exports.HeadingContext),
                                null
                            ]
                        ],
                        children: /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)($e9e9a0d15c3364d7$export$9c75275274db0b56, {
                            visibleMonths: visibleMonths
                        })
                    }),
                    /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)("div", {
                        className: " sd131 _tc131 Uj131 qj131 Za131 ed131",
                        children: Array.from({
                            length: visibleMonths
                        }).map((_, i)=>/*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)($e9e9a0d15c3364d7$export$5bd780d491cfc46c, {
                                months: i
                            }, i))
                    }),
                    isInvalid && /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)((0, $425Kz$reactariacomponentsText.Text), {
                        slot: "errorMessage",
                        className: (0, $a0b996a550bb0891$exports.helpTextStyles)({
                            isInvalid: isInvalid,
                            isDisabled: isDisabled,
                            size: 'M'
                        }),
                        children: errorMessage || stringFormatter.format('calendar.invalidSelection', {
                            selectedCount: 1
                        })
                    })
                ]
            });
        }
    });
});
const $e9e9a0d15c3364d7$export$9c75275274db0b56 = ({ visibleMonths: visibleMonths })=>{
    return /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)((0, $6367bc87eb7d24ad$exports.Header), {
        styles: $e9e9a0d15c3364d7$var$headerStyles,
        children: Array.from({
            length: visibleMonths
        }).map((_, i)=>/*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsxs)("div", {
                className: $e9e9a0d15c3364d7$var$headingStyles,
                children: [
                    i === 0 && /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)($e9e9a0d15c3364d7$export$adb29da0ac001538, {
                        slot: "previous",
                        children: /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)((0, $4f683da24e35b31a$exports.default), {})
                    }),
                    /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)((0, $425Kz$reactariacomponentsCalendar.CalendarHeading), {
                        offset: {
                            months: i
                        },
                        className: $e9e9a0d15c3364d7$var$titleStyles
                    }),
                    i === visibleMonths - 1 && /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)($e9e9a0d15c3364d7$export$adb29da0ac001538, {
                        slot: "next",
                        children: /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)((0, $f76eebe9b8ebc714$exports.default), {})
                    })
                ]
            }, i))
    });
};
const $e9e9a0d15c3364d7$export$5bd780d491cfc46c = (props)=>{
    let rangeCalendarProps = (0, $425Kz$reactariacomponentsslots.useSlottedContext)((0, $425Kz$reactariacomponentsRangeCalendar.RangeCalendarContext));
    let calendarProps = (0, $425Kz$reactariacomponentsslots.useSlottedContext)((0, $425Kz$reactariacomponentsCalendar.CalendarContext));
    let firstDayOfWeek = rangeCalendarProps?.firstDayOfWeek ?? calendarProps?.firstDayOfWeek;
    // use isolation to start a new stacking context so that we can use zIndex -1 for the selection span.
    return /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsxs)((0, $425Kz$reactariacomponentsCalendar.CalendarGrid), {
        className: " __na131 __oe131 _Bb131",
        offset: {
            months: props.months
        },
        children: [
            /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)((0, $425Kz$reactariacomponentsCalendar.CalendarGridHeader), {
                className: "",
                children: (day)=>/*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)($e9e9a0d15c3364d7$var$CalendarHeaderCell, {
                        children: day
                    })
            }),
            /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)((0, $425Kz$reactariacomponentsCalendar.CalendarGridBody), {
                className: "",
                children: (date)=>/*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)($e9e9a0d15c3364d7$var$CalendarCell, {
                        date: date,
                        firstDayOfWeek: firstDayOfWeek
                    })
            })
        ]
    });
};
const $e9e9a0d15c3364d7$export$adb29da0ac001538 = (props1)=>{
    let { direction: direction } = (0, $425Kz$reactariaI18nProvider.useLocale)();
    return /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)("div", {
        className: function anonymous(props) {
            let rules = " ";
            if (props.direction === "rtl") {
                rules += ' -Z4Pn4c-a131';
                rules += ' __Ha131';
            }
            if (props.direction === "rtl") {
                rules += ' -_04Pn4c-a131';
                rules += ' __Ha131';
            }
            return rules;
        }({
            direction: direction
        }),
        children: /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)((0, $6e265ff388155b91$exports.ActionButton), {
            ...props1,
            isQuiet: true,
            children: props1.children
        })
    });
};
const $e9e9a0d15c3364d7$var$CalendarHeaderCell = (props)=>{
    return /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)((0, $425Kz$reactariacomponentsCalendar.CalendarHeaderCell), {
        className: $e9e9a0d15c3364d7$var$headerCellStyles,
        children: props.children
    });
};
const $e9e9a0d15c3364d7$var$CalendarCell = (props)=>{
    let { locale: locale } = (0, $425Kz$reactariaI18nProvider.useLocale)();
    let firstDayOfWeek = props.firstDayOfWeek;
    // Calculate the day and week index based on the date.
    let { dayIndex: dayIndex, weekIndex: weekIndex, lastWeekIndex: lastWeekIndex } = $e9e9a0d15c3364d7$var$useWeekAndDayIndices(props.date, locale, firstDayOfWeek);
    let calendarStateContext = (0, $425Kz$react.useContext)((0, $425Kz$reactariacomponentsCalendar.CalendarStateContext));
    let rangeCalendarStateContext = (0, $425Kz$react.useContext)((0, $425Kz$reactariacomponentsRangeCalendar.RangeCalendarStateContext));
    let state = calendarStateContext ?? rangeCalendarStateContext;
    let isFirstWeek = weekIndex === 0;
    let isLastWeek = weekIndex === lastWeekIndex;
    let isFirstChild = dayIndex === 0;
    let isLastChild = dayIndex === 6;
    return /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)((0, $425Kz$reactariacomponentsCalendar.CalendarCell), {
        date: props.date,
        className: (renderProps)=>$e9e9a0d15c3364d7$var$cellStyles({
                ...renderProps,
                isFirstChild: isFirstChild,
                isLastChild: isLastChild,
                isFirstWeek: isFirstWeek,
                isLastWeek: isLastWeek
            }),
        children: (renderProps)=>/*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)($e9e9a0d15c3364d7$var$CalendarCellInner, {
                ...props,
                weekIndex: weekIndex,
                dayIndex: dayIndex,
                state: state,
                isRangeSelection: !!rangeCalendarStateContext,
                renderProps: renderProps
            })
    });
};
const $e9e9a0d15c3364d7$var$CalendarCellInner = (props)=>{
    let { dayIndex: dayIndex, date: date, renderProps: renderProps, state: state, isRangeSelection: isRangeSelection } = props;
    let ref = (0, $425Kz$react.useRef)(null);
    let { isUnavailable: isUnavailable, formattedDate: formattedDate, isSelected: isSelected, isSelectionStart: isSelectionStart, isSelectionEnd: isSelectionEnd, isInvalid: isInvalid } = renderProps;
    // only apply the selection start/end styles if the start/end date is actually selectable (aka not unavailable)
    // or if the range is invalid and thus we still want to show the styles even if the start/end date is an unavailable one
    isSelectionStart = isSelectionStart && (!isUnavailable || isInvalid);
    isSelectionEnd = isSelectionEnd && (!isUnavailable || isInvalid);
    let isDateInRange = (checkDate)=>{
        if (!('highlightedRange' in state) || !state.highlightedRange) return state.isSelected(checkDate);
        // if invalid, check if date is within the full range boundaries
        if (isInvalid) return checkDate.compare(state.highlightedRange.start) >= 0 && checkDate.compare(state.highlightedRange.end) <= 0;
        return state.isSelected(checkDate);
    };
    let prevDay = date.subtract({
        days: 1
    });
    let nextDay = date.add({
        days: 1
    });
    let isFirstDayInWeek = dayIndex === 0;
    let isLastDayInWeek = dayIndex === 6;
    let isPreviousDayNotSelected = !prevDay || !isDateInRange(prevDay) || prevDay.month !== props.date.month;
    let isNextDayNotSelected = !nextDay || !isDateInRange(nextDay) || nextDay.month !== props.date.month;
    // when invalid, show background for all selected dates (including unavailable) to make continuous range appearance
    // when valid, only show background for available selected dates
    let isBackgroundStyleApplied = isSelected && isRangeSelection && (isInvalid || !isUnavailable) && (isDateInRange(prevDay) || nextDay.month === date.month && isDateInRange(nextDay));
    return /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsxs)("div", {
        className: " _Pc131 Za131 Fb131",
        children: [
            /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsxs)("div", {
                ref: ref,
                style: (0, $2061c83559b50a66$exports.pressScale)(ref, {})(renderProps),
                className: $e9e9a0d15c3364d7$var$cellInnerStyles({
                    ...renderProps,
                    isSelectionStart: isSelectionStart,
                    isSelectionEnd: isSelectionEnd,
                    selectionMode: isRangeSelection ? 'range' : 'single'
                }),
                children: [
                    /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)("div", {
                        className: $e9e9a0d15c3364d7$var$todayStyles(renderProps),
                        role: "presentation"
                    }),
                    /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)("div", {
                        children: formattedDate
                    }),
                    isUnavailable && /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)("div", {
                        className: $e9e9a0d15c3364d7$var$unavailableStyles,
                        role: "presentation"
                    })
                ]
            }),
            isBackgroundStyleApplied && /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)("div", {
                className: $e9e9a0d15c3364d7$var$selectionBackgroundStyles({
                    isInvalid: isInvalid,
                    isFirstDayInWeek: isFirstDayInWeek,
                    isLastDayInWeek: isLastDayInWeek,
                    isSelectionStart: isSelectionStart,
                    isSelectionEnd: isSelectionEnd,
                    isPreviousDayNotSelected: isPreviousDayNotSelected,
                    isNextDayNotSelected: isNextDayNotSelected
                }),
                role: "presentation"
            }),
            isBackgroundStyleApplied && /*#__PURE__*/ (0, $425Kz$reactjsxruntime.jsx)("div", {
                className: $e9e9a0d15c3364d7$var$selectionBorderStyles({
                    isInvalid: isInvalid,
                    isFirstDayInWeek: isFirstDayInWeek,
                    isLastDayInWeek: isLastDayInWeek,
                    isSelectionStart: isSelectionStart,
                    isSelectionEnd: isSelectionEnd,
                    isPreviousDayNotSelected: isPreviousDayNotSelected,
                    isNextDayNotSelected: isNextDayNotSelected
                }),
                role: "presentation"
            })
        ]
    });
};
/**
 * Calculate the week index (0-based) and day index (0-based) for a given date within a month in a
 * calendar.
 *
 * @param date - The date to calculate indices for.
 * @param locale - The locale string (e.g., 'en-US', 'fr-FR', 'hi-IN-u-ca-indian').
 * @param firstDayOfWeek - Optional override for the first day of the week ('sun', 'mon', 'tue',
 *   etc.).
 * @returns Object with weekIndex and dayIndex.
 */ function $e9e9a0d15c3364d7$var$useWeekAndDayIndices(date, locale, firstDayOfWeek) {
    let result = (0, $425Kz$react.useMemo)(()=>{
        // Get the day index within the week (0-6)
        const dayIndex = (0, $425Kz$internationalizeddate.getDayOfWeek)(date, locale, firstDayOfWeek);
        const monthStart = (0, $425Kz$internationalizeddate.startOfMonth)(date);
        // Calculate the week index by finding which week this date falls into
        // within the month's calendar grid
        const monthStartDayOfWeek = (0, $425Kz$internationalizeddate.getDayOfWeek)(monthStart, locale, firstDayOfWeek);
        const dayOfMonth = date.day;
        const weekIndex = Math.floor((dayOfMonth + monthStartDayOfWeek - 1) / 7);
        const lastDayOfMonth = (0, $425Kz$internationalizeddate.startOfMonth)(date).add({
            months: 1
        }).subtract({
            days: 1
        });
        const lastWeekIndex = Math.floor((lastDayOfMonth.day + monthStartDayOfWeek - 1) / 7);
        return {
            weekIndex: weekIndex,
            lastWeekIndex: lastWeekIndex,
            dayIndex: dayIndex
        };
    }, [
        date,
        locale,
        firstDayOfWeek
    ]);
    return result;
}
