// @react-spectrum/s2/dist/private/Card.mjs
import "./Card.css";
import {ActionMenuContext as $8bf3cd2442eb485e$export$7f4b66acaca28bd5} from "./ActionMenu.mjs";
import {AvatarContext as $3ffa2cd930156220$export$a20dad690e1279e2} from "./Avatar.mjs";
import {ButtonContext as $067ea9f64ccd4e8e$export$24d547caef80ccd1, LinkButtonContext as $067ea9f64ccd4e8e$export$1c1a176f3b9e48d3} from "./Button.mjs";
import {Checkbox as $9b405e0f8095dce0$export$48513f6b9f8ce62d} from "./Checkbox.mjs";
import {ContentContext as $8e847109a6ab556d$export$1cbdd774077931b4, FooterContext as $8e847109a6ab556d$export$5630640b68817ed6, TextContext as $8e847109a6ab556d$export$9afb8bc826b033ea} from "./Content.mjs";
import {DividerContext as $bdf52657dac0f070$export$30fbfd376abdf190} from "./Divider.mjs";
import {IllustrationContext as $ac8c32e6775ddd1f$export$207c7561471de4bd} from "../../icons/Icon.mjs";
import {ImageContext as $dbd6f0b2503b938c$export$8151eb379fcf2442} from "./Image.mjs";
import {ImageCoordinator as $4b5e069e9e001e8b$export$1b926c015f09611d} from "./ImageCoordinator.mjs";
import {mergeStyles as $feb886035e0d4633$export$e618dc39ac9ad607} from "../../icons/runtime.mjs";
import {pressScale as $10ea7662e51a285b$export$56e8cba416805d8d} from "./pressScale.mjs";
import {SkeletonContext as $5ad421ec19460c48$export$74e166679b1f49ee, SkeletonWrapper as $5ad421ec19460c48$export$6069cbe61f690103, useIsSkeleton as $5ad421ec19460c48$export$4f8dc7555740235c} from "../../icons/Skeleton.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $jzFZE$jsx, jsxs as $jzFZE$jsxs} from "react/jsx-runtime";
import {composeRenderProps as $jzFZE$composeRenderProps} from "react-aria-components/composeRenderProps";
import {Provider as $jzFZE$Provider, DEFAULT_SLOT as $jzFZE$DEFAULT_SLOT} from "react-aria-components/slots";
import {createContext as $jzFZE$createContext, forwardRef as $jzFZE$forwardRef, useContext as $jzFZE$useContext} from "react";
import {filterDOMProps as $jzFZE$filterDOMProps} from "react-aria/filterDOMProps";
import {inertValue as $jzFZE$inertValue} from "react-aria/private/utils/inertValue";
import {Link as $jzFZE$Link} from "react-aria-components/Link";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 




















const $68e4e6fe083e22fd$var$borderRadius = {
    default: 'lg',
    size: {
        XS: 'default',
        S: 'default'
    }
};
let $68e4e6fe083e22fd$var$card = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let width = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "Z") width = true;
        rules += p[0];
    }
    rules += ' sd131';
    rules += ' _ta131';
    if (!position) rules += ' _Pc131';
    if (props.size === "S") rules += ' ob131';
    else if (props.size === "XS") rules += ' ob131';
    else rules += ' oc131';
    if (props.size === "S") rules += ' nb131';
    else if (props.size === "XS") rules += ' nb131';
    else rules += ' nc131';
    if (props.size === "S") rules += ' kb131';
    else if (props.size === "XS") rules += ' kb131';
    else rules += ' kc131';
    if (props.size === "S") rules += ' jb131';
    else if (props.size === "XS") rules += ' jb131';
    else rules += ' jc131';
    if (props.variant === "secondary") rules += ' -Ovbhqd-gqlRAZb131';
    else if (props.variant === "primary") rules += ' -Ovbhqd-g_g131';
    rules += ' -Ovbhqd-lgb131';
    if (props.variant === "quiet") rules += ' g9131';
    else if (props.variant === "tertiary") rules += ' g9131';
    else rules += ' gEzkoQe131';
    if (props.variant === "quiet") rules += ' _nd131';
    else if (props.variant === "tertiary") {
        if (props.isSelected) rules += ' _nd131';
        else if (props.isFocusVisible) rules += ' _nrC99Ne131';
        else if (props.isHovered) rules += ' _nrC99Ne131';
        else rules += ' _nnvmtGb131';
        rules += ' _nli6j2Ec131';
    } else {
        if (props.isSelected) rules += ' _nLeasBb131';
        else if (props.isFocusVisible) rules += ' _nLeasBb131';
        else if (props.isHovered) rules += ' _nLeasBb131';
        else rules += ' _n4zCLMe131';
        rules += ' _nlJWldjc131';
    }
    rules += ' _yb131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' _Xa131';
    if (props.variant === "quiet") rules += ' _Ne131';
    else rules += ' _Nb131';
    if (props.variant === "quiet") rules += ' Pe131';
    else rules += ' Pb131';
    rules += ' _qc131';
    rules += ' __ca131';
    if (props.isCardView) rules += ' _4c131';
    if (props.isLink) rules += ' ry131';
    if (props.isCardView) {
        if (!width) rules += ' Za131';
    } else if (props.size === "XL") {
        if (!width) rules += ' ZA131';
    } else if (props.size === "L") {
        if (!width) rules += ' Zy131';
    } else if (props.size === "M") {
        if (!width) rules += ' Zt131';
    } else if (props.size === "S") {
        if (!width) rules += ' Zr131';
    } else if (props.size === "XS") {
        if (!width) rules += ' ZI131';
    }
    rules += ' Fb131';
    if (props.density === "spacious") {
        if (props.size === "XL") rules += ' -dAV6Jb-Ti131';
        else if (props.size === "L") rules += ' -dAV6Jb-Th131';
        else if (props.size === "M") rules += ' -dAV6Jb-Tg131';
        else if (props.size === "S") rules += ' -dAV6Jb-Tf131';
        else if (props.size === "XS") rules += ' -dAV6Jb-Te131';
    } else if (props.density === "regular") {
        if (props.size === "XL") rules += ' -dAV6Jb-Th131';
        else if (props.size === "L") rules += ' -dAV6Jb-Tg131';
        else if (props.size === "M") rules += ' -dAV6Jb-Tf131';
        else if (props.size === "S") rules += ' -dAV6Jb-Te131';
        else if (props.size === "XS") rules += ' -dAV6Jb-Tt131';
    } else if (props.density === "compact") {
        if (props.size === "XL") rules += ' -dAV6Jb-Tg131';
        else if (props.size === "L") rules += ' -dAV6Jb-Tf131';
        else if (props.size === "M") rules += ' -dAV6Jb-Te131';
        else if (props.size === "S") rules += ' -dAV6Jb-Tt131';
        else if (props.size === "XS") rules += ' -dAV6Jb-T1DOfn131';
    }
    if (props.variant === "quiet") rules += ' -jXLvjd-Td131';
    else rules += ' -jXLvjd-T5gMYHd131';
    if (props.variant === "quiet") rules += ' -iXLvjd-Sd131';
    else rules += ' -iXLvjd-S5gMYHd131';
    rules += ' TxTaQHd131';
    rules += ' QxTaQHd131';
    rules += ' S0SaQHd131';
    rules += ' R0SaQHd131';
    rules += ' _oa131';
    if (props.variant === "quiet") rules += ' _Le131';
    else if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    return rules;
};
let $68e4e6fe083e22fd$var$selectionIndicator = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' Wr131';
    rules += ' _lr131';
    rules += ' _Ar131';
    rules += ' _zr131';
    rules += ' _9d131';
    if (props.size === "S") rules += ' ob131';
    else if (props.size === "XS") rules += ' ob131';
    else rules += ' oc131';
    if (props.size === "S") rules += ' nb131';
    else if (props.size === "XS") rules += ' nb131';
    else rules += ' nc131';
    if (props.size === "S") rules += ' kb131';
    else if (props.size === "XS") rules += ' kb131';
    else rules += ' kc131';
    if (props.size === "S") rules += ' jb131';
    else if (props.size === "XS") rules += ' jb131';
    else rules += ' jc131';
    rules += ' _Ob131';
    rules += ' _kc131';
    rules += ' hc131';
    rules += ' mc131';
    rules += ' lc131';
    rules += ' _jf131';
    rules += ' ik131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.isSelected) rules += ' _Ib131';
    else rules += ' _Ia131';
    rules += ' OGcUfH131';
    rules += ' _Kc131';
    if (props.isStrokeInner) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' _Mc131';
    return rules;
};
let $68e4e6fe083e22fd$var$preview = function anonymous(props) {
    let rules = " ";
    rules += ' _Pc131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' _Nb131';
    rules += ' Pb131';
    rules += ' I8cyuNb131';
    rules += ' H8cyuNb131';
    rules += ' J39J4qb131';
    rules += ' Gk39J4qb131';
    if (props.isQuiet) {
        if (props.size === "S") rules += ' ob131';
        else if (props.size === "XS") rules += ' ob131';
        else rules += ' oc131';
    }
    if (props.isQuiet) {
        if (props.size === "S") rules += ' nb131';
        else if (props.size === "XS") rules += ' nb131';
        else rules += ' nc131';
    }
    if (props.isQuiet) {
        if (props.size === "S") rules += ' kb131';
        else if (props.size === "XS") rules += ' kb131';
        else rules += ' kc131';
    }
    if (props.isQuiet) {
        if (props.size === "S") rules += ' jb131';
        else if (props.size === "XS") rules += ' jb131';
        else rules += ' jc131';
    }
    if (props.isQuiet) {
        if (props.isSelected) rules += ' _nLeasBb131';
        else if (props.isFocusVisible) rules += ' _nLeasBb131';
        else if (props.isHovered) rules += ' _nLeasBb131';
    }
    if (props.isQuiet) {
        if (props.isFocusVisible) rules += ' _Lf131';
    } else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    return rules;
};
const $68e4e6fe083e22fd$var$image = function anonymous(props) {
    let rules = " ";
    rules += ' Za131';
    if (props.layout === "grid") rules += ' fc131';
    rules += ' _Hb131';
    rules += ' _4c131';
    rules += ' _Ob131';
    return rules;
};
let $68e4e6fe083e22fd$var$title = function anonymous(props) {
    let rules = " ";
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "M") {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    }
    rules += ' xd131';
    rules += ' _xa131';
    rules += ' xX0cczbc131';
    rules += ' _xX0cczba131';
    rules += ' xfd131';
    rules += ' _xfa131';
    rules += ' xfX0cczbc131';
    rules += ' _xfX0cczba131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    rules += ' po131';
    rules += ' __Ea131';
    rules += ' sa131';
    rules += ' __aa131';
    rules += ' __ba131';
    rules += ' zn9DOs131';
    rules += ' yn9DOs131';
    rules += ' Bn9DOs131';
    rules += ' An9DOs131';
    return rules;
};
let $68e4e6fe083e22fd$var$description = function anonymous(props) {
    let rules = " ";
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    } else if (props.size === "M") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    }
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fb131';
    rules += ' _FnuYUwec131';
    rules += ' pt131';
    rules += ' __Ea131';
    rules += ' sa131';
    rules += ' __aa131';
    rules += ' __ba131';
    rules += ' z9u7zPc131';
    rules += ' y9u7zPc131';
    rules += ' B9u7zPc131';
    rules += ' A9u7zPc131';
    return rules;
};
let $68e4e6fe083e22fd$var$content = function anonymous(props) {
    let rules = " ";
    rules += ' se131';
    rules += ' DM8Mfn131';
    rules += ' DlsIuJI4ff4131';
    rules += ' CfLapxc131';
    rules += ' ClsIuJg9IhLd131';
    rules += ' qb131';
    rules += ' _ub131';
    rules += ' ea131';
    rules += ' _cf131';
    if (props.size === "XL") rules += ' Ue131';
    else if (props.size === "L") rules += ' UWJpoB131';
    else if (props.size === "M") rules += ' UWJpoB131';
    else if (props.size === "S") rules += ' Ub131';
    else if (props.size === "XS") rules += ' Ub131';
    rules += ' T5gMYHd131';
    rules += ' Tbd131';
    rules += ' Q69YG2b131';
    rules += ' Qkd131';
    return rules;
};
let $68e4e6fe083e22fd$var$actionMenu = " zgSHDrc131 ygSHDrc131 BgSHDrc131 AgSHDrc131 JTnYEIe131 GTnYEIe131";
let $68e4e6fe083e22fd$var$footer = " sd131 _tc131 ec131 _Ce131 Ue131 qe131 T69YG2b131";
const $68e4e6fe083e22fd$export$cb658fed5fefe1d = /*#__PURE__*/ (0, $jzFZE$createContext)({
    ElementType: 'div',
    layout: 'grid'
});
const $68e4e6fe083e22fd$export$d0b2ee33ebf7d64 = /*#__PURE__*/ (0, $jzFZE$createContext)(null);
const $68e4e6fe083e22fd$var$InternalCardContext = /*#__PURE__*/ (0, $jzFZE$createContext)({
    isQuiet: false,
    size: 'M',
    isSelected: false,
    isHovered: false,
    isFocusVisible: false,
    isPressed: false,
    isCheckboxSelection: true
});
const $68e4e6fe083e22fd$var$actionButtonSize = {
    XS: 'XS',
    S: 'XS',
    M: 'S',
    L: 'M',
    XL: 'L'
};
const $68e4e6fe083e22fd$export$60332b2344f7fe41 = /*#__PURE__*/ (0, $jzFZE$forwardRef)(function Card(props, ref) {
    [props] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $68e4e6fe083e22fd$export$d0b2ee33ebf7d64);
    let { ElementType: ElementType, layout: layout } = (0, $jzFZE$useContext)($68e4e6fe083e22fd$export$cb658fed5fefe1d);
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let { density: density = 'regular', size: size = 'M', variant: variant = 'primary', UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, styles: styles, id: id, ...otherProps } = props;
    let isQuiet = variant === 'quiet';
    let isSkeleton = (0, $5ad421ec19460c48$export$4f8dc7555740235c)();
    let children = /*#__PURE__*/ (0, $jzFZE$jsx)((0, $jzFZE$Provider), {
        values: [
            [
                (0, $dbd6f0b2503b938c$export$8151eb379fcf2442),
                {
                    alt: '',
                    styles: $68e4e6fe083e22fd$var$image({
                        layout: layout
                    })
                }
            ],
            [
                (0, $8e847109a6ab556d$export$9afb8bc826b033ea),
                {
                    slots: {
                        [(0, $jzFZE$DEFAULT_SLOT)]: {},
                        title: {
                            styles: $68e4e6fe083e22fd$var$title({
                                size: size
                            })
                        },
                        description: {
                            styles: $68e4e6fe083e22fd$var$description({
                                size: size
                            })
                        }
                    }
                }
            ],
            [
                (0, $8e847109a6ab556d$export$1cbdd774077931b4),
                {
                    styles: $68e4e6fe083e22fd$var$content({
                        size: size
                    })
                }
            ],
            [
                (0, $bdf52657dac0f070$export$30fbfd376abdf190),
                {
                    size: 'S'
                }
            ],
            [
                (0, $8e847109a6ab556d$export$5630640b68817ed6),
                {
                    styles: $68e4e6fe083e22fd$var$footer
                }
            ],
            [
                (0, $8bf3cd2442eb485e$export$7f4b66acaca28bd5),
                {
                    isQuiet: true,
                    size: $68e4e6fe083e22fd$var$actionButtonSize[size],
                    isDisabled: isSkeleton,
                    // @ts-ignore
                    'data-slot': 'menu',
                    styles: $68e4e6fe083e22fd$var$actionMenu
                }
            ],
            [
                (0, $5ad421ec19460c48$export$74e166679b1f49ee),
                isSkeleton
            ]
        ],
        children: /*#__PURE__*/ (0, $jzFZE$jsx)((0, $4b5e069e9e001e8b$export$1b926c015f09611d), {
            children: typeof props.children === 'function' ? props.children({
                size: size
            }) : props.children
        })
    });
    let press = (0, $10ea7662e51a285b$export$56e8cba416805d8d)(domRef, UNSAFE_style);
    if (ElementType === 'div' && !isSkeleton && props.href) // Standalone Card that has an href should be rendered as a Link.
    // NOTE: In this case, the card must not contain interactive elements.
    return /*#__PURE__*/ (0, $jzFZE$jsx)((0, $jzFZE$Link), {
        ...(0, $jzFZE$filterDOMProps)(otherProps, {
            isLink: true
        }),
        ref: domRef,
        className: (renderProps)=>UNSAFE_className + $68e4e6fe083e22fd$var$card({
                ...renderProps,
                size: size,
                density: density,
                variant: variant,
                isCardView: false,
                isLink: true
            }, styles),
        style: (renderProps)=>// Only the preview in quiet cards scales down on press
            variant === 'quiet' ? UNSAFE_style : press(renderProps),
        children: (renderProps)=>/*#__PURE__*/ (0, $jzFZE$jsx)($68e4e6fe083e22fd$var$InternalCardContext.Provider, {
                value: {
                    size: size,
                    isQuiet: isQuiet,
                    isCheckboxSelection: false,
                    isSelected: false,
                    ...renderProps
                },
                children: children
            })
    });
    if (ElementType === 'div' || isSkeleton) return /*#__PURE__*/ (0, $jzFZE$jsx)("div", {
        ...(0, $jzFZE$filterDOMProps)(otherProps),
        id: id != null ? String(id) : undefined,
        // @ts-ignore - React < 19 compat
        inert: (0, $jzFZE$inertValue)(isSkeleton),
        ref: domRef,
        className: UNSAFE_className + $68e4e6fe083e22fd$var$card({
            size: size,
            density: density,
            variant: variant,
            isCardView: ElementType !== 'div'
        }, styles),
        style: UNSAFE_style,
        children: /*#__PURE__*/ (0, $jzFZE$jsx)($68e4e6fe083e22fd$var$InternalCardContext.Provider, {
            value: {
                size: size,
                isQuiet: isQuiet,
                isCheckboxSelection: false,
                isHovered: false,
                isFocusVisible: false,
                isSelected: false,
                isPressed: false
            },
            children: children
        })
    });
    return /*#__PURE__*/ (0, $jzFZE$jsx)(ElementType, {
        ...props,
        ref: domRef,
        className: (renderProps)=>UNSAFE_className + $68e4e6fe083e22fd$var$card({
                ...renderProps,
                isCardView: true,
                isLink: !!props.href,
                size: size,
                density: density,
                variant: variant
            }, styles),
        style: (renderProps)=>// Only the preview in quiet cards scales down on press
            variant === 'quiet' ? UNSAFE_style : press(renderProps),
        children: ({ selectionMode: selectionMode, selectionBehavior: selectionBehavior, isHovered: isHovered, isFocusVisible: isFocusVisible, isSelected: isSelected, isPressed: isPressed })=>/*#__PURE__*/ (0, $jzFZE$jsxs)($68e4e6fe083e22fd$var$InternalCardContext.Provider, {
                value: {
                    size: size,
                    isQuiet: isQuiet,
                    isCheckboxSelection: selectionMode !== 'none' && selectionBehavior === 'toggle',
                    isHovered: isHovered,
                    isFocusVisible: isFocusVisible,
                    isSelected: isSelected,
                    isPressed: isPressed
                },
                children: [
                    !isQuiet && /*#__PURE__*/ (0, $jzFZE$jsx)($68e4e6fe083e22fd$var$SelectionIndicator, {}),
                    !isQuiet && selectionMode !== 'none' && selectionBehavior === 'toggle' && /*#__PURE__*/ (0, $jzFZE$jsx)($68e4e6fe083e22fd$var$CardCheckbox, {}),
                    /*#__PURE__*/ (0, $jzFZE$jsx)("div", {
                        className: " sc131",
                        children: children
                    })
                ]
            })
    });
});
function $68e4e6fe083e22fd$var$SelectionIndicator() {
    let { size: size, isSelected: isSelected, isQuiet: isQuiet, isCheckboxSelection: isCheckboxSelection } = (0, $jzFZE$useContext)($68e4e6fe083e22fd$var$InternalCardContext);
    return /*#__PURE__*/ (0, $jzFZE$jsx)("div", {
        className: $68e4e6fe083e22fd$var$selectionIndicator({
            size: size,
            isSelected: isSelected,
            // Add an inner stroke only for quiet cards with no checkbox to
            // help distinguish the selected state from the preview.
            isStrokeInner: isQuiet && !isCheckboxSelection
        })
    });
}
function $68e4e6fe083e22fd$var$CardCheckbox() {
    let { size: size } = (0, $jzFZE$useContext)($68e4e6fe083e22fd$var$InternalCardContext);
    return /*#__PURE__*/ (0, $jzFZE$jsx)("div", {
        className: " _Pa131 W5gMYHd131 _A5gMYHd131 _9d131 Tp131 Qp131 Sp131 Rp131 gGcUfH131 ob131 nb131 kb131 jb131 _n4zCLMe131",
        children: /*#__PURE__*/ (0, $jzFZE$jsx)((0, $9b405e0f8095dce0$export$48513f6b9f8ce62d), {
            slot: "selection",
            excludeFromTabOrder: true,
            size: size === 'XS' ? 'S' : size
        })
    });
}
const $68e4e6fe083e22fd$export$115f017852a4bb7c = /*#__PURE__*/ (0, $jzFZE$forwardRef)(function CardPreview(props, ref) {
    let { size: size, isQuiet: isQuiet, isHovered: isHovered, isFocusVisible: isFocusVisible, isSelected: isSelected, isPressed: isPressed, isCheckboxSelection: isCheckboxSelection } = (0, $jzFZE$useContext)($68e4e6fe083e22fd$var$InternalCardContext);
    let { UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    return /*#__PURE__*/ (0, $jzFZE$jsxs)("div", {
        ...(0, $jzFZE$filterDOMProps)(props),
        slot: "preview",
        ref: domRef,
        className: UNSAFE_className + $68e4e6fe083e22fd$var$preview({
            size: size,
            isQuiet: isQuiet,
            isHovered: isHovered,
            isFocusVisible: isFocusVisible,
            isSelected: isSelected
        }),
        style: isQuiet ? (0, $10ea7662e51a285b$export$56e8cba416805d8d)(domRef)({
            isPressed: isPressed
        }) : UNSAFE_style,
        children: [
            isQuiet && /*#__PURE__*/ (0, $jzFZE$jsx)($68e4e6fe083e22fd$var$SelectionIndicator, {}),
            isQuiet && isCheckboxSelection && /*#__PURE__*/ (0, $jzFZE$jsx)($68e4e6fe083e22fd$var$CardCheckbox, {}),
            /*#__PURE__*/ (0, $jzFZE$jsx)("div", {
                className: " oi131 ni131 ki131 ji131 _Nb131 Pb131",
                children: props.children
            })
        ]
    });
});
const $68e4e6fe083e22fd$var$collection = function anonymous(props) {
    let rules = " ";
    rules += ' se131';
    rules += ' DvhgoHd131';
    if (props.size === "S") rules += ' Ua131';
    else if (props.size === "XS") rules += ' Ua131';
    else rules += ' Ub131';
    if (props.size === "S") rules += ' qa131';
    else if (props.size === "XS") rules += ' qa131';
    else rules += ' qb131';
    return rules;
};
const $68e4e6fe083e22fd$var$collectionImage = " Za131 fa131 f5DbKFc131 y5DbKFwoofG131 _Hb131 _Ob131 _4c131";
const $68e4e6fe083e22fd$export$c368ea3e577b1b86 = /*#__PURE__*/ (0, $jzFZE$forwardRef)(function CollectionCardPreview(props, ref) {
    let { size: size } = (0, $jzFZE$useContext)($68e4e6fe083e22fd$var$InternalCardContext);
    return /*#__PURE__*/ (0, $jzFZE$jsx)($68e4e6fe083e22fd$export$115f017852a4bb7c, {
        ...props,
        ref: ref,
        children: /*#__PURE__*/ (0, $jzFZE$jsx)("div", {
            className: $68e4e6fe083e22fd$var$collection({
                size: size
            }),
            children: /*#__PURE__*/ (0, $jzFZE$jsx)((0, $dbd6f0b2503b938c$export$8151eb379fcf2442).Provider, {
                value: {
                    styles: $68e4e6fe083e22fd$var$collectionImage
                },
                children: props.children
            })
        })
    });
});
const $68e4e6fe083e22fd$export$eff93e4e59ccc3b8 = /*#__PURE__*/ (0, $jzFZE$forwardRef)(function AssetCard(props1, ref) {
    let { layout: layout } = (0, $jzFZE$useContext)($68e4e6fe083e22fd$export$cb658fed5fefe1d);
    return /*#__PURE__*/ (0, $jzFZE$jsx)($68e4e6fe083e22fd$export$60332b2344f7fe41, {
        ...props1,
        ref: ref,
        density: "regular",
        children: (0, $jzFZE$composeRenderProps)(props1.children, (children)=>/*#__PURE__*/ (0, $jzFZE$jsx)((0, $jzFZE$Provider), {
                values: [
                    [
                        (0, $dbd6f0b2503b938c$export$8151eb379fcf2442),
                        {
                            alt: '',
                            styles: function anonymous(props) {
                                let rules = " ";
                                rules += ' Za131';
                                if (props.layout === "grid") rules += ' fa131';
                                rules += ' _Ha131';
                                rules += ' _Ob131';
                                rules += ' _4c131';
                                return rules;
                            }({
                                layout: layout
                            })
                        }
                    ],
                    [
                        (0, $ac8c32e6775ddd1f$export$207c7561471de4bd),
                        {
                            render (icon) {
                                return /*#__PURE__*/ (0, $jzFZE$jsx)((0, $5ad421ec19460c48$export$6069cbe61f690103), {
                                    children: /*#__PURE__*/ (0, $jzFZE$jsx)("div", {
                                        className: " sd131 eb131 _Ca131 gH131 fa131",
                                        children: icon
                                    })
                                });
                            },
                            styles: " Fd131 Le131 Ke131 Zc131"
                        }
                    ]
                ],
                children: children
            }))
    });
});
const $68e4e6fe083e22fd$var$avatarSize = {
    XS: 24,
    S: 48,
    M: 64,
    L: 64,
    XL: 80
};
const $68e4e6fe083e22fd$export$78e8870b5d8ae469 = /*#__PURE__*/ (0, $jzFZE$forwardRef)(function UserCard(props, ref) {
    let { size: size = 'M' } = props;
    return /*#__PURE__*/ (0, $jzFZE$jsx)($68e4e6fe083e22fd$export$60332b2344f7fe41, {
        ...props,
        ref: ref,
        density: "spacious",
        children: (0, $jzFZE$composeRenderProps)(props.children, (children)=>/*#__PURE__*/ (0, $jzFZE$jsx)((0, $jzFZE$Provider), {
                values: [
                    [
                        (0, $dbd6f0b2503b938c$export$8151eb379fcf2442),
                        {
                            alt: '',
                            styles: " Za131 fb131 _Hb131 _Ob131 _4c131"
                        }
                    ],
                    [
                        (0, $3ffa2cd930156220$export$a20dad690e1279e2),
                        {
                            size: $68e4e6fe083e22fd$var$avatarSize[size],
                            UNSAFE_style: {
                                '--size': $68e4e6fe083e22fd$var$avatarSize[size] / 16 + 'rem'
                            },
                            styles: " _Pc131 Jy131 Jkymupd0fnV8d131",
                            isOverBackground: true
                        }
                    ]
                ],
                children: children
            }))
    });
});
const $68e4e6fe083e22fd$var$buttonSize = {
    XS: 'S',
    S: 'S',
    M: 'M',
    L: 'L',
    XL: 'XL'
};
const $68e4e6fe083e22fd$export$e6cbe35324ccf844 = /*#__PURE__*/ (0, $jzFZE$forwardRef)(function ProductCard(props1, ref) {
    let { size: size = 'M' } = props1;
    return /*#__PURE__*/ (0, $jzFZE$jsx)($68e4e6fe083e22fd$export$60332b2344f7fe41, {
        ...props1,
        ref: ref,
        density: "spacious",
        children: (0, $jzFZE$composeRenderProps)(props1.children, (children)=>/*#__PURE__*/ (0, $jzFZE$jsx)((0, $jzFZE$Provider), {
                values: [
                    [
                        (0, $dbd6f0b2503b938c$export$8151eb379fcf2442),
                        {
                            slots: {
                                preview: {
                                    alt: '',
                                    styles: " Za131 fd131 _Hb131 _Ob131 _4c131"
                                },
                                thumbnail: {
                                    alt: '',
                                    styles: function anonymous(props) {
                                        let rules = " ";
                                        rules += ' _Pc131';
                                        rules += ' _Ob131';
                                        rules += ' _4c131';
                                        if (props.size === "XL") rules += ' ZD131';
                                        else if (props.size === "L") rules += ' Zx131';
                                        else if (props.size === "M") rules += ' Zw131';
                                        else if (props.size === "S") rules += ' Zv131';
                                        else if (props.size === "XS") rules += ' Zo131';
                                        rules += ' FRuFGYc131';
                                        if (props.size === "S") rules += ' oa131';
                                        else if (props.size === "XS") rules += ' oa131';
                                        else rules += ' ob131';
                                        if (props.size === "S") rules += ' na131';
                                        else if (props.size === "XS") rules += ' na131';
                                        else rules += ' nb131';
                                        if (props.size === "S") rules += ' ka131';
                                        else if (props.size === "XS") rules += ' ka131';
                                        else rules += ' kb131';
                                        if (props.size === "S") rules += ' ja131';
                                        else if (props.size === "XS") rules += ' ja131';
                                        else rules += ' jb131';
                                        rules += ' _Hb131';
                                        rules += ' Jy131';
                                        rules += ' JkymupdLu7Bcc131';
                                        rules += ' _Lf131';
                                        if (props.size === "XS") rules += ' _Mb131';
                                        else rules += ' _Mc131';
                                        rules += ' OEzkoQe131';
                                        if (props.size === "XL") rules += ' -F_-Fy131';
                                        else if (props.size === "L") rules += ' -F_-Fw131';
                                        else if (props.size === "M") rules += ' -F_-Fv131';
                                        else if (props.size === "S") rules += ' -F_-Fu131';
                                        else if (props.size === "XS") rules += ' -F_-Fp131';
                                        return rules;
                                    }({
                                        size: size
                                    })
                                }
                            }
                        }
                    ],
                    [
                        (0, $8e847109a6ab556d$export$5630640b68817ed6),
                        {
                            styles: (0, $feb886035e0d4633$export$e618dc39ac9ad607)($68e4e6fe083e22fd$var$footer, " _Cb131")
                        }
                    ],
                    [
                        (0, $067ea9f64ccd4e8e$export$24d547caef80ccd1),
                        {
                            size: $68e4e6fe083e22fd$var$buttonSize[size]
                        }
                    ],
                    [
                        (0, $067ea9f64ccd4e8e$export$1c1a176f3b9e48d3),
                        {
                            size: $68e4e6fe083e22fd$var$buttonSize[size]
                        }
                    ]
                ],
                children: children
            }))
    });
});


export {$68e4e6fe083e22fd$export$cb658fed5fefe1d as InternalCardViewContext, $68e4e6fe083e22fd$export$d0b2ee33ebf7d64 as CardContext, $68e4e6fe083e22fd$export$60332b2344f7fe41 as Card, $68e4e6fe083e22fd$export$115f017852a4bb7c as CardPreview, $68e4e6fe083e22fd$export$c368ea3e577b1b86 as CollectionCardPreview, $68e4e6fe083e22fd$export$eff93e4e59ccc3b8 as AssetCard, $68e4e6fe083e22fd$export$78e8870b5d8ae469 as UserCard, $68e4e6fe083e22fd$export$e6cbe35324ccf844 as ProductCard};
