// @react-spectrum/s2/dist/private/Card.cjs
require("./Card.css");
var $e5a758dd286599f2$exports = require("./ActionMenu.cjs");
var $38071d9ce246d4cf$exports = require("./Avatar.cjs");
var $25d06cf8d4e72761$exports = require("./Button.cjs");
var $af04e099a53d3e85$exports = require("./Checkbox.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $2d04acb918636de0$exports = require("./Divider.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $053b76ed3d29e13b$exports = require("./Image.cjs");
var $a4f1585b527b9b7a$exports = require("./ImageCoordinator.cjs");
var $308b180f49d82d28$exports = require("../../icons/runtime.cjs");
var $2061c83559b50a66$exports = require("./pressScale.cjs");
var $5eb75e0c130e0669$exports = require("../../icons/Skeleton.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $1B7TP$reactjsxruntime = require("react/jsx-runtime");
var $1B7TP$reactariacomponentscomposeRenderProps = require("react-aria-components/composeRenderProps");
var $1B7TP$reactariacomponentsslots = require("react-aria-components/slots");
var $1B7TP$react = require("react");
var $1B7TP$reactariafilterDOMProps = require("react-aria/filterDOMProps");
var $1B7TP$reactariaprivateutilsinertValue = require("react-aria/private/utils/inertValue");
var $1B7TP$reactariacomponentsLink = require("react-aria-components/Link");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "InternalCardViewContext", () => $230078a1c4ce81d8$export$cb658fed5fefe1d);
$parcel$export(module.exports, "CardContext", () => $230078a1c4ce81d8$export$d0b2ee33ebf7d64);
$parcel$export(module.exports, "Card", () => $230078a1c4ce81d8$export$60332b2344f7fe41);
$parcel$export(module.exports, "CardPreview", () => $230078a1c4ce81d8$export$115f017852a4bb7c);
$parcel$export(module.exports, "CollectionCardPreview", () => $230078a1c4ce81d8$export$c368ea3e577b1b86);
$parcel$export(module.exports, "AssetCard", () => $230078a1c4ce81d8$export$eff93e4e59ccc3b8);
$parcel$export(module.exports, "UserCard", () => $230078a1c4ce81d8$export$78e8870b5d8ae469);
$parcel$export(module.exports, "ProductCard", () => $230078a1c4ce81d8$export$e6cbe35324ccf844);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 




















const $230078a1c4ce81d8$var$borderRadius = {
    default: 'lg',
    size: {
        XS: 'default',
        S: 'default'
    }
};
let $230078a1c4ce81d8$var$card = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let width = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "Z") width = true;
        rules += p[0];
    }
    rules += ' sd131';
    rules += ' _ta131';
    if (!position) rules += ' _Pc131';
    if (props.size === "S") rules += ' ob131';
    else if (props.size === "XS") rules += ' ob131';
    else rules += ' oc131';
    if (props.size === "S") rules += ' nb131';
    else if (props.size === "XS") rules += ' nb131';
    else rules += ' nc131';
    if (props.size === "S") rules += ' kb131';
    else if (props.size === "XS") rules += ' kb131';
    else rules += ' kc131';
    if (props.size === "S") rules += ' jb131';
    else if (props.size === "XS") rules += ' jb131';
    else rules += ' jc131';
    if (props.variant === "secondary") rules += ' -Ovbhqd-gqlRAZb131';
    else if (props.variant === "primary") rules += ' -Ovbhqd-g_g131';
    rules += ' -Ovbhqd-lgb131';
    if (props.variant === "quiet") rules += ' g9131';
    else if (props.variant === "tertiary") rules += ' g9131';
    else rules += ' gEzkoQe131';
    if (props.variant === "quiet") rules += ' _nd131';
    else if (props.variant === "tertiary") {
        if (props.isSelected) rules += ' _nd131';
        else if (props.isFocusVisible) rules += ' _nrC99Ne131';
        else if (props.isHovered) rules += ' _nrC99Ne131';
        else rules += ' _nnvmtGb131';
        rules += ' _nli6j2Ec131';
    } else {
        if (props.isSelected) rules += ' _nLeasBb131';
        else if (props.isFocusVisible) rules += ' _nLeasBb131';
        else if (props.isHovered) rules += ' _nLeasBb131';
        else rules += ' _n4zCLMe131';
        rules += ' _nlJWldjc131';
    }
    rules += ' _yb131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' _Xa131';
    if (props.variant === "quiet") rules += ' _Ne131';
    else rules += ' _Nb131';
    if (props.variant === "quiet") rules += ' Pe131';
    else rules += ' Pb131';
    rules += ' _qc131';
    rules += ' __ca131';
    if (props.isCardView) rules += ' _4c131';
    if (props.isLink) rules += ' ry131';
    if (props.isCardView) {
        if (!width) rules += ' Za131';
    } else if (props.size === "XL") {
        if (!width) rules += ' ZA131';
    } else if (props.size === "L") {
        if (!width) rules += ' Zy131';
    } else if (props.size === "M") {
        if (!width) rules += ' Zt131';
    } else if (props.size === "S") {
        if (!width) rules += ' Zr131';
    } else if (props.size === "XS") {
        if (!width) rules += ' ZI131';
    }
    rules += ' Fb131';
    if (props.density === "spacious") {
        if (props.size === "XL") rules += ' -dAV6Jb-Ti131';
        else if (props.size === "L") rules += ' -dAV6Jb-Th131';
        else if (props.size === "M") rules += ' -dAV6Jb-Tg131';
        else if (props.size === "S") rules += ' -dAV6Jb-Tf131';
        else if (props.size === "XS") rules += ' -dAV6Jb-Te131';
    } else if (props.density === "regular") {
        if (props.size === "XL") rules += ' -dAV6Jb-Th131';
        else if (props.size === "L") rules += ' -dAV6Jb-Tg131';
        else if (props.size === "M") rules += ' -dAV6Jb-Tf131';
        else if (props.size === "S") rules += ' -dAV6Jb-Te131';
        else if (props.size === "XS") rules += ' -dAV6Jb-Tt131';
    } else if (props.density === "compact") {
        if (props.size === "XL") rules += ' -dAV6Jb-Tg131';
        else if (props.size === "L") rules += ' -dAV6Jb-Tf131';
        else if (props.size === "M") rules += ' -dAV6Jb-Te131';
        else if (props.size === "S") rules += ' -dAV6Jb-Tt131';
        else if (props.size === "XS") rules += ' -dAV6Jb-T1DOfn131';
    }
    if (props.variant === "quiet") rules += ' -jXLvjd-Td131';
    else rules += ' -jXLvjd-T5gMYHd131';
    if (props.variant === "quiet") rules += ' -iXLvjd-Sd131';
    else rules += ' -iXLvjd-S5gMYHd131';
    rules += ' TxTaQHd131';
    rules += ' QxTaQHd131';
    rules += ' S0SaQHd131';
    rules += ' R0SaQHd131';
    rules += ' _oa131';
    if (props.variant === "quiet") rules += ' _Le131';
    else if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    return rules;
};
let $230078a1c4ce81d8$var$selectionIndicator = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' Wr131';
    rules += ' _lr131';
    rules += ' _Ar131';
    rules += ' _zr131';
    rules += ' _9d131';
    if (props.size === "S") rules += ' ob131';
    else if (props.size === "XS") rules += ' ob131';
    else rules += ' oc131';
    if (props.size === "S") rules += ' nb131';
    else if (props.size === "XS") rules += ' nb131';
    else rules += ' nc131';
    if (props.size === "S") rules += ' kb131';
    else if (props.size === "XS") rules += ' kb131';
    else rules += ' kc131';
    if (props.size === "S") rules += ' jb131';
    else if (props.size === "XS") rules += ' jb131';
    else rules += ' jc131';
    rules += ' _Ob131';
    rules += ' _kc131';
    rules += ' hc131';
    rules += ' mc131';
    rules += ' lc131';
    rules += ' _jf131';
    rules += ' ik131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.isSelected) rules += ' _Ib131';
    else rules += ' _Ia131';
    rules += ' OGcUfH131';
    rules += ' _Kc131';
    if (props.isStrokeInner) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' _Mc131';
    return rules;
};
let $230078a1c4ce81d8$var$preview = function anonymous(props) {
    let rules = " ";
    rules += ' _Pc131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' _Nb131';
    rules += ' Pb131';
    rules += ' I8cyuNb131';
    rules += ' H8cyuNb131';
    rules += ' J39J4qb131';
    rules += ' Gk39J4qb131';
    if (props.isQuiet) {
        if (props.size === "S") rules += ' ob131';
        else if (props.size === "XS") rules += ' ob131';
        else rules += ' oc131';
    }
    if (props.isQuiet) {
        if (props.size === "S") rules += ' nb131';
        else if (props.size === "XS") rules += ' nb131';
        else rules += ' nc131';
    }
    if (props.isQuiet) {
        if (props.size === "S") rules += ' kb131';
        else if (props.size === "XS") rules += ' kb131';
        else rules += ' kc131';
    }
    if (props.isQuiet) {
        if (props.size === "S") rules += ' jb131';
        else if (props.size === "XS") rules += ' jb131';
        else rules += ' jc131';
    }
    if (props.isQuiet) {
        if (props.isSelected) rules += ' _nLeasBb131';
        else if (props.isFocusVisible) rules += ' _nLeasBb131';
        else if (props.isHovered) rules += ' _nLeasBb131';
    }
    if (props.isQuiet) {
        if (props.isFocusVisible) rules += ' _Lf131';
    } else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    return rules;
};
const $230078a1c4ce81d8$var$image = function anonymous(props) {
    let rules = " ";
    rules += ' Za131';
    if (props.layout === "grid") rules += ' fc131';
    rules += ' _Hb131';
    rules += ' _4c131';
    rules += ' _Ob131';
    return rules;
};
let $230078a1c4ce81d8$var$title = function anonymous(props) {
    let rules = " ";
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "M") {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    }
    rules += ' xd131';
    rules += ' _xa131';
    rules += ' xX0cczbc131';
    rules += ' _xX0cczba131';
    rules += ' xfd131';
    rules += ' _xfa131';
    rules += ' xfX0cczbc131';
    rules += ' _xfX0cczba131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    rules += ' po131';
    rules += ' __Ea131';
    rules += ' sa131';
    rules += ' __aa131';
    rules += ' __ba131';
    rules += ' zn9DOs131';
    rules += ' yn9DOs131';
    rules += ' Bn9DOs131';
    rules += ' An9DOs131';
    return rules;
};
let $230078a1c4ce81d8$var$description = function anonymous(props) {
    let rules = " ";
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    } else if (props.size === "M") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    }
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fb131';
    rules += ' _FnuYUwec131';
    rules += ' pt131';
    rules += ' __Ea131';
    rules += ' sa131';
    rules += ' __aa131';
    rules += ' __ba131';
    rules += ' z9u7zPc131';
    rules += ' y9u7zPc131';
    rules += ' B9u7zPc131';
    rules += ' A9u7zPc131';
    return rules;
};
let $230078a1c4ce81d8$var$content = function anonymous(props) {
    let rules = " ";
    rules += ' se131';
    rules += ' DM8Mfn131';
    rules += ' DlsIuJI4ff4131';
    rules += ' CfLapxc131';
    rules += ' ClsIuJg9IhLd131';
    rules += ' qb131';
    rules += ' _ub131';
    rules += ' ea131';
    rules += ' _cf131';
    if (props.size === "XL") rules += ' Ue131';
    else if (props.size === "L") rules += ' UWJpoB131';
    else if (props.size === "M") rules += ' UWJpoB131';
    else if (props.size === "S") rules += ' Ub131';
    else if (props.size === "XS") rules += ' Ub131';
    rules += ' T5gMYHd131';
    rules += ' Tbd131';
    rules += ' Q69YG2b131';
    rules += ' Qkd131';
    return rules;
};
let $230078a1c4ce81d8$var$actionMenu = " zgSHDrc131 ygSHDrc131 BgSHDrc131 AgSHDrc131 JTnYEIe131 GTnYEIe131";
let $230078a1c4ce81d8$var$footer = " sd131 _tc131 ec131 _Ce131 Ue131 qe131 T69YG2b131";
const $230078a1c4ce81d8$export$cb658fed5fefe1d = /*#__PURE__*/ (0, $1B7TP$react.createContext)({
    ElementType: 'div',
    layout: 'grid'
});
const $230078a1c4ce81d8$export$d0b2ee33ebf7d64 = /*#__PURE__*/ (0, $1B7TP$react.createContext)(null);
const $230078a1c4ce81d8$var$InternalCardContext = /*#__PURE__*/ (0, $1B7TP$react.createContext)({
    isQuiet: false,
    size: 'M',
    isSelected: false,
    isHovered: false,
    isFocusVisible: false,
    isPressed: false,
    isCheckboxSelection: true
});
const $230078a1c4ce81d8$var$actionButtonSize = {
    XS: 'XS',
    S: 'XS',
    M: 'S',
    L: 'M',
    XL: 'L'
};
const $230078a1c4ce81d8$export$60332b2344f7fe41 = /*#__PURE__*/ (0, $1B7TP$react.forwardRef)(function Card(props, ref) {
    [props] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $230078a1c4ce81d8$export$d0b2ee33ebf7d64);
    let { ElementType: ElementType, layout: layout } = (0, $1B7TP$react.useContext)($230078a1c4ce81d8$export$cb658fed5fefe1d);
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let { density: density = 'regular', size: size = 'M', variant: variant = 'primary', UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, styles: styles, id: id, ...otherProps } = props;
    let isQuiet = variant === 'quiet';
    let isSkeleton = (0, $5eb75e0c130e0669$exports.useIsSkeleton)();
    let children = /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)((0, $1B7TP$reactariacomponentsslots.Provider), {
        values: [
            [
                (0, $053b76ed3d29e13b$exports.ImageContext),
                {
                    alt: '',
                    styles: $230078a1c4ce81d8$var$image({
                        layout: layout
                    })
                }
            ],
            [
                (0, $6367bc87eb7d24ad$exports.TextContext),
                {
                    slots: {
                        [(0, $1B7TP$reactariacomponentsslots.DEFAULT_SLOT)]: {},
                        title: {
                            styles: $230078a1c4ce81d8$var$title({
                                size: size
                            })
                        },
                        description: {
                            styles: $230078a1c4ce81d8$var$description({
                                size: size
                            })
                        }
                    }
                }
            ],
            [
                (0, $6367bc87eb7d24ad$exports.ContentContext),
                {
                    styles: $230078a1c4ce81d8$var$content({
                        size: size
                    })
                }
            ],
            [
                (0, $2d04acb918636de0$exports.DividerContext),
                {
                    size: 'S'
                }
            ],
            [
                (0, $6367bc87eb7d24ad$exports.FooterContext),
                {
                    styles: $230078a1c4ce81d8$var$footer
                }
            ],
            [
                (0, $e5a758dd286599f2$exports.ActionMenuContext),
                {
                    isQuiet: true,
                    size: $230078a1c4ce81d8$var$actionButtonSize[size],
                    isDisabled: isSkeleton,
                    // @ts-ignore
                    'data-slot': 'menu',
                    styles: $230078a1c4ce81d8$var$actionMenu
                }
            ],
            [
                (0, $5eb75e0c130e0669$exports.SkeletonContext),
                isSkeleton
            ]
        ],
        children: /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)((0, $a4f1585b527b9b7a$exports.ImageCoordinator), {
            children: typeof props.children === 'function' ? props.children({
                size: size
            }) : props.children
        })
    });
    let press = (0, $2061c83559b50a66$exports.pressScale)(domRef, UNSAFE_style);
    if (ElementType === 'div' && !isSkeleton && props.href) // Standalone Card that has an href should be rendered as a Link.
    // NOTE: In this case, the card must not contain interactive elements.
    return /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)((0, $1B7TP$reactariacomponentsLink.Link), {
        ...(0, $1B7TP$reactariafilterDOMProps.filterDOMProps)(otherProps, {
            isLink: true
        }),
        ref: domRef,
        className: (renderProps)=>UNSAFE_className + $230078a1c4ce81d8$var$card({
                ...renderProps,
                size: size,
                density: density,
                variant: variant,
                isCardView: false,
                isLink: true
            }, styles),
        style: (renderProps)=>// Only the preview in quiet cards scales down on press
            variant === 'quiet' ? UNSAFE_style : press(renderProps),
        children: (renderProps)=>/*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)($230078a1c4ce81d8$var$InternalCardContext.Provider, {
                value: {
                    size: size,
                    isQuiet: isQuiet,
                    isCheckboxSelection: false,
                    isSelected: false,
                    ...renderProps
                },
                children: children
            })
    });
    if (ElementType === 'div' || isSkeleton) return /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)("div", {
        ...(0, $1B7TP$reactariafilterDOMProps.filterDOMProps)(otherProps),
        id: id != null ? String(id) : undefined,
        // @ts-ignore - React < 19 compat
        inert: (0, $1B7TP$reactariaprivateutilsinertValue.inertValue)(isSkeleton),
        ref: domRef,
        className: UNSAFE_className + $230078a1c4ce81d8$var$card({
            size: size,
            density: density,
            variant: variant,
            isCardView: ElementType !== 'div'
        }, styles),
        style: UNSAFE_style,
        children: /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)($230078a1c4ce81d8$var$InternalCardContext.Provider, {
            value: {
                size: size,
                isQuiet: isQuiet,
                isCheckboxSelection: false,
                isHovered: false,
                isFocusVisible: false,
                isSelected: false,
                isPressed: false
            },
            children: children
        })
    });
    return /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)(ElementType, {
        ...props,
        ref: domRef,
        className: (renderProps)=>UNSAFE_className + $230078a1c4ce81d8$var$card({
                ...renderProps,
                isCardView: true,
                isLink: !!props.href,
                size: size,
                density: density,
                variant: variant
            }, styles),
        style: (renderProps)=>// Only the preview in quiet cards scales down on press
            variant === 'quiet' ? UNSAFE_style : press(renderProps),
        children: ({ selectionMode: selectionMode, selectionBehavior: selectionBehavior, isHovered: isHovered, isFocusVisible: isFocusVisible, isSelected: isSelected, isPressed: isPressed })=>/*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsxs)($230078a1c4ce81d8$var$InternalCardContext.Provider, {
                value: {
                    size: size,
                    isQuiet: isQuiet,
                    isCheckboxSelection: selectionMode !== 'none' && selectionBehavior === 'toggle',
                    isHovered: isHovered,
                    isFocusVisible: isFocusVisible,
                    isSelected: isSelected,
                    isPressed: isPressed
                },
                children: [
                    !isQuiet && /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)($230078a1c4ce81d8$var$SelectionIndicator, {}),
                    !isQuiet && selectionMode !== 'none' && selectionBehavior === 'toggle' && /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)($230078a1c4ce81d8$var$CardCheckbox, {}),
                    /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)("div", {
                        className: " sc131",
                        children: children
                    })
                ]
            })
    });
});
function $230078a1c4ce81d8$var$SelectionIndicator() {
    let { size: size, isSelected: isSelected, isQuiet: isQuiet, isCheckboxSelection: isCheckboxSelection } = (0, $1B7TP$react.useContext)($230078a1c4ce81d8$var$InternalCardContext);
    return /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)("div", {
        className: $230078a1c4ce81d8$var$selectionIndicator({
            size: size,
            isSelected: isSelected,
            // Add an inner stroke only for quiet cards with no checkbox to
            // help distinguish the selected state from the preview.
            isStrokeInner: isQuiet && !isCheckboxSelection
        })
    });
}
function $230078a1c4ce81d8$var$CardCheckbox() {
    let { size: size } = (0, $1B7TP$react.useContext)($230078a1c4ce81d8$var$InternalCardContext);
    return /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)("div", {
        className: " _Pa131 W5gMYHd131 _A5gMYHd131 _9d131 Tp131 Qp131 Sp131 Rp131 gGcUfH131 ob131 nb131 kb131 jb131 _n4zCLMe131",
        children: /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)((0, $af04e099a53d3e85$exports.Checkbox), {
            slot: "selection",
            excludeFromTabOrder: true,
            size: size === 'XS' ? 'S' : size
        })
    });
}
const $230078a1c4ce81d8$export$115f017852a4bb7c = /*#__PURE__*/ (0, $1B7TP$react.forwardRef)(function CardPreview(props, ref) {
    let { size: size, isQuiet: isQuiet, isHovered: isHovered, isFocusVisible: isFocusVisible, isSelected: isSelected, isPressed: isPressed, isCheckboxSelection: isCheckboxSelection } = (0, $1B7TP$react.useContext)($230078a1c4ce81d8$var$InternalCardContext);
    let { UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style } = props;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    return /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsxs)("div", {
        ...(0, $1B7TP$reactariafilterDOMProps.filterDOMProps)(props),
        slot: "preview",
        ref: domRef,
        className: UNSAFE_className + $230078a1c4ce81d8$var$preview({
            size: size,
            isQuiet: isQuiet,
            isHovered: isHovered,
            isFocusVisible: isFocusVisible,
            isSelected: isSelected
        }),
        style: isQuiet ? (0, $2061c83559b50a66$exports.pressScale)(domRef)({
            isPressed: isPressed
        }) : UNSAFE_style,
        children: [
            isQuiet && /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)($230078a1c4ce81d8$var$SelectionIndicator, {}),
            isQuiet && isCheckboxSelection && /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)($230078a1c4ce81d8$var$CardCheckbox, {}),
            /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)("div", {
                className: " oi131 ni131 ki131 ji131 _Nb131 Pb131",
                children: props.children
            })
        ]
    });
});
const $230078a1c4ce81d8$var$collection = function anonymous(props) {
    let rules = " ";
    rules += ' se131';
    rules += ' DvhgoHd131';
    if (props.size === "S") rules += ' Ua131';
    else if (props.size === "XS") rules += ' Ua131';
    else rules += ' Ub131';
    if (props.size === "S") rules += ' qa131';
    else if (props.size === "XS") rules += ' qa131';
    else rules += ' qb131';
    return rules;
};
const $230078a1c4ce81d8$var$collectionImage = " Za131 fa131 f5DbKFc131 y5DbKFwoofG131 _Hb131 _Ob131 _4c131";
const $230078a1c4ce81d8$export$c368ea3e577b1b86 = /*#__PURE__*/ (0, $1B7TP$react.forwardRef)(function CollectionCardPreview(props, ref) {
    let { size: size } = (0, $1B7TP$react.useContext)($230078a1c4ce81d8$var$InternalCardContext);
    return /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)($230078a1c4ce81d8$export$115f017852a4bb7c, {
        ...props,
        ref: ref,
        children: /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)("div", {
            className: $230078a1c4ce81d8$var$collection({
                size: size
            }),
            children: /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)((0, $053b76ed3d29e13b$exports.ImageContext).Provider, {
                value: {
                    styles: $230078a1c4ce81d8$var$collectionImage
                },
                children: props.children
            })
        })
    });
});
const $230078a1c4ce81d8$export$eff93e4e59ccc3b8 = /*#__PURE__*/ (0, $1B7TP$react.forwardRef)(function AssetCard(props1, ref) {
    let { layout: layout } = (0, $1B7TP$react.useContext)($230078a1c4ce81d8$export$cb658fed5fefe1d);
    return /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)($230078a1c4ce81d8$export$60332b2344f7fe41, {
        ...props1,
        ref: ref,
        density: "regular",
        children: (0, $1B7TP$reactariacomponentscomposeRenderProps.composeRenderProps)(props1.children, (children)=>/*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)((0, $1B7TP$reactariacomponentsslots.Provider), {
                values: [
                    [
                        (0, $053b76ed3d29e13b$exports.ImageContext),
                        {
                            alt: '',
                            styles: function anonymous(props) {
                                let rules = " ";
                                rules += ' Za131';
                                if (props.layout === "grid") rules += ' fa131';
                                rules += ' _Ha131';
                                rules += ' _Ob131';
                                rules += ' _4c131';
                                return rules;
                            }({
                                layout: layout
                            })
                        }
                    ],
                    [
                        (0, $bde97c91243ed164$exports.IllustrationContext),
                        {
                            render (icon) {
                                return /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)((0, $5eb75e0c130e0669$exports.SkeletonWrapper), {
                                    children: /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)("div", {
                                        className: " sd131 eb131 _Ca131 gH131 fa131",
                                        children: icon
                                    })
                                });
                            },
                            styles: " Fd131 Le131 Ke131 Zc131"
                        }
                    ]
                ],
                children: children
            }))
    });
});
const $230078a1c4ce81d8$var$avatarSize = {
    XS: 24,
    S: 48,
    M: 64,
    L: 64,
    XL: 80
};
const $230078a1c4ce81d8$export$78e8870b5d8ae469 = /*#__PURE__*/ (0, $1B7TP$react.forwardRef)(function UserCard(props, ref) {
    let { size: size = 'M' } = props;
    return /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)($230078a1c4ce81d8$export$60332b2344f7fe41, {
        ...props,
        ref: ref,
        density: "spacious",
        children: (0, $1B7TP$reactariacomponentscomposeRenderProps.composeRenderProps)(props.children, (children)=>/*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)((0, $1B7TP$reactariacomponentsslots.Provider), {
                values: [
                    [
                        (0, $053b76ed3d29e13b$exports.ImageContext),
                        {
                            alt: '',
                            styles: " Za131 fb131 _Hb131 _Ob131 _4c131"
                        }
                    ],
                    [
                        (0, $38071d9ce246d4cf$exports.AvatarContext),
                        {
                            size: $230078a1c4ce81d8$var$avatarSize[size],
                            UNSAFE_style: {
                                '--size': $230078a1c4ce81d8$var$avatarSize[size] / 16 + 'rem'
                            },
                            styles: " _Pc131 Jy131 Jkymupd0fnV8d131",
                            isOverBackground: true
                        }
                    ]
                ],
                children: children
            }))
    });
});
const $230078a1c4ce81d8$var$buttonSize = {
    XS: 'S',
    S: 'S',
    M: 'M',
    L: 'L',
    XL: 'XL'
};
const $230078a1c4ce81d8$export$e6cbe35324ccf844 = /*#__PURE__*/ (0, $1B7TP$react.forwardRef)(function ProductCard(props1, ref) {
    let { size: size = 'M' } = props1;
    return /*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)($230078a1c4ce81d8$export$60332b2344f7fe41, {
        ...props1,
        ref: ref,
        density: "spacious",
        children: (0, $1B7TP$reactariacomponentscomposeRenderProps.composeRenderProps)(props1.children, (children)=>/*#__PURE__*/ (0, $1B7TP$reactjsxruntime.jsx)((0, $1B7TP$reactariacomponentsslots.Provider), {
                values: [
                    [
                        (0, $053b76ed3d29e13b$exports.ImageContext),
                        {
                            slots: {
                                preview: {
                                    alt: '',
                                    styles: " Za131 fd131 _Hb131 _Ob131 _4c131"
                                },
                                thumbnail: {
                                    alt: '',
                                    styles: function anonymous(props) {
                                        let rules = " ";
                                        rules += ' _Pc131';
                                        rules += ' _Ob131';
                                        rules += ' _4c131';
                                        if (props.size === "XL") rules += ' ZD131';
                                        else if (props.size === "L") rules += ' Zx131';
                                        else if (props.size === "M") rules += ' Zw131';
                                        else if (props.size === "S") rules += ' Zv131';
                                        else if (props.size === "XS") rules += ' Zo131';
                                        rules += ' FRuFGYc131';
                                        if (props.size === "S") rules += ' oa131';
                                        else if (props.size === "XS") rules += ' oa131';
                                        else rules += ' ob131';
                                        if (props.size === "S") rules += ' na131';
                                        else if (props.size === "XS") rules += ' na131';
                                        else rules += ' nb131';
                                        if (props.size === "S") rules += ' ka131';
                                        else if (props.size === "XS") rules += ' ka131';
                                        else rules += ' kb131';
                                        if (props.size === "S") rules += ' ja131';
                                        else if (props.size === "XS") rules += ' ja131';
                                        else rules += ' jb131';
                                        rules += ' _Hb131';
                                        rules += ' Jy131';
                                        rules += ' JkymupdLu7Bcc131';
                                        rules += ' _Lf131';
                                        if (props.size === "XS") rules += ' _Mb131';
                                        else rules += ' _Mc131';
                                        rules += ' OEzkoQe131';
                                        if (props.size === "XL") rules += ' -F_-Fy131';
                                        else if (props.size === "L") rules += ' -F_-Fw131';
                                        else if (props.size === "M") rules += ' -F_-Fv131';
                                        else if (props.size === "S") rules += ' -F_-Fu131';
                                        else if (props.size === "XS") rules += ' -F_-Fp131';
                                        return rules;
                                    }({
                                        size: size
                                    })
                                }
                            }
                        }
                    ],
                    [
                        (0, $6367bc87eb7d24ad$exports.FooterContext),
                        {
                            styles: (0, $308b180f49d82d28$exports.mergeStyles)($230078a1c4ce81d8$var$footer, " _Cb131")
                        }
                    ],
                    [
                        (0, $25d06cf8d4e72761$exports.ButtonContext),
                        {
                            size: $230078a1c4ce81d8$var$buttonSize[size]
                        }
                    ],
                    [
                        (0, $25d06cf8d4e72761$exports.LinkButtonContext),
                        {
                            size: $230078a1c4ce81d8$var$buttonSize[size]
                        }
                    ]
                ],
                children: children
            }))
    });
});
