// @react-spectrum/s2/dist/private/CenterBaseline.mjs
import "./CenterBaseline.css";
import {mergeStyles as $feb886035e0d4633$export$e618dc39ac9ad607} from "../../icons/runtime.mjs";
import {jsx as $hT6aV$jsx} from "react/jsx-runtime";
import {filterDOMProps as $hT6aV$filterDOMProps} from "react-aria/filterDOMProps";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 


const $1f4b04be3f24aae3$var$styles = " sd131 eb131";
function $1f4b04be3f24aae3$export$768dac55bb57081d(props) {
    let domProps = (0, $hT6aV$filterDOMProps)(props);
    return /*#__PURE__*/ (0, $hT6aV$jsx)("div", {
        ...domProps,
        slot: props.slot,
        style: props.style,
        className: (0, $feb886035e0d4633$export$e618dc39ac9ad607)($1f4b04be3f24aae3$var$styles, props.styles) + ' ' + $1f4b04be3f24aae3$export$179e9489a09b14ac,
        children: props.children
    });
}
const $1f4b04be3f24aae3$export$179e9489a09b14ac = "FRGfhd";
function $1f4b04be3f24aae3$export$9d7e2342a7e53afa(props = {}) {
    return (icon)=>/*#__PURE__*/ (0, $hT6aV$jsx)($1f4b04be3f24aae3$export$768dac55bb57081d, {
            ...props,
            children: icon
        });
}


export {$1f4b04be3f24aae3$export$768dac55bb57081d as CenterBaseline, $1f4b04be3f24aae3$export$179e9489a09b14ac as centerBaselineBefore, $1f4b04be3f24aae3$export$9d7e2342a7e53afa as centerBaseline};
