// @react-spectrum/s2/dist/private/CenterBaseline.cjs
require("./CenterBaseline.css");
var $308b180f49d82d28$exports = require("../../icons/runtime.cjs");
var $5B7cg$reactjsxruntime = require("react/jsx-runtime");
var $5B7cg$reactariafilterDOMProps = require("react-aria/filterDOMProps");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "CenterBaseline", () => $e991cbcdf82ced71$export$768dac55bb57081d);
$parcel$export(module.exports, "centerBaselineBefore", () => $e991cbcdf82ced71$export$179e9489a09b14ac);
$parcel$export(module.exports, "centerBaseline", () => $e991cbcdf82ced71$export$9d7e2342a7e53afa);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 


const $e991cbcdf82ced71$var$styles = " sd131 eb131";
function $e991cbcdf82ced71$export$768dac55bb57081d(props) {
    let domProps = (0, $5B7cg$reactariafilterDOMProps.filterDOMProps)(props);
    return /*#__PURE__*/ (0, $5B7cg$reactjsxruntime.jsx)("div", {
        ...domProps,
        slot: props.slot,
        style: props.style,
        className: (0, $308b180f49d82d28$exports.mergeStyles)($e991cbcdf82ced71$var$styles, props.styles) + ' ' + $e991cbcdf82ced71$export$179e9489a09b14ac,
        children: props.children
    });
}
const $e991cbcdf82ced71$export$179e9489a09b14ac = "FRGfhd";
function $e991cbcdf82ced71$export$9d7e2342a7e53afa(props = {}) {
    return (icon)=>/*#__PURE__*/ (0, $5B7cg$reactjsxruntime.jsx)($e991cbcdf82ced71$export$768dac55bb57081d, {
            ...props,
            children: icon
        });
}
