// @react-spectrum/s2/dist/private/Checkbox.cjs
require("./Checkbox.css");
var $e991cbcdf82ced71$exports = require("./CenterBaseline.cjs");
var $dba2b18fc5727f9b$exports = require("./Checkmark.cjs");
var $84c37b7bd5c1374e$exports = require("./Dash.cjs");
var $bfa4962d90c8af48$exports = require("./Form.cjs");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $2061c83559b50a66$exports = require("./pressScale.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $gg2RS$reactjsxruntime = require("react/jsx-runtime");
var $gg2RS$reactariacomponentsCheckbox = require("react-aria-components/Checkbox");
var $gg2RS$reactariacomponentsCheckboxGroup = require("react-aria-components/CheckboxGroup");
var $gg2RS$reactariacomponentsslots = require("react-aria-components/slots");
var $gg2RS$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "CheckboxContext", () => $af04e099a53d3e85$export$b085522c77523c51);
$parcel$export(module.exports, "box", () => $af04e099a53d3e85$export$827f4ee28efc37);
$parcel$export(module.exports, "iconStyles", () => $af04e099a53d3e85$export$2c99b98b77550cb4);
$parcel$export(module.exports, "Checkbox", () => $af04e099a53d3e85$export$48513f6b9f8ce62d);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 












const $af04e099a53d3e85$export$b085522c77523c51 = /*#__PURE__*/ (0, $gg2RS$react.createContext)(null);
const $af04e099a53d3e85$var$field = function anonymous(props, overrides) {
    let rules = " ";
    let width = false;
    let gridColumnStart = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "Z") width = true;
        if (p[1] === "z") gridColumnStart = true;
        rules += p[0];
    }
    rules += ' se131';
    if (props.isNoVisibleLabel) rules += ' DtyCQZ131';
    else rules += ' DE5iebb131';
    rules += ' qf131';
    rules += ' _ch131';
    if (props.isInCheckboxGroup) {
        if (!width) rules += ' Zd131';
    } else if (!width) rules += ' ZJ131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.size === "XL") rules += ' pt131';
    else if (props.size === "L") rules += ' pt131';
    else if (props.size === "S") rules += ' pt131';
    else if (props.size === "XS") rules += ' pt131';
    else rules += ' pt131';
    if (props.size === "XL") rules += ' -BhX7R-Fz131';
    else if (props.size === "L") rules += ' -BhX7R-Fv131';
    else if (props.size === "S") rules += ' -BhX7R-Fp131';
    else if (props.size === "XS") rules += ' -BhX7R-Fn131';
    else rules += ' -BhX7R-Fx131';
    if (props.isInCheckboxGroup) {
        if (props.size === "XL") rules += ' Ua131';
        else if (props.size === "L") rules += ' Ua131';
        else if (props.size === "M") rules += ' Uai5a0131';
        else if (props.size === "S") rules += ' Uai5a0131';
    } else rules += ' UVxyPec131';
    if (props.isInForm) {
        if (!gridColumnStart) rules += ' zhJUGr131';
    }
    return rules;
};
const $af04e099a53d3e85$var$wrapper = function anonymous(props) {
    let rules = " ";
    rules += ' se131';
    rules += ' DLwlgod131';
    rules += ' z2mU131';
    rules += ' yzZKy131';
    rules += ' _Pc131';
    rules += ' ea131';
    rules += ' Yc131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.isDisabled) {
        rules += ' pp131';
        rules += ' plc131';
    } else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    }
    rules += ' __ca131';
    return rules;
};
const $af04e099a53d3e85$export$827f4ee28efc37 = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-oa131';
    rules += ' oj131';
    rules += ' nj131';
    rules += ' kj131';
    rules += ' jj131';
    if (props.size === "XL") rules += ' Zm131';
    else if (props.size === "L") rules += ' Zl131';
    else if (props.size === "S") rules += ' Zk131';
    else rules += ' Zu131';
    if (props.size === "XL") rules += ' Fn131';
    else if (props.size === "L") rules += ' Fm131';
    else if (props.size === "S") rules += ' Fl131';
    else rules += ' Ft131';
    rules += ' _va131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _Ca131';
    rules += ' _k7m7Gv131';
    rules += ' h7m7Gv131';
    rules += ' m7m7Gv131';
    rules += ' l7m7Gv131';
    rules += ' _oa131';
    rules += ' _jf131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' _yb131';
    if (props.isSelected) {
        if (props.isDisabled) {
            rules += ' gw131';
            rules += ' gld131';
        } else if (props.isInvalid) {
            if (props.isPressed) rules += ' gt131';
            else if (props.isFocusVisible) rules += ' gt131';
            else if (props.isHovered) rules += ' gt131';
            else rules += ' g_d131';
            rules += ' gl_b131';
        } else {
            if (props.isEmphasized) {
                if (props.isPressed) rules += ' gY131';
                else if (props.isFocusVisible) rules += ' gY131';
                else if (props.isHovered) rules += ' gY131';
                else rules += ' g3131';
            } else {
                if (props.isPressed) rules += ' gu131';
                else if (props.isFocusVisible) rules += ' gu131';
                else if (props.isHovered) rules += ' gu131';
                else rules += ' g0131';
            }
            rules += ' gle131';
        }
    } else {
        rules += ' gX131';
        rules += ' gla131';
    }
    if (props.isSelected) rules += ' iA131';
    else if (props.isDisabled) {
        rules += ' io131';
        rules += ' ilc131';
    } else if (props.isInvalid) {
        if (props.isPressed) rules += ' im131';
        else if (props.isFocusVisible) rules += ' im131';
        else if (props.isHovered) rules += ' im131';
        else rules += ' ip131';
        rules += ' ile131';
    } else {
        if (props.isPressed) rules += ' in131';
        else if (props.isFocusVisible) rules += ' in131';
        else if (props.isHovered) rules += ' in131';
        else rules += ' iv131';
        rules += ' ila131';
    }
    return rules;
};
const $af04e099a53d3e85$export$2c99b98b77550cb4 = " _Ob131 -_8sjo0b-txpP0Sb131 -_8sjo0b-lt8pvGJ131";
const $af04e099a53d3e85$var$smallerSize = {
    S: 'XS',
    M: 'S',
    L: 'M',
    XL: 'L'
};
const $af04e099a53d3e85$export$48513f6b9f8ce62d = /*#__PURE__*/ (0, $gg2RS$react.forwardRef)(function Checkbox({ children: children, ...props1 }, ref) {
    [props1, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props1, ref, $af04e099a53d3e85$export$b085522c77523c51);
    let boxRef = (0, $gg2RS$react.useRef)(null);
    let inputRef = (0, $gg2RS$react.useRef)(null);
    let domRef = (0, $150382364eecc4bc$exports.useFocusableRef)(ref, inputRef);
    let isInForm = !!(0, $gg2RS$react.useContext)((0, $bfa4962d90c8af48$exports.FormContext));
    props1 = (0, $bfa4962d90c8af48$exports.useFormProps)(props1);
    let isInCheckboxGroup = !!(0, $gg2RS$react.useContext)((0, $gg2RS$reactariacomponentsCheckboxGroup.CheckboxGroupStateContext));
    let ctx = (0, $gg2RS$reactariacomponentsslots.useSlottedContext)($af04e099a53d3e85$export$b085522c77523c51, props1.slot);
    return /*#__PURE__*/ (0, $gg2RS$reactjsxruntime.jsx)((0, $gg2RS$reactariacomponentsCheckbox.CheckboxField), {
        ...props1,
        ref: domRef,
        inputRef: inputRef,
        style: props1.UNSAFE_style,
        className: (props1.UNSAFE_className || '') + $af04e099a53d3e85$var$field({
            size: props1.size || 'M',
            isInCheckboxGroup: isInCheckboxGroup,
            isNoVisibleLabel: !children
        }, props1.styles),
        children: ({ isDisabled: isDisabled, isInvalid: isInvalid })=>/*#__PURE__*/ (0, $gg2RS$reactjsxruntime.jsxs)((0, $gg2RS$reactjsxruntime.Fragment), {
                children: [
                    /*#__PURE__*/ (0, $gg2RS$reactjsxruntime.jsx)((0, $gg2RS$reactariacomponentsCheckbox.CheckboxButton), {
                        className: (renderProps)=>$af04e099a53d3e85$var$wrapper({
                                ...renderProps,
                                isInForm: isInForm,
                                size: props1.size || 'M'
                            }),
                        children: (renderProps)=>{
                            let checkbox = /*#__PURE__*/ (0, $gg2RS$reactjsxruntime.jsxs)("div", {
                                ref: boxRef,
                                style: (0, $2061c83559b50a66$exports.pressScale)(boxRef)(renderProps),
                                className: $af04e099a53d3e85$export$827f4ee28efc37({
                                    ...renderProps,
                                    isSelected: renderProps.isSelected || renderProps.isIndeterminate,
                                    size: props1.size || 'M',
                                    isEmphasized: isInCheckboxGroup ? ctx?.isEmphasized : props1.isEmphasized
                                }),
                                children: [
                                    renderProps.isIndeterminate && /*#__PURE__*/ (0, $gg2RS$reactjsxruntime.jsx)((0, $84c37b7bd5c1374e$exports.default), {
                                        size: $af04e099a53d3e85$var$smallerSize[props1.size || 'M'],
                                        className: $af04e099a53d3e85$export$2c99b98b77550cb4
                                    }),
                                    renderProps.isSelected && !renderProps.isIndeterminate && /*#__PURE__*/ (0, $gg2RS$reactjsxruntime.jsx)((0, $dba2b18fc5727f9b$exports.default), {
                                        size: $af04e099a53d3e85$var$smallerSize[props1.size || 'M'],
                                        className: $af04e099a53d3e85$export$2c99b98b77550cb4
                                    })
                                ]
                            });
                            // Only render checkbox without center baseline if no label.
                            // This avoids expanding the checkbox height to the font's line height.
                            if (!children) return checkbox;
                            return /*#__PURE__*/ (0, $gg2RS$reactjsxruntime.jsxs)((0, $gg2RS$reactjsxruntime.Fragment), {
                                children: [
                                    /*#__PURE__*/ (0, $gg2RS$reactjsxruntime.jsx)((0, $e991cbcdf82ced71$exports.CenterBaseline), {
                                        children: checkbox
                                    }),
                                    /*#__PURE__*/ (0, $gg2RS$reactjsxruntime.jsx)("span", {
                                        className: " z3mU131",
                                        children: children
                                    })
                                ]
                            });
                        }
                    }),
                    /*#__PURE__*/ (0, $gg2RS$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.HelpText), {
                        size: isInCheckboxGroup ? $af04e099a53d3e85$var$smallerSize[props1.size || 'M'] : props1.size || 'M',
                        styles: function anonymous(props) {
                            let rules = " ";
                            if (props.isInCheckboxGroup) rules += ' z3mU131';
                            else rules += ' z2mU131';
                            rules += ' Td131';
                            return rules;
                        }({
                            isInCheckboxGroup: isInCheckboxGroup
                        }),
                        isDisabled: isDisabled,
                        isInvalid: isInCheckboxGroup ? false : isInvalid,
                        description: props1.description,
                        showErrorIcon: true,
                        children: props1.errorMessage
                    })
                ]
            })
    });
});
