// @react-spectrum/s2/dist/private/Checkbox.mjs
import "./Checkbox.css";
import {CenterBaseline as $1f4b04be3f24aae3$export$768dac55bb57081d} from "./CenterBaseline.mjs";
import $81cf3d4588736e7d$export$2e2bcd8739ae039 from "./Checkmark.mjs";
import $f17037051ff08dc2$export$2e2bcd8739ae039 from "./Dash.mjs";
import {FormContext as $9b916426527cebe7$export$c24727297075ec6a, useFormProps as $9b916426527cebe7$export$a6b5be5c6b451665} from "./Form.mjs";
import {HelpText as $9afd80978c252040$export$a67c0bc59081311a} from "./Field.mjs";
import {pressScale as $10ea7662e51a285b$export$56e8cba416805d8d} from "./pressScale.mjs";
import {useFocusableRef as $80ecb974df713a30$export$96a734597687c040} from "./useDOMRef.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $5oFhd$jsx, jsxs as $5oFhd$jsxs, Fragment as $5oFhd$Fragment} from "react/jsx-runtime";
import {CheckboxField as $5oFhd$CheckboxField, CheckboxButton as $5oFhd$CheckboxButton} from "react-aria-components/Checkbox";
import {CheckboxGroupStateContext as $5oFhd$CheckboxGroupStateContext} from "react-aria-components/CheckboxGroup";
import {useSlottedContext as $5oFhd$useSlottedContext} from "react-aria-components/slots";
import {createContext as $5oFhd$createContext, forwardRef as $5oFhd$forwardRef, useRef as $5oFhd$useRef, useContext as $5oFhd$useContext} from "react";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 












const $9b405e0f8095dce0$export$b085522c77523c51 = /*#__PURE__*/ (0, $5oFhd$createContext)(null);
const $9b405e0f8095dce0$var$field = function anonymous(props, overrides) {
    let rules = " ";
    let width = false;
    let gridColumnStart = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "Z") width = true;
        if (p[1] === "z") gridColumnStart = true;
        rules += p[0];
    }
    rules += ' se131';
    if (props.isNoVisibleLabel) rules += ' DtyCQZ131';
    else rules += ' DE5iebb131';
    rules += ' qf131';
    rules += ' _ch131';
    if (props.isInCheckboxGroup) {
        if (!width) rules += ' Zd131';
    } else if (!width) rules += ' ZJ131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.size === "XL") rules += ' pt131';
    else if (props.size === "L") rules += ' pt131';
    else if (props.size === "S") rules += ' pt131';
    else if (props.size === "XS") rules += ' pt131';
    else rules += ' pt131';
    if (props.size === "XL") rules += ' -BhX7R-Fz131';
    else if (props.size === "L") rules += ' -BhX7R-Fv131';
    else if (props.size === "S") rules += ' -BhX7R-Fp131';
    else if (props.size === "XS") rules += ' -BhX7R-Fn131';
    else rules += ' -BhX7R-Fx131';
    if (props.isInCheckboxGroup) {
        if (props.size === "XL") rules += ' Ua131';
        else if (props.size === "L") rules += ' Ua131';
        else if (props.size === "M") rules += ' Uai5a0131';
        else if (props.size === "S") rules += ' Uai5a0131';
    } else rules += ' UVxyPec131';
    if (props.isInForm) {
        if (!gridColumnStart) rules += ' zhJUGr131';
    }
    return rules;
};
const $9b405e0f8095dce0$var$wrapper = function anonymous(props) {
    let rules = " ";
    rules += ' se131';
    rules += ' DLwlgod131';
    rules += ' z2mU131';
    rules += ' yzZKy131';
    rules += ' _Pc131';
    rules += ' ea131';
    rules += ' Yc131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.isDisabled) {
        rules += ' pp131';
        rules += ' plc131';
    } else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    }
    rules += ' __ca131';
    return rules;
};
const $9b405e0f8095dce0$export$827f4ee28efc37 = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-oa131';
    rules += ' oj131';
    rules += ' nj131';
    rules += ' kj131';
    rules += ' jj131';
    if (props.size === "XL") rules += ' Zm131';
    else if (props.size === "L") rules += ' Zl131';
    else if (props.size === "S") rules += ' Zk131';
    else rules += ' Zu131';
    if (props.size === "XL") rules += ' Fn131';
    else if (props.size === "L") rules += ' Fm131';
    else if (props.size === "S") rules += ' Fl131';
    else rules += ' Ft131';
    rules += ' _va131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _Ca131';
    rules += ' _k7m7Gv131';
    rules += ' h7m7Gv131';
    rules += ' m7m7Gv131';
    rules += ' l7m7Gv131';
    rules += ' _oa131';
    rules += ' _jf131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' _yb131';
    if (props.isSelected) {
        if (props.isDisabled) {
            rules += ' gw131';
            rules += ' gld131';
        } else if (props.isInvalid) {
            if (props.isPressed) rules += ' gt131';
            else if (props.isFocusVisible) rules += ' gt131';
            else if (props.isHovered) rules += ' gt131';
            else rules += ' g_d131';
            rules += ' gl_b131';
        } else {
            if (props.isEmphasized) {
                if (props.isPressed) rules += ' gY131';
                else if (props.isFocusVisible) rules += ' gY131';
                else if (props.isHovered) rules += ' gY131';
                else rules += ' g3131';
            } else {
                if (props.isPressed) rules += ' gu131';
                else if (props.isFocusVisible) rules += ' gu131';
                else if (props.isHovered) rules += ' gu131';
                else rules += ' g0131';
            }
            rules += ' gle131';
        }
    } else {
        rules += ' gX131';
        rules += ' gla131';
    }
    if (props.isSelected) rules += ' iA131';
    else if (props.isDisabled) {
        rules += ' io131';
        rules += ' ilc131';
    } else if (props.isInvalid) {
        if (props.isPressed) rules += ' im131';
        else if (props.isFocusVisible) rules += ' im131';
        else if (props.isHovered) rules += ' im131';
        else rules += ' ip131';
        rules += ' ile131';
    } else {
        if (props.isPressed) rules += ' in131';
        else if (props.isFocusVisible) rules += ' in131';
        else if (props.isHovered) rules += ' in131';
        else rules += ' iv131';
        rules += ' ila131';
    }
    return rules;
};
const $9b405e0f8095dce0$export$2c99b98b77550cb4 = " _Ob131 -_8sjo0b-txpP0Sb131 -_8sjo0b-lt8pvGJ131";
const $9b405e0f8095dce0$var$smallerSize = {
    S: 'XS',
    M: 'S',
    L: 'M',
    XL: 'L'
};
const $9b405e0f8095dce0$export$48513f6b9f8ce62d = /*#__PURE__*/ (0, $5oFhd$forwardRef)(function Checkbox({ children: children, ...props1 }, ref) {
    [props1, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props1, ref, $9b405e0f8095dce0$export$b085522c77523c51);
    let boxRef = (0, $5oFhd$useRef)(null);
    let inputRef = (0, $5oFhd$useRef)(null);
    let domRef = (0, $80ecb974df713a30$export$96a734597687c040)(ref, inputRef);
    let isInForm = !!(0, $5oFhd$useContext)((0, $9b916426527cebe7$export$c24727297075ec6a));
    props1 = (0, $9b916426527cebe7$export$a6b5be5c6b451665)(props1);
    let isInCheckboxGroup = !!(0, $5oFhd$useContext)((0, $5oFhd$CheckboxGroupStateContext));
    let ctx = (0, $5oFhd$useSlottedContext)($9b405e0f8095dce0$export$b085522c77523c51, props1.slot);
    return /*#__PURE__*/ (0, $5oFhd$jsx)((0, $5oFhd$CheckboxField), {
        ...props1,
        ref: domRef,
        inputRef: inputRef,
        style: props1.UNSAFE_style,
        className: (props1.UNSAFE_className || '') + $9b405e0f8095dce0$var$field({
            size: props1.size || 'M',
            isInCheckboxGroup: isInCheckboxGroup,
            isNoVisibleLabel: !children
        }, props1.styles),
        children: ({ isDisabled: isDisabled, isInvalid: isInvalid })=>/*#__PURE__*/ (0, $5oFhd$jsxs)((0, $5oFhd$Fragment), {
                children: [
                    /*#__PURE__*/ (0, $5oFhd$jsx)((0, $5oFhd$CheckboxButton), {
                        className: (renderProps)=>$9b405e0f8095dce0$var$wrapper({
                                ...renderProps,
                                isInForm: isInForm,
                                size: props1.size || 'M'
                            }),
                        children: (renderProps)=>{
                            let checkbox = /*#__PURE__*/ (0, $5oFhd$jsxs)("div", {
                                ref: boxRef,
                                style: (0, $10ea7662e51a285b$export$56e8cba416805d8d)(boxRef)(renderProps),
                                className: $9b405e0f8095dce0$export$827f4ee28efc37({
                                    ...renderProps,
                                    isSelected: renderProps.isSelected || renderProps.isIndeterminate,
                                    size: props1.size || 'M',
                                    isEmphasized: isInCheckboxGroup ? ctx?.isEmphasized : props1.isEmphasized
                                }),
                                children: [
                                    renderProps.isIndeterminate && /*#__PURE__*/ (0, $5oFhd$jsx)((0, $f17037051ff08dc2$export$2e2bcd8739ae039), {
                                        size: $9b405e0f8095dce0$var$smallerSize[props1.size || 'M'],
                                        className: $9b405e0f8095dce0$export$2c99b98b77550cb4
                                    }),
                                    renderProps.isSelected && !renderProps.isIndeterminate && /*#__PURE__*/ (0, $5oFhd$jsx)((0, $81cf3d4588736e7d$export$2e2bcd8739ae039), {
                                        size: $9b405e0f8095dce0$var$smallerSize[props1.size || 'M'],
                                        className: $9b405e0f8095dce0$export$2c99b98b77550cb4
                                    })
                                ]
                            });
                            // Only render checkbox without center baseline if no label.
                            // This avoids expanding the checkbox height to the font's line height.
                            if (!children) return checkbox;
                            return /*#__PURE__*/ (0, $5oFhd$jsxs)((0, $5oFhd$Fragment), {
                                children: [
                                    /*#__PURE__*/ (0, $5oFhd$jsx)((0, $1f4b04be3f24aae3$export$768dac55bb57081d), {
                                        children: checkbox
                                    }),
                                    /*#__PURE__*/ (0, $5oFhd$jsx)("span", {
                                        className: " z3mU131",
                                        children: children
                                    })
                                ]
                            });
                        }
                    }),
                    /*#__PURE__*/ (0, $5oFhd$jsx)((0, $9afd80978c252040$export$a67c0bc59081311a), {
                        size: isInCheckboxGroup ? $9b405e0f8095dce0$var$smallerSize[props1.size || 'M'] : props1.size || 'M',
                        styles: function anonymous(props) {
                            let rules = " ";
                            if (props.isInCheckboxGroup) rules += ' z3mU131';
                            else rules += ' z2mU131';
                            rules += ' Td131';
                            return rules;
                        }({
                            isInCheckboxGroup: isInCheckboxGroup
                        }),
                        isDisabled: isDisabled,
                        isInvalid: isInCheckboxGroup ? false : isInvalid,
                        description: props1.description,
                        showErrorIcon: true,
                        children: props1.errorMessage
                    })
                ]
            })
    });
});


export {$9b405e0f8095dce0$export$b085522c77523c51 as CheckboxContext, $9b405e0f8095dce0$export$827f4ee28efc37 as box, $9b405e0f8095dce0$export$2c99b98b77550cb4 as iconStyles, $9b405e0f8095dce0$export$48513f6b9f8ce62d as Checkbox};
