// @react-spectrum/s2/dist/private/ClearButton.cjs
require("./ClearButton.css");
var $89dd175101ffc43a$exports = require("./Cross.cjs");
var $2061c83559b50a66$exports = require("./pressScale.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ae1Ho$reactjsxruntime = require("react/jsx-runtime");
var $ae1Ho$reactariacomponentsButton = require("react-aria-components/Button");
var $ae1Ho$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "ClearButton", () => $4fb579ecc195bd2b$export$13ec83e50bf04290);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





const $4fb579ecc195bd2b$var$focusRingStyles = {
    outlineStyle: {
        "default": "none",
        isFocusVisible: "solid"
    },
    outlineColor: {
        "default": "focus-ring",
        forcedColors: "Highlight"
    },
    outlineWidth: 2,
    outlineOffset: 2
};
const $4fb579ecc195bd2b$var$visibleClearButton = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    if (props.isStaticColor) rules += ' Ok131';
    else {
        rules += ' Oh131';
        rules += ' Olc131';
    }
    rules += ' _Mc131';
    rules += ' _Kc131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _Ca131';
    rules += ' Fb131';
    if (props.size === "XL") rules += ' ZF131';
    else if (props.size === "L") rules += ' Zw131';
    else if (props.size === "S") rules += ' Zo131';
    else if (props.size === "XS") rules += ' Zm131';
    else rules += ' ZC131';
    rules += ' _va131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    rules += ' _je131';
    rules += ' g9131';
    rules += ' _oa131';
    rules += ' Td131';
    rules += ' Qd131';
    rules += ' Sd131';
    rules += ' Rd131';
    rules += ' p40ub4c131';
    rules += ' -_8sjo0b-t5ZbAob131';
    return rules;
};
const $4fb579ecc195bd2b$export$13ec83e50bf04290 = /*#__PURE__*/ (0, $ae1Ho$react.forwardRef)(function ClearButton(props, ref) {
    let { size: size = 'M', isStaticColor: isStaticColor = false, ...rest } = props;
    let domRef = (0, $150382364eecc4bc$exports.useFocusableRef)(ref);
    return /*#__PURE__*/ (0, $ae1Ho$reactjsxruntime.jsx)((0, $ae1Ho$reactariacomponentsButton.Button), {
        ...rest,
        ref: domRef,
        style: (0, $2061c83559b50a66$exports.pressScale)(domRef),
        className: (renderProps)=>$4fb579ecc195bd2b$var$visibleClearButton({
                ...renderProps,
                size: size,
                isStaticColor: isStaticColor
            }),
        children: /*#__PURE__*/ (0, $ae1Ho$reactjsxruntime.jsx)((0, $89dd175101ffc43a$exports.default), {
            size: props.size
        })
    });
});
