// @react-spectrum/s2/dist/private/ClearButton.mjs
import "./ClearButton.css";
import $bd983341bd35d639$export$2e2bcd8739ae039 from "./Cross.mjs";
import {pressScale as $10ea7662e51a285b$export$56e8cba416805d8d} from "./pressScale.mjs";
import {useFocusableRef as $80ecb974df713a30$export$96a734597687c040} from "./useDOMRef.mjs";
import {jsx as $9qqo3$jsx} from "react/jsx-runtime";
import {Button as $9qqo3$Button} from "react-aria-components/Button";
import {forwardRef as $9qqo3$forwardRef} from "react";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





const $f6a62e6e68437458$var$focusRingStyles = {
    outlineStyle: {
        "default": "none",
        isFocusVisible: "solid"
    },
    outlineColor: {
        "default": "focus-ring",
        forcedColors: "Highlight"
    },
    outlineWidth: 2,
    outlineOffset: 2
};
const $f6a62e6e68437458$var$visibleClearButton = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    if (props.isStaticColor) rules += ' Ok131';
    else {
        rules += ' Oh131';
        rules += ' Olc131';
    }
    rules += ' _Mc131';
    rules += ' _Kc131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _Ca131';
    rules += ' Fb131';
    if (props.size === "XL") rules += ' ZF131';
    else if (props.size === "L") rules += ' Zw131';
    else if (props.size === "S") rules += ' Zo131';
    else if (props.size === "XS") rules += ' Zm131';
    else rules += ' ZC131';
    rules += ' _va131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    rules += ' _je131';
    rules += ' g9131';
    rules += ' _oa131';
    rules += ' Td131';
    rules += ' Qd131';
    rules += ' Sd131';
    rules += ' Rd131';
    rules += ' p40ub4c131';
    rules += ' -_8sjo0b-t5ZbAob131';
    return rules;
};
const $f6a62e6e68437458$export$13ec83e50bf04290 = /*#__PURE__*/ (0, $9qqo3$forwardRef)(function ClearButton(props, ref) {
    let { size: size = 'M', isStaticColor: isStaticColor = false, ...rest } = props;
    let domRef = (0, $80ecb974df713a30$export$96a734597687c040)(ref);
    return /*#__PURE__*/ (0, $9qqo3$jsx)((0, $9qqo3$Button), {
        ...rest,
        ref: domRef,
        style: (0, $10ea7662e51a285b$export$56e8cba416805d8d)(domRef),
        className: (renderProps)=>$f6a62e6e68437458$var$visibleClearButton({
                ...renderProps,
                size: size,
                isStaticColor: isStaticColor
            }),
        children: /*#__PURE__*/ (0, $9qqo3$jsx)((0, $bd983341bd35d639$export$2e2bcd8739ae039), {
            size: props.size
        })
    });
});


export {$f6a62e6e68437458$export$13ec83e50bf04290 as ClearButton};
