// @react-spectrum/s2/dist/private/ColorSwatchPicker.mjs
import "./ColorSwatchPicker.css";
import {InternalColorSwatchContext as $cd9e1720baab3945$export$2d9e1fe7b5c0ceb6} from "./ColorSwatch.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $bOBHx$jsx, jsxs as $bOBHx$jsxs, Fragment as $bOBHx$Fragment} from "react/jsx-runtime";
import {ColorSwatchPicker as $bOBHx$ColorSwatchPicker, ColorSwatchPickerItem as $bOBHx$ColorSwatchPickerItem} from "react-aria-components/ColorSwatchPicker";
import {createContext as $bOBHx$createContext, forwardRef as $bOBHx$forwardRef} from "react";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





const $ba9c901b6c815fb9$export$7214f50881fc1eaf = /*#__PURE__*/ (0, $bOBHx$createContext)(null);
const $ba9c901b6c815fb9$export$b46792416e3d8515 = /*#__PURE__*/ (0, $bOBHx$forwardRef)(function ColorSwatchPicker(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $ba9c901b6c815fb9$export$7214f50881fc1eaf);
    let { density: density = 'regular', size: size = 'M', rounding: rounding = 'none', UNSAFE_className: UNSAFE_className = '', ...otherProps } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    return /*#__PURE__*/ (0, $bOBHx$jsx)((0, $bOBHx$ColorSwatchPicker), {
        ...otherProps,
        ref: domRef,
        className: UNSAFE_className + function anonymous(props, overrides) {
            let rules = " ";
            rules += (String(overrides || '').match(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g) || []).join('');
            rules += ' sd131';
            rules += ' _wb131';
            if (props.density === "spacious") rules += ' UWJpoB131';
            else if (props.density === "regular") rules += ' Ub131';
            else if (props.density === "compact") rules += ' Ua131';
            if (props.density === "spacious") rules += ' qWJpoB131';
            else if (props.density === "regular") rules += ' qb131';
            else if (props.density === "compact") rules += ' qa131';
            return rules;
        }({
            density: density
        }, props.styles),
        children: /*#__PURE__*/ (0, $bOBHx$jsx)((0, $cd9e1720baab3945$export$2d9e1fe7b5c0ceb6).Provider, {
            value: {
                useWrapper: $ba9c901b6c815fb9$var$useWrapper,
                size: size,
                rounding: rounding
            },
            children: props.children
        })
    });
});
function $ba9c901b6c815fb9$var$useWrapper(swatch, color, rounding) {
    return /*#__PURE__*/ (0, $bOBHx$jsx)((0, $bOBHx$ColorSwatchPickerItem), {
        color: color,
        className: (renderProps)=>(function anonymous(props) {
                let rules = " ";
                if (props.isFocusVisible) rules += ' _Lf131';
                else rules += ' _Le131';
                rules += ' Oh131';
                rules += ' Olc131';
                rules += ' _Mc131';
                rules += ' _Kd131';
                rules += ' _Pc131';
                if (props.rounding === "full") rules += ' og131';
                else if (props.rounding === "default") rules += ' oa131';
                else if (props.rounding === "none") rules += ' od131';
                if (props.rounding === "full") rules += ' ng131';
                else if (props.rounding === "default") rules += ' na131';
                else if (props.rounding === "none") rules += ' nd131';
                if (props.rounding === "full") rules += ' kg131';
                else if (props.rounding === "default") rules += ' ka131';
                else if (props.rounding === "none") rules += ' kd131';
                if (props.rounding === "full") rules += ' jg131';
                else if (props.rounding === "default") rules += ' ja131';
                else if (props.rounding === "none") rules += ' jd131';
                rules += ' __ca131';
                return rules;
            })({
                ...renderProps,
                rounding: rounding
            }),
        children: ({ isSelected: isSelected })=>/*#__PURE__*/ (0, $bOBHx$jsxs)((0, $bOBHx$Fragment), {
                children: [
                    swatch,
                    isSelected && /*#__PURE__*/ (0, $bOBHx$jsx)("div", {
                        "aria-hidden": true,
                        className: " _Pa131 _Ob131 Wr131 _lr131 _Ar131 _zr131 _oa131 in131 _jf131 _kc131 hc131 mc131 lc131 Og131 _Lf131 _Mc131 _Kc131 _yb131 oi131 ni131 ki131 ji131"
                    })
                ]
            })
    });
}


export {$ba9c901b6c815fb9$export$7214f50881fc1eaf as ColorSwatchPickerContext, $ba9c901b6c815fb9$export$b46792416e3d8515 as ColorSwatchPicker};
