// @react-spectrum/s2/dist/private/ContextualHelp.mjs
import "./ContextualHelp.css";
import {ActionButton as $da878a05ab4a403e$export$cfc7921d29ef7b80} from "./ActionButton.mjs";
import {ContentContext as $8e847109a6ab556d$export$1cbdd774077931b4, FooterContext as $8e847109a6ab556d$export$5630640b68817ed6, HeadingContext as $8e847109a6ab556d$export$d688439359537581, TextContext as $8e847109a6ab556d$export$9afb8bc826b033ea} from "./Content.mjs";
import {dialogInner as $9bbc115952dac5a6$export$6af31f836f9aa6f2} from "./Dialog.mjs";
import {DialogTrigger as $66262086fa190fcb$export$2e1e1122cf0cba88} from "./DialogTrigger.mjs";
import $410bd93c9b5189d7$export$2e2bcd8739ae039 from "../../icons/HelpCircle.mjs";
import $4c8f17dac3ecefd9$export$2e2bcd8739ae039 from "../../icons/InfoCircle.mjs";
import $bcd4U$intlStringsmjs from "./intlStrings.mjs";
import {mergeStyles as $feb886035e0d4633$export$e618dc39ac9ad607} from "../../icons/runtime.mjs";
import {Popover as $88b746eba92c8d0d$export$5b6b19405a83ff9d} from "./Popover.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $bcd4U$jsx, jsxs as $bcd4U$jsxs} from "react/jsx-runtime";
import {Provider as $bcd4U$Provider, DEFAULT_SLOT as $bcd4U$DEFAULT_SLOT} from "react-aria-components/slots";
import {createContext as $bcd4U$createContext, forwardRef as $bcd4U$forwardRef} from "react";
import {filterDOMProps as $bcd4U$filterDOMProps} from "react-aria/filterDOMProps";
import {mergeProps as $bcd4U$mergeProps} from "react-aria/mergeProps";
import {TextContext as $bcd4U$TextContext} from "react-aria-components/Text";
import {useId as $bcd4U$useId} from "react-aria/useId";
import {useLabels as $bcd4U$useLabels} from "react-aria/private/utils/useLabels";
import {useLocalizedStringFormatter as $bcd4U$useLocalizedStringFormatter} from "react-aria/useLocalizedStringFormatter";


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}



















const $8e176595bc49ca3f$var$wrappingDiv = " NOBje7c131 ZOBje7c131 Th131 Qh131 Sh131 Rh131 _oa131 Fb131";
const $8e176595bc49ca3f$var$headingStyles = " uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-e131 vx131 xe131 _xa131 xX0cczbd131 _xX0cczba131 xfe131 _xfa131 xfX0cczbd131 _xfX0cczba131 _Fd131 _FnuYUweb131 po131 Jy131 Gv131 Iy131 Hy131";
function $8e176595bc49ca3f$export$218201d297fb4e0d(props) {
    let { children: children, ...popoverProps } = props;
    let titleId = (0, $bcd4U$useId)();
    return /*#__PURE__*/ (0, $bcd4U$jsx)((0, $88b746eba92c8d0d$export$5b6b19405a83ff9d), {
        padding: "none",
        hideArrow: true,
        "aria-labelledby": titleId,
        ...popoverProps,
        children: /*#__PURE__*/ (0, $bcd4U$jsx)("div", {
            className: $8e176595bc49ca3f$var$wrappingDiv,
            children: /*#__PURE__*/ (0, $bcd4U$jsx)("div", {
                className: (0, $feb886035e0d4633$export$e618dc39ac9ad607)((0, $9bbc115952dac5a6$export$6af31f836f9aa6f2), " od131 nd131 kd131 jd131 Jnqb3Ob131 Gnqb3Ob131 Inqb3Ob131 Hnqb3Ob131 TjCFGYc131 Qh131 Sh131 Rh131 -T_-Th131"),
                children: /*#__PURE__*/ (0, $bcd4U$jsx)((0, $bcd4U$Provider), {
                    values: [
                        [
                            (0, $bcd4U$TextContext),
                            {
                                slots: {
                                    [(0, $bcd4U$DEFAULT_SLOT)]: {}
                                }
                            }
                        ],
                        // Make sure to clear context from above Menu
                        [
                            (0, $8e847109a6ab556d$export$9afb8bc826b033ea),
                            null
                        ],
                        [
                            (0, $8e847109a6ab556d$export$d688439359537581),
                            {
                                styles: $8e176595bc49ca3f$var$headingStyles,
                                slots: {
                                    // needed so combobox/picker does not need to provide slot="title" to their provided
                                    // ContextualHelp (they get the aria-labelled by from the button)
                                    // otherwise, use the heading if available aka unavaiable menu item
                                    [(0, $bcd4U$DEFAULT_SLOT)]: {
                                        styles: $8e176595bc49ca3f$var$headingStyles
                                    },
                                    title: {
                                        id: titleId,
                                        styles: $8e176595bc49ca3f$var$headingStyles,
                                        level: 2
                                    }
                                }
                            }
                        ],
                        [
                            (0, $8e847109a6ab556d$export$1cbdd774077931b4),
                            {
                                styles: " uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-c131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131"
                            }
                        ],
                        [
                            (0, $8e847109a6ab556d$export$5630640b68817ed6),
                            {
                                styles: " uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-c131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131 JC131"
                            }
                        ]
                    ],
                    children: children
                })
            })
        })
    });
}
const $8e176595bc49ca3f$export$41b9afaaba473494 = /*#__PURE__*/ (0, $bcd4U$createContext)(null);
const $8e176595bc49ca3f$export$7d3cdb256c2ba320 = /*#__PURE__*/ (0, $bcd4U$forwardRef)(function ContextualHelp(props, ref) {
    let stringFormatter = (0, $bcd4U$useLocalizedStringFormatter)((0, ($parcel$interopDefault($bcd4U$intlStringsmjs))), '@react-spectrum/s2');
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $8e176595bc49ca3f$export$41b9afaaba473494);
    let { children: children, defaultOpen: defaultOpen, containerPadding: containerPadding = 8, size: size = 'XS', crossOffset: crossOffset, isOpen: isOpen, onOpenChange: onOpenChange, placement: placement = 'bottom start', shouldFlip: shouldFlip, UNSAFE_className: UNSAFE_className, UNSAFE_style: UNSAFE_style, styles: styles, variant: variant = 'help' } = props;
    // In a FieldLabel we're getting the context's aria-labeledby, so we need to
    // manually set the aria-label after useLabels() to keep the order of label
    // then ContextualHelp variant
    let labelProps = (0, $bcd4U$useLabels)(props);
    let label = stringFormatter.format(`contextualhelp.${variant}`);
    labelProps['aria-label'] = labelProps['aria-label'] ? labelProps['aria-label'] + ' ' + label : label;
    let buttonProps = (0, $bcd4U$filterDOMProps)(props, {
        labelable: true
    });
    return /*#__PURE__*/ (0, $bcd4U$jsxs)((0, $66262086fa190fcb$export$2e1e1122cf0cba88), {
        isOpen: isOpen,
        defaultOpen: defaultOpen,
        onOpenChange: onOpenChange,
        children: [
            /*#__PURE__*/ (0, $bcd4U$jsx)((0, $da878a05ab4a403e$export$cfc7921d29ef7b80), {
                slot: null,
                ref: ref,
                size: size,
                ...(0, $bcd4U$mergeProps)(buttonProps, labelProps),
                UNSAFE_style: UNSAFE_style,
                UNSAFE_className: UNSAFE_className,
                styles: styles,
                isQuiet: true,
                children: variant === 'info' ? /*#__PURE__*/ (0, $bcd4U$jsx)((0, $4c8f17dac3ecefd9$export$2e2bcd8739ae039), {}) : /*#__PURE__*/ (0, $bcd4U$jsx)((0, $410bd93c9b5189d7$export$2e2bcd8739ae039), {})
            }),
            /*#__PURE__*/ (0, $bcd4U$jsx)($8e176595bc49ca3f$export$218201d297fb4e0d, {
                placement: placement,
                shouldFlip: shouldFlip,
                containerPadding: containerPadding,
                offset: 8,
                crossOffset: crossOffset,
                children: children
            })
        ]
    });
});


export {$8e176595bc49ca3f$export$218201d297fb4e0d as ContextualHelpPopover, $8e176595bc49ca3f$export$41b9afaaba473494 as ContextualHelpContext, $8e176595bc49ca3f$export$7d3cdb256c2ba320 as ContextualHelp};
