// @react-spectrum/s2/dist/private/CustomDialog.mjs
import "./CustomDialog.css";
import {Modal as $c46d159bd615e407$export$2b77a92f1a5ad772} from "./Modal.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {jsx as $4wOFe$jsx} from "react/jsx-runtime";
import {composeRenderProps as $4wOFe$composeRenderProps} from "react-aria-components/composeRenderProps";
import {forwardRef as $4wOFe$forwardRef} from "react";
import {Dialog as $4wOFe$Dialog, OverlayTriggerStateContext as $4wOFe$OverlayTriggerStateContext} from "react-aria-components/Dialog";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





const $6e8e8fdd66697ef1$var$dialogStyle = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let width = false;
    let height = false;
    let maxWidth = false;
    let maxHeight = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L|F|M|K)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "Z") width = true;
        if (p[1] === "F") height = true;
        if (p[1] === "L") maxWidth = true;
        if (p[1] === "K") maxHeight = true;
        rules += p[0];
    }
    if (props.padding === "none") rules += ' Td131';
    else if (props.padding === "default") {
        rules += ' Th131';
        rules += ' Tnk131';
    }
    if (props.padding === "none") rules += ' Qd131';
    else if (props.padding === "default") {
        rules += ' Qh131';
        rules += ' Qnk131';
    }
    if (props.padding === "none") rules += ' Sd131';
    else if (props.padding === "default") {
        rules += ' Sh131';
        rules += ' Snk131';
    }
    if (props.padding === "none") rules += ' Rd131';
    else if (props.padding === "default") {
        rules += ' Rh131';
        rules += ' Rnk131';
    }
    rules += ' _oa131';
    rules += ' _Le131';
    rules += ' oi131';
    rules += ' ni131';
    rules += ' ki131';
    rules += ' ji131';
    rules += ' _Na131';
    rules += ' Pa131';
    if (!position) rules += ' _Pc131';
    if (!width) rules += ' Za131';
    if (!height) rules += ' Fb131';
    if (!maxWidth) rules += ' L40ub4c131';
    if (!maxHeight) rules += ' K40ub4c131';
    return rules;
};
const $6e8e8fdd66697ef1$export$82e1fa7406db6ec = /*#__PURE__*/ (0, $4wOFe$forwardRef)(function CustomDialog(props, ref) {
    let { size: size, isDismissible: isDismissible, isKeyboardDismissDisabled: isKeyboardDismissDisabled, padding: padding = 'default' } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    return /*#__PURE__*/ (0, $4wOFe$jsx)((0, $c46d159bd615e407$export$2b77a92f1a5ad772), {
        size: size,
        isDismissable: isDismissible,
        isKeyboardDismissDisabled: isKeyboardDismissDisabled,
        children: /*#__PURE__*/ (0, $4wOFe$jsx)((0, $4wOFe$Dialog), {
            ...props,
            ref: domRef,
            style: props.UNSAFE_style,
            className: (props.UNSAFE_className || '') + $6e8e8fdd66697ef1$var$dialogStyle({
                padding: padding
            }, props.styles),
            children: (0, $4wOFe$composeRenderProps)(props.children, (children)=>// Reset OverlayTriggerStateContext so the buttons inside the dialog don't retain their hover state.
                /*#__PURE__*/ (0, $4wOFe$jsx)((0, $4wOFe$OverlayTriggerStateContext).Provider, {
                    value: null,
                    children: children
                }))
        })
    });
});


export {$6e8e8fdd66697ef1$export$82e1fa7406db6ec as CustomDialog};
