// @react-spectrum/s2/dist/private/DatePicker.cjs
require("./DatePicker.css");
var $e9e9a0d15c3364d7$exports = require("./Calendar.cjs");
var $1dfd5738cdf83d1e$exports = require("../../icons/Calendar.cjs");
var $d19642654b699f84$exports = require("./DateField.cjs");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $4526404114e78c80$exports = require("./intlStrings.cjs");
var $493371ef39bf7a55$exports = require("./Popover.cjs");
var $2061c83559b50a66$exports = require("./pressScale.cjs");
var $7b93ad26130ad3f8$exports = require("./TimeField.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $2B5ON$reactjsxruntime = require("react/jsx-runtime");
var $2B5ON$reactariacomponentsDatePicker = require("react-aria-components/DatePicker");
var $2B5ON$reactariacomponentsButton = require("react-aria-components/Button");
var $2B5ON$reactariacomponentsslots = require("react-aria-components/slots");
var $2B5ON$react = require("react");
var $2B5ON$reactariacomponentsDialog = require("react-aria-components/Dialog");
var $2B5ON$reactariacomponentsForm = require("react-aria-components/Form");
var $2B5ON$reactariauseLocalizedStringFormatter = require("react-aria/useLocalizedStringFormatter");


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "DatePickerContext", () => $e6aad0776f57e0e9$export$cf316c7f3b44c11e);
$parcel$export(module.exports, "timeField", () => $e6aad0776f57e0e9$export$5320dd0e349ed44d);
$parcel$export(module.exports, "DatePicker", () => $e6aad0776f57e0e9$export$5109c6dd95d8fb00);
$parcel$export(module.exports, "CalendarButton", () => $e6aad0776f57e0e9$export$adb29da0ac001538);
$parcel$export(module.exports, "CalendarPopover", () => $e6aad0776f57e0e9$export$f3ce207574a3cffe);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 

















const $e6aad0776f57e0e9$export$cf316c7f3b44c11e = /*#__PURE__*/ (0, $2B5ON$react.createContext)(null);
const $e6aad0776f57e0e9$var$inputButton = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-oa131';
    rules += ' oj131';
    rules += ' nj131';
    rules += ' kj131';
    rules += ' jj131';
    rules += ' _Pc131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "M") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "M") {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "M") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "M") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isDisabled) rules += ' pp131';
    else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    }
    rules += ' pla131';
    rules += ' ri131';
    rules += ' sd131';
    rules += ' _Wa131';
    rules += ' _je131';
    rules += ' eb131';
    rules += ' _Ca131';
    if (props.size === "XL") rules += ' ZC131';
    else if (props.size === "L") rules += ' Zo131';
    else if (props.size === "M") rules += ' Zm131';
    else if (props.size === "S") rules += ' Zu131';
    rules += ' Fd131';
    rules += ' Iw131';
    rules += ' fa131';
    rules += ' _va131';
    rules += ' Yd131';
    rules += ' Yle131';
    rules += ' Xb131';
    rules += ' Xlb131';
    rules += ' _2b131';
    rules += ' _2lb131';
    if (props.isDisabled) rules += ' gH131';
    else if (props.isOpen) rules += ' gF131';
    else {
        if (props.isPressed) rules += ' gF131';
        else if (props.isFocusVisible) rules += ' gF131';
        else if (props.isHovered) rules += ' gF131';
        else rules += ' gH131';
    }
    if (props.isDisabled) rules += ' gld131';
    else if (props.isOpen) rules += ' gle131';
    else if (props.isHovered) rules += ' gle131';
    else rules += ' glc131';
    return rules;
};
const $e6aad0776f57e0e9$export$5320dd0e349ed44d = " _vb131 _ua131 Na131 Ze4mUs131";
const $e6aad0776f57e0e9$export$5109c6dd95d8fb00 = /*#__PURE__*/ (0, $2B5ON$react.forwardRef)(function DatePicker(props1, ref) {
    [props1, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props1, ref, $e6aad0776f57e0e9$export$cf316c7f3b44c11e);
    let stringFormatter = (0, $2B5ON$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    let { label: label, contextualHelp: contextualHelp, description: descriptionMessage, errorMessage: errorMessage, isRequired: isRequired, size: size = 'M', labelPosition: labelPosition = 'top', necessityIndicator: necessityIndicator, labelAlign: labelAlign = 'start', UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className, styles: styles, placeholderValue: placeholderValue, maxVisibleMonths: maxVisibleMonths = 1, createCalendar: createCalendar, ...dateFieldProps } = props1;
    let formContext = (0, $2B5ON$react.useContext)((0, $2B5ON$reactariacomponentsForm.FormContext));
    let [buttonHasFocus, setButtonHasFocus] = (0, $2B5ON$react.useState)(false);
    return /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $2B5ON$reactariacomponentsDatePicker.DatePicker), {
        ref: ref,
        isRequired: isRequired,
        ...dateFieldProps,
        style: UNSAFE_style,
        className: (UNSAFE_className || '') + function anonymous(props, overrides) {
            let rules = " ";
            let gridColumnStart = false;
            let gridColumnEnd = false;
            let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
            for (let p of matches){
                if (p[1] === "z") gridColumnStart = true;
                if (p[1] === "y") gridColumnEnd = true;
                rules += p[0];
            }
            rules += ' se131';
            if (props.isInForm) {
                if (!gridColumnStart) rules += ' z2mU131';
            }
            if (props.isInForm) {
                if (props.labelPosition === "side") {
                    if (!gridColumnEnd) rules += ' yvoofG131';
                }
            }
            if (props.isInForm) rules += ' DLwlgod131';
            else {
                if (props.labelPosition === "side") rules += ' DPlN5qb131';
                else if (props.labelPosition === "top") rules += ' DM8Mfn131';
            }
            if (props.labelPosition === "side") rules += ' EPlN5qb131';
            else if (props.labelPosition === "top") rules += ' E4w1sLc131';
            if (props.labelPosition === "side") rules += ' CbRM95c131';
            else if (props.labelPosition === "top") rules += ' Cxaocre131';
            if (props.size === "XL") {
                rules += ' -_6BNtrc-e131';
                rules += ' vx131';
            } else if (props.size === "L") {
                rules += ' -_6BNtrc-d131';
                rules += ' vx131';
            } else if (props.size === "S") {
                rules += ' -_6BNtrc-b131';
                rules += ' vx131';
            } else if (props.size === "XS") {
                rules += ' -_6BNtrc-a131';
                rules += ' vx131';
            } else {
                rules += ' -_6BNtrc-c131';
                rules += ' vx131';
            }
            rules += ' ea131';
            rules += ' _Fd131';
            rules += ' _FnuYUweb131';
            if (props.size === "XL") rules += ' -BhX7R-Fz131';
            else if (props.size === "L") rules += ' -BhX7R-Fv131';
            else if (props.size === "S") rules += ' -BhX7R-Fp131';
            else if (props.size === "XS") rules += ' -BhX7R-Fn131';
            else rules += ' -BhX7R-Fx131';
            rules += ' -AJjOLd-UVkF0Db131';
            rules += ' qg131';
            rules += ' __ca131';
            return rules;
        }({
            isInForm: !!formContext,
            labelPosition: labelPosition,
            size: size
        }, styles),
        children: ({ isDisabled: isDisabled, isInvalid: isInvalid, isOpen: isOpen, state: state })=>{
            let placeholder = placeholderValue ?? undefined;
            let timePlaceholder = placeholder && 'hour' in placeholder ? placeholder : undefined;
            let timeMinValue = props1.minValue && 'hour' in props1.minValue ? props1.minValue : undefined;
            let timeMaxValue = props1.maxValue && 'hour' in props1.maxValue ? props1.maxValue : undefined;
            let timeGranularity = state.granularity === 'hour' || state.granularity === 'minute' || state.granularity === 'second' ? state.granularity : undefined;
            let showTimeField = !!timeGranularity;
            // Ideally, we could omit errorMessage here and let S2 Calendar read it from RAC's CalendarContext which already contains the resolved value via calendarProps from useDatePicker.
            // However, RAC's CalendarProps omits errorMessage, so reading it back from the context would require an unsafe cast. Instead, we resolve it here using the same logic as useDatePicker.
            let resolvedErrorMessage = typeof errorMessage === 'function' ? errorMessage(state.displayValidation) : errorMessage || state.displayValidation.validationErrors.join(' ');
            return /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsxs)((0, $2B5ON$reactjsxruntime.Fragment), {
                children: [
                    /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldLabel), {
                        isDisabled: isDisabled,
                        isRequired: isRequired,
                        size: size,
                        labelPosition: labelPosition,
                        labelAlign: labelAlign,
                        necessityIndicator: necessityIndicator,
                        contextualHelp: contextualHelp,
                        children: label
                    }),
                    /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsxs)((0, $a0b996a550bb0891$exports.FieldGroup), {
                        role: "presentation",
                        shouldTurnOffFocusRing: buttonHasFocus,
                        isDisabled: isDisabled,
                        isInvalid: isInvalid,
                        size: size,
                        styles: function anonymous(props) {
                            let rules = " ";
                            rules += ' zXcEWr131';
                            rules += ' yXcEWr131';
                            rules += ' BXcEWr131';
                            rules += ' AXcEWr131';
                            if (props.size === "XL") rules += ' Nl131';
                            else if (props.size === "L") rules += ' Nj131';
                            else if (props.size === "S") rules += ' Nf131';
                            else if (props.size === "XS") rules += ' Ne131';
                            else rules += ' Nk131';
                            if (props.isQuiet) rules += ' _qd131';
                            else rules += ' _qtb131';
                            if (props.size === "XL") rules += ' -Anxdcc-Zt131';
                            else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
                            else if (props.size === "S") rules += ' -Anxdcc-Zr131';
                            else rules += ' -Anxdcc-Z3fn9H131';
                            rules += ' __v6TNLWb131';
                            rules += ' __Wb131';
                            rules += ' SMBFGYc131';
                            rules += ' RfBFGYc131';
                            rules += ' -S_-Sv131';
                            if (props.size === "XL") rules += ' -R_-RWJpoB131';
                            else if (props.size === "L") rules += ' -R_-RWJpoB131';
                            else if (props.size === "M") rules += ' -R_-Rp131';
                            else if (props.size === "S") rules += ' -R_-Rj131';
                            return rules;
                        }({
                            size: size
                        }),
                        children: [
                            /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $d19642654b699f84$exports.DateInputContainer), {
                                children: /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $d19642654b699f84$exports.DateInput), {})
                            }),
                            /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $d19642654b699f84$exports.InvalidIndicator), {
                                isInvalid: isInvalid,
                                isDisabled: isDisabled
                            }),
                            /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)($e6aad0776f57e0e9$export$adb29da0ac001538, {
                                isOpen: isOpen,
                                size: size,
                                setButtonHasFocus: setButtonHasFocus
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsxs)($e6aad0776f57e0e9$export$f3ce207574a3cffe, {
                        shouldFlip: props1.shouldFlip,
                        children: [
                            /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $e9e9a0d15c3364d7$exports.Calendar), {
                                visibleMonths: maxVisibleMonths,
                                createCalendar: createCalendar,
                                errorMessage: resolvedErrorMessage
                            }),
                            showTimeField && /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)("div", {
                                className: " sd131 Ul131 ql131 _qb131",
                                children: /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $7b93ad26130ad3f8$exports.TimeField), {
                                    styles: $e6aad0776f57e0e9$export$5320dd0e349ed44d,
                                    label: stringFormatter.format('datepicker.time'),
                                    value: state.timeValue,
                                    // TODO: why do i need the cast?
                                    onChange: (v)=>state.setTimeValue(v),
                                    placeholderValue: timePlaceholder,
                                    granularity: timeGranularity,
                                    minValue: timeMinValue,
                                    maxValue: timeMaxValue,
                                    hourCycle: props1.hourCycle,
                                    hideTimeZone: props1.hideTimeZone
                                })
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.HelpText), {
                        size: size,
                        isDisabled: isDisabled,
                        isInvalid: isInvalid,
                        description: descriptionMessage,
                        children: resolvedErrorMessage
                    })
                ]
            });
        }
    });
});
function $e6aad0776f57e0e9$export$f3ce207574a3cffe(props) {
    return /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $493371ef39bf7a55$exports.Popover), {
        ...props,
        hideArrow: true,
        padding: "none",
        children: /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)("div", {
            className: " Sf131 Rf131 Th131 Qh131 _Na131 Pa131 sd131 _ta131 Ul131 ql131 _oa131 Za131 Fb131",
            children: /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $2B5ON$reactariacomponentsDialog.Dialog), {
                children: /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $2B5ON$reactariacomponentsslots.Provider), {
                    values: [
                        [
                            (0, $2B5ON$reactariacomponentsDialog.OverlayTriggerStateContext),
                            null
                        ]
                    ],
                    children: props.children
                })
            })
        })
    });
}
function $e6aad0776f57e0e9$export$adb29da0ac001538(props) {
    let buttonRef = (0, $2B5ON$react.useRef)(null);
    let { isOpen: isOpen, size: size, setButtonHasFocus: setButtonHasFocus } = props;
    return /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $2B5ON$reactariacomponentsButton.Button), {
        ref: buttonRef,
        // Prevent press scale from sticking while DatePicker is open.
        // @ts-ignore
        isPressed: false,
        onFocusChange: setButtonHasFocus,
        style: (0, $2061c83559b50a66$exports.pressScale)(buttonRef),
        className: (renderProps)=>$e6aad0776f57e0e9$var$inputButton({
                ...renderProps,
                size: size,
                isOpen: isOpen
            }),
        children: /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $2B5ON$reactariacomponentsslots.Provider), {
            values: [
                [
                    (0, $bde97c91243ed164$exports.IconContext),
                    {
                        styles: " -_8sjo0b-t5ZbAob131 Za8Mfn131 Fa8Mfn131"
                    }
                ]
            ],
            children: /*#__PURE__*/ (0, $2B5ON$reactjsxruntime.jsx)((0, $1dfd5738cdf83d1e$exports.default), {})
        })
    });
}
