// @react-spectrum/s2/dist/private/DateField.cjs
require("./DateField.css");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $fCG98$reactjsxruntime = require("react/jsx-runtime");
var $fCG98$reactariacomponentsDateField = require("react-aria-components/DateField");
var $fCG98$react = require("react");
var $fCG98$reactariacomponentsForm = require("react-aria-components/Form");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "DateFieldContext", () => $d19642654b699f84$export$7b3e670c86da5fe8);
$parcel$export(module.exports, "DateField", () => $d19642654b699f84$export$d9781c7894a82487);
$parcel$export(module.exports, "DateInputContainer", () => $d19642654b699f84$export$3f08d3c48fa697);
$parcel$export(module.exports, "DateInput", () => $d19642654b699f84$export$7edc06cf1783b30f);
$parcel$export(module.exports, "InvalidIndicator", () => $d19642654b699f84$export$19696856283ab185);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





const $d19642654b699f84$export$7b3e670c86da5fe8 = /*#__PURE__*/ (0, $fCG98$react.createContext)(null);
const $d19642654b699f84$var$segmentContainer = " _ub131 _vb131 Na131 Fb131 _Na131 Pc131 -dLGswc-h3TDrc131 sd131 eb131 __Wb131";
const $d19642654b699f84$var$dateSegment = function anonymous(props) {
    let rules = " ";
    rules += ' _Le131';
    rules += ' -zNggOc-RWHrbc131';
    if (props.isFocused) rules += ' g_h131';
    else rules += ' g9131';
    if (props.isFocused) rules += ' gle131';
    else rules += ' gl9131';
    if (props.isFocused) rules += ' px131';
    if (props.isFocused) rules += ' ple131';
    rules += ' oLvNfn131';
    rules += ' nLvNfn131';
    rules += ' kLvNfn131';
    rules += ' jLvNfn131';
    if (props.isPunctuation) rules += ' Sd131';
    else rules += ' Sj131';
    if (props.isPunctuation) rules += ' Rd131';
    else rules += ' Rj131';
    rules += ' Tj131';
    rules += ' Qj131';
    rules += ' _yb131';
    return rules;
};
const $d19642654b699f84$var$iconStyles = " sd131 eb131 _Cb131";
const $d19642654b699f84$export$d9781c7894a82487 = /*#__PURE__*/ (0, $fCG98$react.forwardRef)(function DateField(props1, ref) {
    [props1, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props1, ref, $d19642654b699f84$export$7b3e670c86da5fe8);
    let { label: label, contextualHelp: contextualHelp, description: descriptionMessage, errorMessage: errorMessage, isRequired: isRequired, size: size = 'M', labelPosition: labelPosition = 'top', necessityIndicator: necessityIndicator, labelAlign: labelAlign = 'start', UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className, styles: styles, ...dateFieldProps } = props1;
    let formContext = (0, $fCG98$react.useContext)((0, $fCG98$reactariacomponentsForm.FormContext));
    return /*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsx)((0, $fCG98$reactariacomponentsDateField.DateField), {
        ref: ref,
        isRequired: isRequired,
        ...dateFieldProps,
        style: UNSAFE_style,
        className: (UNSAFE_className || '') + function anonymous(props, overrides) {
            let rules = " ";
            let gridColumnStart = false;
            let gridColumnEnd = false;
            let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
            for (let p of matches){
                if (p[1] === "z") gridColumnStart = true;
                if (p[1] === "y") gridColumnEnd = true;
                rules += p[0];
            }
            rules += ' se131';
            if (props.isInForm) {
                if (!gridColumnStart) rules += ' z2mU131';
            }
            if (props.isInForm) {
                if (props.labelPosition === "side") {
                    if (!gridColumnEnd) rules += ' yvoofG131';
                }
            }
            if (props.isInForm) rules += ' DLwlgod131';
            else {
                if (props.labelPosition === "side") rules += ' DPlN5qb131';
                else if (props.labelPosition === "top") rules += ' DM8Mfn131';
            }
            if (props.labelPosition === "side") rules += ' EPlN5qb131';
            else if (props.labelPosition === "top") rules += ' E4w1sLc131';
            if (props.labelPosition === "side") rules += ' CbRM95c131';
            else if (props.labelPosition === "top") rules += ' Cxaocre131';
            if (props.size === "XL") {
                rules += ' -_6BNtrc-e131';
                rules += ' vx131';
            } else if (props.size === "L") {
                rules += ' -_6BNtrc-d131';
                rules += ' vx131';
            } else if (props.size === "S") {
                rules += ' -_6BNtrc-b131';
                rules += ' vx131';
            } else if (props.size === "XS") {
                rules += ' -_6BNtrc-a131';
                rules += ' vx131';
            } else {
                rules += ' -_6BNtrc-c131';
                rules += ' vx131';
            }
            rules += ' ea131';
            rules += ' _Fd131';
            rules += ' _FnuYUweb131';
            if (props.size === "XL") rules += ' -BhX7R-Fz131';
            else if (props.size === "L") rules += ' -BhX7R-Fv131';
            else if (props.size === "S") rules += ' -BhX7R-Fp131';
            else if (props.size === "XS") rules += ' -BhX7R-Fn131';
            else rules += ' -BhX7R-Fx131';
            rules += ' -AJjOLd-UVkF0Db131';
            rules += ' qg131';
            rules += ' __ca131';
            return rules;
        }({
            isInForm: !!formContext,
            labelPosition: labelPosition,
            size: size
        }, styles),
        children: ({ isDisabled: isDisabled, isInvalid: isInvalid })=>{
            return /*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsxs)((0, $fCG98$reactjsxruntime.Fragment), {
                children: [
                    /*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldLabel), {
                        isDisabled: isDisabled,
                        isRequired: isRequired,
                        size: size,
                        labelPosition: labelPosition,
                        labelAlign: labelAlign,
                        necessityIndicator: necessityIndicator,
                        contextualHelp: contextualHelp,
                        children: label
                    }),
                    /*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsxs)((0, $a0b996a550bb0891$exports.FieldGroup), {
                        role: "presentation",
                        isDisabled: isDisabled,
                        isInvalid: isInvalid,
                        size: size,
                        styles: function anonymous(props) {
                            let rules = " ";
                            rules += ' zXcEWr131';
                            rules += ' yXcEWr131';
                            rules += ' BXcEWr131';
                            rules += ' AXcEWr131';
                            if (props.size === "XL") rules += ' Nl131';
                            else if (props.size === "L") rules += ' Nj131';
                            else if (props.size === "S") rules += ' Nf131';
                            else if (props.size === "XS") rules += ' Ne131';
                            else rules += ' Nk131';
                            if (props.isQuiet) rules += ' _qd131';
                            else rules += ' _qtb131';
                            if (props.size === "XL") rules += ' -Anxdcc-Zt131';
                            else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
                            else if (props.size === "S") rules += ' -Anxdcc-Zr131';
                            else rules += ' -Anxdcc-Z3fn9H131';
                            rules += ' __v6TNLWb131';
                            rules += ' __Wb131';
                            rules += ' SMBFGYc131';
                            rules += ' RfBFGYc131';
                            rules += ' -S_-Sv131';
                            rules += ' -R_-Rv131';
                            return rules;
                        }({
                            size: size
                        }),
                        children: [
                            /*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsx)($d19642654b699f84$export$3f08d3c48fa697, {
                                children: /*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsx)($d19642654b699f84$export$7edc06cf1783b30f, {})
                            }),
                            /*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsx)($d19642654b699f84$export$19696856283ab185, {
                                isInvalid: isInvalid,
                                isDisabled: isDisabled
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.HelpText), {
                        size: size,
                        isDisabled: isDisabled,
                        isInvalid: isInvalid,
                        description: descriptionMessage,
                        children: errorMessage
                    })
                ]
            });
        }
    });
});
function $d19642654b699f84$export$3f08d3c48fa697(props) {
    return /*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsx)("div", {
        className: $d19642654b699f84$var$segmentContainer,
        children: props.children
    });
}
function $d19642654b699f84$export$7edc06cf1783b30f(props) {
    return /*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsx)((0, $fCG98$reactariacomponentsDateField.DateInput), {
        className: "",
        ...props,
        children: (segment)=>/*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsx)((0, $fCG98$reactariacomponentsDateField.DateSegment), {
                segment: segment,
                className: (renderProps)=>$d19642654b699f84$var$dateSegment({
                        ...renderProps,
                        isPunctuation: segment.type === 'literal'
                    })
            })
    });
}
function $d19642654b699f84$export$19696856283ab185(props) {
    return props.isInvalid ? /*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsx)("div", {
        className: $d19642654b699f84$var$iconStyles,
        children: /*#__PURE__*/ (0, $fCG98$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldErrorIcon), {
            isDisabled: props.isDisabled
        })
    }) : null;
}
