// @react-spectrum/s2/dist/private/DateField.mjs
import "./DateField.css";
import {FieldErrorIcon as $9afd80978c252040$export$bda0d075e0e342c9, FieldGroup as $9afd80978c252040$export$f0fe469072bc6950, FieldLabel as $9afd80978c252040$export$1acdcf5a973a8414, HelpText as $9afd80978c252040$export$a67c0bc59081311a} from "./Field.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $7wrC3$jsx, jsxs as $7wrC3$jsxs, Fragment as $7wrC3$Fragment} from "react/jsx-runtime";
import {DateField as $7wrC3$DateField, DateInput as $7wrC3$DateInput, DateSegment as $7wrC3$DateSegment} from "react-aria-components/DateField";
import {createContext as $7wrC3$createContext, forwardRef as $7wrC3$forwardRef, useContext as $7wrC3$useContext} from "react";
import {FormContext as $7wrC3$FormContext} from "react-aria-components/Form";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





const $0e78e2b1ea2e947e$export$7b3e670c86da5fe8 = /*#__PURE__*/ (0, $7wrC3$createContext)(null);
const $0e78e2b1ea2e947e$var$segmentContainer = " _ub131 _vb131 Na131 Fb131 _Na131 Pc131 -dLGswc-h3TDrc131 sd131 eb131 __Wb131";
const $0e78e2b1ea2e947e$var$dateSegment = function anonymous(props) {
    let rules = " ";
    rules += ' _Le131';
    rules += ' -zNggOc-RWHrbc131';
    if (props.isFocused) rules += ' g_h131';
    else rules += ' g9131';
    if (props.isFocused) rules += ' gle131';
    else rules += ' gl9131';
    if (props.isFocused) rules += ' px131';
    if (props.isFocused) rules += ' ple131';
    rules += ' oLvNfn131';
    rules += ' nLvNfn131';
    rules += ' kLvNfn131';
    rules += ' jLvNfn131';
    if (props.isPunctuation) rules += ' Sd131';
    else rules += ' Sj131';
    if (props.isPunctuation) rules += ' Rd131';
    else rules += ' Rj131';
    rules += ' Tj131';
    rules += ' Qj131';
    rules += ' _yb131';
    return rules;
};
const $0e78e2b1ea2e947e$var$iconStyles = " sd131 eb131 _Cb131";
const $0e78e2b1ea2e947e$export$d9781c7894a82487 = /*#__PURE__*/ (0, $7wrC3$forwardRef)(function DateField(props1, ref) {
    [props1, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props1, ref, $0e78e2b1ea2e947e$export$7b3e670c86da5fe8);
    let { label: label, contextualHelp: contextualHelp, description: descriptionMessage, errorMessage: errorMessage, isRequired: isRequired, size: size = 'M', labelPosition: labelPosition = 'top', necessityIndicator: necessityIndicator, labelAlign: labelAlign = 'start', UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className, styles: styles, ...dateFieldProps } = props1;
    let formContext = (0, $7wrC3$useContext)((0, $7wrC3$FormContext));
    return /*#__PURE__*/ (0, $7wrC3$jsx)((0, $7wrC3$DateField), {
        ref: ref,
        isRequired: isRequired,
        ...dateFieldProps,
        style: UNSAFE_style,
        className: (UNSAFE_className || '') + function anonymous(props, overrides) {
            let rules = " ";
            let gridColumnStart = false;
            let gridColumnEnd = false;
            let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
            for (let p of matches){
                if (p[1] === "z") gridColumnStart = true;
                if (p[1] === "y") gridColumnEnd = true;
                rules += p[0];
            }
            rules += ' se131';
            if (props.isInForm) {
                if (!gridColumnStart) rules += ' z2mU131';
            }
            if (props.isInForm) {
                if (props.labelPosition === "side") {
                    if (!gridColumnEnd) rules += ' yvoofG131';
                }
            }
            if (props.isInForm) rules += ' DLwlgod131';
            else {
                if (props.labelPosition === "side") rules += ' DPlN5qb131';
                else if (props.labelPosition === "top") rules += ' DM8Mfn131';
            }
            if (props.labelPosition === "side") rules += ' EPlN5qb131';
            else if (props.labelPosition === "top") rules += ' E4w1sLc131';
            if (props.labelPosition === "side") rules += ' CbRM95c131';
            else if (props.labelPosition === "top") rules += ' Cxaocre131';
            if (props.size === "XL") {
                rules += ' -_6BNtrc-e131';
                rules += ' vx131';
            } else if (props.size === "L") {
                rules += ' -_6BNtrc-d131';
                rules += ' vx131';
            } else if (props.size === "S") {
                rules += ' -_6BNtrc-b131';
                rules += ' vx131';
            } else if (props.size === "XS") {
                rules += ' -_6BNtrc-a131';
                rules += ' vx131';
            } else {
                rules += ' -_6BNtrc-c131';
                rules += ' vx131';
            }
            rules += ' ea131';
            rules += ' _Fd131';
            rules += ' _FnuYUweb131';
            if (props.size === "XL") rules += ' -BhX7R-Fz131';
            else if (props.size === "L") rules += ' -BhX7R-Fv131';
            else if (props.size === "S") rules += ' -BhX7R-Fp131';
            else if (props.size === "XS") rules += ' -BhX7R-Fn131';
            else rules += ' -BhX7R-Fx131';
            rules += ' -AJjOLd-UVkF0Db131';
            rules += ' qg131';
            rules += ' __ca131';
            return rules;
        }({
            isInForm: !!formContext,
            labelPosition: labelPosition,
            size: size
        }, styles),
        children: ({ isDisabled: isDisabled, isInvalid: isInvalid })=>{
            return /*#__PURE__*/ (0, $7wrC3$jsxs)((0, $7wrC3$Fragment), {
                children: [
                    /*#__PURE__*/ (0, $7wrC3$jsx)((0, $9afd80978c252040$export$1acdcf5a973a8414), {
                        isDisabled: isDisabled,
                        isRequired: isRequired,
                        size: size,
                        labelPosition: labelPosition,
                        labelAlign: labelAlign,
                        necessityIndicator: necessityIndicator,
                        contextualHelp: contextualHelp,
                        children: label
                    }),
                    /*#__PURE__*/ (0, $7wrC3$jsxs)((0, $9afd80978c252040$export$f0fe469072bc6950), {
                        role: "presentation",
                        isDisabled: isDisabled,
                        isInvalid: isInvalid,
                        size: size,
                        styles: function anonymous(props) {
                            let rules = " ";
                            rules += ' zXcEWr131';
                            rules += ' yXcEWr131';
                            rules += ' BXcEWr131';
                            rules += ' AXcEWr131';
                            if (props.size === "XL") rules += ' Nl131';
                            else if (props.size === "L") rules += ' Nj131';
                            else if (props.size === "S") rules += ' Nf131';
                            else if (props.size === "XS") rules += ' Ne131';
                            else rules += ' Nk131';
                            if (props.isQuiet) rules += ' _qd131';
                            else rules += ' _qtb131';
                            if (props.size === "XL") rules += ' -Anxdcc-Zt131';
                            else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
                            else if (props.size === "S") rules += ' -Anxdcc-Zr131';
                            else rules += ' -Anxdcc-Z3fn9H131';
                            rules += ' __v6TNLWb131';
                            rules += ' __Wb131';
                            rules += ' SMBFGYc131';
                            rules += ' RfBFGYc131';
                            rules += ' -S_-Sv131';
                            rules += ' -R_-Rv131';
                            return rules;
                        }({
                            size: size
                        }),
                        children: [
                            /*#__PURE__*/ (0, $7wrC3$jsx)($0e78e2b1ea2e947e$export$3f08d3c48fa697, {
                                children: /*#__PURE__*/ (0, $7wrC3$jsx)($0e78e2b1ea2e947e$export$7edc06cf1783b30f, {})
                            }),
                            /*#__PURE__*/ (0, $7wrC3$jsx)($0e78e2b1ea2e947e$export$19696856283ab185, {
                                isInvalid: isInvalid,
                                isDisabled: isDisabled
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $7wrC3$jsx)((0, $9afd80978c252040$export$a67c0bc59081311a), {
                        size: size,
                        isDisabled: isDisabled,
                        isInvalid: isInvalid,
                        description: descriptionMessage,
                        children: errorMessage
                    })
                ]
            });
        }
    });
});
function $0e78e2b1ea2e947e$export$3f08d3c48fa697(props) {
    return /*#__PURE__*/ (0, $7wrC3$jsx)("div", {
        className: $0e78e2b1ea2e947e$var$segmentContainer,
        children: props.children
    });
}
function $0e78e2b1ea2e947e$export$7edc06cf1783b30f(props) {
    return /*#__PURE__*/ (0, $7wrC3$jsx)((0, $7wrC3$DateInput), {
        className: "",
        ...props,
        children: (segment)=>/*#__PURE__*/ (0, $7wrC3$jsx)((0, $7wrC3$DateSegment), {
                segment: segment,
                className: (renderProps)=>$0e78e2b1ea2e947e$var$dateSegment({
                        ...renderProps,
                        isPunctuation: segment.type === 'literal'
                    })
            })
    });
}
function $0e78e2b1ea2e947e$export$19696856283ab185(props) {
    return props.isInvalid ? /*#__PURE__*/ (0, $7wrC3$jsx)("div", {
        className: $0e78e2b1ea2e947e$var$iconStyles,
        children: /*#__PURE__*/ (0, $7wrC3$jsx)((0, $9afd80978c252040$export$bda0d075e0e342c9), {
            isDisabled: props.isDisabled
        })
    }) : null;
}


export {$0e78e2b1ea2e947e$export$7b3e670c86da5fe8 as DateFieldContext, $0e78e2b1ea2e947e$export$d9781c7894a82487 as DateField, $0e78e2b1ea2e947e$export$3f08d3c48fa697 as DateInputContainer, $0e78e2b1ea2e947e$export$7edc06cf1783b30f as DateInput, $0e78e2b1ea2e947e$export$19696856283ab185 as InvalidIndicator};
