// @react-spectrum/s2/dist/private/Dialog.cjs
require("./Dialog.css");
var $e1016b8947dbadb8$exports = require("./ButtonGroup.cjs");
var $2f907cb84c6e9e75$exports = require("./CloseButton.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $053b76ed3d29e13b$exports = require("./Image.cjs");
var $8e678305a8c10028$exports = require("./Modal.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $WjfVF$reactjsxruntime = require("react/jsx-runtime");
var $WjfVF$reactariacomponentscomposeRenderProps = require("react-aria-components/composeRenderProps");
var $WjfVF$react = require("react");
var $WjfVF$reactariacomponentsDialog = require("react-aria-components/Dialog");
var $WjfVF$reactariacomponentsslots = require("react-aria-components/slots");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "dialogInner", () => $0f4636c7093b9233$export$6af31f836f9aa6f2);
$parcel$export(module.exports, "Dialog", () => $0f4636c7093b9233$export$3ddf2d174ce01153);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 










const $0f4636c7093b9233$var$image = " Za131 FC131 _va131 _Hb131";
const $0f4636c7093b9233$var$heading = " _ub131 Jy131 Gy131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-g131 vx131 xd131 _xa131 xX0cczbc131 _xX0cczba131 xfd131 _xfa131 xfX0cczbc131 _xfX0cczba131 _Fd131 _FnuYUweb131 po131";
const $0f4636c7093b9233$var$header = " uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-d131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131";
const $0f4636c7093b9233$var$content = " _ub131 _v7GYNyba131 Pa131 P7GYNybe131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-d131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131 Sk131 Rk131";
const $0f4636c7093b9233$var$footer = " _ub131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-d131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131";
const $0f4636c7093b9233$var$buttonGroup = " IM131 La131";
const $0f4636c7093b9233$export$6af31f836f9aa6f2 = " sd131 _ta131 _ub131 K40ub4c131 _oa131 _Le131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 oi131 ni131 ki131 ji131 _Na131 Pa131";
const $0f4636c7093b9233$export$3ddf2d174ce01153 = /*#__PURE__*/ (0, $WjfVF$react.forwardRef)(function Dialog(props, ref) {
    let { size: size = 'M', isDismissible: isDismissible, isKeyboardDismissDisabled: isKeyboardDismissDisabled } = props;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    return /*#__PURE__*/ (0, $WjfVF$reactjsxruntime.jsx)((0, $8e678305a8c10028$exports.Modal), {
        size: size,
        isDismissable: isDismissible,
        isKeyboardDismissDisabled: isKeyboardDismissDisabled,
        children: /*#__PURE__*/ (0, $WjfVF$reactjsxruntime.jsx)((0, $WjfVF$reactariacomponentsDialog.Dialog), {
            ...props,
            ref: domRef,
            style: props.UNSAFE_style,
            className: (props.UNSAFE_className || '') + $0f4636c7093b9233$export$6af31f836f9aa6f2,
            children: (0, $WjfVF$reactariacomponentscomposeRenderProps.composeRenderProps)(props.children, (children)=>// Render the children multiple times inside the wrappers we need to implement the layout.
                // Each instance hides certain children so that they are all rendered in the correct locations.
                // Reset OverlayTriggerStateContext so the buttons inside the dialog don't retain their hover state.
                /*#__PURE__*/ (0, $WjfVF$reactjsxruntime.jsxs)((0, $WjfVF$reactariacomponentsDialog.OverlayTriggerStateContext).Provider, {
                    value: null,
                    children: [
                        /*#__PURE__*/ (0, $WjfVF$reactjsxruntime.jsx)((0, $WjfVF$reactariacomponentsslots.Provider), {
                            values: [
                                [
                                    (0, $053b76ed3d29e13b$exports.ImageContext),
                                    {
                                        styles: $0f4636c7093b9233$var$image
                                    }
                                ],
                                [
                                    (0, $6367bc87eb7d24ad$exports.HeadingContext),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $6367bc87eb7d24ad$exports.HeaderContext),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $6367bc87eb7d24ad$exports.ContentContext),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $6367bc87eb7d24ad$exports.FooterContext),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $e1016b8947dbadb8$exports.ButtonGroupContext),
                                    {
                                        isHidden: true
                                    }
                                ]
                            ],
                            children: children
                        }),
                        /*#__PURE__*/ (0, $WjfVF$reactjsxruntime.jsxs)("div", {
                            className: function anonymous(props) {
                                let rules = " ";
                                rules += ' sd131';
                                rules += ' ed131';
                                rules += ' qg131';
                                rules += ' Sk131';
                                if (props.isDismissible) rules += ' Re131';
                                else rules += ' Rk131';
                                rules += ' Te131';
                                return rules;
                            }({
                                isDismissible: props.isDismissible
                            }),
                            children: [
                                /*#__PURE__*/ (0, $WjfVF$reactjsxruntime.jsx)("div", {
                                    className: " sd131 _ub131 Jz131 Jay131 GC131 Gay131 qj131 Ue131 _ta131 _tnc131 ed131 enb131",
                                    children: /*#__PURE__*/ (0, $WjfVF$reactjsxruntime.jsx)((0, $WjfVF$reactariacomponentsslots.Provider), {
                                        values: [
                                            [
                                                (0, $053b76ed3d29e13b$exports.ImageContext),
                                                {
                                                    hidden: true
                                                }
                                            ],
                                            [
                                                (0, $6367bc87eb7d24ad$exports.HeadingContext),
                                                {
                                                    styles: $0f4636c7093b9233$var$heading
                                                }
                                            ],
                                            [
                                                (0, $6367bc87eb7d24ad$exports.HeaderContext),
                                                {
                                                    styles: $0f4636c7093b9233$var$header
                                                }
                                            ],
                                            [
                                                (0, $6367bc87eb7d24ad$exports.ContentContext),
                                                {
                                                    isHidden: true
                                                }
                                            ],
                                            [
                                                (0, $6367bc87eb7d24ad$exports.FooterContext),
                                                {
                                                    isHidden: true
                                                }
                                            ],
                                            [
                                                (0, $e1016b8947dbadb8$exports.ButtonGroupContext),
                                                {
                                                    isHidden: true
                                                }
                                            ]
                                        ],
                                        children: children
                                    })
                                }),
                                props.isDismissible && /*#__PURE__*/ (0, $WjfVF$reactjsxruntime.jsx)((0, $2f907cb84c6e9e75$exports.CloseButton), {
                                    styles: " Gx131"
                                })
                            ]
                        }),
                        /*#__PURE__*/ (0, $WjfVF$reactjsxruntime.jsx)((0, $WjfVF$reactariacomponentsslots.Provider), {
                            values: [
                                [
                                    (0, $053b76ed3d29e13b$exports.ImageContext),
                                    {
                                        hidden: true
                                    }
                                ],
                                [
                                    (0, $6367bc87eb7d24ad$exports.HeadingContext),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $6367bc87eb7d24ad$exports.HeaderContext),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $6367bc87eb7d24ad$exports.ContentContext),
                                    {
                                        styles: $0f4636c7093b9233$var$content
                                    }
                                ],
                                [
                                    (0, $6367bc87eb7d24ad$exports.FooterContext),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $e1016b8947dbadb8$exports.ButtonGroupContext),
                                    {
                                        isHidden: true
                                    }
                                ]
                            ],
                            children: children
                        }),
                        /*#__PURE__*/ (0, $WjfVF$reactjsxruntime.jsx)("div", {
                            className: " sd131 Sk131 Rk131 Qk131 Tk131 Tad131 Uj131 qj131 eb131 _wb131",
                            children: /*#__PURE__*/ (0, $WjfVF$reactjsxruntime.jsx)((0, $WjfVF$reactariacomponentsslots.Provider), {
                                values: [
                                    [
                                        (0, $053b76ed3d29e13b$exports.ImageContext),
                                        {
                                            hidden: true
                                        }
                                    ],
                                    [
                                        (0, $6367bc87eb7d24ad$exports.HeadingContext),
                                        {
                                            isHidden: true
                                        }
                                    ],
                                    [
                                        (0, $6367bc87eb7d24ad$exports.HeaderContext),
                                        {
                                            isHidden: true
                                        }
                                    ],
                                    [
                                        (0, $6367bc87eb7d24ad$exports.ContentContext),
                                        {
                                            isHidden: true
                                        }
                                    ],
                                    [
                                        (0, $6367bc87eb7d24ad$exports.FooterContext),
                                        {
                                            styles: $0f4636c7093b9233$var$footer
                                        }
                                    ],
                                    [
                                        (0, $e1016b8947dbadb8$exports.ButtonGroupContext),
                                        {
                                            isHidden: props.isDismissible,
                                            styles: $0f4636c7093b9233$var$buttonGroup,
                                            align: 'end'
                                        }
                                    ]
                                ],
                                children: children
                            })
                        })
                    ]
                }))
        })
    });
});
