// @react-spectrum/s2/dist/private/Dialog.mjs
import "./Dialog.css";
import {ButtonGroupContext as $472eaa9a9b344b9a$export$4c574c91bb0be0e2} from "./ButtonGroup.mjs";
import {CloseButton as $a9cda54c4f47ce52$export$de65de8213222d10} from "./CloseButton.mjs";
import {ContentContext as $8e847109a6ab556d$export$1cbdd774077931b4, FooterContext as $8e847109a6ab556d$export$5630640b68817ed6, HeaderContext as $8e847109a6ab556d$export$e0e4026c12a8bdbb, HeadingContext as $8e847109a6ab556d$export$d688439359537581} from "./Content.mjs";
import {ImageContext as $dbd6f0b2503b938c$export$8151eb379fcf2442} from "./Image.mjs";
import {Modal as $c46d159bd615e407$export$2b77a92f1a5ad772} from "./Modal.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {jsx as $9Rtwv$jsx, jsxs as $9Rtwv$jsxs} from "react/jsx-runtime";
import {composeRenderProps as $9Rtwv$composeRenderProps} from "react-aria-components/composeRenderProps";
import {forwardRef as $9Rtwv$forwardRef} from "react";
import {Dialog as $9Rtwv$Dialog, OverlayTriggerStateContext as $9Rtwv$OverlayTriggerStateContext} from "react-aria-components/Dialog";
import {Provider as $9Rtwv$Provider} from "react-aria-components/slots";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 










const $9bbc115952dac5a6$var$image = " Za131 FC131 _va131 _Hb131";
const $9bbc115952dac5a6$var$heading = " _ub131 Jy131 Gy131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-g131 vx131 xd131 _xa131 xX0cczbc131 _xX0cczba131 xfd131 _xfa131 xfX0cczbc131 _xfX0cczba131 _Fd131 _FnuYUweb131 po131";
const $9bbc115952dac5a6$var$header = " uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-d131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131";
const $9bbc115952dac5a6$var$content = " _ub131 _v7GYNyba131 Pa131 P7GYNybe131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-d131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131 Sk131 Rk131";
const $9bbc115952dac5a6$var$footer = " _ub131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-d131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131";
const $9bbc115952dac5a6$var$buttonGroup = " IM131 La131";
const $9bbc115952dac5a6$export$6af31f836f9aa6f2 = " sd131 _ta131 _ub131 K40ub4c131 _oa131 _Le131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 oi131 ni131 ki131 ji131 _Na131 Pa131";
const $9bbc115952dac5a6$export$3ddf2d174ce01153 = /*#__PURE__*/ (0, $9Rtwv$forwardRef)(function Dialog(props, ref) {
    let { size: size = 'M', isDismissible: isDismissible, isKeyboardDismissDisabled: isKeyboardDismissDisabled } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    return /*#__PURE__*/ (0, $9Rtwv$jsx)((0, $c46d159bd615e407$export$2b77a92f1a5ad772), {
        size: size,
        isDismissable: isDismissible,
        isKeyboardDismissDisabled: isKeyboardDismissDisabled,
        children: /*#__PURE__*/ (0, $9Rtwv$jsx)((0, $9Rtwv$Dialog), {
            ...props,
            ref: domRef,
            style: props.UNSAFE_style,
            className: (props.UNSAFE_className || '') + $9bbc115952dac5a6$export$6af31f836f9aa6f2,
            children: (0, $9Rtwv$composeRenderProps)(props.children, (children)=>// Render the children multiple times inside the wrappers we need to implement the layout.
                // Each instance hides certain children so that they are all rendered in the correct locations.
                // Reset OverlayTriggerStateContext so the buttons inside the dialog don't retain their hover state.
                /*#__PURE__*/ (0, $9Rtwv$jsxs)((0, $9Rtwv$OverlayTriggerStateContext).Provider, {
                    value: null,
                    children: [
                        /*#__PURE__*/ (0, $9Rtwv$jsx)((0, $9Rtwv$Provider), {
                            values: [
                                [
                                    (0, $dbd6f0b2503b938c$export$8151eb379fcf2442),
                                    {
                                        styles: $9bbc115952dac5a6$var$image
                                    }
                                ],
                                [
                                    (0, $8e847109a6ab556d$export$d688439359537581),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $8e847109a6ab556d$export$e0e4026c12a8bdbb),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $8e847109a6ab556d$export$1cbdd774077931b4),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $8e847109a6ab556d$export$5630640b68817ed6),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $472eaa9a9b344b9a$export$4c574c91bb0be0e2),
                                    {
                                        isHidden: true
                                    }
                                ]
                            ],
                            children: children
                        }),
                        /*#__PURE__*/ (0, $9Rtwv$jsxs)("div", {
                            className: function anonymous(props) {
                                let rules = " ";
                                rules += ' sd131';
                                rules += ' ed131';
                                rules += ' qg131';
                                rules += ' Sk131';
                                if (props.isDismissible) rules += ' Re131';
                                else rules += ' Rk131';
                                rules += ' Te131';
                                return rules;
                            }({
                                isDismissible: props.isDismissible
                            }),
                            children: [
                                /*#__PURE__*/ (0, $9Rtwv$jsx)("div", {
                                    className: " sd131 _ub131 Jz131 Jay131 GC131 Gay131 qj131 Ue131 _ta131 _tnc131 ed131 enb131",
                                    children: /*#__PURE__*/ (0, $9Rtwv$jsx)((0, $9Rtwv$Provider), {
                                        values: [
                                            [
                                                (0, $dbd6f0b2503b938c$export$8151eb379fcf2442),
                                                {
                                                    hidden: true
                                                }
                                            ],
                                            [
                                                (0, $8e847109a6ab556d$export$d688439359537581),
                                                {
                                                    styles: $9bbc115952dac5a6$var$heading
                                                }
                                            ],
                                            [
                                                (0, $8e847109a6ab556d$export$e0e4026c12a8bdbb),
                                                {
                                                    styles: $9bbc115952dac5a6$var$header
                                                }
                                            ],
                                            [
                                                (0, $8e847109a6ab556d$export$1cbdd774077931b4),
                                                {
                                                    isHidden: true
                                                }
                                            ],
                                            [
                                                (0, $8e847109a6ab556d$export$5630640b68817ed6),
                                                {
                                                    isHidden: true
                                                }
                                            ],
                                            [
                                                (0, $472eaa9a9b344b9a$export$4c574c91bb0be0e2),
                                                {
                                                    isHidden: true
                                                }
                                            ]
                                        ],
                                        children: children
                                    })
                                }),
                                props.isDismissible && /*#__PURE__*/ (0, $9Rtwv$jsx)((0, $a9cda54c4f47ce52$export$de65de8213222d10), {
                                    styles: " Gx131"
                                })
                            ]
                        }),
                        /*#__PURE__*/ (0, $9Rtwv$jsx)((0, $9Rtwv$Provider), {
                            values: [
                                [
                                    (0, $dbd6f0b2503b938c$export$8151eb379fcf2442),
                                    {
                                        hidden: true
                                    }
                                ],
                                [
                                    (0, $8e847109a6ab556d$export$d688439359537581),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $8e847109a6ab556d$export$e0e4026c12a8bdbb),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $8e847109a6ab556d$export$1cbdd774077931b4),
                                    {
                                        styles: $9bbc115952dac5a6$var$content
                                    }
                                ],
                                [
                                    (0, $8e847109a6ab556d$export$5630640b68817ed6),
                                    {
                                        isHidden: true
                                    }
                                ],
                                [
                                    (0, $472eaa9a9b344b9a$export$4c574c91bb0be0e2),
                                    {
                                        isHidden: true
                                    }
                                ]
                            ],
                            children: children
                        }),
                        /*#__PURE__*/ (0, $9Rtwv$jsx)("div", {
                            className: " sd131 Sk131 Rk131 Qk131 Tk131 Tad131 Uj131 qj131 eb131 _wb131",
                            children: /*#__PURE__*/ (0, $9Rtwv$jsx)((0, $9Rtwv$Provider), {
                                values: [
                                    [
                                        (0, $dbd6f0b2503b938c$export$8151eb379fcf2442),
                                        {
                                            hidden: true
                                        }
                                    ],
                                    [
                                        (0, $8e847109a6ab556d$export$d688439359537581),
                                        {
                                            isHidden: true
                                        }
                                    ],
                                    [
                                        (0, $8e847109a6ab556d$export$e0e4026c12a8bdbb),
                                        {
                                            isHidden: true
                                        }
                                    ],
                                    [
                                        (0, $8e847109a6ab556d$export$1cbdd774077931b4),
                                        {
                                            isHidden: true
                                        }
                                    ],
                                    [
                                        (0, $8e847109a6ab556d$export$5630640b68817ed6),
                                        {
                                            styles: $9bbc115952dac5a6$var$footer
                                        }
                                    ],
                                    [
                                        (0, $472eaa9a9b344b9a$export$4c574c91bb0be0e2),
                                        {
                                            isHidden: props.isDismissible,
                                            styles: $9bbc115952dac5a6$var$buttonGroup,
                                            align: 'end'
                                        }
                                    ]
                                ],
                                children: children
                            })
                        })
                    ]
                }))
        })
    });
});


export {$9bbc115952dac5a6$export$6af31f836f9aa6f2 as dialogInner, $9bbc115952dac5a6$export$3ddf2d174ce01153 as Dialog};
