// @react-spectrum/s2/dist/private/Disclosure.mjs
import "./Disclosure.css";
import {ActionButtonContext as $da878a05ab4a403e$export$2ff682723a46d413} from "./ActionButton.mjs";
import {CenterBaseline as $1f4b04be3f24aae3$export$768dac55bb57081d} from "./CenterBaseline.mjs";
import $4b69f02ec06b9226$export$2e2bcd8739ae039 from "./Chevron.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $7VHtn$jsx, jsxs as $7VHtn$jsxs} from "react/jsx-runtime";
import {Button as $7VHtn$Button} from "react-aria-components/Button";
import {Provider as $7VHtn$Provider, useSlottedContext as $7VHtn$useSlottedContext} from "react-aria-components/slots";
import {Disclosure as $7VHtn$Disclosure, DisclosureStateContext as $7VHtn$DisclosureStateContext, DisclosurePanel as $7VHtn$DisclosurePanel} from "react-aria-components/Disclosure";
import {filterDOMProps as $7VHtn$filterDOMProps} from "react-aria/filterDOMProps";
import {Heading as $7VHtn$Heading} from "react-aria-components/Heading";
import {createContext as $7VHtn$createContext, forwardRef as $7VHtn$forwardRef, useContext as $7VHtn$useContext} from "react";
import {useLocale as $7VHtn$useLocale} from "react-aria/I18nProvider";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 












const $cf1d656285911959$export$d665dd135a51b28a = /*#__PURE__*/ (0, $7VHtn$createContext)(null);
const $cf1d656285911959$var$disclosure = function anonymous(props, overrides) {
    let rules = " ";
    let minWidth = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "N") minWidth = true;
        rules += p[0];
    }
    rules += ' po131';
    if (props.isQuiet) rules += ' _ka131';
    else rules += ' _kb131';
    if (props.isInGroup) {
        rules += ' ha131';
        if (props.isQuiet) rules += ' hka131';
        else rules += ' hkb131';
    } else if (props.isQuiet) rules += ' ha131';
    else rules += ' hb131';
    rules += ' ma131';
    rules += ' la131';
    rules += ' _jf131';
    rules += ' ir131';
    if (!minWidth) rules += ' Ng131';
    return rules;
};
const $cf1d656285911959$export$74a362b31437ec83 = /*#__PURE__*/ (0, $7VHtn$forwardRef)(function Disclosure(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $cf1d656285911959$export$d665dd135a51b28a);
    let { size: size = 'M', density: density = 'regular', isQuiet: isQuiet, UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '' } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let isInGroup = (0, $7VHtn$useContext)($cf1d656285911959$export$d665dd135a51b28a) !== null;
    return /*#__PURE__*/ (0, $7VHtn$jsx)((0, $7VHtn$Provider), {
        values: [
            [
                $cf1d656285911959$export$d665dd135a51b28a,
                {
                    size: size,
                    isQuiet: isQuiet,
                    density: density
                }
            ]
        ],
        children: /*#__PURE__*/ (0, $7VHtn$jsx)((0, $7VHtn$Disclosure), {
            ...props,
            ref: domRef,
            style: UNSAFE_style,
            className: (UNSAFE_className ?? '') + $cf1d656285911959$var$disclosure({
                isQuiet: isQuiet,
                isInGroup: isInGroup
            }, props.styles),
            children: props.children
        })
    });
});
const $cf1d656285911959$var$headingStyle = " Jy131 Gy131 Iy131 Hy131 _ub131 sd131 _vb131 Na131";
const $cf1d656285911959$var$buttonStyles = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kb131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-f131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "M") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    rules += ' xd131';
    rules += ' _xa131';
    rules += ' xX0cczbc131';
    rules += ' _xX0cczba131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    if (props.isDisabled) {
        rules += ' pp131';
        rules += ' plc131';
    } else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
        rules += ' plb131';
    }
    rules += ' sd131';
    rules += ' _ub131';
    rules += ' ea131';
    rules += ' S6FcKR131';
    rules += ' R6FcKR131';
    rules += ' Tu7Fubd131';
    rules += ' Qu7Fubd131';
    rules += ' U6FcKR131';
    rules += ' q6FcKR131';
    rules += ' MAyFGYc131';
    rules += ' Za131';
    if (props.isPressed) rules += ' gnSjEFb131';
    else if (props.isHovered) rules += ' gabY4te131';
    else if (props.isFocusVisible) rules += ' gabY4te131';
    else rules += ' g9131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' _kJ8iPYd131';
    rules += ' hXMFGYc131';
    rules += ' ma131';
    rules += ' la131';
    if (props.isQuiet) rules += ' ob131';
    else if (props.isFocusVisible) rules += ' ob131';
    else rules += ' od131';
    if (props.isQuiet) rules += ' nb131';
    else if (props.isFocusVisible) rules += ' nb131';
    else rules += ' nd131';
    if (props.isQuiet) rules += ' kb131';
    else if (props.isFocusVisible) rules += ' kb131';
    else rules += ' kd131';
    if (props.isQuiet) rules += ' jb131';
    else if (props.isFocusVisible) rules += ' jb131';
    else rules += ' jd131';
    rules += ' _Wd131';
    rules += ' __ca131';
    if (props.size === "XL") {
        if (props.density === "spacious") rules += ' -M_-Mj131';
        else if (props.density === "regular") rules += ' -M_-Mk131';
        else if (props.density === "compact") rules += ' -M_-Mh131';
    } else if (props.size === "L") {
        if (props.density === "spacious") rules += ' -M_-Mk131';
        else if (props.density === "regular") rules += ' -M_-Mh131';
        else if (props.density === "compact") rules += ' -M_-Mi131';
    } else if (props.size === "M") {
        if (props.density === "spacious") rules += ' -M_-Mh131';
        else if (props.density === "regular") rules += ' -M_-Mi131';
        else if (props.density === "compact") rules += ' -M_-Mg131';
    } else if (props.size === "S") {
        if (props.density === "spacious") rules += ' -M_-Mi131';
        else if (props.density === "regular") rules += ' -M_-Mg131';
        else if (props.density === "compact") rules += ' -M_-Me131';
    }
    rules += ' -_k_-_ka131';
    rules += ' -h_-ha131';
    return rules;
};
const $cf1d656285911959$var$chevronStyles = function anonymous(props) {
    let rules = " ";
    if (props.isExpanded) rules += ' _Sd131';
    else if (props.isRTL) rules += ' _Sc131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' -_8sjo0b-t5ZbAob131';
    rules += ' _va131';
    return rules;
};
const $cf1d656285911959$var$InternalDisclosureHeader = /*#__PURE__*/ (0, $7VHtn$createContext)(null);
function $cf1d656285911959$var$DisclosureHeaderWithForwardRef(props, ref) {
    let { UNSAFE_className: UNSAFE_className, UNSAFE_style: UNSAFE_style, children: children } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let { size: size, isQuiet: isQuiet, density: density } = (0, $7VHtn$useSlottedContext)($cf1d656285911959$export$d665dd135a51b28a);
    // Shift button size down by 2 for compact density, 1 for regular/spacious to ensure there is space between the top and bottom of the action button and container
    let newSize = size;
    const sizes = [
        'XS',
        'S',
        'M',
        'L',
        'XL'
    ];
    const currentIndex = sizes.indexOf(size ?? 'M');
    const shift = density === 'compact' ? 2 : 1;
    newSize = sizes[Math.max(0, currentIndex - shift)];
    return /*#__PURE__*/ (0, $7VHtn$jsx)((0, $7VHtn$Provider), {
        values: [
            [
                (0, $da878a05ab4a403e$export$2ff682723a46d413),
                {
                    size: newSize,
                    isQuiet: isQuiet
                }
            ],
            [
                $cf1d656285911959$var$InternalDisclosureHeader,
                {}
            ]
        ],
        children: /*#__PURE__*/ (0, $7VHtn$jsx)("div", {
            style: UNSAFE_style,
            className: (UNSAFE_className ?? '') + " sd131 eb131 Ub131 qb131",
            ref: domRef,
            children: children
        })
    });
}
const $cf1d656285911959$export$1e037ddd32f5b8ac = /*#__PURE__*/ (0, $7VHtn$forwardRef)($cf1d656285911959$var$DisclosureHeaderWithForwardRef);
const $cf1d656285911959$export$7843c6a5b3e340a2 = /*#__PURE__*/ (0, $7VHtn$forwardRef)(function DisclosureTitle(props, ref) {
    let { level: level = 3, UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '', ...otherProps } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    const domProps = (0, $7VHtn$filterDOMProps)(otherProps);
    let { direction: direction } = (0, $7VHtn$useLocale)();
    let { isExpanded: isExpanded } = (0, $7VHtn$useContext)((0, $7VHtn$DisclosureStateContext));
    let { size: size, density: density, isQuiet: isQuiet } = (0, $7VHtn$useSlottedContext)($cf1d656285911959$export$d665dd135a51b28a);
    let isRTL = direction === 'rtl';
    let buttonTrigger = /*#__PURE__*/ (0, $7VHtn$jsx)((0, $7VHtn$Heading), {
        ...domProps,
        level: level,
        ref: domRef,
        style: UNSAFE_style,
        className: (UNSAFE_className ?? '') + $cf1d656285911959$var$headingStyle,
        children: /*#__PURE__*/ (0, $7VHtn$jsxs)((0, $7VHtn$Button), {
            className: (renderProps)=>$cf1d656285911959$var$buttonStyles({
                    ...renderProps,
                    size: size,
                    density: density,
                    isQuiet: isQuiet
                }),
            slot: "trigger",
            children: [
                /*#__PURE__*/ (0, $7VHtn$jsx)((0, $1f4b04be3f24aae3$export$768dac55bb57081d), {
                    children: /*#__PURE__*/ (0, $7VHtn$jsx)((0, $4b69f02ec06b9226$export$2e2bcd8739ae039), {
                        size: size,
                        className: $cf1d656285911959$var$chevronStyles({
                            isExpanded: isExpanded,
                            isRTL: isRTL
                        }),
                        "aria-hidden": "true"
                    })
                }),
                props.children
            ]
        })
    });
    let ctx = (0, $7VHtn$useContext)($cf1d656285911959$var$InternalDisclosureHeader);
    if (ctx) return buttonTrigger;
    return /*#__PURE__*/ (0, $7VHtn$jsx)($cf1d656285911959$export$1e037ddd32f5b8ac, {
        children: buttonTrigger
    });
});
const $cf1d656285911959$var$panelStyles = " uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-d131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131 FfwmW1d131 _Nb131 Pb131 Ysgyfc131 Ydklsgce131 Xb131 Xdklsgcb131 _2b131 _2dklsgcb131";
const $cf1d656285911959$var$panelInner = function anonymous(props) {
    let rules = " ";
    rules += ' Tt131';
    rules += ' Qf131';
    if (props.size === "XL") rules += ' Sz5EUbe131';
    else if (props.size === "L") rules += ' Se131';
    else if (props.size === "M") rules += ' SLabwmd131';
    else if (props.size === "S") rules += ' St131';
    if (props.size === "XL") rules += ' Rz5EUbe131';
    else if (props.size === "L") rules += ' Re131';
    else if (props.size === "M") rules += ' RLabwmd131';
    else if (props.size === "S") rules += ' Rt131';
    return rules;
};
const $cf1d656285911959$export$feabaa331e1d464c = /*#__PURE__*/ (0, $7VHtn$forwardRef)(function DisclosurePanel(props, ref) {
    let { UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '', ...otherProps } = props;
    const domProps = (0, $7VHtn$filterDOMProps)(otherProps);
    let { size: size } = (0, $7VHtn$useSlottedContext)($cf1d656285911959$export$d665dd135a51b28a);
    let panelRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    return /*#__PURE__*/ (0, $7VHtn$jsx)((0, $7VHtn$DisclosurePanel), {
        ...domProps,
        ref: panelRef,
        style: UNSAFE_style,
        className: (UNSAFE_className ?? '') + $cf1d656285911959$var$panelStyles,
        children: /*#__PURE__*/ (0, $7VHtn$jsx)("div", {
            className: $cf1d656285911959$var$panelInner({
                size: size
            }),
            children: props.children
        })
    });
});


export {$cf1d656285911959$export$d665dd135a51b28a as DisclosureContext, $cf1d656285911959$export$74a362b31437ec83 as Disclosure, $cf1d656285911959$export$1e037ddd32f5b8ac as DisclosureHeader, $cf1d656285911959$export$7843c6a5b3e340a2 as DisclosureTitle, $cf1d656285911959$export$feabaa331e1d464c as DisclosurePanel};
