// @react-spectrum/s2/dist/private/Divider.cjs
require("./Divider.css");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $98jss$reactjsxruntime = require("react/jsx-runtime");
var $98jss$react = require("react");
var $98jss$reactariacomponentsSeparator = require("react-aria-components/Separator");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "DividerContext", () => $2d04acb918636de0$export$30fbfd376abdf190);
$parcel$export(module.exports, "divider", () => $2d04acb918636de0$export$e8c9778bf415dd08);
$parcel$export(module.exports, "Divider", () => $2d04acb918636de0$export$2e0a83ec2e27ecbb);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 




const $2d04acb918636de0$export$30fbfd376abdf190 = /*#__PURE__*/ (0, $98jss$react.createContext)(null);
const $2d04acb918636de0$export$e8c9778bf415dd08 = function anonymous(props, overrides) {
    let rules = " ";
    let alignSelf = false;
    let marginTop = false;
    let marginBottom = false;
    let marginInlineStart = false;
    let marginInlineEnd = false;
    let flexGrow = false;
    let flexShrink = false;
    let width = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_d") alignSelf = true;
        if (p[1] === "J") marginTop = true;
        if (p[1] === "G") marginBottom = true;
        if (p[1] === "I") marginInlineStart = true;
        if (p[1] === "H") marginInlineEnd = true;
        if (p[1] === "_u") flexGrow = true;
        if (p[1] === "_v") flexShrink = true;
        if (p[1] === "Z") width = true;
        rules += p[0];
    }
    if (props.staticColor === "white") rules += ' -Ovbhqd-gcwrnr131';
    else if (props.staticColor === "black") rules += ' -Ovbhqd-g4hn3s131';
    if (!alignSelf) rules += ' _df131';
    if (props.isStaticColor) {
        if (props.size === "L") rules += ' gi131';
        else rules += ' gg131';
    } else if (props.size === "L") rules += ' g0131';
    else rules += ' gF131';
    rules += ' gl_a131';
    rules += ' _je131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    if (!marginTop) rules += ' Jy131';
    if (!marginBottom) rules += ' Gy131';
    if (!marginInlineStart) rules += ' Iy131';
    if (!marginInlineEnd) rules += ' Hy131';
    if (!flexGrow) rules += ' _ua131';
    if (!flexShrink) rules += ' _va131';
    if (props.orientation === "horizontal") {
        if (props.size === "L") rules += ' FT4Nfn131';
        else if (props.size === "S") rules += ' FceNfn131';
        else rules += ' FLvNfn131';
    }
    if (props.orientation === "vertical") {
        if (props.size === "L") {
            if (!width) rules += ' ZT4Nfn131';
        } else if (props.size === "S") {
            if (!width) rules += ' ZceNfn131';
        } else if (!width) rules += ' ZLvNfn131';
    }
    return rules;
};
const $2d04acb918636de0$export$2e0a83ec2e27ecbb = /*#__PURE__*/ (0, $98jss$react.forwardRef)(function Divider(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $2d04acb918636de0$export$30fbfd376abdf190);
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    return /*#__PURE__*/ (0, $98jss$reactjsxruntime.jsx)((0, $98jss$reactariacomponentsSeparator.Separator), {
        ...props,
        ref: domRef,
        style: props.UNSAFE_style,
        className: (props.UNSAFE_className || '') + $2d04acb918636de0$export$e8c9778bf415dd08({
            size: props.size || 'M',
            orientation: props.orientation || 'horizontal',
            staticColor: props.staticColor,
            isStaticColor: !!props.staticColor
        }, props.styles)
    });
});
