// @react-spectrum/s2/dist/private/dnd-utils.cjs
require("./dnd-utils.css");
var $ae5b71e1d2f5c8ba$exports = require("./DragHandle.cjs");
var $c1AfU$reactjsxruntime = require("react/jsx-runtime");
var $c1AfU$reactariacomponentsButton = require("react-aria-components/Button");
var $c1AfU$reactariacomponentsuseDragAndDrop = require("react-aria-components/useDragAndDrop");
var $c1AfU$react = require("react");
var $c1AfU$reactariaVisuallyHidden = require("react-aria/VisuallyHidden");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "InsertionIndicator", () => $f1f08ef5de95f8eb$export$2c0bab5914a9d088);
$parcel$export(module.exports, "DragHandleButton", () => $f1f08ef5de95f8eb$export$4c99a519d851f90c);
/*
 * Copyright 2026 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





let $f1f08ef5de95f8eb$var$insertionIndicatorWrapper = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' Wr131';
    rules += ' _lr131';
    rules += ' _Ar131';
    rules += ' _zr131';
    rules += ' sd131';
    rules += ' eb131';
    if (props.isRoot) rules += ' Iy131';
    else rules += ' IyEbn5c131';
    if (props.isDropTarget) rules += ' poPoFpe131';
    else rules += ' pw131';
    if (props.isDropTarget) rules += ' pld131';
    if (props.isDropTarget) rules += ' -_3xYcJd-gX131';
    else rules += ' -_3xYcJd-g9131';
    if (props.isDropTarget) rules += ' -_3xYcJd-lga131';
    rules += ' _yb131';
    return rules;
};
function $f1f08ef5de95f8eb$export$2c0bab5914a9d088({ target: target }) {
    let { dropState: dropState } = (0, $c1AfU$react.useContext)((0, $c1AfU$reactariacomponentsuseDragAndDrop.DragAndDropContext)) ?? {};
    let level = 0;
    if (target.type === 'item' && dropState?.collection) level = dropState.collection.getItem(target.key)?.level ?? 0;
    return /*#__PURE__*/ (0, $c1AfU$reactjsxruntime.jsx)((0, $c1AfU$reactariacomponentsuseDragAndDrop.DropIndicator), {
        className: "",
        target: target,
        children: ({ isDropTarget: isDropTarget })=>/*#__PURE__*/ (0, $c1AfU$reactjsxruntime.jsx)("div", {
                className: $f1f08ef5de95f8eb$var$insertionIndicatorWrapper({
                    isDropTarget: isDropTarget,
                    isRoot: level === 0
                }),
                style: level > 0 ? {
                    '--drop-level': level
                } : undefined,
                children: /*#__PURE__*/ (0, $c1AfU$reactjsxruntime.jsxs)("svg", {
                    "aria-hidden": true,
                    width: "100%",
                    height: "12",
                    children: [
                        /*#__PURE__*/ (0, $c1AfU$reactjsxruntime.jsx)("line", {
                            x1: "0",
                            y1: "6",
                            x2: "100%",
                            y2: "6",
                            strokeWidth: "2",
                            stroke: "currentColor"
                        }),
                        /*#__PURE__*/ (0, $c1AfU$reactjsxruntime.jsx)("circle", {
                            cx: "6",
                            cy: "6",
                            r: "5",
                            strokeWidth: "2",
                            stroke: "currentColor",
                            style: {
                                fill: 'var(--indicator-circle-bg)'
                            }
                        }),
                        /*#__PURE__*/ (0, $c1AfU$reactjsxruntime.jsx)("circle", {
                            cx: "100%",
                            cy: "6",
                            r: "5",
                            strokeWidth: "2",
                            stroke: "currentColor",
                            style: {
                                fill: 'var(--indicator-circle-bg)'
                            },
                            transform: "translate(-6, 0)"
                        })
                    ]
                })
            })
    });
}
let $f1f08ef5de95f8eb$var$dragButton = function anonymous(props) {
    let rules = " ";
    rules += ' p40ub4c131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _Ca131';
    rules += ' Fo131';
    rules += ' Zi131';
    rules += ' Td131';
    rules += ' Qd131';
    rules += ' Sd131';
    rules += ' Rd131';
    rules += ' Jy131';
    rules += ' Gy131';
    rules += ' Iy131';
    rules += ' Hy131';
    rules += ' g9131';
    rules += ' _je131';
    rules += ' oa131';
    rules += ' na131';
    rules += ' ka131';
    rules += ' ja131';
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' -_8sjo0b-t5ZbAob131';
    return rules;
};
function $f1f08ef5de95f8eb$export$4c99a519d851f90c({ isFocusVisibleWithin: isFocusVisibleWithin }) {
    let { visuallyHiddenProps: visuallyHiddenProps } = (0, $c1AfU$reactariaVisuallyHidden.useVisuallyHidden)();
    return /*#__PURE__*/ (0, $c1AfU$reactjsxruntime.jsx)((0, $c1AfU$reactariacomponentsButton.Button), {
        slot: "drag",
        style: !isFocusVisibleWithin ? visuallyHiddenProps.style : {},
        className: $f1f08ef5de95f8eb$var$dragButton,
        children: /*#__PURE__*/ (0, $c1AfU$reactjsxruntime.jsx)((0, $ae5b71e1d2f5c8ba$exports.default), {
            size: "M"
        })
    });
}
