// @react-spectrum/s2/dist/private/dnd-utils.mjs
import "./dnd-utils.css";
import $8b63ac1a4fa1df22$export$2e2bcd8739ae039 from "./DragHandle.mjs";
import {jsx as $7m476$jsx, jsxs as $7m476$jsxs} from "react/jsx-runtime";
import {Button as $7m476$Button} from "react-aria-components/Button";
import {DragAndDropContext as $7m476$DragAndDropContext, DropIndicator as $7m476$DropIndicator} from "react-aria-components/useDragAndDrop";
import {useContext as $7m476$useContext} from "react";
import {useVisuallyHidden as $7m476$useVisuallyHidden} from "react-aria/VisuallyHidden";

/*
 * Copyright 2026 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





let $3c1ab1fd8f31a3e2$var$insertionIndicatorWrapper = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' Wr131';
    rules += ' _lr131';
    rules += ' _Ar131';
    rules += ' _zr131';
    rules += ' sd131';
    rules += ' eb131';
    if (props.isRoot) rules += ' Iy131';
    else rules += ' IyEbn5c131';
    if (props.isDropTarget) rules += ' poPoFpe131';
    else rules += ' pw131';
    if (props.isDropTarget) rules += ' pld131';
    if (props.isDropTarget) rules += ' -_3xYcJd-gX131';
    else rules += ' -_3xYcJd-g9131';
    if (props.isDropTarget) rules += ' -_3xYcJd-lga131';
    rules += ' _yb131';
    return rules;
};
function $3c1ab1fd8f31a3e2$export$2c0bab5914a9d088({ target: target }) {
    let { dropState: dropState } = (0, $7m476$useContext)((0, $7m476$DragAndDropContext)) ?? {};
    let level = 0;
    if (target.type === 'item' && dropState?.collection) level = dropState.collection.getItem(target.key)?.level ?? 0;
    return /*#__PURE__*/ (0, $7m476$jsx)((0, $7m476$DropIndicator), {
        className: "",
        target: target,
        children: ({ isDropTarget: isDropTarget })=>/*#__PURE__*/ (0, $7m476$jsx)("div", {
                className: $3c1ab1fd8f31a3e2$var$insertionIndicatorWrapper({
                    isDropTarget: isDropTarget,
                    isRoot: level === 0
                }),
                style: level > 0 ? {
                    '--drop-level': level
                } : undefined,
                children: /*#__PURE__*/ (0, $7m476$jsxs)("svg", {
                    "aria-hidden": true,
                    width: "100%",
                    height: "12",
                    children: [
                        /*#__PURE__*/ (0, $7m476$jsx)("line", {
                            x1: "0",
                            y1: "6",
                            x2: "100%",
                            y2: "6",
                            strokeWidth: "2",
                            stroke: "currentColor"
                        }),
                        /*#__PURE__*/ (0, $7m476$jsx)("circle", {
                            cx: "6",
                            cy: "6",
                            r: "5",
                            strokeWidth: "2",
                            stroke: "currentColor",
                            style: {
                                fill: 'var(--indicator-circle-bg)'
                            }
                        }),
                        /*#__PURE__*/ (0, $7m476$jsx)("circle", {
                            cx: "100%",
                            cy: "6",
                            r: "5",
                            strokeWidth: "2",
                            stroke: "currentColor",
                            style: {
                                fill: 'var(--indicator-circle-bg)'
                            },
                            transform: "translate(-6, 0)"
                        })
                    ]
                })
            })
    });
}
let $3c1ab1fd8f31a3e2$var$dragButton = function anonymous(props) {
    let rules = " ";
    rules += ' p40ub4c131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _Ca131';
    rules += ' Fo131';
    rules += ' Zi131';
    rules += ' Td131';
    rules += ' Qd131';
    rules += ' Sd131';
    rules += ' Rd131';
    rules += ' Jy131';
    rules += ' Gy131';
    rules += ' Iy131';
    rules += ' Hy131';
    rules += ' g9131';
    rules += ' _je131';
    rules += ' oa131';
    rules += ' na131';
    rules += ' ka131';
    rules += ' ja131';
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' -_8sjo0b-t5ZbAob131';
    return rules;
};
function $3c1ab1fd8f31a3e2$export$4c99a519d851f90c({ isFocusVisibleWithin: isFocusVisibleWithin }) {
    let { visuallyHiddenProps: visuallyHiddenProps } = (0, $7m476$useVisuallyHidden)();
    return /*#__PURE__*/ (0, $7m476$jsx)((0, $7m476$Button), {
        slot: "drag",
        style: !isFocusVisibleWithin ? visuallyHiddenProps.style : {},
        className: $3c1ab1fd8f31a3e2$var$dragButton,
        children: /*#__PURE__*/ (0, $7m476$jsx)((0, $8b63ac1a4fa1df22$export$2e2bcd8739ae039), {
            size: "M"
        })
    });
}


export {$3c1ab1fd8f31a3e2$export$2c0bab5914a9d088 as InsertionIndicator, $3c1ab1fd8f31a3e2$export$4c99a519d851f90c as DragHandleButton};
