// @react-spectrum/s2/dist/private/DragPreview.cjs
require("./DragPreview.css");
var $e991cbcdf82ced71$exports = require("./CenterBaseline.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $ee7b4c497f520c08$exports = require("./utils.cjs");
var $j7gUG$reactjsxruntime = require("react/jsx-runtime");
var $j7gUG$reactariacomponentsslots = require("react-aria-components/slots");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "label", () => $35d2da6f10cb8588$export$1237798dc640739a);
$parcel$export(module.exports, "description", () => $35d2da6f10cb8588$export$6d59db4903f20f7d);
$parcel$export(module.exports, "iconCenterWrapper", () => $35d2da6f10cb8588$export$31d1842a0fd736e6);
$parcel$export(module.exports, "icon", () => $35d2da6f10cb8588$export$1ca1ec8b29a4ce27);
$parcel$export(module.exports, "DragPreview", () => $35d2da6f10cb8588$export$905ab40ac2179daa);
/*
 * Copyright 2026 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





let $35d2da6f10cb8588$export$1237798dc640739a = function anonymous(props) {
    let rules = " ";
    rules += ' zv6x9r131';
    rules += ' yv6x9r131';
    rules += ' Bv6x9r131';
    rules += ' Av6x9r131';
    rules += ' _dc131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    rules += ' p40ub4c131';
    rules += ' _Nc131';
    rules += ' Pc131';
    rules += ' _Yb131';
    if (props.overflowMode === "wrap") rules += ' _7b131';
    else rules += ' _7c131';
    return rules;
};
let $35d2da6f10cb8588$export$6d59db4903f20f7d = function anonymous(props) {
    let rules = " ";
    rules += ' z9u7zPc131';
    rules += ' y9u7zPc131';
    rules += ' B9u7zPc131';
    rules += ' A9u7zPc131';
    rules += ' _dc131';
    rules += ' _Nc131';
    rules += ' Pc131';
    rules += ' _Yb131';
    if (props.overflowMode === "wrap") rules += ' _7b131';
    else rules += ' _7c131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-b131';
    rules += ' vx131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    if (props.isDisabled) rules += ' pp131';
    else {
        if (props.isPressed) rules += ' pt131';
        else if (props.isFocusVisible) rules += ' pt131';
        else if (props.isHovered) rules += ' pt131';
        else rules += ' pv131';
    }
    rules += ' pl40ub4c131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    return rules;
};
let $35d2da6f10cb8588$export$31d1842a0fd736e6 = " sd131 z2U5Crc131 y2U5Crc131 B2U5Crc131 AvoofG131 _dc131 eb131";
let $35d2da6f10cb8588$export$1ca1ec8b29a4ce27 = " sb131 ZuUmpre131 FuUmpre131 Ht131 Hsu131 -_8sjo0b-t5ZbAob131";
let $35d2da6f10cb8588$export$6b6fc962de371ec4 = " _Pc131";
let $35d2da6f10cb8588$export$12d0dcd183478a04 = " _Pa131 _9a131 WE131 _EE131 Zd4Eyud131 Fb131 ob131 nb131 kb131 jb131 _kb131 hb131 mb131 lb131 _jf131 iw131 gX131";
let $35d2da6f10cb8588$export$4a61298387975868 = function anonymous(props) {
    let rules = " ";
    rules += ' _oa131';
    rules += ' Sd131';
    rules += ' Rd131';
    rules += ' Tt131';
    rules += ' Qt131';
    rules += ' gX131';
    if (props.isPressed) rules += ' po131';
    else if (props.isFocusVisible) rules += ' po131';
    else if (props.isHovered) rules += ' po131';
    else rules += ' pt131';
    rules += ' _Pc131';
    rules += ' se131';
    rules += ' CQyEEqe131';
    rules += ' DBqynpb131';
    rules += ' Exumf9b131';
    rules += ' ea131';
    if (props.scale === "large") rules += ' MODqQub131';
    else rules += ' Mh131';
    rules += ' Zd4Eyud131';
    rules += ' ob131';
    rules += ' nb131';
    rules += ' kb131';
    rules += ' jb131';
    rules += ' _kb131';
    rules += ' hb131';
    rules += ' mb131';
    rules += ' lb131';
    rules += ' _jf131';
    rules += ' iw131';
    return rules;
};
let $35d2da6f10cb8588$export$f28dddd30893324d = " z0yNlr131 y0yNlr131 B0yNlr131 A0yNlr131 _dc131 St131 Rt131 Tj131 Qj131 oa131 na131 ka131 ja131 g3131 gle131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-b131 vx131 xd131 _xa131 xX0cczbc131 _xX0cczba131 _Fd131 _FnuYUweb131 px131 ple131 _yb131";
function $35d2da6f10cb8588$export$905ab40ac2179daa(props) {
    let { items: items, overflowMode: overflowMode } = props;
    let isDraggingMultiple = items.length > 1;
    let itemLabel = items[0]?.['text/plain'] ?? '';
    let scale = (0, $ee7b4c497f520c08$exports.useScale)();
    return /*#__PURE__*/ (0, $j7gUG$reactjsxruntime.jsxs)("div", {
        className: $35d2da6f10cb8588$export$6b6fc962de371ec4,
        children: [
            isDraggingMultiple && /*#__PURE__*/ (0, $j7gUG$reactjsxruntime.jsx)("div", {
                className: $35d2da6f10cb8588$export$12d0dcd183478a04
            }),
            /*#__PURE__*/ (0, $j7gUG$reactjsxruntime.jsx)("div", {
                className: $35d2da6f10cb8588$export$4a61298387975868({
                    scale: scale
                }),
                children: /*#__PURE__*/ (0, $j7gUG$reactjsxruntime.jsxs)((0, $j7gUG$reactariacomponentsslots.Provider), {
                    values: [
                        [
                            (0, $6367bc87eb7d24ad$exports.TextContext),
                            {
                                slots: {
                                    [(0, $j7gUG$reactariacomponentsslots.DEFAULT_SLOT)]: {
                                        styles: $35d2da6f10cb8588$export$1237798dc640739a({
                                            overflowMode: overflowMode
                                        })
                                    },
                                    label: {
                                        styles: $35d2da6f10cb8588$export$1237798dc640739a({
                                            overflowMode: overflowMode
                                        })
                                    },
                                    description: {
                                        styles: $35d2da6f10cb8588$export$6d59db4903f20f7d({
                                            overflowMode: overflowMode
                                        })
                                    }
                                }
                            }
                        ],
                        [
                            (0, $bde97c91243ed164$exports.IconContext),
                            {
                                slots: {
                                    icon: {
                                        render: (0, $e991cbcdf82ced71$exports.centerBaseline)({
                                            slot: 'icon',
                                            styles: $35d2da6f10cb8588$export$31d1842a0fd736e6
                                        }),
                                        styles: $35d2da6f10cb8588$export$1ca1ec8b29a4ce27
                                    }
                                }
                            }
                        ]
                    ],
                    children: [
                        props.children ?? /*#__PURE__*/ (0, $j7gUG$reactjsxruntime.jsx)((0, $6367bc87eb7d24ad$exports.Text), {
                            children: itemLabel
                        }),
                        isDraggingMultiple && /*#__PURE__*/ (0, $j7gUG$reactjsxruntime.jsx)("div", {
                            className: $35d2da6f10cb8588$export$f28dddd30893324d,
                            children: items.length
                        })
                    ]
                })
            })
        ]
    });
}
