// @react-spectrum/s2/dist/private/Field.mjs
import "./Field.css";
import $e17a4836f4b6cc28$export$2e2bcd8739ae039 from "../../icons/AlertTriangle.mjs";
import $8373af657584cab2$export$2e2bcd8739ae039 from "./Asterisk.mjs";
import {centerBaseline as $1f4b04be3f24aae3$export$9d7e2342a7e53afa, CenterBaseline as $1f4b04be3f24aae3$export$768dac55bb57081d, centerBaselineBefore as $1f4b04be3f24aae3$export$179e9489a09b14ac} from "./CenterBaseline.mjs";
import {ContextualHelpContext as $8e176595bc49ca3f$export$41b9afaaba473494} from "./ContextualHelp.mjs";
import {IconContext as $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed} from "../../icons/Icon.mjs";
import $7A6HT$intlStringsmjs from "./intlStrings.mjs";
import {mergeStyles as $feb886035e0d4633$export$e618dc39ac9ad607} from "../../icons/runtime.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {jsxs as $7A6HT$jsxs, jsx as $7A6HT$jsx, Fragment as $7A6HT$Fragment} from "react/jsx-runtime";
import {composeRenderProps as $7A6HT$composeRenderProps} from "react-aria-components/composeRenderProps";
import {FieldError as $7A6HT$FieldError} from "react-aria-components/FieldError";
import {forwardRef as $7A6HT$forwardRef, useContext as $7A6HT$useContext} from "react";
import {getEventTarget as $7A6HT$getEventTarget} from "react-aria/private/utils/shadowdom/DOMFunctions";
import {Group as $7A6HT$Group} from "react-aria-components/Group";
import {InputContext as $7A6HT$InputContext, Input as $7A6HT$Input} from "react-aria-components/Input";
import {Label as $7A6HT$Label} from "react-aria-components/Label";
import {Provider as $7A6HT$Provider} from "react-aria-components/slots";
import {Text as $7A6HT$Text} from "react-aria-components/Text";
import {useId as $7A6HT$useId} from "react-aria/useId";
import {useLocalizedStringFormatter as $7A6HT$useLocalizedStringFormatter} from "react-aria/useLocalizedStringFormatter";


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 



















const $9afd80978c252040$export$1acdcf5a973a8414 = /*#__PURE__*/ (0, $7A6HT$forwardRef)(function FieldLabel(props, ref) {
    let stringFormatter = (0, $7A6HT$useLocalizedStringFormatter)((0, ($parcel$interopDefault($7A6HT$intlStringsmjs))), '@react-spectrum/s2');
    let { isDisabled: isDisabled, isRequired: isRequired, size: size = 'M', necessityIndicator: necessityIndicator = 'icon', includeNecessityIndicatorInAccessibilityName: includeNecessityIndicatorInAccessibilityName = false, labelAlign: labelAlign, labelPosition: labelPosition, staticColor: staticColor, contextualHelp: contextualHelp, isQuiet: isQuiet, UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '', ...labelProps } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let contextualHelpId = (0, $7A6HT$useId)();
    let fallbackLabelPropsId = (0, $7A6HT$useId)();
    if (contextualHelp && !labelProps.id) labelProps.id = fallbackLabelPropsId;
    if (!props.children) return null;
    return /*#__PURE__*/ (0, $7A6HT$jsxs)("div", {
        className: function anonymous(props) {
            let rules = " ";
            rules += ' zv6x9r131';
            rules += ' yv6x9r131';
            rules += ' Bv6x9r131';
            rules += ' Av6x9r131';
            rules += ' sf131';
            if (props.labelAlign === "end") rules += ' _Wb131';
            else if (props.labelAlign === "start") rules += ' _Wd131';
            if (props.labelPosition === "top") rules += ' QgOnRt131';
            if (props.isQuiet) rules += ' _qd131';
            else if (props.labelPosition === "top") rules += ' _qb131';
            return rules;
        }({
            labelAlign: labelAlign,
            labelPosition: labelPosition,
            isQuiet: isQuiet
        }),
        children: [
            /*#__PURE__*/ (0, $7A6HT$jsxs)((0, $7A6HT$Label), {
                ...labelProps,
                ref: domRef,
                style: UNSAFE_style,
                className: UNSAFE_className + (0, $feb886035e0d4633$export$e618dc39ac9ad607)(function anonymous(props) {
                    let rules = " ";
                    if (props.size === "XL") {
                        rules += ' uk131';
                        rules += ' ucJ9TBTb131';
                        rules += ' ud3Euai131';
                        rules += ' uea131';
                        rules += ' ugb131';
                        rules += ' uhd131';
                        rules += ' uje131';
                        rules += ' u2NhKxcl131';
                        rules += ' uic131';
                    } else if (props.size === "L") {
                        rules += ' uk131';
                        rules += ' ucJ9TBTb131';
                        rules += ' ud3Euai131';
                        rules += ' uea131';
                        rules += ' ugb131';
                        rules += ' uhd131';
                        rules += ' uje131';
                        rules += ' u2NhKxcl131';
                        rules += ' uic131';
                    } else if (props.size === "S") {
                        rules += ' uk131';
                        rules += ' ucJ9TBTb131';
                        rules += ' ud3Euai131';
                        rules += ' uea131';
                        rules += ' ugb131';
                        rules += ' uhd131';
                        rules += ' uje131';
                        rules += ' u2NhKxcl131';
                        rules += ' uic131';
                    } else if (props.size === "XS") {
                        rules += ' uk131';
                        rules += ' ucJ9TBTb131';
                        rules += ' ud3Euai131';
                        rules += ' uea131';
                        rules += ' ugb131';
                        rules += ' uhd131';
                        rules += ' uje131';
                        rules += ' u2NhKxcl131';
                        rules += ' uic131';
                    } else {
                        rules += ' uk131';
                        rules += ' ucJ9TBTb131';
                        rules += ' ud3Euai131';
                        rules += ' uea131';
                        rules += ' ugb131';
                        rules += ' uhd131';
                        rules += ' uje131';
                        rules += ' u2NhKxcl131';
                        rules += ' uic131';
                    }
                    if (props.size === "XL") {
                        rules += ' -_6BNtrc-e131';
                        rules += ' vx131';
                    } else if (props.size === "L") {
                        rules += ' -_6BNtrc-d131';
                        rules += ' vx131';
                    } else if (props.size === "S") {
                        rules += ' -_6BNtrc-b131';
                        rules += ' vx131';
                    } else if (props.size === "XS") {
                        rules += ' -_6BNtrc-a131';
                        rules += ' vx131';
                    } else {
                        rules += ' -_6BNtrc-c131';
                        rules += ' vx131';
                    }
                    if (props.size === "XL") {
                        rules += ' xb131';
                        rules += ' _xa131';
                    } else if (props.size === "L") {
                        rules += ' xb131';
                        rules += ' _xa131';
                    } else if (props.size === "S") {
                        rules += ' xb131';
                        rules += ' _xa131';
                    } else if (props.size === "XS") {
                        rules += ' xb131';
                        rules += ' _xa131';
                    } else {
                        rules += ' xb131';
                        rules += ' _xa131';
                    }
                    if (props.size === "XL") {
                        rules += ' _Fd131';
                        rules += ' _FnuYUweb131';
                    } else if (props.size === "L") {
                        rules += ' _Fd131';
                        rules += ' _FnuYUweb131';
                    } else if (props.size === "S") {
                        rules += ' _Fd131';
                        rules += ' _FnuYUweb131';
                    } else if (props.size === "XS") {
                        rules += ' _Fd131';
                        rules += ' _FnuYUweb131';
                    } else {
                        rules += ' _Fd131';
                        rules += ' _FnuYUweb131';
                    }
                    if (props.isStaticColor) rules += ' pl131';
                    else if (props.isDisabled) rules += ' pp131';
                    else rules += ' pv131';
                    rules += ' plb131';
                    rules += ' ri131';
                    return rules;
                }({
                    labelPosition: labelPosition,
                    isDisabled: isDisabled,
                    size: size,
                    isStaticColor: !!staticColor
                }), props.styles),
                children: [
                    props.children,
                    (isRequired || necessityIndicator === 'label') && /*#__PURE__*/ (0, $7A6HT$jsxs)("span", {
                        className: " _7c131",
                        children: [
                            "\xa0",
                            necessityIndicator === 'icon' && /*#__PURE__*/ (0, $7A6HT$jsx)((0, $8373af657584cab2$export$2e2bcd8739ae039), {
                                size: size === 'S' ? 'M' : size,
                                className: " -_8sjo0b-t5ZbAob131",
                                "aria-label": includeNecessityIndicatorInAccessibilityName ? stringFormatter.format('label.(required)') : undefined,
                                "aria-hidden": !includeNecessityIndicatorInAccessibilityName
                            }),
                            necessityIndicator === 'label' && /* The necessity label is hidden to screen readers if the field is required because
               * aria-required is set on the field in that case. That will already be announced,
               * so no need to duplicate it here. If optional, we do want it to be announced here.
               */ /*#__PURE__*/ (0, $7A6HT$jsx)("span", {
                                "aria-hidden": !includeNecessityIndicatorInAccessibilityName ? isRequired : undefined,
                                children: isRequired ? stringFormatter.format('label.(required)') : stringFormatter.format('label.(optional)')
                            })
                        ]
                    })
                ]
            }),
            contextualHelp && /*#__PURE__*/ (0, $7A6HT$jsxs)("span", {
                className: " _7c131",
                children: [
                    "\xa0",
                    /*#__PURE__*/ (0, $7A6HT$jsx)((0, $1f4b04be3f24aae3$export$768dac55bb57081d), {
                        styles: " sh131 Fa131",
                        children: /*#__PURE__*/ (0, $7A6HT$jsx)((0, $8e176595bc49ca3f$export$41b9afaaba473494).Provider, {
                            value: {
                                id: contextualHelpId,
                                'aria-labelledby': labelProps?.id ? `${labelProps.id} ${contextualHelpId}` : undefined,
                                size: size === 'L' || size === 'XL' ? 'S' : 'XS'
                            },
                            children: contextualHelp
                        })
                    })
                ]
            })
        ]
    });
});
const $9afd80978c252040$var$fieldGroupStyles = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isDisabled) {
        rules += ' pp131';
        rules += ' plc131';
    } else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
        rules += ' plb131';
    }
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _oa131';
    rules += ' SMBFGYc131';
    rules += ' RfBFGYc131';
    if (props.size === "XL") rules += ' Nl131';
    else if (props.size === "L") rules += ' Nj131';
    else if (props.size === "S") rules += ' Nf131';
    else if (props.size === "XS") rules += ' Ne131';
    else rules += ' Nk131';
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-ob131';
    rules += ' oj131';
    rules += ' nj131';
    rules += ' kj131';
    rules += ' jj131';
    rules += ' FRuFGYc131';
    rules += ' zXcEWr131';
    rules += ' yXcEWr131';
    rules += ' BXcEWr131';
    rules += ' AXcEWr131';
    if (props.isQuiet) rules += ' _qd131';
    else rules += ' _qtb131';
    if (props.size === "XL") rules += ' -Anxdcc-Zt131';
    else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
    else if (props.size === "S") rules += ' -Anxdcc-Zr131';
    else rules += ' -Anxdcc-Z3fn9H131';
    rules += ' __v6TNLWb131';
    rules += ' _kc131';
    rules += ' hc131';
    rules += ' mCPFGYc131';
    rules += ' l5OFGYc131';
    rules += ' _jf131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.isDisabled) {
        rules += ' iq131';
        rules += ' ilc131';
    } else if (props.isFocusWithin) {
        if (props.isInvalid) rules += ' im131';
        else rules += ' in131';
        rules += ' ild131';
    } else if (props.isInvalid) {
        if (props.isPressed) rules += ' im131';
        else if (props.isFocusVisible) rules += ' im131';
        else if (props.isHovered) rules += ' im131';
        else rules += ' ip131';
        rules += ' ile131';
    } else {
        if (props.isPressed) rules += ' io131';
        else if (props.isFocusVisible) rules += ' io131';
        else if (props.isHovered) rules += ' io131';
        else rules += ' iq131';
        rules += ' ila131';
    }
    rules += ' gX131';
    rules += ' gl1kG7o131';
    if (props.isDisabled) rules += ' ri131';
    else rules += ' rD131';
    if (props.size === "XL") rules += ' -F_-Fz131';
    else if (props.size === "L") rules += ' -F_-Fv131';
    else if (props.size === "S") rules += ' -F_-Fp131';
    else if (props.size === "XS") rules += ' -F_-Fn131';
    else rules += ' -F_-Fx131';
    rules += ' -S_-Sv131';
    rules += ' -R_-Rv131';
    rules += ' -m_-mc131';
    rules += ' -l_-lc131';
    return rules;
};
const $9afd80978c252040$export$f0fe469072bc6950 = /*#__PURE__*/ (0, $7A6HT$forwardRef)(function FieldGroup(props, ref) {
    let { children: children, prefix: prefix, shouldTurnOffFocusRing: shouldTurnOffFocusRing, ...otherProps } = props;
    let ctx = (0, $7A6HT$useContext)((0, $7A6HT$InputContext));
    let prefixId = (0, $7A6HT$useId)();
    let newAriaLabelledby = ctx?.['aria-labelledby'];
    if (prefix) newAriaLabelledby = newAriaLabelledby ? `${newAriaLabelledby} ${prefixId}` : prefixId;
    return /*#__PURE__*/ (0, $7A6HT$jsxs)((0, $7A6HT$Group), {
        ref: ref,
        ...otherProps,
        onPointerDown: (e)=>{
            // Forward focus to input element when clicking on a non-interactive child (e.g. icon or padding)
            if (e.pointerType === 'mouse' && !(0, $7A6HT$getEventTarget)(e).closest('button,input,textarea,[role="button"]')) {
                e.preventDefault();
                e.currentTarget.querySelector('input, textarea')?.focus();
            }
        },
        onTouchEnd: (e)=>{
            let target = (0, $7A6HT$getEventTarget)(e);
            if (!target.isContentEditable && !target.closest('button,input,textarea,[role="button"]')) {
                e.preventDefault();
                e.currentTarget.querySelector('input, textarea')?.focus();
            }
        },
        style: props.UNSAFE_style,
        className: (renderProps)=>(props.UNSAFE_className || '') + ' ' + (0, $1f4b04be3f24aae3$export$179e9489a09b14ac) + (0, $feb886035e0d4633$export$e618dc39ac9ad607)($9afd80978c252040$var$fieldGroupStyles({
                ...renderProps,
                isFocusWithin: shouldTurnOffFocusRing ? false : renderProps.isFocusWithin,
                isFocusVisible: shouldTurnOffFocusRing ? false : renderProps.isFocusVisible,
                size: props.size || 'M'
            }), props.styles),
        children: [
            props.prefix ? /*#__PURE__*/ (0, $7A6HT$jsx)((0, $7A6HT$Provider), {
                values: [
                    [
                        (0, $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed),
                        {
                            styles: " ZuUmpre131 FuUmpre131 -_8sjo0b-t5ZbAob131"
                        }
                    ]
                ],
                children: /*#__PURE__*/ (0, $7A6HT$jsx)((0, $1f4b04be3f24aae3$export$768dac55bb57081d), {
                    id: prefixId,
                    styles: " pn131 _va131 Ht131 Hsu131",
                    children: props.prefix
                })
            }) : null,
            /*#__PURE__*/ (0, $7A6HT$jsx)((0, $7A6HT$InputContext).Consumer, {
                children: (ctx)=>/*#__PURE__*/ (0, $7A6HT$jsx)((0, $7A6HT$InputContext).Provider, {
                        value: {
                            ...ctx,
                            'aria-labelledby': newAriaLabelledby
                        },
                        children: children
                    })
            })
        ]
    });
});
const $9afd80978c252040$export$f5b8910cec6cf069 = /*#__PURE__*/ (0, $7A6HT$forwardRef)(function Input(props, ref) {
    let { UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, styles: styles, ...otherProps } = props;
    return /*#__PURE__*/ (0, $7A6HT$jsx)((0, $7A6HT$Input), {
        ...otherProps,
        ref: ref,
        style: UNSAFE_style,
        className: UNSAFE_className + (0, $feb886035e0d4633$export$e618dc39ac9ad607)(" Td131 Qd131 Sd131 Rd131 g9131 p40ub4c131 pYFIK2bn131 pYFIK2blc131 u40ub4c131 v40ub4c131 x40ub4c131 _xa131 _ub131 _vb131 Na131 Za131 _Le131 _je131 _Nc131 Pc131 _Yb131 _7c131", styles)
    });
});
const $9afd80978c252040$export$13b1fd5431595bf6 = function anonymous(props) {
    let rules = " ";
    rules += ' zzBWQ8d131';
    rules += ' yzBWQ8d131';
    rules += ' BzBWQ8d131';
    rules += ' AzBWQ8d131';
    rules += ' sd131';
    rules += ' ea131';
    rules += ' Uc131';
    rules += ' Usd131';
    rules += ' qc131';
    rules += ' qsd131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isDisabled) {
        rules += ' pp131';
        rules += ' plc131';
    } else if (props.isInvalid) {
        rules += ' pq131';
        rules += ' plQBRyrc131';
    } else rules += ' pv131';
    rules += ' -_8sjo0b-t5ZbAob131';
    rules += ' _qb131';
    rules += ' TgOnRt131';
    if (props.isDisabled) rules += ' ri131';
    else rules += ' rD131';
    return rules;
};
function $9afd80978c252040$export$a67c0bc59081311a(props1) {
    let domDescriptionRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(props1.descriptionRef || null);
    let domErrorRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(props1.errorRef || null);
    if (!props1.isInvalid && props1.description) return /*#__PURE__*/ (0, $7A6HT$jsx)((0, $7A6HT$Text), {
        slot: "description",
        ref: domDescriptionRef,
        className: (0, $feb886035e0d4633$export$e618dc39ac9ad607)($9afd80978c252040$export$13b1fd5431595bf6({
            size: props1.size || 'M',
            isDisabled: props1.isDisabled
        }), props1.styles),
        children: props1.description
    });
    if (!props1.isInvalid) return null;
    return /*#__PURE__*/ (0, $7A6HT$jsx)((0, $7A6HT$FieldError), {
        ...props1,
        ref: domErrorRef,
        className: (renderProps)=>(0, $feb886035e0d4633$export$e618dc39ac9ad607)($9afd80978c252040$export$13b1fd5431595bf6({
                ...renderProps,
                size: props1.size || 'M',
                isDisabled: props1.isDisabled
            }), props1.styles),
        children: (0, $7A6HT$composeRenderProps)(props1.children, (children, { validationErrors: validationErrors })=>/*#__PURE__*/ (0, $7A6HT$jsxs)((0, $7A6HT$Fragment), {
                children: [
                    props1.showErrorIcon && /*#__PURE__*/ (0, $7A6HT$jsx)((0, $1f4b04be3f24aae3$export$768dac55bb57081d), {
                        children: /*#__PURE__*/ (0, $7A6HT$jsx)((0, $e17a4836f4b6cc28$export$2e2bcd8739ae039), {
                            // @ts-ignore - ts doesn't know that AlertIcon is an icon and not a raw SVG
                            styles: function anonymous(props) {
                                let rules = " ";
                                if (props.size === "XL") rules += ' Zp131';
                                else if (props.size === "L") rules += ' Zn131';
                                else if (props.size === "M") rules += ' Zm131';
                                else if (props.size === "S") rules += ' Zu131';
                                else if (props.size === "XS") rules += ' Zk131';
                                if (props.size === "XL") rules += ' Fq131';
                                else if (props.size === "L") rules += ' Fo131';
                                else if (props.size === "M") rules += ' Fn131';
                                else if (props.size === "S") rules += ' Ft131';
                                else if (props.size === "XS") rules += ' Fl131';
                                return rules;
                            }({
                                size: props1.size || 'M'
                            })
                        })
                    }),
                    /*#__PURE__*/ (0, $7A6HT$jsx)("span", {
                        children: children || validationErrors.join(' ')
                    })
                ]
            }))
    });
}
function $9afd80978c252040$export$bda0d075e0e342c9(props) {
    return /*#__PURE__*/ (0, $7A6HT$jsx)((0, $7A6HT$Provider), {
        values: [
            [
                (0, $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed),
                {
                    render: (0, $1f4b04be3f24aae3$export$9d7e2342a7e53afa)({
                        slot: 'icon',
                        styles: " _Ja131 _va131 -_8sjo0b-tq131 -_8sjo0b-ltQBRyrc131"
                    }),
                    styles: " ZuUmpre131 FuUmpre131 It131 Isu131 HAxCW0c131 _va131"
                }
            ]
        ],
        children: !props.isDisabled && /*#__PURE__*/ (0, $7A6HT$jsx)((0, $e17a4836f4b6cc28$export$2e2bcd8739ae039), {})
    });
}


export {$9afd80978c252040$export$1acdcf5a973a8414 as FieldLabel, $9afd80978c252040$export$f0fe469072bc6950 as FieldGroup, $9afd80978c252040$export$f5b8910cec6cf069 as Input, $9afd80978c252040$export$13b1fd5431595bf6 as helpTextStyles, $9afd80978c252040$export$a67c0bc59081311a as HelpText, $9afd80978c252040$export$bda0d075e0e342c9 as FieldErrorIcon};
