// @react-spectrum/s2/dist/private/Field.cjs
require("./Field.css");
var $7e7cdbd2b8ae2467$exports = require("../../icons/AlertTriangle.cjs");
var $5d3efbee7071458b$exports = require("./Asterisk.cjs");
var $e991cbcdf82ced71$exports = require("./CenterBaseline.cjs");
var $bab829476f85a155$exports = require("./ContextualHelp.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $4526404114e78c80$exports = require("./intlStrings.cjs");
var $308b180f49d82d28$exports = require("../../icons/runtime.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $4hgIp$reactjsxruntime = require("react/jsx-runtime");
var $4hgIp$reactariacomponentscomposeRenderProps = require("react-aria-components/composeRenderProps");
var $4hgIp$reactariacomponentsFieldError = require("react-aria-components/FieldError");
var $4hgIp$react = require("react");
var $4hgIp$reactariaprivateutilsshadowdomDOMFunctions = require("react-aria/private/utils/shadowdom/DOMFunctions");
var $4hgIp$reactariacomponentsGroup = require("react-aria-components/Group");
var $4hgIp$reactariacomponentsInput = require("react-aria-components/Input");
var $4hgIp$reactariacomponentsLabel = require("react-aria-components/Label");
var $4hgIp$reactariacomponentsslots = require("react-aria-components/slots");
var $4hgIp$reactariacomponentsText = require("react-aria-components/Text");
var $4hgIp$reactariauseId = require("react-aria/useId");
var $4hgIp$reactariauseLocalizedStringFormatter = require("react-aria/useLocalizedStringFormatter");


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "FieldLabel", () => $a0b996a550bb0891$export$1acdcf5a973a8414);
$parcel$export(module.exports, "FieldGroup", () => $a0b996a550bb0891$export$f0fe469072bc6950);
$parcel$export(module.exports, "Input", () => $a0b996a550bb0891$export$f5b8910cec6cf069);
$parcel$export(module.exports, "helpTextStyles", () => $a0b996a550bb0891$export$13b1fd5431595bf6);
$parcel$export(module.exports, "HelpText", () => $a0b996a550bb0891$export$a67c0bc59081311a);
$parcel$export(module.exports, "FieldErrorIcon", () => $a0b996a550bb0891$export$bda0d075e0e342c9);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 



















const $a0b996a550bb0891$export$1acdcf5a973a8414 = /*#__PURE__*/ (0, $4hgIp$react.forwardRef)(function FieldLabel(props, ref) {
    let stringFormatter = (0, $4hgIp$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    let { isDisabled: isDisabled, isRequired: isRequired, size: size = 'M', necessityIndicator: necessityIndicator = 'icon', includeNecessityIndicatorInAccessibilityName: includeNecessityIndicatorInAccessibilityName = false, labelAlign: labelAlign, labelPosition: labelPosition, staticColor: staticColor, contextualHelp: contextualHelp, isQuiet: isQuiet, UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '', ...labelProps } = props;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let contextualHelpId = (0, $4hgIp$reactariauseId.useId)();
    let fallbackLabelPropsId = (0, $4hgIp$reactariauseId.useId)();
    if (contextualHelp && !labelProps.id) labelProps.id = fallbackLabelPropsId;
    if (!props.children) return null;
    return /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsxs)("div", {
        className: function anonymous(props) {
            let rules = " ";
            rules += ' zv6x9r131';
            rules += ' yv6x9r131';
            rules += ' Bv6x9r131';
            rules += ' Av6x9r131';
            rules += ' sf131';
            if (props.labelAlign === "end") rules += ' _Wb131';
            else if (props.labelAlign === "start") rules += ' _Wd131';
            if (props.labelPosition === "top") rules += ' QgOnRt131';
            if (props.isQuiet) rules += ' _qd131';
            else if (props.labelPosition === "top") rules += ' _qb131';
            return rules;
        }({
            labelAlign: labelAlign,
            labelPosition: labelPosition,
            isQuiet: isQuiet
        }),
        children: [
            /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsxs)((0, $4hgIp$reactariacomponentsLabel.Label), {
                ...labelProps,
                ref: domRef,
                style: UNSAFE_style,
                className: UNSAFE_className + (0, $308b180f49d82d28$exports.mergeStyles)(function anonymous(props) {
                    let rules = " ";
                    if (props.size === "XL") {
                        rules += ' uk131';
                        rules += ' ucJ9TBTb131';
                        rules += ' ud3Euai131';
                        rules += ' uea131';
                        rules += ' ugb131';
                        rules += ' uhd131';
                        rules += ' uje131';
                        rules += ' u2NhKxcl131';
                        rules += ' uic131';
                    } else if (props.size === "L") {
                        rules += ' uk131';
                        rules += ' ucJ9TBTb131';
                        rules += ' ud3Euai131';
                        rules += ' uea131';
                        rules += ' ugb131';
                        rules += ' uhd131';
                        rules += ' uje131';
                        rules += ' u2NhKxcl131';
                        rules += ' uic131';
                    } else if (props.size === "S") {
                        rules += ' uk131';
                        rules += ' ucJ9TBTb131';
                        rules += ' ud3Euai131';
                        rules += ' uea131';
                        rules += ' ugb131';
                        rules += ' uhd131';
                        rules += ' uje131';
                        rules += ' u2NhKxcl131';
                        rules += ' uic131';
                    } else if (props.size === "XS") {
                        rules += ' uk131';
                        rules += ' ucJ9TBTb131';
                        rules += ' ud3Euai131';
                        rules += ' uea131';
                        rules += ' ugb131';
                        rules += ' uhd131';
                        rules += ' uje131';
                        rules += ' u2NhKxcl131';
                        rules += ' uic131';
                    } else {
                        rules += ' uk131';
                        rules += ' ucJ9TBTb131';
                        rules += ' ud3Euai131';
                        rules += ' uea131';
                        rules += ' ugb131';
                        rules += ' uhd131';
                        rules += ' uje131';
                        rules += ' u2NhKxcl131';
                        rules += ' uic131';
                    }
                    if (props.size === "XL") {
                        rules += ' -_6BNtrc-e131';
                        rules += ' vx131';
                    } else if (props.size === "L") {
                        rules += ' -_6BNtrc-d131';
                        rules += ' vx131';
                    } else if (props.size === "S") {
                        rules += ' -_6BNtrc-b131';
                        rules += ' vx131';
                    } else if (props.size === "XS") {
                        rules += ' -_6BNtrc-a131';
                        rules += ' vx131';
                    } else {
                        rules += ' -_6BNtrc-c131';
                        rules += ' vx131';
                    }
                    if (props.size === "XL") {
                        rules += ' xb131';
                        rules += ' _xa131';
                    } else if (props.size === "L") {
                        rules += ' xb131';
                        rules += ' _xa131';
                    } else if (props.size === "S") {
                        rules += ' xb131';
                        rules += ' _xa131';
                    } else if (props.size === "XS") {
                        rules += ' xb131';
                        rules += ' _xa131';
                    } else {
                        rules += ' xb131';
                        rules += ' _xa131';
                    }
                    if (props.size === "XL") {
                        rules += ' _Fd131';
                        rules += ' _FnuYUweb131';
                    } else if (props.size === "L") {
                        rules += ' _Fd131';
                        rules += ' _FnuYUweb131';
                    } else if (props.size === "S") {
                        rules += ' _Fd131';
                        rules += ' _FnuYUweb131';
                    } else if (props.size === "XS") {
                        rules += ' _Fd131';
                        rules += ' _FnuYUweb131';
                    } else {
                        rules += ' _Fd131';
                        rules += ' _FnuYUweb131';
                    }
                    if (props.isStaticColor) rules += ' pl131';
                    else if (props.isDisabled) rules += ' pp131';
                    else rules += ' pv131';
                    rules += ' plb131';
                    rules += ' ri131';
                    return rules;
                }({
                    labelPosition: labelPosition,
                    isDisabled: isDisabled,
                    size: size,
                    isStaticColor: !!staticColor
                }), props.styles),
                children: [
                    props.children,
                    (isRequired || necessityIndicator === 'label') && /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsxs)("span", {
                        className: " _7c131",
                        children: [
                            "\xa0",
                            necessityIndicator === 'icon' && /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $5d3efbee7071458b$exports.default), {
                                size: size === 'S' ? 'M' : size,
                                className: " -_8sjo0b-t5ZbAob131",
                                "aria-label": includeNecessityIndicatorInAccessibilityName ? stringFormatter.format('label.(required)') : undefined,
                                "aria-hidden": !includeNecessityIndicatorInAccessibilityName
                            }),
                            necessityIndicator === 'label' && /* The necessity label is hidden to screen readers if the field is required because
               * aria-required is set on the field in that case. That will already be announced,
               * so no need to duplicate it here. If optional, we do want it to be announced here.
               */ /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)("span", {
                                "aria-hidden": !includeNecessityIndicatorInAccessibilityName ? isRequired : undefined,
                                children: isRequired ? stringFormatter.format('label.(required)') : stringFormatter.format('label.(optional)')
                            })
                        ]
                    })
                ]
            }),
            contextualHelp && /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsxs)("span", {
                className: " _7c131",
                children: [
                    "\xa0",
                    /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $e991cbcdf82ced71$exports.CenterBaseline), {
                        styles: " sh131 Fa131",
                        children: /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $bab829476f85a155$exports.ContextualHelpContext).Provider, {
                            value: {
                                id: contextualHelpId,
                                'aria-labelledby': labelProps?.id ? `${labelProps.id} ${contextualHelpId}` : undefined,
                                size: size === 'L' || size === 'XL' ? 'S' : 'XS'
                            },
                            children: contextualHelp
                        })
                    })
                ]
            })
        ]
    });
});
const $a0b996a550bb0891$var$fieldGroupStyles = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isDisabled) {
        rules += ' pp131';
        rules += ' plc131';
    } else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
        rules += ' plb131';
    }
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _oa131';
    rules += ' SMBFGYc131';
    rules += ' RfBFGYc131';
    if (props.size === "XL") rules += ' Nl131';
    else if (props.size === "L") rules += ' Nj131';
    else if (props.size === "S") rules += ' Nf131';
    else if (props.size === "XS") rules += ' Ne131';
    else rules += ' Nk131';
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-ob131';
    rules += ' oj131';
    rules += ' nj131';
    rules += ' kj131';
    rules += ' jj131';
    rules += ' FRuFGYc131';
    rules += ' zXcEWr131';
    rules += ' yXcEWr131';
    rules += ' BXcEWr131';
    rules += ' AXcEWr131';
    if (props.isQuiet) rules += ' _qd131';
    else rules += ' _qtb131';
    if (props.size === "XL") rules += ' -Anxdcc-Zt131';
    else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
    else if (props.size === "S") rules += ' -Anxdcc-Zr131';
    else rules += ' -Anxdcc-Z3fn9H131';
    rules += ' __v6TNLWb131';
    rules += ' _kc131';
    rules += ' hc131';
    rules += ' mCPFGYc131';
    rules += ' l5OFGYc131';
    rules += ' _jf131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.isDisabled) {
        rules += ' iq131';
        rules += ' ilc131';
    } else if (props.isFocusWithin) {
        if (props.isInvalid) rules += ' im131';
        else rules += ' in131';
        rules += ' ild131';
    } else if (props.isInvalid) {
        if (props.isPressed) rules += ' im131';
        else if (props.isFocusVisible) rules += ' im131';
        else if (props.isHovered) rules += ' im131';
        else rules += ' ip131';
        rules += ' ile131';
    } else {
        if (props.isPressed) rules += ' io131';
        else if (props.isFocusVisible) rules += ' io131';
        else if (props.isHovered) rules += ' io131';
        else rules += ' iq131';
        rules += ' ila131';
    }
    rules += ' gX131';
    rules += ' gl1kG7o131';
    if (props.isDisabled) rules += ' ri131';
    else rules += ' rD131';
    if (props.size === "XL") rules += ' -F_-Fz131';
    else if (props.size === "L") rules += ' -F_-Fv131';
    else if (props.size === "S") rules += ' -F_-Fp131';
    else if (props.size === "XS") rules += ' -F_-Fn131';
    else rules += ' -F_-Fx131';
    rules += ' -S_-Sv131';
    rules += ' -R_-Rv131';
    rules += ' -m_-mc131';
    rules += ' -l_-lc131';
    return rules;
};
const $a0b996a550bb0891$export$f0fe469072bc6950 = /*#__PURE__*/ (0, $4hgIp$react.forwardRef)(function FieldGroup(props, ref) {
    let { children: children, prefix: prefix, shouldTurnOffFocusRing: shouldTurnOffFocusRing, ...otherProps } = props;
    let ctx = (0, $4hgIp$react.useContext)((0, $4hgIp$reactariacomponentsInput.InputContext));
    let prefixId = (0, $4hgIp$reactariauseId.useId)();
    let newAriaLabelledby = ctx?.['aria-labelledby'];
    if (prefix) newAriaLabelledby = newAriaLabelledby ? `${newAriaLabelledby} ${prefixId}` : prefixId;
    return /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsxs)((0, $4hgIp$reactariacomponentsGroup.Group), {
        ref: ref,
        ...otherProps,
        onPointerDown: (e)=>{
            // Forward focus to input element when clicking on a non-interactive child (e.g. icon or padding)
            if (e.pointerType === 'mouse' && !(0, $4hgIp$reactariaprivateutilsshadowdomDOMFunctions.getEventTarget)(e).closest('button,input,textarea,[role="button"]')) {
                e.preventDefault();
                e.currentTarget.querySelector('input, textarea')?.focus();
            }
        },
        onTouchEnd: (e)=>{
            let target = (0, $4hgIp$reactariaprivateutilsshadowdomDOMFunctions.getEventTarget)(e);
            if (!target.isContentEditable && !target.closest('button,input,textarea,[role="button"]')) {
                e.preventDefault();
                e.currentTarget.querySelector('input, textarea')?.focus();
            }
        },
        style: props.UNSAFE_style,
        className: (renderProps)=>(props.UNSAFE_className || '') + ' ' + (0, $e991cbcdf82ced71$exports.centerBaselineBefore) + (0, $308b180f49d82d28$exports.mergeStyles)($a0b996a550bb0891$var$fieldGroupStyles({
                ...renderProps,
                isFocusWithin: shouldTurnOffFocusRing ? false : renderProps.isFocusWithin,
                isFocusVisible: shouldTurnOffFocusRing ? false : renderProps.isFocusVisible,
                size: props.size || 'M'
            }), props.styles),
        children: [
            props.prefix ? /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $4hgIp$reactariacomponentsslots.Provider), {
                values: [
                    [
                        (0, $bde97c91243ed164$exports.IconContext),
                        {
                            styles: " ZuUmpre131 FuUmpre131 -_8sjo0b-t5ZbAob131"
                        }
                    ]
                ],
                children: /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $e991cbcdf82ced71$exports.CenterBaseline), {
                    id: prefixId,
                    styles: " pn131 _va131 Ht131 Hsu131",
                    children: props.prefix
                })
            }) : null,
            /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $4hgIp$reactariacomponentsInput.InputContext).Consumer, {
                children: (ctx)=>/*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $4hgIp$reactariacomponentsInput.InputContext).Provider, {
                        value: {
                            ...ctx,
                            'aria-labelledby': newAriaLabelledby
                        },
                        children: children
                    })
            })
        ]
    });
});
const $a0b996a550bb0891$export$f5b8910cec6cf069 = /*#__PURE__*/ (0, $4hgIp$react.forwardRef)(function Input(props, ref) {
    let { UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, styles: styles, ...otherProps } = props;
    return /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $4hgIp$reactariacomponentsInput.Input), {
        ...otherProps,
        ref: ref,
        style: UNSAFE_style,
        className: UNSAFE_className + (0, $308b180f49d82d28$exports.mergeStyles)(" Td131 Qd131 Sd131 Rd131 g9131 p40ub4c131 pYFIK2bn131 pYFIK2blc131 u40ub4c131 v40ub4c131 x40ub4c131 _xa131 _ub131 _vb131 Na131 Za131 _Le131 _je131 _Nc131 Pc131 _Yb131 _7c131", styles)
    });
});
const $a0b996a550bb0891$export$13b1fd5431595bf6 = function anonymous(props) {
    let rules = " ";
    rules += ' zzBWQ8d131';
    rules += ' yzBWQ8d131';
    rules += ' BzBWQ8d131';
    rules += ' AzBWQ8d131';
    rules += ' sd131';
    rules += ' ea131';
    rules += ' Uc131';
    rules += ' Usd131';
    rules += ' qc131';
    rules += ' qsd131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isDisabled) {
        rules += ' pp131';
        rules += ' plc131';
    } else if (props.isInvalid) {
        rules += ' pq131';
        rules += ' plQBRyrc131';
    } else rules += ' pv131';
    rules += ' -_8sjo0b-t5ZbAob131';
    rules += ' _qb131';
    rules += ' TgOnRt131';
    if (props.isDisabled) rules += ' ri131';
    else rules += ' rD131';
    return rules;
};
function $a0b996a550bb0891$export$a67c0bc59081311a(props1) {
    let domDescriptionRef = (0, $150382364eecc4bc$exports.useDOMRef)(props1.descriptionRef || null);
    let domErrorRef = (0, $150382364eecc4bc$exports.useDOMRef)(props1.errorRef || null);
    if (!props1.isInvalid && props1.description) return /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $4hgIp$reactariacomponentsText.Text), {
        slot: "description",
        ref: domDescriptionRef,
        className: (0, $308b180f49d82d28$exports.mergeStyles)($a0b996a550bb0891$export$13b1fd5431595bf6({
            size: props1.size || 'M',
            isDisabled: props1.isDisabled
        }), props1.styles),
        children: props1.description
    });
    if (!props1.isInvalid) return null;
    return /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $4hgIp$reactariacomponentsFieldError.FieldError), {
        ...props1,
        ref: domErrorRef,
        className: (renderProps)=>(0, $308b180f49d82d28$exports.mergeStyles)($a0b996a550bb0891$export$13b1fd5431595bf6({
                ...renderProps,
                size: props1.size || 'M',
                isDisabled: props1.isDisabled
            }), props1.styles),
        children: (0, $4hgIp$reactariacomponentscomposeRenderProps.composeRenderProps)(props1.children, (children, { validationErrors: validationErrors })=>/*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsxs)((0, $4hgIp$reactjsxruntime.Fragment), {
                children: [
                    props1.showErrorIcon && /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $e991cbcdf82ced71$exports.CenterBaseline), {
                        children: /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $7e7cdbd2b8ae2467$exports.default), {
                            // @ts-ignore - ts doesn't know that AlertIcon is an icon and not a raw SVG
                            styles: function anonymous(props) {
                                let rules = " ";
                                if (props.size === "XL") rules += ' Zp131';
                                else if (props.size === "L") rules += ' Zn131';
                                else if (props.size === "M") rules += ' Zm131';
                                else if (props.size === "S") rules += ' Zu131';
                                else if (props.size === "XS") rules += ' Zk131';
                                if (props.size === "XL") rules += ' Fq131';
                                else if (props.size === "L") rules += ' Fo131';
                                else if (props.size === "M") rules += ' Fn131';
                                else if (props.size === "S") rules += ' Ft131';
                                else if (props.size === "XS") rules += ' Fl131';
                                return rules;
                            }({
                                size: props1.size || 'M'
                            })
                        })
                    }),
                    /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)("span", {
                        children: children || validationErrors.join(' ')
                    })
                ]
            }))
    });
}
function $a0b996a550bb0891$export$bda0d075e0e342c9(props) {
    return /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $4hgIp$reactariacomponentsslots.Provider), {
        values: [
            [
                (0, $bde97c91243ed164$exports.IconContext),
                {
                    render: (0, $e991cbcdf82ced71$exports.centerBaseline)({
                        slot: 'icon',
                        styles: " _Ja131 _va131 -_8sjo0b-tq131 -_8sjo0b-ltQBRyrc131"
                    }),
                    styles: " ZuUmpre131 FuUmpre131 It131 Isu131 HAxCW0c131 _va131"
                }
            ]
        ],
        children: !props.isDisabled && /*#__PURE__*/ (0, $4hgIp$reactjsxruntime.jsx)((0, $7e7cdbd2b8ae2467$exports.default), {})
    });
}
