// @react-spectrum/s2/dist/private/FullscreenDialog.cjs
require("./FullscreenDialog.css");
var $e1016b8947dbadb8$exports = require("./ButtonGroup.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $8e678305a8c10028$exports = require("./Modal.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $Yi1VH$reactjsxruntime = require("react/jsx-runtime");
var $Yi1VH$reactariacomponentscomposeRenderProps = require("react-aria-components/composeRenderProps");
var $Yi1VH$react = require("react");
var $Yi1VH$reactariacomponentsDialog = require("react-aria-components/Dialog");
var $Yi1VH$reactariacomponentsslots = require("react-aria-components/slots");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "FullscreenDialog", () => $59876d274d9473a9$export$7082b7e805d2626f);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 








const $59876d274d9473a9$var$heading = " zPCJngb131 yPCJngb131 BPCJngb131 APCJngb131 _ub131 Jy131 Gy131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-g131 vx131 xd131 _xa131 xX0cczbc131 _xX0cczba131 xfd131 _xfa131 xfX0cczbc131 _xfX0cczba131 _Fd131 _FnuYUweb131 po131";
const $59876d274d9473a9$var$header = " z6Akec131 y6Akec131 B6Akec131 A6Akec131 InM131 HnM131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-d131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131";
const $59876d274d9473a9$var$content = " zk52g2d131 yk52g2d131 Bk52g2d131 Ak52g2d131 _ub131 Pa131 PJVVdVe131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-d131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131";
const $59876d274d9473a9$var$buttonGroup = " z4AkfTc131 y4AkfTc131 B4AkfTc131 A4AkfTc131 IM131 La131";
const $59876d274d9473a9$export$6af31f836f9aa6f2 = " se131 CQpyMed131 Cns3y4td131 DM8Mfn131 Dn4w1sLc131 EfcKqZ131 EnFfvwed131 Th131 Tnk131 Qh131 Qnk131 Sh131 Snk131 Rh131 Rnk131 ql131 qnj131 K40ub4c131 Fb131 _oa131 _Le131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 oi131 ni131 ki131 ji131 _Na131 Pa131";
const $59876d274d9473a9$export$7082b7e805d2626f = /*#__PURE__*/ (0, $Yi1VH$react.forwardRef)(function FullscreenDialog(props, ref) {
    let { variant: variant = 'fullscreen', isKeyboardDismissDisabled: isKeyboardDismissDisabled } = props;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    return /*#__PURE__*/ (0, $Yi1VH$reactjsxruntime.jsx)((0, $8e678305a8c10028$exports.Modal), {
        size: variant,
        isKeyboardDismissDisabled: isKeyboardDismissDisabled,
        children: /*#__PURE__*/ (0, $Yi1VH$reactjsxruntime.jsx)((0, $Yi1VH$reactariacomponentsDialog.Dialog), {
            ...props,
            ref: domRef,
            style: props.UNSAFE_style,
            className: (props.UNSAFE_className || '') + $59876d274d9473a9$export$6af31f836f9aa6f2,
            children: (0, $Yi1VH$reactariacomponentscomposeRenderProps.composeRenderProps)(props.children, (children)=>// Reset OverlayTriggerStateContext so the buttons inside the dialog don't retain their hover state.
                /*#__PURE__*/ (0, $Yi1VH$reactjsxruntime.jsx)((0, $Yi1VH$reactariacomponentsDialog.OverlayTriggerStateContext).Provider, {
                    value: null,
                    children: /*#__PURE__*/ (0, $Yi1VH$reactjsxruntime.jsx)((0, $Yi1VH$reactariacomponentsslots.Provider), {
                        values: [
                            [
                                (0, $6367bc87eb7d24ad$exports.HeadingContext),
                                {
                                    styles: $59876d274d9473a9$var$heading
                                }
                            ],
                            [
                                (0, $6367bc87eb7d24ad$exports.HeaderContext),
                                {
                                    styles: $59876d274d9473a9$var$header
                                }
                            ],
                            [
                                (0, $6367bc87eb7d24ad$exports.ContentContext),
                                {
                                    styles: $59876d274d9473a9$var$content
                                }
                            ],
                            [
                                (0, $e1016b8947dbadb8$exports.ButtonGroupContext),
                                {
                                    styles: $59876d274d9473a9$var$buttonGroup
                                }
                            ]
                        ],
                        children: children
                    })
                }))
        })
    });
});
