// @react-spectrum/s2/dist/private/FullscreenDialog.mjs
import "./FullscreenDialog.css";
import {ButtonGroupContext as $472eaa9a9b344b9a$export$4c574c91bb0be0e2} from "./ButtonGroup.mjs";
import {ContentContext as $8e847109a6ab556d$export$1cbdd774077931b4, HeaderContext as $8e847109a6ab556d$export$e0e4026c12a8bdbb, HeadingContext as $8e847109a6ab556d$export$d688439359537581} from "./Content.mjs";
import {Modal as $c46d159bd615e407$export$2b77a92f1a5ad772} from "./Modal.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {jsx as $900FG$jsx} from "react/jsx-runtime";
import {composeRenderProps as $900FG$composeRenderProps} from "react-aria-components/composeRenderProps";
import {forwardRef as $900FG$forwardRef} from "react";
import {Dialog as $900FG$Dialog, OverlayTriggerStateContext as $900FG$OverlayTriggerStateContext} from "react-aria-components/Dialog";
import {Provider as $900FG$Provider} from "react-aria-components/slots";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 








const $8e7a38ff5b9798e1$var$heading = " zPCJngb131 yPCJngb131 BPCJngb131 APCJngb131 _ub131 Jy131 Gy131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-g131 vx131 xd131 _xa131 xX0cczbc131 _xX0cczba131 xfd131 _xfa131 xfX0cczbc131 _xfX0cczba131 _Fd131 _FnuYUweb131 po131";
const $8e7a38ff5b9798e1$var$header = " z6Akec131 y6Akec131 B6Akec131 A6Akec131 InM131 HnM131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-d131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131";
const $8e7a38ff5b9798e1$var$content = " zk52g2d131 yk52g2d131 Bk52g2d131 Ak52g2d131 _ub131 Pa131 PJVVdVe131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 -_6BNtrc-d131 vx131 xb131 _xa131 _Fb131 _FnuYUwec131 pt131";
const $8e7a38ff5b9798e1$var$buttonGroup = " z4AkfTc131 y4AkfTc131 B4AkfTc131 A4AkfTc131 IM131 La131";
const $8e7a38ff5b9798e1$export$6af31f836f9aa6f2 = " se131 CQpyMed131 Cns3y4td131 DM8Mfn131 Dn4w1sLc131 EfcKqZ131 EnFfvwed131 Th131 Tnk131 Qh131 Qnk131 Sh131 Snk131 Rh131 Rnk131 ql131 qnj131 K40ub4c131 Fb131 _oa131 _Le131 uk131 ucJ9TBTb131 ud3Euai131 uea131 ugb131 uhd131 uje131 u2NhKxcl131 uic131 oi131 ni131 ki131 ji131 _Na131 Pa131";
const $8e7a38ff5b9798e1$export$7082b7e805d2626f = /*#__PURE__*/ (0, $900FG$forwardRef)(function FullscreenDialog(props, ref) {
    let { variant: variant = 'fullscreen', isKeyboardDismissDisabled: isKeyboardDismissDisabled } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    return /*#__PURE__*/ (0, $900FG$jsx)((0, $c46d159bd615e407$export$2b77a92f1a5ad772), {
        size: variant,
        isKeyboardDismissDisabled: isKeyboardDismissDisabled,
        children: /*#__PURE__*/ (0, $900FG$jsx)((0, $900FG$Dialog), {
            ...props,
            ref: domRef,
            style: props.UNSAFE_style,
            className: (props.UNSAFE_className || '') + $8e7a38ff5b9798e1$export$6af31f836f9aa6f2,
            children: (0, $900FG$composeRenderProps)(props.children, (children)=>// Reset OverlayTriggerStateContext so the buttons inside the dialog don't retain their hover state.
                /*#__PURE__*/ (0, $900FG$jsx)((0, $900FG$OverlayTriggerStateContext).Provider, {
                    value: null,
                    children: /*#__PURE__*/ (0, $900FG$jsx)((0, $900FG$Provider), {
                        values: [
                            [
                                (0, $8e847109a6ab556d$export$d688439359537581),
                                {
                                    styles: $8e7a38ff5b9798e1$var$heading
                                }
                            ],
                            [
                                (0, $8e847109a6ab556d$export$e0e4026c12a8bdbb),
                                {
                                    styles: $8e7a38ff5b9798e1$var$header
                                }
                            ],
                            [
                                (0, $8e847109a6ab556d$export$1cbdd774077931b4),
                                {
                                    styles: $8e7a38ff5b9798e1$var$content
                                }
                            ],
                            [
                                (0, $472eaa9a9b344b9a$export$4c574c91bb0be0e2),
                                {
                                    styles: $8e7a38ff5b9798e1$var$buttonGroup
                                }
                            ]
                        ],
                        children: children
                    })
                }))
        })
    });
});


export {$8e7a38ff5b9798e1$export$6af31f836f9aa6f2 as dialogInner, $8e7a38ff5b9798e1$export$7082b7e805d2626f as FullscreenDialog};
