// @react-spectrum/s2/dist/private/IllustratedMessage.cjs
require("./IllustratedMessage.css");
var $e1016b8947dbadb8$exports = require("./ButtonGroup.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $jcfA8$reactjsxruntime = require("react/jsx-runtime");
var $jcfA8$reactariacomponentsslots = require("react-aria-components/slots");
var $jcfA8$react = require("react");
var $jcfA8$reactariafilterDOMProps = require("react-aria/filterDOMProps");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "IllustratedMessageContext", () => $55f1b8940d3e02db$export$d661ef89ad42d682);
$parcel$export(module.exports, "IllustratedMessage", () => $55f1b8940d3e02db$export$406dbc84c317ece0);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 








const $55f1b8940d3e02db$var$illustratedMessage = function anonymous(props, overrides) {
    let rules = " ";
    let maxWidth = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L|F|M|K)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "L") maxWidth = true;
        rules += p[0];
    }
    rules += ' se131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.size === "XL") rules += ' pt131';
    else if (props.size === "L") rules += ' pt131';
    else if (props.size === "S") rules += ' pt131';
    else if (props.size === "XS") rules += ' pt131';
    else rules += ' pt131';
    if (props.orientation === "horizontal") {
        if (!maxWidth) rules += ' Lj131';
    } else if (props.orientation === "vertical") {
        if (!maxWidth) rules += ' Li131';
    }
    if (props.orientation === "horizontal") rules += ' CLJajOe131';
    else if (props.orientation === "vertical") rules += ' CPmblS131';
    if (props.orientation === "horizontal") rules += ' E6zxP8c131';
    else if (props.orientation === "vertical") {
        if (props.size === "L") rules += ' EdL799131';
        else rules += ' E02fTrb131';
    }
    if (props.orientation === "horizontal") rules += ' DrSmbR131';
    if (props.orientation === "horizontal") rules += ' _Dc131';
    else if (props.orientation === "vertical") rules += ' _Da131';
    if (props.orientation === "vertical") rules += ' _Wa131';
    return rules;
};
const $55f1b8940d3e02db$var$illustration = function anonymous(props) {
    let rules = " ";
    rules += ' zTZ8xtd131';
    rules += ' yTZ8xtd131';
    rules += ' BTZ8xtd131';
    rules += ' ATZ8xtd131';
    rules += ' _dc131';
    if (props.isDropTarget) rules += ' -_8sjo0b-pu131';
    else rules += ' -_8sjo0b-pt131';
    return rules;
};
const $55f1b8940d3e02db$var$heading = function anonymous(props) {
    let rules = " ";
    rules += ' zPCJngb131';
    rules += ' yPCJngb131';
    rules += ' BPCJngb131';
    rules += ' APCJngb131';
    if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "M") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "L") {
        rules += ' -_6BNtrc-g131';
        rules += ' vx131';
    } else if (props.size === "M") {
        rules += ' -_6BNtrc-f131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    }
    if (props.size === "L") {
        rules += ' xd131';
        rules += ' _xa131';
        rules += ' xX0cczbc131';
        rules += ' _xX0cczba131';
        rules += ' xfd131';
        rules += ' _xfa131';
        rules += ' xfX0cczbc131';
        rules += ' _xfX0cczba131';
    } else if (props.size === "M") {
        rules += ' xd131';
        rules += ' _xa131';
        rules += ' xX0cczbc131';
        rules += ' _xX0cczba131';
        rules += ' xfd131';
        rules += ' _xfa131';
        rules += ' xfX0cczbc131';
        rules += ' _xfX0cczba131';
    } else if (props.size === "S") {
        rules += ' xd131';
        rules += ' _xa131';
        rules += ' xX0cczbc131';
        rules += ' _xX0cczba131';
        rules += ' xfd131';
        rules += ' _xfa131';
        rules += ' xfX0cczbc131';
        rules += ' _xfX0cczba131';
    }
    if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "M") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.size === "L") rules += ' po131';
    else if (props.size === "M") rules += ' po131';
    else if (props.size === "S") rules += ' po131';
    rules += ' _dd131';
    rules += ' Jy131';
    rules += ' Gy131';
    rules += ' Iy131';
    rules += ' Hy131';
    return rules;
};
const $55f1b8940d3e02db$var$content = function anonymous(props) {
    let rules = " ";
    if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "M") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "L") {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    } else if (props.size === "M") {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    }
    if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "M") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "L") {
        rules += ' _Fb131';
        rules += ' _FnuYUwec131';
    } else if (props.size === "M") {
        rules += ' _Fb131';
        rules += ' _FnuYUwec131';
    } else if (props.size === "S") {
        rules += ' _Fb131';
        rules += ' _FnuYUwec131';
    }
    if (props.size === "L") rules += ' pt131';
    else if (props.size === "M") rules += ' pt131';
    else if (props.size === "S") rules += ' pt131';
    rules += ' zk52g2d131';
    rules += ' yk52g2d131';
    rules += ' Bk52g2d131';
    rules += ' Ak52g2d131';
    rules += ' _de131';
    return rules;
};
const $55f1b8940d3e02db$var$buttonGroup = " zmlKpq131 ymlKpq131 BmlKpq131 AmlKpq131 JC131";
const $55f1b8940d3e02db$export$d661ef89ad42d682 = /*#__PURE__*/ (0, $jcfA8$react.createContext)(null);
const $55f1b8940d3e02db$export$406dbc84c317ece0 = /*#__PURE__*/ (0, $jcfA8$react.forwardRef)(function IllustratedMessage(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $55f1b8940d3e02db$export$d661ef89ad42d682);
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let { children: children, orientation: orientation = 'horizontal', size: size = 'M', UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, isInDropZone: isInDropZone = false, isDropTarget: isDropTarget = false, ...otherProps } = props;
    return /*#__PURE__*/ (0, $jcfA8$reactjsxruntime.jsx)("div", {
        ...(0, $jcfA8$reactariafilterDOMProps.filterDOMProps)(otherProps),
        style: UNSAFE_style,
        className: UNSAFE_className + $55f1b8940d3e02db$var$illustratedMessage({
            size: props.size || 'M',
            orientation: props.orientation || 'vertical'
        }, props.styles),
        ref: domRef,
        children: /*#__PURE__*/ (0, $jcfA8$reactjsxruntime.jsx)((0, $jcfA8$reactariacomponentsslots.Provider), {
            values: [
                [
                    (0, $6367bc87eb7d24ad$exports.HeadingContext),
                    {
                        styles: $55f1b8940d3e02db$var$heading({
                            orientation: orientation,
                            size: size
                        })
                    }
                ],
                [
                    (0, $6367bc87eb7d24ad$exports.ContentContext),
                    {
                        styles: $55f1b8940d3e02db$var$content({
                            size: size
                        })
                    }
                ],
                [
                    (0, $bde97c91243ed164$exports.IllustrationContext),
                    {
                        size: size === 'L' ? 'L' : 'M',
                        styles: $55f1b8940d3e02db$var$illustration({
                            orientation: orientation,
                            size: size,
                            isInDropZone: isInDropZone,
                            isDropTarget: isDropTarget
                        })
                    }
                ],
                [
                    (0, $e1016b8947dbadb8$exports.ButtonGroupContext),
                    {
                        styles: $55f1b8940d3e02db$var$buttonGroup
                    }
                ]
            ],
            children: children
        })
    });
});
