// @react-spectrum/s2/dist/private/InlineAlert.cjs
require("./InlineAlert.css");
var $7e7cdbd2b8ae2467$exports = require("../../icons/AlertTriangle.cjs");
var $6391be254c189366$exports = require("../../icons/CheckmarkCircle.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $0ed6e07b499b9797$exports = require("../../icons/InfoCircle.cjs");
var $4526404114e78c80$exports = require("./intlStrings.cjs");
var $e861f394b642505b$exports = require("../../icons/AlertDiamond.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $3stts$reactjsxruntime = require("react/jsx-runtime");
var $3stts$react = require("react");
var $3stts$reactariacomponentsslots = require("react-aria-components/slots");
var $3stts$reactariafilterDOMProps = require("react-aria/filterDOMProps");
var $3stts$reactariauseFocusRing = require("react-aria/useFocusRing");
var $3stts$reactariauseLocalizedStringFormatter = require("react-aria/useLocalizedStringFormatter");


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "InlineAlertContext", () => $72bf3a9f94a75d9f$export$c0de4eca0a8bf011);
$parcel$export(module.exports, "InlineAlert", () => $72bf3a9f94a75d9f$export$a3b2c96db9b0eb71);
/*
 * Copyright 2023 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 














const $72bf3a9f94a75d9f$export$c0de4eca0a8bf011 = /*#__PURE__*/ (0, $3stts$react.createContext)(null);
const $72bf3a9f94a75d9f$var$inlineAlert = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        rules += p[0];
    }
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    rules += ' sg131';
    if (!position) rules += ' _Pc131';
    rules += ' _oa131';
    rules += ' Th131';
    rules += ' Qh131';
    rules += ' Sh131';
    rules += ' Rh131';
    rules += ' oc131';
    rules += ' nc131';
    rules += ' kc131';
    rules += ' jc131';
    rules += ' _jf131';
    rules += ' _kc131';
    rules += ' hc131';
    rules += ' mc131';
    rules += ' lc131';
    if (props.fillStyle === "boldFill") rules += ' iA131';
    else if (props.fillStyle === "subtleFill") rules += ' iA131';
    else if (props.fillStyle === "border") {
        if (props.variant === "neutral") rules += ' iy131';
        else if (props.variant === "negative") rules += ' it131';
        else if (props.variant === "notice") rules += ' is131';
        else if (props.variant === "positive") rules += ' il131';
        else if (props.variant === "informative") rules += ' ix131';
    }
    if (props.variant === "neutral") {
        if (props.fillStyle === "boldFill") rules += ' g6131';
        else if (props.fillStyle === "subtleFill") rules += ' gI131';
        else if (props.fillStyle === "border") rules += ' gX131';
    } else if (props.variant === "negative") {
        if (props.fillStyle === "boldFill") rules += ' gB131';
        else if (props.fillStyle === "subtleFill") rules += ' gU131';
        else if (props.fillStyle === "border") rules += ' gX131';
    } else if (props.variant === "notice") {
        if (props.fillStyle === "boldFill") rules += ' gQ131';
        else if (props.fillStyle === "subtleFill") rules += ' gV131';
        else if (props.fillStyle === "border") rules += ' gX131';
    } else if (props.variant === "positive") {
        if (props.fillStyle === "boldFill") rules += ' g1131';
        else if (props.fillStyle === "subtleFill") rules += ' gA131';
        else if (props.fillStyle === "border") rules += ' gX131';
    } else if (props.variant === "informative") {
        if (props.fillStyle === "boldFill") rules += ' g2131';
        else if (props.fillStyle === "subtleFill") rules += ' gG131';
        else if (props.fillStyle === "border") rules += ' gX131';
    }
    return rules;
};
const $72bf3a9f94a75d9f$var$icon = function anonymous(props) {
    let rules = " ";
    rules += ' __wa131';
    if (props.fillStyle === "boldFill") {
        if (props.variant === "notice") rules += ' -_8sjo0b-tcwrnr131';
        else rules += ' -_8sjo0b-tw131';
    } else if (props.fillStyle === "subtleFill") {
        if (props.variant === "neutral") rules += ' -_8sjo0b-tf131';
        else if (props.variant === "negative") rules += ' -_8sjo0b-tq131';
        else if (props.variant === "notice") rules += ' -_8sjo0b-tq131';
        else if (props.variant === "positive") rules += ' -_8sjo0b-tt131';
        else if (props.variant === "informative") rules += ' -_8sjo0b-tu131';
    } else if (props.fillStyle === "border") {
        if (props.variant === "neutral") rules += ' -_8sjo0b-tf131';
        else if (props.variant === "negative") rules += ' -_8sjo0b-tq131';
        else if (props.variant === "notice") rules += ' -_8sjo0b-tm131';
        else if (props.variant === "positive") rules += ' -_8sjo0b-tt131';
        else if (props.variant === "informative") rules += ' -_8sjo0b-tu131';
    }
    return rules;
};
let $72bf3a9f94a75d9f$var$ICONS = {
    informative: (0, $0ed6e07b499b9797$exports.default),
    positive: (0, $6391be254c189366$exports.default),
    notice: (0, $e861f394b642505b$exports.default),
    negative: (0, $7e7cdbd2b8ae2467$exports.default),
    neutral: undefined
};
const $72bf3a9f94a75d9f$var$heading = function anonymous(props) {
    let rules = " ";
    rules += ' Jy131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-c131';
    rules += ' vx131';
    rules += ' xd131';
    rules += ' _xa131';
    rules += ' xX0cczbc131';
    rules += ' _xX0cczba131';
    rules += ' xfd131';
    rules += ' _xfa131';
    rules += ' xfX0cczbc131';
    rules += ' _xfX0cczba131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    if (props.fillStyle === "boldFill") {
        if (props.variant === "notice") rules += ' pg131';
        else rules += ' px131';
    } else rules += ' po131';
    return rules;
};
const $72bf3a9f94a75d9f$var$content = function anonymous(props) {
    let rules = " ";
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-c131';
    rules += ' vx131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fb131';
    rules += ' _FnuYUwec131';
    if (props.fillStyle === "boldFill") {
        if (props.variant === "notice") rules += ' pg131';
        else rules += ' px131';
    } else rules += ' pt131';
    return rules;
};
const $72bf3a9f94a75d9f$export$a3b2c96db9b0eb71 = /*#__PURE__*/ (0, $3stts$react.forwardRef)(function InlineAlert(props, ref) {
    let stringFormatter = (0, $3stts$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $72bf3a9f94a75d9f$export$c0de4eca0a8bf011);
    let { children: children, variant: variant = 'neutral', fillStyle: fillStyle = 'border', autoFocus: autoFocus } = props;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let Icon = null;
    let iconAlt = '';
    if (variant in $72bf3a9f94a75d9f$var$ICONS) {
        Icon = $72bf3a9f94a75d9f$var$ICONS[variant];
        if (Icon) iconAlt = stringFormatter.format(`inlinealert.${variant}`);
    }
    let { isFocusVisible: isFocusVisible, focusProps: focusProps } = (0, $3stts$reactariauseFocusRing.useFocusRing)({
        autoFocus: props.autoFocus
    });
    let autoFocusRef = (0, $3stts$react.useRef)(props.autoFocus);
    (0, $3stts$react.useEffect)(()=>{
        if (autoFocusRef.current && domRef.current) domRef.current.focus();
        autoFocusRef.current = false;
    }, [
        domRef
    ]);
    return /*#__PURE__*/ (0, $3stts$reactjsxruntime.jsx)("div", {
        ...(0, $3stts$reactariafilterDOMProps.filterDOMProps)(props),
        ...focusProps,
        ref: domRef,
        tabIndex: autoFocus ? -1 : undefined,
        autoFocus: autoFocus,
        role: "alert",
        style: props.UNSAFE_style,
        className: (props.UNSAFE_className || '') + $72bf3a9f94a75d9f$var$inlineAlert({
            variant: variant,
            fillStyle: fillStyle,
            isFocusVisible: isFocusVisible
        }, props.styles),
        children: /*#__PURE__*/ (0, $3stts$reactjsxruntime.jsxs)((0, $3stts$reactariacomponentsslots.Provider), {
            values: [
                [
                    (0, $6367bc87eb7d24ad$exports.HeadingContext),
                    {
                        styles: $72bf3a9f94a75d9f$var$heading({
                            fillStyle: fillStyle
                        })
                    }
                ],
                [
                    (0, $6367bc87eb7d24ad$exports.ContentContext),
                    {
                        styles: $72bf3a9f94a75d9f$var$content({
                            fillStyle: fillStyle
                        })
                    }
                ],
                [
                    (0, $bde97c91243ed164$exports.IconContext),
                    {
                        styles: $72bf3a9f94a75d9f$var$icon({
                            variant: variant,
                            fillStyle: fillStyle
                        })
                    }
                ]
            ],
            children: [
                Icon && /*#__PURE__*/ (0, $3stts$reactjsxruntime.jsx)(Icon, {
                    "aria-label": iconAlt
                }),
                children
            ]
        })
    });
});
