// @react-spectrum/s2/dist/private/InlineAlert.mjs
import "./InlineAlert.css";
import $e17a4836f4b6cc28$export$2e2bcd8739ae039 from "../../icons/AlertTriangle.mjs";
import $22a7d72fc05e138d$export$2e2bcd8739ae039 from "../../icons/CheckmarkCircle.mjs";
import {ContentContext as $8e847109a6ab556d$export$1cbdd774077931b4, HeadingContext as $8e847109a6ab556d$export$d688439359537581} from "./Content.mjs";
import {IconContext as $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed} from "../../icons/Icon.mjs";
import $4c8f17dac3ecefd9$export$2e2bcd8739ae039 from "../../icons/InfoCircle.mjs";
import $7sfGs$intlStringsmjs from "./intlStrings.mjs";
import $a951460932d22d7b$export$2e2bcd8739ae039 from "../../icons/AlertDiamond.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $7sfGs$jsx, jsxs as $7sfGs$jsxs} from "react/jsx-runtime";
import {createContext as $7sfGs$createContext, forwardRef as $7sfGs$forwardRef, useRef as $7sfGs$useRef, useEffect as $7sfGs$useEffect} from "react";
import {Provider as $7sfGs$Provider} from "react-aria-components/slots";
import {filterDOMProps as $7sfGs$filterDOMProps} from "react-aria/filterDOMProps";
import {useFocusRing as $7sfGs$useFocusRing} from "react-aria/useFocusRing";
import {useLocalizedStringFormatter as $7sfGs$useLocalizedStringFormatter} from "react-aria/useLocalizedStringFormatter";


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}
/*
 * Copyright 2023 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 














const $fc2cf6b397804b9c$export$c0de4eca0a8bf011 = /*#__PURE__*/ (0, $7sfGs$createContext)(null);
const $fc2cf6b397804b9c$var$inlineAlert = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        rules += p[0];
    }
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    rules += ' sg131';
    if (!position) rules += ' _Pc131';
    rules += ' _oa131';
    rules += ' Th131';
    rules += ' Qh131';
    rules += ' Sh131';
    rules += ' Rh131';
    rules += ' oc131';
    rules += ' nc131';
    rules += ' kc131';
    rules += ' jc131';
    rules += ' _jf131';
    rules += ' _kc131';
    rules += ' hc131';
    rules += ' mc131';
    rules += ' lc131';
    if (props.fillStyle === "boldFill") rules += ' iA131';
    else if (props.fillStyle === "subtleFill") rules += ' iA131';
    else if (props.fillStyle === "border") {
        if (props.variant === "neutral") rules += ' iy131';
        else if (props.variant === "negative") rules += ' it131';
        else if (props.variant === "notice") rules += ' is131';
        else if (props.variant === "positive") rules += ' il131';
        else if (props.variant === "informative") rules += ' ix131';
    }
    if (props.variant === "neutral") {
        if (props.fillStyle === "boldFill") rules += ' g6131';
        else if (props.fillStyle === "subtleFill") rules += ' gI131';
        else if (props.fillStyle === "border") rules += ' gX131';
    } else if (props.variant === "negative") {
        if (props.fillStyle === "boldFill") rules += ' gB131';
        else if (props.fillStyle === "subtleFill") rules += ' gU131';
        else if (props.fillStyle === "border") rules += ' gX131';
    } else if (props.variant === "notice") {
        if (props.fillStyle === "boldFill") rules += ' gQ131';
        else if (props.fillStyle === "subtleFill") rules += ' gV131';
        else if (props.fillStyle === "border") rules += ' gX131';
    } else if (props.variant === "positive") {
        if (props.fillStyle === "boldFill") rules += ' g1131';
        else if (props.fillStyle === "subtleFill") rules += ' gA131';
        else if (props.fillStyle === "border") rules += ' gX131';
    } else if (props.variant === "informative") {
        if (props.fillStyle === "boldFill") rules += ' g2131';
        else if (props.fillStyle === "subtleFill") rules += ' gG131';
        else if (props.fillStyle === "border") rules += ' gX131';
    }
    return rules;
};
const $fc2cf6b397804b9c$var$icon = function anonymous(props) {
    let rules = " ";
    rules += ' __wa131';
    if (props.fillStyle === "boldFill") {
        if (props.variant === "notice") rules += ' -_8sjo0b-tcwrnr131';
        else rules += ' -_8sjo0b-tw131';
    } else if (props.fillStyle === "subtleFill") {
        if (props.variant === "neutral") rules += ' -_8sjo0b-tf131';
        else if (props.variant === "negative") rules += ' -_8sjo0b-tq131';
        else if (props.variant === "notice") rules += ' -_8sjo0b-tq131';
        else if (props.variant === "positive") rules += ' -_8sjo0b-tt131';
        else if (props.variant === "informative") rules += ' -_8sjo0b-tu131';
    } else if (props.fillStyle === "border") {
        if (props.variant === "neutral") rules += ' -_8sjo0b-tf131';
        else if (props.variant === "negative") rules += ' -_8sjo0b-tq131';
        else if (props.variant === "notice") rules += ' -_8sjo0b-tm131';
        else if (props.variant === "positive") rules += ' -_8sjo0b-tt131';
        else if (props.variant === "informative") rules += ' -_8sjo0b-tu131';
    }
    return rules;
};
let $fc2cf6b397804b9c$var$ICONS = {
    informative: (0, $4c8f17dac3ecefd9$export$2e2bcd8739ae039),
    positive: (0, $22a7d72fc05e138d$export$2e2bcd8739ae039),
    notice: (0, $a951460932d22d7b$export$2e2bcd8739ae039),
    negative: (0, $e17a4836f4b6cc28$export$2e2bcd8739ae039),
    neutral: undefined
};
const $fc2cf6b397804b9c$var$heading = function anonymous(props) {
    let rules = " ";
    rules += ' Jy131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-c131';
    rules += ' vx131';
    rules += ' xd131';
    rules += ' _xa131';
    rules += ' xX0cczbc131';
    rules += ' _xX0cczba131';
    rules += ' xfd131';
    rules += ' _xfa131';
    rules += ' xfX0cczbc131';
    rules += ' _xfX0cczba131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    if (props.fillStyle === "boldFill") {
        if (props.variant === "notice") rules += ' pg131';
        else rules += ' px131';
    } else rules += ' po131';
    return rules;
};
const $fc2cf6b397804b9c$var$content = function anonymous(props) {
    let rules = " ";
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-c131';
    rules += ' vx131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fb131';
    rules += ' _FnuYUwec131';
    if (props.fillStyle === "boldFill") {
        if (props.variant === "notice") rules += ' pg131';
        else rules += ' px131';
    } else rules += ' pt131';
    return rules;
};
const $fc2cf6b397804b9c$export$a3b2c96db9b0eb71 = /*#__PURE__*/ (0, $7sfGs$forwardRef)(function InlineAlert(props, ref) {
    let stringFormatter = (0, $7sfGs$useLocalizedStringFormatter)((0, ($parcel$interopDefault($7sfGs$intlStringsmjs))), '@react-spectrum/s2');
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $fc2cf6b397804b9c$export$c0de4eca0a8bf011);
    let { children: children, variant: variant = 'neutral', fillStyle: fillStyle = 'border', autoFocus: autoFocus } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let Icon = null;
    let iconAlt = '';
    if (variant in $fc2cf6b397804b9c$var$ICONS) {
        Icon = $fc2cf6b397804b9c$var$ICONS[variant];
        if (Icon) iconAlt = stringFormatter.format(`inlinealert.${variant}`);
    }
    let { isFocusVisible: isFocusVisible, focusProps: focusProps } = (0, $7sfGs$useFocusRing)({
        autoFocus: props.autoFocus
    });
    let autoFocusRef = (0, $7sfGs$useRef)(props.autoFocus);
    (0, $7sfGs$useEffect)(()=>{
        if (autoFocusRef.current && domRef.current) domRef.current.focus();
        autoFocusRef.current = false;
    }, [
        domRef
    ]);
    return /*#__PURE__*/ (0, $7sfGs$jsx)("div", {
        ...(0, $7sfGs$filterDOMProps)(props),
        ...focusProps,
        ref: domRef,
        tabIndex: autoFocus ? -1 : undefined,
        autoFocus: autoFocus,
        role: "alert",
        style: props.UNSAFE_style,
        className: (props.UNSAFE_className || '') + $fc2cf6b397804b9c$var$inlineAlert({
            variant: variant,
            fillStyle: fillStyle,
            isFocusVisible: isFocusVisible
        }, props.styles),
        children: /*#__PURE__*/ (0, $7sfGs$jsxs)((0, $7sfGs$Provider), {
            values: [
                [
                    (0, $8e847109a6ab556d$export$d688439359537581),
                    {
                        styles: $fc2cf6b397804b9c$var$heading({
                            fillStyle: fillStyle
                        })
                    }
                ],
                [
                    (0, $8e847109a6ab556d$export$1cbdd774077931b4),
                    {
                        styles: $fc2cf6b397804b9c$var$content({
                            fillStyle: fillStyle
                        })
                    }
                ],
                [
                    (0, $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed),
                    {
                        styles: $fc2cf6b397804b9c$var$icon({
                            variant: variant,
                            fillStyle: fillStyle
                        })
                    }
                ]
            ],
            children: [
                Icon && /*#__PURE__*/ (0, $7sfGs$jsx)(Icon, {
                    "aria-label": iconAlt
                }),
                children
            ]
        })
    });
});


export {$fc2cf6b397804b9c$export$c0de4eca0a8bf011 as InlineAlertContext, $fc2cf6b397804b9c$export$a3b2c96db9b0eb71 as InlineAlert};
