// @react-spectrum/s2/dist/private/LabeledValue.cjs
require("./LabeledValue.css");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $bfa4962d90c8af48$exports = require("./Form.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $cDYdk$reactjsxruntime = require("react/jsx-runtime");
var $cDYdk$internationalizeddate = require("@internationalized/date");
var $cDYdk$react = require("react");
var $cDYdk$reactariafilterDOMProps = require("react-aria/filterDOMProps");
var $cDYdk$reactariauseDateFormatter = require("react-aria/useDateFormatter");
var $cDYdk$reactariauseListFormatter = require("react-aria/useListFormatter");
var $cDYdk$reactariauseNumberFormatter = require("react-aria/useNumberFormatter");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "LabeledValueContext", () => $ad3c2d5408b92dda$export$8ba7143a95895992);
$parcel$export(module.exports, "LabeledValue", () => $ad3c2d5408b92dda$export$d1328f67a56fa517);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 










const $ad3c2d5408b92dda$export$8ba7143a95895992 = /*#__PURE__*/ (0, $cDYdk$react.createContext)(null);
const $ad3c2d5408b92dda$var$fieldStyles = function anonymous(props, overrides) {
    let rules = " ";
    let gridColumnStart = false;
    let gridColumnEnd = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "z") gridColumnStart = true;
        if (p[1] === "y") gridColumnEnd = true;
        rules += p[0];
    }
    rules += ' se131';
    if (props.isInForm) {
        if (!gridColumnStart) rules += ' z2mU131';
    }
    if (props.isInForm) {
        if (props.labelPosition === "side") {
            if (!gridColumnEnd) rules += ' yvoofG131';
        }
    }
    if (props.isInForm) rules += ' DLwlgod131';
    else {
        if (props.labelPosition === "side") rules += ' DPlN5qb131';
        else if (props.labelPosition === "top") rules += ' DM8Mfn131';
    }
    if (props.labelPosition === "side") rules += ' EPlN5qb131';
    else if (props.labelPosition === "top") rules += ' E4w1sLc131';
    if (props.labelPosition === "side") rules += ' CbRM95c131';
    else if (props.labelPosition === "top") rules += ' Cxaocre131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    rules += ' ea131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    if (props.size === "XL") rules += ' -BhX7R-Fz131';
    else if (props.size === "L") rules += ' -BhX7R-Fv131';
    else if (props.size === "S") rules += ' -BhX7R-Fp131';
    else if (props.size === "XS") rules += ' -BhX7R-Fn131';
    else rules += ' -BhX7R-Fx131';
    rules += ' -AJjOLd-UVkF0Db131';
    rules += ' qg131';
    rules += ' __ca131';
    return rules;
};
const $ad3c2d5408b92dda$var$valueStyles = function anonymous(props) {
    let rules = " ";
    rules += ' zXcEWr131';
    rules += ' yXcEWr131';
    rules += ' BXcEWr131';
    rules += ' AXcEWr131';
    if (props.size === "XL") rules += ' Nl131';
    else if (props.size === "L") rules += ' Nj131';
    else if (props.size === "S") rules += ' Nf131';
    else if (props.size === "XS") rules += ' Ne131';
    else rules += ' Nk131';
    if (props.isQuiet) rules += ' _qd131';
    else rules += ' _qtb131';
    if (props.size === "XL") rules += ' -Anxdcc-Zt131';
    else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
    else if (props.size === "S") rules += ' -Anxdcc-Zr131';
    else rules += ' -Anxdcc-Z3fn9H131';
    rules += ' __v6TNLWb131';
    if (props.isInForm) {
        if (props.size === "XL") rules += ' Mk131';
        else if (props.size === "L") rules += ' Mh131';
        else if (props.size === "S") rules += ' Mg131';
        else if (props.size === "XS") rules += ' Mf131';
        else rules += ' Mi131';
    }
    rules += ' sd131';
    rules += ' eb131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.size === "XL") rules += ' pt131';
    else if (props.size === "L") rules += ' pt131';
    else if (props.size === "S") rules += ' pt131';
    else if (props.size === "XS") rules += ' pt131';
    else rules += ' pt131';
    return rules;
};
const $ad3c2d5408b92dda$export$d1328f67a56fa517 = /*#__PURE__*/ (0, $cDYdk$react.forwardRef)(function LabeledValue(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $ad3c2d5408b92dda$export$8ba7143a95895992);
    props = (0, $bfa4962d90c8af48$exports.useFormProps)(props);
    let { label: label, value: value, formatOptions: formatOptions, size: size = 'M', labelPosition: labelPosition = 'top', labelAlign: labelAlign = 'start', contextualHelp: contextualHelp, UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, styles: styles, ...otherProps } = props;
    let formContext = (0, $cDYdk$react.useContext)((0, $bfa4962d90c8af48$exports.FormContext));
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    (0, $cDYdk$react.useEffect)(()=>{
        if (domRef?.current && domRef.current.querySelectorAll('input, [contenteditable], textarea').length > 0) throw new Error('LabeledValue cannot contain an editable value.');
    }, [
        domRef
    ]);
    let children;
    if (Array.isArray(value)) children = /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsx)($ad3c2d5408b92dda$var$FormattedStringList, {
        value: value,
        formatOptions: formatOptions
    });
    else if (typeof value === 'number') children = /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsx)($ad3c2d5408b92dda$var$FormattedNumber, {
        value: value,
        formatOptions: formatOptions
    });
    else if (typeof value === 'object' && value !== null && 'start' in value && typeof value.start === 'number') children = /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsx)($ad3c2d5408b92dda$var$FormattedNumber, {
        value: value,
        formatOptions: formatOptions
    });
    else if (typeof value === 'object' && value !== null && 'start' in value) children = /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsx)($ad3c2d5408b92dda$var$FormattedDate, {
        value: value,
        formatOptions: formatOptions
    });
    else if (value instanceof Date || typeof value === 'object' && value !== null && ('calendar' in value || 'hour' in value)) children = /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsx)($ad3c2d5408b92dda$var$FormattedDate, {
        value: value,
        formatOptions: formatOptions
    });
    else if (typeof value === 'string') children = value;
    else if (/*#__PURE__*/ (0, $cDYdk$react.isValidElement)(value)) children = value;
    return /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsxs)("div", {
        ...(0, $cDYdk$reactariafilterDOMProps.filterDOMProps)(otherProps),
        ref: domRef,
        style: UNSAFE_style,
        className: UNSAFE_className + $ad3c2d5408b92dda$var$fieldStyles({
            isInForm: !!formContext,
            labelPosition: labelPosition,
            size: size
        }, styles),
        children: [
            /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldLabel), {
                elementType: "span",
                size: size,
                labelPosition: labelPosition,
                labelAlign: labelAlign,
                contextualHelp: contextualHelp,
                children: label
            }),
            /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsx)("span", {
                className: $ad3c2d5408b92dda$var$valueStyles({
                    isInForm: !!formContext,
                    size: size,
                    labelPosition: labelPosition
                }),
                children: children
            })
        ]
    });
});
function $ad3c2d5408b92dda$var$FormattedStringList({ value: value, formatOptions: formatOptions }) {
    let formatter = (0, $cDYdk$reactariauseListFormatter.useListFormatter)(formatOptions ?? {});
    return /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsx)((0, $cDYdk$reactjsxruntime.Fragment), {
        children: formatter.format(value)
    });
}
function $ad3c2d5408b92dda$var$FormattedNumber({ value: value, formatOptions: formatOptions }) {
    let formatter = (0, $cDYdk$reactariauseNumberFormatter.useNumberFormatter)(formatOptions);
    if (typeof value === 'object') return /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsx)((0, $cDYdk$reactjsxruntime.Fragment), {
        children: formatter.formatRange(value.start, value.end)
    });
    return /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsx)((0, $cDYdk$reactjsxruntime.Fragment), {
        children: formatter.format(value)
    });
}
function $ad3c2d5408b92dda$var$FormattedDate({ value: value, formatOptions: formatOptions }) {
    if (!formatOptions) formatOptions = $ad3c2d5408b92dda$var$getDefaultFormatOptions('start' in value ? value.start : value);
    let dateFormatter = (0, $cDYdk$reactariauseDateFormatter.useDateFormatter)(formatOptions);
    let timeZone = dateFormatter.resolvedOptions().timeZone || (0, $cDYdk$internationalizeddate.getLocalTimeZone)();
    if ('start' in value && 'end' in value) {
        let start = $ad3c2d5408b92dda$var$convertDateTime(value.start, timeZone);
        let end = $ad3c2d5408b92dda$var$convertDateTime(value.end, timeZone);
        return /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsx)((0, $cDYdk$reactjsxruntime.Fragment), {
            children: dateFormatter.formatRange(start, end)
        });
    }
    return /*#__PURE__*/ (0, $cDYdk$reactjsxruntime.jsx)((0, $cDYdk$reactjsxruntime.Fragment), {
        children: dateFormatter.format($ad3c2d5408b92dda$var$convertDateTime(value, timeZone))
    });
}
function $ad3c2d5408b92dda$var$convertDateTime(value, timeZone) {
    if ('timeZone' in value) return value.toDate();
    else if ('calendar' in value) return value.toDate(timeZone);
    else if (!(value instanceof Date)) return (0, $cDYdk$internationalizeddate.toCalendarDateTime)((0, $cDYdk$internationalizeddate.today)((0, $cDYdk$internationalizeddate.getLocalTimeZone)()), value).toDate(timeZone);
    return value;
}
function $ad3c2d5408b92dda$var$getDefaultFormatOptions(value) {
    if (value instanceof Date) return {
        dateStyle: 'long',
        timeStyle: 'short'
    };
    else if ('timeZone' in value) return {
        year: 'numeric',
        month: 'long',
        day: 'numeric',
        hour: 'numeric',
        minute: 'numeric',
        timeZone: value.timeZone,
        timeZoneName: 'short'
    };
    else if ('hour' in value && 'year' in value) return {
        dateStyle: 'long',
        timeStyle: 'short'
    };
    else if ('hour' in value) return {
        timeStyle: 'short'
    };
    else return {
        dateStyle: 'long'
    };
}
