// @react-spectrum/s2/dist/private/ListView.mjs
import "./ListView.css";
import {ActionButtonGroupContext as $10401bdc118cbc90$export$33f5f2f2cb85d743} from "./ActionButtonGroup.mjs";
import {ActionMenuContext as $8bf3cd2442eb485e$export$7f4b66acaca28bd5} from "./ActionMenu.mjs";
import {centerBaseline as $1f4b04be3f24aae3$export$9d7e2342a7e53afa} from "./CenterBaseline.mjs";
import {Checkbox as $9b405e0f8095dce0$export$48513f6b9f8ce62d} from "./Checkbox.mjs";
import $4b69f02ec06b9226$export$2e2bcd8739ae039 from "./Chevron.mjs";
import {description as $ba354da66aaa1d20$export$6d59db4903f20f7d, DragPreview as $ba354da66aaa1d20$export$905ab40ac2179daa, icon as $ba354da66aaa1d20$export$1ca1ec8b29a4ce27, iconCenterWrapper as $ba354da66aaa1d20$export$31d1842a0fd736e6, label as $ba354da66aaa1d20$export$1237798dc640739a} from "./DragPreview.mjs";
import {DragHandleButton as $3c1ab1fd8f31a3e2$export$4c99a519d851f90c, InsertionIndicator as $3c1ab1fd8f31a3e2$export$2c0bab5914a9d088} from "./dnd-utils.mjs";
import {IconContext as $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed} from "../../icons/Icon.mjs";
import {ImageContext as $dbd6f0b2503b938c$export$8151eb379fcf2442} from "./Image.mjs";
import $alyq6$intlStringsmjs from "./intlStrings.mjs";
import $d9f847a2895a93dc$export$2e2bcd8739ae039 from "./LinkOut.mjs";
import {ProgressCircle as $41ddd91dbbf0c389$export$c79b9d6b4cc92af7} from "./ProgressCircle.mjs";
import {Text as $8e847109a6ab556d$export$5f1af8db9871e1d6, TextContext as $8e847109a6ab556d$export$9afb8bc826b033ea} from "./Content.mjs";
import {useActionBarContainer as $f21f2186348fbc5b$export$13f32e21845e01d0} from "./ActionBar.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {useScale as $859432f3b3573fcb$export$a8d2043b2d807f4d} from "./utils.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $alyq6$jsx, jsxs as $alyq6$jsxs, Fragment as $alyq6$Fragment} from "react/jsx-runtime";
import {CheckboxContext as $alyq6$CheckboxContext} from "react-aria-components/Checkbox";
import {Collection as $alyq6$Collection} from "react-aria/Collection";
import {CollectionRendererContext as $alyq6$CollectionRendererContext, DefaultCollectionRenderer as $alyq6$DefaultCollectionRenderer} from "react-aria-components/CollectionBuilder";
import {useSlottedContext as $alyq6$useSlottedContext, Provider as $alyq6$Provider, DEFAULT_SLOT as $alyq6$DEFAULT_SLOT} from "react-aria-components/slots";
import {createContext as $alyq6$createContext, forwardRef as $alyq6$forwardRef, useRef as $alyq6$useRef, useContext as $alyq6$useContext} from "react";
import {GridListLoadMoreItem as $alyq6$GridListLoadMoreItem, GridList as $alyq6$GridList, GridListItem as $alyq6$GridListItem} from "react-aria-components/GridList";
import {Virtualizer as $alyq6$Virtualizer} from "react-aria-components/Virtualizer";
import {ListLayout as $alyq6$ListLayout} from "react-stately/useVirtualizerState";
import {useLocale as $alyq6$useLocale} from "react-aria/I18nProvider";
import {useLocalizedStringFormatter as $alyq6$useLocalizedStringFormatter} from "react-aria/useLocalizedStringFormatter";


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}
/*
 * Copyright 2026 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 



























const $6d4c8b365b6e9016$export$870039b0abfe3de0 = /*#__PURE__*/ (0, $alyq6$createContext)(null);
let $6d4c8b365b6e9016$var$InternalListViewContext = /*#__PURE__*/ (0, $alyq6$createContext)({});
const $6d4c8b365b6e9016$var$listViewWrapper = function anonymous(props, overrides) {
    let rules = " ";
    let minHeight = false;
    let minWidth = false;
    let position = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L|F|M|K)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "M") minHeight = true;
        if (p[1] === "N") minWidth = true;
        if (p[1] === "_P") position = true;
        rules += p[0];
    }
    if (!minHeight) rules += ' Ma131';
    if (!minWidth) rules += ' Ng131';
    rules += ' sd131';
    rules += ' _Bb131';
    rules += ' __ca131';
    if (!position) rules += ' _Pc131';
    rules += ' _Nb131';
    rules += ' Pb131';
    rules += ' -qVUQke-ob131';
    return rules;
};
// similar to tableview, use this approach so we can actually have a 2px outline when root dropping.
// cant do a external box shadow due to the clipping that is applied on the wrapper element...
// an inset box shadow runs into problems with the item background clipping the box shadow...
const $6d4c8b365b6e9016$var$rootDropOutline = "IJKxRb";
// When any row has a trailing icon, reserve space so actions align.
const $6d4c8b365b6e9016$var$hasTrailingIconRows = ':has([data-has-trailing-icon]) [role="row"]';
const $6d4c8b365b6e9016$var$dropTargetBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 10%)]";
const $6d4c8b365b6e9016$var$listView = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    if (props.isQuiet) rules += ' _Ka131';
    else rules += ' _Kb131';
    rules += ' _4c131';
    rules += ' Ma131';
    rules += ' Na131';
    rules += ' Za131';
    rules += ' Fb131';
    rules += ' _oa131';
    rules += ' _Na131';
    rules += ' Pa131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.isDropTarget) {
        rules += ' glzvDBe131';
        rules += ' gla131';
    } else if (props.isQuiet) rules += ' g9131';
    else rules += ' gX131';
    if (props.isQuiet) rules += ' od131';
    else rules += ' ob131';
    if (props.isQuiet) rules += ' nd131';
    else rules += ' nb131';
    if (props.isQuiet) rules += ' kd131';
    else rules += ' kb131';
    if (props.isQuiet) rules += ' jd131';
    else rules += ' jb131';
    rules += ' iq131';
    rules += ' ila131';
    if (props.isQuiet) rules += ' _ka131';
    else rules += ' _kb131';
    if (props.isQuiet) rules += ' ha131';
    else rules += ' hb131';
    if (props.isQuiet) rules += ' ma131';
    else rules += ' mb131';
    if (props.isQuiet) rules += ' la131';
    else rules += ' lb131';
    rules += ' _jf131';
    rules += ' _yb131';
    rules += ' -Ev4N4b-Zd131';
    rules += ' -Ev4N4b-1bNH0cZuUmpre131';
    return rules;
};
class $6d4c8b365b6e9016$export$6f8e4757b55cd070 extends (0, $alyq6$ListLayout) {
    getDropTargetLayoutInfo(target) {
        let layoutInfo = super.getDropTargetLayoutInfo(target);
        layoutInfo.zIndex = 1;
        layoutInfo.allowOverflow = true;
        return layoutInfo;
    }
}
const $6d4c8b365b6e9016$export$84d0dd190d551cd1 = /*#__PURE__*/ (0, $alyq6$forwardRef)(function ListView(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $6d4c8b365b6e9016$export$870039b0abfe3de0);
    let { children: children, isQuiet: isQuiet, selectionStyle: selectionStyle = 'checkbox', overflowMode: overflowMode = 'truncate', loadingState: loadingState, onLoadMore: onLoadMore, renderEmptyState: userRenderEmptyState, hideLinkOutIcon: hideLinkOutIcon = false, dragAndDropHooks: dragAndDropHooks, ...otherProps } = props;
    let scale = (0, $859432f3b3573fcb$export$a8d2043b2d807f4d)();
    if (dragAndDropHooks && dragAndDropHooks.renderDragPreview == null) dragAndDropHooks.renderDragPreview = (items)=>/*#__PURE__*/ (0, $alyq6$jsx)((0, $ba354da66aaa1d20$export$905ab40ac2179daa), {
            items: items,
            overflowMode: overflowMode
        });
    if (dragAndDropHooks) dragAndDropHooks.renderDropIndicator = (target)=>/*#__PURE__*/ (0, $alyq6$jsx)((0, $3c1ab1fd8f31a3e2$export$2c0bab5914a9d088), {
            target: target
        });
    let stringFormatter = (0, $alyq6$useLocalizedStringFormatter)((0, ($parcel$interopDefault($alyq6$intlStringsmjs))), '@react-spectrum/s2');
    let rowHeight = scale === 'large' ? 50 : 40;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let scrollRef = (0, $alyq6$useRef)(null);
    let isLoading = loadingState === 'loading' || loadingState === 'loadingMore';
    let renderEmptyState;
    if (userRenderEmptyState != null && !isLoading) renderEmptyState = (renderProps)=>/*#__PURE__*/ (0, $alyq6$jsx)("div", {
            className: $6d4c8b365b6e9016$var$emptyStateWrapper,
            children: /*#__PURE__*/ (0, $alyq6$jsx)((0, $alyq6$CollectionRendererContext).Provider, {
                value: (0, $alyq6$DefaultCollectionRenderer),
                children: userRenderEmptyState(renderProps)
            })
        });
    else if (loadingState === 'loading') renderEmptyState = ()=>/*#__PURE__*/ (0, $alyq6$jsx)("div", {
            className: $6d4c8b365b6e9016$var$centeredWrapper,
            children: /*#__PURE__*/ (0, $alyq6$jsx)("div", {
                className: $6d4c8b365b6e9016$var$loadingSpinnerWrapper,
                children: /*#__PURE__*/ (0, $alyq6$jsx)((0, $41ddd91dbbf0c389$export$c79b9d6b4cc92af7), {
                    isIndeterminate: true,
                    "aria-label": stringFormatter.format('table.loading')
                })
            })
        });
    let loadMoreSpinner = onLoadMore ? /*#__PURE__*/ (0, $alyq6$jsx)((0, $alyq6$GridListLoadMoreItem), {
        isLoading: loadingState === 'loadingMore',
        onLoadMore: onLoadMore,
        className: " Fb131 Za131 Tt131 Qt131",
        children: /*#__PURE__*/ (0, $alyq6$jsx)("div", {
            className: $6d4c8b365b6e9016$var$centeredWrapper,
            children: /*#__PURE__*/ (0, $alyq6$jsx)("div", {
                className: $6d4c8b365b6e9016$var$loadingSpinnerWrapper,
                children: /*#__PURE__*/ (0, $alyq6$jsx)((0, $41ddd91dbbf0c389$export$c79b9d6b4cc92af7), {
                    isIndeterminate: true,
                    "aria-label": stringFormatter.format('table.loadingMore')
                })
            })
        })
    }) : null;
    let wrappedChildren;
    let gridListProps = otherProps;
    if (typeof children === 'function' && otherProps.items) {
        let { items: items, dependencies: dependencies = [], ...rest } = otherProps;
        gridListProps = rest;
        wrappedChildren = /*#__PURE__*/ (0, $alyq6$jsxs)((0, $alyq6$Fragment), {
            children: [
                /*#__PURE__*/ (0, $alyq6$jsx)((0, $alyq6$Collection), {
                    items: items,
                    dependencies: dependencies,
                    children: children
                }),
                loadMoreSpinner
            ]
        });
    } else wrappedChildren = /*#__PURE__*/ (0, $alyq6$jsxs)((0, $alyq6$Fragment), {
        children: [
            children,
            loadMoreSpinner
        ]
    });
    let { selectedKeys: selectedKeys, onSelectionChange: onSelectionChange, actionBar: actionBar, actionBarHeight: actionBarHeight } = (0, $f21f2186348fbc5b$export$13f32e21845e01d0)({
        ...props,
        scrollRef: scrollRef
    });
    return /*#__PURE__*/ (0, $alyq6$jsxs)("div", {
        ref: domRef,
        className: (props.UNSAFE_className || '') + $6d4c8b365b6e9016$var$listViewWrapper(null, props.styles) + ' ' + $6d4c8b365b6e9016$var$rootDropOutline,
        style: props.UNSAFE_style,
        children: [
            /*#__PURE__*/ (0, $alyq6$jsx)((0, $alyq6$Virtualizer), {
                layout: $6d4c8b365b6e9016$export$6f8e4757b55cd070,
                layoutOptions: {
                    estimatedRowHeight: rowHeight,
                    loaderHeight: 60,
                    dropIndicatorThickness: 0
                },
                children: /*#__PURE__*/ (0, $alyq6$jsx)($6d4c8b365b6e9016$var$InternalListViewContext.Provider, {
                    value: {
                        isQuiet: isQuiet,
                        selectionStyle: selectionStyle,
                        overflowMode: overflowMode,
                        scale: scale,
                        hideLinkOutIcon: hideLinkOutIcon
                    },
                    children: /*#__PURE__*/ (0, $alyq6$jsx)((0, $alyq6$GridList), {
                        ref: scrollRef,
                        ...gridListProps,
                        dragAndDropHooks: dragAndDropHooks,
                        selectionBehavior: selectionStyle === 'highlight' ? 'replace' : 'toggle',
                        selectionMode: gridListProps.selectionMode,
                        renderEmptyState: renderEmptyState,
                        style: {
                            paddingBottom: actionBarHeight > 0 ? actionBarHeight + 8 : 0,
                            scrollPaddingBottom: actionBarHeight > 0 ? actionBarHeight + 8 : 0
                        },
                        className: (renderProps)=>$6d4c8b365b6e9016$var$listView({
                                ...renderProps,
                                isQuiet: isQuiet,
                                isDropTarget: renderProps.isDropTarget
                            }),
                        selectedKeys: selectedKeys,
                        defaultSelectedKeys: undefined,
                        onSelectionChange: onSelectionChange,
                        children: wrappedChildren
                    })
                })
            }),
            actionBar
        ]
    });
});
const $6d4c8b365b6e9016$var$selectedBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 7%)]";
const $6d4c8b365b6e9016$var$selectedActiveBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 10%)]";
// TODO: removed the background color in HCM for highlight selection since it made it hard to see the focus
// ring of the drag button, this matches v3 anyways. thoughts?
const $6d4c8b365b6e9016$var$listitem = function anonymous(props) {
    let rules = " ";
    rules += ' _Le131';
    rules += ' _oa131';
    rules += ' qh131';
    rules += ' Sd131';
    rules += ' Rd131';
    rules += ' Tt131';
    rules += ' Qt131';
    rules += ' g9131';
    if (props.isDisabled) rules += ' pp131';
    else if (props.isSelected) {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    } else {
        if (props.isPressed) rules += ' pt131';
        else if (props.isFocusVisible) rules += ' pt131';
        else if (props.isHovered) rules += ' pt131';
        else rules += ' pv131';
    }
    if (props.isDisabled) rules += ' plc131';
    else if (props.isSelected) {
        if (props.selectionStyle === "highlight") rules += ' ple131';
    } else rules += ' plb131';
    rules += ' _Pc131';
    rules += ' z2mU131';
    rules += ' yzZKy131';
    rules += ' se131';
    rules += ' CTJxasc131';
    rules += ' Ds5idie131';
    rules += ' Exumf9b131';
    rules += ' UDNKd2dai5a0131';
    rules += ' ea131';
    if (props.scale === "large") rules += ' MODqQub131';
    else rules += ' Mh131';
    rules += ' _Xa131';
    if (props.isLink) rules += ' ry131';
    else rules += ' ri131';
    if (props.isSelected) {
        if (props.selectionStyle === "checkbox") rules += ' -EH8uVb-iq131';
        else if (props.selectionStyle === "highlight") {
            rules += ' -EH8uVb-iw131';
            rules += ' -EH8uVb-lid131';
        }
    } else {
        rules += ' -EH8uVb-iq131';
        rules += ' -EH8uVb-lia131';
    }
    rules += ' _ka131';
    if (props.isLastItem) {
        if (props.isQuiet) rules += ' ha131';
        else rules += ' hb131';
    } else rules += ' hb131';
    rules += ' ma131';
    rules += ' la131';
    rules += ' _jf131';
    if (props.isSelected) rules += ' iA131';
    else if (props.isNextSelected) rules += ' iA131';
    else rules += ' i4pzfu131';
    rules += ' ila131';
    rules += ' -Bu7z1c-ob131';
    rules += ' _yb131';
    return rules;
};
const $6d4c8b365b6e9016$var$insetBorderRadius = 'calc(var(--radius) - 1px)';
const $6d4c8b365b6e9016$var$rootDropSelectedRowBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 28%)]";
const $6d4c8b365b6e9016$var$rowDropBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 10%)]";
const $6d4c8b365b6e9016$var$rootRowDropStyles = {
    // Unlike v3 tableview, v3 listview has the same background color for the listview itself and the rows when
    // dropping on root
    default: $6d4c8b365b6e9016$var$dropTargetBackground,
    isSelected: $6d4c8b365b6e9016$var$rootDropSelectedRowBackground,
    forcedColors: 'Background'
};
const $6d4c8b365b6e9016$var$rowDropStyles = {
    // Also unlike v3, dropping on a selected row vs a non selected row doesn't have any difference in background color
    default: $6d4c8b365b6e9016$var$rowDropBackground,
    isSelected: $6d4c8b365b6e9016$var$rowDropBackground,
    forcedColors: 'Background'
};
const $6d4c8b365b6e9016$var$listRowBackground = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' _9a131';
    if (props.isFirstItem) rules += ' Wr131';
    else if (props.isPrevSelected) rules += ' Wr131';
    else if (props.isSelected) rules += ' WLPOtrc131';
    else rules += ' Wr131';
    rules += ' Wlr131';
    rules += ' _Er131';
    rules += ' _Rr131';
    if (props.isSelected) rules += ' _lLPOtrc131';
    else rules += ' _lr131';
    if (props.isDropTarget) {
        if (props.isSelected) rules += ' glzvDBe131';
        else rules += ' glzvDBe131';
        rules += ' gla131';
    } else {
        if (props.isSelected) {
            if (props.selectionStyle === "highlight") {
                if (props.isPressed) rules += ' gaZwDBe131';
                else if (props.isHovered) rules += ' gaZwDBe131';
                else rules += ' glzvDBe131';
            } else if (props.selectionStyle === "checkbox") {
                if (props.isFocusVisible) rules += ' gRfT7e131';
                else if (props.isPressed) rules += ' gRfT7e131';
                else if (props.isHovered) rules += ' gRfT7e131';
                else rules += ' gj1B7i131';
            }
        } else if (props.isPressed) {
            if (props.selectionStyle === "checkbox") rules += ' gRfT7e131';
            else rules += ' g7IonAe131';
        } else if (props.isHovered) {
            if (props.selectionStyle === "checkbox") rules += ' gj1B7i131';
            else rules += ' gVQkD8c131';
        } else rules += ' gs9yXOb131';
        if (props.selectionStyle === "highlight") {
            if (props.isSelected) rules += ' gle131';
            else rules += ' gl9131';
        } else rules += ' gl9131';
        if (props.isSelected) rules += ' gRQaNpdubHDBe131';
        else rules += ' gRQaNpdlzvDBe131';
        rules += ' gRQaNpdla131';
    }
    if (props.isDropTarget) rules += ' o7E7Gic131';
    else if (props.isPrevNotSelected) {
        if (props.isQuiet) rules += ' ob131';
        else if (props.isSelected) {
            if (props.selectionStyle === "highlight") rules += ' o7E7Gic131';
            else if (props.selectionStyle === "checkbox") rules += ' od131';
        }
    } else if (props.isSelected) rules += ' od131';
    else if (props.isQuiet) rules += ' ob131';
    if (props.isDropTarget) rules += ' n7E7Gic131';
    else if (props.isPrevNotSelected) {
        if (props.isQuiet) rules += ' nb131';
        else if (props.isSelected) {
            if (props.selectionStyle === "highlight") rules += ' n7E7Gic131';
            else if (props.selectionStyle === "checkbox") rules += ' nd131';
        }
    } else if (props.isSelected) rules += ' nd131';
    else if (props.isQuiet) rules += ' nb131';
    if (props.isDropTarget) rules += ' k7E7Gic131';
    else if (props.isNextNotSelected) {
        if (props.isQuiet) rules += ' kb131';
        else if (props.isSelected) {
            if (props.selectionStyle === "highlight") rules += ' k7E7Gic131';
            else if (props.selectionStyle === "checkbox") rules += ' kd131';
        }
    } else if (props.isSelected) rules += ' kd131';
    else if (props.isQuiet) rules += ' kb131';
    if (props.isDropTarget) rules += ' j7E7Gic131';
    else if (props.isNextNotSelected) {
        if (props.isQuiet) rules += ' jb131';
        else if (props.isSelected) {
            if (props.selectionStyle === "highlight") rules += ' j7E7Gic131';
            else if (props.selectionStyle === "checkbox") rules += ' jd131';
        }
    } else if (props.isSelected) rules += ' jd131';
    else if (props.isQuiet) rules += ' jb131';
    if (props.isDropTarget) rules += ' _kc131';
    else if (props.isPrevSelected) rules += ' _ka131';
    else {
        if (props.selectionStyle === "highlight") rules += ' _kb131';
        else if (props.selectionStyle === "checkbox") rules += ' _ka131';
    }
    if (props.isDropTarget) rules += ' hc131';
    else if (props.isNextSelected) rules += ' ha131';
    else {
        if (props.selectionStyle === "highlight") rules += ' hb131';
        else if (props.selectionStyle === "checkbox") rules += ' ha131';
    }
    if (props.isDropTarget) rules += ' mc131';
    else {
        if (props.selectionStyle === "highlight") rules += ' mb131';
        else if (props.selectionStyle === "checkbox") rules += ' ma131';
    }
    if (props.isDropTarget) rules += ' lc131';
    else {
        if (props.selectionStyle === "highlight") rules += ' lb131';
        else if (props.selectionStyle === "checkbox") rules += ' la131';
    }
    rules += ' _jf131';
    if (props.isDropTarget) rules += ' ix131';
    else if (props.isSelected) {
        if (props.selectionStyle === "highlight") rules += ' i4pzfu131';
        else if (props.selectionStyle === "checkbox") rules += ' iA131';
    } else rules += ' iA131';
    if (props.isDropTarget) rules += ' ild131';
    return rules;
};
let $6d4c8b365b6e9016$var$listRowFocusRing = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    if (props.selectionStyle === "highlight") rules += ' Ola131';
    else rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _MlkNNfn131';
    rules += ' _Kb131';
    rules += ' _KlToPtrc131';
    rules += ' _Pa131';
    if (props.isFirstItem) rules += ' Wr131';
    else rules += ' WLPOtrc131';
    if (props.selectionStyle === "highlight") rules += ' _lLPOtrc131';
    else if (props.selectionStyle === "checkbox") {
        if (props.isNextSelected) rules += ' _lr131';
        else rules += ' _lLPOtrc131';
    }
    rules += ' _Ar131';
    rules += ' _zr131';
    if (props.isQuiet) rules += ' ob131';
    else rules += ' o7E7Gic131';
    if (props.isQuiet) rules += ' nb131';
    else rules += ' n7E7Gic131';
    if (props.isQuiet) rules += ' kb131';
    else rules += ' k7E7Gic131';
    if (props.isQuiet) rules += ' jb131';
    else rules += ' j7E7Gic131';
    rules += ' _9c131';
    rules += ' _Ob131';
    return rules;
};
let $6d4c8b365b6e9016$var$rowWrapper = " sc131";
let $6d4c8b365b6e9016$var$image = " z2U5Crc131 y2U5Crc131 B2U5Crc131 AvoofG131 Ht131 Hsu131 _dc131 oa131 na131 ka131 ja131 ZC131 fa131 _Hb131";
let $6d4c8b365b6e9016$var$actionButtonGroup = " zkG38H131 ykG38H131 BkG38H131 AvoofG131 _dc131 __Ac131 It131 Isu131";
let $6d4c8b365b6e9016$var$listActionMenu = " z6Nrfjc131 y6Nrfjc131 B6Nrfjc131 AvoofG131 _dc131";
const $6d4c8b365b6e9016$var$listCheckbox = function anonymous(props) {
    let rules = " ";
    rules += ' z2CW8rb131';
    rules += ' y2CW8rb131';
    rules += ' B2CW8rb131';
    rules += ' AvoofG131';
    rules += ' _dc131';
    rules += ' Hv131';
    if (props.isDisabled) rules += ' _6b131';
    else rules += ' _6c131';
    return rules;
};
const $6d4c8b365b6e9016$var$listTrailingIcon = " znWsohc131 ynWsohc131 BnWsohc131 AvoofG131 _dc131 sd131 eb131 It131 Isu131";
let $6d4c8b365b6e9016$var$dragButtonContainer = " zdYeQw131 ydYeQw131 BdYeQw131 AvoofG131 _dc131 sd131 eb131 Zi131";
const $6d4c8b365b6e9016$var$centeredWrapper = " sd131 eb131 _Ca131 Za131 Fb131";
const $6d4c8b365b6e9016$var$loadingSpinnerWrapper = " TuNalie131 QuNalie131 SuNalie131 RuNalie131";
const $6d4c8b365b6e9016$var$emptyStateWrapper = " sd131 eb131 _Ca131 Za131 Fb131 _oa131 Tf131 Qf131 Sf131 Rf131";
function $6d4c8b365b6e9016$var$ListSelectionCheckbox({ isDisabled: isDisabled }) {
    let selectionContext = (0, $alyq6$useSlottedContext)((0, $alyq6$CheckboxContext), 'selection');
    let isSelectionDisabled = isDisabled || !!selectionContext?.isDisabled;
    return /*#__PURE__*/ (0, $alyq6$jsx)("div", {
        className: $6d4c8b365b6e9016$var$listCheckbox({
            isDisabled: isSelectionDisabled
        }),
        children: /*#__PURE__*/ (0, $alyq6$jsx)((0, $9b405e0f8095dce0$export$48513f6b9f8ce62d), {
            slot: "selection"
        })
    });
}
function $6d4c8b365b6e9016$export$c6bde0c04b033c0e(props1) {
    let ref = (0, $alyq6$useRef)(null);
    let { hasChildItems: hasChildItems, ...otherProps } = props1;
    let isLink = props1.href != null;
    let isLinkOut = isLink && props1.target === '_blank';
    let { isQuiet: isQuiet, selectionStyle: selectionStyle, overflowMode: overflowMode, scale: scale, hideLinkOutIcon: hideLinkOutIcon = false } = (0, $alyq6$useContext)($6d4c8b365b6e9016$var$InternalListViewContext);
    let textValue = props1.textValue || (typeof props1.children === 'string' ? props1.children : undefined);
    let { direction: direction } = (0, $alyq6$useLocale)();
    let hasTrailingIcon = hasChildItems || isLinkOut && !hideLinkOutIcon;
    return /*#__PURE__*/ (0, $alyq6$jsx)((0, $alyq6$GridListItem), {
        ...otherProps,
        textValue: textValue,
        ref: ref,
        ...hasTrailingIcon ? {
            'data-has-trailing-icon': ''
        } : {},
        className: (renderProps)=>$6d4c8b365b6e9016$var$listitem({
                ...renderProps,
                isLink: isLink,
                isQuiet: isQuiet,
                scale: scale,
                selectionStyle: selectionStyle,
                isPrevNotSelected: !$6d4c8b365b6e9016$var$isPrevSelected(renderProps.id, renderProps.state),
                isNextSelected: $6d4c8b365b6e9016$var$isNextSelected(renderProps.id, renderProps.state),
                isNextNotSelected: !$6d4c8b365b6e9016$var$isNextSelected(renderProps.id, renderProps.state),
                isFirstItem: $6d4c8b365b6e9016$var$isFirstItem(renderProps.id, renderProps.state),
                isLastItem: $6d4c8b365b6e9016$var$isLastItem(renderProps.id, renderProps.state)
            }),
        children: (renderProps)=>{
            let { children: children } = props1;
            let { selectionMode: selectionMode, selectionBehavior: selectionBehavior, isDisabled: isDisabled, id: id, state: state, allowsDragging: allowsDragging, isFocusVisibleWithin: isFocusVisibleWithin } = renderProps;
            return /*#__PURE__*/ (0, $alyq6$jsx)((0, $alyq6$Provider), {
                values: [
                    [
                        (0, $8e847109a6ab556d$export$9afb8bc826b033ea),
                        {
                            slots: {
                                [(0, $alyq6$DEFAULT_SLOT)]: {
                                    styles: (0, $ba354da66aaa1d20$export$1237798dc640739a)({
                                        ...renderProps,
                                        overflowMode: overflowMode
                                    })
                                },
                                label: {
                                    styles: (0, $ba354da66aaa1d20$export$1237798dc640739a)({
                                        ...renderProps,
                                        overflowMode: overflowMode
                                    })
                                },
                                description: {
                                    styles: (0, $ba354da66aaa1d20$export$6d59db4903f20f7d)({
                                        ...renderProps,
                                        overflowMode: overflowMode
                                    })
                                }
                            }
                        }
                    ],
                    [
                        (0, $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed),
                        {
                            slots: {
                                icon: {
                                    render: (0, $1f4b04be3f24aae3$export$9d7e2342a7e53afa)({
                                        slot: 'icon',
                                        styles: (0, $ba354da66aaa1d20$export$31d1842a0fd736e6)
                                    }),
                                    styles: (0, $ba354da66aaa1d20$export$1ca1ec8b29a4ce27)
                                }
                            }
                        }
                    ],
                    [
                        (0, $dbd6f0b2503b938c$export$8151eb379fcf2442),
                        {
                            styles: $6d4c8b365b6e9016$var$image
                        }
                    ],
                    [
                        (0, $10401bdc118cbc90$export$33f5f2f2cb85d743),
                        {
                            styles: $6d4c8b365b6e9016$var$actionButtonGroup,
                            size: 'S',
                            isQuiet: true
                        }
                    ],
                    [
                        (0, $8bf3cd2442eb485e$export$7f4b66acaca28bd5),
                        {
                            styles: $6d4c8b365b6e9016$var$listActionMenu,
                            isQuiet: true,
                            size: 'S',
                            isDisabled: isDisabled
                        }
                    ]
                ],
                children: /*#__PURE__*/ (0, $alyq6$jsxs)("div", {
                    className: $6d4c8b365b6e9016$var$rowWrapper,
                    children: [
                        /*#__PURE__*/ (0, $alyq6$jsx)("div", {
                            className: $6d4c8b365b6e9016$var$listRowBackground({
                                ...renderProps,
                                selectionStyle: selectionStyle,
                                isQuiet: isQuiet,
                                isDropTarget: renderProps.isDropTarget,
                                isPrevSelected: $6d4c8b365b6e9016$var$isPrevSelected(id, state),
                                isNextSelected: $6d4c8b365b6e9016$var$isNextSelected(id, state),
                                isPrevNotSelected: !$6d4c8b365b6e9016$var$isPrevSelected(id, state),
                                isNextNotSelected: !$6d4c8b365b6e9016$var$isNextSelected(id, state),
                                isFirstItem: $6d4c8b365b6e9016$var$isFirstItem(id, state),
                                isLastItem: $6d4c8b365b6e9016$var$isLastItem(id, state)
                            })
                        }),
                        renderProps.isFocusVisible && /*#__PURE__*/ (0, $alyq6$jsx)("div", {
                            className: $6d4c8b365b6e9016$var$listRowFocusRing({
                                ...renderProps,
                                selectionStyle: selectionStyle,
                                isQuiet: isQuiet,
                                isPrevSelected: $6d4c8b365b6e9016$var$isPrevSelected(id, state),
                                isNextSelected: $6d4c8b365b6e9016$var$isNextSelected(id, state),
                                isPrevNotSelected: !$6d4c8b365b6e9016$var$isPrevSelected(id, state),
                                isNextNotSelected: !$6d4c8b365b6e9016$var$isNextSelected(id, state),
                                isFirstItem: $6d4c8b365b6e9016$var$isFirstItem(id, state),
                                isLastItem: $6d4c8b365b6e9016$var$isLastItem(id, state)
                            })
                        }),
                        allowsDragging && /*#__PURE__*/ (0, $alyq6$jsx)("div", {
                            className: $6d4c8b365b6e9016$var$dragButtonContainer,
                            children: !isDisabled && /*#__PURE__*/ (0, $alyq6$jsx)((0, $3c1ab1fd8f31a3e2$export$4c99a519d851f90c), {
                                isFocusVisibleWithin: isFocusVisibleWithin
                            })
                        }),
                        selectionMode !== 'none' && selectionBehavior === 'toggle' && /*#__PURE__*/ (0, $alyq6$jsx)($6d4c8b365b6e9016$var$ListSelectionCheckbox, {
                            isDisabled: isDisabled
                        }),
                        typeof children === 'string' ? /*#__PURE__*/ (0, $alyq6$jsx)((0, $8e847109a6ab556d$export$5f1af8db9871e1d6), {
                            slot: "label",
                            children: children
                        }) : children,
                        isLinkOut && !hideLinkOutIcon && /*#__PURE__*/ (0, $alyq6$jsx)("div", {
                            className: $6d4c8b365b6e9016$var$listTrailingIcon,
                            children: /*#__PURE__*/ (0, $alyq6$jsx)((0, $d9f847a2895a93dc$export$2e2bcd8739ae039), {
                                size: "M",
                                className: function anonymous(props) {
                                    let rules = " ";
                                    if (props.direction === "rtl") {
                                        rules += ' -Z4Pn4c-a131';
                                        rules += ' __Ha131';
                                    }
                                    rules += ' -_8sjo0b-t5ZbAob131';
                                    return rules;
                                }({
                                    direction: direction
                                })
                            })
                        }),
                        hasChildItems && !isLinkOut && /*#__PURE__*/ (0, $alyq6$jsx)("div", {
                            className: $6d4c8b365b6e9016$var$listTrailingIcon,
                            children: /*#__PURE__*/ (0, $alyq6$jsx)((0, $4b69f02ec06b9226$export$2e2bcd8739ae039), {
                                className: function anonymous(props) {
                                    let rules = " ";
                                    if (props.direction === "rtl") {
                                        rules += ' -Z4Pn4c-a131';
                                        rules += ' __Ha131';
                                    } else if (props.direction === "ltr") {
                                        rules += ' -Z4Pn4c-b131';
                                        rules += ' __Ha131';
                                    }
                                    if (props.direction === "rtl") {
                                        rules += ' -_04Pn4c-a131';
                                        rules += ' __Ha131';
                                    } else if (props.direction === "ltr") {
                                        rules += ' -_04Pn4c-b131';
                                        rules += ' __Ha131';
                                    }
                                    rules += ' -_8sjo0b-t5ZbAob131';
                                    return rules;
                                }({
                                    direction: direction
                                })
                            })
                        })
                    ]
                })
            });
        }
    });
}
function $6d4c8b365b6e9016$var$isNextSelected(id, state) {
    if (id == null || !state) return false;
    let keyAfter = state.collection.getKeyAfter(id);
    return keyAfter != null && state.selectionManager.isSelected(keyAfter);
}
function $6d4c8b365b6e9016$var$isPrevSelected(id, state) {
    if (id == null || !state) return false;
    let keyBefore = state.collection.getKeyBefore(id);
    return keyBefore != null && state.selectionManager.isSelected(keyBefore);
}
function $6d4c8b365b6e9016$var$isFirstItem(id, state) {
    if (id == null || !state) return false;
    return state.collection.getFirstKey() === id;
}
function $6d4c8b365b6e9016$var$isLastItem(id, state) {
    if (id == null || !state) return false;
    let key = state.collection.getLastKey();
    let node = key ? state.collection.getItem(key) : null;
    return node ? node.key === id : false;
}


export {$6d4c8b365b6e9016$export$870039b0abfe3de0 as ListViewContext, $6d4c8b365b6e9016$export$6f8e4757b55cd070 as S2ListLayout, $6d4c8b365b6e9016$export$84d0dd190d551cd1 as ListView, $6d4c8b365b6e9016$export$c6bde0c04b033c0e as ListViewItem};
