// @react-spectrum/s2/dist/private/Meter.mjs
import "./Meter.css";
import {FieldLabel as $9afd80978c252040$export$1acdcf5a973a8414} from "./Field.mjs";
import {SkeletonWrapper as $5ad421ec19460c48$export$6069cbe61f690103} from "../../icons/Skeleton.mjs";
import {Text as $8e847109a6ab556d$export$5f1af8db9871e1d6} from "./Content.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $blcZa$jsx, jsxs as $blcZa$jsxs, Fragment as $blcZa$Fragment} from "react/jsx-runtime";
import {Meter as $blcZa$Meter} from "react-aria-components/Meter";
import {createContext as $blcZa$createContext, forwardRef as $blcZa$forwardRef} from "react";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 







const $bb56943300b294c6$export$8b645da15a96b44f = /*#__PURE__*/ (0, $blcZa$createContext)(null);
const $bb56943300b294c6$var$wrapper = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let minWidth = false;
    let maxWidth = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "N") minWidth = true;
        if (p[1] === "L") maxWidth = true;
        rules += p[0];
    }
    if (props.staticColor === "white") rules += ' -Ovbhqd-gcwrnr131';
    else if (props.staticColor === "black") rules += ' -Ovbhqd-g4hn3s131';
    if (!position) rules += ' _Pc131';
    rules += ' se131';
    if (props.labelPosition === "side") rules += ' DPlN5qb131';
    else if (props.labelPosition === "top") rules += ' Dxumf9b131';
    if (props.labelPosition === "side") rules += ' Cvv4gue131';
    else if (props.labelPosition === "top") rules += ' CC0Gyo131';
    rules += ' ea131';
    rules += ' _Bb131';
    if (!minWidth) rules += ' Nl131';
    if (!maxWidth) rules += ' Lk131';
    if (props.size === "XL") rules += ' -BhX7R-Fz131';
    else if (props.size === "L") rules += ' -BhX7R-Fv131';
    else if (props.size === "S") rules += ' -BhX7R-Fp131';
    else if (props.size === "XS") rules += ' -BhX7R-Fn131';
    else rules += ' -BhX7R-Fx131';
    rules += ' -dTMmze-Fe131';
    rules += ' -AJjOLd-UCwrhQe131';
    rules += ' qg131';
    return rules;
};
const $bb56943300b294c6$var$valueStyles = function anonymous(props) {
    let rules = " ";
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isStaticColor) rules += ' pl131';
    else if (props.isDisabled) rules += ' pp131';
    else rules += ' pv131';
    rules += ' plb131';
    rules += ' ri131';
    rules += ' z8bmXs131';
    rules += ' y8bmXs131';
    rules += ' B8bmXs131';
    rules += ' A8bmXs131';
    return rules;
};
const $bb56943300b294c6$var$trackStyles = function anonymous(props) {
    let rules = " ";
    rules += ' zMY0fn131';
    rules += ' yMY0fn131';
    rules += ' BMY0fn131';
    rules += ' AMY0fn131';
    if (props.size === "XL") rules += ' Nl131';
    else if (props.size === "L") rules += ' Nj131';
    else if (props.size === "S") rules += ' Nf131';
    else if (props.size === "XS") rules += ' Ne131';
    else rules += ' Nk131';
    if (props.isQuiet) rules += ' _qd131';
    else rules += ' _qtb131';
    if (props.size === "XL") rules += ' -Anxdcc-Zt131';
    else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
    else if (props.size === "S") rules += ' -Anxdcc-Zr131';
    else rules += ' -Anxdcc-Z3fn9H131';
    rules += ' __v6TNLWb131';
    rules += ' _Nc131';
    rules += ' Pc131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    if (props.isStaticColor) rules += ' gh131';
    else rules += ' gE131';
    rules += ' glb131';
    rules += ' _Ma131';
    rules += ' _Mlb131';
    rules += ' _Le131';
    rules += ' _Llf131';
    rules += ' Oj131';
    rules += ' Olb131';
    rules += ' _9c131';
    if (props.size === "XL") rules += ' Fj131';
    else if (props.size === "L") rules += ' Fi131';
    else if (props.size === "M") rules += ' Fg131';
    else if (props.size === "S") rules += ' Fe131';
    else rules += ' Fg131';
    return rules;
};
const $bb56943300b294c6$var$fillStyles = function anonymous(props) {
    let rules = " ";
    rules += ' Fb131';
    rules += ' _je131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    if (props.isStaticColor) rules += ' gj131';
    else if (props.variant === "negative") rules += ' gFLVMS131';
    else if (props.variant === "notice") rules += ' g9VrKkd131';
    else if (props.variant === "positive") rules += ' gp17Jm131';
    else rules += ' gG1ze4d131';
    rules += ' glc131';
    return rules;
};
const $bb56943300b294c6$export$62e3ae2a4090b879 = /*#__PURE__*/ (0, $blcZa$forwardRef)(function Meter(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $bb56943300b294c6$export$8b645da15a96b44f);
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let { label: label, size: size = 'M', staticColor: staticColor, styles: styles, UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, variant: variant = 'informative', labelPosition: labelPosition = 'top', ...groupProps } = props;
    let isStaticColor = !!staticColor;
    return /*#__PURE__*/ (0, $blcZa$jsx)((0, $blcZa$Meter), {
        ...groupProps,
        ref: domRef,
        style: UNSAFE_style,
        className: UNSAFE_className + $bb56943300b294c6$var$wrapper({
            size: size,
            variant: variant,
            staticColor: staticColor,
            labelPosition: labelPosition
        }, styles),
        children: ({ percentage: percentage, valueText: valueText })=>/*#__PURE__*/ (0, $blcZa$jsxs)((0, $blcZa$Fragment), {
                children: [
                    label && /*#__PURE__*/ (0, $blcZa$jsx)((0, $9afd80978c252040$export$1acdcf5a973a8414), {
                        size: size,
                        labelAlign: "start",
                        labelPosition: labelPosition,
                        staticColor: staticColor,
                        children: label
                    }),
                    label && /*#__PURE__*/ (0, $blcZa$jsx)((0, $8e847109a6ab556d$export$5f1af8db9871e1d6), {
                        styles: $bb56943300b294c6$var$valueStyles({
                            size: size,
                            labelAlign: 'end',
                            isStaticColor: isStaticColor
                        }),
                        children: valueText
                    }),
                    /*#__PURE__*/ (0, $blcZa$jsx)((0, $5ad421ec19460c48$export$6069cbe61f690103), {
                        children: /*#__PURE__*/ (0, $blcZa$jsx)("div", {
                            className: $bb56943300b294c6$var$trackStyles({
                                isStaticColor: isStaticColor,
                                size: size
                            }),
                            children: /*#__PURE__*/ (0, $blcZa$jsx)("div", {
                                className: $bb56943300b294c6$var$fillStyles({
                                    isStaticColor: isStaticColor,
                                    variant: variant
                                }),
                                style: {
                                    width: percentage + '%'
                                }
                            })
                        })
                    })
                ]
            })
    });
});


export {$bb56943300b294c6$export$8b645da15a96b44f as MeterContext, $bb56943300b294c6$export$62e3ae2a4090b879 as Meter};
