// @react-spectrum/s2/dist/private/Modal.cjs
require("./Modal.css");
var $914a06b3a1c679bc$exports = require("./Provider.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $j6TWk$reactjsxruntime = require("react/jsx-runtime");
var $j6TWk$react = require("react");
var $j6TWk$reactariacomponentsModal = require("react-aria-components/Modal");
var $j6TWk$reactariaI18nProvider = require("react-aria/I18nProvider");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "Modal", () => $8e678305a8c10028$export$2b77a92f1a5ad772);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





const $8e678305a8c10028$var$modalOverlayStyles = function anonymous(props) {
    let rules = " ";
    if (props.colorScheme === "dark") rules += ' -ICUGx-_pa131';
    else if (props.colorScheme === "light") rules += ' -ICUGx-_pb131';
    else if (props.colorScheme === "light dark") rules += ' -ICUGx-_pc131';
    rules += ' _pK8eoyd131';
    if (props.colorScheme === "dark") rules += ' -NvXmrd-_ZLmU131';
    else if (props.colorScheme === "light") rules += ' -NvXmrd-_ZN8Gi4c131';
    else if (props.colorScheme === "light dark") {
        rules += ' -NvXmrd-_ZN8Gi4c131';
        rules += ' -NvXmrd-D1v0V_ZLmU131';
    }
    if (props.colorScheme === "dark") rules += ' -j9blY-_ZN8Gi4c131';
    else if (props.colorScheme === "light") rules += ' -j9blY-_ZLmU131';
    else if (props.colorScheme === "light dark") {
        rules += ' -j9blY-_ZLmU131';
        rules += ' -j9blY-D1v0V_ZN8Gi4c131';
    }
    rules += ' _Pa131';
    rules += ' Wr131';
    rules += ' _Er131';
    rules += ' Za131';
    rules += ' FKem6Ob131';
    rules += ' _Bb131';
    rules += ' g8131';
    if (props.isExiting) rules += ' _Ia131';
    else if (props.isEntering) rules += ' _Ia131';
    rules += ' Yf131';
    if (props.isExiting) rules += ' Xa131';
    else rules += ' Xd131';
    rules += ' _2b131';
    return rules;
};
const $8e678305a8c10028$var$modalWrapper = function anonymous(props) {
    let rules = " ";
    rules += ' _Pe131';
    rules += ' Wr131';
    rules += ' _Er131';
    rules += ' Za131';
    rules += ' FE9bXU131';
    rules += ' sd131';
    if (props.size === "fullscreenTakeover") rules += ' ed131';
    else rules += ' eb131';
    rules += ' _Ca131';
    return rules;
};
const $8e678305a8c10028$export$2b77a92f1a5ad772 = /*#__PURE__*/ (0, $j6TWk$react.forwardRef)(function Modal(props1, ref) {
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let colorScheme = (0, $j6TWk$react.useContext)((0, $914a06b3a1c679bc$exports.ColorSchemeContext));
    let { locale: locale, direction: direction } = (0, $j6TWk$reactariaI18nProvider.useLocale)();
    // TODO: should we pass through lang and dir props in RAC?
    let modalRef = (0, $j6TWk$react.useCallback)((el)=>{
        domRef.current = el;
        if (el) {
            el.lang = locale;
            el.dir = direction;
        }
    }, [
        locale,
        direction,
        domRef
    ]);
    return /*#__PURE__*/ (0, $j6TWk$reactjsxruntime.jsx)((0, $j6TWk$reactariacomponentsModal.ModalOverlay), {
        ...props1,
        className: (renderProps)=>$8e678305a8c10028$var$modalOverlayStyles({
                ...renderProps,
                colorScheme: colorScheme
            }),
        children: /*#__PURE__*/ (0, $j6TWk$reactjsxruntime.jsx)("div", {
            className: $8e678305a8c10028$var$modalWrapper({
                size: props1.size
            }),
            style: {
                containerType: 'size'
            },
            children: /*#__PURE__*/ (0, $j6TWk$reactjsxruntime.jsx)((0, $j6TWk$reactariacomponentsModal.Modal), {
                ...props1,
                ref: modalRef,
                className: (renderProps)=>(function anonymous(props) {
                        let rules = " ";
                        rules += ' sd131';
                        rules += ' _ta131';
                        if (props.size === "fullscreenTakeover") rules += ' od131';
                        else rules += ' oe131';
                        if (props.size === "fullscreenTakeover") rules += ' nd131';
                        else rules += ' ne131';
                        if (props.size === "fullscreenTakeover") rules += ' kd131';
                        else rules += ' ke131';
                        if (props.size === "fullscreenTakeover") rules += ' jd131';
                        else rules += ' je131';
                        if (props.size === "fullscreenTakeover") rules += ' Za131';
                        else if (props.size === "fullscreen") rules += ' ZGqsG7b131';
                        else if (props.size === "XL") rules += ' ZbRWaYd131';
                        else if (props.size === "L") rules += ' Zb4QY4131';
                        else if (props.size === "M") rules += ' ZILqIYb131';
                        else if (props.size === "S") rules += ' ZA131';
                        if (props.size === "fullscreenTakeover") rules += ' Fb131';
                        else if (props.size === "fullscreen") rules += ' FGqsG7b131';
                        if (props.size === "fullscreenTakeover") rules += ' Lp131';
                        else if (props.size === "fullscreen") rules += ' Lp131';
                        else rules += ' Lc131';
                        if (props.size === "fullscreenTakeover") rules += ' Ki131';
                        else if (props.size === "fullscreen") rules += ' Ki131';
                        else rules += ' KjVOfn131';
                        if (props.size === "fullscreenTakeover") rules += ' QILlin131';
                        rules += ' -Ovbhqd-g_g131';
                        rules += ' gEzkoQe131';
                        if (props.isExiting) rules += ' _Ia131';
                        else if (props.isEntering) rules += ' _Ia131';
                        if (props.isEntering) {
                            rules += ' -_8PloMd-w131';
                            rules += ' __Ya131';
                        }
                        rules += ' YmenWad131';
                        if (props.isExiting) rules += ' Xa131';
                        else rules += ' Xd131';
                        rules += ' _2b131';
                        if (props.isExiting) rules += ' _1a131';
                        else rules += ' _1b131';
                        rules += ' _Lf131';
                        rules += ' _Mb131';
                        rules += ' Oj131';
                        return rules;
                    })({
                        ...renderProps,
                        size: props1.size
                    })
            })
        })
    });
});
