// @react-spectrum/s2/dist/private/NotificationBadge.cjs
require("./NotificationBadge.css");
var $4526404114e78c80$exports = require("./intlStrings.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $62rny$reactjsxruntime = require("react/jsx-runtime");
var $62rny$reactariafilterDOMProps = require("react-aria/filterDOMProps");
var $62rny$internationalizednumber = require("@internationalized/number");
var $62rny$react = require("react");
var $62rny$reactariaI18nProvider = require("react-aria/I18nProvider");
var $62rny$reactariauseLocalizedStringFormatter = require("react-aria/useLocalizedStringFormatter");


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "NotificationBadgeContext", () => $cd7301fb33ba8690$export$9ab75c96ebaa396c);
$parcel$export(module.exports, "NotificationBadge", () => $cd7301fb33ba8690$export$529cab0ca6be204e);
/*
 * Copyright 2025 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 








const $cd7301fb33ba8690$export$9ab75c96ebaa396c = /*#__PURE__*/ (0, $62rny$react.createContext)(null);
const $cd7301fb33ba8690$var$badge = function anonymous(props, overrides) {
    let rules = " ";
    let width = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "Z") width = true;
        rules += p[0];
    }
    if (props.isDisabled) rules += ' sk131';
    else rules += ' sd131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "M") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    }
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    if (props.isStaticColor) rules += ' ph131';
    else rules += ' px131';
    rules += ' plb131';
    rules += ' _jlf131';
    rules += ' _klb131';
    rules += ' hlb131';
    rules += ' mlb131';
    rules += ' llb131';
    rules += ' ila131';
    rules += ' _Ca131';
    rules += ' eb131';
    rules += ' gqMFGYc131';
    rules += ' FRuFGYc131';
    if (props.isSingleDigit) rules += ' fa131';
    else if (props.isIndicatorOnly) rules += ' fa131';
    if (!width) rules += ' ZK131';
    if (props.isDoubleDigit) rules += ' Sv131';
    if (props.isDoubleDigit) rules += ' Rv131';
    rules += ' oh131';
    rules += ' nh131';
    rules += ' kh131';
    rules += ' jh131';
    if (props.isStaticColor) rules += ' -g_-g_c131';
    else rules += ' -g_-g2131';
    rules += ' -g_-lgb131';
    if (props.size === "XL") {
        if (props.isIndicatorOnly) rules += ' -F_-Fspvr8c131';
        else rules += ' -F_-Fm131';
    } else if (props.size === "L") {
        if (props.isIndicatorOnly) rules += ' -F_-Fspvr8c131';
        else rules += ' -F_-Ft131';
    } else if (props.size === "M") {
        if (props.isIndicatorOnly) rules += ' -F_-Fi131';
        else rules += ' -F_-FLb8KJ131';
    } else if (props.size === "S") {
        if (props.isIndicatorOnly) rules += ' -F_-Fi131';
        else rules += ' -F_-Fk131';
    }
    return rules;
};
const $cd7301fb33ba8690$export$529cab0ca6be204e = /*#__PURE__*/ (0, $62rny$react.forwardRef)(function Badge(props, ref) {
    let stringFormatter = (0, $62rny$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $cd7301fb33ba8690$export$9ab75c96ebaa396c);
    let { size: size = 'S', value: value, isDisabled: isDisabled = false, staticColor: staticColor, ...otherProps } = props;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let { locale: locale } = (0, $62rny$reactariaI18nProvider.useLocale)();
    let formattedValue = '';
    let isIndicatorOnly = false;
    let isSingleDigit = false;
    let isDoubleDigit = false;
    if (value == null) isIndicatorOnly = true;
    else if (value <= 0) throw new Error('Value cannot be negative or zero');
    else if (!Number.isInteger(value)) throw new Error('Value must be a positive integer');
    else {
        formattedValue = new (0, $62rny$internationalizednumber.NumberFormatter)(locale).format(Math.min(value, 99));
        let length = Math.log(value <= 99 ? value : 99) * Math.LOG10E + 1 | 0; // for positive integers (https://stackoverflow.com/questions/14879691/get-number-of-digits-with-javascript)
        if (length === 1) isSingleDigit = true;
        else if (length === 2) isDoubleDigit = true;
        if (value > 99) formattedValue = stringFormatter.format('notificationbadge.plus', {
            notifications: formattedValue
        });
    }
    let ariaLabel = props['aria-label'] || undefined;
    if (ariaLabel === undefined && isIndicatorOnly) ariaLabel = stringFormatter.format('notificationbadge.indicatorOnly');
    return /*#__PURE__*/ (0, $62rny$reactjsxruntime.jsx)("span", {
        ...(0, $62rny$reactariafilterDOMProps.filterDOMProps)(otherProps, {
            labelable: true
        }),
        role: ariaLabel && 'img',
        "aria-label": ariaLabel,
        className: (props.UNSAFE_className || '') + $cd7301fb33ba8690$var$badge({
            size: size,
            isIndicatorOnly: isIndicatorOnly,
            isSingleDigit: isSingleDigit,
            isDoubleDigit: isDoubleDigit,
            isDisabled: isDisabled,
            isStaticColor: !!staticColor
        }, props.styles),
        style: props.UNSAFE_style,
        ref: domRef,
        children: formattedValue
    });
});
