// @react-spectrum/s2/dist/private/NumberField.cjs
require("./NumberField.css");
var $54686c7c3f031ee2$exports = require("./Add.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $84c37b7bd5c1374e$exports = require("./Dash.cjs");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $bfa4962d90c8af48$exports = require("./Form.cjs");
var $2061c83559b50a66$exports = require("./pressScale.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $fwzUK$reactjsxruntime = require("react/jsx-runtime");
var $fwzUK$reactariacomponentsButton = require("react-aria-components/Button");
var $fwzUK$reactariacomponentsNumberField = require("react-aria-components/NumberField");
var $fwzUK$reactariacomponentsslots = require("react-aria-components/slots");
var $fwzUK$react = require("react");
var $fwzUK$reactariafilterDOMProps = require("react-aria/filterDOMProps");
var $fwzUK$reactariacomponentsInput = require("react-aria-components/Input");
var $fwzUK$reactariamergeProps = require("react-aria/mergeProps");
var $fwzUK$reactariamergeRefs = require("react-aria/mergeRefs");
var $fwzUK$reactariauseButton = require("react-aria/useButton");
var $fwzUK$reactariauseFocusRing = require("react-aria/useFocusRing");
var $fwzUK$reactariauseHover = require("react-aria/useHover");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "NumberFieldContext", () => $a366ef8f467b1cb5$export$b414a48cf5dcbc11);
$parcel$export(module.exports, "NumberField", () => $a366ef8f467b1cb5$export$63c5fa0b2fdccd2e);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 


















const $a366ef8f467b1cb5$export$b414a48cf5dcbc11 = /*#__PURE__*/ (0, $fwzUK$react.createContext)(null);
const $a366ef8f467b1cb5$var$inputButton = function anonymous(props) {
    let rules = " ";
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-oa131';
    rules += ' oj131';
    rules += ' nj131';
    rules += ' kj131';
    rules += ' jj131';
    rules += ' sd131';
    rules += ' _Le131';
    rules += ' _Wa131';
    rules += ' _je131';
    rules += ' eb131';
    rules += ' _Ca131';
    if (props.size === "XL") rules += ' ZC131';
    else if (props.size === "L") rules += ' Zo131';
    else if (props.size === "M") rules += ' Zm131';
    else if (props.size === "S") rules += ' Zu131';
    rules += ' Fd131';
    if (props.type === "increment") rules += ' Iy131';
    else rules += ' Iw131';
    rules += ' fa131';
    rules += ' _va131';
    rules += ' Ma131';
    rules += ' Yd131';
    rules += ' Yle131';
    rules += ' Xb131';
    rules += ' Xlb131';
    rules += ' _2b131';
    rules += ' _2lb131';
    if (props.isDisabled) rules += ' gH131';
    else {
        if (props.isPressed) rules += ' gF131';
        else if (props.isFocusVisible) rules += ' gF131';
        else if (props.isHovered) rules += ' gF131';
        else rules += ' gH131';
    }
    if (props.isDisabled) rules += ' gld131';
    else if (props.isHovered) rules += ' gle131';
    else rules += ' glc131';
    if (props.isDisabled) rules += ' pp131';
    else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    }
    rules += ' pla131';
    rules += ' ri131';
    return rules;
};
const $a366ef8f467b1cb5$var$iconStyles = " _va131 -_8sjo0b-t5ZbAob131";
const $a366ef8f467b1cb5$var$stepperContainerStyles = function anonymous(props) {
    let rules = " ";
    rules += ' sd131';
    rules += ' _tc131';
    if (props.size === "XL") rules += ' Ue131';
    else if (props.size === "L") rules += ' Ue131';
    else if (props.size === "M") rules += ' Ub131';
    else if (props.size === "S") rules += ' Ue131';
    if (props.size === "XL") rules += ' qe131';
    else if (props.size === "L") rules += ' qe131';
    else if (props.size === "M") rules += ' qb131';
    else if (props.size === "S") rules += ' qe131';
    if (props.size === "XL") rules += ' HWJpoB131';
    else if (props.size === "L") rules += ' HWJpoB131';
    else if (props.size === "M") rules += ' Hs131';
    else if (props.size === "S") rules += ' Hr131';
    return rules;
};
const $a366ef8f467b1cb5$export$63c5fa0b2fdccd2e = /*#__PURE__*/ (0, $fwzUK$react.forwardRef)(function NumberField(props1, ref) {
    [props1, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props1, ref, $a366ef8f467b1cb5$export$b414a48cf5dcbc11);
    props1 = (0, $bfa4962d90c8af48$exports.useFormProps)(props1);
    let { label: label, contextualHelp: contextualHelp, description: descriptionMessage, errorMessage: errorMessage, hideStepper: hideStepper, isRequired: isRequired, size: size = 'M', labelPosition: labelPosition = 'top', necessityIndicator: necessityIndicator, labelAlign: labelAlign = 'start', UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, ...numberFieldProps } = props1;
    let formContext = (0, $fwzUK$react.useContext)((0, $bfa4962d90c8af48$exports.FormContext));
    let domRef = (0, $fwzUK$react.useRef)(null);
    let inputRef = (0, $fwzUK$react.useRef)(null);
    let decrementButtonRef = (0, $fwzUK$react.useRef)(null);
    let incrementButtonRef = (0, $fwzUK$react.useRef)(null);
    // Expose imperative interface for ref
    (0, $fwzUK$react.useImperativeHandle)(ref, ()=>({
            ...(0, $150382364eecc4bc$exports.createFocusableRef)(domRef, inputRef),
            select () {
                if (inputRef.current) inputRef.current.select();
            },
            getInputElement () {
                return inputRef.current;
            }
        }));
    return /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsx)((0, $fwzUK$reactariacomponentsNumberField.NumberField), {
        ref: domRef,
        isRequired: isRequired,
        ...numberFieldProps,
        style: UNSAFE_style,
        className: UNSAFE_className + function anonymous(props, overrides) {
            let rules = " ";
            let gridColumnStart = false;
            let gridColumnEnd = false;
            let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
            for (let p of matches){
                if (p[1] === "z") gridColumnStart = true;
                if (p[1] === "y") gridColumnEnd = true;
                rules += p[0];
            }
            rules += ' se131';
            if (props.isInForm) {
                if (!gridColumnStart) rules += ' z2mU131';
            }
            if (props.isInForm) {
                if (props.labelPosition === "side") {
                    if (!gridColumnEnd) rules += ' yvoofG131';
                }
            }
            if (props.isInForm) rules += ' DLwlgod131';
            else {
                if (props.labelPosition === "side") rules += ' DPlN5qb131';
                else if (props.labelPosition === "top") rules += ' DM8Mfn131';
            }
            if (props.labelPosition === "side") rules += ' EPlN5qb131';
            else if (props.labelPosition === "top") rules += ' E4w1sLc131';
            if (props.labelPosition === "side") rules += ' CbRM95c131';
            else if (props.labelPosition === "top") rules += ' Cxaocre131';
            if (props.size === "XL") {
                rules += ' -_6BNtrc-e131';
                rules += ' vx131';
            } else if (props.size === "L") {
                rules += ' -_6BNtrc-d131';
                rules += ' vx131';
            } else if (props.size === "S") {
                rules += ' -_6BNtrc-b131';
                rules += ' vx131';
            } else if (props.size === "XS") {
                rules += ' -_6BNtrc-a131';
                rules += ' vx131';
            } else {
                rules += ' -_6BNtrc-c131';
                rules += ' vx131';
            }
            rules += ' ea131';
            rules += ' _Fd131';
            rules += ' _FnuYUweb131';
            if (props.size === "XL") rules += ' -BhX7R-Fz131';
            else if (props.size === "L") rules += ' -BhX7R-Fv131';
            else if (props.size === "S") rules += ' -BhX7R-Fp131';
            else if (props.size === "XS") rules += ' -BhX7R-Fn131';
            else rules += ' -BhX7R-Fx131';
            rules += ' -AJjOLd-UVkF0Db131';
            rules += ' qg131';
            rules += ' __ca131';
            return rules;
        }({
            isInForm: !!formContext,
            labelPosition: labelPosition,
            size: size
        }, props1.styles),
        children: ({ isDisabled: isDisabled, isInvalid: isInvalid })=>{
            return /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsxs)((0, $fwzUK$reactjsxruntime.Fragment), {
                children: [
                    /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldLabel), {
                        isDisabled: isDisabled,
                        isRequired: isRequired,
                        size: size,
                        labelPosition: labelPosition,
                        labelAlign: labelAlign,
                        necessityIndicator: necessityIndicator,
                        contextualHelp: contextualHelp,
                        children: label
                    }),
                    /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsxs)((0, $a0b996a550bb0891$exports.FieldGroup), {
                        prefix: props1.prefix,
                        size: size,
                        styles: function anonymous(props) {
                            let rules = " ";
                            rules += ' zXcEWr131';
                            rules += ' yXcEWr131';
                            rules += ' BXcEWr131';
                            rules += ' AXcEWr131';
                            if (props.size === "XL") rules += ' Nl131';
                            else if (props.size === "L") rules += ' Nj131';
                            else if (props.size === "S") rules += ' Nf131';
                            else if (props.size === "XS") rules += ' Ne131';
                            else rules += ' Nk131';
                            if (props.isQuiet) rules += ' _qd131';
                            else rules += ' _qtb131';
                            if (props.size === "XL") rules += ' -Anxdcc-Zt131';
                            else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
                            else if (props.size === "S") rules += ' -Anxdcc-Zr131';
                            else rules += ' -Anxdcc-Z3fn9H131';
                            rules += ' __v6TNLWb131';
                            rules += ' SMBFGYc131';
                            rules += ' RfBFGYc131';
                            rules += ' -S_-Sv131';
                            if (props.isStepperHidden) rules += ' -R_-Rv131';
                            else rules += ' -R_-Rd131';
                            return rules;
                        }({
                            size: size,
                            isStepperHidden: hideStepper
                        }),
                        children: [
                            /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsx)((0, $fwzUK$reactariacomponentsInput.InputContext).Consumer, {
                                children: (ctx)=>/*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsx)((0, $fwzUK$reactariacomponentsInput.InputContext).Provider, {
                                        value: {
                                            ...ctx,
                                            ref: (0, $fwzUK$reactariamergeRefs.mergeRefs)(ctx?.ref, inputRef)
                                        },
                                        children: /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.Input), {})
                                    })
                            }),
                            isInvalid && /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldErrorIcon), {
                                isDisabled: isDisabled
                            }),
                            !hideStepper && /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsxs)("div", {
                                className: $a366ef8f467b1cb5$var$stepperContainerStyles({
                                    size: size
                                }),
                                children: [
                                    /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsx)($a366ef8f467b1cb5$var$StepButton, {
                                        ref: decrementButtonRef,
                                        slot: "decrement",
                                        style: (renderProps)=>(0, $2061c83559b50a66$exports.pressScale)(decrementButtonRef)(renderProps),
                                        className: (renderProps)=>$a366ef8f467b1cb5$var$inputButton({
                                                ...renderProps,
                                                type: 'decrement',
                                                size: size
                                            }),
                                        children: /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsx)((0, $84c37b7bd5c1374e$exports.default), {
                                            size: size,
                                            className: $a366ef8f467b1cb5$var$iconStyles
                                        })
                                    }),
                                    /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsx)($a366ef8f467b1cb5$var$StepButton, {
                                        ref: incrementButtonRef,
                                        slot: "increment",
                                        style: (renderProps)=>(0, $2061c83559b50a66$exports.pressScale)(incrementButtonRef)(renderProps),
                                        className: (renderProps)=>$a366ef8f467b1cb5$var$inputButton({
                                                ...renderProps,
                                                type: 'increment',
                                                size: size
                                            }),
                                        children: /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsx)((0, $54686c7c3f031ee2$exports.default), {
                                            size: size,
                                            className: $a366ef8f467b1cb5$var$iconStyles
                                        })
                                    })
                                ]
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.HelpText), {
                        size: size,
                        isDisabled: isDisabled,
                        isInvalid: isInvalid,
                        description: descriptionMessage,
                        children: errorMessage
                    })
                ]
            });
        }
    });
});
const $a366ef8f467b1cb5$var$additionalButtonHTMLAttributes = new Set([
    'form',
    'formAction',
    'formEncType',
    'formMethod',
    'formNoValidate',
    'formTarget',
    'name',
    'value'
]);
let $a366ef8f467b1cb5$var$StepButton = /*#__PURE__*/ (0, $fwzUK$react.forwardRef)((props, ref)=>{
    [props, ref] = (0, $fwzUK$reactariacomponentsslots.useContextProps)(props, ref, (0, $fwzUK$reactariacomponentsButton.ButtonContext));
    let ctx = props;
    /**
   * Must use div for now because Safari pointer event bugs on disabled form elements.
   * Link https://bugs.webkit.org/show_bug.cgi?id=219188.
   */ let { buttonProps: buttonProps, isPressed: isPressed } = (0, $fwzUK$reactariauseButton.useButton)({
        ...props,
        elementType: 'div'
    }, ref);
    let { focusProps: focusProps, isFocused: isFocused, isFocusVisible: isFocusVisible } = (0, $fwzUK$reactariauseFocusRing.useFocusRing)(props);
    let { hoverProps: hoverProps, isHovered: isHovered } = (0, $fwzUK$reactariauseHover.useHover)(props);
    let renderProps = $a366ef8f467b1cb5$var$useRenderProps({
        ...props,
        values: {
            isHovered: isHovered,
            isPressed: isPressed,
            isFocused: isFocused,
            isFocusVisible: isFocusVisible,
            isDisabled: props.isDisabled || false
        },
        defaultClassName: 'react-aria-Button'
    });
    return /*#__PURE__*/ (0, $fwzUK$reactjsxruntime.jsx)("div", {
        ...(0, $fwzUK$reactariafilterDOMProps.filterDOMProps)(props, {
            propNames: $a366ef8f467b1cb5$var$additionalButtonHTMLAttributes
        }),
        ...(0, $fwzUK$reactariamergeProps.mergeProps)(buttonProps, focusProps, hoverProps),
        ...renderProps,
        ref: ref,
        slot: props.slot || undefined,
        "data-disabled": props.isDisabled || undefined,
        "data-pressed": ctx.isPressed || isPressed || undefined,
        "data-hovered": isHovered || undefined,
        "data-focused": isFocused || undefined,
        "data-focus-visible": isFocusVisible || undefined
    });
});
// replace from RAC
function $a366ef8f467b1cb5$var$useRenderProps(props) {
    let { className: className, style: style, children: children, defaultClassName: defaultClassName, defaultChildren: defaultChildren, defaultStyle: defaultStyle, values: values } = props;
    return (0, $fwzUK$react.useMemo)(()=>{
        let computedClassName;
        let computedStyle;
        let computedChildren;
        if (typeof className === 'function') computedClassName = className({
            ...values,
            defaultClassName: defaultClassName
        });
        else computedClassName = className;
        if (typeof style === 'function') computedStyle = style({
            ...values,
            defaultStyle: defaultStyle || {}
        });
        else computedStyle = style;
        if (typeof children === 'function') computedChildren = children({
            ...values,
            defaultChildren: defaultChildren
        });
        else if (children == null) computedChildren = defaultChildren;
        else computedChildren = children;
        return {
            className: computedClassName ?? defaultClassName,
            style: computedStyle || defaultStyle ? {
                ...defaultStyle,
                ...computedStyle
            } : undefined,
            children: computedChildren ?? defaultChildren,
            'data-rac': ''
        };
    }, [
        className,
        style,
        children,
        defaultClassName,
        defaultChildren,
        defaultStyle,
        values
    ]);
}
