// @react-spectrum/s2/dist/private/Picker.cjs
require("./Picker.css");
var $38071d9ce246d4cf$exports = require("./Avatar.cjs");
var $af04e099a53d3e85$exports = require("./Checkbox.cjs");
var $e991cbcdf82ced71$exports = require("./CenterBaseline.cjs");
var $e741ea6b88ce4866$exports = require("./Menu.cjs");
var $dba2b18fc5727f9b$exports = require("./Checkmark.cjs");
var $1df9f1c9262ce5df$exports = require("./Chevron.cjs");
var $a2d5b35c01bebcd6$exports = require("./ComboBox.cjs");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $bfa4962d90c8af48$exports = require("./Form.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $4526404114e78c80$exports = require("./intlStrings.cjs");
var $308b180f49d82d28$exports = require("../../icons/runtime.cjs");
var $493371ef39bf7a55$exports = require("./Popover.cjs");
var $2061c83559b50a66$exports = require("./pressScale.cjs");
var $1c116ea4b0d2d023$exports = require("./ProgressCircle.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ee7b4c497f520c08$exports = require("./utils.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $ceqeb$reactjsxruntime = require("react/jsx-runtime");
var $ceqeb$reactariacomponentsListBox = require("react-aria-components/ListBox");
var $ceqeb$reactariacomponentsSelect = require("react-aria-components/Select");
var $ceqeb$reactariacomponentsButton = require("react-aria-components/Button");
var $ceqeb$reactariaCollection = require("react-aria/Collection");
var $ceqeb$reactariacomponentsslots = require("react-aria-components/slots");
var $ceqeb$reactariaprivatecollectionsHidden = require("react-aria/private/collections/Hidden");
var $ceqeb$reactariaprivateutilsshadowdomShadowTreeWalker = require("react-aria/private/utils/shadowdom/ShadowTreeWalker");
var $ceqeb$reactariaprivateutilsdomHelpers = require("react-aria/private/utils/domHelpers");
var $ceqeb$reactariaprivateutilsisFocusable = require("react-aria/private/utils/isFocusable");
var $ceqeb$reactstatelyuseVirtualizerState = require("react-stately/useVirtualizerState");
var $ceqeb$reactariaprivateinteractionsPressResponder = require("react-aria/private/interactions/PressResponder");
var $ceqeb$react = require("react");
var $ceqeb$reactariaprivateutilsuseGlobalListeners = require("react-aria/private/utils/useGlobalListeners");
var $ceqeb$reactariaI18nProvider = require("react-aria/I18nProvider");
var $ceqeb$reactariauseLocalizedStringFormatter = require("react-aria/useLocalizedStringFormatter");
var $ceqeb$reactariaprivateutilsuseId = require("react-aria/private/utils/useId");
var $ceqeb$reactariacomponentsVirtualizer = require("react-aria-components/Virtualizer");


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "PickerContext", () => $b9c30139cd63c62a$export$b72b8dade9393cd5);
$parcel$export(module.exports, "Picker", () => $b9c30139cd63c62a$export$ba25329847403e11);
$parcel$export(module.exports, "PickerItem", () => $b9c30139cd63c62a$export$d601881f38163e28);
$parcel$export(module.exports, "PickerSection", () => $b9c30139cd63c62a$export$a340772aca6155cd);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 




































const $b9c30139cd63c62a$export$b72b8dade9393cd5 = /*#__PURE__*/ (0, $ceqeb$react.createContext)(null);
const $b9c30139cd63c62a$var$inputButton = function anonymous(props) {
    let rules = " ";
    if (props.isQuiet) rules += ' _Le131';
    else if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isDisabled) rules += ' pp131';
    else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    }
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _oa131';
    rules += ' SMBFGYc131';
    rules += ' RfBFGYc131';
    if (props.size === "XL") rules += ' Nl131';
    else if (props.size === "L") rules += ' Nj131';
    else if (props.size === "S") rules += ' Nf131';
    else if (props.size === "XS") rules += ' Ne131';
    else rules += ' Nk131';
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-ob131';
    rules += ' oj131';
    rules += ' nj131';
    rules += ' kj131';
    rules += ' jj131';
    rules += ' qc131';
    rules += ' qsd131';
    rules += ' -Qg41Lb-IAxCW0c131';
    rules += ' -Qg41Lb-gLWx3bIy131';
    rules += ' FRuFGYc131';
    rules += ' zXcEWr131';
    rules += ' yXcEWr131';
    rules += ' BXcEWr131';
    rules += ' AXcEWr131';
    if (props.isQuiet) rules += ' _qd131';
    else rules += ' _qtb131';
    if (props.size === "XL") rules += ' -Anxdcc-Zt131';
    else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
    else if (props.size === "S") rules += ' -Anxdcc-Zr131';
    else rules += ' -Anxdcc-Z3fn9H131';
    rules += ' __v6TNLWb131';
    rules += ' _Pc131';
    rules += ' _Wd131';
    rules += ' _je131';
    rules += ' _jlf131';
    if (props.isDisabled) rules += ' ilc131';
    else rules += ' ilb131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.isQuiet) rules += ' g9131';
    else if (props.isDisabled) rules += ' gH131';
    else if (props.isOpen) rules += ' gF131';
    else {
        if (props.isPressed) rules += ' gF131';
        else if (props.isFocusVisible) rules += ' gF131';
        else if (props.isHovered) rules += ' gF131';
        else rules += ' gH131';
    }
    if (props.isQuiet) rules += ' Ln131';
    rules += ' __ca131';
    if (props.size === "XL") rules += ' -F_-Fz131';
    else if (props.size === "L") rules += ' -F_-Fv131';
    else if (props.size === "S") rules += ' -F_-Fp131';
    else if (props.size === "XS") rules += ' -F_-Fn131';
    else rules += ' -F_-Fx131';
    if (props.isQuiet) rules += ' -S_-Sd131';
    else rules += ' -S_-Sv131';
    if (props.isQuiet) rules += ' -R_-Rd131';
    else rules += ' -R_-Rv131';
    return rules;
};
const $b9c30139cd63c62a$var$quietFocusLine = " Za131 FLvNfn131 _Pa131 _lr131 og131 ng131 kg131 jg131 g_h131 gle131";
let $b9c30139cd63c62a$export$b1e5508a851be14d = function anonymous(props) {
    let rules = " ";
    rules += ' _Le131';
    rules += ' se131';
    rules += ' Za131';
    if (props.size === "XL") rules += ' Dk92HIe131';
    else if (props.size === "L") rules += ' DyaX9Fb131';
    else if (props.size === "M") rules += ' Deb5jcb131';
    else if (props.size === "S") rules += ' DGTLJAe131';
    rules += ' _oa131';
    rules += ' K40ub4c131';
    rules += ' _Na131';
    rules += ' Pa131';
    rules += ' Tt131';
    rules += ' Qt131';
    rules += ' St131';
    rules += ' Rt131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    rules += ' __ya131';
    return rules;
};
const $b9c30139cd63c62a$var$invalidBorder = function anonymous(props) {
    let rules = " ";
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-ob131';
    rules += ' oj131';
    rules += ' nj131';
    rules += ' kj131';
    rules += ' jj131';
    rules += ' _Pa131';
    rules += ' Wr131';
    rules += ' _Er131';
    rules += ' _lr131';
    rules += ' _Rr131';
    rules += ' _Ob131';
    rules += ' _jf131';
    rules += ' _kc131';
    rules += ' hc131';
    rules += ' mc131';
    rules += ' lc131';
    if (props.isPressed) rules += ' im131';
    else if (props.isFocusVisible) rules += ' im131';
    else if (props.isHovered) rules += ' im131';
    else rules += ' ip131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    return rules;
};
const $b9c30139cd63c62a$var$valueStyles = function anonymous(props) {
    let rules = " ";
    if (props.isQuiet) rules += ' _ua131';
    else rules += ' _ub131';
    rules += ' _Nc131';
    rules += ' Pc131';
    rules += ' _Yb131';
    rules += ' _7c131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' Fb131';
    return rules;
};
const $b9c30139cd63c62a$var$iconStyles = function anonymous(props) {
    let rules = " ";
    rules += ' _va131';
    rules += ' _Sd131';
    rules += ' -_8sjo0b-t5ZbAob131';
    if (props.isLoading) rules += ' pp131';
    return rules;
};
const $b9c30139cd63c62a$var$avatar = " z2U5Crc131 y2U5Crc131 B2U5Crc131 A2U5Crc131 Ht131 Hsu131";
const $b9c30139cd63c62a$var$loadingWrapperStyles = " z2mU131 yzZKy131 sd131 eb131 _Ca131 Jv131 Gv131";
const $b9c30139cd63c62a$var$progressCircleStyles = function anonymous(props) {
    let rules = " ";
    if (props.size === "XL") rules += ' Zp131';
    else if (props.size === "L") rules += ' Zn131';
    else if (props.size === "M") rules += ' Zm131';
    else if (props.size === "S") rules += ' Zu131';
    if (props.size === "XL") rules += ' Fq131';
    else if (props.size === "L") rules += ' Fo131';
    else if (props.size === "M") rules += ' Fn131';
    else if (props.size === "S") rules += ' Ft131';
    return rules;
};
let $b9c30139cd63c62a$var$InternalPickerContext = /*#__PURE__*/ (0, $ceqeb$react.createContext)({
    size: 'M'
});
let $b9c30139cd63c62a$var$InsideSelectValueContext = /*#__PURE__*/ (0, $ceqeb$react.createContext)(false);
const $b9c30139cd63c62a$export$ba25329847403e11 = /*#__PURE__*/ (0, $ceqeb$react.forwardRef)(function Picker(props1, ref) {
    let stringFormatter = (0, $ceqeb$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    [props1, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props1, ref, $b9c30139cd63c62a$export$b72b8dade9393cd5);
    let domRef = (0, $150382364eecc4bc$exports.useFocusableRef)(ref);
    let formContext = (0, $ceqeb$react.useContext)((0, $bfa4962d90c8af48$exports.FormContext));
    props1 = (0, $bfa4962d90c8af48$exports.useFormProps)(props1);
    let { direction: direction = 'bottom', align: align = 'start', shouldFlip: shouldFlip = true, menuWidth: menuWidth, label: label, description: descriptionMessage, errorMessage: errorMessage, children: children, items: items, size: size = 'M', labelPosition: labelPosition = 'top', labelAlign: labelAlign = 'start', necessityIndicator: necessityIndicator, UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, placeholder: placeholder = stringFormatter.format('picker.placeholder'), isQuiet: isQuiet, loadingState: loadingState, renderValue: renderValue, onLoadMore: onLoadMore, ...pickerProps } = props1;
    // Better way to encode this into a style? need to account for flipping
    let menuOffset;
    if (size === 'S') menuOffset = 6;
    else if (size === 'M') menuOffset = 6;
    else if (size === 'L') menuOffset = 7;
    else menuOffset = 8;
    let renderer;
    let showButtonSpinner = (0, $ceqeb$react.useMemo)(()=>loadingState === 'loading', [
        loadingState
    ]);
    let spinnerId = (0, $ceqeb$reactariaprivateutilsuseId.useSlotId)([
        showButtonSpinner
    ]);
    let listBoxLoadingCircle = /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactariacomponentsListBox.ListBoxLoadMoreItem), {
        className: $b9c30139cd63c62a$var$loadingWrapperStyles,
        isLoading: loadingState === 'loadingMore',
        onLoadMore: onLoadMore,
        children: /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)($b9c30139cd63c62a$var$PickerProgressCircle, {
            size: size,
            "aria-label": stringFormatter.format('table.loadingMore')
        })
    });
    if (typeof children === 'function' && items) renderer = /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsxs)((0, $ceqeb$reactjsxruntime.Fragment), {
        children: [
            /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactariaCollection.Collection), {
                items: items,
                dependencies: props1.dependencies,
                children: children
            }),
            listBoxLoadingCircle
        ]
    });
    else renderer = /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsxs)((0, $ceqeb$reactjsxruntime.Fragment), {
        children: [
            children,
            listBoxLoadingCircle
        ]
    });
    let scale = (0, $ee7b4c497f520c08$exports.useScale)();
    let { direction: dir } = (0, $ceqeb$reactariaI18nProvider.useLocale)();
    let RTLFlipOffset = dir === 'rtl' ? -1 : 1;
    return /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactariacomponentsSelect.Select), {
        ...pickerProps,
        "aria-describedby": spinnerId,
        placeholder: placeholder,
        style: UNSAFE_style,
        className: UNSAFE_className + function anonymous(props, overrides) {
            let rules = " ";
            let gridColumnStart = false;
            let gridColumnEnd = false;
            let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
            for (let p of matches){
                if (p[1] === "z") gridColumnStart = true;
                if (p[1] === "y") gridColumnEnd = true;
                rules += p[0];
            }
            rules += ' se131';
            if (props.isInForm) {
                if (!gridColumnStart) rules += ' z2mU131';
            }
            if (props.isInForm) {
                if (props.labelPosition === "side") {
                    if (!gridColumnEnd) rules += ' yvoofG131';
                }
            }
            if (props.isInForm) rules += ' DLwlgod131';
            else {
                if (props.labelPosition === "side") rules += ' DPlN5qb131';
                else if (props.labelPosition === "top") rules += ' DM8Mfn131';
            }
            if (props.labelPosition === "side") rules += ' EPlN5qb131';
            else if (props.labelPosition === "top") rules += ' E4w1sLc131';
            if (props.labelPosition === "side") rules += ' CbRM95c131';
            else if (props.labelPosition === "top") rules += ' Cxaocre131';
            if (props.size === "XL") {
                rules += ' -_6BNtrc-e131';
                rules += ' vx131';
            } else if (props.size === "L") {
                rules += ' -_6BNtrc-d131';
                rules += ' vx131';
            } else if (props.size === "S") {
                rules += ' -_6BNtrc-b131';
                rules += ' vx131';
            } else if (props.size === "XS") {
                rules += ' -_6BNtrc-a131';
                rules += ' vx131';
            } else {
                rules += ' -_6BNtrc-c131';
                rules += ' vx131';
            }
            rules += ' ea131';
            rules += ' _Fd131';
            rules += ' _FnuYUweb131';
            if (props.size === "XL") rules += ' -BhX7R-Fz131';
            else if (props.size === "L") rules += ' -BhX7R-Fv131';
            else if (props.size === "S") rules += ' -BhX7R-Fp131';
            else if (props.size === "XS") rules += ' -BhX7R-Fn131';
            else rules += ' -BhX7R-Fx131';
            rules += ' -AJjOLd-UVkF0Db131';
            rules += ' qg131';
            rules += ' __ca131';
            return rules;
        }({
            isInForm: !!formContext,
            labelPosition: labelPosition,
            size: size
        }, props1.styles),
        children: ({ isDisabled: isDisabled, isOpen: isOpen, isFocusVisible: isFocusVisible, isInvalid: isInvalid, isRequired: isRequired })=>/*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactjsxruntime.Fragment), {
                children: /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsxs)($b9c30139cd63c62a$var$InternalPickerContext.Provider, {
                    value: {
                        size: size
                    },
                    children: [
                        /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldLabel), {
                            includeNecessityIndicatorInAccessibilityName: true,
                            isDisabled: isDisabled,
                            isRequired: isRequired,
                            size: size,
                            labelPosition: labelPosition,
                            labelAlign: labelAlign,
                            isQuiet: isQuiet,
                            necessityIndicator: necessityIndicator,
                            contextualHelp: props1.contextualHelp,
                            children: label
                        }),
                        /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)($b9c30139cd63c62a$var$PickerButton, {
                            loadingState: loadingState,
                            renderValue: renderValue,
                            isOpen: isOpen,
                            isQuiet: isQuiet,
                            isFocusVisible: isFocusVisible,
                            size: size,
                            isInvalid: isInvalid,
                            isDisabled: isDisabled,
                            buttonRef: domRef,
                            loadingCircle: /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)($b9c30139cd63c62a$var$PickerProgressCircle, {
                                id: spinnerId,
                                size: size,
                                "aria-label": stringFormatter.format('table.loading')
                            })
                        }),
                        /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.HelpText), {
                            size: size,
                            isDisabled: isDisabled,
                            isInvalid: isInvalid,
                            description: descriptionMessage,
                            children: errorMessage
                        }),
                        /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactariacomponentsVirtualizer.Virtualizer), {
                            layout: (0, $ceqeb$reactstatelyuseVirtualizerState.ListLayout),
                            layoutOptions: {
                                estimatedRowHeight: 32,
                                estimatedHeadingHeight: 50,
                                padding: 8,
                                loaderHeight: (0, $a2d5b35c01bebcd6$exports.LOADER_ROW_HEIGHTS)[size][scale]
                            },
                            children: /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $493371ef39bf7a55$exports.Popover), {
                                hideArrow: true,
                                padding: "none",
                                offset: menuOffset,
                                crossOffset: isQuiet ? RTLFlipOffset * -12 : undefined,
                                placement: `${direction} ${align}`,
                                shouldFlip: shouldFlip,
                                UNSAFE_style: {
                                    width: menuWidth && !isQuiet ? `${menuWidth}px` : undefined
                                },
                                styles: function anonymous(props) {
                                    let rules = " ";
                                    if (props.isQuiet) rules += ' Nh131';
                                    else rules += ' NuHSBze131';
                                    if (props.isQuiet) rules += ' ZbWuche131';
                                    else rules += ' ZuHSBze131';
                                    return rules;
                                }(props1),
                                children: /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)("div", {
                                    className: " sd131 Za131 Fb131",
                                    children: /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactariacomponentsslots.Provider), {
                                        values: [
                                            [
                                                (0, $6367bc87eb7d24ad$exports.HeaderContext),
                                                {
                                                    styles: (0, $a2d5b35c01bebcd6$exports.listboxHeader)({
                                                        size: size
                                                    })
                                                }
                                            ],
                                            [
                                                (0, $6367bc87eb7d24ad$exports.HeadingContext),
                                                {
                                                    // @ts-ignore
                                                    role: 'presentation',
                                                    styles: (0, $e741ea6b88ce4866$exports.sectionHeading)
                                                }
                                            ],
                                            [
                                                (0, $6367bc87eb7d24ad$exports.TextContext),
                                                {
                                                    slots: {
                                                        description: {
                                                            styles: (0, $e741ea6b88ce4866$exports.description)({
                                                                size: size,
                                                                isFocused: false,
                                                                isDisabled: false
                                                            })
                                                        }
                                                    }
                                                }
                                            ]
                                        ],
                                        children: /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactariacomponentsListBox.ListBox), {
                                            dependencies: props1.dependencies,
                                            items: items,
                                            className: (0, $a2d5b35c01bebcd6$exports.listbox)({
                                                size: size
                                            }),
                                            children: renderer
                                        })
                                    })
                                })
                            })
                        })
                    ]
                })
            })
    });
});
function $b9c30139cd63c62a$var$PickerProgressCircle(props) {
    let { id: id, size: size, 'aria-label': ariaLabel } = props;
    return /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $1c116ea4b0d2d023$exports.ProgressCircle), {
        id: id,
        isIndeterminate: true,
        size: "S",
        "aria-label": ariaLabel,
        styles: $b9c30139cd63c62a$var$progressCircleStyles({
            size: size
        })
    });
}
const $b9c30139cd63c62a$var$avatarSize = {
    S: 16,
    M: 20,
    L: 22,
    XL: 26
};
// https://w3c.github.io/aria/#widget_roles
let $b9c30139cd63c62a$var$INTERACTIVE_ARIA_ROLES = new Set([
    'application',
    'button',
    'checkbox',
    'combobox',
    'gridcell',
    'link',
    'menuitem',
    'menuitemcheckbox',
    'menuitemradio',
    'option',
    'radio',
    'searchbox',
    'separator',
    'slider',
    'spinbutton',
    'switch',
    'tab',
    'textbox',
    'treeitem'
]);
// Needs to be hidable component or otherwise the PressResponder throws a warning when rendered in the fake DOM and tries to register
const $b9c30139cd63c62a$var$PickerButton = (0, $ceqeb$reactariaprivatecollectionsHidden.createHideableComponent)(function PickerButton(props) {
    let { isOpen: isOpen, isQuiet: isQuiet, isFocusVisible: isFocusVisible, size: size, isInvalid: isInvalid, isDisabled: isDisabled, loadingState: loadingState, loadingCircle: loadingCircle, buttonRef: buttonRef, renderValue: renderValue } = props;
    let stringFormatter = (0, $ceqeb$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    let renderValueRef = (0, $ceqeb$react.useRef)(null);
    let state = (0, $ceqeb$react.useContext)((0, $ceqeb$reactariacomponentsSelect.SelectStateContext));
    (0, $ceqeb$react.useEffect)(()=>{
        if (process.env.NODE_ENV === 'production' || !renderValue) return;
        if (!renderValueRef.current) return;
        let doc = (0, $ceqeb$reactariaprivateutilsdomHelpers.getOwnerDocument)(renderValueRef.current);
        let walker = (0, $ceqeb$reactariaprivateutilsshadowdomShadowTreeWalker.createShadowTreeWalker)(doc, renderValueRef.current, NodeFilter.SHOW_ELEMENT, {
            acceptNode (node) {
                let role = node.getAttribute('role');
                let interactive = (0, $ceqeb$reactariaprivateutilsisFocusable.isFocusable)(node) || role != null && $b9c30139cd63c62a$var$INTERACTIVE_ARIA_ROLES.has(role);
                return interactive ? NodeFilter.FILTER_ACCEPT : NodeFilter.FILTER_SKIP;
            }
        });
        let next = walker.nextNode();
        if (next) console.warn("Picker's value should not have interactive children for accessibility.");
    }, [
        state.selectedItems,
        renderValue
    ]);
    // For mouse interactions, pickers open on press start. When the popover underlay appears
    // it covers the trigger button, causing onPressEnd to fire immediately and no press scaling
    // to occur. We override this by listening for pointerup on the document ourselves.
    let [isPressed, setPressed] = (0, $ceqeb$react.useState)(false);
    let { addGlobalListener: addGlobalListener } = (0, $ceqeb$reactariaprivateutilsuseGlobalListeners.useGlobalListeners)();
    let onPressStart = (e)=>{
        if (e.pointerType !== 'mouse') return;
        setPressed(true);
        addGlobalListener(document, 'pointerup', ()=>{
            setPressed(false);
        }, {
            once: true,
            capture: true
        });
    };
    return /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactariaprivateinteractionsPressResponder.PressResponder), {
        onPressStart: onPressStart,
        isPressed: isPressed,
        children: /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactariacomponentsButton.Button), {
            ref: buttonRef,
            style: (renderProps)=>(0, $2061c83559b50a66$exports.pressScale)(buttonRef)(renderProps),
            // Prevent press scale from sticking while Picker is open.
            // @ts-ignore
            isPressed: false,
            className: (renderProps)=>$b9c30139cd63c62a$var$inputButton({
                    ...renderProps,
                    size: size,
                    isOpen: isOpen,
                    isQuiet: isQuiet
                }),
            children: (renderProps)=>/*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsxs)((0, $ceqeb$reactjsxruntime.Fragment), {
                    children: [
                        /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactariacomponentsSelect.SelectValue), {
                            className: $b9c30139cd63c62a$var$valueStyles({
                                isQuiet: isQuiet
                            }) + (renderValue ? '' : " pqYuhc"),
                            children: ({ selectedItems: selectedItems, defaultChildren: defaultChildren })=>{
                                const selectedValues = selectedItems.filter((item)=>item != null);
                                const defaultRenderedValue = selectedItems.length <= 1 ? defaultChildren : /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $6367bc87eb7d24ad$exports.Text), {
                                    slot: "label",
                                    children: stringFormatter.format('picker.selectedCount', {
                                        count: selectedItems.length
                                    })
                                });
                                const renderedValue = selectedItems.length > 0 && renderValue ? renderValue(selectedValues) : defaultRenderedValue;
                                return /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactariacomponentsslots.Provider), {
                                    values: [
                                        [
                                            (0, $bde97c91243ed164$exports.IconContext),
                                            {
                                                slots: {
                                                    icon: {
                                                        render: (0, $e991cbcdf82ced71$exports.centerBaseline)({
                                                            slot: 'icon',
                                                            styles: (0, $e741ea6b88ce4866$exports.iconCenterWrapper)
                                                        }),
                                                        styles: (0, $e741ea6b88ce4866$exports.icon)
                                                    }
                                                }
                                            }
                                        ],
                                        [
                                            (0, $38071d9ce246d4cf$exports.AvatarContext),
                                            {
                                                slots: {
                                                    avatar: {
                                                        size: $b9c30139cd63c62a$var$avatarSize[size ?? 'M'],
                                                        styles: $b9c30139cd63c62a$var$avatar
                                                    }
                                                }
                                            }
                                        ],
                                        [
                                            (0, $6367bc87eb7d24ad$exports.TextContext),
                                            {
                                                slots: {
                                                    description: {},
                                                    [(0, $ceqeb$reactariacomponentsslots.DEFAULT_SLOT)]: {
                                                        styles: " sb131 _ub131 _Nc131 Pc131 _Yb131 _7c131",
                                                        // @ts-ignore
                                                        'data-slot': 'label'
                                                    },
                                                    label: {
                                                        styles: " sb131 _ub131 _Nc131 Pc131 _Yb131 _7c131",
                                                        // @ts-ignore not technically necessary, but good for consistency
                                                        'data-slot': 'label'
                                                    }
                                                }
                                            }
                                        ],
                                        [
                                            $b9c30139cd63c62a$var$InsideSelectValueContext,
                                            true
                                        ]
                                    ],
                                    children: renderValue ? /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)("div", {
                                        ref: renderValueRef,
                                        style: {
                                            display: 'contents'
                                        },
                                        children: renderedValue
                                    }) : renderedValue
                                });
                            }
                        }),
                        isInvalid && /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldErrorIcon), {
                            isDisabled: isDisabled
                        }),
                        loadingState === 'loading' && !isOpen && loadingCircle,
                        /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $1df9f1c9262ce5df$exports.default), {
                            size: size,
                            className: $b9c30139cd63c62a$var$iconStyles({
                                isLoading: loadingState === 'loading'
                            })
                        }),
                        isFocusVisible && isQuiet && /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)("span", {
                            className: $b9c30139cd63c62a$var$quietFocusLine
                        }),
                        isInvalid && !isDisabled && !isQuiet && // @ts-ignore known limitation detecting functions from the theme
                        /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)("div", {
                            className: $b9c30139cd63c62a$var$invalidBorder({
                                ...renderProps,
                                size: size
                            })
                        })
                    ]
                })
        })
    });
});
const $b9c30139cd63c62a$var$checkmarkIconSize = {
    S: 'XS',
    M: 'M',
    L: 'L',
    XL: 'XL'
};
function $b9c30139cd63c62a$export$d601881f38163e28(props) {
    let ref = (0, $ceqeb$react.useRef)(null);
    let isLink = props.href != null;
    let { size: size } = (0, $ceqeb$react.useContext)($b9c30139cd63c62a$var$InternalPickerContext);
    return /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactariacomponentsListBox.ListBoxItem), {
        ...props,
        ref: ref,
        textValue: props.textValue || (typeof props.children === 'string' ? props.children : undefined),
        style: (0, $2061c83559b50a66$exports.pressScale)(ref, props.UNSAFE_style),
        className: (renderProps)=>(props.UNSAFE_className || '') + (0, $a2d5b35c01bebcd6$exports.listboxItem)({
                ...renderProps,
                size: size,
                isLink: isLink
            }, props.styles),
        children: (renderProps)=>{
            let { children: children } = props;
            let checkboxRenderProps = {
                ...renderProps,
                size: size,
                isFocused: false,
                isFocusVisible: false,
                isIndeterminate: false,
                isReadOnly: false,
                isInvalid: false,
                isRequired: false
            };
            return /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)($b9c30139cd63c62a$var$DefaultProvider, {
                context: (0, $bde97c91243ed164$exports.IconContext),
                value: {
                    slots: {
                        icon: {
                            render: (0, $e991cbcdf82ced71$exports.centerBaseline)({
                                slot: 'icon',
                                styles: (0, $e741ea6b88ce4866$exports.iconCenterWrapper)
                            }),
                            styles: (0, $e741ea6b88ce4866$exports.icon)
                        }
                    }
                },
                children: /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)($b9c30139cd63c62a$var$DefaultProvider, {
                    context: (0, $38071d9ce246d4cf$exports.AvatarContext),
                    value: {
                        slots: {
                            avatar: {
                                size: $b9c30139cd63c62a$var$avatarSize[size],
                                styles: $b9c30139cd63c62a$var$avatar
                            }
                        }
                    },
                    children: /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsxs)($b9c30139cd63c62a$var$DefaultProvider, {
                        context: (0, $6367bc87eb7d24ad$exports.TextContext),
                        value: {
                            slots: {
                                [(0, $ceqeb$reactariacomponentsslots.DEFAULT_SLOT)]: {
                                    styles: (0, $e741ea6b88ce4866$exports.label)({
                                        size: size
                                    })
                                },
                                label: {
                                    styles: (0, $e741ea6b88ce4866$exports.label)({
                                        size: size
                                    })
                                },
                                description: {
                                    styles: (0, $e741ea6b88ce4866$exports.description)({
                                        ...renderProps,
                                        size: size
                                    })
                                }
                            }
                        },
                        children: [
                            renderProps.selectionMode === 'single' && !isLink && /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $dba2b18fc5727f9b$exports.default), {
                                size: $b9c30139cd63c62a$var$checkmarkIconSize[size],
                                className: (0, $e741ea6b88ce4866$exports.checkmark)({
                                    ...renderProps,
                                    size: size
                                })
                            }),
                            renderProps.selectionMode === 'multiple' && !isLink && /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)("div", {
                                className: (0, $308b180f49d82d28$exports.mergeStyles)((0, $e741ea6b88ce4866$exports.checkbox), (0, $af04e099a53d3e85$exports.box)(checkboxRenderProps)),
                                children: /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $dba2b18fc5727f9b$exports.default), {
                                    size: size,
                                    className: (0, $af04e099a53d3e85$exports.iconStyles)
                                })
                            }),
                            typeof children === 'string' ? /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $6367bc87eb7d24ad$exports.Text), {
                                slot: "label",
                                children: children
                            }) : children
                        ]
                    })
                })
            });
        }
    });
}
// A Context.Provider that only sets a value if not inside SelectValue.
function $b9c30139cd63c62a$var$DefaultProvider({ context: context, value: value, children: children }) {
    let inSelectValue = (0, $ceqeb$react.useContext)($b9c30139cd63c62a$var$InsideSelectValueContext);
    if (inSelectValue) return children;
    return /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)(context.Provider, {
        value: value,
        children: children
    });
}
function $b9c30139cd63c62a$export$a340772aca6155cd(props) {
    let { size: size } = (0, $ceqeb$react.useContext)($b9c30139cd63c62a$var$InternalPickerContext);
    return /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsxs)((0, $ceqeb$reactjsxruntime.Fragment), {
        children: [
            /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $ceqeb$reactariacomponentsListBox.ListBoxSection), {
                ...props,
                children: props.children
            }),
            /*#__PURE__*/ (0, $ceqeb$reactjsxruntime.jsx)((0, $a2d5b35c01bebcd6$exports.Divider), {
                size: size
            })
        ]
    });
}
