// @react-spectrum/s2/dist/private/Picker.mjs
import "./Picker.css";
import {AvatarContext as $3ffa2cd930156220$export$a20dad690e1279e2} from "./Avatar.mjs";
import {box as $9b405e0f8095dce0$export$827f4ee28efc37, iconStyles as $9b405e0f8095dce0$export$2c99b98b77550cb4} from "./Checkbox.mjs";
import {centerBaseline as $1f4b04be3f24aae3$export$9d7e2342a7e53afa} from "./CenterBaseline.mjs";
import {checkbox as $13afb0ea5f0ed767$export$b07e517d22efa1ca, checkmark as $13afb0ea5f0ed767$export$292abbf31ed842a, description as $13afb0ea5f0ed767$export$6d59db4903f20f7d, icon as $13afb0ea5f0ed767$export$1ca1ec8b29a4ce27, iconCenterWrapper as $13afb0ea5f0ed767$export$31d1842a0fd736e6, label as $13afb0ea5f0ed767$export$1237798dc640739a, sectionHeading as $13afb0ea5f0ed767$export$300ac6f106ef584d} from "./Menu.mjs";
import $81cf3d4588736e7d$export$2e2bcd8739ae039 from "./Checkmark.mjs";
import $4b69f02ec06b9226$export$2e2bcd8739ae039 from "./Chevron.mjs";
import {Divider as $a2f262fac286a920$export$2e0a83ec2e27ecbb, listbox as $a2f262fac286a920$export$9901d82449b93cea, listboxHeader as $a2f262fac286a920$export$c040ba7a724e8da1, listboxItem as $a2f262fac286a920$export$68265fb354482bae, LOADER_ROW_HEIGHTS as $a2f262fac286a920$export$83379f5245567d78} from "./ComboBox.mjs";
import {FieldErrorIcon as $9afd80978c252040$export$bda0d075e0e342c9, FieldLabel as $9afd80978c252040$export$1acdcf5a973a8414, HelpText as $9afd80978c252040$export$a67c0bc59081311a} from "./Field.mjs";
import {FormContext as $9b916426527cebe7$export$c24727297075ec6a, useFormProps as $9b916426527cebe7$export$a6b5be5c6b451665} from "./Form.mjs";
import {HeaderContext as $8e847109a6ab556d$export$e0e4026c12a8bdbb, HeadingContext as $8e847109a6ab556d$export$d688439359537581, Text as $8e847109a6ab556d$export$5f1af8db9871e1d6, TextContext as $8e847109a6ab556d$export$9afb8bc826b033ea} from "./Content.mjs";
import {IconContext as $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed} from "../../icons/Icon.mjs";
import $c77I5$intlStringsmjs from "./intlStrings.mjs";
import {mergeStyles as $feb886035e0d4633$export$e618dc39ac9ad607} from "../../icons/runtime.mjs";
import {Popover as $88b746eba92c8d0d$export$5b6b19405a83ff9d} from "./Popover.mjs";
import {pressScale as $10ea7662e51a285b$export$56e8cba416805d8d} from "./pressScale.mjs";
import {ProgressCircle as $41ddd91dbbf0c389$export$c79b9d6b4cc92af7} from "./ProgressCircle.mjs";
import {useFocusableRef as $80ecb974df713a30$export$96a734597687c040} from "./useDOMRef.mjs";
import {useScale as $859432f3b3573fcb$export$a8d2043b2d807f4d} from "./utils.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $c77I5$jsx, jsxs as $c77I5$jsxs, Fragment as $c77I5$Fragment} from "react/jsx-runtime";
import {ListBoxLoadMoreItem as $c77I5$ListBoxLoadMoreItem, ListBox as $c77I5$ListBox, ListBoxItem as $c77I5$ListBoxItem, ListBoxSection as $c77I5$ListBoxSection} from "react-aria-components/ListBox";
import {Select as $c77I5$Select, SelectStateContext as $c77I5$SelectStateContext, SelectValue as $c77I5$SelectValue} from "react-aria-components/Select";
import {Button as $c77I5$Button} from "react-aria-components/Button";
import {Collection as $c77I5$Collection} from "react-aria/Collection";
import {Provider as $c77I5$Provider, DEFAULT_SLOT as $c77I5$DEFAULT_SLOT} from "react-aria-components/slots";
import {createHideableComponent as $c77I5$createHideableComponent} from "react-aria/private/collections/Hidden";
import {createShadowTreeWalker as $c77I5$createShadowTreeWalker} from "react-aria/private/utils/shadowdom/ShadowTreeWalker";
import {getOwnerDocument as $c77I5$getOwnerDocument} from "react-aria/private/utils/domHelpers";
import {isFocusable as $c77I5$isFocusable} from "react-aria/private/utils/isFocusable";
import {ListLayout as $c77I5$ListLayout} from "react-stately/useVirtualizerState";
import {PressResponder as $c77I5$PressResponder} from "react-aria/private/interactions/PressResponder";
import {createContext as $c77I5$createContext, forwardRef as $c77I5$forwardRef, useContext as $c77I5$useContext, useMemo as $c77I5$useMemo, useRef as $c77I5$useRef, useEffect as $c77I5$useEffect, useState as $c77I5$useState} from "react";
import {useGlobalListeners as $c77I5$useGlobalListeners} from "react-aria/private/utils/useGlobalListeners";
import {useLocale as $c77I5$useLocale} from "react-aria/I18nProvider";
import {useLocalizedStringFormatter as $c77I5$useLocalizedStringFormatter} from "react-aria/useLocalizedStringFormatter";
import {useSlotId as $c77I5$useSlotId} from "react-aria/private/utils/useId";
import {Virtualizer as $c77I5$Virtualizer} from "react-aria-components/Virtualizer";


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 




































const $b41c15d27fbaff7d$export$b72b8dade9393cd5 = /*#__PURE__*/ (0, $c77I5$createContext)(null);
const $b41c15d27fbaff7d$var$inputButton = function anonymous(props) {
    let rules = " ";
    if (props.isQuiet) rules += ' _Le131';
    else if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isDisabled) rules += ' pp131';
    else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    }
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _oa131';
    rules += ' SMBFGYc131';
    rules += ' RfBFGYc131';
    if (props.size === "XL") rules += ' Nl131';
    else if (props.size === "L") rules += ' Nj131';
    else if (props.size === "S") rules += ' Nf131';
    else if (props.size === "XS") rules += ' Ne131';
    else rules += ' Nk131';
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-ob131';
    rules += ' oj131';
    rules += ' nj131';
    rules += ' kj131';
    rules += ' jj131';
    rules += ' qc131';
    rules += ' qsd131';
    rules += ' -Qg41Lb-IAxCW0c131';
    rules += ' -Qg41Lb-gLWx3bIy131';
    rules += ' FRuFGYc131';
    rules += ' zXcEWr131';
    rules += ' yXcEWr131';
    rules += ' BXcEWr131';
    rules += ' AXcEWr131';
    if (props.isQuiet) rules += ' _qd131';
    else rules += ' _qtb131';
    if (props.size === "XL") rules += ' -Anxdcc-Zt131';
    else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
    else if (props.size === "S") rules += ' -Anxdcc-Zr131';
    else rules += ' -Anxdcc-Z3fn9H131';
    rules += ' __v6TNLWb131';
    rules += ' _Pc131';
    rules += ' _Wd131';
    rules += ' _je131';
    rules += ' _jlf131';
    if (props.isDisabled) rules += ' ilc131';
    else rules += ' ilb131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.isQuiet) rules += ' g9131';
    else if (props.isDisabled) rules += ' gH131';
    else if (props.isOpen) rules += ' gF131';
    else {
        if (props.isPressed) rules += ' gF131';
        else if (props.isFocusVisible) rules += ' gF131';
        else if (props.isHovered) rules += ' gF131';
        else rules += ' gH131';
    }
    if (props.isQuiet) rules += ' Ln131';
    rules += ' __ca131';
    if (props.size === "XL") rules += ' -F_-Fz131';
    else if (props.size === "L") rules += ' -F_-Fv131';
    else if (props.size === "S") rules += ' -F_-Fp131';
    else if (props.size === "XS") rules += ' -F_-Fn131';
    else rules += ' -F_-Fx131';
    if (props.isQuiet) rules += ' -S_-Sd131';
    else rules += ' -S_-Sv131';
    if (props.isQuiet) rules += ' -R_-Rd131';
    else rules += ' -R_-Rv131';
    return rules;
};
const $b41c15d27fbaff7d$var$quietFocusLine = " Za131 FLvNfn131 _Pa131 _lr131 og131 ng131 kg131 jg131 g_h131 gle131";
let $b41c15d27fbaff7d$export$b1e5508a851be14d = function anonymous(props) {
    let rules = " ";
    rules += ' _Le131';
    rules += ' se131';
    rules += ' Za131';
    if (props.size === "XL") rules += ' Dk92HIe131';
    else if (props.size === "L") rules += ' DyaX9Fb131';
    else if (props.size === "M") rules += ' Deb5jcb131';
    else if (props.size === "S") rules += ' DGTLJAe131';
    rules += ' _oa131';
    rules += ' K40ub4c131';
    rules += ' _Na131';
    rules += ' Pa131';
    rules += ' Tt131';
    rules += ' Qt131';
    rules += ' St131';
    rules += ' Rt131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    rules += ' __ya131';
    return rules;
};
const $b41c15d27fbaff7d$var$invalidBorder = function anonymous(props) {
    let rules = " ";
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-ob131';
    rules += ' oj131';
    rules += ' nj131';
    rules += ' kj131';
    rules += ' jj131';
    rules += ' _Pa131';
    rules += ' Wr131';
    rules += ' _Er131';
    rules += ' _lr131';
    rules += ' _Rr131';
    rules += ' _Ob131';
    rules += ' _jf131';
    rules += ' _kc131';
    rules += ' hc131';
    rules += ' mc131';
    rules += ' lc131';
    if (props.isPressed) rules += ' im131';
    else if (props.isFocusVisible) rules += ' im131';
    else if (props.isHovered) rules += ' im131';
    else rules += ' ip131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    return rules;
};
const $b41c15d27fbaff7d$var$valueStyles = function anonymous(props) {
    let rules = " ";
    if (props.isQuiet) rules += ' _ua131';
    else rules += ' _ub131';
    rules += ' _Nc131';
    rules += ' Pc131';
    rules += ' _Yb131';
    rules += ' _7c131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' Fb131';
    return rules;
};
const $b41c15d27fbaff7d$var$iconStyles = function anonymous(props) {
    let rules = " ";
    rules += ' _va131';
    rules += ' _Sd131';
    rules += ' -_8sjo0b-t5ZbAob131';
    if (props.isLoading) rules += ' pp131';
    return rules;
};
const $b41c15d27fbaff7d$var$avatar = " z2U5Crc131 y2U5Crc131 B2U5Crc131 A2U5Crc131 Ht131 Hsu131";
const $b41c15d27fbaff7d$var$loadingWrapperStyles = " z2mU131 yzZKy131 sd131 eb131 _Ca131 Jv131 Gv131";
const $b41c15d27fbaff7d$var$progressCircleStyles = function anonymous(props) {
    let rules = " ";
    if (props.size === "XL") rules += ' Zp131';
    else if (props.size === "L") rules += ' Zn131';
    else if (props.size === "M") rules += ' Zm131';
    else if (props.size === "S") rules += ' Zu131';
    if (props.size === "XL") rules += ' Fq131';
    else if (props.size === "L") rules += ' Fo131';
    else if (props.size === "M") rules += ' Fn131';
    else if (props.size === "S") rules += ' Ft131';
    return rules;
};
let $b41c15d27fbaff7d$var$InternalPickerContext = /*#__PURE__*/ (0, $c77I5$createContext)({
    size: 'M'
});
let $b41c15d27fbaff7d$var$InsideSelectValueContext = /*#__PURE__*/ (0, $c77I5$createContext)(false);
const $b41c15d27fbaff7d$export$ba25329847403e11 = /*#__PURE__*/ (0, $c77I5$forwardRef)(function Picker(props1, ref) {
    let stringFormatter = (0, $c77I5$useLocalizedStringFormatter)((0, ($parcel$interopDefault($c77I5$intlStringsmjs))), '@react-spectrum/s2');
    [props1, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props1, ref, $b41c15d27fbaff7d$export$b72b8dade9393cd5);
    let domRef = (0, $80ecb974df713a30$export$96a734597687c040)(ref);
    let formContext = (0, $c77I5$useContext)((0, $9b916426527cebe7$export$c24727297075ec6a));
    props1 = (0, $9b916426527cebe7$export$a6b5be5c6b451665)(props1);
    let { direction: direction = 'bottom', align: align = 'start', shouldFlip: shouldFlip = true, menuWidth: menuWidth, label: label, description: descriptionMessage, errorMessage: errorMessage, children: children, items: items, size: size = 'M', labelPosition: labelPosition = 'top', labelAlign: labelAlign = 'start', necessityIndicator: necessityIndicator, UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, placeholder: placeholder = stringFormatter.format('picker.placeholder'), isQuiet: isQuiet, loadingState: loadingState, renderValue: renderValue, onLoadMore: onLoadMore, ...pickerProps } = props1;
    // Better way to encode this into a style? need to account for flipping
    let menuOffset;
    if (size === 'S') menuOffset = 6;
    else if (size === 'M') menuOffset = 6;
    else if (size === 'L') menuOffset = 7;
    else menuOffset = 8;
    let renderer;
    let showButtonSpinner = (0, $c77I5$useMemo)(()=>loadingState === 'loading', [
        loadingState
    ]);
    let spinnerId = (0, $c77I5$useSlotId)([
        showButtonSpinner
    ]);
    let listBoxLoadingCircle = /*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$ListBoxLoadMoreItem), {
        className: $b41c15d27fbaff7d$var$loadingWrapperStyles,
        isLoading: loadingState === 'loadingMore',
        onLoadMore: onLoadMore,
        children: /*#__PURE__*/ (0, $c77I5$jsx)($b41c15d27fbaff7d$var$PickerProgressCircle, {
            size: size,
            "aria-label": stringFormatter.format('table.loadingMore')
        })
    });
    if (typeof children === 'function' && items) renderer = /*#__PURE__*/ (0, $c77I5$jsxs)((0, $c77I5$Fragment), {
        children: [
            /*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$Collection), {
                items: items,
                dependencies: props1.dependencies,
                children: children
            }),
            listBoxLoadingCircle
        ]
    });
    else renderer = /*#__PURE__*/ (0, $c77I5$jsxs)((0, $c77I5$Fragment), {
        children: [
            children,
            listBoxLoadingCircle
        ]
    });
    let scale = (0, $859432f3b3573fcb$export$a8d2043b2d807f4d)();
    let { direction: dir } = (0, $c77I5$useLocale)();
    let RTLFlipOffset = dir === 'rtl' ? -1 : 1;
    return /*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$Select), {
        ...pickerProps,
        "aria-describedby": spinnerId,
        placeholder: placeholder,
        style: UNSAFE_style,
        className: UNSAFE_className + function anonymous(props, overrides) {
            let rules = " ";
            let gridColumnStart = false;
            let gridColumnEnd = false;
            let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
            for (let p of matches){
                if (p[1] === "z") gridColumnStart = true;
                if (p[1] === "y") gridColumnEnd = true;
                rules += p[0];
            }
            rules += ' se131';
            if (props.isInForm) {
                if (!gridColumnStart) rules += ' z2mU131';
            }
            if (props.isInForm) {
                if (props.labelPosition === "side") {
                    if (!gridColumnEnd) rules += ' yvoofG131';
                }
            }
            if (props.isInForm) rules += ' DLwlgod131';
            else {
                if (props.labelPosition === "side") rules += ' DPlN5qb131';
                else if (props.labelPosition === "top") rules += ' DM8Mfn131';
            }
            if (props.labelPosition === "side") rules += ' EPlN5qb131';
            else if (props.labelPosition === "top") rules += ' E4w1sLc131';
            if (props.labelPosition === "side") rules += ' CbRM95c131';
            else if (props.labelPosition === "top") rules += ' Cxaocre131';
            if (props.size === "XL") {
                rules += ' -_6BNtrc-e131';
                rules += ' vx131';
            } else if (props.size === "L") {
                rules += ' -_6BNtrc-d131';
                rules += ' vx131';
            } else if (props.size === "S") {
                rules += ' -_6BNtrc-b131';
                rules += ' vx131';
            } else if (props.size === "XS") {
                rules += ' -_6BNtrc-a131';
                rules += ' vx131';
            } else {
                rules += ' -_6BNtrc-c131';
                rules += ' vx131';
            }
            rules += ' ea131';
            rules += ' _Fd131';
            rules += ' _FnuYUweb131';
            if (props.size === "XL") rules += ' -BhX7R-Fz131';
            else if (props.size === "L") rules += ' -BhX7R-Fv131';
            else if (props.size === "S") rules += ' -BhX7R-Fp131';
            else if (props.size === "XS") rules += ' -BhX7R-Fn131';
            else rules += ' -BhX7R-Fx131';
            rules += ' -AJjOLd-UVkF0Db131';
            rules += ' qg131';
            rules += ' __ca131';
            return rules;
        }({
            isInForm: !!formContext,
            labelPosition: labelPosition,
            size: size
        }, props1.styles),
        children: ({ isDisabled: isDisabled, isOpen: isOpen, isFocusVisible: isFocusVisible, isInvalid: isInvalid, isRequired: isRequired })=>/*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$Fragment), {
                children: /*#__PURE__*/ (0, $c77I5$jsxs)($b41c15d27fbaff7d$var$InternalPickerContext.Provider, {
                    value: {
                        size: size
                    },
                    children: [
                        /*#__PURE__*/ (0, $c77I5$jsx)((0, $9afd80978c252040$export$1acdcf5a973a8414), {
                            includeNecessityIndicatorInAccessibilityName: true,
                            isDisabled: isDisabled,
                            isRequired: isRequired,
                            size: size,
                            labelPosition: labelPosition,
                            labelAlign: labelAlign,
                            isQuiet: isQuiet,
                            necessityIndicator: necessityIndicator,
                            contextualHelp: props1.contextualHelp,
                            children: label
                        }),
                        /*#__PURE__*/ (0, $c77I5$jsx)($b41c15d27fbaff7d$var$PickerButton, {
                            loadingState: loadingState,
                            renderValue: renderValue,
                            isOpen: isOpen,
                            isQuiet: isQuiet,
                            isFocusVisible: isFocusVisible,
                            size: size,
                            isInvalid: isInvalid,
                            isDisabled: isDisabled,
                            buttonRef: domRef,
                            loadingCircle: /*#__PURE__*/ (0, $c77I5$jsx)($b41c15d27fbaff7d$var$PickerProgressCircle, {
                                id: spinnerId,
                                size: size,
                                "aria-label": stringFormatter.format('table.loading')
                            })
                        }),
                        /*#__PURE__*/ (0, $c77I5$jsx)((0, $9afd80978c252040$export$a67c0bc59081311a), {
                            size: size,
                            isDisabled: isDisabled,
                            isInvalid: isInvalid,
                            description: descriptionMessage,
                            children: errorMessage
                        }),
                        /*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$Virtualizer), {
                            layout: (0, $c77I5$ListLayout),
                            layoutOptions: {
                                estimatedRowHeight: 32,
                                estimatedHeadingHeight: 50,
                                padding: 8,
                                loaderHeight: (0, $a2f262fac286a920$export$83379f5245567d78)[size][scale]
                            },
                            children: /*#__PURE__*/ (0, $c77I5$jsx)((0, $88b746eba92c8d0d$export$5b6b19405a83ff9d), {
                                hideArrow: true,
                                padding: "none",
                                offset: menuOffset,
                                crossOffset: isQuiet ? RTLFlipOffset * -12 : undefined,
                                placement: `${direction} ${align}`,
                                shouldFlip: shouldFlip,
                                UNSAFE_style: {
                                    width: menuWidth && !isQuiet ? `${menuWidth}px` : undefined
                                },
                                styles: function anonymous(props) {
                                    let rules = " ";
                                    if (props.isQuiet) rules += ' Nh131';
                                    else rules += ' NuHSBze131';
                                    if (props.isQuiet) rules += ' ZbWuche131';
                                    else rules += ' ZuHSBze131';
                                    return rules;
                                }(props1),
                                children: /*#__PURE__*/ (0, $c77I5$jsx)("div", {
                                    className: " sd131 Za131 Fb131",
                                    children: /*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$Provider), {
                                        values: [
                                            [
                                                (0, $8e847109a6ab556d$export$e0e4026c12a8bdbb),
                                                {
                                                    styles: (0, $a2f262fac286a920$export$c040ba7a724e8da1)({
                                                        size: size
                                                    })
                                                }
                                            ],
                                            [
                                                (0, $8e847109a6ab556d$export$d688439359537581),
                                                {
                                                    // @ts-ignore
                                                    role: 'presentation',
                                                    styles: (0, $13afb0ea5f0ed767$export$300ac6f106ef584d)
                                                }
                                            ],
                                            [
                                                (0, $8e847109a6ab556d$export$9afb8bc826b033ea),
                                                {
                                                    slots: {
                                                        description: {
                                                            styles: (0, $13afb0ea5f0ed767$export$6d59db4903f20f7d)({
                                                                size: size,
                                                                isFocused: false,
                                                                isDisabled: false
                                                            })
                                                        }
                                                    }
                                                }
                                            ]
                                        ],
                                        children: /*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$ListBox), {
                                            dependencies: props1.dependencies,
                                            items: items,
                                            className: (0, $a2f262fac286a920$export$9901d82449b93cea)({
                                                size: size
                                            }),
                                            children: renderer
                                        })
                                    })
                                })
                            })
                        })
                    ]
                })
            })
    });
});
function $b41c15d27fbaff7d$var$PickerProgressCircle(props) {
    let { id: id, size: size, 'aria-label': ariaLabel } = props;
    return /*#__PURE__*/ (0, $c77I5$jsx)((0, $41ddd91dbbf0c389$export$c79b9d6b4cc92af7), {
        id: id,
        isIndeterminate: true,
        size: "S",
        "aria-label": ariaLabel,
        styles: $b41c15d27fbaff7d$var$progressCircleStyles({
            size: size
        })
    });
}
const $b41c15d27fbaff7d$var$avatarSize = {
    S: 16,
    M: 20,
    L: 22,
    XL: 26
};
// https://w3c.github.io/aria/#widget_roles
let $b41c15d27fbaff7d$var$INTERACTIVE_ARIA_ROLES = new Set([
    'application',
    'button',
    'checkbox',
    'combobox',
    'gridcell',
    'link',
    'menuitem',
    'menuitemcheckbox',
    'menuitemradio',
    'option',
    'radio',
    'searchbox',
    'separator',
    'slider',
    'spinbutton',
    'switch',
    'tab',
    'textbox',
    'treeitem'
]);
// Needs to be hidable component or otherwise the PressResponder throws a warning when rendered in the fake DOM and tries to register
const $b41c15d27fbaff7d$var$PickerButton = (0, $c77I5$createHideableComponent)(function PickerButton(props) {
    let { isOpen: isOpen, isQuiet: isQuiet, isFocusVisible: isFocusVisible, size: size, isInvalid: isInvalid, isDisabled: isDisabled, loadingState: loadingState, loadingCircle: loadingCircle, buttonRef: buttonRef, renderValue: renderValue } = props;
    let stringFormatter = (0, $c77I5$useLocalizedStringFormatter)((0, ($parcel$interopDefault($c77I5$intlStringsmjs))), '@react-spectrum/s2');
    let renderValueRef = (0, $c77I5$useRef)(null);
    let state = (0, $c77I5$useContext)((0, $c77I5$SelectStateContext));
    (0, $c77I5$useEffect)(()=>{
        if (process.env.NODE_ENV === 'production' || !renderValue) return;
        if (!renderValueRef.current) return;
        let doc = (0, $c77I5$getOwnerDocument)(renderValueRef.current);
        let walker = (0, $c77I5$createShadowTreeWalker)(doc, renderValueRef.current, NodeFilter.SHOW_ELEMENT, {
            acceptNode (node) {
                let role = node.getAttribute('role');
                let interactive = (0, $c77I5$isFocusable)(node) || role != null && $b41c15d27fbaff7d$var$INTERACTIVE_ARIA_ROLES.has(role);
                return interactive ? NodeFilter.FILTER_ACCEPT : NodeFilter.FILTER_SKIP;
            }
        });
        let next = walker.nextNode();
        if (next) console.warn("Picker's value should not have interactive children for accessibility.");
    }, [
        state.selectedItems,
        renderValue
    ]);
    // For mouse interactions, pickers open on press start. When the popover underlay appears
    // it covers the trigger button, causing onPressEnd to fire immediately and no press scaling
    // to occur. We override this by listening for pointerup on the document ourselves.
    let [isPressed, setPressed] = (0, $c77I5$useState)(false);
    let { addGlobalListener: addGlobalListener } = (0, $c77I5$useGlobalListeners)();
    let onPressStart = (e)=>{
        if (e.pointerType !== 'mouse') return;
        setPressed(true);
        addGlobalListener(document, 'pointerup', ()=>{
            setPressed(false);
        }, {
            once: true,
            capture: true
        });
    };
    return /*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$PressResponder), {
        onPressStart: onPressStart,
        isPressed: isPressed,
        children: /*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$Button), {
            ref: buttonRef,
            style: (renderProps)=>(0, $10ea7662e51a285b$export$56e8cba416805d8d)(buttonRef)(renderProps),
            // Prevent press scale from sticking while Picker is open.
            // @ts-ignore
            isPressed: false,
            className: (renderProps)=>$b41c15d27fbaff7d$var$inputButton({
                    ...renderProps,
                    size: size,
                    isOpen: isOpen,
                    isQuiet: isQuiet
                }),
            children: (renderProps)=>/*#__PURE__*/ (0, $c77I5$jsxs)((0, $c77I5$Fragment), {
                    children: [
                        /*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$SelectValue), {
                            className: $b41c15d27fbaff7d$var$valueStyles({
                                isQuiet: isQuiet
                            }) + (renderValue ? '' : " pqYuhc"),
                            children: ({ selectedItems: selectedItems, defaultChildren: defaultChildren })=>{
                                const selectedValues = selectedItems.filter((item)=>item != null);
                                const defaultRenderedValue = selectedItems.length <= 1 ? defaultChildren : /*#__PURE__*/ (0, $c77I5$jsx)((0, $8e847109a6ab556d$export$5f1af8db9871e1d6), {
                                    slot: "label",
                                    children: stringFormatter.format('picker.selectedCount', {
                                        count: selectedItems.length
                                    })
                                });
                                const renderedValue = selectedItems.length > 0 && renderValue ? renderValue(selectedValues) : defaultRenderedValue;
                                return /*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$Provider), {
                                    values: [
                                        [
                                            (0, $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed),
                                            {
                                                slots: {
                                                    icon: {
                                                        render: (0, $1f4b04be3f24aae3$export$9d7e2342a7e53afa)({
                                                            slot: 'icon',
                                                            styles: (0, $13afb0ea5f0ed767$export$31d1842a0fd736e6)
                                                        }),
                                                        styles: (0, $13afb0ea5f0ed767$export$1ca1ec8b29a4ce27)
                                                    }
                                                }
                                            }
                                        ],
                                        [
                                            (0, $3ffa2cd930156220$export$a20dad690e1279e2),
                                            {
                                                slots: {
                                                    avatar: {
                                                        size: $b41c15d27fbaff7d$var$avatarSize[size ?? 'M'],
                                                        styles: $b41c15d27fbaff7d$var$avatar
                                                    }
                                                }
                                            }
                                        ],
                                        [
                                            (0, $8e847109a6ab556d$export$9afb8bc826b033ea),
                                            {
                                                slots: {
                                                    description: {},
                                                    [(0, $c77I5$DEFAULT_SLOT)]: {
                                                        styles: " sb131 _ub131 _Nc131 Pc131 _Yb131 _7c131",
                                                        // @ts-ignore
                                                        'data-slot': 'label'
                                                    },
                                                    label: {
                                                        styles: " sb131 _ub131 _Nc131 Pc131 _Yb131 _7c131",
                                                        // @ts-ignore not technically necessary, but good for consistency
                                                        'data-slot': 'label'
                                                    }
                                                }
                                            }
                                        ],
                                        [
                                            $b41c15d27fbaff7d$var$InsideSelectValueContext,
                                            true
                                        ]
                                    ],
                                    children: renderValue ? /*#__PURE__*/ (0, $c77I5$jsx)("div", {
                                        ref: renderValueRef,
                                        style: {
                                            display: 'contents'
                                        },
                                        children: renderedValue
                                    }) : renderedValue
                                });
                            }
                        }),
                        isInvalid && /*#__PURE__*/ (0, $c77I5$jsx)((0, $9afd80978c252040$export$bda0d075e0e342c9), {
                            isDisabled: isDisabled
                        }),
                        loadingState === 'loading' && !isOpen && loadingCircle,
                        /*#__PURE__*/ (0, $c77I5$jsx)((0, $4b69f02ec06b9226$export$2e2bcd8739ae039), {
                            size: size,
                            className: $b41c15d27fbaff7d$var$iconStyles({
                                isLoading: loadingState === 'loading'
                            })
                        }),
                        isFocusVisible && isQuiet && /*#__PURE__*/ (0, $c77I5$jsx)("span", {
                            className: $b41c15d27fbaff7d$var$quietFocusLine
                        }),
                        isInvalid && !isDisabled && !isQuiet && // @ts-ignore known limitation detecting functions from the theme
                        /*#__PURE__*/ (0, $c77I5$jsx)("div", {
                            className: $b41c15d27fbaff7d$var$invalidBorder({
                                ...renderProps,
                                size: size
                            })
                        })
                    ]
                })
        })
    });
});
const $b41c15d27fbaff7d$var$checkmarkIconSize = {
    S: 'XS',
    M: 'M',
    L: 'L',
    XL: 'XL'
};
function $b41c15d27fbaff7d$export$d601881f38163e28(props) {
    let ref = (0, $c77I5$useRef)(null);
    let isLink = props.href != null;
    let { size: size } = (0, $c77I5$useContext)($b41c15d27fbaff7d$var$InternalPickerContext);
    return /*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$ListBoxItem), {
        ...props,
        ref: ref,
        textValue: props.textValue || (typeof props.children === 'string' ? props.children : undefined),
        style: (0, $10ea7662e51a285b$export$56e8cba416805d8d)(ref, props.UNSAFE_style),
        className: (renderProps)=>(props.UNSAFE_className || '') + (0, $a2f262fac286a920$export$68265fb354482bae)({
                ...renderProps,
                size: size,
                isLink: isLink
            }, props.styles),
        children: (renderProps)=>{
            let { children: children } = props;
            let checkboxRenderProps = {
                ...renderProps,
                size: size,
                isFocused: false,
                isFocusVisible: false,
                isIndeterminate: false,
                isReadOnly: false,
                isInvalid: false,
                isRequired: false
            };
            return /*#__PURE__*/ (0, $c77I5$jsx)($b41c15d27fbaff7d$var$DefaultProvider, {
                context: (0, $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed),
                value: {
                    slots: {
                        icon: {
                            render: (0, $1f4b04be3f24aae3$export$9d7e2342a7e53afa)({
                                slot: 'icon',
                                styles: (0, $13afb0ea5f0ed767$export$31d1842a0fd736e6)
                            }),
                            styles: (0, $13afb0ea5f0ed767$export$1ca1ec8b29a4ce27)
                        }
                    }
                },
                children: /*#__PURE__*/ (0, $c77I5$jsx)($b41c15d27fbaff7d$var$DefaultProvider, {
                    context: (0, $3ffa2cd930156220$export$a20dad690e1279e2),
                    value: {
                        slots: {
                            avatar: {
                                size: $b41c15d27fbaff7d$var$avatarSize[size],
                                styles: $b41c15d27fbaff7d$var$avatar
                            }
                        }
                    },
                    children: /*#__PURE__*/ (0, $c77I5$jsxs)($b41c15d27fbaff7d$var$DefaultProvider, {
                        context: (0, $8e847109a6ab556d$export$9afb8bc826b033ea),
                        value: {
                            slots: {
                                [(0, $c77I5$DEFAULT_SLOT)]: {
                                    styles: (0, $13afb0ea5f0ed767$export$1237798dc640739a)({
                                        size: size
                                    })
                                },
                                label: {
                                    styles: (0, $13afb0ea5f0ed767$export$1237798dc640739a)({
                                        size: size
                                    })
                                },
                                description: {
                                    styles: (0, $13afb0ea5f0ed767$export$6d59db4903f20f7d)({
                                        ...renderProps,
                                        size: size
                                    })
                                }
                            }
                        },
                        children: [
                            renderProps.selectionMode === 'single' && !isLink && /*#__PURE__*/ (0, $c77I5$jsx)((0, $81cf3d4588736e7d$export$2e2bcd8739ae039), {
                                size: $b41c15d27fbaff7d$var$checkmarkIconSize[size],
                                className: (0, $13afb0ea5f0ed767$export$292abbf31ed842a)({
                                    ...renderProps,
                                    size: size
                                })
                            }),
                            renderProps.selectionMode === 'multiple' && !isLink && /*#__PURE__*/ (0, $c77I5$jsx)("div", {
                                className: (0, $feb886035e0d4633$export$e618dc39ac9ad607)((0, $13afb0ea5f0ed767$export$b07e517d22efa1ca), (0, $9b405e0f8095dce0$export$827f4ee28efc37)(checkboxRenderProps)),
                                children: /*#__PURE__*/ (0, $c77I5$jsx)((0, $81cf3d4588736e7d$export$2e2bcd8739ae039), {
                                    size: size,
                                    className: (0, $9b405e0f8095dce0$export$2c99b98b77550cb4)
                                })
                            }),
                            typeof children === 'string' ? /*#__PURE__*/ (0, $c77I5$jsx)((0, $8e847109a6ab556d$export$5f1af8db9871e1d6), {
                                slot: "label",
                                children: children
                            }) : children
                        ]
                    })
                })
            });
        }
    });
}
// A Context.Provider that only sets a value if not inside SelectValue.
function $b41c15d27fbaff7d$var$DefaultProvider({ context: context, value: value, children: children }) {
    let inSelectValue = (0, $c77I5$useContext)($b41c15d27fbaff7d$var$InsideSelectValueContext);
    if (inSelectValue) return children;
    return /*#__PURE__*/ (0, $c77I5$jsx)(context.Provider, {
        value: value,
        children: children
    });
}
function $b41c15d27fbaff7d$export$a340772aca6155cd(props) {
    let { size: size } = (0, $c77I5$useContext)($b41c15d27fbaff7d$var$InternalPickerContext);
    return /*#__PURE__*/ (0, $c77I5$jsxs)((0, $c77I5$Fragment), {
        children: [
            /*#__PURE__*/ (0, $c77I5$jsx)((0, $c77I5$ListBoxSection), {
                ...props,
                children: props.children
            }),
            /*#__PURE__*/ (0, $c77I5$jsx)((0, $a2f262fac286a920$export$2e0a83ec2e27ecbb), {
                size: size
            })
        ]
    });
}


export {$b41c15d27fbaff7d$export$b72b8dade9393cd5 as PickerContext, $b41c15d27fbaff7d$export$b1e5508a851be14d as menu, $b41c15d27fbaff7d$export$ba25329847403e11 as Picker, $b41c15d27fbaff7d$export$d601881f38163e28 as PickerItem, $b41c15d27fbaff7d$export$a340772aca6155cd as PickerSection};
