// @react-spectrum/s2/dist/private/ProgressBar.cjs
require("./ProgressBar.css");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $308b180f49d82d28$exports = require("../../icons/runtime.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $gIpiX$reactjsxruntime = require("react/jsx-runtime");
var $gIpiX$reactariacomponentsProgressBar = require("react-aria-components/ProgressBar");
var $gIpiX$react = require("react");
var $gIpiX$reactariaI18nProvider = require("react-aria/I18nProvider");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "ProgressBarContext", () => $83e59061dd392311$export$e9f3bf65a26ce129);
$parcel$export(module.exports, "ProgressBar", () => $83e59061dd392311$export$c17561cb55d4db30);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 







const $83e59061dd392311$export$e9f3bf65a26ce129 = /*#__PURE__*/ (0, $gIpiX$react.createContext)(null);
const $83e59061dd392311$var$indeterminateLTR = "VHPsfc";
const $83e59061dd392311$var$indeterminateRTL = "reCXe";
const $83e59061dd392311$var$wrapper = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let minWidth = false;
    let maxWidth = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "N") minWidth = true;
        if (p[1] === "L") maxWidth = true;
        rules += p[0];
    }
    if (props.staticColor === "white") rules += ' -Ovbhqd-gcwrnr131';
    else if (props.staticColor === "black") rules += ' -Ovbhqd-g4hn3s131';
    if (!position) rules += ' _Pc131';
    rules += ' se131';
    if (props.isIndeterminate) {
        if (props.labelPosition === "side") rules += ' DPlN5qb131';
        else if (props.labelPosition === "top") rules += ' DM8Mfn131';
    } else {
        if (props.labelPosition === "side") rules += ' DPlN5qb131';
        else if (props.labelPosition === "top") rules += ' Dxumf9b131';
    }
    if (props.isIndeterminate) {
        if (props.labelPosition === "side") rules += ' C2Yfxtb131';
        else if (props.labelPosition === "top") rules += ' C0J5Chb131';
    } else {
        if (props.labelPosition === "side") rules += ' Cvv4gue131';
        else if (props.labelPosition === "top") rules += ' CC0Gyo131';
    }
    rules += ' ea131';
    rules += ' _Bb131';
    if (!minWidth) rules += ' Nl131';
    if (!maxWidth) rules += ' Lk131';
    if (props.size === "XL") rules += ' -BhX7R-Fz131';
    else if (props.size === "L") rules += ' -BhX7R-Fv131';
    else if (props.size === "S") rules += ' -BhX7R-Fp131';
    else if (props.size === "XS") rules += ' -BhX7R-Fn131';
    else rules += ' -BhX7R-Fx131';
    rules += ' -dTMmze-Fe131';
    rules += ' -AJjOLd-UCwrhQe131';
    rules += ' qg131';
    return rules;
};
const $83e59061dd392311$var$valueStyles = function anonymous(props) {
    let rules = " ";
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isStaticColor) rules += ' pl131';
    else if (props.isDisabled) rules += ' pp131';
    else rules += ' pv131';
    rules += ' plb131';
    rules += ' ri131';
    rules += ' z8bmXs131';
    rules += ' y8bmXs131';
    rules += ' B8bmXs131';
    rules += ' A8bmXs131';
    return rules;
};
const $83e59061dd392311$var$trackStyles = function anonymous(props) {
    let rules = " ";
    rules += ' zMY0fn131';
    rules += ' yMY0fn131';
    rules += ' BMY0fn131';
    rules += ' AMY0fn131';
    if (props.size === "XL") rules += ' Nl131';
    else if (props.size === "L") rules += ' Nj131';
    else if (props.size === "S") rules += ' Nf131';
    else if (props.size === "XS") rules += ' Ne131';
    else rules += ' Nk131';
    if (props.isQuiet) rules += ' _qd131';
    else rules += ' _qtb131';
    if (props.size === "XL") rules += ' -Anxdcc-Zt131';
    else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
    else if (props.size === "S") rules += ' -Anxdcc-Zr131';
    else rules += ' -Anxdcc-Z3fn9H131';
    rules += ' __v6TNLWb131';
    rules += ' _Nc131';
    rules += ' Pc131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    if (props.isStaticColor) rules += ' gh131';
    else rules += ' gE131';
    rules += ' glb131';
    rules += ' _Ma131';
    rules += ' _Mlb131';
    rules += ' _Le131';
    rules += ' _Llf131';
    rules += ' Oj131';
    rules += ' Olb131';
    rules += ' _9c131';
    if (props.size === "XL") rules += ' Fj131';
    else if (props.size === "L") rules += ' Fi131';
    else if (props.size === "M") rules += ' Fg131';
    else if (props.size === "S") rules += ' Fe131';
    else rules += ' Fg131';
    return rules;
};
const $83e59061dd392311$var$fill = function anonymous(props) {
    let rules = " ";
    rules += ' Fb131';
    rules += ' _je131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    if (props.isStaticColor) rules += ' gj131';
    else rules += ' g2131';
    rules += ' glc131';
    if (props.isIndeterminate) rules += ' Za131';
    if (props.isIndeterminate) rules += ' _0e131';
    return rules;
};
const $83e59061dd392311$var$indeterminateAnimation = function anonymous(props) {
    let rules = " ";
    if (props.direction === "rtl") rules += ' _g2FRDs131';
    else if (props.direction === "ltr") rules += ' _gXZ1Q4d131';
    rules += ' _ea131';
    rules += ' _hb131';
    rules += ' __ea131';
    rules += ' _8d131';
    rules += ' _Pc131';
    return rules;
};
const $83e59061dd392311$export$c17561cb55d4db30 = /*#__PURE__*/ (0, $gIpiX$react.forwardRef)(function ProgressBar(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $83e59061dd392311$export$e9f3bf65a26ce129);
    let { label: label, size: size = 'M', staticColor: staticColor, isIndeterminate: isIndeterminate, labelPosition: labelPosition = 'top', UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '' } = props;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let { direction: direction } = (0, $gIpiX$reactariaI18nProvider.useLocale)();
    let isStaticColor = !!staticColor;
    return /*#__PURE__*/ (0, $gIpiX$reactjsxruntime.jsx)((0, $gIpiX$reactariacomponentsProgressBar.ProgressBar), {
        ...props,
        ref: domRef,
        style: UNSAFE_style,
        className: UNSAFE_className + $83e59061dd392311$var$wrapper({
            ...props,
            size: size,
            labelPosition: labelPosition,
            staticColor: staticColor
        }, props.styles),
        children: ({ percentage: percentage, valueText: valueText })=>/*#__PURE__*/ (0, $gIpiX$reactjsxruntime.jsxs)((0, $gIpiX$reactjsxruntime.Fragment), {
                children: [
                    label && /*#__PURE__*/ (0, $gIpiX$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldLabel), {
                        size: size,
                        labelAlign: "start",
                        labelPosition: labelPosition,
                        staticColor: staticColor,
                        children: label
                    }),
                    label && !isIndeterminate && /*#__PURE__*/ (0, $gIpiX$reactjsxruntime.jsx)("span", {
                        className: $83e59061dd392311$var$valueStyles({
                            size: size,
                            labelAlign: 'end',
                            isStaticColor: isStaticColor
                        }),
                        children: valueText
                    }),
                    /*#__PURE__*/ (0, $gIpiX$reactjsxruntime.jsx)("div", {
                        className: $83e59061dd392311$var$trackStyles({
                            isStaticColor: isStaticColor,
                            size: size
                        }),
                        children: /*#__PURE__*/ (0, $gIpiX$reactjsxruntime.jsx)("div", {
                            className: (0, $308b180f49d82d28$exports.mergeStyles)($83e59061dd392311$var$fill({
                                ...props,
                                isStaticColor: isStaticColor
                            }), isIndeterminate ? $83e59061dd392311$var$indeterminateAnimation({
                                direction: direction
                            }) : null),
                            style: {
                                width: isIndeterminate ? undefined : percentage + '%'
                            }
                        })
                    })
                ]
            })
    });
});
