// @react-spectrum/s2/dist/private/RadioGroup.mjs
import "./RadioGroup.css";
import {CenterBaseline as $1f4b04be3f24aae3$export$768dac55bb57081d} from "./CenterBaseline.mjs";
import {FieldLabel as $9afd80978c252040$export$1acdcf5a973a8414, HelpText as $9afd80978c252040$export$a67c0bc59081311a} from "./Field.mjs";
import {FormContext as $9b916426527cebe7$export$c24727297075ec6a, useFormProps as $9b916426527cebe7$export$a6b5be5c6b451665} from "./Form.mjs";
import {pressScale as $10ea7662e51a285b$export$56e8cba416805d8d} from "./pressScale.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8, useFocusableRef as $80ecb974df713a30$export$96a734597687c040} from "./useDOMRef.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $fZVHt$jsx, jsxs as $fZVHt$jsxs, Fragment as $fZVHt$Fragment} from "react/jsx-runtime";
import {RadioGroup as $fZVHt$RadioGroup, RadioField as $fZVHt$RadioField, RadioButton as $fZVHt$RadioButton} from "react-aria-components/RadioGroup";
import {createContext as $fZVHt$createContext, forwardRef as $fZVHt$forwardRef, useContext as $fZVHt$useContext, useRef as $fZVHt$useRef} from "react";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 








const $9a2afae66771094f$export$a79eda4ff50e30b6 = /*#__PURE__*/ (0, $fZVHt$createContext)(null);
const $9a2afae66771094f$export$a98f0dcb43a68a25 = /*#__PURE__*/ (0, $fZVHt$forwardRef)(function RadioGroup(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $9a2afae66771094f$export$a79eda4ff50e30b6);
    let formContext = (0, $fZVHt$useContext)((0, $9b916426527cebe7$export$c24727297075ec6a));
    props = (0, $9b916426527cebe7$export$a6b5be5c6b451665)(props);
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let { label: label, description: description, errorMessage: errorMessage, children: children, isEmphasized: isEmphasized, labelPosition: labelPosition = 'top', labelAlign: labelAlign = 'start', necessityIndicator: necessityIndicator = 'icon', size: size = 'M', orientation: orientation = 'vertical', UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, ...groupProps } = props;
    return /*#__PURE__*/ (0, $fZVHt$jsx)((0, $fZVHt$RadioGroup), {
        ...groupProps,
        orientation: orientation,
        ref: domRef,
        style: UNSAFE_style,
        className: UNSAFE_className + function anonymous(props, overrides) {
            let rules = " ";
            let gridColumnStart = false;
            let gridColumnEnd = false;
            let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
            for (let p of matches){
                if (p[1] === "z") gridColumnStart = true;
                if (p[1] === "y") gridColumnEnd = true;
                rules += p[0];
            }
            rules += ' se131';
            if (props.isInForm) {
                if (!gridColumnStart) rules += ' z2mU131';
            }
            if (props.isInForm) {
                if (props.labelPosition === "side") {
                    if (!gridColumnEnd) rules += ' yvoofG131';
                }
            }
            if (props.isInForm) rules += ' DLwlgod131';
            else {
                if (props.labelPosition === "side") rules += ' DPlN5qb131';
                else if (props.labelPosition === "top") rules += ' DM8Mfn131';
            }
            if (props.labelPosition === "side") rules += ' EPlN5qb131';
            else if (props.labelPosition === "top") rules += ' E4w1sLc131';
            if (props.labelPosition === "side") rules += ' CbRM95c131';
            else if (props.labelPosition === "top") rules += ' Cxaocre131';
            if (props.size === "XL") {
                rules += ' -_6BNtrc-e131';
                rules += ' vx131';
            } else if (props.size === "L") {
                rules += ' -_6BNtrc-d131';
                rules += ' vx131';
            } else if (props.size === "S") {
                rules += ' -_6BNtrc-b131';
                rules += ' vx131';
            } else if (props.size === "XS") {
                rules += ' -_6BNtrc-a131';
                rules += ' vx131';
            } else {
                rules += ' -_6BNtrc-c131';
                rules += ' vx131';
            }
            rules += ' ea131';
            rules += ' _Fd131';
            rules += ' _FnuYUweb131';
            if (props.size === "XL") rules += ' -BhX7R-Fz131';
            else if (props.size === "L") rules += ' -BhX7R-Fv131';
            else if (props.size === "S") rules += ' -BhX7R-Fp131';
            else if (props.size === "XS") rules += ' -BhX7R-Fn131';
            else rules += ' -BhX7R-Fx131';
            rules += ' -AJjOLd-UVxyPec131';
            rules += ' qg131';
            rules += ' __ca131';
            return rules;
        }({
            size: size,
            labelPosition: labelPosition,
            isInForm: !!formContext
        }, props.styles),
        children: ({ isDisabled: isDisabled, isInvalid: isInvalid })=>/*#__PURE__*/ (0, $fZVHt$jsxs)((0, $fZVHt$Fragment), {
                children: [
                    /*#__PURE__*/ (0, $fZVHt$jsx)((0, $9afd80978c252040$export$1acdcf5a973a8414), {
                        isDisabled: isDisabled,
                        isRequired: props.isRequired,
                        size: size,
                        labelPosition: labelPosition,
                        labelAlign: labelAlign,
                        isQuiet: true,
                        necessityIndicator: necessityIndicator,
                        contextualHelp: props.contextualHelp,
                        children: label
                    }),
                    /*#__PURE__*/ (0, $fZVHt$jsx)("div", {
                        className: function anonymous(props) {
                            let rules = " ";
                            rules += ' zXcEWr131';
                            rules += ' yXcEWr131';
                            rules += ' BXcEWr131';
                            rules += ' AXcEWr131';
                            rules += ' sd131';
                            if (props.orientation === "horizontal") rules += ' _tc131';
                            else if (props.orientation === "vertical") rules += ' _ta131';
                            if (props.orientation === "horizontal") rules += ' _wb131';
                            rules += ' ql131';
                            rules += ' UgOnRt131';
                            return rules;
                        }({
                            orientation: orientation
                        }),
                        children: /*#__PURE__*/ (0, $fZVHt$jsx)((0, $9b916426527cebe7$export$c24727297075ec6a).Provider, {
                            value: {
                                ...formContext,
                                size: size,
                                isEmphasized: isEmphasized
                            },
                            children: children
                        })
                    }),
                    /*#__PURE__*/ (0, $fZVHt$jsx)((0, $9afd80978c252040$export$a67c0bc59081311a), {
                        size: size,
                        isDisabled: isDisabled,
                        isInvalid: isInvalid,
                        description: description,
                        showErrorIcon: true,
                        children: errorMessage
                    })
                ]
            })
    });
});
const $9a2afae66771094f$var$radioField = function anonymous(props, overrides) {
    let rules = " ";
    let gridColumnStart = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "z") gridColumnStart = true;
        rules += p[0];
    }
    rules += ' se131';
    if (props.isNoVisibleLabel) rules += ' DtyCQZ131';
    else rules += ' DE5iebb131';
    rules += ' qf131';
    rules += ' _ch131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.size === "XL") rules += ' pt131';
    else if (props.size === "L") rules += ' pt131';
    else if (props.size === "S") rules += ' pt131';
    else if (props.size === "XS") rules += ' pt131';
    else rules += ' pt131';
    if (props.size === "XL") rules += ' -BhX7R-Fz131';
    else if (props.size === "L") rules += ' -BhX7R-Fv131';
    else if (props.size === "S") rules += ' -BhX7R-Fp131';
    else if (props.size === "XS") rules += ' -BhX7R-Fn131';
    else rules += ' -BhX7R-Fx131';
    if (props.size === "XL") rules += ' Ua131';
    else if (props.size === "L") rules += ' Ua131';
    else if (props.size === "M") rules += ' Uai5a0131';
    else if (props.size === "S") rules += ' Uai5a0131';
    if (props.isInForm) {
        if (!gridColumnStart) rules += ' zhJUGr131';
    }
    return rules;
};
const $9a2afae66771094f$var$wrapper = function anonymous(props) {
    let rules = " ";
    rules += ' se131';
    rules += ' DLwlgod131';
    rules += ' z2mU131';
    rules += ' yzZKy131';
    rules += ' _Pc131';
    rules += ' ea131';
    rules += ' Yc131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.isDisabled) {
        rules += ' pp131';
        rules += ' plc131';
    } else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    }
    rules += ' __ca131';
    return rules;
};
const $9a2afae66771094f$var$circle = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (props.size === "XL") rules += ' Zm131';
    else if (props.size === "L") rules += ' Zl131';
    else if (props.size === "S") rules += ' Zk131';
    else rules += ' Zu131';
    rules += ' FRuFGYc131';
    rules += ' _va131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _Ca131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    rules += ' _jf131';
    rules += ' _oa131';
    if (props.isSelected) rules += ' _kUXU7p131';
    else rules += ' _k7m7Gv131';
    if (props.isSelected) rules += ' hUXU7p131';
    else rules += ' h7m7Gv131';
    if (props.isSelected) rules += ' mUXU7p131';
    else rules += ' m7m7Gv131';
    if (props.isSelected) rules += ' lUXU7p131';
    else rules += ' l7m7Gv131';
    rules += ' _yb131';
    rules += ' gX131';
    if (props.isDisabled) {
        rules += ' io131';
        rules += ' ilc131';
    } else if (props.isInvalid) {
        if (props.isPressed) rules += ' im131';
        else if (props.isFocusVisible) rules += ' im131';
        else if (props.isHovered) rules += ' im131';
        else rules += ' ip131';
        rules += ' ile131';
    } else if (props.isSelected) {
        if (props.isEmphasized) {
            if (props.isPressed) rules += ' iu131';
            else if (props.isFocusVisible) rules += ' iu131';
            else if (props.isHovered) rules += ' iu131';
            else rules += ' iw131';
        }
        rules += ' ild131';
    } else {
        if (props.isPressed) rules += ' in131';
        else if (props.isFocusVisible) rules += ' in131';
        else if (props.isHovered) rules += ' in131';
        else rules += ' iv131';
        rules += ' ila131';
    }
    if (props.size === "XL") rules += ' -F_-Fn131';
    else if (props.size === "L") rules += ' -F_-Fm131';
    else if (props.size === "S") rules += ' -F_-Fl131';
    else rules += ' -F_-Ft131';
    return rules;
};
const $9a2afae66771094f$var$smallerSize = {
    S: 'XS',
    M: 'S',
    L: 'M',
    XL: 'L'
};
const $9a2afae66771094f$export$d7b12c4107be0d61 = /*#__PURE__*/ (0, $fZVHt$forwardRef)(function Radio(props, ref) {
    let { children: children, UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style } = props;
    let circleRef = (0, $fZVHt$useRef)(null);
    let inputRef = (0, $fZVHt$useRef)(null);
    let domRef = (0, $80ecb974df713a30$export$96a734597687c040)(ref, inputRef);
    let isInForm = !!(0, $fZVHt$useContext)((0, $9b916426527cebe7$export$c24727297075ec6a));
    let { size: size = 'M', ...allProps } = (0, $9b916426527cebe7$export$a6b5be5c6b451665)(props);
    return /*#__PURE__*/ (0, $fZVHt$jsx)((0, $fZVHt$RadioField), {
        ...allProps,
        ref: domRef,
        inputRef: inputRef,
        style: UNSAFE_style,
        className: (renderProps)=>UNSAFE_className + $9a2afae66771094f$var$radioField({
                ...renderProps,
                isInForm: isInForm,
                size: size,
                isNoVisibleLabel: !children
            }, allProps.styles),
        children: (renderProps)=>/*#__PURE__*/ (0, $fZVHt$jsxs)((0, $fZVHt$Fragment), {
                children: [
                    /*#__PURE__*/ (0, $fZVHt$jsx)((0, $fZVHt$RadioButton), {
                        className: (renderProps)=>$9a2afae66771094f$var$wrapper({
                                ...renderProps,
                                isInForm: isInForm,
                                size: size
                            }),
                        children: (renderProps)=>/*#__PURE__*/ (0, $fZVHt$jsxs)((0, $fZVHt$Fragment), {
                                children: [
                                    /*#__PURE__*/ (0, $fZVHt$jsx)((0, $1f4b04be3f24aae3$export$768dac55bb57081d), {
                                        children: /*#__PURE__*/ (0, $fZVHt$jsx)("div", {
                                            ref: circleRef,
                                            style: (0, $10ea7662e51a285b$export$56e8cba416805d8d)(circleRef)(renderProps),
                                            className: $9a2afae66771094f$var$circle({
                                                ...renderProps,
                                                isEmphasized: allProps.isEmphasized,
                                                isSelected: renderProps.isSelected,
                                                size: size
                                            })
                                        })
                                    }),
                                    children
                                ]
                            })
                    }),
                    /*#__PURE__*/ (0, $fZVHt$jsx)((0, $9afd80978c252040$export$a67c0bc59081311a), {
                        size: $9a2afae66771094f$var$smallerSize[size],
                        styles: " z3mU131 Td131",
                        isDisabled: renderProps.isDisabled,
                        description: props.description
                    })
                ]
            })
    });
});


export {$9a2afae66771094f$export$a79eda4ff50e30b6 as RadioGroupContext, $9a2afae66771094f$export$a98f0dcb43a68a25 as RadioGroup, $9a2afae66771094f$export$d7b12c4107be0d61 as Radio};
