// @react-spectrum/s2/dist/private/S2_ChevronSize200.cjs
var $76jmY$reactjsxruntime = require("react/jsx-runtime");
require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $1895c28190a0d28f$export$2e2bcd8739ae039);


const $1895c28190a0d28f$var$SvgS2ChevronSize200 = (props)=>/*#__PURE__*/ (0, $76jmY$reactjsxruntime.jsx)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 12,
        height: 12,
        viewBox: "0 0 12 12",
        ...props,
        children: /*#__PURE__*/ (0, $76jmY$reactjsxruntime.jsx)("path", {
            fill: "var(--iconPrimary, #222)",
            d: "M9.301 6c0-.049-.02-.095-.028-.143-.01-.068-.013-.136-.039-.2a.9.9 0 0 0-.2-.302L4.343.663a.912.912 0 0 0-1.29 1.29L7.102 6l-4.047 4.048a.912.912 0 0 0 1.289 1.289l4.691-4.692a.9.9 0 0 0 .2-.302c.026-.064.028-.132.04-.2C9.28 6.095 9.3 6.05 9.3 6"
        })
    });
var $1895c28190a0d28f$export$2e2bcd8739ae039 = $1895c28190a0d28f$var$SvgS2ChevronSize200;
