// @react-spectrum/s2/dist/private/S2_DragHandleSize300.mjs
import {jsxs as $iEEOH$jsxs, jsx as $iEEOH$jsx} from "react/jsx-runtime";
import "react";



const $2328c0ed6835818a$var$SvgS2DragHandleSize300 = (props)=>/*#__PURE__*/ (0, $iEEOH$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 14,
        height: 14,
        viewBox: "0 0 14 14",
        ...props,
        children: [
            /*#__PURE__*/ (0, $iEEOH$jsx)("circle", {
                cx: 4.6,
                cy: 11.8,
                r: 1.2,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $iEEOH$jsx)("circle", {
                cx: 4.6,
                cy: 7,
                r: 1.2,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $iEEOH$jsx)("circle", {
                cx: 4.6,
                cy: 2.2,
                r: 1.2,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $iEEOH$jsx)("circle", {
                cx: 9.4,
                cy: 11.8,
                r: 1.2,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $iEEOH$jsx)("circle", {
                cx: 9.4,
                cy: 7,
                r: 1.2,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $iEEOH$jsx)("circle", {
                cx: 9.4,
                cy: 2.2,
                r: 1.2,
                fill: "var(--iconPrimary, #222)"
            })
        ]
    });
var $2328c0ed6835818a$export$2e2bcd8739ae039 = $2328c0ed6835818a$var$SvgS2DragHandleSize300;


export {$2328c0ed6835818a$export$2e2bcd8739ae039 as default};
