// @react-spectrum/s2/dist/private/S2_DragHandleSize300.cjs
var $cMWFR$reactjsxruntime = require("react/jsx-runtime");
require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $b0491803a28e72c2$export$2e2bcd8739ae039);


const $b0491803a28e72c2$var$SvgS2DragHandleSize300 = (props)=>/*#__PURE__*/ (0, $cMWFR$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 14,
        height: 14,
        viewBox: "0 0 14 14",
        ...props,
        children: [
            /*#__PURE__*/ (0, $cMWFR$reactjsxruntime.jsx)("circle", {
                cx: 4.6,
                cy: 11.8,
                r: 1.2,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $cMWFR$reactjsxruntime.jsx)("circle", {
                cx: 4.6,
                cy: 7,
                r: 1.2,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $cMWFR$reactjsxruntime.jsx)("circle", {
                cx: 4.6,
                cy: 2.2,
                r: 1.2,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $cMWFR$reactjsxruntime.jsx)("circle", {
                cx: 9.4,
                cy: 11.8,
                r: 1.2,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $cMWFR$reactjsxruntime.jsx)("circle", {
                cx: 9.4,
                cy: 7,
                r: 1.2,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $cMWFR$reactjsxruntime.jsx)("circle", {
                cx: 9.4,
                cy: 2.2,
                r: 1.2,
                fill: "var(--iconPrimary, #222)"
            })
        ]
    });
var $b0491803a28e72c2$export$2e2bcd8739ae039 = $b0491803a28e72c2$var$SvgS2DragHandleSize300;
