// @react-spectrum/s2/dist/private/S2_DragHandleSize100.mjs
import {jsxs as $1wFAy$jsxs, jsx as $1wFAy$jsx} from "react/jsx-runtime";
import "react";



const $0adf62121b55c8dc$var$SvgS2DragHandleSize100 = (props)=>/*#__PURE__*/ (0, $1wFAy$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 10,
        height: 10,
        viewBox: "0 0 10 10",
        ...props,
        children: [
            /*#__PURE__*/ (0, $1wFAy$jsx)("circle", {
                cx: 3,
                cy: 9,
                r: 1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $1wFAy$jsx)("circle", {
                cx: 3,
                cy: 5,
                r: 1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $1wFAy$jsx)("circle", {
                cx: 3,
                cy: 1,
                r: 1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $1wFAy$jsx)("circle", {
                cx: 7,
                cy: 9,
                r: 1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $1wFAy$jsx)("circle", {
                cx: 7,
                cy: 5,
                r: 1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $1wFAy$jsx)("circle", {
                cx: 7,
                cy: 1,
                r: 1,
                fill: "var(--iconPrimary, #222)"
            })
        ]
    });
var $0adf62121b55c8dc$export$2e2bcd8739ae039 = $0adf62121b55c8dc$var$SvgS2DragHandleSize100;


export {$0adf62121b55c8dc$export$2e2bcd8739ae039 as default};
