// @react-spectrum/s2/dist/private/S2_DragHandleSize100.cjs
var $byuoM$reactjsxruntime = require("react/jsx-runtime");
require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $88d6d23073bc9ab9$export$2e2bcd8739ae039);


const $88d6d23073bc9ab9$var$SvgS2DragHandleSize100 = (props)=>/*#__PURE__*/ (0, $byuoM$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 10,
        height: 10,
        viewBox: "0 0 10 10",
        ...props,
        children: [
            /*#__PURE__*/ (0, $byuoM$reactjsxruntime.jsx)("circle", {
                cx: 3,
                cy: 9,
                r: 1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $byuoM$reactjsxruntime.jsx)("circle", {
                cx: 3,
                cy: 5,
                r: 1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $byuoM$reactjsxruntime.jsx)("circle", {
                cx: 3,
                cy: 1,
                r: 1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $byuoM$reactjsxruntime.jsx)("circle", {
                cx: 7,
                cy: 9,
                r: 1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $byuoM$reactjsxruntime.jsx)("circle", {
                cx: 7,
                cy: 5,
                r: 1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $byuoM$reactjsxruntime.jsx)("circle", {
                cx: 7,
                cy: 1,
                r: 1,
                fill: "var(--iconPrimary, #222)"
            })
        ]
    });
var $88d6d23073bc9ab9$export$2e2bcd8739ae039 = $88d6d23073bc9ab9$var$SvgS2DragHandleSize100;
