// @react-spectrum/s2/dist/private/S2_DragHandleSize75.cjs
var $2GD9x$reactjsxruntime = require("react/jsx-runtime");
require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $68edbeacb9887ed7$export$2e2bcd8739ae039);


const $68edbeacb9887ed7$var$SvgS2DragHandleSize75 = (props)=>/*#__PURE__*/ (0, $2GD9x$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 10,
        height: 10,
        viewBox: "0 0 10 10",
        ...props,
        children: [
            /*#__PURE__*/ (0, $2GD9x$reactjsxruntime.jsx)("circle", {
                cx: 3.2,
                cy: 8.6,
                r: 0.9,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $2GD9x$reactjsxruntime.jsx)("circle", {
                cx: 3.2,
                cy: 5,
                r: 0.9,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $2GD9x$reactjsxruntime.jsx)("circle", {
                cx: 3.2,
                cy: 1.4,
                r: 0.9,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $2GD9x$reactjsxruntime.jsx)("circle", {
                cx: 6.8,
                cy: 8.6,
                r: 0.9,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $2GD9x$reactjsxruntime.jsx)("circle", {
                cx: 6.8,
                cy: 5,
                r: 0.9,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $2GD9x$reactjsxruntime.jsx)("circle", {
                cx: 6.8,
                cy: 1.4,
                r: 0.9,
                fill: "var(--iconPrimary, #222)"
            })
        ]
    });
var $68edbeacb9887ed7$export$2e2bcd8739ae039 = $68edbeacb9887ed7$var$SvgS2DragHandleSize75;
