// @react-spectrum/s2/dist/private/S2_DragHandleSize75.mjs
import {jsxs as $2MJfm$jsxs, jsx as $2MJfm$jsx} from "react/jsx-runtime";
import "react";



const $c6fba7d3aa542a55$var$SvgS2DragHandleSize75 = (props)=>/*#__PURE__*/ (0, $2MJfm$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 10,
        height: 10,
        viewBox: "0 0 10 10",
        ...props,
        children: [
            /*#__PURE__*/ (0, $2MJfm$jsx)("circle", {
                cx: 3.2,
                cy: 8.6,
                r: 0.9,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $2MJfm$jsx)("circle", {
                cx: 3.2,
                cy: 5,
                r: 0.9,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $2MJfm$jsx)("circle", {
                cx: 3.2,
                cy: 1.4,
                r: 0.9,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $2MJfm$jsx)("circle", {
                cx: 6.8,
                cy: 8.6,
                r: 0.9,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $2MJfm$jsx)("circle", {
                cx: 6.8,
                cy: 5,
                r: 0.9,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $2MJfm$jsx)("circle", {
                cx: 6.8,
                cy: 1.4,
                r: 0.9,
                fill: "var(--iconPrimary, #222)"
            })
        ]
    });
var $c6fba7d3aa542a55$export$2e2bcd8739ae039 = $c6fba7d3aa542a55$var$SvgS2DragHandleSize75;


export {$c6fba7d3aa542a55$export$2e2bcd8739ae039 as default};
