// @react-spectrum/s2/dist/private/S2_DragHandleSize200.mjs
import {jsxs as $1QzFU$jsxs, jsx as $1QzFU$jsx} from "react/jsx-runtime";
import "react";



const $1792142a9047937d$var$SvgS2DragHandleSize200 = (props)=>/*#__PURE__*/ (0, $1QzFU$jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 12,
        height: 12,
        viewBox: "0 0 12 12",
        ...props,
        children: [
            /*#__PURE__*/ (0, $1QzFU$jsx)("circle", {
                cx: 3.8,
                cy: 10.4,
                r: 1.1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $1QzFU$jsx)("circle", {
                cx: 3.8,
                cy: 6,
                r: 1.1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $1QzFU$jsx)("circle", {
                cx: 3.8,
                cy: 1.6,
                r: 1.1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $1QzFU$jsx)("circle", {
                cx: 8.2,
                cy: 10.4,
                r: 1.1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $1QzFU$jsx)("circle", {
                cx: 8.2,
                cy: 6,
                r: 1.1,
                fill: "var(--iconPrimary, #222)"
            }),
            /*#__PURE__*/ (0, $1QzFU$jsx)("circle", {
                cx: 8.2,
                cy: 1.6,
                r: 1.1,
                fill: "var(--iconPrimary, #222)"
            })
        ]
    });
var $1792142a9047937d$export$2e2bcd8739ae039 = $1792142a9047937d$var$SvgS2DragHandleSize200;


export {$1792142a9047937d$export$2e2bcd8739ae039 as default};
