// @react-spectrum/s2/dist/private/S2_MoveHorizontalTableWidget.cjs
var $fZCTT$reactjsxruntime = require("react/jsx-runtime");
require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "default", () => $23350f45c699333e$export$2e2bcd8739ae039);


const $23350f45c699333e$var$SvgS2MoveHorizontalTableWidget = (props)=>/*#__PURE__*/ (0, $fZCTT$reactjsxruntime.jsxs)("svg", {
        xmlns: "http://www.w3.org/2000/svg",
        width: 16,
        height: 16,
        viewBox: "0 0 16 16",
        ...props,
        children: [
            /*#__PURE__*/ (0, $fZCTT$reactjsxruntime.jsx)("circle", {
                cx: 8,
                cy: 8,
                r: 8
            }),
            /*#__PURE__*/ (0, $fZCTT$reactjsxruntime.jsx)("path", {
                fill: "var(--iconPrimary, #fff)",
                d: "M10 11.75a.75.75 0 0 1-.53-1.28L11.94 8 9.47 5.53a.75.75 0 1 1 1.06-1.06l3 3a.75.75 0 0 1 0 1.06l-3 3a.75.75 0 0 1-.53.22M6 11.75a.75.75 0 0 1-.53-.22l-3-3a.75.75 0 0 1 0-1.06l3-3a.75.75 0 1 1 1.06 1.06L4.06 8l2.47 2.47A.75.75 0 0 1 6 11.75"
            })
        ]
    });
var $23350f45c699333e$export$2e2bcd8739ae039 = $23350f45c699333e$var$SvgS2MoveHorizontalTableWidget;
