// @react-spectrum/s2/dist/private/SearchField.cjs
require("./SearchField.css");
var $e991cbcdf82ced71$exports = require("./CenterBaseline.cjs");
var $4fb579ecc195bd2b$exports = require("./ClearButton.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $bfa4962d90c8af48$exports = require("./Form.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $7218fc0a0ade65d7$exports = require("../../icons/Search.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $fsjBd$reactjsxruntime = require("react/jsx-runtime");
var $fsjBd$reactariacomponentsSearchField = require("react-aria-components/SearchField");
var $fsjBd$reactariacomponentsslots = require("react-aria-components/slots");
var $fsjBd$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "SearchFieldContext", () => $f0dfe14258307d4b$export$d1c4e4c63cb03a11);
$parcel$export(module.exports, "SearchField", () => $f0dfe14258307d4b$export$b94867ecbd698f21);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 











const $f0dfe14258307d4b$export$d1c4e4c63cb03a11 = /*#__PURE__*/ (0, $fsjBd$react.createContext)(null);
const $f0dfe14258307d4b$export$b94867ecbd698f21 = /*#__PURE__*/ (0, $fsjBd$react.forwardRef)(function SearchField(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $f0dfe14258307d4b$export$d1c4e4c63cb03a11);
    let formContext = (0, $fsjBd$react.useContext)((0, $bfa4962d90c8af48$exports.FormContext));
    props = (0, $bfa4962d90c8af48$exports.useFormProps)(props);
    let { label: label, description: description, errorMessage: errorMessage, necessityIndicator: necessityIndicator, labelPosition: labelPosition = 'top', labelAlign: labelAlign = 'start', UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, ...searchFieldProps } = props;
    let domRef = (0, $fsjBd$react.useRef)(null);
    let inputRef = (0, $fsjBd$react.useRef)(null);
    // Expose imperative interface for ref
    (0, $fsjBd$react.useImperativeHandle)(ref, ()=>({
            ...(0, $150382364eecc4bc$exports.createFocusableRef)(domRef, inputRef),
            select () {
                if (inputRef.current) inputRef.current.select();
            },
            getInputElement () {
                return inputRef.current;
            }
        }));
    return /*#__PURE__*/ (0, $fsjBd$reactjsxruntime.jsx)((0, $fsjBd$reactariacomponentsSearchField.SearchField), {
        ...searchFieldProps,
        ref: domRef,
        style: UNSAFE_style,
        className: UNSAFE_className + function anonymous(props, overrides) {
            let rules = " ";
            let gridColumnStart = false;
            let gridColumnEnd = false;
            let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
            for (let p of matches){
                if (p[1] === "z") gridColumnStart = true;
                if (p[1] === "y") gridColumnEnd = true;
                rules += p[0];
            }
            rules += ' se131';
            if (props.isInForm) {
                if (!gridColumnStart) rules += ' z2mU131';
            }
            if (props.isInForm) {
                if (props.labelPosition === "side") {
                    if (!gridColumnEnd) rules += ' yvoofG131';
                }
            }
            if (props.isInForm) rules += ' DLwlgod131';
            else {
                if (props.labelPosition === "side") rules += ' DPlN5qb131';
                else if (props.labelPosition === "top") rules += ' DM8Mfn131';
            }
            if (props.labelPosition === "side") rules += ' EPlN5qb131';
            else if (props.labelPosition === "top") rules += ' E4w1sLc131';
            if (props.labelPosition === "side") rules += ' CbRM95c131';
            else if (props.labelPosition === "top") rules += ' Cxaocre131';
            if (props.size === "XL") {
                rules += ' -_6BNtrc-e131';
                rules += ' vx131';
            } else if (props.size === "L") {
                rules += ' -_6BNtrc-d131';
                rules += ' vx131';
            } else if (props.size === "S") {
                rules += ' -_6BNtrc-b131';
                rules += ' vx131';
            } else if (props.size === "XS") {
                rules += ' -_6BNtrc-a131';
                rules += ' vx131';
            } else {
                rules += ' -_6BNtrc-c131';
                rules += ' vx131';
            }
            rules += ' ea131';
            rules += ' _Fd131';
            rules += ' _FnuYUweb131';
            if (props.size === "XL") rules += ' -BhX7R-Fz131';
            else if (props.size === "L") rules += ' -BhX7R-Fv131';
            else if (props.size === "S") rules += ' -BhX7R-Fp131';
            else if (props.size === "XS") rules += ' -BhX7R-Fn131';
            else rules += ' -BhX7R-Fx131';
            rules += ' -AJjOLd-UVkF0Db131';
            rules += ' qg131';
            rules += ' __ca131';
            rules += ' -Qg41Lb-JAxCW0c131';
            if (props.isDisabled) {
                rules += ' pp131';
                rules += ' plc131';
            } else {
                if (props.isPressed) rules += ' po131';
                else if (props.isFocusVisible) rules += ' po131';
                else if (props.isHovered) rules += ' po131';
                else rules += ' pt131';
            }
            return rules;
        }({
            size: props.size,
            labelPosition: labelPosition,
            isInForm: !!formContext
        }, props.styles),
        children: ({ isDisabled: isDisabled, isInvalid: isInvalid, isEmpty: isEmpty })=>/*#__PURE__*/ (0, $fsjBd$reactjsxruntime.jsxs)((0, $fsjBd$reactjsxruntime.Fragment), {
                children: [
                    label && /*#__PURE__*/ (0, $fsjBd$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldLabel), {
                        isDisabled: isDisabled,
                        isRequired: props.isRequired,
                        size: props.size,
                        labelPosition: labelPosition,
                        labelAlign: labelAlign,
                        necessityIndicator: necessityIndicator,
                        contextualHelp: props.contextualHelp,
                        children: label
                    }),
                    /*#__PURE__*/ (0, $fsjBd$reactjsxruntime.jsxs)((0, $a0b996a550bb0891$exports.FieldGroup), {
                        isDisabled: isDisabled,
                        size: props.size,
                        styles: " og131 ng131 kg131 jg131 Sw131 Rd131",
                        children: [
                            /*#__PURE__*/ (0, $fsjBd$reactjsxruntime.jsx)((0, $fsjBd$reactariacomponentsslots.Provider), {
                                values: [
                                    [
                                        (0, $bde97c91243ed164$exports.IconContext),
                                        {
                                            render: (0, $e991cbcdf82ced71$exports.centerBaseline)({
                                                slot: 'icon',
                                                styles: " _va131 Ht131 Hsu131 -_8sjo0b-t5ZbAob131"
                                            }),
                                            styles: " ZuUmpre131 FuUmpre131 I67logd131"
                                        }
                                    ]
                                ],
                                children: /*#__PURE__*/ (0, $fsjBd$reactjsxruntime.jsx)((0, $7218fc0a0ade65d7$exports.default), {})
                            }),
                            /*#__PURE__*/ (0, $fsjBd$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.Input), {
                                ref: inputRef,
                                UNSAFE_className: "zc3JFe"
                            }),
                            !isEmpty && !searchFieldProps.isReadOnly && /*#__PURE__*/ (0, $fsjBd$reactjsxruntime.jsx)((0, $4fb579ecc195bd2b$exports.ClearButton), {
                                size: props.size
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $fsjBd$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.HelpText), {
                        size: props.size,
                        isDisabled: isDisabled,
                        isInvalid: isInvalid,
                        description: description,
                        children: errorMessage
                    })
                ]
            })
    });
});
