// @react-spectrum/s2/dist/private/SegmentedControl.mjs
import "./SegmentedControl.css";
import {centerBaseline as $1f4b04be3f24aae3$export$9d7e2342a7e53afa} from "./CenterBaseline.mjs";
import {IconContext as $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed} from "../../icons/Icon.mjs";
import {pressScale as $10ea7662e51a285b$export$56e8cba416805d8d} from "./pressScale.mjs";
import {Text as $8e847109a6ab556d$export$5f1af8db9871e1d6, TextContext as $8e847109a6ab556d$export$9afb8bc826b033ea} from "./Content.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8, useFocusableRef as $80ecb974df713a30$export$96a734597687c040} from "./useDOMRef.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $ae3Tm$jsx, jsxs as $ae3Tm$jsxs, Fragment as $ae3Tm$Fragment} from "react/jsx-runtime";
import {Provider as $ae3Tm$Provider, DEFAULT_SLOT as $ae3Tm$DEFAULT_SLOT} from "react-aria-components/slots";
import {createContext as $ae3Tm$createContext, forwardRef as $ae3Tm$forwardRef, useContext as $ae3Tm$useContext, useRef as $ae3Tm$useRef, useCallback as $ae3Tm$useCallback} from "react";
import {TextContext as $ae3Tm$TextContext} from "react-aria-components/Text";
import {SelectionIndicator as $ae3Tm$SelectionIndicator} from "react-aria-components/SelectionIndicator";
import {ToggleButton as $ae3Tm$ToggleButton} from "react-aria-components/ToggleButton";
import {ToggleButtonGroup as $ae3Tm$ToggleButtonGroup, ToggleGroupStateContext as $ae3Tm$ToggleGroupStateContext} from "react-aria-components/ToggleButtonGroup";
import {useLayoutEffect as $ae3Tm$useLayoutEffect} from "react-aria/private/utils/useLayoutEffect";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 













const $c679c2152e03f8bf$export$c083c44d8a07dd35 = /*#__PURE__*/ (0, $ae3Tm$createContext)(null);
const $c679c2152e03f8bf$var$segmentedControl = function anonymous(props, overrides) {
    let rules = " ";
    let width = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "Z") width = true;
        rules += p[0];
    }
    rules += ' sd131';
    rules += ' Ub131';
    rules += ' qb131';
    rules += ' gH131';
    rules += ' ob131';
    rules += ' nb131';
    rules += ' kb131';
    rules += ' jb131';
    if (!width) rules += ' ZJ131';
    return rules;
};
const $c679c2152e03f8bf$var$controlItem = function anonymous(props, overrides) {
    let rules = " ";
    let minWidth = false;
    let position = false;
    let flexGrow = false;
    let flexBasis = false;
    let flexShrink = false;
    let zIndex = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "N") minWidth = true;
        if (p[1] === "_P") position = true;
        if (p[1] === "_u") flexGrow = true;
        if (p[1] === "_s") flexBasis = true;
        if (p[1] === "_v") flexShrink = true;
        if (p[1] === "_9") zIndex = true;
        rules += p[0];
    }
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isDisabled) rules += ' pp131';
    else if (props.isSelected) {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    } else {
        if (props.isPressed) rules += ' pt131';
        else if (props.isFocusVisible) rules += ' pt131';
        else if (props.isHovered) rules += ' pt131';
        else rules += ' pv131';
    }
    if (props.isSelected) rules += ' ple131';
    else if (props.isDisabled) rules += ' plc131';
    else rules += ' plb131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' _oa131';
    rules += ' Sv131';
    rules += ' SgLWx3bd131';
    rules += ' Rv131';
    rules += ' RgLWx3bd131';
    if (props.size === "XL") {
        if (!minWidth) rules += ' Nl131';
    } else if (props.size === "L") {
        if (!minWidth) rules += ' Nj131';
    } else if (props.size === "S") {
        if (!minWidth) rules += ' Nf131';
    } else if (props.size === "XS") {
        if (!minWidth) rules += ' Ne131';
    } else if (!minWidth) rules += ' Nk131';
    if (props.size === "XL") rules += ' -SFnbic-_JaP8ec131';
    else if (props.size === "L") rules += ' -SFnbic-_J0K2hn131';
    else if (props.size === "S") rules += ' -SFnbic-_JJOW22b131';
    else if (props.size === "XS") rules += ' -SFnbic-_JICefdd131';
    else rules += ' -SFnbic-_Jb131';
    rules += ' -Bu7z1c-ob131';
    rules += ' oj131';
    rules += ' nj131';
    rules += ' kj131';
    rules += ' jj131';
    rules += ' qc131';
    rules += ' qsd131';
    rules += ' -Qg41Lb-IAxCW0c131';
    rules += ' -Qg41Lb-gLWx3bIy131';
    rules += ' FRuFGYc131';
    rules += ' _Ca131';
    if (!position) rules += ' _Pc131';
    rules += ' _yb131';
    if (props.isJustified) {
        if (!flexGrow) rules += ' _ub131';
    }
    if (props.isJustified) {
        if (!flexBasis) rules += ' _sb131';
    }
    if (!flexShrink) rules += ' _va131';
    rules += ' _7c131';
    rules += ' __ca131';
    rules += ' _4c131';
    rules += ' g9131';
    rules += ' _je131';
    rules += ' -_8sjo0b-t5ZbAob131';
    if (props.isSelected) {
        if (!zIndex) rules += ' _9b131';
    } else if (!zIndex) rules += ' _9c131';
    if (props.size === "XL") rules += ' -F_-Fz131';
    else if (props.size === "L") rules += ' -F_-Fv131';
    else if (props.size === "S") rules += ' -F_-Fp131';
    else if (props.size === "XS") rules += ' -F_-Fn131';
    else rules += ' -F_-Fx131';
    return rules;
};
const $c679c2152e03f8bf$var$slider = function anonymous(props) {
    let rules = " ";
    rules += ' gX131';
    if (props.isDisabled) rules += ' gld131';
    else rules += ' gle131';
    rules += ' Wr131';
    rules += ' _Er131';
    rules += ' Za131';
    rules += ' Fb131';
    rules += ' _qg131';
    rules += ' Yx6BKnd131';
    rules += ' Ydklsgce131';
    rules += ' Xc131';
    rules += ' _2a131';
    rules += ' _Pa131';
    rules += ' _oa131';
    rules += ' _jf131';
    rules += ' _kc131';
    rules += ' hc131';
    rules += ' mc131';
    rules += ' lc131';
    if (props.isDisabled) rules += ' iq131';
    else rules += ' in131';
    if (props.isDisabled) rules += ' ilc131';
    else rules += ' ild131';
    rules += ' oc131';
    rules += ' nc131';
    rules += ' kc131';
    rules += ' jc131';
    return rules;
};
const $c679c2152e03f8bf$var$InternalSegmentedControlContext = /*#__PURE__*/ (0, $ae3Tm$createContext)({});
const $c679c2152e03f8bf$export$668709c620d0b8e2 = /*#__PURE__*/ (0, $ae3Tm$forwardRef)(function SegmentedControl(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $c679c2152e03f8bf$export$c083c44d8a07dd35);
    let { defaultSelectedKey: defaultSelectedKey, selectedKey: selectedKey, onSelectionChange: onSelectionChange } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let onChange = (values)=>{
        if (onSelectionChange) {
            let firstKey = values.values().next().value;
            if (firstKey != null) onSelectionChange(firstKey);
        }
    };
    return /*#__PURE__*/ (0, $ae3Tm$jsx)((0, $ae3Tm$ToggleButtonGroup), {
        ...props,
        selectedKeys: selectedKey != null ? [
            selectedKey
        ] : undefined,
        defaultSelectedKeys: defaultSelectedKey != null ? [
            defaultSelectedKey
        ] : undefined,
        disallowEmptySelection: true,
        ref: domRef,
        orientation: "horizontal",
        style: props.UNSAFE_style,
        onSelectionChange: onChange,
        className: (props.UNSAFE_className || '') + $c679c2152e03f8bf$var$segmentedControl(null, props.styles),
        "aria-label": props['aria-label'],
        children: /*#__PURE__*/ (0, $ae3Tm$jsx)($c679c2152e03f8bf$var$DefaultSelectionTracker, {
            defaultValue: defaultSelectedKey,
            value: selectedKey,
            isJustified: props.isJustified,
            children: props.children
        })
    });
});
function $c679c2152e03f8bf$var$DefaultSelectionTracker(props) {
    let state = (0, $ae3Tm$useContext)((0, $ae3Tm$ToggleGroupStateContext));
    let isRegistered = (0, $ae3Tm$useRef)(!(props.defaultValue == null && props.value == null));
    // default select the first available item
    let register = (0, $ae3Tm$useCallback)((value)=>{
        if (state && !isRegistered.current) {
            isRegistered.current = true;
            state.toggleKey(value);
        }
    // eslint-disable-next-line react-hooks/exhaustive-deps
    }, []);
    return /*#__PURE__*/ (0, $ae3Tm$jsx)((0, $ae3Tm$Provider), {
        values: [
            [
                $c679c2152e03f8bf$var$InternalSegmentedControlContext,
                {
                    register: register,
                    isJustified: props.isJustified
                }
            ]
        ],
        children: props.children
    });
}
const $c679c2152e03f8bf$export$4fc12f2742e8cf3f = /*#__PURE__*/ (0, $ae3Tm$forwardRef)(function SegmentedControlItem(props, ref) {
    let domRef = (0, $80ecb974df713a30$export$96a734597687c040)(ref);
    let divRef = (0, $ae3Tm$useRef)(null);
    let { register: register, isJustified: isJustified } = (0, $ae3Tm$useContext)($c679c2152e03f8bf$var$InternalSegmentedControlContext);
    (0, $ae3Tm$useLayoutEffect)(()=>{
        register?.(props.id);
    }, [
        register,
        props.id
    ]);
    return /*#__PURE__*/ (0, $ae3Tm$jsx)((0, $ae3Tm$ToggleButton), {
        ...props,
        ref: domRef,
        style: props.UNSAFE_style,
        className: (renderProps)=>(props.UNSAFE_className || '') + $c679c2152e03f8bf$var$controlItem({
                ...renderProps,
                isJustified: isJustified
            }, props.styles),
        children: ({ isPressed: isPressed, isDisabled: isDisabled })=>/*#__PURE__*/ (0, $ae3Tm$jsxs)((0, $ae3Tm$Fragment), {
                children: [
                    /*#__PURE__*/ (0, $ae3Tm$jsx)((0, $ae3Tm$SelectionIndicator), {
                        className: $c679c2152e03f8bf$var$slider({
                            isDisabled: isDisabled
                        })
                    }),
                    /*#__PURE__*/ (0, $ae3Tm$jsx)((0, $ae3Tm$Provider), {
                        values: [
                            [
                                (0, $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed),
                                {
                                    render: (0, $1f4b04be3f24aae3$export$9d7e2342a7e53afa)({
                                        slot: 'icon',
                                        styles: " _Ja131 _va131"
                                    })
                                }
                            ],
                            [
                                (0, $ae3Tm$TextContext),
                                {
                                    slots: {
                                        [(0, $ae3Tm$DEFAULT_SLOT)]: {}
                                    }
                                }
                            ],
                            [
                                (0, $8e847109a6ab556d$export$9afb8bc826b033ea),
                                {
                                    styles: " _Jb131 _Nc131 Pc131 _Yb131 _7c131"
                                }
                            ]
                        ],
                        children: /*#__PURE__*/ (0, $ae3Tm$jsx)("div", {
                            ref: divRef,
                            style: (0, $10ea7662e51a285b$export$56e8cba416805d8d)(divRef)({
                                isPressed: isPressed
                            }),
                            className: " sd131 Uc131 Usd131 qc131 qsd131 Yd131 Xb131 _2b131 eb131 Na131",
                            children: typeof props.children === 'string' ? /*#__PURE__*/ (0, $ae3Tm$jsx)((0, $8e847109a6ab556d$export$5f1af8db9871e1d6), {
                                children: props.children
                            }) : props.children
                        })
                    })
                ]
            })
    });
});


export {$c679c2152e03f8bf$export$c083c44d8a07dd35 as SegmentedControlContext, $c679c2152e03f8bf$export$668709c620d0b8e2 as SegmentedControl, $c679c2152e03f8bf$export$4fc12f2742e8cf3f as SegmentedControlItem};
