// @react-spectrum/s2/dist/private/SelectBoxGroup.mjs
import "./SelectBoxGroup.css";
import {box as $9b405e0f8095dce0$export$827f4ee28efc37, iconStyles as $9b405e0f8095dce0$export$2c99b98b77550cb4} from "./Checkbox.mjs";
import $81cf3d4588736e7d$export$2e2bcd8739ae039 from "./Checkmark.mjs";
import {IllustrationContext as $ac8c32e6775ddd1f$export$207c7561471de4bd} from "../../icons/Icon.mjs";
import {pressScale as $10ea7662e51a285b$export$56e8cba416805d8d} from "./pressScale.mjs";
import {TextContext as $8e847109a6ab556d$export$9afb8bc826b033ea} from "./Content.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $hsEDv$jsx, jsxs as $hsEDv$jsxs, Fragment as $hsEDv$Fragment} from "react/jsx-runtime";
import {Provider as $hsEDv$Provider, DEFAULT_SLOT as $hsEDv$DEFAULT_SLOT} from "react-aria-components/slots";
import {ListBoxItem as $hsEDv$ListBoxItem, ListBox as $hsEDv$ListBox} from "react-aria-components/ListBox";
import {createContext as $hsEDv$createContext, useContext as $hsEDv$useContext, useRef as $hsEDv$useRef, forwardRef as $hsEDv$forwardRef, useMemo as $hsEDv$useMemo} from "react";
import {useFocusVisible as $hsEDv$useFocusVisible} from "react-aria/useFocusVisible";

/*
 * Copyright 2025 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 










const $e1ab128dc6578df7$var$SelectBoxContext = /*#__PURE__*/ (0, $hsEDv$createContext)({
    orientation: 'vertical'
});
const $e1ab128dc6578df7$export$cad91f7528e8dc08 = /*#__PURE__*/ (0, $hsEDv$createContext)(null);
const $e1ab128dc6578df7$var$labelOnly = ':has([slot=label]):not(:has([slot=description]))';
const $e1ab128dc6578df7$var$noIllustration = ':not(:has([slot=illustration]))';
const $e1ab128dc6578df7$var$selectBoxStyles = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let width = false;
    let minWidth = false;
    let maxWidth = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "Z") width = true;
        if (p[1] === "N") minWidth = true;
        if (p[1] === "L") maxWidth = true;
        rules += p[0];
    }
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    rules += ' se131';
    rules += ' __yM8Mfn131';
    if (!position) rules += ' _Pc131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-c131';
    rules += ' vx131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    if (props.isDisabled) rules += ' pp131';
    rules += ' ri131';
    rules += ' _oa131';
    rules += ' _Nc131';
    rules += ' Pc131';
    if (props.orientation === "horizontal") {
        if (!width) rules += ' ZAK24ue131';
    } else if (!width) rules += ' Z75oWv131';
    if (props.orientation === "horizontal") rules += ' Fd131';
    else rules += ' F75oWv131';
    if (props.orientation === "horizontal") {
        if (!minWidth) rules += ' NBzUiud131';
    } else if (!minWidth) rules += ' NQGV3L131';
    if (props.orientation === "horizontal") rules += ' -_5yLdZ-ZILqIYb131';
    else rules += ' -_5yLdZ-Z75oWv131';
    if (!maxWidth) rules += ' Lqz07Zc131';
    if (props.orientation === "horizontal") rules += ' MKniM131';
    else rules += ' MQGV3L131';
    if (props.orientation === "horizontal") rules += ' KDvO7ld131';
    else rules += ' K75oWv131';
    if (props.orientation === "horizontal") rules += ' Tf131';
    else rules += ' Th131';
    if (props.orientation === "horizontal") rules += ' Qf131';
    else rules += ' Qh131';
    if (props.orientation === "horizontal") rules += ' Sk131';
    if (props.orientation === "horizontal") rules += ' Rh131';
    if (props.orientation === "horizontal") {
        rules += ' CtXiS9b131';
        rules += ' CbWiMddxbX3ib131';
    } else if (props.orientation === "vertical") rules += ' CfwsTtd131';
    if (props.orientation === "horizontal") {
        rules += ' E5HJqj131';
        rules += ' E1099pdfYMhHd131';
    } else if (props.orientation === "vertical") rules += ' EY7ZLHe131';
    if (props.orientation === "horizontal") rules += ' DJSAqje131';
    if (props.orientation === "vertical") rules += ' _cb131';
    rules += ' oc131';
    rules += ' nc131';
    rules += ' kc131';
    rules += ' jc131';
    rules += ' _jf131';
    if (props.isDisabled) rules += ' iA131';
    else if (props.isSelected) rules += ' in131';
    else rules += ' iA131';
    if (props.isDisabled) rules += ' gH131';
    else rules += ' g_g131';
    if (props.isDisabled) rules += ' _nd131';
    else if (props.isSelected) rules += ' _nLeasBb131';
    else if (props.isHovered) rules += ' _nLeasBb131';
    else rules += ' _n4zCLMe131';
    rules += ' _kc131';
    rules += ' hc131';
    rules += ' mc131';
    rules += ' lc131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    return rules;
};
const $e1ab128dc6578df7$var$illustrationContainer = function anonymous(props) {
    let rules = " ";
    rules += ' zTZ8xtd131';
    rules += ' yTZ8xtd131';
    rules += ' BTZ8xtd131';
    rules += ' ATZ8xtd131';
    rules += ' _dc131';
    rules += ' __Ab131';
    rules += ' Nl131';
    rules += ' Mk131';
    if (props.isDisabled) rules += ' -_8sjo0b-pp131';
    else {
        if (props.isPressed) rules += ' -_8sjo0b-po131';
        else if (props.isFocusVisible) rules += ' -_8sjo0b-po131';
        else if (props.isHovered) rules += ' -_8sjo0b-po131';
        else rules += ' -_8sjo0b-pt131';
    }
    return rules;
};
const $e1ab128dc6578df7$var$descriptionText = function anonymous(props) {
    let rules = " ";
    rules += ' z9u7zPc131';
    rules += ' y9u7zPc131';
    rules += ' B9u7zPc131';
    rules += ' A9u7zPc131';
    rules += ' _dc131';
    if (props.orientation === "vertical") rules += ' sk131';
    else rules += ' sb131';
    rules += ' _Nc131';
    rules += ' Pc131';
    if (props.orientation === "horizontal") rules += ' _Wd131';
    else rules += ' _Wa131';
    if (props.isDisabled) rules += ' pp131';
    else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    }
    return rules;
};
const $e1ab128dc6578df7$var$labelText = function anonymous(props) {
    let rules = " ";
    rules += ' zv6x9r131';
    rules += ' yv6x9r131';
    rules += ' Bv6x9r131';
    rules += ' Av6x9r131';
    rules += ' _dc131';
    if (props.orientation === "horizontal") rules += ' __Ad131';
    else rules += ' __Ab131';
    rules += ' Za131';
    rules += ' _Nc131';
    rules += ' Pc131';
    rules += ' Na131';
    if (props.orientation === "horizontal") rules += ' _Wd131';
    else rules += ' _Wa131';
    rules += ' _7c131';
    rules += ' _Yb131';
    if (props.orientation === "horizontal") {
        rules += ' xd131';
        rules += ' _xa131';
        rules += ' xX0cczbc131';
        rules += ' _xX0cczba131';
    }
    if (props.isDisabled) rules += ' pp131';
    else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    }
    return rules;
};
const $e1ab128dc6578df7$var$gridStyles = function anonymous(props, overrides) {
    let rules = " ";
    rules += (String(overrides || '').match(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g) || []).join('');
    rules += ' se131';
    rules += ' __yM8Mfn131';
    rules += ' Uj131';
    rules += ' qj131';
    rules += ' _Ca131';
    if (props.orientation === "vertical") rules += ' -kN2jP-Z75oWv131';
    else if (props.orientation === "horizontal") rules += ' -kN2jP-ZAK24ue131';
    if (props.orientation === "vertical") rules += ' -LWGJg-ZQGV3L131';
    else if (props.orientation === "horizontal") rules += ' -LWGJg-ZBzUiud131';
    if (props.orientation === "vertical") rules += ' DnG8gle131';
    else if (props.orientation === "horizontal") rules += ' DnG8gle131';
    return rules;
};
function $e1ab128dc6578df7$export$ebde6d4e7862fe9d(props) {
    let { children: children, isDisabled: individualDisabled = false, UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className, styles: styles, ...otherProps } = props;
    let { orientation: orientation = 'vertical', isDisabled: groupDisabled = false } = (0, $hsEDv$useContext)($e1ab128dc6578df7$var$SelectBoxContext);
    const isDisabled = individualDisabled || groupDisabled;
    const ref = (0, $hsEDv$useRef)(null);
    let { isFocusVisible: isFocusVisible } = (0, $hsEDv$useFocusVisible)();
    return /*#__PURE__*/ (0, $hsEDv$jsx)((0, $hsEDv$ListBoxItem), {
        isDisabled: isDisabled,
        ref: ref,
        className: (renderProps)=>(UNSAFE_className || '') + $e1ab128dc6578df7$var$selectBoxStyles({
                ...renderProps,
                isFocusVisible: isFocusVisible && renderProps.isFocused,
                orientation: orientation
            }, styles),
        style: (0, $10ea7662e51a285b$export$56e8cba416805d8d)(ref, UNSAFE_style),
        ...otherProps,
        children: ({ isSelected: isSelected, isDisabled: isDisabled, isHovered: isHovered, selectionMode: selectionMode })=>{
            return /*#__PURE__*/ (0, $hsEDv$jsxs)((0, $hsEDv$Fragment), {
                children: [
                    /*#__PURE__*/ (0, $hsEDv$jsx)("div", {
                        className: " _Pa131 WJ131 _AI131 _Ob131",
                        "aria-hidden": "true",
                        children: !isDisabled && selectionMode === 'multiple' && /*#__PURE__*/ (0, $hsEDv$jsx)("div", {
                            className: (0, $9b405e0f8095dce0$export$827f4ee28efc37)({
                                isSelected: isSelected,
                                isDisabled: isDisabled,
                                size: 'M'
                            }),
                            children: /*#__PURE__*/ (0, $hsEDv$jsx)((0, $81cf3d4588736e7d$export$2e2bcd8739ae039), {
                                size: "S",
                                className: (0, $9b405e0f8095dce0$export$2c99b98b77550cb4)
                            })
                        })
                    }),
                    /*#__PURE__*/ (0, $hsEDv$jsx)((0, $hsEDv$Provider), {
                        values: [
                            [
                                (0, $ac8c32e6775ddd1f$export$207c7561471de4bd),
                                {
                                    size: 'S',
                                    styles: $e1ab128dc6578df7$var$illustrationContainer({
                                        size: 'S',
                                        orientation: orientation,
                                        isDisabled: isDisabled,
                                        isHovered: isHovered
                                    })
                                }
                            ],
                            [
                                (0, $8e847109a6ab556d$export$9afb8bc826b033ea),
                                {
                                    slots: {
                                        [(0, $hsEDv$DEFAULT_SLOT)]: {
                                            styles: $e1ab128dc6578df7$var$labelText({
                                                orientation: orientation,
                                                isDisabled: isDisabled,
                                                isHovered: isHovered
                                            })
                                        },
                                        label: {
                                            styles: $e1ab128dc6578df7$var$labelText({
                                                orientation: orientation,
                                                isDisabled: isDisabled,
                                                isHovered: isHovered
                                            })
                                        },
                                        description: {
                                            styles: $e1ab128dc6578df7$var$descriptionText({
                                                orientation: orientation,
                                                isDisabled: isDisabled,
                                                isHovered: isHovered
                                            })
                                        }
                                    }
                                }
                            ]
                        ],
                        children: children
                    })
                ]
            });
        }
    });
}
const $e1ab128dc6578df7$export$da87cf13fc2ef3d8 = /*#__PURE__*/ (0, $hsEDv$forwardRef)(function SelectBoxGroup(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $e1ab128dc6578df7$export$cad91f7528e8dc08);
    let { children: children, selectionMode: selectionMode = 'single', orientation: orientation = 'vertical', isDisabled: isDisabled = false, UNSAFE_className: UNSAFE_className, UNSAFE_style: UNSAFE_style, styles: styles, ...otherProps } = props;
    const selectBoxContextValue = (0, $hsEDv$useMemo)(()=>({
            orientation: orientation,
            isDisabled: isDisabled
        }), [
        orientation,
        isDisabled
    ]);
    return /*#__PURE__*/ (0, $hsEDv$jsx)($e1ab128dc6578df7$var$SelectBoxContext.Provider, {
        value: selectBoxContextValue,
        children: /*#__PURE__*/ (0, $hsEDv$jsx)((0, $hsEDv$ListBox), {
            selectionMode: selectionMode,
            layout: "grid",
            orientation: orientation,
            className: (UNSAFE_className || '') + $e1ab128dc6578df7$var$gridStyles({
                orientation: orientation
            }, styles),
            style: UNSAFE_style,
            ...otherProps,
            children: children
        })
    });
});


export {$e1ab128dc6578df7$export$cad91f7528e8dc08 as SelectBoxGroupContext, $e1ab128dc6578df7$export$ebde6d4e7862fe9d as SelectBox, $e1ab128dc6578df7$export$da87cf13fc2ef3d8 as SelectBoxGroup};
