// @react-spectrum/s2/dist/private/Slider.cjs
require("./Slider.css");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $bfa4962d90c8af48$exports = require("./Form.cjs");
var $308b180f49d82d28$exports = require("../../icons/runtime.cjs");
var $2061c83559b50a66$exports = require("./pressScale.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $5DeaH$reactjsxruntime = require("react/jsx-runtime");
var $5DeaH$reactariacomponentsSlider = require("react-aria-components/Slider");
var $5DeaH$react = require("react");
var $5DeaH$reactariaI18nProvider = require("react-aria/I18nProvider");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "SliderContext", () => $e937566cb5c6349c$export$e99164f0030f3bff);
$parcel$export(module.exports, "track", () => $e937566cb5c6349c$export$6b2a7d5132615086);
$parcel$export(module.exports, "thumbContainer", () => $e937566cb5c6349c$export$7814bee2738c4c3);
$parcel$export(module.exports, "thumbHitArea", () => $e937566cb5c6349c$export$f120a3c6038107a5);
$parcel$export(module.exports, "thumb", () => $e937566cb5c6349c$export$becc9140d0e846f8);
$parcel$export(module.exports, "upperTrack", () => $e937566cb5c6349c$export$1e1dc99b6e2204f1);
$parcel$export(module.exports, "filledTrack", () => $e937566cb5c6349c$export$847ceef3bc944d02);
$parcel$export(module.exports, "SliderBase", () => $e937566cb5c6349c$export$9418495bb635ebde);
$parcel$export(module.exports, "Slider", () => $e937566cb5c6349c$export$472062a354075cee);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 









const $e937566cb5c6349c$export$e99164f0030f3bff = /*#__PURE__*/ (0, $5DeaH$react.createContext)(null);
const $e937566cb5c6349c$var$slider = function anonymous(props, overrides) {
    let rules = " ";
    rules += (String(overrides || '').match(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g) || []).join('');
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.isDisabled) rules += ' pp131';
    else {
        rules += ' pv131';
        rules += ' plb131';
    }
    if (props.labelPosition === "side") rules += ' eb131';
    if (props.isInForm) rules += ' qg131';
    else if (props.size === "XL") rules += ' qj131';
    else if (props.size === "L") rules += ' qi131';
    else if (props.size === "M") rules += ' ql131';
    else if (props.size === "S") rules += ' ql131';
    return rules;
};
const $e937566cb5c6349c$var$labelContainer = function anonymous(props) {
    let rules = " ";
    if (props.labelPosition === "top") rules += ' se131';
    rules += ' zv6x9r131';
    rules += ' yv6x9r131';
    rules += ' Bv6x9r131';
    rules += ' Av6x9r131';
    rules += ' Za131';
    if (props.labelPosition === "top") rules += ' C09aAFd131';
    if (props.labelPosition === "top") rules += ' Dxumf9b131';
    if (props.labelPosition === "side") {
        if (props.labelAlign === "end") rules += ' _Wb131';
        else if (props.labelAlign === "start") rules += ' _Wd131';
    }
    rules += ' -AJjOLd-Qd131';
    return rules;
};
const $e937566cb5c6349c$var$output = function anonymous(props) {
    let rules = " ";
    rules += ' zChk6v131';
    rules += ' yChk6v131';
    rules += ' BChk6v131';
    rules += ' AChk6v131';
    if (props.labelPosition === "side") {
        if (props.isInForm) rules += ' _Wb131';
        else if (props.direction === "rtl") rules += ' _Wb131';
        else if (props.direction === "ltr") rules += ' _Wd131';
    } else if (props.labelPosition === "top") {
        if (props.direction === "rtl") rules += ' _Wd131';
        else if (props.direction === "ltr") rules += ' _Wb131';
    }
    return rules;
};
let $e937566cb5c6349c$export$6b2a7d5132615086 = function anonymous(props) {
    let rules = " ";
    rules += ' zCPUPs131';
    rules += ' yCPUPs131';
    rules += ' BCPUPs131';
    rules += ' ACPUPs131';
    rules += ' _Pc131';
    rules += ' Za131';
    if (props.size === "XL") rules += ' Fz131';
    else if (props.size === "L") rules += ' Fv131';
    else if (props.size === "M") rules += ' Fx131';
    else if (props.size === "S") rules += ' Fp131';
    return rules;
};
let $e937566cb5c6349c$export$7814bee2738c4c3 = function anonymous(props) {
    let rules = " ";
    if (props.size === "XL") rules += ' Zo131';
    else if (props.size === "L") rules += ' Zn131';
    else if (props.size === "M") rules += ' Zm131';
    else if (props.size === "S") rules += ' Zl131';
    if (props.size === "XL") rules += ' Fp131';
    else if (props.size === "L") rules += ' Fo131';
    else if (props.size === "M") rules += ' Fn131';
    else if (props.size === "S") rules += ' Fm131';
    rules += ' sg131';
    rules += ' _Pa131';
    rules += ' WF131';
    return rules;
};
let $e937566cb5c6349c$export$f120a3c6038107a5 = function anonymous(props) {
    let rules = " ";
    if (props.thumbStyle === "precise") {
        if (props.size === "XL") rules += ' Zp131';
        else if (props.size === "L") rules += ' Zo131';
        else if (props.size === "M") rules += ' Zn131';
        else if (props.size === "S") rules += ' Zm131';
    } else if (props.thumbStyle === "default") {
        if (props.size === "XL") rules += ' Zo131';
        else if (props.size === "L") rules += ' Zn131';
        else if (props.size === "M") rules += ' Zm131';
        else if (props.size === "S") rules += ' Zl131';
    }
    if (props.thumbStyle === "precise") {
        if (props.size === "XL") rules += ' Fq131';
        else if (props.size === "L") rules += ' Fp131';
        else if (props.size === "M") rules += ' Fo131';
        else if (props.size === "S") rules += ' Fn131';
    } else if (props.thumbStyle === "default") {
        if (props.size === "XL") rules += ' Fp131';
        else if (props.size === "L") rules += ' Fo131';
        else if (props.size === "M") rules += ' Fn131';
        else if (props.size === "S") rules += ' Fm131';
    }
    return rules;
};
let $e937566cb5c6349c$export$becc9140d0e846f8 = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    rules += ' sg131';
    rules += ' _oa131';
    rules += ' _Pa131';
    rules += ' WF131';
    rules += ' _EF131';
    rules += ' _Zd131';
    if (props.thumbStyle === "precise") rules += ' Zg131';
    else if (props.thumbStyle === "default") {
        if (props.size === "XL") rules += ' Zo131';
        else if (props.size === "L") rules += ' Zn131';
        else if (props.size === "M") rules += ' Zm131';
        else if (props.size === "S") rules += ' Zl131';
    }
    if (props.thumbStyle === "precise") {
        if (props.size === "XL") rules += ' Fq131';
        else if (props.size === "L") rules += ' Fp131';
        else if (props.size === "M") rules += ' Fo131';
        else if (props.size === "S") rules += ' Fn131';
    } else if (props.thumbStyle === "default") {
        if (props.size === "XL") rules += ' Fp131';
        else if (props.size === "L") rules += ' Fo131';
        else if (props.size === "M") rules += ' Fn131';
        else if (props.size === "S") rules += ' Fm131';
    }
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    rules += ' _jf131';
    rules += ' _kc131';
    rules += ' hc131';
    rules += ' mc131';
    rules += ' lc131';
    if (props.isDisabled) rules += ' iq131';
    else if (props.isDragging) rules += ' in131';
    else if (props.isHovered) rules += ' in131';
    else rules += ' iv131';
    if (props.isDisabled) rules += ' ilc131';
    else rules += ' ila131';
    rules += ' gX131';
    rules += ' glb131';
    return rules;
};
const $e937566cb5c6349c$var$trackStyling = {
    height: {
        trackStyle: {
            thin: 4,
            thick: 16
        }
    },
    top: '50%',
    borderRadius: {
        trackStyle: {
            thin: 'lg',
            thick: 'sm'
        }
    }
};
let $e937566cb5c6349c$export$1e1dc99b6e2204f1 = function anonymous(props) {
    let rules = " ";
    if (props.trackStyle === "thick") rules += ' Ft131';
    else if (props.trackStyle === "thin") rules += ' Fe131';
    rules += ' WF131';
    if (props.trackStyle === "thick") rules += ' oa131';
    else if (props.trackStyle === "thin") rules += ' oc131';
    if (props.trackStyle === "thick") rules += ' na131';
    else if (props.trackStyle === "thin") rules += ' nc131';
    if (props.trackStyle === "thick") rules += ' ka131';
    else if (props.trackStyle === "thin") rules += ' kc131';
    if (props.trackStyle === "thick") rules += ' ja131';
    else if (props.trackStyle === "thin") rules += ' jc131';
    rules += ' _Pa131';
    if (props.isDisabled) rules += ' gH131';
    else {
        if (props.isStaticColor) rules += ' gh131';
        else rules += ' gE131';
        rules += ' glb131';
    }
    rules += ' -_8PloMd-m131';
    rules += ' __Ya131';
    rules += ' Za131';
    rules += ' _oa131';
    rules += ' _Lf131';
    rules += ' _MEjZtrc131';
    rules += ' _KnkFfic131';
    rules += ' Oj131';
    if (props.isDisabled) rules += ' OldZOE7131';
    else rules += ' Olb131';
    return rules;
};
let $e937566cb5c6349c$export$847ceef3bc944d02 = function anonymous(props) {
    let rules = " ";
    if (props.trackStyle === "thick") rules += ' Ft131';
    else if (props.trackStyle === "thin") rules += ' Fe131';
    rules += ' WF131';
    if (props.trackStyle === "thick") rules += ' oa131';
    else if (props.trackStyle === "thin") rules += ' oc131';
    if (props.trackStyle === "thick") rules += ' na131';
    else if (props.trackStyle === "thin") rules += ' nc131';
    if (props.trackStyle === "thick") rules += ' ka131';
    else if (props.trackStyle === "thin") rules += ' kc131';
    if (props.trackStyle === "thick") rules += ' ja131';
    else if (props.trackStyle === "thin") rules += ' jc131';
    rules += ' _Pa131';
    if (props.isDisabled) rules += ' gH131';
    else if (props.isEmphasized) rules += ' g3131';
    else rules += ' g_i131';
    if (props.isDisabled) rules += ' gld131';
    else rules += ' gle131';
    rules += ' _oa131';
    rules += ' _jf131';
    rules += ' _kEjZtrc131';
    rules += ' hEjZtrc131';
    rules += ' mEjZtrc131';
    rules += ' lEjZtrc131';
    rules += ' iA131';
    if (props.isDisabled) rules += ' ilc131';
    else rules += ' ilb131';
    rules += ' -_8PloMd-m131';
    rules += ' __Ya131';
    return rules;
};
function $e937566cb5c6349c$export$9418495bb635ebde(props1) {
    let formContext = (0, $5DeaH$react.useContext)((0, $bfa4962d90c8af48$exports.FormContext));
    props1 = (0, $bfa4962d90c8af48$exports.useFormProps)(props1);
    let { label: label, labelPosition: labelPosition = 'top', labelAlign: labelAlign = 'start', size: size = 'M', minValue: minValue = 0, maxValue: maxValue = 100 } = props1;
    let { direction: direction } = (0, $5DeaH$reactariaI18nProvider.useLocale)();
    return /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsx)((0, $5DeaH$reactariacomponentsSlider.Slider), {
        ...props1,
        ref: props1.sliderRef,
        className: (renderProps)=>(props1.UNSAFE_className || '') + (0, $308b180f49d82d28$exports.mergeStyles)(function anonymous(props) {
                let rules = " ";
                rules += ' se131';
                if (props.isInForm) rules += ' z2mU131';
                if (props.isInForm) {
                    if (props.labelPosition === "side") rules += ' yvoofG131';
                }
                if (props.isInForm) rules += ' DLwlgod131';
                else {
                    if (props.labelPosition === "side") rules += ' DPlN5qb131';
                    else if (props.labelPosition === "top") rules += ' DM8Mfn131';
                }
                if (props.labelPosition === "side") rules += ' EPlN5qb131';
                else if (props.labelPosition === "top") rules += ' E4w1sLc131';
                if (props.labelPosition === "side") rules += ' CbRM95c131';
                else if (props.labelPosition === "top") rules += ' Cxaocre131';
                if (props.size === "XL") {
                    rules += ' -_6BNtrc-e131';
                    rules += ' vx131';
                } else if (props.size === "L") {
                    rules += ' -_6BNtrc-d131';
                    rules += ' vx131';
                } else if (props.size === "S") {
                    rules += ' -_6BNtrc-b131';
                    rules += ' vx131';
                } else if (props.size === "XS") {
                    rules += ' -_6BNtrc-a131';
                    rules += ' vx131';
                } else {
                    rules += ' -_6BNtrc-c131';
                    rules += ' vx131';
                }
                rules += ' ea131';
                rules += ' _Fd131';
                rules += ' _FnuYUweb131';
                if (props.size === "XL") rules += ' -BhX7R-Fz131';
                else if (props.size === "L") rules += ' -BhX7R-Fv131';
                else if (props.size === "S") rules += ' -BhX7R-Fp131';
                else if (props.size === "XS") rules += ' -BhX7R-Fn131';
                else rules += ' -BhX7R-Fx131';
                rules += ' -AJjOLd-UVkF0Db131';
                rules += ' qg131';
                rules += ' __ca131';
                return rules;
            }({
                labelPosition: labelPosition,
                isInForm: !!formContext
            }), $e937566cb5c6349c$var$slider({
                ...renderProps,
                labelPosition: labelPosition,
                size: size,
                isInForm: !!formContext
            }, props1.styles)),
        children: ({ state: state })=>{
            let maxLabelLength = 0;
            switch(state.values.length){
                case 1:
                    maxLabelLength = Math.max([
                        ...state.getFormattedValue(minValue)
                    ].length, [
                        ...state.getFormattedValue(maxValue)
                    ].length);
                    break;
                case 2:
                    maxLabelLength = Math.max([
                        ...state.getFormattedValue([
                            minValue,
                            minValue + (props1.step || 1)
                        ])
                    ].length, [
                        ...state.getFormattedValue([
                            maxValue - (props1.step || 1),
                            maxValue
                        ])
                    ].length);
                    break;
                default:
                    throw new Error('Only sliders with 1 or 2 handles are supported!');
            }
            let outputValue = /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsx)((0, $5DeaH$reactariacomponentsSlider.SliderOutput), {
                style: {
                    width: `${maxLabelLength}ch`,
                    minWidth: `${maxLabelLength}ch`,
                    fontVariantNumeric: 'tabular-nums'
                },
                className: $e937566cb5c6349c$var$output({
                    direction: direction,
                    labelPosition: labelPosition,
                    isInForm: !!formContext
                })
            });
            return /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsxs)((0, $5DeaH$reactjsxruntime.Fragment), {
                children: [
                    /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsxs)("div", {
                        className: $e937566cb5c6349c$var$labelContainer({
                            labelPosition: labelPosition,
                            labelAlign: labelAlign
                        }),
                        children: [
                            /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldLabel), {
                                isDisabled: props1.isDisabled,
                                size: props1.size,
                                labelPosition: labelPosition,
                                labelAlign: labelAlign,
                                contextualHelp: props1.contextualHelp,
                                children: label
                            }),
                            labelPosition === 'top' && outputValue
                        ]
                    }),
                    /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsxs)("div", {
                        className: function anonymous(props) {
                            let rules = " ";
                            rules += ' zXcEWr131';
                            rules += ' yXcEWr131';
                            rules += ' BXcEWr131';
                            rules += ' AXcEWr131';
                            if (props.size === "XL") rules += ' Nl131';
                            else if (props.size === "L") rules += ' Nj131';
                            else if (props.size === "S") rules += ' Nf131';
                            else if (props.size === "XS") rules += ' Ne131';
                            else rules += ' Nk131';
                            if (props.isQuiet) rules += ' _qd131';
                            else rules += ' _qtb131';
                            if (props.size === "XL") rules += ' -Anxdcc-Zt131';
                            else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
                            else if (props.size === "S") rules += ' -Anxdcc-Zr131';
                            else rules += ' -Anxdcc-Z3fn9H131';
                            rules += ' __v6TNLWb131';
                            rules += ' sh131';
                            rules += ' eb131';
                            if (props.size === "XL") rules += ' Uj131';
                            else if (props.size === "L") rules += ' Ui131';
                            else rules += ' Ul131';
                            if (props.size === "XL") rules += ' qj131';
                            else if (props.size === "L") rules += ' qi131';
                            else rules += ' ql131';
                            return rules;
                        }({
                            size: size
                        }),
                        children: [
                            props1.children,
                            labelPosition === 'side' && outputValue
                        ]
                    })
                ]
            });
        }
    });
}
const $e937566cb5c6349c$export$472062a354075cee = /*#__PURE__*/ (0, $5DeaH$react.forwardRef)(function Slider(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $e937566cb5c6349c$export$e99164f0030f3bff);
    let formContext = (0, $5DeaH$react.useContext)((0, $bfa4962d90c8af48$exports.FormContext));
    props = (0, $bfa4962d90c8af48$exports.useFormProps)(props);
    let { labelPosition: labelPosition = 'top', size: size = 'M', fillOffset: fillOffset, isEmphasized: isEmphasized, trackStyle: trackStyle = 'thin', thumbStyle: thumbStyle = 'default' } = props;
    let thumbRef = (0, $5DeaH$react.useRef)(null);
    let inputRef = (0, $5DeaH$react.useRef)(null); // TODO: need to pass inputRef to SliderThumb when we release the next version of RAC 1.3.0
    let domRef = (0, $150382364eecc4bc$exports.useFocusableRef)(ref, inputRef);
    let isStaticColor = props['PRIVATE_staticColor'];
    return /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsx)($e937566cb5c6349c$export$9418495bb635ebde, {
        ...props,
        sliderRef: domRef,
        children: /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsx)((0, $5DeaH$reactariacomponentsSlider.SliderTrack), {
            className: $e937566cb5c6349c$export$6b2a7d5132615086({
                size: size,
                labelPosition: labelPosition,
                isInForm: !!formContext
            }),
            children: ({ isDisabled: isDisabled })=>{
                return /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsxs)((0, $5DeaH$reactjsxruntime.Fragment), {
                    children: [
                        /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsx)("div", {
                            className: $e937566cb5c6349c$export$1e1dc99b6e2204f1({
                                isDisabled: isDisabled,
                                isStaticColor: isStaticColor,
                                trackStyle: trackStyle
                            }),
                            children: /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsx)((0, $5DeaH$reactariacomponentsSlider.SliderFill), {
                                offset: fillOffset,
                                className: $e937566cb5c6349c$export$847ceef3bc944d02({
                                    isDisabled: isDisabled,
                                    isEmphasized: isEmphasized,
                                    trackStyle: trackStyle
                                })
                            })
                        }),
                        /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsx)((0, $5DeaH$reactariacomponentsSlider.SliderThumb), {
                            className: $e937566cb5c6349c$export$7814bee2738c4c3,
                            index: 0,
                            name: props.name,
                            form: props.form,
                            ref: thumbRef,
                            style: (renderProps)=>(0, $2061c83559b50a66$exports.pressScale)(thumbRef, {
                                    transform: 'translate(-50%, -50%)'
                                })({
                                    ...renderProps,
                                    isPressed: renderProps.isDragging
                                }),
                            children: (renderProps)=>/*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsx)("div", {
                                    className: $e937566cb5c6349c$export$f120a3c6038107a5({
                                        size: size
                                    }),
                                    children: /*#__PURE__*/ (0, $5DeaH$reactjsxruntime.jsx)("div", {
                                        className: $e937566cb5c6349c$export$becc9140d0e846f8({
                                            ...renderProps,
                                            size: size,
                                            thumbStyle: thumbStyle
                                        })
                                    })
                                })
                        })
                    ]
                });
            }
        })
    });
});
