// @react-spectrum/s2/dist/private/StatusLight.mjs
import "./StatusLight.css";
import {CenterBaseline as $1f4b04be3f24aae3$export$768dac55bb57081d} from "./CenterBaseline.mjs";
import {Text as $8e847109a6ab556d$export$5f1af8db9871e1d6} from "./Content.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {useIsSkeleton as $5ad421ec19460c48$export$4f8dc7555740235c} from "../../icons/Skeleton.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsxs as $2PVgW$jsxs, jsx as $2PVgW$jsx} from "react/jsx-runtime";
import {createContext as $2PVgW$createContext, forwardRef as $2PVgW$forwardRef} from "react";
import {filterDOMProps as $2PVgW$filterDOMProps} from "react-aria/filterDOMProps";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 







const $cd71613dba8ee178$export$d8f8c0b091d8cb9b = /*#__PURE__*/ (0, $2PVgW$createContext)(null);
const $cd71613dba8ee178$var$wrapper = function anonymous(props, overrides) {
    let rules = " ";
    let width = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "Z") width = true;
        rules += p[0];
    }
    rules += ' sd131';
    rules += ' Uc131';
    rules += ' Usd131';
    rules += ' qc131';
    rules += ' qsd131';
    rules += ' ea131';
    if (!width) rules += ' ZJ131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.variant === "neutral") rules += ' pn131';
    else rules += ' pt131';
    rules += ' __ca131';
    return rules;
};
const $cd71613dba8ee178$var$light = function anonymous(props) {
    let rules = " ";
    if (props.size === "XL") rules += ' Zk131';
    else if (props.size === "L") rules += ' Zj131';
    else if (props.size === "M") rules += ' Zi131';
    else if (props.size === "S") rules += ' Zh131';
    if (props.size === "XL") rules += ' Fl131';
    else if (props.size === "L") rules += ' Fk131';
    else if (props.size === "M") rules += ' Fj131';
    else if (props.size === "S") rules += ' Fi131';
    if (props.isSkeleton) rules += ' tn131';
    else if (props.variant === "silver") rules += ' te131';
    else if (props.variant === "brown") rules += ' th131';
    else if (props.variant === "cinnamon") rules += ' tj131';
    else if (props.variant === "turquoise") rules += ' tb131';
    else if (props.variant === "pink") rules += ' to131';
    else if (props.variant === "yellow") rules += ' tl131';
    else if (props.variant === "seafoam") rules += ' tc131';
    else if (props.variant === "indigo") rules += ' td131';
    else if (props.variant === "magenta") rules += ' tp131';
    else if (props.variant === "purple") rules += ' ti131';
    else if (props.variant === "fuchsia") rules += ' tk131';
    else if (props.variant === "cyan") rules += ' ts131';
    else if (props.variant === "chartreuse") rules += ' tg131';
    else if (props.variant === "celery") rules += ' tv131';
    else if (props.variant === "negative") rules += ' tq131';
    else if (props.variant === "notice") rules += ' tm131';
    else if (props.variant === "positive") rules += ' tt131';
    else if (props.variant === "neutral") rules += ' tf131';
    else if (props.variant === "informative") rules += ' tu131';
    rules += ' _Ne131';
    rules += ' Pe131';
    return rules;
};
const $cd71613dba8ee178$export$5f84c37a31c6e41c = /*#__PURE__*/ (0, $2PVgW$forwardRef)(function StatusLight(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $cd71613dba8ee178$export$d8f8c0b091d8cb9b);
    let { children: children, size: size = 'M', variant: variant = 'neutral', role: role, UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style, styles: styles } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let isSkeleton = (0, $5ad421ec19460c48$export$4f8dc7555740235c)();
    if (!children && !props['aria-label'] && process.env.NODE_ENV !== 'production') console.warn('If no children are provided, an aria-label must be specified');
    if (!role && (props['aria-label'] || props['aria-labelledby']) && process.env.NODE_ENV !== 'production') console.warn('A labelled StatusLight must have a role.');
    return /*#__PURE__*/ (0, $2PVgW$jsxs)("div", {
        ...(0, $2PVgW$filterDOMProps)(props, {
            labelable: !!role
        }),
        ref: domRef,
        role: role,
        style: UNSAFE_style,
        className: UNSAFE_className + $cd71613dba8ee178$var$wrapper({
            size: size,
            variant: variant
        }, styles),
        children: [
            /*#__PURE__*/ (0, $2PVgW$jsx)((0, $1f4b04be3f24aae3$export$768dac55bb57081d), {
                children: /*#__PURE__*/ (0, $2PVgW$jsx)("svg", {
                    className: $cd71613dba8ee178$var$light({
                        size: size,
                        variant: variant,
                        isSkeleton: isSkeleton
                    }),
                    "aria-hidden": "true",
                    children: /*#__PURE__*/ (0, $2PVgW$jsx)("circle", {
                        r: "50%",
                        cx: "50%",
                        cy: "50%"
                    })
                })
            }),
            /*#__PURE__*/ (0, $2PVgW$jsx)((0, $8e847109a6ab556d$export$5f1af8db9871e1d6), {
                children: children
            })
        ]
    });
});


export {$cd71613dba8ee178$export$d8f8c0b091d8cb9b as StatusLightContext, $cd71613dba8ee178$export$5f84c37a31c6e41c as StatusLight};
