// @react-spectrum/s2/dist/private/Switch.cjs
require("./Switch.css");
var $e991cbcdf82ced71$exports = require("./CenterBaseline.cjs");
var $bfa4962d90c8af48$exports = require("./Form.cjs");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $2061c83559b50a66$exports = require("./pressScale.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $bxoLC$reactjsxruntime = require("react/jsx-runtime");
var $bxoLC$react = require("react");
var $bxoLC$reactariacomponentsSwitch = require("react-aria-components/Switch");
var $bxoLC$reactariaI18nProvider = require("react-aria/I18nProvider");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "SwitchContext", () => $0bb99ae32af8596c$export$8699e3b644d5a28a);
$parcel$export(module.exports, "Switch", () => $0bb99ae32af8596c$export$b5d5cf8927ab7262);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 









const $0bb99ae32af8596c$export$8699e3b644d5a28a = /*#__PURE__*/ (0, $bxoLC$react.createContext)(null);
const $0bb99ae32af8596c$var$field = function anonymous(props, overrides) {
    let rules = " ";
    let width = false;
    let gridColumnStart = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "Z") width = true;
        if (p[1] === "z") gridColumnStart = true;
        rules += p[0];
    }
    rules += ' se131';
    if (props.isNoVisibleLabel) rules += ' DtyCQZ131';
    else rules += ' DE5iebb131';
    rules += ' qf131';
    if (!width) rules += ' ZJ131';
    if (props.size === "XL") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "L") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "S") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else if (props.size === "XS") {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    } else {
        rules += ' uk131';
        rules += ' ucJ9TBTb131';
        rules += ' ud3Euai131';
        rules += ' uea131';
        rules += ' ugb131';
        rules += ' uhd131';
        rules += ' uje131';
        rules += ' u2NhKxcl131';
        rules += ' uic131';
    }
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    if (props.size === "XL") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "L") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "S") {
        rules += ' xb131';
        rules += ' _xa131';
    } else if (props.size === "XS") {
        rules += ' xb131';
        rules += ' _xa131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    if (props.size === "XL") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "L") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "S") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else if (props.size === "XS") {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    } else {
        rules += ' _Fd131';
        rules += ' _FnuYUweb131';
    }
    if (props.size === "XL") rules += ' pt131';
    else if (props.size === "L") rules += ' pt131';
    else if (props.size === "S") rules += ' pt131';
    else if (props.size === "XS") rules += ' pt131';
    else rules += ' pt131';
    if (props.size === "XL") rules += ' -BhX7R-Fz131';
    else if (props.size === "L") rules += ' -BhX7R-Fv131';
    else if (props.size === "S") rules += ' -BhX7R-Fp131';
    else if (props.size === "XS") rules += ' -BhX7R-Fn131';
    else rules += ' -BhX7R-Fx131';
    rules += ' UVxyPec131';
    if (props.isInForm) {
        if (!gridColumnStart) rules += ' zhJUGr131';
    }
    return rules;
};
const $0bb99ae32af8596c$var$wrapper = function anonymous(props) {
    let rules = " ";
    rules += ' se131';
    rules += ' DLwlgod131';
    rules += ' z2mU131';
    rules += ' yzZKy131';
    rules += ' ea131';
    rules += ' Yc131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.isDisabled) {
        rules += ' pp131';
        rules += ' plc131';
    } else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    }
    rules += ' __ca131';
    return rules;
};
const $0bb99ae32af8596c$var$track = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    rules += ' -ERv7Xb-ZmDIzbc131';
    if (props.size === "XL") rules += ' -LsrJQc-Fn131';
    else if (props.size === "L") rules += ' -LsrJQc-Fm131';
    else if (props.size === "S") rules += ' -LsrJQc-Fl131';
    else rules += ' -LsrJQc-Ft131';
    rules += ' ZAkyXj131';
    rules += ' Fpk5ROc131';
    rules += ' _oa131';
    rules += ' _kc131';
    rules += ' hc131';
    rules += ' mc131';
    rules += ' lc131';
    rules += ' _jf131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' _yb131';
    if (props.isSelected) rules += ' iA131';
    else if (props.isDisabled) {
        rules += ' io131';
        rules += ' ilc131';
    } else {
        if (props.isPressed) rules += ' in131';
        else if (props.isFocusVisible) rules += ' in131';
        else if (props.isHovered) rules += ' in131';
        else rules += ' iv131';
        rules += ' ila131';
    }
    if (props.isSelected) {
        if (props.isDisabled) {
            rules += ' gw131';
            rules += ' gld131';
        } else {
            if (props.isEmphasized) {
                if (props.isPressed) rules += ' gY131';
                else if (props.isFocusVisible) rules += ' gY131';
                else if (props.isHovered) rules += ' gY131';
                else rules += ' g3131';
            } else {
                if (props.isPressed) rules += ' gu131';
                else if (props.isFocusVisible) rules += ' gu131';
                else if (props.isHovered) rules += ' gu131';
                else rules += ' g0131';
            }
            rules += ' gle131';
        }
    } else rules += ' gX131';
    return rules;
};
const $0bb99ae32af8596c$var$handle = function anonymous(props) {
    let rules = " ";
    rules += ' Fb131';
    rules += ' fa131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    if (props.isSelected) rules += ' gX131';
    else if (props.isDisabled) {
        rules += ' gw131';
        rules += ' gld131';
    } else {
        if (props.isPressed) rules += ' gu131';
        else if (props.isFocusVisible) rules += ' gu131';
        else if (props.isHovered) rules += ' gu131';
        else rules += ' g0131';
    }
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    return rules;
};
// Use an inline style to calculate the transform so we can combine it with the press scale.
const $0bb99ae32af8596c$var$transformStyle = ({ isSelected: isSelected, direction: direction })=>{
    // In the default state, the handle is 8px smaller than the track. When selected it grows to 6px smaller than the track.
    // Normally this could be calculated as a scale transform with (trackHeight - 8px) / trackHeight, however,
    // CSS does not allow division with units. To solve this we use a 3d perspective transform. Perspective is the
    // distance from the Z=0 plane to the viewer. Since we want to scale the handle by a fixed amount and we cannot divide
    // by a value with units, we can set the Z translation to a fixed amount and change the perspective in order to achieve
    // the desired effect. Given the following formula:
    //
    //   scale = perspective / (perspective - translateZ)
    //
    // and desired scale factors (accounting for the 2px border on each side of the track):
    //
    //   defaultScale = (trackHeight - 8px) / (trackHeight - 4px)
    //   selectedScale = (trackHeight - 6px) / (trackHeight - 4px)
    //
    // we can solve for the perspective needed in each case where translateZ is hard coded to -4px:
    //
    //    defaultPerspective = trackHeight - 8px
    //    selectedPerspective = 2 * (trackHeight - 6px)
    const placement = direction === 'ltr' ? 'translateX(calc(var(--trackWidth) - 100% - 4px)) perspective(calc(2 * (var(--trackHeight) - 6px))) translateZ(-4px)' : 'translateX(calc(100% - var(--trackWidth) + 4px)) perspective(calc(2 * (var(--trackHeight) - 6px))) translateZ(-4px)';
    return {
        transform: isSelected ? placement : 'perspective(calc(var(--trackHeight) - 8px)) translateZ(-4px)'
    };
};
const $0bb99ae32af8596c$export$b5d5cf8927ab7262 = /*#__PURE__*/ (0, $bxoLC$react.forwardRef)(function Switch(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $0bb99ae32af8596c$export$8699e3b644d5a28a);
    let { children: children, UNSAFE_className: UNSAFE_className = '', UNSAFE_style: UNSAFE_style } = props;
    let inputRef = (0, $bxoLC$react.useRef)(null);
    let domRef = (0, $150382364eecc4bc$exports.useFocusableRef)(ref, inputRef);
    let handleRef = (0, $bxoLC$react.useRef)(null);
    let isInForm = !!(0, $bxoLC$react.useContext)((0, $bfa4962d90c8af48$exports.FormContext));
    let { direction: direction } = (0, $bxoLC$reactariaI18nProvider.useLocale)();
    props = (0, $bfa4962d90c8af48$exports.useFormProps)(props);
    return /*#__PURE__*/ (0, $bxoLC$reactjsxruntime.jsx)((0, $bxoLC$reactariacomponentsSwitch.SwitchField), {
        ...props,
        ref: domRef,
        inputRef: inputRef,
        style: UNSAFE_style,
        className: (renderProps)=>UNSAFE_className + $0bb99ae32af8596c$var$field({
                ...renderProps,
                isInForm: isInForm,
                size: props.size || 'M',
                isNoVisibleLabel: !children
            }, props.styles),
        children: ({ isDisabled: isDisabled, isInvalid: isInvalid })=>/*#__PURE__*/ (0, $bxoLC$reactjsxruntime.jsxs)((0, $bxoLC$reactjsxruntime.Fragment), {
                children: [
                    /*#__PURE__*/ (0, $bxoLC$reactjsxruntime.jsx)((0, $bxoLC$reactariacomponentsSwitch.SwitchButton), {
                        className: (renderProps)=>$0bb99ae32af8596c$var$wrapper({
                                ...renderProps,
                                isInForm: isInForm,
                                size: props.size || 'M'
                            }),
                        children: (renderProps)=>/*#__PURE__*/ (0, $bxoLC$reactjsxruntime.jsxs)((0, $bxoLC$reactjsxruntime.Fragment), {
                                children: [
                                    /*#__PURE__*/ (0, $bxoLC$reactjsxruntime.jsx)((0, $e991cbcdf82ced71$exports.CenterBaseline), {
                                        children: /*#__PURE__*/ (0, $bxoLC$reactjsxruntime.jsx)("div", {
                                            className: $0bb99ae32af8596c$var$track({
                                                ...renderProps,
                                                size: props.size || 'M',
                                                isEmphasized: props.isEmphasized
                                            }),
                                            children: /*#__PURE__*/ (0, $bxoLC$reactjsxruntime.jsx)("div", {
                                                ref: handleRef,
                                                style: (0, $2061c83559b50a66$exports.pressScale)(handleRef, $0bb99ae32af8596c$var$transformStyle)({
                                                    ...renderProps,
                                                    direction: direction
                                                }),
                                                className: $0bb99ae32af8596c$var$handle(renderProps)
                                            })
                                        })
                                    }),
                                    children
                                ]
                            })
                    }),
                    /*#__PURE__*/ (0, $bxoLC$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.HelpText), {
                        size: props.size || 'M',
                        styles: " z2mU131 Td131",
                        isDisabled: isDisabled,
                        isInvalid: isInvalid,
                        description: props.description,
                        showErrorIcon: true,
                        children: props.errorMessage
                    })
                ]
            })
    });
});
