// @react-spectrum/s2/dist/private/TableView.cjs
require("./TableView.css");
var $6e265ff388155b91$exports = require("./ActionButton.cjs");
var $e1016b8947dbadb8$exports = require("./ButtonGroup.cjs");
var $af04e099a53d3e85$exports = require("./Checkbox.cjs");
var $c3e72d94a25defe0$exports = require("../../icons/Checkmark.cjs");
var $1df9f1c9262ce5df$exports = require("./Chevron.cjs");
var $8ecd53081d7d50cc$exports = require("../../icons/Close.cjs");
var $a05aa0b0345f1846$exports = require("./CustomDialog.cjs");
var $fc196655d58ce18c$exports = require("./DialogContainer.cjs");
var $f1f08ef5de95f8eb$exports = require("./dnd-utils.cjs");
var $35d2da6f10cb8588$exports = require("./DragPreview.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $4526404114e78c80$exports = require("./intlStrings.cjs");
var $e741ea6b88ce4866$exports = require("./Menu.cjs");
var $23350f45c699333e$exports = require("./S2_MoveHorizontalTableWidget.cjs");
var $1c116ea4b0d2d023$exports = require("./ProgressCircle.cjs");
var $22bf2407c3628a21$exports = require("../../icons/SortDown.cjs");
var $2197830b90e71c31$exports = require("../../icons/SortUp.cjs");
var $25d06cf8d4e72761$exports = require("./Button.cjs");
var $85bf454149fe9ddc$exports = require("./ActionBar.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $3112837ad519e72a$exports = require("../../icons/useMediaQuery.cjs");
var $ee7b4c497f520c08$exports = require("./utils.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $iLVc8$reactjsxruntime = require("react/jsx-runtime");
var $iLVc8$reactariacomponentsButton = require("react-aria-components/Button");
var $iLVc8$reactariacomponentsTable = require("react-aria-components/Table");
var $iLVc8$reactariaCollection = require("react-aria/Collection");
var $iLVc8$reactariacomponentsCollectionBuilder = require("react-aria-components/CollectionBuilder");
var $iLVc8$reactariacomponentsslots = require("react-aria-components/slots");
var $iLVc8$reactariacomponentsForm = require("react-aria-components/Form");
var $iLVc8$reactariaprivateutilsshadowdomDOMFunctions = require("react-aria/private/utils/shadowdom/DOMFunctions");
var $iLVc8$reactariaprivateutilsdomHelpers = require("react-aria/private/utils/domHelpers");
var $iLVc8$reactariacomponentsVirtualizer = require("react-aria-components/Virtualizer");
var $iLVc8$reactariacomponentsDialog = require("react-aria-components/Dialog");
var $iLVc8$reactariacomponentsCheckbox = require("react-aria-components/Checkbox");
var $iLVc8$reactariacomponentsPopover = require("react-aria-components/Popover");
var $iLVc8$react = require("react");
var $iLVc8$reactariaprivateutilsuseLayoutEffect = require("react-aria/private/utils/useLayoutEffect");
var $iLVc8$reactariaI18nProvider = require("react-aria/I18nProvider");
var $iLVc8$reactariauseLocalizedStringFormatter = require("react-aria/useLocalizedStringFormatter");
var $iLVc8$reactariauseObjectRef = require("react-aria/useObjectRef");
var $iLVc8$reactariaVisuallyHidden = require("react-aria/VisuallyHidden");


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "TableContext", () => $bed37377ec7a3a9e$export$93e4b0b2cc49b648);
$parcel$export(module.exports, "TableView", () => $bed37377ec7a3a9e$export$b3c27e869d856b7);
$parcel$export(module.exports, "TableBody", () => $bed37377ec7a3a9e$export$76ccd210b9029917);
$parcel$export(module.exports, "Column", () => $bed37377ec7a3a9e$export$816b5d811295e6bc);
$parcel$export(module.exports, "TableHeader", () => $bed37377ec7a3a9e$export$f850895b287ef28e);
$parcel$export(module.exports, "Cell", () => $bed37377ec7a3a9e$export$f6f0c3fe4ec306ea);
$parcel$export(module.exports, "EditableCell", () => $bed37377ec7a3a9e$export$d27c35c47f085992);
$parcel$export(module.exports, "Row", () => $bed37377ec7a3a9e$export$b59bdbef9ce70de2);
$parcel$export(module.exports, "TableFooter", () => $bed37377ec7a3a9e$export$1f116082bba1f9a8);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 









































let $bed37377ec7a3a9e$var$InternalTableContext = /*#__PURE__*/ (0, $iLVc8$react.createContext)({});
const $bed37377ec7a3a9e$var$tableWrapper = function anonymous(props, overrides) {
    let rules = " ";
    let minHeight = false;
    let minWidth = false;
    let position = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L|F|M|K)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "M") minHeight = true;
        if (p[1] === "N") minWidth = true;
        if (p[1] === "_P") position = true;
        rules += p[0];
    }
    if (!minHeight) rules += ' Ma131';
    if (!minWidth) rules += ' Na131';
    rules += ' sd131';
    rules += ' _Bb131';
    rules += ' __ca131';
    if (!position) rules += ' _Pc131';
    rules += ' _Nb131';
    rules += ' Pb131';
    return rules;
};
// due to the overflow: clip for actionbar, render the outline via a pseudo element so it isn't
// clipped. 6px radius matches the table's border radius
const $bed37377ec7a3a9e$var$rootDropOutline = "pUEtOc";
const $bed37377ec7a3a9e$var$dropTargetBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 10%)]";
const $bed37377ec7a3a9e$var$table = function anonymous(props) {
    let rules = " ";
    rules += ' Za131';
    rules += ' Fb131';
    rules += ' _oa131';
    rules += ' _4c131';
    rules += ' Ma131';
    rules += ' Na131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Na131';
    rules += ' Pa131';
    if (props.isDropTarget) rules += ' glzvDBe131';
    else if (props.isQuiet) rules += ' g9131';
    else rules += ' gX131';
    rules += ' gla131';
    if (props.isDropTarget) rules += ' ix131';
    else rules += ' iq131';
    if (props.isDropTarget) rules += ' ild131';
    else rules += ' ila131';
    rules += ' _jf131';
    if (props.isQuiet) rules += ' _ka131';
    else rules += ' _kb131';
    if (props.isQuiet) rules += ' ha131';
    else rules += ' hb131';
    if (props.isQuiet) rules += ' ma131';
    else rules += ' mb131';
    if (props.isQuiet) rules += ' la131';
    else rules += ' lb131';
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (props.isQuiet) rules += ' od131';
    else rules += ' of131';
    if (props.isQuiet) rules += ' nd131';
    else rules += ' nf131';
    if (props.isQuiet) rules += ' kd131';
    else rules += ' kf131';
    if (props.isQuiet) rules += ' jd131';
    else rules += ' jf131';
    rules += ' _Um131';
    if (props.isDragAndDrop) {
        if (props.isCheckboxSelection) rules += ' _Tn131';
        else rules += ' _Ti131';
    } else if (props.isCheckboxSelection) rules += ' _Tk131';
    rules += ' _yb131';
    if (props.isCheckboxSelection) rules += ' -dBGRde-ZosIvTc131';
    else rules += ' -dBGRde-ZwbbKhe131';
    rules += ' -lPUMEc-Zu131';
    return rules;
};
// component-height-100
const $bed37377ec7a3a9e$var$DEFAULT_HEADER_HEIGHT = {
    medium: 32,
    large: 40
};
const $bed37377ec7a3a9e$var$ROW_HEIGHTS = {
    compact: {
        medium: 32,
        large: 40
    },
    regular: {
        medium: 40,
        large: 50
    },
    spacious: {
        medium: 48,
        large: 60
    }
};
class $bed37377ec7a3a9e$export$4e03bdf0174fd602 extends (0, $iLVc8$reactariacomponentsVirtualizer.TableLayout) {
    isStickyColumn(node) {
        return node.props.isSticky;
    }
    buildCollection() {
        let rowGroups = super.buildCollection();
        if (rowGroups.length < 2) return [];
        let { layoutInfo: layoutInfo } = rowGroups[1];
        // TableLayout's buildCollection always sets the body width to the max width between the header width, but
        // we want the body to be sticky and only as wide as the table so it is always in view if loading/empty
        let isEmptyOrLoading = this.virtualizer?.collection.size === 0;
        if (isEmptyOrLoading) layoutInfo.rect.width = this.virtualizer.size.width - 80;
        return rowGroups;
    }
    buildLoader(node, x, y) {
        let layoutNode = super.buildLoader(node, x, y);
        let { layoutInfo: layoutInfo } = layoutNode;
        layoutInfo.allowOverflow = true;
        layoutInfo.rect.width = this.virtualizer.size.width;
        // If performing first load or empty, the body will be sticky so we don't want to apply sticky to the loader, otherwise it will
        // affect the positioning of the empty state renderer
        let collection = this.virtualizer.collection;
        let isEmptyOrLoading = collection?.size === 0;
        layoutInfo.isSticky = !isEmptyOrLoading;
        return layoutNode;
    }
    buildRowGroup(y, node) {
        let layoutNode = super.buildRowGroup(y, node);
        if (node.type !== 'tablebody') return layoutNode;
        let { layoutInfo: layoutInfo } = layoutNode;
        // Needs overflow for sticky loader
        layoutInfo.allowOverflow = true;
        // If loading or empty, we'll want the body to be sticky and centered
        let isEmptyOrLoading = this.virtualizer?.collection.size === 0;
        if (isEmptyOrLoading) {
            layoutInfo.rect = new (0, $iLVc8$reactariacomponentsVirtualizer.Rect)(40, 40, this.virtualizer.size.width - 80, this.virtualizer.size.height - 80);
            layoutInfo.isSticky = true;
        }
        return {
            ...layoutNode,
            layoutInfo: layoutInfo
        };
    }
    buildRow(node, x, y) {
        let layoutNode = super.buildRow(node, x, y);
        layoutNode.layoutInfo.allowOverflow = true;
        // Needs overflow for sticky selection/drag cells
        return layoutNode;
    }
    buildTableHeader() {
        let layoutNode = super.buildTableHeader();
        // Needs overflow for sticky selection/drag column
        layoutNode.layoutInfo.allowOverflow = true;
        return layoutNode;
    }
    buildColumn(node, x, y) {
        let layoutNode = super.buildColumn(node, x, y);
        // Needs overflow for the resize handle
        layoutNode.layoutInfo.allowOverflow = true;
        return layoutNode;
    }
    getDropTargetLayoutInfo(target) {
        let layoutInfo = super.getDropTargetLayoutInfo(target);
        layoutInfo.zIndex = 1;
        layoutInfo.allowOverflow = true;
        return layoutInfo;
    }
}
const $bed37377ec7a3a9e$export$93e4b0b2cc49b648 = /*#__PURE__*/ (0, $iLVc8$react.createContext)(null);
const $bed37377ec7a3a9e$export$b3c27e869d856b7 = /*#__PURE__*/ (0, $iLVc8$react.forwardRef)(function TableView(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $bed37377ec7a3a9e$export$93e4b0b2cc49b648);
    let { UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className, isQuiet: isQuiet = false, density: density = 'regular', overflowMode: overflowMode = 'truncate', styles: styles, loadingState: loadingState, onResize: propsOnResize, onResizeStart: propsOnResizeStart, onResizeEnd: propsOnResizeEnd, onAction: onAction, onLoadMore: onLoadMore, selectionMode: selectionMode = 'none', selectionStyle: selectionStyle = 'checkbox', dragAndDropHooks: dragAndDropHooks, disabledBehavior: disabledBehavior = 'all', ...otherProps } = props;
    if (dragAndDropHooks && dragAndDropHooks.renderDragPreview == null) dragAndDropHooks.renderDragPreview = (items)=>/*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $35d2da6f10cb8588$exports.DragPreview), {
            items: items,
            overflowMode: overflowMode
        });
    if (dragAndDropHooks) dragAndDropHooks.renderDropIndicator = (target)=>/*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $f1f08ef5de95f8eb$exports.InsertionIndicator), {
            target: target
        });
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let scale = (0, $ee7b4c497f520c08$exports.useScale)();
    // Starts when the user selects resize from the menu, ends when resizing ends
    // used to control the visibility of the resizer Nubbin
    let [isInResizeMode, setIsInResizeMode] = (0, $iLVc8$react.useState)(false);
    let onResizeStart = (0, $iLVc8$react.useCallback)((widths)=>{
        propsOnResizeStart?.(widths);
    }, [
        propsOnResizeStart
    ]);
    let onResizeEnd = (0, $iLVc8$react.useCallback)((widths)=>{
        setIsInResizeMode(false);
        propsOnResizeEnd?.(widths);
    }, [
        propsOnResizeEnd,
        setIsInResizeMode
    ]);
    let context = (0, $iLVc8$react.useMemo)(()=>({
            isQuiet: isQuiet,
            density: density,
            overflowMode: overflowMode,
            loadingState: loadingState,
            onLoadMore: onLoadMore,
            isInResizeMode: isInResizeMode,
            setIsInResizeMode: setIsInResizeMode,
            selectionMode: selectionMode,
            selectionStyle: selectionStyle,
            disabledBehavior: disabledBehavior
        }), [
        isQuiet,
        density,
        overflowMode,
        loadingState,
        onLoadMore,
        isInResizeMode,
        setIsInResizeMode,
        selectionMode,
        selectionStyle,
        disabledBehavior
    ]);
    let scrollRef = (0, $iLVc8$react.useRef)(null);
    let isCheckboxSelection = selectionMode === 'multiple' || selectionMode === 'single';
    let isDragAndDrop = !!dragAndDropHooks?.useDraggableCollectionState;
    let { selectedKeys: selectedKeys, onSelectionChange: onSelectionChange, actionBar: actionBar, actionBarHeight: actionBarHeight } = (0, $85bf454149fe9ddc$exports.useActionBarContainer)({
        ...props,
        scrollRef: scrollRef
    });
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactariacomponentsTable.ResizableTableContainer), {
        // TODO: perhaps this ref should be attached to the RACTable but it expects a table type ref which isn't true in the virtualized case
        ref: domRef,
        onResize: propsOnResize,
        onResizeEnd: onResizeEnd,
        onResizeStart: onResizeStart,
        className: (UNSAFE_className || '') + $bed37377ec7a3a9e$var$tableWrapper(null, styles) + ' ' + $bed37377ec7a3a9e$var$rootDropOutline,
        style: UNSAFE_style,
        children: [
            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsVirtualizer.Virtualizer), {
                layout: $bed37377ec7a3a9e$export$4e03bdf0174fd602,
                layoutOptions: {
                    rowHeight: overflowMode === 'wrap' ? undefined : $bed37377ec7a3a9e$var$ROW_HEIGHTS[density][scale],
                    estimatedRowHeight: overflowMode === 'wrap' ? $bed37377ec7a3a9e$var$ROW_HEIGHTS[density][scale] : undefined,
                    // No need for estimated headingHeight since the headers aren't affected by overflow mode: wrap
                    headingHeight: $bed37377ec7a3a9e$var$DEFAULT_HEADER_HEIGHT[scale],
                    loaderHeight: 60,
                    dropIndicatorThickness: 0
                },
                children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$InternalTableContext.Provider, {
                    value: context,
                    children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsTable.Table), {
                        ref: scrollRef,
                        style: {
                            // Fix webkit bug where scrollbars appear above the checkboxes/other table elements
                            WebkitTransform: 'translateZ(0)',
                            // Add padding at the bottom when the action bar is visible so users can scroll to the last items.
                            // Also add scroll padding so navigating with the keyboard doesn't go behind the action bar.
                            paddingBottom: actionBarHeight > 0 ? actionBarHeight + 8 : 0,
                            scrollPaddingBottom: actionBarHeight > 0 ? actionBarHeight + 8 : 0
                        },
                        className: (renderProps)=>$bed37377ec7a3a9e$var$table({
                                ...renderProps,
                                isCheckboxSelection: isCheckboxSelection,
                                isDragAndDrop: isDragAndDrop,
                                isQuiet: isQuiet
                            }),
                        selectionBehavior: selectionStyle === 'highlight' ? 'replace' : 'toggle',
                        selectionMode: selectionMode,
                        onRowAction: onAction,
                        dragAndDropHooks: dragAndDropHooks,
                        disabledBehavior: disabledBehavior,
                        ...otherProps,
                        selectedKeys: selectedKeys,
                        defaultSelectedKeys: undefined,
                        onSelectionChange: onSelectionChange
                    })
                })
            }),
            actionBar
        ]
    });
});
const $bed37377ec7a3a9e$var$centeredWrapper = " sd131 eb131 _Ca131 Za131 Fb131";
const $bed37377ec7a3a9e$export$76ccd210b9029917 = /*#__PURE__*/ (0, $iLVc8$react.forwardRef)(function TableBody(props, ref) {
    let { items: items, renderEmptyState: renderEmptyState, children: children, dependencies: dependencies = [] } = props;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let { loadingState: loadingState, onLoadMore: onLoadMore } = (0, $iLVc8$react.useContext)($bed37377ec7a3a9e$var$InternalTableContext);
    let isLoading = loadingState === 'loading' || loadingState === 'loadingMore';
    let emptyRender;
    let renderer = children;
    let stringFormatter = (0, $iLVc8$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    // TODO: still is offset strangely if loadingMore when there aren't any items in the table, see http://localhost:6006/?path=/story/tableview--empty-state&args=loadingState:loadingMore
    // This is because we don't distinguish between loadingMore and loading in the layout, resulting in a different rect being used to build the body. Perhaps can be considered as a user error
    // if they pass loadingMore without having any other items in the table. Alternatively, could update the layout so it knows the current loading state.
    let loadMoreSpinner = /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsTable.TableLoadMoreItem), {
        isLoading: loadingState === 'loadingMore',
        onLoadMore: onLoadMore,
        className: " Fb131 Za131",
        children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)("div", {
            className: $bed37377ec7a3a9e$var$centeredWrapper,
            children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $1c116ea4b0d2d023$exports.ProgressCircle), {
                isIndeterminate: true,
                "aria-label": stringFormatter.format('table.loadingMore')
            })
        })
    });
    // If the user is rendering their rows in dynamic fashion, wrap their render function in Collection so we can inject
    // the loader. Otherwise it is a static renderer and thus we can simply add the table loader after
    // TODO: this assumes that the user isn't providing their children in some wrapper though and/or isn't doing a map of children
    // (though I guess they wouldn't provide items then so the check for this is still valid in the latter case)...
    if (typeof children === 'function' && items) renderer = /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactjsxruntime.Fragment), {
        children: [
            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariaCollection.Collection), {
                items: items,
                dependencies: dependencies,
                children: children
            }),
            loadMoreSpinner
        ]
    });
    else renderer = /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactjsxruntime.Fragment), {
        children: [
            children,
            loadMoreSpinner
        ]
    });
    if (renderEmptyState != null && !isLoading) emptyRender = (props)=>/*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)("div", {
            className: $bed37377ec7a3a9e$var$centeredWrapper,
            children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsCollectionBuilder.CollectionRendererContext).Provider, {
                value: (0, $iLVc8$reactariacomponentsCollectionBuilder.DefaultCollectionRenderer),
                children: renderEmptyState(props)
            })
        });
    else if (loadingState === 'loading') emptyRender = ()=>/*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)("div", {
            className: $bed37377ec7a3a9e$var$centeredWrapper,
            children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $1c116ea4b0d2d023$exports.ProgressCircle), {
                isIndeterminate: true,
                "aria-label": stringFormatter.format('table.loading')
            })
        });
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsTable.TableBody), {
        // @ts-ignore
        ref: domRef,
        className: " Fb131",
        ...props,
        renderEmptyState: emptyRender,
        dependencies: [
            loadingState
        ],
        children: renderer
    });
});
const $bed37377ec7a3a9e$var$cellFocus = {
    outlineStyle: {
        default: 'none',
        isFocusVisible: 'solid'
    },
    outlineOffset: -2,
    outlineWidth: 2,
    outlineColor: {
        default: 'focus-ring',
        forcedColors: {
            default: 'Highlight'
        }
    },
    borderRadius: '[5px]',
    // We need to render the cell focus ring on top of the cell divider if there is one. The divider has a z-index of 1.
    zIndex: 2
};
function $bed37377ec7a3a9e$var$CellFocusRing() {
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)("div", {
        role: "presentation",
        className: function anonymous(props) {
            let rules = " ";
            if (props.isFocusVisible) rules += ' _Lf131';
            else rules += ' _Le131';
            rules += ' _Kb131';
            rules += ' _Mc131';
            rules += ' Oh131';
            rules += ' Olc131';
            rules += ' osmOfn131';
            rules += ' nsmOfn131';
            rules += ' ksmOfn131';
            rules += ' jsmOfn131';
            rules += ' _9d131';
            rules += ' _Pa131';
            rules += ' Wr131';
            rules += ' _lr131';
            rules += ' _Ar131';
            rules += ' _zr131';
            rules += ' _Ob131';
            return rules;
        }({
            isFocusVisible: true
        })
    });
}
const $bed37377ec7a3a9e$var$columnStyles = function anonymous(props) {
    let rules = " ";
    rules += ' F40ub4c131';
    rules += ' _oa131';
    if (props.isPressed) rules += ' po131';
    else if (props.isFocusVisible) rules += ' po131';
    else if (props.isHovered) rules += ' po131';
    else rules += ' pt131';
    rules += ' plb131';
    if (props.isMenu) rules += ' Sd131';
    else rules += ' Sf131';
    if (props.isMenu) rules += ' Rd131';
    else rules += ' Rf131';
    if (props.align === "end") rules += ' _Wb131';
    else if (props.align === "center") rules += ' _Wa131';
    else if (props.align === "start") rules += ' _Wd131';
    rules += ' _Le131';
    rules += ' _Pc131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' xd131';
    rules += ' _xa131';
    rules += ' xX0cczbc131';
    rules += ' _xX0cczba131';
    rules += ' sd131';
    rules += ' iq131';
    rules += ' ila131';
    if (props.isQuiet) rules += ' _kb131';
    else rules += ' _ka131';
    rules += ' hb131';
    rules += ' ma131';
    if (props.isMenu) rules += ' lb131';
    else rules += ' la131';
    rules += ' _jf131';
    rules += ' _yb131';
    return rules;
};
const $bed37377ec7a3a9e$export$816b5d811295e6bc = /*#__PURE__*/ (0, $iLVc8$react.forwardRef)(function Column(props, ref) {
    let { isQuiet: isQuiet } = (0, $iLVc8$react.useContext)($bed37377ec7a3a9e$var$InternalTableContext);
    let { allowsResizing: allowsResizing, children: children, align: align = 'start' } = props;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let isMenu = allowsResizing || !!props.menuItems;
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsTable.Column), {
        ...props,
        ref: domRef,
        style: {
            borderInlineEndColor: 'transparent'
        },
        className: (renderProps)=>$bed37377ec7a3a9e$var$columnStyles({
                ...renderProps,
                isMenu: isMenu,
                align: align,
                isQuiet: isQuiet
            }),
        children: ({ allowsSorting: allowsSorting, sortDirection: sortDirection, isFocusVisible: isFocusVisible, sort: sort, startResize: startResize })=>/*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactjsxruntime.Fragment), {
                children: [
                    isFocusVisible && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$CellFocusRing, {}),
                    isMenu ? /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$ColumnWithMenu, {
                        isColumnResizable: allowsResizing,
                        menuItems: props.menuItems,
                        allowsSorting: allowsSorting,
                        sortDirection: sortDirection,
                        sort: sort,
                        startResize: startResize,
                        align: align,
                        children: children
                    }) : /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$ColumnContents, {
                        align: align,
                        allowsSorting: allowsSorting,
                        sortDirection: sortDirection,
                        children: children
                    })
                ]
            })
    });
});
const $bed37377ec7a3a9e$var$columnContentWrapper = function anonymous(props) {
    let rules = " ";
    rules += ' Na131';
    rules += ' sd131';
    rules += ' eb131';
    rules += ' Za131';
    if (props.align === "end") rules += ' _Cb131';
    else if (props.align === "center") rules += ' _Ca131';
    else if (props.align === "default") rules += ' _Cg131';
    return rules;
};
const $bed37377ec7a3a9e$var$sortIcon = function anonymous(props) {
    let rules = " ";
    rules += ' ZXJkxGe131';
    rules += ' FXJkxGe131';
    rules += ' _va131';
    if (props.isButton) {
        rules += ' Ht131';
        rules += ' Hsu131';
    } else rules += ' Hv131';
    rules += ' _5b131';
    rules += ' -_8sjo0b-t5ZbAob131';
    return rules;
};
function $bed37377ec7a3a9e$var$ColumnContents(props) {
    let { align: align, allowsSorting: allowsSorting, sortDirection: sortDirection, children: children } = props;
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)("div", {
        className: $bed37377ec7a3a9e$var$columnContentWrapper({
            align: align
        }),
        children: [
            allowsSorting && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsslots.Provider), {
                values: [
                    [
                        (0, $bde97c91243ed164$exports.IconContext),
                        {
                            styles: $bed37377ec7a3a9e$var$sortIcon({})
                        }
                    ]
                ],
                children: sortDirection != null && (sortDirection === 'ascending' ? /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $2197830b90e71c31$exports.default), {}) : /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $22bf2407c3628a21$exports.default), {}))
            }),
            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)("span", {
                className: $bed37377ec7a3a9e$var$columnHeaderText,
                children: children
            })
        ]
    });
}
const $bed37377ec7a3a9e$var$resizableMenuButtonWrapper = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' _Kb131';
    rules += ' _Mc131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' osmOfn131';
    rules += ' nsmOfn131';
    rules += ' ksmOfn131';
    rules += ' jsmOfn131';
    rules += ' _9d131';
    rules += ' pt131';
    rules += ' Za131';
    rules += ' _Pc131';
    rules += ' sd131';
    rules += ' eb131';
    if (props.align === "end") rules += ' _Cb131';
    else if (props.align === "center") rules += ' _Ca131';
    else if (props.align === "default") rules += ' _Cg131';
    rules += ' Sf131';
    rules += ' Rf131';
    rules += ' g9131';
    rules += ' _je131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' xd131';
    rules += ' _xa131';
    rules += ' xX0cczbc131';
    rules += ' _xX0cczba131';
    return rules;
};
const $bed37377ec7a3a9e$var$resizerHandleContainer = function anonymous(props) {
    let rules = " ";
    if (props.isInResizeMode) rules += ' sb131';
    else if (props.isResizing) rules += ' sb131';
    else rules += ' sjBAxlb131';
    rules += ' Zj131';
    rules += ' Fb131';
    rules += ' _Pa131';
    rules += ' Wr131';
    rules += ' _zdYbyj131';
    if (props.resizableDirection === "both") rules += ' rk131';
    else if (props.resizableDirection === "right") rules += ' rF131';
    else if (props.resizableDirection === "left") rules += ' rj131';
    else rules += ' rt131';
    return rules;
};
const $bed37377ec7a3a9e$var$resizerHandle = function anonymous(props) {
    let rules = " ";
    if (props.isResizing) rules += ' g5qAiPc131';
    else if (props.isFocusVisible) rules += ' g5qAiPc131';
    else rules += ' gE131';
    if (props.isResizing) rules += ' gle131';
    else if (props.isFocusVisible) rules += ' gle131';
    else rules += ' gla131';
    if (props.isResizing) rules += ' Fc131';
    else rules += ' Fb131';
    if (props.isResizing) rules += ' Zf131';
    else rules += ' Ze131';
    rules += ' _Pa131';
    rules += ' _AWJpoB131';
    return rules;
};
const $bed37377ec7a3a9e$var$columnHeaderText = " _Nc131 Pc131 _Yb131 _7c131 NXJkxGe131 _ua131 _vb131 _sd131";
const $bed37377ec7a3a9e$var$chevronIcon = " _Sd131 It131 Isu131 NXJkxGe131 _va131 -_8sjo0b-t5ZbAob131";
const $bed37377ec7a3a9e$var$nubbin = " _Pa131 Wr131 _AlhfIS131 ZXJkxGe131 FXJkxGe131 t5qAiPc131 tla131 -_8sjo0b-tw131 -_8sjo0b-lt8pvGJ131";
function $bed37377ec7a3a9e$var$ColumnWithMenu(props) {
    let { allowsSorting: allowsSorting, sortDirection: sortDirection, sort: sort, startResize: startResize, children: children, align: align, isColumnResizable: isColumnResizable, menuItems: menuItems } = props;
    let { setIsInResizeMode: setIsInResizeMode, isInResizeMode: isInResizeMode } = (0, $iLVc8$react.useContext)($bed37377ec7a3a9e$var$InternalTableContext);
    let stringFormatter = (0, $iLVc8$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    const onMenuSelect = (key)=>{
        switch(key){
            case 'sort-asc':
                sort('ascending');
                break;
            case 'sort-desc':
                sort('descending');
                break;
            case 'resize':
                setIsInResizeMode?.(true);
                startResize();
                break;
        }
    };
    let items = (0, $iLVc8$react.useMemo)(()=>{
        let options = [];
        if (isColumnResizable) options = [
            {
                label: stringFormatter.format('table.resizeColumn'),
                id: 'resize'
            }
        ];
        if (allowsSorting) options = [
            {
                label: stringFormatter.format('table.sortAscending'),
                id: 'sort-asc'
            },
            {
                label: stringFormatter.format('table.sortDescending'),
                id: 'sort-desc'
            },
            ...options
        ];
        return options;
    // eslint-disable-next-line react-hooks/exhaustive-deps
    }, [
        allowsSorting,
        isColumnResizable
    ]);
    let buttonAlignment = 'start';
    let menuAlign = 'start';
    if (align === 'center') buttonAlignment = 'center';
    else if (align === 'end') {
        buttonAlignment = 'end';
        menuAlign = 'end';
    }
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactjsxruntime.Fragment), {
        children: [
            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $e741ea6b88ce4866$exports.MenuTrigger), {
                align: menuAlign,
                children: [
                    /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactariacomponentsButton.Button), {
                        className: (renderProps)=>$bed37377ec7a3a9e$var$resizableMenuButtonWrapper({
                                ...renderProps,
                                align: buttonAlignment
                            }),
                        children: [
                            allowsSorting && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsslots.Provider), {
                                values: [
                                    [
                                        (0, $bde97c91243ed164$exports.IconContext),
                                        {
                                            styles: $bed37377ec7a3a9e$var$sortIcon({
                                                isButton: true
                                            })
                                        }
                                    ]
                                ],
                                children: sortDirection != null && (sortDirection === 'ascending' ? /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $2197830b90e71c31$exports.default), {}) : /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $22bf2407c3628a21$exports.default), {}))
                            }),
                            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)("div", {
                                className: $bed37377ec7a3a9e$var$columnHeaderText,
                                children: children
                            }),
                            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $1df9f1c9262ce5df$exports.default), {
                                size: "M",
                                className: $bed37377ec7a3a9e$var$chevronIcon
                            })
                        ]
                    }),
                    /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $e741ea6b88ce4866$exports.Menu), {
                        onAction: onMenuSelect,
                        styles: " Nn131",
                        children: [
                            items.length > 0 && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $e741ea6b88ce4866$exports.MenuSection), {
                                children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariaCollection.Collection), {
                                    items: items,
                                    children: (item)=>/*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $e741ea6b88ce4866$exports.MenuItem), {
                                            children: item?.label
                                        })
                                })
                            }),
                            menuItems
                        ]
                    })
                ]
            }),
            isColumnResizable && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)("div", {
                "data-react-aria-prevent-focus": "true",
                children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsTable.ColumnResizer), {
                    "data-react-aria-prevent-focus": "true",
                    className: ({ resizableDirection: resizableDirection, isResizing: isResizing })=>$bed37377ec7a3a9e$var$resizerHandleContainer({
                            resizableDirection: resizableDirection,
                            isResizing: isResizing,
                            isInResizeMode: isInResizeMode
                        }),
                    children: ({ isFocusVisible: isFocusVisible, isResizing: isResizing })=>/*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactjsxruntime.Fragment), {
                            children: [
                                /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$ResizerIndicator, {
                                    isFocusVisible: isFocusVisible,
                                    isResizing: isResizing
                                }),
                                (isFocusVisible || isInResizeMode) && isResizing && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)("div", {
                                    className: $bed37377ec7a3a9e$var$nubbin,
                                    children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $23350f45c699333e$exports.default), {})
                                })
                            ]
                        })
                })
            })
        ]
    });
}
function $bed37377ec7a3a9e$var$ResizerIndicator({ isFocusVisible: isFocusVisible, isResizing: isResizing }) {
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)("div", {
        className: $bed37377ec7a3a9e$var$resizerHandle({
            isFocusVisible: isFocusVisible,
            isResizing: isResizing
        })
    });
}
const $bed37377ec7a3a9e$var$tableHeader = function anonymous(props) {
    let rules = " ";
    rules += ' Fb131';
    rules += ' Za131';
    rules += ' gL131';
    rules += ' gl9131';
    rules += ' JLPOtrc131';
    if (props.isHovered) rules += ' -xLp2wc-sb131';
    else rules += ' -xLp2wc-sk131';
    return rules;
};
const $bed37377ec7a3a9e$var$selectAllCheckbox = "";
const $bed37377ec7a3a9e$var$selectAllCheckboxColumn = function anonymous(props) {
    let rules = " ";
    rules += ' Sd131';
    rules += ' SZbReHcf131';
    rules += ' Rd131';
    rules += ' R3n0jFt131';
    rules += ' Td131';
    rules += ' Qd131';
    rules += ' Fb131';
    rules += ' _oa131';
    rules += ' _Le131';
    rules += ' _Pc131';
    rules += ' sd131';
    rules += ' _cb131';
    rules += ' eb131';
    rules += ' _Cg131';
    rules += ' iq131';
    rules += ' ila131';
    rules += ' ma131';
    rules += ' la131';
    if (props.isQuiet) rules += ' _kb131';
    else rules += ' _ka131';
    rules += ' hb131';
    rules += ' _jf131';
    rules += ' gL131';
    rules += ' gl9131';
    return rules;
};
const $bed37377ec7a3a9e$export$f850895b287ef28e = /*#__PURE__*/ (0, $iLVc8$react.forwardRef)(function TableHeader({ columns: columns, dependencies: dependencies, children: children }, ref) {
    let scale = (0, $ee7b4c497f520c08$exports.useScale)();
    let { selectionBehavior: selectionBehavior, selectionMode: selectionMode, allowsDragging: allowsDragging } = (0, $iLVc8$reactariacomponentsTable.useTableOptions)();
    let { isQuiet: isQuiet, selectionStyle: selectionStyle } = (0, $iLVc8$react.useContext)($bed37377ec7a3a9e$var$InternalTableContext);
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let stringFormatter = (0, $iLVc8$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactariacomponentsTable.TableHeader), {
        // @ts-ignore
        ref: domRef,
        className: $bed37377ec7a3a9e$var$tableHeader,
        children: [
            allowsDragging && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsTable.Column), {
                // @ts-ignore
                isSticky: true,
                width: scale === 'medium' ? 16 : 20,
                minWidth: scale === 'medium' ? 16 : 20,
                className: $bed37377ec7a3a9e$var$selectAllCheckboxColumn({
                    isQuiet: isQuiet
                }),
                children: ({ isFocusVisible: isFocusVisible })=>/*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactjsxruntime.Fragment), {
                        children: [
                            isFocusVisible && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$CellFocusRing, {}),
                            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariaVisuallyHidden.VisuallyHidden), {
                                children: stringFormatter.format('table.drag')
                            })
                        ]
                    })
            }),
            selectionBehavior === 'toggle' && selectionStyle === 'checkbox' && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsTable.Column), {
                // Also isSticky prop is applied just for the layout, will decide what the RAC api should be later
                // @ts-ignore
                isSticky: true,
                width: scale === 'medium' ? 40 : 52,
                minWidth: scale === 'medium' ? 40 : 52,
                className: $bed37377ec7a3a9e$var$selectAllCheckboxColumn({
                    isQuiet: isQuiet
                }),
                children: ({ isFocusVisible: isFocusVisible })=>/*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactjsxruntime.Fragment), {
                        children: [
                            selectionMode === 'single' && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactjsxruntime.Fragment), {
                                children: [
                                    isFocusVisible && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$CellFocusRing, {}),
                                    /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$VisuallyHiddenSelectAllLabel, {})
                                ]
                            }),
                            selectionMode === 'multiple' && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $af04e099a53d3e85$exports.Checkbox), {
                                styles: $bed37377ec7a3a9e$var$selectAllCheckbox,
                                slot: "selection"
                            })
                        ]
                    })
            }),
            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariaCollection.Collection), {
                items: columns,
                dependencies: dependencies,
                children: children
            })
        ]
    });
});
function $bed37377ec7a3a9e$var$VisuallyHiddenSelectAllLabel() {
    let checkboxProps = (0, $iLVc8$reactariacomponentsslots.useSlottedContext)((0, $iLVc8$reactariacomponentsCheckbox.CheckboxContext), 'selection');
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariaVisuallyHidden.VisuallyHidden), {
        children: checkboxProps?.['aria-label']
    });
}
const $bed37377ec7a3a9e$var$commonCellStyles = {
    position: 'relative',
    color: '--rowTextColor',
    outlineStyle: 'none',
    paddingX: 16 // table-edge-to-content
};
const $bed37377ec7a3a9e$var$treeColumnStyles = {
    '--treeColumnPadding': {
        type: 'width',
        value: {
            default: 16,
            isTreeColumnWithNoChildren: 36
        }
    },
    paddingStart: {
        default: 16,
        isTreeColumn: 'calc(var(--treeColumnPadding) + (var(--table-row-level, 1) - 1) * var(--indent))'
    }
};
const $bed37377ec7a3a9e$var$cell = function anonymous(props) {
    let rules = " ";
    rules += ' _Pc131';
    rules += ' pBvFblc131';
    rules += ' _Le131';
    if (props.isTreeColumn) rules += ' S7cyhDe131';
    else rules += ' Sf131';
    rules += ' Rf131';
    if (props.isTreeColumnWithNoChildren) rules += ' -WWxPD-Zv131';
    else rules += ' -WWxPD-Zu131';
    if (props.density === "spacious") rules += ' Mk131';
    else if (props.density === "compact") rules += ' Mi131';
    else rules += ' Mh131';
    rules += ' _oa131';
    rules += ' Fb131';
    rules += ' Za131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    rules += ' eb131';
    rules += ' sd131';
    return rules;
};
const $bed37377ec7a3a9e$var$divider = " sb131 _Pa131 ZceNfn131 Fb131 _zr131 gp95UEb131 _9c131";
const $bed37377ec7a3a9e$var$stickyCell = {
    backgroundColor: 'gray-25'
};
const $bed37377ec7a3a9e$var$checkboxCellStyle = " _Pc131 pBvFblc131 _Le131 Sf131 Rt131 gs9yXOb131 sd131 _cb131 eb131 _Cg131 FBJsHLe131 ha131";
const $bed37377ec7a3a9e$var$dragCellStyle = " _Pc131 pBvFblc131 _Le131 Sp131 Rp131 gs9yXOb131 _cb131 FBJsHLe131 ha131";
const $bed37377ec7a3a9e$var$cellContent = function anonymous(props) {
    let rules = " ";
    if (props.isSticky) rules += ' _Ne131';
    else rules += ' _Nc131';
    if (props.isSticky) rules += ' Pe131';
    else rules += ' Pc131';
    if (props.isSticky) rules += ' _Ya131';
    else rules += ' _Yb131';
    if (props.overflowMode === "wrap") rules += ' _7b131';
    else rules += ' _7c131';
    if (props.align === "end") rules += ' _Wb131';
    else if (props.align === "center") rules += ' _Wa131';
    else if (props.align === "start") rules += ' _Wd131';
    rules += ' Za131';
    rules += ' ZZbReHce4mUs131';
    rules += ' _Bb131';
    if (props.isSticky) rules += ' Td131';
    else rules += ' Tp131';
    if (props.isSticky) rules += ' Qd131';
    else rules += ' Qp131';
    if (props.isSticky) rules += ' Sd131';
    else rules += ' Sp131';
    if (props.isSticky) rules += ' Rd131';
    else rules += ' Rp131';
    if (props.isSticky) rules += ' Jy131';
    else rules += ' Jb131';
    if (props.isSticky) rules += ' Gy131';
    else rules += ' Gb131';
    if (props.isSticky) rules += ' Iy131';
    else rules += ' Ib131';
    if (props.isSticky) rules += ' Hy131';
    else rules += ' Hb131';
    if (props.isSticky) rules += ' gs9yXOb131';
    else rules += ' g9131';
    return rules;
};
const $bed37377ec7a3a9e$export$f6f0c3fe4ec306ea = /*#__PURE__*/ (0, $iLVc8$react.forwardRef)(function Cell(props, ref) {
    let { children: children, isSticky: isSticky, showDivider: showDivider = false, align: align, textValue: textValue, ...otherProps } = props;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let tableVisualOptions = (0, $iLVc8$react.useContext)($bed37377ec7a3a9e$var$InternalTableContext);
    textValue ||= typeof children === 'string' ? children : undefined;
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsTable.Cell), {
        ref: domRef,
        // Also isSticky prop is applied just for the layout, will decide what the RAC api should be later
        // @ts-ignore
        isSticky: isSticky,
        className: (renderProps)=>$bed37377ec7a3a9e$var$cell({
                ...renderProps,
                ...tableVisualOptions,
                isDivider: showDivider,
                isTreeColumnWithNoChildren: renderProps.isTreeColumn && !renderProps.hasChildItems
            }),
        textValue: textValue,
        ...otherProps,
        children: ({ id: id, isFocusVisible: isFocusVisible, hasChildItems: hasChildItems, isTreeColumn: isTreeColumn, isExpanded: isExpanded, isDisabled: isDisabled })=>/*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactjsxruntime.Fragment), {
                children: [
                    showDivider && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)("div", {
                        className: $bed37377ec7a3a9e$var$divider
                    }),
                    hasChildItems && isTreeColumn && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$ExpandableRowChevron, {
                        isDisabled: isDisabled,
                        isExpanded: isExpanded
                    }, id),
                    /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)("span", {
                        className: $bed37377ec7a3a9e$var$cellContent({
                            ...tableVisualOptions,
                            isSticky: isSticky,
                            align: align || 'start'
                        }),
                        children: children
                    }),
                    isFocusVisible && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$CellFocusRing, {})
                ]
            })
    });
});
const $bed37377ec7a3a9e$var$expandButton = function anonymous(props) {
    let rules = " ";
    rules += ' z8dW5Qb131';
    rules += ' y8dW5Qb131';
    rules += ' B8dW5Qb131';
    rules += ' A8dW5Qb131';
    if (props.isDisabled) {
        rules += ' pp131';
        rules += ' plc131';
    } else rules += ' p40ub4c131';
    rules += ' Fb131';
    rules += ' Zw131';
    rules += ' Id131';
    rules += ' sd131';
    rules += ' _wb131';
    rules += ' _cb131';
    rules += ' _Ca131';
    rules += ' _Le131';
    rules += ' ri131';
    if (props.isExpanded) {
        if (props.isRTL) rules += ' _Za131';
        else rules += ' _Zb131';
    }
    rules += ' Td131';
    rules += ' Qd131';
    rules += ' Sd131';
    rules += ' Rd131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' g9131';
    rules += ' _je131';
    rules += ' __ca131';
    if (props.isHidden) rules += ' _6b131';
    return rules;
};
function $bed37377ec7a3a9e$var$ExpandableRowChevron(props1) {
    let ref = (0, $iLVc8$react.useRef)(null);
    let { isExpanded: isExpanded, isHidden: isHidden } = props1;
    let { direction: direction } = (0, $iLVc8$reactariaI18nProvider.useLocale)();
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsButton.Button), {
        ...props1,
        ref: ref,
        slot: "chevron",
        className: (renderProps)=>$bed37377ec7a3a9e$var$expandButton({
                ...renderProps,
                isExpanded: isExpanded,
                isRTL: direction === 'rtl',
                isHidden: isHidden
            }),
        children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $1df9f1c9262ce5df$exports.default), {
            className: function anonymous(props) {
                let rules = " ";
                if (props.direction === "rtl") {
                    rules += ' -Z4Pn4c-a131';
                    rules += ' __Ha131';
                } else if (props.direction === "ltr") {
                    rules += ' -Z4Pn4c-b131';
                    rules += ' __Ha131';
                }
                if (props.direction === "rtl") {
                    rules += ' -_04Pn4c-a131';
                    rules += ' __Ha131';
                } else if (props.direction === "ltr") {
                    rules += ' -_04Pn4c-b131';
                    rules += ' __Ha131';
                }
                rules += ' -_8sjo0b-t5ZbAob131';
                return rules;
            }({
                direction: direction
            })
        })
    });
}
const $bed37377ec7a3a9e$var$editableCell = function anonymous(props) {
    let rules = " ";
    rules += ' _Pc131';
    if (props.isDisabled) {
        rules += ' pp131';
        rules += ' plc131';
    } else {
        if (props.isSaving) {
            if (props.isPressed) rules += ' pt131';
            else if (props.isFocusVisible) rules += ' pt131';
            else if (props.isHovered) rules += ' pt131';
            else rules += ' pv131';
        } else {
            if (props.isPressed) rules += ' po131';
            else if (props.isFocusVisible) rules += ' po131';
            else if (props.isHovered) rules += ' po131';
            else rules += ' pt131';
        }
        rules += ' plb131';
    }
    rules += ' _Le131';
    if (props.isTreeColumn) rules += ' S7cyhDe131';
    else rules += ' Sf131';
    rules += ' Rf131';
    if (props.isTreeColumnWithNoChildren) rules += ' -WWxPD-Zv131';
    else rules += ' -WWxPD-Zu131';
    rules += ' Tu7Fubd131';
    rules += ' Qu7Fubd131';
    rules += ' _oa131';
    rules += ' FBJsHLe131';
    rules += ' Za131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    rules += ' eb131';
    rules += ' sd131';
    if (props.isDivider) rules += ' _jf131';
    else rules += ' _je131';
    if (props.isDivider) rules += ' lb131';
    else rules += ' la131';
    rules += ' iq131';
    rules += ' ila131';
    return rules;
};
let $bed37377ec7a3a9e$var$editPopover = function anonymous(props, overrides) {
    let rules = " ";
    let minWidth = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "N") minWidth = true;
        rules += p[0];
    }
    if (props.colorScheme === "dark") rules += ' -ICUGx-_pa131';
    else if (props.colorScheme === "light") rules += ' -ICUGx-_pb131';
    else if (props.colorScheme === "light dark") rules += ' -ICUGx-_pc131';
    rules += ' _pK8eoyd131';
    if (props.colorScheme === "dark") rules += ' -NvXmrd-_ZLmU131';
    else if (props.colorScheme === "light") rules += ' -NvXmrd-_ZN8Gi4c131';
    else if (props.colorScheme === "light dark") {
        rules += ' -NvXmrd-_ZN8Gi4c131';
        rules += ' -NvXmrd-D1v0V_ZLmU131';
    }
    if (props.colorScheme === "dark") rules += ' -j9blY-_ZN8Gi4c131';
    else if (props.colorScheme === "light") rules += ' -j9blY-_ZLmU131';
    else if (props.colorScheme === "light dark") {
        rules += ' -j9blY-_ZLmU131';
        rules += ' -j9blY-D1v0V_ZN8Gi4c131';
    }
    rules += ' -Ovbhqd-g_g131';
    rules += ' gEzkoQe131';
    rules += ' kb131';
    rules += ' jb131';
    rules += ' _nLeasBb131';
    rules += ' _jf131';
    rules += ' _kb131';
    rules += ' hb131';
    rules += ' mb131';
    rules += ' lb131';
    rules += ' ir131';
    rules += ' ila131';
    rules += ' _ob131';
    rules += ' _Bb131';
    if (props.isExiting) rules += ' _Ob131';
    rules += ' _Le131';
    if (!minWidth) rules += ' NuHSBze131';
    rules += ' Tt131';
    rules += ' Qt131';
    rules += ' St131';
    rules += ' Rt131';
    rules += ' sd131';
    rules += ' eb131';
    return rules;
};
const $bed37377ec7a3a9e$export$d27c35c47f085992 = /*#__PURE__*/ (0, $iLVc8$react.forwardRef)(function EditableCell(props, ref) {
    let { children: children, showDivider: showDivider = false, textValue: textValue, isSaving: isSaving, ...otherProps } = props;
    let tableVisualOptions = (0, $iLVc8$react.useContext)($bed37377ec7a3a9e$var$InternalTableContext);
    let domRef = (0, $iLVc8$reactariauseObjectRef.useObjectRef)(ref);
    textValue ||= typeof children === 'string' ? children : undefined;
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsTable.Cell), {
        ref: domRef,
        className: (renderProps)=>$bed37377ec7a3a9e$var$editableCell({
                ...renderProps,
                ...tableVisualOptions,
                isDivider: showDivider,
                isSaving: isSaving,
                isTreeColumnWithNoChildren: renderProps.isTreeColumn && !renderProps.hasChildItems
            }),
        textValue: textValue,
        ...otherProps,
        children: ({ id: id, isFocusVisible: isFocusVisible, hasChildItems: hasChildItems, isTreeColumn: isTreeColumn, isExpanded: isExpanded, isDisabled: isDisabled })=>/*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactjsxruntime.Fragment), {
                children: [
                    hasChildItems && isTreeColumn && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$ExpandableRowChevron, {
                        isDisabled: isDisabled,
                        isExpanded: isExpanded
                    }, id),
                    /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$EditableCellInner, {
                        ...props,
                        isFocusVisible: isFocusVisible,
                        cellRef: domRef
                    })
                ]
            })
    });
});
const $bed37377ec7a3a9e$var$nonTextInputTypes = new Set([
    'checkbox',
    'radio',
    'range',
    'color',
    'file',
    'image',
    'button',
    'submit',
    'reset'
]);
function $bed37377ec7a3a9e$var$EditableCellInner(props1) {
    let { children: children, align: align, renderEditing: renderEditing, isSaving: isSaving, onSubmit: onSubmit, isFocusVisible: isFocusVisible, cellRef: cellRef, action: action, onCancel: onCancel } = props1;
    let [isOpen, setIsOpen] = (0, $iLVc8$react.useState)(false);
    let popoverRef = (0, $iLVc8$react.useRef)(null);
    let formRef = (0, $iLVc8$react.useRef)(null);
    let [triggerWidth, setTriggerWidth] = (0, $iLVc8$react.useState)(0);
    let [tableWidth, setTableWidth] = (0, $iLVc8$react.useState)(0);
    let [verticalOffset, setVerticalOffset] = (0, $iLVc8$react.useState)(0);
    let tableVisualOptions = (0, $iLVc8$react.useContext)($bed37377ec7a3a9e$var$InternalTableContext);
    let stringFormatter = (0, $iLVc8$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    let dialogRef = (0, $iLVc8$react.useRef)(null);
    let { density: density } = (0, $iLVc8$react.useContext)($bed37377ec7a3a9e$var$InternalTableContext);
    let size = 'M';
    if (density === 'compact') size = 'S';
    else if (density === 'spacious') size = 'L';
    // Popover positioning
    (0, $iLVc8$reactariaprivateutilsuseLayoutEffect.useLayoutEffect)(()=>{
        if (!isOpen) return;
        let width = cellRef.current?.clientWidth || 0;
        let cell = cellRef.current;
        let boundingRect = cell?.parentElement?.getBoundingClientRect();
        let verticalOffset = (boundingRect?.top ?? 0) - (boundingRect?.bottom ?? 0);
        let tableWidth = cellRef.current?.closest('[role="grid"],[role="treegrid"]')?.clientWidth || 0;
        setTriggerWidth(width);
        setVerticalOffset(verticalOffset);
        setTableWidth(tableWidth);
    }, [
        cellRef,
        density,
        isOpen
    ]);
    // Auto select the entire text range of the autofocused input on overlay opening
    // Maybe replace with FocusScope or one of those utilities
    (0, $iLVc8$react.useEffect)(()=>{
        if (isOpen) {
            let activeElement = (0, $iLVc8$reactariaprivateutilsshadowdomDOMFunctions.getActiveElement)((0, $iLVc8$reactariaprivateutilsdomHelpers.getOwnerDocument)(formRef.current));
            if (activeElement && (0, $iLVc8$reactariaprivateutilsshadowdomDOMFunctions.nodeContains)(formRef.current, activeElement) && // not going to handle contenteditable https://stackoverflow.com/questions/6139107/programmatically-select-text-in-a-contenteditable-html-element
            // seems like an edge case anyways
            (activeElement instanceof HTMLInputElement && !$bed37377ec7a3a9e$var$nonTextInputTypes.has(activeElement.type) || activeElement instanceof HTMLTextAreaElement) && typeof activeElement.select === 'function') activeElement.select();
        }
    }, [
        isOpen
    ]);
    let cancel = (0, $iLVc8$react.useCallback)(()=>{
        setIsOpen(false);
        onCancel?.();
    }, [
        onCancel
    ]);
    let isMobile = !(0, $3112837ad519e72a$exports.useMediaQuery)('(hover: hover) and (pointer: fine)');
    // Can't differentiate between Dialog click outside dismissal and Escape key dismissal
    let prevIsOpen = (0, $iLVc8$react.useRef)(isOpen);
    (0, $iLVc8$react.useEffect)(()=>{
        let dialog = dialogRef.current?.UNSAFE_getDOMNode();
        if (isOpen && dialog && !prevIsOpen.current) {
            let handler = (e)=>{
                if (e.key === 'Escape') {
                    cancel();
                    e.stopPropagation();
                    e.preventDefault();
                }
            };
            dialog.addEventListener('keydown', handler);
            prevIsOpen.current = isOpen;
            return ()=>{
                dialog.removeEventListener('keydown', handler);
            };
        }
        prevIsOpen.current = isOpen;
    }, [
        isOpen,
        cancel
    ]);
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactariacomponentsslots.Provider), {
        values: [
            [
                (0, $iLVc8$reactariacomponentsButton.ButtonContext),
                null
            ],
            [
                (0, $6e265ff388155b91$exports.ActionButtonContext),
                {
                    slots: {
                        [(0, $iLVc8$reactariacomponentsslots.DEFAULT_SLOT)]: {},
                        edit: {
                            onPress: ()=>setIsOpen(true),
                            isPending: isSaving,
                            isQuiet: !isSaving,
                            size: size,
                            excludeFromTabOrder: true,
                            styles: function anonymous(props) {
                                let rules = " ";
                                if (props.isForcedVisible) rules += ' _6c131';
                                else rules += ' _6b131';
                                rules += ' _6JGzCfcc131';
                                rules += ' _6hcIDbbc131';
                                rules += ' _6sc131';
                                return rules;
                            }({
                                isForcedVisible: isOpen || !!isSaving
                            })
                        }
                    }
                }
            ]
        ],
        children: [
            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)("span", {
                className: $bed37377ec7a3a9e$var$cellContent({
                    ...tableVisualOptions,
                    align: align || 'start'
                }),
                children: children
            }),
            isFocusVisible && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$CellFocusRing, {}),
            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactariacomponentsslots.Provider), {
                values: [
                    [
                        (0, $6e265ff388155b91$exports.ActionButtonContext),
                        null
                    ]
                ],
                children: [
                    !isMobile && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsPopover.Popover), {
                        isOpen: isOpen,
                        onOpenChange: setIsOpen,
                        ref: popoverRef,
                        shouldCloseOnInteractOutside: ()=>{
                            if (!(0, $iLVc8$reactariaprivateutilsshadowdomDOMFunctions.isFocusWithin)(popoverRef.current)) return false;
                            formRef.current?.requestSubmit();
                            return false;
                        },
                        triggerRef: cellRef,
                        "aria-label": props1['aria-label'] ?? stringFormatter.format('table.editCell'),
                        offset: verticalOffset,
                        placement: "bottom start",
                        style: {
                            minWidth: `min(${triggerWidth}px, ${tableWidth}px)`,
                            maxWidth: `${tableWidth}px`,
                            // Override default z-index from useOverlayPosition. We use isolation: isolate instead.
                            zIndex: undefined
                        },
                        className: $bed37377ec7a3a9e$var$editPopover,
                        children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsslots.Provider), {
                            values: [
                                [
                                    (0, $iLVc8$reactariacomponentsDialog.OverlayTriggerStateContext),
                                    null
                                ]
                            ],
                            children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactariacomponentsForm.Form), {
                                ref: formRef,
                                action: action,
                                onSubmit: (e)=>{
                                    onSubmit?.(e);
                                    setIsOpen(false);
                                },
                                className: " Za131 sd131 ed131 Ul131 ql131",
                                style: {
                                    '--input-width': `calc(${triggerWidth}px - 32px)`
                                },
                                children: [
                                    renderEditing(),
                                    /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)("div", {
                                        className: " sd131 _tc131 ea131 _va131 _ua131",
                                        children: [
                                            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $6e265ff388155b91$exports.ActionButton), {
                                                isQuiet: true,
                                                onPress: cancel,
                                                "aria-label": stringFormatter.format('table.cancel'),
                                                children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $8ecd53081d7d50cc$exports.default), {})
                                            }),
                                            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $6e265ff388155b91$exports.ActionButton), {
                                                isQuiet: true,
                                                type: "submit",
                                                "aria-label": stringFormatter.format('table.save'),
                                                children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $c3e72d94a25defe0$exports.default), {})
                                            })
                                        ]
                                    })
                                ]
                            })
                        })
                    }),
                    isMobile && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $fc196655d58ce18c$exports.DialogContainer), {
                        onDismiss: ()=>formRef.current?.requestSubmit(),
                        children: isOpen && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $a05aa0b0345f1846$exports.CustomDialog), {
                            ref: dialogRef,
                            isDismissible: true,
                            isKeyboardDismissDisabled: true,
                            "aria-label": props1['aria-label'] ?? stringFormatter.format('table.editCell'),
                            children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactariacomponentsForm.Form), {
                                ref: formRef,
                                action: action,
                                onSubmit: (e)=>{
                                    onSubmit?.(e);
                                    setIsOpen(false);
                                },
                                className: " Za131 sd131 _ta131 ed131 Ul131 ql131",
                                children: [
                                    renderEditing(),
                                    /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $e1016b8947dbadb8$exports.ButtonGroup), {
                                        align: "end",
                                        styles: " _dd131",
                                        children: [
                                            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $25d06cf8d4e72761$exports.Button), {
                                                onPress: cancel,
                                                variant: "secondary",
                                                fillStyle: "outline",
                                                children: "Cancel"
                                            }),
                                            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $25d06cf8d4e72761$exports.Button), {
                                                type: "submit",
                                                variant: "accent",
                                                children: "Save"
                                            })
                                        ]
                                    })
                                ]
                            })
                        })
                    })
                ]
            })
        ]
    });
}
// Use color-mix instead of transparency so sticky cells work correctly.
const $bed37377ec7a3a9e$var$selectedBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 7%)]";
const $bed37377ec7a3a9e$var$selectedActiveBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 10%)]";
// TODO: I made these up, not sure if there is a great way to go from v3 values to
// S2. Overall the root drop color should be lighter than the row color during a root drop
// which should be lighter than a selected row during root drop. Those root drop row colors should also be darker
// than if the row is the drop target itself
const $bed37377ec7a3a9e$var$rootDropRowBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 17%)]";
const $bed37377ec7a3a9e$var$rootDropSelectedRowBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 28%)]";
const $bed37377ec7a3a9e$var$rowDropBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 10%)]";
const $bed37377ec7a3a9e$var$rowDropSelectedBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 15%)]";
const $bed37377ec7a3a9e$var$rootRowDropStyles = {
    default: $bed37377ec7a3a9e$var$rootDropRowBackground,
    isSelected: $bed37377ec7a3a9e$var$rootDropSelectedRowBackground,
    forcedColors: 'Background'
};
const $bed37377ec7a3a9e$var$rowDropStyles = {
    default: $bed37377ec7a3a9e$var$rowDropBackground,
    isSelected: $bed37377ec7a3a9e$var$rowDropSelectedBackground,
    forcedColors: 'Background'
};
const $bed37377ec7a3a9e$var$rowBackgroundColor = {
    default: {
        default: 'gray-25',
        isQuiet: '--s2-container-bg'
    },
    forcedColors: {
        default: 'Background'
    },
    isHovered: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 7%)]",
    isPressed: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 10%)]",
    isSelected: {
        default: $bed37377ec7a3a9e$var$selectedBackground,
        isFocusVisibleWithin: $bed37377ec7a3a9e$var$selectedActiveBackground,
        isHovered: $bed37377ec7a3a9e$var$selectedActiveBackground,
        isPressed: $bed37377ec7a3a9e$var$selectedActiveBackground // table-selected-row-background-color, opacity /15
    },
    selectionStyle: {
        highlight: {
            default: 'gray-25',
            isQuiet: '--s2-container-bg',
            isHovered: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 7%)]",
            isPressed: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 10%)]",
            isSelected: {
                default: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 10%)]",
                isHovered: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 15%)]",
                isPressed: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 15%)]",
                forcedColors: 'Highlight'
            }
        }
    },
    isInFooter: 'gray-200',
    ':is([role="grid"][data-drop-target] *)': $bed37377ec7a3a9e$var$rootRowDropStyles,
    isDropTarget: $bed37377ec7a3a9e$var$rowDropStyles
};
const $bed37377ec7a3a9e$var$rowTextColor = {
    default: {
        "default": "neutral-subdued",
        isHovered: "neutral-subdued:hovered",
        isFocusVisible: "neutral-subdued:focused",
        isPressed: "neutral-subdued:pressed"
    },
    isSelected: {
        "default": "neutral",
        isHovered: "neutral:hovered",
        isFocusVisible: "neutral:focused",
        isPressed: "neutral:pressed"
    },
    forcedColors: {
        default: 'ButtonText',
        isSelected: {
            selectionStyle: {
                highlight: 'HighlightText'
            }
        }
    },
    isDisabled: {
        default: 'disabled',
        forcedColors: 'GrayText'
    },
    isInFooter: 'neutral'
};
const $bed37377ec7a3a9e$var$row = function anonymous(props) {
    let rules = " ";
    rules += ' Fb131';
    rules += ' _Pc131';
    rules += ' _oa131';
    rules += ' gs9yXOb131';
    if (props.isDropTarget) {
        if (props.isSelected) rules += ' -qH4xad-gaZwDBe131';
        else rules += ' -qH4xad-glzvDBe131';
        rules += ' -qH4xad-lga131';
    } else {
        if (props.isInFooter) rules += ' -qH4xad-gF131';
        else if (props.selectionStyle === "highlight") {
            if (props.isSelected) {
                if (props.isPressed) rules += ' -qH4xad-gaZwDBe131';
                else if (props.isHovered) rules += ' -qH4xad-gaZwDBe131';
                else rules += ' -qH4xad-glzvDBe131';
                rules += ' -qH4xad-lge131';
            } else if (props.isPressed) rules += ' -qH4xad-gRfT7e131';
            else if (props.isHovered) rules += ' -qH4xad-gj1B7i131';
            else if (props.isQuiet) rules += ' -qH4xad-gEzkoQe131';
            else rules += ' -qH4xad-gX131';
        } else if (props.isSelected) {
            if (props.isPressed) rules += ' -qH4xad-gRfT7e131';
            else if (props.isHovered) rules += ' -qH4xad-gRfT7e131';
            else if (props.isFocusVisibleWithin) rules += ' -qH4xad-gRfT7e131';
            else rules += ' -qH4xad-gj1B7i131';
        } else if (props.isPressed) rules += ' -qH4xad-gRfT7e131';
        else if (props.isHovered) rules += ' -qH4xad-gj1B7i131';
        else {
            if (props.isQuiet) rules += ' -qH4xad-gEzkoQe131';
            else rules += ' -qH4xad-gX131';
            rules += ' -qH4xad-lga131';
        }
        if (props.isSelected) rules += ' -qH4xad-RQaNpdgubHDBe131';
        else rules += ' -qH4xad-RQaNpdgiyxDBe131';
        rules += ' -qH4xad-RQaNpdlga131';
    }
    if (props.isInFooter) rules += ' -Fadoyc-pt131';
    else if (props.isDisabled) {
        rules += ' -Fadoyc-pp131';
        rules += ' -Fadoyc-lpc131';
    } else {
        if (props.isSelected) {
            if (props.isPressed) rules += ' -Fadoyc-po131';
            else if (props.isFocusVisible) rules += ' -Fadoyc-po131';
            else if (props.isHovered) rules += ' -Fadoyc-po131';
            else rules += ' -Fadoyc-pt131';
        } else {
            if (props.isPressed) rules += ' -Fadoyc-pt131';
            else if (props.isFocusVisible) rules += ' -Fadoyc-pt131';
            else if (props.isHovered) rules += ' -Fadoyc-pt131';
            else rules += ' -Fadoyc-pv131';
        }
        if (props.isSelected) {
            if (props.selectionStyle === "highlight") rules += ' -Fadoyc-lpe131';
        } else rules += ' -Fadoyc-lpb131';
    }
    rules += ' _Le131';
    if (props.selectionStyle === "highlight") {
        if (props.isNextSelected) rules += ' -mtnI9b-kd131';
        else if (props.isSelected) rules += ' -mtnI9b-ksmOfn131';
        else rules += ' -mtnI9b-kd131';
    } else rules += ' -mtnI9b-kd131';
    if (props.selectionStyle === "highlight") {
        if (props.isPrevSelected) rules += ' -GkFu3c-od131';
        else if (props.isSelected) rules += ' -GkFu3c-osmOfn131';
        else rules += ' -GkFu3c-od131';
    } else rules += ' -GkFu3c-od131';
    rules += ' kM1g6bb131';
    rules += ' jM1g6bb131';
    rules += ' ow2GT5b131';
    rules += ' nw2GT5b131';
    if (props.isPrevSelected) rules += ' -K0jSNe-ZDWMfn131';
    else {
        if (props.selectionStyle === "highlight") rules += ' -K0jSNe-Ze131';
        else if (props.selectionStyle === "checkbox") rules += ' -K0jSNe-ZDWMfn131';
    }
    if (props.isNextSelected) rules += ' -ACwbMe-ZDWMfn131';
    else {
        if (props.selectionStyle === "highlight") rules += ' -ACwbMe-Ze131';
        else if (props.selectionStyle === "checkbox") rules += ' -ACwbMe-ZDWMfn131';
    }
    if (props.selectionStyle === "highlight") rules += ' -YiueL-Ze131';
    else if (props.selectionStyle === "checkbox") rules += ' -YiueL-ZDWMfn131';
    rules += ' _ka131';
    if (props.selectionStyle === "checkbox") rules += ' hb131';
    else if (props.selectionStyle === "highlight") rules += ' ha131';
    rules += ' ma131';
    rules += ' la131';
    rules += ' _jf131';
    if (props.selectionStyle === "checkbox") {
        rules += ' iq131';
        rules += ' ila131';
    } else if (props.selectionStyle === "highlight") rules += ' iA131';
    rules += ' -JW3yBe-iq131';
    if (props.isSelected) rules += ' -JW3yBe-lid131';
    else rules += ' -JW3yBe-lia131';
    rules += ' -wFhyBe-iw131';
    rules += ' -wFhyBe-lid131';
    if (props.isSelected) rules += ' -EH8uVb-iuZovEb131';
    else rules += ' -EH8uVb-iA131';
    if (props.selectionStyle === "highlight") {
        if (props.isNextSelected) rules += ' _nCOSyG131';
        else rules += ' _nAMrXze131';
    }
    if (props.isFocusVisible) rules += ' _nlGDXwhb131';
    if (props.isInFooter) {
        rules += ' xd131';
        rules += ' _xa131';
        rules += ' xX0cczbc131';
        rules += ' _xX0cczba131';
    } else {
        rules += ' xb131';
        rules += ' _xa131';
    }
    rules += ' _Bb131';
    rules += ' _yb131';
    return rules;
};
// Sticky cells (the drag cell, and the checkbox cell when present) get an inline z-index=2 applied by the virtualizer's layout
// To ensure that the highlight selection border is painted above the stick cells, set z-index to 3
const $bed37377ec7a3a9e$var$highlightSelectionBorder = "XvgEBe";
const $bed37377ec7a3a9e$var$focusIndicator = "_7GNHP";
const $bed37377ec7a3a9e$var$selectionCheckbox = " _6c131 _6Lcn87b131";
const $bed37377ec7a3a9e$export$b59bdbef9ce70de2 = /*#__PURE__*/ (0, $iLVc8$react.forwardRef)(function Row({ id: id, columns: columns, children: children, dependencies: dependencies = [], ...otherProps }, ref) {
    let { selectionBehavior: selectionBehavior, selectionMode: selectionMode, allowsDragging: allowsDragging } = (0, $iLVc8$reactariacomponentsTable.useTableOptions)();
    let { selectionStyle: selectionStyle, ...tableVisualOptions } = (0, $iLVc8$react.useContext)($bed37377ec7a3a9e$var$InternalTableContext);
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let isInFooter = (0, $iLVc8$react.useContext)($bed37377ec7a3a9e$var$FooterContext);
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsxs)((0, $iLVc8$reactariacomponentsTable.Row), {
        // @ts-ignore
        ref: domRef,
        id: id,
        dependencies: [
            ...dependencies,
            columns
        ],
        isDisabled: isInFooter,
        disabledBehavior: isInFooter ? 'selection' : undefined,
        className: (renderProps)=>$bed37377ec7a3a9e$var$row({
                ...renderProps,
                ...tableVisualOptions,
                selectionStyle: selectionStyle,
                isInFooter: isInFooter,
                isNextSelected: $bed37377ec7a3a9e$export$801a7cfaf3e363f5(renderProps.id, renderProps.state),
                isPrevSelected: $bed37377ec7a3a9e$export$aebccab86af12e73(renderProps.id, renderProps.state)
            }) + (renderProps.isFocusVisible || renderProps.isDropTarget ? ' ' + $bed37377ec7a3a9e$var$focusIndicator : '') + (selectionStyle === 'highlight' ? ' ' + $bed37377ec7a3a9e$var$highlightSelectionBorder : ''),
        ...otherProps,
        children: [
            allowsDragging && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsTable.Cell), {
                // @ts-ignore
                isSticky: true,
                className: $bed37377ec7a3a9e$var$dragCellStyle,
                children: ({ isFocusVisibleWithinRow: isFocusVisibleWithinRow })=>!(otherProps.isDisabled && tableVisualOptions.disabledBehavior === 'all') && /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $f1f08ef5de95f8eb$exports.DragHandleButton), {
                        isFocusVisibleWithin: isFocusVisibleWithinRow
                    })
            }),
            selectionMode !== 'none' && selectionBehavior === 'toggle' && selectionStyle === 'checkbox' && // Not sure what we want to do with this className, in Cell it currently overrides the className that would have been applied.
            // The `spread` otherProps must be after className in Cell.
            // @ts-ignore
            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$export$f6f0c3fe4ec306ea, {
                isSticky: true,
                className: $bed37377ec7a3a9e$var$checkboxCellStyle,
                children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $af04e099a53d3e85$exports.Checkbox), {
                    slot: "selection",
                    styles: $bed37377ec7a3a9e$var$selectionCheckbox
                })
            }),
            /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariaCollection.Collection), {
                items: columns,
                dependencies: [
                    ...dependencies,
                    columns
                ],
                children: children
            })
        ]
    });
});
const $bed37377ec7a3a9e$var$FooterContext = /*#__PURE__*/ (0, $iLVc8$react.createContext)(false);
const $bed37377ec7a3a9e$export$1f116082bba1f9a8 = /*#__PURE__*/ (0, $iLVc8$react.forwardRef)(function TableFooter(props, ref) {
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    return /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)($bed37377ec7a3a9e$var$FooterContext.Provider, {
        value: true,
        children: /*#__PURE__*/ (0, $iLVc8$reactjsxruntime.jsx)((0, $iLVc8$reactariacomponentsTable.TableFooter), {
            ...props,
            ref: domRef
        })
    });
});
function $bed37377ec7a3a9e$export$801a7cfaf3e363f5(id, state) {
    if (id == null || !state) return false;
    let keyAfter = state.collection.getKeyAfter(id);
    return keyAfter != null && state.selectionManager.isSelected(keyAfter);
}
function $bed37377ec7a3a9e$export$aebccab86af12e73(id, state) {
    if (id == null || !state) return false;
    let keyBefore = state.collection.getKeyBefore(id);
    return keyBefore != null && state.selectionManager.isSelected(keyBefore);
}
