// @react-spectrum/s2/dist/private/TabsPicker.cjs
require("./TabsPicker.css");
var $e991cbcdf82ced71$exports = require("./CenterBaseline.cjs");
var $e741ea6b88ce4866$exports = require("./Menu.cjs");
var $dba2b18fc5727f9b$exports = require("./Checkmark.cjs");
var $1df9f1c9262ce5df$exports = require("./Chevron.cjs");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $493371ef39bf7a55$exports = require("./Popover.cjs");
var $2061c83559b50a66$exports = require("./pressScale.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $bfa4962d90c8af48$exports = require("./Form.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $7pV4n$reactjsxruntime = require("react/jsx-runtime");
var $7pV4n$reactariacomponentsSelect = require("react-aria-components/Select");
var $7pV4n$reactariacomponentsButton = require("react-aria-components/Button");
var $7pV4n$reactariacomponentsslots = require("react-aria-components/slots");
var $7pV4n$reactariacomponentsListBox = require("react-aria-components/ListBox");
var $7pV4n$react = require("react");
var $7pV4n$reactariaI18nProvider = require("react-aria/I18nProvider");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "Picker", () => $9d2c38978395d44a$export$ba25329847403e11);
$parcel$export(module.exports, "PickerItem", () => $9d2c38978395d44a$export$d601881f38163e28);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 


















const $9d2c38978395d44a$export$b72b8dade9393cd5 = /*#__PURE__*/ (0, $7pV4n$react.createContext)(null);
const $9d2c38978395d44a$var$inputButton = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    rules += ' zXcEWr131';
    rules += ' yXcEWr131';
    rules += ' BXcEWr131';
    rules += ' AXcEWr131';
    if (props.size === "XL") rules += ' Nl131';
    else if (props.size === "L") rules += ' Nj131';
    else if (props.size === "S") rules += ' Nf131';
    else if (props.size === "XS") rules += ' Ne131';
    else rules += ' Nk131';
    if (props.isQuiet) rules += ' _qd131';
    else rules += ' _qtb131';
    if (props.size === "XL") rules += ' -Anxdcc-Zt131';
    else if (props.size === "L") rules += ' -Anxdcc-ZStdtne131';
    else if (props.size === "S") rules += ' -Anxdcc-Zr131';
    else rules += ' -Anxdcc-Z3fn9H131';
    rules += ' __v6TNLWb131';
    rules += ' _Pc131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-c131';
    rules += ' vx131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    if (props.isDisabled) rules += ' pp131';
    else {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    }
    rules += ' sd131';
    rules += ' _Wd131';
    rules += ' _je131';
    rules += ' oa131';
    rules += ' na131';
    rules += ' ka131';
    rules += ' ja131';
    rules += ' eb131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' qc131';
    rules += ' qsd131';
    rules += ' SMBFGYc131';
    rules += ' RfBFGYc131';
    rules += ' g9131';
    if (props.isQuiet) rules += ' Ln131';
    rules += ' __ca131';
    if (props.density === "compact") rules += ' Fx131';
    else rules += ' Fz131';
    rules += ' _oa131';
    rules += ' -S_-Sd131';
    rules += ' -R_-Rd131';
    return rules;
};
let $9d2c38978395d44a$export$b1e5508a851be14d = function anonymous(props) {
    let rules = " ";
    rules += ' _Le131';
    rules += ' se131';
    rules += ' Deb5jcb131';
    rules += ' _oa131';
    rules += ' Za131';
    rules += ' K40ub4c131';
    rules += ' _Na131';
    rules += ' Pa131';
    rules += ' Tt131';
    rules += ' Qt131';
    rules += ' St131';
    rules += ' Rt131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    if (props.size === "XL") {
        rules += ' -_6BNtrc-e131';
        rules += ' vx131';
    } else if (props.size === "L") {
        rules += ' -_6BNtrc-d131';
        rules += ' vx131';
    } else if (props.size === "S") {
        rules += ' -_6BNtrc-b131';
        rules += ' vx131';
    } else if (props.size === "XS") {
        rules += ' -_6BNtrc-a131';
        rules += ' vx131';
    } else {
        rules += ' -_6BNtrc-c131';
        rules += ' vx131';
    }
    return rules;
};
const $9d2c38978395d44a$var$valueStyles = " _ua131 _Nc131 Pc131 _Yb131 _7c131 sd131 eb131 Fb131";
const $9d2c38978395d44a$var$iconStyles = " _va131 _Sd131 -_8sjo0b-t5ZbAob131";
const $9d2c38978395d44a$var$iconCenterWrapper = function anonymous(props) {
    let rules = " ";
    rules += ' sd131';
    rules += ' z2U5Crc131';
    rules += ' y2U5Crc131';
    rules += ' B2U5Crc131';
    rules += ' A2U5Crc131';
    if (props.labelBehavior === "hide") rules += ' SssxDec131';
    return rules;
};
let $9d2c38978395d44a$var$InsideSelectValueContext = /*#__PURE__*/ (0, $7pV4n$react.createContext)(false);
function $9d2c38978395d44a$var$Picker(props1, ref) {
    [props1, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props1, ref, $9d2c38978395d44a$export$b72b8dade9393cd5);
    let domRef = (0, $150382364eecc4bc$exports.useFocusableRef)(ref);
    props1 = (0, $bfa4962d90c8af48$exports.useFormProps)(props1);
    let { direction: direction = 'bottom', align: align = 'start', shouldFlip: shouldFlip = true, children: children, items: items, density: density, labelBehavior: labelBehavior = 'show', valueId: valueId, ...pickerProps } = props1;
    let isQuiet = true;
    const menuOffset = 6;
    const size = 'M';
    let ariaLabelledby = props1['aria-labelledby'] ?? '';
    let { direction: dir } = (0, $7pV4n$reactariaI18nProvider.useLocale)();
    let RTLFlipOffset = dir === 'rtl' ? -1 : 1;
    return /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsxs)("div", {
        children: [
            /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)((0, $7pV4n$reactariacomponentsSelect.Select), {
                ...pickerProps,
                className: "",
                "aria-labelledby": `${labelBehavior === 'hide' ? valueId : ''} ${ariaLabelledby}`,
                children: ({ isOpen: isOpen })=>/*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsxs)((0, $7pV4n$reactjsxruntime.Fragment), {
                        children: [
                            /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldLabel), {
                                isQuiet: isQuiet
                            }),
                            /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsxs)((0, $7pV4n$reactariacomponentsButton.Button), {
                                ref: domRef,
                                style: (renderProps)=>(0, $2061c83559b50a66$exports.pressScale)(domRef)(renderProps),
                                // Prevent press scale from sticking while Picker is open.
                                // @ts-ignore
                                isPressed: false,
                                className: (renderProps)=>$9d2c38978395d44a$var$inputButton({
                                        ...renderProps,
                                        size: 'M',
                                        isOpen: isOpen,
                                        isQuiet: isQuiet,
                                        density: density
                                    }),
                                children: [
                                    /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)((0, $7pV4n$reactariacomponentsSelect.SelectValue), {
                                        className: $9d2c38978395d44a$var$valueStyles + ' ' + "WzII9b",
                                        children: ({ defaultChildren: defaultChildren })=>{
                                            return /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)((0, $7pV4n$reactariacomponentsslots.Provider), {
                                                values: [
                                                    [
                                                        (0, $bde97c91243ed164$exports.IconContext),
                                                        {
                                                            slots: {
                                                                icon: {
                                                                    render: (0, $e991cbcdf82ced71$exports.centerBaseline)({
                                                                        slot: 'icon',
                                                                        styles: $9d2c38978395d44a$var$iconCenterWrapper({
                                                                            labelBehavior: labelBehavior
                                                                        })
                                                                    }),
                                                                    styles: (0, $e741ea6b88ce4866$exports.icon)
                                                                }
                                                            }
                                                        }
                                                    ],
                                                    [
                                                        (0, $6367bc87eb7d24ad$exports.TextContext),
                                                        {
                                                            slots: {
                                                                // Default slot is useful when converting other collections to PickerItems.
                                                                [(0, $7pV4n$reactariacomponentsslots.DEFAULT_SLOT)]: {
                                                                    id: valueId,
                                                                    styles: function anonymous(props) {
                                                                        let rules = " ";
                                                                        if (props.labelBehavior === "hide") rules += ' sk131';
                                                                        else rules += ' sb131';
                                                                        rules += ' _ub131';
                                                                        rules += ' _Nc131';
                                                                        rules += ' Pc131';
                                                                        rules += ' _Yb131';
                                                                        rules += ' _7c131';
                                                                        return rules;
                                                                    }({
                                                                        labelBehavior: labelBehavior
                                                                    })
                                                                }
                                                            }
                                                        }
                                                    ],
                                                    [
                                                        $9d2c38978395d44a$var$InsideSelectValueContext,
                                                        true
                                                    ]
                                                ],
                                                children: defaultChildren
                                            });
                                        }
                                    }),
                                    /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)((0, $1df9f1c9262ce5df$exports.default), {
                                        size: size,
                                        className: $9d2c38978395d44a$var$iconStyles
                                    })
                                ]
                            }),
                            /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)((0, $493371ef39bf7a55$exports.Popover), {
                                hideArrow: true,
                                offset: menuOffset,
                                crossOffset: RTLFlipOffset * -12,
                                placement: `${direction} ${align}`,
                                shouldFlip: shouldFlip,
                                styles: " Nh131 ZbWuche131",
                                children: /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)("div", {
                                    className: " sd131 Za131 Fb131",
                                    children: /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)((0, $7pV4n$reactariacomponentsslots.Provider), {
                                        values: [
                                            [
                                                (0, $6367bc87eb7d24ad$exports.HeaderContext),
                                                {
                                                    styles: (0, $e741ea6b88ce4866$exports.sectionHeader)({
                                                        size: size
                                                    })
                                                }
                                            ],
                                            [
                                                (0, $6367bc87eb7d24ad$exports.HeadingContext),
                                                {
                                                    // @ts-ignore
                                                    role: 'presentation',
                                                    styles: (0, $e741ea6b88ce4866$exports.sectionHeading)
                                                }
                                            ],
                                            [
                                                (0, $6367bc87eb7d24ad$exports.TextContext),
                                                {
                                                    slots: {
                                                        description: {
                                                            styles: (0, $e741ea6b88ce4866$exports.description)({
                                                                size: size,
                                                                isFocused: false,
                                                                isDisabled: false
                                                            })
                                                        }
                                                    }
                                                }
                                            ]
                                        ],
                                        children: /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)((0, $7pV4n$reactariacomponentsListBox.ListBox), {
                                            items: items,
                                            className: $9d2c38978395d44a$export$b1e5508a851be14d,
                                            children: children
                                        })
                                    })
                                })
                            })
                        ]
                    })
            }),
            /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)($9d2c38978395d44a$var$TabLine, {
                isDisabled: props1.isDisabled
            })
        ]
    });
}
/**
 * Pickers allow users to choose a single option from a collapsible list of options when space is
 * limited.
 */ let $9d2c38978395d44a$export$ba25329847403e11 = /*#__PURE__*/ (0, $7pV4n$react.forwardRef)($9d2c38978395d44a$var$Picker);
const $9d2c38978395d44a$var$selectedIndicator = function anonymous(props) {
    let rules = " ";
    if (props.isDisabled) rules += ' gH131';
    else rules += ' g0131';
    if (props.isDisabled) rules += ' gld131';
    else rules += ' gle131';
    rules += ' FLvNfn131';
    rules += ' _je131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    rules += ' Jk7Otrc131';
    rules += ' Xa131';
    rules += ' _2b131';
    return rules;
};
function $9d2c38978395d44a$var$TabLine(props) {
    return /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)("div", {
        className: $9d2c38978395d44a$var$selectedIndicator(props)
    });
}
function $9d2c38978395d44a$export$d601881f38163e28(props) {
    let ref = (0, $7pV4n$react.useRef)(null);
    let isLink = props.href != null;
    const size = 'M';
    return /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)((0, $7pV4n$reactariacomponentsListBox.ListBoxItem), {
        ...props,
        ref: ref,
        textValue: props.textValue || (typeof props.children === 'string' ? props.children : undefined),
        style: (0, $2061c83559b50a66$exports.pressScale)(ref, props.UNSAFE_style),
        className: (renderProps)=>(props.UNSAFE_className || '') + (0, $e741ea6b88ce4866$exports.menuitem)({
                ...renderProps,
                size: size,
                isLink: isLink
            }, props.styles),
        children: (renderProps)=>{
            let { children: children } = props;
            return /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)($9d2c38978395d44a$var$DefaultProvider, {
                context: (0, $bde97c91243ed164$exports.IconContext),
                value: {
                    slots: {
                        icon: {
                            render: (0, $e991cbcdf82ced71$exports.centerBaseline)({
                                slot: 'icon',
                                styles: $9d2c38978395d44a$var$iconCenterWrapper({})
                            }),
                            styles: (0, $e741ea6b88ce4866$exports.icon)
                        }
                    }
                },
                children: /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsxs)($9d2c38978395d44a$var$DefaultProvider, {
                    context: (0, $6367bc87eb7d24ad$exports.TextContext),
                    value: {
                        slots: {
                            [(0, $7pV4n$reactariacomponentsslots.DEFAULT_SLOT)]: {
                                styles: (0, $e741ea6b88ce4866$exports.label)({
                                    size: size
                                })
                            }
                        }
                    },
                    children: [
                        !isLink && /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)((0, $dba2b18fc5727f9b$exports.default), {
                            size: size,
                            className: (0, $e741ea6b88ce4866$exports.checkmark)({
                                ...renderProps,
                                size: size
                            })
                        }),
                        typeof children === 'string' ? /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)((0, $6367bc87eb7d24ad$exports.Text), {
                            children: children
                        }) : children
                    ]
                })
            });
        }
    });
}
// A Context.Provider that only sets a value if not inside SelectValue.
function $9d2c38978395d44a$var$DefaultProvider({ context: context, value: value, children: children }) {
    let inSelectValue = (0, $7pV4n$react.useContext)($9d2c38978395d44a$var$InsideSelectValueContext);
    if (inSelectValue) return children;
    return /*#__PURE__*/ (0, $7pV4n$reactjsxruntime.jsx)(context.Provider, {
        value: value,
        children: children
    });
}
