// @react-spectrum/s2/dist/private/Tabs.cjs
require("./Tabs.css");
var $e991cbcdf82ced71$exports = require("./CenterBaseline.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $9d2c38978395d44a$exports = require("./TabsPicker.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $5oxdw$reactjsxruntime = require("react/jsx-runtime");
var $5oxdw$react = require("react");
var $5oxdw$reactariacomponentsTabs = require("react-aria-components/Tabs");
var $5oxdw$reactariaCollectionBuilder = require("react-aria/CollectionBuilder");
var $5oxdw$reactariacomponentsslots = require("react-aria-components/slots");
var $5oxdw$reactariacomponentsGroup = require("react-aria-components/Group");
var $5oxdw$reactariaprivateutilsinertValue = require("react-aria/private/utils/inertValue");
var $5oxdw$reactariacomponentsSelectionIndicator = require("react-aria-components/SelectionIndicator");
var $5oxdw$reactstatelyuseControlledState = require("react-stately/useControlledState");
var $5oxdw$reactariaprivateutilsuseEffectEvent = require("react-aria/private/utils/useEffectEvent");
var $5oxdw$reactariaprivatefocususeHasTabbableChild = require("react-aria/private/focus/useHasTabbableChild");
var $5oxdw$reactariauseId = require("react-aria/useId");
var $5oxdw$reactariaprivateutilsuseLabels = require("react-aria/private/utils/useLabels");
var $5oxdw$reactariaprivateutilsuseLayoutEffect = require("react-aria/private/utils/useLayoutEffect");
var $5oxdw$reactariaI18nProvider = require("react-aria/I18nProvider");
var $5oxdw$reactariaprivateutilsuseResizeObserver = require("react-aria/private/utils/useResizeObserver");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "TabsContext", () => $b27519d6a701105b$export$cfa7aa87c26e7d1f);
$parcel$export(module.exports, "Tabs", () => $b27519d6a701105b$export$b2539bed5023c21c);
$parcel$export(module.exports, "TabList", () => $b27519d6a701105b$export$e51a686c67fdaa2d);
$parcel$export(module.exports, "Tab", () => $b27519d6a701105b$export$3e41faf802a29e71);
$parcel$export(module.exports, "TabPanel", () => $b27519d6a701105b$export$3d96ec278d3efce4);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





















const $b27519d6a701105b$export$cfa7aa87c26e7d1f = /*#__PURE__*/ (0, $5oxdw$react.createContext)(null);
const $b27519d6a701105b$var$InternalTabsContext = /*#__PURE__*/ (0, $5oxdw$react.createContext)({});
const $b27519d6a701105b$var$CollapseContext = /*#__PURE__*/ (0, $5oxdw$react.createContext)({
    showTabs: true,
    menuId: '',
    valueId: '',
    tabs: []
});
const $b27519d6a701105b$var$tabs = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let flexShrink = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L|F|M|K)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "_v") flexShrink = true;
        rules += p[0];
    }
    if (!position) rules += ' _Pc131';
    rules += ' sd131';
    if (!flexShrink) rules += ' _va131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-c131';
    rules += ' vx131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    rules += ' pt131';
    if (props.orientation === "horizontal") rules += ' _ta131';
    return rules;
};
const $b27519d6a701105b$export$b2539bed5023c21c = /*#__PURE__*/ (0, $5oxdw$react.forwardRef)(function Tabs(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $b27519d6a701105b$export$cfa7aa87c26e7d1f);
    let { density: density = 'regular', isDisabled: isDisabled, disabledKeys: disabledKeys, orientation: orientation = 'horizontal', labelBehavior: labelBehavior = 'show' } = props;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let [value, setValue] = (0, $5oxdw$reactstatelyuseControlledState.useControlledState)(props.selectedKey, props.defaultSelectedKey ?? null, props.onSelectionChange);
    if (!props['aria-label'] && !props['aria-labelledby']) throw new Error('An aria-label or aria-labelledby prop is required on Tabs for accessibility.');
    let tablistRef = (0, $5oxdw$react.useRef)(null);
    return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)((0, $5oxdw$reactariacomponentsslots.Provider), {
        values: [
            [
                $b27519d6a701105b$var$InternalTabsContext,
                {
                    density: density,
                    isDisabled: isDisabled,
                    orientation: orientation,
                    disabledKeys: disabledKeys,
                    selectedKey: value,
                    tablistRef: tablistRef,
                    onSelectionChange: setValue,
                    labelBehavior: labelBehavior,
                    'aria-label': props['aria-label'],
                    'aria-labelledby': props['aria-labelledby']
                }
            ]
        ],
        children: /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)((0, $5oxdw$reactariaCollectionBuilder.CollectionBuilder), {
            content: props.children,
            children: (collection)=>/*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)($b27519d6a701105b$var$CollapsingTabs, {
                    ...props,
                    selectedKey: value,
                    onSelectionChange: setValue,
                    collection: collection,
                    containerRef: domRef
                })
        })
    });
});
const $b27519d6a701105b$var$tablist = function anonymous(props) {
    let rules = " ";
    rules += ' sd131';
    if (props.orientation === "horizontal") {
        if (props.labelBehavior === "hide") {
            if (props.density === "regular") rules += ' Uj131';
            else if (props.density === "compact") rules += ' Ul131';
        } else if (props.density === "regular") rules += ' Up131';
        else if (props.density === "compact") rules += ' Uj131';
    }
    if (props.orientation === "horizontal") {
        if (props.labelBehavior === "hide") {
            if (props.density === "regular") rules += ' qj131';
            else if (props.density === "compact") rules += ' ql131';
        } else if (props.density === "regular") rules += ' qp131';
        else if (props.density === "compact") rules += ' qj131';
    }
    if (props.orientation === "vertical") rules += ' _ta131';
    if (props.orientation === "vertical") rules += ' Hz131';
    if (props.orientation === "vertical") rules += ' Ix131';
    rules += ' Nq131';
    return rules;
};
const $b27519d6a701105b$var$tablistWrapper = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let minWidth = false;
    let flexShrink = false;
    let flexGrow = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "N") minWidth = true;
        if (p[1] === "_v") flexShrink = true;
        if (p[1] === "_u") flexGrow = true;
        rules += p[0];
    }
    if (!position) rules += ' _Pc131';
    if (!minWidth) rules += ' Na131';
    if (!flexShrink) rules += ' _va131';
    if (!flexGrow) rules += ' _ua131';
    return rules;
};
function $b27519d6a701105b$export$e51a686c67fdaa2d(props) {
    let { showTabs: showTabs, menuId: menuId, valueId: valueId, tabs: tabs, listRef: listRef, onSelectionChange: onSelectionChange, ariaLabel: ariaLabel, ariaDescribedBy: ariaDescribedBy } = (0, $5oxdw$react.useContext)($b27519d6a701105b$var$CollapseContext) ?? {};
    let { density: density, orientation: orientation, labelBehavior: labelBehavior } = (0, $5oxdw$react.useContext)($b27519d6a701105b$var$InternalTabsContext);
    if (showTabs) return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)($b27519d6a701105b$var$TabListInner, {
        ...props
    });
    return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsxs)("div", {
        className: $b27519d6a701105b$var$tablistWrapper(null, props.styles),
        children: [
            listRef && /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)("div", {
                className: $b27519d6a701105b$var$tablist({
                    orientation: orientation,
                    labelBehavior: labelBehavior,
                    density: density
                }),
                children: /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)($b27519d6a701105b$var$HiddenTabs, {
                    items: tabs,
                    density: density,
                    listRef: listRef
                })
            }),
            /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)($b27519d6a701105b$var$TabsMenu, {
                id: menuId,
                valueId: valueId,
                items: tabs,
                onSelectionChange: onSelectionChange,
                "aria-label": ariaLabel,
                "aria-describedby": ariaDescribedBy
            })
        ]
    });
}
function $b27519d6a701105b$var$TabListInner(props) {
    let { tablistRef: tablistRef, orientation: orientation, density: density, labelBehavior: labelBehavior, 'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy } = (0, $5oxdw$react.useContext)($b27519d6a701105b$var$InternalTabsContext) ?? {};
    let { tabs: tabs, listRef: listRef } = (0, $5oxdw$react.useContext)($b27519d6a701105b$var$CollapseContext) ?? {};
    return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsxs)("div", {
        style: props.UNSAFE_style,
        className: (props.UNSAFE_className || '') + $b27519d6a701105b$var$tablistWrapper(null, props.styles),
        children: [
            listRef && /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)("div", {
                className: $b27519d6a701105b$var$tablist({
                    orientation: orientation,
                    labelBehavior: labelBehavior,
                    density: density
                }),
                children: /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)($b27519d6a701105b$var$HiddenTabs, {
                    items: tabs,
                    density: density,
                    listRef: listRef
                })
            }),
            /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)((0, $5oxdw$reactariacomponentsTabs.TabList), {
                ...props,
                "aria-label": ariaLabel,
                "aria-labelledby": ariaLabelledBy,
                ref: tablistRef,
                className: (renderProps)=>$b27519d6a701105b$var$tablist({
                        ...renderProps,
                        labelBehavior: labelBehavior,
                        density: density
                    })
            })
        ]
    });
}
const $b27519d6a701105b$var$selectedIndicator = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    if (props.isDisabled) rules += ' gH131';
    else rules += ' g0131';
    if (props.isDisabled) rules += ' gld131';
    else rules += ' gle131';
    if (props.orientation === "horizontal") rules += ' FLvNfn131';
    else rules += ' Fb131';
    if (props.orientation === "vertical") rules += ' ZLvNfn131';
    else rules += ' Za131';
    rules += ' _qg131';
    rules += ' YIRyJjb131';
    rules += ' Ydklsgce131';
    rules += ' Xc131';
    rules += ' _2a131';
    rules += ' _lr131';
    if (props.orientation === "vertical") rules += ' Wr131';
    if (props.orientation === "horizontal") rules += ' _Er131';
    if (props.orientation === "vertical") rules += ' _Aa131';
    rules += ' _je131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    return rules;
};
const $b27519d6a701105b$var$tab = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let flexShrink = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "_v") flexShrink = true;
        rules += p[0];
    }
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    rules += ' sd131';
    if (props.isDisabled) rules += ' pp131';
    else if (props.isSelected) {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    } else {
        if (props.isPressed) rules += ' pt131';
        else if (props.isFocusVisible) rules += ' pt131';
        else if (props.isHovered) rules += ' pt131';
        else rules += ' pv131';
    }
    if (props.isDisabled) rules += ' plc131';
    else if (props.isSelected) rules += ' pld131';
    rules += ' oa131';
    rules += ' na131';
    rules += ' ka131';
    rules += ' ja131';
    rules += ' Uc131';
    rules += ' Usd131';
    rules += ' qc131';
    rules += ' qsd131';
    if (props.orientation === "horizontal") {
        if (props.density === "regular") rules += ' Fz131';
        else if (props.density === "compact") rules += ' Fx131';
    }
    if (props.orientation === "vertical") {
        if (props.density === "regular") rules += ' Mk131';
        else if (props.density === "compact") rules += ' Mi131';
    }
    rules += ' eb131';
    if (!position) rules += ' _Pc131';
    rules += ' ri131';
    rules += ' _Xa131';
    if (!flexShrink) rules += ' _va131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.labelBehavior === "hide") rules += ' SssxDec131';
    if (props.labelBehavior === "hide") rules += ' RssxDec131';
    rules += ' __ca131';
    return rules;
};
const $b27519d6a701105b$var$icon = " sb131 _va131 -_8sjo0b-t5ZbAob131";
function $b27519d6a701105b$export$3e41faf802a29e71(props1) {
    let { density: density, orientation: orientation, labelBehavior: labelBehavior } = (0, $5oxdw$react.useContext)($b27519d6a701105b$var$InternalTabsContext) ?? {};
    let contentId = (0, $5oxdw$reactariauseId.useId)();
    let ariaLabelledBy = props1['aria-labelledby'] || '';
    return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)((0, $5oxdw$reactariacomponentsTabs.Tab), {
        ...props1,
        // @ts-ignore
        originalProps: props1,
        "aria-labelledby": `${labelBehavior === 'hide' ? contentId : ''} ${ariaLabelledBy}`,
        style: props1.UNSAFE_style,
        className: (renderProps)=>(props1.UNSAFE_className || '') + $b27519d6a701105b$var$tab({
                ...renderProps,
                density: density,
                labelBehavior: labelBehavior,
                orientation: orientation
            }, props1.styles),
        children: ({ isMenu: // @ts-ignore
        isMenu, isDisabled: isDisabled })=>{
            if (isMenu) return props1.children;
            else return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)((0, $5oxdw$reactariacomponentsslots.Provider), {
                values: [
                    [
                        (0, $6367bc87eb7d24ad$exports.TextContext),
                        {
                            id: contentId,
                            styles: function anonymous(props) {
                                let rules = " ";
                                rules += ' _Jb131';
                                if (props.labelBehavior === "hide") rules += ' sk131';
                                return rules;
                            }({
                                labelBehavior: labelBehavior
                            })
                        }
                    ],
                    [
                        (0, $bde97c91243ed164$exports.IconContext),
                        {
                            render: (0, $e991cbcdf82ced71$exports.centerBaseline)({
                                slot: 'icon',
                                styles: " _Ja131"
                            }),
                            styles: $b27519d6a701105b$var$icon
                        }
                    ]
                ],
                children: /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)($b27519d6a701105b$var$TabInner, {
                    orientation: orientation,
                    isDisabled: isDisabled,
                    children: typeof props1.children === 'string' ? /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)((0, $6367bc87eb7d24ad$exports.Text), {
                        children: props1.children
                    }) : props1.children
                })
            });
        }
    });
}
function $b27519d6a701105b$var$TabInner({ isDisabled: isDisabled, orientation: orientation, children: children }) {
    let ref = (0, $5oxdw$react.useRef)(null);
    let isHidden = (0, $5oxdw$react.useContext)($b27519d6a701105b$var$HiddenTabsContext);
    return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsxs)((0, $5oxdw$reactjsxruntime.Fragment), {
        children: [
            !isHidden && /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)((0, $5oxdw$reactariacomponentsSelectionIndicator.SelectionIndicator), {
                ref: ref,
                className: $b27519d6a701105b$var$selectedIndicator({
                    isDisabled: isDisabled,
                    orientation: orientation
                })
            }),
            children
        ]
    });
}
const $b27519d6a701105b$var$tabPanel = function anonymous(props, overrides) {
    let rules = " ";
    let marginTop = false;
    let flexGrow = false;
    let minHeight = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L|F|M|K)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "J") marginTop = true;
        if (p[1] === "_u") flexGrow = true;
        if (p[1] === "M") minHeight = true;
        rules += p[0];
    }
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (!marginTop) rules += ' Js131';
    rules += ' pt131';
    if (!flexGrow) rules += ' _ub131';
    if (!minHeight) rules += ' Ma131';
    if (props.isInert) rules += ' sk131';
    else rules += ' sb131';
    return rules;
};
function $b27519d6a701105b$export$3d96ec278d3efce4(props) {
    let { showTabs: showTabs } = (0, $5oxdw$react.useContext)($b27519d6a701105b$var$CollapseContext);
    let { selectedKey: selectedKey } = (0, $5oxdw$react.useContext)($b27519d6a701105b$var$InternalTabsContext);
    if (showTabs) return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)((0, $5oxdw$reactariacomponentsTabs.TabPanel), {
        ...props,
        style: props.UNSAFE_style,
        className: (renderProps)=>(props.UNSAFE_className ?? '') + $b27519d6a701105b$var$tabPanel(renderProps, props.styles)
    });
    if (props.id !== selectedKey) return null;
    return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)($b27519d6a701105b$var$CollapsedTabPanel, {
        ...props
    });
}
function $b27519d6a701105b$var$CollapsedTabPanel(props) {
    // eslint-disable-next-line @typescript-eslint/no-unused-vars
    let { UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '', id: id, shouldForceMount: shouldForceMount, ...otherProps } = props;
    let { menuId: menuId, valueId: valueId } = (0, $5oxdw$react.useContext)($b27519d6a701105b$var$CollapseContext);
    let ref = (0, $5oxdw$react.useRef)(null);
    let tabIndex = (0, $5oxdw$reactariaprivatefocususeHasTabbableChild.useHasTabbableChild)(ref) ? undefined : 0;
    return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)((0, $5oxdw$reactariacomponentsGroup.Group), {
        ...otherProps,
        ref: ref,
        "aria-labelledby": menuId + ' ' + valueId,
        tabIndex: tabIndex,
        style: UNSAFE_style,
        className: (renderProps)=>UNSAFE_className + $b27519d6a701105b$var$tabPanel(renderProps, props.styles)
    });
}
function $b27519d6a701105b$var$isEveryTabDisabled(collection, disabledKeys) {
    let testKey = null;
    if (collection && collection.size > 0) {
        testKey = collection.getFirstKey();
        let index = 0;
        while(testKey && index < collection.size){
            // We have to check if the item in the collection has a key in disabledKeys or has the isDisabled prop set directly on it
            if (!disabledKeys.has(testKey) && !collection.getItem(testKey)?.props?.isDisabled) return false;
            testKey = collection.getKeyAfter(testKey);
            index++;
        }
        return true;
    }
    return false;
}
const $b27519d6a701105b$var$HiddenTabsContext = /*#__PURE__*/ (0, $5oxdw$react.createContext)(false);
let $b27519d6a701105b$var$HiddenTabs = function(props) {
    let { listRef: listRef, items: items = [], size: size, density: density } = props;
    return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)("div", {
        // @ts-ignore
        inert: (0, $5oxdw$reactariaprivateutilsinertValue.inertValue)(true),
        ref: listRef,
        className: " s40ub4c131 _t40ub4c131 U40ub4c131 q40ub4c131 _w40ub4c131 _Pa131 Wr131 _lr131 _Ar131 _zr131 _6b131 _Nc131 Pc131 _Ia131",
        children: /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)($b27519d6a701105b$var$HiddenTabsContext.Provider, {
            value: true,
            children: items.map((item)=>{
                // pull off individual props as an allow list, don't want refs or other props getting through
                return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)("div", {
                    "data-hidden-tab": true,
                    style: item.props.UNSAFE_style,
                    className: item.props.className({
                        size: size,
                        density: density
                    }),
                    children: item.props.children({
                        size: size,
                        density: density
                    })
                }, item.key);
            })
        })
    });
};
let $b27519d6a701105b$var$TabsMenu = (props1)=>{
    let { id: id, items: items, 'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy, valueId: valueId } = props1;
    let { density: density, onSelectionChange: _onSelectionChange, selectedKey: selectedKey, isDisabled: isDisabled, disabledKeys: disabledKeys, labelBehavior: labelBehavior } = (0, $5oxdw$react.useContext)($b27519d6a701105b$var$InternalTabsContext);
    let onSelectionChange = (0, $5oxdw$react.useCallback)((key)=>{
        if (key != null) _onSelectionChange?.(key);
    }, [
        _onSelectionChange
    ]);
    let state = (0, $5oxdw$react.useContext)((0, $5oxdw$reactariacomponentsTabs.TabListStateContext));
    let allKeysDisabled = (0, $5oxdw$react.useMemo)(()=>{
        return $b27519d6a701105b$var$isEveryTabDisabled(state?.collection, disabledKeys ? new Set(disabledKeys) : new Set());
    }, [
        state?.collection,
        disabledKeys
    ]);
    let labelProps = (0, $5oxdw$reactariaprivateutilsuseLabels.useLabels)({
        id: id,
        'aria-label': ariaLabel,
        'aria-labelledby': ariaLabelledBy
    });
    return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)("div", {
        className: function anonymous(props) {
            let rules = " ";
            rules += ' sd131';
            rules += ' _va131';
            rules += ' eb131';
            if (props.density === "regular") rules += ' Fz131';
            else if (props.density === "compact") rules += ' Fx131';
            return rules;
        }({
            density: density
        }),
        children: /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)((0, $9d2c38978395d44a$exports.Picker), {
            id: id,
            valueId: valueId,
            ...labelProps,
            "aria-describedby": props1['aria-describedby'],
            "aria-details": props1['aria-details'],
            isDisabled: isDisabled || allKeysDisabled,
            density: density,
            labelBehavior: labelBehavior,
            items: items,
            disabledKeys: disabledKeys,
            selectedKey: selectedKey,
            onSelectionChange: onSelectionChange,
            children: (item)=>{
                return /*#__PURE__*/ (0, $5oxdw$react.createElement)((0, $9d2c38978395d44a$exports.PickerItem), {
                    ...item.props.originalProps,
                    isDisabled: isDisabled || allKeysDisabled,
                    key: item.key
                }, item.props.children({
                    density: density,
                    isMenu: true
                }));
            }
        })
    });
};
let $b27519d6a701105b$var$CollapsingTabs = ({ collection: collection, containerRef: containerRef, ...props })=>{
    let { orientation: orientation = 'horizontal', onSelectionChange: onSelectionChange } = props;
    let [showItems, _setShowItems] = (0, $5oxdw$react.useState)(true);
    showItems = orientation === 'vertical' ? true : showItems;
    let setShowItems = (0, $5oxdw$react.useCallback)((value)=>{
        if (orientation === 'vertical') // if orientation is vertical, we always show the items
        _setShowItems(true);
        else _setShowItems(value);
    }, [
        orientation
    ]);
    let { direction: direction } = (0, $5oxdw$reactariaI18nProvider.useLocale)();
    let children = (0, $5oxdw$react.useMemo)(()=>[
            ...collection
        ], [
        collection
    ]);
    let listRef = (0, $5oxdw$react.useRef)(null);
    let updateOverflow = ()=>{
        if (orientation === 'vertical' || !listRef.current || !containerRef?.current) return;
        let container = listRef.current;
        let containerRect = container.getBoundingClientRect();
        let tabs = container.querySelectorAll('[data-hidden-tab]');
        let lastTab = tabs[tabs.length - 1];
        let lastTabRect = lastTab.getBoundingClientRect();
        if (direction === 'ltr') setShowItems?.(lastTabRect.right <= containerRect.right);
        else setShowItems?.(lastTabRect.left >= containerRect.left);
    };
    let updateOverflowEffect = (0, $5oxdw$reactariaprivateutilsuseEffectEvent.useEffectEvent)(updateOverflow);
    (0, $5oxdw$reactariaprivateutilsuseResizeObserver.useResizeObserver)({
        ref: containerRef,
        onResize: updateOverflow
    });
    (0, $5oxdw$reactariaprivateutilsuseLayoutEffect.useLayoutEffect)(()=>{
        if (collection.size > 0) queueMicrotask(updateOverflowEffect);
    }, [
        collection.size
    ]);
    // start with null so that the first render won't have a flicker
    let prevOrientation = (0, $5oxdw$react.useRef)(null);
    (0, $5oxdw$reactariaprivateutilsuseLayoutEffect.useLayoutEffect)(()=>{
        if (collection.size > 0 && prevOrientation.current !== orientation) updateOverflowEffect();
        prevOrientation.current = orientation;
    }, [
        collection.size,
        orientation
    ]);
    (0, $5oxdw$react.useEffect)(()=>{
        // Recalculate visible tags when fonts are loaded.
        document.fonts?.ready.then(()=>updateOverflowEffect());
    }, []);
    let menuId = (0, $5oxdw$reactariauseId.useId)();
    let valueId = (0, $5oxdw$reactariauseId.useId)();
    let contents;
    if (showItems) contents = /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)((0, $5oxdw$reactariacomponentsTabs.Tabs), {
        ...props,
        className: "",
        style: {
            display: 'contents'
        },
        children: props.children
    });
    else contents = /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)((0, $5oxdw$reactjsxruntime.Fragment), {
        children: /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)($b27519d6a701105b$var$CollapseContext.Provider, {
            value: {
                showTabs: false,
                tabs: children,
                menuId: menuId,
                valueId: valueId,
                listRef: listRef,
                onSelectionChange: onSelectionChange,
                ariaLabel: props['aria-label'],
                ariaDescribedBy: props['aria-labelledby']
            },
            children: props.children
        })
    });
    return /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)("div", {
        style: props.UNSAFE_style,
        className: (props.UNSAFE_className || '') + $b27519d6a701105b$var$tabs({
            orientation: orientation
        }, props.styles),
        ref: containerRef,
        children: /*#__PURE__*/ (0, $5oxdw$reactjsxruntime.jsx)($b27519d6a701105b$var$CollapseContext.Provider, {
            value: {
                showTabs: true,
                menuId: menuId,
                valueId: valueId,
                tabs: children,
                listRef: listRef
            },
            children: contents
        })
    });
};
