// @react-spectrum/s2/dist/private/Tabs.mjs
import "./Tabs.css";
import {centerBaseline as $1f4b04be3f24aae3$export$9d7e2342a7e53afa} from "./CenterBaseline.mjs";
import {IconContext as $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed} from "../../icons/Icon.mjs";
import {Picker as $0067ea932a992b6a$export$ba25329847403e11, PickerItem as $0067ea932a992b6a$export$d601881f38163e28} from "./TabsPicker.mjs";
import {Text as $8e847109a6ab556d$export$5f1af8db9871e1d6, TextContext as $8e847109a6ab556d$export$9afb8bc826b033ea} from "./Content.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $8bRfI$jsx, jsxs as $8bRfI$jsxs, Fragment as $8bRfI$Fragment} from "react/jsx-runtime";
import {createContext as $8bRfI$createContext, forwardRef as $8bRfI$forwardRef, useRef as $8bRfI$useRef, useContext as $8bRfI$useContext, useCallback as $8bRfI$useCallback, useMemo as $8bRfI$useMemo, createElement as $8bRfI$createElement, useState as $8bRfI$useState, useEffect as $8bRfI$useEffect} from "react";
import {TabList as $8bRfI$TabList, Tab as $8bRfI$Tab, TabPanel as $8bRfI$TabPanel, TabListStateContext as $8bRfI$TabListStateContext, Tabs as $8bRfI$Tabs} from "react-aria-components/Tabs";
import {CollectionBuilder as $8bRfI$CollectionBuilder} from "react-aria/CollectionBuilder";
import {Provider as $8bRfI$Provider} from "react-aria-components/slots";
import {Group as $8bRfI$Group} from "react-aria-components/Group";
import {inertValue as $8bRfI$inertValue} from "react-aria/private/utils/inertValue";
import {SelectionIndicator as $8bRfI$SelectionIndicator} from "react-aria-components/SelectionIndicator";
import {useControlledState as $8bRfI$useControlledState} from "react-stately/useControlledState";
import {useEffectEvent as $8bRfI$useEffectEvent} from "react-aria/private/utils/useEffectEvent";
import {useHasTabbableChild as $8bRfI$useHasTabbableChild} from "react-aria/private/focus/useHasTabbableChild";
import {useId as $8bRfI$useId} from "react-aria/useId";
import {useLabels as $8bRfI$useLabels} from "react-aria/private/utils/useLabels";
import {useLayoutEffect as $8bRfI$useLayoutEffect} from "react-aria/private/utils/useLayoutEffect";
import {useLocale as $8bRfI$useLocale} from "react-aria/I18nProvider";
import {useResizeObserver as $8bRfI$useResizeObserver} from "react-aria/private/utils/useResizeObserver";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





















const $79e2b4a1b34d1592$export$cfa7aa87c26e7d1f = /*#__PURE__*/ (0, $8bRfI$createContext)(null);
const $79e2b4a1b34d1592$var$InternalTabsContext = /*#__PURE__*/ (0, $8bRfI$createContext)({});
const $79e2b4a1b34d1592$var$CollapseContext = /*#__PURE__*/ (0, $8bRfI$createContext)({
    showTabs: true,
    menuId: '',
    valueId: '',
    tabs: []
});
const $79e2b4a1b34d1592$var$tabs = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let flexShrink = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L|F|M|K)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "_v") flexShrink = true;
        rules += p[0];
    }
    if (!position) rules += ' _Pc131';
    rules += ' sd131';
    if (!flexShrink) rules += ' _va131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-c131';
    rules += ' vx131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    rules += ' pt131';
    if (props.orientation === "horizontal") rules += ' _ta131';
    return rules;
};
const $79e2b4a1b34d1592$export$b2539bed5023c21c = /*#__PURE__*/ (0, $8bRfI$forwardRef)(function Tabs(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $79e2b4a1b34d1592$export$cfa7aa87c26e7d1f);
    let { density: density = 'regular', isDisabled: isDisabled, disabledKeys: disabledKeys, orientation: orientation = 'horizontal', labelBehavior: labelBehavior = 'show' } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let [value, setValue] = (0, $8bRfI$useControlledState)(props.selectedKey, props.defaultSelectedKey ?? null, props.onSelectionChange);
    if (!props['aria-label'] && !props['aria-labelledby']) throw new Error('An aria-label or aria-labelledby prop is required on Tabs for accessibility.');
    let tablistRef = (0, $8bRfI$useRef)(null);
    return /*#__PURE__*/ (0, $8bRfI$jsx)((0, $8bRfI$Provider), {
        values: [
            [
                $79e2b4a1b34d1592$var$InternalTabsContext,
                {
                    density: density,
                    isDisabled: isDisabled,
                    orientation: orientation,
                    disabledKeys: disabledKeys,
                    selectedKey: value,
                    tablistRef: tablistRef,
                    onSelectionChange: setValue,
                    labelBehavior: labelBehavior,
                    'aria-label': props['aria-label'],
                    'aria-labelledby': props['aria-labelledby']
                }
            ]
        ],
        children: /*#__PURE__*/ (0, $8bRfI$jsx)((0, $8bRfI$CollectionBuilder), {
            content: props.children,
            children: (collection)=>/*#__PURE__*/ (0, $8bRfI$jsx)($79e2b4a1b34d1592$var$CollapsingTabs, {
                    ...props,
                    selectedKey: value,
                    onSelectionChange: setValue,
                    collection: collection,
                    containerRef: domRef
                })
        })
    });
});
const $79e2b4a1b34d1592$var$tablist = function anonymous(props) {
    let rules = " ";
    rules += ' sd131';
    if (props.orientation === "horizontal") {
        if (props.labelBehavior === "hide") {
            if (props.density === "regular") rules += ' Uj131';
            else if (props.density === "compact") rules += ' Ul131';
        } else if (props.density === "regular") rules += ' Up131';
        else if (props.density === "compact") rules += ' Uj131';
    }
    if (props.orientation === "horizontal") {
        if (props.labelBehavior === "hide") {
            if (props.density === "regular") rules += ' qj131';
            else if (props.density === "compact") rules += ' ql131';
        } else if (props.density === "regular") rules += ' qp131';
        else if (props.density === "compact") rules += ' qj131';
    }
    if (props.orientation === "vertical") rules += ' _ta131';
    if (props.orientation === "vertical") rules += ' Hz131';
    if (props.orientation === "vertical") rules += ' Ix131';
    rules += ' Nq131';
    return rules;
};
const $79e2b4a1b34d1592$var$tablistWrapper = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let minWidth = false;
    let flexShrink = false;
    let flexGrow = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "N") minWidth = true;
        if (p[1] === "_v") flexShrink = true;
        if (p[1] === "_u") flexGrow = true;
        rules += p[0];
    }
    if (!position) rules += ' _Pc131';
    if (!minWidth) rules += ' Na131';
    if (!flexShrink) rules += ' _va131';
    if (!flexGrow) rules += ' _ua131';
    return rules;
};
function $79e2b4a1b34d1592$export$e51a686c67fdaa2d(props) {
    let { showTabs: showTabs, menuId: menuId, valueId: valueId, tabs: tabs, listRef: listRef, onSelectionChange: onSelectionChange, ariaLabel: ariaLabel, ariaDescribedBy: ariaDescribedBy } = (0, $8bRfI$useContext)($79e2b4a1b34d1592$var$CollapseContext) ?? {};
    let { density: density, orientation: orientation, labelBehavior: labelBehavior } = (0, $8bRfI$useContext)($79e2b4a1b34d1592$var$InternalTabsContext);
    if (showTabs) return /*#__PURE__*/ (0, $8bRfI$jsx)($79e2b4a1b34d1592$var$TabListInner, {
        ...props
    });
    return /*#__PURE__*/ (0, $8bRfI$jsxs)("div", {
        className: $79e2b4a1b34d1592$var$tablistWrapper(null, props.styles),
        children: [
            listRef && /*#__PURE__*/ (0, $8bRfI$jsx)("div", {
                className: $79e2b4a1b34d1592$var$tablist({
                    orientation: orientation,
                    labelBehavior: labelBehavior,
                    density: density
                }),
                children: /*#__PURE__*/ (0, $8bRfI$jsx)($79e2b4a1b34d1592$var$HiddenTabs, {
                    items: tabs,
                    density: density,
                    listRef: listRef
                })
            }),
            /*#__PURE__*/ (0, $8bRfI$jsx)($79e2b4a1b34d1592$var$TabsMenu, {
                id: menuId,
                valueId: valueId,
                items: tabs,
                onSelectionChange: onSelectionChange,
                "aria-label": ariaLabel,
                "aria-describedby": ariaDescribedBy
            })
        ]
    });
}
function $79e2b4a1b34d1592$var$TabListInner(props) {
    let { tablistRef: tablistRef, orientation: orientation, density: density, labelBehavior: labelBehavior, 'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy } = (0, $8bRfI$useContext)($79e2b4a1b34d1592$var$InternalTabsContext) ?? {};
    let { tabs: tabs, listRef: listRef } = (0, $8bRfI$useContext)($79e2b4a1b34d1592$var$CollapseContext) ?? {};
    return /*#__PURE__*/ (0, $8bRfI$jsxs)("div", {
        style: props.UNSAFE_style,
        className: (props.UNSAFE_className || '') + $79e2b4a1b34d1592$var$tablistWrapper(null, props.styles),
        children: [
            listRef && /*#__PURE__*/ (0, $8bRfI$jsx)("div", {
                className: $79e2b4a1b34d1592$var$tablist({
                    orientation: orientation,
                    labelBehavior: labelBehavior,
                    density: density
                }),
                children: /*#__PURE__*/ (0, $8bRfI$jsx)($79e2b4a1b34d1592$var$HiddenTabs, {
                    items: tabs,
                    density: density,
                    listRef: listRef
                })
            }),
            /*#__PURE__*/ (0, $8bRfI$jsx)((0, $8bRfI$TabList), {
                ...props,
                "aria-label": ariaLabel,
                "aria-labelledby": ariaLabelledBy,
                ref: tablistRef,
                className: (renderProps)=>$79e2b4a1b34d1592$var$tablist({
                        ...renderProps,
                        labelBehavior: labelBehavior,
                        density: density
                    })
            })
        ]
    });
}
const $79e2b4a1b34d1592$var$selectedIndicator = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    if (props.isDisabled) rules += ' gH131';
    else rules += ' g0131';
    if (props.isDisabled) rules += ' gld131';
    else rules += ' gle131';
    if (props.orientation === "horizontal") rules += ' FLvNfn131';
    else rules += ' Fb131';
    if (props.orientation === "vertical") rules += ' ZLvNfn131';
    else rules += ' Za131';
    rules += ' _qg131';
    rules += ' YIRyJjb131';
    rules += ' Ydklsgce131';
    rules += ' Xc131';
    rules += ' _2a131';
    rules += ' _lr131';
    if (props.orientation === "vertical") rules += ' Wr131';
    if (props.orientation === "horizontal") rules += ' _Er131';
    if (props.orientation === "vertical") rules += ' _Aa131';
    rules += ' _je131';
    rules += ' og131';
    rules += ' ng131';
    rules += ' kg131';
    rules += ' jg131';
    return rules;
};
const $79e2b4a1b34d1592$var$tab = function anonymous(props, overrides) {
    let rules = " ";
    let position = false;
    let flexShrink = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "_P") position = true;
        if (p[1] === "_v") flexShrink = true;
        rules += p[0];
    }
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    rules += ' sd131';
    if (props.isDisabled) rules += ' pp131';
    else if (props.isSelected) {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    } else {
        if (props.isPressed) rules += ' pt131';
        else if (props.isFocusVisible) rules += ' pt131';
        else if (props.isHovered) rules += ' pt131';
        else rules += ' pv131';
    }
    if (props.isDisabled) rules += ' plc131';
    else if (props.isSelected) rules += ' pld131';
    rules += ' oa131';
    rules += ' na131';
    rules += ' ka131';
    rules += ' ja131';
    rules += ' Uc131';
    rules += ' Usd131';
    rules += ' qc131';
    rules += ' qsd131';
    if (props.orientation === "horizontal") {
        if (props.density === "regular") rules += ' Fz131';
        else if (props.density === "compact") rules += ' Fx131';
    }
    if (props.orientation === "vertical") {
        if (props.density === "regular") rules += ' Mk131';
        else if (props.density === "compact") rules += ' Mi131';
    }
    rules += ' eb131';
    if (!position) rules += ' _Pc131';
    rules += ' ri131';
    rules += ' _Xa131';
    if (!flexShrink) rules += ' _va131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    if (props.labelBehavior === "hide") rules += ' SssxDec131';
    if (props.labelBehavior === "hide") rules += ' RssxDec131';
    rules += ' __ca131';
    return rules;
};
const $79e2b4a1b34d1592$var$icon = " sb131 _va131 -_8sjo0b-t5ZbAob131";
function $79e2b4a1b34d1592$export$3e41faf802a29e71(props1) {
    let { density: density, orientation: orientation, labelBehavior: labelBehavior } = (0, $8bRfI$useContext)($79e2b4a1b34d1592$var$InternalTabsContext) ?? {};
    let contentId = (0, $8bRfI$useId)();
    let ariaLabelledBy = props1['aria-labelledby'] || '';
    return /*#__PURE__*/ (0, $8bRfI$jsx)((0, $8bRfI$Tab), {
        ...props1,
        // @ts-ignore
        originalProps: props1,
        "aria-labelledby": `${labelBehavior === 'hide' ? contentId : ''} ${ariaLabelledBy}`,
        style: props1.UNSAFE_style,
        className: (renderProps)=>(props1.UNSAFE_className || '') + $79e2b4a1b34d1592$var$tab({
                ...renderProps,
                density: density,
                labelBehavior: labelBehavior,
                orientation: orientation
            }, props1.styles),
        children: ({ isMenu: // @ts-ignore
        isMenu, isDisabled: isDisabled })=>{
            if (isMenu) return props1.children;
            else return /*#__PURE__*/ (0, $8bRfI$jsx)((0, $8bRfI$Provider), {
                values: [
                    [
                        (0, $8e847109a6ab556d$export$9afb8bc826b033ea),
                        {
                            id: contentId,
                            styles: function anonymous(props) {
                                let rules = " ";
                                rules += ' _Jb131';
                                if (props.labelBehavior === "hide") rules += ' sk131';
                                return rules;
                            }({
                                labelBehavior: labelBehavior
                            })
                        }
                    ],
                    [
                        (0, $ac8c32e6775ddd1f$export$a49f528ae1a4d0ed),
                        {
                            render: (0, $1f4b04be3f24aae3$export$9d7e2342a7e53afa)({
                                slot: 'icon',
                                styles: " _Ja131"
                            }),
                            styles: $79e2b4a1b34d1592$var$icon
                        }
                    ]
                ],
                children: /*#__PURE__*/ (0, $8bRfI$jsx)($79e2b4a1b34d1592$var$TabInner, {
                    orientation: orientation,
                    isDisabled: isDisabled,
                    children: typeof props1.children === 'string' ? /*#__PURE__*/ (0, $8bRfI$jsx)((0, $8e847109a6ab556d$export$5f1af8db9871e1d6), {
                        children: props1.children
                    }) : props1.children
                })
            });
        }
    });
}
function $79e2b4a1b34d1592$var$TabInner({ isDisabled: isDisabled, orientation: orientation, children: children }) {
    let ref = (0, $8bRfI$useRef)(null);
    let isHidden = (0, $8bRfI$useContext)($79e2b4a1b34d1592$var$HiddenTabsContext);
    return /*#__PURE__*/ (0, $8bRfI$jsxs)((0, $8bRfI$Fragment), {
        children: [
            !isHidden && /*#__PURE__*/ (0, $8bRfI$jsx)((0, $8bRfI$SelectionIndicator), {
                ref: ref,
                className: $79e2b4a1b34d1592$var$selectedIndicator({
                    isDisabled: isDisabled,
                    orientation: orientation
                })
            }),
            children
        ]
    });
}
const $79e2b4a1b34d1592$var$tabPanel = function anonymous(props, overrides) {
    let rules = " ";
    let marginTop = false;
    let flexGrow = false;
    let minHeight = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L|F|M|K)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "J") marginTop = true;
        if (p[1] === "_u") flexGrow = true;
        if (p[1] === "M") minHeight = true;
        rules += p[0];
    }
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kd131';
    if (!marginTop) rules += ' Js131';
    rules += ' pt131';
    if (!flexGrow) rules += ' _ub131';
    if (!minHeight) rules += ' Ma131';
    if (props.isInert) rules += ' sk131';
    else rules += ' sb131';
    return rules;
};
function $79e2b4a1b34d1592$export$3d96ec278d3efce4(props) {
    let { showTabs: showTabs } = (0, $8bRfI$useContext)($79e2b4a1b34d1592$var$CollapseContext);
    let { selectedKey: selectedKey } = (0, $8bRfI$useContext)($79e2b4a1b34d1592$var$InternalTabsContext);
    if (showTabs) return /*#__PURE__*/ (0, $8bRfI$jsx)((0, $8bRfI$TabPanel), {
        ...props,
        style: props.UNSAFE_style,
        className: (renderProps)=>(props.UNSAFE_className ?? '') + $79e2b4a1b34d1592$var$tabPanel(renderProps, props.styles)
    });
    if (props.id !== selectedKey) return null;
    return /*#__PURE__*/ (0, $8bRfI$jsx)($79e2b4a1b34d1592$var$CollapsedTabPanel, {
        ...props
    });
}
function $79e2b4a1b34d1592$var$CollapsedTabPanel(props) {
    // eslint-disable-next-line @typescript-eslint/no-unused-vars
    let { UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '', id: id, shouldForceMount: shouldForceMount, ...otherProps } = props;
    let { menuId: menuId, valueId: valueId } = (0, $8bRfI$useContext)($79e2b4a1b34d1592$var$CollapseContext);
    let ref = (0, $8bRfI$useRef)(null);
    let tabIndex = (0, $8bRfI$useHasTabbableChild)(ref) ? undefined : 0;
    return /*#__PURE__*/ (0, $8bRfI$jsx)((0, $8bRfI$Group), {
        ...otherProps,
        ref: ref,
        "aria-labelledby": menuId + ' ' + valueId,
        tabIndex: tabIndex,
        style: UNSAFE_style,
        className: (renderProps)=>UNSAFE_className + $79e2b4a1b34d1592$var$tabPanel(renderProps, props.styles)
    });
}
function $79e2b4a1b34d1592$var$isEveryTabDisabled(collection, disabledKeys) {
    let testKey = null;
    if (collection && collection.size > 0) {
        testKey = collection.getFirstKey();
        let index = 0;
        while(testKey && index < collection.size){
            // We have to check if the item in the collection has a key in disabledKeys or has the isDisabled prop set directly on it
            if (!disabledKeys.has(testKey) && !collection.getItem(testKey)?.props?.isDisabled) return false;
            testKey = collection.getKeyAfter(testKey);
            index++;
        }
        return true;
    }
    return false;
}
const $79e2b4a1b34d1592$var$HiddenTabsContext = /*#__PURE__*/ (0, $8bRfI$createContext)(false);
let $79e2b4a1b34d1592$var$HiddenTabs = function(props) {
    let { listRef: listRef, items: items = [], size: size, density: density } = props;
    return /*#__PURE__*/ (0, $8bRfI$jsx)("div", {
        // @ts-ignore
        inert: (0, $8bRfI$inertValue)(true),
        ref: listRef,
        className: " s40ub4c131 _t40ub4c131 U40ub4c131 q40ub4c131 _w40ub4c131 _Pa131 Wr131 _lr131 _Ar131 _zr131 _6b131 _Nc131 Pc131 _Ia131",
        children: /*#__PURE__*/ (0, $8bRfI$jsx)($79e2b4a1b34d1592$var$HiddenTabsContext.Provider, {
            value: true,
            children: items.map((item)=>{
                // pull off individual props as an allow list, don't want refs or other props getting through
                return /*#__PURE__*/ (0, $8bRfI$jsx)("div", {
                    "data-hidden-tab": true,
                    style: item.props.UNSAFE_style,
                    className: item.props.className({
                        size: size,
                        density: density
                    }),
                    children: item.props.children({
                        size: size,
                        density: density
                    })
                }, item.key);
            })
        })
    });
};
let $79e2b4a1b34d1592$var$TabsMenu = (props1)=>{
    let { id: id, items: items, 'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy, valueId: valueId } = props1;
    let { density: density, onSelectionChange: _onSelectionChange, selectedKey: selectedKey, isDisabled: isDisabled, disabledKeys: disabledKeys, labelBehavior: labelBehavior } = (0, $8bRfI$useContext)($79e2b4a1b34d1592$var$InternalTabsContext);
    let onSelectionChange = (0, $8bRfI$useCallback)((key)=>{
        if (key != null) _onSelectionChange?.(key);
    }, [
        _onSelectionChange
    ]);
    let state = (0, $8bRfI$useContext)((0, $8bRfI$TabListStateContext));
    let allKeysDisabled = (0, $8bRfI$useMemo)(()=>{
        return $79e2b4a1b34d1592$var$isEveryTabDisabled(state?.collection, disabledKeys ? new Set(disabledKeys) : new Set());
    }, [
        state?.collection,
        disabledKeys
    ]);
    let labelProps = (0, $8bRfI$useLabels)({
        id: id,
        'aria-label': ariaLabel,
        'aria-labelledby': ariaLabelledBy
    });
    return /*#__PURE__*/ (0, $8bRfI$jsx)("div", {
        className: function anonymous(props) {
            let rules = " ";
            rules += ' sd131';
            rules += ' _va131';
            rules += ' eb131';
            if (props.density === "regular") rules += ' Fz131';
            else if (props.density === "compact") rules += ' Fx131';
            return rules;
        }({
            density: density
        }),
        children: /*#__PURE__*/ (0, $8bRfI$jsx)((0, $0067ea932a992b6a$export$ba25329847403e11), {
            id: id,
            valueId: valueId,
            ...labelProps,
            "aria-describedby": props1['aria-describedby'],
            "aria-details": props1['aria-details'],
            isDisabled: isDisabled || allKeysDisabled,
            density: density,
            labelBehavior: labelBehavior,
            items: items,
            disabledKeys: disabledKeys,
            selectedKey: selectedKey,
            onSelectionChange: onSelectionChange,
            children: (item)=>{
                return /*#__PURE__*/ (0, $8bRfI$createElement)((0, $0067ea932a992b6a$export$d601881f38163e28), {
                    ...item.props.originalProps,
                    isDisabled: isDisabled || allKeysDisabled,
                    key: item.key
                }, item.props.children({
                    density: density,
                    isMenu: true
                }));
            }
        })
    });
};
let $79e2b4a1b34d1592$var$CollapsingTabs = ({ collection: collection, containerRef: containerRef, ...props })=>{
    let { orientation: orientation = 'horizontal', onSelectionChange: onSelectionChange } = props;
    let [showItems, _setShowItems] = (0, $8bRfI$useState)(true);
    showItems = orientation === 'vertical' ? true : showItems;
    let setShowItems = (0, $8bRfI$useCallback)((value)=>{
        if (orientation === 'vertical') // if orientation is vertical, we always show the items
        _setShowItems(true);
        else _setShowItems(value);
    }, [
        orientation
    ]);
    let { direction: direction } = (0, $8bRfI$useLocale)();
    let children = (0, $8bRfI$useMemo)(()=>[
            ...collection
        ], [
        collection
    ]);
    let listRef = (0, $8bRfI$useRef)(null);
    let updateOverflow = ()=>{
        if (orientation === 'vertical' || !listRef.current || !containerRef?.current) return;
        let container = listRef.current;
        let containerRect = container.getBoundingClientRect();
        let tabs = container.querySelectorAll('[data-hidden-tab]');
        let lastTab = tabs[tabs.length - 1];
        let lastTabRect = lastTab.getBoundingClientRect();
        if (direction === 'ltr') setShowItems?.(lastTabRect.right <= containerRect.right);
        else setShowItems?.(lastTabRect.left >= containerRect.left);
    };
    let updateOverflowEffect = (0, $8bRfI$useEffectEvent)(updateOverflow);
    (0, $8bRfI$useResizeObserver)({
        ref: containerRef,
        onResize: updateOverflow
    });
    (0, $8bRfI$useLayoutEffect)(()=>{
        if (collection.size > 0) queueMicrotask(updateOverflowEffect);
    }, [
        collection.size
    ]);
    // start with null so that the first render won't have a flicker
    let prevOrientation = (0, $8bRfI$useRef)(null);
    (0, $8bRfI$useLayoutEffect)(()=>{
        if (collection.size > 0 && prevOrientation.current !== orientation) updateOverflowEffect();
        prevOrientation.current = orientation;
    }, [
        collection.size,
        orientation
    ]);
    (0, $8bRfI$useEffect)(()=>{
        // Recalculate visible tags when fonts are loaded.
        document.fonts?.ready.then(()=>updateOverflowEffect());
    }, []);
    let menuId = (0, $8bRfI$useId)();
    let valueId = (0, $8bRfI$useId)();
    let contents;
    if (showItems) contents = /*#__PURE__*/ (0, $8bRfI$jsx)((0, $8bRfI$Tabs), {
        ...props,
        className: "",
        style: {
            display: 'contents'
        },
        children: props.children
    });
    else contents = /*#__PURE__*/ (0, $8bRfI$jsx)((0, $8bRfI$Fragment), {
        children: /*#__PURE__*/ (0, $8bRfI$jsx)($79e2b4a1b34d1592$var$CollapseContext.Provider, {
            value: {
                showTabs: false,
                tabs: children,
                menuId: menuId,
                valueId: valueId,
                listRef: listRef,
                onSelectionChange: onSelectionChange,
                ariaLabel: props['aria-label'],
                ariaDescribedBy: props['aria-labelledby']
            },
            children: props.children
        })
    });
    return /*#__PURE__*/ (0, $8bRfI$jsx)("div", {
        style: props.UNSAFE_style,
        className: (props.UNSAFE_className || '') + $79e2b4a1b34d1592$var$tabs({
            orientation: orientation
        }, props.styles),
        ref: containerRef,
        children: /*#__PURE__*/ (0, $8bRfI$jsx)($79e2b4a1b34d1592$var$CollapseContext.Provider, {
            value: {
                showTabs: true,
                menuId: menuId,
                valueId: valueId,
                tabs: children,
                listRef: listRef
            },
            children: contents
        })
    });
};


export {$79e2b4a1b34d1592$export$cfa7aa87c26e7d1f as TabsContext, $79e2b4a1b34d1592$export$b2539bed5023c21c as Tabs, $79e2b4a1b34d1592$export$e51a686c67fdaa2d as TabList, $79e2b4a1b34d1592$export$3e41faf802a29e71 as Tab, $79e2b4a1b34d1592$export$3d96ec278d3efce4 as TabPanel};
