// @react-spectrum/s2/dist/private/TextField.mjs
import "./TextField.css";
import {createFocusableRef as $80ecb974df713a30$export$79d69eee6ae4b329} from "./useDOMRef.mjs";
import {FieldErrorIcon as $9afd80978c252040$export$bda0d075e0e342c9, FieldGroup as $9afd80978c252040$export$f0fe469072bc6950, FieldLabel as $9afd80978c252040$export$1acdcf5a973a8414, HelpText as $9afd80978c252040$export$a67c0bc59081311a, Input as $9afd80978c252040$export$f5b8910cec6cf069} from "./Field.mjs";
import {FormContext as $9b916426527cebe7$export$c24727297075ec6a, useFormProps as $9b916426527cebe7$export$a6b5be5c6b451665} from "./Form.mjs";
import {TextContext as $8e847109a6ab556d$export$9afb8bc826b033ea} from "./Content.mjs";
import {useSpectrumContextProps as $5ce63c423902f47d$export$764f6146fadd77f7} from "../../icons/useSpectrumContextProps.mjs";
import {jsx as $fsr8S$jsx, jsxs as $fsr8S$jsxs, Fragment as $fsr8S$Fragment} from "react/jsx-runtime";
import {TextAreaContext as $fsr8S$TextAreaContext, TextArea as $fsr8S$TextArea} from "react-aria-components/TextArea";
import {TextField as $fsr8S$TextField} from "react-aria-components/TextField";
import {composeRenderProps as $fsr8S$composeRenderProps} from "react-aria-components/composeRenderProps";
import {Provider as $fsr8S$Provider, useSlottedContext as $fsr8S$useSlottedContext} from "react-aria-components/slots";
import {createContext as $fsr8S$createContext, forwardRef as $fsr8S$forwardRef, useRef as $fsr8S$useRef, useContext as $fsr8S$useContext, useImperativeHandle as $fsr8S$useImperativeHandle} from "react";
import {InputContext as $fsr8S$InputContext} from "react-aria-components/Input";
import {mergeRefs as $fsr8S$mergeRefs} from "react-aria/mergeRefs";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 












const $d946f50010ba937f$export$2129e27b3ef0d483 = /*#__PURE__*/ (0, $fsr8S$createContext)(null);
const $d946f50010ba937f$export$2c73285ae9390cec = /*#__PURE__*/ (0, $fsr8S$forwardRef)(function TextField(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $d946f50010ba937f$export$2129e27b3ef0d483);
    return /*#__PURE__*/ (0, $fsr8S$jsx)($d946f50010ba937f$export$d22444a338b6e3c2, {
        ...props,
        ref: ref,
        children: /*#__PURE__*/ (0, $fsr8S$jsx)((0, $9afd80978c252040$export$f5b8910cec6cf069), {})
    });
});
const $d946f50010ba937f$export$2dc6166a7e65358c = /*#__PURE__*/ (0, $fsr8S$createContext)(null);
const $d946f50010ba937f$export$f5c9f3c2c4054eec = /*#__PURE__*/ (0, $fsr8S$forwardRef)(function TextArea(props, ref) {
    [props, ref] = (0, $5ce63c423902f47d$export$764f6146fadd77f7)(props, ref, $d946f50010ba937f$export$2dc6166a7e65358c);
    return /*#__PURE__*/ (0, $fsr8S$jsx)($d946f50010ba937f$export$d22444a338b6e3c2, {
        ...props,
        ref: ref,
        fieldGroupCss: " ea131 Fd131",
        children: /*#__PURE__*/ (0, $fsr8S$jsx)($d946f50010ba937f$var$TextAreaInput, {})
    });
});
const $d946f50010ba937f$export$d22444a338b6e3c2 = /*#__PURE__*/ (0, $fsr8S$forwardRef)(function TextFieldBase(props, ref) {
    let inputRef = (0, $fsr8S$useRef)(null);
    let domRef = (0, $fsr8S$useRef)(null);
    let formContext = (0, $fsr8S$useContext)((0, $9b916426527cebe7$export$c24727297075ec6a));
    props = (0, $9b916426527cebe7$export$a6b5be5c6b451665)(props);
    let { label: label, description: description, errorMessage: errorMessage, necessityIndicator: necessityIndicator, labelPosition: labelPosition = 'top', labelAlign: labelAlign = 'start', fieldGroupCss: fieldGroupCss, UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '', ...textFieldProps } = props;
    // Expose imperative interface for ref
    (0, $fsr8S$useImperativeHandle)(ref, ()=>({
            ...(0, $80ecb974df713a30$export$79d69eee6ae4b329)(domRef, inputRef),
            select () {
                if (inputRef.current) inputRef.current.select();
            },
            getInputElement () {
                return inputRef.current;
            }
        }));
    return /*#__PURE__*/ (0, $fsr8S$jsx)((0, $fsr8S$TextField), {
        ...textFieldProps,
        ref: domRef,
        style: UNSAFE_style,
        className: UNSAFE_className + function anonymous(props, overrides) {
            let rules = " ";
            let gridColumnStart = false;
            let gridColumnEnd = false;
            let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
            for (let p of matches){
                if (p[1] === "z") gridColumnStart = true;
                if (p[1] === "y") gridColumnEnd = true;
                rules += p[0];
            }
            rules += ' se131';
            if (props.isInForm) {
                if (!gridColumnStart) rules += ' z2mU131';
            }
            if (props.isInForm) {
                if (props.labelPosition === "side") {
                    if (!gridColumnEnd) rules += ' yvoofG131';
                }
            }
            if (props.isInForm) rules += ' DLwlgod131';
            else {
                if (props.labelPosition === "side") rules += ' DPlN5qb131';
                else if (props.labelPosition === "top") rules += ' DM8Mfn131';
            }
            if (props.labelPosition === "side") rules += ' EPlN5qb131';
            else if (props.labelPosition === "top") rules += ' E4w1sLc131';
            if (props.labelPosition === "side") rules += ' CbRM95c131';
            else if (props.labelPosition === "top") rules += ' Cxaocre131';
            if (props.size === "XL") {
                rules += ' -_6BNtrc-e131';
                rules += ' vx131';
            } else if (props.size === "L") {
                rules += ' -_6BNtrc-d131';
                rules += ' vx131';
            } else if (props.size === "S") {
                rules += ' -_6BNtrc-b131';
                rules += ' vx131';
            } else if (props.size === "XS") {
                rules += ' -_6BNtrc-a131';
                rules += ' vx131';
            } else {
                rules += ' -_6BNtrc-c131';
                rules += ' vx131';
            }
            rules += ' ea131';
            rules += ' _Fd131';
            rules += ' _FnuYUweb131';
            if (props.size === "XL") rules += ' -BhX7R-Fz131';
            else if (props.size === "L") rules += ' -BhX7R-Fv131';
            else if (props.size === "S") rules += ' -BhX7R-Fp131';
            else if (props.size === "XS") rules += ' -BhX7R-Fn131';
            else rules += ' -BhX7R-Fx131';
            rules += ' -AJjOLd-UVkF0Db131';
            rules += ' qg131';
            rules += ' __ca131';
            return rules;
        }({
            size: props.size,
            labelPosition: labelPosition,
            isInForm: !!formContext
        }, props.styles),
        children: (0, $fsr8S$composeRenderProps)(props.children, (children, { isDisabled: isDisabled, isInvalid: isInvalid })=>/*#__PURE__*/ (0, $fsr8S$jsxs)((0, $fsr8S$Fragment), {
                children: [
                    /*#__PURE__*/ (0, $fsr8S$jsx)((0, $9afd80978c252040$export$1acdcf5a973a8414), {
                        isDisabled: isDisabled,
                        isRequired: props.isRequired,
                        size: props.size,
                        labelPosition: labelPosition,
                        labelAlign: labelAlign,
                        necessityIndicator: necessityIndicator,
                        contextualHelp: props.contextualHelp,
                        children: label
                    }),
                    /*#__PURE__*/ (0, $fsr8S$jsx)((0, $9afd80978c252040$export$f0fe469072bc6950), {
                        prefix: props.prefix,
                        size: props.size,
                        styles: fieldGroupCss,
                        children: /*#__PURE__*/ (0, $fsr8S$jsxs)((0, $fsr8S$Provider), {
                            values: [
                                [
                                    (0, $8e847109a6ab556d$export$9afb8bc826b033ea),
                                    {}
                                ]
                            ],
                            children: [
                                /*#__PURE__*/ (0, $fsr8S$jsx)((0, $fsr8S$InputContext).Consumer, {
                                    children: (ctx)=>/*#__PURE__*/ (0, $fsr8S$jsx)((0, $fsr8S$InputContext).Provider, {
                                            value: {
                                                ...ctx,
                                                ref: (0, $fsr8S$mergeRefs)(ctx?.ref, inputRef)
                                            },
                                            children: children
                                        })
                                }),
                                isInvalid && /*#__PURE__*/ (0, $fsr8S$jsx)((0, $9afd80978c252040$export$bda0d075e0e342c9), {
                                    isDisabled: isDisabled
                                })
                            ]
                        })
                    }),
                    /*#__PURE__*/ (0, $fsr8S$jsx)((0, $9afd80978c252040$export$a67c0bc59081311a), {
                        size: props.size,
                        isDisabled: isDisabled,
                        isInvalid: isInvalid,
                        description: description,
                        children: errorMessage
                    })
                ]
            }))
    });
});
function $d946f50010ba937f$var$TextAreaInput() {
    // Force re-render when value changes so we update the height.
    let { placeholder: placeholder } = (0, $fsr8S$useSlottedContext)((0, $fsr8S$TextAreaContext)) ?? {};
    let onHeightChange = (input)=>{
        // TODO: only do this if an explicit height is not given?
        if (input) {
            let prevAlignment = input.style.alignSelf;
            let prevOverflow = input.style.overflow;
            // Firefox scroll position is lost when overflow: 'hidden' is applied so we skip applying it.
            // The measure/applied height is also incorrect/reset if we turn on and off
            // overflow: hidden in Firefox https://bugzilla.mozilla.org/show_bug.cgi?id=1787062
            let isFirefox = 'MozAppearance' in input.style;
            if (!isFirefox) input.style.overflow = 'hidden';
            input.style.alignSelf = 'start';
            input.style.height = 'auto';
            // offsetHeight - clientHeight accounts for the border/padding.
            input.style.height = `${input.scrollHeight + (input.offsetHeight - input.clientHeight)}px`;
            input.style.overflow = prevOverflow;
            input.style.alignSelf = prevAlignment;
        }
    };
    let { ref: ref } = (0, $fsr8S$useSlottedContext)((0, $fsr8S$InputContext)) ?? {};
    return /*#__PURE__*/ (0, $fsr8S$jsx)((0, $fsr8S$TextArea), {
        ref: (0, $fsr8S$mergeRefs)(onHeightChange, ref),
        // Workaround for baseline alignment bug in Safari.
        // https://bugs.webkit.org/show_bug.cgi?id=142968
        placeholder: placeholder ?? ' ',
        className: function anonymous(props) {
            let rules = " ";
            rules += ' Sd131';
            rules += ' Rd131';
            rules += ' Tu7Fubd131';
            rules += ' Qu7Fubd131';
            rules += ' MAyFGYc131';
            rules += ' _oa131';
            rules += ' g9131';
            rules += ' p40ub4c131';
            rules += ' pYFIK2bn131';
            rules += ' pYFIK2blc131';
            rules += ' u40ub4c131';
            rules += ' v40ub4c131';
            rules += ' x40ub4c131';
            rules += ' _xa131';
            rules += ' _F40ub4c131';
            rules += ' _ub131';
            rules += ' Na131';
            rules += ' _Le131';
            rules += ' _je131';
            rules += ' _Qc131';
            rules += ' _Nc131';
            if (props.size === "XL") rules += ' -M_-Mk131';
            else if (props.size === "L") rules += ' -M_-Mh131';
            else if (props.size === "S") rules += ' -M_-Mg131';
            else if (props.size === "XS") rules += ' -M_-Mf131';
            else rules += ' -M_-Mi131';
            return rules;
        }
    });
}


export {$d946f50010ba937f$export$2129e27b3ef0d483 as TextFieldContext, $d946f50010ba937f$export$2c73285ae9390cec as TextField, $d946f50010ba937f$export$d22444a338b6e3c2 as TextFieldBase, $d946f50010ba937f$export$2dc6166a7e65358c as TextAreaContext, $d946f50010ba937f$export$f5c9f3c2c4054eec as TextArea};
