// @react-spectrum/s2/dist/private/TextField.cjs
require("./TextField.css");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $a0b996a550bb0891$exports = require("./Field.cjs");
var $bfa4962d90c8af48$exports = require("./Form.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $tjrHw$reactjsxruntime = require("react/jsx-runtime");
var $tjrHw$reactariacomponentsTextArea = require("react-aria-components/TextArea");
var $tjrHw$reactariacomponentsTextField = require("react-aria-components/TextField");
var $tjrHw$reactariacomponentscomposeRenderProps = require("react-aria-components/composeRenderProps");
var $tjrHw$reactariacomponentsslots = require("react-aria-components/slots");
var $tjrHw$react = require("react");
var $tjrHw$reactariacomponentsInput = require("react-aria-components/Input");
var $tjrHw$reactariamergeRefs = require("react-aria/mergeRefs");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "TextFieldContext", () => $cce7e7de6810e1b5$export$2129e27b3ef0d483);
$parcel$export(module.exports, "TextField", () => $cce7e7de6810e1b5$export$2c73285ae9390cec);
$parcel$export(module.exports, "TextAreaContext", () => $cce7e7de6810e1b5$export$2dc6166a7e65358c);
$parcel$export(module.exports, "TextArea", () => $cce7e7de6810e1b5$export$f5c9f3c2c4054eec);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 












const $cce7e7de6810e1b5$export$2129e27b3ef0d483 = /*#__PURE__*/ (0, $tjrHw$react.createContext)(null);
const $cce7e7de6810e1b5$export$2c73285ae9390cec = /*#__PURE__*/ (0, $tjrHw$react.forwardRef)(function TextField(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $cce7e7de6810e1b5$export$2129e27b3ef0d483);
    return /*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsx)($cce7e7de6810e1b5$export$d22444a338b6e3c2, {
        ...props,
        ref: ref,
        children: /*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.Input), {})
    });
});
const $cce7e7de6810e1b5$export$2dc6166a7e65358c = /*#__PURE__*/ (0, $tjrHw$react.createContext)(null);
const $cce7e7de6810e1b5$export$f5c9f3c2c4054eec = /*#__PURE__*/ (0, $tjrHw$react.forwardRef)(function TextArea(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $cce7e7de6810e1b5$export$2dc6166a7e65358c);
    return /*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsx)($cce7e7de6810e1b5$export$d22444a338b6e3c2, {
        ...props,
        ref: ref,
        fieldGroupCss: " ea131 Fd131",
        children: /*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsx)($cce7e7de6810e1b5$var$TextAreaInput, {})
    });
});
const $cce7e7de6810e1b5$export$d22444a338b6e3c2 = /*#__PURE__*/ (0, $tjrHw$react.forwardRef)(function TextFieldBase(props, ref) {
    let inputRef = (0, $tjrHw$react.useRef)(null);
    let domRef = (0, $tjrHw$react.useRef)(null);
    let formContext = (0, $tjrHw$react.useContext)((0, $bfa4962d90c8af48$exports.FormContext));
    props = (0, $bfa4962d90c8af48$exports.useFormProps)(props);
    let { label: label, description: description, errorMessage: errorMessage, necessityIndicator: necessityIndicator, labelPosition: labelPosition = 'top', labelAlign: labelAlign = 'start', fieldGroupCss: fieldGroupCss, UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '', ...textFieldProps } = props;
    // Expose imperative interface for ref
    (0, $tjrHw$react.useImperativeHandle)(ref, ()=>({
            ...(0, $150382364eecc4bc$exports.createFocusableRef)(domRef, inputRef),
            select () {
                if (inputRef.current) inputRef.current.select();
            },
            getInputElement () {
                return inputRef.current;
            }
        }));
    return /*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsx)((0, $tjrHw$reactariacomponentsTextField.TextField), {
        ...textFieldProps,
        ref: domRef,
        style: UNSAFE_style,
        className: UNSAFE_className + function anonymous(props, overrides) {
            let rules = " ";
            let gridColumnStart = false;
            let gridColumnEnd = false;
            let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L)[^\s]+/g);
            for (let p of matches){
                if (p[1] === "z") gridColumnStart = true;
                if (p[1] === "y") gridColumnEnd = true;
                rules += p[0];
            }
            rules += ' se131';
            if (props.isInForm) {
                if (!gridColumnStart) rules += ' z2mU131';
            }
            if (props.isInForm) {
                if (props.labelPosition === "side") {
                    if (!gridColumnEnd) rules += ' yvoofG131';
                }
            }
            if (props.isInForm) rules += ' DLwlgod131';
            else {
                if (props.labelPosition === "side") rules += ' DPlN5qb131';
                else if (props.labelPosition === "top") rules += ' DM8Mfn131';
            }
            if (props.labelPosition === "side") rules += ' EPlN5qb131';
            else if (props.labelPosition === "top") rules += ' E4w1sLc131';
            if (props.labelPosition === "side") rules += ' CbRM95c131';
            else if (props.labelPosition === "top") rules += ' Cxaocre131';
            if (props.size === "XL") {
                rules += ' -_6BNtrc-e131';
                rules += ' vx131';
            } else if (props.size === "L") {
                rules += ' -_6BNtrc-d131';
                rules += ' vx131';
            } else if (props.size === "S") {
                rules += ' -_6BNtrc-b131';
                rules += ' vx131';
            } else if (props.size === "XS") {
                rules += ' -_6BNtrc-a131';
                rules += ' vx131';
            } else {
                rules += ' -_6BNtrc-c131';
                rules += ' vx131';
            }
            rules += ' ea131';
            rules += ' _Fd131';
            rules += ' _FnuYUweb131';
            if (props.size === "XL") rules += ' -BhX7R-Fz131';
            else if (props.size === "L") rules += ' -BhX7R-Fv131';
            else if (props.size === "S") rules += ' -BhX7R-Fp131';
            else if (props.size === "XS") rules += ' -BhX7R-Fn131';
            else rules += ' -BhX7R-Fx131';
            rules += ' -AJjOLd-UVkF0Db131';
            rules += ' qg131';
            rules += ' __ca131';
            return rules;
        }({
            size: props.size,
            labelPosition: labelPosition,
            isInForm: !!formContext
        }, props.styles),
        children: (0, $tjrHw$reactariacomponentscomposeRenderProps.composeRenderProps)(props.children, (children, { isDisabled: isDisabled, isInvalid: isInvalid })=>/*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsxs)((0, $tjrHw$reactjsxruntime.Fragment), {
                children: [
                    /*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldLabel), {
                        isDisabled: isDisabled,
                        isRequired: props.isRequired,
                        size: props.size,
                        labelPosition: labelPosition,
                        labelAlign: labelAlign,
                        necessityIndicator: necessityIndicator,
                        contextualHelp: props.contextualHelp,
                        children: label
                    }),
                    /*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldGroup), {
                        prefix: props.prefix,
                        size: props.size,
                        styles: fieldGroupCss,
                        children: /*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsxs)((0, $tjrHw$reactariacomponentsslots.Provider), {
                            values: [
                                [
                                    (0, $6367bc87eb7d24ad$exports.TextContext),
                                    {}
                                ]
                            ],
                            children: [
                                /*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsx)((0, $tjrHw$reactariacomponentsInput.InputContext).Consumer, {
                                    children: (ctx)=>/*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsx)((0, $tjrHw$reactariacomponentsInput.InputContext).Provider, {
                                            value: {
                                                ...ctx,
                                                ref: (0, $tjrHw$reactariamergeRefs.mergeRefs)(ctx?.ref, inputRef)
                                            },
                                            children: children
                                        })
                                }),
                                isInvalid && /*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.FieldErrorIcon), {
                                    isDisabled: isDisabled
                                })
                            ]
                        })
                    }),
                    /*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsx)((0, $a0b996a550bb0891$exports.HelpText), {
                        size: props.size,
                        isDisabled: isDisabled,
                        isInvalid: isInvalid,
                        description: description,
                        children: errorMessage
                    })
                ]
            }))
    });
});
function $cce7e7de6810e1b5$var$TextAreaInput() {
    // Force re-render when value changes so we update the height.
    let { placeholder: placeholder } = (0, $tjrHw$reactariacomponentsslots.useSlottedContext)((0, $tjrHw$reactariacomponentsTextArea.TextAreaContext)) ?? {};
    let onHeightChange = (input)=>{
        // TODO: only do this if an explicit height is not given?
        if (input) {
            let prevAlignment = input.style.alignSelf;
            let prevOverflow = input.style.overflow;
            // Firefox scroll position is lost when overflow: 'hidden' is applied so we skip applying it.
            // The measure/applied height is also incorrect/reset if we turn on and off
            // overflow: hidden in Firefox https://bugzilla.mozilla.org/show_bug.cgi?id=1787062
            let isFirefox = 'MozAppearance' in input.style;
            if (!isFirefox) input.style.overflow = 'hidden';
            input.style.alignSelf = 'start';
            input.style.height = 'auto';
            // offsetHeight - clientHeight accounts for the border/padding.
            input.style.height = `${input.scrollHeight + (input.offsetHeight - input.clientHeight)}px`;
            input.style.overflow = prevOverflow;
            input.style.alignSelf = prevAlignment;
        }
    };
    let { ref: ref } = (0, $tjrHw$reactariacomponentsslots.useSlottedContext)((0, $tjrHw$reactariacomponentsInput.InputContext)) ?? {};
    return /*#__PURE__*/ (0, $tjrHw$reactjsxruntime.jsx)((0, $tjrHw$reactariacomponentsTextArea.TextArea), {
        ref: (0, $tjrHw$reactariamergeRefs.mergeRefs)(onHeightChange, ref),
        // Workaround for baseline alignment bug in Safari.
        // https://bugs.webkit.org/show_bug.cgi?id=142968
        placeholder: placeholder ?? ' ',
        className: function anonymous(props) {
            let rules = " ";
            rules += ' Sd131';
            rules += ' Rd131';
            rules += ' Tu7Fubd131';
            rules += ' Qu7Fubd131';
            rules += ' MAyFGYc131';
            rules += ' _oa131';
            rules += ' g9131';
            rules += ' p40ub4c131';
            rules += ' pYFIK2bn131';
            rules += ' pYFIK2blc131';
            rules += ' u40ub4c131';
            rules += ' v40ub4c131';
            rules += ' x40ub4c131';
            rules += ' _xa131';
            rules += ' _F40ub4c131';
            rules += ' _ub131';
            rules += ' Na131';
            rules += ' _Le131';
            rules += ' _je131';
            rules += ' _Qc131';
            rules += ' _Nc131';
            if (props.size === "XL") rules += ' -M_-Mk131';
            else if (props.size === "L") rules += ' -M_-Mh131';
            else if (props.size === "S") rules += ' -M_-Mg131';
            else if (props.size === "XS") rules += ' -M_-Mf131';
            else rules += ' -M_-Mi131';
            return rules;
        }
    });
}
