// @react-spectrum/s2/dist/private/ToggleButton.cjs
require("./ToggleButton.css");
var $6e265ff388155b91$exports = require("./ActionButton.cjs");
var $e991cbcdf82ced71$exports = require("./CenterBaseline.cjs");
var $411768293e842abb$exports = require("./CornerTriangle.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $2061c83559b50a66$exports = require("./pressScale.cjs");
var $5eb75e0c130e0669$exports = require("../../icons/Skeleton.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $428d056a8f9b19e5$exports = require("./ToggleButtonGroup.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $bfa4962d90c8af48$exports = require("./Form.cjs");
var $ac757a4c2bd72aee$exports = require("../../icons/useSpectrumContextProps.cjs");
var $kyLWK$reactjsxruntime = require("react/jsx-runtime");
var $kyLWK$reactariacomponentsslots = require("react-aria-components/slots");
var $kyLWK$react = require("react");
var $kyLWK$reactariacomponentsToggleButton = require("react-aria-components/ToggleButton");
var $kyLWK$reactariaI18nProvider = require("react-aria/I18nProvider");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "ToggleButtonContext", () => $d16e5203405a91b7$export$43506d75ebd2e218);
$parcel$export(module.exports, "ToggleButton", () => $d16e5203405a91b7$export$d2b052e7b4be1756);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 















const $d16e5203405a91b7$export$43506d75ebd2e218 = /*#__PURE__*/ (0, $kyLWK$react.createContext)(null);
const $d16e5203405a91b7$export$d2b052e7b4be1756 = /*#__PURE__*/ (0, $kyLWK$react.forwardRef)(function ToggleButton(props, ref) {
    [props, ref] = (0, $ac757a4c2bd72aee$exports.useSpectrumContextProps)(props, ref, $d16e5203405a91b7$export$43506d75ebd2e218);
    props = (0, $bfa4962d90c8af48$exports.useFormProps)(props);
    let domRef = (0, $150382364eecc4bc$exports.useFocusableRef)(ref);
    let ctx = (0, $kyLWK$reactariacomponentsslots.useSlottedContext)((0, $428d056a8f9b19e5$exports.ToggleButtonGroupContext));
    let isInGroup = !!ctx;
    let { density: density = 'regular', isJustified: isJustified, orientation: orientation = 'horizontal', staticColor: staticColor = props.staticColor, isQuiet: isQuiet = props.isQuiet, isEmphasized: isEmphasized = props.isEmphasized, size: size = props.size || 'M', isDisabled: isDisabled = props.isDisabled } = ctx || {};
    let { holdAffordance: holdAffordance } = props;
    let { direction: direction } = (0, $kyLWK$reactariaI18nProvider.useLocale)();
    return /*#__PURE__*/ (0, $kyLWK$reactjsxruntime.jsxs)((0, $kyLWK$reactariacomponentsToggleButton.ToggleButton), {
        ...props,
        isDisabled: isDisabled,
        ref: domRef,
        style: (0, $2061c83559b50a66$exports.pressScale)(domRef, props.UNSAFE_style),
        className: (renderProps)=>(props.UNSAFE_className || '') + (0, $6e265ff388155b91$exports.btnStyles)({
                ...renderProps,
                staticColor: staticColor,
                isStaticColor: !!staticColor,
                size: size,
                isQuiet: isQuiet,
                isEmphasized: isEmphasized,
                isPending: false,
                density: density,
                isJustified: isJustified,
                orientation: orientation,
                isInGroup: isInGroup
            }, props.styles),
        children: [
            /*#__PURE__*/ (0, $kyLWK$reactjsxruntime.jsx)((0, $kyLWK$reactariacomponentsslots.Provider), {
                values: [
                    [
                        (0, $5eb75e0c130e0669$exports.SkeletonContext),
                        null
                    ],
                    [
                        (0, $6367bc87eb7d24ad$exports.TextContext),
                        {
                            styles: " TAzEcG131 QAzEcG131 _Jb131 _Nc131 Pc131 _Yb131 _7c131"
                        }
                    ],
                    [
                        (0, $bde97c91243ed164$exports.IconContext),
                        {
                            render: (0, $e991cbcdf82ced71$exports.centerBaseline)({
                                slot: 'icon',
                                styles: " _Ja131"
                            }),
                            styles: " ZuUmpre131 FuUmpre131 I67logd131 _va131"
                        }
                    ]
                ],
                children: typeof props.children === 'string' ? /*#__PURE__*/ (0, $kyLWK$reactjsxruntime.jsx)((0, $6367bc87eb7d24ad$exports.Text), {
                    children: props.children
                }) : props.children
            }),
            holdAffordance && /*#__PURE__*/ (0, $kyLWK$reactjsxruntime.jsx)((0, $411768293e842abb$exports.default), {
                size: size === 'XS' ? 'S' : size,
                className: function anonymous(props) {
                    let rules = " ";
                    rules += ' _Pa131';
                    if (props.size === "XL") rules += ' _zWJpoB131';
                    else if (props.size === "L") rules += ' _zC1otk131';
                    else if (props.size === "M") rules += ' _zE131';
                    else if (props.size === "S") rules += ' _zwtfhvc131';
                    else if (props.size === "XS") rules += ' _zwtfhvc131';
                    if (props.size === "XL") rules += ' _lWJpoB131';
                    else if (props.size === "L") rules += ' _lC1otk131';
                    else if (props.size === "M") rules += ' _lE131';
                    else if (props.size === "S") rules += ' _lwtfhvc131';
                    else if (props.size === "XS") rules += ' _lwtfhvc131';
                    if (props.direction === "rtl") {
                        rules += ' -Z4Pn4c-a131';
                        rules += ' __Ha131';
                    }
                    return rules;
                }({
                    direction: direction,
                    size: size
                })
            })
        ]
    });
});
