// @react-spectrum/s2/dist/private/Tooltip.mjs
import "./Tooltip.css";
import {ColorSchemeContext as $9affbfa4e13b9582$export$374e0245597d19a3} from "./Provider.mjs";
import {useDOMRef as $80ecb974df713a30$export$c2c55ef9111cafd8} from "./useDOMRef.mjs";
import {jsx as $iNhwA$jsx, jsxs as $iNhwA$jsxs, Fragment as $iNhwA$Fragment} from "react/jsx-runtime";
import {Tooltip as $iNhwA$Tooltip, OverlayArrow as $iNhwA$OverlayArrow, TooltipTrigger as $iNhwA$TooltipTrigger} from "react-aria-components/Tooltip";
import {createContext as $iNhwA$createContext, forwardRef as $iNhwA$forwardRef, useContext as $iNhwA$useContext, useState as $iNhwA$useState, useCallback as $iNhwA$useCallback} from "react";
import {useLocale as $iNhwA$useLocale} from "react-aria/I18nProvider";

/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





const $4c20647d6e5d8fdf$var$tooltip = function anonymous(props) {
    let rules = " ";
    if (props.colorScheme === "dark") rules += ' -ICUGx-_pa131';
    else if (props.colorScheme === "light") rules += ' -ICUGx-_pb131';
    else if (props.colorScheme === "light dark") rules += ' -ICUGx-_pc131';
    rules += ' _pK8eoyd131';
    if (props.colorScheme === "dark") rules += ' -NvXmrd-_ZLmU131';
    else if (props.colorScheme === "light") rules += ' -NvXmrd-_ZN8Gi4c131';
    else if (props.colorScheme === "light dark") {
        rules += ' -NvXmrd-_ZN8Gi4c131';
        rules += ' -NvXmrd-D1v0V_ZLmU131';
    }
    if (props.colorScheme === "dark") rules += ' -j9blY-_ZN8Gi4c131';
    else if (props.colorScheme === "light") rules += ' -j9blY-_ZLmU131';
    else if (props.colorScheme === "light dark") {
        rules += ' -j9blY-_ZLmU131';
        rules += ' -j9blY-D1v0V_ZN8Gi4c131';
    }
    rules += ' _Ca131';
    rules += ' eb131';
    rules += ' Le131';
    rules += ' MAyFGYc131';
    rules += ' _oa131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-b131';
    rules += ' vx131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    rules += ' pr131';
    rules += ' plb131';
    rules += ' _kJ8iPYd131';
    rules += ' hXMFGYc131';
    rules += ' mlb131';
    rules += ' llb131';
    rules += ' _jlf131';
    rules += ' ilA131';
    rules += ' g0131';
    rules += ' ob131';
    rules += ' nb131';
    rules += ' kb131';
    rules += ' jb131';
    rules += ' Sv131';
    rules += ' Rv131';
    rules += ' Tu7Fubd131';
    rules += ' Qu7Fubd131';
    rules += ' Yd131';
    rules += ' Xc131';
    if (props.isExiting) rules += ' _2c131';
    if (props.placement === "right") {
        if (props.isExiting) {
            rules += ' -_7PloMd-m131';
            rules += ' __Ya131';
        } else if (props.isEntering) {
            rules += ' -_7PloMd-m131';
            rules += ' __Ya131';
        }
    } else if (props.placement === "left") {
        if (props.isExiting) {
            rules += ' -_7PloMd-G131';
            rules += ' __Ya131';
        } else if (props.isEntering) {
            rules += ' -_7PloMd-G131';
            rules += ' __Ya131';
        }
    }
    if (props.placement === "bottom") {
        if (props.isExiting) {
            rules += ' -_8PloMd-l131';
            rules += ' __Ya131';
        } else if (props.isEntering) {
            rules += ' -_8PloMd-l131';
            rules += ' __Ya131';
        }
    } else if (props.placement === "top") {
        if (props.isExiting) {
            rules += ' -_8PloMd-F131';
            rules += ' __Ya131';
        } else if (props.isEntering) {
            rules += ' -_8PloMd-F131';
            rules += ' __Ya131';
        }
    }
    if (props.isExiting) rules += ' _Ia131';
    else if (props.isEntering) rules += ' _Ia131';
    rules += ' -nTSCV-LTAEPe131';
    rules += ' -M_-Mg131';
    rules += ' -_k_-l_kb131';
    rules += ' -h_-lhb131';
    return rules;
};
const $4c20647d6e5d8fdf$var$arrowStyles = function anonymous(props) {
    let rules = " ";
    rules += ' sb131';
    rules += ' tr131';
    rules += ' Zi131';
    rules += ' Ff131';
    if (props.placement === "right") rules += ' _Sd131';
    else if (props.placement === "left") rules += ' _Sa131';
    else if (props.placement === "bottom") rules += ' _Sc131';
    else if (props.placement === "top") rules += ' _Sb131';
    if (props.placement === "right") {
        rules += ' -_7PloMd-y131';
        rules += ' __Ya131';
    } else if (props.placement === "left") {
        rules += ' -_7PloMd-e131';
        rules += ' __Ya131';
    }
    return rules;
};
let $4c20647d6e5d8fdf$var$InternalTooltipTriggerContext = /*#__PURE__*/ (0, $iNhwA$createContext)({});
const $4c20647d6e5d8fdf$export$28c660c63b792dea = /*#__PURE__*/ (0, $iNhwA$forwardRef)(function Tooltip(props, ref) {
    let { children: children, UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '' } = props;
    let domRef = (0, $80ecb974df713a30$export$c2c55ef9111cafd8)(ref);
    let { containerPadding: containerPadding, crossOffset: crossOffset, placement: placement = 'top', shouldFlip: shouldFlip } = (0, $iNhwA$useContext)($4c20647d6e5d8fdf$var$InternalTooltipTriggerContext);
    let colorScheme = (0, $iNhwA$useContext)((0, $9affbfa4e13b9582$export$374e0245597d19a3));
    let { locale: locale, direction: direction } = (0, $iNhwA$useLocale)();
    let [borderRadius, setBorderRadius] = (0, $iNhwA$useState)(0);
    // TODO: should we pass through lang and dir props in RAC?
    let tooltipRef = (0, $iNhwA$useCallback)((el)=>{
        domRef.current = el;
        if (el) {
            el.lang = locale;
            el.dir = direction;
            let spectrumBorderRadius = typeof window !== 'undefined' ? window.getComputedStyle(el).borderRadius : '';
            if (spectrumBorderRadius !== '') setBorderRadius(parseInt(spectrumBorderRadius, 10));
        }
    }, [
        locale,
        direction,
        domRef
    ]);
    return /*#__PURE__*/ (0, $iNhwA$jsx)((0, $iNhwA$Tooltip), {
        ...props,
        arrowBoundaryOffset: borderRadius,
        containerPadding: containerPadding,
        crossOffset: crossOffset,
        offset: 9,
        placement: placement,
        shouldFlip: shouldFlip,
        ref: tooltipRef,
        style: UNSAFE_style,
        className: (renderProps)=>UNSAFE_className + $4c20647d6e5d8fdf$var$tooltip({
                ...renderProps,
                colorScheme: colorScheme
            }),
        children: (renderProps)=>/*#__PURE__*/ (0, $iNhwA$jsxs)((0, $iNhwA$Fragment), {
                children: [
                    /*#__PURE__*/ (0, $iNhwA$jsx)((0, $iNhwA$OverlayArrow), {
                        className: "",
                        children: /*#__PURE__*/ (0, $iNhwA$jsx)("svg", {
                            className: $4c20647d6e5d8fdf$var$arrowStyles(renderProps),
                            xmlns: "http://www.w3.org/2000/svg",
                            viewBox: "0 0 10 5",
                            children: /*#__PURE__*/ (0, $iNhwA$jsx)("path", {
                                d: "M4.29289 4.29289L0 0H10L5.70711 4.29289C5.31658 4.68342 4.68342 4.68342 4.29289 4.29289Z"
                            })
                        })
                    }),
                    children
                ]
            })
    });
});
function $4c20647d6e5d8fdf$export$8c610744efcf8a1d(props) {
    let { containerPadding: containerPadding, crossOffset: crossOffset, placement: placement, shouldFlip: shouldFlip, ...triggerProps } = props;
    return /*#__PURE__*/ (0, $iNhwA$jsx)((0, $iNhwA$TooltipTrigger), {
        ...triggerProps,
        children: /*#__PURE__*/ (0, $iNhwA$jsx)($4c20647d6e5d8fdf$var$InternalTooltipTriggerContext.Provider, {
            value: {
                containerPadding: containerPadding,
                crossOffset: crossOffset,
                placement: placement,
                shouldFlip: shouldFlip
            },
            children: props.children
        })
    });
}
function $4c20647d6e5d8fdf$export$92ee03cda75aac2b(props) {
    return /*#__PURE__*/ (0, $iNhwA$jsx)("div", {});
}


export {$4c20647d6e5d8fdf$export$28c660c63b792dea as Tooltip, $4c20647d6e5d8fdf$export$8c610744efcf8a1d as TooltipTrigger, $4c20647d6e5d8fdf$export$92ee03cda75aac2b as CombinedTooltip};
