// @react-spectrum/s2/dist/private/Tooltip.cjs
require("./Tooltip.css");
var $914a06b3a1c679bc$exports = require("./Provider.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $8KZCJ$reactjsxruntime = require("react/jsx-runtime");
var $8KZCJ$reactariacomponentsTooltip = require("react-aria-components/Tooltip");
var $8KZCJ$react = require("react");
var $8KZCJ$reactariaI18nProvider = require("react-aria/I18nProvider");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "Tooltip", () => $d3b6a84aebe66f8d$export$28c660c63b792dea);
$parcel$export(module.exports, "TooltipTrigger", () => $d3b6a84aebe66f8d$export$8c610744efcf8a1d);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 





const $d3b6a84aebe66f8d$var$tooltip = function anonymous(props) {
    let rules = " ";
    if (props.colorScheme === "dark") rules += ' -ICUGx-_pa131';
    else if (props.colorScheme === "light") rules += ' -ICUGx-_pb131';
    else if (props.colorScheme === "light dark") rules += ' -ICUGx-_pc131';
    rules += ' _pK8eoyd131';
    if (props.colorScheme === "dark") rules += ' -NvXmrd-_ZLmU131';
    else if (props.colorScheme === "light") rules += ' -NvXmrd-_ZN8Gi4c131';
    else if (props.colorScheme === "light dark") {
        rules += ' -NvXmrd-_ZN8Gi4c131';
        rules += ' -NvXmrd-D1v0V_ZLmU131';
    }
    if (props.colorScheme === "dark") rules += ' -j9blY-_ZN8Gi4c131';
    else if (props.colorScheme === "light") rules += ' -j9blY-_ZLmU131';
    else if (props.colorScheme === "light dark") {
        rules += ' -j9blY-_ZLmU131';
        rules += ' -j9blY-D1v0V_ZN8Gi4c131';
    }
    rules += ' _Ca131';
    rules += ' eb131';
    rules += ' Le131';
    rules += ' MAyFGYc131';
    rules += ' _oa131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-b131';
    rules += ' vx131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    rules += ' pr131';
    rules += ' plb131';
    rules += ' _kJ8iPYd131';
    rules += ' hXMFGYc131';
    rules += ' mlb131';
    rules += ' llb131';
    rules += ' _jlf131';
    rules += ' ilA131';
    rules += ' g0131';
    rules += ' ob131';
    rules += ' nb131';
    rules += ' kb131';
    rules += ' jb131';
    rules += ' Sv131';
    rules += ' Rv131';
    rules += ' Tu7Fubd131';
    rules += ' Qu7Fubd131';
    rules += ' Yd131';
    rules += ' Xc131';
    if (props.isExiting) rules += ' _2c131';
    if (props.placement === "right") {
        if (props.isExiting) {
            rules += ' -_7PloMd-m131';
            rules += ' __Ya131';
        } else if (props.isEntering) {
            rules += ' -_7PloMd-m131';
            rules += ' __Ya131';
        }
    } else if (props.placement === "left") {
        if (props.isExiting) {
            rules += ' -_7PloMd-G131';
            rules += ' __Ya131';
        } else if (props.isEntering) {
            rules += ' -_7PloMd-G131';
            rules += ' __Ya131';
        }
    }
    if (props.placement === "bottom") {
        if (props.isExiting) {
            rules += ' -_8PloMd-l131';
            rules += ' __Ya131';
        } else if (props.isEntering) {
            rules += ' -_8PloMd-l131';
            rules += ' __Ya131';
        }
    } else if (props.placement === "top") {
        if (props.isExiting) {
            rules += ' -_8PloMd-F131';
            rules += ' __Ya131';
        } else if (props.isEntering) {
            rules += ' -_8PloMd-F131';
            rules += ' __Ya131';
        }
    }
    if (props.isExiting) rules += ' _Ia131';
    else if (props.isEntering) rules += ' _Ia131';
    rules += ' -nTSCV-LTAEPe131';
    rules += ' -M_-Mg131';
    rules += ' -_k_-l_kb131';
    rules += ' -h_-lhb131';
    return rules;
};
const $d3b6a84aebe66f8d$var$arrowStyles = function anonymous(props) {
    let rules = " ";
    rules += ' sb131';
    rules += ' tr131';
    rules += ' Zi131';
    rules += ' Ff131';
    if (props.placement === "right") rules += ' _Sd131';
    else if (props.placement === "left") rules += ' _Sa131';
    else if (props.placement === "bottom") rules += ' _Sc131';
    else if (props.placement === "top") rules += ' _Sb131';
    if (props.placement === "right") {
        rules += ' -_7PloMd-y131';
        rules += ' __Ya131';
    } else if (props.placement === "left") {
        rules += ' -_7PloMd-e131';
        rules += ' __Ya131';
    }
    return rules;
};
let $d3b6a84aebe66f8d$var$InternalTooltipTriggerContext = /*#__PURE__*/ (0, $8KZCJ$react.createContext)({});
const $d3b6a84aebe66f8d$export$28c660c63b792dea = /*#__PURE__*/ (0, $8KZCJ$react.forwardRef)(function Tooltip(props, ref) {
    let { children: children, UNSAFE_style: UNSAFE_style, UNSAFE_className: UNSAFE_className = '' } = props;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let { containerPadding: containerPadding, crossOffset: crossOffset, placement: placement = 'top', shouldFlip: shouldFlip } = (0, $8KZCJ$react.useContext)($d3b6a84aebe66f8d$var$InternalTooltipTriggerContext);
    let colorScheme = (0, $8KZCJ$react.useContext)((0, $914a06b3a1c679bc$exports.ColorSchemeContext));
    let { locale: locale, direction: direction } = (0, $8KZCJ$reactariaI18nProvider.useLocale)();
    let [borderRadius, setBorderRadius] = (0, $8KZCJ$react.useState)(0);
    // TODO: should we pass through lang and dir props in RAC?
    let tooltipRef = (0, $8KZCJ$react.useCallback)((el)=>{
        domRef.current = el;
        if (el) {
            el.lang = locale;
            el.dir = direction;
            let spectrumBorderRadius = typeof window !== 'undefined' ? window.getComputedStyle(el).borderRadius : '';
            if (spectrumBorderRadius !== '') setBorderRadius(parseInt(spectrumBorderRadius, 10));
        }
    }, [
        locale,
        direction,
        domRef
    ]);
    return /*#__PURE__*/ (0, $8KZCJ$reactjsxruntime.jsx)((0, $8KZCJ$reactariacomponentsTooltip.Tooltip), {
        ...props,
        arrowBoundaryOffset: borderRadius,
        containerPadding: containerPadding,
        crossOffset: crossOffset,
        offset: 9,
        placement: placement,
        shouldFlip: shouldFlip,
        ref: tooltipRef,
        style: UNSAFE_style,
        className: (renderProps)=>UNSAFE_className + $d3b6a84aebe66f8d$var$tooltip({
                ...renderProps,
                colorScheme: colorScheme
            }),
        children: (renderProps)=>/*#__PURE__*/ (0, $8KZCJ$reactjsxruntime.jsxs)((0, $8KZCJ$reactjsxruntime.Fragment), {
                children: [
                    /*#__PURE__*/ (0, $8KZCJ$reactjsxruntime.jsx)((0, $8KZCJ$reactariacomponentsTooltip.OverlayArrow), {
                        className: "",
                        children: /*#__PURE__*/ (0, $8KZCJ$reactjsxruntime.jsx)("svg", {
                            className: $d3b6a84aebe66f8d$var$arrowStyles(renderProps),
                            xmlns: "http://www.w3.org/2000/svg",
                            viewBox: "0 0 10 5",
                            children: /*#__PURE__*/ (0, $8KZCJ$reactjsxruntime.jsx)("path", {
                                d: "M4.29289 4.29289L0 0H10L5.70711 4.29289C5.31658 4.68342 4.68342 4.68342 4.29289 4.29289Z"
                            })
                        })
                    }),
                    children
                ]
            })
    });
});
function $d3b6a84aebe66f8d$export$8c610744efcf8a1d(props) {
    let { containerPadding: containerPadding, crossOffset: crossOffset, placement: placement, shouldFlip: shouldFlip, ...triggerProps } = props;
    return /*#__PURE__*/ (0, $8KZCJ$reactjsxruntime.jsx)((0, $8KZCJ$reactariacomponentsTooltip.TooltipTrigger), {
        ...triggerProps,
        children: /*#__PURE__*/ (0, $8KZCJ$reactjsxruntime.jsx)($d3b6a84aebe66f8d$var$InternalTooltipTriggerContext.Provider, {
            value: {
                containerPadding: containerPadding,
                crossOffset: crossOffset,
                placement: placement,
                shouldFlip: shouldFlip
            },
            children: props.children
        })
    });
}
function $d3b6a84aebe66f8d$export$92ee03cda75aac2b(props) {
    return /*#__PURE__*/ (0, $8KZCJ$reactjsxruntime.jsx)("div", {});
}
