// @react-spectrum/s2/dist/private/TreeView.cjs
require("./TreeView.css");
var $7a26131f6144af2b$exports = require("./ActionButtonGroup.cjs");
var $e5a758dd286599f2$exports = require("./ActionMenu.cjs");
var $e991cbcdf82ced71$exports = require("./CenterBaseline.cjs");
var $af04e099a53d3e85$exports = require("./Checkbox.cjs");
var $1df9f1c9262ce5df$exports = require("./Chevron.cjs");
var $f1f08ef5de95f8eb$exports = require("./dnd-utils.cjs");
var $35d2da6f10cb8588$exports = require("./DragPreview.cjs");
var $bde97c91243ed164$exports = require("../../icons/Icon.cjs");
var $4526404114e78c80$exports = require("./intlStrings.cjs");
var $1c116ea4b0d2d023$exports = require("./ProgressCircle.cjs");
var $387d58020e2a8028$exports = require("./ListView.cjs");
var $6367bc87eb7d24ad$exports = require("./Content.cjs");
var $85bf454149fe9ddc$exports = require("./ActionBar.cjs");
var $150382364eecc4bc$exports = require("./useDOMRef.cjs");
var $ee7b4c497f520c08$exports = require("./utils.cjs");
var $6K4Mt$reactjsxruntime = require("react/jsx-runtime");
var $6K4Mt$reactariacomponentsButton = require("react-aria-components/Button");
var $6K4Mt$reactariacomponentsslots = require("react-aria-components/slots");
var $6K4Mt$reactariacomponentsTree = require("react-aria-components/Tree");
var $6K4Mt$react = require("react");
var $6K4Mt$reactariaI18nProvider = require("react-aria/I18nProvider");
var $6K4Mt$reactariauseLocalizedStringFormatter = require("react-aria/useLocalizedStringFormatter");
var $6K4Mt$reactariacomponentsVirtualizer = require("react-aria-components/Virtualizer");


function $parcel$interopDefault(a) {
  return a && a.__esModule ? a.default : a;
}

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "TreeView", () => $8a2a97a0cb171e69$export$6940b0d9c820eca7);
$parcel$export(module.exports, "TreeViewItem", () => $8a2a97a0cb171e69$export$6e77ea6719814e9c);
$parcel$export(module.exports, "TreeViewItemContent", () => $8a2a97a0cb171e69$export$9a5779ed3fade674);
$parcel$export(module.exports, "TreeViewLoadMoreItem", () => $8a2a97a0cb171e69$export$6ceebe2a029205c);
/*
 * Copyright 2024 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 






















const $8a2a97a0cb171e69$var$TreeRendererContext = /*#__PURE__*/ (0, $6K4Mt$react.createContext)({});
const $8a2a97a0cb171e69$var$treeViewWrapper = function anonymous(props, overrides) {
    let rules = " ";
    let minHeight = false;
    let height = false;
    let minWidth = false;
    let position = false;
    let matches = String(overrides || '').matchAll(/(?:^|\s)(J|G|I|H|_u|_v|_s|__A|_d|_J|z|y|B|A|_P|_9|W|_l|_A|_z|_6|Z|N|L|F|M|K)[^\s]+/g);
    for (let p of matches){
        if (p[1] === "M") minHeight = true;
        if (p[1] === "F") height = true;
        if (p[1] === "N") minWidth = true;
        if (p[1] === "_P") position = true;
        rules += p[0];
    }
    if (!minHeight) rules += ' Ma131';
    if (!height) rules += ' Fb131';
    if (!minWidth) rules += ' NMaAbMd131';
    rules += ' sd131';
    rules += ' _Bb131';
    rules += ' __ca131';
    if (!position) rules += ' _Pc131';
    rules += ' _Nb131';
    rules += ' Pb131';
    rules += ' -dBGRde-ZVvYXid131';
    rules += ' -qVUQke-oa131';
    return rules;
};
// TODO: can't use border cuz it means the contents of the tree are no longer
// flush with its container. Outline and box shadow are affected by the outlineOffset we
// have in the tree styles and result in the rows clipping the drop outline. This approach
// avoids that
const $8a2a97a0cb171e69$var$rootDropOutline = "_2RDoQb";
// These are the same as ListView. we didn't have v3 tree dnd and dont have designs so to be adjusted later
const $8a2a97a0cb171e69$var$dropTargetBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 10%)]";
const $8a2a97a0cb171e69$var$rootDropSelectedRowBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 28%)]";
const $8a2a97a0cb171e69$var$rowDropBackground = "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(59, 99, 251), rgb(86, 129, 255)) 10%)]";
const $8a2a97a0cb171e69$var$rootRowDropStyles = {
    default: $8a2a97a0cb171e69$var$dropTargetBackground,
    isSelected: $8a2a97a0cb171e69$var$rootDropSelectedRowBackground,
    forcedColors: 'Background'
};
const $8a2a97a0cb171e69$var$rowDropStyles = {
    default: $8a2a97a0cb171e69$var$rowDropBackground,
    isSelected: $8a2a97a0cb171e69$var$rowDropBackground,
    forcedColors: 'Background'
};
// TODO: the below is needed so the borders of the top and bottom row isn't cut off if the TreeView is wrapped within a container by always reserving the 2px needed for the
// keyboard focus ring. Perhaps find a different way of rendering the outlines since the top of the item doesn't
// scroll into view due to how the ring is offset. Alternatively, have the tree render the top/bottom outline like it does in Listview
const $8a2a97a0cb171e69$var$tree = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    rules += ' Oh131';
    rules += ' Olc131';
    rules += ' _Mc131';
    rules += ' _Kb131';
    rules += ' _4c131';
    rules += ' Ma131';
    rules += ' Na131';
    rules += ' Za131';
    rules += ' Fb131';
    rules += ' _Na131';
    rules += ' Pa131';
    rules += ' _oa131';
    if (props.isDropTarget) {
        rules += ' glzvDBe131';
        rules += ' gla131';
    }
    if (props.isEmpty) rules += ' _Ca131';
    if (props.isEmpty) rules += ' eb131';
    rules += ' -lPUMEc-Zu131';
    return rules;
};
let $8a2a97a0cb171e69$var$InternalTreeViewContext = /*#__PURE__*/ (0, $6K4Mt$react.createContext)({});
const $8a2a97a0cb171e69$export$6940b0d9c820eca7 = /*#__PURE__*/ (0, $6K4Mt$react.forwardRef)(function TreeView(props, ref) {
    let { children: children, selectionStyle: selectionStyle = 'checkbox', UNSAFE_className: UNSAFE_className, UNSAFE_style: UNSAFE_style, dragAndDropHooks: dragAndDropHooks } = props;
    let scale = (0, $ee7b4c497f520c08$exports.useScale)();
    if (dragAndDropHooks && dragAndDropHooks.renderDragPreview == null) dragAndDropHooks.renderDragPreview = (items)=>/*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $35d2da6f10cb8588$exports.DragPreview), {
            items: items
        });
    let hasCheckbox = props.selectionMode !== 'none' && selectionStyle !== 'highlight';
    if (dragAndDropHooks) dragAndDropHooks.renderDropIndicator = (target)=>/*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $f1f08ef5de95f8eb$exports.InsertionIndicator), {
            target: target
        });
    let renderer;
    if (typeof children === 'function') renderer = children;
    let domRef = (0, $150382364eecc4bc$exports.useDOMRef)(ref);
    let scrollRef = (0, $6K4Mt$react.useRef)(null);
    let { selectedKeys: selectedKeys, onSelectionChange: onSelectionChange, actionBar: actionBar, actionBarHeight: actionBarHeight } = (0, $85bf454149fe9ddc$exports.useActionBarContainer)({
        ...props,
        scrollRef: scrollRef
    });
    return /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsxs)("div", {
        ref: domRef,
        className: (UNSAFE_className ?? '') + $8a2a97a0cb171e69$var$treeViewWrapper({
            hasCheckbox: hasCheckbox
        }, props.styles) + ' ' + $8a2a97a0cb171e69$var$rootDropOutline,
        style: UNSAFE_style,
        children: [
            /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $6K4Mt$reactariacomponentsVirtualizer.Virtualizer), {
                layout: (0, $387d58020e2a8028$exports.S2ListLayout),
                layoutOptions: {
                    rowHeight: scale === 'large' ? 50 : 40,
                    dropIndicatorThickness: 0
                },
                children: /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)($8a2a97a0cb171e69$var$TreeRendererContext.Provider, {
                    value: {
                        renderer: renderer
                    },
                    children: /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)($8a2a97a0cb171e69$var$InternalTreeViewContext.Provider, {
                        value: {
                            selectionStyle: selectionStyle
                        },
                        children: /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $6K4Mt$reactariacomponentsTree.Tree), {
                            ...props,
                            dragAndDropHooks: dragAndDropHooks,
                            style: {
                                paddingBottom: actionBarHeight > 0 ? actionBarHeight + 8 : 0,
                                scrollPaddingBottom: actionBarHeight > 0 ? actionBarHeight + 8 : 0
                            },
                            className: (renderProps)=>$8a2a97a0cb171e69$var$tree(renderProps),
                            selectionBehavior: selectionStyle === 'highlight' ? 'replace' : 'toggle',
                            selectedKeys: selectedKeys,
                            defaultSelectedKeys: undefined,
                            onSelectionChange: onSelectionChange,
                            ref: scrollRef,
                            children: props.children
                        })
                    })
                })
            }),
            actionBar
        ]
    });
});
const $8a2a97a0cb171e69$var$rowBackgroundColor = {
    default: '--s2-container-bg',
    isFocusVisibleWithin: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 7%)]",
    isHovered: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 7%)]",
    isPressed: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 7%)]",
    isSelected: {
        default: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 7%)]",
        isFocusVisibleWithin: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 10%)]",
        isHovered: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 10%)]",
        isPressed: "[color-mix(in srgb, light-dark(rgb(255, 255, 255), rgb(17, 17, 17)), light-dark(rgb(19, 19, 19), rgb(242, 242, 242)) 10%)]"
    },
    forcedColors: {
        default: 'Background'
    }
};
const $8a2a97a0cb171e69$var$treeRow = function anonymous(props) {
    let rules = " ";
    if (props.isDropTarget) rules += ' _Lf131';
    else rules += ' _Le131';
    if (props.isDropTarget) rules += ' _Mc131';
    if (props.isDropTarget) rules += ' _Kb131';
    if (props.isDropTarget) rules += ' Oh131';
    if (props.isDropTarget) rules += ' Olc131';
    rules += ' _Pc131';
    rules += ' sd131';
    rules += ' Fv131';
    rules += ' Za131';
    rules += ' _oa131';
    rules += ' uk131';
    rules += ' ucJ9TBTb131';
    rules += ' ud3Euai131';
    rules += ' uea131';
    rules += ' ugb131';
    rules += ' uhd131';
    rules += ' uje131';
    rules += ' u2NhKxcl131';
    rules += ' uic131';
    rules += ' -_6BNtrc-c131';
    rules += ' vx131';
    rules += ' xb131';
    rules += ' _xa131';
    rules += ' _Fd131';
    rules += ' _FnuYUweb131';
    if (props.isSelected) {
        if (props.isPressed) rules += ' po131';
        else if (props.isFocusVisible) rules += ' po131';
        else if (props.isHovered) rules += ' po131';
        else rules += ' pt131';
    } else {
        if (props.isPressed) rules += ' pt131';
        else if (props.isFocusVisible) rules += ' pt131';
        else if (props.isHovered) rules += ' pt131';
        else rules += ' pv131';
    }
    rules += ' plb131';
    if (props.isLink) rules += ' ry131';
    else rules += ' ri131';
    if (props.isSelected) {
        if (props.isPressed) rules += ' -qH4xad-gRfT7e131';
        else if (props.isHovered) rules += ' -qH4xad-gRfT7e131';
        else if (props.isFocusVisibleWithin) rules += ' -qH4xad-gRfT7e131';
        else rules += ' -qH4xad-gj1B7i131';
    } else if (props.isPressed) rules += ' -qH4xad-gj1B7i131';
    else if (props.isHovered) rules += ' -qH4xad-gj1B7i131';
    else if (props.isFocusVisibleWithin) rules += ' -qH4xad-gj1B7i131';
    else rules += ' -qH4xad-gEzkoQe131';
    rules += ' -qH4xad-lga131';
    rules += ' -_8Gqznb-oa131';
    rules += ' oa131';
    rules += ' na131';
    rules += ' ka131';
    rules += ' ja131';
    return rules;
};
const $8a2a97a0cb171e69$var$treeCellGrid = function anonymous(props) {
    let rules = " ";
    rules += ' se131';
    rules += ' Za131';
    rules += ' Fb131';
    rules += ' _oa131';
    rules += ' oa131';
    rules += ' na131';
    rules += ' ka131';
    rules += ' ja131';
    rules += ' _cb131';
    rules += ' eb131';
    rules += ' DtBRkae131';
    rules += ' EM8Mfn131';
    rules += ' C4pAWc131';
    rules += ' Rp131';
    if (props.selectionStyle === "highlight") {
        if (props.isSelected) rules += ' ple131';
    } else {
        if (props.isDisabled) rules += ' pp131';
        rules += ' plb131';
    }
    if (props.isFocusVisible) rules += ' -_7xg3bb-Oh131';
    else rules += ' -_7xg3bb-O8Izf8c131';
    rules += ' -_7xg3bb-lOc131';
    rules += ' -pKMv1c-Oh131';
    rules += ' -pKMv1c-lOc131';
    if (props.selectionStyle === "highlight") rules += ' -EH8uVb-iw131';
    else rules += ' -EH8uVb-iA131';
    rules += ' -EH8uVb-lia131';
    rules += ' _yb131';
    return rules;
};
const $8a2a97a0cb171e69$var$treeRowBackground = function anonymous(props) {
    let rules = " ";
    rules += ' _Pa131';
    rules += ' _9a131';
    rules += ' Wr131';
    rules += ' _lr131';
    rules += ' _Ar131';
    rules += ' _zr131';
    if (props.isDropTarget) {
        if (props.isSelected) rules += ' glzvDBe131';
        else rules += ' glzvDBe131';
        rules += ' gla131';
    } else {
        if (props.selectionStyle === "highlight") {
            if (props.isSelected) {
                if (props.isFocusVisible) rules += ' gaZwDBe131';
                else if (props.isPressed) rules += ' gaZwDBe131';
                else if (props.isHovered) rules += ' gaZwDBe131';
                else rules += ' glzvDBe131';
                rules += ' gle131';
            } else rules += ' gs9yXOb131';
        } else {
            rules += ' gs9yXOb131';
            rules += ' gla131';
        }
        if (props.isSelected) rules += ' gRQwW1ubHDBe131';
        else rules += ' gRQwW1lzvDBe131';
        rules += ' gRQwW1la131';
    }
    if (props.isPreviousSelected) {
        if (props.isSelected) rules += ' od131';
        else rules += ' o4VHKAc131';
    } else rules += ' o4VHKAc131';
    if (props.isPreviousSelected) {
        if (props.isSelected) rules += ' nd131';
        else rules += ' n4VHKAc131';
    } else rules += ' n4VHKAc131';
    if (props.isNextSelected) {
        if (props.isSelected) rules += ' kd131';
        else rules += ' k4VHKAc131';
    } else rules += ' k4VHKAc131';
    if (props.isNextSelected) {
        if (props.isSelected) rules += ' jd131';
        else rules += ' j4VHKAc131';
    } else rules += ' j4VHKAc131';
    if (props.isPreviousSelected) rules += ' _ka131';
    else rules += ' _kb131';
    if (props.isNextSelected) rules += ' ha131';
    else rules += ' hb131';
    rules += ' mb131';
    rules += ' lb131';
    rules += ' _jf131';
    if (props.isSelected) rules += ' i4pzfu131';
    else rules += ' iA131';
    return rules;
};
const $8a2a97a0cb171e69$var$treeCheckbox = function anonymous(props) {
    let rules = " ";
    rules += ' zoRzcu131';
    rules += ' yoRzcu131';
    rules += ' BoRzcu131';
    rules += ' AoRzcu131';
    rules += ' Iv131';
    rules += ' Hy131';
    rules += ' Rd131';
    if (props.isDisabled) rules += ' _6b131';
    else rules += ' _6c131';
    return rules;
};
const $8a2a97a0cb171e69$var$treeIcon = " z2U5Crc131 y2U5Crc131 B2U5Crc131 A2U5Crc131 Ht131 Hsu131 -_8sjo0b-t5ZbAob131";
const $8a2a97a0cb171e69$var$treeContent = " zk52g2d131 yk52g2d131 Bk52g2d131 Ak52g2d131 _Yb131 _7c131 _Nc131 Pc131";
const $8a2a97a0cb171e69$var$treeActions = " zkG38H131 ykG38H131 BkG38H131 AkG38H131 Ir131 Hs131";
const $8a2a97a0cb171e69$var$treeActionMenu = " z6Nrfjc131 y6Nrfjc131 B6Nrfjc131 A6Nrfjc131";
const $8a2a97a0cb171e69$var$treeDragButtonContainer = " zqEWDzc131 yqEWDzc131 BqEWDzc131 AqEWDzc131 sd131 eb131 _Ca131 Zi131";
let $8a2a97a0cb171e69$var$treeRowFocusRing = function anonymous(props) {
    let rules = " ";
    if (props.isFocusVisible) rules += ' _Lf131';
    else rules += ' _Le131';
    if (props.selectionStyle === "highlight") {
        rules += ' Oh131';
        rules += ' Ola131';
    } else {
        rules += ' Oh131';
        rules += ' Olc131';
    }
    rules += ' _Mc131';
    rules += ' _MlkNNfn131';
    rules += ' _Kb131';
    rules += ' _KlToPtrc131';
    rules += ' _Pa131';
    if (props.isFirstItem) rules += ' Wr131';
    else rules += ' WLPOtrc131';
    if (props.selectionStyle === "highlight") {
        if (props.isSelected) {
            if (props.isNextSelected) rules += ' _lr131';
            else rules += ' _lr131';
        } else if (props.isNextSelected) rules += ' _lLPOtrc131';
        else rules += ' _lr131';
    } else if (props.selectionStyle === "checkbox") rules += ' _lr131';
    rules += ' _Ar131';
    rules += ' _zr131';
    rules += ' oa131';
    rules += ' na131';
    rules += ' ka131';
    rules += ' ja131';
    rules += ' _9c131';
    rules += ' _Ob131';
    return rules;
};
const $8a2a97a0cb171e69$export$6e77ea6719814e9c = (props)=>{
    let { href: href } = props;
    return /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $6K4Mt$reactariacomponentsTree.TreeItem), {
        ...props,
        className: (renderProps)=>$8a2a97a0cb171e69$var$treeRow({
                ...renderProps,
                isLink: !!href
            })
    });
};
const $8a2a97a0cb171e69$export$9a5779ed3fade674 = (props)=>{
    let { children: children } = props;
    let scale = (0, $ee7b4c497f520c08$exports.useScale)();
    let { selectionStyle: selectionStyle } = (0, $6K4Mt$react.useContext)($8a2a97a0cb171e69$var$InternalTreeViewContext);
    return /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $6K4Mt$reactariacomponentsTree.TreeItemContent), {
        children: ({ isExpanded: isExpanded, hasChildItems: hasChildItems, selectionMode: selectionMode, selectionBehavior: selectionBehavior, isDisabled: isDisabled, isSelected: isSelected, id: id, state: state, isHovered: isHovered, isFocusVisible: isFocusVisible, isFocusVisibleWithin: isFocusVisibleWithin, allowsDragging: allowsDragging, isDropTarget: isDropTarget })=>{
            return /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsxs)("div", {
                className: $8a2a97a0cb171e69$var$treeCellGrid({
                    isDisabled: isDisabled,
                    isNextSelected: $8a2a97a0cb171e69$var$isNextSelected(id, state),
                    isSelected: isSelected,
                    selectionStyle: selectionStyle
                }),
                children: [
                    /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)("div", {
                        className: $8a2a97a0cb171e69$var$treeRowBackground({
                            isHovered: isHovered,
                            isFocusVisible: isFocusVisible,
                            isSelected: isSelected,
                            selectionStyle: selectionStyle,
                            isNextSelected: $8a2a97a0cb171e69$var$isNextSelected(id, state),
                            isPreviousSelected: $8a2a97a0cb171e69$var$isPrevSelected(id, state),
                            isDropTarget: isDropTarget
                        })
                    }),
                    isFocusVisible && /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)("div", {
                        className: $8a2a97a0cb171e69$var$treeRowFocusRing({
                            isFocusVisible: isFocusVisible,
                            selectionStyle: selectionStyle,
                            isSelected: isSelected,
                            isNextSelected: $8a2a97a0cb171e69$var$isNextSelected(id, state),
                            isFirstItem: $8a2a97a0cb171e69$var$isFirstItem(id, state)
                        })
                    }),
                    allowsDragging && /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)("div", {
                        className: $8a2a97a0cb171e69$var$treeDragButtonContainer,
                        children: !isDisabled && /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $f1f08ef5de95f8eb$exports.DragHandleButton), {
                            isFocusVisibleWithin: isFocusVisibleWithin
                        })
                    }),
                    selectionMode !== 'none' && selectionBehavior === 'toggle' && selectionStyle !== 'highlight' && // TODO: add transition?
                    /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)("div", {
                        className: $8a2a97a0cb171e69$var$treeCheckbox({
                            isDisabled: isDisabled || !state.selectionManager.canSelectItem(id) || state.disabledKeys.has(id)
                        }),
                        children: /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $af04e099a53d3e85$exports.Checkbox), {
                            slot: "selection"
                        })
                    }),
                    /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)("div", {
                        className: " z7fSmKc131 y7fSmKc131 B7fSmKc131 A7fSmKc131 ZecPNad131"
                    }),
                    /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)($8a2a97a0cb171e69$var$ExpandableRowChevron, {
                        isDisabled: isDisabled,
                        isExpanded: isExpanded,
                        scale: scale,
                        isHidden: !hasChildItems
                    }),
                    /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $6K4Mt$reactariacomponentsslots.Provider), {
                        values: [
                            [
                                (0, $6367bc87eb7d24ad$exports.TextContext),
                                {
                                    styles: $8a2a97a0cb171e69$var$treeContent
                                }
                            ],
                            [
                                (0, $bde97c91243ed164$exports.IconContext),
                                {
                                    render: (0, $e991cbcdf82ced71$exports.centerBaseline)({
                                        slot: 'icon',
                                        styles: $8a2a97a0cb171e69$var$treeIcon
                                    }),
                                    styles: " ZuUmpre131 FuUmpre131 _va131"
                                }
                            ],
                            [
                                (0, $7a26131f6144af2b$exports.ActionButtonGroupContext),
                                {
                                    styles: $8a2a97a0cb171e69$var$treeActions,
                                    isDisabled: isDisabled
                                }
                            ],
                            [
                                (0, $e5a758dd286599f2$exports.ActionMenuContext),
                                {
                                    styles: $8a2a97a0cb171e69$var$treeActionMenu,
                                    isQuiet: true,
                                    isDisabled: isDisabled
                                }
                            ]
                        ],
                        children: typeof children === 'string' ? /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $6367bc87eb7d24ad$exports.Text), {
                            children: children
                        }) : children
                    })
                ]
            });
        }
    });
};
const $8a2a97a0cb171e69$var$centeredWrapper = " sd131 eb131 _Ca131 Za131 Fb131";
const $8a2a97a0cb171e69$export$6ceebe2a029205c = (props)=>{
    let { loadingState: loadingState, onLoadMore: onLoadMore } = props;
    let stringFormatter = (0, $6K4Mt$reactariauseLocalizedStringFormatter.useLocalizedStringFormatter)((0, ($parcel$interopDefault($4526404114e78c80$exports))), '@react-spectrum/s2');
    let isLoading = loadingState === 'loading' || loadingState === 'loadingMore';
    return /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $6K4Mt$reactariacomponentsTree.TreeLoadMoreItem), {
        isLoading: isLoading,
        onLoadMore: onLoadMore,
        className: " Za131 Js131 Gs131",
        children: ()=>{
            return /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)("div", {
                className: $8a2a97a0cb171e69$var$centeredWrapper,
                children: /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $1c116ea4b0d2d023$exports.ProgressCircle), {
                    isIndeterminate: true,
                    "aria-label": stringFormatter.format('table.loadingMore')
                })
            });
        }
    });
};
const $8a2a97a0cb171e69$var$expandButton = function anonymous(props) {
    let rules = " ";
    rules += ' z8dW5Qb131';
    rules += ' y8dW5Qb131';
    rules += ' B8dW5Qb131';
    rules += ' A8dW5Qb131';
    if (props.isDisabled) {
        rules += ' pp131';
        rules += ' plc131';
    } else rules += ' p40ub4c131';
    rules += ' Fv131';
    rules += ' Zw131';
    rules += ' sd131';
    rules += ' _wb131';
    rules += ' _cb131';
    rules += ' _Ca131';
    rules += ' _Le131';
    rules += ' ri131';
    if (props.isExpanded) {
        if (props.isRTL) rules += ' _Za131';
        else rules += ' _Zb131';
    }
    rules += ' Td131';
    rules += ' Qd131';
    rules += ' Sd131';
    rules += ' Rd131';
    rules += ' Yd131';
    rules += ' Xb131';
    rules += ' _2b131';
    rules += ' g9131';
    rules += ' _je131';
    rules += ' __ca131';
    if (props.isHidden) rules += ' _6b131';
    return rules;
};
function $8a2a97a0cb171e69$var$ExpandableRowChevron(props1) {
    let expandButtonRef = (0, $6K4Mt$react.useRef)(null);
    let [fullProps, ref] = (0, $6K4Mt$reactariacomponentsslots.useContextProps)({
        ...props1,
        slot: 'chevron'
    }, expandButtonRef, (0, $6K4Mt$reactariacomponentsButton.ButtonContext));
    let { isExpanded: isExpanded, scale: scale, isHidden: isHidden } = fullProps;
    let { direction: direction } = (0, $6K4Mt$reactariaI18nProvider.useLocale)();
    return /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $6K4Mt$reactariacomponentsButton.Button), {
        ...props1,
        ref: ref,
        slot: "chevron",
        className: (renderProps)=>$8a2a97a0cb171e69$var$expandButton({
                ...renderProps,
                isExpanded: isExpanded,
                isRTL: direction === 'rtl',
                scale: scale,
                isHidden: isHidden
            }),
        children: /*#__PURE__*/ (0, $6K4Mt$reactjsxruntime.jsx)((0, $1df9f1c9262ce5df$exports.default), {
            className: function anonymous(props) {
                let rules = " ";
                if (props.direction === "rtl") {
                    rules += ' -Z4Pn4c-a131';
                    rules += ' __Ha131';
                } else if (props.direction === "ltr") {
                    rules += ' -Z4Pn4c-b131';
                    rules += ' __Ha131';
                }
                if (props.direction === "rtl") {
                    rules += ' -_04Pn4c-a131';
                    rules += ' __Ha131';
                } else if (props.direction === "ltr") {
                    rules += ' -_04Pn4c-b131';
                    rules += ' __Ha131';
                }
                rules += ' -_8sjo0b-t5ZbAob131';
                return rules;
            }({
                direction: direction
            })
        })
    });
}
function $8a2a97a0cb171e69$var$isNextSelected(id, state) {
    if (id == null || !state) return false;
    let keyAfter = state.collection.getKeyAfter(id);
    // We need to skip non-item nodes because the selection manager will map non-item nodes to their parent before checking selection
    let node = keyAfter != null ? state.collection.getItem(keyAfter) : null;
    while(node && node.type !== 'item' && keyAfter != null){
        keyAfter = state.collection.getKeyAfter(keyAfter);
        node = keyAfter != null ? state.collection.getItem(keyAfter) : null;
    }
    return keyAfter != null && state.selectionManager.isSelected(keyAfter);
}
function $8a2a97a0cb171e69$var$isPrevSelected(id, state) {
    if (id == null || !state) return false;
    let keyBefore = state.collection.getKeyBefore(id);
    return keyBefore != null && state.selectionManager.isSelected(keyBefore);
}
function $8a2a97a0cb171e69$var$isFirstItem(id, state) {
    if (id == null || !state) return false;
    return state.collection.getFirstKey() === id;
}
