// @react-spectrum/s2/dist/private/useDOMRef.cjs
var $iHEPm$react = require("react");


function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "createFocusableRef", () => $150382364eecc4bc$export$79d69eee6ae4b329);
$parcel$export(module.exports, "useDOMRef", () => $150382364eecc4bc$export$c2c55ef9111cafd8);
$parcel$export(module.exports, "useFocusableRef", () => $150382364eecc4bc$export$96a734597687c040);
/*
 * Copyright 2026 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 
function $150382364eecc4bc$export$a5795cc979dfae80(ref) {
    return {
        UNSAFE_getDOMNode () {
            return ref.current;
        }
    };
}
function $150382364eecc4bc$export$79d69eee6ae4b329(domRef, focusableRef) {
    let resolvedFocusableRef = focusableRef || domRef;
    return {
        ...$150382364eecc4bc$export$a5795cc979dfae80(domRef),
        focus () {
            if (resolvedFocusableRef.current) resolvedFocusableRef.current.focus();
        }
    };
}
function $150382364eecc4bc$export$c2c55ef9111cafd8(ref) {
    let domRef = (0, $iHEPm$react.useRef)(null);
    (0, $iHEPm$react.useImperativeHandle)(ref, ()=>$150382364eecc4bc$export$a5795cc979dfae80(domRef));
    return domRef;
}
function $150382364eecc4bc$export$96a734597687c040(ref, focusableRef) {
    let domRef = (0, $iHEPm$react.useRef)(null);
    (0, $iHEPm$react.useImperativeHandle)(ref, ()=>$150382364eecc4bc$export$79d69eee6ae4b329(domRef, focusableRef));
    return domRef;
}
function $150382364eecc4bc$export$c7e28c72a4823176(ref) {
    return {
        get current () {
            return ref.current && ref.current.UNSAFE_getDOMNode();
        }
    };
}
function $150382364eecc4bc$export$1d5cc31d9d8df817(ref) {
    return (0, $iHEPm$react.useMemo)(()=>$150382364eecc4bc$export$c7e28c72a4823176(ref), [
        ref
    ]);
}
