// @react-spectrum/s2/dist/private/useDOMRef.mjs
import {useRef as $jbe5b$useRef, useImperativeHandle as $jbe5b$useImperativeHandle, useMemo as $jbe5b$useMemo} from "react";

/*
 * Copyright 2026 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */ 
function $80ecb974df713a30$export$a5795cc979dfae80(ref) {
    return {
        UNSAFE_getDOMNode () {
            return ref.current;
        }
    };
}
function $80ecb974df713a30$export$79d69eee6ae4b329(domRef, focusableRef) {
    let resolvedFocusableRef = focusableRef || domRef;
    return {
        ...$80ecb974df713a30$export$a5795cc979dfae80(domRef),
        focus () {
            if (resolvedFocusableRef.current) resolvedFocusableRef.current.focus();
        }
    };
}
function $80ecb974df713a30$export$c2c55ef9111cafd8(ref) {
    let domRef = (0, $jbe5b$useRef)(null);
    (0, $jbe5b$useImperativeHandle)(ref, ()=>$80ecb974df713a30$export$a5795cc979dfae80(domRef));
    return domRef;
}
function $80ecb974df713a30$export$96a734597687c040(ref, focusableRef) {
    let domRef = (0, $jbe5b$useRef)(null);
    (0, $jbe5b$useImperativeHandle)(ref, ()=>$80ecb974df713a30$export$79d69eee6ae4b329(domRef, focusableRef));
    return domRef;
}
function $80ecb974df713a30$export$c7e28c72a4823176(ref) {
    return {
        get current () {
            return ref.current && ref.current.UNSAFE_getDOMNode();
        }
    };
}
function $80ecb974df713a30$export$1d5cc31d9d8df817(ref) {
    return (0, $jbe5b$useMemo)(()=>$80ecb974df713a30$export$c7e28c72a4823176(ref), [
        ref
    ]);
}


export {$80ecb974df713a30$export$a5795cc979dfae80 as createDOMRef, $80ecb974df713a30$export$79d69eee6ae4b329 as createFocusableRef, $80ecb974df713a30$export$c2c55ef9111cafd8 as useDOMRef, $80ecb974df713a30$export$96a734597687c040 as useFocusableRef, $80ecb974df713a30$export$c7e28c72a4823176 as unwrapDOMRef, $80ecb974df713a30$export$1d5cc31d9d8df817 as useUnwrapDOMRef};
